<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evo-devo dynamics of hominin brain size
Authors: Gonzalez-Forero, M.
Score: 42.8, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421
Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Evo-devo dynamics of hominin brain size
Authors: Gonzalez-Forero, M.
Score: 42.8, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421
Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Evo-devo dynamics of hominin brain size
Authors: Gonzalez-Forero, M.
Score: 42.8, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421
Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Evo-devo dynamics of hominin brain size\nAuthors: Gonzalez-Forero, M.\nScore: 42.8, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421\nBrain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution.",
  "keywords": [
    
  ],
  "articleBody": " Evo-devo dynamics of hominin brain size\nAuthors: Gonzalez-Forero, M.\nScore: 42.8, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421\nBrain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution. Analysis shows that in this model the brain expands because it is \"socio-genetically\" correlated with developmentally late preovulatory ovarian follicles, not because brain size is directly selected for. The socio-genetic correlation causing the recovered hominin brain expansion is generated over development by ecology and possibly culture. Thus, in this model, direct selection that does not favour brain expansion provides a force that developmental constraints divert causing hominin-brain expansion.\nLarge-scale analyses reveal the contribution of adaptive evolution in pathogenic and non-pathogenic fungal species\nAuthors: Pereira, D.; Bolton, M. D.; Friesen, T. L.; Stephan, W.; Dutheil, J. Y.; Stukenbrock, E. H.\nScore: 23.6, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555124\nGenome studies of fungal pathogens have presented evidence for exceptionally high rates of evolution. It has been proposed that rapid adaptation is a hallmark of pathogen evolution that facilitates the invasion of new host niches and the overcoming of intervention strategies such as fungicide applications and drug treatments. To which extent high levels of genetic variation within and between species correlate with adaptive protein evolution in fungi more generally has so far not been explored. In this study, we addressed the contribution of adaptive evolution relative to genetic drift in 20 fungal species, hereby exploring genetic variation in 2,478 fungal genomes. We reannotated positions of protein-coding genes to obtain a high-quality dataset of 234,427 full-length core gene and 25,612 accessory gene alignments. We applied an extension of the McDonald-Kreitman test that models the distributions of fitness effects to infer the rate of adaptive ({omega}A) and non-adaptive ({omega}NA) non-synonymous substitutions in protein-coding genes. To explore the relevance of recombination on local adaptation rates, we inferred the population genomic recombination rate for all 20 species. Our analyses reveal extensive variation in rates of adaptation and show that high rates of adaptation are not a hallmark of a pathogenic lifestyle. Up to 83% of non-synonymous substitutions are adaptive in the species Parastagonospora nodorum. However, non-synonymous substitutions in other species, including the prominent rice-infecting pathogen Magnaporthe oryzae, are predominantly non-adaptive (neutral or slightly deleterious). Correlating adaptation measures with effective population size and recombination rate, we show that effective population size is a primary determinant of adaptive evolution in fungi. At the genome scale, recombination rate variation explains variation in both {omega}A and {omega}NA. Finally, we demonstrate the robustness of our estimates using simulations. We underline the value of population genetic principles in studies of fungal evolution, and we highlight the importance of demographic processes in adaptive evolution of pathogenic and non-pathogenic species.\nAn efficient and robust ABC approach to infer the rate and strength of adaptation\nAuthors: Murga-Moreno, J.; Casillas, S.; Barbadilla, A.; Uricchio, L.; Enard, D.\nScore: 16.2, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555322\nInferring the effects of positive selection on genomes remains a critical step in characterizing the ultimate and proximate causes of adaptation across species, and quantifying positive selection remains a challenge due to the confounding effects of many other evolutionary processes. Robust and efficient approaches for adaptation inference could help characterize the rate and strength of adaptation in non-model species for which demographic history, mutational processes, and recombination patterns are not currently well-described. Here, we introduce an efficient and user-friendly extension of of the McDonald-Kreitman test (ABC-MK) for quantifying long-term protein adaptation in specific lineages of interest. We characterize the performance of our approach with forward simulations and find that it is robust to many demographic perturbations and positive selection configurations, demonstrating its suitability for applications to non-model genomes. We apply ABC-MK to the human proteome and a set of known Virus Interacting Proteins (VIPs) to test long-term adaptation in VIPs, testing for specific pathogenic players associated with human evolution. We find substantially stronger signatures of positive selection on RNA-VIPs than DNA-VIPs, suggesting that RNA viruses may be may be an important driver of human adaptation over deep evolutionary time scales.\nA cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae\nAuthors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.\nScore: 23.8, Published: 2023-08-29 DOI: 10.1101/2023.08.24.554571\nWe provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles, and yet these two muscle cell phenotypes are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians. TeaserSingle cell transcriptomic atlas across the jellyfish life cycle reveals increased cellular complexity associated with the medusa form.\nHuman deleterious mutation rate implies high fitness variance, with declining mean fitness compensated by rarer beneficial mutations of larger effect\nAuthors: Matheson, J. D.; Masel, J.; Bertram, J.\nScore: 13.7, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555871\nEach new human has an expected U_d=2-10 new deleterious mutations. This deluge of deleterious mutations cannot all be purged, and therefore accumulate in a declining fitness ratchet. Using a novel simulation framework designed to efficiently handle genome-wide linkage disequilibria across many segregating sites, we find that rarer, beneficial mutations of larger effect are sufficient to compensate fitness declines due to the fixation of many slightly deleterious mutations. Drift barrier theory posits a similar asymmetric pattern of fixations to explain ratcheting genome size and complexity, but in our theory, the cause is U_d\u003e1 rather than small population size. In our simulations, U_d~2-10 generates high within-population variance in relative fitness; two individuals will typically differ in fitness by 15-40%. U_d~2-10 also slows net adaptation by ~13%-39%. Surprisingly, fixation rates are more sensitive to changes in the beneficial than the deleterious mutation rate, e.g. a 10% increase in overall mutation rate leads to faster adaptation; this puts to rest dysgenic fears about increasing mutation rates due to rising paternal age.\nRemote homolog detection places insect chemoreceptors in a cryptic protein superfamily spanning the tree of life\nAuthors: Himmel, N. J.; Moi, D.; Benton, R.\nScore: 12.9, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555482\nMany proteins exist in the so-called 'twilight zone' of sequence alignment, where low pairwise sequence identity makes it difficult to determine homology and phylogeny. As protein tertiary structure is often more conserved, recent advances in ab initio protein folding have made structure-based identification of putative homologs feasible. However, structural screening and phylogenetics are in their infancy, particularly for twilight zone proteins. We present a pipeline for the identification and characterization of distant homologs, and apply it to 7-transmembrane domain ion channels (7TMICs), a protein group founded by insect Odorant and Gustatory receptors. Previous sequence and limited structure-based searches identified putatively-related proteins, mainly in other animals and plants. However, very few 7TMICs have been identified in non-animal, non-plant taxa. Moreover, these proteins' remarkable sequence dissimilarity made it uncertain if disparate 7TMIC types (Gr/Or, Grl, GRL, DUF3537, PHTF and GrlHz) are homologous or convergent, leaving their evolutionary history unresolved. Our pipeline identified thousands of new 7TMICs in archaea, bacteria and unicellular eukaryotes. Using graph-based analyses and protein language models to extract family-wide signatures, we demonstrate that 7TMICs have structure and sequence similarity, supporting homology. Through sequence and structure-based phylogenetics, we classify eukaryotic 7TMICs into two families (Class-A and Class-B), which are the result of a gene duplication predating the split(s) leading to Amorphea (animals, fungi and allies) and Diaphoretickes (plants and allies). Our work reveals 7TMICs as a cryptic superfamily with origins close to the evolution of cellular life. More generally, this study serves as a methodological proof of principle for the identification of extremely distant protein homologs.\nUncovering the mosaic evolution of carnivoran skeletal systems\nAuthors: Law, C. J.; Hlusko, L. J.; Tseng, J.\nScore: 11.9, Published: 2023-09-04 DOI: 10.1101/2023.09.03.556127\nThe diversity of vertebrate skeletal forms is often attributed to adaptations to distinct ecological factors such as diet, locomotion, and sensory environment. Although the adaptive evolution of cranial, appendicular, and vertebral skeletal systems is well studied in vertebrates, comprehensive investigations of all skeletal components are rarely performed simultaneously. Consequently, we know little of how modes of evolution differ among skeletal components. We collected 103 linear measurements from 208 osteological museum specimens to capture skeletal variation across the major components of the skeletal systems of 119 extant carnivores. With these data, we tested if ecological and phylogenetic effects led to distinct modes of evolution among the cranial, appendicular, and vertebral skeletal regions by fitting multivariate evolutionary models. We found mosaic evolution in which only the mandible, hindlimb, and posterior region of the vertebral column showed evidence of adaptation towards ecological regimes whereas the remaining skeletal components followed clade-specific evolutionary shifts. We hypothesize that the decoupled evolution of individual skeletal components may have led to the origination of distinct adaptive zones and morphologies among extant carnivoran families that reflect phylogenetic hierarchies. Overall, our work highlights the importance of examining multiple skeletal components simultaneously in ecomorphological analyses. Ongoing work integrating the fossil and paleoenvironmental record will further govern carnivoran diversity we see today and reveal the complexity of evolutionary processes in multicomponent systems.\nWhat is the best method for estimating ancestral states from discrete characters?\nAuthors: Keating, J. N.\nScore: 10.9, Published: 2023-09-02 DOI: 10.1101/2023.08.31.555762\nAncestral state estimation is a formal phylogenetic method for inferring the nature of ancestors and performing tests of character evolution. As such, it is among the most important tools available to evolutionary biologists. However, there are a profusion of methods available, the accuracy of which remains unclear. Here I use a simulation approach to test between parsimony and likelihood methods for estimating ancestral states from discrete binary characters. I simulate 500 characters using 15 different Markov generating models, a range of tree sizes (8-256 tips) and three topologies representing end members of tree symmetry and branch length heterogeneity. Simulated tip states were subject to ancestral state estimation under the Equal Rates (ER) and All-Rates-Different (ARD) models, as well as under parsimony assuming accelerated transformations (ACCTRAN). The results demonstrate that both parsimony and likelihood approaches obtain high accuracy applied to trees with more tips. Parsimony performs poorly when trees contain long branches, whereas the ER model performs well across simulations and is reasonably robust to model violation. The ER model frequently outperforms the ARD model, even when data are simulated using unequal rates. Furthermore, the ER model exhibits less transition rate error when compared to ER models. These results suggest that ARD models may be overparameterized when character data is limited. Surprisingly, difference in error between likelihood models is a poor predictor of difference in model fitness; better fitting models are not necessarily more accurate. However there is a strong correlation between model uncertainty and model error; likelihood models with more certain ancestral state estimates are typically more accurate. Using empirical morphological datasets, I demonstrate that applying different methods often results in substantively different ancestral state estimates. However, ER ancestral state estimates are frequently supported by either parsimony or ARD estimates. Finally, I highlight the importance of incorporating fossils in ancestral state estimation. Fossils increase the total number of tips, break long branches and lower average branch length and overall branch length heterogeneity of trees. These factors will all contribute to increasing the accuracy of ancestral state estimates, irrespective of the method used.\nChloramphenicol reduces phage resistance evolution by suppressing bacterial cell surface mutants\nAuthors: Parab, L.; Romeyer Dherbey, J.; Rivera, N.; Schwarz, M.; Bertels, F.\nScore: 11.5, Published: 2023-08-28 DOI: 10.1101/2023.08.28.552763\nBacteriophages infect Gram-negative bacteria by attaching to molecules present on the bacterial outer membrane, often lipopolysaccharides (LPS). Modification of the LPS can lead to phage resistance. LPS modifications also impact antibiotic susceptibility, allowing for phage-antibiotic synergism. The mechanism for these synergistic interactions is unclear. Here, we show that antibiotics affect the evolution of phage resistance using phage {Phi}X174 and Escherichia coli C wildtype. We use a collection of E. coli C LPS mutants, each of which is resistant to {Phi}X174, and has either a \"rough\" or \"deep-rough\" LPS phenotype. Growth of deep rough mutants is inhibited at subinhibitory chloramphenicol concentrations. In contrast, gentamicin has no major effect on growth. Hypothesis testing shows that treating E. coli C wildtype with {Phi}X174 and chloramphenicol eliminates deep rough mutants, and reduces phage resistance evolution. Our results show that differential survival of phage resistant mutants with antibiotics explains phage-antibiotic synergism in our model system.\nGenetic basis of variation in thermal developmental plasticity for Drosophila melanogaster body pigmentation\nAuthors: Lafuente, E.; Duneau, D. F.; Beldade, P.\nScore: 8.3, Published: 2023-09-03 DOI: 10.1101/2023.09.03.556093\nSeasonal differences in insect pigmentation are attributed to the influence of ambient temperature on pigmentation development. This thermal plasticity is adaptive and heritable, thereby capable of evolving. However, the specific genes contributing to the variation in plasticity that can drive its evolution remain largely unknown. To address this, we analyzed pigmentation and pigmentation plasticity in Drosophila melanogaster. We measured two components of pigmentation in the thorax and abdomen: overall darkness and the proportion of length covered by darker pattern elements (a trident in the thorax and bands in the abdomen) in females from two developmental temperatures (17{degrees}C or 28{degrees}C) and 191 genotypes. Using a GWAS approach to identify the genetic basis of variation in pigmentation and its response to temperature, we identified numerous dispersed QTLs, including some mapping to melanogenesis genes (yellow, ebony, and tan). Remarkably, we observed limited overlap between QTLs for variation within specific temperatures and those influencing thermal plasticity, as well as minimal overlap between plasticity QTLs across pigmentation components and across body parts. For most traits, consistent with selection favoring the retention of plasticity, we found that lower plasticity alleles were often at lower frequencies. The functional analysis of selected candidate QTLs and pigmentation genes largely confirmed their contributions to variation in pigmentation and/or pigmentation plasticity. Overall, our study reveals the existence and underlying basis of extensive and trait-specific genetic variation for pigmentation and pigmentation plasticity, offering a rich reservoir of raw material for natural selection to shape the independent evolution of these traits.\n",
  "wordCount" : "2560",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.20.533421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.20.533421" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.20.533421">
        <p class="paperTitle">Evo-devo dynamics of hominin brain size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.20.533421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.20.533421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Forero, M.</p>
        <p class="info">Score: 42.8, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.20.533421' target='https://doi.org/10.1101/2023.03.20.533421'> 10.1101/2023.03.20.533421</a></p>
        <p class="abstract">Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution. Analysis shows that in this model the brain expands because it is &#34;socio-genetically&#34; correlated with developmentally late preovulatory ovarian follicles, not because brain size is directly selected for. The socio-genetic correlation causing the recovered hominin brain expansion is generated over development by ecology and possibly culture. Thus, in this model, direct selection that does not favour brain expansion provides a force that developmental constraints divert causing hominin-brain expansion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555124">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555124" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555124">
        <p class="paperTitle">Large-scale analyses reveal the contribution of adaptive evolution in pathogenic and non-pathogenic fungal species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555124" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555124" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pereira, D.; Bolton, M. D.; Friesen, T. L.; Stephan, W.; Dutheil, J. Y.; Stukenbrock, E. H.</p>
        <p class="info">Score: 23.6, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555124' target='https://doi.org/10.1101/2023.08.28.555124'> 10.1101/2023.08.28.555124</a></p>
        <p class="abstract">Genome studies of fungal pathogens have presented evidence for exceptionally high rates of evolution. It has been proposed that rapid adaptation is a hallmark of pathogen evolution that facilitates the invasion of new host niches and the overcoming of intervention strategies such as fungicide applications and drug treatments. To which extent high levels of genetic variation within and between species correlate with adaptive protein evolution in fungi more generally has so far not been explored. In this study, we addressed the contribution of adaptive evolution relative to genetic drift in 20 fungal species, hereby exploring genetic variation in 2,478 fungal genomes. We reannotated positions of protein-coding genes to obtain a high-quality dataset of 234,427 full-length core gene and 25,612 accessory gene alignments. We applied an extension of the McDonald-Kreitman test that models the distributions of fitness effects to infer the rate of adaptive ({omega}A) and non-adaptive ({omega}NA) non-synonymous substitutions in protein-coding genes. To explore the relevance of recombination on local adaptation rates, we inferred the population genomic recombination rate for all 20 species. Our analyses reveal extensive variation in rates of adaptation and show that high rates of adaptation are not a hallmark of a pathogenic lifestyle. Up to 83% of non-synonymous substitutions are adaptive in the species Parastagonospora nodorum. However, non-synonymous substitutions in other species, including the prominent rice-infecting pathogen Magnaporthe oryzae, are predominantly non-adaptive (neutral or slightly deleterious). Correlating adaptation measures with effective population size and recombination rate, we show that effective population size is a primary determinant of adaptive evolution in fungi. At the genome scale, recombination rate variation explains variation in both {omega}A and {omega}NA. Finally, we demonstrate the robustness of our estimates using simulations. We underline the value of population genetic principles in studies of fungal evolution, and we highlight the importance of demographic processes in adaptive evolution of pathogenic and non-pathogenic species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555322">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555322" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555322">
        <p class="paperTitle">An efficient and robust ABC approach to infer the rate and strength of adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555322" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555322" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murga-Moreno, J.; Casillas, S.; Barbadilla, A.; Uricchio, L.; Enard, D.</p>
        <p class="info">Score: 16.2, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555322' target='https://doi.org/10.1101/2023.08.29.555322'> 10.1101/2023.08.29.555322</a></p>
        <p class="abstract">Inferring the effects of positive selection on genomes remains a critical step in characterizing the ultimate and proximate causes of adaptation across species, and quantifying positive selection remains a challenge due to the confounding effects of many other evolutionary processes. Robust and efficient approaches for adaptation inference could help characterize the rate and strength of adaptation in non-model species for which demographic history, mutational processes, and recombination patterns are not currently well-described. Here, we introduce an efficient and user-friendly extension of of the McDonald-Kreitman test (ABC-MK) for quantifying long-term protein adaptation in specific lineages of interest. We characterize the performance of our approach with forward simulations and find that it is robust to many demographic perturbations and positive selection configurations, demonstrating its suitability for applications to non-model genomes. We apply ABC-MK to the human proteome and a set of known Virus Interacting Proteins (VIPs) to test long-term adaptation in VIPs, testing for specific pathogenic players associated with human evolution. We find substantially stronger signatures of positive selection on RNA-VIPs than DNA-VIPs, suggesting that RNA viruses may be may be an important driver of human adaptation over deep evolutionary time scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554571">
        <p class="paperTitle">A cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.</p>
        <p class="info">Score: 23.8, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554571' target='https://doi.org/10.1101/2023.08.24.554571'> 10.1101/2023.08.24.554571</a></p>
        <p class="abstract">We provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles, and yet these two muscle cell phenotypes are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians.

TeaserSingle cell transcriptomic atlas across the jellyfish life cycle reveals increased cellular complexity associated with the medusa form.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555871">
        <p class="paperTitle">Human deleterious mutation rate implies high fitness variance, with declining mean fitness compensated by rarer beneficial mutations of larger effect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matheson, J. D.; Masel, J.; Bertram, J.</p>
        <p class="info">Score: 13.7, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555871' target='https://doi.org/10.1101/2023.09.01.555871'> 10.1101/2023.09.01.555871</a></p>
        <p class="abstract">Each new human has an expected U_d=2-10 new deleterious mutations. This deluge of deleterious mutations cannot all be purged, and therefore accumulate in a declining fitness ratchet. Using a novel simulation framework designed to efficiently handle genome-wide linkage disequilibria across many segregating sites, we find that rarer, beneficial mutations of larger effect are sufficient to compensate fitness declines due to the fixation of many slightly deleterious mutations. Drift barrier theory posits a similar asymmetric pattern of fixations to explain ratcheting genome size and complexity, but in our theory, the cause is U_d&gt;1 rather than small population size. In our simulations, U_d~2-10 generates high within-population variance in relative fitness; two individuals will typically differ in fitness by 15-40%. U_d~2-10 also slows net adaptation by ~13%-39%. Surprisingly, fixation rates are more sensitive to changes in the beneficial than the deleterious mutation rate, e.g. a 10% increase in overall mutation rate leads to faster adaptation; this puts to rest dysgenic fears about increasing mutation rates due to rising paternal age.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555482">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555482" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555482">
        <p class="paperTitle">Remote homolog detection places insect chemoreceptors in a cryptic protein superfamily spanning the tree of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555482" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555482" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Himmel, N. J.; Moi, D.; Benton, R.</p>
        <p class="info">Score: 12.9, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555482' target='https://doi.org/10.1101/2023.08.30.555482'> 10.1101/2023.08.30.555482</a></p>
        <p class="abstract">Many proteins exist in the so-called &#39;twilight zone&#39; of sequence alignment, where low pairwise sequence identity makes it difficult to determine homology and phylogeny. As protein tertiary structure is often more conserved, recent advances in ab initio protein folding have made structure-based identification of putative homologs feasible. However, structural screening and phylogenetics are in their infancy, particularly for twilight zone proteins. We present a pipeline for the identification and characterization of distant homologs, and apply it to 7-transmembrane domain ion channels (7TMICs), a protein group founded by insect Odorant and Gustatory receptors. Previous sequence and limited structure-based searches identified putatively-related proteins, mainly in other animals and plants. However, very few 7TMICs have been identified in non-animal, non-plant taxa. Moreover, these proteins&#39; remarkable sequence dissimilarity made it uncertain if disparate 7TMIC types (Gr/Or, Grl, GRL, DUF3537, PHTF and GrlHz) are homologous or convergent, leaving their evolutionary history unresolved. Our pipeline identified thousands of new 7TMICs in archaea, bacteria and unicellular eukaryotes. Using graph-based analyses and protein language models to extract family-wide signatures, we demonstrate that 7TMICs have structure and sequence similarity, supporting homology. Through sequence and structure-based phylogenetics, we classify eukaryotic 7TMICs into two families (Class-A and Class-B), which are the result of a gene duplication predating the split(s) leading to Amorphea (animals, fungi and allies) and Diaphoretickes (plants and allies). Our work reveals 7TMICs as a cryptic superfamily with origins close to the evolution of cellular life. More generally, this study serves as a methodological proof of principle for the identification of extremely distant protein homologs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556127">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556127" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556127">
        <p class="paperTitle">Uncovering the mosaic evolution of carnivoran skeletal systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556127" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556127" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Law, C. J.; Hlusko, L. J.; Tseng, J.</p>
        <p class="info">Score: 11.9, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556127' target='https://doi.org/10.1101/2023.09.03.556127'> 10.1101/2023.09.03.556127</a></p>
        <p class="abstract">The diversity of vertebrate skeletal forms is often attributed to adaptations to distinct ecological factors such as diet, locomotion, and sensory environment. Although the adaptive evolution of cranial, appendicular, and vertebral skeletal systems is well studied in vertebrates, comprehensive investigations of all skeletal components are rarely performed simultaneously. Consequently, we know little of how modes of evolution differ among skeletal components. We collected 103 linear measurements from 208 osteological museum specimens to capture skeletal variation across the major components of the skeletal systems of 119 extant carnivores. With these data, we tested if ecological and phylogenetic effects led to distinct modes of evolution among the cranial, appendicular, and vertebral skeletal regions by fitting multivariate evolutionary models. We found mosaic evolution in which only the mandible, hindlimb, and posterior region of the vertebral column showed evidence of adaptation towards ecological regimes whereas the remaining skeletal components followed clade-specific evolutionary shifts. We hypothesize that the decoupled evolution of individual skeletal components may have led to the origination of distinct adaptive zones and morphologies among extant carnivoran families that reflect phylogenetic hierarchies. Overall, our work highlights the importance of examining multiple skeletal components simultaneously in ecomorphological analyses. Ongoing work integrating the fossil and paleoenvironmental record will further govern carnivoran diversity we see today and reveal the complexity of evolutionary processes in multicomponent systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555762">
        <p class="paperTitle">What is the best method for estimating ancestral states from discrete characters?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keating, J. N.</p>
        <p class="info">Score: 10.9, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555762' target='https://doi.org/10.1101/2023.08.31.555762'> 10.1101/2023.08.31.555762</a></p>
        <p class="abstract">Ancestral state estimation is a formal phylogenetic method for inferring the nature of ancestors and performing tests of character evolution. As such, it is among the most important tools available to evolutionary biologists. However, there are a profusion of methods available, the accuracy of which remains unclear. Here I use a simulation approach to test between parsimony and likelihood methods for estimating ancestral states from discrete binary characters. I simulate 500 characters using 15 different Markov generating models, a range of tree sizes (8-256 tips) and three topologies representing end members of tree symmetry and branch length heterogeneity. Simulated tip states were subject to ancestral state estimation under the Equal Rates (ER) and All-Rates-Different (ARD) models, as well as under parsimony assuming accelerated transformations (ACCTRAN). The results demonstrate that both parsimony and likelihood approaches obtain high accuracy applied to trees with more tips. Parsimony performs poorly when trees contain long branches, whereas the ER model performs well across simulations and is reasonably robust to model violation. The ER model frequently outperforms the ARD model, even when data are simulated using unequal rates. Furthermore, the ER model exhibits less transition rate error when compared to ER models. These results suggest that ARD models may be overparameterized when character data is limited. Surprisingly, difference in error between likelihood models is a poor predictor of difference in model fitness; better fitting models are not necessarily more accurate. However there is a strong correlation between model uncertainty and model error; likelihood models with more certain ancestral state estimates are typically more accurate. Using empirical morphological datasets, I demonstrate that applying different methods often results in substantively different ancestral state estimates. However, ER ancestral state estimates are frequently supported by either parsimony or ARD estimates. Finally, I highlight the importance of incorporating fossils in ancestral state estimation. Fossils increase the total number of tips, break long branches and lower average branch length and overall branch length heterogeneity of trees. These factors will all contribute to increasing the accuracy of ancestral state estimates, irrespective of the method used.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.552763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.552763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.552763">
        <p class="paperTitle">Chloramphenicol reduces phage resistance evolution by suppressing bacterial cell surface mutants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.552763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.552763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parab, L.; Romeyer Dherbey, J.; Rivera, N.; Schwarz, M.; Bertels, F.</p>
        <p class="info">Score: 11.5, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.552763' target='https://doi.org/10.1101/2023.08.28.552763'> 10.1101/2023.08.28.552763</a></p>
        <p class="abstract">Bacteriophages infect Gram-negative bacteria by attaching to molecules present on the bacterial outer membrane, often lipopolysaccharides (LPS). Modification of the LPS can lead to phage resistance. LPS modifications also impact antibiotic susceptibility, allowing for phage-antibiotic synergism. The mechanism for these synergistic interactions is unclear. Here, we show that antibiotics affect the evolution of phage resistance using phage {Phi}X174 and Escherichia coli C wildtype. We use a collection of E. coli C LPS mutants, each of which is resistant to {Phi}X174, and has either a &#34;rough&#34; or &#34;deep-rough&#34; LPS phenotype. Growth of deep rough mutants is inhibited at subinhibitory chloramphenicol concentrations. In contrast, gentamicin has no major effect on growth. Hypothesis testing shows that treating E. coli C wildtype with {Phi}X174 and chloramphenicol eliminates deep rough mutants, and reduces phage resistance evolution. Our results show that differential survival of phage resistant mutants with antibiotics explains phage-antibiotic synergism in our model system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556093">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556093" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556093">
        <p class="paperTitle">Genetic basis of variation in thermal developmental plasticity for Drosophila melanogaster body pigmentation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556093" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556093" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lafuente, E.; Duneau, D. F.; Beldade, P.</p>
        <p class="info">Score: 8.3, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556093' target='https://doi.org/10.1101/2023.09.03.556093'> 10.1101/2023.09.03.556093</a></p>
        <p class="abstract">Seasonal differences in insect pigmentation are attributed to the influence of ambient temperature on pigmentation development. This thermal plasticity is adaptive and heritable, thereby capable of evolving. However, the specific genes contributing to the variation in plasticity that can drive its evolution remain largely unknown. To address this, we analyzed pigmentation and pigmentation plasticity in Drosophila melanogaster. We measured two components of pigmentation in the thorax and abdomen: overall darkness and the proportion of length covered by darker pattern elements (a trident in the thorax and bands in the abdomen) in females from two developmental temperatures (17{degrees}C or 28{degrees}C) and 191 genotypes. Using a GWAS approach to identify the genetic basis of variation in pigmentation and its response to temperature, we identified numerous dispersed QTLs, including some mapping to melanogenesis genes (yellow, ebony, and tan). Remarkably, we observed limited overlap between QTLs for variation within specific temperatures and those influencing thermal plasticity, as well as minimal overlap between plasticity QTLs across pigmentation components and across body parts. For most traits, consistent with selection favoring the retention of plasticity, we found that lower plasticity alleles were often at lower frequencies. The functional analysis of selected candidate QTLs and pigmentation genes largely confirmed their contributions to variation in pigmentation and/or pigmentation plasticity. Overall, our study reveals the existence and underlying basis of extensive and trait-specific genetic variation for pigmentation and pigmentation plasticity, offering a rich reservoir of raw material for natural selection to shape the independent evolution of these traits.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
