<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects
Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.
Score: 73.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects
Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.
Score: 73." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects
Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.
Score: 73."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world\u0026#39;s most common underwater architects\nAuthors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.\nScore: 73.",
  "keywords": [
    
  ],
  "articleBody": " Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world's most common underwater architects\nAuthors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.\nScore: 73.2, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572910\nCaddisflies (Trichoptera) are among the most diverse groups of freshwater animals, with more than 16,000 described species. They play an outsized role in freshwater ecology and environmental engineering in streams, rivers, and lakes. Because of this, they are frequently used as indicator organisms in biomonitoring programs. Despite their importance, key questions concerning the evolutionary history of caddisflies, such as the timing and origin of larval case-making, have been unanswered due to the lack of a well-resolved phylogenetic tree. To shed light on these questions in Trichoptera, we estimated a phylogenetic tree using a combination of transcriptomes and targeted enrichment data for 206 species, representing 48 of 52 extant families and 174 genera. We calibrated and dated the tree with a set of 33 carefully selected fossils. The first caddisflies originated in the Permian, and the major suborders began to diversify in the Triassic. Ancestral state reconstruction and diversification analysis revealed that portable case-making evolved in three separate lineages, and shifts in diversification occurred in concert with key evolutionary innovations other than case-making.\nDistinct ancient structural polymorphisms control heterodichogamy in walnuts and hickories\nAuthors: Groh, J. S.; Vik, D. C.; Stevens, K. A.; Brown, P. J.; Langley, C. H.; Coop, G.\nScore: 16.4, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573205\nThe maintenance of stable mating type polymorphisms is a classic example of balancing selection, underlying the nearly ubiquitous 50/50 sex ratio in species with separate sexes. One lesser known but intriguing example of a balanced mating polymorphism in angiosperms is heterodichogamy - polymorphism for opposing directions of dichogamy (temporal separation of male and female function in hermaphrodites) within a flowering season. This mating system is common throughout Juglandaceae, the family that includes globally important and iconic nut and timber crops - walnuts (), as well as pecan and other hickories (). We show that two distinct genetic systems control heterodichogamous mating systems in these genera. In each genus, we find ancient (\u003e50Mya) structural variants that segregate as trans-species polymorphisms. The locus maps to a ca. 20kb structural variant adjacent to gene with likely function in the trehalose-6-phosphate signaling pathway, which is known to regulate flowering time in Arabidopsis and inflorescence architecture in maize. Notably, an insertion in the dominant (protogynous) haplotype contains 8-12 complex tandem duplicates of the 3' UTR of the gene, and transpecies polymorphism within the gene is restricted to the 3' UTR. In contrast, the heterodichogamy locus maps to a ca. 200-450 kb cluster of tightly linked polymorphisms across 20 genes, some of which have known roles in flowering and are differentially expressed between morphs in floral primordia. The dominant (protogynous) haplotype in pecan, which is nearly always heterozygous and appears to rarely recombine, shows reduced genetic diversity and is over twice as long as its recessive counterpart due to proliferation of multiple transposable element families. We did not detect either genetic system in another heterodichogamous genus within the Juglandaceae, suggesting that additional genetic systems for heterodichogamy may yet remain undiscovered.\nEvolution of Multicellularity Genes in the Lead Up to the Great Oxidation Event\nAuthors: Boden, J. S.; Nieves-Morion, M.; Nurnberg, D. J.; Arevalo, S.; Flores, E.; Sanchez-Baracaldo, P.\nScore: 14.2, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573081\nCyanobacteria are among the most morphologically diverse prokaryotic phyla on Earth. Their morphotypes range from unicellular to multicellular filaments, yet mechanisms underlying the evolution of filamentous morphologies remain unknown. Here, we implement phylogenomic, Bayesian molecular clock and gene-tree-species-tree reconciliation analyses to estimate when genes encoding cell-cell joining structures first evolved. We also characterise septal structures and measure intercellular communication rates in non-model and early-branching filamentous strains. Our results suggest that genes encoding septal proteins (namely sepJ, sepI, and fraE) and potentially pattern formation (hetR) evolved in the Neoarchaean 2.6-2.7 billion years (Ga) ago. Later, at the start of the Great Oxygenation Event approximately 2.5 Ga, genes involved in cellular differentiation (namely hetZ, patU3 and hglK) appeared. Our results predict that early-branching lineages like Pseudanabaena were capable of intercellular communication, but further innovations in cellular differentiation were needed to drive ecological expansion on a scale large enough to permanently oxygenate Earth's atmosphere.\nAncestral neural circuits potentiate the origin of a female sexual behavior\nAuthors: Li, M.; Chen, D. S.; Junker, I. P.; Szorenyi, F. I.; Chen, G. H.; Berger, A. J.; Comeault, A. A.; Matute, D. R.; Ding, Y.\nScore: 13.0, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570174\nCourtship interactions are remarkably diverse in form and complexity among species. How neural circuits evolve to encode new behaviors that are functionally integrated into these dynamic social interactions is unknown. Here we report a recently originated female sexual behavior in the island endemic Drosophila species D. santomea, where females signal receptivity to male courtship songs by spreading their wings, which in turn promotes prolonged songs in courting males. Copulation success depends on this female signal and correlates with males ability to adjust his singing in such a social feedback loop. Functional comparison of sexual circuitry across species suggests that a pair of descending neurons, which integrates male song stimuli and female internal state to control a conserved female abdominal behavior, drives wing spreading in D. santomea. This co-option occurred through the refinement of a pre-existing, plastic circuit that can be optogenetically activated in an outgroup species. Combined, our results show that the ancestral potential of a socially-tuned key circuit node to engage the wing motor program facilitates the expression of a new female behavior in appropriate sensory and motivational contexts. More broadly, our work provides insights into the evolution of social behaviors, particularly female behaviors, and the underlying neural mechanisms.\nEvolution in microbial microcosms is highly parallel regardless of the presence of interacting species\nAuthors: Meroz, N.; Livny, T.; Toledano, G.; Sorokin, Y.; Tovi, N.; Friedman, J.\nScore: 19.1, Published: 2023-12-14 DOI: 10.1101/2023.12.14.571477\nDuring laboratory evolution, replicate bacterial populations often follow similar trajectories, thus their evolution is potentially predictable. However, predicting the evolution of natural populations, which are commonly embedded in multispecies communities, would prove extremely difficult if adaptations are contingent on the identity of the interacting species. The extent to which adaptations typically depend on coevolving partners remains poorly understood, since coevolution is commonly studied using small-scale experiments involving few species, making it challenging to extract general trends. To address this knowledge gap, we study the adaptations that occurred in strains of each of 11 species that were either evolved in monoculture or in multiple pairwise co-cultures. While we detect slight but significant partner-specific effects we find that the majority of evolutionary changes that occur are robust across strains that evolved with different partners; species growth abilities increase by a similar factor regardless of partners identity, shifts in community compositions and interactions are similar between pairs of coevolved and separately evolved strains, and the majority of parallelly mutated genes were detected in multiple biotic conditions. We hypothesized that these results might arise from the fact that ancestral strains are maladapted to the abiotic environment, thus having a pool of adaptations that are beneficial regardless of the biotic partners. Therefore, we conducted a second experiment with strains that were pre-adapted to the abiotic conditions before being co-cultured. We find that even after [~]400 generations of pre-adaptation, evolution is surprisingly non-partner-specific. Further work is required in order to elucidate the factors that influence partner-specificity of coevolution, but our results suggest that selection imposed by the biotic environment may play a secondary role to that imposed by abiotic conditions, making predictions regarding coevolutionary dynamics less challenging than previously thought.\nEpiFusion: Joint inference of the effective reproduction number by integrating phylodynamic and epidemiological modelling with particle filtering\nAuthors: Judge, C.; Vaughan, T.; Russell, T.; Abbott, S.; du Plessis, L.; Stadler, T.; Brady, O.; Hill, S.\nScore: 9.7, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572106\nAccurately estimating the effective reproduction number (Rt) of a circulating pathogen is a fundamental challenge in the study of infectious disease. The fields of epidemiology and pathogen phylodynamics both share this goal, but to date, methodologies and data employed by each remain largely distinct. Here we present EpiFusion: a joint approach that can be used to harness the complementary strengths of each field to improve estimation of outbreak dynamics for large and poorly sampled epidemics, such as arboviral or respiratory outbreaks, and validate it for retrospective analysis. We propose a model of Rt that estimates outbreak trajectories conditional upon both phylodynamic (time-scaled trees estimated from genetic sequences) and epidemiological (case incidence) data. We simulate stochastic outbreak trajectories that are weighted according to epidemiological and phylodynamic observation models and fit using particle Markov Chain Monte Carlo. To assess performance, we test EpiFusion on simulated outbreaks in which transmission and/or surveillance rapidly changes and find that using EpiFusion to combine epidemiological and phylodynamic data maintains accuracy and increases certainty in trajectory and Rt estimates, compared to when each data type is used alone. Finally, we benchmark EpiFusions performance against existing methods to estimate Rt and demonstrate advances in efficiency and accuracy. Importantly, our approach scales efficiently with dataset size, including the use of phylogenetic trees generated from large genomic datasets. EpiFusion is designed to accommodate future extensions that will improve its utility, such as introduction of population structure, accommodations for phylogenetic uncertainty, and the ability to weight the contributions of genomic or case incidence to the inference. Author SummaryUnderstanding infectious disease spread is fundamental to protecting public health, but can be challenging as disease spread is a phenomenon that cannot be directly observed. So, epidemiologists use data in conjunction with mathematical models to estimate disease dynamics. Often, combinations of different models and data can be used to answer the same questions - for example traditional epidemiology commonly uses case incidence data (the number of people who have tested positive for a disease at a certain time) whereas phylodynamic models use pathogen genomic sequence data and our knowledge of their evolution to model disease population dynamics. Each of these approaches have strengths and limitations, and data of each type can be sparse or biased, particularly in rapidly developing outbreaks or lower-middle income countries. An increasing number of approaches attempt to fix this problem by incorporating diverse concepts and data types together in their models. We aim to contribute to this movement by introducing EpiFusion, a modelling framework that makes improvements on efficiency and temporal resolution. EpiFusion uses particle filtering to simulate epidemic trajectories over time and weight their likelihood according to both case incidence data and a phylogenetic tree using separate observation models, resulting in the inference of trajectories in agreement with both sets of data. Improvements in our ability to accurately and confidently model pathogen spread help us to respond to infectious disease outbreaks and improve public health.\nAssociation of poultry vaccination with the interspecies transmission and molecular evolution of H5 subtype avian influenza virus\nAuthors: Li, B.; Raghwani, J.; Hill, S. C.; Francois, S.; Lefrancq, N.; Liang, Y.; Wang, Z.; Dong, L.; Lemey, P.; Pybus, O. G.; Tian, H.\nScore: 8.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572711\nThe effectiveness of vaccinating poultry in preventing the transmission of highly pathogenic avian influenza viruses (AIVs) has been questioned for years and its impact on wild birds is uncertain 1-3. Here we reconstruct movements of H5 subtype AIV lineages among vaccinated poultry, unvaccinated poultry, and wild birds, worldwide from 1996 to 2023. We find that lineage transitions among host types are lagged and that movements from wild birds to unvaccinated poultry were more frequent than those from wild birds to vaccinated poultry. However, we also find that the HA gene of the AIV lineage that circulated predominately among Chinese poultry with high vaccination coverage underwent faster evolution and greater nonsynonymous divergence than other lineages. Further, this Chinese poultry lineage contained more codons inferred to be under positive selection, including at known antigenic sites, and its rates of nonsynonymous divergence and adaptative fixation increased after mass poultry vaccination began. Our results indicate that the epidemiological, ecological and evolutionary consequences of widespread AIV vaccination in poultry may be linked in complex ways, and that much work is needed to better understand how such interventions may affect AIV transmission to, within and from wild birds.\nThe last days of Aporia crataegi (L.) in Britain: evaluating genomic erosion in an extirpated butterfly\nAuthors: Whitla, R.; Hens, K.; Breuker, C. J.; Shreve, T. G.; Arif, S.\nScore: 6.1, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572305\nCurrent rates of climate change and habitat degradation are causing biodiversity declines globally, with global extinction rates far beyond estimated background extinction rates. Studies on vertebrates highlight how conservation genomics can be an effective tool to identify and manage threatened populations. However, it is unclear how metrics of genomic erosion, commonly applied in conservation genomics, translate to invertebrate species which can have markedly different population sizes and life history strategies. The Black-veined White butterfly (Aporia crataegi) was extirpated from Britain in the 1920s. Here, we sequenced historical DNA from 17 museum specimens collected between 1854 and 1924 to determine levels of genomic erosion in the species as it approached extirpation. We compare these results to those from the Common Blue butterfly (Polyommatus icarus) - a species with stable demographic trends in Great Britain. We provide evidence for a bottleneck of British butterflies around the period of post-glacial colonisation of the British Isles. Despite a 4X difference in effective population sizes around the time of post-glacial colonisation, reduction in genome-wide heterozygosity (our metric for genetic diversity) in A. crataegi (17.9%) was comparable to that in P. icarus (14.5%). More symptomatic of A. crataegis disappearance were marked increases in size and proportion of runs-of-homozygosity (RoH) and accumulation of putatively mildly and weakly deleterious variants. Our results support the notion that metrics with more direct links to host fitness and function, such as inbreeding and genetic load, may be more informative of population decline and thus more valuable in identifying vulnerable insect populations.\nOne million years of solitude: the rapid evolution of de novo protein structure and complex\nAuthors: Chen, J.; Li, Q.; Xia, S.; Arsala, D.; Sosa, D.; Wang, D.; Long, M.\nScore: 6.1, Published: 2023-12-26 DOI: 10.1101/2023.12.24.573215\nRecent studies have established that de novo genes, evolving from non-coding sequences, enhance protein diversity through a stepwise process. However, the pattern and rate of their structural evolution over time remain unclear. Here, we addressed these issues within a timeframe of ~1 million years (97% of de novo genes). We found that de novo genes evolve faster than gene duplicates in the intrinsic disordered regions (IDRs, such as random coils), secondary structural elements (such as -helix and {beta}-strand), hydrophobicity, and molecular recognition features (MoRFs). Specifically, we observed an 8-14% reduction in random coils and IDR lengths per million years per protein, and a 2.3-6.5% increase in structured elements, hydrophobicity, and MoRFs. These patterns of structural evolution align with changes in amino acid composition over time. We also revealed significantly higher positive charges but smaller molecular weights for de novo proteins than duplicates. Tertiary structure predictions demonstrated that de novo proteins, though not typically well-folded on their own, readily form low-energy, compact complexes with extensive residue contacts and conformational flexibility, suggesting a faster-binding scenario in de novo proteins to promote interaction. Our findings illuminate the rapid evolution of protein structure in the early life of de novo proteins, originating from noncoding sequences, highlighting their quick transformation into active, complex-forming components within a remarkably short evolutionary timeframe.\nShared features of blastula and neural crest stem cells evolved at the base of vertebrates\nAuthors: York, J.; Rao, A.; Huber, P. B.; Schock, E. N.; Montequin, A.; Rigney, S.; LaBonne, C.\nScore: 5.8, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572714\nThe neural crest is vertebrate-specific stem cell population that helped drive the origin and evolution of the vertebrate clade. A distinguishing feature of these stem cells is their multi-germ layer potential, which has drawn developmental and evolutionary parallels to another stem cell population--pluripotent embryonic stem cells (animal pole cells or ES cells) of the vertebrate blastula. Here, we investigate the evolutionary origins of neural crest potential by comparing neural crest and pluripotency gene regulatory networks (GRNs) in both jawed (Xenopus) and jawless (lamprey) vertebrates. Through comparative gene expression analysis and transcriptomics, we reveal an ancient evolutionary origin of shared regulatory factors between neural crest and pluripotency GRNs that dates back to the last common ancestor of extant vertebrates. Focusing on the key pluripotency factor pou5 (formerly oct4), we show that the lamprey genome encodes a pou5 ortholog that is expressed in animal pole cells, as in jawed vertebrates, but is absent from the neural crest. However, gain-of-function experiments show that both lamprey and Xenopus pou5 enhance neural crest formation, suggesting that pou5 was lost from the neural crest of jawless vertebrates. Finally, we show that pou5 is required for neural crest specification in jawed vertebrates and that it acquired novel neural crest-enhancing activity after evolving from an ancestral pou3-like clade that lacks this functionality. We propose that a pluripotency-neural crest GRN was assembled in stem vertebrates and that the multi-germ layer potential of the neural crest evolved by deploying this regulatory program.\n",
  "wordCount" : "2894",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572910">
        <p class="paperTitle">Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.</p>
        <p class="info">Score: 73.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572910' target='https://doi.org/10.1101/2023.12.21.572910'> 10.1101/2023.12.21.572910</a></p>
        <p class="abstract">Caddisflies (Trichoptera) are among the most diverse groups of freshwater animals, with more than 16,000 described species. They play an outsized role in freshwater ecology and environmental engineering in streams, rivers, and lakes. Because of this, they are frequently used as indicator organisms in biomonitoring programs. Despite their importance, key questions concerning the evolutionary history of caddisflies, such as the timing and origin of larval case-making, have been unanswered due to the lack of a well-resolved phylogenetic tree. To shed light on these questions in Trichoptera, we estimated a phylogenetic tree using a combination of transcriptomes and targeted enrichment data for 206 species, representing 48 of 52 extant families and 174 genera. We calibrated and dated the tree with a set of 33 carefully selected fossils. The first caddisflies originated in the Permian, and the major suborders began to diversify in the Triassic. Ancestral state reconstruction and diversification analysis revealed that portable case-making evolved in three separate lineages, and shifts in diversification occurred in concert with key evolutionary innovations other than case-making.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573205">
        <p class="paperTitle">Distinct ancient structural polymorphisms control heterodichogamy in walnuts and hickories</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groh, J. S.; Vik, D. C.; Stevens, K. A.; Brown, P. J.; Langley, C. H.; Coop, G.</p>
        <p class="info">Score: 16.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573205' target='https://doi.org/10.1101/2023.12.23.573205'> 10.1101/2023.12.23.573205</a></p>
        <p class="abstract">The maintenance of stable mating type polymorphisms is a classic example of balancing selection, underlying the nearly ubiquitous 50/50 sex ratio in species with separate sexes. One lesser known but intriguing example of a balanced mating polymorphism in angiosperms is heterodichogamy - polymorphism for opposing directions of dichogamy (temporal separation of male and female function in hermaphrodites) within a flowering season. This mating system is common throughout Juglandaceae, the family that includes globally important and iconic nut and timber crops - walnuts (&lt;Juglans/&gt;), as well as pecan and other hickories (&lt;Carya/&gt;). We show that two distinct genetic systems control heterodichogamous mating systems in these genera. In each genus, we find ancient (&gt;50Mya) structural variants that segregate as trans-species polymorphisms. The &lt;Juglans/&gt; locus maps to a ca. 20kb structural variant adjacent to gene with likely function in the trehalose-6-phosphate signaling pathway, which is known to regulate flowering time in Arabidopsis and inflorescence architecture in maize. Notably, an insertion in the dominant (protogynous) haplotype contains 8-12 complex tandem duplicates of the 3&#39; UTR of the gene, and transpecies polymorphism within the gene is restricted to the 3&#39; UTR. In contrast, the &lt;Carya/&gt; heterodichogamy locus maps to a ca. 200-450 kb cluster of tightly linked polymorphisms across 20 genes, some of which have known roles in flowering and are differentially expressed between morphs in floral primordia. The dominant (protogynous) haplotype in pecan, which is nearly always heterozygous and appears to rarely recombine, shows reduced genetic diversity and is over twice as long as its recessive counterpart due to proliferation of multiple transposable element families. We did not detect either genetic system in another heterodichogamous genus within the Juglandaceae, suggesting that additional genetic systems for heterodichogamy may yet remain undiscovered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573081">
        <p class="paperTitle">Evolution of Multicellularity Genes in the Lead Up to the Great Oxidation Event</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boden, J. S.; Nieves-Morion, M.; Nurnberg, D. J.; Arevalo, S.; Flores, E.; Sanchez-Baracaldo, P.</p>
        <p class="info">Score: 14.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573081' target='https://doi.org/10.1101/2023.12.23.573081'> 10.1101/2023.12.23.573081</a></p>
        <p class="abstract">Cyanobacteria are among the most morphologically diverse prokaryotic phyla on Earth. Their morphotypes range from unicellular to multicellular filaments, yet mechanisms underlying the evolution of filamentous morphologies remain unknown. Here, we implement phylogenomic, Bayesian molecular clock and gene-tree-species-tree reconciliation analyses to estimate when genes encoding cell-cell joining structures first evolved. We also characterise septal structures and measure intercellular communication rates in non-model and early-branching filamentous strains. Our results suggest that genes encoding septal proteins (namely sepJ, sepI, and fraE) and potentially pattern formation (hetR) evolved in the Neoarchaean 2.6-2.7 billion years (Ga) ago. Later, at the start of the Great Oxygenation Event approximately 2.5 Ga, genes involved in cellular differentiation (namely hetZ, patU3 and hglK) appeared. Our results predict that early-branching lineages like Pseudanabaena were capable of intercellular communication, but further innovations in cellular differentiation were needed to drive ecological expansion on a scale large enough to permanently oxygenate Earth&#39;s atmosphere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570174">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570174" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570174">
        <p class="paperTitle">Ancestral neural circuits potentiate the origin of a female sexual behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570174" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570174" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, M.; Chen, D. S.; Junker, I. P.; Szorenyi, F. I.; Chen, G. H.; Berger, A. J.; Comeault, A. A.; Matute, D. R.; Ding, Y.</p>
        <p class="info">Score: 13.0, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570174' target='https://doi.org/10.1101/2023.12.05.570174'> 10.1101/2023.12.05.570174</a></p>
        <p class="abstract">Courtship interactions are remarkably diverse in form and complexity among species. How neural circuits evolve to encode new behaviors that are functionally integrated into these dynamic social interactions is unknown. Here we report a recently originated female sexual behavior in the island endemic Drosophila species D. santomea, where females signal receptivity to male courtship songs by spreading their wings, which in turn promotes prolonged songs in courting males. Copulation success depends on this female signal and correlates with males ability to adjust his singing in such a social feedback loop. Functional comparison of sexual circuitry across species suggests that a pair of descending neurons, which integrates male song stimuli and female internal state to control a conserved female abdominal behavior, drives wing spreading in D. santomea. This co-option occurred through the refinement of a pre-existing, plastic circuit that can be optogenetically activated in an outgroup species. Combined, our results show that the ancestral potential of a socially-tuned key circuit node to engage the wing motor program facilitates the expression of a new female behavior in appropriate sensory and motivational contexts. More broadly, our work provides insights into the evolution of social behaviors, particularly female behaviors, and the underlying neural mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571477" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571477">
        <p class="paperTitle">Evolution in microbial microcosms is highly parallel regardless of the presence of interacting species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meroz, N.; Livny, T.; Toledano, G.; Sorokin, Y.; Tovi, N.; Friedman, J.</p>
        <p class="info">Score: 19.1, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571477' target='https://doi.org/10.1101/2023.12.14.571477'> 10.1101/2023.12.14.571477</a></p>
        <p class="abstract">During laboratory evolution, replicate bacterial populations often follow similar trajectories, thus their evolution is potentially predictable. However, predicting the evolution of natural populations, which are commonly embedded in multispecies communities, would prove extremely difficult if adaptations are contingent on the identity of the interacting species. The extent to which adaptations typically depend on coevolving partners remains poorly understood, since coevolution is commonly studied using small-scale experiments involving few species, making it challenging to extract general trends. To address this knowledge gap, we study the adaptations that occurred in strains of each of 11 species that were either evolved in monoculture or in multiple pairwise co-cultures. While we detect slight but significant partner-specific effects we find that the majority of evolutionary changes that occur are robust across strains that evolved with different partners; species growth abilities increase by a similar factor regardless of partners identity, shifts in community compositions and interactions are similar between pairs of coevolved and separately evolved strains, and the majority of parallelly mutated genes were detected in multiple biotic conditions. We hypothesized that these results might arise from the fact that ancestral strains are maladapted to the abiotic environment, thus having a pool of adaptations that are beneficial regardless of the biotic partners. Therefore, we conducted a second experiment with strains that were pre-adapted to the abiotic conditions before being co-cultured. We find that even after [~]400 generations of pre-adaptation, evolution is surprisingly non-partner-specific. Further work is required in order to elucidate the factors that influence partner-specificity of coevolution, but our results suggest that selection imposed by the biotic environment may play a secondary role to that imposed by abiotic conditions, making predictions regarding coevolutionary dynamics less challenging than previously thought.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572106">
        <p class="paperTitle">EpiFusion: Joint inference of the effective reproduction number by integrating phylodynamic and epidemiological modelling with particle filtering</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Judge, C.; Vaughan, T.; Russell, T.; Abbott, S.; du Plessis, L.; Stadler, T.; Brady, O.; Hill, S.</p>
        <p class="info">Score: 9.7, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572106' target='https://doi.org/10.1101/2023.12.18.572106'> 10.1101/2023.12.18.572106</a></p>
        <p class="abstract">Accurately estimating the effective reproduction number (Rt) of a circulating pathogen is a fundamental challenge in the study of infectious disease. The fields of epidemiology and pathogen phylodynamics both share this goal, but to date, methodologies and data employed by each remain largely distinct. Here we present EpiFusion: a joint approach that can be used to harness the complementary strengths of each field to improve estimation of outbreak dynamics for large and poorly sampled epidemics, such as arboviral or respiratory outbreaks, and validate it for retrospective analysis. We propose a model of Rt that estimates outbreak trajectories conditional upon both phylodynamic (time-scaled trees estimated from genetic sequences) and epidemiological (case incidence) data. We simulate stochastic outbreak trajectories that are weighted according to epidemiological and phylodynamic observation models and fit using particle Markov Chain Monte Carlo. To assess performance, we test EpiFusion on simulated outbreaks in which transmission and/or surveillance rapidly changes and find that using EpiFusion to combine epidemiological and phylodynamic data maintains accuracy and increases certainty in trajectory and Rt estimates, compared to when each data type is used alone. Finally, we benchmark EpiFusions performance against existing methods to estimate Rt and demonstrate advances in efficiency and accuracy. Importantly, our approach scales efficiently with dataset size, including the use of phylogenetic trees generated from large genomic datasets. EpiFusion is designed to accommodate future extensions that will improve its utility, such as introduction of population structure, accommodations for phylogenetic uncertainty, and the ability to weight the contributions of genomic or case incidence to the inference.

Author SummaryUnderstanding infectious disease spread is fundamental to protecting public health, but can be challenging as disease spread is a phenomenon that cannot be directly observed. So, epidemiologists use data in conjunction with mathematical models to estimate disease dynamics. Often, combinations of different models and data can be used to answer the same questions - for example  traditional epidemiology commonly uses case incidence data (the number of people who have tested positive for a disease at a certain time) whereas phylodynamic models use pathogen genomic sequence data and our knowledge of their evolution to model disease population dynamics. Each of these approaches have strengths and limitations, and data of each type can be sparse or biased, particularly in rapidly developing outbreaks or lower-middle income countries. An increasing number of approaches attempt to fix this problem by incorporating diverse concepts and data types together in their models. We aim to contribute to this movement by introducing EpiFusion, a modelling framework that makes improvements on efficiency and temporal resolution. EpiFusion uses particle filtering to simulate epidemic trajectories over time and weight their likelihood according to both case incidence data and a phylogenetic tree using separate observation models, resulting in the inference of trajectories in agreement with both sets of data. Improvements in our ability to accurately and confidently model pathogen spread help us to respond to infectious disease outbreaks and improve public health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572711">
        <p class="paperTitle">Association of poultry vaccination with the interspecies transmission and molecular evolution of H5 subtype avian influenza virus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, B.; Raghwani, J.; Hill, S. C.; Francois, S.; Lefrancq, N.; Liang, Y.; Wang, Z.; Dong, L.; Lemey, P.; Pybus, O. G.; Tian, H.</p>
        <p class="info">Score: 8.2, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572711' target='https://doi.org/10.1101/2023.12.20.572711'> 10.1101/2023.12.20.572711</a></p>
        <p class="abstract">The effectiveness of vaccinating poultry in preventing the transmission of highly pathogenic avian influenza viruses (AIVs) has been questioned for years and its impact on wild birds is uncertain 1-3. Here we reconstruct movements of H5 subtype AIV lineages among vaccinated poultry, unvaccinated poultry, and wild birds, worldwide from 1996 to 2023. We find that lineage transitions among host types are lagged and that movements from wild birds to unvaccinated poultry were more frequent than those from wild birds to vaccinated poultry. However, we also find that the HA gene of the AIV lineage that circulated predominately among Chinese poultry with high vaccination coverage underwent faster evolution and greater nonsynonymous divergence than other lineages. Further, this Chinese poultry lineage contained more codons inferred to be under positive selection, including at known antigenic sites, and its rates of nonsynonymous divergence and adaptative fixation increased after mass poultry vaccination began. Our results indicate that the epidemiological, ecological and evolutionary consequences of widespread AIV vaccination in poultry may be linked in complex ways, and that much work is needed to better understand how such interventions may affect AIV transmission to, within and from wild birds.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572305">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572305" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572305">
        <p class="paperTitle">The last days of Aporia crataegi (L.) in Britain: evaluating genomic erosion in an extirpated butterfly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572305" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572305" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitla, R.; Hens, K.; Breuker, C. J.; Shreve, T. G.; Arif, S.</p>
        <p class="info">Score: 6.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572305' target='https://doi.org/10.1101/2023.12.19.572305'> 10.1101/2023.12.19.572305</a></p>
        <p class="abstract">Current rates of climate change and habitat degradation are causing biodiversity declines globally, with global extinction rates far beyond estimated background extinction rates. Studies on vertebrates highlight how conservation genomics can be an effective tool to identify and manage threatened populations. However, it is unclear how metrics of genomic erosion, commonly applied in conservation genomics, translate to invertebrate species which can have markedly different population sizes and life history strategies. The Black-veined White butterfly (Aporia crataegi) was extirpated from Britain in the 1920s. Here, we sequenced historical DNA from 17 museum specimens collected between 1854 and 1924 to determine levels of genomic erosion in the species as it approached extirpation. We compare these results to those from the Common Blue butterfly (Polyommatus icarus) - a species with stable demographic trends in Great Britain. We provide evidence for a bottleneck of British butterflies around the period of post-glacial colonisation of the British Isles. Despite a 4X difference in effective population sizes around the time of post-glacial colonisation, reduction in genome-wide heterozygosity (our metric for genetic diversity) in A. crataegi (17.9%) was comparable to that in P. icarus (14.5%). More symptomatic of A. crataegis disappearance were marked increases in size and proportion of runs-of-homozygosity (RoH) and accumulation of putatively mildly and weakly deleterious variants. Our results support the notion that metrics with more direct links to host fitness and function, such as inbreeding and genetic load, may be more informative of population decline and thus more valuable in identifying vulnerable insect populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573215">
        <p class="paperTitle">One million years of solitude: the rapid evolution of de novo protein structure and complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Li, Q.; Xia, S.; Arsala, D.; Sosa, D.; Wang, D.; Long, M.</p>
        <p class="info">Score: 6.1, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573215' target='https://doi.org/10.1101/2023.12.24.573215'> 10.1101/2023.12.24.573215</a></p>
        <p class="abstract">Recent studies have established that de novo genes, evolving from non-coding sequences, enhance protein diversity through a stepwise process. However, the pattern and rate of their structural evolution over time remain unclear. Here, we addressed these issues within a timeframe of ~1 million years (97% of de novo genes). We found that de novo genes evolve faster than gene duplicates in the intrinsic disordered regions (IDRs, such as random coils), secondary structural elements (such as -helix and {beta}-strand), hydrophobicity, and molecular recognition features (MoRFs). Specifically, we observed an 8-14% reduction in random coils and IDR lengths per million years per protein, and a 2.3-6.5% increase in structured elements, hydrophobicity, and MoRFs. These patterns of structural evolution align with changes in amino acid composition over time. We also revealed significantly higher positive charges but smaller molecular weights for de novo proteins than duplicates. Tertiary structure predictions demonstrated that de novo proteins, though not typically well-folded on their own, readily form low-energy, compact complexes with extensive residue contacts and conformational flexibility, suggesting a faster-binding scenario in de novo proteins to promote interaction. Our findings illuminate the rapid evolution of protein structure in the early life of de novo proteins, originating from noncoding sequences, highlighting their quick transformation into active, complex-forming components within a remarkably short evolutionary timeframe.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572714">
        <p class="paperTitle">Shared features of blastula and neural crest stem cells evolved at the base of vertebrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: York, J.; Rao, A.; Huber, P. B.; Schock, E. N.; Montequin, A.; Rigney, S.; LaBonne, C.</p>
        <p class="info">Score: 5.8, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572714' target='https://doi.org/10.1101/2023.12.21.572714'> 10.1101/2023.12.21.572714</a></p>
        <p class="abstract">The neural crest is vertebrate-specific stem cell population that helped drive the origin and evolution of the vertebrate clade. A distinguishing feature of these stem cells is their multi-germ layer potential, which has drawn developmental and evolutionary parallels to another stem cell population--pluripotent embryonic stem cells (animal pole cells or ES cells) of the vertebrate blastula. Here, we investigate the evolutionary origins of neural crest potential by comparing neural crest and pluripotency gene regulatory networks (GRNs) in both jawed (Xenopus) and jawless (lamprey) vertebrates. Through comparative gene expression analysis and transcriptomics, we reveal an ancient evolutionary origin of shared regulatory factors between neural crest and pluripotency GRNs that dates back to the last common ancestor of extant vertebrates. Focusing on the key pluripotency factor pou5 (formerly oct4), we show that the lamprey genome encodes a pou5 ortholog that is expressed in animal pole cells, as in jawed vertebrates, but is absent from the neural crest. However, gain-of-function experiments show that both lamprey and Xenopus pou5 enhance neural crest formation, suggesting that pou5 was lost from the neural crest of jawless vertebrates. Finally, we show that pou5 is required for neural crest specification in jawed vertebrates and that it acquired novel neural crest-enhancing activity after evolving from an ancestral pou3-like clade that lacks this functionality. We propose that a pluripotency-neural crest GRN was assembled in stem vertebrates and that the multi-germ layer potential of the neural crest evolved by deploying this regulatory program.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
