<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 61.8, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 61.8, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 61.8, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 61.8, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe.",
  "keywords": [
    
  ],
  "articleBody": " Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 61.8, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.\nVariation in Neural Crest Development and Functional Divergence of sox10 Paralogs Contribute to Morphological Diversification in Cichlid Fishes\nAuthors: Marconi, A.; Vernaz, G.; Karunaratna, A.; Ngochera, M. J.; Durbin, R.; Santos, M. E.\nScore: 36.5, Published: 2024-01-31 DOI: 10.1101/2024.01.30.578004\nNeural crest (NC) is a vertebrate-specific embryonic progenitor cell population at the basis of important vertebrate features such as the craniofacial skeleton and pigmentation patterns. Despite the wide-ranging variation of NC-derived traits across vertebrates, the contribution of NC to species diversification remains largely unexplored. Here, by leveraging the adaptive diversity of African Great Lakes cichlid species, we combined comparative transcriptomics and population genomics to investigate the role of NC development in morphological diversification. Our analysis revealed substantial differences in transcriptional landscapes across somitogenesis, an embryonic period coinciding with NC development and migration. Notably, several NC-related gene expression clusters showed both species-specific divergence in transcriptional landscapes and signatures of positive selection. Specifically, we identified two paralogs of the sox10 gene as prime NC-related candidates contributing to interspecific morphological variation, which displayed remarkable spatio-temporal expression variation in cichlids. Finally, through CRISPR-KO mutants, we experimentally validated the functional divergence between sox10 paralogs, with the acquisition of a novel role in cichlid skeletogenesis by sox10-like. Our study demonstrates the central role of NC-related processes - in particular those controlled by sox10s - in generating morphological diversification among closely-related species and lays the groundwork for further investigations into the mechanisms underpinning vertebrate NC diversification.\nParallel evolution in mosquito vectors - a duplicated esterase locus is associated with resistance to pirimiphos-methyl in An. gambiae\nAuthors: Nagi, S. C.; Lucas, E. R.; Egyir-Yawson, A.; Essandoh, J.; Dadzie, S.; Chabi, J.; Djogbenou, L. S.; Medjigbodo, A. A.; Edi, C. V.; Ketoh, G.; Koudou, B. G.; Ashraf, F.; Clarkson, C. S.; Miles, A.; Weetman, D.; Donnelly, M. J.\nScore: 13.7, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578361\nThe primary control methods for the African malaria mosquito, Anopheles gambiae, are based on insecticidal interventions. Emerging resistance to these compounds is therefore of major concern to malaria control programmes. The organophosphate, pirimiphos-methyl, is a relatively new chemical in the vector control armoury but is now widely used in indoor residual spray campaigns. Whilst generally effective, phenotypic resistance has developed in some areas in malaria vectors. Here, we used a population genomic approach to identify novel mechanisms of resistance to pirimiphos-methyl in Anopheles gambiae s.l mosquitoes. In multiple populations, we found large and repeated signals of selection at a locus containing a cluster of detoxification enzymes, some of whose orthologs are known to confer resistance to organophosphates in Culex pipiens. Close examination revealed a pair of alpha-esterases, Coeae1f and Coeae2f, and a complex and diverse pattern of haplotypes under selection in An. gambiae, An. coluzzii and An. arabiensis. As in Cx. pipiens, copy number variation seems to play a role in the evolution of insecticide resistance at this locus. We used diplotype clustering to examine whether these signals arise from parallel evolution or adaptive introgression. Using whole-genome sequenced phenotyped samples, we found that in West Africa, a copy number variant in Anopheles gambiae is associated with resistance to pirimiphos-methyl. Overall, we demonstrate a striking example of contemporary parallel evolution which has important implications for malaria control programmes.\nEvolutionary innovation in conserved regulatory elements across the mammalian tree of life\nAuthors: Uebbing, S.; Kocher, A. A.; Baumgartner, M.; Ji, Y.; Bai, S.; Xing, X.; Nottoli, T.; Noonan, J. P.\nScore: 9.2, Published: 2024-01-31 DOI: 10.1101/2024.01.31.578197\nTranscriptional enhancers orchestrate cell type- and time point-specific gene expression programs. Evolution of enhancer sequences can alter target gene expression without causing detrimental misexpression in other contexts. It has long been thought that this modularity allows evolutionary changes in enhancers to escape pleiotropic constraints, which is especially important for evolutionary constrained developmental patterning genes. However, there is still little data supporting this hypothesis. Here we identified signatures of accelerated evolution in conserved enhancer elements across the mammalian phylogeny. We found that pleiotropic genes involved in gene regulatory and developmental processes were enriched for accelerated sequence evolution within their enhancer elements. These genes were associated with an excess number of enhancers compared to other genes, and due to this they exhibit a substantial degree of sequence acceleration over all their enhancers combined. We provide evidence that sequence acceleration is associated with turnover of regulatory function. We studied one acceleration event in depth and found that its sequence evolution led to the emergence of a new enhancer activity domain that may be involved in the evolution of digit reduction in hoofed mammals. Our results provide tangible evidence that enhancer evolution has been a frequent contributor to modifications involving constrained developmental signaling genes in mammals.\nReproductive innovation enabled radiation in the deep sea during an ecological crisis\nAuthors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.\nScore: 115.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380\nMajor ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate. A phylogenomic reconstruction of the evolutionary history of anglerfishes provides a strong inference for the convergent evolution of permanently-fusing deep-sea anglerfishes and their degenerate immune genes. Our results support that sexual parasitism was enabled by the degeneration of adaptive immunity and ancestral sexual size dimorphism. The combination of these traits facilitated the transition of pelagic anglerfishes into novel ecologies available in the deep open oceans after evolving from benthic ancestors. These results show how seemingly unrelated physiological and reproductive traits interact synergistically to drive evolutionary radiation in novel environments.\nLineage-resolved analysis of embryonic gene expression evolution in C. elegans and C. briggsae\nAuthors: Large, C. R. L.; Khanal, R.; Hillier, L. W.; Huynh, C.; Kubo, C.; Kim, J.; Waterston, R.; Murray, J. I.\nScore: 7.6, Published: 2024-02-06 DOI: 10.1101/2024.02.03.578695\nWhat constraints govern the evolution of gene expression patterns across development remains a fundamental question of evolutionary biology. The advent of single-cell sequencing opens the possibility of learning these constraints by systematically profiling homologous cells across different organisms. The nematode C. elegans is a well-studied model for embryonic development, and its invariant lineage that is conserved with other Caenorhabditis species makes it an ideal model to directly compare gene expression between homologous progenitor and terminal cell types across evolution. We have measured the spatiotemporal divergence of gene expression across embryogenesis by collecting, annotating and comparing the transcriptomes of homologous embryonic progenitors and terminal cell types, using a dataset comprising \u003e200,000 C. elegans cells and \u003e190,000 C. briggsae cells. We find a high level of similarity in gene expression programs between the species despite tens of millions of years of evolutionary divergence, consistent with their conserved developmental lineages. Even still, thousands of genes show divergence in their cell-type specific expression patterns, and these are enriched for categories involved in environmental response and behavior. Comparing the degree of expression conservation across cell types reveals that certain cell types such as neurons, have diverged more than others such as the intestine and body wall muscle. Taken together, this work identifies likely constraints on the evolution of developmental gene expression and provides a powerful resource for addressing diverse evolutionary questions.\nQuantifying internal conflicts and their threats to organismal form and fitness\nAuthors: Schenkel, M. A.; Patten, M. M.; Agren, J. A.\nScore: 7.5, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578856\nEvolutionary biologists often treat organisms as both fitness-maximizing agents and as the primary level at which adaptation manifests. Yet, genes and cells may also seek to optimize their fitness by distorting the Mendelian rules of transmission or by influencing organismal traits for their own benefit. Organismal form and fitness are therefore threatened from within by selfish genes and cells. However, to what extent such internal conflicts actually harm individual organisms and threaten our concept of the organism as the sole bearer of adaptation remains unclear. We introduce a mathematical framework to capture the threat posed by internal conflicts and develop two metrics to measure their various forms of harm. We name these metrics fitness unity and trait unity, and use them to refer to the threats posed by internal conflicts to an organism's role as the optimizing agent and the strategy wielded to achieve that optimization, respectively. We apply our framework to two examples of internal conflicts, genomic imprinting and sex ratio distortion, to illustrate how such harms from internal conflict may be quantified. We conclude by discussing the conditions under which internal conflict becomes sufficiently disruptive to organisms that it no longer makes sense to think of them as unified fitness-maximizing agents, but instead as adaptive compromises of multiple competing sub-agents.\nA comprehensive examination of Chelicerate genomes reveals no evidence for a whole genome duplication among spiders and scorpions\nAuthors: Thomas, G. W. C.; McKibben, M. T. W.; Hahn, M. W.; Barker, M. S.\nScore: 7.0, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578966\nWhole genome duplications (WGDs) can be a key event in evolution, playing a role in both adaptation and speciation. While WGDs are common throughout the history of plants, only a few examples have been proposed in metazoans. Among these, recent proposals of WGD events in Chelicerates, the group of Arthropods that includes horseshoe crabs, ticks, scorpions, and spiders, include several rounds in the history of horseshoe crabs, with an additional WGD proposed in the ancestor of spiders and scorpions. However, many of these inferences are based on evidence from only a small portion of the genome (in particular, the Hox gene cluster); therefore, genome-wide inferences with broader species sampling may give a clearer picture of WGDs in this clade. Here, we investigate signals of WGD in Chelicerates using whole genomes from 17 species. We employ multiple methods to look for these signals, including gene tree analysis of thousands of gene families, comparisons of synteny, and signals of divergence among within-species paralogs. We test several scenarios of WGD in Chelicerates using multiple species trees as a backbone for all hypotheses. While we do find support for at least one WGD in the ancestral horseshoe crab lineage, we find no evidence for a WGD in the history of spiders and scorpions using any genome-scale method. This study not only sheds light on genome evolution and phylogenetics within Chelicerates, but also demonstrates how a combination of comparative methods can be used to investigate signals of ancient WGDs.\nGelatinous matrix, an original convergent evolution to cope with oligotrophy in Nassellaria (Radiolaria)\nAuthors: Llopis Monferrer, N.; Romac, S.; Laget, M.; Biard, T.; Sandin, M. M.\nScore: 16.1, Published: 2024-02-01 DOI: 10.1101/2024.01.22.576765\nRadiolaria are heterotrophic protists abundant in the worlds oceans playing important roles in oceanic biogeochemical cycles. Some species can host photosynthetic algae thus contributing to primary production. Such mixotrophic behaviour is believed to explain their ecological success in oligotrophic waters, notably Collodaria, a group of exclusively mixotrophic radiolarians embedded in a gelatinous matrix. Sampling oligotrophic California Current communities revealed an abundant, rarely observed population of Nassellaria of the genus Phlebarachnium, the only Radiolaria known to live within a gelatinous matrix besides Collodaria. Phylogenetic reconstruction of the ribosomal DNA suggests that these two distantly related lineages within Nassellaria independently developed the ability to produce a gelatinous matrix [~]120 million years ago. Molecular analyses identify photosynthetic symbionts as Scrippsiella sp., a common dinoflagellate symbiont of other planktonic groups. By matching our physical samples with their genetic signature, we identified these rarely observed organisms in global metabarcoding datasets, revealing a strong biogeographic affinity to oligotrophic water masses. Our results suggest that the gelatinous matrix is an exclusive adaptation to oligotrophic waters although further research is required to evaluate the similarities between the gelatinous matrices of Collodaria and Phlebarachnium. We hypothesize that such an original convergent evolution could increase the effective volume to weight ratio favoring prey contact and capture and provide an advantageous microenvironment for symbionts, enhancing ecological success in nutrient-depleted waters. This study advances our understanding of the evolution of eukaryotic diversity and highlights the specific advantages of certain adaptations, specifically when evolution occurs independently among various lineages.\nTarget Enrichment Data Uncovers Rapid Radiation, Whole Genome Duplication, and Extensive Hybridization in Slipper Orchid Genus Cypripedium L.\nAuthors: Lagou, L. J.; Kadereit, G.; Morales-Briones, D. F.\nScore: 7.6, Published: 2024-01-27 DOI: 10.1101/2024.01.24.577114\nBackground and AimsCypripedium is the most widespread and morphologically diverse genus of slipper orchids. Despite several published phylogenies based on Sanger sequencing data, the topology and monophyly of its infrageneric taxa remained uncertain. Here, we aimed to reconstruct a robust section-level phylogeny of Cypripedium and explore its evolutionary history using target capture data for the first time. MethodsWe used the orchid-specific bait set \"Orchidaceae963\" to reconstruct the phylogeny of Cypripedium based on 614 nuclear loci, covering 11 out of 13 sections. Subsequently, we investigated tree discordance, estimated divergence times and ancestral ranges, searched for anomaly zones, polytomies, and diversification rate shifts, and identified gene duplication and hybridization events. Key ResultsAll sections were recovered as monophyletic, contrary to the subsections within sect. Cypripedium. Although the two subclades within this section did not correspond to its two subsections, they matched the geographic distribution of their species. Additionally, we discovered high levels of discordance in the short backbone branches of the genus and within sect. Cypripedium, which can be attributed to gene duplication and hybridization events, a potential whole genome duplication, and incomplete lineage sorting caused by rapid radiation. Our biogeographic analysis suggested a Neotropical origin of the genus during the Early Miocene ([~]20 Ma). The rapid radiations at the backbone likely occurred in Southeast Asia around the Middle Miocene Climatic Transition ([~]15-13 Ma), followed by several independent dispersals back to the New World. Moreover, the Pliocene-Quaternary glacial cycles may have contributed to further speciation and reticulate evolution, giving rise to a hybrid swarm within sect. Cypripedium. ConclusionsOur study provided novel insights into the evolutionary history of Cypripedium based on high-throughput molecular data, shedding light on the dynamics of its distribution and diversity patterns from its origin to the present.\n",
  "wordCount" : "2573",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576074">
        <p class="paperTitle">Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cain, F. E.; Yamaguchi, K.</p>
        <p class="info">Score: 61.8, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576074' target='https://doi.org/10.1101/2024.01.17.576074'> 10.1101/2024.01.17.576074</a></p>
        <p class="abstract">The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578004" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578004">
        <p class="paperTitle">Variation in Neural Crest Development and Functional Divergence of sox10 Paralogs Contribute to Morphological Diversification in Cichlid Fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marconi, A.; Vernaz, G.; Karunaratna, A.; Ngochera, M. J.; Durbin, R.; Santos, M. E.</p>
        <p class="info">Score: 36.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578004' target='https://doi.org/10.1101/2024.01.30.578004'> 10.1101/2024.01.30.578004</a></p>
        <p class="abstract">Neural crest (NC) is a vertebrate-specific embryonic progenitor cell population at the basis of important vertebrate features such as the craniofacial skeleton and pigmentation patterns. Despite the wide-ranging variation of NC-derived traits across vertebrates, the contribution of NC to species diversification remains largely unexplored. Here, by leveraging the adaptive diversity of African Great Lakes cichlid species, we combined comparative transcriptomics and population genomics to investigate the role of NC development in morphological diversification. Our analysis revealed substantial differences in transcriptional landscapes across somitogenesis, an embryonic period coinciding with NC development and migration. Notably, several NC-related gene expression clusters showed both species-specific divergence in transcriptional landscapes and signatures of positive selection. Specifically, we identified two paralogs of the sox10 gene as prime NC-related candidates contributing to interspecific morphological variation, which displayed remarkable spatio-temporal expression variation in cichlids. Finally, through CRISPR-KO mutants, we experimentally validated the functional divergence between sox10 paralogs, with the acquisition of a novel role in cichlid skeletogenesis by sox10-like. Our study demonstrates the central role of NC-related processes - in particular those controlled by sox10s - in generating morphological diversification among closely-related species and lays the groundwork for further investigations into the mechanisms underpinning vertebrate NC diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578361">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578361" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578361">
        <p class="paperTitle">Parallel evolution in mosquito vectors - a duplicated esterase locus is associated with resistance to pirimiphos-methyl in An. gambiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578361" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578361" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nagi, S. C.; Lucas, E. R.; Egyir-Yawson, A.; Essandoh, J.; Dadzie, S.; Chabi, J.; Djogbenou, L. S.; Medjigbodo, A. A.; Edi, C. V.; Ketoh, G.; Koudou, B. G.; Ashraf, F.; Clarkson, C. S.; Miles, A.; Weetman, D.; Donnelly, M. J.</p>
        <p class="info">Score: 13.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578361' target='https://doi.org/10.1101/2024.02.01.578361'> 10.1101/2024.02.01.578361</a></p>
        <p class="abstract">The primary control methods for the African malaria mosquito, Anopheles gambiae, are based on insecticidal interventions. Emerging resistance to these compounds is therefore of major concern to malaria control programmes. The organophosphate, pirimiphos-methyl, is a relatively new chemical in the vector control armoury but is now widely used in indoor residual spray campaigns. Whilst generally effective, phenotypic resistance has developed in some areas in malaria vectors. Here, we used a population genomic approach to identify novel mechanisms of resistance to pirimiphos-methyl in Anopheles gambiae s.l mosquitoes. In multiple populations, we found large and repeated signals of selection at a locus containing a cluster of detoxification enzymes, some of whose orthologs are known to confer resistance to organophosphates in Culex pipiens. Close examination revealed a pair of alpha-esterases, Coeae1f and Coeae2f, and a complex and diverse pattern of haplotypes under selection in An. gambiae, An. coluzzii and An. arabiensis. As in Cx. pipiens, copy number variation seems to play a role in the evolution of insecticide resistance at this locus. We used diplotype clustering to examine whether these signals arise from parallel evolution or adaptive introgression. Using whole-genome sequenced phenotyped samples, we found that in West Africa, a copy number variant in Anopheles gambiae is associated with resistance to pirimiphos-methyl. Overall, we demonstrate a striking example of contemporary parallel evolution which has important implications for malaria control programmes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578197" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578197">
        <p class="paperTitle">Evolutionary innovation in conserved regulatory elements across the mammalian tree of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uebbing, S.; Kocher, A. A.; Baumgartner, M.; Ji, Y.; Bai, S.; Xing, X.; Nottoli, T.; Noonan, J. P.</p>
        <p class="info">Score: 9.2, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578197' target='https://doi.org/10.1101/2024.01.31.578197'> 10.1101/2024.01.31.578197</a></p>
        <p class="abstract">Transcriptional enhancers orchestrate cell type- and time point-specific gene expression programs. Evolution of enhancer sequences can alter target gene expression without causing detrimental misexpression in other contexts. It has long been thought that this modularity allows evolutionary changes in enhancers to escape pleiotropic constraints, which is especially important for evolutionary constrained developmental patterning genes. However, there is still little data supporting this hypothesis. Here we identified signatures of accelerated evolution in conserved enhancer elements across the mammalian phylogeny. We found that pleiotropic genes involved in gene regulatory and developmental processes were enriched for accelerated sequence evolution within their enhancer elements. These genes were associated with an excess number of enhancers compared to other genes, and due to this they exhibit a substantial degree of sequence acceleration over all their enhancers combined. We provide evidence that sequence acceleration is associated with turnover of regulatory function. We studied one acceleration event in depth and found that its sequence evolution led to the emergence of a new enhancer activity domain that may be involved in the evolution of digit reduction in hoofed mammals. Our results provide tangible evidence that enhancer evolution has been a frequent contributor to modifications involving constrained developmental signaling genes in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575380">
        <p class="paperTitle">Reproductive innovation enabled radiation in the deep sea during an ecological crisis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.</p>
        <p class="info">Score: 115.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575380' target='https://doi.org/10.1101/2024.01.12.575380'> 10.1101/2024.01.12.575380</a></p>
        <p class="abstract">Major ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate. A phylogenomic reconstruction of the evolutionary history of anglerfishes provides a strong inference for the convergent evolution of permanently-fusing deep-sea anglerfishes and their degenerate immune genes. Our results support that sexual parasitism was enabled by the degeneration of adaptive immunity and ancestral sexual size dimorphism. The combination of these traits facilitated the transition of pelagic anglerfishes into novel ecologies available in the deep open oceans after evolving from benthic ancestors. These results show how seemingly unrelated physiological and reproductive traits interact synergistically to drive evolutionary radiation in novel environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.578695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.578695" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.578695">
        <p class="paperTitle">Lineage-resolved analysis of embryonic gene expression evolution in C. elegans and C. briggsae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.578695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.578695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Large, C. R. L.; Khanal, R.; Hillier, L. W.; Huynh, C.; Kubo, C.; Kim, J.; Waterston, R.; Murray, J. I.</p>
        <p class="info">Score: 7.6, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.578695' target='https://doi.org/10.1101/2024.02.03.578695'> 10.1101/2024.02.03.578695</a></p>
        <p class="abstract">What constraints govern the evolution of gene expression patterns across development remains a fundamental question of evolutionary biology. The advent of single-cell sequencing opens the possibility of learning these constraints by systematically profiling homologous cells across different organisms. The nematode C. elegans is a well-studied model for embryonic development, and its invariant lineage that is conserved with other Caenorhabditis species makes it an ideal model to directly compare gene expression between homologous progenitor and terminal cell types across evolution. We have measured the spatiotemporal divergence of gene expression across embryogenesis by collecting, annotating and comparing the transcriptomes of homologous embryonic progenitors and terminal cell types, using a dataset comprising &gt;200,000 C. elegans cells and &gt;190,000 C. briggsae cells. We find a high level of similarity in gene expression programs between the species despite tens of millions of years of evolutionary divergence, consistent with their conserved developmental lineages. Even still, thousands of genes show divergence in their cell-type specific expression patterns, and these are enriched for categories involved in environmental response and behavior. Comparing the degree of expression conservation across cell types reveals that certain cell types such as neurons, have diverged more than others such as the intestine and body wall muscle. Taken together, this work identifies likely constraints on the evolution of developmental gene expression and provides a powerful resource for addressing diverse evolutionary questions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578856" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578856">
        <p class="paperTitle">Quantifying internal conflicts and their threats to organismal form and fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schenkel, M. A.; Patten, M. M.; Agren, J. A.</p>
        <p class="info">Score: 7.5, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578856' target='https://doi.org/10.1101/2024.02.05.578856'> 10.1101/2024.02.05.578856</a></p>
        <p class="abstract">Evolutionary biologists often treat organisms as both fitness-maximizing agents and as the primary level at which adaptation manifests. Yet, genes and cells may also seek to optimize their fitness by distorting the Mendelian rules of transmission or by influencing organismal traits for their own benefit. Organismal form and fitness are therefore threatened from within by selfish genes and cells. However, to what extent such internal conflicts actually harm individual organisms and threaten our concept of the organism as the sole bearer of adaptation remains unclear. We introduce a mathematical framework to capture the threat posed by internal conflicts and develop two metrics to measure their various forms of harm. We name these metrics fitness unity and trait unity, and use them to refer to the threats posed by internal conflicts to an organism&#39;s role as the optimizing agent and the strategy wielded to achieve that optimization, respectively. We apply our framework to two examples of internal conflicts, genomic imprinting and sex ratio distortion, to illustrate how such harms from internal conflict may be quantified. We conclude by discussing the conditions under which internal conflict becomes sufficiently disruptive to organisms that it no longer makes sense to think of them as unified fitness-maximizing agents, but instead as adaptive compromises of multiple competing sub-agents.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578966" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578966">
        <p class="paperTitle">A comprehensive examination of Chelicerate genomes reveals no evidence for a whole genome duplication among spiders and scorpions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, G. W. C.; McKibben, M. T. W.; Hahn, M. W.; Barker, M. S.</p>
        <p class="info">Score: 7.0, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578966' target='https://doi.org/10.1101/2024.02.05.578966'> 10.1101/2024.02.05.578966</a></p>
        <p class="abstract">Whole genome duplications (WGDs) can be a key event in evolution, playing a role in both adaptation and speciation. While WGDs are common throughout the history of plants, only a few examples have been proposed in metazoans. Among these, recent proposals of WGD events in Chelicerates, the group of Arthropods that includes horseshoe crabs, ticks, scorpions, and spiders, include several rounds in the history of horseshoe crabs, with an additional WGD proposed in the ancestor of spiders and scorpions. However, many of these inferences are based on evidence from only a small portion of the genome (in particular, the Hox gene cluster); therefore, genome-wide inferences with broader species sampling may give a clearer picture of WGDs in this clade. Here, we investigate signals of WGD in Chelicerates using whole genomes from 17 species. We employ multiple methods to look for these signals, including gene tree analysis of thousands of gene families, comparisons of synteny, and signals of divergence among within-species paralogs. We test several scenarios of WGD in Chelicerates using multiple species trees as a backbone for all hypotheses. While we do find support for at least one WGD in the ancestral horseshoe crab lineage, we find no evidence for a WGD in the history of spiders and scorpions using any genome-scale method. This study not only sheds light on genome evolution and phylogenetics within Chelicerates, but also demonstrates how a combination of comparative methods can be used to investigate signals of ancient WGDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576765" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576765">
        <p class="paperTitle">Gelatinous matrix, an original convergent evolution to cope with oligotrophy in Nassellaria (Radiolaria)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Llopis Monferrer, N.; Romac, S.; Laget, M.; Biard, T.; Sandin, M. M.</p>
        <p class="info">Score: 16.1, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576765' target='https://doi.org/10.1101/2024.01.22.576765'> 10.1101/2024.01.22.576765</a></p>
        <p class="abstract">Radiolaria are heterotrophic protists abundant in the worlds oceans playing important roles in oceanic biogeochemical cycles. Some species can host photosynthetic algae thus contributing to primary production. Such mixotrophic behaviour is believed to explain their ecological success in oligotrophic waters, notably Collodaria, a group of exclusively mixotrophic radiolarians embedded in a gelatinous matrix. Sampling oligotrophic California Current communities revealed an abundant, rarely observed population of Nassellaria of the genus Phlebarachnium, the only Radiolaria known to live within a gelatinous matrix besides Collodaria. Phylogenetic reconstruction of the ribosomal DNA suggests that these two distantly related lineages within Nassellaria independently developed the ability to produce a gelatinous matrix [~]120 million years ago. Molecular analyses identify photosynthetic symbionts as Scrippsiella sp., a common dinoflagellate symbiont of other planktonic groups. By matching our physical samples with their genetic signature, we identified these rarely observed organisms in global metabarcoding datasets, revealing a strong biogeographic affinity to oligotrophic water masses. Our results suggest that the gelatinous matrix is an exclusive adaptation to oligotrophic waters although further research is required to evaluate the similarities between the gelatinous matrices of Collodaria and Phlebarachnium. We hypothesize that such an original convergent evolution could increase the effective volume to weight ratio favoring prey contact and capture and provide an advantageous microenvironment for symbionts, enhancing ecological success in nutrient-depleted waters. This study advances our understanding of the evolution of eukaryotic diversity and highlights the specific advantages of certain adaptations, specifically when evolution occurs independently among various lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577114" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577114">
        <p class="paperTitle">Target Enrichment Data Uncovers Rapid Radiation, Whole Genome Duplication, and Extensive Hybridization in Slipper Orchid Genus Cypripedium L.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lagou, L. J.; Kadereit, G.; Morales-Briones, D. F.</p>
        <p class="info">Score: 7.6, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577114' target='https://doi.org/10.1101/2024.01.24.577114'> 10.1101/2024.01.24.577114</a></p>
        <p class="abstract">Background and AimsCypripedium is the most widespread and morphologically diverse genus of slipper orchids. Despite several published phylogenies based on Sanger sequencing data, the topology and monophyly of its infrageneric taxa remained uncertain. Here, we aimed to reconstruct a robust section-level phylogeny of Cypripedium and explore its evolutionary history using target capture data for the first time.

MethodsWe used the orchid-specific bait set &#34;Orchidaceae963&#34; to reconstruct the phylogeny of Cypripedium based on 614 nuclear loci, covering 11 out of 13 sections. Subsequently, we investigated tree discordance, estimated divergence times and ancestral ranges, searched for anomaly zones, polytomies, and diversification rate shifts, and identified gene duplication and hybridization events.

Key ResultsAll sections were recovered as monophyletic, contrary to the subsections within sect. Cypripedium. Although the two subclades within this section did not correspond to its two subsections, they matched the geographic distribution of their species. Additionally, we discovered high levels of discordance in the short backbone branches of the genus and within sect. Cypripedium, which can be attributed to gene duplication and hybridization events, a potential whole genome duplication, and incomplete lineage sorting caused by rapid radiation. Our biogeographic analysis suggested a Neotropical origin of the genus during the Early Miocene ([~]20 Ma). The rapid radiations at the backbone likely occurred in Southeast Asia around the Middle Miocene Climatic Transition ([~]15-13 Ma), followed by several independent dispersals back to the New World. Moreover, the Pliocene-Quaternary glacial cycles may have contributed to further speciation and reticulate evolution, giving rise to a hybrid swarm within sect. Cypripedium.

ConclusionsOur study provided novel insights into the evolutionary history of Cypripedium based on high-throughput molecular data, shedding light on the dynamics of its distribution and diversity patterns from its origin to the present.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
