<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Allelic gene conversion softens selective sweeps
Authors: Schrider, D. R.
Score: 20.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570141
The prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele &#34;sweep&#34; through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Allelic gene conversion softens selective sweeps
Authors: Schrider, D. R.
Score: 20.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570141
The prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele &#34;sweep&#34; through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Allelic gene conversion softens selective sweeps
Authors: Schrider, D. R.
Score: 20.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570141
The prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele &#34;sweep&#34; through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Allelic gene conversion softens selective sweeps\nAuthors: Schrider, D. R.\nScore: 20.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570141\nThe prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele \u0026#34;sweep\u0026#34; through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection.",
  "keywords": [
    
  ],
  "articleBody": " Allelic gene conversion softens selective sweeps\nAuthors: Schrider, D. R.\nScore: 20.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570141\nThe prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele \"sweep\" through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection. Two models of selective sweeps have been proposed: classical sweeps, or \"hard sweeps\", in which a single copy of the adaptive allele sweeps to fixation, and \"soft sweeps\", in which multiple distinct copies of the adaptive allele leave descendants after the sweep. Soft sweeps can be the outcome of recurrent mutation to the adaptive allele, or the presence of standing genetic variation consisting of multiple copies of the adaptive allele prior to the onset of selection. Importantly, soft sweeps will be common when populations can rapidly adapt to novel selective pressures, either because of a high mutation rate or because adaptive alleles are already present. The prevalence of soft sweeps is especially controversial, and it has been noted that selection on standing variation or recurrent mutations may not always produce soft sweeps. Here, we show that the inverse is true: selection on single-origin de novo mutations may often result in an outcome that is indistinguishable from a soft sweep. This is made possible by allelic gene conversion, which \"softens\" hard sweeps by copying the adaptive allele onto multiple genetic backgrounds, a process we refer to as a \"pseudo-soft\" sweep. We carried out a simulation study examining the impact of gene conversion on sweeps from a single de novo variant in models of human, Drosophila, and Arabidopsis populations. The fraction of simulations in which gene conversion had produced multiple haplotypes with the adaptive allele upon fixation was appreciable. Indeed, under realistic demographic histories and gene conversion rates, even if selection always acts on a single-origin mutation, sweeps involving multiple haplotypes are more likely than hard sweeps in large populations, especially when selection is not extremely strong. Thus, even when the mutation rate is low or there is no standing variation, hard sweeps are expected to be the exception rather than the rule in large populations. These results also imply that the presence of signatures of soft sweeps does not necessarily mean that adaptation has been especially rapid or is not mutation limited.\nReproductive life history may modulate selection on the germline mutation rate\nAuthors: Zhu, L.; Beichman, A. C.; Harris, K.\nScore: 19.2, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570457\nA long lifespan is a potential liability from the point of view of mutation accumulation, allowing more time for cancer-causing mutations to accumulate in somatic tissues. In practice, however, long-lived organisms do not seem to suffer more cancer mortality than short-lived organisms, a pattern known as \"Petos Paradox.\" One suggested resolution to Petos Paradox is that long lifespans may facilitate the evolution of better somatic DNA repair. This comes into some conflict with the predictions of the drift-barrier hypothesis of mutation rate evolution, which infers that germline DNA repair efficacy should be best in short-lived organisms with high genetic diversity and efficient natural selection. Here, we show that this conflict can be resolved by modeling the tendency of long generation times to increase the strength of selection against mutator alleles that affect clocklike mutational processes. By considering the empirical joint distribution of generation time and effective population size, we show that stronger selection against mutator alleles in long-lived organisms is likely to be partially but not completely offset by their tendency to have low effective population sizes. This model resolves a conflict between the predictions of theories of germline and somatic mutation rate evolution, leading to the prediction that both germline and somatic DNA repair should be most efficient in long-lived organisms despite their lower genetic diversity. Significance StatementAll cells accumulate mutations due to DNA damage and replication errors. Mutations affecting the germline can harm future offspring, while mutations in other tissues can threaten longevity. Previous theories have predicted that somatic DNA repair should be most effective in long-lived organisms, but that germline DNA repair should instead be most effective in organisms with high genetic diversity that tend to be short-lived. We model how reproductive age should affect germline mutation rate evolution and show that this may reverse the prediction that high-diversity species should have more efficient selection for effective germline DNA repair. By demonstrating how population size and lifespan likely shape selection on mutagenesis, we are able to harmonize predictions about germline and somatic mutation rate evolution.\nExamining the role of oxygen-binding proteins on the early evolution of multicellularity\nAuthors: Wong, W.; Bravo, P.; Yunker, P. J.; Ratcliff, W. C.; Burnetti, A. J.\nScore: 21.9, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569647\nOxygen availability is a key factor in the evolution of multicellularity, as larger and more sophisticated organisms often require mechanisms allowing efficient oxygen delivery to their tissues. One such mechanism is the presence of oxygen-binding proteins, such as globins and hemerythrins, which arose in the ancestor of bilaterian animals. Despite their importance, the precise mechanisms by which oxygen-binding proteins influenced the early stages of multicellular evolution under varying environmental oxygen levels are not yet clear. We addressed this knowledge gap by heterologously expressing the oxygen binding proteins myoglobin and myohemerythrin in snowflake yeast, a model system of simple, undifferentiated multicellularity. These proteins increased the depth and rate of oxygen diffusion, increasing the fitness of snowflake yeast growing aerobically. Experiments show that, paradoxically, oxygen-binding proteins confer a greater fitness benefit for larger organisms under high, not low, O2 conditions. We show via biophysical modeling that this is because facilitated diffusion is more efficient when oxygen is abundant, transporting a greater quantity of O2 which can be used for metabolism. By alleviating anatomical diffusion limitations to oxygen consumption, the evolution of O2-binding proteins in the oxygen-rich Neoproterozoic may have been a key breakthrough enabling the evolution of increasingly large, complex multicellular metazoan lineages.\nRapid turnover of pathogen-blocking Wolbachia and their incompatibility loci\nAuthors: Shropshire, D.; Conner, W. R.; Vanderpool, D.; Hoffmann, A. A.; Turelli, M.; Cooper, B. S.\nScore: 13.9, Published: 2023-12-05 DOI: 10.1101/2023.12.04.569981\nAt least half of all insect species carry maternally inherited Wolbachia alphaproteobacteria, making Wolbachia the most common endosymbionts in nature. Wolbachia spread to high frequencies is often due to cytoplasmic incompatibility (CI), a Wolbachia-induced sperm modification that kills embryos without Wolbachia. Several CI-causing Wolbachia variants, including wMel from Drosophila melanogaster, also block viruses. Establishing pathogen-blocking wMel in natural Aedes aegypti mosquito populations has reduced dengue disease incidence, with one study reporting about 85% reduction when wMel frequency is high. However, wMel transinfection establishment is challenging in many environments, highlighting the importance of identifying CI-causing Wolbachia variants that stably persist in diverse hosts and habitats. We demonstrate that wMel-like variants have naturally established in widely distributed holometabolous dipteran and hymenopteran insects that diverged approximately 350 million years ago, with wMel variants spreading rapidly among these hosts over only the last 100,000 years. Wolbachia genomes contain prophages that encode CI-causing operons (cifs). These cifs move among Wolbachia genomes - with and without prophages - even more rapidly than Wolbachia move among insect hosts. Our results shed light on how rapid host switching and horizontal gene transfer contribute to Wolbachia and cif diversity in nature. The diverse wMel variants we report here from hosts present in different climates offer many new options for broadening Wolbachia-based biocontrol of diseases and pests. Significance StatementWolbachia bacteria transinfected into Aedes aegypti mosquitoes (in particular the wMel Wolbachia strain) are being widely released to suppress the transmission of dengue and other insect-vectored arboviruses. The bacterial strain wMel originated from Drosophila melanogaster, but very little is known about its distribution and spread to other drosophilids and distantly related insects. Here we show that there has been a rapid rate of spread of Wolbachia strains closely related to wMel not just into other flies but also into wasps, indicating a large pool of wMel variants that can be employed in vector-borne disease control. Wolbachia prophage genes essential to this spread have evolved particularly rapidly.\nPhylogenomics reveals an almost perfect polytomy among the almost ungulates (Paenungulata)\nAuthors: Bowman, J.; Enard, D.; Lynch, V. J.\nScore: 11.3, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570590\nPhylogenetic studies have resolved most relationships among Eutherian Orders. However, the branching order of elephants (Proboscidea), hyraxes (Hyracoidea), and sea cows (Sirenia) (i.e., the Paenungulata) has remained uncertain since at least 1758, when Linnaeus grouped elephants and manatees into a single Order (Bruta) to the exclusion of hyraxes. Subsequent morphological, molecular, and large-scale phylogenomic datasets have reached conflicting conclusions on the branching order within Paenungulates. We use a phylogenomic dataset of alignments from 13,388 protein-coding genes across 261 Eutherian mammals to infer phylogenetic relationships within Paenungulates. We find that gene trees almost equally support the three alternative resolutions of Paenungulate relationships and that despite strong support for a Proboscidea+Hyracoidea split in the multispecies coalescent (MSC) tree, there is significant evidence for gene tree uncertainty, incomplete lineage sorting, and introgression among Proboscidea, Hyracoidea, and Sirenia. Indeed, only 8-10% of genes have statistically significant phylogenetic signal to reject the hypothesis of a Paenungulate polytomy. These data indicate little support for any resolution for the branching order Proboscidea, Hyracoidea, and Sirenia within Paenungulata and suggest that Paenungulata may be as close to a real, or at least unresolvable, polytomy as possible.\nPatterns of physical activity in hunter-gatherer children compared with US and UK children.\nAuthors: Kretschmer, L.; Dyble, M.; Chaudhary, N.; Bann, D.; Salali, G. D.\nScore: 17.8, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569171\nSedentary lifestyles, mismatched with our active foraging history, contribute to escalating rates of non-communicable diseases. Contemporary hunter-gatherers appear to be highly active, but little is known about physical activity levels in hunter-gatherer children. We analysed 150 days of accelerometer data from 51 BaYaka hunter-gatherer children (aged 3-18) in the Republic of Congo, comparing it with British and American children (MCS and NHANES). BaYaka children were highly active, engaging in over 3 hours of moderate-to-vigorous physical activity (MVPA) daily, surpassing British adolescents by over 70 minutes. In US children activity declined with age; while in BaYaka children activity increased with age, unaffected by gender. Reflecting their foraging lifestyle, activity patterns varied within and between days, yet all children consistently rose with the sun. These findings highlight the impact of a foraging upbringing on childrens activity levels, providing a benchmark for understanding childhood physical activity and wellbeing.\nPhysical constraints during Snowball Earth drive the evolution of multicellularity\nAuthors: Crockett, W. W.; Shaw, J.; Simpson, C.; Kempes, C. P.\nScore: 10.6, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570654\nMolecular and fossil evidence suggest that complex eukaryotic multicellularity evolved during the late Neoproterozoic era, coincident with Snowball Earth glaciations, where ice sheets covered most of the globe. During this period, environmental conditions--such as sea water temperature and the availability of photosynthetically active light in the oceans--likely changed dramatically. Such changes would have had significant effects on both resource availability and optimal phenotypes. Here, we construct and apply mechanistic models to explore (i) how environmental changes during Snowball Earth and biophysical constraints generated selective pressures and (ii) how these pressures may have had differential effects on organisms with different forms of biological organization. By testing a series of alternate--and commonly debated--hypotheses, we demonstrate how multicellularity was likely acquired differently in eukaryotes and prokaryotes due to selective differences in the biophysical and metabolic regimes they experience: decreasing temperatures and resource-availability instigated by the onset of glaciations generated selective pressures towards smaller sizes in organisms in a diffusive regime and towards larger sizes in motile heterotrophs. These results suggest that changing environmental conditions during Snowball Earth glaciations gave multicellular eukaryotes an evolutionary advantage, paving the way for the complex multicellular lineages that followed.\nMolecular Characterization of the Sea Lamprey Retina Illuminates the Evolutionary Origin of Retinal Cell Types\nAuthors: Wang, J.; Zhang, L.; Cavallini, M.; Pahlevan, A.; Sun, J.; Morshedian, A.; Fain, G. L.; Sampath, A. P.; Peng, Y.-R.\nScore: 10.4, Published: 2023-12-12 DOI: 10.1101/2023.12.10.571000\nThe lamprey, a primitive jawless vertebrate whose ancestors diverged from all other vertebrates over 500 million years ago, offers a unique window into the primordial formation of the retina. Using single-cell RNA-sequencing, we characterized retinal cell types in lamprey and compared their molecular differentiation and regulatory networks with those in mouse and other jawed vertebrates. Our analysis revealed six cell classes and 74 distinct cell types. We discovered multiple conserved cell types shared between jawless and jawed lineages, including notably rods and cones, ON and OFF bipolar cells, and starburst amacrine cells. The conservation of these cell types indicates their emergence early in vertebrate evolution, highlighting the primal designs of retinal circuits for the rod pathway, ON-OFF discrimination, and direction selectivity. In contrast to this evidence for conservation, the pathways of diversification for amacrine cells and retinal ganglion cells appear to have distinctly diverged between the two lineages. Furthermore, we inferred master regulators in specifying retinal cell classes in both lamprey and macaque and identified common regulatory elements across species, underscoring the ancestral nature of the molecular origins governing retinal cell classes. Altogether, our characterization of the lamprey retina illuminates the evolutionary origin of visual processing in the retina.\nGenetic architecture, spatial heterogeneity, and the coevolutionary arms race between newts and snakes\nAuthors: Caudill, V.; Ralph, P. L.\nScore: 8.7, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570693\nCoevolution between two species can lead to exaggerated phenotypes that vary in a correlated manner across space. However, the conditions under which we expect such spatially varying coevolutionary patterns in polygenic traits are not well-understood. We investigate the coevolutionary dynamics between two species undergoing reciprocal adaptation across space and time, using simulations inspired by the Taricha newt - Thamnophis garter snake system. One striking observation from this system is that newts in some areas carry much more tetrodotoxin than in other areas, and garter snakes that live near more toxic newts tend to be more resistant to this toxin, a correlation seen across several broad geographic areas. Furthermore, snakes seem to be \"winning\" the coevolutinary arms race, i.e., having a high level of resistance compared to local newt toxicity, despite substantial variation in both toxicity and resistance across the range. We explore how possible genetic architectures of the toxin and resistance traits would affect the coevolutionary dynamics by manipulating both mutation rate and effect size of mutations across many simulations. We find that coevolutionary dynamics alone were not sufficient in our simulations to produce the striking mosaic of levels of toxicity and resistance observed in nature, but simulations with ecological heterogeneity (in trait costliness or interaction rate) did produce such patterns. We also find that in simulations, newts tend to \"win\" across most combinations of genetic architectures, although the species with higher mutational genetic variance tends to have an advantage.\nIndividual differences in developmental trajectory leave a male polyphenic signature in bulb mite populations\nAuthors: Deere, J. A.; Smallegange, I. M.\nScore: 10.4, Published: 2023-11-30 DOI: 10.1101/2023.02.06.527265\nDevelopmental plasticity alters phenotypes and can in that way change the response to selection. When alternative phenotypes show different life history trajectories, developmental plasticity can also affect, and be affected by, population size-structure in an eco-evolutionary interaction. Developmental plasticity often functions to anticipate future conditions but it can also mitigate current stress conditions. Both types of developmental plasticity have evolved under different selections and this raises the question if they underlie different eco-evolutionary population dynamics. Here, we tested, in a long-term population experiment using the male polyphenic bulb mite (Rhizoglyphus robini), if the selective harvesting of juveniles of different developmental stages concurrently alters population size (ecological response) and male adult phenotype expression (evolutionary response) in line with eco-evolutionary predictions that assume the male polyphenism is anticipatory or mitigating. We found that the frequency of adult males that expressed costly (fighter) morphology was lowest under the most severe juvenile harvesting conditions. This response cannot be explained if we assume that adult male phenotype expression is to anticipate adult (mating) conditions because, in that case, only the manipulation of adult performance would have an effect. Instead, we suggest that juveniles mitigate their increased mortality risk by expediating ontogeny to forego the development of costly morphology and mature quicker but as a defenceless scrambler. If, like in mammals and birds where early-life stress effects are extensively studied, we account for such pre-adult viability selection in coldblooded species, it would allow us to (i) better characterise natural selection on trait development like male polyphenisms, (ii) understand how it can affect the response to other selections in adulthood, and (iii) understand how such trait dynamics influence, and are influenced by, population dynamics.\n",
  "wordCount" : "2774",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570141">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570141" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570141">
        <p class="paperTitle">Allelic gene conversion softens selective sweeps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570141" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570141" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schrider, D. R.</p>
        <p class="info">Score: 20.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570141' target='https://doi.org/10.1101/2023.12.05.570141'> 10.1101/2023.12.05.570141</a></p>
        <p class="abstract">The prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele &#34;sweep&#34; through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection. Two models of selective sweeps have been proposed: classical sweeps, or &#34;hard sweeps&#34;, in which a single copy of the adaptive allele sweeps to fixation, and &#34;soft sweeps&#34;, in which multiple distinct copies of the adaptive allele leave descendants after the sweep. Soft sweeps can be the outcome of recurrent mutation to the adaptive allele, or the presence of standing genetic variation consisting of multiple copies of the adaptive allele prior to the onset of selection. Importantly, soft sweeps will be common when populations can rapidly adapt to novel selective pressures, either because of a high mutation rate or because adaptive alleles are already present. The prevalence of soft sweeps is especially controversial, and it has been noted that selection on standing variation or recurrent mutations may not always produce soft sweeps. Here, we show that the inverse is true: selection on single-origin de novo mutations may often result in an outcome that is indistinguishable from a soft sweep. This is made possible by allelic gene conversion, which &#34;softens&#34; hard sweeps by copying the adaptive allele onto multiple genetic backgrounds, a process we refer to as a &#34;pseudo-soft&#34; sweep. We carried out a simulation study examining the impact of gene conversion on sweeps from a single de novo variant in models of human, Drosophila, and Arabidopsis populations. The fraction of simulations in which gene conversion had produced multiple haplotypes with the adaptive allele upon fixation was appreciable. Indeed, under realistic demographic histories and gene conversion rates, even if selection always acts on a single-origin mutation, sweeps involving multiple haplotypes are more likely than hard sweeps in large populations, especially when selection is not extremely strong. Thus, even when the mutation rate is low or there is no standing variation, hard sweeps are expected to be the exception rather than the rule in large populations. These results also imply that the presence of signatures of soft sweeps does not necessarily mean that adaptation has been especially rapid or is not mutation limited.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570457">
        <p class="paperTitle">Reproductive life history may modulate selection on the germline mutation rate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, L.; Beichman, A. C.; Harris, K.</p>
        <p class="info">Score: 19.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570457' target='https://doi.org/10.1101/2023.12.06.570457'> 10.1101/2023.12.06.570457</a></p>
        <p class="abstract">A long lifespan is a potential liability from the point of view of mutation accumulation, allowing more time for cancer-causing mutations to accumulate in somatic tissues. In practice, however, long-lived organisms do not seem to suffer more cancer mortality than short-lived organisms, a pattern known as &#34;Petos Paradox.&#34; One suggested resolution to Petos Paradox is that long lifespans may facilitate the evolution of better somatic DNA repair. This comes into some conflict with the predictions of the drift-barrier hypothesis of mutation rate evolution, which infers that germline DNA repair efficacy should be best in short-lived organisms with high genetic diversity and efficient natural selection. Here, we show that this conflict can be resolved by modeling the tendency of long generation times to increase the strength of selection against mutator alleles that affect clocklike mutational processes. By considering the empirical joint distribution of generation time and effective population size, we show that stronger selection against mutator alleles in long-lived organisms is likely to be partially but not completely offset by their tendency to have low effective population sizes. This model resolves a conflict between the predictions of theories of germline and somatic mutation rate evolution, leading to the prediction that both germline and somatic DNA repair should be most efficient in long-lived organisms despite their lower genetic diversity.

Significance StatementAll cells accumulate mutations due to DNA damage and replication errors. Mutations affecting the germline can harm future offspring, while mutations in other tissues can threaten longevity. Previous theories have predicted that somatic DNA repair should be most effective in long-lived organisms, but that germline DNA repair should instead be most effective in organisms with high genetic diversity that tend to be short-lived. We model how reproductive age should affect germline mutation rate evolution and show that this may reverse the prediction that high-diversity species should have more efficient selection for effective germline DNA repair. By demonstrating how population size and lifespan likely shape selection on mutagenesis, we are able to harmonize predictions about germline and somatic mutation rate evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569647">
        <p class="paperTitle">Examining the role of oxygen-binding proteins on the early evolution of multicellularity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, W.; Bravo, P.; Yunker, P. J.; Ratcliff, W. C.; Burnetti, A. J.</p>
        <p class="info">Score: 21.9, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569647' target='https://doi.org/10.1101/2023.12.01.569647'> 10.1101/2023.12.01.569647</a></p>
        <p class="abstract">Oxygen availability is a key factor in the evolution of multicellularity, as larger and more sophisticated organisms often require mechanisms allowing efficient oxygen delivery to their tissues. One such mechanism is the presence of oxygen-binding proteins, such as globins and hemerythrins, which arose in the ancestor of bilaterian animals. Despite their importance, the precise mechanisms by which oxygen-binding proteins influenced the early stages of multicellular evolution under varying environmental oxygen levels are not yet clear. We addressed this knowledge gap by heterologously expressing the oxygen binding proteins myoglobin and myohemerythrin in snowflake yeast, a model system of simple, undifferentiated multicellularity. These proteins increased the depth and rate of oxygen diffusion, increasing the fitness of snowflake yeast growing aerobically. Experiments show that, paradoxically, oxygen-binding proteins confer a greater fitness benefit for larger organisms under high, not low, O2 conditions. We show via biophysical modeling that this is because facilitated diffusion is more efficient when oxygen is abundant, transporting a greater quantity of O2 which can be used for metabolism. By alleviating anatomical diffusion limitations to oxygen consumption, the evolution of O2-binding proteins in the oxygen-rich Neoproterozoic may have been a key breakthrough enabling the evolution of increasingly large, complex multicellular metazoan lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569981">
        <p class="paperTitle">Rapid turnover of pathogen-blocking Wolbachia and their incompatibility loci</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shropshire, D.; Conner, W. R.; Vanderpool, D.; Hoffmann, A. A.; Turelli, M.; Cooper, B. S.</p>
        <p class="info">Score: 13.9, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569981' target='https://doi.org/10.1101/2023.12.04.569981'> 10.1101/2023.12.04.569981</a></p>
        <p class="abstract">At least half of all insect species carry maternally inherited Wolbachia alphaproteobacteria, making Wolbachia the most common endosymbionts in nature. Wolbachia spread to high frequencies is often due to cytoplasmic incompatibility (CI), a Wolbachia-induced sperm modification that kills embryos without Wolbachia. Several CI-causing Wolbachia variants, including wMel from Drosophila melanogaster, also block viruses. Establishing pathogen-blocking wMel in natural Aedes aegypti mosquito populations has reduced dengue disease incidence, with one study reporting about 85% reduction when wMel frequency is high. However, wMel transinfection establishment is challenging in many environments, highlighting the importance of identifying CI-causing Wolbachia variants that stably persist in diverse hosts and habitats. We demonstrate that wMel-like variants have naturally established in widely distributed holometabolous dipteran and hymenopteran insects that diverged approximately 350 million years ago, with wMel variants spreading rapidly among these hosts over only the last 100,000 years. Wolbachia genomes contain prophages that encode CI-causing operons (cifs). These cifs move among Wolbachia genomes - with and without prophages - even more rapidly than Wolbachia move among insect hosts. Our results shed light on how rapid host switching and horizontal gene transfer contribute to Wolbachia and cif diversity in nature. The diverse wMel variants we report here from hosts present in different climates offer many new options for broadening Wolbachia-based biocontrol of diseases and pests.

Significance StatementWolbachia bacteria transinfected into Aedes aegypti mosquitoes (in particular the wMel Wolbachia strain) are being widely released to suppress the transmission of dengue and other insect-vectored arboviruses. The bacterial strain wMel originated from Drosophila melanogaster, but very little is known about its distribution and spread to other drosophilids and distantly related insects. Here we show that there has been a rapid rate of spread of Wolbachia strains closely related to wMel not just into other flies but also into wasps, indicating a large pool of wMel variants that can be employed in vector-borne disease control. Wolbachia prophage genes essential to this spread have evolved particularly rapidly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570590">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570590" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570590">
        <p class="paperTitle">Phylogenomics reveals an almost perfect polytomy among the almost ungulates (Paenungulata)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570590" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570590" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bowman, J.; Enard, D.; Lynch, V. J.</p>
        <p class="info">Score: 11.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570590' target='https://doi.org/10.1101/2023.12.07.570590'> 10.1101/2023.12.07.570590</a></p>
        <p class="abstract">Phylogenetic studies have resolved most relationships among Eutherian Orders. However, the branching order of elephants (Proboscidea), hyraxes (Hyracoidea), and sea cows (Sirenia) (i.e., the Paenungulata) has remained uncertain since at least 1758, when Linnaeus grouped elephants and manatees into a single Order (Bruta) to the exclusion of hyraxes. Subsequent morphological, molecular, and large-scale phylogenomic datasets have reached conflicting conclusions on the branching order within Paenungulates. We use a phylogenomic dataset of alignments from 13,388 protein-coding genes across 261 Eutherian mammals to infer phylogenetic relationships within Paenungulates. We find that gene trees almost equally support the three alternative resolutions of Paenungulate relationships and that despite strong support for a Proboscidea&#43;Hyracoidea split in the multispecies coalescent (MSC) tree, there is significant evidence for gene tree uncertainty, incomplete lineage sorting, and introgression among Proboscidea, Hyracoidea, and Sirenia. Indeed, only 8-10% of genes have statistically significant phylogenetic signal to reject the hypothesis of a Paenungulate polytomy. These data indicate little support for any resolution for the branching order Proboscidea, Hyracoidea, and Sirenia within Paenungulata and suggest that Paenungulata may be as close to a real, or at least unresolvable, polytomy as possible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569171">
        <p class="paperTitle">Patterns of physical activity in hunter-gatherer children compared with US and UK children.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kretschmer, L.; Dyble, M.; Chaudhary, N.; Bann, D.; Salali, G. D.</p>
        <p class="info">Score: 17.8, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569171' target='https://doi.org/10.1101/2023.11.29.569171'> 10.1101/2023.11.29.569171</a></p>
        <p class="abstract">Sedentary lifestyles, mismatched with our active foraging history, contribute to escalating rates of non-communicable diseases. Contemporary hunter-gatherers appear to be highly active, but little is known about physical activity levels in hunter-gatherer children. We analysed 150 days of accelerometer data from 51 BaYaka hunter-gatherer children (aged 3-18) in the Republic of Congo, comparing it with British and American children (MCS and NHANES). BaYaka children were highly active, engaging in over 3 hours of moderate-to-vigorous physical activity (MVPA) daily, surpassing British adolescents by over 70 minutes. In US children activity declined with age; while in BaYaka children activity increased with age, unaffected by gender. Reflecting their foraging lifestyle, activity patterns varied within and between days, yet all children consistently rose with the sun. These findings highlight the impact of a foraging upbringing on childrens activity levels, providing a benchmark for understanding childhood physical activity and wellbeing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570654">
        <p class="paperTitle">Physical constraints during Snowball Earth drive the evolution of multicellularity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crockett, W. W.; Shaw, J.; Simpson, C.; Kempes, C. P.</p>
        <p class="info">Score: 10.6, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570654' target='https://doi.org/10.1101/2023.12.07.570654'> 10.1101/2023.12.07.570654</a></p>
        <p class="abstract">Molecular and fossil evidence suggest that complex eukaryotic multicellularity evolved during the late Neoproterozoic era, coincident with Snowball Earth glaciations, where ice sheets covered most of the globe. During this period, environmental conditions--such as sea water temperature and the availability of photosynthetically active light in the oceans--likely changed dramatically. Such changes would have had significant effects on both resource availability and optimal phenotypes. Here, we construct and apply mechanistic models to explore (i) how environmental changes during Snowball Earth and biophysical constraints generated selective pressures and (ii) how these pressures may have had differential effects on organisms with different forms of biological organization. By testing a series of alternate--and commonly debated--hypotheses, we demonstrate how multicellularity was likely acquired differently in eukaryotes and prokaryotes due to selective differences in the biophysical and metabolic regimes they experience: decreasing temperatures and resource-availability instigated by the onset of glaciations generated selective pressures towards smaller sizes in organisms in a diffusive regime and towards larger sizes in motile heterotrophs. These results suggest that changing environmental conditions during Snowball Earth glaciations gave multicellular eukaryotes an evolutionary advantage, paving the way for the complex multicellular lineages that followed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.571000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.571000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.571000">
        <p class="paperTitle">Molecular Characterization of the Sea Lamprey Retina Illuminates the Evolutionary Origin of Retinal Cell Types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.571000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.571000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Zhang, L.; Cavallini, M.; Pahlevan, A.; Sun, J.; Morshedian, A.; Fain, G. L.; Sampath, A. P.; Peng, Y.-R.</p>
        <p class="info">Score: 10.4, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.571000' target='https://doi.org/10.1101/2023.12.10.571000'> 10.1101/2023.12.10.571000</a></p>
        <p class="abstract">The lamprey, a primitive jawless vertebrate whose ancestors diverged from all other vertebrates over 500 million years ago, offers a unique window into the primordial formation of the retina. Using single-cell RNA-sequencing, we characterized retinal cell types in lamprey and compared their molecular differentiation and regulatory networks with those in mouse and other jawed vertebrates. Our analysis revealed six cell classes and 74 distinct cell types. We discovered multiple conserved cell types shared between jawless and jawed lineages, including notably rods and cones, ON and OFF bipolar cells, and starburst amacrine cells. The conservation of these cell types indicates their emergence early in vertebrate evolution, highlighting the primal designs of retinal circuits for the rod pathway, ON-OFF discrimination, and direction selectivity. In contrast to this evidence for conservation, the pathways of diversification for amacrine cells and retinal ganglion cells appear to have distinctly diverged between the two lineages. Furthermore, we inferred master regulators in specifying retinal cell classes in both lamprey and macaque and identified common regulatory elements across species, underscoring the ancestral nature of the molecular origins governing retinal cell classes. Altogether, our characterization of the lamprey retina illuminates the evolutionary origin of visual processing in the retina.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570693">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570693" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570693">
        <p class="paperTitle">Genetic architecture, spatial heterogeneity, and the coevolutionary arms race between newts and snakes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570693" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570693" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caudill, V.; Ralph, P. L.</p>
        <p class="info">Score: 8.7, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570693' target='https://doi.org/10.1101/2023.12.07.570693'> 10.1101/2023.12.07.570693</a></p>
        <p class="abstract">Coevolution between two species can lead to exaggerated phenotypes that vary in a correlated manner across space. However, the conditions under which we expect such spatially varying coevolutionary patterns in polygenic traits are not well-understood. We investigate the coevolutionary dynamics between two species undergoing reciprocal adaptation across space and time, using simulations inspired by the Taricha newt - Thamnophis garter snake system. One striking observation from this system is that newts in some areas carry much more tetrodotoxin than in other areas, and garter snakes that live near more toxic newts tend to be more resistant to this toxin, a correlation seen across several broad geographic areas. Furthermore, snakes seem to be &#34;winning&#34; the coevolutinary arms race, i.e., having a high level of resistance compared to local newt toxicity, despite substantial variation in both toxicity and resistance across the range. We explore how possible genetic architectures of the toxin and resistance traits would affect the coevolutionary dynamics by manipulating both mutation rate and effect size of mutations across many simulations. We find that coevolutionary dynamics alone were not sufficient in our simulations to produce the striking mosaic of levels of toxicity and resistance observed in nature, but simulations with ecological heterogeneity (in trait costliness or interaction rate) did produce such patterns. We also find that in simulations, newts tend to &#34;win&#34; across most combinations of genetic architectures, although the species with higher mutational genetic variance tends to have an advantage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.06.527265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.06.527265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.06.527265">
        <p class="paperTitle">Individual differences in developmental trajectory leave a male polyphenic signature in bulb mite populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.06.527265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.06.527265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deere, J. A.; Smallegange, I. M.</p>
        <p class="info">Score: 10.4, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.06.527265' target='https://doi.org/10.1101/2023.02.06.527265'> 10.1101/2023.02.06.527265</a></p>
        <p class="abstract">Developmental plasticity alters phenotypes and can in that way change the response to selection. When alternative phenotypes show different life history trajectories, developmental plasticity can also affect, and be affected by, population size-structure in an eco-evolutionary interaction. Developmental plasticity often functions to anticipate future conditions but it can also mitigate current stress conditions. Both types of developmental plasticity have evolved under different selections and this raises the question if they underlie different eco-evolutionary population dynamics. Here, we tested, in a long-term population experiment using the male polyphenic bulb mite (Rhizoglyphus robini), if the selective harvesting of juveniles of different developmental stages concurrently alters population size (ecological response) and male adult phenotype expression (evolutionary response) in line with eco-evolutionary predictions that assume the male polyphenism is anticipatory or mitigating. We found that the frequency of adult males that expressed costly (fighter) morphology was lowest under the most severe juvenile harvesting conditions. This response cannot be explained if we assume that adult male phenotype expression is to anticipate adult (mating) conditions because, in that case, only the manipulation of adult performance would have an effect. Instead, we suggest that juveniles mitigate their increased mortality risk by expediating ontogeny to forego the development of costly morphology and mature quicker but as a defenceless scrambler. If, like in mammals and birds where early-life stress effects are extensively studied, we account for such pre-adult viability selection in coldblooded species, it would allow us to (i) better characterise natural selection on trait development like male polyphenisms, (ii) understand how it can affect the response to other selections in adulthood, and (iii) understand how such trait dynamics influence, and are influenced by, population dynamics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
