<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.",
  "keywords": [
    
  ],
  "articleBody": " The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.\nScore: 110.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561165\nInfectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.\nDuplication and neofunctionalization of a horizontally-transferred xyloglucanase as a facet of the red queen co-evolutionary dynamic\nAuthors: Attah, V.; Milner, D. S.; Fang, Y.; Yan, X.; Leonard, G.; Heitman, J.; Talbot, N. J.; Richards, T. A.\nScore: 13.1, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561229\nOomycetes are heterotrophic protists that share phenotypic similarities with fungi, including the ability to cause plant diseases, but branch in a separate and distant region of the eukaryotic tree of life. It has been suggested that multiple horizontal gene transfers (HGTs) from fungi-to-oomycetes contributed to the evolution of plant-pathogenic traits. These HGTs are predicted to include secreted proteins that degrade plant cell walls. This is a key trait in the pathology of many oomycetes, as the plant cell wall represents a primary barrier to pathogen invasion and a rich source of carbohydrates. Many of the HGT gene families identified have undergone multiple rounds of duplication. Using a combination of phylogenomic analysis and functional assays, we investigate the diversification of a horizontally-transferred xyloglucanase gene family in the model oomycete species Phytophthora sojae. Our analyses detect 11 genes retained in P. sojae among a complex pattern of gene duplications and losses. Using a phenotype assay, based on heterologous expression in yeast, we show that eight of these paralogs have xyloglucanase function, including variants with distinct protein characteristics, such as a long-disordered C-terminal extension that can increase xyloglucanase activity. The functional xyloglucanase variants analysed subtend an ancestral node close to the fungi-oomycetes gene transfer, suggesting the horizontally-transferred gene was a bona fide xyloglucanase. Expression of xyloglucanase paralogs in Nicotiana benthamiana triggers distinct patterns of reactive oxygen species (ROS) generation, demonstrating that enzyme variants differentially stimulate pattern-triggered immunity in plants. Mass spectrometry of detectable enzymatic products demonstrates that some paralogs catalyze production of variant breakdown profiles, suggesting that secretion of multiple xyloglucanase variants increases efficiency of xyloglucan breakdown, as well as potentially diversifying the range of Damage-Associated Molecular Patterns (DAMPs) released during pathogen attack. We suggest that such patterns of protein neofunctionalization, and variant host responses, represent an aspect of the Red Queen host-pathogen co-evolutionary dynamic. Significance StatementThe oomycetes are a diverse group of eukaryotic microbes that include some of the most devastating pathogens of plants. Oomycetes perceive, invade, and colonize plants in similar ways to fungi, in part because they acquired the genes to attack and feed on plants from fungi. These genes are predicted to be useful to oomycete plant pathogens because they have undergone multiple rounds of gene duplication. One key enzyme for attacking plant cell wall structures is called xyloglucanase. Xyloglucanase in the oomycetes has undergone multiple rounds of gene duplication, leading to variants including an enzyme with a C-terminal extension that increases activity. Some xyloglucanase variants trigger unique patterns of reactive oxygen species (ROS) in planta, and generate different profiles of cell wall breakdown products - such outcomes could act to mystify and increase the workload of the plant immune system, allowing successful pathogens to proliferate.\nComplex genomic landscape of inversion polymorphism in Europe's most destructive forest pest\nAuthors: Mykhailenko, A.; Zielinski, P.; Bednarz, A.; Schlyter, F.; Andersson, M. N.; Antunes, B.; Borowski, Z.; Krokene, P.; Melin, M.; Morales-Garcia, J.; Müller, J.; Nowak, Z.; Schebeck, M.; Stauffer, C.; Viiri, H.; Babik, W.; Nadachowska-Brzyska, K.\nScore: 9.2, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561670\nIn many species, polymorphic inversions underlie complex phenotypic polymorphisms and appear to facilitate local adaptation in the face of gene flow. Multiple polymorphic inversions can co-occur in a genome, but the prevalence, evolutionary significance, and limits to complexity of genomic inversion landscapes remain poorly understood. Here, we examine genome-wide variation in one of Europes most destructive forest pests, the spruce bark beetle Ips typographus, scan for polymorphic inversions, and test whether inversions are involved in key adaptations in this species. We sampled 244 individuals from 18 populations across the species European range and, using a whole-genome resequencing approach, identified 27 polymorphic inversions covering at least 28% of the genome. The inversions vary in size and in levels of intra-inversion recombination, are highly polymorphic across the species range, and often overlap, forming an extremely complex genomic architecture. We show that the heterogeneous inversion landscape is likely maintained by the combined action of several evolutionary forces and that inversions are enriched in odorant receptor genes encoding key elements of recognition pathways for host plants, mates, and symbiotic fungi. Our results indicate that the genome of this major forest pest of growing social, political, and economic importance harbors one of the most complex inversion landscapes described to date and is pushing the limits of genomic architecture complexity.\nThe maintenance of genetic polymorphism in sexually antagonistic traits\nAuthors: Flintham, E.; Savolainen, V.; Otto, S.; Reuter, M.; Mullon, C.\nScore: 8.0, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561678\nSelection often favours different trait values in males and females, leading to genetic conflicts between the sexes, or sexual antagonism. Theory suggests that such conflict can maintain genetic variation by generating balancing selection. However, most of this theory is based on insights from models of single loci with fixed fitness effects. It is thus unclear how readily sexual antagonism drives balancing selection at loci where the fitness effects are not arbitrarily set but emerge from the genotype-phenotype map and the fitness landscape. Here we use mathematical models and computer simulations to investigate the evolution of traits that have different optima in males and females and so experience sexually antagonistic selection. We examine the consequences of such selection for the maintenance and nature of genetic variation under different assumptions of genetic architecture and for different fitness landscapes. We show that the conditions necessary to maintain variation are significantly more restrictive than previously appreciated. This is because the majority of typical sex-specific fitness landscapes generate stabilising selection on traits, and so favour the fixation of a single homozygote genotype that encodes the phenotype with the highest sex-averaged fitness. Instead, the maintenance of genetic polymorphism requires extremely, if not implausibly, strong conflict between males and females in order to disfavour extreme and intermediate phenotypes by generating diversifying selection. Furthermore, in all cases, we find that distinctive patterns of genetic variation produced by sexual antagonism arise rarely and are often transient, and so are difficult to observe in genomic data. Taken together, our results indicate that sex-specific selection is not a straightforward source of balanced genetic polymorphism and leaves few traces in the genome. Rather, we suggest that sexual antagonism is most likely to maintain genetic variation when traits are determined by a single large effect locus where genetic constraints lead to a limited range of possible alleles.\nRelative Homoplasy Index: A New Cross-comparable Metric for Quantifying Homoplasy in Discrete Character Datasets\nAuthors: Steell, E. M.; Hsiang, A. Y.; Field, D. J.\nScore: 7.4, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561677\nHomoplasy is among the main hinderances to phylogenetic inference. However, investigating patterns of homoplasy can also improve our understanding of macroevolution, for instance by revealing evolutionary constraints on morphology, or highlighting convergent form-function relationships. Several methods have been proposed to quantify the extent of homoplasy in discrete character matrices, but the consistency index (CI) and retention index (RI) have remained the most widely used for decades, with little recent scrutiny of their function. Here, we test the performance of CI and RI using simulated and empirical datasets and investigate patterns of homoplasy with different matrix scenarios. In addition, we describe and test a new scaled metric, the relative homoplasy index (RHI), implemented in the R statistical environment. The results suggest that, unlike the RI, the CI does not constitute a direct measure of homoplasy. However, the RI consistently underestimates the extent of homoplasy in phylogenetic character-taxon matrices, particularly in datasets characterised by high levels of homoplasy. By contrast, RHI - the newly proposed metric - outperforms both methods in sensitivity to homoplasy levels, and is scaled between zero and one for comparison of values between different datasets. Using both simulated and empirical phylogenetic datasets, we show that relative levels of homoplasy remain constant with the addition of novel characters, and, in contrast to earlier work, decrease with the addition of taxa. Our results help illuminate the inherent properties of homoplasy in cladistic matrices, opening new potential avenues of research for investigating patterns of homoplasy in macroevolutionary studies.\nContrasting patterns of presence-absence variation of NLRS within S. chilense are mainly shaped by past demographic history\nAuthors: Silva Arias, G. A.; Gagnon, E.; Hembrom, S.; Fastner, A.; Khan, M. R.; Stam, R.; Tellier, A.\nScore: 6.7, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562278\nUnderstanding the evolution of pathogen resistance genes (also known as NLRs) within a species requires a comprehensive examination of factors that affect gene loss and gain. We present a new reference genome of Solanum chilense, that leads to an increased number and more accurate annotation of NLRs. Next, using a target-capture approach, we quantify the presence-absence variation (PAV) of NLR loci across 20 populations from different habitats. We build a rigorous pipeline to validate the identification of PAV of NLRs, then show that PAV is larger within populations than between populations, suggesting that maintenance of NLR diversity is linked to population dynamics. Furthermore, the amount of PAV is not correlated with the NLR presence in gene clusters in the genome, but rather with the past demographic history of the species, with loss of NLRs in diverging populations at the distribution edges and smaller population sizes. Finally, using a redundancy analysis, we find limited evidence of PAV being linked to environmental gradients. Our results contradict the classic assumptions of the important selective role of PAV for NLRs, and suggest that NLRs PAV is driven by random processes (and weak selection) in an outcrossing plant with high nucleotide diversity.\nDietary restriction extends lifespan across different temperatures in the fly\nAuthors: Phillips, E. J.; Simons, M.\nScore: 5.6, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562222\nDietary restriction (DR) has been consistently shown to extend lifespan across a range of taxa. However, it has recently been reported that DR does not extend lifespan at certain, namely lower, temperatures in flies (Drosophila melanogaster). Similar to the interpretation of other findings that appear to question DR's universality, this finding has been interpreted as being an artefact of benign laboratory conditions. Here we re-test this hypothesis, now using a strain that shows robust lifespan extension at 25C (across three prior experiments), and using a range of 5 diets across two temperatures, 18C and 21C. We found the DR longevity response to be robust, extending lifespan irrespective of temperature. We measured fecundity as a positive control for the DR phenotype, and found, as predicted, that DR reduces egg laying. It will be important for results that question DR as a phenotype to not be overinterpreted readily, as with a substantially larger sample size and a larger range of diets we were unable to replicate this prior work. Differences in experimental setup, genetic lines used and variation in the diet-lifespan reaction norm we suggest are responsible for this discrepancy. In addition, starting with a strain and conditions that show a lifespan extension by DR, as we do here, and then changing environment and/or genotype promises a more robust test of DR modulating factors.\nThe role of location propagation for translocon complex relocation during eukaryogenesis\nAuthors: Carilo, I.; Senju, Y.; Robinson, R. C.\nScore: 8.0, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560972\nIn all domains of life, the translationally-active ribosome-translocon complex inserts nascent transmembrane proteins into, and processes and transports signal peptide-containing proteins across, membranes1,2. Eukaryotic translocons are anchored in the endoplasmic reticulum, while the prokaryotic complexes reside in cell membranes. DNA sequence analyses indicate that the eukaryotic Sec61/OST/TRAP translocon is inherited from an Asgard archaea ancestor3,4. However, the mechanism for translocon migration from a peripheral membrane to an internal cellular compartment (the proto-endoplasmic reticulum) during eukaryogenesis is unknown. Here we show that Asgard and eukaryotic ribosome-translocon complexes are intercompatible. We find that fluorescently-tagged Asgard translocon proteins from Candidatus Prometheoarchaeum syntrophicum strain MK-D15, a Lokiarchaeon confirmed to contain no internal cellular membranes, are targeted to the eukaryotic endoplasmic reticulum on ectopic expression. Our data demonstrate that the location of existing ribosome-translocon complexes, at the protein level, determines the future placement of yet to be translated translocon subunits. This principle predicts that during eukaryogenesis, under positive selection pressure, the relocation of a few translocon complexes to the proto-endoplasmic reticulum will have propagated the new translocon location, leading to their loss from the cell membrane.\nInsights into deuterostome evolution from the biphasic transcriptional programmes of hemichordates\nAuthors: Perez-Posada, A.; Lin, C.-Y.; Fan, T.-P.; Lin, C.-Y.; Chen, Y.-C.; Gomez-Skarmeta, J. L.; Yu, J.-K.; Su, Y.-H.; Tena, J. J.\nScore: 18.9, Published: 2023-10-11 DOI: 10.1101/2022.06.10.495707\nEvolutionary history of deuterostomes remains unsolved and is intimately related to the origin of chordates. Among deuterostomes, hemichordates and echinoderms (collectively called Ambulacraria) are sister groups of chordates. Comparative studies involving these three groups provide valuable insights into deuterostome evolution. Indirect developing hemichordates produce planktonic larvae that bear resemblance to echinoderm larvae before undergoing metamorphosis into an adult body plan with anteroposterior polarity homologous to that of chordates. Therefore, understanding the developmental processes of indirect-developing hemichordates can help understand the evolution of deuterostomes and the origins of chordates. In this study, we analysed the transcriptomes and chromatin accessibility of multiple developmental stages of the indirect-developing hemichordate Ptychodera flava and discovered that it exhibits a biphasic developmental program controlled by distinct sets of transcription factors and their corresponding regulatory elements. Comparative analyses of transcriptomes and network analyses revealed that the gastrula transcriptome is relatively ancient, and the TFs orchestrating its gene expression are highly interconnected in networks of cis-regulatory interactions. Comparing the developmental transcriptomes of hemichordates, echinoderms, and amphioxus, revealed high conservation of gene expression during gastrulation that extends to the neurula stages of amphioxus, along with remarkable similarity in larval transcriptomes across the three species. Additionally, we show that P. flava possesses conserved interactions of transcription factors necessary for the development of echinoderm endomesoderm and chordate axial mesoderm, including conserved cis-regulatory elements of the FoxA transcription factor that is central to the two networks. These findings suggest the existence of a deuterostome phylotypic stage during gastrulation governed by gene regulatory networks with conserved cis-regulatory interactions. Conversely, integration of gene expression data with synteny data revealed that gene expression recapitulates the independent evolutionary history of the Ancestral Linkage Groups that underwent rearrangements in each deuterostome lineage, suggesting a potential role of genome rearrangement during the evolution of larval strategies in hemichordates and deuterostome body plans.\nHidden evolutionary constraints dictate the retention of coronavirus accessory genes\nAuthors: Goldstein, S. A.; Feeley, T. M.; Babler, K. M.; Hilbert, Z. A.; Downhour, D. M.; Moshiri, N.; Elde, N. C.\nScore: 4.1, Published: 2023-10-14 DOI: 10.1101/2023.10.12.561935\nGenetic innovation is fundamental to the ability of viruses to adapt in the face of host immunity. Coronaviruses exhibit many mechanisms of innovation given flexibility in genomic composition relative to most RNA virus families (1-5). Examples include the acquisition of unique accessory genes that can originate by capture of cellular genes or through duplication and divergence of existing viral genes (6-8). Accessory genes may be influential in dictating viral host range and cellular tropism, but little is known about how selection acts on these variable regions of virus genomes. We used experimental evolution of mouse hepatitis virus (MHV) with an inactive native phosphodiesterase, NS2, that encodes a complementing cellular AKAP7 gene (9), to simulate the capture of a host gene and found hidden patterns of constraint that determine the fate of coronavirus accessory genes. After courses of serial infection, AKAP7 was retained under strong selection but rapidly lost under relaxed selection. In contrast, the gene encoding inactive NS2, ORF2, remained intact, suggesting it is under cryptic evolutionary constraint. Guided by the retention of ORF2 and hints of similar patterns in related betacoronaviruses, we analyzed the evolution of SARS-CoV-2 ORF8, which arose via gene duplication (6) and contains premature stop codons in several globally successful lineages. As with MHV ORF2, the coding-defective SARS-CoV-2 ORF8 gene remains largely intact, mirroring patterns observed during MHV experimental evolution and extending these findings to viruses currently adapting to humans. Retention of inactive genes challenges assumptions on the dynamics of gene loss in virus genomes and can help guide evolutionary analysis of emerging and pandemic coronaviruses.\n",
  "wordCount" : "3132",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561165">
        <p class="paperTitle">The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.</p>
        <p class="info">Score: 110.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561165' target='https://doi.org/10.1101/2023.10.06.561165'> 10.1101/2023.10.06.561165</a></p>
        <p class="abstract">Infectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561229">
        <p class="paperTitle">Duplication and neofunctionalization of a horizontally-transferred xyloglucanase as a facet of the red queen co-evolutionary dynamic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Attah, V.; Milner, D. S.; Fang, Y.; Yan, X.; Leonard, G.; Heitman, J.; Talbot, N. J.; Richards, T. A.</p>
        <p class="info">Score: 13.1, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561229' target='https://doi.org/10.1101/2023.10.09.561229'> 10.1101/2023.10.09.561229</a></p>
        <p class="abstract">Oomycetes are heterotrophic protists that share phenotypic similarities with fungi, including the ability to cause plant diseases, but branch in a separate and distant region of the eukaryotic tree of life. It has been suggested that multiple horizontal gene transfers (HGTs) from fungi-to-oomycetes contributed to the evolution of plant-pathogenic traits. These HGTs are predicted to include secreted proteins that degrade plant cell walls. This is a key trait in the pathology of many oomycetes, as the plant cell wall represents a primary barrier to pathogen invasion and a rich source of carbohydrates. Many of the HGT gene families identified have undergone multiple rounds of duplication. Using a combination of phylogenomic analysis and functional assays, we investigate the diversification of a horizontally-transferred xyloglucanase gene family in the model oomycete species Phytophthora sojae. Our analyses detect 11 genes retained in P. sojae among a complex pattern of gene duplications and losses. Using a phenotype assay, based on heterologous expression in yeast, we show that eight of these paralogs have xyloglucanase function, including variants with distinct protein characteristics, such as a long-disordered C-terminal extension that can increase xyloglucanase activity. The functional xyloglucanase variants analysed subtend an ancestral node close to the fungi-oomycetes gene transfer, suggesting the horizontally-transferred gene was a bona fide xyloglucanase. Expression of xyloglucanase paralogs in Nicotiana benthamiana triggers distinct patterns of reactive oxygen species (ROS) generation, demonstrating that enzyme variants differentially stimulate pattern-triggered immunity in plants. Mass spectrometry of detectable enzymatic products demonstrates that some paralogs catalyze production of variant breakdown profiles, suggesting that secretion of multiple xyloglucanase variants increases efficiency of xyloglucan breakdown, as well as potentially diversifying the range of Damage-Associated Molecular Patterns (DAMPs) released during pathogen attack. We suggest that such patterns of protein neofunctionalization, and variant host responses, represent an aspect of the Red Queen host-pathogen co-evolutionary dynamic.

Significance StatementThe oomycetes are a diverse group of eukaryotic microbes that include some of the most devastating pathogens of plants. Oomycetes perceive, invade, and colonize plants in similar ways to fungi, in part because they acquired the genes to attack and feed on plants from fungi. These genes are predicted to be useful to oomycete plant pathogens because they have undergone multiple rounds of gene duplication. One key enzyme for attacking plant cell wall structures is called xyloglucanase. Xyloglucanase in the oomycetes has undergone multiple rounds of gene duplication, leading to variants including an enzyme with a C-terminal extension that increases activity. Some xyloglucanase variants trigger unique patterns of reactive oxygen species (ROS) in planta, and generate different profiles of cell wall breakdown products - such outcomes could act to mystify and increase the workload of the plant immune system, allowing successful pathogens to proliferate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561670">
        <p class="paperTitle">Complex genomic landscape of inversion polymorphism in Europe&#39;s most destructive forest pest</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mykhailenko, A.; Zielinski, P.; Bednarz, A.; Schlyter, F.; Andersson, M. N.; Antunes, B.; Borowski, Z.; Krokene, P.; Melin, M.; Morales-Garcia, J.; Müller, J.; Nowak, Z.; Schebeck, M.; Stauffer, C.; Viiri, H.; Babik, W.; Nadachowska-Brzyska, K.</p>
        <p class="info">Score: 9.2, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561670' target='https://doi.org/10.1101/2023.10.10.561670'> 10.1101/2023.10.10.561670</a></p>
        <p class="abstract">In many species, polymorphic inversions underlie complex phenotypic polymorphisms and appear to facilitate local adaptation in the face of gene flow. Multiple polymorphic inversions can co-occur in a genome, but the prevalence, evolutionary significance, and limits to complexity of genomic inversion landscapes remain poorly understood. Here, we examine genome-wide variation in one of Europes most destructive forest pests, the spruce bark beetle Ips typographus, scan for polymorphic inversions, and test whether inversions are involved in key adaptations in this species. We sampled 244 individuals from 18 populations across the species European range and, using a whole-genome resequencing approach, identified 27 polymorphic inversions covering at least 28% of the genome. The inversions vary in size and in levels of intra-inversion recombination, are highly polymorphic across the species range, and often overlap, forming an extremely complex genomic architecture. We show that the heterogeneous inversion landscape is likely maintained by the combined action of several evolutionary forces and that inversions are enriched in odorant receptor genes encoding key elements of recognition pathways for host plants, mates, and symbiotic fungi. Our results indicate that the genome of this major forest pest of growing social, political, and economic importance harbors one of the most complex inversion landscapes described to date and is pushing the limits of genomic architecture complexity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561678">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561678" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561678">
        <p class="paperTitle">The maintenance of genetic polymorphism in sexually antagonistic traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561678" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561678" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Flintham, E.; Savolainen, V.; Otto, S.; Reuter, M.; Mullon, C.</p>
        <p class="info">Score: 8.0, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561678' target='https://doi.org/10.1101/2023.10.10.561678'> 10.1101/2023.10.10.561678</a></p>
        <p class="abstract">Selection often favours different trait values in males and females, leading to genetic conflicts between the sexes, or sexual antagonism. Theory suggests that such conflict can maintain genetic variation by generating balancing selection. However, most of this theory is based on insights from models of single loci with fixed fitness effects. It is thus unclear how readily sexual antagonism drives balancing selection at loci where the fitness effects are not arbitrarily set but emerge from the genotype-phenotype map and the fitness landscape. Here we use mathematical models and computer simulations to investigate the evolution of traits that have different optima in males and females and so experience sexually antagonistic selection. We examine the consequences of such selection for the maintenance and nature of genetic variation under different assumptions of genetic architecture and for different fitness landscapes. We show that the conditions necessary to maintain variation are significantly more restrictive than previously appreciated. This is because the majority of typical sex-specific fitness landscapes generate stabilising selection on traits, and so favour the fixation of a single homozygote genotype that encodes the phenotype with the highest sex-averaged fitness. Instead, the maintenance of genetic polymorphism requires extremely, if not implausibly, strong conflict between males and females in order to disfavour extreme and intermediate phenotypes by generating diversifying selection. Furthermore, in all cases, we find that distinctive patterns of genetic variation produced by sexual antagonism arise rarely and are often transient, and so are difficult to observe in genomic data. Taken together, our results indicate that sex-specific selection is not a straightforward source of balanced genetic polymorphism and leaves few traces in the genome. Rather, we suggest that sexual antagonism is most likely to maintain genetic variation when traits are determined by a single large effect locus where genetic constraints lead to a limited range of possible alleles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561677">
        <p class="paperTitle">Relative Homoplasy Index: A New Cross-comparable Metric for Quantifying Homoplasy in Discrete Character Datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steell, E. M.; Hsiang, A. Y.; Field, D. J.</p>
        <p class="info">Score: 7.4, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561677' target='https://doi.org/10.1101/2023.10.10.561677'> 10.1101/2023.10.10.561677</a></p>
        <p class="abstract">Homoplasy is among the main hinderances to phylogenetic inference. However, investigating patterns of homoplasy can also improve our understanding of macroevolution, for instance by revealing evolutionary constraints on morphology, or highlighting convergent form-function relationships. Several methods have been proposed to quantify the extent of homoplasy in discrete character matrices, but the consistency index (CI) and retention index (RI) have remained the most widely used for decades, with little recent scrutiny of their function. Here, we test the performance of CI and RI using simulated and empirical datasets and investigate patterns of homoplasy with different matrix scenarios. In addition, we describe and test a new scaled metric, the relative homoplasy index (RHI), implemented in the R statistical environment. The results suggest that, unlike the RI, the CI does not constitute a direct measure of homoplasy. However, the RI consistently underestimates the extent of homoplasy in phylogenetic character-taxon matrices, particularly in datasets characterised by high levels of homoplasy. By contrast, RHI - the newly proposed metric - outperforms both methods in sensitivity to homoplasy levels, and is scaled between zero and one for comparison of values between different datasets. Using both simulated and empirical phylogenetic datasets, we show that relative levels of homoplasy remain constant with the addition of novel characters, and, in contrast to earlier work, decrease with the addition of taxa. Our results help illuminate the inherent properties of homoplasy in cladistic matrices, opening new potential avenues of research for investigating patterns of homoplasy in macroevolutionary studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562278">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562278" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562278">
        <p class="paperTitle">Contrasting patterns of presence-absence variation of NLRS within S. chilense are mainly shaped by past demographic history</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562278" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562278" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silva Arias, G. A.; Gagnon, E.; Hembrom, S.; Fastner, A.; Khan, M. R.; Stam, R.; Tellier, A.</p>
        <p class="info">Score: 6.7, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562278' target='https://doi.org/10.1101/2023.10.13.562278'> 10.1101/2023.10.13.562278</a></p>
        <p class="abstract">Understanding the evolution of pathogen resistance genes (also known as NLRs) within a species requires a comprehensive examination of factors that affect gene loss and gain. We present a new reference genome of Solanum chilense, that leads to an increased number and more accurate annotation of NLRs. Next, using a target-capture approach, we quantify the presence-absence variation (PAV) of NLR loci across 20 populations from different habitats. We build a rigorous pipeline to validate the identification of PAV of NLRs, then show that PAV is larger within populations than between populations, suggesting that maintenance of NLR diversity is linked to population dynamics. Furthermore, the amount of PAV is not correlated with the NLR presence in gene clusters in the genome, but rather with the past demographic history of the species, with loss of NLRs in diverging populations at the distribution edges and smaller population sizes. Finally, using a redundancy analysis, we find limited evidence of PAV being linked to environmental gradients. Our results contradict the classic assumptions of the important selective role of PAV for NLRs, and suggest that NLRs PAV is driven by random processes (and weak selection) in an outcrossing plant with high nucleotide diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562222" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562222">
        <p class="paperTitle">Dietary restriction extends lifespan across different temperatures in the fly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phillips, E. J.; Simons, M.</p>
        <p class="info">Score: 5.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562222' target='https://doi.org/10.1101/2023.10.13.562222'> 10.1101/2023.10.13.562222</a></p>
        <p class="abstract">Dietary restriction (DR) has been consistently shown to extend lifespan across a range of taxa. However, it has recently been reported that DR does not extend lifespan at certain, namely lower, temperatures in flies (Drosophila melanogaster). Similar to the interpretation of other findings that appear to question DR&#39;s universality, this finding has been interpreted as being an artefact of benign laboratory conditions. Here we re-test this hypothesis, now using a strain that shows robust lifespan extension at 25C (across three prior experiments), and using a range of 5 diets across two temperatures, 18C and 21C. We found the DR longevity response to be robust, extending lifespan irrespective of temperature. We measured fecundity as a positive control for the DR phenotype, and found, as predicted, that DR reduces egg laying. It will be important for results that question DR as a phenotype to not be overinterpreted readily, as with a substantially larger sample size and a larger range of diets we were unable to replicate this prior work. Differences in experimental setup, genetic lines used and variation in the diet-lifespan reaction norm we suggest are responsible for this discrepancy. In addition, starting with a strain and conditions that show a lifespan extension by DR, as we do here, and then changing environment and/or genotype promises a more robust test of DR modulating factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560972">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560972" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560972">
        <p class="paperTitle">The role of location propagation for translocon complex relocation during eukaryogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560972" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560972" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carilo, I.; Senju, Y.; Robinson, R. C.</p>
        <p class="info">Score: 8.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560972' target='https://doi.org/10.1101/2023.10.04.560972'> 10.1101/2023.10.04.560972</a></p>
        <p class="abstract">In all domains of life, the translationally-active ribosome-translocon complex inserts nascent transmembrane proteins into, and processes and transports signal peptide-containing proteins across, membranes1,2. Eukaryotic translocons are anchored in the endoplasmic reticulum, while the prokaryotic complexes reside in cell membranes. DNA sequence analyses indicate that the eukaryotic Sec61/OST/TRAP translocon is inherited from an Asgard archaea ancestor3,4. However, the mechanism for translocon migration from a peripheral membrane to an internal cellular compartment (the proto-endoplasmic reticulum) during eukaryogenesis is unknown. Here we show that Asgard and eukaryotic ribosome-translocon complexes are intercompatible. We find that fluorescently-tagged Asgard translocon proteins from Candidatus Prometheoarchaeum syntrophicum strain MK-D15, a Lokiarchaeon confirmed to contain no internal cellular membranes, are targeted to the eukaryotic endoplasmic reticulum on ectopic expression. Our data demonstrate that the location of existing ribosome-translocon complexes, at the protein level, determines the future placement of yet to be translated translocon subunits. This principle predicts that during eukaryogenesis, under positive selection pressure, the relocation of a few translocon complexes to the proto-endoplasmic reticulum will have propagated the new translocon location, leading to their loss from the cell membrane.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.06.10.495707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.06.10.495707" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.06.10.495707">
        <p class="paperTitle">Insights into deuterostome evolution from the biphasic transcriptional programmes of hemichordates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.06.10.495707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.06.10.495707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perez-Posada, A.; Lin, C.-Y.; Fan, T.-P.; Lin, C.-Y.; Chen, Y.-C.; Gomez-Skarmeta, J. L.; Yu, J.-K.; Su, Y.-H.; Tena, J. J.</p>
        <p class="info">Score: 18.9, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.06.10.495707' target='https://doi.org/10.1101/2022.06.10.495707'> 10.1101/2022.06.10.495707</a></p>
        <p class="abstract">Evolutionary history of deuterostomes remains unsolved and is intimately related to the origin of chordates. Among deuterostomes, hemichordates and echinoderms (collectively called Ambulacraria) are sister groups of chordates. Comparative studies involving these three groups provide valuable insights into deuterostome evolution. Indirect developing hemichordates produce planktonic larvae that bear resemblance to echinoderm larvae before undergoing metamorphosis into an adult body plan with anteroposterior polarity homologous to that of chordates. Therefore, understanding the developmental processes of indirect-developing hemichordates can help understand the evolution of deuterostomes and the origins of chordates. In this study, we analysed the transcriptomes and chromatin accessibility of multiple developmental stages of the indirect-developing hemichordate Ptychodera flava and discovered that it exhibits a biphasic developmental program controlled by distinct sets of transcription factors and their corresponding regulatory elements. Comparative analyses of transcriptomes and network analyses revealed that the gastrula transcriptome is relatively ancient, and the TFs orchestrating its gene expression are highly interconnected in networks of cis-regulatory interactions. Comparing the developmental transcriptomes of hemichordates, echinoderms, and amphioxus, revealed high conservation of gene expression during gastrulation that extends to the neurula stages of amphioxus, along with remarkable similarity in larval transcriptomes across the three species. Additionally, we show that P. flava possesses conserved interactions of transcription factors necessary for the development of echinoderm endomesoderm and chordate axial mesoderm, including conserved cis-regulatory elements of the FoxA transcription factor that is central to the two networks. These findings suggest the existence of a deuterostome phylotypic stage during gastrulation governed by gene regulatory networks with conserved cis-regulatory interactions. Conversely, integration of gene expression data with synteny data revealed that gene expression recapitulates the independent evolutionary history of the Ancestral Linkage Groups that underwent rearrangements in each deuterostome lineage, suggesting a potential role of genome rearrangement during the evolution of larval strategies in hemichordates and deuterostome body plans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.561935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.561935" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.561935">
        <p class="paperTitle">Hidden evolutionary constraints dictate the retention of coronavirus accessory genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.561935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.561935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goldstein, S. A.; Feeley, T. M.; Babler, K. M.; Hilbert, Z. A.; Downhour, D. M.; Moshiri, N.; Elde, N. C.</p>
        <p class="info">Score: 4.1, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.561935' target='https://doi.org/10.1101/2023.10.12.561935'> 10.1101/2023.10.12.561935</a></p>
        <p class="abstract">Genetic innovation is fundamental to the ability of viruses to adapt in the face of host immunity. Coronaviruses exhibit many mechanisms of innovation given flexibility in genomic composition relative to most RNA virus families (1-5). Examples include the acquisition of unique accessory genes that can originate by capture of cellular genes or through duplication and divergence of existing viral genes (6-8). Accessory genes may be influential in dictating viral host range and cellular tropism, but little is known about how selection acts on these variable regions of virus genomes. We used experimental evolution of mouse hepatitis virus (MHV) with an inactive native phosphodiesterase, NS2, that encodes a complementing cellular AKAP7 gene (9), to simulate the capture of a host gene and found hidden patterns of constraint that determine the fate of coronavirus accessory genes. After courses of serial infection, AKAP7 was retained under strong selection but rapidly lost under relaxed selection. In contrast, the gene encoding inactive NS2, ORF2, remained intact, suggesting it is under cryptic evolutionary constraint. Guided by the retention of ORF2 and hints of similar patterns in related betacoronaviruses, we analyzed the evolution of SARS-CoV-2 ORF8, which arose via gene duplication (6) and contains premature stop codons in several globally successful lineages. As with MHV ORF2, the coding-defective SARS-CoV-2 ORF8 gene remains largely intact, mirroring patterns observed during MHV experimental evolution and extending these findings to viruses currently adapting to humans. Retention of inactive genes challenges assumptions on the dynamics of gene loss in virus genomes and can help guide evolutionary analysis of emerging and pandemic coronaviruses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
