<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Fundamental equations linking methylation dynamics to maximum lifespan in mammals
Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.
Score: 84.6, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643
We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Fundamental equations linking methylation dynamics to maximum lifespan in mammals
Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.
Score: 84.6, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643
We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Fundamental equations linking methylation dynamics to maximum lifespan in mammals
Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.
Score: 84.6, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643
We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Fundamental equations linking methylation dynamics to maximum lifespan in mammals\nAuthors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.\nScore: 84.6, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643\nWe establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset.",
  "keywords": [
    
  ],
  "articleBody": " Fundamental equations linking methylation dynamics to maximum lifespan in mammals\nAuthors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.\nScore: 84.6, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643\nWe establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset. Our research leads to three key findings: First, a reciprocal relationship exists between the average rate of methylation change (AROCM) in bivalent promoter regions and maximum lifespan: AROCM {propto} 1/MaxLifespan. Second, the correlation between age and average methylation bears no relation to maximum lifespan, Cor(Methly,Age) {perp} MaxLifespan. Third, the rate of methylation change in young animals is related to that in old animals: Young animals AROCM {propto} Old AROCM. These findings hinge on the chromatin context, as different results emerge when defining AROCM using different chromatin states. Our analytical framework is versatile and readily extendable to a broad range of other molecular assessments. Overall, our study demonstrates that epigenetic aging rates in specific chromatin states exhibit an inverse relationship with maximum lifespan in mammals.\nComplete phylogeny of Micrathena spiders suggests multiple dispersal events among Neotropical rainforests, islands, and landmasses, and indicates Andean orogeny promotes speciation\nAuthors: Magalhaes, I. L. F.; Martins, P. H.; Faleiro, B. T.; Vidigal, T. H. D. A.; Santos, F. R.; Carvalho, L. S.; Santos, A. J.\nScore: 75.6, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565210\nThe Neotropical region is the most diverse on the planet, largely due to its mosaic of tropical rainforests. Multiple tectonic and climatic processes have been hypothesized to play a role in generating this diversity; these include Andean orogeny, closure of the Panama Isthmus, the hypothesized GAARlandia land bridge, and putative historical connections among currently isolated forests. Micrathena spiny spiders (Araneidae) include [~]120 species distributed mostly in forests from Mexico to Argentina, including the Antilles. Here, we use it as a model to study the biogeographic history of Neotropical rainforests by estimating a complete, dated phylogeny using morphological data for all species and molecular data for a subsample of 79 species. This resulted in a phylogeny that is mostly robust and supports most of the previously recognized species groups, although with uncertainty in the phylogenetic position of some species, especially those lacking sequences. The genus began diversifying about 25 million years ago. We use an event-based approach and biogeographic stochastic mapping to estimate ancestral distributions and the timing and direction of dispersal events, and to identify areas where diversity was generated, while accounting for phylogenetic uncertainty. Andean cloud forests generated the majority of species through in-situ speciation, but the Amazon was the major source of species for adjacent areas through dispersal; on the other extreme, the Dry Diagonal received species from other areas but generated very little diversity. Species exchange between Central and South America was intense, with [~]23 dispersal events beginning at least 20 million years ago, indicating that Micrathena dispersed between these continents before closure of the Isthmus of Panama. We also inferred [~]4 over-water dispersal events from Central and North America to the Antilles, which happened in the last 20 million years, and thus much after the proposed age of the GAARlandia land bridge. We identified an important species exchange route between the Amazon and the Atlantic forests. Sampling all species of the genus was fundamental to some of the conclusions above, especially in identifying the Andes as the area that generated the majority of species. This study highlights the importance of a solid and complete taxonomic sampling in biogeographic studies.\nLong-term serial passaging of SARS-CoV-2 reveals signatures of convergent evolution\nAuthors: Foster, C. S. P.; Walker, G. J.; Jean, T.; Wong, M.; Brassil, L.; Isaacs, S.; Lu, Y.; Turville, S. G.; Kelleher, A.; Rawlinson, W. D.\nScore: 52.3, Published: 2023-11-06 DOI: 10.1101/2023.11.02.565396\nMeasures to control the COVID-19 pandemic such as antiviral therapy and vaccination have been challenged by ongoing virus evolution under antiviral and immune pressures. Understanding viral evolutionary dynamics is crucial for responding to SARS-CoV-2, and preparing for the next pandemic, by informing prediction of virus adaptation, public health strategies, and design of broadly effective therapies. Whole-genome sequencing (WGS) of SARS-CoV-2 during the pandemic enabled fine-grained studies of virus evolution in the human population. Serial passaging in vitro offers a controlled environment to investigate the emergence and persistence of genetic variants that may confer selective advantage. Nine virus lineages, including four \"variants of concern\" and three former \"variants under investigation\" as designated by the World Health Organisation, were chosen to investigate intra- and inter-lineage evolution through long-term serial passaging in Vero E6 cells. Viruses were sampled over at least 33 passages (range 33-100) and analysed using WGS to examine evolutionary dynamics and identify key mutations with implications for virus fitness, transmissibility, and immune evasion. All passages continued to replicate in culture, despite regular accumulation of mutations. There was evidence of convergent acquisition of mutations both across passage lines and compared with contemporaneous SARS CoV-2 clinical sequences from population studies. Some of these convergent mutations are hypothesised to be important in proliferation of SARS-CoV-2 lineages, such as by evading host immune responses (e.g. S:A67V, S:H655Y). Given these mutations arose in vitro, in the absence of a multicellular host immune response, this suggests virus genome mutation resulted from stochastic events, rather than immune-driven mutation. There was a regular gain and loss of low-frequency variants during serial passaging, but some became fixed in subsequent multiple passages, suggesting either a benefit of the mutation in vitro, or at least a lack of deleterious effect. Our findings reveal valuable insights into the evolution of SARS-CoV-2 by quantitatively investigating evolutionary dynamics of the virus over the greatest number of serial passages to date. Knowledge of these evolutionary trends will be useful for public health and the development of antiviral and vaccine measures to reduce the effects of SARS CoV-2 infection on the human population.\nGeographic patterns of living tetrapod diversity reveal the signature of global diversification dynamics\nAuthors: Tejero-Cicuendez, H.; Menendez, I.; Steell, E. M.; Navalon, G.; Blanco, F.; Smid, J.\nScore: 21.5, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565445\nBiodiversity is distributed unevenly among lineages and regions, and understanding the processes generating these global patterns is a central goal in evolutionary research, particularly in light of the current biodiversity crisis. Here, we integrate phylogenetic relatedness with species diversity patterns in four major clades of living tetrapods (amphibians, squamates, birds and mammals) to approach this challenge. We studied geographic patterns of richness-corrected phylogenetic diversity (residual PD), identifying regions where species are phylogenetically more closely or distantly related than expected by richness. We explored the effect of different factors in residual PD: recent speciation rates, temporal trends of lineage accumulation, and environmental variables. Specifically, we searched for evolutionary and ecological differences between regions of high and low residual PD. Our results reveal a nuanced relationship between recent speciation rates and residual PD, underscoring the role of recent speciation events in structuring current biogeographic patterns. Furthermore, we found differences between endothermic and ectothermic tetrapods in response to temperature and precipitation, highlighting the pivotal role of thermal physiology in shaping diversity dynamics. By illuminating the multifaceted factors underpinning global diversity patterns, our study represents a significant advancement towards more effective and holistic conservation approaches that are crucial to facing ongoing environmental challenges.\nHsp90 shapes adaptation by controlling the fitness consequences of regulatory variation\nAuthors: Jakobson, C.; Aguilar-Rodriguez, J.; Jarosz, D. F.\nScore: 16.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564848\nThe essential stress-responsive chaperone Hsp90 impacts development and adaptation from microbes to humans. Yet despite evidence of its role in evolution, pathogenesis, and oncogenic transformation, the molecular mechanisms by which Hsp90 alters the consequences of mutations remain vigorously debated. Here we exploit the power of nucleotide-resolution genetic mapping in Saccharomyces cerevisiae to uncover more than 1,000 natural variant-to-phenotype associations governed by this molecular chaperone. Strikingly, Hsp90 more frequently modified the phenotypic effects of cis-regulatory variation than variants that altered protein sequence. Moreover, these interactions made the largest contribution to Hsp90-dependent heredity. Nearly all interacting variants--both regulatory and protein-coding--fell within clients of Hsp90 or targets of its direct binding partners. Hsp90 activity affected mutations in evolutionarily young genes, segmental deletions, and heterozygotes, highlighting its influence on variation central to evolutionary novelty. Reconciling the diverse epistatic effects of this chaperone, synthetic transcriptional regulation and reconstructions of natural alleles by genome editing revealed a central role for Hsp90 in regulating the fundamental relationship between activity and phenotype. Our findings establish that non-coding variation is a core driver of Hsp90s influence on heredity, offering a mechanistic explanation for the chaperones strong effects on evolution and development across species.\nExperimental evolution of multicellularity via cuboidal cell packing in fission yeast\nAuthors: Pineau, R.; Kahn, P. C.; Lac, D. T.; Denning, M.; Wong, W.; Ratcliff, W.; Bozdag, G. O.\nScore: 15.3, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565576\nThe evolution of multicellularity represents a major transition in lifes history, enabling the rise of complex organisms. Multicellular groups can evolve through multiple developmental modes, but a common step is the formation of permanent cell-cell attachments after division. The characteristics of the multicellular morphology which emerges has profound consequences for the subsequent evolution of a nascent multicellular lineage, but little prior work has examined these dynamics directly. Here we examine a widespread yet understudied emergent multicellular morphology: cuboidal packing. Extinct and extant multicellular organisms across the tree of life have evolved to form groups in which spherical cells divide but remain attached, forming approximately cubic subunits. To experimentally investigate the evolution of cuboidal cell packing, we used settling selection to favor the evolution of simple multicellularity in unicellular, spherical Schizosaccharomyces pombe yeast. Multicellular clusters with cuboidal organization rapidly evolved, displacing the unicellular ancestor. These clusters displayed key hallmarks of an evolutionary transition in individuality: groups possess an emergent life cycle driven by physical fracture, group size is heritable, and they respond to group-level selection via multicellular adaptation. In 2/5 lineages, group formation was driven by mutations in the ACE2 gene, preventing daughter cell separation after division. Remarkably, ACE2 mutations also underlie the transition to multicellularity in Saccharomyces cerevisiae and C. galabrata, lineages last shared a common ancestor \u003e300 million years ago. Our results provide insight into the evolution of cuboidal cell packing, an understudied multicellular morphology, and highlights the deeply convergent potential for a transition to multicellular individuality within fungi.\nEcological diversification in an adaptive radiation of plants: the role of de novo mutation and introgression\nAuthors: Stone, B. W.; Wessinger, C. A.\nScore: 14.1, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565185\nAdaptive radiations are characterized by rapid ecological diversification and speciation events, leading to fuzzy species boundaries between ecologically differentiated species. Adaptive radiations are therefore key systems for understanding how species are formed and maintained, including the role of de novo mutations vs. pre-existing variation in ecological adaptation and the genome-wide consequences of hybridization events. For example, adaptive introgression, where beneficial alleles are transferred between lineages through hybridization, may fuel diversification in adaptive radiations and facilitate adaptation to new environments. In this study, we employed whole-genome resequencing data to investigate the evolutionary origin of hummingbird-pollinated flowers and to characterize genome-wide patterns of phylogenetic discordance and introgression in Penstemon subgenus Dasanthera, a small and diverse adaptive radiation of plants. We found that magenta hummingbird-adapted flowers have apparently evolved twice from ancestral blue-violet bee-pollinated flowers within this radiation. These apparently independent shifts in flower color are accompanied by a variety of inactivating mutations to a key anthocyanin pathway enzyme, suggesting de novo mutations underlie parallel evolution of this trait. Although patterns of introgression and phylogenetic discordance were heterogenous across the genome, a strong effect of gene density suggests that, in general, natural selection opposes introgression and maintains genetic differentiation in gene-rich genomic regions.\nA whole-body micro-CT scan library that captures the skeletal diversity of Lake Malawi cichlid fishes\nAuthors: Bucklow, C. V.; Genner, M. J.; Turner, G. F.; Maclaine, J.; Benson, R.; Verd, B.\nScore: 20.2, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565162\nHere we describe a dataset of freely available, readily processed, whole-body CT-scans of 56 species (116 specimens) of Lake Malawi cichlid fishes that captures a considerable majority of the morphological variation present in this remarkable adaptive radiation. We contextualise the scanned specimens within a discussion of their respective ecomorphological groupings and suggest possible macroevolutionary studies that could be conducted with these data. We also describe a methodology to efficiently CT-scan (on average) 23 specimens per hour, limiting scanning time and alleviating the financial cost whilst maintaining high resolution. We demonstrate the utility of this method by reconstructing 3D models of multiple bones from multiple specimens within the dataset. We hope this dataset will enable further morphological study of this fascinating system and permit wider-scale comparisons with other cichlid adaptive radiations.\nPrecise age estimation in clonal species using a somatic genetic clock\nAuthors: Yu, L.; Renton, J.; Burian, A.; Khachaturyan, M.; Kotta, J.; Stachowicz, J. C.; DuBois, K.; Baums, I. B.; Werner, B.; Reusch, T. B. H.\nScore: 10.7, Published: 2023-11-09 DOI: 10.1101/2023.11.07.566010\nAge and longevity are key parameters for demography and life-history evolution of organisms. In clonal species, a widespread life history among animals, plants, algae and fungi, the sexually produced offspring (the genet) grows indeterminately by producing iterative modules, or ramets. The age of large genets often remains elusive, while estimates based on their spatial extent as proxy for age are unreliable. Here, we present a method for age estimation using a molecular clock based on the accumulation of fixed somatic genetic variation (SoGV) that segregates among ramets of the same genet. Using a stochastic model of a generic clonal organism, we demonstrate that the accumulation of fixed SoGV via somatic genetic drift will approach linearity after a short lag phase, and is determined by the mitotic mutation rate, without direct dependence on asexual generation time. The lag phase decreased with lower stem cell population size (N), number of founder cells for the formation of new modules (N0), and the ratio of symmetric vs. asymmetric stem cell divisions. We apply the somatic genetic clock to the clonal plant model Zostera marina (eelgrass) and show that linearity is approached within a few years. Taking advantage of two long-term cultivation experiments for Z. marina (4 and 17 years respectively) as calibration points, we find genet ages up to 1,403 years in a global data set of 20 eelgrass populations. The somatic genetic clock is applicable to any multicellular clonal species where a small number of founder cells are recruited to form new ramets, opening novel research avenues to study longevity and hence, demography and population dynamics of clonal species.\nFrom museum drawer to tree: historical DNA phylogenomics clarifies the systematics of rare dung beetles (Coleoptera: Scarabaeinae) from museum collections\nAuthors: Lopes, F.; Gunter, N.; Gillett, C. P. D. T.; Montanaro, G.; Rossini, M.; Losacco, F.; Daniel, G. M.; Straube, N.; Tarasov, S.\nScore: 29.5, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564347\nAlthough several methods exist for extracting and sequencing historical DNA originating from drypreserved insect specimens deposited in natural history museums, no consensus exists as to what is the optimal approach. We demonstrate that a customized, low-cost archival DNA extraction protocol ([~] {euro}10 per sample), in combination with Ultraconserved Elements (UCEs), is an effective tool for insect phylogenomic studies. We successfully tested our approach by sequencing DNA from scarab dung beetles preserved in both wet and dry collections, including unique primary type and rare historical specimens from internationally important natural history museums in London, Paris and Helsinki. The focal specimens comprise enigmatic dung beetle genera (Nesosisyphus, Onychotechus and Helictopleurus) that varied in age and preservation. The oldest specimen, the holotype of the now possibly extinct Mauritian endemic Nesosisyphus rotundatus, was collected in 1944. We obtained high-quality DNA from all studied specimens to enable the generation of a UCE-based dataset that revealed an insightful and well-supported phylogenetic tree of dung beetles. The resulting phylogeny suggested the reclassification of Onychotechus (previously incertae sedis) within the tribe Coprini. Our approach demonstrates the feasibility and effectiveness of combining DNA data from historic and recent museum specimens to provide novel insights. The proposed archival DNA protocol is available at DOI 10.17504/protocols.io.81wgbybqyvpk/v1 HighlightsO_LIWe combined custom low-cost archival DNA extractions and Ultraconserved Element phylogenomics C_LIO_LIDNA from rare museum specimens of enigmatic dung beetles revealed their phylogenetic connections C_LIO_LIGenomic data was obtained from the holotype of a potentially extinct monoinsular endemic species C_LIO_LIGenomic data allowed a rare and enigmatic species of previously unknown affinity to be classified C_LIO_LIThe morphology of museum specimens remained intact following non-destructive DNA extraction C_LI O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=80 SRC=\"FIGDIR/small/564347v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (24K): org.highwire.dtl.DTLVardef@34aad0org.highwire.dtl.DTLVardef@1ba597dorg.highwire.dtl.DTLVardef@1493c2dorg.highwire.dtl.DTLVardef@10dd2eb_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2803",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.21.541643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.21.541643" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.21.541643">
        <p class="paperTitle">Fundamental equations linking methylation dynamics to maximum lifespan in mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.21.541643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.21.541643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.</p>
        <p class="info">Score: 84.6, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.21.541643' target='https://doi.org/10.1101/2023.05.21.541643'> 10.1101/2023.05.21.541643</a></p>
        <p class="abstract">We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset. Our research leads to three key findings: First, a reciprocal relationship exists between the average rate of methylation change (AROCM) in bivalent promoter regions and maximum lifespan: AROCM {propto} 1/MaxLifespan. Second, the correlation between age and average methylation bears no relation to maximum lifespan, Cor(Methly,Age) {perp} MaxLifespan. Third, the rate of methylation change in young animals is related to that in old animals: Young animals AROCM {propto} Old AROCM. These findings hinge on the chromatin context, as different results emerge when defining AROCM using different chromatin states. Our analytical framework is versatile and readily extendable to a broad range of other molecular assessments. Overall, our study demonstrates that epigenetic aging rates in specific chromatin states exhibit an inverse relationship with maximum lifespan in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565210">
        <p class="paperTitle">Complete phylogeny of Micrathena spiders suggests multiple dispersal events among Neotropical rainforests, islands, and landmasses, and indicates Andean orogeny promotes speciation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Magalhaes, I. L. F.; Martins, P. H.; Faleiro, B. T.; Vidigal, T. H. D. A.; Santos, F. R.; Carvalho, L. S.; Santos, A. J.</p>
        <p class="info">Score: 75.6, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565210' target='https://doi.org/10.1101/2023.11.01.565210'> 10.1101/2023.11.01.565210</a></p>
        <p class="abstract">The Neotropical region is the most diverse on the planet, largely due to its mosaic of tropical rainforests. Multiple tectonic and climatic processes have been hypothesized to play a role in generating this diversity; these include Andean orogeny, closure of the Panama Isthmus, the hypothesized GAARlandia land bridge, and putative historical connections among currently isolated forests. Micrathena spiny spiders (Araneidae) include [~]120 species distributed mostly in forests from Mexico to Argentina, including the Antilles. Here, we use it as a model to study the biogeographic history of Neotropical rainforests by estimating a complete, dated phylogeny using morphological data for all species and molecular data for a subsample of 79 species. This resulted in a phylogeny that is mostly robust and supports most of the previously recognized species groups, although with uncertainty in the phylogenetic position of some species, especially those lacking sequences. The genus began diversifying about 25 million years ago. We use an event-based approach and biogeographic stochastic mapping to estimate ancestral distributions and the timing and direction of dispersal events, and to identify areas where diversity was generated, while accounting for phylogenetic uncertainty. Andean cloud forests generated the majority of species through in-situ speciation, but the Amazon was the major source of species for adjacent areas through dispersal; on the other extreme, the Dry Diagonal received species from other areas but generated very little diversity. Species exchange between Central and South America was intense, with [~]23 dispersal events beginning at least 20 million years ago, indicating that Micrathena dispersed between these continents before closure of the Isthmus of Panama. We also inferred [~]4 over-water dispersal events from Central and North America to the Antilles, which happened in the last 20 million years, and thus much after the proposed age of the GAARlandia land bridge. We identified an important species exchange route between the Amazon and the Atlantic forests. Sampling all species of the genus was fundamental to some of the conclusions above, especially in identifying the Andes as the area that generated the majority of species. This study highlights the importance of a solid and complete taxonomic sampling in biogeographic studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565396">
        <p class="paperTitle">Long-term serial passaging of SARS-CoV-2 reveals signatures of convergent evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foster, C. S. P.; Walker, G. J.; Jean, T.; Wong, M.; Brassil, L.; Isaacs, S.; Lu, Y.; Turville, S. G.; Kelleher, A.; Rawlinson, W. D.</p>
        <p class="info">Score: 52.3, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565396' target='https://doi.org/10.1101/2023.11.02.565396'> 10.1101/2023.11.02.565396</a></p>
        <p class="abstract">Measures to control the COVID-19 pandemic such as antiviral therapy and vaccination have been challenged by ongoing virus evolution under antiviral and immune pressures. Understanding viral evolutionary dynamics is crucial for responding to SARS-CoV-2, and preparing for the next pandemic, by informing prediction of virus adaptation, public health strategies, and design of broadly effective therapies. Whole-genome sequencing (WGS) of SARS-CoV-2 during the pandemic enabled fine-grained studies of virus evolution in the human population. Serial passaging in vitro offers a controlled environment to investigate the emergence and persistence of genetic variants that may confer selective advantage.

Nine virus lineages, including four &#34;variants of concern&#34; and three former &#34;variants under investigation&#34; as designated by the World Health Organisation, were chosen to investigate intra- and inter-lineage evolution through long-term serial passaging in Vero E6 cells. Viruses were sampled over at least 33 passages (range 33-100) and analysed using WGS to examine evolutionary dynamics and identify key mutations with implications for virus fitness, transmissibility, and immune evasion. All passages continued to replicate in culture, despite regular accumulation of mutations. There was evidence of convergent acquisition of mutations both across passage lines and compared with contemporaneous SARS CoV-2 clinical sequences from population studies. Some of these convergent mutations are hypothesised to be important in proliferation of SARS-CoV-2 lineages, such as by evading host immune responses (e.g. S:A67V, S:H655Y). Given these mutations arose in vitro, in the absence of a multicellular host immune response, this suggests virus genome mutation resulted from stochastic events, rather than immune-driven mutation. There was a regular gain and loss of low-frequency variants during serial passaging, but some became fixed in subsequent multiple passages, suggesting either a benefit of the mutation in vitro, or at least a lack of deleterious effect. Our findings reveal valuable insights into the evolution of SARS-CoV-2 by quantitatively investigating evolutionary dynamics of the virus over the greatest number of serial passages to date. Knowledge of these evolutionary trends will be useful for public health and the development of antiviral and vaccine measures to reduce the effects of SARS CoV-2 infection on the human population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565445">
        <p class="paperTitle">Geographic patterns of living tetrapod diversity reveal the signature of global diversification dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tejero-Cicuendez, H.; Menendez, I.; Steell, E. M.; Navalon, G.; Blanco, F.; Smid, J.</p>
        <p class="info">Score: 21.5, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565445' target='https://doi.org/10.1101/2023.11.03.565445'> 10.1101/2023.11.03.565445</a></p>
        <p class="abstract">Biodiversity is distributed unevenly among lineages and regions, and understanding the processes generating these global patterns is a central goal in evolutionary research, particularly in light of the current biodiversity crisis. Here, we integrate phylogenetic relatedness with species diversity patterns in four major clades of living tetrapods (amphibians, squamates, birds and mammals) to approach this challenge. We studied geographic patterns of richness-corrected phylogenetic diversity (residual PD), identifying regions where species are phylogenetically more closely or distantly related than expected by richness. We explored the effect of different factors in residual PD: recent speciation rates, temporal trends of lineage accumulation, and environmental variables. Specifically, we searched for evolutionary and ecological differences between regions of high and low residual PD. Our results reveal a nuanced relationship between recent speciation rates and residual PD, underscoring the role of recent speciation events in structuring current biogeographic patterns. Furthermore, we found differences between endothermic and ectothermic tetrapods in response to temperature and precipitation, highlighting the pivotal role of thermal physiology in shaping diversity dynamics. By illuminating the multifaceted factors underpinning global diversity patterns, our study represents a significant advancement towards more effective and holistic conservation approaches that are crucial to facing ongoing environmental challenges.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564848">
        <p class="paperTitle">Hsp90 shapes adaptation by controlling the fitness consequences of regulatory variation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jakobson, C.; Aguilar-Rodriguez, J.; Jarosz, D. F.</p>
        <p class="info">Score: 16.6, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564848' target='https://doi.org/10.1101/2023.10.30.564848'> 10.1101/2023.10.30.564848</a></p>
        <p class="abstract">The essential stress-responsive chaperone Hsp90 impacts development and adaptation from microbes to humans. Yet despite evidence of its role in evolution, pathogenesis, and oncogenic transformation, the molecular mechanisms by which Hsp90 alters the consequences of mutations remain vigorously debated. Here we exploit the power of nucleotide-resolution genetic mapping in Saccharomyces cerevisiae to uncover more than 1,000 natural variant-to-phenotype associations governed by this molecular chaperone. Strikingly, Hsp90 more frequently modified the phenotypic effects of cis-regulatory variation than variants that altered protein sequence. Moreover, these interactions made the largest contribution to Hsp90-dependent heredity. Nearly all interacting variants--both regulatory and protein-coding--fell within clients of Hsp90 or targets of its direct binding partners. Hsp90 activity affected mutations in evolutionarily young genes, segmental deletions, and heterozygotes, highlighting its influence on variation central to evolutionary novelty. Reconciling the diverse epistatic effects of this chaperone, synthetic transcriptional regulation and reconstructions of natural alleles by genome editing revealed a central role for Hsp90 in regulating the fundamental relationship between activity and phenotype. Our findings establish that non-coding variation is a core driver of Hsp90s influence on heredity, offering a mechanistic explanation for the chaperones strong effects on evolution and development across species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565576">
        <p class="paperTitle">Experimental evolution of multicellularity via cuboidal cell packing in fission yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pineau, R.; Kahn, P. C.; Lac, D. T.; Denning, M.; Wong, W.; Ratcliff, W.; Bozdag, G. O.</p>
        <p class="info">Score: 15.3, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565576' target='https://doi.org/10.1101/2023.11.03.565576'> 10.1101/2023.11.03.565576</a></p>
        <p class="abstract">The evolution of multicellularity represents a major transition in lifes history, enabling the rise of complex organisms. Multicellular groups can evolve through multiple developmental modes, but a common step is the formation of permanent cell-cell attachments after division. The characteristics of the multicellular morphology which emerges has profound consequences for the subsequent evolution of a nascent multicellular lineage, but little prior work has examined these dynamics directly. Here we examine a widespread yet understudied emergent multicellular morphology: cuboidal packing. Extinct and extant multicellular organisms across the tree of life have evolved to form groups in which spherical cells divide but remain attached, forming approximately cubic subunits. To experimentally investigate the evolution of cuboidal cell packing, we used settling selection to favor the evolution of simple multicellularity in unicellular, spherical Schizosaccharomyces pombe yeast. Multicellular clusters with cuboidal organization rapidly evolved, displacing the unicellular ancestor. These clusters displayed key hallmarks of an evolutionary transition in individuality: groups possess an emergent life cycle driven by physical fracture, group size is heritable, and they respond to group-level selection via multicellular adaptation. In 2/5 lineages, group formation was driven by mutations in the ACE2 gene, preventing daughter cell separation after division. Remarkably, ACE2 mutations also underlie the transition to multicellularity in Saccharomyces cerevisiae and C. galabrata, lineages last shared a common ancestor &gt;300 million years ago. Our results provide insight into the evolution of cuboidal cell packing, an understudied multicellular morphology, and highlights the deeply convergent potential for a transition to multicellular individuality within fungi.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565185">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565185" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565185">
        <p class="paperTitle">Ecological diversification in an adaptive radiation of plants: the role of de novo mutation and introgression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565185" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565185" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stone, B. W.; Wessinger, C. A.</p>
        <p class="info">Score: 14.1, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565185' target='https://doi.org/10.1101/2023.11.01.565185'> 10.1101/2023.11.01.565185</a></p>
        <p class="abstract">Adaptive radiations are characterized by rapid ecological diversification and speciation events, leading to fuzzy species boundaries between ecologically differentiated species. Adaptive radiations are therefore key systems for understanding how species are formed and maintained, including the role of de novo mutations vs. pre-existing variation in ecological adaptation and the genome-wide consequences of hybridization events. For example, adaptive introgression, where beneficial alleles are transferred between lineages through hybridization, may fuel diversification in adaptive radiations and facilitate adaptation to new environments. In this study, we employed whole-genome resequencing data to investigate the evolutionary origin of hummingbird-pollinated flowers and to characterize genome-wide patterns of phylogenetic discordance and introgression in Penstemon subgenus Dasanthera, a small and diverse adaptive radiation of plants. We found that magenta hummingbird-adapted flowers have apparently evolved twice from ancestral blue-violet bee-pollinated flowers within this radiation. These apparently independent shifts in flower color are accompanied by a variety of inactivating mutations to a key anthocyanin pathway enzyme, suggesting de novo mutations underlie parallel evolution of this trait. Although patterns of introgression and phylogenetic discordance were heterogenous across the genome, a strong effect of gene density suggests that, in general, natural selection opposes introgression and maintains genetic differentiation in gene-rich genomic regions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565162">
        <p class="paperTitle">A whole-body micro-CT scan library that captures the skeletal diversity of Lake Malawi cichlid fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bucklow, C. V.; Genner, M. J.; Turner, G. F.; Maclaine, J.; Benson, R.; Verd, B.</p>
        <p class="info">Score: 20.2, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565162' target='https://doi.org/10.1101/2023.11.01.565162'> 10.1101/2023.11.01.565162</a></p>
        <p class="abstract">Here we describe a dataset of freely available, readily processed, whole-body CT-scans of 56 species (116 specimens) of Lake Malawi cichlid fishes that captures a considerable majority of the morphological variation present in this remarkable adaptive radiation. We contextualise the scanned specimens within a discussion of their respective ecomorphological groupings and suggest possible macroevolutionary studies that could be conducted with these data. We also describe a methodology to efficiently CT-scan (on average) 23 specimens per hour, limiting scanning time and alleviating the financial cost whilst maintaining high resolution. We demonstrate the utility of this method by reconstructing 3D models of multiple bones from multiple specimens within the dataset. We hope this dataset will enable further morphological study of this fascinating system and permit wider-scale comparisons with other cichlid adaptive radiations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566010" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566010">
        <p class="paperTitle">Precise age estimation in clonal species using a somatic genetic clock</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, L.; Renton, J.; Burian, A.; Khachaturyan, M.; Kotta, J.; Stachowicz, J. C.; DuBois, K.; Baums, I. B.; Werner, B.; Reusch, T. B. H.</p>
        <p class="info">Score: 10.7, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566010' target='https://doi.org/10.1101/2023.11.07.566010'> 10.1101/2023.11.07.566010</a></p>
        <p class="abstract">Age and longevity are key parameters for demography and life-history evolution of organisms. In clonal species, a widespread life history among animals, plants, algae and fungi, the sexually produced offspring (the genet) grows indeterminately by producing iterative modules, or ramets. The age of large genets often remains elusive, while estimates based on their spatial extent as proxy for age are unreliable. Here, we present a method for age estimation using a molecular clock based on the accumulation of fixed somatic genetic variation (SoGV) that segregates among ramets of the same genet. Using a stochastic model of a generic clonal organism, we demonstrate that the accumulation of fixed SoGV via somatic genetic drift will approach linearity after a short lag phase, and is determined by the mitotic mutation rate, without direct dependence on asexual generation time. The lag phase decreased with lower stem cell population size (N), number of founder cells for the formation of new modules (N0), and the ratio of symmetric vs. asymmetric stem cell divisions. We apply the somatic genetic clock to the clonal plant model Zostera marina (eelgrass) and show that linearity is approached within a few years. Taking advantage of two long-term cultivation experiments for Z. marina (4 and 17 years respectively) as calibration points, we find genet ages up to 1,403 years in a global data set of 20 eelgrass populations. The somatic genetic clock is applicable to any multicellular clonal species where a small number of founder cells are recruited to form new ramets, opening novel research avenues to study longevity and hence, demography and population dynamics of clonal species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564347">
        <p class="paperTitle">From museum drawer to tree: historical DNA phylogenomics clarifies the systematics of rare dung beetles (Coleoptera: Scarabaeinae) from museum collections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopes, F.; Gunter, N.; Gillett, C. P. D. T.; Montanaro, G.; Rossini, M.; Losacco, F.; Daniel, G. M.; Straube, N.; Tarasov, S.</p>
        <p class="info">Score: 29.5, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564347' target='https://doi.org/10.1101/2023.10.27.564347'> 10.1101/2023.10.27.564347</a></p>
        <p class="abstract">Although several methods exist for extracting and sequencing historical DNA originating from drypreserved insect specimens deposited in natural history museums, no consensus exists as to what is the optimal approach. We demonstrate that a customized, low-cost archival DNA extraction protocol ([~] {euro}10 per sample), in combination with Ultraconserved Elements (UCEs), is an effective tool for insect phylogenomic studies. We successfully tested our approach by sequencing DNA from scarab dung beetles preserved in both wet and dry collections, including unique primary type and rare historical specimens from internationally important natural history museums in London, Paris and Helsinki. The focal specimens comprise enigmatic dung beetle genera (Nesosisyphus, Onychotechus and Helictopleurus) that varied in age and preservation. The oldest specimen, the holotype of the now possibly extinct Mauritian endemic Nesosisyphus rotundatus, was collected in 1944. We obtained high-quality DNA from all studied specimens to enable the generation of a UCE-based dataset that revealed an insightful and well-supported phylogenetic tree of dung beetles. The resulting phylogeny suggested the reclassification of Onychotechus (previously incertae sedis) within the tribe Coprini. Our approach demonstrates the feasibility and effectiveness of combining DNA data from historic and recent museum specimens to provide novel insights. The proposed archival DNA protocol is available at DOI 10.17504/protocols.io.81wgbybqyvpk/v1

HighlightsO_LIWe combined custom low-cost archival DNA extractions and Ultraconserved Element phylogenomics
C_LIO_LIDNA from rare museum specimens of enigmatic dung beetles revealed their phylogenetic connections
C_LIO_LIGenomic data was obtained from the holotype of a potentially extinct monoinsular endemic species
C_LIO_LIGenomic data allowed a rare and enigmatic species of previously unknown affinity to be classified
C_LIO_LIThe morphology of museum specimens remained intact following non-destructive DNA extraction
C_LI



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=80 SRC=&#34;FIGDIR/small/564347v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (24K):
org.highwire.dtl.DTLVardef@34aad0org.highwire.dtl.DTLVardef@1ba597dorg.highwire.dtl.DTLVardef@1493c2dorg.highwire.dtl.DTLVardef@10dd2eb_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
