<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A non-adaptive explanation for macroevolutionary patterns in the evolution of complex multicellularity
Authors: Bingham, E. P.; Ratcliff, W. C.
Score: 92.1, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566713
&#34;Complex multicellularity&#34;, conventionally defined as large organisms with many specialized cell types, has evolved five times independently in eukaryotes, but never within prokaryotes. A number hypotheses have been proposed to explain this phenomenon, most of which posit that eukaryotes evolved key traits (e.g., dynamic cytoskeletons, alternative mechanisms of gene regulation, or subcellular compartments) which were a necessary prerequisite for the evolution of complex multicellularity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="A non-adaptive explanation for macroevolutionary patterns in the evolution of complex multicellularity
Authors: Bingham, E. P.; Ratcliff, W. C.
Score: 92.1, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566713
&#34;Complex multicellularity&#34;, conventionally defined as large organisms with many specialized cell types, has evolved five times independently in eukaryotes, but never within prokaryotes. A number hypotheses have been proposed to explain this phenomenon, most of which posit that eukaryotes evolved key traits (e.g., dynamic cytoskeletons, alternative mechanisms of gene regulation, or subcellular compartments) which were a necessary prerequisite for the evolution of complex multicellularity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="A non-adaptive explanation for macroevolutionary patterns in the evolution of complex multicellularity
Authors: Bingham, E. P.; Ratcliff, W. C.
Score: 92.1, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566713
&#34;Complex multicellularity&#34;, conventionally defined as large organisms with many specialized cell types, has evolved five times independently in eukaryotes, but never within prokaryotes. A number hypotheses have been proposed to explain this phenomenon, most of which posit that eukaryotes evolved key traits (e.g., dynamic cytoskeletons, alternative mechanisms of gene regulation, or subcellular compartments) which were a necessary prerequisite for the evolution of complex multicellularity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "A non-adaptive explanation for macroevolutionary patterns in the evolution of complex multicellularity\nAuthors: Bingham, E. P.; Ratcliff, W. C.\nScore: 92.1, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566713\n\u0026#34;Complex multicellularity\u0026#34;, conventionally defined as large organisms with many specialized cell types, has evolved five times independently in eukaryotes, but never within prokaryotes. A number hypotheses have been proposed to explain this phenomenon, most of which posit that eukaryotes evolved key traits (e.g., dynamic cytoskeletons, alternative mechanisms of gene regulation, or subcellular compartments) which were a necessary prerequisite for the evolution of complex multicellularity.",
  "keywords": [
    
  ],
  "articleBody": " A non-adaptive explanation for macroevolutionary patterns in the evolution of complex multicellularity\nAuthors: Bingham, E. P.; Ratcliff, W. C.\nScore: 92.1, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566713\n\"Complex multicellularity\", conventionally defined as large organisms with many specialized cell types, has evolved five times independently in eukaryotes, but never within prokaryotes. A number hypotheses have been proposed to explain this phenomenon, most of which posit that eukaryotes evolved key traits (e.g., dynamic cytoskeletons, alternative mechanisms of gene regulation, or subcellular compartments) which were a necessary prerequisite for the evolution of complex multicellularity. Here we propose an alternative, non-adaptive hypothesis for this broad macroevolutionary pattern. By binning cells into groups with finite genetic bottlenecks between generations, the evolution of multicellularity greatly reduces the effective population size (Ne) of cellular populations, increasing the role of genetic drift in evolutionary change. While both prokaryotes and eukaryotes experience this phenomenon, they have opposite responses to drift: mutational biases in eukaryotes tend to drive genomic expansion, providing additional raw genetic material for subsequent multicellular innovation, while prokaryotes generally face genomic erosion. These effects become more severe as organisms evolve larger size and more stringent genetic bottlenecks between generations-- both of which are hallmarks of complex multicellularity. Taken together, we hypothesize that it is these idiosyncratic lineagespecific mutational biases, rather than cell-biological innovations within eukaryotes, that underpins the long-term divergent evolution of complex multicellularity across the tree of life.\nTranslocations spur population growth but exacerbate inbreeding in an imperiled species\nAuthors: Linderoth, T.; Deaner, L.; Chen, N.; Bowman, R.; Boughton, R.; Fitzpatrick, S. W.\nScore: 19.7, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566550\nLand and natural resource usage that supports human society can pose a risk to the survival of other species, spurring biodiversity loss. In extreme cases, when development threatens the existence of individuals, wildlife managers may perform mitigation translocations, relocating individuals out of harms way. We investigated the efficacy of mitigation translocations as a conservation strategy in Federally Threatened Florida Scrub-Jays using a dataset that provided unprecedented resolution into both the demographic and genomic outcomes of translocations. Over the course of seven years, a total of fourteen groups (51 jays) from five subpopulations that had been declining from agriculture and lack-of-fire driven habitat degradation were translocated to a larger site of more contiguous restored habitat with only four family groups, to mitigate for loss of these subpopulations from mining activity. Habitat restoration and translocations established a core population that increased 10-fold in size after only 17 years from the first translocations. Pedigree analyses of this population revealed that a small subset of mostly translocated individuals fueled the demographic expansion, with a single breeding pair responsible for [~]24% of the ancestral genetic contributions since 2008. Genomic comparisons between translocated individuals and individuals from the core population before and after translocations revealed that the high reproductive skew led to increased inbreeding and loss of genetic diversity. This study stresses the importance of demographic and genetic monitoring following translocations, and that subsequent, genetic-rescue-oriented translocations may be necessary in mitigation scenarios to counter the genetic consequences of reproductive skew in fragmented populations. Significance StatementThere is ongoing debate surrounding the effectiveness of mitigation-driven translocations for conservation, however we show that translocations to mitigate the effects of mining on Federally Threatened Florida Scrub-Jays spurred population growth; a major boon to the viability of this species. We translocated individuals from at-risk subpopulations that were demographic sinks into recently restored habitat, which quickly established a rapidly growing core population. We demonstrate that demographic and genetic recovery do not necessarily go hand-in-hand, as reproduction was highly skewed towards a small subset of mostly translocated individuals, which increased inbreeding and eroded genetic diversity. This stresses the importance of demographic and genetic monitoring for identifying reproductive skew, allowing for adaptive management that addresses inbreeding and achieves broader conservation goals.\nChallenges in estimating species age from phylogenetic trees\nAuthors: Calderon del Cid, C.; Hauffe, T.; Carrillo, J. D.; Warnock, R. C. M.; Silvestro, D.\nScore: 15.5, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567731\nAim. Species age, the elapsed time since origination, can give an insight into how species longevity might influence eco-evolutionary dynamics and has been hypothesized to influence extinction risk. Traditionally, species ages have been measured in the fossil record. However, recently, numerous studies have attempted to estimate the ages of extant species from the branch lengths of time-calibrated phylogenies. This approach poses problems because phylogenetic trees contain direct information about species identity only at the tips and not along the branches. Here, we show that taxon sampling, extinction, and different assumptions about speciation modes can significantly alter the relationship between true species age and phylogenetic branch lengths, leading to high error rates. We find that these biases can lead to erroneous interpretations of eco-evolutionary patterns derived from the comparison between phylogenetic age and other traits, such as extinction risk. Innovation. For bifurcating speciation, which is the default assumption in most analyses, we propose a probabilistic approach to improve the estimation of species ages, based on the properties of a birth-death process. We show that our model can reduce the error by one order of magnitude under cases of high extinction. Main conclusion. Our results call for caution in interpreting the relationship between phylogenetic ages and eco-evolutionary traits, and show that, under some assumptions, it is possible to obtain better approximations of species age by combining information from branch lengths with the expectations of a birth-death process.\nThe explosive radiation of the Neotropical Tillandsia subgenus Tillandsia (Bromeliaceae) has been facilitated by pervasive hybridization\nAuthors: Yardeni, G.; Barfuss, M. H. J.; Till, W.; Thornton, M. R.; Groot Crego, C.; Lexer, C.; Leroy, T.; Paun, O.\nScore: 14.4, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567341\nThe recent rapid radiation of Tillandsia subgenus Tillandsia (Bromeliaceae) provides an attractive system to study the drivers and limits of species diversification. This species-rich Neotropical monocot clade includes predominantly epiphytic species displaying vast phenotypic diversity. Recent in-depth phylogenomic work revealed that the subgenus originated within the last 7 MY while expanding through one major event from South into Central America within the last 5 MY. However, disagreements between phylogenies and lack of resolution at shallow nodes suggested that hybridization occurred throughout the radiation, together with frequent incomplete lineage sorting and/or considerable gene family evolution. We used whole-genome resequencing data and a newly available reference genome to explore the evolutionary history of 34 representative ingroup species employing both a tree-based and a network approach. Our results indicate that lineage co-occurrence does not predict relatedness and confirm significant deviations from a tree-like structure, coupled with pervasive gene tree discordance. Focusing on hybridization, ABBA-BABA and related statistics were used to infer the rates and relative timing of introgression, while topology weighting uncovered high heterogeneity of the phylogenetic signal along the genome. High rates of hybridization within and among clades suggest that, in contrast to previous hypotheses, the expansion of subgenus Tillandsia into Central America proceeded in several dispersal events, punctuated by episodes of diversification and gene flow. Network analysis revealed reticulation as a prominent propeller during radiation and establishment in different ecological niches. This work contributes a plant example of prevalent hybridization during rapid species diversification, supporting the hypothesis that interspecific gene flow facilitates explosive diversification.\nPromoter capture drives the emergence of proto-genes in Escherichia coli\nAuthors: uz-Zaman, M. H.; D'Alton, S.; Barrick, J. E.; Ochman, H.\nScore: 14.3, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567300\nThe phenomenon of de novo gene birth--the emergence of genes from non-genic sequences--has received considerable attention due to the widespread occurrence of genes that are unique to particular species or genomes. Most instances of de novo gene birth have been recognized through comparative analyses of genome sequences in eukaryotes, despite the abundance of novel, lineage-specific genes in bacteria and the relative ease with which bacteria can be studied in an experimental context. Here, we explore the genetic record of the Escherichia coli Long-Term Evolution Experiment (LTEE) for changes indicative of \"proto-genic\" phases of new gene birth in which non-genic sequences evolve stable transcription and/or translation. Over the time-span of the LTEE, non-genic regions are frequently transcribed, translated and differentially expressed, thereby serving as raw material for new gene emergence. Most proto-genes result either from insertion element activity or chromosomal translocations that fused pre-existing regulatory sequences to regions that were not expressed in the LTEE ancestor. Additionally, we identified instances of proto-gene emergence in which a previously unexpressed sequence was transcribed after formation of an upstream promoter. Tracing the origin of the causative mutations, we discovered that most occurred early in the history of the LTEE, often within the first 20,000 generations, and became fixed soon after emergence. Our findings show that proto-genes emerge frequently within evolving populations, persist stably, and can serve as potential substrates for new gene formation.\nMass lysis of bacterial predators drives the enrichment of antibiotic resistance in soil microbial communities\nAuthors: Saha, S.; Kalathera, J.; Sumi, T. S.; Mane, V.; Zimmermann, S.; Waschina, S.; Pande, S.\nScore: 14.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567171\nWhile studies on anthropogenic activities and antibiotic resistance are numerous, the impact of microbial interactions on resistance in complex communities remains uncertain. Here we demonstrate a correlation between the presence of Myxococcus xanthus in natural soil communities and the abundance of antibiotic-resistant bacteria. Further, introducing M. xanthus isolates also enriches antibiotic resistance. This is due to the mass lysis of M. xanthus cells, which results in a toxic environment that fosters the proliferation of pre-existing resistant bacteria rather than de novo resistance evolution. Metagenomic analysis revealed that this enrichment is not limited to the tested antibiotics in culture-based methods, indicating its broader relevance. Crucially, these findings go beyond laboratory settings, showing M. xanthus introduction enriches resistant isolates in natural soil communities. Finally, we demonstrate that the mass lysis of M. xanthus cells during starvation-induced development (key aspect of the lifecycle of M. xanthus) also results in the enrichment of antibiotic resistance in soil communities. Together, we demonstrate how life-history traits in bacterial predators, like M. xanthus, significantly impact antibiotic resistomes in nature. This study also highlights the complex dynamics at play in the evolution and maintenance of antibiotic resistance, emphasizing the role of interspecies interactions in shaping antibiotic resistance profiles.\nInduction of apoptosis by double-stranded RNA was present in the last common ancestor of cnidarian and bilaterian animals\nAuthors: Kozlovski, I.; Jaimes-Becerra, A.; Sharoni, T.; Lewandowska, M.; Karmi, O.; Moran, Y.\nScore: 12.7, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567790\nApoptosis, a major form of programmed cell death, is an essential component of host defense against invading intracellular pathogens. Viruses encode inhibitors of apoptosis to evade host responses during infection, and to support their own replication and survival. Therefore, hosts and their viruses are entangled in a constant evolutionary arms race to control apoptosis. Until now, apoptosis in the context of the antiviral immune system has been almost exclusively studied in vertebrates. This limited phyletic sampling makes it impossible to determine whether a similar mechanism existed in the last common ancestor of animals. Here, we established assays to probe apoptosis in the sea anemone Nematostella vectensis, a model species of Cnidaria, a phylum that diverged approximately 600 million years ago from the rest of animals. We show that polyinosinic:polycytidylic acid (poly I:C), a synthetic long double-stranded RNA mimicking viral RNA and a primary ligand for the vertebrate RLR melanoma differentiation-associated protein 5 (MDA5), is sufficient to induce apoptosis in N. vectensis. Furthermore, at the transcriptomic level, apoptosis related genes are significantly enriched upon poly(I:C) exposure in N. vectensis as well as bilaterian invertebrates. Our phylogenetic analysis of caspase family genes in N. vectensis reveals conservation of all four caspase genes involved in apoptosis in mammals and revealed a cnidarian-specific caspase gene which was strongly upregulated. Altogether, our findings suggest that apoptosis in response to a viral challenge is a functionally conserved mechanism that can be traced back to the last common ancestor of Bilateria and Cnidaria.\nIdiosyncrasies unveiled: examining the pace, patterns and predictors of biotic diversification in peninsular India\nAuthors: Roy, P.; Joshi, J.\nScore: 12.6, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567174\nUnderstanding the tempo and mode of diversification and their consequence for tropical biodiversity remains a key challenge as different clades and regions exemplify distinct patterns and processes. We examined diversification rates and their drivers across 34 well-studied endemic lineages in peninsular India, one of the oldest regions of differentiation in the Oriental Realm, using birth-death models and their variations. We show that 18 lineages supported gradual species accumulation, suggesting that the historic stability of the landscape was an important driver. Additionally, paleotemperature, Miocene aridification, monsoon intensification and existing species diversity explained time-varying diversification patterns among the other 16 lineages. Net-diversification rates influenced differences in species diversity more than clade ages of peninsular Indian clades, where speciation rates mainly drove diversification, as extinction rates were low across taxa. Our results emphasise the importance of regional biogeographic, phylogenetic and geoclimatic history on the diversification dynamics among tropical landscapes.\nMatreex: compact and interactive visualisation for scalable studies of large gene families\nAuthors: Rossier, V.; Train, C.; Nevers, Y.; Robinson-Rechavi, M.; Dessimoz, C.\nScore: 26.0, Published: 2023-11-15 DOI: 10.1101/2023.02.18.529053\nStudying gene family evolution strongly benefits from insightful visualisations. However, the evergrowing number of sequenced genomes is leading to increasingly larger gene families, which challenges existing gene tree visualisations. Indeed, most of them present users with a dilemma: display complete but intractable gene trees, or collapse subtrees, thereby hiding their childrens information. Here, we introduce Matreex, a new dynamic tool to scale-up the visualisation of gene families. Matreexs key idea is to use \"phylogenetic\" profiles, which are dense representations of gene repertoires, to minimise the information loss when collapsing subtrees. We illustrate Matreex usefulness with three biological applications. First, we demonstrate on the MutS family the power of combining gene trees and phylogenetic profiles to delve into precise evolutionary analyses of large multi-copy gene families. Secondly, by displaying 22 intraflagellar transport gene families across 622 species cumulating 5500 representatives, we show how Matreex can be used to automate large-scale analyses of gene presence-absence. Notably, we report for the first time the complete loss of intraflagellar transport in the myxozoan Thelohanellus kitauei. Finally, using the textbook example of visual opsins, we show Matreexs potential to create easily interpretable figures for teaching and outreach. Matreex is available from the Python Package Index (pip install matreex) with the source code and documentation available at https://github.com/DessimozLab/matreex.\nInference of the demographic histories and selective effects of human gut commensal microbiota over the course of human history\nAuthors: Mah, J. C.; Lohmueller, K. E.; Garud, N.\nScore: 17.9, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566454\nDespite the importance of gut commensal microbiota to human health, there is little knowledge about their evolutionary histories, including their population demographic histories and their distributions of fitness effects (DFE) of new mutations. Here, we infer the demographic histories and DFEs of 27 of the most highly prevalent and abundant commensal gut microbial species in North Americans over timescales exceeding human generations using a collection of lineages inferred from a panel of healthy hosts. We find overall reductions in genetic variation among commensal gut microbes sampled from a Western population relative to an African rural population. Additionally, some species in North American microbiomes display contractions in population size and others expansions, potentially occurring at several key historical moments in human history. DFEs across species vary from highly to mildly deleterious, with accessory genes experiencing more drift compared to core genes. Within genera, DFEs tend to be more congruent, reflective of underlying phylogenetic relationships. Taken together, these findings suggest that human commensal gut microbes have distinct evolutionary histories, possibly reflecting the unique roles of individual members of the microbiome.\n",
  "wordCount" : "2589",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566713">
        <p class="paperTitle">A non-adaptive explanation for macroevolutionary patterns in the evolution of complex multicellularity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bingham, E. P.; Ratcliff, W. C.</p>
        <p class="info">Score: 92.1, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566713' target='https://doi.org/10.1101/2023.11.11.566713'> 10.1101/2023.11.11.566713</a></p>
        <p class="abstract">&#34;Complex multicellularity&#34;, conventionally defined as large organisms with many specialized cell types, has evolved five times independently in eukaryotes, but never within prokaryotes. A number hypotheses have been proposed to explain this phenomenon, most of which posit that eukaryotes evolved key traits (e.g., dynamic cytoskeletons, alternative mechanisms of gene regulation, or subcellular compartments) which were a necessary prerequisite for the evolution of complex multicellularity. Here we propose an alternative, non-adaptive hypothesis for this broad macroevolutionary pattern. By binning cells into groups with finite genetic bottlenecks between generations, the evolution of multicellularity greatly reduces the effective population size (Ne) of cellular populations, increasing the role of genetic drift in evolutionary change. While both prokaryotes and eukaryotes experience this phenomenon, they have opposite responses to drift: mutational biases in eukaryotes tend to drive genomic expansion, providing additional raw genetic material for subsequent multicellular innovation, while prokaryotes generally face genomic erosion. These effects become more severe as organisms evolve larger size and more stringent genetic bottlenecks between generations-- both of which are hallmarks of complex multicellularity. Taken together, we hypothesize that it is these idiosyncratic lineagespecific mutational biases, rather than cell-biological innovations within eukaryotes, that underpins the long-term divergent evolution of complex multicellularity across the tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566550">
        <p class="paperTitle">Translocations spur population growth but exacerbate inbreeding in an imperiled species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Linderoth, T.; Deaner, L.; Chen, N.; Bowman, R.; Boughton, R.; Fitzpatrick, S. W.</p>
        <p class="info">Score: 19.7, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566550' target='https://doi.org/10.1101/2023.11.11.566550'> 10.1101/2023.11.11.566550</a></p>
        <p class="abstract">Land and natural resource usage that supports human society can pose a risk to the survival of other species, spurring biodiversity loss. In extreme cases, when development threatens the existence of individuals, wildlife managers may perform mitigation translocations, relocating individuals out of harms way. We investigated the efficacy of mitigation translocations as a conservation strategy in Federally Threatened Florida Scrub-Jays using a dataset that provided unprecedented resolution into both the demographic and genomic outcomes of translocations. Over the course of seven years, a total of fourteen groups (51 jays) from five subpopulations that had been declining from agriculture and lack-of-fire driven habitat degradation were translocated to a larger site of more contiguous restored habitat with only four family groups, to mitigate for loss of these subpopulations from mining activity. Habitat restoration and translocations established a core population that increased 10-fold in size after only 17 years from the first translocations. Pedigree analyses of this population revealed that a small subset of mostly translocated individuals fueled the demographic expansion, with a single breeding pair responsible for [~]24% of the ancestral genetic contributions since 2008. Genomic comparisons between translocated individuals and individuals from the core population before and after translocations revealed that the high reproductive skew led to increased inbreeding and loss of genetic diversity. This study stresses the importance of demographic and genetic monitoring following translocations, and that subsequent, genetic-rescue-oriented translocations may be necessary in mitigation scenarios to counter the genetic consequences of reproductive skew in fragmented populations.

Significance StatementThere is ongoing debate surrounding the effectiveness of mitigation-driven translocations for conservation, however we show that translocations to mitigate the effects of mining on Federally Threatened Florida Scrub-Jays spurred population growth; a major boon to the viability of this species. We translocated individuals from at-risk subpopulations that were demographic sinks into recently restored habitat, which quickly established a rapidly growing core population. We demonstrate that demographic and genetic recovery do not necessarily go hand-in-hand, as reproduction was highly skewed towards a small subset of mostly translocated individuals, which increased inbreeding and eroded genetic diversity. This stresses the importance of demographic and genetic monitoring for identifying reproductive skew, allowing for adaptive management that addresses inbreeding and achieves broader conservation goals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567731">
        <p class="paperTitle">Challenges in estimating species age from phylogenetic trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calderon del Cid, C.; Hauffe, T.; Carrillo, J. D.; Warnock, R. C. M.; Silvestro, D.</p>
        <p class="info">Score: 15.5, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567731' target='https://doi.org/10.1101/2023.11.19.567731'> 10.1101/2023.11.19.567731</a></p>
        <p class="abstract">Aim. Species age, the elapsed time since origination, can give an insight into how species longevity might influence eco-evolutionary dynamics and has been hypothesized to influence extinction risk. Traditionally, species ages have been measured in the fossil record. However, recently, numerous studies have attempted to estimate the ages of extant species from the branch lengths of time-calibrated phylogenies. This approach poses problems because phylogenetic trees contain direct information about species identity only at the tips and not along the branches. Here, we show that taxon sampling, extinction, and different assumptions about speciation modes can significantly alter the relationship between true species age and phylogenetic branch lengths, leading to high error rates. We find that these biases can lead to erroneous interpretations of eco-evolutionary patterns derived from the comparison between phylogenetic age and other traits, such as extinction risk. Innovation. For bifurcating speciation, which is the default assumption in most analyses, we propose a probabilistic approach to improve the estimation of species ages, based on the properties of a birth-death process. We show that our model can reduce the error by one order of magnitude under cases of high extinction. Main conclusion. Our results call for caution in interpreting the relationship between phylogenetic ages and eco-evolutionary traits, and show that, under some assumptions, it is possible to obtain better approximations of species age by combining information from branch lengths with the expectations of a birth-death process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567341">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567341" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567341">
        <p class="paperTitle">The explosive radiation of the Neotropical Tillandsia subgenus Tillandsia (Bromeliaceae) has been facilitated by pervasive hybridization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567341" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567341" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yardeni, G.; Barfuss, M. H. J.; Till, W.; Thornton, M. R.; Groot Crego, C.; Lexer, C.; Leroy, T.; Paun, O.</p>
        <p class="info">Score: 14.4, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567341' target='https://doi.org/10.1101/2023.11.16.567341'> 10.1101/2023.11.16.567341</a></p>
        <p class="abstract">The recent rapid radiation of Tillandsia subgenus Tillandsia (Bromeliaceae) provides an attractive system to study the drivers and limits of species diversification. This species-rich Neotropical monocot clade includes predominantly epiphytic species displaying vast phenotypic diversity. Recent in-depth phylogenomic work revealed that the subgenus originated within the last 7 MY while expanding through one major event from South into Central America within the last 5 MY. However, disagreements between phylogenies and lack of resolution at shallow nodes suggested that hybridization occurred throughout the radiation, together with frequent incomplete lineage sorting and/or considerable gene family evolution. We used whole-genome resequencing data and a newly available reference genome to explore the evolutionary history of 34 representative ingroup species employing both a tree-based and a network approach. Our results indicate that lineage co-occurrence does not predict relatedness and confirm significant deviations from a tree-like structure, coupled with pervasive gene tree discordance. Focusing on hybridization, ABBA-BABA and related statistics were used to infer the rates and relative timing of introgression, while topology weighting uncovered high heterogeneity of the phylogenetic signal along the genome. High rates of hybridization within and among clades suggest that, in contrast to previous hypotheses, the expansion of subgenus Tillandsia into Central America proceeded in several dispersal events, punctuated by episodes of diversification and gene flow. Network analysis revealed reticulation as a prominent propeller during radiation and establishment in different ecological niches. This work contributes a plant example of prevalent hybridization during rapid species diversification, supporting the hypothesis that interspecific gene flow facilitates explosive diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567300">
        <p class="paperTitle">Promoter capture drives the emergence of proto-genes in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: uz-Zaman, M. H.; D&#39;Alton, S.; Barrick, J. E.; Ochman, H.</p>
        <p class="info">Score: 14.3, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567300' target='https://doi.org/10.1101/2023.11.15.567300'> 10.1101/2023.11.15.567300</a></p>
        <p class="abstract">The phenomenon of de novo gene birth--the emergence of genes from non-genic sequences--has received considerable attention due to the widespread occurrence of genes that are unique to particular species or genomes. Most instances of de novo gene birth have been recognized through comparative analyses of genome sequences in eukaryotes, despite the abundance of novel, lineage-specific genes in bacteria and the relative ease with which bacteria can be studied in an experimental context. Here, we explore the genetic record of the Escherichia coli Long-Term Evolution Experiment (LTEE) for changes indicative of &#34;proto-genic&#34; phases of new gene birth in which non-genic sequences evolve stable transcription and/or translation. Over the time-span of the LTEE, non-genic regions are frequently transcribed, translated and differentially expressed, thereby serving as raw material for new gene emergence. Most proto-genes result either from insertion element activity or chromosomal translocations that fused pre-existing regulatory sequences to regions that were not expressed in the LTEE ancestor. Additionally, we identified instances of proto-gene emergence in which a previously unexpressed sequence was transcribed after formation of an upstream promoter. Tracing the origin of the causative mutations, we discovered that most occurred early in the history of the LTEE, often within the first 20,000 generations, and became fixed soon after emergence. Our findings show that proto-genes emerge frequently within evolving populations, persist stably, and can serve as potential substrates for new gene formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567171">
        <p class="paperTitle">Mass lysis of bacterial predators drives the enrichment of antibiotic resistance in soil microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saha, S.; Kalathera, J.; Sumi, T. S.; Mane, V.; Zimmermann, S.; Waschina, S.; Pande, S.</p>
        <p class="info">Score: 14.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567171' target='https://doi.org/10.1101/2023.11.20.567171'> 10.1101/2023.11.20.567171</a></p>
        <p class="abstract">While studies on anthropogenic activities and antibiotic resistance are numerous, the impact of microbial interactions on resistance in complex communities remains uncertain. Here we demonstrate a correlation between the presence of Myxococcus xanthus in natural soil communities and the abundance of antibiotic-resistant bacteria. Further, introducing M. xanthus isolates also enriches antibiotic resistance. This is due to the mass lysis of M. xanthus cells, which results in a toxic environment that fosters the proliferation of pre-existing resistant bacteria rather than de novo resistance evolution. Metagenomic analysis revealed that this enrichment is not limited to the tested antibiotics in culture-based methods, indicating its broader relevance. Crucially, these findings go beyond laboratory settings, showing M. xanthus introduction enriches resistant isolates in natural soil communities. Finally, we demonstrate that the mass lysis of M. xanthus cells during starvation-induced development (key aspect of the lifecycle of M. xanthus) also results in the enrichment of antibiotic resistance in soil communities. Together, we demonstrate how life-history traits in bacterial predators, like M. xanthus, significantly impact antibiotic resistomes in nature. This study also highlights the complex dynamics at play in the evolution and maintenance of antibiotic resistance, emphasizing the role of interspecies interactions in shaping antibiotic resistance profiles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567790">
        <p class="paperTitle">Induction of apoptosis by double-stranded RNA was present in the last common ancestor of cnidarian and bilaterian animals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozlovski, I.; Jaimes-Becerra, A.; Sharoni, T.; Lewandowska, M.; Karmi, O.; Moran, Y.</p>
        <p class="info">Score: 12.7, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567790' target='https://doi.org/10.1101/2023.11.20.567790'> 10.1101/2023.11.20.567790</a></p>
        <p class="abstract">Apoptosis, a major form of programmed cell death, is an essential component of host defense against invading intracellular pathogens. Viruses encode inhibitors of apoptosis to evade host responses during infection, and to support their own replication and survival. Therefore, hosts and their viruses are entangled in a constant evolutionary arms race to control apoptosis. Until now, apoptosis in the context of the antiviral immune system has been almost exclusively studied in vertebrates. This limited phyletic sampling makes it impossible to determine whether a similar mechanism existed in the last common ancestor of animals. Here, we established assays to probe apoptosis in the sea anemone Nematostella vectensis, a model species of Cnidaria, a phylum that diverged approximately 600 million years ago from the rest of animals. We show that polyinosinic:polycytidylic acid (poly I:C), a synthetic long double-stranded RNA mimicking viral RNA and a primary ligand for the vertebrate RLR melanoma differentiation-associated protein 5 (MDA5), is sufficient to induce apoptosis in N. vectensis. Furthermore, at the transcriptomic level, apoptosis related genes are significantly enriched upon poly(I:C) exposure in N. vectensis as well as bilaterian invertebrates. Our phylogenetic analysis of caspase family genes in N. vectensis reveals conservation of all four caspase genes involved in apoptosis in mammals and revealed a cnidarian-specific caspase gene which was strongly upregulated. Altogether, our findings suggest that apoptosis in response to a viral challenge is a functionally conserved mechanism that can be traced back to the last common ancestor of Bilateria and Cnidaria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567174">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567174" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567174">
        <p class="paperTitle">Idiosyncrasies unveiled: examining the pace, patterns and predictors of biotic diversification in peninsular India</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567174" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567174" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roy, P.; Joshi, J.</p>
        <p class="info">Score: 12.6, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567174' target='https://doi.org/10.1101/2023.11.15.567174'> 10.1101/2023.11.15.567174</a></p>
        <p class="abstract">Understanding the tempo and mode of diversification and their consequence for tropical biodiversity remains a key challenge as different clades and regions exemplify distinct patterns and processes. We examined diversification rates and their drivers across 34 well-studied endemic lineages in peninsular India, one of the oldest regions of differentiation in the Oriental Realm, using birth-death models and their variations. We show that 18 lineages supported gradual species accumulation, suggesting that the historic stability of the landscape was an important driver. Additionally, paleotemperature, Miocene aridification, monsoon intensification and existing species diversity explained time-varying diversification patterns among the other 16 lineages. Net-diversification rates influenced differences in species diversity more than clade ages of peninsular Indian clades, where speciation rates mainly drove diversification, as extinction rates were low across taxa. Our results emphasise the importance of regional biogeographic, phylogenetic and geoclimatic history on the diversification dynamics among tropical landscapes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.18.529053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.18.529053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.18.529053">
        <p class="paperTitle">Matreex: compact and interactive visualisation for scalable studies of large gene families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.18.529053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.18.529053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossier, V.; Train, C.; Nevers, Y.; Robinson-Rechavi, M.; Dessimoz, C.</p>
        <p class="info">Score: 26.0, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.18.529053' target='https://doi.org/10.1101/2023.02.18.529053'> 10.1101/2023.02.18.529053</a></p>
        <p class="abstract">Studying gene family evolution strongly benefits from insightful visualisations. However, the evergrowing number of sequenced genomes is leading to increasingly larger gene families, which challenges existing gene tree visualisations. Indeed, most of them present users with a dilemma: display complete but intractable gene trees, or collapse subtrees, thereby hiding their childrens information. Here, we introduce Matreex, a new dynamic tool to scale-up the visualisation of gene families. Matreexs key idea is to use &#34;phylogenetic&#34; profiles, which are dense representations of gene repertoires, to minimise the information loss when collapsing subtrees. We illustrate Matreex usefulness with three biological applications. First, we demonstrate on the MutS family the power of combining gene trees and phylogenetic profiles to delve into precise evolutionary analyses of large multi-copy gene families. Secondly, by displaying 22 intraflagellar transport gene families across 622 species cumulating 5500 representatives, we show how Matreex can be used to automate large-scale analyses of gene presence-absence. Notably, we report for the first time the complete loss of intraflagellar transport in the myxozoan Thelohanellus kitauei. Finally, using the textbook example of visual opsins, we show Matreexs potential to create easily interpretable figures for teaching and outreach. Matreex is available from the Python Package Index (pip install matreex) with the source code and documentation available at https://github.com/DessimozLab/matreex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566454">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566454" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566454">
        <p class="paperTitle">Inference of the demographic histories and selective effects of human gut commensal microbiota over the course of human history</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566454" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566454" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mah, J. C.; Lohmueller, K. E.; Garud, N.</p>
        <p class="info">Score: 17.9, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566454' target='https://doi.org/10.1101/2023.11.09.566454'> 10.1101/2023.11.09.566454</a></p>
        <p class="abstract">Despite the importance of gut commensal microbiota to human health, there is little knowledge about their evolutionary histories, including their population demographic histories and their distributions of fitness effects (DFE) of new mutations. Here, we infer the demographic histories and DFEs of 27 of the most highly prevalent and abundant commensal gut microbial species in North Americans over timescales exceeding human generations using a collection of lineages inferred from a panel of healthy hosts. We find overall reductions in genetic variation among commensal gut microbes sampled from a Western population relative to an African rural population. Additionally, some species in North American microbiomes display contractions in population size and others expansions, potentially occurring at several key historical moments in human history. DFEs across species vary from highly to mildly deleterious, with accessory genes experiencing more drift compared to core genes. Within genera, DFEs tend to be more congruent, reflective of underlying phylogenetic relationships. Taken together, these findings suggest that human commensal gut microbes have distinct evolutionary histories, possibly reflecting the unique roles of individual members of the microbiome.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
