<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression
Authors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.
Score: 93.1, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548947
Molecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression
Authors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.
Score: 93.1, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548947
Molecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression
Authors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.
Score: 93.1, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548947
Molecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression\nAuthors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.\nScore: 93.1, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548947\nMolecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion.",
  "keywords": [
    
  ],
  "articleBody": " Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression\nAuthors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.\nScore: 93.1, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548947\nMolecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion. Here, we introduce Clockor2, a client-side web application for conducting RTT regression. Clockor2 uniquely allows users to quickly fit local and global molecular clocks, thus handling the increasing complexity of genomic datasets that sample beyond the assumption homogeneous host populations. Clockor2 is efficient, handling trees of up to the order of 104 tips, with significant speed increases compared to other RTT regression applications. Although clockor2 is written as a web application, all data processing happens on the client-side, meaning that data never leaves the users computer. Clockor2 is freely available at https://clockor2.github.io/.\nEnamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; RuÌˆther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.\nScore: 275.6, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547326\nThe evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.\nEvaluating evidence for co-geography in the Anopheles Plasmodium host-parasite system\nAuthors: Rehmann, C. T.; Ralph, P. L.; Kern, A. D.\nScore: 27.3, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549405\nThe often tight association between parasites and their hosts means that under certain scenarios, the evolutionary histories of the two species can become closely coupled. Using spatial genetic inference, we identify a potential signal of correlated dispersal in the Anopheles gambiae and Plasmodium falciparum host-parasite system in Sub-Saharan Africa, as seen through a between-species correlation of the differences between geographic sampling location and geographic location predicted from the genome. An explanation for this correlation is common patterns of dispersal and gene flow between these species, but we also consider possible statistical artifacts due to uneven spatial distribution of sampling locations. Using continuous-space population genetics simulations, we investigate the degree to which uneven distribution of sampling locations leads to bias in prediction of spatial location from genetic data and implement methods to counter this effect. We demonstrate that while algorithmic bias presents a problem in inference from spatiogenetic data, the correlation structure between A. gambiae and P. falciparum predictions cannot be attributed to spatial bias alone, and is thus likely a genetic signal of co-dispersal in a host-parasite system.\nUsing big sequencing data to identify chronic SARS-Coronavirus-2 infections\nAuthors: Harari, S.; Miller, D.; Fleishon, S.; Burstein, D.; Stern, A.\nScore: 24.9, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549184\nThe evolution of SARS-Coronavirus-2 (SARS-CoV-2) has been characterized by the periodic emergence of highly divergent variants, many of which may have arisen during chronic infections of immunocompromised individuals. Here, we harness a global phylogeny of [~]11.7 million SARS-CoV-2 genomes and search for clades composed of sequences with identical metadata (location, age, and sex) spanning more than 21 days. We postulate that such clades represent repeated sampling from the same chronically infected individual. A set of 271 such chronic-like clades was inferred, and displayed signatures of an elevated rate of adaptive evolution, in line with validated chronic infections. More than 70% of adaptive mutations present in currently circulating variants are found in BA.1 chronic-like clades that predate the circulating variants by months, demonstrating the predictive nature of such clades. We find that in chronic-like clades the probability of observing adaptive mutations is approximately 10-20 higher than that in global transmission chains. We next employ language models to find mutations most predictive of chronic infections and use them to infer hundreds of additional chronic-like clades in the absence of metadata and phylogenetic information. Our proposed approach presents an innovative method for mining extensive sequencing data and providing valuable insights into future evolutionary patterns.\nThe recency and geographical origins of the bat viruses ancestral to SARS-CoV and SARS-CoV-2\nAuthors: Pekar, J. E.; Lytras, S.; Ghafari, M.; Magee, A. F.; Parker, E.; Havens, J. L.; Katzourakis, A.; Vasylyeva, T. I.; Suchard, M. A.; Hughes, A. C.; Hughes, J.; Robertson, D. L.; Dellicour, S.; Worobey, M.; Wertheim, J. O.; Lemey, P.\nScore: 105.6, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548617\nThe emergence of SARS-CoV in 2002 and SARS-CoV-2 in 2019 has led to increased sampling of related sarbecoviruses circulating primarily in horseshoe bats. These viruses undergo frequent recombination and exhibit spatial structuring across Asia. Employing recombination-aware phylogenetic inference on bat sarbecoviruses, we find that the closest-inferred bat virus ancestors of SARS-CoV and SARS-CoV-2 existed just [~]1-3 years prior to their emergence in humans. Phylogeographic analyses examining the movement of related sarbecoviruses demonstrate that they traveled at similar rates to their horseshoe bat hosts and have been circulating for thousands of years in Asia. The closest-inferred bat virus ancestor of SARS-CoV likely circulated in western China, and that of SARS-CoV-2 likely circulated in a region comprising southwest China and northern Laos, both a substantial distance from where they emerged. This distance and recency indicate that the direct ancestors of SARS-CoV and SARS-CoV-2 could not have reached their respective sites of emergence via the bat reservoir alone. Our recombination-aware dating and phylogeographic analyses reveal a more accurate inference of evolutionary history than performing only whole-genome or single gene analyses. These results can guide future sampling efforts and demonstrate that viral genomic fragments extremely closely related to SARS-CoV and SARS-CoV-2 were circulating in horseshoe bats, confirming their importance as the reservoir species for SARS viruses.\nAdaptive introgression of a visual preference gene\nAuthors: Rossi, M.; Hausmann, A. H.; Alcami, P.; Moest, M. H.; Wright, D. S.; Kuo, C.-Y.; Lozano, D.; Maulana, A.; Melo-Florez, L.; Rueda-Munoz, G.; McMahon, S.; Linares, M.; McMillan, W. O.; Pardo-Diaz, C.; Salazar, C.; Merrill, R. M.\nScore: 31.4, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548653\nVisual preferences are important drivers of mate choice and sexual selection, but little is known of how they evolve at the genetic level. Here we take advantage of the diversity of bright warning patterns displayed by Heliconius butterflies, which are also used during mate choice. We show that two Heliconius species have evolved the same visual mating preferences for females with red patterns by exchanging genetic material through hybridization. Extensive behavioral experiments reveal that male preferences are associated with a genomic region of increased admixture between these two species. Variation in neural expression of regucalcin1, located within this introgressed region, correlates with visual preference across populations, and disruption of regucalcin1 with CRISPR/Cas9 impairs courtship towards conspecific females, proving a direct link between gene and behavior. Our results support a role for hybridization during behavioral evolution, and show how visually-guided behaviors contributing to adaptation and speciation are encoded within the genome.\nEvolution of compound eye morphology underlies differences in vision between closely related Drosophila species\nAuthors: Buffry, A. D.; Currea, J. P.; Franke, F. A.; Palavalli-Nettimi, R.; Bodey, A. P.; Rau, C.; Samadi, N.; GstoÌˆhl, S. J.; SchlepuÌˆtz, C. M.; McGregor, A. P.; Sumner-Rooney, L.; Theobald, J.; Kittelmann, M.\nScore: 15.9, Published: 2023-07-16 DOI: 10.1101/2023.07.16.549164\nInsects have evolved complex visual systems and display an astonishing range of adaptations for diverse ecological niches. Differences in eye size within and between Drosophila species provide the opportunity to study the impact of eye structure on vision. Here we further explored differences in D. mauritiana and its sibling species D. simulans and confirmed that D. mauritiana have rapidly evolved larger eyes as a result of more and wider ommatidia than D. simulans since their recent common ancestor. The functional impact of eye size, and specifically ommatidia size, is often only estimated based on the rigid surface morphology of the compound eye. Therefore, we used 3D synchrotron radiation tomography to measure optical parameters in 3D, predict optical capacity, and compare the modelled vision to in vivo optomotor responses. Our optical models predicted higher contrast sensitivity for D. mauritiana, which we verified by presenting sinusoidal gratings to tethered flies in a flight arena. Similarly, we confirmed the higher spatial acuity predicted for Drosophila simulans with smaller ommatidia and found evidence for higher temporal resolution.\nA positive association between gut microbiota diversity and vertebrate host performance in a field experiment\nAuthors: HaÌˆrer, A.; Thompson, K. A.; Schluter, D.; Rennison, D. J.\nScore: 15.6, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549938\nThe vertebrate gut microbiota is a critical determinant of organismal function, yet it remains unclear if and how gut microbial communities affect host fitness under natural conditions. Here, we investigate associations between growth rate (a fitness proxy) and gut microbiota diversity and composition in a field experiment with threespine stickleback fish (Gasterosteus aculeatus). We detected on average 63% more bacterial taxa in the guts of high-fitness fish compared to low-fitness fish (i.e., higher -diversity), suggesting that higher diversity promotes host growth. The microbial communities of high-fitness fish had higher similarity (i.e., lower {beta}-diversity) than low-fitness fish, supporting the Anna Karenina principle-that there are fewer ways to have a functional microbiota than a dysfunctional microbiota. Our findings provide a basis for functional tests of the fitness consequences of host-microbiota interactions.\nDistinct ossification trade-offs illuminate the shoulder girdle reconfiguration at the water-to-land transition.\nAuthors: Wei, J.; Wood, T. W. P.; Flaherty, K.; Enny, A.; Andrescavage, A.; Brazer, D.; Navon, D.; Stewart, T. A.; Cohen, H.; Shanabag, A.; Kuroda, S.; Braasch, I.; Nakamura, T.\nScore: 12.5, Published: 2023-07-17 DOI: 10.1101/2023.07.17.547998\nThe mechanisms of the pectoral girdle transformation at the origin of terrestrial locomotion in vertebrates remains an outstanding problem in evolutionary biology1. The loss of dermal bones and the enlargement of endochondral bones resulted in the disarticulation of the pectoral girdle from the skull and the formation of the neck during the fish-to-tetrapod transition2-5. Despite the functional implications of this skeletal shift in the emergence of terrestrial vertebrates, the underlying genetic-developmental alterations have remained enigmatic. Here, we discovered that in zebrafish pectoral girdle mesodermal cells expressing gli3, a transcription factor in the Hedgehog signaling pathway, contribute to both dermal and endochondral bones. We show that Gli3 regulates expression of activin A receptor type 1-like, a BMP type 1 receptor lost in tetrapod lineages, and thereby determines endochondral and dermal ossification. Intriguingly, Gli and Hedgehog compound knockout fish exhibited an unexpected combination of actinopterygian fish and stem-tetrapod pectoral girdle characteristics. These ontogenetic and anatomical data suggest that a trade-off between the two distinct ossification pathways is a deeply embedded developmental program in bony fishes, with potential for tuning of this trade-off to generate novel pectoral girdle forms akin to stem-tetrapods at the dawn of vertebrate terrestrialization.\nThe super-pangenome of Populus unveil genomic facets for adaptation and diversification in widespread forest trees\nAuthors: Shi, T.; Zhang, X.; Hou, Y.; Jiang, Y.; Jia, C.; Lai, Q.; Dan, X.; Feng, J.; Feng, J.; Ma, T.; Wu, J.; Liu, S.; Zhang, L.; Long, Z.; Zhang, Y.; Zhang, J.; Chen, L.; R. Street, N.; K. Ingvarsson, P.; Liu, J.; Yin, T.; Wang, J.\nScore: 8.4, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549473\nUnderstanding the underlying mechanisms between genome evolution and phenotypic and adaptive innovations is a key goal of evolutionary studies. Poplars are the world's most widely distributed and cultivated trees, with extensive phenotypic diversity and environmental adaptability. Here we report a genus-level super-pangenome of 19 Populus genomes. After integrating pan-genomes with transcriptomes, methylomes and chromatin accessibility mapping, we reveal that the evolutionary fate of pan-genes and duplicated genes are largely associated with local genomic landscapes of regulatory and epigenetic architectures. Further comparative genomic analyses enabled to identify 142,202 structural variations (SVs) across species, which overlap with substantial genes and play key roles in both phenotypic and adaptive divergence. We experimentally validated a ~180 bp presence/absence variant located in the promoter of the CUC2 gene, which contributed critically to leaf serration divergence between species. Together, this first super-pangenome resource in forest trees will not only accelerate molecular functional studies and genetic breeding of this globally important tree genus, but also lays a foundation for our understanding of tree biology.\n",
  "wordCount" : "2277",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548947">
        <p class="paperTitle">Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.</p>
        <p class="info">Score: 93.1, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548947' target='https://doi.org/10.1101/2023.07.13.548947'> 10.1101/2023.07.13.548947</a></p>
        <p class="abstract">Molecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion. Here, we introduce Clockor2, a client-side web application for conducting RTT regression. Clockor2 uniquely allows users to quickly fit local and global molecular clocks, thus handling the increasing complexity of genomic datasets that sample beyond the assumption homogeneous host populations. Clockor2 is efficient, handling trees of up to the order of 104 tips, with significant speed increases compared to other RTT regression applications. Although clockor2 is written as a web application, all data processing happens on the client-side, meaning that data never leaves the users computer. Clockor2 is freely available at https://clockor2.github.io/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547326">
        <p class="paperTitle">Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; RuÌˆther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.</p>
        <p class="info">Score: 275.6, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547326' target='https://doi.org/10.1101/2023.07.03.547326'> 10.1101/2023.07.03.547326</a></p>
        <p class="abstract">The evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549405">
        <p class="paperTitle">Evaluating evidence for co-geography in the Anopheles Plasmodium host-parasite system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rehmann, C. T.; Ralph, P. L.; Kern, A. D.</p>
        <p class="info">Score: 27.3, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549405' target='https://doi.org/10.1101/2023.07.17.549405'> 10.1101/2023.07.17.549405</a></p>
        <p class="abstract">The often tight association between parasites and their hosts means that under certain scenarios, the evolutionary histories of the two species can become closely coupled. Using spatial genetic inference, we identify a potential signal of correlated dispersal in the Anopheles gambiae and Plasmodium falciparum host-parasite system in Sub-Saharan Africa, as seen through a between-species correlation of the differences between geographic sampling location and geographic location predicted from the genome. An explanation for this correlation is common patterns of dispersal and gene flow between these species, but we also consider possible statistical artifacts due to uneven spatial distribution of sampling locations. Using continuous-space population genetics simulations, we investigate the degree to which uneven distribution of sampling locations leads to bias in prediction of spatial location from genetic data and implement methods to counter this effect. We demonstrate that while algorithmic bias presents a problem in inference from spatiogenetic data, the correlation structure between A. gambiae and P. falciparum predictions cannot be attributed to spatial bias alone, and is thus likely a genetic signal of co-dispersal in a host-parasite system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549184">
        <p class="paperTitle">Using big sequencing data to identify chronic SARS-Coronavirus-2 infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harari, S.; Miller, D.; Fleishon, S.; Burstein, D.; Stern, A.</p>
        <p class="info">Score: 24.9, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549184' target='https://doi.org/10.1101/2023.07.16.549184'> 10.1101/2023.07.16.549184</a></p>
        <p class="abstract">The evolution of SARS-Coronavirus-2 (SARS-CoV-2) has been characterized by the periodic emergence of highly divergent variants, many of which may have arisen during chronic infections of immunocompromised individuals. Here, we harness a global phylogeny of [~]11.7 million SARS-CoV-2 genomes and search for clades composed of sequences with identical metadata (location, age, and sex) spanning more than 21 days. We postulate that such clades represent repeated sampling from the same chronically infected individual. A set of 271 such chronic-like clades was inferred, and displayed signatures of an elevated rate of adaptive evolution, in line with validated chronic infections. More than 70% of adaptive mutations present in currently circulating variants are found in BA.1 chronic-like clades that predate the circulating variants by months, demonstrating the predictive nature of such clades. We find that in chronic-like clades the probability of observing adaptive mutations is approximately 10-20 higher than that in global transmission chains. We next employ language models to find mutations most predictive of chronic infections and use them to infer hundreds of additional chronic-like clades in the absence of metadata and phylogenetic information. Our proposed approach presents an innovative method for mining extensive sequencing data and providing valuable insights into future evolutionary patterns.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548617">
        <p class="paperTitle">The recency and geographical origins of the bat viruses ancestral to SARS-CoV and SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pekar, J. E.; Lytras, S.; Ghafari, M.; Magee, A. F.; Parker, E.; Havens, J. L.; Katzourakis, A.; Vasylyeva, T. I.; Suchard, M. A.; Hughes, A. C.; Hughes, J.; Robertson, D. L.; Dellicour, S.; Worobey, M.; Wertheim, J. O.; Lemey, P.</p>
        <p class="info">Score: 105.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548617' target='https://doi.org/10.1101/2023.07.12.548617'> 10.1101/2023.07.12.548617</a></p>
        <p class="abstract">The emergence of SARS-CoV in 2002 and SARS-CoV-2 in 2019 has led to increased sampling of related sarbecoviruses circulating primarily in horseshoe bats. These viruses undergo frequent recombination and exhibit spatial structuring across Asia. Employing recombination-aware phylogenetic inference on bat sarbecoviruses, we find that the closest-inferred bat virus ancestors of SARS-CoV and SARS-CoV-2 existed just [~]1-3 years prior to their emergence in humans. Phylogeographic analyses examining the movement of related sarbecoviruses demonstrate that they traveled at similar rates to their horseshoe bat hosts and have been circulating for thousands of years in Asia. The closest-inferred bat virus ancestor of SARS-CoV likely circulated in western China, and that of SARS-CoV-2 likely circulated in a region comprising southwest China and northern Laos, both a substantial distance from where they emerged. This distance and recency indicate that the direct ancestors of SARS-CoV and SARS-CoV-2 could not have reached their respective sites of emergence via the bat reservoir alone. Our recombination-aware dating and phylogeographic analyses reveal a more accurate inference of evolutionary history than performing only whole-genome or single gene analyses. These results can guide future sampling efforts and demonstrate that viral genomic fragments extremely closely related to SARS-CoV and SARS-CoV-2 were circulating in horseshoe bats, confirming their importance as the reservoir species for SARS viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548653">
        <p class="paperTitle">Adaptive introgression of a visual preference gene</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossi, M.; Hausmann, A. H.; Alcami, P.; Moest, M. H.; Wright, D. S.; Kuo, C.-Y.; Lozano, D.; Maulana, A.; Melo-Florez, L.; Rueda-Munoz, G.; McMahon, S.; Linares, M.; McMillan, W. O.; Pardo-Diaz, C.; Salazar, C.; Merrill, R. M.</p>
        <p class="info">Score: 31.4, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548653' target='https://doi.org/10.1101/2023.07.12.548653'> 10.1101/2023.07.12.548653</a></p>
        <p class="abstract">Visual preferences are important drivers of mate choice and sexual selection, but little is known of how they evolve at the genetic level. Here we take advantage of the diversity of bright warning patterns displayed by Heliconius butterflies, which are also used during mate choice. We show that two Heliconius species have evolved the same visual mating preferences for females with red patterns by exchanging genetic material through hybridization. Extensive behavioral experiments reveal that male preferences are associated with a genomic region of increased admixture between these two species. Variation in neural expression of regucalcin1, located within this introgressed region, correlates with visual preference across populations, and disruption of regucalcin1 with CRISPR/Cas9 impairs courtship towards conspecific females, proving a direct link between gene and behavior. Our results support a role for hybridization during behavioral evolution, and show how visually-guided behaviors contributing to adaptation and speciation are encoded within the genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549164">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549164" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549164">
        <p class="paperTitle">Evolution of compound eye morphology underlies differences in vision between closely related Drosophila species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549164" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549164" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buffry, A. D.; Currea, J. P.; Franke, F. A.; Palavalli-Nettimi, R.; Bodey, A. P.; Rau, C.; Samadi, N.; GstoÌˆhl, S. J.; SchlepuÌˆtz, C. M.; McGregor, A. P.; Sumner-Rooney, L.; Theobald, J.; Kittelmann, M.</p>
        <p class="info">Score: 15.9, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549164' target='https://doi.org/10.1101/2023.07.16.549164'> 10.1101/2023.07.16.549164</a></p>
        <p class="abstract">Insects have evolved complex visual systems and display an astonishing range of adaptations for diverse ecological niches. Differences in eye size within and between Drosophila species provide the opportunity to study the impact of eye structure on vision. Here we further explored differences in D. mauritiana and its sibling species D. simulans and confirmed that D. mauritiana have rapidly evolved larger eyes as a result of more and wider ommatidia than D. simulans since their recent common ancestor. The functional impact of eye size, and specifically ommatidia size, is often only estimated based on the rigid surface morphology of the compound eye. Therefore, we used 3D synchrotron radiation tomography to measure optical parameters in 3D, predict optical capacity, and compare the modelled vision to in vivo optomotor responses. Our optical models predicted higher contrast sensitivity for D. mauritiana, which we verified by presenting sinusoidal gratings to tethered flies in a flight arena. Similarly, we confirmed the higher spatial acuity predicted for Drosophila simulans with smaller ommatidia and found evidence for higher temporal resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549938">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549938" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549938">
        <p class="paperTitle">A positive association between gut microbiota diversity and vertebrate host performance in a field experiment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549938" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549938" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: HaÌˆrer, A.; Thompson, K. A.; Schluter, D.; Rennison, D. J.</p>
        <p class="info">Score: 15.6, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549938' target='https://doi.org/10.1101/2023.07.20.549938'> 10.1101/2023.07.20.549938</a></p>
        <p class="abstract">The vertebrate gut microbiota is a critical determinant of organismal function, yet it remains unclear if and how gut microbial communities affect host fitness under natural conditions. Here, we investigate associations between growth rate (a fitness proxy) and gut microbiota diversity and composition in a field experiment with threespine stickleback fish (Gasterosteus aculeatus). We detected on average 63% more bacterial taxa in the guts of high-fitness fish compared to low-fitness fish (i.e., higher -diversity), suggesting that higher diversity promotes host growth. The microbial communities of high-fitness fish had higher similarity (i.e., lower {beta}-diversity) than low-fitness fish, supporting the Anna Karenina principle-that there are fewer ways to have a functional microbiota than a dysfunctional microbiota. Our findings provide a basis for functional tests of the fitness consequences of host-microbiota interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.547998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.547998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.547998">
        <p class="paperTitle">Distinct ossification trade-offs illuminate the shoulder girdle reconfiguration at the water-to-land transition.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.547998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.547998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, J.; Wood, T. W. P.; Flaherty, K.; Enny, A.; Andrescavage, A.; Brazer, D.; Navon, D.; Stewart, T. A.; Cohen, H.; Shanabag, A.; Kuroda, S.; Braasch, I.; Nakamura, T.</p>
        <p class="info">Score: 12.5, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.547998' target='https://doi.org/10.1101/2023.07.17.547998'> 10.1101/2023.07.17.547998</a></p>
        <p class="abstract">The mechanisms of the pectoral girdle transformation at the origin of terrestrial locomotion in vertebrates remains an outstanding problem in evolutionary biology1. The loss of dermal bones and the enlargement of endochondral bones resulted in the disarticulation of the pectoral girdle from the skull and the formation of the neck during the fish-to-tetrapod transition2-5. Despite the functional implications of this skeletal shift in the emergence of terrestrial vertebrates, the underlying genetic-developmental alterations have remained enigmatic. Here, we discovered that in zebrafish pectoral girdle mesodermal cells expressing gli3, a transcription factor in the Hedgehog signaling pathway, contribute to both dermal and endochondral bones. We show that Gli3 regulates expression of activin A receptor type 1-like, a BMP type 1 receptor lost in tetrapod lineages, and thereby determines endochondral and dermal ossification. Intriguingly, Gli and Hedgehog compound knockout fish exhibited an unexpected combination of actinopterygian fish and stem-tetrapod pectoral girdle characteristics. These ontogenetic and anatomical data suggest that a trade-off between the two distinct ossification pathways is a deeply embedded developmental program in bony fishes, with potential for tuning of this trade-off to generate novel pectoral girdle forms akin to stem-tetrapods at the dawn of vertebrate terrestrialization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549473">
        <p class="paperTitle">The super-pangenome of Populus unveil genomic facets for adaptation and diversification in widespread forest trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, T.; Zhang, X.; Hou, Y.; Jiang, Y.; Jia, C.; Lai, Q.; Dan, X.; Feng, J.; Feng, J.; Ma, T.; Wu, J.; Liu, S.; Zhang, L.; Long, Z.; Zhang, Y.; Zhang, J.; Chen, L.; R. Street, N.; K. Ingvarsson, P.; Liu, J.; Yin, T.; Wang, J.</p>
        <p class="info">Score: 8.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549473' target='https://doi.org/10.1101/2023.07.18.549473'> 10.1101/2023.07.18.549473</a></p>
        <p class="abstract">Understanding the underlying mechanisms between genome evolution and phenotypic and adaptive innovations is a key goal of evolutionary studies. Poplars are the world&#39;s most widely distributed and cultivated trees, with extensive phenotypic diversity and environmental adaptability. Here we report a genus-level super-pangenome of 19 Populus genomes. After integrating pan-genomes with transcriptomes, methylomes and chromatin accessibility mapping, we reveal that the evolutionary fate of pan-genes and duplicated genes are largely associated with local genomic landscapes of regulatory and epigenetic architectures. Further comparative genomic analyses enabled to identify 142,202 structural variations (SVs) across species, which overlap with substantial genes and play key roles in both phenotypic and adaptive divergence. We experimentally validated a ~180 bp presence/absence variant located in the promoter of the CUC2 gene, which contributed critically to leaf serration divergence between species. Together, this first super-pangenome resource in forest trees will not only accelerate molecular functional studies and genetic breeding of this globally important tree genus, but also lays a foundation for our understanding of tree biology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
