<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila
Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.
Score: 18.3, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861
How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila
Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.
Score: 18.3, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861
How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila
Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.
Score: 18.3, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861
How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila\nAuthors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.\nScore: 18.3, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861\nHow evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D.",
  "keywords": [
    
  ],
  "articleBody": " Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila\nAuthors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.\nScore: 18.3, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861\nHow evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D. yakuba, who has undergone a lineage-specific loss of sine song, one of the two major types of male courtship song in Drosophila. Neuroanatomical comparison of song patterning neurons called TN1 across the phylogeny demonstrates a link between the loss of sine song and a reduction both in the number of TN1 neurons and the neurites serving the sine circuit connectivity. Optogenetic activation confirms that TN1 neurons in D. yakuba have lost the ability to drive sine song, while maintaining the ability to drive the singing wing posture. Single-cell transcriptomic comparison shows that D. yakuba specifically lacks a cell type corresponding to TN1A neurons, the TN1 subtype that is essential for sine song. Genetic and developmental manipulation reveals a functional divergence of the sex determination gene doublesex in D. yakuba to reduce TN1 number by promoting apoptosis. Our work illustrates the contribution of motor patterning circuits and cell type changes in behavioral evolution, and uncovers the evolutionary lability of sex determination genes to reconfigure the cellular makeup of neural circuits.\nDeep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait\nAuthors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.\nScore: 24.6, Published: 2024-01-23 DOI: 10.1101/2023.09.29.560175\nThe extraordinary variation in male guppy coloration has proven a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a Genome Wide Association Study (GWAS) of color. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture spanning the entire genome. Unusually, our GWAS results suggest that copy number variation, particularly copies shared between the Y chromosome and the remainder of the genome, is responsible for much of the variation in color in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.\nEarly acting inbreeding depression can evolve as an inbreeding avoidance mechanism\nAuthors: Brandvain, Y. J.; Thomson, L.; Pyhajarvi, T.\nScore: 10.3, Published: 2024-01-25 DOI: 10.1101/2023.03.20.533524\nDespite the potential for mechanical, developmental and/or chemical mechanisms to prevent self-fertilization, incidental self-fertilization is inevitable in many predominantly outcrossing species. In such cases, inbreeding can compromise individual fitness. Unquestionably, much of this inbreeding depression is maladaptive. However, we show that when reproductive compensation allows for the replacement of inviable embryos lost early in development, selection can favor deleterious recessive variants that induce \"self-sacrificial\" death of inbred embryos. Our theoretical results provide numerous testable predictions which could challenge the assumption that inbreeding depression is always maladaptive. Our work is applicable any species that cannot fully avoid inbreeding, exhibits substantial inbreeding depression, and has the potential to compensate embryos lost early in development. In addition to its general applicability our theory suggests that self-sacrificial variants might be responsible for the remarkably low realized selfing rates of gymnosperms with high primary selfing rates, as gymnosperms exhibit strong inbreeding depression, have effective reproductive compensation mechanisms, and cannot evolve chemical self-incompatibility.\nStronger evidence for relaxed selection than adaptive evolution in high-elevation animal mtDNA\nAuthors: Iverson, E. N. K.; Criswell, A.; Havird, J. C.\nScore: 6.4, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576402\nMitochondrial (mt) genes are the subject of many adaptive hypotheses due to the key role of mitochondria in energy production and metabolism. One widespread adaptive hypothesis is that selection imposed by life at high elevation leads to the rapid fixation of beneficial alleles in mtDNA, reflected in the increased rates of mtDNA evolution documented in many high-elevation species. However, the assumption that fast mtDNA evolution is caused by positive, rather than relaxed purifying selection has rarely been tested. Here, we calculated the dN/dS ratio, a metric of nonsynonymous substitution bias, and explicitly tested for relaxed selection in the mtDNA of over 700 species of terrestrial vertebrates, freshwater fishes, and arthropods, with information on elevation and latitudinal range limits, range sizes, and body sizes. We confirmed that mitochondrial genomes of high-elevation taxa have slightly higher dN/dS ratios compared to low-elevation relatives. High-elevation species tend to have smaller ranges, which predict higher dN/dS ratios and more relaxed selection across species and clades, while absolute elevation and latitude do not predict higher dN/dS. We also find a positive relationship between body mass and dN/dS, supporting a role for small effective population size leading to relaxed selection. We conclude that higher mt dN/dS among high-elevation species is more likely to reflect relaxed selection due to smaller ranges and reduced effective population size than adaptation to the environment. Our results highlight the importance of rigorously testing adaptive stories against non-adaptive alternative hypotheses, especially in mt genomes.\nMultiple distinct evolutionary mechanisms govern the dynamics of selfish mitochondrial genomes\nAuthors: Gitschlag, B. L.; Pereira, C. V.; Held, J. P.; McCandlish, D. M.; Patel, M. R.\nScore: 11.2, Published: 2024-01-26 DOI: 10.1101/2022.04.08.487647\nCells possess multiple mitochondrial DNA (mtDNA) copies, which undergo semi-autonomous replication and stochastic inheritance. This enables mutant mtDNA variants to arise and selfishly compete with cooperative (wildtype) mtDNA. Selfish mitochondrial genomes are subject to selection at different levels: they compete against wildtype mtDNA directly within hosts and indirectly through organismal selection. However, determining the relative contributions of selection at different levels has proven challenging. We overcome this challenge by combining mathematical modeling with experiments designed to isolate the levels of selection. Applying this approach to many selfish mitochondrial genotypes in Caenorhabditis elegans revealed an unexpected diversity of evolutionary mechanisms. Some mutant genomes persist at high frequency for many generations, despite a host fitness cost, by aggressively outcompeting cooperative genomes within hosts. Conversely, some mutant genomes persist by evading organismal selection. Strikingly, the mutant genomes vary dramatically in their susceptibility to neutral drift. Although different mechanisms can cause high frequency of selfish mtDNA, we show how they give rise to characteristically different distributions of mutant frequency among individuals. Given that heteroplasmic frequency represents a key determinant of phenotypic severity, this work outlines an evolutionary theoretic framework for predicting the distribution of phenotypic consequences among individuals carrying a selfish mitochondrial genome.\nComplex interactions in the life cycle of a simple parasite shape the evolution of virulence\nAuthors: Silva, L. M.; Koella, J. C.\nScore: 5.9, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577571\nEvolutionary expectations about the virulence of parasites (i.e., the parasite-induced mortality rate of the host) generally describe parasites which grow and are transmitted at the same time. Many parasites, however, go through an incubation period, that is they must grow before they can be transmitted. Moreover, the parasite incubation period and virulence are linked through its growth rate. We here suggest that such interactions make it difficult to predict how virulence will evolve. We illustrate our ideas with a parasite with a simple cycle, the microsporidian Vavraia culicis , which infects the mosquito Anopheles gambiae . We selected the parasite for six generations to kill their host and be transmitted early or late. Selecting for late transmission led to a stronger increasing its exploitation (i.e., cost due to growth), but not pathogenicity per parasite (i.e., cost independent of growth), than selection for early transmission. This was associated with higher virulence, and a shorter parasite life cycle, with the latter having a larger and more rapid production of the infective spores. In response, the host shortened its life cycle and shifted to early fecundity when infected with late-selected spores. Altogether, the results of this study reinforce the need for a reviewed and unified virulence theory and raise important questions for epidemiological and vector control strategies.\nSex-biased migration and demographic history of the big European firefly Lampyris noctiluca\nAuthors: Catalan, A.; Gygax, D.; Candolin, U.; Tusso, S.; Duchen, P.; Hoehna, S.\nScore: 5.5, Published: 2024-01-29 DOI: 10.1101/2024.01.24.577017\nDifferential dispersion between the sexes can impact population structure and connectivity between populations, which in turn, can have an effect on the evolvability capability of a species. Here we explored the demographic history of the big European firefly, Lampyris noctiluca, which exhibits female neoteny. L. noctiluca distribution extends throughout Europe, but nothing is known on how its colonization processes. To investigate this, we produced the first Lampyris genome (653Mb), including an IsoSeq annotation and the identification of the X chromosome. We collected 115 individuals from six populations of L. noctiluca (Finland to Italy) and generated whole genome re-sequencing for each individual. We inferred several population expansions and bottlenecks throughout the Pleistocene that correlate with glaciation events. Surprisingly, we uncovered strong population structure and low gene-flow. We reject a stepwise, south to north, colonization history scenario and instead uncovered a complex demographic history with a putative eastern European origin. Analyzing the evolutionary history of the mitochondrial genome as well as X-linked and autosomal loci, we found evidence of a maternal colonialization of Germany, putatively from a western European population, followed by male-only migration from south of the Alps (Italy). Overall, investigating the demographic history and colonization patterns of a species should form part of an integrative approach of biodiversity research. Our results provide evidence of sex-biased migration which is important to consider for demographic, biogeographic and species delimitation studies.\nMaternal manipulation of offspring size can trigger the evolution of eusociality in promiscuous species\nAuthors: Rees-Baylis, E.; Pen, I.; Kreider, J. J.\nScore: 4.5, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576864\nEusocial organisms typically live in colonies with one reproductive queen supported by thousands of sterile workers. It is widely believed that monogamous mating is a precondition for the evolution of eusociality. Here, we present a theoretical model that simulates a realistic scenario for the evolution of eusociality. In the model, mothers can evolve control over resource allocation to offspring, affecting offspring body size. The offspring can evolve body-size-dependent dispersal, by which they disperse to breed or stay at the nest as helpers. We demonstrate that eusociality evolves, even if mothers are not strictly monogamous, if mothers can constrain their offsprings reproduction by manipulation. We also observe the evolution of social polymorphism with small individuals that help and larger individuals that disperse to breed. Our model unifies the traditional kin selection and maternal manipulation explanations for the evolution of eusociality and demonstrates that - contrary to current consensus belief - eusociality can evolve despite highly promiscuous mating.\nAdaptive and pleiotropic effects of evolution in synonymous sugar environments.\nAuthors: Ahlawat, N.; Venkataraman, P.; Mahilkar, A.; Brajesh, R. G.; Saini, S.\nScore: 3.5, Published: 2024-01-30 DOI: 10.1101/2024.01.28.577607\nAdaptation to an environment is enabled by the accumulation of beneficial mutations. When adapted populations are shifted to other environments, the byproduct or pleiotropic fitness effects of these mutations can be wide-ranged. Since there exists no molecular framework to quantify relatedness of environments, predicting pleiotropic effects based on adaptation has been challenging. In this work, we ask if evolution in highly similar environments elicits correlated adaptive and pleiotropic responses. We evolve replicate populations of Escherichia coli in non-stressful environments that contain either a mixture of glucose and galactose, lactose, or melibiose as the source of carbon. We term these similar sugars as synonymous, since lactose and melibiose are disaccharides made up of glucose and galactose. Therefore, the evolution environments differed only in the way carbon was presented to the bacterial population. After 300 generations of evolution, we see that the adaptive responses of these populations are not predictable. We investigate the pleiotropic effects of adaptation in a range of non-synonymous environments, and show that despite uncorrelated adaptive changes, the nature of pleiotropic effects is largely predictable based on the fitness of the ancestor in the non-home environments. Overall, our results highlight how subtle changes in the environment can alter adaptation, but despite sequence-level variations, pleiotropy is qualitatively predictable.\nOn the annotation of Paleo- and Mesoproterozoic microfossils as fungi\nAuthors: Kanaparthi, D.; Lampe, M.; Krings, M.; Schue, B. S.; Klingl, A.; Schwille, P.; Lueders, T.\nScore: 2.8, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577314\nHow old is the crown group Fungi? Inferences from phylogenetic and fossil-based studies provided far-apart age estimates ranging between 0.75 to 2.7 billion years old. One important criterion for interpreting Paleo- and Mesoproterozoic microfossils as Fungi is their uncanny morphological resemblance with extant fungi. Here, we demonstrate that bacteria exposed to environmental conditions similar to the paleoenvironmental settings where these presumed fungi lived can spontaneously transform into their protoplasts. These protoplasts exhibit morphologies corresponding to those of presumed fungal microfossils. These observations, together with microfossil chemical composition, pose a serious challenge to interpreting Paleo- and Mesoproterozoic microfossils as fungi. Based on these results, we reiterate that morphology is not a reliable indicator of the phylogeny of microfossils older than 2.5Ga.\n",
  "wordCount" : "2214",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576861" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576861">
        <p class="paperTitle">Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.</p>
        <p class="info">Score: 18.3, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576861' target='https://doi.org/10.1101/2024.01.23.576861'> 10.1101/2024.01.23.576861</a></p>
        <p class="abstract">How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D. yakuba, who has undergone a lineage-specific loss of sine song, one of the two major types of male courtship song in Drosophila. Neuroanatomical comparison of song patterning neurons called TN1 across the phylogeny demonstrates a link between the loss of sine song and a reduction both in the number of TN1 neurons and the neurites serving the sine circuit connectivity. Optogenetic activation confirms that TN1 neurons in D. yakuba have lost the ability to drive sine song, while maintaining the ability to drive the singing wing posture. Single-cell transcriptomic comparison shows that D. yakuba specifically lacks a cell type corresponding to TN1A neurons, the TN1 subtype that is essential for sine song. Genetic and developmental manipulation reveals a functional divergence of the sex determination gene doublesex in D. yakuba to reduce TN1 number by promoting apoptosis. Our work illustrates the contribution of motor patterning circuits and cell type changes in behavioral evolution, and uncovers the evolutionary lability of sex determination genes to reconfigure the cellular makeup of neural circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560175">
        <p class="paperTitle">Deep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.</p>
        <p class="info">Score: 24.6, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560175' target='https://doi.org/10.1101/2023.09.29.560175'> 10.1101/2023.09.29.560175</a></p>
        <p class="abstract">The extraordinary variation in male guppy coloration has proven a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a Genome Wide Association Study (GWAS) of color. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture spanning the entire genome. Unusually, our GWAS results suggest that copy number variation, particularly copies shared between the Y chromosome and the remainder of the genome, is responsible for much of the variation in color in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.20.533524">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.20.533524" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.20.533524">
        <p class="paperTitle">Early acting inbreeding depression can evolve as an inbreeding avoidance mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.20.533524" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.20.533524" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brandvain, Y. J.; Thomson, L.; Pyhajarvi, T.</p>
        <p class="info">Score: 10.3, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.20.533524' target='https://doi.org/10.1101/2023.03.20.533524'> 10.1101/2023.03.20.533524</a></p>
        <p class="abstract">Despite the potential for mechanical, developmental and/or chemical mechanisms to prevent self-fertilization, incidental self-fertilization is inevitable in many predominantly outcrossing species. In such cases, inbreeding can compromise individual fitness. Unquestionably, much of this inbreeding depression is maladaptive. However, we show that when reproductive compensation allows for the replacement of inviable embryos lost early in development, selection can favor deleterious recessive variants that induce &#34;self-sacrificial&#34; death of inbred embryos. Our theoretical results provide numerous testable predictions which could challenge the assumption that inbreeding depression is always maladaptive. Our work is applicable any species that cannot fully avoid inbreeding, exhibits substantial inbreeding depression, and has the potential to compensate embryos lost early in development. In addition to its general applicability our theory suggests that self-sacrificial variants might be responsible for the remarkably low realized selfing rates of gymnosperms with high primary selfing rates, as gymnosperms exhibit strong inbreeding depression, have effective reproductive compensation mechanisms, and cannot evolve chemical self-incompatibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576402">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576402" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576402">
        <p class="paperTitle">Stronger evidence for relaxed selection than adaptive evolution in high-elevation animal mtDNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576402" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576402" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iverson, E. N. K.; Criswell, A.; Havird, J. C.</p>
        <p class="info">Score: 6.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576402' target='https://doi.org/10.1101/2024.01.20.576402'> 10.1101/2024.01.20.576402</a></p>
        <p class="abstract">Mitochondrial (mt) genes are the subject of many adaptive hypotheses due to the key role of mitochondria in energy production and metabolism. One widespread adaptive hypothesis is that selection imposed by life at high elevation leads to the rapid fixation of beneficial alleles in mtDNA, reflected in the increased rates of mtDNA evolution documented in many high-elevation species. However, the assumption that fast mtDNA evolution is caused by positive, rather than relaxed purifying selection has rarely been tested. Here, we calculated the dN/dS ratio, a metric of nonsynonymous substitution bias, and explicitly tested for relaxed selection in the mtDNA of over 700 species of terrestrial vertebrates, freshwater fishes, and arthropods, with information on elevation and latitudinal range limits, range sizes, and body sizes. We confirmed that mitochondrial genomes of high-elevation taxa have slightly higher dN/dS ratios compared to low-elevation relatives. High-elevation species tend to have smaller ranges, which predict higher dN/dS ratios and more relaxed selection across species and clades, while absolute elevation and latitude do not predict higher dN/dS. We also find a positive relationship between body mass and dN/dS, supporting a role for small effective population size leading to relaxed selection. We conclude that higher mt dN/dS among high-elevation species is more likely to reflect relaxed selection due to smaller ranges and reduced effective population size than adaptation to the environment. Our results highlight the importance of rigorously testing adaptive stories against non-adaptive alternative hypotheses, especially in mt genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.08.487647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.08.487647" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.08.487647">
        <p class="paperTitle">Multiple distinct evolutionary mechanisms govern the dynamics of selfish mitochondrial genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.08.487647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.08.487647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gitschlag, B. L.; Pereira, C. V.; Held, J. P.; McCandlish, D. M.; Patel, M. R.</p>
        <p class="info">Score: 11.2, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.08.487647' target='https://doi.org/10.1101/2022.04.08.487647'> 10.1101/2022.04.08.487647</a></p>
        <p class="abstract">Cells possess multiple mitochondrial DNA (mtDNA) copies, which undergo semi-autonomous replication and stochastic inheritance. This enables mutant mtDNA variants to arise and selfishly compete with cooperative (wildtype) mtDNA. Selfish mitochondrial genomes are subject to selection at different levels: they compete against wildtype mtDNA directly within hosts and indirectly through organismal selection. However, determining the relative contributions of selection at different levels has proven challenging. We overcome this challenge by combining mathematical modeling with experiments designed to isolate the levels of selection. Applying this approach to many selfish mitochondrial genotypes in Caenorhabditis elegans revealed an unexpected diversity of evolutionary mechanisms. Some mutant genomes persist at high frequency for many generations, despite a host fitness cost, by aggressively outcompeting cooperative genomes within hosts. Conversely, some mutant genomes persist by evading organismal selection. Strikingly, the mutant genomes vary dramatically in their susceptibility to neutral drift. Although different mechanisms can cause high frequency of selfish mtDNA, we show how they give rise to characteristically different distributions of mutant frequency among individuals. Given that heteroplasmic frequency represents a key determinant of phenotypic severity, this work outlines an evolutionary theoretic framework for predicting the distribution of phenotypic consequences among individuals carrying a selfish mitochondrial genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577571" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577571">
        <p class="paperTitle">Complex interactions in the life cycle of a simple parasite shape the evolution of virulence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silva, L. M.; Koella, J. C.</p>
        <p class="info">Score: 5.9, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577571' target='https://doi.org/10.1101/2024.01.28.577571'> 10.1101/2024.01.28.577571</a></p>
        <p class="abstract">Evolutionary expectations about the virulence of parasites (i.e., the parasite-induced mortality rate of the host) generally describe parasites which grow and are transmitted at the same time. Many parasites, however, go through an incubation period, that is they must grow before they can be transmitted. Moreover, the parasite incubation period and virulence are linked through its growth rate. We here suggest that such interactions make it difficult to predict how virulence will evolve. We illustrate our ideas with a parasite with a simple cycle, the microsporidian Vavraia culicis , which infects the mosquito Anopheles gambiae . We selected the parasite for six generations to kill their host and be transmitted early or late. Selecting for late transmission led to a stronger increasing its exploitation (i.e., cost due to growth), but not pathogenicity per parasite (i.e., cost independent of growth), than selection for early transmission. This was associated with higher virulence, and a shorter parasite life cycle, with the latter having a larger and more rapid production of the infective spores. In response, the host shortened its life cycle and shifted to early fecundity when infected with late-selected spores. Altogether, the results of this study reinforce the need for a reviewed and unified virulence theory and raise important questions for epidemiological and vector control strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577017">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577017" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577017">
        <p class="paperTitle">Sex-biased migration and demographic history of the big European firefly Lampyris noctiluca</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577017" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577017" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Catalan, A.; Gygax, D.; Candolin, U.; Tusso, S.; Duchen, P.; Hoehna, S.</p>
        <p class="info">Score: 5.5, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577017' target='https://doi.org/10.1101/2024.01.24.577017'> 10.1101/2024.01.24.577017</a></p>
        <p class="abstract">Differential dispersion between the sexes can impact population structure and connectivity between populations, which in turn, can have an effect on the evolvability capability of a species. Here we explored the demographic history of the big European firefly, Lampyris noctiluca, which exhibits female neoteny. L. noctiluca distribution extends throughout Europe, but nothing is known on how its colonization processes. To investigate this, we produced the first Lampyris genome (653Mb), including an IsoSeq annotation and the identification of the X chromosome. We collected 115 individuals from six populations of L. noctiluca (Finland to Italy) and generated whole genome re-sequencing for each individual. We inferred several population expansions and bottlenecks throughout the Pleistocene that correlate with glaciation events. Surprisingly, we uncovered strong population structure and low gene-flow. We reject a stepwise, south to north, colonization history scenario and instead uncovered a complex demographic history with a putative eastern European origin. Analyzing the evolutionary history of the mitochondrial genome as well as X-linked and autosomal loci, we found evidence of a maternal colonialization of Germany, putatively from a western European population, followed by male-only migration from south of the Alps (Italy). Overall, investigating the demographic history and colonization patterns of a species should form part of an integrative approach of biodiversity research. Our results provide evidence of sex-biased migration which is important to consider for demographic, biogeographic and species delimitation studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576864" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576864">
        <p class="paperTitle">Maternal manipulation of offspring size can trigger the evolution of eusociality in promiscuous species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rees-Baylis, E.; Pen, I.; Kreider, J. J.</p>
        <p class="info">Score: 4.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576864' target='https://doi.org/10.1101/2024.01.23.576864'> 10.1101/2024.01.23.576864</a></p>
        <p class="abstract">Eusocial organisms typically live in colonies with one reproductive queen supported by thousands of sterile workers. It is widely believed that monogamous mating is a precondition for the evolution of eusociality. Here, we present a theoretical model that simulates a realistic scenario for the evolution of eusociality. In the model, mothers can evolve control over resource allocation to offspring, affecting offspring body size. The offspring can evolve body-size-dependent dispersal, by which they disperse to breed or stay at the nest as helpers. We demonstrate that eusociality evolves, even if mothers are not strictly monogamous, if mothers can constrain their offsprings reproduction by manipulation. We also observe the evolution of social polymorphism with small individuals that help and larger individuals that disperse to breed. Our model unifies the traditional kin selection and maternal manipulation explanations for the evolution of eusociality and demonstrates that - contrary to current consensus belief - eusociality can evolve despite highly promiscuous mating.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577607" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577607">
        <p class="paperTitle">Adaptive and pleiotropic effects of evolution in synonymous sugar environments.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahlawat, N.; Venkataraman, P.; Mahilkar, A.; Brajesh, R. G.; Saini, S.</p>
        <p class="info">Score: 3.5, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577607' target='https://doi.org/10.1101/2024.01.28.577607'> 10.1101/2024.01.28.577607</a></p>
        <p class="abstract">Adaptation to an environment is enabled by the accumulation of beneficial mutations. When adapted populations are shifted to other environments, the byproduct or pleiotropic fitness effects of these mutations can be wide-ranged. Since there exists no molecular framework to quantify relatedness of environments, predicting pleiotropic effects based on adaptation has been challenging. In this work, we ask if evolution in highly similar environments elicits correlated adaptive and pleiotropic responses. We evolve replicate populations of Escherichia coli in non-stressful environments that contain either a mixture of glucose and galactose, lactose, or melibiose as the source of carbon. We term these similar sugars as synonymous, since lactose and melibiose are disaccharides made up of glucose and galactose. Therefore, the evolution environments differed only in the way carbon was presented to the bacterial population. After 300 generations of evolution, we see that the adaptive responses of these populations are not predictable. We investigate the pleiotropic effects of adaptation in a range of non-synonymous environments, and show that despite uncorrelated adaptive changes, the nature of pleiotropic effects is largely predictable based on the fitness of the ancestor in the non-home environments. Overall, our results highlight how subtle changes in the environment can alter adaptation, but despite sequence-level variations, pleiotropy is qualitatively predictable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577314" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577314">
        <p class="paperTitle">On the annotation of Paleo- and Mesoproterozoic microfossils as fungi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kanaparthi, D.; Lampe, M.; Krings, M.; Schue, B. S.; Klingl, A.; Schwille, P.; Lueders, T.</p>
        <p class="info">Score: 2.8, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577314' target='https://doi.org/10.1101/2024.01.25.577314'> 10.1101/2024.01.25.577314</a></p>
        <p class="abstract">How old is the crown group Fungi? Inferences from phylogenetic and fossil-based studies provided far-apart age estimates ranging between 0.75 to 2.7 billion years old. One important criterion for interpreting Paleo- and Mesoproterozoic microfossils as Fungi is their uncanny morphological resemblance with extant fungi. Here, we demonstrate that bacteria exposed to environmental conditions similar to the paleoenvironmental settings where these presumed fungi lived can spontaneously transform into their protoplasts. These protoplasts exhibit morphologies corresponding to those of presumed fungal microfossils. These observations, together with microfossil chemical composition, pose a serious challenge to interpreting Paleo- and Mesoproterozoic microfossils as fungi. Based on these results, we reiterate that morphology is not a reliable indicator of the phylogeny of microfossils older than 2.5Ga.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
