<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Quantitative Genetic Model of Background Selection in Humans
Authors: Buffalo, V.; Kern, A. D.
Score: 27.2, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762
Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="A Quantitative Genetic Model of Background Selection in Humans
Authors: Buffalo, V.; Kern, A. D.
Score: 27.2, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762
Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="A Quantitative Genetic Model of Background Selection in Humans
Authors: Buffalo, V.; Kern, A. D.
Score: 27.2, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762
Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "A Quantitative Genetic Model of Background Selection in Humans\nAuthors: Buffalo, V.; Kern, A. D.\nScore: 27.2, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762\nAcross the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This \u0026#34;linked selection signal\u0026#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation.",
  "keywords": [
    
  ],
  "articleBody": " A Quantitative Genetic Model of Background Selection in Humans\nAuthors: Buffalo, V.; Kern, A. D.\nScore: 27.2, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762\nAcross the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This \"linked selection signal\" reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation. To date, statistical efforts to estimate purifying selection parameters from linked selection models have relied on classic Background Selection theory, which is only applicable when new mutations are so deleterious that they cannot fix in the population. Here, we develop a statistical method based on a quantitative genetics view of linked selection, that models how polygenic additive fitness variance distributed along the genome increases the rate of stochastic allele frequency change. By jointly predicting the equilibrium fitness variance and substitution rate due to both strong and weakly deleterious mutations, we estimate the distribution of fitness effects (DFE) and mutation rate across three geographically distinct human samples. While our model can accommodate weaker selection, we find evidence of strong selection operating similarly across all human samples. While our quantitative genetic model of linked selection fits better than previous models, substitution rates of the most constrained sites are poorly predicted. We find that a model incorporating selective interference better predicts observed divergence in conserved regions, but overall our results ultimately suggest considerable uncertainty remains about the processes generating fitness variation in humans.\nDetecting shifts in the mode of chromosomal speciation across the cosmopolitan plant lineage Carex\nAuthors: Tribble, C. M.; Marquez Corro, J. I.; May, M. R.; Hipp, A. L.; Escudero, M.; Zenil-Ferguson, R.\nScore: 24.9, Published: 2023-09-10 DOI: 10.1101/2023.09.05.556433\nRecent studies of angiosperm diversification have focused on the role of polyploidy as a driver of diversification. However, we know far less about the effects of single changes in chromosome number---dysploidy, which can mediate lineage diversification by affecting recombination rates, linkage, and/or reproductive isolation. Modeling the effects of dysploidy on diversification is mathematically and computationally challenging because many states and parameters are required to track changes in individual chromosomes, especially in lineages that have high variability in chromosome number. Additionally, we expect the processes of diversification and dysploidy to vary across clades, which requires modeling process variation to disentangle the effects of the observed trait (chromosome number) from the effects of unobserved traits on diversification. In this work, we propose a new state-dependent diversification model of chromosome evolution that includes numerous character states and explicitly models heterogeneity in the diversification process. Our model includes parameters that functionally link diversification rates to dysploidy rates and differentiate between anagenetic and cladogenetic changes. We apply this model to Carex (Cyperaceae), a model lineage for understanding dysploidy and diversification, leveraging chromosome number information and the most recent time-calibrated phylogenetic tree for over 700 species and subspecies. We recover distinct modes of chromosomal speciation across Carex. In one mode, dysploidy occurs very frequently and drives faster diversification rates. In the other mode, dysploidy is rare and diversification is driven by other factors, unmeasured in our analysis. This study is the first to demonstrate that dysploidy drives diversification in plants while considering unmeasured factors affecting diversification.\nSpeciation in kleptoparasites of oak gall wasps often correlates with shifts into new tree habitats, tree organs, and gall morphospace\nAuthors: Ward, A.; Zhang, Y. M.; Brown, G. E.; Hippee, A. C.; Prior, K. M.; Rollins, S.; Sierra, N.; Sheikh, S. I.; Tribull, C. M.; Forbes, A. A.\nScore: 18.8, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556376\nHost shifts to new plants can drive speciation for plant-feeding insects, but how commonly do host shifts also drive diversification for the parasites of those same insects? Oak gall wasps induce galls on oak trees and shifts to novel tree hosts and new tree organs have been implicated as drivers of oak gall wasp speciation. Gall wasps are themselves attacked by many insect parasites, which must find their hosts on the correct tree species and organ, but which also must navigate the morphologically variable galls with which they interact. Thus, we ask whether host shifts to new trees, organs, or gall morphologies correlate with gall parasite diversification. We delimit species and infer phylogenies for two genera of gall kleptoparasites, Synergus and Ceroptres, reared from a variety of North American oak galls. We find that most species were reared from galls induced by just one gall wasp species, and no parasite species was reared from galls of more than four species. Most kleptoparasite divergence events correlate with shifts to non-ancestral galls. These shifts often involved changes in tree habitat, gall location, and gall morphology. Host shifts are thus implicated in driving diversification for both oak gall wasps and their kleptoparasitic associates.\nConserved switch genes regulate a novel cannibalistic morph after whole genome duplication\nAuthors: Wighard, S.; Sommer, R. J.; Witte, H.\nScore: 21.3, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554244\nDevelopmental plasticity facilitates morphological and behavioural novelty, but associated regulatory mechanisms remain elusive. Nematodes have emerged as a powerful model to study developmental plasticity and its evolution. Here, we show the predatory nematode Allodiplogaster sudhausi evolved an additional third mouth morph, concomitant with whole genome duplication (WGD) and a strong increase in body size. The three mouth morphs are induced by different diets; bacteria, fungi and nematodes. CRISPR experiments indicate that regulation of the third morph involves co-option of a conserved developmental switch gene, which through WGD resulted in two mouth-form regulators. Gene dosage studies revealed a diverged role of these developmental switches, with functional redundancy and quantitative effects in the two mouth-form decisions, respectively. The third morph is cannibalistic and kills kin, whereas the other two morphs do not. Thus, the recent evolution of a new morph relies on pre-existing regulatory mechanisms and adds behavioural and social complexity. One-Sentence SummaryExperimental genetics in a nematode reveals a key role for developmental plasticity in the evolution of nutritional diversity\nExtreme high-elevation mammal surveys reveal unexpectedly high upper range limits of Andean mice\nAuthors: Storz, J.; Quiroga-Carmona, M.; Liphardt, S.; Bautista, N. M.; Opazo, J. C.; Rico, A.; Salazar-Bravo, J.; Good, J.; D'Elia, G.\nScore: 19.5, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554215\nIn the worlds highest mountain ranges, uncertainty about the upper elevational range limits of alpine animals represents a critical knowledge gap regarding the environmental limits of life and presents a problem for detecting range shifts in response to climate change. Here we report results of mountaineering mammal surveys in the Central Andes, which led to the discovery of multiple species of mice living at extreme elevations that far surpass previously assumed range limits for mammals. We live-trapped small mammals from ecologically diverse sites spanning \u003e6700 m of vertical relief, from the desert coast of northern Chile to the summits of the highest volcanoes in the Andes. We used molecular sequence data and whole-genome sequence data to confirm the identities of species that represent new elevational records and to test hypotheses regarding species limits. These discoveries contribute to a new appreciation of the environmental limits of vertebrate life.\nDeep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution\nAuthors: Taylor, A. L.; Starr, T. N.\nScore: 12.2, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279\nSubstitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear. We conducted deep mutational scans to measure the impacts of all single amino acid mutations and single-codon deletions in the spike receptor-binding domain (RBD) on ACE2-binding affinity and protein expression in the recent Omicron BQ.1.1 and XBB.1.5 variants, and we compared mutational patterns to earlier viral strains that we have previously profiled. As with previous RBD deep mutational scans, we find many mutations that are tolerated or even enhance binding to ACE2 receptor. The tolerance of sites to single-codon deletion largely conforms with tolerance to amino acid mutation. Though deletions in the RBD have not yet been seen in dominant lineages, we observe many tolerated deletions including at positions that exhibit indel variation across broader sarbecovirus evolution and in emerging SARS-CoV-2 variants of interest, most notably the well-tolerated {Delta}483 deletion in BA.2.86. The substitutions that distinguish recent viral variants have not induced as dramatic of epistatic perturbations as N501Y, but we identify ongoing epistatic drift in SARS-CoV-2 variants, including interaction between R493Q reversions and mutations at positions 453, 455, and 456, including mutations like F456L that define the newly emerging EG.5 lineage. Our results highlight ongoing drift in the effects of mutations due to epistasis, which may continue to direct SARS-CoV-2 evolution into new regions of sequence space.\nHaplotype independence contributes to evolvability in the long-term absence of sex in a mite\nAuthors: Oztoprak, H.; Gao, S.; Guiglielmoni, N.; Brandt, A.; Zheng, Y.; Becker, C.; Becker, K.; Bednarski, V.; Borgschulte, L.; Burak, K. A.; Dion-Cote, A.-M.; Leonov, V.; Opherden, L.; Shimano, S.; Bast, J.\nScore: 10.7, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556471\nSome unique asexual species persist over time and contradict the consensus that sex is a prerequisite for long-term evolutionary survival. How they escape the dead-end fate remains enigmatic. Here, we generated a haplotype-resolved genome assembly based on a single individual and collected genomic data from worldwide populations of the parthenogenetic diploid oribatid mite Platynothrus peltifer to identify signatures of persistence without sex. We found that haplotypes diverge independently since the transition to asexuality at least 20 my ago. Multiple lines of evidence indicate disparate evolutionary trajectories between haplotypic blocks. Our findings imply that such haplotypic independence can lead to non-canonical routes of evolvability, helping some species to adapt, diversify and persist for millions of years in the absence of sex.\nThe genomic landscape, causes, and consequences of extensive phylogenomic discordance in Old World mice and rats\nAuthors: Thomas, G. W. C.; Hughes, J. J.; Kumon, T.; Berv, J. S.; Nordgren, C. E.; Lampson, M.; Levine, M.; Searle, J. B.; Good, J. M.\nScore: 10.7, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555178\nA species tree is a central concept in evolutionary biology whereby a single branching phylogeny reflects relationships among species. However, the phylogenies of different genomic regions often differ from the species tree. Although tree discordance is often widespread in phylogenomic studies, we still lack a clear understanding of how variation in phylogenetic patterns is shaped by genome biology or the extent to which discordance may compromise comparative studies. We characterized patterns of phylogenomic discordance across the murine rodents (Old World mice and rats) - a large and ecologically diverse group that gave rise to the mouse and rat model systems. Combining new linked-read genome assemblies for seven murine species with eleven published rodent genomes, we first used ultra-conserved elements (UCEs) to infer a robust species tree. We then used whole genomes to examine finer-scale patterns of discordance and found that phylogenies built from proximate chromosomal regions had similar phylogenies. However, there was no relationship between tree similarity and local recombination rates in house mice, suggesting that genetic linkage influences phylogenetic patterns over deeper timescales. This signal may be independent of contemporary recombination landscapes. We also detected a strong influence of linked selection whereby purifying selection at UCEs led to less discordance, while genes experiencing positive selection showed more discordant and variable phylogenetic signals. Finally, we show that assuming a single species tree can result in high error rates when testing for positive selection under different models. Collectively, our results highlight the complex relationship between phylogenetic inference and genome biology and underscore how failure to account for this complexity can mislead comparative genomic studies.\nNatural genomic variation in rice blast genomes is associated with specific heterochromatinmodifications\nAuthors: Rowe, D.; Huang, J.; Zhang, W.; Mishra, D.; Jordan, K. W.; Valent, B.; Liu, S.; Cook, D.\nScore: 12.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555587\nGenome organization in eukaryotes exhibits non-random patterns tied to transcription, replication, and chromatin. However, the driving forces across these processes, and their impacts on genome evolution remain unclear. To address this, we analyzed sequence data from 86 Magnaporthe oryzae isolates, a globally important plant pathogenic fungus, alongside new reference genomes, to investigate DNA sequence variation and the epigenome. Histone modification profiles were obtained through genome-wide chromatin immunoprecipitation-sequencing of the four new reference strains, which revealed that repressive histone marks (H3K27me3, H3K9me3) were associated with SNP and INDEL frequency. Densely grouped SNPs were found to reside in heterochromatin, often outside transposable elements, highlighting the link between heterochromatin and DNA variation. Even when controlling for selection, silent SNP frequency/kb was higher in H3K27me3-associated genes. Effector genes, key to pathogenicity, also displayed this trend. Comparing the reference strains, euchromatic regions were often syntenic, while heterochromatic regions trended towards non-syntenic. Heterochromatin emerged as a major factor associated with diverse DNA variations in M. oryzae populations, even when accounting for selective pressure. This underscores heterochromatins pivotal role in shaping genetic diversity in these mainly asexually reproducing fungi.\nSchizosaccharomyces versatilis represents a distinct evolutionary lineage.\nAuthors: Etherington, G. J.; Nieduszynski, C.; Oliferenko, S.; Gomez Gil, E.; Haerty, W.\nScore: 9.5, Published: 2023-09-06 DOI: 10.1101/2023.09.04.556188\nThe fission yeast species Schizosaccharomyces japonicus is currently divided into two variants - S. japonicus var. japonicus and S. japonicus var. versatilis. Here we examine the var. versatilis isolate CBS5679. The CBS5679 genome shows 88% identity to the reference genome of S. japonicus var. japonicus at the coding sequence level, with phylogenetic analyses suggesting that it has split from the S. japonicus lineage 25 million years ago. The CBS5679 genome contains a reciprocal translocation between chromosomes 1 and 2, together with several large inversions. The products of genes linked to the major translocation are associated with \"metabolism\" and \"cellular assembly\" ontology terms. We further show that CBS5679 does not generate viable progeny with the reference strain of S. japonicus. Although CBS5679 shares closer similarity to the \"type\" strain of var. versatilis as compared to S. japonicus, it is not identical to the type strain, suggesting population structure within var. versatilis. We recommend that the taxonomic status of S. japonicus var. versatilis is raised, with it being treated as a separate species, Schizosaccharomyces versatilis. Take-awayO_LIThe taxonomic status of Schizosaccharomyces versatilis is addressed. C_LIO_LIS. versatilis diverged from S. japonicus around 25 million years ago. C_LIO_LIS. versatilis does not produce viable progeny in crosses with S. japonicus. C_LIO_LIS. versatilis has a reciprocal translocation between chromosomes 1 and 2. C_LIO_LIThe Gene Ontology terms for genes in the translocations are enriched for terms connected to \"metabolism\" and \"cellular assembly\". C_LI\n",
  "wordCount" : "2436",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556762">
        <p class="paperTitle">A Quantitative Genetic Model of Background Selection in Humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buffalo, V.; Kern, A. D.</p>
        <p class="info">Score: 27.2, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556762' target='https://doi.org/10.1101/2023.09.07.556762'> 10.1101/2023.09.07.556762</a></p>
        <p class="abstract">Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation. To date, statistical efforts to estimate purifying selection parameters from linked selection models have relied on classic Background Selection theory, which is only applicable when new mutations are so deleterious that they cannot fix in the population. Here, we develop a statistical method based on a quantitative genetics view of linked selection, that models how polygenic additive fitness variance distributed along the genome increases the rate of stochastic allele frequency change. By jointly predicting the equilibrium fitness variance and substitution rate due to both strong and weakly deleterious mutations, we estimate the distribution of fitness effects (DFE) and mutation rate across three geographically distinct human samples. While our model can accommodate weaker selection, we find evidence of strong selection operating similarly across all human samples. While our quantitative genetic model of linked selection fits better than previous models, substitution rates of the most constrained sites are poorly predicted. We find that a model incorporating selective interference better predicts observed divergence in conserved regions, but overall our results ultimately suggest considerable uncertainty remains about the processes generating fitness variation in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556433">
        <p class="paperTitle">Detecting shifts in the mode of chromosomal speciation across the cosmopolitan plant lineage Carex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tribble, C. M.; Marquez Corro, J. I.; May, M. R.; Hipp, A. L.; Escudero, M.; Zenil-Ferguson, R.</p>
        <p class="info">Score: 24.9, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556433' target='https://doi.org/10.1101/2023.09.05.556433'> 10.1101/2023.09.05.556433</a></p>
        <p class="abstract">Recent studies of angiosperm diversification have focused on the role of polyploidy as a driver of diversification. However, we know far less about the effects of single changes in chromosome number---dysploidy, which can mediate lineage diversification by affecting recombination rates, linkage, and/or reproductive isolation. Modeling the effects of dysploidy on diversification is mathematically and computationally challenging because many states and parameters are required to track changes in individual chromosomes, especially in lineages that have high variability in chromosome number. Additionally, we expect the processes of diversification and dysploidy to vary across clades, which requires modeling process variation to disentangle the effects of the observed trait (chromosome number) from the effects of unobserved traits on diversification. In this work, we propose a new state-dependent diversification model of chromosome evolution that includes numerous character states and explicitly models heterogeneity in the diversification process. Our model includes parameters that functionally link diversification rates to dysploidy rates and differentiate between anagenetic and cladogenetic changes. We apply this model to Carex (Cyperaceae), a model lineage for understanding dysploidy and diversification, leveraging chromosome number information and the most recent time-calibrated phylogenetic tree for over 700 species and subspecies. We recover distinct modes of chromosomal speciation across Carex. In one mode, dysploidy occurs very frequently and drives faster diversification rates. In the other mode, dysploidy is rare and diversification is driven by other factors, unmeasured in our analysis. This study is the first to demonstrate that dysploidy drives diversification in plants while considering unmeasured factors affecting diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556376">
        <p class="paperTitle">Speciation in kleptoparasites of oak gall wasps often correlates with shifts into new tree habitats, tree organs, and gall morphospace</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ward, A.; Zhang, Y. M.; Brown, G. E.; Hippee, A. C.; Prior, K. M.; Rollins, S.; Sierra, N.; Sheikh, S. I.; Tribull, C. M.; Forbes, A. A.</p>
        <p class="info">Score: 18.8, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556376' target='https://doi.org/10.1101/2023.09.07.556376'> 10.1101/2023.09.07.556376</a></p>
        <p class="abstract">Host shifts to new plants can drive speciation for plant-feeding insects, but how commonly do host shifts also drive diversification for the parasites of those same insects? Oak gall wasps induce galls on oak trees and shifts to novel tree hosts and new tree organs have been implicated as drivers of oak gall wasp speciation. Gall wasps are themselves attacked by many insect parasites, which must find their hosts on the correct tree species and organ, but which also must navigate the morphologically variable galls with which they interact. Thus, we ask whether host shifts to new trees, organs, or gall morphologies correlate with gall parasite diversification. We delimit species and infer phylogenies for two genera of gall kleptoparasites, Synergus and Ceroptres, reared from a variety of North American oak galls. We find that most species were reared from galls induced by just one gall wasp species, and no parasite species was reared from galls of more than four species. Most kleptoparasite divergence events correlate with shifts to non-ancestral galls. These shifts often involved changes in tree habitat, gall location, and gall morphology. Host shifts are thus implicated in driving diversification for both oak gall wasps and their kleptoparasitic associates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554244">
        <p class="paperTitle">Conserved switch genes regulate a novel cannibalistic morph after whole genome duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wighard, S.; Sommer, R. J.; Witte, H.</p>
        <p class="info">Score: 21.3, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554244' target='https://doi.org/10.1101/2023.08.22.554244'> 10.1101/2023.08.22.554244</a></p>
        <p class="abstract">Developmental plasticity facilitates morphological and behavioural novelty, but associated regulatory mechanisms remain elusive. Nematodes have emerged as a powerful model to study developmental plasticity and its evolution. Here, we show the predatory nematode Allodiplogaster sudhausi evolved an additional third mouth morph, concomitant with whole genome duplication (WGD) and a strong increase in body size. The three mouth morphs are induced by different diets; bacteria, fungi and nematodes. CRISPR experiments indicate that regulation of the third morph involves co-option of a conserved developmental switch gene, which through WGD resulted in two mouth-form regulators. Gene dosage studies revealed a diverged role of these developmental switches, with functional redundancy and quantitative effects in the two mouth-form decisions, respectively. The third morph is cannibalistic and kills kin, whereas the other two morphs do not. Thus, the recent evolution of a new morph relies on pre-existing regulatory mechanisms and adds behavioural and social complexity.

One-Sentence SummaryExperimental genetics in a nematode reveals a key role for developmental plasticity in the evolution of nutritional diversity</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554215">
        <p class="paperTitle">Extreme high-elevation mammal surveys reveal unexpectedly high upper range limits of Andean mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Storz, J.; Quiroga-Carmona, M.; Liphardt, S.; Bautista, N. M.; Opazo, J. C.; Rico, A.; Salazar-Bravo, J.; Good, J.; D&#39;Elia, G.</p>
        <p class="info">Score: 19.5, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554215' target='https://doi.org/10.1101/2023.08.22.554215'> 10.1101/2023.08.22.554215</a></p>
        <p class="abstract">In the worlds highest mountain ranges, uncertainty about the upper elevational range limits of alpine animals represents a critical knowledge gap regarding the environmental limits of life and presents a problem for detecting range shifts in response to climate change. Here we report results of mountaineering mammal surveys in the Central Andes, which led to the discovery of multiple species of mice living at extreme elevations that far surpass previously assumed range limits for mammals. We live-trapped small mammals from ecologically diverse sites spanning &gt;6700 m of vertical relief, from the desert coast of northern Chile to the summits of the highest volcanoes in the Andes. We used molecular sequence data and whole-genome sequence data to confirm the identities of species that represent new elevational records and to test hypotheses regarding species limits. These discoveries contribute to a new appreciation of the environmental limits of vertebrate life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557279">
        <p class="paperTitle">Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, A. L.; Starr, T. N.</p>
        <p class="info">Score: 12.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557279' target='https://doi.org/10.1101/2023.09.11.557279'> 10.1101/2023.09.11.557279</a></p>
        <p class="abstract">Substitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear. We conducted deep mutational scans to measure the impacts of all single amino acid mutations and single-codon deletions in the spike receptor-binding domain (RBD) on ACE2-binding affinity and protein expression in the recent Omicron BQ.1.1 and XBB.1.5 variants, and we compared mutational patterns to earlier viral strains that we have previously profiled. As with previous RBD deep mutational scans, we find many mutations that are tolerated or even enhance binding to ACE2 receptor. The tolerance of sites to single-codon deletion largely conforms with tolerance to amino acid mutation. Though deletions in the RBD have not yet been seen in dominant lineages, we observe many tolerated deletions including at positions that exhibit indel variation across broader sarbecovirus evolution and in emerging SARS-CoV-2 variants of interest, most notably the well-tolerated {Delta}483 deletion in BA.2.86. The substitutions that distinguish recent viral variants have not induced as dramatic of epistatic perturbations as N501Y, but we identify ongoing epistatic drift in SARS-CoV-2 variants, including interaction between R493Q reversions and mutations at positions 453, 455, and 456, including mutations like F456L that define the newly emerging EG.5 lineage. Our results highlight ongoing drift in the effects of mutations due to epistasis, which may continue to direct SARS-CoV-2 evolution into new regions of sequence space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556471">
        <p class="paperTitle">Haplotype independence contributes to evolvability in the long-term absence of sex in a mite</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oztoprak, H.; Gao, S.; Guiglielmoni, N.; Brandt, A.; Zheng, Y.; Becker, C.; Becker, K.; Bednarski, V.; Borgschulte, L.; Burak, K. A.; Dion-Cote, A.-M.; Leonov, V.; Opherden, L.; Shimano, S.; Bast, J.</p>
        <p class="info">Score: 10.7, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556471' target='https://doi.org/10.1101/2023.09.07.556471'> 10.1101/2023.09.07.556471</a></p>
        <p class="abstract">Some unique asexual species persist over time and contradict the consensus that sex is a prerequisite for long-term evolutionary survival. How they escape the dead-end fate remains enigmatic. Here, we generated a haplotype-resolved genome assembly based on a single individual and collected genomic data from worldwide populations of the parthenogenetic diploid oribatid mite Platynothrus peltifer to identify signatures of persistence without sex. We found that haplotypes diverge independently since the transition to asexuality at least 20 my ago. Multiple lines of evidence indicate disparate evolutionary trajectories between haplotypic blocks. Our findings imply that such haplotypic independence can lead to non-canonical routes of evolvability, helping some species to adapt, diversify and persist for millions of years in the absence of sex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555178">
        <p class="paperTitle">The genomic landscape, causes, and consequences of extensive phylogenomic discordance in Old World mice and rats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, G. W. C.; Hughes, J. J.; Kumon, T.; Berv, J. S.; Nordgren, C. E.; Lampson, M.; Levine, M.; Searle, J. B.; Good, J. M.</p>
        <p class="info">Score: 10.7, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555178' target='https://doi.org/10.1101/2023.08.28.555178'> 10.1101/2023.08.28.555178</a></p>
        <p class="abstract">A species tree is a central concept in evolutionary biology whereby a single branching phylogeny reflects relationships among species. However, the phylogenies of different genomic regions often differ from the species tree. Although tree discordance is often widespread in phylogenomic studies, we still lack a clear understanding of how variation in phylogenetic patterns is shaped by genome biology or the extent to which discordance may compromise comparative studies. We characterized patterns of phylogenomic discordance across the murine rodents (Old World mice and rats) - a large and ecologically diverse group that gave rise to the mouse and rat model systems. Combining new linked-read genome assemblies for seven murine species with eleven published rodent genomes, we first used ultra-conserved elements (UCEs) to infer a robust species tree. We then used whole genomes to examine finer-scale patterns of discordance and found that phylogenies built from proximate chromosomal regions had similar phylogenies. However, there was no relationship between tree similarity and local recombination rates in house mice, suggesting that genetic linkage influences phylogenetic patterns over deeper timescales. This signal may be independent of contemporary recombination landscapes. We also detected a strong influence of linked selection whereby purifying selection at UCEs led to less discordance, while genes experiencing positive selection showed more discordant and variable phylogenetic signals. Finally, we show that assuming a single species tree can result in high error rates when testing for positive selection under different models. Collectively, our results highlight the complex relationship between phylogenetic inference and genome biology and underscore how failure to account for this complexity can mislead comparative genomic studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555587">
        <p class="paperTitle">Natural genomic variation in rice blast genomes is associated with specific heterochromatinmodifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rowe, D.; Huang, J.; Zhang, W.; Mishra, D.; Jordan, K. W.; Valent, B.; Liu, S.; Cook, D.</p>
        <p class="info">Score: 12.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555587' target='https://doi.org/10.1101/2023.08.30.555587'> 10.1101/2023.08.30.555587</a></p>
        <p class="abstract">Genome organization in eukaryotes exhibits non-random patterns tied to transcription, replication, and chromatin. However, the driving forces across these processes, and their impacts on genome evolution remain unclear. To address this, we analyzed sequence data from 86 Magnaporthe oryzae isolates, a globally important plant pathogenic fungus, alongside new reference genomes, to investigate DNA sequence variation and the epigenome. Histone modification profiles were obtained through genome-wide chromatin immunoprecipitation-sequencing of the four new reference strains, which revealed that repressive histone marks (H3K27me3, H3K9me3) were associated with SNP and INDEL frequency. Densely grouped SNPs were found to reside in heterochromatin, often outside transposable elements, highlighting the link between heterochromatin and DNA variation. Even when controlling for selection, silent SNP frequency/kb was higher in H3K27me3-associated genes. Effector genes, key to pathogenicity, also displayed this trend. Comparing the reference strains, euchromatic regions were often syntenic, while heterochromatic regions trended towards non-syntenic. Heterochromatin emerged as a major factor associated with diverse DNA variations in M. oryzae populations, even when accounting for selective pressure. This underscores heterochromatins pivotal role in shaping genetic diversity in these mainly asexually reproducing fungi.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556188">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556188" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556188">
        <p class="paperTitle">Schizosaccharomyces versatilis represents a distinct evolutionary lineage.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556188" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556188" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Etherington, G. J.; Nieduszynski, C.; Oliferenko, S.; Gomez Gil, E.; Haerty, W.</p>
        <p class="info">Score: 9.5, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556188' target='https://doi.org/10.1101/2023.09.04.556188'> 10.1101/2023.09.04.556188</a></p>
        <p class="abstract">The fission yeast species Schizosaccharomyces japonicus is currently divided into two variants - S. japonicus var. japonicus and S. japonicus var. versatilis. Here we examine the var. versatilis isolate CBS5679. The CBS5679 genome shows 88% identity to the reference genome of S. japonicus var. japonicus at the coding sequence level, with phylogenetic analyses suggesting that it has split from the S. japonicus lineage 25 million years ago. The CBS5679 genome contains a reciprocal translocation between chromosomes 1 and 2, together with several large inversions. The products of genes linked to the major translocation are associated with &#34;metabolism&#34; and &#34;cellular assembly&#34; ontology terms. We further show that CBS5679 does not generate viable progeny with the reference strain of S. japonicus. Although CBS5679 shares closer similarity to the &#34;type&#34; strain of var. versatilis as compared to S. japonicus, it is not identical to the type strain, suggesting population structure within var. versatilis. We recommend that the taxonomic status of S. japonicus var. versatilis is raised, with it being treated as a separate species, Schizosaccharomyces versatilis.

Take-awayO_LIThe taxonomic status of Schizosaccharomyces versatilis is addressed.
C_LIO_LIS. versatilis diverged from S. japonicus around 25 million years ago.
C_LIO_LIS. versatilis does not produce viable progeny in crosses with S. japonicus.
C_LIO_LIS. versatilis has a reciprocal translocation between chromosomes 1 and 2.
C_LIO_LIThe Gene Ontology terms for genes in the translocations are enriched for terms connected to &#34;metabolism&#34; and &#34;cellular assembly&#34;.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
