<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution
Authors: Taylor, A. L.; Starr, T. N.
Score: 81.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279
Substitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution
Authors: Taylor, A. L.; Starr, T. N.
Score: 81.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279
Substitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution
Authors: Taylor, A. L.; Starr, T. N.
Score: 81.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279
Substitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution\nAuthors: Taylor, A. L.; Starr, T. N.\nScore: 81.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279\nSubstitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear.",
  "keywords": [
    
  ],
  "articleBody": " Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution\nAuthors: Taylor, A. L.; Starr, T. N.\nScore: 81.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279\nSubstitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear. We conducted deep mutational scans to measure the impacts of all single amino acid mutations and single-codon deletions in the spike receptor-binding domain (RBD) on ACE2-binding affinity and protein expression in the recent Omicron BQ.1.1 and XBB.1.5 variants, and we compared mutational patterns to earlier viral strains that we have previously profiled. As with previous RBD deep mutational scans, we find many mutations that are tolerated or even enhance binding to ACE2 receptor. The tolerance of sites to single-codon deletion largely conforms with tolerance to amino acid mutation. Though deletions in the RBD have not yet been seen in dominant lineages, we observe many tolerated deletions including at positions that exhibit indel variation across broader sarbecovirus evolution and in emerging SARS-CoV-2 variants of interest, most notably the well-tolerated {Delta}483 deletion in BA.2.86. The substitutions that distinguish recent viral variants have not induced as dramatic of epistatic perturbations as N501Y, but we identify ongoing epistatic drift in SARS-CoV-2 variants, including interaction between R493Q reversions and mutations at positions 453, 455, and 456, including mutations like F456L that define the newly emerging EG.5 lineage. Our results highlight ongoing drift in the effects of mutations due to epistasis, which may continue to direct SARS-CoV-2 evolution into new regions of sequence space.\nViperin immunity evolved across the tree of life through serial innovations on a conserved scaffold\nAuthors: Shomar, H.; Georjon, H.; Feng, Y.; Olympio, B.; Tesson, F.; Cury, J.; Wu, F.; Bernheim, A.\nScore: 47.0, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557418\nEvolutionary arms races between cells and viruses drive the rapid diversification of antiviral genes in diverse life forms. Recent discoveries have revealed the existence of shared immune genes between prokaryotes and eukaryotes, showing molecular and mechanistic similarities in their response to viruses. However, the underlying evolutionary dynamics that explain the conservation and adaptation of these antiviral genes remain mostly unexplored. Here, we show that viperins constitute a highly conserved family of immune genes across diverse prokaryotes and eukaryotes, and uncover mechanisms by which they diversified in eukaryotes. Our findings indicate that viperins are enriched in Asgard archaea and widely distributed in all major eukaryotic clades, suggesting their presence in the Last Eukaryotic Common Ancestor (LECA). We show that viperins maintain their immune function by producing antiviral nucleotide analogs. We demonstrate that eukaryotic viperins diversified through serial innovations on the viperin gene, such as the emergence and selection of substrate specificity towards pyrimidine nucleotides, and through partnerships with genes maintained through genetic linkage, notably with nucleotide kinases. These findings unveil biochemical and genomic transitions underlying the adaptation of immune genes shared by prokaryotes and eukaryotes. Our study paves the way for the understanding of the conservation of immunity across domains of life.\nConserved switch genes regulate a novel cannibalistic morph after whole genome duplication\nAuthors: Wighard, S.; Sommer, R. J.; Witte, H.\nScore: 37.4, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554244\nDevelopmental plasticity facilitates morphological and behavioural novelty, but associated regulatory mechanisms remain elusive. Nematodes have emerged as a powerful model to study developmental plasticity and its evolution. Here, we show the predatory nematode Allodiplogaster sudhausi evolved an additional third mouth morph, concomitant with whole genome duplication (WGD) and a strong increase in body size. The three mouth morphs are induced by different diets; bacteria, fungi and nematodes. CRISPR experiments indicate that regulation of the third morph involves co-option of a conserved developmental switch gene, which through WGD resulted in two mouth-form regulators. Gene dosage studies revealed a diverged role of these developmental switches, with functional redundancy and quantitative effects in the two mouth-form decisions, respectively. The third morph is cannibalistic and kills kin, whereas the other two morphs do not. Thus, the recent evolution of a new morph relies on pre-existing regulatory mechanisms and adds behavioural and social complexity. One-Sentence SummaryExperimental genetics in a nematode reveals a key role for developmental plasticity in the evolution of nutritional diversity\nDetecting shifts in the mode of chromosomal speciation across the cosmopolitan plant lineage Carex\nAuthors: Tribble, C. M.; Marquez Corro, J. I.; May, M. R.; Hipp, A. L.; Escudero, M.; Zenil-Ferguson, R.\nScore: 24.9, Published: 2023-09-10 DOI: 10.1101/2023.09.05.556433\nRecent studies of angiosperm diversification have focused on the role of polyploidy as a driver of diversification. However, we know far less about the effects of single changes in chromosome number--dysploidy, which can mediate lineage diversification by affecting recombination rates, linkage, and/or reproductive isolation. Modeling the effects of dysploidy on diversification is mathematically and computationally challenging because many states and parameters are required to track changes in individual chromosomes, especially in lineages that have high variability in chromosome number. Additionally, we expect the processes of diversification and dysploidy to vary across clades, which requires modeling process variation to disentangle the effects of the observed trait (chromosome number) from the effects of unobserved traits on diversification. In this work, we propose a new state-dependent diversification model of chromosome evolution that includes numerous character states and explicitly models heterogeneity in the diversification process. Our model includes parameters that functionally link diversification rates to dysploidy rates and differentiate between anagenetic and cladogenetic changes. We apply this model to Carex (Cyperaceae), a model lineage for understanding dysploidy and diversification, leveraging chromosome number information and the most recent time-calibrated phylogenetic tree for over 700 species and subspecies. We recover distinct modes of chromosomal speciation across Carex. In one mode, dysploidy occurs very frequently and drives faster diversification rates. In the other mode, dysploidy is rare and diversification is driven by other factors, unmeasured in our analysis. This study is the first to demonstrate that dysploidy drives diversification in plants while considering unmeasured factors affecting diversification. Significance StatementEvolutionary genomic rearrangements via the gain and/or loss of chromosomes without losing DNA may cause new species to form, but how this process plays out over a millions-of-years timescale is unclear. We test if there is a detectable effect of chromosome gains/losses on the rate of species formation in a large, diverse group of plants. We demonstrate decisively for the first time that these rearrangements sometimes drive the formation of new species through two evolutionary modes. In one mode, rearrangements happen rapidly and sometimes cause new species to form. In the other mode, rearrangements happen less frequently and are not associated with new species formation; instead, other factors--unmeasured in our analysis--likely drive the formation of new species.\nGC content across insect genomes: phylogenetic patterns, causes and consequences\nAuthors: Kyriacou, R. G.; Mulhair, P. O.; Holland, P. W. H.\nScore: 12.2, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557135\nThe proportions of A:T and G:C nucleotide pairs are often unequal and can vary greatly between animal species and along chromosomes. The causes and consequences of this variation are incompletely understood. The recent release of high-quality genome sequences from the Darwin Tree of Life and other large-scale genome projects provides an opportunity for GC heterogeneity to be compared across a large number of insect species. Here we analyse GC content along chromosomes, and within protein-coding genes and codons, of 150 insect species from four holometabolous orders: Coleoptera, Diptera, Hymenoptera, and Lepidoptera. We find that protein-coding sequences have higher GC content than the genome average, and that Lepidoptera generally have higher GC content than the other three insect orders examined. GC content is higher in small chromosomes in most Lepidoptera species, but this pattern is less consistent in other orders. GC content also increases towards subtelomeric regions within protein-coding genes in Hymenoptera, Coleoptera and, most strikingly, Lepidoptera. Two species of Diptera, Bombylius major and B. discolor, have very atypical genomes with ubiquitous increase in AT content, especially at third codon positions. Despite dramatic AT-biased codon usage, we find no evidence that this has driven divergent protein evolution. We argue that the GC landscape of Lepidoptera, Hymenoptera and Coleoptera genomes is influenced by GC-biased gene conversion, strongest in Lepidoptera, with some outlier taxa affected drastically by counteracting processes.\nHaplotype independence contributes to evolvability in the long-term absence of sex in a mite\nAuthors: Oztoprak, H.; Gao, S.; Guiglielmoni, N.; Brandt, A.; Zheng, Y.; Becker, C.; Becker, K.; Bednarski, V.; Borgschulte, L.; Burak, K. A.; Dion-Cote, A.-M.; Leonov, V.; Opherden, L.; Shimano, S.; Bast, J.\nScore: 12.1, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556471\nSome unique asexual species persist over time and contradict the consensus that sex is a prerequisite for long-term evolutionary survival. How they escape the dead-end fate remains enigmatic. Here, we generated a haplotype-resolved genome assembly based on a single individual and collected genomic data from worldwide populations of the parthenogenetic diploid oribatid mite Platynothrus peltifer to identify signatures of persistence without sex. We found that haplotypes diverge independently since the transition to asexuality at least 20 my ago. Multiple lines of evidence indicate disparate evolutionary trajectories between haplotypic blocks. Our findings imply that such haplotypic independence can lead to non-canonical routes of evolvability, helping some species to adapt, diversify and persist for millions of years in the absence of sex. One-Sentence SummaryFunctionally different chromosome sets in an asexual mite species showcase a survival strategy spanning millions of years.\nA beneficial synonymous substitution in EF-Tu is contingent on genetic background\nAuthors: McGrath, K. M.; Russell, S.; Fer, E.; Garmendia, E.; Hosgel, A.; Baltrus, D.; Kacar, B.\nScore: 10.3, Published: 2023-09-08 DOI: 10.1101/2023.09.06.555949\nSynonymous mutations are changes to DNA sequence that occur within translated genes but which do not affect the protein sequence. Although often referred to as silent mutations, evidence suggests that synonymous mutations can affect gene expression, mRNA stability, and even translation efficiency. A collection of both experimental and bioinformatic data has shown that synonymous mutations can impact cell phenotype, yet less is known about the molecular mechanisms and potential of beneficial or adaptive effects of such changes within evolved populations. Here, we report a beneficial synonymous mutation acquired via experimental evolution in an essential gene variant encoding the translation Elongation Factor protein EF-Tu. We demonstrate that this particular synonymous mutation increases EF-Tu mRNA and protein levels, as well as the polysome abundance on global transcripts. Although presence of the synonymous mutation is clearly causative of such changes, we also demonstrate that fitness benefits are highly contingent on other potentiating mutations present within the genetic background in which the mutation arose. Our results underscore the importance of beneficial synonymous mutations, especially those that affect levels of proteins that are key for cellular processes. ImportanceThis study explores the degree to which synonymous mutations in essential genes can influence adaptation in bacteria. An experimental system whereby an Escherichia coli strain harboring an engineered translation protein Elongation Factor-Tu (EF-Tu) was subjected to laboratory evolution. We find that a synonymous mutation acquired on the gene encoding for EF-Tu is conditionally beneficial for bacterial fitness. Our findings provide insight into the importance of the genetic background when a synonymous substitution is favored by natural selection and how such changes have the potential to impact evolution when critical cellular processes are involved.\nInference of multiple mergers while dating a pathogen phylogeny\nAuthors: Helekal, D.; Koskela, J.; Didelot, X.\nScore: 8.7, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557403\nThe vast majority of pathogen phylogenetic studies do not consider the possibility of multiple merger events being present, where a single node of the tree leads to more than two descendent branches. These events are however likely to occur when studying a relatively small population or if there is high variability in the reproductive chances. Here we consider the problem of detecting the presence of multiple mergers in the context of dating a phylogeny, that is determining the date of each of the nodes. We use the Lambda-coalescent theory as a modelling framework and show how Bayesian inference can be efficiently performed using a Billera-Holmes-Vogtmann space embedding and a customised Markov Chain Monte Carlo sampling scheme. We applied this new analysis methodology to a large number of simulated datasets to show that it is possible to infer if and when multiple merger events occurred, and that the phylogenetic dating is improved as a result of taking this information into account. We also analysed real datasets of Vibrio cholerae and Mycobacterium tuberculosis to demonstrate the relevance of our approach to real pathogen evolutionary epidemiology. We have implemented our new methodology in a R package which is freely available at https://github.com/dhelekal/MMCTime.\nLytic/Lysogenic Transition as a Life-History Switch\nAuthors: Roughgarden, J.\nScore: 18.1, Published: 2023-09-10 DOI: 10.1101/2023.01.03.522622\nThe transition from a lytic to lysogenic life cycle is the most important feature of the life history of temperate viruses. An optimal life-history explanation for this transition is offered based a discrete-time model of phage/bacteria population dynamics featuring infection of bacteria by Poisson sampling of virions from the environment. The time step is the viral latency period. In this model density-dependent viral absorption onto the bacterial surface is sufficient to produce virus/bacteria coexistence and density-dependence in bacterial growth is not needed. The formula for the transition from a lytic to lysogenic phase is termed the \"fitness switch\". The virus switches from lytic to lysogenic when its population grows faster as prophage than as virions produced by lysis of the infected cells. A prophage that benefits the bacterium it infects automatically incurs lower fitness upon leaving the bacterial genome, resulting in its becoming locked into the bacterial genome in what is termed as a \"prophage lock\". The fitness switch qualitatively predicts that environmental enrichment leads to microbialization, fluctuating environmental conditions promote horizontal gene transfer (HGT), and prophage-containing bacteria can integrate into the microbiome of a eukaryotic host forming a functionally integrated tripartite holobiont. These predictions accord more with the \"Piggyback-the-Winner\" hypothesis than with the \"Kill-the-Winner\" hypothesis in virus ecology.\nUnravelling species demographic responses to local and global past ecological disruptions using whole-genome sequencing in volcanic oceanic island organisms\nAuthors: Gabrielli, M.; Leroy, T.; Salmona, J.; Nabholz, B.; Mila, B.; Thebaud, C.\nScore: 7.4, Published: 2023-09-12 DOI: 10.1101/2023.09.12.555930\nDisentangling the effects of ecological disruptions operating at different spatial and temporal scales in shaping past species demography is particularly important in the current context of rapid environmental changes driven by both local and global factors. We argue that volcanic oceanic islands provide useful settings to study the influence of past ecological disruptions operating at local and global scales on population demographic histories. We investigate potential drivers of past population dynamics for three closely related species of passerine birds from two volcanic oceanic islands, Reunion and Mauritius (Mascarene archipelago), with distinct volcanic history. Using ABC and PSMC inferences from complete genomes, we reconstructed the demographic history of the Reunion grey white-eye (Zosterops borbonicus), the Reunion olive white-eye (Z. olivaceus), and the Mauritius grey white-eye (Z. mauritianus), and searched for possible causes underlying similarities or differences between species living on the same or different islands. Both demographic inferences strongly support ancient and long-term expansions in all species. They also reveal different trajectories between species inhabiting different islands, but consistent demographic trajectories in species or populations from the same island. Species from Reunion appear to have experienced synchronous reductions in population size during the Last Glacial Maximum, a trend not seen in the Mauritian species. Overall, this study suggests that local events have played an important role in shaping population trajectories of these island species. It also highlights the potential of our conceptual framework to disentangle the effects of local and global drivers on past species demography and long-term population processes.\n",
  "wordCount" : "2605",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557279">
        <p class="paperTitle">Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, A. L.; Starr, T. N.</p>
        <p class="info">Score: 81.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557279' target='https://doi.org/10.1101/2023.09.11.557279'> 10.1101/2023.09.11.557279</a></p>
        <p class="abstract">Substitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear. We conducted deep mutational scans to measure the impacts of all single amino acid mutations and single-codon deletions in the spike receptor-binding domain (RBD) on ACE2-binding affinity and protein expression in the recent Omicron BQ.1.1 and XBB.1.5 variants, and we compared mutational patterns to earlier viral strains that we have previously profiled. As with previous RBD deep mutational scans, we find many mutations that are tolerated or even enhance binding to ACE2 receptor. The tolerance of sites to single-codon deletion largely conforms with tolerance to amino acid mutation. Though deletions in the RBD have not yet been seen in dominant lineages, we observe many tolerated deletions including at positions that exhibit indel variation across broader sarbecovirus evolution and in emerging SARS-CoV-2 variants of interest, most notably the well-tolerated {Delta}483 deletion in BA.2.86. The substitutions that distinguish recent viral variants have not induced as dramatic of epistatic perturbations as N501Y, but we identify ongoing epistatic drift in SARS-CoV-2 variants, including interaction between R493Q reversions and mutations at positions 453, 455, and 456, including mutations like F456L that define the newly emerging EG.5 lineage. Our results highlight ongoing drift in the effects of mutations due to epistasis, which may continue to direct SARS-CoV-2 evolution into new regions of sequence space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557418">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557418" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557418">
        <p class="paperTitle">Viperin immunity evolved across the tree of life through serial innovations on a conserved scaffold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557418" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557418" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shomar, H.; Georjon, H.; Feng, Y.; Olympio, B.; Tesson, F.; Cury, J.; Wu, F.; Bernheim, A.</p>
        <p class="info">Score: 47.0, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557418' target='https://doi.org/10.1101/2023.09.13.557418'> 10.1101/2023.09.13.557418</a></p>
        <p class="abstract">Evolutionary arms races between cells and viruses drive the rapid diversification of antiviral genes in diverse life forms. Recent discoveries have revealed the existence of shared immune genes between prokaryotes and eukaryotes, showing molecular and mechanistic similarities in their response to viruses. However, the underlying evolutionary dynamics that explain the conservation and adaptation of these antiviral genes remain mostly unexplored. Here, we show that viperins constitute a highly conserved family of immune genes across diverse prokaryotes and eukaryotes, and uncover mechanisms by which they diversified in eukaryotes. Our findings indicate that viperins are enriched in Asgard archaea and widely distributed in all major eukaryotic clades, suggesting their presence in the Last Eukaryotic Common Ancestor (LECA). We show that viperins maintain their immune function by producing antiviral nucleotide analogs. We demonstrate that eukaryotic viperins diversified through serial innovations on the viperin gene, such as the emergence and selection of substrate specificity towards pyrimidine nucleotides, and through partnerships with genes maintained through genetic linkage, notably with nucleotide kinases. These findings unveil biochemical and genomic transitions underlying the adaptation of immune genes shared by prokaryotes and eukaryotes. Our study paves the way for the understanding of the conservation of immunity across domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554244">
        <p class="paperTitle">Conserved switch genes regulate a novel cannibalistic morph after whole genome duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wighard, S.; Sommer, R. J.; Witte, H.</p>
        <p class="info">Score: 37.4, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554244' target='https://doi.org/10.1101/2023.08.22.554244'> 10.1101/2023.08.22.554244</a></p>
        <p class="abstract">Developmental plasticity facilitates morphological and behavioural novelty, but associated regulatory mechanisms remain elusive. Nematodes have emerged as a powerful model to study developmental plasticity and its evolution. Here, we show the predatory nematode Allodiplogaster sudhausi evolved an additional third mouth morph, concomitant with whole genome duplication (WGD) and a strong increase in body size. The three mouth morphs are induced by different diets; bacteria, fungi and nematodes. CRISPR experiments indicate that regulation of the third morph involves co-option of a conserved developmental switch gene, which through WGD resulted in two mouth-form regulators. Gene dosage studies revealed a diverged role of these developmental switches, with functional redundancy and quantitative effects in the two mouth-form decisions, respectively. The third morph is cannibalistic and kills kin, whereas the other two morphs do not. Thus, the recent evolution of a new morph relies on pre-existing regulatory mechanisms and adds behavioural and social complexity.

One-Sentence SummaryExperimental genetics in a nematode reveals a key role for developmental plasticity in the evolution of nutritional diversity</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556433">
        <p class="paperTitle">Detecting shifts in the mode of chromosomal speciation across the cosmopolitan plant lineage Carex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tribble, C. M.; Marquez Corro, J. I.; May, M. R.; Hipp, A. L.; Escudero, M.; Zenil-Ferguson, R.</p>
        <p class="info">Score: 24.9, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556433' target='https://doi.org/10.1101/2023.09.05.556433'> 10.1101/2023.09.05.556433</a></p>
        <p class="abstract">Recent studies of angiosperm diversification have focused on the role of polyploidy as a driver of diversification. However, we know far less about the effects of single changes in chromosome number--dysploidy, which can mediate lineage diversification by affecting recombination rates, linkage, and/or reproductive isolation. Modeling the effects of dysploidy on diversification is mathematically and computationally challenging because many states and parameters are required to track changes in individual chromosomes, especially in lineages that have high variability in chromosome number. Additionally, we expect the processes of diversification and dysploidy to vary across clades, which requires modeling process variation to disentangle the effects of the observed trait (chromosome number) from the effects of unobserved traits on diversification. In this work, we propose a new state-dependent diversification model of chromosome evolution that includes numerous character states and explicitly models heterogeneity in the diversification process. Our model includes parameters that functionally link diversification rates to dysploidy rates and differentiate between anagenetic and cladogenetic changes. We apply this model to Carex (Cyperaceae), a model lineage for understanding dysploidy and diversification, leveraging chromosome number information and the most recent time-calibrated phylogenetic tree for over 700 species and subspecies. We recover distinct modes of chromosomal speciation across Carex. In one mode, dysploidy occurs very frequently and drives faster diversification rates. In the other mode, dysploidy is rare and diversification is driven by other factors, unmeasured in our analysis. This study is the first to demonstrate that dysploidy drives diversification in plants while considering unmeasured factors affecting diversification.

Significance StatementEvolutionary genomic rearrangements via the gain and/or loss of chromosomes without losing DNA may cause new species to form, but how this process plays out over a millions-of-years timescale is unclear. We test if there is a detectable effect of chromosome gains/losses on the rate of species formation in a large, diverse group of plants. We demonstrate decisively for the first time that these rearrangements sometimes drive the formation of new species through two evolutionary modes. In one mode, rearrangements happen rapidly and sometimes cause new species to form. In the other mode, rearrangements happen less frequently and are not associated with new species formation; instead, other factors--unmeasured in our analysis--likely drive the formation of new species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557135">
        <p class="paperTitle">GC content across insect genomes: phylogenetic patterns, causes and consequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kyriacou, R. G.; Mulhair, P. O.; Holland, P. W. H.</p>
        <p class="info">Score: 12.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557135' target='https://doi.org/10.1101/2023.09.11.557135'> 10.1101/2023.09.11.557135</a></p>
        <p class="abstract">The proportions of A:T and G:C nucleotide pairs are often unequal and can vary greatly between animal species and along chromosomes. The causes and consequences of this variation are incompletely understood. The recent release of high-quality genome sequences from the Darwin Tree of Life and other large-scale genome projects provides an opportunity for GC heterogeneity to be compared across a large number of insect species. Here we analyse GC content along chromosomes, and within protein-coding genes and codons, of 150 insect species from four holometabolous orders: Coleoptera, Diptera, Hymenoptera, and Lepidoptera. We find that protein-coding sequences have higher GC content than the genome average, and that Lepidoptera generally have higher GC content than the other three insect orders examined. GC content is higher in small chromosomes in most Lepidoptera species, but this pattern is less consistent in other orders. GC content also increases towards subtelomeric regions within protein-coding genes in Hymenoptera, Coleoptera and, most strikingly, Lepidoptera. Two species of Diptera, Bombylius major and B. discolor, have very atypical genomes with ubiquitous increase in AT content, especially at third codon positions. Despite dramatic AT-biased codon usage, we find no evidence that this has driven divergent protein evolution. We argue that the GC landscape of Lepidoptera, Hymenoptera and Coleoptera genomes is influenced by GC-biased gene conversion, strongest in Lepidoptera, with some outlier taxa affected drastically by counteracting processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556471">
        <p class="paperTitle">Haplotype independence contributes to evolvability in the long-term absence of sex in a mite</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oztoprak, H.; Gao, S.; Guiglielmoni, N.; Brandt, A.; Zheng, Y.; Becker, C.; Becker, K.; Bednarski, V.; Borgschulte, L.; Burak, K. A.; Dion-Cote, A.-M.; Leonov, V.; Opherden, L.; Shimano, S.; Bast, J.</p>
        <p class="info">Score: 12.1, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556471' target='https://doi.org/10.1101/2023.09.07.556471'> 10.1101/2023.09.07.556471</a></p>
        <p class="abstract">Some unique asexual species persist over time and contradict the consensus that sex is a prerequisite for long-term evolutionary survival. How they escape the dead-end fate remains enigmatic. Here, we generated a haplotype-resolved genome assembly based on a single individual and collected genomic data from worldwide populations of the parthenogenetic diploid oribatid mite Platynothrus peltifer to identify signatures of persistence without sex. We found that haplotypes diverge independently since the transition to asexuality at least 20 my ago. Multiple lines of evidence indicate disparate evolutionary trajectories between haplotypic blocks. Our findings imply that such haplotypic independence can lead to non-canonical routes of evolvability, helping some species to adapt, diversify and persist for millions of years in the absence of sex.

One-Sentence SummaryFunctionally different chromosome sets in an asexual mite species showcase a survival strategy spanning millions of years.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.555949">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.555949" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.555949">
        <p class="paperTitle">A beneficial synonymous substitution in EF-Tu is contingent on genetic background</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.555949" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.555949" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McGrath, K. M.; Russell, S.; Fer, E.; Garmendia, E.; Hosgel, A.; Baltrus, D.; Kacar, B.</p>
        <p class="info">Score: 10.3, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.555949' target='https://doi.org/10.1101/2023.09.06.555949'> 10.1101/2023.09.06.555949</a></p>
        <p class="abstract">Synonymous mutations are changes to DNA sequence that occur within translated genes but which do not affect the protein sequence. Although often referred to as silent mutations, evidence suggests that synonymous mutations can affect gene expression, mRNA stability, and even translation efficiency. A collection of both experimental and bioinformatic data has shown that synonymous mutations can impact cell phenotype, yet less is known about the molecular mechanisms and potential of beneficial or adaptive effects of such changes within evolved populations. Here, we report a beneficial synonymous mutation acquired via experimental evolution in an essential gene variant encoding the translation Elongation Factor protein EF-Tu. We demonstrate that this particular synonymous mutation increases EF-Tu mRNA and protein levels, as well as the polysome abundance on global transcripts. Although presence of the synonymous mutation is clearly causative of such changes, we also demonstrate that fitness benefits are highly contingent on other potentiating mutations present within the genetic background in which the mutation arose. Our results underscore the importance of beneficial synonymous mutations, especially those that affect levels of proteins that are key for cellular processes.

ImportanceThis study explores the degree to which synonymous mutations in essential genes can influence adaptation in bacteria. An experimental system whereby an Escherichia coli strain harboring an engineered translation protein Elongation Factor-Tu (EF-Tu) was subjected to laboratory evolution. We find that a synonymous mutation acquired on the gene encoding for EF-Tu is conditionally beneficial for bacterial fitness. Our findings provide insight into the importance of the genetic background when a synonymous substitution is favored by natural selection and how such changes have the potential to impact evolution when critical cellular processes are involved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557403">
        <p class="paperTitle">Inference of multiple mergers while dating a pathogen phylogeny</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Helekal, D.; Koskela, J.; Didelot, X.</p>
        <p class="info">Score: 8.7, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557403' target='https://doi.org/10.1101/2023.09.12.557403'> 10.1101/2023.09.12.557403</a></p>
        <p class="abstract">The vast majority of pathogen phylogenetic studies do not consider the possibility of multiple merger events being present, where a single node of the tree leads to more than two descendent branches. These events are however likely to occur when studying a relatively small population or if there is high variability in the reproductive chances. Here we consider the problem of detecting the presence of multiple mergers in the context of dating a phylogeny, that is determining the date of each of the nodes. We use the Lambda-coalescent theory as a modelling framework and show how Bayesian inference can be efficiently performed using a Billera-Holmes-Vogtmann space embedding and a customised Markov Chain Monte Carlo sampling scheme. We applied this new analysis methodology to a large number of simulated datasets to show that it is possible to infer if and when multiple merger events occurred, and that the phylogenetic dating is improved as a result of taking this information into account. We also analysed real datasets of Vibrio cholerae and Mycobacterium tuberculosis to demonstrate the relevance of our approach to real pathogen evolutionary epidemiology. We have implemented our new methodology in a R package which is freely available at https://github.com/dhelekal/MMCTime.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.03.522622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.03.522622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.03.522622">
        <p class="paperTitle">Lytic/Lysogenic Transition as a Life-History Switch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.03.522622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.03.522622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roughgarden, J.</p>
        <p class="info">Score: 18.1, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.03.522622' target='https://doi.org/10.1101/2023.01.03.522622'> 10.1101/2023.01.03.522622</a></p>
        <p class="abstract">The transition from a lytic to lysogenic life cycle is the most important feature of the life history of temperate viruses. An optimal life-history explanation for this transition is offered based a discrete-time model of phage/bacteria population dynamics featuring infection of bacteria by Poisson sampling of virions from the environment. The time step is the viral latency period. In this model density-dependent viral absorption onto the bacterial surface is sufficient to produce virus/bacteria coexistence and density-dependence in bacterial growth is not needed.

The formula for the transition from a lytic to lysogenic phase is termed the &#34;fitness switch&#34;. The virus switches from lytic to lysogenic when its population grows faster as prophage than as virions produced by lysis of the infected cells.

A prophage that benefits the bacterium it infects automatically incurs lower fitness upon leaving the bacterial genome, resulting in its becoming locked into the bacterial genome in what is termed as a &#34;prophage lock&#34;.

The fitness switch qualitatively predicts that environmental enrichment leads to microbialization, fluctuating environmental conditions promote horizontal gene transfer (HGT), and prophage-containing bacteria can integrate into the microbiome of a eukaryotic host forming a functionally integrated tripartite holobiont. These predictions accord more with the &#34;Piggyback-the-Winner&#34; hypothesis than with the &#34;Kill-the-Winner&#34; hypothesis in virus ecology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.555930">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.555930" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.555930">
        <p class="paperTitle">Unravelling species demographic responses to local and global past ecological disruptions using whole-genome sequencing in volcanic oceanic island organisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.555930" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.555930" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabrielli, M.; Leroy, T.; Salmona, J.; Nabholz, B.; Mila, B.; Thebaud, C.</p>
        <p class="info">Score: 7.4, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.555930' target='https://doi.org/10.1101/2023.09.12.555930'> 10.1101/2023.09.12.555930</a></p>
        <p class="abstract">Disentangling the effects of ecological disruptions operating at different spatial and temporal scales in shaping past species demography is particularly important in the current context of rapid environmental changes driven by both local and global factors. We argue that volcanic oceanic islands provide useful settings to study the influence of past ecological disruptions operating at local and global scales on population demographic histories. We investigate potential drivers of past population dynamics for three closely related species of passerine birds from two volcanic oceanic islands, Reunion and Mauritius (Mascarene archipelago), with distinct volcanic history. Using ABC and PSMC inferences from complete genomes, we reconstructed the demographic history of the Reunion grey white-eye (Zosterops borbonicus), the Reunion olive white-eye (Z. olivaceus), and the Mauritius grey white-eye (Z. mauritianus), and searched for possible causes underlying similarities or differences between species living on the same or different islands. Both demographic inferences strongly support ancient and long-term expansions in all species. They also reveal different trajectories between species inhabiting different islands, but consistent demographic trajectories in species or populations from the same island. Species from Reunion appear to have experienced synchronous reductions in population size during the Last Glacial Maximum, a trend not seen in the Mauritian species. Overall, this study suggests that local events have played an important role in shaping population trajectories of these island species. It also highlights the potential of our conceptual framework to disentangle the effects of local and global drivers on past species demography and long-term population processes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
