<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Origin Of Evolutionary Bifurcation In An Enzyme
Authors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.
Score: 17.4, Published: 2023-11-26 DOI: 10.1101/2023.11.25.568631
Evolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Origin Of Evolutionary Bifurcation In An Enzyme
Authors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.
Score: 17.4, Published: 2023-11-26 DOI: 10.1101/2023.11.25.568631
Evolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Origin Of Evolutionary Bifurcation In An Enzyme
Authors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.
Score: 17.4, Published: 2023-11-26 DOI: 10.1101/2023.11.25.568631
Evolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Origin Of Evolutionary Bifurcation In An Enzyme\nAuthors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.\nScore: 17.4, Published: 2023-11-26 DOI: 10.1101/2023.11.25.568631\nEvolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life.",
  "keywords": [
    
  ],
  "articleBody": " Origin Of Evolutionary Bifurcation In An Enzyme\nAuthors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.\nScore: 17.4, Published: 2023-11-26 DOI: 10.1101/2023.11.25.568631\nEvolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life. Here, we present a detailed molecular description of a bifurcation event that rapidly led to the emergence of two distinct enzymes from a common ancestor. When initiated from two starting points that differed by a single amino acid, the laboratory evolution of a phosphotriesterase (PTE) toward arylester hydrolysis resulted in different genetic and phenotypic outcomes. One trajectory led to a \u003e35,000-fold increase in activity via the reorganization of its active site to achieve exquisite enzyme-substrate complementarity. The second trajectory gave rise to an evolved variant with a [~]500-fold increase in activity, but exhibiting an alternative substrate binding mode resulting from the destabilization of an active site loop. While initial mutations tend to dictate mutational accessibility, we rather observed the gradual divergence and specialisation of each trajectory, following the emergence of distinct molecular interaction networks. Intramolecular epistasis underlay pathway bifurcation by promoting unique synergistic interactions within each trajectory, while restricting the fixation of mutation across pathways. Our results illustrate how distinct molecular outcomes can radiate from a common protein ancestor and give rise to phenotypic diversity.\nEpigenetic predictors of species maximum lifespan and other life history traits in mammals\nAuthors: Li, C. Z.; Haghani, A.; Yan, Q.; Lu, A. T.; Zhang, J.; Fei, Z.; Ernst, J.; Yang, X.; Gladyshev, V. N.; Raj, K.; Seluanov, A.; Gorbunova, V.; Horvath, S.\nScore: 26.8, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565286\nMaximum lifespan is an intrinsic characteristic of a biological species and is defined as the longest time an individual of a species has been reported to survive. By analyzing 15K samples derived from 348 mammalian species representing 25 taxonomic orders we previously identified CpG methylation sites associated with maximum lifespan. Here we present accurate DNA methylation-based (DNAm) predictors of maximum lifespan (r=0.89), average gestation time (r=0.96), and age at sexual maturity (r=0.85). Our DNAm maximum lifespan predictor indicates a potential innate longevity advantage for females over males in 17 mammalian species such as humans, red deer, and cattle. The DNAm maximum lifespan predictions do not vary significantly by caloric restriction and partial reprogramming. Genetic disruptions in the somatotropic axis, which includes growth hormone, IGF-1, and their related receptors, have an impact on DNAm maximum lifespan only in select mouse tissues. Cancer mortality rates in major mammalian orders show no correlation with our epigenetic estimates of life history traits. The DNAm maximum lifespan predictor does not detect variation in lifespan between individuals of the same species, such as between the breeds of dogs. We also present the first prototypes of accurate pan mammalian DNAm predictors of sex and tissue type. Collectively, our findings indicate that maximum lifespan is determined, at least in part, by an epigenetic signature that is an intrinsic property of each species and is distinct from the signatures that relate to individual lifespan, which is unaffected by interventions influencing the mortality risk of individuals.\nSelective sweep probabilities in spatially expanding populations\nAuthors: Stein, A.; Kizhuttil, R.; Bak, M.; Noble, R. J.\nScore: 13.0, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568915\nEvolution during range expansions is an important feature of many biological systems including tumours, microbial communities, and invasive species. A selective sweep is a fundamental process, in which an advantageous mutation evades clonal interference and spreads through the population to fixation. However, most theoretical investigations of selective sweeps have assumed constant population size or have ignored spatial structure. Here we use mathematical modelling and analysis to investigate selective sweep probabilities in populations that grow with constant radial expansion speed. We derive probability distributions for the arrival time and location of the first surviving mutant and hence find simple approximate and exact expressions for selective sweep probabilities in one, two and three dimensions, which are independent of mutation rate. Namely, the selective sweep probability is approximately (1-cwt/cm)d, where cwt and cm are the wildtype and mutant radial expansion speeds, and d is the spatial dimension. Using agent-based simulations, we show that our analytical results accurately predict selective sweep frequencies in the two-dimensional spatial Moran process. We further compare our results with those obtained for alternative growth laws. Parameterizing our model for human tumours, we find that selective sweeps are predicted to be rare except during very early solid tumour growth, thus providing a general, pan-cancer explanation for findings from recent sequencing studies.\nHot springs viruses at Yellowstone National Park have ancient origins and are adapted to their thermophilic hosts.\nAuthors: Benites, L. F.; Stephens, T. G.; Van Etten, J.; James, T.; Christian, W. C.; Barry, K.; Grigoriev, I. V.; McDermott, T. R.; Bhattacharya, D.\nScore: 17.8, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567249\nGeothermal springs in areas such as Yellowstone National Park (YNP), USA house unicellular red algae that dominate the microbial biomass. Little is known about the viruses that infect the biota. Here we used metagenomics to characterize the multi-kingdom infecting virus community associated with red algal mats in three neighboring habitats (creek, endolithic, soil) at Lemonade Creek, YNP to determine their taxonomic composition, predicted gene functions, extent of horizontal gene transfer, and potential links to hosts. We find that despite proximity, each habitat houses a unique collection of viruses, with the giant viruses, Megaviricetes, dominant in all three. The early branching phylogenetic position of genes encoded on metagenome assembled virus genomes (vMAGs) suggests that the YNP lineages are of ancient origins and not the result of multiple invasions from mesophilic habitats. The existence of genomic footprints of adaptation to thermophily in the vMAGs is consistent with this idea. Our study is the first analysis of viruses associated with polyextremophilic red algae that form extensive microbial mats in YNP and are common worldwide at geothermal sites. Although more recent than the earliest forms of prokaryotic life, these eukaryotic algal mats originated ca. 1.5 Bya, and therefore span a significant period of the planets history. Our data are therefore relevant to understanding biotic interactions on the early Earth.\nGene expression shifts in Emperor penguin adaptation to the extreme Antarctic environment\nAuthors: Paris, J. R.; Fernandes, F. A. N.; Pirri, F.; Greco, S.; Gerdol, M.; Pallavicini, A.; Benoiste, M.; Cornec, C.; Zane, L.; Haas, B.; Le Bohec, C.; Trucchi, E.\nScore: 10.4, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569211\nGene expression can accelerate ecological divergence by rapidly tweaking the response of an organism to novel environments, with more divergent environments exerting stronger selection and, supposedly, requiring faster adaptive responses. Organisms adapted to extreme environments provide ideal systems to test this hypothesis, particularly when compared to related species with milder ecological niches. The Emperor penguin (Aptenodytes forsteri) is the only warm-blooded vertebrate breeding in the harsh Antarctic winter, in stark contrast with the less cold-adapted sister species, the King penguin (A. patagonicus). Assembling the first de novo transcriptomes and analysing multi-tissue (brain, kidney, liver, muscle, skin) RNAseq data from natural populations of both species, we quantified the shifts in tissue-enhanced genes, co-expression gene networks, and differentially expressed genes characterising Emperor penguin adaptation to extreme Antarctic ecology. Our analyses revealed the crucial role played by muscle and liver in temperature homeostasis, fasting and whole-body energy metabolism (glucose/insulin regulation, lipid metabolism, fatty acid beta-oxidation, and blood coagulation). Repatterning at the regulatory level appears as more important in the brain of the Emperor penguin, showing the lowest signature of differential gene expression but the largest co-expression gene network shift. Nevertheless, over-expressed genes related to mTOR signalling in the brain and the liver support their central role in cold and fasting responses. Besides contributing to understanding the genetics underlying complex traits, like body energy reservoir management, our results provide a first insight into the role of gene expression in adaptation to one of the most extreme environmental conditions endured by an endotherm.\nMatreex: compact and interactive visualisation for scalable studies of large gene families\nAuthors: Rossier, V.; Train, C.; Nevers, Y.; Robinson-Rechavi, M.; Dessimoz, C.\nScore: 35.0, Published: 2023-11-15 DOI: 10.1101/2023.02.18.529053\nStudying gene family evolution strongly benefits from insightful visualisations. However, the evergrowing number of sequenced genomes is leading to increasingly larger gene families, which challenges existing gene tree visualisations. Indeed, most of them present users with a dilemma: display complete but intractable gene trees, or collapse subtrees, thereby hiding their childrens information. Here, we introduce Matreex, a new dynamic tool to scale-up the visualisation of gene families. Matreexs key idea is to use \"phylogenetic\" profiles, which are dense representations of gene repertoires, to minimise the information loss when collapsing subtrees. We illustrate Matreex usefulness with three biological applications. First, we demonstrate on the MutS family the power of combining gene trees and phylogenetic profiles to delve into precise evolutionary analyses of large multi-copy gene families. Secondly, by displaying 22 intraflagellar transport gene families across 622 species cumulating 5500 representatives, we show how Matreex can be used to automate large-scale analyses of gene presence-absence. Notably, we report for the first time the complete loss of intraflagellar transport in the myxozoan Thelohanellus kitauei. Finally, using the textbook example of visual opsins, we show Matreexs potential to create easily interpretable figures for teaching and outreach. Matreex is available from the Python Package Index (pip install matreex) with the source code and documentation available at https://github.com/DessimozLab/matreex.\nWhole-genome sequencing of major malaria vectors reveals the evolution of new insecticide resistance variants in a longitudinal study in Burkina Faso\nAuthors: Kientega, M.; Clarkson, C. S.; Traore, N.; Hui, T.-Y. J.; O'Loughlin, S. M.; Millogo, A.-A.; Yao, F. A.; Epopa, P. S.; Belem, A. M. G.; Jon, B. H.; Miles, A.; Burt, A.; Diabate, A.\nScore: 10.2, Published: 2023-11-25 DOI: 10.1101/2023.11.20.567800\nIntensive deployment of insecticide based malaria vector control tools results in the rapid evolution of phenotypes resistant to these chemicals. Understanding this process on the genomic level is essential for the deployment of successful interventions. Using whole genome sequencing data of 1409 individual An. Gambiae s.l. collected from 2012 to 2017, we investigated the change in genetic structure and the evolution of the insecticide resistance variants in natural populations over time and space. The results showed similar and constant nucleotide diversity and negative Tajimas D between An. gambiae s.s. and An. coluzzii. PCA and FST showed a clear genetic structure in the An. gambiae s.l. species. Genome-wide FST and H12 scans identified genomic regions under divergent selection and also having an implication in the adaptations to ecological changes. Novel voltage-gated sodium channel pyrethroid resistance target-site alleles (V402L, I1527T) were identified at increasing frequencies alongside the established kdr alleles (Vgsc-L995F, Vgsc-L995S and N1570Y) within the An. gambiae s.l. populations. Organophosphate metabolic resistance markers were also identified, at increasing frequencies, within the An. gambiae s.s. populations from 2012 to 2017, including the SNP Ace1-G280S and its associated duplication. Variants simultaneously identified in the same vector populations raise concerns about the long-term efficacy of new-generation bednets and the recently introduced organophosphate pirimiphos-methyl indoor residual spraying. These findings highlighted the benefit of genomic malaria vector surveillance for the detection of new insecticide resistance variants, the monitoring of the existing resistance variants, and also to get insights into the evolutionary processes driving insecticide resistance. Author SummaryGenomic surveillance of malaria vectors is crucial for understanding the genetic variation in natural vector populations and also guiding the implementation of novel and innovative vector control tools. Application of sequencing technologies in vector studies provide insights on the genetic and evolutionary phenomena of vectors that could have impact on vector control strategies. By analyzing the whole genome data of 1409 wild An. gambiae s.l. mosquito collected between 2012 and 2017, we showed an emergence of novel insecticide resistance markers alongside increasing frequencies of existing insecticide resistance variants over time in Burkina Faso. We showed the benefit of genomic surveillance of malaria vectors for the monitoring of the insecticide resistance variants and also providing insights into the evolutionary processes driving insecticide resistance.\nGenetic architecture of inbreeding depression may explain its persistence in a population of wild red deer\nAuthors: Hewett, A.; Pemberton, J. M.; Johnston, S. E.\nScore: 8.3, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568439\nInbreeding depression is of major concern in declining populations, but relatively little is known about its genetic architecture in wild populations, such as the degree to which it is composed of large or small effect loci and their distribution throughout the genome. We combine fitness and genomic data from a wild population of red deer to investigate the genomic distribution of inbreeding effects. Inspired by the runs of homozygosity (ROH)-based inbreeding coefficient, FROH, we use chromosome-specific inbreeding coefficients (FROHChr) to explore whether the effect of inbreeding varies between chromosomes. Under the assumption that within an individual the probability of being identical-by-descent is equal across all chromosomes, we used a multi-membership model to estimate the deviation of FROHChr from the average inbreeding effect. This novel approach ensures effect sizes are not overestimated whilst maximising the power of our available dataset containing \u003e35,000 autosomal SNPs. We find that most chromosomes confer a minor reduction in fitness-related traits, which when these effects are summed, results in the observed inbreeding depression in birth weight, survival and lifetime breeding success. However, no chromosomes had a significantly detrimental effect compared to the overall effect of inbreeding. We conclude that in this population, inbreeding depression is the result of multiple mild or moderately deleterious mutations spread across all chromosomes. As predicted by genetic theory these mutations will be inefficiently purged, explaining the persistence of inbreeding depression in this population.\nEvolution of hybrid inviability associated with chromosome fusions\nAuthors: Boman, J.; Näsvall, K.; Vila, R.; Wiklund, C.; Backström, N.\nScore: 6.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569355\nChromosomal rearrangements, such as inversions, have received considerable attention in the speciation literature due to their hampering effects on recombination. However, less is known about how other rearrangements, such as chromosome fissions and fusions, can affect the evolution of reproductive isolation. Here, we used crosses between populations of the wood white butterfly (Leptidea sinapis) with different karyotypes to identify genomic regions associated with hybrid inviability. By contrasting allele frequencies between F2 hybrids that survived until adulthood with individuals of the same cohort that succumbed to hybrid incompatibilities, we show that candidate loci for hybrid inviability mainly are situated in fast-evolving regions with reduced recombination rates, especially in regions where chromosome fusions have occurred. Our results show that the extensive variation in chromosome numbers observed across the tree of life can be involved in speciation by being hotspots for the early evolution of postzygotic reproductive isolation.\nPatterns of physical activity in hunter-gatherer children compared with US and UK children.\nAuthors: Kretschmer, L.; Dyble, M.; Chaudhary, N.; Bann, D.; Salali, G. D.\nScore: 6.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569171\nSedentary lifestyles, mismatched with our active foraging history, contribute to escalating rates of non-communicable diseases. Contemporary hunter-gatherers appear to be highly active, but little is known about physical activity levels in hunter-gatherer children. We analysed 150 days of accelerometer data from 51 BaYaka hunter-gatherer children (aged 3-18) in the Republic of Congo, comparing it with British and American children (MCS and NHANES). BaYaka children were highly active, engaging in over 3 hours of moderate-to-vigorous physical activity (MVPA) daily, surpassing British adolescents by over 70 minutes. In US children activity declined with age; while in BaYaka children activity increased with age, unaffected by gender. Reflecting their foraging lifestyle, activity patterns varied within and between days, yet all children consistently rose with the sun. These findings highlight the impact of a foraging upbringing on children's activity levels, providing a benchmark for understanding childhood physical activity and wellbeing.\n",
  "wordCount" : "2622",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568631">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568631" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568631">
        <p class="paperTitle">Origin Of Evolutionary Bifurcation In An Enzyme</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568631" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568631" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.</p>
        <p class="info">Score: 17.4, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568631' target='https://doi.org/10.1101/2023.11.25.568631'> 10.1101/2023.11.25.568631</a></p>
        <p class="abstract">Evolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life. Here, we present a detailed molecular description of a bifurcation event that rapidly led to the emergence of two distinct enzymes from a common ancestor. When initiated from two starting points that differed by a single amino acid, the laboratory evolution of a phosphotriesterase (PTE) toward arylester hydrolysis resulted in different genetic and phenotypic outcomes. One trajectory led to a &gt;35,000-fold increase in activity via the reorganization of its active site to achieve exquisite enzyme-substrate complementarity. The second trajectory gave rise to an evolved variant with a [~]500-fold increase in activity, but exhibiting an alternative substrate binding mode resulting from the destabilization of an active site loop. While initial mutations tend to dictate mutational accessibility, we rather observed the gradual divergence and specialisation of each trajectory, following the emergence of distinct molecular interaction networks. Intramolecular epistasis underlay pathway bifurcation by promoting unique synergistic interactions within each trajectory, while restricting the fixation of mutation across pathways. Our results illustrate how distinct molecular outcomes can radiate from a common protein ancestor and give rise to phenotypic diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565286">
        <p class="paperTitle">Epigenetic predictors of species maximum lifespan and other life history traits in mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, C. Z.; Haghani, A.; Yan, Q.; Lu, A. T.; Zhang, J.; Fei, Z.; Ernst, J.; Yang, X.; Gladyshev, V. N.; Raj, K.; Seluanov, A.; Gorbunova, V.; Horvath, S.</p>
        <p class="info">Score: 26.8, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565286' target='https://doi.org/10.1101/2023.11.02.565286'> 10.1101/2023.11.02.565286</a></p>
        <p class="abstract">Maximum lifespan is an intrinsic characteristic of a biological species and is defined as the longest time an individual of a species has been reported to survive. By analyzing 15K samples derived from 348 mammalian species representing 25 taxonomic orders we previously identified CpG methylation sites associated with maximum lifespan. Here we present accurate DNA methylation-based (DNAm) predictors of maximum lifespan (r=0.89), average gestation time (r=0.96), and age at sexual maturity (r=0.85). Our DNAm maximum lifespan predictor indicates a potential innate longevity advantage for females over males in 17 mammalian species such as humans, red deer, and cattle. The DNAm maximum lifespan predictions do not vary significantly by caloric restriction and partial reprogramming. Genetic disruptions in the somatotropic axis, which includes growth hormone, IGF-1, and their related receptors, have an impact on DNAm maximum lifespan only in select mouse tissues. Cancer mortality rates in major mammalian orders show no correlation with our epigenetic estimates of life history traits. The DNAm maximum lifespan predictor does not detect variation in lifespan between individuals of the same species, such as between the breeds of dogs. We also present the first prototypes of accurate pan mammalian DNAm predictors of sex and tissue type.

Collectively, our findings indicate that maximum lifespan is determined, at least in part, by an epigenetic signature that is an intrinsic property of each species and is distinct from the signatures that relate to individual lifespan, which is unaffected by interventions influencing the mortality risk of individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568915">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568915" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568915">
        <p class="paperTitle">Selective sweep probabilities in spatially expanding populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568915" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568915" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein, A.; Kizhuttil, R.; Bak, M.; Noble, R. J.</p>
        <p class="info">Score: 13.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568915' target='https://doi.org/10.1101/2023.11.27.568915'> 10.1101/2023.11.27.568915</a></p>
        <p class="abstract">Evolution during range expansions is an important feature of many biological systems including tumours, microbial communities, and invasive species. A selective sweep is a fundamental process, in which an advantageous mutation evades clonal interference and spreads through the population to fixation. However, most theoretical investigations of selective sweeps have assumed constant population size or have ignored spatial structure. Here we use mathematical modelling and analysis to investigate selective sweep probabilities in populations that grow with constant radial expansion speed. We derive probability distributions for the arrival time and location of the first surviving mutant and hence find simple approximate and exact expressions for selective sweep probabilities in one, two and three dimensions, which are independent of mutation rate. Namely, the selective sweep probability is approximately (1-cwt/cm)d, where cwt and cm are the wildtype and mutant radial expansion speeds, and d is the spatial dimension. Using agent-based simulations, we show that our analytical results accurately predict selective sweep frequencies in the two-dimensional spatial Moran process. We further compare our results with those obtained for alternative growth laws. Parameterizing our model for human tumours, we find that selective sweeps are predicted to be rare except during very early solid tumour growth, thus providing a general, pan-cancer explanation for findings from recent sequencing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567249">
        <p class="paperTitle">Hot springs viruses at Yellowstone National Park have ancient origins and are adapted to their thermophilic hosts.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benites, L. F.; Stephens, T. G.; Van Etten, J.; James, T.; Christian, W. C.; Barry, K.; Grigoriev, I. V.; McDermott, T. R.; Bhattacharya, D.</p>
        <p class="info">Score: 17.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567249' target='https://doi.org/10.1101/2023.11.15.567249'> 10.1101/2023.11.15.567249</a></p>
        <p class="abstract">Geothermal springs in areas such as Yellowstone National Park (YNP), USA house unicellular red algae that dominate the microbial biomass. Little is known about the viruses that infect the biota. Here we used metagenomics to characterize the multi-kingdom infecting virus community associated with red algal mats in three neighboring habitats (creek, endolithic, soil) at Lemonade Creek, YNP to determine their taxonomic composition, predicted gene functions, extent of horizontal gene transfer, and potential links to hosts. We find that despite proximity, each habitat houses a unique collection of viruses, with the giant viruses, Megaviricetes, dominant in all three. The early branching phylogenetic position of genes encoded on metagenome assembled virus genomes (vMAGs) suggests that the YNP lineages are of ancient origins and not the result of multiple invasions from mesophilic habitats. The existence of genomic footprints of adaptation to thermophily in the vMAGs is consistent with this idea. Our study is the first analysis of viruses associated with polyextremophilic red algae that form extensive microbial mats in YNP and are common worldwide at geothermal sites. Although more recent than the earliest forms of prokaryotic life, these eukaryotic algal mats originated ca. 1.5 Bya, and therefore span a significant period of the planets history. Our data are therefore relevant to understanding biotic interactions on the early Earth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569211">
        <p class="paperTitle">Gene expression shifts in Emperor penguin adaptation to the extreme Antarctic environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paris, J. R.; Fernandes, F. A. N.; Pirri, F.; Greco, S.; Gerdol, M.; Pallavicini, A.; Benoiste, M.; Cornec, C.; Zane, L.; Haas, B.; Le Bohec, C.; Trucchi, E.</p>
        <p class="info">Score: 10.4, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569211' target='https://doi.org/10.1101/2023.11.29.569211'> 10.1101/2023.11.29.569211</a></p>
        <p class="abstract">Gene expression can accelerate ecological divergence by rapidly tweaking the response of an organism to novel environments, with more divergent environments exerting stronger selection and, supposedly, requiring faster adaptive responses. Organisms adapted to extreme environments provide ideal systems to test this hypothesis, particularly when compared to related species with milder ecological niches. The Emperor penguin (Aptenodytes forsteri) is the only warm-blooded vertebrate breeding in the harsh Antarctic winter, in stark contrast with the less cold-adapted sister species, the King penguin (A. patagonicus). Assembling the first de novo transcriptomes and analysing multi-tissue (brain, kidney, liver, muscle, skin) RNAseq data from natural populations of both species, we quantified the shifts in tissue-enhanced genes, co-expression gene networks, and differentially expressed genes characterising Emperor penguin adaptation to extreme Antarctic ecology. Our analyses revealed the crucial role played by muscle and liver in temperature homeostasis, fasting and whole-body energy metabolism (glucose/insulin regulation, lipid metabolism, fatty acid beta-oxidation, and blood coagulation). Repatterning at the regulatory level appears as more important in the brain of the Emperor penguin, showing the lowest signature of differential gene expression but the largest co-expression gene network shift. Nevertheless, over-expressed genes related to mTOR signalling in the brain and the liver support their central role in cold and fasting responses. Besides contributing to understanding the genetics underlying complex traits, like body energy reservoir management, our results provide a first insight into the role of gene expression in adaptation to one of the most extreme environmental conditions endured by an endotherm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.18.529053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.18.529053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.18.529053">
        <p class="paperTitle">Matreex: compact and interactive visualisation for scalable studies of large gene families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.18.529053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.18.529053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossier, V.; Train, C.; Nevers, Y.; Robinson-Rechavi, M.; Dessimoz, C.</p>
        <p class="info">Score: 35.0, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.18.529053' target='https://doi.org/10.1101/2023.02.18.529053'> 10.1101/2023.02.18.529053</a></p>
        <p class="abstract">Studying gene family evolution strongly benefits from insightful visualisations. However, the evergrowing number of sequenced genomes is leading to increasingly larger gene families, which challenges existing gene tree visualisations. Indeed, most of them present users with a dilemma: display complete but intractable gene trees, or collapse subtrees, thereby hiding their childrens information. Here, we introduce Matreex, a new dynamic tool to scale-up the visualisation of gene families. Matreexs key idea is to use &#34;phylogenetic&#34; profiles, which are dense representations of gene repertoires, to minimise the information loss when collapsing subtrees. We illustrate Matreex usefulness with three biological applications. First, we demonstrate on the MutS family the power of combining gene trees and phylogenetic profiles to delve into precise evolutionary analyses of large multi-copy gene families. Secondly, by displaying 22 intraflagellar transport gene families across 622 species cumulating 5500 representatives, we show how Matreex can be used to automate large-scale analyses of gene presence-absence. Notably, we report for the first time the complete loss of intraflagellar transport in the myxozoan Thelohanellus kitauei. Finally, using the textbook example of visual opsins, we show Matreexs potential to create easily interpretable figures for teaching and outreach. Matreex is available from the Python Package Index (pip install matreex) with the source code and documentation available at https://github.com/DessimozLab/matreex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567800">
        <p class="paperTitle">Whole-genome sequencing of major malaria vectors reveals the evolution of new insecticide resistance variants in a longitudinal study in Burkina Faso</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kientega, M.; Clarkson, C. S.; Traore, N.; Hui, T.-Y. J.; O&#39;Loughlin, S. M.; Millogo, A.-A.; Yao, F. A.; Epopa, P. S.; Belem, A. M. G.; Jon, B. H.; Miles, A.; Burt, A.; Diabate, A.</p>
        <p class="info">Score: 10.2, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567800' target='https://doi.org/10.1101/2023.11.20.567800'> 10.1101/2023.11.20.567800</a></p>
        <p class="abstract">Intensive deployment of insecticide based malaria vector control tools results in the rapid evolution of phenotypes resistant to these chemicals. Understanding this process on the genomic level is essential for the deployment of successful interventions. Using whole genome sequencing data of 1409 individual An. Gambiae s.l. collected from 2012 to 2017, we investigated the change in genetic structure and the evolution of the insecticide resistance variants in natural populations over time and space. The results showed similar and constant nucleotide diversity and negative Tajimas D between An. gambiae s.s. and An. coluzzii. PCA and FST showed a clear genetic structure in the An. gambiae s.l. species. Genome-wide FST and H12 scans identified genomic regions under divergent selection and also having an implication in the adaptations to ecological changes. Novel voltage-gated sodium channel pyrethroid resistance target-site alleles (V402L, I1527T) were identified at increasing frequencies alongside the established kdr alleles (Vgsc-L995F, Vgsc-L995S and N1570Y) within the An. gambiae s.l. populations. Organophosphate metabolic resistance markers were also identified, at increasing frequencies, within the An. gambiae s.s. populations from 2012 to 2017, including the SNP Ace1-G280S and its associated duplication. Variants simultaneously identified in the same vector populations raise concerns about the long-term efficacy of new-generation bednets and the recently introduced organophosphate pirimiphos-methyl indoor residual spraying. These findings highlighted the benefit of genomic malaria vector surveillance for the detection of new insecticide resistance variants, the monitoring of the existing resistance variants, and also to get insights into the evolutionary processes driving insecticide resistance.

Author SummaryGenomic surveillance of malaria vectors is crucial for understanding the genetic variation in natural vector populations and also guiding the implementation of novel and innovative vector control tools. Application of sequencing technologies in vector studies provide insights on the genetic and evolutionary phenomena of vectors that could have impact on vector control strategies. By analyzing the whole genome data of 1409 wild An. gambiae s.l. mosquito collected between 2012 and 2017, we showed an emergence of novel insecticide resistance markers alongside increasing frequencies of existing insecticide resistance variants over time in Burkina Faso. We showed the benefit of genomic surveillance of malaria vectors for the monitoring of the insecticide resistance variants and also providing insights into the evolutionary processes driving insecticide resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568439" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568439">
        <p class="paperTitle">Genetic architecture of inbreeding depression may explain its persistence in a population of wild red deer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hewett, A.; Pemberton, J. M.; Johnston, S. E.</p>
        <p class="info">Score: 8.3, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568439' target='https://doi.org/10.1101/2023.11.23.568439'> 10.1101/2023.11.23.568439</a></p>
        <p class="abstract">Inbreeding depression is of major concern in declining populations, but relatively little is known about its genetic architecture in wild populations, such as the degree to which it is composed of large or small effect loci and their distribution throughout the genome. We combine fitness and genomic data from a wild population of red deer to investigate the genomic distribution of inbreeding effects. Inspired by the runs of homozygosity (ROH)-based inbreeding coefficient, FROH, we use chromosome-specific inbreeding coefficients (FROHChr) to explore whether the effect of inbreeding varies between chromosomes. Under the assumption that within an individual the probability of being identical-by-descent is equal across all chromosomes, we used a multi-membership model to estimate the deviation of FROHChr from the average inbreeding effect. This novel approach ensures effect sizes are not overestimated whilst maximising the power of our available dataset containing &gt;35,000 autosomal SNPs. We find that most chromosomes confer a minor reduction in fitness-related traits, which when these effects are summed, results in the observed inbreeding depression in birth weight, survival and lifetime breeding success. However, no chromosomes had a significantly detrimental effect compared to the overall effect of inbreeding. We conclude that in this population, inbreeding depression is the result of multiple mild or moderately deleterious mutations spread across all chromosomes. As predicted by genetic theory these mutations will be inefficiently purged, explaining the persistence of inbreeding depression in this population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569355">
        <p class="paperTitle">Evolution of hybrid inviability associated with chromosome fusions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boman, J.; Näsvall, K.; Vila, R.; Wiklund, C.; Backström, N.</p>
        <p class="info">Score: 6.7, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569355' target='https://doi.org/10.1101/2023.11.30.569355'> 10.1101/2023.11.30.569355</a></p>
        <p class="abstract">Chromosomal rearrangements, such as inversions, have received considerable attention in the speciation literature due to their hampering effects on recombination. However, less is known about how other rearrangements, such as chromosome fissions and fusions, can affect the evolution of reproductive isolation. Here, we used crosses between populations of the wood white butterfly (Leptidea sinapis) with different karyotypes to identify genomic regions associated with hybrid inviability. By contrasting allele frequencies between F2 hybrids that survived until adulthood with individuals of the same cohort that succumbed to hybrid incompatibilities, we show that candidate loci for hybrid inviability mainly are situated in fast-evolving regions with reduced recombination rates, especially in regions where chromosome fusions have occurred. Our results show that the extensive variation in chromosome numbers observed across the tree of life can be involved in speciation by being hotspots for the early evolution of postzygotic reproductive isolation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569171">
        <p class="paperTitle">Patterns of physical activity in hunter-gatherer children compared with US and UK children.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kretschmer, L.; Dyble, M.; Chaudhary, N.; Bann, D.; Salali, G. D.</p>
        <p class="info">Score: 6.1, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569171' target='https://doi.org/10.1101/2023.11.29.569171'> 10.1101/2023.11.29.569171</a></p>
        <p class="abstract">Sedentary lifestyles, mismatched with our active foraging history, contribute to escalating rates of non-communicable diseases. Contemporary hunter-gatherers appear to be highly active, but little is known about physical activity levels in hunter-gatherer children. We analysed 150 days of accelerometer data from 51 BaYaka hunter-gatherer children (aged 3-18) in the Republic of Congo, comparing it with British and American children (MCS and NHANES). BaYaka children were highly active, engaging in over 3 hours of moderate-to-vigorous physical activity (MVPA) daily, surpassing British adolescents by over 70 minutes. In US children activity declined with age; while in BaYaka children activity increased with age, unaffected by gender. Reflecting their foraging lifestyle, activity patterns varied within and between days, yet all children consistently rose with the sun. These findings highlight the impact of a foraging upbringing on children&#39;s activity levels, providing a benchmark for understanding childhood physical activity and wellbeing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
