<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Tertiary-interaction characters enable fast, model-based structural phylogenetics beyond the twilight zone
Authors: Puente-Lelievre, C.; Malik, A. J.; Douglas, J.; Ascher, D.; Baker, M.; Allison, J.; Poole, A. M.; Lundin, D.; Fullmer, M.; Bouckaert, R.; Steinegger, M.; Matzke, N. J.
Score: 70.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571181
Protein structure is more conserved than protein sequence, and therefore may be useful for phylogenetic inference beyond the &#34;twilight zone&#34; where sequence similarity is highly decayed.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Tertiary-interaction characters enable fast, model-based structural phylogenetics beyond the twilight zone
Authors: Puente-Lelievre, C.; Malik, A. J.; Douglas, J.; Ascher, D.; Baker, M.; Allison, J.; Poole, A. M.; Lundin, D.; Fullmer, M.; Bouckaert, R.; Steinegger, M.; Matzke, N. J.
Score: 70.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571181
Protein structure is more conserved than protein sequence, and therefore may be useful for phylogenetic inference beyond the &#34;twilight zone&#34; where sequence similarity is highly decayed." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Tertiary-interaction characters enable fast, model-based structural phylogenetics beyond the twilight zone
Authors: Puente-Lelievre, C.; Malik, A. J.; Douglas, J.; Ascher, D.; Baker, M.; Allison, J.; Poole, A. M.; Lundin, D.; Fullmer, M.; Bouckaert, R.; Steinegger, M.; Matzke, N. J.
Score: 70.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571181
Protein structure is more conserved than protein sequence, and therefore may be useful for phylogenetic inference beyond the &#34;twilight zone&#34; where sequence similarity is highly decayed."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Tertiary-interaction characters enable fast, model-based structural phylogenetics beyond the twilight zone\nAuthors: Puente-Lelievre, C.; Malik, A. J.; Douglas, J.; Ascher, D.; Baker, M.; Allison, J.; Poole, A. M.; Lundin, D.; Fullmer, M.; Bouckaert, R.; Steinegger, M.; Matzke, N. J.\nScore: 70.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571181\nProtein structure is more conserved than protein sequence, and therefore may be useful for phylogenetic inference beyond the \u0026#34;twilight zone\u0026#34; where sequence similarity is highly decayed.",
  "keywords": [
    
  ],
  "articleBody": " Tertiary-interaction characters enable fast, model-based structural phylogenetics beyond the twilight zone\nAuthors: Puente-Lelievre, C.; Malik, A. J.; Douglas, J.; Ascher, D.; Baker, M.; Allison, J.; Poole, A. M.; Lundin, D.; Fullmer, M.; Bouckaert, R.; Steinegger, M.; Matzke, N. J.\nScore: 70.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571181\nProtein structure is more conserved than protein sequence, and therefore may be useful for phylogenetic inference beyond the \"twilight zone\" where sequence similarity is highly decayed. Until recently, structural phylogenetics was constrained by the lack of solved structures for most proteins, and the reliance on phylogenetic distance methods which made it difficult to treat inference and uncertainty statistically. AlphaFold has mostly overcome the first problem by making structural predictions readily available. We address the second problem by redeploying a structural alphabet recently developed for Foldseek, a highly-efficient deep homology search program. For each residue in a structure, Foldseek identifies a tertiary interaction closest-neighbor residue in the structure, and classifies it into one of twenty \"3Di\" states. We test the hypothesis that 3Dis can be used as standard phylogenetic characters using a dataset of 53 structures from the ferritin-like superfamily. We performed 60 IQtree Maximum Likelihood runs to compare structure-free, PDB, and AlphaFold analyses, and default versus custom model sets that include a 3DI-specific rate matrix. Analyses that combine amino acids, 3Di characters, partitioning, and custom models produce the closest match to the structural distances tree of Malik et al. (2020), avoiding the long-branch attraction errors of structure-free analyses. Analyses include standard ultrafast bootstrapping confidence measures, and take minutes instead of weeks to run on desktop computers. These results suggest that structural phylogenetics could soon be routine practice in protein phylogenetics, allowing the re-exploration of many fundamental phylogenetic problems.\nExtreme in Every Way: Exceedingly Low Genetic Diversity in Snow Leopards Due to Persistently Small Population Size\nAuthors: Solari, K. A.; Morgan, S. A.; Poyarkov, A. D.; Weckworth, B.; Samelius, G.; Sharma, K.; Ostrowski, S.; Ramakrishnan, U.; Kubanychbekov, Z.; Kachel, S.; Johansson, O.; Lkhagvajav, P.; Hemmingmoore, H.; Aleksandrov, D.; Bayaraa, M.; Grachev, A.; Korablev, M. P.; Hernandez-Blanco, J. A.; Munkhtsog, B.; Rosenbaum, B.; Rozhnov, V. V.; Rajabi, A. M.; Noori, H.; Armstrong, E. E.; Petrov, D.\nScore: 35.0, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571340\nSnow leopards (Panthera uncia) serve as an umbrella species whose conservation benefits their high-elevation Asian habitat. Their numbers are believed to be in decline due to numerous Anthropogenic threats; however, their conservation is hindered by numerous knowledge gaps. They are the least studied genetically of all big cat species and little is known about their historic population size and range, current population trends, or connectivity across their range. Here, we use whole genome sequencing data for 41 snow leopards (37 newly sequenced) to assess population connectivity, historic population size, and current levels of genetic diversity. Among our samples, we find evidence of a primary genetic divide between the northern and southern part of the range around the Dzungarian Basin and a secondary divide south of Kyrgyzstan around the Taklamakan Desert. However, we find evidence of gene flow, suggesting that barriers between these groups are permeable. Perhaps most noteworthy, we find that snow leopards have the lowest genetic diversity of any big cat species, likely due to a persistently small population size throughout their evolutionary history. Without a large population size or ample standing genetic variation to help buffer them from any forthcoming Anthropogenic challenges, snow leopard persistence may be more tenuous than currently appreciated.\nFast and accurate estimation of selection coefficients and allele histories from ancient and modern DNA\nAuthors: Vaughn, A.; Nielsen, R.\nScore: 31.9, Published: 2023-12-17 DOI: 10.1101/2023.12.16.572012\nWe here present CLUES2, a full-likelihood method to infer natural selection from sequence data that is an extension of the method CLUES. We make several substantial improvements to the CLUES method that greatly increases both its applicability and its speed. We add the ability to use ARGs on ancient data as emissions to the underlying HMM, which enables CLUES2 to use both temporal and linkage information to make estimates of selection coefficients. We also fully implement the ability to estimate distinct selection coefficients in different epochs, which allows for the analysis of changes in selective pressures through time. In addition, we greatly increase the computational efficiency of CLUES2 over CLUES using several approximations to the forward-backward algorithms and develop a new way to reconstruct historic allele frequencies by integrating over the uncertainty in the estimation of the selection coefficients. We illustrate the accuracy of CLUES2 through extensive simulations and validate the importance sampling framework for integrating over the uncertainty in the inference of gene trees. We also show that CLUES2 is well-calibrated by showing that under the null hypothesis, the distribution of log-likelihood ratios follows a chi-squared distribution with the appropriate degrees of freedom. We run CLUES2 on a set of recently published ancient human data from Western Eurasia and test for evidence of changing selection coefficients through time. We find significant evidence of changing selective pressures in several genes correlated with the introduction of agriculture to Europe and the ensuing dietary and demographic shifts of that time. In particular, our analysis supports previous hypotheses of strong selection on lactase persistence during periods of ancient famines and attenuated selection in more modern periods.\nHybrid adaptation is hampered by Haldane's sieve\nAuthors: Bautista, C.; Gagnon-Arsenault, I.; Utrobina, M.; Fijarczyk, A.; Bendixsen, D. P.; Stelkens, R.; Landry, C.\nScore: 19.1, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571924\nHybrids between species exhibit plastic genomic architectures that foster phenotypic diversity. Their genomic instability also incurs costs, potentially limiting adaptation. When challenged to evolve in an environment containing a UV mimetic drug, yeast hybrids have reduced adaptation rates compared to parents. We hypothesized that this reduction could result from a faster accumulation of genomic changes, but we found no such association. Alternatively, we proposed that hybrids might lack access to adaptive mutations occurring in the parents, yet, we identified mutations in the same genes (PDR1 and YRR1), suggesting similar molecular adaptation mechanisms. However, mutations in these genes tended to be homozygous in the parents but heterozygous in the hybrids. We hypothesized that a lower rate of loss of heterozygosity (LOH) in hybrids could limit fitness gain. Using genome editing, we demonstrated that mutations display incomplete dominance, requiring homozygosity to show full impact and to circumvent Haldane's sieve, which favors the fixation of dominant mutations. We used frozen \"fossils\" to track genotype frequency dynamics and confirmed that LOH occurs at a slower pace in hybrids than in parents. Together, these findings show that Haldane's sieve slows down adaptation in hybrids, revealing an intrinsic constraint of hybrid genomic architecture that can limit the role of hybridization in adaptive evolution.\nPopulation Genomics of Postglacial Western Eurasia\nAuthors: Allentoft, M. E.; Sikora, M.; Refoyo-Martinez, A.; Irving-Pease, E. K.; Fischer, A.; Barrie, W.; Ingason, A.; Stenderup, J.; Sjögren, K.-G.; Pearson, A.; Sousa da Mota, B.; Paulsson, B. S.; Halgren, A. S.; Macleod, R.; Schjellerup Jorkov, M. L.; Demeter, F.; Novosolov, M.; Sorensen, L.; Nielsen, P. O.; Henriksen, R. A.; Vimala, T.; McColl, H.; Margaryan, A.; Ilardo, M.; Vaughn, A.; Mortensen, M. F.; Nielsen, A. B.; Hede, M. U.; Rasmussen, P.; Vinner, L.; Renaud, G.; Stern, A. J.; Trolle Jensen, T. Z.; Johannsen, N. N.; Scorrano, G.; Schroeder, H.; Lysdahl, P.; Ramsoe, A.; Skorobogatov, A.; Sc\nScore: 427.9, Published: 2023-12-13 DOI: 10.1101/2022.05.04.490594\nWestern Eurasia witnessed several large-scale human migrations during the Holocene1-5. To investigate the cross-continental impacts we shotgun-sequenced 317 primarily Mesolithic and Neolithic genomes from across Northern and Western Eurasia. These were imputed alongside published data to obtain diploid genotypes from \u003e1,600 ancient humans. Our analyses revealed a Great Divide genomic boundary extending from the Black Sea to the Baltic. Mesolithic hunter-gatherers (HGs) were highly genetically differentiated east and west of this zone, and the impact of the neolithisation was equally disparate. Large-scale ancestry shifts occurred in the west as farming was introduced, including near-total replacements of HGs in many areas, whereas no substantial ancestry shifts happened east of the zone during the same period. Similarly, relatedness decreased in the west from the Neolithic transition onwards, while east of the Urals relatedness remained high until [~]4,000 BP, consistent with persistence of localised HG groups. The boundary dissolved when Yamnaya-related ancestry spread across western Eurasia around 5,000 BP resulting in a second major turnover that reached most parts of Europe within a 1,000-year span. The genetic origin and fate of the Yamnaya have remained elusive but we demonstrate that HGs from the Middle Don region contributed ancestry to them. Yamnaya-groups later admixed with individuals associated with the Globular Amphora Culture before expanding into Europe. Similar turnovers occurred in western Siberia, where we report new genomic data from a Neolithic steppe cline spanning the Siberian forest steppe to Lake Baikal. These prehistoric migrations had profound and lasting effects on the genetic diversity of Eurasian populations.\nSubspecies divergence, hybridisation and the spatial environment shape phylosymbiosis in the microbiome of house mice\nAuthors: Ferreira, S. C. M.; Jarquin-Diaz, V. H.; Planillo, A.; Dureje, L.; Martincova, I.; Kramer-Schadt, S.; Forslund, S. K.; Heitlinger, E.\nScore: 15.0, Published: 2023-12-11 DOI: 10.1101/2023.12.11.571054\nClosely related host species share similar symbionts, yet how host genetics and the environment affect symbiont communities at different stages of host genetic divergence remains largely unknown. Similarly, it is unclear whether host-symbiont associations result from or contribute to host divergence. We examined the intestinal community of 619 wild-caught mice from Germanys European house mouse hybrid zone. Here, hybridisation upon secondary contact reflects divergence and could be traced gradually. Temporal and spatial factors were strong predictors of microbiome composition. Subspecies divergence predicted the composition similarity of the overall microbiome, specifically in the bacteria, parasite and fungal components. The effect of hybridisation was generally weak but significant for the fungal component. We confirmed our results in experiments with wild-derived inbred mice: subspecies genetic distances and hybridisation predicted the overall microbiome composition, and hybridisation further predicted fungal similarities among individuals. Fungi seemed more stable to the community perturbation of infection than other components of the microbiome. Differences between subspecies were more substantial across different microbiome components than those associated with hybridisation. Diverged microbiomes are a product of host divergence and are maintained by host genetics upon large environmental effects. These results provide a unique perspective into the ecoevolutionary processes shaping phylosymbiosis.\nEvolution in microbial microcosms is highly parallel regardless of the presence of interacting species\nAuthors: Meroz, N.; Livny, T.; Toledano, G.; Sorokin, Y.; Tovi, N.; Friedman, J.\nScore: 10.4, Published: 2023-12-14 DOI: 10.1101/2023.12.14.571477\nDuring laboratory evolution, replicate bacterial populations often follow similar trajectories, thus their evolution is potentially predictable. However, predicting the evolution of natural populations, which are commonly embedded in multispecies communities, would prove extremely difficult if adaptations are contingent on the identity of the interacting species. The extent to which adaptations typically depend on coevolving partners remains poorly understood, since coevolution is commonly studied using small-scale experiments involving few species, making it challenging to extract general trends. To address this knowledge gap, we study the adaptations that occurred in strains of each of 11 species that were either evolved in monoculture or in multiple pairwise co-cultures. While we detect slight but significant partner-specific effects we find that the majority of evolutionary changes that occur are robust across strains that evolved with different partners; species growth abilities increase by a similar factor regardless of partners identity, shifts in community compositions and interactions are similar between pairs of coevolved and separately evolved strains, and the majority of parallelly mutated genes were detected in multiple biotic conditions. We hypothesized that these results might arise from the fact that ancestral strains are maladapted to the abiotic environment, thus having a pool of adaptations that are beneficial regardless of the biotic partners. Therefore, we conducted a second experiment with strains that were pre-adapted to the abiotic conditions before being co-cultured. We find that even after [~]400 generations of pre-adaptation, evolution is surprisingly non-partner-specific. Further work is required in order to elucidate the factors that influence partner-specificity of coevolution, but our results suggest that selection imposed by the biotic environment may play a secondary role to that imposed by abiotic conditions, making predictions regarding coevolutionary dynamics less challenging than previously thought.\nPhylogenomics resolves key relationships in Rumex and uncovers a dynamic history of independently evolving sex chromosomes\nAuthors: Hibbins, M. S.; Rifkin, J. L.; Choudhury, B. I.; Voznesenka, O.; Sacchi, B. M.; Yuan, M.; Gong, Y.; Barrett, S. C. H.; Wright, S. I.\nScore: 10.4, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571571\nSex chromosomes have evolved independently many times across eukaryotes. Despite a considerable body of literature on the evolution of sex chromosomes, the causes and consequences of variation in the formation, degeneration, and turnover of sex chromosomes remain poorly understood. Comparative approaches in groups with sexual system variation can be valuable for understanding these questions. Plants are well-suited to such comparative studies, with many lineages containing relatively recent origins of dioecy and sex chromosomes as well as hermaphroditic close relatives. Rumex is a diverse genus of flowering plants harboring significant sexual system variation, including hermaphroditic and dioecious clades with XY sex chromosomes. Previous disagreement in the phylogenetic relationships among key species have rendered the history of sex chromosome evolution uncertain. Resolving this history is important to the development of Rumex as a system for the comparative study of sex chromosome evolution. Here, we leverage new transcriptome assemblies from 11 species representing the major clades in the genus, along with a whole-genome assembly generated for a pivotal hermaphroditic species, to further resolve the phylogeny and history of sex chromosome evolution in Rumex. Using phylogenomic approaches, we find evidence for two independent origins of sex chromosomes and introgression from unsampled taxa in the genus. Comparative genomics reveals massive chromosomal rearrangements in a dioecious species, with evidence for a complex origin of the sex chromosomes through multiple chromosomal fusions. However, we see no evidence of elevated rates of fusion on the sex chromosome in comparison with autosomes, providing no support for an adaptive hypothesis for the sex chromosome expansion. Overall, our results highlight the dynamic nature of sex chromosome systems in Rumex and illustrate the utility of the genus as a model for the comparative study of sex chromosome evolution.\nBoundary effects cause false signals of range expansions in population genomic data\nAuthors: Kemppainen, P.; Schemberi, R.; Momigliano, P.\nScore: 9.0, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570483\nStudying range expansions (REs) is central for understanding genetic variation through space and time as well as for identifying refugia and biological invasions. Range expansions are characterized by serial founder events causing clines of decreasing diversity away from the center of origin and asymmetries in the two-dimensional allele frequency spectra. These asymmetries, summarized by the directionality index ({psi}), are sensitive to REs and persist for longer than clines in genetic diversity. In continuous and finite meta-populations, genetic drift tends to be stronger at the edges of the species distribution. Such boundary effects (BEs) are expected to affect geographic patterns in {psi} as well as genetic diversity. With simulations we show that BEs consistently cause high false positive rates in equilibrium meta-populations when testing for REs. In the simulations, the absolute value of {psi} (|{psi}|) in equilibrium data sets was proportional to the fixation index (FST). By fitting signatures of REs as a function of {varepsilon}=|{psi}|/FST and geographic clines in {psi}, strong evidence for REs could be detected in data from a recent rapid invasion of the cane toad, Rhinella marina, in Australia, but not in 28 previously published empirical data sets from Australian scincid lizards or the Indo-Australasian blacktip shark that were significant for the standard RE tests. Thus, while clinal variation in {psi} is still the most sensitive statistic to REs, in order to detect true signatures of REs in natural populations, its magnitude needs to be considered in relation to the overall levels of genetic structuring in the data.\nIdentification of the Mode of Evolution in Incomplete Carbonate Successions\nAuthors: Hohmann, N.; Koelewijn, J. R.; Burgess, P.; Jarochowska, E.\nScore: 7.3, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572098\nBackground: The fossil record provides the unique opportunity to observe evolution over millions of years, but is known to be incomplete. While incompleteness varies spatially and is hard to estimate for empirical sections, computer simulations of geological processes allow to examine the biasing effects of the incompleteness in silico. We combine simulations of different modes of evolution (stasis, (un)biased random walks) with deposition of carbonate platforms strata to examine how well the mode of evolution can be recovered from fossil time series, and how test results vary between different positions in the carbonate platform and multiple stratigraphic architectures generated by different sea level curves. Results: Stratigraphic architecture and position along an onshore-offshore gradient has only a small influence on the mode of evolution recovered by statistical tests. Tests fail to identify the correct mode of evolution even in the absence of stratigraphic biases, for adequate models, and under excellent sampling conditions. Visual examination of trait evolution in lineages shows that rather than stratigraphic incompleteness, maximum hiatus duration determines how much fossil time series differ from the original evolutionary process. Directional evolution is more prone to the biasing effects of stratigraphy, turning it into apparent punctuated anagenesis. In contrast, stasis remains unaffected. Conclusions: *Tests for the mode of evolution should be reviewed critically, as they do not identify the correct (simulated) mode of evolution, even for adequate models, in the absence of stratigraphic effects, and for exceptionally long time series. *Fossil time series favor the recognition of both stasis and complex, punctuated modes of evolution. *Not stratigraphic incompleteness, but the presence of rare, prolonged gaps has the largest biasing effect on trait evolution. This suggests that incomplete sections with regular hiatus frequency and durations can potentially preserve evolutionary history without major biases.\n",
  "wordCount" : "2950",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571181">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571181" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571181">
        <p class="paperTitle">Tertiary-interaction characters enable fast, model-based structural phylogenetics beyond the twilight zone</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571181" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571181" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Puente-Lelievre, C.; Malik, A. J.; Douglas, J.; Ascher, D.; Baker, M.; Allison, J.; Poole, A. M.; Lundin, D.; Fullmer, M.; Bouckaert, R.; Steinegger, M.; Matzke, N. J.</p>
        <p class="info">Score: 70.0, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571181' target='https://doi.org/10.1101/2023.12.12.571181'> 10.1101/2023.12.12.571181</a></p>
        <p class="abstract">Protein structure is more conserved than protein sequence, and therefore may be useful for phylogenetic inference beyond the &#34;twilight zone&#34; where sequence similarity is highly decayed. Until recently, structural phylogenetics was constrained by the lack of solved structures for most proteins, and the reliance on phylogenetic distance methods which made it difficult to treat inference and uncertainty statistically. AlphaFold has mostly overcome the first problem by making structural predictions readily available. We address the second problem by redeploying a structural alphabet recently developed for Foldseek, a highly-efficient deep homology search program. For each residue in a structure, Foldseek identifies a tertiary interaction closest-neighbor residue in the structure, and classifies it into one of twenty &#34;3Di&#34; states. We test the hypothesis that 3Dis can be used as standard phylogenetic characters using a dataset of 53 structures from the ferritin-like superfamily. We performed 60 IQtree Maximum Likelihood runs to compare structure-free, PDB, and AlphaFold analyses, and default versus custom model sets that include a 3DI-specific rate matrix. Analyses that combine amino acids, 3Di characters, partitioning, and custom models produce the closest match to the structural distances tree of Malik et al. (2020), avoiding the long-branch attraction errors of structure-free analyses. Analyses include standard ultrafast bootstrapping confidence measures, and take minutes instead of weeks to run on desktop computers. These results suggest that structural phylogenetics could soon be routine practice in protein phylogenetics, allowing the re-exploration of many fundamental phylogenetic problems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571340">
        <p class="paperTitle">Extreme in Every Way: Exceedingly Low Genetic Diversity in Snow Leopards Due to Persistently Small Population Size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Solari, K. A.; Morgan, S. A.; Poyarkov, A. D.; Weckworth, B.; Samelius, G.; Sharma, K.; Ostrowski, S.; Ramakrishnan, U.; Kubanychbekov, Z.; Kachel, S.; Johansson, O.; Lkhagvajav, P.; Hemmingmoore, H.; Aleksandrov, D.; Bayaraa, M.; Grachev, A.; Korablev, M. P.; Hernandez-Blanco, J. A.; Munkhtsog, B.; Rosenbaum, B.; Rozhnov, V. V.; Rajabi, A. M.; Noori, H.; Armstrong, E. E.; Petrov, D.</p>
        <p class="info">Score: 35.0, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571340' target='https://doi.org/10.1101/2023.12.14.571340'> 10.1101/2023.12.14.571340</a></p>
        <p class="abstract">Snow leopards (Panthera uncia) serve as an umbrella species whose conservation benefits their high-elevation Asian habitat. Their numbers are believed to be in decline due to numerous Anthropogenic threats; however, their conservation is hindered by numerous knowledge gaps. They are the least studied genetically of all big cat species and little is known about their historic population size and range, current population trends, or connectivity across their range. Here, we use whole genome sequencing data for 41 snow leopards (37 newly sequenced) to assess population connectivity, historic population size, and current levels of genetic diversity. Among our samples, we find evidence of a primary genetic divide between the northern and southern part of the range around the Dzungarian Basin and a secondary divide south of Kyrgyzstan around the Taklamakan Desert. However, we find evidence of gene flow, suggesting that barriers between these groups are permeable. Perhaps most noteworthy, we find that snow leopards have the lowest genetic diversity of any big cat species, likely due to a persistently small population size throughout their evolutionary history. Without a large population size or ample standing genetic variation to help buffer them from any forthcoming Anthropogenic challenges, snow leopard persistence may be more tenuous than currently appreciated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.572012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.572012" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.572012">
        <p class="paperTitle">Fast and accurate estimation of selection coefficients and allele histories from ancient and modern DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.572012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.572012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vaughn, A.; Nielsen, R.</p>
        <p class="info">Score: 31.9, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.572012' target='https://doi.org/10.1101/2023.12.16.572012'> 10.1101/2023.12.16.572012</a></p>
        <p class="abstract">We here present CLUES2, a full-likelihood method to infer natural selection from sequence data that is an extension of the method CLUES. We make several substantial improvements to the CLUES method that greatly increases both its applicability and its speed. We add the ability to use ARGs on ancient data as emissions to the underlying HMM, which enables CLUES2 to use both temporal and linkage information to make estimates of selection coefficients. We also fully implement the ability to estimate distinct selection coefficients in different epochs, which allows for the analysis of changes in selective pressures through time. In addition, we greatly increase the computational efficiency of CLUES2 over CLUES using several approximations to the forward-backward algorithms and develop a new way to reconstruct historic allele frequencies by integrating over the uncertainty in the estimation of the selection coefficients. We illustrate the accuracy of CLUES2 through extensive simulations and validate the importance sampling framework for integrating over the uncertainty in the inference of gene trees. We also show that CLUES2 is well-calibrated by showing that under the null hypothesis, the distribution of log-likelihood ratios follows a chi-squared distribution with the appropriate degrees of freedom. We run CLUES2 on a set of recently published ancient human data from Western Eurasia and test for evidence of changing selection coefficients through time. We find significant evidence of changing selective pressures in several genes correlated with the introduction of agriculture to Europe and the ensuing dietary and demographic shifts of that time. In particular, our analysis supports previous hypotheses of strong selection on lactase persistence during periods of ancient famines and attenuated selection in more modern periods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571924">
        <p class="paperTitle">Hybrid adaptation is hampered by Haldane&#39;s sieve</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bautista, C.; Gagnon-Arsenault, I.; Utrobina, M.; Fijarczyk, A.; Bendixsen, D. P.; Stelkens, R.; Landry, C.</p>
        <p class="info">Score: 19.1, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571924' target='https://doi.org/10.1101/2023.12.15.571924'> 10.1101/2023.12.15.571924</a></p>
        <p class="abstract">Hybrids between species exhibit plastic genomic architectures that foster phenotypic diversity. Their genomic instability also incurs costs, potentially limiting adaptation. When challenged to evolve in an environment containing a UV mimetic drug, yeast hybrids have reduced adaptation rates compared to parents. We hypothesized that this reduction could result from a faster accumulation of genomic changes, but we found no such association. Alternatively, we proposed that hybrids might lack access to adaptive mutations occurring in the parents, yet, we identified mutations in the same genes (PDR1 and YRR1), suggesting similar molecular adaptation mechanisms. However, mutations in these genes tended to be homozygous in the parents but heterozygous in the hybrids. We hypothesized that a lower rate of loss of heterozygosity (LOH) in hybrids could limit fitness gain. Using genome editing, we demonstrated that mutations display incomplete dominance, requiring homozygosity to show full impact and to circumvent Haldane&#39;s sieve, which favors the fixation of dominant mutations. We used frozen &#34;fossils&#34; to track genotype frequency dynamics and confirmed that LOH occurs at a slower pace in hybrids than in parents. Together, these findings show that Haldane&#39;s sieve slows down adaptation in hybrids, revealing an intrinsic constraint of hybrid genomic architecture that can limit the role of hybridization in adaptive evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.04.490594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.04.490594" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.04.490594">
        <p class="paperTitle">Population Genomics of Postglacial Western Eurasia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.04.490594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.04.490594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Allentoft, M. E.; Sikora, M.; Refoyo-Martinez, A.; Irving-Pease, E. K.; Fischer, A.; Barrie, W.; Ingason, A.; Stenderup, J.; Sjögren, K.-G.; Pearson, A.; Sousa da Mota, B.; Paulsson, B. S.; Halgren, A. S.; Macleod, R.; Schjellerup Jorkov, M. L.; Demeter, F.; Novosolov, M.; Sorensen, L.; Nielsen, P. O.; Henriksen, R. A.; Vimala, T.; McColl, H.; Margaryan, A.; Ilardo, M.; Vaughn, A.; Mortensen, M. F.; Nielsen, A. B.; Hede, M. U.; Rasmussen, P.; Vinner, L.; Renaud, G.; Stern, A. J.; Trolle Jensen, T. Z.; Johannsen, N. N.; Scorrano, G.; Schroeder, H.; Lysdahl, P.; Ramsoe, A.; Skorobogatov, A.; Sc</p>
        <p class="info">Score: 427.9, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.04.490594' target='https://doi.org/10.1101/2022.05.04.490594'> 10.1101/2022.05.04.490594</a></p>
        <p class="abstract">Western Eurasia witnessed several large-scale human migrations during the Holocene1-5. To investigate the cross-continental impacts we shotgun-sequenced 317 primarily Mesolithic and Neolithic genomes from across Northern and Western Eurasia. These were imputed alongside published data to obtain diploid genotypes from &gt;1,600 ancient humans. Our analyses revealed a  Great Divide genomic boundary extending from the Black Sea to the Baltic. Mesolithic hunter-gatherers (HGs) were highly genetically differentiated east and west of this zone, and the impact of the neolithisation was equally disparate. Large-scale ancestry shifts occurred in the west as farming was introduced, including near-total replacements of HGs in many areas, whereas no substantial ancestry shifts happened east of the zone during the same period. Similarly, relatedness decreased in the west from the Neolithic transition onwards, while east of the Urals relatedness remained high until [~]4,000 BP, consistent with persistence of localised HG groups. The boundary dissolved when Yamnaya-related ancestry spread across western Eurasia around 5,000 BP resulting in a second major turnover that reached most parts of Europe within a 1,000-year span. The genetic origin and fate of the Yamnaya have remained elusive but we demonstrate that HGs from the Middle Don region contributed ancestry to them. Yamnaya-groups later admixed with individuals associated with the Globular Amphora Culture before expanding into Europe. Similar turnovers occurred in western Siberia, where we report new genomic data from a  Neolithic steppe cline spanning the Siberian forest steppe to Lake Baikal. These prehistoric migrations had profound and lasting effects on the genetic diversity of Eurasian populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571054">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571054" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571054">
        <p class="paperTitle">Subspecies divergence, hybridisation and the spatial environment shape phylosymbiosis in the microbiome of house mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571054" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571054" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferreira, S. C. M.; Jarquin-Diaz, V. H.; Planillo, A.; Dureje, L.; Martincova, I.; Kramer-Schadt, S.; Forslund, S. K.; Heitlinger, E.</p>
        <p class="info">Score: 15.0, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571054' target='https://doi.org/10.1101/2023.12.11.571054'> 10.1101/2023.12.11.571054</a></p>
        <p class="abstract">Closely related host species share similar symbionts, yet how host genetics and the environment affect symbiont communities at different stages of host genetic divergence remains largely unknown. Similarly, it is unclear whether host-symbiont associations result from or contribute to host divergence.

We examined the intestinal community of 619 wild-caught mice from Germanys European house mouse hybrid zone. Here, hybridisation upon secondary contact reflects divergence and could be traced gradually.

Temporal and spatial factors were strong predictors of microbiome composition. Subspecies divergence predicted the composition similarity of the overall microbiome, specifically in the bacteria, parasite and fungal components. The effect of hybridisation was generally weak but significant for the fungal component. We confirmed our results in experiments with wild-derived inbred mice: subspecies genetic distances and hybridisation predicted the overall microbiome composition, and hybridisation further predicted fungal similarities among individuals. Fungi seemed more stable to the community perturbation of infection than other components of the microbiome.

Differences between subspecies were more substantial across different microbiome components than those associated with hybridisation. Diverged microbiomes are a product of host divergence and are maintained by host genetics upon large environmental effects. These results provide a unique perspective into the ecoevolutionary processes shaping phylosymbiosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571477" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571477">
        <p class="paperTitle">Evolution in microbial microcosms is highly parallel regardless of the presence of interacting species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meroz, N.; Livny, T.; Toledano, G.; Sorokin, Y.; Tovi, N.; Friedman, J.</p>
        <p class="info">Score: 10.4, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571477' target='https://doi.org/10.1101/2023.12.14.571477'> 10.1101/2023.12.14.571477</a></p>
        <p class="abstract">During laboratory evolution, replicate bacterial populations often follow similar trajectories, thus their evolution is potentially predictable. However, predicting the evolution of natural populations, which are commonly embedded in multispecies communities, would prove extremely difficult if adaptations are contingent on the identity of the interacting species. The extent to which adaptations typically depend on coevolving partners remains poorly understood, since coevolution is commonly studied using small-scale experiments involving few species, making it challenging to extract general trends. To address this knowledge gap, we study the adaptations that occurred in strains of each of 11 species that were either evolved in monoculture or in multiple pairwise co-cultures. While we detect slight but significant partner-specific effects we find that the majority of evolutionary changes that occur are robust across strains that evolved with different partners; species growth abilities increase by a similar factor regardless of partners identity, shifts in community compositions and interactions are similar between pairs of coevolved and separately evolved strains, and the majority of parallelly mutated genes were detected in multiple biotic conditions. We hypothesized that these results might arise from the fact that ancestral strains are maladapted to the abiotic environment, thus having a pool of adaptations that are beneficial regardless of the biotic partners. Therefore, we conducted a second experiment with strains that were pre-adapted to the abiotic conditions before being co-cultured. We find that even after [~]400 generations of pre-adaptation, evolution is surprisingly non-partner-specific. Further work is required in order to elucidate the factors that influence partner-specificity of coevolution, but our results suggest that selection imposed by the biotic environment may play a secondary role to that imposed by abiotic conditions, making predictions regarding coevolutionary dynamics less challenging than previously thought.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571571">
        <p class="paperTitle">Phylogenomics resolves key relationships in Rumex and uncovers a dynamic history of independently evolving sex chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hibbins, M. S.; Rifkin, J. L.; Choudhury, B. I.; Voznesenka, O.; Sacchi, B. M.; Yuan, M.; Gong, Y.; Barrett, S. C. H.; Wright, S. I.</p>
        <p class="info">Score: 10.4, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571571' target='https://doi.org/10.1101/2023.12.13.571571'> 10.1101/2023.12.13.571571</a></p>
        <p class="abstract">Sex chromosomes have evolved independently many times across eukaryotes. Despite a considerable body of literature on the evolution of sex chromosomes, the causes and consequences of variation in the formation, degeneration, and turnover of sex chromosomes remain poorly understood. Comparative approaches in groups with sexual system variation can be valuable for understanding these questions. Plants are well-suited to such comparative studies, with many lineages containing relatively recent origins of dioecy and sex chromosomes as well as hermaphroditic close relatives. Rumex is a diverse genus of flowering plants harboring significant sexual system variation, including hermaphroditic and dioecious clades with XY sex chromosomes. Previous disagreement in the phylogenetic relationships among key species have rendered the history of sex chromosome evolution uncertain. Resolving this history is important to the development of Rumex as a system for the comparative study of sex chromosome evolution. Here, we leverage new transcriptome assemblies from 11 species representing the major clades in the genus, along with a whole-genome assembly generated for a pivotal hermaphroditic species, to further resolve the phylogeny and history of sex chromosome evolution in Rumex. Using phylogenomic approaches, we find evidence for two independent origins of sex chromosomes and introgression from unsampled taxa in the genus. Comparative genomics reveals massive chromosomal rearrangements in a dioecious species, with evidence for a complex origin of the sex chromosomes through multiple chromosomal fusions. However, we see no evidence of elevated rates of fusion on the sex chromosome in comparison with autosomes, providing no support for an adaptive hypothesis for the sex chromosome expansion. Overall, our results highlight the dynamic nature of sex chromosome systems in Rumex and illustrate the utility of the genus as a model for the comparative study of sex chromosome evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570483">
        <p class="paperTitle">Boundary effects cause false signals of range expansions in population genomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kemppainen, P.; Schemberi, R.; Momigliano, P.</p>
        <p class="info">Score: 9.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570483' target='https://doi.org/10.1101/2023.12.06.570483'> 10.1101/2023.12.06.570483</a></p>
        <p class="abstract">Studying range expansions (REs) is central for understanding genetic variation through space and time as well as for identifying refugia and biological invasions. Range expansions are characterized by serial founder events causing clines of decreasing diversity away from the center of origin and asymmetries in the two-dimensional allele frequency spectra. These asymmetries, summarized by the directionality index ({psi}), are sensitive to REs and persist for longer than clines in genetic diversity. In continuous and finite meta-populations, genetic drift tends to be stronger at the edges of the species distribution. Such boundary effects (BEs) are expected to affect geographic patterns in {psi} as well as genetic diversity. With simulations we show that BEs consistently cause high false positive rates in equilibrium meta-populations when testing for REs. In the simulations, the absolute value of {psi} (|{psi}|) in equilibrium data sets was proportional to the fixation index (FST). By fitting signatures of REs as a function of {varepsilon}=|{psi}|/FST and geographic clines in {psi}, strong evidence for REs could be detected in data from a recent rapid invasion of the cane toad, Rhinella marina, in Australia, but not in 28 previously published empirical data sets from Australian scincid lizards or the Indo-Australasian blacktip shark that were significant for the standard RE tests. Thus, while clinal variation in {psi} is still the most sensitive statistic to REs, in order to detect true signatures of REs in natural populations, its magnitude needs to be considered in relation to the overall levels of genetic structuring in the data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572098">
        <p class="paperTitle">Identification of the Mode of Evolution in Incomplete Carbonate Successions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hohmann, N.; Koelewijn, J. R.; Burgess, P.; Jarochowska, E.</p>
        <p class="info">Score: 7.3, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572098' target='https://doi.org/10.1101/2023.12.18.572098'> 10.1101/2023.12.18.572098</a></p>
        <p class="abstract">Background: The fossil record provides the unique opportunity to observe evolution over millions of years, but is known to be incomplete. While incompleteness varies spatially and is hard to estimate for empirical sections, computer simulations of geological processes allow to examine the biasing effects of the incompleteness in silico. We combine simulations of different modes of evolution (stasis, (un)biased random walks) with deposition of carbonate platforms strata to examine how well the mode of evolution can be recovered from fossil time series, and how test results vary between different positions in the carbonate platform and multiple stratigraphic architectures generated by different sea level curves. Results: Stratigraphic architecture and position along an onshore-offshore gradient has only a small influence on the mode of evolution recovered by statistical tests. Tests fail to identify the correct mode of evolution even in the absence of stratigraphic biases, for adequate models, and under excellent sampling conditions. Visual examination of trait evolution in lineages shows that rather than stratigraphic incompleteness, maximum hiatus duration determines how much fossil time series differ from the original evolutionary process. Directional evolution is more prone to the biasing effects of stratigraphy, turning it into apparent punctuated anagenesis. In contrast, stasis remains unaffected. Conclusions: *Tests for the mode of evolution should be reviewed critically, as they do not identify the correct (simulated) mode of evolution, even for adequate models, in the absence of stratigraphic effects, and for exceptionally long time series. *Fossil time series favor the recognition of both stasis and complex, punctuated modes of evolution. *Not stratigraphic incompleteness, but the presence of rare, prolonged gaps has the largest biasing effect on trait evolution. This suggests that incomplete sections with regular hiatus frequency and durations can potentially preserve evolutionary history without major biases.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
