<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Fundamental equations linking methylation dynamics to maximum lifespan in mammals
Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.
Score: 79.8, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643
We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Fundamental equations linking methylation dynamics to maximum lifespan in mammals
Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.
Score: 79.8, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643
We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Fundamental equations linking methylation dynamics to maximum lifespan in mammals
Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.
Score: 79.8, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643
We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Fundamental equations linking methylation dynamics to maximum lifespan in mammals\nAuthors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.\nScore: 79.8, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643\nWe establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset.",
  "keywords": [
    
  ],
  "articleBody": " Fundamental equations linking methylation dynamics to maximum lifespan in mammals\nAuthors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.\nScore: 79.8, Published: 2023-11-05 DOI: 10.1101/2023.05.21.541643\nWe establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset. Our research leads to three key findings: First, a reciprocal relationship exists between the average rate of methylation change (AROCM) in bivalent promoter regions and maximum lifespan: AROCM ~ 1/MaxLifespan. Second, the correlation between age and average methylation bears no relation to maximum lifespan, Cor(Methly,Age) independent of MaxLifespan. Third, the rate of methylation change in young animals is related to that in old animals: Young animals' AROCM ~ Old AROCM. These findings hinge on the chromatin context, as different results emerge when defining AROCM using different chromatin states. Our analytical framework is versatile and readily extendable to a broad range of other molecular assessments. Overall, our study demonstrates that epigenetic aging rates in specific chromatin states exhibit an inverse relationship with maximum lifespan in mammals.\nComplete phylogeny of Micrathena spiders suggests multiple dispersal events among Neotropical rainforests, islands, and landmasses, and indicates Andean orogeny promotes speciation\nAuthors: Magalhaes, I. L. F.; Martins, P. H.; Faleiro, B. T.; Vidigal, T. H. D. A.; Santos, F. R.; Carvalho, L. S.; Santos, A. J.\nScore: 68.5, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565210\nThe Neotropical region is the most diverse on the planet, largely due to its mosaic of tropical rainforests. Multiple tectonic and climatic processes have been hypothesized to play a role in generating this diversity; these include Andean orogeny, closure of the Panama Isthmus, the hypothesized GAARlandia land bridge, and putative historical connections among currently isolated forests. Micrathena spiny spiders (Araneidae) include [~]120 species distributed mostly in forests from Mexico to Argentina, including the Antilles. Here, we use it as a model to study the biogeographic history of Neotropical rainforests by estimating a complete, dated phylogeny using morphological data for all species and molecular data for a subsample of 79 species. This resulted in a phylogeny that is mostly robust and supports most of the previously recognized species groups, although with uncertainty in the phylogenetic position of some species, especially those lacking sequences. The genus began diversifying about 25 million years ago. We use an event-based approach and biogeographic stochastic mapping to estimate ancestral distributions and the timing and direction of dispersal events, and to identify areas where diversity was generated, while accounting for phylogenetic uncertainty. Andean cloud forests generated the majority of species through in-situ speciation, but the Amazon was the major source of species for adjacent areas through dispersal; on the other extreme, the Dry Diagonal received species from other areas but generated very little diversity. Species exchange between Central and South America was intense, with [~]23 dispersal events beginning at least 20 million years ago, indicating that Micrathena dispersed between these continents before closure of the Isthmus of Panama. We also inferred [~]4 over-water dispersal events from Central and North America to the Antilles, which happened in the last 20 million years, and thus much after the proposed age of the GAARlandia land bridge. We identified an important species exchange route between the Amazon and the Atlantic forests. Sampling all species of the genus was fundamental to some of the conclusions above, especially in identifying the Andes as the area that generated the majority of species. This study highlights the importance of a solid and complete taxonomic sampling in biogeographic studies.\nSpoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s\nAuthors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.\nScore: 30.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564725\nDuring the last few centuries D. melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D. melanogaster. It is a 5216nt LTR retrotransposon of the Ty3/gypsy superfamily. Relying on strains sampled at different times during the last century we show that Spoink invaded worldwide D. melanogaster populations after the P-element between 1983 and 1993. This invasion was likely triggered by a horizontal transfer from the D. willistoni group, much as the P-element. Spoink is probably silenced by the piRNA pathway in natural populations and about 1/3 of the examined strains have an insertion into a canonical piRNA cluster such as 42AB. Given the degree of genetic investigation of D. melanogaster it is surprising that Spoink was able to invade unnoticed.\nFrom museum drawer to tree: historical DNA phylogenomics clarifies the systematics of rare dung beetles (Coleoptera: Scarabaeinae) from museum collections\nAuthors: Lopes, F.; Gunter, N.; Gillett, C. P. D. T.; Montanaro, G.; Rossini, M.; Losacco, F.; Daniel, G. M.; Straube, N.; Tarasov, S.\nScore: 27.6, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564347\nAlthough several methods exist for extracting and sequencing historical DNA originating from drypreserved insect specimens deposited in natural history museums, no consensus exists as to what is the optimal approach. We demonstrate that a customized, low-cost archival DNA extraction protocol ([~] {euro}10 per sample), in combination with Ultraconserved Elements (UCEs), is an effective tool for insect phylogenomic studies. We successfully tested our approach by sequencing DNA from scarab dung beetles preserved in both wet and dry collections, including unique primary type and rare historical specimens from internationally important natural history museums in London, Paris and Helsinki. The focal specimens comprise enigmatic dung beetle genera (Nesosisyphus, Onychotechus and Helictopleurus) that varied in age and preservation. The oldest specimen, the holotype of the now possibly extinct Mauritian endemic Nesosisyphus rotundatus, was collected in 1944. We obtained high-quality DNA from all studied specimens to enable the generation of a UCE-based dataset that revealed an insightful and well-supported phylogenetic tree of dung beetles. The resulting phylogeny suggested the reclassification of Onychotechus (previously incertae sedis) within the tribe Coprini. Our approach demonstrates the feasibility and effectiveness of combining DNA data from historic and recent museum specimens to provide novel insights. The proposed archival DNA protocol is available at DOI 10.17504/protocols.io.81wgbybqyvpk/v1 HighlightsO_LIWe combined custom low-cost archival DNA extractions and Ultraconserved Element phylogenomics C_LIO_LIDNA from rare museum specimens of enigmatic dung beetles revealed their phylogenetic connections C_LIO_LIGenomic data was obtained from the holotype of a potentially extinct monoinsular endemic species C_LIO_LIGenomic data allowed a rare and enigmatic species of previously unknown affinity to be classified C_LIO_LIThe morphology of museum specimens remained intact following non-destructive DNA extraction C_LI O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=80 SRC=\"FIGDIR/small/564347v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (24K): org.highwire.dtl.DTLVardef@cdc35corg.highwire.dtl.DTLVardef@d4feaaorg.highwire.dtl.DTLVardef@19b7e9dorg.highwire.dtl.DTLVardef@195a467_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPhylogenomics reveals the deep ocean as an accelerator for evolutionary diversification in anglerfishes\nAuthors: Miller, E. C.; Faucher, R.; Hart, P. B.; Rincon-Sandoval, M.; Santaquiteria, A.; White, W. T.; Baldwin, C. C.; Miya, M.; Betancur-R, R.; Tornabene, L.; Evans, K.; Arcila, D.\nScore: 25.3, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564281\nColonization of a novel habitat is often followed by radiation in the wake of ecological opportunity. Alternatively, some habitats should be inherently more constraining than others if the challenges of that environment have few evolutionary solutions. We examined the push-and-pull of these factors on evolution following habitat transitions, using anglerfishes (Lophiiformes) as a model. Deep-sea fishes are notoriously difficult to study, and poor sampling has limited progress thus far. Here we present a new phylogeny of anglerfishes with unprecedented taxonomic sampling (1,092 loci and 40% of species), combined with three-dimensional phenotypic data from museum specimens obtained with micro-CT scanning. We use these datasets to examine the tempo and mode of phenotypic and lineage diversification using phylogenetic comparative methods, comparing lineages in shallow and deep benthic versus bathypelagic habitats. Our results show that anglerfishes represent a surprising case where the bathypelagic lineage has greater taxonomic and phenotypic diversity than coastal benthic relatives. This defies expectations based on ecological principles since the bathypelagic zone is the most homogeneous habitat on Earth. Deep-sea anglerfishes experienced rapid lineage diversification concomitant with colonization of the bathypelagic zone from a continental slope ancestor. They display the highest body, skull and jaw shape disparity across lophiiforms. In contrast, reef-associated taxa show strong constraints on shape and low evolutionary rates, contradicting patterns suggested by other shallow marine fishes. We found that Lophiiformes as a whole evolved under an early burst model with subclades occupying distinct body shapes. We further discuss to what extent the bathypelagic clade is a secondary adaptive radiation, or if its diversity can be explained by non-adaptive processes.\nA whole-body micro-CT scan library that captures the skeletal diversity of Lake Malawi cichlid fishes\nAuthors: Bucklow, C. V.; Genner, M. J.; Turner, G. F.; Maclaine, J.; Benson, R.; Verd, B.\nScore: 19.4, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565162\nHere we describe a dataset of freely available, readily processed, whole-body CT-scans of 56 species (116 specimens) of Lake Malawi cichlid fishes that captures a considerable majority of the morphological variation present in this remarkable adaptive radiation. We contextualise the scanned specimens within a discussion of their respective ecomorphological groupings and suggest possible macroevolutionary studies that could be conducted with these data. We also describe a methodology to efficiently CT-scan (on average) 23 specimens per hour, limiting scanning time and alleviating the financial cost whilst maintaining high resolution. We demonstrate the utility of this method by reconstructing 3D models of multiple bones from multiple specimens within the dataset. We hope this dataset will enable further morphological study of this fascinating system and permit wider-scale comparisons with other cichlid adaptive radiations.\nThe distribution of fitness effects during adaptive walks using a simple genetic network\nAuthors: O'Brien, N. L. V.; Holland, B.; Engelstaedter, J.; Ortiz-Barrientos, D.\nScore: 17.2, Published: 2023-10-31 DOI: 10.1101/2023.10.26.564303\nThe tempo and mode of adaptation is dependent on the availability of beneficial alleles. Genetic interactions arising from gene networks can restrict this availability. However, the extent to which networks affect adaptation remains inconclusive. Current models of evolution consider additive genotype-phenotype relationships while often ignoring the contribution to phenotypic variance arising from gene interactions. In this study, we present a novel approach to modeling quantitative traits as products of genetic networks via systems of ordinary differential equations so we can mechanistically explore the effects of network structures on adaptation. We present the first step of this approach by studying a simple gene regulatory network, the negative autoregulation motif (NAR). Using forward-time genetic simulations, we measure adaptive walks towards a phenotypic optimum in both additive and NAR models. A key expectation from adaptive walk theory is that the distribution of fitness effects (DFE) of new beneficial mutations is exponential. We found that although both models strayed from expectation, harboring fewer beneficial mutations than expected, they took a similar number of adaptive steps to reach the optimum. NAR populations were 10% less likely to reach the optimum and took longer to adapt. This was partly due to a complex and largely deleterious DFE from new mutations, and by requiring coordinated changes among its molecular components. This behavior is reminiscent of the cost of complexity, where correlations among traits constrain adaptation. Our results suggest that the interactions emerging from genetic networks can hinder adaptation by generating complex and multi-modal distributions of fitness effects.\nThe brittle star genome illuminates the genetic basis of animal appendage regeneration\nAuthors: Parey, E.; Ortega-Martinez, O.; Delroisse, J.; Piovani, L.; Czarkwiani, A.; Dylus, D.; Arya, S.; Dupont, S.; Thorndyke, M.; Larsson, T.; Johannesson, K.; Buckley, K. M.; Martinez, P.; Oliveri, P.; Marletaz, F.\nScore: 16.9, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564762\nSpecies within nearly all extant animal lineages are capable of regenerating body parts. However, it remains unclear whether the gene expression programme controlling regeneration is evolutionarily conserved. Brittle stars are a species-rich class of echinoderms with outstanding regenerative abilities, but investigations into the genetic bases of regeneration in this group have been hindered by the limited available genomic resources. Here, we report a chromosome-scale genome assembly for the brittle star Amphiura filiformis. We show that the brittle star displays the most rearranged genome amongst echinoderms sequenced to date, featuring a reorganised Hox cluster reminiscent of the rearrangements observed in sea urchins. In addition, we performed an extensive profiling of gene expression throughout brittle star adult arm regeneration and identified sequential waves of gene expression governing wound healing, proliferation and differentiation. We conducted comparative transcriptomic analyses with other invertebrate and vertebrate models for appendage regeneration and uncovered hundreds of genes with conserved expression dynamics, notably during the proliferative phase of regeneration. Our findings emphasise the crucial importance of echinoderms to detect long-range expression conservation between vertebrates and classical invertebrate regeneration model systems.\nExperimental evolution of multicellularity via cuboidal cell packing in fission yeast\nAuthors: Pineau, R.; Kahn, P. C.; Lac, D. T.; Denning, M.; Wong, W.; Ratcliff, W.; Bozdag, G. O.\nScore: 14.8, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565576\nThe evolution of multicellularity represents a major transition in life's history, enabling the rise of complex organisms. Multicellular groups can evolve through multiple developmental modes, but a common step is the formation of permanent cell-cell attachments after division. The characteristics of the multicellular morphology which emerges has profound consequences for the subsequent evolution of a nascent multicellular lineage, but little prior work has examined these dynamics directly. Here we examine a widespread yet understudied emergent multicellular morphology: cuboidal packing. Extinct and extant multicellular organisms across the tree of life have evolved to form groups in which spherical cells divide but remain attached, forming approximately cubic subunits. To experimentally investigate the evolution of cuboidal cell packing, we used settling selection to favor the evolution of simple multicellularity in unicellular, spherical Schizosaccharomyces pombe yeast. Multicellular clusters with cuboidal organization rapidly evolved, displacing the unicellular ancestor. These clusters displayed key hallmarks of an evolutionary transition in individuality: groups possess an emergent life cycle driven by physical fracture, group size is heritable, and they respond to group-level selection via multicellular adaptation. In 2/5 lineages, group formation was driven by mutations in the ACE2 gene, preventing daughter cell separation after division. Remarkably, ACE2 mutations also underlie the transition to multicellularity in Saccharomyces cerevisiae and C. galabrata, lineages last shared a common ancestor \u003e300 million years ago. Our results provide insight into the evolution of cuboidal cell packing, an understudied multicellular morphology, and highlights the deeply convergent potential for a transition to multicellular individuality within fungi.\nGenome of the glacier alga Ancylonema and its insights into the evolution of streptophyte life on ice and land\nAuthors: Bowles, A. M. C.; Williams, T. A.; Donoghue, P. C. J.; Campbell, D. A.; Williamson, C. J.\nScore: 14.3, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563300\nContemporary glaciers and ice sheets are home to communities of streptophyte glacier algae that must balance their requirements for photosynthesis and growth with tolerance of extremes in temperature, desiccation and UV radiation. These same environmental challenges have been hypothesized as the driving force behind the evolution of land plants from streptophyte algal ancestors in the Cryogenian (720-635 million years ago). Here, we sequence, assemble and analyze the metagenome-assembled genome (MAG) of the glacier alga Ancylonema nordenskioldii to investigate its adaptations to life in ice, and whether this represents a vestige of Cryogenian anydrophyte exaptations. Phylogenetic analysis confirms the placement of glacier algae within the sister lineage to land plants, Zygnematophyceae. The MAG is characterized by an expansion of genes involved in high irradiance and UV light tolerance, whilst lineage-specific diversification led to the novel screening pigmentation of glacier algae. We found no support for the hypothesis of a common genomic basis for adaptations to ice and to land in streptophytes. Comparative genomic analysis revealed that reductive genome evolution in the ancestor of Zygnematophyceae correlates with their reductive morphological evolution. This first genome-scale data for glacier algae supports an Ancylonema-specific adaptation to the cryosphere, as well as shedding light on the genome evolution of land plants and Zygnematophyceae.\n",
  "wordCount" : "2674",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.21.541643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.21.541643" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.21.541643">
        <p class="paperTitle">Fundamental equations linking methylation dynamics to maximum lifespan in mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.21.541643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.21.541643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horvath, S.; Zhang, J.; Haghani, A.; Lu, A. T.; Fei, Z.</p>
        <p class="info">Score: 79.8, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.21.541643' target='https://doi.org/10.1101/2023.05.21.541643'> 10.1101/2023.05.21.541643</a></p>
        <p class="abstract">We establish the mathematical foundation that links the rate of change in any molecular biomarker to species lifespan. Specifically, we propose a robust approach that identifies the strong inverse relationship for certain biomarkers using two comprehensive methylation datasets. After examining 54 chromatin states, we found the rates of change of CpG sites in bivalent chromatin states are negatively associated to the lifespans of 90 dog breeds in the first dataset, and the discoveries are further strengthened with 125 mammalian species in the second dataset. Our research leads to three key findings: First, a reciprocal relationship exists between the average rate of methylation change (AROCM) in bivalent promoter regions and maximum lifespan: AROCM ~ 1/MaxLifespan. Second, the correlation between age and average methylation bears no relation to maximum lifespan, Cor(Methly,Age) independent of MaxLifespan. Third, the rate of methylation change in young animals is related to that in old animals: Young animals&#39; AROCM ~ Old AROCM. These findings hinge on the chromatin context, as different results emerge when defining AROCM using different chromatin states. Our analytical framework is versatile and readily extendable to a broad range of other molecular assessments. Overall, our study demonstrates that epigenetic aging rates in specific chromatin states exhibit an inverse relationship with maximum lifespan in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565210">
        <p class="paperTitle">Complete phylogeny of Micrathena spiders suggests multiple dispersal events among Neotropical rainforests, islands, and landmasses, and indicates Andean orogeny promotes speciation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Magalhaes, I. L. F.; Martins, P. H.; Faleiro, B. T.; Vidigal, T. H. D. A.; Santos, F. R.; Carvalho, L. S.; Santos, A. J.</p>
        <p class="info">Score: 68.5, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565210' target='https://doi.org/10.1101/2023.11.01.565210'> 10.1101/2023.11.01.565210</a></p>
        <p class="abstract">The Neotropical region is the most diverse on the planet, largely due to its mosaic of tropical rainforests. Multiple tectonic and climatic processes have been hypothesized to play a role in generating this diversity; these include Andean orogeny, closure of the Panama Isthmus, the hypothesized GAARlandia land bridge, and putative historical connections among currently isolated forests. Micrathena spiny spiders (Araneidae) include [~]120 species distributed mostly in forests from Mexico to Argentina, including the Antilles. Here, we use it as a model to study the biogeographic history of Neotropical rainforests by estimating a complete, dated phylogeny using morphological data for all species and molecular data for a subsample of 79 species. This resulted in a phylogeny that is mostly robust and supports most of the previously recognized species groups, although with uncertainty in the phylogenetic position of some species, especially those lacking sequences. The genus began diversifying about 25 million years ago. We use an event-based approach and biogeographic stochastic mapping to estimate ancestral distributions and the timing and direction of dispersal events, and to identify areas where diversity was generated, while accounting for phylogenetic uncertainty. Andean cloud forests generated the majority of species through in-situ speciation, but the Amazon was the major source of species for adjacent areas through dispersal; on the other extreme, the Dry Diagonal received species from other areas but generated very little diversity. Species exchange between Central and South America was intense, with [~]23 dispersal events beginning at least 20 million years ago, indicating that Micrathena dispersed between these continents before closure of the Isthmus of Panama. We also inferred [~]4 over-water dispersal events from Central and North America to the Antilles, which happened in the last 20 million years, and thus much after the proposed age of the GAARlandia land bridge. We identified an important species exchange route between the Amazon and the Atlantic forests. Sampling all species of the genus was fundamental to some of the conclusions above, especially in identifying the Andes as the area that generated the majority of species. This study highlights the importance of a solid and complete taxonomic sampling in biogeographic studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564725">
        <p class="paperTitle">Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.</p>
        <p class="info">Score: 30.6, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564725' target='https://doi.org/10.1101/2023.10.30.564725'> 10.1101/2023.10.30.564725</a></p>
        <p class="abstract">During the last few centuries D. melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D. melanogaster. It is a 5216nt LTR retrotransposon of the Ty3/gypsy superfamily. Relying on strains sampled at different times during the last century we show that Spoink invaded worldwide D. melanogaster populations after the P-element between 1983 and 1993. This invasion was likely triggered by a horizontal transfer from the D. willistoni group, much as the P-element. Spoink is probably silenced by the piRNA pathway in natural populations and about 1/3 of the examined strains have an insertion into a canonical piRNA cluster such as 42AB. Given the degree of genetic investigation of D. melanogaster it is surprising that Spoink was able to invade unnoticed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564347">
        <p class="paperTitle">From museum drawer to tree: historical DNA phylogenomics clarifies the systematics of rare dung beetles (Coleoptera: Scarabaeinae) from museum collections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopes, F.; Gunter, N.; Gillett, C. P. D. T.; Montanaro, G.; Rossini, M.; Losacco, F.; Daniel, G. M.; Straube, N.; Tarasov, S.</p>
        <p class="info">Score: 27.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564347' target='https://doi.org/10.1101/2023.10.27.564347'> 10.1101/2023.10.27.564347</a></p>
        <p class="abstract">Although several methods exist for extracting and sequencing historical DNA originating from drypreserved insect specimens deposited in natural history museums, no consensus exists as to what is the optimal approach. We demonstrate that a customized, low-cost archival DNA extraction protocol ([~] {euro}10 per sample), in combination with Ultraconserved Elements (UCEs), is an effective tool for insect phylogenomic studies. We successfully tested our approach by sequencing DNA from scarab dung beetles preserved in both wet and dry collections, including unique primary type and rare historical specimens from internationally important natural history museums in London, Paris and Helsinki. The focal specimens comprise enigmatic dung beetle genera (Nesosisyphus, Onychotechus and Helictopleurus) that varied in age and preservation. The oldest specimen, the holotype of the now possibly extinct Mauritian endemic Nesosisyphus rotundatus, was collected in 1944. We obtained high-quality DNA from all studied specimens to enable the generation of a UCE-based dataset that revealed an insightful and well-supported phylogenetic tree of dung beetles. The resulting phylogeny suggested the reclassification of Onychotechus (previously incertae sedis) within the tribe Coprini. Our approach demonstrates the feasibility and effectiveness of combining DNA data from historic and recent museum specimens to provide novel insights. The proposed archival DNA protocol is available at DOI 10.17504/protocols.io.81wgbybqyvpk/v1

HighlightsO_LIWe combined custom low-cost archival DNA extractions and Ultraconserved Element phylogenomics
C_LIO_LIDNA from rare museum specimens of enigmatic dung beetles revealed their phylogenetic connections
C_LIO_LIGenomic data was obtained from the holotype of a potentially extinct monoinsular endemic species
C_LIO_LIGenomic data allowed a rare and enigmatic species of previously unknown affinity to be classified
C_LIO_LIThe morphology of museum specimens remained intact following non-destructive DNA extraction
C_LI



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=80 SRC=&#34;FIGDIR/small/564347v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (24K):
org.highwire.dtl.DTLVardef@cdc35corg.highwire.dtl.DTLVardef@d4feaaorg.highwire.dtl.DTLVardef@19b7e9dorg.highwire.dtl.DTLVardef@195a467_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564281">
        <p class="paperTitle">Phylogenomics reveals the deep ocean as an accelerator for evolutionary diversification in anglerfishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, E. C.; Faucher, R.; Hart, P. B.; Rincon-Sandoval, M.; Santaquiteria, A.; White, W. T.; Baldwin, C. C.; Miya, M.; Betancur-R, R.; Tornabene, L.; Evans, K.; Arcila, D.</p>
        <p class="info">Score: 25.3, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564281' target='https://doi.org/10.1101/2023.10.26.564281'> 10.1101/2023.10.26.564281</a></p>
        <p class="abstract">Colonization of a novel habitat is often followed by radiation in the wake of ecological opportunity. Alternatively, some habitats should be inherently more constraining than others if the challenges of that environment have few evolutionary solutions. We examined the push-and-pull of these factors on evolution following habitat transitions, using anglerfishes (Lophiiformes) as a model. Deep-sea fishes are notoriously difficult to study, and poor sampling has limited progress thus far. Here we present a new phylogeny of anglerfishes with unprecedented taxonomic sampling (1,092 loci and 40% of species), combined with three-dimensional phenotypic data from museum specimens obtained with micro-CT scanning. We use these datasets to examine the tempo and mode of phenotypic and lineage diversification using phylogenetic comparative methods, comparing lineages in shallow and deep benthic versus bathypelagic habitats. Our results show that anglerfishes represent a surprising case where the bathypelagic lineage has greater taxonomic and phenotypic diversity than coastal benthic relatives. This defies expectations based on ecological principles since the bathypelagic zone is the most homogeneous habitat on Earth. Deep-sea anglerfishes experienced rapid lineage diversification concomitant with colonization of the bathypelagic zone from a continental slope ancestor. They display the highest body, skull and jaw shape disparity across lophiiforms. In contrast, reef-associated taxa show strong constraints on shape and low evolutionary rates, contradicting patterns suggested by other shallow marine fishes. We found that Lophiiformes as a whole evolved under an early burst model with subclades occupying distinct body shapes. We further discuss to what extent the bathypelagic clade is a secondary adaptive radiation, or if its diversity can be explained by non-adaptive processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565162">
        <p class="paperTitle">A whole-body micro-CT scan library that captures the skeletal diversity of Lake Malawi cichlid fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bucklow, C. V.; Genner, M. J.; Turner, G. F.; Maclaine, J.; Benson, R.; Verd, B.</p>
        <p class="info">Score: 19.4, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565162' target='https://doi.org/10.1101/2023.11.01.565162'> 10.1101/2023.11.01.565162</a></p>
        <p class="abstract">Here we describe a dataset of freely available, readily processed, whole-body CT-scans of 56 species (116 specimens) of Lake Malawi cichlid fishes that captures a considerable majority of the morphological variation present in this remarkable adaptive radiation. We contextualise the scanned specimens within a discussion of their respective ecomorphological groupings and suggest possible macroevolutionary studies that could be conducted with these data. We also describe a methodology to efficiently CT-scan (on average) 23 specimens per hour, limiting scanning time and alleviating the financial cost whilst maintaining high resolution. We demonstrate the utility of this method by reconstructing 3D models of multiple bones from multiple specimens within the dataset. We hope this dataset will enable further morphological study of this fascinating system and permit wider-scale comparisons with other cichlid adaptive radiations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564303">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564303" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564303">
        <p class="paperTitle">The distribution of fitness effects during adaptive walks using a simple genetic network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564303" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564303" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Brien, N. L. V.; Holland, B.; Engelstaedter, J.; Ortiz-Barrientos, D.</p>
        <p class="info">Score: 17.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564303' target='https://doi.org/10.1101/2023.10.26.564303'> 10.1101/2023.10.26.564303</a></p>
        <p class="abstract">The tempo and mode of adaptation is dependent on the availability of beneficial alleles. Genetic interactions arising from gene networks can restrict this availability. However, the extent to which networks affect adaptation remains inconclusive. Current models of evolution consider additive genotype-phenotype relationships while often ignoring the contribution to phenotypic variance arising from gene interactions. In this study, we present a novel approach to modeling quantitative traits as products of genetic networks via systems of ordinary differential equations so we can mechanistically explore the effects of network structures on adaptation. We present the first step of this approach by studying a simple gene regulatory network, the negative autoregulation motif (NAR). Using forward-time genetic simulations, we measure adaptive walks towards a phenotypic optimum in both additive and NAR models. A key expectation from adaptive walk theory is that the distribution of fitness effects (DFE) of new beneficial mutations is exponential. We found that although both models strayed from expectation, harboring fewer beneficial mutations than expected, they took a similar number of adaptive steps to reach the optimum. NAR populations were 10% less likely to reach the optimum and took longer to adapt. This was partly due to a complex and largely deleterious DFE from new mutations, and by requiring coordinated changes among its molecular components. This behavior is reminiscent of the cost of complexity, where correlations among traits constrain adaptation. Our results suggest that the interactions emerging from genetic networks can hinder adaptation by generating complex and multi-modal distributions of fitness effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564762">
        <p class="paperTitle">The brittle star genome illuminates the genetic basis of animal appendage regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parey, E.; Ortega-Martinez, O.; Delroisse, J.; Piovani, L.; Czarkwiani, A.; Dylus, D.; Arya, S.; Dupont, S.; Thorndyke, M.; Larsson, T.; Johannesson, K.; Buckley, K. M.; Martinez, P.; Oliveri, P.; Marletaz, F.</p>
        <p class="info">Score: 16.9, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564762' target='https://doi.org/10.1101/2023.10.30.564762'> 10.1101/2023.10.30.564762</a></p>
        <p class="abstract">Species within nearly all extant animal lineages are capable of regenerating body parts. However, it remains unclear whether the gene expression programme controlling regeneration is evolutionarily conserved. Brittle stars are a species-rich class of echinoderms with outstanding regenerative abilities, but investigations into the genetic bases of regeneration in this group have been hindered by the limited available genomic resources. Here, we report a chromosome-scale genome assembly for the brittle star Amphiura filiformis. We show that the brittle star displays the most rearranged genome amongst echinoderms sequenced to date, featuring a reorganised Hox cluster reminiscent of the rearrangements observed in sea urchins. In addition, we performed an extensive profiling of gene expression throughout brittle star adult arm regeneration and identified sequential waves of gene expression governing wound healing, proliferation and differentiation. We conducted comparative transcriptomic analyses with other invertebrate and vertebrate models for appendage regeneration and uncovered hundreds of genes with conserved expression dynamics, notably during the proliferative phase of regeneration. Our findings emphasise the crucial importance of echinoderms to detect long-range expression conservation between vertebrates and classical invertebrate regeneration model systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565576">
        <p class="paperTitle">Experimental evolution of multicellularity via cuboidal cell packing in fission yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pineau, R.; Kahn, P. C.; Lac, D. T.; Denning, M.; Wong, W.; Ratcliff, W.; Bozdag, G. O.</p>
        <p class="info">Score: 14.8, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565576' target='https://doi.org/10.1101/2023.11.03.565576'> 10.1101/2023.11.03.565576</a></p>
        <p class="abstract">The evolution of multicellularity represents a major transition in life&#39;s history, enabling the rise of complex organisms. Multicellular groups can evolve through multiple developmental modes, but a common step is the formation of permanent cell-cell attachments after division. The characteristics of the multicellular morphology which emerges has profound consequences for the subsequent evolution of a nascent multicellular lineage, but little prior work has examined these dynamics directly. Here we examine a widespread yet understudied emergent multicellular morphology: cuboidal packing. Extinct and extant multicellular organisms across the tree of life have evolved to form groups in which spherical cells divide but remain attached, forming approximately cubic subunits. To experimentally investigate the evolution of cuboidal cell packing, we used settling selection to favor the evolution of simple multicellularity in unicellular, spherical Schizosaccharomyces pombe yeast. Multicellular clusters with cuboidal organization rapidly evolved, displacing the unicellular ancestor. These clusters displayed key hallmarks of an evolutionary transition in individuality: groups possess an emergent life cycle driven by physical fracture, group size is heritable, and they respond to group-level selection via multicellular adaptation. In 2/5 lineages, group formation was driven by mutations in the ACE2 gene, preventing daughter cell separation after division. Remarkably, ACE2 mutations also underlie the transition to multicellularity in Saccharomyces cerevisiae and C. galabrata, lineages last shared a common ancestor &gt;300 million years ago. Our results provide insight into the evolution of cuboidal cell packing, an understudied multicellular morphology, and highlights the deeply convergent potential for a transition to multicellular individuality within fungi.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563300">
        <p class="paperTitle">Genome of the glacier alga Ancylonema and its insights into the evolution of streptophyte life on ice and land</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bowles, A. M. C.; Williams, T. A.; Donoghue, P. C. J.; Campbell, D. A.; Williamson, C. J.</p>
        <p class="info">Score: 14.3, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563300' target='https://doi.org/10.1101/2023.10.20.563300'> 10.1101/2023.10.20.563300</a></p>
        <p class="abstract">Contemporary glaciers and ice sheets are home to communities of streptophyte glacier algae that must balance their requirements for photosynthesis and growth with tolerance of extremes in temperature, desiccation and UV radiation. These same environmental challenges have been hypothesized as the driving force behind the evolution of land plants from streptophyte algal ancestors in the Cryogenian (720-635 million years ago). Here, we sequence, assemble and analyze the metagenome-assembled genome (MAG) of the glacier alga Ancylonema nordenskioldii to investigate its adaptations to life in ice, and whether this represents a vestige of Cryogenian anydrophyte exaptations. Phylogenetic analysis confirms the placement of glacier algae within the sister lineage to land plants, Zygnematophyceae. The MAG is characterized by an expansion of genes involved in high irradiance and UV light tolerance, whilst lineage-specific diversification led to the novel screening pigmentation of glacier algae. We found no support for the hypothesis of a common genomic basis for adaptations to ice and to land in streptophytes. Comparative genomic analysis revealed that reductive genome evolution in the ancestor of Zygnematophyceae correlates with their reductive morphological evolution. This first genome-scale data for glacier algae supports an Ancylonema-specific adaptation to the cryosphere, as well as shedding light on the genome evolution of land plants and Zygnematophyceae.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
