<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="50,000 years of Evolutionary History of India: Insights from ~2,700 Whole Genome Sequences
Authors: Kerdoncuff, E.; Skov, L.; Patterson, N.; Zhao, W.; Lueng, Y. Y.; Schellenberg, G. D.; Smith, J. A.; Dey, S.; Ganna, A.; Dey, A.; Kardia, S. L. R.; Lee, J.; Moorjani, P.
Score: 116.3, Published: 2024-02-17 DOI: 10.1101/2024.02.15.580575
India has been underrepresented in whole genome sequencing studies. We generated 2,762 high coverage genomes from India - including individuals from most geographic regions, speakers of all major languages, and tribal and caste groups - providing a comprehensive survey of genetic variation in India.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="50,000 years of Evolutionary History of India: Insights from ~2,700 Whole Genome Sequences
Authors: Kerdoncuff, E.; Skov, L.; Patterson, N.; Zhao, W.; Lueng, Y. Y.; Schellenberg, G. D.; Smith, J. A.; Dey, S.; Ganna, A.; Dey, A.; Kardia, S. L. R.; Lee, J.; Moorjani, P.
Score: 116.3, Published: 2024-02-17 DOI: 10.1101/2024.02.15.580575
India has been underrepresented in whole genome sequencing studies. We generated 2,762 high coverage genomes from India - including individuals from most geographic regions, speakers of all major languages, and tribal and caste groups - providing a comprehensive survey of genetic variation in India." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="50,000 years of Evolutionary History of India: Insights from ~2,700 Whole Genome Sequences
Authors: Kerdoncuff, E.; Skov, L.; Patterson, N.; Zhao, W.; Lueng, Y. Y.; Schellenberg, G. D.; Smith, J. A.; Dey, S.; Ganna, A.; Dey, A.; Kardia, S. L. R.; Lee, J.; Moorjani, P.
Score: 116.3, Published: 2024-02-17 DOI: 10.1101/2024.02.15.580575
India has been underrepresented in whole genome sequencing studies. We generated 2,762 high coverage genomes from India - including individuals from most geographic regions, speakers of all major languages, and tribal and caste groups - providing a comprehensive survey of genetic variation in India."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "50,000 years of Evolutionary History of India: Insights from ~2,700 Whole Genome Sequences\nAuthors: Kerdoncuff, E.; Skov, L.; Patterson, N.; Zhao, W.; Lueng, Y. Y.; Schellenberg, G. D.; Smith, J. A.; Dey, S.; Ganna, A.; Dey, A.; Kardia, S. L. R.; Lee, J.; Moorjani, P.\nScore: 116.3, Published: 2024-02-17 DOI: 10.1101/2024.02.15.580575\nIndia has been underrepresented in whole genome sequencing studies. We generated 2,762 high coverage genomes from India - including individuals from most geographic regions, speakers of all major languages, and tribal and caste groups - providing a comprehensive survey of genetic variation in India.",
  "keywords": [
    
  ],
  "articleBody": " 50,000 years of Evolutionary History of India: Insights from ~2,700 Whole Genome Sequences\nAuthors: Kerdoncuff, E.; Skov, L.; Patterson, N.; Zhao, W.; Lueng, Y. Y.; Schellenberg, G. D.; Smith, J. A.; Dey, S.; Ganna, A.; Dey, A.; Kardia, S. L. R.; Lee, J.; Moorjani, P.\nScore: 116.3, Published: 2024-02-17 DOI: 10.1101/2024.02.15.580575\nIndia has been underrepresented in whole genome sequencing studies. We generated 2,762 high coverage genomes from India - including individuals from most geographic regions, speakers of all major languages, and tribal and caste groups - providing a comprehensive survey of genetic variation in India. With these data, we reconstruct the evolutionary history of India through space and time at fine scales. We show that most Indians derive ancestry from three ancestral groups related to ancient Iranian farmers, Eurasian Steppe pastoralists and South Asian hunter-gatherers. We uncover a common source of Iranian-related ancestry from early Neolithic cultures of Central Asia into the ancestors of Ancestral South Indians (ASI), Ancestral North Indians (ANI), Austro-asiatic-related and East Asian-related groups in India. Following these admixtures, India experienced a major demographic shift towards endogamy, resulting in extensive homozygosity and identity-by-descent sharing among individuals. At deep time scales, Indians derive around 1-2% of their ancestry from gene flow from archaic hominins, Neanderthals and Denisovans. By assembling the surviving fragments of archaic ancestry in modern Indians, we recover ~1.5 Gb (or 50%) of the introgressing Neanderthal and ~0.6 Gb (or 20%) of the introgressing Denisovan genomes, more than any other previous archaic ancestry study. Moreover, Indians have the largest variation in Neanderthal ancestry, as well as the highest amount of population-specific Neanderthal segments among worldwide groups. Finally, we demonstrate that most of the genetic variation in Indians stems from a single major migration out of Africa that occurred around 50,000 years ago, with minimal contribution from earlier migration waves. Together, these analyses provide a detailed view of the population history of India and underscore the value of expanding genomic surveys to diverse groups outside Europe.\nEffect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 174.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.\nUnselfish meiotic drive maintains heterozygosity in a parthenogenetic ant\nAuthors: Lacy, K. D.; Hart, T.; Kronauer, D. J. C.\nScore: 37.6, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579553\nAccording to Mendels second law, chromosomes segregate randomly in meiosis. Non-random segregation is primarily known for cases of selfish meiotic drive in females, in which particular alleles bias their own transmission into the oocyte1,2. Here, we report a rare example of unselfish meiotic drive for crossover inheritance in the clonal raider ant, Ooceraea biroi. This species produces diploid offspring parthenogenetically via fusion of two haploid nuclei from the same meiosis3. This process should cause rapid genotypic degeneration due to loss of heterozygosity, which results if crossover recombination is followed by random (Mendelian) segregation of chromosomes4,5. However, by comparing whole genomes of mothers and daughters, we show that loss of heterozygosity is exceedingly rare, raising the possibility that crossovers are infrequent or absent in O. biroi meiosis. Using a combination of cytology and whole genome sequencing, we show that crossover recombination is, in fact, common, but that loss of heterozygosity is avoided because crossover products are faithfully co-inherited. This results from a programmed violation of Mendels law of segregation, such that crossover products segregate together rather than randomly. This discovery highlights an extreme example of cellular \"memory\" of crossovers, which could be a common yet cryptic feature of chromosomal segregation.\nMultiple horizontal mini-chromosome transfers drive genome evolution of clonal blast fungus lineages\nAuthors: Barragan, A. C.; Latorre, S. M.; Malmgren, A.; Harant, A.; Win, J.; Sugihara, Y.; Burbano, H. A.; Kamoun, S.; Langner, T.\nScore: 36.9, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580079\nCrop disease pandemics are often driven by clonal lineages of plant pathogens that reproduce asexually. How these clonal pathogens continuously adapt to their hosts despite harboring limited genetic variation, and in absence of sexual recombination remains elusive. Here, we reveal multiple instances of horizontal chromosome transfer within pandemic clonal lineages of the blast fungus Magnaporthe (Syn. Pyricularia) oryzae. We identified a horizontally transferred 1.2Mb supernumerary mini-chromosome which is remarkably conserved between M. oryzae isolates from both the rice blast fungus lineage and the lineage infecting Indian goosegrass (Eleusine indica), a wild grass that often grows in the proximity of cultivated cereal crops. Furthermore, we show that this mini-chromosome was horizontally acquired by clonal rice blast isolates through at least nine distinct transfer events over the past three centuries. These findings establish horizontal mini-chromosome transfer as a mechanism facilitating genetic exchange among different host-associated blast fungus lineages. We propose that blast fungus populations infecting wild grasses act as genetic reservoirs that drive genome evolution of pandemic clonal lineages that afflict cereal crops.\nA critical reexamination of recovered SARS-CoV-2 sequencing data\nAuthors: Debarre, F.; Hensel, Z.\nScore: 33.8, Published: 2024-02-16 DOI: 10.1101/2024.02.15.580500\nSARS-CoV-2 genomes collected at the onset of the Covid-19 pandemic are valuable because they could help understand how the virus entered the human population. In 2021, Jesse Bloom reported on the recovery of a sequencing dataset that had been removed from the NCBI SRA database at the request of the data generators, a scientific team at Wuhan University. Bloom suggested that the data may have been removed in order to obfuscate the origin of SARS-CoV-2, questioning the generating authors' statements that the samples had been collected on and after January 30, 2020. Here, we show that sample collection dates were published in 2020 together with the sequencing data, and match the dates given by the authors in 2021. We examine mutations in these sequences and confirm that they are entirely consistent with the previously known genetic diversity of SARS-CoV-2 of late January 2020. Finally, we explain how an apparent phylogenetic rooting paradox described by Bloom was resolved by subsequent analysis. Our reanalysis demonstrates that allegations of cover-up or metadata manipulation were unwarranted.\nA long non-coding RNA at the cortex locus controls adaptive colouration in butterflies\nAuthors: Livraghi, L.; Hanly, J. J.; Evans, E.; Wright, C. J.; Loh, L. S.; Mazo-Vargas, A.; Kamrava, K.; Carter, A. T.; van der Heijden, E. S.; Reed, R. D.; Papa, R.; Jiggins, C. D.; Martin, A.\nScore: 27.2, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579710\nEvolutionary variation in the wing pigmentation of butterflies and moths offers striking examples of adaptation by crypsis and mimicry. The cortex locus has been independently mapped as the locus controlling colour polymorphisms in 14 lepidopteran species, suggesting it acts as a genomic hotspot for the diversification of wing patterns, but functional validation through protein-coding knockouts has proven difficult to obtain. Our study unveils the role of a novel long non-coding RNA (lncRNA) which we name ivory, transcribed from the cortex locus, in modulating colour patterning in butterflies. Strikingly, ivory expression prefigures most melanic patterns during pupal development, suggesting an early developmental role in specifying scale identity. To test this, we generated CRISPR mosaic knock-outs in five nymphalid butterfly species and show that ivory mutagenesis yields transformations of dark pigmented scales into white or light-coloured scales. Genotyping of Vanessa cardui germline mutants associates these phenotypes to small on-target deletions at the conserved first exon of ivory. In contrast, cortex germline mutant butterflies with confirmed null alleles lack any wing phenotype, and exclude a colour patterning role for this adjacent gene. Overall, these results show that a lncRNA acts as a master switch of colour pattern specification, and played key roles in the adaptive diversification of colour patterns in butterflies. Significance statementDeciphering the genetic underpinnings of adaptive variation is fundamental for a comprehensive understanding of evolutionary processes. Long non-coding RNAs (lncRNAs) represent an emerging category of genetic modulators within the genome, yet they have been overlooked as a source of phenotypic diversity. In this study, we unveil the pivotal role of a lncRNA in orchestrating colour transitions between dark and light patterns during butterfly wing development. Remarkably, this lncRNA gene is nested within the cortex locus, a genetic region known to control multiple cases of adaptive variation in butterflies and moths, including iconic examples of natural selection. These findings highlight the significant influence of lncRNAs in developmental regulation, and also underscore their potential as key genetic players in the evolutionary process itself.\nCompensatory mutations potentiate constructive neutral evolution by gene duplication\nAuthors: Despres, P. C.; Dube, A. K.; Grenier, J.; Picard, M.-E.; Shi, R.; Landry, C. R.\nScore: 26.8, Published: 2024-02-13 DOI: 10.1101/2024.02.12.579783\nProtein functions generally depend on their assembly into complexes. During evolution, some complexes have transitioned from homomers encoded by a single gene to heteromers encoded by duplicate genes. This transition could occur without adaptive evolution through intermolecular compensatory mutations. Here, we experimentally duplicate and evolve an homodimeric enzyme to examine if and how this could happen. We identify hundreds of deleterious mutations that inactivate individual homodimers but produce functional enzymes when co-expressed as duplicated proteins that heterodimerize. The structure of one such heteromer reveals how both losses of function are buffered through the introduction of asymmetry in the complex that allows them to subfunctionalize. Constructive neutral evolution can thus occur by gene duplication followed by only one deleterious mutation per duplicate. One sentence summaryCompensatory deleterious mutations entangle gene duplicates\nFusion, fission, and scrambling of the bilaterian genome in Bryozoa\nAuthors: Lewin, T. D.; Liao, I. J.-Y.; Chen, M.-E.; Bishop, J. D. D.; Holland, P. W. H.; Luo, Y.-J.\nScore: 25.3, Published: 2024-02-16 DOI: 10.1101/2024.02.15.580425\nGenes are commonly found together on the same chromosome over vast evolutionary distances. This extensive physical gene linkage, known as macrosynteny, can be seen between bilaterian phyla as divergent as Chordata, Echinodermata, Mollusca, and Nemertea and likely reflects the importance of genome organization to gene regulatory landscapes. Here, we report a very different pattern of genome evolution in Bryozoa, an understudied yet ecologically important phylum of colonial invertebrates. Using comparative genomics, we reconstruct the chromosomal evolutionary history of five bryozoans. We infer the ancestral bryozoan genome organization and identify multiple ancient chromosome fusions followed by gene mixing, leading to the near-complete loss of bilaterian linkage groups. A second wave of rearrangements, including chromosome fission, occurred independently in two bryozoan classes, further shuffling bryozoan genomes. We also discover nine chromosomal fusion events shared between bryozoans and brachiopods, supporting the traditional yet highly debated Lophophorata hypothesis. Finally, we show that chromosome fusion and fission processes led to the separation of bryozoan Hox clusters. These findings demonstrate how the canonical bilaterian genome structure has been lost across an entire phylum, suggest a modification to gene regulatory landscapes, and provide a powerful source of phylogenetic information.\nPhylotranscriptomics reveals convergent behavioral evolution associated with shared and unique mechanisms in cavity-nesting songbirds\nAuthors: Lipshutz, S. E.; Hibbins, M. S.; Bentz, A. B.; Buechlin, A. M.; Empson, T. A.; George, E. M.; Hauber, M. E.; Rusch, D. B.; Schelsky, W. M.; Thomas, Q. K.; Torneo, S. J.; Turner, A. M.; Wolf, S. E.; Woodruff, M. J.; Hahn, M. W.; Rosvall, K. A.\nScore: 24.3, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580205\nUncovering the genomic bases of phenotypic adaptation is a major goal in biology, but this has been hard to achieve for complex behavioral traits. Here, we leverage the repeated, independent evolution of obligate cavity-nesting in birds to test the hypothesis that the shared pressure to compete for a limited breeding resource drives convergent behavioral evolution via convergent gene regulatory changes in the brain. Using behavioral assays in the field, hormonal measures of free-living subjects, and transcriptome-wide analyses of the brain in wild-captured males and females, we examined five species pairs across five avian families, each including one obligate cavity-nesting species and a related species with an open cup-nesting or otherwise more flexible nest strategy. Results support the hypothesis of behavioral convergence, with higher levels of territorial aggression in obligate cavity-nesters, particularly among females. Levels of testosterone in circulation were not associated with nest strategy or aggression for either sex, but phylogenetic analyses of individual genes and co-regulated gene networks revealed some shared patterns of gene expression, including a nest strategy-related gene network shared across families and two separate networks linked to aggression only in females. Though associated with convergent behavioral evolution, these genes were not significantly enriched for particular functional pathways, and the scope of convergent gene expression evolution was limited to a small percent of the genome. Together, these observations indicate that replicated evolutionary changes in complex behavior arise via a combination of convergent and lineage-specific evolution of gene regulation.\nChronospaces: an R package for the statistical exploration of divergence times reveals extreme dependence on molecular clocks and gene choice\nAuthors: Mongiardino Koch, N.; Milla Carmona, P.\nScore: 23.2, Published: 2024-02-08 DOI: 10.1101/2024.02.04.578835\nMuch of our understanding of the history of life hinges upon time calibration, the process of assigning absolute times to cladogenetic events. Bayesian approaches to time scaling phylogenetic trees have dramatically grown in complexity, and hinge today upon numerous methodological choices. Arriving at objective justifications for all of these is difficult and time consuming. Thus, divergence times are routinely inferred under only one or a handful of parametric conditions, often chosen arbitrarily. Progress towards building robust biological timescales necessitate the development of better methods to visualize and quantify the sensitivity of results to these decisions. Here, we present an R package that assists in this endeavor through the use of chronospaces, i.e., graphical representations summarizing variation in the node ages contained in time-calibrated trees. We further test this approach using three empirical datasets spanning widely differing timeframes. Our results reveal large differences in the impact of many common methodological decisions, with the choice of clock (uncorrelated vs. autocorrelated) and loci having strong effects on inferred ages. Other decisions have comparatively minor consequences, including the use of the computationally intensive site-heterogeneous model CAT-GTR. Notably, these conclusions are as valid for Cenozoic divergences as they are for the deepest eukaryote nodes. The package chronospace implements a range of graphical and analytical tools that assist in the exploration of sensitivity and the prioritization of computational resources in the inference of divergence times.\n",
  "wordCount" : "2531",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580575" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580575">
        <p class="paperTitle">50,000 years of Evolutionary History of India: Insights from ~2,700 Whole Genome Sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kerdoncuff, E.; Skov, L.; Patterson, N.; Zhao, W.; Lueng, Y. Y.; Schellenberg, G. D.; Smith, J. A.; Dey, S.; Ganna, A.; Dey, A.; Kardia, S. L. R.; Lee, J.; Moorjani, P.</p>
        <p class="info">Score: 116.3, Published: 2024-02-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580575' target='https://doi.org/10.1101/2024.02.15.580575'> 10.1101/2024.02.15.580575</a></p>
        <p class="abstract">India has been underrepresented in whole genome sequencing studies. We generated 2,762 high coverage genomes from India - including individuals from most geographic regions, speakers of all major languages, and tribal and caste groups - providing a comprehensive survey of genetic variation in India. With these data, we reconstruct the evolutionary history of India through space and time at fine scales. We show that most Indians derive ancestry from three ancestral groups related to ancient Iranian farmers, Eurasian Steppe pastoralists and South Asian hunter-gatherers. We uncover a common source of Iranian-related ancestry from early Neolithic cultures of Central Asia into the ancestors of Ancestral South Indians (ASI), Ancestral North Indians (ANI), Austro-asiatic-related and East Asian-related groups in India. Following these admixtures, India experienced a major demographic shift towards endogamy, resulting in extensive homozygosity and identity-by-descent sharing among individuals. At deep time scales, Indians derive around 1-2% of their ancestry from gene flow from archaic hominins, Neanderthals and Denisovans. By assembling the surviving fragments of archaic ancestry in modern Indians, we recover ~1.5 Gb (or 50%) of the introgressing Neanderthal and ~0.6 Gb (or 20%) of the introgressing Denisovan genomes, more than any other previous archaic ancestry study. Moreover, Indians have the largest variation in Neanderthal ancestry, as well as the highest amount of population-specific Neanderthal segments among worldwide groups. Finally, we demonstrate that most of the genetic variation in Indians stems from a single major migration out of Africa that occurred around 50,000 years ago, with minimal contribution from earlier migration waves. Together, these analyses provide a detailed view of the population history of India and underscore the value of expanding genomic surveys to diverse groups outside Europe.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576074">
        <p class="paperTitle">Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cain, F. E.; Yamaguchi, K.</p>
        <p class="info">Score: 174.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576074' target='https://doi.org/10.1101/2024.01.17.576074'> 10.1101/2024.01.17.576074</a></p>
        <p class="abstract">The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579553" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579553">
        <p class="paperTitle">Unselfish meiotic drive maintains heterozygosity in a parthenogenetic ant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacy, K. D.; Hart, T.; Kronauer, D. J. C.</p>
        <p class="info">Score: 37.6, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579553' target='https://doi.org/10.1101/2024.02.09.579553'> 10.1101/2024.02.09.579553</a></p>
        <p class="abstract">According to Mendels second law, chromosomes segregate randomly in meiosis. Non-random segregation is primarily known for cases of selfish meiotic drive in females, in which particular alleles bias their own transmission into the oocyte1,2. Here, we report a rare example of unselfish meiotic drive for crossover inheritance in the clonal raider ant, Ooceraea biroi. This species produces diploid offspring parthenogenetically via fusion of two haploid nuclei from the same meiosis3. This process should cause rapid genotypic degeneration due to loss of heterozygosity, which results if crossover recombination is followed by random (Mendelian) segregation of chromosomes4,5. However, by comparing whole genomes of mothers and daughters, we show that loss of heterozygosity is exceedingly rare, raising the possibility that crossovers are infrequent or absent in O. biroi meiosis. Using a combination of cytology and whole genome sequencing, we show that crossover recombination is, in fact, common, but that loss of heterozygosity is avoided because crossover products are faithfully co-inherited. This results from a programmed violation of Mendels law of segregation, such that crossover products segregate together rather than randomly. This discovery highlights an extreme example of cellular &#34;memory&#34; of crossovers, which could be a common yet cryptic feature of chromosomal segregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580079" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580079">
        <p class="paperTitle">Multiple horizontal mini-chromosome transfers drive genome evolution of clonal blast fungus lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barragan, A. C.; Latorre, S. M.; Malmgren, A.; Harant, A.; Win, J.; Sugihara, Y.; Burbano, H. A.; Kamoun, S.; Langner, T.</p>
        <p class="info">Score: 36.9, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580079' target='https://doi.org/10.1101/2024.02.13.580079'> 10.1101/2024.02.13.580079</a></p>
        <p class="abstract">Crop disease pandemics are often driven by clonal lineages of plant pathogens that reproduce asexually. How these clonal pathogens continuously adapt to their hosts despite harboring limited genetic variation, and in absence of sexual recombination remains elusive. Here, we reveal multiple instances of horizontal chromosome transfer within pandemic clonal lineages of the blast fungus Magnaporthe (Syn. Pyricularia) oryzae. We identified a horizontally transferred 1.2Mb supernumerary mini-chromosome which is remarkably conserved between M. oryzae isolates from both the rice blast fungus lineage and the lineage infecting Indian goosegrass (Eleusine indica), a wild grass that often grows in the proximity of cultivated cereal crops. Furthermore, we show that this mini-chromosome was horizontally acquired by clonal rice blast isolates through at least nine distinct transfer events over the past three centuries. These findings establish horizontal mini-chromosome transfer as a mechanism facilitating genetic exchange among different host-associated blast fungus lineages. We propose that blast fungus populations infecting wild grasses act as genetic reservoirs that drive genome evolution of pandemic clonal lineages that afflict cereal crops.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580500" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580500">
        <p class="paperTitle">A critical reexamination of recovered SARS-CoV-2 sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Debarre, F.; Hensel, Z.</p>
        <p class="info">Score: 33.8, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580500' target='https://doi.org/10.1101/2024.02.15.580500'> 10.1101/2024.02.15.580500</a></p>
        <p class="abstract">SARS-CoV-2 genomes collected at the onset of the Covid-19 pandemic are valuable because they could help understand how the virus entered the human population. In 2021, Jesse Bloom reported on the recovery of a sequencing dataset that had been removed from the NCBI SRA database at the request of the data generators, a scientific team at Wuhan University. Bloom suggested that the data may have been removed in order to obfuscate the origin of SARS-CoV-2, questioning the generating authors&#39; statements that the samples had been collected on and after January 30, 2020. Here, we show that sample collection dates were published in 2020 together with the sequencing data, and match the dates given by the authors in 2021. We examine mutations in these sequences and confirm that they are entirely consistent with the previously known genetic diversity of SARS-CoV-2 of late January 2020. Finally, we explain how an apparent phylogenetic rooting paradox described by Bloom was resolved by subsequent analysis. Our reanalysis demonstrates that allegations of cover-up or metadata manipulation were unwarranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579710" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579710">
        <p class="paperTitle">A long non-coding RNA at the cortex locus controls adaptive colouration in butterflies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Livraghi, L.; Hanly, J. J.; Evans, E.; Wright, C. J.; Loh, L. S.; Mazo-Vargas, A.; Kamrava, K.; Carter, A. T.; van der Heijden, E. S.; Reed, R. D.; Papa, R.; Jiggins, C. D.; Martin, A.</p>
        <p class="info">Score: 27.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579710' target='https://doi.org/10.1101/2024.02.09.579710'> 10.1101/2024.02.09.579710</a></p>
        <p class="abstract">Evolutionary variation in the wing pigmentation of butterflies and moths offers striking examples of adaptation by crypsis and mimicry. The cortex locus has been independently mapped as the locus controlling colour polymorphisms in 14 lepidopteran species, suggesting it acts as a genomic hotspot for the diversification of wing patterns, but functional validation through protein-coding knockouts has proven difficult to obtain. Our study unveils the role of a novel long non-coding RNA (lncRNA) which we name ivory, transcribed from the cortex locus, in modulating colour patterning in butterflies. Strikingly, ivory expression prefigures most melanic patterns during pupal development, suggesting an early developmental role in specifying scale identity. To test this, we generated CRISPR mosaic knock-outs in five nymphalid butterfly species and show that ivory mutagenesis yields transformations of dark pigmented scales into white or light-coloured scales. Genotyping of Vanessa cardui germline mutants associates these phenotypes to small on-target deletions at the conserved first exon of ivory. In contrast, cortex germline mutant butterflies with confirmed null alleles lack any wing phenotype, and exclude a colour patterning role for this adjacent gene. Overall, these results show that a lncRNA acts as a master switch of colour pattern specification, and played key roles in the adaptive diversification of colour patterns in butterflies.

Significance statementDeciphering the genetic underpinnings of adaptive variation is fundamental for a comprehensive understanding of evolutionary processes. Long non-coding RNAs (lncRNAs) represent an emerging category of genetic modulators within the genome, yet they have been overlooked as a source of phenotypic diversity. In this study, we unveil the pivotal role of a lncRNA in orchestrating colour transitions between dark and light patterns during butterfly wing development. Remarkably, this lncRNA gene is nested within the cortex locus, a genetic region known to control multiple cases of adaptive variation in butterflies and moths, including iconic examples of natural selection. These findings highlight the significant influence of lncRNAs in developmental regulation, and also underscore their potential as key genetic players in the evolutionary process itself.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579783">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579783" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579783">
        <p class="paperTitle">Compensatory mutations potentiate constructive neutral evolution by gene duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579783" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579783" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Despres, P. C.; Dube, A. K.; Grenier, J.; Picard, M.-E.; Shi, R.; Landry, C. R.</p>
        <p class="info">Score: 26.8, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579783' target='https://doi.org/10.1101/2024.02.12.579783'> 10.1101/2024.02.12.579783</a></p>
        <p class="abstract">Protein functions generally depend on their assembly into complexes. During evolution, some complexes have transitioned from homomers encoded by a single gene to heteromers encoded by duplicate genes. This transition could occur without adaptive evolution through intermolecular compensatory mutations. Here, we experimentally duplicate and evolve an homodimeric enzyme to examine if and how this could happen. We identify hundreds of deleterious mutations that inactivate individual homodimers but produce functional enzymes when co-expressed as duplicated proteins that heterodimerize. The structure of one such heteromer reveals how both losses of function are buffered through the introduction of asymmetry in the complex that allows them to subfunctionalize. Constructive neutral evolution can thus occur by gene duplication followed by only one deleterious mutation per duplicate.

One sentence summaryCompensatory deleterious mutations entangle gene duplicates</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580425" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580425">
        <p class="paperTitle">Fusion, fission, and scrambling of the bilaterian genome in Bryozoa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lewin, T. D.; Liao, I. J.-Y.; Chen, M.-E.; Bishop, J. D. D.; Holland, P. W. H.; Luo, Y.-J.</p>
        <p class="info">Score: 25.3, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580425' target='https://doi.org/10.1101/2024.02.15.580425'> 10.1101/2024.02.15.580425</a></p>
        <p class="abstract">Genes are commonly found together on the same chromosome over vast evolutionary distances. This extensive physical gene linkage, known as macrosynteny, can be seen between bilaterian phyla as divergent as Chordata, Echinodermata, Mollusca, and Nemertea and likely reflects the importance of genome organization to gene regulatory landscapes. Here, we report a very different pattern of genome evolution in Bryozoa, an understudied yet ecologically important phylum of colonial invertebrates. Using comparative genomics, we reconstruct the chromosomal evolutionary history of five bryozoans. We infer the ancestral bryozoan genome organization and identify multiple ancient chromosome fusions followed by gene mixing, leading to the near-complete loss of bilaterian linkage groups. A second wave of rearrangements, including chromosome fission, occurred independently in two bryozoan classes, further shuffling bryozoan genomes. We also discover nine chromosomal fusion events shared between bryozoans and brachiopods, supporting the traditional yet highly debated Lophophorata hypothesis. Finally, we show that chromosome fusion and fission processes led to the separation of bryozoan Hox clusters. These findings demonstrate how the canonical bilaterian genome structure has been lost across an entire phylum, suggest a modification to gene regulatory landscapes, and provide a powerful source of phylogenetic information.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580205" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580205">
        <p class="paperTitle">Phylotranscriptomics reveals convergent behavioral evolution associated with shared and unique mechanisms in cavity-nesting songbirds</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lipshutz, S. E.; Hibbins, M. S.; Bentz, A. B.; Buechlin, A. M.; Empson, T. A.; George, E. M.; Hauber, M. E.; Rusch, D. B.; Schelsky, W. M.; Thomas, Q. K.; Torneo, S. J.; Turner, A. M.; Wolf, S. E.; Woodruff, M. J.; Hahn, M. W.; Rosvall, K. A.</p>
        <p class="info">Score: 24.3, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580205' target='https://doi.org/10.1101/2024.02.13.580205'> 10.1101/2024.02.13.580205</a></p>
        <p class="abstract">Uncovering the genomic bases of phenotypic adaptation is a major goal in biology, but this has been hard to achieve for complex behavioral traits. Here, we leverage the repeated, independent evolution of obligate cavity-nesting in birds to test the hypothesis that the shared pressure to compete for a limited breeding resource drives convergent behavioral evolution via convergent gene regulatory changes in the brain. Using behavioral assays in the field, hormonal measures of free-living subjects, and transcriptome-wide analyses of the brain in wild-captured males and females, we examined five species pairs across five avian families, each including one obligate cavity-nesting species and a related species with an open cup-nesting or otherwise more flexible nest strategy. Results support the hypothesis of behavioral convergence, with higher levels of territorial aggression in obligate cavity-nesters, particularly among females. Levels of testosterone in circulation were not associated with nest strategy or aggression for either sex, but phylogenetic analyses of individual genes and co-regulated gene networks revealed some shared patterns of gene expression, including a nest strategy-related gene network shared across families and two separate networks linked to aggression only in females. Though associated with convergent behavioral evolution, these genes were not significantly enriched for particular functional pathways, and the scope of convergent gene expression evolution was limited to a small percent of the genome. Together, these observations indicate that replicated evolutionary changes in complex behavior arise via a combination of convergent and lineage-specific evolution of gene regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578835">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578835" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578835">
        <p class="paperTitle">Chronospaces: an R package for the statistical exploration of divergence times reveals extreme dependence on molecular clocks and gene choice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578835" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578835" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mongiardino Koch, N.; Milla Carmona, P.</p>
        <p class="info">Score: 23.2, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578835' target='https://doi.org/10.1101/2024.02.04.578835'> 10.1101/2024.02.04.578835</a></p>
        <p class="abstract">Much of our understanding of the history of life hinges upon time calibration, the process of assigning absolute times to cladogenetic events. Bayesian approaches to time scaling phylogenetic trees have dramatically grown in complexity, and hinge today upon numerous methodological choices. Arriving at objective justifications for all of these is difficult and time consuming. Thus, divergence times are routinely inferred under only one or a handful of parametric conditions, often chosen arbitrarily. Progress towards building robust biological timescales necessitate the development of better methods to visualize and quantify the sensitivity of results to these decisions. Here, we present an R package that assists in this endeavor through the use of chronospaces, i.e., graphical representations summarizing variation in the node ages contained in time-calibrated trees. We further test this approach using three empirical datasets spanning widely differing timeframes. Our results reveal large differences in the impact of many common methodological decisions, with the choice of clock (uncorrelated vs. autocorrelated) and loci having strong effects on inferred ages. Other decisions have comparatively minor consequences, including the use of the computationally intensive site-heterogeneous model CAT-GTR. Notably, these conclusions are as valid for Cenozoic divergences as they are for the deepest eukaryote nodes. The package chronospace implements a range of graphical and analytical tools that assist in the exploration of sensitivity and the prioritization of computational resources in the inference of divergence times.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
