<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Self-assembling viral histones unravel early nucleosome evolution
Authors: Irwin, N. A. T.; Richards, T. A.
Score: 53.1, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576
Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution1,2.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Self-assembling viral histones unravel early nucleosome evolution
Authors: Irwin, N. A. T.; Richards, T. A.
Score: 53.1, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576
Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution1,2." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Self-assembling viral histones unravel early nucleosome evolution
Authors: Irwin, N. A. T.; Richards, T. A.
Score: 53.1, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576
Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution1,2."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Self-assembling viral histones unravel early nucleosome evolution\nAuthors: Irwin, N. A. T.; Richards, T. A.\nScore: 53.1, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576\nNucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution1,2.",
  "keywords": [
    
  ],
  "articleBody": " Self-assembling viral histones unravel early nucleosome evolution\nAuthors: Irwin, N. A. T.; Richards, T. A.\nScore: 53.1, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576\nNucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution1,2. However, functional constraints and the extinction of stem-eukaryotes have concealed how these dynamic systems evolved from simpler histone homologues in Archaea3-5. Viral histones have also previously been identified and are thought to reflect an ancestral state as they often comprise multiple histone paralogues arranged within a single protein, termed histone repeats6-11. Here, using viruses as an alternative source of variation, we expand the known diversity of histones and develop an empirical hypothesis for the origin of the nucleosome. Our analysis identified hundreds of histones with variable domain repeat configurations including histone singlets, doublets, triplets, and quadruplets, the latter comprising the four core histones arranged in series. Viral histone repeats consistently branch between Archaea and eukaryotes in phylogenetic trees and display intermediate functions, self-assembling into eukaryotic-like nucleosomes that stack into archaeal-like oligomers capable of impacting genomic activity and condensing DNA. The linkers conjoining the histone repeats also facilitate nucleosome formation and can promote the assembly of eukaryotic nucleosomes in the bacterium, Escherichia coli. Combining these data, we hypothesize that viral histone repeats represent molecular relics acquired by viruses from stem-eukaryotes during eukaryogenesis and suggest that nucleosome evolution may have proceeded through histone repeat intermediates.\nStrong environmental memory revealed by experimental evolution in static and fluctuating environments\nAuthors: Abreu, C. I.; Mathur, S.; Petrov, D. A.\nScore: 39.4, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739\nEvolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters. However, transitions between conditions may pose entirely new challenges, which could cause deviations from this time-average. To test this, we tracked hundreds of thousands of barcoded yeast lineages evolving in static and fluctuating conditions and subsequently isolated 900 mutants for pooled fitness assays in 15 environments. We find that fitness in fluctuating environments indeed often deviates from the expectation based on static components, leading to fitness non-additivity. Moreover, closer examination reveals that fitness in one component of a fluctuating environment is often strongly influenced by the previous component. We show that this environmental memory is especially common for mutants with high variance in fitness across tested environments, even if the components of the focal fluctuating environment are excluded from this variance. We employ a simple mathematical model and whole-genome sequencing to propose mechanisms underlying this effect, including lag time evolution and sensing mutations. Our results demonstrate that environmental fluctuations have large impacts on fitness and suggest that variance in static environments can explain these impacts.\nThe MUC19 gene in Denisovans, Neanderthals, and Modern Humans: An Evolutionary History of Recurrent Introgression and Natural Selection\nAuthors: Villanea, F. A.; Peede, D.; Kaufman, E. J.; Anorve-Garibay, V.; Dillon, K. E. W.; Zeloni, R.; Marnetto, d.; Moorjani, P.; Jay, F.; Valdmanis, P. N.; Avila-Arcos, M. C.; Huerta-Sanchez, E.\nScore: 25.4, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559202\nAll humans carry a small fraction of archaic ancestry across the genome, the legacy of gene flow from Neanderthals, Denisovans, and other hominids into the ancestors of modern humans. While the effects of Neanderthal ancestry on human fitness and health have been explored more thoroughly, there are fewer examples of adaptive introgression of Denisovan variants. Here, we study the gene MUC19, for which some modern humans carry a Denisovan-like haplotype. MUC19 is a mucin, a glycoprotein that forms gels with various biological functions, from lubrication to immunity. We find the diagnostic variants for the Denisovan-like MUC19 haplotype at high frequencies in admixed Latin American individuals among global population, and at highest frequency in 23 ancient Indigenous American individuals, all predating population admixture with Europeans and Africans. We find that some Neanderthals - Vindija and Chagyrskaya - carry the Denisovan-like MUC19 haplotype, and that it was likely introgressed into human populations through Neanderthal introgression rather than Denisovan introgression. Finally, we find that the Denisovan-like MUC19 haplotype carries a higher copy number of a 30 base-pair variable number tandem repeat relative to the Human-like haplotype, and that copy numbers of this repeat are exceedingly high in American populations. Our results suggest that the Denisovan-like MUC19 haplotype served as the raw genetic material for positive selection as American populations adapted to novel environments during their movement from Beringia into North and then South America.\nTransposable element abundance subtly contributes to lower fitness in maize\nAuthors: Stitzer, M. C.; Khaipho-Burch, M. B.; Hudson, A. I.; Song, B.; Valdes-Franco, J. A.; Ramstein, G. P.; Feschotte, C.; Buckler, E. S.\nScore: 29.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557618\nTransposable elements (TEs) have long been shown to have deleterious effects on the survival and reproduction of their host organism. As TEs are mobile DNA that jump to new positions, this deleterious cost can occur directly, by inserting into genes and regulatory sequences. Classical population genetic theory suggests copy-number dependent selection against TEs is necessary to prevent TEs from expanding so much they take over a genome. Such models have been difficult to interpret when applied to large genomes like maize, where there are hundreds of thousands of TE insertions that collectively make up 85% of the genome. Here, we use nearly 5000 inbred lines from maize mapping populations and a pan-genomic imputation approach to measure TE content. Segregating TE content gives rise to 100 Mb differences between individuals, and populations often show transgressive segregation in TE content. We use replicated phenotypes measured in hybrids across numerous years and environments to empirically measure the fitness costs of TEs. For an annual plant like maize, grain yield is not only a key agronomic phenotype, but also a direct measure of reproductive output. We find weak negative effects of TE accumulation on grain yield, nearing the limit of the efficacy of natural selection in maize. This results in a loss of one kernel ({approx}0.1% of average per-plant yield) for every additional 14 Mb of TE content. This deleterious load is enriched in TEs within 1 kilobase of genes and young TE insertions. Together, we provide rare empirical measurements of the fitness costs of TEs, and suggest that the TEs we see today in the genome have been filtered by selection against their deleterious consequences on maize fitness.\nFemales translate male mRNA transferred during mating\nAuthors: Matzkin, L. M.; Bono, J. M.; Pigage, H.; Allan, C. W.; Diaz, F.; McCoy, J.; Green, C.; Callan, J.; Delahunt, S.\nScore: 15.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.558997\nAlthough RNA is found in the seminal fluid of diverse organisms, it is unknown whether this RNA is functional within females. Here, we develop an experimental proteomic method called VESPA (Variant Enabled SILAC Proteomic Analysis) to test the hypothesis that Drosophila male seminal fluid RNA is translated by females. We find strong evidence for 67 male-derived, female-translated proteins (mdFTPs) in female lower reproductive tracts at six hours postmating, many with predicted functions relevant to reproduction. Gene knockout experiments indicate that genes coding for mdFTPs play diverse roles in postmating interactions, with effects on fertilization efficiency, and the formation and persistence of the insemination reaction mass, a trait hypothesized to be involved in sexual conflict. These findings advance our understanding of reproduction by revealing a novel mechanism of postmating molecular interactions between the sexes that strengthens and extends male influences on reproductive outcomes in previously unrecognized ways. Given the diverse species known to carry RNA in seminal fluid, this discovery has broad significance for understanding molecular mechanisms of cooperation and conflict during reproduction.\nA direct experimental test of Ohno's hypothesis\nAuthors: Mihajlovic, L.; Iyengar, B. R.; Baier, F.; Barbier, I.; Iwaszkiewicz, J.; Zoete, V.; Wagner, A.; Schaerli, Y.\nScore: 14.7, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559237\nGene duplication drives evolution by providing raw material for proteins with novel functions. The oldest and historically most influential hypothesis about the evolutionary fate and potential of duplicated genes has been proposed by Susumu Ohno in 1970. This hypothesis essentially posits that gene duplication can help genes tolerate new mutations and thus facilitates the evolu-tion of new phenotypes Competing hypotheses argue that deleterious mutations will usually inactivate gene duplicates too rapidly for Ohno's hypothesis to work. Here, we provide a first direct experimental test of Ohno's hypothesis. Specifically, we evolved one or two genes en-coding a fluorescent protein in Escherichia coli through multiple rounds of mutagenesis and selection. We then analyzed the genotypic and phenotypic evolutionary dynamics of the evolving populations through high-throughput DNA sequencing, biochemical assays, and engineering of selected variants. In support of Ohno's hypothesis, populations carrying two gene copies displayed higher mutational robustness than those carrying a single gene copy. As a conse-quence, the double-copy populations experienced relaxed purifying selection, evolved higher phenotypic and genetic diversity, carried more mutations and accumulated combinations of key beneficial mutations earlier. However, their phenotypic evolution was not accelerated, possibly because one gene copy rapidly became inactivated by deleterious mutations. Our work provides an experimental platform to test models of evolution by gene duplication, and it supports alter-natives to Ohno's hypothesis.\nDifferential evolution of cooperative traits in aggregative multicellular bacterium Myxococcus xanthus driven by varied population bottleneck sizes\nAuthors: Kalathera, J.; Patel, V.; Pande, S.\nScore: 14.5, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558552\nRepeated population bottlenecks influence the evolution and maintenance of cooperation. However, it remains unclear whether bottlenecks select all cooperative traits expressed by an organism or only a subset of them. Myxococcus xanthus, a social bacterium, displays multiple cooperative traits, including growth, predation, sporulation in multicellular fruiting bodies, and germination. Using laboratory evolution experiment, we investigated the effect of repeated stringent versus relaxed population bottlenecks on the evolution of these four cooperative traits when they were all under selection. We found that only fruiting body formation and growth were positively selected under the stringent regimen, while the other two traits were negatively selected. The pattern was reversed in the relaxed regimen. Additionally, the relaxed regimen led to a significant increase in fitness when competed against ancestors across the entire lifecycle, whereas the stringent treatment did not change competitive fitness. Genomic analysis revealed that mutations in sigma54 interacting protein and DNA-binding response regulator protein are linked with the changes observed in stringent and relaxed regimens respectively. Further, similar trade-offs are also seen among natural populations of M. xanthus. Overall, we demonstrate that different bottleneck sizes drive the evolution of lifecycles in distinct manners, driven by trade-offs between cooperative life history traits.\nGenomic signatures of adaptation in native lizards exposed to human-introduced fire ants\nAuthors: Assis, B. A.; Sullivan, A. P.; Marciniak, S.; Bergey, C. M.; Garcia, V.; Szpiech, Z. A.; Langkilde, T.; Perry, G. H.\nScore: 10.4, Published: 2023-09-26 DOI: 10.1101/2023.09.24.559217\nUnderstanding the process of genetic adaptation in response to human-mediated ecological change will help elucidate the eco-evolutionary impacts of human activity. Red fire ants (Solenopsis invicta) spread across Southeastern USA since their accidental introduction via Port Mobile, Alabama in the 1930s, serving today as both novel venomous predator and novel toxic prey to native eastern fence lizards (Sceloporus undulatus). To identify potential signatures of genetic adaptation in lizards to invasive fire ants, we generated whole genome sequencing data from 420 native fence lizards sampled across three populations, two of which had not been invaded by fire ants (in Tennessee and Arkansas) and one which had been invaded for ~70 years (Alabama). We detected signatures of positive selection exclusive to the exposed Alabama population for genetic variants overlapping genes related to the membrane attack complex of the complement immune system, growth factor pathways, and morphological development. Prior work identified a relationship between increased lizard survival of fire ant attack and longer hind limbs, which lizards use to remove ants from their bodies. Furthermore, we conducted a genome-wide association study with 381 Alabama lizards to identify 24 hind limb length-associated genetic loci. For two loci, positive-effect alleles occur in high frequency and overlap genomic regions that are highly differentiated from the populations naive to fire ants. Collectively, these findings represent plausible genetic adaptations in response to fire ant invasion, whereby morphological differentiation may increase survival against swarming ants and altered immune responses may allow the exploitation of a novel, toxic food resource.\nCoupling of 12 chromosomal inversions maintains a strong barrier to gene flow between ecotypes\nAuthors: Le Moan, A.; Stankowski, S.; Rafajlovic, M.; Martinez Ortega, O.; Faria, R.; Butlin, R.; Johannesson, K.\nScore: 9.9, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558209\nChromosomal rearrangements lead to the coupling of reproductive barriers, but whether and how they contribute to completion of speciation remains unclear. Marine snails of the genus Littorina repeatedly form hybrid zones between taxa segregating for inversion arrangements, providing opportunities to study this question. Here, we analysed two adjacent transects across hybrid zones between large and dwarf ecotypes of Littorina fabalis covering wave exposure gradients in a Swedish island. Applying whole-genome sequences we found 12 putative inversions reaching near differential fixation between the opposite ends of each transect, and being in strong linkage disequilibrium. These inversions cover 20% of the genome and carry 93% of divergent SNPs. Bimodal hybrid zones in both transects indicate that the two ecotypes of Littorina fabalis maintain their genetic and phenotypic integrity following contact due to strong coupling between inversion clines that strengthened the reproductive barrier. The bimodality resulting from the linked inversions extends into collinear parts of the genome, suggesting a genome-wide coupling. Demographic inference suggests that the coupling built up during a period of allopatry, and has been maintained for more than 1K generations after secondary contact. Overall, this study shows that the coupling of multiple chromosomal inversions contributes to strong reproductive isolation. Importantly, two of the inversions overlap with inverted genomic regions associated with ecotype differences in a closely-related species (L. saxatilis), suggesting the same regions, with similar structural variants, repeatedly contribute to ecotype evolution in distinct species.\nPatterns of genomic variation reveal a single evolutionary origin of the wild allotetraploid Mimulus sookensis\nAuthors: Whitener, M. R.; Mangelson, H.; Sweigart, A. L.\nScore: 8.9, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557766\nPolyploidy occurs across the tree of life and is especially common in plants. Because newly formed cytotypes are often incompatible with their progenitors, polyploidy is also said to trigger \"instantaneous\" speciation. If a polyploid can self-fertilize or reproduce asexually, it is even possible for one individual to produce an entirely new lineage. How often this extreme scenario occurs is unclear, with most studies of wild polyploids reporting multiple origins. Here, we explore the evolutionary history of the wild allotetraploid Mimulus sookensis, which was formed through hybridization between self-compatible, diploid species in the Mimulus guttatus complex. We generate a chromosome-scale reference assembly for M. sookensis and define its distinct subgenomes. Despite previous reports suggesting multiple origins of this highly selfing polyploid, we discover patterns of population genomic variation that provide unambiguous support for a single origin, which we estimate occurred [~]71,000 years ago. One M. sookensis subgenome is clearly derived from the selfer M. nasutus, which, based on organellar variation, also appears to be the maternal progenitor. The ancestor of the other subgenome is less certain, but it shares variation with both M. decorus and M. guttatus, two outcrossing diploids that overlap broadly with M. sookensis. Whatever its precise ancestry, this study establishes M. sookensis as an example of instantaneous speciation, likely facilitated by the polyploids predisposition to self-fertilize. With a reference genome for M. sookensis now available and its origin clarified, this wild tetraploid is poised to become a model for understanding the genetic and evolutionary mechanisms of polyploid persistence.\n",
  "wordCount" : "2649",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558576">
        <p class="paperTitle">Self-assembling viral histones unravel early nucleosome evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Irwin, N. A. T.; Richards, T. A.</p>
        <p class="info">Score: 53.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558576' target='https://doi.org/10.1101/2023.09.20.558576'> 10.1101/2023.09.20.558576</a></p>
        <p class="abstract">Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution1,2. However, functional constraints and the extinction of stem-eukaryotes have concealed how these dynamic systems evolved from simpler histone homologues in Archaea3-5. Viral histones have also previously been identified and are thought to reflect an ancestral state as they often comprise multiple histone paralogues arranged within a single protein, termed histone repeats6-11. Here, using viruses as an alternative source of variation, we expand the known diversity of histones and develop an empirical hypothesis for the origin of the nucleosome. Our analysis identified hundreds of histones with variable domain repeat configurations including histone singlets, doublets, triplets, and quadruplets, the latter comprising the four core histones arranged in series. Viral histone repeats consistently branch between Archaea and eukaryotes in phylogenetic trees and display intermediate functions, self-assembling into eukaryotic-like nucleosomes that stack into archaeal-like oligomers capable of impacting genomic activity and condensing DNA. The linkers conjoining the histone repeats also facilitate nucleosome formation and can promote the assembly of eukaryotic nucleosomes in the bacterium, Escherichia coli. Combining these data, we hypothesize that viral histone repeats represent molecular relics acquired by viruses from stem-eukaryotes during eukaryogenesis and suggest that nucleosome evolution may have proceeded through histone repeat intermediates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557739">
        <p class="paperTitle">Strong environmental memory revealed by experimental evolution in static and fluctuating environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abreu, C. I.; Mathur, S.; Petrov, D. A.</p>
        <p class="info">Score: 39.4, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557739' target='https://doi.org/10.1101/2023.09.14.557739'> 10.1101/2023.09.14.557739</a></p>
        <p class="abstract">Evolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters. However, transitions between conditions may pose entirely new challenges, which could cause deviations from this time-average. To test this, we tracked hundreds of thousands of barcoded yeast lineages evolving in static and fluctuating conditions and subsequently isolated 900 mutants for pooled fitness assays in 15 environments. We find that fitness in fluctuating environments indeed often deviates from the expectation based on static components, leading to fitness non-additivity. Moreover, closer examination reveals that fitness in one component of a fluctuating environment is often strongly influenced by the previous component. We show that this environmental memory is especially common for mutants with high variance in fitness across tested environments, even if the components of the focal fluctuating environment are excluded from this variance. We employ a simple mathematical model and whole-genome sequencing to propose mechanisms underlying this effect, including lag time evolution and sensing mutations. Our results demonstrate that environmental fluctuations have large impacts on fitness and suggest that variance in static environments can explain these impacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559202">
        <p class="paperTitle">The MUC19 gene in Denisovans, Neanderthals, and Modern Humans: An Evolutionary History of Recurrent Introgression and Natural Selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Villanea, F. A.; Peede, D.; Kaufman, E. J.; Anorve-Garibay, V.; Dillon, K. E. W.; Zeloni, R.; Marnetto, d.; Moorjani, P.; Jay, F.; Valdmanis, P. N.; Avila-Arcos, M. C.; Huerta-Sanchez, E.</p>
        <p class="info">Score: 25.4, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559202' target='https://doi.org/10.1101/2023.09.25.559202'> 10.1101/2023.09.25.559202</a></p>
        <p class="abstract">All humans carry a small fraction of archaic ancestry across the genome, the legacy of gene flow from Neanderthals, Denisovans, and other hominids into the ancestors of modern humans. While the effects of Neanderthal ancestry on human fitness and health have been explored more thoroughly, there are fewer examples of adaptive introgression of Denisovan variants. Here, we study the gene MUC19, for which some modern humans carry a Denisovan-like haplotype. MUC19 is a mucin, a glycoprotein that forms gels with various biological functions, from lubrication to immunity. We find the diagnostic variants for the Denisovan-like MUC19 haplotype at high frequencies in admixed Latin American individuals among global population, and at highest frequency in 23 ancient Indigenous American individuals, all predating population admixture with Europeans and Africans. We find that some Neanderthals - Vindija and Chagyrskaya - carry the Denisovan-like MUC19 haplotype, and that it was likely introgressed into human populations through Neanderthal introgression rather than Denisovan introgression. Finally, we find that the Denisovan-like MUC19 haplotype carries a higher copy number of a 30 base-pair variable number tandem repeat relative to the Human-like haplotype, and that copy numbers of this repeat are exceedingly high in American populations. Our results suggest that the Denisovan-like MUC19 haplotype served as the raw genetic material for positive selection as American populations adapted to novel environments during their movement from Beringia into North and then South America.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557618">
        <p class="paperTitle">Transposable element abundance subtly contributes to lower fitness in maize</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stitzer, M. C.; Khaipho-Burch, M. B.; Hudson, A. I.; Song, B.; Valdes-Franco, J. A.; Ramstein, G. P.; Feschotte, C.; Buckler, E. S.</p>
        <p class="info">Score: 29.7, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557618' target='https://doi.org/10.1101/2023.09.18.557618'> 10.1101/2023.09.18.557618</a></p>
        <p class="abstract">Transposable elements (TEs) have long been shown to have deleterious effects on the survival and reproduction of their host organism. As TEs are mobile DNA that jump to new positions, this deleterious cost can occur directly, by inserting into genes and regulatory sequences. Classical population genetic theory suggests copy-number dependent selection against TEs is necessary to prevent TEs from expanding so much they take over a genome. Such models have been difficult to interpret when applied to large genomes like maize, where there are hundreds of thousands of TE insertions that collectively make up 85% of the genome. Here, we use nearly 5000 inbred lines from maize mapping populations and a pan-genomic imputation approach to measure TE content. Segregating TE content gives rise to 100 Mb differences between individuals, and populations often show transgressive segregation in TE content. We use replicated phenotypes measured in hybrids across numerous years and environments to empirically measure the fitness costs of TEs. For an annual plant like maize, grain yield is not only a key agronomic phenotype, but also a direct measure of reproductive output. We find weak negative effects of TE accumulation on grain yield, nearing the limit of the efficacy of natural selection in maize. This results in a loss of one kernel ({approx}0.1% of average per-plant yield) for every additional 14 Mb of TE content. This deleterious load is enriched in TEs within 1 kilobase of genes and young TE insertions. Together, we provide rare empirical measurements of the fitness costs of TEs, and suggest that the TEs we see today in the genome have been filtered by selection against their deleterious consequences on maize fitness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558997">
        <p class="paperTitle">Females translate male mRNA transferred during mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matzkin, L. M.; Bono, J. M.; Pigage, H.; Allan, C. W.; Diaz, F.; McCoy, J.; Green, C.; Callan, J.; Delahunt, S.</p>
        <p class="info">Score: 15.2, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558997' target='https://doi.org/10.1101/2023.09.22.558997'> 10.1101/2023.09.22.558997</a></p>
        <p class="abstract">Although RNA is found in the seminal fluid of diverse organisms, it is unknown whether this RNA is functional within females. Here, we develop an experimental proteomic method called VESPA (Variant Enabled SILAC Proteomic Analysis) to test the hypothesis that Drosophila male seminal fluid RNA is translated by females. We find strong evidence for 67 male-derived, female-translated proteins (mdFTPs) in female lower reproductive tracts at six hours postmating, many with predicted functions relevant to reproduction. Gene knockout experiments indicate that genes coding for mdFTPs play diverse roles in postmating interactions, with effects on fertilization efficiency, and the formation and persistence of the insemination reaction mass, a trait hypothesized to be involved in sexual conflict. These findings advance our understanding of reproduction by revealing a novel mechanism of postmating molecular interactions between the sexes that strengthens and extends male influences on reproductive outcomes in previously unrecognized ways. Given the diverse species known to carry RNA in seminal fluid, this discovery has broad significance for understanding molecular mechanisms of cooperation and conflict during reproduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559237">
        <p class="paperTitle">A direct experimental test of Ohno&#39;s hypothesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mihajlovic, L.; Iyengar, B. R.; Baier, F.; Barbier, I.; Iwaszkiewicz, J.; Zoete, V.; Wagner, A.; Schaerli, Y.</p>
        <p class="info">Score: 14.7, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559237' target='https://doi.org/10.1101/2023.09.25.559237'> 10.1101/2023.09.25.559237</a></p>
        <p class="abstract">Gene duplication drives evolution by providing raw material for proteins with novel functions. The oldest and historically most influential hypothesis about the evolutionary fate and potential of duplicated genes has been proposed by Susumu Ohno in 1970. This hypothesis essentially posits that gene duplication can help genes tolerate new mutations and thus facilitates the evolu-tion of new phenotypes Competing hypotheses argue that deleterious mutations will usually inactivate gene duplicates too rapidly for Ohno&#39;s hypothesis to work. Here, we provide a first direct experimental test of Ohno&#39;s hypothesis. Specifically, we evolved one or two genes en-coding a fluorescent protein in Escherichia coli through multiple rounds of mutagenesis and selection. We then analyzed the genotypic and phenotypic evolutionary dynamics of the evolving populations through high-throughput DNA sequencing, biochemical assays, and engineering of selected variants. In support of Ohno&#39;s hypothesis, populations carrying two gene copies displayed higher mutational robustness than those carrying a single gene copy. As a conse-quence, the double-copy populations experienced relaxed purifying selection, evolved higher phenotypic and genetic diversity, carried more mutations and accumulated combinations of key beneficial mutations earlier. However, their phenotypic evolution was not accelerated, possibly because one gene copy rapidly became inactivated by deleterious mutations. Our work provides an experimental platform to test models of evolution by gene duplication, and it supports alter-natives to Ohno&#39;s hypothesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558552">
        <p class="paperTitle">Differential evolution of cooperative traits in aggregative multicellular bacterium Myxococcus xanthus driven by varied population bottleneck sizes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalathera, J.; Patel, V.; Pande, S.</p>
        <p class="info">Score: 14.5, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558552' target='https://doi.org/10.1101/2023.09.20.558552'> 10.1101/2023.09.20.558552</a></p>
        <p class="abstract">Repeated population bottlenecks influence the evolution and maintenance of cooperation. However, it remains unclear whether bottlenecks select all cooperative traits expressed by an organism or only a subset of them. Myxococcus xanthus, a social bacterium, displays multiple cooperative traits, including growth, predation, sporulation in multicellular fruiting bodies, and germination. Using laboratory evolution experiment, we investigated the effect of repeated stringent versus relaxed population bottlenecks on the evolution of these four cooperative traits when they were all under selection. We found that only fruiting body formation and growth were positively selected under the stringent regimen, while the other two traits were negatively selected. The pattern was reversed in the relaxed regimen. Additionally, the relaxed regimen led to a significant increase in fitness when competed against ancestors across the entire lifecycle, whereas the stringent treatment did not change competitive fitness. Genomic analysis revealed that mutations in sigma54 interacting protein and DNA-binding response regulator protein are linked with the changes observed in stringent and relaxed regimens respectively. Further, similar trade-offs are also seen among natural populations of M. xanthus. Overall, we demonstrate that different bottleneck sizes drive the evolution of lifecycles in distinct manners, driven by trade-offs between cooperative life history traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.559217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.559217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.559217">
        <p class="paperTitle">Genomic signatures of adaptation in native lizards exposed to human-introduced fire ants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.559217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.559217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Assis, B. A.; Sullivan, A. P.; Marciniak, S.; Bergey, C. M.; Garcia, V.; Szpiech, Z. A.; Langkilde, T.; Perry, G. H.</p>
        <p class="info">Score: 10.4, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.559217' target='https://doi.org/10.1101/2023.09.24.559217'> 10.1101/2023.09.24.559217</a></p>
        <p class="abstract">Understanding the process of genetic adaptation in response to human-mediated ecological change will help elucidate the eco-evolutionary impacts of human activity. Red fire ants (Solenopsis invicta) spread across Southeastern USA since their accidental introduction via Port Mobile, Alabama in the 1930s, serving today as both novel venomous predator and novel toxic prey to native eastern fence lizards (Sceloporus undulatus). To identify potential signatures of genetic adaptation in lizards to invasive fire ants, we generated whole genome sequencing data from 420 native fence lizards sampled across three populations, two of which had not been invaded by fire ants (in Tennessee and Arkansas) and one which had been invaded for ~70 years (Alabama). We detected signatures of positive selection exclusive to the exposed Alabama population for genetic variants overlapping genes related to the membrane attack complex of the complement immune system, growth factor pathways, and morphological development. Prior work identified a relationship between increased lizard survival of fire ant attack and longer hind limbs, which lizards use to remove ants from their bodies. Furthermore, we conducted a genome-wide association study with 381 Alabama lizards to identify 24 hind limb length-associated genetic loci. For two loci, positive-effect alleles occur in high frequency and overlap genomic regions that are highly differentiated from the populations naive to fire ants. Collectively, these findings represent plausible genetic adaptations in response to fire ant invasion, whereby morphological differentiation may increase survival against swarming ants and altered immune responses may allow the exploitation of a novel, toxic food resource.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558209">
        <p class="paperTitle">Coupling of 12 chromosomal inversions maintains a strong barrier to gene flow between ecotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le Moan, A.; Stankowski, S.; Rafajlovic, M.; Martinez Ortega, O.; Faria, R.; Butlin, R.; Johannesson, K.</p>
        <p class="info">Score: 9.9, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558209' target='https://doi.org/10.1101/2023.09.18.558209'> 10.1101/2023.09.18.558209</a></p>
        <p class="abstract">Chromosomal rearrangements lead to the coupling of reproductive barriers, but whether and how they contribute to completion of speciation remains unclear. Marine snails of the genus Littorina repeatedly form hybrid zones between taxa segregating for inversion arrangements, providing opportunities to study this question. Here, we analysed two adjacent transects across hybrid zones between large and dwarf ecotypes of Littorina fabalis covering wave exposure gradients in a Swedish island. Applying whole-genome sequences we found 12 putative inversions reaching near differential fixation between the opposite ends of each transect, and being in strong linkage disequilibrium. These inversions cover 20% of the genome and carry 93% of divergent SNPs. Bimodal hybrid zones in both transects indicate that the two ecotypes of Littorina fabalis maintain their genetic and phenotypic integrity following contact due to strong coupling between inversion clines that strengthened the reproductive barrier. The bimodality resulting from the linked inversions extends into collinear parts of the genome, suggesting a genome-wide coupling. Demographic inference suggests that the coupling built up during a period of allopatry, and has been maintained for more than 1K generations after secondary contact. Overall, this study shows that the coupling of multiple chromosomal inversions contributes to strong reproductive isolation. Importantly, two of the inversions overlap with inverted genomic regions associated with ecotype differences in a closely-related species (L. saxatilis), suggesting the same regions, with similar structural variants, repeatedly contribute to ecotype evolution in distinct species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557766">
        <p class="paperTitle">Patterns of genomic variation reveal a single evolutionary origin of the wild allotetraploid Mimulus sookensis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitener, M. R.; Mangelson, H.; Sweigart, A. L.</p>
        <p class="info">Score: 8.9, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557766' target='https://doi.org/10.1101/2023.09.14.557766'> 10.1101/2023.09.14.557766</a></p>
        <p class="abstract">Polyploidy occurs across the tree of life and is especially common in plants. Because newly formed cytotypes are often incompatible with their progenitors, polyploidy is also said to trigger &#34;instantaneous&#34; speciation. If a polyploid can self-fertilize or reproduce asexually, it is even possible for one individual to produce an entirely new lineage. How often this extreme scenario occurs is unclear, with most studies of wild polyploids reporting multiple origins. Here, we explore the evolutionary history of the wild allotetraploid Mimulus sookensis, which was formed through hybridization between self-compatible, diploid species in the Mimulus guttatus complex. We generate a chromosome-scale reference assembly for M. sookensis and define its distinct subgenomes. Despite previous reports suggesting multiple origins of this highly selfing polyploid, we discover patterns of population genomic variation that provide unambiguous support for a single origin, which we estimate occurred [~]71,000 years ago. One M. sookensis subgenome is clearly derived from the selfer M. nasutus, which, based on organellar variation, also appears to be the maternal progenitor. The ancestor of the other subgenome is less certain, but it shares variation with both M. decorus and M. guttatus, two outcrossing diploids that overlap broadly with M. sookensis. Whatever its precise ancestry, this study establishes M. sookensis as an example of instantaneous speciation, likely facilitated by the polyploids predisposition to self-fertilize. With a reference genome for M. sookensis now available and its origin clarified, this wild tetraploid is poised to become a model for understanding the genetic and evolutionary mechanisms of polyploid persistence.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
