<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The Selection Landscape and Genetic Legacy of Ancient Eurasians
Authors: Irving-Pease, E. K.; Refoyo-Martinez, A.; Barrie, W.; Ingason, A.; Pearson, A.; Fischer, A.; Sjögren, K.-G.; Halgren, A. S.; Macleod, R.; Demeter, F.; Henriksen, R. A.; Vimala, T.; McColl, H.; Vaughn, A.; Speidel, L.; Stern, A. J.; Scorrano, G.; Ramsoe, A.; Schork, A. J.; Rosengren, A.; Zhao, L.; Kristiansen, K.; Iversen, A. K. N.; Fugger, L.; Sudmant, P. H.; Lawson, D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="The Selection Landscape and Genetic Legacy of Ancient Eurasians
Authors: Irving-Pease, E. K.; Refoyo-Martinez, A.; Barrie, W.; Ingason, A.; Pearson, A.; Fischer, A.; Sjögren, K.-G.; Halgren, A. S.; Macleod, R.; Demeter, F.; Henriksen, R. A.; Vimala, T.; McColl, H.; Vaughn, A.; Speidel, L.; Stern, A. J.; Scorrano, G.; Ramsoe, A.; Schork, A. J.; Rosengren, A.; Zhao, L.; Kristiansen, K.; Iversen, A. K. N.; Fugger, L.; Sudmant, P. H.; Lawson, D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="The Selection Landscape and Genetic Legacy of Ancient Eurasians
Authors: Irving-Pease, E. K.; Refoyo-Martinez, A.; Barrie, W.; Ingason, A.; Pearson, A.; Fischer, A.; Sjögren, K.-G.; Halgren, A. S.; Macleod, R.; Demeter, F.; Henriksen, R. A.; Vimala, T.; McColl, H.; Vaughn, A.; Speidel, L.; Stern, A. J.; Scorrano, G.; Ramsoe, A.; Schork, A. J.; Rosengren, A.; Zhao, L.; Kristiansen, K.; Iversen, A. K. N.; Fugger, L.; Sudmant, P. H.; Lawson, D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "The Selection Landscape and Genetic Legacy of Ancient Eurasians\nAuthors: Irving-Pease, E. K.; Refoyo-Martinez, A.; Barrie, W.; Ingason, A.; Pearson, A.; Fischer, A.; Sjögren, K.-G.; Halgren, A. S.; Macleod, R.; Demeter, F.; Henriksen, R. A.; Vimala, T.; McColl, H.; Vaughn, A.; Speidel, L.; Stern, A. J.; Scorrano, G.; Ramsoe, A.; Schork, A. J.; Rosengren, A.; Zhao, L.; Kristiansen, K.; Iversen, A. K. N.; Fugger, L.; Sudmant, P. H.; Lawson, D.",
  "keywords": [
    
  ],
  "articleBody": " The Selection Landscape and Genetic Legacy of Ancient Eurasians\nAuthors: Irving-Pease, E. K.; Refoyo-Martinez, A.; Barrie, W.; Ingason, A.; Pearson, A.; Fischer, A.; Sjögren, K.-G.; Halgren, A. S.; Macleod, R.; Demeter, F.; Henriksen, R. A.; Vimala, T.; McColl, H.; Vaughn, A.; Speidel, L.; Stern, A. J.; Scorrano, G.; Ramsoe, A.; Schork, A. J.; Rosengren, A.; Zhao, L.; Kristiansen, K.; Iversen, A. K. N.; Fugger, L.; Sudmant, P. H.; Lawson, D. J.; Durbin, R.; Korneliussen, T.; Werge, T.; Allentoft, M. E.; Sikora, M.; Nielsen, R.; Racimo, F.; Willerslev, E.\nScore: 72.1, Published: 2023-10-18 DOI: 10.1101/2022.09.22.509027\nThe Holocene (beginning [~]12,000 years ago) encompassed some of the most significant changes in human evolution, with far-reaching consequences for the dietary, physical, and mental health of present-day populations. Using a dataset of \u003e1600 imputed ancient genomes 1, we modelled the selection landscape during the transition from hunting and gathering, to farming and pastoralism across West Eurasia. We identify major selection signals related to metabolism, including that selection at the FADS cluster began earlier than previously reported, and that selection near the LCT locus predates the emergence of the lactase persistence allele by thousands of years. We also find strong selection in the HLA region, possibly due to increased exposure to pathogens during the Bronze Age. Using ancient individuals to infer local ancestry tracts in \u003e400,000 samples from the UK Biobank, we identify widespread differences in the distribution of Mesolithic, Neolithic, and Bronze Age ancestries across Eurasia. By calculating ancestry-specific polygenic risk scores, we show that height differences between Northern and Southern Europe are associated with differential Steppe ancestry, rather than selection, and that risk alleles for mood-related phenotypes are enriched for Neolithic farmer ancestry, while risk alleles for diabetes and Alzheimers disease are enriched for Western Hunter-gatherer ancestry. Our results suggest that ancient selection and migration were major contributors to the distribution of phenotypic diversity in present-day Europeans.\nEvolution of novel sensory organs in fish with legs\nAuthors: Allard, C. A.; Herbert, A. L.; Krueger, S. P.; Liang, Q.; Walsh, B. L.; Rhyne, A.; Gourlay, A.; Seminara, A.; Baldwin, M. W.; Kingsley, D. M.; Bellono, N.\nScore: 30.1, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562285\nHow do animals evolve new traits? Sea robins are unusual \"walking\" fishes that use leg-like appendages to navigate the seafloor. Here, we show that legs are bona fide sense organs that mediate the unique ability to localize and uncover buried prey. We then probe the developmental and physiological basis of these novel sense organs as a striking example of a major trait gain in evolution. We find certain sea robin species have legs with unique end-organs called papillae that mediate enhanced mechanical and chemical sensitivity to enable predatory digging behavior. Papillae exhibit dense innervation from touch-sensitive neurons, noncanonical epithelial taste receptors, and chemical sensitivity that drives predatory digging behavior. Using a combination of developmental analyses, crosses between species with and without papillae, and interspecies comparisons of sea robins from around the world, we demonstrate that papillae represent a key evolutionary innovation associated with behavioral niche expansion on the seafloor. These discoveries provide a conceptual framework for understanding how molecular, cellular, and tissue-scale adaptations integrate to produce novel organismic traits and behavior.\nThe genetic basis of novel trait gain in walking fish\nAuthors: Herbert, A. L.; Allard, C. A.; McCoy, M. J.; Wucherpfennig, J. I.; Krueger, S. P.; Chen, H. I.; Gourlay, A. N.; Jackson, K. D.; Abbo, L. A.; Bennett, S. H.; Sears, J. D.; Rhyne, A. L.; Bellono, N. W.; Kingsley, D. M.\nScore: 28.3, Published: 2023-10-14 DOI: 10.1101/2023.10.14.562356\nA major goal in biology is to understand how organisms evolve novel traits. Multiple studies have identified genes contributing to regressive evolution, the loss of structures that existed in a recent ancestor. However, fewer examples exist for genes underlying constructive evolution, the gain of novel structures and capabilities in lineages that previously lacked them. Sea robins are fish that have evolved enlarged pectoral fins, six mobile locomotory fin rays (legs) and six novel macroscopic lobes in the central nervous system (CNS) that innervate the corresponding legs. Here, we establish successful husbandry and use a combination of transcriptomics, CRISPR-Cas9 editing, and behavioral assays to identify key transcription factors that are required for leg formation and function in sea robins. We also generate hybrids between two sea robin species with distinct leg morphologies and use allele-specific expression analysis and gene editing to explore the genetic basis of species-specific trait diversity, including a novel sensory gain of function. Collectively, our study establishes sea robins as a new model for studying the genetic basis of novel organ formation, and demonstrates a crucial role for the conserved limb gene tbx3a in the evolution of chemosensory legs in walking fish.\nSerial dilution shapes genetic variation and defines conservation units in Asian elephants\nAuthors: Khan, A.; Sil, M.; Thekaekara, T.; Sinha, I.; Khurana, R.; Sukumar, R.; Ramakrishnan, U.\nScore: 20.8, Published: 2023-09-24 DOI: 10.1101/2023.09.04.556173\nMegaherbivores are primary consumers who provide unique ecosystem services. Given their body size, they are disproportionately threatened in the Anthropocene. Asian elephants are the largest extant terrestrial megaherbivores native to Asia, with 60% of the population found in India. Despite their ecological and cultural importance, the management/conservation units, genetic history, diversity and threats remain understudied. We re-sequenced 31 whole genomes (between 11X - 32X) from all known elephant landscapes in India and identified five management/conservation units corresponding to elephants in northern India, central India and three in southern India. The genetic data reveal signatures of serial colonisation, and a dilution of diversity from north to south of India. The northern populations diverged from other populations more than 70,000 years ago, and have higher genetic diversity, with low inbreeding/high effective size (Pi = 0.0016{+/-}0.0001; FROH\u003e 1MB= 0.09{+/-}0.03). Two of three populations in southern India have low diversity and are inbred with much lower effective sizes than current populations sizes (Pi = 0.0014{+/-}0.00009 and 0.0015{+/-}0.0001; FROH\u003e 1MB= 0.25{+/-}0.09 and 0.17{+/-}0.02). Additionally, future generations are expected to be more inbred since pairs of extant elephants have large tracts of the genome that are already identical. Analyses of genetic load reveals purging of potentially high-effect deleterious alleles in the southern populations and potential dilution of all deleterious alleles from north to south in India. However, southern Indian elephants are highly homozygous for all the deleterious alleles that persist, despite dilution and purging. High homozygosity of deleterious alleles, coupled with low neutral genetic diversity make them high priority for conservation and management attention. Most surprisingly, our study suggests that patterns of genetic diversity and genetic load can correspond to geographic signatures of serial founding events even in large mobile endangered species.\nSkimming genomes for systematics and DNA barcodes of corals\nAuthors: Quattrini, A. M.; McCartin, L. J.; Easton, E. E.; Horowitz, J.; Wirshing, H. H.; Bowers, H.; Mitchell, K.; Sei, M.; McFadden, C. S.; Herrera, S.\nScore: 12.3, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562770\n1: Numerous genomic methods developed over the past two decades have enabled the discovery and extraction of orthologous loci to help resolve phylogenetic relationships across various taxa and scales. Genome skimming (or low-coverage whole genome sequencing) remains a low-cost, promising method to not only extract high-copy loci, but also 100s to 1000s of phylogenetically informative single-copy nuclear loci (e.g., ultraconserved elements [UCEs] and exons) from contemporary and historical museum samples. The subphylum Anthozoa, which includes important ecosystem engineers (e.g., stony corals, black corals, anemones and octocorals) in the marine environment, is in critical need of phylogenetic resolution and thus might benefit from a genome-skimming approach. 2: Genome skimming was conducted on 242 hexacorals and octocorals collected from 1890 to 2022. Using previously developed target-capture baitsets, we bioinformatically obtained UCEs and exons from the genome-skimming data and incorporated them with data from previously published target-capture studies. We also extracted partial to whole mitogenomes and nuclear rRNA genes from the skim data. 3: The mean number of UCE and exon loci extracted from the genome skimming data was 1,837 {+/-} 662 SD for octocorals and 1,422 {+/-} 720 loci for hexacorals; phylogenetic relationships were well resolved within each class. A mean of 1,422 {+/-} 720 loci were obtained from the historical museum specimens, with 1,253 loci recovered from the oldest specimen collected in 1886 and 1,336 loci recovered from a holotype. The nuclear rRNA genes and the majority of mitochondrial genes were successfully obtained from \u003e95% of samples. Out of 99 circularized mitogenomes, 88% were recovered in samples from which we obtained \u003e15M paired-end (PE) reads (\u003e30M total reads); there was more variability in whether mitogenomes were circularized or not in samples with \u003c15M PE reads. 4: Bioinformatically pulling UCEs, exons, mitochondrial genomes, and nuclear rRNA genes from genome skimming is a viable and low-cost option for phylogenetic studies. This approach can be used to review and support taxonomic revisions and reconstruct evolutionary histories, including historical museum and type specimens.\nBayesian inference of relative fitness on high-throughput pooled competition assays\nAuthors: Razo-Mejia, M.; Mani, M.; Petrov, D.\nScore: 11.0, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562365\nThe tracking of lineage frequencies via DNA barcode sequencing enables the quantification of microbial fitness. However, experimental noise coming from biotic and abiotic sources complicates the computation of a reliable inference. We present a Bayesian pipeline to infer relative microbial fitness from high-throughput lineage tracking assays. Our model accounts for multiple sources of noise and propagates uncertainties throughout all parameters in a systematic way. Furthermore, using modern variational inference methods based on automatic differentiation, we are able to scale the inference to a large number of unique barcodes. We extend this core model to analyze multi-environment assays, replicate experiments, and barcodes linked to genotypes. On simulations, our method recovers known parameters within posterior credible intervals. This work provides a generalizable Bayesian framework to analyze lineage tracking experiments. The accompanying open-source software library enables the adoption of principled statistical methods in experimental evolution.\nDietary restriction extends lifespan across different temperatures in the fly\nAuthors: Phillips, E. J.; Simons, M.\nScore: 10.8, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562222\nDietary restriction (DR) has been consistently shown to extend lifespan across a range of taxa. However, it has recently been reported that DR does not extend lifespan at certain, namely lower, temperatures in flies (Drosophila melanogaster). Similar to the interpretation of other findings that appear to question DRs universality, this finding has been interpreted as being an artefact of benign laboratory conditions. Here we re-test this hypothesis, now using a strain that shows robust lifespan extension at 25{degrees}C (across three prior experiments), and using a range of 5 diets across two temperatures, 18{degrees}C and 21{degrees}C. We found the DR longevity response to be robust, extending lifespan irrespective of temperature. We measured fecundity as a positive control for the DR phenotype, and found, as predicted, that DR reduces egg laying. It will be important for results that question DR as a phenotype to not be overinterpreted readily, as with a substantially larger sample size and a larger range of diets we were unable to replicate this prior work. Differences in experimental setup, genetic lines used and variation in the diet-lifespan reaction norm we suggest are responsible for this discrepancy. In addition, starting with a strain and conditions that show a lifespan extension by DR, as we do here, and then changing environment and/or genotype promises a more robust test of DR modulating factors.\nAllopolyploidization from two dioecious ancestors leads to recurrent evolution of sex chromosomes and reversion to autosomes\nAuthors: He, L.; Wang, Y.; Wang, Y.; Zhang, R.-G.; Wang, Y.; Hoerandl, E.; Mank, J. E.; Ming, R.\nScore: 9.2, Published: 2023-10-19 DOI: 10.1101/2023.10.18.562857\nPolyploidization presents an unusual challenge for species with sex chromosomes, as it can lead to complex combinations of sex chromosomes that disrupt reproductive development. This is particularly true for allopolyploidization between species with different sex chromosome systems. Here we assemble haplotype-resolved chromosome-level genomes of a female allotetraploid weeping willow (Salix babylonica) and a male diploid Salix dunnii using Hi-C and PacBio HiFi reads. We use phylogenomics of nuclear and plastid genomes to show that weeping willow arose from crosses between female ancestor from the Salix clade, having XY sex chromosomes on chromosome 7, with a male ancestor from the Vetrix clade, having ancestral XY sex chromosomes on chromosome 15. Our analysis reveals that weeping willow has one pair sex chromosomes, ZW on chromosome 15, that derive from the ancestral XY sex chromosomes in the Vetrix-clade male ancestor, and the X chromosomes on chromosome 7 from the Salix-clade female ancestor has reverted to an autosome. Taken together, our results point to rapid evolution and reversion of sex chromosomes following allopolyploidization in weeping willow.\nPhylogenomics, biogeography, and trait evolution of the Boletaceae (Boletales, Agaricomycetes, Basidiomycota)\nAuthors: Tremble, K.; Henkel, T.; Bradshaw, A.; Domnauer, C.; Furci, G.; Tham, L. X.; Aime, C.; Moncalvo, J.-M.; Dentinger, B.\nScore: 9.0, Published: 2023-10-21 DOI: 10.1101/2023.10.18.563010\nThe species-rich porcini mushroom family Boletaceae is a widespread and well-known group of ectomycorrhizal (ECM) mushroom-forming fungi that has eluded intrafamilial phylogenetic resolution despite many attempts using morphological traits and multi-locus molecular datasets. In this study, we present a genome-wide molecular dataset of 1764 single-copy gene families from a global sampling of 418 Boletaceae specimens. The resulting phylogenetic analysis has strong statistical support for most branches of the tree, including the first statistically robust backbone. The enigmatic Phylloboletellus chloephorus from non-ECM subtropical forests in northwestern Argentina is recovered as an early diverging lineage within the Boletaceae. Time-calibrated branch lengths estimate that the family first arose in the early-to-mid Cretaceous and underwent a rapid radiation in the Eocene, possibly when the ECM nutritional mode arose with the emergence and diversification of ECM angiosperms. Biogeographic reconstructions reveal a complex history of vicariance and episodic long-distance dispersal correlated with historical geologic events, including Gondwanan origins and cladogenesis patterns that parallel its fragmentation. Ancestral state reconstruction of morphological traits predicts that the ancestor of the Boletaceae was lamellate with ornamented basidiospores, contrary to the most common bolete sporocarp morphology. Transition rates indicate that the lamellate hymenophore and sequestrate sporocarp are reversible traits. Together, this study represents the most comprehensively sampled, data-rich molecular phylogeny of the Boletaceae to date, enabling robust inferences of trait evolution and biogeography in the group.\nContrasting patterns of presence-absence variation of NLRS within S. chilense are mainly shaped by past demographic history\nAuthors: Silva Arias, G. A.; Gagnon, E.; Hembrom, S.; Fastner, A.; Khan, M. R.; Stam, R.; Tellier, A.\nScore: 8.4, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562278\nUnderstanding the evolution of pathogen resistance genes (also known as NLRs) within a species requires a comprehensive examination of factors that affect gene loss and gain. We present a new reference genome of Solanum chilense, that leads to an increased number and more accurate annotation of NLRs. Next, using a target-capture approach, we quantify the presence-absence variation (PAV) of NLR loci across 20 populations from different habitats. We build a rigorous pipeline to validate the identification of PAV of NLRs, then show that PAV is larger within populations than between populations, suggesting that maintenance of NLR diversity is linked to population dynamics. Furthermore, the amount of PAV is not correlated with the NLR presence in gene clusters in the genome, but rather with the past demographic history of the species, with loss of NLRs in diverging populations at the distribution edges and smaller population sizes. Finally, using a redundancy analysis, we find limited evidence of PAV being linked to environmental gradients. Our results contradict the classic assumptions of the important selective role of PAV for NLRs, and suggest that NLRs PAV is driven by random processes (and weak selection) in an outcrossing plant with high nucleotide diversity.\n",
  "wordCount" : "2590",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.09.22.509027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.09.22.509027" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.09.22.509027">
        <p class="paperTitle">The Selection Landscape and Genetic Legacy of Ancient Eurasians</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.09.22.509027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.09.22.509027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Irving-Pease, E. K.; Refoyo-Martinez, A.; Barrie, W.; Ingason, A.; Pearson, A.; Fischer, A.; Sjögren, K.-G.; Halgren, A. S.; Macleod, R.; Demeter, F.; Henriksen, R. A.; Vimala, T.; McColl, H.; Vaughn, A.; Speidel, L.; Stern, A. J.; Scorrano, G.; Ramsoe, A.; Schork, A. J.; Rosengren, A.; Zhao, L.; Kristiansen, K.; Iversen, A. K. N.; Fugger, L.; Sudmant, P. H.; Lawson, D. J.; Durbin, R.; Korneliussen, T.; Werge, T.; Allentoft, M. E.; Sikora, M.; Nielsen, R.; Racimo, F.; Willerslev, E.</p>
        <p class="info">Score: 72.1, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.09.22.509027' target='https://doi.org/10.1101/2022.09.22.509027'> 10.1101/2022.09.22.509027</a></p>
        <p class="abstract">The Holocene (beginning [~]12,000 years ago) encompassed some of the most significant changes in human evolution, with far-reaching consequences for the dietary, physical, and mental health of present-day populations. Using a dataset of &gt;1600 imputed ancient genomes 1, we modelled the selection landscape during the transition from hunting and gathering, to farming and pastoralism across West Eurasia. We identify major selection signals related to metabolism, including that selection at the FADS cluster began earlier than previously reported, and that selection near the LCT locus predates the emergence of the lactase persistence allele by thousands of years. We also find strong selection in the HLA region, possibly due to increased exposure to pathogens during the Bronze Age. Using ancient individuals to infer local ancestry tracts in &gt;400,000 samples from the UK Biobank, we identify widespread differences in the distribution of Mesolithic, Neolithic, and Bronze Age ancestries across Eurasia. By calculating ancestry-specific polygenic risk scores, we show that height differences between Northern and Southern Europe are associated with differential Steppe ancestry, rather than selection, and that risk alleles for mood-related phenotypes are enriched for Neolithic farmer ancestry, while risk alleles for diabetes and Alzheimers disease are enriched for Western Hunter-gatherer ancestry. Our results suggest that ancient selection and migration were major contributors to the distribution of phenotypic diversity in present-day Europeans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562285">
        <p class="paperTitle">Evolution of novel sensory organs in fish with legs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Allard, C. A.; Herbert, A. L.; Krueger, S. P.; Liang, Q.; Walsh, B. L.; Rhyne, A.; Gourlay, A.; Seminara, A.; Baldwin, M. W.; Kingsley, D. M.; Bellono, N.</p>
        <p class="info">Score: 30.1, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562285' target='https://doi.org/10.1101/2023.10.14.562285'> 10.1101/2023.10.14.562285</a></p>
        <p class="abstract">How do animals evolve new traits? Sea robins are unusual &#34;walking&#34; fishes that use leg-like appendages to navigate the seafloor. Here, we show that legs are bona fide sense organs that mediate the unique ability to localize and uncover buried prey. We then probe the developmental and physiological basis of these novel sense organs as a striking example of a major trait gain in evolution. We find certain sea robin species have legs with unique end-organs called papillae that mediate enhanced mechanical and chemical sensitivity to enable predatory digging behavior. Papillae exhibit dense innervation from touch-sensitive neurons, noncanonical epithelial taste receptors, and chemical sensitivity that drives predatory digging behavior. Using a combination of developmental analyses, crosses between species with and without papillae, and interspecies comparisons of sea robins from around the world, we demonstrate that papillae represent a key evolutionary innovation associated with behavioral niche expansion on the seafloor. These discoveries provide a conceptual framework for understanding how molecular, cellular, and tissue-scale adaptations integrate to produce novel organismic traits and behavior.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562356">
        <p class="paperTitle">The genetic basis of novel trait gain in walking fish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herbert, A. L.; Allard, C. A.; McCoy, M. J.; Wucherpfennig, J. I.; Krueger, S. P.; Chen, H. I.; Gourlay, A. N.; Jackson, K. D.; Abbo, L. A.; Bennett, S. H.; Sears, J. D.; Rhyne, A. L.; Bellono, N. W.; Kingsley, D. M.</p>
        <p class="info">Score: 28.3, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562356' target='https://doi.org/10.1101/2023.10.14.562356'> 10.1101/2023.10.14.562356</a></p>
        <p class="abstract">A major goal in biology is to understand how organisms evolve novel traits. Multiple studies have identified genes contributing to regressive evolution, the loss of structures that existed in a recent ancestor. However, fewer examples exist for genes underlying constructive evolution, the gain of novel structures and capabilities in lineages that previously lacked them. Sea robins are fish that have evolved enlarged pectoral fins, six mobile locomotory fin rays (legs) and six novel macroscopic lobes in the central nervous system (CNS) that innervate the corresponding legs. Here, we establish successful husbandry and use a combination of transcriptomics, CRISPR-Cas9 editing, and behavioral assays to identify key transcription factors that are required for leg formation and function in sea robins. We also generate hybrids between two sea robin species with distinct leg morphologies and use allele-specific expression analysis and gene editing to explore the genetic basis of species-specific trait diversity, including a novel sensory gain of function. Collectively, our study establishes sea robins as a new model for studying the genetic basis of novel organ formation, and demonstrates a crucial role for the conserved limb gene tbx3a in the evolution of chemosensory legs in walking fish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556173">
        <p class="paperTitle">Serial dilution shapes genetic variation and defines conservation units in Asian elephants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khan, A.; Sil, M.; Thekaekara, T.; Sinha, I.; Khurana, R.; Sukumar, R.; Ramakrishnan, U.</p>
        <p class="info">Score: 20.8, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556173' target='https://doi.org/10.1101/2023.09.04.556173'> 10.1101/2023.09.04.556173</a></p>
        <p class="abstract">Megaherbivores are primary consumers who provide unique ecosystem services. Given their body size, they are disproportionately threatened in the Anthropocene. Asian elephants are the largest extant terrestrial megaherbivores native to Asia, with 60% of the population found in India. Despite their ecological and cultural importance, the management/conservation units, genetic history, diversity and threats remain understudied. We re-sequenced 31 whole genomes (between 11X - 32X) from all known elephant landscapes in India and identified five management/conservation units corresponding to elephants in northern India, central India and three in southern India. The genetic data reveal signatures of serial colonisation, and a dilution of diversity from north to south of India. The northern populations diverged from other populations more than 70,000 years ago, and have higher genetic diversity, with low inbreeding/high effective size (Pi = 0.0016{&#43;/-}0.0001; FROH&gt; 1MB= 0.09{&#43;/-}0.03). Two of three populations in southern India have low diversity and are inbred with much lower effective sizes than current populations sizes (Pi = 0.0014{&#43;/-}0.00009 and 0.0015{&#43;/-}0.0001; FROH&gt; 1MB= 0.25{&#43;/-}0.09 and 0.17{&#43;/-}0.02). Additionally, future generations are expected to be more inbred since pairs of extant elephants have large tracts of the genome that are already identical. Analyses of genetic load reveals purging of potentially high-effect deleterious alleles in the southern populations and potential dilution of all deleterious alleles from north to south in India. However, southern Indian elephants are highly homozygous for all the deleterious alleles that persist, despite dilution and purging. High homozygosity of deleterious alleles, coupled with low neutral genetic diversity make them high priority for conservation and management attention. Most surprisingly, our study suggests that patterns of genetic diversity and genetic load can correspond to geographic signatures of serial founding events even in large mobile endangered species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562770">
        <p class="paperTitle">Skimming genomes for systematics and DNA barcodes of corals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Quattrini, A. M.; McCartin, L. J.; Easton, E. E.; Horowitz, J.; Wirshing, H. H.; Bowers, H.; Mitchell, K.; Sei, M.; McFadden, C. S.; Herrera, S.</p>
        <p class="info">Score: 12.3, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562770' target='https://doi.org/10.1101/2023.10.17.562770'> 10.1101/2023.10.17.562770</a></p>
        <p class="abstract">1: Numerous genomic methods developed over the past two decades have enabled the discovery and extraction of orthologous loci to help resolve phylogenetic relationships across various taxa and scales. Genome skimming (or low-coverage whole genome sequencing) remains a low-cost, promising method to not only extract high-copy loci, but also 100s to 1000s of phylogenetically informative single-copy nuclear loci (e.g., ultraconserved elements [UCEs] and exons) from contemporary and historical museum samples. The subphylum Anthozoa, which includes important ecosystem engineers (e.g., stony corals, black corals, anemones and octocorals) in the marine environment, is in critical need of phylogenetic resolution and thus might benefit from a genome-skimming approach. 2: Genome skimming was conducted on 242 hexacorals and octocorals collected from 1890 to 2022. Using previously developed target-capture baitsets, we bioinformatically obtained UCEs and exons from the genome-skimming data and incorporated them with data from previously published target-capture studies. We also extracted partial to whole mitogenomes and nuclear rRNA genes from the skim data. 3: The mean number of UCE and exon loci extracted from the genome skimming data was 1,837 {&#43;/-} 662 SD for octocorals and 1,422 {&#43;/-} 720 loci for hexacorals; phylogenetic relationships were well resolved within each class. A mean of 1,422 {&#43;/-} 720 loci were obtained from the historical museum specimens, with 1,253 loci recovered from the oldest specimen collected in 1886 and 1,336 loci recovered from a holotype. The nuclear rRNA genes and the majority of mitochondrial genes were successfully obtained from &gt;95% of samples. Out of 99 circularized mitogenomes, 88% were recovered in samples from which we obtained &gt;15M paired-end (PE) reads (&gt;30M total reads); there was more variability in whether mitogenomes were circularized or not in samples with &lt;15M PE reads. 4: Bioinformatically pulling UCEs, exons, mitochondrial genomes, and nuclear rRNA genes from genome skimming is a viable and low-cost option for phylogenetic studies. This approach can be used to review and support taxonomic revisions and reconstruct evolutionary histories, including historical museum and type specimens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562365">
        <p class="paperTitle">Bayesian inference of relative fitness on high-throughput pooled competition assays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razo-Mejia, M.; Mani, M.; Petrov, D.</p>
        <p class="info">Score: 11.0, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562365' target='https://doi.org/10.1101/2023.10.14.562365'> 10.1101/2023.10.14.562365</a></p>
        <p class="abstract">The tracking of lineage frequencies via DNA barcode sequencing enables the quantification of microbial fitness. However, experimental noise coming from biotic and abiotic sources complicates the computation of a reliable inference. We present a Bayesian pipeline to infer relative microbial fitness from high-throughput lineage tracking assays. Our model accounts for multiple sources of noise and propagates uncertainties throughout all parameters in a systematic way. Furthermore, using modern variational inference methods based on automatic differentiation, we are able to scale the inference to a large number of unique barcodes. We extend this core model to analyze multi-environment assays, replicate experiments, and barcodes linked to genotypes. On simulations, our method recovers known parameters within posterior credible intervals. This work provides a generalizable Bayesian framework to analyze lineage tracking experiments. The accompanying open-source software library enables the adoption of principled statistical methods in experimental evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562222" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562222">
        <p class="paperTitle">Dietary restriction extends lifespan across different temperatures in the fly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phillips, E. J.; Simons, M.</p>
        <p class="info">Score: 10.8, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562222' target='https://doi.org/10.1101/2023.10.13.562222'> 10.1101/2023.10.13.562222</a></p>
        <p class="abstract">Dietary restriction (DR) has been consistently shown to extend lifespan across a range of taxa. However, it has recently been reported that DR does not extend lifespan at certain, namely lower, temperatures in flies (Drosophila melanogaster). Similar to the interpretation of other findings that appear to question DRs universality, this finding has been interpreted as being an artefact of benign laboratory conditions. Here we re-test this hypothesis, now using a strain that shows robust lifespan extension at 25{degrees}C (across three prior experiments), and using a range of 5 diets across two temperatures, 18{degrees}C and 21{degrees}C. We found the DR longevity response to be robust, extending lifespan irrespective of temperature. We measured fecundity as a positive control for the DR phenotype, and found, as predicted, that DR reduces egg laying. It will be important for results that question DR as a phenotype to not be overinterpreted readily, as with a substantially larger sample size and a larger range of diets we were unable to replicate this prior work. Differences in experimental setup, genetic lines used and variation in the diet-lifespan reaction norm we suggest are responsible for this discrepancy. In addition, starting with a strain and conditions that show a lifespan extension by DR, as we do here, and then changing environment and/or genotype promises a more robust test of DR modulating factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562857">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562857" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562857">
        <p class="paperTitle">Allopolyploidization from two dioecious ancestors leads to recurrent evolution of sex chromosomes and reversion to autosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562857" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562857" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, L.; Wang, Y.; Wang, Y.; Zhang, R.-G.; Wang, Y.; Hoerandl, E.; Mank, J. E.; Ming, R.</p>
        <p class="info">Score: 9.2, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562857' target='https://doi.org/10.1101/2023.10.18.562857'> 10.1101/2023.10.18.562857</a></p>
        <p class="abstract">Polyploidization presents an unusual challenge for species with sex chromosomes, as it can lead to complex combinations of sex chromosomes that disrupt reproductive development. This is particularly true for allopolyploidization between species with different sex chromosome systems. Here we assemble haplotype-resolved chromosome-level genomes of a female allotetraploid weeping willow (Salix babylonica) and a male diploid Salix dunnii using Hi-C and PacBio HiFi reads. We use phylogenomics of nuclear and plastid genomes to show that weeping willow arose from crosses between female ancestor from the Salix clade, having XY sex chromosomes on chromosome 7, with a male ancestor from the Vetrix clade, having ancestral XY sex chromosomes on chromosome 15. Our analysis reveals that weeping willow has one pair sex chromosomes, ZW on chromosome 15, that derive from the ancestral XY sex chromosomes in the Vetrix-clade male ancestor, and the X chromosomes on chromosome 7 from the Salix-clade female ancestor has reverted to an autosome. Taken together, our results point to rapid evolution and reversion of sex chromosomes following allopolyploidization in weeping willow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.563010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.563010" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.563010">
        <p class="paperTitle">Phylogenomics, biogeography, and trait evolution of the Boletaceae (Boletales, Agaricomycetes, Basidiomycota)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.563010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.563010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tremble, K.; Henkel, T.; Bradshaw, A.; Domnauer, C.; Furci, G.; Tham, L. X.; Aime, C.; Moncalvo, J.-M.; Dentinger, B.</p>
        <p class="info">Score: 9.0, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.563010' target='https://doi.org/10.1101/2023.10.18.563010'> 10.1101/2023.10.18.563010</a></p>
        <p class="abstract">The species-rich porcini mushroom family Boletaceae is a widespread and well-known group of ectomycorrhizal (ECM) mushroom-forming fungi that has eluded intrafamilial phylogenetic resolution despite many attempts using morphological traits and multi-locus molecular datasets. In this study, we present a genome-wide molecular dataset of 1764 single-copy gene families from a global sampling of 418 Boletaceae specimens. The resulting phylogenetic analysis has strong statistical support for most branches of the tree, including the first statistically robust backbone. The enigmatic Phylloboletellus chloephorus from non-ECM subtropical forests in northwestern Argentina is recovered as an early diverging lineage within the Boletaceae. Time-calibrated branch lengths estimate that the family first arose in the early-to-mid Cretaceous and underwent a rapid radiation in the Eocene, possibly when the ECM nutritional mode arose with the emergence and diversification of ECM angiosperms. Biogeographic reconstructions reveal a complex history of vicariance and episodic long-distance dispersal correlated with historical geologic events, including Gondwanan origins and cladogenesis patterns that parallel its fragmentation. Ancestral state reconstruction of morphological traits predicts that the ancestor of the Boletaceae was lamellate with ornamented basidiospores, contrary to the most common bolete sporocarp morphology. Transition rates indicate that the lamellate hymenophore and sequestrate sporocarp are reversible traits. Together, this study represents the most comprehensively sampled, data-rich molecular phylogeny of the Boletaceae to date, enabling robust inferences of trait evolution and biogeography in the group.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562278">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562278" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562278">
        <p class="paperTitle">Contrasting patterns of presence-absence variation of NLRS within S. chilense are mainly shaped by past demographic history</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562278" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562278" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silva Arias, G. A.; Gagnon, E.; Hembrom, S.; Fastner, A.; Khan, M. R.; Stam, R.; Tellier, A.</p>
        <p class="info">Score: 8.4, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562278' target='https://doi.org/10.1101/2023.10.13.562278'> 10.1101/2023.10.13.562278</a></p>
        <p class="abstract">Understanding the evolution of pathogen resistance genes (also known as NLRs) within a species requires a comprehensive examination of factors that affect gene loss and gain. We present a new reference genome of Solanum chilense, that leads to an increased number and more accurate annotation of NLRs. Next, using a target-capture approach, we quantify the presence-absence variation (PAV) of NLR loci across 20 populations from different habitats. We build a rigorous pipeline to validate the identification of PAV of NLRs, then show that PAV is larger within populations than between populations, suggesting that maintenance of NLR diversity is linked to population dynamics. Furthermore, the amount of PAV is not correlated with the NLR presence in gene clusters in the genome, but rather with the past demographic history of the species, with loss of NLRs in diverging populations at the distribution edges and smaller population sizes. Finally, using a redundancy analysis, we find limited evidence of PAV being linked to environmental gradients. Our results contradict the classic assumptions of the important selective role of PAV for NLRs, and suggest that NLRs PAV is driven by random processes (and weak selection) in an outcrossing plant with high nucleotide diversity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
