<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA methylation enables recurrent endogenization of giant viruses in an animal relative
Authors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.
Score: 38.7, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574619
5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="DNA methylation enables recurrent endogenization of giant viruses in an animal relative
Authors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.
Score: 38.7, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574619
5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="DNA methylation enables recurrent endogenization of giant viruses in an animal relative
Authors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.
Score: 38.7, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574619
5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "DNA methylation enables recurrent endogenization of giant viruses in an animal relative\nAuthors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.\nScore: 38.7, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574619\n5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification.",
  "keywords": [
    
  ],
  "articleBody": " DNA methylation enables recurrent endogenization of giant viruses in an animal relative\nAuthors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.\nScore: 38.7, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574619\n5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification. Here we show that the protist closely related to animals Amoebidium appalachense features both transposon and gene body methylation, a pattern reminiscent of invertebrates and plants. Unexpectedly, large hypermethylated regions of the Amoebidium genome derive from viral insertions, including hundreds of endogenized giant viruses contributing 14% of the encoded genes, to an extent never reported before in any eukaryotic genome. Using a combination of inhibitors and functional genomic assays, we demonstrate that 5mC silences these giant virus insertions. Moreover, alternative Amoebidium isolates show polymorphic giant virus insertions, highlighting a dynamic process of infection, endogenization and purging. Our results indicate that 5mC is critical for the controlled co-existence of newly acquired viral DNA into eukaryotic genomes, making Amoebidium a unique model to understand the hybrid origins of eukaryotic genomes.\nA 39.8kb flavi-like virus uses a novel strategy for overcoming the RNA virus error threshold\nAuthors: Petrone, M. E.; Grove, J.; Mifsud, J. C. O.; Parry, R. H.; Marzinelli, E. M.; Holmes, E. C.\nScore: 25.0, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574764\nIt is commonly held that there is a fundamental relationship between genome size and error rate, manifest as a notional \"error threshold\" that sets an upper limit on genome sizes. The genome sizes of RNA viruses, which have intrinsically high mutation rates due to a lack of mechanisms for error correction, must therefore be small to avoid accumulating an excessive number of deleterious mutations that will ultimately lead to population extinction. The proposed exceptions to this evolutionary rule are RNA viruses from the order Nidovirales (such as coronaviruses) that encode an error correcting exonuclease, enabling them to reach genome lengths greater than 40kb. The recent discovery of large genome flavi-like viruses (Flaviviridae), which comprise genomes up to 27kb in length yet seemingly do not encode exonuclease domains, has led to the proposal that a proofreading mechanism is required to facilitate the expansion of RNA virus genomes above 30kb. Herein, we describe a 39.8kb flavi-like virus identified in a Haliclona sponge metatranscriptome that does not encode an exonuclease. Structural analysis revealed that this virus may have instead captured bacterial domains associated with nucleic acid metabolism that have not been previously found in RNA viruses. Phylogenetic analysis placed this virus as a divergent pesti-like lineage, such that we have provisionally termed it Maximus pesti-like virus. This virus represents the first instance of a flavi-like virus achieving a genome size comparable to that of the Nidovirales and demonstrates that RNA viruses have evolved multiple solutions to overcome the error threshold.\nAncestral Sequence Reconstruction as a tool to detect and study de novo gene emergence\nAuthors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.\nScore: 13.9, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573862\nNew protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (\u003c1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute \"easy\" cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.\nA complex mechanism translating variation of a simple genetic architecture into alternative life-histories\nAuthors: Verta, J.-P.; Moustakas-Verho, J.; Donner, I.; Frapin, M.; Ruokolainen, A.; Debes, P. V.; Erkinaro, J.; Primmer, C. R.\nScore: 11.1, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574286\nLinking genes to traits is a central goal in biology. Despite progress in discovering genes associated with trait differences, a poor understanding of the functional mechanisms underlying genetic associations leaves us critically far from connecting genetic and phenotypic variation. This knowledge-gap is particularly large in multifaceted phenotypes of ecological relevance such as life-history traits. Using a multiomic dissection of the genotype-phenotype association in a large-effect maturation age gene - the transcription cofactor vestigial-like 3 (vgll3) - in Atlantic salmon (Salmo salar), we show that vgll3 mediates concerted changes of distinct molecular phenotypes associated with puberty in male gonads. Vgll3 genotype conferring early maturity upregulates key genes controlling androgen production, cellular energy and adiposity, and TGF-{beta} signaling, among others, thereby increasing the likelihood of earlier pubertal initiation. Genotype-dependent developmental trajectories are produced through VGLL3 interaction with distinct transcription factors, thus coordinating differential activation of regulatory pathways. These results reveal a genetically simple, yet functionally complex, architecture underlying alternative life-histories where variation in a single major effect gene produces pleiotropic variation in a spectrum of cellular traits. Our results further suggest that evolution in correlated phenotypes such as exemplified by alternative life history strategies may be mediated by a surprisingly simple genetic architecture.\nMaternal Care Leads to the Evolution of Long, Slow Lives\nAuthors: Zipple, M. N.; Reeve, H. K.; Peniston, J. H.\nScore: 10.9, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574353\nAnimals, and mammals in particular, vary widely in their \"pace of life,\" with some species living long lives and reproducing infrequently (slow life histories) and others living short lives and reproducing often (fast life histories). These species also vary in the importance of maternal care in offspring fitness: in some species, offspring are fully independent of their mothers following a brief period of nutritional input, while others display a long period of continued dependence on mothers well after nutritional dependence. Here we hypothesize that these two axes of variation are causally related to each other, such that extended dependence of offspring on maternal presence leads to the evolution of longer lives at the expense of reproduction. We use a combination of deterministic modeling and stochastic agent-based modeling to explore how empirically-observed links between maternal survival and offspring fitness are likely to shape the evolution of mortality and fertility. Each of our modelling approaches leads to the same conclusion: when maternal survival has strong impacts on the survival of offspring and grandoffspring, populations evolve longer lives with less frequent reproduction. Our results suggest the slow life histories of humans and other primates as well as other long-lived, highly social animals such as hyenas, whales, and elephants, are partially the result of the strong maternal care that these animals display. We have designed our models to be readily parameterized with demographic data that is routinely collected by long-term researchers, which will facilitate more thorough testing of our hypothesis. Significance StatementHumans and other primates live longer lives and reproduce less often than other mammals of similar body mass. What is the cause of these long lives? Here we add to existing hypotheses, including the Mother and Grandmother hypotheses, by arguing that these increased lifespans are partially explained by the intense maternal care that many primates express. Using a combination of deterministic and stochastic modeling approaches, informed by empirical data, we show that stronger connections between maternal survival and offspring fitness leads to selection for longer lives and slower reproduction. Our models suggest that the importance of the mother-offspring relationship, which defines much of human and non-human primate lives, lies at the core of the evolution of our long lives.\nConsistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence\nAuthors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.\nScore: 10.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574070\nThe ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects. This reduction in population size leads to a reduction in genetic diversity, which ultimately results in a reduction in the efficiency of natural selection. Then, what is the source of genetic variation which acts as the raw material for ecological and phenotypic diversification in oceanic archipelagos? Transposable elements (TEs) are mobile genetic elements that have been linked to the generation of genetic diversity, and evidence suggests that TE activity and accumulation along the genome can result from reductions in population size. Here, we use the Hawaiian spiny-leg spider radiation (Tetragnatha) to test whether TE accumulation increases due to demographic processes associated with island colonization. We sequenced and quantified TEs in 23 individuals from the spiny-leg radiation and 4 individuals from its sister radiation, the Hawaiian web-building Tetragnatha. Our results show that founder effects resulting from colonization of new islands have not resulted in TE accumulation over evolutionary time. Specifically, we found no evidence for increase in abundance of specific TE superfamilies, nor an accumulation of young TEs in lineages which have recently colonized a new island or are present in islands with active volcanoes. We also found that the DNA/hAT transposon superfamily is by far the most abundant TE superfamily in the Tetragnatha radiation. This work shows that TE abundance has remained constant for the spiny-leg radiation across the archipelago chronosequence, and TE accumulation is not affected by population oscillations associated with island colonization events. Therefore, despite their known role in the generation of genetic diversity, TE activity does not appear to be the mechanism to explain the evolutionary paradox of the insular Tetragnatha spiny-leg radiation.\nHorizontal transfer and recombination fuel Ty4 retrotransposon evolution in Saccharomyces.\nAuthors: Chen, J.; Garfinkel, D. J.; Bergman, C. M.\nScore: 9.8, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572574\nHorizontal transposon transfer (HTT) plays an important role in the evolution of eukaryotic genomes, however the detailed evolutionary history and impact of most HTT events remain to be elucidated. To better understand the process of HTT in closely-related microbial eukaryotes, we studied Ty4 retrotransposon subfamily content and sequence evolution across the genus Saccharomyces using short- and long-read whole genome sequence data, including new PacBio genome assemblies for two S. mikatae strains. We find evidence for multiple independent HTT events introducing the Tsu4 subfamily into specific lineages of S. paradoxus, S. cerevisiae, S. eubayanus, S. kudriavzevii and the ancestor of the S. mikatae/S. jurei species pair. In both S. mikatae and S. kudriavzevii , we identified novel Ty4 clades that were independently generated through recombination between resident and horizontally-transferred subfamilies. Our results reveal that recurrent HTT and lineage-specific extinction events lead to a complex pattern of Ty4 subfamily content across the genus Saccharomyces. Moreover, our results demonstrate how HTT can lead to coexistence of related retrotransposon subfamilies in the same genome that can fuel evolution of new retrotransposon clades via recombination.\nRetention of an endosymbiont for the production of a single molecule\nAuthors: Garber, A.; Garcia de la Filia Molina, A.; Vea, I. M.; Mongue, A. J.; Ross, L.; McCutcheon, J. P.\nScore: 9.3, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574232\nSap-feeding insects often maintain multiple nutritional endosymbionts, which act in concert to produce compounds essential for insect survival. Many mealybugs have endosymbionts in a nested configuration: one or two bacterial species reside within the cytoplasm of another bacterium, and to-gether these bacteria have genomes which encode interdependent but complete sets of genes needed to produce key nutritional molecules. Here we show that the mealybug Pseudococcus viburni has three endosymbionts, one of which contributes only two genes that produce a single host nutrition-related molecule. All three bacterial endosymbionts have tiny genomes, suggesting that they have been co-evolving inside their insect host for millions of years. SignificanceNutritional endosymbionts synthesize (or contribute to the synthesis of) key metabolites such as essential amino acids and vitamins for their host organism. These nutrients are required by hosts because of their restricted diets, which in the case of mealybugs consists solely of plant phloem sap. Genome sequencing of insect endosymbionts has shown that their genomes can be very small, encoding few genes outside of core bacterial processes and nutrient provisioning. Here we highlight an example that has taken this reductive process to the extreme: a mealybug endosymbiont contributes only a single essential compound, chorismate, to the symbiosis.\nGut microbiota influences foraging onset without affecting division of labor and associated physiological hallmarks in honeybees\nAuthors: Liberti, J.; Frank, E. T.; Kay, T.; Kesner, L.; Monie-Ibanes, M.; Quinn, A.; Schmitt, T.; Keller, L.; Engel, P.\nScore: 10.9, Published: 2023-12-24 DOI: 10.1101/2023.12.20.570781\nGut microbes can impact cognition and behavior, but whether they regulate division of labor in animal societies is unknown. We addressed this question using honeybees since they exhibit division of labor between nurses and foragers and because their gut microbiota can be manipulated. Using automated behavioral tracking and controlling for co-housing effects, we show that gut microbes influence the age at which bees start foraging but have no effects on the time spent in a foraging area and number of foraging trips. Moreover, the gut microbiota did not influence hallmarks of behavioral maturation such as body weight, cuticular hydrocarbon (CHC) profile, hypopharyngeal gland size, and the proportion of bees maturing into foragers. Overall, this study shows that the honeybee gut microbiota does not affect division of labor but rather plays an important function in controlling the onset of bee foraging.\nThe role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication\nAuthors: Monsen, O.; Gronvold, L.; Datsomor, A.; Harvey, T. N.; Kijas, J.; Suh, A. S.-J.; Hvidsten, T. R.; Sandve, S. R.\nScore: 10.3, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573861\nBackgroundTwo of the most potent drivers of genome evolution in eukaryotes are whole genome duplications (WGD) and transposable element (TE) activity. These two mutational forces can also play synergistic roles; WGDs result in both cellular stress and functional redundancy, which would allow TEs to escape host-silencing mechanisms and effectively spread with reduced impact on fitness. As TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE-activity following WGD are likely to impact evolution of gene regulation. However, the role of TEs in genome regulatory remodelling after WGDs is unclear. Here we used the genome of Atlantic salmon, which is known to have experienced massive expansion of TEs after a WGD [~]100 Mya, as a model system to explore the synergistic roles of TEs and WGDs on genome regulatory evolution. ResultsWe identified 61,309 putative TE-CREs in Atlantic salmon using chromatin accessibility data from brain and liver. Of these, 82% were tissue specific to liver (43%) or brain (39%) and TE-CREs originating from retroelements were twice as common as those originating from DNA elements. Signatures of selection shaping TE-CRE evolution were evident from depletion of TEs in open chromatin, a bias in tissue-shared TE-CREs towards older TE-insertions, as well as tissue-specific processes shaping the TE-CRE repertoire. The DTT elements (Tc1-Mariners), which exploded in numbers at the time of the WGD, were significantly less prone to evolve into TE-CREs and significantly less potent in driving or repressing transcription compared to other TE-derived sequences. A minority of TEs (16% of consensus TEs) accounted for the origin of 46% of all TE-CREs, but these CRE-superspreaders were not temporally associated with the WGD. Rather, the majority of TE-CREs, including those found to be significantly associated with gene regulatory evolution and thus found to drive or repress transcription, evolved from TE activity occurring across tens of millions of years following the WGD event. ConclusionOur results do not support a WGD-associated TE-CRE rewiring of gene regulation. Instead we find that TEs from diverse superfamilies have been particularly effective in spreading TE-CREs and shaping gene regulatory networks under tissue-specific selection pressures, across millions of years following the salmonid WGD.\n",
  "wordCount" : "2866",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574619" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574619">
        <p class="paperTitle">DNA methylation enables recurrent endogenization of giant viruses in an animal relative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.</p>
        <p class="info">Score: 38.7, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574619' target='https://doi.org/10.1101/2024.01.08.574619'> 10.1101/2024.01.08.574619</a></p>
        <p class="abstract">5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification. Here we show that the protist closely related to animals Amoebidium appalachense features both transposon and gene body methylation, a pattern reminiscent of invertebrates and plants. Unexpectedly, large hypermethylated regions of the Amoebidium genome derive from viral insertions, including hundreds of endogenized giant viruses contributing 14% of the encoded genes, to an extent never reported before in any eukaryotic genome. Using a combination of inhibitors and functional genomic assays, we demonstrate that 5mC silences these giant virus insertions. Moreover, alternative Amoebidium isolates show polymorphic giant virus insertions, highlighting a dynamic process of infection, endogenization and purging. Our results indicate that 5mC is critical for the controlled co-existence of newly acquired viral DNA into eukaryotic genomes, making Amoebidium a unique model to understand the hybrid origins of eukaryotic genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574764" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574764">
        <p class="paperTitle">A 39.8kb flavi-like virus uses a novel strategy for overcoming the RNA virus error threshold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petrone, M. E.; Grove, J.; Mifsud, J. C. O.; Parry, R. H.; Marzinelli, E. M.; Holmes, E. C.</p>
        <p class="info">Score: 25.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574764' target='https://doi.org/10.1101/2024.01.08.574764'> 10.1101/2024.01.08.574764</a></p>
        <p class="abstract">It is commonly held that there is a fundamental relationship between genome size and error rate, manifest as a notional &#34;error threshold&#34; that sets an upper limit on genome sizes. The genome sizes of RNA viruses, which have intrinsically high mutation rates due to a lack of mechanisms for error correction, must therefore be small to avoid accumulating an excessive number of deleterious mutations that will ultimately lead to population extinction. The proposed exceptions to this evolutionary rule are RNA viruses from the order Nidovirales (such as coronaviruses) that encode an error correcting exonuclease, enabling them to reach genome lengths greater than 40kb. The recent discovery of large genome flavi-like viruses (Flaviviridae), which comprise genomes up to 27kb in length yet seemingly do not encode exonuclease domains, has led to the proposal that a proofreading mechanism is required to facilitate the expansion of RNA virus genomes above 30kb. Herein, we describe a 39.8kb flavi-like virus identified in a Haliclona sponge metatranscriptome that does not encode an exonuclease. Structural analysis revealed that this virus may have instead captured bacterial domains associated with nucleic acid metabolism that have not been previously found in RNA viruses. Phylogenetic analysis placed this virus as a divergent pesti-like lineage, such that we have provisionally termed it Maximus pesti-like virus. This virus represents the first instance of a flavi-like virus achieving a genome size comparable to that of the Nidovirales and demonstrates that RNA viruses have evolved multiple solutions to overcome the error threshold.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573862" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573862">
        <p class="paperTitle">Ancestral Sequence Reconstruction as a tool to detect and study de novo gene emergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.</p>
        <p class="info">Score: 13.9, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573862' target='https://doi.org/10.1101/2024.01.02.573862'> 10.1101/2024.01.02.573862</a></p>
        <p class="abstract">New protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (&lt;1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute &#34;easy&#34; cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574286" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574286">
        <p class="paperTitle">A complex mechanism translating variation of a simple genetic architecture into alternative life-histories</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verta, J.-P.; Moustakas-Verho, J.; Donner, I.; Frapin, M.; Ruokolainen, A.; Debes, P. V.; Erkinaro, J.; Primmer, C. R.</p>
        <p class="info">Score: 11.1, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574286' target='https://doi.org/10.1101/2024.01.05.574286'> 10.1101/2024.01.05.574286</a></p>
        <p class="abstract">Linking genes to traits is a central goal in biology. Despite progress in discovering genes associated with trait differences, a poor understanding of the functional mechanisms underlying genetic associations leaves us critically far from connecting genetic and phenotypic variation. This knowledge-gap is particularly large in multifaceted phenotypes of ecological relevance such as life-history traits. Using a multiomic dissection of the genotype-phenotype association in a large-effect maturation age gene - the transcription cofactor vestigial-like 3 (vgll3) - in Atlantic salmon (Salmo salar), we show that vgll3 mediates concerted changes of distinct molecular phenotypes associated with puberty in male gonads. Vgll3 genotype conferring early maturity upregulates key genes controlling androgen production, cellular energy and adiposity, and TGF-{beta} signaling, among others, thereby increasing the likelihood of earlier pubertal initiation. Genotype-dependent developmental trajectories are produced through VGLL3 interaction with distinct transcription factors, thus coordinating differential activation of regulatory pathways. These results reveal a genetically simple, yet functionally complex, architecture underlying alternative life-histories where variation in a single major effect gene produces pleiotropic variation in a spectrum of cellular traits. Our results further suggest that evolution in correlated phenotypes such as exemplified by alternative life history strategies may be mediated by a surprisingly simple genetic architecture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574353" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574353">
        <p class="paperTitle">Maternal Care Leads to the Evolution of Long, Slow Lives</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zipple, M. N.; Reeve, H. K.; Peniston, J. H.</p>
        <p class="info">Score: 10.9, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574353' target='https://doi.org/10.1101/2024.01.05.574353'> 10.1101/2024.01.05.574353</a></p>
        <p class="abstract">Animals, and mammals in particular, vary widely in their &#34;pace of life,&#34; with some species living long lives and reproducing infrequently (slow life histories) and others living short lives and reproducing often (fast life histories). These species also vary in the importance of maternal care in offspring fitness: in some species, offspring are fully independent of their mothers following a brief period of nutritional input, while others display a long period of continued dependence on mothers well after nutritional dependence. Here we hypothesize that these two axes of variation are causally related to each other, such that extended dependence of offspring on maternal presence leads to the evolution of longer lives at the expense of reproduction. We use a combination of deterministic modeling and stochastic agent-based modeling to explore how empirically-observed links between maternal survival and offspring fitness are likely to shape the evolution of mortality and fertility. Each of our modelling approaches leads to the same conclusion: when maternal survival has strong impacts on the survival of offspring and grandoffspring, populations evolve longer lives with less frequent reproduction. Our results suggest the slow life histories of humans and other primates as well as other long-lived, highly social animals such as hyenas, whales, and elephants, are partially the result of the strong maternal care that these animals display. We have designed our models to be readily parameterized with demographic data that is routinely collected by long-term researchers, which will facilitate more thorough testing of our hypothesis.

Significance StatementHumans and other primates live longer lives and reproduce less often than other mammals of similar body mass. What is the cause of these long lives? Here we add to existing hypotheses, including the Mother and Grandmother hypotheses, by arguing that these increased lifespans are partially explained by the intense maternal care that many primates express. Using a combination of deterministic and stochastic modeling approaches, informed by empirical data, we show that stronger connections between maternal survival and offspring fitness leads to selection for longer lives and slower reproduction. Our models suggest that the importance of the mother-offspring relationship, which defines much of human and non-human primate lives, lies at the core of the evolution of our long lives.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574070" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574070">
        <p class="paperTitle">Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.</p>
        <p class="info">Score: 10.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574070' target='https://doi.org/10.1101/2024.01.03.574070'> 10.1101/2024.01.03.574070</a></p>
        <p class="abstract">The ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects. This reduction in population size leads to a reduction in genetic diversity, which ultimately results in a reduction in the efficiency of natural selection. Then, what is the source of genetic variation which acts as the raw material for ecological and phenotypic diversification in oceanic archipelagos? Transposable elements (TEs) are mobile genetic elements that have been linked to the generation of genetic diversity, and evidence suggests that TE activity and accumulation along the genome can result from reductions in population size. Here, we use the Hawaiian spiny-leg spider radiation (Tetragnatha) to test whether TE accumulation increases due to demographic processes associated with island colonization. We sequenced and quantified TEs in 23 individuals from the spiny-leg radiation and 4 individuals from its sister radiation, the Hawaiian web-building Tetragnatha. Our results show that founder effects resulting from colonization of new islands have not resulted in TE accumulation over evolutionary time. Specifically, we found no evidence for increase in abundance of specific TE superfamilies, nor an accumulation of  young TEs in lineages which have recently colonized a new island or are present in islands with active volcanoes. We also found that the DNA/hAT transposon superfamily is by far the most abundant TE superfamily in the Tetragnatha radiation. This work shows that TE abundance has remained constant for the spiny-leg radiation across the archipelago chronosequence, and TE accumulation is not affected by population oscillations associated with island colonization events. Therefore, despite their known role in the generation of genetic diversity, TE activity does not appear to be the mechanism to explain the evolutionary paradox of the insular Tetragnatha spiny-leg radiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572574">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572574" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572574">
        <p class="paperTitle">Horizontal transfer and recombination fuel Ty4 retrotransposon evolution in Saccharomyces.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572574" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572574" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Garfinkel, D. J.; Bergman, C. M.</p>
        <p class="info">Score: 9.8, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572574' target='https://doi.org/10.1101/2023.12.20.572574'> 10.1101/2023.12.20.572574</a></p>
        <p class="abstract">Horizontal transposon transfer (HTT) plays an important role in the evolution of eukaryotic genomes, however the detailed evolutionary history and impact of most HTT events remain to be elucidated. To better understand the process of HTT in closely-related microbial eukaryotes, we studied Ty4 retrotransposon subfamily content and sequence evolution across the genus Saccharomyces using short- and long-read whole genome sequence data, including new PacBio genome assemblies for two S. mikatae strains. We find evidence for multiple independent HTT events introducing the Tsu4 subfamily into specific lineages of S. paradoxus, S. cerevisiae, S. eubayanus, S. kudriavzevii and the ancestor of the S. mikatae/S. jurei species pair. In both S. mikatae and S. kudriavzevii , we identified novel Ty4 clades that were independently generated through recombination between resident and horizontally-transferred subfamilies. Our results reveal that recurrent HTT and lineage-specific extinction events lead to a complex pattern of Ty4 subfamily content across the genus Saccharomyces. Moreover, our results demonstrate how HTT can lead to coexistence of related retrotransposon subfamilies in the same genome that can fuel evolution of new retrotransposon clades via recombination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574232" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574232">
        <p class="paperTitle">Retention of an endosymbiont for the production of a single molecule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garber, A.; Garcia de la Filia Molina, A.; Vea, I. M.; Mongue, A. J.; Ross, L.; McCutcheon, J. P.</p>
        <p class="info">Score: 9.3, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574232' target='https://doi.org/10.1101/2024.01.04.574232'> 10.1101/2024.01.04.574232</a></p>
        <p class="abstract">Sap-feeding insects often maintain multiple nutritional endosymbionts, which act in concert to produce compounds essential for insect survival. Many mealybugs have endosymbionts in a nested configuration: one or two bacterial species reside within the cytoplasm of another bacterium, and to-gether these bacteria have genomes which encode interdependent but complete sets of genes needed to produce key nutritional molecules. Here we show that the mealybug Pseudococcus viburni has three endosymbionts, one of which contributes only two genes that produce a single host nutrition-related molecule. All three bacterial endosymbionts have tiny genomes, suggesting that they have been co-evolving inside their insect host for millions of years.

SignificanceNutritional endosymbionts synthesize (or contribute to the synthesis of) key metabolites such as essential amino acids and vitamins for their host organism. These nutrients are required by hosts because of their restricted diets, which in the case of mealybugs consists solely of plant phloem sap. Genome sequencing of insect endosymbionts has shown that their genomes can be very small, encoding few genes outside of core bacterial processes and nutrient provisioning. Here we highlight an example that has taken this reductive process to the extreme: a mealybug endosymbiont contributes only a single essential compound, chorismate, to the symbiosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.570781">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.570781" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.570781">
        <p class="paperTitle">Gut microbiota influences foraging onset without affecting division of labor and associated physiological hallmarks in honeybees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.570781" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.570781" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liberti, J.; Frank, E. T.; Kay, T.; Kesner, L.; Monie-Ibanes, M.; Quinn, A.; Schmitt, T.; Keller, L.; Engel, P.</p>
        <p class="info">Score: 10.9, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.570781' target='https://doi.org/10.1101/2023.12.20.570781'> 10.1101/2023.12.20.570781</a></p>
        <p class="abstract">Gut microbes can impact cognition and behavior, but whether they regulate division of labor in animal societies is unknown. We addressed this question using honeybees since they exhibit division of labor between nurses and foragers and because their gut microbiota can be manipulated. Using automated behavioral tracking and controlling for co-housing effects, we show that gut microbes influence the age at which bees start foraging but have no effects on the time spent in a foraging area and number of foraging trips. Moreover, the gut microbiota did not influence hallmarks of behavioral maturation such as body weight, cuticular hydrocarbon (CHC) profile, hypopharyngeal gland size, and the proportion of bees maturing into foragers. Overall, this study shows that the honeybee gut microbiota does not affect division of labor but rather plays an important function in controlling the onset of bee foraging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573861" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573861">
        <p class="paperTitle">The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monsen, O.; Gronvold, L.; Datsomor, A.; Harvey, T. N.; Kijas, J.; Suh, A. S.-J.; Hvidsten, T. R.; Sandve, S. R.</p>
        <p class="info">Score: 10.3, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573861' target='https://doi.org/10.1101/2024.01.02.573861'> 10.1101/2024.01.02.573861</a></p>
        <p class="abstract">BackgroundTwo of the most potent drivers of genome evolution in eukaryotes are whole genome duplications (WGD) and transposable element (TE) activity. These two mutational forces can also play synergistic roles; WGDs result in both cellular stress and functional redundancy, which would allow TEs to escape host-silencing mechanisms and effectively spread with reduced impact on fitness. As TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE-activity following WGD are likely to impact evolution of gene regulation. However, the role of TEs in genome regulatory remodelling after WGDs is unclear. Here we used the genome of Atlantic salmon, which is known to have experienced massive expansion of TEs after a WGD [~]100 Mya, as a model system to explore the synergistic roles of TEs and WGDs on genome regulatory evolution.

ResultsWe identified 61,309 putative TE-CREs in Atlantic salmon using chromatin accessibility data from brain and liver. Of these, 82% were tissue specific to liver (43%) or brain (39%) and TE-CREs originating from retroelements were twice as common as those originating from DNA elements. Signatures of selection shaping TE-CRE evolution were evident from depletion of TEs in open chromatin, a bias in tissue-shared TE-CREs towards older TE-insertions, as well as tissue-specific processes shaping the TE-CRE repertoire. The DTT elements (Tc1-Mariners), which exploded in numbers at the time of the WGD, were significantly less prone to evolve into TE-CREs and significantly less potent in driving or repressing transcription compared to other TE-derived sequences. A minority of TEs (16% of consensus TEs) accounted for the origin of 46% of all TE-CREs, but these  CRE-superspreaders were not temporally associated with the WGD. Rather, the majority of TE-CREs, including those found to be significantly associated with gene regulatory evolution and thus found to drive or repress transcription, evolved from TE activity occurring across tens of millions of years following the WGD event.

ConclusionOur results do not support a WGD-associated TE-CRE rewiring of gene regulation. Instead we find that TEs from diverse superfamilies have been particularly effective in spreading TE-CREs and shaping gene regulatory networks under tissue-specific selection pressures, across millions of years following the salmonid WGD.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
