<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A systematic screen for co-option of transposable elements across the fungal kingdom
Authors: Oggenfuss, U.; Badet, T.; Croll, D.
Score: 39.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563377
How novel protein functions are acquired is a central question in molecular biology. Key paths to novelty include gene duplications, recombination or horizontal acquisition. Transposable elements (TEs) are increasingly recognized as a major source of novel domain-encoding sequences. However, the impact of TE coding sequences on the evolution of the proteome remains understudied.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="A systematic screen for co-option of transposable elements across the fungal kingdom
Authors: Oggenfuss, U.; Badet, T.; Croll, D.
Score: 39.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563377
How novel protein functions are acquired is a central question in molecular biology. Key paths to novelty include gene duplications, recombination or horizontal acquisition. Transposable elements (TEs) are increasingly recognized as a major source of novel domain-encoding sequences. However, the impact of TE coding sequences on the evolution of the proteome remains understudied." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="A systematic screen for co-option of transposable elements across the fungal kingdom
Authors: Oggenfuss, U.; Badet, T.; Croll, D.
Score: 39.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563377
How novel protein functions are acquired is a central question in molecular biology. Key paths to novelty include gene duplications, recombination or horizontal acquisition. Transposable elements (TEs) are increasingly recognized as a major source of novel domain-encoding sequences. However, the impact of TE coding sequences on the evolution of the proteome remains understudied."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "A systematic screen for co-option of transposable elements across the fungal kingdom\nAuthors: Oggenfuss, U.; Badet, T.; Croll, D.\nScore: 39.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563377\nHow novel protein functions are acquired is a central question in molecular biology. Key paths to novelty include gene duplications, recombination or horizontal acquisition. Transposable elements (TEs) are increasingly recognized as a major source of novel domain-encoding sequences. However, the impact of TE coding sequences on the evolution of the proteome remains understudied.",
  "keywords": [
    
  ],
  "articleBody": " A systematic screen for co-option of transposable elements across the fungal kingdom\nAuthors: Oggenfuss, U.; Badet, T.; Croll, D.\nScore: 39.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563377\nHow novel protein functions are acquired is a central question in molecular biology. Key paths to novelty include gene duplications, recombination or horizontal acquisition. Transposable elements (TEs) are increasingly recognized as a major source of novel domain-encoding sequences. However, the impact of TE coding sequences on the evolution of the proteome remains understudied. Here, we analyzed 1,237 genomes spanning the phylogenetic breadth of the fungal kingdom. We scanned proteomes for evidence of co-occurrence of TE-derived domains along with other conventional protein functional domains. We detected more than 13,000 predicted proteins containing potentially TE-derived domain, of which 825 were identified in more than five genomes, indicating that many host-TE fusions may have persisted over long evolutionary time scales. We used the phylogenetic context to identify the origin and retention of individual TE-derived domains. The most common TE-derived domains are helicases derived from Academ, Kolobok or Helitron. We found putative TE co-options at a higher rate in genomes of the Saccharomycotina, providing an unexpected source of protein novelty in these generally TE depleted genomes. We investigated in detail a candidate host-TE fusion with a heterochromatic transcriptional silencing function that may play a role in TE and gene regulation in ascomycetes. The affected gene underwent multiple full or partial losses within the phylum. Overall, our work establishes a kingdom-wide view of how domains likely derived from TEs contribute to the evolution of protein functions.\nSelection leads to false inferences of introgression using popular methods\nAuthors: Smith, M. L.; Hahn, M. W.\nScore: 24.6, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564394\nDetecting introgression between closely related populations or species is a fundamental objective in evolutionary biology. Existing methods for detecting migration and inferring migration rates from population genetic data often assume a neutral model of evolution. Growing evidence of the pervasive impact of selection on large portions of the genome across diverse taxa suggests that this assumption is unrealistic in most empirical systems. Further, ignoring selection has previously been shown to negatively impact demographic inferences (e.g., of population size histories). However, the impacts of biologically realistic selection on inferences of migration remain poorly explored. Here, we simulate data under models of background selection, selective sweeps, and adaptive introgression. We show that ignoring selection leads to false inferences of migration in three popularly used methods (fastsimcoal, {partial}a{partial}i, and BPP). Selection results in the rejection of isolation-only models in favor of isolation-with-migration models and leads to elevated estimates of migration rates across methods. Our results suggest that such methods may be unreliable in many empirical systems, such that new methods that are robust to selection will need to be developed. Article SummaryDetecting migration between closely related populations is a central objective in many evolutionary biology studies. However, popular methods for detecting migration assume a simplified model of evolution. Here, we evaluate the impacts of biologically realistic natural selection, recombination, and mutation on three methods for detecting migration. We find that biological complexity leads to false inferences of migration, suggesting that results should be interpreted with caution and that new methods are needed to make robust inferences of migration across empirical systems.\nPhylogenomics reveals the deep ocean as an accelerator for evolutionary diversification in anglerfishes\nAuthors: Miller, E. C.; Faucher, R.; Hart, P. B.; Rincon-Sandoval, M.; Santaquiteria, A.; White, W. T.; Baldwin, C. C.; Miya, M.; Betancur-R, R.; Tornabene, L.; Evans, K.; Arcila, D.\nScore: 18.9, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564281\nColonization of a novel habitat is often followed by radiation in the wake of ecological opportunity. Alternatively, some habitats should be inherently more constraining than others if the challenges of that environment have few evolutionary solutions. We examined the push-and-pull of these factors on evolution following habitat transitions, using anglerfishes (Lophiiformes) as a model. Deep-sea fishes are notoriously difficult to study, and poor sampling has limited progress thus far. Here we present a new phylogeny of anglerfishes with unprecedented taxonomic sampling (1,092 loci and 40% of species), combined with three-dimensional phenotypic data from museum specimens obtained with micro-CT scanning. We use these datasets to examine the tempo and mode of phenotypic and lineage diversification using phylogenetic comparative methods, comparing lineages in shallow and deep benthic versus bathypelagic habitats. Our results show that anglerfishes represent a surprising case where the bathypelagic lineage has greater taxonomic and phenotypic diversity than coastal benthic relatives. This defies expectations based on ecological principles since the bathypelagic zone is the most homogeneous habitat on Earth. Deep-sea anglerfishes experienced rapid lineage diversification concomitant with colonization of the bathypelagic zone from a continental slope ancestor. They display the highest body, skull and jaw shape disparity across lophiiforms. In contrast, reef-associated taxa show strong constraints on shape and low evolutionary rates, contradicting patterns suggested by other shallow marine fishes. We found that Lophiiformes as a whole evolved under an early burst model with subclades occupying distinct body shapes. We further discuss to what extent the bathypelagic clade is a secondary adaptive radiation, or if its diversity can be explained by non-adaptive processes.\nThe landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.\nScore: 126.1, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561165\nInfectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.\nMobile DNA is replete with hotspots for the de novo emergence of gene regulation\nAuthors: Fuqua, T.; Wagner, A.\nScore: 13.8, Published: 2023-10-25 DOI: 10.1101/2023.10.22.563463\nDNA mutations that create new gene expression are important raw materials for Darwinian evolution. One potential source of new gene regulation is mobile DNA, which can sometimes drive the expression of genes near its insertion site in a genome through outward-directed promoters. However, we do not know how frequent this ability is, nor how frequently mobile DNA may evolve such promoters de novo. Here we address these questions for the insertion sequence family IS3, the most abundant family of a simple form of prokaryotic mobile DNA. First, we estimate that at least 30% of IS3 sequences harbor outward-directed promoters. Second, we combine high-throughput mutagenesis with a massively parallel reporter assay to show that single point mutations suffice to create outward-directed promoters in all the IS3 sequences we studied. We found that in 5.6% of 18607 mutant IS3 sequences, promoter activity emerged de novo. Promoters preferentially arise at emergence hotspots in each IS3 sequence. These hotspots overlap with promoter motifs that already exist or are newly created by mutation. One common route to promoter activity is gaining a -10 box downstream of an existing -35 box, which we call \"Shiko Emergence.\" Overall, we show that mobile DNA has a high latent potential to drive new gene expression. This makes mobile DNA ideal for domestication by its host organism. It also raises intriguing questions about how this potential has evolved.\nPervasive relaxed selection on spermatogenesis genes coincident with the evolution of polygyny in gorillas\nAuthors: Bowman, J. D.; Silva, N.; Schüftan, E.; Almeida, J. M.; Brattig-Correia, R.; Oliveira, R. A.; Tüttelmann, F.; Enard, D.; Navarro-Costa, P.; Lynch, V. J.\nScore: 13.2, Published: 2023-10-30 DOI: 10.1101/2023.10.27.564379\nGorillas have a polygynous social system in which the highest-ranking male has almost exclusive access to females and sires most of the offspring in the troop. Such behavior results in a dramatic reduction of sperm competition, which is ultimately associated with numerous traits that cause low efficacy of gorilla spermatogenesis. However, the molecular basis behind the remarkable erosion of the gorilla male reproductive system remains unknown. Here, we explored the genetic consequences of the polygynous social system in gorillas by testing for altered selection intensity across 13,310 orthologous protein-coding genes from 261 Eutherian mammals. We identified 578 genes with relaxed purifying selection in the gorilla lineage, compared with only 96 that were positively selected. Genes under relaxed purifying selection in gorillas have accumulated numerous deleterious amino acid substitutions, their expression is biased towards male germ cells, and are enriched in functions related to meiosis and sperm biology. We tested the function of gorilla relaxed genes previously not implicated in sperm biology using the Drosophila model system and identified 41 novel spermatogenesis genes required for normal fertility. Furthermore, by exploring exome/genome sequencing data of infertile men with severe spermatogenic impairment, we found that the human orthologs of the gorilla relaxed genes are enriched for loss-of-function variants in infertile men. These data provide compelling evidence that reduced sperm competition in gorillas is associated with relaxed purifying selection on genes related to male reproductive function. The accumulation of deleterious mutations in these genes likely provides the mechanistic basis behind the low efficacy of gorilla spermatogenesis and uncovers new candidate genes for human male infertility.\nConvergent evolution of complex adaptive traits enabled human life at high altitudes\nAuthors: Ferraretti, G.; Rill, A.; Abondio, P.; Smith, K.; Ojeda-Granados, C.; De Fanti, S.; Izzi, M.; Sherpa, P. T.; Cocco, P.; Tiriticco, M.; di Marcello, M.; Dezi, A.; Gnecchi-Ruscone, G. A.; Natali, L.; Corcelli, A.; Marinelli, G.; Garagnani, P.; Peluzzi, D.; Luiselli, D.; Pettener, D.; Sarno, S.; Sazzini, M.\nScore: 13.0, Published: 2023-10-25 DOI: 10.1101/2023.10.24.563738\nConvergent adaptations represent paradigmatic examples of the capacity of natural selection to influence organisms biology. However, the possibility to investigate genetic determinants underpinning convergent complex adaptive traits has been offered only recently by methods for inferring polygenic adaptations from genomic data. Relying on this approach, we demonstrate how high-altitude Andean human groups experienced pervasive selective events at angiogenic pathways, which resemble those previously attested for Himalayan populations despite partial convergence at the single-gene level was observed. This provides unprecedented evidence for the drivers of convergent evolution of enhanced blood perfusion in populations exposed to hypobaric hypoxia for thousands of years.\nSOCfinder: a genomic tool for identifying cooperative genes in bacteria\nAuthors: Belcher, L.; Dewar, A.; Hao, C.; Katz, Z.; Ghoul, M.; West, S.\nScore: 13.9, Published: 2023-10-18 DOI: 10.1101/2023.10.16.562460\nBacteria cooperate by working collaboratively to defend their colonies, share nutrients, and resist antibiotics. Nevertheless, our understanding of these remarkable behaviours primarily comes from studying a few well-characterized species. Consequently, there is a significant gap in our understanding of microbial cooperation, particularly in natural environments. To address this gap, we can use bioinformatic tools to identify cooperative traits and their underlying genes across diverse species. Existing tools address this challenge through two approaches. One approach is to identify genes that encode extracellular proteins, which can provide benefits to neighbouring cells. An alternative approach is to predict gene function using annotation tools. However, these tools have several limitations. Not all extracellular proteins are cooperative, and not all cooperative behaviours are controlled by extracellular proteins. Furthermore, existing functional annotation methods frequently miss known cooperative genes. Here, we introduce SOCfinder as a new tool to find cooperative genes in bacterial genomes. SOCfinder combines information from several methods, considering if a gene is likely to (1) code for an extracellular protein, (2) have a cooperative functional annotation, or (3) be part of the biosynthesis of a cooperative secondary metabolite. We use data on two extensively-studied species (P. aeruginosa \u0026 B. subtilis) to show that SOCfinder is better at finding known cooperative genes than existing tools. We also use theory from population genetics to identify a signature of kin selection in SOCfinder cooperative genes, which is lacking in genes identified by existing tools. SOCfinder opens up a number of exciting directions for future research, and is available to download from https://github.com/lauriebelch/SOCfinder. Data SummaryAll code and associated files are available at https://github.com/lauriebelch/SOCfinder. Impact StatementBacteria cooperate by secreting many molecules outside the cell, where they can provide benefits to other cells. While we know much about how bacteria cooperate in the lab, we know much less about bacterial cooperation in nature. Is cooperation equally important in all species? Are all cooperations equally vulnerable to cheating? To answer these questions, we need a way of identifying cooperative genes across a wide range of genomes. Here, we provide such a method - which we name SOCfinder. SOCfinder allows users to find cooperative genes in any bacterial genome. SOCfinder opens up a number of exciting directions for future research. It will allow detailed studies of non-model species, as well as broad comparative studies across species. These studies will allow cooperation in the wild to be studied in new ways.\nConflicts in natural selection constrain adaptation to climate change in Arabidopsis thaliana\nAuthors: Ruffley, M. R.; Levanthal, L.; Hateley, S.; Rhee, S. Y.; Exposito-Alonso, M.\nScore: 25.7, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562583\nAdaptation to new environments can be impeded if beneficial phenotype combinations cannot coexist due to genetic constraints. To understand how such constraints may hinder plant adaptation to future climates, we compiled a comprehensive database of traits of Arabidopsis thaliana and estimated phenotypic natural selection in common gardens in its native distribution with rainfall limitation treatments. We found a natural selection conflict in drought environments as two drought-adaptive strategies, escape and avoidance, are mutually exclusive in A. thaliana. Traits underlying such strategies, such as flowering time, growth rate, and water use efficiency, are genetically correlated, and we identify novel loci involved in such correlation experiencing antagonistic natural selection. This empirical evidence shows that these adaptive strategies in natural populations are mutually exclusive due to strong genetic correlations amongst traits that limit the possible combinations of phenotypes. Given projections that future climates will become hotter and drier in many temperate regions, we expect an increasing conflict in natural selection among adaptive traits that could slow down or prevent adaptation. Our study underscores the importance of accounting for evolutionary genetic constraints when predicting how species may respond to a changing climate.\nTarget complementarity in cnidarians supports a common origin for animal and plant microRNAs\nAuthors: Admoni, Y.; Fridrich, A.; Razin, T.; Salinas-Saavedra, M.; Rabani, M.; Frank, U.; Moran, Y.\nScore: 28.3, Published: 2023-10-30 DOI: 10.1101/2023.01.08.523153\nmicroRNAs (miRNAs) are important post-transcriptional regulators that activate silencing mechanisms by annealing to mRNA transcripts. While plant miRNAs match their targets with nearly-full complementarity leading to mRNA cleavage, miRNAs in most animals require only a short sequence called seed to inhibit target translation. Recent findings showed that miRNAs in cnidarians, early-branching metazoans, act similarly to plant miRNAs, by exhibiting full complementarity and target cleavage; however, it remained unknown if seed-based regulation was possible in cnidarians. Here, we investigate the miRNA-target complementarity requirements for miRNA activity in the cnidarian Nematostella vectensis. We show that bilaterian-like complementarity of seed-only or seed and supplementary 3' matches are insufficient for miRNA-mediated knockdown. Furthermore, miRNA-target mismatches in the cleavage site decrease knockdown efficiency. Finally, miRNA silencing of a target with three seed binding sites in the 3' untranslated region that mimics typical miRNA targeting was repressed in zebrafish but not in Nematostella and Hydractinia symbiolongicarpus. Altogether, these results unravel striking similarities between plant and cnidarian miRNAs consolidating the evidence for common evolutionary origin of miRNAs in plants and animals.\n",
  "wordCount" : "2870",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563377">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563377" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563377">
        <p class="paperTitle">A systematic screen for co-option of transposable elements across the fungal kingdom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563377" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563377" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oggenfuss, U.; Badet, T.; Croll, D.</p>
        <p class="info">Score: 39.9, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563377' target='https://doi.org/10.1101/2023.10.20.563377'> 10.1101/2023.10.20.563377</a></p>
        <p class="abstract">How novel protein functions are acquired is a central question in molecular biology. Key paths to novelty include gene duplications, recombination or horizontal acquisition. Transposable elements (TEs) are increasingly recognized as a major source of novel domain-encoding sequences. However, the impact of TE coding sequences on the evolution of the proteome remains understudied. Here, we analyzed 1,237 genomes spanning the phylogenetic breadth of the fungal kingdom. We scanned proteomes for evidence of co-occurrence of TE-derived domains along with other conventional protein functional domains. We detected more than 13,000 predicted proteins containing potentially TE-derived domain, of which 825 were identified in more than five genomes, indicating that many host-TE fusions may have persisted over long evolutionary time scales. We used the phylogenetic context to identify the origin and retention of individual TE-derived domains. The most common TE-derived domains are helicases derived from Academ, Kolobok or Helitron. We found putative TE co-options at a higher rate in genomes of the Saccharomycotina, providing an unexpected source of protein novelty in these generally TE depleted genomes. We investigated in detail a candidate host-TE fusion with a heterochromatic transcriptional silencing function that may play a role in TE and gene regulation in ascomycetes. The affected gene underwent multiple full or partial losses within the phylum. Overall, our work establishes a kingdom-wide view of how domains likely derived from TEs contribute to the evolution of protein functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564394">
        <p class="paperTitle">Selection leads to false inferences of introgression using popular methods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, M. L.; Hahn, M. W.</p>
        <p class="info">Score: 24.6, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564394' target='https://doi.org/10.1101/2023.10.27.564394'> 10.1101/2023.10.27.564394</a></p>
        <p class="abstract">Detecting introgression between closely related populations or species is a fundamental objective in evolutionary biology. Existing methods for detecting migration and inferring migration rates from population genetic data often assume a neutral model of evolution. Growing evidence of the pervasive impact of selection on large portions of the genome across diverse taxa suggests that this assumption is unrealistic in most empirical systems. Further, ignoring selection has previously been shown to negatively impact demographic inferences (e.g., of population size histories). However, the impacts of biologically realistic selection on inferences of migration remain poorly explored. Here, we simulate data under models of background selection, selective sweeps, and adaptive introgression. We show that ignoring selection leads to false inferences of migration in three popularly used methods (fastsimcoal, {partial}a{partial}i, and BPP). Selection results in the rejection of isolation-only models in favor of isolation-with-migration models and leads to elevated estimates of migration rates across methods. Our results suggest that such methods may be unreliable in many empirical systems, such that new methods that are robust to selection will need to be developed.

Article SummaryDetecting migration between closely related populations is a central objective in many evolutionary biology studies. However, popular methods for detecting migration assume a simplified model of evolution. Here, we evaluate the impacts of biologically realistic natural selection, recombination, and mutation on three methods for detecting migration. We find that biological complexity leads to false inferences of migration, suggesting that results should be interpreted with caution and that new methods are needed to make robust inferences of migration across empirical systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564281">
        <p class="paperTitle">Phylogenomics reveals the deep ocean as an accelerator for evolutionary diversification in anglerfishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, E. C.; Faucher, R.; Hart, P. B.; Rincon-Sandoval, M.; Santaquiteria, A.; White, W. T.; Baldwin, C. C.; Miya, M.; Betancur-R, R.; Tornabene, L.; Evans, K.; Arcila, D.</p>
        <p class="info">Score: 18.9, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564281' target='https://doi.org/10.1101/2023.10.26.564281'> 10.1101/2023.10.26.564281</a></p>
        <p class="abstract">Colonization of a novel habitat is often followed by radiation in the wake of ecological opportunity. Alternatively, some habitats should be inherently more constraining than others if the challenges of that environment have few evolutionary solutions. We examined the push-and-pull of these factors on evolution following habitat transitions, using anglerfishes (Lophiiformes) as a model. Deep-sea fishes are notoriously difficult to study, and poor sampling has limited progress thus far. Here we present a new phylogeny of anglerfishes with unprecedented taxonomic sampling (1,092 loci and 40% of species), combined with three-dimensional phenotypic data from museum specimens obtained with micro-CT scanning. We use these datasets to examine the tempo and mode of phenotypic and lineage diversification using phylogenetic comparative methods, comparing lineages in shallow and deep benthic versus bathypelagic habitats. Our results show that anglerfishes represent a surprising case where the bathypelagic lineage has greater taxonomic and phenotypic diversity than coastal benthic relatives. This defies expectations based on ecological principles since the bathypelagic zone is the most homogeneous habitat on Earth. Deep-sea anglerfishes experienced rapid lineage diversification concomitant with colonization of the bathypelagic zone from a continental slope ancestor. They display the highest body, skull and jaw shape disparity across lophiiforms. In contrast, reef-associated taxa show strong constraints on shape and low evolutionary rates, contradicting patterns suggested by other shallow marine fishes. We found that Lophiiformes as a whole evolved under an early burst model with subclades occupying distinct body shapes. We further discuss to what extent the bathypelagic clade is a secondary adaptive radiation, or if its diversity can be explained by non-adaptive processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561165">
        <p class="paperTitle">The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.</p>
        <p class="info">Score: 126.1, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561165' target='https://doi.org/10.1101/2023.10.06.561165'> 10.1101/2023.10.06.561165</a></p>
        <p class="abstract">Infectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563463">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563463" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563463">
        <p class="paperTitle">Mobile DNA is replete with hotspots for the de novo emergence of gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563463" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563463" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuqua, T.; Wagner, A.</p>
        <p class="info">Score: 13.8, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563463' target='https://doi.org/10.1101/2023.10.22.563463'> 10.1101/2023.10.22.563463</a></p>
        <p class="abstract">DNA mutations that create new gene expression are important raw materials for Darwinian evolution. One potential source of new gene regulation is mobile DNA, which can sometimes drive the expression of genes near its insertion site in a genome through outward-directed promoters. However, we do not know how frequent this ability is, nor how frequently mobile DNA may evolve such promoters de novo. Here we address these questions for the insertion sequence family IS3, the most abundant family of a simple form of prokaryotic mobile DNA. First, we estimate that at least 30% of IS3 sequences harbor outward-directed promoters. Second, we combine high-throughput mutagenesis with a massively parallel reporter assay to show that single point mutations suffice to create outward-directed promoters in all the IS3 sequences we studied. We found that in 5.6% of 18607 mutant IS3 sequences, promoter activity emerged de novo. Promoters preferentially arise at emergence hotspots in each IS3 sequence. These hotspots overlap with promoter motifs that already exist or are newly created by mutation. One common route to promoter activity is gaining a -10 box downstream of an existing -35 box, which we call &#34;Shiko Emergence.&#34; Overall, we show that mobile DNA has a high latent potential to drive new gene expression. This makes mobile DNA ideal for domestication by its host organism. It also raises intriguing questions about how this potential has evolved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564379">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564379" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564379">
        <p class="paperTitle">Pervasive relaxed selection on spermatogenesis genes coincident with the evolution of polygyny in gorillas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564379" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564379" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bowman, J. D.; Silva, N.; Schüftan, E.; Almeida, J. M.; Brattig-Correia, R.; Oliveira, R. A.; Tüttelmann, F.; Enard, D.; Navarro-Costa, P.; Lynch, V. J.</p>
        <p class="info">Score: 13.2, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564379' target='https://doi.org/10.1101/2023.10.27.564379'> 10.1101/2023.10.27.564379</a></p>
        <p class="abstract">Gorillas have a polygynous social system in which the highest-ranking male has almost exclusive access to females and sires most of the offspring in the troop. Such behavior results in a dramatic reduction of sperm competition, which is ultimately associated with numerous traits that cause low efficacy of gorilla spermatogenesis. However, the molecular basis behind the remarkable erosion of the gorilla male reproductive system remains unknown. Here, we explored the genetic consequences of the polygynous social system in gorillas by testing for altered selection intensity across 13,310 orthologous protein-coding genes from 261 Eutherian mammals. We identified 578 genes with relaxed purifying selection in the gorilla lineage, compared with only 96 that were positively selected. Genes under relaxed purifying selection in gorillas have accumulated numerous deleterious amino acid substitutions, their expression is biased towards male germ cells, and are enriched in functions related to meiosis and sperm biology. We tested the function of gorilla relaxed genes previously not implicated in sperm biology using the Drosophila model system and identified 41 novel spermatogenesis genes required for normal fertility. Furthermore, by exploring exome/genome sequencing data of infertile men with severe spermatogenic impairment, we found that the human orthologs of the gorilla relaxed genes are enriched for loss-of-function variants in infertile men. These data provide compelling evidence that reduced sperm competition in gorillas is associated with relaxed purifying selection on genes related to male reproductive function. The accumulation of deleterious mutations in these genes likely provides the mechanistic basis behind the low efficacy of gorilla spermatogenesis and uncovers new candidate genes for human male infertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563738">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563738" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563738">
        <p class="paperTitle">Convergent evolution of complex adaptive traits enabled human life at high altitudes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563738" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563738" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferraretti, G.; Rill, A.; Abondio, P.; Smith, K.; Ojeda-Granados, C.; De Fanti, S.; Izzi, M.; Sherpa, P. T.; Cocco, P.; Tiriticco, M.; di Marcello, M.; Dezi, A.; Gnecchi-Ruscone, G. A.; Natali, L.; Corcelli, A.; Marinelli, G.; Garagnani, P.; Peluzzi, D.; Luiselli, D.; Pettener, D.; Sarno, S.; Sazzini, M.</p>
        <p class="info">Score: 13.0, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563738' target='https://doi.org/10.1101/2023.10.24.563738'> 10.1101/2023.10.24.563738</a></p>
        <p class="abstract">Convergent adaptations represent paradigmatic examples of the capacity of natural selection to influence organisms biology. However, the possibility to investigate genetic determinants underpinning convergent complex adaptive traits has been offered only recently by methods for inferring polygenic adaptations from genomic data. Relying on this approach, we demonstrate how high-altitude Andean human groups experienced pervasive selective events at angiogenic pathways, which resemble those previously attested for Himalayan populations despite partial convergence at the single-gene level was observed. This provides unprecedented evidence for the drivers of convergent evolution of enhanced blood perfusion in populations exposed to hypobaric hypoxia for thousands of years.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562460" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562460">
        <p class="paperTitle">SOCfinder: a genomic tool for identifying cooperative genes in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Belcher, L.; Dewar, A.; Hao, C.; Katz, Z.; Ghoul, M.; West, S.</p>
        <p class="info">Score: 13.9, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562460' target='https://doi.org/10.1101/2023.10.16.562460'> 10.1101/2023.10.16.562460</a></p>
        <p class="abstract">Bacteria cooperate by working collaboratively to defend their colonies, share nutrients, and resist antibiotics. Nevertheless, our understanding of these remarkable behaviours primarily comes from studying a few well-characterized species. Consequently, there is a significant gap in our understanding of microbial cooperation, particularly in natural environments. To address this gap, we can use bioinformatic tools to identify cooperative traits and their underlying genes across diverse species. Existing tools address this challenge through two approaches. One approach is to identify genes that encode extracellular proteins, which can provide benefits to neighbouring cells. An alternative approach is to predict gene function using annotation tools. However, these tools have several limitations. Not all extracellular proteins are cooperative, and not all cooperative behaviours are controlled by extracellular proteins. Furthermore, existing functional annotation methods frequently miss known cooperative genes. Here, we introduce SOCfinder as a new tool to find cooperative genes in bacterial genomes. SOCfinder combines information from several methods, considering if a gene is likely to (1) code for an extracellular protein, (2) have a cooperative functional annotation, or (3) be part of the biosynthesis of a cooperative secondary metabolite. We use data on two extensively-studied species (P. aeruginosa &amp; B. subtilis) to show that SOCfinder is better at finding known cooperative genes than existing tools. We also use theory from population genetics to identify a signature of kin selection in SOCfinder cooperative genes, which is lacking in genes identified by existing tools. SOCfinder opens up a number of exciting directions for future research, and is available to download from https://github.com/lauriebelch/SOCfinder.

Data SummaryAll code and associated files are available at https://github.com/lauriebelch/SOCfinder.

Impact StatementBacteria cooperate by secreting many molecules outside the cell, where they can provide benefits to other cells. While we know much about how bacteria cooperate in the lab, we know much less about bacterial cooperation in nature. Is cooperation equally important in all species? Are all cooperations equally vulnerable to cheating? To answer these questions, we need a way of identifying cooperative genes across a wide range of genomes. Here, we provide such a method - which we name SOCfinder. SOCfinder allows users to find cooperative genes in any bacterial genome. SOCfinder opens up a number of exciting directions for future research. It will allow detailed studies of non-model species, as well as broad comparative studies across species. These studies will allow cooperation in the wild to be studied in new ways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562583">
        <p class="paperTitle">Conflicts in natural selection constrain adaptation to climate change in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruffley, M. R.; Levanthal, L.; Hateley, S.; Rhee, S. Y.; Exposito-Alonso, M.</p>
        <p class="info">Score: 25.7, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562583' target='https://doi.org/10.1101/2023.10.16.562583'> 10.1101/2023.10.16.562583</a></p>
        <p class="abstract">Adaptation to new environments can be impeded if beneficial phenotype combinations cannot coexist due to genetic constraints. To understand how such constraints may hinder plant adaptation to future climates, we compiled a comprehensive database of traits of Arabidopsis thaliana and estimated phenotypic natural selection in common gardens in its native distribution with rainfall limitation treatments. We found a natural selection conflict in drought environments as two drought-adaptive strategies, escape and avoidance, are mutually exclusive in A. thaliana. Traits underlying such strategies, such as flowering time, growth rate, and water use efficiency, are genetically correlated, and we identify novel loci involved in such correlation experiencing antagonistic natural selection. This empirical evidence shows that these adaptive strategies in natural populations are mutually exclusive due to strong genetic correlations amongst traits that limit the possible combinations of phenotypes. Given projections that future climates will become hotter and drier in many temperate regions, we expect an increasing conflict in natural selection among adaptive traits that could slow down or prevent adaptation. Our study underscores the importance of accounting for evolutionary genetic constraints when predicting how species may respond to a changing climate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.08.523153">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.08.523153" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.08.523153">
        <p class="paperTitle">Target complementarity in cnidarians supports a common origin for animal and plant microRNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.08.523153" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.08.523153" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Admoni, Y.; Fridrich, A.; Razin, T.; Salinas-Saavedra, M.; Rabani, M.; Frank, U.; Moran, Y.</p>
        <p class="info">Score: 28.3, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.08.523153' target='https://doi.org/10.1101/2023.01.08.523153'> 10.1101/2023.01.08.523153</a></p>
        <p class="abstract">microRNAs (miRNAs) are important post-transcriptional regulators that activate silencing mechanisms by annealing to mRNA transcripts. While plant miRNAs match their targets with nearly-full complementarity leading to mRNA cleavage, miRNAs in most animals require only a short sequence called seed to inhibit target translation. Recent findings showed that miRNAs in cnidarians, early-branching metazoans, act similarly to plant miRNAs, by exhibiting full complementarity and target cleavage; however, it remained unknown if seed-based regulation was possible in cnidarians. Here, we investigate the miRNA-target complementarity requirements for miRNA activity in the cnidarian Nematostella vectensis. We show that bilaterian-like complementarity of seed-only or seed and supplementary 3&#39; matches are insufficient for miRNA-mediated knockdown. Furthermore, miRNA-target mismatches in the cleavage site decrease knockdown efficiency. Finally, miRNA silencing of a target with three seed binding sites in the 3&#39; untranslated region that mimics typical miRNA targeting was repressed in zebrafish but not in Nematostella and Hydractinia symbiolongicarpus. Altogether, these results unravel striking similarities between plant and cnidarian miRNAs consolidating the evidence for common evolutionary origin of miRNAs in plants and animals.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
