<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Bacterial ecology and evolution converge on seasonal and decadal scales
Authors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.
Score: 77.1, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579087
Ecology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Bacterial ecology and evolution converge on seasonal and decadal scales
Authors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.
Score: 77.1, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579087
Ecology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Bacterial ecology and evolution converge on seasonal and decadal scales
Authors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.
Score: 77.1, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579087
Ecology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Bacterial ecology and evolution converge on seasonal and decadal scales\nAuthors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.\nScore: 77.1, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579087\nEcology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset.",
  "keywords": [
    
  ],
  "articleBody": " Bacterial ecology and evolution converge on seasonal and decadal scales\nAuthors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.\nScore: 77.1, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579087\nEcology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset. This massive sampling and sequencing effort resulted in the reconstruction of 30,389 metagenomic-assembled genomes (MAGs) over 50% complete, which dereplicated into 2,855 distinct genomes (\u003e96% nucleotide sequence identity). We found both ecological and evolutionary processes occurred at seasonal time scales. There were recurring annual patterns at the species level in abundances, nucleotide diversities ({pi}), and single nucleotide variant (SNV) profiles for the majority of all taxa. During annual blooms, we observed both higher and lower nucleotide diversity, indicating that both ecological differentiation and competition drove evolutionary dynamics. Overlayed upon seasonal patterns, we observed long-term change in 20% of the species SNV profiles including gradual changes, step changes, and disturbances followed by resilience. Most abrupt changes occurred in a single species, suggesting evolutionary drivers are highly specific. Nevertheless, seven members of the abundant Nanopelagicaceae family experienced abrupt change in 2012, an unusually hot and dry year. This shift coincided with increased numbers of genes under selection involved in amino acid and nucleic acid metabolism, suggesting fundamental organic nitrogen compounds drive strain differentiation in the most globally abundant freshwater family. Overall, we observed seasonal and decadal trends in both interspecific ecological and intraspecific evolutionary processes. The convergence of microbial ecology and evolution on the same time scales demonstrates that understanding microbiomes requires a new unified approach that views ecology and evolution as a single continuum.\nEffect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 88.6, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.\nLineage-resolved analysis of embryonic gene expression evolution in C. elegans and C. briggsae\nAuthors: Large, C. R. L.; Khanal, R.; Hillier, L. W.; Huynh, C.; Kubo, C.; Kim, J.; Waterston, R.; Murray, J. I.\nScore: 14.1, Published: 2024-02-06 DOI: 10.1101/2024.02.03.578695\nWhat constraints govern the evolution of gene expression patterns across development remains a fundamental question of evolutionary biology. The advent of single-cell sequencing opens the possibility of learning these constraints by systematically profiling homologous cells across different organisms. The nematode C. elegans is a well-studied model for embryonic development, and its invariant lineage that is conserved with other Caenorhabditis species makes it an ideal model to directly compare gene expression between homologous progenitor and terminal cell types across evolution. We have measured the spatiotemporal divergence of gene expression across embryogenesis by collecting, annotating, and comparing the transcriptomes of homologous embryonic progenitors and terminal cell types, using a dataset comprising \u003e200,000 C. elegans cells and \u003e190,000 C. briggsae cells. We find a high level of similarity in gene expression programs between the species despite tens of millions of years of evolutionary divergence, consistent with their conserved developmental lineages. Even still, thousands of genes show divergence in their cell-type specific expression patterns, and these are enriched for categories involved in environmental response and behavior. Comparing the degree of expression conservation across cell types reveals that certain cell types such as neurons, have diverged more than others such as the intestine and body wall muscle. Taken together, this work identifies likely constraints on the evolution of developmental gene expression and provides a powerful resource for addressing diverse evolutionary questions.\nAtypical landscape of transposable elements in the large genome of Aedes aegypti\nAuthors: Daron, J.; Bergman, A.; Lopez-Maestre, H.; LAMBRECHTS, L.\nScore: 12.6, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579293\nTransposable elements (TEs) contribute significantly to variation in genome size among eukaryotic species, but the factors influencing TE accumulation and diversification are only partially understood. Most of our current knowledge about TE organization, dynamics and evolution derives from investigations in model organisms with a relatively small genome size such as Drosophila melanogaster or Arabidopsis thaliana. Whether the observed patterns hold true in larger genomes remains to be determined. The Diptera order is an ideal taxon to address this question, because it includes a forty-year model of TE biology (D. melanogaster) as well as mosquito species with significantly larger genomes. Here, we use a comparative genomics approach to characterize the genomic forces that have shaped the TE content of the Aedes aegypti genome (1.3 Gb) relative to the Anopheles coluzzii genome (300 Mb) and the D. melanogaster genome (180 Mb). Leveraging a newly developed high-quality TE library for Ae. aegypti, our results reveal a contrasted pattern of TE organization in Ae. aegypti compared to An. coluzzii and D. melanogaster. Our analyses suggest that the substantial TE fraction observed in the Ae. aegypti genome reflect both a high rate of TE transposition and a low rate of TE elimination. Together, our results indicate that TE organization and evolutionary dynamics in the large genome of Ae. aegypti are distinct from those of other dipterans with smaller genomes.\nQuantifying internal conflicts and their threats to organismal form and fitness\nAuthors: Schenkel, M. A.; Patten, M. M.; Agren, J. A.\nScore: 10.9, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578856\nEvolutionary biologists often treat organisms as both fitness-maximizing agents and as the primary level at which adaptation manifests. Yet, genes and cells may also seek to optimize their fitness by distorting the Mendelian rules of transmission or by influencing organismal traits for their own benefit. Organismal form and fitness are therefore threatened from within by selfish genes and cells. However, to what extent such internal conflicts actually harm individual organisms and threaten our concept of the organism as the sole bearer of adaptation remains unclear. We introduce a mathematical framework to capture the threat posed by internal conflicts and develop two metrics to measure their various forms of harm. We name these metrics fitness unity and trait unity, and use them to refer to the threats posed by internal conflicts to an organisms role as the optimizing agent and the strategy wielded to achieve that optimization, respectively. We apply our framework to two examples of internal conflicts, genomic imprinting and sex ratio distortion, to illustrate how such harms from internal conflict may be quantified. We conclude by discussing the conditions under which internal conflict becomes sufficiently disruptive to organisms that it no longer makes sense to think of them as unified fitness-maximizing agents, but instead as adaptive compromises of multiple competing sub-agents.\nMicrobes as manipulators of developmental life-history\nAuthors: Kustra, M. C.; Carrier, T. J.\nScore: 8.5, Published: 2024-02-06 DOI: 10.1101/2024.02.02.578589\nMarine invertebrates mainly reproduce by energy-poor eggs that develop into feeding larvae or energy-rich eggs that develop into non-feeding larvae1-4. Transitions between these reproductive strategies have been studied in detail5-7, yet the evolutionary factor(s) responsible for these switches remains elusive. Here, we use theoretical models to show that microbes with the capacity to manipulate host reproduction are one possible factor. We report that microbial manipulators create a sperm-limited environment that selects for larger eggs by shifting the hosts sex ratio towards female dominance and, as a result, serve as the evolutionary driver of transitions in the developmental life-history for marine invertebrates. Loss of a microbial manipulator can then recover the ancestral developmental life-history. We also document more than a dozen genera of marine invertebrates from throughout the worlds oceans that fit the framework of a microbe-induced switch between these predominate reproductive strategies. We anticipate that microbial manipulators have a yet-to-be appreciated influence on the life-history strategies of marine invertebrates. We find it paramount to understand if transitions in developmental life-history also occur without microbial manipulators as well as if the underlying mechanisms of these manipulations are convergent with terrestrial systems.\nA single giant cell at the origin of an evolutionary innovation\nAuthors: Badiane, A.; Viala, S.; Armisen, D.; Mignerey, I.; Forcet-Vauchel, C.; De Jesus Villanueva, C. N.; Van Belleghem, S.; Crumiere, A.; Santos, E.; Francesconi, M.; Khila, A.\nScore: 8.4, Published: 2024-02-09 DOI: 10.1101/2024.02.08.579433\nKey innovations play a central role in the diversification of lineages, yet our understanding of the mechanisms underlying their emergence remains fragmented. The propelling fan is a key innovation associated with the diversification of Rhagovelia water striders into fast flowing streams. Here, we show that a single giant cell is responsible for the development of the fan throughout embryogenesis. RNA interference against the fan-specific gene family gsha/mogsha depleted the fan in the embryo but did not alter the giant cell, indicating that these genes are not required for specifying the identity of this cell. gsha/mogsha instead seem to activate the expression of cuticular genes during early development, suggesting a role of these genes in regulating the accumulation of fan structural proteins. We also show that Hox genes act to block the giant cell in the fore- and rear-legs, thus restricting its fate to the mid-legs - the pair of legs that generates movement on water. In addition, Hox expression is specifically excluded from the giant cell, thus allowing the expression of the fan's developmental genetic program. This study reveals that a single cell can be sufficient to generate innovative traits that have been central for a lineage to acquire new ecological opportunities and burst into diversification.\nA comprehensive examination of Chelicerate genomes reveals no evidence for a whole genome duplication among spiders and scorpions\nAuthors: Thomas, G. W. C.; McKibben, M. T. W.; Hahn, M. W.; Barker, M. S.\nScore: 8.2, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578966\nWhole genome duplications (WGDs) can be a key event in evolution, playing a role in both adaptation and speciation. While WGDs are common throughout the history of plants, only a few examples have been proposed in metazoans. Among these, recent proposals of WGD events in Chelicerates, the group of Arthropods that includes horseshoe crabs, ticks, scorpions, and spiders, include several rounds in the history of horseshoe crabs, with an additional WGD proposed in the ancestor of spiders and scorpions. However, many of these inferences are based on evidence from only a small portion of the genome (in particular, the Hox gene cluster); therefore, genome-wide inferences with broader species sampling may give a clearer picture of WGDs in this clade. Here, we investigate signals of WGD in Chelicerates using whole genomes from 17 species. We employ multiple methods to look for these signals, including gene tree analysis of thousands of gene families, comparisons of synteny, and signals of divergence among within-species paralogs. We test several scenarios of WGD in Chelicerates using multiple species trees as a backbone for all hypotheses. While we do find support for at least one WGD in the ancestral horseshoe crab lineage, we find no evidence for a WGD in the history of spiders and scorpions using any genome-scale method. This study not only sheds light on genome evolution and phylogenetics within Chelicerates, but also demonstrates how a combination of comparative methods can be used to investigate signals of ancient WGDs.\nVariation in Neural Crest Development and Functional Divergence of sox10 Paralogs Contribute to Morphological Diversification in Cichlid Fishes\nAuthors: Marconi, A.; Vernaz, G.; Karunaratna, A.; Ngochera, M. J.; Durbin, R.; Santos, M. E.\nScore: 36.5, Published: 2024-01-31 DOI: 10.1101/2024.01.30.578004\nNeural crest (NC) is a vertebrate-specific embryonic progenitor cell population at the basis of important vertebrate features such as the craniofacial skeleton and pigmentation patterns. Despite the wide-ranging variation of NC-derived traits across vertebrates, the contribution of NC to species diversification remains largely unexplored. Here, by leveraging the adaptive diversity of African Great Lakes cichlid species, we combined comparative transcriptomics and population genomics to investigate the role of NC development in morphological diversification. Our analysis revealed substantial differences in transcriptional landscapes across somitogenesis, an embryonic period coinciding with NC development and migration. Notably, several NC-related gene expression clusters showed both species-specific divergence in transcriptional landscapes and signatures of positive selection. Specifically, we identified two paralogs of the sox10 gene as prime NC-related candidates contributing to interspecific morphological variation, which displayed remarkable spatio-temporal expression variation in cichlids. Finally, through CRISPR-KO mutants, we experimentally validated the functional divergence between sox10 paralogs, with the acquisition of a novel role in cichlid skeletogenesis by sox10-like. Our study demonstrates the central role of NC-related processes - in particular those controlled by sox10s - in generating morphological diversification among closely-related species and lays the groundwork for further investigations into the mechanisms underpinning vertebrate NC diversification.\nA covariant model of molecular and diversification patterns through time and the history of large clades\nAuthors: Budd, G. E.; Mann, R. P.\nScore: 6.8, Published: 2024-02-06 DOI: 10.1101/2024.02.01.578373\nAO_SCPLOWBSTRACTC_SCPLOWRate shifts in speciation and extinction have been recognised as important contributors to the creation of evolutionary patterns. In particular, the distribution of modern clade sizes is difficult to reconcile with models that do not include them. Although recent advances have allowed rate shifts to be integrated into evolutionary models, these have largely been for the purpose of inferring historical rate shifts across phylogenetic trees. In addition, these models have typically assumed an independence between patterns of diversification and rates of molecular and morphological evolution, despite there being mounting evidence of a connection between them. Here, we develop a new model with two principal goals: first, to explore the general patterns of diversification implied by constantly changing rates, and secondly to integrate diversification, molecular and morphological evolution into a single coherent framework. We thus develop and analyse a covariant birth-death process in which rates of all evolutionary processes (i.e. speciation, extinction and molecular and morphological change) covary continuously, both for each species and through time. We use this model to show that modern diversity is likely to be dominated by a small number of extremely large clades at any historical epoch; that these large clades are expected to be characterised by explosive early radiations accompanied by elevated rates of molecular evolution; and that extant organisms are likely to have evolved from species with unusually fast evolutionary rates. In addition, we show that under such a model, the amount of molecular change along a particular lineage is essentially independent of its height, which further weakens the molecular clock hypothesis. Finally, our model predicts the existence of \"living fossil\" sister groups to large clades that are both species poor and have exhibited slow rates of morphological and molecular change. Although our model is highly stochastic, it includes no special evolutionary moments or epochs. Our results thus demonstrate that the observed historical patterns of evolution can be modelled without invoking special evolutionary mechanisms or innovations that are unique to specific times or taxa, even when they are highly non-uniform: instead they could emerge from a process that is fundamentally homogeneous throughout time.\n",
  "wordCount" : "2657",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579087" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579087">
        <p class="paperTitle">Bacterial ecology and evolution converge on seasonal and decadal scales</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.</p>
        <p class="info">Score: 77.1, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579087' target='https://doi.org/10.1101/2024.02.06.579087'> 10.1101/2024.02.06.579087</a></p>
        <p class="abstract">Ecology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset. This massive sampling and sequencing effort resulted in the reconstruction of 30,389 metagenomic-assembled genomes (MAGs) over 50% complete, which dereplicated into 2,855 distinct genomes (&gt;96% nucleotide sequence identity). We found both ecological and evolutionary processes occurred at seasonal time scales. There were recurring annual patterns at the species level in abundances, nucleotide diversities ({pi}), and single nucleotide variant (SNV) profiles for the majority of all taxa. During annual blooms, we observed both higher and lower nucleotide diversity, indicating that both ecological differentiation and competition drove evolutionary dynamics. Overlayed upon seasonal patterns, we observed long-term change in 20% of the species SNV profiles including gradual changes, step changes, and disturbances followed by resilience. Most abrupt changes occurred in a single species, suggesting evolutionary drivers are highly specific. Nevertheless, seven members of the abundant Nanopelagicaceae family experienced abrupt change in 2012, an unusually hot and dry year. This shift coincided with increased numbers of genes under selection involved in amino acid and nucleic acid metabolism, suggesting fundamental organic nitrogen compounds drive strain differentiation in the most globally abundant freshwater family. Overall, we observed seasonal and decadal trends in both interspecific ecological and intraspecific evolutionary processes. The convergence of microbial ecology and evolution on the same time scales demonstrates that understanding microbiomes requires a new unified approach that views ecology and evolution as a single continuum.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576074">
        <p class="paperTitle">Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cain, F. E.; Yamaguchi, K.</p>
        <p class="info">Score: 88.6, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576074' target='https://doi.org/10.1101/2024.01.17.576074'> 10.1101/2024.01.17.576074</a></p>
        <p class="abstract">The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.578695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.578695" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.578695">
        <p class="paperTitle">Lineage-resolved analysis of embryonic gene expression evolution in C. elegans and C. briggsae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.578695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.578695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Large, C. R. L.; Khanal, R.; Hillier, L. W.; Huynh, C.; Kubo, C.; Kim, J.; Waterston, R.; Murray, J. I.</p>
        <p class="info">Score: 14.1, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.578695' target='https://doi.org/10.1101/2024.02.03.578695'> 10.1101/2024.02.03.578695</a></p>
        <p class="abstract">What constraints govern the evolution of gene expression patterns across development remains a fundamental question of evolutionary biology. The advent of single-cell sequencing opens the possibility of learning these constraints by systematically profiling homologous cells across different organisms. The nematode C. elegans is a well-studied model for embryonic development, and its invariant lineage that is conserved with other Caenorhabditis species makes it an ideal model to directly compare gene expression between homologous progenitor and terminal cell types across evolution. We have measured the spatiotemporal divergence of gene expression across embryogenesis by collecting, annotating, and comparing the transcriptomes of homologous embryonic progenitors and terminal cell types, using a dataset comprising &gt;200,000 C. elegans cells and &gt;190,000 C. briggsae cells. We find a high level of similarity in gene expression programs between the species despite tens of millions of years of evolutionary divergence, consistent with their conserved developmental lineages. Even still, thousands of genes show divergence in their cell-type specific expression patterns, and these are enriched for categories involved in environmental response and behavior. Comparing the degree of expression conservation across cell types reveals that certain cell types such as neurons, have diverged more than others such as the intestine and body wall muscle. Taken together, this work identifies likely constraints on the evolution of developmental gene expression and provides a powerful resource for addressing diverse evolutionary questions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579293" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579293">
        <p class="paperTitle">Atypical landscape of transposable elements in the large genome of Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daron, J.; Bergman, A.; Lopez-Maestre, H.; LAMBRECHTS, L.</p>
        <p class="info">Score: 12.6, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579293' target='https://doi.org/10.1101/2024.02.07.579293'> 10.1101/2024.02.07.579293</a></p>
        <p class="abstract">Transposable elements (TEs) contribute significantly to variation in genome size among eukaryotic species, but the factors influencing TE accumulation and diversification are only partially understood. Most of our current knowledge about TE organization, dynamics and evolution derives from investigations in model organisms with a relatively small genome size such as Drosophila melanogaster or Arabidopsis thaliana. Whether the observed patterns hold true in larger genomes remains to be determined. The Diptera order is an ideal taxon to address this question, because it includes a forty-year model of TE biology (D. melanogaster) as well as mosquito species with significantly larger genomes. Here, we use a comparative genomics approach to characterize the genomic forces that have shaped the TE content of the Aedes aegypti genome (1.3 Gb) relative to the Anopheles coluzzii genome (300 Mb) and the D. melanogaster genome (180 Mb). Leveraging a newly developed high-quality TE library for Ae. aegypti, our results reveal a contrasted pattern of TE organization in Ae. aegypti compared to An. coluzzii and D. melanogaster. Our analyses suggest that the substantial TE fraction observed in the Ae. aegypti genome reflect both a high rate of TE transposition and a low rate of TE elimination. Together, our results indicate that TE organization and evolutionary dynamics in the large genome of Ae. aegypti are distinct from those of other dipterans with smaller genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578856" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578856">
        <p class="paperTitle">Quantifying internal conflicts and their threats to organismal form and fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schenkel, M. A.; Patten, M. M.; Agren, J. A.</p>
        <p class="info">Score: 10.9, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578856' target='https://doi.org/10.1101/2024.02.05.578856'> 10.1101/2024.02.05.578856</a></p>
        <p class="abstract">Evolutionary biologists often treat organisms as both fitness-maximizing agents and as the primary level at which adaptation manifests. Yet, genes and cells may also seek to optimize their fitness by distorting the Mendelian rules of transmission or by influencing organismal traits for their own benefit. Organismal form and fitness are therefore threatened from within by selfish genes and cells. However, to what extent such internal conflicts actually harm individual organisms and threaten our concept of the organism as the sole bearer of adaptation remains unclear. We introduce a mathematical framework to capture the threat posed by internal conflicts and develop two metrics to measure their various forms of harm. We name these metrics fitness unity and trait unity, and use them to refer to the threats posed by internal conflicts to an organisms role as the optimizing agent and the strategy wielded to achieve that optimization, respectively. We apply our framework to two examples of internal conflicts, genomic imprinting and sex ratio distortion, to illustrate how such harms from internal conflict may be quantified. We conclude by discussing the conditions under which internal conflict becomes sufficiently disruptive to organisms that it no longer makes sense to think of them as unified fitness-maximizing agents, but instead as adaptive compromises of multiple competing sub-agents.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578589" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578589">
        <p class="paperTitle">Microbes as manipulators of developmental life-history</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kustra, M. C.; Carrier, T. J.</p>
        <p class="info">Score: 8.5, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578589' target='https://doi.org/10.1101/2024.02.02.578589'> 10.1101/2024.02.02.578589</a></p>
        <p class="abstract">Marine invertebrates mainly reproduce by energy-poor eggs that develop into feeding larvae or energy-rich eggs that develop into non-feeding larvae1-4. Transitions between these reproductive strategies have been studied in detail5-7, yet the evolutionary factor(s) responsible for these switches remains elusive. Here, we use theoretical models to show that microbes with the capacity to manipulate host reproduction are one possible factor. We report that microbial manipulators create a sperm-limited environment that selects for larger eggs by shifting the hosts sex ratio towards female dominance and, as a result, serve as the evolutionary driver of transitions in the developmental life-history for marine invertebrates. Loss of a microbial manipulator can then recover the ancestral developmental life-history. We also document more than a dozen genera of marine invertebrates from throughout the worlds oceans that fit the framework of a microbe-induced switch between these predominate reproductive strategies. We anticipate that microbial manipulators have a yet-to-be appreciated influence on the life-history strategies of marine invertebrates. We find it paramount to understand if transitions in developmental life-history also occur without microbial manipulators as well as if the underlying mechanisms of these manipulations are convergent with terrestrial systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579433" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579433">
        <p class="paperTitle">A single giant cell at the origin of an evolutionary innovation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Badiane, A.; Viala, S.; Armisen, D.; Mignerey, I.; Forcet-Vauchel, C.; De Jesus Villanueva, C. N.; Van Belleghem, S.; Crumiere, A.; Santos, E.; Francesconi, M.; Khila, A.</p>
        <p class="info">Score: 8.4, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579433' target='https://doi.org/10.1101/2024.02.08.579433'> 10.1101/2024.02.08.579433</a></p>
        <p class="abstract">Key innovations play a central role in the diversification of lineages, yet our understanding of the mechanisms underlying their emergence remains fragmented. The propelling fan is a key innovation associated with the diversification of Rhagovelia water striders into fast flowing streams. Here, we show that a single giant cell is responsible for the development of the fan throughout embryogenesis. RNA interference against the fan-specific gene family gsha/mogsha depleted the fan in the embryo but did not alter the giant cell, indicating that these genes are not required for specifying the identity of this cell. gsha/mogsha instead seem to activate the expression of cuticular genes during early development, suggesting a role of these genes in regulating the accumulation of fan structural proteins. We also show that Hox genes act to block the giant cell in the fore- and rear-legs, thus restricting its fate to the mid-legs - the pair of legs that generates movement on water. In addition, Hox expression is specifically excluded from the giant cell, thus allowing the expression of the fan&#39;s developmental genetic program. This study reveals that a single cell can be sufficient to generate innovative traits that have been central for a lineage to acquire new ecological opportunities and burst into diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578966" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578966">
        <p class="paperTitle">A comprehensive examination of Chelicerate genomes reveals no evidence for a whole genome duplication among spiders and scorpions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, G. W. C.; McKibben, M. T. W.; Hahn, M. W.; Barker, M. S.</p>
        <p class="info">Score: 8.2, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578966' target='https://doi.org/10.1101/2024.02.05.578966'> 10.1101/2024.02.05.578966</a></p>
        <p class="abstract">Whole genome duplications (WGDs) can be a key event in evolution, playing a role in both adaptation and speciation. While WGDs are common throughout the history of plants, only a few examples have been proposed in metazoans. Among these, recent proposals of WGD events in Chelicerates, the group of Arthropods that includes horseshoe crabs, ticks, scorpions, and spiders, include several rounds in the history of horseshoe crabs, with an additional WGD proposed in the ancestor of spiders and scorpions. However, many of these inferences are based on evidence from only a small portion of the genome (in particular, the Hox gene cluster); therefore, genome-wide inferences with broader species sampling may give a clearer picture of WGDs in this clade. Here, we investigate signals of WGD in Chelicerates using whole genomes from 17 species. We employ multiple methods to look for these signals, including gene tree analysis of thousands of gene families, comparisons of synteny, and signals of divergence among within-species paralogs. We test several scenarios of WGD in Chelicerates using multiple species trees as a backbone for all hypotheses. While we do find support for at least one WGD in the ancestral horseshoe crab lineage, we find no evidence for a WGD in the history of spiders and scorpions using any genome-scale method. This study not only sheds light on genome evolution and phylogenetics within Chelicerates, but also demonstrates how a combination of comparative methods can be used to investigate signals of ancient WGDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578004" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578004">
        <p class="paperTitle">Variation in Neural Crest Development and Functional Divergence of sox10 Paralogs Contribute to Morphological Diversification in Cichlid Fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marconi, A.; Vernaz, G.; Karunaratna, A.; Ngochera, M. J.; Durbin, R.; Santos, M. E.</p>
        <p class="info">Score: 36.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578004' target='https://doi.org/10.1101/2024.01.30.578004'> 10.1101/2024.01.30.578004</a></p>
        <p class="abstract">Neural crest (NC) is a vertebrate-specific embryonic progenitor cell population at the basis of important vertebrate features such as the craniofacial skeleton and pigmentation patterns. Despite the wide-ranging variation of NC-derived traits across vertebrates, the contribution of NC to species diversification remains largely unexplored. Here, by leveraging the adaptive diversity of African Great Lakes cichlid species, we combined comparative transcriptomics and population genomics to investigate the role of NC development in morphological diversification. Our analysis revealed substantial differences in transcriptional landscapes across somitogenesis, an embryonic period coinciding with NC development and migration. Notably, several NC-related gene expression clusters showed both species-specific divergence in transcriptional landscapes and signatures of positive selection. Specifically, we identified two paralogs of the sox10 gene as prime NC-related candidates contributing to interspecific morphological variation, which displayed remarkable spatio-temporal expression variation in cichlids. Finally, through CRISPR-KO mutants, we experimentally validated the functional divergence between sox10 paralogs, with the acquisition of a novel role in cichlid skeletogenesis by sox10-like. Our study demonstrates the central role of NC-related processes - in particular those controlled by sox10s - in generating morphological diversification among closely-related species and lays the groundwork for further investigations into the mechanisms underpinning vertebrate NC diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578373">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578373" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578373">
        <p class="paperTitle">A covariant model of molecular and diversification patterns through time and the history of large clades</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578373" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578373" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Budd, G. E.; Mann, R. P.</p>
        <p class="info">Score: 6.8, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578373' target='https://doi.org/10.1101/2024.02.01.578373'> 10.1101/2024.02.01.578373</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWRate shifts in speciation and extinction have been recognised as important contributors to the creation of evolutionary patterns. In particular, the distribution of modern clade sizes is difficult to reconcile with models that do not include them. Although recent advances have allowed rate shifts to be integrated into evolutionary models, these have largely been for the purpose of inferring historical rate shifts across phylogenetic trees. In addition, these models have typically assumed an independence between patterns of diversification and rates of molecular and morphological evolution, despite there being mounting evidence of a connection between them. Here, we develop a new model with two principal goals: first, to explore the general patterns of diversification implied by constantly changing rates, and secondly to integrate diversification, molecular and morphological evolution into a single coherent framework. We thus develop and analyse a covariant birth-death process in which rates of all evolutionary processes (i.e. speciation, extinction and molecular and morphological change) covary continuously, both for each species and through time. We use this model to show that modern diversity is likely to be dominated by a small number of extremely large clades at any historical epoch; that these large clades are expected to be characterised by explosive early radiations accompanied by elevated rates of molecular evolution; and that extant organisms are likely to have evolved from species with unusually fast evolutionary rates. In addition, we show that under such a model, the amount of molecular change along a particular lineage is essentially independent of its height, which further weakens the molecular clock hypothesis. Finally, our model predicts the existence of &#34;living fossil&#34; sister groups to large clades that are both species poor and have exhibited slow rates of morphological and molecular change. Although our model is highly stochastic, it includes no special evolutionary moments or epochs. Our results thus demonstrate that the observed historical patterns of evolution can be modelled without invoking special evolutionary mechanisms or innovations that are unique to specific times or taxa, even when they are highly non-uniform: instead they could emerge from a process that is fundamentally homogeneous throughout time.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
