<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders
Authors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.
Score: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573503
Eye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders
Authors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.
Score: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573503
Eye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders
Authors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.
Score: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573503
Eye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders\nAuthors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.\nScore: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573503\nEye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size.",
  "keywords": [
    
  ],
  "articleBody": " Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders\nAuthors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.\nScore: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573503\nEye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size. By contrast, animals possessing larger visual systems can exhibit variable eye sizes within individuals. Spiders have up to four pairs of eyes whose size often varies, but their ontogenetic, static, and evolutionary allometry has not yet been studied in a comparative context. We report variable evolutionary and developmental dynamics in eye size across 1098 individuals in 39 species and 8 families, indicating selective pressures and constraints driving the evolution of different eye pairs and lineages. We observed variation in the relationship between eye and body size not only between taxa and visual ecologies, but between eye pairs within species, indicating that growth dynamics are variable and can be divergently adapted in particular eye pairs. Supplementing our sampling with a phylogenetically comprehensive dataset recently published by Wolff et al. (2022), we confirmed these findings across more than 400 species, found that ecological factors such as visual hunting, web building, and diurnality can also impact eye diameter, and identified significant allometric shifts across spider phylogeny using an unbiased approach, many of which coincide with ecological changes such as visual pursuit hunting strategies. This study represents the first detailed comparative exploration of visual allometry in spiders, revealing striking differences in eye growth not only between families but also within species. Our findings shed light on the relationship between spider visual systems and their diverse ecologies, and how spiders exploit the modular nature of their visual systems to balance selective pressures and optical and energetic constraints.\nBrood care in shell-dwelling cichlids is timed by independent maternal and larval clocks\nAuthors: Parker, A. V.; Stemmer, M.; Graetsch, S.; Dorigo, A.; Rodriguez Ramirez, O.; Adel, A.; Jordan, A. L.; Baier, H.\nScore: 11.9, Published: 2023-12-24 DOI: 10.1101/2023.12.22.573114\nSuccessful brood care requires a finely tuned interplay between caregiving parent and offspring. Cichlids are a diverse group of teleost fish known for their extensive parental care of altricial young. Female Lamprologus ocellatus lay their eggs in the protective confines of an abandoned snail shell, where they raise them to free-swimming larvae. Here we investigated whether the transition to larval independence is driven by the behavior of the mother, the offspring, or an interaction between the two. Using 3D printed shells with a window, we were able to observe the behavior of both mother and offspring inside the nest using automated object detection and tracking. We found that, once hatched, the larvae remain in the deep chambers of the shell, and at 9 dpf, actively emerge from the shell during the day. An inversion of larval phototactic behavior from dark- to light-seeking leads up to this emergence. Removal of the mother causes premature emergence of the larvae by a few days. The natural schedule (emergence at 9 dpf) is restored if the larvae are supplied with fresh water in the mothers absence. In cross-fostering experiments, we found that swapping out the mothers own clutch for older larvae forces a delay in emergence, as the foster mother prevents the larvae from leaving at 9 dpf. We conclude that larval and maternal behavior are each controlled by independent internal clocks, which, although normally synchronized, can be experimentally manipulated and brought into conflict. This study has thus revealed an innate sequence of behavioral adaptations that orchestrate brood care in shell-dwelling cichlids.\nPhylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world's most common underwater architects\nAuthors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.\nScore: 78.7, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572910\nCaddisflies (Trichoptera) are among the most diverse groups of freshwater animals with more than 16,000 described species. They play an outsized role in freshwater ecology and environmental engineering in streams, rivers, and lakes. Because of this, they are frequently used as indicator organisms in biomonitoring programs. Despite their importance, key questions concerning the evolutionary history of caddisflies, such as the timing and origin of larval case-making, have been unanswered due to the lack of a well-resolved phylogenetic tree. To shed light on these questions in Trichoptera, we estimated a phylogenetic tree using a combination of transcriptomes and targeted enrichment data for 206 species, representing 48 of 52 extant families and 174 genera. We calibrated and dated the tree with a set of 33 carefully selected fossils. The first caddisflies originated in the Permian and the major suborders began to diversify in the Triassic. Ancestral state reconstruction and diversification analysis revealed that portable case-making evolved in three separate lineages and shifts in diversification occurred in concert with key evolutionary innovations other than case-making.\nThe role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication\nAuthors: Monsen, O.; Gronvold, L.; Datsomor, A.; Harvey, T. N.; Kijas, J.; Suh, A. S.-J.; Hvidsten, T. R.; Sandve, S. R.\nScore: 4.8, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573861\nBackground: Two of the most potent drivers of genome evolution in eukaryotes are whole genome duplications (WGD) and transposable element (TE) activity. These two mutational forces can also play synergistic roles; WGDs result in both cellular stress and functional redundancy, which would allow TEs to escape host-silencing mechanisms and effectively spread with reduced impact on fitness. As TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE-activity following WGD are likely to impact evolution of gene regulation. However, the role of TEs in genome regulatory remodelling after WGDs is unclear. Here we used the genome of Atlantic salmon, which is known to have experienced massive expansion of TEs after a WGD ~100 Mya, as a model system to explore the synergistic roles of TEs and WGDs on genome regulatory evolution. Results: We identified 61,309 putative TE-CREs in Atlantic salmon using chromatin accessibility data from brain and liver. Of these, 82% were tissue specific to liver (43%) or brain (39%) and TE-CREs originating from retroelements were twice as common as those originating from DNA elements. Signatures of selection shaping TE-CRE evolution were evident from depletion of TEs in open chromatin, a bias in tissue-shared TE-CREs towards older TE-insertions, as well as tissue-specific processes shaping the TE-CRE repertoire. The DTT elements (Tc1-Mariners), which exploded in numbers at the time of the WGD, were significantly less prone to evolve into TE-CREs and significantly less potent in driving or repressing transcription compared to other TE-derived sequences. A minority of TEs (16% of consensus TEs) accounted for the origin of 46% of all TE-CREs, but these 'CRE-superspreaders' were not temporally associated with the WGD. Rather, the majority of TE-CREs, including those found to be significantly associated with gene regulatory evolution and thus found to drive or repress transcription, evolved from TE activity occurring across tens of millions of years following the WGD event. Conclusion: Our results do not support a WGD-associated TE-CRE rewiring of gene regulation. Instead we find that TEs from diverse superfamilies have been particularly effective in spreading TE-CREs and shaping gene regulatory networks under tissue-specific selection pressures, across millions of years following the salmonid WGD.\nSex chromosome dominance in a UV sexual system\nAuthors: Vigneau, J.; Martinho, C.; Zheng, M.; Godfroy, O.; Haas, F. B.; Borg, M.; Coelho, S. M.\nScore: 4.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573518\nThe alternation between multicellular haploid gametophytes and diploid sporophytes is a defining feature of most plant and algal life cycles. In such organisms, male and female sexes are determined in the haploid gametophyte with a female (U) or male (V) sex chromosome. Once the U and V chromosomes unite at fertilisation, sex determination no longer occurs, raising key questions about the fate of UV sex chromosomes in the diploid sporophyte stage of the life cycle. Here, we unravel the genetic and molecular interactions between the U and V chromosomes by assessing transcriptional and chromatin states across the life cycle of the brown alga Ectocarpus alongside ouroboros mutants that decouple life cycle stage from ploidy. We reveal how sex chromosome genes are developmentally regulated across the life cycle, with genes involved in female sex determination in particular undergoing strong down-regulation in the sporophyte. Diploid ouroboros mutants containing both a U and V sex chromosome behave as functional male gametophytes yet still exhibit feminized transcription, suggesting that presence of the V chromosome alone is insufficient to fully suppress female developmental program. Although the silencing of sex chromosome genes in the diploid sporophyte does not appear to correlate with localised changes in chromatin state, small RNAs may play a role in the repression of a female sex-linked gene. Finally, we show how histone H3K79me2 is globally re-configured in the diploid phase of the life cycle, including the sex determining region of the UV sex chromosomes. Contrary to its pattern in the haploid gametophyte, H3K79me2 no longer associates with repressed genes in the diploid sporophyte, suggesting that the function of this histone mark in Ectocarpus may be more complex than previously appreciated.\nPhylogenetic modeling provides evidence for sudden shifts in expression after small-scale duplication in vertebrates and strong support for the ortholog conjecture\nAuthors: Begum, T.; Duchen, P.; Bucao, C.; Robinson-Rechavi, M.\nScore: 4.0, Published: 2023-12-29 DOI: 10.1101/2023.12.29.571877\nGene duplication is a potential source of innovation, but the evolutionary dynamics of functional change are still poorly understood. The \"ortholog conjecture\" posits that such innovation or other functional change is more common after duplication than in its absence, but is still debated. We have applied a Levy model of evolutionary trait jumps to two phenotypes of gene function: average gene expression levels, and tissue specificity. Such instantaneous jumps in trait values typically occur during the transition of organisms into new adaptive zones, and thus provide insights into the role of duplication in adaptation. Using time-calibrated phylogenies of 15 vertebrates, including 5 teleost fishes, we show for the first time that such a mechanism of trait divergence strongly affects paralogs, in addition to other modes of functional evolution. We find that at least 25% of teleost fish small-scale duplicates follow a rapid evolutionary rate shift model for both traits, much more than after speciations. We show that such trait jumps play a major role in the evolution of duplicates. While there is some evidence for more positive selection at the protein-coding level after duplication, this does not appear strongly linked to the presence of jumps in expression. This helps to explain higher phylogenetic independent contrasts between paralogs, supporting the ortholog conjecture. However, genome-wide duplicates (ohnologs) do not support such a trait jump model, and thus follow a different evolutionary dynamic, while also supporting the ortholog conjecture.\nFunctional consequences of reductive protein evolution in a minimal eukaryotic genome\nAuthors: Jiang, J.; Qu, R.; Grigorescu, M.; Zhao, W.; Reinke, A. W.\nScore: 3.0, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573788\nMicrosporidia are parasites with the smallest known eukaryotic genomes. The extent of protein loss in these organisms has been well documented, but much less is known about how compaction of microsporidia proteins affects their function. Taking a comparative genomic approach, we identified microsporidia orthologs of budding yeast proteins and show that these orthologs are enriched for essential yeast genes. We show that the median microsporidia protein is 21% shorter than its yeast counterpart and although extensive protein loss occurred after the divergence of microsporidia, reduced protein sizes were already present in microsporidian relatives. Microsporidia proteins are shorter through reduced domain lengths, diminished linker lengths, and domain loss, with 21% of microsporidia orthologs having lost domains present in yeast. On average, 34% of microsporidia orthologs have lost C-terminal residues essential for function in yeast, including 13 essential domains lost per genome. We also found that microsporidia display distinct phylogenetic patterns of domain loss, with losses occurring in a clade-specific manner. To investigate conservation of function, we used yeast complementation assays to test orthologs from several microsporidia species and their relative Rozella allomycis. These experiments reveal that most microsporidia proteins cannot complement their yeast orthologs, the ability to complement is about three-fold less than observed for R. allomycis orthologs, and proteins that do not complement are more reduced in length than their yeast orthologs. Altogether, our results demonstrate the drastic reduction of microsporidia proteins and show that these reductions have resulted in functional divergence from their fungal ancestors.\nChewing Through Challenges: Exploring the Evolutionary Pathways to Wood-Feeding in Insects\nAuthors: Beza-Beza, C. F.; Wiegmann, B. M.; Ware, J. A.; Petersen, M.; Gunter, N.; Cole, M. E.; Schwarz, M.; Bertone, M. A.; Young, D.; Mikaelyan, A.\nScore: 2.9, Published: 2023-12-28 DOI: 10.1101/2023.12.27.573094\nDecaying wood, while an abundant and stable resource, presents considerable nutritional challenges due to its structural rigidity, chemical recalcitrance, and low nitrogen content. Despite these challenges, certain insect lineages have successfully evolved saproxylophagy (consuming and deriving sustenance from decaying wood), impacting nutrient recycling in ecosystems and carbon sequestration dynamics. This study explores the uneven phylogenetic distribution of saproxylophagy across insects and delves into the evolutionary origins of this trait in disparate insect orders. Employing a comprehensive analysis of gut microbiome data, encompassing both previously published datasets and newly generated data, from both saproxylophagous insects and their non-saproxylophagous relatives, this Hypothesis paper discusses the broader phylogenetic context and potential morphological, physiological, and symbiotic adaptations necessary for this dietary specialization. The study proposes the \"Detritivore-First Hypothesis,\" suggesting an evolutionary pathway to saproxylophagy through detritivory, and highlights the critical role of symbiotic gut microbiomes in the digestion of decaying wood. The article aims to provide a deeper understanding of the macroevolutionary landscape and mechanisms underpinning the multiple origins and distribution of saproxylophagy in insects.\nPhysical constraints during Snowball Earth drive the evolution of multicellularity\nAuthors: Crockett, W. W.; Shaw, J.; Simpson, C.; Kempes, C. P.\nScore: 13.3, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570654\nMolecular and fossil evidence suggest that complex eukaryotic multicellularity evolved during the late Neoproterozoic era, coincident with Snowball Earth glaciations, where ice sheets covered most of the globe. During this period, environmental conditions--such as sea water temperature and the availability of photosynthetically active light in the oceans--likely changed dramatically. Such changes would have had significant effects on both resource availability and optimal phenotypes. Here, we construct and apply mechanistic models to explore (i) how environmental changes during Snowball Earth and biophysical constraints generated selective pressures and (ii) how these pressures may have had differential effects on organisms with different forms of biological organization. By testing a series of alternate--and commonly debated--hypotheses, we demonstrate how multicellularity was likely acquired differently in eukaryotes and prokaryotes due to selective differences in the biophysical and metabolic regimes they experience: decreasing temperatures and resource-availability instigated by the onset of glaciations generated selective pressures towards smaller sizes in organisms in a diffusive regime and towards larger sizes in motile heterotrophs. These results suggest that changing environmental conditions during Snowball Earth glaciations gave multicellular eukaryotes an evolutionary advantage, paving the way for the complex multicellular lineages that followed.\nEvolutionary trends of alternative splicing\nAuthors: de la Fuente, R.; Diaz-Villanueva, W.; Arnau, V.; Moya, A.\nScore: 2.5, Published: 2023-12-27 DOI: 10.1101/2023.12.26.571677\nAlternative splicing is the post-transcriptional mechanism by which many different isoforms are generated from a single gene, contributing to increasing spatio-temporal transcriptome complexity. We propose a novel genome-level measure of alternative splicing, which associates it with a single value for each species. Thus, a comparative analysis of species spanning the whole tree of life has revealed certain evolutionary trends in alternative splicing, prevalence in specific lineages, and relation to genome compositional structures.\n",
  "wordCount" : "2664",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573503">
        <p class="paperTitle">Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.</p>
        <p class="info">Score: 14.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573503' target='https://doi.org/10.1101/2023.12.28.573503'> 10.1101/2023.12.28.573503</a></p>
        <p class="abstract">Eye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size. By contrast, animals possessing larger visual systems can exhibit variable eye sizes within individuals. Spiders have up to four pairs of eyes whose size often varies, but their ontogenetic, static, and evolutionary allometry has not yet been studied in a comparative context. We report variable evolutionary and developmental dynamics in eye size across 1098 individuals in 39 species and 8 families, indicating selective pressures and constraints driving the evolution of different eye pairs and lineages. We observed variation in the relationship between eye and body size not only between taxa and visual ecologies, but between eye pairs within species, indicating that growth dynamics are variable and can be divergently adapted in particular eye pairs. Supplementing our sampling with a phylogenetically comprehensive dataset recently published by Wolff et al. (2022), we confirmed these findings across more than 400 species, found that ecological factors such as visual hunting, web building, and diurnality can also impact eye diameter, and identified significant allometric shifts across spider phylogeny using an unbiased approach, many of which coincide with ecological changes such as visual pursuit hunting strategies. This study represents the first detailed comparative exploration of visual allometry in spiders, revealing striking differences in eye growth not only between families but also within species. Our findings shed light on the relationship between spider visual systems and their diverse ecologies, and how spiders exploit the modular nature of their visual systems to balance selective pressures and optical and energetic constraints.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573114">
        <p class="paperTitle">Brood care in shell-dwelling cichlids is timed by independent maternal and larval clocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parker, A. V.; Stemmer, M.; Graetsch, S.; Dorigo, A.; Rodriguez Ramirez, O.; Adel, A.; Jordan, A. L.; Baier, H.</p>
        <p class="info">Score: 11.9, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573114' target='https://doi.org/10.1101/2023.12.22.573114'> 10.1101/2023.12.22.573114</a></p>
        <p class="abstract">Successful brood care requires a finely tuned interplay between caregiving parent and offspring. Cichlids are a diverse group of teleost fish known for their extensive parental care of altricial young. Female Lamprologus ocellatus lay their eggs in the protective confines of an abandoned snail shell, where they raise them to free-swimming larvae. Here we investigated whether the transition to larval independence is driven by the behavior of the mother, the offspring, or an interaction between the two. Using 3D printed shells with a window, we were able to observe the behavior of both mother and offspring inside the nest using automated object detection and tracking. We found that, once hatched, the larvae remain in the deep chambers of the shell, and at 9 dpf, actively emerge from the shell during the day. An inversion of larval phototactic behavior from dark- to light-seeking leads up to this emergence. Removal of the mother causes premature emergence of the larvae by a few days. The natural schedule (emergence at 9 dpf) is restored if the larvae are supplied with fresh water in the mothers absence. In cross-fostering experiments, we found that swapping out the mothers own clutch for older larvae forces a delay in emergence, as the foster mother prevents the larvae from leaving at 9 dpf. We conclude that larval and maternal behavior are each controlled by independent internal clocks, which, although normally synchronized, can be experimentally manipulated and brought into conflict. This study has thus revealed an innate sequence of behavioral adaptations that orchestrate brood care in shell-dwelling cichlids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572910">
        <p class="paperTitle">Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.</p>
        <p class="info">Score: 78.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572910' target='https://doi.org/10.1101/2023.12.21.572910'> 10.1101/2023.12.21.572910</a></p>
        <p class="abstract">Caddisflies (Trichoptera) are among the most diverse groups of freshwater animals with more than 16,000 described species. They play an outsized role in freshwater ecology and environmental engineering in streams, rivers, and lakes. Because of this, they are frequently used as indicator organisms in biomonitoring programs. Despite their importance, key questions concerning the evolutionary history of caddisflies, such as the timing and origin of larval case-making, have been unanswered due to the lack of a well-resolved phylogenetic tree. To shed light on these questions in Trichoptera, we estimated a phylogenetic tree using a combination of transcriptomes and targeted enrichment data for 206 species, representing 48 of 52 extant families and 174 genera. We calibrated and dated the tree with a set of 33 carefully selected fossils. The first caddisflies originated in the Permian and the major suborders began to diversify in the Triassic. Ancestral state reconstruction and diversification analysis revealed that portable case-making evolved in three separate lineages and shifts in diversification occurred in concert with key evolutionary innovations other than case-making.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573861" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573861">
        <p class="paperTitle">The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monsen, O.; Gronvold, L.; Datsomor, A.; Harvey, T. N.; Kijas, J.; Suh, A. S.-J.; Hvidsten, T. R.; Sandve, S. R.</p>
        <p class="info">Score: 4.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573861' target='https://doi.org/10.1101/2024.01.02.573861'> 10.1101/2024.01.02.573861</a></p>
        <p class="abstract">Background: Two of the most potent drivers of genome evolution in eukaryotes are whole genome duplications (WGD) and transposable element (TE) activity. These two mutational forces can also play synergistic roles; WGDs result in both cellular stress and functional redundancy, which would allow TEs to escape host-silencing mechanisms and effectively spread with reduced impact on fitness. As TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE-activity following WGD are likely to impact evolution of gene regulation. However, the role of TEs in genome regulatory remodelling after WGDs is unclear. Here we used the genome of Atlantic salmon, which is known to have experienced massive expansion of TEs after a WGD ~100 Mya, as a model system to explore the synergistic roles of TEs and WGDs on genome regulatory evolution. Results: We identified 61,309 putative TE-CREs in Atlantic salmon using chromatin accessibility data from brain and liver. Of these, 82% were tissue specific to liver (43%) or brain (39%) and TE-CREs originating from retroelements were twice as common as those originating from DNA elements. Signatures of selection shaping TE-CRE evolution were evident from depletion of TEs in open chromatin, a bias in tissue-shared TE-CREs towards older TE-insertions, as well as tissue-specific processes shaping the TE-CRE repertoire. The DTT elements (Tc1-Mariners), which exploded in numbers at the time of the WGD, were significantly less prone to evolve into TE-CREs and significantly less potent in driving or repressing transcription compared to other TE-derived sequences. A minority of TEs (16% of consensus TEs) accounted for the origin of 46% of all TE-CREs, but these &#39;CRE-superspreaders&#39; were not temporally associated with the WGD. Rather, the majority of TE-CREs, including those found to be significantly associated with gene regulatory evolution and thus found to drive or repress transcription, evolved from TE activity occurring across tens of millions of years following the WGD event. Conclusion: Our results do not support a WGD-associated TE-CRE rewiring of gene regulation. Instead we find that TEs from diverse superfamilies have been particularly effective in spreading TE-CREs and shaping gene regulatory networks under tissue-specific selection pressures, across millions of years following the salmonid WGD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573518">
        <p class="paperTitle">Sex chromosome dominance in a UV sexual system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vigneau, J.; Martinho, C.; Zheng, M.; Godfroy, O.; Haas, F. B.; Borg, M.; Coelho, S. M.</p>
        <p class="info">Score: 4.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573518' target='https://doi.org/10.1101/2023.12.28.573518'> 10.1101/2023.12.28.573518</a></p>
        <p class="abstract">The alternation between multicellular haploid gametophytes and diploid sporophytes is a defining feature of most plant and algal life cycles. In such organisms, male and female sexes are determined in the haploid gametophyte with a female (U) or male (V) sex chromosome. Once the U and V chromosomes unite at fertilisation, sex determination no longer occurs, raising key questions about the fate of UV sex chromosomes in the diploid sporophyte stage of the life cycle. Here, we unravel the genetic and molecular interactions between the U and V chromosomes by assessing transcriptional and chromatin states across the life cycle of the brown alga Ectocarpus alongside ouroboros mutants that decouple life cycle stage from ploidy. We reveal how sex chromosome genes are developmentally regulated across the life cycle, with genes involved in female sex determination in particular undergoing strong down-regulation in the sporophyte. Diploid ouroboros mutants containing both a U and V sex chromosome behave as functional male gametophytes yet still exhibit feminized transcription, suggesting that presence of the V chromosome alone is insufficient to fully suppress female developmental program. Although the silencing of sex chromosome genes in the diploid sporophyte does not appear to correlate with localised changes in chromatin state, small RNAs may play a role in the repression of a female sex-linked gene. Finally, we show how histone H3K79me2 is globally re-configured in the diploid phase of the life cycle, including the sex determining region of the UV sex chromosomes. Contrary to its pattern in the haploid gametophyte, H3K79me2 no longer associates with repressed genes in the diploid sporophyte, suggesting that the function of this histone mark in Ectocarpus may be more complex than previously appreciated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.571877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.571877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.571877">
        <p class="paperTitle">Phylogenetic modeling provides evidence for sudden shifts in expression after small-scale duplication in vertebrates and strong support for the ortholog conjecture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.571877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.571877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Begum, T.; Duchen, P.; Bucao, C.; Robinson-Rechavi, M.</p>
        <p class="info">Score: 4.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.571877' target='https://doi.org/10.1101/2023.12.29.571877'> 10.1101/2023.12.29.571877</a></p>
        <p class="abstract">Gene duplication is a potential source of innovation, but the evolutionary dynamics of functional change are still poorly understood. The &#34;ortholog conjecture&#34; posits that such innovation or other functional change is more common after duplication than in its absence, but is still debated. We have applied a Levy model of evolutionary trait jumps to two phenotypes of gene function: average gene expression levels, and tissue specificity. Such instantaneous jumps in trait values typically occur during the transition of organisms into new adaptive zones, and thus provide insights into the role of duplication in adaptation. Using time-calibrated phylogenies of 15 vertebrates, including 5 teleost fishes, we show for the first time that such a mechanism of trait divergence strongly affects paralogs, in addition to other modes of functional evolution. We find that at least 25% of teleost fish small-scale duplicates follow a rapid evolutionary rate shift model for both traits, much more than after speciations. We show that such trait jumps play a major role in the evolution of duplicates. While there is some evidence for more positive selection at the protein-coding level after duplication, this does not appear strongly linked to the presence of jumps in expression. This helps to explain higher phylogenetic independent contrasts between paralogs, supporting the ortholog conjecture. However, genome-wide duplicates (ohnologs) do not support such a trait jump model, and thus follow a different evolutionary dynamic, while also supporting the ortholog conjecture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573788">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573788" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573788">
        <p class="paperTitle">Functional consequences of reductive protein evolution in a minimal eukaryotic genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573788" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573788" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, J.; Qu, R.; Grigorescu, M.; Zhao, W.; Reinke, A. W.</p>
        <p class="info">Score: 3.0, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573788' target='https://doi.org/10.1101/2023.12.31.573788'> 10.1101/2023.12.31.573788</a></p>
        <p class="abstract">Microsporidia are parasites with the smallest known eukaryotic genomes. The extent of protein loss in these organisms has been well documented, but much less is known about how compaction of microsporidia proteins affects their function. Taking a comparative genomic approach, we identified microsporidia orthologs of budding yeast proteins and show that these orthologs are enriched for essential yeast genes. We show that the median microsporidia protein is 21% shorter than its yeast counterpart and although extensive protein loss occurred after the divergence of microsporidia, reduced protein sizes were already present in microsporidian relatives. Microsporidia proteins are shorter through reduced domain lengths, diminished linker lengths, and domain loss, with 21% of microsporidia orthologs having lost domains present in yeast. On average, 34% of microsporidia orthologs have lost C-terminal residues essential for function in yeast, including 13 essential domains lost per genome. We also found that microsporidia display distinct phylogenetic patterns of domain loss, with losses occurring in a clade-specific manner. To investigate conservation of function, we used yeast complementation assays to test orthologs from several microsporidia species and their relative Rozella allomycis. These experiments reveal that most microsporidia proteins cannot complement their yeast orthologs, the ability to complement is about three-fold less than observed for R. allomycis orthologs, and proteins that do not complement are more reduced in length than their yeast orthologs. Altogether, our results demonstrate the drastic reduction of microsporidia proteins and show that these reductions have resulted in functional divergence from their fungal ancestors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573094">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573094" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573094">
        <p class="paperTitle">Chewing Through Challenges: Exploring the Evolutionary Pathways to Wood-Feeding in Insects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573094" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573094" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beza-Beza, C. F.; Wiegmann, B. M.; Ware, J. A.; Petersen, M.; Gunter, N.; Cole, M. E.; Schwarz, M.; Bertone, M. A.; Young, D.; Mikaelyan, A.</p>
        <p class="info">Score: 2.9, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573094' target='https://doi.org/10.1101/2023.12.27.573094'> 10.1101/2023.12.27.573094</a></p>
        <p class="abstract">Decaying wood, while an abundant and stable resource, presents considerable nutritional challenges due to its structural rigidity, chemical recalcitrance, and low nitrogen content. Despite these challenges, certain insect lineages have successfully evolved saproxylophagy (consuming and deriving sustenance from decaying wood), impacting nutrient recycling in ecosystems and carbon sequestration dynamics. This study explores the uneven phylogenetic distribution of saproxylophagy across insects and delves into the evolutionary origins of this trait in disparate insect orders. Employing a comprehensive analysis of gut microbiome data, encompassing both previously published datasets and newly generated data, from both saproxylophagous insects and their non-saproxylophagous relatives, this Hypothesis paper discusses the broader phylogenetic context and potential morphological, physiological, and symbiotic adaptations necessary for this dietary specialization. The study proposes the &#34;Detritivore-First Hypothesis,&#34; suggesting an evolutionary pathway to saproxylophagy through detritivory, and highlights the critical role of symbiotic gut microbiomes in the digestion of decaying wood. The article aims to provide a deeper understanding of the macroevolutionary landscape and mechanisms underpinning the multiple origins and distribution of saproxylophagy in insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570654">
        <p class="paperTitle">Physical constraints during Snowball Earth drive the evolution of multicellularity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crockett, W. W.; Shaw, J.; Simpson, C.; Kempes, C. P.</p>
        <p class="info">Score: 13.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570654' target='https://doi.org/10.1101/2023.12.07.570654'> 10.1101/2023.12.07.570654</a></p>
        <p class="abstract">Molecular and fossil evidence suggest that complex eukaryotic multicellularity evolved during the late Neoproterozoic era, coincident with Snowball Earth glaciations, where ice sheets covered most of the globe. During this period, environmental conditions--such as sea water temperature and the availability of photosynthetically active light in the oceans--likely changed dramatically. Such changes would have had significant effects on both resource availability and optimal phenotypes. Here, we construct and apply mechanistic models to explore (i) how environmental changes during Snowball Earth and biophysical constraints generated selective pressures and (ii) how these pressures may have had differential effects on organisms with different forms of biological organization. By testing a series of alternate--and commonly debated--hypotheses, we demonstrate how multicellularity was likely acquired differently in eukaryotes and prokaryotes due to selective differences in the biophysical and metabolic regimes they experience: decreasing temperatures and resource-availability instigated by the onset of glaciations generated selective pressures towards smaller sizes in organisms in a diffusive regime and towards larger sizes in motile heterotrophs. These results suggest that changing environmental conditions during Snowball Earth glaciations gave multicellular eukaryotes an evolutionary advantage, paving the way for the complex multicellular lineages that followed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.571677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.571677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.571677">
        <p class="paperTitle">Evolutionary trends of alternative splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.571677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.571677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de la Fuente, R.; Diaz-Villanueva, W.; Arnau, V.; Moya, A.</p>
        <p class="info">Score: 2.5, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.571677' target='https://doi.org/10.1101/2023.12.26.571677'> 10.1101/2023.12.26.571677</a></p>
        <p class="abstract">Alternative splicing is the post-transcriptional mechanism by which many different isoforms are generated from a single gene, contributing to increasing spatio-temporal transcriptome complexity. We propose a novel genome-level measure of alternative splicing, which associates it with a single value for each species. Thus, a comparative analysis of species spanning the whole tree of life has revealed certain evolutionary trends in alternative splicing, prevalence in specific lineages, and relation to genome compositional structures.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
