<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)
Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.
Score: 55.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084
Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)
Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.
Score: 55.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084
Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)
Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.
Score: 55.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084
Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)\nAuthors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.\nScore: 55.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084\nNearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision.",
  "keywords": [
    
  ],
  "articleBody": " New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)\nAuthors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.\nScore: 55.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084\nNearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision. The adaptive changes in sensory abilities of animals during these transitions, both at the genetic and neural levels, are largely unexplored. Butterflies and moths, with their diverse diel-niche shifts, are an ideal group for investigating the gene evolution linked to these transitions. While most butterflies are day-flying, hedylid butterflies are unique in being primarily nocturnal, and they represent an important evolutionary shift from diurnality to nocturnality in this clade. Here, we sequence the first high-quality Hedylidae genome and functionally annotate genes to understand genomic changes associated with shifts in diel niche. Comparing Hedylidae visual genes against day- and night-flying Lepidoptera species revealed that visual genes are highly conserved, with no major losses. However, hedylid butterfly opsins were more similar to nocturnal moths than their diurnal congeners. Tests on the evolutionary rates (dN/dS) confirmed that color vision opsins were under strong selection, similar to nocturnal moths. We propose that a convergent event of sequence evolution took place when these butterflies became nocturnal, approximately 98 million years ago.\nFrequent transitions from night-to-day activity after mass extinctions\nAuthors: Shafer, M. E. R.; Nichols, A. L. A.; Schier, A. F.; Salzburger, W.\nScore: 34.0, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564421\nWhy certain species survive large-scale extinction events while others do not is poorly understood. While the fossil record can provide insights into morphological adaptations that increase survival probabilities, it provides limited information regarding behavioural traits. Here we integrate behavioural data with phylogenetic comparative models to study the evolution of day-night activity patterns (nocturnality and diurnality) in bony fishes, which have persisted through the last four mass extinction events. Our findings in fish, combined with data across all other clades of vertebrates, provide four lines of evidence that nocturnality conferred an evolutionary advantage during mass extinctions, and that frequent nocturnal-to-diurnal transitions facilitated post-extinction diversification: First, phylogenetic reconstruction indicates that the last common ancestors of all vertebrates and of clades of bony vertebrates were nocturnal. Second, in the lineage of bony fishes, which contains over half of all vertebrate species, twice as many transitions between nocturnal and diurnal activity patterns have occurred compared to tetrapods. Third, within a specific ecological niche, different species exhibit distinct temporal activity patterns, suggesting widespread temporal niche partitioning. Fourth, independent bursts in night-to-day transitions followed large-scale extinction events during the last two geological eras in all four major bony vertebrate groups. These observations suggest that ancestral nocturnality and frequent transitions to diurnality helped vertebrates survive and diversify in the face of extinction events, such as those expected during the current \"6th mass extinction\" event caused by anthropogenic climate change.\nOrigin Of Evolutionary Bifurcation In An Enzyme\nAuthors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.\nScore: 17.4, Published: 2023-11-26 DOI: 10.1101/2023.11.25.568631\nEvolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life. Here, we present a detailed molecular description of a bifurcation event that rapidly led to the emergence of two distinct enzymes from a common ancestor. When initiated from two starting points that differed by a single amino acid, the laboratory evolution of a phosphotriesterase (PTE) toward arylester hydrolysis resulted in different genetic and phenotypic outcomes. One trajectory led to a \u003e35,000-fold increase in activity via the reorganization of its active site to achieve exquisite enzyme-substrate complementarity. The second trajectory gave rise to an evolved variant with a [~]500-fold increase in activity, but exhibiting an alternative substrate binding mode resulting from the destabilization of an active site loop. While initial mutations tend to dictate mutational accessibility, we rather observed the gradual divergence and specialisation of each trajectory, following the emergence of distinct molecular interaction networks. Intramolecular epistasis underlay pathway bifurcation by promoting unique synergistic interactions within each trajectory, while restricting the fixation of mutation across pathways. Our results illustrate how distinct molecular outcomes can radiate from a common protein ancestor and give rise to phenotypic diversity.\nEpigenetic predictors of species maximum lifespan and other life history traits in mammals\nAuthors: Li, C. Z.; Haghani, A.; Yan, Q.; Lu, A. T.; Zhang, J.; Fei, Z.; Ernst, J.; Yang, X.; Gladyshev, V. N.; Raj, K.; Seluanov, A.; Gorbunova, V.; Horvath, S.\nScore: 26.8, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565286\nMaximum lifespan is an intrinsic characteristic of a biological species and is defined as the longest time an individual of a species has been reported to survive. By analyzing 15K samples derived from 348 mammalian species representing 25 taxonomic orders we previously identified CpG methylation sites associated with maximum lifespan. Here we present accurate DNA methylation-based (DNAm) predictors of maximum lifespan (r=0.89), average gestation time (r=0.96), and age at sexual maturity (r=0.85). Our DNAm maximum lifespan predictor indicates a potential innate longevity advantage for females over males in 17 mammalian species such as humans, red deer, and cattle. The DNAm maximum lifespan predictions do not vary significantly by caloric restriction and partial reprogramming. Genetic disruptions in the somatotropic axis, which includes growth hormone, IGF-1, and their related receptors, have an impact on DNAm maximum lifespan only in select mouse tissues. Cancer mortality rates in major mammalian orders show no correlation with our epigenetic estimates of life history traits. The DNAm maximum lifespan predictor does not detect variation in lifespan between individuals of the same species, such as between the breeds of dogs. We also present the first prototypes of accurate pan mammalian DNAm predictors of sex and tissue type. Collectively, our findings indicate that maximum lifespan is determined, at least in part, by an epigenetic signature that is an intrinsic property of each species and is distinct from the signatures that relate to individual lifespan, which is unaffected by interventions influencing the mortality risk of individuals.\nFunctional and evolutionary integration of a fungal gene with a bacterial operon\nAuthors: Sun, L.; David, K. T.; Wolters, J. F.; Karlen, S. D.; Goncalves, C.; Opulente, D. A.; LaBella, A. L.; Groenewald, M.; Zhou, X.; Shen, X.-X.; Rokas, A.; Hittinger, C. T.\nScore: 14.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568075\nSiderophores are crucial for iron-scavenging in microorganisms. While many yeasts can uptake siderophores produced by other organisms, they are typically unable to synthesize siderophores themselves. In contrast, Wickerhamiella/Starmerella (W/S) clade yeasts gained the capacity to make the siderophore enterobactin following the remarkable horizontal acquisition of a bacterial operon enabling enterobactin synthesis. Yet, how these yeasts absorb the iron bound by enterobactin remains unresolved. Here, we demonstrate that Enb1 is the key enterobactin importer in the W/S-clade species Starmerella bombicola. Through phylogenomic analyses, we show that ENB1 is present in all W/S clade yeast species that retained the enterobactin biosynthetic genes. Conversely, it is absent in species that lost the ent genes, except for Starmerella stellata, making this species the only cheater in the W/S clade that can utilize enterobactin without producing it. Through phylogenetic analyses, we infer that ENB1 is a fungal gene that likely existed in the W/S clade prior to the acquisition of the ent genes and subsequently experienced multiple gene losses and duplications. Through phylogenetic topology tests, we show that ENB1 likely underwent horizontal gene transfer from an ancient W/S clade yeast to the order Saccharomycetales, which includes the model yeast Saccharomyces cerevisiae, followed by extensive secondary losses. Taken together, these results suggest that the fungal ENB1 and bacterial ent genes were cooperatively integrated into a functional unit within the W/S clade that enabled adaptation to iron-limited environments. This integrated fungal-bacterial circuit and its dynamic evolution determines the extant distribution of yeast enterobactin producers and cheaters.\nPopulation genomic history of the endangered Anatolian and Cyprian mouflons in relation to worldwide wild, feral and domestic sheep lineages\nAuthors: Atag, G.; Kaptan, D.; Yüncü, E.; Vural, K. B.; Mereu, P.; Pirastru, M.; Barbato, M.; Leoni, G. G.; Güler, M. N.; Er, T.; Eker, E.; Yazıcı, T. D.; Kılıc, M. S.; Altınısık, N. E.; Celik, E. A.; Miranda, P. M.; Dehasque, M.; Floridia, V.; Götherström, A.; Bilgin, C. C.; Togan, I.; Günther, T.; Özer, F.; Hadjisterkotis, E.; Somel, M.\nScore: 12.6, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568468\nOnce widespread in their homelands, Anatolian mouflon (Ovis gmelini anatolica) and Cyprian mouflon (Ovis gmelini ophion) were driven to near extinction during the 20th century and are currently listed as endangered populations by the IUCN. While the exact origins of these lineages remain unclear, they have been suggested to be close relatives of domestic sheep or remnants of proto-domestic sheep groups. Here, we study whole genome sequences of n=5 Anatolian mouflons and n=10 Cyprian mouflons in terms of population history and diversity, relative to eight other extant sheep lineages. We find reciprocal genetic affinity between Anatolian and Cyprian mouflons and domestic sheep, higher than all other studied wild sheep genomes, including the Iranian mouflon (Ovis gmelini). Despite similar recent population dynamics, Anatolian and Cyprian mouflons exhibit disparate diversity levels, which can potentially be attributed to founder effects, island isolation, introgression from domestic lineages, or different bottleneck dynamics. The lower relative mutation load found in Cyprian compared to Anatolian mouflons suggests the purging of recessive deleterious variants in the former. This agrees with estimates of a long-term small effective population size in the Cyprian mouflon. Both subspecies harbor considerable numbers of runs of homozygosity (ROH) blocks \u003c2 Mb, which reflects the effect of small population size. Expanding our analyses to worldwide wild and feral Ovis genomes, we observe varying viability metrics among different lineages, and a limited consistency between viability metrics and conservation status. Factors such as recent inbreeding, introgression, and unique population dynamics may contribute to the observed disparities.\nGene acquisition by giant transposons primes eukaryotes for rapid evolution via horizontal gene transfer\nAuthors: Urquhart, A. S.; Gluck-Thaler, E.; Vogan, A. A.\nScore: 11.5, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568313\nHorizontal gene transfer (HGT) disseminates genetic information between species. The significance of HGT in eukaryotes is not well established, with evidence currently limited to isolated examples, typically absent of a mechanism. It has been proposed that mobile elements might be active agents of HGT in eukaryotes, as they are in prokaryotes. To test this hypothesis, we examined a gene cluster, which putatively contributes to formaldehyde resistance and is found within some members of the Starship superfamily of giant transposons. Remarkably, we found four instances where ssf has been independently acquired by distantly related Starships, and show that each acquisition event coincided with the elements horizontal transfer (at least 9 HGT events in total). Our results demonstrate that acquisition of host-beneficial cargo by Starships primes fungi for rapid and repeated adaptation via HGT, revealing and elevating the role of HGT in eukaryotic biology.\nDollo parsimony overestimates ancestral gene content reconstructions\nAuthors: Morante, A. G.; Gueguen, L.; Natsidis, P.; Telford, M. J.; Richter, D. J.\nScore: 9.5, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567878\nAncestral reconstruction is a widely-used technique that has been applied to understand the history of gain and loss of gene families over evolutionary time scales, and to produce hypotheses on how these gains and losses may have influenced the evolutionary trajectories of extant organisms. Ancestral gene content can be reconstructed via different phylogenetic methods, such as maximum likelihood or Bayesian inference, but many current and previous studies employ Dollo parsimony. We hypothesize that Dollo parsimony is not appropriate for ancestral gene content reconstruction inferences based on sequence homology, as Dollo parsimony is derived from the assumption that a complex character can only be gained once in evolutionary history. This premise does not accurately model molecular sequence evolution, in which false orthology can result from sequence convergence (including random sequence similarity and parallel gene gains), non-orthologous homology or lateral gene transfer. The aim of this study is to test Dollo parsimonys suitability for ancestral gene content reconstruction and to compare its inferences with a maximum likelihood-based approach which allows a gene family to be gained more than once within a tree. In order to test our hypothesis, we first compared the performance of the two approaches on a series of artificial datasets each of 5,000 genes that were simulated according to a spectrum of evolutionary rates. Next, we reconstructed protein domain evolution on a phylogeny representing known eukaryotic diversity. We observed that Dollo parsimony produced numerous ancestral gene content overestimations, which were more pronounced at nodes closer to the root of the tree. These observations led us to the conclusion that, confirming our hypothesis, Dollo parsimony is not an appropriate method for ancestral reconstruction studies based on sequence homology.\nInduction of apoptosis by double-stranded RNA was present in the last common ancestor of cnidarian and bilaterian animals\nAuthors: Kozlovski, I.; Jaimes-Becerra, A.; Sharoni, T.; Lewandowska, M.; Karmi, O.; Moran, Y.\nScore: 20.0, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567790\nApoptosis, a major form of programmed cell death, is an essential component of host defense against invading intracellular pathogens. Viruses encode inhibitors of apoptosis to evade host responses during infection, and to support their own replication and survival. Therefore, hosts and their viruses are entangled in a constant evolutionary arms race to control apoptosis. Until now, apoptosis in the context of the antiviral immune system has been almost exclusively studied in vertebrates. This limited phyletic sampling makes it impossible to determine whether a similar mechanism existed in the last common ancestor of animals. Here, we established assays to probe apoptosis in the sea anemone Nematostella vectensis, a model species of Cnidaria, a phylum that diverged approximately 600 million years ago from the rest of animals. We show that polyinosinic:polycytidylic acid (poly I:C), a synthetic long double-stranded RNA mimicking viral RNA and a primary ligand for the vertebrate RLR melanoma differentiation-associated protein 5 (MDA5), is sufficient to induce apoptosis in N. vectensis. Furthermore, at the transcriptomic level, apoptosis related genes are significantly enriched upon poly(I:C) exposure in N. vectensis as well as bilaterian invertebrates. Our phylogenetic analysis of caspase family genes in N. vectensis reveals conservation of all four caspase genes involved in apoptosis in mammals and revealed a cnidarian-specific caspase gene which was strongly upregulated. Altogether, our findings suggest that apoptosis in response to a viral challenge is a functionally conserved mechanism that can be traced back to the last common ancestor of Bilateria and Cnidaria.\nMatreex: compact and interactive visualisation for scalable studies of large gene families\nAuthors: Rossier, V.; Train, C.; Nevers, Y.; Robinson-Rechavi, M.; Dessimoz, C.\nScore: 33.9, Published: 2023-11-15 DOI: 10.1101/2023.02.18.529053\nStudying gene family evolution strongly benefits from insightful visualisations. However, the evergrowing number of sequenced genomes is leading to increasingly larger gene families, which challenges existing gene tree visualisations. Indeed, most of them present users with a dilemma: display complete but intractable gene trees, or collapse subtrees, thereby hiding their childrens information. Here, we introduce Matreex, a new dynamic tool to scale-up the visualisation of gene families. Matreexs key idea is to use \"phylogenetic\" profiles, which are dense representations of gene repertoires, to minimise the information loss when collapsing subtrees. We illustrate Matreex usefulness with three biological applications. First, we demonstrate on the MutS family the power of combining gene trees and phylogenetic profiles to delve into precise evolutionary analyses of large multi-copy gene families. Secondly, by displaying 22 intraflagellar transport gene families across 622 species cumulating 5500 representatives, we show how Matreex can be used to automate large-scale analyses of gene presence-absence. Notably, we report for the first time the complete loss of intraflagellar transport in the myxozoan Thelohanellus kitauei. Finally, using the textbook example of visual opsins, we show Matreexs potential to create easily interpretable figures for teaching and outreach. Matreex is available from the Python Package Index (pip install matreex) with the source code and documentation available at https://github.com/DessimozLab/matreex.\n",
  "wordCount" : "2689",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568084">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568084" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568084">
        <p class="paperTitle">New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568084" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568084" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.</p>
        <p class="info">Score: 55.9, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568084' target='https://doi.org/10.1101/2023.11.21.568084'> 10.1101/2023.11.21.568084</a></p>
        <p class="abstract">Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision. The adaptive changes in sensory abilities of animals during these transitions, both at the genetic and neural levels, are largely unexplored. Butterflies and moths, with their diverse diel-niche shifts, are an ideal group for investigating the gene evolution linked to these transitions. While most butterflies are day-flying, hedylid butterflies are unique in being primarily nocturnal, and they represent an important evolutionary shift from diurnality to nocturnality in this clade. Here, we sequence the first high-quality Hedylidae genome and functionally annotate genes to understand genomic changes associated with shifts in diel niche. Comparing Hedylidae visual genes against day- and night-flying Lepidoptera species revealed that visual genes are highly conserved, with no major losses. However, hedylid butterfly opsins were more similar to nocturnal moths than their diurnal congeners. Tests on the evolutionary rates (dN/dS) confirmed that color vision opsins were under strong selection, similar to nocturnal moths. We propose that a convergent event of sequence evolution took place when these butterflies became nocturnal, approximately 98 million years ago.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564421" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564421">
        <p class="paperTitle">Frequent transitions from night-to-day activity after mass extinctions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafer, M. E. R.; Nichols, A. L. A.; Schier, A. F.; Salzburger, W.</p>
        <p class="info">Score: 34.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564421' target='https://doi.org/10.1101/2023.10.27.564421'> 10.1101/2023.10.27.564421</a></p>
        <p class="abstract">Why certain species survive large-scale extinction events while others do not is poorly understood. While the fossil record can provide insights into morphological adaptations that increase survival probabilities, it provides limited information regarding behavioural traits. Here we integrate behavioural data with phylogenetic comparative models to study the evolution of day-night activity patterns (nocturnality and diurnality) in bony fishes, which have persisted through the last four mass extinction events. Our findings in fish, combined with data across all other clades of vertebrates, provide four lines of evidence that nocturnality conferred an evolutionary advantage during mass extinctions, and that frequent nocturnal-to-diurnal transitions facilitated post-extinction diversification: First, phylogenetic reconstruction indicates that the last common ancestors of all vertebrates and of clades of bony vertebrates were nocturnal. Second, in the lineage of bony fishes, which contains over half of all vertebrate species, twice as many transitions between nocturnal and diurnal activity patterns have occurred compared to tetrapods. Third, within a specific ecological niche, different species exhibit distinct temporal activity patterns, suggesting widespread temporal niche partitioning. Fourth, independent bursts in night-to-day transitions followed large-scale extinction events during the last two geological eras in all four major bony vertebrate groups. These observations suggest that ancestral nocturnality and frequent transitions to diurnality helped vertebrates survive and diversify in the face of extinction events, such as those expected during the current &#34;6th mass extinction&#34; event caused by anthropogenic climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568631">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568631" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568631">
        <p class="paperTitle">Origin Of Evolutionary Bifurcation In An Enzyme</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568631" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568631" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miton, C. M.; Campbell, E. C.; Kaczmarski, J. A.; Feixas, F.; Romero-Rivera, A.; Sandhu, M.; Anderson, D. W.; Shatani, N.; Osuna, S.; Jackson, C. J.; Tokuriki, N.</p>
        <p class="info">Score: 17.4, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568631' target='https://doi.org/10.1101/2023.11.25.568631'> 10.1101/2023.11.25.568631</a></p>
        <p class="abstract">Evolution can lead to significantly distinct outcomes depending on the mutational path taken. Evolutionary bifurcation, in which two mutational trajectories segregate, becoming non-interchangeable over time, is the basis of diversification in all kingdoms of life. Here, we present a detailed molecular description of a bifurcation event that rapidly led to the emergence of two distinct enzymes from a common ancestor. When initiated from two starting points that differed by a single amino acid, the laboratory evolution of a phosphotriesterase (PTE) toward arylester hydrolysis resulted in different genetic and phenotypic outcomes. One trajectory led to a &gt;35,000-fold increase in activity via the reorganization of its active site to achieve exquisite enzyme-substrate complementarity. The second trajectory gave rise to an evolved variant with a [~]500-fold increase in activity, but exhibiting an alternative substrate binding mode resulting from the destabilization of an active site loop. While initial mutations tend to dictate mutational accessibility, we rather observed the gradual divergence and specialisation of each trajectory, following the emergence of distinct molecular interaction networks. Intramolecular epistasis underlay pathway bifurcation by promoting unique synergistic interactions within each trajectory, while restricting the fixation of mutation across pathways. Our results illustrate how distinct molecular outcomes can radiate from a common protein ancestor and give rise to phenotypic diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565286">
        <p class="paperTitle">Epigenetic predictors of species maximum lifespan and other life history traits in mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, C. Z.; Haghani, A.; Yan, Q.; Lu, A. T.; Zhang, J.; Fei, Z.; Ernst, J.; Yang, X.; Gladyshev, V. N.; Raj, K.; Seluanov, A.; Gorbunova, V.; Horvath, S.</p>
        <p class="info">Score: 26.8, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565286' target='https://doi.org/10.1101/2023.11.02.565286'> 10.1101/2023.11.02.565286</a></p>
        <p class="abstract">Maximum lifespan is an intrinsic characteristic of a biological species and is defined as the longest time an individual of a species has been reported to survive. By analyzing 15K samples derived from 348 mammalian species representing 25 taxonomic orders we previously identified CpG methylation sites associated with maximum lifespan. Here we present accurate DNA methylation-based (DNAm) predictors of maximum lifespan (r=0.89), average gestation time (r=0.96), and age at sexual maturity (r=0.85). Our DNAm maximum lifespan predictor indicates a potential innate longevity advantage for females over males in 17 mammalian species such as humans, red deer, and cattle. The DNAm maximum lifespan predictions do not vary significantly by caloric restriction and partial reprogramming. Genetic disruptions in the somatotropic axis, which includes growth hormone, IGF-1, and their related receptors, have an impact on DNAm maximum lifespan only in select mouse tissues. Cancer mortality rates in major mammalian orders show no correlation with our epigenetic estimates of life history traits. The DNAm maximum lifespan predictor does not detect variation in lifespan between individuals of the same species, such as between the breeds of dogs. We also present the first prototypes of accurate pan mammalian DNAm predictors of sex and tissue type.

Collectively, our findings indicate that maximum lifespan is determined, at least in part, by an epigenetic signature that is an intrinsic property of each species and is distinct from the signatures that relate to individual lifespan, which is unaffected by interventions influencing the mortality risk of individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568075">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568075" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568075">
        <p class="paperTitle">Functional and evolutionary integration of a fungal gene with a bacterial operon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568075" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568075" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, L.; David, K. T.; Wolters, J. F.; Karlen, S. D.; Goncalves, C.; Opulente, D. A.; LaBella, A. L.; Groenewald, M.; Zhou, X.; Shen, X.-X.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 14.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568075' target='https://doi.org/10.1101/2023.11.21.568075'> 10.1101/2023.11.21.568075</a></p>
        <p class="abstract">Siderophores are crucial for iron-scavenging in microorganisms. While many yeasts can uptake siderophores produced by other organisms, they are typically unable to synthesize siderophores themselves. In contrast, Wickerhamiella/Starmerella (W/S) clade yeasts gained the capacity to make the siderophore enterobactin following the remarkable horizontal acquisition of a bacterial operon enabling enterobactin synthesis. Yet, how these yeasts absorb the iron bound by enterobactin remains unresolved. Here, we demonstrate that Enb1 is the key enterobactin importer in the W/S-clade species Starmerella bombicola. Through phylogenomic analyses, we show that ENB1 is present in all W/S clade yeast species that retained the enterobactin biosynthetic genes. Conversely, it is absent in species that lost the ent genes, except for Starmerella stellata, making this species the only cheater in the W/S clade that can utilize enterobactin without producing it. Through phylogenetic analyses, we infer that ENB1 is a fungal gene that likely existed in the W/S clade prior to the acquisition of the ent genes and subsequently experienced multiple gene losses and duplications. Through phylogenetic topology tests, we show that ENB1 likely underwent horizontal gene transfer from an ancient W/S clade yeast to the order Saccharomycetales, which includes the model yeast Saccharomyces cerevisiae, followed by extensive secondary losses. Taken together, these results suggest that the fungal ENB1 and bacterial ent genes were cooperatively integrated into a functional unit within the W/S clade that enabled adaptation to iron-limited environments. This integrated fungal-bacterial circuit and its dynamic evolution determines the extant distribution of yeast enterobactin producers and cheaters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568468">
        <p class="paperTitle">Population genomic history of the endangered Anatolian and Cyprian mouflons in relation to worldwide wild, feral and domestic sheep lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atag, G.; Kaptan, D.; Yüncü, E.; Vural, K. B.; Mereu, P.; Pirastru, M.; Barbato, M.; Leoni, G. G.; Güler, M. N.; Er, T.; Eker, E.; Yazıcı, T. D.; Kılıc, M. S.; Altınısık, N. E.; Celik, E. A.; Miranda, P. M.; Dehasque, M.; Floridia, V.; Götherström, A.; Bilgin, C. C.; Togan, I.; Günther, T.; Özer, F.; Hadjisterkotis, E.; Somel, M.</p>
        <p class="info">Score: 12.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568468' target='https://doi.org/10.1101/2023.11.23.568468'> 10.1101/2023.11.23.568468</a></p>
        <p class="abstract">Once widespread in their homelands, Anatolian mouflon (Ovis gmelini anatolica) and Cyprian mouflon (Ovis gmelini ophion) were driven to near extinction during the 20th century and are currently listed as endangered populations by the IUCN. While the exact origins of these lineages remain unclear, they have been suggested to be close relatives of domestic sheep or remnants of proto-domestic sheep groups. Here, we study whole genome sequences of n=5 Anatolian mouflons and n=10 Cyprian mouflons in terms of population history and diversity, relative to eight other extant sheep lineages. We find reciprocal genetic affinity between Anatolian and Cyprian mouflons and domestic sheep, higher than all other studied wild sheep genomes, including the Iranian mouflon (Ovis gmelini). Despite similar recent population dynamics, Anatolian and Cyprian mouflons exhibit disparate diversity levels, which can potentially be attributed to founder effects, island isolation, introgression from domestic lineages, or different bottleneck dynamics. The lower relative mutation load found in Cyprian compared to Anatolian mouflons suggests the purging of recessive deleterious variants in the former. This agrees with estimates of a long-term small effective population size in the Cyprian mouflon. Both subspecies harbor considerable numbers of runs of homozygosity (ROH) blocks &lt;2 Mb, which reflects the effect of small population size. Expanding our analyses to worldwide wild and feral Ovis genomes, we observe varying viability metrics among different lineages, and a limited consistency between viability metrics and conservation status. Factors such as recent inbreeding, introgression, and unique population dynamics may contribute to the observed disparities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568313">
        <p class="paperTitle">Gene acquisition by giant transposons primes eukaryotes for rapid evolution via horizontal gene transfer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Urquhart, A. S.; Gluck-Thaler, E.; Vogan, A. A.</p>
        <p class="info">Score: 11.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568313' target='https://doi.org/10.1101/2023.11.22.568313'> 10.1101/2023.11.22.568313</a></p>
        <p class="abstract">Horizontal gene transfer (HGT) disseminates genetic information between species. The significance of HGT in eukaryotes is not well established, with evidence currently limited to isolated examples, typically absent of a mechanism. It has been proposed that mobile elements might be active agents of HGT in eukaryotes, as they are in prokaryotes. To test this hypothesis, we examined a gene cluster, which putatively contributes to formaldehyde resistance and is found within some members of the Starship superfamily of giant transposons. Remarkably, we found four instances where ssf has been independently acquired by distantly related Starships, and show that each acquisition event coincided with the elements horizontal transfer (at least 9 HGT events in total). Our results demonstrate that acquisition of host-beneficial cargo by Starships primes fungi for rapid and repeated adaptation via HGT, revealing and elevating the role of HGT in eukaryotic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567878">
        <p class="paperTitle">Dollo parsimony overestimates ancestral gene content reconstructions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morante, A. G.; Gueguen, L.; Natsidis, P.; Telford, M. J.; Richter, D. J.</p>
        <p class="info">Score: 9.5, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567878' target='https://doi.org/10.1101/2023.11.20.567878'> 10.1101/2023.11.20.567878</a></p>
        <p class="abstract">Ancestral reconstruction is a widely-used technique that has been applied to understand the history of gain and loss of gene families over evolutionary time scales, and to produce hypotheses on how these gains and losses may have influenced the evolutionary trajectories of extant organisms.

Ancestral gene content can be reconstructed via different phylogenetic methods, such as maximum likelihood or Bayesian inference, but many current and previous studies employ Dollo parsimony. We hypothesize that Dollo parsimony is not appropriate for ancestral gene content reconstruction inferences based on sequence homology, as Dollo parsimony is derived from the assumption that a complex character can only be gained once in evolutionary history. This premise does not accurately model molecular sequence evolution, in which false orthology can result from sequence convergence (including random sequence similarity and parallel gene gains), non-orthologous homology or lateral gene transfer. The aim of this study is to test Dollo parsimonys suitability for ancestral gene content reconstruction and to compare its inferences with a maximum likelihood-based approach which allows a gene family to be gained more than once within a tree.

In order to test our hypothesis, we first compared the performance of the two approaches on a series of artificial datasets each of 5,000 genes that were simulated according to a spectrum of evolutionary rates. Next, we reconstructed protein domain evolution on a phylogeny representing known eukaryotic diversity. We observed that Dollo parsimony produced numerous ancestral gene content overestimations, which were more pronounced at nodes closer to the root of the tree. These observations led us to the conclusion that, confirming our hypothesis, Dollo parsimony is not an appropriate method for ancestral reconstruction studies based on sequence homology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567790">
        <p class="paperTitle">Induction of apoptosis by double-stranded RNA was present in the last common ancestor of cnidarian and bilaterian animals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozlovski, I.; Jaimes-Becerra, A.; Sharoni, T.; Lewandowska, M.; Karmi, O.; Moran, Y.</p>
        <p class="info">Score: 20.0, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567790' target='https://doi.org/10.1101/2023.11.20.567790'> 10.1101/2023.11.20.567790</a></p>
        <p class="abstract">Apoptosis, a major form of programmed cell death, is an essential component of host defense against invading intracellular pathogens. Viruses encode inhibitors of apoptosis to evade host responses during infection, and to support their own replication and survival. Therefore, hosts and their viruses are entangled in a constant evolutionary arms race to control apoptosis. Until now, apoptosis in the context of the antiviral immune system has been almost exclusively studied in vertebrates. This limited phyletic sampling makes it impossible to determine whether a similar mechanism existed in the last common ancestor of animals. Here, we established assays to probe apoptosis in the sea anemone Nematostella vectensis, a model species of Cnidaria, a phylum that diverged approximately 600 million years ago from the rest of animals. We show that polyinosinic:polycytidylic acid (poly I:C), a synthetic long double-stranded RNA mimicking viral RNA and a primary ligand for the vertebrate RLR melanoma differentiation-associated protein 5 (MDA5), is sufficient to induce apoptosis in N. vectensis. Furthermore, at the transcriptomic level, apoptosis related genes are significantly enriched upon poly(I:C) exposure in N. vectensis as well as bilaterian invertebrates. Our phylogenetic analysis of caspase family genes in N. vectensis reveals conservation of all four caspase genes involved in apoptosis in mammals and revealed a cnidarian-specific caspase gene which was strongly upregulated. Altogether, our findings suggest that apoptosis in response to a viral challenge is a functionally conserved mechanism that can be traced back to the last common ancestor of Bilateria and Cnidaria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.18.529053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.18.529053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.18.529053">
        <p class="paperTitle">Matreex: compact and interactive visualisation for scalable studies of large gene families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.18.529053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.18.529053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossier, V.; Train, C.; Nevers, Y.; Robinson-Rechavi, M.; Dessimoz, C.</p>
        <p class="info">Score: 33.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.18.529053' target='https://doi.org/10.1101/2023.02.18.529053'> 10.1101/2023.02.18.529053</a></p>
        <p class="abstract">Studying gene family evolution strongly benefits from insightful visualisations. However, the evergrowing number of sequenced genomes is leading to increasingly larger gene families, which challenges existing gene tree visualisations. Indeed, most of them present users with a dilemma: display complete but intractable gene trees, or collapse subtrees, thereby hiding their childrens information. Here, we introduce Matreex, a new dynamic tool to scale-up the visualisation of gene families. Matreexs key idea is to use &#34;phylogenetic&#34; profiles, which are dense representations of gene repertoires, to minimise the information loss when collapsing subtrees. We illustrate Matreex usefulness with three biological applications. First, we demonstrate on the MutS family the power of combining gene trees and phylogenetic profiles to delve into precise evolutionary analyses of large multi-copy gene families. Secondly, by displaying 22 intraflagellar transport gene families across 622 species cumulating 5500 representatives, we show how Matreex can be used to automate large-scale analyses of gene presence-absence. Notably, we report for the first time the complete loss of intraflagellar transport in the myxozoan Thelohanellus kitauei. Finally, using the textbook example of visual opsins, we show Matreexs potential to create easily interpretable figures for teaching and outreach. Matreex is available from the Python Package Index (pip install matreex) with the source code and documentation available at https://github.com/DessimozLab/matreex.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
