<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.",
  "keywords": [
    
  ],
  "articleBody": " The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.\nScore: 109.6, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561165\nInfectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.\nThe amino acid sequence determines protein abundance through its conformational stability and reduced synthesis cost.\nAuthors: Buric, F.; Viknander, S.; Fu, X.; Lemke, O.; Zrimec, J.; Szyrwiel, L.; Muelleder, M.; Ralser, M.; Zelezniak, A.\nScore: 44.3, Published: 2023-10-04 DOI: 10.1101/2023.10.02.560091\nUnderstanding what drives protein abundance is essential to biology, medicine, and biotechnology. Driven by evolutionary selection, the amino acid sequence is tailored to meet the required abundance of proteomes, underscoring the intricate relationship between sequence and functional demand. Yet, the specific role of amino acid sequences in determining proteome abundance remains elusive. Here, we demonstrate that the amino acid sequence predicts abundance by shaping a proteins conformational stability. We show that increasing the abundance provides metabolic cost benefits, underscoring the evolutionary advantage of maintaining a highly abundant and stable proteome. Specifically, using a deep learning model (BERT), we predict 56% of protein abundance variation in Saccharomyces cerevisiae solely based on amino acid sequence. The model reveals latent factors linking sequence features to protein stability. To probe these relationships, we introduce MGEM (Mutation Guided by an Embedded Manifold), a methodology for guiding protein abundance through sequence modifications. We find that mutations increasing abundance significantly alter protein polarity and hydrophobicity, underscoring a connection between protein stability and abundance. Through molecular dynamics simulations and in vivo experiments in yeast, we confirm that abundance-enhancing mutations result in longer-lasting and more stable protein expression. Importantly, these sequence changes also reduce metabolic costs of protein synthesis, elucidating the evolutionary advantage of cost-effective, high-abundance, stable proteomes. Our findings support the role of amino acid sequence as a pivotal determinant of protein abundance and stability, revealing an evolutionary optimization for metabolic efficiency.\nDarwin's Overlooked Radiation: genomic evidence points to the early stages of a radiation in the Galapagos prickly pear cactus (Opuntia, Cactaceae)\nAuthors: Zapata, F.; Cerca, J.; McCarney, D.; Henriquez, C. L.; Tiamiyu, B. B.; McCormack, J.; Reckling, K. R.; Chaves, J. A.; Rivas-Torres, G.\nScore: 15.0, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560736\nThe Galapagos Islands are a prime example of a natural laboratory for the study of evolutionary radiations. While much attention has been devoted to iconic species like Darwins finches 1-4, the islands offer an equally unique but often overlooked opportunity for plant radiations 5. Yet, compared to their animal counterparts, our understanding of the patterns and processes underpinning Galapagos plant radiations remains relatively limited 6,7. We present evidence of the early stages of a radiation in prickly-pear cactus (Opuntia, Cactaceae), a plant lineage widespread across the archipelago. Phylogenomic and population genomic analyses show that notwithstanding overall low genetic differentiation across populations, there is marked geographic structure that is broadly consistent with current taxonomy and the dynamic paleogeography of the Galapagos. Because such low genetic differentiation stands in stark contrast to the exceptional eco-phenotypic diversity displayed by cacti across islands, it is plausible that phenotypic plasticity precedes genetic divergence and is the source of adaptive evolution, or that introgression between populations facilitates local adaptation. Models of population relationships including admixture indicate that gene flow is common between certain islands, likely facilitated by dispersal via animals known to feed on Opuntia flowers, fruits, and seeds across the archipelago. Scans of genetic differentiation between populations reveal candidate loci associated with seed traits and environmental stressors, suggesting that a combination of biotic interactions and abiotic pressures due to the harsh conditions characterizing island life in a volcanic, equatorial archipelago may underlie the diversification of prickly-pear cacti. Considered in concert, these results are relevant to both the mechanisms of plant eco-phenotypic differentiation and the evolutionary history and conservation of the Galapagos biota.\nDuplication and neofunctionalization of a horizontally-transferred xyloglucanase as a facet of the red queen co-evolutionary dynamic\nAuthors: Attah, V.; Milner, D. S.; Fang, Y.; Yan, X.; Leonard, G.; Heitman, J.; Talbot, N. J.; Richards, T. A.\nScore: 12.1, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561229\nOomycetes are heterotrophic protists that share phenotypic similarities with fungi, including the ability to cause plant diseases, but branch in a separate and distant region of the eukaryotic tree of life. It has been suggested that multiple horizontal gene transfers (HGTs) from fungi-to-oomycetes contributed to the evolution of plant-pathogenic traits. These HGTs are predicted to include secreted proteins that degrade plant cell walls. This is a key trait in the pathology of many oomycetes, as the plant cell wall represents a primary barrier to pathogen invasion and a rich source of carbohydrates. Many of the HGT gene families identified have undergone multiple rounds of duplication. Using a combination of phylogenomic analysis and functional assays, we investigate the diversification of a horizontally-transferred xyloglucanase gene family in the model oomycete species Phytophthora sojae. Our analyses detect 11 genes retained in P. sojae among a complex pattern of gene duplications and losses. Using a phenotype assay, based on heterologous expression in yeast, we show that eight of these paralogs have xyloglucanase function, including variants with distinct protein characteristics, such as a long-disordered C-terminal extension that can increase xyloglucanase activity. The functional xyloglucanase variants analysed subtend an ancestral node close to the fungi-oomycetes gene transfer, suggesting the horizontally-transferred gene was a bona fide xyloglucanase. Expression of xyloglucanase paralogs in Nicotiana benthamiana triggers distinct patterns of reactive oxygen species (ROS) generation, demonstrating that enzyme variants differentially stimulate pattern-triggered immunity in plants. Mass spectrometry of detectable enzymatic products demonstrates that some paralogs catalyze production of variant breakdown profiles, suggesting that secretion of multiple xyloglucanase variants increases efficiency of xyloglucan breakdown, as well as potentially diversifying the range of Damage-Associated Molecular Patterns (DAMPs) released during pathogen attack. We suggest that such patterns of protein neofunctionalization, and variant host responses, represent an aspect of the Red Queen host-pathogen co-evolutionary dynamic. Significance StatementThe oomycetes are a diverse group of eukaryotic microbes that include some of the most devastating pathogens of plants. Oomycetes perceive, invade, and colonize plants in similar ways to fungi, in part because they acquired the genes to attack and feed on plants from fungi. These genes are predicted to be useful to oomycete plant pathogens because they have undergone multiple rounds of gene duplication. One key enzyme for attacking plant cell wall structures is called xyloglucanase. Xyloglucanase in the oomycetes has undergone multiple rounds of gene duplication, leading to variants including an enzyme with a C-terminal extension that increases activity. Some xyloglucanase variants trigger unique patterns of reactive oxygen species (ROS) in planta, and generate different profiles of cell wall breakdown products - such outcomes could act to mystify and increase the workload of the plant immune system, allowing successful pathogens to proliferate.\nRapid evolution of pesticide resistance via adaptation and interspecific introgression in a major North American crop pest\nAuthors: North, H. L.; Fu, Z.; Metz, R.; Stull, M. A.; Johnson, C. D.; Shirley, X.; Crumley, K.; Reisig, D.; Kerns, D. L.; Gilligan, T.; Walsh, T.; Jiggins, C. D.; Sword, G.\nScore: 11.7, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560869\nInsect crop pests threaten global food security. This threat is amplified through the spread of non-native species and the evolution of pesticide resistance, which can be introduced to a population though de novo mutation or gene flow. We investigate these processes in an economically important noctuid crop pest, Helicoverpa zea, which has rapidly evolved resistance to several pesticides. Its sister species Helicoverpa armigera, first detected as an invasive species in Brazil in 2013, introduced the pyrethroid resistance gene CYP337B3 to South American H. zea via introgression. To understand whether this contributes to pesticide resistance in North America, we sequenced 237 H. zea genomes across 10 sample sites in the US. First, we report H. armigera introgression into the North American H. zea population. Two individuals sampled in Texas in 2019 carry H. armigera haplotypes in a 4Mbp region containing CYP337B3. Second, we show that the remarkable dispersal ability of H. zea results in a panmictic North American population. Third, we detect signatures of selection in non-admixed H. zea, identifying a selective sweep at a second pesticide resistance locus with a similar name: CYP333B3. We estimate that its derived allele conferred a [~]4.9% fitness advantage and show that this estimate explains independently observed rare nonsynonymous CYP333B3 mutations approaching fixation over a [~]20-year period. We also detect putative signatures of selection at a kinesin gene associated with Bt resistance. Our results show that pesticide resistance in H. zea evolved rapidly and recently via two independent mechanisms: interspecific introgression and rapid intraspecific adaptation.\nComplex genomic landscape of inversion polymorphism in Europe's most destructive forest pest\nAuthors: Mykhailenko, A.; Zielinski, P.; Bednarz, A.; Schlyter, F.; Andersson, M. N.; Antunes, B.; Borowski, Z.; Krokene, P.; Melin, M.; Morales-Garcia, J.; Müller, J.; Nowak, Z.; Schebeck, M.; Stauffer, C.; Viiri, H.; Babik, W.; Nadachowska-Brzyska, K.\nScore: 9.2, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561670\nIn many species, polymorphic inversions underlie complex phenotypic polymorphisms and appear to facilitate local adaptation in the face of gene flow. Multiple polymorphic inversions can co-occur in a genome, but the prevalence, evolutionary significance, and limits to complexity of genomic inversion landscapes remain poorly understood. Here, we examine genome-wide variation in one of Europe's most destructive forest pests, the spruce bark beetle Ips typographus, scan for polymorphic inversions, and test whether inversions are involved in key adaptations in this species. We sampled 244 individuals from 18 populations across the species' European range and, using a whole-genome resequencing approach, identified 27 polymorphic inversions covering at least 28% of the genome. The inversions vary in size and in levels of intra-inversion recombination, are highly polymorphic across the species range, and often overlap, forming an extremely complex genomic architecture. We show that the heterogeneous inversion landscape is likely maintained by the combined action of several evolutionary forces and that inversions are enriched in odorant receptor genes encoding key elements of recognition pathways for host plants, mates, and symbiotic fungi. Our results indicate that the genome of this major forest pest of growing social, political, and economic importance harbors one of the most complex inversion landscapes described to date and is pushing the limits of genomic architecture complexity.\nRecurrent duplication and diversification of a vital DNA repair gene family across Drosophila\nAuthors: Brand, C. L.; Oliver, G. T.; Farkas, I. Z.; Levine, M. T.\nScore: 34.4, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560779\nMaintaining genome integrity is vital for organismal survival and reproduction. Essential, broadly conserved DNA repair pathways actively preserve genome integrity. However, many DNA repair proteins evolve adaptively. Ecological forces like UV exposure are classically cited as drivers of DNA repair evolution. Intrinsic forces like repetitive DNA, which can also imperil genome integrity, have received less attention. We recently reported that a Drosophila melanogaster-specific DNA satellite array triggered species-specific, adaptive evolution of a DNA repair protein called Spartan/MH. The Spartan family of proteases cleave hazardous, covalent crosslinks that form between DNA and proteins (\"DNA-protein crosslink repair\"). Appreciating that DNA satellites are both ubiquitous and universally fast-evolving, we hypothesized that satellite DNA turnover spurs evolution of DNA-protein crosslink repair beyond D. melanogaster. This hypothesis predicts pervasive Spartan gene family diversification across the Drosophila phylogeny. To study the evolutionary history of the Drosophila Spartan gene family, we conducted population genetic, molecular evolution, phylogenomic, and tissue-specific expression analyses. We uncovered widespread signals of positive selection across multiple Spartan family genes and across multiple evolutionary timescales. We also detected recurrent Spartan family gene duplication, divergence, and gene loss. Finally, we found that ovary-enriched parent genes consistently birthed testis-enriched daughter genes. To account for Drosophila-wide, Spartan family diversification, we introduce a mechanistic model of antagonistic coevolution that links DNA satellite evolution and adaptive regulation of Spartan protease activity. This framework, combined with a recent explosion of genome assemblies that encompass repeat-rich genomic regions, promises to accelerate our understanding of how DNA repeats drive recurrent evolutionary innovation to preserve genome integrity.\nThe maintenance of genetic polymorphism in sexually antagonistic traits\nAuthors: Flintham, E.; Savolainen, V.; Otto, S.; Reuter, M.; Mullon, C.\nScore: 8.0, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561678\nSelection often favours different trait values in males and females, leading to genetic conflicts between the sexes, or sexual antagonism. Theory suggests that such conflict can maintain genetic variation by generating balancing selection. However, most of this theory is based on insights from models of single loci with fixed fitness effects. It is thus unclear how readily sexual antagonism drives balancing selection at loci where the fitness effects are not arbitrarily set but emerge from the genotype-phenotype map and the fitness landscape. Here we use mathematical models and computer simulations to investigate the evolution of traits that have different optima in males and females and so experience sexually antagonistic selection. We examine the consequences of such selection for the maintenance and nature of genetic variation under different assumptions of genetic architecture and for different fitness landscapes. We show that the conditions necessary to maintain variation are significantly more restrictive than previously appreciated. This is because the majority of typical sex-specific fitness landscapes generate stabilising selection on traits, and so favour the fixation of a single homozygote genotype that encodes the phenotype with the highest sex-averaged fitness. Instead, the maintenance of genetic polymorphism requires extremely, if not implausibly, strong conflict between males and females in order to disfavour extreme and intermediate phenotypes by generating diversifying selection. Furthermore, in all cases, we find that distinctive patterns of genetic variation produced by sexual antagonism arise rarely and are often transient, and so are difficult to observe in genomic data. Taken together, our results indicate that sex-specific selection is not a straightforward source of balanced genetic polymorphism and leaves few traces in the genome. Rather, we suggest that sexual antagonism is most likely to maintain genetic variation when traits are determined by a single large effect locus where genetic constraints lead to a limited range of possible alleles.\nTransposable element abundance subtly contributes to lower fitness in maize\nAuthors: Stitzer, M. C.; Khaipho-Burch, M. B.; Hudson, A. I.; Song, B.; Valdes-Franco, J. A.; Ramstein, G. P.; Feschotte, C.; Buckler, E. S.\nScore: 37.3, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557618\nTransposable elements (TEs) have long been shown to have deleterious effects on the survival and reproduction of their host organism. As TEs are mobile DNA that jump to new positions, this deleterious cost can occur directly, by inserting into genes and regulatory sequences. Classical population genetic theory suggests copy-number dependent selection against TEs is necessary to prevent TEs from expanding so much they take over a genome. Such models have been difficult to interpret when applied to large genomes like maize, where there are hundreds of thousands of TE insertions that collectively make up 85% of the genome. Here, we use nearly 5000 inbred lines from maize mapping populations and a pan-genomic imputation approach to measure TE content. Segregating TE content gives rise to 100 Mb differences between individuals, and populations often show transgressive segregation in TE content. We use replicated phenotypes measured in hybrids across numerous years and environments to empirically measure the fitness costs of TEs. For an annual plant like maize, grain yield is not only a key agronomic phenotype, but also a direct measure of reproductive output. We find weak negative effects of TE accumulation on grain yield, nearing the limit of the efficacy of natural selection in maize. This results in a loss of one kernel ({approx}0.1% of average per-plant yield) for every additional 14 Mb of TE content. This deleterious load is enriched in TEs within 1 kilobase of genes and young TE insertions. Together, we provide rare empirical measurements of the fitness costs of TEs, and suggest that the TEs we see today in the genome have been filtered by selection against their deleterious consequences on maize fitness.\nRelative Homoplasy Index: A New Cross-comparable Metric for Quantifying Homoplasy in Discrete Character Datasets\nAuthors: Steell, E. M.; Hsiang, A. Y.; Field, D. J.\nScore: 7.4, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561677\nHomoplasy is among the main hinderances to phylogenetic inference. However, investigating patterns of homoplasy can also improve our understanding of macroevolution, for instance by revealing evolutionary constraints on morphology, or highlighting convergent form-function relationships. Several methods have been proposed to quantify the extent of homoplasy in discrete character matrices, but the consistency index (CI) and retention index (RI) have remained the most widely used for decades, with little recent scrutiny of their function. Here, we test the performance of CI and RI using simulated and empirical datasets and investigate patterns of homoplasy with different matrix scenarios. In addition, we describe and test a new scaled metric, the relative homoplasy index (RHI), implemented in the R statistical environment. The results suggest that, unlike the RI, the CI does not constitute a direct measure of homoplasy. However, the RI consistently underestimates the extent of homoplasy in phylogenetic character-taxon matrices, particularly in datasets characterised by high levels of homoplasy. By contrast, RHI - the newly proposed metric - outperforms both methods in sensitivity to homoplasy levels, and is scaled between zero and one for comparison of values between different datasets. Using both simulated and empirical phylogenetic datasets, we show that relative levels of homoplasy remain constant with the addition of novel characters, and, in contrast to earlier work, decrease with the addition of taxa. Our results help illuminate the inherent properties of homoplasy in cladistic matrices, opening new potential avenues of research for investigating patterns of homoplasy in macroevolutionary studies.\n",
  "wordCount" : "3301",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561165">
        <p class="paperTitle">The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.</p>
        <p class="info">Score: 109.6, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561165' target='https://doi.org/10.1101/2023.10.06.561165'> 10.1101/2023.10.06.561165</a></p>
        <p class="abstract">Infectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560091" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560091">
        <p class="paperTitle">The amino acid sequence determines protein abundance through its conformational stability and reduced synthesis cost.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buric, F.; Viknander, S.; Fu, X.; Lemke, O.; Zrimec, J.; Szyrwiel, L.; Muelleder, M.; Ralser, M.; Zelezniak, A.</p>
        <p class="info">Score: 44.3, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560091' target='https://doi.org/10.1101/2023.10.02.560091'> 10.1101/2023.10.02.560091</a></p>
        <p class="abstract">Understanding what drives protein abundance is essential to biology, medicine, and biotechnology. Driven by evolutionary selection, the amino acid sequence is tailored to meet the required abundance of proteomes, underscoring the intricate relationship between sequence and functional demand. Yet, the specific role of amino acid sequences in determining proteome abundance remains elusive. Here, we demonstrate that the amino acid sequence predicts abundance by shaping a proteins conformational stability. We show that increasing the abundance provides metabolic cost benefits, underscoring the evolutionary advantage of maintaining a highly abundant and stable proteome. Specifically, using a deep learning model (BERT), we predict 56% of protein abundance variation in Saccharomyces cerevisiae solely based on amino acid sequence. The model reveals latent factors linking sequence features to protein stability. To probe these relationships, we introduce MGEM (Mutation Guided by an Embedded Manifold), a methodology for guiding protein abundance through sequence modifications. We find that mutations increasing abundance significantly alter protein polarity and hydrophobicity, underscoring a connection between protein stability and abundance. Through molecular dynamics simulations and in vivo experiments in yeast, we confirm that abundance-enhancing mutations result in longer-lasting and more stable protein expression. Importantly, these sequence changes also reduce metabolic costs of protein synthesis, elucidating the evolutionary advantage of cost-effective, high-abundance, stable proteomes. Our findings support the role of amino acid sequence as a pivotal determinant of protein abundance and stability, revealing an evolutionary optimization for metabolic efficiency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560736">
        <p class="paperTitle">Darwin&#39;s Overlooked Radiation: genomic evidence points to the early stages of a radiation in the Galapagos prickly pear cactus (Opuntia, Cactaceae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zapata, F.; Cerca, J.; McCarney, D.; Henriquez, C. L.; Tiamiyu, B. B.; McCormack, J.; Reckling, K. R.; Chaves, J. A.; Rivas-Torres, G.</p>
        <p class="info">Score: 15.0, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560736' target='https://doi.org/10.1101/2023.10.03.560736'> 10.1101/2023.10.03.560736</a></p>
        <p class="abstract">The Galapagos Islands are a prime example of a natural laboratory for the study of evolutionary radiations. While much attention has been devoted to iconic species like Darwins finches 1-4, the islands offer an equally unique but often overlooked opportunity for plant radiations 5. Yet, compared to their animal counterparts, our understanding of the patterns and processes underpinning Galapagos plant radiations remains relatively limited 6,7. We present evidence of the early stages of a radiation in prickly-pear cactus (Opuntia, Cactaceae), a plant lineage widespread across the archipelago. Phylogenomic and population genomic analyses show that notwithstanding overall low genetic differentiation across populations, there is marked geographic structure that is broadly consistent with current taxonomy and the dynamic paleogeography of the Galapagos. Because such low genetic differentiation stands in stark contrast to the exceptional eco-phenotypic diversity displayed by cacti across islands, it is plausible that phenotypic plasticity precedes genetic divergence and is the source of adaptive evolution, or that introgression between populations facilitates local adaptation. Models of population relationships including admixture indicate that gene flow is common between certain islands, likely facilitated by dispersal via animals known to feed on Opuntia flowers, fruits, and seeds across the archipelago. Scans of genetic differentiation between populations reveal candidate loci associated with seed traits and environmental stressors, suggesting that a combination of biotic interactions and abiotic pressures due to the harsh conditions characterizing island life in a volcanic, equatorial archipelago may underlie the diversification of prickly-pear cacti. Considered in concert, these results are relevant to both the mechanisms of plant eco-phenotypic differentiation and the evolutionary history and conservation of the Galapagos biota.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561229">
        <p class="paperTitle">Duplication and neofunctionalization of a horizontally-transferred xyloglucanase as a facet of the red queen co-evolutionary dynamic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Attah, V.; Milner, D. S.; Fang, Y.; Yan, X.; Leonard, G.; Heitman, J.; Talbot, N. J.; Richards, T. A.</p>
        <p class="info">Score: 12.1, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561229' target='https://doi.org/10.1101/2023.10.09.561229'> 10.1101/2023.10.09.561229</a></p>
        <p class="abstract">Oomycetes are heterotrophic protists that share phenotypic similarities with fungi, including the ability to cause plant diseases, but branch in a separate and distant region of the eukaryotic tree of life. It has been suggested that multiple horizontal gene transfers (HGTs) from fungi-to-oomycetes contributed to the evolution of plant-pathogenic traits. These HGTs are predicted to include secreted proteins that degrade plant cell walls. This is a key trait in the pathology of many oomycetes, as the plant cell wall represents a primary barrier to pathogen invasion and a rich source of carbohydrates. Many of the HGT gene families identified have undergone multiple rounds of duplication. Using a combination of phylogenomic analysis and functional assays, we investigate the diversification of a horizontally-transferred xyloglucanase gene family in the model oomycete species Phytophthora sojae. Our analyses detect 11 genes retained in P. sojae among a complex pattern of gene duplications and losses. Using a phenotype assay, based on heterologous expression in yeast, we show that eight of these paralogs have xyloglucanase function, including variants with distinct protein characteristics, such as a long-disordered C-terminal extension that can increase xyloglucanase activity. The functional xyloglucanase variants analysed subtend an ancestral node close to the fungi-oomycetes gene transfer, suggesting the horizontally-transferred gene was a bona fide xyloglucanase. Expression of xyloglucanase paralogs in Nicotiana benthamiana triggers distinct patterns of reactive oxygen species (ROS) generation, demonstrating that enzyme variants differentially stimulate pattern-triggered immunity in plants. Mass spectrometry of detectable enzymatic products demonstrates that some paralogs catalyze production of variant breakdown profiles, suggesting that secretion of multiple xyloglucanase variants increases efficiency of xyloglucan breakdown, as well as potentially diversifying the range of Damage-Associated Molecular Patterns (DAMPs) released during pathogen attack. We suggest that such patterns of protein neofunctionalization, and variant host responses, represent an aspect of the Red Queen host-pathogen co-evolutionary dynamic.

Significance StatementThe oomycetes are a diverse group of eukaryotic microbes that include some of the most devastating pathogens of plants. Oomycetes perceive, invade, and colonize plants in similar ways to fungi, in part because they acquired the genes to attack and feed on plants from fungi. These genes are predicted to be useful to oomycete plant pathogens because they have undergone multiple rounds of gene duplication. One key enzyme for attacking plant cell wall structures is called xyloglucanase. Xyloglucanase in the oomycetes has undergone multiple rounds of gene duplication, leading to variants including an enzyme with a C-terminal extension that increases activity. Some xyloglucanase variants trigger unique patterns of reactive oxygen species (ROS) in planta, and generate different profiles of cell wall breakdown products - such outcomes could act to mystify and increase the workload of the plant immune system, allowing successful pathogens to proliferate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560869">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560869" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560869">
        <p class="paperTitle">Rapid evolution of pesticide resistance via adaptation and interspecific introgression in a major North American crop pest</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560869" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560869" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: North, H. L.; Fu, Z.; Metz, R.; Stull, M. A.; Johnson, C. D.; Shirley, X.; Crumley, K.; Reisig, D.; Kerns, D. L.; Gilligan, T.; Walsh, T.; Jiggins, C. D.; Sword, G.</p>
        <p class="info">Score: 11.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560869' target='https://doi.org/10.1101/2023.10.04.560869'> 10.1101/2023.10.04.560869</a></p>
        <p class="abstract">Insect crop pests threaten global food security. This threat is amplified through the spread of non-native species and the evolution of pesticide resistance, which can be introduced to a population though de novo mutation or gene flow. We investigate these processes in an economically important noctuid crop pest, Helicoverpa zea, which has rapidly evolved resistance to several pesticides. Its sister species Helicoverpa armigera, first detected as an invasive species in Brazil in 2013, introduced the pyrethroid resistance gene CYP337B3 to South American H. zea via introgression. To understand whether this contributes to pesticide resistance in North America, we sequenced 237 H. zea genomes across 10 sample sites in the US. First, we report H. armigera introgression into the North American H. zea population. Two individuals sampled in Texas in 2019 carry H. armigera haplotypes in a 4Mbp region containing CYP337B3. Second, we show that the remarkable dispersal ability of H. zea results in a panmictic North American population. Third, we detect signatures of selection in non-admixed H. zea, identifying a selective sweep at a second pesticide resistance locus with a similar name: CYP333B3. We estimate that its derived allele conferred a [~]4.9% fitness advantage and show that this estimate explains independently observed rare nonsynonymous CYP333B3 mutations approaching fixation over a [~]20-year period. We also detect putative signatures of selection at a kinesin gene associated with Bt resistance. Our results show that pesticide resistance in H. zea evolved rapidly and recently via two independent mechanisms: interspecific introgression and rapid intraspecific adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561670">
        <p class="paperTitle">Complex genomic landscape of inversion polymorphism in Europe&#39;s most destructive forest pest</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mykhailenko, A.; Zielinski, P.; Bednarz, A.; Schlyter, F.; Andersson, M. N.; Antunes, B.; Borowski, Z.; Krokene, P.; Melin, M.; Morales-Garcia, J.; Müller, J.; Nowak, Z.; Schebeck, M.; Stauffer, C.; Viiri, H.; Babik, W.; Nadachowska-Brzyska, K.</p>
        <p class="info">Score: 9.2, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561670' target='https://doi.org/10.1101/2023.10.10.561670'> 10.1101/2023.10.10.561670</a></p>
        <p class="abstract">In many species, polymorphic inversions underlie complex phenotypic polymorphisms and appear to facilitate local adaptation in the face of gene flow. Multiple polymorphic inversions can co-occur in a genome, but the prevalence, evolutionary significance, and limits to complexity of genomic inversion landscapes remain poorly understood. Here, we examine genome-wide variation in one of Europe&#39;s most destructive forest pests, the spruce bark beetle Ips typographus, scan for polymorphic inversions, and test whether inversions are involved in key adaptations in this species. We sampled 244 individuals from 18 populations across the species&#39; European range and, using a whole-genome resequencing approach, identified 27 polymorphic inversions covering at least 28% of the genome. The inversions vary in size and in levels of intra-inversion recombination, are highly polymorphic across the species range, and often overlap, forming an extremely complex genomic architecture. We show that the heterogeneous inversion landscape is likely maintained by the combined action of several evolutionary forces and that inversions are enriched in odorant receptor genes encoding key elements of recognition pathways for host plants, mates, and symbiotic fungi. Our results indicate that the genome of this major forest pest of growing social, political, and economic importance harbors one of the most complex inversion landscapes described to date and is pushing the limits of genomic architecture complexity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560779" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560779">
        <p class="paperTitle">Recurrent duplication and diversification of a vital DNA repair gene family across Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brand, C. L.; Oliver, G. T.; Farkas, I. Z.; Levine, M. T.</p>
        <p class="info">Score: 34.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560779' target='https://doi.org/10.1101/2023.10.04.560779'> 10.1101/2023.10.04.560779</a></p>
        <p class="abstract">Maintaining genome integrity is vital for organismal survival and reproduction. Essential, broadly conserved DNA repair pathways actively preserve genome integrity. However, many DNA repair proteins evolve adaptively. Ecological forces like UV exposure are classically cited as drivers of DNA repair evolution. Intrinsic forces like repetitive DNA, which can also imperil genome integrity, have received less attention. We recently reported that a Drosophila melanogaster-specific DNA satellite array triggered species-specific, adaptive evolution of a DNA repair protein called Spartan/MH. The Spartan family of proteases cleave hazardous, covalent crosslinks that form between DNA and proteins (&#34;DNA-protein crosslink repair&#34;). Appreciating that DNA satellites are both ubiquitous and universally fast-evolving, we hypothesized that satellite DNA turnover spurs evolution of DNA-protein crosslink repair beyond D. melanogaster. This hypothesis predicts pervasive Spartan gene family diversification across the Drosophila phylogeny. To study the evolutionary history of the Drosophila Spartan gene family, we conducted population genetic, molecular evolution, phylogenomic, and tissue-specific expression analyses. We uncovered widespread signals of positive selection across multiple Spartan family genes and across multiple evolutionary timescales. We also detected recurrent Spartan family gene duplication, divergence, and gene loss. Finally, we found that ovary-enriched parent genes consistently birthed testis-enriched daughter genes. To account for Drosophila-wide, Spartan family diversification, we introduce a mechanistic model of antagonistic coevolution that links DNA satellite evolution and adaptive regulation of Spartan protease activity. This framework, combined with a recent explosion of genome assemblies that encompass repeat-rich genomic regions, promises to accelerate our understanding of how DNA repeats drive recurrent evolutionary innovation to preserve genome integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561678">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561678" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561678">
        <p class="paperTitle">The maintenance of genetic polymorphism in sexually antagonistic traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561678" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561678" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Flintham, E.; Savolainen, V.; Otto, S.; Reuter, M.; Mullon, C.</p>
        <p class="info">Score: 8.0, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561678' target='https://doi.org/10.1101/2023.10.10.561678'> 10.1101/2023.10.10.561678</a></p>
        <p class="abstract">Selection often favours different trait values in males and females, leading to genetic conflicts between the sexes, or sexual antagonism. Theory suggests that such conflict can maintain genetic variation by generating balancing selection. However, most of this theory is based on insights from models of single loci with fixed fitness effects. It is thus unclear how readily sexual antagonism drives balancing selection at loci where the fitness effects are not arbitrarily set but emerge from the genotype-phenotype map and the fitness landscape. Here we use mathematical models and computer simulations to investigate the evolution of traits that have different optima in males and females and so experience sexually antagonistic selection. We examine the consequences of such selection for the maintenance and nature of genetic variation under different assumptions of genetic architecture and for different fitness landscapes. We show that the conditions necessary to maintain variation are significantly more restrictive than previously appreciated. This is because the majority of typical sex-specific fitness landscapes generate stabilising selection on traits, and so favour the fixation of a single homozygote genotype that encodes the phenotype with the highest sex-averaged fitness. Instead, the maintenance of genetic polymorphism requires extremely, if not implausibly, strong conflict between males and females in order to disfavour extreme and intermediate phenotypes by generating diversifying selection. Furthermore, in all cases, we find that distinctive patterns of genetic variation produced by sexual antagonism arise rarely and are often transient, and so are difficult to observe in genomic data. Taken together, our results indicate that sex-specific selection is not a straightforward source of balanced genetic polymorphism and leaves few traces in the genome. Rather, we suggest that sexual antagonism is most likely to maintain genetic variation when traits are determined by a single large effect locus where genetic constraints lead to a limited range of possible alleles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557618">
        <p class="paperTitle">Transposable element abundance subtly contributes to lower fitness in maize</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stitzer, M. C.; Khaipho-Burch, M. B.; Hudson, A. I.; Song, B.; Valdes-Franco, J. A.; Ramstein, G. P.; Feschotte, C.; Buckler, E. S.</p>
        <p class="info">Score: 37.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557618' target='https://doi.org/10.1101/2023.09.18.557618'> 10.1101/2023.09.18.557618</a></p>
        <p class="abstract">Transposable elements (TEs) have long been shown to have deleterious effects on the survival and reproduction of their host organism. As TEs are mobile DNA that jump to new positions, this deleterious cost can occur directly, by inserting into genes and regulatory sequences. Classical population genetic theory suggests copy-number dependent selection against TEs is necessary to prevent TEs from expanding so much they take over a genome. Such models have been difficult to interpret when applied to large genomes like maize, where there are hundreds of thousands of TE insertions that collectively make up 85% of the genome. Here, we use nearly 5000 inbred lines from maize mapping populations and a pan-genomic imputation approach to measure TE content. Segregating TE content gives rise to 100 Mb differences between individuals, and populations often show transgressive segregation in TE content. We use replicated phenotypes measured in hybrids across numerous years and environments to empirically measure the fitness costs of TEs. For an annual plant like maize, grain yield is not only a key agronomic phenotype, but also a direct measure of reproductive output. We find weak negative effects of TE accumulation on grain yield, nearing the limit of the efficacy of natural selection in maize. This results in a loss of one kernel ({approx}0.1% of average per-plant yield) for every additional 14 Mb of TE content. This deleterious load is enriched in TEs within 1 kilobase of genes and young TE insertions. Together, we provide rare empirical measurements of the fitness costs of TEs, and suggest that the TEs we see today in the genome have been filtered by selection against their deleterious consequences on maize fitness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561677">
        <p class="paperTitle">Relative Homoplasy Index: A New Cross-comparable Metric for Quantifying Homoplasy in Discrete Character Datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steell, E. M.; Hsiang, A. Y.; Field, D. J.</p>
        <p class="info">Score: 7.4, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561677' target='https://doi.org/10.1101/2023.10.10.561677'> 10.1101/2023.10.10.561677</a></p>
        <p class="abstract">Homoplasy is among the main hinderances to phylogenetic inference. However, investigating patterns of homoplasy can also improve our understanding of macroevolution, for instance by revealing evolutionary constraints on morphology, or highlighting convergent form-function relationships. Several methods have been proposed to quantify the extent of homoplasy in discrete character matrices, but the consistency index (CI) and retention index (RI) have remained the most widely used for decades, with little recent scrutiny of their function. Here, we test the performance of CI and RI using simulated and empirical datasets and investigate patterns of homoplasy with different matrix scenarios. In addition, we describe and test a new scaled metric, the relative homoplasy index (RHI), implemented in the R statistical environment. The results suggest that, unlike the RI, the CI does not constitute a direct measure of homoplasy. However, the RI consistently underestimates the extent of homoplasy in phylogenetic character-taxon matrices, particularly in datasets characterised by high levels of homoplasy. By contrast, RHI - the newly proposed metric - outperforms both methods in sensitivity to homoplasy levels, and is scaled between zero and one for comparison of values between different datasets. Using both simulated and empirical phylogenetic datasets, we show that relative levels of homoplasy remain constant with the addition of novel characters, and, in contrast to earlier work, decrease with the addition of taxa. Our results help illuminate the inherent properties of homoplasy in cladistic matrices, opening new potential avenues of research for investigating patterns of homoplasy in macroevolutionary studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
