<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.",
  "keywords": [
    
  ],
  "articleBody": " Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.\nScore: 241.1, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547326\nThe evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.\nThe recency and geographical origins of the bat viruses ancestral to SARS-CoV and SARS-CoV-2\nAuthors: Pekar, J. E.; Lytras, S.; Ghafari, M.; Magee, A. F.; Parker, E.; Havens, J. L.; Katzourakis, A.; Vasylyeva, T. I.; Suchard, M. A.; Hughes, A. C.; Hughes, J.; Robertson, D. L.; Dellicour, S.; Worobey, M.; Wertheim, J. O.; Lemey, P.\nScore: 104.5, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548617\nThe emergence of SARS-CoV in 2002 and SARS-CoV-2 in 2019 has led to increased sampling of related sarbecoviruses circulating primarily in horseshoe bats. These viruses undergo frequent recombination and exhibit spatial structuring across Asia. Employing recombination-aware phylogenetic inference on bat sarbecoviruses, we find that the closest-inferred bat virus ancestors of SARS-CoV and SARS-CoV-2 existed just [~]1-3 years prior to their emergence in humans. Phylogeographic analyses examining the movement of related sarbecoviruses demonstrate that they traveled at similar rates to their horseshoe bat hosts and have been circulating for thousands of years in Asia. The closest-inferred bat virus ancestor of SARS-CoV likely circulated in western China, and that of SARS-CoV-2 likely circulated in a region comprising southwest China and northern Laos, both a substantial distance from where they emerged. This distance and recency indicate that the direct ancestors of SARS-CoV and SARS-CoV-2 could not have reached their respective sites of emergence via the bat reservoir alone. Our recombination-aware dating and phylogeographic analyses reveal a more accurate inference of evolutionary history than performing only whole-genome or single gene analyses. These results can guide future sampling efforts and demonstrate that viral genomic fragments extremely closely related to SARS-CoV and SARS-CoV-2 were circulating in horseshoe bats, confirming their importance as the reservoir species for SARS viruses.\nClockor2: Inferring global and local strict molecular clocks using root-to-tip regression\nAuthors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.\nScore: 89.6, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548947\nMolecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion. Here, we introduce Clockor2, a client-side web application for conducting RTT regression. Clockor2 uniquely allows users to quickly fit local and global molecular clocks, thus handling the increasing complexity of genomic datasets that sample beyond the assumption homogeneous host populations. Clockor2 is efficient, handling trees of up to the order of $10^4$ tips, with significant speed increases compared to other RTT regression applications. Although clockor2 is written as a web application, all data processing happens on the client-side, meaning that data never leaves the user's computer. Clockor2 is freely available at https://clockor2.github.io/ .\nPatterns of recombination in snakes reveal a tug of war between PRDM9 and promoter-like features\nAuthors: Hoge, C. R.; de Manuel, M.; Mahgoub, M.; Okami, N.; Fuller, Z. L.; Banerjee, S.; Baker, Z.; Mcnulty, M.; Andolfatto, P.; Macfarlan, T. S.; Schumer, M.; Tzika, A. C.; Przeworski, M.\nScore: 49.1, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548536\nIn vertebrates, there are two known mechanisms by which meiotic recombination is directed to the genome: in humans, mice, and other mammals, recombination occurs almost exclusively where the protein PRDM9 binds, while in species lacking an intact PRDM9, such as birds and canids, recombination rates are elevated near promoter-like features. To test if PRDM9 also directs recombination in non-mammalian vertebrates, we focused on an exemplar species, the corn snake (Pantherophis guttatus). Unlike birds, this species possesses a single, intact PRDM9 ortholog. By inferring historical recombination rates along the genome from patterns of linkage disequilibrium and identifying crossovers in pedigrees, we found that PRDM9 specifies the location of recombination events outside of mammals. However, we also detected an independent effect of promoter-like features on recombination, which is more pronounced on macrothan microchromosomes. Thus, our findings reveal that the uses of PRDM9 and promoter-like features are not mutually-exclusive, and instead reflect a tug of war, which varies in strength along the genome and is more lopsided in some species than others. One sentence summaryWhile the localization of meiotic recombination in vertebrates was previously thought to occur using one of two distinct mechanisms, our analysis of recombination in corn snakes reveals that they and likely other vertebrates use both of these mechanisms.\nThe contribution of gene flow, selection, and genetic drift to five thousand years of human allele frequency change\nAuthors: Simon, A.; Coop, G.\nScore: 47.4, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548607\nGenomic time-series from experimental evolution studies and ancient DNA datasets offer us a chance to more directly observe the interplay of various evolutionary forces. Here we show how the genome-wide variance in allele frequency change between two time points can be decomposed into the contributions of gene flow, genetic drift, and linked selection. In closed populations, the contribution of linked selection is identifiable because it creates covariances between time intervals, and genetic drift does not. However, repeated gene flow between populations can also produce directionality in allele frequency change, creating covariances. We show how to accurately separate the fraction of variance in allele frequency change due to admixture and linked selection in a population receiving gene flow. We use two human ancient DNA datasets, spanning around 5,000 years, as time transects to quantify the contributions to the genome-wide variance in allele frequency change. We find that a large fraction of genome-wide allele frequency change is due to gene flow. In both cases, after correcting for known major gene flow events in those populations, we do not observe a signal of genome-wide linked selection. Thus despite the known role of selection in shaping long-term polymorphism levels, and an increasing number of examples of strong selection on single loci and polygenic scores from ancient DNA, it appears to be gene flow and drift, and not selection, that are the main determinants of recent genome-wide allele frequency change. Our approach should be applicable to the growing number of contemporary and ancient temporal population genomics datasets.\nAdaptive introgression of a visual preference gene\nAuthors: Rossi, M.; Hausmann, A. H.; Alcami, P.; Moest, M. H.; Wright, D. S.; Kuo, C.-Y.; Lozano, D.; Maulana, A.; Melo-Florez, L.; Rueda-Munoz, G.; McMahon, S.; Linares, M.; McMillan, W. O.; Pardo-Diaz, C.; Salazar, C.; Merrill, R. M.\nScore: 29.4, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548653\nVisual preferences are important drivers of mate choice and sexual selection, but little is known of how they evolve at the genetic level. Here we take advantage of the diversity of bright warning patterns displayed by Heliconius butterflies, which are also used during mate choice. We show that two Heliconius species have evolved the same visual mating preferences for females with red patterns by exchanging genetic material through hybridization. Extensive behavioral experiments reveal that male preferences are associated with a genomic region of increased admixture between these two species. Variation in neural expression of regucalcin1, located within this introgressed region, correlates with visual preference across populations, and disruption of regucalcin1 with CRISPR/Cas9 impairs courtship towards conspecific females, proving a direct link between gene and behavior. Our results support a role for hybridization during behavioral evolution, and show how visually-guided behaviors contributing to adaptation and speciation are encoded within the genome.\nWhole-genomes from the extinct Xerces Blue butterfly can help identify declining insect species\nAuthors: de-Dios, T.; Fontsere, C.; Renom, P.; Stiller, J.; Llovera, L.; Uliano-Silva, M.; Sanchez-Gracia, A.; Wright, C.; Lizano, E.; Navarro, A.; Civit, S.; Robbins, R. K.; Blaxter, M.; Marques-Bonet, T.; Vila, R.; Lalueza-Fox, C.\nScore: 104.5, Published: 2023-07-14 DOI: 10.1101/2021.11.08.467457\nThe Xerces Blue (Glaucopsyche xerces) is considered to be the first butterfly to become extinct at global scale in historical times. It was notable for its chalky lavender wings with conspicuous white spots on the ventral wings. The last individuals were collected in their restricted habitat, in the dunes near the Presidio military base in San Francisco, in 1941. We sequenced the genomes of four 80 to 100-year-old Xerces Blue, and seven historical and one modern specimens of its closest relative, the Silvery Blue (G. lygdamus). We compared these to a novel annotated genome of the Green-Underside Blue (G. alexis). Phylogenetic relationships inferred from complete mitochondrial genomes indicate that Xerces Blue was a distinct species that diverged from the Silvery Blue lineage at least 850,000 years ago. Using nuclear genomes, both species experienced population growth during the Eemian interglacial period, but the Xerces Blue decreased to a very low effective population size subsequently, a trend opposite to that observed in the Silvery Blue. Runs of homozygosity and deleterious load in the Xerces Blue were significantly greater than in the Silvery Blue, suggesting a higher incidence of inbreeding. These signals of population decline observed in Xerces Blue could be used to identify and monitor other insects threatened by human activities, whose extinction patterns are still not well known.\nEvaluating evidence for co-geography in the Anopheles Plasmodium host-parasite system\nAuthors: Rehmann, C. T.; Ralph, P. L.; Kern, A. D.\nScore: 25.4, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549405\nThe often tight association between parasites and their hosts means that under certain scenarios, the evolutionary histories of the two species can become closely coupled. Using spatial genetic inference, we identify a potential signal of correlated dispersal in the Anopheles gambiae and Plasmodium falciparum host-parasite system in Sub-Saharan Africa, as seen through a between-species correlation of the differences between geographic sampling location and geographic location predicted from the genome. An explanation for this correlation is common patterns of dispersal and gene flow between these species, but we also consider possible statistical artifacts due to uneven spatial distribution of sampling locations. Using continuous-space population genetics simulations, we investigate the degree to which uneven distribution of sampling locations leads to bias in prediction of spatial location from genetic data and implement methods to counter this effect. We demonstrate that while algorithmic bias presents a problem in inference from spatiogenetic data, the correlation structure between A. gambiae and P. falciparum predictions cannot be attributed to spatial bias alone, and is thus likely a genetic signal of co-dispersal in a host-parasite system.\nEvolution of Evolvability In Rapidly Adapting Populations\nAuthors: Ferrare, J. T.; Good, B. H.\nScore: 24.1, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548717\nMutations can alter the short-term fitness of an organism, as well as the rates and benefits of future mutations. While numerous examples of these evolvability modifiers have been observed in rapidly adapting microbial populations, existing theory struggles to predict when they will be favored by natural selection. Here, we develop a mathematical framework for predicting the fates of new mutations that modify the rates and benefits of future mutations in linked genomic regions. We derive analytical expressions showing how the fixation probabilities of these variants depend on the size of the population and the diversity of competing mutations. We find that competition between linked mutations can dramatically enhance selection for modifiers that increase the benefits of future mutations, even when they impose a strong direct cost on fitness. Conversely, we find that relatively modest direct benefits can be sufficient to drive an evolutionary dead-end to fixation. Our results suggest that subtle differences in evolvability could play an important role in shaping the long-term success of genetic variants in rapidly evolving microbial populations.\nUsing big sequencing data to identify chronic SARS-Coronavirus-2 infections\nAuthors: Harari, S.; Miller, D.; Fleishon, S.; Burstein, D.; Stern, A.\nScore: 21.2, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549184\nThe evolution of SARS-Coronavirus-2 (SARS-CoV-2) has been characterized by the periodic emergence of highly divergent variants, many of which may have arisen during chronic infections of immunocompromised individuals. Here, we harness a global phylogeny of ~11.7 million SARS-CoV-2 genomes and search for clades composed of sequences with identical metadata (location, age, and sex) spanning more than 21 days. We postulate that such clades represent repeated sampling from the same chronically infected individual. A set of 271 such chronic-like clades was inferred, and displayed signatures of an elevated rate of adaptive evolution, in line with validated chronic infections. More than 70% of adaptive mutations present in currently circulating variants are found in BA.1 chronic-like clades that predate the circulating variants by months, demonstrating the predictive nature of such clades. We find that in chronic-like clades the probability of observing adaptive mutations is approximately 10-20 higher than that in global transmission chains. We next employ language models to find mutations most predictive of chronic infections and use them to infer hundreds of additional chronic-like clades in the absence of metadata and phylogenetic information. Our proposed approach presents an innovative method for mining extensive sequencing data and providing valuable insights into future evolutionary patterns.\n",
  "wordCount" : "2417",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547326">
        <p class="paperTitle">Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.</p>
        <p class="info">Score: 241.1, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547326' target='https://doi.org/10.1101/2023.07.03.547326'> 10.1101/2023.07.03.547326</a></p>
        <p class="abstract">The evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548617">
        <p class="paperTitle">The recency and geographical origins of the bat viruses ancestral to SARS-CoV and SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pekar, J. E.; Lytras, S.; Ghafari, M.; Magee, A. F.; Parker, E.; Havens, J. L.; Katzourakis, A.; Vasylyeva, T. I.; Suchard, M. A.; Hughes, A. C.; Hughes, J.; Robertson, D. L.; Dellicour, S.; Worobey, M.; Wertheim, J. O.; Lemey, P.</p>
        <p class="info">Score: 104.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548617' target='https://doi.org/10.1101/2023.07.12.548617'> 10.1101/2023.07.12.548617</a></p>
        <p class="abstract">The emergence of SARS-CoV in 2002 and SARS-CoV-2 in 2019 has led to increased sampling of related sarbecoviruses circulating primarily in horseshoe bats. These viruses undergo frequent recombination and exhibit spatial structuring across Asia. Employing recombination-aware phylogenetic inference on bat sarbecoviruses, we find that the closest-inferred bat virus ancestors of SARS-CoV and SARS-CoV-2 existed just [~]1-3 years prior to their emergence in humans. Phylogeographic analyses examining the movement of related sarbecoviruses demonstrate that they traveled at similar rates to their horseshoe bat hosts and have been circulating for thousands of years in Asia. The closest-inferred bat virus ancestor of SARS-CoV likely circulated in western China, and that of SARS-CoV-2 likely circulated in a region comprising southwest China and northern Laos, both a substantial distance from where they emerged. This distance and recency indicate that the direct ancestors of SARS-CoV and SARS-CoV-2 could not have reached their respective sites of emergence via the bat reservoir alone. Our recombination-aware dating and phylogeographic analyses reveal a more accurate inference of evolutionary history than performing only whole-genome or single gene analyses. These results can guide future sampling efforts and demonstrate that viral genomic fragments extremely closely related to SARS-CoV and SARS-CoV-2 were circulating in horseshoe bats, confirming their importance as the reservoir species for SARS viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548947">
        <p class="paperTitle">Clockor2: Inferring global and local strict molecular clocks using root-to-tip regression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Featherstone, L. A.; Rambaut, A.; Duchene, S.; Wirth, W.</p>
        <p class="info">Score: 89.6, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548947' target='https://doi.org/10.1101/2023.07.13.548947'> 10.1101/2023.07.13.548947</a></p>
        <p class="abstract">Molecular sequence data from rapidly evolving organisms are often sampled at different points in time. Sampling times can then be used for molecular clock calibration. The root-to-tip (RTT) regression is an essential tool to assess the degree to which the data behave in a clock-like fashion. Here, we introduce Clockor2, a client-side web application for conducting RTT regression. Clockor2 uniquely allows users to quickly fit local and global molecular clocks, thus handling the increasing complexity of genomic datasets that sample beyond the assumption homogeneous host populations. Clockor2 is efficient, handling trees of up to the order of $10^4$ tips, with significant speed increases compared to other RTT regression applications. Although clockor2 is written as a web application, all data processing happens on the client-side, meaning that data never leaves the user&#39;s computer. Clockor2 is freely available at https://clockor2.github.io/ .</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548536" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548536">
        <p class="paperTitle">Patterns of recombination in snakes reveal a tug of war between PRDM9 and promoter-like features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoge, C. R.; de Manuel, M.; Mahgoub, M.; Okami, N.; Fuller, Z. L.; Banerjee, S.; Baker, Z.; Mcnulty, M.; Andolfatto, P.; Macfarlan, T. S.; Schumer, M.; Tzika, A. C.; Przeworski, M.</p>
        <p class="info">Score: 49.1, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548536' target='https://doi.org/10.1101/2023.07.11.548536'> 10.1101/2023.07.11.548536</a></p>
        <p class="abstract">In vertebrates, there are two known mechanisms by which meiotic recombination is directed to the genome: in humans, mice, and other mammals, recombination occurs almost exclusively where the protein PRDM9 binds, while in species lacking an intact PRDM9, such as birds and canids, recombination rates are elevated near promoter-like features. To test if PRDM9 also directs recombination in non-mammalian vertebrates, we focused on an exemplar species, the corn snake (Pantherophis guttatus). Unlike birds, this species possesses a single, intact PRDM9 ortholog. By inferring historical recombination rates along the genome from patterns of linkage disequilibrium and identifying crossovers in pedigrees, we found that PRDM9 specifies the location of recombination events outside of mammals. However, we also detected an independent effect of promoter-like features on recombination, which is more pronounced on macrothan microchromosomes. Thus, our findings reveal that the uses of PRDM9 and promoter-like features are not mutually-exclusive, and instead reflect a tug of war, which varies in strength along the genome and is more lopsided in some species than others.

One sentence summaryWhile the localization of meiotic recombination in vertebrates was previously thought to occur using one of two distinct mechanisms, our analysis of recombination in corn snakes reveals that they and likely other vertebrates use both of these mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548607" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548607">
        <p class="paperTitle">The contribution of gene flow, selection, and genetic drift to five thousand years of human allele frequency change</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simon, A.; Coop, G.</p>
        <p class="info">Score: 47.4, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548607' target='https://doi.org/10.1101/2023.07.11.548607'> 10.1101/2023.07.11.548607</a></p>
        <p class="abstract">Genomic time-series from experimental evolution studies and ancient DNA datasets offer us a chance to more directly observe the interplay of various evolutionary forces. Here we show how the genome-wide variance in allele frequency change between two time points can be decomposed into the contributions of gene flow, genetic drift, and linked selection. In closed populations, the contribution of linked selection is identifiable because it creates covariances between time intervals, and genetic drift does not. However, repeated gene flow between populations can also produce directionality in allele frequency change, creating covariances. We show how to accurately separate the fraction of variance in allele frequency change due to admixture and linked selection in a population receiving gene flow. We use two human ancient DNA datasets, spanning around 5,000 years, as time transects to quantify the contributions to the genome-wide variance in allele frequency change. We find that a large fraction of genome-wide allele frequency change is due to gene flow. In both cases, after correcting for known major gene flow events in those populations, we do not observe a signal of genome-wide linked selection. Thus despite the known role of selection in shaping long-term polymorphism levels, and an increasing number of examples of strong selection on single loci and polygenic scores from ancient DNA, it appears to be gene flow and drift, and not selection, that are the main determinants of recent genome-wide allele frequency change. Our approach should be applicable to the growing number of contemporary and ancient temporal population genomics datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548653">
        <p class="paperTitle">Adaptive introgression of a visual preference gene</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossi, M.; Hausmann, A. H.; Alcami, P.; Moest, M. H.; Wright, D. S.; Kuo, C.-Y.; Lozano, D.; Maulana, A.; Melo-Florez, L.; Rueda-Munoz, G.; McMahon, S.; Linares, M.; McMillan, W. O.; Pardo-Diaz, C.; Salazar, C.; Merrill, R. M.</p>
        <p class="info">Score: 29.4, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548653' target='https://doi.org/10.1101/2023.07.12.548653'> 10.1101/2023.07.12.548653</a></p>
        <p class="abstract">Visual preferences are important drivers of mate choice and sexual selection, but little is known of how they evolve at the genetic level. Here we take advantage of the diversity of bright warning patterns displayed by Heliconius butterflies, which are also used during mate choice. We show that two Heliconius species have evolved the same visual mating preferences for females with red patterns by exchanging genetic material through hybridization. Extensive behavioral experiments reveal that male preferences are associated with a genomic region of increased admixture between these two species. Variation in neural expression of regucalcin1, located within this introgressed region, correlates with visual preference across populations, and disruption of regucalcin1 with CRISPR/Cas9 impairs courtship towards conspecific females, proving a direct link between gene and behavior. Our results support a role for hybridization during behavioral evolution, and show how visually-guided behaviors contributing to adaptation and speciation are encoded within the genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.11.08.467457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.11.08.467457" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.11.08.467457">
        <p class="paperTitle">Whole-genomes from the extinct Xerces Blue butterfly can help identify declining insect species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.11.08.467457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.11.08.467457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de-Dios, T.; Fontsere, C.; Renom, P.; Stiller, J.; Llovera, L.; Uliano-Silva, M.; Sanchez-Gracia, A.; Wright, C.; Lizano, E.; Navarro, A.; Civit, S.; Robbins, R. K.; Blaxter, M.; Marques-Bonet, T.; Vila, R.; Lalueza-Fox, C.</p>
        <p class="info">Score: 104.5, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.11.08.467457' target='https://doi.org/10.1101/2021.11.08.467457'> 10.1101/2021.11.08.467457</a></p>
        <p class="abstract">The Xerces Blue (Glaucopsyche xerces) is considered to be the first butterfly to become extinct at global scale in historical times. It was notable for its chalky lavender wings with conspicuous white spots on the ventral wings. The last individuals were collected in their restricted habitat, in the dunes near the Presidio military base in San Francisco, in 1941. We sequenced the genomes of four 80 to 100-year-old Xerces Blue, and seven historical and one modern specimens of its closest relative, the Silvery Blue (G. lygdamus). We compared these to a novel annotated genome of the Green-Underside Blue (G. alexis). Phylogenetic relationships inferred from complete mitochondrial genomes indicate that Xerces Blue was a distinct species that diverged from the Silvery Blue lineage at least 850,000 years ago. Using nuclear genomes, both species experienced population growth during the Eemian interglacial period, but the Xerces Blue decreased to a very low effective population size subsequently, a trend opposite to that observed in the Silvery Blue. Runs of homozygosity and deleterious load in the Xerces Blue were significantly greater than in the Silvery Blue, suggesting a higher incidence of inbreeding. These signals of population decline observed in Xerces Blue could be used to identify and monitor other insects threatened by human activities, whose extinction patterns are still not well known.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549405">
        <p class="paperTitle">Evaluating evidence for co-geography in the Anopheles Plasmodium host-parasite system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rehmann, C. T.; Ralph, P. L.; Kern, A. D.</p>
        <p class="info">Score: 25.4, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549405' target='https://doi.org/10.1101/2023.07.17.549405'> 10.1101/2023.07.17.549405</a></p>
        <p class="abstract">The often tight association between parasites and their hosts means that under certain scenarios, the evolutionary histories of the two species can become closely coupled. Using spatial genetic inference, we identify a potential signal of correlated dispersal in the Anopheles gambiae and Plasmodium falciparum host-parasite system in Sub-Saharan Africa, as seen through a between-species correlation of the differences between geographic sampling location and geographic location predicted from the genome. An explanation for this correlation is common patterns of dispersal and gene flow between these species, but we also consider possible statistical artifacts due to uneven spatial distribution of sampling locations. Using continuous-space population genetics simulations, we investigate the degree to which uneven distribution of sampling locations leads to bias in prediction of spatial location from genetic data and implement methods to counter this effect. We demonstrate that while algorithmic bias presents a problem in inference from spatiogenetic data, the correlation structure between A. gambiae and P. falciparum predictions cannot be attributed to spatial bias alone, and is thus likely a genetic signal of co-dispersal in a host-parasite system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548717">
        <p class="paperTitle">Evolution of Evolvability In Rapidly Adapting Populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferrare, J. T.; Good, B. H.</p>
        <p class="info">Score: 24.1, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548717' target='https://doi.org/10.1101/2023.07.12.548717'> 10.1101/2023.07.12.548717</a></p>
        <p class="abstract">Mutations can alter the short-term fitness of an organism, as well as the rates and benefits of future mutations. While numerous examples of these evolvability modifiers have been observed in rapidly adapting microbial populations, existing theory struggles to predict when they will be favored by natural selection. Here, we develop a mathematical framework for predicting the fates of new mutations that modify the rates and benefits of future mutations in linked genomic regions. We derive analytical expressions showing how the fixation probabilities of these variants depend on the size of the population and the diversity of competing mutations. We find that competition between linked mutations can dramatically enhance selection for modifiers that increase the benefits of future mutations, even when they impose a strong direct cost on fitness. Conversely, we find that relatively modest direct benefits can be sufficient to drive an evolutionary dead-end to fixation. Our results suggest that subtle differences in evolvability could play an important role in shaping the long-term success of genetic variants in rapidly evolving microbial populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549184">
        <p class="paperTitle">Using big sequencing data to identify chronic SARS-Coronavirus-2 infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harari, S.; Miller, D.; Fleishon, S.; Burstein, D.; Stern, A.</p>
        <p class="info">Score: 21.2, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549184' target='https://doi.org/10.1101/2023.07.16.549184'> 10.1101/2023.07.16.549184</a></p>
        <p class="abstract">The evolution of SARS-Coronavirus-2 (SARS-CoV-2) has been characterized by the periodic emergence of highly divergent variants, many of which may have arisen during chronic infections of immunocompromised individuals. Here, we harness a global phylogeny of ~11.7 million SARS-CoV-2 genomes and search for clades composed of sequences with identical metadata (location, age, and sex) spanning more than 21 days. We postulate that such clades represent repeated sampling from the same chronically infected individual. A set of 271 such chronic-like clades was inferred, and displayed signatures of an elevated rate of adaptive evolution, in line with validated chronic infections. More than 70% of adaptive mutations present in currently circulating variants are found in BA.1 chronic-like clades that predate the circulating variants by months, demonstrating the predictive nature of such clades. We find that in chronic-like clades the probability of observing adaptive mutations is approximately 10-20 higher than that in global transmission chains. We next employ language models to find mutations most predictive of chronic infections and use them to infer hundreds of additional chronic-like clades in the absence of metadata and phylogenetic information. Our proposed approach presents an innovative method for mining extensive sequencing data and providing valuable insights into future evolutionary patterns.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
