<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event
Authors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.
Score: 32.0, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427
Most of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event
Authors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.
Score: 32.0, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427
Most of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event
Authors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.
Score: 32.0, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427
Most of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event\nAuthors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.\nScore: 32.0, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427\nMost of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time.",
  "keywords": [
    
  ],
  "articleBody": " An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event\nAuthors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.\nScore: 32.0, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427\nMost of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time. However, the co-evolution of life and the Earth system has left signatures of bacterial metabolism in the geochemical record, most conspicuously the Great Oxidation Event (GOE) [~]2.33 billion years ago (Ga, (Poulton et al. 2021)), in which oxygenic photosynthesis and tectonism (Eguchi, Seales, and Dasgupta 2019) transformed Earths biosphere from dominantly anaerobic to aerobic. Here, we combine machine learning and phylogenetic reconciliation to infer ancestral transitions to aerobic lifestyles during bacterial evolution. Linking these transitions to the GOE provides new constraints to infer the timetree of Bacteria. We find that extant bacterial phyla are truly ancient, having radiated in the Archaean and the Proterozoic: the oldest include Bacillota (Firmicutes), which radiated 3.1-3.7 Ga, Cyanobacteria (3.3-3.5 Ga) and Patescibacteria (3-3.5 Ga). We show that most bacterial phyla were ancestrally anaerobic and that most transitions to an aerobic lifestyle post-dated the GOE. Our analyses trace oxygen production and consumption back to Cyanobacteria. From that starting point, horizontal transfer seeded aerobic lifestyles across bacterial diversity over hundreds of millions of years. Our analyses demonstrate that the diversification of aerobes proceeded in two waves corresponding to the GOE and to a second sustained rise in atmospheric O2 at the dawn of the Palezoic (Krause et al. 2022).\nA conserved ribosomal protein has entirely dissimilar structures in different organisms\nAuthors: Schierholz, L.; Brown, C. R.; Helena-Bueno, K.; Uversky, V. N.; Hirt, R.; Barandun, J.; Melnikov, S. V.\nScore: 34.5, Published: 2023-08-09 DOI: 10.1101/2022.09.21.508910\nRibosomes from different species can markedly differ in their composition by including dozens of ribosomal proteins that are unique to specific lineages but absent in others. However, it remains unknown how ribosomes acquire and specialize new proteins throughout evolution. Here, to help answer this question, we describe the evolution of the ribosomal protein msL1/msL2 that was recently found in ribosomes from the parasitic microorganism clade, microsporidia. We first show that this protein has a conserved location in the ribosome but entirely dissimilar structures in different organisms: in each of the analyzed species, msL1/msL2 exhibits an altered secondary structure, an inverted orientation of the N- and C-termini on the ribosomal binding surface, and a completely transformed three-dimensional fold. We then show that this evolutionary fold switching is likely caused by changes in the msL1/msL2-binding site in the ribosome; specifically, by variations in microsporidian rRNA. These observations allow us to infer an evolutionary scenario in which a small, positively-charged, de novo-born unfolded protein was first captured by rRNA to become part of the ribosome and subsequently underwent complete fold switching to optimize its binding to its evolving ribosomal binding site. Overall, our work provides a striking example that will help us better understand the origin and evolution of new protein components of complex molecular assemblies - thereby enhancing our ability to engineer biological molecules, identify protein homologs, and peer into the history of life on Earth.\nFruitless mating with the exes: the irreversible parthenogenesis in a stick insect\nAuthors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.\nScore: 227.1, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994\nParthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.\nTetrapod vocal evolution: higher frequencies and faster rates of evolution in mammalian vocalizations\nAuthors: Munoz, M. I.; Marsot, M.; Ellers, J.; Halfwerk, W.\nScore: 25.0, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552622\nUsing the voice to produce sound is a widespread form of communication and plays an important role in contexts as diverse as parent-offspring interactions and courtship. Variation in the tempo and mode of vocal signal evolution has been studied in a phylogenetic context within orders or classes, but understanding vocal signal evolution ultimately requires comparison across all major lineages involved. Here we used comparative analyses to investigate the evolution of dominant frequency (i.e., the frequency with the highest energy content) and its association with body weight across 873 species of mammals, birds and frogs. In agreement with previous studies, we found that the negative allometric relationship between body weight and vocal frequency is a general feature of vocal systems. In addition, we found mammals to consistently vocalize at higher frequencies, and evolved their vocalizations at around 6-fold faster rates than those of birds and frogs. Although all three groups strongly rely on vocal communication, our findings show that only mammals have extensively explored the spectral acoustic space. We argue that such high vocal diversity of mammals is made possible by their unique hearing system, which evolved in the context of a small, parental-caring, nocturnal and insectivore ancestor, and has allowed them to detect, and therefore to evolve, a richer array of frequencies than other tetrapods.\nEvidence for ecological tuning of novel anuran biofluorescent signals\nAuthors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.\nScore: 17.6, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432\nOur study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians. We more than tripled the number of species that have been tested for this trait and added representatives from previously untested anuran families. We found evidence for ecological tuning (i.e., the specific adaptation of a signal to the environment in which it is received) of the novel anuran biofluorescent signals. Across groups, the fluorescence excitation peak matches the wavelengths most available at twilight, the light environment in which most frog species are active. Additionally, biofluorescence emission spans both wavelengths of low availability in twilight and the peak sensitivity of green-sensitive rods in the anuran eye, likely increasing contrast of this signal for a conspecific receiver. With evidence of tuning to the ecology and sensory systems of frogs, our results suggest frog biofluorescence is likely functioning in anuran communication.\nA frog adaptive radiation: Ecomorphological evolution in Old-World shrub frogs (Rhacophoridae: Pseudophilautus)\nAuthors: Meegaskumbura, M.; Ellepola, G.; Senevirathne, G.; Manamendra-Arachchi, K.; Wijayathilaka, N.; Pie, M. R.; Sun, D.; Pethiyagoda, R.; Schneider, C. J.\nScore: 9.0, Published: 2023-08-14 DOI: 10.1101/2023.08.12.549244\nEcomorphs result from divergent natural selection, leading to species-rich adaptive radiations. Identifying ecomorphs and the resulting adaptive radiations in frogs is challenging due to conserved morphology and high species diversity. In this study, we demonstrate the ecological and climate specializations that have driven the diversification of shrub frogs of the genus Pseudophilautus in Sri Lanka, a tropical continental island. We use a time-calibrated phylogeny, morphometric analyses, and climate-niche evolution, and identify five ecomorphological categories, including Tree-shrub, Rock-boulder, Leaf-litter, Habitat Generalists, and Canopy forms, and describe their evolution. Body size is the primary factor separating species, and specific body features correlate with habitat type. Ecomorphs likely evolved multiple times in disparate lineages, and in different regions and altitudes, during cold climatic periods owing to monsoon cycles resulting from the Himalayan-Tibetan orogeny. The common ancestor was a medium-sized, wet-adapted, tree-shrub habitat specialist which originated in the late Oligocene. Extreme size classes (diminutive leaf litter forms and large canopy forms) evolved recently and suggest that morphological disparity arose late in diversification, possibly aided by favorable climates. This work will facilitate understanding of adaptive radiations in frogs, which possibly will help uncover the prevalence of subtle adaptive radiations in frogs, just as in tailed-vertebrates.\nGene expression is the main driver of purifying selection in large penguin populations\nAuthors: Trucchi, E.; Massa, P.; Giannelli, F.; Fernandes, F. A. N.; Ancona, L.; Stenseth, N. C.; Ferrer Obiol, J.; Paris, J.; Bertorelle, G.; Le Bohec, C.\nScore: 9.2, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552445\nPurifying selection is the most pervasive type of selection, as it constantly removes deleterious mutations arising in populations, directly scaling with population size. Highly expressed genes appear to accumulate fewer deleterious mutations between divergent species lineages, pointing towards gene expression as an additional driver of purifying selection. However, estimates of the effect of gene expression on segregating deleterious variants in natural populations are lacking, as well as an understanding of the relative contribution of population size and gene expression to overall purifying selection pressure. Here, we analyse genomic and transcriptomic data from two natural populations of closely related sister species with different demographic histories, the Emperor (Aptenodytes forsteri) and the King penguins (A. patagonicus), and demonstrate that purifying selection at the population-level depends on the level of gene expression, with larger effects than population size. Deleterious segregating variants spread less in the population when they are in genes with higher expression rate. Leveraging realistic forward simulations, we estimate that the top 10% of the most highly expressed genes in a genome experience a selection pressure corresponding to an average selection coefficient of -0.1, which decreases to a selection coefficient of -0.01 for the top 50%. Gene expression appears to be a fundamental driver of purifying selection in natural populations, also effective at small population size. We suggest gene expression could be used as a proxy for gene selection coefficients (i.e., distribution of fitness effects), which are notoriously difficult to derive in non-model species under real-world conditions.\nTransmission bottleneck size estimation from de novo viral genetic variation\nAuthors: Shi, T.; Harris, J.; Martin, M. A.; Koelle, K.\nScore: 8.3, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553219\nSequencing of viral infections has become increasingly common over the last decade. Deep sequencing data in particular have proven useful in characterizing the roles that genetic drift and natural selection play in shaping within-host viral populations. They have also been used to estimate transmission bottleneck sizes from identified donor-recipient pairs. These bottleneck sizes quantify the number of viral particles that establish genetic lineages in the recipient host and are important to estimate due to their impact on viral evolution. Current approaches for estimating bottleneck sizes exclusively consider the subset of viral sites that are observed as polymorphic in the donor individual. However, allele frequencies can change dramatically over the course of an individual's infection, such that sites that are polymorphic in the donor at the time of transmission may not be polymorphic in the donor at the time of sampling and allele frequencies at donor-polymorphic sites may change dramatically over the course of a recipient's infection. Because of this, transmission bottleneck sizes estimated using allele frequencies observed at a donor's polymorphic sites may be considerable underestimates of true bottleneck sizes. Here, we present a new statistical approach for instead estimating bottleneck sizes using patterns of viral genetic variation that arose de novo within a recipient individual. Specifically, our approach makes use of the number of clonal viral variants observed in a transmission pair, defined as the number of viral sites that are monomorphic in both the donor and the recipient but carry different alleles. We first test our approach on a simulated dataset and then apply it to both influenza A virus sequence data and SARS-CoV-2 sequence data from identified transmission pairs. Our results confirm the existence of extremely tight transmission bottlenecks for these two respiratory viruses, using an approach that does not tend to underestimate transmission bottleneck sizes.\nCompetition-driven eco-evolutionary feedback reshapes bacteriophage lambda's fitness landscape and enables speciation\nAuthors: Doud, M. B.; Gupta, A.; Li, V.; Medina, S. J.; De La Fuente, C. A.; Meyer, J. R.\nScore: 7.9, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553017\nA major challenge in evolutionary biology is explaining how populations navigate rugged fitness landscapes without getting trapped on local optima. One idea illustrated by adaptive dynamics theory is that as populations adapt, their newly enhanced capacities to exploit resources alter fitness payoffs and restructure the landscape in ways that promote speciation by opening new adaptive pathways. While there have been indirect tests of this theory, none have measured how fitness landscapes deform during adaptation, or test whether these shifts promote diversification. Here, we achieve this by studying bacteriophage {lambda}, a virus that readily speciates into co-existing receptor specialists under controlled laboratory conditions. We used a high-throughput gene editing-phenotyping technology to measure {lambda}'s fitness landscape in the presence of different evolved-{lambda} competitors and found that the fitness effects of individual mutations, and their epistatic interactions, depend on the competitor. Using these empirical data, we simulated {lambda}'s evolution on an unchanging landscape and one that recapitulates how the landscape deforms during evolution. {lambda} heterogeneity only evolved in the shifting landscape regime. This study provides a test of adaptive dynamics, and, more broadly, shows how fitness landscapes dynamically change during adaptation, potentiating phenomena like speciation by opening new adaptive pathways.\nFunctional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification\nAuthors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.\nScore: 53.8, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782\nTransposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene. In Drosophila ovaries, iERVs with functional envelope have adapted their cis-regulatory sequences to be expressed in any somatic cell type, from where they infect the germline. Strikingly, related retroviruses show distinct expression patterns, indicating niche partitioning. In contrast, all non-infectious iERVs that emerged through secondary envelope-loss are specifically expressed in the germline. Co-evolving with iERVs, the genome-protecting piRNA pathway has assimilated iERV promoter and sequence information into piRNA clusters, underscoring the functional significance of iERV expression in somatic niches. We propose that the evolutionary innovation of cell-to-cell infectivity has triggered the adaptive radiation of iERVs through trait diversification and antagonistic virus-host interactions, processes that likely underpin niche-specific expression of endogenous retroviruses in vertebrates as well.\n",
  "wordCount" : "2522",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552427">
        <p class="paperTitle">An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.</p>
        <p class="info">Score: 32.0, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552427' target='https://doi.org/10.1101/2023.08.08.552427'> 10.1101/2023.08.08.552427</a></p>
        <p class="abstract">Most of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time. However, the co-evolution of life and the Earth system has left signatures of bacterial metabolism in the geochemical record, most conspicuously the Great Oxidation Event (GOE) [~]2.33 billion years ago (Ga, (Poulton et al. 2021)), in which oxygenic photosynthesis and tectonism (Eguchi, Seales, and Dasgupta 2019) transformed Earths biosphere from dominantly anaerobic to aerobic. Here, we combine machine learning and phylogenetic reconciliation to infer ancestral transitions to aerobic lifestyles during bacterial evolution. Linking these transitions to the GOE provides new constraints to infer the timetree of Bacteria. We find that extant bacterial phyla are truly ancient, having radiated in the Archaean and the Proterozoic: the oldest include Bacillota (Firmicutes), which radiated 3.1-3.7 Ga, Cyanobacteria (3.3-3.5 Ga) and Patescibacteria (3-3.5 Ga). We show that most bacterial phyla were ancestrally anaerobic and that most transitions to an aerobic lifestyle post-dated the GOE. Our analyses trace oxygen production and consumption back to Cyanobacteria. From that starting point, horizontal transfer seeded aerobic lifestyles across bacterial diversity over hundreds of millions of years. Our analyses demonstrate that the diversification of aerobes proceeded in two waves corresponding to the GOE and to a second sustained rise in atmospheric O2 at the dawn of the Palezoic (Krause et al. 2022).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.09.21.508910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.09.21.508910" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.09.21.508910">
        <p class="paperTitle">A conserved ribosomal protein has entirely dissimilar structures in different organisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.09.21.508910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.09.21.508910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schierholz, L.; Brown, C. R.; Helena-Bueno, K.; Uversky, V. N.; Hirt, R.; Barandun, J.; Melnikov, S. V.</p>
        <p class="info">Score: 34.5, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.09.21.508910' target='https://doi.org/10.1101/2022.09.21.508910'> 10.1101/2022.09.21.508910</a></p>
        <p class="abstract">Ribosomes from different species can markedly differ in their composition by including dozens of ribosomal proteins that are unique to specific lineages but absent in others. However, it remains unknown how ribosomes acquire and specialize new proteins throughout evolution. Here, to help answer this question, we describe the evolution of the ribosomal protein msL1/msL2 that was recently found in ribosomes from the parasitic microorganism clade, microsporidia. We first show that this protein has a conserved location in the ribosome but entirely dissimilar structures in different organisms: in each of the analyzed species, msL1/msL2 exhibits an altered secondary structure, an inverted orientation of the N- and C-termini on the ribosomal binding surface, and a completely transformed three-dimensional fold. We then show that this evolutionary fold switching is likely caused by changes in the msL1/msL2-binding site in the ribosome; specifically, by variations in microsporidian rRNA. These observations allow us to infer an evolutionary scenario in which a small, positively-charged, de novo-born unfolded protein was first captured by rRNA to become part of the ribosome and subsequently underwent complete fold switching to optimize its binding to its evolving ribosomal binding site. Overall, our work provides a striking example that will help us better understand the origin and evolution of new protein components of complex molecular assemblies - thereby enhancing our ability to engineer biological molecules, identify protein homologs, and peer into the history of life on Earth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550994">
        <p class="paperTitle">Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.</p>
        <p class="info">Score: 227.1, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550994' target='https://doi.org/10.1101/2023.07.28.550994'> 10.1101/2023.07.28.550994</a></p>
        <p class="abstract">Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552622">
        <p class="paperTitle">Tetrapod vocal evolution: higher frequencies and faster rates of evolution in mammalian vocalizations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Munoz, M. I.; Marsot, M.; Ellers, J.; Halfwerk, W.</p>
        <p class="info">Score: 25.0, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552622' target='https://doi.org/10.1101/2023.08.09.552622'> 10.1101/2023.08.09.552622</a></p>
        <p class="abstract">Using the voice to produce sound is a widespread form of communication and plays an important role in contexts as diverse as parent-offspring interactions and courtship. Variation in the tempo and mode of vocal signal evolution has been studied in a phylogenetic context within orders or classes, but understanding vocal signal evolution ultimately requires comparison across all major lineages involved. Here we used comparative analyses to investigate the evolution of dominant frequency (i.e., the frequency with the highest energy content) and its association with body weight across 873 species of mammals, birds and frogs. In agreement with previous studies, we found that the negative allometric relationship between body weight and vocal frequency is a general feature of vocal systems. In addition, we found mammals to consistently vocalize at higher frequencies, and evolved their vocalizations at around 6-fold faster rates than those of birds and frogs. Although all three groups strongly rely on vocal communication, our findings show that only mammals have extensively explored the spectral acoustic space. We argue that such high vocal diversity of mammals is made possible by their unique hearing system, which evolved in the context of a small, parental-caring, nocturnal and insectivore ancestor, and has allowed them to detect, and therefore to evolve, a richer array of frequencies than other tetrapods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550432">
        <p class="paperTitle">Evidence for ecological tuning of novel anuran biofluorescent signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.</p>
        <p class="info">Score: 17.6, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550432' target='https://doi.org/10.1101/2023.07.25.550432'> 10.1101/2023.07.25.550432</a></p>
        <p class="abstract">Our study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians. We more than tripled the number of species that have been tested for this trait and added representatives from previously untested anuran families. We found evidence for ecological tuning (i.e., the specific adaptation of a signal to the environment in which it is received) of the novel anuran biofluorescent signals. Across groups, the fluorescence excitation peak matches the wavelengths most available at twilight, the light environment in which most frog species are active. Additionally, biofluorescence emission spans both wavelengths of low availability in twilight and the peak sensitivity of green-sensitive rods in the anuran eye, likely increasing contrast of this signal for a conspecific receiver. With evidence of tuning to the ecology and sensory systems of frogs, our results suggest frog biofluorescence is likely functioning in anuran communication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.549244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.549244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.549244">
        <p class="paperTitle">A frog adaptive radiation: Ecomorphological evolution in Old-World shrub frogs (Rhacophoridae: Pseudophilautus)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.549244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.549244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meegaskumbura, M.; Ellepola, G.; Senevirathne, G.; Manamendra-Arachchi, K.; Wijayathilaka, N.; Pie, M. R.; Sun, D.; Pethiyagoda, R.; Schneider, C. J.</p>
        <p class="info">Score: 9.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.549244' target='https://doi.org/10.1101/2023.08.12.549244'> 10.1101/2023.08.12.549244</a></p>
        <p class="abstract">Ecomorphs result from divergent natural selection, leading to species-rich adaptive radiations. Identifying ecomorphs and the resulting adaptive radiations in frogs is challenging due to conserved morphology and high species diversity. In this study, we demonstrate the ecological and climate specializations that have driven the diversification of shrub frogs of the genus Pseudophilautus in Sri Lanka, a tropical continental island. We use a time-calibrated phylogeny, morphometric analyses, and climate-niche evolution, and identify five ecomorphological categories, including Tree-shrub, Rock-boulder, Leaf-litter, Habitat Generalists, and Canopy forms, and describe their evolution. Body size is the primary factor separating species, and specific body features correlate with habitat type. Ecomorphs likely evolved multiple times in disparate lineages, and in different regions and altitudes, during cold climatic periods owing to monsoon cycles resulting from the Himalayan-Tibetan orogeny. The common ancestor was a medium-sized, wet-adapted, tree-shrub habitat specialist which originated in the late Oligocene. Extreme size classes (diminutive leaf litter forms and large canopy forms) evolved recently and suggest that morphological disparity arose late in diversification, possibly aided by favorable climates. This work will facilitate understanding of adaptive radiations in frogs, which possibly will help uncover the prevalence of subtle adaptive radiations in frogs, just as in tailed-vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552445">
        <p class="paperTitle">Gene expression is the main driver of purifying selection in large penguin populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trucchi, E.; Massa, P.; Giannelli, F.; Fernandes, F. A. N.; Ancona, L.; Stenseth, N. C.; Ferrer Obiol, J.; Paris, J.; Bertorelle, G.; Le Bohec, C.</p>
        <p class="info">Score: 9.2, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552445' target='https://doi.org/10.1101/2023.08.08.552445'> 10.1101/2023.08.08.552445</a></p>
        <p class="abstract">Purifying selection is the most pervasive type of selection, as it constantly removes deleterious mutations arising in populations, directly scaling with population size. Highly expressed genes appear to accumulate fewer deleterious mutations between divergent species lineages, pointing towards gene expression as an additional driver of purifying selection. However, estimates of the effect of gene expression on segregating deleterious variants in natural populations are lacking, as well as an understanding of the relative contribution of population size and gene expression to overall purifying selection pressure. Here, we analyse genomic and transcriptomic data from two natural populations of closely related sister species with different demographic histories, the Emperor (Aptenodytes forsteri) and the King penguins (A. patagonicus), and demonstrate that purifying selection at the population-level depends on the level of gene expression, with larger effects than population size. Deleterious segregating variants spread less in the population when they are in genes with higher expression rate. Leveraging realistic forward simulations, we estimate that the top 10% of the most highly expressed genes in a genome experience a selection pressure corresponding to an average selection coefficient of -0.1, which decreases to a selection coefficient of -0.01 for the top 50%. Gene expression appears to be a fundamental driver of purifying selection in natural populations, also effective at small population size. We suggest gene expression could be used as a proxy for gene selection coefficients (i.e., distribution of fitness effects), which are notoriously difficult to derive in non-model species under real-world conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553219">
        <p class="paperTitle">Transmission bottleneck size estimation from de novo viral genetic variation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, T.; Harris, J.; Martin, M. A.; Koelle, K.</p>
        <p class="info">Score: 8.3, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553219' target='https://doi.org/10.1101/2023.08.14.553219'> 10.1101/2023.08.14.553219</a></p>
        <p class="abstract">Sequencing of viral infections has become increasingly common over the last decade. Deep sequencing data in particular have proven useful in characterizing the roles that genetic drift and natural selection play in shaping within-host viral populations. They have also been used to estimate transmission bottleneck sizes from identified donor-recipient pairs. These bottleneck sizes quantify the number of viral particles that establish genetic lineages in the recipient host and are important to estimate due to their impact on viral evolution. Current approaches for estimating bottleneck sizes exclusively consider the subset of viral sites that are observed as polymorphic in the donor individual. However, allele frequencies can change dramatically over the course of an individual&#39;s infection, such that sites that are polymorphic in the donor at the time of transmission may not be polymorphic in the donor at the time of sampling and allele frequencies at donor-polymorphic sites may change dramatically over the course of a recipient&#39;s infection. Because of this, transmission bottleneck sizes estimated using allele frequencies observed at a donor&#39;s polymorphic sites may be considerable underestimates of true bottleneck sizes. Here, we present a new statistical approach for instead estimating bottleneck sizes using patterns of viral genetic variation that arose de novo within a recipient individual. Specifically, our approach makes use of the number of clonal viral variants observed in a transmission pair, defined as the number of viral sites that are monomorphic in both the donor and the recipient but carry different alleles. We first test our approach on a simulated dataset and then apply it to both influenza A virus sequence data and SARS-CoV-2 sequence data from identified transmission pairs. Our results confirm the existence of extremely tight transmission bottlenecks for these two respiratory viruses, using an approach that does not tend to underestimate transmission bottleneck sizes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553017">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553017" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553017">
        <p class="paperTitle">Competition-driven eco-evolutionary feedback reshapes bacteriophage lambda&#39;s fitness landscape and enables speciation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553017" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553017" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doud, M. B.; Gupta, A.; Li, V.; Medina, S. J.; De La Fuente, C. A.; Meyer, J. R.</p>
        <p class="info">Score: 7.9, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553017' target='https://doi.org/10.1101/2023.08.11.553017'> 10.1101/2023.08.11.553017</a></p>
        <p class="abstract">A major challenge in evolutionary biology is explaining how populations navigate rugged fitness landscapes without getting trapped on local optima. One idea illustrated by adaptive dynamics theory is that as populations adapt, their newly enhanced capacities to exploit resources alter fitness payoffs and restructure the landscape in ways that promote speciation by opening new adaptive pathways. While there have been indirect tests of this theory, none have measured how fitness landscapes deform during adaptation, or test whether these shifts promote diversification. Here, we achieve this by studying bacteriophage {lambda}, a virus that readily speciates into co-existing receptor specialists under controlled laboratory conditions. We used a high-throughput gene editing-phenotyping technology to measure {lambda}&#39;s fitness landscape in the presence of different evolved-{lambda} competitors and found that the fitness effects of individual mutations, and their epistatic interactions, depend on the competitor. Using these empirical data, we simulated {lambda}&#39;s evolution on an unchanging landscape and one that recapitulates how the landscape deforms during evolution. {lambda} heterogeneity only evolved in the shifting landscape regime. This study provides a test of adaptive dynamics, and, more broadly, shows how fitness landscapes dynamically change during adaptation, potentiating phenomena like speciation by opening new adaptive pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551782">
        <p class="paperTitle">Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.</p>
        <p class="info">Score: 53.8, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551782' target='https://doi.org/10.1101/2023.08.03.551782'> 10.1101/2023.08.03.551782</a></p>
        <p class="abstract">Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene. In Drosophila ovaries, iERVs with functional envelope have adapted their cis-regulatory sequences to be expressed in any somatic cell type, from where they infect the germline. Strikingly, related retroviruses show distinct expression patterns, indicating niche partitioning. In contrast, all non-infectious iERVs that emerged through secondary envelope-loss are specifically expressed in the germline. Co-evolving with iERVs, the genome-protecting piRNA pathway has assimilated iERV promoter and sequence information into piRNA clusters, underscoring the functional significance of iERV expression in somatic niches. We propose that the evolutionary innovation of cell-to-cell infectivity has triggered the adaptive radiation of iERVs through trait diversification and antagonistic virus-host interactions, processes that likely underpin niche-specific expression of endogenous retroviruses in vertebrates as well.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
