<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life
Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life
Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life
Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life\nAuthors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.",
  "keywords": [
    
  ],
  "articleBody": " Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life\nAuthors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.; Simoni, S.; Steenwinkel, T. E.; Schiffer, M.; Syed, Z. A.; Takahashi, A.; Wei, K. H.-C.; Yokoyama, T.; Eisen, M. B.; Kopp, A.; Matute, D.; Obbard, D. J.; O'Grady, P. M.; Price, D. K.; Toda, M. J.; Werner, T.; Petrov, D. A.\nScore: 58.9, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560517\nLong-read sequencing is driving rapid progress in genome assembly across all major groups of life, including species of the family Drosophilidae, a longtime model system for genetics, genomics, and evolution. We previously developed a cost-effective hybrid Oxford Nanopore (ONT) long-read and Illumina short-read sequencing approach and used it to assemble 101 drosophilid genomes from laboratory cultures, greatly increasing the number of genome assemblies for this taxonomic group. The next major challenge is to address the laboratory culture bias in taxon sampling by sequencing genomes of species that cannot easily be reared in the lab. Here, we build upon our previous methods to perform amplification-free ONT sequencing of single wild flies obtained either directly from the field or from ethanol-preserved specimens in museum collections, greatly improving the representation of lesser studied drosophilid taxa in whole-genome data. Using Illumina Novaseq X Plus and ONT P2 sequencers with R10.4.1 chemistry, we set a new benchmark for inexpensive hybrid genome assembly at US $150 per genome while assembling genomes from as little as 35 ng of genomic DNA from a single fly. We present 183 new genome assemblies for 179 species as a resource for drosophilid systematics, phylogenetics, and comparative genomics. Of these genomes, 62 are from pooled lab strains and 121 from single adult flies. Despite the sample limitations of working with small insects, most single-fly diploid assemblies are comparable in contiguity (\u003e1Mb contig N50), completeness (\u003e98% complete dipteran BUSCOs), and accuracy (\u003eQV40 genome-wide with ONT R10.4.1) to assemblies from inbred lines. We present a well-resolved multi-locus phylogeny for 360 drosophilid and 4 outgroup species encompassing all publicly available (as of August 2023) genomes for this group. Finally, we present a Progressive Cactus whole-genome, reference-free alignment built from a subset of 298 suitably high-quality drosophilid genomes. The new assemblies and alignment, along with updated laboratory protocols and computational pipelines, are released as an open resource and as a tool for studying evolution at the scale of an entire insect family.\nThe amino acid sequence determines protein abundance through its conformational stability and reduced synthesis cost.\nAuthors: Buric, F.; Viknander, S.; Fu, X.; Lemke, O.; Zrimec, J.; Szyrwiel, L.; Muelleder, M.; Ralser, M.; Zelezniak, A.\nScore: 30.7, Published: 2023-10-04 DOI: 10.1101/2023.10.02.560091\nUnderstanding what drives protein abundance is essential to biology, medicine, and biotechnology. Driven by evolutionary selection, the amino acid sequence is tailored to meet the required abundance of proteomes, underscoring the intricate relationship between sequence and functional demand of proteomes. Yet, the specific role of amino acid sequences in determining proteome abundance remains elusive. Here, we demonstrate that the amino acid sequence predicts abundance by shaping a protein's conformational stability. We show that increasing the abundance provides metabolic cost benefits, underscoring the evolutionary advantage of maintaining a highly abundant and stable proteome. Specifically, using a deep learning model (BERT), we predict 56% of protein abundance variation in Saccharomyces cerevisiae solely based on amino acid sequence. The model reveals latent factors linking sequence features to protein stability. To probe these relationships, we introduce MGEM (Mutation Guided by an Embedded Manifold), a methodology for guiding protein abundance through sequence modifications. We find that mutations increasing abundance significantly alter protein polarity and hydrophobicity, underscoring a connection between protein stability and abundance. Through molecular dynamics simulations and in vivo experiments in yeast, we confirm that abundance-enhancing mutations result in longer-lasting and more stable protein expression. Importantly, these sequence changes also reduce metabolic costs of protein synthesis, elucidating the evolutionary advantage of cost-effective, high-abundance, stable proteomes. Our findings support the role of amino acid sequence as a pivotal determinant of protein abundance and stability, revealing an evolutionary optimization for metabolic efficiency.\nRecurrent duplication and diversification of a vital DNA repair gene family across Drosophila\nAuthors: Brand, C. L.; Oliver, G. T.; Farkas, I. Z.; Levine, M. T.\nScore: 27.0, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560779\nMaintaining genome integrity is vital for organismal survival and reproduction. Essential, broadly conserved DNA repair pathways actively preserve genome integrity. However, many DNA repair proteins evolve adaptively. Ecological forces like UV exposure are classically cited as drivers of DNA repair evolution. Intrinsic forces like repetitive DNA, which can also imperil genome integrity, have received less attention. We recently reported that a Drosophila melanogaster-specific DNA satellite array triggered species-specific, adaptive evolution of a DNA repair protein called Spartan/MH. The Spartan family of proteases cleave hazardous, covalent crosslinks that form between DNA and proteins (DNA-protein crosslink repair). Appreciating that DNA satellites are both ubiquitous and universally fast-evolving, we hypothesized that satellite DNA turnover spurs evolution of DNA-protein crosslink repair beyond D. melanogaster. This hypothesis predicts pervasive Spartan gene family diversification across the Drosophila phylogeny. To study the evolutionary history of the Drosophila Spartan gene family, we conducted population genetic, molecular evolution, phylogenomic, and tissue-specific expression analyses. We uncovered widespread signals of positive selection across multiple Spartan family genes and across multiple evolutionary timescales. We also detected recurrent Spartan family gene duplication, divergence, and gene loss. Finally, we found that ovary-enriched parent genes consistently birthed testis-enriched daughter genes. To account for Drosophila-wide, Spartan family diversification, we introduce a mechanistic model of antagonistic coevolution that links DNA satellite evolution and adaptive regulation of Spartan protease activity. This framework, combined with a recent explosion of genome assemblies that encompass repeat-rich genomic regions, promises to accelerate our understanding of how DNA repeats drive recurrent evolutionary innovation to preserve genome integrity.\nDeep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait\nAuthors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.\nScore: 23.6, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560175\nThe extraordinary variation in male guppy coloration has been a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we used a population with low levels of Y-linkage for color variation to identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments and controlled pedigrees and whole-genome resequencing for Genome Wide Association Study (GWAS) of colour. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture. Unusually, our GWAS results suggest that copy number variation (CNV) is responsible for much of the variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.\nGenome analysis of the Jomon dogs reveals the oldest domestic dog lineage in Eastern Eurasia\nAuthors: Terai, Y.; Xiaokaiti, X.; Gojobori, J.; Arakawa, N.; Sato, T.; Kasai, K.; Machida, K.; Yamazaki, K.; Yamaji, N.; Hongo, H.; Gakuhari, T.\nScore: 20.6, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560089\nDog is the oldest domesticated animal that established close relationships with humans. Due to its ancient origin, when, where, and whether a single or dual domestication event occurred is still under debate. The dogs in the Jomon period (Jomon dogs) in the Japanese archipelago had little change in morphology from 10,000 to 3,000 years ago. Therefore, we expected that the ancient genome of the Jomon dogs would provide a clue to reveal the characteristics of the ancient East Asian dogs. Here, we have sequenced the genomes of three 6000-year-old Jomon dogs, one 3000-4000-year-old Jomon dog, and four late 8th century dogs excavated in Japan. Our analyses suggest that the Jomon dogs are a distinct lineage from the previously known ancient dogs and are one of the oldest among the dogs in East Eurasian lineages. In addition, the genome of the Jomon dogs contained 9.5% of the genome of Japanese wolf ancestry due to a single introgression event. We estimated the proportion of the Jomon dog lineage genome in the genomes of dogs, which indicates that the genomic composition derived from the Jomon dog lineage is one of the major sources of modern dog genomes. Furthermore, we estimated the early admixture events of dogs in East Eurasia by analyzing the ancient genomes of the Jomon dogs. Due to the admixture events, the Jomon dog-derived genome has been one of the genomic sources of a wide range of modern dogs.\nPolyploidy is widespread in Microsporidia\nAuthors: Khalaf, A.; Jaron, K. S.; Blaxter, M.; Lawniczak, M.\nScore: 13.9, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560119\nMicrosporidia are obligate intracellular eukaryotic parasites with an extremely broad host range. They have both economic and public health importance. Ploidy in microsporidia is variable, with a few species formally identified as diploid and one as polyploid. Given the increase in the number of studies sequencing microsporidian genomes, it is now possible to assess ploidy levels across all currently-explored microsporidian diversity. We estimate ploidy for all microsporidian datasets available on the Sequence Read Archive (SRA) using k-mer-based analyses, and demonstrate that polyploidy is widespread in Microsporidia, and that ploidy change is dynamic in the group. Using genome-wide heterozygosity estimates, we also show that polyploid microsporidian genomes are relatively homozygous, and we discuss the implications of these findings on the timing of polyploidisation events and their origin. Author SummaryMicrosporidia are single-celled intracellular parasites, distantly related to fungi, that can infect a broad range of hosts, from humans all the way to protozoans. Exploiting the wealth of microsporidian genomic data available, we use k-mer based analyses to assess ploidy status across the group. Understanding a genomes ploidy is crucial in order to assemble it effectively, and may also be relevant for better understanding a parasites behaviour and life cycle. We show that tetraploidy is present in at least six species in Microsporidia, and that these polyploidisation events are likely to have occurred independently. We discuss why these findings may be paradoxical, given that Microsporidia, like other intracellular parasites, have extremely small, reduced genomes.\nIntegrating Genomics, Collections, and Community Science to Reveal Speciation in a Variable Monitor Lizard (Varanus tristis)\nAuthors: Pavon-Vazquez, C. J.; Fitch, A. J.; Doughty, P.; Donnellan, S. C.; Keogh, J. S.\nScore: 11.4, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560725\nThe accurate characterization of species diversity is a vital prerequisite for ecological and evolutionary research, as well as conservation. Thus, it is necessary to generate robust hypotheses of species limits based on the inference of evolutionary processes. Integrative species delimitation, the inference of species limits based on multiple sources of evidence, can provide unique insight into species diversity and the processes behind it. However, the application of integrative approaches in non-model organisms is often limited by the amount of data that is available. Here, we show how data relevant for species delimitation can be bolstered by incorporating information from tissue collections, museum specimens, and observations made by the wider community. We show how to integrate these data under a hypothesis-driven, integrative framework by identifying the processes generating genetic and phenotypic variation in Varanus tristis, a widespread and variable complex of Australian monitor lizards. Using genomic, morphometric (linear and geometric), coloration, spatial, and environmental data we show that disparity in this complex is inconsistent with intraspecific variation and instead suggests that speciation has occurred. Based on our results, we identify the environmental factors that may have been responsible for the geographic sorting of variation. Our workflow provides a guideline for the integrative analysis of several types of data to identify the occurrence and causes of speciation. Furthermore, our study highlights how community science and machine learning--two tools used here--can be used to accelerate taxonomic research.\nTrade-offs, trade-ups, and high mutational parallelism underlie microbial adaptation to extreme feast/famine\nAuthors: Behringer, M. G.; Ho, W.-C.; Miller, S. F.; Worthan, S. B.; Cen, Z.; Stikeleather, R.; Lynch, M.\nScore: 9.8, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560893\nMicrobes are robust organisms capable of rapidly adapting to complex stress, enabling the colonization of harsh environments. In nature, microbes are regularly challenged by starvation, which is a particularly complex stress because resource limitation often co-occurs with changes in pH, osmolarity, and toxin accumulation created by metabolic waste. Often overlooked are the additional complications introduced by eventual resource replenishment as successful microbes must withstand rapid environmental shifts before swiftly capitalizing on replenished resources to avoid invasion by competing species. To understand how microbes navigate trade-offs between growth and survival, ultimately adapting to thrive in environments with extreme fluctuations, we experimentally evolved 16 Escherichia coli populations for 900 days to repeated feast/famine cycles of 100-day starvation before resource replenishment. Using longitudinal population-genomic analysis, we found that evolution in response to extreme feast/famine is characterized by narrow adaptive trajectories with high mutational parallelism and notable mutational order. Genetic reconstructions reveal that early mutations result in trade-offs for biofilm and motility but trade-ups for growth and survival, as these mutations conferred correlated advantages during both short-term and long-term culture. Our results demonstrate how microbes can navigate the adaptive landscapes of regularly fluctuating conditions and ultimately follow mutational trajectories that confer benefits across diverse environments.\nA genome-scale Opisthokonta tree of life: toward phylogenomic resolution of ancient divergences\nAuthors: Liu, H.; Steenwyk, J. L.; Zhou, X.; Schultz, D. T.; Kocot, K. M.; Shen, X.-X.; Rokas, A.; Li, Y.\nScore: 10.4, Published: 2023-09-21 DOI: 10.1101/2023.09.20.556338\nAbstractAncient divergences within Opisthokonta--a major lineage that includes organisms in the kingdoms Animalia, Fungi, and their unicellular relatives-- remain contentious, hindering investigations of the evolutionary processes that gave rise to two kingdoms and the repeated emergence of iconic phenotypes like multicellularity. Here, we use genome-scale amounts of data to reconstruct the most taxon-rich Opisthokonta tree of life to date (348 species) and place divergences in geologic time, suggesting a Mesoproterozoic origin ([~] 1.11 billion years ago). By dissecting multiple dimensions of phylogenomic error, such as the influence of taxon sampling and model complexity, we found that deep divergences within Holozoa remain unresolved and suggest Pluriformea is either sister to Ichthyosporea and Filozoa (Pluriformea-sister hypothesis) or is monophyletic to Ichthyosporea, forming the Teretosporea lineage (Teretosporea-sister hypothesis). A combination of information theory and sensitivity analyses revealed that the inferred unicellular Holozoa relationships are largely robust to common sources of analytical error, such as insufficient model complexity, and suggest that previous reports likely suffered from insufficient taxon sampling. Our study presents a robust Opisthokonta phylogenomic framework, highlights the challenges in resolving the relationships of unicellular Holozoa, and paves the way for illuminating ancient evolutionary episodes concerning the origin of two kingdoms.\nPlasticity-led evolution as an intrinsic property of developmental gene regulatory networks\nAuthors: Ng, E. T. H.; Kinjo, A. R.\nScore: 9.7, Published: 2023-09-22 DOI: 10.1101/2023.05.25.542372\nThe Modern Evolutionary Synthesis seemingly fails to explain how a population can survive a large environmental change: the pre-existence of heritable variants adapted to the novel environment is too opportunistic, whereas the search for new adaptive mutations after the environmental change is so slow that the population may go extinct. Plasticity-led evolution, the initial environmental induction of a novel adaptive phenotype followed by genetic accommodation, has been proposed to solve this problem. However, the mechanism enabling plasticity-led evolution remains unclear. Here, we present computational models that exhibit behaviors compatible with plasticity-led evolution by extending the Wagner model of gene regulatory networks. The models show adaptive plastic response and the uncovering of cryptic mutations under large environmental changes, followed by genetic accommodation. Moreover, these behaviors are consistently observed over distinct novel environments. We further show that environmental cues, developmental processes, and hierarchical regulation cooperatively amplify the above behaviors and accelerate evolution. These observations suggest plasticity-led evolution is a universal property of complex developmental systems independent of particular mutations.\n",
  "wordCount" : "2700",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560517">
        <p class="paperTitle">Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.; Simoni, S.; Steenwinkel, T. E.; Schiffer, M.; Syed, Z. A.; Takahashi, A.; Wei, K. H.-C.; Yokoyama, T.; Eisen, M. B.; Kopp, A.; Matute, D.; Obbard, D. J.; O&#39;Grady, P. M.; Price, D. K.; Toda, M. J.; Werner, T.; Petrov, D. A.</p>
        <p class="info">Score: 58.9, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560517' target='https://doi.org/10.1101/2023.10.02.560517'> 10.1101/2023.10.02.560517</a></p>
        <p class="abstract">Long-read sequencing is driving rapid progress in genome assembly across all major groups of life, including species of the family Drosophilidae, a longtime model system for genetics, genomics, and evolution. We previously developed a cost-effective hybrid Oxford Nanopore (ONT) long-read and Illumina short-read sequencing approach and used it to assemble 101 drosophilid genomes from laboratory cultures, greatly increasing the number of genome assemblies for this taxonomic group. The next major challenge is to address the laboratory culture bias in taxon sampling by sequencing genomes of species that cannot easily be reared in the lab. Here, we build upon our previous methods to perform amplification-free ONT sequencing of single wild flies obtained either directly from the field or from ethanol-preserved specimens in museum collections, greatly improving the representation of lesser studied drosophilid taxa in whole-genome data. Using Illumina Novaseq X Plus and ONT P2 sequencers with R10.4.1 chemistry, we set a new benchmark for inexpensive hybrid genome assembly at US $150 per genome while assembling genomes from as little as 35 ng of genomic DNA from a single fly. We present 183 new genome assemblies for 179 species as a resource for drosophilid systematics, phylogenetics, and comparative genomics. Of these genomes, 62 are from pooled lab strains and 121 from single adult flies. Despite the sample limitations of working with small insects, most single-fly diploid assemblies are comparable in contiguity (&gt;1Mb contig N50), completeness (&gt;98% complete dipteran BUSCOs), and accuracy (&gt;QV40 genome-wide with ONT R10.4.1) to assemblies from inbred lines. We present a well-resolved multi-locus phylogeny for 360 drosophilid and 4 outgroup species encompassing all publicly available (as of August 2023) genomes for this group. Finally, we present a Progressive Cactus whole-genome, reference-free alignment built from a subset of 298 suitably high-quality drosophilid genomes. The new assemblies and alignment, along with updated laboratory protocols and computational pipelines, are released as an open resource and as a tool for studying evolution at the scale of an entire insect family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560091" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560091">
        <p class="paperTitle">The amino acid sequence determines protein abundance through its conformational stability and reduced synthesis cost.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buric, F.; Viknander, S.; Fu, X.; Lemke, O.; Zrimec, J.; Szyrwiel, L.; Muelleder, M.; Ralser, M.; Zelezniak, A.</p>
        <p class="info">Score: 30.7, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560091' target='https://doi.org/10.1101/2023.10.02.560091'> 10.1101/2023.10.02.560091</a></p>
        <p class="abstract">Understanding what drives protein abundance is essential to biology, medicine, and biotechnology. Driven by evolutionary selection, the amino acid sequence is tailored to meet the required abundance of proteomes, underscoring the intricate relationship between sequence and functional demand of proteomes. Yet, the specific role of amino acid sequences in determining proteome abundance remains elusive. Here, we demonstrate that the amino acid sequence predicts abundance by shaping a protein&#39;s conformational stability. We show that increasing the abundance provides metabolic cost benefits, underscoring the evolutionary advantage of maintaining a highly abundant and stable proteome. Specifically, using a deep learning model (BERT), we predict 56% of protein abundance variation in Saccharomyces cerevisiae solely based on amino acid sequence. The model reveals latent factors linking sequence features to protein stability. To probe these relationships, we introduce MGEM (Mutation Guided by an Embedded Manifold), a methodology for guiding protein abundance through sequence modifications. We find that mutations increasing abundance significantly alter protein polarity and hydrophobicity, underscoring a connection between protein stability and abundance. Through molecular dynamics simulations and in vivo experiments in yeast, we confirm that abundance-enhancing mutations result in longer-lasting and more stable protein expression. Importantly, these sequence changes also reduce metabolic costs of protein synthesis, elucidating the evolutionary advantage of cost-effective, high-abundance, stable proteomes. Our findings support the role of amino acid sequence as a pivotal determinant of protein abundance and stability, revealing an evolutionary optimization for metabolic efficiency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560779" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560779">
        <p class="paperTitle">Recurrent duplication and diversification of a vital DNA repair gene family across Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brand, C. L.; Oliver, G. T.; Farkas, I. Z.; Levine, M. T.</p>
        <p class="info">Score: 27.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560779' target='https://doi.org/10.1101/2023.10.04.560779'> 10.1101/2023.10.04.560779</a></p>
        <p class="abstract">Maintaining genome integrity is vital for organismal survival and reproduction. Essential, broadly conserved DNA repair pathways actively preserve genome integrity. However, many DNA repair proteins evolve adaptively. Ecological forces like UV exposure are classically cited as drivers of DNA repair evolution. Intrinsic forces like repetitive DNA, which can also imperil genome integrity, have received less attention. We recently reported that a Drosophila melanogaster-specific DNA satellite array triggered species-specific, adaptive evolution of a DNA repair protein called Spartan/MH. The Spartan family of proteases cleave hazardous, covalent crosslinks that form between DNA and proteins (DNA-protein crosslink repair). Appreciating that DNA satellites are both ubiquitous and universally fast-evolving, we hypothesized that satellite DNA turnover spurs evolution of DNA-protein crosslink repair beyond D. melanogaster. This hypothesis predicts pervasive Spartan gene family diversification across the Drosophila phylogeny. To study the evolutionary history of the Drosophila Spartan gene family, we conducted population genetic, molecular evolution, phylogenomic, and tissue-specific expression analyses. We uncovered widespread signals of positive selection across multiple Spartan family genes and across multiple evolutionary timescales. We also detected recurrent Spartan family gene duplication, divergence, and gene loss. Finally, we found that ovary-enriched parent genes consistently birthed testis-enriched daughter genes. To account for Drosophila-wide, Spartan family diversification, we introduce a mechanistic model of antagonistic coevolution that links DNA satellite evolution and adaptive regulation of Spartan protease activity. This framework, combined with a recent explosion of genome assemblies that encompass repeat-rich genomic regions, promises to accelerate our understanding of how DNA repeats drive recurrent evolutionary innovation to preserve genome integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560175">
        <p class="paperTitle">Deep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.</p>
        <p class="info">Score: 23.6, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560175' target='https://doi.org/10.1101/2023.09.29.560175'> 10.1101/2023.09.29.560175</a></p>
        <p class="abstract">The extraordinary variation in male guppy coloration has been a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we used a population with low levels of Y-linkage for color variation to identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments and controlled pedigrees and whole-genome resequencing for Genome Wide Association Study (GWAS) of colour. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture. Unusually, our GWAS results suggest that copy number variation (CNV) is responsible for much of the variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560089" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560089">
        <p class="paperTitle">Genome analysis of the Jomon dogs reveals the oldest domestic dog lineage in Eastern Eurasia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terai, Y.; Xiaokaiti, X.; Gojobori, J.; Arakawa, N.; Sato, T.; Kasai, K.; Machida, K.; Yamazaki, K.; Yamaji, N.; Hongo, H.; Gakuhari, T.</p>
        <p class="info">Score: 20.6, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560089' target='https://doi.org/10.1101/2023.09.29.560089'> 10.1101/2023.09.29.560089</a></p>
        <p class="abstract">Dog is the oldest domesticated animal that established close relationships with humans. Due to its ancient origin, when, where, and whether a single or dual domestication event occurred is still under debate. The dogs in the Jomon period (Jomon dogs) in the Japanese archipelago had little change in morphology from 10,000 to 3,000 years ago. Therefore, we expected that the ancient genome of the Jomon dogs would provide a clue to reveal the characteristics of the ancient East Asian dogs. Here, we have sequenced the genomes of three 6000-year-old Jomon dogs, one 3000-4000-year-old Jomon dog, and four late 8th century dogs excavated in Japan. Our analyses suggest that the Jomon dogs are a distinct lineage from the previously known ancient dogs and are one of the oldest among the dogs in East Eurasian lineages. In addition, the genome of the Jomon dogs contained 9.5% of the genome of Japanese wolf ancestry due to a single introgression event. We estimated the proportion of the Jomon dog lineage genome in the genomes of dogs, which indicates that the genomic composition derived from the Jomon dog lineage is one of the major sources of modern dog genomes. Furthermore, we estimated the early admixture events of dogs in East Eurasia by analyzing the ancient genomes of the Jomon dogs. Due to the admixture events, the Jomon dog-derived genome has been one of the genomic sources of a wide range of modern dogs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560119">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560119" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560119">
        <p class="paperTitle">Polyploidy is widespread in Microsporidia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560119" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560119" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khalaf, A.; Jaron, K. S.; Blaxter, M.; Lawniczak, M.</p>
        <p class="info">Score: 13.9, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560119' target='https://doi.org/10.1101/2023.09.29.560119'> 10.1101/2023.09.29.560119</a></p>
        <p class="abstract">Microsporidia are obligate intracellular eukaryotic parasites with an extremely broad host range. They have both economic and public health importance. Ploidy in microsporidia is variable, with a few species formally identified as diploid and one as polyploid. Given the increase in the number of studies sequencing microsporidian genomes, it is now possible to assess ploidy levels across all currently-explored microsporidian diversity. We estimate ploidy for all microsporidian datasets available on the Sequence Read Archive (SRA) using k-mer-based analyses, and demonstrate that polyploidy is widespread in Microsporidia, and that ploidy change is dynamic in the group. Using genome-wide heterozygosity estimates, we also show that polyploid microsporidian genomes are relatively homozygous, and we discuss the implications of these findings on the timing of polyploidisation events and their origin.

Author SummaryMicrosporidia are single-celled intracellular parasites, distantly related to fungi, that can infect a broad range of hosts, from humans all the way to protozoans. Exploiting the wealth of microsporidian genomic data available, we use k-mer based analyses to assess ploidy status across the group. Understanding a genomes ploidy is crucial in order to assemble it effectively, and may also be relevant for better understanding a parasites behaviour and life cycle. We show that tetraploidy is present in at least six species in Microsporidia, and that these polyploidisation events are likely to have occurred independently. We discuss why these findings may be paradoxical, given that Microsporidia, like other intracellular parasites, have extremely small, reduced genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560725">
        <p class="paperTitle">Integrating Genomics, Collections, and Community Science to Reveal Speciation in a Variable Monitor Lizard (Varanus tristis)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pavon-Vazquez, C. J.; Fitch, A. J.; Doughty, P.; Donnellan, S. C.; Keogh, J. S.</p>
        <p class="info">Score: 11.4, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560725' target='https://doi.org/10.1101/2023.10.03.560725'> 10.1101/2023.10.03.560725</a></p>
        <p class="abstract">The accurate characterization of species diversity is a vital prerequisite for ecological and evolutionary research, as well as conservation. Thus, it is necessary to generate robust hypotheses of species limits based on the inference of evolutionary processes. Integrative species delimitation, the inference of species limits based on multiple sources of evidence, can provide unique insight into species diversity and the processes behind it. However, the application of integrative approaches in non-model organisms is often limited by the amount of data that is available. Here, we show how data relevant for species delimitation can be bolstered by incorporating information from tissue collections, museum specimens, and observations made by the wider community. We show how to integrate these data under a hypothesis-driven, integrative framework by identifying the processes generating genetic and phenotypic variation in Varanus tristis, a widespread and variable complex of Australian monitor lizards. Using genomic, morphometric (linear and geometric), coloration, spatial, and environmental data we show that disparity in this complex is inconsistent with intraspecific variation and instead suggests that speciation has occurred. Based on our results, we identify the environmental factors that may have been responsible for the geographic sorting of variation. Our workflow provides a guideline for the integrative analysis of several types of data to identify the occurrence and causes of speciation. Furthermore, our study highlights how community science and machine learning--two tools used here--can be used to accelerate taxonomic research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560893">
        <p class="paperTitle">Trade-offs, trade-ups, and high mutational parallelism underlie microbial adaptation to extreme feast/famine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Behringer, M. G.; Ho, W.-C.; Miller, S. F.; Worthan, S. B.; Cen, Z.; Stikeleather, R.; Lynch, M.</p>
        <p class="info">Score: 9.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560893' target='https://doi.org/10.1101/2023.10.04.560893'> 10.1101/2023.10.04.560893</a></p>
        <p class="abstract">Microbes are robust organisms capable of rapidly adapting to complex stress, enabling the colonization of harsh environments. In nature, microbes are regularly challenged by starvation, which is a particularly complex stress because resource limitation often co-occurs with changes in pH, osmolarity, and toxin accumulation created by metabolic waste. Often overlooked are the additional complications introduced by eventual resource replenishment as successful microbes must withstand rapid environmental shifts before swiftly capitalizing on replenished resources to avoid invasion by competing species. To understand how microbes navigate trade-offs between growth and survival, ultimately adapting to thrive in environments with extreme fluctuations, we experimentally evolved 16 Escherichia coli populations for 900 days to repeated feast/famine cycles of 100-day starvation before resource replenishment. Using longitudinal population-genomic analysis, we found that evolution in response to extreme feast/famine is characterized by narrow adaptive trajectories with high mutational parallelism and notable mutational order. Genetic reconstructions reveal that early mutations result in trade-offs for biofilm and motility but trade-ups for growth and survival, as these mutations conferred correlated advantages during both short-term and long-term culture. Our results demonstrate how microbes can navigate the adaptive landscapes of regularly fluctuating conditions and ultimately follow mutational trajectories that confer benefits across diverse environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.556338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.556338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.556338">
        <p class="paperTitle">A genome-scale Opisthokonta tree of life: toward phylogenomic resolution of ancient divergences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.556338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.556338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, H.; Steenwyk, J. L.; Zhou, X.; Schultz, D. T.; Kocot, K. M.; Shen, X.-X.; Rokas, A.; Li, Y.</p>
        <p class="info">Score: 10.4, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.556338' target='https://doi.org/10.1101/2023.09.20.556338'> 10.1101/2023.09.20.556338</a></p>
        <p class="abstract">AbstractAncient divergences within Opisthokonta--a major lineage that includes organisms in the kingdoms Animalia, Fungi, and their unicellular relatives-- remain contentious, hindering investigations of the evolutionary processes that gave rise to two kingdoms and the repeated emergence of iconic phenotypes like multicellularity. Here, we use genome-scale amounts of data to reconstruct the most taxon-rich Opisthokonta tree of life to date (348 species) and place divergences in geologic time, suggesting a Mesoproterozoic origin ([~] 1.11 billion years ago). By dissecting multiple dimensions of phylogenomic error, such as the influence of taxon sampling and model complexity, we found that deep divergences within Holozoa remain unresolved and suggest Pluriformea is either sister to Ichthyosporea and Filozoa (Pluriformea-sister hypothesis) or is monophyletic to Ichthyosporea, forming the Teretosporea lineage (Teretosporea-sister hypothesis). A combination of information theory and sensitivity analyses revealed that the inferred unicellular Holozoa relationships are largely robust to common sources of analytical error, such as insufficient model complexity, and suggest that previous reports likely suffered from insufficient taxon sampling. Our study presents a robust Opisthokonta phylogenomic framework, highlights the challenges in resolving the relationships of unicellular Holozoa, and paves the way for illuminating ancient evolutionary episodes concerning the origin of two kingdoms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.25.542372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.25.542372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.25.542372">
        <p class="paperTitle">Plasticity-led evolution as an intrinsic property of developmental gene regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.25.542372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.25.542372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ng, E. T. H.; Kinjo, A. R.</p>
        <p class="info">Score: 9.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.25.542372' target='https://doi.org/10.1101/2023.05.25.542372'> 10.1101/2023.05.25.542372</a></p>
        <p class="abstract">The Modern Evolutionary Synthesis seemingly fails to explain how a population can survive a large environmental change: the pre-existence of heritable variants adapted to the novel environment is too opportunistic, whereas the search for new adaptive mutations after the environmental change is so slow that the population may go extinct. Plasticity-led evolution, the initial environmental induction of a novel adaptive phenotype followed by genetic accommodation, has been proposed to solve this problem. However, the mechanism enabling plasticity-led evolution remains unclear. Here, we present computational models that exhibit behaviors compatible with plasticity-led evolution by extending the Wagner model of gene regulatory networks. The models show adaptive plastic response and the uncovering of cryptic mutations under large environmental changes, followed by genetic accommodation. Moreover, these behaviors are consistently observed over distinct novel environments. We further show that environmental cues, developmental processes, and hierarchical regulation cooperatively amplify the above behaviors and accelerate evolution. These observations suggest plasticity-led evolution is a universal property of complex developmental systems independent of particular mutations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
