<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s
Authors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.
Score: 30.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564725
During the last few centuries Drosophila melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s
Authors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.
Score: 30.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564725
During the last few centuries Drosophila melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s
Authors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.
Score: 30.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564725
During the last few centuries Drosophila melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s\nAuthors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.\nScore: 30.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564725\nDuring the last few centuries Drosophila melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D.",
  "keywords": [
    
  ],
  "articleBody": " Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s\nAuthors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.\nScore: 30.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564725\nDuring the last few centuries Drosophila melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D. melanogaster. It is a 5216nt LTR retrotransposon of the Ty3/gypsy superfamily. Relying on strains sampled at different times during the last century we show that Spoink invaded worldwide D. melanogaster populations after the P-element between 1983 and 1993. This invasion was likely triggered by a horizontal transfer from the D. willistoni group, much as the P-element. Spoink is probably silenced by the piRNA pathway in natural populations and about 1/3 of the examined strains have an insertion into a canonical piRNA cluster such as 42AB. Given the degree of genetic investigation of D. melanogaster it is surprising that Spoink was able to invade unnoticed.\nFrom museum drawer to tree: historical DNA phylogenomics clarifies the systematics of rare dung beetles (Coleoptera: Scarabaeinae) from museum collections\nAuthors: Lopes, F.; Gunter, N.; Gillett, C. P. D. T.; Montanaro, G.; Rossini, M.; Losacco, F.; Daniel, G. M.; Straube, N.; Tarasov, S.\nScore: 22.6, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564347\nAlthough several methods exist for extracting and sequencing historical DNA originating from dry-preserved insect specimens deposited in natural history museums, no consensus exists as to what is the optimal approach. We demonstrate that a customized, low-cost archival DNA extraction protocol (~10 EUR per sample), in combination with Ultraconserved Elements (UCEs), is an effective tool for insect phylogenomic studies. We successfully tested our approach by sequencing DNA from scarab dung beetles preserved in both wet and dry collections, including unique primary type and rare historical specimens from internationally important natural history museums in London, Paris and Helsinki. The focal specimens comprise enigmatic dung beetle genera (Nesosisyphus, Onychotechus and Helictopleurus) that varied in age and preservation. The oldest specimen, the holotype of the now possibly extinct Mauritian endemic Nesosisyphus rotundatus, was collected in 1944. We obtained high-quality DNA from all studied specimens to enable the generation of a UCE-based dataset that revealed an insightful and well-supported phylogenetic tree of dung beetles. The resulting phylogeny suggested the reclassification of Onychotechus previously incertae sedis within the tribe Coprini. Our approach demonstrates the feasibility and effectiveness of combining DNA data from historic and recent museum specimens to provide novel insights. The proposed archival DNA protocol is available at DOI 10.17504/protocols.io.81wgbybqyvpk/v1\nPhylogenomics reveals the deep ocean as an accelerator for evolutionary diversification in anglerfishes\nAuthors: Miller, E. C.; Faucher, R.; Hart, P. B.; Rincon-Sandoval, M.; Santaquiteria, A.; White, W. T.; Baldwin, C. C.; Miya, M.; Betancur-R, R.; Tornabene, L.; Evans, K.; Arcila, D.\nScore: 22.5, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564281\nColonization of a novel habitat is often followed by radiation in the wake of ecological opportunity. Alternatively, some habitats should be inherently more constraining than others if the challenges of that environment have few evolutionary solutions. We examined the push-and-pull of these factors on evolution following habitat transitions, using anglerfishes (Lophiiformes) as a model. Deep-sea fishes are notoriously difficult to study, and poor sampling has limited progress thus far. Here we present a new phylogeny of anglerfishes with unprecedented taxonomic sampling (1,092 loci and 40% of species), combined with three-dimensional phenotypic data from museum specimens obtained with micro-CT scanning. We use these datasets to examine the tempo and mode of phenotypic and lineage diversification using phylogenetic comparative methods, comparing lineages in shallow and deep benthic versus bathypelagic habitats. Our results show that anglerfishes represent a surprising case where the bathypelagic lineage has greater taxonomic and phenotypic diversity than coastal benthic relatives. This defies expectations based on ecological principles since the bathypelagic zone is the most homogeneous habitat on Earth. Deep-sea anglerfishes experienced rapid lineage diversification concomitant with colonization of the bathypelagic zone from a continental slope ancestor. They display the highest body, skull and jaw shape disparity across lophiiforms. In contrast, reef-associated taxa show strong constraints on shape and low evolutionary rates, contradicting patterns suggested by other shallow marine fishes. We found that Lophiiformes as a whole evolved under an early burst model with subclades occupying distinct body shapes. We further discuss to what extent the bathypelagic clade is a secondary adaptive radiation, or if its diversity can be explained by non-adaptive processes.\nThe distribution of fitness effects during adaptive walks using a simple genetic network\nAuthors: O'Brien, N. L. V.; Holland, B.; Engelstaedter, J.; Ortiz-Barrientos, D.\nScore: 17.2, Published: 2023-10-31 DOI: 10.1101/2023.10.26.564303\nThe tempo and mode of adaptation is dependent on the availability of beneficial alleles. Genetic interactions arising from gene networks can restrict this availability. However, the extent to which networks affect adaptation remains inconclusive. Current models of evolution consider additive genotypephenotype relationships while often ignoring the contribution of gene interactions to phenotypic variance. In this study, we present a novel approach to modeling quantitative traits as products of genetic networks via systems of ordinary differential equations. This allows us to mechanistically explore the effects of network structures on adaptation. We present the first step of this approach by studying a simple gene regulatory network, the negative autoregulation motif. Using forward-time genetic simulations, we measure adaptive walks towards a phenotypic optimum in both additive and network models. A key expectation from adaptive walk theory is that the distribution of fitness effects of new beneficial mutations is exponential. We found that although both models strayed from expectation, harboring fewer beneficial mutations than expected, they took a similar number of adaptive steps to reach the optimum. Network model populations were 10% less likely to reach the optimum and took longer to adapt. This was partly due to a complex and largely deleterious distribution of fitness effects from new mutations. This behavior is reminiscent of the cost of complexity, where correlations among traits constrain adaptation. Our results suggest that the interactions emerging from genetic networks can hinder adaptation by generating complex and multimodal distributions of fitness effects.\nThe brittle star genome illuminates the genetic basis of animal appendage regeneration\nAuthors: Parey, E.; Ortega-Martinez, O.; Delroisse, J.; Piovani, L.; Czarkwiani, A.; Dylus, D.; Arya, S.; Dupont, S.; Thorndyke, M.; Larsson, T.; Johannesson, K.; Buckley, K. M.; Martinez, P.; Oliveri, P.; Marletaz, F.\nScore: 16.9, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564762\nSpecies within nearly all extant animal lineages are capable of regenerating body parts. However, it remains unclear whether the gene expression programme controlling regeneration is evolutionarily conserved. Brittle stars are a species-rich class of echinoderms with outstanding regenerative abilities, but investigations into the genetic bases of regeneration in this group have been hindered by the limited available genomic resources. Here, we report a chromosome-scale genome assembly for the brittle star Amphiura filiformis. We show that the brittle star displays the most rearranged genome amongst echinoderms sequenced to date, featuring a reorganised Hox cluster reminiscent of the rearrangements observed in sea urchins. In addition, we performed an extensive profiling of gene expression throughout brittle star adult arm regeneration and identified sequential waves of gene expression governing wound healing, proliferation and differentiation. We conducted comparative transcriptomic analyses with other invertebrate and vertebrate models for appendage regeneration and uncovered hundreds of genes with conserved expression dynamics, notably during the proliferative phase of regeneration. Our findings emphasise the crucial importance of echinoderms to detect long-range expression conservation between vertebrates and classical invertebrate regeneration model systems.\nThe landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.\nScore: 138.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561165\nInfectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.\nGenome of the glacier alga Ancylonema and its insights into the evolution of streptophyte life on ice and land\nAuthors: Bowles, A. M. C.; Williams, T. A.; Donoghue, P. C. J.; Campbell, D. A.; Williamson, C. J.\nScore: 14.3, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563300\nContemporary glaciers and ice sheets are home to communities of streptophyte glacier algae that must balance their requirements for photosynthesis and growth with tolerance of extremes in temperature, desiccation and UV radiation. These same environmental challenges have been hypothesized as the driving force behind the evolution of land plants from streptophyte algal ancestors in the Cryogenian (720-635 million years ago). Here, we sequence, assemble and analyze the metagenome-assembled genome (MAG) of the glacier alga Ancylonema nordenskioldii to investigate its adaptations to life in ice, and whether this represents a vestige of Cryogenian anydrophyte exaptations. Phylogenetic analysis confirms the placement of glacier algae within the sister lineage to land plants, Zygnematophyceae. The MAG is characterized by an expansion of genes involved in high irradiance and UV light tolerance, whilst lineage-specific diversification led to the novel screening pigmentation of glacier algae. We found no support for the hypothesis of a common genomic basis for adaptations to ice and to land in streptophytes. Comparative genomic analysis revealed that reductive genome evolution in the ancestor of Zygnematophyceae correlates with their reductive morphological evolution. This first genome-scale data for glacier algae supports an Ancylonema-specific adaptation to the cryosphere, as well as shedding light on the genome evolution of land plants and Zygnematophyceae.\nSelection leads to false inferences of introgression using popular methods\nAuthors: Smith, M. L.; Hahn, M. W.\nScore: 24.6, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564394\nDetecting introgression between closely related populations or species is a fundamental objective in evolutionary biology. Existing methods for detecting migration and inferring migration rates from population genetic data often assume a neutral model of evolution. Growing evidence of the pervasive impact of selection on large portions of the genome across diverse taxa suggests that this assumption is unrealistic in most empirical systems. Further, ignoring selection has previously been shown to negatively impact demographic inferences (e.g., of population size histories). However, the impacts of biologically realistic selection on inferences of migration remain poorly explored. Here, we simulate data under models of background selection, selective sweeps, and adaptive introgression. We show that ignoring selection leads to false inferences of migration in three popularly used methods (fastsimcoal, {partial}a{partial}i, and BPP). Selection results in the rejection of isolation-only models in favor of isolation-with-migration models and leads to elevated estimates of migration rates across methods. Our results suggest that such methods may be unreliable in many empirical systems, such that new methods that are robust to selection will need to be developed. Article SummaryDetecting migration between closely related populations is a central objective in many evolutionary biology studies. However, popular methods for detecting migration assume a simplified model of evolution. Here, we evaluate the impacts of biologically realistic natural selection, recombination, and mutation on three methods for detecting migration. We find that biological complexity leads to false inferences of migration, suggesting that results should be interpreted with caution and that new methods are needed to make robust inferences of migration across empirical systems.\nTarget complementarity in cnidarians supports a common origin for animal and plant microRNAs\nAuthors: Admoni, Y.; Fridrich, A.; Razin, T.; Salinas-Saavedra, M.; Rabani, M.; Frank, U.; Moran, Y.\nScore: 28.8, Published: 2023-10-30 DOI: 10.1101/2023.01.08.523153\nmicroRNAs (miRNAs) are important post-transcriptional regulators that activate silencing mechanisms by annealing to mRNA transcripts. While plant miRNAs match their targets with nearly-full complementarity leading to mRNA cleavage, miRNAs in most animals require only a short sequence called seed to inhibit target translation. Recent findings showed that miRNAs in cnidarians, early-branching metazoans, act similarly to plant miRNAs, by exhibiting full complementarity and target cleavage; however, it remained unknown if seed- based regulation was possible in cnidarians. Here, we investigate the miRNA-target complementarity requirements for miRNA activity in the cnidarian Nematostella vectensis. We show that bilaterian-like complementarity of seed-only or seed and supplementary 3 matches are insufficient for miRNA-mediated knockdown. Furthermore, miRNA-target mismatches in the cleavage site decrease knockdown efficiency. Finally, miRNA silencing of a target with three seed binding sites in the 3 untranslated region that mimics typical miRNA targeting was repressed in zebrafish but not in Nematostella and Hydractinia symbiolongicarpus. Altogether, these results unravel striking similarities between plant and cnidarian miRNAs consolidating the evidence for common evolutionary origin of miRNAs in plants and animals.\nA whole-body micro-CT scan library that captures the skeletal diversity of Lake Malawi cichlid fishes\nAuthors: Bucklow, C. V.; Genner, M. J.; Turner, G. F.; Maclaine, J.; Benson, R.; Verd, B.\nScore: 9.8, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565162\nHere we describe a dataset of freely available, readily processed, whole-body CT-scans of 56 species (116 specimens) of Lake Malawi cichlid fishes that captures a considerable majority of the morphological variation present in this remarkable adaptive radiation. We contextualise the scanned specimens within a discussion of their respective ecomorphological groupings and suggest possible macroevolutionary studies that could be conducted with these data. We also describe a methodology to efficiently CT-scan (on average) 23 specimens per hour, limiting scanning time and alleviating the financial cost whilst maintaining high resolution. We demonstrate the utility of this method by reconstructing 3D models of multiple bones from multiple specimens within the dataset. We hope this dataset will enable further morphological study of this fascinating system and permit wider-scale comparisons with other cichlid adaptive radiations.\n",
  "wordCount" : "2592",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564725">
        <p class="paperTitle">Spoink, a LTR retrotransposon, invaded D. melanogaster populations in the 1990s</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pianezza, R.; Scarpa, A.; Narayanan, P.; Signor, S.; Kofler, R.</p>
        <p class="info">Score: 30.6, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564725' target='https://doi.org/10.1101/2023.10.30.564725'> 10.1101/2023.10.30.564725</a></p>
        <p class="abstract">During the last few centuries Drosophila melanogaster populations were invaded by several transposable elements, the most recent of which was thought to be the P-element between 1950 and 1980. Here we describe a novel TE, which we named Spoink, that has invaded D. melanogaster. It is a 5216nt LTR retrotransposon of the Ty3/gypsy superfamily. Relying on strains sampled at different times during the last century we show that Spoink invaded worldwide D. melanogaster populations after the P-element between 1983 and 1993. This invasion was likely triggered by a horizontal transfer from the D. willistoni group, much as the P-element. Spoink is probably silenced by the piRNA pathway in natural populations and about 1/3 of the examined strains have an insertion into a canonical piRNA cluster such as 42AB. Given the degree of genetic investigation of D. melanogaster it is surprising that Spoink was able to invade unnoticed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564347">
        <p class="paperTitle">From museum drawer to tree: historical DNA phylogenomics clarifies the systematics of rare dung beetles (Coleoptera: Scarabaeinae) from museum collections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopes, F.; Gunter, N.; Gillett, C. P. D. T.; Montanaro, G.; Rossini, M.; Losacco, F.; Daniel, G. M.; Straube, N.; Tarasov, S.</p>
        <p class="info">Score: 22.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564347' target='https://doi.org/10.1101/2023.10.27.564347'> 10.1101/2023.10.27.564347</a></p>
        <p class="abstract">Although several methods exist for extracting and sequencing historical DNA originating from dry-preserved insect specimens deposited in natural history museums, no consensus exists as to what is the optimal approach. We demonstrate that a customized, low-cost archival DNA extraction protocol (~10 EUR per sample), in combination with Ultraconserved Elements (UCEs), is an effective tool for insect phylogenomic studies. We successfully tested our approach by sequencing DNA from scarab dung beetles preserved in both wet and dry collections, including unique primary type and rare historical specimens from internationally important natural history museums in London, Paris and Helsinki. The focal specimens comprise enigmatic dung beetle genera (Nesosisyphus, Onychotechus and Helictopleurus) that varied in age and preservation. The oldest specimen, the holotype of the now possibly extinct Mauritian endemic Nesosisyphus rotundatus, was collected in 1944. We obtained high-quality DNA from all studied specimens to enable the generation of a UCE-based dataset that revealed an insightful and well-supported phylogenetic tree of dung beetles. The resulting phylogeny suggested the reclassification of Onychotechus previously incertae sedis within the tribe Coprini. Our approach demonstrates the feasibility and effectiveness of combining DNA data from historic and recent museum specimens to provide novel insights. The proposed archival DNA protocol is available at DOI 10.17504/protocols.io.81wgbybqyvpk/v1</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564281">
        <p class="paperTitle">Phylogenomics reveals the deep ocean as an accelerator for evolutionary diversification in anglerfishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, E. C.; Faucher, R.; Hart, P. B.; Rincon-Sandoval, M.; Santaquiteria, A.; White, W. T.; Baldwin, C. C.; Miya, M.; Betancur-R, R.; Tornabene, L.; Evans, K.; Arcila, D.</p>
        <p class="info">Score: 22.5, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564281' target='https://doi.org/10.1101/2023.10.26.564281'> 10.1101/2023.10.26.564281</a></p>
        <p class="abstract">Colonization of a novel habitat is often followed by radiation in the wake of ecological opportunity. Alternatively, some habitats should be inherently more constraining than others if the challenges of that environment have few evolutionary solutions. We examined the push-and-pull of these factors on evolution following habitat transitions, using anglerfishes (Lophiiformes) as a model. Deep-sea fishes are notoriously difficult to study, and poor sampling has limited progress thus far. Here we present a new phylogeny of anglerfishes with unprecedented taxonomic sampling (1,092 loci and 40% of species), combined with three-dimensional phenotypic data from museum specimens obtained with micro-CT scanning. We use these datasets to examine the tempo and mode of phenotypic and lineage diversification using phylogenetic comparative methods, comparing lineages in shallow and deep benthic versus bathypelagic habitats. Our results show that anglerfishes represent a surprising case where the bathypelagic lineage has greater taxonomic and phenotypic diversity than coastal benthic relatives. This defies expectations based on ecological principles since the bathypelagic zone is the most homogeneous habitat on Earth. Deep-sea anglerfishes experienced rapid lineage diversification concomitant with colonization of the bathypelagic zone from a continental slope ancestor. They display the highest body, skull and jaw shape disparity across lophiiforms. In contrast, reef-associated taxa show strong constraints on shape and low evolutionary rates, contradicting patterns suggested by other shallow marine fishes. We found that Lophiiformes as a whole evolved under an early burst model with subclades occupying distinct body shapes. We further discuss to what extent the bathypelagic clade is a secondary adaptive radiation, or if its diversity can be explained by non-adaptive processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564303">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564303" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564303">
        <p class="paperTitle">The distribution of fitness effects during adaptive walks using a simple genetic network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564303" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564303" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Brien, N. L. V.; Holland, B.; Engelstaedter, J.; Ortiz-Barrientos, D.</p>
        <p class="info">Score: 17.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564303' target='https://doi.org/10.1101/2023.10.26.564303'> 10.1101/2023.10.26.564303</a></p>
        <p class="abstract">The tempo and mode of adaptation is dependent on the availability of beneficial alleles. Genetic interactions arising from gene networks can restrict this availability. However, the extent to which networks affect adaptation remains inconclusive. Current models of evolution consider additive genotypephenotype relationships while often ignoring the contribution of gene interactions to phenotypic variance. In this study, we present a novel approach to modeling quantitative traits as products of genetic networks via systems of ordinary differential equations. This allows us to mechanistically explore the effects of network structures on adaptation. We present the first step of this approach by studying a simple gene regulatory network, the negative autoregulation motif. Using forward-time genetic simulations, we measure adaptive walks towards a phenotypic optimum in both additive and network models. A key expectation from adaptive walk theory is that the distribution of fitness effects of new beneficial mutations is exponential. We found that although both models strayed from expectation, harboring fewer beneficial mutations than expected, they took a similar number of adaptive steps to reach the optimum. Network model populations were 10% less likely to reach the optimum and took longer to adapt. This was partly due to a complex and largely deleterious distribution of fitness effects from new mutations. This behavior is reminiscent of the cost of complexity, where correlations among traits constrain adaptation. Our results suggest that the interactions emerging from genetic networks can hinder adaptation by generating complex and multimodal distributions of fitness effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564762">
        <p class="paperTitle">The brittle star genome illuminates the genetic basis of animal appendage regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parey, E.; Ortega-Martinez, O.; Delroisse, J.; Piovani, L.; Czarkwiani, A.; Dylus, D.; Arya, S.; Dupont, S.; Thorndyke, M.; Larsson, T.; Johannesson, K.; Buckley, K. M.; Martinez, P.; Oliveri, P.; Marletaz, F.</p>
        <p class="info">Score: 16.9, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564762' target='https://doi.org/10.1101/2023.10.30.564762'> 10.1101/2023.10.30.564762</a></p>
        <p class="abstract">Species within nearly all extant animal lineages are capable of regenerating body parts. However, it remains unclear whether the gene expression programme controlling regeneration is evolutionarily conserved. Brittle stars are a species-rich class of echinoderms with outstanding regenerative abilities, but investigations into the genetic bases of regeneration in this group have been hindered by the limited available genomic resources. Here, we report a chromosome-scale genome assembly for the brittle star Amphiura filiformis. We show that the brittle star displays the most rearranged genome amongst echinoderms sequenced to date, featuring a reorganised Hox cluster reminiscent of the rearrangements observed in sea urchins. In addition, we performed an extensive profiling of gene expression throughout brittle star adult arm regeneration and identified sequential waves of gene expression governing wound healing, proliferation and differentiation. We conducted comparative transcriptomic analyses with other invertebrate and vertebrate models for appendage regeneration and uncovered hundreds of genes with conserved expression dynamics, notably during the proliferative phase of regeneration. Our findings emphasise the crucial importance of echinoderms to detect long-range expression conservation between vertebrates and classical invertebrate regeneration model systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561165">
        <p class="paperTitle">The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.</p>
        <p class="info">Score: 138.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561165' target='https://doi.org/10.1101/2023.10.06.561165'> 10.1101/2023.10.06.561165</a></p>
        <p class="abstract">Infectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood1. To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from [~]6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates [~]5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals2. However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe3,4. This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563300">
        <p class="paperTitle">Genome of the glacier alga Ancylonema and its insights into the evolution of streptophyte life on ice and land</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bowles, A. M. C.; Williams, T. A.; Donoghue, P. C. J.; Campbell, D. A.; Williamson, C. J.</p>
        <p class="info">Score: 14.3, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563300' target='https://doi.org/10.1101/2023.10.20.563300'> 10.1101/2023.10.20.563300</a></p>
        <p class="abstract">Contemporary glaciers and ice sheets are home to communities of streptophyte glacier algae that must balance their requirements for photosynthesis and growth with tolerance of extremes in temperature, desiccation and UV radiation. These same environmental challenges have been hypothesized as the driving force behind the evolution of land plants from streptophyte algal ancestors in the Cryogenian (720-635 million years ago). Here, we sequence, assemble and analyze the metagenome-assembled genome (MAG) of the glacier alga Ancylonema nordenskioldii to investigate its adaptations to life in ice, and whether this represents a vestige of Cryogenian anydrophyte exaptations. Phylogenetic analysis confirms the placement of glacier algae within the sister lineage to land plants, Zygnematophyceae. The MAG is characterized by an expansion of genes involved in high irradiance and UV light tolerance, whilst lineage-specific diversification led to the novel screening pigmentation of glacier algae. We found no support for the hypothesis of a common genomic basis for adaptations to ice and to land in streptophytes. Comparative genomic analysis revealed that reductive genome evolution in the ancestor of Zygnematophyceae correlates with their reductive morphological evolution. This first genome-scale data for glacier algae supports an Ancylonema-specific adaptation to the cryosphere, as well as shedding light on the genome evolution of land plants and Zygnematophyceae.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564394">
        <p class="paperTitle">Selection leads to false inferences of introgression using popular methods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, M. L.; Hahn, M. W.</p>
        <p class="info">Score: 24.6, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564394' target='https://doi.org/10.1101/2023.10.27.564394'> 10.1101/2023.10.27.564394</a></p>
        <p class="abstract">Detecting introgression between closely related populations or species is a fundamental objective in evolutionary biology. Existing methods for detecting migration and inferring migration rates from population genetic data often assume a neutral model of evolution. Growing evidence of the pervasive impact of selection on large portions of the genome across diverse taxa suggests that this assumption is unrealistic in most empirical systems. Further, ignoring selection has previously been shown to negatively impact demographic inferences (e.g., of population size histories). However, the impacts of biologically realistic selection on inferences of migration remain poorly explored. Here, we simulate data under models of background selection, selective sweeps, and adaptive introgression. We show that ignoring selection leads to false inferences of migration in three popularly used methods (fastsimcoal, {partial}a{partial}i, and BPP). Selection results in the rejection of isolation-only models in favor of isolation-with-migration models and leads to elevated estimates of migration rates across methods. Our results suggest that such methods may be unreliable in many empirical systems, such that new methods that are robust to selection will need to be developed.

Article SummaryDetecting migration between closely related populations is a central objective in many evolutionary biology studies. However, popular methods for detecting migration assume a simplified model of evolution. Here, we evaluate the impacts of biologically realistic natural selection, recombination, and mutation on three methods for detecting migration. We find that biological complexity leads to false inferences of migration, suggesting that results should be interpreted with caution and that new methods are needed to make robust inferences of migration across empirical systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.08.523153">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.08.523153" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.08.523153">
        <p class="paperTitle">Target complementarity in cnidarians supports a common origin for animal and plant microRNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.08.523153" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.08.523153" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Admoni, Y.; Fridrich, A.; Razin, T.; Salinas-Saavedra, M.; Rabani, M.; Frank, U.; Moran, Y.</p>
        <p class="info">Score: 28.8, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.08.523153' target='https://doi.org/10.1101/2023.01.08.523153'> 10.1101/2023.01.08.523153</a></p>
        <p class="abstract">microRNAs (miRNAs) are important post-transcriptional regulators that activate silencing mechanisms by annealing to mRNA transcripts. While plant miRNAs match their targets with nearly-full complementarity leading to mRNA cleavage, miRNAs in most animals require only a short sequence called  seed to inhibit target translation. Recent findings showed that miRNAs in cnidarians, early-branching metazoans, act similarly to plant miRNAs, by exhibiting full complementarity and target cleavage; however, it remained unknown if seed- based regulation was possible in cnidarians. Here, we investigate the miRNA-target complementarity requirements for miRNA activity in the cnidarian Nematostella vectensis. We show that bilaterian-like complementarity of seed-only or seed and supplementary 3 matches are insufficient for miRNA-mediated knockdown. Furthermore, miRNA-target mismatches in the cleavage site decrease knockdown efficiency. Finally, miRNA silencing of a target with three seed binding sites in the 3 untranslated region that mimics typical miRNA targeting was repressed in zebrafish but not in Nematostella and Hydractinia symbiolongicarpus. Altogether, these results unravel striking similarities between plant and cnidarian miRNAs consolidating the evidence for common evolutionary origin of miRNAs in plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565162">
        <p class="paperTitle">A whole-body micro-CT scan library that captures the skeletal diversity of Lake Malawi cichlid fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bucklow, C. V.; Genner, M. J.; Turner, G. F.; Maclaine, J.; Benson, R.; Verd, B.</p>
        <p class="info">Score: 9.8, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565162' target='https://doi.org/10.1101/2023.11.01.565162'> 10.1101/2023.11.01.565162</a></p>
        <p class="abstract">Here we describe a dataset of freely available, readily processed, whole-body CT-scans of 56 species (116 specimens) of Lake Malawi cichlid fishes that captures a considerable majority of the morphological variation present in this remarkable adaptive radiation. We contextualise the scanned specimens within a discussion of their respective ecomorphological groupings and suggest possible macroevolutionary studies that could be conducted with these data. We also describe a methodology to efficiently CT-scan (on average) 23 specimens per hour, limiting scanning time and alleviating the financial cost whilst maintaining high resolution. We demonstrate the utility of this method by reconstructing 3D models of multiple bones from multiple specimens within the dataset. We hope this dataset will enable further morphological study of this fascinating system and permit wider-scale comparisons with other cichlid adaptive radiations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
