<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.",
  "keywords": [
    
  ],
  "articleBody": " Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.\nScore: 286.9, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547326\nThe evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.\nMachine learning illuminates how diet influences the evolution of yeast galactose metabolism\nAuthors: Harrison, M.-C.; Ubbelohde, E. J.; LaBella, A. L.; Opulente, D. A.; Wolters, J. F.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Hittinger, C. T.; Rokas, A.\nScore: 22.9, Published: 2023-07-23 DOI: 10.1101/2023.07.20.549758\nHow genomic differences contribute to phenotypic differences across species is a major question in biology. The recently characterized genomes, isolation environments, and qualitative patterns of growth on 122 sources and conditions of 1,154 strains from 1,049 fungal species (nearly all known) in the subphylum Saccharomycotina provide a powerful, yet complex, dataset for addressing this question. In recent years, machine learning has been successfully used in diverse analyses of biological big data. Using a random forest classification algorithm trained on these genomic, metabolic, and/or environmental data, we predicted growth on several carbon sources and conditions with high accuracy from presence/absence patterns of genes and of growth in other conditions. Known structural genes involved in assimilation of these sources were important features contributing to prediction accuracy, whereas isolation environmental data were poor predictors. By further examining growth on galactose, we found that it can be predicted with high accuracy from either genomic (92.6%) or growth data in 120 other conditions (83.3%) but not from isolation environment data (65.7%). When we combined genomic and growth data, we noted that prediction accuracy was even higher (93.4%) and that, after the GALactose utilization genes, the most important feature for predicting growth on galactose was growth on galactitol. These data raised the hypothesis that several species in two orders, Serinales and Pichiales (containing Candida auris and the genus Ogataea, respectively), have an alternative galactose utilization pathway because they lack the GAL genes. Growth and biochemical assays of several of these species confirmed that they utilize galactose through an oxidoreductive D-galactose pathway, rather than the canonical GAL pathway. We conclude that machine learning is a powerful tool for investigating the evolution of the yeast genotype-phenotype map and that it can help uncover novel biology, even in well-studied traits.\nPhylogeny and evolution of the SARS-CoV-2 spike gene from December 2022 to February 2023\nAuthors: Kao, H.-W.; Kao, H.-W.\nScore: 20.8, Published: 2023-07-24 DOI: 10.1101/2023.07.23.549423\nBackground: By the end of 2022, new variants of SARS-CoV-2, such as BQ.1.1.10, BA.4.6.3, XBB, and CH.1.1, emerged with higher fitness than BA.5. Methods: The file (spikeprot0304), which contains spike protein sequences, isolates collected before March, 4, 2023, was downloaded from Global Initiative on Sharing All Influenza Data (GISAID). A total of 188 different spike protein sequences were chosen, of which their isolates were collected from December 2022 to February 2023. These sequences did not contain undetermined amino acid X, and each spike protein sequence had at least 100 identical isolate sequences in GISAID. Phylogenetic trees were reconstructed using IQ-TREE and MrBayes softwares. A median-join network was reconstructed using PopART software. Selection analyses were conducted using site model of PAML software. Results: The phylogenetic tree of the spike DNA sequences revealed that the majority of variants belonged to three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. The median network showed that these lineages had at least six major diversifying centers. The spike DNA sequences of these diversifying centers had the representative accession IDs (EPI_ISL_) of 16040256 (BN.1.2), 15970311 (BA.5), 16028739 (BA.5.11), 16028774 (BQ.1), 16027638 (BQ.1.1.23), and 16044705 (XBB.1.5). Selection analyses revealed 26 amino-acid sites under positive selection. These sites included L5, V83, W152, G181, N185, V213, H245, Y248, D253, S255, S256, G257, R346, R408, K444, V445, G446, N450, L452, N460, F486, Q613, Q675, T883, P1162, and V1264. Conclusion: The spike proteins of SARS-CoV-2 from December 2022 to February 2023 were characterized by a swarm of variants that were evolved from three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. These lineages had at least six diversifying centers. Selection analysis identified 26 amino acid sites were under positive selection. Continued surveillance and research are necessary to monitor the evolution and potential impact of these variants on public health.\nA positive association between gut microbiota diversity and vertebrate host performance in a field experiment\nAuthors: Härer, A.; Thompson, K. A.; Schluter, D.; Rennison, D. J.\nScore: 16.9, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549938\nThe vertebrate gut microbiota is a critical determinant of organismal function, yet it remains unclear if and how gut microbial communities affect host fitness under natural conditions. Here, we investigate associations between growth rate (a fitness proxy) and gut microbiota diversity and composition in a field experiment with threespine stickleback fish (Gasterosteus aculeatus). We detected on average 63% more bacterial taxa in the guts of high-fitness fish compared to low-fitness fish (i.e., higher -diversity), suggesting that higher diversity promotes host growth. The microbial communities of high-fitness fish had higher similarity (i.e., lower {beta}-diversity) than low-fitness fish, supporting the Anna Karenina principle-- that there are fewer ways to have a functional microbiota than a dysfunctional microbiota. Our findings provide a basis for functional tests of the fitness consequences of host-microbiota interactions. Significance statementThe vertebrate gut microbiota is important for many aspects of their hosts biology--such as nutrient metabolism and defense against pathogens--that could ultimately affect host fitness. However, studies investigating the effects of gut microbiota composition on vertebrate host fitness under natural conditions remain exceedingly rare. We tested for associations between gut microbiota diversity and growth rate (a fitness proxy) in threespine stickleback fish reared in large outdoor ponds. We found evidence that a more diverse gut microbiota was predictive of higher growth rate and therefore increased host fitness. Notably, high-fitness fish had higher gut microbiota similarity to one another than did low-fitness fish, providing experimental evidence for the Anna Karenina principle--that there are fewer ways to have a functional microbiota than a dysfunctional microbiota--as it relates to host fitness.\nEvolution of compound eye morphology underlies differences in vision between closely related Drosophila species\nAuthors: Buffry, A. D.; Currea, J. P.; Franke, F. A.; Palavalli-Nettimi, R.; Bodey, A. P.; Rau, C.; Samadi, N.; Gstöhl, S. J.; Schlepütz, C. M.; McGregor, A. P.; Sumner-Rooney, L.; Theobald, J.; Kittelmann, M.\nScore: 17.2, Published: 2023-07-16 DOI: 10.1101/2023.07.16.549164\nInsects have evolved complex visual systems and display an astonishing range of adaptations for diverse ecological niches. Differences in eye size within and between Drosophila species provide the opportunity to study the impact of eye structure on vision. Here we further explored differences in D. mauritiana and its sibling species D. simulans and confirmed that D. mauritiana have rapidly evolved larger eyes as a result of more and wider ommatidia than D. simulans since their recent common ancestor. The functional impact of eye size, and specifically ommatidia size, is often only estimated based on the rigid surface morphology of the compound eye. Therefore, we used 3D synchrotron radiation tomography to measure optical parameters in 3D, predict optical capacity, and compare the modelled vision to in vivo optomotor responses. Our optical models predicted higher contrast sensitivity for D. mauritiana, which we verified by presenting sinusoidal gratings to tethered flies in a flight arena. Similarly, we confirmed the higher spatial acuity predicted for Drosophila simulans with smaller ommatidia and found evidence for higher temporal resolution.\nOn the structure of an evolutionary transition: dependence and cohesion in incipient endosymbioses\nAuthors: Athreya, G.; Czuppon, P.; Gokhale, C. S.\nScore: 8.8, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549359\nHow did biological organisms become so complex? Endosymbiosis is the prototypical example of an egalitarian evolutionary transition and is central to the origins of many complex biological systems. Why do only some symbioses undergo evolutionary transitions, and how does the host-symbiont relationship change during this process? Here, we characterise endosymbiosis by two emergent collective-level properties: the relationship between the host and symbiont (mutual dependence) and reproduction (reproductive cohesion). Using methods from adaptive dynamics, we study the co-evolution of the traits underlying these properties. Our central result is a robust demonstration that even when investments in dependence and cohesion are uncorrelated, host-symbiont mutual dependence arises faster than reproductive cohesion. Further, we show that the collectives formation and shared fate, coupled with different generation times of the host and symbiont, leads to an emergent asymmetry in how much they invest in the collective. Lastly, we account for biological realism that previous models have ignored and show that this can preclude a successful evolutionary transition towards stable endosymbiosis. Together, this work uncovers a fundamental property of endosymbioses, highlighting the immense effects of simple ecological factors and providing a clear way forward for theoretical and empirical investigations.\nThe super-pangenome of Populus unveil genomic facets for adaptation and diversification in widespread forest trees\nAuthors: Shi, T.; Zhang, X.; Hou, Y.; Jiang, Y.; Jia, C.; Lai, Q.; Dan, X.; Feng, J.; Feng, J.; Ma, T.; Wu, J.; Liu, S.; Zhang, L.; Long, Z.; Zhang, Y.; Zhang, J.; Chen, L.; R. Street, N.; K. Ingvarsson, P.; Liu, J.; Yin, T.; Wang, J.\nScore: 8.4, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549473\nUnderstanding the underlying mechanisms between genome evolution and phenotypic and adaptive innovations is a key goal of evolutionary studies. Poplars are the worlds most widely distributed and cultivated trees, with extensive phenotypic diversity and environmental adaptability. Here we report a genus-level super-pangenome of 19 Populus genomes. After integrating pan-genomes with transcriptomes, methylomes and chromatin accessibility mapping, we reveal that the evolutionary fate of pan-genes and duplicated genes are largely associated with local genomic landscapes of regulatory and epigenetic architectures. Further comparative genomic analyses enabled to identify 142,202 structural variations (SVs) across species, which overlap with substantial genes and play key roles in both phenotypic and adaptive divergence. We experimentally validated a [~]180 bp presence/absence variant located in the promoter of the CUC2 gene, which contributed critically to leaf serration divergence between species. Together, this first super-pangenome resource in forest trees will not only accelerate molecular functional studies and genetic breeding of this globally important tree genus, but also lays a foundation for our understanding of tree biology.\nMultiple axes of visual system diversity in Ithomiini, an ecologically diverse tribe of mimetic butterflies\nAuthors: Wainwright, J. B.; Schofield, C.; Conway, M.; Phillips, D.; Martin-Silverstone, E.; Brodrick, E. A.; Cicconardi, F.; How, M. J.; Roberts, N. W.; Montgomery, S.\nScore: 8.4, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549450\nThe striking structural variation seen in arthropod visual systems can be explained by availability of light within habitats coupled with developmental and physiological constraints. However, little is currently known about how fine-scale variation in visual structures arise across shorter evolutionary and ecological scales. In this study, we characterise patterns of interspecific, intraspecific and intraindividual variation in the visual system of four ithomiine butterfly species. These species are part of a diverse Neotropical radiation where changes in mimetic colouration are associated with fine-scale shifts in ecology, such as microhabitat preference. By using a combination of selection analyses on visual opsin sequences, in-vivo ophthalmoscopy, micro-computed tomography (micro-CT), immunohistochemistry, confocal microscopy, and neural tracing, we quantify and describe physiological, anatomical, and molecular traits involved in visual processing. Using these data, we provide evidence of substantial variation within the visual systems of Ithomiini, including i) habitat-associated relaxed selection on visual opsins, ii) interspecific shifts in visual system physiology and anatomy, and iii) extensive sexual dimorphism, including the complete absence of a butterfly-specific optic neuropil in the males of some species. We conclude that considerable visual system variation can exist within diverse insect radiations, hinting at the evolutionary lability of these systems to rapidly develop specialisations to distinct visual ecologies, with selection acting at both the perceptual, processing, and molecular level. Summary statementPhysiological, anatomical, and molecular evidence of extensive visual system variation within a diverse butterfly radiation, hinting at the lability of visual systems to evolve specialisations to distinct visual environments.\nUsing big sequencing data to identify chronic SARS-Coronavirus-2 infections\nAuthors: Harari, S.; Miller, D.; Fleishon, S.; Burstein, D.; Stern, A.\nScore: 26.7, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549184\nThe evolution of SARS-Coronavirus-2 (SARS-CoV-2) has been characterized by the periodic emergence of highly divergent variants, many of which may have arisen during chronic infections of immunocompromised individuals. Here, we harness a global phylogeny of [~]11.7 million SARS-CoV-2 genomes and search for clades composed of sequences with identical metadata (location, age, and sex) spanning more than 21 days. We postulate that such clades represent repeated sampling from the same chronically infected individual. A set of 271 such chronic-like clades was inferred, and displayed signatures of an elevated rate of adaptive evolution, in line with validated chronic infections. More than 70% of adaptive mutations present in currently circulating variants are found in BA.1 chronic-like clades that predate the circulating variants by months, demonstrating the predictive nature of such clades. We find that in chronic-like clades the probability of observing adaptive mutations is approximately 10-20 higher than that in global transmission chains. We next employ language models to find mutations most predictive of chronic infections and use them to infer hundreds of additional chronic-like clades in the absence of metadata and phylogenetic information. Our proposed approach presents an innovative method for mining extensive sequencing data and providing valuable insights into future evolutionary patterns.\nThe influence of gene flow on population viability in an isolated urban caracal population\nAuthors: Kyriazis, C. C.; Serieys, L. E.; Bishop, J. M.; Drouilly, M.; Viljoen, S.; Wayne, R. K.; Lohmueller, K. E.\nScore: 7.7, Published: 2023-07-23 DOI: 10.1101/2023.07.20.549918\nWildlife populations are becoming increasingly fragmented by anthropogenic development. Such small and isolated populations often face an elevated risk of extinction, in part due to inbreeding depression. Here, we examine the genomic consequences of urbanization in a caracal (Caracal caracal) population that has become isolated in the Cape Peninsula region of the city of Cape Town, South Africa and is thought to number [~]50 individuals. We document low levels of migration into the population over the past [~]75 years, with an estimated rate of 1.3 effective migrants per generation. As a consequence of this isolation and small population size, levels of inbreeding are elevated in the contemporary Cape Peninsula population (mean FROH\u003e1Mb=0.20). Inbreeding primarily manifests as long runs of homozygosity \u003e10Mb, consistent with the effects of isolation due to the rapid recent growth of Cape Town. To explore how reduced migration and elevated inbreeding may impact future population dynamics, we parameterized an eco-evolutionary simulation model. We find that if migration rates do not change in the future, the population is expected to decline only slightly, with a low projected risk of extinction. However, if migration rates decline or anthropogenic mortality rates increase, the potential risk of extinction is greatly elevated. To avert a population decline, we suggest that translocating migrants into the Cape Peninsula to initiate a genetic rescue may be warranted in the near future. Our analysis highlights the utility of genomic datasets coupled with computational simulation models for investigating the influence of gene flow on population viability.\n",
  "wordCount" : "2780",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547326">
        <p class="paperTitle">Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.</p>
        <p class="info">Score: 286.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547326' target='https://doi.org/10.1101/2023.07.03.547326'> 10.1101/2023.07.03.547326</a></p>
        <p class="abstract">The evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549758">
        <p class="paperTitle">Machine learning illuminates how diet influences the evolution of yeast galactose metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harrison, M.-C.; Ubbelohde, E. J.; LaBella, A. L.; Opulente, D. A.; Wolters, J. F.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Hittinger, C. T.; Rokas, A.</p>
        <p class="info">Score: 22.9, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549758' target='https://doi.org/10.1101/2023.07.20.549758'> 10.1101/2023.07.20.549758</a></p>
        <p class="abstract">How genomic differences contribute to phenotypic differences across species is a major question in biology. The recently characterized genomes, isolation environments, and qualitative patterns of growth on 122 sources and conditions of 1,154 strains from 1,049 fungal species (nearly all known) in the subphylum Saccharomycotina provide a powerful, yet complex, dataset for addressing this question. In recent years, machine learning has been successfully used in diverse analyses of biological big data. Using a random forest classification algorithm trained on these genomic, metabolic, and/or environmental data, we predicted growth on several carbon sources and conditions with high accuracy from presence/absence patterns of genes and of growth in other conditions. Known structural genes involved in assimilation of these sources were important features contributing to prediction accuracy, whereas isolation environmental data were poor predictors. By further examining growth on galactose, we found that it can be predicted with high accuracy from either genomic (92.6%) or growth data in 120 other conditions (83.3%) but not from isolation environment data (65.7%). When we combined genomic and growth data, we noted that prediction accuracy was even higher (93.4%) and that, after the GALactose utilization genes, the most important feature for predicting growth on galactose was growth on galactitol. These data raised the hypothesis that several species in two orders, Serinales and Pichiales (containing Candida auris and the genus Ogataea, respectively), have an alternative galactose utilization pathway because they lack the GAL genes. Growth and biochemical assays of several of these species confirmed that they utilize galactose through an oxidoreductive D-galactose pathway, rather than the canonical GAL pathway. We conclude that machine learning is a powerful tool for investigating the evolution of the yeast genotype-phenotype map and that it can help uncover novel biology, even in well-studied traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.549423">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.549423" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.549423">
        <p class="paperTitle">Phylogeny and evolution of the SARS-CoV-2 spike gene from December 2022 to February 2023</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.549423" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.549423" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kao, H.-W.; Kao, H.-W.</p>
        <p class="info">Score: 20.8, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.549423' target='https://doi.org/10.1101/2023.07.23.549423'> 10.1101/2023.07.23.549423</a></p>
        <p class="abstract">Background: By the end of 2022, new variants of SARS-CoV-2, such as BQ.1.1.10, BA.4.6.3, XBB, and CH.1.1, emerged with higher fitness than BA.5. Methods: The file (spikeprot0304), which contains spike protein sequences, isolates collected before March, 4, 2023, was downloaded from Global Initiative on Sharing All Influenza Data (GISAID). A total of 188 different spike protein sequences were chosen, of which their isolates were collected from December 2022 to February 2023. These sequences did not contain undetermined amino acid X, and each spike protein sequence had at least 100 identical isolate sequences in GISAID. Phylogenetic trees were reconstructed using IQ-TREE and MrBayes softwares. A median-join network was reconstructed using PopART software. Selection analyses were conducted using site model of PAML software. Results: The phylogenetic tree of the spike DNA sequences revealed that the majority of variants belonged to three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. The median network showed that these lineages had at least six major diversifying centers. The spike DNA sequences of these diversifying centers had the representative accession IDs (EPI_ISL_) of 16040256 (BN.1.2), 15970311 (BA.5), 16028739 (BA.5.11), 16028774 (BQ.1), 16027638 (BQ.1.1.23), and 16044705 (XBB.1.5). Selection analyses revealed 26 amino-acid sites under positive selection. These sites included L5, V83, W152, G181, N185, V213, H245, Y248, D253, S255, S256, G257, R346, R408, K444, V445, G446, N450, L452, N460, F486, Q613, Q675, T883, P1162, and V1264. Conclusion: The spike proteins of SARS-CoV-2 from December 2022 to February 2023 were characterized by a swarm of variants that were evolved from three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. These lineages had at least six diversifying centers. Selection analysis identified 26 amino acid sites were under positive selection. Continued surveillance and research are necessary to monitor the evolution and potential impact of these variants on public health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549938">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549938" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549938">
        <p class="paperTitle">A positive association between gut microbiota diversity and vertebrate host performance in a field experiment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549938" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549938" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Härer, A.; Thompson, K. A.; Schluter, D.; Rennison, D. J.</p>
        <p class="info">Score: 16.9, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549938' target='https://doi.org/10.1101/2023.07.20.549938'> 10.1101/2023.07.20.549938</a></p>
        <p class="abstract">The vertebrate gut microbiota is a critical determinant of organismal function, yet it remains unclear if and how gut microbial communities affect host fitness under natural conditions. Here, we investigate associations between growth rate (a fitness proxy) and gut microbiota diversity and composition in a field experiment with threespine stickleback fish (Gasterosteus aculeatus). We detected on average 63% more bacterial taxa in the guts of high-fitness fish compared to low-fitness fish (i.e., higher -diversity), suggesting that higher diversity promotes host growth. The microbial communities of high-fitness fish had higher similarity (i.e., lower {beta}-diversity) than low-fitness fish, supporting the Anna Karenina principle-- that there are fewer ways to have a functional microbiota than a dysfunctional microbiota. Our findings provide a basis for functional tests of the fitness consequences of host-microbiota interactions.

Significance statementThe vertebrate gut microbiota is important for many aspects of their hosts biology--such as nutrient metabolism and defense against pathogens--that could ultimately affect host fitness. However, studies investigating the effects of gut microbiota composition on vertebrate host fitness under natural conditions remain exceedingly rare. We tested for associations between gut microbiota diversity and growth rate (a fitness proxy) in threespine stickleback fish reared in large outdoor ponds. We found evidence that a more diverse gut microbiota was predictive of higher growth rate and therefore increased host fitness. Notably, high-fitness fish had higher gut microbiota similarity to one another than did low-fitness fish, providing experimental evidence for the Anna Karenina principle--that there are fewer ways to have a functional microbiota than a dysfunctional microbiota--as it relates to host fitness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549164">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549164" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549164">
        <p class="paperTitle">Evolution of compound eye morphology underlies differences in vision between closely related Drosophila species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549164" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549164" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buffry, A. D.; Currea, J. P.; Franke, F. A.; Palavalli-Nettimi, R.; Bodey, A. P.; Rau, C.; Samadi, N.; Gstöhl, S. J.; Schlepütz, C. M.; McGregor, A. P.; Sumner-Rooney, L.; Theobald, J.; Kittelmann, M.</p>
        <p class="info">Score: 17.2, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549164' target='https://doi.org/10.1101/2023.07.16.549164'> 10.1101/2023.07.16.549164</a></p>
        <p class="abstract">Insects have evolved complex visual systems and display an astonishing range of adaptations for diverse ecological niches. Differences in eye size within and between Drosophila species provide the opportunity to study the impact of eye structure on vision. Here we further explored differences in D. mauritiana and its sibling species D. simulans and confirmed that D. mauritiana have rapidly evolved larger eyes as a result of more and wider ommatidia than D. simulans since their recent common ancestor. The functional impact of eye size, and specifically ommatidia size, is often only estimated based on the rigid surface morphology of the compound eye. Therefore, we used 3D synchrotron radiation tomography to measure optical parameters in 3D, predict optical capacity, and compare the modelled vision to in vivo optomotor responses. Our optical models predicted higher contrast sensitivity for D. mauritiana, which we verified by presenting sinusoidal gratings to tethered flies in a flight arena. Similarly, we confirmed the higher spatial acuity predicted for Drosophila simulans with smaller ommatidia and found evidence for higher temporal resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549359">
        <p class="paperTitle">On the structure of an evolutionary transition: dependence and cohesion in incipient endosymbioses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Athreya, G.; Czuppon, P.; Gokhale, C. S.</p>
        <p class="info">Score: 8.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549359' target='https://doi.org/10.1101/2023.07.17.549359'> 10.1101/2023.07.17.549359</a></p>
        <p class="abstract">How did biological organisms become so complex? Endosymbiosis is the prototypical example of an egalitarian evolutionary transition and is central to the origins of many complex biological systems. Why do only some symbioses undergo evolutionary transitions, and how does the host-symbiont relationship change during this process? Here, we characterise endosymbiosis by two emergent collective-level properties: the relationship between the host and symbiont (mutual dependence) and reproduction (reproductive cohesion). Using methods from adaptive dynamics, we study the co-evolution of the traits underlying these properties. Our central result is a robust demonstration that even when investments in dependence and cohesion are uncorrelated, host-symbiont mutual dependence arises faster than reproductive cohesion. Further, we show that the collectives formation and shared fate, coupled with different generation times of the host and symbiont, leads to an emergent asymmetry in how much they invest in the collective. Lastly, we account for biological realism that previous models have ignored and show that this can preclude a successful evolutionary transition towards stable endosymbiosis. Together, this work uncovers a fundamental property of endosymbioses, highlighting the immense effects of simple ecological factors and providing a clear way forward for theoretical and empirical investigations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549473">
        <p class="paperTitle">The super-pangenome of Populus unveil genomic facets for adaptation and diversification in widespread forest trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, T.; Zhang, X.; Hou, Y.; Jiang, Y.; Jia, C.; Lai, Q.; Dan, X.; Feng, J.; Feng, J.; Ma, T.; Wu, J.; Liu, S.; Zhang, L.; Long, Z.; Zhang, Y.; Zhang, J.; Chen, L.; R. Street, N.; K. Ingvarsson, P.; Liu, J.; Yin, T.; Wang, J.</p>
        <p class="info">Score: 8.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549473' target='https://doi.org/10.1101/2023.07.18.549473'> 10.1101/2023.07.18.549473</a></p>
        <p class="abstract">Understanding the underlying mechanisms between genome evolution and phenotypic and adaptive innovations is a key goal of evolutionary studies. Poplars are the worlds most widely distributed and cultivated trees, with extensive phenotypic diversity and environmental adaptability. Here we report a genus-level super-pangenome of 19 Populus genomes. After integrating pan-genomes with transcriptomes, methylomes and chromatin accessibility mapping, we reveal that the evolutionary fate of pan-genes and duplicated genes are largely associated with local genomic landscapes of regulatory and epigenetic architectures. Further comparative genomic analyses enabled to identify 142,202 structural variations (SVs) across species, which overlap with substantial genes and play key roles in both phenotypic and adaptive divergence. We experimentally validated a [~]180 bp presence/absence variant located in the promoter of the CUC2 gene, which contributed critically to leaf serration divergence between species. Together, this first super-pangenome resource in forest trees will not only accelerate molecular functional studies and genetic breeding of this globally important tree genus, but also lays a foundation for our understanding of tree biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549450" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549450">
        <p class="paperTitle">Multiple axes of visual system diversity in Ithomiini, an ecologically diverse tribe of mimetic butterflies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wainwright, J. B.; Schofield, C.; Conway, M.; Phillips, D.; Martin-Silverstone, E.; Brodrick, E. A.; Cicconardi, F.; How, M. J.; Roberts, N. W.; Montgomery, S.</p>
        <p class="info">Score: 8.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549450' target='https://doi.org/10.1101/2023.07.18.549450'> 10.1101/2023.07.18.549450</a></p>
        <p class="abstract">The striking structural variation seen in arthropod visual systems can be explained by availability of light within habitats coupled with developmental and physiological constraints. However, little is currently known about how fine-scale variation in visual structures arise across shorter evolutionary and ecological scales. In this study, we characterise patterns of interspecific, intraspecific and intraindividual variation in the visual system of four ithomiine butterfly species. These species are part of a diverse Neotropical radiation where changes in mimetic colouration are associated with fine-scale shifts in ecology, such as microhabitat preference. By using a combination of selection analyses on visual opsin sequences, in-vivo ophthalmoscopy, micro-computed tomography (micro-CT), immunohistochemistry, confocal microscopy, and neural tracing, we quantify and describe physiological, anatomical, and molecular traits involved in visual processing. Using these data, we provide evidence of substantial variation within the visual systems of Ithomiini, including i) habitat-associated relaxed selection on visual opsins, ii) interspecific shifts in visual system physiology and anatomy, and iii) extensive sexual dimorphism, including the complete absence of a butterfly-specific optic neuropil in the males of some species. We conclude that considerable visual system variation can exist within diverse insect radiations, hinting at the evolutionary lability of these systems to rapidly develop specialisations to distinct visual ecologies, with selection acting at both the perceptual, processing, and molecular level.

Summary statementPhysiological, anatomical, and molecular evidence of extensive visual system variation within a diverse butterfly radiation, hinting at the lability of visual systems to evolve specialisations to distinct visual environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549184">
        <p class="paperTitle">Using big sequencing data to identify chronic SARS-Coronavirus-2 infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harari, S.; Miller, D.; Fleishon, S.; Burstein, D.; Stern, A.</p>
        <p class="info">Score: 26.7, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549184' target='https://doi.org/10.1101/2023.07.16.549184'> 10.1101/2023.07.16.549184</a></p>
        <p class="abstract">The evolution of SARS-Coronavirus-2 (SARS-CoV-2) has been characterized by the periodic emergence of highly divergent variants, many of which may have arisen during chronic infections of immunocompromised individuals. Here, we harness a global phylogeny of [~]11.7 million SARS-CoV-2 genomes and search for clades composed of sequences with identical metadata (location, age, and sex) spanning more than 21 days. We postulate that such clades represent repeated sampling from the same chronically infected individual. A set of 271 such chronic-like clades was inferred, and displayed signatures of an elevated rate of adaptive evolution, in line with validated chronic infections. More than 70% of adaptive mutations present in currently circulating variants are found in BA.1 chronic-like clades that predate the circulating variants by months, demonstrating the predictive nature of such clades. We find that in chronic-like clades the probability of observing adaptive mutations is approximately 10-20 higher than that in global transmission chains. We next employ language models to find mutations most predictive of chronic infections and use them to infer hundreds of additional chronic-like clades in the absence of metadata and phylogenetic information. Our proposed approach presents an innovative method for mining extensive sequencing data and providing valuable insights into future evolutionary patterns.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549918">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549918" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549918">
        <p class="paperTitle">The influence of gene flow on population viability in an isolated urban caracal population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549918" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549918" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kyriazis, C. C.; Serieys, L. E.; Bishop, J. M.; Drouilly, M.; Viljoen, S.; Wayne, R. K.; Lohmueller, K. E.</p>
        <p class="info">Score: 7.7, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549918' target='https://doi.org/10.1101/2023.07.20.549918'> 10.1101/2023.07.20.549918</a></p>
        <p class="abstract">Wildlife populations are becoming increasingly fragmented by anthropogenic development. Such small and isolated populations often face an elevated risk of extinction, in part due to inbreeding depression. Here, we examine the genomic consequences of urbanization in a caracal (Caracal caracal) population that has become isolated in the Cape Peninsula region of the city of Cape Town, South Africa and is thought to number [~]50 individuals. We document low levels of migration into the population over the past [~]75 years, with an estimated rate of 1.3 effective migrants per generation. As a consequence of this isolation and small population size, levels of inbreeding are elevated in the contemporary Cape Peninsula population (mean FROH&gt;1Mb=0.20). Inbreeding primarily manifests as long runs of homozygosity &gt;10Mb, consistent with the effects of isolation due to the rapid recent growth of Cape Town. To explore how reduced migration and elevated inbreeding may impact future population dynamics, we parameterized an eco-evolutionary simulation model. We find that if migration rates do not change in the future, the population is expected to decline only slightly, with a low projected risk of extinction. However, if migration rates decline or anthropogenic mortality rates increase, the potential risk of extinction is greatly elevated. To avert a population decline, we suggest that translocating migrants into the Cape Peninsula to initiate a genetic rescue may be warranted in the near future. Our analysis highlights the utility of genomic datasets coupled with computational simulation models for investigating the influence of gene flow on population viability.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
