<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects
Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.
Score: 78.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects
Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.
Score: 78." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects
Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.
Score: 78."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world\u0026#39;s most common underwater architects\nAuthors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.\nScore: 78.",
  "keywords": [
    
  ],
  "articleBody": " Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world's most common underwater architects\nAuthors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.\nScore: 78.4, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572910\nCaddisflies (Trichoptera) are among the most diverse groups of freshwater animals with more than 16,000 described species. They play an outsized role in freshwater ecology and environmental engineering in streams, rivers, and lakes. Because of this, they are frequently used as indicator organisms in biomonitoring programs. Despite their importance, key questions concerning the evolutionary history of caddisflies, such as the timing and origin of larval case-making, have been unanswered due to the lack of a well-resolved phylogenetic tree. To shed light on these questions in Trichoptera, we estimated a phylogenetic tree using a combination of transcriptomes and targeted enrichment data for 206 species, representing 48 of 52 extant families and 174 genera. We calibrated and dated the tree with a set of 33 carefully selected fossils. The first caddisflies originated in the Permian and the major suborders began to diversify in the Triassic. Ancestral state reconstruction and diversification analysis revealed that portable case-making evolved in three separate lineages and shifts in diversification occurred in concert with key evolutionary innovations other than case-making.\nDistinct ancient structural polymorphisms control heterodichogamy in walnuts and hickories\nAuthors: Groh, J. S.; Vik, D. C.; Stevens, K. A.; Brown, P. J.; Langley, C. H.; Coop, G.\nScore: 17.1, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573205\nThe maintenance of stable mating type polymorphisms is a classic example of balancing selection, underlying the nearly ubiquitous 50/50 sex ratio in species with separate sexes. One lesser known but intriguing example of a balanced mating polymorphism in angiosperms is heterodichogamy - polymorphism for opposing directions of dichogamy (temporal separation of male and female function in hermaphrodites) within a flowering season. This mating system is common throughout Juglandaceae, the family that includes globally important and iconic nut and timber crops - walnuts (Juglans), as well as pecan and other hickories (Carya). We show that two distinct genetic systems control heterodichogamous mating systems in these genera. In each genus, we find ancient (\u003e50Mya) structural variants that segregate as trans-species polymorphisms. The Juglans locus maps to a ca. 20kb structural variant adjacent to gene with likely function in the trehalose-6-phosphate signaling pathway, which is known to regulate flowering time in Arabidopsis and inflorescence architecture in maize. Notably, an insertion in the dominant (protogynous) haplotype contains 8-12 complex tandem duplicates of the 3 UTR of the gene, and transpecies polymorphism within the gene is restricted to the 3 UTR. In contrast, the Carya heterodichogamy locus maps to a ca. 200-450 kb cluster of tightly linked polymorphisms across 20 genes, some of which have known roles in flowering and are differentially expressed between morphs in floral primordia. The dominant (protogynous) haplotype in pecan, which is nearly always heterozygous and appears to rarely recombine, shows reduced genetic diversity and is over twice as long as its recessive counterpart due to proliferation of multiple transposable element families. We did not detect either genetic system in another heterodichogamous genus within the Juglandaceae, suggesting that additional genetic systems for heterodichogamy may yet remain undiscovered.\nEvolution of Multicellularity Genes in the Lead Up to the Great Oxidation Event\nAuthors: Boden, J. S.; Nieves-Morion, M.; Nurnberg, D. J.; Arevalo, S.; Flores, E.; Sanchez-Baracaldo, P.\nScore: 15.1, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573081\nCyanobacteria are among the most morphologically diverse prokaryotic phyla on Earth. Their morphotypes range from unicellular to multicellular filaments, yet mechanisms underlying the evolution of filamentous morphologies remain unknown. Here, we implement phylogenomic, Bayesian molecular clock and gene-tree-species-tree reconciliation analyses to estimate when genes encoding cell-cell joining structures first evolved. We also characterise septal structures and measure intercellular communication rates in non-model and early-branching filamentous strains. Our results suggest that genes encoding septal proteins (namely sepJ, sepI, and fraE) and potentially pattern formation (hetR) evolved in the Neoarchaean [~]2.6-2.7 billion years (Ga) ago. Later, at the start of the Great Oxygenation Event [~]2.5 Ga, genes involved in cellular differentiation (namely hetZ, patU3 and hglK) appeared. Our results predict that early-branching lineages like Pseudanabaena were capable of intercellular communication, but further innovations in cellular differentiation were needed to drive ecological expansion on a scale large enough to permanently oxygenate Earths atmosphere.\nComplex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders\nAuthors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.\nScore: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573503\nEye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size. By contrast, animals possessing larger visual systems can exhibit variable eye sizes within individuals. Spiders have up to four pairs of eyes whose size often varies, but their ontogenetic, static, and evolutionary allometry has not yet been studied in a comparative context. We report variable evolutionary and developmental dynamics in eye size across 1098 individuals in 39 species and 8 families, indicating selective pressures and constraints driving the evolution of different eye pairs and lineages. We observed variation in the relationship between eye and body size not only between taxa and visual ecologies, but between eye pairs within species, indicating that growth dynamics are variable and can be divergently adapted in particular eye pairs. Supplementing our sampling with a phylogenetically comprehensive dataset recently published by Wolff et al. (2022), we confirmed these findings across more than 400 species, found that ecological factors such as visual hunting, web building, and diurnality can also impact eye diameter, and identified significant allometric shifts across spider phylogeny using an unbiased approach, many of which coincide with ecological changes such as visual pursuit hunting strategies. This study represents the first detailed comparative exploration of visual allometry in spiders, revealing striking differences in eye growth not only between families but also within species. Our findings shed light on the relationship between spider visual systems and their diverse ecologies, and how spiders exploit the modular nature of their visual systems to balance selective pressures and optical and energetic constraints.\nBrood care in shell-dwelling cichlids is timed by independent maternal and larval clocks\nAuthors: Parker, A. V.; Stemmer, M.; Graetsch, S.; Dorigo, A.; Rodriguez Ramirez, O.; Adel, A.; Jordan, A. L.; Baier, H.\nScore: 11.9, Published: 2023-12-24 DOI: 10.1101/2023.12.22.573114\nSuccessful brood care requires a finely tuned interplay between caregiving parent and offspring. Cichlids are a diverse group of teleost fish known for their extensive parental care of altricial young. Female Lamprologus ocellatus lay their eggs in the protective confines of an abandoned snail shell, where they raise them to free-swimming larvae. Here we investigated whether the transition to larval independence is driven by the behavior of the mother, the offspring, or an interaction between the two. Using 3D printed shells with a window, we were able to observe the behavior of both mother and offspring inside the nest using automated object detection and tracking. We found that, once hatched, the larvae remain in the deep chambers of the shell, and at 9 dpf, actively emerge from the shell during the day. An inversion of larval phototactic behavior from dark- to light-seeking leads up to this emergence. Removal of the mother causes premature emergence of the larvae by a few days. The natural schedule (emergence at 9 dpf) is restored if the larvae are supplied with fresh water in the mothers absence. In cross-fostering experiments, we found that swapping out the mothers own clutch for older larvae forces a delay in emergence, as the foster mother prevents the larvae from leaving at 9 dpf. We conclude that larval and maternal behavior are each controlled by independent internal clocks, which, although normally synchronized, can be experimentally manipulated and brought into conflict. This study has thus revealed an innate sequence of behavioral adaptations that orchestrate brood care in shell-dwelling cichlids.\nOne million years of solitude: the rapid evolution of de novo protein structure and complex\nAuthors: Chen, J.; Li, Q.; Xia, S.; Arsala, D.; Sosa, D.; Wang, D.; Long, M.\nScore: 6.3, Published: 2023-12-26 DOI: 10.1101/2023.12.24.573215\nRecent studies have established that de novo genes, evolving from non-coding sequences, enhance protein diversity through a stepwise process. However, the pattern and rate of their structural evolution over time remain unclear. Here, we addressed these issues within a timeframe of ~1 million years (97% of de novo genes). We found that de novo genes evolve faster than gene duplicates in the intrinsic disordered regions (IDRs, such as random coils), secondary structural elements (such as -helix and {beta}-strand), hydrophobicity, and molecular recognition features (MoRFs). Specifically, we observed an 8-14% reduction in random coils and IDR lengths per million years per protein, and a 2.3-6.5% increase in structured elements, hydrophobicity, and MoRFs. These patterns of structural evolution align with changes in amino acid composition over time. We also revealed significantly higher positive charges but smaller molecular weights for de novo proteins than duplicates. Tertiary structure predictions demonstrated that de novo proteins, though not typically well-folded on their own, readily form low-energy, compact complexes with extensive residue contacts and conformational flexibility, suggesting a faster-binding scenario in de novo proteins to promote interaction. Our findings illuminate the rapid evolution of protein structure in the early life of de novo proteins, originating from noncoding sequences, highlighting their quick transformation into active, complex-forming components within a remarkably short evolutionary timeframe.\nHorizontal transfers between fungal Fusarium species contributed to successive outbreaks of coffee wilt disease\nAuthors: Peck, L. D.; Llewellyn, T.; Bennetot, B.; O'Donnell, S.; Nowell, R. W.; Ryan, M. J.; Flood, J.; Rodriguez de la Vega, R. C.; Ropars, J.; Giraud, T.; Spanu, P. D.; Barraclough, T. G.\nScore: 5.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572981\nOutbreaks of fungal disease have devastated plants and animals throughout history. Over the past century, the repeated emergence of coffee wilt disease caused by the fungal pathogen Fusarium xylarioides severely impacted coffee production across sub-Saharan Africa. To improve the disease management of such pathogens, it is crucial to understand their genetic structure and evolutionary potential. We compared the genomes of 13 historic strains spanning six decades and multiple disease outbreaks to investigate population structure and host specialisation. We found F. xylarioides comprises at least four distinct lineages: one host-specific to Coffea arabica, one to C. canephora var. robusta, and two historic lineages isolated from various Coffea species. Mapping variation onto a new long-read reference genome showed that host-specificity appears to be acquired through horizontal transfer of effector genes from members of the F. oxysporum species complex. This species complex is known to cause wilt disease in over 100 plant species. Multiple transfers into the F. xylarioides populations matched to different parts of the F. oxysporum mobile pathogenicity chromosome and were enriched in effector genes and transposons. Effector genes in this region and other horizontally transferred carbohydrate-active enzymes important in the breakdown of plant cell walls were shown by transcriptomics to be highly expressed during infection of C. arabica by the fungal arabica strains. Widespread sharing of specific transposons between F. xylarioides and F. oxysporum, and the presence of large Starship elements, indicate that transposons were involved in horizontal transfers. Our results support the hypothesis that horizontal gene transfers contributed to the repeated emergence of this fungal disease.\nGenome evolution is surprisingly predictable after initial hybridization\nAuthors: Langdon, Q. K.; Groh, J. S.; Aguillon, S.; Powell, D. L.; Gunn, T. R.; Payne, C. Y.; Baczenas, J. J.; Donny, A.; Dodge, T. O.; Du, K.; Schartl, M.; Rios-Cardenas, O.; Gutierrez-Rodriguez, C.; Morris, M.; Schumer, M.\nScore: 4.8, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572897\nOver the past two decades, evolutionary biologists have come to appreciate that hybridization, or genetic exchange between distinct lineages, is remarkably common - not just in particular lineages but in taxonomic groups across the tree of life. As a result, the genomes of many modern species harbor regions inherited from related species. This observation has raised fundamental questions about the degree to which the genomic outcomes of hybridization are repeatable and the degree to which natural selection drives such repeatability. However, a lack of appropriate systems to answer these questions has limited empirical progress in this area. Here, we leverage independently formed hybrid populations between the swordtail fish Xiphophorus birchmanni and X. cortezi to address this fundamental question. We find that local ancestry in one hybrid population is remarkably predictive of local ancestry in another, demographically independent hybrid population. Applying newly developed methods, we can attribute much of this repeatability to strong selection in the earliest generations after initial hybridization. We complement these analyses with time-series data that demonstrates that ancestry at regions under selection has remained stable over the past [~]40 generations of evolution. Finally, we compare our results to the well-studied X. birchmannixX. malinche hybrid populations and conclude that deeper evolutionary divergence has resulted in stronger selection and higher repeatability in patterns of local ancestry in hybrids between X. birchmanni and X. cortezi.\nGenomic analysis of Ancylistes closterii, an enigmatic alga parasitic fungus in the arthropod-associated Entomophthoromycotina\nAuthors: Seto, K.; James, T. Y.\nScore: 4.1, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573025\nRecent advances in fungal genome sequencing have dramatically altered our understanding of the phylogeny and evolution of Fungi. However, there are still many poorly studied obligate parasitic or symbiotic fungi for which we lack any genomic information or knowledge of where they fit in the fungal phylogeny. Ancylistes, an endoparasite of desmid green algae, is such an understudied fungal genus. This genus has been taxonomically placed in the group of arthropod pathogens and saprobes, Entomophthoromycotina in Zoopagomycota. Understanding the phylogenetic position of Ancylistes provides insights into the nutritional evolution of Zoopagomycota, which is primarily composed of animal-associated fungi. In this study, we found and cultivated Ancylistes closterii with its host Closterium sp. and sequenced its genome to investigate its phylogenetic position and evolution. Phylogenetic analyses using rDNA and genome-scale datasets showed that A. closterii was sister to other Entomophthoromycotina fungi, confirming the taxonomic position of Ancylistes. Despite the ecological distinctiveness between Ancylistes and other Entomophthoromycotina fungi, our comparative genomic analyses revealed many shared traits of these fungi such as lineage-specific subtilases and hybrid histidine kinases. Ancylistes also possessed unique genes among Zoopagomycota fungi, such as plant cell wall degrading enzymes which could be important for infection of algae. SignificanceImproved taxon sampling is important for inferring a robust phylogeny of Fungi. However, there are still poorly studied obligate parasitic taxa whose DNA sequencing is challenging, especially in Zoopagomycota, one of the early diverging lineages of Fungi. This study focused on a long-neglected algal parasite, Ancylistes closterii, which belongs to the arthropod-associated group, Entomophthoromycotina. We rediscovered A. closterii and established a dual culture of fungus and its host alga, which enabled the first molecular analysis of this enigmatic parasite. Our results provide new insights into the nutritional evolution of primarily animal-associated Zoopagomycota.\nMulticellularity and increasing Reynolds number impact on the evolutionary shift in photo-induced ciliary response in Volvocales\nAuthors: Ueki, N.; Wakabayashi, K.-i.\nScore: 3.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573154\nVolvocales include species with different cell numbers and sizes, reflecting a history of gradual size increase evolution. Unicellular species live in low Reynolds-number (Re) environments where viscous forces dominate, whereas multicellular species live in higher Re environments with nonnegligible inertial forces. Despite significant changes in the physical environment, during the evolution of multicellularity they maintained photobehaviors (i.e., photoshock and phototactic responses), which allows them to survive under changing light conditions. In this study, we classified photo-induced ciliary responses in Volvocales into four patterns: temporal waveform conversion, no obvious response, pause in ciliary beating, temporal changes in ciliary beating directions. We found that which species exhibit which pattern depends on Re associated with the individual size of each species rather than phylogenetic relationships. These results suggest that species with increased cell numbers acquired their responses adapted to higher Re fluid environments. Significance StatementVolvocales green algae include species with various cell numbers and are excellent organisms for studying the evolution of multicellularity. They exhibit photobehaviors by changing the pattern of ciliary beating, which could be categorized into four patterns. We found that the difference in patterns among the organisms is due to the Reynolds number, the ratio of viscous and inertia forces, rather than their phylogenetic relationships. This study indicates that the fluid environment was an important factor in natural selection for behavioral changes in microalgae during evolution. The results link evolution and physics while contributing to the design of micromachines.\n",
  "wordCount" : "2809",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572910">
        <p class="paperTitle">Phylogenomics recovers multiple origins of portable case-making in caddisflies (Insecta: Trichoptera), the world&#39;s most common underwater architects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frandsen, P. B.; Holzenthal, R. W.; Espeland, M.; Breinholt, J.; Thomas, J. A.; Simon, S.; Kawahara, A. Y.; Plotkin, D.; Hotaling, S.; Li, Y.; Nelson, C. R.; Niehuis, O.; Mayer, C.; Podsiadlowski, L.; Donath, A.; Misof, B.; Lemmon, E.; Lemmon, A. R.; Morse, J. C.; Pauls, S. U.; Zhou, X.</p>
        <p class="info">Score: 78.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572910' target='https://doi.org/10.1101/2023.12.21.572910'> 10.1101/2023.12.21.572910</a></p>
        <p class="abstract">Caddisflies (Trichoptera) are among the most diverse groups of freshwater animals with more than 16,000 described species. They play an outsized role in freshwater ecology and environmental engineering in streams, rivers, and lakes. Because of this, they are frequently used as indicator organisms in biomonitoring programs. Despite their importance, key questions concerning the evolutionary history of caddisflies, such as the timing and origin of larval case-making, have been unanswered due to the lack of a well-resolved phylogenetic tree. To shed light on these questions in Trichoptera, we estimated a phylogenetic tree using a combination of transcriptomes and targeted enrichment data for 206 species, representing 48 of 52 extant families and 174 genera. We calibrated and dated the tree with a set of 33 carefully selected fossils. The first caddisflies originated in the Permian and the major suborders began to diversify in the Triassic. Ancestral state reconstruction and diversification analysis revealed that portable case-making evolved in three separate lineages and shifts in diversification occurred in concert with key evolutionary innovations other than case-making.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573205">
        <p class="paperTitle">Distinct ancient structural polymorphisms control heterodichogamy in walnuts and hickories</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groh, J. S.; Vik, D. C.; Stevens, K. A.; Brown, P. J.; Langley, C. H.; Coop, G.</p>
        <p class="info">Score: 17.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573205' target='https://doi.org/10.1101/2023.12.23.573205'> 10.1101/2023.12.23.573205</a></p>
        <p class="abstract">The maintenance of stable mating type polymorphisms is a classic example of balancing selection, underlying the nearly ubiquitous 50/50 sex ratio in species with separate sexes. One lesser known but intriguing example of a balanced mating polymorphism in angiosperms is heterodichogamy - polymorphism for opposing directions of dichogamy (temporal separation of male and female function in hermaphrodites) within a flowering season. This mating system is common throughout Juglandaceae, the family that includes globally important and iconic nut and timber crops - walnuts (Juglans), as well as pecan and other hickories (Carya). We show that two distinct genetic systems control heterodichogamous mating systems in these genera. In each genus, we find ancient (&gt;50Mya) structural variants that segregate as trans-species polymorphisms. The Juglans locus maps to a ca. 20kb structural variant adjacent to gene with likely function in the trehalose-6-phosphate signaling pathway, which is known to regulate flowering time in Arabidopsis and inflorescence architecture in maize. Notably, an insertion in the dominant (protogynous) haplotype contains 8-12 complex tandem duplicates of the 3 UTR of the gene, and transpecies polymorphism within the gene is restricted to the 3 UTR. In contrast, the Carya heterodichogamy locus maps to a ca. 200-450 kb cluster of tightly linked polymorphisms across 20 genes, some of which have known roles in flowering and are differentially expressed between morphs in floral primordia. The dominant (protogynous) haplotype in pecan, which is nearly always heterozygous and appears to rarely recombine, shows reduced genetic diversity and is over twice as long as its recessive counterpart due to proliferation of multiple transposable element families. We did not detect either genetic system in another heterodichogamous genus within the Juglandaceae, suggesting that additional genetic systems for heterodichogamy may yet remain undiscovered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573081">
        <p class="paperTitle">Evolution of Multicellularity Genes in the Lead Up to the Great Oxidation Event</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boden, J. S.; Nieves-Morion, M.; Nurnberg, D. J.; Arevalo, S.; Flores, E.; Sanchez-Baracaldo, P.</p>
        <p class="info">Score: 15.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573081' target='https://doi.org/10.1101/2023.12.23.573081'> 10.1101/2023.12.23.573081</a></p>
        <p class="abstract">Cyanobacteria are among the most morphologically diverse prokaryotic phyla on Earth. Their morphotypes range from unicellular to multicellular filaments, yet mechanisms underlying the evolution of filamentous morphologies remain unknown. Here, we implement phylogenomic, Bayesian molecular clock and gene-tree-species-tree reconciliation analyses to estimate when genes encoding cell-cell joining structures first evolved. We also characterise septal structures and measure intercellular communication rates in non-model and early-branching filamentous strains. Our results suggest that genes encoding septal proteins (namely sepJ, sepI, and fraE) and potentially pattern formation (hetR) evolved in the Neoarchaean [~]2.6-2.7 billion years (Ga) ago. Later, at the start of the Great Oxygenation Event [~]2.5 Ga, genes involved in cellular differentiation (namely hetZ, patU3 and hglK) appeared. Our results predict that early-branching lineages like Pseudanabaena were capable of intercellular communication, but further innovations in cellular differentiation were needed to drive ecological expansion on a scale large enough to permanently oxygenate Earths atmosphere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573503">
        <p class="paperTitle">Complex allometric relationships and ecological factors shape the development and evolution of eye size in the modular visual system of spiders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chong, K. L.; Grahn, A.; Perl, C. D.; Sumner-Rooney, L.</p>
        <p class="info">Score: 14.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573503' target='https://doi.org/10.1101/2023.12.28.573503'> 10.1101/2023.12.28.573503</a></p>
        <p class="abstract">Eye size affects many aspects of visual function, most notably including contrast sensitivity and achievable spatial resolution, but eyes are costly to grow and maintain. The allometry of eyes can provide insight to this trade-off in the context of visual ecology, but to date, this has mainly been explored in species that have a single pair of eyes of equal size. By contrast, animals possessing larger visual systems can exhibit variable eye sizes within individuals. Spiders have up to four pairs of eyes whose size often varies, but their ontogenetic, static, and evolutionary allometry has not yet been studied in a comparative context. We report variable evolutionary and developmental dynamics in eye size across 1098 individuals in 39 species and 8 families, indicating selective pressures and constraints driving the evolution of different eye pairs and lineages. We observed variation in the relationship between eye and body size not only between taxa and visual ecologies, but between eye pairs within species, indicating that growth dynamics are variable and can be divergently adapted in particular eye pairs. Supplementing our sampling with a phylogenetically comprehensive dataset recently published by Wolff et al. (2022), we confirmed these findings across more than 400 species, found that ecological factors such as visual hunting, web building, and diurnality can also impact eye diameter, and identified significant allometric shifts across spider phylogeny using an unbiased approach, many of which coincide with ecological changes such as visual pursuit hunting strategies. This study represents the first detailed comparative exploration of visual allometry in spiders, revealing striking differences in eye growth not only between families but also within species. Our findings shed light on the relationship between spider visual systems and their diverse ecologies, and how spiders exploit the modular nature of their visual systems to balance selective pressures and optical and energetic constraints.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573114">
        <p class="paperTitle">Brood care in shell-dwelling cichlids is timed by independent maternal and larval clocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parker, A. V.; Stemmer, M.; Graetsch, S.; Dorigo, A.; Rodriguez Ramirez, O.; Adel, A.; Jordan, A. L.; Baier, H.</p>
        <p class="info">Score: 11.9, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573114' target='https://doi.org/10.1101/2023.12.22.573114'> 10.1101/2023.12.22.573114</a></p>
        <p class="abstract">Successful brood care requires a finely tuned interplay between caregiving parent and offspring. Cichlids are a diverse group of teleost fish known for their extensive parental care of altricial young. Female Lamprologus ocellatus lay their eggs in the protective confines of an abandoned snail shell, where they raise them to free-swimming larvae. Here we investigated whether the transition to larval independence is driven by the behavior of the mother, the offspring, or an interaction between the two. Using 3D printed shells with a window, we were able to observe the behavior of both mother and offspring inside the nest using automated object detection and tracking. We found that, once hatched, the larvae remain in the deep chambers of the shell, and at 9 dpf, actively emerge from the shell during the day. An inversion of larval phototactic behavior from dark- to light-seeking leads up to this emergence. Removal of the mother causes premature emergence of the larvae by a few days. The natural schedule (emergence at 9 dpf) is restored if the larvae are supplied with fresh water in the mothers absence. In cross-fostering experiments, we found that swapping out the mothers own clutch for older larvae forces a delay in emergence, as the foster mother prevents the larvae from leaving at 9 dpf. We conclude that larval and maternal behavior are each controlled by independent internal clocks, which, although normally synchronized, can be experimentally manipulated and brought into conflict. This study has thus revealed an innate sequence of behavioral adaptations that orchestrate brood care in shell-dwelling cichlids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573215">
        <p class="paperTitle">One million years of solitude: the rapid evolution of de novo protein structure and complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Li, Q.; Xia, S.; Arsala, D.; Sosa, D.; Wang, D.; Long, M.</p>
        <p class="info">Score: 6.3, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573215' target='https://doi.org/10.1101/2023.12.24.573215'> 10.1101/2023.12.24.573215</a></p>
        <p class="abstract">Recent studies have established that de novo genes, evolving from non-coding sequences, enhance protein diversity through a stepwise process. However, the pattern and rate of their structural evolution over time remain unclear. Here, we addressed these issues within a timeframe of ~1 million years (97% of de novo genes). We found that de novo genes evolve faster than gene duplicates in the intrinsic disordered regions (IDRs, such as random coils), secondary structural elements (such as -helix and {beta}-strand), hydrophobicity, and molecular recognition features (MoRFs). Specifically, we observed an 8-14% reduction in random coils and IDR lengths per million years per protein, and a 2.3-6.5% increase in structured elements, hydrophobicity, and MoRFs. These patterns of structural evolution align with changes in amino acid composition over time. We also revealed significantly higher positive charges but smaller molecular weights for de novo proteins than duplicates. Tertiary structure predictions demonstrated that de novo proteins, though not typically well-folded on their own, readily form low-energy, compact complexes with extensive residue contacts and conformational flexibility, suggesting a faster-binding scenario in de novo proteins to promote interaction. Our findings illuminate the rapid evolution of protein structure in the early life of de novo proteins, originating from noncoding sequences, highlighting their quick transformation into active, complex-forming components within a remarkably short evolutionary timeframe.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572981">
        <p class="paperTitle">Horizontal transfers between fungal Fusarium species contributed to successive outbreaks of coffee wilt disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peck, L. D.; Llewellyn, T.; Bennetot, B.; O&#39;Donnell, S.; Nowell, R. W.; Ryan, M. J.; Flood, J.; Rodriguez de la Vega, R. C.; Ropars, J.; Giraud, T.; Spanu, P. D.; Barraclough, T. G.</p>
        <p class="info">Score: 5.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572981' target='https://doi.org/10.1101/2023.12.22.572981'> 10.1101/2023.12.22.572981</a></p>
        <p class="abstract">Outbreaks of fungal disease have devastated plants and animals throughout history. Over the past century, the repeated emergence of coffee wilt disease caused by the fungal pathogen Fusarium xylarioides severely impacted coffee production across sub-Saharan Africa. To improve the disease management of such pathogens, it is crucial to understand their genetic structure and evolutionary potential. We compared the genomes of 13 historic strains spanning six decades and multiple disease outbreaks to investigate population structure and host specialisation. We found F. xylarioides comprises at least four distinct lineages: one host-specific to Coffea arabica, one to C. canephora var. robusta, and two historic lineages isolated from various Coffea species. Mapping variation onto a new long-read reference genome showed that host-specificity appears to be acquired through horizontal transfer of effector genes from members of the F. oxysporum species complex. This species complex is known to cause wilt disease in over 100 plant species. Multiple transfers into the F. xylarioides populations matched to different parts of the F. oxysporum mobile pathogenicity chromosome and were enriched in effector genes and transposons. Effector genes in this region and other horizontally transferred carbohydrate-active enzymes important in the breakdown of plant cell walls were shown by transcriptomics to be highly expressed during infection of C. arabica by the fungal arabica strains. Widespread sharing of specific transposons between F. xylarioides and F. oxysporum, and the presence of large Starship elements, indicate that transposons were involved in horizontal transfers. Our results support the hypothesis that horizontal gene transfers contributed to the repeated emergence of this fungal disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572897">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572897" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572897">
        <p class="paperTitle">Genome evolution is surprisingly predictable after initial hybridization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572897" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572897" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Langdon, Q. K.; Groh, J. S.; Aguillon, S.; Powell, D. L.; Gunn, T. R.; Payne, C. Y.; Baczenas, J. J.; Donny, A.; Dodge, T. O.; Du, K.; Schartl, M.; Rios-Cardenas, O.; Gutierrez-Rodriguez, C.; Morris, M.; Schumer, M.</p>
        <p class="info">Score: 4.8, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572897' target='https://doi.org/10.1101/2023.12.21.572897'> 10.1101/2023.12.21.572897</a></p>
        <p class="abstract">Over the past two decades, evolutionary biologists have come to appreciate that hybridization, or genetic exchange between distinct lineages, is remarkably common - not just in particular lineages but in taxonomic groups across the tree of life. As a result, the genomes of many modern species harbor regions inherited from related species. This observation has raised fundamental questions about the degree to which the genomic outcomes of hybridization are repeatable and the degree to which natural selection drives such repeatability. However, a lack of appropriate systems to answer these questions has limited empirical progress in this area. Here, we leverage independently formed hybrid populations between the swordtail fish Xiphophorus birchmanni and X. cortezi to address this fundamental question. We find that local ancestry in one hybrid population is remarkably predictive of local ancestry in another, demographically independent hybrid population. Applying newly developed methods, we can attribute much of this repeatability to strong selection in the earliest generations after initial hybridization. We complement these analyses with time-series data that demonstrates that ancestry at regions under selection has remained stable over the past [~]40 generations of evolution. Finally, we compare our results to the well-studied X. birchmannixX. malinche hybrid populations and conclude that deeper evolutionary divergence has resulted in stronger selection and higher repeatability in patterns of local ancestry in hybrids between X. birchmanni and X. cortezi.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573025">
        <p class="paperTitle">Genomic analysis of Ancylistes closterii, an enigmatic alga parasitic fungus in the arthropod-associated Entomophthoromycotina</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seto, K.; James, T. Y.</p>
        <p class="info">Score: 4.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573025' target='https://doi.org/10.1101/2023.12.22.573025'> 10.1101/2023.12.22.573025</a></p>
        <p class="abstract">Recent advances in fungal genome sequencing have dramatically altered our understanding of the phylogeny and evolution of Fungi. However, there are still many poorly studied obligate parasitic or symbiotic fungi for which we lack any genomic information or knowledge of where they fit in the fungal phylogeny. Ancylistes, an endoparasite of desmid green algae, is such an understudied fungal genus. This genus has been taxonomically placed in the group of arthropod pathogens and saprobes, Entomophthoromycotina in Zoopagomycota. Understanding the phylogenetic position of Ancylistes provides insights into the nutritional evolution of Zoopagomycota, which is primarily composed of animal-associated fungi. In this study, we found and cultivated Ancylistes closterii with its host Closterium sp. and sequenced its genome to investigate its phylogenetic position and evolution. Phylogenetic analyses using rDNA and genome-scale datasets showed that A. closterii was sister to other Entomophthoromycotina fungi, confirming the taxonomic position of Ancylistes. Despite the ecological distinctiveness between Ancylistes and other Entomophthoromycotina fungi, our comparative genomic analyses revealed many shared traits of these fungi such as lineage-specific subtilases and hybrid histidine kinases. Ancylistes also possessed unique genes among Zoopagomycota fungi, such as plant cell wall degrading enzymes which could be important for infection of algae.

SignificanceImproved taxon sampling is important for inferring a robust phylogeny of Fungi. However, there are still poorly studied obligate parasitic taxa whose DNA sequencing is challenging, especially in Zoopagomycota, one of the early diverging lineages of Fungi. This study focused on a long-neglected algal parasite, Ancylistes closterii, which belongs to the arthropod-associated group, Entomophthoromycotina. We rediscovered A. closterii and established a dual culture of fungus and its host alga, which enabled the first molecular analysis of this enigmatic parasite. Our results provide new insights into the nutritional evolution of primarily animal-associated Zoopagomycota.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573154">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573154" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573154">
        <p class="paperTitle">Multicellularity and increasing Reynolds number impact on the evolutionary shift in photo-induced ciliary response in Volvocales</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573154" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573154" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ueki, N.; Wakabayashi, K.-i.</p>
        <p class="info">Score: 3.9, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573154' target='https://doi.org/10.1101/2023.12.22.573154'> 10.1101/2023.12.22.573154</a></p>
        <p class="abstract">Volvocales include species with different cell numbers and sizes, reflecting a history of gradual size increase evolution. Unicellular species live in low Reynolds-number (Re) environments where viscous forces dominate, whereas multicellular species live in higher Re environments with nonnegligible inertial forces. Despite significant changes in the physical environment, during the evolution of multicellularity they maintained photobehaviors (i.e., photoshock and phototactic responses), which allows them to survive under changing light conditions. In this study, we classified photo-induced ciliary responses in Volvocales into four patterns: temporal waveform conversion, no obvious response, pause in ciliary beating, temporal changes in ciliary beating directions. We found that which species exhibit which pattern depends on Re associated with the individual size of each species rather than phylogenetic relationships. These results suggest that species with increased cell numbers acquired their responses adapted to higher Re fluid environments.

Significance StatementVolvocales green algae include species with various cell numbers and are excellent organisms for studying the evolution of multicellularity. They exhibit photobehaviors by changing the pattern of ciliary beating, which could be categorized into four patterns. We found that the difference in patterns among the organisms is due to the Reynolds number, the ratio of viscous and inertia forces, rather than their phylogenetic relationships. This study indicates that the fluid environment was an important factor in natural selection for behavioral changes in microalgae during evolution. The results link evolution and physics while contributing to the design of micromachines.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
