<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.",
  "keywords": [
    
  ],
  "articleBody": " Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.\nScore: 88.0, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547326\nThe evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.\nA Neanderthal/Denisovan GLI3 variant contributes to anatomical variations in mice\nAuthors: Agata, A.; Ohtsuka, S.; Noji, R.; Gotoh, H.; Ono, K.; Nomura, T.\nScore: 54.6, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547394\nChanges in genomic structures underlie phenotypic diversification in organisms. Amino acid-changing mutations affect pleiotropic functions of proteins, although little is known about how mutated proteins are adapted in existing developmental programs. Here we investigate the biological effects of a variant of the GLI3 transcription factor (GLI3R1537C) carried in Neanderthals and Denisovans, which are extinct hominins close to modern humans. R1537C does not compromise protein stability or GLI3 activator-dependent transcriptional activities. In contrast, R1537C affects the regulation of downstream target genes associated with developmental processes. Furthermore, genome-edited mice carrying the Neanderthal/Denisovan GLI3 mutation exhibited various alterations in skeletal morphology. Our data suggest that an extinct hominin-type GLI3 contributes to species-specific anatomical variations, which were tolerated by relaxed constraint in developmental programs during human evolution.\nPhylogenetic conflicts: distinguishing gene flow from incomplete lineage sorting\nAuthors: Galtier, N.\nScore: 21.9, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547897\nAO_SCPLOWBSTRACTC_SCPLOWGene flow and incomplete lineage sorting are two distinct sources of phylogenetic conflict, i.e., gene trees that differ in topology from each other and from the species tree. Distinguishing between the two processes is a key objective of current evolutionary genomics. This is most often pursued via the so-called ABBA-BABA type of method, which relies on a prediction of symmetry of gene tree discordance made by the incomplete lineage sorting hypothesis. Gene flow, however, need not be asymmetric, and the ABBA-BABA hypothesis-testing approach is not a proper way of measuring gene flow prevalence. We introduce Aphid, a maximum-likelihood method aimed at quantifying the sources of phylogenetic conflict via topology and branch length analysis of three-species gene trees. Aphid draws information from the fact that gene trees affected by gene flow tend to have shorter branches, and gene trees affected by incomplete lineage sorting longer branches, than the average gene tree. Accounting for the among-loci variance in mutation rate and gene flow time, Aphid returns estimates of the speciation times and ancestral effective population size, and a posterior assessment of the contribution of gene flow and incomplete lineage sorting to the conflict. Simulations suggest that Aphid is reasonably robust to a wide range of conditions. An analysis of exon trees in mammals illustrates the potential of the approach. Notably, Aphid predicts that roughly half of the human/chimpanzee/gorilla phylogenetic conflict is due to ancient gene flow. This also translates in older estimated speciation times and a smaller estimated effective population size in this group, compared to existing analyses assuming no gene flow.\nBeyond venomous fangs: Uloboridae spiders have lost their venom apparatus but not their toxins\nAuthors: Peng, X.; Dederichs, T.; Lüddecke, T.; Michalik, P.; Mucciolo, A.; Robinson-Rechavi, M.; Zancolli, G.\nScore: 25.9, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546488\nVenom is one of the most potent chemical secretions in nature which has played a significant role in the evolutionary success of many animal groups, including spiders. However, the family Uloboridae has seemingly lost the venom-producing organs, leaving the presence and function of toxins in question. Uloborids employ a distinct hunting method involving extensive silk-wrapping and regurgitation of digestive fluids onto the whole prey, yet the precise mechanism by which prey is immobilized remains unknown. One hypothesis is that toxins may have shifted from the venom to other secretions that come into contact with prey, such as silk or digestive fluids. Here, we use a combination of histology and multi-tissue transcriptomics, to 1) investigate the absence of venom glands, and 2) examine the expression of venom toxins in different body parts of Uluborus plumipes. Our findings indicate that U. plumipes effectively does not possess venom glands, nor the duct opening in the fangs. However, we identified putative neurotoxins that are highly expressed in the digestive gland, suggesting that these may contribute to prey immobilization. Among the most highly expressed were U3-aranetoxins, U24-ctenitoxins, and a defensin, although the latter was not predicted to have neurotoxic activity. These results support our hypothesis that Uloboridae spiders have lost their venom apparatus, but not their toxins. These \"digestive toxins\" might be responsible for complete prey immobilisation and represent an alternative hunting strategy that replaced the loss of a frontal venom system.\nViruses in laboratory Drosophila and their impact on host gene expression\nAuthors: Kuyateh, O.; Obbard, D.\nScore: 17.0, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548260\nDrosophila melanogaster has one of the best characterized antiviral immune responses among invertebrates. However, relatively few easily-transmitted natural virus isolates are available, and so many Drosophila experiments have been performed using artificial infection routes and artificial host-virus combinations. These may not reflect natural infections, especially for subtle phenotypes such as gene expression. Here, to explore the laboratory virus community and to better understand how natural virus infections induce changes in gene expression, we analyse seven publicly available D. melanogaster transcriptomic sequencing libraries that were originally sequenced for projects unrelated to virus infection. We find ten known viruses--including five that have not been experimentally isolated--but no previously unknown viruses. Our analysis of host gene expression found numerous genes were differentially expressed in flies that were naturally infected with a virus. For example, flies infected with nora virus showed patterns of gene expression consistent with intestinal vacuolization and host attempted repair via the upd3 JAK/STAT pathway. We also found marked sex-differences in virus-induced differential gene expression. Our results show that natural virus infection in laboratory Drosophila does indeed induce detectable changes in gene expression, suggesting that this may form an important background condition for experimental studies in the laboratory.\nBursts of novel composite gene families at major nodes in animal evolution\nAuthors: Mulhair, P. O.; Moran, R. J.; Pathmanathan, J. s.; Sussfeld, D.; Creevey, C. J.; Siu-Ting, K.; Whelan, F. J.; Pisani, D.; Constantinides, B.; Pelletier, E.; lopez, p.; Bapteste, E.; McInerney, J. O.; O'Connell, M. J.\nScore: 14.1, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548381\nA molecular level perspective on how novel phenotypes evolve is contingent on our understanding of how genomes evolve through time, and of particular interest is how novel elements emerge or are lost. Mechanisms of protein evolution such as gene duplication have been well established. Studies of gene fusion events show they often generate novel functions and adaptive benefits. Identifying gene fusion and fission events on a genome scale allows us to establish the mode and tempo of emergence of composite genes across the animal tree of life, and allows us to test the repeatability of evolution in terms of determining how often composite genes can arise independently. Here we show that ~5% of all animal gene families are composite, and their phylogenetic distribution suggests an abrupt, rather than gradual, emergence during animal evolution. We find that gene fusion occurs at a higher rate than fission (73.3% vs 25.4%) in animal composite genes, but many gene fusions (79% of the 73.3%) have more complex patterns including subsequent fission or loss. We demonstrate that nodes such as Bilateria, Euteleostomi, and Eutheria, have significantly higher rates of accumulation of composite genes. We observe that in general Deuterostomes have a greater amount of composite genes as compared to Protostomes. Intriguingly, up to 41% of composite gene families have evolved independently in different clades showing that the same solutions to protein innovation have evolved time and again in animals.\nSugar assimilation underlying dietary evolution of Neotropical bats\nAuthors: Camacho, J.; Bernal-Rivera, A.; Robb, S.; Russell, J.; Yi, K.; Tsuchiya, D.; Wang, Y.; Murillo-Garcia, O. E.; Rohner, N.\nScore: 14.8, Published: 2023-07-03 DOI: 10.1101/2023.07.02.547432\nSpecializations in animal diets drive selective demands on morphology, anatomy, and physiology. Studying adaptations linked to diet evolution benefits from examining Neotropical bats, a remarkable group with high taxonomic and trophic diversity. In this study we performed glucose tolerance tests on wild-caught bats, which revealed distinct responses to three sugars present in different foods: trehalose (insects), sucrose and glucose (fruits and nectar). Insect eating bats responded most strongly to trehalose, while bats with nectar and fruit-based diets exhibited heightened response to glucose and sucrose, reaching blood sugar levels over 600 mg/dL. To reveal positive selection in sugar assimilation genes we performed genome analysis of 22 focal bat species and 2 outgroup species. We identified selection in the ancestral vespertilionid branch (insect-eaters) for the digestive enzyme Trehalase, while sucrase-isomaltase exhibited selection in branches leading to omnivorous and nectar diets. Unexpectedly, the insect-eating lineage Myotis exhibited sucrase-isomaltase selection, potentially explaining their heightened sucrose assimilation. Furthermore, the glucose transporters SLC2A3 and SLC2A2 showed selection in lineages associated with nectar and blood diets. By examining cellular features of the small intestine, we discovered that sugar consumption strongly impacted numerous digestive traits, providing valuable insight into the physiological implications of the identified molecular adaptations. To elucidate this further, we used HCR RNA-FISH to perform single molecule ex vivo gene expression analysis of enterocyte response to a glucose meal in three focal species. We observed unusually high activity in the glucose transporter Slc2a2 during the fasted state of nectar bats that did not change upon feeding. Comparatively, nectar bats exhibited an enhanced capacity for intestinal absorption of dietary sugar primarily through Slc2a2, while fruit bats relied on increasing levels of Slc5a1. Overall, this study highlights the intricate interplay between molecular, morphological, and physiological aspects of diet evolution and provides new insights to our understanding of sugar assimilation mechanisms in mammals. HighlightsO_LISugar assimilation differences emphasize metabolic adaptations to diet C_LIO_LIGlucose tolerance tests provide a quick and practical assessment of dietary ecology C_LIO_LIBat genomes exhibit positive selection on digestive enzymes and glucose transporters C_LIO_LIGlucose absorption differences can be explained by gut anatomy C_LIO_LIExtreme blood glucose (\u003e650 mg/dL) coincides with constitutive expression of apical Slc2a2 C_LIO_LIThe regulation of Sglt1 and apical Slc2a2 highlights differences in blood glucose levels C_LI Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=106 SRC=\"FIGDIR/small/547432v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@426655org.highwire.dtl.DTLVardef@14c9735org.highwire.dtl.DTLVardef@15b35f6org.highwire.dtl.DTLVardef@133d43b_HPS_FORMAT_FIGEXP M_FIG C_FIG\nLarge-scale phylogenomics uncovers a complex evolutionary history and extensive ancestral gene flow in an African primate radiation\nAuthors: Jensen, A.; Swift, F.; de Vries, D.; Beck, R.; Kuderna, L. F. K.; Knauf, S.; Chuma, I. S.; Keyyu, J. D.; Kitchener, A. C.; Farh, K.; Rogers, J.; Marques-Bonet, T.; Detwiler, K. M.; Roos, C.; Guschanski, K.\nScore: 22.0, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545890\nUnderstanding the drivers of speciation is fundamental in evolutionary biology, and recent studies highlight hybridization as a potential facilitator of adaptive radiations. Using whole-genome sequencing data from 22 species of guenons (tribe Cercopithecini), one of the worlds largest primate radiations, we show that rampant gene flow characterizes their evolutionary history, and identify ancient hybridization across deeply divergent lineages differing in ecology, morphology and karyotypes. Lineages experiencing gene flow tend to be more species-rich than non-admixed lineages. Mitochondrial transfers between distant lineages were likely facilitated by co-introgression of co-adapted nuclear variants. Although the genomic landscapes of introgression were largely lineage specific, we found that genes with immune functions were overrepresented in introgressing regions, in line with adaptive introgression, whereas genes involved in pigmentation and morphology may contribute to reproductive isolation. This study provides important insights into the prevalence, role and outcomes of ancestral hybridization in a large mammalian radiation.\nWidespread but spotty: Atlantic cod is a complex of five species through ecological speciation.\nAuthors: Halldorsdottir, K.; Llanos-Garrido, A.; Einarsson, A. W.; Sackton, T. B.; Arnason, E.\nScore: 15.0, Published: 2023-07-06 DOI: 10.1101/2023.06.14.544921\nMarine environments are home to a significant portion of fish and other organisms biodiversity. However, our knowledge and empirical evidence of speciation in the marine realm is quite limited. Here we use genomics and study population differentiation and speciation in the Atlantic cod, a marine teleost of biological and economic importance. The Atlantic cod genome is liable to recurrent selective sweeps showing the importance of selective processes [1]. We show that the non-inversion part of the genome separates individuals into five clusters. At the four chromosomes carrying large inversions that define ecotypic variation, individuals also cluster into the same five clusters in conjunction with individuals inversion genotypes. Within each cluster, individuals homozygous or heterozygous for the inversions are divergent and there is an adaptive divergence of each type of inversion homozygote among clusters. These interchromosomal correlations imply the reproductive isolation of five cryptic species. We show that the inversion haplotypes are ancient [2] trans-species balanced polymorphisms that would evolve towards homozygous lethality under Mullers ratchet [3-5]. This and recombination suppression of inversions. However, gene-flux [6] from gene conversion [7] rescues homozygous fitness, and recombination and selection lead to the evolution of habitat specialists through ecological speciation [8, 9]. The specialized genotypes actively seek out their preferred habitat resulting in a widespread but patchy distribution that contradicts the traditional geographical speciation model and the stock concept in fisheries biology. Our results highlight the power of genomics in studying selection and speciation while informing conservation and management of essential fisheries resources.\nPatterns of recombination in snakes reveal a tug of war between PRDM9 and promoter-like features\nAuthors: Hoge, C. R.; de Manuel, M.; Mahgoub, M.; Okami, N.; Fuller, Z. L.; Banerjee, S.; Baker, Z.; Mcnulty, M.; Andolfatto, P.; Macfarlan, T. S.; Schumer, M.; Tzika, A. C.; Przeworski, M.\nScore: 11.8, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548536\nIn vertebrates, there are two known mechanisms by which meiotic recombination is directed to the genome: in humans, mice, and other mammals, recombination occurs almost exclusively where the protein PRDM9 binds, while in species lacking an intact PRDM9, such as birds and canids, recombination rates are elevated near promoter-like features. To test if PRDM9 also directs recombination in non-mammalian vertebrates, we focused on an exemplar species, the corn snake (Pantherophis guttatus). Unlike birds, this species possesses a single, intact PRDM9 ortholog. By inferring historical recombination rates along the genome from patterns of linkage disequilibrium and identifying crossovers in pedigrees, we found that PRDM9 specifies the location of recombination events outside of mammals. However, we also detected an independent effect of promoter-like features on recombination, which is more pronounced on macro- than microchromosomes. Thus, our findings reveal that the uses of PRDM9 and promoter-like features are not mutually-exclusive, and instead reflect a tug of war, which varies in strength along the genome and is more lopsided in some species than others.\n",
  "wordCount" : "2684",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547326">
        <p class="paperTitle">Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.</p>
        <p class="info">Score: 88.0, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547326' target='https://doi.org/10.1101/2023.07.03.547326'> 10.1101/2023.07.03.547326</a></p>
        <p class="abstract">The evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547394">
        <p class="paperTitle">A Neanderthal/Denisovan GLI3 variant contributes to anatomical variations in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Agata, A.; Ohtsuka, S.; Noji, R.; Gotoh, H.; Ono, K.; Nomura, T.</p>
        <p class="info">Score: 54.6, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547394' target='https://doi.org/10.1101/2023.07.03.547394'> 10.1101/2023.07.03.547394</a></p>
        <p class="abstract">Changes in genomic structures underlie phenotypic diversification in organisms. Amino acid-changing mutations affect pleiotropic functions of proteins, although little is known about how mutated proteins are adapted in existing developmental programs. Here we investigate the biological effects of a variant of the GLI3 transcription factor (GLI3R1537C) carried in Neanderthals and Denisovans, which are extinct hominins close to modern humans. R1537C does not compromise protein stability or GLI3 activator-dependent transcriptional activities. In contrast, R1537C affects the regulation of downstream target genes associated with developmental processes. Furthermore, genome-edited mice carrying the Neanderthal/Denisovan GLI3 mutation exhibited various alterations in skeletal morphology. Our data suggest that an extinct hominin-type GLI3 contributes to species-specific anatomical variations, which were tolerated by relaxed constraint in developmental programs during human evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547897">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547897" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547897">
        <p class="paperTitle">Phylogenetic conflicts: distinguishing gene flow from incomplete lineage sorting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547897" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547897" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Galtier, N.</p>
        <p class="info">Score: 21.9, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547897' target='https://doi.org/10.1101/2023.07.06.547897'> 10.1101/2023.07.06.547897</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWGene flow and incomplete lineage sorting are two distinct sources of phylogenetic conflict, i.e., gene trees that differ in topology from each other and from the species tree. Distinguishing between the two processes is a key objective of current evolutionary genomics. This is most often pursued via the so-called ABBA-BABA type of method, which relies on a prediction of symmetry of gene tree discordance made by the incomplete lineage sorting hypothesis. Gene flow, however, need not be asymmetric, and the ABBA-BABA hypothesis-testing approach is not a proper way of measuring gene flow prevalence. We introduce Aphid, a maximum-likelihood method aimed at quantifying the sources of phylogenetic conflict via topology and branch length analysis of three-species gene trees. Aphid draws information from the fact that gene trees affected by gene flow tend to have shorter branches, and gene trees affected by incomplete lineage sorting longer branches, than the average gene tree. Accounting for the among-loci variance in mutation rate and gene flow time, Aphid returns estimates of the speciation times and ancestral effective population size, and a posterior assessment of the contribution of gene flow and incomplete lineage sorting to the conflict. Simulations suggest that Aphid is reasonably robust to a wide range of conditions. An analysis of exon trees in mammals illustrates the potential of the approach. Notably, Aphid predicts that roughly half of the human/chimpanzee/gorilla phylogenetic conflict is due to ancient gene flow. This also translates in older estimated speciation times and a smaller estimated effective population size in this group, compared to existing analyses assuming no gene flow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546488">
        <p class="paperTitle">Beyond venomous fangs: Uloboridae spiders have lost their venom apparatus but not their toxins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, X.; Dederichs, T.; Lüddecke, T.; Michalik, P.; Mucciolo, A.; Robinson-Rechavi, M.; Zancolli, G.</p>
        <p class="info">Score: 25.9, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546488' target='https://doi.org/10.1101/2023.06.26.546488'> 10.1101/2023.06.26.546488</a></p>
        <p class="abstract">Venom is one of the most potent chemical secretions in nature which has played a significant role in the evolutionary success of many animal groups, including spiders. However, the family Uloboridae has seemingly lost the venom-producing organs, leaving the presence and function of toxins in question. Uloborids employ a distinct hunting method involving extensive silk-wrapping and regurgitation of digestive fluids onto the whole prey, yet the precise mechanism by which prey is immobilized remains unknown. One hypothesis is that toxins may have shifted from the venom to other secretions that come into contact with prey, such as silk or digestive fluids. Here, we use a combination of histology and multi-tissue transcriptomics, to 1) investigate the absence of venom glands, and 2) examine the expression of venom toxins in different body parts of Uluborus plumipes. Our findings indicate that U. plumipes effectively does not possess venom glands, nor the duct opening in the fangs. However, we identified putative neurotoxins that are highly expressed in the digestive gland, suggesting that these may contribute to prey immobilization. Among the most highly expressed were U3-aranetoxins, U24-ctenitoxins, and a defensin, although the latter was not predicted to have neurotoxic activity. These results support our hypothesis that Uloboridae spiders have lost their venom apparatus, but not their toxins. These &#34;digestive toxins&#34; might be responsible for complete prey immobilisation and represent an alternative hunting strategy that replaced the loss of a frontal venom system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548260">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548260" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548260">
        <p class="paperTitle">Viruses in laboratory Drosophila and their impact on host gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548260" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548260" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuyateh, O.; Obbard, D.</p>
        <p class="info">Score: 17.0, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548260' target='https://doi.org/10.1101/2023.07.10.548260'> 10.1101/2023.07.10.548260</a></p>
        <p class="abstract">Drosophila melanogaster has one of the best characterized antiviral immune responses among invertebrates. However, relatively few easily-transmitted natural virus isolates are available, and so many Drosophila experiments have been performed using artificial infection routes and artificial host-virus combinations. These may not reflect natural infections, especially for subtle phenotypes such as gene expression. Here, to explore the laboratory virus community and to better understand how natural virus infections induce changes in gene expression, we analyse seven publicly available D. melanogaster transcriptomic sequencing libraries that were originally sequenced for projects unrelated to virus infection. We find ten known viruses--including five that have not been experimentally isolated--but no previously unknown viruses. Our analysis of host gene expression found numerous genes were differentially expressed in flies that were naturally infected with a virus. For example, flies infected with nora virus showed patterns of gene expression consistent with intestinal vacuolization and host attempted repair via the upd3 JAK/STAT pathway. We also found marked sex-differences in virus-induced differential gene expression. Our results show that natural virus infection in laboratory Drosophila does indeed induce detectable changes in gene expression, suggesting that this may form an important background condition for experimental studies in the laboratory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548381">
        <p class="paperTitle">Bursts of novel composite gene families at major nodes in animal evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mulhair, P. O.; Moran, R. J.; Pathmanathan, J. s.; Sussfeld, D.; Creevey, C. J.; Siu-Ting, K.; Whelan, F. J.; Pisani, D.; Constantinides, B.; Pelletier, E.; lopez, p.; Bapteste, E.; McInerney, J. O.; O&#39;Connell, M. J.</p>
        <p class="info">Score: 14.1, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548381' target='https://doi.org/10.1101/2023.07.10.548381'> 10.1101/2023.07.10.548381</a></p>
        <p class="abstract">A molecular level perspective on how novel phenotypes evolve is contingent on our understanding of how genomes evolve through time, and of particular interest is how novel elements emerge or are lost. Mechanisms of protein evolution such as gene duplication have been well established. Studies of gene fusion events show they often generate novel functions and adaptive benefits. Identifying gene fusion and fission events on a genome scale allows us to establish the mode and tempo of emergence of composite genes across the animal tree of life, and allows us to test the repeatability of evolution in terms of determining how often composite genes can arise independently. Here we show that ~5% of all animal gene families are composite, and their phylogenetic distribution suggests an abrupt, rather than gradual, emergence during animal evolution. We find that gene fusion occurs at a higher rate than fission (73.3% vs 25.4%) in animal composite genes, but many gene fusions (79% of the 73.3%) have more complex patterns including subsequent fission or loss. We demonstrate that nodes such as Bilateria, Euteleostomi, and Eutheria, have significantly higher rates of accumulation of composite genes. We observe that in general Deuterostomes have a greater amount of composite genes as compared to Protostomes. Intriguingly, up to 41% of composite gene families have evolved independently in different clades showing that the same solutions to protein innovation have evolved time and again in animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547432">
        <p class="paperTitle">Sugar assimilation underlying dietary evolution of Neotropical bats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Camacho, J.; Bernal-Rivera, A.; Robb, S.; Russell, J.; Yi, K.; Tsuchiya, D.; Wang, Y.; Murillo-Garcia, O. E.; Rohner, N.</p>
        <p class="info">Score: 14.8, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547432' target='https://doi.org/10.1101/2023.07.02.547432'> 10.1101/2023.07.02.547432</a></p>
        <p class="abstract">Specializations in animal diets drive selective demands on morphology, anatomy, and physiology. Studying adaptations linked to diet evolution benefits from examining Neotropical bats, a remarkable group with high taxonomic and trophic diversity. In this study we performed glucose tolerance tests on wild-caught bats, which revealed distinct responses to three sugars present in different foods: trehalose (insects), sucrose and glucose (fruits and nectar). Insect eating bats responded most strongly to trehalose, while bats with nectar and fruit-based diets exhibited heightened response to glucose and sucrose, reaching blood sugar levels over 600 mg/dL. To reveal positive selection in sugar assimilation genes we performed genome analysis of 22 focal bat species and 2 outgroup species. We identified selection in the ancestral vespertilionid branch (insect-eaters) for the digestive enzyme Trehalase, while sucrase-isomaltase exhibited selection in branches leading to omnivorous and nectar diets. Unexpectedly, the insect-eating lineage Myotis exhibited sucrase-isomaltase selection, potentially explaining their heightened sucrose assimilation. Furthermore, the glucose transporters SLC2A3 and SLC2A2 showed selection in lineages associated with nectar and blood diets. By examining cellular features of the small intestine, we discovered that sugar consumption strongly impacted numerous digestive traits, providing valuable insight into the physiological implications of the identified molecular adaptations. To elucidate this further, we used HCR RNA-FISH to perform single molecule ex vivo gene expression analysis of enterocyte response to a glucose meal in three focal species. We observed unusually high activity in the glucose transporter Slc2a2 during the fasted state of nectar bats that did not change upon feeding. Comparatively, nectar bats exhibited an enhanced capacity for intestinal absorption of dietary sugar primarily through Slc2a2, while fruit bats relied on increasing levels of Slc5a1. Overall, this study highlights the intricate interplay between molecular, morphological, and physiological aspects of diet evolution and provides new insights to our understanding of sugar assimilation mechanisms in mammals.

HighlightsO_LISugar assimilation differences emphasize metabolic adaptations to diet
C_LIO_LIGlucose tolerance tests provide a quick and practical assessment of dietary ecology
C_LIO_LIBat genomes exhibit positive selection on digestive enzymes and glucose transporters
C_LIO_LIGlucose absorption differences can be explained by gut anatomy
C_LIO_LIExtreme blood glucose (&gt;650 mg/dL) coincides with constitutive expression of apical Slc2a2
C_LIO_LIThe regulation of Sglt1 and apical Slc2a2 highlights differences in blood glucose levels
C_LI

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=106 SRC=&#34;FIGDIR/small/547432v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@426655org.highwire.dtl.DTLVardef@14c9735org.highwire.dtl.DTLVardef@15b35f6org.highwire.dtl.DTLVardef@133d43b_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545890" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545890">
        <p class="paperTitle">Large-scale phylogenomics uncovers a complex evolutionary history and extensive ancestral gene flow in an African primate radiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jensen, A.; Swift, F.; de Vries, D.; Beck, R.; Kuderna, L. F. K.; Knauf, S.; Chuma, I. S.; Keyyu, J. D.; Kitchener, A. C.; Farh, K.; Rogers, J.; Marques-Bonet, T.; Detwiler, K. M.; Roos, C.; Guschanski, K.</p>
        <p class="info">Score: 22.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545890' target='https://doi.org/10.1101/2023.06.21.545890'> 10.1101/2023.06.21.545890</a></p>
        <p class="abstract">Understanding the drivers of speciation is fundamental in evolutionary biology, and recent studies highlight hybridization as a potential facilitator of adaptive radiations. Using whole-genome sequencing data from 22 species of guenons (tribe Cercopithecini), one of the worlds largest primate radiations, we show that rampant gene flow characterizes their evolutionary history, and identify ancient hybridization across deeply divergent lineages differing in ecology, morphology and karyotypes. Lineages experiencing gene flow tend to be more species-rich than non-admixed lineages. Mitochondrial transfers between distant lineages were likely facilitated by co-introgression of co-adapted nuclear variants. Although the genomic landscapes of introgression were largely lineage specific, we found that genes with immune functions were overrepresented in introgressing regions, in line with adaptive introgression, whereas genes involved in pigmentation and morphology may contribute to reproductive isolation. This study provides important insights into the prevalence, role and outcomes of ancestral hybridization in a large mammalian radiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.544921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.544921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.544921">
        <p class="paperTitle">Widespread but spotty: Atlantic cod is a complex of five species through ecological speciation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.544921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.544921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Halldorsdottir, K.; Llanos-Garrido, A.; Einarsson, A. W.; Sackton, T. B.; Arnason, E.</p>
        <p class="info">Score: 15.0, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.544921' target='https://doi.org/10.1101/2023.06.14.544921'> 10.1101/2023.06.14.544921</a></p>
        <p class="abstract">Marine environments are home to a significant portion of fish and other organisms biodiversity. However, our knowledge and empirical evidence of speciation in the marine realm is quite limited. Here we use genomics and study population differentiation and speciation in the Atlantic cod, a marine teleost of biological and economic importance. The Atlantic cod genome is liable to recurrent selective sweeps showing the importance of selective processes [1]. We show that the non-inversion part of the genome separates individuals into five clusters. At the four chromosomes carrying large inversions that define ecotypic variation, individuals also cluster into the same five clusters in conjunction with individuals inversion genotypes. Within each cluster, individuals homozygous or heterozygous for the inversions are divergent and there is an adaptive divergence of each type of inversion homozygote among clusters. These interchromosomal correlations imply the reproductive isolation of five cryptic species. We show that the inversion haplotypes are ancient [2] trans-species balanced polymorphisms that would evolve towards homozygous lethality under Mullers ratchet [3-5]. This and recombination suppression of inversions. However, gene-flux [6] from gene conversion [7] rescues homozygous fitness, and recombination and selection lead to the evolution of habitat specialists through ecological speciation [8, 9]. The specialized genotypes actively seek out their preferred habitat resulting in a widespread but patchy distribution that contradicts the traditional geographical speciation model and the stock concept in fisheries biology. Our results highlight the power of genomics in studying selection and speciation while informing conservation and management of essential fisheries resources.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548536" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548536">
        <p class="paperTitle">Patterns of recombination in snakes reveal a tug of war between PRDM9 and promoter-like features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoge, C. R.; de Manuel, M.; Mahgoub, M.; Okami, N.; Fuller, Z. L.; Banerjee, S.; Baker, Z.; Mcnulty, M.; Andolfatto, P.; Macfarlan, T. S.; Schumer, M.; Tzika, A. C.; Przeworski, M.</p>
        <p class="info">Score: 11.8, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548536' target='https://doi.org/10.1101/2023.07.11.548536'> 10.1101/2023.07.11.548536</a></p>
        <p class="abstract">In vertebrates, there are two known mechanisms by which meiotic recombination is directed to the genome: in humans, mice, and other mammals, recombination occurs almost exclusively where the protein PRDM9 binds, while in species lacking an intact PRDM9, such as birds and canids, recombination rates are elevated near promoter-like features. To test if PRDM9 also directs recombination in non-mammalian vertebrates, we focused on an exemplar species, the corn snake (Pantherophis guttatus). Unlike birds, this species possesses a single, intact PRDM9 ortholog. By inferring historical recombination rates along the genome from patterns of linkage disequilibrium and identifying crossovers in pedigrees, we found that PRDM9 specifies the location of recombination events outside of mammals. However, we also detected an independent effect of promoter-like features on recombination, which is more pronounced on macro- than microchromosomes. Thus, our findings reveal that the uses of PRDM9 and promoter-like features are not mutually-exclusive, and instead reflect a tug of war, which varies in strength along the genome and is more lopsided in some species than others.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
