<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 13.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 13.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 13.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 13.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe.",
  "keywords": [
    
  ],
  "articleBody": " Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 13.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.\nReproductive innovation enabled radiation in the deep sea during an ecological crisis\nAuthors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.\nScore: 105.9, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380\nMajor ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate. A phylogenomic reconstruction of the evolutionary history of anglerfishes provides a strong inference for the convergent evolution of permanently-fusing deep-sea anglerfishes and their degenerate immune genes. Our results support that sexual parasitism was enabled by the degeneration of adaptive immunity and ancestral sexual size dimorphism. The combination of these traits facilitated the transition of pelagic anglerfishes into novel ecologies available in the deep open oceans after evolving from benthic ancestors. These results show how seemingly unrelated physiological and reproductive traits interact synergistically to drive evolutionary radiation in novel environments.\nGenomic evidence for supergene control of Darwin's 'complex marriage arrangement' - the tristylous floral polymorphism\nAuthors: Xue, H.; Gong, Y.; Wright, S. I.; Barrett, S. C. H.\nScore: 6.6, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575583\nTristyly is a sexual polymorphism characterized by three flower morphs with reciprocal stigma and anther heights controlled by two epistatically interacting diallelic loci (S and M), hypothesized to be supergenes. Chromosome-level genome assemblies of Eichhornia paniculata identified the S- and M-loci. The S-locus is a supergene consisting of two divergent alleles: The S-allele (2.51 Mb), with three S-allele specific genes hemizygous in most S-morph plants and the s-allele (596 kb) with five s-allele specific genes. Two of the S-allele specific genes, LAZY1-S and HRGP-S, were specifically expressed in styles and stamens, respectively, making them tristyly candidate genes. The M-locus contains one gene (LAZY1-M) present in the M-allele but absent from the m-allele and which is homologous to LAZY1-S. Estimates of allele age are consistent with theory predicting that the S-locus evolved before the M-locus. Re-use of the same gene family highlights the potential role of gene duplication in the evolution of epistatic multilocus polymorphisms.\nRecurrent Evolutionary Innovations in Rodent and Primate Schlafen Genes\nAuthors: Mordier, J.; Fraisse, M.; Cohen-Tannoudji, M.; Molaro, A.\nScore: 20.0, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575368\nSCHLAFEN proteins are a large family of RNase-related enzymes carrying essential immune and developmental functions. Despite these important roles, Schlafen genes display varying degrees of evolutionary conservation in mammals. While this appears to influence their molecular activities, a detailed understanding of these evolutionary innovations is still lacking. Here, we used in depth phylogenomic approaches to characterize the evolutionary trajectories and selective forces shaping mammalian Schlafen genes. We traced lineage-specific Schlafen amplifications and found that recent duplicates evolved under distinct selective forces, supporting repeated sub-functionalization cycles. Codon-level natural selection analyses in primates and rodents, identified recurrent positive selection over Schlafen protein domains engaged in viral interactions. Combining crystal structures with machine learning predictions, we discovered a novel class of rapidly evolving residues enriched at the contact interface of SCHLAFEN protein dimers. Our results suggest that inter Schlafen compatibilities are under strong selective pressures and are likely to impact their molecular functions. We posit that cycles of genetic conflicts with pathogens and between paralogs drove Schlafens recurrent evolutionary innovations in mammals.\nParasite contingency loci and the evolution of host specificity: Simple sequence repeats mediate Bartonella adaptation to a wild rodent host\nAuthors: Rodriguez-Pastor, R.; Knossow, N.; Shahar, N.; Hasik, A. Z.; Deatherage, D. E.; Gutierrez, R.; Harrus, S.; Zaman, L. E.; Lenski, R. E.; Barrick, J. E.; Hawlena, H.\nScore: 5.5, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576196\nParasites can adapt to better exploit their hosts on many timescales, ranging from within a single infection to across serial infections of multiple hosts. However, little is known about how the genomes of parasites in natural communities evolve when they face diverse hosts. We investigated how Bartonella bacteria that circulate in rodent communities in the dunes of the Negev Desert in Israel adapt to different hosts. We propagated 15 Bartonella populations through infections of either a single rodent species (Gerbillus andersoni or Gerbillus pyramidum) or alternating between the two. After 20 rodent passages, strains with de novo mutations outcompeted the ancestor in most lines. Mutations in two mononucleotide simple sequence repeats (SSRs) dominated the evolutionary dynamics and caused frameshifts in the same adhesin gene. These mutations appeared exclusively in populations that encountered G. andersoni, and they improved the ability to exploit this host. Similar SSRs in other genes are conserved and exhibit ON/OFF variation in Bartonella isolates from the Negev Desert dunes. Our results suggest that SSR-based contingency loci are important not only for rapidly and reversibly generating antigenic variation to escape immune responses but may also play a prominent role in the evolution of host specificity.\nAncestral Sequence Reconstruction as a tool to detect and study de novo gene emergence\nAuthors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.\nScore: 29.6, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573862\nNew protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (\u003c1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute \"easy\" cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.\nCell type and regulatory analysis in amphioxus illuminates evolutionary origin of the vertebrate head\nAuthors: Markos, A.; Kubovciak, J.; Mikula Mrstakova, S.; Paces, J.; Zitova, A.; Kozmik, Z.; Machacova, S.; Kozmik, Z.; Kozmikova, I.\nScore: 4.8, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576194\nTo shed light on the enigmatic origin of the vertebrate head, our study employs an integrated approach that combines single-cell transcriptomics, perturbations in signalling pathways, and cis-regulatory analysis in amphioxus, a close relative of chordate common ancestor. Through cell type characterization, we identified the presence of a prechordal plate, pre-migratory and migratory neural crest-like cell populations in the developing amphioxus embryo. Functional analysis established conserved roles of the Nodal and Hedgehog signalling pathways in prechordal plate, and of Wnt signalling pathway in neural crest development. Furthermore, the trans-species transgenic experiments provided evidence of cis-regulatory level homology within the chordate lineage. Our findings provide evidence that the key features of vertebrate head development can be traced back to the common ancestor of all chordates. One Sentence SummaryCell populations forming the vertebrate head are present in the close relative of chordate common ancestor.\nEmpirical evidence of resource dependent evolution of payoff matrices in Saccharomyces cerevisiae populations.\nAuthors: Venkataraman, P.; Mahilkar, A.; Raj, N.; Saini, S.\nScore: 4.7, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575778\nIn evolutionary game theory, a relative comparison of the cost and benefit associated with obtaining a resource, called payoff, is used as an indicator of fitness of an organism. Such payoff matrices are used to understand complex inter-species and intra-species interactions like cooperation, mutualism, and altruism. In the absence of any empirical data, the evolution of these payoff matrices has been investigated theoretically by tweaking well-established game theory models. In this paper, we present empirical evidence of three types of resource-dependent changes in the payoff matrices of evolving Saccharomyces cerevisiae populations. We show that depending on the carbon source and participating genotypes, the payoff matrix could either (a) evolve quantitatively yet maintain a cheater-cooperator game, (b) change qualitatively such that the cheater-cooperator game collapses, or (c) change qualitatively to result in the birth of a cheater-cooperator game. Our results highlight the need to consider the dynamic nature of payoff matrices while making even short-term predictions about population interactions and dynamics.\nMicrobial diversity and ecological complexity emerging from environmental variation and horizontal gene transfer in a simple mathematical model.\nAuthors: Babajanyan, S. G.; Garushyants, S. K.; Wolf, Y. I.; Koonin, E. V.\nScore: 4.0, Published: 2024-01-21 DOI: 10.1101/2024.01.17.576128\nMicrobiomes are generally characterized by high diversity of coexisting microbial species and strains that remains stable within a broad range of conditions. However, under fixed conditions, microbial ecology conforms with the exclusion principle under which two populations competing for the same resource within the same niche cannot coexist because the less fit population inevitably goes extinct. To explore the conditions for stabilization of microbial diversity, we developed a simple mathematical model consisting of two competing populations that could exchange a single gene allele via horizontal gene transfer (HGT). We found that, although in a fixed environment, with unbiased HGT, the system obeyed the exclusion principle, in an oscillating environment, within large regions of the phase space bounded by the rates of reproduction and HGT, the two populations coexist. Moreover, depending on the parameter combination, all three major types of symbiosis obtained, namely, pure competition, host-parasite relationship and mutualism. In each of these regimes, certain parameter combinations provided for synergy, that is, a greater total abundance of both populations compared to the abundance of the winning population in the fixed environments. These findings show that basic phenomena that are universal in microbial communities, environmental variation and HGT, provide for stabilization of microbial diversity and ecological complexity.\nConvergent reductive evolution of cyanobacteria in symbiosis with Dinophysiales dinoflagellates\nAuthors: Nakayama, T.; Nomura, M.; Yabuki, A.; Shiba, K.; Inaba, K.; Inagaki, Y.\nScore: 4.8, Published: 2024-01-11 DOI: 10.1101/2024.01.11.574452\nThe diversity of marine cyanobacteria has been extensively studied due to their vital roles in ocean primary production. However, little is understood about the diversity of cyanobacterial species involved in symbiotic relationships. In this study, we successfully sequenced the complete genome of a cyanobacterium in symbiosis with Citharistes regius, a dinoflagellate species thriving in the open ocean. A phylogenomic analysis revealed that the cyanobacterium (CregCyn) belongs to the marine picocyanobacterial lineage, akin to another cyanobacterial symbiont (OmCyn) of a different dinoflagellate closely related to Citharistes. Nevertheless, these two symbionts are distinct lineages, suggesting independent origins of their symbiotic lifestyles. Despite the distinct origins, the genome analyses of CregCyn revealed shared characteristics with OmCyn, including an obligate symbiotic relationship with the host dinoflagellates and a degree of genome reduction. In contrast, a detailed analysis of genome subregions unveiled that the CregCyn genome carries genomic islands that are not found in the OmCyn genome. The presence of the genomic islands implies that exogenous genes have been integrated into the CregCyn genome at some point in its evolution. This study contributes to our understanding of the complex history of the symbiosis between dinoflagellates and cyanobacteria, as well as the genomic diversity of marine picocyanobacteria.\n",
  "wordCount" : "2256",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576074">
        <p class="paperTitle">Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cain, F. E.; Yamaguchi, K.</p>
        <p class="info">Score: 13.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576074' target='https://doi.org/10.1101/2024.01.17.576074'> 10.1101/2024.01.17.576074</a></p>
        <p class="abstract">The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575380">
        <p class="paperTitle">Reproductive innovation enabled radiation in the deep sea during an ecological crisis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.</p>
        <p class="info">Score: 105.9, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575380' target='https://doi.org/10.1101/2024.01.12.575380'> 10.1101/2024.01.12.575380</a></p>
        <p class="abstract">Major ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate. A phylogenomic reconstruction of the evolutionary history of anglerfishes provides a strong inference for the convergent evolution of permanently-fusing deep-sea anglerfishes and their degenerate immune genes. Our results support that sexual parasitism was enabled by the degeneration of adaptive immunity and ancestral sexual size dimorphism. The combination of these traits facilitated the transition of pelagic anglerfishes into novel ecologies available in the deep open oceans after evolving from benthic ancestors. These results show how seemingly unrelated physiological and reproductive traits interact synergistically to drive evolutionary radiation in novel environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575583" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575583">
        <p class="paperTitle">Genomic evidence for supergene control of Darwin&#39;s &#39;complex marriage arrangement&#39; - the tristylous floral polymorphism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xue, H.; Gong, Y.; Wright, S. I.; Barrett, S. C. H.</p>
        <p class="info">Score: 6.6, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575583' target='https://doi.org/10.1101/2024.01.15.575583'> 10.1101/2024.01.15.575583</a></p>
        <p class="abstract">Tristyly is a sexual polymorphism characterized by three flower morphs with reciprocal stigma and anther heights controlled by two epistatically interacting diallelic loci (S and M), hypothesized to be supergenes. Chromosome-level genome assemblies of Eichhornia paniculata identified the S- and M-loci. The S-locus is a supergene consisting of two divergent alleles: The S-allele (2.51 Mb), with three S-allele specific genes hemizygous in most S-morph plants and the s-allele (596 kb) with five s-allele specific genes. Two of the S-allele specific genes, LAZY1-S and HRGP-S, were specifically expressed in styles and stamens, respectively, making them tristyly candidate genes. The M-locus contains one gene (LAZY1-M) present in the M-allele but absent from the m-allele and which is homologous to LAZY1-S. Estimates of allele age are consistent with theory predicting that the S-locus evolved before the M-locus. Re-use of the same gene family highlights the potential role of gene duplication in the evolution of epistatic multilocus polymorphisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575368" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575368">
        <p class="paperTitle">Recurrent Evolutionary Innovations in Rodent and Primate Schlafen Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mordier, J.; Fraisse, M.; Cohen-Tannoudji, M.; Molaro, A.</p>
        <p class="info">Score: 20.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575368' target='https://doi.org/10.1101/2024.01.12.575368'> 10.1101/2024.01.12.575368</a></p>
        <p class="abstract">SCHLAFEN proteins are a large family of RNase-related enzymes carrying essential immune and developmental functions. Despite these important roles, Schlafen genes display varying degrees of evolutionary conservation in mammals. While this appears to influence their molecular activities, a detailed understanding of these evolutionary innovations is still lacking. Here, we used in depth phylogenomic approaches to characterize the evolutionary trajectories and selective forces shaping mammalian Schlafen genes. We traced lineage-specific Schlafen amplifications and found that recent duplicates evolved under distinct selective forces, supporting repeated sub-functionalization cycles. Codon-level natural selection analyses in primates and rodents, identified recurrent positive selection over Schlafen protein domains engaged in viral interactions. Combining crystal structures with machine learning predictions, we discovered a novel class of rapidly evolving residues enriched at the contact interface of SCHLAFEN protein dimers. Our results suggest that inter Schlafen compatibilities are under strong selective pressures and are likely to impact their molecular functions. We posit that cycles of genetic conflicts with pathogens and between paralogs drove Schlafens recurrent evolutionary innovations in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576196" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576196">
        <p class="paperTitle">Parasite contingency loci and the evolution of host specificity: Simple sequence repeats mediate Bartonella adaptation to a wild rodent host</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodriguez-Pastor, R.; Knossow, N.; Shahar, N.; Hasik, A. Z.; Deatherage, D. E.; Gutierrez, R.; Harrus, S.; Zaman, L. E.; Lenski, R. E.; Barrick, J. E.; Hawlena, H.</p>
        <p class="info">Score: 5.5, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576196' target='https://doi.org/10.1101/2024.01.18.576196'> 10.1101/2024.01.18.576196</a></p>
        <p class="abstract">Parasites can adapt to better exploit their hosts on many timescales, ranging from within a single infection to across serial infections of multiple hosts. However, little is known about how the genomes of parasites in natural communities evolve when they face diverse hosts. We investigated how Bartonella bacteria that circulate in rodent communities in the dunes of the Negev Desert in Israel adapt to different hosts. We propagated 15 Bartonella populations through infections of either a single rodent species (Gerbillus andersoni or Gerbillus pyramidum) or alternating between the two. After 20 rodent passages, strains with de novo mutations outcompeted the ancestor in most lines. Mutations in two mononucleotide simple sequence repeats (SSRs) dominated the evolutionary dynamics and caused frameshifts in the same adhesin gene. These mutations appeared exclusively in populations that encountered G. andersoni, and they improved the ability to exploit this host. Similar SSRs in other genes are conserved and exhibit ON/OFF variation in Bartonella isolates from the Negev Desert dunes. Our results suggest that SSR-based contingency loci are important not only for rapidly and reversibly generating antigenic variation to escape immune responses but may also play a prominent role in the evolution of host specificity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573862" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573862">
        <p class="paperTitle">Ancestral Sequence Reconstruction as a tool to detect and study de novo gene emergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.</p>
        <p class="info">Score: 29.6, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573862' target='https://doi.org/10.1101/2024.01.02.573862'> 10.1101/2024.01.02.573862</a></p>
        <p class="abstract">New protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (&lt;1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute &#34;easy&#34; cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576194" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576194">
        <p class="paperTitle">Cell type and regulatory analysis in amphioxus illuminates evolutionary origin of the vertebrate head</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Markos, A.; Kubovciak, J.; Mikula Mrstakova, S.; Paces, J.; Zitova, A.; Kozmik, Z.; Machacova, S.; Kozmik, Z.; Kozmikova, I.</p>
        <p class="info">Score: 4.8, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576194' target='https://doi.org/10.1101/2024.01.18.576194'> 10.1101/2024.01.18.576194</a></p>
        <p class="abstract">To shed light on the enigmatic origin of the vertebrate head, our study employs an integrated approach that combines single-cell transcriptomics, perturbations in signalling pathways, and cis-regulatory analysis in amphioxus, a close relative of chordate common ancestor. Through cell type characterization, we identified the presence of a prechordal plate, pre-migratory and migratory neural crest-like cell populations in the developing amphioxus embryo. Functional analysis established conserved roles of the Nodal and Hedgehog signalling pathways in prechordal plate, and of Wnt signalling pathway in neural crest development. Furthermore, the trans-species transgenic experiments provided evidence of cis-regulatory level homology within the chordate lineage. Our findings provide evidence that the key features of vertebrate head development can be traced back to the common ancestor of all chordates.

One Sentence SummaryCell populations forming the vertebrate head are present in the close relative of chordate common ancestor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575778" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575778">
        <p class="paperTitle">Empirical evidence of resource dependent evolution of payoff matrices in Saccharomyces cerevisiae populations.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkataraman, P.; Mahilkar, A.; Raj, N.; Saini, S.</p>
        <p class="info">Score: 4.7, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575778' target='https://doi.org/10.1101/2024.01.15.575778'> 10.1101/2024.01.15.575778</a></p>
        <p class="abstract">In evolutionary game theory, a relative comparison of the cost and benefit associated with obtaining a resource, called payoff, is used as an indicator of fitness of an organism. Such payoff matrices are used to understand complex inter-species and intra-species interactions like cooperation, mutualism, and altruism. In the absence of any empirical data, the evolution of these payoff matrices has been investigated theoretically by tweaking well-established game theory models. In this paper, we present empirical evidence of three types of resource-dependent changes in the payoff matrices of evolving Saccharomyces cerevisiae populations. We show that depending on the carbon source and participating genotypes, the payoff matrix could either (a) evolve quantitatively yet maintain a cheater-cooperator game, (b) change qualitatively such that the cheater-cooperator game collapses, or (c) change qualitatively to result in the birth of a cheater-cooperator game. Our results highlight the need to consider the dynamic nature of payoff matrices while making even short-term predictions about population interactions and dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576128" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576128">
        <p class="paperTitle">Microbial diversity and ecological complexity emerging from environmental variation and horizontal gene transfer in a simple mathematical model.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Babajanyan, S. G.; Garushyants, S. K.; Wolf, Y. I.; Koonin, E. V.</p>
        <p class="info">Score: 4.0, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576128' target='https://doi.org/10.1101/2024.01.17.576128'> 10.1101/2024.01.17.576128</a></p>
        <p class="abstract">Microbiomes are generally characterized by high diversity of coexisting microbial species and strains that remains stable within a broad range of conditions. However, under fixed conditions, microbial ecology conforms with the exclusion principle under which two populations competing for the same resource within the same niche cannot coexist because the less fit population inevitably goes extinct. To explore the conditions for stabilization of microbial diversity, we developed a simple mathematical model consisting of two competing populations that could exchange a single gene allele via horizontal gene transfer (HGT). We found that, although in a fixed environment, with unbiased HGT, the system obeyed the exclusion principle, in an oscillating environment, within large regions of the phase space bounded by the rates of reproduction and HGT, the two populations coexist. Moreover, depending on the parameter combination, all three major types of symbiosis obtained, namely, pure competition, host-parasite relationship and mutualism. In each of these regimes, certain parameter combinations provided for synergy, that is, a greater total abundance of both populations compared to the abundance of the winning population in the fixed environments. These findings show that basic phenomena that are universal in microbial communities, environmental variation and HGT, provide for stabilization of microbial diversity and ecological complexity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.574452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.574452" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.574452">
        <p class="paperTitle">Convergent reductive evolution of cyanobacteria in symbiosis with Dinophysiales dinoflagellates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.574452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.574452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakayama, T.; Nomura, M.; Yabuki, A.; Shiba, K.; Inaba, K.; Inagaki, Y.</p>
        <p class="info">Score: 4.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.574452' target='https://doi.org/10.1101/2024.01.11.574452'> 10.1101/2024.01.11.574452</a></p>
        <p class="abstract">The diversity of marine cyanobacteria has been extensively studied due to their vital roles in ocean primary production. However, little is understood about the diversity of cyanobacterial species involved in symbiotic relationships. In this study, we successfully sequenced the complete genome of a cyanobacterium in symbiosis with Citharistes regius, a dinoflagellate species thriving in the open ocean. A phylogenomic analysis revealed that the cyanobacterium (CregCyn) belongs to the marine picocyanobacterial lineage, akin to another cyanobacterial symbiont (OmCyn) of a different dinoflagellate closely related to Citharistes. Nevertheless, these two symbionts are distinct lineages, suggesting independent origins of their symbiotic lifestyles. Despite the distinct origins, the genome analyses of CregCyn revealed shared characteristics with OmCyn, including an obligate symbiotic relationship with the host dinoflagellates and a degree of genome reduction. In contrast, a detailed analysis of genome subregions unveiled that the CregCyn genome carries genomic islands that are not found in the OmCyn genome. The presence of the genomic islands implies that exogenous genes have been integrated into the CregCyn genome at some point in its evolution. This study contributes to our understanding of the complex history of the symbiosis between dinoflagellates and cyanobacteria, as well as the genomic diversity of marine picocyanobacteria.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
