<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus
Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.",
  "keywords": [
    
  ],
  "articleBody": " Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus\nAuthors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.\nScore: 34.1, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547326\nThe evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.\nDeep Divergences Among Cryptic Clades of Epipedobates Poison Frogs\nAuthors: Lopez-Hervas, K.; Santos, J. C.; Ron, S. R.; Betancourth-Cundar, M.; Cannatella, D. C.; Tarvin, R. D.\nScore: 32.1, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547117\nPoison frogs (Dendrobatidae) are famous for their aposematic species, having a combination of diverse color patterns and defensive skin toxins, yet most species in this family are inconspicuous (cryptic) and considered non-aposematic. Epipedobates is among the youngest genus-level clades of Dendrobatidae that includes both aposematic and cryptic species. Using mitochondrial and nuclear markers, we demonstrate deep genetic divergences among cryptic species of Epipedobates but relatively shallow genetic divergences among conspicuous species. Our phylogenetic analysis includes broad geographic sampling of the cryptic lineages typically identified as E. boulengeri and E. espinosai, which reveals two putative new species, one in west-central Colombia (E. sp. 1) and the other in north-central Ecuador (E. aff. espinosai). We conclude that E. darwinwallacei is a junior subjective synonym of E. espinosai. We also clarify the geographic distributions of cryptic Epipedobates species including the widespread E. boulengeri. We provide a qualitative assessment of the phenotypic diversity in each nominal species, with a focus on the color and pattern of cryptic species. We conclude that Epipedobates contains eight known valid species, six of which are cryptic. A relaxed molecular clock analysis suggests that the most recent common ancestor of Epipedobates is ~11.1 million years old, which nearly doubles previous estimates. Last, genetic information points to a center of species diversity in the Choco at the southwestern border of Colombia with Ecuador.\nSeveral independent adaptations of archaea to hypersaline environments\nAuthors: Baker, B. A.; Gutierrez-Preciado, A.; Rodriguez del Rio, A.; McCarthy, C.; Lopez-Garcia, P.; Huerta-Cepas, J.; Susko, E.; Roger, A. J.; Eme, L.; Moreira, D.\nScore: 25.3, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547478\nSeveral archaeal lineages thrive in high, saturating salt concentrations. These extremely halophilic archaea, including Halobacteria, Nanohaloarchaeota, Methanonatronarchaeia, and Haloplasmatales, must maintain osmotic equilibrium with their environment. For this, they use a 'salt-in' strategy, which involves pumping molar concentrations of potassium into the cells, which, in turn, has led to extensive proteome-wide modifications to prevent protein aggregation. However, the evolutionary history underlying these adaptations remains poorly understood. In particular, the number of times that these dramatic proteome-sweeping changes occurred is unclear due to the conflicting phylogenetic positions found for several of these lineages. Here, we present a resolved phylogeny of extremely halophilic archaea obtained using improved taxon sampling and state-of-the-art phylogenetic approaches designed to cope with the strong compositional biases of their proteomes. We describe two new uncultured lineages, Afararchaeaceae and Asboarchaeaceae, which break the long branches at the base of Haloarchaea and Nanohaloarchaeota, respectively. Our extensive phylogenomic analyses show that at least four independent adaptations to extreme halophily occurred during archaeal evolution. Finally, gene-tree/species-tree reconciliation suggests that gene duplication and horizontal gene transfer played an important role in this process, for example, by spreading key genes (such as those encoding potassium transporters) across the various extremely halophilic lineages.\nStanding genetic variation and de novo mutations underlie parallel evolution of island bird phenotypes\nAuthors: Estandia, A.; Sendell-Price, A. T.; Robertson, B. C.; Clegg, S. M.\nScore: 19.4, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546893\nParallel evolution occurs when the same trait evolves in closely related lineages in response to similar ecological contexts and provides some of the best examples of determinism in evolutionary biology. However, from a genetic standpoint, this process can be driven by either new mutations that appear independently in each diverging population or by selection on existing genetic variation common to both lineages. Small bodied birds, for example, tend to increase in size after they colonise a new island, following what is known as the island rule. Such is the case of the Silvereye, a prolific natural coloniser of southwest Pacific islands. Island forms of this bird species increase in body size after they establish, with the pattern and pace of change consistent with directional natural selection and evident even in the most recent colonisations within the last 200 years. The system provides an exceptional opportunity to explore the genomic basis of repeated body size evolution. We sequenced 377 whole genomes from 31 different Silvereye populations, which revealed that both mechanisms are at play: in some lineages, new mutations are highly associated with body and bill size, but there are also highly associated polymorphisms present across all populations. Our research sheds light on the genomic basis of repeated body size evolution and emphasises that multiple molecular mechanisms can underlie similar evolutionary trajectories even within a single taxon.\nReconstructing the evolutionary history of human, simian, and prosimian immunodeficiency viruses\nAuthors: Ghafari, M.; Simmonds, P.; Katzourakis, A.\nScore: 19.3, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546833\nWhile paleovirological evidence supports the idea that the ancestors of modern simian (SIV) and prosimian (pSIV) immunodeficiency viruses have been evolving in nonhuman primates for millions of years, standard molecular clock methods calibrated using contemporary sequences significantly underestimate their times of origin. This discrepancy is attributed to the time-dependent nature of evolutionary rate estimates whereby the rate of virus evolution varies with respect to the time window of its measurement. While biogeographical calibrations may provide better estimates across intermediate timescales, there is currently no unifying framework that can allow inference of lentiviral ages across all relevant timescales. Here we showed that by correcting for such time-dependent rate effects using the recently developed Prisoner-of-War evolutionary model, we can successfully reconstruct the evolutionary history of human, simian, and prosimian immunodeficiency viruses across vastly different timescales and allowed estimates of the age of lineages that gave rise to HIV-1 and HIV-2. The model also predicted that the most recent common ancestor of SIV and pSIV lived around 21 million years ago, suggesting that the most likely explanation for the origins of primate lentiviruses is a terrestrial transfer of lentiviruses between African and Malagasy primates during the last episode of colonisation of Madagascar. Infections have entered via a land bridge in a group of mammals or through a nonprimate vector or transfer mediated by an aerial vector species. We also found that the most recent common ancestor of SIVs lived nearly a million years ago and that some SIV lineages codiverged with their hosts for several hundreds of thousands of years. The predicted long evolutionary timescales of SIVs and potential for close virus/host co-adaptation is consistent with their reduced or minimal pathogenicity in their native hosts, contrasting with the very recent evolutionary origins of highly pathogenic HIV strains in humans.\nAncient Sheep Genomes reveal four Millennia of North European Short-Tailed Sheep in the Baltic Sea region\nAuthors: Larsson, M. N.; Morell Miranda, P.; Pan, L.; Basak Vural, K.; Kaptan, D.; Rodrigues Soares, A. E.; Kivikero, H.; Kantanen, J.; Somel, M.; Özer, F.; Johansson, A. M.; Stora, J.; Günther, T.\nScore: 20.9, Published: 2023-06-27 DOI: 10.1101/2023.06.26.544912\nSheep are among the earliest domesticated livestock species, with a wide variety of breeds present today. However, it remains unclear how far back this breed diversity goes, with formal documentation only dating back a few centuries. North European short-tailed breeds are often assumed to be among the oldest domestic sheep populations, even thought to represent relicts of the earliest sheep expansions during the Neolithic period reaching Scandinavia less than 6000 years ago. This study sequenced the genomes (up to 11.6X) of five sheep remains from the Baltic islands of Gotland and [A]land, dating from Late Neolithic ([~]4100 calBP) to historical times ([~]1600 CE). Our findings indicate that these ancient sheep largely possessed the genetic characteristics of modern North European short-tailed breeds, suggesting a substantial degree of long-term continuity of this breed type in the Baltic Sea region. Despite the wide temporal spread, population genetic analyses show high levels of affinity between the ancient genomes and they also exhibit higher genetic diversity when compared to modern breeds, implying a loss of diversity in recent centuries associated with breed formation. Finally, we see a potential signature of an even earlier, genetically different form of sheep in Scandinavia as these samples do not represent the first sheep in Northern Europe. Our results shed light on the development of breeds in Northern Europe specifically as well as the development of genetic diversity in sheep breeds, and their expansion from the domestication center in general.\nThe genome sequence of the Montseny horsehair worm, Gordionus montsenyensis sp. nov., a key resource to investigate Ecdysozoa evolution\nAuthors: Eleftheriadi, K.; Guiglielmoni, N.; Salces-Ortiz, J.; Vargas-Chavez, C.; Martinez-Redondo, G. I.; Gut, M.; Flot, J.-F.; Schmidt-Rhaesa, A.; Fernandez, R.\nScore: 14.9, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546503\nNematomorpha, also known as Gordiacea or Gordian worms, are a phylum of parasitic organisms that belong to the Ecdysozoa, a clade of invertebrate animals characterized by molting. They are one of the most scientifically neglected animal phyla, and many aspects of their biology and evolution are still unknown, partially due to the lack of genomic resources for this phylum. As part of the European Reference Genome Atlas pilot effort to generate reference genomes for European biodiversity, we present the taxonomic description and chromosome-level genome assembly of a newly described species of Nematomorpha (Gordionus montsenyensis Schmidt-Rhaesa \u0026 Fernandez sp. nov.). The final assembly has a total length of 288 Mb in 398 scaffolds with an N50 of 52.6 Mb, 96% of which is scaffolded into 6 pseudochromosomes. The circular mitochondrial genome was also assembled into a 15-kilobases sequence. Gene annotation predicted 10,819 protein-coding genes in the nuclear genome. In this piece of work, we contribute a key genomic resource to not only explore the evolution of Ecdysozoa, but also to further our understanding on the genomic basis of parasitic lifestyles. In addition, we describe a species new to science from this enigmatic animal phyla.\nEpistasis Arises from Shifting the Rate-Limiting Step during Enzyme Evolution\nAuthors: Frohlich, C.; Bunzel, H. A.; Buda, K.; Mulholland, A. J.; van der Kamp, M. W.; Johnsen, P. J.; Leiros, H.-K. S.; Tokuriki, N.\nScore: 14.5, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547057\nThe molecular mechanisms by which epistasis boosts enzyme activity remain elusive, undermining our ability to predict the evolution of pathogens and engineer novel biocatalysts. Here, we reveal how directed evolution of a {beta}-lactamase yielded highly epistatic activity enhancements. Evolution selected four mutations that increase antibiotic resistance 40-fold, despite their marginal individual effects ([\u0026le;] 2-fold). Synergistic improvements coincided with the introduction of super-stochiometric burst kinetics, indicating that epistasis is rooted in the enzymes conformational dynamics. Kinetic, structural, and dynamical analyses reveal that epistasis was driven by distinct effects of each mutation on the catalytic cycle. The first mutation acquired during evolution increases protein flexibility and accelerates substrate binding, which is rate-limiting in the wild-type enzyme. The ensuing mutations predominantly boosted the chemical steps by fine-tuning substrate interactions. Our work identifies an overlooked cause for epistasis: changing the rate-limiting step can result in substantial positive synergy boosting enzyme activity.\nHigh germline mutation rates but not extreme population size outbreaks influence genetic diversity in crown-of-thorns sea stars\nAuthors: Popovic, I.; Bergeron, L.; Bozec, Y.-M.; Waldvogel, A.-M.; Howitt, S. M.; Damjanovic, K.; Patel, F.; Cabrera, M. G.; Wörheide, G.; Uthicke, S.; Riginos, C.\nScore: 12.6, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546961\nLewontin's paradox, the observation that levels of genetic diversity ({pi}) among animals do not scale linearly with variation in census population sizes (Nc), is an evolutionary conundrum, where the most extreme mismatches between {pi} and Nc are found for highly abundant marine invertebrates. Yet, whether new mutations influence {pi} relative to extrinsic processes remains unknown for most taxa. Here, we provide the first direct germline mutation rate () estimate for a marine invertebrate, using high-coverage (60x) whole-genome sequencing of wild-caught Acanthaster cf. solaris crown-of-thorns sea stars (Echinodermata). We also provide empirical estimates of adult Nc in Australia's Great Barrier Reef to jointly examine the determinants of {pi}. Based on direct observations of 63 de novo mutations across 14 parent-offspring trios, the A. cf. solaris mean was 9.13 x 10-09 mutations per-site per-generation (95% CI: 6.51 x 10-09 to 1.18 x 10-08). This value exceeds estimates for other invertebrates, showing greater concordance with reported vertebrate germline mutation rates. Lower-than-expected Ne (~70,000-180,000) and low Ne/Nc values (0.0047-0.048) indicated significant genetic drift and weak influences of contemporary population outbreaks on long-term {pi}. Our findings of elevated and low Ne in A. cf. solaris may help explain high mutational loads and extreme polymorphism levels observed in some marine invertebrate taxa and are consistent with evolving in response to Ne (drift-barrier hypothesis). This study advances our understanding of the processes controlling levels of natural genetic variation and provides new data valuable for further testing hypotheses about mutation rate evolution across animal phyla.\nEnrichment of hard sweeps on the X chromosome compared to autosomes in six Drosophila species\nAuthors: Harris, M.; Kim, B. Y.; Garud, N. R.\nScore: 12.3, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545888\nThe X chromosome, being hemizygous in males, is exposed one third of the time increasing the visibility of new mutations to natural selection, potentially leading to different evolutionary dynamics than autosomes. Recently, we found an enrichment of hard selective sweeps over soft selective sweeps on the X chromosome relative to the autosomes in a North American population of Drosophila melanogaster. To understand whether this enrichment is a universal feature of evolution on the X chromosome, we analyze diversity patterns across six commonly studied Drosophila species. We find an increased proportion of regions with steep reductions in diversity on the X chromosome compared to autosomes, accompanied by elevated homozygosity and an increased proportion of singletons. To assess if these signatures are consistent with positive selection, we simulate a wide variety of evolutionary scenarios spanning variations in demography, mutation rate, recombination rate, background selection, hard sweeps, and soft sweeps, and find that the diversity patterns observed on the X are most consistent with hard sweeps. Our findings highlight the importance of sex chromosomes in driving evolutionary processes and suggest that hard sweeps have played a significant role in shaping diversity patterns on the X chromosome across multiple Drosophila species. SignificanceUnderstanding the unique evolutionary dynamics of the X chromosome is a longstanding question in evolutionary biology given its central role in speciation, fertility, and sexual dimorphism. Recently, we found that hard sweeps, which are indicative of gradual adaptation, are more common on the X chromosome than on the autosomes in a North American population of Drosophila melanogaster. This is because new mutations on the X chromosome are more visible to natural selection due to male hemizygosity. Here, we analyze multiple whole genome sequences from six Drosophila species and find that this pattern is common across all species. Our study demonstrates the power of comparative population genetics in unraveling the forces shaping genetic variation within and between species.\n",
  "wordCount" : "2770",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547326">
        <p class="paperTitle">Enamel proteins reveal biological sex and genetic variability within southern African Paranthropus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madupe, P. P.; Koenig, C.; Patramanis, I.; Rüther, P. L.; Hlazo, N.; Mackie, M.; Tawane, M.; Krueger, J.; Taurozzi, A. J.; Troche, G. B.; Kibii, J.; Pickering, R.; Dickinson, M. R.; Sahle, Y.; Kgotleng, D.; Musiba, C.; Manthi, F.; Bell, L.; du Plessis, M.; Gilbert, C.; Zipfel, B.; Kuderna, L.; Lizano, E.; Welker, F.; Kyriakidou, P.; Cox, J.; Mollereau, C.; Tokarski, C.; Blackburn, J.; Ramos-Madrigal, J.; Marques-Bonet, T.; Penkman, K. E. H.; Zanolli, C.; Schroeder, L.; Racimo, F.; Olsen, J. V.; Ackermann, R. R.; Cappellini, E.</p>
        <p class="info">Score: 34.1, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547326' target='https://doi.org/10.1101/2023.07.03.547326'> 10.1101/2023.07.03.547326</a></p>
        <p class="abstract">The evolutionary relationships among extinct African hominin taxa are highly debated and largely unresolved, due in part to a lack of molecular data. Even within taxa, it is not always clear, based on morphology alone, whether ranges of variation are due to sexual dimorphism versus potentially undescribed taxonomic diversity. For Paranthropus robustus, a Pleistocene hominin found only in South Africa, both phylogenetic relationships to other taxa 1,2 and the nature of intraspecific variation 3-6 are still disputed. Here we report the mass spectrometric (MS) sequencing of enamel proteomes from four ca. 2 million year (Ma) old dental specimens attributed morphologically to P. robustus, from the site of Swartkrans. The identification of AMELY-specific peptides and semi-quantitative MS data analysis enabled us to determine the biological sex of all the specimens. Our combined molecular and morphometric data also provide compelling evidence of a significant degree of variation within southern African Paranthropus, as previously suggested based on morphology alone 6. Finally, the molecular data also confirm the taxonomic placement of Paranthropus within the hominin clade. This study demonstrates the feasibility of recovering informative Early Pleistocene hominin enamel proteins from Africa. Crucially, it also shows how the analysis of these proteins can contribute to understanding whether hominin morphological variation is due to sexual dimorphism or to taxonomic differences. We anticipate that this approach can be widely applied to geologically-comparable sites within South Africa, and possibly more broadly across the continent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547117">
        <p class="paperTitle">Deep Divergences Among Cryptic Clades of Epipedobates Poison Frogs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez-Hervas, K.; Santos, J. C.; Ron, S. R.; Betancourth-Cundar, M.; Cannatella, D. C.; Tarvin, R. D.</p>
        <p class="info">Score: 32.1, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547117' target='https://doi.org/10.1101/2023.06.29.547117'> 10.1101/2023.06.29.547117</a></p>
        <p class="abstract">Poison frogs (Dendrobatidae) are famous for their aposematic species, having a combination of diverse color patterns and defensive skin toxins, yet most species in this family are inconspicuous (cryptic) and considered non-aposematic. Epipedobates is among the youngest genus-level clades of Dendrobatidae that includes both aposematic and cryptic species. Using mitochondrial and nuclear markers, we demonstrate deep genetic divergences among cryptic species of Epipedobates but relatively shallow genetic divergences among conspicuous species. Our phylogenetic analysis includes broad geographic sampling of the cryptic lineages typically identified as E. boulengeri and E. espinosai, which reveals two putative new species, one in west-central Colombia (E. sp. 1) and the other in north-central Ecuador (E. aff. espinosai). We conclude that E. darwinwallacei is a junior subjective synonym of E. espinosai. We also clarify the geographic distributions of cryptic Epipedobates species including the widespread E. boulengeri. We provide a qualitative assessment of the phenotypic diversity in each nominal species, with a focus on the color and pattern of cryptic species. We conclude that Epipedobates contains eight known valid species, six of which are cryptic. A relaxed molecular clock analysis suggests that the most recent common ancestor of Epipedobates is ~11.1 million years old, which nearly doubles previous estimates. Last, genetic information points to a center of species diversity in the Choco at the southwestern border of Colombia with Ecuador.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547478">
        <p class="paperTitle">Several independent adaptations of archaea to hypersaline environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baker, B. A.; Gutierrez-Preciado, A.; Rodriguez del Rio, A.; McCarthy, C.; Lopez-Garcia, P.; Huerta-Cepas, J.; Susko, E.; Roger, A. J.; Eme, L.; Moreira, D.</p>
        <p class="info">Score: 25.3, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547478' target='https://doi.org/10.1101/2023.07.03.547478'> 10.1101/2023.07.03.547478</a></p>
        <p class="abstract">Several archaeal lineages thrive in high, saturating salt concentrations. These extremely halophilic archaea, including Halobacteria, Nanohaloarchaeota, Methanonatronarchaeia, and Haloplasmatales, must maintain osmotic equilibrium with their environment. For this, they use a &#39;salt-in&#39; strategy, which involves pumping molar concentrations of potassium into the cells, which, in turn, has led to extensive proteome-wide modifications to prevent protein aggregation. However, the evolutionary history underlying these adaptations remains poorly understood. In particular, the number of times that these dramatic proteome-sweeping changes occurred is unclear due to the conflicting phylogenetic positions found for several of these lineages. Here, we present a resolved phylogeny of extremely halophilic archaea obtained using improved taxon sampling and state-of-the-art phylogenetic approaches designed to cope with the strong compositional biases of their proteomes. We describe two new uncultured lineages, Afararchaeaceae and Asboarchaeaceae, which break the long branches at the base of Haloarchaea and Nanohaloarchaeota, respectively. Our extensive phylogenomic analyses show that at least four independent adaptations to extreme halophily occurred during archaeal evolution. Finally, gene-tree/species-tree reconciliation suggests that gene duplication and horizontal gene transfer played an important role in this process, for example, by spreading key genes (such as those encoding potassium transporters) across the various extremely halophilic lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546893">
        <p class="paperTitle">Standing genetic variation and de novo mutations underlie parallel evolution of island bird phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Estandia, A.; Sendell-Price, A. T.; Robertson, B. C.; Clegg, S. M.</p>
        <p class="info">Score: 19.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546893' target='https://doi.org/10.1101/2023.06.29.546893'> 10.1101/2023.06.29.546893</a></p>
        <p class="abstract">Parallel evolution occurs when the same trait evolves in closely related lineages in response to similar ecological contexts and provides some of the best examples of determinism in evolutionary biology. However, from a genetic standpoint, this process can be driven by either new mutations that appear independently in each diverging population or by selection on existing genetic variation common to both lineages. Small bodied birds, for example, tend to increase in size after they colonise a new island, following what is known as the  island rule. Such is the case of the Silvereye, a prolific natural coloniser of southwest Pacific islands. Island forms of this bird species increase in body size after they establish, with the pattern and pace of change consistent with directional natural selection and evident even in the most recent colonisations within the last 200 years. The system provides an exceptional opportunity to explore the genomic basis of repeated body size evolution. We sequenced 377 whole genomes from 31 different Silvereye populations, which revealed that both mechanisms are at play: in some lineages, new mutations are highly associated with body and bill size, but there are also highly associated polymorphisms present across all populations. Our research sheds light on the genomic basis of repeated body size evolution and emphasises that multiple molecular mechanisms can underlie similar evolutionary trajectories even within a single taxon.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546833">
        <p class="paperTitle">Reconstructing the evolutionary history of human, simian, and prosimian immunodeficiency viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghafari, M.; Simmonds, P.; Katzourakis, A.</p>
        <p class="info">Score: 19.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546833' target='https://doi.org/10.1101/2023.06.28.546833'> 10.1101/2023.06.28.546833</a></p>
        <p class="abstract">While paleovirological evidence supports the idea that the ancestors of modern simian (SIV) and prosimian (pSIV) immunodeficiency viruses have been evolving in nonhuman primates for millions of years, standard molecular clock methods calibrated using contemporary sequences significantly underestimate their times of origin. This discrepancy is attributed to the time-dependent nature of evolutionary rate estimates whereby the rate of virus evolution varies with respect to the time window of its measurement. While biogeographical calibrations may provide better estimates across intermediate timescales, there is currently no unifying framework that can allow inference of lentiviral ages across all relevant timescales. Here we showed that by correcting for such time-dependent rate effects using the recently developed Prisoner-of-War evolutionary model, we can successfully reconstruct the evolutionary history of human, simian, and prosimian immunodeficiency viruses across vastly different timescales and allowed estimates of the age of lineages that gave rise to HIV-1 and HIV-2. The model also predicted that the most recent common ancestor of SIV and pSIV lived around 21 million years ago, suggesting that the most likely explanation for the origins of primate lentiviruses is a terrestrial transfer of lentiviruses between African and Malagasy primates during the last episode of colonisation of Madagascar. Infections have entered via a land bridge in a group of mammals or through a nonprimate vector or transfer mediated by an aerial vector species. We also found that the most recent common ancestor of SIVs lived nearly a million years ago and that some SIV lineages codiverged with their hosts for several hundreds of thousands of years. The predicted long evolutionary timescales of SIVs and potential for close virus/host co-adaptation is consistent with their reduced or minimal pathogenicity in their native hosts, contrasting with the very recent evolutionary origins of highly pathogenic HIV strains in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.544912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.544912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.544912">
        <p class="paperTitle">Ancient Sheep Genomes reveal four Millennia of North European Short-Tailed Sheep in the Baltic Sea region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.544912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.544912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsson, M. N.; Morell Miranda, P.; Pan, L.; Basak Vural, K.; Kaptan, D.; Rodrigues Soares, A. E.; Kivikero, H.; Kantanen, J.; Somel, M.; Özer, F.; Johansson, A. M.; Stora, J.; Günther, T.</p>
        <p class="info">Score: 20.9, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.544912' target='https://doi.org/10.1101/2023.06.26.544912'> 10.1101/2023.06.26.544912</a></p>
        <p class="abstract">Sheep are among the earliest domesticated livestock species, with a wide variety of breeds present today. However, it remains unclear how far back this breed diversity goes, with formal documentation only dating back a few centuries. North European short-tailed breeds are often assumed to be among the oldest domestic sheep populations, even thought to represent relicts of the earliest sheep expansions during the Neolithic period reaching Scandinavia less than 6000 years ago. This study sequenced the genomes (up to 11.6X) of five sheep remains from the Baltic islands of Gotland and [A]land, dating from Late Neolithic ([~]4100 calBP) to historical times ([~]1600 CE). Our findings indicate that these ancient sheep largely possessed the genetic characteristics of modern North European short-tailed breeds, suggesting a substantial degree of long-term continuity of this breed type in the Baltic Sea region. Despite the wide temporal spread, population genetic analyses show high levels of affinity between the ancient genomes and they also exhibit higher genetic diversity when compared to modern breeds, implying a loss of diversity in recent centuries associated with breed formation. Finally, we see a potential signature of an even earlier, genetically different form of sheep in Scandinavia as these samples do not represent the first sheep in Northern Europe. Our results shed light on the development of breeds in Northern Europe specifically as well as the development of genetic diversity in sheep breeds, and their expansion from the domestication center in general.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546503">
        <p class="paperTitle">The genome sequence of the Montseny horsehair worm, Gordionus montsenyensis sp. nov., a key resource to investigate Ecdysozoa evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eleftheriadi, K.; Guiglielmoni, N.; Salces-Ortiz, J.; Vargas-Chavez, C.; Martinez-Redondo, G. I.; Gut, M.; Flot, J.-F.; Schmidt-Rhaesa, A.; Fernandez, R.</p>
        <p class="info">Score: 14.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546503' target='https://doi.org/10.1101/2023.06.26.546503'> 10.1101/2023.06.26.546503</a></p>
        <p class="abstract">Nematomorpha, also known as Gordiacea or Gordian worms, are a phylum of parasitic organisms that belong to the Ecdysozoa, a clade of invertebrate animals characterized by molting. They are one of the most scientifically neglected animal phyla, and many aspects of their biology and evolution are still unknown, partially due to the lack of genomic resources for this phylum. As part of the European Reference Genome Atlas pilot effort to generate reference genomes for European biodiversity, we present the taxonomic description and chromosome-level genome assembly of a newly described species of Nematomorpha (Gordionus montsenyensis Schmidt-Rhaesa &amp; Fernandez sp. nov.). The final assembly has a total length of 288 Mb in 398 scaffolds with an N50 of 52.6 Mb, 96% of which is scaffolded into 6 pseudochromosomes. The circular mitochondrial genome was also assembled into a 15-kilobases sequence. Gene annotation predicted 10,819 protein-coding genes in the nuclear genome. In this piece of work, we contribute a key genomic resource to not only explore the evolution of Ecdysozoa, but also to further our understanding on the genomic basis of parasitic lifestyles. In addition, we describe a species new to science from this enigmatic animal phyla.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547057">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547057" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547057">
        <p class="paperTitle">Epistasis Arises from Shifting the Rate-Limiting Step during Enzyme Evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547057" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547057" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frohlich, C.; Bunzel, H. A.; Buda, K.; Mulholland, A. J.; van der Kamp, M. W.; Johnsen, P. J.; Leiros, H.-K. S.; Tokuriki, N.</p>
        <p class="info">Score: 14.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547057' target='https://doi.org/10.1101/2023.06.29.547057'> 10.1101/2023.06.29.547057</a></p>
        <p class="abstract">The molecular mechanisms by which epistasis boosts enzyme activity remain elusive, undermining our ability to predict the evolution of pathogens and engineer novel biocatalysts. Here, we reveal how directed evolution of a {beta}-lactamase yielded highly epistatic activity enhancements. Evolution selected four mutations that increase antibiotic resistance 40-fold, despite their marginal individual effects ([&amp;le;] 2-fold). Synergistic improvements coincided with the introduction of super-stochiometric burst kinetics, indicating that epistasis is rooted in the enzymes conformational dynamics. Kinetic, structural, and dynamical analyses reveal that epistasis was driven by distinct effects of each mutation on the catalytic cycle. The first mutation acquired during evolution increases protein flexibility and accelerates substrate binding, which is rate-limiting in the wild-type enzyme. The ensuing mutations predominantly boosted the chemical steps by fine-tuning substrate interactions. Our work identifies an overlooked cause for epistasis: changing the rate-limiting step can result in substantial positive synergy boosting enzyme activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546961">
        <p class="paperTitle">High germline mutation rates but not extreme population size outbreaks influence genetic diversity in crown-of-thorns sea stars</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Popovic, I.; Bergeron, L.; Bozec, Y.-M.; Waldvogel, A.-M.; Howitt, S. M.; Damjanovic, K.; Patel, F.; Cabrera, M. G.; Wörheide, G.; Uthicke, S.; Riginos, C.</p>
        <p class="info">Score: 12.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546961' target='https://doi.org/10.1101/2023.06.28.546961'> 10.1101/2023.06.28.546961</a></p>
        <p class="abstract">Lewontin&#39;s paradox, the observation that levels of genetic diversity ({pi}) among animals do not scale linearly with variation in census population sizes (Nc), is an evolutionary conundrum, where the most extreme mismatches between {pi} and Nc are found for highly abundant marine invertebrates. Yet, whether new mutations influence {pi} relative to extrinsic processes remains unknown for most taxa. Here, we provide the first direct germline mutation rate () estimate for a marine invertebrate, using high-coverage (60x) whole-genome sequencing of wild-caught Acanthaster cf. solaris crown-of-thorns sea stars (Echinodermata). We also provide empirical estimates of adult Nc in Australia&#39;s Great Barrier Reef to jointly examine the determinants of {pi}. Based on direct observations of 63 de novo mutations across 14 parent-offspring trios, the A. cf. solaris mean  was 9.13 x 10-09 mutations per-site per-generation (95% CI: 6.51 x 10-09 to 1.18 x 10-08). This value exceeds estimates for other invertebrates, showing greater concordance with reported vertebrate germline mutation rates. Lower-than-expected Ne (~70,000-180,000) and low Ne/Nc values (0.0047-0.048) indicated significant genetic drift and weak influences of contemporary population outbreaks on long-term {pi}. Our findings of elevated  and low Ne in A. cf. solaris may help explain high mutational loads and extreme polymorphism levels observed in some marine invertebrate taxa and are consistent with  evolving in response to Ne (drift-barrier hypothesis). This study advances our understanding of the processes controlling levels of natural genetic variation and provides new data valuable for further testing hypotheses about mutation rate evolution across animal phyla.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545888" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545888">
        <p class="paperTitle">Enrichment of hard sweeps on the X chromosome compared to autosomes in six  Drosophila  species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, M.; Kim, B. Y.; Garud, N. R.</p>
        <p class="info">Score: 12.3, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545888' target='https://doi.org/10.1101/2023.06.21.545888'> 10.1101/2023.06.21.545888</a></p>
        <p class="abstract">The X chromosome, being hemizygous in males, is exposed one third of the time increasing the visibility of new mutations to natural selection, potentially leading to different evolutionary dynamics than autosomes. Recently, we found an enrichment of hard selective sweeps over soft selective sweeps on the X chromosome relative to the autosomes in a North American population of Drosophila melanogaster. To understand whether this enrichment is a universal feature of evolution on the X chromosome, we analyze diversity patterns across six commonly studied Drosophila species. We find an increased proportion of regions with steep reductions in diversity on the X chromosome compared to autosomes, accompanied by elevated homozygosity and an increased proportion of singletons. To assess if these signatures are consistent with positive selection, we simulate a wide variety of evolutionary scenarios spanning variations in demography, mutation rate, recombination rate, background selection, hard sweeps, and soft sweeps, and find that the diversity patterns observed on the X are most consistent with hard sweeps. Our findings highlight the importance of sex chromosomes in driving evolutionary processes and suggest that hard sweeps have played a significant role in shaping diversity patterns on the X chromosome across multiple Drosophila species.

SignificanceUnderstanding the unique evolutionary dynamics of the X chromosome is a longstanding question in evolutionary biology given its central role in speciation, fertility, and sexual dimorphism. Recently, we found that hard sweeps, which are indicative of gradual adaptation, are more common on the X chromosome than on the autosomes in a North American population of Drosophila melanogaster. This is because new mutations on the X chromosome are more visible to natural selection due to male hemizygosity. Here, we analyze multiple whole genome sequences from six Drosophila species and find that this pattern is common across all species. Our study demonstrates the power of comparative population genetics in unraveling the forces shaping genetic variation within and between species.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
