<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Strong environmental memory revealed by experimental evolution in static and fluctuating environments
Authors: Abreu, C. I.; Mathur, S.; Petrov, D. A.
Score: 42.1, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739
Evolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Strong environmental memory revealed by experimental evolution in static and fluctuating environments
Authors: Abreu, C. I.; Mathur, S.; Petrov, D. A.
Score: 42.1, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739
Evolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Strong environmental memory revealed by experimental evolution in static and fluctuating environments
Authors: Abreu, C. I.; Mathur, S.; Petrov, D. A.
Score: 42.1, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739
Evolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Strong environmental memory revealed by experimental evolution in static and fluctuating environments\nAuthors: Abreu, C. I.; Mathur, S.; Petrov, D. A.\nScore: 42.1, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739\nEvolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters.",
  "keywords": [
    
  ],
  "articleBody": " Strong environmental memory revealed by experimental evolution in static and fluctuating environments\nAuthors: Abreu, C. I.; Mathur, S.; Petrov, D. A.\nScore: 42.1, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739\nEvolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters. However, transitions between conditions may pose entirely new challenges, which could cause deviations from this time-average. To test this, we tracked hundreds of thousands of barcoded yeast lineages evolving in static and fluctuating conditions and subsequently isolated 900 mutants for pooled fitness assays in 15 environments. We find that fitness in fluctuating environments indeed often deviates from the expectation based on static components, leading to fitness non-additivity. Moreover, closer examination reveals that fitness in one component of a fluctuating environment is often strongly influenced by the previous component. We show that this environmental memory is especially common for mutants with high variance in fitness across tested environments, even if the components of the focal fluctuating environment are excluded from this variance. We employ a simple mathematical model and whole-genome sequencing to propose mechanisms underlying this effect, including lag time evolution and sensing mutations. Our results demonstrate that environmental fluctuations have large impacts on fitness and suggest that variance in static environments can explain these impacts.\nThe MUC19 gene in Denisovans, Neanderthals, and Modern Humans: An Evolutionary History of Recurrent Introgression and Natural Selection\nAuthors: Villanea, F. A.; Peede, D.; Kaufman, E. J.; Anorve-Garibay, V.; Dillon, K. E. W.; Zeloni, R.; Marnetto, d.; Moorjani, P.; Jay, F.; Valdmanis, P. N.; Avila-Arcos, M. C.; Huerta-Sanchez, E.\nScore: 38.7, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559202\nAll humans carry a small fraction of archaic ancestry across the genome, the legacy of gene flow from Neanderthals, Denisovans, and other hominids into the ancestors of modern humans. While the effects of Neanderthal ancestry on human fitness and health have been explored more thoroughly, there are fewer examples of adaptive introgression of Denisovan variants. Here, we study the gene MUC19, for which some modern humans carry a Denisovan-like haplotype. MUC19 is a mucin, a glycoprotein that forms gels with various biological functions, from lubrication to immunity. We find the diagnostic variants for the Denisovan-like MUC19 haplotype at high frequencies in admixed Latin American individuals among global population, and at highest frequency in 23 ancient Indigenous American individuals, all predating population admixture with Europeans and Africans. We find that some Neanderthals--Vindija and Chagyrskaya--carry the Denisovan-like MUC19 haplotype, and that it was likely introgressed into human populations through Neanderthal introgression rather than Denisovan introgression. Finally, we find that the Denisovan-like MUC19 haplotype carries a higher copy number of a 30 base-pair variable number tandem repeat relative to the Human-like haplotype, and that copy numbers of this repeat are exceedingly high in American populations. Our results suggest that the Denisovan-like MUC19 haplotype served as the raw genetic material for positive selection as American populations adapted to novel environments during their movement from Beringia into North and then South America.\nWorking together to control mutation: how collective peroxide detoxification determines microbial mutation rate plasticity\nAuthors: Green, R.; Wang, H.; Botchey, C.; Zhang, N.; Wadsworth, C.; McBain, A.; Paszek, P.; Krasovec, R.; Knight, C. G.\nScore: 22.2, Published: 2023-09-27 DOI: 10.1101/2023.09.27.557722\nMutagenesis is responsive to many environmental factors. Evolution therefore depends on the environment not only for selection but also in determining the variation available in a population. One such environmental dependency is the inverse relationship between mutation rates and population density in many microbial species. Here we determine the mechanism responsible for this mutation rate plasticity. Using dynamical computational modelling and in vivo mutation rate estimation we show that the negative relationship between mutation rate and population density arises from the collective ability of microbial populations to control concentrations of hydrogen peroxide. We demonstrate a loss of this density-associated mutation rate plasticity when Escherichia coli populations are deficient in the degradation of hydrogen peroxide. We further show that the reduction in mutation rate in denser populations is restored in peroxide degradation-deficient cells by the presence of wild-type cells in a mixed population. Together, these model-guided experiments provide a mechanistic explanation for density-associated mutation rate plasticity, applicable across all domains of life, and frames mutation rate as a dynamic trait shaped by microbial community composition.\nDifferential evolution of cooperative traits in aggregative multicellular bacterium Myxococcus xanthus driven by varied population bottleneck sizes\nAuthors: Kalathera, J.; Patel, V.; Pande, S.\nScore: 17.8, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558552\nRepeated population bottlenecks influence the evolution and maintenance of cooperation1,2. However, it remains unclear whether bottlenecks select all cooperative traits expressed by an organism or only a subset of them. Myxococcus xanthus, a social bacterium, displays multiple cooperative traits, including growth, predation, sporulation in multicellular fruiting bodies, and germination 3-6. Using laboratory evolution experiment, we investigated the effect of repeated stringent versus relaxed population bottlenecks on the evolution of these four cooperative traits when they were all under selection. We found that only fruiting body formation and growth were positively selected under the stringent regimen, while the other two traits were negatively selected. The pattern was reversed in the relaxed regimen. Additionally, the relaxed regimen led to a significant increase in fitness when competed against ancestors across the entire lifecycle, whereas the stringent treatment did not change competitive fitness. Genomic analysis revealed that mutations in {sigma}54 interacting protein and DNA-binding response regulator protein are linked with the changes observed in stringent and relaxed regimens respectively. Further, similar trade-offs are also seen among natural populations of M. xanthus. Overall, we demonstrate that different bottleneck sizes drive the evolution of lifecycles in distinct manners, driven by trade-offs between cooperative life history traits.\nA direct experimental test of Ohno's hypothesis\nAuthors: Mihajlovic, L.; Iyengar, B. R.; Baier, F.; Barbier, I.; Iwaszkiewicz, J.; Zoete, V.; Wagner, A.; Schaerli, Y.\nScore: 16.5, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559237\nGene duplication drives evolution by providing raw material for proteins with novel functions. The oldest and historically most influential hypothesis about the evolutionary fate and potential of duplicated genes has been proposed by Susumu Ohno in 1970. This hypothesis essentially posits that gene duplication can help genes tolerate new mutations and thus facilitates the evolution of new phenotypes Competing hypotheses argue that deleterious mutations will usually inactivate gene duplicates too rapidly for Ohnos hypothesis to work. Here, we provide a first direct experimental test of Ohnos hypothesis. Specifically, we evolved one or two genes encoding a fluorescent protein in Escherichia coli through multiple rounds of mutagenesis and selection. We then analyzed the genotypic and phenotypic evolutionary dynamics of the evolving populations through high-throughput DNA sequencing, biochemical assays, and engineering of selected variants. In support of Ohnos hypothesis, populations carrying two gene copies displayed higher mutational robustness than those carrying a single gene copy. As a consequence, the double-copy populations experienced relaxed purifying selection, evolved higher phenotypic and genetic diversity, carried more mutations and accumulated combinations of key beneficial mutations earlier. However, their phenotypic evolution was not accelerated, possibly because one gene copy rapidly became inactivated by deleterious mutations. Our work provides an experimental platform to test models of evolution by gene duplication, and it supports alternatives to Ohnos hypothesis.\nGenomic signatures of adaptation in native lizards exposed to human-introduced fire ants\nAuthors: Assis, B. A.; Sullivan, A. P.; Marciniak, S.; Bergey, C. M.; Garcia, V.; Szpiech, Z. A.; Langkilde, T.; Perry, G. H.\nScore: 12.8, Published: 2023-09-26 DOI: 10.1101/2023.09.24.559217\nUnderstanding the process of genetic adaptation in response to human-mediated ecological change will help elucidate the eco-evolutionary impacts of human activity. Red fire ants (Solenopsis invicta) spread across Southeastern USA since their accidental introduction via Port Mobile, Alabama in the 1930s, serving today as both novel venomous predator and novel toxic prey to native eastern fence lizards (Sceloporus undulatus). To identify potential signatures of genetic adaptation in lizards to invasive fire ants, we generated whole genome sequencing data from 420 native fence lizards sampled across three populations, two of which had not been invaded by fire ants (in Tennessee and Arkansas) and one which had been invaded for [~]70 years (Alabama). We detected signatures of positive selection exclusive to the exposed Alabama population for genetic variants overlapping genes related to the membrane attack complex of the complement immune system, growth factor pathways, and morphological development. Prior work identified a relationship between increased lizard survival of fire ant attack and longer hind limbs, which lizards use to remove ants from their bodies. Furthermore, we conducted a genome-wide association study with 381 Alabama lizards to identify 24 hind limb length-associated genetic loci. For two loci, positive-effect alleles occur in high frequency and overlap genomic regions that are highly differentiated from the populations naive to fire ants. Collectively, these findings represent plausible genetic adaptations in response to fire ant invasion, whereby morphological differentiation may increase survival against swarming ants and altered immune responses may allow the exploitation of a novel, toxic food resource. Significance statementHuman activity can force interactions between species from distinct ecological backgrounds. These interactions can consequently impose novel selective pressures on endemic populations via predation or disruption of ecological niches through community-wide effects. While some endemic taxa have been able to adapt biologically to these disruptions, we do not have a full understanding of the underlying genetic processes that may allow it. Here we identify genomic signatures of recent adaptation nearby genes involved in morphological and immunological processes in native fence lizards that are consistent with pressures imposed by the venomous, predatory fire ants introduced by humans. These signatures are largely absent from lizard populations that are naive to fire ants.\nPhased assembly of neo-sex chromosomes reveals extensive Y degeneration and rapid genome evolution in Rumex hastatulus\nAuthors: Sacchi, B. M.; Humphries, Z.; Kruzlicova, J.; Bodlakova, M.; Pyne, C.; Choudhury, B.; Gong, Y.; Bacovsky, V.; Hobza, R.; Barrett, S. C. H.; Wright, S. I.\nScore: 12.7, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559509\nY chromosomes are thought to undergo progressive degeneration due to stepwise loss of recombination and subsequent reduction in selection efficiency. However, the timescales over which degeneration occurs and the evolutionary forces driving degeneration remain unclear. In order to characterize the evolution of sex chromosomes on multiple timescales, we generated a high-quality phased genome assembly of the massive older (7-9 MYA) and neo (\u003c200,000 years) sex chromosomes in the XYY cytotype of the plant Rumex hastatulus, along with a hermaphroditic outgroup Rumex salicifolius. Our assemblies confirmed the neo-sex chromosomes were formed by two key events: an X-autosome fusion and a reciprocal translocation between the homologous autosome and the Y chromosome. The enormous sex-linked regions of the X (296 MB) and the two Y chromosomes (503 MB) both arose in a large repeat-rich genome with low recombination, however the complete loss of recombination on the Y still led to over 30% gene loss and massive rearrangements over a short timescale. In the older sex-linked region, there has been a major increase in the abundance of transposable elements, including into and near genes. In the neo sex-linked regions, we observe evidence of extensive chromosome rearrangements before gene degeneration and loss. Overall, we observe extensive degeneration during the first 10 million years of Y chromosome evolution, but not immediate genomic degeneration on very short timescales. Our results highlight that even when sex chromosomes emerge from repetitive regions of already-low recombination, the complete loss of recombination on the Y chromosome still leads to a substantial increase in repetitive element content and gene degeneration.\nAssessing the role of mitonuclear interactions on mitochondrial function and organismal fitness in natural Drosophila populations\nAuthors: Bettinazzi, S.; Liang, J.; Rodriguez, E.; Bonneau, M.; Holt, R.; Whitehead, B.; Dowling, D.; Lane, N.; Camus, F.\nScore: 9.6, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559268\nMitochondrial metabolism is regulated by a series of enzyme complexes, whose function depends on effective interactions between proteins and RNA encoded by the mitochondrial and nuclear genomes. These epistatic interactions are in turn highly sensitive to the environment. Many studies have found that mitochondrial haplotype frequencies of various taxa associate with latitude or altitude, leading to the hypothesis that mitochondrial genomes may respond to thermal selection and contribute to local adaptation. We used a Drosophila melanogaster panel comprising native (coadapted) populations from the extremes of the Australian east-coast cline, and generated mitonuclear cybrid populations. Our results indicate a strong phenotypic impact of mitonuclear interactions in cybrid lines, involving an apparent trade-off between aerobic capacity and key fitness aspects such as reproduction, growth, and survival. Overall, our study shows that naturally-occurring mitonuclear disruptions can have a meaningful impact on phenotypes, potentially influencing future ecological adaptation and population persistence.\nViperin immunity evolved across the tree of life through serial innovations on a conserved scaffold\nAuthors: Shomar, H.; Georjon, H.; Feng, Y.; Olympio, B.; Tesson, F.; Cury, J.; Wu, F.; Bernheim, A.\nScore: 56.6, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557418\nEvolutionary arms races between cells and viruses drive the rapid diversification of antiviral genes in diverse life forms. Recent discoveries have revealed the existence of shared immune genes between prokaryotes and eukaryotes, showing molecular and mechanistic similarities in their response to viruses. However, the underlying evolutionary dynamics that explain the conservation and adaptation of these antiviral genes remain mostly unexplored. Here, we show that viperins constitute a highly conserved family of immune genes across diverse prokaryotes and eukaryotes, and uncover mechanisms by which they diversified in eukaryotes. Our findings indicate that viperins are enriched in Asgard archaea and widely distributed in all major eukaryotic clades, suggesting their presence in the Last Eukaryotic Common Ancestor (LECA). We show that viperins maintain their immune function by producing antiviral nucleotide analogs. We demonstrate that eukaryotic viperins diversified through serial innovations on the viperin gene, such as the emergence and selection of substrate specificity towards pyrimidine nucleotides, and through partnerships with genes maintained through genetic linkage, notably with nucleotide kinases. These findings unveil biochemical and genomic transitions underlying the adaptation of immune genes shared by prokaryotes and eukaryotes. Our study paves the way for the understanding of the conservation of immunity across domains of life.\nPatterns of genomic variation reveal a single evolutionary origin of the wild allotetraploid Mimulus sookensis\nAuthors: Whitener, M. R.; Mangelson, H.; Sweigart, A. L.\nScore: 9.4, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557766\nPolyploidy occurs across the tree of life and is especially common in plants. Because newly formed cytotypes are often incompatible with their progenitors, polyploidy is also said to trigger \"instantaneous\" speciation. If a polyploid can self-fertilize or reproduce asexually, it is even possible for one individual to produce an entirely new lineage. How often this extreme scenario occurs is unclear, with most studies of wild polyploids reporting multiple origins. Here, we explore the evolutionary history of the wild allotetraploid Mimulus sookensis, which was formed through hybridization between self-compatible, diploid species in the Mimulus guttatus complex. We generate a chromosome-scale reference assembly for M. sookensis and define its distinct subgenomes. Despite previous reports suggesting multiple origins of this highly selfing polyploid, we discover patterns of population genomic variation that provide unambiguous support for a single origin, which we estimate occurred [~]71,000 years ago. One M. sookensis subgenome is clearly derived from the selfer M. nasutus, which, based on organellar variation, also appears to be the maternal progenitor. The ancestor of the other subgenome is less certain, but it shares variation with both M. decorus and M. guttatus, two outcrossing diploids that overlap broadly with M. sookensis. Whatever its precise ancestry, this study establishes M. sookensis as an example of instantaneous speciation, likely facilitated by the polyploids predisposition to self-fertilize. With a reference genome for M. sookensis now available and its origin clarified, this wild tetraploid is poised to become a model for understanding the genetic and evolutionary mechanisms of polyploid persistence.\n",
  "wordCount" : "2648",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557739">
        <p class="paperTitle">Strong environmental memory revealed by experimental evolution in static and fluctuating environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abreu, C. I.; Mathur, S.; Petrov, D. A.</p>
        <p class="info">Score: 42.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557739' target='https://doi.org/10.1101/2023.09.14.557739'> 10.1101/2023.09.14.557739</a></p>
        <p class="abstract">Evolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters. However, transitions between conditions may pose entirely new challenges, which could cause deviations from this time-average. To test this, we tracked hundreds of thousands of barcoded yeast lineages evolving in static and fluctuating conditions and subsequently isolated 900 mutants for pooled fitness assays in 15 environments. We find that fitness in fluctuating environments indeed often deviates from the expectation based on static components, leading to fitness non-additivity. Moreover, closer examination reveals that fitness in one component of a fluctuating environment is often strongly influenced by the previous component. We show that this environmental memory is especially common for mutants with high variance in fitness across tested environments, even if the components of the focal fluctuating environment are excluded from this variance. We employ a simple mathematical model and whole-genome sequencing to propose mechanisms underlying this effect, including lag time evolution and sensing mutations. Our results demonstrate that environmental fluctuations have large impacts on fitness and suggest that variance in static environments can explain these impacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559202">
        <p class="paperTitle">The MUC19 gene in Denisovans, Neanderthals, and Modern Humans: An Evolutionary History of Recurrent Introgression and Natural Selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Villanea, F. A.; Peede, D.; Kaufman, E. J.; Anorve-Garibay, V.; Dillon, K. E. W.; Zeloni, R.; Marnetto, d.; Moorjani, P.; Jay, F.; Valdmanis, P. N.; Avila-Arcos, M. C.; Huerta-Sanchez, E.</p>
        <p class="info">Score: 38.7, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559202' target='https://doi.org/10.1101/2023.09.25.559202'> 10.1101/2023.09.25.559202</a></p>
        <p class="abstract">All humans carry a small fraction of archaic ancestry across the genome, the legacy of gene flow from Neanderthals, Denisovans, and other hominids into the ancestors of modern humans. While the effects of Neanderthal ancestry on human fitness and health have been explored more thoroughly, there are fewer examples of adaptive introgression of Denisovan variants. Here, we study the gene MUC19, for which some modern humans carry a Denisovan-like haplotype. MUC19 is a mucin, a glycoprotein that forms gels with various biological functions, from lubrication to immunity. We find the diagnostic variants for the Denisovan-like MUC19 haplotype at high frequencies in admixed Latin American individuals among global population, and at highest frequency in 23 ancient Indigenous American individuals, all predating population admixture with Europeans and Africans. We find that some Neanderthals--Vindija and Chagyrskaya--carry the Denisovan-like MUC19 haplotype, and that it was likely introgressed into human populations through Neanderthal introgression rather than Denisovan introgression. Finally, we find that the Denisovan-like MUC19 haplotype carries a higher copy number of a 30 base-pair variable number tandem repeat relative to the Human-like haplotype, and that copy numbers of this repeat are exceedingly high in American populations. Our results suggest that the Denisovan-like MUC19 haplotype served as the raw genetic material for positive selection as American populations adapted to novel environments during their movement from Beringia into North and then South America.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.557722">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.557722" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.557722">
        <p class="paperTitle">Working together to control mutation: how collective peroxide detoxification determines microbial mutation rate plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.557722" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.557722" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Green, R.; Wang, H.; Botchey, C.; Zhang, N.; Wadsworth, C.; McBain, A.; Paszek, P.; Krasovec, R.; Knight, C. G.</p>
        <p class="info">Score: 22.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.557722' target='https://doi.org/10.1101/2023.09.27.557722'> 10.1101/2023.09.27.557722</a></p>
        <p class="abstract">Mutagenesis is responsive to many environmental factors. Evolution therefore depends on the environment not only for selection but also in determining the variation available in a population. One such environmental dependency is the inverse relationship between mutation rates and population density in many microbial species. Here we determine the mechanism responsible for this mutation rate plasticity. Using dynamical computational modelling and in vivo mutation rate estimation we show that the negative relationship between mutation rate and population density arises from the collective ability of microbial populations to control concentrations of hydrogen peroxide. We demonstrate a loss of this density-associated mutation rate plasticity when Escherichia coli populations are deficient in the degradation of hydrogen peroxide. We further show that the reduction in mutation rate in denser populations is restored in peroxide degradation-deficient cells by the presence of wild-type cells in a mixed population. Together, these model-guided experiments provide a mechanistic explanation for density-associated mutation rate plasticity, applicable across all domains of life, and frames mutation rate as a dynamic trait shaped by microbial community composition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558552">
        <p class="paperTitle">Differential evolution of cooperative traits in aggregative multicellular bacterium Myxococcus xanthus driven by varied population bottleneck sizes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalathera, J.; Patel, V.; Pande, S.</p>
        <p class="info">Score: 17.8, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558552' target='https://doi.org/10.1101/2023.09.20.558552'> 10.1101/2023.09.20.558552</a></p>
        <p class="abstract">Repeated population bottlenecks influence the evolution and maintenance of cooperation1,2. However, it remains unclear whether bottlenecks select all cooperative traits expressed by an organism or only a subset of them. Myxococcus xanthus, a social bacterium, displays multiple cooperative traits, including growth, predation, sporulation in multicellular fruiting bodies, and germination 3-6. Using laboratory evolution experiment, we investigated the effect of repeated stringent versus relaxed population bottlenecks on the evolution of these four cooperative traits when they were all under selection. We found that only fruiting body formation and growth were positively selected under the stringent regimen, while the other two traits were negatively selected. The pattern was reversed in the relaxed regimen. Additionally, the relaxed regimen led to a significant increase in fitness when competed against ancestors across the entire lifecycle, whereas the stringent treatment did not change competitive fitness. Genomic analysis revealed that mutations in {sigma}54 interacting protein and DNA-binding response regulator protein are linked with the changes observed in stringent and relaxed regimens respectively. Further, similar trade-offs are also seen among natural populations of M. xanthus. Overall, we demonstrate that different bottleneck sizes drive the evolution of lifecycles in distinct manners, driven by trade-offs between cooperative life history traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559237">
        <p class="paperTitle">A direct experimental test of Ohno&#39;s hypothesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mihajlovic, L.; Iyengar, B. R.; Baier, F.; Barbier, I.; Iwaszkiewicz, J.; Zoete, V.; Wagner, A.; Schaerli, Y.</p>
        <p class="info">Score: 16.5, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559237' target='https://doi.org/10.1101/2023.09.25.559237'> 10.1101/2023.09.25.559237</a></p>
        <p class="abstract">Gene duplication drives evolution by providing raw material for proteins with novel functions. The oldest and historically most influential hypothesis about the evolutionary fate and potential of duplicated genes has been proposed by Susumu Ohno in 1970. This hypothesis essentially posits that gene duplication can help genes tolerate new mutations and thus facilitates the evolution of new phenotypes Competing hypotheses argue that deleterious mutations will usually inactivate gene duplicates too rapidly for Ohnos hypothesis to work. Here, we provide a first direct experimental test of Ohnos hypothesis. Specifically, we evolved one or two genes encoding a fluorescent protein in Escherichia coli through multiple rounds of mutagenesis and selection. We then analyzed the genotypic and phenotypic evolutionary dynamics of the evolving populations through high-throughput DNA sequencing, biochemical assays, and engineering of selected variants. In support of Ohnos hypothesis, populations carrying two gene copies displayed higher mutational robustness than those carrying a single gene copy. As a consequence, the double-copy populations experienced relaxed purifying selection, evolved higher phenotypic and genetic diversity, carried more mutations and accumulated combinations of key beneficial mutations earlier. However, their phenotypic evolution was not accelerated, possibly because one gene copy rapidly became inactivated by deleterious mutations. Our work provides an experimental platform to test models of evolution by gene duplication, and it supports alternatives to Ohnos hypothesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.559217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.559217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.559217">
        <p class="paperTitle">Genomic signatures of adaptation in native lizards exposed to human-introduced fire ants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.559217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.559217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Assis, B. A.; Sullivan, A. P.; Marciniak, S.; Bergey, C. M.; Garcia, V.; Szpiech, Z. A.; Langkilde, T.; Perry, G. H.</p>
        <p class="info">Score: 12.8, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.559217' target='https://doi.org/10.1101/2023.09.24.559217'> 10.1101/2023.09.24.559217</a></p>
        <p class="abstract">Understanding the process of genetic adaptation in response to human-mediated ecological change will help elucidate the eco-evolutionary impacts of human activity. Red fire ants (Solenopsis invicta) spread across Southeastern USA since their accidental introduction via Port Mobile, Alabama in the 1930s, serving today as both novel venomous predator and novel toxic prey to native eastern fence lizards (Sceloporus undulatus). To identify potential signatures of genetic adaptation in lizards to invasive fire ants, we generated whole genome sequencing data from 420 native fence lizards sampled across three populations, two of which had not been invaded by fire ants (in Tennessee and Arkansas) and one which had been invaded for [~]70 years (Alabama). We detected signatures of positive selection exclusive to the exposed Alabama population for genetic variants overlapping genes related to the membrane attack complex of the complement immune system, growth factor pathways, and morphological development. Prior work identified a relationship between increased lizard survival of fire ant attack and longer hind limbs, which lizards use to remove ants from their bodies. Furthermore, we conducted a genome-wide association study with 381 Alabama lizards to identify 24 hind limb length-associated genetic loci. For two loci, positive-effect alleles occur in high frequency and overlap genomic regions that are highly differentiated from the populations naive to fire ants. Collectively, these findings represent plausible genetic adaptations in response to fire ant invasion, whereby morphological differentiation may increase survival against swarming ants and altered immune responses may allow the exploitation of a novel, toxic food resource.

Significance statementHuman activity can force interactions between species from distinct ecological backgrounds. These interactions can consequently impose novel selective pressures on endemic populations via predation or disruption of ecological niches through community-wide effects. While some endemic taxa have been able to adapt biologically to these disruptions, we do not have a full understanding of the underlying genetic processes that may allow it. Here we identify genomic signatures of recent adaptation nearby genes involved in morphological and immunological processes in native fence lizards that are consistent with pressures imposed by the venomous, predatory fire ants introduced by humans. These signatures are largely absent from lizard populations that are naive to fire ants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559509">
        <p class="paperTitle">Phased assembly of neo-sex chromosomes reveals extensive Y degeneration and rapid genome evolution in Rumex hastatulus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sacchi, B. M.; Humphries, Z.; Kruzlicova, J.; Bodlakova, M.; Pyne, C.; Choudhury, B.; Gong, Y.; Bacovsky, V.; Hobza, R.; Barrett, S. C. H.; Wright, S. I.</p>
        <p class="info">Score: 12.7, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559509' target='https://doi.org/10.1101/2023.09.26.559509'> 10.1101/2023.09.26.559509</a></p>
        <p class="abstract">Y chromosomes are thought to undergo progressive degeneration due to stepwise loss of recombination and subsequent reduction in selection efficiency. However, the timescales over which degeneration occurs and the evolutionary forces driving degeneration remain unclear. In order to characterize the evolution of sex chromosomes on multiple timescales, we generated a high-quality phased genome assembly of the massive older (7-9 MYA) and neo (&lt;200,000 years) sex chromosomes in the XYY cytotype of the plant Rumex hastatulus, along with a hermaphroditic outgroup Rumex salicifolius. Our assemblies confirmed the neo-sex chromosomes were formed by two key events: an X-autosome fusion and a reciprocal translocation between the homologous autosome and the Y chromosome. The enormous sex-linked regions of the X (296 MB) and the two Y chromosomes (503 MB) both arose in a large repeat-rich genome with low recombination, however the complete loss of recombination on the Y still led to over 30% gene loss and massive rearrangements over a short timescale. In the older sex-linked region, there has been a major increase in the abundance of transposable elements, including into and near genes. In the neo sex-linked regions, we observe evidence of extensive chromosome rearrangements before gene degeneration and loss. Overall, we observe extensive degeneration during the first 10 million years of Y chromosome evolution, but not immediate genomic degeneration on very short timescales. Our results highlight that even when sex chromosomes emerge from repetitive regions of already-low recombination, the complete loss of recombination on the Y chromosome still leads to a substantial increase in repetitive element content and gene degeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559268">
        <p class="paperTitle">Assessing the role of mitonuclear interactions on mitochondrial function and organismal fitness in natural Drosophila populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bettinazzi, S.; Liang, J.; Rodriguez, E.; Bonneau, M.; Holt, R.; Whitehead, B.; Dowling, D.; Lane, N.; Camus, F.</p>
        <p class="info">Score: 9.6, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559268' target='https://doi.org/10.1101/2023.09.25.559268'> 10.1101/2023.09.25.559268</a></p>
        <p class="abstract">Mitochondrial metabolism is regulated by a series of enzyme complexes, whose function depends on effective interactions between proteins and RNA encoded by the mitochondrial and nuclear genomes. These epistatic interactions are in turn highly sensitive to the environment. Many studies have found that mitochondrial haplotype frequencies of various taxa associate with latitude or altitude, leading to the hypothesis that mitochondrial genomes may respond to thermal selection and contribute to local adaptation. We used a Drosophila melanogaster panel comprising native (coadapted) populations from the extremes of the Australian east-coast cline, and generated mitonuclear cybrid populations. Our results indicate a strong phenotypic impact of mitonuclear interactions in cybrid lines, involving an apparent trade-off between aerobic capacity and key fitness aspects such as reproduction, growth, and survival. Overall, our study shows that naturally-occurring mitonuclear disruptions can have a meaningful impact on phenotypes, potentially influencing future ecological adaptation and population persistence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557418">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557418" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557418">
        <p class="paperTitle">Viperin immunity evolved across the tree of life through serial innovations on a conserved scaffold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557418" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557418" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shomar, H.; Georjon, H.; Feng, Y.; Olympio, B.; Tesson, F.; Cury, J.; Wu, F.; Bernheim, A.</p>
        <p class="info">Score: 56.6, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557418' target='https://doi.org/10.1101/2023.09.13.557418'> 10.1101/2023.09.13.557418</a></p>
        <p class="abstract">Evolutionary arms races between cells and viruses drive the rapid diversification of antiviral genes in diverse life forms. Recent discoveries have revealed the existence of shared immune genes between prokaryotes and eukaryotes, showing molecular and mechanistic similarities in their response to viruses. However, the underlying evolutionary dynamics that explain the conservation and adaptation of these antiviral genes remain mostly unexplored. Here, we show that viperins constitute a highly conserved family of immune genes across diverse prokaryotes and eukaryotes, and uncover mechanisms by which they diversified in eukaryotes. Our findings indicate that viperins are enriched in Asgard archaea and widely distributed in all major eukaryotic clades, suggesting their presence in the Last Eukaryotic Common Ancestor (LECA). We show that viperins maintain their immune function by producing antiviral nucleotide analogs. We demonstrate that eukaryotic viperins diversified through serial innovations on the viperin gene, such as the emergence and selection of substrate specificity towards pyrimidine nucleotides, and through partnerships with genes maintained through genetic linkage, notably with nucleotide kinases. These findings unveil biochemical and genomic transitions underlying the adaptation of immune genes shared by prokaryotes and eukaryotes. Our study paves the way for the understanding of the conservation of immunity across domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557766">
        <p class="paperTitle">Patterns of genomic variation reveal a single evolutionary origin of the wild allotetraploid Mimulus sookensis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitener, M. R.; Mangelson, H.; Sweigart, A. L.</p>
        <p class="info">Score: 9.4, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557766' target='https://doi.org/10.1101/2023.09.14.557766'> 10.1101/2023.09.14.557766</a></p>
        <p class="abstract">Polyploidy occurs across the tree of life and is especially common in plants. Because newly formed cytotypes are often incompatible with their progenitors, polyploidy is also said to trigger &#34;instantaneous&#34; speciation. If a polyploid can self-fertilize or reproduce asexually, it is even possible for one individual to produce an entirely new lineage. How often this extreme scenario occurs is unclear, with most studies of wild polyploids reporting multiple origins. Here, we explore the evolutionary history of the wild allotetraploid Mimulus sookensis, which was formed through hybridization between self-compatible, diploid species in the Mimulus guttatus complex. We generate a chromosome-scale reference assembly for M. sookensis and define its distinct subgenomes. Despite previous reports suggesting multiple origins of this highly selfing polyploid, we discover patterns of population genomic variation that provide unambiguous support for a single origin, which we estimate occurred [~]71,000 years ago. One M. sookensis subgenome is clearly derived from the selfer M. nasutus, which, based on organellar variation, also appears to be the maternal progenitor. The ancestor of the other subgenome is less certain, but it shares variation with both M. decorus and M. guttatus, two outcrossing diploids that overlap broadly with M. sookensis. Whatever its precise ancestry, this study establishes M. sookensis as an example of instantaneous speciation, likely facilitated by the polyploids predisposition to self-fertilize. With a reference genome for M. sookensis now available and its origin clarified, this wild tetraploid is poised to become a model for understanding the genetic and evolutionary mechanisms of polyploid persistence.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
