<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect
Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.
Score: 203.2, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994
Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect
Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.
Score: 203.2, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994
Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect
Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.
Score: 203.2, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994
Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect\nAuthors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.\nScore: 203.2, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994\nParthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection.",
  "keywords": [
    
  ],
  "articleBody": " Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect\nAuthors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.\nScore: 203.2, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994\nParthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.\nVenom tradeoff shapes interspecific interactions, physiology and reproduction\nAuthors: Surm, J. M.; Birch, S. J.; Macrander, J.; Jaimes-Becerra, A.; Fridrich, A.; Aharoni, R.; Rozenblat, R.; Sharabany, J.; Appelbaum, L.; Reitzel, A. M.; Moran, Y.\nScore: 30.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550294\nThe ability of an animal to effectively capture prey and defend against predators is pivotal for its survival. Venom, a mixture of many toxin proteins, shapes predator-prey interactions. Here, we use the sea anemone Nematostella vectensis to test how toxin genotypes impact predator-prey interactions. We developed a new genetic manipulation tool which significantly reduces both RNA and protein levels of Nv1, a major neurotoxin. In concert we recently discovered a native population of Nematostella that has lost Nv1.We demonstrate that these anemones lacking Nv1, have reduced ability to defend themselves against grass shrimp, a native predator. Additionally, secreted Nv1 can act indirectly in defense by attracting mummichog fish, which are known to prey on grass shrimp. This unravels a tritrophic interaction acting in animal defense at the molecular level. Additionally, our work reveals an evolutionary tradeoff, as the reduction of Nv1 levels causes faster growth and increased sexual and asexual reproductive rates.\nWhole genome approach to the structure and dynamics of Cacajao wild populations\nAuthors: Hermosilla-Albala, N.; Silva, F. E.; Cuadros-Espinoza, S.; Fontsere, C.; Valenzuela-Seba, A.; Gut, M.; Kelley, J. L.; Ruibal-Puertas, S.; Alentorn-Moron, P.; Faella, A.; Lizano, E.; Farias, I.; Hrbek, T.; Valsecchi, J.; Gut, I. G.; Rogers, J.; Farh, K. K.-H.; Kuderna, L. F. K.; Boubli, J. P.; Marques-Bonet, T.\nScore: 11.1, Published: 2023-07-25 DOI: 10.1101/2023.07.22.550156\nUakari monkeys (genus Cacajao) are endemic to the Amazon rainforest. These have been divided into two main groups: bald and black, mainly based on phenotypic and ecological differences, for which eight taxonomic species have been described. We present 48 geo-localized high coverage whole genomes from uakaris in wild populations across their habitat range. The resolution in terms of sequencing depth provided by this dataset has allowed us to closely explore the structure and evolutionary past of these populations inhabiting the highly dynamic and diverse Amazon rainforest, unavailable until now. Also, we have refined previous phylogenetic estimates by presenting the first whole genome phylogenies available for Cacajao, which were built based on 1Mb and 250kb windows respectively. In this context, bald and black uakaris showed differing genetic diversity ranges, group-specific non-synonymous variation enriched for pathogen-related pathways, and strong population structure, both among and within species. Despite the latter, connectivity was common inside each group whilst strongly shaped by allopatric barriers. Strikingly, even though their habitat ranges partially overlap, no migration was detected between these two groups. Lastly, demographic inference was performed employing a maximum likelihood approach, with which we i) obtained overall higher effective population size estimates for bald extant and ancestral populations than black in agreement with genetic diversity patterns, and ii) found the evolutionary divergence times in the genus were as old as one million years, depicting a quite recent evolutionary history. With this work we increased the representation of wild non-human primates in genomic studies, particularly that of platyrrhine lineages such as Cacajao.\nDevelopmental noise and phenotypic plasticity are correlated in Drosophila simulans\nAuthors: Saito, K.; Tsuboi, M.; Takahashi, Y.\nScore: 8.7, Published: 2023-07-29 DOI: 10.1101/2023.07.28.550919\nNon-genetic variation is the phenotypic variation induced by the differential expression of a genotype in response to varying environmental cues and is broadly categorized into two types: phenotypic plasticity and developmental noise. These variation aspects have been suggested to play an important role in adaptive evolution; however, the mechanisms by which these two types of non-genetic variations influence the evolutionary process are currently poorly understood. Using a machine-learning based phenotyping tool, we independently quantified the phenotypic plasticity and developmental noise in the wing morphological traits of a fruit fly Drosophila simulans. Utilizing a rearing experiment, we demonstrated plastic responses in both wing size and shape as well as non-zero heritability of both phenotypic plasticity and developmental noise, which suggests that adaptive phenotypic plasticity can evolve via genetic accommodation in the wing morphology of D. simulans. We found a positive correlation between phenotypic plasticity and developmental noise, while the correlation between the plastic response to three kinds of environmental factors that were examined (nutrient condition, temperature, and light-dark cycle) were poor. These results suggest that phenotypic plasticity and developmental noise contribute to evolvability in a similar manner, however, the mechanisms that underlie the correspondence between these two variation types remains to be elucidated. Lay SummaryNon-genetic variations consist of phenotypic plasticity and developmental noise, and these variations have been suggested to influence the direction and the rate of evolution. However, the role of phenotypic plasticity and developmental noise in evolutionary process is still poorly understood. Using a rearing experiment, we examined the heritability of plasticity and developmental noise, the correlation of the strength of plastic response to three kinds of environmental factors, and the relationship between plasticity and developmental noise in wing size and wing shape in Drosophila simulans. We found that the degree of phenotypic plasticity and developmental noise were heritable, and positively correlated with each other. Our results suggest that there two non-genetic variations dependently affect the direction and the rate of evolution together.\nTogether Inbreeding and Reproductive Compensation Favor Lethal t-Haplotypes\nAuthors: Munasinghe, M.; Brandvain, Y. J.\nScore: 8.6, Published: 2023-07-26 DOI: 10.1101/2023.07.26.550691\nMale mice who are heterozygous for distorting and non-distorting alleles at the t-haplotype transmit the driving t-haplotype around 90% of the time - a drastic departure from Mendelian expectations. This selfish act comes at a cost. The mechanism underlying transmission distortion in this system causes severe sterility in males homozygous for the drive alleles, ultimately preventing its fixation. Curiously, many driving t-haplotypes also induce embryonic lethality in both sexes when homozygous, however, this is neither universal nor a necessity for this distortion mechanism. Charlesworth provided an adaptive explanation for the evolution of lethal t-haplotypes in a population segregating for distorting and non-distorting t alleles - namely, if mothers compensate by replacing dead embryos with new offspring (or by transferring energy to surviving offspring), a recessive lethal can be favored because it effectively allows mothers the opportunity to trade in infertile males for potentially fertile offspring. However, this model requires near complete reproductive compensation for the invasion of the lethal t-haplotype and produces an equilibrium frequency of lethal drivers well below what is observed in nature. We show that even low levels of systemic inbreeding, which we model as brother-sister mating, allows lethal t-haplotypes to invade with much lower levels of reproductive compensation. Furthermore, inbreeding allows these lethal haplotypes to largely displace the ancestral male-sterile haplotypes. Our results show that together inbreeding and reproductive compensation move expected equilibria closer to observed haplotype frequencies in natural populations and occur under more reasonable parameter combinations.\nHow Veeries vary: Whole genome sequencing resolves fine-scale genetic structure in a long-distance migratory bird, Catharus fuscescens\nAuthors: Kimmitt, A. A.; Pegan, T. M.; Jones, A. W.; Winker, K.; Winger, B.\nScore: 6.7, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550389\nFine-scale resolution of spatial genetic structure is important for understanding a species evolutionary history and contemporary genetic diversity. For high-latitude species with high dispersal ability, such as long-distance migratory birds, populations typically exhibit little genetic structure due to high gene flow and recent postglacial expansion. Some migratory birds, however, show high breeding site fidelity, which might reduce gene flow such that population genetic structure could be detectable with sufficient genomic data. We sequenced over 120 low-coverage whole genomes from across the breeding range of a long-distance migratory bird, the Veery (Catharus fuscescens). As this species breeding range extends across both historically glaciated and unglaciated regions in North America, we evaluated whether contemporary patterns of structure and genetic diversity are consistent with historical population isolation in glacial refugia. We found strong evidence for isolation by distance across the breeding range, as well as significant population structure between southern Appalachian and northern populations. However, patterns of genetic diversity did not support southern Appalachia as a glacial refugium. Resolution of isolation by distance across the breeding range was sufficient to assign likely breeding origins of individuals sampled in this species poorly understood South American nonbreeding range, demonstrating the potential to assess migratory connectivity in this species using genomic data. Overall, our findings suggest that isolation by distance yields subtle associations between genetic structure and geography across the breeding range even in the absence of obvious historical vicariance or contemporary barriers to dispersal.\nPhylogeny and evolution of the SARS-CoV-2 spike gene from December 2022 to February 2023\nAuthors: Kao, H.-W.; Kao, H.-W.\nScore: 24.5, Published: 2023-07-24 DOI: 10.1101/2023.07.23.549423\nBackgroundBy the end of 2022, new variants of SARS-CoV-2, such as BQ.1.1.10, BA.4.6.3, XBB, and CH.1.1, emerged with higher fitness than BA.5. MethodsThe file (spikeprot0304), which contains spike protein sequences, isolates collected before March, 4, 2023, was downloaded from Global Initiative on Sharing All Influenza Data (GISAID). A total of 188 different spike protein sequences were chosen, of which their isolates were collected from December 2022 to February 2023. These sequences did not contain undetermined amino acid X, and each spike protein sequence had at least 100 identical isolate sequences in GISAID. Phylogenetic trees were reconstructed using IQ-TREE and MrBayes softwares. A median-join network was reconstructed using PopART software. Selection analyses were conducted using site model of PAML software. ResultsThe phylogenetic tree of the spike DNA sequences revealed that the majority of variants belonged to three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. The median network showed that these lineages had at least six major diversifying centers. The spike DNA sequences of these diversifying centers had the representative accession IDs (EPI_ISL_) of 16040256 (BN.1.2), 15970311 (BA.5), 16028739 (BA.5.11), 16028774 (BQ.1), 16027638 (BQ.1.1.23), and 16044705 (XBB.1.5). Selection analyses revealed 26 amino-acid sites under positive selection. These sites included L5, V83, W152, G181, N185, V213, H245, Y248, D253, S255, S256, G257, R346, R408, K444, V445, G446, N450, L452, N460, F486, Q613, Q675, T883, P1162, and V1264. ConclusionThe spike proteins of SARS-CoV-2 from December 2022 to February 2023 were characterized by a swarm of variants that were evolved from three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. These lineages had at least six diversifying centers. Selection analysis identified 26 amino acid sites were under positive selection. Continued surveillance and research are necessary to monitor the evolution and potential impact of these variants on public health.\nMore than just density: the role of egg number, food volume and container dimensions in mediating larval competition in Drosophila melanogaster\nAuthors: Venkitachalam, S.; V.S., S.; Joshi, A.\nScore: 6.1, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550621\nFlies of the genus Drosophila have been extensively used as model systems to study competition. Experiments using larval crowding in these species have furthered our understanding of competition ecology, stress-adaptation, density-dependent selection and population dynamics. Historically, larval competition has been induced by crowding the larvae in high density cultures, compared to low-density controls. However, recent studies have shown that two larval cultures having the same total eggs per mL food density, with different absolute quantities of eggs and food, can differ greatly in their density-specific fitness functions. Similarly, populations adapted to two types of cultures achieving the same density through different means, can also evolve different traits. Thus, it is clear that there is more to larval crowding than just eggs/food density, which has until now been the benchmark variable for quantifying larval crowding in Drosophila studies. In the current study, we explore the consequences of implementing crowding in different ways, using a three-way factorial experiment with egg number and food volume, cast into different food column heights or cylindrical vials with different diameters. We find that not just the same density, but cultures having the same egg number and food volume combination but experienced in food columns of varying height and diameter can have very different pre-adult survivorship and development times. We further propose a new variable for quantifying larval crowding - effective density, which is the density within the larval feeding band, a volume of food close to the surface, with access to air, wherein a majority of the larvae feed. We show that effective density is a much better predictor of the outcomes of competition than the popularly used total eggs/food density.\nThe evolution of competitive effectiveness and tolerance in populations of Drosophila melanogaster adapted to chronic larval crowding at varying combinations of egg number and food volume\nAuthors: Venkitachalam, S.; Temura, C.; Kokile, R.; Joshi, A.\nScore: 5.5, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550285\nThe theory of density-dependent selection posits that genotypic fitness can vary depending on the population density. Several long-term selection experiments on outbred populations of Drosophila adapted to chronically high larval densities have shown that the most common evolutionary response of such rearing is an increase in larval competitive ability. Some authors have proposed that a better understanding of the evolution of competitive ability can be achieved by its partitioning into effectiveness and tolerance components. Effectiveness is the amount of competitive inhibition imposed by a competitor on another, whereas tolerance is the degree to which a competitor can withstand inhibition. In this study, we have explored the evolution of effectiveness and tolerance components of competitive ability using three sets of outbred populations of D. melanogaster adapted to chronic larval crowding at different respective combinations of egg number, food volume and container dimensions. Effectiveness and tolerance were found to be dependent on the particular selection regime, the starting food amount and the trait used as the outcome of competition. Eclosion, dry biomass and dry weight per fly distributions over time indicated that competitive ability can also express itself in a time-dependent manner. The results suggest that larval competition and the evolution of competitive ability in Drosophila are extremely nuanced.\nA cline within an ecotype of the yellow monkeyflower, Mimulus guttatus\nAuthors: Zambiasi, T.; Lowry, D. B.\nScore: 5.4, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550335\nPremiseA key goal of evolutionary biologists is to understand how and why adaptive genetic variation is partitioned within species. In the yellow monkeyflower, Mimulus guttatus (syn. Erythranthe guttata), coastal perennial populations collectively constitute a single genetically and morphologically differentiated ecotype. While the distinctiveness of the coastal ecotype has now been well documented, there is also variation in environmental factors across the range of the coastal ecotype that could drive differentiation among its component populations in a more continuous way. MethodsBased on previous observations of a potential cline within this ecotype, we quantified plant height across coastal perennial accessions from 69 total populations in two greenhouse common garden experiments. To evaluate possible environmental factors driving the relationship between plant height and latitude, we regressed height against multiple climatic factors, including temperature, precipitation, and coastal wind speeds. ResultsIn both experiments, plant height was negatively correlated with latitude. Mimulus height correlated positively with annual precipitation and with mean wind speed, to a lesser degree, and negatively with annual mean temperature. We hypothesize that one or more of these factors drove clinal variation within the coastal ecotype. ConclusionOverall, our study illustrates the complexity of how the distribution of environmental variation can both simultaneously drive the evolution of distinct ecotypes as well as continuous clines within those ecotypes. These results are discussed in the context of the classic criticisms of ecotypes being intermediates in the process of speciation.\n",
  "wordCount" : "2762",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550994">
        <p class="paperTitle">Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.</p>
        <p class="info">Score: 203.2, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550994' target='https://doi.org/10.1101/2023.07.28.550994'> 10.1101/2023.07.28.550994</a></p>
        <p class="abstract">Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550294">
        <p class="paperTitle">Venom tradeoff shapes interspecific interactions, physiology and reproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Surm, J. M.; Birch, S. J.; Macrander, J.; Jaimes-Becerra, A.; Fridrich, A.; Aharoni, R.; Rozenblat, R.; Sharabany, J.; Appelbaum, L.; Reitzel, A. M.; Moran, Y.</p>
        <p class="info">Score: 30.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550294' target='https://doi.org/10.1101/2023.07.24.550294'> 10.1101/2023.07.24.550294</a></p>
        <p class="abstract">The ability of an animal to effectively capture prey and defend against predators is pivotal for its survival. Venom, a mixture of many toxin proteins, shapes predator-prey interactions. Here, we use the sea anemone Nematostella vectensis to test how toxin genotypes impact predator-prey interactions. We developed a new genetic manipulation tool which significantly reduces both RNA and protein levels of Nv1, a major neurotoxin. In concert we recently discovered a native population of Nematostella that has lost Nv1.We demonstrate that these anemones lacking Nv1, have reduced ability to defend themselves against grass shrimp, a native predator. Additionally, secreted Nv1 can act indirectly in defense by attracting mummichog fish, which are known to prey on grass shrimp. This unravels a tritrophic interaction acting in animal defense at the molecular level. Additionally, our work reveals an evolutionary tradeoff, as the reduction of Nv1 levels causes faster growth and increased sexual and asexual reproductive rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550156">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550156" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550156">
        <p class="paperTitle">Whole genome approach to the structure and dynamics of Cacajao wild populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550156" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550156" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hermosilla-Albala, N.; Silva, F. E.; Cuadros-Espinoza, S.; Fontsere, C.; Valenzuela-Seba, A.; Gut, M.; Kelley, J. L.; Ruibal-Puertas, S.; Alentorn-Moron, P.; Faella, A.; Lizano, E.; Farias, I.; Hrbek, T.; Valsecchi, J.; Gut, I. G.; Rogers, J.; Farh, K. K.-H.; Kuderna, L. F. K.; Boubli, J. P.; Marques-Bonet, T.</p>
        <p class="info">Score: 11.1, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550156' target='https://doi.org/10.1101/2023.07.22.550156'> 10.1101/2023.07.22.550156</a></p>
        <p class="abstract">Uakari monkeys (genus Cacajao) are endemic to the Amazon rainforest. These have been divided into two main groups: bald and black, mainly based on phenotypic and ecological differences, for which eight taxonomic species have been described. We present 48 geo-localized high coverage whole genomes from uakaris in wild populations across their habitat range. The resolution in terms of sequencing depth provided by this dataset has allowed us to closely explore the structure and evolutionary past of these populations inhabiting the highly dynamic and diverse Amazon rainforest, unavailable until now. Also, we have refined previous phylogenetic estimates by presenting the first whole genome phylogenies available for Cacajao, which were built based on 1Mb and 250kb windows respectively. In this context, bald and black uakaris showed differing genetic diversity ranges, group-specific non-synonymous variation enriched for pathogen-related pathways, and strong population structure, both among and within species. Despite the latter, connectivity was common inside each group whilst strongly shaped by allopatric barriers. Strikingly, even though their habitat ranges partially overlap, no migration was detected between these two groups. Lastly, demographic inference was performed employing a maximum likelihood approach, with which we i) obtained overall higher effective population size estimates for bald extant and ancestral populations than black in agreement with genetic diversity patterns, and ii) found the evolutionary divergence times in the genus were as old as one million years, depicting a quite recent evolutionary history. With this work we increased the representation of wild non-human primates in genomic studies, particularly that of platyrrhine lineages such as Cacajao.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550919">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550919" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550919">
        <p class="paperTitle">Developmental noise and phenotypic plasticity are correlated in Drosophila simulans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550919" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550919" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saito, K.; Tsuboi, M.; Takahashi, Y.</p>
        <p class="info">Score: 8.7, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550919' target='https://doi.org/10.1101/2023.07.28.550919'> 10.1101/2023.07.28.550919</a></p>
        <p class="abstract">Non-genetic variation is the phenotypic variation induced by the differential expression of a genotype in response to varying environmental cues and is broadly categorized into two types: phenotypic plasticity and developmental noise. These variation aspects have been suggested to play an important role in adaptive evolution; however, the mechanisms by which these two types of non-genetic variations influence the evolutionary process are currently poorly understood. Using a machine-learning based phenotyping tool, we independently quantified the phenotypic plasticity and developmental noise in the wing morphological traits of a fruit fly Drosophila simulans. Utilizing a rearing experiment, we demonstrated plastic responses in both wing size and shape as well as non-zero heritability of both phenotypic plasticity and developmental noise, which suggests that adaptive phenotypic plasticity can evolve via genetic accommodation in the wing morphology of D. simulans. We found a positive correlation between phenotypic plasticity and developmental noise, while the correlation between the plastic response to three kinds of environmental factors that were examined (nutrient condition, temperature, and light-dark cycle) were poor. These results suggest that phenotypic plasticity and developmental noise contribute to evolvability in a similar manner, however, the mechanisms that underlie the correspondence between these two variation types remains to be elucidated.

Lay SummaryNon-genetic variations consist of phenotypic plasticity and developmental noise, and these variations have been suggested to influence the direction and the rate of evolution. However, the role of phenotypic plasticity and developmental noise in evolutionary process is still poorly understood. Using a rearing experiment, we examined the heritability of plasticity and developmental noise, the correlation of the strength of plastic response to three kinds of environmental factors, and the relationship between plasticity and developmental noise in wing size and wing shape in Drosophila simulans. We found that the degree of phenotypic plasticity and developmental noise were heritable, and positively correlated with each other. Our results suggest that there two non-genetic variations dependently affect the direction and the rate of evolution together.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550691">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550691" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550691">
        <p class="paperTitle">Together Inbreeding and Reproductive Compensation Favor Lethal t-Haplotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550691" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550691" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Munasinghe, M.; Brandvain, Y. J.</p>
        <p class="info">Score: 8.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550691' target='https://doi.org/10.1101/2023.07.26.550691'> 10.1101/2023.07.26.550691</a></p>
        <p class="abstract">Male mice who are heterozygous for distorting and non-distorting alleles at the t-haplotype transmit the driving t-haplotype around 90% of the time - a drastic departure from Mendelian expectations. This selfish act comes at a cost. The mechanism underlying transmission distortion in this system causes severe sterility in males homozygous for the drive alleles, ultimately preventing its fixation. Curiously, many driving t-haplotypes also induce embryonic lethality in both sexes when homozygous, however, this is neither universal nor a necessity for this distortion mechanism. Charlesworth provided an adaptive explanation for the evolution of lethal t-haplotypes in a population segregating for distorting and non-distorting t alleles - namely, if mothers compensate by replacing dead embryos with new offspring (or by transferring energy to surviving offspring), a recessive lethal can be favored because it effectively allows mothers the opportunity to trade in infertile males for potentially fertile offspring. However, this model requires near complete reproductive compensation for the invasion of the lethal t-haplotype and produces an equilibrium frequency of lethal drivers well below what is observed in nature. We show that even low levels of systemic inbreeding, which we model as brother-sister mating, allows lethal t-haplotypes to invade with much lower levels of reproductive compensation. Furthermore, inbreeding allows these lethal haplotypes to largely displace the ancestral male-sterile haplotypes. Our results show that together inbreeding and reproductive compensation move expected equilibria closer to observed haplotype frequencies in natural populations and occur under more reasonable parameter combinations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550389">
        <p class="paperTitle">How Veeries vary: Whole genome sequencing resolves fine-scale genetic structure in a long-distance migratory bird,  Catharus fuscescens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kimmitt, A. A.; Pegan, T. M.; Jones, A. W.; Winker, K.; Winger, B.</p>
        <p class="info">Score: 6.7, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550389' target='https://doi.org/10.1101/2023.07.25.550389'> 10.1101/2023.07.25.550389</a></p>
        <p class="abstract">Fine-scale resolution of spatial genetic structure is important for understanding a species evolutionary history and contemporary genetic diversity. For high-latitude species with high dispersal ability, such as long-distance migratory birds, populations typically exhibit little genetic structure due to high gene flow and recent postglacial expansion. Some migratory birds, however, show high breeding site fidelity, which might reduce gene flow such that population genetic structure could be detectable with sufficient genomic data. We sequenced over 120 low-coverage whole genomes from across the breeding range of a long-distance migratory bird, the Veery (Catharus fuscescens). As this species breeding range extends across both historically glaciated and unglaciated regions in North America, we evaluated whether contemporary patterns of structure and genetic diversity are consistent with historical population isolation in glacial refugia. We found strong evidence for isolation by distance across the breeding range, as well as significant population structure between southern Appalachian and northern populations. However, patterns of genetic diversity did not support southern Appalachia as a glacial refugium. Resolution of isolation by distance across the breeding range was sufficient to assign likely breeding origins of individuals sampled in this species poorly understood South American nonbreeding range, demonstrating the potential to assess migratory connectivity in this species using genomic data. Overall, our findings suggest that isolation by distance yields subtle associations between genetic structure and geography across the breeding range even in the absence of obvious historical vicariance or contemporary barriers to dispersal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.549423">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.549423" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.549423">
        <p class="paperTitle">Phylogeny and evolution of the SARS-CoV-2 spike gene from December 2022 to February 2023</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.549423" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.549423" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kao, H.-W.; Kao, H.-W.</p>
        <p class="info">Score: 24.5, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.549423' target='https://doi.org/10.1101/2023.07.23.549423'> 10.1101/2023.07.23.549423</a></p>
        <p class="abstract">BackgroundBy the end of 2022, new variants of SARS-CoV-2, such as BQ.1.1.10, BA.4.6.3, XBB, and CH.1.1, emerged with higher fitness than BA.5.

MethodsThe file (spikeprot0304), which contains spike protein sequences, isolates collected before March, 4, 2023, was downloaded from Global Initiative on Sharing All Influenza Data (GISAID). A total of 188 different spike protein sequences were chosen, of which their isolates were collected from December 2022 to February 2023. These sequences did not contain undetermined amino acid X, and each spike protein sequence had at least 100 identical isolate sequences in GISAID. Phylogenetic trees were reconstructed using IQ-TREE and MrBayes softwares. A median-join network was reconstructed using PopART software. Selection analyses were conducted using site model of PAML software.

ResultsThe phylogenetic tree of the spike DNA sequences revealed that the majority of variants belonged to three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. The median network showed that these lineages had at least six major diversifying centers. The spike DNA sequences of these diversifying centers had the representative accession IDs (EPI_ISL_) of 16040256 (BN.1.2), 15970311 (BA.5), 16028739 (BA.5.11), 16028774 (BQ.1), 16027638 (BQ.1.1.23), and 16044705 (XBB.1.5). Selection analyses revealed 26 amino-acid sites under positive selection. These sites included L5, V83, W152, G181, N185, V213, H245, Y248, D253, S255, S256, G257, R346, R408, K444, V445, G446, N450, L452, N460, F486, Q613, Q675, T883, P1162, and V1264.

ConclusionThe spike proteins of SARS-CoV-2 from December 2022 to February 2023 were characterized by a swarm of variants that were evolved from three major lineages: BA.2 (BA.1.1.529.2), BA.5 (BA.1.1.529.5), and XBB. These lineages had at least six diversifying centers. Selection analysis identified 26 amino acid sites were under positive selection. Continued surveillance and research are necessary to monitor the evolution and potential impact of these variants on public health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550621">
        <p class="paperTitle">More than just density: the role of egg number, food volume and container dimensions in mediating larval competition in Drosophila melanogaster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkitachalam, S.; V.S., S.; Joshi, A.</p>
        <p class="info">Score: 6.1, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550621' target='https://doi.org/10.1101/2023.07.26.550621'> 10.1101/2023.07.26.550621</a></p>
        <p class="abstract">Flies of the genus Drosophila have been extensively used as model systems to study competition. Experiments using larval crowding in these species have furthered our understanding of competition ecology, stress-adaptation, density-dependent selection and population dynamics. Historically, larval competition has been induced by crowding the larvae in high density cultures, compared to low-density controls. However, recent studies have shown that two larval cultures having the same total eggs per mL food density, with different absolute quantities of eggs and food, can differ greatly in their density-specific fitness functions. Similarly, populations adapted to two types of cultures achieving the same density through different means, can also evolve different traits. Thus, it is clear that there is more to larval crowding than just eggs/food density, which has until now been the benchmark variable for quantifying larval crowding in Drosophila studies. In the current study, we explore the consequences of implementing crowding in different ways, using a three-way factorial experiment with egg number and food volume, cast into different food column heights or cylindrical vials with different diameters. We find that not just the same density, but cultures having the same egg number and food volume combination but experienced in food columns of varying height and diameter can have very different pre-adult survivorship and development times. We further propose a new variable for quantifying larval crowding - effective density, which is the density within the larval feeding band, a volume of food close to the surface, with access to air, wherein a majority of the larvae feed. We show that effective density is a much better predictor of the outcomes of competition than the popularly used total eggs/food density.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550285">
        <p class="paperTitle">The evolution of competitive effectiveness and tolerance in populations of Drosophila melanogaster adapted to chronic larval crowding at varying combinations of egg number and food volume</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkitachalam, S.; Temura, C.; Kokile, R.; Joshi, A.</p>
        <p class="info">Score: 5.5, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550285' target='https://doi.org/10.1101/2023.07.24.550285'> 10.1101/2023.07.24.550285</a></p>
        <p class="abstract">The theory of density-dependent selection posits that genotypic fitness can vary depending on the population density. Several long-term selection experiments on outbred populations of Drosophila adapted to chronically high larval densities have shown that the most common evolutionary response of such rearing is an increase in larval competitive ability. Some authors have proposed that a better understanding of the evolution of competitive ability can be achieved by its partitioning into effectiveness and tolerance components. Effectiveness is the amount of competitive inhibition imposed by a competitor on another, whereas tolerance is the degree to which a competitor can withstand inhibition. In this study, we have explored the evolution of effectiveness and tolerance components of competitive ability using three sets of outbred populations of D. melanogaster adapted to chronic larval crowding at different respective combinations of egg number, food volume and container dimensions. Effectiveness and tolerance were found to be dependent on the particular selection regime, the starting food amount and the trait used as the outcome of competition. Eclosion, dry biomass and dry weight per fly distributions over time indicated that competitive ability can also express itself in a time-dependent manner. The results suggest that larval competition and the evolution of competitive ability in Drosophila are extremely nuanced.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550335">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550335" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550335">
        <p class="paperTitle">A cline within an ecotype of the yellow monkeyflower, Mimulus guttatus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550335" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550335" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zambiasi, T.; Lowry, D. B.</p>
        <p class="info">Score: 5.4, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550335' target='https://doi.org/10.1101/2023.07.24.550335'> 10.1101/2023.07.24.550335</a></p>
        <p class="abstract">PremiseA key goal of evolutionary biologists is to understand how and why adaptive genetic variation is partitioned within species. In the yellow monkeyflower, Mimulus guttatus (syn. Erythranthe guttata), coastal perennial populations collectively constitute a single genetically and morphologically differentiated ecotype. While the distinctiveness of the coastal ecotype has now been well documented, there is also variation in environmental factors across the range of the coastal ecotype that could drive differentiation among its component populations in a more continuous way.

MethodsBased on previous observations of a potential cline within this ecotype, we quantified plant height across coastal perennial accessions from 69 total populations in two greenhouse common garden experiments. To evaluate possible environmental factors driving the relationship between plant height and latitude, we regressed height against multiple climatic factors, including temperature, precipitation, and coastal wind speeds.

ResultsIn both experiments, plant height was negatively correlated with latitude. Mimulus height correlated positively with annual precipitation and with mean wind speed, to a lesser degree, and negatively with annual mean temperature. We hypothesize that one or more of these factors drove clinal variation within the coastal ecotype.

ConclusionOverall, our study illustrates the complexity of how the distribution of environmental variation can both simultaneously drive the evolution of distinct ecotypes as well as continuous clines within those ecotypes. These results are discussed in the context of the classic criticisms of ecotypes being intermediates in the process of speciation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
