<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genomic and ecological factors shaping specialism and generalism across an entire subphylum
Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Genomic and ecological factors shaping specialism and generalism across an entire subphylum
Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T09:51:07+00:00" />
<meta property="article:modified_time" content="2023-06-28T09:51:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Genomic and ecological factors shaping specialism and generalism across an entire subphylum
Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://yorks0n.github.io/TRxiv2/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Genomic and ecological factors shaping specialism and generalism across an entire subphylum\nAuthors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J.",
  "keywords": [
    
  ],
  "articleBody": " Genomic and ecological factors shaping specialism and generalism across an entire subphylum\nAuthors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J. H.; Hulfachor, A. B.; Kurtzman, C. P.; Sampaio, J. P.; Goncalves, P.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Rokas, A.; Hittinger, C. T.\nScore: 69.9, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545611\nOrganisms exhibit extensive breadth in the range of niches that they inhabit, from very narrow (specialists) to very broad (generalists). The two major paradigms proposed to explain niche breadth variation invoke either trade-offs between performance efficiency and breadth or the presence of specific intrinsic (e.g., genomic) and extrinsic (e.g., ecological) factors. To gain insights into niche breadth evolution, we assembled an unprecedented amount of genomic, metabolic, and ecological data from nearly all known known species of the Saccharomycotina, an ancient subphylum of fungi that exhibits extensive genomic, metabolic, and ecological diversity. We curated and sequenced the genomes of 1,154 yeast strains from 1,090 species, quantitatively measured the growth of 843 species in 24 carbon and nitrogen sources, and constructed an environmental ontology of 1,088 species. We classified yeasts into three niche breadth categories (specialists, standard, or generalists), and we tested for trade-offs between breadth and growth rate, measured coevolution between traits, and quantified the contributions of intrinsic and extrinsic factors to phenotype. Differences between generalists and specialists were not explained by trade-offs or by extrinsic factors. Instead, we found strong evidence that large interspecific differences in carbon breadth stem from intrinsic genomic differences in metabolic genes and linked generalism to variation in specific metabolic pathways. From this vast dataset, intrinsic genomic factors emerge as a primary driving force of microbial niche breadths.\nChance favors the prepared genomes: horizontal transfer shapes the emergence of antibiotic resistance mutations in core genes.\nAuthors: Coluzzi, C.; Guillemet, M.; Mazzamurro, F.; Touchon, M.; Godfroid, M.; Achaz, G.; Glaser, P.; Rocha, E. P. C.\nScore: 51.4, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545734\nBacterial lineages vary in the frequency with which they acquire novel traits, like antibiotic resistance or virulence. While previous studies have highlighted the impact of the genetic background on the successful acquisition of novel traits through horizontal gene transfer, the impact of the latter on the subsequent evolution of bacterial genomes by point mutations remains poorly understood. Here, we studied the evolution of resistance to quinolones in thousands of Escherichia coli genomes. Resistance-conferring point mutations in the core genes are frequent and accumulate very quickly. We searched for gene gains and losses significantly associated with the subsequent acquisition of these resistance mutations. This revealed 60 groups of genes in genetic linkage whose gain or loss induced a change in the probability of subsequently becoming resistant to quinolones by point mutations in gyrA and parC. Although some of these chronologies may reflect epidemiological trends, most of these groups encoded functions that were previously associated with antibiotic resistance, tolerance, or persistence, often specifically under quinolone treatment. A lot of the largest groups were found in prophages or plasmids, and they usually increased the likelihood of subsequent resistance mutations. Conversely groups of lost genes were typically small and chromosomal. Quinolone resistance was among the first resistances acquired in the extant lineages of E. coli and its acquisition was associated with an increased likelihood of acquiring other types of resistances, including to aminoglycosides and beta-lactams. Our findings suggest that gene flow shapes the subsequent fixation rate of adaptive mutations in core genes. Given the substantial gene flow within bacterial genomes, interactions between horizontal transfer and point mutations in core genes may be key to the success of adaptation processes.\nProtein evolvability under rewired genetic codes\nAuthors: Rozhonova, H.; Marti-Gomez, C.; McCandlish, D. M.; Payne, J. L.\nScore: 33.2, Published: 2023-06-23 DOI: 10.1101/2023.06.20.545706\nThe standard genetic code defines the rules of translation for nearly every life form on Earth. It also determines the amino acid changes accessible via single-nucleotide mutations, thus influencing protein evolvability -- the ability of mutation to bring forth adaptive variation in protein function. One of the most striking features of the standard genetic code is its robustness to mutation, yet it remains an open question whether this robustness facilitates or frustrates protein evolvability. To answer this question, we use data from massively-parallel sequence-to-function assays to construct and analyze empirical adaptive landscapes under hundreds of thousands of rewired genetic codes, including those of codon compression schemes relevant to protein engineering and synthetic biology. We find that robust genetic codes tend to enhance protein evolvability by rendering smooth adaptive landscapes with few peaks, which are readily accessible from throughout sequence space. By constructing low-dimensional visualizations of these landscapes, which each comprise more than 16 million mRNA sequences, we demonstrate that alternative genetic codes can radically alter the topological features of the network of high-fitness genotypes. Whereas the genetic codes that optimize evolvability depend to some extent on the detailed relationship between amino acid sequence and protein function, we also uncover general design principles for engineering non-standard genetic codes for enhanced and diminished evolvability, which may facilitate directed protein evolution experiments and the biocontainment of synthetic organisms, respectively. Our findings demonstrate that the standard genetic code, a critical and near-universal cellular information processing system, not only mitigates replication and translation errors as compared to most alternative genetic codes, but also facilitates predictable and directional adaptive evolution by enabling evolving populations to readily find mutational paths to adaptation.\n241,000 to 335,000 Years Old Rock Engravings Made by Homo naledi in the Rising Star Cave system, South Africa.\nAuthors: Berger, L. R.; Hawks, J.; Fuentes, A.; van Rooyen, D.; Tsikoane, M.; Ramalepa, M.; Nkwe, S.; Molopyane, K.\nScore: 1757.5, Published: 2023-06-05 DOI: 10.1101/2023.06.01.543133\nThe production of painted, etched or engraved designs on cave walls or other surfaces is recognized as a major cognitive step in human evolution. Such intentional designs, which are widely interpreted as signifying, recording, and transmitting information in a durable manner were once considered exclusive to Late Pleistocene Homo sapiens. Recent work has demonstrated that other hominin groups also made such marks, including Neanderthals (Rodriguez-Vidal et al., 2014; Hoffmann et al., 2018), and possibly Middle-Pleistocene Homo erectus (Joordens et al., 2015). Such durable signs indicate an intentionality characteristic of meaning-making (Kissel and Fuentes 2018) which has been argued to require significant levels of cognitive abilities not found in species with smaller brain sizes (Parkington, 2010). In fact, the evolution of such meaning-making symbols is thought to be a core aspect of what it means to be \"human\" (Henshilwood, 2009). Here we present the first known example of abstract patterns and shapes engraved within the Dinaledi subsystem of the Rising Star Cave in South Africa. We identified markings incised into the dolomitic limestone walls of the cave. The engravings described here are deeply impressed cross-hatchings and other geometric shapes. The surfaces bearing these engravings appear to have been prepared and smoothed. In some areas there is residue that creates a sheen on the surface possibly indicating repeated handling or rubbing of the rock, and there is evidence of the application of dirt or sand to the surface by non-natural processes. Homo naledi entered this part of the cave system and buried bodies within the both the Dinaledi Chamber and adjacent Hill Antechamber between 241 and 335 ka (Dirks et al., 2017; Robbins et al., 2021, Berger et al, 2023a). The engravings described here are found on a pillar in the Hill Antechamber that extends into the natural fissure corridor that links the two chambers and we associate them with H. naledi.\nrphylopic: An R package for fetching, transforming, and visualising PhyloPic silhouettes\nAuthors: Gearty, W.; Jones, L. A.\nScore: 23.1, Published: 2023-06-26 DOI: 10.1101/2023.06.22.546191\nData visualisation is vital for data exploration, analysis, and communication in research. Moreover, it can bridge gaps between researchers and the general public by making research findings more accessible and engaging. Today, researchers increasingly conduct their data analyses in programming languages such as R and Python. The availability of data visualisation tools within these environments supports the generation of reproducible data analyses and visualisation workflows. However, resources from external databases are frequently required for data visualisation necessitating integration between these databases and the platforms in which researchers conduct their analyses. Here, we introduce rphylopic, an R package for fetching, transforming, and visualising silhouettes of organisms from the PhyloPic database. In addition to making over 7,000 organism silhouettes available within the R programming language, rphylopic empowers users to modify the appearance of these silhouettes for ultimate customisability when coding production-quality visualisations in both base R and ggplot2 workflows. In this work, we provide details about how the package can be installed, its implementation, and potential use cases. For the latter, we showcase three examples across the ecology and evolutionary biology spectrum. Our hope is that rphylopic will make it easier for biologists to develop more accessible and engaging data visualisations by making external resources readily accessible, customisable, and usable within R. In turn, by integrating into existing workflows, rphylopic helps to ensure that science is reproducible and accessible.\nUnraveling the Genomic Diversity and Evolutionary History of Captive Tigers in the United States\nAuthors: Armstrong, E. E.; Mooney, J. A.; Solari, K. A.; Kim, B. Y.; Barsh, G. S.; Grant, V. B.; Greenbaum, G.; Kaelin, C. B.; Panchenko, K.; Pickrell, J. K.; Rosenberg, N.; Ryder, O. A.; Yokoyama, T.; Ramakrishnan, U.; Petrov, D. A.; Hadly, E. A.\nScore: 19.9, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545608\nGenetic and genomic studies of rare and endangered species have focused broadly on describing diversity patterns and resolving phylogenetic relationships, with the overarching goal of informing conservation efforts. However, many studies do not consider genetic reserves that are potentially housed in captive populations. For tigers (Panthera tigris) in particular, captive individuals vastly outnumber those in the wild, and their diversity remains largely unexplored. Here, we present the first large-scale genetic study of the private (non-zoo) captive tiger population in the United States (U.S.), also known as Generic tigers. We find that the U.S. Generic tiger population has an admixture fingerprint comprising all six extant wild tiger subspecies (P. t. altaica, Amur; P. t. tigris, Bengal; P. t. corbetti, Indochinese; P. t. jacksoni, Malayan; P. t. amoyensis, South China; P. t. sumatrae, Sumatran). We show that the Generic tiger population has a comparable amount of genetic diversity to most wild subspecies, relatively few private variants, and fewer deleterious mutations. We also observe inbreeding coefficients that are similar to wild populations, suggesting that inbreeding in captive populations is not prevalent as previously thought, although there are some individuals within the Generic population that are quite inbred. Our results reflect the complex demographic history of the Generic tiger population in the U.S. Additionally, we develop a reference panel for tigers and show that it can be used with imputation to accurately distinguish individuals and assign ancestry even with ultra-low coverage (0.25x) data. We anticipate this comprehensive study and panel will propel future research and preservation of tigers in the U.S. and globally.\nA multi-layered integrative analysis reveals a cholesterol metabolic program in outer radial glia with implications for human brain evolution\nAuthors: Moriano, J.; Leonardi, O.; Vitriolo, A.; Testa, G.; Boeckx, C.\nScore: 17.1, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546307\nThe definition of molecular and cellular mechanisms contributing to evolutionary divergences in brain ontogenetic trajectories is essential to formulate hypotheses about the emergence of our species. Yet the functional dissection of evolutionary modifications derived in the Homo sapiens lineage at an appropriate level of granularity remains particularly challenging. Capitalizing on recent single-cell sequencing efforts that have massively profiled neural stem cells from the developing human cortex, we develop an integrative computational framework in which we perform (i) trajectory inference and gene regulatory network reconstruction, (ii) (pseudo)time-informed non-negative matrix factorization for learning the dynamics of gene expression programs, and (iii) paleogenomic analysis for a higher-resolution mapping of the regulatory landscape where our species acquired derived mutations in comparison to our closest relatives. We provide evidence for cell type-specific activation and regulation of gene expression programs during indirect neurogenesis. In particular, our analysis uncovers a zinc-finger transcription factor, KLF6, as a key regulator of a cholesterol metabolic program specifically in outer radial glia. Our strategy allows us to further probe whether the (semi)discrete gene expression programs identified have been under selective pressures in our species lineage. A cartography of the regulatory landscape impacted by Homo sapiens derived transcription factor binding site disruptions reveals signals of selection clustering around regulatory regions associated with GLI3, a well-known regulator of the radial glial cell cycle. As a whole, our study contributes to the evidence of significant changes impacting metabolic pathways in recent human brain evolution\nAncient Sheep Genomes reveal four Millennia of North European Short-Tailed Sheep in the Baltic Sea region\nAuthors: Larsson, M. N.; Morell Miranda, P.; Pan, L.; Basak Vural, K.; Kaptan, D.; Rodrigues Soares, A. E.; Kivikero, H.; Kantanen, J.; Somel, M.; Özer, F.; Johansson, A. M.; Stora, J.; Günther, T.\nScore: 15.9, Published: 2023-06-27 DOI: 10.1101/2023.06.26.544912\nSheep are among the earliest domesticated livestock species, with a wide variety of breeds present today. However, it remains unclear how far back this breed diversity goes, with formal documentation only dating back a few centuries. North European short tail breeds are often assumed to be among the oldest domestic sheep populations, even thought to represent relicts of the earliest sheep expansions during the Neolithic period reaching Scandinavia less than 6000 years ago. This study sequenced the genomes (up to 11.6X) of five sheep remains from the Baltic islands of Gotland and Aringland, dating from Late Neolithic (~4100 calBP) to historical times (~1600 CE). Our findings indicate that these ancient sheep already possessed the genetic characteristics of modern North European short tail breeds, suggesting a long-term continuity of this breed type in the Baltic Sea region. Despite the wide temporal spread, population genetic analyses show high levels of affinity between the ancient genomes and they exhibit higher genetic diversity compared to modern breeds, implying a loss of diversity in recent centuries associated with breed formation. Finally, we see a potential signature of an even earlier, genetically different form of sheep in Scandinavia as these samples do not represent the first sheep in Northern Europe. Our results shed light on the development of breeds in Northern Europe specifically and the development of genetic diversity in sheep breeds and their expansion from the domestication center in general.\nBurials and engravings in a small-brained hominin, Homo naledi, from the late Pleistocene: contexts and evolutionary implications\nAuthors: Fuentes, A.; Kissel, M.; Spikins, P.; Molopyane, K.; Hawks, J.; Berger, L. R.\nScore: 1457.8, Published: 2023-06-05 DOI: 10.1101/2023.06.01.543135\nData from recent explorations in the Dinaledi subsystem illustrates one of the earliest examples of a mortuary practice in hominins and offers the earliest evidence of multiple interments and funerary actions, as well as evidence of the early creation of meaning making by a hominin. The hominin undertaking these behaviors was the small-brained Homo naledi. These data call into question several key assumptions about behavioral and cognitive evolution in Pleistocene hominins. The evidence from Dinaledi push back the temporal origins of mortuary and funerary behaviors and associate the creation of meaning making with a small-brained species and thus challenge key assumptions about the role and importance of encephalization in human evolution. This suggests that the hominin socio-cognitive niche and its relation to meaning-making activities is more diverse than previously thought. The association of these activities in subterranean spaces accessed and modified by the small brained species Homo naledi impacts assertations that technological and cognitive advances in human evolution are associated solely with the evolution of larger brains. One-Sentence SummaryBurials and related meaning making in a small-brained hominin alter our understandings of human evolution.\nTRAILS: tree reconstruction of ancestry using incomplete lineage sorting\nAuthors: Rivas-Gonzalez, I.; Schierup, M. H.; Wakeley, J.; Hobolth, A.\nScore: 13.0, Published: 2023-06-23 DOI: 10.1101/2023.06.22.546039\nAO_SCPLOWBSTRACTC_SCPLOWGenome-wide genealogies of multiple species carry detailed information about demographic and selection processes on individual branches of the phylogeny. Here, we introduce TRAILS, a hidden Markov model that accurately infers time-resolved population genetics parameters, such as ancestral effective population sizes and speciation times, for ancestral branches using a multi-species alignment of three species and an outgroup. TRAILS leverages the information contained in incomplete lineage sorting fragments by modelling genealogies along the genome as rooted three-leaved trees, each with a topology and two coalescent events happening in discretized time intervals within the phylogeny. Posterior decoding of the hidden Markov model can be used to infer the ancestral recombination graph for the alignment and details on demographic changes within a branch. Since TRAILS performs posterior decoding at the base-pair level, genome-wide scans based on the posterior probabilities can be devised to detect deviations from neutrality. Using TRAILS on a human-chimp-gorilla-orangutan alignment, we recover speciation parameters and extract information about the topology and coalescent times at high resolution.\n",
  "wordCount" : "2840",
  "inLanguage": "en",
  "datePublished": "2023-06-28T09:51:07Z",
  "dateModified": "2023-06-28T09:51:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545611">
        <p class="paperTitle">Genomic and ecological factors shaping specialism and generalism across an entire subphylum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J. H.; Hulfachor, A. B.; Kurtzman, C. P.; Sampaio, J. P.; Goncalves, P.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 69.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545611' target='https://doi.org/10.1101/2023.06.19.545611'> 10.1101/2023.06.19.545611</a></p>
        <p class="abstract">Organisms exhibit extensive breadth in the range of niches that they inhabit, from very narrow (specialists) to very broad (generalists). The two major paradigms proposed to explain niche breadth variation invoke either trade-offs between performance efficiency and breadth or the presence of specific intrinsic (e.g., genomic) and extrinsic (e.g., ecological) factors. To gain insights into niche breadth evolution, we assembled an unprecedented amount of genomic, metabolic, and ecological data from nearly all known known species of the Saccharomycotina, an ancient subphylum of fungi that exhibits extensive genomic, metabolic, and ecological diversity. We curated and sequenced the genomes of 1,154 yeast strains from 1,090 species, quantitatively measured the growth of 843 species in 24 carbon and nitrogen sources, and constructed an environmental ontology of 1,088 species. We classified yeasts into three niche breadth categories (specialists, standard, or generalists), and we tested for trade-offs between breadth and growth rate, measured coevolution between traits, and quantified the contributions of intrinsic and extrinsic factors to phenotype. Differences between generalists and specialists were not explained by trade-offs or by extrinsic factors. Instead, we found strong evidence that large interspecific differences in carbon breadth stem from intrinsic genomic differences in metabolic genes and linked generalism to variation in specific metabolic pathways. From this vast dataset, intrinsic genomic factors emerge as a primary driving force of microbial niche breadths.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545734">
        <p class="paperTitle">Chance favors the prepared genomes: horizontal transfer shapes the emergence of antibiotic resistance mutations in core genes.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coluzzi, C.; Guillemet, M.; Mazzamurro, F.; Touchon, M.; Godfroid, M.; Achaz, G.; Glaser, P.; Rocha, E. P. C.</p>
        <p class="info">Score: 51.4, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545734' target='https://doi.org/10.1101/2023.06.20.545734'> 10.1101/2023.06.20.545734</a></p>
        <p class="abstract">Bacterial lineages vary in the frequency with which they acquire novel traits, like antibiotic resistance or virulence. While previous studies have highlighted the impact of the genetic background on the successful acquisition of novel traits through horizontal gene transfer, the impact of the latter on the subsequent evolution of bacterial genomes by point mutations remains poorly understood. Here, we studied the evolution of resistance to quinolones in thousands of Escherichia coli genomes. Resistance-conferring point mutations in the core genes are frequent and accumulate very quickly. We searched for gene gains and losses significantly associated with the subsequent acquisition of these resistance mutations. This revealed 60 groups of genes in genetic linkage whose gain or loss induced a change in the probability of subsequently becoming resistant to quinolones by point mutations in gyrA and parC. Although some of these chronologies may reflect epidemiological trends, most of these groups encoded functions that were previously associated with antibiotic resistance, tolerance, or persistence, often specifically under quinolone treatment. A lot of the largest groups were found in prophages or plasmids, and they usually increased the likelihood of subsequent resistance mutations. Conversely groups of lost genes were typically small and chromosomal. Quinolone resistance was among the first resistances acquired in the extant lineages of E. coli and its acquisition was associated with an increased likelihood of acquiring other types of resistances, including to aminoglycosides and beta-lactams. Our findings suggest that gene flow shapes the subsequent fixation rate of adaptive mutations in core genes. Given the substantial gene flow within bacterial genomes, interactions between horizontal transfer and point mutations in core genes may be key to the success of adaptation processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545706">
        <p class="paperTitle">Protein evolvability under rewired genetic codes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rozhonova, H.; Marti-Gomez, C.; McCandlish, D. M.; Payne, J. L.</p>
        <p class="info">Score: 33.2, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545706' target='https://doi.org/10.1101/2023.06.20.545706'> 10.1101/2023.06.20.545706</a></p>
        <p class="abstract">The standard genetic code defines the rules of translation for nearly every life form on Earth. It also determines the amino acid changes accessible via single-nucleotide mutations, thus influencing protein evolvability -- the ability of mutation to bring forth adaptive variation in protein function. One of the most striking features of the standard genetic code is its robustness to mutation, yet it remains an open question whether this robustness facilitates or frustrates protein evolvability. To answer this question, we use data from massively-parallel sequence-to-function assays to construct and analyze empirical adaptive landscapes under hundreds of thousands of rewired genetic codes, including those of codon compression schemes relevant to protein engineering and synthetic biology. We find that robust genetic codes tend to enhance protein evolvability by rendering smooth adaptive landscapes with few peaks, which are readily accessible from throughout sequence space. By constructing low-dimensional visualizations of these landscapes, which each comprise more than 16 million mRNA sequences, we demonstrate that alternative genetic codes can radically alter the topological features of the network of high-fitness genotypes. Whereas the genetic codes that optimize evolvability depend to some extent on the detailed relationship between amino acid sequence and protein function, we also uncover general design principles for engineering non-standard genetic codes for enhanced and diminished evolvability, which may facilitate directed protein evolution experiments and the biocontainment of synthetic organisms, respectively. Our findings demonstrate that the standard genetic code, a critical and near-universal cellular information processing system, not only mitigates replication and translation errors as compared to most alternative genetic codes, but also facilitates predictable and directional adaptive evolution by enabling evolving populations to readily find mutational paths to adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543133">
        <p class="paperTitle">241,000 to 335,000 Years Old Rock Engravings Made by Homo naledi in the Rising Star Cave system, South Africa.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berger, L. R.; Hawks, J.; Fuentes, A.; van Rooyen, D.; Tsikoane, M.; Ramalepa, M.; Nkwe, S.; Molopyane, K.</p>
        <p class="info">Score: 1757.5, Published: 2023-06-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543133' target='https://doi.org/10.1101/2023.06.01.543133'> 10.1101/2023.06.01.543133</a></p>
        <p class="abstract">The production of painted, etched or engraved designs on cave walls or other surfaces is recognized as a major cognitive step in human evolution. Such intentional designs, which are widely interpreted as signifying, recording, and transmitting information in a durable manner were once considered exclusive to Late Pleistocene Homo sapiens. Recent work has demonstrated that other hominin groups also made such marks, including Neanderthals (Rodriguez-Vidal et al., 2014; Hoffmann et al., 2018), and possibly Middle-Pleistocene Homo erectus (Joordens et al., 2015). Such durable signs indicate an intentionality characteristic of meaning-making (Kissel and Fuentes 2018) which has been argued to require significant levels of cognitive abilities not found in species with smaller brain sizes (Parkington, 2010). In fact, the evolution of such meaning-making symbols is thought to be a core aspect of what it means to be &#34;human&#34; (Henshilwood, 2009). Here we present the first known example of abstract patterns and shapes engraved within the Dinaledi subsystem of the Rising Star Cave in South Africa. We identified markings incised into the dolomitic limestone walls of the cave. The engravings described here are deeply impressed cross-hatchings and other geometric shapes. The surfaces bearing these engravings appear to have been prepared and smoothed. In some areas there is residue that creates a sheen on the surface possibly indicating repeated handling or rubbing of the rock, and there is evidence of the application of dirt or sand to the surface by non-natural processes. Homo naledi entered this part of the cave system and buried bodies within the both the Dinaledi Chamber and adjacent Hill Antechamber between 241 and 335 ka (Dirks et al., 2017; Robbins et al., 2021, Berger et al, 2023a). The engravings described here are found on a pillar in the Hill Antechamber that extends into the natural fissure corridor that links the two chambers and we associate them with H. naledi.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546191" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546191">
        <p class="paperTitle">rphylopic: An R package for fetching, transforming, and visualising PhyloPic silhouettes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gearty, W.; Jones, L. A.</p>
        <p class="info">Score: 23.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546191' target='https://doi.org/10.1101/2023.06.22.546191'> 10.1101/2023.06.22.546191</a></p>
        <p class="abstract">Data visualisation is vital for data exploration, analysis, and communication in research. Moreover, it can bridge gaps between researchers and the general public by making research findings more accessible and engaging. Today, researchers increasingly conduct their data analyses in programming languages such as R and Python. The availability of data visualisation tools within these environments supports the generation of reproducible data analyses and visualisation workflows. However, resources from external databases are frequently required for data visualisation necessitating integration between these databases and the platforms in which researchers conduct their analyses. Here, we introduce rphylopic, an R package for fetching, transforming, and visualising silhouettes of organisms from the PhyloPic database. In addition to making over 7,000 organism silhouettes available within the R programming language, rphylopic empowers users to modify the appearance of these silhouettes for ultimate customisability when coding production-quality visualisations in both base R and ggplot2 workflows. In this work, we provide details about how the package can be installed, its implementation, and potential use cases. For the latter, we showcase three examples across the ecology and evolutionary biology spectrum. Our hope is that rphylopic will make it easier for biologists to develop more accessible and engaging data visualisations by making external resources readily accessible, customisable, and usable within R. In turn, by integrating into existing workflows, rphylopic helps to ensure that science is reproducible and accessible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545608">
        <p class="paperTitle">Unraveling the Genomic Diversity and Evolutionary History of Captive Tigers in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Armstrong, E. E.; Mooney, J. A.; Solari, K. A.; Kim, B. Y.; Barsh, G. S.; Grant, V. B.; Greenbaum, G.; Kaelin, C. B.; Panchenko, K.; Pickrell, J. K.; Rosenberg, N.; Ryder, O. A.; Yokoyama, T.; Ramakrishnan, U.; Petrov, D. A.; Hadly, E. A.</p>
        <p class="info">Score: 19.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545608' target='https://doi.org/10.1101/2023.06.19.545608'> 10.1101/2023.06.19.545608</a></p>
        <p class="abstract">Genetic and genomic studies of rare and endangered species have focused broadly on describing diversity patterns and resolving phylogenetic relationships, with the overarching goal of informing conservation efforts. However, many studies do not consider genetic reserves that are potentially housed in captive populations. For tigers (Panthera tigris) in particular, captive individuals vastly outnumber those in the wild, and their diversity remains largely unexplored. Here, we present the first large-scale genetic study of the private (non-zoo) captive tiger population in the United States (U.S.), also known as  Generic tigers. We find that the U.S. Generic tiger population has an admixture fingerprint comprising all six extant wild tiger subspecies (P. t. altaica, Amur; P. t. tigris, Bengal; P. t. corbetti, Indochinese; P. t. jacksoni, Malayan; P. t. amoyensis, South China; P. t. sumatrae, Sumatran). We show that the Generic tiger population has a comparable amount of genetic diversity to most wild subspecies, relatively few private variants, and fewer deleterious mutations. We also observe inbreeding coefficients that are similar to wild populations, suggesting that inbreeding in captive populations is not prevalent as previously thought, although there are some individuals within the Generic population that are quite inbred. Our results reflect the complex demographic history of the Generic tiger population in the U.S. Additionally, we develop a reference panel for tigers and show that it can be used with imputation to accurately distinguish individuals and assign ancestry even with ultra-low coverage (0.25x) data. We anticipate this comprehensive study and panel will propel future research and preservation of tigers in the U.S. and globally.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546307">
        <p class="paperTitle">A multi-layered integrative analysis reveals a cholesterol metabolic program in outer radial glia with implications for human brain evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moriano, J.; Leonardi, O.; Vitriolo, A.; Testa, G.; Boeckx, C.</p>
        <p class="info">Score: 17.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546307' target='https://doi.org/10.1101/2023.06.23.546307'> 10.1101/2023.06.23.546307</a></p>
        <p class="abstract">The definition of molecular and cellular mechanisms contributing to evolutionary divergences in brain ontogenetic trajectories is essential to formulate hypotheses about the emergence of our species. Yet the functional dissection of evolutionary modifications derived in the Homo sapiens lineage at an appropriate level of granularity remains particularly challenging. Capitalizing on recent single-cell sequencing efforts that have massively profiled neural stem cells from the developing human cortex, we develop an integrative computational framework in which we perform (i) trajectory inference and gene regulatory network reconstruction, (ii) (pseudo)time-informed non-negative matrix factorization for learning the dynamics of gene expression programs, and (iii) paleogenomic analysis for a higher-resolution mapping of the regulatory landscape where our species acquired derived mutations in comparison to our closest relatives. We provide evidence for cell type-specific activation and regulation of gene expression programs during indirect neurogenesis. In particular, our analysis uncovers a zinc-finger transcription factor, KLF6, as a key regulator of a cholesterol metabolic program specifically in outer radial glia. Our strategy allows us to further probe whether the (semi)discrete gene expression programs identified have been under selective pressures in our species lineage. A cartography of the regulatory landscape impacted by Homo sapiens derived transcription factor binding site disruptions reveals signals of selection clustering around regulatory regions associated with GLI3, a well-known regulator of the radial glial cell cycle. As a whole, our study contributes to the evidence of significant changes impacting metabolic pathways in recent human brain evolution</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.544912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.544912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.544912">
        <p class="paperTitle">Ancient Sheep Genomes reveal four Millennia of North European Short-Tailed Sheep in the Baltic Sea region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.544912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.544912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsson, M. N.; Morell Miranda, P.; Pan, L.; Basak Vural, K.; Kaptan, D.; Rodrigues Soares, A. E.; Kivikero, H.; Kantanen, J.; Somel, M.; Özer, F.; Johansson, A. M.; Stora, J.; Günther, T.</p>
        <p class="info">Score: 15.9, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.544912' target='https://doi.org/10.1101/2023.06.26.544912'> 10.1101/2023.06.26.544912</a></p>
        <p class="abstract">Sheep are among the earliest domesticated livestock species, with a wide variety of breeds present today. However, it remains unclear how far back this breed diversity goes, with formal documentation only dating back a few centuries. North European short tail breeds are often assumed to be among the oldest domestic sheep populations, even thought to represent relicts of the earliest sheep expansions during the Neolithic period reaching Scandinavia less than 6000 years ago. This study sequenced the genomes (up to 11.6X) of five sheep remains from the Baltic islands of Gotland and Aringland, dating from Late Neolithic (~4100 calBP) to historical times (~1600 CE). Our findings indicate that these ancient sheep already possessed the genetic characteristics of modern North European short tail breeds, suggesting a long-term continuity of this breed type in the Baltic Sea region. Despite the wide temporal spread, population genetic analyses show high levels of affinity between the ancient genomes and they exhibit higher genetic diversity compared to modern breeds, implying a loss of diversity in recent centuries associated with breed formation. Finally, we see a potential signature of an even earlier, genetically different form of sheep in Scandinavia as these samples do not represent the first sheep in Northern Europe. Our results shed light on the development of breeds in Northern Europe specifically and the development of genetic diversity in sheep breeds and their expansion from the domestication center in general.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543135">
        <p class="paperTitle">Burials and engravings in a small-brained hominin, Homo naledi, from the late Pleistocene: contexts and evolutionary implications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuentes, A.; Kissel, M.; Spikins, P.; Molopyane, K.; Hawks, J.; Berger, L. R.</p>
        <p class="info">Score: 1457.8, Published: 2023-06-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543135' target='https://doi.org/10.1101/2023.06.01.543135'> 10.1101/2023.06.01.543135</a></p>
        <p class="abstract">Data from recent explorations in the Dinaledi subsystem illustrates one of the earliest examples of a mortuary practice in hominins and offers the earliest evidence of multiple interments and funerary actions, as well as evidence of the early creation of meaning making by a hominin. The hominin undertaking these behaviors was the small-brained Homo naledi. These data call into question several key assumptions about behavioral and cognitive evolution in Pleistocene hominins. The evidence from Dinaledi push back the temporal origins of mortuary and funerary behaviors and associate the creation of meaning making with a small-brained species and thus challenge key assumptions about the role and importance of encephalization in human evolution. This suggests that the hominin socio-cognitive niche and its relation to meaning-making activities is more diverse than previously thought. The association of these activities in subterranean spaces accessed and modified by the small brained species Homo naledi impacts assertations that technological and cognitive advances in human evolution are associated solely with the evolution of larger brains.

One-Sentence SummaryBurials and related meaning making in a small-brained hominin alter our understandings of human evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546039">
        <p class="paperTitle">TRAILS: tree reconstruction of ancestry using incomplete lineage sorting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivas-Gonzalez, I.; Schierup, M. H.; Wakeley, J.; Hobolth, A.</p>
        <p class="info">Score: 13.0, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546039' target='https://doi.org/10.1101/2023.06.22.546039'> 10.1101/2023.06.22.546039</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWGenome-wide genealogies of multiple species carry detailed information about demographic and selection processes on individual branches of the phylogeny. Here, we introduce TRAILS, a hidden Markov model that accurately infers time-resolved population genetics parameters, such as ancestral effective population sizes and speciation times, for ancestral branches using a multi-species alignment of three species and an outgroup. TRAILS leverages the information contained in incomplete lineage sorting fragments by modelling genealogies along the genome as rooted three-leaved trees, each with a topology and two coalescent events happening in discretized time intervals within the phylogeny. Posterior decoding of the hidden Markov model can be used to infer the ancestral recombination graph for the alignment and details on demographic changes within a branch. Since TRAILS performs posterior decoding at the base-pair level, genome-wide scans based on the posterior probabilities can be devised to detect deviations from neutrality. Using TRAILS on a human-chimp-gorilla-orangutan alignment, we recover speciation parameters and extract information about the topology and coalescent times at high resolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
