<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Self-assembling viral histones unravel early nucleosome evolution
Authors: Irwin, N. A. T.; Richards, T. A.
Score: 51.2, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576
Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Self-assembling viral histones unravel early nucleosome evolution
Authors: Irwin, N. A. T.; Richards, T. A.
Score: 51.2, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576
Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Self-assembling viral histones unravel early nucleosome evolution
Authors: Irwin, N. A. T.; Richards, T. A.
Score: 51.2, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576
Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Self-assembling viral histones unravel early nucleosome evolution\nAuthors: Irwin, N. A. T.; Richards, T. A.\nScore: 51.2, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576\nNucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution.",
  "keywords": [
    
  ],
  "articleBody": " Self-assembling viral histones unravel early nucleosome evolution\nAuthors: Irwin, N. A. T.; Richards, T. A.\nScore: 51.2, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558576\nNucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution. However, functional constraints and the extinction of stem-eukaryotes have concealed how these dynamic systems evolved from simpler histone homologues in Archaea. Viral histones have also previously been identified and are thought to reflect an ancestral state as they often comprise multiple histone paralogues arranged within a single protein, termed histone repeats. Here, using viruses as an alternative source of variation, we expand the known diversity of histones and develop an empirical hypothesis for the origin of the nucleosome. Our analysis identified hundreds of histones with variable domain repeat configurations including histone singlets, doublets, triplets, and quadruplets, the latter comprising the four core histones arranged in series. Viral histone repeats consistently branch between Archaea and eukaryotes in phylogenetic trees and display intermediate functions, self-assembling into eukaryotic-like nucleosomes that stack into archaeal-like oligomers capable of impacting genomic activity and condensing DNA. The linkers conjoining the histone repeats also facilitate nucleosome formation and can promote the assembly of eukaryotic nucleosomes in the bacterium, Escherichia coli. Combining these data, we hypothesize that viral histone repeats represent molecular relics acquired by viruses from stem-eukaryotes during eukaryogenesis and suggest that nucleosome evolution may have proceeded through histone repeat intermediates.\nTransposable element abundance subtly contributes to lower fitness in maize\nAuthors: Stitzer, M. C.; Khaipho-Burch, M. B.; Hudson, A. I.; Song, B.; Valdes-Franco, J. A.; Ramstein, G. P.; Feschotte, C.; Buckler, E. S.\nScore: 25.8, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557618\nTransposable elements (TEs) have long been shown to have deleterious effects on the survival and reproduction of their host organism. As TEs are mobile DNA that jump to new positions, this deleterious cost can occur directly, by inserting into genes and regulatory sequences. Classical population genetic theory suggests copy-number dependent selection against TEs is necessary to prevent TEs from expanding so much they take over a genome. Such models have been difficult to interpret when applied to large genomes like maize, where there are hundreds of thousands of TE insertions that collectively make up 85% of the genome. Here, we use nearly 5000 inbred lines from maize mapping populations and a pan-genomic imputation approach to measure TE content. Segregating TE content gives rise to 100 Mb differences between individuals, and populations often show transgressive segregation in TE content. We use replicated phenotypes measured in hybrids across numerous years and environments to empirically measure the fitness costs of TEs. For an annual plant like maize, grain yield is not only a key agronomic phenotype, but also a direct measure of reproductive output. We find weak negative effects of TE accumulation on grain yield, nearing the limit of the efficacy of natural selection in maize. This results in a loss of one kernel ({approx}0.1% of average per-plant yield) for every additional 14 Mb of TE content. This deleterious load is enriched in TEs within 1 kilobase of genes and young TE insertions. Together, we provide rare empirical measurements of the fitness costs of TEs, and suggest that the TEs we see today in the genome have been filtered by selection against their deleterious consequences on maize fitness.\nGene regulatory network that shaped the evolution of larval sensory organ in Cnidaria\nAuthors: Gilbert, E.; Craggs, J.; Modepalli, V.\nScore: 24.4, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557727\nAmong non-bilaterian animals, a larval apical sensory organ with integrated neurons is only found in cnidarians. Within cnidarians, an apical organ with a ciliary tuft is mainly found in Actiniaria. Whether this apical tuft has evolved independently in Actiniaria or alternatively originated in the common ancestor of Cnidaria and Bilateria and was lost in specific groups is uncertain. We generated transcriptomes of the apical domain during the planula stage of four species representing three key groups of cnidarians: Aurelia aurita (Scyphozoa), Nematostella vectensis (Actiniaria), and Acropora millepora \u0026 Acropora tenuis (Scleractinia). We showed that the canonical genes implicated in patterning the apical domain of Nematostella are largely absent in Aurelia, indicating that scyphozoans and anthozoans do not share apical organ homology. In contrast, the apical domain of the scleractinian planula shares gene expression pattern with Nematostella. By comparing the larval single-cell transcriptomes, we revealed the apical organ cell type of Scleractinia and confirmed its homology to Actiniaria. However, Fgfa2, a vital regulator of the regionalisation of the Nematostella apical organ, is absent in the scleractinian genome. Likewise, we found that FoxJ1 and 245 genes associated with cilia are exclusively expressed in the Nematostella apical domain, which is in line with the presence of ciliary apical tuft in Actiniaria and its absence in Scleractinia and Scyphozoa. Our findings suggest that the common ancestor of cnidarians and bilaterians lacked an apical organ with a ciliary tuft, and it could have evolved independently in several clades of cnidarians and bilaterians.\nThe evolution of transposable elements in Brachypodium distachyon is governed by purifying selection, while neutral and adaptive processes play a minor role\nAuthors: Horvath, R.; Minadakis, N.; Bourgeois, Y. X. C.; Roulin, A.\nScore: 13.2, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557873\nBackgroundUnderstanding how plants adapt to changing environments and the potential contribution of transposable elements (TEs) to this process is a key question in evolutionary genomics. While TEs have recently been put forward as active players in the context of adaptation, few studies have thoroughly investigated their precise role in plant evolution. Here we used the wild Mediterranean grass Brachypodium distachyon as a model species to identify and quantify the forces acting on TEs during the adaptation of this species to various conditions, across its entire geographic range. ResultsUsing sequencing data from more than 320 natural B. distachyon accessions and a suite of population genomics approaches, we reveal that putatively adaptive TE polymorphisms are rare in wild B. distachyon populations. After accounting for changes in past TE activity, we show that only a small proportion of TE polymorphisms evolved neutrally (\u003c 10%), while the vast majority of them are under moderate purifying selection regardless of their distance to genes. ConclusionsTE polymorphisms should not be ignored when conducting evolutionary studies, as they can be linked to adaptation. However, our study clearly shows that while they have a large potential to cause phenotypic variation in B. distachyon, they are not favored during evolution and adaptation over other types of mutations (such as point mutations) in this species.\nDiverse signatures of convergent evolution in cacti-associated yeasts\nAuthors: Goncalves, C.; Harrison, M.-C.; Steenwyk, J. L.; Opulente, D. A.; LaBella, A. L.; Wolters, J. F.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Hittinger, C. T.; Rokas, A.\nScore: 11.7, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557833\nMany distantly related organisms have convergently evolved traits and lifestyles that enable them to live in similar ecological environments. However, the extent of phenotypic convergence evolving through the same or distinct genetic trajectories remains an open question. Here, we leverage a comprehensive dataset of genomic and phenotypic data from 1,049 yeast species in the subphylum Saccharomycotina (Kingdom Fungi, Phylum Ascomycota) to explore signatures of convergent evolution in cactophilic yeasts, ecological specialists associated with cacti. We inferred that the ecological association of yeasts with cacti arose independently [~]17 times. Using machine-learning, we further found that cactophily can be predicted with 76% accuracy from functional genomic and phenotypic data. The most informative feature for predicting cactophily was thermotolerance, which is likely associated with duplication and altered evolutionary rates of genes impacting the cell envelope in several cactophilic lineages. We also identified horizontal gene transfer and duplication events of plant cell wall-degrading enzymes in distantly related cactophilic clades, suggesting that putatively adaptive traits evolved through disparate molecular mechanisms. Remarkably, multiple cactophilic lineages and their close relatives are emerging human opportunistic pathogens, suggesting that the cactophilic lifestyle--and perhaps more generally lifestyles favoring thermotolerance--may preadapt yeasts to cause human disease. This work underscores the potential of a multifaceted approach involving high throughput genomic and phenotypic data to shed light onto ecological adaptation and highlights how convergent evolution to wild environments could facilitate the transition to human pathogenicity.\nSperm length evolution in relation to body mass is shaped by multiple trade-offs in tetrapods\nAuthors: Kocillari, L.; Cattelan, S.; Rasotto, M. B.; Seno, F.; Maritan, A.; Pilastro, A.\nScore: 12.1, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557314\nSperm size is highly variable across species and is influenced by various factors including fertilization mode, female reproductive traits and sperm competition. Despite considerable efforts, many questions about sperm size variation remain open. Variation in body size may affect sperm size evolution through its influence on these factors, but the extent to which sperm size variation is linked to body mass remains elusive. In this study, we use the general theory of Pareto Optimality to investigate the relationship between sperm size and body mass across tetrapods. We find that tetrapods fall within a triangular-shaped Pareto front in the trait space of body mass and sperm length suggesting that the evolution of sperm size in relation to body size is shaped by trade-offs. We then explore the three main factors predicted to influence sperm size evolution, namely sperm competition, clutch size and genome size. Our results demonstrate that body mass optimally shapes sperm size evolution in tetrapods mainly through its association with sperm competition and clutch size. Finally, we show that the triangular-shaped Pareto front is maintained when tested separately within mammals, birds, endothermic species and internal fertilizers, suggesting that similar evolutionary trade-offs characterize the evolution of sperm size in relation to body size within taxonomic/phylogenetic and functional subgroups of tetrapods. This study provides insights into the evolutionary mechanisms driving interspecific sperm size variation and highlights the importance of considering multiple trade-offs in optimizing reproductive traits.\nFemales translate male mRNA transferred during mating\nAuthors: Matzkin, L. M.; Bono, J. M.; Pigage, H.; Allan, C. W.; Diaz, F.; McCoy, J.; Green, C.; Callan, J.; Delahunt, S.\nScore: 10.5, Published: 2023-09-23 DOI: 10.1101/2023.09.22.558997\nAlthough RNA is found in the seminal fluid of diverse organisms, it is unknown whether this RNA is functional within females. Here, we develop an experimental proteomic method called VESPA (Variant Enabled SILAC Proteomic Analysis) to test the hypothesis that Drosophila male seminal fluid RNA is translated by females. We find strong evidence for 67 male-derived, female-translated proteins (mdFTPs) in female lower reproductive tracts at six hours postmating, many with predicted functions relevant to reproduction. Gene knockout experiments indicate that genes coding for mdFTPs play diverse roles in postmating interactions, with effects on fertilization efficiency, and the formation and persistence of the insemination reaction mass, a trait hypothesized to be involved in sexual conflict. These findings advance our understanding of reproduction by revealing a novel mechanism of postmating molecular interactions between the sexes that strengthens and extends male influences on reproductive outcomes in previously unrecognized ways. Given the diverse species known to carry RNA in seminal fluid, this discovery has broad significance for understanding molecular mechanisms of cooperation and conflict during reproduction.\nDeep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution\nAuthors: Taylor, A. L.; Starr, T. N.\nScore: 70.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557279\nSubstitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear. We conducted deep mutational scans to measure the impacts of all single amino acid mutations and single-codon deletions in the spike receptor-binding domain (RBD) on ACE2-binding affinity and protein expression in the recent Omicron BQ.1.1 and XBB.1.5 variants, and we compared mutational patterns to earlier viral strains that we have previously profiled. As with previous RBD deep mutational scans, we find many mutations that are tolerated or even enhance binding to ACE2 receptor. The tolerance of sites to single-codon deletion largely conforms with tolerance to amino acid mutation. Though deletions in the RBD have not yet been seen in dominant lineages, we observe many tolerated deletions including at positions that exhibit indel variation across broader sarbecovirus evolution and in emerging SARS-CoV-2 variants of interest, most notably the well-tolerated {Delta}483 deletion in BA.2.86. The substitutions that distinguish recent viral variants have not induced as dramatic of epistatic perturbations as N501Y, but we identify ongoing epistatic drift in SARS-CoV-2 variants, including interaction between R493Q reversions and mutations at positions 453, 455, and 456, including mutations like F456L that define the newly emerging EG.5 lineage. Our results highlight ongoing drift in the effects of mutations due to epistasis, which may continue to direct SARS-CoV-2 evolution into new regions of sequence space.\nGenomic consequences of isolation and inbreeding in an island dingo population\nAuthors: Leon-Apodaca, A. V.; Kumar, M.; del Castillo, A.; Conroy, G. C.; Lamont, R. W.; Ogbourne, S.; Cairns, K. M.; Borburgh, L.; Behrendorff, L.; Subramanian, S.; Szpiech, Z. A.\nScore: 13.7, Published: 2023-09-20 DOI: 10.1101/2023.09.15.557950\nDingoes come from an ancient canid lineage that originated in East Asia around 8000-11,000 years BP. As Australias largest terrestrial predator, dingoes play an important ecological role. A small population exists on a world heritage listed offshore island, Kgari (formerly Fraser Island). While dingoes on Kgari are protected, dingoes on the mainland are only protected in certain regions, and, because of its controversial status in Australia, lethal control is commonly used for management. Concern regarding the persistence of dingoes on Kgari has risen due to their low genetic diversity and elevated inbreeding levels. However, whole-genome sequencing data is lacking from this population. Here, we include five new whole-genome sequences of Kgari dingoes. We analyze a total of 18 whole genome sequences of dingoes sampled from mainland Australia and Kgari to assess the genomic consequences of their demographic histories. Long (\u003e1 Mb) runs of homozygosity (ROH) -- indicators of inbreeding -- are elevated in all sampled dingoes. However, Kgari dingoes showed significantly higher levels of very long ROH (\u003e5 Mb), providing genomic evidence for small population size, isolation, inbreeding, and a strong founder effect. Our results suggest that, despite current levels of inbreeding, the Kgari population is purging strongly deleterious mutations, which, in the absence of further reductions in population size, may facilitate the persistence of small populations despite low genetic diversity and isolation. However, there is little to no purging of mildly deleterious alleles, which may have important long-term consequences, and should be considered by conservation and management programs.\nCoupling of 12 chromosomal inversions maintains a strong barrier to gene flow between ecotypes\nAuthors: Le Moan, A.; Stankowski, S.; Rafajlovic, M.; Martinez Ortega, O.; Faria, R.; Butlin, R.; Johannesson, K.\nScore: 6.8, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558209\nChromosomal rearrangements lead to the coupling of reproductive barriers, but whether and how they contribute to completion of speciation remains unclear. Marine snails of the genus Littorina repeatedly form hybrid zones between taxa segregating for inversion arrangements, providing opportunities to study this question. Here, we analysed two adjacent transects across hybrid zones between large and dwarf ecotypes of Littorina fabalis covering wave exposure gradients in a Swedish island. Applying whole-genome sequences we found 12 putative inversions reaching near differential fixation between the opposite ends of each transect, and being in strong linkage disequilibrium. These inversions cover 20% of the genome and carry 93% of divergent SNPs. Bimodal hybrid zones in both transects indicate that the two ecotypes of Littorina fabalis maintain their genetic and phenotypic integrity following contact due to strong coupling between inversion clines that strengthened the reproductive barrier. The bimodality resulting from the linked inversions extends into collinear parts of the genome, suggesting a genome-wide coupling. Demographic inference suggests that the coupling built up during a period of allopatry, and has been maintained for more than 1K generations after secondary contact. Overall, this study shows that the coupling of multiple chromosomal inversions contributes to strong reproductive isolation. Importantly, two of the inversions overlap with inverted genomic regions associated with ecotype differences in a closely-related species (L. saxatilis), suggesting the same regions, with similar structural variants, repeatedly contribute to ecotype evolution in distinct species.\n",
  "wordCount" : "2705",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558576">
        <p class="paperTitle">Self-assembling viral histones unravel early nucleosome evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Irwin, N. A. T.; Richards, T. A.</p>
        <p class="info">Score: 51.2, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558576' target='https://doi.org/10.1101/2023.09.20.558576'> 10.1101/2023.09.20.558576</a></p>
        <p class="abstract">Nucleosomes are a core-component of eukaryotic nuclei, forming the structural basis of chromatin and co-ordinating processes from gene expression to chromosome segregation. Composed of a DNA-protein complex consisting of the four individual histones, H2A, H2B, H3, and H4, the nucleosome and its associated functions were key innovations during eukaryotic evolution. However, functional constraints and the extinction of stem-eukaryotes have concealed how these dynamic systems evolved from simpler histone homologues in Archaea. Viral histones have also previously been identified and are thought to reflect an ancestral state as they often comprise multiple histone paralogues arranged within a single protein, termed histone repeats. Here, using viruses as an alternative source of variation, we expand the known diversity of histones and develop an empirical hypothesis for the origin of the nucleosome. Our analysis identified hundreds of histones with variable domain repeat configurations including histone singlets, doublets, triplets, and quadruplets, the latter comprising the four core histones arranged in series. Viral histone repeats consistently branch between Archaea and eukaryotes in phylogenetic trees and display intermediate functions, self-assembling into eukaryotic-like nucleosomes that stack into archaeal-like oligomers capable of impacting genomic activity and condensing DNA. The linkers conjoining the histone repeats also facilitate nucleosome formation and can promote the assembly of eukaryotic nucleosomes in the bacterium, Escherichia coli. Combining these data, we hypothesize that viral histone repeats represent molecular relics acquired by viruses from stem-eukaryotes during eukaryogenesis and suggest that nucleosome evolution may have proceeded through histone repeat intermediates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557618">
        <p class="paperTitle">Transposable element abundance subtly contributes to lower fitness in maize</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stitzer, M. C.; Khaipho-Burch, M. B.; Hudson, A. I.; Song, B.; Valdes-Franco, J. A.; Ramstein, G. P.; Feschotte, C.; Buckler, E. S.</p>
        <p class="info">Score: 25.8, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557618' target='https://doi.org/10.1101/2023.09.18.557618'> 10.1101/2023.09.18.557618</a></p>
        <p class="abstract">Transposable elements (TEs) have long been shown to have deleterious effects on the survival and reproduction of their host organism. As TEs are mobile DNA that jump to new positions, this deleterious cost can occur directly, by inserting into genes and regulatory sequences. Classical population genetic theory suggests copy-number dependent selection against TEs is necessary to prevent TEs from expanding so much they take over a genome. Such models have been difficult to interpret when applied to large genomes like maize, where there are hundreds of thousands of TE insertions that collectively make up 85% of the genome. Here, we use nearly 5000 inbred lines from maize mapping populations and a pan-genomic imputation approach to measure TE content. Segregating TE content gives rise to 100 Mb differences between individuals, and populations often show transgressive segregation in TE content. We use replicated phenotypes measured in hybrids across numerous years and environments to empirically measure the fitness costs of TEs. For an annual plant like maize, grain yield is not only a key agronomic phenotype, but also a direct measure of reproductive output. We find weak negative effects of TE accumulation on grain yield, nearing the limit of the efficacy of natural selection in maize. This results in a loss of one kernel ({approx}0.1% of average per-plant yield) for every additional 14 Mb of TE content. This deleterious load is enriched in TEs within 1 kilobase of genes and young TE insertions. Together, we provide rare empirical measurements of the fitness costs of TEs, and suggest that the TEs we see today in the genome have been filtered by selection against their deleterious consequences on maize fitness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557727">
        <p class="paperTitle">Gene regulatory network that shaped the evolution of larval sensory organ in Cnidaria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilbert, E.; Craggs, J.; Modepalli, V.</p>
        <p class="info">Score: 24.4, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557727' target='https://doi.org/10.1101/2023.09.14.557727'> 10.1101/2023.09.14.557727</a></p>
        <p class="abstract">Among non-bilaterian animals, a larval apical sensory organ with integrated neurons is only found in cnidarians. Within cnidarians, an apical organ with a ciliary tuft is mainly found in Actiniaria. Whether this apical tuft has evolved independently in Actiniaria or alternatively originated in the common ancestor of Cnidaria and Bilateria and was lost in specific groups is uncertain. We generated transcriptomes of the apical domain during the planula stage of four species representing three key groups of cnidarians: Aurelia aurita (Scyphozoa), Nematostella vectensis (Actiniaria), and Acropora millepora &amp; Acropora tenuis (Scleractinia). We showed that the canonical genes implicated in patterning the apical domain of Nematostella are largely absent in Aurelia, indicating that scyphozoans and anthozoans do not share apical organ homology. In contrast, the apical domain of the scleractinian planula shares gene expression pattern with Nematostella. By comparing the larval single-cell transcriptomes, we revealed the apical organ cell type of Scleractinia and confirmed its homology to Actiniaria. However, Fgfa2, a vital regulator of the regionalisation of the Nematostella apical organ, is absent in the scleractinian genome. Likewise, we found that FoxJ1 and 245 genes associated with cilia are exclusively expressed in the Nematostella apical domain, which is in line with the presence of ciliary apical tuft in Actiniaria and its absence in Scleractinia and Scyphozoa. Our findings suggest that the common ancestor of cnidarians and bilaterians lacked an apical organ with a ciliary tuft, and it could have evolved independently in several clades of cnidarians and bilaterians.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557873">
        <p class="paperTitle">The evolution of transposable elements in Brachypodium distachyon is governed by purifying selection, while neutral and adaptive processes play a minor role</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horvath, R.; Minadakis, N.; Bourgeois, Y. X. C.; Roulin, A.</p>
        <p class="info">Score: 13.2, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557873' target='https://doi.org/10.1101/2023.09.15.557873'> 10.1101/2023.09.15.557873</a></p>
        <p class="abstract">BackgroundUnderstanding how plants adapt to changing environments and the potential contribution of transposable elements (TEs) to this process is a key question in evolutionary genomics. While TEs have recently been put forward as active players in the context of adaptation, few studies have thoroughly investigated their precise role in plant evolution. Here we used the wild Mediterranean grass Brachypodium distachyon as a model species to identify and quantify the forces acting on TEs during the adaptation of this species to various conditions, across its entire geographic range.

ResultsUsing sequencing data from more than 320 natural B. distachyon accessions and a suite of population genomics approaches, we reveal that putatively adaptive TE polymorphisms are rare in wild B. distachyon populations. After accounting for changes in past TE activity, we show that only a small proportion of TE polymorphisms evolved neutrally (&lt; 10%), while the vast majority of them are under moderate purifying selection regardless of their distance to genes.

ConclusionsTE polymorphisms should not be ignored when conducting evolutionary studies, as they can be linked to adaptation. However, our study clearly shows that while they have a large potential to cause phenotypic variation in B. distachyon, they are not favored during evolution and adaptation over other types of mutations (such as point mutations) in this species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557833">
        <p class="paperTitle">Diverse signatures of convergent evolution in cacti-associated yeasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goncalves, C.; Harrison, M.-C.; Steenwyk, J. L.; Opulente, D. A.; LaBella, A. L.; Wolters, J. F.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Hittinger, C. T.; Rokas, A.</p>
        <p class="info">Score: 11.7, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557833' target='https://doi.org/10.1101/2023.09.14.557833'> 10.1101/2023.09.14.557833</a></p>
        <p class="abstract">Many distantly related organisms have convergently evolved traits and lifestyles that enable them to live in similar ecological environments. However, the extent of phenotypic convergence evolving through the same or distinct genetic trajectories remains an open question. Here, we leverage a comprehensive dataset of genomic and phenotypic data from 1,049 yeast species in the subphylum Saccharomycotina (Kingdom Fungi, Phylum Ascomycota) to explore signatures of convergent evolution in cactophilic yeasts, ecological specialists associated with cacti. We inferred that the ecological association of yeasts with cacti arose independently [~]17 times. Using machine-learning, we further found that cactophily can be predicted with 76% accuracy from functional genomic and phenotypic data. The most informative feature for predicting cactophily was thermotolerance, which is likely associated with duplication and altered evolutionary rates of genes impacting the cell envelope in several cactophilic lineages. We also identified horizontal gene transfer and duplication events of plant cell wall-degrading enzymes in distantly related cactophilic clades, suggesting that putatively adaptive traits evolved through disparate molecular mechanisms. Remarkably, multiple cactophilic lineages and their close relatives are emerging human opportunistic pathogens, suggesting that the cactophilic lifestyle--and perhaps more generally lifestyles favoring thermotolerance--may preadapt yeasts to cause human disease. This work underscores the potential of a multifaceted approach involving high throughput genomic and phenotypic data to shed light onto ecological adaptation and highlights how convergent evolution to wild environments could facilitate the transition to human pathogenicity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557314" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557314">
        <p class="paperTitle">Sperm length evolution in relation to body mass is shaped by multiple trade-offs in tetrapods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kocillari, L.; Cattelan, S.; Rasotto, M. B.; Seno, F.; Maritan, A.; Pilastro, A.</p>
        <p class="info">Score: 12.1, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557314' target='https://doi.org/10.1101/2023.09.12.557314'> 10.1101/2023.09.12.557314</a></p>
        <p class="abstract">Sperm size is highly variable across species and is influenced by various factors including fertilization mode, female reproductive traits and sperm competition. Despite considerable efforts, many questions about sperm size variation remain open. Variation in body size may affect sperm size evolution through its influence on these factors, but the extent to which sperm size variation is linked to body mass remains elusive. In this study, we use the general theory of Pareto Optimality to investigate the relationship between sperm size and body mass across tetrapods. We find that tetrapods fall within a triangular-shaped Pareto front in the trait space of body mass and sperm length suggesting that the evolution of sperm size in relation to body size is shaped by trade-offs. We then explore the three main factors predicted to influence sperm size evolution, namely sperm competition, clutch size and genome size. Our results demonstrate that body mass optimally shapes sperm size evolution in tetrapods mainly through its association with sperm competition and clutch size. Finally, we show that the triangular-shaped Pareto front is maintained when tested separately within mammals, birds, endothermic species and internal fertilizers, suggesting that similar evolutionary trade-offs characterize the evolution of sperm size in relation to body size within taxonomic/phylogenetic and functional subgroups of tetrapods. This study provides insights into the evolutionary mechanisms driving interspecific sperm size variation and highlights the importance of considering multiple trade-offs in optimizing reproductive traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558997">
        <p class="paperTitle">Females translate male mRNA transferred during mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matzkin, L. M.; Bono, J. M.; Pigage, H.; Allan, C. W.; Diaz, F.; McCoy, J.; Green, C.; Callan, J.; Delahunt, S.</p>
        <p class="info">Score: 10.5, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558997' target='https://doi.org/10.1101/2023.09.22.558997'> 10.1101/2023.09.22.558997</a></p>
        <p class="abstract">Although RNA is found in the seminal fluid of diverse organisms, it is unknown whether this RNA is functional within females. Here, we develop an experimental proteomic method called VESPA (Variant Enabled SILAC Proteomic Analysis) to test the hypothesis that Drosophila male seminal fluid RNA is translated by females. We find strong evidence for 67 male-derived, female-translated proteins (mdFTPs) in female lower reproductive tracts at six hours postmating, many with predicted functions relevant to reproduction. Gene knockout experiments indicate that genes coding for mdFTPs play diverse roles in postmating interactions, with effects on fertilization efficiency, and the formation and persistence of the insemination reaction mass, a trait hypothesized to be involved in sexual conflict. These findings advance our understanding of reproduction by revealing a novel mechanism of postmating molecular interactions between the sexes that strengthens and extends male influences on reproductive outcomes in previously unrecognized ways. Given the diverse species known to carry RNA in seminal fluid, this discovery has broad significance for understanding molecular mechanisms of cooperation and conflict during reproduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557279">
        <p class="paperTitle">Deep mutational scans of XBB.1.5 and BQ.1.1 reveal ongoing epistatic drift during SARS-CoV-2 evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, A. L.; Starr, T. N.</p>
        <p class="info">Score: 70.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557279' target='https://doi.org/10.1101/2023.09.11.557279'> 10.1101/2023.09.11.557279</a></p>
        <p class="abstract">Substitutions that fix between SARS-CoV-2 variants can transform the mutational landscape of future evolution via epistasis. For example, large epistatic shifts in mutational effects caused by N501Y underlied the original emergence of Omicron variants, but whether such large epistatic saltations continue to define ongoing SARS-CoV-2 evolution remains unclear. We conducted deep mutational scans to measure the impacts of all single amino acid mutations and single-codon deletions in the spike receptor-binding domain (RBD) on ACE2-binding affinity and protein expression in the recent Omicron BQ.1.1 and XBB.1.5 variants, and we compared mutational patterns to earlier viral strains that we have previously profiled. As with previous RBD deep mutational scans, we find many mutations that are tolerated or even enhance binding to ACE2 receptor. The tolerance of sites to single-codon deletion largely conforms with tolerance to amino acid mutation. Though deletions in the RBD have not yet been seen in dominant lineages, we observe many tolerated deletions including at positions that exhibit indel variation across broader sarbecovirus evolution and in emerging SARS-CoV-2 variants of interest, most notably the well-tolerated {Delta}483 deletion in BA.2.86. The substitutions that distinguish recent viral variants have not induced as dramatic of epistatic perturbations as N501Y, but we identify ongoing epistatic drift in SARS-CoV-2 variants, including interaction between R493Q reversions and mutations at positions 453, 455, and 456, including mutations like F456L that define the newly emerging EG.5 lineage. Our results highlight ongoing drift in the effects of mutations due to epistasis, which may continue to direct SARS-CoV-2 evolution into new regions of sequence space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557950">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557950" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557950">
        <p class="paperTitle">Genomic consequences of isolation and inbreeding in an island dingo population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557950" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557950" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leon-Apodaca, A. V.; Kumar, M.; del Castillo, A.; Conroy, G. C.; Lamont, R. W.; Ogbourne, S.; Cairns, K. M.; Borburgh, L.; Behrendorff, L.; Subramanian, S.; Szpiech, Z. A.</p>
        <p class="info">Score: 13.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557950' target='https://doi.org/10.1101/2023.09.15.557950'> 10.1101/2023.09.15.557950</a></p>
        <p class="abstract">Dingoes come from an ancient canid lineage that originated in East Asia around 8000-11,000 years BP. As Australias largest terrestrial predator, dingoes play an important ecological role. A small population exists on a world heritage listed offshore island, Kgari (formerly Fraser Island). While dingoes on Kgari are protected, dingoes on the mainland are only protected in certain regions, and, because of its controversial status in Australia, lethal control is commonly used for management. Concern regarding the persistence of dingoes on Kgari has risen due to their low genetic diversity and elevated inbreeding levels. However, whole-genome sequencing data is lacking from this population. Here, we include five new whole-genome sequences of Kgari dingoes. We analyze a total of 18 whole genome sequences of dingoes sampled from mainland Australia and Kgari to assess the genomic consequences of their demographic histories. Long (&gt;1 Mb) runs of homozygosity (ROH) -- indicators of inbreeding -- are elevated in all sampled dingoes. However, Kgari dingoes showed significantly higher levels of very long ROH (&gt;5 Mb), providing genomic evidence for small population size, isolation, inbreeding, and a strong founder effect. Our results suggest that, despite current levels of inbreeding, the Kgari population is purging strongly deleterious mutations, which, in the absence of further reductions in population size, may facilitate the persistence of small populations despite low genetic diversity and isolation. However, there is little to no purging of mildly deleterious alleles, which may have important long-term consequences, and should be considered by conservation and management programs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558209">
        <p class="paperTitle">Coupling of 12 chromosomal inversions maintains a strong barrier to gene flow between ecotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le Moan, A.; Stankowski, S.; Rafajlovic, M.; Martinez Ortega, O.; Faria, R.; Butlin, R.; Johannesson, K.</p>
        <p class="info">Score: 6.8, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558209' target='https://doi.org/10.1101/2023.09.18.558209'> 10.1101/2023.09.18.558209</a></p>
        <p class="abstract">Chromosomal rearrangements lead to the coupling of reproductive barriers, but whether and how they contribute to completion of speciation remains unclear. Marine snails of the genus Littorina repeatedly form hybrid zones between taxa segregating for inversion arrangements, providing opportunities to study this question. Here, we analysed two adjacent transects across hybrid zones between large and dwarf ecotypes of Littorina fabalis covering wave exposure gradients in a Swedish island. Applying whole-genome sequences we found 12 putative inversions reaching near differential fixation between the opposite ends of each transect, and being in strong linkage disequilibrium. These inversions cover 20% of the genome and carry 93% of divergent SNPs. Bimodal hybrid zones in both transects indicate that the two ecotypes of Littorina fabalis maintain their genetic and phenotypic integrity following contact due to strong coupling between inversion clines that strengthened the reproductive barrier. The bimodality resulting from the linked inversions extends into collinear parts of the genome, suggesting a genome-wide coupling. Demographic inference suggests that the coupling built up during a period of allopatry, and has been maintained for more than 1K generations after secondary contact. Overall, this study shows that the coupling of multiple chromosomal inversions contributes to strong reproductive isolation. Importantly, two of the inversions overlap with inverted genomic regions associated with ecotype differences in a closely-related species (L. saxatilis), suggesting the same regions, with similar structural variants, repeatedly contribute to ecotype evolution in distinct species.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
