<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reproductive innovation enabled radiation in the deep sea during an ecological crisis
Authors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.
Score: 96.3, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380
Major ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Reproductive innovation enabled radiation in the deep sea during an ecological crisis
Authors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.
Score: 96.3, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380
Major ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Reproductive innovation enabled radiation in the deep sea during an ecological crisis
Authors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.
Score: 96.3, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380
Major ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Reproductive innovation enabled radiation in the deep sea during an ecological crisis\nAuthors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.\nScore: 96.3, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380\nMajor ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate.",
  "keywords": [
    
  ],
  "articleBody": " Reproductive innovation enabled radiation in the deep sea during an ecological crisis\nAuthors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.\nScore: 96.3, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575380\nMajor ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate. A phylogenomic reconstruction of the evolutionary history of anglerfishes provides a strong inference for the convergent evolution of permanently-fusing deep-sea anglerfishes and their degenerate immune genes. Our results support that sexual parasitism was enabled by the degeneration of adaptive immunity and ancestral sexual size dimorphism. The combination of these traits facilitated the transition of pelagic anglerfishes into novel ecologies available in the deep open oceans after evolving from benthic ancestors. These results show how seemingly unrelated physiological and reproductive traits interact synergistically to drive evolutionary radiation in novel environments.\nA 39.8kb flavi-like virus uses a novel strategy for overcoming the RNA virus error threshold\nAuthors: Petrone, M. E.; Grove, J.; Mifsud, J. C. O.; Parry, R. H.; Marzinelli, E. M.; Holmes, E. C.\nScore: 42.1, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574764\nIt is commonly held that there is a fundamental relationship between genome size and error rate, manifest as a notional \"error threshold\" that sets an upper limit on genome sizes. The genome sizes of RNA viruses, which have intrinsically high mutation rates due to a lack of mechanisms for error correction, must therefore be small to avoid accumulating an excessive number of deleterious mutations that will ultimately lead to population extinction. The proposed exceptions to this evolutionary rule are RNA viruses from the order Nidovirales (such as coronaviruses) that encode an error correcting exonuclease, enabling them to reach genome lengths greater than 40kb. The recent discovery of large genome flavi-like viruses (Flaviviridae), which comprise genomes up to 27kb in length yet seemingly do not encode exonuclease domains, has led to the proposal that a proofreading mechanism is required to facilitate the expansion of RNA virus genomes above 30kb. Herein, we describe a 39.8kb flavi-like virus identified in a Haliclona sponge metatranscriptome that does not encode an exonuclease. Structural analysis revealed that this virus may have instead captured bacterial domains associated with nucleic acid metabolism that have not been previously found in RNA viruses. Phylogenetic analysis placed this virus as a divergent pesti-like lineage, such that we have provisionally termed it Maximus pesti-like virus. This virus represents the first instance of a flavi-like virus achieving a genome size comparable to that of the Nidovirales and demonstrates that RNA viruses have evolved multiple solutions to overcome the error threshold.\nDNA methylation enables recurrent endogenization of giant viruses in an animal relative\nAuthors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.\nScore: 55.8, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574619\n5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification. Here we show that the protist closely related to animals Amoebidium appalachense features both transposon and gene body methylation, a pattern reminiscent of invertebrates and plants. Unexpectedly, large hypermethylated regions of the Amoebidium genome derive from viral insertions, including hundreds of endogenized giant viruses contributing 14% of the encoded genes, to an extent never reported before in any eukaryotic genome. Using a combination of inhibitors and functional genomic assays, we demonstrate that 5mC silences these giant virus insertions. Moreover, alternative Amoebidium isolates show polymorphic giant virus insertions, highlighting a dynamic process of infection, endogenization and purging. Our results indicate that 5mC is critical for the controlled co-existence of newly acquired viral DNA into eukaryotic genomes, making Amoebidium a unique model to understand the hybrid origins of eukaryotic genomes.\nAncestral Sequence Reconstruction as a tool to detect and study de novo gene emergence\nAuthors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.\nScore: 25.5, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573862\nNew protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (\u003c1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute \"easy\" cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.\nPervasive selective sweeps across human gut microbiomes\nAuthors: Garud, N.; Wolff, R.\nScore: 16.8, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573162\nThe human gut microbiome is composed of a highly diverse consortia of species which are continually evolving within and across hosts. The ability to identify adaptations common to many host gut microbiomes would not only reveal shared selection pressures across hosts, but also key drivers of functional differentiation of the microbiome that may affect community structure and host traits. However, to date there has not been a systematic scan for adaptations that have spread across host microbiomes. Here, we develop a novel selection scan statistic, named the integrated linkage disequilibrium score (iLDS), that can detect the spread of adaptive haplotypes across host microbiomes via migration and horizontal gene transfer. Specifically, iLDS leverages signals of hitchhiking of deleterious variants with the beneficial variant, a common feature of adaptive evolution. We find that iLDS is capable of detecting simulated and known cases of selection, and moreover is robust to potential confounders that can also elevate LD. Application of the statistic to [~]20 common commensal gut species from a large cohort of healthy, Western adults reveals pervasive spread of selected alleles across human microbiomes mediated by horizontal gene transfer. Among the candidate selective sweeps recovered by iLDS is an enrichment for genes involved in the metabolism of maltodextrin, a synthetic starch that has recently become a widespread component of Western diets. In summary, we demonstrate that selective sweeps across host microbiomes are a common feature of the evolution of the human gut microbiome.\nEvidence for compensatory evolution within pleiotropic regulatory elements\nAuthors: Kliesmete, Z.; Orchard, P.; Lee, V. Y. K.; Geuder, J.; Krauss, S. M.; Ohnuki, M.; Jocher, J.; Vieth, B.; Enard, W.; Hellmann, I.\nScore: 14.5, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575014\nPleiotropy, measured as expression breadth across tissues, is one of the best predictors for protein sequence and expression conservation. In this study, we investigated its effect on the evolution of cis-regulatory elements (CREs). To this end, we carefully reanalyzed the Epigenomics Roadmap data for nine fetal tissues, assigning a measure of pleiotropic degree to nearly half a million CREs. To assess the functional conservation of CREs, we generated ATAC-seq and RNA-seq data from humans and macaques. We found that more pleiotropic CREs exhibit greater conservation in accessibility, and the mRNA expression levels of the associated genes are more conserved. This trend of higher conservation for higher degrees of pleiotropy persists when analyzing the transcription factor binding repertoire. In contrast, simple DNA sequence conservation of orthologous sites between species tends to be even lower for pleiotropic CREs than for species-specific CREs. Combining various lines of evidence, we suggest that the lack of sequence conservation for functionally conserved pleiotropic elements is due to compensatory evolution within these large pleiotropic CREs. Furthermore, for less pleiotropic CREs, we find an indication of compensation across CREs. This suggests that pleiotropy is also a good predictor for the functional conservation of CREs, but this is not reflected in the sequence conservation for pleiotropic CREs.\nRecurrent Evolutionary Innovations in Rodent and Primate Schlafen Genes\nAuthors: Mordier, J.; Fraisse, M.; Cohen-Tannoudji, M.; Molaro, A.\nScore: 13.7, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575368\nSCHLAFEN proteins are a large family of RNase-related enzymes carrying essential immune and developmental functions. Despite these important roles, Schlafen genes display varying degrees of evolutionary conservation in mammals. While this appears to influence their molecular activities, a detailed understanding of these evolutionary innovations is still lacking. Here, we used in depth phylogenomic approaches to characterize the evolutionary trajectories and selective forces shaping mammalian Schlafen genes. We traced lineage-specific Schlafen amplifications and found that recent duplicates evolved under distinct selective forces, supporting repeated sub-functionalization cycles. Codon-level natural selection analyses in primates and rodents, identified recurrent positive selection over Schlafen protein domains engaged in viral interactions. Combining crystal structures with machine learning predictions, we discovered a novel class of rapidly evolving residues enriched at the contact interface of SCHLAFEN protein dimers. Our results suggest that inter Schlafen compatibilities are under strong selective pressures and are likely to impact their molecular functions. We posit that cycles of genetic conflicts with pathogens and between paralogs drove Schlafens recurrent evolutionary innovations in mammals.\nNortheast India: Genetic Inconsistency across Ethnicity and Geography\nAuthors: Bankura, B.; Basak, B.; Singh, P. P.; Vanlalruata, A.; Chatterjee, A.; Ghosh, S.; Tamang, R.; Hazarika, M.; Chaubey, G.; Das, M.\nScore: 7.0, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574778\nThe genetic landscape of South Asia is intriguing. In a vast geographical region like India, mostly genes follow geography, whereas in a smaller geographical region e.g., Nepal, the genetic landscape is more akin to languages. Moreover, in neighboring regions like East and Southeast Asia, language is the major denominator of the genes. However, these are not just the two alternatives, in many geographical pockets, ethnicity plays a stronger role than either of these. Being a crossroads of South and Southeast Asia, the Himalayan region is mostly populated by people speaking the Trans-Himalayan language. Archaeological and anthropological studies suggest migration and cultural diffusion in this region with the Tibetan plateau in the North. To understand this complex pattern, we have performed a fine-grained genetic analysis of the major Mizo population and its various clans living in the Himalayan geography. We have investigated 110 individuals belonging to seven different clans of Mizo people for a hundred thousand autosomal markers. We have used various statistical methods and tested the role of ethnicity and geography in shaping the genetic landscape of the Himalayan region. In contrast to the East and Southeast Asian genetic landscape, our results suggested that the fine-scaled genetic structure of Northeast India is much more complex and a mixture of interplay between languages, ethnicity, and geography. Allelic frequency-based analyses indicated that a novel Trans-Himalayan ancestry unite all the populations of this region.\nA new story of four Hexapoda classes: Protura as the sister to all other hexapods\nAuthors: Du, S.; Tihelka, E.; Yu, D.; Chen, W.-J.; Bu, Y.; Cai, C.; Engel, M. S.; Luan, Y.-X.; Zhang, F.\nScore: 7.6, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574592\nInsects represent the most diverse animal group, yet previous phylogenetic analyses based on the morphological and molecular data have failed to agree on the evolutionary relationships of early insects and their six-legged relatives (together constituting the clade Hexapoda). In particular, the phylogenetic positions of the three early-diverging hexapod groups, the coneheads (Protura), springtails (Collembola), and two-pronged bristletails (Diplura), have been debated for over a century, with alternative topologies implying drastically different scenarios of the evolution of the insect body plan and hexapod terrestrialisation. We addressed this issue by sampling of all hexapod orders, and experimented with a broad range of across-site compositional heterogeneous models designed to tackle ancient divergences. Our analyses support Protura as the earliest-diverging hexapod lineage (Protura-sister) and Collembola as a sister group to the Diplura, a clade we refer to as Antennomusculata characterised by the shared possession of internal muscles in the antennal flagellum. The universally recognized Ellipura hypothesis is recovered under the site-homogenous LG model. Our cross-validation analysis shows that the CAT-GTR model that recovers Protura-sister fits significantly better than homogenous model. Furthermore, as a very unusual group, Protura as the first diverging lineage of hexapods is also supported by other lines of evidence, such as mitogenomics, comparative embryology, and sperm morphology. The backbone phylogeny of hexapods recovered in this study will facilitate the exploration of the underpinnings of hexapod terrestrialisation and mega-diversity.\nOrigin and evolutionary trajectories of brown algal sex chromosomes\nAuthors: Barrera-Redondo, J.; Lipinska, A. P.; Liu, P.; Dinatale, E.; Cossard, G.; Bogaert, K.; Hoshino, M.; Avia, K.; Avdievich, E.; Leiria, G.; Liesner, D.; Luthringer, R.; Godfroy, O.; Heesch, S.; Nehr, Z.; Peters, A. F.; Hoarau, G.; Pearson, G. A.; Aury, J.-M.; Wincker, P.; Denoeud, F.; Cock, J. M.; Haas, F. B.; Coelho, S. M.\nScore: 2.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575685\nSex chromosomes fall into three classes: XX/XY, ZW/ZZ and U/V systems. The rise, evolution and demise of U/V systems have remained enigmatic to date. Here, we analyze genomes spanning the entire brown algal phylogeny to decipher their sex-determination evolutionary history. The birth of U/V sex chromosomes evolved more than 250 million years ago, when a pivotal male-determinant located in a discrete region in proto-U and proto-V chromosomes ceased recombining. Over time, nested inversions led to step-wise expansions, accompanying increasing morphological complexity and sexual differentiation of brown seaweeds. Unlike XX/XY and ZW/ZZ, U/V evolve mainly by gene gain, showing minimal degeneration. They are structurally dynamic, and act as genomic 'cradles' fostering the birth of new genes. Our analyses show that hermaphroditism arose from ancestral males that acquired U-specific genes by ectopic recombination, and that in the transition from a U/V to an XX/XY system, V-specific genes moved down the genetic hierarchy of sex determination. Both events lead to the demise of U and V and erosion of their specific genomic characteristics. Taken together, our findings offer a comprehensive model of U/V sex chromosome evolution.\n",
  "wordCount" : "2545",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575380">
        <p class="paperTitle">Reproductive innovation enabled radiation in the deep sea during an ecological crisis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brownstein, C.; Zapfe, K.; Lott, S.; Harrington, R. C.; Ghezelayagh, A.; Dornburg, A.; Near, T. J.</p>
        <p class="info">Score: 96.3, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575380' target='https://doi.org/10.1101/2024.01.12.575380'> 10.1101/2024.01.12.575380</a></p>
        <p class="abstract">Major ecological transitions are thought to fuel evolutionary radiations, but whether they are contingent on the evolution of certain traits is unclear. We show that the rapid ecological transition of anglerfishes into pelagic habitats during a period of major global warming coincided with the origins of sexual parasitism, in which male anglerfishes temporarily attach or permanently fuse to females to mate. A phylogenomic reconstruction of the evolutionary history of anglerfishes provides a strong inference for the convergent evolution of permanently-fusing deep-sea anglerfishes and their degenerate immune genes. Our results support that sexual parasitism was enabled by the degeneration of adaptive immunity and ancestral sexual size dimorphism. The combination of these traits facilitated the transition of pelagic anglerfishes into novel ecologies available in the deep open oceans after evolving from benthic ancestors. These results show how seemingly unrelated physiological and reproductive traits interact synergistically to drive evolutionary radiation in novel environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574764" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574764">
        <p class="paperTitle">A 39.8kb flavi-like virus uses a novel strategy for overcoming the RNA virus error threshold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petrone, M. E.; Grove, J.; Mifsud, J. C. O.; Parry, R. H.; Marzinelli, E. M.; Holmes, E. C.</p>
        <p class="info">Score: 42.1, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574764' target='https://doi.org/10.1101/2024.01.08.574764'> 10.1101/2024.01.08.574764</a></p>
        <p class="abstract">It is commonly held that there is a fundamental relationship between genome size and error rate, manifest as a notional &#34;error threshold&#34; that sets an upper limit on genome sizes. The genome sizes of RNA viruses, which have intrinsically high mutation rates due to a lack of mechanisms for error correction, must therefore be small to avoid accumulating an excessive number of deleterious mutations that will ultimately lead to population extinction. The proposed exceptions to this evolutionary rule are RNA viruses from the order Nidovirales (such as coronaviruses) that encode an error correcting exonuclease, enabling them to reach genome lengths greater than 40kb. The recent discovery of large genome flavi-like viruses (Flaviviridae), which comprise genomes up to 27kb in length yet seemingly do not encode exonuclease domains, has led to the proposal that a proofreading mechanism is required to facilitate the expansion of RNA virus genomes above 30kb. Herein, we describe a 39.8kb flavi-like virus identified in a Haliclona sponge metatranscriptome that does not encode an exonuclease. Structural analysis revealed that this virus may have instead captured bacterial domains associated with nucleic acid metabolism that have not been previously found in RNA viruses. Phylogenetic analysis placed this virus as a divergent pesti-like lineage, such that we have provisionally termed it Maximus pesti-like virus. This virus represents the first instance of a flavi-like virus achieving a genome size comparable to that of the Nidovirales and demonstrates that RNA viruses have evolved multiple solutions to overcome the error threshold.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574619" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574619">
        <p class="paperTitle">DNA methylation enables recurrent endogenization of giant viruses in an animal relative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarre, L. A.; Kim, I. V.; Ovchinnikov, V.; Olivetta, M.; Suga, H.; Dudin, O.; Sebe-Pedros, A.; de Mendoza, A.</p>
        <p class="info">Score: 55.8, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574619' target='https://doi.org/10.1101/2024.01.08.574619'> 10.1101/2024.01.08.574619</a></p>
        <p class="abstract">5-methylcytosine (5mC) is a widespread silencing mechanism that controls genomic parasites. However, in many eukaryotes 5mC has gained complex roles in gene regulation beyond parasite control. Animals are a quintessential case for 5mC evolution, as they show widespread variability across lineages, ranging from gene regulation and transposable element control to loss of this base modification. Here we show that the protist closely related to animals Amoebidium appalachense features both transposon and gene body methylation, a pattern reminiscent of invertebrates and plants. Unexpectedly, large hypermethylated regions of the Amoebidium genome derive from viral insertions, including hundreds of endogenized giant viruses contributing 14% of the encoded genes, to an extent never reported before in any eukaryotic genome. Using a combination of inhibitors and functional genomic assays, we demonstrate that 5mC silences these giant virus insertions. Moreover, alternative Amoebidium isolates show polymorphic giant virus insertions, highlighting a dynamic process of infection, endogenization and purging. Our results indicate that 5mC is critical for the controlled co-existence of newly acquired viral DNA into eukaryotic genomes, making Amoebidium a unique model to understand the hybrid origins of eukaryotic genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573862" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573862">
        <p class="paperTitle">Ancestral Sequence Reconstruction as a tool to detect and study de novo gene emergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.</p>
        <p class="info">Score: 25.5, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573862' target='https://doi.org/10.1101/2024.01.02.573862'> 10.1101/2024.01.02.573862</a></p>
        <p class="abstract">New protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (&lt;1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute &#34;easy&#34; cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573162">
        <p class="paperTitle">Pervasive selective sweeps across human gut microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garud, N.; Wolff, R.</p>
        <p class="info">Score: 16.8, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573162' target='https://doi.org/10.1101/2023.12.22.573162'> 10.1101/2023.12.22.573162</a></p>
        <p class="abstract">The human gut microbiome is composed of a highly diverse consortia of species which are continually evolving within and across hosts. The ability to identify adaptations common to many host gut microbiomes would not only reveal shared selection pressures across hosts, but also key drivers of functional differentiation of the microbiome that may affect community structure and host traits. However, to date there has not been a systematic scan for adaptations that have spread across host microbiomes. Here, we develop a novel selection scan statistic, named the integrated linkage disequilibrium score (iLDS), that can detect the spread of adaptive haplotypes across host microbiomes via migration and horizontal gene transfer. Specifically, iLDS leverages signals of hitchhiking of deleterious variants with the beneficial variant, a common feature of adaptive evolution. We find that iLDS is capable of detecting simulated and known cases of selection, and moreover is robust to potential confounders that can also elevate LD. Application of the statistic to [~]20 common commensal gut species from a large cohort of healthy, Western adults reveals pervasive spread of selected alleles across human microbiomes mediated by horizontal gene transfer. Among the candidate selective sweeps recovered by iLDS is an enrichment for genes involved in the metabolism of maltodextrin, a synthetic starch that has recently become a widespread component of Western diets. In summary, we demonstrate that selective sweeps across host microbiomes are a common feature of the evolution of the human gut microbiome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575014" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575014">
        <p class="paperTitle">Evidence for compensatory evolution within pleiotropic regulatory elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kliesmete, Z.; Orchard, P.; Lee, V. Y. K.; Geuder, J.; Krauss, S. M.; Ohnuki, M.; Jocher, J.; Vieth, B.; Enard, W.; Hellmann, I.</p>
        <p class="info">Score: 14.5, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575014' target='https://doi.org/10.1101/2024.01.10.575014'> 10.1101/2024.01.10.575014</a></p>
        <p class="abstract">Pleiotropy, measured as expression breadth across tissues, is one of the best predictors for protein sequence and expression conservation. In this study, we investigated its effect on the evolution of cis-regulatory elements (CREs). To this end, we carefully reanalyzed the Epigenomics Roadmap data for nine fetal tissues, assigning a measure of pleiotropic degree to nearly half a million CREs. To assess the functional conservation of CREs, we generated ATAC-seq and RNA-seq data from humans and macaques. We found that more pleiotropic CREs exhibit greater conservation in accessibility, and the mRNA expression levels of the associated genes are more conserved. This trend of higher conservation for higher degrees of pleiotropy persists when analyzing the transcription factor binding repertoire. In contrast, simple DNA sequence conservation of orthologous sites between species tends to be even lower for pleiotropic CREs than for species-specific CREs. Combining various lines of evidence, we suggest that the lack of sequence conservation for functionally conserved pleiotropic elements is due to compensatory evolution within these large pleiotropic CREs. Furthermore, for less pleiotropic CREs, we find an indication of compensation across CREs. This suggests that pleiotropy is also a good predictor for the functional conservation of CREs, but this is not reflected in the sequence conservation for pleiotropic CREs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575368" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575368">
        <p class="paperTitle">Recurrent Evolutionary Innovations in Rodent and Primate Schlafen Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mordier, J.; Fraisse, M.; Cohen-Tannoudji, M.; Molaro, A.</p>
        <p class="info">Score: 13.7, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575368' target='https://doi.org/10.1101/2024.01.12.575368'> 10.1101/2024.01.12.575368</a></p>
        <p class="abstract">SCHLAFEN proteins are a large family of RNase-related enzymes carrying essential immune and developmental functions. Despite these important roles, Schlafen genes display varying degrees of evolutionary conservation in mammals. While this appears to influence their molecular activities, a detailed understanding of these evolutionary innovations is still lacking. Here, we used in depth phylogenomic approaches to characterize the evolutionary trajectories and selective forces shaping mammalian Schlafen genes. We traced lineage-specific Schlafen amplifications and found that recent duplicates evolved under distinct selective forces, supporting repeated sub-functionalization cycles. Codon-level natural selection analyses in primates and rodents, identified recurrent positive selection over Schlafen protein domains engaged in viral interactions. Combining crystal structures with machine learning predictions, we discovered a novel class of rapidly evolving residues enriched at the contact interface of SCHLAFEN protein dimers. Our results suggest that inter Schlafen compatibilities are under strong selective pressures and are likely to impact their molecular functions. We posit that cycles of genetic conflicts with pathogens and between paralogs drove Schlafens recurrent evolutionary innovations in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574778" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574778">
        <p class="paperTitle">Northeast India: Genetic Inconsistency across Ethnicity and Geography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bankura, B.; Basak, B.; Singh, P. P.; Vanlalruata, A.; Chatterjee, A.; Ghosh, S.; Tamang, R.; Hazarika, M.; Chaubey, G.; Das, M.</p>
        <p class="info">Score: 7.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574778' target='https://doi.org/10.1101/2024.01.09.574778'> 10.1101/2024.01.09.574778</a></p>
        <p class="abstract">The genetic landscape of South Asia is intriguing. In a vast geographical region like India, mostly genes follow geography, whereas in a smaller geographical region e.g., Nepal, the genetic landscape is more akin to languages. Moreover, in neighboring regions like East and Southeast Asia, language is the major denominator of the genes. However, these are not just the two alternatives, in many geographical pockets, ethnicity plays a stronger role than either of these. Being a crossroads of South and Southeast Asia, the Himalayan region is mostly populated by people speaking the Trans-Himalayan language. Archaeological and anthropological studies suggest migration and cultural diffusion in this region with the Tibetan plateau in the North. To understand this complex pattern, we have performed a fine-grained genetic analysis of the major Mizo population and its various clans living in the Himalayan geography. We have investigated 110 individuals belonging to seven different clans of Mizo people for a hundred thousand autosomal markers. We have used various statistical methods and tested the role of ethnicity and geography in shaping the genetic landscape of the Himalayan region. In contrast to the East and Southeast Asian genetic landscape, our results suggested that the fine-scaled genetic structure of Northeast India is much more complex and a mixture of interplay between languages, ethnicity, and geography. Allelic frequency-based analyses indicated that a novel Trans-Himalayan ancestry unite all the populations of this region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574592" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574592">
        <p class="paperTitle">A new story of four Hexapoda classes: Protura as the sister to all other hexapods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, S.; Tihelka, E.; Yu, D.; Chen, W.-J.; Bu, Y.; Cai, C.; Engel, M. S.; Luan, Y.-X.; Zhang, F.</p>
        <p class="info">Score: 7.6, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574592' target='https://doi.org/10.1101/2024.01.08.574592'> 10.1101/2024.01.08.574592</a></p>
        <p class="abstract">Insects represent the most diverse animal group, yet previous phylogenetic analyses based on the morphological and molecular data have failed to agree on the evolutionary relationships of early insects and their six-legged relatives (together constituting the clade Hexapoda). In particular, the phylogenetic positions of the three early-diverging hexapod groups, the coneheads (Protura), springtails (Collembola), and two-pronged bristletails (Diplura), have been debated for over a century, with alternative topologies implying drastically different scenarios of the evolution of the insect body plan and hexapod terrestrialisation. We addressed this issue by sampling of all hexapod orders, and experimented with a broad range of across-site compositional heterogeneous models designed to tackle ancient divergences. Our analyses support Protura as the earliest-diverging hexapod lineage (Protura-sister) and Collembola as a sister group to the Diplura, a clade we refer to as  Antennomusculata characterised by the shared possession of internal muscles in the antennal flagellum. The universally recognized  Ellipura hypothesis is recovered under the site-homogenous LG model. Our cross-validation analysis shows that the CAT-GTR model that recovers Protura-sister fits significantly better than homogenous model. Furthermore, as a very unusual group, Protura as the first diverging lineage of hexapods is also supported by other lines of evidence, such as mitogenomics, comparative embryology, and sperm morphology. The backbone phylogeny of hexapods recovered in this study will facilitate the exploration of the underpinnings of hexapod terrestrialisation and mega-diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575685">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575685" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575685">
        <p class="paperTitle">Origin and evolutionary trajectories of brown algal sex chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575685" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575685" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barrera-Redondo, J.; Lipinska, A. P.; Liu, P.; Dinatale, E.; Cossard, G.; Bogaert, K.; Hoshino, M.; Avia, K.; Avdievich, E.; Leiria, G.; Liesner, D.; Luthringer, R.; Godfroy, O.; Heesch, S.; Nehr, Z.; Peters, A. F.; Hoarau, G.; Pearson, G. A.; Aury, J.-M.; Wincker, P.; Denoeud, F.; Cock, J. M.; Haas, F. B.; Coelho, S. M.</p>
        <p class="info">Score: 2.9, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575685' target='https://doi.org/10.1101/2024.01.15.575685'> 10.1101/2024.01.15.575685</a></p>
        <p class="abstract">Sex chromosomes fall into three classes: XX/XY, ZW/ZZ and U/V systems. The rise, evolution and demise of U/V systems have remained enigmatic to date. Here, we analyze genomes spanning the entire brown algal phylogeny to decipher their sex-determination evolutionary history. The birth of U/V sex chromosomes evolved more than 250 million years ago, when a pivotal male-determinant located in a discrete region in proto-U and proto-V chromosomes ceased recombining. Over time, nested inversions led to step-wise expansions, accompanying increasing morphological complexity and sexual differentiation of brown seaweeds. Unlike XX/XY and ZW/ZZ, U/V evolve mainly by gene gain, showing minimal degeneration. They are structurally dynamic, and act as genomic &#39;cradles&#39; fostering the birth of new genes. Our analyses show that hermaphroditism arose from ancestral males that acquired U-specific genes by ectopic recombination, and that in the transition from a U/V to an XX/XY system, V-specific genes moved down the genetic hierarchy of sex determination. Both events lead to the demise of U and V and erosion of their specific genomic characteristics. Taken together, our findings offer a comprehensive model of U/V sex chromosome evolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
