<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Epigenetic priming of embryonic lineages in the mammalian epiblast
Authors: Sendra, M.; McDole, K.; Jimenez-Carretero, D.; de Dios-Hourcade, J.; Temino Valbuena, S.; Raiola, M.; Guignard, L.; Keller, P. J.; Sanchez-Cabo, F.; Dominguez, J. N.; Torres, M.
Score: 19.9, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575188
Understanding the diversification of mammalian cell lineages is an essential to embryonic development, organ regeneration and tissue engineering. Shortly after implantation in the uterus, the pluripotent cells of the mammalian epiblast generate the three germ layers: ectoderm, mesoderm and endoderm1.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Epigenetic priming of embryonic lineages in the mammalian epiblast
Authors: Sendra, M.; McDole, K.; Jimenez-Carretero, D.; de Dios-Hourcade, J.; Temino Valbuena, S.; Raiola, M.; Guignard, L.; Keller, P. J.; Sanchez-Cabo, F.; Dominguez, J. N.; Torres, M.
Score: 19.9, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575188
Understanding the diversification of mammalian cell lineages is an essential to embryonic development, organ regeneration and tissue engineering. Shortly after implantation in the uterus, the pluripotent cells of the mammalian epiblast generate the three germ layers: ectoderm, mesoderm and endoderm1." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Epigenetic priming of embryonic lineages in the mammalian epiblast
Authors: Sendra, M.; McDole, K.; Jimenez-Carretero, D.; de Dios-Hourcade, J.; Temino Valbuena, S.; Raiola, M.; Guignard, L.; Keller, P. J.; Sanchez-Cabo, F.; Dominguez, J. N.; Torres, M.
Score: 19.9, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575188
Understanding the diversification of mammalian cell lineages is an essential to embryonic development, organ regeneration and tissue engineering. Shortly after implantation in the uterus, the pluripotent cells of the mammalian epiblast generate the three germ layers: ectoderm, mesoderm and endoderm1."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Epigenetic priming of embryonic lineages in the mammalian epiblast\nAuthors: Sendra, M.; McDole, K.; Jimenez-Carretero, D.; de Dios-Hourcade, J.; Temino Valbuena, S.; Raiola, M.; Guignard, L.; Keller, P. J.; Sanchez-Cabo, F.; Dominguez, J. N.; Torres, M.\nScore: 19.9, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575188\nUnderstanding the diversification of mammalian cell lineages is an essential to embryonic development, organ regeneration and tissue engineering. Shortly after implantation in the uterus, the pluripotent cells of the mammalian epiblast generate the three germ layers: ectoderm, mesoderm and endoderm1.",
  "keywords": [
    
  ],
  "articleBody": " Epigenetic priming of embryonic lineages in the mammalian epiblast\nAuthors: Sendra, M.; McDole, K.; Jimenez-Carretero, D.; de Dios-Hourcade, J.; Temino Valbuena, S.; Raiola, M.; Guignard, L.; Keller, P. J.; Sanchez-Cabo, F.; Dominguez, J. N.; Torres, M.\nScore: 19.9, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575188\nUnderstanding the diversification of mammalian cell lineages is an essential to embryonic development, organ regeneration and tissue engineering. Shortly after implantation in the uterus, the pluripotent cells of the mammalian epiblast generate the three germ layers: ectoderm, mesoderm and endoderm1. Although clonal analyses suggest early specification of epiblast cells towards particular cell lineages2-4, single-cell transcriptomes do not identify lineage-specific markers in the epiblast5-11 and thus, the molecular regulation of such specification remains unknow. Here, we studied the epigenetic landscape of single epiblast cells, which revealed lineage priming towards endoderm, ectoderm or mesoderm. Unexpectedly, epiblast cells with mesodermal priming show a strong signature for the endothelial/endocardial fate, suggesting early specification of this lineage aside from other mesoderm. Through clonal analysis and live imaging, we show that endothelial precursors show early lineage divergence from the rest of mesodermal derivatives. In particular, cardiomyocytes and endocardial cells show limited lineage relationship, despite being temporally and spatially co-recruited during gastrulation. Furthermore, analysing the live tracks of single cells through unsupervised classification of cell migratory activity, we found early behavioral divergence of endothelial precursors shortly after the onset of mesoderm migration towards the cardiogenic area. These results provide a new model for the phenotypically silent specification of mammalian cell lineages in pluripotent cells of the epiblast and modify current knowledge on the sequence and timing of cardiovascular lineages diversification12,13.\nEnvironmental and molecular control of tissue-specific ionocyte differentiation in zebrafish\nAuthors: Peloggia, J.; Lush, M. E.; Tsai, Y.-Y.; Wood, C.; Piotrowski, T.\nScore: 17.2, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575421\nOrganisms adjust their physiology to cope with environmental fluctuations and maintain fitness. These adaptations occur via genetic changes over multiple generations or through acclimation, a set of reversible phenotypic changes that confer resilience to the individual. Aquatic organisms are subject to dramatic seasonal fluctuations in water salinity, which can affect the function of lateral line mechanosensory hair cells. To maintain hair cell function when salinity decreases, ion-regulating cells, Neuromast-associated ionocytes (Nm ionocytes), increase in number and invade lateral line neuromasts. How environmental changes trigger this adaptive differentiation of Nm ionocytes and how these cells are specified is still unknown. Here, we identify Nm ionocyte progenitors as foxi3a/foxi3b-expressing skin cells and show that their differentiation is associated with sequential activation of different Notch pathway components, which control ionocyte survival. We demonstrate that new Nm ionocytes are rapidly specified by absolute salinity levels, independently of stress response pathways. We further show that Nm ionocyte differentiation is selectively triggered by depletion of specific ions, such as Ca2+ and Na+/Cl-, but not by low K+ levels, and is independent of media osmolarity. Finally, we demonstrate that hair cell activity plays a role in Nm ionocyte recruitment and that systemic factors are not necessary for Nm ionocyte induction. In summary, we have identified how environmental changes activate a signaling cascade that triggers basal skin cell progenitors to differentiate into Nm ionocytes and invade lateral line organs. This adaptive behavior is an example of physiological plasticity that may prove essential for survival in changing climates.\nAn inappropriate decline in ribosome levels drives a diverse set of neurodevelopmental disorders\nAuthors: Ni, C.; Yu, L.; Vona, B.; Park, D.; Wei, Y.; Schmitz, D.; Wei, Y.; Ding, Y.; Sakurai, M.; Ballard, E.; Liu, Y.; Kumar, A.; Xing, C.; Kim, H.-G.; Ekmekci, C.; Karimiani, E.; Imannezhad, S.; Eghbal, F.; Badv, R.; Schwaibold, E.; Dehghani, M.; Mehrjardi, M.; Metanat, Z.; Eslamiyeh, H.; Khouj, E.; Alhajj, S.; Chedrawi, A.; Alves, C.; Houlden, H.; Kruer, M.; Alkuraya, F.; Cenik, C.; Maroofian, R.; Wu, J.; Buszczak, M.\nScore: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.09.574708\nMany neurodevelopmental defects are linked to perturbations in genes involved in housekeeping functions, such as those encoding ribosome biogenesis factors. However, how reductions in ribosome biogenesis can result in tissue and developmental specific defects remains a mystery. Here we describe new allelic variants in the ribosome biogenesis factor AIRIM primarily associated with neurodevelopmental disorders. Using human cerebral organoids in combination with proteomic analysis, single-cell transcriptome analysis across multiple developmental stages, and single organoid translatome analysis, we identify a previously unappreciated mechanism linking changes in ribosome levels and the timing of cell fate specification during early brain development. We find ribosome levels decrease during neuroepithelial differentiation, making differentiating cells particularly vulnerable to perturbations in ribosome biogenesis during this time. Reduced ribosome availability more profoundly impacts the translation of specific transcripts, disrupting both survival and cell fate commitment of transitioning neuroepithelia. Enhancing mTOR activity by both genetic and pharmacologic approaches ameliorates the growth and developmental defects associated with intellectual disability linked variants, identifying potential treatment options for specific brain ribosomopathies. This work reveals the cellular and molecular origins of protein synthesis defect-related disorders of human brain development. HighlightsO_LIAIRIM variants reduce ribosome levels specifically in neural progenitor cells. C_LIO_LIInappropriately low ribosome levels cause a transient delay in radial glia fate commitment. C_LIO_LIReduced ribosome levels impair translation of a selected subset of mRNAs. C_LIO_LIGenetic and pharmacologic activation of mTORC1 suppresses AIRIM-linked phenotypes. C_LI\nFoxO transcription factors actuate the formative pluripotency specific gene expression programme\nAuthors: Santini, L.; Kowald, S.; Sestini, G.; Rivron, N. C.; Leeb, M.\nScore: 7.3, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575494\nNaive pluripotency is sustained by a self-reinforcing gene regulatory network (GRN) comprising core and naive pluripotency-specific transcription factors (TFs). Upon exiting naive pluripotency, ES cells transition through a formative post-implantation-like pluripotent state, where they acquire competence for lineage-choice. However, the mechanisms underlying disengagement from the naive GRN and initiation of the formative GRN are unclear. Here, we demonstrate that phosphorylated AKT acts as a gatekeeper that prevents nuclear localization of FoxO TFs in naive ESCs. PTEN-mediated reduction of AKT activity upon exit from naive pluripotency allows nuclear entry of FoxO TFs, enforcing a cell fate transition by binding and activating formative pluripotency-specific enhancers. Indeed, FoxO TFs are necessary and sufficient for transition from the naive to the formative pluripotent state. Our work uncovers a pivotal role for FoxO TFs and AKT signalling in mechanisms establishing formative post-implantation pluripotency, a critical early embryonic cell fate transition.\nSingle-cell transcriptional dynamics in a living vertebrate\nAuthors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.\nScore: 27.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574108\nThe ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos. We addressed this deficit by applying MS2-MCP mRNA labeling to the quantification of transcription in zebrafish, a model vertebrate. We developed a platform of transgenic organisms, light sheet fluorescence microscopy, and optimized image analysis that enables visualization and quantification of MS2 reporters. We used these tools to obtain the first single-cell, real-time measurements of transcriptional dynamics of the segmentation clock. Our measurements challenge the traditional view of smooth clock oscillations and instead suggest a model of discrete transcriptional bursts that are organized in space and time. Together, these results highlight how measuring single-cell transcriptional activity can reveal unexpected features of gene regulation and how this data can fuel the dialogue between theory and experiment.\nA spatiotemporally resolved atlas of mRNA decay in the C. elegans embryo reveals differential regulation of mRNA stability across stages and cell types\nAuthors: Peng, F.; Nordgren, C. E.; Murray, J. I.\nScore: 5.3, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575757\nDuring embryonic development, cells undergo dynamic changes in gene expression that are required for appropriate cell fate specification. Although both transcription and mRNA degradation contribute to gene expression dynamics, patterns of mRNA decay are less well-understood. Here we directly measured spatiotemporally resolved mRNA decay rates transcriptome-wide throughout C. elegans embryogenesis by transcription inhibition followed by bulk and single-cell RNA-sequencing. This allowed us to calculate mRNA half-lives within specific cell types and developmental stages and identify differentially regulated mRNA decay throughout embryonic development. We identified transcript features that are correlated with mRNA stability and found that mRNA decay rates are associated with distinct peaks in gene expression over time. Moreover, we provide evidence that, on average, mRNA is more stable in the germline compared to in the soma and in later embryonic stages compared to in earlier stages. This work suggests that differential mRNA decay across cell states and time helps to shape developmental gene expression, and it provides a valuable resource for studies of mRNA turnover regulatory mechanisms.\nDifferential tissue deformability underlies shape divergence of the embryonic brain and spinal cord under fluid pressure\nAuthors: McLaren, S. B. P.; Xue, S.-L.; Ding, S.; Winkel, A.; Baldwin, O.; Dwarakacherla, S.; Franze, K.; Hannezo, E.; Xiong, F.\nScore: 3.4, Published: 2024-01-12 DOI: 10.1101/2024.01.12.575349\nAn expanded brain enables the complex behaviours of vertebrates that promote their adaptation in diverse ecological niches1-3. Initial morphological differences between the brain and spinal cord emerge as the antero-posteriorly pat erned neural plate folds to form the neural tube4-7 during embryonic development. Following neural tube closure, a dramatic expansion of the brain diverges its shape from the spinal cord8, setting their distinct morphologies for further development9,10. How the brain and the spinal cord expand differentially remains unclear. Here, using the chicken embryo as a model, we show that the hindbrain expands through dorsal tissue thinning under a positive hydrostatic pressure from the neural tube lumen11,12 while the dorsal spinal cord shape resists the same pressure. Using magnetic droplets and atomic force microscopy, we reveal that the dorsal tissue in the hindbrain is more fluid than in the spinal cord. The dorsal hindbrain harbours more migratory neural crest cells13 and exhibits reduced apical actin and a disorganised laminin matrix compared to the dorsal spinal cord. Blocking the activity of neural crest-associated matrix metalloproteinases inhibited dorsal tissue thinning, leading to abnormal brain morphology. Transplanting early dorsal hindbrain cells to the spinal cord was sufficient to create a region with expanded brain-like morphology including a thinned-out roof. Our [fi]ndings open new questions in vertebrate head evolution and neural tube defects, and suggest a general role of mechanical pre-pat ern in creating shape differences in epithelial tubes.\nTranscriptomic analysis of pubertal and adult virgin mouse mammary epithelial and stromal cell populations\nAuthors: Heijmans, N.; Wiese, K. E.; Jonkers, J.; van Amerongen, R.\nScore: 3.4, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575148\nConflicting data exist as to how mammary epithelial cell proliferation changes during the reproductive cycle. To study the effect of endogenous hormone fluctuations on gene expression in the mouse mammary gland, we performed bulk RNAseq analyses of epithelial and stromal cell populations that were isolated either during puberty or at different stages of the adult virgin estrous cycle. Our data confirm prior findings that proliferative changes do not occur in every mouse in every cycle. We also show that during the estrous cycle the main gene expression changes occur in adipocytes and other stromal cells. Finally, we present a comprehensive overview of the Wnt gene expression landscape in different mammary gland cell types in pubertal and adult mice. This work contributes to understanding the effects of physiological hormone fluctuations and locally produced signaling molecules on gene expression changes in the mammary gland during the reproductive cycle and should be a useful resource for future studies investigating gene expression patterns in different cell types across different developmental timepoints.\nMorphologically defined substages of tail tip morphogenesis in C. elegans males\nAuthors: Kiontke, K. C.; Fernandez, P.; Woronik, A.; Fitch, D. H. A.\nScore: 3.0, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575265\nBackground Sex-specific morphogenesis occurs in C. elegans in the vulva of the hermaphrodite and in the male tail during the last larval stage. Temporal progression of vulva morphogenesis has been described in fine detail. However, a similar precise description of male tail morphogenesis was lacking. Results We here describe morphogenesis of the male tail at time points matching vulva development with special focus on morphogenesis of the tail tip. Using fluorescent reporters, we follow changes in cell shapes, cell fusions, nuclear migration, modifications in the basement membrane and formation of a new apical extracellular matrix at the end of the tail. Conclusion Our analysis answers two open questions about tail tip morphogenesis (TTM) by showing that one of the four tail tip cells, hyp11, remains separate while the other cells fuse with each other and with two additional tail cells to form a ventral tail syncytium. This fusion begins early during TTM but is only completed towards the end of the process. This work provides a framework for future investigations of cell-biological factors that drive male tail morpogenesis.\nAltered Extracellular Matrix Structure and Elevated Stiffness in a Brain Organoid Model for Disease\nAuthors: Karlinski Zur, M.; Bhattacharya, B.; Ben Dor, S.; Solomonov, I.; Savidor, A.; Sapir, T.; Harris, T.; Olender, T.; Sagi, I.; Schmidt, R.; Schwarz, J.; Buxboim, A.; Reiner, O.\nScore: 2.2, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574777\nThe viscoelasticity of tissues impacts their shape, as well as the growth and differentiation of their cells. Nevertheless, little is known about changes in viscoelastic properties during brain malformations. Lissencephaly is a severe malformation of cortical development caused by LIS1 mutations, which results in a lack of cortical convolutions. Here, we show that human-derived brain organoids with LIS1 mutation are stiffer than control ones at multiple developmental times. This stiffening is accompanied by abnormal ECM expression and organization, as well as elevated water content, as measured by diffusion-weighted MRI. Proteolytic cleavage of ECM components by short-term treatment with the catalytic subunit of MMP9 reduced the stiffening and water diffusion levels of mutated organoids to control levels. Finally, based on the molecular and rheological properties obtained, we generated a computational microstructure mechanical model that can successfully predict mechanical changes that follow differential ECM localization and integrity in the developing brain. Overall, our study reveals that LIS1 is essential for the expression and organization of ECM proteins during brain development, and its mutation leads to a substantial viscoelastic change. To our knowledge, this is the first study to elucidate how tissue mechanics change in disease states using human brain organoids.\n",
  "wordCount" : "2358",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575188">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575188" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575188">
        <p class="paperTitle">Epigenetic priming of embryonic lineages in the mammalian epiblast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575188" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575188" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sendra, M.; McDole, K.; Jimenez-Carretero, D.; de Dios-Hourcade, J.; Temino Valbuena, S.; Raiola, M.; Guignard, L.; Keller, P. J.; Sanchez-Cabo, F.; Dominguez, J. N.; Torres, M.</p>
        <p class="info">Score: 19.9, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575188' target='https://doi.org/10.1101/2024.01.11.575188'> 10.1101/2024.01.11.575188</a></p>
        <p class="abstract">Understanding the diversification of mammalian cell lineages is an essential to embryonic development, organ regeneration and tissue engineering. Shortly after implantation in the uterus, the pluripotent cells of the mammalian epiblast generate the three germ layers: ectoderm, mesoderm and endoderm1. Although clonal analyses suggest early specification of epiblast cells towards particular cell lineages2-4, single-cell transcriptomes do not identify lineage-specific markers in the epiblast5-11 and thus, the molecular regulation of such specification remains unknow. Here, we studied the epigenetic landscape of single epiblast cells, which revealed lineage priming towards endoderm, ectoderm or mesoderm. Unexpectedly, epiblast cells with mesodermal priming show a strong signature for the endothelial/endocardial fate, suggesting early specification of this lineage aside from other mesoderm. Through clonal analysis and live imaging, we show that endothelial precursors show early lineage divergence from the rest of mesodermal derivatives. In particular, cardiomyocytes and endocardial cells show limited lineage relationship, despite being temporally and spatially co-recruited during gastrulation. Furthermore, analysing the live tracks of single cells through unsupervised classification of cell migratory activity, we found early behavioral divergence of endothelial precursors shortly after the onset of mesoderm migration towards the cardiogenic area. These results provide a new model for the phenotypically silent specification of mammalian cell lineages in pluripotent cells of the epiblast and modify current knowledge on the sequence and timing of cardiovascular lineages diversification12,13.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575421" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575421">
        <p class="paperTitle">Environmental and molecular control of tissue-specific ionocyte differentiation in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peloggia, J.; Lush, M. E.; Tsai, Y.-Y.; Wood, C.; Piotrowski, T.</p>
        <p class="info">Score: 17.2, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575421' target='https://doi.org/10.1101/2024.01.12.575421'> 10.1101/2024.01.12.575421</a></p>
        <p class="abstract">Organisms adjust their physiology to cope with environmental fluctuations and maintain fitness. These adaptations occur via genetic changes over multiple generations or through acclimation, a set of reversible phenotypic changes that confer resilience to the individual. Aquatic organisms are subject to dramatic seasonal fluctuations in water salinity, which can affect the function of lateral line mechanosensory hair cells. To maintain hair cell function when salinity decreases, ion-regulating cells, Neuromast-associated ionocytes (Nm ionocytes), increase in number and invade lateral line neuromasts. How environmental changes trigger this adaptive differentiation of Nm ionocytes and how these cells are specified is still unknown. Here, we identify Nm ionocyte progenitors as foxi3a/foxi3b-expressing skin cells and show that their differentiation is associated with sequential activation of different Notch pathway components, which control ionocyte survival. We demonstrate that new Nm ionocytes are rapidly specified by absolute salinity levels, independently of stress response pathways. We further show that Nm ionocyte differentiation is selectively triggered by depletion of specific ions, such as Ca2&#43; and Na&#43;/Cl-, but not by low K&#43; levels, and is independent of media osmolarity. Finally, we demonstrate that hair cell activity plays a role in Nm ionocyte recruitment and that systemic factors are not necessary for Nm ionocyte induction. In summary, we have identified how environmental changes activate a signaling cascade that triggers basal skin cell progenitors to differentiate into Nm ionocytes and invade lateral line organs. This adaptive behavior is an example of physiological plasticity that may prove essential for survival in changing climates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574708" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574708">
        <p class="paperTitle">An inappropriate decline in ribosome levels drives a diverse set of neurodevelopmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ni, C.; Yu, L.; Vona, B.; Park, D.; Wei, Y.; Schmitz, D.; Wei, Y.; Ding, Y.; Sakurai, M.; Ballard, E.; Liu, Y.; Kumar, A.; Xing, C.; Kim, H.-G.; Ekmekci, C.; Karimiani, E.; Imannezhad, S.; Eghbal, F.; Badv, R.; Schwaibold, E.; Dehghani, M.; Mehrjardi, M.; Metanat, Z.; Eslamiyeh, H.; Khouj, E.; Alhajj, S.; Chedrawi, A.; Alves, C.; Houlden, H.; Kruer, M.; Alkuraya, F.; Cenik, C.; Maroofian, R.; Wu, J.; Buszczak, M.</p>
        <p class="info">Score: 13.0, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574708' target='https://doi.org/10.1101/2024.01.09.574708'> 10.1101/2024.01.09.574708</a></p>
        <p class="abstract">Many neurodevelopmental defects are linked to perturbations in genes involved in housekeeping functions, such as those encoding ribosome biogenesis factors. However, how reductions in ribosome biogenesis can result in tissue and developmental specific defects remains a mystery. Here we describe new allelic variants in the ribosome biogenesis factor AIRIM primarily associated with neurodevelopmental disorders. Using human cerebral organoids in combination with proteomic analysis, single-cell transcriptome analysis across multiple developmental stages, and single organoid translatome analysis, we identify a previously unappreciated mechanism linking changes in ribosome levels and the timing of cell fate specification during early brain development. We find ribosome levels decrease during neuroepithelial differentiation, making differentiating cells particularly vulnerable to perturbations in ribosome biogenesis during this time. Reduced ribosome availability more profoundly impacts the translation of specific transcripts, disrupting both survival and cell fate commitment of transitioning neuroepithelia. Enhancing mTOR activity by both genetic and pharmacologic approaches ameliorates the growth and developmental defects associated with intellectual disability linked variants, identifying potential treatment options for specific brain ribosomopathies. This work reveals the cellular and molecular origins of protein synthesis defect-related disorders of human brain development.

HighlightsO_LIAIRIM variants reduce ribosome levels specifically in neural progenitor cells.
C_LIO_LIInappropriately low ribosome levels cause a transient delay in radial glia fate commitment.
C_LIO_LIReduced ribosome levels impair translation of a selected subset of mRNAs.
C_LIO_LIGenetic and pharmacologic activation of mTORC1 suppresses AIRIM-linked phenotypes.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575494">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575494" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575494">
        <p class="paperTitle">FoxO transcription factors actuate the formative pluripotency specific gene expression programme</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575494" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575494" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santini, L.; Kowald, S.; Sestini, G.; Rivron, N. C.; Leeb, M.</p>
        <p class="info">Score: 7.3, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575494' target='https://doi.org/10.1101/2024.01.13.575494'> 10.1101/2024.01.13.575494</a></p>
        <p class="abstract">Naive pluripotency is sustained by a self-reinforcing gene regulatory network (GRN) comprising core and naive pluripotency-specific transcription factors (TFs). Upon exiting naive pluripotency, ES cells transition through a formative post-implantation-like pluripotent state, where they acquire competence for lineage-choice. However, the mechanisms underlying disengagement from the naive GRN and initiation of the formative GRN are unclear. Here, we demonstrate that phosphorylated AKT acts as a gatekeeper that prevents nuclear localization of FoxO TFs in naive ESCs. PTEN-mediated reduction of AKT activity upon exit from naive pluripotency allows nuclear entry of FoxO TFs, enforcing a cell fate transition by binding and activating formative pluripotency-specific enhancers. Indeed, FoxO TFs are necessary and sufficient for transition from the naive to the formative pluripotent state. Our work uncovers a pivotal role for FoxO TFs and AKT signalling in mechanisms establishing formative post-implantation pluripotency, a critical early embryonic cell fate transition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574108" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574108">
        <p class="paperTitle">Single-cell transcriptional dynamics in a living vertebrate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.</p>
        <p class="info">Score: 27.3, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574108' target='https://doi.org/10.1101/2024.01.03.574108'> 10.1101/2024.01.03.574108</a></p>
        <p class="abstract">The ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos. We addressed this deficit by applying MS2-MCP mRNA labeling to the quantification of transcription in zebrafish, a model vertebrate. We developed a platform of transgenic organisms, light sheet fluorescence microscopy, and optimized image analysis that enables visualization and quantification of MS2 reporters. We used these tools to obtain the first single-cell, real-time measurements of transcriptional dynamics of the segmentation clock. Our measurements challenge the traditional view of smooth clock oscillations and instead suggest a model of discrete transcriptional bursts that are organized in space and time. Together, these results highlight how measuring single-cell transcriptional activity can reveal unexpected features of gene regulation and how this data can fuel the dialogue between theory and experiment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575757" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575757">
        <p class="paperTitle">A spatiotemporally resolved atlas of mRNA decay in the C. elegans embryo reveals differential regulation of mRNA stability across stages and cell types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, F.; Nordgren, C. E.; Murray, J. I.</p>
        <p class="info">Score: 5.3, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575757' target='https://doi.org/10.1101/2024.01.15.575757'> 10.1101/2024.01.15.575757</a></p>
        <p class="abstract">During embryonic development, cells undergo dynamic changes in gene expression that are required for appropriate cell fate specification. Although both transcription and mRNA degradation contribute to gene expression dynamics, patterns of mRNA decay are less well-understood. Here we directly measured spatiotemporally resolved mRNA decay rates transcriptome-wide throughout C. elegans embryogenesis by transcription inhibition followed by bulk and single-cell RNA-sequencing. This allowed us to calculate mRNA half-lives within specific cell types and developmental stages and identify differentially regulated mRNA decay throughout embryonic development. We identified transcript features that are correlated with mRNA stability and found that mRNA decay rates are associated with distinct peaks in gene expression over time. Moreover, we provide evidence that, on average, mRNA is more stable in the germline compared to in the soma and in later embryonic stages compared to in earlier stages. This work suggests that differential mRNA decay across cell states and time helps to shape developmental gene expression, and it provides a valuable resource for studies of mRNA turnover regulatory mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575349" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575349">
        <p class="paperTitle">Differential tissue deformability underlies shape divergence of the embryonic brain and spinal cord under fluid pressure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McLaren, S. B. P.; Xue, S.-L.; Ding, S.; Winkel, A.; Baldwin, O.; Dwarakacherla, S.; Franze, K.; Hannezo, E.; Xiong, F.</p>
        <p class="info">Score: 3.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575349' target='https://doi.org/10.1101/2024.01.12.575349'> 10.1101/2024.01.12.575349</a></p>
        <p class="abstract">An expanded brain enables the complex behaviours of vertebrates that promote their adaptation in diverse ecological niches1-3. Initial morphological differences between the brain and spinal cord emerge as the antero-posteriorly pat erned neural plate folds to form the neural tube4-7 during embryonic development. Following neural tube closure, a dramatic expansion of the brain diverges its shape from the spinal cord8, setting their distinct morphologies for further development9,10. How the brain and the spinal cord expand differentially remains unclear. Here, using the chicken embryo as a model, we show that the hindbrain expands through dorsal tissue thinning under a positive hydrostatic pressure from the neural tube lumen11,12 while the dorsal spinal cord shape resists the same pressure. Using magnetic droplets and atomic force microscopy, we reveal that the dorsal tissue in the hindbrain is more fluid than in the spinal cord. The dorsal hindbrain harbours more migratory neural crest cells13 and exhibits reduced apical actin and a disorganised laminin matrix compared to the dorsal spinal cord. Blocking the activity of neural crest-associated matrix metalloproteinases inhibited dorsal tissue thinning, leading to abnormal brain morphology. Transplanting early dorsal hindbrain cells to the spinal cord was sufficient to create a region with expanded brain-like morphology including a thinned-out roof. Our [fi]ndings open new questions in vertebrate head evolution and neural tube defects, and suggest a general role of mechanical pre-pat ern in creating shape differences in epithelial tubes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575148" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575148">
        <p class="paperTitle">Transcriptomic analysis of pubertal and adult virgin mouse mammary epithelial and stromal cell populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heijmans, N.; Wiese, K. E.; Jonkers, J.; van Amerongen, R.</p>
        <p class="info">Score: 3.4, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575148' target='https://doi.org/10.1101/2024.01.11.575148'> 10.1101/2024.01.11.575148</a></p>
        <p class="abstract">Conflicting data exist as to how mammary epithelial cell proliferation changes during the reproductive cycle. To study the effect of endogenous hormone fluctuations on gene expression in the mouse mammary gland, we performed bulk RNAseq analyses of epithelial and stromal cell populations that were isolated either during puberty or at different stages of the adult virgin estrous cycle. Our data confirm prior findings that proliferative changes do not occur in every mouse in every cycle. We also show that during the estrous cycle the main gene expression changes occur in adipocytes and other stromal cells. Finally, we present a comprehensive overview of the Wnt gene expression landscape in different mammary gland cell types in pubertal and adult mice. This work contributes to understanding the effects of physiological hormone fluctuations and locally produced signaling molecules on gene expression changes in the mammary gland during the reproductive cycle and should be a useful resource for future studies investigating gene expression patterns in different cell types across different developmental timepoints.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575265" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575265">
        <p class="paperTitle">Morphologically defined substages of tail tip morphogenesis in C. elegans males</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kiontke, K. C.; Fernandez, P.; Woronik, A.; Fitch, D. H. A.</p>
        <p class="info">Score: 3.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575265' target='https://doi.org/10.1101/2024.01.11.575265'> 10.1101/2024.01.11.575265</a></p>
        <p class="abstract">Background Sex-specific morphogenesis occurs in C. elegans in the vulva of the hermaphrodite and in the male tail during the last larval stage. Temporal progression of vulva morphogenesis has been described in fine detail. However, a similar precise description of male tail morphogenesis was lacking. Results We here describe morphogenesis of the male tail at time points matching vulva development with special focus on morphogenesis of the tail tip. Using fluorescent reporters, we follow changes in cell shapes, cell fusions, nuclear migration, modifications in the basement membrane and formation of a new apical extracellular matrix at the end of the tail. Conclusion Our analysis answers two open questions about tail tip morphogenesis (TTM) by showing that one of the four tail tip cells, hyp11, remains separate while the other cells fuse with each other and with two additional tail cells to form a ventral tail syncytium. This fusion begins early during TTM but is only completed towards the end of the process. This work provides a framework for future investigations of cell-biological factors that drive male tail morpogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574777" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574777">
        <p class="paperTitle">Altered Extracellular Matrix Structure and Elevated Stiffness in a Brain Organoid Model for Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karlinski Zur, M.; Bhattacharya, B.; Ben Dor, S.; Solomonov, I.; Savidor, A.; Sapir, T.; Harris, T.; Olender, T.; Sagi, I.; Schmidt, R.; Schwarz, J.; Buxboim, A.; Reiner, O.</p>
        <p class="info">Score: 2.2, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574777' target='https://doi.org/10.1101/2024.01.09.574777'> 10.1101/2024.01.09.574777</a></p>
        <p class="abstract">The viscoelasticity of tissues impacts their shape, as well as the growth and differentiation of their cells. Nevertheless, little is known about changes in viscoelastic properties during brain malformations. Lissencephaly is a severe malformation of cortical development caused by LIS1 mutations, which results in a lack of cortical convolutions. Here, we show that human-derived brain organoids with LIS1 mutation are stiffer than control ones at multiple developmental times. This stiffening is accompanied by abnormal ECM expression and organization, as well as elevated water content, as measured by diffusion-weighted MRI. Proteolytic cleavage of ECM components by short-term treatment with the catalytic subunit of MMP9 reduced the stiffening and water diffusion levels of mutated organoids to control levels. Finally, based on the molecular and rheological properties obtained, we generated a computational microstructure mechanical model that can successfully predict mechanical changes that follow differential ECM localization and integrity in the developing brain. Overall, our study reveals that LIS1 is essential for the expression and organization of ECM proteins during brain development, and its mutation leads to a substantial viscoelastic change. To our knowledge, this is the first study to elucidate how tissue mechanics change in disease states using human brain organoids.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
