<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sox9 links biliary maturation to branching morphogenesis
Authors: Hrncir, H. R.; Bombin, S.; Goodloe, B.; Hogan, C. B.; Jadi, O.; Gracz, A. D.
Score: 12.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574730
Branching morphogenesis couples cellular differentiation with development of tissue architecture. Intrahepatic bile duct (IHBD) morphogenesis is initiated with biliary epithelial cell (BEC) specification and eventually forms a heterogeneous network of large ducts and small ductules. Here, we show that Sox9 is required for developmental establishment of small ductules.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Sox9 links biliary maturation to branching morphogenesis
Authors: Hrncir, H. R.; Bombin, S.; Goodloe, B.; Hogan, C. B.; Jadi, O.; Gracz, A. D.
Score: 12.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574730
Branching morphogenesis couples cellular differentiation with development of tissue architecture. Intrahepatic bile duct (IHBD) morphogenesis is initiated with biliary epithelial cell (BEC) specification and eventually forms a heterogeneous network of large ducts and small ductules. Here, we show that Sox9 is required for developmental establishment of small ductules." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Sox9 links biliary maturation to branching morphogenesis
Authors: Hrncir, H. R.; Bombin, S.; Goodloe, B.; Hogan, C. B.; Jadi, O.; Gracz, A. D.
Score: 12.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574730
Branching morphogenesis couples cellular differentiation with development of tissue architecture. Intrahepatic bile duct (IHBD) morphogenesis is initiated with biliary epithelial cell (BEC) specification and eventually forms a heterogeneous network of large ducts and small ductules. Here, we show that Sox9 is required for developmental establishment of small ductules."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Sox9 links biliary maturation to branching morphogenesis\nAuthors: Hrncir, H. R.; Bombin, S.; Goodloe, B.; Hogan, C. B.; Jadi, O.; Gracz, A. D.\nScore: 12.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574730\nBranching morphogenesis couples cellular differentiation with development of tissue architecture. Intrahepatic bile duct (IHBD) morphogenesis is initiated with biliary epithelial cell (BEC) specification and eventually forms a heterogeneous network of large ducts and small ductules. Here, we show that Sox9 is required for developmental establishment of small ductules.",
  "keywords": [
    
  ],
  "articleBody": " Sox9 links biliary maturation to branching morphogenesis\nAuthors: Hrncir, H. R.; Bombin, S.; Goodloe, B.; Hogan, C. B.; Jadi, O.; Gracz, A. D.\nScore: 12.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574730\nBranching morphogenesis couples cellular differentiation with development of tissue architecture. Intrahepatic bile duct (IHBD) morphogenesis is initiated with biliary epithelial cell (BEC) specification and eventually forms a heterogeneous network of large ducts and small ductules. Here, we show that Sox9 is required for developmental establishment of small ductules. IHBDs emerge as a webbed structure by E15.5 and undergo morphological maturation through 2 weeks of age. Developmental knockout of Sox9 leads to decreased postnatal branching morphogenesis, manifesting as loss of ductules in adult livers. In the absence of Sox9, BECs fail to mature and exhibit elevated TGF-{beta} signaling and Activin A. Activin A induces developmental gene expression and morphological defects in BEC organoids and represses ductule formation in postnatal livers. Our data demonstrate that adult IHBD morphology and BEC maturation is regulated by the Sox9-dependent formation of precursors to ductules during development, mediated in part by downregulation of Activin A.\nEnvironmental and molecular control of tissue-specific ionocyte differentiation in zebrafish\nAuthors: Peloggia, J.; Lush, M. E.; Tsai, Y.-Y.; Wood, C.; Piotrowski, T.\nScore: 26.3, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575421\nOrganisms adjust their physiology to cope with environmental fluctuations and maintain fitness. These adaptations occur via genetic changes over multiple generations or through acclimation, a set of reversible phenotypic changes that confer resilience to the individual. Aquatic organisms are subject to dramatic seasonal fluctuations in water salinity, which can affect the function of lateral line mechanosensory hair cells. To maintain hair cell function when salinity decreases, ion-regulating cells, Neuromast-associated ionocytes (Nm ionocytes), increase in number and invade lateral line neuromasts. How environmental changes trigger this adaptive differentiation of Nm ionocytes and how these cells are specified is still unknown. Here, we identify Nm ionocyte progenitors as foxi3a/foxi3b-expressing skin cells and show that their differentiation is associated with sequential activation of different Notch pathway components, which control ionocyte survival. We demonstrate that new Nm ionocytes are rapidly specified by absolute salinity levels, independently of stress response pathways. We further show that Nm ionocyte differentiation is selectively triggered by depletion of specific ions, such as Ca2+ and Na+/Cl-, but not by low K+ levels, and is independent of media osmolarity. Finally, we demonstrate that hair cell activity plays a role in Nm ionocyte recruitment and that systemic factors are not necessary for Nm ionocyte induction. In summary, we have identified how environmental changes activate a signaling cascade that triggers basal skin cell progenitors to differentiate into Nm ionocytes and invade lateral line organs. This adaptive behavior is an example of physiological plasticity that may prove essential for survival in changing climates.\nPerturbing nuclear glycosylation in the mouse preimplantation embryo slows down embryonic growth\nAuthors: Formichetti, S.; Chitnavis, U.; Sadowska, A.; Liu, N.; Boskovic, A.; Boulard, M.\nScore: 12.0, Published: 2024-01-22 DOI: 10.1101/2024.01.22.576677\nThe only known form of intracellular protein glycosylation (O-GlcNAc) is reversible and has been mapped on thousands of cytoplasmic and nuclear proteins, including RNA polymerase II, transcription factors and chromatin modifiers. The O-GlcNAc modification is catalyzed by a single enzyme known as O-GlcNAc Transferase (OGT), that is required for mammalian early development. Remarkably, the regulatory function of protein O-GlcNAcylation in the embryo as well as the embryonic O-GlcNAc proteome remain unknown. Here, we devised a new method to enzymatically remove O-GlcNAc from preimplantation embryonic nuclei, where it accumulates coincidently with embryonic genome activation (EGA). Unexpectedly, the depletion of nuclear O-GlcNAc to undetectable levels has no impact on EGA, but dampens the transcriptional activation of the translational machinery, and triggers a spindle checkpoint response. These molecular alterations were phenotypically associated with a developmental delay starting from early cleavage stages and persisting after embryo implantation, establishing a novel link between nuclear glycosylation and embryonic growth.\nWnt signaling alters CTCF binding patterns and global chromatin structure\nAuthors: Nordin, A.; Chakraborty, C.; Jonasson, M.; Dano, O.; Zambanini, G.; Pagella, P.; Remeseiro, S.; Cantu, C.\nScore: 10.9, Published: 2024-01-22 DOI: 10.1101/2024.01.22.576696\nWnt signaling plays a pivotal role during development, stem cell maintenance, and tissue homeostasis. Upon Wnt pathway activation, {beta}-catenin translocates to the nucleus where it binds the TCF/LEF transcription factors to drive the context-specific expression of Wnt target genes. Coordinating gene expression programs in vertebrates requires a complex interplay between the regulatory and the 3D organization of the genome. However, the impact of Wnt signaling on genome structure has been poorly explored. Here we investigated how Wnt signaling activation influences the binding patterns of CTCF, one of the core architectural proteins that helps establish the 3D genome organization be demarcating topologically associated domains (TAD). This study uncovered a series of CTCF rearrangements under Wnt, that we termed RUW. Notably, RUW sites that were gained upon Wnt activation were typically dependent on {beta}-catenin and were characterized by both CTCF and TCF/LEF binding. Accordingly, many CTCF RUWs aligned with {beta}-catenin binding patterns, and {beta}-catenin and CTCF co-localized in vivo in discreet nuclear puncta only up-on pathway activation. Genome-wide investigation of CTCF-mediated 3D genomic interactions upon Wnt pathway stimulation supported the role of the identified RUWs in mediating Wnt-dependent chromatin loops. Lastly, targeted disruption of selected CTCF binding sites demonstrated their functional contribution to Wnt target gene regulation, implicating regulation of the 3D genomic structure in the execution of tran-scriptional programs orchestrated by developmental pathways.\nA spatiotemporally resolved atlas of mRNA decay in the C. elegans embryo reveals differential regulation of mRNA stability across stages and cell types\nAuthors: Peng, F.; Nordgren, C. E.; Murray, J. I.\nScore: 11.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575757\nDuring embryonic development, cells undergo dynamic changes in gene expression that are required for appropriate cell fate specification. Although both transcription and mRNA degradation contribute to gene expression dynamics, patterns of mRNA decay are less well-understood. Here we directly measured spatiotemporally resolved mRNA decay rates transcriptome-wide throughout C. elegans embryogenesis by transcription inhibition followed by bulk and single-cell RNA-sequencing. This allowed us to calculate mRNA half-lives within specific cell types and developmental stages and identify differentially regulated mRNA decay throughout embryonic development. We identified transcript features that are correlated with mRNA stability and found that mRNA decay rates are associated with distinct peaks in gene expression over time. Moreover, we provide evidence that, on average, mRNA is more stable in the germline compared to in the soma and in later embryonic stages compared to in earlier stages. This work suggests that differential mRNA decay across cell states and time helps to shape developmental gene expression, and it provides a valuable resource for studies of mRNA turnover regulatory mechanisms.\nNeutrophil immune profile controls spinal cord regeneration in zebrafish\nAuthors: de Sena-Tomas, C.; Lameira, L.; Naique Taborda, P.; Laborde, A.; Orger, M. B.; de Oliveira, S.; Saude, L.\nScore: 6.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576035\nSpinal cord injury triggers a strong innate inflammatory response in both non-regenerative mammals and regenerative zebrafish. Neutrophils are the first immune population to be recruited to the injury site. Yet, their role in the repair process, particularly in a regenerative context, remains largely unknown. Here, we show that, following rapid recruitment to the injured spinal cord, neutrophils mostly reverse migrate throughout the zebrafish body. In addition, promoting neutrophil inflammation resolution by inhibiting Cxcr4 boosts cellular and functional regeneration. Neutrophil-specific RNA-seq analysis reveals an enhanced activation state that correlates with a transient increase in tnf- expression in macrophage/microglia populations. Conversely, blocking neutrophil recruitment through Cxcr1/2 inhibition diminishes the presence of macrophage/microglia at the injury site and impairs spinal cord regeneration. Altogether, these findings provide new insights into the role of neutrophils in spinal cord regeneration, emphasizing the significant impact of their immune profile on the outcome of the repair process. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=185 HEIGHT=200 SRC=\"FIGDIR/small/576035v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@30d66borg.highwire.dtl.DTLVardef@4d28a5org.highwire.dtl.DTLVardef@1a797faorg.highwire.dtl.DTLVardef@1eac5ea_HPS_FORMAT_FIGEXP M_FIG C_FIG\nGene module reconstruction elucidates cellular differentiation processes and the regulatory logic of specialized secretion\nAuthors: Wang, Y.; Liu, J.; Du, L. Y.; Wyss, J. L.; Farrell, J. A.; Schier, A. F.\nScore: 11.5, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573643\nDuring differentiation, cells become structurally and functionally specialized, but comprehensive views of the underlying remodeling processes are elusive. Here, we leverage scRNA-seq developmental trajectories to reconstruct differentiation using two secretory tissues as a model system - the zebrafish notochord and hatching gland. First, we present an approach to integrate expression and functional similarities for gene module identification, revealing dozens of gene modules representing known and newly associated differentiation processes and their temporal ordering. Second, we focused on the unfolded protein response (UPR) transducer module to study how general versus cell-type specific secretory functions are regulated. By profiling loss- and gain-of-function embryos, we found that the UPR transcription factors creb3l1, creb3l2, and xbp1 are master regulators of a general secretion program. creb3l1/creb3l2 additionally activate an extracellular matrix secretion program, while xbp1 partners with bhlha15 to activate a gland-specific secretion program. Our study offers a multi-source integrated approach for functional gene module identification and illustrates how transcription factors confer general and specialized cellular functions.\nAutomated Cell Lineage Reconstruction using Label-Free 4D Microscopy\nAuthors: Waliman, M.; Johnson, R. L.; Natesan, G.; Tan, S.; Santella, A.; Hong, R. L.; Shah, P. K.\nScore: 2.6, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576449\nHere we describe embGAN, a deep learning pipeline that addresses the challenge of automated cell detection and tracking in label-free 3D time lapse imaging. embGAN requires no manual data annotation for training, learns robust detections that exhibits a high degree of scale invariance and generalizes well to images acquired in multiple labs on multiple instruments.\nCytoneme-mediated intercellular signaling in keratinocytes essential for epidermal remodeling\nAuthors: Wang, Y.; He, Q.; Has, O.; Forouzesh, K.; Eom, D. S.\nScore: 3.1, Published: 2024-01-20 DOI: 10.1101/2023.11.08.566303\nThe skin, the bodys largest organ, functions as a primary defense mechanism. Epidermal stem cells supply undifferentiated keratinocytes that differentiate as they migrate toward the outermost skin layer. Although such a replenishment process is disrupted in various human skin diseases, its underlying mechanisms remain elusive. With high-resolution live imaging and in vivo manipulations, we revealed that Notch signaling between keratinocytes is mediated by signaling filopodia called cytonemes and is essential for proper keratinocyte differentiation and proliferation. Inhibiting keratinocyte cytonemes reduces Notch within undifferentiated keratinocytes, leading to abnormal differentiation and hyperproliferation, resembling human skin disease phenotypes. Overproduction of Interleukin (IL)-17 signal, associated with skin diseases like psoriasis, induces psoriatic phenotypes via cytonemes in zebrafish. Clathrin interactor 1 (clint1), crucial for epidermal homeostasis in zebrafish and humans, regulates cytoneme extension. Our study suggests that intercellular signaling between keratinocytes via cytonemes is critical for epidermal maintenance, and its misregulation could be an origin of human skin diseases.\nDyrk1a is required for craniofacial development in Xenopus laevis.\nAuthors: Dickinson, A. J. G.; Wahl, S. E.; Litovchick, L.; Sesay, F.; Johnson, H. K.\nScore: 2.8, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575394\nLoss of function mutations in the dual specificity tyrosine-phosphorylation-regulated kinase 1A (DYRK1A) gene are associated with craniofacial malformations in humans. Here we characterized the effects of deficient DYRK1A in craniofacial development using a developmental model, Xenopus laevis. Dyrk1a mRNA and protein was expressed throughout the developing head and was enriched in the branchial arches which contribute to the face and jaw. Consistently, reduced Dyrk1a function, using dyrk1a morpholinos and pharmacological inhibitors, resulted in orofacial malformations including hypotelorism, altered mouth shape, slanted eyes, and narrower face accompanied by smaller jaw cartilage and muscle. Inhibition of Dyrk1a function resulted in misexpression of key craniofacial regulators including transcription factors and members of the retinoic acid signaling pathway. Two such regulators, sox9 and pax3 are required for neural crest development and their decreased expression corresponds with smaller neural crest domains within the branchial arches. Finally, we determined that the smaller size of the faces, jaw elements and neural crest domains in embryos deficient in Dyrk1a could be explained by increased cell death and decreased proliferation. This study is the first to provide insight into why craniofacial birth defects might arise in humans with DYRK1A mutations.\n",
  "wordCount" : "1987",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.574730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.574730" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.574730">
        <p class="paperTitle">Sox9 links biliary maturation to branching morphogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.574730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.574730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hrncir, H. R.; Bombin, S.; Goodloe, B.; Hogan, C. B.; Jadi, O.; Gracz, A. D.</p>
        <p class="info">Score: 12.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.574730' target='https://doi.org/10.1101/2024.01.15.574730'> 10.1101/2024.01.15.574730</a></p>
        <p class="abstract">Branching morphogenesis couples cellular differentiation with development of tissue architecture. Intrahepatic bile duct (IHBD) morphogenesis is initiated with biliary epithelial cell (BEC) specification and eventually forms a heterogeneous network of large ducts and small ductules. Here, we show that Sox9 is required for developmental establishment of small ductules. IHBDs emerge as a webbed structure by E15.5 and undergo morphological maturation through 2 weeks of age. Developmental knockout of Sox9 leads to decreased postnatal branching morphogenesis, manifesting as loss of ductules in adult livers. In the absence of Sox9, BECs fail to mature and exhibit elevated TGF-{beta} signaling and Activin A. Activin A induces developmental gene expression and morphological defects in BEC organoids and represses ductule formation in postnatal livers. Our data demonstrate that adult IHBD morphology and BEC maturation is regulated by the Sox9-dependent formation of precursors to ductules during development, mediated in part by downregulation of Activin A.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575421" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575421">
        <p class="paperTitle">Environmental and molecular control of tissue-specific ionocyte differentiation in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peloggia, J.; Lush, M. E.; Tsai, Y.-Y.; Wood, C.; Piotrowski, T.</p>
        <p class="info">Score: 26.3, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575421' target='https://doi.org/10.1101/2024.01.12.575421'> 10.1101/2024.01.12.575421</a></p>
        <p class="abstract">Organisms adjust their physiology to cope with environmental fluctuations and maintain fitness. These adaptations occur via genetic changes over multiple generations or through acclimation, a set of reversible phenotypic changes that confer resilience to the individual. Aquatic organisms are subject to dramatic seasonal fluctuations in water salinity, which can affect the function of lateral line mechanosensory hair cells. To maintain hair cell function when salinity decreases, ion-regulating cells, Neuromast-associated ionocytes (Nm ionocytes), increase in number and invade lateral line neuromasts. How environmental changes trigger this adaptive differentiation of Nm ionocytes and how these cells are specified is still unknown. Here, we identify Nm ionocyte progenitors as foxi3a/foxi3b-expressing skin cells and show that their differentiation is associated with sequential activation of different Notch pathway components, which control ionocyte survival. We demonstrate that new Nm ionocytes are rapidly specified by absolute salinity levels, independently of stress response pathways. We further show that Nm ionocyte differentiation is selectively triggered by depletion of specific ions, such as Ca2&#43; and Na&#43;/Cl-, but not by low K&#43; levels, and is independent of media osmolarity. Finally, we demonstrate that hair cell activity plays a role in Nm ionocyte recruitment and that systemic factors are not necessary for Nm ionocyte induction. In summary, we have identified how environmental changes activate a signaling cascade that triggers basal skin cell progenitors to differentiate into Nm ionocytes and invade lateral line organs. This adaptive behavior is an example of physiological plasticity that may prove essential for survival in changing climates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576677" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576677">
        <p class="paperTitle">Perturbing nuclear glycosylation in the mouse preimplantation embryo slows down embryonic growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Formichetti, S.; Chitnavis, U.; Sadowska, A.; Liu, N.; Boskovic, A.; Boulard, M.</p>
        <p class="info">Score: 12.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576677' target='https://doi.org/10.1101/2024.01.22.576677'> 10.1101/2024.01.22.576677</a></p>
        <p class="abstract">The only known form of intracellular protein glycosylation (O-GlcNAc) is reversible and has been mapped on thousands of cytoplasmic and nuclear proteins, including RNA polymerase II, transcription factors and chromatin modifiers. The O-GlcNAc modification is catalyzed by a single enzyme known as O-GlcNAc Transferase (OGT), that is required for mammalian early development. Remarkably, the regulatory function of protein O-GlcNAcylation in the embryo as well as the embryonic O-GlcNAc proteome remain unknown. Here, we devised a new method to enzymatically remove O-GlcNAc from preimplantation embryonic nuclei, where it accumulates coincidently with embryonic genome activation (EGA). Unexpectedly, the depletion of nuclear O-GlcNAc to undetectable levels has no impact on EGA, but dampens the transcriptional activation of the translational machinery, and triggers a spindle checkpoint response. These molecular alterations were phenotypically associated with a developmental delay starting from early cleavage stages and persisting after embryo implantation, establishing a novel link between nuclear glycosylation and embryonic growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576696">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576696" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576696">
        <p class="paperTitle">Wnt signaling alters CTCF binding patterns and global chromatin structure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576696" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576696" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nordin, A.; Chakraborty, C.; Jonasson, M.; Dano, O.; Zambanini, G.; Pagella, P.; Remeseiro, S.; Cantu, C.</p>
        <p class="info">Score: 10.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576696' target='https://doi.org/10.1101/2024.01.22.576696'> 10.1101/2024.01.22.576696</a></p>
        <p class="abstract">Wnt signaling plays a pivotal role during development, stem cell maintenance, and tissue homeostasis. Upon Wnt pathway activation, {beta}-catenin translocates to the nucleus where it binds the TCF/LEF transcription factors to drive the context-specific expression of Wnt target genes. Coordinating gene expression programs in vertebrates requires a complex interplay between the regulatory and the 3D organization of the genome. However, the impact of Wnt signaling on genome structure has been poorly explored. Here we investigated how Wnt signaling activation influences the binding patterns of CTCF, one of the core architectural proteins that helps establish the 3D genome organization be demarcating topologically associated domains (TAD). This study uncovered a series of CTCF rearrangements under Wnt, that we termed RUW. Notably, RUW sites that were gained upon Wnt activation were typically dependent on {beta}-catenin and were characterized by both CTCF and TCF/LEF binding. Accordingly, many CTCF RUWs aligned with {beta}-catenin binding patterns, and {beta}-catenin and CTCF co-localized in vivo in discreet nuclear puncta only up-on pathway activation. Genome-wide investigation of CTCF-mediated 3D genomic interactions upon Wnt pathway stimulation supported the role of the identified RUWs in mediating Wnt-dependent chromatin loops. Lastly, targeted disruption of selected CTCF binding sites demonstrated their functional contribution to Wnt target gene regulation, implicating regulation of the 3D genomic structure in the execution of tran-scriptional programs orchestrated by developmental pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575757" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575757">
        <p class="paperTitle">A spatiotemporally resolved atlas of mRNA decay in the C. elegans embryo reveals differential regulation of mRNA stability across stages and cell types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, F.; Nordgren, C. E.; Murray, J. I.</p>
        <p class="info">Score: 11.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575757' target='https://doi.org/10.1101/2024.01.15.575757'> 10.1101/2024.01.15.575757</a></p>
        <p class="abstract">During embryonic development, cells undergo dynamic changes in gene expression that are required for appropriate cell fate specification. Although both transcription and mRNA degradation contribute to gene expression dynamics, patterns of mRNA decay are less well-understood. Here we directly measured spatiotemporally resolved mRNA decay rates transcriptome-wide throughout C. elegans embryogenesis by transcription inhibition followed by bulk and single-cell RNA-sequencing. This allowed us to calculate mRNA half-lives within specific cell types and developmental stages and identify differentially regulated mRNA decay throughout embryonic development. We identified transcript features that are correlated with mRNA stability and found that mRNA decay rates are associated with distinct peaks in gene expression over time. Moreover, we provide evidence that, on average, mRNA is more stable in the germline compared to in the soma and in later embryonic stages compared to in earlier stages. This work suggests that differential mRNA decay across cell states and time helps to shape developmental gene expression, and it provides a valuable resource for studies of mRNA turnover regulatory mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576035">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576035" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576035">
        <p class="paperTitle">Neutrophil immune profile controls spinal cord regeneration in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576035" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576035" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Sena-Tomas, C.; Lameira, L.; Naique Taborda, P.; Laborde, A.; Orger, M. B.; de Oliveira, S.; Saude, L.</p>
        <p class="info">Score: 6.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576035' target='https://doi.org/10.1101/2024.01.17.576035'> 10.1101/2024.01.17.576035</a></p>
        <p class="abstract">Spinal cord injury triggers a strong innate inflammatory response in both non-regenerative mammals and regenerative zebrafish. Neutrophils are the first immune population to be recruited to the injury site. Yet, their role in the repair process, particularly in a regenerative context, remains largely unknown. Here, we show that, following rapid recruitment to the injured spinal cord, neutrophils mostly reverse migrate throughout the zebrafish body. In addition, promoting neutrophil inflammation resolution by inhibiting Cxcr4 boosts cellular and functional regeneration. Neutrophil-specific RNA-seq analysis reveals an enhanced activation state that correlates with a transient increase in tnf- expression in macrophage/microglia populations. Conversely, blocking neutrophil recruitment through Cxcr1/2 inhibition diminishes the presence of macrophage/microglia at the injury site and impairs spinal cord regeneration. Altogether, these findings provide new insights into the role of neutrophils in spinal cord regeneration, emphasizing the significant impact of their immune profile on the outcome of the repair process.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=185 HEIGHT=200 SRC=&#34;FIGDIR/small/576035v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@30d66borg.highwire.dtl.DTLVardef@4d28a5org.highwire.dtl.DTLVardef@1a797faorg.highwire.dtl.DTLVardef@1eac5ea_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573643" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573643">
        <p class="paperTitle">Gene module reconstruction elucidates cellular differentiation processes and the regulatory logic of specialized secretion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Liu, J.; Du, L. Y.; Wyss, J. L.; Farrell, J. A.; Schier, A. F.</p>
        <p class="info">Score: 11.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573643' target='https://doi.org/10.1101/2023.12.29.573643'> 10.1101/2023.12.29.573643</a></p>
        <p class="abstract">During differentiation, cells become structurally and functionally specialized, but comprehensive views of the underlying remodeling processes are elusive. Here, we leverage scRNA-seq developmental trajectories to reconstruct differentiation using two secretory tissues as a model system - the zebrafish notochord and hatching gland. First, we present an approach to integrate expression and functional similarities for gene module identification, revealing dozens of gene modules representing known and newly associated differentiation processes and their temporal ordering. Second, we focused on the unfolded protein response (UPR) transducer module to study how general versus cell-type specific secretory functions are regulated. By profiling loss- and gain-of-function embryos, we found that the UPR transcription factors creb3l1, creb3l2, and xbp1 are master regulators of a general secretion program. creb3l1/creb3l2 additionally activate an extracellular matrix secretion program, while xbp1 partners with bhlha15 to activate a gland-specific secretion program. Our study offers a multi-source integrated approach for functional gene module identification and illustrates how transcription factors confer general and specialized cellular functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576449" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576449">
        <p class="paperTitle">Automated Cell Lineage Reconstruction using Label-Free 4D Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waliman, M.; Johnson, R. L.; Natesan, G.; Tan, S.; Santella, A.; Hong, R. L.; Shah, P. K.</p>
        <p class="info">Score: 2.6, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576449' target='https://doi.org/10.1101/2024.01.20.576449'> 10.1101/2024.01.20.576449</a></p>
        <p class="abstract">Here we describe embGAN, a deep learning pipeline that addresses the challenge of automated cell detection and tracking in label-free 3D time lapse imaging. embGAN requires no manual data annotation for training, learns robust detections that exhibits a high degree of scale invariance and generalizes well to images acquired in multiple labs on multiple instruments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566303">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566303" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566303">
        <p class="paperTitle">Cytoneme-mediated intercellular signaling in keratinocytes essential for epidermal remodeling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566303" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566303" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; He, Q.; Has, O.; Forouzesh, K.; Eom, D. S.</p>
        <p class="info">Score: 3.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566303' target='https://doi.org/10.1101/2023.11.08.566303'> 10.1101/2023.11.08.566303</a></p>
        <p class="abstract">The skin, the bodys largest organ, functions as a primary defense mechanism. Epidermal stem cells supply undifferentiated keratinocytes that differentiate as they migrate toward the outermost skin layer. Although such a replenishment process is disrupted in various human skin diseases, its underlying mechanisms remain elusive. With high-resolution live imaging and in vivo manipulations, we revealed that Notch signaling between keratinocytes is mediated by signaling filopodia called cytonemes and is essential for proper keratinocyte differentiation and proliferation. Inhibiting keratinocyte cytonemes reduces Notch within undifferentiated keratinocytes, leading to abnormal differentiation and hyperproliferation, resembling human skin disease phenotypes. Overproduction of Interleukin (IL)-17 signal, associated with skin diseases like psoriasis, induces psoriatic phenotypes via cytonemes in zebrafish. Clathrin interactor 1 (clint1), crucial for epidermal homeostasis in zebrafish and humans, regulates cytoneme extension. Our study suggests that intercellular signaling between keratinocytes via cytonemes is critical for epidermal maintenance, and its misregulation could be an origin of human skin diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575394" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575394">
        <p class="paperTitle">Dyrk1a is required for craniofacial development in Xenopus laevis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dickinson, A. J. G.; Wahl, S. E.; Litovchick, L.; Sesay, F.; Johnson, H. K.</p>
        <p class="info">Score: 2.8, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575394' target='https://doi.org/10.1101/2024.01.13.575394'> 10.1101/2024.01.13.575394</a></p>
        <p class="abstract">Loss of function mutations in the dual specificity tyrosine-phosphorylation-regulated kinase 1A (DYRK1A) gene are associated with craniofacial malformations in humans. Here we characterized the effects of deficient DYRK1A in craniofacial development using a developmental model, Xenopus laevis. Dyrk1a mRNA and protein was expressed throughout the developing head and was enriched in the branchial arches which contribute to the face and jaw. Consistently, reduced Dyrk1a function, using dyrk1a morpholinos and pharmacological inhibitors, resulted in orofacial malformations including hypotelorism, altered mouth shape, slanted eyes, and narrower face accompanied by smaller jaw cartilage and muscle. Inhibition of Dyrk1a function resulted in misexpression of key craniofacial regulators including transcription factors and members of the retinoic acid signaling pathway. Two such regulators, sox9 and pax3 are required for neural crest development and their decreased expression corresponds with smaller neural crest domains within the branchial arches. Finally, we determined that the smaller size of the faces, jaw elements and neural crest domains in embryos deficient in Dyrk1a could be explained by increased cell death and decreased proliferation. This study is the first to provide insight into why craniofacial birth defects might arise in humans with DYRK1A mutations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
