<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction
Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction
Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction
Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction\nAuthors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.",
  "keywords": [
    
  ],
  "articleBody": " Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction\nAuthors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.\nScore: 32.8, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552148\nYoung blood plasma is known to confer beneficial effects on various organs in mice and rats. However, it was not known whether plasma from young pigs rejuvenates old rat tissues at the epigenetic level; whether it alters the epigenetic clock, which is a highly accurate molecular biomarker of aging. To address this question, we developed and validated six different epigenetic clocks for rat tissues that are based on DNA methylation values derived from n=613 tissue samples. As indicated by their respective names, the rat pan-tissue clock can be applied to DNA methylation profiles from all rat tissues, while the rat brain-, liver-, and blood clocks apply to the corresponding tissue types. We also developed two epigenetic clocks that apply to both human and rat tissues by adding n=1366 human tissue samples to the training data. We employed these six rat clocks to investigate the rejuvenation effects of a porcine plasma fraction treatment in different rat tissues. The treatment more than halved the epigenetic ages of blood, heart, and liver tissue. A less pronounced, but statistically significant, rejuvenation effect could be observed in the hypothalamus. The treatment was accompanied by progressive improvement in the function of these organs as ascertained through numerous biochemical/physiological biomarkers and behavioral responses to assess cognitive functions. An immunoglobulin G (IgG) N-glycosylation pattern shift from pro-to anti-inflammatory also indicated reversal of glycan aging. Overall, this study demonstrates that a young porcine plasma-derived treatment markedly reverses aging in rats according to epigenetic clocks, IgG glycans, and other biomarkers of aging.\nSphingolipid metabolism is spatially regulated in the developing embryo by SOXE genes\nAuthors: Piacentino, M. L.; Fasse, A. J.; Camacho-Avila, A.; Grabylnikov, I.; Bronner, M. E.\nScore: 7.7, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552770\nDuring epithelial-to-mesenchymal transition (EMT), significant rearrangements occur in plasma membrane protein and lipid content that are important for membrane function and acquisition of cell motility. To gain insight into how neural crest cells regulate their lipid content at the transcriptional level during EMT, here we identify critical enhancer sequences that regulate the expression of SMPD3, a gene responsible for sphingomyelin hydrolysis to produce ceramide, which is necessary for neural crest EMT. We uncovered three enhancer regions within the first intron of the SMPD3 locus that drive reporter expression in distinct spatial and temporal domains, together collectively recapitulating the expression domains of endogenous SMPD3 within the ectodermal lineages. We further dissected one enhancer that is specifically active in the migrating neural crest. By mutating putative transcriptional input sites or knocking down upstream regulators, we find that the SoxE-family transcription factors Sox9 and Sox10 regulate the expression of SMPD3 in migrating neural crest cells. Together these results shed light on how core components of developmental gene regulatory networks interact with metabolic effector genes to control changes in membrane lipid content. HighlightsO_LISMPD3 is expressed in the neural tube, neural crest, and notochord during early development C_LIO_LISMPD3 expression is regulated by at least three intronic enhancers C_LIO_LISox10 and its binding sites are required for expression by a migratory neural crest-specific SMPD3 enhancer C_LIO_LISox10 is a positive regulator of endogenous SMPD3 expression during neural crest migration C_LI\nSingle-cell transcriptome landscape of developing fetal gonads defines somatic cell lineage specification in humans\nAuthors: Lardenois, A.; Suglia, A.; Moore, C. L.; Evrard, B.; Noel, L.; Rivaud, P.; Besson, A.; Toupin, M.; Leonard, S.; Lesne, L.; Coiffec, I.; Nef, s.; Lavoue, V.; Collin, O.; Chedotal, A.; Mazaud-Guittot, S.; Chalmel, F.; Rolland, A. D.\nScore: 4.7, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552336\nGonad development is an exciting model to study cell fate commitment. However, the specification and differentiation of somatic cell lineages within the testis and the ovary are incompletely characterized, especially in humans. In fact, a better understanding of sex determination first requires the identification of all the cell types involved and of their dynamic expression programs. Here we present a comprehensive analysis of approximately 128,000 single cells collected from 33 fetal testes and ovaries between 5 and 12 postconceptional weeks. In particular, a focused analysis of somatic cells allowed us to identify a common population of bipotential progenitors derived from the coelomic epithelium of both male and female gonads and capable of committing to either a steroidogenic or a supporting fate. Moreover, we have shown that early supporting cells, prior to further differentiation into Sertoli or granulosa cells, also give rise to the rete testis/ovarii lineage. Finally, we found that the ovary retains the capacity to feed the supporting cell pool for an extended period of time, directly from the surface epithelial cells and, bypassing the bipotential progenitor step. Altogether, our results provide an unprecedented revisiting of the human gonadal sex determination process.\nSIN-3 transcriptional coregulator maintains mitochondrial hormesis and polyamine flux\nAuthors: Palladino, F.; Giovannetti, M.; Fabrizio, P.; Nicolle, O.; Bedet, C.; Rodriguez-Palero, M.; Michaux, G.; Artal-Sanz, M.; Witting, M.\nScore: 3.7, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552272\nMitochondrial function relies on the coordinated transcription of mitochondrial and nuclear genomes to assemble respiratory chain complexes. Across species, the SIN3 coregulator influences mitochondrial functions, but how its loss impacts mitochondrial homeostasis and metabolism in the context of a whole organism is unknown. Exploring this link is important because SIN3 haploinsufficiency causes intellectual disability/autism syndromes and SIN3 plays an important role in tumor biology. Here we show that loss of C. elegans SIN-3 results in transcriptional deregulation of mitochondrial- and nuclear encoded mitochondrial genes, potentially leading to mito-nuclear imbalance. Consistent with impaired mitochondrial function, sin-3 mutants show extensive mitochondrial fragmentation by transmission electron microscopy (TEM) and in vivo imaging, and altered oxygen consumption. Metabolomic analysis of sin-3 mutant animals identifies a signature of mitochondria stress, and deregulation of methionine flux resulting in decreased S-adenosyl methionine (SAM), and increased polyamine levels. Our results identify SIN3 as a key regulator of mitochondrial dynamics and metabolic flux, with important implications for human pathologies.\nPRDM16 co-operates with LHX2 to shape the human brain\nAuthors: Suresh, V.; Bhattacharya, B.; Tshuva, R. Y.; Danan Gotthold, M.; Olender, T.; Bose, M.; Ben Zeev, B.; Smith, R. S.; Pradhan, S. J.; Tole, S.; Galande, S.; Harwell, C.; Baizabal, M.; Reiner, O.\nScore: 3.0, Published: 2023-08-12 DOI: 10.1101/2023.08.12.553065\nPRDM16 is a dynamic transcriptional regulator of various stem cell niches, including adipocytic, hematopoietic, cardiac progenitors, and neural stem cells. PRDM16 has been suggested to contribute to 1p36 deletion syndrome, one of the most prevalent subtelomeric microdeletion syndromes. We report a patient with a de novo nonsense mutation in the PRDM16 coding sequence, accompanied by lissencephaly and microcephaly features. Human stem cells were genetically modified to mimic this mutation, generating cortical organoids that exhibited altered cell cycle dynamics. RNA sequencing of cortical organoids at day 32 unveiled changes in cell adhesion and WNT-signaling pathways. ChIP-seq of PRDM16 identified binding sites in postmortem human fetal cortex, indicating the conservation of PRDM16 binding to developmental genes in mice and humans, potentially at enhancer sites. A shared motif between PRDM16 and LHX2 was identified and further examined through comparison with LHX2 ChIP-seq data from mice. These results suggested a collaborative partnership between PRDM16 and LHX2 in regulating a common set of genes and pathways in cortical radial glia cells, possibly via their synergistic involvement in cortical development.\nDynamic modes of Notch transcription hubs conferring memory and stochastic activation revealed by live imaging the co-activator Mastermind\nAuthors: deHaro- Arbona, F. J.; Roussos, C.; Baloul, S.; Townson, J. M.; Gomez-Lamarca, M.; Bray, S. J.\nScore: 2.2, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552629\nDevelopmental programming involves the accurate conversion of signaling levels and dynamics to transcriptional outputs. The transcriptional relay in the Notch pathway relies on nuclear complexes containing the coactivator Mastermind (Mam). By tracking these complexes in real time, we reveal that they promote formation of a dynamic transcription hub in Notch-ON nuclei which concentrates key factors including Mediator CDK module. The composition of the hub is labile and persists after Notch withdrawal conferring a memory that enables rapid reformation. Surprisingly, only a third of Notch-ON hubs progress to a state with nascent transcription, that correlates with Polymerase II and core Mediator recruitment. The probability is increased by a second signal. The discovery that target-gene transcription is probabilistic has far-reaching implications because it implies that stochastic differences in Notch pathway output can arise downstream of receptor activation.\nData-Driven 3D Shape Analysis Reveals Cell Shape-Fate Relationships in Zebrafish Lateral Line Neuromast\nAuthors: Hewitt, M. N.; Cruz, I.; Raible, D. W.\nScore: 2.0, Published: 2023-08-11 DOI: 10.1101/2023.08.09.552694\nCell shape is a powerful readout of cell state, fate, and function. With the advent of sophisticated microscopes, image segmentation algorithms, and numerical shape representations, it is becoming more feasible to study cell shape in developing tissues. However, few studies have analyzed cell shape in three dimensions in living, intact organisms. Here, we took advantage of the favorable imaging qualities of zebrafish lateral line neuromasts to generate a dataset of high resolution images with labeled cells and nuclei. Using a custom Python-based workflow, we performed semi-automated, 3D cell and nucleus segmentation. We then used spherical harmonics and principal components analysis to distill neuromast cell and nuclear shape variation into several interpretable, biologically meaningful parameters. We found that neuromast cell and nuclear shapes vary with cell location and identity. The distinction between hair cells and support cells was discrete and accounted for much of the variation in neuromast cell and nucleus shape, which allowed us to train classifiers to predict hair cell identity from cell and nucleus shape features. Using markers for support cell subpopulations, we found that support cell subtypes also had different shapes from each other; however, shape features did not distinguish as sharply between support cell subtypes, suggesting that support cells vary continuously in shape. To investigate the effects of genetic perturbation that results in loss of a cell type on neuromast cell shape, we examined atoh1a mutants that lack hair cells. We found that neuromasts from atoh1a mutants lacked the cell shape phenotype associated with hair cells, but did not exhibit a mutant-specific cell shape. Our results demonstrate the utility of using 3D cell shape features to characterize, compare, and classify cells in a living, developing organism.\nSclerotome-derived vascular smooth muscle progenitors contribute to the hematopoietic stem cell specification niche\nAuthors: Kelley, C. M.; Glenn, N. O.; Gays, D.; Santoro, M. M.; Clements, W. K.\nScore: 2.0, Published: 2023-08-11 DOI: 10.1101/2023.08.09.552695\nHaematopoietic stem cells (HSCs) are the self-renewing progenitors that continuously populate the haemato-immune cell lineages throughout life, and constitute the therapeutic component of bone marrow transplants. A major biomedical goal has been to understand the native specification of HSCs during embryonic development as a means to inform in vitro directed differentiation of pluripotent stem cells. Across vertebrate phyla, HSCs derive from haemogenic endothelium in the ventral floor of the primitive dorsal aorta (DA), also known as the descending aorta in mammals. Competent HSC-fated cells in the endothelium likely receive instructive signaling from neighbouring cells that constitute a \"specification niche.\" We previously showed that experimental manipulations leading to defects in the most ventral compartment of the somite, the sclerotome, are correlated with HSC defects, raising the possibility that sclerotome patterning is required for HSC specification. Here we show that in zebrafish, specific sclerotome-derived cells contact the DA immediately prior to the emergence of HSCs. These cells subsequently give rise to vascular smooth muscle cells (VSMCs). When sclerotome patterning is disrupted, VSMCs are diminished, and HSC specification fails. We conclude that sclerotome-derived VSMC progenitors contribute to the embryonic HSC specification niche, most likely by providing unknown HSC inductive signals.\nm6A mRNA Methylation Regulates Early Pancreatic β-Cell Differentiation\nAuthors: Kahraman, S.; De Jesus, D. F.; Wei, J.; Brown, N. K.; Zou, Z.; Hu, J.; He, C.; Kulkarni, R. N.\nScore: 3.0, Published: 2023-08-03 DOI: 10.1101/2023.08.03.551675\nN6-methyladenosine (m6A) is the most abundant chemical modification in mRNA, and plays important roles in human and mouse embryonic stem cell pluripotency, maintenance, and differentiation. We have recently reported, for the first time, the role of m6A in the postnatal control of {beta}-cell function in physiological states and in Type 1 and 2 Diabetes. However, the precise mechanisms by which m6A acts to regulate the development of human and mouse {beta}-cells are unexplored. Here, we show that the m6A landscape is dynamic during human pancreas development, and that METTL14, one of the m6A writer complex proteins, is essential for the early differentiation of both human and mouse {beta}-cells.\nLymphatic vessel development in human embryos\nAuthors: Yamaguchi, S.; Minamide, N.; Imai, H.; Ikeda, T.; Watanabe, M.; Imanaka-Yoshida, K.; Maruyama, K.\nScore: 1.5, Published: 2023-08-15 DOI: 10.1101/2023.08.12.553102\nLymphatic vessel development has been a subject of research for about 120 years. Studies employing mice and zebrafish models have elucidated that lymphatic endothelial cells (LECs) predominantly differentiate from venous endothelial cells via the expression of transcription factor Prospero homeobox protein 1 (Prox1), a master regulator of lymphatic vessel development. On the other hand, it has been found that LECs can also be generated from undifferentiated mesodermal or hemogenic endothelial cells, suggesting potential diversity in their origins depending on the organ or anatomical location. However, knowledge of human lymphatic vessel development remains limited. Here, we examined early lymphatic development in humans by analyzing 31 embryos and three 9-week old fetuses. We found that human embryos produce Prox1-expressing LECs in and around the cardinal veins, which converged to form initial lymph sacs. Furthermore, we also examined lymphatic vessel development in the heart, lungs, lower jaw, mesentery, intestines and kidneys. Lymphatic vessels appeared to develop at different rates in each organ and to display temporal differences in marker expression. These observation showed the possibility that there could exist different patterns of lymphatic vessel development across organs, which may reflect different cellular origins or developmental signaling in each organ. Our study elucidates early human lymphatic vessel development, which facilitates understanding of lymphatic vessel evolution and phylogenetics, and also leads to illuminate the etiology of congenital lymph-related diseases, such as lymphatic malformations.\n",
  "wordCount" : "2374",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552148">
        <p class="paperTitle">Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.</p>
        <p class="info">Score: 32.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552148' target='https://doi.org/10.1101/2023.08.06.552148'> 10.1101/2023.08.06.552148</a></p>
        <p class="abstract">Young blood plasma is known to confer beneficial effects on various organs in mice and rats. However, it was not known whether plasma from young pigs rejuvenates old rat tissues at the epigenetic level; whether it alters the epigenetic clock, which is a highly accurate molecular biomarker of aging. To address this question, we developed and validated six different epigenetic clocks for rat tissues that are based on DNA methylation values derived from n=613 tissue samples. As indicated by their respective names, the rat pan-tissue clock can be applied to DNA methylation profiles from all rat tissues, while the rat brain-, liver-, and blood clocks apply to the corresponding tissue types. We also developed two epigenetic clocks that apply to both human and rat tissues by adding n=1366 human tissue samples to the training data. We employed these six rat clocks to investigate the rejuvenation effects of a porcine plasma fraction treatment in different rat tissues. The treatment more than halved the epigenetic ages of blood, heart, and liver tissue. A less pronounced, but statistically significant, rejuvenation effect could be observed in the hypothalamus. The treatment was accompanied by progressive improvement in the function of these organs as ascertained through numerous biochemical/physiological biomarkers and behavioral responses to assess cognitive functions. An immunoglobulin G (IgG) N-glycosylation pattern shift from pro-to anti-inflammatory also indicated reversal of glycan aging. Overall, this study demonstrates that a young porcine plasma-derived treatment markedly reverses aging in rats according to epigenetic clocks, IgG glycans, and other biomarkers of aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552770">
        <p class="paperTitle">Sphingolipid metabolism is spatially regulated in the developing embryo by SOXE genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piacentino, M. L.; Fasse, A. J.; Camacho-Avila, A.; Grabylnikov, I.; Bronner, M. E.</p>
        <p class="info">Score: 7.7, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552770' target='https://doi.org/10.1101/2023.08.10.552770'> 10.1101/2023.08.10.552770</a></p>
        <p class="abstract">During epithelial-to-mesenchymal transition (EMT), significant rearrangements occur in plasma membrane protein and lipid content that are important for membrane function and acquisition of cell motility. To gain insight into how neural crest cells regulate their lipid content at the transcriptional level during EMT, here we identify critical enhancer sequences that regulate the expression of SMPD3, a gene responsible for sphingomyelin hydrolysis to produce ceramide, which is necessary for neural crest EMT. We uncovered three enhancer regions within the first intron of the SMPD3 locus that drive reporter expression in distinct spatial and temporal domains, together collectively recapitulating the expression domains of endogenous SMPD3 within the ectodermal lineages. We further dissected one enhancer that is specifically active in the migrating neural crest. By mutating putative transcriptional input sites or knocking down upstream regulators, we find that the SoxE-family transcription factors Sox9 and Sox10 regulate the expression of SMPD3 in migrating neural crest cells. Together these results shed light on how core components of developmental gene regulatory networks interact with metabolic effector genes to control changes in membrane lipid content.

HighlightsO_LISMPD3 is expressed in the neural tube, neural crest, and notochord during early development
C_LIO_LISMPD3 expression is regulated by at least three intronic enhancers
C_LIO_LISox10 and its binding sites are required for expression by a migratory neural crest-specific SMPD3 enhancer
C_LIO_LISox10 is a positive regulator of endogenous SMPD3 expression during neural crest migration
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552336">
        <p class="paperTitle">Single-cell transcriptome landscape of developing fetal gonads defines somatic cell lineage specification in humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lardenois, A.; Suglia, A.; Moore, C. L.; Evrard, B.; Noel, L.; Rivaud, P.; Besson, A.; Toupin, M.; Leonard, S.; Lesne, L.; Coiffec, I.; Nef, s.; Lavoue, V.; Collin, O.; Chedotal, A.; Mazaud-Guittot, S.; Chalmel, F.; Rolland, A. D.</p>
        <p class="info">Score: 4.7, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552336' target='https://doi.org/10.1101/2023.08.07.552336'> 10.1101/2023.08.07.552336</a></p>
        <p class="abstract">Gonad development is an exciting model to study cell fate commitment. However, the specification and differentiation of somatic cell lineages within the testis and the ovary are incompletely characterized, especially in humans. In fact, a better understanding of sex determination first requires the identification of all the cell types involved and of their dynamic expression programs. Here we present a comprehensive analysis of approximately 128,000 single cells collected from 33 fetal testes and ovaries between 5 and 12 postconceptional weeks. In particular, a focused analysis of somatic cells allowed us to identify a common population of bipotential progenitors derived from the coelomic epithelium of both male and female gonads and capable of committing to either a steroidogenic or a supporting fate. Moreover, we have shown that early supporting cells, prior to further differentiation into Sertoli or granulosa cells, also give rise to the rete testis/ovarii lineage. Finally, we found that the ovary retains the capacity to feed the supporting cell pool for an extended period of time, directly from the surface epithelial cells and, bypassing the bipotential progenitor step. Altogether, our results provide an unprecedented revisiting of the human gonadal sex determination process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552272">
        <p class="paperTitle">SIN-3 transcriptional coregulator maintains mitochondrial hormesis and polyamine flux</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Palladino, F.; Giovannetti, M.; Fabrizio, P.; Nicolle, O.; Bedet, C.; Rodriguez-Palero, M.; Michaux, G.; Artal-Sanz, M.; Witting, M.</p>
        <p class="info">Score: 3.7, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552272' target='https://doi.org/10.1101/2023.08.07.552272'> 10.1101/2023.08.07.552272</a></p>
        <p class="abstract">Mitochondrial function relies on the coordinated transcription of mitochondrial and nuclear genomes to assemble respiratory chain complexes. Across species, the SIN3 coregulator influences mitochondrial functions, but how its loss impacts mitochondrial homeostasis and metabolism in the context of a whole organism is unknown. Exploring this link is important because SIN3 haploinsufficiency causes intellectual disability/autism syndromes and SIN3 plays an important role in tumor biology. Here we show that loss of C. elegans SIN-3 results in transcriptional deregulation of mitochondrial- and nuclear encoded mitochondrial genes, potentially leading to mito-nuclear imbalance. Consistent with impaired mitochondrial function, sin-3 mutants show extensive mitochondrial fragmentation by transmission electron microscopy (TEM) and in vivo imaging, and altered oxygen consumption. Metabolomic analysis of sin-3 mutant animals identifies a signature of mitochondria stress, and deregulation of methionine flux resulting in decreased S-adenosyl methionine (SAM), and increased polyamine levels. Our results identify SIN3 as a key regulator of mitochondrial dynamics and metabolic flux, with important implications for human pathologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553065" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553065">
        <p class="paperTitle">PRDM16 co-operates with LHX2 to shape the human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suresh, V.; Bhattacharya, B.; Tshuva, R. Y.; Danan Gotthold, M.; Olender, T.; Bose, M.; Ben Zeev, B.; Smith, R. S.; Pradhan, S. J.; Tole, S.; Galande, S.; Harwell, C.; Baizabal, M.; Reiner, O.</p>
        <p class="info">Score: 3.0, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553065' target='https://doi.org/10.1101/2023.08.12.553065'> 10.1101/2023.08.12.553065</a></p>
        <p class="abstract">PRDM16 is a dynamic transcriptional regulator of various stem cell niches, including adipocytic, hematopoietic, cardiac progenitors, and neural stem cells. PRDM16 has been suggested to contribute to 1p36 deletion syndrome, one of the most prevalent subtelomeric microdeletion syndromes. We report a patient with a de novo nonsense mutation in the PRDM16 coding sequence, accompanied by lissencephaly and microcephaly features. Human stem cells were genetically modified to mimic this mutation, generating cortical organoids that exhibited altered cell cycle dynamics. RNA sequencing of cortical organoids at day 32 unveiled changes in cell adhesion and WNT-signaling pathways. ChIP-seq of PRDM16 identified binding sites in postmortem human fetal cortex, indicating the conservation of PRDM16 binding to developmental genes in mice and humans, potentially at enhancer sites. A shared motif between PRDM16 and LHX2 was identified and further examined through comparison with LHX2 ChIP-seq data from mice. These results suggested a collaborative partnership between PRDM16 and LHX2 in regulating a common set of genes and pathways in cortical radial glia cells, possibly via their synergistic involvement in cortical development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552629">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552629" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552629">
        <p class="paperTitle">Dynamic modes of Notch transcription hubs conferring memory and stochastic activation revealed by live imaging the co-activator Mastermind</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552629" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552629" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: deHaro- Arbona, F. J.; Roussos, C.; Baloul, S.; Townson, J. M.; Gomez-Lamarca, M.; Bray, S. J.</p>
        <p class="info">Score: 2.2, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552629' target='https://doi.org/10.1101/2023.08.10.552629'> 10.1101/2023.08.10.552629</a></p>
        <p class="abstract">Developmental programming involves the accurate conversion of signaling levels and dynamics to transcriptional outputs. The transcriptional relay in the Notch pathway relies on nuclear complexes containing the coactivator Mastermind (Mam). By tracking these complexes in real time, we reveal that they promote formation of a dynamic transcription hub in Notch-ON nuclei which concentrates key factors including Mediator CDK module. The composition of the hub is labile and persists after Notch withdrawal conferring a memory that enables rapid reformation. Surprisingly, only a third of Notch-ON hubs progress to a state with nascent transcription, that correlates with Polymerase II and core Mediator recruitment. The probability is increased by a second signal. The discovery that target-gene transcription is probabilistic has far-reaching implications because it implies that stochastic differences in Notch pathway output can arise downstream of receptor activation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552694">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552694" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552694">
        <p class="paperTitle">Data-Driven 3D Shape Analysis Reveals Cell Shape-Fate Relationships in Zebrafish Lateral Line Neuromast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552694" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552694" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hewitt, M. N.; Cruz, I.; Raible, D. W.</p>
        <p class="info">Score: 2.0, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552694' target='https://doi.org/10.1101/2023.08.09.552694'> 10.1101/2023.08.09.552694</a></p>
        <p class="abstract">Cell shape is a powerful readout of cell state, fate, and function. With the advent of sophisticated microscopes, image segmentation algorithms, and numerical shape representations, it is becoming more feasible to study cell shape in developing tissues. However, few studies have analyzed cell shape in three dimensions in living, intact organisms. Here, we took advantage of the favorable imaging qualities of zebrafish lateral line neuromasts to generate a dataset of high resolution images with labeled cells and nuclei. Using a custom Python-based workflow, we performed semi-automated, 3D cell and nucleus segmentation. We then used spherical harmonics and principal components analysis to distill neuromast cell and nuclear shape variation into several interpretable, biologically meaningful parameters. We found that neuromast cell and nuclear shapes vary with cell location and identity. The distinction between hair cells and support cells was discrete and accounted for much of the variation in neuromast cell and nucleus shape, which allowed us to train classifiers to predict hair cell identity from cell and nucleus shape features. Using markers for support cell subpopulations, we found that support cell subtypes also had different shapes from each other; however, shape features did not distinguish as sharply between support cell subtypes, suggesting that support cells vary continuously in shape. To investigate the effects of genetic perturbation that results in loss of a cell type on neuromast cell shape, we examined atoh1a mutants that lack hair cells. We found that neuromasts from atoh1a mutants lacked the cell shape phenotype associated with hair cells, but did not exhibit a mutant-specific cell shape. Our results demonstrate the utility of using 3D cell shape features to characterize, compare, and classify cells in a living, developing organism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552695" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552695">
        <p class="paperTitle">Sclerotome-derived vascular smooth muscle progenitors contribute to the hematopoietic stem cell specification niche</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelley, C. M.; Glenn, N. O.; Gays, D.; Santoro, M. M.; Clements, W. K.</p>
        <p class="info">Score: 2.0, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552695' target='https://doi.org/10.1101/2023.08.09.552695'> 10.1101/2023.08.09.552695</a></p>
        <p class="abstract">Haematopoietic stem cells (HSCs) are the self-renewing progenitors that continuously populate the haemato-immune cell lineages throughout life, and constitute the therapeutic component of bone marrow transplants. A major biomedical goal has been to understand the native specification of HSCs during embryonic development as a means to inform in vitro directed differentiation of pluripotent stem cells. Across vertebrate phyla, HSCs derive from haemogenic endothelium in the ventral floor of the primitive dorsal aorta (DA), also known as the descending aorta in mammals. Competent HSC-fated cells in the endothelium likely receive instructive signaling from neighbouring cells that constitute a &#34;specification niche.&#34; We previously showed that experimental manipulations leading to defects in the most ventral compartment of the somite, the sclerotome, are correlated with HSC defects, raising the possibility that sclerotome patterning is required for HSC specification. Here we show that in zebrafish, specific sclerotome-derived cells contact the DA immediately prior to the emergence of HSCs. These cells subsequently give rise to vascular smooth muscle cells (VSMCs). When sclerotome patterning is disrupted, VSMCs are diminished, and HSC specification fails. We conclude that sclerotome-derived VSMC progenitors contribute to the embryonic HSC specification niche, most likely by providing unknown HSC inductive signals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551675">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551675" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551675">
        <p class="paperTitle">m6A mRNA Methylation Regulates Early Pancreatic β-Cell Differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551675" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551675" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kahraman, S.; De Jesus, D. F.; Wei, J.; Brown, N. K.; Zou, Z.; Hu, J.; He, C.; Kulkarni, R. N.</p>
        <p class="info">Score: 3.0, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551675' target='https://doi.org/10.1101/2023.08.03.551675'> 10.1101/2023.08.03.551675</a></p>
        <p class="abstract">N6-methyladenosine (m6A) is the most abundant chemical modification in mRNA, and plays important roles in human and mouse embryonic stem cell pluripotency, maintenance, and differentiation. We have recently reported, for the first time, the role of m6A in the postnatal control of {beta}-cell function in physiological states and in Type 1 and 2 Diabetes. However, the precise mechanisms by which m6A acts to regulate the development of human and mouse {beta}-cells are unexplored. Here, we show that the m6A landscape is dynamic during human pancreas development, and that METTL14, one of the m6A writer complex proteins, is essential for the early differentiation of both human and mouse {beta}-cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553102">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553102" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553102">
        <p class="paperTitle">Lymphatic vessel development in human embryos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553102" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553102" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yamaguchi, S.; Minamide, N.; Imai, H.; Ikeda, T.; Watanabe, M.; Imanaka-Yoshida, K.; Maruyama, K.</p>
        <p class="info">Score: 1.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553102' target='https://doi.org/10.1101/2023.08.12.553102'> 10.1101/2023.08.12.553102</a></p>
        <p class="abstract">Lymphatic vessel development has been a subject of research for about 120 years. Studies employing mice and zebrafish models have elucidated that lymphatic endothelial cells (LECs) predominantly differentiate from venous endothelial cells via the expression of transcription factor Prospero homeobox protein 1 (Prox1), a master regulator of lymphatic vessel development. On the other hand, it has been found that LECs can also be generated from undifferentiated mesodermal or hemogenic endothelial cells, suggesting potential diversity in their origins depending on the organ or anatomical location. However, knowledge of human lymphatic vessel development remains limited. Here, we examined early lymphatic development in humans by analyzing 31 embryos and three 9-week old fetuses. We found that human embryos produce Prox1-expressing LECs in and around the cardinal veins, which converged to form initial lymph sacs. Furthermore, we also examined lymphatic vessel development in the heart, lungs, lower jaw, mesentery, intestines and kidneys. Lymphatic vessels appeared to develop at different rates in each organ and to display temporal differences in marker expression. These observation showed the possibility that there could exist different patterns of lymphatic vessel development across organs, which may reflect different cellular origins or developmental signaling in each organ. Our study elucidates early human lymphatic vessel development, which facilitates understanding of lymphatic vessel evolution and phylogenetics, and also leads to illuminate the etiology of congenital lymph-related diseases, such as lymphatic malformations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
