<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles
Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.
Score: 66.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233
Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles
Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.
Score: 66.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233
Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles
Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.
Score: 66.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233
Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles\nAuthors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.\nScore: 66.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233\nRecently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs.",
  "keywords": [
    
  ],
  "articleBody": " Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles\nAuthors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.\nScore: 66.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233\nRecently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs. This atlas catalogs hundreds to thousands of cell types, of which we annotated 250. This still leaves many clusters to be fully characterized, in particular in the brain. Furthermore, with single-nuclei sequencing, all information about the spatial location of the cells and of the mRNAs within these cells is lost. Here, we provide a solution to this problem. In a proof of concept study, we have applied spatial transcriptomics using a selected gene panel to pinpoint the locations of 150 mRNA species in the adult fly. This enabled us to map unknown cell types identified in the Fly Cell Atlas to their spatial locations in the brain. Additionally, spatial transcriptomics discovered interesting principles of mRNA localization in large crowded muscle cells that may spark future mechanistic investigations. Furthermore, we present a set of computational tools that will allow for easier integration of spatial transcriptomics and single-cell datasets.\nAn integrated transcriptomic cell atlas of human neural organoids\nAuthors: He, Z.; Dony, L.; Fleck, J. S.; Szalata, A.; Li, K. X.; Sliskovic, I.; Lin, H.-C.; Santel, M.; Atamian, A.; Quadrato, G.; Sun, J.; Pasca, S. P.; Camp, G.; Theis, F.; Treutlein, B.\nScore: 34.1, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561097\nNeural tissues generated from human pluripotent stem cells in vitro (known as neural organoids) are becoming useful tools to study human brain development, evolution and disease. The characterization of neural organoids using single-cell genomic methods has revealed a large diversity of neural cell types with molecular signatures similar to those observed in primary human brain tissue. However, it is unclear which domains of the human nervous system are covered by existing protocols. It is also difficult to quantitatively assess variation between protocols and the specific cell states in organoids as compared to primary counterparts. Single-cell transcriptome data from primary tissue and neural organoids derived with guided or un-guided approaches and under diverse conditions combined with large-scale integrative analyses make it now possible to address these challenges. Recent advances in computational methodology enable the generation of integrated atlases across many data sets. Here, we integrated 36 single-cell transcriptomics data sets spanning 26 protocols into one integrated human neural organoid cell atlas (HNOCA) totaling over 1.7 million cells. We harmonize cell type annotations by incorporating reference data sets from the developing human brain. By mapping to the developing human brain reference, we reveal which primary cell states have been generated in vitro, and which are under-represented. We further compare transcriptomic profiles of neuronal populations in organoids to their counterparts in the developing human brain. To support rapid organoid phenotyping and quantitative assessment of new protocols, we provide a programmatic interface to browse the atlas and query new data sets, and showcase the power of the atlas to annotate new query data sets and evaluate new organoid protocols. Taken together, the HNOCA will be useful to assess the fidelity of organoids, characterize perturbed and diseased states and facilitate protocol development in the future.\nScale-independent topological interactions drive the first fate decision in the Drosophila embryo\nAuthors: Hur, W.; Mukherjee, A.; Hayden, L.; Lu, Z.; Chao, A.; Mitchell, N. P.; Streichan, S. J.; Vergassola, M.; Di Talia, S.\nScore: 16.8, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561879\nDuring embryogenesis, the earliest cell fate decision is often linked to nuclear positioning, whose control arises from the integration of the cell cycle oscillator and associated cytoskeletal dynamics. Yet, the mechanisms that ensure that the correct number of nuclei move to the appropriate place remain poorly understood. Here, using light sheet microscopy, we show that in Drosophila embryos spindle orientation controls which nuclei migrate towards the cortex and which remains inside the embryo, thereby determining nuclear fate and the number of cells undergoing development. Combining computational methods inspired by integral geometry and manipulations of cell cycle genes, we show that spindle orientation is controlled by topological spindle-spindle interactions and not by internuclear distance. Using arguments describing the behavior of space-filling systems, we develop a theory for topological dependency in microtubule structures. Our work shows how topological interplay of microtubule mechanics can ensure robust control of density and cell fate determination.\nFin elaboration via anterior-posterior regulation by Hedgehog signaling in teleosts\nAuthors: Tanaka, Y.; Okayama, S.; Ansai, S.; Abe, G.; Tamura, K.\nScore: 9.1, Published: 2023-10-11 DOI: 10.1101/2023.10.10.557878\nFins in fishes are appendages that serve to facilitate maneuvering in water. Compared to their ancestral state, teleosts have reduced radial bones in their paired fin skeletons and have acquired elaborated, agile paired fins. We found that mutation of Hedgehog interacting protein (Hhip), encoding an antagonist of Hedgehog signaling, leads to an increase of radial bones in a localized region and replicates the ancestral state. Interestingly, the caudal fin, which has undergone a reduction of skeleton structure in teleosts, as well as the paired fins, exhibit a regional-specific branching of the 2nd hypural in hhip-/- mutant zebrafish. These results imply that Hedgehog signaling is repressed during fin development in teleosts compared to the ancestral state. Molecular genetic analyses show that shhb, one of the Sonic hedgehog (Shh) genes, encoding one of the Hedgehog ligands in teleosts, is expressed during subdivision of endochondral components in paired fin skeletal development, and that mutation of shhb leads to hypoplasia of the paired fin skeletons. Therefore, we suggest that paired and caudal fins in fishes possess a specific region susceptible to Hedgehog signaling. The reduction of radial bones by repressive regulation of Hedgehog signaling may induce fin elaboration in teleosts.\nAn in vivo drug screen reveals that cyclooxygenase 2-derived prostaglandin D2 promotes spinal cord neurogenesis.\nAuthors: Gonzalez-Llera, L.; Sobrido-Camean, D.; Quelle-Regaldie, A.; SANCHEZ, L.; Barreiro-Iglesias, A.\nScore: 8.0, Published: 2023-10-08 DOI: 10.1101/2023.10.05.561099\nThe study of neurogenesis is essential to understand fundamental developmental processes and for the development of cell replacement therapies for central nervous system disorders. Here, we designed an in vivo drug screening protocol in developing zebrafish to find new molecules and signalling pathways regulating neurogenesis in the ventral spinal cord. This unbiased drug screen revealed that 4 cyclooxygenase (COX) inhibitors reduce the generation of serotonergic interneurons in the developing spinal cord. These results fitted very nicely with available single cell RNAseq data revealing that the floor plate cells show differential expression of 1 of the 2 COX2 zebrafish genes (ptgs2a). Indeed, several selective COX2 inhibitors and 2 different morpholinos against ptgs2a also caused a significant reduction in the number of serotonergic neurons in the ventral spinal cord and led to locomotor deficits. Single cell RNAseq data and different pharmacological manipulations further revealed that COX2-floor plate-derived prostaglandin D2 promotes neurogenesis in the developing spinal cord by promoting mitotic activity in progenitor cells. Rescue experiments using a phosphodiesterase-4 inhibitor suggest that intracellular changes in cAMP levels underlie the effects of COX inhibitors on neurogenesis and locomotion. Our study provides compelling in vivo evidence showing that prostaglandin signalling promotes neurogenesis in the ventral spinal cord.\nLong-term three-dimensional high-resolution imaging of live unlabeled small intestinal organoids using low-coherence holotomography\nAuthors: Lee, M. J.; Lee, J.; Ha, J.; Kim, G.; Kim, H.-J.; Lee, S.; Koo, B.-K.; Park, Y.\nScore: 18.0, Published: 2023-09-16 DOI: 10.1101/2023.09.16.558039\nThe prevailing challenges in live unlabeled high-resolution imaging of native organoids stem from technical issues like complex sample handling and optical scattering in three-dimensional architectures. In this study, we introduce low-coherence holotomography as an advanced, label-free, quantitative imaging modality, designed to overcome related technical obstacles for long-term live imaging of 3D organoids. We successfully captured high-resolution morphological intricacies and dynamic events within mouse small intestinal organoids at a subcellular resolution. Furthermore, this method provides a unique advantage in differentiating between viable and non-viable organoids, thereby expanding its potential applications in organoid-based research.\nOpen top multi sample dual view light sheet microscope for live imaging of large multicellular systems\nAuthors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.\nScore: 100.3, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925\nMulticellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging. To bridge these wide spatiotemporal scales, we present an open top dual view and dual illumination light sheet microscope dedicated for live imaging of large specimens at single cell resolution. The novel configuration of objectives together with a flexible and customizable multi-well mounting system combines dual view with high throughput multi-position imaging. We use the newly developed microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids.\nVisualisation of gene expression within the context of tissues: an X-ray computed tomography-based multimodal approach\nAuthors: Kairiss, K.; Sokolova, N.; Zilova, L.; Schlagheck, C.; Reinhardt, R.; Baumbach, T.; Farago, T.; van de Kamp, T.; Wittbrodt, J.; Weinhardt, V.\nScore: 7.6, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561491\nThe development of an organism is orchestrated by the spatial and temporal expression of genes. Accurate visualisation of gene expression patterns in the context of the surrounding tissues offers a glimpse into the mechanisms that drive morphogenesis. We developed correlative light-sheet fluorescence microscopy and X-ray computed tomography approach to map gene expression patterns to the whole organisms 3D anatomy at cellular resolution. We show that this multimodal approach is applicable to gene expression visualised by protein-specific antibodies and fluorescence RNA in situ hybridisation, offering a detailed understanding of individual phenotypic variations in model organisms. Furthermore, the approach provides a unique possibility to identify tissues together with their 3D cellular and molecular composition in anatomically less-defined in vitro models, such as organoids. We anticipate that the visual and quantitative insights into the 3D distribution of gene expression within tissue architecture, by the multimodal approach developed here, will be equally valuable for reference atlases of model organisms development, as well as for comprehensive screens and morphogenesis studies of in vitro models.\nRapid generation of single-insertion transgenics by Tol2 transposition in zebrafish\nAuthors: Kalvaityte, M.; Gabrilaviciute, S.; Balciunas, D.\nScore: 6.7, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561928\nThe Tol2 transposable element is the most widely used transgenesis tool in zebrafish. However, its high activity almost always leads to multiple unlinked integrations of the transgenic cassette in F1 fish. Each of these transgenes is susceptible to position effects from surrounding regulatory landscape, which can lead to altered expression and, consequently, activity. Scientists therefore must strike a balance between the need to maximize reproducibility by establishing single-insertion transgenic lines and the need to complete experiments within a reasonable timeframe. In this article, we introduce a simple competitive dilution strategy for rapid generation of single-insertion transgenics. By using cry:BFP reporter plasmid as a competitor, we achieved a nearly fourfold reduction in the number of the transgene of interest (TgOI) integrations while simultaneously increasing the proportion of single-insertion F1 generation transgenics to over 50%. We also observed variations in TgOI expression among independent single-insertion transgenics, highlighting that the commonly used ubiquitous ubb promoter is susceptible to position effects. Wide application of our competitive dilution strategy will save time, reduce animal usage, and improve reproducibility of zebrafish research.\nDifferences in binding preferences for XIST partners are observed in mammals with different early pregnancy morphologies\nAuthors: Tsagakis, I.; Tinning, H.; Malo-Estepa, I.; Whitehouse, A.; O'Connell, M. J.; Forde, N.; Aspden, J.\nScore: 6.8, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560814\nIn comparison to protein-coding genes, long non-coding RNAs are poorly conserved between species at the sequence level yet they often perform conserved and essential regulatory roles. The lncRNA XIST mediates X chromosome inactivation (XCI) through the interaction of numerous proteins, most of which bind to its repeat regions. Whilst XIST is present across placental mammals, its sequence is divergent. In addition, the timing and mechanistic details of the process of XCI also varies across placental mammals. Here, we sought to determine whether XIST interactor proteins previously identified in a model mammalian species (mouse) also bind in other mammals which exhibit divergent timing of XCI (human and bovine) and differing pre-implantation embryo morphology. We show that XIST and its putative protein partners are coordinately expressed in the endometria of mammals with different pre-implantation embryo morphologies. RNA immunoprecipitations revealed that human WTAP, SPEN, hnRNPK and CIZ1 proteins bind human XIST. CIZ1-XIST binding was also detected in bovine. In both human and cow, we found CIZ1 binds to the repeat E element of XIST. However, human CIZ1 protein is not able to bind bovine repeat E, indicating a species-specific binding interaction. Together these data indicate that several of the key RNA-protein interactions with XIST are common across placental mammals, even though we observe divergence in both the RNA sequence of XIST, and early embryo morphologies. This work sheds light upon the evolution of RNA-protein binding interactions, revealing that the binding events can be conserved even when the precise mechanism of binding has changed. Statements and DeclarationsAuthors have no financial or non-financial interests that are directly or indirectly related to the work submitted for publication.\n",
  "wordCount" : "2222",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561233">
        <p class="paperTitle">Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.</p>
        <p class="info">Score: 66.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561233' target='https://doi.org/10.1101/2023.10.06.561233'> 10.1101/2023.10.06.561233</a></p>
        <p class="abstract">Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs. This atlas catalogs hundreds to thousands of cell types, of which we annotated 250. This still leaves many clusters to be fully characterized, in particular in the brain. Furthermore, with single-nuclei sequencing, all information about the spatial location of the cells and of the mRNAs within these cells is lost. Here, we provide a solution to this problem. In a proof of concept study, we have applied spatial transcriptomics using a selected gene panel to pinpoint the locations of 150 mRNA species in the adult fly. This enabled us to map unknown cell types identified in the Fly Cell Atlas to their spatial locations in the brain. Additionally, spatial transcriptomics discovered interesting principles of mRNA localization in large crowded muscle cells that may spark future mechanistic investigations. Furthermore, we present a set of computational tools that will allow for easier integration of spatial transcriptomics and single-cell datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561097">
        <p class="paperTitle">An integrated transcriptomic cell atlas of human neural organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Z.; Dony, L.; Fleck, J. S.; Szalata, A.; Li, K. X.; Sliskovic, I.; Lin, H.-C.; Santel, M.; Atamian, A.; Quadrato, G.; Sun, J.; Pasca, S. P.; Camp, G.; Theis, F.; Treutlein, B.</p>
        <p class="info">Score: 34.1, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561097' target='https://doi.org/10.1101/2023.10.05.561097'> 10.1101/2023.10.05.561097</a></p>
        <p class="abstract">Neural tissues generated from human pluripotent stem cells in vitro (known as neural organoids) are becoming useful tools to study human brain development, evolution and disease. The characterization of neural organoids using single-cell genomic methods has revealed a large diversity of neural cell types with molecular signatures similar to those observed in primary human brain tissue. However, it is unclear which domains of the human nervous system are covered by existing protocols. It is also difficult to quantitatively assess variation between protocols and the specific cell states in organoids as compared to primary counterparts. Single-cell transcriptome data from primary tissue and neural organoids derived with guided or un-guided approaches and under diverse conditions combined with large-scale integrative analyses make it now possible to address these challenges. Recent advances in computational methodology enable the generation of integrated atlases across many data sets. Here, we integrated 36 single-cell transcriptomics data sets spanning 26 protocols into one integrated human neural organoid cell atlas (HNOCA) totaling over 1.7 million cells. We harmonize cell type annotations by incorporating reference data sets from the developing human brain. By mapping to the developing human brain reference, we reveal which primary cell states have been generated in vitro, and which are under-represented. We further compare transcriptomic profiles of neuronal populations in organoids to their counterparts in the developing human brain. To support rapid organoid phenotyping and quantitative assessment of new protocols, we provide a programmatic interface to browse the atlas and query new data sets, and showcase the power of the atlas to annotate new query data sets and evaluate new organoid protocols. Taken together, the HNOCA will be useful to assess the fidelity of organoids, characterize perturbed and diseased states and facilitate protocol development in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561879">
        <p class="paperTitle">Scale-independent topological interactions drive the first fate decision in the Drosophila embryo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hur, W.; Mukherjee, A.; Hayden, L.; Lu, Z.; Chao, A.; Mitchell, N. P.; Streichan, S. J.; Vergassola, M.; Di Talia, S.</p>
        <p class="info">Score: 16.8, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561879' target='https://doi.org/10.1101/2023.10.11.561879'> 10.1101/2023.10.11.561879</a></p>
        <p class="abstract">During embryogenesis, the earliest cell fate decision is often linked to nuclear positioning, whose control arises from the integration of the cell cycle oscillator and associated cytoskeletal dynamics. Yet, the mechanisms that ensure that the correct number of nuclei move to the appropriate place remain poorly understood. Here, using light sheet microscopy, we show that in Drosophila embryos spindle orientation controls which nuclei migrate towards the cortex and which remains inside the embryo, thereby determining nuclear fate and the number of cells undergoing development. Combining computational methods inspired by integral geometry and manipulations of cell cycle genes, we show that spindle orientation is controlled by topological spindle-spindle interactions and not by internuclear distance. Using arguments describing the behavior of space-filling systems, we develop a theory for topological dependency in microtubule structures. Our work shows how topological interplay of microtubule mechanics can ensure robust control of density and cell fate determination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.557878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.557878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.557878">
        <p class="paperTitle">Fin elaboration via anterior-posterior regulation by Hedgehog signaling in teleosts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.557878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.557878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tanaka, Y.; Okayama, S.; Ansai, S.; Abe, G.; Tamura, K.</p>
        <p class="info">Score: 9.1, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.557878' target='https://doi.org/10.1101/2023.10.10.557878'> 10.1101/2023.10.10.557878</a></p>
        <p class="abstract">Fins in fishes are appendages that serve to facilitate maneuvering in water. Compared to their ancestral state, teleosts have reduced radial bones in their paired fin skeletons and have acquired elaborated, agile paired fins. We found that mutation of Hedgehog interacting protein (Hhip), encoding an antagonist of Hedgehog signaling, leads to an increase of radial bones in a localized region and replicates the ancestral state. Interestingly, the caudal fin, which has undergone a reduction of skeleton structure in teleosts, as well as the paired fins, exhibit a regional-specific branching of the 2nd hypural in hhip-/- mutant zebrafish. These results imply that Hedgehog signaling is repressed during fin development in teleosts compared to the ancestral state. Molecular genetic analyses show that shhb, one of the Sonic hedgehog (Shh) genes, encoding one of the Hedgehog ligands in teleosts, is expressed during subdivision of endochondral components in paired fin skeletal development, and that mutation of shhb leads to hypoplasia of the paired fin skeletons. Therefore, we suggest that paired and caudal fins in fishes possess a specific region susceptible to Hedgehog signaling. The reduction of radial bones by repressive regulation of Hedgehog signaling may induce fin elaboration in teleosts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561099">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561099" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561099">
        <p class="paperTitle">An in vivo drug screen reveals that cyclooxygenase 2-derived prostaglandin D2 promotes spinal cord neurogenesis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561099" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561099" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Llera, L.; Sobrido-Camean, D.; Quelle-Regaldie, A.; SANCHEZ, L.; Barreiro-Iglesias, A.</p>
        <p class="info">Score: 8.0, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561099' target='https://doi.org/10.1101/2023.10.05.561099'> 10.1101/2023.10.05.561099</a></p>
        <p class="abstract">The study of neurogenesis is essential to understand fundamental developmental processes and for the development of cell replacement therapies for central nervous system disorders. Here, we designed an in vivo drug screening protocol in developing zebrafish to find new molecules and signalling pathways regulating neurogenesis in the ventral spinal cord. This unbiased drug screen revealed that 4 cyclooxygenase (COX) inhibitors reduce the generation of serotonergic interneurons in the developing spinal cord. These results fitted very nicely with available single cell RNAseq data revealing that the floor plate cells show differential expression of 1 of the 2 COX2 zebrafish genes (ptgs2a). Indeed, several selective COX2 inhibitors and 2 different morpholinos against ptgs2a also caused a significant reduction in the number of serotonergic neurons in the ventral spinal cord and led to locomotor deficits. Single cell RNAseq data and different pharmacological manipulations further revealed that COX2-floor plate-derived prostaglandin D2 promotes neurogenesis in the developing spinal cord by promoting mitotic activity in progenitor cells. Rescue experiments using a phosphodiesterase-4 inhibitor suggest that intracellular changes in cAMP levels underlie the effects of COX inhibitors on neurogenesis and locomotion. Our study provides compelling in vivo evidence showing that prostaglandin signalling promotes neurogenesis in the ventral spinal cord.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.558039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.558039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.558039">
        <p class="paperTitle">Long-term three-dimensional high-resolution imaging of live unlabeled small intestinal organoids using low-coherence holotomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.558039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.558039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. J.; Lee, J.; Ha, J.; Kim, G.; Kim, H.-J.; Lee, S.; Koo, B.-K.; Park, Y.</p>
        <p class="info">Score: 18.0, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.558039' target='https://doi.org/10.1101/2023.09.16.558039'> 10.1101/2023.09.16.558039</a></p>
        <p class="abstract">The prevailing challenges in live unlabeled high-resolution imaging of native organoids stem from technical issues like complex sample handling and optical scattering in three-dimensional architectures. In this study, we introduce low-coherence holotomography as an advanced, label-free, quantitative imaging modality, designed to overcome related technical obstacles for long-term live imaging of 3D organoids. We successfully captured high-resolution morphological intricacies and dynamic events within mouse small intestinal organoids at a subcellular resolution. Furthermore, this method provides a unique advantage in differentiating between viable and non-viable organoids, thereby expanding its potential applications in organoid-based research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559925">
        <p class="paperTitle">Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.</p>
        <p class="info">Score: 100.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559925' target='https://doi.org/10.1101/2023.09.28.559925'> 10.1101/2023.09.28.559925</a></p>
        <p class="abstract">Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging. To bridge these wide spatiotemporal scales, we present an open top dual view and dual illumination light sheet microscope dedicated for live imaging of large specimens at single cell resolution. The novel configuration of objectives together with a flexible and customizable multi-well mounting system combines dual view with high throughput multi-position imaging. We use the newly developed microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561491">
        <p class="paperTitle">Visualisation of gene expression within the context of tissues: an X-ray computed tomography-based multimodal approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kairiss, K.; Sokolova, N.; Zilova, L.; Schlagheck, C.; Reinhardt, R.; Baumbach, T.; Farago, T.; van de Kamp, T.; Wittbrodt, J.; Weinhardt, V.</p>
        <p class="info">Score: 7.6, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561491' target='https://doi.org/10.1101/2023.10.09.561491'> 10.1101/2023.10.09.561491</a></p>
        <p class="abstract">The development of an organism is orchestrated by the spatial and temporal expression of genes. Accurate visualisation of gene expression patterns in the context of the surrounding tissues offers a glimpse into the mechanisms that drive morphogenesis. We developed correlative light-sheet fluorescence microscopy and X-ray computed tomography approach to map gene expression patterns to the whole organisms 3D anatomy at cellular resolution. We show that this multimodal approach is applicable to gene expression visualised by protein-specific antibodies and fluorescence RNA in situ hybridisation, offering a detailed understanding of individual phenotypic variations in model organisms. Furthermore, the approach provides a unique possibility to identify tissues together with their 3D cellular and molecular composition in anatomically less-defined in vitro models, such as organoids. We anticipate that the visual and quantitative insights into the 3D distribution of gene expression within tissue architecture, by the multimodal approach developed here, will be equally valuable for reference atlases of model organisms development, as well as for comprehensive screens and morphogenesis studies of in vitro models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561928">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561928" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561928">
        <p class="paperTitle">Rapid generation of single-insertion transgenics by Tol2 transposition in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561928" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561928" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalvaityte, M.; Gabrilaviciute, S.; Balciunas, D.</p>
        <p class="info">Score: 6.7, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561928' target='https://doi.org/10.1101/2023.10.11.561928'> 10.1101/2023.10.11.561928</a></p>
        <p class="abstract">The Tol2 transposable element is the most widely used transgenesis tool in zebrafish. However, its high activity almost always leads to multiple unlinked integrations of the transgenic cassette in F1 fish. Each of these transgenes is susceptible to position effects from surrounding regulatory landscape, which can lead to altered expression and, consequently, activity. Scientists therefore must strike a balance between the need to maximize reproducibility by establishing single-insertion transgenic lines and the need to complete experiments within a reasonable timeframe. In this article, we introduce a simple competitive dilution strategy for rapid generation of single-insertion transgenics. By using cry:BFP reporter plasmid as a competitor, we achieved a nearly fourfold reduction in the number of the transgene of interest (TgOI) integrations while simultaneously increasing the proportion of single-insertion F1 generation transgenics to over 50%. We also observed variations in TgOI expression among independent single-insertion transgenics, highlighting that the commonly used ubiquitous ubb promoter is susceptible to position effects. Wide application of our competitive dilution strategy will save time, reduce animal usage, and improve reproducibility of zebrafish research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560814">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560814" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560814">
        <p class="paperTitle">Differences in binding preferences for XIST partners are observed in mammals with different early pregnancy morphologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560814" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560814" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsagakis, I.; Tinning, H.; Malo-Estepa, I.; Whitehouse, A.; O&#39;Connell, M. J.; Forde, N.; Aspden, J.</p>
        <p class="info">Score: 6.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560814' target='https://doi.org/10.1101/2023.10.04.560814'> 10.1101/2023.10.04.560814</a></p>
        <p class="abstract">In comparison to protein-coding genes, long non-coding RNAs are poorly conserved between species at the sequence level yet they often perform conserved and essential regulatory roles. The lncRNA XIST mediates X chromosome inactivation (XCI) through the interaction of numerous proteins, most of which bind to its repeat regions. Whilst XIST is present across placental mammals, its sequence is divergent. In addition, the timing and mechanistic details of the process of XCI also varies across placental mammals. Here, we sought to determine whether XIST interactor proteins previously identified in a model mammalian species (mouse) also bind in other mammals which exhibit divergent timing of XCI (human and bovine) and differing pre-implantation embryo morphology. We show that XIST and its putative protein partners are coordinately expressed in the endometria of mammals with different pre-implantation embryo morphologies. RNA immunoprecipitations revealed that human WTAP, SPEN, hnRNPK and CIZ1 proteins bind human XIST. CIZ1-XIST binding was also detected in bovine. In both human and cow, we found CIZ1 binds to the repeat E element of XIST. However, human CIZ1 protein is not able to bind bovine repeat E, indicating a species-specific binding interaction. Together these data indicate that several of the key RNA-protein interactions with XIST are common across placental mammals, even though we observe divergence in both the RNA sequence of XIST, and early embryo morphologies. This work sheds light upon the evolution of RNA-protein binding interactions, revealing that the binding events can be conserved even when the precise mechanism of binding has changed.

Statements and DeclarationsAuthors have no financial or non-financial interests that are directly or indirectly related to the work submitted for publication.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
