<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Morphodynamics of human early brain organoid development
Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.
Score: 53.2, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827
Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Morphodynamics of human early brain organoid development
Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.
Score: 53.2, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827
Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Morphodynamics of human early brain organoid development
Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.
Score: 53.2, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827
Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Morphodynamics of human early brain organoid development\nAuthors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.\nScore: 53.2, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827\nBrain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development.",
  "keywords": [
    
  ],
  "articleBody": " Morphodynamics of human early brain organoid development\nAuthors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.\nScore: 53.2, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827\nBrain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrixinduced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of nontelencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.\nDynamics of chromatin accessibility during human first-trimester neurodevelopment\nAuthors: Mannens, C. C. A.; Hu, L.; Lonnerberg, P.; Schipper, M.; Reagor, C.; Li, X.; He, X.; Barker, R. A.; Sundstrom, E.; Posthuma, D.; Linnarsson, S.\nScore: 32.9, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553878\nThe human brain is capable of highly complex functions that develops through a tightly organized cascade of patterning events, expressed transcription factors and changes in chromatin accessibility. While extensive datasets exist describing gene expression across the developing brain with single-cell resolution, similar atlases of chromatin accessibility have been primarily focused on the forebrain. Here, we focus on the chromatin landscape and paired gene expression across the developing human brain to provide a comprehensive single cell atlas during the first trimester (6 - 13 post-conceptional weeks). We identified 135 clusters across half a million nuclei and using the multiomic measurements linked candidate cis-regulatory elements (cCREs) to gene expression. We found an increase in the number of accessible regions driven both by age and neuronal differentiation. Using a convolutional neural network we identified putative functional TF-binding sites in enhancers characterizing neuronal subtypes and we applied this model to cCREs upstream of ESRRB to elucidate its activation mechanism. Finally, by linking disease-associated SNPs to cCREs we validated putative pathogenic mechanisms in several diseases and identified midbrain-derived GABAergic neurons as being the most vulnerable to major depressive disorder related mutations. Together, our findings provide a higher degree of detail to some key gene regulatory mechanisms underlying the emergence of cell types during the first trimester. We anticipate this resource to be a valuable reference for future studies related to human neurodevelopment, such as identifying cell type specific enhancers that can be used for highly specific targeting in in vitro models.\nPrimate-expressed EPIREGULIN promotes basal progenitor proliferation in the developing neocortex\nAuthors: Cubillos, P.; Ditzer, N.; Kolodziejczyk, A.; Schwenk, G.; Hoffmann, J.; Schütze, T. M.; Derihaci, R. P.; Birdir, C.; Köllner, J. E. M.; Petzold, A.; Sarov, M.; Martin, U.; Long, K. R.; Wimberger, P.; Albert, M.\nScore: 17.9, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554446\nNeocortex expansion during evolution is linked to higher numbers of neurons thought to result from increased proliferative capacity and neurogenic potential of basal progenitor cells (BPs) during development. Here we show that EREG, encoding the growth factor EPIREGULIN, is expressed in the human developing neocortex and in gorilla organoids, but not in the mouse neocortex. Addition of EPIREGULIN to the mouse neocortex increases proliferation of BPs via EGFR-mediated signaling, whereas ablation of EREG in human cortical organoids reduces BP proliferation. Addition of EPIREGULIN to cortical organoids promotes a further increase in proliferation of gorilla but not human BPs. Finally, we identify putative cis-regulatory elements that may contribute to inter-species differences in EREG expression. Overall, our results suggest that species-specific expression of EPIREGULIN may contribute to increased neocortex size in primates by providing a pro-proliferative signal to BPs in the subventricular zone progenitor niche.\nHow a reaction-diffusion signal can control spinal cord regeneration in axolotls: A modelling study\nAuthors: Caliaro, V.; Peurichard, D.; Chara, O.\nScore: 15.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554065\nAxolotls are uniquely able to completely regenerate the spinal cord after amputation. The underlying governing mechanisms of this regenerative response have not yet been fully elucidated. We previously found that spinal cord regeneration is mainly driven by cell cycle acceleration of ependymal cells, recruited by a hypothetical signal propagating from the injury. However, the nature of the signal and its propagation remain unknown. In this theoretical study, we investigated whether the regeneration-inducing signal can follow a reaction-diffusion process. We developed a computational model, validated it with experimental data and showed that the signal dynamics can be understood in terms of reaction-diffusion mechanism. By developing a theory of the regenerating outgrowth in the limit of fast reaction-diffusion, we demonstrate that control of regenerative response solely relies on cell-to-signal sensitivity and the signal reaction-diffusion characteristic length. This study lays foundations for further identification of the signal controlling regeneration of the spinal cord.\nA mechanically regulated liquid-liquid phase separation of the transcriptional regulator Tono instructs muscle development\nAuthors: Zhang, X.; Avellaneda, J.; Spletter, M. L.; Lemke, S. B.; Mangeol, P.; Habermann, B. H.; Schnorrer, F.\nScore: 10.5, Published: 2023-08-28 DOI: 10.1101/2023.08.27.555003\nMuscle morphogenesis is a multi-step program, starting with myoblast fusion, followed by myotube-tendon attachment and sarcomere assembly, with subsequent sarcomere maturation, mitochondrial amplification and specialisation. The correct chronological order of these steps requires precise control of the transcriptional regulators and their effectors. How this regulation is achieved during muscle development is not well understood. In a genome-wide RNAi screen in Drosophila, we identified the BTB-zinc finger protein Tono (CG32121) as a muscle-specific transcriptional regulator. tono mutant flight muscles display severe deficits in mitochondria and sarcomere maturation, resulting in uncontrolled contractile forces causing muscle atrophy during development. Tono protein is expressed during sarcomere maturation and localises in distinct condensates in flight muscle nuclei. Interestingly, internal pressure exerted by the maturing sarcomeres deforms the muscle nuclei into elongated shapes and changes the Tono condensates, suggesting that Tono senses the mechanical status of the muscle cells. Indeed, external mechanical pressure on the muscles triggers rapid liquid-liquid phase separation of Tono utilising its BTB domain. Thus, we propose that Tono senses high mechanical pressure to in turn adapt muscle transcription specifically at the sarcomere maturation stage. Consistently, tono mutant muscles display specific defects in a transcriptional switch that represses early muscle differentiation genes and boosts late ones. We hypothesise that a similar mechano-responsive regulation mechanism may control the activity of related BTB-zinc finger proteins that, if mutated, can result in uncontrolled force production in human muscle.\nThe molecular basis of macrochaete diversification highlighted by a single-cell atlas of Bicyclus anynana butterfly pupal forewings\nAuthors: Prakash, A.; Dion, E.; Monteiro, A.\nScore: 10.5, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554425\nButterfly wings display a large diversity of cell types, including scale cells of different colors, shapes, and sizes, yet the molecular basis of such diversity is poorly understood. Scales are single-cellular projections homologous to sensory bristles in other arthropods, and different scale types are often found intermingled on the wing, alongside other sensory and epidermal cell types, making the analysis of the complete transcriptomes of each scale type difficult to achieve. To explore scale cell diversity at a transcriptomic level we employed single-cell RNA sequencing of [~]5200 large cells (\u003e6 {micro}m) from 22.5-25-hour male pupal forewings of the butterfly Bicyclus anynana. Using single-cell unsupervised, transcriptome clustering, based on shared and differentially expressed genes, followed by in situ hybridization, immunofluorescence, and CRISPR/Cas9 editing of candidate cell type specific genes, we annotate various cell types on the wing. We identify the molecular identities of non-innervated scale cells at various stages of differentiation, innervated sensory cell types, and pheromone producing glandular cells. We further show that senseless, a zinc finger transcription factor, and HR38, a hormone receptor, determine the identity, size, and color of different scale cell types and are important regulators of scale cell differentiation. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=100 SRC=\"FIGDIR/small/554425v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (29K): org.highwire.dtl.DTLVardef@1c4575eorg.highwire.dtl.DTLVardef@143f5d8org.highwire.dtl.DTLVardef@c16196org.highwire.dtl.DTLVardef@1ede21d_HPS_FORMAT_FIGEXP M_FIG C_FIG\nFOXL2 interaction with different binding partners regulates the dynamics of ovarian development\nAuthors: Migale, R.; Neumann, M.; Mitter, R.; Rafiee, M.-R.; Wood, S.; Olsen, J.; Lovell-Badge, R.\nScore: 24.2, Published: 2023-08-24 DOI: 10.1101/2023.04.14.536403\nThe transcription factor FOXL2 is required in ovarian somatic cells for female fertility. Differential timing of Foxl2 deletion, in embryonic versus adult mouse ovary, leads to distinctive outcomes suggesting different roles across development. Here, we comprehensively investigated FOXL2s role through a multi-omics approach to characterise gene expression dynamics and chromatin accessibility changes, coupled with genome-wide identification of FOXL2 targets and on-chromatin interacting partners in granulosa cells across ovarian development. We found that FOXL2 regulates more targets postnatally, through interaction with factors regulating primordial follicle activation (PFA) and steroidogenesis. Deletion of one interactor, Ubiquitin specific protease 7 (USP7), induces PFA blockage, impaired ovary development and sterility. Our datasets constitute a comprehensive resource for exploration of the molecular mechanisms of ovarian development and causes of female infertility.\nInfection with the chytrid fungus (Batrachochytrium dendrobatidis) may distort sex ratio via sex reversal but not sex-dependent mortality in a common Eurasian amphibian\nAuthors: Ujszegi, J.; Ujhegyi, N.; Balogh, E.; Miko, Z.; Kasler, A.; Hettyey, A.; Bokony, V.\nScore: 6.4, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554788\nOne of the major factors driving the currently ongoing biodiversity crisis is the anthropogenic spread of infectious diseases. Diseases can have conspicuous consequences, such as mass mortality events, but may also exert covert but similarly severe effects, such as sex ratio distortion via sex-biased mortality or sex reversal. Chytridiomycosis, caused by the fungal pathogen Batrachochytrium dendrobatidis (Bd) is among the most important threats to amphibian biodiversity. Yet, whether Bd infection can skew sex ratios in amphibians is currently unknown, although such a hidden effect may cause the already dwindling amphibian populations to collapse. To investigate this possibility, we collected common toad (Bufo bufo) tadpoles from a natural habitat in Hungary, and continuously treated them until metamorphosis with sterile Bd culture medium (control), or a liquid culture of a Hungarian or a Spanish Bd isolate. Three months after metamorphosis we dissected the individuals and diagnosed their phenotypic and genetic sex. Bd prevalence was high in animals that died during the experiment but was almost zero at sexing. Survival was generally low in the control group, but it was further lowered by the Bd treatments. We did not observe sex-dependent mortality in either treatment. However, treatment with the Spanish Bd isolate significantly increased the frequency of sex reversal: 3 out of 9 genetic females developed into phenotypic males. Based on our results, Bd infection may have the potential to affect sex ratio in common toads through female-to-male sex reversal, but future research is needed to ascertain the generality of these findings.\nVolumetric trans-scale imaging of massive quantity of heterogeneous cell populations in centimeter-wide tissue and embryo\nAuthors: Ichimura, T.; Kakizuka, T.; Sato, Y.; Itano, K.; Seiriki, K.; Hashimoto, H.; Itoga, H.; Onami, S.; Nagai, T.\nScore: 4.8, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553997\nWe established a volumetric trans-scale imaging system with an ultra-large field-of-view (FOV) that enables simultaneous observation of millions of cellular dynamics in centimeter-wide three-dimensional (3D) tissues and embryos. Using a custom-made giant lens system with a magnification of 2x and a numerical aperture (NA) of 0.25, and a CMOS camera with more than 100 megapixels, we built a trans-scale scope AMATERAS-2, and realized fluorescence imaging with a lateral spatial resolution of approximately 1.2 {micro}m across an FOV of approximately 1.5 x 1.0 cm2. The 3D resolving capability was realized through a combination of optical and computational sectioning techniques tailored for our low-power imaging system. We applied the imaging technique to 1.2 cm-wide section of mouse brain, and successfully observed various regions of the brain with sub-cellular resolution in a single FOV. We also performed time-lapse imaging of a 1-cm-wide vascular network during quail embryo development for over 24 hours, visualizing the movement of over 4.0 x 105 vascular endothelial cells and quantitatively analyzing their dynamics. Our results demonstrate the potential of this technique in accelerating production of comprehensive reference maps of all cells in organisms and tissues, which contributes to understanding developmental processes, brain functions, and pathogenesis of disease, as well as high-throughput quality check of tissues used for transplantation medicine.\nReversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction\nAuthors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.\nScore: 53.9, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552148\nYoung blood plasma is known to confer beneficial effects on various organs in mice and rats. However, it was not known whether plasma from young pigs rejuvenates old rat tissues at the epigenetic level; whether it alters the epigenetic clock, which is a highly accurate molecular biomarker of aging. To address this question, we developed and validated six different epigenetic clocks for rat tissues that are based on DNA methylation values derived from n=613 tissue samples. As indicated by their respective names, the rat pan-tissue clock can be applied to DNA methylation profiles from all rat tissues, while the rat brain-, liver-, and blood clocks apply to the corresponding tissue types. We also developed two epigenetic clocks that apply to both human and rat tissues by adding n=1366 human tissue samples to the training data. We employed these six rat clocks to investigate the rejuvenation effects of a porcine plasma fraction treatment in different rat tissues. The treatment more than halved the epigenetic ages of blood, heart, and liver tissue. A less pronounced, but statistically significant, rejuvenation effect could be observed in the hypothalamus. The treatment was accompanied by progressive improvement in the function of these organs as ascertained through numerous biochemical/physiological biomarkers and behavioral responses to assess cognitive functions. An immunoglobulin G (IgG) N-glycosylation pattern shift from pro-to anti-inflammatory also indicated reversal of glycan aging. Overall, this study demonstrates that a young porcine plasma-derived treatment markedly reverses aging in rats according to epigenetic clocks, IgG glycans, and other biomarkers of aging.\n",
  "wordCount" : "2491",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553827">
        <p class="paperTitle">Morphodynamics of human early brain organoid development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.</p>
        <p class="info">Score: 53.2, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553827' target='https://doi.org/10.1101/2023.08.21.553827'> 10.1101/2023.08.21.553827</a></p>
        <p class="abstract">Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrixinduced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of nontelencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553878">
        <p class="paperTitle">Dynamics of chromatin accessibility during human first-trimester neurodevelopment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mannens, C. C. A.; Hu, L.; Lonnerberg, P.; Schipper, M.; Reagor, C.; Li, X.; He, X.; Barker, R. A.; Sundstrom, E.; Posthuma, D.; Linnarsson, S.</p>
        <p class="info">Score: 32.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553878' target='https://doi.org/10.1101/2023.08.18.553878'> 10.1101/2023.08.18.553878</a></p>
        <p class="abstract">The human brain is capable of highly complex functions that develops through a tightly organized cascade of patterning events, expressed transcription factors and changes in chromatin accessibility. While extensive datasets exist describing gene expression across the developing brain with single-cell resolution, similar atlases of chromatin accessibility have been primarily focused on the forebrain. Here, we focus on the chromatin landscape and paired gene expression across the developing human brain to provide a comprehensive single cell atlas during the first trimester (6 - 13 post-conceptional weeks). We identified 135 clusters across half a million nuclei and using the multiomic measurements linked candidate cis-regulatory elements (cCREs) to gene expression. We found an increase in the number of accessible regions driven both by age and neuronal differentiation. Using a convolutional neural network we identified putative functional TF-binding sites in enhancers characterizing neuronal subtypes and we applied this model to cCREs upstream of ESRRB to elucidate its activation mechanism. Finally, by linking disease-associated SNPs to cCREs we validated putative pathogenic mechanisms in several diseases and identified midbrain-derived GABAergic neurons as being the most vulnerable to major depressive disorder related mutations. Together, our findings provide a higher degree of detail to some key gene regulatory mechanisms underlying the emergence of cell types during the first trimester. We anticipate this resource to be a valuable reference for future studies related to human neurodevelopment, such as identifying cell type specific enhancers that can be used for highly specific targeting in in vitro models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554446">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554446" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554446">
        <p class="paperTitle">Primate-expressed EPIREGULIN promotes basal progenitor proliferation in the developing neocortex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554446" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554446" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cubillos, P.; Ditzer, N.; Kolodziejczyk, A.; Schwenk, G.; Hoffmann, J.; Schütze, T. M.; Derihaci, R. P.; Birdir, C.; Köllner, J. E. M.; Petzold, A.; Sarov, M.; Martin, U.; Long, K. R.; Wimberger, P.; Albert, M.</p>
        <p class="info">Score: 17.9, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554446' target='https://doi.org/10.1101/2023.08.23.554446'> 10.1101/2023.08.23.554446</a></p>
        <p class="abstract">Neocortex expansion during evolution is linked to higher numbers of neurons thought to result from increased proliferative capacity and neurogenic potential of basal progenitor cells (BPs) during development. Here we show that EREG, encoding the growth factor EPIREGULIN, is expressed in the human developing neocortex and in gorilla organoids, but not in the mouse neocortex. Addition of EPIREGULIN to the mouse neocortex increases proliferation of BPs via EGFR-mediated signaling, whereas ablation of EREG in human cortical organoids reduces BP proliferation. Addition of EPIREGULIN to cortical organoids promotes a further increase in proliferation of gorilla but not human BPs. Finally, we identify putative cis-regulatory elements that may contribute to inter-species differences in EREG expression. Overall, our results suggest that species-specific expression of EPIREGULIN may contribute to increased neocortex size in primates by providing a pro-proliferative signal to BPs in the subventricular zone progenitor niche.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554065" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554065">
        <p class="paperTitle">How a reaction-diffusion signal can control spinal cord regeneration in axolotls: A modelling study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caliaro, V.; Peurichard, D.; Chara, O.</p>
        <p class="info">Score: 15.9, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554065' target='https://doi.org/10.1101/2023.08.21.554065'> 10.1101/2023.08.21.554065</a></p>
        <p class="abstract">Axolotls are uniquely able to completely regenerate the spinal cord after amputation. The underlying governing mechanisms of this regenerative response have not yet been fully elucidated. We previously found that spinal cord regeneration is mainly driven by cell cycle acceleration of ependymal cells, recruited by a hypothetical signal propagating from the injury. However, the nature of the signal and its propagation remain unknown. In this theoretical study, we investigated whether the regeneration-inducing signal can follow a reaction-diffusion process. We developed a computational model, validated it with experimental data and showed that the signal dynamics can be understood in terms of reaction-diffusion mechanism. By developing a theory of the regenerating outgrowth in the limit of fast reaction-diffusion, we demonstrate that control of regenerative response solely relies on cell-to-signal sensitivity and the signal reaction-diffusion characteristic length. This study lays foundations for further identification of the signal controlling regeneration of the spinal cord.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.555003">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.555003" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.555003">
        <p class="paperTitle">A mechanically regulated liquid-liquid phase separation of the transcriptional regulator Tono instructs muscle development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.555003" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.555003" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, X.; Avellaneda, J.; Spletter, M. L.; Lemke, S. B.; Mangeol, P.; Habermann, B. H.; Schnorrer, F.</p>
        <p class="info">Score: 10.5, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.555003' target='https://doi.org/10.1101/2023.08.27.555003'> 10.1101/2023.08.27.555003</a></p>
        <p class="abstract">Muscle morphogenesis is a multi-step program, starting with myoblast fusion, followed by myotube-tendon attachment and sarcomere assembly, with subsequent sarcomere maturation, mitochondrial amplification and specialisation. The correct chronological order of these steps requires precise control of the transcriptional regulators and their effectors. How this regulation is achieved during muscle development is not well understood. In a genome-wide RNAi screen in Drosophila, we identified the BTB-zinc finger protein Tono (CG32121) as a muscle-specific transcriptional regulator. tono mutant flight muscles display severe deficits in mitochondria and sarcomere maturation, resulting in uncontrolled contractile forces causing muscle atrophy during development. Tono protein is expressed during sarcomere maturation and localises in distinct condensates in flight muscle nuclei. Interestingly, internal pressure exerted by the maturing sarcomeres deforms the muscle nuclei into elongated shapes and changes the Tono condensates, suggesting that Tono senses the mechanical status of the muscle cells. Indeed, external mechanical pressure on the muscles triggers rapid liquid-liquid phase separation of Tono utilising its BTB domain. Thus, we propose that Tono senses high mechanical pressure to in turn adapt muscle transcription specifically at the sarcomere maturation stage. Consistently, tono mutant muscles display specific defects in a transcriptional switch that represses early muscle differentiation genes and boosts late ones. We hypothesise that a similar mechano-responsive regulation mechanism may control the activity of related BTB-zinc finger proteins that, if mutated, can result in uncontrolled force production in human muscle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554425">
        <p class="paperTitle">The molecular basis of macrochaete diversification highlighted by a single-cell atlas of Bicyclus anynana butterfly pupal forewings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prakash, A.; Dion, E.; Monteiro, A.</p>
        <p class="info">Score: 10.5, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554425' target='https://doi.org/10.1101/2023.08.23.554425'> 10.1101/2023.08.23.554425</a></p>
        <p class="abstract">Butterfly wings display a large diversity of cell types, including scale cells of different colors, shapes, and sizes, yet the molecular basis of such diversity is poorly understood. Scales are single-cellular projections homologous to sensory bristles in other arthropods, and different scale types are often found intermingled on the wing, alongside other sensory and epidermal cell types, making the analysis of the complete transcriptomes of each scale type difficult to achieve. To explore scale cell diversity at a transcriptomic level we employed single-cell RNA sequencing of [~]5200 large cells (&gt;6 {micro}m) from 22.5-25-hour male pupal forewings of the butterfly Bicyclus anynana. Using single-cell unsupervised, transcriptome clustering, based on shared and differentially expressed genes, followed by in situ hybridization, immunofluorescence, and CRISPR/Cas9 editing of candidate cell type specific genes, we annotate various cell types on the wing. We identify the molecular identities of non-innervated scale cells at various stages of differentiation, innervated sensory cell types, and pheromone producing glandular cells. We further show that senseless, a zinc finger transcription factor, and HR38, a hormone receptor, determine the identity, size, and color of different scale cell types and are important regulators of scale cell differentiation.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=100 SRC=&#34;FIGDIR/small/554425v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (29K):
org.highwire.dtl.DTLVardef@1c4575eorg.highwire.dtl.DTLVardef@143f5d8org.highwire.dtl.DTLVardef@c16196org.highwire.dtl.DTLVardef@1ede21d_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.14.536403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.14.536403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.14.536403">
        <p class="paperTitle">FOXL2 interaction with different binding partners regulates the dynamics of ovarian development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.14.536403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.14.536403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Migale, R.; Neumann, M.; Mitter, R.; Rafiee, M.-R.; Wood, S.; Olsen, J.; Lovell-Badge, R.</p>
        <p class="info">Score: 24.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.14.536403' target='https://doi.org/10.1101/2023.04.14.536403'> 10.1101/2023.04.14.536403</a></p>
        <p class="abstract">The transcription factor FOXL2 is required in ovarian somatic cells for female fertility. Differential timing of Foxl2 deletion, in embryonic versus adult mouse ovary, leads to distinctive outcomes suggesting different roles across development. Here, we comprehensively investigated FOXL2s role through a multi-omics approach to characterise gene expression dynamics and chromatin accessibility changes, coupled with genome-wide identification of FOXL2 targets and on-chromatin interacting partners in granulosa cells across ovarian development. We found that FOXL2 regulates more targets postnatally, through interaction with factors regulating primordial follicle activation (PFA) and steroidogenesis. Deletion of one interactor, Ubiquitin specific protease 7 (USP7), induces PFA blockage, impaired ovary development and sterility. Our datasets constitute a comprehensive resource for exploration of the molecular mechanisms of ovarian development and causes of female infertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554788">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554788" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554788">
        <p class="paperTitle">Infection with the chytrid fungus (Batrachochytrium dendrobatidis) may distort sex ratio via sex reversal but not sex-dependent mortality in a common Eurasian amphibian</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554788" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554788" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ujszegi, J.; Ujhegyi, N.; Balogh, E.; Miko, Z.; Kasler, A.; Hettyey, A.; Bokony, V.</p>
        <p class="info">Score: 6.4, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554788' target='https://doi.org/10.1101/2023.08.25.554788'> 10.1101/2023.08.25.554788</a></p>
        <p class="abstract">One of the major factors driving the currently ongoing biodiversity crisis is the anthropogenic spread of infectious diseases. Diseases can have conspicuous consequences, such as mass mortality events, but may also exert covert but similarly severe effects, such as sex ratio distortion via sex-biased mortality or sex reversal. Chytridiomycosis, caused by the fungal pathogen Batrachochytrium dendrobatidis (Bd) is among the most important threats to amphibian biodiversity. Yet, whether Bd infection can skew sex ratios in amphibians is currently unknown, although such a hidden effect may cause the already dwindling amphibian populations to collapse. To investigate this possibility, we collected common toad (Bufo bufo) tadpoles from a natural habitat in Hungary, and continuously treated them until metamorphosis with sterile Bd culture medium (control), or a liquid culture of a Hungarian or a Spanish Bd isolate. Three months after metamorphosis we dissected the individuals and diagnosed their phenotypic and genetic sex. Bd prevalence was high in animals that died during the experiment but was almost zero at sexing. Survival was generally low in the control group, but it was further lowered by the Bd treatments. We did not observe sex-dependent mortality in either treatment. However, treatment with the Spanish Bd isolate significantly increased the frequency of sex reversal: 3 out of 9 genetic females developed into phenotypic males. Based on our results, Bd infection may have the potential to affect sex ratio in common toads through female-to-male sex reversal, but future research is needed to ascertain the generality of these findings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553997">
        <p class="paperTitle">Volumetric trans-scale imaging of massive quantity of heterogeneous cell populations in centimeter-wide tissue and embryo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ichimura, T.; Kakizuka, T.; Sato, Y.; Itano, K.; Seiriki, K.; Hashimoto, H.; Itoga, H.; Onami, S.; Nagai, T.</p>
        <p class="info">Score: 4.8, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553997' target='https://doi.org/10.1101/2023.08.21.553997'> 10.1101/2023.08.21.553997</a></p>
        <p class="abstract">We established a volumetric trans-scale imaging system with an ultra-large field-of-view (FOV) that enables simultaneous observation of millions of cellular dynamics in centimeter-wide three-dimensional (3D) tissues and embryos. Using a custom-made giant lens system with a magnification of 2x and a numerical aperture (NA) of 0.25, and a CMOS camera with more than 100 megapixels, we built a trans-scale scope AMATERAS-2, and realized fluorescence imaging with a lateral spatial resolution of approximately 1.2 {micro}m across an FOV of approximately 1.5 x 1.0 cm2. The 3D resolving capability was realized through a combination of optical and computational sectioning techniques tailored for our low-power imaging system. We applied the imaging technique to 1.2 cm-wide section of mouse brain, and successfully observed various regions of the brain with sub-cellular resolution in a single FOV. We also performed time-lapse imaging of a 1-cm-wide vascular network during quail embryo development for over 24 hours, visualizing the movement of over 4.0 x 105 vascular endothelial cells and quantitatively analyzing their dynamics. Our results demonstrate the potential of this technique in accelerating production of comprehensive reference maps of all cells in organisms and tissues, which contributes to understanding developmental processes, brain functions, and pathogenesis of disease, as well as high-throughput quality check of tissues used for transplantation medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552148">
        <p class="paperTitle">Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.</p>
        <p class="info">Score: 53.9, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552148' target='https://doi.org/10.1101/2023.08.06.552148'> 10.1101/2023.08.06.552148</a></p>
        <p class="abstract">Young blood plasma is known to confer beneficial effects on various organs in mice and rats. However, it was not known whether plasma from young pigs rejuvenates old rat tissues at the epigenetic level; whether it alters the epigenetic clock, which is a highly accurate molecular biomarker of aging. To address this question, we developed and validated six different epigenetic clocks for rat tissues that are based on DNA methylation values derived from n=613 tissue samples. As indicated by their respective names, the rat pan-tissue clock can be applied to DNA methylation profiles from all rat tissues, while the rat brain-, liver-, and blood clocks apply to the corresponding tissue types. We also developed two epigenetic clocks that apply to both human and rat tissues by adding n=1366 human tissue samples to the training data. We employed these six rat clocks to investigate the rejuvenation effects of a porcine plasma fraction treatment in different rat tissues. The treatment more than halved the epigenetic ages of blood, heart, and liver tissue. A less pronounced, but statistically significant, rejuvenation effect could be observed in the hypothalamus. The treatment was accompanied by progressive improvement in the function of these organs as ascertained through numerous biochemical/physiological biomarkers and behavioral responses to assess cognitive functions. An immunoglobulin G (IgG) N-glycosylation pattern shift from pro-to anti-inflammatory also indicated reversal of glycan aging. Overall, this study demonstrates that a young porcine plasma-derived treatment markedly reverses aging in rats according to epigenetic clocks, IgG glycans, and other biomarkers of aging.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
