<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genetically encoded affinity reagents (GEARs): A toolkit for visualizing and manipulating endogenous protein function in vivo.
Authors: Boswell, C. W.; Hoppe, C.; Sherrard, A.; Miao, L.; Kojima, M. L.; Krishna, S.; Musaev, D.; Zhao, N.; Stasevich, T. J.; Giraldez, A. J.
Score: 35.4, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567075
Probing endogenous protein localization and function in vivo remains challenging due to laborious gene targeting and monofunctional alleles. Here, we develop a multifunctional, universal, and adaptable toolkit based on genetically encoded affinity reagents (GEARs).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Genetically encoded affinity reagents (GEARs): A toolkit for visualizing and manipulating endogenous protein function in vivo.
Authors: Boswell, C. W.; Hoppe, C.; Sherrard, A.; Miao, L.; Kojima, M. L.; Krishna, S.; Musaev, D.; Zhao, N.; Stasevich, T. J.; Giraldez, A. J.
Score: 35.4, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567075
Probing endogenous protein localization and function in vivo remains challenging due to laborious gene targeting and monofunctional alleles. Here, we develop a multifunctional, universal, and adaptable toolkit based on genetically encoded affinity reagents (GEARs)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Genetically encoded affinity reagents (GEARs): A toolkit for visualizing and manipulating endogenous protein function in vivo.
Authors: Boswell, C. W.; Hoppe, C.; Sherrard, A.; Miao, L.; Kojima, M. L.; Krishna, S.; Musaev, D.; Zhao, N.; Stasevich, T. J.; Giraldez, A. J.
Score: 35.4, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567075
Probing endogenous protein localization and function in vivo remains challenging due to laborious gene targeting and monofunctional alleles. Here, we develop a multifunctional, universal, and adaptable toolkit based on genetically encoded affinity reagents (GEARs)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Genetically encoded affinity reagents (GEARs): A toolkit for visualizing and manipulating endogenous protein function in vivo.\nAuthors: Boswell, C. W.; Hoppe, C.; Sherrard, A.; Miao, L.; Kojima, M. L.; Krishna, S.; Musaev, D.; Zhao, N.; Stasevich, T. J.; Giraldez, A. J.\nScore: 35.4, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567075\nProbing endogenous protein localization and function in vivo remains challenging due to laborious gene targeting and monofunctional alleles. Here, we develop a multifunctional, universal, and adaptable toolkit based on genetically encoded affinity reagents (GEARs).",
  "keywords": [
    
  ],
  "articleBody": " Genetically encoded affinity reagents (GEARs): A toolkit for visualizing and manipulating endogenous protein function in vivo.\nAuthors: Boswell, C. W.; Hoppe, C.; Sherrard, A.; Miao, L.; Kojima, M. L.; Krishna, S.; Musaev, D.; Zhao, N.; Stasevich, T. J.; Giraldez, A. J.\nScore: 35.4, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567075\nProbing endogenous protein localization and function in vivo remains challenging due to laborious gene targeting and monofunctional alleles. Here, we develop a multifunctional, universal, and adaptable toolkit based on genetically encoded affinity reagents (GEARs). GEARs use nanobodies and single chain variable fragments (scFv), which recognize small epitopes, enabling fluorescent visualization and selective degradation of protein targets in vivo. Furthermore, we delineate a CRISPR/Cas9-based epitope tagging pipeline to demonstrate its utility for producing knock-in alleles that have broad multifunctionality. We use GEARs to examine the native behaviour of the pioneer transcription factor Nanog and the planar cell polarity protein Vangl2 during early zebrafish development. Together, this toolkit provides a versatile system for probing and perturbing endogenous protein function while circumventing challenges associated with conventional gene targeting and is broadly available to the model organism community.\nMicroinjection, gene knockdown, and CRISPR-mediated gene knock-in in the hard coral, Astrangia poculata\nAuthors: Warner, J. F.; Besemer, R.; Schickle, A.; Changsut, I. V.; Borbee, E.; Sharp, K.; Babonis, L. S.\nScore: 33.3, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567385\nCnidarians have become valuable models for understanding many aspects of developmental biology including the evolution of body plan diversity, novel cell type specification, and regeneration. Most of our understanding of gene function during early development in cnidarians comes from a small number of experimental systems including the sea anemone, Nematostella vectensis. Few molecular tools have been developed for use in hard corals, limiting our understanding of this diverse and ecologically important clade. Here, we report the development of a suite of tools for manipulating and analyzing gene expression during early development in the northern star coral, Astrangia poculata. We present methods for gene knockdown using short hairpin RNAs, gene overexpression using exogenous mRNAs, and endogenous gene tagging using CRISPR-mediated gene knock-in. Combined with our ability to control spawning in the laboratory, these tools make A. poculata a tractable experimental system for investigative studies of coral development. Further application of these tools will enable functional analyses of embryonic patterning and morphogenesis across Anthozoa and open new frontiers in coral biology research.\nSelf-organised pattern formation in the developing neural tube by a temporal relay of BMP signalling\nAuthors: Lehr, S.; Brueckner, D. B.; Greunz-Schindler, M.; Minchington, T. G.; Merrin, J.; Hannezo, E.; Kicheva, A.\nScore: 23.3, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567070\nDeveloping tissues interpret dynamic changes in morphogen activity to generate cell type diversity. To quantitatively study BMP signalling dynamics in the vertebrate neural tube, we developed a new ES cell differentiation system tailored for growing tissues. Differentiating cells form striking self-organised patterns of dorsal neural tube cell types driven by sequential phases of BMP signalling that are observed both in vitro and in vivo. Data-driven biophysical modelling showed that these dynamics result from coupling fast negative feedback with slow positive regulation of signalling by the specification of an endogenous BMP source. Thus, in contrast to relays that propagate morphogen signalling in space, we uncover a BMP signalling relay that operates in time. This mechanism allows rapid initial concentrationsensitive response that is robustly terminated, thereby regulating balanced sequential cell type generation. Altogether, our study provides an experimental and theoretical framework to understand how signalling dynamics are exploited in developing tissues.\nOOCYTE AND EMBRYO CULTURE UNDER OIL PROFOUNDLY ALTERS EFFECTIVE CONCENTRATIONS OF SMALL MOLECULE INHIBITORS\nAuthors: Remillard Labrosse, G.; Cohen, S.; Boucher, E.; Gagnon, K.; Vasilev, F.; Mihajlovic, A. I.; FitzHarris, G.\nScore: 26.9, Published: 2023-11-10 DOI: 10.1101/2023.11.10.566607\nCulture of oocytes and embryos in media under oil is a cornerstone of fertility treatment, and extensively employed in experimental investigation of early mammalian development. It has been noted anecdotally by some that certain small molecule inhibitors might lose activity in oil-covered culture systems, presumably by drug partitioning into the oil. Here we took a pseudo-pharmacological approach to appraise this formally. Using different culture dish designs with defined media:oil volume ratios, we show that the EC50 of the widely employed microtubule poison nocodazole shifts as a function of the media:oil ratio, such that nocodazole concentrations that prevent cell division in oil-free culture fail to in oil-covered media drops. Relatively subtle changes in culture dish design lead to measurable changes in EC50. This effect is not specific to one type of culture oil, and can be readily observed both in oocyte and embryo culture experiments. We subsequently applied a similar approach to a small panel of widely employed cell cycle-related inhibitors, finding that most lose activity in standard oil-covered oocyte/embryo culture systems. Our data suggest that loss of small molecule activity in oil-covered oocyte and embryo culture is a widespread phenomenon with potentially far-reaching implications for data reproducibility, and we recommend avoiding oil-covered culture for experiments employing inhibitors/drugs wherever possible.\nGenome organization regulates nuclear pore complex formation and promotes differentiation during Drosophila oogenesis\nAuthors: Kotb, N. M.; Ulukaya, G.; Chavan, A.; Nguyen, S. C.; Proskauer, L.; Joyce, E. F.; Hasson, D.; Jagannathan, M.; Rangan, P.\nScore: 18.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567233\nGenome organization can regulate gene expression and promote cell fate transitions. The differentiation of germline stem cells (GSCs) to oocytes in Drosophila involves changes in genome organization mediated by heterochromatin and the nuclear pore complex (NPC). Heterochromatin represses germ-cell genes during differentiation and NPCs anchor these silenced genes to the nuclear periphery, maintaining silencing to allow for oocyte development. Surprisingly, we find that genome organization also contributes to NPC formation, mediated by the transcription factor Stonewall (Stwl). As GSCs differentiate, Stwl accumulates at boundaries between silenced and active gene compartments. Stwl at these boundaries plays a pivotal role in transitioning germ-cell genes into a silenced state and activating a group of oocyte genes and Nucleoporins (Nups). The upregulation of these Nups during differentiation is crucial for NPC formation and further genome organization. Thus, crosstalk between genome architecture and NPCs is essential for successful cell fate transitions.\nClock driven waves of Tbx6 expression prefigure somite boundaries\nAuthors: Venzin, O. F.; Jollivet, C.; Chiaruttini, N.; Rosspopoff, O.; Helsens, C.; Morelli, L. G.; Uriu, K.; Oates, A. C.\nScore: 19.6, Published: 2023-11-10 DOI: 10.1101/2023.11.09.566373\nThe segmented body plan of vertebrates is established during embryogenesis by periodic and sequential formation of multi-cellular structures called somites. Somitogenesis is an example of patterning by a biological oscillator, the segmentation clock, which manifests as traveling waves of oscillating Hes/Her gene expression, reiterating during the formation of each1-3. How these waves are converted into the striped Mesp gene expression pattern that prefigures morphological somite boundaries4-8 remains unclear. Here, we image this conversion in real-time at single-cell resolution in zebrafish, using light-sheet microscopy of a novel reporter of Tbx6, a key activator of Mesp expression. We observe cellular oscillations and kinematic waves of Tbx6 expression that are driven by Hes/Her genes. Tbx6 waves arrest precisely in boundary cells that eventually express Mesp, thereby prefiguring the Mesp pattern, whereas Hes/Her waves do not. Although Hes/Her oscillations began before somitogenesis9-11, the first Tbx6 wave defines the boundary cells of the anterior-most somite, forming the head-trunk interface. Our findings imply that Tbx6 acts as a genetic clutch, converting Her/Hes pacemaker waves into Mesp stripes. We propose that this clock design shields the pacemaker from external perturbations, allowing flexible and robust patterning, making it of interest for organoids and tissue-engineering.\nThe development of the adult nervous system in the annelid Owenia fusiformis\nAuthors: Carrillo-Baltodano, A. M.; Donnellan, R. D.; Williams, E. A.; Jekely, G.; Martin-Duran, J. M.\nScore: 15.2, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567050\nBackgroundThe evolutionary origins of animal nervous systems remain contentious because we still have a limited understanding of neural development in most major animal clades. Annelids -- a species-rich group with centralised nervous systems -- have played central roles in hypotheses about the origins of animal nervous systems. However, most studies have focused on adults of deeply nested species in the annelid tree. Recently, Owenia fusiformis has emerged as an informative species to reconstruct ancestral traits in Annelida, given its phylogenetic position within the sister clade to all remaining annelids. MethodsCombining immunohistochemistry of the conserved neuropeptides FVamide-lir, RYamide-lir, RGWamide-lir and MIP-lir with gene expression, we comprehensively characterise neural development from larva to adulthood in Owenia fusiformis. ResultsThe early larval nervous system comprises a neuropeptide-rich apical organ connected through peripheral nerves to a prototroch ring and the chaetal sac. There are seven sensory neurons in the prototroch. A bilobed brain forms below the apical organ and connects to the ventral nerve cord of the developing juvenile. During metamorphosis, the brain compresses, becoming ring-shaped, and the trunk nervous system develops several longitudinal cords and segmented lateral nerves. ConclusionsOur findings reveal the formation and reorganisation of the nervous system during the life cycle of O. fusiformis, an early-branching annelid. Despite its apparent neuroanatomical simplicity, this species has a diverse peptidergic nervous system, exhibiting morphological similarities with other annelids, particularly at the larval stages. Our work supports the importance of neuropeptides in animal nervous systems and the evolution of biphasic life cycles.\nThe cell cycle controls spindle architecture in Arabidopsis by modulating the augmin pathway\nAuthors: Motta, M. R.; Nedelec, F.; Woelken, E.; Saville, H.; Jacquerie, C.; Pastuglia, M.; Stolze, S. C.; Van De Slijke, E.; Chen, P.; Boettger, L.; Belcram, K.; Nakagami, H.; De Jaeger, G.; Bouchez, D.; Schnittger, A.\nScore: 8.7, Published: 2023-11-14 DOI: 10.1101/2023.11.14.567058\nTo ensure an even segregation of chromosomes during somatic cell division, eukaryotes rely on specific microtubule structures called mitotic spindles. There are, however, striking differences in overall spindle organization among eukaryotic super groups, and in particular little is known about how spindle architecture is determined in plants. As a foundation for our work, we have measured prime characteristics of Arabidopsis mitotic spindles and built a three-dimensional dynamic model of the Arabidopsis mitotic spindle using Cytosim. Next, we identified the cell-cycle regulator CYCLIN-DEPENDENT KINASE B1 (CDKB1) together with its cyclin partner CYCB3;1 as key regulators of spindle shape and organization in Arabidopsis. Loss of CDKB1 function resulted in a high number of astral microtubules that are normally absent from plant spindles, as opposed to animal ones. We identified an augmin complex member, ENDOSPERM DEFECTIVE1 (EDE1), as a substrate of the CDKB1;1-CYCB3;1 complex. A non-phosphorylatable mutant of EDE1 displayed spindles with extended pole-to-pole distance, resembling the phenotypes of cycb3;1 and cdkb1 mutants. Moreover, we found that the mutated EDE1 version associated less efficiently with spindle microtubules. Consistently, reducing the level of augmin in Cytosim simulations largely recapitulated the spindle phenotypes observed in cycb3;1 and cdkb1 mutants. Our results emphasize the importance of cell cycle-dependent phospho-control of the mitotic spindle in plant cells. They also support the validity of our computational model as a framework for the exploration of mechanisms controlling the organization of the spindle in plants and in other species.\nSnowball: a novel gene family required for developmental patterning in fruiting bodies of mushroom-forming fungi (Agaricomycetes)\nAuthors: Földi, C.; Merenyi, Z.; Balint, B.; Csernetics, A.; Miklovics, N.; Wu, H.; Hegedüs, B.; Viragh, M.; Hou, Z.; Liu, X.-B.; Galgoczy, L.; Nagy, L.\nScore: 8.0, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566867\nThe morphogenesis of sexual fruiting bodies of fungi is a complex process determined by a genetically encoded program. Fruiting bodies reached the highest complexity levels in the Agaricomycetes, yet, the underlying genetics is currently poorly known. In this work, we functionally characterized a highly conserved unannotated gene termed snb1, whose expression level increases rapidly during fruiting body initiation. According to phylogenetic analyses, orthologues of snb1 are present in almost all agaricomycetes and may represent a novel conserved gene family that plays a substantial role in fruiting body development. We disrupted snb1 using CRISPR/Cas9 in the agaricomycete model organism Coprinopsis cinerea. Snb1 deletion mutants formed unique, snowball-shaped, rudimentary fruiting bodies that could not differentiate caps, stipes and lamellae. We took advantage of this phenotype to study fruiting body differentiation using RNA-Seq analyses. This revealed differentially regulated genes and gene families that, based on wild-type RNA-Seq data, were upregulated early during development and showed tissue-specific expression, underscoring their potential role in differentiation. Taken together, the novel gene family of snb1 and the differentially expressed genes in the snb1 mutants provide valuable insights into the complex mechanisms underlying developmental patterning in the Agaricomycetes.\nRediscovering the Rete Ovarii: a secreting auxiliary structure to the ovary\nAuthors: Anbarci, D. N.; McKey, J.; Levic, D. S.; Bagnat, M.; Capel, B.\nScore: 6.3, Published: 2023-11-11 DOI: 10.1101/2023.11.08.566085\nThe rete ovarii (RO) is an appendage of the ovary that has been given little attention. Although the RO appears in drawings of the ovary in early versions of Grays Anatomy, it disappeared from recent textbooks, and is often dismissed as a functionless vestige in the adult ovary. Using PAX8 immunostaining and confocal microscopy, we characterized the fetal development of the RO in the context of the ovary. The RO consists of three distinct regions that persist in adult life, the intraovarian rete (IOR), the extraovarian rete (EOR), and the connecting rete (CR). While the cells of the IOR appear to form solid cords within the ovary, the EOR rapidly develops into a convoluted tubular epithelium ending in a distal dilated tip. Cells of the EOR are ciliated and exhibit cellular trafficking capabilities. The CR, connecting the EOR to the IOR, gradually acquires tubular epithelial characteristics by birth. Using microinjections into the distal dilated tip of the EOR, we found that luminal contents flow towards the ovary. Mass spectrometry revealed that the EOR lumen contains secreted proteins potentially important for ovarian function. We show that the cells of the EOR are closely associated with vasculature and macrophages, and are contacted by neuronal projections. The direct proximity of the RO to the ovary and its integration with the extraovarian landscape suggest that it plays an important role in ovary development and homeostasis.\n",
  "wordCount" : "2311",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567075">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567075" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567075">
        <p class="paperTitle">Genetically encoded affinity reagents (GEARs): A toolkit for visualizing and manipulating endogenous protein function in vivo.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567075" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567075" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boswell, C. W.; Hoppe, C.; Sherrard, A.; Miao, L.; Kojima, M. L.; Krishna, S.; Musaev, D.; Zhao, N.; Stasevich, T. J.; Giraldez, A. J.</p>
        <p class="info">Score: 35.4, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567075' target='https://doi.org/10.1101/2023.11.15.567075'> 10.1101/2023.11.15.567075</a></p>
        <p class="abstract">Probing endogenous protein localization and function in vivo remains challenging due to laborious gene targeting and monofunctional alleles. Here, we develop a multifunctional, universal, and adaptable toolkit based on genetically encoded affinity reagents (GEARs). GEARs use nanobodies and single chain variable fragments (scFv), which recognize small epitopes, enabling fluorescent visualization and selective degradation of protein targets in vivo. Furthermore, we delineate a CRISPR/Cas9-based epitope tagging pipeline to demonstrate its utility for producing knock-in alleles that have broad multifunctionality. We use GEARs to examine the native behaviour of the pioneer transcription factor Nanog and the planar cell polarity protein Vangl2 during early zebrafish development. Together, this toolkit provides a versatile system for probing and perturbing endogenous protein function while circumventing challenges associated with conventional gene targeting and is broadly available to the model organism community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567385">
        <p class="paperTitle">Microinjection, gene knockdown, and CRISPR-mediated gene knock-in in the hard coral, Astrangia poculata</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Warner, J. F.; Besemer, R.; Schickle, A.; Changsut, I. V.; Borbee, E.; Sharp, K.; Babonis, L. S.</p>
        <p class="info">Score: 33.3, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567385' target='https://doi.org/10.1101/2023.11.16.567385'> 10.1101/2023.11.16.567385</a></p>
        <p class="abstract">Cnidarians have become valuable models for understanding many aspects of developmental biology including the evolution of body plan diversity, novel cell type specification, and regeneration. Most of our understanding of gene function during early development in cnidarians comes from a small number of experimental systems including the sea anemone, Nematostella vectensis. Few molecular tools have been developed for use in hard corals, limiting our understanding of this diverse and ecologically important clade. Here, we report the development of a suite of tools for manipulating and analyzing gene expression during early development in the northern star coral, Astrangia poculata. We present methods for gene knockdown using short hairpin RNAs, gene overexpression using exogenous mRNAs, and endogenous gene tagging using CRISPR-mediated gene knock-in. Combined with our ability to control spawning in the laboratory, these tools make A. poculata a tractable experimental system for investigative studies of coral development. Further application of these tools will enable functional analyses of embryonic patterning and morphogenesis across Anthozoa and open new frontiers in coral biology research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567070" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567070">
        <p class="paperTitle">Self-organised pattern formation in the developing neural tube by a temporal relay of BMP signalling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lehr, S.; Brueckner, D. B.; Greunz-Schindler, M.; Minchington, T. G.; Merrin, J.; Hannezo, E.; Kicheva, A.</p>
        <p class="info">Score: 23.3, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567070' target='https://doi.org/10.1101/2023.11.15.567070'> 10.1101/2023.11.15.567070</a></p>
        <p class="abstract">Developing tissues interpret dynamic changes in morphogen activity to generate cell type diversity. To quantitatively study BMP signalling dynamics in the vertebrate neural tube, we developed a new ES cell differentiation system tailored for growing tissues. Differentiating cells form striking self-organised patterns of dorsal neural tube cell types driven by sequential phases of BMP signalling that are observed both in vitro and in vivo. Data-driven biophysical modelling showed that these dynamics result from coupling fast negative feedback with slow positive regulation of signalling by the specification of an endogenous BMP source. Thus, in contrast to relays that propagate morphogen signalling in space, we uncover a BMP signalling relay that operates in time. This mechanism allows rapid initial concentrationsensitive response that is robustly terminated, thereby regulating balanced sequential cell type generation. Altogether, our study provides an experimental and theoretical framework to understand how signalling dynamics are exploited in developing tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566607" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566607">
        <p class="paperTitle">OOCYTE AND EMBRYO CULTURE UNDER OIL PROFOUNDLY ALTERS EFFECTIVE CONCENTRATIONS OF SMALL MOLECULE INHIBITORS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Remillard Labrosse, G.; Cohen, S.; Boucher, E.; Gagnon, K.; Vasilev, F.; Mihajlovic, A. I.; FitzHarris, G.</p>
        <p class="info">Score: 26.9, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566607' target='https://doi.org/10.1101/2023.11.10.566607'> 10.1101/2023.11.10.566607</a></p>
        <p class="abstract">Culture of oocytes and embryos in media under oil is a cornerstone of fertility treatment, and extensively employed in experimental investigation of early mammalian development. It has been noted anecdotally by some that certain small molecule inhibitors might lose activity in oil-covered culture systems, presumably by drug partitioning into the oil. Here we took a pseudo-pharmacological approach to appraise this formally. Using different culture dish designs with defined media:oil volume ratios, we show that the EC50 of the widely employed microtubule poison nocodazole shifts as a function of the media:oil ratio, such that nocodazole concentrations that prevent cell division in oil-free culture fail to in oil-covered media drops. Relatively subtle changes in culture dish design lead to measurable changes in EC50. This effect is not specific to one type of culture oil, and can be readily observed both in oocyte and embryo culture experiments. We subsequently applied a similar approach to a small panel of widely employed cell cycle-related inhibitors, finding that most lose activity in standard oil-covered oocyte/embryo culture systems. Our data suggest that loss of small molecule activity in oil-covered oocyte and embryo culture is a widespread phenomenon with potentially far-reaching implications for data reproducibility, and we recommend avoiding oil-covered culture for experiments employing inhibitors/drugs wherever possible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567233">
        <p class="paperTitle">Genome organization regulates nuclear pore complex formation and promotes differentiation during Drosophila oogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kotb, N. M.; Ulukaya, G.; Chavan, A.; Nguyen, S. C.; Proskauer, L.; Joyce, E. F.; Hasson, D.; Jagannathan, M.; Rangan, P.</p>
        <p class="info">Score: 18.2, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567233' target='https://doi.org/10.1101/2023.11.15.567233'> 10.1101/2023.11.15.567233</a></p>
        <p class="abstract">Genome organization can regulate gene expression and promote cell fate transitions. The differentiation of germline stem cells (GSCs) to oocytes in Drosophila involves changes in genome organization mediated by heterochromatin and the nuclear pore complex (NPC). Heterochromatin represses germ-cell genes during differentiation and NPCs anchor these silenced genes to the nuclear periphery, maintaining silencing to allow for oocyte development. Surprisingly, we find that genome organization also contributes to NPC formation, mediated by the transcription factor Stonewall (Stwl). As GSCs differentiate, Stwl accumulates at boundaries between silenced and active gene compartments. Stwl at these boundaries plays a pivotal role in transitioning germ-cell genes into a silenced state and activating a group of oocyte genes and Nucleoporins (Nups). The upregulation of these Nups during differentiation is crucial for NPC formation and further genome organization. Thus, crosstalk between genome architecture and NPCs is essential for successful cell fate transitions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566373">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566373" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566373">
        <p class="paperTitle">Clock driven waves of Tbx6 expression prefigure somite boundaries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566373" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566373" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venzin, O. F.; Jollivet, C.; Chiaruttini, N.; Rosspopoff, O.; Helsens, C.; Morelli, L. G.; Uriu, K.; Oates, A. C.</p>
        <p class="info">Score: 19.6, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566373' target='https://doi.org/10.1101/2023.11.09.566373'> 10.1101/2023.11.09.566373</a></p>
        <p class="abstract">The segmented body plan of vertebrates is established during embryogenesis by periodic and sequential formation of multi-cellular structures called somites. Somitogenesis is an example of patterning by a biological oscillator, the segmentation clock, which manifests as traveling waves of oscillating Hes/Her gene expression, reiterating during the formation of each1-3. How these waves are converted into the striped Mesp gene expression pattern that prefigures morphological somite boundaries4-8 remains unclear. Here, we image this conversion in real-time at single-cell resolution in zebrafish, using light-sheet microscopy of a novel reporter of Tbx6, a key activator of Mesp expression. We observe cellular oscillations and kinematic waves of Tbx6 expression that are driven by Hes/Her genes. Tbx6 waves arrest precisely in boundary cells that eventually express Mesp, thereby prefiguring the Mesp pattern, whereas Hes/Her waves do not. Although Hes/Her oscillations began before somitogenesis9-11, the first Tbx6 wave defines the boundary cells of the anterior-most somite, forming the head-trunk interface. Our findings imply that Tbx6 acts as a genetic clutch, converting Her/Hes pacemaker waves into Mesp stripes. We propose that this clock design shields the pacemaker from external perturbations, allowing flexible and robust patterning, making it of interest for organoids and tissue-engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567050">
        <p class="paperTitle">The development of the adult nervous system in the annelid Owenia fusiformis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrillo-Baltodano, A. M.; Donnellan, R. D.; Williams, E. A.; Jekely, G.; Martin-Duran, J. M.</p>
        <p class="info">Score: 15.2, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567050' target='https://doi.org/10.1101/2023.11.14.567050'> 10.1101/2023.11.14.567050</a></p>
        <p class="abstract">BackgroundThe evolutionary origins of animal nervous systems remain contentious because we still have a limited understanding of neural development in most major animal clades. Annelids -- a species-rich group with centralised nervous systems -- have played central roles in hypotheses about the origins of animal nervous systems. However, most studies have focused on adults of deeply nested species in the annelid tree. Recently, Owenia fusiformis has emerged as an informative species to reconstruct ancestral traits in Annelida, given its phylogenetic position within the sister clade to all remaining annelids.

MethodsCombining immunohistochemistry of the conserved neuropeptides FVamide-lir, RYamide-lir, RGWamide-lir and MIP-lir with gene expression, we comprehensively characterise neural development from larva to adulthood in Owenia fusiformis.

ResultsThe early larval nervous system comprises a neuropeptide-rich apical organ connected through peripheral nerves to a prototroch ring and the chaetal sac. There are seven sensory neurons in the prototroch. A bilobed brain forms below the apical organ and connects to the ventral nerve cord of the developing juvenile. During metamorphosis, the brain compresses, becoming ring-shaped, and the trunk nervous system develops several longitudinal cords and segmented lateral nerves.

ConclusionsOur findings reveal the formation and reorganisation of the nervous system during the life cycle of O. fusiformis, an early-branching annelid. Despite its apparent neuroanatomical simplicity, this species has a diverse peptidergic nervous system, exhibiting morphological similarities with other annelids, particularly at the larval stages. Our work supports the importance of neuropeptides in animal nervous systems and the evolution of biphasic life cycles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567058">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567058" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567058">
        <p class="paperTitle">The cell cycle controls spindle architecture in Arabidopsis by modulating the augmin pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567058" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567058" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Motta, M. R.; Nedelec, F.; Woelken, E.; Saville, H.; Jacquerie, C.; Pastuglia, M.; Stolze, S. C.; Van De Slijke, E.; Chen, P.; Boettger, L.; Belcram, K.; Nakagami, H.; De Jaeger, G.; Bouchez, D.; Schnittger, A.</p>
        <p class="info">Score: 8.7, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567058' target='https://doi.org/10.1101/2023.11.14.567058'> 10.1101/2023.11.14.567058</a></p>
        <p class="abstract">To ensure an even segregation of chromosomes during somatic cell division, eukaryotes rely on specific microtubule structures called mitotic spindles. There are, however, striking differences in overall spindle organization among eukaryotic super groups, and in particular little is known about how spindle architecture is determined in plants. As a foundation for our work, we have measured prime characteristics of Arabidopsis mitotic spindles and built a three-dimensional dynamic model of the Arabidopsis mitotic spindle using Cytosim. Next, we identified the cell-cycle regulator CYCLIN-DEPENDENT KINASE B1 (CDKB1) together with its cyclin partner CYCB3;1 as key regulators of spindle shape and organization in Arabidopsis. Loss of CDKB1 function resulted in a high number of astral microtubules that are normally absent from plant spindles, as opposed to animal ones. We identified an augmin complex member, ENDOSPERM DEFECTIVE1 (EDE1), as a substrate of the CDKB1;1-CYCB3;1 complex. A non-phosphorylatable mutant of EDE1 displayed spindles with extended pole-to-pole distance, resembling the phenotypes of cycb3;1 and cdkb1 mutants. Moreover, we found that the mutated EDE1 version associated less efficiently with spindle microtubules. Consistently, reducing the level of augmin in Cytosim simulations largely recapitulated the spindle phenotypes observed in cycb3;1 and cdkb1 mutants. Our results emphasize the importance of cell cycle-dependent phospho-control of the mitotic spindle in plant cells. They also support the validity of our computational model as a framework for the exploration of mechanisms controlling the organization of the spindle in plants and in other species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566867">
        <p class="paperTitle">Snowball: a novel gene family required for developmental patterning in fruiting bodies of mushroom-forming fungi (Agaricomycetes)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Földi, C.; Merenyi, Z.; Balint, B.; Csernetics, A.; Miklovics, N.; Wu, H.; Hegedüs, B.; Viragh, M.; Hou, Z.; Liu, X.-B.; Galgoczy, L.; Nagy, L.</p>
        <p class="info">Score: 8.0, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566867' target='https://doi.org/10.1101/2023.11.13.566867'> 10.1101/2023.11.13.566867</a></p>
        <p class="abstract">The morphogenesis of sexual fruiting bodies of fungi is a complex process determined by a genetically encoded program. Fruiting bodies reached the highest complexity levels in the Agaricomycetes, yet, the underlying genetics is currently poorly known. In this work, we functionally characterized a highly conserved unannotated gene termed snb1, whose expression level increases rapidly during fruiting body initiation. According to phylogenetic analyses, orthologues of snb1 are present in almost all agaricomycetes and may represent a novel conserved gene family that plays a substantial role in fruiting body development. We disrupted snb1 using CRISPR/Cas9 in the agaricomycete model organism Coprinopsis cinerea. Snb1 deletion mutants formed unique, snowball-shaped, rudimentary fruiting bodies that could not differentiate caps, stipes and lamellae. We took advantage of this phenotype to study fruiting body differentiation using RNA-Seq analyses. This revealed differentially regulated genes and gene families that, based on wild-type RNA-Seq data, were upregulated early during development and showed tissue-specific expression, underscoring their potential role in differentiation. Taken together, the novel gene family of snb1 and the differentially expressed genes in the snb1 mutants provide valuable insights into the complex mechanisms underlying developmental patterning in the Agaricomycetes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566085">
        <p class="paperTitle">Rediscovering the Rete Ovarii: a secreting auxiliary structure to the ovary</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anbarci, D. N.; McKey, J.; Levic, D. S.; Bagnat, M.; Capel, B.</p>
        <p class="info">Score: 6.3, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566085' target='https://doi.org/10.1101/2023.11.08.566085'> 10.1101/2023.11.08.566085</a></p>
        <p class="abstract">The rete ovarii (RO) is an appendage of the ovary that has been given little attention. Although the RO appears in drawings of the ovary in early versions of Grays Anatomy, it disappeared from recent textbooks, and is often dismissed as a functionless vestige in the adult ovary. Using PAX8 immunostaining and confocal microscopy, we characterized the fetal development of the RO in the context of the ovary. The RO consists of three distinct regions that persist in adult life, the intraovarian rete (IOR), the extraovarian rete (EOR), and the connecting rete (CR). While the cells of the IOR appear to form solid cords within the ovary, the EOR rapidly develops into a convoluted tubular epithelium ending in a distal dilated tip. Cells of the EOR are ciliated and exhibit cellular trafficking capabilities. The CR, connecting the EOR to the IOR, gradually acquires tubular epithelial characteristics by birth. Using microinjections into the distal dilated tip of the EOR, we found that luminal contents flow towards the ovary. Mass spectrometry revealed that the EOR lumen contains secreted proteins potentially important for ovarian function. We show that the cells of the EOR are closely associated with vasculature and macrophages, and are contacted by neuronal projections. The direct proximity of the RO to the ovary and its integration with the extraovarian landscape suggest that it plays an important role in ovary development and homeostasis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
