<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles
Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.
Score: 65.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233
Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles
Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.
Score: 65.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233
Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles
Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.
Score: 65.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233
Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles\nAuthors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.\nScore: 65.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233\nRecently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs.",
  "keywords": [
    
  ],
  "articleBody": " Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles\nAuthors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.\nScore: 65.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233\nRecently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs. This atlas catalogs hundreds to thousands of cell types, of which we annotated 250. This still leaves many clusters to be fully characterized, in particular in the brain. Furthermore, with single-nuclei sequencing, all information about the spatial location of the cells and of the mRNAs within these cells is lost. Here, we provide a solution to this problem. In a proof of concept study, we have applied spatial transcriptomics using a selected gene panel to pinpoint the locations of 150 mRNA species in the adult fly. This enabled us to map unknown cell types identified in the Fly Cell Atlas to their spatial locations in the brain. Additionally, spatial transcriptomics discovered interesting principles of mRNA localization in large crowded muscle cells that may spark future mechanistic investigations. Furthermore, we present a set of computational tools that will allow for easier integration of spatial transcriptomics and single-cell datasets.\nAn integrated transcriptomic cell atlas of human neural organoids\nAuthors: He, Z.; Dony, L.; Fleck, J. S.; Szalata, A.; Li, K. X.; Sliskovic, I.; Lin, H.-C.; Santel, M.; Atamian, A.; Quadrato, G.; Sun, J.; Pasca, S. P.; Camp, G.; Theis, F.; Treutlein, B.\nScore: 34.4, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561097\nNeural tissues generated from human pluripotent stem cells in vitro (known as neural organoids) are becoming useful tools to study human brain development, evolution and disease. The characterization of neural organoids using single-cell genomic methods has revealed a large diversity of neural cell types with molecular signatures similar to those observed in primary human brain tissue. However, it is unclear which domains of the human nervous system are covered by existing protocols. It is also difficult to quantitatively assess variation between protocols and the specific cell states in organoids as compared to primary counterparts. Single-cell transcriptome data from primary tissue and neural organoids derived with guided or un-guided approaches and under diverse conditions combined with large-scale integrative analyses make it now possible to address these challenges. Recent advances in computational methodology enable the generation of integrated atlases across many data sets. Here, we integrated 36 single-cell transcriptomics data sets spanning 26 protocols into one integrated human neural organoid cell atlas (HNOCA) totaling over 1.7 million cells. We harmonize cell type annotations by incorporating reference data sets from the developing human brain. By mapping to the developing human brain reference, we reveal which primary cell states have been generated in vitro, and which are under-represented. We further compare transcriptomic profiles of neuronal populations in organoids to their counterparts in the developing human brain. To support rapid organoid phenotyping and quantitative assessment of new protocols, we provide a programmatic interface to browse the atlas and query new data sets, and showcase the power of the atlas to annotate new query data sets and evaluate new organoid protocols. Taken together, the HNOCA will be useful to assess the fidelity of organoids, characterize perturbed and diseased states and facilitate protocol development in the future.\nOpen top multi sample dual view light sheet microscope for live imaging of large multicellular systems\nAuthors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.\nScore: 98.1, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925\nMulticellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging. To bridge these wide spatiotemporal scales, we present an open top dual view and dual illumination light sheet microscope dedicated for live imaging of large specimens at single cell resolution. The novel configuration of objectives together with a flexible and customizable multi-well mounting system combines dual view with high throughput multi-position imaging. We use the newly developed microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids.\nSoxB1 transcription factors are essential for initiating and maintaining the neural plate border gene expression\nAuthors: Schock, E. N.; York, J. R.; Li, A. P.; Tu, A. Y.; LaBonne, C.\nScore: 10.0, Published: 2023-09-29 DOI: 10.1101/2023.09.28.560033\nSoxB1 transcription factors (Sox2/3) are well known for their role in early neural fate specification in the embryo, but little is known about functional roles for SoxB1 factors in non-neural ectodermal cell types, such as the neural plate border (NPB). Using Xenopus laevis, we set out to determine if SoxB1 transcription factors have a regulatory function in NPB formation. Herein, we show that SoxB1 factors are necessary for NPB formation, and that prolonged SoxB1 factor activity blocks the transition from a NPB to a neural crest state. Using ChIP-seq we demonstrate that Sox3 is enriched upstream of NPB genes in early NPB cells and, surprisingly, in blastula stem cells. Depletion of SoxB1 factors in blastula stem cells results in downregulation of NPB genes. Finally, we identify Pou5f3 factors as a potential SoxB1 partners in regulating the formation of the NPB and show their combined activity is needed to maintain NPB gene expression. Together, these data identify a novel role for SoxB1 factors in the establishment and maintenance of the NPB, in part through partnership with Pou5f3 factors.\nMyocardin-related transcription factors regulate morphogenetic events in vertebrate embryos by controlling F-actin organization and apical constriction\nAuthors: Itoh, K.; Ossipova, O.; Matsuda, M.; Sokol, S. Y.\nScore: 8.7, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559818\nMyocardin-related transcription factors (Mrtfa and Mrtfb), also known as megakaryoblastic leukemia proteins (Mkl1/MAL and Mkl2), associate with serum response factor (Srf) to regulate transcription in response to actin dynamics, however, the functions of Mrtfs in early vertebrate embryos remain largely unknown. Here we document the requirement of Mrtfs for blastopore closure at gastrulation and neural plate folding in Xenopus early embryos. Both stimulation and inhibition of Mrtf activity caused similar gross morphological phenotypes, yet the effects on F-actin distribution and cell behavior were different. Suppressing Mrtf-dependent transcription reduced overall F-actin levels and inhibited apical constriction during gastrulation and neurulation. By contrast, constitutively active Mrtf caused tricellular junction remodeling and induced apical constriction in superficial ectoderm. The underlying mechanism appeared distinct from the one utilized by known apical constriction inducers. We propose that the regulation of apical constriction is among the primary cellular responses to Mrtf. Our findings highlight a dedicated role of specific transcription factors, Mrtfs, in early morphogenetic processes.\nDifferences in binding preferences for XIST partners are observed in mammals with different early pregnancy morphologies\nAuthors: Tsagakis, I.; Tinning, H.; Malo-Estepa, I.; Whitehouse, A.; O'Connell, M. J.; Forde, N.; Aspden, J.\nScore: 6.8, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560814\nIn comparison to protein-coding genes, long non-coding RNAs are poorly conserved between species at the sequence level yet they often perform conserved and essential regulatory roles. The lncRNA XIST mediates X chromosome inactivation (XCI) through the interaction of numerous proteins, most of which bind to its repeat regions. Whilst XIST is present across placental mammals, its sequence is divergent. In addition, the timing and mechanistic details of the process of XCI also varies across placental mammals. Here, we sought to determine whether XIST interactor proteins previously identified in a model mammalian species (mouse) also bind in other mammals which exhibit divergent timing of XCI (human and bovine) and differing pre-implantation embryo morphology. We show that XIST and its putative protein partners are coordinately expressed in the endometria of mammals with different pre-implantation embryo morphologies. RNA immunoprecipitations revealed that human WTAP, SPEN, hnRNPK and CIZ1 proteins bind human XIST. CIZ1-XIST binding was also detected in bovine. In both human and cow, we found CIZ1 binds to the repeat E element of XIST. However, human CIZ1 protein is not able to bind bovine repeat E, indicating a species-specific binding interaction. Together these data indicate that several of the key RNA-protein interactions with XIST are common across placental mammals, even though we observe divergence in both the RNA sequence of XIST, and early embryo morphologies. This work sheds light upon the evolution of RNA-protein binding interactions, revealing that the binding events can be conserved even when the precise mechanism of binding has changed.\nAn in vivo drug screen reveals that cyclooxygenase 2-derived prostaglandin D2 promotes spinal cord neurogenesis.\nAuthors: Gonzalez-Llera, L.; Sobrido-Camean, D.; Quelle-Regaldie, A.; SANCHEZ, L.; Barreiro-Iglesias, A.\nScore: 6.5, Published: 2023-10-08 DOI: 10.1101/2023.10.05.561099\nThe study of neurogenesis is essential to understand fundamental developmental processes and for the development of cell replacement therapies for central nervous system disorders. Here, we designed an in vivo drug screening protocol in developing zebrafish to find new molecules and signalling pathways regulating neurogenesis in the ventral spinal cord. This unbiased drug screen revealed that 4 cyclooxygenase (COX) inhibitors reduce the generation of serotonergic interneurons in the developing spinal cord. These results fitted very nicely with available single cell RNAseq data revealing that the floor plate cells show differential expression of 1 of the 2 COX2 zebrafish genes (ptgs2a). Indeed, several selective COX2 inhibitors and 2 different morpholinos against ptgs2a also caused a significant reduction in the number of serotonergic neurons in the ventral spinal cord and led to locomotor deficits. Single cell RNAseq data and different pharmacological manipulations further revealed that COX2-floor plate-derived prostaglandin D2 promotes neurogenesis in the developing spinal cord by promoting mitotic activity in progenitor cells. Rescue experiments using a phosphodiesterase-4 inhibitor suggest that intracellular changes in cAMP levels underlie the effects of COX inhibitors on neurogenesis and locomotion. Our study provides compelling in vivo evidence showing that prostaglandin signalling promotes neurogenesis in the ventral spinal cord.\nProtein phosphatase 1 regulates core PCP signaling\nAuthors: Song, S.; Cho, B.; Weiner, A. T.; Nissen, S. B.; Naharros, I. O.; Bosch, P. S.; Suyama, K.; Hu, Y.; He, L.; Svinkina, T.; Udeshi, N.; Carr, S. A.; Perrimon, N.; Axelrod, J. D.\nScore: 9.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.556998\nAbstractPCP signaling polarizes epithelial cells within the plane of an epithelium. Core PCP signaling components adopt asymmetric subcellular localizations within cells to both polarize and coordinate polarity between cells. Achieving subcellular asymmetry requires additional effectors, including some mediating post-translational modifications of core components. Identification of such proteins is challenging due to pleiotropy. We used mass spectrometry-based proximity labeling proteomics to identify such regulators in the Drosophila wing. We identified the catalytic subunit of Protein Phosphatase1, Pp1-87B, and show that it regulates core protein polarization. Pp1-87B interacts with the core protein Van Gogh and at least one Serine/Threonine kinase, Dco/CKIc, that is known to regulate PCP. Pp1-87B modulates Van Gogh subcellular localization and directs its dephosphorylation in vivo. PNUTS, a Pp1 regulatory subunit, also modulates PCP. While the direct substrate(s) of Pp1-87B in control of PCP is not known, our data support the model that cycling between phosphorylated and unphosphorylated forms of one or more core PCP components may regulate acquisition of asymmetry. Finally, our screen serves as a resource for identifying additional regulators of PCP signaling.\nFunctional opsin patterning for Drosophila color vision is established through signaling pathways in adjacent object-detection neurons\nAuthors: Kitamata, M.; Otake, Y.; Kitagori, H.; Zhang, X.; Maki, Y.; Boku, R.; Takeuchi, M.; Nakagoshi, H.\nScore: 8.2, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560118\nVision is mainly based on two different tasks, object detection and color discrimination through activities of photoreceptor (PR) cells. Drosophila compound eye consists of [~]800 ommatidia. Every ommatidium contains eight PR cells; six outer cells (R1-R6) and two inner cells (R7 and R8) by which object detection and color vision are achieved, respectively. Expression of opsin genes in R7 and R8 is highly coordinated through the instructive signal from R7 to R8, and two major ommatidial subtypes are distributed stochastically; pale type expresses Rh3/Rh5, while yellow type expresses Rh4/Rh6 in R7/R8. The homeodomain protein Defective proventriculus (Dve) is expressed in yellow-type R7 and in six outer PRs, and it is involved in Rh3 repression to specify the yellow-type R7. dve mutant eyes exhibited atypical coupling, Rh3/Rh6 and Rh4/Rh5, indicating that the Dve activity is required for proper opsin coupling. Surprisingly, Dve activity in R1 is required for instructive signal, whereas those in R6 and R7 block the signal. Our results indicate that functional coupling of two different neurons is established through signaling pathways from adjacent neurons that are functionally different. Summary StatementDve activity is required for proper opsin coupling of two neurons that is established through signaling pathways from adjacent neurons that are functionally different.\nFolate depletion alters mouse trophoblast stem cell regulation in vitro\nAuthors: Rakoczy, J.; Watson, E. D.\nScore: 7.4, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558846\nMaternal folate deficiency increases risk of congenital malformations, yet its effect on placenta development is unclear. Here, we investigated how folate-depleted culture medium affects the developmental potential of mouse trophoblast stem cells (TSCs). When cultured in stem cell conditions, TSC viability was unaffected by folate depletion, but ectopic differentiation of several trophoblast cell subtypes occurred. When cultured in conditions that promote differentiation, folate-depleted TSCs were driven towards a syncytiotrophoblast cell fate potentially at the expense of other lineages. Additionally, trophoblast giant cell nuclei were small implicating folate in the regulation of endoreduplication. Therefore, dietary folate intake likely promotes placenta development.\n",
  "wordCount" : "2242",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561233">
        <p class="paperTitle">Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.</p>
        <p class="info">Score: 65.9, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561233' target='https://doi.org/10.1101/2023.10.06.561233'> 10.1101/2023.10.06.561233</a></p>
        <p class="abstract">Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs. This atlas catalogs hundreds to thousands of cell types, of which we annotated 250. This still leaves many clusters to be fully characterized, in particular in the brain. Furthermore, with single-nuclei sequencing, all information about the spatial location of the cells and of the mRNAs within these cells is lost. Here, we provide a solution to this problem. In a proof of concept study, we have applied spatial transcriptomics using a selected gene panel to pinpoint the locations of 150 mRNA species in the adult fly. This enabled us to map unknown cell types identified in the Fly Cell Atlas to their spatial locations in the brain. Additionally, spatial transcriptomics discovered interesting principles of mRNA localization in large crowded muscle cells that may spark future mechanistic investigations. Furthermore, we present a set of computational tools that will allow for easier integration of spatial transcriptomics and single-cell datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561097">
        <p class="paperTitle">An integrated transcriptomic cell atlas of human neural organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Z.; Dony, L.; Fleck, J. S.; Szalata, A.; Li, K. X.; Sliskovic, I.; Lin, H.-C.; Santel, M.; Atamian, A.; Quadrato, G.; Sun, J.; Pasca, S. P.; Camp, G.; Theis, F.; Treutlein, B.</p>
        <p class="info">Score: 34.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561097' target='https://doi.org/10.1101/2023.10.05.561097'> 10.1101/2023.10.05.561097</a></p>
        <p class="abstract">Neural tissues generated from human pluripotent stem cells in vitro (known as neural organoids) are becoming useful tools to study human brain development, evolution and disease. The characterization of neural organoids using single-cell genomic methods has revealed a large diversity of neural cell types with molecular signatures similar to those observed in primary human brain tissue. However, it is unclear which domains of the human nervous system are covered by existing protocols. It is also difficult to quantitatively assess variation between protocols and the specific cell states in organoids as compared to primary counterparts. Single-cell transcriptome data from primary tissue and neural organoids derived with guided or un-guided approaches and under diverse conditions combined with large-scale integrative analyses make it now possible to address these challenges. Recent advances in computational methodology enable the generation of integrated atlases across many data sets. Here, we integrated 36 single-cell transcriptomics data sets spanning 26 protocols into one integrated human neural organoid cell atlas (HNOCA) totaling over 1.7 million cells. We harmonize cell type annotations by incorporating reference data sets from the developing human brain. By mapping to the developing human brain reference, we reveal which primary cell states have been generated in vitro, and which are under-represented. We further compare transcriptomic profiles of neuronal populations in organoids to their counterparts in the developing human brain. To support rapid organoid phenotyping and quantitative assessment of new protocols, we provide a programmatic interface to browse the atlas and query new data sets, and showcase the power of the atlas to annotate new query data sets and evaluate new organoid protocols. Taken together, the HNOCA will be useful to assess the fidelity of organoids, characterize perturbed and diseased states and facilitate protocol development in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559925">
        <p class="paperTitle">Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.</p>
        <p class="info">Score: 98.1, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559925' target='https://doi.org/10.1101/2023.09.28.559925'> 10.1101/2023.09.28.559925</a></p>
        <p class="abstract">Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging. To bridge these wide spatiotemporal scales, we present an open top dual view and dual illumination light sheet microscope dedicated for live imaging of large specimens at single cell resolution. The novel configuration of objectives together with a flexible and customizable multi-well mounting system combines dual view with high throughput multi-position imaging. We use the newly developed microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.560033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.560033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.560033">
        <p class="paperTitle">SoxB1 transcription factors are essential for initiating and maintaining the neural plate border gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.560033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.560033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schock, E. N.; York, J. R.; Li, A. P.; Tu, A. Y.; LaBonne, C.</p>
        <p class="info">Score: 10.0, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.560033' target='https://doi.org/10.1101/2023.09.28.560033'> 10.1101/2023.09.28.560033</a></p>
        <p class="abstract">SoxB1 transcription factors (Sox2/3) are well known for their role in early neural fate specification in the embryo, but little is known about functional roles for SoxB1 factors in non-neural ectodermal cell types, such as the neural plate border (NPB). Using Xenopus laevis, we set out to determine if SoxB1 transcription factors have a regulatory function in NPB formation. Herein, we show that SoxB1 factors are necessary for NPB formation, and that prolonged SoxB1 factor activity blocks the transition from a NPB to a neural crest state. Using ChIP-seq we demonstrate that Sox3 is enriched upstream of NPB genes in early NPB cells and, surprisingly, in blastula stem cells. Depletion of SoxB1 factors in blastula stem cells results in downregulation of NPB genes. Finally, we identify Pou5f3 factors as a potential SoxB1 partners in regulating the formation of the NPB and show their combined activity is needed to maintain NPB gene expression. Together, these data identify a novel role for SoxB1 factors in the establishment and maintenance of the NPB, in part through partnership with Pou5f3 factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559818">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559818" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559818">
        <p class="paperTitle">Myocardin-related transcription factors regulate morphogenetic events in vertebrate embryos by controlling F-actin organization and apical constriction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559818" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559818" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Itoh, K.; Ossipova, O.; Matsuda, M.; Sokol, S. Y.</p>
        <p class="info">Score: 8.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559818' target='https://doi.org/10.1101/2023.09.27.559818'> 10.1101/2023.09.27.559818</a></p>
        <p class="abstract">Myocardin-related transcription factors (Mrtfa and Mrtfb), also known as megakaryoblastic leukemia proteins (Mkl1/MAL and Mkl2), associate with serum response factor (Srf) to regulate transcription in response to actin dynamics, however, the functions of Mrtfs in early vertebrate embryos remain largely unknown. Here we document the requirement of Mrtfs for blastopore closure at gastrulation and neural plate folding in Xenopus early embryos. Both stimulation and inhibition of Mrtf activity caused similar gross morphological phenotypes, yet the effects on F-actin distribution and cell behavior were different. Suppressing Mrtf-dependent transcription reduced overall F-actin levels and inhibited apical constriction during gastrulation and neurulation. By contrast, constitutively active Mrtf caused tricellular junction remodeling and induced apical constriction in superficial ectoderm. The underlying mechanism appeared distinct from the one utilized by known apical constriction inducers. We propose that the regulation of apical constriction is among the primary cellular responses to Mrtf. Our findings highlight a dedicated role of specific transcription factors, Mrtfs, in early morphogenetic processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560814">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560814" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560814">
        <p class="paperTitle">Differences in binding preferences for XIST partners are observed in mammals with different early pregnancy morphologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560814" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560814" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsagakis, I.; Tinning, H.; Malo-Estepa, I.; Whitehouse, A.; O&#39;Connell, M. J.; Forde, N.; Aspden, J.</p>
        <p class="info">Score: 6.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560814' target='https://doi.org/10.1101/2023.10.04.560814'> 10.1101/2023.10.04.560814</a></p>
        <p class="abstract">In comparison to protein-coding genes, long non-coding RNAs are poorly conserved between species at the sequence level yet they often perform conserved and essential regulatory roles. The lncRNA XIST mediates X chromosome inactivation (XCI) through the interaction of numerous proteins, most of which bind to its repeat regions. Whilst XIST is present across placental mammals, its sequence is divergent. In addition, the timing and mechanistic details of the process of XCI also varies across placental mammals. Here, we sought to determine whether XIST interactor proteins previously identified in a model mammalian species (mouse) also bind in other mammals which exhibit divergent timing of XCI (human and bovine) and differing pre-implantation embryo morphology. We show that XIST and its putative protein partners are coordinately expressed in the endometria of mammals with different pre-implantation embryo morphologies. RNA immunoprecipitations revealed that human WTAP, SPEN, hnRNPK and CIZ1 proteins bind human XIST. CIZ1-XIST binding was also detected in bovine. In both human and cow, we found CIZ1 binds to the repeat E element of XIST. However, human CIZ1 protein is not able to bind bovine repeat E, indicating a species-specific binding interaction. Together these data indicate that several of the key RNA-protein interactions with XIST are common across placental mammals, even though we observe divergence in both the RNA sequence of XIST, and early embryo morphologies. This work sheds light upon the evolution of RNA-protein binding interactions, revealing that the binding events can be conserved even when the precise mechanism of binding has changed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561099">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561099" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561099">
        <p class="paperTitle">An in vivo drug screen reveals that cyclooxygenase 2-derived prostaglandin D2 promotes spinal cord neurogenesis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561099" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561099" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Llera, L.; Sobrido-Camean, D.; Quelle-Regaldie, A.; SANCHEZ, L.; Barreiro-Iglesias, A.</p>
        <p class="info">Score: 6.5, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561099' target='https://doi.org/10.1101/2023.10.05.561099'> 10.1101/2023.10.05.561099</a></p>
        <p class="abstract">The study of neurogenesis is essential to understand fundamental developmental processes and for the development of cell replacement therapies for central nervous system disorders. Here, we designed an in vivo drug screening protocol in developing zebrafish to find new molecules and signalling pathways regulating neurogenesis in the ventral spinal cord. This unbiased drug screen revealed that 4 cyclooxygenase (COX) inhibitors reduce the generation of serotonergic interneurons in the developing spinal cord. These results fitted very nicely with available single cell RNAseq data revealing that the floor plate cells show differential expression of 1 of the 2 COX2 zebrafish genes (ptgs2a). Indeed, several selective COX2 inhibitors and 2 different morpholinos against ptgs2a also caused a significant reduction in the number of serotonergic neurons in the ventral spinal cord and led to locomotor deficits. Single cell RNAseq data and different pharmacological manipulations further revealed that COX2-floor plate-derived prostaglandin D2 promotes neurogenesis in the developing spinal cord by promoting mitotic activity in progenitor cells. Rescue experiments using a phosphodiesterase-4 inhibitor suggest that intracellular changes in cAMP levels underlie the effects of COX inhibitors on neurogenesis and locomotion. Our study provides compelling in vivo evidence showing that prostaglandin signalling promotes neurogenesis in the ventral spinal cord.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.556998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.556998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.556998">
        <p class="paperTitle">Protein phosphatase 1 regulates core PCP signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.556998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.556998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Song, S.; Cho, B.; Weiner, A. T.; Nissen, S. B.; Naharros, I. O.; Bosch, P. S.; Suyama, K.; Hu, Y.; He, L.; Svinkina, T.; Udeshi, N.; Carr, S. A.; Perrimon, N.; Axelrod, J. D.</p>
        <p class="info">Score: 9.0, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.556998' target='https://doi.org/10.1101/2023.09.12.556998'> 10.1101/2023.09.12.556998</a></p>
        <p class="abstract">AbstractPCP signaling polarizes epithelial cells within the plane of an epithelium. Core PCP signaling components adopt asymmetric subcellular localizations within cells to both polarize and coordinate polarity between cells. Achieving subcellular asymmetry requires additional effectors, including some mediating post-translational modifications of core components. Identification of such proteins is challenging due to pleiotropy. We used mass spectrometry-based proximity labeling proteomics to identify such regulators in the Drosophila wing. We identified the catalytic subunit of Protein Phosphatase1, Pp1-87B, and show that it regulates core protein polarization. Pp1-87B interacts with the core protein Van Gogh and at least one Serine/Threonine kinase, Dco/CKIc, that is known to regulate PCP. Pp1-87B modulates Van Gogh subcellular localization and directs its dephosphorylation in vivo. PNUTS, a Pp1 regulatory subunit, also modulates PCP. While the direct substrate(s) of Pp1-87B in control of PCP is not known, our data support the model that cycling between phosphorylated and unphosphorylated forms of one or more core PCP components may regulate acquisition of asymmetry. Finally, our screen serves as a resource for identifying additional regulators of PCP signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560118">
        <p class="paperTitle">Functional opsin patterning for Drosophila color vision is established through signaling pathways in adjacent object-detection neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kitamata, M.; Otake, Y.; Kitagori, H.; Zhang, X.; Maki, Y.; Boku, R.; Takeuchi, M.; Nakagoshi, H.</p>
        <p class="info">Score: 8.2, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560118' target='https://doi.org/10.1101/2023.09.29.560118'> 10.1101/2023.09.29.560118</a></p>
        <p class="abstract">Vision is mainly based on two different tasks, object detection and color discrimination through activities of photoreceptor (PR) cells. Drosophila compound eye consists of [~]800 ommatidia. Every ommatidium contains eight PR cells; six outer cells (R1-R6) and two inner cells (R7 and R8) by which object detection and color vision are achieved, respectively. Expression of opsin genes in R7 and R8 is highly coordinated through the instructive signal from R7 to R8, and two major ommatidial subtypes are distributed stochastically; pale type expresses Rh3/Rh5, while yellow type expresses Rh4/Rh6 in R7/R8. The homeodomain protein Defective proventriculus (Dve) is expressed in yellow-type R7 and in six outer PRs, and it is involved in Rh3 repression to specify the yellow-type R7. dve mutant eyes exhibited atypical coupling, Rh3/Rh6 and Rh4/Rh5, indicating that the Dve activity is required for proper opsin coupling. Surprisingly, Dve activity in R1 is required for instructive signal, whereas those in R6 and R7 block the signal. Our results indicate that functional coupling of two different neurons is established through signaling pathways from adjacent neurons that are functionally different.

Summary StatementDve activity is required for proper opsin coupling of two neurons that is established through signaling pathways from adjacent neurons that are functionally different.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.558846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.558846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.558846">
        <p class="paperTitle">Folate depletion alters mouse trophoblast stem cell regulation in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.558846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.558846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rakoczy, J.; Watson, E. D.</p>
        <p class="info">Score: 7.4, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.558846' target='https://doi.org/10.1101/2023.09.29.558846'> 10.1101/2023.09.29.558846</a></p>
        <p class="abstract">Maternal folate deficiency increases risk of congenital malformations, yet its effect on placenta development is unclear. Here, we investigated how folate-depleted culture medium affects the developmental potential of mouse trophoblast stem cells (TSCs). When cultured in stem cell conditions, TSC viability was unaffected by folate depletion, but ectopic differentiation of several trophoblast cell subtypes occurred. When cultured in conditions that promote differentiation, folate-depleted TSCs were driven towards a syncytiotrophoblast cell fate potentially at the expense of other lineages. Additionally, trophoblast giant cell nuclei were small implicating folate in the regulation of endoreduplication. Therefore, dietary folate intake likely promotes placenta development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
