<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CXCR3-CXCL11 signaling restricts angiogenesis and promotes pericyte recruitment
Authors: Goeckel, M. E.; Lee, J.; Levitas, A.; Colijn, S.; Mun, G.; Burton, Z.; Chintalapati, B.; Yin, Y.; Abello, J.; Stratman, A. N.
Score: 14.1, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557842
Endothelial cell (EC)-pericyte interactions are known to remodel in response to hemodynamic forces, yet there is a lack of mechanistic understanding of the signaling pathways that underlie these events. Here, we have identified a novel signaling network regulated by blood flow in ECs--the chemokine receptor, CXCR3, and one of its ligands, CXCL11--that delimits EC angiogenic potential and suppresses pericyte recruitment during development through regulation of pdgfb expression in ECs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="CXCR3-CXCL11 signaling restricts angiogenesis and promotes pericyte recruitment
Authors: Goeckel, M. E.; Lee, J.; Levitas, A.; Colijn, S.; Mun, G.; Burton, Z.; Chintalapati, B.; Yin, Y.; Abello, J.; Stratman, A. N.
Score: 14.1, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557842
Endothelial cell (EC)-pericyte interactions are known to remodel in response to hemodynamic forces, yet there is a lack of mechanistic understanding of the signaling pathways that underlie these events. Here, we have identified a novel signaling network regulated by blood flow in ECs--the chemokine receptor, CXCR3, and one of its ligands, CXCL11--that delimits EC angiogenic potential and suppresses pericyte recruitment during development through regulation of pdgfb expression in ECs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="CXCR3-CXCL11 signaling restricts angiogenesis and promotes pericyte recruitment
Authors: Goeckel, M. E.; Lee, J.; Levitas, A.; Colijn, S.; Mun, G.; Burton, Z.; Chintalapati, B.; Yin, Y.; Abello, J.; Stratman, A. N.
Score: 14.1, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557842
Endothelial cell (EC)-pericyte interactions are known to remodel in response to hemodynamic forces, yet there is a lack of mechanistic understanding of the signaling pathways that underlie these events. Here, we have identified a novel signaling network regulated by blood flow in ECs--the chemokine receptor, CXCR3, and one of its ligands, CXCL11--that delimits EC angiogenic potential and suppresses pericyte recruitment during development through regulation of pdgfb expression in ECs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "CXCR3-CXCL11 signaling restricts angiogenesis and promotes pericyte recruitment\nAuthors: Goeckel, M. E.; Lee, J.; Levitas, A.; Colijn, S.; Mun, G.; Burton, Z.; Chintalapati, B.; Yin, Y.; Abello, J.; Stratman, A. N.\nScore: 14.1, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557842\nEndothelial cell (EC)-pericyte interactions are known to remodel in response to hemodynamic forces, yet there is a lack of mechanistic understanding of the signaling pathways that underlie these events. Here, we have identified a novel signaling network regulated by blood flow in ECs--the chemokine receptor, CXCR3, and one of its ligands, CXCL11--that delimits EC angiogenic potential and suppresses pericyte recruitment during development through regulation of pdgfb expression in ECs.",
  "keywords": [
    
  ],
  "articleBody": " CXCR3-CXCL11 signaling restricts angiogenesis and promotes pericyte recruitment\nAuthors: Goeckel, M. E.; Lee, J.; Levitas, A.; Colijn, S.; Mun, G.; Burton, Z.; Chintalapati, B.; Yin, Y.; Abello, J.; Stratman, A. N.\nScore: 14.1, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557842\nEndothelial cell (EC)-pericyte interactions are known to remodel in response to hemodynamic forces, yet there is a lack of mechanistic understanding of the signaling pathways that underlie these events. Here, we have identified a novel signaling network regulated by blood flow in ECs--the chemokine receptor, CXCR3, and one of its ligands, CXCL11--that delimits EC angiogenic potential and suppresses pericyte recruitment during development through regulation of pdgfb expression in ECs. In vitro modeling of EC-pericyte interactions demonstrates that suppression of EC-specific CXCR3 signaling leads to loss of pericyte association with EC tubes. In vivo, phenotypic defects are particularly noted in the cranial vasculature, where we see a loss of pericyte association with and expansion of the vasculature in zebrafish treated with the Cxcr3 inhibitor AMG487. We also demonstrate using flow modeling platforms that CXCR3-deficient ECs are more elongated, move more slowly, and have impaired EC-EC junctions compared to their control counterparts. Together these data suggest that CXCR3 signaling in ECs drives vascular stabilization events during development.\nLong-term three-dimensional high-resolution imaging of live unlabeled small intestinal organoids using low-coherence holotomography\nAuthors: Lee, M. J.; Lee, J.; Ha, J.; Kim, G.; Kim, H.-J.; Lee, S.; Koo, B.-K.; Park, Y.\nScore: 7.5, Published: 2023-09-16 DOI: 10.1101/2023.09.16.558039\nThe prevailing challenges in live unlabeled high-resolution imaging of native organoids stem from technical issues like complex sample handling and optical scattering in three-dimensional architectures. In this study, we introduce low-coherence holotomography as an advanced, label-free, quantitative imaging modality, designed to overcome related technical obstacles for long-term live imaging of 3D organoids. We successfully captured high-resolution morphological intricacies and dynamic events within mouse small intestinal organoids at a subcellular resolution. Furthermore, this method provides a unique advantage in differentiating between viable and non-viable organoids, thereby expanding its potential applications in organoid-based research.\nSingle-cell mitochondrial variant enrichment resolved clonal tracking and spatial architecture in human embryonic hematopoiesis\nAuthors: Xue, Y.; Chao, Y.; Lin, X.; Huang, Y.; Ho, J. W. K.; Sugimura, R.\nScore: 7.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558215\nThe ability to perform lineage tracing at the single-cell level is critical to reconstructing dynamic transitions during cell differentiation. However, prospective tracing approaches inevitably encounter outstanding challenges including barcoding precision, barcode diversity, and detection efficiency, which can skew inferred lineage relationships. Human pluripotent stem cells (hPSC) even face risks of DNA-damage-induced toxicity-related cell death. We explored the use of naturally occurring somatic mutations in mitochondrial transcripts detected in single-cell RNA-seq as genetic lineage barcodes in HPSC. In this study, we used an enrichment of scRNA-seq mitochondrial reads and a robust computational method to identify clonally relevant mitochondrial variants as endogenous genetic barcodes for clonal tracking of early embryonic hematopoiesis from hPSC. We modeled the development of embryonic tissues from hPSCs and delineated the ontogeny of hematopoietic cells by mitochondrial variant lineage tracing. We identified multiple waves of erythropoiesis in time-series scRNA-seq data. We further applied mitochondrial variant lineage tracing to spatial transcriptomics and identified the clonal development of both erythro-myeloid progenitors and their niche cells. Our study highlights the power of mitochondrial variants as an endogenous marker for clonal tracking in modeling human development from stem cells.\nSensitivity of the timing of Drosophila pupal wing morphogenesis to external perturbations\nAuthors: Piscitello-Gomez, R.; Mahmoud, A.; Dye, N. A.; Eaton, S.\nScore: 4.9, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558969\nThe morphogenesis of the pupal wing of Drosophila melanogaster is a powerful model for understanding the physical mechanisms of tissue morphogenesis. During this process, the wing undergoes a dramatic reshaping where the hinge region contracts and pulls on the blade region, triggering a dynamic pattern of cell elongation changes, cell divisions, and cell neighborhood changes Etournay et al. (2015). Over our several year study of this process, we noticed that sometime after the year 2016, the onset of morphogenesis became consistently delayed by [~]6 h compared to previous years. Thus, the peak of cell elongation was now observed at 28 hours after puparium formation (hAPF) rather than at 20-22 hAPF. We show here that this delay is independent of genetic background and that the speed of the overall process is relatively unchanged, implying that an environmental factor causes a shift in the onset of the process relative to the start of pupariation. We then investigated how various environmental factors influence the timing of pupal wing morphogenesis, using the dynamics of cell elongation as a proxy. We find that imaging conditions, temperature, diet, Wolbachia infection, and the light/dark cycle have relatively little effect. Preliminary data show that the spectra of light used to rear the flies affects cell elongation dynamics, although the delay was not completely reverted and more experiments are needed to know precisely how this variable affects morphogenesis. In sum, our work suggests the presence of a gate controlling the onset of pupal wing morphogenesis that is influenced by a specific, yet unclear, environmental factor. Once the gate is released, pupal wing morphogenesis occurs at a fairly constant rate, remarkably robust to many environmental variables.\nThe asynchrony in the exit from naive pluripotency cannot be explained by differences in the cell cycle phase\nAuthors: Jayaram, S.; Romeike, M.; Buecker, C.\nScore: 4.7, Published: 2023-09-15 DOI: 10.1101/2023.09.15.557731\nDevelopment is characterized by consecutive cell state transitions that build on each other and ultimately lead to the generation of the numerous different cell types found in the organism. During each of these transitions, cells change their gene expression profiles and take on new identities. Cell state transitions have to be tightly coordinated with proliferation to ensure simultaneous growth and differentiation. The exit from naive pluripotency is an ideal model system for studying the temporal coordination of proliferation and differentiation. Individual cells initiate differentiation earlier compared to others, thereby leading to an asynchronous exit from naive pluripotency. One of the major differences among the cells of the starting population of mouse embryonic stem cells (mESCs) is the cell cycle status, and could therefore be an underlying cause of the differences in the onset of the exit from naive pluripotency. However, through comprehensive analysis including single cell RNA sequencing (scRNA-seq), cell cycle synchronization, and perturbation experiments, we demonstrate here that the cell cycle phase at the initiation of differentiation does not influence the timing of the exit from naive pluripotency.\nA robust knock-in approach using a minimal promoter and a minicircle\nAuthors: Keating, M.; Hagle, R.; Osorio-Mendez, D.; Rodriguez, A.; Almutawa, S.; Kang, J.\nScore: 4.1, Published: 2023-09-16 DOI: 10.1101/2023.09.15.558008\nKnock-in reporter (KI) animals are essential tools in biomedical research to study gene expression impacting diverse biological events. While CRISPR/Cas9-mediated genome editing allows for the successful generation of KI animals, several factors should be considered, such as low expression of the target gene, prevention of bacterial DNA integration, and in-frame editing. To circumvent these challenges, we developed a new strategy that utilizes minicircle technology and introduces a minimal promoter. We demonstrated that minicircles serve as an efficient donor DNA in zebrafish, significantly enhancing KI events compared to plasmids containing bacterial backbones. In an attempt to generate a KI reporter for scn8ab, we precisely integrated a fluorescence gene at the start codon. However, the seamlessly integrated reporter was unable to direct expression that recapitulates endogenous scn8ab expression. To overcome this obstacle, we introduced the hsp70 minimal promoter to provide an ectopic transcription initiation site and succeeded in establishing stable KI transgenic reporters for scn8ab. This strategy also created a fgf20b KI reporter line with a high success rate. Furthermore, our data revealed that an unexpectedly edited genome can inappropriately influence the integrated reporter gene expression, highlighting the importance of selecting a proper KI line. Overall, our approach utilizing a minicircle and an ectopic promoter establishes a robust and efficient strategy for KI generation, expanding our capacity to create KI animals.\nAtypical heat shock transcription factor HSF5 is critical for male meiotic prophase under non-stress conditions\nAuthors: Yoshimura, S.; Shimada, R.; Kikuchi, K.; Kawagoe, S.; Abe, H.; Iisaka, S.; Fujimura, S.; Yasunaga, K.-i.; Usuki, S.; Tani, N.; Ohba, T.; Kondoh, E.; Saio, T.; Araki, K.; Ishiguro, K.-i.\nScore: 3.7, Published: 2023-09-22 DOI: 10.1101/2023.09.19.557986\nMeiotic prophase progression is differently regulated in males and females. In males, pachytene transition during meiotic prophase is accompanied by robust alteration in gene expression. However, how gene expression is regulated differently to ensure meiotic prophase completion in males remains elusive. Herein, we identified HSF5 as a male germ cell-specific heat shock transcription factor (HSF) for meiotic prophase progression. Genetic analyses and single-cell RNA-sequencing demonstrated that HSF5 is essential for progression beyond the pachytene stage under non-stress conditions rather than heat stress. Chromatin binding analysis in vivo and DNA-binding assays in vitro suggested that HSF5 binds to promoters in a subset of genes associated with chromatin organization. HSF5 recognizes a DNA motif different from typical heat shock elements recognized by other canonical HSFs. This study suggests that HSF5 is an atypical HSF that enforces the gene expression program for pachytene transition during meiotic prophase in males.\nMechanistic Regulation of Planarian Shape During Growth and Degrowth\nAuthors: Ko, J. M.; Reginato, W.; Lobo, D.\nScore: 3.1, Published: 2023-09-16 DOI: 10.1101/2023.09.15.557968\nAdult planarians can grow when fed and degrow (shrink) when starved while maintaining their whole-body shape with correct proportions. Different planarian morphogens are expressed at the anterior-posterior poles, medio-lateral border, and midline to provide positional information signals for the specification of different tissues at the right locations. However, it is currently unknown how these signals are coordinated together during feeding or starvation and how they modulate the differential tissue growth or degrowth necessary to form correct whole-body shapes. Here we investigate the dynamics of planarian shape during growth and degrowth together with a theoretical study to evaluate the mechanisms that regulate whole-body proportions and shape. We found that the planarian body proportions scale isometrically following similar linear rates during growth and degrowth, but that fed worms are significantly wider than starved worms. By combining a descriptive model of planarian shape and size with a mechanistic model of anterior-posterior and medio-lateral signaling calibrated with a novel machine learning methodology, we demonstrate that the feedback loop between these positional information signals and the shape they control can regulate the planarian whole-body shape during growth. Furthermore, the model can predict the correct shape and size dynamics during degrowth due to an increase in apoptosis rate and pole signal during starvation. These results offer mechanistic insights into planarian shape and size dynamics and the regulation of whole-body morphologies.\nThe Chordate Origins of Heart Regeneration\nAuthors: Schuster, K. J.; Christiaen, L.\nScore: 3.0, Published: 2023-09-22 DOI: 10.1101/2023.09.19.558507\nThe human heart is infamous for not healing after infarction in adults, prompting biomedical interest in species that can regenerate damaged hearts. In such animals as zebrafish and neonatal mice, cardiac repair relies on remaining heart tissue supporting cardiomyocyte proliferation. Natural de novo cardiogenesis in post-embryonic stages thus remains elusive. Here we show that the tunicate Ciona, an ascidian among the closest living relatives to the vertebrates, can survive complete chemogenetic ablation of the heart and loss of cardiac function, and recover both cardiac tissue and contractility. As in vertebrates, Ciona heart regeneration relies on Bone Morphogenetic Protein (BMP) signaling-dependent proliferation of cardiomyocytes, providing insights into the evolutionary origins of regenerative cardiogenesis in chordates. Remarkably, prospective lineage tracing by photoconversion of the fluorescent protein Kaede suggested that new cardiomyocytes can emerge from endodermal lineages in post-metamorphic animals, providing an unprecedented case of regenerative de novo cardiogenesis. Finally, while embryos cannot compensate for early losses of the cardiogenic lineage, forming heartless juveniles, developing animals gain their regenerative ability during metamorphosis, uncovering a fundamental transition between deterministic embryogenesis and regulative post-embryonic development.\nSystematic Perturbation of Thousands of Retroviral LTRs in Mouse Embryos\nAuthors: Yang, J.; Cook, L.; Chen, Z.\nScore: 3.0, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558531\nIn mammals, many retrotransposons are de-repressed during zygotic genome activation (ZGA). However, their functions in early development remain elusive largely due to the challenge to simultaneously manipulate thousands of retrotransposon insertions in embryos. Here, we employed epigenome editing to perturb the long terminal repeat (LTR) MT2_Mm, a well-known ZGA and totipotency marker that exists in [~]2667 insertions throughout the mouse genome. CRISPRi robustly repressed 2485 ([~] 93%) MT2_Mm insertions and 1090 ([~]55%) insertions of the closely related MT2C_Mm in 2-cell embryos. Remarkably, such perturbation caused down-regulation of hundreds of ZGA genes at the 2-cell stage and embryonic arrest mostly at the morula stage. Mechanistically, MT2_Mm/MT2C_Mm primarily served as alternative ZGA promoters activated by OBOX proteins. Thus, through unprecedented large-scale epigenome editing, we addressed to what extent MT2_Mm/MT2C_Mm regulates ZGA and preimplantation development. Our approach could be adapted to systematically perturb retrotransposons in other mammalian embryos as it doesnt require transgenic animals.\n",
  "wordCount" : "2133",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557842" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557842">
        <p class="paperTitle">CXCR3-CXCL11 signaling restricts angiogenesis and promotes pericyte recruitment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goeckel, M. E.; Lee, J.; Levitas, A.; Colijn, S.; Mun, G.; Burton, Z.; Chintalapati, B.; Yin, Y.; Abello, J.; Stratman, A. N.</p>
        <p class="info">Score: 14.1, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557842' target='https://doi.org/10.1101/2023.09.16.557842'> 10.1101/2023.09.16.557842</a></p>
        <p class="abstract">Endothelial cell (EC)-pericyte interactions are known to remodel in response to hemodynamic forces, yet there is a lack of mechanistic understanding of the signaling pathways that underlie these events. Here, we have identified a novel signaling network regulated by blood flow in ECs--the chemokine receptor, CXCR3, and one of its ligands, CXCL11--that delimits EC angiogenic potential and suppresses pericyte recruitment during development through regulation of pdgfb expression in ECs. In vitro modeling of EC-pericyte interactions demonstrates that suppression of EC-specific CXCR3 signaling leads to loss of pericyte association with EC tubes. In vivo, phenotypic defects are particularly noted in the cranial vasculature, where we see a loss of pericyte association with and expansion of the vasculature in zebrafish treated with the Cxcr3 inhibitor AMG487. We also demonstrate using flow modeling platforms that CXCR3-deficient ECs are more elongated, move more slowly, and have impaired EC-EC junctions compared to their control counterparts. Together these data suggest that CXCR3 signaling in ECs drives vascular stabilization events during development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.558039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.558039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.558039">
        <p class="paperTitle">Long-term three-dimensional high-resolution imaging of live unlabeled small intestinal organoids using low-coherence holotomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.558039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.558039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. J.; Lee, J.; Ha, J.; Kim, G.; Kim, H.-J.; Lee, S.; Koo, B.-K.; Park, Y.</p>
        <p class="info">Score: 7.5, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.558039' target='https://doi.org/10.1101/2023.09.16.558039'> 10.1101/2023.09.16.558039</a></p>
        <p class="abstract">The prevailing challenges in live unlabeled high-resolution imaging of native organoids stem from technical issues like complex sample handling and optical scattering in three-dimensional architectures. In this study, we introduce low-coherence holotomography as an advanced, label-free, quantitative imaging modality, designed to overcome related technical obstacles for long-term live imaging of 3D organoids. We successfully captured high-resolution morphological intricacies and dynamic events within mouse small intestinal organoids at a subcellular resolution. Furthermore, this method provides a unique advantage in differentiating between viable and non-viable organoids, thereby expanding its potential applications in organoid-based research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558215">
        <p class="paperTitle">Single-cell mitochondrial variant enrichment resolved clonal tracking and spatial architecture in human embryonic hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xue, Y.; Chao, Y.; Lin, X.; Huang, Y.; Ho, J. W. K.; Sugimura, R.</p>
        <p class="info">Score: 7.0, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558215' target='https://doi.org/10.1101/2023.09.18.558215'> 10.1101/2023.09.18.558215</a></p>
        <p class="abstract">The ability to perform lineage tracing at the single-cell level is critical to reconstructing dynamic transitions during cell differentiation. However, prospective tracing approaches inevitably encounter outstanding challenges including barcoding precision, barcode diversity, and detection efficiency, which can skew inferred lineage relationships. Human pluripotent stem cells (hPSC) even face risks of DNA-damage-induced toxicity-related cell death. We explored the use of naturally occurring somatic mutations in mitochondrial transcripts detected in single-cell RNA-seq as genetic lineage barcodes in HPSC. In this study, we used an enrichment of scRNA-seq mitochondrial reads and a robust computational method to identify clonally relevant mitochondrial variants as endogenous genetic barcodes for clonal tracking of early embryonic hematopoiesis from hPSC. We modeled the development of embryonic tissues from hPSCs and delineated the ontogeny of hematopoietic cells by mitochondrial variant lineage tracing. We identified multiple waves of erythropoiesis in time-series scRNA-seq data. We further applied mitochondrial variant lineage tracing to spatial transcriptomics and identified the clonal development of both erythro-myeloid progenitors and their niche cells. Our study highlights the power of mitochondrial variants as an endogenous marker for clonal tracking in modeling human development from stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558969">
        <p class="paperTitle">Sensitivity of the timing of Drosophila pupal wing morphogenesis to external perturbations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piscitello-Gomez, R.; Mahmoud, A.; Dye, N. A.; Eaton, S.</p>
        <p class="info">Score: 4.9, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558969' target='https://doi.org/10.1101/2023.09.22.558969'> 10.1101/2023.09.22.558969</a></p>
        <p class="abstract">The morphogenesis of the pupal wing of Drosophila melanogaster is a powerful model for understanding the physical mechanisms of tissue morphogenesis. During this process, the wing undergoes a dramatic reshaping where the hinge region contracts and pulls on the blade region, triggering a dynamic pattern of cell elongation changes, cell divisions, and cell neighborhood changes Etournay et al. (2015). Over our several year study of this process, we noticed that sometime after the year 2016, the onset of morphogenesis became consistently delayed by [~]6 h compared to previous years. Thus, the peak of cell elongation was now observed at 28 hours after puparium formation (hAPF) rather than at 20-22 hAPF. We show here that this delay is independent of genetic background and that the speed of the overall process is relatively unchanged, implying that an environmental factor causes a shift in the onset of the process relative to the start of pupariation. We then investigated how various environmental factors influence the timing of pupal wing morphogenesis, using the dynamics of cell elongation as a proxy. We find that imaging conditions, temperature, diet, Wolbachia infection, and the light/dark cycle have relatively little effect. Preliminary data show that the spectra of light used to rear the flies affects cell elongation dynamics, although the delay was not completely reverted and more experiments are needed to know precisely how this variable affects morphogenesis. In sum, our work suggests the presence of a gate controlling the onset of pupal wing morphogenesis that is influenced by a specific, yet unclear, environmental factor. Once the gate is released, pupal wing morphogenesis occurs at a fairly constant rate, remarkably robust to many environmental variables.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557731">
        <p class="paperTitle">The asynchrony in the exit from naive pluripotency cannot be explained by differences in the cell cycle phase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jayaram, S.; Romeike, M.; Buecker, C.</p>
        <p class="info">Score: 4.7, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557731' target='https://doi.org/10.1101/2023.09.15.557731'> 10.1101/2023.09.15.557731</a></p>
        <p class="abstract">Development is characterized by consecutive cell state transitions that build on each other and ultimately lead to the generation of the numerous different cell types found in the organism. During each of these transitions, cells change their gene expression profiles and take on new identities. Cell state transitions have to be tightly coordinated with proliferation to ensure simultaneous growth and differentiation. The exit from naive pluripotency is an ideal model system for studying the temporal coordination of proliferation and differentiation. Individual cells initiate differentiation earlier compared to others, thereby leading to an asynchronous exit from naive pluripotency. One of the major differences among the cells of the starting population of mouse embryonic stem cells (mESCs) is the cell cycle status, and could therefore be an underlying cause of the differences in the onset of the exit from naive pluripotency. However, through comprehensive analysis including single cell RNA sequencing (scRNA-seq), cell cycle synchronization, and perturbation experiments, we demonstrate here that the cell cycle phase at the initiation of differentiation does not influence the timing of the exit from naive pluripotency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.558008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.558008" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.558008">
        <p class="paperTitle">A robust knock-in approach using a minimal promoter and a minicircle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.558008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.558008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keating, M.; Hagle, R.; Osorio-Mendez, D.; Rodriguez, A.; Almutawa, S.; Kang, J.</p>
        <p class="info">Score: 4.1, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.558008' target='https://doi.org/10.1101/2023.09.15.558008'> 10.1101/2023.09.15.558008</a></p>
        <p class="abstract">Knock-in reporter (KI) animals are essential tools in biomedical research to study gene expression impacting diverse biological events. While CRISPR/Cas9-mediated genome editing allows for the successful generation of KI animals, several factors should be considered, such as low expression of the target gene, prevention of bacterial DNA integration, and in-frame editing. To circumvent these challenges, we developed a new strategy that utilizes minicircle technology and introduces a minimal promoter. We demonstrated that minicircles serve as an efficient donor DNA in zebrafish, significantly enhancing KI events compared to plasmids containing bacterial backbones. In an attempt to generate a KI reporter for scn8ab, we precisely integrated a fluorescence gene at the start codon. However, the seamlessly integrated reporter was unable to direct expression that recapitulates endogenous scn8ab expression. To overcome this obstacle, we introduced the hsp70 minimal promoter to provide an ectopic transcription initiation site and succeeded in establishing stable KI transgenic reporters for scn8ab. This strategy also created a fgf20b KI reporter line with a high success rate. Furthermore, our data revealed that an unexpectedly edited genome can inappropriately influence the integrated reporter gene expression, highlighting the importance of selecting a proper KI line. Overall, our approach utilizing a minicircle and an ectopic promoter establishes a robust and efficient strategy for KI generation, expanding our capacity to create KI animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.557986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.557986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.557986">
        <p class="paperTitle">Atypical heat shock transcription factor HSF5 is critical for male meiotic prophase under non-stress conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.557986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.557986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoshimura, S.; Shimada, R.; Kikuchi, K.; Kawagoe, S.; Abe, H.; Iisaka, S.; Fujimura, S.; Yasunaga, K.-i.; Usuki, S.; Tani, N.; Ohba, T.; Kondoh, E.; Saio, T.; Araki, K.; Ishiguro, K.-i.</p>
        <p class="info">Score: 3.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.557986' target='https://doi.org/10.1101/2023.09.19.557986'> 10.1101/2023.09.19.557986</a></p>
        <p class="abstract">Meiotic prophase progression is differently regulated in males and females. In males, pachytene transition during meiotic prophase is accompanied by robust alteration in gene expression. However, how gene expression is regulated differently to ensure meiotic prophase completion in males remains elusive. Herein, we identified HSF5 as a male germ cell-specific heat shock transcription factor (HSF) for meiotic prophase progression. Genetic analyses and single-cell RNA-sequencing demonstrated that HSF5 is essential for progression beyond the pachytene stage under non-stress conditions rather than heat stress. Chromatin binding analysis in vivo and DNA-binding assays in vitro suggested that HSF5 binds to promoters in a subset of genes associated with chromatin organization. HSF5 recognizes a DNA motif different from typical heat shock elements recognized by other canonical HSFs. This study suggests that HSF5 is an atypical HSF that enforces the gene expression program for pachytene transition during meiotic prophase in males.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557968">
        <p class="paperTitle">Mechanistic Regulation of Planarian Shape During Growth and Degrowth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ko, J. M.; Reginato, W.; Lobo, D.</p>
        <p class="info">Score: 3.1, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557968' target='https://doi.org/10.1101/2023.09.15.557968'> 10.1101/2023.09.15.557968</a></p>
        <p class="abstract">Adult planarians can grow when fed and degrow (shrink) when starved while maintaining their whole-body shape with correct proportions. Different planarian morphogens are expressed at the anterior-posterior poles, medio-lateral border, and midline to provide positional information signals for the specification of different tissues at the right locations. However, it is currently unknown how these signals are coordinated together during feeding or starvation and how they modulate the differential tissue growth or degrowth necessary to form correct whole-body shapes. Here we investigate the dynamics of planarian shape during growth and degrowth together with a theoretical study to evaluate the mechanisms that regulate whole-body proportions and shape. We found that the planarian body proportions scale isometrically following similar linear rates during growth and degrowth, but that fed worms are significantly wider than starved worms. By combining a descriptive model of planarian shape and size with a mechanistic model of anterior-posterior and medio-lateral signaling calibrated with a novel machine learning methodology, we demonstrate that the feedback loop between these positional information signals and the shape they control can regulate the planarian whole-body shape during growth. Furthermore, the model can predict the correct shape and size dynamics during degrowth due to an increase in apoptosis rate and pole signal during starvation. These results offer mechanistic insights into planarian shape and size dynamics and the regulation of whole-body morphologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558507">
        <p class="paperTitle">The Chordate Origins of Heart Regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schuster, K. J.; Christiaen, L.</p>
        <p class="info">Score: 3.0, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558507' target='https://doi.org/10.1101/2023.09.19.558507'> 10.1101/2023.09.19.558507</a></p>
        <p class="abstract">The human heart is infamous for not healing after infarction in adults, prompting biomedical interest in species that can regenerate damaged hearts. In such animals as zebrafish and neonatal mice, cardiac repair relies on remaining heart tissue supporting cardiomyocyte proliferation. Natural de novo cardiogenesis in post-embryonic stages thus remains elusive. Here we show that the tunicate Ciona, an ascidian among the closest living relatives to the vertebrates, can survive complete chemogenetic ablation of the heart and loss of cardiac function, and recover both cardiac tissue and contractility. As in vertebrates, Ciona heart regeneration relies on Bone Morphogenetic Protein (BMP) signaling-dependent proliferation of cardiomyocytes, providing insights into the evolutionary origins of regenerative cardiogenesis in chordates. Remarkably, prospective lineage tracing by photoconversion of the fluorescent protein Kaede suggested that new cardiomyocytes can emerge from endodermal lineages in post-metamorphic animals, providing an unprecedented case of regenerative de novo cardiogenesis. Finally, while embryos cannot compensate for early losses of the cardiogenic lineage, forming heartless juveniles, developing animals gain their regenerative ability during metamorphosis, uncovering a fundamental transition between deterministic embryogenesis and regulative post-embryonic development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558531">
        <p class="paperTitle">Systematic Perturbation of Thousands of Retroviral LTRs in Mouse Embryos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J.; Cook, L.; Chen, Z.</p>
        <p class="info">Score: 3.0, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558531' target='https://doi.org/10.1101/2023.09.19.558531'> 10.1101/2023.09.19.558531</a></p>
        <p class="abstract">In mammals, many retrotransposons are de-repressed during zygotic genome activation (ZGA). However, their functions in early development remain elusive largely due to the challenge to simultaneously manipulate thousands of retrotransposon insertions in embryos. Here, we employed epigenome editing to perturb the long terminal repeat (LTR) MT2_Mm, a well-known ZGA and totipotency marker that exists in [~]2667 insertions throughout the mouse genome. CRISPRi robustly repressed 2485 ([~] 93%) MT2_Mm insertions and 1090 ([~]55%) insertions of the closely related MT2C_Mm in 2-cell embryos. Remarkably, such perturbation caused down-regulation of hundreds of ZGA genes at the 2-cell stage and embryonic arrest mostly at the morula stage. Mechanistically, MT2_Mm/MT2C_Mm primarily served as alternative ZGA promoters activated by OBOX proteins. Thus, through unprecedented large-scale epigenome editing, we addressed to what extent MT2_Mm/MT2C_Mm regulates ZGA and preimplantation development. Our approach could be adapted to systematically perturb retrotransposons in other mammalian embryos as it doesnt require transgenic animals.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
