<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems
Authors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.
Score: 95.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925
Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems
Authors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.
Score: 95.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925
Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems
Authors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.
Score: 95.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925
Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems\nAuthors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.\nScore: 95.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925\nMulticellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging.",
  "keywords": [
    
  ],
  "articleBody": " Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems\nAuthors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.\nScore: 95.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559925\nMulticellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging. To bridge these wide spatiotemporal scales, we present an open top dual view and dual illumination light sheet microscope dedicated for live imaging of large specimens at single cell resolution. The novel configuration of objectives together with a flexible and customizable multi-well mounting system combines dual view with high throughput multi-position imaging. We use the newly developed microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids.\nSpatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles\nAuthors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.\nScore: 51.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561233\nRecently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs. This atlas catalogs hundreds to thousands of cell types, of which we annotated 250. This still leaves many clusters to be fully characterized, in particular in the brain. Furthermore, with single-nuclei sequencing, all information about the spatial location of the cells and of the mRNAs within these cells is lost. Here, we provide a solution to this problem. In a proof of concept study, we have applied spatial transcriptomics using a selected gene panel to pinpoint the locations of 150 mRNA species in the adult fly. This enabled us to map unknown cell types identified in the Fly Cell Atlas to their spatial locations in the brain. Additionally, spatial transcriptomics discovered interesting principles of mRNA localization in large crowded muscle cells that may spark future mechanistic investigations. Furthermore, we present a set of computational tools that will allow for easier integration of spatial transcriptomics and single-cell datasets.\nAn integrated transcriptomic cell atlas of human neural organoids\nAuthors: He, Z.; Dony, L.; Fleck, J. S.; Szalata, A.; Li, K. X.; Sliskovic, I.; Lin, H.-C.; Santel, M.; Atamian, A.; Quadrato, G.; Sun, J.; Pasca, S. P.; Camp, G.; Theis, F.; Treutlein, B.\nScore: 22.0, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561097\nNeural tissues generated from human pluripotent stem cells in vitro (known as neural organoids) are becoming useful tools to study human brain development, evolution and disease. The characterization of neural organoids using single-cell genomic methods has revealed a large diversity of neural cell types with molecular signatures similar to those observed in primary human brain tissue. However, it is unclear which domains of the human nervous system are covered by existing protocols. It is also difficult to quantitatively assess variation between protocols and the specific cell states in organoids as compared to primary counterparts. Single-cell transcriptome data from primary tissue and neural organoids derived with guided or unguided approaches and under diverse conditions combined with large-scale integrative analyses make it now possible to address these challenges. Recent advances in computational methodology enable the generation of integrated atlases across many data sets. Here, we integrated 36 single-cell transcriptomics data sets spanning 26 protocols into one integrated human neural organoid cell atlas (HNOCA) totaling over 1.7 million cells. We harmonize cell type annotations by incorporating reference data sets from the developing human brain. By mapping to the developing human brain reference, we reveal which primary cell states have been generated in vitro, and which are under-represented. We further compare transcriptomic profiles of neuronal populations in organoids to their counterparts in the developing human brain. To support rapid organoid phenotyping and quantitative assessment of new protocols, we provide a programmatic interface to browse the atlas and query new data sets, and showcase the power of the atlas to annotate new query data sets and evaluate new organoid protocols. Taken together, the HNOCA will be useful to assess the fidelity of organoids, characterize perturbed and diseased states and facilitate protocol development in the future.\nEpididymal acquired sperm microRNAs modify post-fertilization embryonic gene expression\nAuthors: Trigg, N. A.; Conine, C. C.\nScore: 15.7, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560377\nSperm small RNAs have emerged as important non-genetic contributors to embryogenesis and offspring health. A subset of sperm small RNAs are thought to be acquired during epididymal transit. However, the transfer of RNAs from the somatic epididymis to sperm has been questioned, and the identity of the specific small RNAs transferred remains unclear. Here, we employ Cre/Lox genetics to generate germline- and epididymal-specific Dgcr8 conditional knockout mice to investigate the dynamics of sperm microRNAs and their function in the early embryo. Testicular sperm from germline specific Dgcr8 knockout mice have reduced levels of 98 microRNAs. Enthrallingly, following epididymal transit the abundance of 59% of these microRNAs are restored to control levels. Conversely, sperm from epididymal Dgcr8 knockouts displayed a reduction of \u003e 3.4-fold in 25 miRNAs. This substantial loss of epididymal miRNAs in sperm was accompanied by transcriptomic changes in the embryo which was rescued by microinjection of epididymal miRNAs. These findings ultimately demonstrate the acquisition of miRNAs from the soma by sperm during epididymal transit and their subsequent regulation of post-fertilization embryonic gene expression.\nSingle-cell transcription mapping of murine and human mammary organoids responses to female hormones\nAuthors: Ruiz-Ortiz, J.; Lewis, S. M.; Ciccone, M. F.; Chatterjee, D.; Henry, S.; Siepel, A.; Dos Santos, C. O.\nScore: 15.1, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559971\nDuring female adolescence and pregnancy, rising levels of hormones result in a cyclic source of signals that control the development of mammary tissue. While such alterations are well understood from a whole-gland perspective, the alterations that such hormones bring to organoid cultures derived from mammary glands have yet to be fully mapped. This is of special importance given that organoids are considered suitable systems to understand cross species breast development. Here we utilized single-cell transcriptional profiling to delineate responses of murine and human normal breast organoid systems to female hormones across evolutionary distinct species. Collectively, our study represents a molecular atlas of epithelial dynamics in response to estrogen and pregnancy hormones.\nSoxB1 transcription factors are essential for initiating and maintaining the neural plate border gene expression\nAuthors: Schock, E. N.; York, J. R.; Li, A. P.; Tu, A. Y.; LaBonne, C.\nScore: 8.3, Published: 2023-09-29 DOI: 10.1101/2023.09.28.560033\nSoxB1 transcription factors (Sox2/3) are well known for their role in early neural fate specification in the embryo, but little is known about functional roles for SoxB1 factors in non-neural ectodermal cell types, such as the neural plate border (NPB). Using Xenopus laevis, we set out to determine if SoxB1 transcription factors have a regulatory function in NPB formation. Herein, we show that SoxB1 factors are necessary for NPB formation, and that prolonged SoxB1 factor activity blocks the transition from a NPB to a neural crest state. Using ChIP-seq we demonstrate that Sox3 is enriched upstream of NPB genes in early NPB cells and, surprisingly, in blastula stem cells. Depletion of SoxB1 factors in blastula stem cells results in downregulation of NPB genes. Finally, we identify Pou5f3 factors as a potential SoxB1 partners in regulating the formation of the NPB and show their combined activity is needed to maintain NPB gene expression. Together, these data identify a novel role for SoxB1 factors in the establishment and maintenance of the NPB, in part through partnership with Pou5f3 factors.\nMapping the developmental potential of mouse inner ear organoids at single-cell resolution\nAuthors: Waldhaus, J.; Jiang, L.; Liu, L.; Liu, J.; Duncan, R. K.\nScore: 10.8, Published: 2023-09-09 DOI: 10.1101/2023.09.08.556845\nInner ear organoids recapitulate development and are intended to generate cell types of the otic lineage for applications such as basic science research and cell replacement strategies. Here, we use single-cell sequencing to study the cellular heterogeneity of late-stage mouse inner ear organoid sensory epithelia, which we validated by comparison with data sets of the mouse cochlea and vestibular epithelia. We resolved supporting cell sub-types, cochlear like hair cells, and vestibular Type I and Type II like hair cells. While cochlear like hair cells aligned best with an outer hair cell trajectory, vestibular like hair cells followed developmental trajectories similar to in vivo programs branching into Type II and then Type I extrastriolar hair cells. These results highlight the transcriptional accuracy of the organoid developmental program but will also inform future strategies to improve synaptic connectivity and regional specification.\nSystematic Perturbation of Thousands of Retroviral LTRs in Mouse Embryos\nAuthors: Yang, J.; Cook, L.; Chen, Z.\nScore: 10.4, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558531\nIn mammals, many retrotransposons are de-repressed during zygotic genome activation (ZGA). However, their functions in early development remain elusive largely due to the challenge to simultaneously manipulate thousands of retrotransposon insertions in embryos. Here, we employed epigenome editing to perturb the long terminal repeat (LTR) MT2_Mm, a well-known ZGA and totipotency marker that exists in [~]2667 insertions throughout the mouse genome. CRISPRi robustly repressed 2485 ([~] 93%) MT2_Mm insertions and 1090 ([~]55%) insertions of the closely related MT2C_Mm in 2-cell embryos. Remarkably, such perturbation caused down-regulation of hundreds of ZGA genes at the 2-cell stage and embryonic arrest mostly at the morula stage. Mechanistically, MT2_Mm/MT2C_Mm primarily served as alternative ZGA promoters activated by OBOX proteins. Thus, through unprecedented large-scale epigenome editing, we addressed to what extent MT2_Mm/MT2C_Mm regulates ZGA and preimplantation development. Our approach could be adapted to systematically perturb retrotransposons in other mammalian embryos as it doesnt require transgenic animals.\nSegmentation in tapeworms as a modified form of flatworm posterior regeneration involving Wnt and Hedgehog signalling\nAuthors: Jarero, F.; Baillie, A.; Riddiford, N.; Montagne, J.; Koziol, U.; Olson, P. D.\nScore: 6.1, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560491\nTapeworms are parasitic flatworms that lack key features conventionally used to define the head-tail axis in free-living organisms, resulting in long standing questions regarding the true orientation of their main body axis. As adults, most tapeworms also exhibit a segmented body which has been considered an adaptation unique to the group. Anteroposterior (AP) patterning in free-living flatworms is controlled by {beta}-catenin-dependent Wnt signalling and positional control genes are expressed by their musculature in highly regionalised domains. Here we investigate the expression of Wnt and Hedgehog components during the strobilar phase of the tapeworm life cycle during which larval tissues are lost and replaced through the continuous production of new tissues. Results reveal previously unidentified centres of signalling associated with their neuromuscular system and show that segments are marked by secondary, AP axes in agreement with the polarity of the primary body axis. Neuromuscular expression and communication between Wnt and Hedgehog signalling is consistent with embryonic and regenerative growth in planarians and is a common mechanism for establishing AP-polarised boundaries in a diverse range of segmented animals. Taken together our results suggest that segmentation in tapeworms represents a modified form of posterior regeneration, a common feature among flatworms.\nFunctional opsin patterning for Drosophila color vision is established through signaling pathways in adjacent object-detection neurons\nAuthors: Kitamata, M.; Otake, Y.; Kitagori, H.; Zhang, X.; Maki, Y.; Boku, R.; Takeuchi, M.; Nakagoshi, H.\nScore: 8.2, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560118\nVision is mainly based on two different tasks, object detection and color discrimination through activities of photoreceptor (PR) cells. Drosophila compound eye consists of [~]800 ommatidia. Every ommatidium contains eight PR cells; six outer cells (R1-R6) and two inner cells (R7 and R8) by which object detection and color vision are achieved, respectively. Expression of opsin genes in R7 and R8 is highly coordinated through the instructive signal from R7 to R8, and two major ommatidial subtypes are distributed stochastically; pale type expresses Rh3/Rh5, while yellow type expresses Rh4/Rh6 in R7/R8. The homeodomain protein Defective proventriculus (Dve) is expressed in yellow-type R7 and in six outer PRs, and it is involved in Rh3 repression to specify the yellow-type R7. dve mutant eyes exhibited atypical coupling, Rh3/Rh6 and Rh4/Rh5, indicating that the Dve activity is required for proper opsin coupling. Surprisingly, Dve activity in R1 is required for instructive signal, whereas those in R6 and R7 block the signal. Our results indicate that functional coupling of two different neurons is established through signaling pathways from adjacent neurons that are functionally different. Summary StatementDve activity is required for proper opsin coupling of two neurons that is established through signaling pathways from adjacent neurons that are functionally different.\n",
  "wordCount" : "2104",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559925">
        <p class="paperTitle">Open top multi sample dual view light sheet microscope for live imaging of large multicellular systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moos, F.; Suppinger, S.; de Medeiros, G.; Oost, K.; Boni, A.; Remy, C.; Weevers, S.; Tsiairis, C. D.; Strnad, P.; Liberali, P.</p>
        <p class="info">Score: 95.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559925' target='https://doi.org/10.1101/2023.09.28.559925'> 10.1101/2023.09.28.559925</a></p>
        <p class="abstract">Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms composed of several thousands of cells, making live imaging of such samples challenging. To bridge these wide spatiotemporal scales, we present an open top dual view and dual illumination light sheet microscope dedicated for live imaging of large specimens at single cell resolution. The novel configuration of objectives together with a flexible and customizable multi-well mounting system combines dual view with high throughput multi-position imaging. We use the newly developed microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561233">
        <p class="paperTitle">Spatial transcriptomics in adult Drosophila reveals new cell types in the brain and identifies subcellular mRNA patterns in muscles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janssens, J.; Mangeol, P.; Hecker, N.; Partel, G.; Spanier, K.; Ismail, J. N.; Hulselmans, G. J.; Aerts, S.; Schnorrer, F.</p>
        <p class="info">Score: 51.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561233' target='https://doi.org/10.1101/2023.10.06.561233'> 10.1101/2023.10.06.561233</a></p>
        <p class="abstract">Recently, we have achieved a significant milestone with the creation of the Fly Cell Atlas. This single-nuclei atlas encompasses the entire fly, covering the entire head and body, in addition to all major organs. This atlas catalogs hundreds to thousands of cell types, of which we annotated 250. This still leaves many clusters to be fully characterized, in particular in the brain. Furthermore, with single-nuclei sequencing, all information about the spatial location of the cells and of the mRNAs within these cells is lost. Here, we provide a solution to this problem. In a proof of concept study, we have applied spatial transcriptomics using a selected gene panel to pinpoint the locations of 150 mRNA species in the adult fly. This enabled us to map unknown cell types identified in the Fly Cell Atlas to their spatial locations in the brain. Additionally, spatial transcriptomics discovered interesting principles of mRNA localization in large crowded muscle cells that may spark future mechanistic investigations. Furthermore, we present a set of computational tools that will allow for easier integration of spatial transcriptomics and single-cell datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561097">
        <p class="paperTitle">An integrated transcriptomic cell atlas of human neural organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Z.; Dony, L.; Fleck, J. S.; Szalata, A.; Li, K. X.; Sliskovic, I.; Lin, H.-C.; Santel, M.; Atamian, A.; Quadrato, G.; Sun, J.; Pasca, S. P.; Camp, G.; Theis, F.; Treutlein, B.</p>
        <p class="info">Score: 22.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561097' target='https://doi.org/10.1101/2023.10.05.561097'> 10.1101/2023.10.05.561097</a></p>
        <p class="abstract">Neural tissues generated from human pluripotent stem cells in vitro (known as neural organoids) are becoming useful tools to study human brain development, evolution and disease. The characterization of neural organoids using single-cell genomic methods has revealed a large diversity of neural cell types with molecular signatures similar to those observed in primary human brain tissue. However, it is unclear which domains of the human nervous system are covered by existing protocols. It is also difficult to quantitatively assess variation between protocols and the specific cell states in organoids as compared to primary counterparts. Single-cell transcriptome data from primary tissue and neural organoids derived with guided or unguided approaches and under diverse conditions combined with large-scale integrative analyses make it now possible to address these challenges. Recent advances in computational methodology enable the generation of integrated atlases across many data sets. Here, we integrated 36 single-cell transcriptomics data sets spanning 26 protocols into one integrated human neural organoid cell atlas (HNOCA) totaling over 1.7 million cells. We harmonize cell type annotations by incorporating reference data sets from the developing human brain. By mapping to the developing human brain reference, we reveal which primary cell states have been generated in vitro, and which are under-represented. We further compare transcriptomic profiles of neuronal populations in organoids to their counterparts in the developing human brain. To support rapid organoid phenotyping and quantitative assessment of new protocols, we provide a programmatic interface to browse the atlas and query new data sets, and showcase the power of the atlas to annotate new query data sets and evaluate new organoid protocols. Taken together, the HNOCA will be useful to assess the fidelity of organoids, characterize perturbed and diseased states and facilitate protocol development in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560377">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560377" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560377">
        <p class="paperTitle">Epididymal acquired sperm microRNAs modify post-fertilization embryonic gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560377" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560377" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trigg, N. A.; Conine, C. C.</p>
        <p class="info">Score: 15.7, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560377' target='https://doi.org/10.1101/2023.10.01.560377'> 10.1101/2023.10.01.560377</a></p>
        <p class="abstract">Sperm small RNAs have emerged as important non-genetic contributors to embryogenesis and offspring health. A subset of sperm small RNAs are thought to be acquired during epididymal transit. However, the transfer of RNAs from the somatic epididymis to sperm has been questioned, and the identity of the specific small RNAs transferred remains unclear. Here, we employ Cre/Lox genetics to generate germline- and epididymal-specific Dgcr8 conditional knockout mice to investigate the dynamics of sperm microRNAs and their function in the early embryo. Testicular sperm from germline specific Dgcr8 knockout mice have reduced levels of 98 microRNAs. Enthrallingly, following epididymal transit the abundance of 59% of these microRNAs are restored to control levels. Conversely, sperm from epididymal Dgcr8 knockouts displayed a reduction of &gt; 3.4-fold in 25 miRNAs. This substantial loss of epididymal miRNAs in sperm was accompanied by transcriptomic changes in the embryo which was rescued by microinjection of epididymal miRNAs. These findings ultimately demonstrate the acquisition of miRNAs from the soma by sperm during epididymal transit and their subsequent regulation of post-fertilization embryonic gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559971">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559971" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559971">
        <p class="paperTitle">Single-cell transcription mapping of murine and human mammary organoids responses to female hormones</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559971" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559971" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruiz-Ortiz, J.; Lewis, S. M.; Ciccone, M. F.; Chatterjee, D.; Henry, S.; Siepel, A.; Dos Santos, C. O.</p>
        <p class="info">Score: 15.1, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559971' target='https://doi.org/10.1101/2023.09.28.559971'> 10.1101/2023.09.28.559971</a></p>
        <p class="abstract">During female adolescence and pregnancy, rising levels of hormones result in a cyclic source of signals that control the development of mammary tissue. While such alterations are well understood from a whole-gland perspective, the alterations that such hormones bring to organoid cultures derived from mammary glands have yet to be fully mapped. This is of special importance given that organoids are considered suitable systems to understand cross species breast development. Here we utilized single-cell transcriptional profiling to delineate responses of murine and human normal breast organoid systems to female hormones across evolutionary distinct species. Collectively, our study represents a molecular atlas of epithelial dynamics in response to estrogen and pregnancy hormones.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.560033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.560033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.560033">
        <p class="paperTitle">SoxB1 transcription factors are essential for initiating and maintaining the neural plate border gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.560033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.560033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schock, E. N.; York, J. R.; Li, A. P.; Tu, A. Y.; LaBonne, C.</p>
        <p class="info">Score: 8.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.560033' target='https://doi.org/10.1101/2023.09.28.560033'> 10.1101/2023.09.28.560033</a></p>
        <p class="abstract">SoxB1 transcription factors (Sox2/3) are well known for their role in early neural fate specification in the embryo, but little is known about functional roles for SoxB1 factors in non-neural ectodermal cell types, such as the neural plate border (NPB). Using Xenopus laevis, we set out to determine if SoxB1 transcription factors have a regulatory function in NPB formation. Herein, we show that SoxB1 factors are necessary for NPB formation, and that prolonged SoxB1 factor activity blocks the transition from a NPB to a neural crest state. Using ChIP-seq we demonstrate that Sox3 is enriched upstream of NPB genes in early NPB cells and, surprisingly, in blastula stem cells. Depletion of SoxB1 factors in blastula stem cells results in downregulation of NPB genes. Finally, we identify Pou5f3 factors as a potential SoxB1 partners in regulating the formation of the NPB and show their combined activity is needed to maintain NPB gene expression. Together, these data identify a novel role for SoxB1 factors in the establishment and maintenance of the NPB, in part through partnership with Pou5f3 factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556845">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556845" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556845">
        <p class="paperTitle">Mapping the developmental potential of mouse inner ear organoids at single-cell resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556845" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556845" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waldhaus, J.; Jiang, L.; Liu, L.; Liu, J.; Duncan, R. K.</p>
        <p class="info">Score: 10.8, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556845' target='https://doi.org/10.1101/2023.09.08.556845'> 10.1101/2023.09.08.556845</a></p>
        <p class="abstract">Inner ear organoids recapitulate development and are intended to generate cell types of the otic lineage for applications such as basic science research and cell replacement strategies. Here, we use single-cell sequencing to study the cellular heterogeneity of late-stage mouse inner ear organoid sensory epithelia, which we validated by comparison with data sets of the mouse cochlea and vestibular epithelia. We resolved supporting cell sub-types, cochlear like hair cells, and vestibular Type I and Type II like hair cells. While cochlear like hair cells aligned best with an outer hair cell trajectory, vestibular like hair cells followed developmental trajectories similar to in vivo programs branching into Type II and then Type I extrastriolar hair cells. These results highlight the transcriptional accuracy of the organoid developmental program but will also inform future strategies to improve synaptic connectivity and regional specification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558531">
        <p class="paperTitle">Systematic Perturbation of Thousands of Retroviral LTRs in Mouse Embryos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J.; Cook, L.; Chen, Z.</p>
        <p class="info">Score: 10.4, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558531' target='https://doi.org/10.1101/2023.09.19.558531'> 10.1101/2023.09.19.558531</a></p>
        <p class="abstract">In mammals, many retrotransposons are de-repressed during zygotic genome activation (ZGA). However, their functions in early development remain elusive largely due to the challenge to simultaneously manipulate thousands of retrotransposon insertions in embryos. Here, we employed epigenome editing to perturb the long terminal repeat (LTR) MT2_Mm, a well-known ZGA and totipotency marker that exists in [~]2667 insertions throughout the mouse genome. CRISPRi robustly repressed 2485 ([~] 93%) MT2_Mm insertions and 1090 ([~]55%) insertions of the closely related MT2C_Mm in 2-cell embryos. Remarkably, such perturbation caused down-regulation of hundreds of ZGA genes at the 2-cell stage and embryonic arrest mostly at the morula stage. Mechanistically, MT2_Mm/MT2C_Mm primarily served as alternative ZGA promoters activated by OBOX proteins. Thus, through unprecedented large-scale epigenome editing, we addressed to what extent MT2_Mm/MT2C_Mm regulates ZGA and preimplantation development. Our approach could be adapted to systematically perturb retrotransposons in other mammalian embryos as it doesnt require transgenic animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560491">
        <p class="paperTitle">Segmentation in tapeworms as a modified form of flatworm posterior regeneration involving Wnt and Hedgehog signalling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jarero, F.; Baillie, A.; Riddiford, N.; Montagne, J.; Koziol, U.; Olson, P. D.</p>
        <p class="info">Score: 6.1, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560491' target='https://doi.org/10.1101/2023.10.02.560491'> 10.1101/2023.10.02.560491</a></p>
        <p class="abstract">Tapeworms are parasitic flatworms that lack key features conventionally used to define the head-tail axis in free-living organisms, resulting in long standing questions regarding the true orientation of their main body axis. As adults, most tapeworms also exhibit a segmented body which has been considered an adaptation unique to the group. Anteroposterior (AP) patterning in free-living flatworms is controlled by {beta}-catenin-dependent Wnt signalling and positional control genes are expressed by their musculature in highly regionalised domains. Here we investigate the expression of Wnt and Hedgehog components during the strobilar phase of the tapeworm life cycle during which larval tissues are lost and replaced through the continuous production of new tissues. Results reveal previously unidentified centres of signalling associated with their neuromuscular system and show that segments are marked by secondary, AP axes in agreement with the polarity of the primary body axis. Neuromuscular expression and communication between Wnt and Hedgehog signalling is consistent with embryonic and regenerative growth in planarians and is a common mechanism for establishing AP-polarised boundaries in a diverse range of segmented animals. Taken together our results suggest that segmentation in tapeworms represents a modified form of posterior regeneration, a common feature among flatworms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560118">
        <p class="paperTitle">Functional opsin patterning for Drosophila color vision is established through signaling pathways in adjacent object-detection neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kitamata, M.; Otake, Y.; Kitagori, H.; Zhang, X.; Maki, Y.; Boku, R.; Takeuchi, M.; Nakagoshi, H.</p>
        <p class="info">Score: 8.2, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560118' target='https://doi.org/10.1101/2023.09.29.560118'> 10.1101/2023.09.29.560118</a></p>
        <p class="abstract">Vision is mainly based on two different tasks, object detection and color discrimination through activities of photoreceptor (PR) cells. Drosophila compound eye consists of [~]800 ommatidia. Every ommatidium contains eight PR cells; six outer cells (R1-R6) and two inner cells (R7 and R8) by which object detection and color vision are achieved, respectively. Expression of opsin genes in R7 and R8 is highly coordinated through the instructive signal from R7 to R8, and two major ommatidial subtypes are distributed stochastically; pale type expresses Rh3/Rh5, while yellow type expresses Rh4/Rh6 in R7/R8. The homeodomain protein Defective proventriculus (Dve) is expressed in yellow-type R7 and in six outer PRs, and it is involved in Rh3 repression to specify the yellow-type R7. dve mutant eyes exhibited atypical coupling, Rh3/Rh6 and Rh4/Rh5, indicating that the Dve activity is required for proper opsin coupling. Surprisingly, Dve activity in R1 is required for instructive signal, whereas those in R6 and R7 block the signal. Our results indicate that functional coupling of two different neurons is established through signaling pathways from adjacent neurons that are functionally different.

Summary StatementDve activity is required for proper opsin coupling of two neurons that is established through signaling pathways from adjacent neurons that are functionally different.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
