<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Meta-Atlas of the Developing Human Cortex Identifies Modules Driving Cell Subtype Specification
Authors: Nano, P. R.; Fazzari, E.; Azizad, D.; Nguyen, C. V.; Wang, S.; Kan, R. L.; Wick, B.; Haeussler, M.; Bhaduri, A.
Score: 26.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557406
Human brain development requires the generation of hundreds of diverse cell types, a process targeted by recent single-cell transcriptomic profiling efforts. Through a meta-analysis of seven of these published datasets, we have generated 225 meta-modules - gene co-expression networks that can describe mechanisms underlying cortical development.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="A Meta-Atlas of the Developing Human Cortex Identifies Modules Driving Cell Subtype Specification
Authors: Nano, P. R.; Fazzari, E.; Azizad, D.; Nguyen, C. V.; Wang, S.; Kan, R. L.; Wick, B.; Haeussler, M.; Bhaduri, A.
Score: 26.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557406
Human brain development requires the generation of hundreds of diverse cell types, a process targeted by recent single-cell transcriptomic profiling efforts. Through a meta-analysis of seven of these published datasets, we have generated 225 meta-modules - gene co-expression networks that can describe mechanisms underlying cortical development." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="A Meta-Atlas of the Developing Human Cortex Identifies Modules Driving Cell Subtype Specification
Authors: Nano, P. R.; Fazzari, E.; Azizad, D.; Nguyen, C. V.; Wang, S.; Kan, R. L.; Wick, B.; Haeussler, M.; Bhaduri, A.
Score: 26.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557406
Human brain development requires the generation of hundreds of diverse cell types, a process targeted by recent single-cell transcriptomic profiling efforts. Through a meta-analysis of seven of these published datasets, we have generated 225 meta-modules - gene co-expression networks that can describe mechanisms underlying cortical development."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "A Meta-Atlas of the Developing Human Cortex Identifies Modules Driving Cell Subtype Specification\nAuthors: Nano, P. R.; Fazzari, E.; Azizad, D.; Nguyen, C. V.; Wang, S.; Kan, R. L.; Wick, B.; Haeussler, M.; Bhaduri, A.\nScore: 26.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557406\nHuman brain development requires the generation of hundreds of diverse cell types, a process targeted by recent single-cell transcriptomic profiling efforts. Through a meta-analysis of seven of these published datasets, we have generated 225 meta-modules - gene co-expression networks that can describe mechanisms underlying cortical development.",
  "keywords": [
    
  ],
  "articleBody": " A Meta-Atlas of the Developing Human Cortex Identifies Modules Driving Cell Subtype Specification\nAuthors: Nano, P. R.; Fazzari, E.; Azizad, D.; Nguyen, C. V.; Wang, S.; Kan, R. L.; Wick, B.; Haeussler, M.; Bhaduri, A.\nScore: 26.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557406\nHuman brain development requires the generation of hundreds of diverse cell types, a process targeted by recent single-cell transcriptomic profiling efforts. Through a meta-analysis of seven of these published datasets, we have generated 225 meta-modules - gene co-expression networks that can describe mechanisms underlying cortical development. Several meta-modules have potential roles in both establishing and refining cortical cell type identities, and we validated their spatiotemporal expression in primary human cortical tissues. These include meta-module 20, associated with FEZF2+ deep layer neurons. Half of meta-module 20 genes are putative FEZF2 targets, including TSHZ3, a transcription factor associated with neurodevelopmental disorders. Human cortical organoid experiments validated that both factors are necessary for deep layer neuron specification. Importantly, subtle manipulations of these factors drive slight changes in meta-module activity that cascade into strong differences in cell fate - demonstrating how of our meta-atlas can engender further mechanistic analyses of cortical fate specification.\nSingle-cell epigenomic reconstruction of developmental trajectories in human neural organoid systems from pluripotency\nAuthors: Zenk, F.; Fleck, J. S.; Jansen, S. M. J.; Kashanian, B.; Eisinger, B.; Santel, M.; Dupre, J. S.; Camp, G.; Treutlein, B.\nScore: 19.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557341\nHuman cell type diversity emerges through a highly regulated series of fate restrictions from pluripotent progenitors. Fate restriction is orchestrated in part through epigenetic modifications at genes and regulatory elements, however it has been difficult to study these mechanisms in humans. Here, we use organoid models of the human central nervous system and establish single-cell profiling of histone modifications (H3K27ac, H3K27me3, H3K4me3) in organoid cells over a time course to reconstruct epigenomic trajectories governing cell identity acquisition from human pluripotency. We capture transitions from pluripotency through neuroepithelium, to retinal and brain region specification, as well as differentiation from progenitors to neuronal and glial terminal states. We find that switching of repressive and activating epigenetic modifications can precede and predict decisions at each stage, providing a temporal census of gene regulatory elements and transcription factors that we characterize in a gene regulatory network underlying human cerebral fate acquisition. We use transcriptome and chromatin accessibility measurements in the same cell from a human developing brain to validate this regulatory mode in a primary tissue. We show that abolishing histone 3 lysine 27 trimethylation (H3K27me3) through inhibition of the polycomb group protein Embryonic Ectoderm Development (EED) at the neuroectoderm stage disrupts fate restriction and leads to aberrant cell fate acquisition, ultimately influencing cell type composition in brain organoids. Altogether, our single-cell genome wide map of histone modifications during human neural organoid development serves as a blueprint (https://episcape.ethz.ch) to explore human cell fate decisions in normal physiology and in neurodevelopmental disorders. More broadly, this approach can be used to study human epigenomic trajectory mechanisms in any human organoid system. SummaryUnguided neural organoids reveal widespread and dynamic switching of epigenetic modifications during development and recapitulate fate restriction from pluripotency to terminally differentiated cells of the human central nervous system.\nmiR-137 conferred robustness to the territorial restriction of the neural plate border\nAuthors: Scatturice, L. A.; Vazquez, N.; Strobl-Mazzulla, P. H.\nScore: 16.8, Published: 2023-09-09 DOI: 10.1101/2023.09.08.556830\nThe neural plate border (NPB) of vertebrate embryos segregated from the neural plate (NP) and epidermal regions, and comprised an intermingled group of progenitors with multiple fate potential. Recent studies have shown that during the gastrula stage, TFAP2A acts as a pioneer factor in remodeling the epigenetic landscape required to activate components of the NPB induction program. Here we show that Tfap2a has two highly conserved binding sites for miR-137 and both display a reciprocal expression pattern at the NPB and NP respectively. In addition, ectopic miR-137 expression reduced TFAP2A, whereas its functional inhibition expanded their territorial distribution overlapping with PAX7. Furthermore, we demonstrated that loss of the de novo DNA methyltransferase DNMT3A expanded miR-137 expression to the NPB. Bisulfite sequencing showed a significantly higher level of non-canonical CpH methylation at the promoter of miR-137 when we compared NPB and NP samples. Our finding shows that miR-137 contributes to the robustness of NPB territorial restriction in vertebrate development.\nAn actomyosin network organizes niche morphology and responds to feedback from recruited stem cells\nAuthors: Warder, B. N.; Nelson, K. A.; Sui, J.; Anllo, L.; DiNardo, S.\nScore: 6.7, Published: 2023-09-09 DOI: 10.1101/2023.09.08.556877\nStem cells often rely on signals from a niche, which in many tissues adopts a precise morphology. What remains elusive is how niches are formed, and how morphology impacts function. To address this, we leverage the Drosophila gonadal niche, combining genetic tractability with live-imaging. We have previously shown that proper positioning of niche cells relies on signaling from visceral mesoderm. Here, we show that once positioned, niche cells robustly polarize filamentous actin and Non-muscle Myosin II (MyoII) towards neighboring germ cells. Actomyosin tension along the niche periphery generates a highly reproducible smoothened contour. Without contractility, niches are misshapen and exhibit defects in their ability to regulate germline stem cell behavior. We additionally show that germ cells aid in polarizing MyoII within niche cells, and that extrinsic input is required for niche morphogenesis and function. Our work reveals a feedback mechanism where stem cells shape the niche that guides their behavior.\nIdentification of amnion progenitor-like cells at the amnion-epiblast boundary in the primate peri-gastrula\nAuthors: Sekulovski, N.; Juga, L. L.; Cortez, C. L.; Czerwinski, M.; Whorton, A. E.; Spence, J. R.; Kropp Schmidt, J.; Golos, T.; Gumucio, D. L.; Lin, C.-W.; Taniguchi, K.\nScore: 6.6, Published: 2023-09-09 DOI: 10.1101/2023.09.07.556553\nAmniogenesis is triggered in a collection of pluripotent epiblast cells as the human embryo implants. To gain insights into the critical but poorly understood transcriptional machinery governing amnion fate determination, we examined the evolving transcriptome of a developing human pluripotent stem cell-derived amnion model at the single cell level. This analysis revealed a previously unrecognized CLDN10+ amnion progenitor cell type and enabled the identification of several markers specific to the different stages of lineage progression. Strikingly, we found that expression of CLDN10 is restricted to the amnion-epiblast boundary region in peri-gastrula cynomolgus macaque embryos, bolstering the notion that, at this stage, the amnion-epiblast boundary is a site of active amniogenesis. Bioinformatic analysis of published primate peri-gastrula single cell sequencing data further confirmed that CLDN10 is expressed in cells progressing to amnion. Overall, this study identifies a previously unrecognized CLDN10+ amnion progenitor-like cell population and validates the presence of these fate transitioning cells at the boundary of the developing amnion.\nLong-term three-dimensional high-resolution imaging of live unlabeled small intestinal organoids using low-coherence holotomography\nAuthors: Lee, M. J.; Lee, J.; Ha, J.; Kim, G.; Kim, H.-J.; Lee, S.; Koo, B.-K.; Park, Y.\nScore: 4.3, Published: 2023-09-16 DOI: 10.1101/2023.09.16.558039\nThe prevailing challenges in live unlabeled high-resolution imaging of native organoids stem from technical issues like complex sample handling and optical scattering in three-dimensional architectures. In this study, we introduce low-coherence holotomography as an advanced, label-free, quantitative imaging modality, designed to overcome related technical obstacles for long-term live imaging of 3D organoids. We successfully captured high-resolution morphological intricacies and dynamic events within mouse small intestinal organoids at a subcellular resolution. Furthermore, this method provides a unique advantage in differentiating between viable and non-viable organoids, thereby expanding its potential applications in organoid-based research.\nGerm cells do not progress through spermatogenesis in the infertile zebrafish testis\nAuthors: Sposato, A. L.; Llewellyn, D. R.; Weber, J. M.; Hollins, H. L.; Schrock, M. N.; Farrell, J. A.; Gagnon, J. A.\nScore: 15.3, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556432\nVertebrate spermatogonial stem cells maintain sperm production over the lifetime of an animal but fertility declines with age. While morphological studies have greatly informed our understanding of typical spermatogenesis, the molecular and cellular mechanisms underlying spermatogenesis are not yet understood, particularly with respect to the onset of fertility. We used single-cell RNA sequencing to generate a developmental atlas of the zebrafish testis. Using 5 timepoints across the adult life of a zebrafish, we described cellular profiles in the testis during and after fertility. While all germ cell stages of spermatogenesis are detected in testes from fertile adult zebrafish, testes from older infertile males only contained spermatogonia and a reduced population of spermatocytes. These remaining germ cells are transcriptionally distinct from fertile spermatogonia. Immune cells including macrophages and lymphocytes drastically increase in abundance in infertile testes. Our developmental atlas reveals the cellular changes as the testis ages and defines a molecular roadmap for the regulation of male fertility.\nA robust knock-in approach using a minimal promoter and a minicircle\nAuthors: Keating, M.; Hagle, R.; Osorio-Mendez, D.; Rodriguez, A.; Almutawa, S.; Kang, J.\nScore: 4.1, Published: 2023-09-16 DOI: 10.1101/2023.09.15.558008\nKnock-in reporter (KI) animals are essential tools in biomedical research to study gene expression impacting diverse biological events. While CRISPR/Cas9-mediated genome editing allows for the successful generation of KI animals, several factors should be considered, such as low expression of the target gene, prevention of bacterial DNA integration, and in-frame editing. To circumvent these challenges, we developed a new strategy that utilizes minicircle technology and introduces a minimal promoter. We demonstrated that minicircles serve as an efficient donor DNA in zebrafish, significantly enhancing KI events compared to plasmids containing bacterial backbones. In an attempt to generate a KI reporter for scn8ab, we precisely integrated a fluorescence gene at the start codon. However, the seamlessly integrated reporter was unable to direct expression that recapitulates endogenous scn8ab expression. To overcome this obstacle, we introduced the hsp70 minimal promoter to provide an ectopic transcription initiation site and succeeded in establishing stable KI transgenic reporters for scn8ab. This strategy also created a fgf20b KI reporter line with a high success rate. Furthermore, our data revealed that an unexpectedly edited genome can inappropriately influence the integrated reporter gene expression, highlighting the importance of selecting a proper KI line. Overall, our approach utilizing a minicircle and an ectopic promoter establishes a robust and efficient strategy for KI generation, expanding our capacity to create KI animals.\nCerebellar granule cell migration and folia development requires Mllt11/Af1q\nAuthors: Blommers, M.; Stanton-Turcotte, D.; Witt, E. A.; Heidari, M.; Iulianella, A.\nScore: 3.8, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557458\nThe organization of neurons into distinct layers, known as lamination, is a common feature of the nervous system. This process, which arises from the direct coupling of neurogenesis and neuronal migration, plays a crucial role in the development of the cerebellum, a structure exhibiting a distinct cytoarchitecture with cells arranged in discrete layers. Disruptions to neuronal migration and lamination can lead to various neurodevelopmental disorders, highlighting the significance of understanding their underlying regulators. Here, we report a role for a microtubule-interacting protein Mllt11/Af1q (Myeloid/lymphoid or mixed-lineage leukemia; translocated to chromosome 11/All1 Fused Gene From Chromosome 1q) in the migration of cerebellar granule cells (GCs). We show that Mllt11 may serve a similar role in both tangential and radial migration of excitatory GCs. Loss of Mllt11 led to an accumulation of GC precursors in the rhombic lip region and a reduction in the number of GCs successfully populating developing folia. Consequently, this results in smaller folia and an overall reduction in cerebellar size. Furthermore, analysis of the anchoring centres reveals disruptions in the perinatal folia cytoarchitecture, including alterations in the Bergmann glia fiber orientation and reduced infolding of the Purkinje cell plate. Taken together, the findings reported herein demonstrate a role for Mllt11 in regulating neuronal migration within the developing cerebellum, which is necessary for its proper neuroanatomical organization.\nMaternal IL-10 restricts fetal emergency myelopoiesis\nAuthors: Collins, A.; Swann, J. W.; Proven, M. A.; Patel, C.; Mitchell, C. A.; Kasbekar, M.; Dellorusso, P. V.; Passegue, E.\nScore: 3.6, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557548\nNeonates, in contrast to adults, are highly susceptible to inflammation and infection. Here we investigate how late fetal liver (FL) mouse hematopoietic stem and progenitor cells (HSPC) respond to inflammation, testing the hypothesis that deficits in engagement of emergency myelopoiesis (EM) pathways limit neutrophil output and contribute to perinatal neutropenia. We show that despite similar molecular wiring as adults, fetal HSPCs have limited production of myeloid cells at steady state and fail to activate a classical EM transcriptional program. Moreover, we find that fetal HSPCs are capable of responding to EM-inducing inflammatory stimuli in vitro, but are restricted by maternal anti-inflammatory factors, primarily interleukin-10 (IL-10), from activating EM pathways in utero. Accordingly, we demonstrate that loss of maternal IL-10 restores EM activation in fetal HSPCs but at the cost of premature parturition. These results reveal the evolutionary trade-off inherent in maternal anti-inflammatory responses that maintain pregnancy but render the fetus unresponsive to EM activation signals and susceptible to infection. HIGHLIGHTSO_LIThe structure of the HSPC compartment is conserved from late fetal to adult life. C_LIO_LIFetal HSPCs have diminished steady-state myeloid cell production compared to adult. C_LIO_LIFetal HSPCs are restricted from engaging in emergency myelopoiesis by maternal IL-10. C_LIO_LIRestriction of emergency myelopoiesis may explain neutropenia in septic neonates. C_LI eTOC BLURBFetal hematopoietic stem and progenitor cells are restricted from activating emergency myelopoiesis pathways by maternal IL-10, resulting in inadequate myeloid cell production in response to inflammatory challenges and contributing to neonatal neutropenia.\n",
  "wordCount" : "2194",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557406">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557406" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557406">
        <p class="paperTitle">A Meta-Atlas of the Developing Human Cortex Identifies Modules Driving Cell Subtype Specification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557406" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557406" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nano, P. R.; Fazzari, E.; Azizad, D.; Nguyen, C. V.; Wang, S.; Kan, R. L.; Wick, B.; Haeussler, M.; Bhaduri, A.</p>
        <p class="info">Score: 26.0, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557406' target='https://doi.org/10.1101/2023.09.12.557406'> 10.1101/2023.09.12.557406</a></p>
        <p class="abstract">Human brain development requires the generation of hundreds of diverse cell types, a process targeted by recent single-cell transcriptomic profiling efforts. Through a meta-analysis of seven of these published datasets, we have generated 225 meta-modules - gene co-expression networks that can describe mechanisms underlying cortical development. Several meta-modules have potential roles in both establishing and refining cortical cell type identities, and we validated their spatiotemporal expression in primary human cortical tissues. These include meta-module 20, associated with FEZF2&#43; deep layer neurons. Half of meta-module 20 genes are putative FEZF2 targets, including TSHZ3, a transcription factor associated with neurodevelopmental disorders. Human cortical organoid experiments validated that both factors are necessary for deep layer neuron specification. Importantly, subtle manipulations of these factors drive slight changes in meta-module activity that cascade into strong differences in cell fate - demonstrating how of our meta-atlas can engender further mechanistic analyses of cortical fate specification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557341">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557341" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557341">
        <p class="paperTitle">Single-cell epigenomic reconstruction of developmental trajectories in human neural organoid systems from pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557341" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557341" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zenk, F.; Fleck, J. S.; Jansen, S. M. J.; Kashanian, B.; Eisinger, B.; Santel, M.; Dupre, J. S.; Camp, G.; Treutlein, B.</p>
        <p class="info">Score: 19.9, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557341' target='https://doi.org/10.1101/2023.09.12.557341'> 10.1101/2023.09.12.557341</a></p>
        <p class="abstract">Human cell type diversity emerges through a highly regulated series of fate restrictions from pluripotent progenitors. Fate restriction is orchestrated in part through epigenetic modifications at genes and regulatory elements, however it has been difficult to study these mechanisms in humans. Here, we use organoid models of the human central nervous system and establish single-cell profiling of histone modifications (H3K27ac, H3K27me3, H3K4me3) in organoid cells over a time course to reconstruct epigenomic trajectories governing cell identity acquisition from human pluripotency. We capture transitions from pluripotency through neuroepithelium, to retinal and brain region specification, as well as differentiation from progenitors to neuronal and glial terminal states. We find that switching of repressive and activating epigenetic modifications can precede and predict decisions at each stage, providing a temporal census of gene regulatory elements and transcription factors that we characterize in a gene regulatory network underlying human cerebral fate acquisition. We use transcriptome and chromatin accessibility measurements in the same cell from a human developing brain to validate this regulatory mode in a primary tissue. We show that abolishing histone 3 lysine 27 trimethylation (H3K27me3) through inhibition of the polycomb group protein Embryonic Ectoderm Development (EED) at the neuroectoderm stage disrupts fate restriction and leads to aberrant cell fate acquisition, ultimately influencing cell type composition in brain organoids. Altogether, our single-cell genome wide map of histone modifications during human neural organoid development serves as a blueprint (https://episcape.ethz.ch) to explore human cell fate decisions in normal physiology and in neurodevelopmental disorders. More broadly, this approach can be used to study human epigenomic trajectory mechanisms in any human organoid system.

SummaryUnguided neural organoids reveal widespread and dynamic switching of epigenetic modifications during development and recapitulate fate restriction from pluripotency to terminally differentiated cells of the human central nervous system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556830">
        <p class="paperTitle">miR-137 conferred robustness to the territorial restriction of the neural plate border</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scatturice, L. A.; Vazquez, N.; Strobl-Mazzulla, P. H.</p>
        <p class="info">Score: 16.8, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556830' target='https://doi.org/10.1101/2023.09.08.556830'> 10.1101/2023.09.08.556830</a></p>
        <p class="abstract">The neural plate border (NPB) of vertebrate embryos segregated from the neural plate (NP) and epidermal regions, and comprised an intermingled group of progenitors with multiple fate potential. Recent studies have shown that during the gastrula stage, TFAP2A acts as a pioneer factor in remodeling the epigenetic landscape required to activate components of the NPB induction program. Here we show that Tfap2a has two highly conserved binding sites for miR-137 and both display a reciprocal expression pattern at the NPB and NP respectively. In addition, ectopic miR-137 expression reduced TFAP2A, whereas its functional inhibition expanded their territorial distribution overlapping with PAX7. Furthermore, we demonstrated that loss of the de novo DNA methyltransferase DNMT3A expanded miR-137 expression to the NPB. Bisulfite sequencing showed a significantly higher level of non-canonical CpH methylation at the promoter of miR-137 when we compared NPB and NP samples. Our finding shows that miR-137 contributes to the robustness of NPB territorial restriction in vertebrate development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556877">
        <p class="paperTitle">An actomyosin network organizes niche morphology and responds to feedback from recruited stem cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Warder, B. N.; Nelson, K. A.; Sui, J.; Anllo, L.; DiNardo, S.</p>
        <p class="info">Score: 6.7, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556877' target='https://doi.org/10.1101/2023.09.08.556877'> 10.1101/2023.09.08.556877</a></p>
        <p class="abstract">Stem cells often rely on signals from a niche, which in many tissues adopts a precise morphology. What remains elusive is how niches are formed, and how morphology impacts function. To address this, we leverage the Drosophila gonadal niche, combining genetic tractability with live-imaging. We have previously shown that proper positioning of niche cells relies on signaling from visceral mesoderm. Here, we show that once positioned, niche cells robustly polarize filamentous actin and Non-muscle Myosin II (MyoII) towards neighboring germ cells. Actomyosin tension along the niche periphery generates a highly reproducible smoothened contour. Without contractility, niches are misshapen and exhibit defects in their ability to regulate germline stem cell behavior. We additionally show that germ cells aid in polarizing MyoII within niche cells, and that extrinsic input is required for niche morphogenesis and function. Our work reveals a feedback mechanism where stem cells shape the niche that guides their behavior.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556553" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556553">
        <p class="paperTitle">Identification of amnion progenitor-like cells at the amnion-epiblast boundary in the primate peri-gastrula</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sekulovski, N.; Juga, L. L.; Cortez, C. L.; Czerwinski, M.; Whorton, A. E.; Spence, J. R.; Kropp Schmidt, J.; Golos, T.; Gumucio, D. L.; Lin, C.-W.; Taniguchi, K.</p>
        <p class="info">Score: 6.6, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556553' target='https://doi.org/10.1101/2023.09.07.556553'> 10.1101/2023.09.07.556553</a></p>
        <p class="abstract">Amniogenesis is triggered in a collection of pluripotent epiblast cells as the human embryo implants. To gain insights into the critical but poorly understood transcriptional machinery governing amnion fate determination, we examined the evolving transcriptome of a developing human pluripotent stem cell-derived amnion model at the single cell level. This analysis revealed a previously unrecognized CLDN10&#43; amnion progenitor cell type and enabled the identification of several markers specific to the different stages of lineage progression. Strikingly, we found that expression of CLDN10 is restricted to the amnion-epiblast boundary region in peri-gastrula cynomolgus macaque embryos, bolstering the notion that, at this stage, the amnion-epiblast boundary is a site of active amniogenesis. Bioinformatic analysis of published primate peri-gastrula single cell sequencing data further confirmed that CLDN10 is expressed in cells progressing to amnion. Overall, this study identifies a previously unrecognized CLDN10&#43; amnion progenitor-like cell population and validates the presence of these fate transitioning cells at the boundary of the developing amnion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.558039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.558039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.558039">
        <p class="paperTitle">Long-term three-dimensional high-resolution imaging of live unlabeled small intestinal organoids using low-coherence holotomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.558039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.558039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. J.; Lee, J.; Ha, J.; Kim, G.; Kim, H.-J.; Lee, S.; Koo, B.-K.; Park, Y.</p>
        <p class="info">Score: 4.3, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.558039' target='https://doi.org/10.1101/2023.09.16.558039'> 10.1101/2023.09.16.558039</a></p>
        <p class="abstract">The prevailing challenges in live unlabeled high-resolution imaging of native organoids stem from technical issues like complex sample handling and optical scattering in three-dimensional architectures. In this study, we introduce low-coherence holotomography as an advanced, label-free, quantitative imaging modality, designed to overcome related technical obstacles for long-term live imaging of 3D organoids. We successfully captured high-resolution morphological intricacies and dynamic events within mouse small intestinal organoids at a subcellular resolution. Furthermore, this method provides a unique advantage in differentiating between viable and non-viable organoids, thereby expanding its potential applications in organoid-based research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556432">
        <p class="paperTitle">Germ cells do not progress through spermatogenesis in the infertile zebrafish testis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sposato, A. L.; Llewellyn, D. R.; Weber, J. M.; Hollins, H. L.; Schrock, M. N.; Farrell, J. A.; Gagnon, J. A.</p>
        <p class="info">Score: 15.3, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556432' target='https://doi.org/10.1101/2023.09.05.556432'> 10.1101/2023.09.05.556432</a></p>
        <p class="abstract">Vertebrate spermatogonial stem cells maintain sperm production over the lifetime of an animal but fertility declines with age. While morphological studies have greatly informed our understanding of typical spermatogenesis, the molecular and cellular mechanisms underlying spermatogenesis are not yet understood, particularly with respect to the onset of fertility. We used single-cell RNA sequencing to generate a developmental atlas of the zebrafish testis. Using 5 timepoints across the adult life of a zebrafish, we described cellular profiles in the testis during and after fertility. While all germ cell stages of spermatogenesis are detected in testes from fertile adult zebrafish, testes from older infertile males only contained spermatogonia and a reduced population of spermatocytes. These remaining germ cells are transcriptionally distinct from fertile spermatogonia. Immune cells including macrophages and lymphocytes drastically increase in abundance in infertile testes. Our developmental atlas reveals the cellular changes as the testis ages and defines a molecular roadmap for the regulation of male fertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.558008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.558008" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.558008">
        <p class="paperTitle">A robust knock-in approach using a minimal promoter and a minicircle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.558008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.558008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keating, M.; Hagle, R.; Osorio-Mendez, D.; Rodriguez, A.; Almutawa, S.; Kang, J.</p>
        <p class="info">Score: 4.1, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.558008' target='https://doi.org/10.1101/2023.09.15.558008'> 10.1101/2023.09.15.558008</a></p>
        <p class="abstract">Knock-in reporter (KI) animals are essential tools in biomedical research to study gene expression impacting diverse biological events. While CRISPR/Cas9-mediated genome editing allows for the successful generation of KI animals, several factors should be considered, such as low expression of the target gene, prevention of bacterial DNA integration, and in-frame editing. To circumvent these challenges, we developed a new strategy that utilizes minicircle technology and introduces a minimal promoter. We demonstrated that minicircles serve as an efficient donor DNA in zebrafish, significantly enhancing KI events compared to plasmids containing bacterial backbones. In an attempt to generate a KI reporter for scn8ab, we precisely integrated a fluorescence gene at the start codon. However, the seamlessly integrated reporter was unable to direct expression that recapitulates endogenous scn8ab expression. To overcome this obstacle, we introduced the hsp70 minimal promoter to provide an ectopic transcription initiation site and succeeded in establishing stable KI transgenic reporters for scn8ab. This strategy also created a fgf20b KI reporter line with a high success rate. Furthermore, our data revealed that an unexpectedly edited genome can inappropriately influence the integrated reporter gene expression, highlighting the importance of selecting a proper KI line. Overall, our approach utilizing a minicircle and an ectopic promoter establishes a robust and efficient strategy for KI generation, expanding our capacity to create KI animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557458">
        <p class="paperTitle">Cerebellar granule cell migration and folia development requires Mllt11/Af1q</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blommers, M.; Stanton-Turcotte, D.; Witt, E. A.; Heidari, M.; Iulianella, A.</p>
        <p class="info">Score: 3.8, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557458' target='https://doi.org/10.1101/2023.09.12.557458'> 10.1101/2023.09.12.557458</a></p>
        <p class="abstract">The organization of neurons into distinct layers, known as lamination, is a common feature of the nervous system. This process, which arises from the direct coupling of neurogenesis and neuronal migration, plays a crucial role in the development of the cerebellum, a structure exhibiting a distinct cytoarchitecture with cells arranged in discrete layers. Disruptions to neuronal migration and lamination can lead to various neurodevelopmental disorders, highlighting the significance of understanding their underlying regulators. Here, we report a role for a microtubule-interacting protein Mllt11/Af1q (Myeloid/lymphoid or mixed-lineage leukemia; translocated to chromosome 11/All1 Fused Gene From Chromosome 1q) in the migration of cerebellar granule cells (GCs). We show that Mllt11 may serve a similar role in both tangential and radial migration of excitatory GCs. Loss of Mllt11 led to an accumulation of GC precursors in the rhombic lip region and a reduction in the number of GCs successfully populating developing folia. Consequently, this results in smaller folia and an overall reduction in cerebellar size. Furthermore, analysis of the anchoring centres reveals disruptions in the perinatal folia cytoarchitecture, including alterations in the Bergmann glia fiber orientation and reduced infolding of the Purkinje cell plate. Taken together, the findings reported herein demonstrate a role for Mllt11 in regulating neuronal migration within the developing cerebellum, which is necessary for its proper neuroanatomical organization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557548">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557548" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557548">
        <p class="paperTitle">Maternal IL-10 restricts fetal emergency myelopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557548" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557548" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, A.; Swann, J. W.; Proven, M. A.; Patel, C.; Mitchell, C. A.; Kasbekar, M.; Dellorusso, P. V.; Passegue, E.</p>
        <p class="info">Score: 3.6, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557548' target='https://doi.org/10.1101/2023.09.13.557548'> 10.1101/2023.09.13.557548</a></p>
        <p class="abstract">Neonates, in contrast to adults, are highly susceptible to inflammation and infection. Here we investigate how late fetal liver (FL) mouse hematopoietic stem and progenitor cells (HSPC) respond to inflammation, testing the hypothesis that deficits in engagement of emergency myelopoiesis (EM) pathways limit neutrophil output and contribute to perinatal neutropenia. We show that despite similar molecular wiring as adults, fetal HSPCs have limited production of myeloid cells at steady state and fail to activate a classical EM transcriptional program. Moreover, we find that fetal HSPCs are capable of responding to EM-inducing inflammatory stimuli in vitro, but are restricted by maternal anti-inflammatory factors, primarily interleukin-10 (IL-10), from activating EM pathways in utero. Accordingly, we demonstrate that loss of maternal IL-10 restores EM activation in fetal HSPCs but at the cost of premature parturition. These results reveal the evolutionary trade-off inherent in maternal anti-inflammatory responses that maintain pregnancy but render the fetus unresponsive to EM activation signals and susceptible to infection.

HIGHLIGHTSO_LIThe structure of the HSPC compartment is conserved from late fetal to adult life.
C_LIO_LIFetal HSPCs have diminished steady-state myeloid cell production compared to adult.
C_LIO_LIFetal HSPCs are restricted from engaging in emergency myelopoiesis by maternal IL-10.
C_LIO_LIRestriction of emergency myelopoiesis may explain neutropenia in septic neonates.
C_LI

eTOC BLURBFetal hematopoietic stem and progenitor cells are restricted from activating emergency myelopoiesis pathways by maternal IL-10, resulting in inadequate myeloid cell production in response to inflammatory challenges and contributing to neonatal neutropenia.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
