<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-cell transcriptional dynamics in a living vertebrate
Authors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.
Score: 22.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574108
The ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Single-cell transcriptional dynamics in a living vertebrate
Authors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.
Score: 22.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574108
The ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Single-cell transcriptional dynamics in a living vertebrate
Authors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.
Score: 22.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574108
The ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Single-cell transcriptional dynamics in a living vertebrate\nAuthors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.\nScore: 22.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574108\nThe ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos.",
  "keywords": [
    
  ],
  "articleBody": " Single-cell transcriptional dynamics in a living vertebrate\nAuthors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.\nScore: 22.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574108\nThe ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos. We addressed this deficit by applying MS2-MCP mRNA labeling to the quantification of transcription in zebrafish, a model vertebrate. We developed a platform of transgenic organisms, light sheet fluorescence microscopy, and optimized image analysis that enables visualization and quantification of MS2 reporters. We used these tools to obtain the first single-cell, real-time measurements of transcriptional dynamics of the segmentation clock. Our measurements challenge the traditional view of smooth clock oscillations and instead suggest a model of discrete transcriptional bursts that are organized in space and time. Together, these results highlight how measuring single-cell transcriptional activity can reveal unexpected features of gene regulation and how this data can fuel the dialogue between theory and experiment.\nLoss of αBa-crystallin, but not αA-crystallin, increases age-related cataract in the zebrafish lens\nAuthors: Posner, M.; Garver, T.; Kaye, T.; Brdicka, S.; Suttle, M.; Patterson, B.; Farnsworth, D. R.\nScore: 6.7, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574085\nThe vertebrate eye lens is an unusual organ in that most of its cells lack nuclei and the ability to replace aging protein. The small heat shock protein -crystallins evolved to become key components of this lens, possibly because of their ability to prevent aggregation of aging protein that would otherwise lead to lens opacity. Most vertebrates express two -crystallins, A- and B-crystallin, and mutations in each are linked to human cataract. In a mouse knockout model only the loss of A-crystallin led to early-stage lens cataract. We have used the zebrafish as a model system to investigate the role of -crystallins during lens development. Interestingly, while zebrafish express one lens-specific A-crystallin gene (cryaa), they express two B-crystallin genes, with one evolving lens specificity (cryaba) and the other retaining the broad expression of its mammalian ortholog (cryabb). In this study we used individual mutant zebrafish lines for all three -crystallin genes to determine the impact of their loss on age-related cataract. Surprisingly, unlike mouse knockout models, we found that the loss of the Ba-crystallin gene cryaba led to an increase in lens opacity compared to cryaa null fish at 24 months of age. Loss of A-crystallin did not increase the prevalence of cataract. We also used single cell RNA-Seq and RT-qPCR data to show a shift in the lens expression of zebrafish -crystallins between 5 and 10 days post fertilization (dpf), with 5 and 6 dpf lenses expressing cryaa almost exclusively, and expression of cryaba and cryabb becoming more prominent after 10 dpf. These data show that cryaa is the primary -crystallin during early lens development, while the protective role for cryaba becomes more important during lens aging. This study is the first to quantify cataract prevalence in wild-type zebrafish, showing that lens opacities develop in approximately 25% of fish by 18 months of age. None of the three -crystallin mutants showed a compensatory increase in the expression of the remaining two crystallins, or in the abundant {beta}B1-crystallin. Overall, these findings indicate an ontogenetic shift in the functional importance of individual -crystallins during zebrafish lens development. Our finding that the lens-specific zebrafish Ba-crystallin plays the leading role in preventing age-related cataract adds a new twist to our understanding of vertebrate lens evolution.\nCooperation between Nodal and FGF signals regulates zebrafish cardiac cell migration and heart morphogenesis\nAuthors: Gonzalez, V.; Grant, M. G.; Suzuki, M.; Christophers, B.; Williams, J. R.; Burdine, R. D.\nScore: 4.2, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574380\nAsymmetric vertebrate heart development is driven by an intricate sequence of morphogenetic cell movements, the coordination of which requires precise interpretation of signaling cues by heart primordia. Here we show that Nodal functions cooperatively with FGF during heart tube formation and asymmetric placement. Both pathways act as migratory stimuli for cardiac progenitor cells (CPCs), but FGF is dispensable for directing heart tube asymmetry, which is governed by Nodal. We further find that Nodal controls CPC migration by inducing left-right asymmetries in the formation of actin-based protrusions in CPCs. Additionally, we define a developmental window in which FGF signals are required for proper heart looping and show cooperativity between FGF and Nodal in this process. We present evidence FGF may promote heart looping through addition of the secondary heart field. Finally, we demonstrate that loss of FGF signaling affects proper development of the atrioventricular canal (AVC), which likely contributes to abnormal chamber morphologies in FGF-deficient hearts. Together, our data shed insight into how the spatiotemporal dynamics of signaling cues regulate the cellular behaviors underlying organ morphogenesis.\nTolerance thresholds underlie responses to DNA damage during germline development\nAuthors: Jansen, G.; Gebert, D.; Ravindra Kumar, T.; Simmons, E.; Murphy, S.; Karam Teixeira, F.\nScore: 2.1, Published: 2024-01-07 DOI: 10.1101/2024.01.07.574510\nSelfish DNA modules like transposable elements (TEs) are particularly active in the germline, the lineage that passes genetic information across generations. New TE insertions can disrupt genes and impair the functionality and viability of germ cells. However, we find that in P-M hybrid dysgenesis in Drosophila, a sterility syndrome triggered by the P-element DNA transposon, germ cells harbour unexpectedly few new TE insertions, despite accumulating DNA double-strand breaks (DSBs) and inducing cell cycle arrest. Using an engineered CRISPR-Cas9 system, we show that generating DSBs at silenced P-elements or other non-coding sequences is sufficient to induce germ cell loss independently of gene disruption. Indeed, we demonstrate that both developing and adult mitotic germ cells are sensitive to DSBs in a dosage-dependent manner. Following the mitotic-to-meiotic transition, however, germ cells become more tolerant to DSBs, completing oogenesis regardless of the accumulated genome damage. Our findings establish DNA damage tolerance thresholds as crucial safeguards of genome integrity during germline development.\nSingle cell proteomics characterization of bone marrow hematopoiesis with distinct Ras pathway lesions.\nAuthors: Karra, L.; Finger, A.-M.; Shechtman, L.; Krush, M.; Meng-Yao Huang, R.; Prinz, M.; Tennvooren, I.; Bahl, K.; Hysenaj, L.; Gonzalez, P.; combes, a. J.; Gonzalez, H.; Arguello, R.; Spitzer, M.; Roose, J.\nScore: 4.9, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572584\nNormal hematopoiesis requires constant prolific production of different blood cell lineages by multipotent hematopoietic stem cells (HSC). Stem- and progenitor- cells need to balance dormancy with proliferation. How genetic alterations impact frequency, lineage potential, and metabolism of HSC is largely unknown. Here, we compared induced expression of KRASG12D or RasGRP1 to normal hematopoiesis. At low-resolution, both Ras pathway lesions result in skewing towards myeloid lineages. Single-cell resolution CyTOF proteomics unmasked an expansion of HSC- and progenitor- compartments for RasGRP1, contrasted by a depletion for KRASG12D. SCENITH quantitates protein synthesis with single-cell precision and corroborated that immature cells display low metabolic SCENITH rates. Both RasGRP1 and KRASG12D elevated mean SCENITH signals in immature cells. However, RasGRP1-overexpressing stem cells retain a metabolically quiescent cell-fraction, whereas this fraction diminishes for KRASG12D. Our temporal single cell proteomics and metabolomics datasets provide a resource of mechanistic insights into altered hematopoiesis at single cell resolution.\nModularity of the segmentation clock and morphogenesis\nAuthors: Hammond, J. E.; Baker, R. E.; Verd, B.\nScore: 1.8, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574679\nVertebrates have evolved great diversity in the number of segments dividing trunk body features such as the musculature, nervous system, and skeleton, however the developmental origin for this evolvability is poorly understood. The number of segments is determined by the number of somites, transient blocks of mesodermal tissue, that form in the early embryo. These form rhythmically at a rate thought to be determined by the dynamics of a molecular oscillator known as the segmentation clock, and the morphogenesis of the pre-somitic mesoderm (PSM) is thought to control the total length of time over which somites are formed. Here we explore whether the clock and PSM morphogenesis exhibit developmental modularity, as independent evolution of these two processes may explain the high degree of evolvability observed in vertebrate segment number. Using a computational model of the clock and PSM development parameterised for Zebrafish (Danio rerio), we find that the clock is broadly robust to variation in morphogenetic processes such as cell ingression, motility, compaction, and cell division. We show that this robustness is in part determined by the length of the PSM and the strength of phase coupling in the clock. As previous studies report no changes to morphogenesis upon perturbing the clock, we conclude based on our study that the clock and morphogenesis of the PSM exhibit developmental modularity.\nPersistence of the primary somatosensory system in zebrafish\nAuthors: Navajas Acedo, J.\nScore: 51.6, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572352\nThe somatosensory system detects peripheral stimuli that are translated into behaviors necessary for survival. Fishes and amphibians possess two somatosensory systems in the trunk: the primary somatosensory system, formed by the Rohon-Beard neurons, and the secondary somatosensory system, formed by the neural crest cell-derived neurons of the Dorsal Root Ganglia. Rohon-Beard neurons have been characterized as a transient population that mostly disappears during the first days of life and is functionally replaced by the Dorsal Root Ganglia. Here, I follow Rohon-Beard neurons in vivo and show that the entire repertoire remains present in zebrafish from 1-day post-fertilization until the juvenile stage, 15-days post-fertilization. These data indicate that zebrafish retain two complete somatosensory systems until at least up to a developmental stage when the animals display complex behavioral repertoires.\nGap junction mediated bioelectric coordination is required for slow muscle development, organization, and function\nAuthors: Lukowicz-Bedford, R. M.; Eisen, J. S.; Miller, A. C.\nScore: 3.6, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572619\nBioelectrical signaling, intercellular communication facilitated by membrane potential and electrochemical coupling, is emerging as a key regulator of animal development. Gap junction (GJ) channels can mediate bioelectric signaling by creating a fast, direct pathway between cells for the movement of ions and other small molecules. In vertebrates, GJ channels are formed by a highly conserved transmembrane protein family called the Connexins. The connexin gene family is large and complex, presenting a challenge in identifying the specific Connexins that create channels within developing and mature tissues. Using the embryonic zebrafish neuromuscular system as a model, we identify a connexin conserved across vertebrate lineages, gjd4, which encodes the Cx46.8 protein, that mediates bioelectric signaling required for appropriate slow muscle development and function. Through a combination of mutant analysis and in vivo imaging we show that gjd4/Cx46.8 creates GJ channels specifically in developing slow muscle cells. Using genetics, pharmacology, and calcium imaging we find that spinal cord generated neural activity is transmitted to developing slow muscle cells and synchronized activity spreads via gjd4/Cx46.8 GJ channels. Finally, we show that bioelectrical signal propagation within the developing neuromuscular system is required for appropriate myofiber organization, and that disruption leads to defects in behavior. Our work reveals the molecular basis for GJ communication among developing muscle cells and reveals how perturbations to bioelectric signaling in the neuromuscular system may contribute to developmental myopathies. Moreover, this work underscores a critical motif of signal propagation between organ systems and highlights the pivotal role played by GJ communication in coordinating bioelectric signaling during development.\nEndosomal-lysosomal organellar assembly (ELYSA) structures coordinate lysosomal degradation systems through mammalian oocyte-to-embryo transition\nAuthors: Satouh, Y.; Tatebe, T.; Tanida, I.; Yamaguchi, J.; Uchiyama, Y.; Sato, K.\nScore: 2.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573616\nMouse oocytes mature from the germinal vesicle (GV) to meiosis II (MII) stage and are fertilized by a single sperm. After fertilization, the embryo degrades parts of the maternal components via lysosomal degradation systems, including autophagy and endocytosis, as zygotic gene expression begins during embryogenesis. Here, we demonstrate that endosomal-lysosomal organelles form assembly structures (ELYSAs) in the periphery of the oocyte plasma membrane. Small ELYSA structures appeared in GV oocytes and combined to form large structures in MII oocytes. The ELYSA forms a large spherical structure with tubular-vesicular structures approximately 2-10 m in diameter and includes cytosolic components at the MII stage. After fertilization, ELYSA structures are gradually disassembled, and immature endosomes and lysosomes appear at the early 2-cell stages. The V1-subunit of vacuolar ATPase is recruited to these endosomes and lysosomes and causes further acidification of endosomal-lysosomal organelles, suggesting that the ELYSA maintains endosomal-lysosomal activity in a static state in oocytes for timely activation during early development.\nAn adaptive numerical method for multi-cellular simulations of organ development and disease\nAuthors: Osborne, J. M.\nScore: 1.2, Published: 2024-01-07 DOI: 10.1101/2024.01.06.574290\nIn recent years, multi-cellular models, where cells are represented as individual interacting entities, are becoming ever popular. This has led to a proliferation of novel methods and simulation tools. The first aim of this paper is to review the numerical methods utilised by multi-cellular modelling tools and to demonstrate which numerical methods are appropriate for simulations of tissue and organ development and disease. The second aim is to introduce an adaptive time-stepping algorithm and to demonstrate its efficiency and accuracy. We focus on off-lattice, mechanics based, models where cell movement is defined by a series of differential equations, derived by considering a force balance. We see that many numerical methods have been used, ranging from simple Forward Euler approaches through to higher order single-step methods like Runge-Kutta 4 and multi-step methods like Adams-Bashforth 2. Through a series of exemplar multi-cellular simulations, we see that if: care is taken to have events (births deaths and re-meshing/re-arrangements) occur on common time-steps; and boundaries are imposed on all sub-steps of numerical methods or implemented using forces, then all numerical methods can converge with the correct order. We introduce an adaptive time-stepping method and demonstrate that the best compromise between L{infty} error and run-time is to use Runge-Kutta 4 with an increased time-step and moderate adaptivity. We see that a judicious choice of numerical method can speed the simulation up by a factor of 10 and a further speed up by a factor of 4 can be achieved by using an adaptive time-step.\n",
  "wordCount" : "2328",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574108" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574108">
        <p class="paperTitle">Single-cell transcriptional dynamics in a living vertebrate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eck, E.; Moretti, B.; Schlomann, B. H.; Bragantini, J.; Lange, M.; Zhao, X.; VijayKumar, S.; Valentin, G.; Loureiro, C.; Soroldoni, D.; Royer, L. A.; Oates, A. C.; Garcia, H. H.</p>
        <p class="info">Score: 22.3, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574108' target='https://doi.org/10.1101/2024.01.03.574108'> 10.1101/2024.01.03.574108</a></p>
        <p class="abstract">The ability to quantify transcriptional dynamics in individual cells via live imaging has revolutionized our understanding of gene regulation. However, such measurements are lacking in the context of vertebrate embryos. We addressed this deficit by applying MS2-MCP mRNA labeling to the quantification of transcription in zebrafish, a model vertebrate. We developed a platform of transgenic organisms, light sheet fluorescence microscopy, and optimized image analysis that enables visualization and quantification of MS2 reporters. We used these tools to obtain the first single-cell, real-time measurements of transcriptional dynamics of the segmentation clock. Our measurements challenge the traditional view of smooth clock oscillations and instead suggest a model of discrete transcriptional bursts that are organized in space and time. Together, these results highlight how measuring single-cell transcriptional activity can reveal unexpected features of gene regulation and how this data can fuel the dialogue between theory and experiment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574085" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574085">
        <p class="paperTitle">Loss of αBa-crystallin, but not αA-crystallin, increases age-related cataract in the zebrafish lens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Posner, M.; Garver, T.; Kaye, T.; Brdicka, S.; Suttle, M.; Patterson, B.; Farnsworth, D. R.</p>
        <p class="info">Score: 6.7, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574085' target='https://doi.org/10.1101/2024.01.03.574085'> 10.1101/2024.01.03.574085</a></p>
        <p class="abstract">The vertebrate eye lens is an unusual organ in that most of its cells lack nuclei and the ability to replace aging protein. The small heat shock protein -crystallins evolved to become key components of this lens, possibly because of their ability to prevent aggregation of aging protein that would otherwise lead to lens opacity. Most vertebrates express two -crystallins, A- and B-crystallin, and mutations in each are linked to human cataract. In a mouse knockout model only the loss of A-crystallin led to early-stage lens cataract. We have used the zebrafish as a model system to investigate the role of -crystallins during lens development. Interestingly, while zebrafish express one lens-specific A-crystallin gene (cryaa), they express two B-crystallin genes, with one evolving lens specificity (cryaba) and the other retaining the broad expression of its mammalian ortholog (cryabb). In this study we used individual mutant zebrafish lines for all three -crystallin genes to determine the impact of their loss on age-related cataract. Surprisingly, unlike mouse knockout models, we found that the loss of the Ba-crystallin gene cryaba led to an increase in lens opacity compared to cryaa null fish at 24 months of age. Loss of A-crystallin did not increase the prevalence of cataract. We also used single cell RNA-Seq and RT-qPCR data to show a shift in the lens expression of zebrafish -crystallins between 5 and 10 days post fertilization (dpf), with 5 and 6 dpf lenses expressing cryaa almost exclusively, and expression of cryaba and cryabb becoming more prominent after 10 dpf. These data show that cryaa is the primary -crystallin during early lens development, while the protective role for cryaba becomes more important during lens aging. This study is the first to quantify cataract prevalence in wild-type zebrafish, showing that lens opacities develop in approximately 25% of fish by 18 months of age. None of the three -crystallin mutants showed a compensatory increase in the expression of the remaining two crystallins, or in the abundant {beta}B1-crystallin. Overall, these findings indicate an ontogenetic shift in the functional importance of individual -crystallins during zebrafish lens development. Our finding that the lens-specific zebrafish Ba-crystallin plays the leading role in preventing age-related cataract adds a new twist to our understanding of vertebrate lens evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574380">
        <p class="paperTitle">Cooperation between Nodal and FGF signals regulates zebrafish cardiac cell migration and heart morphogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez, V.; Grant, M. G.; Suzuki, M.; Christophers, B.; Williams, J. R.; Burdine, R. D.</p>
        <p class="info">Score: 4.2, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574380' target='https://doi.org/10.1101/2024.01.05.574380'> 10.1101/2024.01.05.574380</a></p>
        <p class="abstract">Asymmetric vertebrate heart development is driven by an intricate sequence of morphogenetic cell movements, the coordination of which requires precise interpretation of signaling cues by heart primordia. Here we show that Nodal functions cooperatively with FGF during heart tube formation and asymmetric placement. Both pathways act as migratory stimuli for cardiac progenitor cells (CPCs), but FGF is dispensable for directing heart tube asymmetry, which is governed by Nodal. We further find that Nodal controls CPC migration by inducing left-right asymmetries in the formation of actin-based protrusions in CPCs. Additionally, we define a developmental window in which FGF signals are required for proper heart looping and show cooperativity between FGF and Nodal in this process. We present evidence FGF may promote heart looping through addition of the secondary heart field. Finally, we demonstrate that loss of FGF signaling affects proper development of the atrioventricular canal (AVC), which likely contributes to abnormal chamber morphologies in FGF-deficient hearts. Together, our data shed insight into how the spatiotemporal dynamics of signaling cues regulate the cellular behaviors underlying organ morphogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574510">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574510" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574510">
        <p class="paperTitle">Tolerance thresholds underlie responses to DNA damage during germline development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574510" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574510" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansen, G.; Gebert, D.; Ravindra Kumar, T.; Simmons, E.; Murphy, S.; Karam Teixeira, F.</p>
        <p class="info">Score: 2.1, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574510' target='https://doi.org/10.1101/2024.01.07.574510'> 10.1101/2024.01.07.574510</a></p>
        <p class="abstract">Selfish DNA modules like transposable elements (TEs) are particularly active in the germline, the lineage that passes genetic information across generations. New TE insertions can disrupt genes and impair the functionality and viability of germ cells. However, we find that in P-M hybrid dysgenesis in Drosophila, a sterility syndrome triggered by the P-element DNA transposon, germ cells harbour unexpectedly few new TE insertions, despite accumulating DNA double-strand breaks (DSBs) and inducing cell cycle arrest. Using an engineered CRISPR-Cas9 system, we show that generating DSBs at silenced P-elements or other non-coding sequences is sufficient to induce germ cell loss independently of gene disruption. Indeed, we demonstrate that both developing and adult mitotic germ cells are sensitive to DSBs in a dosage-dependent manner. Following the mitotic-to-meiotic transition, however, germ cells become more tolerant to DSBs, completing oogenesis regardless of the accumulated genome damage. Our findings establish DNA damage tolerance thresholds as crucial safeguards of genome integrity during germline development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572584">
        <p class="paperTitle">Single cell proteomics characterization of bone marrow hematopoiesis with distinct Ras pathway lesions.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karra, L.; Finger, A.-M.; Shechtman, L.; Krush, M.; Meng-Yao Huang, R.; Prinz, M.; Tennvooren, I.; Bahl, K.; Hysenaj, L.; Gonzalez, P.; combes, a. J.; Gonzalez, H.; Arguello, R.; Spitzer, M.; Roose, J.</p>
        <p class="info">Score: 4.9, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572584' target='https://doi.org/10.1101/2023.12.20.572584'> 10.1101/2023.12.20.572584</a></p>
        <p class="abstract">Normal hematopoiesis requires constant prolific production of different blood cell lineages by multipotent hematopoietic stem cells (HSC). Stem- and progenitor- cells need to balance dormancy with proliferation. How genetic alterations impact frequency, lineage potential, and metabolism of HSC is largely unknown. Here, we compared induced expression of KRASG12D or RasGRP1 to normal hematopoiesis. At low-resolution, both Ras pathway lesions result in skewing towards myeloid lineages. Single-cell resolution CyTOF proteomics unmasked an expansion of HSC- and progenitor- compartments for RasGRP1, contrasted by a depletion for KRASG12D. SCENITH quantitates protein synthesis with single-cell precision and corroborated that immature cells display low metabolic SCENITH rates. Both RasGRP1 and KRASG12D elevated mean SCENITH signals in immature cells. However, RasGRP1-overexpressing stem cells retain a metabolically quiescent cell-fraction, whereas this fraction diminishes for KRASG12D. Our temporal single cell proteomics and metabolomics datasets provide a resource of mechanistic insights into altered hematopoiesis at single cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574679" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574679">
        <p class="paperTitle">Modularity of the segmentation clock and morphogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hammond, J. E.; Baker, R. E.; Verd, B.</p>
        <p class="info">Score: 1.8, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574679' target='https://doi.org/10.1101/2024.01.08.574679'> 10.1101/2024.01.08.574679</a></p>
        <p class="abstract">Vertebrates have evolved great diversity in the number of segments dividing trunk body features such as the musculature, nervous system, and skeleton, however the developmental origin for this evolvability is poorly understood. The number of segments is determined by the number of somites, transient blocks of mesodermal tissue, that form in the early embryo. These form rhythmically at a rate thought to be determined by the dynamics of a molecular oscillator known as the segmentation clock, and the morphogenesis of the pre-somitic mesoderm (PSM) is thought to control the total length of time over which somites are formed. Here we explore whether the clock and PSM morphogenesis exhibit developmental modularity, as independent evolution of these two processes may explain the high degree of evolvability observed in vertebrate segment number. Using a computational model of the clock and PSM development parameterised for Zebrafish (Danio rerio), we find that the clock is broadly robust to variation in morphogenetic processes such as cell ingression, motility, compaction, and cell division. We show that this robustness is in part determined by the length of the PSM and the strength of phase coupling in the clock. As previous studies report no changes to morphogenesis upon perturbing the clock, we conclude based on our study that the clock and morphogenesis of the PSM exhibit developmental modularity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572352" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572352">
        <p class="paperTitle">Persistence of the primary somatosensory system in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Navajas Acedo, J.</p>
        <p class="info">Score: 51.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572352' target='https://doi.org/10.1101/2023.12.19.572352'> 10.1101/2023.12.19.572352</a></p>
        <p class="abstract">The somatosensory system detects peripheral stimuli that are translated into behaviors necessary for survival. Fishes and amphibians possess two somatosensory systems in the trunk: the primary somatosensory system, formed by the Rohon-Beard neurons, and the secondary somatosensory system, formed by the neural crest cell-derived neurons of the Dorsal Root Ganglia. Rohon-Beard neurons have been characterized as a transient population that mostly disappears during the first days of life and is functionally replaced by the Dorsal Root Ganglia. Here, I follow Rohon-Beard neurons in vivo and show that the entire repertoire remains present in zebrafish from 1-day post-fertilization until the juvenile stage, 15-days post-fertilization. These data indicate that zebrafish retain two complete somatosensory systems until at least up to a developmental stage when the animals display complex behavioral repertoires.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572619" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572619">
        <p class="paperTitle">Gap junction mediated bioelectric coordination is required for slow muscle development, organization, and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lukowicz-Bedford, R. M.; Eisen, J. S.; Miller, A. C.</p>
        <p class="info">Score: 3.6, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572619' target='https://doi.org/10.1101/2023.12.20.572619'> 10.1101/2023.12.20.572619</a></p>
        <p class="abstract">Bioelectrical signaling, intercellular communication facilitated by membrane potential and electrochemical coupling, is emerging as a key regulator of animal development. Gap junction (GJ) channels can mediate bioelectric signaling by creating a fast, direct pathway between cells for the movement of ions and other small molecules. In vertebrates, GJ channels are formed by a highly conserved transmembrane protein family called the Connexins. The connexin gene family is large and complex, presenting a challenge in identifying the specific Connexins that create channels within developing and mature tissues. Using the embryonic zebrafish neuromuscular system as a model, we identify a connexin conserved across vertebrate lineages, gjd4, which encodes the Cx46.8 protein, that mediates bioelectric signaling required for appropriate slow muscle development and function. Through a combination of mutant analysis and in vivo imaging we show that gjd4/Cx46.8 creates GJ channels specifically in developing slow muscle cells. Using genetics, pharmacology, and calcium imaging we find that spinal cord generated neural activity is transmitted to developing slow muscle cells and synchronized activity spreads via gjd4/Cx46.8 GJ channels. Finally, we show that bioelectrical signal propagation within the developing neuromuscular system is required for appropriate myofiber organization, and that disruption leads to defects in behavior. Our work reveals the molecular basis for GJ communication among developing muscle cells and reveals how perturbations to bioelectric signaling in the neuromuscular system may contribute to developmental myopathies. Moreover, this work underscores a critical motif of signal propagation between organ systems and highlights the pivotal role played by GJ communication in coordinating bioelectric signaling during development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573616">
        <p class="paperTitle">Endosomal-lysosomal organellar assembly (ELYSA) structures coordinate lysosomal degradation systems through mammalian oocyte-to-embryo transition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Satouh, Y.; Tatebe, T.; Tanida, I.; Yamaguchi, J.; Uchiyama, Y.; Sato, K.</p>
        <p class="info">Score: 2.2, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573616' target='https://doi.org/10.1101/2023.12.29.573616'> 10.1101/2023.12.29.573616</a></p>
        <p class="abstract">Mouse oocytes mature from the germinal vesicle (GV) to meiosis II (MII) stage and are fertilized by a single sperm. After fertilization, the embryo degrades parts of the maternal components via lysosomal degradation systems, including autophagy and endocytosis, as zygotic gene expression begins during embryogenesis. Here, we demonstrate that endosomal-lysosomal organelles form assembly structures (ELYSAs) in the periphery of the oocyte plasma membrane. Small ELYSA structures appeared in GV oocytes and combined to form large structures in MII oocytes. The ELYSA forms a large spherical structure with tubular-vesicular structures approximately 2-10 m in diameter and includes cytosolic components at the MII stage. After fertilization, ELYSA structures are gradually disassembled, and immature endosomes and lysosomes appear at the early 2-cell stages. The V1-subunit of vacuolar ATPase is recruited to these endosomes and lysosomes and causes further acidification of endosomal-lysosomal organelles, suggesting that the ELYSA maintains endosomal-lysosomal activity in a static state in oocytes for timely activation during early development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574290">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574290" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574290">
        <p class="paperTitle">An adaptive numerical method for multi-cellular simulations of organ development and disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574290" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574290" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Osborne, J. M.</p>
        <p class="info">Score: 1.2, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574290' target='https://doi.org/10.1101/2024.01.06.574290'> 10.1101/2024.01.06.574290</a></p>
        <p class="abstract">In recent years, multi-cellular models, where cells are represented as individual interacting entities, are becoming ever popular. This has led to a proliferation of novel methods and simulation tools. The first aim of this paper is to review the numerical methods utilised by multi-cellular modelling tools and to demonstrate which numerical methods are appropriate for simulations of tissue and organ development and disease. The second aim is to introduce an adaptive time-stepping algorithm and to demonstrate its efficiency and accuracy. We focus on off-lattice, mechanics based, models where cell movement is defined by a series of differential equations, derived by considering a force balance. We see that many numerical methods have been used, ranging from simple Forward Euler approaches through to higher order single-step methods like Runge-Kutta 4 and multi-step methods like Adams-Bashforth 2. Through a series of exemplar multi-cellular simulations, we see that if: care is taken to have events (births deaths and re-meshing/re-arrangements) occur on common time-steps; and boundaries are imposed on all sub-steps of numerical methods or implemented using forces, then all numerical methods can converge with the correct order. We introduce an adaptive time-stepping method and demonstrate that the best compromise between L{infty} error and run-time is to use Runge-Kutta 4 with an increased time-step and moderate adaptivity. We see that a judicious choice of numerical method can speed the simulation up by a factor of 10 and a further speed up by a factor of 4 can be achieved by using an adaptive time-step.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
