<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods
Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.
Score: 36.3, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381
Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods
Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.
Score: 36.3, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381
Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods
Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.
Score: 36.3, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381
Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods\nAuthors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.\nScore: 36.3, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381\nCell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP.",
  "keywords": [
    
  ],
  "articleBody": " PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods\nAuthors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.\nScore: 36.3, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381\nCell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP. To achieve sensitive labelling, we amplified the fluorescent signal by fusing s36GFP to the ultra-bright protein mNeonGreen. Secretor cells transfer s36GFP-mNeonGreen to neighbours while retaining a nuclear mCherry, making it straightforward to unambiguously identify, isolate, and profile neighbours of secretor cells. Transfer of the s36GFP-mNeonGreen fusion occurs within minutes, enabling even relatively transient interactions to be captured or tracked over time with live-imaging or flow cytometry. To enhance flexibility of use, we incorporated HaloTag technology to facilitate colour- of-choice labelling. To further increase the range of applications, we engineered a customisable single-plasmid construct composed of interchangeable components with option to incorporate any additional transgene. This allows users to manipulate the properties of a cell, while at the same time applying a fluorescent label to the surrounding cells. PUFFFIN offers a simple, sensitive, customisable approach to profile non-cell-autonomous responses to natural or induced changes in cell identity or behaviour in mammalian cells. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=168 SRC=\"FIGDIR/small/556381v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (38K): org.highwire.dtl.DTLVardef@1f5f624org.highwire.dtl.DTLVardef@f2e193org.highwire.dtl.DTLVardef@1b77d75org.highwire.dtl.DTLVardef@18da6d9_HPS_FORMAT_FIGEXP M_FIG C_FIG\nGerm cells do not progress through spermatogenesis in the infertile zebrafish testis\nAuthors: Sposato, A. L.; Llewellyn, D. R.; Weber, J. M.; Hollins, H. L.; Schrock, M. N.; Farrell, J. A.; Gagnon, J. A.\nScore: 14.3, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556432\nVertebrate spermatogonial stem cells maintain sperm production over the lifetime of an animal but fertility declines with age. While morphological studies have greatly informed our understanding of typical spermatogenesis, the molecular and cellular mechanisms underlying spermatogenesis are not yet understood, particularly with respect to the onset of fertility. We used single-cell RNA sequencing to generate a developmental atlas of the zebrafish testis. Using 5 timepoints across the adult life of a zebrafish, we described cellular profiles in the testis during and after fertility. While all germ cell stages of spermatogenesis are detected in testes from fertile adult zebrafish, testes from older infertile males only contained spermatogonia and a reduced population of spermatocytes. These remaining germ cells are transcriptionally distinct from fertile spermatogonia. Immune cells including macrophages and lymphocytes drastically increase in abundance in infertile testes. Our developmental atlas reveals the cellular changes as the testis ages and defines a molecular roadmap for the regulation of male fertility.\nKLF7 is a general inducer of human pluripotency\nAuthors: Arboit, M.; Zorzan, I.; Pellegrini, M.; Martini, P.; Carbognin, E.; Martello, G.\nScore: 11.8, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556189\nPluripotency is the capacity to give rise to all differentiated cells of the body and the germ line and is governed by a self-reinforcing network of transcription factors. The forced expression of only some of these factors enables the reprogramming of somatic cells to pluripotency. In murine cells, several kruppel-like factors (KLFs) have been identified as stabilisers and inducers of pluripotency. Human somatic cells are routinely reprogrammed by expression of KLF4 in combination with OCT4, SOX2 and cMYC (OSKM). An extensive transcriptome analysis revealed, however, that KLF4 is barely expressed in conventional human pluripotent stem cells (PSCs). Here we show that KLF7 is robustly expressed in conventional human PSCs and it allows transcription factor-mediated somatic reprogramming. Moreover, we find that KLF7 is highly expressed in naive PSCs and its forced expression in conventional hPSCs induces upregulation of naive markers and boosts efficiency of chemical resetting to naive PSCs, overall suggesting that KLF7 is a general human pluripotency factor and an inducer of pluripotency.\nmiR-137 conferred robustness to the territorial restriction of the neural plate border\nAuthors: Scatturice, L. A.; Vazquez, N.; Strobl-Mazzulla, P. H.\nScore: 9.6, Published: 2023-09-09 DOI: 10.1101/2023.09.08.556830\nThe neural plate border (NPB) of vertebrate embryos segregated from the neural plate (NP) and epidermal regions, and comprised an intermingled group of progenitors with multiple fate potential. Recent studies have shown that during the gastrula stage, TFAP2A acts as a pioneer factor in remodeling the epigenetic landscape required to activate components of the NPB induction program. Here we show that Tfap2a has two highly conserved binding sites for miR-137 and both display a reciprocal expression pattern at the NPB and NP respectively. In addition, ectopic miR-137 expression reduced TFAP2A, whereas its functional inhibition expanded their territorial distribution overlapping with PAX7. Furthermore, we demonstrated that loss of the de novo DNA methyltransferase DNMT3A expanded miR-137 expression to the NPB. Bisulfite sequencing showed a significantly higher level of non-canonical CpH methylation at the promoter of miR-137 when we compared NPB and NP samples. Our finding shows that miR-137 contributes to the robustness of NPB territorial restriction in vertebrate development.\nMorphodynamics of human early brain organoid development\nAuthors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.\nScore: 86.0, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827\nBrain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrixinduced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of nontelencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.\nSpatiotemporal dynamics of cytokines expression dictate fetal liver hematopoiesis.\nAuthors: Peixoto, M. M.; Soares-da-Silva, F.; Bonnet, V.; Ronteix, G.; Santos, R. F.; Mailhe, M.-P.; Feng, X.; Pereira, J.; Azzoni, E.; Anselmi, G.; de Bruijn, M.; Baroud, C.; Pinto-do-O, P.; Cumano, A.\nScore: 8.7, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554612\nDuring embryogenesis, yolk-sac and intra-embryonic-derived hematopoietic progenitors, comprising the precursors of adult hematopoietic stem cells, converge into the fetal liver. With a new staining strategy, we defined all non-hematopoietic components of the fetal liver and found that hepatoblasts are the major producers of hematopoietic growth factors. We identified mesothelial cells, a novel component of the stromal compartment, producing Kit ligand, a major hematopoietic cytokine. A high-definition imaging dataset analyzed using a deep-learning based pipeline allowed the unambiguous identification of hematopoietic and stromal populations, and enabled determining a neighboring network composition, at the single cell resolution. Throughout active hematopoiesis, progenitors preferentially associate with hepatoblasts, but not with stellate or endothelial cells. We found that, unlike yolk sac-derived progenitors, intra-embryonic progenitors respond to a chemokine gradient created by CXCL12-producing stellate cells. These results revealed that FL hematopoiesis is a spatiotemporal dynamic process, defined by an environment characterized by low cytokine concentrations.\nPrimate-expressed EPIREGULIN promotes basal progenitor proliferation in the developing neocortex\nAuthors: Cubillos, P.; Ditzer, N.; Kolodziejczyk, A.; Schwenk, G.; Hoffmann, J.; Schütze, T. M.; Derihaci, R. P.; Birdir, C.; Köllner, J. E. M.; Petzold, A.; Sarov, M.; Martin, U.; Long, K. R.; Wimberger, P.; Albert, M.\nScore: 29.6, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554446\nNeocortex expansion during evolution is linked to higher numbers of neurons thought to result from increased proliferative capacity and neurogenic potential of basal progenitor cells (BPs) during development. Here we show that EREG, encoding the growth factor EPIREGULIN, is expressed in the human developing neocortex and in gorilla organoids, but not in the mouse neocortex. Addition of EPIREGULIN to the mouse neocortex increases proliferation of BPs via EGFR-mediated signaling, whereas ablation of EREG in human cortical organoids reduces BP proliferation. Addition of EPIREGULIN to cortical organoids promotes a further increase in proliferation of gorilla but not human BPs. Finally, we identify putative cis-regulatory elements that may contribute to inter-species differences in EREG expression. Overall, our results suggest that species-specific expression of EPIREGULIN may contribute to increased neocortex size in primates by providing a pro-proliferative signal to BPs in the subventricular zone progenitor niche.\nDifferential regulation of the proteome and phosphosproteome along the dorso-ventral axis of the early Drosophila embryo\nAuthors: Gomez, J. M.; Nolte, H.; Vogelsang, E.; Dey, B.; Takeda, M.; Giudice, G.; Faxel, M.; Cepraga, A.; Zinzen, R. P.; Krueger, M.; Petsalaki, E.; Wang, Y.-C.; Leptin, M.\nScore: 11.4, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554590\nThe initially homogeneous epithelium of the early Drosophila embryo differentiates into regional subpopulations with different behaviours and physical properties that are needed for morphogenesis. The factors at top of the genetic hierarchy that control these behaviours are known, but many of their targets are not. To understand how proteins work together to mediate differential cellular activities, we studied in an unbiased manner the proteomes and phosphoproteomes of the three main cell populations along the dorso-ventral axis during gastrulation using mutant embryos that represent the different populations. We detected 6111 protein groups and 6259 phosphosites of which 3399 and 3433 respectively, were differentially regulated. The changes in phosphosite abundance did not correlate with changes in host protein abundance, showing phosphorylation to be a regulatory step during gastrulation. Hierarchical clustering of protein groups and phosphosites identified clusters that contain known fate determinants such as Doc1, Sog, Snail and Twist. The recovery of the appropriate known marker proteins in each of the different mutants we used validated the approach, but also revealed that two mutations that both interfere with the dorsal fate pathway, Toll10B and serpin27aex do this in very different manners. Diffused network analyses within each cluster point to microtubule components as one of the main groups of regulated proteins. Functional studies on the role of microtubules provide the proof of principle that microtubules have different functions in different domains along the DV axis of the embryo.\nA distinct class of hematopoietic stem cells develop from the human yellow bone marrow\nAuthors: Nguyen, T. T.; Loureiro, Z. Y.; Desai, A.; DeSouza, T.; Joyce, S.; Khair, L.; Samant, A.; Cirka, H.; Solivan-Rivera, J.; Ziegler, R.; Brehm, M.; Messina, L. M.; Corvera, S.\nScore: 9.4, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555167\nAging and metabolic diseases are accompanied by systemic inflammation, but the mechanisms that induce this state are not known. We developed a human bone-marrow organoid system to explore mechanisms underlying metabolic-disease associated systemic inflammation. We find that a distinct type of hematopoietic stem cell (HSC) develops in the adipose-rich, yellow bone marrow, which is known to gradually replace the hematopoietic red marrow as we age and during metabolic disease. Unlike HSCs derived from the red bone marrow, HSCs derived from the yellow bone marrow have higher proliferation rates, increase myeloid differentiation, skew towards pro-inflammatory M1 macrophage differentiation, and express a distinct transcriptomic profile associated with responsiveness to wounding. Yellow marrow-derived HSCs express higher levels of the leptin receptor, which we find to be further increased in patients with type 2 diabetes. Our work demonstrates that the human long bone yellow marrow is a niche for a distinct class of HSCs which could underlie hematopoietic dysfunction during aging and metabolic disease processes suggesting a shared inflammaging mechanism.\nAutomated staging of zebrafish embryos with deep learning\nAuthors: Jones, R. A.; Renshaw, M. J.; Devenport, D.; Barry, D. J.\nScore: 12.2, Published: 2023-08-17 DOI: 10.1101/2023.01.13.523922\nThe zebrafish (Danio rerio), is an important biomedical model organism used in many disciplines. The phenomenon of developmental delay in zebrafish embryos has been widely reported as part of a mutant or treatment-induced phenotype. However, the detection and quantification of these delays is often achieved through manual observation with reference to staging guides, which is both time-consuming and subjective. We recently reported a machine learning-based classifier, capable of quantifying the developmental delay between two populations of zebrafish embryos. Here, we build on that work by introducing a deep learning-based model (KimmelNet) that has been trained to predict the age (hours post fertilisation) of populations of zebrafish embryos. We show that when KimmelNet is tested on 2D brightfield images of zebrafish embryos, the predictions generated agree closely with those expected from established approaches to staging. Random sampling of the test data demonstrate that KimmelNet can be used to detect developmental delay between two populations with high confidence based on as few as 100 images of each population. Finally, we show that KimmelNet generalises to previously unseen data, with limited transfer learning improving this performance significantly. With the ability to analyse tens of thousands of standard brightfield microscopy images on a timescale of minutes, we envisage that KimmelNet will be a valuable resource for the developmental biology community. Furthermore, the approach we have used could easily be adapted to generate models for other organisms.\n",
  "wordCount" : "2260",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556381">
        <p class="paperTitle">PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.</p>
        <p class="info">Score: 36.3, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556381' target='https://doi.org/10.1101/2023.09.06.556381'> 10.1101/2023.09.06.556381</a></p>
        <p class="abstract">Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP. To achieve sensitive labelling, we amplified the fluorescent signal by fusing s36GFP to the ultra-bright protein mNeonGreen. Secretor cells transfer s36GFP-mNeonGreen to neighbours while retaining a nuclear mCherry, making it straightforward to unambiguously identify, isolate, and profile neighbours of secretor cells. Transfer of the s36GFP-mNeonGreen fusion occurs within minutes, enabling even relatively transient interactions to be captured or tracked over time with live-imaging or flow cytometry. To enhance flexibility of use, we incorporated HaloTag technology to facilitate colour- of-choice labelling. To further increase the range of applications, we engineered a customisable single-plasmid construct composed of interchangeable components with option to incorporate any additional transgene. This allows users to manipulate the properties of a cell, while at the same time applying a fluorescent label to the surrounding cells. PUFFFIN offers a simple, sensitive, customisable approach to profile non-cell-autonomous responses to natural or induced changes in cell identity or behaviour in mammalian cells.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=168 SRC=&#34;FIGDIR/small/556381v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (38K):
org.highwire.dtl.DTLVardef@1f5f624org.highwire.dtl.DTLVardef@f2e193org.highwire.dtl.DTLVardef@1b77d75org.highwire.dtl.DTLVardef@18da6d9_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556432">
        <p class="paperTitle">Germ cells do not progress through spermatogenesis in the infertile zebrafish testis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sposato, A. L.; Llewellyn, D. R.; Weber, J. M.; Hollins, H. L.; Schrock, M. N.; Farrell, J. A.; Gagnon, J. A.</p>
        <p class="info">Score: 14.3, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556432' target='https://doi.org/10.1101/2023.09.05.556432'> 10.1101/2023.09.05.556432</a></p>
        <p class="abstract">Vertebrate spermatogonial stem cells maintain sperm production over the lifetime of an animal but fertility declines with age. While morphological studies have greatly informed our understanding of typical spermatogenesis, the molecular and cellular mechanisms underlying spermatogenesis are not yet understood, particularly with respect to the onset of fertility. We used single-cell RNA sequencing to generate a developmental atlas of the zebrafish testis. Using 5 timepoints across the adult life of a zebrafish, we described cellular profiles in the testis during and after fertility. While all germ cell stages of spermatogenesis are detected in testes from fertile adult zebrafish, testes from older infertile males only contained spermatogonia and a reduced population of spermatocytes. These remaining germ cells are transcriptionally distinct from fertile spermatogonia. Immune cells including macrophages and lymphocytes drastically increase in abundance in infertile testes. Our developmental atlas reveals the cellular changes as the testis ages and defines a molecular roadmap for the regulation of male fertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556189">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556189" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556189">
        <p class="paperTitle">KLF7 is a general inducer of human pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556189" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556189" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arboit, M.; Zorzan, I.; Pellegrini, M.; Martini, P.; Carbognin, E.; Martello, G.</p>
        <p class="info">Score: 11.8, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556189' target='https://doi.org/10.1101/2023.09.06.556189'> 10.1101/2023.09.06.556189</a></p>
        <p class="abstract">Pluripotency is the capacity to give rise to all differentiated cells of the body and the germ line and is governed by a self-reinforcing network of transcription factors. The forced expression of only some of these factors enables the reprogramming of somatic cells to pluripotency. In murine cells, several kruppel-like factors (KLFs) have been identified as stabilisers and inducers of pluripotency. Human somatic cells are routinely reprogrammed by expression of KLF4 in combination with OCT4, SOX2 and cMYC (OSKM). An extensive transcriptome analysis revealed, however, that KLF4 is barely expressed in conventional human pluripotent stem cells (PSCs). Here we show that KLF7 is robustly expressed in conventional human PSCs and it allows transcription factor-mediated somatic reprogramming. Moreover, we find that KLF7 is highly expressed in naive PSCs and its forced expression in conventional hPSCs induces upregulation of naive markers and boosts efficiency of chemical resetting to naive PSCs, overall suggesting that KLF7 is a general human pluripotency factor and an inducer of pluripotency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556830">
        <p class="paperTitle">miR-137 conferred robustness to the territorial restriction of the neural plate border</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scatturice, L. A.; Vazquez, N.; Strobl-Mazzulla, P. H.</p>
        <p class="info">Score: 9.6, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556830' target='https://doi.org/10.1101/2023.09.08.556830'> 10.1101/2023.09.08.556830</a></p>
        <p class="abstract">The neural plate border (NPB) of vertebrate embryos segregated from the neural plate (NP) and epidermal regions, and comprised an intermingled group of progenitors with multiple fate potential. Recent studies have shown that during the gastrula stage, TFAP2A acts as a pioneer factor in remodeling the epigenetic landscape required to activate components of the NPB induction program. Here we show that Tfap2a has two highly conserved binding sites for miR-137 and both display a reciprocal expression pattern at the NPB and NP respectively. In addition, ectopic miR-137 expression reduced TFAP2A, whereas its functional inhibition expanded their territorial distribution overlapping with PAX7. Furthermore, we demonstrated that loss of the de novo DNA methyltransferase DNMT3A expanded miR-137 expression to the NPB. Bisulfite sequencing showed a significantly higher level of non-canonical CpH methylation at the promoter of miR-137 when we compared NPB and NP samples. Our finding shows that miR-137 contributes to the robustness of NPB territorial restriction in vertebrate development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553827">
        <p class="paperTitle">Morphodynamics of human early brain organoid development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.</p>
        <p class="info">Score: 86.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553827' target='https://doi.org/10.1101/2023.08.21.553827'> 10.1101/2023.08.21.553827</a></p>
        <p class="abstract">Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrixinduced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of nontelencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554612">
        <p class="paperTitle">Spatiotemporal dynamics of cytokines expression dictate fetal liver hematopoiesis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peixoto, M. M.; Soares-da-Silva, F.; Bonnet, V.; Ronteix, G.; Santos, R. F.; Mailhe, M.-P.; Feng, X.; Pereira, J.; Azzoni, E.; Anselmi, G.; de Bruijn, M.; Baroud, C.; Pinto-do-O, P.; Cumano, A.</p>
        <p class="info">Score: 8.7, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554612' target='https://doi.org/10.1101/2023.08.24.554612'> 10.1101/2023.08.24.554612</a></p>
        <p class="abstract">During embryogenesis, yolk-sac and intra-embryonic-derived hematopoietic progenitors, comprising the precursors of adult hematopoietic stem cells, converge into the fetal liver. With a new staining strategy, we defined all non-hematopoietic components of the fetal liver and found that hepatoblasts are the major producers of hematopoietic growth factors. We identified mesothelial cells, a novel component of the stromal compartment, producing Kit ligand, a major hematopoietic cytokine.

A high-definition imaging dataset analyzed using a deep-learning based pipeline allowed the unambiguous identification of hematopoietic and stromal populations, and enabled determining a neighboring network composition, at the single cell resolution.

Throughout active hematopoiesis, progenitors preferentially associate with hepatoblasts, but not with stellate or endothelial cells. We found that, unlike yolk sac-derived progenitors, intra-embryonic progenitors respond to a chemokine gradient created by CXCL12-producing stellate cells. These results revealed that FL hematopoiesis is a spatiotemporal dynamic process, defined by an environment characterized by low cytokine concentrations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554446">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554446" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554446">
        <p class="paperTitle">Primate-expressed EPIREGULIN promotes basal progenitor proliferation in the developing neocortex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554446" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554446" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cubillos, P.; Ditzer, N.; Kolodziejczyk, A.; Schwenk, G.; Hoffmann, J.; Schütze, T. M.; Derihaci, R. P.; Birdir, C.; Köllner, J. E. M.; Petzold, A.; Sarov, M.; Martin, U.; Long, K. R.; Wimberger, P.; Albert, M.</p>
        <p class="info">Score: 29.6, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554446' target='https://doi.org/10.1101/2023.08.23.554446'> 10.1101/2023.08.23.554446</a></p>
        <p class="abstract">Neocortex expansion during evolution is linked to higher numbers of neurons thought to result from increased proliferative capacity and neurogenic potential of basal progenitor cells (BPs) during development. Here we show that EREG, encoding the growth factor EPIREGULIN, is expressed in the human developing neocortex and in gorilla organoids, but not in the mouse neocortex. Addition of EPIREGULIN to the mouse neocortex increases proliferation of BPs via EGFR-mediated signaling, whereas ablation of EREG in human cortical organoids reduces BP proliferation. Addition of EPIREGULIN to cortical organoids promotes a further increase in proliferation of gorilla but not human BPs. Finally, we identify putative cis-regulatory elements that may contribute to inter-species differences in EREG expression. Overall, our results suggest that species-specific expression of EPIREGULIN may contribute to increased neocortex size in primates by providing a pro-proliferative signal to BPs in the subventricular zone progenitor niche.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554590">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554590" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554590">
        <p class="paperTitle">Differential regulation of the proteome and phosphosproteome along the dorso-ventral axis of the early Drosophila embryo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554590" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554590" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gomez, J. M.; Nolte, H.; Vogelsang, E.; Dey, B.; Takeda, M.; Giudice, G.; Faxel, M.; Cepraga, A.; Zinzen, R. P.; Krueger, M.; Petsalaki, E.; Wang, Y.-C.; Leptin, M.</p>
        <p class="info">Score: 11.4, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554590' target='https://doi.org/10.1101/2023.08.24.554590'> 10.1101/2023.08.24.554590</a></p>
        <p class="abstract">The initially homogeneous epithelium of the early Drosophila embryo differentiates into regional subpopulations with different behaviours and physical properties that are needed for morphogenesis. The factors at top of the genetic hierarchy that control these behaviours are known, but many of their targets are not. To understand how proteins work together to mediate differential cellular activities, we studied in an unbiased manner the proteomes and phosphoproteomes of the three main cell populations along the dorso-ventral axis during gastrulation using mutant embryos that represent the different populations. We detected 6111 protein groups and 6259 phosphosites of which 3399 and 3433 respectively, were differentially regulated. The changes in phosphosite abundance did not correlate with changes in host protein abundance, showing phosphorylation to be a regulatory step during gastrulation. Hierarchical clustering of protein groups and phosphosites identified clusters that contain known fate determinants such as Doc1, Sog, Snail and Twist. The recovery of the appropriate known marker proteins in each of the different mutants we used validated the approach, but also revealed that two mutations that both interfere with the dorsal fate pathway, Toll10B and serpin27aex do this in very different manners. Diffused network analyses within each cluster point to microtubule components as one of the main groups of regulated proteins. Functional studies on the role of microtubules provide the proof of principle that microtubules have different functions in different domains along the DV axis of the embryo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555167">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555167" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555167">
        <p class="paperTitle">A distinct class of hematopoietic stem cells develop from the human yellow bone marrow</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555167" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555167" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nguyen, T. T.; Loureiro, Z. Y.; Desai, A.; DeSouza, T.; Joyce, S.; Khair, L.; Samant, A.; Cirka, H.; Solivan-Rivera, J.; Ziegler, R.; Brehm, M.; Messina, L. M.; Corvera, S.</p>
        <p class="info">Score: 9.4, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555167' target='https://doi.org/10.1101/2023.08.29.555167'> 10.1101/2023.08.29.555167</a></p>
        <p class="abstract">Aging and metabolic diseases are accompanied by systemic inflammation, but the mechanisms that induce this state are not known. We developed a human bone-marrow organoid system to explore mechanisms underlying metabolic-disease associated systemic inflammation. We find that a distinct type of hematopoietic stem cell (HSC) develops in the adipose-rich, yellow bone marrow, which is known to gradually replace the hematopoietic red marrow as we age and during metabolic disease. Unlike HSCs derived from the red bone marrow, HSCs derived from the yellow bone marrow have higher proliferation rates, increase myeloid differentiation, skew towards pro-inflammatory M1 macrophage differentiation, and express a distinct transcriptomic profile associated with responsiveness to wounding. Yellow marrow-derived HSCs express higher levels of the leptin receptor, which we find to be further increased in patients with type 2 diabetes. Our work demonstrates that the human long bone yellow marrow is a niche for a distinct class of HSCs which could underlie hematopoietic dysfunction during aging and metabolic disease processes suggesting a shared inflammaging mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.13.523922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.13.523922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.13.523922">
        <p class="paperTitle">Automated staging of zebrafish embryos with deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.13.523922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.13.523922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, R. A.; Renshaw, M. J.; Devenport, D.; Barry, D. J.</p>
        <p class="info">Score: 12.2, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.13.523922' target='https://doi.org/10.1101/2023.01.13.523922'> 10.1101/2023.01.13.523922</a></p>
        <p class="abstract">The zebrafish (Danio rerio), is an important biomedical model organism used in many disciplines. The phenomenon of developmental delay in zebrafish embryos has been widely reported as part of a mutant or treatment-induced phenotype. However, the detection and quantification of these delays is often achieved through manual observation with reference to staging guides, which is both time-consuming and subjective. We recently reported a machine learning-based classifier, capable of quantifying the developmental delay between two populations of zebrafish embryos. Here, we build on that work by introducing a deep learning-based model (KimmelNet) that has been trained to predict the age (hours post fertilisation) of populations of zebrafish embryos. We show that when KimmelNet is tested on 2D brightfield images of zebrafish embryos, the predictions generated agree closely with those expected from established approaches to staging. Random sampling of the test data demonstrate that KimmelNet can be used to detect developmental delay between two populations with high confidence based on as few as 100 images of each population. Finally, we show that KimmelNet generalises to previously unseen data, with limited transfer learning improving this performance significantly. With the ability to analyse tens of thousands of standard brightfield microscopy images on a timescale of minutes, we envisage that KimmelNet will be a valuable resource for the developmental biology community. Furthermore, the approach we have used could easily be adapted to generate models for other organisms.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
