<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction
Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction
Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction
Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction\nAuthors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.",
  "keywords": [
    
  ],
  "articleBody": " Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction\nAuthors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.\nScore: 51.4, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552148\nYoung blood plasma is known to confer beneficial effects on various organs in mice and rats. However, it was not known whether plasma from young pigs rejuvenates old rat tissues at the epigenetic level; whether it alters the epigenetic clock, which is a highly accurate molecular biomarker of aging. To address this question, we developed and validated six different epigenetic clocks for rat tissues that are based on DNA methylation values derived from n=613 tissue samples. As indicated by their respective names, the rat pan-tissue clock can be applied to DNA methylation profiles from all rat tissues, while the rat brain-, liver-, and blood clocks apply to the corresponding tissue types. We also developed two epigenetic clocks that apply to both human and rat tissues by adding n=1366 human tissue samples to the training data. We employed these six rat clocks to investigate the rejuvenation effects of a porcine plasma fraction treatment in different rat tissues. The treatment more than halved the epigenetic ages of blood, heart, and liver tissue. A less pronounced, but statistically significant, rejuvenation effect could be observed in the hypothalamus. The treatment was accompanied by progressive improvement in the function of these organs as ascertained through numerous biochemical/physiological biomarkers and behavioral responses to assess cognitive functions. An immunoglobulin G (IgG) N-glycosylation pattern shift from pro-to anti-inflammatory also indicated reversal of glycan aging. Overall, this study demonstrates that a young porcine plasma-derived treatment markedly reverses aging in rats according to epigenetic clocks, IgG glycans, and other biomarkers of aging.\nNucleotide sharing through gap junctions buffers replication stress\nAuthors: Boumard, B.; Le Meur, G.; Maalouf, T.; El-Hajj, M.; Bauer, R.; Bardin, A. J.\nScore: 16.8, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553366\nCell proliferation underlying tissue growth and homeostasis, requires high levels of metabolites such as deoxynucleotides (dNTPs). The dNTP pool is known to be tightly cell-autonomously regulated through the de novo synthesis and salvage pathways. Here, we reveal that nucleotides can be provided to cells non-autonomously by surrounding cells of the tissue. Drosophila adult intestinal stem cells are highly sensitive to nucleotide depletion whereas wing progenitor cells are not. While adult intestinal stem cells lack gap junctions and their associated capacity to buffer dNTP pools, wing progenitor cells share nucleotides through connections to surrounding cells via gap junctions, allowing buffering of replication stress induced by nucleotide pool depletion. These findings have broad implications, suggesting that gap junction connectivity may define how sensitive cells are to changes in intracellular dNTP levels and replication stress.\nDynamics of chromatin accessibility during human first-trimester neurodevelopment\nAuthors: Mannens, C. C. A.; Hu, L.; Lonnerberg, P.; Schipper, M.; Reagor, C.; Li, X.; He, X.; Barker, R. A.; Sundstrom, E.; Posthuma, D.; Linnarsson, S.\nScore: 10.2, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553878\nThe human brain is capable of highly complex functions that develops through a tightly organized cascade of patterning events, expressed transcription factors and changes in chromatin accessibility. While extensive datasets exist describing gene expression across the developing brain with single-cell resolution, similar atlases of chromatin accessibility have been primarily focused on the forebrain. Here, we focus on the chromatin landscape and paired gene expression across the developing human brain to provide a comprehensive single cell atlas during the first trimester (6 - 13 post-conceptional weeks). We identified 135 clusters across half a million nuclei and using the multiomic measurements linked candidate cis-regulatory elements (cCREs) to gene expression. We found an increase in the number of accessible regions driven both by age and neuronal differentiation. Using a convolutional neural network we identified putative functional TF-binding sites in enhancers characterizing neuronal subtypes and we applied this model to cCREs upstream of ESRRB to elucidate its activation mechanism. Finally, by linking disease-associated SNPs to cCREs we validated putative pathogenic mechanisms in several diseases and identified midbrain-derived GABAergic neurons as being the most vulnerable to major depressive disorder related mutations. Together, our findings provide a higher degree of detail to some key gene regulatory mechanisms underlying the emergence of cell types during the first trimester. We anticipate this resource to be a valuable reference for future studies related to human neurodevelopment, such as identifying cell type specific enhancers that can be used for highly specific targeting in in vitro models.\nGlobal decoupling of cell differentiation from cell division in early embryo development\nAuthors: Kukreja, K.; Patel, N.; Megason, S. G.; Klein, A. M.\nScore: 51.9, Published: 2023-07-30 DOI: 10.1101/2023.07.29.551123\nAs tissues develop, cells divide and differentiate concurrently. Conflicting evidence shows that cell division is either dispensable or required for formation of cell types. To determine the role of cell division in differentiation, we arrested the cell cycle in zebrafish embryos using two independent approaches and profiled them at single-cell resolution. We show that cell division is dispensable for differentiation of all embryonic tissues during initial cell type differentiation from early gastrulation to the end of segmentation. In the absence of cell division, differentiation slows down in some cell types, and cells exhibit global stress responses. While differentiation is robust to blocking cell division, the proportions of cells across cell states are not. This work simplifies our understanding of the role of cell division in development and showcases the utility of combining embryo-wide perturbations with single-cell RNA sequencing to uncover the role of common biological processes across multiple tissues.\nHow a reaction-diffusion signal can control spinal cord regeneration in axolotls: A modelling study\nAuthors: Caliaro, V.; Peurichard, D.; Chara, O.\nScore: 5.5, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554065\nAxolotls are uniquely able to completely regenerate the spinal cord after amputation. The underlying governing mechanisms of this regenerative response have not yet been fully elucidated. We previously found that spinal cord regeneration is mainly driven by cell cycle acceleration of ependymal cells, recruited by a hypothetical signal propagating from the injury. However, the nature of the signal and its propagation remain unknown. In this theoretical study, we investigated whether the regeneration-inducing signal can follow a reaction-diffusion process. We developed a computational model, validated it with experimental data and showed that the signal dynamics can be understood in terms of reaction-diffusion mechanism. By developing a theory of the regenerating outgrowth in the limit of fast reaction-diffusion, we demonstrate that control of regenerative response solely relies on cell-to-signal sensitivity and the signal reaction-diffusion characteristic length. This study lays foundations for further identification of the signal controlling regeneration of the spinal cord.\nLineage-based scaling of germline intercellular bridges during oogenesis\nAuthors: Shaikh, U.; Sherlock, K.; Wilson, J.; Gilliland, W.; Lewellyn, L.\nScore: 3.9, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553876\nThe size of subcellular structures must be tightly controlled to maintain normal cell function; this is especially important when cells are part of developing tissues or organs. Despite its importance, few studies have determined how the size of organelles or other structures is maintained during tissue growth, when cells are growing, dividing, and rearranging. The developing egg chamber is a powerful model in which to study the relative growth rates of subcellular structures. The egg chamber contains a cluster of sixteen germ cells, which are connected through intercellular bridges called ring canals. Ring canals are formed following incomplete cytokinesis after each of four germ cell divisions. As the egg chamber grows, the nurse cells and the ring canals that connect them increase in size. Here, we demonstrate that ring canal size scaling is related to their lineage; the largest, first born ring canals grow at a relatively slower rate than ring canals derived from subsequent mitotic divisions. This lineage-based scaling relationship is maintained even if directed transport is reduced, ring canal size is altered, or if the germ cells go through an additional mitotic division. Further, we propose that changes in ring canal scaling could provide a mechanism to alter egg size.\nThe longitudinal single-cell panorama of cynomolgus monkey ovary throughout lifespan revealed a conserved ovarian clock between primates\nAuthors: Yan, L.; Long, X.; Zhao, Y.; Zhao, F.; Tu, W.; Yang, Q.; Qian, J.; Zhai, J.; Wang, M.; Hu, Y.; He, B.; Su, Y.; Jiang, X.; Gao, F.; Wang, H.; Guo, F.\nScore: 3.2, Published: 2023-08-16 DOI: 10.1101/2023.08.14.553309\nOvarian function is critical for female fertility and impacts reproductive longevity. It is of great importance to accurately predict the aging process within the ovary for fertility assessment and disease diagnosis. However, cell metrics for evaluating the ovarian aging rate are still in urgent need, and molecular insights into ovarian development and dysfunction during the primate life course are also limited. Here, we reported large-scale ovarian cell atlas of consecutive development of cynomolgus monkeys across 22 years with 20 time points, covering the foetal, newborn, prepubertal, pubertal, adult, perimenopausal and menopausal stages. We characterized and validated distinct molecular signatures of each cluster of cells within primate ovaries, and uncovered a previously undocumented RHOXF1-positive oocyte type during primordial follicle assembly in primates. Furthermore, the constitution and developmental trajectories of primate germ cells, granulosa cells and stromal / theca cells were also elucidated, and their precursors were identified. More importantly, dynamics of cellular compositions were unravelled through the ovarian development, featured by granulosa, epithelial, stromal, and immune cells that showed strong temporal heterogeneity spanning lifetime, whilst referred to the key function during the corresponding stages. Based on the correlations of each cell type with age and stage-specific molecular dynamics, we further constructed a transcriptomic ovarian clock which could perceive an effective biological age prediction of the ovary and further applied to humans. The findings reveal granulosa, epithelial, and stromal cells as the highest performance predictors of ovarian biological age, while highlighting the crucial role of AGE-RAGE and Relaxin signaling pathways in regulating ovarian aging. Our work not only provide valuable resource for obtaining insights into the development, aging and dysfunction of key organs, but also establish a transcriptomic clock to predict biological ovarian aging thus to be potential clinical implementation in future.\nMorphodynamics of human early brain organoid development\nAuthors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.\nScore: 3.0, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827\nBrain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrix-induced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of non-telencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.\nTrabeculations of the porcine and human cardiac ventricles are different in number but similar in total volume.\nAuthors: Jensen, B.; Salvatori, D.; Schouten, J.; Meijborg, V. M.; Lauridsen, H.; Agger, P.\nScore: 2.5, Published: 2023-08-19 DOI: 10.1101/2023.08.17.553743\nAn intricate meshwork of trabeculations lines the luminal side of cardiac ventricles. Compaction, a developmental process, is thought to reduce trabeculations by adding them to the neighboring compact wall which is then enlarged. When pig, a plausible cardiac donor for xenotransplantation, is compared to human, the ventricular walls appear to have fewer trabeculations. We hypothesized the trabecular volume is proportionally smaller in pig than in human. Macroscopically, we observed in sixteen pig hearts that the ventricular walls harbor few but large trabeculations. Close inspection revealed a high number of tiny trabeculations, a few hundred, within the recesses of the large trabeculations. While tiny, these were still larger than embryonic trabeculations and even when considering their number, the total tally of trabeculations in pig was much fewer than in human. Volumetrics based on high-resolution MRI of additional six pig hearts compared to six human hearts, revealed the left ventricles were not significantly differently trabeculated (21.5 versus 22.8%, respectively), and the porcine right ventricles were only slightly less trabeculated (42.1 versus 49.3%, respectively). We then analyzed volumetrically ten pig embryonic hearts from gestational day 14 to 35. The trabecular and compact layer always grew, as did the intertrabecular recesses, in contrast to what compaction predicts. The proportions of the trabecular and compact layers changed substantially, nonetheless, due to differences in their growth rate rather than compaction. In conclusion, processes that affect the trabecular morphology do not necessarily affect the proportion of trabecular-to-compact myocardium and they are then distinct from compaction.\nPiezo regulates epithelial topology and promotes precision in organ size control\nAuthors: Kumar, N.; Mim, M. S.; Levis, M.; Unger, M.; Miranda, G.; Robinett, T.; Zartman, J. J.\nScore: 2.2, Published: 2023-08-18 DOI: 10.1101/2023.08.16.553584\nMechanosensitive Piezo channels regulate cell division through calcium-mediated activation of ERK signaling or activate Rho signaling to mediate cell extrusion and cell death. However, systems-level functions of Piezo in regulating organogenesis remain poorly understood. Here, we demonstrate that Piezo controls epithelial cell topology to ensure precise organ growth through the integration of live imaging experiments with pharmacological and genetic perturbations and computational modeling. Notably, knockout or knockdown of Piezo led to bilateral asymmetry in wing phenotypes. While pharmacological activation of Piezo stimulated an increase in the frequency of spikes in cytosolic Ca2+, we discovered that Piezo overexpression counterintuitively reduces Ca2+ signaling dynamics. Knockdown of Piezo inhibited proliferation and decreased apoptosis, resulting in an overall increase in epithelial overcrowding. In contrast, either genetic overexpression or pharmacological activation of Piezo increased cell proliferation and cell removal through basal extrusion. Surprisingly, Piezo overexpression increased the hexagonality of cellular topology. To test whether Piezo regulates cell topology, we formulated computational simulations to investigate how expression levels of Piezo protein regulate cell proliferation and apoptosis through modulation of the cut-off tension required for Piezo channel activation. Quantitative analysis validated computational simulation predictions of how perturbations to Piezo impacted epithelial topology. Overall, our findings demonstrate that Piezo promotes robustness in regulating epithelial topology and is necessary for precise organ size control.\n",
  "wordCount" : "2531",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552148">
        <p class="paperTitle">Reversal of Biological Age in Multiple Rat Organs by Young Porcine Plasma Fraction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horvath, S.; Singh, K.; Raj, K.; Khairnar, S.; Sanghavi, A.; Shrivastava, A.; Zoller, J. A.; Li, C. Z.; Herenu, C. B.; Canatelli-Mallat, M.; Lehmann, M.; Habazin, S.; Vuckovic, F.; Solberg-Woods, L.; Garcia Martinez, A.; Wang, T.; Chiavellini, P.; Levine, A. J.; Chen, H.; Brooke, R. T.; Gordevicius, J.; Lauc, G.; Goya, R. G.; Katcher, H. L.</p>
        <p class="info">Score: 51.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552148' target='https://doi.org/10.1101/2023.08.06.552148'> 10.1101/2023.08.06.552148</a></p>
        <p class="abstract">Young blood plasma is known to confer beneficial effects on various organs in mice and rats. However, it was not known whether plasma from young pigs rejuvenates old rat tissues at the epigenetic level; whether it alters the epigenetic clock, which is a highly accurate molecular biomarker of aging. To address this question, we developed and validated six different epigenetic clocks for rat tissues that are based on DNA methylation values derived from n=613 tissue samples. As indicated by their respective names, the rat pan-tissue clock can be applied to DNA methylation profiles from all rat tissues, while the rat brain-, liver-, and blood clocks apply to the corresponding tissue types. We also developed two epigenetic clocks that apply to both human and rat tissues by adding n=1366 human tissue samples to the training data. We employed these six rat clocks to investigate the rejuvenation effects of a porcine plasma fraction treatment in different rat tissues. The treatment more than halved the epigenetic ages of blood, heart, and liver tissue. A less pronounced, but statistically significant, rejuvenation effect could be observed in the hypothalamus. The treatment was accompanied by progressive improvement in the function of these organs as ascertained through numerous biochemical/physiological biomarkers and behavioral responses to assess cognitive functions. An immunoglobulin G (IgG) N-glycosylation pattern shift from pro-to anti-inflammatory also indicated reversal of glycan aging. Overall, this study demonstrates that a young porcine plasma-derived treatment markedly reverses aging in rats according to epigenetic clocks, IgG glycans, and other biomarkers of aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553366">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553366" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553366">
        <p class="paperTitle">Nucleotide sharing through gap junctions buffers replication stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553366" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553366" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boumard, B.; Le Meur, G.; Maalouf, T.; El-Hajj, M.; Bauer, R.; Bardin, A. J.</p>
        <p class="info">Score: 16.8, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553366' target='https://doi.org/10.1101/2023.08.15.553366'> 10.1101/2023.08.15.553366</a></p>
        <p class="abstract">Cell proliferation underlying tissue growth and homeostasis, requires high levels of metabolites such as deoxynucleotides (dNTPs). The dNTP pool is known to be tightly cell-autonomously regulated through the de novo synthesis and salvage pathways. Here, we reveal that nucleotides can be provided to cells non-autonomously by surrounding cells of the tissue. Drosophila adult intestinal stem cells are highly sensitive to nucleotide depletion whereas wing progenitor cells are not. While adult intestinal stem cells lack gap junctions and their associated capacity to buffer dNTP pools, wing progenitor cells share nucleotides through connections to surrounding cells via gap junctions, allowing buffering of replication stress induced by nucleotide pool depletion. These findings have broad implications, suggesting that gap junction connectivity may define how sensitive cells are to changes in intracellular dNTP levels and replication stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553878">
        <p class="paperTitle">Dynamics of chromatin accessibility during human first-trimester neurodevelopment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mannens, C. C. A.; Hu, L.; Lonnerberg, P.; Schipper, M.; Reagor, C.; Li, X.; He, X.; Barker, R. A.; Sundstrom, E.; Posthuma, D.; Linnarsson, S.</p>
        <p class="info">Score: 10.2, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553878' target='https://doi.org/10.1101/2023.08.18.553878'> 10.1101/2023.08.18.553878</a></p>
        <p class="abstract">The human brain is capable of highly complex functions that develops through a tightly organized cascade of patterning events, expressed transcription factors and changes in chromatin accessibility. While extensive datasets exist describing gene expression across the developing brain with single-cell resolution, similar atlases of chromatin accessibility have been primarily focused on the forebrain. Here, we focus on the chromatin landscape and paired gene expression across the developing human brain to provide a comprehensive single cell atlas during the first trimester (6 - 13 post-conceptional weeks). We identified 135 clusters across half a million nuclei and using the multiomic measurements linked candidate cis-regulatory elements (cCREs) to gene expression. We found an increase in the number of accessible regions driven both by age and neuronal differentiation. Using a convolutional neural network we identified putative functional TF-binding sites in enhancers characterizing neuronal subtypes and we applied this model to cCREs upstream of ESRRB to elucidate its activation mechanism. Finally, by linking disease-associated SNPs to cCREs we validated putative pathogenic mechanisms in several diseases and identified midbrain-derived GABAergic neurons as being the most vulnerable to major depressive disorder related mutations. Together, our findings provide a higher degree of detail to some key gene regulatory mechanisms underlying the emergence of cell types during the first trimester. We anticipate this resource to be a valuable reference for future studies related to human neurodevelopment, such as identifying cell type specific enhancers that can be used for highly specific targeting in in vitro models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551123">
        <p class="paperTitle">Global decoupling of cell differentiation from cell division in early embryo development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kukreja, K.; Patel, N.; Megason, S. G.; Klein, A. M.</p>
        <p class="info">Score: 51.9, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551123' target='https://doi.org/10.1101/2023.07.29.551123'> 10.1101/2023.07.29.551123</a></p>
        <p class="abstract">As tissues develop, cells divide and differentiate concurrently. Conflicting evidence shows that cell division is either dispensable or required for formation of cell types. To determine the role of cell division in differentiation, we arrested the cell cycle in zebrafish embryos using two independent approaches and profiled them at single-cell resolution. We show that cell division is dispensable for differentiation of all embryonic tissues during initial cell type differentiation from early gastrulation to the end of segmentation. In the absence of cell division, differentiation slows down in some cell types, and cells exhibit global stress responses. While differentiation is robust to blocking cell division, the proportions of cells across cell states are not. This work simplifies our understanding of the role of cell division in development and showcases the utility of combining embryo-wide perturbations with single-cell RNA sequencing to uncover the role of common biological processes across multiple tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554065" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554065">
        <p class="paperTitle">How a reaction-diffusion signal can control spinal cord regeneration in axolotls: A modelling study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caliaro, V.; Peurichard, D.; Chara, O.</p>
        <p class="info">Score: 5.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554065' target='https://doi.org/10.1101/2023.08.21.554065'> 10.1101/2023.08.21.554065</a></p>
        <p class="abstract">Axolotls are uniquely able to completely regenerate the spinal cord after amputation. The underlying governing mechanisms of this regenerative response have not yet been fully elucidated. We previously found that spinal cord regeneration is mainly driven by cell cycle acceleration of ependymal cells, recruited by a hypothetical signal propagating from the injury. However, the nature of the signal and its propagation remain unknown. In this theoretical study, we investigated whether the regeneration-inducing signal can follow a reaction-diffusion process. We developed a computational model, validated it with experimental data and showed that the signal dynamics can be understood in terms of reaction-diffusion mechanism. By developing a theory of the regenerating outgrowth in the limit of fast reaction-diffusion, we demonstrate that control of regenerative response solely relies on cell-to-signal sensitivity and the signal reaction-diffusion characteristic length. This study lays foundations for further identification of the signal controlling regeneration of the spinal cord.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553876">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553876" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553876">
        <p class="paperTitle">Lineage-based scaling of germline intercellular bridges during oogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553876" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553876" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shaikh, U.; Sherlock, K.; Wilson, J.; Gilliland, W.; Lewellyn, L.</p>
        <p class="info">Score: 3.9, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553876' target='https://doi.org/10.1101/2023.08.18.553876'> 10.1101/2023.08.18.553876</a></p>
        <p class="abstract">The size of subcellular structures must be tightly controlled to maintain normal cell function; this is especially important when cells are part of developing tissues or organs. Despite its importance, few studies have determined how the size of organelles or other structures is maintained during tissue growth, when cells are growing, dividing, and rearranging. The developing egg chamber is a powerful model in which to study the relative growth rates of subcellular structures. The egg chamber contains a cluster of sixteen germ cells, which are connected through intercellular bridges called ring canals. Ring canals are formed following incomplete cytokinesis after each of four germ cell divisions. As the egg chamber grows, the nurse cells and the ring canals that connect them increase in size. Here, we demonstrate that ring canal size scaling is related to their lineage; the largest, first born ring canals grow at a relatively slower rate than ring canals derived from subsequent mitotic divisions. This lineage-based scaling relationship is maintained even if directed transport is reduced, ring canal size is altered, or if the germ cells go through an additional mitotic division. Further, we propose that changes in ring canal scaling could provide a mechanism to alter egg size.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553309" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553309">
        <p class="paperTitle">The longitudinal single-cell panorama of cynomolgus monkey ovary throughout lifespan revealed a conserved ovarian clock between primates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yan, L.; Long, X.; Zhao, Y.; Zhao, F.; Tu, W.; Yang, Q.; Qian, J.; Zhai, J.; Wang, M.; Hu, Y.; He, B.; Su, Y.; Jiang, X.; Gao, F.; Wang, H.; Guo, F.</p>
        <p class="info">Score: 3.2, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553309' target='https://doi.org/10.1101/2023.08.14.553309'> 10.1101/2023.08.14.553309</a></p>
        <p class="abstract">Ovarian function is critical for female fertility and impacts reproductive longevity. It is of great importance to accurately predict the aging process within the ovary for fertility assessment and disease diagnosis. However, cell metrics for evaluating the ovarian aging rate are still in urgent need, and molecular insights into ovarian development and dysfunction during the primate life course are also limited. Here, we reported large-scale ovarian cell atlas of consecutive development of cynomolgus monkeys across 22 years with 20 time points, covering the foetal, newborn, prepubertal, pubertal, adult, perimenopausal and menopausal stages. We characterized and validated distinct molecular signatures of each cluster of cells within primate ovaries, and uncovered a previously undocumented RHOXF1-positive oocyte type during primordial follicle assembly in primates. Furthermore, the constitution and developmental trajectories of primate germ cells, granulosa cells and stromal / theca cells were also elucidated, and their precursors were identified. More importantly, dynamics of cellular compositions were unravelled through the ovarian development, featured by granulosa, epithelial, stromal, and immune cells that showed strong temporal heterogeneity spanning lifetime, whilst referred to the key function during the corresponding stages. Based on the correlations of each cell type with age and stage-specific molecular dynamics, we further constructed a transcriptomic ovarian clock which could perceive an effective biological age prediction of the ovary and further applied to humans. The findings reveal granulosa, epithelial, and stromal cells as the highest performance predictors of ovarian biological age, while highlighting the crucial role of AGE-RAGE and Relaxin signaling pathways in regulating ovarian aging. Our work not only provide valuable resource for obtaining insights into the development, aging and dysfunction of key organs, but also establish a transcriptomic clock to predict biological ovarian aging thus to be potential clinical implementation in future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553827">
        <p class="paperTitle">Morphodynamics of human early brain organoid development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.</p>
        <p class="info">Score: 3.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553827' target='https://doi.org/10.1101/2023.08.21.553827'> 10.1101/2023.08.21.553827</a></p>
        <p class="abstract">Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrix-induced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of non-telencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553743">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553743" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553743">
        <p class="paperTitle">Trabeculations of the porcine and human cardiac ventricles are different in number but similar in total volume.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553743" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553743" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jensen, B.; Salvatori, D.; Schouten, J.; Meijborg, V. M.; Lauridsen, H.; Agger, P.</p>
        <p class="info">Score: 2.5, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553743' target='https://doi.org/10.1101/2023.08.17.553743'> 10.1101/2023.08.17.553743</a></p>
        <p class="abstract">An intricate meshwork of trabeculations lines the luminal side of cardiac ventricles. Compaction, a developmental process, is thought to reduce trabeculations by adding them to the neighboring compact wall which is then enlarged. When pig, a plausible cardiac donor for xenotransplantation, is compared to human, the ventricular walls appear to have fewer trabeculations. We hypothesized the trabecular volume is proportionally smaller in pig than in human. Macroscopically, we observed in sixteen pig hearts that the ventricular walls harbor few but large trabeculations. Close inspection revealed a high number of tiny trabeculations, a few hundred, within the recesses of the large trabeculations. While tiny, these were still larger than embryonic trabeculations and even when considering their number, the total tally of trabeculations in pig was much fewer than in human. Volumetrics based on high-resolution MRI of additional six pig hearts compared to six human hearts, revealed the left ventricles were not significantly differently trabeculated (21.5 versus 22.8%, respectively), and the porcine right ventricles were only slightly less trabeculated (42.1 versus 49.3%, respectively). We then analyzed volumetrically ten pig embryonic hearts from gestational day 14 to 35. The trabecular and compact layer always grew, as did the intertrabecular recesses, in contrast to what compaction predicts. The proportions of the trabecular and compact layers changed substantially, nonetheless, due to differences in their growth rate rather than compaction. In conclusion, processes that affect the trabecular morphology do not necessarily affect the proportion of trabecular-to-compact myocardium and they are then distinct from compaction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553584">
        <p class="paperTitle">Piezo regulates epithelial topology and promotes precision in organ size control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, N.; Mim, M. S.; Levis, M.; Unger, M.; Miranda, G.; Robinett, T.; Zartman, J. J.</p>
        <p class="info">Score: 2.2, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553584' target='https://doi.org/10.1101/2023.08.16.553584'> 10.1101/2023.08.16.553584</a></p>
        <p class="abstract">Mechanosensitive Piezo channels regulate cell division through calcium-mediated activation of ERK signaling or activate Rho signaling to mediate cell extrusion and cell death. However, systems-level functions of Piezo in regulating organogenesis remain poorly understood. Here, we demonstrate that Piezo controls epithelial cell topology to ensure precise organ growth through the integration of live imaging experiments with pharmacological and genetic perturbations and computational modeling. Notably, knockout or knockdown of Piezo led to bilateral asymmetry in wing phenotypes. While pharmacological activation of Piezo stimulated an increase in the frequency of spikes in cytosolic Ca2&#43;, we discovered that Piezo overexpression counterintuitively reduces Ca2&#43; signaling dynamics. Knockdown of Piezo inhibited proliferation and decreased apoptosis, resulting in an overall increase in epithelial overcrowding. In contrast, either genetic overexpression or pharmacological activation of Piezo increased cell proliferation and cell removal through basal extrusion. Surprisingly, Piezo overexpression increased the hexagonality of cellular topology. To test whether Piezo regulates cell topology, we formulated computational simulations to investigate how expression levels of Piezo protein regulate cell proliferation and apoptosis through modulation of the cut-off tension required for Piezo channel activation. Quantitative analysis validated computational simulation predictions of how perturbations to Piezo impacted epithelial topology. Overall, our findings demonstrate that Piezo promotes robustness in regulating epithelial topology and is necessary for precise organ size control.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
