<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>developmental biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>developmental biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods
Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.
Score: 34.5, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381
Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/developmental-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="developmental biology" />
<meta property="og:description" content="PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods
Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.
Score: 34.5, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381
Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/developmental-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="developmental biology"/>
<meta name="twitter:description" content="PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods
Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.
Score: 34.5, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381
Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "developmental biology",
      "item": "https://trxiv.yorks0n.com/posts/developmental-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "developmental biology",
  "name": "developmental biology",
  "description": "PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods\nAuthors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.\nScore: 34.5, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381\nCell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP.",
  "keywords": [
    
  ],
  "articleBody": " PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods\nAuthors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.\nScore: 34.5, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556381\nCell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP. To achieve sensitive labelling, we amplified the fluorescent signal by fusing s36GFP to the ultra-bright protein mNeonGreen. Secretor cells transfer s36GFP-mNeonGreen to neighbours while retaining a nuclear mCherry, making it straightforward to unambiguously identify, isolate, and profile neighbours of secretor cells. Transfer of the s36GFP-mNeonGreen fusion occurs within minutes, enabling even relatively transient interactions to be captured or tracked over time with live-imaging or flow cytometry. To enhance flexibility of use, we incorporated HaloTag technology to facilitate colour-of-choice labelling. To further increase the range of applications, we engineered a customisable single-plasmid construct composed of interchangeable components with option to incorporate any additional transgene. This allows users to manipulate the properties of a cell, while at the same time applying a fluorescent label to the surrounding cells. PUFFFIN offers a simple, sensitive, customisable approach to profile non-cell-autonomous responses to natural or induced changes in cell identity or behaviour in mammalian cells\nA single-cell atlas of pig gastrulation as a resource for comparative embryology\nAuthors: Simpson, L.; Strange, A.; Klisch, D.; Kraunsoe, S.; Azami, T.; Goszczynski, D.; Le, T.; Planells, B.; Holmes, N.; Sang, F.; Henson, S.; Loose, M. W.; Nichols, J.; Alberio, R.\nScore: 25.8, Published: 2023-09-02 DOI: 10.1101/2023.08.31.555712\nEarly mammalian gastrulations cell-fate decisions are poorly understood due to difficulties obtaining non-rodent embryos. The bilaminar disc of pig embryos mirrors humans, making them a useful proxy for studying gastrulation. Here we present a single-cell transcriptomic atlas of pig gastrulation, revealing cell-fate emergence dynamics, as well as conserved and divergent gene programs governing early porcine, primate, and murine development. We highlight heterochronicity in extraembryonic cell-type development, despite the broad conservation of cell-type-specific transcriptional programs. We apply these findings in combination with functional investigations, to outline conserved spatial, molecular, and temporal events during definitive endoderm (DE). We find early FOXA2+/TBXT-embryonic disc cells directly from DE, contrasting later-emerging FOXA2/TBXT+ node/notochord progenitors. Unlike mesoderm, none of these progenitors undergo epithelial-to-mesenchymal transition. DE/Node fate hinges on balanced WNT and hypoblast-derived NODAL, which is extinguished upon DE differentiation. These findings emphasise the interplay between temporal and topological signalling in early fate decisions during gastrulation.\nVolumetric trans-scale imaging of massive quantity of heterogeneous cell populations in centimeter-wide tissue and embryo\nAuthors: Ichimura, T.; Kakizuka, T.; Sato, Y.; Itano, K.; Seiriki, K.; Hashimoto, H.; Itoga, H.; Onami, S.; Nagai, T.\nScore: 22.3, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553997\nWe established a volumetric trans-scale imaging system with an ultra-large field-of-view (FOV) that enables simultaneous observation of millions of cellular dynamics in centimeter-wide three-dimensional (3D) tissues and embryos. Using a custom-made giant lens system with a magnification of 2x and a numerical aperture (NA) of 0.25, and a CMOS camera with more than 100 megapixels, we built a trans-scale scope AMATERAS-2, and realized fluorescence imaging with a lateral spatial resolution of approximately 1.2 {micro}m across an FOV of approximately 1.5 x 1.0 cm2. The 3D resolving capability was realized through a combination of optical and computational sectioning techniques tailored for our low-power imaging system. We applied the imaging technique to 1.2 cm-wide section of mouse brain, and successfully observed various regions of the brain with sub-cellular resolution in a single FOV. We also performed time-lapse imaging of a 1-cm-wide vascular network during quail embryo development for over 24 hours, visualizing the movement of over 4.0 x 105 vascular endothelial cells and quantitatively analyzing their dynamics. Our results demonstrate the potential of this technique in accelerating production of comprehensive reference maps of all cells in organisms and tissues, which contributes to understanding developmental processes, brain functions, and pathogenesis of disease, as well as high-throughput quality check of tissues used for transplantation medicine.\nMorphogen gradients can convey position and time in growing tissues\nAuthors: Vetter, R.; Iber, D.\nScore: 27.0, Published: 2023-09-01 DOI: 10.1101/2023.08.28.555133\nDuring development, cells need to make fate decisions according to their position and the developmental timepoint. Morphogen gradients provide positional information, but how timing is controlled has remained elusive. In the mouse neural tube, morphogen signalling is transient and peaks shortly before the onset of differentiation. Here, we show that in morphogen gradients with constant decay length, cells experience transient, biphasic concentration profiles if the morphogen source expands in parallel with the uniformly growing tissue. This novel patterning paradigm offers a simple mechanism for the simultaneous control of position and time by morphogen gradients, and might apply in many patterning systems, as uniform growth is observed widely in development.\nKLF7 is a general inducer of human pluripotency\nAuthors: Arboit, M.; Zorzan, I.; Pellegrini, M.; Martini, P.; Carbognin, E.; Martello, G.\nScore: 11.3, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556189\nPluripotency is the capacity to give rise to all differentiated cells of the body and the germ line and is governed by a self-reinforcing network of transcription factors. The forced expression of only some of these factors enables the reprogramming of somatic cells to pluripotency. In murine cells, several kruppel-like factors (KLFs) have been identified as stabilisers and inducers of pluripotency. Human somatic cells are routinely reprogrammed by expression of KLF4 in combination with OCT4, SOX2 and cMYC (OSKM). An extensive transcriptome analysis revealed, however, that KLF4 is barely expressed in conventional human pluripotent stem cells (PSCs). Here we show that KLF7 is robustly expressed in conventional human PSCs and it allows transcription factor-mediated somatic reprogramming. Moreover, we find that KLF7 is highly expressed in naive PSCs and its forced expression in conventional hPSCs induces upregulation of naive markers and boosts efficiency of chemical resetting to naive PSCs, overall suggesting that KLF7 is a general human pluripotency factor and an inducer of pluripotency.\nMorphodynamics of human early brain organoid development\nAuthors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.\nScore: 86.3, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553827\nBrain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrixinduced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of nontelencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.\nGerm cells do not progress through spermatogenesis in the infertile zebrafish testis\nAuthors: Sposato, A. L.; Llewellyn, D. R.; Weber, J. M.; Hollins, H. L.; Schrock, M. N.; Farrell, J. A.; Gagnon, J. A.\nScore: 10.9, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556432\nVertebrate spermatogonial stem cells maintain sperm production over the lifetime of an animal but fertility declines with age. While morphological studies have greatly informed our understanding of typical spermatogenesis, the molecular and cellular mechanisms underlying spermatogenesis are not yet understood, particularly with respect to the onset of fertility. We used single-cell RNA sequencing to generate a developmental atlas of the zebrafish testis. Using 5 timepoints across the adult life of a zebrafish, we described cellular profiles in the testis during and after fertility. While all germ cell stages of spermatogenesis are detected in testes from fertile adult zebrafish, testes from older infertile males only contained spermatogonia and a reduced population of spermatocytes. These remaining germ cells are transcriptionally distinct from fertile spermatogonia. Immune cells including macrophages and lymphocytes drastically increase in abundance in infertile testes. Our developmental atlas reveals the cellular changes as the testis ages and defines a molecular roadmap for the regulation of male fertility.\nCombinatorial Wnt signaling landscape during brachiopod anteroposterior patterning\nAuthors: Vellutini, B. C.; Martin-Duran, J. M.; Borve, A.; Hejnol, A.\nScore: 9.5, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556047\nBackgroundWnt signaling pathways play crucial roles in animal development. They establish embryonic axes, specify cell fates, and regulate tissue morphogenesis from the early embryo to organo-genesis. It is becoming increasingly recognized that these distinct developmental outcomes depend upon dynamic interactions between multiple ligands, receptors, antagonists, and other pathway modulators, consolidating the view that a combinatorial \"code\" controls the output of Wnt signaling. However, due to the lack of comprehensive analyses of Wnt components in several animal groups, it remains unclear if specific combinations always give rise to specific outcomes, and if these combinatorial patterns are conserved throughout evolution. ResultsIn this work, we investigate the combinatorial expression of Wnt signaling components during the axial patterning of the brachiopod Terebratalia transversa. We find that T. transversa has a conserved repertoire of ligands, receptors, and antagonists. These genes are expressed throughout embryogenesis but undergo significant upregulation during axial elongation. At this stage, Frizzled domains occupy broad regions across the body while Wnt domains are narrower and distributed in partially overlapping patches; antagonists are mostly restricted to the anterior end. Based on their combinatorial expression, we identify a series of unique transcriptional subregions along the anteroposterior axis that coincide with the different morphological subdivisions of the brachiopod larval body. When comparing these data across the animal phylogeny, we find that the expression of Frizzled genes is relatively conserved, whereas the expression of Wnt genes is more variable. ConclusionsOur results suggest that the differential activation of Wnt signaling pathways may play a role in regionalizing the anteroposterior axis of brachiopod larvae. More generally, our analyses suggest that changes in the receptor context of Wnt ligands may act as a mechanism for the evolution and diversification of the body axis of bilaterians.\nModular control of time and space during vertebrate axis segmentation\nAuthors: Seleit, A.; Brettell, I.; Fitzgerald, T. W.; Vibe, C.; Loosli, F.; Wittbrodt, J.; Naruse, K.; Birney, E.; Aulehla, A.\nScore: 27.5, Published: 2023-08-31 DOI: 10.1101/2023.08.30.555457\nHow temporal and spatial control of developmental processes are linked remains a fundamental question. Do underlying mechanisms form a single functional unit or are these dissociable modules? We address this question by studying the periodic process of embryonic axis segmentation, using genetic crosses of inbred medaka fish strains representing two species, Oryzias sakaizumii and latipes. Our analysis revealed correlated interspecies differences with regard to the timing of segmentation, the size of segments and of the presomitic mesoderm (PSM), from which segments are periodically formed. We then did interspecies crosses and real-time imaging quantifications, which revealed extensive phenotypic variation in [~]600 F2 embryos. Importantly, while the F2 analysis showed correlated changes of PSM and segment size, these spatial measures were not correlated to the timing of segmentation. This shows that the control of time and space of axis segmentation can, in principle, be decoupled. In line with this finding, we identified, using developmental quantitative trait loci (devQTL) mapping, distinct chromosomal regions linked to either the control of segmentation timing or PSM size. We were able to validate the devQTL findings using a CRISPR/Cas9 loss-of-function approach on several candidate genes in vivo. Combined, this study reveals that a developmental constraint mechanism underlies spatial scaling of axis segmentation, while its spatial and temporal control are dissociable modules. Our findings emphasise the need to reveal the selective constraints linking these modules in the natural environment.\nPrimate-expressed EPIREGULIN promotes basal progenitor proliferation in the developing neocortex\nAuthors: Cubillos, P.; Ditzer, N.; Kolodziejczyk, A.; Schwenk, G.; Hoffmann, J.; Schütze, T. M.; Derihaci, R. P.; Birdir, C.; Köllner, J. E. M.; Petzold, A.; Sarov, M.; Martin, U.; Long, K. R.; Wimberger, P.; Albert, M.\nScore: 29.6, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554446\nNeocortex expansion during evolution is linked to higher numbers of neurons thought to result from increased proliferative capacity and neurogenic potential of basal progenitor cells (BPs) during development. Here we show that EREG, encoding the growth factor EPIREGULIN, is expressed in the human developing neocortex and in gorilla organoids, but not in the mouse neocortex. Addition of EPIREGULIN to the mouse neocortex increases proliferation of BPs via EGFR-mediated signaling, whereas ablation of EREG in human cortical organoids reduces BP proliferation. Addition of EPIREGULIN to cortical organoids promotes a further increase in proliferation of gorilla but not human BPs. Finally, we identify putative cis-regulatory elements that may contribute to inter-species differences in EREG expression. Overall, our results suggest that species-specific expression of EPIREGULIN may contribute to increased neocortex size in primates by providing a pro-proliferative signal to BPs in the subventricular zone progenitor niche.\n",
  "wordCount" : "2258",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/developmental-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      developmental biology
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556381">
        <p class="paperTitle">PUFFFIN: A novel, ultra-bright, customisable, single plasmid system for labelling cell neighbourhoods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lebek, T.; Malaguti, M.; Elfick, A.; Lowell, S.</p>
        <p class="info">Score: 34.5, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556381' target='https://doi.org/10.1101/2023.09.06.556381'> 10.1101/2023.09.06.556381</a></p>
        <p class="abstract">Cell communication orchestrates development, homeostasis, and disease. Understanding these processes depends on effective methods to study how cells influence their neighbours. Here, we present Positive Ultra-bright Fluorescent Fusion For Identifying Neighbours (PUFFFIN), a novel cell neighbour-labelling system based upon secretion and uptake of a positively supercharged fluorescent protein s36GFP. To achieve sensitive labelling, we amplified the fluorescent signal by fusing s36GFP to the ultra-bright protein mNeonGreen. Secretor cells transfer s36GFP-mNeonGreen to neighbours while retaining a nuclear mCherry, making it straightforward to unambiguously identify, isolate, and profile neighbours of secretor cells. Transfer of the s36GFP-mNeonGreen fusion occurs within minutes, enabling even relatively transient interactions to be captured or tracked over time with live-imaging or flow cytometry. To enhance flexibility of use, we incorporated HaloTag technology to facilitate colour-of-choice labelling. To further increase the range of applications, we engineered a customisable single-plasmid construct composed of interchangeable components with option to incorporate any additional transgene. This allows users to manipulate the properties of a cell, while at the same time applying a fluorescent label to the surrounding cells. PUFFFIN offers a simple, sensitive, customisable approach to profile non-cell-autonomous responses to natural or induced changes in cell identity or behaviour in mammalian cells</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555712" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555712">
        <p class="paperTitle">A single-cell atlas of pig gastrulation as a resource for comparative embryology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simpson, L.; Strange, A.; Klisch, D.; Kraunsoe, S.; Azami, T.; Goszczynski, D.; Le, T.; Planells, B.; Holmes, N.; Sang, F.; Henson, S.; Loose, M. W.; Nichols, J.; Alberio, R.</p>
        <p class="info">Score: 25.8, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555712' target='https://doi.org/10.1101/2023.08.31.555712'> 10.1101/2023.08.31.555712</a></p>
        <p class="abstract">Early mammalian gastrulations cell-fate decisions are poorly understood due to difficulties obtaining non-rodent embryos. The bilaminar disc of pig embryos mirrors humans, making them a useful proxy for studying gastrulation. Here we present a single-cell transcriptomic atlas of pig gastrulation, revealing cell-fate emergence dynamics, as well as conserved and divergent gene programs governing early porcine, primate, and murine development. We highlight heterochronicity in extraembryonic cell-type development, despite the broad conservation of cell-type-specific transcriptional programs. We apply these findings in combination with functional investigations, to outline conserved spatial, molecular, and temporal events during definitive endoderm (DE). We find early FOXA2&#43;/TBXT-embryonic disc cells directly from DE, contrasting later-emerging FOXA2/TBXT&#43; node/notochord progenitors. Unlike mesoderm, none of these progenitors undergo epithelial-to-mesenchymal transition. DE/Node fate hinges on balanced WNT and hypoblast-derived NODAL, which is extinguished upon DE differentiation. These findings emphasise the interplay between temporal and topological signalling in early fate decisions during gastrulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553997">
        <p class="paperTitle">Volumetric trans-scale imaging of massive quantity of heterogeneous cell populations in centimeter-wide tissue and embryo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ichimura, T.; Kakizuka, T.; Sato, Y.; Itano, K.; Seiriki, K.; Hashimoto, H.; Itoga, H.; Onami, S.; Nagai, T.</p>
        <p class="info">Score: 22.3, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553997' target='https://doi.org/10.1101/2023.08.21.553997'> 10.1101/2023.08.21.553997</a></p>
        <p class="abstract">We established a volumetric trans-scale imaging system with an ultra-large field-of-view (FOV) that enables simultaneous observation of millions of cellular dynamics in centimeter-wide three-dimensional (3D) tissues and embryos. Using a custom-made giant lens system with a magnification of 2x and a numerical aperture (NA) of 0.25, and a CMOS camera with more than 100 megapixels, we built a trans-scale scope AMATERAS-2, and realized fluorescence imaging with a lateral spatial resolution of approximately 1.2 {micro}m across an FOV of approximately 1.5 x 1.0 cm2. The 3D resolving capability was realized through a combination of optical and computational sectioning techniques tailored for our low-power imaging system. We applied the imaging technique to 1.2 cm-wide section of mouse brain, and successfully observed various regions of the brain with sub-cellular resolution in a single FOV. We also performed time-lapse imaging of a 1-cm-wide vascular network during quail embryo development for over 24 hours, visualizing the movement of over 4.0 x 105 vascular endothelial cells and quantitatively analyzing their dynamics. Our results demonstrate the potential of this technique in accelerating production of comprehensive reference maps of all cells in organisms and tissues, which contributes to understanding developmental processes, brain functions, and pathogenesis of disease, as well as high-throughput quality check of tissues used for transplantation medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555133">
        <p class="paperTitle">Morphogen gradients can convey position and time in growing tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vetter, R.; Iber, D.</p>
        <p class="info">Score: 27.0, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555133' target='https://doi.org/10.1101/2023.08.28.555133'> 10.1101/2023.08.28.555133</a></p>
        <p class="abstract">During development, cells need to make fate decisions according to their position and the developmental timepoint. Morphogen gradients provide positional information, but how timing is controlled has remained elusive. In the mouse neural tube, morphogen signalling is transient and peaks shortly before the onset of differentiation. Here, we show that in morphogen gradients with constant decay length, cells experience transient, biphasic concentration profiles if the morphogen source expands in parallel with the uniformly growing tissue. This novel patterning paradigm offers a simple mechanism for the simultaneous control of position and time by morphogen gradients, and might apply in many patterning systems, as uniform growth is observed widely in development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556189">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556189" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556189">
        <p class="paperTitle">KLF7 is a general inducer of human pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556189" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556189" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arboit, M.; Zorzan, I.; Pellegrini, M.; Martini, P.; Carbognin, E.; Martello, G.</p>
        <p class="info">Score: 11.3, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556189' target='https://doi.org/10.1101/2023.09.06.556189'> 10.1101/2023.09.06.556189</a></p>
        <p class="abstract">Pluripotency is the capacity to give rise to all differentiated cells of the body and the germ line and is governed by a self-reinforcing network of transcription factors. The forced expression of only some of these factors enables the reprogramming of somatic cells to pluripotency. In murine cells, several kruppel-like factors (KLFs) have been identified as stabilisers and inducers of pluripotency. Human somatic cells are routinely reprogrammed by expression of KLF4 in combination with OCT4, SOX2 and cMYC (OSKM). An extensive transcriptome analysis revealed, however, that KLF4 is barely expressed in conventional human pluripotent stem cells (PSCs). Here we show that KLF7 is robustly expressed in conventional human PSCs and it allows transcription factor-mediated somatic reprogramming. Moreover, we find that KLF7 is highly expressed in naive PSCs and its forced expression in conventional hPSCs induces upregulation of naive markers and boosts efficiency of chemical resetting to naive PSCs, overall suggesting that KLF7 is a general human pluripotency factor and an inducer of pluripotency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553827">
        <p class="paperTitle">Morphodynamics of human early brain organoid development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jain, A.; Gut, G.; Sanchis Calleja, F.; Okamoto, R.; Streib, S.; He, Z.; Zenk, F.; Santel, M.; Seimiya, M.; Holtackers, R.; Jansen, S. M. J.; Camp, J. G.; Treutlein, B.</p>
        <p class="info">Score: 86.3, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553827' target='https://doi.org/10.1101/2023.08.21.553827'> 10.1101/2023.08.21.553827</a></p>
        <p class="abstract">Brain organoids enable the mechanistic study of human brain development, and provide opportunities to explore self-organization in unconstrained developmental systems. Here, we establish long-term, live light sheet microscopy on unguided brain organoids generated from fluorescently labeled human induced pluripotent stem cells, which enables tracking of tissue morphology, cell behaviors, and subcellular features over weeks of organoid development. We provide a novel dual-channel, multi-mosaic and multi-protein labeling strategy combined with a computational demultiplexing approach to enable simultaneous quantification of distinct subcellular features during organoid development. We track Actin, Tubulin, plasma membrane, nucleus, and nuclear envelope dynamics, and quantify cell morphometric and alignment changes during tissue state transitions including neuroepithelial induction, maturation, lumenization, and brain regionalization. Based on imaging and single-cell transcriptome modalities, we find that lumenal expansion and cell morphotype composition within the developing neuroepithelium are associated with modulation of gene expression programs involving extracellular matrix (ECM) pathway regulators and mechanosensing. We show that an extrinsically provided matrix enhances lumen expansion as well as telencephalon formation, and unguided organoids grown in the absence of an extrinsic matrix have altered morphologies with increased neural crest and caudalized tissue identity. Matrixinduced regional guidance and lumen morphogenesis are linked to the WNT and Hippo (YAP1) signaling pathways, including spatially restricted induction of the Wnt Ligand Secretion Mediator (WLS) that marks the earliest emergence of nontelencephalic brain regions. Altogether, our work provides a new inroad into studying human brain morphodynamics, and supports a view that matrix-linked mechanosensing dynamics play a central role during brain regionalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556432">
        <p class="paperTitle">Germ cells do not progress through spermatogenesis in the infertile zebrafish testis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sposato, A. L.; Llewellyn, D. R.; Weber, J. M.; Hollins, H. L.; Schrock, M. N.; Farrell, J. A.; Gagnon, J. A.</p>
        <p class="info">Score: 10.9, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556432' target='https://doi.org/10.1101/2023.09.05.556432'> 10.1101/2023.09.05.556432</a></p>
        <p class="abstract">Vertebrate spermatogonial stem cells maintain sperm production over the lifetime of an animal but fertility declines with age. While morphological studies have greatly informed our understanding of typical spermatogenesis, the molecular and cellular mechanisms underlying spermatogenesis are not yet understood, particularly with respect to the onset of fertility. We used single-cell RNA sequencing to generate a developmental atlas of the zebrafish testis. Using 5 timepoints across the adult life of a zebrafish, we described cellular profiles in the testis during and after fertility. While all germ cell stages of spermatogenesis are detected in testes from fertile adult zebrafish, testes from older infertile males only contained spermatogonia and a reduced population of spermatocytes. These remaining germ cells are transcriptionally distinct from fertile spermatogonia. Immune cells including macrophages and lymphocytes drastically increase in abundance in infertile testes. Our developmental atlas reveals the cellular changes as the testis ages and defines a molecular roadmap for the regulation of male fertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556047">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556047" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556047">
        <p class="paperTitle">Combinatorial Wnt signaling landscape during brachiopod anteroposterior patterning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556047" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556047" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vellutini, B. C.; Martin-Duran, J. M.; Borve, A.; Hejnol, A.</p>
        <p class="info">Score: 9.5, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556047' target='https://doi.org/10.1101/2023.09.03.556047'> 10.1101/2023.09.03.556047</a></p>
        <p class="abstract">BackgroundWnt signaling pathways play crucial roles in animal development. They establish embryonic axes, specify cell fates, and regulate tissue morphogenesis from the early embryo to organo-genesis. It is becoming increasingly recognized that these distinct developmental outcomes depend upon dynamic interactions between multiple ligands, receptors, antagonists, and other pathway modulators, consolidating the view that a combinatorial &#34;code&#34; controls the output of Wnt signaling. However, due to the lack of comprehensive analyses of Wnt components in several animal groups, it remains unclear if specific combinations always give rise to specific outcomes, and if these combinatorial patterns are conserved throughout evolution.

ResultsIn this work, we investigate the combinatorial expression of Wnt signaling components during the axial patterning of the brachiopod Terebratalia transversa. We find that T. transversa has a conserved repertoire of ligands, receptors, and antagonists. These genes are expressed throughout embryogenesis but undergo significant upregulation during axial elongation. At this stage, Frizzled domains occupy broad regions across the body while Wnt domains are narrower and distributed in partially overlapping patches; antagonists are mostly restricted to the anterior end. Based on their combinatorial expression, we identify a series of unique transcriptional subregions along the anteroposterior axis that coincide with the different morphological subdivisions of the brachiopod larval body. When comparing these data across the animal phylogeny, we find that the expression of Frizzled genes is relatively conserved, whereas the expression of Wnt genes is more variable.

ConclusionsOur results suggest that the differential activation of Wnt signaling pathways may play a role in regionalizing the anteroposterior axis of brachiopod larvae. More generally, our analyses suggest that changes in the receptor context of Wnt ligands may act as a mechanism for the evolution and diversification of the body axis of bilaterians.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555457">
        <p class="paperTitle">Modular control of time and space during vertebrate axis segmentation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seleit, A.; Brettell, I.; Fitzgerald, T. W.; Vibe, C.; Loosli, F.; Wittbrodt, J.; Naruse, K.; Birney, E.; Aulehla, A.</p>
        <p class="info">Score: 27.5, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555457' target='https://doi.org/10.1101/2023.08.30.555457'> 10.1101/2023.08.30.555457</a></p>
        <p class="abstract">How temporal and spatial control of developmental processes are linked remains a fundamental question. Do underlying mechanisms form a single functional unit or are these dissociable modules?

We address this question by studying the periodic process of embryonic axis segmentation, using genetic crosses of inbred medaka fish strains representing two species, Oryzias sakaizumii and latipes. Our analysis revealed correlated interspecies differences with regard to the timing of segmentation, the size of segments and of the presomitic mesoderm (PSM), from which segments are periodically formed. We then did interspecies crosses and real-time imaging quantifications, which revealed extensive phenotypic variation in [~]600 F2 embryos. Importantly, while the F2 analysis showed correlated changes of PSM and segment size, these spatial measures were not correlated to the timing of segmentation. This shows that the control of time and space of axis segmentation can, in principle, be decoupled. In line with this finding, we identified, using developmental quantitative trait loci (devQTL) mapping, distinct chromosomal regions linked to either the control of segmentation timing or PSM size. We were able to validate the devQTL findings using a CRISPR/Cas9 loss-of-function approach on several candidate genes in vivo.

Combined, this study reveals that a developmental constraint mechanism underlies spatial scaling of axis segmentation, while its spatial and temporal control are dissociable modules. Our findings emphasise the need to reveal the selective constraints linking these modules in the natural environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554446">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554446" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554446">
        <p class="paperTitle">Primate-expressed EPIREGULIN promotes basal progenitor proliferation in the developing neocortex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554446" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554446" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cubillos, P.; Ditzer, N.; Kolodziejczyk, A.; Schwenk, G.; Hoffmann, J.; Schütze, T. M.; Derihaci, R. P.; Birdir, C.; Köllner, J. E. M.; Petzold, A.; Sarov, M.; Martin, U.; Long, K. R.; Wimberger, P.; Albert, M.</p>
        <p class="info">Score: 29.6, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554446' target='https://doi.org/10.1101/2023.08.23.554446'> 10.1101/2023.08.23.554446</a></p>
        <p class="abstract">Neocortex expansion during evolution is linked to higher numbers of neurons thought to result from increased proliferative capacity and neurogenic potential of basal progenitor cells (BPs) during development. Here we show that EREG, encoding the growth factor EPIREGULIN, is expressed in the human developing neocortex and in gorilla organoids, but not in the mouse neocortex. Addition of EPIREGULIN to the mouse neocortex increases proliferation of BPs via EGFR-mediated signaling, whereas ablation of EREG in human cortical organoids reduces BP proliferation. Addition of EPIREGULIN to cortical organoids promotes a further increase in proliferation of gorilla but not human BPs. Finally, we identify putative cis-regulatory elements that may contribute to inter-species differences in EREG expression. Overall, our results suggest that species-specific expression of EPIREGULIN may contribute to increased neocortex size in primates by providing a pro-proliferative signal to BPs in the subventricular zone progenitor niche.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
