<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>hematology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>hematology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Calling for diversity: improving transfusion safety through high-throughput blood group microarray genotyping
Authors: Wittig, M.; Steiert, T. A.; ElAbd, H.; Degenhardt, F.; Valenti, L.; Prati, D.; Ronzoni, L.; Bujanda, L.; Banales, J. M.; Blay, N.; Invernizzi, P.; Buti, M.; Albillos, A.; Fernandez, J.; Sacchi, N.; Julia, A.; Latiano, A.; de Cid, R.; D&#39;Amato, M.; Asselta, R.; Laudes, M.; Lieb, W.; Juhl, D.; Gassner, C.; Franke, A.
Score: 3.8, Published: 2023-12-18 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/hematology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="hematology" />
<meta property="og:description" content="Calling for diversity: improving transfusion safety through high-throughput blood group microarray genotyping
Authors: Wittig, M.; Steiert, T. A.; ElAbd, H.; Degenhardt, F.; Valenti, L.; Prati, D.; Ronzoni, L.; Bujanda, L.; Banales, J. M.; Blay, N.; Invernizzi, P.; Buti, M.; Albillos, A.; Fernandez, J.; Sacchi, N.; Julia, A.; Latiano, A.; de Cid, R.; D&#39;Amato, M.; Asselta, R.; Laudes, M.; Lieb, W.; Juhl, D.; Gassner, C.; Franke, A.
Score: 3.8, Published: 2023-12-18 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/hematology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="hematology"/>
<meta name="twitter:description" content="Calling for diversity: improving transfusion safety through high-throughput blood group microarray genotyping
Authors: Wittig, M.; Steiert, T. A.; ElAbd, H.; Degenhardt, F.; Valenti, L.; Prati, D.; Ronzoni, L.; Bujanda, L.; Banales, J. M.; Blay, N.; Invernizzi, P.; Buti, M.; Albillos, A.; Fernandez, J.; Sacchi, N.; Julia, A.; Latiano, A.; de Cid, R.; D&#39;Amato, M.; Asselta, R.; Laudes, M.; Lieb, W.; Juhl, D.; Gassner, C.; Franke, A.
Score: 3.8, Published: 2023-12-18 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "hematology",
      "item": "https://trxiv.yorks0n.com/posts/hematology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "hematology",
  "name": "hematology",
  "description": "Calling for diversity: improving transfusion safety through high-throughput blood group microarray genotyping\nAuthors: Wittig, M.; Steiert, T. A.; ElAbd, H.; Degenhardt, F.; Valenti, L.; Prati, D.; Ronzoni, L.; Bujanda, L.; Banales, J. M.; Blay, N.; Invernizzi, P.; Buti, M.; Albillos, A.; Fernandez, J.; Sacchi, N.; Julia, A.; Latiano, A.; de Cid, R.; D\u0026#39;Amato, M.; Asselta, R.; Laudes, M.; Lieb, W.; Juhl, D.; Gassner, C.; Franke, A.\nScore: 3.8, Published: 2023-12-18 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Calling for diversity: improving transfusion safety through high-throughput blood group microarray genotyping\nAuthors: Wittig, M.; Steiert, T. A.; ElAbd, H.; Degenhardt, F.; Valenti, L.; Prati, D.; Ronzoni, L.; Bujanda, L.; Banales, J. M.; Blay, N.; Invernizzi, P.; Buti, M.; Albillos, A.; Fernandez, J.; Sacchi, N.; Julia, A.; Latiano, A.; de Cid, R.; D'Amato, M.; Asselta, R.; Laudes, M.; Lieb, W.; Juhl, D.; Gassner, C.; Franke, A.\nScore: 3.8, Published: 2023-12-18 DOI: 10.1101/2023.12.15.23299980\nBlood transfusions, conducted between donors compatible in their red blood cell (RBC) antigens, play a life-saving role in transfusion medicine. Genetic differences at blood group loci between ethnicities result in diversity and altered frequency of RBC antigens that need to be considered in blood transfusion. Consequently, comprehensive, and accurate blood group antigen typing is especially relevant for inter-ethnicity blood transfusions and for minorities underrepresented in the donor population. Blood group microarray genotyping is a cost-efficient and scalable method for comprehensive blood group typing. Previously, however, microarray typing has been challenging for the clinically important blood group systems Rh and MNS, as these feature highly paralogous genomic loci leading to mixed signals. We here present an approach for accurately typing blood group systems, including Rh and MNS variations, that we benchmarked in an ethnically diverse cohort. We tested its performance using gold-standard, diagnostic-grade MALDI-TOF data from 1,052-samples, including 334 HGDP-CEPH diversity panel samples and applied the approach to 4,999 samples of a COVID-19 genetics study. Overall, we obtained a 99.95% benchmarking concordance and 99.43% call rate. In summary, we provide a highly accurate and cost-efficient high-throughput genotyping method for comprehensive blood group analysis that is also suitable for ethnically diverse sample sets.\nSingle-cell analysis of bone marrow CD8+ T cells in Myeloid Neoplasms predicts response to treatment with Azacitidine\nAuthors: Tasis, A.; Papaioannou, N.; Grigoriou, M.; Paschalidis, N.; Loukogiannaki, K.; Filia, A.; Katsiki, K.; Lamprianidou, E.; Papadopoulos, V.; Rimpa, C. M.; Chatzigeorgiou, A.; Kourtzelis, I.; Gerasimou, P.; Kyprianou, I.; Kosteas, P.; Liakopoulos, P.; Liapis, K.; Kolovos, P.; Chavakis, T.; Alissafi, T.; Kotsianidis, I.; Mitroulis, I.\nScore: 1.5, Published: 2023-12-31 DOI: 10.1101/2023.12.30.23300608\nTreatment with hypomethylating agents (HMA), and specifically azacitidine (AZA), is the standard of care for patients with higher-risk myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) that are not eligible to receive intensive chemotherapy. Despite research efforts, it is not possible to predict response to treatment with HMA. In this study, we aimed to identify immune cell signatures in the bone marrow (BM) associated with treatment outcomes. By employing mass cytometry, we performed an in-depth immunophenotypic analysis in BM samples deriving from patients with myeloid neoplasms prior to treatment initiation. We identified an increased pre-treatment frequency of a CD8+ T cell subset, characterized as CD57+CXCR3+CCR7-CD45RA+, in patients with MDS and AML who did not respond to treatment with AZA, compared to responders. Furthermore, an increased baseline frequency (\u003e29%) of CD57+CXCR3+CD8+ T cells was correlated with poor overall survival. We further engaged scRNA-seq to assess the transcriptional profile of BM CD8+ T cells from treatment-naive patients with MDS and AML, to identify molecular signatures in CD8+ T subpopulations associated with favorable outcomes. Response to treatment was positively associated with enrichment of IFN-mediated pathways coupled with enhanced cytotoxic signature, whereas enrichment of the TGF-{beta} signaling pathway was observed in cell clusters from non-responders. Together, this study identified a specific CD57+CXCR3+ CD8+ T cell population with predictive value in patients with MDS and AML treated with AZA and characterized molecular signatures in CD8+ T cells linked to cytokine signaling that were associated with treatment outcomes. Key-pointsO_LIThe frequency of bone marrow CD57+CXCR3+ CD8+ T cell subset predicts response to azacitidine in MDS and AML patients. C_LIO_LIscRNA-seq analysis linked IFN pathway and TGF-{beta} pathway signature in CD8+ T cells with response and failure to treatment, respectively. C_LI\n",
  "wordCount" : "624",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/hematology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      hematology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.23299980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.23299980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.23299980">
        <p class="paperTitle">Calling for diversity: improving transfusion safety through high-throughput blood group microarray genotyping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.23299980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.23299980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wittig, M.; Steiert, T. A.; ElAbd, H.; Degenhardt, F.; Valenti, L.; Prati, D.; Ronzoni, L.; Bujanda, L.; Banales, J. M.; Blay, N.; Invernizzi, P.; Buti, M.; Albillos, A.; Fernandez, J.; Sacchi, N.; Julia, A.; Latiano, A.; de Cid, R.; D&#39;Amato, M.; Asselta, R.; Laudes, M.; Lieb, W.; Juhl, D.; Gassner, C.; Franke, A.</p>
        <p class="info">Score: 3.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.23299980' target='https://doi.org/10.1101/2023.12.15.23299980'> 10.1101/2023.12.15.23299980</a></p>
        <p class="abstract">Blood transfusions, conducted between donors compatible in their red blood cell (RBC) antigens, play a life-saving role in transfusion medicine. Genetic differences at blood group loci between ethnicities result in diversity and altered frequency of RBC antigens that need to be considered in blood transfusion. Consequently, comprehensive, and accurate blood group antigen typing is especially relevant for inter-ethnicity blood transfusions and for minorities underrepresented in the donor population. Blood group microarray genotyping is a cost-efficient and scalable method for comprehensive blood group typing. Previously, however, microarray typing has been challenging for the clinically important blood group systems Rh and MNS, as these feature highly paralogous genomic loci leading to mixed signals. We here present an approach for accurately typing blood group systems, including Rh and MNS variations, that we benchmarked in an ethnically diverse cohort. We tested its performance using gold-standard, diagnostic-grade MALDI-TOF data from 1,052-samples, including 334 HGDP-CEPH diversity panel samples and applied the approach to 4,999 samples of a COVID-19 genetics study. Overall, we obtained a 99.95% benchmarking concordance and 99.43% call rate. In summary, we provide a highly accurate and cost-efficient high-throughput genotyping method for comprehensive blood group analysis that is also suitable for ethnically diverse sample sets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.23300608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.23300608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.23300608">
        <p class="paperTitle">Single-cell analysis of bone marrow CD8&#43; T cells in Myeloid Neoplasms predicts response to treatment with Azacitidine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.23300608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.23300608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tasis, A.; Papaioannou, N.; Grigoriou, M.; Paschalidis, N.; Loukogiannaki, K.; Filia, A.; Katsiki, K.; Lamprianidou, E.; Papadopoulos, V.; Rimpa, C. M.; Chatzigeorgiou, A.; Kourtzelis, I.; Gerasimou, P.; Kyprianou, I.; Kosteas, P.; Liakopoulos, P.; Liapis, K.; Kolovos, P.; Chavakis, T.; Alissafi, T.; Kotsianidis, I.; Mitroulis, I.</p>
        <p class="info">Score: 1.5, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.23300608' target='https://doi.org/10.1101/2023.12.30.23300608'> 10.1101/2023.12.30.23300608</a></p>
        <p class="abstract">Treatment with hypomethylating agents (HMA), and specifically azacitidine (AZA), is the standard of care for patients with higher-risk myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) that are not eligible to receive intensive chemotherapy. Despite research efforts, it is not possible to predict response to treatment with HMA. In this study, we aimed to identify immune cell signatures in the bone marrow (BM) associated with treatment outcomes. By employing mass cytometry, we performed an in-depth immunophenotypic analysis in BM samples deriving from patients with myeloid neoplasms prior to treatment initiation. We identified an increased pre-treatment frequency of a CD8&#43; T cell subset, characterized as CD57&#43;CXCR3&#43;CCR7-CD45RA&#43;, in patients with MDS and AML who did not respond to treatment with AZA, compared to responders. Furthermore, an increased baseline frequency (&gt;29%) of CD57&#43;CXCR3&#43;CD8&#43; T cells was correlated with poor overall survival. We further engaged scRNA-seq to assess the transcriptional profile of BM CD8&#43; T cells from treatment-naive patients with MDS and AML, to identify molecular signatures in CD8&#43; T subpopulations associated with favorable outcomes. Response to treatment was positively associated with enrichment of IFN-mediated pathways coupled with enhanced cytotoxic signature, whereas enrichment of the TGF-{beta} signaling pathway was observed in cell clusters from non-responders. Together, this study identified a specific CD57&#43;CXCR3&#43; CD8&#43; T cell population with predictive value in patients with MDS and AML treated with AZA and characterized molecular signatures in CD8&#43; T cells linked to cytokine signaling that were associated with treatment outcomes.

Key-pointsO_LIThe frequency of bone marrow CD57&#43;CXCR3&#43; CD8&#43; T cell subset predicts response to azacitidine in MDS and AML patients.
C_LIO_LIscRNA-seq analysis linked IFN pathway and TGF-{beta} pathway signature in CD8&#43; T cells with response and failure to treatment, respectively.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
