<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="snPATHO-seq: unlocking the pathology archives
Authors: Wang, T.; Harvey, K.; Escudero Morales, J.; Kiedik, B.; Al-Eryani, G.; Greenwald, A. C.; Kalavros, N.; SegatovDezem, F.; Ma, Y.; Pita-Juarez, Y.; Wise, K.; Degletagne, C.; Elz, A.; Johanneson, J.; Pakiam, F.; Ryu, H.; Newell, E. W.; Tonon, L.; Kohlway, A.; Drennon, T.; Hadadianpour, A.; Abousoud, J.; Scott, R.; Lund, P.; Durruthy, J.; Vallejos, A. F.; Kaczorowski, D.; Warren, J.; Butler, L. M.; O&#39;Toole, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="snPATHO-seq: unlocking the pathology archives
Authors: Wang, T.; Harvey, K.; Escudero Morales, J.; Kiedik, B.; Al-Eryani, G.; Greenwald, A. C.; Kalavros, N.; SegatovDezem, F.; Ma, Y.; Pita-Juarez, Y.; Wise, K.; Degletagne, C.; Elz, A.; Johanneson, J.; Pakiam, F.; Ryu, H.; Newell, E. W.; Tonon, L.; Kohlway, A.; Drennon, T.; Hadadianpour, A.; Abousoud, J.; Scott, R.; Lund, P.; Durruthy, J.; Vallejos, A. F.; Kaczorowski, D.; Warren, J.; Butler, L. M.; O&#39;Toole, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="snPATHO-seq: unlocking the pathology archives
Authors: Wang, T.; Harvey, K.; Escudero Morales, J.; Kiedik, B.; Al-Eryani, G.; Greenwald, A. C.; Kalavros, N.; SegatovDezem, F.; Ma, Y.; Pita-Juarez, Y.; Wise, K.; Degletagne, C.; Elz, A.; Johanneson, J.; Pakiam, F.; Ryu, H.; Newell, E. W.; Tonon, L.; Kohlway, A.; Drennon, T.; Hadadianpour, A.; Abousoud, J.; Scott, R.; Lund, P.; Durruthy, J.; Vallejos, A. F.; Kaczorowski, D.; Warren, J.; Butler, L. M.; O&#39;Toole, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "snPATHO-seq: unlocking the pathology archives\nAuthors: Wang, T.; Harvey, K.; Escudero Morales, J.; Kiedik, B.; Al-Eryani, G.; Greenwald, A. C.; Kalavros, N.; SegatovDezem, F.; Ma, Y.; Pita-Juarez, Y.; Wise, K.; Degletagne, C.; Elz, A.; Johanneson, J.; Pakiam, F.; Ryu, H.; Newell, E. W.; Tonon, L.; Kohlway, A.; Drennon, T.; Hadadianpour, A.; Abousoud, J.; Scott, R.; Lund, P.; Durruthy, J.; Vallejos, A. F.; Kaczorowski, D.; Warren, J.; Butler, L. M.; O\u0026#39;Toole, S.",
  "keywords": [
    
  ],
  "articleBody": " snPATHO-seq: unlocking the pathology archives\nAuthors: Wang, T.; Harvey, K.; Escudero Morales, J.; Kiedik, B.; Al-Eryani, G.; Greenwald, A. C.; Kalavros, N.; SegatovDezem, F.; Ma, Y.; Pita-Juarez, Y.; Wise, K.; Degletagne, C.; Elz, A.; Johanneson, J.; Pakiam, F.; Ryu, H.; Newell, E. W.; Tonon, L.; Kohlway, A.; Drennon, T.; Hadadianpour, A.; Abousoud, J.; Scott, R.; Lund, P.; Durruthy, J.; Vallejos, A. F.; Kaczorowski, D.; Warren, J.; Butler, L. M.; O'Toole, S.; Plummer, J. T.; Vlachos, I.; Lundeberg, J.; Swarbrick, A.; Martelotto, L. G.\nScore: 37.2, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570700\nFormalin-fixed paraffin-embedded (FFPE) samples are valuable but underutilized in single-cell omics research due to their low DNA and RNA quality. In this study, leveraging recent single-cell genomic technology advances, we introduce a versatile method to derive high-quality single-nucleus transcriptomic data from FFPE samples.\nStructural Basis of Î¼-Opioid Receptor-Targeting by a Nanobody Antagonist\nAuthors: Yu, J.; Kumar, A.; Zhang, X.; Martin, C.; Raia, P.; Kohl, A.; Laeremans, T.; Steyaert, J.; Manglik, A.; Ballet, S.; Boland, A.; Stoeber, M.\nScore: 30.0, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570395\nThe -opioid receptor (OR), a prototypical member of the G protein-coupled receptor (GPCR) family, is the molecular target of opioid analgesics such as morphine and fentanyl. Due to the limitations and severe side effects of currently available opioid drugs, there is considerable interest in developing novel modulators of OR function. Most GPCR ligands today are small molecules, however biologics, including antibodies and nanobodies, are emerging as alternative therapeutics with clear advantages such as affinity and target selectivity. Here, we describe the nanobody NbE, which selectively binds to the OR and acts as an antagonist. We functionally characterize NbE as an extracellular and genetically encoded {micro}OR ligand and uncover the molecular basis for {micro}OR antagonism by solving the cryo-EM structure of the NbE-{micro}OR complex. NbE displays a unique ligand binding mode and achieves {micro}OR selectivity by interactions with the orthosteric pocket and extracellular receptor loops. Based on a {beta}-hairpin loop formed by NbE that deeply inserts into the {micro}OR and centers most binding contacts, we design short peptide analogues that retain {micro}OR antagonism. The work illustrates the potential of nanobodies to uniquely engage with GPCRs and describes novel OR ligands that can serve as a basis for therapeutic developments.\nEnhancer cooperativity can compensate for loss of activity over large genomic distances\nAuthors: Thomas, H. F.; Feng, S.; Huber, M.; LOUBIERE, V.; Vanina, D.; Pitasi, M.; Stark, A.; Buecker, C.\nScore: 28.5, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570399\nEnhancers are short DNA sequences that activate their target promoter from a distance; however, increasing the genomic distance between the enhancer and the promoter decreases expression levels. Many genes are controlled by combinations of multiple enhancers, yet the interaction and cooperation of individual enhancer elements is not well understood. Here, we developed a novel synthetic platform that allows building complex regulatory landscapes from the bottom up. We tested the system by integrating individual enhancers at different distances and revealed that the strength of an enhancer determines how strongly it is affected by increased genomic distance. Furthermore, synergy between two enhancer elements depends on the distance at which the two elements are integrated: introducing a weak enhancer between a strong enhancer and the promoter strongly increases reporter gene expression, allowing enhancers to activate from increased genomic distances.\nIdentification of key residues in MERS-CoV and SARS-CoV-2 main proteases for resistance against clinically applied inhibitors nirmatrelvir and ensitrelvir\nAuthors: Krismer, L.; Schoeppe, H.; Rauch, S.; Bante, D.; Sprenger, B.; Naschberger, A.; Costacurta, F.; Fuerst, A.; Sauerwein, A.; Rupp, B.; Kaserer, T.; von Laer, D.; Heilmann, E.\nScore: 31.2, Published: 2023-12-04 DOI: 10.1101/2023.12.04.569917\nThe Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an epidemic, zoonotically emerging pathogen initially reported in Saudi Arabia in 2012. MERS-CoV has the potential to mutate or recombine with other coronaviruses, thus acquiring the ability to efficiently spread among humans and become pandemic. Its high mortality rate of up to 35 % and the absence of effective targeted therapies call for the development of antiviral drugs for this pathogen. Since the beginning of the SARS-CoV-2 pandemic, extensive research has focused on identifying protease inhibitors for the treatment of SARS-CoV-2. Our intention was therefore to assess whether these protease inhibitors are viable options for combating MERS-CoV. To that end, we used previously established protease assays to quantify inhibition of the SARS-CoV-2 and MERS-CoV main proteases. Furthermore, we selected MERS-CoV-Mpro mutants resistant against nirmatrelvir, the most effective inhibitor of this protease, with a safe, surrogate virus-based system, and suggest putative resistance mechanisms. Notably, nirmatrelvir demonstrated effectiveness against various viral proteases, illustrating its potential as a broad-spectrum coronavirus inhibitor. To adress the inherent resistance of MERS-CoV-Mpro to ensitrelvir, we applied directed mutagenesis to a key ensitrelvir-interacting residue and provided structural models. One-Sentence SummaryWe investigate antivirals for MERS-CoV with a pool of SARS-CoV-2 antiviral drugs and study potential resistances developing against those drugs.\nBarcode 100K Specimens: In a Single Nanopore Run\nAuthors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.\nScore: 77.9, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282\nIt is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.\nStructural basis of tRNA recognition by the m3C-RNA-methyltransferase METTL6 in complex with SerRS seryl-tRNA synthetase\nAuthors: Throll, P.; Dolce, L. G.; Lastres, P. R.; Arnold, K.; Tengo, L.; Basu, S.; Kaiser, S.; Schneider, R.; Kowalinski, E.\nScore: 14.7, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570192\nMethylation of cytosine 32 in the anticodon loop of tRNAs to 3-methylcytosine (m3C) is crucial for cellular translation fidelity 1. Misregulation of the RNA methyltransferases setting this modification can cause aggressive cancers and metabolic disturbances 2,3. However, our understanding of the substrate selection and catalysis mode of the m3C RNA methyltransferases is currently still lacking. Here, we report the cryo-electron microscopy structure of the m3C tRNA methyltransferase METTL6 in complex with seryl-tRNA synthetase (SerRS) and their common substrate tRNASer. Through the complex structure, we identify the tRNA binding domain of METTL6. We show that SerRS acts as the tRNASer substrate selection factor for METTL6. We reveal how METTL6 and SerRS jointly coordinate the long variable arm of tRNASer in their interface. We demonstrate that SerRS augments the methylation activity of METTL6 and that direct contacts between METTL6 and SerRS are necessary for efficient tRNASer methylation. Finally, based on the structure of METTL6 in complex with SerRS and tRNASer, we postulate a universal tRNA binding mode for m3C RNA methyltransferases including METTL2 and METTL8, suggesting that these mammalian paralogues use similar ways to engage their respective tRNA substrates and co-factors.\nNucleolar Pol II interactome reveals TBPL1, PAF1, and Pol I at intergenic rDNA drive rRNA biogenesis\nAuthors: Khosraviani, N.; Yerlici, V. T.; St-Germain, J.; Hou, Y. Y.; Cao, S. B.; Ghali, C.; Krishnan, R.; Hakem, R.; Raught, B.; Mekhail, K.\nScore: 10.8, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570972\nNucleolar ribosomal DNA (rDNA) repeats control ribosome manufacturing. rDNA harbors a ribosomal RNA (rRNA) gene and an intergenic spacer (IGS). RNA polymerase (Pol) I transcribes rRNA genes yielding the rRNA components of ribosomes. Pol II at the IGS induces rRNA production by preventing Pol I from excessively synthesizing IGS non-coding RNAs (ncRNAs) that can disrupt nucleoli. At the IGS, Pol II regulatory processes and whether Pol I function can be beneficial remain unknown. Here, we identify IGS Pol II regulators, uncovering nucleolar optimization via IGS Pol I. Compartment-enriched proximity-dependent biotin identification (compBioID) showed enrichment of the TATA-less promoter-binding TBPL1 and transcription regulator PAF1 with IGS Pol II. TBPL1 localizes to TCT motifs, driving Pol II and Pol I and maintaining its baseline ncRNA levels. PAF1 promotes Pol II elongation, preventing unscheduled R-loops that hyper-restrain IGS Pol I and its ncRNAs. PAF1 or TBPL1 deficiency disrupts nucleolar organization and rRNA biogenesis. In PAF1-deficient cells, repressing unscheduled IGS R-loops rescues nucleolar organization and rRNA production. Depleting IGS Pol I-dependent ncRNAs is sufficient to compromise nucleoli. We present the interactome of nucleolar Pol II and show its control by TBPL1 and PAF1 ensures IGS Pol I ncRNAs maintaining nucleolar structure and operation.\nEfficient scar-free knock-ins of several kilobases by engineered CRISPR/Cas endonucleases\nAuthors: Schreiber, T.; Prange, A.; Schaefer, P.; Iwen, T.; Gruetzner, R.; Marillonnet, S.; Tissier, A. F.\nScore: 62.6, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568561\nIn plants and mammals, non-homologous end-joining is the dominant pathway to repair DNA double strand breaks, making it challenging to generate knock-in events. Using a transient assay in Nicotiana benthamiana we identified two groups of exonucleases, respectively from the Herpes Virus and from the bacteriophage T7 families, hat confer up to 38-fold increase in HDR frequencies when fused to Cas9. We achieved precise and scar-free insertion of several kilobases of DNA both in transient and stable transformation systems. In Arabidopsis thaliana, fusion of Cas9 to a Herpes Virus family exonuclease leads to 10-fold higher frequencies of knock-ins. Our results open perspectives for the routine production of knock-in and gene replacement events in plants. One-Sentence SummaryFusions of CRISPR endonucleases to specific 5'-exonucleases leads to significant increase in scar-free multikilobase knock-ins.\nThe Effect of Pseudoknot Base Pairing on Cotranscriptional Structural Switching of the Fluoride Riboswitch\nAuthors: Hertz, L. M.; White, E. N.; Kuznedelov, K.; Cheng, L.; Yu, A. M.; Kakkaramadam, R.; Severinov, K.; Chen, A.; Lucks, J. B.\nScore: 10.1, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570056\nA central question in biology is how RNA sequence changes influence dynamic conformational changes during cotranscriptional folding. Here we investigated this question through the study of transcriptional fluoride riboswitches, non-coding RNAs that sense the fluoride anion through the coordinated folding and rearrangement of a pseudoknotted aptamer domain and a downstream intrinsic terminator expression platform. Using a combination of E. coli RNA polymerase in vitro transcription and cellular gene expression assays, we characterized the function of mesophilic and thermophilic fluoride riboswitch variants. We showed that only variants containing the mesophilic pseudoknot function at 37 {degrees}C. We next systematically varied the pseudoknot sequence and found that a single wobble base pair is critical for function. Characterizing thermophilic variants at 65 {degrees}C through Thermus aquaticus RNA polymerase in vitro transcription showed the importance of this wobble pair for function even at elevated temperatures. Finally, we performed all-atom molecular dynamics simulations which supported the experimental findings, visualized the RNA structure switching process, and provided insight into the important role of magnesium ions. Together these studies provide deeper insights into the role of riboswitch sequence in influencing folding and function that will be important for understanding of RNA-based gene regulation and for synthetic biology applications.\nWRN Inhibition Leads to its Chromatin-Associated Degradation Via the PIAS4-RNF4-p97/VCP Axis\nAuthors: Rodriguez Perez, F.; Natwick, D. E.; Schiff, L.; Huey, M.; McSwiggen, D.; Heckert, A.; Ortega, J.; Loo, M.; Miranda, R.; Morrison, H. M.; Butler, R.; Cheng, K.; Filbin, J.; Gonzalez, E.; Rand, M.; Tang, Y.; Ho, J. J.; Zhang, Z.; Anderson, D. J.; Bashore, C.; Basham, S.\nScore: 9.6, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570895\nSynthetic lethality, the concept in which the co-occurrence of two genetic events leads to cell death while either single event alone does not, is an attractive strategy for targeted cancer therapies. A recent example of synthetic lethality as a therapeutic paradigm is the observation that cancer cells with high levels of microsatellite instability (MSI-H) are dependent on the Werner (WRN) RecQ helicase for survival. However, the mechanisms that regulate WRN spatiotemporal dynamics are not fully understood. In this study, we used our single molecule tracking (SMT) platform in combination with a recently disclosed WRN inhibitor to gain insights into WRNs dynamic localization within the nuclei of live cancer cells. We observe that WRN inhibition results in the helicase becoming trapped on chromatin, requiring p97/VCP for extraction and shuttling to the proteasome for degradation. Interestingly, this sequence of events resulting in WRN degradation appears to be MSI-H dependent. Using a phenotypic screen, we identify the PIAS4-RNF4 axis as the pathway responsible for WRN degradation and show that co-inhibition of WRN and SUMOylation has an additive toxic effect in MSI-H cells. Taken together, our work elucidates a novel regulatory mechanism for WRN. Gaining a deeper understanding into this regulatory pathway for WRN can aid in the identification of new high value targets for targeted cancer therapies.\n",
  "wordCount" : "2266",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570700">
        <p class="paperTitle">snPATHO-seq: unlocking the pathology archives</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, T.; Harvey, K.; Escudero Morales, J.; Kiedik, B.; Al-Eryani, G.; Greenwald, A. C.; Kalavros, N.; SegatovDezem, F.; Ma, Y.; Pita-Juarez, Y.; Wise, K.; Degletagne, C.; Elz, A.; Johanneson, J.; Pakiam, F.; Ryu, H.; Newell, E. W.; Tonon, L.; Kohlway, A.; Drennon, T.; Hadadianpour, A.; Abousoud, J.; Scott, R.; Lund, P.; Durruthy, J.; Vallejos, A. F.; Kaczorowski, D.; Warren, J.; Butler, L. M.; O&#39;Toole, S.; Plummer, J. T.; Vlachos, I.; Lundeberg, J.; Swarbrick, A.; Martelotto, L. G.</p>
        <p class="info">Score: 37.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570700' target='https://doi.org/10.1101/2023.12.07.570700'> 10.1101/2023.12.07.570700</a></p>
        <p class="abstract">Formalin-fixed paraffin-embedded (FFPE) samples are valuable but underutilized in single-cell omics research due to their low DNA and RNA quality. In this study, leveraging recent single-cell genomic technology advances, we introduce a versatile method to derive high-quality single-nucleus transcriptomic data from FFPE samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570395">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570395" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570395">
        <p class="paperTitle">Structural Basis of Î¼-Opioid Receptor-Targeting by a Nanobody Antagonist</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570395" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570395" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, J.; Kumar, A.; Zhang, X.; Martin, C.; Raia, P.; Kohl, A.; Laeremans, T.; Steyaert, J.; Manglik, A.; Ballet, S.; Boland, A.; Stoeber, M.</p>
        <p class="info">Score: 30.0, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570395' target='https://doi.org/10.1101/2023.12.06.570395'> 10.1101/2023.12.06.570395</a></p>
        <p class="abstract">The -opioid receptor (OR), a prototypical member of the G protein-coupled receptor (GPCR) family, is the molecular target of opioid analgesics such as morphine and fentanyl. Due to the limitations and severe side effects of currently available opioid drugs, there is considerable interest in developing novel modulators of OR function. Most GPCR ligands today are small molecules, however biologics, including antibodies and nanobodies, are emerging as alternative therapeutics with clear advantages such as affinity and target selectivity. Here, we describe the nanobody NbE, which selectively binds to the OR and acts as an antagonist. We functionally characterize NbE as an extracellular and genetically encoded {micro}OR ligand and uncover the molecular basis for {micro}OR antagonism by solving the cryo-EM structure of the NbE-{micro}OR complex. NbE displays a unique ligand binding mode and achieves {micro}OR selectivity by interactions with the orthosteric pocket and extracellular receptor loops. Based on a {beta}-hairpin loop formed by NbE that deeply inserts into the {micro}OR and centers most binding contacts, we design short peptide analogues that retain {micro}OR antagonism. The work illustrates the potential of nanobodies to uniquely engage with GPCRs and describes novel OR ligands that can serve as a basis for therapeutic developments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570399">
        <p class="paperTitle">Enhancer cooperativity can compensate for loss of activity over large genomic distances</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, H. F.; Feng, S.; Huber, M.; LOUBIERE, V.; Vanina, D.; Pitasi, M.; Stark, A.; Buecker, C.</p>
        <p class="info">Score: 28.5, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570399' target='https://doi.org/10.1101/2023.12.06.570399'> 10.1101/2023.12.06.570399</a></p>
        <p class="abstract">Enhancers are short DNA sequences that activate their target promoter from a distance; however, increasing the genomic distance between the enhancer and the promoter decreases expression levels. Many genes are controlled by combinations of multiple enhancers, yet the interaction and cooperation of individual enhancer elements is not well understood. Here, we developed a novel synthetic platform that allows building complex regulatory landscapes from the bottom up. We tested the system by integrating individual enhancers at different distances and revealed that the strength of an enhancer determines how strongly it is affected by increased genomic distance. Furthermore, synergy between two enhancer elements depends on the distance at which the two elements are integrated: introducing a weak enhancer between a strong enhancer and the promoter strongly increases reporter gene expression, allowing enhancers to activate from increased genomic distances.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569917">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569917" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569917">
        <p class="paperTitle">Identification of key residues in MERS-CoV and SARS-CoV-2 main proteases for resistance against clinically applied inhibitors nirmatrelvir and ensitrelvir</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569917" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569917" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krismer, L.; Schoeppe, H.; Rauch, S.; Bante, D.; Sprenger, B.; Naschberger, A.; Costacurta, F.; Fuerst, A.; Sauerwein, A.; Rupp, B.; Kaserer, T.; von Laer, D.; Heilmann, E.</p>
        <p class="info">Score: 31.2, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569917' target='https://doi.org/10.1101/2023.12.04.569917'> 10.1101/2023.12.04.569917</a></p>
        <p class="abstract">The Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an epidemic, zoonotically emerging pathogen initially reported in Saudi Arabia in 2012. MERS-CoV has the potential to mutate or recombine with other coronaviruses, thus acquiring the ability to efficiently spread among humans and become pandemic. Its high mortality rate of up to 35 % and the absence of effective targeted therapies call for the development of antiviral drugs for this pathogen. Since the beginning of the SARS-CoV-2 pandemic, extensive research has focused on identifying protease inhibitors for the treatment of SARS-CoV-2. Our intention was therefore to assess whether these protease inhibitors are viable options for combating MERS-CoV. To that end, we used previously established protease assays to quantify inhibition of the SARS-CoV-2 and MERS-CoV main proteases. Furthermore, we selected MERS-CoV-Mpro mutants resistant against nirmatrelvir, the most effective inhibitor of this protease, with a safe, surrogate virus-based system, and suggest putative resistance mechanisms. Notably, nirmatrelvir demonstrated effectiveness against various viral proteases, illustrating its potential as a broad-spectrum coronavirus inhibitor. To adress the inherent resistance of MERS-CoV-Mpro to ensitrelvir, we applied directed mutagenesis to a key ensitrelvir-interacting residue and provided structural models.

One-Sentence SummaryWe investigate antivirals for MERS-CoV with a pool of SARS-CoV-2 antiviral drugs and study potential resistances developing against those drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569282">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569282" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569282">
        <p class="paperTitle">Barcode 100K Specimens: In a Single Nanopore Run</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569282" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569282" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.</p>
        <p class="info">Score: 77.9, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569282' target='https://doi.org/10.1101/2023.11.29.569282'> 10.1101/2023.11.29.569282</a></p>
        <p class="abstract">It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570192" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570192">
        <p class="paperTitle">Structural basis of tRNA recognition by the m3C-RNA-methyltransferase METTL6 in complex with SerRS seryl-tRNA synthetase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Throll, P.; Dolce, L. G.; Lastres, P. R.; Arnold, K.; Tengo, L.; Basu, S.; Kaiser, S.; Schneider, R.; Kowalinski, E.</p>
        <p class="info">Score: 14.7, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570192' target='https://doi.org/10.1101/2023.12.05.570192'> 10.1101/2023.12.05.570192</a></p>
        <p class="abstract">Methylation of cytosine 32 in the anticodon loop of tRNAs to 3-methylcytosine (m3C) is crucial for cellular translation fidelity 1. Misregulation of the RNA methyltransferases setting this modification can cause aggressive cancers and metabolic disturbances 2,3. However, our understanding of the substrate selection and catalysis mode of the m3C RNA methyltransferases is currently still lacking. Here, we report the cryo-electron microscopy structure of the m3C tRNA methyltransferase METTL6 in complex with seryl-tRNA synthetase (SerRS) and their common substrate tRNASer. Through the complex structure, we identify the tRNA binding domain of METTL6. We show that SerRS acts as the tRNASer substrate selection factor for METTL6. We reveal how METTL6 and SerRS jointly coordinate the long variable arm of tRNASer in their interface. We demonstrate that SerRS augments the methylation activity of METTL6 and that direct contacts between METTL6 and SerRS are necessary for efficient tRNASer methylation. Finally, based on the structure of METTL6 in complex with SerRS and tRNASer, we postulate a universal tRNA binding mode for m3C RNA methyltransferases including METTL2 and METTL8, suggesting that these mammalian paralogues use similar ways to engage their respective tRNA substrates and co-factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.570972">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.570972" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.570972">
        <p class="paperTitle">Nucleolar Pol II interactome reveals TBPL1, PAF1, and Pol I at intergenic rDNA drive rRNA biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.570972" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.570972" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khosraviani, N.; Yerlici, V. T.; St-Germain, J.; Hou, Y. Y.; Cao, S. B.; Ghali, C.; Krishnan, R.; Hakem, R.; Raught, B.; Mekhail, K.</p>
        <p class="info">Score: 10.8, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.570972' target='https://doi.org/10.1101/2023.12.10.570972'> 10.1101/2023.12.10.570972</a></p>
        <p class="abstract">Nucleolar ribosomal DNA (rDNA) repeats control ribosome manufacturing. rDNA harbors a ribosomal RNA (rRNA) gene and an intergenic spacer (IGS). RNA polymerase (Pol) I transcribes rRNA genes yielding the rRNA components of ribosomes. Pol II at the IGS induces rRNA production by preventing Pol I from excessively synthesizing IGS non-coding RNAs (ncRNAs) that can disrupt nucleoli. At the IGS, Pol II regulatory processes and whether Pol I function can be beneficial remain unknown. Here, we identify IGS Pol II regulators, uncovering nucleolar optimization via IGS Pol I. Compartment-enriched proximity-dependent biotin identification (compBioID) showed enrichment of the TATA-less promoter-binding TBPL1 and transcription regulator PAF1 with IGS Pol II. TBPL1 localizes to TCT motifs, driving Pol II and Pol I and maintaining its baseline ncRNA levels. PAF1 promotes Pol II elongation, preventing unscheduled R-loops that hyper-restrain IGS Pol I and its ncRNAs. PAF1 or TBPL1 deficiency disrupts nucleolar organization and rRNA biogenesis. In PAF1-deficient cells, repressing unscheduled IGS R-loops rescues nucleolar organization and rRNA production. Depleting IGS Pol I-dependent ncRNAs is sufficient to compromise nucleoli. We present the interactome of nucleolar Pol II and show its control by TBPL1 and PAF1 ensures IGS Pol I ncRNAs maintaining nucleolar structure and operation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568561">
        <p class="paperTitle">Efficient scar-free knock-ins of several kilobases by engineered CRISPR/Cas endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schreiber, T.; Prange, A.; Schaefer, P.; Iwen, T.; Gruetzner, R.; Marillonnet, S.; Tissier, A. F.</p>
        <p class="info">Score: 62.6, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568561' target='https://doi.org/10.1101/2023.11.24.568561'> 10.1101/2023.11.24.568561</a></p>
        <p class="abstract">In plants and mammals, non-homologous end-joining is the dominant pathway to repair DNA double strand breaks, making it challenging to generate knock-in events. Using a transient assay in Nicotiana benthamiana we identified two groups of exonucleases, respectively from the Herpes Virus and from the bacteriophage T7 families, hat confer up to 38-fold increase in HDR frequencies when fused to Cas9. We achieved precise and scar-free insertion of several kilobases of DNA both in transient and stable transformation systems. In Arabidopsis thaliana, fusion of Cas9 to a Herpes Virus family exonuclease leads to 10-fold higher frequencies of knock-ins. Our results open perspectives for the routine production of knock-in and gene replacement events in plants.

One-Sentence SummaryFusions of CRISPR endonucleases to specific 5&#39;-exonucleases leads to significant increase in scar-free multikilobase knock-ins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570056">
        <p class="paperTitle">The Effect of Pseudoknot Base Pairing on Cotranscriptional Structural Switching of the Fluoride Riboswitch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hertz, L. M.; White, E. N.; Kuznedelov, K.; Cheng, L.; Yu, A. M.; Kakkaramadam, R.; Severinov, K.; Chen, A.; Lucks, J. B.</p>
        <p class="info">Score: 10.1, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570056' target='https://doi.org/10.1101/2023.12.05.570056'> 10.1101/2023.12.05.570056</a></p>
        <p class="abstract">A central question in biology is how RNA sequence changes influence dynamic conformational changes during cotranscriptional folding. Here we investigated this question through the study of transcriptional fluoride riboswitches, non-coding RNAs that sense the fluoride anion through the coordinated folding and rearrangement of a pseudoknotted aptamer domain and a downstream intrinsic terminator expression platform. Using a combination of E. coli RNA polymerase in vitro transcription and cellular gene expression assays, we characterized the function of mesophilic and thermophilic fluoride riboswitch variants. We showed that only variants containing the mesophilic pseudoknot function at 37 {degrees}C. We next systematically varied the pseudoknot sequence and found that a single wobble base pair is critical for function. Characterizing thermophilic variants at 65 {degrees}C through Thermus aquaticus RNA polymerase in vitro transcription showed the importance of this wobble pair for function even at elevated temperatures. Finally, we performed all-atom molecular dynamics simulations which supported the experimental findings, visualized the RNA structure switching process, and provided insight into the important role of magnesium ions. Together these studies provide deeper insights into the role of riboswitch sequence in influencing folding and function that will be important for understanding of RNA-based gene regulation and for synthetic biology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570895" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570895">
        <p class="paperTitle">WRN Inhibition Leads to its Chromatin-Associated Degradation Via the PIAS4-RNF4-p97/VCP Axis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodriguez Perez, F.; Natwick, D. E.; Schiff, L.; Huey, M.; McSwiggen, D.; Heckert, A.; Ortega, J.; Loo, M.; Miranda, R.; Morrison, H. M.; Butler, R.; Cheng, K.; Filbin, J.; Gonzalez, E.; Rand, M.; Tang, Y.; Ho, J. J.; Zhang, Z.; Anderson, D. J.; Bashore, C.; Basham, S.</p>
        <p class="info">Score: 9.6, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570895' target='https://doi.org/10.1101/2023.12.08.570895'> 10.1101/2023.12.08.570895</a></p>
        <p class="abstract">Synthetic lethality, the concept in which the co-occurrence of two genetic events leads to cell death while either single event alone does not, is an attractive strategy for targeted cancer therapies. A recent example of synthetic lethality as a therapeutic paradigm is the observation that cancer cells with high levels of microsatellite instability (MSI-H) are dependent on the Werner (WRN) RecQ helicase for survival. However, the mechanisms that regulate WRN spatiotemporal dynamics are not fully understood. In this study, we used our single molecule tracking (SMT) platform in combination with a recently disclosed WRN inhibitor to gain insights into WRNs dynamic localization within the nuclei of live cancer cells. We observe that WRN inhibition results in the helicase becoming trapped on chromatin, requiring p97/VCP for extraction and shuttling to the proteasome for degradation. Interestingly, this sequence of events resulting in WRN degradation appears to be MSI-H dependent. Using a phenotypic screen, we identify the PIAS4-RNF4 axis as the pathway responsible for WRN degradation and show that co-inhibition of WRN and SUMOylation has an additive toxic effect in MSI-H cells. Taken together, our work elucidates a novel regulatory mechanism for WRN. Gaining a deeper understanding into this regulatory pathway for WRN can aid in the identification of new high value targets for targeted cancer therapies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
