<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses
Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.
Score: 241.2, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871
TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses
Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.
Score: 241.2, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871
TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses
Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.
Score: 241.2, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871
TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses\nAuthors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.\nScore: 241.2, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871\nTnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown.",
  "keywords": [
    
  ],
  "articleBody": " Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses\nAuthors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.\nScore: 241.2, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871\nTnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.\nPreserved respiratory chain capacity and physiology in mice with profoundly reduced levels of mitochondrial respirasomes\nAuthors: Milenkovic, D.; Misic, J.; Hevler, J. F.; Molinie, T.; Chung, I.; Atanassov, I.; Li, X.; Filograna, R.; Mesaros, A.; Mourier, A.; Heck, A. J. R.; Hirst, J.; Larsson, N.-G.\nScore: 33.0, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545560\nThe mammalian respiratory chain complexes I, III2 and IV (CI, CIII2 and CIV) are critical for cellular bioenergetics and form a stable assembly, the respirasome (CI- CIII2-CIV), that is biochemically and structurally well documented. The role of the respirasome in bioenergetics and regulation of metabolism is subject to intense debate and is difficult to study because the individual respiratory chain complexes coexist together with high levels of respirasomes. To critically investigate the in vivo role of the respirasome, we generated homozygous knock-in mice that have normal levels of respiratory chain complexes but profoundly decreased levels of respirasomes. Surprisingly, the mutant mice are healthy, with preserved respiratory chain capacity and normal exercise performance. Our findings show that high levels of respirasomes are dispensable for maintaining bioenergetics and physiology in the mouse, but raises questions about their alternate functions, such as relating to regulation of protein stability and prevention of age-associated protein aggregation.\nBRG1 establishes the neuroectodermal chromatin landscape to restrict dorsal cell fates\nAuthors: Hoffman, J. A.; Muse, G. W.; Langer, L. F.; Gandara, I.; Ward, J. M.; Archer, T. K.\nScore: 8.5, Published: 2023-06-24 DOI: 10.1101/2023.06.24.546365\nCell fate decisions are achieved with gene expression changes driven by lineage-specific transcription factors (TFs). These TFs depend on chromatin remodelers including the BAF complex to activate target genes. BAF complex subunits are essential for development and frequently mutated in cancer. Thus, interrogating how BAF complexes contribute to cell fate decisions is critical for human health. We examined the requirement for the catalytic BAF subunit BRG1 in neural progenitor cell (NPC) specification from human embryonic stem cells. During the earliest stages of differentiation, BRG1 was required to establish chromatin accessibility at neuroectoderm-specific enhancers. BRG1 depletion resulted in abnormal NPC populations that differentially expressed neuroectodermal TFs, were more prone to neuronal differentiation, and precociously formed neural crest lineages. These findings demonstrate that BRG1 mediates NPC specification by ensuring proper expression of lineage-specific TFs and appropriate activation of their transcriptional programs.\nWolbachia-mediated resistance to Zika virus infection in Aedes aegypti is dominated by diverse transcriptional regulation and weak evolutionary pressures\nAuthors: Boehm, E. C.; Jaeger, A. S.; Ries, H. J.; Castaneda, D.; Weiler, A. M.; Valencia, C. C.; Weger-Lucarelli, J.; Ebel, G.; O'Connor, S. L.; Friedrich, T. C.; Zamanian, M.; Aliota, M. T.\nScore: 7.9, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546271\nA promising candidate for arbovirus control and prevention relies on replacing arbovirus-susceptible Aedes aegypti populations with mosquitoes that have been colonized by the intracellular bacterium Wolbachia and thus have a reduced capacity to transmit arboviruses. This reduced capacity to transmit arboviruses is mediated through a phenomenon referred to as pathogen blocking. Pathogen blocking has primarily been proposed as a tool to control dengue virus (DENV) transmission, however it works against a range of viruses, including Zika virus (ZIKV). Despite years of research, the molecular mechanisms underlying pathogen blocking still need to be better understood. Here, we used RNA-seq to characterize mosquito gene transcription dynamics in Ae. aegypti infected with the wMel strain of Wolbachia that are being released by the World Mosquito Program in Medellin, Colombia. Comparative analyses using ZIKV-infected, uninfected tissues, and mosquitoes without Wolbachia revealed that the influence of wMel on mosquito gene transcription is multifactorial. Importantly, because Wolbachia limits, but does not completely prevent, replication of ZIKV and other viruses in coinfected mosquitoes, there is a possibility that these viruses could evolve resistance to pathogen blocking. Therefore, to understand the influence of Wolbachia on within-host ZIKV evolution, we characterized the genetic diversity of molecularly barcoded ZIKV virus populations replicating in Wolbachia-infected mosquitoes and found that within-host ZIKV evolution was subject to weak purifying selection and, unexpectedly, loose anatomical bottlenecks in the presence and absence of Wolbachia. Together, these findings suggest that there is no clear transcriptional profile associated with Wolbachia-mediated ZIKV restriction, and that there is no evidence for ZIKV escape from this restriction in our system. Author SummaryWhen Wolbachia bacteria infect Aedes aegypti mosquitoes, they dramatically reduce the mosquitoes susceptibility to infection with a range of arthropod-borne viruses, including Zika virus (ZIKV). Although this pathogen-blocking effect has been widely recognized, its mechanisms remain unclear. Furthermore, because Wolbachia limits, but does not completely prevent, replication of ZIKV and other viruses in coinfected mosquitoes, there is a possibility that these viruses could evolve resistance to Wolbachia-mediated blocking. Here, we use host transcriptomics and viral genome sequencing to examine the mechanisms of ZIKV pathogen blocking by Wolbachia and viral evolutionary dynamics in Ae. aegypti mosquitoes. We find complex transcriptome patterns that do not suggest a single clear mechanism for pathogen blocking. We also find no evidence that Wolbachia exerts detectable selective pressures on ZIKV in coinfected mosquitoes. Together our data suggest that it may be difficult for ZIKV to evolve Wolbachia resistance, perhaps due to the complexity of the pathogen blockade mechanism.\nEfficient Formation of Single-copy Human Artificial Chromosomes\nAuthors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.\nScore: 7.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547284\nLarge DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through ~125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a ~750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.\nMulti-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation\nAuthors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.\nScore: 7.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546730\nPartial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.\nTarget-directed microRNA degradation regulates developmental microRNA expression and embryonic growth in mammals\nAuthors: Jones, B. T.; Han, J.; Zhang, H.; Hammer, R. E.; Evers, B. M.; Rakheja, D.; Acharya, A.; Mendell, J. T.\nScore: 6.5, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546601\nMicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that play critical roles in development and disease. Target-directed miRNA degradation (TDMD), a pathway in which miRNAs that bind to specialized targets with extensive complementarity are rapidly decayed, has emerged as a potent mechanism of controlling miRNA levels. Nevertheless, the biological role and scope of miRNA regulation by TDMD in mammals remains poorly understood. To address these questions, we generated mice with constitutive or conditional deletion of Zswim8, which encodes an essential TDMD factor. Loss of Zswim8 resulted in developmental defects in heart and lung, growth restriction, and perinatal lethality. Small RNA sequencing of embryonic tissues revealed widespread miRNA regulation by TDMD and greatly expanded the known catalog of miRNAs regulated by this pathway. These experiments also uncovered novel features of TDMD-regulated miRNAs, including their enrichment in co-transcribed clusters and examples in which TDMD underlies arm switching, a phenomenon wherein the dominant strand of a miRNA precursor changes in different tissues or conditions. Importantly, deletion of two miRNAs, miR-322 and miR-503, rescued growth of Zswim8 null embryos, directly implicating the TDMD pathway as a regulator of mammalian body size. These data illuminate the broad landscape and developmental role of TDMD in mammals.\nDNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor\nAuthors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.\nScore: 17.2, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516\nThe human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin. However, how HUSH and MORC2 operate alongside DNA methylation, a central epigenetic regulator of repeat transcription, remains poorly understood. Here we interrogate this relationship in human neural progenitor cells (hNPCs), a somatic model of brain development that tolerates removal of DNA methyltransferase DNMT1. Upon loss of MORC2 or HUSH subunit TASOR in hNPCs, L1s remain silenced by robust promoter methylation. However, genome demethylation and activation of evolutionarily-young L1s attracts MORC2 binding. Simultaneous depletion of DNMT1 and MORC2 causes massive accumulation of L1 transcripts. We identify the same mechanistic hierarchy at pericentromeric -satellites and clustered protocadherin genes, repetitive elements important for chromosome structure and neurodevelopment respectively. Our data delineate the independent epigenetic control of repeats in somatic cells, with implications for understanding the vital functions of HUSH-MORC2 in hypomethylated contexts throughout human development.\nLoss of endothelial cell heterogeneity in arteries after obesogenic diet\nAuthors: Dunaway, L. S.; Luse, M. A.; Nyshadham, S.; Bulut, G.; Alencar, G. F.; Chavkin, N. W.; Cortese-Krott, M.; Hirschi, K. K.; Isakson, B. E.\nScore: 11.8, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546320\nBackgroundIt is well recognized that obesity leads to arterial endothelial dysfunction and cardiovascular disease. However, the progression to endothelial dysfunction is not clear. Endothelial cells (ECs) adapt to the unique needs of their resident tissue and respond to systemic metabolic perturbations. We sought to better understand how obesity affects EC phenotypes in different tissues specifically focusing on mitochondrial gene expression. MethodsWe performed bulk RNA sequencing (RNA-seq) and single cell RNA-seq (scRNA-seq) on mesenteric and adipose ECs isolated from normal chow (NC) and high fat diet (HFD) fed mice. Differential gene expression, gene ontology pathway, and transcription factor analyses were performed. We further investigated our hypothesis in humans using published human adipose single nuclei RNA-seq (snRNA-seq) data. ResultsBulk RNA-seq revealed higher mitochondrial gene expression in adipose ECs compared to mesenteric ECs in both NC and HFD mice. We then performed scRNA-seq and categorized EC clusters as arterial, capillary, venous, or lymphatic. HFD decreased the number of differentially expressed genes between mesenteric and adipose ECs in all subtypes, but the largest effect was seen in arterial ECs. Further analysis of arterial ECs revealed genes coding for mitochondrial oxidative phosphorylation proteins were enriched in adipose compared to mesentery under NC conditions. In HFD mice, these genes were decreased in adipose ECs becoming similar to mesenteric ECs. Transcription factor analysis revealed C/EBP and PPAR{gamma}, both known to regulate lipid handling and metabolism, had high specificity scores in the NC adipose artery ECs. These findings were recapitulated in snRNA-seq data from human adipose. ConclusionsThese data suggest mesenteric and adipose arterial ECs metabolize lipids differently and the transcriptional phenotype of these two vascular beds converge in obesity, in part, due to downregulation of PPAR{gamma} and C/EBP in adipose artery ECs. This work lays the foundation for investigating vascular bed specific adaptations to obesity.\nThe influence of diet on gut microbiome and body mass dynamics in a capital-breeding migratory bird\nAuthors: Jones, I. L.; Marsh, K.; Handby, T.; Hopkins, K.; Slezacek, J.; Bearhop, S.; Harrison, X.\nScore: 5.9, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546090\nGut-associated microbial communities are known to play a vital role in the health and fitness of their hosts. Though studies investigating the factors associated with among-individual variation in microbiome structure in wild animal species are increasing, knowledge of this variation at the individual level is scarce, despite the clear link between microbiome and nutritional status uncovered in humans and model organisms. Here, we combine detailed life history, foraging preference, and faecal microbiome data to investigate the relationship between diet, microbiome stability and rates of body mass gain in a migratory capital-breeding bird, the light-bellied Brent goose (Branta bernicla hrota). Our findings suggest that there is a significant relationship between foraging and the diversity, composition and stability of the faecal microbiome. We also suggest a link between foraging phenotype and the rates of mass gain during the spring staging of a capital breeder. This study offers rare insight into individual-level temporal dynamics of the gut microbiome of a wild host. Further work is needed to uncover the functional link between individual dietary choices, gut microbiome structure and stability, and the implications this has for the reproductive success of this capital breeder.\n",
  "wordCount" : "2519",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544871">
        <p class="paperTitle">Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.</p>
        <p class="info">Score: 241.2, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544871' target='https://doi.org/10.1101/2023.06.13.544871'> 10.1101/2023.06.13.544871</a></p>
        <p class="abstract">TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545560">
        <p class="paperTitle">Preserved respiratory chain capacity and physiology in mice with profoundly reduced levels of mitochondrial respirasomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Milenkovic, D.; Misic, J.; Hevler, J. F.; Molinie, T.; Chung, I.; Atanassov, I.; Li, X.; Filograna, R.; Mesaros, A.; Mourier, A.; Heck, A. J. R.; Hirst, J.; Larsson, N.-G.</p>
        <p class="info">Score: 33.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545560' target='https://doi.org/10.1101/2023.06.19.545560'> 10.1101/2023.06.19.545560</a></p>
        <p class="abstract">The mammalian respiratory chain complexes I, III2 and IV (CI, CIII2 and CIV) are critical for cellular bioenergetics and form a stable assembly, the respirasome (CI- CIII2-CIV), that is biochemically and structurally well documented. The role of the respirasome in bioenergetics and regulation of metabolism is subject to intense debate and is difficult to study because the individual respiratory chain complexes coexist together with high levels of respirasomes. To critically investigate the in vivo role of the respirasome, we generated homozygous knock-in mice that have normal levels of respiratory chain complexes but profoundly decreased levels of respirasomes. Surprisingly, the mutant mice are healthy, with preserved respiratory chain capacity and normal exercise performance. Our findings show that high levels of respirasomes are dispensable for maintaining bioenergetics and physiology in the mouse, but raises questions about their alternate functions, such as relating to regulation of protein stability and prevention of age-associated protein aggregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.546365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.546365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.546365">
        <p class="paperTitle">BRG1 establishes the neuroectodermal chromatin landscape to restrict dorsal cell fates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.546365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.546365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffman, J. A.; Muse, G. W.; Langer, L. F.; Gandara, I.; Ward, J. M.; Archer, T. K.</p>
        <p class="info">Score: 8.5, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.546365' target='https://doi.org/10.1101/2023.06.24.546365'> 10.1101/2023.06.24.546365</a></p>
        <p class="abstract">Cell fate decisions are achieved with gene expression changes driven by lineage-specific transcription factors (TFs). These TFs depend on chromatin remodelers including the BAF complex to activate target genes. BAF complex subunits are essential for development and frequently mutated in cancer. Thus, interrogating how BAF complexes contribute to cell fate decisions is critical for human health. We examined the requirement for the catalytic BAF subunit BRG1 in neural progenitor cell (NPC) specification from human embryonic stem cells. During the earliest stages of differentiation, BRG1 was required to establish chromatin accessibility at neuroectoderm-specific enhancers. BRG1 depletion resulted in abnormal NPC populations that differentially expressed neuroectodermal TFs, were more prone to neuronal differentiation, and precociously formed neural crest lineages. These findings demonstrate that BRG1 mediates NPC specification by ensuring proper expression of lineage-specific TFs and appropriate activation of their transcriptional programs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546271" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546271">
        <p class="paperTitle">Wolbachia-mediated resistance to Zika virus infection in Aedes aegypti is dominated by diverse transcriptional regulation and weak evolutionary pressures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boehm, E. C.; Jaeger, A. S.; Ries, H. J.; Castaneda, D.; Weiler, A. M.; Valencia, C. C.; Weger-Lucarelli, J.; Ebel, G.; O&#39;Connor, S. L.; Friedrich, T. C.; Zamanian, M.; Aliota, M. T.</p>
        <p class="info">Score: 7.9, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546271' target='https://doi.org/10.1101/2023.06.26.546271'> 10.1101/2023.06.26.546271</a></p>
        <p class="abstract">A promising candidate for arbovirus control and prevention relies on replacing arbovirus-susceptible Aedes aegypti populations with mosquitoes that have been colonized by the intracellular bacterium Wolbachia and thus have a reduced capacity to transmit arboviruses. This reduced capacity to transmit arboviruses is mediated through a phenomenon referred to as pathogen blocking. Pathogen blocking has primarily been proposed as a tool to control dengue virus (DENV) transmission, however it works against a range of viruses, including Zika virus (ZIKV). Despite years of research, the molecular mechanisms underlying pathogen blocking still need to be better understood. Here, we used RNA-seq to characterize mosquito gene transcription dynamics in Ae. aegypti infected with the wMel strain of Wolbachia that are being released by the World Mosquito Program in Medellin, Colombia. Comparative analyses using ZIKV-infected, uninfected tissues, and mosquitoes without Wolbachia revealed that the influence of wMel on mosquito gene transcription is multifactorial. Importantly, because Wolbachia limits, but does not completely prevent, replication of ZIKV and other viruses in coinfected mosquitoes, there is a possibility that these viruses could evolve resistance to pathogen blocking. Therefore, to understand the influence of Wolbachia on within-host ZIKV evolution, we characterized the genetic diversity of molecularly barcoded ZIKV virus populations replicating in Wolbachia-infected mosquitoes and found that within-host ZIKV evolution was subject to weak purifying selection and, unexpectedly, loose anatomical bottlenecks in the presence and absence of Wolbachia. Together, these findings suggest that there is no clear transcriptional profile associated with Wolbachia-mediated ZIKV restriction, and that there is no evidence for ZIKV escape from this restriction in our system.

Author SummaryWhen Wolbachia bacteria infect Aedes aegypti mosquitoes, they dramatically reduce the mosquitoes susceptibility to infection with a range of arthropod-borne viruses, including Zika virus (ZIKV). Although this pathogen-blocking effect has been widely recognized, its mechanisms remain unclear. Furthermore, because Wolbachia limits, but does not completely prevent, replication of ZIKV and other viruses in coinfected mosquitoes, there is a possibility that these viruses could evolve resistance to Wolbachia-mediated blocking. Here, we use host transcriptomics and viral genome sequencing to examine the mechanisms of ZIKV pathogen blocking by Wolbachia and viral evolutionary dynamics in Ae. aegypti mosquitoes. We find complex transcriptome patterns that do not suggest a single clear mechanism for pathogen blocking. We also find no evidence that Wolbachia exerts detectable selective pressures on ZIKV in coinfected mosquitoes. Together our data suggest that it may be difficult for ZIKV to evolve Wolbachia resistance, perhaps due to the complexity of the pathogen blockade mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547284">
        <p class="paperTitle">Efficient Formation of Single-copy Human Artificial Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.</p>
        <p class="info">Score: 7.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547284' target='https://doi.org/10.1101/2023.06.30.547284'> 10.1101/2023.06.30.547284</a></p>
        <p class="abstract">Large DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through ~125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a ~750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546730">
        <p class="paperTitle">Multi-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.</p>
        <p class="info">Score: 7.1, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546730' target='https://doi.org/10.1101/2023.06.30.546730'> 10.1101/2023.06.30.546730</a></p>
        <p class="abstract">Partial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546601">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546601" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546601">
        <p class="paperTitle">Target-directed microRNA degradation regulates developmental microRNA expression and embryonic growth in mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546601" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546601" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, B. T.; Han, J.; Zhang, H.; Hammer, R. E.; Evers, B. M.; Rakheja, D.; Acharya, A.; Mendell, J. T.</p>
        <p class="info">Score: 6.5, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546601' target='https://doi.org/10.1101/2023.06.26.546601'> 10.1101/2023.06.26.546601</a></p>
        <p class="abstract">MicroRNAs (miRNAs) are post-transcriptional regulators of gene expression that play critical roles in development and disease. Target-directed miRNA degradation (TDMD), a pathway in which miRNAs that bind to specialized targets with extensive complementarity are rapidly decayed, has emerged as a potent mechanism of controlling miRNA levels. Nevertheless, the biological role and scope of miRNA regulation by TDMD in mammals remains poorly understood. To address these questions, we generated mice with constitutive or conditional deletion of Zswim8, which encodes an essential TDMD factor. Loss of Zswim8 resulted in developmental defects in heart and lung, growth restriction, and perinatal lethality. Small RNA sequencing of embryonic tissues revealed widespread miRNA regulation by TDMD and greatly expanded the known catalog of miRNAs regulated by this pathway. These experiments also uncovered novel features of TDMD-regulated miRNAs, including their enrichment in co-transcribed clusters and examples in which TDMD underlies  arm switching, a phenomenon wherein the dominant strand of a miRNA precursor changes in different tissues or conditions. Importantly, deletion of two miRNAs, miR-322 and miR-503, rescued growth of Zswim8 null embryos, directly implicating the TDMD pathway as a regulator of mammalian body size. These data illuminate the broad landscape and developmental role of TDMD in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545516">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545516" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545516">
        <p class="paperTitle">DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545516" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545516" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.</p>
        <p class="info">Score: 17.2, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545516' target='https://doi.org/10.1101/2023.06.21.545516'> 10.1101/2023.06.21.545516</a></p>
        <p class="abstract">The human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin. However, how HUSH and MORC2 operate alongside DNA methylation, a central epigenetic regulator of repeat transcription, remains poorly understood. Here we interrogate this relationship in human neural progenitor cells (hNPCs), a somatic model of brain development that tolerates removal of DNA methyltransferase DNMT1. Upon loss of MORC2 or HUSH subunit TASOR in hNPCs, L1s remain silenced by robust promoter methylation. However, genome demethylation and activation of evolutionarily-young L1s attracts MORC2 binding. Simultaneous depletion of DNMT1 and MORC2 causes massive accumulation of L1 transcripts. We identify the same mechanistic hierarchy at pericentromeric -satellites and clustered protocadherin genes, repetitive elements important for chromosome structure and neurodevelopment respectively. Our data delineate the independent epigenetic control of repeats in somatic cells, with implications for understanding the vital functions of HUSH-MORC2 in hypomethylated contexts throughout human development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546320">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546320" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546320">
        <p class="paperTitle">Loss of endothelial cell heterogeneity in arteries after obesogenic diet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546320" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546320" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dunaway, L. S.; Luse, M. A.; Nyshadham, S.; Bulut, G.; Alencar, G. F.; Chavkin, N. W.; Cortese-Krott, M.; Hirschi, K. K.; Isakson, B. E.</p>
        <p class="info">Score: 11.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546320' target='https://doi.org/10.1101/2023.06.23.546320'> 10.1101/2023.06.23.546320</a></p>
        <p class="abstract">BackgroundIt is well recognized that obesity leads to arterial endothelial dysfunction and cardiovascular disease. However, the progression to endothelial dysfunction is not clear. Endothelial cells (ECs) adapt to the unique needs of their resident tissue and respond to systemic metabolic perturbations. We sought to better understand how obesity affects EC phenotypes in different tissues specifically focusing on mitochondrial gene expression.

MethodsWe performed bulk RNA sequencing (RNA-seq) and single cell RNA-seq (scRNA-seq) on mesenteric and adipose ECs isolated from normal chow (NC) and high fat diet (HFD) fed mice. Differential gene expression, gene ontology pathway, and transcription factor analyses were performed. We further investigated our hypothesis in humans using published human adipose single nuclei RNA-seq (snRNA-seq) data.

ResultsBulk RNA-seq revealed higher mitochondrial gene expression in adipose ECs compared to mesenteric ECs in both NC and HFD mice. We then performed scRNA-seq and categorized EC clusters as arterial, capillary, venous, or lymphatic. HFD decreased the number of differentially expressed genes between mesenteric and adipose ECs in all subtypes, but the largest effect was seen in arterial ECs. Further analysis of arterial ECs revealed genes coding for mitochondrial oxidative phosphorylation proteins were enriched in adipose compared to mesentery under NC conditions. In HFD mice, these genes were decreased in adipose ECs becoming similar to mesenteric ECs. Transcription factor analysis revealed C/EBP and PPAR{gamma}, both known to regulate lipid handling and metabolism, had high specificity scores in the NC adipose artery ECs. These findings were recapitulated in snRNA-seq data from human adipose.

ConclusionsThese data suggest mesenteric and adipose arterial ECs metabolize lipids differently and the transcriptional phenotype of these two vascular beds converge in obesity, in part, due to downregulation of PPAR{gamma} and C/EBP in adipose artery ECs. This work lays the foundation for investigating vascular bed specific adaptations to obesity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546090" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546090">
        <p class="paperTitle">The influence of diet on gut microbiome and body mass dynamics in a capital-breeding migratory bird</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, I. L.; Marsh, K.; Handby, T.; Hopkins, K.; Slezacek, J.; Bearhop, S.; Harrison, X.</p>
        <p class="info">Score: 5.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546090' target='https://doi.org/10.1101/2023.06.22.546090'> 10.1101/2023.06.22.546090</a></p>
        <p class="abstract">Gut-associated microbial communities are known to play a vital role in the health and fitness of their hosts. Though studies investigating the factors associated with among-individual variation in microbiome structure in wild animal species are increasing, knowledge of this variation at the individual level is scarce, despite the clear link between microbiome and nutritional status uncovered in humans and model organisms. Here, we combine detailed life history, foraging preference, and faecal microbiome data to investigate the relationship between diet, microbiome stability and rates of body mass gain in a migratory capital-breeding bird, the light-bellied Brent goose (Branta bernicla hrota). Our findings suggest that there is a significant relationship between foraging and the diversity, composition and stability of the faecal microbiome. We also suggest a link between foraging phenotype and the rates of mass gain during the spring staging of a capital breeder. This study offers rare insight into individual-level temporal dynamics of the gut microbiome of a wild host. Further work is needed to uncover the functional link between individual dietary choices, gut microbiome structure and stability, and the implications this has for the reproductive success of this capital breeder.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
