<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Interplay of condensation and chromatin binding underlies BRD4 targeting
Authors: Strom, A. R.; Eeftens, J. M.; Polyachenko, Y.; Weaver, C. J.; Watanabe, H.-F.; Bracha, D.; Orlovsky, N. D.; Jumper, C. C.; Jacobs, W. M.; Brangwynne, C. P.
Score: 26.8, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579384
Nuclear compartments form via biomolecular phase separation, mediated through multivalent properties of biomolecules concentrated within condensates. Certain compartments are associated with specific chromatin regions, including transcriptional initiation condensates, which are composed of transcription factors and transcriptional machinery, and form at acetylated regions including enhancer and promoter loci.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Interplay of condensation and chromatin binding underlies BRD4 targeting
Authors: Strom, A. R.; Eeftens, J. M.; Polyachenko, Y.; Weaver, C. J.; Watanabe, H.-F.; Bracha, D.; Orlovsky, N. D.; Jumper, C. C.; Jacobs, W. M.; Brangwynne, C. P.
Score: 26.8, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579384
Nuclear compartments form via biomolecular phase separation, mediated through multivalent properties of biomolecules concentrated within condensates. Certain compartments are associated with specific chromatin regions, including transcriptional initiation condensates, which are composed of transcription factors and transcriptional machinery, and form at acetylated regions including enhancer and promoter loci." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Interplay of condensation and chromatin binding underlies BRD4 targeting
Authors: Strom, A. R.; Eeftens, J. M.; Polyachenko, Y.; Weaver, C. J.; Watanabe, H.-F.; Bracha, D.; Orlovsky, N. D.; Jumper, C. C.; Jacobs, W. M.; Brangwynne, C. P.
Score: 26.8, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579384
Nuclear compartments form via biomolecular phase separation, mediated through multivalent properties of biomolecules concentrated within condensates. Certain compartments are associated with specific chromatin regions, including transcriptional initiation condensates, which are composed of transcription factors and transcriptional machinery, and form at acetylated regions including enhancer and promoter loci."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Interplay of condensation and chromatin binding underlies BRD4 targeting\nAuthors: Strom, A. R.; Eeftens, J. M.; Polyachenko, Y.; Weaver, C. J.; Watanabe, H.-F.; Bracha, D.; Orlovsky, N. D.; Jumper, C. C.; Jacobs, W. M.; Brangwynne, C. P.\nScore: 26.8, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579384\nNuclear compartments form via biomolecular phase separation, mediated through multivalent properties of biomolecules concentrated within condensates. Certain compartments are associated with specific chromatin regions, including transcriptional initiation condensates, which are composed of transcription factors and transcriptional machinery, and form at acetylated regions including enhancer and promoter loci.",
  "keywords": [
    
  ],
  "articleBody": " Interplay of condensation and chromatin binding underlies BRD4 targeting\nAuthors: Strom, A. R.; Eeftens, J. M.; Polyachenko, Y.; Weaver, C. J.; Watanabe, H.-F.; Bracha, D.; Orlovsky, N. D.; Jumper, C. C.; Jacobs, W. M.; Brangwynne, C. P.\nScore: 26.8, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579384\nNuclear compartments form via biomolecular phase separation, mediated through multivalent properties of biomolecules concentrated within condensates. Certain compartments are associated with specific chromatin regions, including transcriptional initiation condensates, which are composed of transcription factors and transcriptional machinery, and form at acetylated regions including enhancer and promoter loci. While protein self-interactions, especially within low-complexity and intrinsically disordered regions, are known to mediate condensation, the role of substrate-binding interactions in regulating the formation and function of biomolecular condensates is under-explored. Here, utilizing live-cell experiments in parallel with coarse-grained simulations, we investigate how chromatin interaction of the transcription factor BRD4 modulates its condensate formation. We find that both kinetic and thermodynamic properties of BRD4 condensation are affected by chromatin binding: nucleation rate is sensitive to BRD4-chromatin interactions, providing an explanation for the selective formation of BRD4 condensates at acetylated chromatin regions, and thermodynamically, multivalent acetylated chromatin sites provide a platform for BRD4 clustering below the concentration required for off-chromatin condensation. This provides a molecular and physical explanation of the relationship between nuclear condensates and epigenetically modified chromatin that results in their mutual spatiotemporal regulation, suggesting that epigenetic modulation is an important mechanism by which the cell targets transcriptional condensates to specific chromatin loci.\nStructure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation\nAuthors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.\nScore: 22.7, Published: 2024-02-10 DOI: 10.1101/2024.02.10.579772\nHuman cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry. HCMV vaccine candidates containing gB have demonstrated moderate clinical efficacy, but no HCMV vaccine has been FDA-approved. Here, we used structure-based design to identify and characterize amino acid substitutions that stabilize gB in its metastable prefusion conformation. One variant containing two engineered interprotomer disulfide bonds and two cavity-filling substitutions (gB-C7), displayed increased expression and thermostability. A 2.8 [A] resolution cryo-electron microscopy structure shows that gB-C7 adopts a prefusion-like conformation, revealing additional structural elements at the membrane-distal apex. Unlike previous observations for several class I viral fusion proteins, mice immunized with postfusion or prefusion-stabilized forms of soluble gB protein displayed similar neutralizing antibody titers, here specifically against an HCMV laboratory strain on fibroblasts. Collectively, these results identify initial strategies to stabilize class III viral fusion proteins and provide tools to probe gB-directed antibody responses. TeaserA structure-based design campaign leads to stabilization of the class III viral fusion protein from HCMV in a prefusion-like conformation.\nIntroducing synthetic thermostable RNase inhibitors to single-cell RNA-seq\nAuthors: Noble, J. C.; Lentini, A.; Hageman-Jensen, M.; Sandberg, R.; Reinius, B.\nScore: 16.6, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579116\nSingle-cell RNA-sequencing (scRNAseq) is revolutionizing biomedicine, propelled by advances in methodology, ease of use, and cost reduction of library preparation. Over the past decade, there have been remarkable technical improvements in most aspects of single-cell transcriptomics. Yet, there has been little to no progress in advancing RNase inhibition despite that maintained RNA integrity is critical during cell collection, storage, and cDNA library generation. Here, we demonstrate that a synthetic thermostable RNase inhibitor yields single-cell libraries of equal or superior quality compared to ubiquitously used protein-based recombinant RNase inhibitors (RRIs). Importantly, the synthetic RNase inhibitor provides additional unique improvements in reproducibility and throughput, enables new experimental workflows including heat cycles, and can reduce the need for dry-ice transports. In summary, replacing RRIs represents a substantial advancement in the field of single-cell transcriptomics.\nChAHP2 and ChAHP control diverse retrotransposons by complementary activities\nAuthors: Ahel, J.; Pandey, A.; Schwaiger, M.; Mohn, F.; Basters, A.; Kempf, G.; Andriollo, A.; Kaaij, L.; Hess, D.; Bühler, M.\nScore: 14.4, Published: 2024-02-07 DOI: 10.1101/2024.02.05.578923\nRetrotransposon control in mammals is an intricate process that is effectuated by a broad network of chromatin regulatory pathways. We previously discovered ChAHP, a protein complex with repressive activity against SINE retrotransposons, composed of the transcription factor ADNP, chromatin remodeler CHD4, and HP1 proteins. Here we identify ChAHP2, a protein complex homologous to ChAHP, wherein ADNP is replaced by ADNP2. ChAHP2 is predominantly targeted to ERVs and LINEs, via HP1{beta}-mediated binding of H3K9 trimethylated histones. We further demonstrate that ChAHP also binds these elements in a mechanistically equivalent manner to ChAHP2, and distinct from DNA sequence-specific recruitment at SINEs. Genetic ablation of ADNP2 alleviates ERV and LINE1 repression, which is synthetically exacerbated by additional depletion of ADNP. Together, our results reveal that the ChAHP and ChAHP2 complexes function to control both non-autonomous and autonomous retrotransposons by complementary activities, further adding to the complexity of mammalian transposon control.\nProgrammable RNA writing with trans-splicing\nAuthors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.\nScore: 128.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223\nRNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing \u0026 Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5' or 3' trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington's disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntingtons disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.\nRewriting endogenous human transcripts with trans-splicing\nAuthors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.\nScore: 84.0, Published: 2024-01-30 DOI: 10.1101/2024.01.29.577779\nSplicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.\nSingle-molecule chromatin configurations link transcription factor binding to expression in human cells\nAuthors: Doughty, B. R.; Hinks, M. M.; Schaepe, J. M.; Marinov, G. K.; Thurm, A. R.; Rios-Martinez, C.; Parks, B. E.; Tan, Y.; Marklund, E.; Dubocanin, D.; Bintu, L.; Greenleaf, W. J.\nScore: 74.3, Published: 2024-02-04 DOI: 10.1101/2024.02.02.578660\nThe binding of multiple transcription factors (TFs) to genomic enhancers activates gene expression in mammalian cells. However, the molecular details that link enhancer sequence to TF binding, promoter state, and gene expression levels remain opaque. We applied single-molecule footprinting (SMF) to measure the simultaneous occupancy of TFs, nucleosomes, and components of the transcription machinery on engineered enhancer/promoter constructs with variable numbers of TF binding sites for both a synthetic and an endogenous TF. We find that activation domains enhance a TFs capacity to compete with nucleosomes for binding to DNA in a BAF-dependent manner, TF binding on nucleosome-free DNA is consistent with independent binding between TFs, and average TF occupancy linearly contributes to promoter activation rates. We also decompose TF strength into separable binding and activation terms, which can be tuned and perturbed independently. Finally, we develop thermodynamic and kinetic models that quantitatively predict both the binding microstates observed at the enhancer and subsequent time-dependent gene expression. This work provides a template for quantitative dissection of distinct contributors to gene activation, including the activity of chromatin remodelers, TF activation domains, chromatin acetylation, TF concentration, TF binding affinity, and TF binding site configuration.\nA rapid, cost-effective, colorimetric LAMP assay (CLASS) for detecting invasive malaria vector, Anopheles stephensi\nAuthors: Rafferty, C.; Raise, G.; Scaife, J.; Abongo, B.; Omondi, S.; Milanoi, S.; Muchoki, M.; Onyango, B.; Ochomo, E.; Zohdy, S.\nScore: 10.1, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579110\nAnopheles stephensi, an invasive malaria vector in Africa, has the potential to impact the landscape of malaria on the continent, threatening to put an additional 126 million people per year at risk of malaria, largely in peri-urban/urban areas. To accelerate the early detection and rapid response to An. stephensi and ensure no gains made in malaria control and elimination are lost, it is critical to confirm the presence of the species and the geographic extent of its spread to inform control. However, morphological identification may be misinterpreted if specimens are damaged and existing molecular species confirmation assays require specialized laboratory equipment and training and may be challenging to interpret, requiring additional sequencing confirmation. A colorimetric rapid loop-mediated isothermal amplification (LAMP) assay for molecular An. stephensi species identification was developed and optimized. The colorimetric assay requires only a heat source and reagents and can be used with or without DNA extraction resulting in positive color change in 30-35 minutes. To determine analytical sensitivity, a 1:10 dilution series of the DNA extract was conducted showing 100% assay sensitivity down to 0.003 nanograms. To determine specificity, three different An. stephensi laboratory strains (STE2, SDA 500, UCI), 8 other Anopheles mosquito species, and Aedes aegypti were compared, and the results indicated 100% specificity across these species. To determine use without the need for DNA extraction, samples evaluated included a single mosquito leg, whole adult or larval mosquitoes, and pooled DNA extract from several mosquito species. A total of 1687 individual reactions were tested during optimization and all LAMP assay results were compared against the conventional PCR assay and confirmed through Sanger sequencing. To validate the optimized assay on wild caught specimens, DNA extracted from 12 wild caught, sequence-confirmed An. stephensi from Marsabit, Kenya, were tested and the colorimetric assay was accurate in identifying all of the specimens as An. stephensi. The assay described presents an opportunity to accelerate An. stephensi molecular identification in new and existing locations in Africa, within its endemic range, and globally. These findings present a simple, rapid, unique alternative to existing PCR and sequencing-based An. stephensi species identification and confirmation strategies. With additional field validation studies, molecular screening tools like the colorimetric LAMP-based An. stephensi species identification (CLASS) assay fill an important gap of rapid confirmation of this invasive vector and presents an ideal opportunity to better understand the spread of the species in Africa and other recently invaded areas, thus accelerating a response to mitigate its long-term impacts on malaria on the continent.\nThe MYCN oncoprotein and helicases DDX17 and DDX5 have opposite effects on the production of chimeric transcripts in neuroblastoma cells\nAuthors: Clerc, V.; Valat, J.; Grand, X.; Fontrodona, N.; Bazire, M.; Rama, N.; Auboeuf, D.; Gibert, B.; Mortreux, F.; Bourgeois, C. F.\nScore: 6.6, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578895\nDEAD box helicases DDX17 and DDX5 control the termination of transcription and the associated cleavage of the 3 end of transcripts. Here we show that the transcriptional readthrough induced by their depletion in neuroblastoma cells also results in increased production of chimeric transcripts from tandemly oriented genes. Analysis of neuroblastoma tumours in which chimeric transcripts are abundant revealed that low expression of the DDX17 and DDX5 genes is associated with poor overall patient survival. Low DDX17 expression is also significantly associated with high-risk tumours and is inversely correlated with MYCN oncogene amplification, suggesting a link between these two factors. We demonstrate that changes in MYCN expression do not affect the expression of either helicase, but alter transcription termination leading to the production of chimeric transcripts. We provide evidence that MYCN acts on termination through its direct binding to the 3 region of genes and that it interacts with DDX17, suggesting that it may inhibit the activity of the helicase. Collectively, our work reveals a novel function of MYCN in transcription termination and suggests that the deregulation of MYCN and DDX17/DDX5 expression in neuroblastoma may lead to the expression of non-canonical and potentially harmful RNA molecules.\nA Mycobacterium tuberculosis secreted virulence factor disrupts host snRNP biogenesis\nAuthors: Chauhan, K.; Passricha, N.; Kapoor, Y.; Datta, D.; Arora, N.; Das, M.; Rao, K.; Singh, L.; Dutt, R.; Sharma, R.; Sarkar, B.; Malakkar, B.; Kalam, H.; Saini, P.; Mehra, L.; Das, P.; Ahuja, V.; Singhal, A.; Nandicoori, V.; Kumar, D.\nScore: 5.4, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579120\nMycobacterium tuberculosis (Mtb) infection of human macrophages triggers extensive dysregulation of host alternative splicing (AS). Here, we present an unexpected mechanism exploited by the Mtb to achieve the same. We identify several bacterial secretory proteins that interact with the host splicing factors (SFs) and alter select RNA splicing events in vitro, in cell and during ex vivo infections. We show bacterial proteins disrupt AS by impairing Small nuclear ribonucleoprotein (snRNP) biogenesis, primarily by interacting with and sequestering U5snRNA and SNRPF, key spliceosome constituents. These interactions are driven, in part, by one of the identified Mtb proteins, Hsr1, whose access to the host cytosol is ESX1-dependent. The Mtb{Delta}hsr1 is unable to interact and sequester U5snRNA or SNRPF and fails to alter host AS. Consequently, Mtb{Delta}hsr1 shows compromised survival and pathogenesis in macrophages and mice. Lungs of infected mice show distinct SNRPF staining, which is hsr1 dependent. Similar distinctive SNRPF staining is also noted in human tuberculous granulomas. Moreover, monocytes from tuberculosis patients show substantial splicing dysregulation, indicating splicing defects as a key determinant behind systemic tuberculosis pathology. We propose mycobacterial exploitation of the evolutionarily conserved mechanism of snRNP biogenesis, aided its adaptation for human pathogenesis and provides a unique therapeutic opportunity.\n",
  "wordCount" : "2562",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579384" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579384">
        <p class="paperTitle">Interplay of condensation and chromatin binding underlies BRD4 targeting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strom, A. R.; Eeftens, J. M.; Polyachenko, Y.; Weaver, C. J.; Watanabe, H.-F.; Bracha, D.; Orlovsky, N. D.; Jumper, C. C.; Jacobs, W. M.; Brangwynne, C. P.</p>
        <p class="info">Score: 26.8, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579384' target='https://doi.org/10.1101/2024.02.07.579384'> 10.1101/2024.02.07.579384</a></p>
        <p class="abstract">Nuclear compartments form via biomolecular phase separation, mediated through multivalent properties of biomolecules concentrated within condensates. Certain compartments are associated with specific chromatin regions, including transcriptional initiation condensates, which are composed of transcription factors and transcriptional machinery, and form at acetylated regions including enhancer and promoter loci. While protein self-interactions, especially within low-complexity and intrinsically disordered regions, are known to mediate condensation, the role of substrate-binding interactions in regulating the formation and function of biomolecular condensates is under-explored. Here, utilizing live-cell experiments in parallel with coarse-grained simulations, we investigate how chromatin interaction of the transcription factor BRD4 modulates its condensate formation. We find that both kinetic and thermodynamic properties of BRD4 condensation are affected by chromatin binding: nucleation rate is sensitive to BRD4-chromatin interactions, providing an explanation for the selective formation of BRD4 condensates at acetylated chromatin regions, and thermodynamically, multivalent acetylated chromatin sites provide a platform for BRD4 clustering below the concentration required for off-chromatin condensation. This provides a molecular and physical explanation of the relationship between nuclear condensates and epigenetically modified chromatin that results in their mutual spatiotemporal regulation, suggesting that epigenetic modulation is an important mechanism by which the cell targets transcriptional condensates to specific chromatin loci.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579772" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579772">
        <p class="paperTitle">Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.</p>
        <p class="info">Score: 22.7, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579772' target='https://doi.org/10.1101/2024.02.10.579772'> 10.1101/2024.02.10.579772</a></p>
        <p class="abstract">Human cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry. HCMV vaccine candidates containing gB have demonstrated moderate clinical efficacy, but no HCMV vaccine has been FDA-approved. Here, we used structure-based design to identify and characterize amino acid substitutions that stabilize gB in its metastable prefusion conformation. One variant containing two engineered interprotomer disulfide bonds and two cavity-filling substitutions (gB-C7), displayed increased expression and thermostability. A 2.8 [A] resolution cryo-electron microscopy structure shows that gB-C7 adopts a prefusion-like conformation, revealing additional structural elements at the membrane-distal apex. Unlike previous observations for several class I viral fusion proteins, mice immunized with postfusion or prefusion-stabilized forms of soluble gB protein displayed similar neutralizing antibody titers, here specifically against an HCMV laboratory strain on fibroblasts. Collectively, these results identify initial strategies to stabilize class III viral fusion proteins and provide tools to probe gB-directed antibody responses.

TeaserA structure-based design campaign leads to stabilization of the class III viral fusion protein from HCMV in a prefusion-like conformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579116" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579116">
        <p class="paperTitle">Introducing synthetic thermostable RNase inhibitors to single-cell RNA-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noble, J. C.; Lentini, A.; Hageman-Jensen, M.; Sandberg, R.; Reinius, B.</p>
        <p class="info">Score: 16.6, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579116' target='https://doi.org/10.1101/2024.02.06.579116'> 10.1101/2024.02.06.579116</a></p>
        <p class="abstract">Single-cell RNA-sequencing (scRNAseq) is revolutionizing biomedicine, propelled by advances in methodology, ease of use, and cost reduction of library preparation. Over the past decade, there have been remarkable technical improvements in most aspects of single-cell transcriptomics. Yet, there has been little to no progress in advancing RNase inhibition despite that maintained RNA integrity is critical during cell collection, storage, and cDNA library generation. Here, we demonstrate that a synthetic thermostable RNase inhibitor yields single-cell libraries of equal or superior quality compared to ubiquitously used protein-based recombinant RNase inhibitors (RRIs). Importantly, the synthetic RNase inhibitor provides additional unique improvements in reproducibility and throughput, enables new experimental workflows including heat cycles, and can reduce the need for dry-ice transports.

In summary, replacing RRIs represents a substantial advancement in the field of single-cell transcriptomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578923">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578923" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578923">
        <p class="paperTitle">ChAHP2 and ChAHP control diverse retrotransposons by complementary activities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578923" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578923" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahel, J.; Pandey, A.; Schwaiger, M.; Mohn, F.; Basters, A.; Kempf, G.; Andriollo, A.; Kaaij, L.; Hess, D.; Bühler, M.</p>
        <p class="info">Score: 14.4, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578923' target='https://doi.org/10.1101/2024.02.05.578923'> 10.1101/2024.02.05.578923</a></p>
        <p class="abstract">Retrotransposon control in mammals is an intricate process that is effectuated by a broad network of chromatin regulatory pathways. We previously discovered ChAHP, a protein complex with repressive activity against SINE retrotransposons, composed of the transcription factor ADNP, chromatin remodeler CHD4, and HP1 proteins. Here we identify ChAHP2, a protein complex homologous to ChAHP, wherein ADNP is replaced by ADNP2. ChAHP2 is predominantly targeted to ERVs and LINEs, via HP1{beta}-mediated binding of H3K9 trimethylated histones. We further demonstrate that ChAHP also binds these elements in a mechanistically equivalent manner to ChAHP2, and distinct from DNA sequence-specific recruitment at SINEs. Genetic ablation of ADNP2 alleviates ERV and LINE1 repression, which is synthetically exacerbated by additional depletion of ADNP. Together, our results reveal that the ChAHP and ChAHP2 complexes function to control both non-autonomous and autonomous retrotransposons by complementary activities, further adding to the complexity of mammalian transposon control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578223" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578223">
        <p class="paperTitle">Programmable RNA writing with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.</p>
        <p class="info">Score: 128.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578223' target='https://doi.org/10.1101/2024.01.31.578223'> 10.1101/2024.01.31.578223</a></p>
        <p class="abstract">RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington&#39;s disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntingtons disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577779">
        <p class="paperTitle">Rewriting endogenous human transcripts with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 84.0, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577779' target='https://doi.org/10.1101/2024.01.29.577779'> 10.1101/2024.01.29.577779</a></p>
        <p class="abstract">Splicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578660" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578660">
        <p class="paperTitle">Single-molecule chromatin configurations link transcription factor binding to expression in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doughty, B. R.; Hinks, M. M.; Schaepe, J. M.; Marinov, G. K.; Thurm, A. R.; Rios-Martinez, C.; Parks, B. E.; Tan, Y.; Marklund, E.; Dubocanin, D.; Bintu, L.; Greenleaf, W. J.</p>
        <p class="info">Score: 74.3, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578660' target='https://doi.org/10.1101/2024.02.02.578660'> 10.1101/2024.02.02.578660</a></p>
        <p class="abstract">The binding of multiple transcription factors (TFs) to genomic enhancers activates gene expression in mammalian cells. However, the molecular details that link enhancer sequence to TF binding, promoter state, and gene expression levels remain opaque. We applied single-molecule footprinting (SMF) to measure the simultaneous occupancy of TFs, nucleosomes, and components of the transcription machinery on engineered enhancer/promoter constructs with variable numbers of TF binding sites for both a synthetic and an endogenous TF. We find that activation domains enhance a TFs capacity to compete with nucleosomes for binding to DNA in a BAF-dependent manner, TF binding on nucleosome-free DNA is consistent with independent binding between TFs, and average TF occupancy linearly contributes to promoter activation rates. We also decompose TF strength into separable binding and activation terms, which can be tuned and perturbed independently. Finally, we develop thermodynamic and kinetic models that quantitatively predict both the binding microstates observed at the enhancer and subsequent time-dependent gene expression. This work provides a template for quantitative dissection of distinct contributors to gene activation, including the activity of chromatin remodelers, TF activation domains, chromatin acetylation, TF concentration, TF binding affinity, and TF binding site configuration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579110" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579110">
        <p class="paperTitle">A rapid, cost-effective, colorimetric LAMP assay (CLASS) for detecting invasive malaria vector, Anopheles stephensi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rafferty, C.; Raise, G.; Scaife, J.; Abongo, B.; Omondi, S.; Milanoi, S.; Muchoki, M.; Onyango, B.; Ochomo, E.; Zohdy, S.</p>
        <p class="info">Score: 10.1, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579110' target='https://doi.org/10.1101/2024.02.06.579110'> 10.1101/2024.02.06.579110</a></p>
        <p class="abstract">Anopheles stephensi, an invasive malaria vector in Africa, has the potential to impact the landscape of malaria on the continent, threatening to put an additional 126 million people per year at risk of malaria, largely in peri-urban/urban areas. To accelerate the early detection and rapid response to An. stephensi and ensure no gains made in malaria control and elimination are lost, it is critical to confirm the presence of the species and the geographic extent of its spread to inform control. However, morphological identification may be misinterpreted if specimens are damaged and existing molecular species confirmation assays require specialized laboratory equipment and training and may be challenging to interpret, requiring additional sequencing confirmation. A colorimetric rapid loop-mediated isothermal amplification (LAMP) assay for molecular An. stephensi species identification was developed and optimized. The colorimetric assay requires only a heat source and reagents and can be used with or without DNA extraction resulting in positive color change in 30-35 minutes. To determine analytical sensitivity, a 1:10 dilution series of the DNA extract was conducted showing 100% assay sensitivity down to 0.003 nanograms. To determine specificity, three different An. stephensi laboratory strains (STE2, SDA 500, UCI), 8 other Anopheles mosquito species, and Aedes aegypti were compared, and the results indicated 100% specificity across these species. To determine use without the need for DNA extraction, samples evaluated included a single mosquito leg, whole adult or larval mosquitoes, and pooled DNA extract from several mosquito species. A total of 1687 individual reactions were tested during optimization and all LAMP assay results were compared against the conventional PCR assay and confirmed through Sanger sequencing. To validate the optimized assay on wild caught specimens, DNA extracted from 12 wild caught, sequence-confirmed An. stephensi from Marsabit, Kenya, were tested and the colorimetric assay was accurate in identifying all of the specimens as An. stephensi. The assay described presents an opportunity to accelerate An. stephensi molecular identification in new and existing locations in Africa, within its endemic range, and globally. These findings present a simple, rapid, unique alternative to existing PCR and sequencing-based An. stephensi species identification and confirmation strategies. With additional field validation studies, molecular screening tools like the colorimetric LAMP-based An. stephensi species identification (CLASS) assay fill an important gap of rapid confirmation of this invasive vector and presents an ideal opportunity to better understand the spread of the species in Africa and other recently invaded areas, thus accelerating a response to mitigate its long-term impacts on malaria on the continent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578895" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578895">
        <p class="paperTitle">The MYCN oncoprotein and helicases DDX17 and DDX5 have opposite effects on the production of chimeric transcripts in neuroblastoma cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clerc, V.; Valat, J.; Grand, X.; Fontrodona, N.; Bazire, M.; Rama, N.; Auboeuf, D.; Gibert, B.; Mortreux, F.; Bourgeois, C. F.</p>
        <p class="info">Score: 6.6, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578895' target='https://doi.org/10.1101/2024.02.05.578895'> 10.1101/2024.02.05.578895</a></p>
        <p class="abstract">DEAD box helicases DDX17 and DDX5 control the termination of transcription and the associated cleavage of the 3 end of transcripts. Here we show that the transcriptional readthrough induced by their depletion in neuroblastoma cells also results in increased production of chimeric transcripts from tandemly oriented genes. Analysis of neuroblastoma tumours in which chimeric transcripts are abundant revealed that low expression of the DDX17 and DDX5 genes is associated with poor overall patient survival. Low DDX17 expression is also significantly associated with high-risk tumours and is inversely correlated with MYCN oncogene amplification, suggesting a link between these two factors. We demonstrate that changes in MYCN expression do not affect the expression of either helicase, but alter transcription termination leading to the production of chimeric transcripts. We provide evidence that MYCN acts on termination through its direct binding to the 3 region of genes and that it interacts with DDX17, suggesting that it may inhibit the activity of the helicase. Collectively, our work reveals a novel function of MYCN in transcription termination and suggests that the deregulation of MYCN and DDX17/DDX5 expression in neuroblastoma may lead to the expression of non-canonical and potentially harmful RNA molecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579120">
        <p class="paperTitle">A Mycobacterium tuberculosis secreted virulence factor disrupts host snRNP biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chauhan, K.; Passricha, N.; Kapoor, Y.; Datta, D.; Arora, N.; Das, M.; Rao, K.; Singh, L.; Dutt, R.; Sharma, R.; Sarkar, B.; Malakkar, B.; Kalam, H.; Saini, P.; Mehra, L.; Das, P.; Ahuja, V.; Singhal, A.; Nandicoori, V.; Kumar, D.</p>
        <p class="info">Score: 5.4, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579120' target='https://doi.org/10.1101/2024.02.06.579120'> 10.1101/2024.02.06.579120</a></p>
        <p class="abstract">Mycobacterium tuberculosis (Mtb) infection of human macrophages triggers extensive dysregulation of host alternative splicing (AS). Here, we present an unexpected mechanism exploited by the Mtb to achieve the same. We identify several bacterial secretory proteins that interact with the host splicing factors (SFs) and alter select RNA splicing events in vitro, in cell and during ex vivo infections. We show bacterial proteins disrupt AS by impairing Small nuclear ribonucleoprotein (snRNP) biogenesis, primarily by interacting with and sequestering U5snRNA and SNRPF, key spliceosome constituents. These interactions are driven, in part, by one of the identified Mtb proteins, Hsr1, whose access to the host cytosol is ESX1-dependent. The Mtb{Delta}hsr1 is unable to interact and sequester U5snRNA or SNRPF and fails to alter host AS. Consequently, Mtb{Delta}hsr1 shows compromised survival and pathogenesis in macrophages and mice. Lungs of infected mice show distinct SNRPF staining, which is hsr1 dependent. Similar distinctive SNRPF staining is also noted in human tuberculous granulomas. Moreover, monocytes from tuberculosis patients show substantial splicing dysregulation, indicating splicing defects as a key determinant behind systemic tuberculosis pathology. We propose mycobacterial exploitation of the evolutionarily conserved mechanism of snRNP biogenesis, aided its adaptation for human pathogenesis and provides a unique therapeutic opportunity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
