<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure and mechanism of Zorya anti-phage defense system
Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.
Score: 55.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097
Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Structure and mechanism of Zorya anti-phage defense system
Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.
Score: 55.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097
Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Structure and mechanism of Zorya anti-phage defense system
Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.
Score: 55.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097
Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Structure and mechanism of Zorya anti-phage defense system\nAuthors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.\nScore: 55.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097\nZorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion.",
  "keywords": [
    
  ],
  "articleBody": " Structure and mechanism of Zorya anti-phage defense system\nAuthors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.\nScore: 55.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097\nZorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion. It consists of a predicted membrane-embedded complex (ZorAB) and soluble components that differ among Zorya subtypes, notably ZorC and ZorD, in type I Zorya systems. Here, we reveal the molecular basis of the Zorya defense system using cryo-electron microscopy, mutagenesis, fluorescence microscopy, proteomics, and functional studies. We demonstrate that ZorAB shares the stoichiometry of other 5:2 inner membrane ion-driven rotary motors. Additionally, ZorA5B2 features a dimeric ZorB peptidoglycan binding domain and a pentameric alpha-helical coiled-coil tail made of ZorA that projects approximately 700 angstrom into the cytoplasm. We further characterize the structure and function of the soluble Zorya components, ZorC and ZorD, and find that they harbour DNA binding and nuclease activity, respectively. Comprehensive functional and mutational analyses demonstrates that all Zorya components work in concert to protect bacterial cells against invading phages. We present evidence that ZorAB operates as an ion-driven motor that becomes activated and anchors to the cell wall upon sensing of cell envelope perturbations during phage invasion. Subsequently, ZorAB transfers the phage invasion signal through the ZorA cytoplasmic tail to the soluble effectors, which function to prevent phage propagation. In summary, our study elucidates the foundational mechanisms of Zorya function and reveals a novel triggering signal for the rapid activation of an anti-phage defense system.\nBarcode 100K Specimens: In a Single Nanopore Run\nAuthors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.\nScore: 117.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282\nIt is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.\nTranscriptional memory is conferred by combined heritable maintenance and local removal of selective chromatin modifications\nAuthors: Mikulski, P.; Tehrani, S. S. H.; Kogan, A.; Abdul-Zani, I.; Shell, E.; Ryan, B. J.; Jansen, L. E. T.\nScore: 30.7, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571619\nInterferon-{gamma} (IFN{gamma}) transiently activates genes involved in inflammation and innate immunity. A subset of targets maintain a mitotically heritable memory of prior IFN{gamma} exposure resulting in hyperactivation upon reexposure. Here we discovered that the active chromatin marks H3K4me1, H3K14Ac and H4K16Ac are established during IFN{gamma} priming and selectively maintained on a cluster of GBP genes for at least 7 days in dividing cells in the absence of transcription. The histone acetyltransferase KAT7 is required for the accelerated GBP reactivation upon reexposure to IFN{gamma}. In naive cells, we find the GBP cluster is maintained in low-level repressive chromatin marked by H3K27me3 limiting priming in a PRC2-dependent manner. Unexpectedly, IFN{gamma} results in transient accumulation of this repressive mark but is then selectively depleted from primed GBP genes during the memory phase facilitating hyperactivation of primed cells. Furthermore, we identified a cis-regulatory element that makes transient, long-range contacts across the GBP cluster and acts as a repressor, primarily to curb the hyperactivation of previously IFN{gamma}-primed cells. Combined our results identify the putative chromatin basis for long-term transcriptional memory of interferon signalling that may contribute to enhanced innate immunity.\nHigh order expression dependencies finely resolve cryptic states and subtypes in single cell data\nAuthors: Jansma, A.; Yao, Y.; Wolfe, J.; Del Debbio, L.; Beentjes, S. V.; Ponting, C. P.; Khamseh, A.\nScore: 18.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572232\nSingle cells are typically typed by clustering in reduced dimensional transcriptome space. Here we introduce Stator, a novel method, workflow and app that reveals cell types, subtypes and states without relying on local proximity of cells in gene expression space. Rather, Stator derives higher-order gene expression dependencies from a sparse gene-by-cell expression matrix. From these dependencies the method multiply labels the same single cell according to type, sub-type and state (activation, differentiation or cell cycle sub-phase). By applying the method to data from mouse embryonic brain, and human healthy or diseased liver, we show how Stator first recapitulates other methods' cell type labels, and then reveals combinatorial gene expression markers of cell type, state, and disease at higher resolution. By allowing multiple state labels for single cells we reveal cell type fates of embryonic progenitor cells and liver cancer states associated with patient survival.\nRapid DNA unwinding accelerates genome editing by engineered CRISPR-Cas9\nAuthors: Eggers, A. R.; Chen, K.; Soczek, K. M.; Tuck, O. T.; Doherty, E. E.; Thornton, B. W.; Xu, B.; Trinidad, M. I.; Doudna, J. A.\nScore: 18.8, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571777\nThermostable CRISPR-Cas9 enzymes could improve genome editing efficiency and delivery due to extended protein lifetimes. However, initial experimentation demonstrated Geobacillus stearothermophilus Cas9 (GeoCas9) to be virtually inactive when used in cultured human cells. Laboratory-evolved variants of GeoCas9 overcome this natural limitation by acquiring mutations in the wedge (WED) domain that produce \u003e100-fold higher genome editing levels. Cryo-EM structures of the wildtype and improved GeoCas9 (iGeoCas9) enzymes reveal extended contacts between the WED domain of iGeoCas9 and DNA substrates. Biochemical analysis shows that iGeoCas9 accelerates DNA unwinding to capture substrates under the magnesium-restricted conditions typical of mammalian but not bacterial cells. These findings enabled rational engineering of other Cas9 orthologs to enhance genome editing levels, pointing to a general strategy for editing enzyme improvement. Together, these results uncover a new role for the Cas9 WED domain in DNA unwinding and demonstrate how accelerated target unwinding dramatically improves Cas9-induced genome editing activity.\nA proteolytic AAA+ machine poised to unfold a protein substrate\nAuthors: Ghanbarpour, A.; Sauer, R. T.; Davis, J. H.\nScore: 18.6, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571662\nAAA+ proteolytic machines unfold proteins prior to degradation. Cryo-EM of a ClpXP-substrate complex reveals a postulated but heretofore unseen intermediate in substrate unfolding/degradation. The natively folded substrate is drawn tightly against the ClpX channel by interactions between axial pore loops and the substrate degron tail, and by contacts with the native substrate that are, in part, enabled by movement of one ClpX subunit out of the typically observed hexameric spiral.\nRAPIDASH: A tag-free enrichment of ribosome-associated proteins reveals compositional dynamics in embryonic tissues and stimulated macrophages\nAuthors: Susanto, T. T.; Hung, V.; Levine, A. G.; Kerr, C. H.; Yoo, Y.; Chen, Y.; Oses-Prieto, J. A.; Fromm, L.; Fujii, K.; Wernig, M.; Burlingame, A.; Ruggero, D.; Barna, M.\nScore: 11.4, Published: 2023-12-07 DOI: 10.1101/2023.12.07.570613\nRibosomes are emerging as direct regulators of gene expression, with ribosome-associated proteins (RAPs) allowing ribosomes to modulate translational control. However, a lack of technologies to enrich RAPs across many sample types has prevented systematic analysis of RAP number, dynamics, and functions. Here, we have developed a label-free methodology called RAPIDASH to enrich ribosomes and RAPs from any sample. We applied RAPIDASH to mouse embryonic tissues and identified hundreds of potential RAPs, including DHX30 and LLPH, two forebrain RAPs important for neurodevelopment. We identified a critical role of LLPH in neural development that is linked to the translation of genes with long coding sequences. Finally, we characterized ribosome composition remodeling during immune activation and observed extensive changes post-stimulation. RAPIDASH has therefore enabled the discovery of RAPs ranging from those with neuroregulatory functions to those activated by immune stimuli, thereby providing critical insights into how ribosomes are remodeled.\nThe SMC5/6 complex is required for maintenance of genome integrity upon APOBEC3A-mediated replication stress\nAuthors: O'Leary, D. R.; Hansen, A. R.; Fingerman, D. F.; Tran, T. A.; Harris, B. R.; Hayer, K. E.; DeWeerd, R. A.; Tennekoon, M.; Meroni, A.; Szeto, J.; Weitzman, M. D.; Bednarski, J. J.; Shalem, O.; Vindigni, A.; Green, A. M.\nScore: 11.4, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568952\nMutational patterns consistent with the activity of the APOBEC3 cytidine deaminases are evident in more than half of human cancer genomes. APOBEC3-mediated mutagenesis is genotoxic when uncontrolled due to accumulation of base mutations, replication stress, and DNA breaks. In particular, the APOBEC3A family member is a potent enzyme with nuclear localization that causes substantial DNA damage in experimental systems and human tumors. However, the spectrum of genome-protective mechanisms that ensure genome stability in cells with active APOBEC3A is unknown. Through a genome-wide functional screen, we identify the Structural Maintenance of Chromosomes 5/6 (SMC5/6) complex as essential for cell viability when APOBEC3A is expressed. Cells depleted of SMC5/6 incurred substantial DNA damage when APOBEC3A was active, as reflected by increased DNA breaks, DNA damage signaling, and defective proliferation. We observed an absence of APOBEC3A mutagenesis in human tumors with dysfunction of SMC5/6, consistent with synthetic lethality. APOBEC3A is known to act on ssDNA at replication forks. We observed increased DNA damage in replicating cells in the absence of SMC5/6, suggestive of replication forks as a source of DNA breaks. We interrogated replication fork dynamics by DNA fiber spreading and found a consistent increase in the length of replication tracks upon APOBEC3A activity across multiple cell lines. Increased replication fork length was dependent on Primpol, consistent with a repriming mechanism downstream of APOBEC3A-induced lesions. Loss of SMC5/6 resulted in abrogation of fork elongation in cells with active APOBEC3A, along with increased DNA breaks. Our findings indicate that increased length of replication forks in response to APOBEC3A is a genome-protective response and is dependent on intact SMC5/6. Therefore, SMC5/6 may be a therapeutic vulnerability in tumors in which APOBEC3A is active.\nRluA is the major mRNA pseudouridine synthase in Escherichia coli\nAuthors: Schaening Burgos, C.; Li, G.-W.; Gilbert, W. V.\nScore: 9.6, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570740\nPseudouridine ({Psi}) is an ubiquitous RNA modification, present in the tRNAs and rRNAs of species across all domains of life. Conserved pseudouridine synthases modify the mRNAs of diverse eukaryotes, but the modification has yet to be identified in bacterial mRNAs. Here, we report the discovery of pseudouridines in mRNA from E. coli. By testing the mRNA modification capacity of all 11 known pseudouridine synthases, we identify RluA as the predominant mRNA-modifying enzyme. RluA, a known tRNA and 23S rRNA pseudouridine synthase, modifies at least 31 of the 44 high-confidence sites we identified in E. coli mRNAs. Using RNA structure probing data to inform secondary structures, we show that the target sites of RluA occur in a common sequence and structural motif comprised of a {Psi}URAA sequence located in the loop of a short hairpin. This recognition element is shared with previously identified target sites of RluA in tRNAs and rRNA. Overall, our work identifies pseudouridine in key mRNAs and suggests the capacity of {Psi} to regulate the transcripts that contain it. Author SummaryWhile RNAs are initially transcribed using only the nucleotides A, G, C and U, these can be enzymatically modified into many different post-transcriptional modifications. tRNAs and rRNAs across all domains of life are modified with pseudouridine, an isoform of uridine that is inserted by highly conserved enzymes. In many eukaryotes, it has been demonstrated that some of these enzymes can also insert pseudouridines in mRNA, where they are poised to impact gene expression through their effects on the transcript. Here we show that protein-coding transcripts in E. coli are also modified with pseudouridine, indicating that mRNA pseudouridylation is also a highly conserved activity. RluA is the main mRNA pseudouridine synthase, introducing the modification into the transcripts of dozens of protein coding genes with high specificity. Its target sites are defined by a combined sequence and secondary structure motif. Two additional enzymes, RluC and RluD, introduce a few additional sites. All three of these enzymes belong to the same sub-family of pseudouridine synthases, and homologs of these also modify mRNAs in humans. Thus, mRNA modification by these enzymes might be a conserved activity with the capacity to impact gene regulation.\nH7 influenza A viruses bind sialyl-LewisX, a potential intermediate receptor between species\nAuthors: Spruit, C.; Palme, D. I.; Li, T.; Rios Carrasco, M.; Gabarroca Garcia, A.; Sweet, I.; Kuryshko, M.; Maliepaard, J. C. L.; Reiding, K. R.; Scheibner, D.; Boons, G.-J.; Abdelwhab, E. M.; de Vries, R. P.\nScore: 6.0, Published: 2023-12-17 DOI: 10.1101/2023.12.15.571923\nInfluenza A viruses (IAVs) can overcome species barriers by adaptation of the receptor binding site of the hemagglutinin (HA). To initiate infection, HAs bind to glycan receptors with terminal sialic acids, which are either N-acetylneuraminic acid (NeuAc) or N-glycolylneuraminic acid (NeuGc), the latter is mainly found in horses and pigs but not in birds and humans. We investigated the influence of previously identified equine NeuGc-adapting mutations (S128T, I130V, A135E, T189A, and K193R) in avian H7 IAVs in vitro and in vivo. We observed that these mutations negatively affected viral replication in chicken cells, but not in duck cells, and positively affected replication in horse cells. In vivo, the mutations reduced virus virulence and mortality in chickens. Ducks excreted high viral loads for a longer time than chickens, although they appeared clinically healthy. To elucidate why chickens and ducks were infected by these viruses despite the absence of NeuGc, we re-evaluated the receptor binding of H7 HAs using glycan microarray and flow cytometry studies. This revealed that mutated avian H7 HAs also bound to 2,3-linked NeuAc and sialyl-LewisX, which have an additional fucose moiety in their terminal epitope, explaining why infection of ducks and chickens was possible. Interestingly, the 2,3-linked NeuAc and sialyl-LewisX epitopes were only bound when presented on tri-antennary N-glycans, emphasizing the importance of investigating the fine receptor specificities of IAVs. In conclusion, the binding of NeuGc-adapted H7 IAV to sialyl-LewisX enables viral replication and shedding by chickens and ducks, potentially facilitating interspecies transmission of equine-adapted H7 IAVs.\n",
  "wordCount" : "2511",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572097">
        <p class="paperTitle">Structure and mechanism of Zorya anti-phage defense system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.</p>
        <p class="info">Score: 55.5, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572097' target='https://doi.org/10.1101/2023.12.18.572097'> 10.1101/2023.12.18.572097</a></p>
        <p class="abstract">Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion. It consists of a predicted membrane-embedded complex (ZorAB) and soluble components that differ among Zorya subtypes, notably ZorC and ZorD, in type I Zorya systems. Here, we reveal the molecular basis of the Zorya defense system using cryo-electron microscopy, mutagenesis, fluorescence microscopy, proteomics, and functional studies. We demonstrate that ZorAB shares the stoichiometry of other 5:2 inner membrane ion-driven rotary motors. Additionally, ZorA5B2 features a dimeric ZorB peptidoglycan binding domain and a pentameric alpha-helical coiled-coil tail made of ZorA that projects approximately 700 angstrom into the cytoplasm. We further characterize the structure and function of the soluble Zorya components, ZorC and ZorD, and find that they harbour DNA binding and nuclease activity, respectively. Comprehensive functional and mutational analyses demonstrates that all Zorya components work in concert to protect bacterial cells against invading phages. We present evidence that ZorAB operates as an ion-driven motor that becomes activated and anchors to the cell wall upon sensing of cell envelope perturbations during phage invasion. Subsequently, ZorAB transfers the phage invasion signal through the ZorA cytoplasmic tail to the soluble effectors, which function to prevent phage propagation. In summary, our study elucidates the foundational mechanisms of Zorya function and reveals a novel triggering signal for the rapid activation of an anti-phage defense system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569282">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569282" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569282">
        <p class="paperTitle">Barcode 100K Specimens: In a Single Nanopore Run</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569282" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569282" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.</p>
        <p class="info">Score: 117.3, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569282' target='https://doi.org/10.1101/2023.11.29.569282'> 10.1101/2023.11.29.569282</a></p>
        <p class="abstract">It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571619" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571619">
        <p class="paperTitle">Transcriptional memory is conferred by combined heritable maintenance and local removal of selective chromatin modifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mikulski, P.; Tehrani, S. S. H.; Kogan, A.; Abdul-Zani, I.; Shell, E.; Ryan, B. J.; Jansen, L. E. T.</p>
        <p class="info">Score: 30.7, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571619' target='https://doi.org/10.1101/2023.12.16.571619'> 10.1101/2023.12.16.571619</a></p>
        <p class="abstract">Interferon-{gamma} (IFN{gamma}) transiently activates genes involved in inflammation and innate immunity. A subset of targets maintain a mitotically heritable memory of prior IFN{gamma} exposure resulting in hyperactivation upon reexposure. Here we discovered that the active chromatin marks H3K4me1, H3K14Ac and H4K16Ac are established during IFN{gamma} priming and selectively maintained on a cluster of GBP genes for at least 7 days in dividing cells in the absence of transcription. The histone acetyltransferase KAT7 is required for the accelerated GBP reactivation upon reexposure to IFN{gamma}. In naive cells, we find the GBP cluster is maintained in low-level repressive chromatin marked by H3K27me3 limiting priming in a PRC2-dependent manner. Unexpectedly, IFN{gamma} results in transient accumulation of this repressive mark but is then selectively depleted from primed GBP genes during the memory phase facilitating hyperactivation of primed cells. Furthermore, we identified a cis-regulatory element that makes transient, long-range contacts across the GBP cluster and acts as a repressor, primarily to curb the hyperactivation of previously IFN{gamma}-primed cells. Combined our results identify the putative chromatin basis for long-term transcriptional memory of interferon signalling that may contribute to enhanced innate immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572232">
        <p class="paperTitle">High order expression dependencies finely resolve cryptic states and subtypes in single cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansma, A.; Yao, Y.; Wolfe, J.; Del Debbio, L.; Beentjes, S. V.; Ponting, C. P.; Khamseh, A.</p>
        <p class="info">Score: 18.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572232' target='https://doi.org/10.1101/2023.12.18.572232'> 10.1101/2023.12.18.572232</a></p>
        <p class="abstract">Single cells are typically typed by clustering in reduced dimensional transcriptome space. Here we introduce Stator, a novel method, workflow and app that reveals cell types, subtypes and states without relying on local proximity of cells in gene expression space. Rather, Stator derives higher-order gene expression dependencies from a sparse gene-by-cell expression matrix. From these dependencies the method multiply labels the same single cell according to type, sub-type and state (activation, differentiation or cell cycle sub-phase). By applying the method to data from mouse embryonic brain, and human healthy or diseased liver, we show how Stator first recapitulates other methods&#39; cell type labels, and then reveals combinatorial gene expression markers of cell type, state, and disease at higher resolution. By allowing multiple state labels for single cells we reveal cell type fates of embryonic progenitor cells and liver cancer states associated with patient survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571777">
        <p class="paperTitle">Rapid DNA unwinding accelerates genome editing by engineered CRISPR-Cas9</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eggers, A. R.; Chen, K.; Soczek, K. M.; Tuck, O. T.; Doherty, E. E.; Thornton, B. W.; Xu, B.; Trinidad, M. I.; Doudna, J. A.</p>
        <p class="info">Score: 18.8, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571777' target='https://doi.org/10.1101/2023.12.14.571777'> 10.1101/2023.12.14.571777</a></p>
        <p class="abstract">Thermostable CRISPR-Cas9 enzymes could improve genome editing efficiency and delivery due to extended protein lifetimes. However, initial experimentation demonstrated Geobacillus stearothermophilus Cas9 (GeoCas9) to be virtually inactive when used in cultured human cells. Laboratory-evolved variants of GeoCas9 overcome this natural limitation by acquiring mutations in the wedge (WED) domain that produce &gt;100-fold higher genome editing levels. Cryo-EM structures of the wildtype and improved GeoCas9 (iGeoCas9) enzymes reveal extended contacts between the WED domain of iGeoCas9 and DNA substrates. Biochemical analysis shows that iGeoCas9 accelerates DNA unwinding to capture substrates under the magnesium-restricted conditions typical of mammalian but not bacterial cells. These findings enabled rational engineering of other Cas9 orthologs to enhance genome editing levels, pointing to a general strategy for editing enzyme improvement. Together, these results uncover a new role for the Cas9 WED domain in DNA unwinding and demonstrate how accelerated target unwinding dramatically improves Cas9-induced genome editing activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571662">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571662" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571662">
        <p class="paperTitle">A proteolytic AAA&#43; machine poised to unfold a protein substrate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571662" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571662" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghanbarpour, A.; Sauer, R. T.; Davis, J. H.</p>
        <p class="info">Score: 18.6, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571662' target='https://doi.org/10.1101/2023.12.14.571662'> 10.1101/2023.12.14.571662</a></p>
        <p class="abstract">AAA&#43; proteolytic machines unfold proteins prior to degradation. Cryo-EM of a ClpXP-substrate complex reveals a postulated but heretofore unseen intermediate in substrate unfolding/degradation. The natively folded substrate is drawn tightly against the ClpX channel by interactions between axial pore loops and the substrate degron tail, and by contacts with the native substrate that are, in part, enabled by movement of one ClpX subunit out of the typically observed hexameric spiral.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570613">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570613" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570613">
        <p class="paperTitle">RAPIDASH: A tag-free enrichment of ribosome-associated proteins reveals compositional dynamics in embryonic tissues and stimulated macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570613" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570613" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Susanto, T. T.; Hung, V.; Levine, A. G.; Kerr, C. H.; Yoo, Y.; Chen, Y.; Oses-Prieto, J. A.; Fromm, L.; Fujii, K.; Wernig, M.; Burlingame, A.; Ruggero, D.; Barna, M.</p>
        <p class="info">Score: 11.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570613' target='https://doi.org/10.1101/2023.12.07.570613'> 10.1101/2023.12.07.570613</a></p>
        <p class="abstract">Ribosomes are emerging as direct regulators of gene expression, with ribosome-associated proteins (RAPs) allowing ribosomes to modulate translational control. However, a lack of technologies to enrich RAPs across many sample types has prevented systematic analysis of RAP number, dynamics, and functions. Here, we have developed a label-free methodology called RAPIDASH to enrich ribosomes and RAPs from any sample. We applied RAPIDASH to mouse embryonic tissues and identified hundreds of potential RAPs, including DHX30 and LLPH, two forebrain RAPs important for neurodevelopment. We identified a critical role of LLPH in neural development that is linked to the translation of genes with long coding sequences. Finally, we characterized ribosome composition remodeling during immune activation and observed extensive changes post-stimulation. RAPIDASH has therefore enabled the discovery of RAPs ranging from those with neuroregulatory functions to those activated by immune stimuli, thereby providing critical insights into how ribosomes are remodeled.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568952">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568952" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568952">
        <p class="paperTitle">The SMC5/6 complex is required for maintenance of genome integrity upon APOBEC3A-mediated replication stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568952" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568952" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Leary, D. R.; Hansen, A. R.; Fingerman, D. F.; Tran, T. A.; Harris, B. R.; Hayer, K. E.; DeWeerd, R. A.; Tennekoon, M.; Meroni, A.; Szeto, J.; Weitzman, M. D.; Bednarski, J. J.; Shalem, O.; Vindigni, A.; Green, A. M.</p>
        <p class="info">Score: 11.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568952' target='https://doi.org/10.1101/2023.11.28.568952'> 10.1101/2023.11.28.568952</a></p>
        <p class="abstract">Mutational patterns consistent with the activity of the APOBEC3 cytidine deaminases are evident in more than half of human cancer genomes. APOBEC3-mediated mutagenesis is genotoxic when uncontrolled due to accumulation of base mutations, replication stress, and DNA breaks. In particular, the APOBEC3A family member is a potent enzyme with nuclear localization that causes substantial DNA damage in experimental systems and human tumors. However, the spectrum of genome-protective mechanisms that ensure genome stability in cells with active APOBEC3A is unknown. Through a genome-wide functional screen, we identify the Structural Maintenance of Chromosomes 5/6 (SMC5/6) complex as essential for cell viability when APOBEC3A is expressed. Cells depleted of SMC5/6 incurred substantial DNA damage when APOBEC3A was active, as reflected by increased DNA breaks, DNA damage signaling, and defective proliferation. We observed an absence of APOBEC3A mutagenesis in human tumors with dysfunction of SMC5/6, consistent with synthetic lethality. APOBEC3A is known to act on ssDNA at replication forks. We observed increased DNA damage in replicating cells in the absence of SMC5/6, suggestive of replication forks as a source of DNA breaks. We interrogated replication fork dynamics by DNA fiber spreading and found a consistent increase in the length of replication tracks upon APOBEC3A activity across multiple cell lines. Increased replication fork length was dependent on Primpol, consistent with a repriming mechanism downstream of APOBEC3A-induced lesions. Loss of SMC5/6 resulted in abrogation of fork elongation in cells with active APOBEC3A, along with increased DNA breaks. Our findings indicate that increased length of replication forks in response to APOBEC3A is a genome-protective response and is dependent on intact SMC5/6. Therefore, SMC5/6 may be a therapeutic vulnerability in tumors in which APOBEC3A is active.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570740">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570740" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570740">
        <p class="paperTitle">RluA is the major mRNA pseudouridine synthase in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570740" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570740" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schaening Burgos, C.; Li, G.-W.; Gilbert, W. V.</p>
        <p class="info">Score: 9.6, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570740' target='https://doi.org/10.1101/2023.12.08.570740'> 10.1101/2023.12.08.570740</a></p>
        <p class="abstract">Pseudouridine ({Psi}) is an ubiquitous RNA modification, present in the tRNAs and rRNAs of species across all domains of life. Conserved pseudouridine synthases modify the mRNAs of diverse eukaryotes, but the modification has yet to be identified in bacterial mRNAs. Here, we report the discovery of pseudouridines in mRNA from E. coli. By testing the mRNA modification capacity of all 11 known pseudouridine synthases, we identify RluA as the predominant mRNA-modifying enzyme. RluA, a known tRNA and 23S rRNA pseudouridine synthase, modifies at least 31 of the 44 high-confidence sites we identified in E. coli mRNAs. Using RNA structure probing data to inform secondary structures, we show that the target sites of RluA occur in a common sequence and structural motif comprised of a {Psi}URAA sequence located in the loop of a short hairpin. This recognition element is shared with previously identified target sites of RluA in tRNAs and rRNA. Overall, our work identifies pseudouridine in key mRNAs and suggests the capacity of {Psi} to regulate the transcripts that contain it.

Author SummaryWhile RNAs are initially transcribed using only the nucleotides A, G, C and U, these can be enzymatically modified into many different post-transcriptional modifications. tRNAs and rRNAs across all domains of life are modified with pseudouridine, an isoform of uridine that is inserted by highly conserved enzymes. In many eukaryotes, it has been demonstrated that some of these enzymes can also insert pseudouridines in mRNA, where they are poised to impact gene expression through their effects on the transcript. Here we show that protein-coding transcripts in E. coli are also modified with pseudouridine, indicating that mRNA pseudouridylation is also a highly conserved activity. RluA is the main mRNA pseudouridine synthase, introducing the modification into the transcripts of dozens of protein coding genes with high specificity. Its target sites are defined by a combined sequence and secondary structure motif. Two additional enzymes, RluC and RluD, introduce a few additional sites. All three of these enzymes belong to the same sub-family of pseudouridine synthases, and homologs of these also modify mRNAs in humans. Thus, mRNA modification by these enzymes might be a conserved activity with the capacity to impact gene regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571923">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571923" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571923">
        <p class="paperTitle">H7 influenza A viruses bind sialyl-LewisX, a potential intermediate receptor between species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571923" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571923" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spruit, C.; Palme, D. I.; Li, T.; Rios Carrasco, M.; Gabarroca Garcia, A.; Sweet, I.; Kuryshko, M.; Maliepaard, J. C. L.; Reiding, K. R.; Scheibner, D.; Boons, G.-J.; Abdelwhab, E. M.; de Vries, R. P.</p>
        <p class="info">Score: 6.0, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571923' target='https://doi.org/10.1101/2023.12.15.571923'> 10.1101/2023.12.15.571923</a></p>
        <p class="abstract">Influenza A viruses (IAVs) can overcome species barriers by adaptation of the receptor binding site of the hemagglutinin (HA). To initiate infection, HAs bind to glycan receptors with terminal sialic acids, which are either N-acetylneuraminic acid (NeuAc) or N-glycolylneuraminic acid (NeuGc), the latter is mainly found in horses and pigs but not in birds and humans. We investigated the influence of previously identified equine NeuGc-adapting mutations (S128T, I130V, A135E, T189A, and K193R) in avian H7 IAVs in vitro and in vivo. We observed that these mutations negatively affected viral replication in chicken cells, but not in duck cells, and positively affected replication in horse cells. In vivo, the mutations reduced virus virulence and mortality in chickens. Ducks excreted high viral loads for a longer time than chickens, although they appeared clinically healthy. To elucidate why chickens and ducks were infected by these viruses despite the absence of NeuGc, we re-evaluated the receptor binding of H7 HAs using glycan microarray and flow cytometry studies. This revealed that mutated avian H7 HAs also bound to 2,3-linked NeuAc and sialyl-LewisX, which have an additional fucose moiety in their terminal epitope, explaining why infection of ducks and chickens was possible. Interestingly, the 2,3-linked NeuAc and sialyl-LewisX epitopes were only bound when presented on tri-antennary N-glycans, emphasizing the importance of investigating the fine receptor specificities of IAVs. In conclusion, the binding of NeuGc-adapted H7 IAV to sialyl-LewisX enables viral replication and shedding by chickens and ducks, potentially facilitating interspecies transmission of equine-adapted H7 IAVs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
