<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing
Authors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.
Score: 17.6, Published: 2023-12-31 DOI: 10.1101/2023.12.28.573526
The Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing
Authors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.
Score: 17.6, Published: 2023-12-31 DOI: 10.1101/2023.12.28.573526
The Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing
Authors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.
Score: 17.6, Published: 2023-12-31 DOI: 10.1101/2023.12.28.573526
The Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing\nAuthors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.\nScore: 17.6, Published: 2023-12-31 DOI: 10.1101/2023.12.28.573526\nThe Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood.",
  "keywords": [
    
  ],
  "articleBody": " Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing\nAuthors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.\nScore: 17.6, Published: 2023-12-31 DOI: 10.1101/2023.12.28.573526\nThe Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood. In this study, we have identified a second HuSH complex, termed HuSH2, centered around TASOR2, a paralog of the core TASOR protein in HuSH. Our findings indicate that every subunit in both HuSH and HuSH2 has an important role in achieving precise genomic localization to distinct, non-overlapping genomic loci. We utilized in silico protein structure prediction to simulate the interactions between MPP8 and both TASOR paralogs. Drawing on the insights gained from these predictions, we implemented amino acid substitutions that interfered with the binding of MPP8 to each HuSH complex. Leveraging these MPP8 transgenes and other constructs, we identified an important role played by the relative quantities of HuSH complexes in controlling the activity of LINE-1 elements. Furthermore, our results suggest that dynamic changes in TASOR and TASOR2 expression enable cells to finely tune the extent of HuSH-mediated silencing. Our study provides insights into the intricate interplay between HuSH complexes, illuminating their important role in the regulation of retrotransposon silencing. Key PointsO_LIThe identification of a previously unknown HuSH2 complex, with TASOR2 as its central component. C_LIO_LIHuSH and HuSH2 complexes exhibit unique genomic localization patterns within the human genome. C_LIO_LIDisruption of the delicate balance between the two HuSH complexes results in the desilencing of LINE-1 and other transposable elements. C_LIO_LITASOR and TASOR2 engage in a competitive interaction for the HuSH subunit MPP8. C_LIO_LIThe localization of MPP8 to either HuSH or HuSH2 sites is intricately regulated by its interaction with TASOR and TASOR2. C_LI\nShieldin and CST co-orchestrate DNA polymerase-dependent tailed-end joining reactions independently of 53BP1-governed repair pathway choice\nAuthors: King, A.; Reichl, P.; Metson, J. S.; Parker, R.; Munro, D.; Oliveira, C.; Becker, J. R.; Biggs, D.; Preece, C.; Davies, B.; Chapman, J. R.\nScore: 17.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572534\n53BP1 regulates DNA end-joining in lymphocytes, diversifying immune antigen receptors. This involves nucleosome-bound 53BP1 at DNA double-stranded breaks (DSBs) recruiting RIF1 and shieldin, a poorly understood DNA-binding complex. The 53BP1-RIF1-shieldin axis is pathological in BRCA1-mutated cancers, blocking homologous recombination (HR) and driving illegitimate non-homologous end-joining (NHEJ). However, how this axis regulates DNA end-joining and HR suppression remains unresolved. We investigated shieldin and its interplay with CST, a complex recently implicated in 53BP1-dependent activities. Immunophenotypically, mice lacking shieldin or CST are equivalent, with class-switch recombination co-reliant on both complexes. ATM-dependent DNA damage signalling underpins this cooperation, inducing physical interactions between these complexes that reveal shieldin as a DSB-responsive CST adaptor. Furthermore, DNA polymerase {zeta} functions downstream of shieldin, establishing DNA fill-in synthesis as the physiological function of shieldin-CST. Lastly, 53BP1 suppresses HR and promotes NHEJ in BRCA1-deficient mice and cells independently of shieldin. These findings showcase the resilience of the 53BP1 pathway, achieved through the collaboration of chromatin-bound 53BP1 complexes and DNA end-processing effector proteins.\nStructure of the hepatitis C virus E1/E2 envelope proteins in a homodimeric complex\nAuthors: Augestad, E.; Olesen, C. H.; Groenberg, C.; Soerensen, A.; Velazquez-Moctezuma, R.; Fanalista, M.; Bukh, J.; Wang, K.; Gourdon, P.; Prentoe, J.\nScore: 10.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573427\nWorldwide, 58 million individuals suffer from chronic hepatitis C virus (HCV) infection, a primary driver of liver cancer. The HCV envelope proteins, E1 and E2, form a heterodimer, which is the target for neutralizing antibodies. Despite high-resolution structural models of partial heterodimer elements, the structural landscape of higher-order E1/E2 oligomers remains unexplored. We determined a ~3.5 [A] cryo-electron microscopy structure of membrane-extracted HCV E1/E2 in a homodimeric arrangement. This structure includes detailed information on the homodimer interface, the E2-binding pocket for hypervariable region 1, antigenic site 412 conformation, and the organization of the E1/E2 transmembrane regions, including one internal to E1. This higher-order E1/E2 assembly could play a pivotal role in the design of novel vaccine antigens better mimicking E1/E2 complexes on the HCV particle.\nGenetic activation of canonical RNA interference in mice\nAuthors: Buccheri, V.; Pasulka, J.; Malik, R.; Loubalova, Z.; Taborska, E.; Horvat, F.; Kulmann, M. I. R.; Jenickova, I.; Prochazka, J.; Sedlacek, R.; Svoboda, P.\nScore: 9.3, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573630\nCanonical RNA interference (RNAi) is sequence-specific mRNA degradation guided by small interfering RNAs (siRNAs) made from double-stranded RNA (dsRNA) by RNase III Dicer. RNAi has different roles including gene regulation, antiviral immunity or defense against transposable elements. In mammals, RNAi is constrained by Dicer, which is adapted to produce microRNAs, another class of small RNAs. However, RNAi exists in mouse oocytes, which employs a truncated Dicer variant. A homozygous mutation to express only the truncated variant ({Delta}HEL1) causes dysregulation of microRNAs and perinatal lethality in mice. Here, we report the phenotype and RNAi activity in Dicer{Delta}HEL1/wtmice, which are viable, show minimal miRNome changes but their endogenous siRNA levels are increased by an order of magnitude. We show that siRNA abundance is limited by available dsRNA but not by PKR, a dsRNA sensor of innate immunity. Expressing dsRNA from a transgene, functional RNAi in vivo was induced in heart. Dicer{Delta}HEL1/wt mice thus represent a new model for researching mammalian canonical RNAi in vivo and offer an unprecedented platform for addressing claims about its biological roles.\nUnraveling the mechanism of centromere inactivation in human aging\nAuthors: Sikder, S.; Baek, S.; McNeil, T.; Dalal, Y.\nScore: 8.9, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573721\nAging involves a range of genetic, epigenetic, and physiological alterations. A key characteristic of aged cells is the loss of global heterochromatin, accompanied by a reduction in canonical histone levels. In this study, we track the fate of centromeres during aging in human cells. Our findings reveal that the centromeric histone H3 variant CENP-A is downregulated in aged cells, in a p53-dependent manner. We observe repression of centromeric noncoding transcription through an epigenetic mechanism via recruitment of a lysine-specific demethylase 1 (LSD1/KDM1A) to centromeres. This suppression results in defective de novo CENP-A loading at aging centromeres. By dual inhibition of p53 and LSD1/KDM1A in aged cells, we mitigate the reduction in centromeric proteins and centromeric transcripts, leading to mitotic rejuvenation of these cells. These results offer insights into a novel mechanism for centromeric inactivation during aging and provide potential strategies to reactivate centromeres.\nModulation of prion protein expression through cryptic splice site manipulation\nAuthors: Gentile, J. E.; Corridon, T. L.; Mortberg, M. A.; D'Souza, E. N.; Whiffin, N.; Minikel, E. V.; Vallabh, S. M.\nScore: 15.2, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572439\nLowering expression of prion protein (PrP) is a well-validated therapeutic strategy in prion disease, but additional modalities are urgently needed. In other diseases, small molecules have proven capable of modulating pre-mRNA splicing, sometimes by forcing inclusion of cryptic exons that reduce gene expression. Here, we characterize a cryptic exon located in human PRNPs sole intron and evaluate its potential to reduce PrP expression through incorporation into the 5 untranslated region (5UTR). This exon is homologous to exon 2 in non-primate species, but contains a start codon that would yield an upstream open reading frame (uORF) with a stop codon prior to a splice site if included in PRNP mRNA, potentially downregulating PrP expression through translational repression or nonsense-mediated decay. We establish a minigene transfection system and test a panel of splice site alterations, identifying mutants that reduce PrP expression by as much as 78%. Our findings nominate a new therapeutic target for lowering PrP.\nStructural basis of tRNA recognition by the m3C-RNA-methyltransferase METTL6 in complex with SerRS seryl-tRNA synthetase\nAuthors: Throll, P.; Dolce, L. G.; Lastres, P. R.; Arnold, K.; Tengo, L.; Basu, S.; Kaiser, S.; Schneider, R.; Kowalinski, E.\nScore: 18.9, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570192\nMethylation of cytosine 32 in the anticodon loop of tRNAs to 3-methylcytosine (m3C) is crucial for cellular translation fidelity 1. Misregulation of the RNA methyltransferases setting this modification can cause aggressive cancers and metabolic disturbances 2,3. However, our understanding of the substrate selection and catalysis mode of the m3C RNA methyltransferases is currently still lacking. Here, we report the cryo-electron microscopy structure of the m3C tRNA methyltransferase METTL6 in complex with seryl-tRNA synthetase (SerRS) and their common substrate tRNASer. Through the complex structure, we identify the tRNA binding domain of METTL6. We show that SerRS acts as the tRNASer substrate selection factor for METTL6. We reveal how METTL6 and SerRS jointly coordinate the long variable arm of tRNASer in their interface. We demonstrate that SerRS augments the methylation activity of METTL6 and that direct contacts between METTL6 and SerRS are necessary for efficient tRNASer methylation. Finally, based on the structure of METTL6 in complex with SerRS and tRNASer, we postulate a universal tRNA binding mode for m3C RNA methyltransferases including METTL2 and METTL8, suggesting that these mammalian paralogues use similar ways to engage their respective tRNA substrates and co-factors.\nDNA Methylation-Based High-Resolution Mapping of Long-Distance Chromosomal Interactions in Nucleosome-Depleted Regions\nAuthors: Li, Y.; Lee, J.; Bai, L.\nScore: 4.1, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573430\n3C-based methods have significantly advanced our understanding of 3D genome organization. However, it remains a formidable task to precisely capture long-range chromosomal interactions between individual loci, such as those between promoters and distal enhancers. Here, we present Methyltransferase Targeting-based chromosome Architecture Capture (MTAC), a method that maps the contacts between a target site (viewpoint) and the rest of the genome in budding yeast with high resolution and sensitivity. MTAC detects hundreds of intra-and inter-chromosomal interactions within nucleosome-depleted regions (NDRs) that cannot be captured by 4C, Hi-C, or Micro-C. By applying MTAC to various viewpoints, we find that 1) tethering to the nuclear pore complex (NPC) is a major mechanism that mediates long-distance chromosomal interactions in yeast, 2) genes co-regulated by methionine assemble into inter-chromosomal clusters near NPCs upon activation, 3) mediated by condensin, the mating locus forms a highly specific interaction with the recombination enhancer (RE) in a mating-type specific manner, and 4) correlation of MTAC signals among NDRs reveal spatial mixing and segregation of the genome. Overall, these results demonstrate MTAC as a powerful tool to resolve fine-scale long-distance chromosomal interactions and provide insights into the 3D genome organization.\nGenomic Transfer via Membrane Vesicle: A Strategy of Giant Phage phiKZ for Early Infection\nAuthors: Antonova, D.; Nichiporenko, A.; Sobinina, M.; Vishnyakov, I. E.; Moiseenko, A.; Kurdyumova, I.; Khodorkovskii, M.; Sokolova, O. S.; Yakunina, M. V.\nScore: 3.5, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573766\nDuring infection, the giant phiKZ phage forms a specialized structure at the center of the host cell called the phage nucleus. This structure is crucial for safeguarding viral DNA against bacterial nucleases and for segregating the transcriptional activities of late genes. Here, we describe a morphological entity, the early phage infection vesicle (EPI vesicle), which appears to be responsible for earlier gene segregation at the beginning of the infection process. Using cryo-electron microscopy, electron tomography, and fluorescence microscopy with membrane-specific dyes, we found that the EPI vesicle is enclosed in a lipid bilayer originating, apparently, from the inner membrane of the bacterial cell. Our investigations further disclose that the phiKZ EPI vesicle contains both viral DNA and viral RNA polymerase (vRNAP). We have observed that the EPI vesicle migrates from the cell pole to the center, displaying co-localization with ChmA, the primary protein of the phage nucleus. While phage DNA is transported into the phage nucleus after phage maturation, the EPI vesicle remains outside. We hypothesized that the EPI vesicle acts as a membrane transport agent, efficiently delivering phage DNA to the phage nucleus while protecting it from the nucleases of the bacterium.\nCLSY docking to Pol IV requires a conserved domain critical for small RNA biogenesis and transposon silencing.\nAuthors: Felgines, L.; Rymen, B.; Martins, L. M.; Xu, G.; Matteoli, C.; Himber, C.; Zhou, M.; Eis, J.; Coruh, C.; Böhrer, M.; Kuhn, L.; Chicher, J.; Pandey, V.; Hammann, P.; Wohlschlegel, J.; Waltz, F.; Law, J. A.; Blevins, T.\nScore: 3.5, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573199\nEukaryotes must balance the need for gene transcription by RNA polymerase II (Pol II) against the danger of mutations caused by transposable element (TE) proliferation. In plants, these gene expression and TE silencing activities are divided between different RNA polymerases. Specifically, RNA polymerase IV (Pol IV), which evolved from Pol II, transcribes TEs to generate small interfering RNAs (siRNAs) that guide DNA methylation and block TE transcription by Pol II. While the Pol IV complex is recruited to TEs via SNF2-like CLASSY (CLSY) proteins, how Pol IV partners with the CLSYs remains unknown. Here we identified a conserved CYC-YPMF motif that is specific to Pol IV and is positioned on the complex exterior. Furthermore, we found that this motif is essential for the co-purification of all four CLSYs with Pol IV, but that only one CLSY is present in any given Pol IV complex. These findings support a \"one CLSY per Pol IV\" model where the CYC-YPMF motif acts as a CLSY-docking site. Indeed, mutations in and around this motif phenocopy pol iv null mutants. Together, these findings provide structural and functional insights into a critical protein feature that distinguishes Pol IV from other RNA polymerases, allowing it to promote genome stability by targeting TEs for silencing.\n",
  "wordCount" : "2216",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573526">
        <p class="paperTitle">Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.</p>
        <p class="info">Score: 17.6, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573526' target='https://doi.org/10.1101/2023.12.28.573526'> 10.1101/2023.12.28.573526</a></p>
        <p class="abstract">The Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood. In this study, we have identified a second HuSH complex, termed HuSH2, centered around TASOR2, a paralog of the core TASOR protein in HuSH. Our findings indicate that every subunit in both HuSH and HuSH2 has an important role in achieving precise genomic localization to distinct, non-overlapping genomic loci. We utilized in silico protein structure prediction to simulate the interactions between MPP8 and both TASOR paralogs. Drawing on the insights gained from these predictions, we implemented amino acid substitutions that interfered with the binding of MPP8 to each HuSH complex. Leveraging these MPP8 transgenes and other constructs, we identified an important role played by the relative quantities of HuSH complexes in controlling the activity of LINE-1 elements. Furthermore, our results suggest that dynamic changes in TASOR and TASOR2 expression enable cells to finely tune the extent of HuSH-mediated silencing. Our study provides insights into the intricate interplay between HuSH complexes, illuminating their important role in the regulation of retrotransposon silencing.

Key PointsO_LIThe identification of a previously unknown HuSH2 complex, with TASOR2 as its central component.
C_LIO_LIHuSH and HuSH2 complexes exhibit unique genomic localization patterns within the human genome.
C_LIO_LIDisruption of the delicate balance between the two HuSH complexes results in the desilencing of LINE-1 and other transposable elements.
C_LIO_LITASOR and TASOR2 engage in a competitive interaction for the HuSH subunit MPP8.
C_LIO_LIThe localization of MPP8 to either HuSH or HuSH2 sites is intricately regulated by its interaction with TASOR and TASOR2.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572534" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572534">
        <p class="paperTitle">Shieldin and CST co-orchestrate DNA polymerase-dependent tailed-end joining reactions independently of 53BP1-governed repair pathway choice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: King, A.; Reichl, P.; Metson, J. S.; Parker, R.; Munro, D.; Oliveira, C.; Becker, J. R.; Biggs, D.; Preece, C.; Davies, B.; Chapman, J. R.</p>
        <p class="info">Score: 17.7, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572534' target='https://doi.org/10.1101/2023.12.20.572534'> 10.1101/2023.12.20.572534</a></p>
        <p class="abstract">53BP1 regulates DNA end-joining in lymphocytes, diversifying immune antigen receptors. This involves nucleosome-bound 53BP1 at DNA double-stranded breaks (DSBs) recruiting RIF1 and shieldin, a poorly understood DNA-binding complex. The 53BP1-RIF1-shieldin axis is pathological in BRCA1-mutated cancers, blocking homologous recombination (HR) and driving illegitimate non-homologous end-joining (NHEJ). However, how this axis regulates DNA end-joining and HR suppression remains unresolved.

We investigated shieldin and its interplay with CST, a complex recently implicated in 53BP1-dependent activities. Immunophenotypically, mice lacking shieldin or CST are equivalent, with class-switch recombination co-reliant on both complexes. ATM-dependent DNA damage signalling underpins this cooperation, inducing physical interactions between these complexes that reveal shieldin as a DSB-responsive CST adaptor. Furthermore, DNA polymerase {zeta} functions downstream of shieldin, establishing DNA fill-in synthesis as the physiological function of shieldin-CST. Lastly, 53BP1 suppresses HR and promotes NHEJ in BRCA1-deficient mice and cells independently of shieldin. These findings showcase the resilience of the 53BP1 pathway, achieved through the collaboration of chromatin-bound 53BP1 complexes and DNA end-processing effector proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573427">
        <p class="paperTitle">Structure of the hepatitis C virus E1/E2 envelope proteins in a homodimeric complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Augestad, E.; Olesen, C. H.; Groenberg, C.; Soerensen, A.; Velazquez-Moctezuma, R.; Fanalista, M.; Bukh, J.; Wang, K.; Gourdon, P.; Prentoe, J.</p>
        <p class="info">Score: 10.8, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573427' target='https://doi.org/10.1101/2023.12.27.573427'> 10.1101/2023.12.27.573427</a></p>
        <p class="abstract">Worldwide, 58 million individuals suffer from chronic hepatitis C virus (HCV) infection, a primary driver of liver cancer. The HCV envelope proteins, E1 and E2, form a heterodimer, which is the target for neutralizing antibodies. Despite high-resolution structural models of partial heterodimer elements, the structural landscape of higher-order E1/E2 oligomers remains unexplored. We determined a ~3.5 [A] cryo-electron microscopy structure of membrane-extracted HCV E1/E2 in a homodimeric arrangement. This structure includes detailed information on the homodimer interface, the E2-binding pocket for hypervariable region 1, antigenic site 412 conformation, and the organization of the E1/E2 transmembrane regions, including one internal to E1. This higher-order E1/E2 assembly could play a pivotal role in the design of novel vaccine antigens better mimicking E1/E2 complexes on the HCV particle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573630">
        <p class="paperTitle">Genetic activation of canonical RNA interference in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buccheri, V.; Pasulka, J.; Malik, R.; Loubalova, Z.; Taborska, E.; Horvat, F.; Kulmann, M. I. R.; Jenickova, I.; Prochazka, J.; Sedlacek, R.; Svoboda, P.</p>
        <p class="info">Score: 9.3, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573630' target='https://doi.org/10.1101/2023.12.29.573630'> 10.1101/2023.12.29.573630</a></p>
        <p class="abstract">Canonical RNA interference (RNAi) is sequence-specific mRNA degradation guided by small interfering RNAs (siRNAs) made from double-stranded RNA (dsRNA) by RNase III Dicer. RNAi has different roles including gene regulation, antiviral immunity or defense against transposable elements. In mammals, RNAi is constrained by Dicer, which is adapted to produce microRNAs, another class of small RNAs. However, RNAi exists in mouse oocytes, which employs a truncated Dicer variant. A homozygous mutation to express only the truncated variant ({Delta}HEL1) causes dysregulation of microRNAs and perinatal lethality in mice. Here, we report the phenotype and RNAi activity in Dicer{Delta}HEL1/wtmice, which are viable, show minimal miRNome changes but their endogenous siRNA levels are increased by an order of magnitude. We show that siRNA abundance is limited by available dsRNA but not by PKR, a dsRNA sensor of innate immunity. Expressing dsRNA from a transgene, functional RNAi in vivo was induced in heart. Dicer{Delta}HEL1/wt mice thus represent a new model for researching mammalian canonical RNAi in vivo and offer an unprecedented platform for addressing claims about its biological roles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573721">
        <p class="paperTitle">Unraveling the mechanism of centromere inactivation in human aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikder, S.; Baek, S.; McNeil, T.; Dalal, Y.</p>
        <p class="info">Score: 8.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573721' target='https://doi.org/10.1101/2023.12.30.573721'> 10.1101/2023.12.30.573721</a></p>
        <p class="abstract">Aging involves a range of genetic, epigenetic, and physiological alterations. A key characteristic of aged cells is the loss of global heterochromatin, accompanied by a reduction in canonical histone levels. In this study, we track the fate of centromeres during aging in human cells. Our findings reveal that the centromeric histone H3 variant CENP-A is downregulated in aged cells, in a p53-dependent manner. We observe repression of centromeric noncoding transcription through an epigenetic mechanism via recruitment of a lysine-specific demethylase 1 (LSD1/KDM1A) to centromeres. This suppression results in defective de novo CENP-A loading at aging centromeres. By dual inhibition of p53 and LSD1/KDM1A in aged cells, we mitigate the reduction in centromeric proteins and centromeric transcripts, leading to mitotic rejuvenation of these cells. These results offer insights into a novel mechanism for centromeric inactivation during aging and provide potential strategies to reactivate centromeres.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572439" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572439">
        <p class="paperTitle">Modulation of prion protein expression through cryptic splice site manipulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gentile, J. E.; Corridon, T. L.; Mortberg, M. A.; D&#39;Souza, E. N.; Whiffin, N.; Minikel, E. V.; Vallabh, S. M.</p>
        <p class="info">Score: 15.2, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572439' target='https://doi.org/10.1101/2023.12.19.572439'> 10.1101/2023.12.19.572439</a></p>
        <p class="abstract">Lowering expression of prion protein (PrP) is a well-validated therapeutic strategy in prion disease, but additional modalities are urgently needed. In other diseases, small molecules have proven capable of modulating pre-mRNA splicing, sometimes by forcing inclusion of cryptic exons that reduce gene expression. Here, we characterize a cryptic exon located in human PRNPs sole intron and evaluate its potential to reduce PrP expression through incorporation into the 5 untranslated region (5UTR). This exon is homologous to exon 2 in non-primate species, but contains a start codon that would yield an upstream open reading frame (uORF) with a stop codon prior to a splice site if included in PRNP mRNA, potentially downregulating PrP expression through translational repression or nonsense-mediated decay. We establish a minigene transfection system and test a panel of splice site alterations, identifying mutants that reduce PrP expression by as much as 78%. Our findings nominate a new therapeutic target for lowering PrP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570192" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570192">
        <p class="paperTitle">Structural basis of tRNA recognition by the m3C-RNA-methyltransferase METTL6 in complex with SerRS seryl-tRNA synthetase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Throll, P.; Dolce, L. G.; Lastres, P. R.; Arnold, K.; Tengo, L.; Basu, S.; Kaiser, S.; Schneider, R.; Kowalinski, E.</p>
        <p class="info">Score: 18.9, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570192' target='https://doi.org/10.1101/2023.12.05.570192'> 10.1101/2023.12.05.570192</a></p>
        <p class="abstract">Methylation of cytosine 32 in the anticodon loop of tRNAs to 3-methylcytosine (m3C) is crucial for cellular translation fidelity 1. Misregulation of the RNA methyltransferases setting this modification can cause aggressive cancers and metabolic disturbances 2,3. However, our understanding of the substrate selection and catalysis mode of the m3C RNA methyltransferases is currently still lacking. Here, we report the cryo-electron microscopy structure of the m3C tRNA methyltransferase METTL6 in complex with seryl-tRNA synthetase (SerRS) and their common substrate tRNASer. Through the complex structure, we identify the tRNA binding domain of METTL6. We show that SerRS acts as the tRNASer substrate selection factor for METTL6. We reveal how METTL6 and SerRS jointly coordinate the long variable arm of tRNASer in their interface. We demonstrate that SerRS augments the methylation activity of METTL6 and that direct contacts between METTL6 and SerRS are necessary for efficient tRNASer methylation. Finally, based on the structure of METTL6 in complex with SerRS and tRNASer, we postulate a universal tRNA binding mode for m3C RNA methyltransferases including METTL2 and METTL8, suggesting that these mammalian paralogues use similar ways to engage their respective tRNA substrates and co-factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573430" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573430">
        <p class="paperTitle">DNA Methylation-Based High-Resolution Mapping of Long-Distance Chromosomal Interactions in Nucleosome-Depleted Regions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Lee, J.; Bai, L.</p>
        <p class="info">Score: 4.1, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573430' target='https://doi.org/10.1101/2023.12.27.573430'> 10.1101/2023.12.27.573430</a></p>
        <p class="abstract">3C-based methods have significantly advanced our understanding of 3D genome organization. However, it remains a formidable task to precisely capture long-range chromosomal interactions between individual loci, such as those between promoters and distal enhancers. Here, we present Methyltransferase Targeting-based chromosome Architecture Capture (MTAC), a method that maps the contacts between a target site (viewpoint) and the rest of the genome in budding yeast with high resolution and sensitivity. MTAC detects hundreds of intra-and inter-chromosomal interactions within nucleosome-depleted regions (NDRs) that cannot be captured by 4C, Hi-C, or Micro-C. By applying MTAC to various viewpoints, we find that 1) tethering to the nuclear pore complex (NPC) is a major mechanism that mediates long-distance chromosomal interactions in yeast, 2) genes co-regulated by methionine assemble into inter-chromosomal clusters near NPCs upon activation, 3) mediated by condensin, the mating locus forms a highly specific interaction with the recombination enhancer (RE) in a mating-type specific manner, and 4) correlation of MTAC signals among NDRs reveal spatial mixing and segregation of the genome. Overall, these results demonstrate MTAC as a powerful tool to resolve fine-scale long-distance chromosomal interactions and provide insights into the 3D genome organization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573766">
        <p class="paperTitle">Genomic Transfer via Membrane Vesicle: A Strategy of Giant Phage phiKZ for Early Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antonova, D.; Nichiporenko, A.; Sobinina, M.; Vishnyakov, I. E.; Moiseenko, A.; Kurdyumova, I.; Khodorkovskii, M.; Sokolova, O. S.; Yakunina, M. V.</p>
        <p class="info">Score: 3.5, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573766' target='https://doi.org/10.1101/2023.12.31.573766'> 10.1101/2023.12.31.573766</a></p>
        <p class="abstract">During infection, the giant phiKZ phage forms a specialized structure at the center of the host cell called the phage nucleus. This structure is crucial for safeguarding viral DNA against bacterial nucleases and for segregating the transcriptional activities of late genes. Here, we describe a morphological entity, the early phage infection vesicle (EPI vesicle), which appears to be responsible for earlier gene segregation at the beginning of the infection process. Using cryo-electron microscopy, electron tomography, and fluorescence microscopy with membrane-specific dyes, we found that the EPI vesicle is enclosed in a lipid bilayer originating, apparently, from the inner membrane of the bacterial cell. Our investigations further disclose that the phiKZ EPI vesicle contains both viral DNA and viral RNA polymerase (vRNAP). We have observed that the EPI vesicle migrates from the cell pole to the center, displaying co-localization with ChmA, the primary protein of the phage nucleus. While phage DNA is transported into the phage nucleus after phage maturation, the EPI vesicle remains outside. We hypothesized that the EPI vesicle acts as a membrane transport agent, efficiently delivering phage DNA to the phage nucleus while protecting it from the nucleases of the bacterium.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573199">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573199" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573199">
        <p class="paperTitle">CLSY docking to Pol IV requires a conserved domain critical for small RNA biogenesis and transposon silencing.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573199" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573199" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Felgines, L.; Rymen, B.; Martins, L. M.; Xu, G.; Matteoli, C.; Himber, C.; Zhou, M.; Eis, J.; Coruh, C.; Böhrer, M.; Kuhn, L.; Chicher, J.; Pandey, V.; Hammann, P.; Wohlschlegel, J.; Waltz, F.; Law, J. A.; Blevins, T.</p>
        <p class="info">Score: 3.5, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573199' target='https://doi.org/10.1101/2023.12.26.573199'> 10.1101/2023.12.26.573199</a></p>
        <p class="abstract">Eukaryotes must balance the need for gene transcription by RNA polymerase II (Pol II) against the danger of mutations caused by transposable element (TE) proliferation. In plants, these gene expression and TE silencing activities are divided between different RNA polymerases. Specifically, RNA polymerase IV (Pol IV), which evolved from Pol II, transcribes TEs to generate small interfering RNAs (siRNAs) that guide DNA methylation and block TE transcription by Pol II. While the Pol IV complex is recruited to TEs via SNF2-like CLASSY (CLSY) proteins, how Pol IV partners with the CLSYs remains unknown. Here we identified a conserved CYC-YPMF motif that is specific to Pol IV and is positioned on the complex exterior. Furthermore, we found that this motif is essential for the co-purification of all four CLSYs with Pol IV, but that only one CLSY is present in any given Pol IV complex. These findings support a &#34;one CLSY per Pol IV&#34; model where the CYC-YPMF motif acts as a CLSY-docking site. Indeed, mutations in and around this motif phenocopy pol iv null mutants. Together, these findings provide structural and functional insights into a critical protein feature that distinguishes Pol IV from other RNA polymerases, allowing it to promote genome stability by targeting TEs for silencing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
