<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Air monitoring by nanopore sequencing
Authors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.
Score: 33.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572325
While the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Air monitoring by nanopore sequencing
Authors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.
Score: 33.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572325
While the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Air monitoring by nanopore sequencing
Authors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.
Score: 33.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572325
While the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Air monitoring by nanopore sequencing\nAuthors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.\nScore: 33.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572325\nWhile the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects.",
  "keywords": [
    
  ],
  "articleBody": " Air monitoring by nanopore sequencing\nAuthors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.\nScore: 33.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572325\nWhile the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects. We here show that nanopore shotgun sequencing can robustly assess the air microbiome in combination with active air sampling through liquid impingement and tailored computational analysis. We provide laboratory and computational protocols for air microbiome profiling, and reliably assess the taxonomic composition of the core air microbiome in controlled and natural environments. Based on de novo assemblies from the long sequencing reads, we further identify the taxa of several air microbiota down to the species level and annotate their ecosystem functions, including their potential role in natural biodegradation processes. As air monitoring by nanopore sequencing can be employed in an automatable, fast, and portable manner for in situ applications all around the world, we envision that this approach can help holistically explore the role of the air microbiome.\nInterplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing\nAuthors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.\nScore: 10.5, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573526\nThe Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood. In this study, we have identified a second HuSH complex, termed HuSH2, centered around TASOR2, a paralog of the core TASOR protein in HuSH. Our findings indicate that every subunit in both HuSH and HuSH2 has an important role in achieving precise genomic localization to distinct, non-overlapping genomic loci. We utilized in silico protein structure prediction to simulate the interactions between MPP8 and both TASOR paralogs. Drawing on the insights gained from these predictions, we implemented amino acid substitutions that interfered with the binding of MPP8 to each HuSH complex. Leveraging these MPP8 transgenes and other constructs, we identified an important role played by the relative quantities of HuSH complexes in controlling the activity of LINE-1 elements. Furthermore, our results suggest that dynamic changes in TASOR and TASOR2 expression enable cells to finely tune the extent of HuSH-mediated silencing. Our study provides insights into the intricate interplay between HuSH complexes, illuminating their important role in the regulation of retrotransposon silencing. Key PointsO_LIThe identification of a previously unknown HuSH2 complex, with TASOR2 as its central component. C_LIO_LIHuSH and HuSH2 complexes exhibit unique genomic localization patterns within the human genome. C_LIO_LIDisruption of the delicate balance between the two HuSH complexes results in the desilencing of LINE-1 and other transposable elements. C_LIO_LITASOR and TASOR2 engage in a competitive interaction for the HuSH subunit MPP8. C_LIO_LIThe localization of MPP8 to either HuSH or HuSH2 sites is intricately regulated by its interaction with TASOR and TASOR2. C_LI\nStructure of the hepatitis C virus E1/E2 envelope proteins in a homodimeric complex\nAuthors: Augestad, E.; Olesen, C. H.; Groenberg, C.; Soerensen, A.; Velazquez-Moctezuma, R.; Fanalista, M.; Bukh, J.; Wang, K.; Gourdon, P.; Prentoe, J.\nScore: 10.3, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573427\nWorldwide, 58 million individuals suffer from chronic hepatitis C virus (HCV) infection, a primary driver of liver cancer. The HCV envelope proteins, E1 and E2, form a heterodimer, which is the target for neutralizing antibodies. Despite high-resolution structural models of partial heterodimer elements, the structural landscape of higher-order E1/E2 oligomers remains unexplored. We determined a ~3.5 [A] cryo-electron microscopy structure of membrane-extracted HCV E1/E2 in a homodimeric arrangement. This structure includes detailed information on the homodimer interface, the E2-binding pocket for hypervariable region 1, antigenic site 412 conformation, and the organization of the E1/E2 transmembrane regions, including one internal to E1. This higher-order E1/E2 assembly could play a pivotal role in the design of novel vaccine antigens better mimicking E1/E2 complexes on the HCV particle.\nGenetic activation of canonical RNA interference in mice\nAuthors: Buccheri, V.; Pasulka, J.; Malik, R.; Loubalova, Z.; Taborska, E.; Horvat, F.; Kulmann, M. I. R.; Jenickova, I.; Prochazka, J.; Sedlacek, R.; Svoboda, P.\nScore: 7.0, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573630\nCanonical RNA interference (RNAi) is sequence-specific mRNA degradation guided by small interfering RNAs (siRNAs) made from double-stranded RNA (dsRNA) by RNase III Dicer. RNAi has different roles including gene regulation, antiviral immunity or defense against transposable elements. In mammals, RNAi is constrained by Dicer, which is adapted to produce microRNAs, another class of small RNAs. However, RNAi exists in mouse oocytes, which employs a truncated Dicer variant. A homozygous mutation to express only the truncated variant ({Delta}HEL1) causes dysregulation of microRNAs and perinatal lethality in mice. Here, we report the phenotype and RNAi activity in Dicer{Delta}HEL1/wt mice, which are viable, show minimal miRNome changes but their endogenous siRNA levels are increased by an order of magnitude. We show that siRNA abundance is limited by available dsRNA but not by PKR, a dsRNA sensor of innate immunity. Expressing dsRNA from a transgene, functional RNAi in vivo was induced in heart. Dicer{Delta}HEL1/wt mice thus represent a new model for researching mammalian canonical RNAi in vivo and offer an unprecedented platform for addressing claims about its biological roles.\nRNA 5' terminal nucleotide determines the strength of the RIG-I/IFN signaling pathway\nAuthors: Wolczyk, M.; Szymanski, J.; Trus, I.; Naz, Z.; Bolembach, A.; Choudhury, N. R.; Tame, T.; Konuc, C.; Nowak, E.; Spanos, C.; Rappsilber, J.; Michlewski, G.\nScore: 12.9, Published: 2023-12-22 DOI: 10.1101/2023.12.22.573000\nThe interferon (IFN) response is crucial for antiviral activity, but its overstimulation can lead to a wide range of autoimmune disorders. The cytoplasmic pattern recognition receptor RIG-I detects viral RNAs and endogenous polymerase III transcripts carrying a 5'-triphosphate (5'- ppp) or 5'-diphosphate (5'-pp) moiety, triggering an IFN immune response. While many viral RNAs initiate with 5'-ppp-adenosine (5'-pppA) and most endogenous Pol III transcripts in higher eukaryotes start with 5'-ppp-guanosine (5'-pppG), no apparent reason for this bias has been identified so far. Here we demonstrate that RNAs initiating with 5'-pppA trigger a much stronger RIG-I/IFN response than those starting with 5'-pppG. The structures of the matching RNA pairs have been confirmed to be similar, suggesting that functional differences in RIG-I/IFN pathway stimulation cannot be primarily explained by an altered conformation. We show that several GTP-binding proteins interact with 5'-pppG RNAs creating a steric hindrance and protecting these RNAs from activating RIG-I. Our findings suggest that 5'-pppG RNAs may allow some RNA viruses and Pol III transcripts to evade detection by cellular immune sensors. These results provide a new insight into the mechanism of sequence-dependent RIG-I/IFN pathway activation and have significant implications for understanding the antiviral response to RNA viruses and the contribution of Pol III-derived RNAs to autoimmune disorders.\nFully automated workflow for integrated sample digestion and Evotip loading enabling high-throughput clinical proteomics\nAuthors: Kverneland, A. H.; Harking, F.; Vej-Nielsen, J. M.; Huusfeldt, M.; Bekker-Jensen, D. B.; Svane, I. M.; Bache, N.; Olsen, J. V.\nScore: 5.6, Published: 2023-12-22 DOI: 10.1101/2023.12.22.573056\nProtein identification and quantification is an important tool for biomarker discovery. With the increased sensitivity and speed of modern mass spectrometers, sample-preparation remains a bottleneck for studying large cohorts. To address this issue, we prepared and evaluated a simple and efficient workflow on the Opentrons OT-2 (OT-2) robot that combines sample digestion, cleanup and Evotip loading in a fully automated manner, allowing the processing of up to 192 samples in 6 hours. Our results demonstrate a highly sensitive workflow yielding both reproducibility and stability even at low sample inputs. The workflow is optimized for minimal sample starting amount to reduce the costs for reagents needed for sample preparation, which is critical when analyzing large biological cohorts. Building on the digesting workflow, we incorporated an automated phosphopeptide enrichment step using magnetic Ti-IMAC beads. This allows for a fully automated proteome and phosphoproteome sample preparation in a single step with high sensitivity. Using the integrated workflow, we evaluated the effects of cancer immune therapy on the plasma proteome in metastatic melanoma patients.\nModulation of prion protein expression through cryptic splice site manipulation\nAuthors: Gentile, J. E.; Corridon, T. L.; Mortberg, M. A.; D'Souza, E. N.; Whiffin, N.; Minikel, E. V.; Vallabh, S. M.\nScore: 15.2, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572439\nLowering expression of prion protein (PrP) is a well-validated therapeutic strategy in prion disease, but additional modalities are urgently needed. In other diseases, small molecules have proven capable of modulating pre-mRNA splicing, sometimes by forcing inclusion of cryptic exons that reduce gene expression. Here, we characterize a cryptic exon located in human PRNPs sole intron and evaluate its potential to reduce PrP expression through incorporation into the 5 untranslated region (5UTR). This exon is homologous to exon 2 in non-primate species, but contains a start codon that would yield an upstream open reading frame (uORF) with a stop codon prior to a splice site if included in PRNP mRNA, potentially downregulating PrP expression through translational repression or nonsense-mediated decay. We establish a minigene transfection system and test a panel of splice site alterations, identifying mutants that reduce PrP expression by as much as 78%. Our findings nominate a new therapeutic target for lowering PrP.\nDNA Methylation-Based High-Resolution Mapping of Long-Distance Chromosomal Interactions in Nucleosome-Depleted Regions\nAuthors: Li, Y.; Lee, J.; Bai, L.\nScore: 4.1, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573430\n3C-based methods have significantly advanced our understanding of 3D genome organization. However, it remains a formidable task to precisely capture long-range chromosomal interactions between individual loci, such as those between promoters and distal enhancers. Here, we present Methyltransferase Targeting-based chromosome Architecture Capture (MTAC), a method that maps the contacts between a target site (viewpoint) and the rest of the genome in budding yeast with high resolution and sensitivity. MTAC detects hundreds of intra-and inter-chromosomal interactions within nucleosome-depleted regions (NDRs) that cannot be captured by 4C, Hi-C, or Micro-C. By applying MTAC to various viewpoints, we find that 1) tethering to the nuclear pore complex (NPC) is a major mechanism that mediates long-distance chromosomal interactions in yeast, 2) genes co-regulated by methionine assemble into inter-chromosomal clusters near NPCs upon activation, 3) mediated by condensin, the mating locus forms a highly specific interaction with the recombination enhancer (RE) in a mating-type specific manner, and 4) correlation of MTAC signals among NDRs reveal spatial mixing and segregation of the genome. Overall, these results demonstrate MTAC as a powerful tool to resolve fine-scale long-distance chromosomal interactions and provide insights into the 3D genome organization.\nConcerted SUMO-targeted ubiquitin ligase activities of TOPORS and RNF4 are essential for stress management and cell proliferation\nAuthors: Liu, J. C. Y.; Ackermann, L.; Hoffmann, S.; Gal, Z.; Hendriks, I. A.; Jain, C.; Morlot, L.; Tatham, M. H.; McLelland, G.-L.; Hay, R. T.; Nielsen, M. L.; Brummelkamp, T. R.; Haahr, P.; Mailand, N.\nScore: 15.5, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572718\nProtein SUMOylation provides a principal driving force for cellular stress responses including DNA-protein crosslink (DPC) repair and arsenic-induced PML body degradation. In genome-scale screens, we identified the human E3 ligase TOPORS as a key effector of SUMO-dependent DPC resolution. We demonstrate that TOPORS promotes DPC repair by functioning as a SUMO-targeted ubiquitin ligase (STUbL) for DPCs, combining ubiquitin ligase activity through its RING domain with poly-SUMO chain binding via a cluster of SUMO-interacting motifs, analogous to the STUbL RNF4. Surprisingly, the STUbL activities of TOPORS and RNF4 are both required for SUMO-dependent DPC repair, PML degradation and other stress responses, making overlapping and distinct contributions to ubiquitin chain formation on SUMOylated targets to enable p97/VCP unfoldase recruitment. Combined loss of TOPORS and RNF4 is synthetic lethal even in unstressed cells, leading to defective clearance of SUMOylated proteins from chromatin accompanied by cell cycle arrest and apoptosis. Together, our findings establish TOPORS as a novel STUbL whose concerted action with RNF4 defines a general mechanistic principle in crucial cellular processes governed by direct SUMO-ubiquitin crosstalk. HighlightsO_LIThe RING E3 ligase TOPORS is required for SUMO-dependent DPC repair C_LIO_LITOPORS is a novel SUMO-targeted ubiquitin ligase (STUbL) C_LIO_LITOPORS promotes multiple STUbL-driven processes in conjunction with RNF4 C_LIO_LICombined TOPORS and RNF4 loss is synthetic lethal in human cells C_LI\nRapid DNA unwinding accelerates genome editing by engineered CRISPR-Cas9\nAuthors: Eggers, A. R.; Chen, K.; Soczek, K. M.; Tuck, O. T.; Doherty, E. E.; Thornton, B. W.; Xu, B.; Trinidad, M. I.; Doudna, J. A.\nScore: 34.2, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571777\nThermostable CRISPR-Cas9 enzymes could improve genome editing efficiency and delivery due to extended protein lifetimes. However, initial experimentation demonstrated Geobacillus stearothermophilus Cas9 (GeoCas9) to be virtually inactive when used in cultured human cells. Laboratory-evolved variants of GeoCas9 overcome this natural limitation by acquiring mutations in the wedge (WED) domain that produce \u003e100-fold higher genome editing levels. Cryo-EM structures of the wildtype and improved GeoCas9 (iGeoCas9) enzymes reveal extended contacts between the WED domain of iGeoCas9 and DNA substrates. Biochemical analysis shows that iGeoCas9 accelerates DNA unwinding to capture substrates under the magnesium-restricted conditions typical of mammalian but not bacterial cells. These findings enabled rational engineering of other Cas9 orthologs to enhance genome editing levels, pointing to a general strategy for editing enzyme improvement. Together, these results uncover a new role for the Cas9 WED domain in DNA unwinding and demonstrate how accelerated target unwinding dramatically improves Cas9-induced genome editing activity.\n",
  "wordCount" : "2233",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572325">
        <p class="paperTitle">Air monitoring by nanopore sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.</p>
        <p class="info">Score: 33.8, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572325' target='https://doi.org/10.1101/2023.12.19.572325'> 10.1101/2023.12.19.572325</a></p>
        <p class="abstract">While the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects. We here show that nanopore shotgun sequencing can robustly assess the air microbiome in combination with active air sampling through liquid impingement and tailored computational analysis. We provide laboratory and computational protocols for air microbiome profiling, and reliably assess the taxonomic composition of the core air microbiome in controlled and natural environments. Based on de novo assemblies from the long sequencing reads, we further identify the taxa of several air microbiota down to the species level and annotate their ecosystem functions, including their potential role in natural biodegradation processes. As air monitoring by nanopore sequencing can be employed in an automatable, fast, and portable manner for in situ applications all around the world, we envision that this approach can help holistically explore the role of the air microbiome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573526">
        <p class="paperTitle">Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.</p>
        <p class="info">Score: 10.5, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573526' target='https://doi.org/10.1101/2023.12.28.573526'> 10.1101/2023.12.28.573526</a></p>
        <p class="abstract">The Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood. In this study, we have identified a second HuSH complex, termed HuSH2, centered around TASOR2, a paralog of the core TASOR protein in HuSH. Our findings indicate that every subunit in both HuSH and HuSH2 has an important role in achieving precise genomic localization to distinct, non-overlapping genomic loci. We utilized in silico protein structure prediction to simulate the interactions between MPP8 and both TASOR paralogs. Drawing on the insights gained from these predictions, we implemented amino acid substitutions that interfered with the binding of MPP8 to each HuSH complex. Leveraging these MPP8 transgenes and other constructs, we identified an important role played by the relative quantities of HuSH complexes in controlling the activity of LINE-1 elements. Furthermore, our results suggest that dynamic changes in TASOR and TASOR2 expression enable cells to finely tune the extent of HuSH-mediated silencing. Our study provides insights into the intricate interplay between HuSH complexes, illuminating their important role in the regulation of retrotransposon silencing.

Key PointsO_LIThe identification of a previously unknown HuSH2 complex, with TASOR2 as its central component.
C_LIO_LIHuSH and HuSH2 complexes exhibit unique genomic localization patterns within the human genome.
C_LIO_LIDisruption of the delicate balance between the two HuSH complexes results in the desilencing of LINE-1 and other transposable elements.
C_LIO_LITASOR and TASOR2 engage in a competitive interaction for the HuSH subunit MPP8.
C_LIO_LIThe localization of MPP8 to either HuSH or HuSH2 sites is intricately regulated by its interaction with TASOR and TASOR2.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573427">
        <p class="paperTitle">Structure of the hepatitis C virus E1/E2 envelope proteins in a homodimeric complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Augestad, E.; Olesen, C. H.; Groenberg, C.; Soerensen, A.; Velazquez-Moctezuma, R.; Fanalista, M.; Bukh, J.; Wang, K.; Gourdon, P.; Prentoe, J.</p>
        <p class="info">Score: 10.3, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573427' target='https://doi.org/10.1101/2023.12.27.573427'> 10.1101/2023.12.27.573427</a></p>
        <p class="abstract">Worldwide, 58 million individuals suffer from chronic hepatitis C virus (HCV) infection, a primary driver of liver cancer. The HCV envelope proteins, E1 and E2, form a heterodimer, which is the target for neutralizing antibodies. Despite high-resolution structural models of partial heterodimer elements, the structural landscape of higher-order E1/E2 oligomers remains unexplored. We determined a ~3.5 [A] cryo-electron microscopy structure of membrane-extracted HCV E1/E2 in a homodimeric arrangement. This structure includes detailed information on the homodimer interface, the E2-binding pocket for hypervariable region 1, antigenic site 412 conformation, and the organization of the E1/E2 transmembrane regions, including one internal to E1. This higher-order E1/E2 assembly could play a pivotal role in the design of novel vaccine antigens better mimicking E1/E2 complexes on the HCV particle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573630">
        <p class="paperTitle">Genetic activation of canonical RNA interference in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buccheri, V.; Pasulka, J.; Malik, R.; Loubalova, Z.; Taborska, E.; Horvat, F.; Kulmann, M. I. R.; Jenickova, I.; Prochazka, J.; Sedlacek, R.; Svoboda, P.</p>
        <p class="info">Score: 7.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573630' target='https://doi.org/10.1101/2023.12.29.573630'> 10.1101/2023.12.29.573630</a></p>
        <p class="abstract">Canonical RNA interference (RNAi) is sequence-specific mRNA degradation guided by small interfering RNAs (siRNAs) made from double-stranded RNA (dsRNA) by RNase III Dicer. RNAi has different roles including gene regulation, antiviral immunity or defense against transposable elements. In mammals, RNAi is constrained by Dicer, which is adapted to produce microRNAs, another class of small RNAs. However, RNAi exists in mouse oocytes, which employs a truncated Dicer variant. A homozygous mutation to express only the truncated variant ({Delta}HEL1) causes dysregulation of microRNAs and perinatal lethality in mice. Here, we report the phenotype and RNAi activity in Dicer{Delta}HEL1/wt mice, which are viable, show minimal miRNome changes but their endogenous siRNA levels are increased by an order of magnitude. We show that siRNA abundance is limited by available dsRNA but not by PKR, a dsRNA sensor of innate immunity. Expressing dsRNA from a transgene, functional RNAi in vivo was induced in heart. Dicer{Delta}HEL1/wt mice thus represent a new model for researching mammalian canonical RNAi in vivo and offer an unprecedented platform for addressing claims about its biological roles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573000">
        <p class="paperTitle">RNA 5&#39; terminal nucleotide determines the strength of the RIG-I/IFN signaling pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolczyk, M.; Szymanski, J.; Trus, I.; Naz, Z.; Bolembach, A.; Choudhury, N. R.; Tame, T.; Konuc, C.; Nowak, E.; Spanos, C.; Rappsilber, J.; Michlewski, G.</p>
        <p class="info">Score: 12.9, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573000' target='https://doi.org/10.1101/2023.12.22.573000'> 10.1101/2023.12.22.573000</a></p>
        <p class="abstract">The interferon (IFN) response is crucial for antiviral activity, but its overstimulation can lead to a wide range of autoimmune disorders. The cytoplasmic pattern recognition receptor RIG-I detects viral RNAs and endogenous polymerase III transcripts carrying a 5&#39;-triphosphate (5&#39;- ppp) or 5&#39;-diphosphate (5&#39;-pp) moiety, triggering an IFN immune response. While many viral RNAs initiate with 5&#39;-ppp-adenosine (5&#39;-pppA) and most endogenous Pol III transcripts in higher eukaryotes start with 5&#39;-ppp-guanosine (5&#39;-pppG), no apparent reason for this bias has been identified so far. Here we demonstrate that RNAs initiating with 5&#39;-pppA trigger a much stronger RIG-I/IFN response than those starting with 5&#39;-pppG. The structures of the matching RNA pairs have been confirmed to be similar, suggesting that functional differences in RIG-I/IFN pathway stimulation cannot be primarily explained by an altered conformation. We show that several GTP-binding proteins interact with 5&#39;-pppG RNAs creating a steric hindrance and protecting these RNAs from activating RIG-I. Our findings suggest that 5&#39;-pppG RNAs may allow some RNA viruses and Pol III transcripts to evade detection by cellular immune sensors. These results provide a new insight into the mechanism of sequence-dependent RIG-I/IFN pathway activation and have significant implications for understanding the antiviral response to RNA viruses and the contribution of Pol III-derived RNAs to autoimmune disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573056">
        <p class="paperTitle">Fully automated workflow for integrated sample digestion and Evotip loading enabling high-throughput clinical proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kverneland, A. H.; Harking, F.; Vej-Nielsen, J. M.; Huusfeldt, M.; Bekker-Jensen, D. B.; Svane, I. M.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 5.6, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573056' target='https://doi.org/10.1101/2023.12.22.573056'> 10.1101/2023.12.22.573056</a></p>
        <p class="abstract">Protein identification and quantification is an important tool for biomarker discovery. With the increased sensitivity and speed of modern mass spectrometers, sample-preparation remains a bottleneck for studying large cohorts. To address this issue, we prepared and evaluated a simple and efficient workflow on the Opentrons OT-2 (OT-2) robot that combines sample digestion, cleanup and Evotip loading in a fully automated manner, allowing the processing of up to 192 samples in 6 hours. Our results demonstrate a highly sensitive workflow yielding both reproducibility and stability even at low sample inputs. The workflow is optimized for minimal sample starting amount to reduce the costs for reagents needed for sample preparation, which is critical when analyzing large biological cohorts. Building on the digesting workflow, we incorporated an automated phosphopeptide enrichment step using magnetic Ti-IMAC beads. This allows for a fully automated proteome and phosphoproteome sample preparation in a single step with high sensitivity. Using the integrated workflow, we evaluated the effects of cancer immune therapy on the plasma proteome in metastatic melanoma patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572439" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572439">
        <p class="paperTitle">Modulation of prion protein expression through cryptic splice site manipulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gentile, J. E.; Corridon, T. L.; Mortberg, M. A.; D&#39;Souza, E. N.; Whiffin, N.; Minikel, E. V.; Vallabh, S. M.</p>
        <p class="info">Score: 15.2, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572439' target='https://doi.org/10.1101/2023.12.19.572439'> 10.1101/2023.12.19.572439</a></p>
        <p class="abstract">Lowering expression of prion protein (PrP) is a well-validated therapeutic strategy in prion disease, but additional modalities are urgently needed. In other diseases, small molecules have proven capable of modulating pre-mRNA splicing, sometimes by forcing inclusion of cryptic exons that reduce gene expression. Here, we characterize a cryptic exon located in human PRNPs sole intron and evaluate its potential to reduce PrP expression through incorporation into the 5 untranslated region (5UTR). This exon is homologous to exon 2 in non-primate species, but contains a start codon that would yield an upstream open reading frame (uORF) with a stop codon prior to a splice site if included in PRNP mRNA, potentially downregulating PrP expression through translational repression or nonsense-mediated decay. We establish a minigene transfection system and test a panel of splice site alterations, identifying mutants that reduce PrP expression by as much as 78%. Our findings nominate a new therapeutic target for lowering PrP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573430" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573430">
        <p class="paperTitle">DNA Methylation-Based High-Resolution Mapping of Long-Distance Chromosomal Interactions in Nucleosome-Depleted Regions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Lee, J.; Bai, L.</p>
        <p class="info">Score: 4.1, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573430' target='https://doi.org/10.1101/2023.12.27.573430'> 10.1101/2023.12.27.573430</a></p>
        <p class="abstract">3C-based methods have significantly advanced our understanding of 3D genome organization. However, it remains a formidable task to precisely capture long-range chromosomal interactions between individual loci, such as those between promoters and distal enhancers. Here, we present Methyltransferase Targeting-based chromosome Architecture Capture (MTAC), a method that maps the contacts between a target site (viewpoint) and the rest of the genome in budding yeast with high resolution and sensitivity. MTAC detects hundreds of intra-and inter-chromosomal interactions within nucleosome-depleted regions (NDRs) that cannot be captured by 4C, Hi-C, or Micro-C. By applying MTAC to various viewpoints, we find that 1) tethering to the nuclear pore complex (NPC) is a major mechanism that mediates long-distance chromosomal interactions in yeast, 2) genes co-regulated by methionine assemble into inter-chromosomal clusters near NPCs upon activation, 3) mediated by condensin, the mating locus forms a highly specific interaction with the recombination enhancer (RE) in a mating-type specific manner, and 4) correlation of MTAC signals among NDRs reveal spatial mixing and segregation of the genome. Overall, these results demonstrate MTAC as a powerful tool to resolve fine-scale long-distance chromosomal interactions and provide insights into the 3D genome organization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572718" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572718">
        <p class="paperTitle">Concerted SUMO-targeted ubiquitin ligase activities of TOPORS and RNF4 are essential for stress management and cell proliferation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, J. C. Y.; Ackermann, L.; Hoffmann, S.; Gal, Z.; Hendriks, I. A.; Jain, C.; Morlot, L.; Tatham, M. H.; McLelland, G.-L.; Hay, R. T.; Nielsen, M. L.; Brummelkamp, T. R.; Haahr, P.; Mailand, N.</p>
        <p class="info">Score: 15.5, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572718' target='https://doi.org/10.1101/2023.12.20.572718'> 10.1101/2023.12.20.572718</a></p>
        <p class="abstract">Protein SUMOylation provides a principal driving force for cellular stress responses including DNA-protein crosslink (DPC) repair and arsenic-induced PML body degradation. In genome-scale screens, we identified the human E3 ligase TOPORS as a key effector of SUMO-dependent DPC resolution. We demonstrate that TOPORS promotes DPC repair by functioning as a SUMO-targeted ubiquitin ligase (STUbL) for DPCs, combining ubiquitin ligase activity through its RING domain with poly-SUMO chain binding via a cluster of SUMO-interacting motifs, analogous to the STUbL RNF4. Surprisingly, the STUbL activities of TOPORS and RNF4 are both required for SUMO-dependent DPC repair, PML degradation and other stress responses, making overlapping and distinct contributions to ubiquitin chain formation on SUMOylated targets to enable p97/VCP unfoldase recruitment. Combined loss of TOPORS and RNF4 is synthetic lethal even in unstressed cells, leading to defective clearance of SUMOylated proteins from chromatin accompanied by cell cycle arrest and apoptosis. Together, our findings establish TOPORS as a novel STUbL whose concerted action with RNF4 defines a general mechanistic principle in crucial cellular processes governed by direct SUMO-ubiquitin crosstalk.

HighlightsO_LIThe RING E3 ligase TOPORS is required for SUMO-dependent DPC repair
C_LIO_LITOPORS is a novel SUMO-targeted ubiquitin ligase (STUbL)
C_LIO_LITOPORS promotes multiple STUbL-driven processes in conjunction with RNF4
C_LIO_LICombined TOPORS and RNF4 loss is synthetic lethal in human cells
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571777">
        <p class="paperTitle">Rapid DNA unwinding accelerates genome editing by engineered CRISPR-Cas9</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eggers, A. R.; Chen, K.; Soczek, K. M.; Tuck, O. T.; Doherty, E. E.; Thornton, B. W.; Xu, B.; Trinidad, M. I.; Doudna, J. A.</p>
        <p class="info">Score: 34.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571777' target='https://doi.org/10.1101/2023.12.14.571777'> 10.1101/2023.12.14.571777</a></p>
        <p class="abstract">Thermostable CRISPR-Cas9 enzymes could improve genome editing efficiency and delivery due to extended protein lifetimes. However, initial experimentation demonstrated Geobacillus stearothermophilus Cas9 (GeoCas9) to be virtually inactive when used in cultured human cells. Laboratory-evolved variants of GeoCas9 overcome this natural limitation by acquiring mutations in the wedge (WED) domain that produce &gt;100-fold higher genome editing levels. Cryo-EM structures of the wildtype and improved GeoCas9 (iGeoCas9) enzymes reveal extended contacts between the WED domain of iGeoCas9 and DNA substrates. Biochemical analysis shows that iGeoCas9 accelerates DNA unwinding to capture substrates under the magnesium-restricted conditions typical of mammalian but not bacterial cells. These findings enabled rational engineering of other Cas9 orthologs to enhance genome editing levels, pointing to a general strategy for editing enzyme improvement. Together, these results uncover a new role for the Cas9 WED domain in DNA unwinding and demonstrate how accelerated target unwinding dramatically improves Cas9-induced genome editing activity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
