<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment
Authors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.
Score: 22.0, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550782
Here we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment
Authors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.
Score: 22.0, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550782
Here we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment
Authors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.
Score: 22.0, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550782
Here we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment\nAuthors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.\nScore: 22.0, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550782\nHere we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3.",
  "keywords": [
    
  ],
  "articleBody": " Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment\nAuthors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.\nScore: 22.0, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550782\nHere we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3.4 [A]. The complex was generated by cotranslational insertion into preformed nanodisc membranes using cell-free synthesis in E. coli lysates. It is the first structure obtained by this detergent-free strategy and the first GPCR/Gs complex structure in lipid environment. Structural comparison with the inactive conformation of H2R and the inactive and Gq-coupled active state of H1R together with structure-guided functional experiments reveal molecular insights into the specificity of ligand binding and G protein coupling for this receptor family. We demonstrate lipid-modulated folding of cell-free synthesized H2R, its agonist-dependent internalization and its interaction with endogenously synthesized H1R and H2R in HEK293 cells by applying a recently developed nanotransfer technique.\nSystematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function\nAuthors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.\nScore: 20.6, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550466\nCatalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits. Individual deletions of subunits of either CERF, RSF, ACF, WICH or NoRC subcomplexes only moderately affect the chromatin landscape, while removal of the NURF-specific subunit BPTF leads to drastic reduction in chromatin accessibility and SNF2H ATPase localization around CTCF sites. While this affects adjacent nucleosome occupancy, it only modestly impacts CTCF binding itself. In the absence of accessibility, the structural function of CTCF is nevertheless impaired resulting in lower occupancy of cohesin and cohesin release factor, and reduced physical insulation at these sites, highlighting the need of NURF-mediated remodeling for open chromatin and proper CTCF function. These results separate local CTCF binding from insulator function in nuclear organization and reveal a specific role for NURF in mediating SNF2H localization and chromatin opening at bound CTCF sites. They designate local accessibility as critical for cohesin positioning and establishment of physical insulation.\nCyanobacterial α-carboxysome carbonic anhydrase is allosterically regulated by the Rubisco substrate RuBP\nAuthors: Pulsford, S. B.; Outram, M. A.; Forster, B.; Rhodes, T.; Williams, S. J.; Badger, M. R.; Price, G. D.; Jackson, C. J.; Long, B. M.\nScore: 11.8, Published: 2023-07-31 DOI: 10.1101/2023.07.31.551272\nCyanobacterial CO2 concentrating mechanisms (CCMs) sequester a globally significant proportion of carbon into the biosphere. Proteinaceous microcompartments, called carboxysomes, play a critical role in CCM function, housing two enzymes to enhance CO2 fixation: carbonic anhydrase (CA) and Rubisco. Despite its importance, our current understanding of the carboxysomal CAs found in -cyanobacteria, CsoSCA, remains limited, particularly regarding the regulation of its activity. Here, we present the first structural and biochemical study of CsoSCA from the cyanobacterium Cyanobium PCC7001. Our results show that the Cyanobium CsoSCA is allosterically activated by the Rubisco substrate ribulose-1,5-bisphosphate (RuBP), and forms a hexameric trimer of dimers. Comprehensive phylogenetic and mutational analyses are consistent with this regulation appearing exclusively in cyanobacterial -carboxysome CAs. These findings clarify the biologically relevant oligomeric state of -carboxysomal CAs and advance our understanding of the regulation of photosynthesis in this globally dominant lineage.\nVCF1 is an unconventional p97/VCP cofactor promoting recognition of ubiquitylated p97-UFD1-NPL4 substrates\nAuthors: Mirsanaye, A. S.; Hoffmann, S.; Weisser, M. B.; Mund, A.; Lopez-Mendez, B.; Typas, D.; van den Boom, J.; Benedict, B.; Hendriks, I. A.; Nielsen, M. L.; Meyer, H.; Duxin, J. P.; Montoya, G.; Mailand, N.\nScore: 8.5, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550448\nThe hexameric AAA+ ATPase p97/VCP functions as an essential mediator of ubiquitin-dependent cellular processes, extracting ubiquitylated proteins from macromolecular complexes or membranes by catalyzing their unfolding. p97 is directed to ubiquitylated client proteins via multiple cofactors, most of which interact with the p97 N-domain. Here, we discovered that FAM104A, a protein of unknown function that we named VCF1 (VCP/p97 Cofactor FAM104 1), acts as a novel p97 cofactor in human cells. Detailed structure-function studies revealed that VCF1 directly binds p97 via a conserved novel -helical motif that recognizes the p97 N-domain with unusually high affinity, exceeding that of other cofactors. We show that VCF1 engages in joint p97 complex formation with the heterodimeric primary p97 cofactor UFD1-NPL4 and promotes p97-UFD1-NPL4-dependent proteasomal degradation of ubiquitylated substrates in cells. Mechanistically, VCF1 indirectly stimulates UFD1-NPL4 interactions with ubiquitin conjugates via its binding to p97 but has no intrinsic affinity for ubiquitin. Collectively, our findings establish VCF1 as an unconventional p97 cofactor that promotes p97-dependent protein turnover by facilitating p97-UFD1-NPL4 recruitment to ubiquitylated targets.\nSmart Leginon enables cryoEM screening automation across multiple grids\nAuthors: Sawh-Gopal, A.; Ishemgulova, A.; Chua, E. Y. D.; Aragon, M. F.; Mendez, J. H.; Eng, E. T.; Noble, A. J.\nScore: 7.5, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550406\nAdvancements in cryo-electron microscopy (cryoEM) techniques over the past decade have allowed structural biologists to routinely resolve macromolecular protein complexes to near-atomic resolution. The general workflow of the entire cryoEM pipeline involves iterating between sample preparation, cryoEM grid preparation, and sample/grid screening before moving on to high-resolution data collection. Iterating between sample/grid preparation and screening is typically a major bottleneck for researchers, as every iterative experiment must optimize for sample concentration, buffer conditions, grid material, grid hole size, ice thickness, and protein particle behavior in the ice, amongst other variables. Furthermore, once these variables are satisfactorily determined, grids prepared under identical conditions vary widely in whether they are ready for data collection, so additional screening sessions prior to selecting optimal grids for high-resolution data collection are recommended. This sample/grid preparation and screening process often consumes several dozen grids and days of operator time at the microscope. Furthermore, the screening process is limited to operator/microscope availability and microscope accessibility. Here, we demonstrate how to use Leginon and Smart Leginon Autoscreen to automate the majority of cryoEM grid screening. Autoscreen combines machine learning, computer vision algorithms, and microscope-handling algorithms to remove the need for constant manual operator input. Autoscreen can autonomously load and image grids with multi-scale imaging using an automated specimen-exchange cassette system, resulting in unattended grid screening for an entire cassette. As a result, operator time for screening 12 grids may be reduced to [~]10 minutes with Autoscreen compared to [~]6 hours using previous methods which are hampered by their inability to account for high variability between grids. This protocol and video tutorial first introduces basic Leginon setup and functionality, then demonstrates Autoscreen functionality step-by-step from the creation of a template session to the end of a 12 grid automated screening session. SummaryCryoEM multi-grid screening is often a tedious process that demands hours of attention. Here, this protocol shows how to set up standard Leginon collection and Smart Leginon Autoscreen to automate this process. This protocol can be applied to the majority of cryoEM holey foil grids.\nZNF512B binds RBBP4 via a variant NuRD interaction motif and aggregates chromatin in a NuRD complex-independent manner\nAuthors: Wunderlich, T. M.; Deshpande, C.; Paasche, L. W.; Friedrich, T.; Diegmueller, F.; Daus, N.; Naseer, H.; Stebel, S. E.; Leers, J.; Lan, J.; Trinh, V. T.; Vazquez, O.; Butter, F.; Bartkuhn, M.; Mackay, J. P.; Hake, S. B.\nScore: 6.3, Published: 2023-07-31 DOI: 10.1101/2023.07.31.551264\nThe evolutionarily conserved histone variant H2A.Z plays a crucial role in various DNA-based processes but the underlying mechanisms by which it acts are not completely understood. Recently, we identified the zinc finger protein ZNF512B as an H2A.Z-, HMG20A- and PWWP2A-associated protein. Here, we report that ZNF512B binds the nucleosome remodeling and deacetylase (NuRD) complex. We discover a conserved amino acid sequence within ZNF512B that resembles the NuRD-interaction motif (NIM) previously identified in FOG-1 and other transcriptional regulators. By solving the crystal structure of this motif bound to the NuRD component RBBP4 and by applying several biochemical assays we demonstrate that this internal NIM is both necessary and sufficient for robust NuRD binding. Transcriptome analyses and reporter assays identify ZNF512B as a repressor of gene expression that can act in both NuRD-dependent and -independent ways. Surprisingly, high levels of ZNF512B expression lead to nuclear protein and chromatin aggregation foci that form independent of the interaction with the NuRD complex but depend on the zinc finger domains of ZNF512B. Our study has implications for diseases in which ZNF512B expression is deregulated, such as cancer and neurodegenerative diseases, and hint at the existence of more proteins as potential NuRD interactors.\nInteraction of SCoV-2 NSP7 or NSP8 alone with NSP12 causes constriction of the RNA entry channel: Implications for novel RdRp inhibitor drug discovery\nAuthors: Singh, D.; Kushwaha, T.; Kulandaisamy, R.; Kumar, V.; Baswal, K.; Tiwari, S. H.; Ghorai, A.; Kumar, M.; Kumar, S.; Polamarasetty, A.; Sehgal, D.; Katika, M. R.; Gadde, S.; Cote, M.; Kayampeta, S. R.; Appaiahgari, M. B.; Inampudi, K. K.\nScore: 5.9, Published: 2023-07-31 DOI: 10.1101/2023.07.26.550660\nRNA-dependent RNA polymerase (RdRP) is a critical component of the RNA virus life cycle, including SCoV-2. Among the Coronavirus-encoded proteins, non-structural protein 12 (NSP12) exhibits polymerase activity in collaboration with one unit of NSP7 and two units of NSP8, constituting the RdRp holoenzyme. While there is abundant information on SCoV-2 RdRp-mediated RNA replication, the influence of interplay among NSP12, NSP7, and NSP8 on template RNA binding and primer extension activity remains relatively unexplored and poorly understood. Here, we recreated a functional RdRp holoenzyme in vitro using recombinant SCoV-2 NSP12, NSP7, and NSP8, and established its functional activity. Subsequently, molecular interactions among the NSPs in the presence of a variety of templates and their effects on polymerase activity were studied, wherein we found that NSP12 alone exhibited notable polymerase activity that increased significantly in the presence of NSP7 and NSP8. However, this activity was completely shut down, and the template RNA-primer complex was detached from NSP12 when one of the two cofactors was present. Through computational analysis, we found that the template RNA entry channel was more constricted in the presence of one of the two cofactors, which was relatively more constricted in the presence of NSP8 compared to that in the presence of NSP7. In conclusion, we report that NSP7 and NSP8 together synergise to enhance the activity of NSP12, but antagonise when present alone. Our findings have implications for novel drug development, and compounds that obstruct the binding of NSP7 or NSP8 to NSP12 can have lethal effects on viral RNA replication.\nLeucine suppresses glucagon secretion from pancreatic islets by directly modulating alpha-cell cAMP\nAuthors: Knuth, E. R.; Foster, H. R.; Jin, E.; Merrins, M. J.\nScore: 4.8, Published: 2023-07-31 DOI: 10.1101/2023.07.31.551113\nObjective: Pancreatic islets are nutrient sensors that regulate organismal blood glucose homeostasis. Glucagon release from the pancreatic alpha-cell is important under fasted, fed, and hypoglycemic conditions, yet metabolic regulation of alpha-cells remains poorly understood. Here, we identified a previously unexplored role for physiological levels of leucine, which is classically regarded as a beta-cell fuel, in the intrinsic regulation of alpha-cell glucagon release. Methods: GcgCreERT:CAMPER and GcgCreERT:GCaMP6s mice were generated to perform dynamic, high-throughput functional measurements of alpha-cell cAMP and Ca2+ within the intact islet. Islet perifusion assays were used for simultaneous, time-resolved measurements of glucagon and insulin release from mouse and human islets. The effects of leucine were compared with glucose and the mitochondrial fuels 2-aminobicyclo(2,2,1)heptane-2-carboxylic acid (BCH, non-metabolized leucine analog that activates glutamate dehydrogenase), alpha-ketoisocaproate (KIC, leucine metabolite), and methyl-succinate (complex II fuel). CYN154806 (Sstr2 antagonist), diazoxide (KATP activator, which prevents Ca2+-dependent exocytosis from alpha, beta, and delta-cells), and dispersed alpha-cells were used to inhibit islet paracrine signaling and identify alpha-cell intrinsic effects. Results: Mimicking the effect of glucose, leucine strongly suppressed amino acid-stimulated glucagon secretion. Mechanistically, leucine dose-dependently reduced alpha-cell cAMP at physiological concentrations, with an IC50 of 57, 440, and 1162 uM at 2, 6, and 10 mM glucose, without affecting alpha-cell Ca2+. Leucine also reduced alpha-cell cAMP in islets treated with Sstr2 antagonist or diazoxide, as well as dispersed alpha-cells, indicating an alpha-cell intrinsic effect. The effect of leucine was matched by KIC and the glutamate dehydrogenase activator BCH, but not methyl-succinate, indicating a dependence on mitochondrial anaplerosis. Glucose, which stimulates anaplerosis via pyruvate carboxylase, had the same suppressive effect on alpha-cell cAMP but with lower potency. Similarly to mouse islets, leucine suppressed glucagon secretion from human islets under hypoglycemic conditions. Conclusions: These findings highlight an important role for physiological levels of leucine in the metabolic regulation of alpha-cell cAMP and glucagon secretion. Leucine functions primarily through an alpha-cell intrinsic effect that is dependent on glutamate dehydrogenase, in addition to the well-established alpha-cell regulation by beta/delta-cell paracrine signaling. Our results suggest that mitochondrial anaplerosis-cataplerosis facilitates the glucagonostatic effect of both leucine and glucose, which cooperatively suppress alpha-cell tone by reducing cAMP.\nMulti-omic temporal landscape of plasma and synovial fluid-derived extracellular vesicles using an experimental model of equine osteoarthritis\nAuthors: Anderson, J. R.; Johnson, E.; Jenkins, R.; Jacobsen, S.; Green, D.; Walters, M.; Bundgaard, L.; Hausmans, B. A.; van den Akker, G.; Welting, T. J.; Chabronova, A.; Kharaz, Y. A.; Clarke, E. J.; James, V.; Peffers, M. J.\nScore: 4.3, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551464\nExtracellular vesicles contribute to osteoarthritis pathogenesis through their release into joint tissues and synovial fluid. Limited studies have profiled extracellular vesicles in osteoarthritic biofluids, such as plasma and synovial fluid. Due to the potential involvement in osteoarthritis pathogenesis, synovial fluid-derived extracellular vesicles have the potential to be direct biomarkers in the causal pathway of disease but also enable understanding of their role in disease progression. Utilizing a temporal model of early osteoarthritis, we defined the changes in matched synovial fluid and plasma-derived extracellular vesicle small non-coding RNA and protein cargo using small RNA sequencing and mass spectrometry proteomics. We explored the data with a multi-omic approach including time series clustering, factor analysis and gene enrichment interrogation. Chondrocyte signalling induced by temporal synovial fluid-derived extracellular vesicles derived from the model were analysed using luciferase-based transcription factor activity assays. Extracellular vesicle protein cargo appears to be more important during osteoarthritis progression than small non-coding RNA cargo. Cluster analysis revealed plasma-extracellular vesicles represented a time-dependant response to osteoarthritis induction, were principally derived from protein cargo and were associated with supramolecular complexes. Clusters for synovial fluid-derived extracellular vesicles were associated with an initial osteoarthritis response and represented immune/inflammatory pathways. Factor analysis revealed that plasma-derived extracellular vesicles correlated with day post induction and were primarily composed of proteins which may modulate lipid metabolism in osteoarthritis. Synovial fluid-derived extracellular vesicles significant factors represented intermediate filament and supramolecular complexes reflecting tissue repair responses to osteoarthritis induction. There was a significant interaction between time and osteoarthritis for cAMP response element, Nuclear factor-kappa B response element, serum response element and serum response factor response element reporters with a trend for osteoarthritis synovial fluid-derived EVs at later time points to have a more pronounced effect. Local and systemic osteoarthritis-associated changes in extracellular vesicle cargo profiles in this in vivo model provided a unique opportunity to understand their role in disease propagation and progression and may represent novel biomarkers to stage osteoarthritis.\nCryoEM insights into RNA primer synthesis by the human primosome\nAuthors: Yin, Z.; Kilkenny, M. L.; Ker, D.-S.; Pellegrini, L.\nScore: 5.5, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549859\nEukaryotic DNA replication depends on the primosome - a complex of DNA polymerase alpha (Pol ) and primase - to initiate DNA synthesis by polymerisation of an RNA - DNA primer. Primer synthesis requires the tight coordination of primase and polymerase activities. Recent cryo-electron microscopy (cryoEM) analyses have elucidated the extensive conformational transitions required for RNA primer handover between primase and Pol and primer elongation by Pol . Because of the intrinsic flexibility of the primosome however, structural information about the initiation of RNA primer synthesis is still lacking. Here, we capture cryoEM snapshots of the priming reaction to reveal the conformational trajectory of the human primosome that brings the PRIM1 and PRIM2 subunits of primase together, poised for RNA synthesis. Furthermore, we provide experimental evidence for the continuous association of primase subunit PRIM2 with the RNA primer during primer synthesis and for how both initiation and termination of RNA primer polymerisation are licensed by specific rearrangements of POLA1, the polymerase subunit of Pol . Our findings fill a critical gap in our understanding of the conformational changes that underpin the synthesis of the RNA primer by the primosome. Together with existing evidence, they provide a complete description of the structural dynamics of the human primosome during DNA replication initiation.\n",
  "wordCount" : "2783",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550782">
        <p class="paperTitle">Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.</p>
        <p class="info">Score: 22.0, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550782' target='https://doi.org/10.1101/2023.07.27.550782'> 10.1101/2023.07.27.550782</a></p>
        <p class="abstract">Here we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3.4 [A]. The complex was generated by cotranslational insertion into preformed nanodisc membranes using cell-free synthesis in E. coli lysates. It is the first structure obtained by this detergent-free strategy and the first GPCR/Gs complex structure in lipid environment. Structural comparison with the inactive conformation of H2R and the inactive and Gq-coupled active state of H1R together with structure-guided functional experiments reveal molecular insights into the specificity of ligand binding and G protein coupling for this receptor family. We demonstrate lipid-modulated folding of cell-free synthesized H2R, its agonist-dependent internalization and its interaction with endogenously synthesized H1R and H2R in HEK293 cells by applying a recently developed nanotransfer technique.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550466">
        <p class="paperTitle">Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.</p>
        <p class="info">Score: 20.6, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550466' target='https://doi.org/10.1101/2023.07.25.550466'> 10.1101/2023.07.25.550466</a></p>
        <p class="abstract">Catalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits. Individual deletions of subunits of either CERF, RSF, ACF, WICH or NoRC subcomplexes only moderately affect the chromatin landscape, while removal of the NURF-specific subunit BPTF leads to drastic reduction in chromatin accessibility and SNF2H ATPase localization around CTCF sites. While this affects adjacent nucleosome occupancy, it only modestly impacts CTCF binding itself. In the absence of accessibility, the structural function of CTCF is nevertheless impaired resulting in lower occupancy of cohesin and cohesin release factor, and reduced physical insulation at these sites, highlighting the need of NURF-mediated remodeling for open chromatin and proper CTCF function.

These results separate local CTCF binding from insulator function in nuclear organization and reveal a specific role for NURF in mediating SNF2H localization and chromatin opening at bound CTCF sites. They designate local accessibility as critical for cohesin positioning and establishment of physical insulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551272">
        <p class="paperTitle">Cyanobacterial α-carboxysome carbonic anhydrase is allosterically regulated by the Rubisco substrate RuBP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pulsford, S. B.; Outram, M. A.; Forster, B.; Rhodes, T.; Williams, S. J.; Badger, M. R.; Price, G. D.; Jackson, C. J.; Long, B. M.</p>
        <p class="info">Score: 11.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551272' target='https://doi.org/10.1101/2023.07.31.551272'> 10.1101/2023.07.31.551272</a></p>
        <p class="abstract">Cyanobacterial CO2 concentrating mechanisms (CCMs) sequester a globally significant proportion of carbon into the biosphere. Proteinaceous microcompartments, called carboxysomes, play a critical role in CCM function, housing two enzymes to enhance CO2 fixation: carbonic anhydrase (CA) and Rubisco. Despite its importance, our current understanding of the carboxysomal CAs found in -cyanobacteria, CsoSCA, remains limited, particularly regarding the regulation of its activity. Here, we present the first structural and biochemical study of CsoSCA from the cyanobacterium Cyanobium PCC7001. Our results show that the Cyanobium CsoSCA is allosterically activated by the Rubisco substrate ribulose-1,5-bisphosphate (RuBP), and forms a hexameric trimer of dimers. Comprehensive phylogenetic and mutational analyses are consistent with this regulation appearing exclusively in cyanobacterial -carboxysome CAs. These findings clarify the biologically relevant oligomeric state of -carboxysomal CAs and advance our understanding of the regulation of photosynthesis in this globally dominant lineage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550448">
        <p class="paperTitle">VCF1 is an unconventional p97/VCP cofactor promoting recognition of ubiquitylated p97-UFD1-NPL4 substrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mirsanaye, A. S.; Hoffmann, S.; Weisser, M. B.; Mund, A.; Lopez-Mendez, B.; Typas, D.; van den Boom, J.; Benedict, B.; Hendriks, I. A.; Nielsen, M. L.; Meyer, H.; Duxin, J. P.; Montoya, G.; Mailand, N.</p>
        <p class="info">Score: 8.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550448' target='https://doi.org/10.1101/2023.07.25.550448'> 10.1101/2023.07.25.550448</a></p>
        <p class="abstract">The hexameric AAA&#43; ATPase p97/VCP functions as an essential mediator of ubiquitin-dependent cellular processes, extracting ubiquitylated proteins from macromolecular complexes or membranes by catalyzing their unfolding. p97 is directed to ubiquitylated client proteins via multiple cofactors, most of which interact with the p97 N-domain. Here, we discovered that FAM104A, a protein of unknown function that we named VCF1 (VCP/p97 Cofactor FAM104 1), acts as a novel p97 cofactor in human cells. Detailed structure-function studies revealed that VCF1 directly binds p97 via a conserved novel -helical motif that recognizes the p97 N-domain with unusually high affinity, exceeding that of other cofactors. We show that VCF1 engages in joint p97 complex formation with the heterodimeric primary p97 cofactor UFD1-NPL4 and promotes p97-UFD1-NPL4-dependent proteasomal degradation of ubiquitylated substrates in cells. Mechanistically, VCF1 indirectly stimulates UFD1-NPL4 interactions with ubiquitin conjugates via its binding to p97 but has no intrinsic affinity for ubiquitin. Collectively, our findings establish VCF1 as an unconventional p97 cofactor that promotes p97-dependent protein turnover by facilitating p97-UFD1-NPL4 recruitment to ubiquitylated targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550406">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550406" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550406">
        <p class="paperTitle">Smart Leginon enables cryoEM screening automation across multiple grids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550406" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550406" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sawh-Gopal, A.; Ishemgulova, A.; Chua, E. Y. D.; Aragon, M. F.; Mendez, J. H.; Eng, E. T.; Noble, A. J.</p>
        <p class="info">Score: 7.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550406' target='https://doi.org/10.1101/2023.07.24.550406'> 10.1101/2023.07.24.550406</a></p>
        <p class="abstract">Advancements in cryo-electron microscopy (cryoEM) techniques over the past decade have allowed structural biologists to routinely resolve macromolecular protein complexes to near-atomic resolution. The general workflow of the entire cryoEM pipeline involves iterating between sample preparation, cryoEM grid preparation, and sample/grid screening before moving on to high-resolution data collection. Iterating between sample/grid preparation and screening is typically a major bottleneck for researchers, as every iterative experiment must optimize for sample concentration, buffer conditions, grid material, grid hole size, ice thickness, and protein particle behavior in the ice, amongst other variables. Furthermore, once these variables are satisfactorily determined, grids prepared under identical conditions vary widely in whether they are ready for data collection, so additional screening sessions prior to selecting optimal grids for high-resolution data collection are recommended. This sample/grid preparation and screening process often consumes several dozen grids and days of operator time at the microscope. Furthermore, the screening process is limited to operator/microscope availability and microscope accessibility. Here, we demonstrate how to use Leginon and Smart Leginon Autoscreen to automate the majority of cryoEM grid screening. Autoscreen combines machine learning, computer vision algorithms, and microscope-handling algorithms to remove the need for constant manual operator input. Autoscreen can autonomously load and image grids with multi-scale imaging using an automated specimen-exchange cassette system, resulting in unattended grid screening for an entire cassette. As a result, operator time for screening 12 grids may be reduced to [~]10 minutes with Autoscreen compared to [~]6 hours using previous methods which are hampered by their inability to account for high variability between grids. This protocol and video tutorial first introduces basic Leginon setup and functionality, then demonstrates Autoscreen functionality step-by-step from the creation of a template session to the end of a 12 grid automated screening session.

SummaryCryoEM multi-grid screening is often a tedious process that demands hours of attention. Here, this protocol shows how to set up standard Leginon collection and Smart Leginon Autoscreen to automate this process. This protocol can be applied to the majority of cryoEM holey foil grids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551264">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551264" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551264">
        <p class="paperTitle">ZNF512B binds RBBP4 via a variant NuRD interaction motif and aggregates chromatin in a NuRD complex-independent manner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551264" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551264" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wunderlich, T. M.; Deshpande, C.; Paasche, L. W.; Friedrich, T.; Diegmueller, F.; Daus, N.; Naseer, H.; Stebel, S. E.; Leers, J.; Lan, J.; Trinh, V. T.; Vazquez, O.; Butter, F.; Bartkuhn, M.; Mackay, J. P.; Hake, S. B.</p>
        <p class="info">Score: 6.3, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551264' target='https://doi.org/10.1101/2023.07.31.551264'> 10.1101/2023.07.31.551264</a></p>
        <p class="abstract">The evolutionarily conserved histone variant H2A.Z plays a crucial role in various DNA-based processes but the underlying mechanisms by which it acts are not completely understood. Recently, we identified the zinc finger protein ZNF512B as an H2A.Z-, HMG20A- and PWWP2A-associated protein. Here, we report that ZNF512B binds the nucleosome remodeling and deacetylase (NuRD) complex. We discover a conserved amino acid sequence within ZNF512B that resembles the NuRD-interaction motif (NIM) previously identified in FOG-1 and other transcriptional regulators. By solving the crystal structure of this motif bound to the NuRD component RBBP4 and by applying several biochemical assays we demonstrate that this internal NIM is both necessary and sufficient for robust NuRD binding. Transcriptome analyses and reporter assays identify ZNF512B as a repressor of gene expression that can act in both NuRD-dependent and -independent ways. Surprisingly, high levels of ZNF512B expression lead to nuclear protein and chromatin aggregation foci that form independent of the interaction with the NuRD complex but depend on the zinc finger domains of ZNF512B. Our study has implications for diseases in which ZNF512B expression is deregulated, such as cancer and neurodegenerative diseases, and hint at the existence of more proteins as potential NuRD interactors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550660" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550660">
        <p class="paperTitle">Interaction of SCoV-2 NSP7 or NSP8 alone with NSP12 causes constriction of the RNA entry channel: Implications for novel RdRp inhibitor drug discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, D.; Kushwaha, T.; Kulandaisamy, R.; Kumar, V.; Baswal, K.; Tiwari, S. H.; Ghorai, A.; Kumar, M.; Kumar, S.; Polamarasetty, A.; Sehgal, D.; Katika, M. R.; Gadde, S.; Cote, M.; Kayampeta, S. R.; Appaiahgari, M. B.; Inampudi, K. K.</p>
        <p class="info">Score: 5.9, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550660' target='https://doi.org/10.1101/2023.07.26.550660'> 10.1101/2023.07.26.550660</a></p>
        <p class="abstract">RNA-dependent RNA polymerase (RdRP) is a critical component of the RNA virus life cycle, including SCoV-2. Among the Coronavirus-encoded proteins, non-structural protein 12 (NSP12) exhibits polymerase activity in collaboration with one unit of NSP7 and two units of NSP8, constituting the RdRp holoenzyme. While there is abundant information on SCoV-2 RdRp-mediated RNA replication, the influence of interplay among NSP12, NSP7, and NSP8 on template RNA binding and primer extension activity remains relatively unexplored and poorly understood. Here, we recreated a functional RdRp holoenzyme in vitro using recombinant SCoV-2 NSP12, NSP7, and NSP8, and established its functional activity. Subsequently, molecular interactions among the NSPs in the presence of a variety of templates and their effects on polymerase activity were studied, wherein we found that NSP12 alone exhibited notable polymerase activity that increased significantly in the presence of NSP7 and NSP8. However, this activity was completely shut down, and the template RNA-primer complex was detached from NSP12 when one of the two cofactors was present. Through computational analysis, we found that the template RNA entry channel was more constricted in the presence of one of the two cofactors, which was relatively more constricted in the presence of NSP8 compared to that in the presence of NSP7. In conclusion, we report that NSP7 and NSP8 together synergise to enhance the activity of NSP12, but antagonise when present alone. Our findings have implications for novel drug development, and compounds that obstruct the binding of NSP7 or NSP8 to NSP12 can have lethal effects on viral RNA replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551113">
        <p class="paperTitle">Leucine suppresses glucagon secretion from pancreatic islets by directly modulating alpha-cell cAMP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knuth, E. R.; Foster, H. R.; Jin, E.; Merrins, M. J.</p>
        <p class="info">Score: 4.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551113' target='https://doi.org/10.1101/2023.07.31.551113'> 10.1101/2023.07.31.551113</a></p>
        <p class="abstract">Objective: Pancreatic islets are nutrient sensors that regulate organismal blood glucose homeostasis. Glucagon release from the pancreatic alpha-cell is important under fasted, fed, and hypoglycemic conditions, yet metabolic regulation of alpha-cells remains poorly understood. Here, we identified a previously unexplored role for physiological levels of leucine, which is classically regarded as a beta-cell fuel, in the intrinsic regulation of alpha-cell glucagon release. Methods: GcgCreERT:CAMPER and GcgCreERT:GCaMP6s mice were generated to perform dynamic, high-throughput functional measurements of alpha-cell cAMP and Ca2&#43; within the intact islet. Islet perifusion assays were used for simultaneous, time-resolved measurements of glucagon and insulin release from mouse and human islets. The effects of leucine were compared with glucose and the mitochondrial fuels 2-aminobicyclo(2,2,1)heptane-2-carboxylic acid (BCH, non-metabolized leucine analog that activates glutamate dehydrogenase), alpha-ketoisocaproate (KIC, leucine metabolite), and methyl-succinate (complex II fuel). CYN154806 (Sstr2 antagonist), diazoxide (KATP activator, which prevents Ca2&#43;-dependent exocytosis from alpha, beta, and delta-cells), and dispersed alpha-cells were used to inhibit islet paracrine signaling and identify alpha-cell intrinsic effects. Results: Mimicking the effect of glucose, leucine strongly suppressed amino acid-stimulated glucagon secretion. Mechanistically, leucine dose-dependently reduced alpha-cell cAMP at physiological concentrations, with an IC50 of 57, 440, and 1162 uM at 2, 6, and 10 mM glucose, without affecting alpha-cell Ca2&#43;. Leucine also reduced alpha-cell cAMP in islets treated with Sstr2 antagonist or diazoxide, as well as dispersed alpha-cells, indicating an alpha-cell intrinsic effect. The effect of leucine was matched by KIC and the glutamate dehydrogenase activator BCH, but not methyl-succinate, indicating a dependence on mitochondrial anaplerosis. Glucose, which stimulates anaplerosis via pyruvate carboxylase, had the same suppressive effect on alpha-cell cAMP but with lower potency. Similarly to mouse islets, leucine suppressed glucagon secretion from human islets under hypoglycemic conditions. Conclusions: These findings highlight an important role for physiological levels of leucine in the metabolic regulation of alpha-cell cAMP and glucagon secretion. Leucine functions primarily through an alpha-cell intrinsic effect that is dependent on glutamate dehydrogenase, in addition to the well-established alpha-cell regulation by beta/delta-cell paracrine signaling. Our results suggest that mitochondrial anaplerosis-cataplerosis facilitates the glucagonostatic effect of both leucine and glucose, which cooperatively suppress alpha-cell tone by reducing cAMP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551464">
        <p class="paperTitle">Multi-omic temporal landscape of plasma and synovial fluid-derived extracellular vesicles using an experimental model of equine osteoarthritis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderson, J. R.; Johnson, E.; Jenkins, R.; Jacobsen, S.; Green, D.; Walters, M.; Bundgaard, L.; Hausmans, B. A.; van den Akker, G.; Welting, T. J.; Chabronova, A.; Kharaz, Y. A.; Clarke, E. J.; James, V.; Peffers, M. J.</p>
        <p class="info">Score: 4.3, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551464' target='https://doi.org/10.1101/2023.08.01.551464'> 10.1101/2023.08.01.551464</a></p>
        <p class="abstract">Extracellular vesicles contribute to osteoarthritis pathogenesis through their release into joint tissues and synovial fluid. Limited studies have profiled extracellular vesicles in osteoarthritic biofluids, such as plasma and synovial fluid. Due to the potential involvement in osteoarthritis pathogenesis, synovial fluid-derived extracellular vesicles have the potential to be direct biomarkers in the causal pathway of disease but also enable understanding of their role in disease progression. Utilizing a temporal model of early osteoarthritis, we defined the changes in matched synovial fluid and plasma-derived extracellular vesicle small non-coding RNA and protein cargo using small RNA sequencing and mass spectrometry proteomics. We explored the data with a multi-omic approach including time series clustering, factor analysis and gene enrichment interrogation. Chondrocyte signalling induced by temporal synovial fluid-derived extracellular vesicles derived from the model were analysed using luciferase-based transcription factor activity assays. Extracellular vesicle protein cargo appears to be more important during osteoarthritis progression than small non-coding RNA cargo. Cluster analysis revealed plasma-extracellular vesicles represented a time-dependant response to osteoarthritis induction, were principally derived from protein cargo and were associated with supramolecular complexes. Clusters for synovial fluid-derived extracellular vesicles were associated with an initial osteoarthritis response and represented immune/inflammatory pathways. Factor analysis revealed that plasma-derived extracellular vesicles correlated with day post induction and were primarily composed of proteins which may modulate lipid metabolism in osteoarthritis. Synovial fluid-derived extracellular vesicles significant factors represented intermediate filament and supramolecular complexes reflecting tissue repair responses to osteoarthritis induction. There was a significant interaction between time and osteoarthritis for cAMP response element, Nuclear factor-kappa B response element, serum response element and serum response factor response element reporters with a trend for osteoarthritis synovial fluid-derived EVs at later time points to have a more pronounced effect. Local and systemic osteoarthritis-associated changes in extracellular vesicle cargo profiles in this in vivo model provided a unique opportunity to understand their role in disease propagation and progression and may represent novel biomarkers to stage osteoarthritis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549859">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549859" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549859">
        <p class="paperTitle">CryoEM insights into RNA primer synthesis by the human primosome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549859" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549859" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yin, Z.; Kilkenny, M. L.; Ker, D.-S.; Pellegrini, L.</p>
        <p class="info">Score: 5.5, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549859' target='https://doi.org/10.1101/2023.07.20.549859'> 10.1101/2023.07.20.549859</a></p>
        <p class="abstract">Eukaryotic DNA replication depends on the primosome - a complex of DNA polymerase alpha (Pol ) and primase - to initiate DNA synthesis by polymerisation of an RNA - DNA primer. Primer synthesis requires the tight coordination of primase and polymerase activities. Recent cryo-electron microscopy (cryoEM) analyses have elucidated the extensive conformational transitions required for RNA primer handover between primase and Pol  and primer elongation by Pol . Because of the intrinsic flexibility of the primosome however, structural information about the initiation of RNA primer synthesis is still lacking. Here, we capture cryoEM snapshots of the priming reaction to reveal the conformational trajectory of the human primosome that brings the PRIM1 and PRIM2 subunits of primase together, poised for RNA synthesis. Furthermore, we provide experimental evidence for the continuous association of primase subunit PRIM2 with the RNA primer during primer synthesis and for how both initiation and termination of RNA primer polymerisation are licensed by specific rearrangements of POLA1, the polymerase subunit of Pol . Our findings fill a critical gap in our understanding of the conformational changes that underpin the synthesis of the RNA primer by the primosome. Together with existing evidence, they provide a complete description of the structural dynamics of the human primosome during DNA replication initiation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
