<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses
Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.
Score: 243.6, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871
TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses
Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.
Score: 243.6, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871
TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses
Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.
Score: 243.6, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871
TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses\nAuthors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.\nScore: 243.6, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871\nTnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown.",
  "keywords": [
    
  ],
  "articleBody": " Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses\nAuthors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.\nScore: 243.6, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871\nTnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.\nPreserved respiratory chain capacity and physiology in mice with profoundly reduced levels of mitochondrial respirasomes\nAuthors: Milenkovic, D.; Misic, J.; Hevler, J. F.; Molinie, T.; Chung, I.; Atanassov, I.; Li, X.; Filograna, R.; Mesaros, A.; Mourier, A.; Heck, A. J. R.; Hirst, J.; Larsson, N.-G.\nScore: 35.1, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545560\nThe mammalian respiratory chain complexes I, III2 and IV (CI, CIII2 and CIV) are critical for cellular bioenergetics and form a stable assembly, the respirasome (CI- CIII2-CIV), that is biochemically and structurally well documented. The role of the respirasome in bioenergetics and regulation of metabolism is subject to intense debate and is difficult to study because the individual respiratory chain complexes coexist together with high levels of respirasomes. To critically investigate the in vivo role of the respirasome, we generated homozygous knock-in mice that have normal levels of respiratory chain complexes but profoundly decreased levels of respirasomes. Surprisingly, the mutant mice are healthy, with preserved respiratory chain capacity and normal exercise performance. Our findings show that high levels of respirasomes are dispensable for maintaining bioenergetics and physiology in the mouse, but raises questions about their alternate functions, such as relating to regulation of protein stability and prevention of age-associated protein aggregation.\nEfficient Formation of Single-copy Human Artificial Chromosomes\nAuthors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.\nScore: 29.0, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547284\nLarge DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through ~125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a ~750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.\nChemogenetic profiling of ubiquitin-like modifier pathways identifies NFATC2IP as a mediator of SUMO-dependent genome integrity\nAuthors: Cho, T.; Zhao, Y.; Olivieri, M.; Hoeg, L.; Setiaputra, D.; Durocher, D.\nScore: 14.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547196\nThe post-translational modification of proteins by ubiquitin and ubiquitin-like polypeptides controls multiple cellular processes including the abundance of a large fraction of the proteome. We applied genome-scale CRISPR/Cas9 screens to elucidate the genetic architecture of the response to inhibition of ubiquitin, NEDD8 and SUMO conjugation pathways as well as inhibition of the p97/VCP segregase. This effort identified 395 genes whose disruption alters the fitness of human cells when faced with perturbations in these pathways. We validated that the TMED2 and TMED10 proteins, which are localized to the secretory pathway, promote resistance to p97/VCP inhibition and also characterized NFATC2IP, an evolutionarily conserved protein harboring SUMO-like domains as a major player in promoting genomic integrity when SUMOylation is inhibited. We propose that NFATC2IP acts in interphase cells to promote the SUMO-dependent E3 ligase activity of the SMC5/SMC6 complex, which is critical for SUMO-dependent genome integrity.\nMulti-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation\nAuthors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.\nScore: 13.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546730\nPartial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.\nDNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor\nAuthors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.\nScore: 21.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516\nThe human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin. However, how HUSH and MORC2 operate alongside DNA methylation, a central epigenetic regulator of repeat transcription, remains poorly understood. Here we interrogate this relationship in human neural progenitor cells (hNPCs), a somatic model of brain development that tolerates removal of DNA methyltransferase DNMT1. Upon loss of MORC2 or HUSH subunit TASOR in hNPCs, L1s remain silenced by robust promoter methylation. However, genome demethylation and activation of evolutionarily-young L1s attracts MORC2 binding. Simultaneous depletion of DNMT1 and MORC2 causes massive accumulation of L1 transcripts. We identify the same mechanistic hierarchy at pericentromeric -satellites and clustered protocadherin genes, repetitive elements important for chromosome structure and neurodevelopment respectively. Our data delineate the independent epigenetic control of repeats in somatic cells, with implications for understanding the vital functions of HUSH-MORC2 in hypomethylated contexts throughout human development.\nA Spatiotemporal Proteomic Map of Human Adipogenesis\nAuthors: Klingelhuber, F.; Frendo-Cumbo, S.; Hmeadi, O.; Taylor, A. J.; Kakimoto, P.; Ribicic, S.; Couchet, M.; Wabitsch, M.; Mueller, T. D.; Ryden, M.; Mejhert, N.; Krahmer, N.\nScore: 10.9, Published: 2023-07-03 DOI: 10.1101/2023.07.01.547321\nWhite adipocytes function as the major energy reservoir in humans by storing substantial amounts of triglycerides and their dysfunction is associated with metabolic disorders. However, the mechanisms underlying cellular specialization during adipogenesis remain unknown. Here, using a high-sensitivity-high throughput workflow, we generated a spatiotemporal proteomic atlas of human adipogenesis encompassing information for ~8.000 proteins. Our systematic approach gives insights into cellular remodeling and the spatial reorganization of metabolic pathways to optimize cells for lipid accumulation and highlights the coordinated regulation of protein localization and abundance during adipogenesis. More specifically, we identified a compartment-specific regulation of protein levels and localization changes of metabolic enzymes to reprogram branched chain amino acid and one-carbon metabolism to provide building blocks and reduction equivalents for lipid synthesis. Additionally, we identified C19orf12 as a differentiation induced and adipocyte-specific lipid droplet (LD) protein, which interacts with the translocase of the outer membrane (TOM) complex of LD associated mitochondria and regulates adipocyte lipid storage. Overall, our study provides a comprehensive resource for understanding human adipogenesis and for future discoveries in the field.\nDiscovery of novel DNA cytosine deaminase activities enables a nondestructive single-enzyme methylation sequencing method for base resolution high-coverage methylome mapping of cell-free and ultra-low input DNA\nAuthors: Vaisvila, R.; Johnson, S. R.; Yan, B.; Dai, N.; Bourkia, B. M.; Correa, I. R.; Yigit, E.; Sun, Z.\nScore: 10.6, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547047\nCytosine deaminases have important uses in the detection of epigenetic modifications and in genome editing. However, the range of applications of deaminases is limited by a small number of well characterized enzymes. To expand the toolkit of deaminases, we developed an in-vitro approach that bypasses a major hurdle with their severe toxicity in expression hosts. We systematically assayed the activity of 175 putative cytosine deaminases on an unprecedented variety of substrates with epigenetically relevant base modifications. We found enzymes with high activity on double- and single-stranded DNA in various sequence contexts including novel CpG-specific deaminases, as well as enzymes without sequence preference. We also report, for the first time, enzymes that do not deaminate modified cytosines. The remarkable diversity of cytosine deaminases opens new avenues for biotechnological and medical applications. Using a newly discovered non-specific, modification-sensitive double-stranded DNA deaminase, we developed a nondestructive single-enzyme 5-methylctyosine sequencing (SEM-seq) method. SEM-seq enables accurate, high-coverage, base-resolution methylome mapping of scarce biological material including clinically relevant cell-free DNA (cfDNA) and single-cell equivalent 10 pg input DNA. Using SEM-seq, we generated highly reproducible base-resolution 5mC maps, accounting for nearly 80% of CpG islands for a low input human cfDNA sample offering valuable information for identifying potential biomarkers for detection of early-stage cancer and other diseases. This streamlined protocol will enable robust, high-throughput, high-coverage epigenome profiling of challenging samples in research and clinical settings.\nIdentification of fish spawning based on size-selected sampling and diurnal concentration changes of environmental DNA resulting from sperm-release\nAuthors: Tsuji, S.; Shibata, N.\nScore: 9.8, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545639\nEnvironmental DNA (eDNA) analysis is a useful method for detecting spawning behaviour by observing elevated eDNA concentrations and/or nuclear/mitochondrial DNA ratios. However, little is known about the characteristics and dynamics of sperm-derived eDNA, which is key to detection. This study focused on changes in eDNA particle size distribution (PSD) and concentrations pre-, post-, and during spawning of Plecoglossus altivelis, to investigate the potential for semi-selective recovery of sperm-derived eDNA and the importance of sampling time for improving detection sensitivity. Firstly, PSD changes between pre- and post-spawning were investigated by comparing concentrations and proportions of eDNA obtained from filters with different pore sizes. Secondly, the diurnal changes in eDNA concentration were monitored at the peak of the spawning season by collecting river water every 1-3 hours. Results showed that eDNA related to sperm-head size increased at post-spawning, and eDNA concentrations had significant diurnal changes with a peak during the spawning time window. These findings suggest that semi-selective recovery of sperm-derived eDNA based on particle size and/or sampling during the spawning time window with increased concentrations can improve the detection sensitivity of eDNA-based spawning surveys. This study provides essential fundamental information for advancing eDNA-based spawning surveys and contributes to their further development. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=107 SRC=\"FIGDIR/small/545639v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@134432dorg.highwire.dtl.DTLVardef@267657org.highwire.dtl.DTLVardef@1236480org.highwire.dtl.DTLVardef@9b3503_HPS_FORMAT_FIGEXP M_FIG C_FIG\nThe Unexpected Match: STAT3-NORAD Interaction, a Novel Link in Antiviral Defense\nAuthors: Argoetti, A.; Shalev, D.; Polyak, G.; Shima, N.; Biran, H.; Lahav, T.; Hashimshony, T.; Mandel-Gutfreund, Y.\nScore: 8.1, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546999\nIn the intricate biological landscape, evolution often repurposes familiar elements for novel roles. This study reveals an interaction between the long non-coding RNA NORAD, noted for its role in DNA stability, and the immune-related transcription factor STAT3. Our findings indicate that NORADs binding to STAT3 facilitates its nuclear entry, suppressing the antiviral response. In the absence of NORAD, STAT3 remains cytoplasmic, enabling STAT1 to activate this response. Evidence from viral infections and clinical samples reinforces this unique role for NORAD. Intriguingly, while other functions of NORAD are conserved in evolution, this newly discovered role is unique to humans, owing to the introduction of an ALU element in hominoids. This discovery sheds new light on the evolution of antiviral defenses.\n",
  "wordCount" : "2136",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544871">
        <p class="paperTitle">Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.</p>
        <p class="info">Score: 243.6, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544871' target='https://doi.org/10.1101/2023.06.13.544871'> 10.1101/2023.06.13.544871</a></p>
        <p class="abstract">TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545560">
        <p class="paperTitle">Preserved respiratory chain capacity and physiology in mice with profoundly reduced levels of mitochondrial respirasomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Milenkovic, D.; Misic, J.; Hevler, J. F.; Molinie, T.; Chung, I.; Atanassov, I.; Li, X.; Filograna, R.; Mesaros, A.; Mourier, A.; Heck, A. J. R.; Hirst, J.; Larsson, N.-G.</p>
        <p class="info">Score: 35.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545560' target='https://doi.org/10.1101/2023.06.19.545560'> 10.1101/2023.06.19.545560</a></p>
        <p class="abstract">The mammalian respiratory chain complexes I, III2 and IV (CI, CIII2 and CIV) are critical for cellular bioenergetics and form a stable assembly, the respirasome (CI- CIII2-CIV), that is biochemically and structurally well documented. The role of the respirasome in bioenergetics and regulation of metabolism is subject to intense debate and is difficult to study because the individual respiratory chain complexes coexist together with high levels of respirasomes. To critically investigate the in vivo role of the respirasome, we generated homozygous knock-in mice that have normal levels of respiratory chain complexes but profoundly decreased levels of respirasomes. Surprisingly, the mutant mice are healthy, with preserved respiratory chain capacity and normal exercise performance. Our findings show that high levels of respirasomes are dispensable for maintaining bioenergetics and physiology in the mouse, but raises questions about their alternate functions, such as relating to regulation of protein stability and prevention of age-associated protein aggregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547284">
        <p class="paperTitle">Efficient Formation of Single-copy Human Artificial Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.</p>
        <p class="info">Score: 29.0, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547284' target='https://doi.org/10.1101/2023.06.30.547284'> 10.1101/2023.06.30.547284</a></p>
        <p class="abstract">Large DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through ~125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a ~750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547196" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547196">
        <p class="paperTitle">Chemogenetic profiling of ubiquitin-like modifier pathways identifies NFATC2IP as a mediator of SUMO-dependent genome integrity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cho, T.; Zhao, Y.; Olivieri, M.; Hoeg, L.; Setiaputra, D.; Durocher, D.</p>
        <p class="info">Score: 14.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547196' target='https://doi.org/10.1101/2023.06.30.547196'> 10.1101/2023.06.30.547196</a></p>
        <p class="abstract">The post-translational modification of proteins by ubiquitin and ubiquitin-like polypeptides controls multiple cellular processes including the abundance of a large fraction of the proteome. We applied genome-scale CRISPR/Cas9 screens to elucidate the genetic architecture of the response to inhibition of ubiquitin, NEDD8 and SUMO conjugation pathways as well as inhibition of the p97/VCP segregase. This effort identified 395 genes whose disruption alters the fitness of human cells when faced with perturbations in these pathways. We validated that the TMED2 and TMED10 proteins, which are localized to the secretory pathway, promote resistance to p97/VCP inhibition and also characterized NFATC2IP, an evolutionarily conserved protein harboring SUMO-like domains as a major player in promoting genomic integrity when SUMOylation is inhibited. We propose that NFATC2IP acts in interphase cells to promote the SUMO-dependent E3 ligase activity of the SMC5/SMC6 complex, which is critical for SUMO-dependent genome integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546730">
        <p class="paperTitle">Multi-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.</p>
        <p class="info">Score: 13.1, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546730' target='https://doi.org/10.1101/2023.06.30.546730'> 10.1101/2023.06.30.546730</a></p>
        <p class="abstract">Partial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545516">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545516" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545516">
        <p class="paperTitle">DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545516" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545516" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.</p>
        <p class="info">Score: 21.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545516' target='https://doi.org/10.1101/2023.06.21.545516'> 10.1101/2023.06.21.545516</a></p>
        <p class="abstract">The human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin. However, how HUSH and MORC2 operate alongside DNA methylation, a central epigenetic regulator of repeat transcription, remains poorly understood. Here we interrogate this relationship in human neural progenitor cells (hNPCs), a somatic model of brain development that tolerates removal of DNA methyltransferase DNMT1. Upon loss of MORC2 or HUSH subunit TASOR in hNPCs, L1s remain silenced by robust promoter methylation. However, genome demethylation and activation of evolutionarily-young L1s attracts MORC2 binding. Simultaneous depletion of DNMT1 and MORC2 causes massive accumulation of L1 transcripts. We identify the same mechanistic hierarchy at pericentromeric -satellites and clustered protocadherin genes, repetitive elements important for chromosome structure and neurodevelopment respectively. Our data delineate the independent epigenetic control of repeats in somatic cells, with implications for understanding the vital functions of HUSH-MORC2 in hypomethylated contexts throughout human development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.547321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.547321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.547321">
        <p class="paperTitle">A Spatiotemporal Proteomic Map of Human Adipogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.547321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.547321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klingelhuber, F.; Frendo-Cumbo, S.; Hmeadi, O.; Taylor, A. J.; Kakimoto, P.; Ribicic, S.; Couchet, M.; Wabitsch, M.; Mueller, T. D.; Ryden, M.; Mejhert, N.; Krahmer, N.</p>
        <p class="info">Score: 10.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.547321' target='https://doi.org/10.1101/2023.07.01.547321'> 10.1101/2023.07.01.547321</a></p>
        <p class="abstract">White adipocytes function as the major energy reservoir in humans by storing substantial amounts of triglycerides and their dysfunction is associated with metabolic disorders. However, the mechanisms underlying cellular specialization during adipogenesis remain unknown. Here, using a high-sensitivity-high throughput workflow, we generated a spatiotemporal proteomic atlas of human adipogenesis encompassing information for ~8.000 proteins. Our systematic approach gives insights into cellular remodeling and the spatial reorganization of metabolic pathways to optimize cells for lipid accumulation and highlights the coordinated regulation of protein localization and abundance during adipogenesis. More specifically, we identified a compartment-specific regulation of protein levels and localization changes of metabolic enzymes to reprogram branched chain amino acid and one-carbon metabolism to provide building blocks and reduction equivalents for lipid synthesis. Additionally, we identified C19orf12 as a differentiation induced and adipocyte-specific lipid droplet (LD) protein, which interacts with the translocase of the outer membrane (TOM) complex of LD associated mitochondria and regulates adipocyte lipid storage. Overall, our study provides a comprehensive resource for understanding human adipogenesis and for future discoveries in the field.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547047">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547047" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547047">
        <p class="paperTitle">Discovery of novel DNA cytosine deaminase activities enables a nondestructive single-enzyme methylation sequencing method for base resolution high-coverage methylome mapping of cell-free and ultra-low input DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547047" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547047" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vaisvila, R.; Johnson, S. R.; Yan, B.; Dai, N.; Bourkia, B. M.; Correa, I. R.; Yigit, E.; Sun, Z.</p>
        <p class="info">Score: 10.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547047' target='https://doi.org/10.1101/2023.06.29.547047'> 10.1101/2023.06.29.547047</a></p>
        <p class="abstract">Cytosine deaminases have important uses in the detection of epigenetic modifications and in genome editing. However, the range of applications of deaminases is limited by a small number of well characterized enzymes. To expand the toolkit of deaminases, we developed an in-vitro approach that bypasses a major hurdle with their severe toxicity in expression hosts. We systematically assayed the activity of 175 putative cytosine deaminases on an unprecedented variety of substrates with epigenetically relevant base modifications. We found enzymes with high activity on double- and single-stranded DNA in various sequence contexts including novel CpG-specific deaminases, as well as enzymes without sequence preference. We also report, for the first time, enzymes that do not deaminate modified cytosines. The remarkable diversity of cytosine deaminases opens new avenues for biotechnological and medical applications. Using a newly discovered non-specific, modification-sensitive double-stranded DNA deaminase, we developed a nondestructive single-enzyme 5-methylctyosine sequencing (SEM-seq) method. SEM-seq enables accurate, high-coverage, base-resolution methylome mapping of scarce biological material including clinically relevant cell-free DNA (cfDNA) and single-cell equivalent 10 pg input DNA. Using SEM-seq, we generated highly reproducible base-resolution 5mC maps, accounting for nearly 80% of CpG islands for a low input human cfDNA sample offering valuable information for identifying potential biomarkers for detection of early-stage cancer and other diseases. This streamlined protocol will enable robust, high-throughput, high-coverage epigenome profiling of challenging samples in research and clinical settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545639">
        <p class="paperTitle">Identification of fish spawning based on size-selected sampling and diurnal concentration changes of environmental DNA resulting from sperm-release</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsuji, S.; Shibata, N.</p>
        <p class="info">Score: 9.8, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545639' target='https://doi.org/10.1101/2023.06.19.545639'> 10.1101/2023.06.19.545639</a></p>
        <p class="abstract">Environmental DNA (eDNA) analysis is a useful method for detecting spawning behaviour by observing elevated eDNA concentrations and/or nuclear/mitochondrial DNA ratios. However, little is known about the characteristics and dynamics of sperm-derived eDNA, which is key to detection. This study focused on changes in eDNA particle size distribution (PSD) and concentrations pre-, post-, and during spawning of Plecoglossus altivelis, to investigate the potential for semi-selective recovery of sperm-derived eDNA and the importance of sampling time for improving detection sensitivity. Firstly, PSD changes between pre- and post-spawning were investigated by comparing concentrations and proportions of eDNA obtained from filters with different pore sizes. Secondly, the diurnal changes in eDNA concentration were monitored at the peak of the spawning season by collecting river water every 1-3 hours. Results showed that eDNA related to sperm-head size increased at post-spawning, and eDNA concentrations had significant diurnal changes with a peak during the spawning time window. These findings suggest that semi-selective recovery of sperm-derived eDNA based on particle size and/or sampling during the spawning time window with increased concentrations can improve the detection sensitivity of eDNA-based spawning surveys. This study provides essential fundamental information for advancing eDNA-based spawning surveys and contributes to their further development.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=107 SRC=&#34;FIGDIR/small/545639v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@134432dorg.highwire.dtl.DTLVardef@267657org.highwire.dtl.DTLVardef@1236480org.highwire.dtl.DTLVardef@9b3503_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546999">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546999" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546999">
        <p class="paperTitle">The Unexpected Match: STAT3-NORAD Interaction, a Novel Link in Antiviral Defense</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546999" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546999" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Argoetti, A.; Shalev, D.; Polyak, G.; Shima, N.; Biran, H.; Lahav, T.; Hashimshony, T.; Mandel-Gutfreund, Y.</p>
        <p class="info">Score: 8.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546999' target='https://doi.org/10.1101/2023.06.29.546999'> 10.1101/2023.06.29.546999</a></p>
        <p class="abstract">In the intricate biological landscape, evolution often repurposes familiar elements for novel roles. This study reveals an interaction between the long non-coding RNA NORAD, noted for its role in DNA stability, and the immune-related transcription factor STAT3. Our findings indicate that NORADs binding to STAT3 facilitates its nuclear entry, suppressing the antiviral response. In the absence of NORAD, STAT3 remains cytoplasmic, enabling STAT1 to activate this response. Evidence from viral infections and clinical samples reinforces this unique role for NORAD. Intriguingly, while other functions of NORAD are conserved in evolution, this newly discovered role is unique to humans, owing to the introduction of an ALU element in hominoids. This discovery sheds new light on the evolution of antiviral defenses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
