<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA
Authors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.
Score: 112.0, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565735
We recently identified glycoRNA-a previously undescribed glycoconjugate-which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA
Authors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.
Score: 112.0, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565735
We recently identified glycoRNA-a previously undescribed glycoconjugate-which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA
Authors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.
Score: 112.0, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565735
We recently identified glycoRNA-a previously undescribed glycoconjugate-which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA\nAuthors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.\nScore: 112.0, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565735\nWe recently identified glycoRNA-a previously undescribed glycoconjugate-which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described.",
  "keywords": [
    
  ],
  "articleBody": " The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA\nAuthors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.\nScore: 112.0, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565735\nWe recently identified glycoRNA-a previously undescribed glycoconjugate-which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described. Here we develop a sensitive and scalable protocol to detect and characterize native glycoRNAs. Leveraging periodate oxidation and aldehyde ligation (rPAL) and Sequential Window Acquisition of all Theoretical Mass Spectra (SWATH-MS), we identified the modified RNA base 3-(3-amino-3-carboxypropyl)uridine (acp3U) as a site of attachment of N-glycans in glycoRNA. The sensitivity and robustness of rPAL provided the first evidence of a direct glycan-RNA linkage, and its flexibility will enable further characterization of glycoRNA biology.\nStructural basis for plasmid restriction by SMC JET nuclease\nAuthors: Roisne-Hamelin, F.; Liu, H. W.; Taschner, M.; Li, Y.; Gruber, S.\nScore: 38.8, Published: 2023-11-03 DOI: 10.1101/2023.11.03.565488\nDNA loop-extruding SMC complexes play crucial roles in chromosome folding and DNA immunity. Prokaryotic SMC Wadjet (JET) complexes limit the spread of circular plasmids through DNA cleavage; yet the mechanisms for target recognition are unresolved. We show that artificial DNA circularization renders linear DNA susceptible to JET cleavage. Unlike free DNA, JET cleaves immobilized plasmid DNA at a specific site, the plasmid-anchoring point, showing that the anchor hinders DNA extrusion but not DNA cleavage implying that residual unextruded DNA is cleaved. Structures of plasmid-bound JetABC reveal two presumably stalled SMC motor units that are drastically rearranged from the resting state, together entrapping a U-shaped DNA segment, which is further converted to kinked V-shaped cleavage substrate by JetD nuclease binding. Our findings uncover mechanical bending of residual unextruded DNA as principle for non-self DNA recognition and molecular signature for plasmid cleavage. We elucidate key elements of SMC loop extrusion including motor directionality and the structure of a DNA-holding state.\nLARP1 senses free ribosomes to coordinate supply and demand of ribosomal proteins\nAuthors: Saba, J. A.; Huang, Z.; Schole, K. L.; Ye, X.; Bhatt, S. D.; Li, Y.; Timp, W.; Cheng, J.; Green, R.\nScore: 22.5, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565189\nTerminal oligopyrimidine motif-containing mRNAs (TOPs) encode all ribosomal proteins in mammals and are regulated to tune ribosome synthesis to cell state. Previous studies implicate LARP1 in 40S- or 80S-ribosome complexes that repress and stabilize TOPs. However, a mechanistic understanding of how LARP1 and TOPs interact with these complexes to coordinate TOP outcomes is lacking. Here, we show that LARP1 senses the cellular supply of ribosomes by directly binding non-translating ribosomal subunits. Cryo-EM structures reveal a previously uncharacterized domain of LARP1 bound to and occluding the 40S mRNA channel. Free cytosolic ribosomes induce sequestration of TOPs in repressed 80S-LARP1-TOP complexes independent of alterations in mTOR signaling. Together, this work demonstrates a general ribosome-sensing function of LARP1 that allows it to tune ribosome protein synthesis to cellular demand. One-Sentence SummaryLARP1 directly binds free ribosomal subunits to repress TOP mRNAs\nqTAG: An adaptable CRISPR-based endogenous tagging protocol using optimized repair cassettes\nAuthors: Philip, R.; Sharma, A.; Matellan, L.; Erpf, A. C.; Hsu, W.-H.; Tkach, J. M.; Wyatt, H. D. M.; Pelletier, L.\nScore: 18.3, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565029\nEndogenous tagging makes it possible to study a proteins localization, dynamics, and function within its native regulatory context by genetically inserting a sequence encoding a functional tag into the reading frame of a gene. Here, we introduce the \"quickTAG,\" or qTAG system, a versatile collection of optimized repair cassettes designed to make CRISPR-mediated tagging more accessible. By partitioning the cassette to include both a desired tag sequence linked with a selectable marker, integrations can be quickly isolated post-editing. These constructs also include several key features that enhance flexibility and ease of use, such as: cassette designs for N- and C-terminus tagging; standardized cloning adaptors to simplify the incorporation of homology arms for HDR or MMEJ-based repairs; restriction sites next to each genetic element within the cassette for easy switching of tags and selectable markers; and the inclusion of lox sites flanking the selectable marker to allow marker gene removal following integration. Our library of [~]120 plasmids is available on Addgene and includes ready-to-use constructs targeting frequently overexpressed genes as well as ready-to-clone cassettes to tag your own genes of interest. These cassettes were engineered to harbor a variety of tags for fluorescence imaging, proximity-dependent biotinylation, epitope tagging, and targeted protein degradation. Alongside a comprehensive protocol, the qTAG system will provide a framework to streamline endogenous tagging in addition to serving as an open resource for researchers to adapt and tailor for their own purposes.\nThe human mitochondrial mRNA structurome reveals mechanisms of gene expression\nAuthors: Moran, J. C.; Brivanlou, A.; Brischigliaro, M.; Fontanesi, F.; Rouskin, S.; Barrientos, A.\nScore: 14.9, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564750\nThe mammalian mitochondrial genome encodes thirteen oxidative phosphorylation system proteins, crucial in aerobic energy transduction. These proteins are translated from 9 monocistronic and 2 bicistronic transcripts, whose native structures remain unexplored, leaving fundamental molecular determinants of mitochondrial gene expression unknown. To address this gap, we developed a mitoDMS-MaPseq approach and used DREEM clustering to resolve the native human mitochondrial mt-mRNA structurome. We gained insights into mt-mRNA biology and translation regulatory mechanisms, including a unique programmed ribosomal frameshifting for the ATP8/ATP6 transcript. Furthermore, absence of the mt-mRNA maintenance factor LRPPRC led to a mitochondrial transcriptome structured differently, with specific mRNA regions exhibiting increased or decreased structuredness. This highlights the role of LRPPRC in maintaining mRNA folding to promote mt-mRNA stabilization and efficient translation. In conclusion, our mt-mRNA folding maps reveal novel mitochondrial gene expression mechanisms, serving as a detailed reference and tool for studying them in different physiological and pathological contexts.\nHuman promoter directionality is determined by transcriptional initiation and the opposing activities of INTS11 and CDK9\nAuthors: Eaton, J. D.; Board, J.; Davidson, L.; Estell, C.; West, S.\nScore: 11.0, Published: 2023-11-02 DOI: 10.1101/2023.11.02.565285\nRNA polymerase II (RNAPII) transcription initiates bidirectionally at many human protein-coding genes. Sense transcription usually dominates and leads to messenger RNA production, whereas antisense transcription rapidly terminates. The basis for this directionality is not fully understood. Here, we show that sense transcriptional initiation is more efficient than in the antisense direction, which provides initial directionality. After transcription begins, the opposing functions of Integrator (INTS11) and cyclin-dependent kinase 9 (CDK9) maintain directionality. INTS11 terminates antisense transcription, whereas sense transcription is protected from INTS11-dependent attenuation by CDK9. INTS11 terminates transcription in both directions upon CDK9 inhibition, and the engineered recruitment of CDK9 prevents attenuation by INTS11. Therefore, transcriptional initiation and the opposing activities of CDK9 and INTS11 explain the attenuation of antisense transcription, the more extensive nature of sense transcription, and promoter directionality.\nMulti-modal skin atlas identifies a multicellular immune-stromal community associated with altered cornification and specific T cell expansion in atopic dermatitis\nAuthors: Fiskin, E.; Eraslan, G.; Alora-Palli, M. B.; Leyva-Castillo, J. M.; Kim, S.; Choe, H.; Lareau, C. A.; Lau, H.; Finan, E. P.; Teixeira-Soldano, I.; LaBere, B.; Chu, A.; Woods, B.; Chou, J.; Slyper, M.; Waldman, J.; Islam, S.; Schneider, L.; Phipatanakul, W.; Platt, C.; Rozenblatt-Rosen, O.; Delorey, T. M.; Deguine, J.; Smith, G. P.; Geha, R. S.; Regev, A.; Xavier, R. J.\nScore: 10.9, Published: 2023-10-31 DOI: 10.1101/2023.10.29.563503\nIn healthy skin, a cutaneous immune system maintains the balance between tolerance towards innocuous environmental antigens and immune responses against pathological agents. In atopic dermatitis (AD), barrier and immune dysfunction result in chronic tissue inflammation. Our understanding of the skin tissue ecosystem in AD remains incomplete with regard to the hallmarks of pathological barrier formation, and cellular state and clonal composition of disease-promoting cells. Here, we generated a multi-modal cell census of 310,691 cells spanning 86 cell subsets from whole skin tissue of 19 adult individuals, including non-lesional and lesional skin from 11 AD patients, and integrated it with 396,321 cells from four studies into a comprehensive human skin cell atlas in health and disease. Reconstruction of human keratinocyte differentiation from basal to cornified layers revealed a disrupted cornification trajectory in AD. This disrupted epithelial differentiation was associated with signals from a unique immune and stromal multicellular community comprised of MMP12+ dendritic cells (DCs), mature migratory DCs, cycling ILCs, NK cells, inflammatory CCL19+ IL4I1+ fibroblasts, and clonally expanded IL13+IL22+IL26+ T cells with overlapping type 2 and type 17 characteristics. Cell subsets within this immune and stromal multicellular community were connected by multiple inter-cellular positive feedback loops predicted to impact community assembly and maintenance. AD GWAS gene expression was enriched both in disrupted cornified keratinocytes and in cell subsets from the lesional immune and stromal multicellular community including IL13+IL22+IL26+ T cells and ILCs, suggesting that epithelial or immune dysfunction in the context of the observed cellular communication network can initiate and then converge towards AD. Our work highlights specific, disease-associated cell subsets and interactions as potential targets in progression and resolution of chronic inflammation.\nPervasive translational control of photosynthesis genes during photomorphogenesis is acquired by C4 genes\nAuthors: Llorens, I. R.; Lastovka, F.; Schreier, T. B.; Singh, P.; Chung, B. Y.; Hibberd, J. M.\nScore: 9.5, Published: 2023-10-27 DOI: 10.1101/2023.10.27.563924\nC4 photosynthesis allows increased efficiency and has evolved in more than sixty-six plant lineages. Underpinning this repeated appearance of the C4 pathway is a major transcriptional reprogramming of photosynthesis genes. Here we investigated whether evolution has also significantly modified translational control by defining the translational dynamics of C3 rice and C4 sorghum during photomorphogenesis. In the dark rice photosynthesis transcripts are low abundance but highly translated. After exposure to light translational efficiency declines. The same phenomena occur in sorghum but in addition C4 cycle genes show this response. We propose a model in which translational control of photosynthesis genes permits a rapid response to light and that this translational regulation is gained by C4 genes during the evolution of the C4 pathway.\nStructural basis of RfaH-mediated transcription-translation coupling\nAuthors: Molodtsov, V.; Wang, C.; Kaelber, J. T.; Blaha, G.; Ebright, R. H.\nScore: 5.3, Published: 2023-11-06 DOI: 10.1101/2023.11.05.565726\nThe NusG paralog RfaH mediates bacterial transcription-translation coupling on genes that contain a DNA sequence element, termed an ops site, required for pausing RNA polymerase (RNAP) and for loading RfaH onto the paused RNAP. Here we report cryo-EM structures of transcription-translation complexes (TTCs) containing RfaH. The results show that RfaH bridges RNAP and the ribosome, with the RfaH N-terminal domain interacting with RNAP, and with the RfaH C-terminal domain interacting with the ribosome. The results show that the distribution of translational and orientational positions of RNAP relative to the ribosome in RfaH-coupled TTCs is more restricted than in NusG-coupled TTCs, due to the more restricted flexibility of the RfaH interdomain linker. The results further show that the structural organization of RfaH-coupled TTCs in the \"loading state,\" in which RNAP and RfaH are located at the ops site during formation of the TTC, is the same as the structural organization of RfaH-coupled TTCs in the \"loaded state,\" in which RNAP and RfaH are located at positions downstream of the ops site during function of the TTC. The results define the structural organization of RfaH-containing TTCs and set the stage for analysis of functions of RfaH during translation initiation and transcription-translation coupling.\nBrazilin is a Natural Product Inhibitor of the NLRP3 Inflammasome\nAuthors: McMahon, E. A.; El-Sayed, S.; Green, J. P.; Hoyle, C.; Fitzpatrick, L.; Jones, E.; Corrie, E.; Kelly, R. L.; Challinor, M.; Freeman, S.; Bryce, R. A.; Lawrence, C. B.; Brough, D.; Kasher, P. R.\nScore: 6.0, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564348\nExcessive or aberrant NLRP3 inflammasome activation has been implicated in the progression and initiation of many inflammatory conditions; however, currently no NLRP3 inflammasome inhibitors have been approved for therapeutic use in the clinic. Here we have identified that the natural product brazilin effectively inhibits both priming and activation of the NLRP3 inflammasome in cultured murine macrophages, a human iPSC microglial cell line and in a mouse model of acute peritoneal inflammation. Through computational modelling, we predict that brazilin can adopt a favourable binding pose within a site of the NLRP3 protein which is essential for its conformational activation. Our results not only encourage further evaluation of brazilin as a therapeutic agent for NLRP3-related inflammatory diseases, but also introduce this small-molecule as a promising scaffold structure for the development of derivative NLRP3 inhibitor compounds.\n",
  "wordCount" : "2071",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565735">
        <p class="paperTitle">The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.</p>
        <p class="info">Score: 112.0, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565735' target='https://doi.org/10.1101/2023.11.06.565735'> 10.1101/2023.11.06.565735</a></p>
        <p class="abstract">We recently identified glycoRNA-a previously undescribed glycoconjugate-which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described. Here we develop a sensitive and scalable protocol to detect and characterize native glycoRNAs. Leveraging periodate oxidation and aldehyde ligation (rPAL) and Sequential Window Acquisition of all Theoretical Mass Spectra (SWATH-MS), we identified the modified RNA base 3-(3-amino-3-carboxypropyl)uridine (acp3U) as a site of attachment of N-glycans in glycoRNA. The sensitivity and robustness of rPAL provided the first evidence of a direct glycan-RNA linkage, and its flexibility will enable further characterization of glycoRNA biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565488">
        <p class="paperTitle">Structural basis for plasmid restriction by SMC JET nuclease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roisne-Hamelin, F.; Liu, H. W.; Taschner, M.; Li, Y.; Gruber, S.</p>
        <p class="info">Score: 38.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565488' target='https://doi.org/10.1101/2023.11.03.565488'> 10.1101/2023.11.03.565488</a></p>
        <p class="abstract">DNA loop-extruding SMC complexes play crucial roles in chromosome folding and DNA immunity. Prokaryotic SMC Wadjet (JET) complexes limit the spread of circular plasmids through DNA cleavage; yet the mechanisms for target recognition are unresolved. We show that artificial DNA circularization renders linear DNA susceptible to JET cleavage. Unlike free DNA, JET cleaves immobilized plasmid DNA at a specific site, the plasmid-anchoring point, showing that the anchor hinders DNA extrusion but not DNA cleavage implying that residual unextruded DNA is cleaved. Structures of plasmid-bound JetABC reveal two presumably stalled SMC motor units that are drastically rearranged from the resting state, together entrapping a U-shaped DNA segment, which is further converted to kinked V-shaped cleavage substrate by JetD nuclease binding. Our findings uncover mechanical bending of residual unextruded DNA as principle for non-self DNA recognition and molecular signature for plasmid cleavage. We elucidate key elements of SMC loop extrusion including motor directionality and the structure of a DNA-holding state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565189">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565189" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565189">
        <p class="paperTitle">LARP1 senses free ribosomes to coordinate supply and demand of ribosomal proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565189" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565189" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saba, J. A.; Huang, Z.; Schole, K. L.; Ye, X.; Bhatt, S. D.; Li, Y.; Timp, W.; Cheng, J.; Green, R.</p>
        <p class="info">Score: 22.5, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565189' target='https://doi.org/10.1101/2023.11.01.565189'> 10.1101/2023.11.01.565189</a></p>
        <p class="abstract">Terminal oligopyrimidine motif-containing mRNAs (TOPs) encode all ribosomal proteins in mammals and are regulated to tune ribosome synthesis to cell state. Previous studies implicate LARP1 in 40S- or 80S-ribosome complexes that repress and stabilize TOPs. However, a mechanistic understanding of how LARP1 and TOPs interact with these complexes to coordinate TOP outcomes is lacking. Here, we show that LARP1 senses the cellular supply of ribosomes by directly binding non-translating ribosomal subunits. Cryo-EM structures reveal a previously uncharacterized domain of LARP1 bound to and occluding the 40S mRNA channel. Free cytosolic ribosomes induce sequestration of TOPs in repressed 80S-LARP1-TOP complexes independent of alterations in mTOR signaling. Together, this work demonstrates a general ribosome-sensing function of LARP1 that allows it to tune ribosome protein synthesis to cellular demand.

One-Sentence SummaryLARP1 directly binds free ribosomal subunits to repress TOP mRNAs</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565029">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565029" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565029">
        <p class="paperTitle">qTAG: An adaptable CRISPR-based endogenous tagging protocol using optimized repair cassettes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565029" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565029" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Philip, R.; Sharma, A.; Matellan, L.; Erpf, A. C.; Hsu, W.-H.; Tkach, J. M.; Wyatt, H. D. M.; Pelletier, L.</p>
        <p class="info">Score: 18.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565029' target='https://doi.org/10.1101/2023.11.01.565029'> 10.1101/2023.11.01.565029</a></p>
        <p class="abstract">Endogenous tagging makes it possible to study a proteins localization, dynamics, and function within its native regulatory context by genetically inserting a sequence encoding a functional tag into the reading frame of a gene. Here, we introduce the &#34;quickTAG,&#34; or qTAG system, a versatile collection of optimized repair cassettes designed to make CRISPR-mediated tagging more accessible. By partitioning the cassette to include both a desired tag sequence linked with a selectable marker, integrations can be quickly isolated post-editing. These constructs also include several key features that enhance flexibility and ease of use, such as: cassette designs for N- and C-terminus tagging; standardized cloning adaptors to simplify the incorporation of homology arms for HDR or MMEJ-based repairs; restriction sites next to each genetic element within the cassette for easy switching of tags and selectable markers; and the inclusion of lox sites flanking the selectable marker to allow marker gene removal following integration. Our library of [~]120 plasmids is available on Addgene and includes ready-to-use constructs targeting frequently overexpressed genes as well as ready-to-clone cassettes to tag your own genes of interest. These cassettes were engineered to harbor a variety of tags for fluorescence imaging, proximity-dependent biotinylation, epitope tagging, and targeted protein degradation. Alongside a comprehensive protocol, the qTAG system will provide a framework to streamline endogenous tagging in addition to serving as an open resource for researchers to adapt and tailor for their own purposes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564750">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564750" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564750">
        <p class="paperTitle">The human mitochondrial mRNA structurome reveals mechanisms of gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564750" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564750" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moran, J. C.; Brivanlou, A.; Brischigliaro, M.; Fontanesi, F.; Rouskin, S.; Barrientos, A.</p>
        <p class="info">Score: 14.9, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564750' target='https://doi.org/10.1101/2023.10.31.564750'> 10.1101/2023.10.31.564750</a></p>
        <p class="abstract">The mammalian mitochondrial genome encodes thirteen oxidative phosphorylation system proteins, crucial in aerobic energy transduction. These proteins are translated from 9 monocistronic and 2 bicistronic transcripts, whose native structures remain unexplored, leaving fundamental molecular determinants of mitochondrial gene expression unknown. To address this gap, we developed a mitoDMS-MaPseq approach and used DREEM clustering to resolve the native human mitochondrial mt-mRNA structurome. We gained insights into mt-mRNA biology and translation regulatory mechanisms, including a unique programmed ribosomal frameshifting for the ATP8/ATP6 transcript. Furthermore, absence of the mt-mRNA maintenance factor LRPPRC led to a mitochondrial transcriptome structured differently, with specific mRNA regions exhibiting increased or decreased structuredness. This highlights the role of LRPPRC in maintaining mRNA folding to promote mt-mRNA stabilization and efficient translation. In conclusion, our mt-mRNA folding maps reveal novel mitochondrial gene expression mechanisms, serving as a detailed reference and tool for studying them in different physiological and pathological contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565285">
        <p class="paperTitle">Human promoter directionality is determined by transcriptional initiation and the opposing activities of INTS11 and CDK9</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eaton, J. D.; Board, J.; Davidson, L.; Estell, C.; West, S.</p>
        <p class="info">Score: 11.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565285' target='https://doi.org/10.1101/2023.11.02.565285'> 10.1101/2023.11.02.565285</a></p>
        <p class="abstract">RNA polymerase II (RNAPII) transcription initiates bidirectionally at many human protein-coding genes. Sense transcription usually dominates and leads to messenger RNA production, whereas antisense transcription rapidly terminates. The basis for this directionality is not fully understood. Here, we show that sense transcriptional initiation is more efficient than in the antisense direction, which provides initial directionality. After transcription begins, the opposing functions of Integrator (INTS11) and cyclin-dependent kinase 9 (CDK9) maintain directionality. INTS11 terminates antisense transcription, whereas sense transcription is protected from INTS11-dependent attenuation by CDK9. INTS11 terminates transcription in both directions upon CDK9 inhibition, and the engineered recruitment of CDK9 prevents attenuation by INTS11. Therefore, transcriptional initiation and the opposing activities of CDK9 and INTS11 explain the attenuation of antisense transcription, the more extensive nature of sense transcription, and promoter directionality.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.563503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.563503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.563503">
        <p class="paperTitle">Multi-modal skin atlas identifies a multicellular immune-stromal community associated with altered cornification and specific T cell expansion in atopic dermatitis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.563503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.563503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fiskin, E.; Eraslan, G.; Alora-Palli, M. B.; Leyva-Castillo, J. M.; Kim, S.; Choe, H.; Lareau, C. A.; Lau, H.; Finan, E. P.; Teixeira-Soldano, I.; LaBere, B.; Chu, A.; Woods, B.; Chou, J.; Slyper, M.; Waldman, J.; Islam, S.; Schneider, L.; Phipatanakul, W.; Platt, C.; Rozenblatt-Rosen, O.; Delorey, T. M.; Deguine, J.; Smith, G. P.; Geha, R. S.; Regev, A.; Xavier, R. J.</p>
        <p class="info">Score: 10.9, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.563503' target='https://doi.org/10.1101/2023.10.29.563503'> 10.1101/2023.10.29.563503</a></p>
        <p class="abstract">In healthy skin, a cutaneous immune system maintains the balance between tolerance towards innocuous environmental antigens and immune responses against pathological agents. In atopic dermatitis (AD), barrier and immune dysfunction result in chronic tissue inflammation. Our understanding of the skin tissue ecosystem in AD remains incomplete with regard to the hallmarks of pathological barrier formation, and cellular state and clonal composition of disease-promoting cells. Here, we generated a multi-modal cell census of 310,691 cells spanning 86 cell subsets from whole skin tissue of 19 adult individuals, including non-lesional and lesional skin from 11 AD patients, and integrated it with 396,321 cells from four studies into a comprehensive human skin cell atlas in health and disease. Reconstruction of human keratinocyte differentiation from basal to cornified layers revealed a disrupted cornification trajectory in AD. This disrupted epithelial differentiation was associated with signals from a unique immune and stromal multicellular community comprised of MMP12&#43; dendritic cells (DCs), mature migratory DCs, cycling ILCs, NK cells, inflammatory CCL19&#43; IL4I1&#43; fibroblasts, and clonally expanded IL13&#43;IL22&#43;IL26&#43; T cells with overlapping type 2 and type 17 characteristics. Cell subsets within this immune and stromal multicellular community were connected by multiple inter-cellular positive feedback loops predicted to impact community assembly and maintenance. AD GWAS gene expression was enriched both in disrupted cornified keratinocytes and in cell subsets from the lesional immune and stromal multicellular community including IL13&#43;IL22&#43;IL26&#43; T cells and ILCs, suggesting that epithelial or immune dysfunction in the context of the observed cellular communication network can initiate and then converge towards AD. Our work highlights specific, disease-associated cell subsets and interactions as potential targets in progression and resolution of chronic inflammation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.563924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.563924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.563924">
        <p class="paperTitle">Pervasive translational control of photosynthesis genes during photomorphogenesis is acquired by C4 genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.563924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.563924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Llorens, I. R.; Lastovka, F.; Schreier, T. B.; Singh, P.; Chung, B. Y.; Hibberd, J. M.</p>
        <p class="info">Score: 9.5, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.563924' target='https://doi.org/10.1101/2023.10.27.563924'> 10.1101/2023.10.27.563924</a></p>
        <p class="abstract">C4 photosynthesis allows increased efficiency and has evolved in more than sixty-six plant lineages. Underpinning this repeated appearance of the C4 pathway is a major transcriptional reprogramming of photosynthesis genes. Here we investigated whether evolution has also significantly modified translational control by defining the translational dynamics of C3 rice and C4 sorghum during photomorphogenesis. In the dark rice photosynthesis transcripts are low abundance but highly translated. After exposure to light translational efficiency declines. The same phenomena occur in sorghum but in addition C4 cycle genes show this response. We propose a model in which translational control of photosynthesis genes permits a rapid response to light and that this translational regulation is gained by C4 genes during the evolution of the C4 pathway.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.05.565726">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.05.565726" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.05.565726">
        <p class="paperTitle">Structural basis of RfaH-mediated transcription-translation coupling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.05.565726" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.05.565726" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Molodtsov, V.; Wang, C.; Kaelber, J. T.; Blaha, G.; Ebright, R. H.</p>
        <p class="info">Score: 5.3, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.05.565726' target='https://doi.org/10.1101/2023.11.05.565726'> 10.1101/2023.11.05.565726</a></p>
        <p class="abstract">The NusG paralog RfaH mediates bacterial transcription-translation coupling on genes that contain a DNA sequence element, termed an ops site, required for pausing RNA polymerase (RNAP) and for loading RfaH onto the paused RNAP. Here we report cryo-EM structures of transcription-translation complexes (TTCs) containing RfaH. The results show that RfaH bridges RNAP and the ribosome, with the RfaH N-terminal domain interacting with RNAP, and with the RfaH C-terminal domain interacting with the ribosome. The results show that the distribution of translational and orientational positions of RNAP relative to the ribosome in RfaH-coupled TTCs is more restricted than in NusG-coupled TTCs, due to the more restricted flexibility of the RfaH interdomain linker. The results further show that the structural organization of RfaH-coupled TTCs in the &#34;loading state,&#34; in which RNAP and RfaH are located at the ops site during formation of the TTC, is the same as the structural organization of RfaH-coupled TTCs in the &#34;loaded state,&#34; in which RNAP and RfaH are located at positions downstream of the ops site during function of the TTC. The results define the structural organization of RfaH-containing TTCs and set the stage for analysis of functions of RfaH during translation initiation and transcription-translation coupling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564348">
        <p class="paperTitle">Brazilin is a Natural Product Inhibitor of the NLRP3 Inflammasome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McMahon, E. A.; El-Sayed, S.; Green, J. P.; Hoyle, C.; Fitzpatrick, L.; Jones, E.; Corrie, E.; Kelly, R. L.; Challinor, M.; Freeman, S.; Bryce, R. A.; Lawrence, C. B.; Brough, D.; Kasher, P. R.</p>
        <p class="info">Score: 6.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564348' target='https://doi.org/10.1101/2023.10.30.564348'> 10.1101/2023.10.30.564348</a></p>
        <p class="abstract">Excessive or aberrant NLRP3 inflammasome activation has been implicated in the progression and initiation of many inflammatory conditions; however, currently no NLRP3 inflammasome inhibitors have been approved for therapeutic use in the clinic. Here we have identified that the natural product brazilin effectively inhibits both priming and activation of the NLRP3 inflammasome in cultured murine macrophages, a human iPSC microglial cell line and in a mouse model of acute peritoneal inflammation. Through computational modelling, we predict that brazilin can adopt a favourable binding pose within a site of the NLRP3 protein which is essential for its conformational activation. Our results not only encourage further evaluation of brazilin as a therapeutic agent for NLRP3-related inflammatory diseases, but also introduce this small-molecule as a promising scaffold structure for the development of derivative NLRP3 inhibitor compounds.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
