<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function
Authors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.
Score: 20.6, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550466
Catalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function
Authors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.
Score: 20.6, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550466
Catalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function
Authors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.
Score: 20.6, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550466
Catalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function\nAuthors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.\nScore: 20.6, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550466\nCatalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits.",
  "keywords": [
    
  ],
  "articleBody": " Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function\nAuthors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.\nScore: 20.6, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550466\nCatalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits. Individual deletions of subunits of either CERF, RSF, ACF, WICH or NoRC subcomplexes only moderately affect the chromatin landscape, while removal of the NURF-specific subunit BPTF leads to drastic reduction in chromatin accessibility and SNF2H ATPase localization around CTCF sites. While this affects adjacent nucleosome occupancy, it only modestly impacts CTCF binding itself. In the absence of accessibility, the structural function of CTCF is nevertheless impaired resulting in lower occupancy of cohesin and cohesin release factor, and reduced physical insulation at these sites, highlighting the need of NURF-mediated remodeling for open chromatin and proper CTCF function. These results separate local CTCF binding from insulator function in nuclear organization and reveal a specific role for NURF in mediating SNF2H localization and chromatin opening at bound CTCF sites. They designate local accessibility as critical for cohesin positioning and establishment of physical insulation.\nCryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment\nAuthors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.\nScore: 19.2, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550782\nHere we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3.4 [A]. The complex was generated by cotranslational insertion into preformed nanodisc membranes using cell-free synthesis in E. coli lysates. It is the first structure obtained by this detergent-free strategy and the first GPCR/Gs complex structure in lipid environment. Structural comparison with the inactive conformation of H2R and the inactive and Gq-coupled active state of H1R together with structure-guided functional experiments reveal molecular insights into the specificity of ligand binding and G protein coupling for this receptor family. We demonstrate lipid-modulated folding of cell-free synthesized H2R, its agonist-dependent internalization and its interaction with endogenously synthesized H1R and H2R in HEK293 cells by applying a recently developed nanotransfer technique.\nVCF1 is an unconventional p97/VCP cofactor promoting recognition of ubiquitylated p97-UFD1-NPL4 substrates\nAuthors: Mirsanaye, A. S.; Hoffmann, S.; Weisser, M. B.; Mund, A.; Lopez-Mendez, B.; Typas, D.; van den Boom, J.; Benedict, B.; Hendriks, I. A.; Nielsen, M. L.; Meyer, H.; Duxin, J. P.; Montoya, G.; Mailand, N.\nScore: 8.5, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550448\nThe hexameric AAA+ ATPase p97/VCP functions as an essential mediator of ubiquitin-dependent cellular processes, extracting ubiquitylated proteins from macromolecular complexes or membranes by catalyzing their unfolding. p97 is directed to ubiquitylated client proteins via multiple cofactors, most of which interact with the p97 N-domain. Here, we discovered that FAM104A, a protein of unknown function that we named VCF1 (VCP/p97 Cofactor FAM104 1), acts as a novel p97 cofactor in human cells. Detailed structure-function studies revealed that VCF1 directly binds p97 via a conserved novel -helical motif that recognizes the p97 N-domain with unusually high affinity, exceeding that of other cofactors. We show that VCF1 engages in joint p97 complex formation with the heterodimeric primary p97 cofactor UFD1-NPL4 and promotes p97-UFD1-NPL4-dependent proteasomal degradation of ubiquitylated substrates in cells. Mechanistically, VCF1 indirectly stimulates UFD1-NPL4 interactions with ubiquitin conjugates via its binding to p97 but has no intrinsic affinity for ubiquitin. Collectively, our findings establish VCF1 as an unconventional p97 cofactor that promotes p97-dependent protein turnover by facilitating p97-UFD1-NPL4 recruitment to ubiquitylated targets.\nCysQuant: simultaneous quantification of cysteine oxidation and protein abundance using data dependent or independent acquisition mass spectrometry\nAuthors: Huang, J.; Staes, A.; Impens, F.; Demichev, V.; Van Breusegem, F.; Gevaert, K.; Willems, P.\nScore: 8.0, Published: 2023-07-24 DOI: 10.1101/2023.07.22.550135\nProtein cysteinyl thiols are susceptible to reduction-oxidation reactions that can influence protein function. Accurate quantification of cysteine oxidation is therefore crucial for decoding protein redox regulation. Here, we present CysQuant, a novel approach for simultaneous quantification of cysteine oxidation degrees and protein abundancies. CysQuant involves light/heavy iodoacetamide isotopologues for differential labeling of reduced and reversibly oxidized cysteines analyzed by data-dependent acquisition (DDA) or data-independent acquisition mass spectrometry (DIA-MS). Using plexDIA with in silico predicted spectral libraries, we quantified an average of 18% cysteine oxidation in Arabidopsis thaliana by DIA-MS, including a subset of highly oxidized cysteines forming disulfide bridges in AlphaFold2 predicted structures. Applying CysQuant to Arabidopsis seedlings exposed to excessive light, we successfully quantified the well-established increased reduction of Calvin-Benson cycle enzymes and discovered yet uncharacterized redox-sensitive disulfides in chloroplastic enzymes. Overall, CysQuant is a highly versatile tool for assessing the cysteine modification status that can be widely applied across various mass spectrometry platforms and organisms.\nUnconventional stabilization of the human T-cell leukemia virus type 1 immature Gag lattice\nAuthors: Obr, M.; Percipalle, M.; Chernikova, D.; Yang, H.; Thader, A.; Pinke, G.; Porley, D.; Mansky, L. M.; Dick, R.; Schur, F. K.\nScore: 7.9, Published: 2023-07-24 DOI: 10.1101/2023.07.24.548988\nHuman T-cell leukemia virus type 1 (HTLV-1) has an atypical immature particle morphology compared to other retroviruses. This indicates that these particles are formed in a way that is unique. Here we report the results of cryo-electron tomography (cryo-ET) studies of HTLV-1 virus-like particles (VLPs) assembled in vitro, as well as derived from cells. This work shows that HTLV-1 employs an unconventional mechanism of Gag-Gag interactions to form the immature viral lattice. Analysis of high-resolution structural information from immature CA tubular arrays reveals that the primary stabilizing component in HTLV-1 is CA-NTD. Mutagenesis and biophysical analysis support this observation. This distinguishes HTLV-1 from other retroviruses, in which the stabilization is provided primarily by the CA-CTD. These results are the first to provide structural details of the quaternary arrangement of Gag for an immature deltaretrovirus, and this helps explain why HTLV-1 particles are morphologically distinct.\nScaling of an antibody validation procedure enables quantification of antibody performance in major research applications\nAuthors: Ayoubi, R.; Ryan, J.; Biddle, M. S.; Alshafie, W.; Fotouhi, M.; Bolivar, S. G.; Moleon, V. R.; Eckmann, P.; Worrall, D.; McDowell, I.; Southern, K.; Reintsch, W.; Durcan, T. M.; Brown, C. M.; Bandrowski, A. E.; Virk, H. S.; Edwards, A. M.; McPherson, P. S.; Laflamme, C.\nScore: 10.2, Published: 2023-07-22 DOI: 10.1101/2023.06.01.543292\nAntibodies are critical reagents to detect and characterize proteins. It is commonly understood that many commercial antibodies do not recognize their intended targets, but information on the scope of the problem remains largely anecdotal, and as such, feasibility of the goal of at least one potent and specific antibody targeting each protein in a proteome cannot be assessed. Focusing on antibodies for human proteins, we have scaled a standardized characterization approach using parental and knockout cell lines to assess the performance of 614 commercial antibodies for 65 neuroscience-related proteins. Side-by-side comparisons of all antibodies against each target, obtained from multiple commercial partners, demonstrates that: i) more than 50% of all antibodies failed in one or more tests, ii) yet, [~]50-75% of the protein set was covered by at least one high-performing antibody, depending on application, suggesting that coverage of human proteins by commercial antibodies is significant; and iii) recombinant antibodies performed better than monoclonal or polyclonal antibodies. The hundreds of underperforming antibodies identified in this study were found to have been used in a large number of published articles, which should raise alarm. Encouragingly, more than half of the underperforming commercial antibodies were reassessed by the manufacturers, and many had alterations to their recommended usage or were removed from the market. This first such study helps demonstrate the scale of the antibody specificity problem but also suggests an efficient strategy toward achieving coverage of the human proteome; mine the existing commercial antibody repertoire, and use the data to focus new renewable antibody generation efforts.\nSmart Leginon enables cryoEM screening automation across multiple grids\nAuthors: Sawh-Gopal, A.; Ishemgulova, A.; Chua, E. Y. D.; Aragon, M. F.; Mendez, J. H.; Eng, E. T.; Noble, A. J.\nScore: 6.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550406\nAdvancements in cryo-electron microscopy (cryoEM) techniques over the past decade have allowed structural biologists to routinely resolve macromolecular protein complexes to near-atomic resolution. The general workflow of the entire cryoEM pipeline involves iterating between sample preparation, cryoEM grid preparation, and sample/grid screening before moving on to high-resolution data collection. Iterating between sample/grid preparation and screening is typically a major bottleneck for researchers, as every iterative experiment must optimize for sample concentration, buffer conditions, grid material, grid hole size, ice thickness, and protein particle behavior in the ice, amongst other variables. Furthermore, once these variables are satisfactorily determined, grids prepared under identical conditions vary widely in whether they are ready for data collection, so additional screening sessions prior to selecting optimal grids for high-resolution data collection are recommended. This sample/grid preparation and screening process often consumes several dozen grids and days of operator time at the microscope. Furthermore, the screening process is limited to operator/microscope availability and microscope accessibility. Here, we demonstrate how to use Leginon and Smart Leginon Autoscreen to automate the majority of cryoEM grid screening. Autoscreen combines machine learning, computer vision algorithms, and microscope-handling algorithms to remove the need for constant manual operator input. Autoscreen can autonomously load and image grids with multi-scale imaging using an automated specimen-exchange cassette system, resulting in unattended grid screening for an entire cassette. As a result, operator time for screening 12 grids may be reduced to [~]10 minutes with Autoscreen compared to [~]6 hours using previous methods which are hampered by their inability to account for high variability between grids. This protocol and video tutorial first introduces basic Leginon setup and functionality, then demonstrates Autoscreen functionality step-by-step from the creation of a template session to the end of a 12 grid automated screening session. SummaryCryoEM multi-grid screening is often a tedious process that demands hours of attention. Here, this protocol shows how to set up standard Leginon collection and Smart Leginon Autoscreen to automate this process. This protocol can be applied to the majority of cryoEM holey foil grids.\nInteraction of SCoV-2 NSP7 or NSP8 alone may cause constriction of the RNA entry channel in NSP12: Implications for novel RdRp inhibitor drug discovery\nAuthors: Singh, D.; Kushwaha, T.; Kulandaisamy, R.; Kumar, V.; Baswal, K.; Tiwari, S. H.; Ghorai, A.; Kumar, M.; Kumar, S.; Polamarasetty, A.; Sehgal, D.; Katika, M. R.; Gadde, S.; Cote, M.; Kayampeta, S. R.; Appaiahgari, M. B.; Inampudi, K. K.\nScore: 5.9, Published: 2023-07-27 DOI: 10.1101/2023.07.26.550660\nRNA-dependent RNA polymerase (RdRP) is a critical component of the RNA virus life cycle, including SCoV-2. Among the Coronavirus-encoded proteins, non-structural protein 12 (NSP12) exhibits polymerase activity in collaboration with one unit of NSP7 and two units of NSP8, constituting the RdRp holoenzyme. While there is abundant information on SCoV-2 RdRp-mediated RNA replication, the influence of interplay among NSP12, NSP7, and NSP8 on template RNA binding and primer extension activity remains relatively unexplored and poorly understood. Here, we recreated a functional RdRp holoenzyme in vitro using recombinant SCoV-2 NSP12, NSP7, and NSP8, and established its functional activity. Subsequently, molecular interactions among the NSPs in the presence of a variety of templates and their effects on polymerase activity were studied, wherein we found that NSP12 alone exhibited notable polymerase activity that increased significantly in the presence of NSP7 and NSP8. However, this activity was completely shut down, and the template RNA primer complex was detached from NSP12 when one of the to cofactors was present. Through computational analysis, we found that the template RNA entry channel was more constricted in the presence of one of the two cofactors, which was relatively more constricted in the presence of NSP8 compared to that in the presence of NSP7. In conclusion, we report that NSP7 and NSP8 together synergise to enhance the activity of NSP12, but antagonise when present alone. Our findings have implications for novel drug development, and compounds that obstruct the binding of NSP7 or NSP8 to NSP12 can have lethal effects on viral RNA replication.\nMechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction\nAuthors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.\nScore: 49.9, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549907\nKinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved. How this is accomplished and regulated, and how kinetochore-mediated chromosome segregation occurs at a molecular level remain major outstanding questions. Here we describe the cryo-electron microscopy (cryo-EM) structure of the budding yeast outer kinetochore Ndc80 and Dam1 ring complexes assembled onto microtubules. We observe coordinated interactions of the outer kinetochore complexes through multiple interfaces, in addition to a short staple within the Dam1 subunit that facilitates Dam1c ring assembly. Perturbation of these interfaces results in loss of yeast viability. Force-rupture assays indicated this is a consequence of substantial reductions in kinetochore - microtubule binding strength. EC-mediated phosphorylation of Ndc80c-Dam1c interfaces would drive complex disassembly, whereas Dam1 staple phosphorylation would promote Dam1c ring disassembly, explaining how kinetochore - microtubule attachments are destabilized and reset by the EC mechanism. One-Sentence SummaryPhosphorylation disrupts the outer kinetochore to regulate kinetochore-microtubule attachments in mitotic error correction.\nSystematic mapping and modeling of 3D enhancer-promoter interactions in early mouse embryonic lineages reveal regulatory principles that determine the levels and cell-type specificity of gene expression\nAuthors: Murphy, D.; Salataj, E.; Di Giammartino, D. C.; Rodriguez-Hernaez, J.; Kloetgen, A.; Garg, V.; Char, E.; Uyehara, C. M.; Ee, L.-s.; Lee, U.; Stadtfeld, M.; Hadjantonakis, A.-K.; Tsirigos, A.; Polyzos, A.; Apostolou, E.\nScore: 30.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549714\nMammalian embryogenesis commences with two pivotal and binary cell fate decisions that give rise to three essential lineages, the trophectoderm (TE), the epiblast (EPI) and the primitive endoderm (PrE). Although key signaling pathways and transcription factors that control these early embryonic decisions have been identified, the non-coding regulatory elements via which transcriptional regulators enact these fates remain understudied. To address this gap, we have characterized, at a genome-wide scale, enhancer activity and 3D connectivity in embryo-derived stem cell lines that represent each of the early developmental fates. We observed extensive enhancer remodeling and fine-scale 3D chromatin rewiring among the three lineages, which strongly associate with transcriptional changes, although there are distinct groups of genes that are irresponsive to topological changes. In each lineage, a high degree of connectivity or \"hubness\" positively correlates with levels of gene expression and enriches for cell-type specific and essential genes. Genes within 3D hubs also show a significantly stronger probability of coregulation across lineages, compared to genes in linear proximity or within the same contact domains. By incorporating 3D chromatin features, we build a novel predictive model for transcriptional regulation (3D-HiChAT), which outperformed models that use only 1D promoter or proximal variables in predicting levels and cell-type specificity of gene expression. Using 3D-HiChAT, we performed genome-wide in silico perturbations to nominate candidate functional enhancers and hubs in each cell lineage, and with CRISPRi experiments we validated several novel enhancers that control expression of one or more genes in their respective lineages. Our study comprehensively identifies 3D regulatory hubs associated with the earliest mammalian lineages and describes their relationship to gene expression and cell identity, providing a framework to understand lineage-specific transcriptional behaviors. HIGHLIGHTS- Cell lines representing early embryonic lineages undergo drastic enhancer remodeling and fine-scale 3D chromatin reorganization - Highly interacting 3D hubs strongly enrich for highly expressed, cell-type specific and essential genes - 3D chromatin features greatly improve prediction of cell-type specific gene expression compared to 1D promoter features - In silico and experimental perturbations identify novel enhancers regulating the expression of two or more genes in early embryonic lineages\n",
  "wordCount" : "2677",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550466">
        <p class="paperTitle">Systematic assessment of ISWI subunits reveals that NURF creates local accessibility for proper CTCF function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iurlaro, M.; Masoni, F.; Wirbelauer, C.; Iskar, M.; Burger, L.; Schuebeler, D.</p>
        <p class="info">Score: 20.6, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550466' target='https://doi.org/10.1101/2023.07.25.550466'> 10.1101/2023.07.25.550466</a></p>
        <p class="abstract">Catalytic activity of the ISWI family of remodelers is critical for nucleosomal organization and DNA binding of transcription factors, including the insulator protein CTCF. To define which subcomplex mediates these diverse functions, we derived a panel of isogenic mouse stem cell lines each lacking one of six ISWI accessory subunits. Individual deletions of subunits of either CERF, RSF, ACF, WICH or NoRC subcomplexes only moderately affect the chromatin landscape, while removal of the NURF-specific subunit BPTF leads to drastic reduction in chromatin accessibility and SNF2H ATPase localization around CTCF sites. While this affects adjacent nucleosome occupancy, it only modestly impacts CTCF binding itself. In the absence of accessibility, the structural function of CTCF is nevertheless impaired resulting in lower occupancy of cohesin and cohesin release factor, and reduced physical insulation at these sites, highlighting the need of NURF-mediated remodeling for open chromatin and proper CTCF function.

These results separate local CTCF binding from insulator function in nuclear organization and reveal a specific role for NURF in mediating SNF2H localization and chromatin opening at bound CTCF sites. They designate local accessibility as critical for cohesin positioning and establishment of physical insulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550782">
        <p class="paperTitle">Cryo-EM structure of cell-free synthesized human histamine H2 receptor coupled to heterotrimeric Gs protein in lipid nanodisc environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koeck, Z.; Schnelle, K.; Persechino, M.; Umbach, S.; Schihada, H.; Januliene, D.; Parey, K.; Pockes, S.; Kolb, P.; Doetsch, V.; Moeller, A.; Hilger, D.; Bernhard, F.</p>
        <p class="info">Score: 19.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550782' target='https://doi.org/10.1101/2023.07.27.550782'> 10.1101/2023.07.27.550782</a></p>
        <p class="abstract">Here we describe the cryo-electron microscopy structure of the human histamine 2 receptor (H2R) in an active conformation with bound histamine and in complex with Gs heterotrimeric protein at an overall resolution of 3.4 [A]. The complex was generated by cotranslational insertion into preformed nanodisc membranes using cell-free synthesis in E. coli lysates. It is the first structure obtained by this detergent-free strategy and the first GPCR/Gs complex structure in lipid environment. Structural comparison with the inactive conformation of H2R and the inactive and Gq-coupled active state of H1R together with structure-guided functional experiments reveal molecular insights into the specificity of ligand binding and G protein coupling for this receptor family. We demonstrate lipid-modulated folding of cell-free synthesized H2R, its agonist-dependent internalization and its interaction with endogenously synthesized H1R and H2R in HEK293 cells by applying a recently developed nanotransfer technique.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550448">
        <p class="paperTitle">VCF1 is an unconventional p97/VCP cofactor promoting recognition of ubiquitylated p97-UFD1-NPL4 substrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mirsanaye, A. S.; Hoffmann, S.; Weisser, M. B.; Mund, A.; Lopez-Mendez, B.; Typas, D.; van den Boom, J.; Benedict, B.; Hendriks, I. A.; Nielsen, M. L.; Meyer, H.; Duxin, J. P.; Montoya, G.; Mailand, N.</p>
        <p class="info">Score: 8.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550448' target='https://doi.org/10.1101/2023.07.25.550448'> 10.1101/2023.07.25.550448</a></p>
        <p class="abstract">The hexameric AAA&#43; ATPase p97/VCP functions as an essential mediator of ubiquitin-dependent cellular processes, extracting ubiquitylated proteins from macromolecular complexes or membranes by catalyzing their unfolding. p97 is directed to ubiquitylated client proteins via multiple cofactors, most of which interact with the p97 N-domain. Here, we discovered that FAM104A, a protein of unknown function that we named VCF1 (VCP/p97 Cofactor FAM104 1), acts as a novel p97 cofactor in human cells. Detailed structure-function studies revealed that VCF1 directly binds p97 via a conserved novel -helical motif that recognizes the p97 N-domain with unusually high affinity, exceeding that of other cofactors. We show that VCF1 engages in joint p97 complex formation with the heterodimeric primary p97 cofactor UFD1-NPL4 and promotes p97-UFD1-NPL4-dependent proteasomal degradation of ubiquitylated substrates in cells. Mechanistically, VCF1 indirectly stimulates UFD1-NPL4 interactions with ubiquitin conjugates via its binding to p97 but has no intrinsic affinity for ubiquitin. Collectively, our findings establish VCF1 as an unconventional p97 cofactor that promotes p97-dependent protein turnover by facilitating p97-UFD1-NPL4 recruitment to ubiquitylated targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550135">
        <p class="paperTitle">CysQuant: simultaneous quantification of cysteine oxidation and protein abundance using data dependent or independent acquisition mass spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, J.; Staes, A.; Impens, F.; Demichev, V.; Van Breusegem, F.; Gevaert, K.; Willems, P.</p>
        <p class="info">Score: 8.0, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550135' target='https://doi.org/10.1101/2023.07.22.550135'> 10.1101/2023.07.22.550135</a></p>
        <p class="abstract">Protein cysteinyl thiols are susceptible to reduction-oxidation reactions that can influence protein function. Accurate quantification of cysteine oxidation is therefore crucial for decoding protein redox regulation. Here, we present CysQuant, a novel approach for simultaneous quantification of cysteine oxidation degrees and protein abundancies. CysQuant involves light/heavy iodoacetamide isotopologues for differential labeling of reduced and reversibly oxidized cysteines analyzed by data-dependent acquisition (DDA) or data-independent acquisition mass spectrometry (DIA-MS). Using plexDIA with in silico predicted spectral libraries, we quantified an average of 18% cysteine oxidation in Arabidopsis thaliana by DIA-MS, including a subset of highly oxidized cysteines forming disulfide bridges in AlphaFold2 predicted structures. Applying CysQuant to Arabidopsis seedlings exposed to excessive light, we successfully quantified the well-established increased reduction of Calvin-Benson cycle enzymes and discovered yet uncharacterized redox-sensitive disulfides in chloroplastic enzymes. Overall, CysQuant is a highly versatile tool for assessing the cysteine modification status that can be widely applied across various mass spectrometry platforms and organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.548988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.548988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.548988">
        <p class="paperTitle">Unconventional stabilization of the human T-cell leukemia virus type 1 immature Gag lattice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.548988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.548988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Obr, M.; Percipalle, M.; Chernikova, D.; Yang, H.; Thader, A.; Pinke, G.; Porley, D.; Mansky, L. M.; Dick, R.; Schur, F. K.</p>
        <p class="info">Score: 7.9, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.548988' target='https://doi.org/10.1101/2023.07.24.548988'> 10.1101/2023.07.24.548988</a></p>
        <p class="abstract">Human T-cell leukemia virus type 1 (HTLV-1) has an atypical immature particle morphology compared to other retroviruses. This indicates that these particles are formed in a way that is unique. Here we report the results of cryo-electron tomography (cryo-ET) studies of HTLV-1 virus-like particles (VLPs) assembled in vitro, as well as derived from cells. This work shows that HTLV-1 employs an unconventional mechanism of Gag-Gag interactions to form the immature viral lattice. Analysis of high-resolution structural information from immature CA tubular arrays reveals that the primary stabilizing component in HTLV-1 is CA-NTD. Mutagenesis and biophysical analysis support this observation. This distinguishes HTLV-1 from other retroviruses, in which the stabilization is provided primarily by the CA-CTD. These results are the first to provide structural details of the quaternary arrangement of Gag for an immature deltaretrovirus, and this helps explain why HTLV-1 particles are morphologically distinct.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543292">
        <p class="paperTitle">Scaling of an antibody validation procedure enables quantification of antibody performance in major research applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ayoubi, R.; Ryan, J.; Biddle, M. S.; Alshafie, W.; Fotouhi, M.; Bolivar, S. G.; Moleon, V. R.; Eckmann, P.; Worrall, D.; McDowell, I.; Southern, K.; Reintsch, W.; Durcan, T. M.; Brown, C. M.; Bandrowski, A. E.; Virk, H. S.; Edwards, A. M.; McPherson, P. S.; Laflamme, C.</p>
        <p class="info">Score: 10.2, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543292' target='https://doi.org/10.1101/2023.06.01.543292'> 10.1101/2023.06.01.543292</a></p>
        <p class="abstract">Antibodies are critical reagents to detect and characterize proteins. It is commonly understood that many commercial antibodies do not recognize their intended targets, but information on the scope of the problem remains largely anecdotal, and as such, feasibility of the goal of at least one potent and specific antibody targeting each protein in a proteome cannot be assessed. Focusing on antibodies for human proteins, we have scaled a standardized characterization approach using parental and knockout cell lines to assess the performance of 614 commercial antibodies for 65 neuroscience-related proteins. Side-by-side comparisons of all antibodies against each target, obtained from multiple commercial partners, demonstrates that: i) more than 50% of all antibodies failed in one or more tests, ii) yet, [~]50-75% of the protein set was covered by at least one high-performing antibody, depending on application, suggesting that coverage of human proteins by commercial antibodies is significant; and iii) recombinant antibodies performed better than monoclonal or polyclonal antibodies. The hundreds of underperforming antibodies identified in this study were found to have been used in a large number of published articles, which should raise alarm. Encouragingly, more than half of the underperforming commercial antibodies were reassessed by the manufacturers, and many had alterations to their recommended usage or were removed from the market. This first such study helps demonstrate the scale of the antibody specificity problem but also suggests an efficient strategy toward achieving coverage of the human proteome; mine the existing commercial antibody repertoire, and use the data to focus new renewable antibody generation efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550406">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550406" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550406">
        <p class="paperTitle">Smart Leginon enables cryoEM screening automation across multiple grids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550406" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550406" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sawh-Gopal, A.; Ishemgulova, A.; Chua, E. Y. D.; Aragon, M. F.; Mendez, J. H.; Eng, E. T.; Noble, A. J.</p>
        <p class="info">Score: 6.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550406' target='https://doi.org/10.1101/2023.07.24.550406'> 10.1101/2023.07.24.550406</a></p>
        <p class="abstract">Advancements in cryo-electron microscopy (cryoEM) techniques over the past decade have allowed structural biologists to routinely resolve macromolecular protein complexes to near-atomic resolution. The general workflow of the entire cryoEM pipeline involves iterating between sample preparation, cryoEM grid preparation, and sample/grid screening before moving on to high-resolution data collection. Iterating between sample/grid preparation and screening is typically a major bottleneck for researchers, as every iterative experiment must optimize for sample concentration, buffer conditions, grid material, grid hole size, ice thickness, and protein particle behavior in the ice, amongst other variables. Furthermore, once these variables are satisfactorily determined, grids prepared under identical conditions vary widely in whether they are ready for data collection, so additional screening sessions prior to selecting optimal grids for high-resolution data collection are recommended. This sample/grid preparation and screening process often consumes several dozen grids and days of operator time at the microscope. Furthermore, the screening process is limited to operator/microscope availability and microscope accessibility. Here, we demonstrate how to use Leginon and Smart Leginon Autoscreen to automate the majority of cryoEM grid screening. Autoscreen combines machine learning, computer vision algorithms, and microscope-handling algorithms to remove the need for constant manual operator input. Autoscreen can autonomously load and image grids with multi-scale imaging using an automated specimen-exchange cassette system, resulting in unattended grid screening for an entire cassette. As a result, operator time for screening 12 grids may be reduced to [~]10 minutes with Autoscreen compared to [~]6 hours using previous methods which are hampered by their inability to account for high variability between grids. This protocol and video tutorial first introduces basic Leginon setup and functionality, then demonstrates Autoscreen functionality step-by-step from the creation of a template session to the end of a 12 grid automated screening session.

SummaryCryoEM multi-grid screening is often a tedious process that demands hours of attention. Here, this protocol shows how to set up standard Leginon collection and Smart Leginon Autoscreen to automate this process. This protocol can be applied to the majority of cryoEM holey foil grids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550660" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550660">
        <p class="paperTitle">Interaction of SCoV-2 NSP7 or NSP8 alone may cause constriction of the RNA entry channel in NSP12: Implications for novel RdRp inhibitor drug discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, D.; Kushwaha, T.; Kulandaisamy, R.; Kumar, V.; Baswal, K.; Tiwari, S. H.; Ghorai, A.; Kumar, M.; Kumar, S.; Polamarasetty, A.; Sehgal, D.; Katika, M. R.; Gadde, S.; Cote, M.; Kayampeta, S. R.; Appaiahgari, M. B.; Inampudi, K. K.</p>
        <p class="info">Score: 5.9, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550660' target='https://doi.org/10.1101/2023.07.26.550660'> 10.1101/2023.07.26.550660</a></p>
        <p class="abstract">RNA-dependent RNA polymerase (RdRP) is a critical component of the RNA virus life cycle, including SCoV-2. Among the Coronavirus-encoded proteins, non-structural protein 12 (NSP12) exhibits polymerase activity in collaboration with one unit of NSP7 and two units of NSP8, constituting the RdRp holoenzyme. While there is abundant information on SCoV-2 RdRp-mediated RNA replication, the influence of interplay among NSP12, NSP7, and NSP8 on template RNA binding and primer extension activity remains relatively unexplored and poorly understood. Here, we recreated a functional RdRp holoenzyme in vitro using recombinant SCoV-2 NSP12, NSP7, and NSP8, and established its functional activity. Subsequently, molecular interactions among the NSPs in the presence of a variety of templates and their effects on polymerase activity were studied, wherein we found that NSP12 alone exhibited notable polymerase activity that increased significantly in the presence of NSP7 and NSP8. However, this activity was completely shut down, and the template RNA primer complex was detached from NSP12 when one of the to cofactors was present. Through computational analysis, we found that the template RNA entry channel was more constricted in the presence of one of the two cofactors, which was relatively more constricted in the presence of NSP8 compared to that in the presence of NSP7. In conclusion, we report that NSP7 and NSP8 together synergise to enhance the activity of NSP12, but antagonise when present alone. Our findings have implications for novel drug development, and compounds that obstruct the binding of NSP7 or NSP8 to NSP12 can have lethal effects on viral RNA replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549907">
        <p class="paperTitle">Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.</p>
        <p class="info">Score: 49.9, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549907' target='https://doi.org/10.1101/2023.07.20.549907'> 10.1101/2023.07.20.549907</a></p>
        <p class="abstract">Kinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved. How this is accomplished and regulated, and how kinetochore-mediated chromosome segregation occurs at a molecular level remain major outstanding questions. Here we describe the cryo-electron microscopy (cryo-EM) structure of the budding yeast outer kinetochore Ndc80 and Dam1 ring complexes assembled onto microtubules. We observe coordinated interactions of the outer kinetochore complexes through multiple interfaces, in addition to a short staple within the Dam1 subunit that facilitates Dam1c ring assembly. Perturbation of these interfaces results in loss of yeast viability. Force-rupture assays indicated this is a consequence of substantial reductions in kinetochore - microtubule binding strength. EC-mediated phosphorylation of Ndc80c-Dam1c interfaces would drive complex disassembly, whereas Dam1 staple phosphorylation would promote Dam1c ring disassembly, explaining how kinetochore - microtubule attachments are destabilized and reset by the EC mechanism.

One-Sentence SummaryPhosphorylation disrupts the outer kinetochore to regulate kinetochore-microtubule attachments in mitotic error correction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549714">
        <p class="paperTitle">Systematic mapping and modeling of 3D enhancer-promoter interactions in early mouse embryonic lineages reveal regulatory principles that determine the levels and cell-type specificity of gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murphy, D.; Salataj, E.; Di Giammartino, D. C.; Rodriguez-Hernaez, J.; Kloetgen, A.; Garg, V.; Char, E.; Uyehara, C. M.; Ee, L.-s.; Lee, U.; Stadtfeld, M.; Hadjantonakis, A.-K.; Tsirigos, A.; Polyzos, A.; Apostolou, E.</p>
        <p class="info">Score: 30.2, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549714' target='https://doi.org/10.1101/2023.07.19.549714'> 10.1101/2023.07.19.549714</a></p>
        <p class="abstract">Mammalian embryogenesis commences with two pivotal and binary cell fate decisions that give rise to three essential lineages, the trophectoderm (TE), the epiblast (EPI) and the primitive endoderm (PrE). Although key signaling pathways and transcription factors that control these early embryonic decisions have been identified, the non-coding regulatory elements via which transcriptional regulators enact these fates remain understudied. To address this gap, we have characterized, at a genome-wide scale, enhancer activity and 3D connectivity in embryo-derived stem cell lines that represent each of the early developmental fates. We observed extensive enhancer remodeling and fine-scale 3D chromatin rewiring among the three lineages, which strongly associate with transcriptional changes, although there are distinct groups of genes that are irresponsive to topological changes. In each lineage, a high degree of connectivity or &#34;hubness&#34; positively correlates with levels of gene expression and enriches for cell-type specific and essential genes. Genes within 3D hubs also show a significantly stronger probability of coregulation across lineages, compared to genes in linear proximity or within the same contact domains. By incorporating 3D chromatin features, we build a novel predictive model for transcriptional regulation (3D-HiChAT), which outperformed models that use only 1D promoter or proximal variables in predicting levels and cell-type specificity of gene expression. Using 3D-HiChAT, we performed genome-wide in silico perturbations to nominate candidate functional enhancers and hubs in each cell lineage, and with CRISPRi experiments we validated several novel enhancers that control expression of one or more genes in their respective lineages. Our study comprehensively identifies 3D regulatory hubs associated with the earliest mammalian lineages and describes their relationship to gene expression and cell identity, providing a framework to understand lineage-specific transcriptional behaviors.

HIGHLIGHTS- Cell lines representing early embryonic lineages undergo drastic enhancer remodeling and fine-scale 3D chromatin reorganization
- Highly interacting 3D hubs strongly enrich for highly expressed, cell-type specific and essential genes
- 3D chromatin features greatly improve prediction of cell-type specific gene expression compared to 1D promoter features
- In silico and experimental perturbations identify novel enhancers regulating the expression of two or more genes in early embryonic lineages</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
