<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction
Authors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.
Score: 50.2, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549907
Kinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction
Authors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.
Score: 50.2, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549907
Kinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction
Authors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.
Score: 50.2, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549907
Kinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction\nAuthors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.\nScore: 50.2, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549907\nKinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved.",
  "keywords": [
    
  ],
  "articleBody": " Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction\nAuthors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.\nScore: 50.2, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549907\nKinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved. How this is accomplished and regulated, and how kinetochore-mediated chromosome segregation occurs at a molecular level remain major outstanding questions. Here we describe the cryo-electron microscopy (cryo-EM) structure of the budding yeast outer kinetochore Ndc80 and Dam1 ring complexes assembled onto microtubules. We observe coordinated interactions of the outer kinetochore complexes through multiple interfaces, in addition to a short staple within the Dam1 subunit that facilitates Dam1c ring assembly. Perturbation of these interfaces results in loss of yeast viability. Force-rupture assays indicated this is a consequence of substantial reductions in kinetochore - microtubule binding strength. EC-mediated phosphorylation of Ndc80c-Dam1c interfaces would drive complex disassembly, whereas Dam1 staple phosphorylation would promote Dam1c ring disassembly, explaining how kinetochore - microtubule attachments are destabilized and reset by the EC mechanism. One-Sentence SummaryPhosphorylation disrupts the outer kinetochore to regulate kinetochore-microtubule attachments in mitotic error correction.\nSystematic mapping and modeling of 3D enhancer-promoter interactions in early mouse embryonic lineages reveal regulatory principles that determine the levels and cell-type specificity of gene expression\nAuthors: Murphy, D.; Salataj, E.; Di Giammartino, D. C.; Rodriguez-Hernaez, J.; Kloetgen, A.; Garg, V.; Char, E.; Uyehara, C. M.; Ee, L.-s.; Lee, U.; Stadtfeld, M.; Hadjantonakis, A.-K.; Tsirigos, A.; Polyzos, A.; Apostolou, E.\nScore: 28.4, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549714\nMammalian embryogenesis commences with two pivotal and binary cell fate decisions that give rise to three essential lineages, the trophectoderm (TE), the epiblast (EPI) and the primitive endoderm (PrE). Although key signaling pathways and transcription factors that control these early embryonic decisions have been identified, the non-coding regulatory elements via which transcriptional regulators enact these fates remain understudied. To address this gap, we have characterized, at a genome-wide scale, enhancer activity and 3D connectivity in embryo-derived stem cell lines that represent each of the early developmental fates. We observed extensive enhancer remodeling and fine-scale 3D chromatin rewiring among the three lineages, which strongly associate with transcriptional changes, although there are distinct groups of genes that are irresponsive to topological changes. In each lineage, a high degree of connectivity or \"hubness\" positively correlates with levels of gene expression and enriches for cell-type specific and essential genes. Genes within 3D hubs also show a significantly stronger probability of coregulation across lineages, compared to genes in linear proximity or within the same contact domains. By incorporating 3D chromatin features, we build a novel predictive model for transcriptional regulation (3D-HiChAT), which outperformed models that use only 1D promoter or proximal variables in predicting levels and cell-type specificity of gene expression. Using 3D-HiChAT, we performed genome-wide in silico perturbations to nominate candidate functional enhancers and hubs in each cell lineage, and with CRISPRi experiments we validated several novel enhancers that control expression of one or more genes in their respective lineages. Our study comprehensively identifies 3D regulatory hubs associated with the earliest mammalian lineages and describes their relationship to gene expression and cell identity, providing a framework to understand lineage-specific transcriptional behaviors. HIGHLIGHTS- Cell lines representing early embryonic lineages undergo drastic enhancer remodeling and fine-scale 3D chromatin reorganization - Highly interacting 3D hubs strongly enrich for highly expressed, cell-type specific and essential genes - 3D chromatin features greatly improve prediction of cell-type specific gene expression compared to 1D promoter features - In silico and experimental perturbations identify novel enhancers regulating the expression of two or more genes in early embryonic lineages\nEfficient Formation of Single-copy Human Artificial Chromosomes\nAuthors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.\nScore: 67.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547284\nLarge DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells. One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.\nChainsaw: protein domain segmentation with fullyconvolutional neural networks\nAuthors: Wells, J.; Hawkins-Hooker, A.; Bordin, N.; Orengo, C.; Paige, B.\nScore: 19.3, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549732\nProtein domains are fundamental units of protein structure and play a pivotal role in understanding folding, function, evolution, and design. The advent of accurate structure prediction techniques has resulted in an influx of new structural data, making the partitioning of these structures into domains essential for inferring evolutionary relationships and functional classification. Currently, the most performant computational methods for parsing protein domains are structure-based methods that rely on unsupervised heuristics to determine the optimal segmentation. Given the availability of large curated databases of domain annotations, we find it worthwhile to revisit structure-based approaches but adopt a supervised learning approach. This paper presents Chainsaw, a novel supervised learning approach to domain parsing that achieves accuracy that surpasses current state-of-the-art methods. The Chainsaw method uses a residual convolutional neural network which is trained to predict the adjacency matrix over residues, where element ai j is an estimate for the probability that residues i and j are in the same domain. Domain assignments are then derived from this soft-adjacency matrix using a non-learned algorithm that is equivalent to maximising the likelihood of the assignments given the set of pairwise probabilities expressed in the soft-adjacency matrix. We benchmark the method against recent state-of-the-art structure-based methods and show that it outperforms them, achieving an NDO score of 0.87 vs 0.80 for the next closest method when predicting a non-redundant test set of multi-domain proteins.\nUnconventional stabilization of the human T-cell leukemia virus type 1 immature Gag lattice\nAuthors: Obr, M.; Percipalle, M.; Chernikova, D.; Yang, H.; Thader, A.; Pinke, G.; Porley, D.; Mansky, L. M.; Dick, R.; Schur, F. K.\nScore: 7.9, Published: 2023-07-24 DOI: 10.1101/2023.07.24.548988\nHuman T-cell leukemia virus type 1 (HTLV-1) has an atypical immature particle morphology compared to other retroviruses. This indicates that these particles are formed in a way that is unique. Here we report the results of cryo-electron tomography (cryo-ET) studies of HTLV-1 virus-like particles (VLPs) assembled in vitro, as well as derived from cells. This work shows that HTLV-1 employs an unconventional mechanism of Gag-Gag interactions to form the immature viral lattice. Analysis of high-resolution structural information from immature CA tubular arrays reveals that the primary stabilizing component in HTLV-1 is CA-NTD. Mutagenesis and biophysical analysis support this observation. This distinguishes HTLV-1 from other retroviruses, in which the stabilization is provided primarily by the CA-CTD. These results are the first to provide structural details of the quaternary arrangement of Gag for an immature deltaretrovirus, and this helps explain why HTLV-1 particles are morphologically distinct.\nCysQuant: simultaneous quantification of cysteine oxidation and protein abundance using data dependent or independent acquisition mass spectrometry\nAuthors: Huang, J.; Staes, A.; Impens, F.; Demichev, V.; Van Breusegem, F.; Gevaert, K.; Willems, P.\nScore: 7.5, Published: 2023-07-24 DOI: 10.1101/2023.07.22.550135\nProtein cysteinyl thiols are susceptible to reduction-oxidation reactions that can influence protein function. Accurate quantification of cysteine oxidation is therefore crucial for decoding protein redox regulation. Here, we present CysQuant, a novel approach for simultaneous quantification of cysteine oxidation degrees and protein abundancies. CysQuant involves light/heavy iodoacetamide isotopologues for differential labeling of reduced and reversibly oxidized cysteines analyzed by data-dependent acquisition (DDA) or data-independent acquisition mass spectrometry (DIA-MS). Using plexDIA with in silico predicted spectral libraries, we quantified an average of 18% cysteine oxidation in Arabidopsis thaliana by DIA-MS, including a subset of highly oxidized cysteines forming disulfide bridges in AlphaFold2 predicted structures. Applying CysQuant to Arabidopsis seedlings exposed to excessive light, we successfully quantified the well-established increased reduction of Calvin-Benson cycle enzymes and discovered yet uncharacterized redox-sensitive disulfides in chloroplastic enzymes. Overall, CysQuant is a highly versatile tool for assessing the cysteine modification status that can be widely applied across various mass spectrometry platforms and organisms.\nScaling of an antibody validation procedure enables quantification of antibody performance in major research applications\nAuthors: Ayoubi, R.; Ryan, J.; Biddle, M. S.; Alshafie, W.; Fotouhi, M.; Bolivar, S. G.; Moleon, V. R.; Eckmann, P.; Worrall, D.; McDowell, I.; Southern, K.; Reintsch, W.; Durcan, T. M.; Brown, C. M.; Bandrowski, A. E.; Virk, H. S.; Edwards, A. M.; McPherson, P. S.; Laflamme, C.\nScore: 10.2, Published: 2023-07-22 DOI: 10.1101/2023.06.01.543292\nAntibodies are critical reagents to detect and characterize proteins. It is commonly understood that many commercial antibodies do not recognize their intended targets, but information on the scope of the problem remains largely anecdotal, and as such, feasibility of the goal of at least one potent and specific antibody targeting each protein in a proteome cannot be assessed. Focusing on antibodies for human proteins, we have scaled a standardized characterization approach using parental and knockout cell lines to assess the performance of 614 commercial antibodies for 65 neuroscience-related proteins. Side-by-side comparisons of all antibodies against each target, obtained from multiple commercial partners, demonstrates that: i) more than 50% of all antibodies failed in one or more tests, ii) yet, [~]50-75% of the protein set was covered by at least one high-performing antibody, depending on application, suggesting that coverage of human proteins by commercial antibodies is significant; and iii) recombinant antibodies performed better than monoclonal or polyclonal antibodies. The hundreds of underperforming antibodies identified in this study were found to have been used in a large number of published articles, which should raise alarm. Encouragingly, more than half of the underperforming commercial antibodies were reassessed by the manufacturers, and many had alterations to their recommended usage or were removed from the market. This first such study helps demonstrate the scale of the antibody specificity problem but also suggests an efficient strategy toward achieving coverage of the human proteome; mine the existing commercial antibody repertoire, and use the data to focus new renewable antibody generation efforts.\nTur1 regulates alternative TSS usage in Cryptococcus\nAuthors: Dang, T. T. V.; Maufrais, C.; Colin, J.; Moyrand, F.; Mouyna, I.; Coppee, J.-Y.; Onyishi, C. U.; Lipecka, J.; Guerrera, I. C.; May, R. C.; Janbon, G.\nScore: 5.2, Published: 2023-07-18 DOI: 10.1101/2023.07.18.549460\nAlternative transcription start site (TSS) usage regulation has been identified as a major means of gene expression regulation in metazoans. However, in fungi, its impact remains elusive as its study has thus far been restricted to model yeasts. Here, we first re-analysed TSS-seq data to define genuine TSS clusters in two species of pathogenic Cryptococcus. We identified two types of TSS clusters associated with specific DNA sequence motifs. Our analysis also revealed that alternative TSS usage regulation in response to environmental cues is widespread in Cryptococcus, altering gene expression and protein targeting. Importantly, we performed a forward genetic screen to identify a unique transcription factor (TF) named Tur1, which regulates alternative TSS (altTSS) usage genome-wide when cells switch from exponential phase to stationary phase. Tur1 has been previously shown to be essential for virulence in C. neoformans. We demonstrated here that a tur1{Delta} mutant strain is more sensitive to superoxide stress and phagocytosed more efficiently by macrophages than the wild-type (WT) strain.\nComplexity and dynamics of in organello translation landscape assessed by high-resolution mitochondrial ribosome profiling\nAuthors: Wakigawa, T.; Mito, M.; Yamashiro, H.; Tomuro, K.; Tomizawa, K.; Chujo, T.; Nagao, A.; Suzuki, T.; Wei, F.-Y.; Shichino, Y.; Suzuki, T.; Iwasaki, S.\nScore: 5.2, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549812\nEukaryotes house an additional protein synthesis system within the mitochondria. Given that mitochondrial translation dictates OXPHOS complex abundance and ATP levels in cells, exhaustive, quantitative, and high-resolution delineation of mitoribosome traversal is needed. Here, we developed a technique for high-resolution mitochondrial ribosome profiling and unveiled the tight regulation of mammalian in organello translation. Our approach assessed the stoichiometry and kinetics of mitochondrial translation flux, such as the absolute numbers of mitoribosomes on a transcript and the elongation rate, initiation rate, and lifetime rounds of translation of individual transcripts. We also surveyed the impacts of modifications at anticodon stem loop in mt-tRNAs, including all possible decorations at 34th position, by deleting the corresponding enzymes and harnessing patient-derived mtDNA A3243G cells. Moreover, retapamulin-assisted profiling and disome profiling unveiled cryptic translation initiation sites at subcognate codons and programmed mitoribosome collision sites across the mitochondrial transcriptome, respectively. Our work provides a useful resource for delineating protein synthesis within this indispensable organelle.\nA simple mechanism for epigenetic inheritance of silent chromatin.\nAuthors: Yuan, A. H.; Moazed, D.\nScore: 5.1, Published: 2023-07-18 DOI: 10.1101/2023.07.18.549577\nMechanisms enabling genetically identical cells to differentially regulate gene expression are complex and central to organismal development and evolution. While gene silencing pathways involving sequence-specific recruitment of histone-modifying enzymes are prevalent in nature, examples of sequence-independent heritable gene silencing are scarce. Studies of Schizosaccharomyces pombe indicate that sequence-independent propagation of heterochromatin can occur but requires numerous multisubunit protein complexes and their various activities. Such complexity has precluded a coherent articulation of the minimal requirements for heritable gene silencing by conventional approaches. Here, we take an unconventional approach to defining these requirements by engineering sequence-independent silent chromatin inheritance in Saccharomyces cerevisiae. The memory-conferring mechanism is remarkably simple and requires only two proteins, one that recognizes histone H3 methylation and deacetylates histone H4, and another that recognizes unmodified H4 and catalyzes H3 methylation. These bilingual \"read-write\" proteins form an interdependent positive feedback loop capable of transmitting sequence-independent silent information over multiple generations.\n",
  "wordCount" : "2348",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549907">
        <p class="paperTitle">Mechanism of outer kinetochore assembly on microtubules and its regulation by mitotic error correction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muir, K. W.; Batters, C.; Dendooven, T.; Yang, J.; Zhang, Z.; Burt, A.; Barford, D.</p>
        <p class="info">Score: 50.2, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549907' target='https://doi.org/10.1101/2023.07.20.549907'> 10.1101/2023.07.20.549907</a></p>
        <p class="abstract">Kinetochores couple chromosomes to the mitotic spindle and transduce the energy of microtubule depolymerization to segregate the genome during cell division. Kinetochore - microtubule attachments are often initially erroneous and subject to a mitotic error correction (EC) mechanism that drives their turnover until biorientation is achieved. How this is accomplished and regulated, and how kinetochore-mediated chromosome segregation occurs at a molecular level remain major outstanding questions. Here we describe the cryo-electron microscopy (cryo-EM) structure of the budding yeast outer kinetochore Ndc80 and Dam1 ring complexes assembled onto microtubules. We observe coordinated interactions of the outer kinetochore complexes through multiple interfaces, in addition to a short staple within the Dam1 subunit that facilitates Dam1c ring assembly. Perturbation of these interfaces results in loss of yeast viability. Force-rupture assays indicated this is a consequence of substantial reductions in kinetochore - microtubule binding strength. EC-mediated phosphorylation of Ndc80c-Dam1c interfaces would drive complex disassembly, whereas Dam1 staple phosphorylation would promote Dam1c ring disassembly, explaining how kinetochore - microtubule attachments are destabilized and reset by the EC mechanism.

One-Sentence SummaryPhosphorylation disrupts the outer kinetochore to regulate kinetochore-microtubule attachments in mitotic error correction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549714">
        <p class="paperTitle">Systematic mapping and modeling of 3D enhancer-promoter interactions in early mouse embryonic lineages reveal regulatory principles that determine the levels and cell-type specificity of gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murphy, D.; Salataj, E.; Di Giammartino, D. C.; Rodriguez-Hernaez, J.; Kloetgen, A.; Garg, V.; Char, E.; Uyehara, C. M.; Ee, L.-s.; Lee, U.; Stadtfeld, M.; Hadjantonakis, A.-K.; Tsirigos, A.; Polyzos, A.; Apostolou, E.</p>
        <p class="info">Score: 28.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549714' target='https://doi.org/10.1101/2023.07.19.549714'> 10.1101/2023.07.19.549714</a></p>
        <p class="abstract">Mammalian embryogenesis commences with two pivotal and binary cell fate decisions that give rise to three essential lineages, the trophectoderm (TE), the epiblast (EPI) and the primitive endoderm (PrE). Although key signaling pathways and transcription factors that control these early embryonic decisions have been identified, the non-coding regulatory elements via which transcriptional regulators enact these fates remain understudied. To address this gap, we have characterized, at a genome-wide scale, enhancer activity and 3D connectivity in embryo-derived stem cell lines that represent each of the early developmental fates. We observed extensive enhancer remodeling and fine-scale 3D chromatin rewiring among the three lineages, which strongly associate with transcriptional changes, although there are distinct groups of genes that are irresponsive to topological changes. In each lineage, a high degree of connectivity or &#34;hubness&#34; positively correlates with levels of gene expression and enriches for cell-type specific and essential genes. Genes within 3D hubs also show a significantly stronger probability of coregulation across lineages, compared to genes in linear proximity or within the same contact domains. By incorporating 3D chromatin features, we build a novel predictive model for transcriptional regulation (3D-HiChAT), which outperformed models that use only 1D promoter or proximal variables in predicting levels and cell-type specificity of gene expression. Using 3D-HiChAT, we performed genome-wide in silico perturbations to nominate candidate functional enhancers and hubs in each cell lineage, and with CRISPRi experiments we validated several novel enhancers that control expression of one or more genes in their respective lineages. Our study comprehensively identifies 3D regulatory hubs associated with the earliest mammalian lineages and describes their relationship to gene expression and cell identity, providing a framework to understand lineage-specific transcriptional behaviors.

HIGHLIGHTS- Cell lines representing early embryonic lineages undergo drastic enhancer remodeling and fine-scale 3D chromatin reorganization
- Highly interacting 3D hubs strongly enrich for highly expressed, cell-type specific and essential genes
- 3D chromatin features greatly improve prediction of cell-type specific gene expression compared to 1D promoter features
- In silico and experimental perturbations identify novel enhancers regulating the expression of two or more genes in early embryonic lineages</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547284">
        <p class="paperTitle">Efficient Formation of Single-copy Human Artificial Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.</p>
        <p class="info">Score: 67.1, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547284' target='https://doi.org/10.1101/2023.06.30.547284'> 10.1101/2023.06.30.547284</a></p>
        <p class="abstract">Large DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.

One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549732">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549732" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549732">
        <p class="paperTitle">Chainsaw: protein domain segmentation with fullyconvolutional neural networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549732" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549732" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wells, J.; Hawkins-Hooker, A.; Bordin, N.; Orengo, C.; Paige, B.</p>
        <p class="info">Score: 19.3, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549732' target='https://doi.org/10.1101/2023.07.19.549732'> 10.1101/2023.07.19.549732</a></p>
        <p class="abstract">Protein domains are fundamental units of protein structure and play a pivotal role in understanding folding, function, evolution, and design. The advent of accurate structure prediction techniques has resulted in an influx of new structural data, making the partitioning of these structures into domains essential for inferring evolutionary relationships and functional classification. Currently, the most performant computational methods for parsing protein domains are structure-based methods that rely on unsupervised heuristics to determine the optimal segmentation. Given the availability of large curated databases of domain annotations, we find it worthwhile to revisit structure-based approaches but adopt a supervised learning approach. This paper presents Chainsaw, a novel supervised learning approach to domain parsing that achieves accuracy that surpasses current state-of-the-art methods. The Chainsaw method uses a residual convolutional neural network which is trained to predict the adjacency matrix over residues, where element ai j is an estimate for the probability that residues i and j are in the same domain. Domain assignments are then derived from this soft-adjacency matrix using a non-learned algorithm that is equivalent to maximising the likelihood of the assignments given the set of pairwise probabilities expressed in the soft-adjacency matrix. We benchmark the method against recent state-of-the-art structure-based methods and show that it outperforms them, achieving an NDO score of 0.87 vs 0.80 for the next closest method when predicting a non-redundant test set of multi-domain proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.548988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.548988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.548988">
        <p class="paperTitle">Unconventional stabilization of the human T-cell leukemia virus type 1 immature Gag lattice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.548988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.548988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Obr, M.; Percipalle, M.; Chernikova, D.; Yang, H.; Thader, A.; Pinke, G.; Porley, D.; Mansky, L. M.; Dick, R.; Schur, F. K.</p>
        <p class="info">Score: 7.9, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.548988' target='https://doi.org/10.1101/2023.07.24.548988'> 10.1101/2023.07.24.548988</a></p>
        <p class="abstract">Human T-cell leukemia virus type 1 (HTLV-1) has an atypical immature particle morphology compared to other retroviruses. This indicates that these particles are formed in a way that is unique. Here we report the results of cryo-electron tomography (cryo-ET) studies of HTLV-1 virus-like particles (VLPs) assembled in vitro, as well as derived from cells. This work shows that HTLV-1 employs an unconventional mechanism of Gag-Gag interactions to form the immature viral lattice. Analysis of high-resolution structural information from immature CA tubular arrays reveals that the primary stabilizing component in HTLV-1 is CA-NTD. Mutagenesis and biophysical analysis support this observation. This distinguishes HTLV-1 from other retroviruses, in which the stabilization is provided primarily by the CA-CTD. These results are the first to provide structural details of the quaternary arrangement of Gag for an immature deltaretrovirus, and this helps explain why HTLV-1 particles are morphologically distinct.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550135">
        <p class="paperTitle">CysQuant: simultaneous quantification of cysteine oxidation and protein abundance using data dependent or independent acquisition mass spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, J.; Staes, A.; Impens, F.; Demichev, V.; Van Breusegem, F.; Gevaert, K.; Willems, P.</p>
        <p class="info">Score: 7.5, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550135' target='https://doi.org/10.1101/2023.07.22.550135'> 10.1101/2023.07.22.550135</a></p>
        <p class="abstract">Protein cysteinyl thiols are susceptible to reduction-oxidation reactions that can influence protein function. Accurate quantification of cysteine oxidation is therefore crucial for decoding protein redox regulation. Here, we present CysQuant, a novel approach for simultaneous quantification of cysteine oxidation degrees and protein abundancies. CysQuant involves light/heavy iodoacetamide isotopologues for differential labeling of reduced and reversibly oxidized cysteines analyzed by data-dependent acquisition (DDA) or data-independent acquisition mass spectrometry (DIA-MS). Using plexDIA with in silico predicted spectral libraries, we quantified an average of 18% cysteine oxidation in Arabidopsis thaliana by DIA-MS, including a subset of highly oxidized cysteines forming disulfide bridges in AlphaFold2 predicted structures. Applying CysQuant to Arabidopsis seedlings exposed to excessive light, we successfully quantified the well-established increased reduction of Calvin-Benson cycle enzymes and discovered yet uncharacterized redox-sensitive disulfides in chloroplastic enzymes. Overall, CysQuant is a highly versatile tool for assessing the cysteine modification status that can be widely applied across various mass spectrometry platforms and organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543292">
        <p class="paperTitle">Scaling of an antibody validation procedure enables quantification of antibody performance in major research applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ayoubi, R.; Ryan, J.; Biddle, M. S.; Alshafie, W.; Fotouhi, M.; Bolivar, S. G.; Moleon, V. R.; Eckmann, P.; Worrall, D.; McDowell, I.; Southern, K.; Reintsch, W.; Durcan, T. M.; Brown, C. M.; Bandrowski, A. E.; Virk, H. S.; Edwards, A. M.; McPherson, P. S.; Laflamme, C.</p>
        <p class="info">Score: 10.2, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543292' target='https://doi.org/10.1101/2023.06.01.543292'> 10.1101/2023.06.01.543292</a></p>
        <p class="abstract">Antibodies are critical reagents to detect and characterize proteins. It is commonly understood that many commercial antibodies do not recognize their intended targets, but information on the scope of the problem remains largely anecdotal, and as such, feasibility of the goal of at least one potent and specific antibody targeting each protein in a proteome cannot be assessed. Focusing on antibodies for human proteins, we have scaled a standardized characterization approach using parental and knockout cell lines to assess the performance of 614 commercial antibodies for 65 neuroscience-related proteins. Side-by-side comparisons of all antibodies against each target, obtained from multiple commercial partners, demonstrates that: i) more than 50% of all antibodies failed in one or more tests, ii) yet, [~]50-75% of the protein set was covered by at least one high-performing antibody, depending on application, suggesting that coverage of human proteins by commercial antibodies is significant; and iii) recombinant antibodies performed better than monoclonal or polyclonal antibodies. The hundreds of underperforming antibodies identified in this study were found to have been used in a large number of published articles, which should raise alarm. Encouragingly, more than half of the underperforming commercial antibodies were reassessed by the manufacturers, and many had alterations to their recommended usage or were removed from the market. This first such study helps demonstrate the scale of the antibody specificity problem but also suggests an efficient strategy toward achieving coverage of the human proteome; mine the existing commercial antibody repertoire, and use the data to focus new renewable antibody generation efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549460" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549460">
        <p class="paperTitle">Tur1 regulates alternative TSS usage in Cryptococcus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dang, T. T. V.; Maufrais, C.; Colin, J.; Moyrand, F.; Mouyna, I.; Coppee, J.-Y.; Onyishi, C. U.; Lipecka, J.; Guerrera, I. C.; May, R. C.; Janbon, G.</p>
        <p class="info">Score: 5.2, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549460' target='https://doi.org/10.1101/2023.07.18.549460'> 10.1101/2023.07.18.549460</a></p>
        <p class="abstract">Alternative transcription start site (TSS) usage regulation has been identified as a major means of gene expression regulation in metazoans. However, in fungi, its impact remains elusive as its study has thus far been restricted to model yeasts. Here, we first re-analysed TSS-seq data to define genuine TSS clusters in two species of pathogenic Cryptococcus. We identified two types of TSS clusters associated with specific DNA sequence motifs. Our analysis also revealed that alternative TSS usage regulation in response to environmental cues is widespread in Cryptococcus, altering gene expression and protein targeting. Importantly, we performed a forward genetic screen to identify a unique transcription factor (TF) named Tur1, which regulates alternative TSS (altTSS) usage genome-wide when cells switch from exponential phase to stationary phase. Tur1 has been previously shown to be essential for virulence in C. neoformans. We demonstrated here that a tur1{Delta} mutant strain is more sensitive to superoxide stress and phagocytosed more efficiently by macrophages than the wild-type (WT) strain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549812">
        <p class="paperTitle">Complexity and dynamics of in organello translation landscape assessed by high-resolution mitochondrial ribosome profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wakigawa, T.; Mito, M.; Yamashiro, H.; Tomuro, K.; Tomizawa, K.; Chujo, T.; Nagao, A.; Suzuki, T.; Wei, F.-Y.; Shichino, Y.; Suzuki, T.; Iwasaki, S.</p>
        <p class="info">Score: 5.2, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549812' target='https://doi.org/10.1101/2023.07.19.549812'> 10.1101/2023.07.19.549812</a></p>
        <p class="abstract">Eukaryotes house an additional protein synthesis system within the mitochondria. Given that mitochondrial translation dictates OXPHOS complex abundance and ATP levels in cells, exhaustive, quantitative, and high-resolution delineation of mitoribosome traversal is needed. Here, we developed a technique for high-resolution mitochondrial ribosome profiling and unveiled the tight regulation of mammalian in organello translation. Our approach assessed the stoichiometry and kinetics of mitochondrial translation flux, such as the absolute numbers of mitoribosomes on a transcript and the elongation rate, initiation rate, and lifetime rounds of translation of individual transcripts. We also surveyed the impacts of modifications at anticodon stem loop in mt-tRNAs, including all possible decorations at 34th position, by deleting the corresponding enzymes and harnessing patient-derived mtDNA A3243G cells. Moreover, retapamulin-assisted profiling and disome profiling unveiled cryptic translation initiation sites at subcognate codons and programmed mitoribosome collision sites across the mitochondrial transcriptome, respectively. Our work provides a useful resource for delineating protein synthesis within this indispensable organelle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549577">
        <p class="paperTitle">A simple mechanism for epigenetic inheritance of silent chromatin.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, A. H.; Moazed, D.</p>
        <p class="info">Score: 5.1, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549577' target='https://doi.org/10.1101/2023.07.18.549577'> 10.1101/2023.07.18.549577</a></p>
        <p class="abstract">Mechanisms enabling genetically identical cells to differentially regulate gene expression are complex and central to organismal development and evolution. While gene silencing pathways involving sequence-specific recruitment of histone-modifying enzymes are prevalent in nature, examples of sequence-independent heritable gene silencing are scarce. Studies of Schizosaccharomyces pombe indicate that sequence-independent propagation of heterochromatin can occur but requires numerous multisubunit protein complexes and their various activities. Such complexity has precluded a coherent articulation of the minimal requirements for heritable gene silencing by conventional approaches. Here, we take an unconventional approach to defining these requirements by engineering sequence-independent silent chromatin inheritance in Saccharomyces cerevisiae. The memory-conferring mechanism is remarkably simple and requires only two proteins, one that recognizes histone H3 methylation and deacetylates histone H4, and another that recognizes unmodified H4 and catalyzes H3 methylation. These bilingual &#34;read-write&#34; proteins form an interdependent positive feedback loop capable of transmitting sequence-independent silent information over multiple generations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
