<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Programmable RNA writing with trans-splicing
Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.
Score: 118.8, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223
RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Programmable RNA writing with trans-splicing
Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.
Score: 118.8, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223
RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Programmable RNA writing with trans-splicing
Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.
Score: 118.8, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223
RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Programmable RNA writing with trans-splicing\nAuthors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.\nScore: 118.8, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223\nRNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing \u0026amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5\u0026#39; or 3\u0026#39; trans-splicing.",
  "keywords": [
    
  ],
  "articleBody": " Programmable RNA writing with trans-splicing\nAuthors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.\nScore: 118.8, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223\nRNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing \u0026 Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5' or 3' trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington's disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntingtons disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.\nRewriting endogenous human transcripts with trans-splicing\nAuthors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.\nScore: 73.8, Published: 2024-01-30 DOI: 10.1101/2024.01.29.577779\nSplicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.\nSingle-molecule chromatin configurations link transcription factor binding to expression in human cells\nAuthors: Doughty, B. R.; Hinks, M. M.; Schaepe, J. M.; Marinov, G. K.; Thurm, A. R.; Rios-Martinez, C.; Parks, B. E.; Tan, Y.; Marklund, E.; Dubocanin, D.; Bintu, L.; Greenleaf, W. J.\nScore: 64.8, Published: 2024-02-04 DOI: 10.1101/2024.02.02.578660\nThe binding of multiple transcription factors (TFs) to genomic enhancers activates gene expression in mammalian cells. However, the molecular details that link enhancer sequence to TF binding, promoter state, and gene expression levels remain opaque. We applied single-molecule footprinting (SMF) to measure the simultaneous occupancy of TFs, nucleosomes, and components of the transcription machinery on engineered enhancer/promoter constructs with variable numbers of TF binding sites for both a synthetic and an endogenous TF. We find that activation domains enhance a TFs capacity to compete with nucleosomes for binding to DNA in a BAF-dependent manner, TF binding on nucleosome-free DNA is consistent with independent binding between TFs, and average TF occupancy linearly contributes to promoter activation rates. We also decompose TF strength into separable binding and activation terms, which can be tuned and perturbed independently. Finally, we develop thermodynamic and kinetic models that quantitatively predict both the binding microstates observed at the enhancer and subsequent time-dependent gene expression. This work provides a template for quantitative dissection of distinct contributors to gene activation, including the activity of chromatin remodelers, TF activation domains, chromatin acetylation, TF concentration, TF binding affinity, and TF binding site configuration.\nSingle-cell Rapid Capture Hybridization sequencing (scRaCH-seq) to reliably detect isoform usage and coding mutations in targeted genes at a single-cell level\nAuthors: Peng, H.; Jabbari, J. S.; Tian, L.; Chua, C. C.; Anstee, N. S.; Amin, N.; Wei, A. H.; Davidson, N. M.; Roberts, A. W.; Huang, D. C. S.; Ritchie, M. E.; Thijssen, R.\nScore: 14.1, Published: 2024-01-30 DOI: 10.1101/2024.01.30.577942\nSingle-cell long-read sequencing has transformed our understanding of isoform usage and the mutation heterogeneity between cells. Despite unbiased in-depth analysis, the low sequencing throughput often results in insufficient read coverage thereby limiting our ability to perform mutation calling for specific genes. Here, we developed a single-cell Rapid Capture Hybridization sequencing (scRaCH-seq) method that demonstrated high specificity and efficiency in capturing targeted transcripts using long-read sequencing, allowing an in-depth analysis of mutation status and transcript usage for genes of interest. The method includes creating a probe panel for transcript capture, using barcoded primers for pooling and efficient sequencing via Oxford Nanopore Technologies platforms. scRaCH-seq is applicable to stored and indexed single-cell cDNA which allows analysis to be combined with existing short-read RNA-seq datasets. In our investigation of BTK and SF3B1 genes in samples from patients with chronic lymphocytic leukaemia (CLL), we detected SF3B1 isoforms and mutations with high sensitivity. Integration with short-read scRNA-seq data revealed significant gene expression differences in SF3B1-mutated CLL cells, though it did not impact the sensitivity of the anti-cancer drug venetoclax. scRaCH-seqs capability to study long-read transcripts of multiple genes makes it a powerful tool for single-cell genomics.\nNELF coordinates Pol II transcription termination and DNA replication initiation\nAuthors: Nakayama, C.; Daigaku, Y.; Aoi, Y.; Fang, Q.; Kimura, H.; Shilatifard, A.; Tellier, M.; Nojima, T.\nScore: 13.4, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578294\nRegulation of RNA polymerase II (Pol II) transcription is closely associated with cell proliferation. However, it remains unclear how the Pol II transcription program is altered in cancer to favour cell growth. Here, we find that gene expression of NELFCD, a known negative elongation factor, is up-regulated in colorectal tumours. To dissect the direct role of NELF-C on Pol II transcription in such cancer, we employed an auxin-dependent protein degradation system for NELF-C in combination with nascent transcript sequencing technologies. Strikingly, we demonstrated that the acute loss of NELF-C protein globally perturbs Pol II transcription termination and also increases transcription elongation rate, independently of promoter-proximal Pol II pausing. This results in Pol II transcription into DNA replication initiation zones, and may link to failure of the cell cycle transition into S phase. We anticipate that NELF will be a potential therapeutic target to restrict colorectal cancers by promoting transcription-replication conflict. HIGHLIGHTSO_LIExpression of NELFCD transcript is up-regulated in colorectal tumors C_LIO_LINELF-C protein is mandatory for the transition between G1-S phases during cell cycle C_LIO_LINELF-C loss impairs transcription termination independently of Pol II promoter-proximal pausing C_LIO_LINELF-C loss leads Pol II to invade DNA replication initiation zones C_LI\nAn endothelial SOX18-mevalonate pathway axis enables repurposing of statins for infantile hemangioma\nAuthors: Holm, A.; Graus, M.; Wylie-Sears, J.; Borgelt, L.; Tan, J. W. H.; Nasim, S.; Chung, L.; Jain, A.; Sun, M.; Sun, L.; Brouillard, P.; Lekwuttikarn, R.; Kozakewich, H.; Qi, Y. J.; Teng, J.; Mulliken, J.; Vikkula, M.; Francois, M.; Bischoff, J.\nScore: 11.6, Published: 2024-02-02 DOI: 10.1101/2024.01.29.577829\nInfantile hemangioma (IH) is the most common tumor in children and a paradigm for pathological vasculogenesis, angiogenesis and regression. Propranolol is the mainstay of treatment for IH. It inhibits hemangioma vessel formation via a {beta}-adrenergic receptor independent off-target effect of its R(+) enantiomer on the endothelial specific transcription factor sex-determining region Y (SRY) box transcription factor 18 (SOX18). Transcriptomic profiling of patient-derived hemangioma stem cells uncovered the mevalonate pathway (MVP) as a target of R(+) propranolol. Loss of SOX18 function confirmed R(+) propranolol mode of action on the MVP. Functional validation in preclinical IH models revealed that statins - targeting the MVP - are potent inhibitors of hemangioma vessel formation. We propose a novel SOX18-MVP-axis as a central regulator of IH pathogenesis and suggest statin repurposing to treat IH. Our findings reveal novel pleiotropic effects of beta-blockers and statins acting on the SOX18-MVP axis to disable an endothelial specific program in IH, which may impact other scenarios involving pathological vasculogenesis and angiogenesis.\nTranscriptome analysis of the tardigrade Hypsibius exemplaris exposed to the DNA-damaging agent bleomycin\nAuthors: Yoshida, Y.; Hirayama, A.; Arakawa, K.\nScore: 10.2, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578372\nTardigrades are microscopic animals that are renowned for their capabilities of tolerating near-complete desiccation by entering an ametabolic state called anhydrobiosis. However, many species also show high tolerance against radiation in the active state as well, suggesting cross-tolerance via the anhydrobiosis mechanism. Previous studies utilized indirect DNA damaging agents to identify core components of the cross-tolerance machinery; however, it was difficult to distinguish whether transcriptomic changes were the result of DNA damage or residual oxidative stress. To this end, we performed transcriptome analysis on bleomycin-exposed Hypsibius exemplaris. We observed induction of several tardigrade-specific gene families that may be the core components of the cross-tolerance mechanism. We also identified an enrichment of the tryptophan metabolism pathway, which metabolomic analysis suggested the engagement of this pathway in stress tolerance. These results provide several candidates for the core component of the cross-tolerance, as well as possible anhydrobiosis machinery.\nTAF2 condensation in nuclear speckles links basal transcription factor TFIID to RNA splicing\nAuthors: Bhuiyan, T.; Mendoza Sanchez, P. K.; Arecco, N.; Kim, J.; Nizamuddin, S.; Prunotto, A.; Tekman, M.; Biniossek, M. L.; Koidl, S.; Hugel, T.; Arnold, S. J.\nScore: 4.5, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578926\nTFIID is an essential basal transcription factor, crucial for RNA polymerase II (pol II) promoter recognition and transcription initiation. The TFIID complex consists of the TATA-binding protein (TBP) and 13 TBP-associated factors (TAFs) that contain intrinsically disordered regions (IDRs) with currently unknown functions. Here, we show that a conserved IDR drives TAF2 condensation in nuclear speckles, independently of other TFIID subunits. Quantitative mass spectrometry analyses reveal that the TAF2 IDR specifically interacts with the nuclear speckle and spliceosome-associated protein SRRM2. Consequently, TAF2 recruits SRRM2 to TFIID to form non-canonical TFIID-SRRM2 complexes. Reduced SRRM2 recruitment elicits alternative splicing events in RNAs coding for proteins involved in transcription and transmembrane transport. Further, genome-wide binding analyses suggest TAF2 shuttling between nuclear speckles and pol II promoters. This study identifies an IDR of the basal transcription machinery as a molecular guide for protein partitioning into nuclear compartments, controlling protein complex composition and pre-mRNA splicing.\nSparse CBX2 nucleates many Polycomb proteins to promote facultative heterochromatinization of Polycomb target genes\nAuthors: Ingersoll, S.; Trouth, A.; Luo, X.; Espinoza, A.; Wen, J.; Tucker, J.; Astatike, K.; Phiel, C. J.; Kutateladze, T. G.; WU, T. P.; Ramachandran, S.; Ren, X.\nScore: 4.1, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578969\nFacultative heterochromatinization of genomic regulators by Polycomb proteins is essential in development and differentiation; however, the mechanisms remain obscure. Using genetic engineering, molecular approaches, and live-cell single-molecule imaging, we quantify the number of proteins within condensates formed through liquid-liquid phase separation (LLPS) and find that approximately 3 CBX2 proteins nucleate many PRC1 and PRC2 subunits to form one non-stoichiometric condensate. We demonstrate that sparse CBX2 prevents Polycomb proteins from migrating to constitutive heterochromatin, demarcates the spatial boundaries of facultative heterochromatin, controls the deposition of H3K27me3, regulates transcription, and impacts cellular differentiation. Furthermore, we show that LLPS of CBX2 is required for the demarcation and deposition of H3K27me3 and is essential for cellular differentiation. Our findings uncover new functional roles of LLPS in the formation of facultative heterochromatin and unravel a new mechanism by which low-abundant proteins nucleate many other proteins to form compartments that enable them to execute their functions.\nA Mycobacterium tuberculosis secreted virulence factor disrupts host snRNP biogenesis\nAuthors: Chauhan, K.; Passricha, N.; Kapoor, Y.; Datta, D.; Arora, N.; Das, M.; Rao, K.; Singh, L.; Dutt, R.; Sharma, R.; Sarkar, B.; Malakkar, B.; Kalam, H.; Saini, P.; Mehra, L.; Das, P.; Ahuja, V.; Singhal, A.; Nandicoori, V.; Kumar, D.\nScore: 3.7, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579120\nMycobacterium tuberculosis (Mtb) infection of human macrophages triggers extensive dysregulation of host alternative splicing (AS). Here, we present an unexpected mechanism exploited by the Mtb to achieve the same. We identify several bacterial secretory proteins that interact with the host splicing factors (SFs) and alter select RNA splicing events in vitro, in cell and during ex vivo infections. We show bacterial proteins disrupt AS by impairing Small nuclear ribonucleoprotein (snRNP) biogenesis, primarily by interacting with and sequestering U5snRNA and SNRPF, key spliceosome constituents. These interactions are driven, in part, by one of the identified Mtb proteins, Hsr1, whose access to the host cytosol is ESX1-dependent. The MtbDhsr1 is unable to interact and sequester U5snRNA or SNRPF and fails to alter host AS. Consequently, MtbDhsr1 shows compromised survival and pathogenesis in macrophages and mice. Lungs of infected mice show distinct SNRPF staining, which is hsr1 dependent. Similar distinctive SNRPF staining is also noted in human tuberculous granulomas. Moreover, monocytes from tuberculosis patients show substantial splicing dysregulation, indicating splicing defects as a key determinant behind systemic tuberculosis pathology. We propose mycobacterial exploitation of the evolutionarily conserved mechanism of snRNP biogenesis, aided its adaptation for human pathogenesis and provides a unique therapeutic opportunity.\n",
  "wordCount" : "2310",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578223" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578223">
        <p class="paperTitle">Programmable RNA writing with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.</p>
        <p class="info">Score: 118.8, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578223' target='https://doi.org/10.1101/2024.01.31.578223'> 10.1101/2024.01.31.578223</a></p>
        <p class="abstract">RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington&#39;s disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntingtons disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577779">
        <p class="paperTitle">Rewriting endogenous human transcripts with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 73.8, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577779' target='https://doi.org/10.1101/2024.01.29.577779'> 10.1101/2024.01.29.577779</a></p>
        <p class="abstract">Splicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578660" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578660">
        <p class="paperTitle">Single-molecule chromatin configurations link transcription factor binding to expression in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doughty, B. R.; Hinks, M. M.; Schaepe, J. M.; Marinov, G. K.; Thurm, A. R.; Rios-Martinez, C.; Parks, B. E.; Tan, Y.; Marklund, E.; Dubocanin, D.; Bintu, L.; Greenleaf, W. J.</p>
        <p class="info">Score: 64.8, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578660' target='https://doi.org/10.1101/2024.02.02.578660'> 10.1101/2024.02.02.578660</a></p>
        <p class="abstract">The binding of multiple transcription factors (TFs) to genomic enhancers activates gene expression in mammalian cells. However, the molecular details that link enhancer sequence to TF binding, promoter state, and gene expression levels remain opaque. We applied single-molecule footprinting (SMF) to measure the simultaneous occupancy of TFs, nucleosomes, and components of the transcription machinery on engineered enhancer/promoter constructs with variable numbers of TF binding sites for both a synthetic and an endogenous TF. We find that activation domains enhance a TFs capacity to compete with nucleosomes for binding to DNA in a BAF-dependent manner, TF binding on nucleosome-free DNA is consistent with independent binding between TFs, and average TF occupancy linearly contributes to promoter activation rates. We also decompose TF strength into separable binding and activation terms, which can be tuned and perturbed independently. Finally, we develop thermodynamic and kinetic models that quantitatively predict both the binding microstates observed at the enhancer and subsequent time-dependent gene expression. This work provides a template for quantitative dissection of distinct contributors to gene activation, including the activity of chromatin remodelers, TF activation domains, chromatin acetylation, TF concentration, TF binding affinity, and TF binding site configuration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577942" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577942">
        <p class="paperTitle">Single-cell Rapid Capture Hybridization sequencing (scRaCH-seq) to reliably detect isoform usage and coding mutations in targeted genes at a single-cell level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, H.; Jabbari, J. S.; Tian, L.; Chua, C. C.; Anstee, N. S.; Amin, N.; Wei, A. H.; Davidson, N. M.; Roberts, A. W.; Huang, D. C. S.; Ritchie, M. E.; Thijssen, R.</p>
        <p class="info">Score: 14.1, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577942' target='https://doi.org/10.1101/2024.01.30.577942'> 10.1101/2024.01.30.577942</a></p>
        <p class="abstract">Single-cell long-read sequencing has transformed our understanding of isoform usage and the mutation heterogeneity between cells. Despite unbiased in-depth analysis, the low sequencing throughput often results in insufficient read coverage thereby limiting our ability to perform mutation calling for specific genes. Here, we developed a single-cell Rapid Capture Hybridization sequencing (scRaCH-seq) method that demonstrated high specificity and efficiency in capturing targeted transcripts using long-read sequencing, allowing an in-depth analysis of mutation status and transcript usage for genes of interest. The method includes creating a probe panel for transcript capture, using barcoded primers for pooling and efficient sequencing via Oxford Nanopore Technologies platforms. scRaCH-seq is applicable to stored and indexed single-cell cDNA which allows analysis to be combined with existing short-read RNA-seq datasets. In our investigation of BTK and SF3B1 genes in samples from patients with chronic lymphocytic leukaemia (CLL), we detected SF3B1 isoforms and mutations with high sensitivity. Integration with short-read scRNA-seq data revealed significant gene expression differences in SF3B1-mutated CLL cells, though it did not impact the sensitivity of the anti-cancer drug venetoclax. scRaCH-seqs capability to study long-read transcripts of multiple genes makes it a powerful tool for single-cell genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578294" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578294">
        <p class="paperTitle">NELF coordinates Pol II transcription termination and DNA replication initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakayama, C.; Daigaku, Y.; Aoi, Y.; Fang, Q.; Kimura, H.; Shilatifard, A.; Tellier, M.; Nojima, T.</p>
        <p class="info">Score: 13.4, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578294' target='https://doi.org/10.1101/2024.01.31.578294'> 10.1101/2024.01.31.578294</a></p>
        <p class="abstract">Regulation of RNA polymerase II (Pol II) transcription is closely associated with cell proliferation. However, it remains unclear how the Pol II transcription program is altered in cancer to favour cell growth. Here, we find that gene expression of NELFCD, a known negative elongation factor, is up-regulated in colorectal tumours. To dissect the direct role of NELF-C on Pol II transcription in such cancer, we employed an auxin-dependent protein degradation system for NELF-C in combination with nascent transcript sequencing technologies. Strikingly, we demonstrated that the acute loss of NELF-C protein globally perturbs Pol II transcription termination and also increases transcription elongation rate, independently of promoter-proximal Pol II pausing. This results in Pol II transcription into DNA replication initiation zones, and may link to failure of the cell cycle transition into S phase. We anticipate that NELF will be a potential therapeutic target to restrict colorectal cancers by promoting transcription-replication conflict.

HIGHLIGHTSO_LIExpression of NELFCD transcript is up-regulated in colorectal tumors
C_LIO_LINELF-C protein is mandatory for the transition between G1-S phases during cell cycle
C_LIO_LINELF-C loss impairs transcription termination independently of Pol II promoter-proximal pausing
C_LIO_LINELF-C loss leads Pol II to invade DNA replication initiation zones
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577829">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577829" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577829">
        <p class="paperTitle">An endothelial SOX18-mevalonate pathway axis enables repurposing of statins for infantile hemangioma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577829" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577829" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holm, A.; Graus, M.; Wylie-Sears, J.; Borgelt, L.; Tan, J. W. H.; Nasim, S.; Chung, L.; Jain, A.; Sun, M.; Sun, L.; Brouillard, P.; Lekwuttikarn, R.; Kozakewich, H.; Qi, Y. J.; Teng, J.; Mulliken, J.; Vikkula, M.; Francois, M.; Bischoff, J.</p>
        <p class="info">Score: 11.6, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577829' target='https://doi.org/10.1101/2024.01.29.577829'> 10.1101/2024.01.29.577829</a></p>
        <p class="abstract">Infantile hemangioma (IH) is the most common tumor in children and a paradigm for pathological vasculogenesis, angiogenesis and regression. Propranolol is the mainstay of treatment for IH. It inhibits hemangioma vessel formation via a {beta}-adrenergic receptor independent off-target effect of its R(&#43;) enantiomer on the endothelial specific transcription factor sex-determining region Y (SRY) box transcription factor 18 (SOX18). Transcriptomic profiling of patient-derived hemangioma stem cells uncovered the mevalonate pathway (MVP) as a target of R(&#43;) propranolol. Loss of SOX18 function confirmed R(&#43;) propranolol mode of action on the MVP. Functional validation in preclinical IH models revealed that statins - targeting the MVP - are potent inhibitors of hemangioma vessel formation. We propose a novel SOX18-MVP-axis as a central regulator of IH pathogenesis and suggest statin repurposing to treat IH. Our findings reveal novel pleiotropic effects of beta-blockers and statins acting on the SOX18-MVP axis to disable an endothelial specific program in IH, which may impact other scenarios involving pathological vasculogenesis and angiogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578372" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578372">
        <p class="paperTitle">Transcriptome analysis of the tardigrade Hypsibius exemplaris exposed to the DNA-damaging agent bleomycin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoshida, Y.; Hirayama, A.; Arakawa, K.</p>
        <p class="info">Score: 10.2, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578372' target='https://doi.org/10.1101/2024.02.01.578372'> 10.1101/2024.02.01.578372</a></p>
        <p class="abstract">Tardigrades are microscopic animals that are renowned for their capabilities of tolerating near-complete desiccation by entering an ametabolic state called anhydrobiosis. However, many species also show high tolerance against radiation in the active state as well, suggesting cross-tolerance via the anhydrobiosis mechanism. Previous studies utilized indirect DNA damaging agents to identify core components of the cross-tolerance machinery; however, it was difficult to distinguish whether transcriptomic changes were the result of DNA damage or residual oxidative stress. To this end, we performed transcriptome analysis on bleomycin-exposed Hypsibius exemplaris. We observed induction of several tardigrade-specific gene families that may be the core components of the cross-tolerance mechanism. We also identified an enrichment of the tryptophan metabolism pathway, which metabolomic analysis suggested the engagement of this pathway in stress tolerance. These results provide several candidates for the core component of the cross-tolerance, as well as possible anhydrobiosis machinery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578926" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578926">
        <p class="paperTitle">TAF2 condensation in nuclear speckles links basal transcription factor TFIID to RNA splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhuiyan, T.; Mendoza Sanchez, P. K.; Arecco, N.; Kim, J.; Nizamuddin, S.; Prunotto, A.; Tekman, M.; Biniossek, M. L.; Koidl, S.; Hugel, T.; Arnold, S. J.</p>
        <p class="info">Score: 4.5, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578926' target='https://doi.org/10.1101/2024.02.05.578926'> 10.1101/2024.02.05.578926</a></p>
        <p class="abstract">TFIID is an essential basal transcription factor, crucial for RNA polymerase II (pol II) promoter recognition and transcription initiation. The TFIID complex consists of the TATA-binding protein (TBP) and 13 TBP-associated factors (TAFs) that contain intrinsically disordered regions (IDRs) with currently unknown functions. Here, we show that a conserved IDR drives TAF2 condensation in nuclear speckles, independently of other TFIID subunits. Quantitative mass spectrometry analyses reveal that the TAF2 IDR specifically interacts with the nuclear speckle and spliceosome-associated protein SRRM2. Consequently, TAF2 recruits SRRM2 to TFIID to form non-canonical TFIID-SRRM2 complexes. Reduced SRRM2 recruitment elicits alternative splicing events in RNAs coding for proteins involved in transcription and transmembrane transport. Further, genome-wide binding analyses suggest TAF2 shuttling between nuclear speckles and pol II promoters. This study identifies an IDR of the basal transcription machinery as a molecular guide for protein partitioning into nuclear compartments, controlling protein complex composition and pre-mRNA splicing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578969" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578969">
        <p class="paperTitle">Sparse CBX2 nucleates many Polycomb proteins to promote facultative heterochromatinization of Polycomb target genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ingersoll, S.; Trouth, A.; Luo, X.; Espinoza, A.; Wen, J.; Tucker, J.; Astatike, K.; Phiel, C. J.; Kutateladze, T. G.; WU, T. P.; Ramachandran, S.; Ren, X.</p>
        <p class="info">Score: 4.1, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578969' target='https://doi.org/10.1101/2024.02.05.578969'> 10.1101/2024.02.05.578969</a></p>
        <p class="abstract">Facultative heterochromatinization of genomic regulators by Polycomb proteins is essential in development and differentiation; however, the mechanisms remain obscure. Using genetic engineering, molecular approaches, and live-cell single-molecule imaging, we quantify the number of proteins within condensates formed through liquid-liquid phase separation (LLPS) and find that approximately 3 CBX2 proteins nucleate many PRC1 and PRC2 subunits to form one non-stoichiometric condensate. We demonstrate that sparse CBX2 prevents Polycomb proteins from migrating to constitutive heterochromatin, demarcates the spatial boundaries of facultative heterochromatin, controls the deposition of H3K27me3, regulates transcription, and impacts cellular differentiation. Furthermore, we show that LLPS of CBX2 is required for the demarcation and deposition of H3K27me3 and is essential for cellular differentiation. Our findings uncover new functional roles of LLPS in the formation of facultative heterochromatin and unravel a new mechanism by which low-abundant proteins nucleate many other proteins to form compartments that enable them to execute their functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579120">
        <p class="paperTitle">A Mycobacterium tuberculosis secreted virulence factor disrupts host snRNP biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chauhan, K.; Passricha, N.; Kapoor, Y.; Datta, D.; Arora, N.; Das, M.; Rao, K.; Singh, L.; Dutt, R.; Sharma, R.; Sarkar, B.; Malakkar, B.; Kalam, H.; Saini, P.; Mehra, L.; Das, P.; Ahuja, V.; Singhal, A.; Nandicoori, V.; Kumar, D.</p>
        <p class="info">Score: 3.7, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579120' target='https://doi.org/10.1101/2024.02.06.579120'> 10.1101/2024.02.06.579120</a></p>
        <p class="abstract">Mycobacterium tuberculosis (Mtb) infection of human macrophages triggers extensive dysregulation of host alternative splicing (AS). Here, we present an unexpected mechanism exploited by the Mtb to achieve the same. We identify several bacterial secretory proteins that interact with the host splicing factors (SFs) and alter select RNA splicing events in vitro, in cell and during ex vivo infections. We show bacterial proteins disrupt AS by impairing Small nuclear ribonucleoprotein (snRNP) biogenesis, primarily by interacting with and sequestering U5snRNA and SNRPF, key spliceosome constituents. These interactions are driven, in part, by one of the identified Mtb proteins, Hsr1, whose access to the host cytosol is ESX1-dependent. The MtbDhsr1 is unable to interact and sequester U5snRNA or SNRPF and fails to alter host AS. Consequently, MtbDhsr1 shows compromised survival and pathogenesis in macrophages and mice. Lungs of infected mice show distinct SNRPF staining, which is hsr1 dependent. Similar distinctive SNRPF staining is also noted in human tuberculous granulomas. Moreover, monocytes from tuberculosis patients show substantial splicing dysregulation, indicating splicing defects as a key determinant behind systemic tuberculosis pathology. We propose mycobacterial exploitation of the evolutionarily conserved mechanism of snRNP biogenesis, aided its adaptation for human pathogenesis and provides a unique therapeutic opportunity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
