<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure and mechanism of Zorya anti-phage defense system
Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.
Score: 67.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097
Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Structure and mechanism of Zorya anti-phage defense system
Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.
Score: 67.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097
Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Structure and mechanism of Zorya anti-phage defense system
Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.
Score: 67.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097
Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Structure and mechanism of Zorya anti-phage defense system\nAuthors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.\nScore: 67.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097\nZorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion.",
  "keywords": [
    
  ],
  "articleBody": " Structure and mechanism of Zorya anti-phage defense system\nAuthors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.\nScore: 67.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572097\nZorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion. It consists of a predicted membrane-embedded complex (ZorAB) and soluble components that differ among Zorya subtypes, notably ZorC and ZorD, in type I Zorya systems. Here, we reveal the molecular basis of the Zorya defense system using cryo-electron microscopy, mutagenesis, fluorescence microscopy, proteomics, and functional studies. We demonstrate that ZorAB shares the stoichiometry of other 5:2 inner membrane ion-driven rotary motors. Additionally, ZorA5B2 features a dimeric ZorB peptidoglycan binding domain and a pentameric -helical coiled-coil tail made of ZorA that projects approximately 700 [A] into the cytoplasm. We further characterize the structure and function of the soluble Zorya components, ZorC and ZorD, and find that they harbour DNA binding and nuclease activity, respectively. Comprehensive functional and mutational analyses demonstrates that all Zorya components work in concert to protect bacterial cells against invading phages. We present evidence that ZorAB operates as an ion-driven motor that becomes activated and anchors to the cell wall upon sensing of cell envelope perturbations during phage invasion. Subsequently, ZorAB transfers the phage invasion signal through the ZorA cytoplasmic tail to the soluble effectors, which function to prevent phage propagation. In summary, our study elucidates the foundational mechanisms of Zorya function and reveals a novel triggering signal for the rapid activation of an anti-phage defense system.\nTranscriptional memory is conferred by combined heritable maintenance and local removal of selective chromatin modifications\nAuthors: Mikulski, P.; Tehrani, S. S. H.; Kogan, A.; Abdul-Zani, I.; Shell, E.; Ryan, B. J.; Jansen, L. E. T.\nScore: 31.9, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571619\nInterferon-{gamma} (IFN{gamma}) transiently activates genes involved in inflammation and innate immunity. A subset of targets maintain a mitotically heritable memory of prior IFN{gamma} exposure resulting in hyperactivation upon reexposure. Here we discovered that the active chromatin marks H3K4me1, H3K14Ac and H4K16Ac are established during IFN{gamma} priming and selectively maintained on a cluster of GBP genes for at least 7 days in dividing cells in the absence of transcription. The histone acetyltransferase KAT7 is required for the accelerated GBP reactivation upon reexposure to IFN{gamma}. In naive cells, we find the GBP cluster is maintained in low-level repressive chromatin marked by H3K27me3 limiting priming in a PRC2-dependent manner. Unexpectedly, IFN{gamma} results in transient accumulation of this repressive mark but is then selectively depleted from primed GBP genes during the memory phase facilitating hyperactivation of primed cells. Furthermore, we identified a cis-regulatory element that makes transient, long-range contacts across the GBP cluster and acts as a repressor, primarily to curb the hyperactivation of previously IFN{gamma}-primed cells. Combined our results identify the putative chromatin basis for long-term transcriptional memory of interferon signalling that may contribute to enhanced innate immunity.\nRapid DNA unwinding accelerates genome editing by engineered CRISPR-Cas9\nAuthors: Eggers, A. R.; Chen, K.; Soczek, K. M.; Tuck, O. T.; Doherty, E. E.; Thornton, B. W.; Xu, B.; Trinidad, M. I.; Doudna, J. A.\nScore: 31.5, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571777\nThermostable CRISPR-Cas9 enzymes could improve genome editing efficiency and delivery due to extended protein lifetimes. However, initial experimentation demonstrated Geobacillus stearothermophilus Cas9 (GeoCas9) to be virtually inactive when used in cultured human cells. Laboratory-evolved variants of GeoCas9 overcome this natural limitation by acquiring mutations in the wedge (WED) domain that produce \u003e100-fold higher genome editing levels. Cryo-EM structures of the wildtype and improved GeoCas9 (iGeoCas9) enzymes reveal extended contacts between the WED domain of iGeoCas9 and DNA substrates. Biochemical analysis shows that iGeoCas9 accelerates DNA unwinding to capture substrates under the magnesium-restricted conditions typical of mammalian but not bacterial cells. These findings enabled rational engineering of other Cas9 orthologs to enhance genome editing levels, pointing to a general strategy for editing enzyme improvement. Together, these results uncover a new role for the Cas9 WED domain in DNA unwinding and demonstrate how accelerated target unwinding dramatically improves Cas9-induced genome editing activity.\nHigh order expression dependencies finely resolve cryptic states and subtypes in single cell data\nAuthors: Jansma, A.; Yao, Y.; Wolfe, J.; Del Debbio, L.; Beentjes, S. V.; Ponting, C. P.; Khamseh, A.\nScore: 25.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572232\nAO_SCPLOWBSTRACTC_SCPLOWSingle cells are typically typed by clustering in reduced dimensional transcriptome space. Here we introduce Stator, a novel method, workflow and app that reveals cell types, subtypes and states without relying on local proximity of cells in gene expression space. Rather, Stator derives higher-order gene expression dependencies from a sparse gene-by-cell expression matrix. From these dependencies the method multiply labels the same single cell according to type, sub-type and state (activation, differentiation or cell cycle sub-phase). By applying the method to data from mouse embryonic brain, and human healthy or diseased liver, we show how Stator first recapitulates other methods cell type labels, and then reveals combinatorial gene expression markers of cell type, state, and disease at higher resolution. By allowing multiple state labels for single cells we reveal cell type fates of embryonic progenitor cells and liver cancer states associated with patient survival.\nSTOPGAP, an open-source package for template matching, subtomogram alignment, and classification\nAuthors: Wan, W.; Khavnekar, S.; Wagner, J.\nScore: 19.5, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572665\nCryo-electron tomography (cryo-ET) enables molecular-resolution 3D imaging of complex biological specimens such as viral particles, cellular sections, and in some cases, whole cells. This enables the structural characterization of molecules in their near-native environments, without the need for purification or separation, thereby preserving biological information such as conformational states and spatial relationships between different molecular species. Subtomogram averaging is an image processing workflow that allows users to leverage cryo-ET data to identify and localize target molecules, determine high-resolution structures of repeating molecular species, and classifying different conformational states. Here we describe STOPGAP, an open-source package for subtomogram averaging designed to provide users with fine control over each of these steps. In providing detailed descriptions of the image processing algorithms that STOPGAP uses, we intend for this manuscript to also serve as a technical resource to users as well as further community-driven software development.\nSMC motor proteins extrude DNA asymmetrically and contain a direction switch\nAuthors: Barth, R.; Davidson, I.; van der Torre, J.; Taschner, M.; Gruber, S.; Peters, J.-M.; Dekker, C.\nScore: 17.3, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572892\nStructural Maintenance of Chromosomes (SMC) complexes organize the genome via DNA loop extrusion. While some SMCs were reported to do so symmetrically, reeling DNA from both sides into the extruded DNA loop simultaneously, others perform loop extrusion asymmetrically toward one direction only. The mechanism underlying this variability remains unclear. Here, we examine the directionality of DNA loop extrusion by SMCs using in vitro single-molecule experiments. We find that cohesin and SMC5/6 do not reel in DNA from both sides, as reported before, but instead extrude DNA asymmetrically, while the direction can switch over time. Asymmetric DNA loop extrusion thus is the shared mechanism across all eukaryotic SMC complexes. For cohesin, direction switches strongly correlate with the turnover of the subunit NIPBL, during which DNA strand switching may occur. STAG1 stabilizes NIPBL on cohesin, preventing NIPBL turnover and direction switches. The findings reveal that SMCs, surprisingly, contain a direction switch subunit.\nCDCA7 is a hemimethylated DNA adaptor for the nucleosome remodeler HELLS\nAuthors: Wassing, I. E.; Nishiyama, A.; Hiruta, M.; Jia, Q.; Shikimachi, R.; Kikuchi, A.; Sugimura, K.; Hong, X.; Chiba, Y.; Peng, J.; Jenness, C.; Nakanishi, M.; Zhao, L.; Arita, K.; Funabiki, H.\nScore: 15.6, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572350\nMutations of the SNF2 family ATPase HELLS and its activator CDCA7 cause immunodeficiency-centromeric instability-facial anomalies (ICF) syndrome, characterized by hypomethylation at heterochromatin. The unique zinc-finger domain, zf-4CXXC_R1, of CDCA7 is widely conserved across eukaryotes but is absent from species that lack HELLS and DNA methyltransferases, implying its specialized relation with methylated DNA. Here we demonstrate that zf-4CXXC_R1 acts as a hemimethylated DNA sensor. The zf-4CXXC_R1 domain of CDCA7 selectively binds to DNA with a hemimethylated CpG, but not unmethylated or fully methylated CpG, and ICF disease mutations eliminated this binding. CDCA7 and HELLS interact via their N-terminal alpha helices, through which HELLS is recruited to hemimethylated DNA. While placement of a hemimethylated CpG within the nucleosome core particle can hinder its recognition by CDCA7, cryo-EM structure analysis of the CDCA7-nucleosome complex suggests that zf-4CXXC_R1 recognizes a hemimethylated CpG in the major groove at linker DNA. Our study provides insights into how the CDCA7-HELLS nucleosome remodeling complex uniquely assists maintenance DNA methylation.\nAir monitoring by nanopore sequencing\nAuthors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.\nScore: 15.4, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572325\nWhile the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects. We here show that nanopore shotgun sequencing can robustly assess the air microbiome in combination with active air sampling through liquid impingement and tailored computational analysis. We provide laboratory and computational protocols for air microbiome profiling, and reliably assess the taxonomic composition of the core air microbiome in controlled and natural environments. Based on de novo assemblies from the long sequencing reads, we further identify the taxa of several air microbiota down to the species level and annotate their ecosystem functions, including their potential role in natural biodegradation processes. As air monitoring by nanopore sequencing can be employed in an automatable, fast, and portable manner for in situ applications all around the world, we envision that this approach can help holistically explore the role of the air microbiome.\nBarcode 100K Specimens: In a Single Nanopore Run\nAuthors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.\nScore: 116.8, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282\nIt is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.\nConcerted SUMO-targeted ubiquitin ligase activities of TOPORS and RNF4 are essential for stress management and cell proliferation\nAuthors: Liu, J. C. Y.; Ackermann, L.; Hoffmann, S.; Gal, Z.; Hendriks, I. A.; Jain, C.; Morlot, L.; Tatham, M. H.; McLelland, G.-L.; Hay, R. T.; Nielsen, M. L.; Brummelkamp, T. R.; Haahr, P.; Mailand, N.\nScore: 12.1, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572718\nProtein SUMOylation provides a principal driving force for cellular stress responses including DNA-protein crosslink (DPC) repair and arsenic-induced PML body degradation. In genome-scale screens, we identified the human E3 ligase TOPORS as a key effector of SUMO-dependent DPC resolution. We demonstrate that TOPORS promotes DPC repair by functioning as a SUMO-targeted ubiquitin ligase (STUbL) for DPCs, combining ubiquitin ligase activity through its RING domain with poly-SUMO chain binding via a cluster of SUMO-interacting motifs, analogous to the STUbL RNF4. Surprisingly, the STUbL activities of TOPORS and RNF4 are both required for SUMO-dependent DPC repair, PML degradation and other stress responses, making overlapping and distinct contributions to ubiquitin chain formation on SUMOylated targets to enable p97/VCP unfoldase recruitment. Combined loss of TOPORS and RNF4 is synthetic lethal even in unstressed cells, leading to defective clearance of SUMOylated proteins from chromatin accompanied by cell cycle arrest and apoptosis. Together, our findings establish TOPORS as a novel STUbL whose concerted action with RNF4 defines a general mechanistic principle in crucial cellular processes governed by direct SUMO-ubiquitin crosstalk.\n",
  "wordCount" : "2184",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572097">
        <p class="paperTitle">Structure and mechanism of Zorya anti-phage defense system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, H.; Hughes, T. C. D.; Popp, P. F.; Roa-Eguiara, A.; Martin, F. J. O.; Rutbeek, N. R.; Hendriks, I. A.; Payne, L. J.; Yan, Y.; Sousa, V. K. d.; Wang, Y.; Nielsen, M. L.; Berry, R. M.; Erhardt, M.; Jackson, S. A.; Taylor, N. M. I.</p>
        <p class="info">Score: 67.0, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572097' target='https://doi.org/10.1101/2023.12.18.572097'> 10.1101/2023.12.18.572097</a></p>
        <p class="abstract">Zorya is a recently identified and widely distributed bacterial immune system, which protects against phage invasion. It consists of a predicted membrane-embedded complex (ZorAB) and soluble components that differ among Zorya subtypes, notably ZorC and ZorD, in type I Zorya systems. Here, we reveal the molecular basis of the Zorya defense system using cryo-electron microscopy, mutagenesis, fluorescence microscopy, proteomics, and functional studies. We demonstrate that ZorAB shares the stoichiometry of other 5:2 inner membrane ion-driven rotary motors. Additionally, ZorA5B2 features a dimeric ZorB peptidoglycan binding domain and a pentameric -helical coiled-coil tail made of ZorA that projects approximately 700 [A] into the cytoplasm. We further characterize the structure and function of the soluble Zorya components, ZorC and ZorD, and find that they harbour DNA binding and nuclease activity, respectively. Comprehensive functional and mutational analyses demonstrates that all Zorya components work in concert to protect bacterial cells against invading phages. We present evidence that ZorAB operates as an ion-driven motor that becomes activated and anchors to the cell wall upon sensing of cell envelope perturbations during phage invasion. Subsequently, ZorAB transfers the phage invasion signal through the ZorA cytoplasmic tail to the soluble effectors, which function to prevent phage propagation. In summary, our study elucidates the foundational mechanisms of Zorya function and reveals a novel triggering signal for the rapid activation of an anti-phage defense system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571619" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571619">
        <p class="paperTitle">Transcriptional memory is conferred by combined heritable maintenance and local removal of selective chromatin modifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mikulski, P.; Tehrani, S. S. H.; Kogan, A.; Abdul-Zani, I.; Shell, E.; Ryan, B. J.; Jansen, L. E. T.</p>
        <p class="info">Score: 31.9, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571619' target='https://doi.org/10.1101/2023.12.16.571619'> 10.1101/2023.12.16.571619</a></p>
        <p class="abstract">Interferon-{gamma} (IFN{gamma}) transiently activates genes involved in inflammation and innate immunity. A subset of targets maintain a mitotically heritable memory of prior IFN{gamma} exposure resulting in hyperactivation upon reexposure. Here we discovered that the active chromatin marks H3K4me1, H3K14Ac and H4K16Ac are established during IFN{gamma} priming and selectively maintained on a cluster of GBP genes for at least 7 days in dividing cells in the absence of transcription. The histone acetyltransferase KAT7 is required for the accelerated GBP reactivation upon reexposure to IFN{gamma}. In naive cells, we find the GBP cluster is maintained in low-level repressive chromatin marked by H3K27me3 limiting priming in a PRC2-dependent manner. Unexpectedly, IFN{gamma} results in transient accumulation of this repressive mark but is then selectively depleted from primed GBP genes during the memory phase facilitating hyperactivation of primed cells. Furthermore, we identified a cis-regulatory element that makes transient, long-range contacts across the GBP cluster and acts as a repressor, primarily to curb the hyperactivation of previously IFN{gamma}-primed cells. Combined our results identify the putative chromatin basis for long-term transcriptional memory of interferon signalling that may contribute to enhanced innate immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571777">
        <p class="paperTitle">Rapid DNA unwinding accelerates genome editing by engineered CRISPR-Cas9</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eggers, A. R.; Chen, K.; Soczek, K. M.; Tuck, O. T.; Doherty, E. E.; Thornton, B. W.; Xu, B.; Trinidad, M. I.; Doudna, J. A.</p>
        <p class="info">Score: 31.5, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571777' target='https://doi.org/10.1101/2023.12.14.571777'> 10.1101/2023.12.14.571777</a></p>
        <p class="abstract">Thermostable CRISPR-Cas9 enzymes could improve genome editing efficiency and delivery due to extended protein lifetimes. However, initial experimentation demonstrated Geobacillus stearothermophilus Cas9 (GeoCas9) to be virtually inactive when used in cultured human cells. Laboratory-evolved variants of GeoCas9 overcome this natural limitation by acquiring mutations in the wedge (WED) domain that produce &gt;100-fold higher genome editing levels. Cryo-EM structures of the wildtype and improved GeoCas9 (iGeoCas9) enzymes reveal extended contacts between the WED domain of iGeoCas9 and DNA substrates. Biochemical analysis shows that iGeoCas9 accelerates DNA unwinding to capture substrates under the magnesium-restricted conditions typical of mammalian but not bacterial cells. These findings enabled rational engineering of other Cas9 orthologs to enhance genome editing levels, pointing to a general strategy for editing enzyme improvement. Together, these results uncover a new role for the Cas9 WED domain in DNA unwinding and demonstrate how accelerated target unwinding dramatically improves Cas9-induced genome editing activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572232">
        <p class="paperTitle">High order expression dependencies finely resolve cryptic states and subtypes in single cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansma, A.; Yao, Y.; Wolfe, J.; Del Debbio, L.; Beentjes, S. V.; Ponting, C. P.; Khamseh, A.</p>
        <p class="info">Score: 25.0, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572232' target='https://doi.org/10.1101/2023.12.18.572232'> 10.1101/2023.12.18.572232</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWSingle cells are typically typed by clustering in reduced dimensional transcriptome space. Here we introduce Stator, a novel method, workflow and app that reveals cell types, subtypes and states without relying on local proximity of cells in gene expression space. Rather, Stator derives higher-order gene expression dependencies from a sparse gene-by-cell expression matrix. From these dependencies the method multiply labels the same single cell according to type, sub-type and state (activation, differentiation or cell cycle sub-phase). By applying the method to data from mouse embryonic brain, and human healthy or diseased liver, we show how Stator first recapitulates other methods cell type labels, and then reveals combinatorial gene expression markers of cell type, state, and disease at higher resolution. By allowing multiple state labels for single cells we reveal cell type fates of embryonic progenitor cells and liver cancer states associated with patient survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572665">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572665" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572665">
        <p class="paperTitle">STOPGAP, an open-source package for template matching, subtomogram alignment, and classification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572665" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572665" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wan, W.; Khavnekar, S.; Wagner, J.</p>
        <p class="info">Score: 19.5, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572665' target='https://doi.org/10.1101/2023.12.20.572665'> 10.1101/2023.12.20.572665</a></p>
        <p class="abstract">Cryo-electron tomography (cryo-ET) enables molecular-resolution 3D imaging of complex biological specimens such as viral particles, cellular sections, and in some cases, whole cells. This enables the structural characterization of molecules in their near-native environments, without the need for purification or separation, thereby preserving biological information such as conformational states and spatial relationships between different molecular species. Subtomogram averaging is an image processing workflow that allows users to leverage cryo-ET data to identify and localize target molecules, determine high-resolution structures of repeating molecular species, and classifying different conformational states. Here we describe STOPGAP, an open-source package for subtomogram averaging designed to provide users with fine control over each of these steps. In providing detailed descriptions of the image processing algorithms that STOPGAP uses, we intend for this manuscript to also serve as a technical resource to users as well as further community-driven software development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572892">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572892" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572892">
        <p class="paperTitle">SMC motor proteins extrude DNA asymmetrically and contain a direction switch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572892" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572892" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barth, R.; Davidson, I.; van der Torre, J.; Taschner, M.; Gruber, S.; Peters, J.-M.; Dekker, C.</p>
        <p class="info">Score: 17.3, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572892' target='https://doi.org/10.1101/2023.12.21.572892'> 10.1101/2023.12.21.572892</a></p>
        <p class="abstract">Structural Maintenance of Chromosomes (SMC) complexes organize the genome via DNA loop extrusion. While some SMCs were reported to do so symmetrically, reeling DNA from both sides into the extruded DNA loop simultaneously, others perform loop extrusion asymmetrically toward one direction only. The mechanism underlying this variability remains unclear. Here, we examine the directionality of DNA loop extrusion by SMCs using in vitro single-molecule experiments. We find that cohesin and SMC5/6 do not reel in DNA from both sides, as reported before, but instead extrude DNA asymmetrically, while the direction can switch over time. Asymmetric DNA loop extrusion thus is the shared mechanism across all eukaryotic SMC complexes. For cohesin, direction switches strongly correlate with the turnover of the subunit NIPBL, during which DNA strand switching may occur. STAG1 stabilizes NIPBL on cohesin, preventing NIPBL turnover and direction switches. The findings reveal that SMCs, surprisingly, contain a direction switch subunit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572350">
        <p class="paperTitle">CDCA7 is a hemimethylated DNA adaptor for the nucleosome remodeler HELLS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wassing, I. E.; Nishiyama, A.; Hiruta, M.; Jia, Q.; Shikimachi, R.; Kikuchi, A.; Sugimura, K.; Hong, X.; Chiba, Y.; Peng, J.; Jenness, C.; Nakanishi, M.; Zhao, L.; Arita, K.; Funabiki, H.</p>
        <p class="info">Score: 15.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572350' target='https://doi.org/10.1101/2023.12.19.572350'> 10.1101/2023.12.19.572350</a></p>
        <p class="abstract">Mutations of the SNF2 family ATPase HELLS and its activator CDCA7 cause immunodeficiency-centromeric instability-facial anomalies (ICF) syndrome, characterized by hypomethylation at heterochromatin. The unique zinc-finger domain, zf-4CXXC_R1, of CDCA7 is widely conserved across eukaryotes but is absent from species that lack HELLS and DNA methyltransferases, implying its specialized relation with methylated DNA. Here we demonstrate that zf-4CXXC_R1 acts as a hemimethylated DNA sensor. The zf-4CXXC_R1 domain of CDCA7 selectively binds to DNA with a hemimethylated CpG, but not unmethylated or fully methylated CpG, and ICF disease mutations eliminated this binding. CDCA7 and HELLS interact via their N-terminal alpha helices, through which HELLS is recruited to hemimethylated DNA. While placement of a hemimethylated CpG within the nucleosome core particle can hinder its recognition by CDCA7, cryo-EM structure analysis of the CDCA7-nucleosome complex suggests that zf-4CXXC_R1 recognizes a hemimethylated CpG in the major groove at linker DNA. Our study provides insights into how the CDCA7-HELLS nucleosome remodeling complex uniquely assists maintenance DNA methylation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572325">
        <p class="paperTitle">Air monitoring by nanopore sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reska, T.; Pozdniakova, S.; Borras, S.; Schloter, M.; Canas, L.; Fenn, A.; Rodo, X.; Winkler, B.; Schnitzler, J.-P.; Urban, L.</p>
        <p class="info">Score: 15.4, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572325' target='https://doi.org/10.1101/2023.12.19.572325'> 10.1101/2023.12.19.572325</a></p>
        <p class="abstract">While the air microbiome and its diversity are known to be essential for human health and ecosystem resilience, the current lack of holistic air microbial diversity monitoring means that little is known about the air microbiomes composition, distribution, or functional effects. We here show that nanopore shotgun sequencing can robustly assess the air microbiome in combination with active air sampling through liquid impingement and tailored computational analysis. We provide laboratory and computational protocols for air microbiome profiling, and reliably assess the taxonomic composition of the core air microbiome in controlled and natural environments. Based on de novo assemblies from the long sequencing reads, we further identify the taxa of several air microbiota down to the species level and annotate their ecosystem functions, including their potential role in natural biodegradation processes. As air monitoring by nanopore sequencing can be employed in an automatable, fast, and portable manner for in situ applications all around the world, we envision that this approach can help holistically explore the role of the air microbiome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569282">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569282" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569282">
        <p class="paperTitle">Barcode 100K Specimens: In a Single Nanopore Run</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569282" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569282" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.</p>
        <p class="info">Score: 116.8, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569282' target='https://doi.org/10.1101/2023.11.29.569282'> 10.1101/2023.11.29.569282</a></p>
        <p class="abstract">It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572718" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572718">
        <p class="paperTitle">Concerted SUMO-targeted ubiquitin ligase activities of TOPORS and RNF4 are essential for stress management and cell proliferation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, J. C. Y.; Ackermann, L.; Hoffmann, S.; Gal, Z.; Hendriks, I. A.; Jain, C.; Morlot, L.; Tatham, M. H.; McLelland, G.-L.; Hay, R. T.; Nielsen, M. L.; Brummelkamp, T. R.; Haahr, P.; Mailand, N.</p>
        <p class="info">Score: 12.1, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572718' target='https://doi.org/10.1101/2023.12.20.572718'> 10.1101/2023.12.20.572718</a></p>
        <p class="abstract">Protein SUMOylation provides a principal driving force for cellular stress responses including DNA-protein crosslink (DPC) repair and arsenic-induced PML body degradation. In genome-scale screens, we identified the human E3 ligase TOPORS as a key effector of SUMO-dependent DPC resolution. We demonstrate that TOPORS promotes DPC repair by functioning as a SUMO-targeted ubiquitin ligase (STUbL) for DPCs, combining ubiquitin ligase activity through its RING domain with poly-SUMO chain binding via a cluster of SUMO-interacting motifs, analogous to the STUbL RNF4. Surprisingly, the STUbL activities of TOPORS and RNF4 are both required for SUMO-dependent DPC repair, PML degradation and other stress responses, making overlapping and distinct contributions to ubiquitin chain formation on SUMOylated targets to enable p97/VCP unfoldase recruitment. Combined loss of TOPORS and RNF4 is synthetic lethal even in unstressed cells, leading to defective clearance of SUMOylated proteins from chromatin accompanied by cell cycle arrest and apoptosis. Together, our findings establish TOPORS as a novel STUbL whose concerted action with RNF4 defines a general mechanistic principle in crucial cellular processes governed by direct SUMO-ubiquitin crosstalk.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
