<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes
Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.
Score: 59.9, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727
RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes
Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.
Score: 59.9, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727
RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes
Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.
Score: 59.9, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727
RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes\nAuthors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.\nScore: 59.9, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727\nRNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing.",
  "keywords": [
    
  ],
  "articleBody": " Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes\nAuthors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.\nScore: 59.9, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727\nRNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing. Some of these enzymes are components of insertion sequences (IS) in the IS200/IS605 and IS607 transposon families. Both IS families encode a TnpA transposase and TnpB nuclease, an RNA-guided enzyme ancestral to CRISPR-Cas12. In eukaryotes and their viruses, TnpB homologs occur as two distinct types, Fanzor1 and Fanzor2. We analyzed the evolutionary relationships between prokaryotic TnpBs and eukaryotic Fanzors, revealing that a clade of IS607 TnpBs with unusual active site arrangement found primarily in Cyanobacteriota likely gave rise to both types of Fanzors. The wide-spread nature of Fanzors imply that the properties of this particular group of IS607 TnpBs were particularly suited to adaptation and evolution in eukaryotes and their viruses. Experimental characterization of a prokaryotic IS607 TnpB and virally encoded Fanzor1s uncovered features that may have fostered coevolution between TnpBs/Fanzors and their cognate transposases. Our results provide insight into the evolutionary origins of a ubiquitous family of RNA-guided proteins that shows remarkable conservation across domains of life.\nOrchestration of pluripotent stem cell genome reactivation during mitotic exit\nAuthors: Placzek, S.; Vanzan, L.; Suter, D. M.\nScore: 19.0, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552605\nThe maintenance of cell identity faces many challenges during mitosis, as most DNA-binding proteins are evicted from DNA and transcription is virtually abolished. How cells maintain their identity through cell division and faithfully re-initiate gene expression during mitotic exit is unclear. Here, we developed a novel reporter system enabling cell cycle synchronization-free separation of pluripotent stem cells in temporal bins of \u003c 30 minutes during mitotic exit. This allowed us to quantify genome-wide reactivation of transcription, sequential changes in chromatin accessibility, and re-binding of the pluripotency transcription factors OCT4, SOX2, and NANOG (OSN). We found that transcriptional activity progressively ramped up after mitosis and that OSN rapidly reoccupied the genome during the anaphase-telophase transition. We also demonstrate transcription factor-specific, dynamic relocation patterns and a hierarchical reorganization of the OSN binding landscape governed by OCT4 and SOX2. Our study sheds light on the dynamic orchestration of transcriptional reactivation after mitosis.\nThe SARM1 TIR domain produces glycocyclic ADPR molecules as minor products\nAuthors: Garb, J.; Amitai, G.; Lu, A.; Ofir, G.; Brandis, A.; Mehlman, T.; Kranzusch, P.; Sorek, R.\nScore: 17.6, Published: 2023-08-10 DOI: 10.1101/2023.08.10.552750\nSterile alpha and TIR motif-containing 1 (SARM1) is a protein involved in programmed death of injured axons. Following axon injury or a drug-induced insult, the TIR domain of SARM1 degrades the essential molecule nicotinamide adenine dinucleotide (NAD+), leading to a form of axonal death called Wallerian degeneration. Degradation of NAD+ by SARM1 is essential for the Wallerian degeneration process, but accumulating evidence suggest that other activities of SARM1, beyond the mere degradation of NAD+, may be necessary for programmed axonal death. In this study we show that the TIR domains of both human and fruit fly SARM1 produce 1''-2' and 1''-3' glycocyclic ADP-ribose (gcADPR) molecules as minor products. As previously reported, we observed that SARM1 TIR domains mostly convert NAD+ to ADPR (for human SARM1) or cADPR (in the case of SARM1 from Drosophila melanogaster). However, we now show that human and Drosophila SARM1 additionally convert [~]0.1-0.5% of NAD+ into gcADPR molecules. We find that SARM1 TIR domains produce gcADPR molecules both when purified in vitro and when expressed in bacterial cells. Given that gcADPR is a second messenger involved in programmed cell death in bacteria and likely in plants, we propose that gcADPR may play a role in SARM1-induced programmed axonal death in animals.\nN-glycosylation as a eukaryotic protective mechanism against protein aggregation\nAuthors: Duran-Romana, R.; Houben, B.; De Vleeschouwer, M.; Louros, N.; Wilson, M. P.; Matthijs, G.; Schymkowitz, J.; Rousseau, F.\nScore: 12.9, Published: 2023-08-11 DOI: 10.1101/2023.08.11.552904\nThe tendency for proteins to form aggregates is an inherent part of every proteome and arises from the self-assembly of short protein segments called aggregation-prone regions (APRs). While post-translational modifications (PTMs) have been implicated in modulating protein aggregation, their direct role in APRs remains poorly understood. In this study, we used a combination of proteome-wide computational analyses and biochemical techniques to investigate the potential involvement of PTMs in aggregation regulation. Our findings reveal that while most PTM types are disfavored near APRs, N-glycosylation is enriched and evolutionarily selected, especially in proteins prone to misfolding. Experimentally, we show that N-glycosylation inhibits the aggregation of peptides in vitro through steric hindrance. Moreover, mining existing proteomics data, we find that the loss of N-glycans at the flanks of APRs leads to specific protein aggregation in Neuro2a cells. Our results point towards a novel intrinsic role for N-glycosylation, directly preventing protein aggregation in eukaryotes.\nAePUb promoter length modulates gene expression in Aedes aegypti\nAuthors: Anderson, M. A. E.; Leftwich, P. T.; Wilson, R.; Carabajal Paladino, L. Z.; Basu, S.; Rooney, S.; Adelman, Z.; Alphey, L.\nScore: 12.9, Published: 2023-08-11 DOI: 10.1101/2023.08.11.553001\nMolecular tools for modulating transgene expression in Aedes aegypti are few. Here we demonstrate that adjustments to the AePUb promoter length can alter expression levels of two reporter proteins in Ae. aegypti cell culture and in mosquitoes. This provides a simple means for increasing or decreasing expression of a gene of interest and easy translation from cells to whole insects.\nMYC and MAX drive the reactivation of the genome after mitosis\nAuthors: Gonzalez, I.; Chervova, A.; Escoll Guerrero, P.; Altamirano-Pacheco, L.; Mueller, F.; Dubois, A.; Navarro, P.\nScore: 11.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553120\nShortly after cell division, a robust wave of hyper-transcription reactivates the genome.1-3 This phenomenon is particularly pronounced in pluripotent cells,4 which necessitate rapid transcriptome reactivation to maintain their undifferentiated state and prevent premature differentiation. While recent work has illuminated how specific groups of genes are reactivated,4-8 the mechanisms enabling the global, efficient and accurate post-mitotic reactivation of the genome remain unknown. Here we elucidate the direct involvement of the MYC/MAX transcription factors in the post-mitotic reactivation of pluripotent mouse embryonic stem cells. While MYC undergoes extensive phosphorylation and largely dissociates from its DNA binding sites during mitosis, we report that MAX remains bound to its targets, preferentially at promoters, and facilitates early recruitment of MYC following mitosis. Through the application of MYC/MAX heterodimerization inhibitors, we demonstrate their indispensable role in sustaining hyper-transcription in ES cells, including during the critical transition from mitosis to G1 phase. Our findings uncover a novel role for MAX in mitotic bookmarking, highlighting its pivotal role in post-mitotic MYC recruitment and the re-establishment of high global transcription levels. These findings hold significant implications for medically relevant contexts, particularly when cell proliferation is of paramount importance.9 We anticipate that the study of mitotic bookmarking by MYC and MAX and of the effects of anti-cancer drugs targeting MYC/MAX interactions in such process10-12 will be relevant for our understanding of cancer and its potential treatments.\nStructural basis of archaeal FttA-dependent transcription termination\nAuthors: Wang, C.; Molodtsov, V.; Sanders, T. J.; Marshall, C. J.; Firlar, E.; Kaelber, J. T.; Santangelo, T. J.; Ebright, R. H.\nScore: 8.4, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552649\nThe ribonuclease FttA mediates factor-dependent transcription termination in archaea1-3. Here, we report the structure of a Thermococcus kodakarensis transcription pre-termination complex comprising FttA, Spt4, Spt5, and a transcription elongation complex (TEC). The structure shows that FttA interacts with the TEC in a manner that enables RNA to proceed directly from the TEC RNA-exit channel to the FttA catalytic center and that enables endonucleolytic cleavage of RNA by FttA, followed by 5[-\u0026gt;]3 exonucleolytic cleavage of RNA by FttA and concomitant 5[-\u0026gt;]3 translocation of FttA on RNA, to apply mechanical force to the TEC and trigger termination. The structure further reveals that Spt5 bridges FttA and the TEC, explaining how Spt5 stimulates FttA-dependent termination. The results reveal functional analogy between bacterial and archaeal factor-dependent termination, reveal functional homology between archaeal and eukaryotic factor-dependent termination, and reveal fundamental mechanistic similarities in factor-dependent termination in the three domains of life: bacterial, archaeal, and eukaryotic. One sentence summaryCryo-EM reveals the structure of the archaeal FttA pre-termination complex\nShort oligomers rather than rings of human RAD52 promote single-strand annealing\nAuthors: Kharlamova, M. A.; Kushwah, M. S.; Jachowski, T. J.; Subramaniam, S.; Stewart, A. F.; Kukura, P.; Schäffer, E.\nScore: 7.3, Published: 2023-08-11 DOI: 10.1101/2023.08.11.553006\nGenome maintenance and stability rely on the repair of DNA double-strand breaks. The break repair can be mediated by the single-strand annealing protein RAD52. RAD52 forms rings that are thought to promote annealing. However, RAD52s annealing activity decreases with increasing concentrations that favor ring formation. Thus, which oligomeric form and how RAD52 anneals DNA strands and detects sequence homology is unclear. We combine mass photometry with biochemical assays to quantify oligomeric states of human RAD52 with and without DNA and put forward an alternative mechanism illustrating the critical role of short oligomers for single-stranded DNA annealing. We found that while truncated RAD52 formed undecameric rings at nanomolar concentrations, full-length RAD52 was mostly monomeric at lower nanomolar, physiological concentrations. At higher concentrations, it formed rings with a variable stoichiometry from heptamers to tridecamers. At low concentrations, with hardly any rings present, RAD52 already promoted single-strand annealing. Rings and short oligomers could bind at least two single DNA strands, but if complementary strands were both bound to rings annealing was inhibited. Our findings suggest that single-strand annealing and homology detection is mediated by short oligomers of RAD52 instead of rings.\nRNA binding limits the ligand induced transcriptional potential of estrogen receptor-alpha\nAuthors: Soota, D.; Saravanan, B.; Mann, R.; Kharbanda, T.; Notani, D.\nScore: 6.0, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552751\nTranscription factors (TFs) primarily regulate gene expression by binding to DNA through their DNA binding domains (DBD). Additionally, approximately half of these TFs also interact with RNA. However, the role of RNA in enabling TF binding on chromatin and subsequent transcription is poorly understood. Estrogen receptor- (ER) is one such TF that activates genes in response to estrogen stimulation. Here, we report that ER interacts with various types of RNAs in ligand dependent manner via its RNA binding motif in hinge region. RNA binding defects lead to a global loss of ER binding in the genome, particularly at weaker ER motifs. In the absence of RNA binding, the ER exhibits dynamic behavior in the nucleus and unexpectedly, the dynamic binding coincides with robust polymerase loading on ER bound chromatin regions. The higher occupancy of PolII was recapitulated by robust ligand induced transcription of ER-regulated genes. Collectively, our results suggest that RNA interactions strengthen ER binding to chromatin limiting the ligand-dependent transcriptional upregulation of estrogen-induced genes.\nMolecular basis for proofreading by the unique exonuclease domain of Family-D DNA polymerases\nAuthors: Betancurt-Anzola, L.; Martinez-Carranza, M.; Delarue, M.; Zatopek, K.; Gardner, A.; Sauguet, L.\nScore: 5.7, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552591\nReplicative DNA polymerases duplicate entire genomes at high fidelity. This feature is shared among the three domains of life and is facilitated by their dual polymerase and exonuclease activities. Family D replicative DNA polymerases (PolD), found exclusively in Archaea, contain an unusual RNA polymerase-like catalytic core, and a unique Mre11-like proofreading active site. Here, we present cryo-EM structures of PolD trapped in a proofreading mode, revealing an unanticipated correction mechanism that extends the repertoire of protein domains known to be involved in DNA proofreading. Based on our experimental structures, mutants of PolD were designed and their contribution to mismatch bypass and exonuclease kinetics was determined. This study sheds light on the convergent evolution of structurally distinct families of DNA polymerases, and the domain acquisition and exchange mechanism that occurred during the evolution of the replisome in the three domains of life.\n",
  "wordCount" : "1972",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552727">
        <p class="paperTitle">Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.</p>
        <p class="info">Score: 59.9, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552727' target='https://doi.org/10.1101/2023.08.09.552727'> 10.1101/2023.08.09.552727</a></p>
        <p class="abstract">RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing. Some of these enzymes are components of insertion sequences (IS) in the IS200/IS605 and IS607 transposon families. Both IS families encode a TnpA transposase and TnpB nuclease, an RNA-guided enzyme ancestral to CRISPR-Cas12. In eukaryotes and their viruses, TnpB homologs occur as two distinct types, Fanzor1 and Fanzor2. We analyzed the evolutionary relationships between prokaryotic TnpBs and eukaryotic Fanzors, revealing that a clade of IS607 TnpBs with unusual active site arrangement found primarily in Cyanobacteriota likely gave rise to both types of Fanzors. The wide-spread nature of Fanzors imply that the properties of this particular group of IS607 TnpBs were particularly suited to adaptation and evolution in eukaryotes and their viruses. Experimental characterization of a prokaryotic IS607 TnpB and virally encoded Fanzor1s uncovered features that may have fostered coevolution between TnpBs/Fanzors and their cognate transposases. Our results provide insight into the evolutionary origins of a ubiquitous family of RNA-guided proteins that shows remarkable conservation across domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552605">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552605" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552605">
        <p class="paperTitle">Orchestration of pluripotent stem cell genome reactivation during mitotic exit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552605" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552605" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Placzek, S.; Vanzan, L.; Suter, D. M.</p>
        <p class="info">Score: 19.0, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552605' target='https://doi.org/10.1101/2023.08.09.552605'> 10.1101/2023.08.09.552605</a></p>
        <p class="abstract">The maintenance of cell identity faces many challenges during mitosis, as most DNA-binding proteins are evicted from DNA and transcription is virtually abolished. How cells maintain their identity through cell division and faithfully re-initiate gene expression during mitotic exit is unclear. Here, we developed a novel reporter system enabling cell cycle synchronization-free separation of pluripotent stem cells in temporal bins of &lt; 30 minutes during mitotic exit. This allowed us to quantify genome-wide reactivation of transcription, sequential changes in chromatin accessibility, and re-binding of the pluripotency transcription factors OCT4, SOX2, and NANOG (OSN). We found that transcriptional activity progressively ramped up after mitosis and that OSN rapidly reoccupied the genome during the anaphase-telophase transition. We also demonstrate transcription factor-specific, dynamic relocation patterns and a hierarchical reorganization of the OSN binding landscape governed by OCT4 and SOX2. Our study sheds light on the dynamic orchestration of transcriptional reactivation after mitosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552750">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552750" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552750">
        <p class="paperTitle">The SARM1 TIR domain produces glycocyclic ADPR molecules as minor products</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552750" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552750" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garb, J.; Amitai, G.; Lu, A.; Ofir, G.; Brandis, A.; Mehlman, T.; Kranzusch, P.; Sorek, R.</p>
        <p class="info">Score: 17.6, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552750' target='https://doi.org/10.1101/2023.08.10.552750'> 10.1101/2023.08.10.552750</a></p>
        <p class="abstract">Sterile alpha and TIR motif-containing 1 (SARM1) is a protein involved in programmed death of injured axons. Following axon injury or a drug-induced insult, the TIR domain of SARM1 degrades the essential molecule nicotinamide adenine dinucleotide (NAD&#43;), leading to a form of axonal death called Wallerian degeneration. Degradation of NAD&#43; by SARM1 is essential for the Wallerian degeneration process, but accumulating evidence suggest that other activities of SARM1, beyond the mere degradation of NAD&#43;, may be necessary for programmed axonal death. In this study we show that the TIR domains of both human and fruit fly SARM1 produce 1&#39;&#39;-2&#39; and 1&#39;&#39;-3&#39; glycocyclic ADP-ribose (gcADPR) molecules as minor products. As previously reported, we observed that SARM1 TIR domains mostly convert NAD&#43; to ADPR (for human SARM1) or cADPR (in the case of SARM1 from Drosophila melanogaster). However, we now show that human and Drosophila SARM1 additionally convert [~]0.1-0.5% of NAD&#43; into gcADPR molecules. We find that SARM1 TIR domains produce gcADPR molecules both when purified in vitro and when expressed in bacterial cells. Given that gcADPR is a second messenger involved in programmed cell death in bacteria and likely in plants, we propose that gcADPR may play a role in SARM1-induced programmed axonal death in animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552904">
        <p class="paperTitle">N-glycosylation as a eukaryotic protective mechanism against protein aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duran-Romana, R.; Houben, B.; De Vleeschouwer, M.; Louros, N.; Wilson, M. P.; Matthijs, G.; Schymkowitz, J.; Rousseau, F.</p>
        <p class="info">Score: 12.9, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552904' target='https://doi.org/10.1101/2023.08.11.552904'> 10.1101/2023.08.11.552904</a></p>
        <p class="abstract">The tendency for proteins to form aggregates is an inherent part of every proteome and arises from the self-assembly of short protein segments called aggregation-prone regions (APRs). While post-translational modifications (PTMs) have been implicated in modulating protein aggregation, their direct role in APRs remains poorly understood. In this study, we used a combination of proteome-wide computational analyses and biochemical techniques to investigate the potential involvement of PTMs in aggregation regulation. Our findings reveal that while most PTM types are disfavored near APRs, N-glycosylation is enriched and evolutionarily selected, especially in proteins prone to misfolding. Experimentally, we show that N-glycosylation inhibits the aggregation of peptides in vitro through steric hindrance. Moreover, mining existing proteomics data, we find that the loss of N-glycans at the flanks of APRs leads to specific protein aggregation in Neuro2a cells. Our results point towards a novel intrinsic role for N-glycosylation, directly preventing protein aggregation in eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553001" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553001">
        <p class="paperTitle">AePUb promoter length modulates gene expression in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderson, M. A. E.; Leftwich, P. T.; Wilson, R.; Carabajal Paladino, L. Z.; Basu, S.; Rooney, S.; Adelman, Z.; Alphey, L.</p>
        <p class="info">Score: 12.9, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553001' target='https://doi.org/10.1101/2023.08.11.553001'> 10.1101/2023.08.11.553001</a></p>
        <p class="abstract">Molecular tools for modulating transgene expression in Aedes aegypti are few. Here we demonstrate that adjustments to the AePUb promoter length can alter expression levels of two reporter proteins in Ae. aegypti cell culture and in mosquitoes. This provides a simple means for increasing or decreasing expression of a gene of interest and easy translation from cells to whole insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553120">
        <p class="paperTitle">MYC and MAX drive the reactivation of the genome after mitosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez, I.; Chervova, A.; Escoll Guerrero, P.; Altamirano-Pacheco, L.; Mueller, F.; Dubois, A.; Navarro, P.</p>
        <p class="info">Score: 11.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553120' target='https://doi.org/10.1101/2023.08.13.553120'> 10.1101/2023.08.13.553120</a></p>
        <p class="abstract">Shortly after cell division, a robust wave of hyper-transcription reactivates the genome.1-3 This phenomenon is particularly pronounced in pluripotent cells,4 which necessitate rapid transcriptome reactivation to maintain their undifferentiated state and prevent premature differentiation. While recent work has illuminated how specific groups of genes are reactivated,4-8 the mechanisms enabling the global, efficient and accurate post-mitotic reactivation of the genome remain unknown. Here we elucidate the direct involvement of the MYC/MAX transcription factors in the post-mitotic reactivation of pluripotent mouse embryonic stem cells. While MYC undergoes extensive phosphorylation and largely dissociates from its DNA binding sites during mitosis, we report that MAX remains bound to its targets, preferentially at promoters, and facilitates early recruitment of MYC following mitosis. Through the application of MYC/MAX heterodimerization inhibitors, we demonstrate their indispensable role in sustaining hyper-transcription in ES cells, including during the critical transition from mitosis to G1 phase. Our findings uncover a novel role for MAX in mitotic bookmarking, highlighting its pivotal role in post-mitotic MYC recruitment and the re-establishment of high global transcription levels. These findings hold significant implications for medically relevant contexts, particularly when cell proliferation is of paramount importance.9 We anticipate that the study of mitotic bookmarking by MYC and MAX and of the effects of anti-cancer drugs targeting MYC/MAX interactions in such process10-12 will be relevant for our understanding of cancer and its potential treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552649">
        <p class="paperTitle">Structural basis of archaeal FttA-dependent transcription termination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, C.; Molodtsov, V.; Sanders, T. J.; Marshall, C. J.; Firlar, E.; Kaelber, J. T.; Santangelo, T. J.; Ebright, R. H.</p>
        <p class="info">Score: 8.4, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552649' target='https://doi.org/10.1101/2023.08.09.552649'> 10.1101/2023.08.09.552649</a></p>
        <p class="abstract">The ribonuclease FttA mediates factor-dependent transcription termination in archaea1-3. Here, we report the structure of a Thermococcus kodakarensis transcription pre-termination complex comprising FttA, Spt4, Spt5, and a transcription elongation complex (TEC). The structure shows that FttA interacts with the TEC in a manner that enables RNA to proceed directly from the TEC RNA-exit channel to the FttA catalytic center and that enables endonucleolytic cleavage of RNA by FttA, followed by 5[-&amp;gt;]3 exonucleolytic cleavage of RNA by FttA and concomitant 5[-&amp;gt;]3 translocation of FttA on RNA, to apply mechanical force to the TEC and trigger termination. The structure further reveals that Spt5 bridges FttA and the TEC, explaining how Spt5 stimulates FttA-dependent termination. The results reveal functional analogy between bacterial and archaeal factor-dependent termination, reveal functional homology between archaeal and eukaryotic factor-dependent termination, and reveal fundamental mechanistic similarities in factor-dependent termination in the three domains of life: bacterial, archaeal, and eukaryotic.

One sentence summaryCryo-EM reveals the structure of the archaeal FttA pre-termination complex</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553006">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553006" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553006">
        <p class="paperTitle">Short oligomers rather than rings of human RAD52 promote single-strand annealing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553006" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553006" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kharlamova, M. A.; Kushwah, M. S.; Jachowski, T. J.; Subramaniam, S.; Stewart, A. F.; Kukura, P.; Schäffer, E.</p>
        <p class="info">Score: 7.3, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553006' target='https://doi.org/10.1101/2023.08.11.553006'> 10.1101/2023.08.11.553006</a></p>
        <p class="abstract">Genome maintenance and stability rely on the repair of DNA double-strand breaks. The break repair can be mediated by the single-strand annealing protein RAD52. RAD52 forms rings that are thought to promote annealing. However, RAD52s annealing activity decreases with increasing concentrations that favor ring formation. Thus, which oligomeric form and how RAD52 anneals DNA strands and detects sequence homology is unclear. We combine mass photometry with biochemical assays to quantify oligomeric states of human RAD52 with and without DNA and put forward an alternative mechanism illustrating the critical role of short oligomers for single-stranded DNA annealing. We found that while truncated RAD52 formed undecameric rings at nanomolar concentrations, full-length RAD52 was mostly monomeric at lower nanomolar, physiological concentrations. At higher concentrations, it formed rings with a variable stoichiometry from heptamers to tridecamers. At low concentrations, with hardly any rings present, RAD52 already promoted single-strand annealing. Rings and short oligomers could bind at least two single DNA strands, but if complementary strands were both bound to rings annealing was inhibited. Our findings suggest that single-strand annealing and homology detection is mediated by short oligomers of RAD52 instead of rings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552751">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552751" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552751">
        <p class="paperTitle">RNA binding limits the ligand induced transcriptional potential of estrogen receptor-alpha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552751" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552751" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soota, D.; Saravanan, B.; Mann, R.; Kharbanda, T.; Notani, D.</p>
        <p class="info">Score: 6.0, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552751' target='https://doi.org/10.1101/2023.08.10.552751'> 10.1101/2023.08.10.552751</a></p>
        <p class="abstract">Transcription factors (TFs) primarily regulate gene expression by binding to DNA through their DNA binding domains (DBD). Additionally, approximately half of these TFs also interact with RNA. However, the role of RNA in enabling TF binding on chromatin and subsequent transcription is poorly understood. Estrogen receptor- (ER) is one such TF that activates genes in response to estrogen stimulation. Here, we report that ER interacts with various types of RNAs in ligand dependent manner via its RNA binding motif in hinge region. RNA binding defects lead to a global loss of ER binding in the genome, particularly at weaker ER motifs. In the absence of RNA binding, the ER exhibits dynamic behavior in the nucleus and unexpectedly, the dynamic binding coincides with robust polymerase loading on ER bound chromatin regions. The higher occupancy of PolII was recapitulated by robust ligand induced transcription of ER-regulated genes. Collectively, our results suggest that RNA interactions strengthen ER binding to chromatin limiting the ligand-dependent transcriptional upregulation of estrogen-induced genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552591">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552591" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552591">
        <p class="paperTitle">Molecular basis for proofreading by the unique exonuclease domain of Family-D DNA polymerases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552591" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552591" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Betancurt-Anzola, L.; Martinez-Carranza, M.; Delarue, M.; Zatopek, K.; Gardner, A.; Sauguet, L.</p>
        <p class="info">Score: 5.7, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552591' target='https://doi.org/10.1101/2023.08.09.552591'> 10.1101/2023.08.09.552591</a></p>
        <p class="abstract">Replicative DNA polymerases duplicate entire genomes at high fidelity. This feature is shared among the three domains of life and is facilitated by their dual polymerase and exonuclease activities. Family D replicative DNA polymerases (PolD), found exclusively in Archaea, contain an unusual RNA polymerase-like catalytic core, and a unique Mre11-like proofreading active site. Here, we present cryo-EM structures of PolD trapped in a proofreading mode, revealing an unanticipated correction mechanism that extends the repertoire of protein domains known to be involved in DNA proofreading. Based on our experimental structures, mutants of PolD were designed and their contribution to mismatch bypass and exonuclease kinetics was determined. This study sheds light on the convergent evolution of structurally distinct families of DNA polymerases, and the domain acquisition and exchange mechanism that occurred during the evolution of the replisome in the three domains of life.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
