<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A paradigm for regulation at the effector interface with RNA-binding proteins
Authors: Shah, K.; He, S.; Turner, D. J.; Corbo, J.; Rebbani, K.; Bateman, J.; Cheloufi, S.; Igreja, C.; Valkov, E.; Murn, J.
Score: 12.2, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558714
RNA-binding proteins (RBPs) are key regulators of gene expression, but how RBPs convey regulatory instructions to the core effectors of RNA processing is unclear. We address this problem by dissecting the effector interface responsible for regulation by a developmentally essential RBP.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="A paradigm for regulation at the effector interface with RNA-binding proteins
Authors: Shah, K.; He, S.; Turner, D. J.; Corbo, J.; Rebbani, K.; Bateman, J.; Cheloufi, S.; Igreja, C.; Valkov, E.; Murn, J.
Score: 12.2, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558714
RNA-binding proteins (RBPs) are key regulators of gene expression, but how RBPs convey regulatory instructions to the core effectors of RNA processing is unclear. We address this problem by dissecting the effector interface responsible for regulation by a developmentally essential RBP." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="A paradigm for regulation at the effector interface with RNA-binding proteins
Authors: Shah, K.; He, S.; Turner, D. J.; Corbo, J.; Rebbani, K.; Bateman, J.; Cheloufi, S.; Igreja, C.; Valkov, E.; Murn, J.
Score: 12.2, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558714
RNA-binding proteins (RBPs) are key regulators of gene expression, but how RBPs convey regulatory instructions to the core effectors of RNA processing is unclear. We address this problem by dissecting the effector interface responsible for regulation by a developmentally essential RBP."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "A paradigm for regulation at the effector interface with RNA-binding proteins\nAuthors: Shah, K.; He, S.; Turner, D. J.; Corbo, J.; Rebbani, K.; Bateman, J.; Cheloufi, S.; Igreja, C.; Valkov, E.; Murn, J.\nScore: 12.2, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558714\nRNA-binding proteins (RBPs) are key regulators of gene expression, but how RBPs convey regulatory instructions to the core effectors of RNA processing is unclear. We address this problem by dissecting the effector interface responsible for regulation by a developmentally essential RBP.",
  "keywords": [
    
  ],
  "articleBody": " A paradigm for regulation at the effector interface with RNA-binding proteins\nAuthors: Shah, K.; He, S.; Turner, D. J.; Corbo, J.; Rebbani, K.; Bateman, J.; Cheloufi, S.; Igreja, C.; Valkov, E.; Murn, J.\nScore: 12.2, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558714\nRNA-binding proteins (RBPs) are key regulators of gene expression, but how RBPs convey regulatory instructions to the core effectors of RNA processing is unclear. We address this problem by dissecting the effector interface responsible for regulation by a developmentally essential RBP. We show that the sequence-specific RBP Unkempt, in addition to its RNA-binding domain, contains an extensive intrinsically disordered region (IDR) and a dimerization domain, all of which distinctly contribute to Unkempts function. Within IDR, we identify an array of linear motifs that support direct contact with two central effectors, the CCR4-NOT complex and PABPC. The spatial arrangement, dual-purpose motifs, and obligate homodimerization allow for high-avidity effector interactions that are mandatory for the accuracy of RNA targeting, translational repression, and control of cell morphology by Unkempt. Our study establishes the molecular assembly and functional principles of an RBP- effector interface, with general implications for the evolution and function of RBP-operated regulatory networks.\nCCAR1 promotes DNA repair via an unanticipated role in alternative splicing\nAuthors: Karasu, M. E.; Joseph, B.; Mironov, A.; Schroeder, M. S.; Gvozdenovic, A.; Zavolan, M.; Corn, J. E.\nScore: 19.3, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557780\nDNA repair is directly performed by hundreds of core factors and indirectly regulated by thousands of others. We massively expanded a CRISPR inhibition and Cas9-editing screening system to discover factors indirectly modulating homology directed repair (HDR) in the context of [~]18000 individual gene knockdowns. We focused on CCAR1, a poorly understood gene that we found reduced both HDR and interstrand crosslink repair, phenocopying the loss of the Fanconi Anemia pathway. CCAR1 loss abrogated FANCA protein without substantial reduction in the level of its mRNA or that of other FA genes. We instead found that CCAR1 prevents inclusion of a poison exon in FANCA. Transcriptomic analysis revealed that the CCAR1 splicing modulatory activity is not limited to FANCA, and it instead regulates widespread changes in alternative splicing that would damage coding sequences in mouse and human cells. CCAR1 therefore has an unanticipated function as a splicing fidelity factor.\nSingle-Cell Analysis Reveals Cryptic Prophage Protease LfgB Protects Escherichia coli During Oxidative Stress by Cleaving Antitoxin MqsA\nAuthors: Fernandez-Garcia, L.; Gao, X.; Battisti, M.; Kirigo, J.; Garcia-Contreras, R.; Tomas, M.; Guo, Y.; Wang, X.; Wood, T. K.\nScore: 10.6, Published: 2023-09-23 DOI: 10.1101/2023.09.23.559088\nAlthough toxin/antitoxin (TA) systems are ubiquitous, beyond phage inhibition and mobile element stabilization, their role in host metabolism is obscure. One of the best-characterized TA systems is MqsR/MqsA of Escherichia coli, which has been linked previously to protecting this gastrointestinal species during the stress it encounters from the bile salt deoxycholate as it colonizes humans. However, some recent whole-population studies have challenged the role of toxins such as MqsR in bacterial physiology, since the mqsRA locus is induced over a hundred-fold during stress, but a phenotype was not found upon its deletion. Here, we investigate further the role MqsR/MqsA by utilizing single cells and demonstrate that upon oxidative stress, the TA system MqsR/MqsA has a heterogeneous effect on the transcriptome of single cells. Furthermore, we discovered that MqsR activation leads to induction of the poorly-characterized yfjXY ypjJ yfjZF operon of cryptic prophage CP4-57. Moreover, deletion of yfjY makes the cells sensitive to H2O2, acid, and heat stress, and this phenotype was complemented. Hence, we recommend yfjY be renamed to lfgB (less fatality gene B). Critically, MqsA represses lfgB by binding the operon promoter, and LfgB is a protease that degrades MqsA to derepress rpoS and facilitate the stress response. Therefore, the MqsR/MqsA TA system facilitates the stress response through cryptic phage protease LfgB.\nMitochondrial transplantation and its impact on infectious disease progression: a pilot study\nAuthors: Benson, T.; Patel, S. P.; Albensi, B. C.; Mahajan, V. B.; Adlimoghaddam, A.; Sikora, S.; Saito, H.\nScore: 10.8, Published: 2023-09-13 DOI: 10.1101/2023.09.08.556161\nBACKGROUNDMitochondrial transplantation has recently gained prominence as a novel technique primarily focused on addressing ischemiareperfusion injuries and rare mitochondrial mutation diseases. Platelets, abundant in the bloodstream, play a crucial role in immune function. Upon activation, platelets release mitochondria encapsulated within extracellular vesicles, here referred to as \"mitlets\". These mitlets exhibit a preference for being internalized by immune cells circulating in the bloodstream, enhancing their cellular energetics. Herein, we hypothesized that the transplantation of mitlets between young animals and aged animals may exert a significant influence on the progression of infectious diseases. STUDY DESIGN AND METHODSIn this study, murine models of Influenza H1N1 infection and sepsis were employed to investigate disease dynamics. Specifically, mitlets isolated from young and healthy mice were transplanted into cohorts of mice of the same age afflicted by H1N1 infection, or into aged mice subjected to polymicrobial infection and sepsis. Survival outcomes and the quantification of cytokine levels were assessed across experimental groups to elucidate the potential therapeutic effects of mitlet transplantation. RESULTSIn the matched-age H1N1 infection model, as predicted, mitlet transplantation did not yield a statistically significant improvement in survival, although it did show a trend towards a reduction in the circulating inflammatory cytokine burden. In the young-to-old sepsis model, the transplantation of mitlets was associated with a significant enhancement in survival rates and a substantial reduction in bacterial loads and circulating cytokine levels. DISCUSSIONOur findings suggest that mitochondrial transplantation may constitute a safe and promising avenue for enhancing the immune systems capacity to counter infectious threats. This pilot investigation sets the stage for further exploration. It is plausible that in the future, immune senescence resulting from diminished mitochondrial energy production could be ameliorated through such transplantation interventions. As a consequence, this approach holds substantial potential as a novel immunotherapeutic strategy for the management of infectious diseases.\nSpinal cord pathology in a Dravet Syndrome mouse model\nAuthors: Antinao Diaz, J. F.; Chilcott, E.; Keegan, A.; Schorge, S.; Waddington, S. N.; Karda, R.\nScore: 8.8, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558962\nObjectives: Dravet syndrome is a severe epileptic encephalopathy that begins in early childhood. More than 80% of patients with Dravet syndrome exhibit a haploinsufficiency in SCN1A, which encodes the voltage-gated sodium ion channel NaV1.1. The epilepsy is believed be caused by specific deficit of SCN1A in inhibitory interneurons of the hippocampus. However, the aetiology of other symptoms including gait disturbances, ataxia, cardiac issues and dysautonomia is less clear. Methods: In an Scn1a knock-out (Scn1a-/-) mouse model which recapitulates clinical phenotypes, we assessed NaV1.1 and neuroinflammation throughout the central nervous system. Results: Consistent with current understanding, wild-type expression of NaV1.1 transcript and protein were absent in knock-out mice in the prefrontal cortex, striatum, hippocampus, thalamus, and cerebellum. Increased GFAP was detected in the brain only in the hippocampus. Transcript and protein were detected in wild-type cervical, thoracic and lumbar spinal cord but not in knock-out mice. Unexpectedly, GFAP was increased in all three spinal regions. Therefore, we proceeded to perform transcriptomic analysis of cortex, hippocampus and spinal cord. Pathways associated with monooxygenase activity, fatty acid ligases and lactate transporters were highly dysregulated in the spinal cord. Conclusion: The existence and relevance of pathology of the spinal cord in Dravet syndrome has received scant attention. Our findings are consistent with some systemic symptoms of Dravet syndrome, with the benefits of treatments which may modulate the astrocyte-neuron lactate shuttle such as Stiripentol and ketogenic dietary regimes, and with the efficacy of intrathecal delivery of therapeutics.\nA cryo-ET study of ciliary rootlet organization\nAuthors: van Hoorn, C.; Carter, A. P.\nScore: 41.3, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556114\nCiliary rootlets are striated bundles of filaments that connect the base of cilia to internal cellular structures. Rootlets are critical for the sensory and motile functions of cilia. However, the mechanisms underlying these functions remain unknown, in part due to a lack of structural information of rootlet organization. In this study, we obtain 3D reconstructions of membrane-associated and purified rootlets using cryo-electron tomography (cryo-ET). We show that flexible protrusions on the rootlet surface, which emanate from the cross-striations, connect to intracellular membranes. In purified rootlets, the striations were classified into amorphous (A)-bands, associated with accumulations on the rootlet surface, and discrete (D)-bands corresponding to punctate lines of density that run through the rootlet. These striations connect a flexible network of longitudinal filaments. Subtomogram averaging suggests the filaments consist of two intertwined coiled coils. The rootlets filamentous architecture, with frequent membrane-connecting cross-striations, lends itself well for anchoring large membranes in the cell.\nSpatial Transcriptomics Reveals the Requirement of ADGRG6 in Maintaining Chondrocyte Homeostasis in Mouse Growth Plates\nAuthors: Bian, F.; Hansen, V.; Feng, H. C.; Chen, Y.; Gray, R.; Wu, C.-L.; Liu, Z.\nScore: 7.2, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558739\nThe growth plate is essential for maintaining skeletal growth; however, the mechanisms governing postnatal growth plate homeostasis are poorly understood. Here we show that ADGRG6/GPR126, a cartilage-enriched G protein-coupled receptor (GPCR), is dispensable for embryonic limb development but is required for postnatal growth plate homeostasis. Adgrg6 ablation in osteochondral progenitor cells or postnatal chondrocytes leads to reduced cellularity and impaired maintenance of the resting zone in the growth plate, coupled with increased cell death and reduced cell proliferation. Adgrg6 mutant growth plates also exhibit disorganized extracellular matrix structures and dysregulated hypertrophic differentiation. Furthermore, using a novel spatial transcriptomics workflow that applies to FFPE tissue sections of mineralized mouse knee joints, we demonstrate that Adgrg6 ablation leads to reduced SOX9 expression, induced Indian hedgehog (IHH) signaling, and a precocious chondrogenic-to-osteogenic conversion of the growth plate chondrocytes that may be driven by increased POSTN/integrin receptor signaling. We further demonstrated that ADGRG6 regulates the proper formation of the resting zone growth plate by maintaining the PTHrP and SOX9-positive cell populations. Altogether, our findings elucidate the essential role of ADGRG6 in maintaining chondrocyte fate, survival, and homeostasis of the postnatal growth plates.\nLeishmania Ribosomal Protein (RP) paralogous genes compensate each others expression maintaining protein native levels\nAuthors: Borges, F. d. S.; Quilles Junior, J. C.; Lorenzon, L. B.; Espada, C. R.; Freitas-Castro, F.; Defina, T. P. A.; Holetz, F. B.; Cruz, A. K.\nScore: 8.1, Published: 2023-09-15 DOI: 10.1101/2023.09.15.557908\nIn the protozoan parasite Leishmania, most of the genes encoding for ribosomal proteins (RPs) are present as two or more copies in the genome, their untranslated regions (UTRs) are predominantly divergent, and might be associated with a distinct regulation of the paralogous genes expression. Here, we investigated the expression profiles of two RPs (S16 and L13a) encoded by duplicated genes in Leishmania major. The genes encoding for S16 protein have identical CDSs and divergent UTRs while the L13a CDSs diverge in two amino acids with divergent UTRs. Using CRISPR/Cas9 genomic editing system, we generated knockout ({Delta}) and tagged transfectants for each paralog of L13a and S16. Combining tagged and {Delta} cell lines we show that the expression of both RPS16 and RPL13a isoforms differ throughout the parasite development with one of the isoforms being always more abundant than its respective copy. Additionally, compensatory expression was observed for each paralog when one of the isoforms was deleted, evidencing functional conservation of these proteins. Such phenomenon is related to post-translational processes, since the compensation happened at the protein levels, with no alterations observed at the transcript levels. Ribosomal profiles for RPL13a point out a standard behavior for these paralogues as already reported for other RPs in trypanosomatids, showing its interaction with heavy RNA-protein complexes. The identification of sets of proteins binding specifically to the 3UTRs of either the high or less abundant transcripts suggests a possible role of these proteins to differently control the levels of expression of these RP genes. In addition, conserved cis-elements were identified in the 3UTRs of RPS16 or RPL13a; among these, exclusive cis-elements for the more or for the less expressed transcripts were identified.\nMatrix selection for the visualization of small molecules and lipids in brain tumors using untargeted MALDI-TOF mass spectrometry imaging\nAuthors: Lu, T.; Freytag, L.; Narayana, V. K.; Moore, Z.; Oliver, S. J.; Valkovic, A.; Nijagal, B.; Peterson, A.; De Souza, D. P.; McConville, M. J.; Whittle, J. R.; Best, S. A.; Freytag, S.\nScore: 6.2, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559427\nMatrix assisted laser desorption/ionization mass spectrometry imaging allows the study of metabolic activity in the tumor microenvironment of brain cancers. The detectable metabolites within these tumors are contingent upon the choice of matrix, deposition technique, and polarity setting. In this study, we compared the performance of three different matrices, two deposition techniques, and use of positive and negative polarity in two different brain cancer types and across two species. Optimal combinations were confirmed by comparative analysis of lipid and small molecule abundance using liquid chromatography mass spectrometry and RNA sequencing assessing differential metabolites between normal and tumor regions. Our findings indicate that the recrystallized alpha cyano 4 hydroxycinnamic acid matrix in positive polarity offered superior performance for both detected metabolites and consistency with other techniques. Beyond these implications for brain cancer, our work establishes a workflow to identify optimal matrices for spatial metabolomics studies.\nDNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency\nAuthors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.\nScore: 61.3, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557729\nIn mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)-deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions. To formally assess how 5mC shapes the H3K27me3 landscape, we profiled the epigenome of naive and differentiated cells in the presence and absence of the DNA methylation machinery. Surprisingly, we found that 5mC accumulation is not required to restrict most H3K27me3 domains. We went on to show that this 5mC-independent H3K27me3 restriction is mediated by aberrant expression of the PRC2 antagonist Ezhip. At the regions where 5mC appears to genuinely supplant H3K27me3, we identified 68 candidate genes that appeared to require 5mC deposition and/or H3K27me3 depletion for their activation in differentiated cells. Employing site-directed epigenome editing to directly modulate 5mC levels, we demonstrated that 5mC deposition is sufficient to antagonize H3K27me3 deposition and confer gene activation at individual candidates. Altogether, we systematically measured the antagonistic interplay between 5mC and H3K27me3 in a system that recapitulates early embryonic dynamics. Our results suggest that H3K27me3 restraint depends on 5mC, both directly and indirectly. This study also reveals a non-canonical role of 5mC in gene activation, which may be important not only for normal development but also for cancer progression, as oncogenic cells frequently exhibit dynamic replacement of 5mC for H3K27me3 and vice versa.\n",
  "wordCount" : "2444",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558714">
        <p class="paperTitle">A paradigm for regulation at the effector interface with RNA-binding proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shah, K.; He, S.; Turner, D. J.; Corbo, J.; Rebbani, K.; Bateman, J.; Cheloufi, S.; Igreja, C.; Valkov, E.; Murn, J.</p>
        <p class="info">Score: 12.2, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558714' target='https://doi.org/10.1101/2023.09.20.558714'> 10.1101/2023.09.20.558714</a></p>
        <p class="abstract">RNA-binding proteins (RBPs) are key regulators of gene expression, but how RBPs convey regulatory instructions to the core effectors of RNA processing is unclear. We address this problem by dissecting the effector interface responsible for regulation by a developmentally essential RBP. We show that the sequence-specific RBP Unkempt, in addition to its RNA-binding domain, contains an extensive intrinsically disordered region (IDR) and a dimerization domain, all of which distinctly contribute to Unkempts function. Within IDR, we identify an array of linear motifs that support direct contact with two central effectors, the CCR4-NOT complex and PABPC. The spatial arrangement, dual-purpose motifs, and obligate homodimerization allow for high-avidity effector interactions that are mandatory for the accuracy of RNA targeting, translational repression, and control of cell morphology by Unkempt. Our study establishes the molecular assembly and functional principles of an RBP- effector interface, with general implications for the evolution and function of RBP-operated regulatory networks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557780">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557780" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557780">
        <p class="paperTitle">CCAR1 promotes DNA repair via an unanticipated role in alternative splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557780" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557780" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karasu, M. E.; Joseph, B.; Mironov, A.; Schroeder, M. S.; Gvozdenovic, A.; Zavolan, M.; Corn, J. E.</p>
        <p class="info">Score: 19.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557780' target='https://doi.org/10.1101/2023.09.18.557780'> 10.1101/2023.09.18.557780</a></p>
        <p class="abstract">DNA repair is directly performed by hundreds of core factors and indirectly regulated by thousands of others. We massively expanded a CRISPR inhibition and Cas9-editing screening system to discover factors indirectly modulating homology directed repair (HDR) in the context of [~]18000 individual gene knockdowns. We focused on CCAR1, a poorly understood gene that we found reduced both HDR and interstrand crosslink repair, phenocopying the loss of the Fanconi Anemia pathway. CCAR1 loss abrogated FANCA protein without substantial reduction in the level of its mRNA or that of other FA genes. We instead found that CCAR1 prevents inclusion of a poison exon in FANCA. Transcriptomic analysis revealed that the CCAR1 splicing modulatory activity is not limited to FANCA, and it instead regulates widespread changes in alternative splicing that would damage coding sequences in mouse and human cells. CCAR1 therefore has an unanticipated function as a splicing fidelity factor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559088">
        <p class="paperTitle">Single-Cell Analysis Reveals Cryptic Prophage Protease LfgB Protects Escherichia coli During Oxidative Stress by Cleaving Antitoxin MqsA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Garcia, L.; Gao, X.; Battisti, M.; Kirigo, J.; Garcia-Contreras, R.; Tomas, M.; Guo, Y.; Wang, X.; Wood, T. K.</p>
        <p class="info">Score: 10.6, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559088' target='https://doi.org/10.1101/2023.09.23.559088'> 10.1101/2023.09.23.559088</a></p>
        <p class="abstract">Although toxin/antitoxin (TA) systems are ubiquitous, beyond phage inhibition and mobile element stabilization, their role in host metabolism is obscure. One of the best-characterized TA systems is MqsR/MqsA of Escherichia coli, which has been linked previously to protecting this gastrointestinal species during the stress it encounters from the bile salt deoxycholate as it colonizes humans. However, some recent whole-population studies have challenged the role of toxins such as MqsR in bacterial physiology, since the mqsRA locus is induced over a hundred-fold during stress, but a phenotype was not found upon its deletion. Here, we investigate further the role MqsR/MqsA by utilizing single cells and demonstrate that upon oxidative stress, the TA system MqsR/MqsA has a heterogeneous effect on the transcriptome of single cells. Furthermore, we discovered that MqsR activation leads to induction of the poorly-characterized yfjXY ypjJ yfjZF operon of cryptic prophage CP4-57. Moreover, deletion of yfjY makes the cells sensitive to H2O2, acid, and heat stress, and this phenotype was complemented. Hence, we recommend yfjY be renamed to lfgB (less fatality gene B). Critically, MqsA represses lfgB by binding the operon promoter, and LfgB is a protease that degrades MqsA to derepress rpoS and facilitate the stress response. Therefore, the MqsR/MqsA TA system facilitates the stress response through cryptic phage protease LfgB.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556161">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556161" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556161">
        <p class="paperTitle">Mitochondrial transplantation and its impact on infectious disease progression: a pilot study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556161" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556161" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benson, T.; Patel, S. P.; Albensi, B. C.; Mahajan, V. B.; Adlimoghaddam, A.; Sikora, S.; Saito, H.</p>
        <p class="info">Score: 10.8, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556161' target='https://doi.org/10.1101/2023.09.08.556161'> 10.1101/2023.09.08.556161</a></p>
        <p class="abstract">BACKGROUNDMitochondrial transplantation has recently gained prominence as a novel technique primarily focused on addressing ischemiareperfusion injuries and rare mitochondrial mutation diseases. Platelets, abundant in the bloodstream, play a crucial role in immune function. Upon activation, platelets release mitochondria encapsulated within extracellular vesicles, here referred to as &#34;mitlets&#34;. These mitlets exhibit a preference for being internalized by immune cells circulating in the bloodstream, enhancing their cellular energetics. Herein, we hypothesized that the transplantation of mitlets between young animals and aged animals may exert a significant influence on the progression of infectious diseases.

STUDY DESIGN AND METHODSIn this study, murine models of Influenza H1N1 infection and sepsis were employed to investigate disease dynamics. Specifically, mitlets isolated from young and healthy mice were transplanted into cohorts of mice of the same age afflicted by H1N1 infection, or into aged mice subjected to polymicrobial infection and sepsis. Survival outcomes and the quantification of cytokine levels were assessed across experimental groups to elucidate the potential therapeutic effects of mitlet transplantation.

RESULTSIn the matched-age H1N1 infection model, as predicted, mitlet transplantation did not yield a statistically significant improvement in survival, although it did show a trend towards a reduction in the circulating inflammatory cytokine burden. In the young-to-old sepsis model, the transplantation of mitlets was associated with a significant enhancement in survival rates and a substantial reduction in bacterial loads and circulating cytokine levels.

DISCUSSIONOur findings suggest that mitochondrial transplantation may constitute a safe and promising avenue for enhancing the immune systems capacity to counter infectious threats. This pilot investigation sets the stage for further exploration. It is plausible that in the future, immune senescence resulting from diminished mitochondrial energy production could be ameliorated through such transplantation interventions. As a consequence, this approach holds substantial potential as a novel immunotherapeutic strategy for the management of infectious diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558962">
        <p class="paperTitle">Spinal cord pathology in a Dravet Syndrome mouse model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antinao Diaz, J. F.; Chilcott, E.; Keegan, A.; Schorge, S.; Waddington, S. N.; Karda, R.</p>
        <p class="info">Score: 8.8, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558962' target='https://doi.org/10.1101/2023.09.22.558962'> 10.1101/2023.09.22.558962</a></p>
        <p class="abstract">Objectives: Dravet syndrome is a severe epileptic encephalopathy that begins in early childhood. More than 80% of patients with Dravet syndrome exhibit a haploinsufficiency in SCN1A, which encodes the voltage-gated sodium ion channel NaV1.1. The epilepsy is believed be caused by specific deficit of SCN1A in inhibitory interneurons of the hippocampus. However, the aetiology of other symptoms including gait disturbances, ataxia, cardiac issues and dysautonomia is less clear. Methods: In an Scn1a knock-out (Scn1a-/-) mouse model which recapitulates clinical phenotypes, we assessed NaV1.1 and neuroinflammation throughout the central nervous system. Results: Consistent with current understanding, wild-type expression of NaV1.1 transcript and protein were absent in knock-out mice in the prefrontal cortex, striatum, hippocampus, thalamus, and cerebellum. Increased GFAP was detected in the brain only in the hippocampus. Transcript and protein were detected in wild-type cervical, thoracic and lumbar spinal cord but not in knock-out mice. Unexpectedly, GFAP was increased in all three spinal regions. Therefore, we proceeded to perform transcriptomic analysis of cortex, hippocampus and spinal cord. Pathways associated with monooxygenase activity, fatty acid ligases and lactate transporters were highly dysregulated in the spinal cord. Conclusion: The existence and relevance of pathology of the spinal cord in Dravet syndrome has received scant attention. Our findings are consistent with some systemic symptoms of Dravet syndrome, with the benefits of treatments which may modulate the astrocyte-neuron lactate shuttle such as Stiripentol and ketogenic dietary regimes, and with the efficacy of intrathecal delivery of therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556114">
        <p class="paperTitle">A cryo-ET study of ciliary rootlet organization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Hoorn, C.; Carter, A. P.</p>
        <p class="info">Score: 41.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556114' target='https://doi.org/10.1101/2023.09.03.556114'> 10.1101/2023.09.03.556114</a></p>
        <p class="abstract">Ciliary rootlets are striated bundles of filaments that connect the base of cilia to internal cellular structures. Rootlets are critical for the sensory and motile functions of cilia. However, the mechanisms underlying these functions remain unknown, in part due to a lack of structural information of rootlet organization. In this study, we obtain 3D reconstructions of membrane-associated and purified rootlets using cryo-electron tomography (cryo-ET). We show that flexible protrusions on the rootlet surface, which emanate from the cross-striations, connect to intracellular membranes. In purified rootlets, the striations were classified into amorphous (A)-bands, associated with accumulations on the rootlet surface, and discrete (D)-bands corresponding to punctate lines of density that run through the rootlet. These striations connect a flexible network of longitudinal filaments. Subtomogram averaging suggests the filaments consist of two intertwined coiled coils. The rootlets filamentous architecture, with frequent membrane-connecting cross-striations, lends itself well for anchoring large membranes in the cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558739">
        <p class="paperTitle">Spatial Transcriptomics Reveals the Requirement of ADGRG6 in Maintaining Chondrocyte Homeostasis in Mouse Growth Plates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bian, F.; Hansen, V.; Feng, H. C.; Chen, Y.; Gray, R.; Wu, C.-L.; Liu, Z.</p>
        <p class="info">Score: 7.2, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558739' target='https://doi.org/10.1101/2023.09.21.558739'> 10.1101/2023.09.21.558739</a></p>
        <p class="abstract">The growth plate is essential for maintaining skeletal growth; however, the mechanisms governing postnatal growth plate homeostasis are poorly understood. Here we show that ADGRG6/GPR126, a cartilage-enriched G protein-coupled receptor (GPCR), is dispensable for embryonic limb development but is required for postnatal growth plate homeostasis. Adgrg6 ablation in osteochondral progenitor cells or postnatal chondrocytes leads to reduced cellularity and impaired maintenance of the resting zone in the growth plate, coupled with increased cell death and reduced cell proliferation. Adgrg6 mutant growth plates also exhibit disorganized extracellular matrix structures and dysregulated hypertrophic differentiation. Furthermore, using a novel spatial transcriptomics workflow that applies to FFPE tissue sections of mineralized mouse knee joints, we demonstrate that Adgrg6 ablation leads to reduced SOX9 expression, induced Indian hedgehog (IHH) signaling, and a precocious chondrogenic-to-osteogenic conversion of the growth plate chondrocytes that may be driven by increased POSTN/integrin receptor signaling. We further demonstrated that ADGRG6 regulates the proper formation of the resting zone growth plate by maintaining the PTHrP and SOX9-positive cell populations. Altogether, our findings elucidate the essential role of ADGRG6 in maintaining chondrocyte fate, survival, and homeostasis of the postnatal growth plates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557908">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557908" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557908">
        <p class="paperTitle">Leishmania Ribosomal Protein (RP) paralogous genes compensate each others expression maintaining protein native levels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557908" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557908" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Borges, F. d. S.; Quilles Junior, J. C.; Lorenzon, L. B.; Espada, C. R.; Freitas-Castro, F.; Defina, T. P. A.; Holetz, F. B.; Cruz, A. K.</p>
        <p class="info">Score: 8.1, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557908' target='https://doi.org/10.1101/2023.09.15.557908'> 10.1101/2023.09.15.557908</a></p>
        <p class="abstract">In the protozoan parasite Leishmania, most of the genes encoding for ribosomal proteins (RPs) are present as two or more copies in the genome, their untranslated regions (UTRs) are predominantly divergent, and might be associated with a distinct regulation of the paralogous genes expression. Here, we investigated the expression profiles of two RPs (S16 and L13a) encoded by duplicated genes in Leishmania major. The genes encoding for S16 protein have identical CDSs and divergent UTRs while the L13a CDSs diverge in two amino acids with divergent UTRs. Using CRISPR/Cas9 genomic editing system, we generated knockout ({Delta}) and tagged transfectants for each paralog of L13a and S16. Combining tagged and {Delta} cell lines we show that the expression of both RPS16 and RPL13a isoforms differ throughout the parasite development with one of the isoforms being always more abundant than its respective copy. Additionally, compensatory expression was observed for each paralog when one of the isoforms was deleted, evidencing functional conservation of these proteins. Such phenomenon is related to post-translational processes, since the compensation happened at the protein levels, with no alterations observed at the transcript levels. Ribosomal profiles for RPL13a point out a standard behavior for these paralogues as already reported for other RPs in trypanosomatids, showing its interaction with heavy RNA-protein complexes. The identification of sets of proteins binding specifically to the 3UTRs of either the high or less abundant transcripts suggests a possible role of these proteins to differently control the levels of expression of these RP genes. In addition, conserved cis-elements were identified in the 3UTRs of RPS16 or RPL13a; among these, exclusive cis-elements for the more or for the less expressed transcripts were identified.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559427">
        <p class="paperTitle">Matrix selection for the visualization of small molecules and lipids in brain tumors using untargeted MALDI-TOF mass spectrometry imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, T.; Freytag, L.; Narayana, V. K.; Moore, Z.; Oliver, S. J.; Valkovic, A.; Nijagal, B.; Peterson, A.; De Souza, D. P.; McConville, M. J.; Whittle, J. R.; Best, S. A.; Freytag, S.</p>
        <p class="info">Score: 6.2, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559427' target='https://doi.org/10.1101/2023.09.25.559427'> 10.1101/2023.09.25.559427</a></p>
        <p class="abstract">Matrix assisted laser desorption/ionization mass spectrometry imaging allows the study of metabolic activity in the tumor microenvironment of brain cancers. The detectable metabolites within these tumors are contingent upon the choice of matrix, deposition technique, and polarity setting. In this study, we compared the performance of three different matrices, two deposition techniques, and use of positive and negative polarity in two different brain cancer types and across two species. Optimal combinations were confirmed by comparative analysis of lipid and small molecule abundance using liquid chromatography mass spectrometry and RNA sequencing assessing differential metabolites between normal and tumor regions. Our findings indicate that the recrystallized alpha cyano 4 hydroxycinnamic acid matrix in positive polarity offered superior performance for both detected metabolites and consistency with other techniques. Beyond these implications for brain cancer, our work establishes a workflow to identify optimal matrices for spatial metabolomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557729" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557729">
        <p class="paperTitle">DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.</p>
        <p class="info">Score: 61.3, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557729' target='https://doi.org/10.1101/2023.09.14.557729'> 10.1101/2023.09.14.557729</a></p>
        <p class="abstract">In mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)-deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions. To formally assess how 5mC shapes the H3K27me3 landscape, we profiled the epigenome of naive and differentiated cells in the presence and absence of the DNA methylation machinery. Surprisingly, we found that 5mC accumulation is not required to restrict most H3K27me3 domains. We went on to show that this 5mC-independent H3K27me3 restriction is mediated by aberrant expression of the PRC2 antagonist Ezhip. At the regions where 5mC appears to genuinely supplant H3K27me3, we identified 68 candidate genes that appeared to require 5mC deposition and/or H3K27me3 depletion for their activation in differentiated cells. Employing site-directed epigenome editing to directly modulate 5mC levels, we demonstrated that 5mC deposition is sufficient to antagonize H3K27me3 deposition and confer gene activation at individual candidates. Altogether, we systematically measured the antagonistic interplay between 5mC and H3K27me3 in a system that recapitulates early embryonic dynamics. Our results suggest that H3K27me3 restraint depends on 5mC, both directly and indirectly. This study also reveals a non-canonical role of 5mC in gene activation, which may be important not only for normal development but also for cancer progression, as oncogenic cells frequently exhibit dynamic replacement of 5mC for H3K27me3 and vice versa.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
