<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization
Authors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.
Score: 43.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576763
CryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization
Authors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.
Score: 43.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576763
CryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization
Authors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.
Score: 43.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576763
CryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization\nAuthors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.\nScore: 43.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576763\nCryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids.",
  "keywords": [
    
  ],
  "articleBody": " CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization\nAuthors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.\nScore: 43.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576763\nCryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids. We demonstrate that vitreous ice may be produced by plunging clipped grids with purified proteins into liquid ethane and that clipped grids may be reused several times for different protein samples. Furthermore, we demonstrate the vitrification of thin areas of cells prepared on gold-coated, pre-clipped grids.\nThe conformational landscape of human transthyretin revealed by cryo-EM\nAuthors: Basanta, B.; Nugroho, K.; Yan, N. Y.; Kline, G. M.; Powers, E. T.; Tsai, F. J.; Wu, M.; Hansel-Harris, A.; Chen, J. S.; Forli, S.; Kelly, J. W.; Lander, G. C.\nScore: 25.0, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576879\nTransthyretin (TTR) is a natively tetrameric thyroxine transporter found in blood and cerebrospinal fluid whose misfolding and aggregation causes transthyretin amyloidosis. A rational drug design campaign identified the small molecule tafamidis (Vyndaqel/Vyndamax) as an effective stabilizer of the native TTR fold, and this aggregation inhibitor is regulatory agency-approved for the treatment of TTR amyloidosis. Despite 50 years of structural studies on TTR and this triumph of structure-based drug design, there remains a notable dearth of structural information available to understand ligand binding allostery and amyloidogenic TTR unfolding intermediates. We used single-particle cryo-electron microscopy (cryo-EM) to investigate the conformational landscape of this 55 kiloDalton tetramer in the absence and presence of one or two ligands, revealing inherent asymmetries in the tetrameric architecture and previously unobserved conformational states. These findings provide critical mechanistic insights into negatively cooperative ligand binding and the structural pathways responsible for TTR amyloidogenesis. This study underscores the capacity of cryo-EM to provide new insights into protein structures that have been historically considered too small to visualize and to identify pharmacological targets suppressed by the confines of the crystal lattice, opening uncharted territory in structure-based drug design.\nPeptidyl-tRNA hydrolysis rate influences the efficiency of nonsense-mediated mRNA decay\nAuthors: Kolakada, D.; Fu, R.; Biziaev, N.; Shuvalov, A.; Lore, M.; Campbell, A. E.; Cortazar, M. A.; Sajek, M. P.; Hesselberth, J. R.; Mukherjee, N.; Alkalaeva, E.; Jagannathan, S.\nScore: 11.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575080\nNonsense-mediated mRNA decay (NMD) is a quality control mechanism that prevents the accumulation of harmful truncated proteins by degrading transcripts with premature termination codons (PTCs). NMD efficiency varies across many contexts, but the factors that influence this variability remain poorly understood. Here, we find an enrichment of glycine (Gly) codons preceding a PTC in common nonsense variants in contrast with a depletion of Gly codons preceding a normal termination codon (NTC). Gly-PTC contexts have higher NMD activity compared to an alanine-PTC context, and this effect is stronger on NMD substrates with long 3UTRs. We used a massively parallel reporter assay to test all possible combinations of -2 and -1 codons, the PTC, and the +4 nucleotide to assess comprehensively how PTC sequence context affects NMD efficiency. A random forest classifier revealed that peptidyl-tRNA hydrolysis rate during translation termination was the most important feature in discriminating high and low NMD activity. We show with in vitro biochemical assays that Gly-TC contexts have the slowest termination rate compared to other codons. Furthermore, Gly-PTC enrichment is most pronounced in genes that tolerate loss-of-function variants, suggesting that enhanced NMD of Gly-PTC context has shaped the evolution of PTCs. Based on these findings, we propose that NMD efficiency is modulated by the \"window of opportunity\" offered by peptidyl tRNA hydrolysis rate and thus, translation termination kinetics.\nCryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples\nAuthors: Arimura, Y.; Konishi, H. A.; Funabiki, H.\nScore: 53.3, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576499\nHere, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to \u003c 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1.8 associated nucleosomes isolated from interphase and metaphase chromosomes in Xenopus egg extract. In combination with Duplicated Selection To Exclude Rubbish particles (DuSTER), a particle curation method that removes low signal-to-noise ratio particles, we also resolved the 3D cryo-EM structures of H1.8-bound nucleoplasmin NPM2, revealing distinct open and closed structural variants. Our approaches offer structural insights into how the chaperone NPM2 interacts with its targets and the cell cycle-dependent regulation of H1.8 association to nucleosomes.\nAnalyzing the implications of protein folding delay caused by translation\nAuthors: Houben, B.; Duran-Romana, R.; Fernandez Migens, P.; Rousseau, F.; Schymkowitz, J.\nScore: 7.4, Published: 2024-01-27 DOI: 10.1101/2024.01.27.577370\nBecause of vectorial protein production, residues that interact in the native protein structure but are distantly separated in the primary sequence are unavailable simultaneously. Instead, there is a temporal delay during which the N-terminal interaction partner is vulnerable to off-pathway, non-native interactions. In this analysis, we introduce \"FoldDelay\" (FD), a metric that integrates the topological pattern of atomic interactions of the native structure with translation kinetics to quantify such time delays. The FD metric reveals that many proteins, particularly at eukaryotic translation rates, exhibit residues with FDs in the range of tens of seconds. These residues, predominantly in well-structured, buried regions, often coincide with predicted aggregation-prone regions. We show a correlation between FD and co-translational engagement by the yeast Hsp70 chaperone Ssb, suggesting that fold-delayed regions have a propensity to misfold. In support of this, we show that proteins with high FDs are more frequently co-translationally ubiquitinated and prone to aggregate upon Ssb deletion. Finally, we find that FD cannot be adequately reduced through codon optimization, highlighting the importance of co-translational chaperones to shield these vulnerable regions. This work offers insights into co-translational proteostasis and the delicate balance between efficient folding and potential misfolding and aggregation during translation.\nEngineered minimal type I CRISPR-Cas system for transcriptional activation and base editing in human cells\nAuthors: Guo, J.; Gong, L.; Yu, H.; Li, M.; Liu, Z.; Fan, S.; Yang, C.; Zhao, D.; Han, J.; Xiang, H.\nScore: 4.0, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577312\nType I CRISPR-Cas systems are widespread and have exhibited remarkable versatility and efficiency in genome editing and gene regulation in prokaryotes. However, due to the multi-subunit composition and large size, their application in eukaryotes has not been thoroughly investigated. Here, we demonstrate that the type I-F2 Cascade, the most compact among type I systems and significantly smaller than SpCas9, can be developed into programmable tools for use in human cells. For transcriptional activation, the efficiency of the tool based on the engineered I-F2 system can match or surpass that of dCas9. Besides, narrow editing windows limit the application of base editors. Although the R-loop formed by Cascade is much wider than that by Cas9 or Cas12, the potential of base editing with Cascade has not yet been explored. We successfully created a base editor with the I-F2 Cascade, which induces a considerably wide editing window ([~]30 nt) with a bimodal distribution. The wide editing window can expand the range of targetable sites and can be useful for disrupting functional sequences and genetic screening. The editing efficiency can achieve 50% in human cells. This research underscores the application potential of compact type I systems in eukaryotes and developed a new base editor with an extraordinary wide editing window.\nHelicases DDX5 and DDX17 promote Hepatitis B Virus transcription termination heterogeneity in infected human hepatocytes\nAuthors: Chapus, F.; Giraud, G.; Huchon, P.; Charre, C.; Goldsmith, C.; Roda, M.; Martinez, M. G.; Fresquet, J.; Diederichs, A.; Locatelli, M.; Polveche, H.; Grand, X.; Scholtes, C.; Chemin, I.; Hernandez Vargas, H.; Rivoire, M.; Bourgeois, C.; Zoulim, F.; Testoni, B.\nScore: 4.9, Published: 2024-01-18 DOI: 10.1101/2024.01.17.575990\nBackground \u0026 AimsTranscription termination fine tunes gene expression and contributes to specify the function of RNAs in eukaryotic cells. Transcription termination of hepatitis B virus (HBV) is subjected to the recognition of the canonical polyadenylation signal (cPAS) common to all viral transcripts. The regulation of the usage of this cPAS and its impact on viral gene expression and replication is currently unknown. Approach \u0026 ResultsTo unravel the regulation of HBV transcript termination, we implemented a 3 RACE-PCR assay coupled to single molecule sequencing both in in vitro infected hepatocytes and in chronically infected patients. The detection of a previously unidentified transcriptional readthrough indicated that the cPAS was not systematically recognized during HBV replication in vitro and in vivo. Gene expression downregulation experiments demonstrated a role for the RNA helicases DDX5 and DDX17 in promoting viral transcriptional readthrough, which was, in turn, associated to HBV RNA destabilization and decreased HBx protein expression. RNA and chromatin immunoprecipitation, together with mutation of cPAS sequence suggested a direct role of DDX5 and DDX17 in functionally linking cPAS recognition to transcriptional readthrough, HBV RNA stability and replication. ConclusionsOur findings identify DDX5 and DDX17 as crucial determinants for HBV transcriptional fidelity and as host restriction factors for HBV replication.\nDiscovery and Characterization of a Pan-betacoronavirus S2-binding antibody\nAuthors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.\nScore: 323.3, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575741\nSUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.\nTwo paralogous PHD finger proteins participate inParamecium tetraurelia's natural genome editing\nAuthors: Häussermann, L.; Singh, A.; Swart, E. C.\nScore: 2.8, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576875\nThe unicellular eukaryote Paramecium tetraurelia contains functionally distinct nuclei: germline micronuclei (MICs) and a somatic macronucleus (MAC). During sexual reproduction, the MIC genome is reorganized into a new MAC genome and the old MAC is lost. Almost 45,000 unique Internal Eliminated Sequences (IESs) distributed throughout the genome require precise excision to guarantee a functional new MAC genome. Here, we characterize a pair of paralogous PHD finger proteins involved in DNA elimination. DevPF1, the early-expressed paralog, is present in only some of the gametic and post-zygotic nuclei during meiosis. Both DevPF1 and DevPF2 localize in the new developing MACs, where IESs excision occurs. In DevPF2 knockdown (KD) long IESs are preferentially retained and late-expressed small RNAs decrease; no length preference for retained IESs was observed in DevPF1-KD and development-specific small RNAs were abolished. The expression of at least two genes from the new MAC with roles in genome reorganization seems to be influenced by DevPF1- and DevPF2-KD. Thus, both PHD fingers are crucial for new MAC genome development, with distinct functions, potentially via regulation of non-coding and coding transcription in the MICs and new MACs.\nBeyond static structures: quantitative dynamics in the eukaryotic RNA exosome complex\nAuthors: Liebau, J.; Lazzaretti, D.; Bichler, A.; Pilsl, M.; Sprangers, R.\nScore: 2.6, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577622\nMolecular machines play pivotal roles in all biological processes. Most structural methods, however, are unable to directly probe molecular motions. We demonstrate that dedicated NMR experiments can provide quantitative insights into functionally important dynamic regions in very large asymmetric protein complexes. We establish this for the 410 kDa eukaryotic RNA exosome complex that contains ten distinct protein chains. NMR data reveal site-specific interactions and conformational changes in regions that are invisible in static cryo-EM and crystal structures. In particular, we identify a flexible plug region that blocks an aberrant route of the RNA to the active site. Our work thus demonstrates that a combination of state-of-the-art structural biology methods can provide quantitative insights into molecular machines that go well beyond static images.\n",
  "wordCount" : "2109",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576763" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576763">
        <p class="paperTitle">CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.</p>
        <p class="info">Score: 43.1, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576763' target='https://doi.org/10.1101/2024.01.23.576763'> 10.1101/2024.01.23.576763</a></p>
        <p class="abstract">CryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids. We demonstrate that vitreous ice may be produced by plunging clipped grids with purified proteins into liquid ethane and that clipped grids may be reused several times for different protein samples. Furthermore, we demonstrate the vitrification of thin areas of cells prepared on gold-coated, pre-clipped grids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576879" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576879">
        <p class="paperTitle">The conformational landscape of human transthyretin revealed by cryo-EM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Basanta, B.; Nugroho, K.; Yan, N. Y.; Kline, G. M.; Powers, E. T.; Tsai, F. J.; Wu, M.; Hansel-Harris, A.; Chen, J. S.; Forli, S.; Kelly, J. W.; Lander, G. C.</p>
        <p class="info">Score: 25.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576879' target='https://doi.org/10.1101/2024.01.23.576879'> 10.1101/2024.01.23.576879</a></p>
        <p class="abstract">Transthyretin (TTR) is a natively tetrameric thyroxine transporter found in blood and cerebrospinal fluid whose misfolding and aggregation causes transthyretin amyloidosis. A rational drug design campaign identified the small molecule tafamidis (Vyndaqel/Vyndamax) as an effective stabilizer of the native TTR fold, and this aggregation inhibitor is regulatory agency-approved for the treatment of TTR amyloidosis. Despite 50 years of structural studies on TTR and this triumph of structure-based drug design, there remains a notable dearth of structural information available to understand ligand binding allostery and amyloidogenic TTR unfolding intermediates. We used single-particle cryo-electron microscopy (cryo-EM) to investigate the conformational landscape of this 55 kiloDalton tetramer in the absence and presence of one or two ligands, revealing inherent asymmetries in the tetrameric architecture and previously unobserved conformational states. These findings provide critical mechanistic insights into negatively cooperative ligand binding and the structural pathways responsible for TTR amyloidogenesis. This study underscores the capacity of cryo-EM to provide new insights into protein structures that have been historically considered too small to visualize and to identify pharmacological targets suppressed by the confines of the crystal lattice, opening uncharted territory in structure-based drug design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575080" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575080">
        <p class="paperTitle">Peptidyl-tRNA hydrolysis rate influences the efficiency of nonsense-mediated mRNA decay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolakada, D.; Fu, R.; Biziaev, N.; Shuvalov, A.; Lore, M.; Campbell, A. E.; Cortazar, M. A.; Sajek, M. P.; Hesselberth, J. R.; Mukherjee, N.; Alkalaeva, E.; Jagannathan, S.</p>
        <p class="info">Score: 11.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575080' target='https://doi.org/10.1101/2024.01.10.575080'> 10.1101/2024.01.10.575080</a></p>
        <p class="abstract">Nonsense-mediated mRNA decay (NMD) is a quality control mechanism that prevents the accumulation of harmful truncated proteins by degrading transcripts with premature termination codons (PTCs). NMD efficiency varies across many contexts, but the factors that influence this variability remain poorly understood. Here, we find an enrichment of glycine (Gly) codons preceding a PTC in common nonsense variants in contrast with a depletion of Gly codons preceding a normal termination codon (NTC). Gly-PTC contexts have higher NMD activity compared to an alanine-PTC context, and this effect is stronger on NMD substrates with long 3UTRs. We used a massively parallel reporter assay to test all possible combinations of -2 and -1 codons, the PTC, and the &#43;4 nucleotide to assess comprehensively how PTC sequence context affects NMD efficiency. A random forest classifier revealed that peptidyl-tRNA hydrolysis rate during translation termination was the most important feature in discriminating high and low NMD activity. We show with in vitro biochemical assays that Gly-TC contexts have the slowest termination rate compared to other codons. Furthermore, Gly-PTC enrichment is most pronounced in genes that tolerate loss-of-function variants, suggesting that enhanced NMD of Gly-PTC context has shaped the evolution of PTCs. Based on these findings, we propose that NMD efficiency is modulated by the &#34;window of opportunity&#34; offered by peptidyl tRNA hydrolysis rate and thus, translation termination kinetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576499" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576499">
        <p class="paperTitle">Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arimura, Y.; Konishi, H. A.; Funabiki, H.</p>
        <p class="info">Score: 53.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576499' target='https://doi.org/10.1101/2024.01.21.576499'> 10.1101/2024.01.21.576499</a></p>
        <p class="abstract">Here, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to &lt; 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1.8 associated nucleosomes isolated from interphase and metaphase chromosomes in Xenopus egg extract. In combination with Duplicated Selection To Exclude Rubbish particles (DuSTER), a particle curation method that removes low signal-to-noise ratio particles, we also resolved the 3D cryo-EM structures of H1.8-bound nucleoplasmin NPM2, revealing distinct open and closed structural variants. Our approaches offer structural insights into how the chaperone NPM2 interacts with its targets and the cell cycle-dependent regulation of H1.8 association to nucleosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577370">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577370" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577370">
        <p class="paperTitle">Analyzing the implications of protein folding delay caused by translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577370" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577370" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Houben, B.; Duran-Romana, R.; Fernandez Migens, P.; Rousseau, F.; Schymkowitz, J.</p>
        <p class="info">Score: 7.4, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577370' target='https://doi.org/10.1101/2024.01.27.577370'> 10.1101/2024.01.27.577370</a></p>
        <p class="abstract">Because of vectorial protein production, residues that interact in the native protein structure but are distantly separated in the primary sequence are unavailable simultaneously. Instead, there is a temporal delay during which the N-terminal interaction partner is vulnerable to off-pathway, non-native interactions. In this analysis, we introduce &#34;FoldDelay&#34; (FD), a metric that integrates the topological pattern of atomic interactions of the native structure with translation kinetics to quantify such time delays. The FD metric reveals that many proteins, particularly at eukaryotic translation rates, exhibit residues with FDs in the range of tens of seconds. These residues, predominantly in well-structured, buried regions, often coincide with predicted aggregation-prone regions. We show a correlation between FD and co-translational engagement by the yeast Hsp70 chaperone Ssb, suggesting that fold-delayed regions have a propensity to misfold. In support of this, we show that proteins with high FDs are more frequently co-translationally ubiquitinated and prone to aggregate upon Ssb deletion. Finally, we find that FD cannot be adequately reduced through codon optimization, highlighting the importance of co-translational chaperones to shield these vulnerable regions. This work offers insights into co-translational proteostasis and the delicate balance between efficient folding and potential misfolding and aggregation during translation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577312">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577312" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577312">
        <p class="paperTitle">Engineered minimal type I CRISPR-Cas system for transcriptional activation and base editing in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577312" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577312" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, J.; Gong, L.; Yu, H.; Li, M.; Liu, Z.; Fan, S.; Yang, C.; Zhao, D.; Han, J.; Xiang, H.</p>
        <p class="info">Score: 4.0, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577312' target='https://doi.org/10.1101/2024.01.26.577312'> 10.1101/2024.01.26.577312</a></p>
        <p class="abstract">Type I CRISPR-Cas systems are widespread and have exhibited remarkable versatility and efficiency in genome editing and gene regulation in prokaryotes. However, due to the multi-subunit composition and large size, their application in eukaryotes has not been thoroughly investigated. Here, we demonstrate that the type I-F2 Cascade, the most compact among type I systems and significantly smaller than SpCas9, can be developed into programmable tools for use in human cells. For transcriptional activation, the efficiency of the tool based on the engineered I-F2 system can match or surpass that of dCas9. Besides, narrow editing windows limit the application of base editors. Although the R-loop formed by Cascade is much wider than that by Cas9 or Cas12, the potential of base editing with Cascade has not yet been explored. We successfully created a base editor with the I-F2 Cascade, which induces a considerably wide editing window ([~]30 nt) with a bimodal distribution. The wide editing window can expand the range of targetable sites and can be useful for disrupting functional sequences and genetic screening. The editing efficiency can achieve 50% in human cells. This research underscores the application potential of compact type I systems in eukaryotes and developed a new base editor with an extraordinary wide editing window.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575990">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575990" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575990">
        <p class="paperTitle">Helicases DDX5 and DDX17 promote Hepatitis B Virus transcription termination heterogeneity in infected human hepatocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575990" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575990" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chapus, F.; Giraud, G.; Huchon, P.; Charre, C.; Goldsmith, C.; Roda, M.; Martinez, M. G.; Fresquet, J.; Diederichs, A.; Locatelli, M.; Polveche, H.; Grand, X.; Scholtes, C.; Chemin, I.; Hernandez Vargas, H.; Rivoire, M.; Bourgeois, C.; Zoulim, F.; Testoni, B.</p>
        <p class="info">Score: 4.9, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575990' target='https://doi.org/10.1101/2024.01.17.575990'> 10.1101/2024.01.17.575990</a></p>
        <p class="abstract">Background &amp; AimsTranscription termination fine tunes gene expression and contributes to specify the function of RNAs in eukaryotic cells. Transcription termination of hepatitis B virus (HBV) is subjected to the recognition of the canonical polyadenylation signal (cPAS) common to all viral transcripts. The regulation of the usage of this cPAS and its impact on viral gene expression and replication is currently unknown.

Approach &amp; ResultsTo unravel the regulation of HBV transcript termination, we implemented a 3 RACE-PCR assay coupled to single molecule sequencing both in in vitro infected hepatocytes and in chronically infected patients. The detection of a previously unidentified transcriptional readthrough indicated that the cPAS was not systematically recognized during HBV replication in vitro and in vivo. Gene expression downregulation experiments demonstrated a role for the RNA helicases DDX5 and DDX17 in promoting viral transcriptional readthrough, which was, in turn, associated to HBV RNA destabilization and decreased HBx protein expression. RNA and chromatin immunoprecipitation, together with mutation of cPAS sequence suggested a direct role of DDX5 and DDX17 in functionally linking cPAS recognition to transcriptional readthrough, HBV RNA stability and replication.

ConclusionsOur findings identify DDX5 and DDX17 as crucial determinants for HBV transcriptional fidelity and as host restriction factors for HBV replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575741" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575741">
        <p class="paperTitle">Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.</p>
        <p class="info">Score: 323.3, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575741' target='https://doi.org/10.1101/2024.01.15.575741'> 10.1101/2024.01.15.575741</a></p>
        <p class="abstract">SUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576875" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576875">
        <p class="paperTitle">Two paralogous PHD finger proteins participate inParamecium tetraurelia&#39;s natural genome editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Häussermann, L.; Singh, A.; Swart, E. C.</p>
        <p class="info">Score: 2.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576875' target='https://doi.org/10.1101/2024.01.23.576875'> 10.1101/2024.01.23.576875</a></p>
        <p class="abstract">The unicellular eukaryote Paramecium tetraurelia contains functionally distinct nuclei: germline micronuclei (MICs) and a somatic macronucleus (MAC). During sexual reproduction, the MIC genome is reorganized into a new MAC genome and the old MAC is lost. Almost 45,000 unique Internal Eliminated Sequences (IESs) distributed throughout the genome require precise excision to guarantee a functional new MAC genome. Here, we characterize a pair of paralogous PHD finger proteins involved in DNA elimination. DevPF1, the early-expressed paralog, is present in only some of the gametic and post-zygotic nuclei during meiosis. Both DevPF1 and DevPF2 localize in the new developing MACs, where IESs excision occurs. In DevPF2 knockdown (KD) long IESs are preferentially retained and late-expressed small RNAs decrease; no length preference for retained IESs was observed in DevPF1-KD and development-specific small RNAs were abolished. The expression of at least two genes from the new MAC with roles in genome reorganization seems to be influenced by DevPF1- and DevPF2-KD. Thus, both PHD fingers are crucial for new MAC genome development, with distinct functions, potentially via regulation of non-coding and coding transcription in the MICs and new MACs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577622" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577622">
        <p class="paperTitle">Beyond static structures: quantitative dynamics in the eukaryotic RNA exosome complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liebau, J.; Lazzaretti, D.; Bichler, A.; Pilsl, M.; Sprangers, R.</p>
        <p class="info">Score: 2.6, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577622' target='https://doi.org/10.1101/2024.01.28.577622'> 10.1101/2024.01.28.577622</a></p>
        <p class="abstract">Molecular machines play pivotal roles in all biological processes. Most structural methods, however, are unable to directly probe molecular motions. We demonstrate that dedicated NMR experiments can provide quantitative insights into functionally important dynamic regions in very large asymmetric protein complexes. We establish this for the 410 kDa eukaryotic RNA exosome complex that contains ten distinct protein chains. NMR data reveal site-specific interactions and conformational changes in regions that are invisible in static cryo-EM and crystal structures. In particular, we identify a flexible plug region that blocks an aberrant route of the RNA to the active site. Our work thus demonstrates that a combination of state-of-the-art structural biology methods can provide quantitative insights into molecular machines that go well beyond static images.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
