<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Brain and cancer associated binding domain mutations provide insight into CTCF&#39;s relationship with chromatin and its ability to act as a chromatin organizer
Authors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.
Score: 17.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070
Although only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Brain and cancer associated binding domain mutations provide insight into CTCF&#39;s relationship with chromatin and its ability to act as a chromatin organizer
Authors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.
Score: 17.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070
Although only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Brain and cancer associated binding domain mutations provide insight into CTCF&#39;s relationship with chromatin and its ability to act as a chromatin organizer
Authors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.
Score: 17.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070
Although only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Brain and cancer associated binding domain mutations provide insight into CTCF\u0026#39;s relationship with chromatin and its ability to act as a chromatin organizer\nAuthors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.\nScore: 17.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070\nAlthough only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown.",
  "keywords": [
    
  ],
  "articleBody": " Brain and cancer associated binding domain mutations provide insight into CTCF's relationship with chromatin and its ability to act as a chromatin organizer\nAuthors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.\nScore: 17.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070\nAlthough only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown. To investigate the relationship between CTCF and chromatin we applied a combination of imaging, structural and molecular approaches, using a series of brain and cancer associated CTCF mutations that act as CTCF perturbations. We demonstrate that binding and the functional impact of WT and mutant CTCF depend not only on the unique binding properties of each protein, but also on the genomic context of bound sites and enrichment of motifs for expressed TFs abutting these sites. Our studies also highlight the reciprocal relationship between CTCF and chromatin, demonstrating that the unique binding properties of WT and mutant proteins have a distinct impact on accessibility, TF binding, cohesin overlap, chromatin interactivity and gene expression programs, providing insight into their cancer and brain related effects.\nCAP peptide artificially induces insect gall\nAuthors: Sakamoto, T.; Kimura, S.; Nakayama, T.; Sato, M. P.; Shirasawa, K.; Sato, M. H.; Hirano, T.\nScore: 15.8, Published: 2024-01-07 DOI: 10.1101/2024.01.06.574462\nGalls caused by gall-inducing insects in their host plants clearly illustrate the concept of extended phenotype, which refers to traits expressed in a host organism when manipulated by a parasite. Candidate effector molecules involved in gall formation, such as phytohormones, amino acids, and proteins, have been reported in numerous studies. However, to date, no attempts to artificially regenerate gall structures using effector candidates have been reported. In this study, we tested the peptide from Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 proteins, CAP peptide as a gall-inducing effector candidate obtained from transcripts isolated from the horned gall aphid, (Schlechtendalia chinensis) through in silico screening and the Arabidopsis-based gall-forming assay, which is a bioassay system for analysing the molecular mechanisms of gall formation. Furthermore, we succeeded in generating an artificial gall in the host plant Veronica peregrina, without any insect parasitism, using three minimal effector elements: CAP peptide, auxin, and cytokinin. Given the strong similarities observed in organ structure with a central cavity and three types of tissue and gene expression patterns between the native and artificial galls, we concluded that CAP peptide is a general gall-inducing effector peptide secreted by gall-inducing insects.\nGenome - nuclear lamina interactions are multivalent and cooperative\nAuthors: Dauban, L.; Eder, M.; de Haas, M.; Franceschini-Santos, V. H.; Yanez-Cuna, J. O.; van Schaik, T.; Leemans, C.; Rademaker, K.; Martinez-Ara, M.; Martinovic, M.; de Wit, E.; van Steensel, B.\nScore: 14.7, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574825\nLamina-associated domains (LADs) are megabase-sized genomic regions that interact with the nuclear lamina (NL). It is not yet understood how their interactions with the NL are encoded in their DNA. Here, we designed an efficient LAD \"scrambling\" approach, based on transposon-mediated local hopping of loxP recombination sites, to generate series of large deletions and inversions that span LADs and flanking sequences. Mapping of NL interactions in these rearrangements revealed that a single LAD contacts the NL through multiple regions that act cooperatively or redundantly; some have more affinity for the NL than others and can pull neighbouring sequences to the NL. Genes drawn towards the NL showed often, but not always, reduced expression and increased H3K9me3 levels. Furthermore, neighbouring LADs can cooperatively interact with the NL when placed close enough to each other. These results elucidate principles that govern the positioning of megabase-sized genomic regions inside the cell nucleus. HighlightsO_LIEfficient generation of LAD rearrangements by transposon hopping of loxP elements. C_LIO_LINL interactions are multivalent, with some subregions being more potent tethers than others. C_LIO_LILADs can cooperate to promote their association to the NL. C_LIO_LIChanges in NL association are often accompanied by changes in gene activity and H3K9me3. C_LI\nA dual histone code specifies the binding of heterochromatin protein Rhino to a subset of piRNA source loci\nAuthors: Akkouche, A.; Kneuss, E.; Bornelöv, S.; Renaud, Y.; Eastwood, E. L.; van Lopik, J.; Gueguen, N.; Jiang, M.; Creixell, P.; Maupetit-Mehouas, S.; Nicholson, B. C.; Brasset, E.; Hannon, G. J.\nScore: 14.2, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575256\nAnimal germ cells deploy a specialized small RNA-based silencing system, called the PIWI-interacting RNA (piRNA) pathway, to prevent unwanted expression of transposable elements and maintain genome integrity. In Drosophila germ cells, the majority of piRNA populations originate from dual-strand piRNA clusters, genomic regions highly enriched in transposon fragments, via an elaborate protein machinery centred on the heterochromatin protein 1 homolog, Rhino. Although Rhino binds to peptides carrying trimethylated H3K9 in vitro, it is not fully understood why in vivo only a fraction of H3K9me3-decorated heterochromatin is occupied by Rhino. Recent work uncovered that Rhino is recruited to a subset of piRNA clusters by the zinc finger protein Kipferl. Here we identify a Kipferl-independent mode of Rhino targeting that is dependent on the histone H3 lysine 27 methyltransferase Enhancer of Zeste and the presence of H3K9me3 and H3K27me3 marks. At Kipferl-independent sites, we find that Rhino, through its dimeric chromodomain, specifically binds to loci marked by both H3K9me3 and H3K27me3. These results expand our understanding of the characteristic binding profile of the heterochromatin protein Rhino. Our work reveals a role for dual histone modifications in defining the binding specificity of a chromatin protein.\nThe mitochondrial mRNA stabilizing protein, SLIRP, regulates skeletal muscle mitochondrial structure and respiration by exercise-recoverable mechanisms\nAuthors: Pham, T. C. P.; Raun, S. H.; Havula, E.; Henriquez-Olguin, C.; Rubalcava-Gracia, D.; Frank, E.; Fritzen, A. M.; Jannig, P. R.; Andersen, N. R.; Kruse, R.; Ali, M.; Halling, J. F.; Ringholm, S.; Needham, E. J.; Hansen, S.; Lemminger, A. K.; Schjerling, P.; Petersen, M. H.; de Almeida, M. E.; Jensen, T. E.; Kiens, B.; Hostrup, M.; Larsen, S.; Ortenblad, N.; Hojlund, K.; Kjaer, M.; Ruas, J. L.; Trifunovic, A.; Wojtaszewski, J. F. P.; Nielsen, J.; Qvortrup, K.; Pilegaard, H.; Richter, E. A.; Sylow, L.\nScore: 16.0, Published: 2024-01-15 DOI: 10.1101/2023.10.30.564600\nThe decline in mitochondrial function is associated with decreased muscle mass and strength in multiple conditions, including sarcopenia, type 2 diabetes, and cancer. Treatment of these diseases could include improving mitochondrial function, however, there are limited and equivocal data regarding the molecular cues that control muscle mitochondrial plasticity. Here we uncover the mitochondrial-mRNA-stabilizing protein SLIRP, in complex with LRPPRC, as a PGC-11 target that regulates mitochondrial structure, respiration, and mitochondrially-encoded-mRNA pools in skeletal muscle. Exercise training effectively counteracted mitochondrial defects induced by loss of LRPPRC/SLIRP, despite sustained low mitochondrially-encoded-mRNA pools, via a mechanism likely including increased mitoribosome translation capacity. In human skeletal muscle, exercise training robustly increased SLIRP and LRPPRC protein content across exercise modalities and sexes, yet this increase was less prominent in type 2 diabetes. Our work identifies a new mechanism of posttranscriptional mitochondrial regulation in skeletal muscle through mitochondrial mRNA stabilization. Moreover, it emphasizes exercise as an effective approach to alleviate mitochondrial dysfunction.\nA CPC-shelterin-BTR axis regulates mitotic telomere deprotection\nAuthors: Romero-Zamora, D.; Rogers, S.; Low, R. R. J.; Robinson, A. B.; Page, S. G.; Lane, B. J. E.; Lamm, N.; Ishikawa, F.; Hayashi, M. T.; Cesare, A. J.\nScore: 9.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574754\nTelomeres prevent ATM activation by sequestering chromosome termini within telomere loops (t-loops). Mitotic arrest promotes telomere linearity and a localized ATM-dependent telomere DNA damage response (DDR) through an unknown mechanism. Using unbiased interactomics, biochemical screening, molecular biology, and super-resolution imaging, we found that mitotic arrest-dependent (MAD) telomere deprotection requires the combined activities of the Chromosome passenger complex (CPC) on shelterin, and the BLM-TOP3A-RMI1/2 (BTR) complex on t-loops. During mitotic arrest, the CPC component Aurora Kinase B (AURKB) phosphorylated both the TRF1 hinge and TRF2 basic domains. The former enhanced CPC and TRF1 interaction through the CPC Survivin subunit, while the latter promoted telomere linearity, telomere DDR activation dependent upon BTR double Holliday junction dissolution activity, and mitotic death. We identify that the TRF2 basic domain functions in mitosis-specific telomere protection and reveal TRF1 regulation over a physiological ATM-dependent telomere DDR. The data demonstrate that MAD telomere deprotection is a sophisticated active mechanism that exposes telomere ends to signal mitotic stress.\nOptimization of Anticodon Edited Transfer RNAs (ACE-tRNAs) Function in Translation for Suppression of Nonsense Mutations\nAuthors: Porter, J. J.; Ko, W.; Sorensen, E. G.; Lueck, J. D.\nScore: 5.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575297\nNonsense suppressor tRNAs or AntiCodon-Edited tRNAs (ACE-tRNAs) have long been envisioned as a therapeutic approach to overcome genetic diseases resulting from the introduction of premature stop codons (PTCs). The ACE-tRNA approach for the rescue of PTCs has been hampered by ineffective delivery through available modalities for gene therapy. Here we have screened a series of ACE-tRNA expression cassette sequence libraries containing \u003e1,800 members to optimize ACE-tRNA function and provide a roadmap for optimization in the future. By optimizing PTC suppression efficiency of ACE-tRNAs, we have decreased the amount of ACE-tRNA required by [~]16-fold for the most common cystic fibrosis-causing PTCs.\nChromatin remodeling initiates transcriptional and post-transcriptional silencing in Neurospora\nAuthors: Carlier, F.; Castro Ramirez, S.; Kilani, J.; Chehboub, S.; Gladyshev, E.\nScore: 4.0, Published: 2024-01-15 DOI: 10.1101/2024.01.15.575522\nIn many eukaryotes, including mammals, transcriptional (heterochromatin-mediated) and post-transcriptional (RNAi-mediated) silencing of repetitive DNA can be initiated by poorly understood mechanisms independent of strong sequence-specific cues. We found that in the fungus Neurospora crassa both types of silencing were induced by the aberrant binding of transcription factors to a synthetic repetitive locus represented by the tetO operator array. Transcriptional silencing of the array was mediated by canonical constitutive heterochromatin. On the other hand, post-transcriptional silencing resembled quelling, but occurred normally in the absence of RAD51 and RAD52, two principal recombination factors. Therefore, this process was named recombination-independent quelling (RIQ). Initiation of RIQ and heterochromatin was associated with perturbed chromatin, and both processes required SAD-6, a SWI/SNF chromatin remodeler orthologous to ATRX. Without SAD-6, and while still occupied by transcription factors, the locus featured an extreme state that was associated with strong nucleosome depletion and high MNase sensitivity, however, this state was unable to promote silencing. We also found that native AT-rich DNA (which comprises relics of transposable elements in N. crassa) could undergo RIQ according to the same principles. These results suggested a general model in which the de novo silencing can be induced by the active remodeling of a perturbed (nucleosome-depleted) chromatin state.\nGeneration of a bloodstream form Trypanosoma brucei double glycosyltransferase null mutant competent in receptor-mediated endocytosis of transferrin\nAuthors: Duncan, S. M.; Carbajo, C. G.; Nagar, R.; Zhong, Q.; Breen, C.; Ferguson, M. A.; Tiengwe, C.\nScore: 3.2, Published: 2024-01-14 DOI: 10.1101/2024.01.14.575564\nThe bloodstream form of Trypanosoma brucei expresses large poly-N-acetyllactosamine (pNAL) chains on complex N-glycans of a subset of glycoproteins. It has been hypothesised that pNAL may be required for receptor-mediated endocytosis. African trypanosomes contain a unique family of glycosyltransferases, the GT67 family. Two of these, TbGT10 and TbGT8, have been shown to be involved in pNAL biosynthesis in bloodstream form Trypanosoma brucei, raising the possibility that deleting both enzymes simultaneously might abolish pNAL biosynthesis and provide clues to pNAL function and/or essentiality. In this paper, we describe the creation of a TbGT10 null mutant containing a single TbGT8 allele that can be excised upon the addition of rapamycin and, from that, a TbGT10 and TbGT8 double null mutant. These mutants were analysed by lectin blotting, glycopeptide methylation linkage analysis and flow cytometry. The data show that the mutants are defective, but not abrogated, in pNAL synthesis, suggesting that other GT67 family members can compensate to some degree for loss of TbGT10 and TbGT8. Despite there being residual pNAL synthesis in these mutants, certain glycoproteins appear to be particularly affected. These include the lysosomal CBP1B serine carboxypeptidase, cell surface ESAG2 and the ESAG6 subunit of the essential parasite transferrin receptor (TfR). The pNAL deficient TfR in the mutants continued to function normally with respect to protein stability, transferrin binding, receptor mediated endocytosis of transferrin and subcellular localisation. Further the pNAL deficient mutants were as viable as wild type parasites in vitro and in in vivo mouse infection experiments. Although we were able to reproduce the inhibition of transferrin uptake with high concentrations of pNAL structural analogues (N-acetylchito-oligosaccharides), this effect disappeared at lower concentrations that still inhibited tomato lectin uptake; i.e., at concentrations able to outcompete lectin-pNAL binding. Based on these findings, we recommend revision of the pNAL-dependent receptor mediated endocytosis hypothesis.\nIntegrative analysis in head and neck cancer reveals distinct role of miRNome and methylome as tumour epigenetic drivers\nAuthors: Mandic, K.; Milutin Gasperov, N.; Bozinovic, K.; Dediol, E.; Sincic, N.; Grce, M.; Sabol, I.; Baresic, A.\nScore: 3.2, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575141\nHead and neck cancer (HNC) is the sixth most common malignancy worldwide with a relatively low 5-year survival rate, mainly because it is diagnosed at a late stage. Current literature suggests that epigenetics is the main driving force for this type of cancer and that HPV infection plays a role in a subset of HNC. In this study, we aimed to find potential epigenetic biomarkers by integrating miRNome, methylome and transcriptome analyses. From the fresh tissue samples of HNC tumours, we selected a group for miRNome, methylome and transcriptome profiling in comparison with appropriate control samples. Bioinformatic analyses were performed in R v4.2.2. Count normalization and group differential expression for mRNA and previously obtained miRNA count data were performed (DESeq2_v1.36). Gene set enrichment analysis was performed and visualised (gProfiler2_v0.2.1). Identification of miRNA targets was performed by querying miRTarBase (multiMiR_v1.18.0). For the integrative analysis, we performed kmeans clustering using 8-12 clusters and nstart 100 (stats_v4.2.2). Transcriptome analysis divided the samples into cancer and control clusters without reference to HPV status or anatomic location. The differentially expressed genes (n=2781) were associated with signaling pathways of tumour progression. We identified a cluster of genes under the control of the transcription factor E2F that were significantly underexpressed in cancer tissue along with T-cell immunity and transcriptional regulation related genes. Among tumour overexpressed genes, we found genes related to cell cycle regulation and vasculature. In this Integrative study we showed distinct role of miRNome and methylome as epigenetic drivers in HNC.\n",
  "wordCount" : "2398",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575070" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575070">
        <p class="paperTitle">Brain and cancer associated binding domain mutations provide insight into CTCF&#39;s relationship with chromatin and its ability to act as a chromatin organizer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.</p>
        <p class="info">Score: 17.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575070' target='https://doi.org/10.1101/2024.01.11.575070'> 10.1101/2024.01.11.575070</a></p>
        <p class="abstract">Although only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown. To investigate the relationship between CTCF and chromatin we applied a combination of imaging, structural and molecular approaches, using a series of brain and cancer associated CTCF mutations that act as CTCF perturbations. We demonstrate that binding and the functional impact of WT and mutant CTCF depend not only on the unique binding properties of each protein, but also on the genomic context of bound sites and enrichment of motifs for expressed TFs abutting these sites. Our studies also highlight the reciprocal relationship between CTCF and chromatin, demonstrating that the unique binding properties of WT and mutant proteins have a distinct impact on accessibility, TF binding, cohesin overlap, chromatin interactivity and gene expression programs, providing insight into their cancer and brain related effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574462">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574462" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574462">
        <p class="paperTitle">CAP peptide artificially induces insect gall</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574462" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574462" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakamoto, T.; Kimura, S.; Nakayama, T.; Sato, M. P.; Shirasawa, K.; Sato, M. H.; Hirano, T.</p>
        <p class="info">Score: 15.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574462' target='https://doi.org/10.1101/2024.01.06.574462'> 10.1101/2024.01.06.574462</a></p>
        <p class="abstract">Galls caused by gall-inducing insects in their host plants clearly illustrate the concept of  extended phenotype, which refers to traits expressed in a host organism when manipulated by a parasite. Candidate effector molecules involved in gall formation, such as phytohormones, amino acids, and proteins, have been reported in numerous studies. However, to date, no attempts to artificially regenerate gall structures using effector candidates have been reported. In this study, we tested the peptide from Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 proteins, CAP peptide as a gall-inducing effector candidate obtained from transcripts isolated from the horned gall aphid, (Schlechtendalia chinensis) through in silico screening and the Arabidopsis-based gall-forming assay, which is a bioassay system for analysing the molecular mechanisms of gall formation. Furthermore, we succeeded in generating an artificial gall in the host plant Veronica peregrina, without any insect parasitism, using three minimal effector elements: CAP peptide, auxin, and cytokinin. Given the strong similarities observed in organ structure with a central cavity and three types of tissue and gene expression patterns between the native and artificial galls, we concluded that CAP peptide is a general gall-inducing effector peptide secreted by gall-inducing insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574825">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574825" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574825">
        <p class="paperTitle">Genome - nuclear lamina interactions are multivalent and cooperative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574825" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574825" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dauban, L.; Eder, M.; de Haas, M.; Franceschini-Santos, V. H.; Yanez-Cuna, J. O.; van Schaik, T.; Leemans, C.; Rademaker, K.; Martinez-Ara, M.; Martinovic, M.; de Wit, E.; van Steensel, B.</p>
        <p class="info">Score: 14.7, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574825' target='https://doi.org/10.1101/2024.01.10.574825'> 10.1101/2024.01.10.574825</a></p>
        <p class="abstract">Lamina-associated domains (LADs) are megabase-sized genomic regions that interact with the nuclear lamina (NL). It is not yet understood how their interactions with the NL are encoded in their DNA. Here, we designed an efficient LAD &#34;scrambling&#34; approach, based on transposon-mediated local hopping of loxP recombination sites, to generate series of large deletions and inversions that span LADs and flanking sequences. Mapping of NL interactions in these rearrangements revealed that a single LAD contacts the NL through multiple regions that act cooperatively or redundantly; some have more affinity for the NL than others and can pull neighbouring sequences to the NL. Genes drawn towards the NL showed often, but not always, reduced expression and increased H3K9me3 levels. Furthermore, neighbouring LADs can cooperatively interact with the NL when placed close enough to each other. These results elucidate principles that govern the positioning of megabase-sized genomic regions inside the cell nucleus.

HighlightsO_LIEfficient generation of LAD rearrangements by transposon hopping of loxP elements.
C_LIO_LINL interactions are multivalent, with some subregions being more potent tethers than others.
C_LIO_LILADs can cooperate to promote their association to the NL.
C_LIO_LIChanges in NL association are often accompanied by changes in gene activity and H3K9me3.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575256">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575256" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575256">
        <p class="paperTitle">A dual histone code specifies the binding of heterochromatin protein Rhino to a subset of piRNA source loci</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575256" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575256" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akkouche, A.; Kneuss, E.; Bornelöv, S.; Renaud, Y.; Eastwood, E. L.; van Lopik, J.; Gueguen, N.; Jiang, M.; Creixell, P.; Maupetit-Mehouas, S.; Nicholson, B. C.; Brasset, E.; Hannon, G. J.</p>
        <p class="info">Score: 14.2, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575256' target='https://doi.org/10.1101/2024.01.11.575256'> 10.1101/2024.01.11.575256</a></p>
        <p class="abstract">Animal germ cells deploy a specialized small RNA-based silencing system, called the PIWI-interacting RNA (piRNA) pathway, to prevent unwanted expression of transposable elements and maintain genome integrity. In Drosophila germ cells, the majority of piRNA populations originate from dual-strand piRNA clusters, genomic regions highly enriched in transposon fragments, via an elaborate protein machinery centred on the heterochromatin protein 1 homolog, Rhino. Although Rhino binds to peptides carrying trimethylated H3K9 in vitro, it is not fully understood why in vivo only a fraction of H3K9me3-decorated heterochromatin is occupied by Rhino. Recent work uncovered that Rhino is recruited to a subset of piRNA clusters by the zinc finger protein Kipferl. Here we identify a Kipferl-independent mode of Rhino targeting that is dependent on the histone H3 lysine 27 methyltransferase Enhancer of Zeste and the presence of H3K9me3 and H3K27me3 marks. At Kipferl-independent sites, we find that Rhino, through its dimeric chromodomain, specifically binds to loci marked by both H3K9me3 and H3K27me3. These results expand our understanding of the characteristic binding profile of the heterochromatin protein Rhino. Our work reveals a role for dual histone modifications in defining the binding specificity of a chromatin protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564600">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564600" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564600">
        <p class="paperTitle">The mitochondrial mRNA stabilizing protein, SLIRP, regulates skeletal muscle mitochondrial structure and respiration by exercise-recoverable mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564600" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564600" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pham, T. C. P.; Raun, S. H.; Havula, E.; Henriquez-Olguin, C.; Rubalcava-Gracia, D.; Frank, E.; Fritzen, A. M.; Jannig, P. R.; Andersen, N. R.; Kruse, R.; Ali, M.; Halling, J. F.; Ringholm, S.; Needham, E. J.; Hansen, S.; Lemminger, A. K.; Schjerling, P.; Petersen, M. H.; de Almeida, M. E.; Jensen, T. E.; Kiens, B.; Hostrup, M.; Larsen, S.; Ortenblad, N.; Hojlund, K.; Kjaer, M.; Ruas, J. L.; Trifunovic, A.; Wojtaszewski, J. F. P.; Nielsen, J.; Qvortrup, K.; Pilegaard, H.; Richter, E. A.; Sylow, L.</p>
        <p class="info">Score: 16.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564600' target='https://doi.org/10.1101/2023.10.30.564600'> 10.1101/2023.10.30.564600</a></p>
        <p class="abstract">The decline in mitochondrial function is associated with decreased muscle mass and strength in multiple conditions, including sarcopenia, type 2 diabetes, and cancer. Treatment of these diseases could include improving mitochondrial function, however, there are limited and equivocal data regarding the molecular cues that control muscle mitochondrial plasticity. Here we uncover the mitochondrial-mRNA-stabilizing protein SLIRP, in complex with LRPPRC, as a PGC-11 target that regulates mitochondrial structure, respiration, and mitochondrially-encoded-mRNA pools in skeletal muscle. Exercise training effectively counteracted mitochondrial defects induced by loss of LRPPRC/SLIRP, despite sustained low mitochondrially-encoded-mRNA pools, via a mechanism likely including increased mitoribosome translation capacity. In human skeletal muscle, exercise training robustly increased SLIRP and LRPPRC protein content across exercise modalities and sexes, yet this increase was less prominent in type 2 diabetes. Our work identifies a new mechanism of posttranscriptional mitochondrial regulation in skeletal muscle through mitochondrial mRNA stabilization. Moreover, it emphasizes exercise as an effective approach to alleviate mitochondrial dysfunction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574754" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574754">
        <p class="paperTitle">A CPC-shelterin-BTR axis regulates mitotic telomere deprotection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Romero-Zamora, D.; Rogers, S.; Low, R. R. J.; Robinson, A. B.; Page, S. G.; Lane, B. J. E.; Lamm, N.; Ishikawa, F.; Hayashi, M. T.; Cesare, A. J.</p>
        <p class="info">Score: 9.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574754' target='https://doi.org/10.1101/2024.01.09.574754'> 10.1101/2024.01.09.574754</a></p>
        <p class="abstract">Telomeres prevent ATM activation by sequestering chromosome termini within telomere loops (t-loops). Mitotic arrest promotes telomere linearity and a localized ATM-dependent telomere DNA damage response (DDR) through an unknown mechanism. Using unbiased interactomics, biochemical screening, molecular biology, and super-resolution imaging, we found that mitotic arrest-dependent (MAD) telomere deprotection requires the combined activities of the Chromosome passenger complex (CPC) on shelterin, and the BLM-TOP3A-RMI1/2 (BTR) complex on t-loops. During mitotic arrest, the CPC component Aurora Kinase B (AURKB) phosphorylated both the TRF1 hinge and TRF2 basic domains. The former enhanced CPC and TRF1 interaction through the CPC Survivin subunit, while the latter promoted telomere linearity, telomere DDR activation dependent upon BTR double Holliday junction dissolution activity, and mitotic death. We identify that the TRF2 basic domain functions in mitosis-specific telomere protection and reveal TRF1 regulation over a physiological ATM-dependent telomere DDR. The data demonstrate that MAD telomere deprotection is a sophisticated active mechanism that exposes telomere ends to signal mitotic stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575297" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575297">
        <p class="paperTitle">Optimization of Anticodon Edited Transfer RNAs (ACE-tRNAs) Function in Translation for Suppression of Nonsense Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porter, J. J.; Ko, W.; Sorensen, E. G.; Lueck, J. D.</p>
        <p class="info">Score: 5.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575297' target='https://doi.org/10.1101/2024.01.11.575297'> 10.1101/2024.01.11.575297</a></p>
        <p class="abstract">Nonsense suppressor tRNAs or AntiCodon-Edited tRNAs (ACE-tRNAs) have long been envisioned as a therapeutic approach to overcome genetic diseases resulting from the introduction of premature stop codons (PTCs). The ACE-tRNA approach for the rescue of PTCs has been hampered by ineffective delivery through available modalities for gene therapy. Here we have screened a series of ACE-tRNA expression cassette sequence libraries containing &gt;1,800 members to optimize ACE-tRNA function and provide a roadmap for optimization in the future. By optimizing PTC suppression efficiency of ACE-tRNAs, we have decreased the amount of ACE-tRNA required by [~]16-fold for the most common cystic fibrosis-causing PTCs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575522" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575522">
        <p class="paperTitle">Chromatin remodeling initiates transcriptional and post-transcriptional silencing in Neurospora</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carlier, F.; Castro Ramirez, S.; Kilani, J.; Chehboub, S.; Gladyshev, E.</p>
        <p class="info">Score: 4.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575522' target='https://doi.org/10.1101/2024.01.15.575522'> 10.1101/2024.01.15.575522</a></p>
        <p class="abstract">In many eukaryotes, including mammals, transcriptional (heterochromatin-mediated) and post-transcriptional (RNAi-mediated) silencing of repetitive DNA can be initiated by poorly understood mechanisms independent of strong sequence-specific cues. We found that in the fungus Neurospora crassa both types of silencing were induced by the aberrant binding of transcription factors to a synthetic repetitive locus represented by the tetO operator array. Transcriptional silencing of the array was mediated by canonical constitutive heterochromatin. On the other hand, post-transcriptional silencing resembled quelling, but occurred normally in the absence of RAD51 and RAD52, two principal recombination factors. Therefore, this process was named recombination-independent quelling (RIQ). Initiation of RIQ and heterochromatin was associated with perturbed chromatin, and both processes required SAD-6, a SWI/SNF chromatin remodeler orthologous to ATRX. Without SAD-6, and while still occupied by transcription factors, the locus featured an extreme state that was associated with strong nucleosome depletion and high MNase sensitivity, however, this state was unable to promote silencing. We also found that native AT-rich DNA (which comprises relics of transposable elements in N. crassa) could undergo RIQ according to the same principles. These results suggested a general model in which the de novo silencing can be induced by the active remodeling of a perturbed (nucleosome-depleted) chromatin state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575564">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575564" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575564">
        <p class="paperTitle">Generation of a bloodstream form Trypanosoma brucei double glycosyltransferase null mutant competent in receptor-mediated endocytosis of transferrin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575564" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575564" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duncan, S. M.; Carbajo, C. G.; Nagar, R.; Zhong, Q.; Breen, C.; Ferguson, M. A.; Tiengwe, C.</p>
        <p class="info">Score: 3.2, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575564' target='https://doi.org/10.1101/2024.01.14.575564'> 10.1101/2024.01.14.575564</a></p>
        <p class="abstract">The bloodstream form of Trypanosoma brucei expresses large poly-N-acetyllactosamine (pNAL) chains on complex N-glycans of a subset of glycoproteins. It has been hypothesised that pNAL may be required for receptor-mediated endocytosis. African trypanosomes contain a unique family of glycosyltransferases, the GT67 family. Two of these, TbGT10 and TbGT8, have been shown to be involved in pNAL biosynthesis in bloodstream form Trypanosoma brucei, raising the possibility that deleting both enzymes simultaneously might abolish pNAL biosynthesis and provide clues to pNAL function and/or essentiality. In this paper, we describe the creation of a TbGT10 null mutant containing a single TbGT8 allele that can be excised upon the addition of rapamycin and, from that, a TbGT10 and TbGT8 double null mutant. These mutants were analysed by lectin blotting, glycopeptide methylation linkage analysis and flow cytometry. The data show that the mutants are defective, but not abrogated, in pNAL synthesis, suggesting that other GT67 family members can compensate to some degree for loss of TbGT10 and TbGT8. Despite there being residual pNAL synthesis in these mutants, certain glycoproteins appear to be particularly affected. These include the lysosomal CBP1B serine carboxypeptidase, cell surface ESAG2 and the ESAG6 subunit of the essential parasite transferrin receptor (TfR). The pNAL deficient TfR in the mutants continued to function normally with respect to protein stability, transferrin binding, receptor mediated endocytosis of transferrin and subcellular localisation. Further the pNAL deficient mutants were as viable as wild type parasites in vitro and in in vivo mouse infection experiments. Although we were able to reproduce the inhibition of transferrin uptake with high concentrations of pNAL structural analogues (N-acetylchito-oligosaccharides), this effect disappeared at lower concentrations that still inhibited tomato lectin uptake; i.e., at concentrations able to outcompete lectin-pNAL binding. Based on these findings, we recommend revision of the pNAL-dependent receptor mediated endocytosis hypothesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575141">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575141" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575141">
        <p class="paperTitle">Integrative analysis in head and neck cancer reveals distinct role of miRNome and methylome as tumour epigenetic drivers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575141" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575141" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mandic, K.; Milutin Gasperov, N.; Bozinovic, K.; Dediol, E.; Sincic, N.; Grce, M.; Sabol, I.; Baresic, A.</p>
        <p class="info">Score: 3.2, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575141' target='https://doi.org/10.1101/2024.01.11.575141'> 10.1101/2024.01.11.575141</a></p>
        <p class="abstract">Head and neck cancer (HNC) is the sixth most common malignancy worldwide with a relatively low 5-year survival rate, mainly because it is diagnosed at a late stage. Current literature suggests that epigenetics is the main driving force for this type of cancer and that HPV infection plays a role in a subset of HNC. In this study, we aimed to find potential epigenetic biomarkers by integrating miRNome, methylome and transcriptome analyses. From the fresh tissue samples of HNC tumours, we selected a group for miRNome, methylome and transcriptome profiling in comparison with appropriate control samples. Bioinformatic analyses were performed in R v4.2.2. Count normalization and group differential expression for mRNA and previously obtained miRNA count data were performed (DESeq2_v1.36). Gene set enrichment analysis was performed and visualised (gProfiler2_v0.2.1). Identification of miRNA targets was performed by querying miRTarBase (multiMiR_v1.18.0). For the integrative analysis, we performed kmeans clustering using 8-12 clusters and nstart 100 (stats_v4.2.2). Transcriptome analysis divided the samples into cancer and control clusters without reference to HPV status or anatomic location. The differentially expressed genes (n=2781) were associated with signaling pathways of tumour progression. We identified a cluster of genes under the control of the transcription factor E2F that were significantly underexpressed in cancer tissue along with T-cell immunity and transcriptional regulation related genes. Among tumour overexpressed genes, we found genes related to cell cycle regulation and vasculature. In this Integrative study we showed distinct role of miRNome and methylome as epigenetic drivers in HNC.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
