<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells
Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.
Score: 24.2, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756
Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells
Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.
Score: 24.2, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756
Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells
Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.
Score: 24.2, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756
Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells\nAuthors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.\nScore: 24.2, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756\nDeveloping therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate.",
  "keywords": [
    
  ],
  "articleBody": " Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells\nAuthors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.\nScore: 24.2, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756\nDeveloping therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate. We show three examples where GLS expression increases during coronavirus infection of host cells, and another in which GLS2 is upregulated. The viruses hijack the metabolic machinery responsible for glutamine metabolism to generate the building blocks for biosynthetic processes and satisfy the bioenergetic requirements demanded by the glutamine addiction of virus-infected host cells. We demonstrate how genetic silencing of glutaminase enzymes reduces coronavirus infection and that newer members of two classes of small molecule allosteric inhibitors targeting these enzymes, designated as SU1, a pan-GLS/GLS2 inhibitor, and UP4, which is specific for GLS, block viral replication in mammalian epithelial cells. Overall, these findings highlight the importance of glutamine metabolism for coronavirus replication in human cells and show that glutaminase inhibitors can block coronavirus infection and thereby may represent a novel class of anti-viral drug candidates. TeaserInhibitors targeting glutaminase enzymes block coronavirus replication and may represent a new class of anti-viral drugs.\n53BP1 interacts with the RNA primer from Okazaki fragments to support their processing during unperturbed DNA replication\nAuthors: Leriche, M.; Bonnet, C.; Jana, J.; Chhetri, G.; Mennour, S.; Martineau, S.; Pennaneach, V.; BUSSO, D.; Veaute, X.; Bertrand, P.; lambert, s. A.; Somyajit, K.; Uguen, P.; Vagner, S.\nScore: 14.2, Published: 2023-09-27 DOI: 10.1101/2023.09.27.559698\nRNA-binding proteins are found at replication forks, but their direct interaction with DNA-embedded RNA species that inevitably shape physiological DNA replication remains unexplored. Here we report that 53BP1, involved in the DNA damage and replication stress response, is an RNA-binding protein that directly interacts with Okazaki fragments, in the absence of any external stress. The bulk chromatin association of 53BP1 shows dramatic dependence on PRIM1, which synthesizes the RNA primer of Okazaki fragments. The direct recruitment of 53BP1 to nascent DNA shows susceptibility to in situ ribonuclease A treatment. Conversely, depletion of FEN1, which results in the accumulation of uncleaved RNA primers, leads to an upregulation of 53BP1 levels at the replication forks, suggesting that RNA primers contribute to the recruitment of 53BP1 at the lagging DNA strand. 53BP1 depletion induces an accumulation of S phase poly(ADP-ribose), which constitutes a sensor of unligated Okazaki fragments. Collectively, our data indicate that 53BP1, distinct from its canonical mode of chromatin-binding, is anchored at the replication fork through its RNA-binding activity, highlighting the role of an RNA-protein interaction at DNA replication forks.\nNucleolar dynamics are determined by the ordered assembly of the ribosome\nAuthors: Sheu-Gruttadauria, J.; Yan, X.; Stuurman, N.; Floor, S. N.; Vale, R. D.\nScore: 13.2, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559432\nRibosome biogenesis is coordinated within the nucleolus, a biomolecular condensate that exhibits dynamic material properties that are thought to be important for nucleolar function. However, the relationship between ribosome assembly and nucleolar dynamics is not clear. Here, we screened 364 genes involved in ribosome biogenesis and RNA metabolism for their impact on dynamics of the nucleolus, as measured by automated, high-throughput fluorescence recovery after photobleaching (FRAP) of the nucleolar scaffold protein NPM1. This screen revealed that gene knockdowns that caused accumulation of early rRNA intermediates were associated with nucleolar rigidification, while accumulation of late intermediates led to increased fluidity. These shifts in dynamics were accompanied by distinct changes in nucleolar morphology. We also found that genes involved in mRNA processing impact nucleolar dynamics, revealing connections between ribosome biogenesis and other RNA processing pathways. Together, this work defines mechanistic ties between ribosome assembly and the biophysical features of the nucleolus, while establishing a toolbox for understanding how molecular dynamics impact function across other biomolecular condensates.\nBacterial histone HBb from Bdellovibrio bacteriovorus compacts DNA by bending\nAuthors: Hu, Y.; Schwab, S.; Deiss, S.; Escudeiro, P.; Joiner, J. D.; Hartmann, M. D.; Lupas, A. N.; Alvarez, B. H.; Alva, V.; Dame, R. T.\nScore: 27.1, Published: 2023-09-27 DOI: 10.1101/2023.02.26.530074\nHistones are essential for genome compaction and transcription regulation in eukaryotes, where they assemble into octamers to form the nucleosome core. In contrast, archaeal histones assemble into dimers that form hypernucleosomes upon DNA binding. Although histone homologs have recently been identified in bacteria, their DNA-binding characteristics remain largely unexplored. Our study reveals that the bacterial histone HBb is indispensable for the survival of Bdellovibrio bacteriovorus, suggesting critical roles in DNA organization and gene regulation. We also elucidate the crystal structure of the HBb dimer at 1.06 [A] resolution and employ various biophysico-chemical approaches to show its ability to bind and compact DNA in a sequence-independent manner. This binding induces DNA bending, similar to that observed with bacterial HU/IHF family proteins. Finally, using DNA affinity purification and sequencing, we reveal that HBb binds along the entire genomic DNA of B. bacteriovorus without sequence specificity. These unique DNA-binding properties of bacterial histones, distinct from their archaeal and eukaryotic counterparts, highlight the diverse roles that histones can play in DNA organization across the domains of life. Author SummaryHistones, traditionally known for organizing and regulating DNA in eukaryotes and archaea, have recently been discovered in bacteria, opening up a new frontier in our understanding of genome organization across the domains of life. Our study investigates the largely unexplored DNA-binding properties of bacterial histones, focusing on HBb in Bdellovibrio bacteriovorus. We reveal that HBb is essential for bacterial survival and exhibits DNA-binding akin to the bacterial nucleoid-associated HU/IHF family proteins. Contrary to eukaryotic and archaeal histones that wrap DNA, HBb bends DNA without sequence specificity. This work not only broadens our understanding of DNA organization across different life forms but also suggests that bacterial histones may have diverse roles in genome organization, distinct from their eukaryotic and archaeal counterparts.\nNon-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity\nAuthors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O'Riordan, M. X. D.; O'Meara, T. R.\nScore: 10.2, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560560\nInfection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated. While recognition of microbe-associated molecular patterns can activate IRE1, it is unclear whether this depends on the canonical role of IRE1 in detecting misfolded proteins. Here, we report that Candida albicans infection of macrophages results in IRE1 activation through C-type lectin receptor signaling, reinforcing a role for IRE1 as a central regulator of host responses to infection by a broad range of pathogens. However, IRE1 activation was not preceded by protein misfolding in response to either C. albicans infection or lipopolysaccharide treatment, implicating a non-canonical mode of IRE1 activation after recognition of microbial patterns. Investigation of the phenotypic consequences of IRE1 activation in macrophage antimicrobial responses revealed that IRE1 activity enhances the fungicidal activity of macrophages. Macrophages lacking IRE1 activity displayed inefficient phagolysosomal fusion, enabling C. albicans to evade fungal killing and escape the phagosome. Together, these data provide mechanistic insight for the non-canonical activation of IRE1 during infection, and reveal central roles for IRE1 in macrophage antifungal responses.\nAltered chromatin occupancy of patient-associated H4 mutants misregulate neuronal differentiation\nAuthors: Feng, L.; Barrows, D.; Zhong, L.; Matlik, K.; Porter, E. G.; Djomo, A. M.; Yau, I.; Soshnev, A. A.; Carroll, T. S.; Wen, D.; Hatten, M. E.; Garcia, B. A.; Allis, C. D.\nScore: 6.7, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560141\nChromatin is a crucial regulator of gene expression and tightly controls development across species. Mutations in only one copy of multiple histone genes were identified in children with developmental disorders characterized by microcephaly, but their mechanistic roles in development remain unclear. Here we focus on dominant mutations affecting histone H4 lysine 91. These H4K91 mutants form aberrant nuclear puncta at specific heterochromatin regions. Mechanistically, H4K91 mutants demonstrate enhanced binding to the histone variant H3.3, and ablation of H3.3 or the H3.3-specific chaperone DAXX diminishes the mutant localization to chromatin. Our functional studies demonstrate that H4K91 mutant expression increases chromatin accessibility, alters developmental gene expression through accelerating pro-neural differentiation, and causes reduced mouse brain size in vivo, reminiscent of the microcephaly phenotypes of patients. Together, our studies unveil a distinct molecular pathogenic mechanism from other known histone mutants, where H4K91 mutants misregulate cell fate during development through abnormal genomic localization.\nMolecular principles of CRISPR-Cas13 mismatch intolerance enable selective silencing of point-mutated oncogenic RNA with single-base precision\nAuthors: Shembrey, C.; Yang, R.; Casan, J.; Hu, W.; Sadras, T.; Prasad, K.; Shortt, J.; Johnstone, R. W.; Trapani, J. A.; Ekert, P. G.; Fareh, M.\nScore: 6.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.557083\nSingle nucleotide variants (SNVs) are extremely prevalent in human cancers. For instance, KRAS mutations occur in over 90% of pancreatic cancers and [~]40% of colorectal cancers. Virtually all KRAS mutations are SNVs, most of which remain clinically unactionable. The programmable RNA nuclease CRISPR-Cas13 has been deployed to specifically target RNAs such as overexpressed oncogenes and fusion transcripts. However, silencing oncogenic SNVs with single-base precision remains extremely challenging due to the intrinsic mismatch tolerance of Cas13. Here, we developed a comprehensive mutagenesis analysis of target-spacer interactions at single-nucleotide resolution, which revealed key spacer nucleotide positions intolerant to mismatches. We show that introducing synthetic mismatches at these precise positions enables de nova design of CRISPR RNA (crRNA) with strong preferential silencing of SNV transcripts. We demonstrate that our top-performing crRNAs possess prominent SNV-selectivity with dose-dependent silencing activity against all KRAS G12 variants at both the RNA and protein levels with minimal off-target silencing of wildtype KRAS. We applied these design principles to effectively silence oncogenic NRAS G12D and BRAF V600E transcripts, underscoring the adaptability of this platform to silence various SNVs. These findings demonstrate that the CRISPR Cas13 system can be reprogrammed to target mutant transcripts with single-base precision, showcasing the tremendous potential of this tool in personalized transcriptome editing. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=176 SRC=\"FIGDIR/small/557083v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (58K): org.highwire.dtl.DTLVardef@104d05forg.highwire.dtl.DTLVardef@10c2f58org.highwire.dtl.DTLVardef@f87463org.highwire.dtl.DTLVardef@1a13c30_HPS_FORMAT_FIGEXP M_FIG C_FIG\nInterleukin 11 therapy causes acute heart failure and its use in patients should be reconsidered\nAuthors: Sweeney, M.; O'Fee, K.; Villanueva-Hayes, C.; Rahman, E.; Lee, M.; Maatz, H.; Lindberg, E. L.; Vanezis, K.; Andrew, I.; Jennings, E. R.; Lim, W.-W.; Widjaja, A. A.; Hübner, N.; Barton, P. J. R.; Cook, S. A.\nScore: 6.5, Published: 2023-10-01 DOI: 10.1101/2023.09.30.560259\nBackground: Interleukin 11 (IL11) was initially thought important for platelet production, which led to recombinant IL11 being developed as a drug to treat thrombocytopenia. IL11 was later found to be redundant for haematopoiesis and its use in patients is associated with unexplained cardiac side effects. Here we identify previously unappreciated and direct cardiomyocyte toxicities associated with IL11 therapy. Methods: We injected recombinant mouse lL11 (rmIL11) into mice and studied its molecular effects in the heart using immunoblotting, qRT-PCR, bulk RNA-seq, single nuclei RNA-seq (snRNA-seq) and ATAC-seq. The physiological impact of IL11 was assessed by echocardiography in vivo and using cardiomyocyte contractility assays in vitro. To determine the activity of IL11 specifically in cardiomyocytes we made two cardiomyocyte-specific Il11ra1 knockout mouse models using either AAV9-mediated and Tnnt2-restricted (vCMKO) or Myh6 (m6CMKO) Cre expression and an Il11ra1 floxed mouse strain. In pharmacologic studies, we studied the effects of JAK/STAT inhibition on rmIL11-induced cardiac toxicities. Results: Injection of rmIL11 caused acute and dose-dependent impairment of left ventricular ejection fraction (saline (2 {micro}L/kg), 60.4{+/-} 3.1; rmIL11 (200 mcg/kg), 31.6{+/-} 2.0; p\u003c0.0001, n=5). Following rmIL11 injection, myocardial STAT3 and JNK phosphorylation were increased and bulk RNA-seq revealed upregulation of pro-inflammatory pathways (TNFa, NFkB and JAK/STAT) and perturbed calcium handling. SnRNA-seq showed rmIL11-induced expression of stress factors (Ankrd1, Ankrd23, Xirp2), activator protein-1 (AP-1) transcription factor genes and Nppb in the cardiomyocyte compartment. Following rmIL11 injection, ATAC-seq identified epigenetic enrichment of the Ankrd1 and Nppb genes and stress-responsive, AP-1 transcription factor binding sites. Cardiomyocyte-specific effects were examined in vCMKO and m6CMKO mice, which were both protected from rmIL11-induced left ventricular impairment and molecular pathobiologies. In mechanistic studies, inhibition of JAK/STAT signalling with either ruxolitinib or tofacitinib prevented rmIL11-induced cardiac dysfunction. Conclusions: Injection of IL11 directly activates JAK/STAT3 in cardiomyocytes to cause acute heart failure. Our data overturn the earlier assumption that IL11 is cardioprotective and explain the serious cardiac side effects associated with IL11 therapy, which questions its continued use in patients.\nA conserved class of viral RNA structures regulate translation reinitiation through dynamic ribosome interactions\nAuthors: Sherlock, M.; Langeberg, C.; Segar, K.; Kieft, J. S.\nScore: 6.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560040\nCertain viruses encode proteins downstream of the main coding region expressed through non-canonical translation termination-reinitiation events. A class of RNA structure located upstream of the first stop codon within viral mRNAs directly binds the ribosome, preventing ribosome recycling and inducing reinitiation. We used bioinformatic methods to identify new examples of viral reinitiation-stimulating RNAs and experimentally verified their secondary structure and function. We determined the structure of a representative viral RNA-ribosome complex using cryoEM. 3D classification and variability analyses reveal that the viral RNA structure can sample a range of conformations while remaining tethered to the ribosome, which enables the ribosome to find a reinitiation start site within a limited range of mRNA sequence. Evaluating the conserved features and constraints of this entire RNA class in the context of the cryoEM reconstruction provides insight into mechanisms enabling reinitiation, a translation regulation strategy employed by many other viral and eukaryotic systems.\nStability of Genomic Imprinting and X-Chromosome Inactivation in the Aging Brain\nAuthors: Mancino, S.; Seneviratne, J.; Mupo, A.; Krueger, F.; Oxley, D.; Eckersley-Maslin, M.; da Rocha, S. T.\nScore: 6.2, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560184\nEpigenetic drift is a hallmark of aging that contributes to the irreversible decline in organismal fitness ultimately leading to aging-related diseases. Epigenetic modifications regulate the cellular memory of the epigenetic processes of genomic imprinting and X-chromosome inactivation to ensure monoallelic expression of imprinted and X-linked genes. Whether epigenetic drift affects maintenance of genomic imprinting and X-chromosome inactivation has not been comprehensively studied. Here, we investigate the allele-specific transcriptional and epigenetic signatures of the aging brain, by comparing juvenile and old hybrid mice obtained from C57BL/6J (BL6) \u0026 CAST/EiJ (CAST) reciprocal crosses, with an emphasis on the hippocampus (HCP). We confirm that the aged HCP shows an increase of DNA hydroxymethylation, a sign of epigenetic drift, and a typical aging transcriptional signature. Genomic imprinting was found to be largely unaffected with stable parent-of-origin-specific DNA methylation in HCP, but also other brain regions such as the cerebellum (CB), nucleus accumbens, hypothalamus and prefrontal cortex. Consistently, transcriptomics analysis confirmed unaltered imprinting expression in the aged HCP. An exception are three novel non-coding transcripts (B230209E15Rik, Ube2nl and A330076H08Rik) at the Prader-Willi syndrome/Angelman syndrome (PWS/AS) imprinted locus which lose strict monoallelic expression during aging. Like imprinting, X-chromosome inactivation was remarkably stable with no signs of aging-driven skewing or relaxation of monoallelic expression of X-linked genes. Our study provides a valuable resource for evaluating monoallelic expression in the aging brain and reveals that, despite epigenetic drift during aging, genomic imprinting and X-chromosome inactivation remain predominantly stable throughout the process of physiological aging in the mouse brain.\n",
  "wordCount" : "2572",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559756">
        <p class="paperTitle">Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.</p>
        <p class="info">Score: 24.2, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559756' target='https://doi.org/10.1101/2023.09.27.559756'> 10.1101/2023.09.27.559756</a></p>
        <p class="abstract">Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate. We show three examples where GLS expression increases during coronavirus infection of host cells, and another in which GLS2 is upregulated. The viruses hijack the metabolic machinery responsible for glutamine metabolism to generate the building blocks for biosynthetic processes and satisfy the bioenergetic requirements demanded by the  glutamine addiction of virus-infected host cells. We demonstrate how genetic silencing of glutaminase enzymes reduces coronavirus infection and that newer members of two classes of small molecule allosteric inhibitors targeting these enzymes, designated as SU1, a pan-GLS/GLS2 inhibitor, and UP4, which is specific for GLS, block viral replication in mammalian epithelial cells. Overall, these findings highlight the importance of glutamine metabolism for coronavirus replication in human cells and show that glutaminase inhibitors can block coronavirus infection and thereby may represent a novel class of anti-viral drug candidates.

TeaserInhibitors targeting glutaminase enzymes block coronavirus replication and may represent a new class of anti-viral drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559698">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559698" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559698">
        <p class="paperTitle">53BP1 interacts with the RNA primer from Okazaki fragments to support their processing during unperturbed DNA replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559698" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559698" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leriche, M.; Bonnet, C.; Jana, J.; Chhetri, G.; Mennour, S.; Martineau, S.; Pennaneach, V.; BUSSO, D.; Veaute, X.; Bertrand, P.; lambert, s. A.; Somyajit, K.; Uguen, P.; Vagner, S.</p>
        <p class="info">Score: 14.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559698' target='https://doi.org/10.1101/2023.09.27.559698'> 10.1101/2023.09.27.559698</a></p>
        <p class="abstract">RNA-binding proteins are found at replication forks, but their direct interaction with DNA-embedded RNA species that inevitably shape physiological DNA replication remains unexplored. Here we report that 53BP1, involved in the DNA damage and replication stress response, is an RNA-binding protein that directly interacts with Okazaki fragments, in the absence of any external stress. The bulk chromatin association of 53BP1 shows dramatic dependence on PRIM1, which synthesizes the RNA primer of Okazaki fragments. The direct recruitment of 53BP1 to nascent DNA shows susceptibility to in situ ribonuclease A treatment. Conversely, depletion of FEN1, which results in the accumulation of uncleaved RNA primers, leads to an upregulation of 53BP1 levels at the replication forks, suggesting that RNA primers contribute to the recruitment of 53BP1 at the lagging DNA strand. 53BP1 depletion induces an accumulation of S phase poly(ADP-ribose), which constitutes a sensor of unligated Okazaki fragments. Collectively, our data indicate that 53BP1, distinct from its canonical mode of chromatin-binding, is anchored at the replication fork through its RNA-binding activity, highlighting the role of an RNA-protein interaction at DNA replication forks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559432">
        <p class="paperTitle">Nucleolar dynamics are determined by the ordered assembly of the ribosome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheu-Gruttadauria, J.; Yan, X.; Stuurman, N.; Floor, S. N.; Vale, R. D.</p>
        <p class="info">Score: 13.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559432' target='https://doi.org/10.1101/2023.09.26.559432'> 10.1101/2023.09.26.559432</a></p>
        <p class="abstract">Ribosome biogenesis is coordinated within the nucleolus, a biomolecular condensate that exhibits dynamic material properties that are thought to be important for nucleolar function. However, the relationship between ribosome assembly and nucleolar dynamics is not clear. Here, we screened 364 genes involved in ribosome biogenesis and RNA metabolism for their impact on dynamics of the nucleolus, as measured by automated, high-throughput fluorescence recovery after photobleaching (FRAP) of the nucleolar scaffold protein NPM1. This screen revealed that gene knockdowns that caused accumulation of early rRNA intermediates were associated with nucleolar rigidification, while accumulation of late intermediates led to increased fluidity. These shifts in dynamics were accompanied by distinct changes in nucleolar morphology. We also found that genes involved in mRNA processing impact nucleolar dynamics, revealing connections between ribosome biogenesis and other RNA processing pathways. Together, this work defines mechanistic ties between ribosome assembly and the biophysical features of the nucleolus, while establishing a toolbox for understanding how molecular dynamics impact function across other biomolecular condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.26.530074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.26.530074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.26.530074">
        <p class="paperTitle">Bacterial histone HBb from Bdellovibrio bacteriovorus compacts DNA by bending</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.26.530074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.26.530074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Y.; Schwab, S.; Deiss, S.; Escudeiro, P.; Joiner, J. D.; Hartmann, M. D.; Lupas, A. N.; Alvarez, B. H.; Alva, V.; Dame, R. T.</p>
        <p class="info">Score: 27.1, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.26.530074' target='https://doi.org/10.1101/2023.02.26.530074'> 10.1101/2023.02.26.530074</a></p>
        <p class="abstract">Histones are essential for genome compaction and transcription regulation in eukaryotes, where they assemble into octamers to form the nucleosome core. In contrast, archaeal histones assemble into dimers that form hypernucleosomes upon DNA binding. Although histone homologs have recently been identified in bacteria, their DNA-binding characteristics remain largely unexplored. Our study reveals that the bacterial histone HBb is indispensable for the survival of Bdellovibrio bacteriovorus, suggesting critical roles in DNA organization and gene regulation. We also elucidate the crystal structure of the HBb dimer at 1.06 [A] resolution and employ various biophysico-chemical approaches to show its ability to bind and compact DNA in a sequence-independent manner. This binding induces DNA bending, similar to that observed with bacterial HU/IHF family proteins. Finally, using DNA affinity purification and sequencing, we reveal that HBb binds along the entire genomic DNA of B. bacteriovorus without sequence specificity. These unique DNA-binding properties of bacterial histones, distinct from their archaeal and eukaryotic counterparts, highlight the diverse roles that histones can play in DNA organization across the domains of life.

Author SummaryHistones, traditionally known for organizing and regulating DNA in eukaryotes and archaea, have recently been discovered in bacteria, opening up a new frontier in our understanding of genome organization across the domains of life. Our study investigates the largely unexplored DNA-binding properties of bacterial histones, focusing on HBb in Bdellovibrio bacteriovorus. We reveal that HBb is essential for bacterial survival and exhibits DNA-binding akin to the bacterial nucleoid-associated HU/IHF family proteins. Contrary to eukaryotic and archaeal histones that wrap DNA, HBb bends DNA without sequence specificity. This work not only broadens our understanding of DNA organization across different life forms but also suggests that bacterial histones may have diverse roles in genome organization, distinct from their eukaryotic and archaeal counterparts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560560">
        <p class="paperTitle">Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O&#39;Riordan, M. X. D.; O&#39;Meara, T. R.</p>
        <p class="info">Score: 10.2, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560560' target='https://doi.org/10.1101/2023.10.02.560560'> 10.1101/2023.10.02.560560</a></p>
        <p class="abstract">Infection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated. While recognition of microbe-associated molecular patterns can activate IRE1, it is unclear whether this depends on the canonical role of IRE1 in detecting misfolded proteins. Here, we report that Candida albicans infection of macrophages results in IRE1 activation through C-type lectin receptor signaling, reinforcing a role for IRE1 as a central regulator of host responses to infection by a broad range of pathogens. However, IRE1 activation was not preceded by protein misfolding in response to either C. albicans infection or lipopolysaccharide treatment, implicating a non-canonical mode of IRE1 activation after recognition of microbial patterns. Investigation of the phenotypic consequences of IRE1 activation in macrophage antimicrobial responses revealed that IRE1 activity enhances the fungicidal activity of macrophages. Macrophages lacking IRE1 activity displayed inefficient phagolysosomal fusion, enabling C. albicans to evade fungal killing and escape the phagosome. Together, these data provide mechanistic insight for the non-canonical activation of IRE1 during infection, and reveal central roles for IRE1 in macrophage antifungal responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560141">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560141" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560141">
        <p class="paperTitle">Altered chromatin occupancy of patient-associated H4 mutants misregulate neuronal differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560141" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560141" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, L.; Barrows, D.; Zhong, L.; Matlik, K.; Porter, E. G.; Djomo, A. M.; Yau, I.; Soshnev, A. A.; Carroll, T. S.; Wen, D.; Hatten, M. E.; Garcia, B. A.; Allis, C. D.</p>
        <p class="info">Score: 6.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560141' target='https://doi.org/10.1101/2023.09.29.560141'> 10.1101/2023.09.29.560141</a></p>
        <p class="abstract">Chromatin is a crucial regulator of gene expression and tightly controls development across species. Mutations in only one copy of multiple histone genes were identified in children with developmental disorders characterized by microcephaly, but their mechanistic roles in development remain unclear. Here we focus on dominant mutations affecting histone H4 lysine 91. These H4K91 mutants form aberrant nuclear puncta at specific heterochromatin regions. Mechanistically, H4K91 mutants demonstrate enhanced binding to the histone variant H3.3, and ablation of H3.3 or the H3.3-specific chaperone DAXX diminishes the mutant localization to chromatin. Our functional studies demonstrate that H4K91 mutant expression increases chromatin accessibility, alters developmental gene expression through accelerating pro-neural differentiation, and causes reduced mouse brain size in vivo, reminiscent of the microcephaly phenotypes of patients. Together, our studies unveil a distinct molecular pathogenic mechanism from other known histone mutants, where H4K91 mutants misregulate cell fate during development through abnormal genomic localization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.557083">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.557083" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.557083">
        <p class="paperTitle">Molecular principles of CRISPR-Cas13 mismatch intolerance enable selective silencing of point-mutated oncogenic RNA with single-base precision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.557083" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.557083" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shembrey, C.; Yang, R.; Casan, J.; Hu, W.; Sadras, T.; Prasad, K.; Shortt, J.; Johnstone, R. W.; Trapani, J. A.; Ekert, P. G.; Fareh, M.</p>
        <p class="info">Score: 6.7, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.557083' target='https://doi.org/10.1101/2023.09.26.557083'> 10.1101/2023.09.26.557083</a></p>
        <p class="abstract">Single nucleotide variants (SNVs) are extremely prevalent in human cancers. For instance, KRAS mutations occur in over 90% of pancreatic cancers and [~]40% of colorectal cancers. Virtually all KRAS mutations are SNVs, most of which remain clinically unactionable. The programmable RNA nuclease CRISPR-Cas13 has been deployed to specifically target RNAs such as overexpressed oncogenes and fusion transcripts. However, silencing oncogenic SNVs with single-base precision remains extremely challenging due to the intrinsic mismatch tolerance of Cas13. Here, we developed a comprehensive mutagenesis analysis of target-spacer interactions at single-nucleotide resolution, which revealed key spacer nucleotide positions intolerant to mismatches. We show that introducing synthetic mismatches at these precise positions enables de nova design of CRISPR RNA (crRNA) with strong preferential silencing of SNV transcripts. We demonstrate that our top-performing crRNAs possess prominent SNV-selectivity with dose-dependent silencing activity against all KRAS G12 variants at both the RNA and protein levels with minimal off-target silencing of wildtype KRAS. We applied these design principles to effectively silence oncogenic NRAS G12D and BRAF V600E transcripts, underscoring the adaptability of this platform to silence various SNVs. These findings demonstrate that the CRISPR Cas13 system can be reprogrammed to target mutant transcripts with single-base precision, showcasing the tremendous potential of this tool in personalized transcriptome editing.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=176 SRC=&#34;FIGDIR/small/557083v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (58K):
org.highwire.dtl.DTLVardef@104d05forg.highwire.dtl.DTLVardef@10c2f58org.highwire.dtl.DTLVardef@f87463org.highwire.dtl.DTLVardef@1a13c30_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560259">
        <p class="paperTitle">Interleukin 11 therapy causes acute heart failure and its use in patients should be reconsidered</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sweeney, M.; O&#39;Fee, K.; Villanueva-Hayes, C.; Rahman, E.; Lee, M.; Maatz, H.; Lindberg, E. L.; Vanezis, K.; Andrew, I.; Jennings, E. R.; Lim, W.-W.; Widjaja, A. A.; Hübner, N.; Barton, P. J. R.; Cook, S. A.</p>
        <p class="info">Score: 6.5, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560259' target='https://doi.org/10.1101/2023.09.30.560259'> 10.1101/2023.09.30.560259</a></p>
        <p class="abstract">Background: Interleukin 11 (IL11) was initially thought important for platelet production, which led to recombinant IL11 being developed as a drug to treat thrombocytopenia. IL11 was later found to be redundant for haematopoiesis and its use in patients is associated with unexplained cardiac side effects. Here we identify previously unappreciated and direct cardiomyocyte toxicities associated with IL11 therapy. Methods: We injected recombinant mouse lL11 (rmIL11) into mice and studied its molecular effects in the heart using immunoblotting, qRT-PCR, bulk RNA-seq, single nuclei RNA-seq (snRNA-seq) and ATAC-seq. The physiological impact of IL11 was assessed by echocardiography in vivo and using cardiomyocyte contractility assays in vitro. To determine the activity of IL11 specifically in cardiomyocytes we made two cardiomyocyte-specific Il11ra1 knockout mouse models using either AAV9-mediated and Tnnt2-restricted (vCMKO) or Myh6 (m6CMKO) Cre expression and an Il11ra1 floxed mouse strain. In pharmacologic studies, we studied the effects of JAK/STAT inhibition on rmIL11-induced cardiac toxicities. Results: Injection of rmIL11 caused acute and dose-dependent impairment of left ventricular ejection fraction (saline (2 {micro}L/kg), 60.4{&#43;/-} 3.1; rmIL11 (200 mcg/kg), 31.6{&#43;/-} 2.0; p&lt;0.0001, n=5). Following rmIL11 injection, myocardial STAT3 and JNK phosphorylation were increased and bulk RNA-seq revealed upregulation of pro-inflammatory pathways (TNFa, NFkB and JAK/STAT) and perturbed calcium handling. SnRNA-seq showed rmIL11-induced expression of stress factors (Ankrd1, Ankrd23, Xirp2), activator protein-1 (AP-1) transcription factor genes and Nppb in the cardiomyocyte compartment. Following rmIL11 injection, ATAC-seq identified epigenetic enrichment of the Ankrd1 and Nppb genes and stress-responsive, AP-1 transcription factor binding sites. Cardiomyocyte-specific effects were examined in vCMKO and m6CMKO mice, which were both protected from rmIL11-induced left ventricular impairment and molecular pathobiologies. In mechanistic studies, inhibition of JAK/STAT signalling with either ruxolitinib or tofacitinib prevented rmIL11-induced cardiac dysfunction. Conclusions: Injection of IL11 directly activates JAK/STAT3 in cardiomyocytes to cause acute heart failure. Our data overturn the earlier assumption that IL11 is cardioprotective and explain the serious cardiac side effects associated with IL11 therapy, which questions its continued use in patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560040">
        <p class="paperTitle">A conserved class of viral RNA structures regulate translation reinitiation through dynamic ribosome interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sherlock, M.; Langeberg, C.; Segar, K.; Kieft, J. S.</p>
        <p class="info">Score: 6.5, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560040' target='https://doi.org/10.1101/2023.09.29.560040'> 10.1101/2023.09.29.560040</a></p>
        <p class="abstract">Certain viruses encode proteins downstream of the main coding region expressed through non-canonical translation termination-reinitiation events. A class of RNA structure located upstream of the first stop codon within viral mRNAs directly binds the ribosome, preventing ribosome recycling and inducing reinitiation. We used bioinformatic methods to identify new examples of viral reinitiation-stimulating RNAs and experimentally verified their secondary structure and function. We determined the structure of a representative viral RNA-ribosome complex using cryoEM. 3D classification and variability analyses reveal that the viral RNA structure can sample a range of conformations while remaining tethered to the ribosome, which enables the ribosome to find a reinitiation start site within a limited range of mRNA sequence. Evaluating the conserved features and constraints of this entire RNA class in the context of the cryoEM reconstruction provides insight into mechanisms enabling reinitiation, a translation regulation strategy employed by many other viral and eukaryotic systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560184">
        <p class="paperTitle">Stability of Genomic Imprinting and X-Chromosome Inactivation in the Aging Brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mancino, S.; Seneviratne, J.; Mupo, A.; Krueger, F.; Oxley, D.; Eckersley-Maslin, M.; da Rocha, S. T.</p>
        <p class="info">Score: 6.2, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560184' target='https://doi.org/10.1101/2023.09.29.560184'> 10.1101/2023.09.29.560184</a></p>
        <p class="abstract">Epigenetic drift is a hallmark of aging that contributes to the irreversible decline in organismal fitness ultimately leading to aging-related diseases. Epigenetic modifications regulate the cellular memory of the epigenetic processes of genomic imprinting and X-chromosome inactivation to ensure monoallelic expression of imprinted and X-linked genes. Whether epigenetic drift affects maintenance of genomic imprinting and X-chromosome inactivation has not been comprehensively studied. Here, we investigate the allele-specific transcriptional and epigenetic signatures of the aging brain, by comparing juvenile and old hybrid mice obtained from C57BL/6J (BL6) &amp; CAST/EiJ (CAST) reciprocal crosses, with an emphasis on the hippocampus (HCP). We confirm that the aged HCP shows an increase of DNA hydroxymethylation, a sign of epigenetic drift, and a typical aging transcriptional signature. Genomic imprinting was found to be largely unaffected with stable parent-of-origin-specific DNA methylation in HCP, but also other brain regions such as the cerebellum (CB), nucleus accumbens, hypothalamus and prefrontal cortex. Consistently, transcriptomics analysis confirmed unaltered imprinting expression in the aged HCP. An exception are three novel non-coding transcripts (B230209E15Rik, Ube2nl and A330076H08Rik) at the Prader-Willi syndrome/Angelman syndrome (PWS/AS) imprinted locus which lose strict monoallelic expression during aging. Like imprinting, X-chromosome inactivation was remarkably stable with no signs of aging-driven skewing or relaxation of monoallelic expression of X-linked genes. Our study provides a valuable resource for evaluating monoallelic expression in the aging brain and reveals that, despite epigenetic drift during aging, genomic imprinting and X-chromosome inactivation remain predominantly stable throughout the process of physiological aging in the mouse brain.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
