<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells
Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.
Score: 24.6, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756
Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells
Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.
Score: 24.6, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756
Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells
Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.
Score: 24.6, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756
Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells\nAuthors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.\nScore: 24.6, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756\nDeveloping therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate.",
  "keywords": [
    
  ],
  "articleBody": " Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells\nAuthors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.\nScore: 24.6, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559756\nDeveloping therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate. We show three examples where GLS expression increases during coronavirus infection of host cells, and another in which GLS2 is upregulated. The viruses hijack the metabolic machinery responsible for glutamine metabolism to generate the building blocks for biosynthetic processes and satisfy the bioenergetic requirements demanded by the \"glutamine addiction\" of virus-infected host cells. We demonstrate how genetic silencing of glutaminase enzymes reduces coronavirus infection and that newer members of two classes of small molecule allosteric inhibitors targeting these enzymes, designated as SU1, a pan-GLS/GLS2 inhibitor, and UP4, which is specific for GLS, block viral replication in mammalian epithelial cells. Overall, these findings highlight the importance of glutamine metabolism for coronavirus replication in human cells and show that glutaminase inhibitors can block coronavirus infection and thereby may represent a novel class of anti-viral drug candidates.\n53BP1 interacts with the RNA primer from Okazaki fragments to support their processing during unperturbed DNA replication\nAuthors: Leriche, M.; Bonnet, C.; Jana, J.; Chhetri, G.; Mennour, S.; Martineau, S.; Pennaneach, V.; BUSSO, D.; Veaute, X.; Bertrand, P.; lambert, s. A.; Somyajit, K.; Uguen, P.; Vagner, S.\nScore: 14.2, Published: 2023-09-27 DOI: 10.1101/2023.09.27.559698\nRNA-binding proteins are found at replication forks, but their direct interaction with DNA-embedded RNA species that inevitably shape physiological DNA replication remains unexplored. Here we report that 53BP1, involved in the DNA damage and replication stress response, is an RNA-binding protein that directly interacts with Okazaki fragments, in the absence of any external stress. The bulk chromatin association of 53BP1 shows dramatic dependence on PRIM1, which synthesizes the RNA primer of Okazaki fragments. The direct recruitment of 53BP1 to nascent DNA shows susceptibility to in situ ribonuclease A treatment. Conversely, depletion of FEN1, which results in the accumulation of uncleaved RNA primers, leads to an upregulation of 53BP1 levels at the replication forks, suggesting that RNA primers contribute to the recruitment of 53BP1 at the lagging DNA strand. 53BP1 depletion induces an accumulation of S phase poly(ADP-ribose), which constitutes a sensor of unligated Okazaki fragments. Collectively, our data indicate that 53BP1, distinct from its canonical mode of chromatin-binding, is anchored at the replication fork through its RNA-binding activity, highlighting the role of an RNA-protein interaction at DNA replication forks.\nNucleolar dynamics are determined by the ordered assembly of the ribosome\nAuthors: Sheu-Gruttadauria, J.; Yan, X.; Stuurman, N.; Floor, S. N.; Vale, R. D.\nScore: 13.2, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559432\nRibosome biogenesis is coordinated within the nucleolus, a biomolecular condensate that exhibits dynamic material properties that are thought to be important for nucleolar function. However, the relationship between ribosome assembly and nucleolar dynamics is not clear. Here, we screened 364 genes involved in ribosome biogenesis and RNA metabolism for their impact on dynamics of the nucleolus, as measured by automated, high-throughput fluorescence recovery after photobleaching (FRAP) of the nucleolar scaffold protein NPM1. This screen revealed that gene knockdowns that caused accumulation of early rRNA intermediates were associated with nucleolar rigidification, while accumulation of late intermediates led to increased fluidity. These shifts in dynamics were accompanied by distinct changes in nucleolar morphology. We also found that genes involved in mRNA processing impact nucleolar dynamics, revealing connections between ribosome biogenesis and other RNA processing pathways. Together, this work defines mechanistic ties between ribosome assembly and the biophysical features of the nucleolus, while establishing a toolbox for understanding how molecular dynamics impact function across other biomolecular condensates.\nSpinal cord pathology in a Dravet Syndrome mouse model\nAuthors: Antinao Diaz, J. F.; Chilcott, E.; Keegan, A.; Schorge, S.; Waddington, S. N.; Karda, R.\nScore: 11.7, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558962\nSummaryO_ST_ABSObjectivesC_ST_ABSDravet syndrome is a severe epileptic encephalopathy that begins in early childhood. More than 80% of patients with Dravet syndrome exhibit a haploinsufficiency in SCN1A, which encodes the voltage-gated sodium ion channel NaV1.1. The epilepsy is believed be caused by specific deficit of SCN1A in inhibitory interneurons of the hippocampus. However, the aetiology of other symptoms including gait disturbances, ataxia, cardiac issues and dysautonomia is less clear. MethodsIn an Scn1a knock-out (Scn1a-/-) mouse model which recapitulates clinical phenotypes, we assessed NaV1.1 and neuroinflammation throughout the central nervous system. ResultsConsistent with current understanding, wild-type expression of NaV1.1 transcript and protein were absent in knock-out mice in the prefrontal cortex, striatum, hippocampus, thalamus, and cerebellum. Increased GFAP was detected in the brain only in the hippocampus. Transcript and protein were detected in wild-type cervical, thoracic and lumbar spinal cord but not in knock-out mice. Unexpectedly, GFAP was increased in all three spinal regions. Therefore, we proceeded to perform transcriptomic analysis of cortex, hippocampus and spinal cord. Pathways associated with monooxygenase activity, fatty acid ligases and lactate transporters were highly dysregulated in the spinal cord. ConclusionThe existence and relevance of pathology of the spinal cord in Dravet syndrome has received scant attention. Our findings are consistent with some systemic symptoms of Dravet syndrome, with the benefits of treatments which may modulate the astrocyte-neuron lactate shuttle such as Stiripentol and ketogenic dietary regimes, and with the efficacy of intrathecal delivery of therapeutics. Key PointsO_LIDecrease of endogenous Scn1a and NaV1.1 expression in Scn1a-/- mice has a widespread impact on the gene expression profile in the spinal cord. C_LIO_LIIncreased GFAP expression observed in the spinal cord of Scn1a-/- mice. C_LIO_LIDifferentially expressed genes related to monooxygenase activity, fatty acid ligases and lactate transporters in cervical spinal cord of Scn1a-/- mice. C_LI\nBacterial histone HBb from Bdellovibrio bacteriovorus compacts DNA by bending\nAuthors: Hu, Y.; Schwab, S.; Deiss, S.; Escudeiro, P.; Joiner, J. D.; Hartmann, M. D.; Lupas, A. N.; Alvarez, B. H.; Alva, V.; Dame, R. T.\nScore: 27.1, Published: 2023-09-27 DOI: 10.1101/2023.02.26.530074\nHistones are essential for genome compaction and transcription regulation in eukaryotes, where they assemble into octamers to form the nucleosome core. In contrast, archaeal histones assemble into dimers that form hypernucleosomes upon DNA binding. Although histone homologs have recently been identified in bacteria, their DNA-binding characteristics remain largely unexplored. Our study reveals that the bacterial histone HBb is indispensable for the survival of Bdellovibrio bacteriovorus, suggesting critical roles in DNA organization and gene regulation. We also elucidate the crystal structure of the HBb dimer at 1.06 [A] resolution and employ various biophysico-chemical approaches to show its ability to bind and compact DNA in a sequence-independent manner. This binding induces DNA bending, similar to that observed with bacterial HU/IHF family proteins. Finally, using DNA affinity purification and sequencing, we reveal that HBb binds along the entire genomic DNA of B. bacteriovorus without sequence specificity. These unique DNA-binding properties of bacterial histones, distinct from their archaeal and eukaryotic counterparts, highlight the diverse roles that histones can play in DNA organization across the domains of life. Author SummaryHistones, traditionally known for organizing and regulating DNA in eukaryotes and archaea, have recently been discovered in bacteria, opening up a new frontier in our understanding of genome organization across the domains of life. Our study investigates the largely unexplored DNA-binding properties of bacterial histones, focusing on HBb in Bdellovibrio bacteriovorus. We reveal that HBb is essential for bacterial survival and exhibits DNA-binding akin to the bacterial nucleoid-associated HU/IHF family proteins. Contrary to eukaryotic and archaeal histones that wrap DNA, HBb bends DNA without sequence specificity. This work not only broadens our understanding of DNA organization across different life forms but also suggests that bacterial histones may have diverse roles in genome organization, distinct from their eukaryotic and archaeal counterparts.\nSingle-Cell Analysis Reveals Cryptic Prophage Protease LfgB Protects Escherichia coli During Oxidative Stress by Cleaving Antitoxin MqsA\nAuthors: Fernandez-Garcia, L.; Gao, X.; Battisti, M.; Kirigo, J.; Garcia-Contreras, R.; Tomas, M.; Guo, Y.; Wang, X.; Wood, T. K.\nScore: 9.7, Published: 2023-09-23 DOI: 10.1101/2023.09.23.559088\nAlthough toxin/antitoxin (TA) systems are ubiquitous, beyond phage inhibition and mobile element stabilization, their role in host metabolism is obscure. One of the best-characterized TA systems is MqsR/MqsA of Escherichia coli, which has been linked previously to protecting this gastrointestinal species during the stress it encounters from the bile salt deoxycholate as it colonizes humans. However, some recent whole-population studies have challenged the role of toxins such as MqsR in bacterial physiology, since the mqsRA locus is induced over a hundred-fold during stress, but a phenotype was not found upon its deletion. Here, we investigate further the role MqsR/MqsA by utilizing single cells and demonstrate that upon oxidative stress, the TA system MqsR/MqsA has a heterogeneous effect on the transcriptome of single cells. Furthermore, we discovered that MqsR activation leads to induction of the poorly-characterized yfjXY ypjJ yfjZF operon of cryptic prophage CP4-57. Moreover, deletion of yfjY makes the cells sensitive to H2O2, acid, and heat stress, and this phenotype was complemented. Hence, we recommend yfjY be renamed to lfgB (less fatality gene B). Critically, MqsA represses lfgB by binding the operon promoter, and LfgB is a protease that degrades MqsA to derepress rpoS and facilitate the stress response. Therefore, the MqsR/MqsA TA system facilitates the stress response through cryptic phage protease LfgB.\nMatrix selection for the visualization of small molecules and lipids in brain tumors using untargeted MALDI-TOF mass spectrometry imaging\nAuthors: Lu, T.; Freytag, L.; Narayana, V. K.; Moore, Z.; Oliver, S. J.; Valkovic, A.; Nijagal, B.; Peterson, A.; De Souza, D. P.; McConville, M. J.; Whittle, J. R.; Best, S. A.; Freytag, S.\nScore: 8.0, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559427\nMatrix-assisted laser desorption/ionization mass spectrometry imaging allows the study of metabolic activity in the tumor microenvironment of brain cancers. The detectable metabolites within these tumors are contingent upon the choice of matrix, deposition technique, and polarity setting. In this study, we compared the performance of three different matrices, two deposition techniques, and use of positive and negative polarity in two different brain cancer types and across two species. Optimal combinations were confirmed by comparative analysis of lipid and small molecule abundance using liquid chromatography-mass spectrometry and RNA-sequencing assessing differential metabolites between normal and tumor regions. Our findings indicate that the recrystallized cyano-4-hydroxycinnamic acid matrix in positive polarity offered superior performance for both detected metabolites and consistency with other techniques. Beyond these implications for brain cancer, our work establishes a workflow to identify optimal matrices for spatial metabolomics studies.\nSpatial Transcriptomics Reveals the Requirement of ADGRG6 in Maintaining Chondrocyte Homeostasis in Mouse Growth Plates\nAuthors: Bian, F.; Hansen, V.; Feng, H. C.; Chen, Y.; Gray, R.; Wu, C.-L.; Liu, Z.\nScore: 7.7, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558739\nThe growth plate is essential for maintaining skeletal growth; however, the mechanisms governing postnatal growth plate homeostasis are poorly understood. Here we show that ADGRG6/GPR126, a cartilage-enriched G protein-coupled receptor (GPCR), is dispensable for embryonic limb development but is required for postnatal growth plate homeostasis. Adgrg6 ablation in osteochondral progenitor cells or postnatal chondrocytes leads to reduced cellularity and impaired maintenance of the resting zone in the growth plate, coupled with increased cell death and reduced cell proliferation. Adgrg6 mutant growth plates also exhibit disorganized extracellular matrix structures and dysregulated hypertrophic differentiation. Furthermore, using a novel spatial transcriptomics workflow that applies to FFPE tissue sections of mineralized mouse knee joints, we demonstrate that Adgrg6 ablation leads to reduced SOX9 expression, induced Indian hedgehog (IHH) signaling, and a precocious chondrogenic-to-osteogenic conversion of the growth plate chondrocytes that may be driven by increased POSTN/integrin receptor signaling. We further demonstrated that ADGRG6 regulates the proper formation of the resting zone growth plate by maintaining the PTHrP and SOX9-positive cell populations. Altogether, our findings elucidate the essential role of ADGRG6 in maintaining chondrocyte fate, survival, and homeostasis of the postnatal growth plates.\nMolecular principles of CRISPR-Cas13 mismatch intolerance enable selective silencing of point-mutated oncogenic RNA with single-base precision\nAuthors: Shembrey, C.; Yang, R.; Casan, J.; Hu, W.; Sadras, T.; Prasad, K.; Shortt, J.; Johnstone, R. W.; Trapani, J. A.; Ekert, P. G.; Fareh, M.\nScore: 6.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.557083\nSingle nucleotide variants (SNVs) are extremely prevalent in human cancers. For instance, KRAS mutations occur in over 90% of pancreatic cancers and [~]40% of colorectal cancers. Virtually all KRAS mutations are SNVs, most of which remain clinically unactionable. The programmable RNA nuclease CRISPR-Cas13 has been deployed to specifically target RNAs such as overexpressed oncogenes and fusion transcripts. However, silencing oncogenic SNVs with single-base precision remains extremely challenging due to the intrinsic mismatch tolerance of Cas13. Here, we developed a comprehensive mutagenesis analysis of target-spacer interactions at single-nucleotide resolution, which revealed key spacer nucleotide positions intolerant to mismatches. We show that introducing synthetic mismatches at these precise positions enables de nova design of CRISPR RNA (crRNA) with strong preferential silencing of SNV transcripts. We demonstrate that our top-performing crRNAs possess prominent SNV-selectivity with dose-dependent silencing activity against all KRAS G12 variants at both the RNA and protein levels with minimal off-target silencing of wildtype KRAS. We applied these design principles to effectively silence oncogenic NRAS G12D and BRAF V600E transcripts, underscoring the adaptability of this platform to silence various SNVs. These findings demonstrate that the CRISPR Cas13 system can be reprogrammed to target mutant transcripts with single-base precision, showcasing the tremendous potential of this tool in personalized transcriptome editing. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=176 SRC=\"FIGDIR/small/557083v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (58K): org.highwire.dtl.DTLVardef@1846402org.highwire.dtl.DTLVardef@c1fb94org.highwire.dtl.DTLVardef@16bd5d6org.highwire.dtl.DTLVardef@a40aac_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMulti-signal regulation of the GSK-3β homolog Rim11 governs meiosis entry in yeast\nAuthors: Kociemba, J.; Joergensen, A. C. S.; Tadic, N.; Harris, A.; Sideri, T.; Chan, W. Y.; Ibrahim, F.; Unal, E.; Skehel, M.; Shahrezaei, V.; Arguello-Miranda, O.; van Werven, F. J.\nScore: 5.6, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558844\nStarvation of budding yeast diploid cells induces the cell-fate program that drives meiosis and spore formation. Transcription activation of early meiotic genes (EMGs) requires the transcription activator Ime1, its DNA-binding partner Ume6, and GSK-3{beta} kinase Rim11. Phosphorylation of Ume6 by Rim11 is key for EMG activation. We report that Rim11 integrates multiple input signals to control Ume6 phosphorylation and EMG transcription. Under nutrient-rich conditions PKA represses Rim11 to low levels while TORC1 keeps Rim11 localized to the cytoplasm. Inhibiting PKA and TORC1 induces Rim11 expression and nuclear localization. Remarkably, nuclear Rim11 is required, but not sufficient, for Rim11-dependent Ume6 phosphorylation. Additionally, Ime1 is an essential anchor protein for phosphorylating Ume6. Subsequently, Ume6-Ime1 coactivator complexes form that drive EMG transcription. Our results demonstrate how varied signalling inputs (PKA/TORC1/Ime1) integrated by Rim11 determine EMG expression and entry into meiosis. We propose that the signalling-regulatory network described here generates robustness in cell-fate control.\n",
  "wordCount" : "2472",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559756">
        <p class="paperTitle">Inhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greene, K. S.; Choi, A.; Chen, M.; Yang, N.; Li, R.; Qiu, Y.; Lukey, M. J.; Rojas, K. S.; Shen, J.; Wilson, K. F.; Katt, W. P.; Whittaker, G. R.; Cerione, R. A.</p>
        <p class="info">Score: 24.6, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559756' target='https://doi.org/10.1101/2023.09.27.559756'> 10.1101/2023.09.27.559756</a></p>
        <p class="abstract">Developing therapeutic strategies against COVID-19 has gained widespread interest given the likelihood that new viral variants will continue to emerge. Here we describe one potential therapeutic strategy which involves targeting members of the glutaminase family of mitochondrial metabolic enzymes (GLS and GLS2), which catalyze the first step in glutamine metabolism, the hydrolysis of glutamine to glutamate. We show three examples where GLS expression increases during coronavirus infection of host cells, and another in which GLS2 is upregulated. The viruses hijack the metabolic machinery responsible for glutamine metabolism to generate the building blocks for biosynthetic processes and satisfy the bioenergetic requirements demanded by the &#34;glutamine addiction&#34; of virus-infected host cells. We demonstrate how genetic silencing of glutaminase enzymes reduces coronavirus infection and that newer members of two classes of small molecule allosteric inhibitors targeting these enzymes, designated as SU1, a pan-GLS/GLS2 inhibitor, and UP4, which is specific for GLS, block viral replication in mammalian epithelial cells. Overall, these findings highlight the importance of glutamine metabolism for coronavirus replication in human cells and show that glutaminase inhibitors can block coronavirus infection and thereby may represent a novel class of anti-viral drug candidates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559698">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559698" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559698">
        <p class="paperTitle">53BP1 interacts with the RNA primer from Okazaki fragments to support their processing during unperturbed DNA replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559698" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559698" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leriche, M.; Bonnet, C.; Jana, J.; Chhetri, G.; Mennour, S.; Martineau, S.; Pennaneach, V.; BUSSO, D.; Veaute, X.; Bertrand, P.; lambert, s. A.; Somyajit, K.; Uguen, P.; Vagner, S.</p>
        <p class="info">Score: 14.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559698' target='https://doi.org/10.1101/2023.09.27.559698'> 10.1101/2023.09.27.559698</a></p>
        <p class="abstract">RNA-binding proteins are found at replication forks, but their direct interaction with DNA-embedded RNA species that inevitably shape physiological DNA replication remains unexplored. Here we report that 53BP1, involved in the DNA damage and replication stress response, is an RNA-binding protein that directly interacts with Okazaki fragments, in the absence of any external stress. The bulk chromatin association of 53BP1 shows dramatic dependence on PRIM1, which synthesizes the RNA primer of Okazaki fragments. The direct recruitment of 53BP1 to nascent DNA shows susceptibility to in situ ribonuclease A treatment. Conversely, depletion of FEN1, which results in the accumulation of uncleaved RNA primers, leads to an upregulation of 53BP1 levels at the replication forks, suggesting that RNA primers contribute to the recruitment of 53BP1 at the lagging DNA strand. 53BP1 depletion induces an accumulation of S phase poly(ADP-ribose), which constitutes a sensor of unligated Okazaki fragments. Collectively, our data indicate that 53BP1, distinct from its canonical mode of chromatin-binding, is anchored at the replication fork through its RNA-binding activity, highlighting the role of an RNA-protein interaction at DNA replication forks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559432">
        <p class="paperTitle">Nucleolar dynamics are determined by the ordered assembly of the ribosome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheu-Gruttadauria, J.; Yan, X.; Stuurman, N.; Floor, S. N.; Vale, R. D.</p>
        <p class="info">Score: 13.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559432' target='https://doi.org/10.1101/2023.09.26.559432'> 10.1101/2023.09.26.559432</a></p>
        <p class="abstract">Ribosome biogenesis is coordinated within the nucleolus, a biomolecular condensate that exhibits dynamic material properties that are thought to be important for nucleolar function. However, the relationship between ribosome assembly and nucleolar dynamics is not clear. Here, we screened 364 genes involved in ribosome biogenesis and RNA metabolism for their impact on dynamics of the nucleolus, as measured by automated, high-throughput fluorescence recovery after photobleaching (FRAP) of the nucleolar scaffold protein NPM1. This screen revealed that gene knockdowns that caused accumulation of early rRNA intermediates were associated with nucleolar rigidification, while accumulation of late intermediates led to increased fluidity. These shifts in dynamics were accompanied by distinct changes in nucleolar morphology. We also found that genes involved in mRNA processing impact nucleolar dynamics, revealing connections between ribosome biogenesis and other RNA processing pathways. Together, this work defines mechanistic ties between ribosome assembly and the biophysical features of the nucleolus, while establishing a toolbox for understanding how molecular dynamics impact function across other biomolecular condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558962">
        <p class="paperTitle">Spinal cord pathology in a Dravet Syndrome mouse model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antinao Diaz, J. F.; Chilcott, E.; Keegan, A.; Schorge, S.; Waddington, S. N.; Karda, R.</p>
        <p class="info">Score: 11.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558962' target='https://doi.org/10.1101/2023.09.22.558962'> 10.1101/2023.09.22.558962</a></p>
        <p class="abstract">SummaryO_ST_ABSObjectivesC_ST_ABSDravet syndrome is a severe epileptic encephalopathy that begins in early childhood. More than 80% of patients with Dravet syndrome exhibit a haploinsufficiency in SCN1A, which encodes the voltage-gated sodium ion channel NaV1.1. The epilepsy is believed be caused by specific deficit of SCN1A in inhibitory interneurons of the hippocampus. However, the aetiology of other symptoms including gait disturbances, ataxia, cardiac issues and dysautonomia is less clear.

MethodsIn an Scn1a knock-out (Scn1a-/-) mouse model which recapitulates clinical phenotypes, we assessed NaV1.1 and neuroinflammation throughout the central nervous system.

ResultsConsistent with current understanding, wild-type expression of NaV1.1 transcript and protein were absent in knock-out mice in the prefrontal cortex, striatum, hippocampus, thalamus, and cerebellum. Increased GFAP was detected in the brain only in the hippocampus. Transcript and protein were detected in wild-type cervical, thoracic and lumbar spinal cord but not in knock-out mice. Unexpectedly, GFAP was increased in all three spinal regions. Therefore, we proceeded to perform transcriptomic analysis of cortex, hippocampus and spinal cord. Pathways associated with monooxygenase activity, fatty acid ligases and lactate transporters were highly dysregulated in the spinal cord.

ConclusionThe existence and relevance of pathology of the spinal cord in Dravet syndrome has received scant attention. Our findings are consistent with some systemic symptoms of Dravet syndrome, with the benefits of treatments which may modulate the astrocyte-neuron lactate shuttle such as Stiripentol and ketogenic dietary regimes, and with the efficacy of intrathecal delivery of therapeutics.

Key PointsO_LIDecrease of endogenous Scn1a and NaV1.1 expression in Scn1a-/- mice has a widespread impact on the gene expression profile in the spinal cord.
C_LIO_LIIncreased GFAP expression observed in the spinal cord of Scn1a-/- mice.
C_LIO_LIDifferentially expressed genes related to monooxygenase activity, fatty acid ligases and lactate transporters in cervical spinal cord of Scn1a-/- mice.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.26.530074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.26.530074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.26.530074">
        <p class="paperTitle">Bacterial histone HBb from Bdellovibrio bacteriovorus compacts DNA by bending</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.26.530074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.26.530074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Y.; Schwab, S.; Deiss, S.; Escudeiro, P.; Joiner, J. D.; Hartmann, M. D.; Lupas, A. N.; Alvarez, B. H.; Alva, V.; Dame, R. T.</p>
        <p class="info">Score: 27.1, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.26.530074' target='https://doi.org/10.1101/2023.02.26.530074'> 10.1101/2023.02.26.530074</a></p>
        <p class="abstract">Histones are essential for genome compaction and transcription regulation in eukaryotes, where they assemble into octamers to form the nucleosome core. In contrast, archaeal histones assemble into dimers that form hypernucleosomes upon DNA binding. Although histone homologs have recently been identified in bacteria, their DNA-binding characteristics remain largely unexplored. Our study reveals that the bacterial histone HBb is indispensable for the survival of Bdellovibrio bacteriovorus, suggesting critical roles in DNA organization and gene regulation. We also elucidate the crystal structure of the HBb dimer at 1.06 [A] resolution and employ various biophysico-chemical approaches to show its ability to bind and compact DNA in a sequence-independent manner. This binding induces DNA bending, similar to that observed with bacterial HU/IHF family proteins. Finally, using DNA affinity purification and sequencing, we reveal that HBb binds along the entire genomic DNA of B. bacteriovorus without sequence specificity. These unique DNA-binding properties of bacterial histones, distinct from their archaeal and eukaryotic counterparts, highlight the diverse roles that histones can play in DNA organization across the domains of life.

Author SummaryHistones, traditionally known for organizing and regulating DNA in eukaryotes and archaea, have recently been discovered in bacteria, opening up a new frontier in our understanding of genome organization across the domains of life. Our study investigates the largely unexplored DNA-binding properties of bacterial histones, focusing on HBb in Bdellovibrio bacteriovorus. We reveal that HBb is essential for bacterial survival and exhibits DNA-binding akin to the bacterial nucleoid-associated HU/IHF family proteins. Contrary to eukaryotic and archaeal histones that wrap DNA, HBb bends DNA without sequence specificity. This work not only broadens our understanding of DNA organization across different life forms but also suggests that bacterial histones may have diverse roles in genome organization, distinct from their eukaryotic and archaeal counterparts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559088">
        <p class="paperTitle">Single-Cell Analysis Reveals Cryptic Prophage Protease LfgB Protects Escherichia coli During Oxidative Stress by Cleaving Antitoxin MqsA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Garcia, L.; Gao, X.; Battisti, M.; Kirigo, J.; Garcia-Contreras, R.; Tomas, M.; Guo, Y.; Wang, X.; Wood, T. K.</p>
        <p class="info">Score: 9.7, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559088' target='https://doi.org/10.1101/2023.09.23.559088'> 10.1101/2023.09.23.559088</a></p>
        <p class="abstract">Although toxin/antitoxin (TA) systems are ubiquitous, beyond phage inhibition and mobile element stabilization, their role in host metabolism is obscure. One of the best-characterized TA systems is MqsR/MqsA of Escherichia coli, which has been linked previously to protecting this gastrointestinal species during the stress it encounters from the bile salt deoxycholate as it colonizes humans. However, some recent whole-population studies have challenged the role of toxins such as MqsR in bacterial physiology, since the mqsRA locus is induced over a hundred-fold during stress, but a phenotype was not found upon its deletion. Here, we investigate further the role MqsR/MqsA by utilizing single cells and demonstrate that upon oxidative stress, the TA system MqsR/MqsA has a heterogeneous effect on the transcriptome of single cells. Furthermore, we discovered that MqsR activation leads to induction of the poorly-characterized yfjXY ypjJ yfjZF operon of cryptic prophage CP4-57. Moreover, deletion of yfjY makes the cells sensitive to H2O2, acid, and heat stress, and this phenotype was complemented. Hence, we recommend yfjY be renamed to lfgB (less fatality gene B). Critically, MqsA represses lfgB by binding the operon promoter, and LfgB is a protease that degrades MqsA to derepress rpoS and facilitate the stress response. Therefore, the MqsR/MqsA TA system facilitates the stress response through cryptic phage protease LfgB.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559427">
        <p class="paperTitle">Matrix selection for the visualization of small molecules and lipids in brain tumors using untargeted MALDI-TOF mass spectrometry imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, T.; Freytag, L.; Narayana, V. K.; Moore, Z.; Oliver, S. J.; Valkovic, A.; Nijagal, B.; Peterson, A.; De Souza, D. P.; McConville, M. J.; Whittle, J. R.; Best, S. A.; Freytag, S.</p>
        <p class="info">Score: 8.0, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559427' target='https://doi.org/10.1101/2023.09.25.559427'> 10.1101/2023.09.25.559427</a></p>
        <p class="abstract">Matrix-assisted laser desorption/ionization mass spectrometry imaging allows the study of metabolic activity in the tumor microenvironment of brain cancers. The detectable metabolites within these tumors are contingent upon the choice of matrix, deposition technique, and polarity setting. In this study, we compared the performance of three different matrices, two deposition techniques, and use of positive and negative polarity in two different brain cancer types and across two species. Optimal combinations were confirmed by comparative analysis of lipid and small molecule abundance using liquid chromatography-mass spectrometry and RNA-sequencing assessing differential metabolites between normal and tumor regions. Our findings indicate that the recrystallized  cyano-4-hydroxycinnamic acid matrix in positive polarity offered superior performance for both detected metabolites and consistency with other techniques. Beyond these implications for brain cancer, our work establishes a workflow to identify optimal matrices for spatial metabolomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558739">
        <p class="paperTitle">Spatial Transcriptomics Reveals the Requirement of ADGRG6 in Maintaining Chondrocyte Homeostasis in Mouse Growth Plates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bian, F.; Hansen, V.; Feng, H. C.; Chen, Y.; Gray, R.; Wu, C.-L.; Liu, Z.</p>
        <p class="info">Score: 7.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558739' target='https://doi.org/10.1101/2023.09.21.558739'> 10.1101/2023.09.21.558739</a></p>
        <p class="abstract">The growth plate is essential for maintaining skeletal growth; however, the mechanisms governing postnatal growth plate homeostasis are poorly understood. Here we show that ADGRG6/GPR126, a cartilage-enriched G protein-coupled receptor (GPCR), is dispensable for embryonic limb development but is required for postnatal growth plate homeostasis. Adgrg6 ablation in osteochondral progenitor cells or postnatal chondrocytes leads to reduced cellularity and impaired maintenance of the resting zone in the growth plate, coupled with increased cell death and reduced cell proliferation. Adgrg6 mutant growth plates also exhibit disorganized extracellular matrix structures and dysregulated hypertrophic differentiation. Furthermore, using a novel spatial transcriptomics workflow that applies to FFPE tissue sections of mineralized mouse knee joints, we demonstrate that Adgrg6 ablation leads to reduced SOX9 expression, induced Indian hedgehog (IHH) signaling, and a precocious chondrogenic-to-osteogenic conversion of the growth plate chondrocytes that may be driven by increased POSTN/integrin receptor signaling. We further demonstrated that ADGRG6 regulates the proper formation of the resting zone growth plate by maintaining the PTHrP and SOX9-positive cell populations. Altogether, our findings elucidate the essential role of ADGRG6 in maintaining chondrocyte fate, survival, and homeostasis of the postnatal growth plates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.557083">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.557083" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.557083">
        <p class="paperTitle">Molecular principles of CRISPR-Cas13 mismatch intolerance enable selective silencing of point-mutated oncogenic RNA with single-base precision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.557083" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.557083" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shembrey, C.; Yang, R.; Casan, J.; Hu, W.; Sadras, T.; Prasad, K.; Shortt, J.; Johnstone, R. W.; Trapani, J. A.; Ekert, P. G.; Fareh, M.</p>
        <p class="info">Score: 6.7, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.557083' target='https://doi.org/10.1101/2023.09.26.557083'> 10.1101/2023.09.26.557083</a></p>
        <p class="abstract">Single nucleotide variants (SNVs) are extremely prevalent in human cancers. For instance, KRAS mutations occur in over 90% of pancreatic cancers and [~]40% of colorectal cancers. Virtually all KRAS mutations are SNVs, most of which remain clinically unactionable. The programmable RNA nuclease CRISPR-Cas13 has been deployed to specifically target RNAs such as overexpressed oncogenes and fusion transcripts. However, silencing oncogenic SNVs with single-base precision remains extremely challenging due to the intrinsic mismatch tolerance of Cas13. Here, we developed a comprehensive mutagenesis analysis of target-spacer interactions at single-nucleotide resolution, which revealed key spacer nucleotide positions intolerant to mismatches. We show that introducing synthetic mismatches at these precise positions enables de nova design of CRISPR RNA (crRNA) with strong preferential silencing of SNV transcripts. We demonstrate that our top-performing crRNAs possess prominent SNV-selectivity with dose-dependent silencing activity against all KRAS G12 variants at both the RNA and protein levels with minimal off-target silencing of wildtype KRAS. We applied these design principles to effectively silence oncogenic NRAS G12D and BRAF V600E transcripts, underscoring the adaptability of this platform to silence various SNVs. These findings demonstrate that the CRISPR Cas13 system can be reprogrammed to target mutant transcripts with single-base precision, showcasing the tremendous potential of this tool in personalized transcriptome editing.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=176 SRC=&#34;FIGDIR/small/557083v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (58K):
org.highwire.dtl.DTLVardef@1846402org.highwire.dtl.DTLVardef@c1fb94org.highwire.dtl.DTLVardef@16bd5d6org.highwire.dtl.DTLVardef@a40aac_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558844">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558844" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558844">
        <p class="paperTitle">Multi-signal regulation of the GSK-3β homolog Rim11 governs meiosis entry in yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558844" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558844" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kociemba, J.; Joergensen, A. C. S.; Tadic, N.; Harris, A.; Sideri, T.; Chan, W. Y.; Ibrahim, F.; Unal, E.; Skehel, M.; Shahrezaei, V.; Arguello-Miranda, O.; van Werven, F. J.</p>
        <p class="info">Score: 5.6, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558844' target='https://doi.org/10.1101/2023.09.21.558844'> 10.1101/2023.09.21.558844</a></p>
        <p class="abstract">Starvation of budding yeast diploid cells induces the cell-fate program that drives meiosis and spore formation. Transcription activation of early meiotic genes (EMGs) requires the transcription activator Ime1, its DNA-binding partner Ume6, and GSK-3{beta} kinase Rim11. Phosphorylation of Ume6 by Rim11 is key for EMG activation. We report that Rim11 integrates multiple input signals to control Ume6 phosphorylation and EMG transcription. Under nutrient-rich conditions PKA represses Rim11 to low levels while TORC1 keeps Rim11 localized to the cytoplasm. Inhibiting PKA and TORC1 induces Rim11 expression and nuclear localization. Remarkably, nuclear Rim11 is required, but not sufficient, for Rim11-dependent Ume6 phosphorylation. Additionally, Ime1 is an essential anchor protein for phosphorylating Ume6. Subsequently, Ume6-Ime1 coactivator complexes form that drive EMG transcription. Our results demonstrate how varied signalling inputs (PKA/TORC1/Ime1) integrated by Rim11 determine EMG expression and entry into meiosis. We propose that the signalling-regulatory network described here generates robustness in cell-fate control.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
