<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="De novo basecalling of m6A modifications at single molecule and single nucleotide resolution
Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.
Score: 47.5, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801
RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="De novo basecalling of m6A modifications at single molecule and single nucleotide resolution
Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.
Score: 47.5, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801
RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="De novo basecalling of m6A modifications at single molecule and single nucleotide resolution
Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.
Score: 47.5, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801
RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "De novo basecalling of m6A modifications at single molecule and single nucleotide resolution\nAuthors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.\nScore: 47.5, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801\nRNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules.",
  "keywords": [
    
  ],
  "articleBody": " De novo basecalling of m6A modifications at single molecule and single nucleotide resolution\nAuthors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.\nScore: 47.5, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801\nRNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules. Herein, we introduce m6ABasecaller, an innovative method enabling direct basecalling of m6A modifications from raw nanopore signals within individual RNA molecules. This approach facilitates de novo prediction of m6A modifications with precision down to the single nucleotide and single molecule levels, without the need of paired knockout or control conditions. Using the m6ABasecaller, we find that the median transcriptome-wide m6A modification stoichiometry is [~]10-15% in human, mouse and zebrafish. Furthermore, we show that m6A modifications affect polyA tail lengths, exhibit a propensity for co-occurrence within the same RNA molecules, and show relatively consistent stoichiometry levels across isoforms. We further validate the m6ABasecaller by treating mESC with increasing concentrations of STM2457, a METTL3 inhibitor as well as in inducible METTL3 knockout systems. Overall, this work demonstrates the feasibility de novo basecalling of m6A modifications, opening novel avenues for the application of nanopore sequencing to samples with limited RNA availability and for which control knockout conditions are unavailable, such as patient-derived samples.\nCreation of de novo cryptic splicing for ALS/FTD precision medicine\nAuthors: Wilkins, O. G.; Chien, M. Z. Y. J.; Wlaschin, J. J.; Pisliakova, M.; Thompson, D.; Digby, H.; Simkin, R. L.; Diaz, J. A.; Mehta, P. R.; Keuss, M. J.; Zanovello, M.; Brown, A.-L.; Harley, P.; Darbey, A.; Karda, R.; Fisher, E. M. C.; Cunningham, T. J.; Le Pichon, C. E.; Ule, J.; Fratta, P.\nScore: 46.2, Published: 2023-11-15 DOI: 10.1101/2023.11.15.565967\nA system enabling the expression of therapeutic proteins specifically in diseased cells would be transformative, providing greatly increased safety and the possibility of pre-emptive treatment. Here we describe \"TDP-REG\", a precision medicine approach primarily for amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), which exploits the cryptic splicing events that occur in cells with TDP-43 loss-of-function (TDP-LOF) in order to drive expression specifically in diseased cells. In addition to modifying existing cryptic exons for this purpose, we develop a deep-learning-powered algorithm for generating customisable cryptic splicing events, which can be embedded within virtually any coding sequence. By placing part of a coding sequence within a novel cryptic exon, we tightly couple protein expression to TDP-LOF. Protein expression is activated by TDP-LOF in vitro and in vivo, including TDP-LOF induced by cytoplasmic TDP-43 aggregation. In addition to generating a variety of fluorescent and luminescent reporters, we use this system to perform TDP-LOF-dependent genomic prime editing to ablate the UNC13A cryptic donor splice site. Furthermore, we design a panel of tightly gated, autoregulating vectors encoding a TDP-43/Raver1 fusion protein, which rescue key pathological cryptic splicing events. In summary, we combine deep-learning and rational design to create sophisticated splicing sensors, resulting in a platform that provides far safer therapeutics for neurodegeneration, potentially even enabling preemptive treatment of at-risk individuals. One-Sentence SummaryWe engineer TDP-43-regulated cryptic exons, enabling exceptionally precise activation of gene therapies in diseased neurons.\nThe embryonic DNA methylation program modulates the cis-regulatory landscape via CTCF antagonism\nAuthors: Monteagudo-Sanchez, A.; Richard Albert, J.; Scarpa, M.; Noordermeer, D.; Greenberg, M. V. C.\nScore: 33.8, Published: 2023-11-16 DOI: 10.1101/2023.11.16.567349\nDuring mammalian embryogenesis, both the 5-cytosine DNA methylation (5meC) landscape and three-dimensional (3D) chromatin architecture are profoundly remodeled during a process known as \"epigenetic reprogramming.\" An understudied aspect of epigenetic reprogramming is how the 5meC flux, per se, affects the 3D genome. This is pertinent given the 5meC- sensitivity of DNA binding for a key regulator of chromosome folding: CTCF. We profiled the CTCF binding landscape using a mouse embryonic stem cell (ESC) differentiation protocol that models the exit of naive pluripotency, wherein global DNA methylation levels start low and increase to somatic levels within four days. We took advantage of the fact that mouse ESCs lacking DNA methylation machinery exhibit globally similar differentiation dynamics, thus allowing for dissection of more subtle effects of CTCF misregulation on gene expression. We carried this out by performing CTCF HiChIP in both wild-type and mutant conditions to assess aberrant CTCF-CTCF contacts in the absence of 5meC. We went on to assess the impact that misregulated CTCF binding has on cis-regulatory contacts by performing H3K27ac HiChIP, given that H3K27ac is enriched on active promoters and enhancers. Using DNA methylation epigenome editing, we were able to directly demonstrate that the DNA methyl- mark is able to impact CTCF binding. Finally, a detailed dissection of the imprinted Zdbf2 gene showed how 5meC-antagonism of CTCF allows for proper gene regulation during differentiation. This work provides a comprehensive overview of how DNA methylation impacts the 3D genome in a relevant model for early embryonic events.\nReversible histone deacetylase activity catalyzes lysine acylation\nAuthors: Tsusaka, T.; Najar, M. A.; Schwarz, B.; Bohrnsen, E.; Oses-Prieto, J. A.; Lee, C.; Burlingame, A. L.; Bosio, C. M.; Burslem, G. M.; Goldberg, E. L.\nScore: 20.4, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567549\nStarvation and low carbohydrate diets lead to the accumulation of the ketone body, {beta}-hydroxybutyrate (BHB), whose blood concentrations increase more than 10-fold into the millimolar range. In addition to providing a carbon source, BHB accumulation triggers lysine {beta}-hydroxybutyrylation (Kbhb) of proteins via unknown mechanisms. As with other lysine acylation events, Kbhb marks can be removed by histone deacetylases (HDACs). Here, we report that class I HDACs unexpectedly catalyze protein lysine modification with {beta}-hydroxybutyrate (BHB). Mutational analyses of the HDAC2 active site reveal a shared reliance on key amino acids for classical deacetylation and non-canonical HDAC-catalyzed {beta}-hydroxybutyrylation. Also consistent with reverse HDAC activity, Kbhb formation is driven by mass action and substrate availability. This reverse HDAC activity is not limited to BHB but also extends to multiple short-chain fatty acids. The reversible activity of class I HDACs described here represents a novel mechanism of PTM deposition relevant to metabolically-sensitive proteome modifications.\nA synthetic CRISPR-Cas nuclease with expanded enzymatic activities\nAuthors: Jabalera, Y.; Tascon, I.; Samperio, S.; Lopez-Alonso, J. P.; Gonzalez-Lopez, M.; Aransay, A. M.; Abascal-Palacios, G.; Ubarretxena-Belandia, I.; Perez-Jimenez, R.\nScore: 20.1, Published: 2023-11-12 DOI: 10.1101/2023.11.12.566739\nClustered regularly interspaced short palindromic repeats (CRISPR)-associated endonucleases have revolutionized biotechnology for their potential as programmable genome editors. Yet, most natural nucleases and their variants have limitations. Here, we report a fully synthetic CRISPR-associated (Cas) nuclease (-synCas) designed by Ancestral Sequence Reconstruction (ASR) that displays a set of robust and distinct targeting properties, not found in any other known CRISPR-Cas Class 2 system. We show that -synCas is a PAMless nuclease able to catalyse RNA-guided, specific cleavage of dsDNA, ssDNA and ssRNA. The synthetic enzyme is also capable of sequence-nonspecific degradation of dsDNA, ssDNA and ssRNA following activation by complementary dsDNA, ssDNA and ssRNA targets. Furthermore, -synCas exhibits a robust genome editing activity in human cells and bacteria. Cryo-electron microscopy structures of -synCas ternary and quaternary complexes provide a framework to understand the structural basis for its expanded enzymatic activities. The capability for programmable multimodal targeting of virtually any nucleic acid sequence distinguishes -synCas as a promising new tool to extend current CRISPR-based technologies.\nSerotonin transporter-dependent histone serotonylation in placenta contributes to the neurodevelopmental transcriptome\nAuthors: Chan, J. C.; Alenina, N.; Cunningham, A. M.; Ramakrishnan, A.; Shen, L.; Bader, M.; Maze, I.\nScore: 12.3, Published: 2023-11-14 DOI: 10.1101/2023.11.14.567020\nBrain development requires appropriate regulation of serotonin (5-HT) signaling from distinct tissue sources across embryogenesis. At the maternal-fetal interface, the placenta is thought to be an important contributor of offspring brain 5-HT and is critical to overall fetal health. Yet, how placental 5-HT is acquired, and the mechanisms through which 5-HT influences placental functions, are not well understood. Recently, our group identified a novel epigenetic role for 5-HT, in which 5-HT can be added to histone proteins to regulate transcription, a process called H3 serotonylation. Here, we show that H3 serotonylation undergoes dynamic regulation during placental development, corresponding to gene expression changes that are known to influence key metabolic processes. Using transgenic mice, we demonstrate that placental H3 serotonylation largely depends on 5-HT uptake by the serotonin transporter (SERT/SLC6A4). SERT deletion robustly reduces enrichment of H3 serotonylation across the placental genome, and disrupts neurodevelopmental gene networks in early embryonic brain tissues. Thus, these findings suggest a novel role for H3 serotonylation in coordinating placental transcription at the intersection of maternal physiology and offspring brain development.\nOmecamtiv mecarbil and Mavacamten target the same myosin pocket despite antagonistic effects in heart contraction\nAuthors: Auguin, D.; Robert-Paganin, J.; Rety, S.; Kikuti, C.; David, A.; Theumer, G.; Schmidt, A. W.; Knoelker, H.-J.; Houdusse, A.\nScore: 9.7, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567213\nInherited cardiomyopathies are amongst the most common cardiac diseases worldwide, leading in the late-stage to heart failure and death. The most promising treatments against these diseases are small-molecules directly modulating the force produced by Beta-cardiac myosin, the molecular motor driving heart contraction. Two of these molecules that produce antagonistic effects on cardiac contractility have completed clinical phase 3 trials: the activator Omecamtiv mecarbil and the inhibitor Mavacamten. In this work, we reveal by X-ray crystallography that both drugs target the same pocket and stabilize a pre-stroke structural state, with only few local differences. All atoms molecular dynamics simulations reveal how these molecules can have antagonistic impact on the allostery of the motor by comparing Beta-cardiac myosin in the apo form or bound to Omecamtiv mecarbil or Mavacamten. Altogether, our results provide the framework for rational drug development for the purpose of personalized medicine.\nCircadian Control of Histone Turnover During Cardiac Development and Growth\nAuthors: Arrieta, A.; Chapski, D. J.; Reese, A.; Kimball, T. H.; Song, K.; Rosa-Garrido, M.; Vondriska, T. M.\nScore: 8.8, Published: 2023-11-14 DOI: 10.1101/2023.11.14.567086\nRationale: During postnatal cardiac hypertrophy, cardiomyocytes undergo mitotic exit, relying on DNA replication-independent mechanisms of histone turnover to maintain chromatin organization and gene transcription. In other tissues, circadian oscillations in nucleosome occupancy influence clock-controlled gene expression, suggesting an unrecognized role for the circadian clock in temporal control of histone turnover and coordinate cardiomyocyte gene expression. Objective: To elucidate roles for the master circadian transcription factor, Bmal1, in histone turnover, chromatin organization, and myocyte-specific gene expression and cell growth in the neonatal period. Methods and Results: Bmal1 knockdown in neonatal rat ventricular myocytes (NRVM) decreased myocyte size, total cellular protein, and transcription of the fetal hypertrophic gene Nppb following treatment with increasing serum concentrations or the -adrenergic agonist phenylephrine (PE). Bmal1 knockdown decreased expression of clock-controlled genes Per2 and Tcap, and salt-inducible kinase 1 (Sik1) which was identified via gene ontology analysis of Bmal1 targets upregulated in adult versus embryonic hearts. Epigenomic analyses revealed co-localized chromatin accessibility and Bmal1 localization in the Sik1 promoter. Bmal1 knockdown impaired Per2 and Sik1 promoter accessibility as measured by MNase-qPCR and impaired histone turnover indicated by metabolic labeling of acid-soluble chromatin fractions and immunoblots of total and chromatin-associated core histones. Sik1 knockdown basally increased myocyte size, while simultaneously impairing and driving Nppb and Per2 transcription, respectively. Conclusions: Bmal1 is required for neonatal myocyte growth, replication-independent histone turnover, and chromatin organization at the Sik1 promoter. Sik1 represents a novel clock-controlled gene that coordinates myocyte growth with hypertrophic and clock-controlled gene transcription.\nSterility-Independent Enhancement of Proteasome Function via Floxuridine-Triggered Detoxification in C. elegans\nAuthors: Dubey, A. A.; Szulc, N. A.; Piechota, M.; Serwa, R. A.; Pokrzywa, W.\nScore: 7.4, Published: 2023-11-12 DOI: 10.1101/2023.11.11.566706\nThe ubiquitin-proteasome system (UPS) functionality is vital for proteostasis, contributing to stress resilience, lifespan, and thermal adaptability. In Caenorhabditis elegans, proteasome constituents such as the RPN-6 and PBS-6 subunits or the PSME-3 activator are respectively linked to heat resistance, longevity at moderate cold (15{degrees}C), and survival at low temperatures (4{degrees}C). Since the inhibition of germline stem cells proliferation is associated with robust proteostasis in worms, we utilized floxuridine (FUdR), a compound known for inducing sterility, to examine whether it could reinforce UPS under proteasome dysfunction, particularly to foster cold survival. We demonstrate that FUdR promotes proteasome resilience during its inhibition or subunit deficits, supporting normal lifespan and facilitating adaptation to cold. FUdRs elevation of the UPS activity occurs independently of main proteostasis regulators and is partly driven by SKN-1-regulated transcription, especially under reduced proteasome function. Additionally, we uncover a FUdR-stimulated detoxification pathway, distinct from both SKN-1 and the germline, with GST-24 emerging as a critical mediator of the UPS buffering. This research underscores FUdRs role in the UPS modulation and its contribution to survival of worms in low-temperature conditions. HIGHLIGHTSO_LIFloxuridine (FUdR) enhances ubiquitin-proteasome system activity in C. elegans, independent of primary proteostasis regulators. C_LIO_LIFUdR permits worms to maintain a normal lifespan and facilitates their adaptation to cold in the context of proteasome deficits. C_LIO_LIActing independently of the germline and SKN-1, FUdR triggers a detoxification pathway, with GST-24 as a pivotal component in modulating the ubiquitin-proteasome system. C_LI\nN-terminal toxin signal peptides efficiently load therapeutics into a natural nano-injection system\nAuthors: Steiner-Rebrova, E. M.; Ejaz, R. N.; Kielkopf, C. S.; Perez Ruiz, M.; Marin-Arraiza, L.; Hendriks, I. A.; Nissen, J. N.; Pozdnyakova, I.; Pape, T.; Regaiolo, A.; Goetz, K.; Heermann, R.; Rasmussen, S.; Nielsen, M. L.; Taylor, N. M. I.\nScore: 6.6, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566157\nTargeted delivery of therapeutics to specific cells is a major bottleneck towards personalized medicine. The extracellular injection system (eCIS) of Serratia entomophila, the antifeeding prophage (Afp), promises potential for drug delivery purposes. However, the precise mechanism of action, toxin location, and Afp loading remain unclear. Here, we reveal a minimal N-terminal signal peptide (NtSP) of the toxin Afp18, that plays a key role in toxin packing. By engineering fusion proteins, we demonstrate that Afp18s NtSP can shuttle effectors for Afp loading. We packed non-eCIS effectors, including CRISPR-Cas protein Cas{Phi}-2 from Biggiephage, and a human antimicrobial peptide, LL37, into Afp. Additionally, NtSPs from eCIS effectors of other species facilitate loading of Cas{Phi}-2 into Afp. We observed cargo being packed inside the Afp tail tube through cryo-EM single particle analysis. The presented results enhance our understanding of eCIS toxin packing and contribute to their development as targeted delivery systems. TeaserA novel use of the Afp nano injection systems N-terminal signal peptide in targeted therapeutics delivery\n",
  "wordCount" : "2370",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566801" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566801">
        <p class="paperTitle">De novo basecalling of m6A modifications at single molecule and single nucleotide resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.</p>
        <p class="info">Score: 47.5, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566801' target='https://doi.org/10.1101/2023.11.13.566801'> 10.1101/2023.11.13.566801</a></p>
        <p class="abstract">RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules. Herein, we introduce m6ABasecaller, an innovative method enabling direct basecalling of m6A modifications from raw nanopore signals within individual RNA molecules. This approach facilitates de novo prediction of m6A modifications with precision down to the single nucleotide and single molecule levels, without the need of paired knockout or control conditions. Using the m6ABasecaller, we find that the median transcriptome-wide m6A modification stoichiometry is [~]10-15% in human, mouse and zebrafish. Furthermore, we show that m6A modifications affect polyA tail lengths, exhibit a propensity for co-occurrence within the same RNA molecules, and show relatively consistent stoichiometry levels across isoforms. We further validate the m6ABasecaller by treating mESC with increasing concentrations of STM2457, a METTL3 inhibitor as well as in inducible METTL3 knockout systems. Overall, this work demonstrates the feasibility de novo basecalling of m6A modifications, opening novel avenues for the application of nanopore sequencing to samples with limited RNA availability and for which control knockout conditions are unavailable, such as patient-derived samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.565967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.565967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.565967">
        <p class="paperTitle">Creation of de novo cryptic splicing for ALS/FTD precision medicine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.565967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.565967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wilkins, O. G.; Chien, M. Z. Y. J.; Wlaschin, J. J.; Pisliakova, M.; Thompson, D.; Digby, H.; Simkin, R. L.; Diaz, J. A.; Mehta, P. R.; Keuss, M. J.; Zanovello, M.; Brown, A.-L.; Harley, P.; Darbey, A.; Karda, R.; Fisher, E. M. C.; Cunningham, T. J.; Le Pichon, C. E.; Ule, J.; Fratta, P.</p>
        <p class="info">Score: 46.2, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.565967' target='https://doi.org/10.1101/2023.11.15.565967'> 10.1101/2023.11.15.565967</a></p>
        <p class="abstract">A system enabling the expression of therapeutic proteins specifically in diseased cells would be transformative, providing greatly increased safety and the possibility of pre-emptive treatment. Here we describe &#34;TDP-REG&#34;, a precision medicine approach primarily for amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), which exploits the cryptic splicing events that occur in cells with TDP-43 loss-of-function (TDP-LOF) in order to drive expression specifically in diseased cells. In addition to modifying existing cryptic exons for this purpose, we develop a deep-learning-powered algorithm for generating customisable cryptic splicing events, which can be embedded within virtually any coding sequence. By placing part of a coding sequence within a novel cryptic exon, we tightly couple protein expression to TDP-LOF. Protein expression is activated by TDP-LOF in vitro and in vivo, including TDP-LOF induced by cytoplasmic TDP-43 aggregation. In addition to generating a variety of fluorescent and luminescent reporters, we use this system to perform TDP-LOF-dependent genomic prime editing to ablate the UNC13A cryptic donor splice site. Furthermore, we design a panel of tightly gated, autoregulating vectors encoding a TDP-43/Raver1 fusion protein, which rescue key pathological cryptic splicing events. In summary, we combine deep-learning and rational design to create sophisticated splicing sensors, resulting in a platform that provides far safer therapeutics for neurodegeneration, potentially even enabling preemptive treatment of at-risk individuals.

One-Sentence SummaryWe engineer TDP-43-regulated cryptic exons, enabling exceptionally precise activation of gene therapies in diseased neurons.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567349">
        <p class="paperTitle">The embryonic DNA methylation program modulates the cis-regulatory landscape via CTCF antagonism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monteagudo-Sanchez, A.; Richard Albert, J.; Scarpa, M.; Noordermeer, D.; Greenberg, M. V. C.</p>
        <p class="info">Score: 33.8, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567349' target='https://doi.org/10.1101/2023.11.16.567349'> 10.1101/2023.11.16.567349</a></p>
        <p class="abstract">During mammalian embryogenesis, both the 5-cytosine DNA methylation (5meC) landscape and three-dimensional (3D) chromatin architecture are profoundly remodeled during a process known as &#34;epigenetic reprogramming.&#34; An understudied aspect of epigenetic reprogramming is how the 5meC flux, per se, affects the 3D genome. This is pertinent given the 5meC- sensitivity of DNA binding for a key regulator of chromosome folding: CTCF. We profiled the CTCF binding landscape using a mouse embryonic stem cell (ESC) differentiation protocol that models the exit of naive pluripotency, wherein global DNA methylation levels start low and increase to somatic levels within four days. We took advantage of the fact that mouse ESCs lacking DNA methylation machinery exhibit globally similar differentiation dynamics, thus allowing for dissection of more subtle effects of CTCF misregulation on gene expression. We carried this out by performing CTCF HiChIP in both wild-type and mutant conditions to assess aberrant CTCF-CTCF contacts in the absence of 5meC. We went on to assess the impact that misregulated CTCF binding has on cis-regulatory contacts by performing H3K27ac HiChIP, given that H3K27ac is enriched on active promoters and enhancers. Using DNA methylation epigenome editing, we were able to directly demonstrate that the DNA methyl- mark is able to impact CTCF binding. Finally, a detailed dissection of the imprinted Zdbf2 gene showed how 5meC-antagonism of CTCF allows for proper gene regulation during differentiation. This work provides a comprehensive overview of how DNA methylation impacts the 3D genome in a relevant model for early embryonic events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567549">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567549" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567549">
        <p class="paperTitle">Reversible histone deacetylase activity catalyzes lysine acylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567549" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567549" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsusaka, T.; Najar, M. A.; Schwarz, B.; Bohrnsen, E.; Oses-Prieto, J. A.; Lee, C.; Burlingame, A. L.; Bosio, C. M.; Burslem, G. M.; Goldberg, E. L.</p>
        <p class="info">Score: 20.4, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567549' target='https://doi.org/10.1101/2023.11.17.567549'> 10.1101/2023.11.17.567549</a></p>
        <p class="abstract">Starvation and low carbohydrate diets lead to the accumulation of the ketone body, {beta}-hydroxybutyrate (BHB), whose blood concentrations increase more than 10-fold into the millimolar range. In addition to providing a carbon source, BHB accumulation triggers lysine {beta}-hydroxybutyrylation (Kbhb) of proteins via unknown mechanisms. As with other lysine acylation events, Kbhb marks can be removed by histone deacetylases (HDACs). Here, we report that class I HDACs unexpectedly catalyze protein lysine modification with {beta}-hydroxybutyrate (BHB). Mutational analyses of the HDAC2 active site reveal a shared reliance on key amino acids for classical deacetylation and non-canonical HDAC-catalyzed {beta}-hydroxybutyrylation. Also consistent with reverse HDAC activity, Kbhb formation is driven by mass action and substrate availability. This reverse HDAC activity is not limited to BHB but also extends to multiple short-chain fatty acids. The reversible activity of class I HDACs described here represents a novel mechanism of PTM deposition relevant to metabolically-sensitive proteome modifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566739">
        <p class="paperTitle">A synthetic CRISPR-Cas nuclease with expanded enzymatic activities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jabalera, Y.; Tascon, I.; Samperio, S.; Lopez-Alonso, J. P.; Gonzalez-Lopez, M.; Aransay, A. M.; Abascal-Palacios, G.; Ubarretxena-Belandia, I.; Perez-Jimenez, R.</p>
        <p class="info">Score: 20.1, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566739' target='https://doi.org/10.1101/2023.11.12.566739'> 10.1101/2023.11.12.566739</a></p>
        <p class="abstract">Clustered regularly interspaced short palindromic repeats (CRISPR)-associated endonucleases have revolutionized biotechnology for their potential as programmable genome editors. Yet, most natural nucleases and their variants have limitations. Here, we report a fully synthetic CRISPR-associated (Cas) nuclease (-synCas) designed by Ancestral Sequence Reconstruction (ASR) that displays a set of robust and distinct targeting properties, not found in any other known CRISPR-Cas Class 2 system. We show that -synCas is a PAMless nuclease able to catalyse RNA-guided, specific cleavage of dsDNA, ssDNA and ssRNA. The synthetic enzyme is also capable of sequence-nonspecific degradation of dsDNA, ssDNA and ssRNA following activation by complementary dsDNA, ssDNA and ssRNA targets. Furthermore, -synCas exhibits a robust genome editing activity in human cells and bacteria. Cryo-electron microscopy structures of -synCas ternary and quaternary complexes provide a framework to understand the structural basis for its expanded enzymatic activities. The capability for programmable multimodal targeting of virtually any nucleic acid sequence distinguishes -synCas as a promising new tool to extend current CRISPR-based technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567020">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567020" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567020">
        <p class="paperTitle">Serotonin transporter-dependent histone serotonylation in placenta contributes to the neurodevelopmental transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567020" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567020" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, J. C.; Alenina, N.; Cunningham, A. M.; Ramakrishnan, A.; Shen, L.; Bader, M.; Maze, I.</p>
        <p class="info">Score: 12.3, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567020' target='https://doi.org/10.1101/2023.11.14.567020'> 10.1101/2023.11.14.567020</a></p>
        <p class="abstract">Brain development requires appropriate regulation of serotonin (5-HT) signaling from distinct tissue sources across embryogenesis. At the maternal-fetal interface, the placenta is thought to be an important contributor of offspring brain 5-HT and is critical to overall fetal health. Yet, how placental 5-HT is acquired, and the mechanisms through which 5-HT influences placental functions, are not well understood. Recently, our group identified a novel epigenetic role for 5-HT, in which 5-HT can be added to histone proteins to regulate transcription, a process called H3 serotonylation. Here, we show that H3 serotonylation undergoes dynamic regulation during placental development, corresponding to gene expression changes that are known to influence key metabolic processes. Using transgenic mice, we demonstrate that placental H3 serotonylation largely depends on 5-HT uptake by the serotonin transporter (SERT/SLC6A4). SERT deletion robustly reduces enrichment of H3 serotonylation across the placental genome, and disrupts neurodevelopmental gene networks in early embryonic brain tissues. Thus, these findings suggest a novel role for H3 serotonylation in coordinating placental transcription at the intersection of maternal physiology and offspring brain development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567213">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567213" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567213">
        <p class="paperTitle">Omecamtiv mecarbil and Mavacamten target the same myosin pocket despite antagonistic effects in heart contraction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567213" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567213" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Auguin, D.; Robert-Paganin, J.; Rety, S.; Kikuti, C.; David, A.; Theumer, G.; Schmidt, A. W.; Knoelker, H.-J.; Houdusse, A.</p>
        <p class="info">Score: 9.7, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567213' target='https://doi.org/10.1101/2023.11.15.567213'> 10.1101/2023.11.15.567213</a></p>
        <p class="abstract">Inherited cardiomyopathies are amongst the most common cardiac diseases worldwide, leading in the late-stage to heart failure and death. The most promising treatments against these diseases are small-molecules directly modulating the force produced by Beta-cardiac myosin, the molecular motor driving heart contraction. Two of these molecules that produce antagonistic effects on cardiac contractility have completed clinical phase 3 trials: the activator Omecamtiv mecarbil and the inhibitor Mavacamten. In this work, we reveal by X-ray crystallography that both drugs target the same pocket and stabilize a pre-stroke structural state, with only few local differences. All atoms molecular dynamics simulations reveal how these molecules can have antagonistic impact on the allostery of the motor by comparing Beta-cardiac myosin in the apo form or bound to Omecamtiv mecarbil or Mavacamten. Altogether, our results provide the framework for rational drug development for the purpose of personalized medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567086" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567086">
        <p class="paperTitle">Circadian Control of Histone Turnover During Cardiac Development and Growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arrieta, A.; Chapski, D. J.; Reese, A.; Kimball, T. H.; Song, K.; Rosa-Garrido, M.; Vondriska, T. M.</p>
        <p class="info">Score: 8.8, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567086' target='https://doi.org/10.1101/2023.11.14.567086'> 10.1101/2023.11.14.567086</a></p>
        <p class="abstract">Rationale: During postnatal cardiac hypertrophy, cardiomyocytes undergo mitotic exit, relying on DNA replication-independent mechanisms of histone turnover to maintain chromatin organization and gene transcription. In other tissues, circadian oscillations in nucleosome occupancy influence clock-controlled gene expression, suggesting an unrecognized role for the circadian clock in temporal control of histone turnover and coordinate cardiomyocyte gene expression. Objective: To elucidate roles for the master circadian transcription factor, Bmal1, in histone turnover, chromatin organization, and myocyte-specific gene expression and cell growth in the neonatal period. Methods and Results: Bmal1 knockdown in neonatal rat ventricular myocytes (NRVM) decreased myocyte size, total cellular protein, and transcription of the fetal hypertrophic gene Nppb following treatment with increasing serum concentrations or the -adrenergic agonist phenylephrine (PE). Bmal1 knockdown decreased expression of clock-controlled genes Per2 and Tcap, and salt-inducible kinase 1 (Sik1) which was identified via gene ontology analysis of Bmal1 targets upregulated in adult versus embryonic hearts. Epigenomic analyses revealed co-localized chromatin accessibility and Bmal1 localization in the Sik1 promoter. Bmal1 knockdown impaired Per2 and Sik1 promoter accessibility as measured by MNase-qPCR and impaired histone turnover indicated by metabolic labeling of acid-soluble chromatin fractions and immunoblots of total and chromatin-associated core histones. Sik1 knockdown basally increased myocyte size, while simultaneously impairing and driving Nppb and Per2 transcription, respectively. Conclusions: Bmal1 is required for neonatal myocyte growth, replication-independent histone turnover, and chromatin organization at the Sik1 promoter. Sik1 represents a novel clock-controlled gene that coordinates myocyte growth with hypertrophic and clock-controlled gene transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566706">
        <p class="paperTitle">Sterility-Independent Enhancement of Proteasome Function via Floxuridine-Triggered Detoxification in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dubey, A. A.; Szulc, N. A.; Piechota, M.; Serwa, R. A.; Pokrzywa, W.</p>
        <p class="info">Score: 7.4, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566706' target='https://doi.org/10.1101/2023.11.11.566706'> 10.1101/2023.11.11.566706</a></p>
        <p class="abstract">The ubiquitin-proteasome system (UPS) functionality is vital for proteostasis, contributing to stress resilience, lifespan, and thermal adaptability. In Caenorhabditis elegans, proteasome constituents such as the RPN-6 and PBS-6 subunits or the PSME-3 activator are respectively linked to heat resistance, longevity at moderate cold (15{degrees}C), and survival at low temperatures (4{degrees}C). Since the inhibition of germline stem cells proliferation is associated with robust proteostasis in worms, we utilized floxuridine (FUdR), a compound known for inducing sterility, to examine whether it could reinforce UPS under proteasome dysfunction, particularly to foster cold survival. We demonstrate that FUdR promotes proteasome resilience during its inhibition or subunit deficits, supporting normal lifespan and facilitating adaptation to cold. FUdRs elevation of the UPS activity occurs independently of main proteostasis regulators and is partly driven by SKN-1-regulated transcription, especially under reduced proteasome function. Additionally, we uncover a FUdR-stimulated detoxification pathway, distinct from both SKN-1 and the germline, with GST-24 emerging as a critical mediator of the UPS buffering. This research underscores FUdRs role in the UPS modulation and its contribution to survival of worms in low-temperature conditions.

HIGHLIGHTSO_LIFloxuridine (FUdR) enhances ubiquitin-proteasome system activity in C. elegans, independent of primary proteostasis regulators.
C_LIO_LIFUdR permits worms to maintain a normal lifespan and facilitates their adaptation to cold in the context of proteasome deficits.
C_LIO_LIActing independently of the germline and SKN-1, FUdR triggers a detoxification pathway, with GST-24 as a pivotal component in modulating the ubiquitin-proteasome system.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566157">
        <p class="paperTitle">N-terminal toxin signal peptides efficiently load therapeutics into a natural nano-injection system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steiner-Rebrova, E. M.; Ejaz, R. N.; Kielkopf, C. S.; Perez Ruiz, M.; Marin-Arraiza, L.; Hendriks, I. A.; Nissen, J. N.; Pozdnyakova, I.; Pape, T.; Regaiolo, A.; Goetz, K.; Heermann, R.; Rasmussen, S.; Nielsen, M. L.; Taylor, N. M. I.</p>
        <p class="info">Score: 6.6, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566157' target='https://doi.org/10.1101/2023.11.13.566157'> 10.1101/2023.11.13.566157</a></p>
        <p class="abstract">Targeted delivery of therapeutics to specific cells is a major bottleneck towards personalized medicine. The extracellular injection system (eCIS) of Serratia entomophila, the antifeeding prophage (Afp), promises potential for drug delivery purposes. However, the precise mechanism of action, toxin location, and Afp loading remain unclear. Here, we reveal a minimal N-terminal signal peptide (NtSP) of the toxin Afp18, that plays a key role in toxin packing. By engineering fusion proteins, we demonstrate that Afp18s NtSP can shuttle effectors for Afp loading. We packed non-eCIS effectors, including CRISPR-Cas protein Cas{Phi}-2 from Biggiephage, and a human antimicrobial peptide, LL37, into Afp. Additionally, NtSPs from eCIS effectors of other species facilitate loading of Cas{Phi}-2 into Afp. We observed cargo being packed inside the Afp tail tube through cryo-EM single particle analysis. The presented results enhance our understanding of eCIS toxin packing and contribute to their development as targeted delivery systems.

TeaserA novel use of the Afp nano injection systems N-terminal signal peptide in targeted therapeutics delivery</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
