<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The apparent loss of PRC2 chromatin occupancy as an artefact of RNA depletion
Authors: Healy, E.; Zhang, Q.; Gail, E. H.; Agius, S. C.; Sun, G.; Bullen, M.; Pandey, V.; Das, P. P.; Polo, J. M.; Davidovich, C.
Score: 60.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553488
RNA has been implicated in the recruitment of chromatin modifiers, and previous studies have provided evidence in favour and against this idea. RNase treatment of chromatin is a prevalent tool for the study of RNA-mediated regulation of chromatin modifiers, but the limitations of this approach remain unclear.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="The apparent loss of PRC2 chromatin occupancy as an artefact of RNA depletion
Authors: Healy, E.; Zhang, Q.; Gail, E. H.; Agius, S. C.; Sun, G.; Bullen, M.; Pandey, V.; Das, P. P.; Polo, J. M.; Davidovich, C.
Score: 60.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553488
RNA has been implicated in the recruitment of chromatin modifiers, and previous studies have provided evidence in favour and against this idea. RNase treatment of chromatin is a prevalent tool for the study of RNA-mediated regulation of chromatin modifiers, but the limitations of this approach remain unclear." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="The apparent loss of PRC2 chromatin occupancy as an artefact of RNA depletion
Authors: Healy, E.; Zhang, Q.; Gail, E. H.; Agius, S. C.; Sun, G.; Bullen, M.; Pandey, V.; Das, P. P.; Polo, J. M.; Davidovich, C.
Score: 60.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553488
RNA has been implicated in the recruitment of chromatin modifiers, and previous studies have provided evidence in favour and against this idea. RNase treatment of chromatin is a prevalent tool for the study of RNA-mediated regulation of chromatin modifiers, but the limitations of this approach remain unclear."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "The apparent loss of PRC2 chromatin occupancy as an artefact of RNA depletion\nAuthors: Healy, E.; Zhang, Q.; Gail, E. H.; Agius, S. C.; Sun, G.; Bullen, M.; Pandey, V.; Das, P. P.; Polo, J. M.; Davidovich, C.\nScore: 60.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553488\nRNA has been implicated in the recruitment of chromatin modifiers, and previous studies have provided evidence in favour and against this idea. RNase treatment of chromatin is a prevalent tool for the study of RNA-mediated regulation of chromatin modifiers, but the limitations of this approach remain unclear.",
  "keywords": [
    
  ],
  "articleBody": " The apparent loss of PRC2 chromatin occupancy as an artefact of RNA depletion\nAuthors: Healy, E.; Zhang, Q.; Gail, E. H.; Agius, S. C.; Sun, G.; Bullen, M.; Pandey, V.; Das, P. P.; Polo, J. M.; Davidovich, C.\nScore: 60.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553488\nRNA has been implicated in the recruitment of chromatin modifiers, and previous studies have provided evidence in favour and against this idea. RNase treatment of chromatin is a prevalent tool for the study of RNA-mediated regulation of chromatin modifiers, but the limitations of this approach remain unclear. One of the most studied chromatin modifiers in the context of RNA-mediated regulation is the H3K27me3 methyltransferase Polycomb Repressive Complex 2 (PRC2). RNase A treatment during chromatin immunoprecipitation (RNase-ChIP or rChIP) reduces the occupancy of PRC2 on chromatin. This led to suggestions of an \"RNA bridge\" between PRC2 and chromatin. Here we show that RNase A treatment during chromatin immunoprecipitation leads to the apparent loss of all facultative heterochromatin, including both PRC2 and its H3K27me3 mark genome wide. This phenomenon persists in mouse embryonic stem cells, human cancer cells and human-induced pluripotent stem cells. We track this observation to a global gain of chromatin that artificially reduces ChIP signals from facultative heterochromatin during normalisation. Our results point to substantial limitations in using RNase A treatment for mapping RNA-dependent chromatin occupancy and invalidate conclusions that were previously established for PRC2 based on this assay.\nApparent RNA bridging between PRC2 and chromatin is an artefact of non-specific chromatin precipitation upon RNA degradation\nAuthors: Hickman, A. H.; Jenner, R. G.\nScore: 19.9, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553503\nPolycomb repressive complex 2 (PRC2) modifies chromatin to maintain repression of genes specific for other cell lineages and is essential for development. In vitro, RNA inhibits PRC2 catalytic activity and interaction with nucleosomes but the effect of RNA on PRC2 in cells has been unclear, with studies concluding that RNA either antagonises or promotes the association of PRC2 with chromatin. The addition of RNase A to chromatin immunoprecipitation reactions has been reported to reduce detection of PRC2 target sites, suggesting the existence of an RNA bridge that connects PRC2 to chromatin. Here, we show that this apparent loss of PRC2 chromatin association after RNase A treatment is due to non-specific chromatin precipitation. RNA degradation causes chromatin to precipitate out of solution, thereby reducing the enrichment of specific DNA sequences in chromatin immunoprecipitation reactions. Maintaining chromatin solubility by the addition of poly-L-glutamic acid (PGA) rescues detection of PRC2 chromatin association upon RNA degradation. These findings undermine support for the model that RNA bridges PRC2 and chromatin in cells.\nEukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes\nAuthors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.\nScore: 59.5, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727\nRNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing. Some of these enzymes are components of insertion sequences (IS) in the IS200/IS605 and IS607 transposon families. Both IS families encode a TnpA transposase and TnpB nuclease, an RNA-guided enzyme ancestral to CRISPR-Cas12. In eukaryotes and their viruses, TnpB homologs occur as two distinct types, Fanzor1 and Fanzor2. We analyzed the evolutionary relationships between prokaryotic TnpBs and eukaryotic Fanzors, revealing that a clade of IS607 TnpBs with unusual active site arrangement found primarily in Cyanobacteriota likely gave rise to both types of Fanzors. The wide-spread nature of Fanzors imply that the properties of this particular group of IS607 TnpBs were particularly suited to adaptation and evolution in eukaryotes and their viruses. Experimental characterization of a prokaryotic IS607 TnpB and virally encoded Fanzor1s uncovered features that may have fostered coevolution between TnpBs/Fanzors and their cognate transposases. Our results provide insight into the evolutionary origins of a ubiquitous family of RNA-guided proteins that shows remarkable conservation across domains of life.\nTazobactam selects for multidrug resistance\nAuthors: Holden, E. R.; Yasir, M.; Turner, A. K.; Charles, I. G.; Webber, M. A.\nScore: 12.2, Published: 2023-08-15 DOI: 10.1101/2023.08.15.553388\nPiperacillin-Tazobactam is a {beta}-lactam/{beta}-lactamase inhibitor combination which is amongst the most prescribed antimicrobials in hospital medicine. Piperacillin is inactivated by commonly carried resistance enzymes, but tazobactam inhibits these allowing successful treatment. The effect of piperacillin on Gram-negative bacteria has been widely studied, but less attention has been paid to the effects of tazobactam. We used a massive transposon mutagenesis approach (TraDIS-Xpress) to determine the genes in Escherichia coli that affect survival when exposed to piperacillin and tazobactam, separately and together. We found significant differences in the selective pressure of the two drugs: a striking finding was that multiple efflux pump families and regulators were essential for survival in the presence of tazobactam, but only one efflux system was beneficial for piperacillin. Additionally, we identified the shikimate kinase AroK as a potential target for tazobactam. This method also found that genes involved in DNA replication and repair reduced E. coli susceptibility to a combination of piperacillin and tazobactam, not seen from either drug treatment alone. Treatment of E. coli and Klebsiella pneumoniae with piperacillin and/or tazobactam selected for mutants with reduced susceptibility, and SNP analyses supported the TraDIS-Xpress findings that tazobactam selects for changes in membrane permeability and maintenance associated with multidrug-resistance. Increased efflux activity is an important foundation of multidrug resistance in human pathogens, therefore the finding that tazobactam can select for this is concerning. These findings could have consequences for antibiotic prescription and should inform the development of future {beta}-lactamase inhibitors to reduce the global increase in multidrug-resistant infections.\nAePUb promoter length modulates gene expression in Aedes aegypti\nAuthors: Anderson, M. A. E.; Leftwich, P. T.; Wilson, R.; Carabajal Paladino, L. Z.; Basu, S.; Rooney, S.; Adelman, Z.; Alphey, L.\nScore: 12.9, Published: 2023-08-11 DOI: 10.1101/2023.08.11.553001\nMolecular tools for modulating transgene expression in Aedes aegypti are few. Here we demonstrate that adjustments to the AePUb promoter length can alter expression levels of two reporter proteins in Ae. aegypti cell culture and in mosquitoes. This provides a simple means for increasing or decreasing expression of a gene of interest and easy translation from cells to whole insects.\nMYC and MAX drive the reactivation of the genome after mitosis\nAuthors: Gonzalez, I.; Chervova, A.; Escoll Guerrero, P.; Altamirano-Pacheco, L.; Mueller, F.; Dubois, A.; Navarro, P.\nScore: 10.8, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553120\nShortly after cell division, a robust wave of hyper-transcription reactivates the genome1-3. This phenomenon is particularly pronounced in pluripotent cells4, which necessitate rapid transcrip-tome reactivation to maintain their undifferentiated state and prevent premature differentiation. While recent work has illuminated how specific groups of genes are reactivated4-8, the mechanisms enabling the global, efficient and accurate post-mitotic reactivation of the genome remain unknown. Here we elucidate the direct involvement of the MYC/MAX transcription factors in the post-mitotic reactivation of pluripotent mouse embryonic stem cells. While MYC undergoes extensive phosphorylation and largely dissociates from its DNA binding sites during mitosis, we report that MAX remains bound to its targets, preferentially at promoters, and facilitates early recruitment of MYC following mitosis. Through the application of MYC/MAX heterodimerization inhibitors, we demonstrate their indispensable role in sustaining hyper-transcription in ES cells, including during the critical transition from mitosis to G1 phase. Our findings uncover a novel role for MAX in mitotic book-marking, highlighting its pivotal role in post-mitotic MYC recruitment and the re-establishment of high global transcription levels. These findings hold significant implications for medically relevant contexts, particularly when cell proliferation is of paramount importance9. We anticipate that the study of mitotic bookmarking by MYC and MAX and of the effects of anticancer drugs targeting MYC/MAX interactions in such process10-12 will be relevant for our understanding of cancer and its potential treatments.\nHighly multiplexed mRNA quantitation with CRISPR-Cas13\nAuthors: Kang, B.; Zhang, J.; Schwoerer, M. P.; Nelson, A. N.; Schoeman, E.; Ploss, A.; Myhrvold, C.\nScore: 10.7, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553527\nRNA quantitation tools are often either high-throughput or cost-effective, but rarely are they both. Existing methods can profile the transcriptome at great expense or are limited to quantifying a handful of genes by labor constraints. A technique that permits more throughput at a reduced cost could enable multi-gene kinetic studies, gene regulatory network analysis, and combinatorial genetic screens. Here, we introduce quantitative Combinatorial Arrayed Reactions for Multiplexed Evaluation of Nucleic acids (qCARMEN): an RNA quantitation technique which leverages the programmable RNA-targeting capabilities of CRISPR-Cas13 to address this challenge by quantifying over 4,500 gene-sample pairs in a single experiment. Using qCARMEN, we studied the response profiles of interferon-stimulated genes (ISGs) during interferon (IFN) stimulation and flavivirus infection. Additionally, we observed isoform switching kinetics during epithelial-mesenchymal transition. qCARMEN is a simple and inexpensive technique that greatly enhances the scalability of RNA quantitation for novel applications with performance similar to gold-standard methods.\nAn anciently diverged family of RNA binding proteins maintain correct splicing of ultra-long exons through cryptic splice site repression.\nAuthors: Siachisumo, C.; Luzzi, S.; Aldalaqan, S.; Hysenaj, G.; Dalgliesh, C.; Cheung, K.; Gazzara, M. R.; Yonchev, I. D.; James, K.; Kheirollahi Chadegani, M.; Ehrmann, I. E.; Smith, G. R.; Cockell, S. J.; Munkley, J.; wilson, s. a.; Barash, Y.; Elliott, D.\nScore: 8.6, Published: 2023-08-15 DOI: 10.1101/2023.08.15.553384\nWe previously showed that the germ cell specific nuclear protein RBMXL2 represses cryptic splicing patterns during meiosis and is required for male fertility. It has remained unknown whether RBMXL2 evolved its role in splicing repression to deal with the transcriptionally permissive environment of meiosis or might fulfil a function required in all cells. RBMXL2 evolved from the X-linked RBMX gene, which is silenced during meiosis due to sex chromosome inactivation. Here we find that like RBMXL2, RBMX primarily operates as a splicing repressor in somatic cells, and specifically regulates a distinct class of exons that exceed the median human exon size. RBMX protein-RNA interactions are enriched within ultra-long exons, particularly within genes involved in genome stability, and RBMX represses the selection of cryptic splice sites that would compromise gene function. These similarities in overall function suggested that RBMXL2 during meiosis might replace the otherwise ubiquitous RBMX protein. To test this prediction we carried out inducible expression of RBMXL2 and the more distantly related RBMY protein in somatic cells, finding each could rescue aberrant patterns of RNA processing in response to RBMX depletion. The C-terminal disordered domain of RBMXL2 is sufficient to rescue proper splicing control after RBMX depletion. Our data indicate that RBMXL2 replaces RBMX during meiosis, and these proteins have maintained parallel roles that must have been conserved over at least 200 million years of mammalian evolution. We propose RBMX family proteins are important for the splicing inclusion of ultra-long exons because these are particularly susceptible to disruption by cryptic splice site selection.\nN-glycosylation as a eukaryotic protective mechanism against protein aggregation\nAuthors: Duran-Romana, R.; Houben, B.; De Vleeschouwer, M.; Louros, N.; Wilson, M. P.; Matthijs, G.; Schymkowitz, J.; Rousseau, F.\nScore: 13.4, Published: 2023-08-11 DOI: 10.1101/2023.08.11.552904\nThe tendency for proteins to form aggregates is an inherent part of every proteome and arises from the self-assembly of short protein segments called aggregation-prone regions (APRs). While post-translational modifications (PTMs) have been implicated in modulating protein aggregation, their direct role in APRs remains poorly understood. In this study, we used a combination of proteome-wide computational analyses and biochemical techniques to investigate the potential involvement of PTMs in aggregation regulation. Our findings reveal that while most PTM types are disfavored near APRs, N-glycosylation is enriched and evolutionarily selected, especially in proteins prone to misfolding. Experimentally, we show that N-glycosylation inhibits the aggregation of peptides in vitro through steric hindrance. Moreover, mining existing proteomics data, we find that the loss of N-glycans at the flanks of APRs leads to specific protein aggregation in Neuro2a cells. Our results point towards a novel intrinsic role for N-glycosylation, directly preventing protein aggregation in eukaryotes.\nGene editing in the Chagas disease vector Rhodnius prolixus by Cas9-mediated ReMOT Control\nAuthors: Lima, L.; Berni, M.; Mota, J.; Bressan, D.; Julio, A.; Cavalcante, R.; Macias, V.; Li, Z.; Rasgon, J. L.; Bier, E.; Araujo, H. M.\nScore: 6.7, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553172\nRhodnius prolixus is currently the model vector of choice for studying Chagas disease transmission, a debilitating disease caused by Trypanosoma cruzi parasites. Despite the broad use of double-stranded RNA interference for the knockdown of gene function in R. prolixus, transgenesis and gene editing protocols are still lacking. Here we tested Receptor-Mediated Ovary Transduction of Cargo (ReMOT Control) and direct parental injection of CRISPR (DIPA-CRISPR) for the maternal delivery of CRISPR/Cas9 elements to developing R. prolixus oocytes, and strategies for the identification of insertions/deletions (indels) in target loci of resulting gene-edited G0 nymphs. We demonstrate successful ReMOT Control-mediated gene editing of the eye color markers Rp-scarlet and Rp-white, and the cuticle color marker Rp-yellow, with highest effectiveness obtained using the ovary-targeting BtKV ligand. These results provide proof-of-concepts for generating somatic mutations in R. prolixus and potentially for generating germline edited lines in triatomines. Our studies also suggest that optimal strategies for recovery of mutations include performing multiple gRNA injections and the use of visible phenotypes such as those displayed in the Rp-scarlet, Rp-white and Rp-yellow loci for future Co-CRISPR experiments. These results will lay the foundation for gene editing protocols for triatomines and could lead to the development of novel control strategies for vectors of Chagas disease. Author SummaryRhodnius prolixus is an insect vector of the protozoan Trypanossoma cruzi, causative agent of debilitating Chagas disease. To fight the spread of the disease, it has been suggested that biological control of the insect should be attempted. Gene editing by the novel CRISPR methodology holds great promise in this sense, as it enables to target almost any gene in the genome for mutagenesis, thus allowing the control of insect physiology and reproduction. Here we have tested protocols for the delivery of CRISPR reagents as an attempt to enable genome editing of the vector. Our results show that maternal delivery of CRISPR by the ReMOT Control method is efficient for mutating eye and cuticle color genes in the resulting nymphs, generating edited animals with red eyes, white eyes or a yellow cuticle. This is the first report of gene editing in a vector of Chagas disease and should lay the basis to produce modified animals either unable to carry the T. cruzi parasite or to reproduce.\n",
  "wordCount" : "2395",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553488">
        <p class="paperTitle">The apparent loss of PRC2 chromatin occupancy as an artefact of RNA depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Healy, E.; Zhang, Q.; Gail, E. H.; Agius, S. C.; Sun, G.; Bullen, M.; Pandey, V.; Das, P. P.; Polo, J. M.; Davidovich, C.</p>
        <p class="info">Score: 60.0, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553488' target='https://doi.org/10.1101/2023.08.16.553488'> 10.1101/2023.08.16.553488</a></p>
        <p class="abstract">RNA has been implicated in the recruitment of chromatin modifiers, and previous studies have provided evidence in favour and against this idea. RNase treatment of chromatin is a prevalent tool for the study of RNA-mediated regulation of chromatin modifiers, but the limitations of this approach remain unclear. One of the most studied chromatin modifiers in the context of RNA-mediated regulation is the H3K27me3 methyltransferase Polycomb Repressive Complex 2 (PRC2). RNase A treatment during chromatin immunoprecipitation (RNase-ChIP or rChIP) reduces the occupancy of PRC2 on chromatin. This led to suggestions of an &#34;RNA bridge&#34; between PRC2 and chromatin. Here we show that RNase A treatment during chromatin immunoprecipitation leads to the apparent loss of all facultative heterochromatin, including both PRC2 and its H3K27me3 mark genome wide. This phenomenon persists in mouse embryonic stem cells, human cancer cells and human-induced pluripotent stem cells. We track this observation to a global gain of chromatin that artificially reduces ChIP signals from facultative heterochromatin during normalisation. Our results point to substantial limitations in using RNase A treatment for mapping RNA-dependent chromatin occupancy and invalidate conclusions that were previously established for PRC2 based on this assay.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553503">
        <p class="paperTitle">Apparent RNA bridging between PRC2 and chromatin is an artefact of non-specific chromatin precipitation upon RNA degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hickman, A. H.; Jenner, R. G.</p>
        <p class="info">Score: 19.9, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553503' target='https://doi.org/10.1101/2023.08.16.553503'> 10.1101/2023.08.16.553503</a></p>
        <p class="abstract">Polycomb repressive complex 2 (PRC2) modifies chromatin to maintain repression of genes specific for other cell lineages and is essential for development. In vitro, RNA inhibits PRC2 catalytic activity and interaction with nucleosomes but the effect of RNA on PRC2 in cells has been unclear, with studies concluding that RNA either antagonises or promotes the association of PRC2 with chromatin. The addition of RNase A to chromatin immunoprecipitation reactions has been reported to reduce detection of PRC2 target sites, suggesting the existence of an RNA bridge that connects PRC2 to chromatin. Here, we show that this apparent loss of PRC2 chromatin association after RNase A treatment is due to non-specific chromatin precipitation. RNA degradation causes chromatin to precipitate out of solution, thereby reducing the enrichment of specific DNA sequences in chromatin immunoprecipitation reactions. Maintaining chromatin solubility by the addition of poly-L-glutamic acid (PGA) rescues detection of PRC2 chromatin association upon RNA degradation. These findings undermine support for the model that RNA bridges PRC2 and chromatin in cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552727">
        <p class="paperTitle">Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.</p>
        <p class="info">Score: 59.5, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552727' target='https://doi.org/10.1101/2023.08.09.552727'> 10.1101/2023.08.09.552727</a></p>
        <p class="abstract">RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing. Some of these enzymes are components of insertion sequences (IS) in the IS200/IS605 and IS607 transposon families. Both IS families encode a TnpA transposase and TnpB nuclease, an RNA-guided enzyme ancestral to CRISPR-Cas12. In eukaryotes and their viruses, TnpB homologs occur as two distinct types, Fanzor1 and Fanzor2. We analyzed the evolutionary relationships between prokaryotic TnpBs and eukaryotic Fanzors, revealing that a clade of IS607 TnpBs with unusual active site arrangement found primarily in Cyanobacteriota likely gave rise to both types of Fanzors. The wide-spread nature of Fanzors imply that the properties of this particular group of IS607 TnpBs were particularly suited to adaptation and evolution in eukaryotes and their viruses. Experimental characterization of a prokaryotic IS607 TnpB and virally encoded Fanzor1s uncovered features that may have fostered coevolution between TnpBs/Fanzors and their cognate transposases. Our results provide insight into the evolutionary origins of a ubiquitous family of RNA-guided proteins that shows remarkable conservation across domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553388">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553388" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553388">
        <p class="paperTitle">Tazobactam selects for multidrug resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553388" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553388" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holden, E. R.; Yasir, M.; Turner, A. K.; Charles, I. G.; Webber, M. A.</p>
        <p class="info">Score: 12.2, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553388' target='https://doi.org/10.1101/2023.08.15.553388'> 10.1101/2023.08.15.553388</a></p>
        <p class="abstract">Piperacillin-Tazobactam is a {beta}-lactam/{beta}-lactamase inhibitor combination which is amongst the most prescribed antimicrobials in hospital medicine. Piperacillin is inactivated by commonly carried resistance enzymes, but tazobactam inhibits these allowing successful treatment. The effect of piperacillin on Gram-negative bacteria has been widely studied, but less attention has been paid to the effects of tazobactam. We used a massive transposon mutagenesis approach (TraDIS-Xpress) to determine the genes in Escherichia coli that affect survival when exposed to piperacillin and tazobactam, separately and together. We found significant differences in the selective pressure of the two drugs: a striking finding was that multiple efflux pump families and regulators were essential for survival in the presence of tazobactam, but only one efflux system was beneficial for piperacillin. Additionally, we identified the shikimate kinase AroK as a potential target for tazobactam. This method also found that genes involved in DNA replication and repair reduced E. coli susceptibility to a combination of piperacillin and tazobactam, not seen from either drug treatment alone. Treatment of E. coli and Klebsiella pneumoniae with piperacillin and/or tazobactam selected for mutants with reduced susceptibility, and SNP analyses supported the TraDIS-Xpress findings that tazobactam selects for changes in membrane permeability and maintenance associated with multidrug-resistance. Increased efflux activity is an important foundation of multidrug resistance in human pathogens, therefore the finding that tazobactam can select for this is concerning. These findings could have consequences for antibiotic prescription and should inform the development of future {beta}-lactamase inhibitors to reduce the global increase in multidrug-resistant infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553001" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553001">
        <p class="paperTitle">AePUb promoter length modulates gene expression in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderson, M. A. E.; Leftwich, P. T.; Wilson, R.; Carabajal Paladino, L. Z.; Basu, S.; Rooney, S.; Adelman, Z.; Alphey, L.</p>
        <p class="info">Score: 12.9, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553001' target='https://doi.org/10.1101/2023.08.11.553001'> 10.1101/2023.08.11.553001</a></p>
        <p class="abstract">Molecular tools for modulating transgene expression in Aedes aegypti are few. Here we demonstrate that adjustments to the AePUb promoter length can alter expression levels of two reporter proteins in Ae. aegypti cell culture and in mosquitoes. This provides a simple means for increasing or decreasing expression of a gene of interest and easy translation from cells to whole insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553120">
        <p class="paperTitle">MYC and MAX drive the reactivation of the genome after mitosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez, I.; Chervova, A.; Escoll Guerrero, P.; Altamirano-Pacheco, L.; Mueller, F.; Dubois, A.; Navarro, P.</p>
        <p class="info">Score: 10.8, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553120' target='https://doi.org/10.1101/2023.08.13.553120'> 10.1101/2023.08.13.553120</a></p>
        <p class="abstract">Shortly after cell division, a robust wave of hyper-transcription reactivates the genome1-3. This phenomenon is particularly pronounced in pluripotent cells4, which necessitate rapid transcrip-tome reactivation to maintain their undifferentiated state and prevent premature differentiation. While recent work has illuminated how specific groups of genes are reactivated4-8, the mechanisms enabling the global, efficient and accurate post-mitotic reactivation of the genome remain unknown. Here we elucidate the direct involvement of the MYC/MAX transcription factors in the post-mitotic reactivation of pluripotent mouse embryonic stem cells. While MYC undergoes extensive phosphorylation and largely dissociates from its DNA binding sites during mitosis, we report that MAX remains bound to its targets, preferentially at promoters, and facilitates early recruitment of MYC following mitosis. Through the application of MYC/MAX heterodimerization inhibitors, we demonstrate their indispensable role in sustaining hyper-transcription in ES cells, including during the critical transition from mitosis to G1 phase. Our findings uncover a novel role for MAX in mitotic book-marking, highlighting its pivotal role in post-mitotic MYC recruitment and the re-establishment of high global transcription levels. These findings hold significant implications for medically relevant contexts, particularly when cell proliferation is of paramount importance9. We anticipate that the study of mitotic bookmarking by MYC and MAX and of the effects of anticancer drugs targeting MYC/MAX interactions in such process10-12 will be relevant for our understanding of cancer and its potential treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553527">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553527" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553527">
        <p class="paperTitle">Highly multiplexed mRNA quantitation with CRISPR-Cas13</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553527" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553527" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kang, B.; Zhang, J.; Schwoerer, M. P.; Nelson, A. N.; Schoeman, E.; Ploss, A.; Myhrvold, C.</p>
        <p class="info">Score: 10.7, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553527' target='https://doi.org/10.1101/2023.08.16.553527'> 10.1101/2023.08.16.553527</a></p>
        <p class="abstract">RNA quantitation tools are often either high-throughput or cost-effective, but rarely are they both. Existing methods can profile the transcriptome at great expense or are limited to quantifying a handful of genes by labor constraints. A technique that permits more throughput at a reduced cost could enable multi-gene kinetic studies, gene regulatory network analysis, and combinatorial genetic screens. Here, we introduce quantitative Combinatorial Arrayed Reactions for Multiplexed Evaluation of Nucleic acids (qCARMEN): an RNA quantitation technique which leverages the programmable RNA-targeting capabilities of CRISPR-Cas13 to address this challenge by quantifying over 4,500 gene-sample pairs in a single experiment. Using qCARMEN, we studied the response profiles of interferon-stimulated genes (ISGs) during interferon (IFN) stimulation and flavivirus infection. Additionally, we observed isoform switching kinetics during epithelial-mesenchymal transition. qCARMEN is a simple and inexpensive technique that greatly enhances the scalability of RNA quantitation for novel applications with performance similar to gold-standard methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553384" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553384">
        <p class="paperTitle">An anciently diverged family of RNA binding proteins maintain correct splicing of ultra-long exons through cryptic splice site repression.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Siachisumo, C.; Luzzi, S.; Aldalaqan, S.; Hysenaj, G.; Dalgliesh, C.; Cheung, K.; Gazzara, M. R.; Yonchev, I. D.; James, K.; Kheirollahi Chadegani, M.; Ehrmann, I. E.; Smith, G. R.; Cockell, S. J.; Munkley, J.; wilson, s. a.; Barash, Y.; Elliott, D.</p>
        <p class="info">Score: 8.6, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553384' target='https://doi.org/10.1101/2023.08.15.553384'> 10.1101/2023.08.15.553384</a></p>
        <p class="abstract">We previously showed that the germ cell specific nuclear protein RBMXL2 represses cryptic splicing patterns during meiosis and is required for male fertility. It has remained unknown whether RBMXL2 evolved its role in splicing repression to deal with the transcriptionally permissive environment of meiosis or might fulfil a function required in all cells. RBMXL2 evolved from the X-linked RBMX gene, which is silenced during meiosis due to sex chromosome inactivation. Here we find that like RBMXL2, RBMX primarily operates as a splicing repressor in somatic cells, and specifically regulates a distinct class of exons that exceed the median human exon size. RBMX protein-RNA interactions are enriched within ultra-long exons, particularly within genes involved in genome stability, and RBMX represses the selection of cryptic splice sites that would compromise gene function. These similarities in overall function suggested that RBMXL2 during meiosis might replace the otherwise ubiquitous RBMX protein. To test this prediction we carried out inducible expression of RBMXL2 and the more distantly related RBMY protein in somatic cells, finding each could rescue aberrant patterns of RNA processing in response to RBMX depletion. The C-terminal disordered domain of RBMXL2 is sufficient to rescue proper splicing control after RBMX depletion. Our data indicate that RBMXL2 replaces RBMX during meiosis, and these proteins have maintained parallel roles that must have been conserved over at least 200 million years of mammalian evolution. We propose RBMX family proteins are important for the splicing inclusion of ultra-long exons because these are particularly susceptible to disruption by cryptic splice site selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552904">
        <p class="paperTitle">N-glycosylation as a eukaryotic protective mechanism against protein aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duran-Romana, R.; Houben, B.; De Vleeschouwer, M.; Louros, N.; Wilson, M. P.; Matthijs, G.; Schymkowitz, J.; Rousseau, F.</p>
        <p class="info">Score: 13.4, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552904' target='https://doi.org/10.1101/2023.08.11.552904'> 10.1101/2023.08.11.552904</a></p>
        <p class="abstract">The tendency for proteins to form aggregates is an inherent part of every proteome and arises from the self-assembly of short protein segments called aggregation-prone regions (APRs). While post-translational modifications (PTMs) have been implicated in modulating protein aggregation, their direct role in APRs remains poorly understood. In this study, we used a combination of proteome-wide computational analyses and biochemical techniques to investigate the potential involvement of PTMs in aggregation regulation. Our findings reveal that while most PTM types are disfavored near APRs, N-glycosylation is enriched and evolutionarily selected, especially in proteins prone to misfolding. Experimentally, we show that N-glycosylation inhibits the aggregation of peptides in vitro through steric hindrance. Moreover, mining existing proteomics data, we find that the loss of N-glycans at the flanks of APRs leads to specific protein aggregation in Neuro2a cells. Our results point towards a novel intrinsic role for N-glycosylation, directly preventing protein aggregation in eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553172">
        <p class="paperTitle">Gene editing in the Chagas disease vector Rhodnius prolixus by Cas9-mediated ReMOT Control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lima, L.; Berni, M.; Mota, J.; Bressan, D.; Julio, A.; Cavalcante, R.; Macias, V.; Li, Z.; Rasgon, J. L.; Bier, E.; Araujo, H. M.</p>
        <p class="info">Score: 6.7, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553172' target='https://doi.org/10.1101/2023.08.14.553172'> 10.1101/2023.08.14.553172</a></p>
        <p class="abstract">Rhodnius prolixus is currently the model vector of choice for studying Chagas disease transmission, a debilitating disease caused by Trypanosoma cruzi parasites. Despite the broad use of double-stranded RNA interference for the knockdown of gene function in R. prolixus, transgenesis and gene editing protocols are still lacking. Here we tested Receptor-Mediated Ovary Transduction of Cargo (ReMOT Control) and direct parental injection of CRISPR (DIPA-CRISPR) for the maternal delivery of CRISPR/Cas9 elements to developing R. prolixus oocytes, and strategies for the identification of insertions/deletions (indels) in target loci of resulting gene-edited G0 nymphs. We demonstrate successful ReMOT Control-mediated gene editing of the eye color markers Rp-scarlet and Rp-white, and the cuticle color marker Rp-yellow, with highest effectiveness obtained using the ovary-targeting BtKV ligand. These results provide proof-of-concepts for generating somatic mutations in R. prolixus and potentially for generating germline edited lines in triatomines. Our studies also suggest that optimal strategies for recovery of mutations include performing multiple gRNA injections and the use of visible phenotypes such as those displayed in the Rp-scarlet, Rp-white and Rp-yellow loci for future Co-CRISPR experiments. These results will lay the foundation for gene editing protocols for triatomines and could lead to the development of novel control strategies for vectors of Chagas disease.

Author SummaryRhodnius prolixus is an insect vector of the protozoan Trypanossoma cruzi, causative agent of debilitating Chagas disease. To fight the spread of the disease, it has been suggested that biological control of the insect should be attempted. Gene editing by the novel CRISPR methodology holds great promise in this sense, as it enables to target almost any gene in the genome for mutagenesis, thus allowing the control of insect physiology and reproduction. Here we have tested protocols for the delivery of CRISPR reagents as an attempt to enable genome editing of the vector. Our results show that maternal delivery of CRISPR by the ReMOT Control method is efficient for mutating eye and cuticle color genes in the resulting nymphs, generating edited animals with red eyes, white eyes or a yellow cuticle. This is the first report of gene editing in a vector of Chagas disease and should lay the basis to produce modified animals either unable to carry the T. cruzi parasite or to reproduce.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
