<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="De novo basecalling of m6A modifications at single molecule and single nucleotide resolution
Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.
Score: 45.6, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801
RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="De novo basecalling of m6A modifications at single molecule and single nucleotide resolution
Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.
Score: 45.6, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801
RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="De novo basecalling of m6A modifications at single molecule and single nucleotide resolution
Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.
Score: 45.6, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801
RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "De novo basecalling of m6A modifications at single molecule and single nucleotide resolution\nAuthors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.\nScore: 45.6, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801\nRNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules.",
  "keywords": [
    
  ],
  "articleBody": " De novo basecalling of m6A modifications at single molecule and single nucleotide resolution\nAuthors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.\nScore: 45.6, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566801\nRNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules. Herein, we introduce m6ABasecaller, an innovative method enabling direct basecalling of m6A modifications from raw nanopore signals within individual RNA molecules. This approach facilitates de novo prediction of m6A modifications with precision down to the single nucleotide and single molecule levels, without the need of paired knockout or control conditions. Using the m6ABasecaller, we find that the median transcriptome-wide m6A modification stoichiometry is ~10-15% in human, mouse and zebrafish. Furthermore, we show that m6A modifications affect polyA tail lengths, exhibit a propensity for co-occurrence within the same RNA molecules, and show relatively consistent stoichiometry levels across isoforms. We further validate the m6ABasecaller by treating mESC with increasing concentrations of STM2457, a METTL3 inhibitor as well as in inducible METTL3 knockout systems. Overall, this work demonstrates the feasibility de novo basecalling of m6A modifications, opening novel avenues for the application of nanopore sequencing to samples with limited RNA availability and for which control knockout conditions are unavailable, such as patient-derived samples.\nNaP-TRAP, a novel massively parallel reporter assay to quantify translation control.\nAuthors: Strayer, E. C.; Krishna, S.; Lee, H.; Vejnar, C.; Beaudoin, J.-D.; Giraldez, A. J.\nScore: 21.2, Published: 2023-11-09 DOI: 10.1101/2023.11.09.566434\nThe cis-regulatory elements encoded in a mRNA determine its stability and translational output. While there has been a considerable effort to understand the factors driving mRNA stability, the regulatory frameworks governing translational control remain elusive. We have developed a novel massively parallel reporter assay (MPRA) to measure mRNA translation, Nascent Peptide Translating Ribosome Affinity Purification (NaP-TRAP). NaP-TRAP measures translation in a frame specific manner through the immunocapture of epitope tagged nascent peptides of reporter mRNAs. In contrast to existing MPRA methods, NaP-TRAP does not require specialized equipment and is readily adaptable to steady-state and dynamic model systems. We have employed NaP-TRAP to quantify Kozak strength and the regulatory landscapes of 5 UTRs in the developing zebrafish embryo and in human cells, characterizing general and developmentally dynamic cis-regulatory elements. To this end, we identify U-rich motifs as general enhancers, and upstream ORFs and GC-rich motifs as global repressors of translation. We also observe a translational switch during the maternal-to-zygotic transition, where C-rich motifs shift from repressors to prominent activators of translation. Conversely, we show that microRNA sites in the 5 UTR repress translation following the zygotic expression of miR-430. Together these results demonstrate that NaP-TRAP is a versatile, accessible, and powerful method to decode the regulatory functions of UTRs across different systems.\nA synthetic CRISPR-Cas nuclease with expanded enzymatic activities\nAuthors: Jabalera, Y.; Tascon, I.; Samperio, S.; Lopez-Alonso, J. P.; Gonzalez-Lopez, M.; Aransay, A. M.; Abascal-Palacios, G.; Ubarretxena-Belandia, I.; Perez-Jimenez, R.\nScore: 20.1, Published: 2023-11-12 DOI: 10.1101/2023.11.12.566739\nClustered regularly interspaced short palindromic repeats (CRISPR)-associated endonucleases have revolutionized biotechnology for their potential as programmable genome editors. Yet, most natural nucleases and their variants have limitations. Here, we report a fully synthetic CRISPR-associated (Cas) nuclease (-synCas) designed by Ancestral Sequence Reconstruction (ASR) that displays a set of robust and distinct targeting properties, not found in any other known CRISPR-Cas Class 2 system. We show that -synCas is a PAMless nuclease able to catalyse RNA-guided, specific cleavage of dsDNA, ssDNA and ssRNA. The synthetic enzyme is also capable of sequence-nonspecific degradation of dsDNA, ssDNA and ssRNA following activation by complementary dsDNA, ssDNA and ssRNA targets. Furthermore, -synCas exhibits a robust genome editing activity in human cells and bacteria. Cryo-electron microscopy structures of -synCas ternary and quaternary complexes provide a framework to understand the structural basis for its expanded enzymatic activities. The capability for programmable multimodal targeting of virtually any nucleic acid sequence distinguishes -synCas as a promising new tool to extend current CRISPR-based technologies.\nBDNF controls phosphorylation and transcriptional networks governing cytoskeleton organization and axonal regeneration\nAuthors: Vargas, J. N. S.; Brown, A.-L.; Sun, K.; Hagemann, C.; Geary, B.; Villarroel-Campos, D.; Bryce-Smith, S.; Zanovello, M.; Lombardo, M.; Ryandov, E.; Stepic, A.; Meysman, L.; Majewski, S.; Tosolini, A.; Secrier, M.; Keuss, M. J.; Serio, A.; Sleigh, J. N.; Fratta, P.; Schiavo, G.\nScore: 18.3, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565775\nAxonal degeneration underlies neuromuscular disorders and neuropathies. Dysregulation of neurotrophic factors, such as brain-derived neurotrophic factor (BDNF), in the peripheral nervous system has long been established to exacerbate axonopathy. However, the molecular programs controlled by BDNF that facilitate axonal regeneration and transport are not well-understood. Here, we unravel the transcriptomic and phosphorylation landscape shaped by BDNF in human iPSC-derived motor neurons. Using SLAM-Seq, we reveal BDNF stimulation increases global transcription rate in motor neurons and governs gene regulatory networks that converge with those engaged during axonal repair/outgrowth. Phosphoproteomic analyses demonstrate that BDNF remodels the phosphorylation landscape of cytoskeletal-binding proteins, especially of structural microtubule-associated proteins. Importantly, the localized axonal-specific activation of ERK1/2 is necessary for BDNF to enhance axonal transport of neurotrophin-containing signaling endosomes and to potentiate axonal regeneration after axotomy. Collectively, this work unveils a novel molecular paradigm that positions BDNF as a core regulator of transcriptional and phosphorylation programs driving axonal regeneration and transport in human motor neurons. HIGHLIGHTSO_LISLAM-seq reveals increased global transcriptional rate by BDNF in human motor neurons C_LIO_LIBDNF modulates transcriptional programs that potentiate axonal outgrowth/regeneration C_LIO_LIBDNF governs the phosphorylation landscape of cytoskeletal-binding proteins C_LIO_LIAxonal BDNF-ERK1/2 signaling controls axonal transport and axonal regeneration C_LI\nEvolutionary origin and structural ligand mimicry by the inserted domain of alpha-integrin proteins\nAuthors: Hollis, J. A.; Chan, M. C.; Malik, H. S.; Campbell, M. G.\nScore: 16.5, Published: 2023-11-06 DOI: 10.1101/2023.11.05.565221\nHeterodimeric integrin proteins transmit signals through conformational changes upon ligand binding between their alpha () and beta ({beta}) subunits. Early in chordate evolution, some subunits acquired an \"inserted\" (I) domain, which expanded their ligand binding capacity but simultaneously obstructed the ancestral ligand-binding pocket. While this would seemingly impede conventional ligand-mediated integrin activation, it was proposed that the I domain itself could serve both as a ligand replacement and an activation trigger. Here, we provide compelling evidence in support of this longstanding hypothesis using high-resolution cryo-electron microscopy structures of two distinct integrin complexes: the ligand-free and E-cadherin-bound states of the E{beta}7 integrin with the I domain, as well as the 4{beta}7 integrin lacking the I domain in both a ligand-free state and bound to MadCAM-1. We trace the evolutionary origin of the I domain to an ancestral collagen-collagen interaction domain. Our analyses illuminate how the I domain intrinsically mimics an extrinsic ligand, enabling integrins to undergo the canonical allosteric cascade of conformational activation and dramatically expanding the range of cellular communication mechanisms in vertebrates.\nMime-seq 2.0: a method to sequence microRNAs from specific mouse cell types\nAuthors: Mandlbauer, A.; Sun, Q.; Popitsch, N.; Schwickert, T.; Spanova, M.; Wang, J.; Ameres, S. L.; Busslinger, M.; Cochella, L.\nScore: 16.7, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565867\nMany microRNAs (miRNAs) are expressed with high spatiotemporal specificity during organismal development, with some being limited to rare cell types, often embedded in complex tissues. Yet most miRNA profiling efforts remain at the tissue and organ levels. To overcome challenges in accessing the microRNomes from tissue-embedded cells, we had previously developed mime-seq (miRNome by methylation dependent sequencing), a technique in which cell-specific miRNA methylation in C. elegans and Drosophila enabled chemo-selective sequencing without the need for cell sorting or biochemical purification. Here, we present mime-seq 2.0 for profiling miRNAs from specific mouse cell types. We engineered a chimeric RNA methyltransferase that is tethered to Argonaute and efficiently methylates miRNAs at their 3-terminal 2OH in mouse and human cell lines. We also generated a transgenic mouse for conditional expression of this methyltransferase, which can be used to direct methylation of miRNAs in a cell-type of choice. We validated the use of this mouse by profiling miRNAs from B cells and bone marrow plasma cells.\nCohesin complex oligomerization maintains end-tethering at DNA double-strand breaks\nAuthors: Phipps, J.; Toulouze, M.; Ducrot, C.; Costa, R.; Brocas, C.; Dubrana, K.\nScore: 14.2, Published: 2023-11-08 DOI: 10.1101/2023.11.08.566226\nDNA double-strand breaks (DSB) must be repaired to ensure genome stability. Crucially, DSB ends must be kept together for timely repair. In Saccharomyces cerevisiae, two poorly understood pathways mediate DSB end-tethering. One employs the Mre11-Rad50-Xrs2 (MRX) complex to physically bridge DSB ends. Another requires the conversion of DSB ends into single-strand DNA (ssDNA) by Exo1, but the bridging proteins are unknown. We uncover that cohesin, its loader and Smc5/6 act with Exo1 to tether DSB ends. Remarkably, cohesin specifically impaired in oligomerization fails to tether DSB ends, revealing a new function for cohesin oligomerization. In addition to the known importance of sister chromatid cohesion, microscopy-based microfluidic experiments unveil a new role for cohesin in repair by ensuring DSB end-tethering. Altogether, our findings demonstrate that oligomerization of cohesin prevents DSB end separation and promotes DSB repair, revealing a novel mode of action and role for cohesin in safeguarding genome integrity.\nThe modified RNA base acp3U is an attachment site for N-glycans in glycoRNA\nAuthors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.\nScore: 118.5, Published: 2023-11-10 DOI: 10.1101/2023.11.06.565735\nWe recently identified glycoRNA--a previously undescribed glycoconjugate--which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described. Here we develop a sensitive and scalable protocol to detect and characterize native glycoRNAs. Leveraging periodate oxidation and aldehyde ligation (rPAL) and Sequential Window Acquisition of all Theoretical Mass Spectra (SWATH-MS), we identified the modified RNA base 3-(3-amino-3-carboxypropyl)uridine (acp3U) as a site of attachment of N-glycans in glycoRNA. The sensitivity and robustness of rPAL provided the first evidence of a direct glycan-RNA linkage, and its flexibility will enable further characterization of glycoRNA biology.\nSterility-Independent Enhancement of Proteasome Function via Floxuridine-Triggered Detoxification in C. elegans\nAuthors: Dubey, A. A.; Szulc, N. A.; Piechota, M.; Serwa, R. A.; Pokrzywa, W.\nScore: 7.4, Published: 2023-11-12 DOI: 10.1101/2023.11.11.566706\nThe ubiquitin-proteasome system (UPS) functionality is vital for proteostasis, contributing to stress resilience, lifespan, and thermal adaptability. In Caenorhabditis elegans, proteasome constituents such as the RPN-6 and PBS-6 subunits or the PSME-3 activator are respectively linked to heat resistance, longevity at moderate cold (15{degrees}C), and survival at low temperatures (4{degrees}C). Since the inhibition of germline stem cells proliferation is associated with robust proteostasis in worms, we utilized floxuridine (FUdR), a compound known for inducing sterility, to examine whether it could reinforce UPS under proteasome dysfunction, particularly to foster cold survival. We demonstrate that FUdR promotes proteasome resilience during its inhibition or subunit deficits, supporting normal lifespan and facilitating adaptation to cold. FUdRs elevation of the UPS activity occurs independently of main proteostasis regulators and is partly driven by SKN-1-regulated transcription, especially under reduced proteasome function. Additionally, we uncover a FUdR-stimulated detoxification pathway, distinct from both SKN-1 and the germline, with GST-24 emerging as a critical mediator of the UPS buffering. This research underscores FUdRs role in the UPS modulation and its contribution to survival of worms in low-temperature conditions. HIGHLIGHTSO_LIFloxuridine (FUdR) enhances ubiquitin-proteasome system activity in C. elegans, independent of primary proteostasis regulators. C_LIO_LIFUdR permits worms to maintain a normal lifespan and facilitates their adaptation to cold in the context of proteasome deficits. C_LIO_LIActing independently of the germline and SKN-1, FUdR triggers a detoxification pathway, with GST-24 as a pivotal component in modulating the ubiquitin-proteasome system. C_LI\nAblation of telomerase reverse transcriptase in Leishmania major results in a senescent-like phenotype and loss of infectivity\nAuthors: Shiburah, M. E.; De Oliveira, B. C. D.; Endalamaw, H. B.; Silva, D. A.; Assis, L. H. d. C.; Barreto, R. M.; Batista, M. M.; Soeiro, M. d. N. C.; Menozzi, B. D.; Langoni, H.; Aoki, J. I.; Coelho, A. C.; CANO, M. I. N.\nScore: 7.0, Published: 2023-11-10 DOI: 10.1101/2023.11.10.566596\nThe lack of efficient human vaccines and effective nontoxic drugs for leishmaniasis necessitates a search for new therapeutic targets. The telomere environment could provide potential targets against leishmaniasis. TERT, the telomerase reverse transcriptase component, has been on the radar for new therapeutic options against several diseases for more than two decades. In this study, we constructed a full deletion (LmTERT-/-) and an ORF disruption (LmN420) of the gene encoding the TERT component of Leishmania major. LmTERT-/- and LmN420 parasites showed replicative and proliferative defects, growth impairment, cell cycle alterations, increased DNA damage, and progressive telomere shortening. Blockage of parasite altruism and the presence of autophagosomes characteristic of a senescent-like phenotype were also detected. LmTERT-/- and LmN420 parasites caused either micro lesion development or no visible lesions in mouse footpads and reduced infectivity in macrophages. While our checks to see if telomere erosion had reached the SCG genes involved in lipophosphoglycan modification showed no changes, our proteomic assessment revealed a downregulation of a metacyclic-associated protein. Complementation of the knockout lineages using the WT LmTERT restored some of the lost phenotypes. Therefore, we speculate that the pleiotropic effects of the loss of LmTERT advance the case for using it as a drug target against the parasite.\n",
  "wordCount" : "2227",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566801" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566801">
        <p class="paperTitle">De novo basecalling of m6A modifications at single molecule and single nucleotide resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cruciani, S.; Delgado Tejedor, A.; Pryszcz, L.; Medina, R.; Llovera, L.; Novoa, E. M.</p>
        <p class="info">Score: 45.6, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566801' target='https://doi.org/10.1101/2023.11.13.566801'> 10.1101/2023.11.13.566801</a></p>
        <p class="abstract">RNA modifications hold pivotal roles in shaping the fate and function of RNA molecules. Although nanopore sequencing technologies have proven successful at transcriptome-wide detection of RNA modifications, current algorithms are limited to predicting modifications at a per-site level rather than within individual RNA molecules. Herein, we introduce m6ABasecaller, an innovative method enabling direct basecalling of m6A modifications from raw nanopore signals within individual RNA molecules. This approach facilitates de novo prediction of m6A modifications with precision down to the single nucleotide and single molecule levels, without the need of paired knockout or control conditions. Using the m6ABasecaller, we find that the median transcriptome-wide m6A modification stoichiometry is ~10-15% in human, mouse and zebrafish. Furthermore, we show that m6A modifications affect polyA tail lengths, exhibit a propensity for co-occurrence within the same RNA molecules, and show relatively consistent stoichiometry levels across isoforms. We further validate the m6ABasecaller by treating mESC with increasing concentrations of STM2457, a METTL3 inhibitor as well as in inducible METTL3 knockout systems. Overall, this work demonstrates the feasibility de novo basecalling of m6A modifications, opening novel avenues for the application of nanopore sequencing to samples with limited RNA availability and for which control knockout conditions are unavailable, such as patient-derived samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566434">
        <p class="paperTitle">NaP-TRAP, a novel massively parallel reporter assay to quantify translation control.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strayer, E. C.; Krishna, S.; Lee, H.; Vejnar, C.; Beaudoin, J.-D.; Giraldez, A. J.</p>
        <p class="info">Score: 21.2, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566434' target='https://doi.org/10.1101/2023.11.09.566434'> 10.1101/2023.11.09.566434</a></p>
        <p class="abstract">The cis-regulatory elements encoded in a mRNA determine its stability and translational output. While there has been a considerable effort to understand the factors driving mRNA stability, the regulatory frameworks governing translational control remain elusive. We have developed a novel massively parallel reporter assay (MPRA) to measure mRNA translation, Nascent Peptide Translating Ribosome Affinity Purification (NaP-TRAP). NaP-TRAP measures translation in a frame specific manner through the immunocapture of epitope tagged nascent peptides of reporter mRNAs. In contrast to existing MPRA methods, NaP-TRAP does not require specialized equipment and is readily adaptable to steady-state and dynamic model systems. We have employed NaP-TRAP to quantify Kozak strength and the regulatory landscapes of 5 UTRs in the developing zebrafish embryo and in human cells, characterizing general and developmentally dynamic cis-regulatory elements. To this end, we identify U-rich motifs as general enhancers, and upstream ORFs and GC-rich motifs as global repressors of translation. We also observe a translational switch during the maternal-to-zygotic transition, where C-rich motifs shift from repressors to prominent activators of translation. Conversely, we show that microRNA sites in the 5 UTR repress translation following the zygotic expression of miR-430. Together these results demonstrate that NaP-TRAP is a versatile, accessible, and powerful method to decode the regulatory functions of UTRs across different systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566739">
        <p class="paperTitle">A synthetic CRISPR-Cas nuclease with expanded enzymatic activities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jabalera, Y.; Tascon, I.; Samperio, S.; Lopez-Alonso, J. P.; Gonzalez-Lopez, M.; Aransay, A. M.; Abascal-Palacios, G.; Ubarretxena-Belandia, I.; Perez-Jimenez, R.</p>
        <p class="info">Score: 20.1, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566739' target='https://doi.org/10.1101/2023.11.12.566739'> 10.1101/2023.11.12.566739</a></p>
        <p class="abstract">Clustered regularly interspaced short palindromic repeats (CRISPR)-associated endonucleases have revolutionized biotechnology for their potential as programmable genome editors. Yet, most natural nucleases and their variants have limitations. Here, we report a fully synthetic CRISPR-associated (Cas) nuclease (-synCas) designed by Ancestral Sequence Reconstruction (ASR) that displays a set of robust and distinct targeting properties, not found in any other known CRISPR-Cas Class 2 system. We show that -synCas is a PAMless nuclease able to catalyse RNA-guided, specific cleavage of dsDNA, ssDNA and ssRNA. The synthetic enzyme is also capable of sequence-nonspecific degradation of dsDNA, ssDNA and ssRNA following activation by complementary dsDNA, ssDNA and ssRNA targets. Furthermore, -synCas exhibits a robust genome editing activity in human cells and bacteria. Cryo-electron microscopy structures of -synCas ternary and quaternary complexes provide a framework to understand the structural basis for its expanded enzymatic activities. The capability for programmable multimodal targeting of virtually any nucleic acid sequence distinguishes -synCas as a promising new tool to extend current CRISPR-based technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565775">
        <p class="paperTitle">BDNF controls phosphorylation and transcriptional networks governing cytoskeleton organization and axonal regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vargas, J. N. S.; Brown, A.-L.; Sun, K.; Hagemann, C.; Geary, B.; Villarroel-Campos, D.; Bryce-Smith, S.; Zanovello, M.; Lombardo, M.; Ryandov, E.; Stepic, A.; Meysman, L.; Majewski, S.; Tosolini, A.; Secrier, M.; Keuss, M. J.; Serio, A.; Sleigh, J. N.; Fratta, P.; Schiavo, G.</p>
        <p class="info">Score: 18.3, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565775' target='https://doi.org/10.1101/2023.11.06.565775'> 10.1101/2023.11.06.565775</a></p>
        <p class="abstract">Axonal degeneration underlies neuromuscular disorders and neuropathies. Dysregulation of neurotrophic factors, such as brain-derived neurotrophic factor (BDNF), in the peripheral nervous system has long been established to exacerbate axonopathy. However, the molecular programs controlled by BDNF that facilitate axonal regeneration and transport are not well-understood. Here, we unravel the transcriptomic and phosphorylation landscape shaped by BDNF in human iPSC-derived motor neurons. Using SLAM-Seq, we reveal BDNF stimulation increases global transcription rate in motor neurons and governs gene regulatory networks that converge with those engaged during axonal repair/outgrowth. Phosphoproteomic analyses demonstrate that BDNF remodels the phosphorylation landscape of cytoskeletal-binding proteins, especially of structural microtubule-associated proteins. Importantly, the localized axonal-specific activation of ERK1/2 is necessary for BDNF to enhance axonal transport of neurotrophin-containing signaling endosomes and to potentiate axonal regeneration after axotomy. Collectively, this work unveils a novel molecular paradigm that positions BDNF as a core regulator of transcriptional and phosphorylation programs driving axonal regeneration and transport in human motor neurons.

HIGHLIGHTSO_LISLAM-seq reveals increased global transcriptional rate by BDNF in human motor neurons
C_LIO_LIBDNF modulates transcriptional programs that potentiate axonal outgrowth/regeneration
C_LIO_LIBDNF governs the phosphorylation landscape of cytoskeletal-binding proteins
C_LIO_LIAxonal BDNF-ERK1/2 signaling controls axonal transport and axonal regeneration
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.05.565221">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.05.565221" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.05.565221">
        <p class="paperTitle">Evolutionary origin and structural ligand mimicry by the inserted domain of alpha-integrin proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.05.565221" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.05.565221" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hollis, J. A.; Chan, M. C.; Malik, H. S.; Campbell, M. G.</p>
        <p class="info">Score: 16.5, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.05.565221' target='https://doi.org/10.1101/2023.11.05.565221'> 10.1101/2023.11.05.565221</a></p>
        <p class="abstract">Heterodimeric integrin proteins transmit signals through conformational changes upon ligand binding between their alpha () and beta ({beta}) subunits. Early in chordate evolution, some  subunits acquired an &#34;inserted&#34; (I) domain, which expanded their ligand binding capacity but simultaneously obstructed the ancestral ligand-binding pocket. While this would seemingly impede conventional ligand-mediated integrin activation, it was proposed that the I domain itself could serve both as a ligand replacement and an activation trigger. Here, we provide compelling evidence in support of this longstanding hypothesis using high-resolution cryo-electron microscopy structures of two distinct integrin complexes: the ligand-free and E-cadherin-bound states of the E{beta}7 integrin with the I domain, as well as the 4{beta}7 integrin lacking the I domain in both a ligand-free state and bound to MadCAM-1. We trace the evolutionary origin of the I domain to an ancestral collagen-collagen interaction domain. Our analyses illuminate how the I domain intrinsically mimics an extrinsic ligand, enabling integrins to undergo the canonical allosteric cascade of conformational activation and dramatically expanding the range of cellular communication mechanisms in vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565867">
        <p class="paperTitle">Mime-seq 2.0: a method to sequence microRNAs from specific mouse cell types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mandlbauer, A.; Sun, Q.; Popitsch, N.; Schwickert, T.; Spanova, M.; Wang, J.; Ameres, S. L.; Busslinger, M.; Cochella, L.</p>
        <p class="info">Score: 16.7, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565867' target='https://doi.org/10.1101/2023.11.06.565867'> 10.1101/2023.11.06.565867</a></p>
        <p class="abstract">Many microRNAs (miRNAs) are expressed with high spatiotemporal specificity during organismal development, with some being limited to rare cell types, often embedded in complex tissues. Yet most miRNA profiling efforts remain at the tissue and organ levels. To overcome challenges in accessing the microRNomes from tissue-embedded cells, we had previously developed mime-seq (miRNome by methylation dependent sequencing), a technique in which cell-specific miRNA methylation in C. elegans and Drosophila enabled chemo-selective sequencing without the need for cell sorting or biochemical purification. Here, we present mime-seq 2.0 for profiling miRNAs from specific mouse cell types. We engineered a chimeric RNA methyltransferase that is tethered to Argonaute and efficiently methylates miRNAs at their 3-terminal 2OH in mouse and human cell lines. We also generated a transgenic mouse for conditional expression of this methyltransferase, which can be used to direct methylation of miRNAs in a cell-type of choice. We validated the use of this mouse by profiling miRNAs from B cells and bone marrow plasma cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566226">
        <p class="paperTitle">Cohesin complex oligomerization maintains end-tethering at DNA double-strand breaks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phipps, J.; Toulouze, M.; Ducrot, C.; Costa, R.; Brocas, C.; Dubrana, K.</p>
        <p class="info">Score: 14.2, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566226' target='https://doi.org/10.1101/2023.11.08.566226'> 10.1101/2023.11.08.566226</a></p>
        <p class="abstract">DNA double-strand breaks (DSB) must be repaired to ensure genome stability. Crucially, DSB ends must be kept together for timely repair. In Saccharomyces cerevisiae, two poorly understood pathways mediate DSB end-tethering. One employs the Mre11-Rad50-Xrs2 (MRX) complex to physically bridge DSB ends. Another requires the conversion of DSB ends into single-strand DNA (ssDNA) by Exo1, but the bridging proteins are unknown. We uncover that cohesin, its loader and Smc5/6 act with Exo1 to tether DSB ends. Remarkably, cohesin specifically impaired in oligomerization fails to tether DSB ends, revealing a new function for cohesin oligomerization. In addition to the known importance of sister chromatid cohesion, microscopy-based microfluidic experiments unveil a new role for cohesin in repair by ensuring DSB end-tethering. Altogether, our findings demonstrate that oligomerization of cohesin prevents DSB end separation and promotes DSB repair, revealing a novel mode of action and role for cohesin in safeguarding genome integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565735">
        <p class="paperTitle">The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xie, Y.; Hemberger, H.; Till, N. A.; Chai, P.; Watkins, C.; Lebedenko, C. G.; Caldwell, R. M.; George, B.; Bertozzi, C. R.; Garcia, B. A.; Flynn, R. A.</p>
        <p class="info">Score: 118.5, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565735' target='https://doi.org/10.1101/2023.11.06.565735'> 10.1101/2023.11.06.565735</a></p>
        <p class="abstract">We recently identified glycoRNA--a previously undescribed glycoconjugate--which consists of RNAs modified with secretory N-glycans and presented on the cell surface. While previous work supported a covalent linkage between RNA and glycans, the direct chemical nature of the RNA-glycan connection was not described. Here we develop a sensitive and scalable protocol to detect and characterize native glycoRNAs. Leveraging periodate oxidation and aldehyde ligation (rPAL) and Sequential Window Acquisition of all Theoretical Mass Spectra (SWATH-MS), we identified the modified RNA base 3-(3-amino-3-carboxypropyl)uridine (acp3U) as a site of attachment of N-glycans in glycoRNA. The sensitivity and robustness of rPAL provided the first evidence of a direct glycan-RNA linkage, and its flexibility will enable further characterization of glycoRNA biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566706">
        <p class="paperTitle">Sterility-Independent Enhancement of Proteasome Function via Floxuridine-Triggered Detoxification in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dubey, A. A.; Szulc, N. A.; Piechota, M.; Serwa, R. A.; Pokrzywa, W.</p>
        <p class="info">Score: 7.4, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566706' target='https://doi.org/10.1101/2023.11.11.566706'> 10.1101/2023.11.11.566706</a></p>
        <p class="abstract">The ubiquitin-proteasome system (UPS) functionality is vital for proteostasis, contributing to stress resilience, lifespan, and thermal adaptability. In Caenorhabditis elegans, proteasome constituents such as the RPN-6 and PBS-6 subunits or the PSME-3 activator are respectively linked to heat resistance, longevity at moderate cold (15{degrees}C), and survival at low temperatures (4{degrees}C). Since the inhibition of germline stem cells proliferation is associated with robust proteostasis in worms, we utilized floxuridine (FUdR), a compound known for inducing sterility, to examine whether it could reinforce UPS under proteasome dysfunction, particularly to foster cold survival. We demonstrate that FUdR promotes proteasome resilience during its inhibition or subunit deficits, supporting normal lifespan and facilitating adaptation to cold. FUdRs elevation of the UPS activity occurs independently of main proteostasis regulators and is partly driven by SKN-1-regulated transcription, especially under reduced proteasome function. Additionally, we uncover a FUdR-stimulated detoxification pathway, distinct from both SKN-1 and the germline, with GST-24 emerging as a critical mediator of the UPS buffering. This research underscores FUdRs role in the UPS modulation and its contribution to survival of worms in low-temperature conditions.

HIGHLIGHTSO_LIFloxuridine (FUdR) enhances ubiquitin-proteasome system activity in C. elegans, independent of primary proteostasis regulators.
C_LIO_LIFUdR permits worms to maintain a normal lifespan and facilitates their adaptation to cold in the context of proteasome deficits.
C_LIO_LIActing independently of the germline and SKN-1, FUdR triggers a detoxification pathway, with GST-24 as a pivotal component in modulating the ubiquitin-proteasome system.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566596">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566596" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566596">
        <p class="paperTitle">Ablation of telomerase reverse transcriptase in Leishmania major results in a senescent-like phenotype and loss of infectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566596" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566596" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiburah, M. E.; De Oliveira, B. C. D.; Endalamaw, H. B.; Silva, D. A.; Assis, L. H. d. C.; Barreto, R. M.; Batista, M. M.; Soeiro, M. d. N. C.; Menozzi, B. D.; Langoni, H.; Aoki, J. I.; Coelho, A. C.; CANO, M. I. N.</p>
        <p class="info">Score: 7.0, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566596' target='https://doi.org/10.1101/2023.11.10.566596'> 10.1101/2023.11.10.566596</a></p>
        <p class="abstract">The lack of efficient human vaccines and effective nontoxic drugs for leishmaniasis necessitates a search for new therapeutic targets. The telomere environment could provide potential targets against leishmaniasis. TERT, the telomerase reverse transcriptase component, has been on the radar for new therapeutic options against several diseases for more than two decades. In this study, we constructed a full deletion (LmTERT-/-) and an ORF disruption (LmN420) of the gene encoding the TERT component of Leishmania major. LmTERT-/- and LmN420 parasites showed replicative and proliferative defects, growth impairment, cell cycle alterations, increased DNA damage, and progressive telomere shortening. Blockage of parasite altruism and the presence of autophagosomes characteristic of a senescent-like phenotype were also detected. LmTERT-/- and LmN420 parasites caused either micro lesion development or no visible lesions in mouse footpads and reduced infectivity in macrophages. While our checks to see if telomere erosion had reached the SCG genes involved in lipophosphoglycan modification showed no changes, our proteomic assessment revealed a downregulation of a metacyclic-associated protein. Complementation of the knockout lineages using the WT LmTERT restored some of the lost phenotypes. Therefore, we speculate that the pleiotropic effects of the loss of LmTERT advance the case for using it as a drug target against the parasite.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
