<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A universal molecular mechanism driving aging
Authors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.
Score: 43.3, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574476
How cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="A universal molecular mechanism driving aging
Authors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.
Score: 43.3, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574476
How cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="A universal molecular mechanism driving aging
Authors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.
Score: 43.3, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574476
How cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "A universal molecular mechanism driving aging\nAuthors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.\nScore: 43.3, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574476\nHow cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin.",
  "keywords": [
    
  ],
  "articleBody": " A universal molecular mechanism driving aging\nAuthors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.\nScore: 43.3, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574476\nHow cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin. This creates a more permissive local environment for G4 formation in subsequent generations. As a result, G4s gradually accumulate on promoters throughout mitosis, driving clock-like DNA hypomethylation and chromatin opening. In patients and in vitro models, loss-of-function mutations in the G4-resolving enzymes WRN, BLM and ERCC8 accelerate the erosion of the epigenomic landscape around G4. G4-driven epigenomic aging is strongly correlated with biological age and is conserved in yeast, nematodes, insects, fish, rodents, and humans. Our results revealed a universal molecular mechanism of aging and provided mechanistic insight into how G-quadruplex processor mutations drive premature aging.\nCAP peptide artificially induces insect gall\nAuthors: Sakamoto, T.; Kimura, S.; Nakayama, T.; Sato, M. P.; Shirasawa, K.; Sato, M. H.; Hirano, T.\nScore: 14.3, Published: 2024-01-07 DOI: 10.1101/2024.01.06.574462\nGalls caused by gall-inducing insects in their host plants clearly illustrate the concept of extended phenotype, which refers to traits expressed in a host organism when manipulated by a parasite. Candidate effector molecules involved in gall formation, such as phytohormones, amino acids, and proteins, have been reported in numerous studies. However, to date, no attempts to artificially regenerate gall structures using effector candidates have been reported. In this study, we tested the peptide from Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 proteins, CAP peptide as a gall-inducing effector candidate obtained from transcripts isolated from the horned gall aphid, (Schlechtendalia chinensis) through in silico screening and the Arabidopsis-based gall-forming assay, which is a bioassay system for analysing the molecular mechanisms of gall formation. Furthermore, we succeeded in generating an artificial gall in the host plant Veronica peregrina, without any insect parasitism, using three minimal effector elements: CAP peptide, auxin, and cytokinin. Given the strong similarities observed in organ structure with a central cavity and three types of tissue and gene expression patterns between the native and artificial galls, we concluded that CAP peptide is a general gall-inducing effector peptide secreted by gall-inducing insects.\nGenome - nuclear lamina interactions are multivalent and cooperative\nAuthors: Dauban, L.; Eder, M.; de Haas, M.; Franceschini-Santos, V. H.; Yanez-Cuna, J. O.; van Schaik, T.; Leemans, C.; Rademaker, K.; Martinez-Ara, M.; Martinovic, M.; de Wit, E.; van Steensel, B.\nScore: 12.7, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574825\nLamina-associated domains (LADs) are megabase-sized genomic regions that interact with the nuclear lamina (NL). It is not yet understood how their interactions with the NL are encoded in their DNA. Here, we designed an efficient LAD \"scrambling\" approach, based on transposon-mediated local hopping of loxP recombination sites, to generate series of large deletions and inversions that span LADs and flanking sequences. Mapping of NL interactions in these rearrangements revealed that a single LAD contacts the NL through multiple regions that act cooperatively or redundantly; some have more affinity for the NL than others and can pull neighbouring sequences to the NL. Genes drawn towards the NL showed often, but not always, reduced expression and increased H3K9me3 levels. Furthermore, neighbouring LADs can cooperatively interact with the NL when placed close enough to each other. These results elucidate principles that govern the positioning of megabase-sized genomic regions inside the cell nucleus. HighlightsO_LIEfficient generation of LAD rearrangements by transposon hopping of loxP elements. C_LIO_LINL interactions are multivalent, with some subregions being more potent tethers than others. C_LIO_LILADs can cooperate to promote their association to the NL. C_LIO_LIChanges in NL association are often accompanied by changes in gene activity and H3K9me3. C_LI\nBrain and cancer associated binding domain mutations provide insight into CTCF's relationship with chromatin and its ability to act as a chromatin organizer\nAuthors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.\nScore: 12.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070\nAlthough only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown. To investigate the relationship between CTCF and chromatin we applied a combination of imaging, structural and molecular approaches, using a series of brain and cancer associated CTCF mutations that act as CTCF perturbations. We demonstrate that binding and the functional impact of WT and mutant CTCF depend not only on the unique binding properties of each protein, but also on the genomic context of bound sites and enrichment of motifs for expressed TFs abutting these sites. Our studies also highlight the reciprocal relationship between CTCF and chromatin, demonstrating that the unique binding properties of WT and mutant proteins have a distinct impact on accessibility, TF binding, cohesin overlap, chromatin interactivity and gene expression programs, providing insight into their cancer and brain related effects.\nA dual histone code specifies the binding of heterochromatin protein Rhino to a subset of piRNA source loci\nAuthors: Akkouche, A.; Kneuss, E.; Bornelöv, S.; Renaud, Y.; Eastwood, E. L.; van Lopik, J.; Gueguen, N.; Jiang, M.; Creixell, P.; Maupetit-Mehouas, S.; Nicholson, B. C.; Brasset, E.; Hannon, G. J.\nScore: 11.9, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575256\nAnimal germ cells deploy a specialized small RNA-based silencing system, called the PIWI-interacting RNA (piRNA) pathway, to prevent unwanted expression of transposable elements and maintain genome integrity. In Drosophila germ cells, the majority of piRNA populations originate from dual-strand piRNA clusters, genomic regions highly enriched in transposon fragments, via an elaborate protein machinery centred on the heterochromatin protein 1 homolog, Rhino. Although Rhino binds to peptides carrying trimethylated H3K9 in vitro, it is not fully understood why in vivo only a fraction of H3K9me3-decorated heterochromatin is occupied by Rhino. Recent work uncovered that Rhino is recruited to a subset of piRNA clusters by the zinc finger protein Kipferl. Here we identify a Kipferl-independent mode of Rhino targeting that is dependent on the histone H3 lysine 27 methyltransferase Enhancer of Zeste and the presence of H3K9me3 and H3K27me3 marks. At Kipferl-independent sites, we find that Rhino, through its dimeric chromodomain, specifically binds to loci marked by both H3K9me3 and H3K27me3. These results expand our understanding of the characteristic binding profile of the heterochromatin protein Rhino. Our work reveals a role for dual histone modifications in defining the binding specificity of a chromatin protein.\nA CPC-shelterin-BTR axis regulates mitotic telomere deprotection\nAuthors: Romero-Zamora, D.; Rogers, S.; Low, R. R. J.; Robinson, A. B.; Page, S. G.; Lane, B. J. E.; Lamm, N.; Ishikawa, F.; Hayashi, M. T.; Cesare, A. J.\nScore: 9.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574754\nTelomeres prevent ATM activation by sequestering chromosome termini within telomere loops (t-loops). Mitotic arrest promotes telomere linearity and a localized ATM-dependent telomere DNA damage response (DDR) through an unknown mechanism. Using unbiased interactomics, biochemical screening, molecular biology, and super-resolution imaging, we found that mitotic arrest-dependent (MAD) telomere deprotection requires the combined activities of the Chromosome passenger complex (CPC) on shelterin, and the BLM-TOP3A-RMI1/2 (BTR) complex on t-loops. During mitotic arrest, the CPC component Aurora Kinase B (AURKB) phosphorylated both the TRF1 hinge and TRF2 basic domains. The former enhanced CPC and TRF1 interaction through the CPC Survivin subunit, while the latter promoted telomere linearity, telomere DDR activation dependent upon BTR double Holliday junction dissolution activity, and mitotic death. We identify that the TRF2 basic domain functions in mitosis-specific telomere protection and reveal TRF1 regulation over a physiological ATM-dependent telomere DDR. The data demonstrate that MAD telomere deprotection is a sophisticated active mechanism that exposes telomere ends to signal mitotic stress.\nOptimization of Anticodon Edited Transfer RNAs (ACE-tRNAs) Function in Translation for Suppression of Nonsense Mutations\nAuthors: Porter, J. J.; Ko, W.; Sorensen, E. G.; Lueck, J. D.\nScore: 5.3, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575297\nNonsense suppressor tRNAs or AntiCodon-Edited tRNAs (ACE-tRNAs) have long been envisioned as a therapeutic approach to overcome genetic diseases resulting from the introduction of premature stop codons (PTCs). The ACE-tRNA approach for the rescue of PTCs has been hampered by ineffective delivery through available modalities for gene therapy. Here we have screened a series of ACE-tRNA expression cassette sequence libraries containing \u003e1,800 members to optimize ACE-tRNA function and provide a roadmap for optimization in the future. By optimizing PTC suppression efficiency of ACE-tRNAs, we have decreased the amount of ACE-tRNA required by ~16-fold for the most common cystic fibrosis-causing PTCs.\nThe V-ATPase/ATG16L1 axis is controlled by the V1H subunit\nAuthors: Timimi, L.; Wrobel, A. G.; Chiduza, G. N.; Maslen, S. L.; Torres-Mendez, A.; Montaner, B.; Davis, C.; Skehel, M.; Rubinstein, J. L.; Schreiber, A. L.; Beale, R.\nScore: 13.9, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572309\nDefects in organellar acidification indicate compromised or infected compartments. Recruitment of the autophagy-related ATG16L1 complex to pathologically de-acidified compartments targets ubiquitin-like ATG8 molecules to perturbed membranes. How this process is coupled to pH gradient disruption is unclear. Here, we reveal a direct role for the V1H subunit of the V-ATPase proton pump in recruiting ATG16L1. The interaction between V1H and ATG16L1 occurs within assembled V-ATPases, but not dissociated V1 complexes. This selectivity allows recruitment to be coupled to changes in V-ATPase assembly that follow pH dissipation. Cells lacking V1H undergo canonical macroautophagy but are unable to recruit ATG16L1 in response to influenza infection or ionophore drugs. We identify a loop within V1H that mediates ATG16L1 binding, which is absent in a neuronal isoform of V1H. Thus, V1H controls ATG16L1 recruitment in response to proton gradient dissipation, suggesting that the V-ATPase acts autonomously as a cell-intrinsic damage sensor.\nProteome profiling of cutaneous leishmaniasis lesions due to dermotropic Leishmania donovani in Sri Lanka\nAuthors: Manamperi, N. H.; Edirisinghe, N. M.; Wijesinghe, H.; Pathiraja, L.; Pathirana, N.; Wanasinghe, V. S.; De Silva, C. G.; Abeyewickreme, W.; Karunaweera, N. D.\nScore: 2.8, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574579\nCharacterization of the host response in cutaneous leishmaniasis (CL) through proteome profiling has gained limited insights in leishmaniasis research, in comparison to that of the parasite. The primary objective of this study was to comprehensively analyze the proteomic profile of the skin lesions tissues in patients with CL, by mass spectrometry, and subsequent validation of these findings through immunohistochemical methods. Sixty-seven proteins exhibited significant differential expression between tissues of CL lesions and healthy controls (p\u003c0.01), representing numerous enriched biological processes within the lesion tissue, as evident by both the Kyoto Encyclopedia of Genes and Genomes (KEGG) and Reactome databases. Among these, the integrated endoplasmic reticulum stress response (IERSR) emerges as a pathway characterized by the up-regulated proteins in CL tissues compared to healthy skin. Expression of endoplasmic reticulum (ER) stress sensors, inositol-requiring enzyme-1 (IRE1), protein kinase RNA-like ER kinase (PERK), and activating transcription factor 6 (ATF6) in lesion tissue was validated by immunohistochemistry. In conclusion, proteomic profiling of skin lesions carried out as a discovery phase study revealed a multitude of probable immunological and pathological mechanisms operating in patients with CL in Sri Lanka, which needs to be further elaborated using more in-depth and targeted investigations. Author SummaryCutaneous leishmaniasis (CL), is a skin infection caused by a type of single-celled parasite. These parasites are usually transmitted through the bite of infected sandflies. In Sri Lanka, CL is caused by a parasite type that usually causes a more severe disease form, known as visceral leishmaniasis. Interaction between the parasite and the human host is important in determining the disease outcome and hence, we conducted a study to look at the proteins in the skin lesions of people with CL using a technique called mass spectrometry. We found 67 proteins that were different between CL lesions and healthy skin. These proteins are involved in various processes in the body, and one specific process called the integrated endoplasmic reticulum stress response (IERSR) was more active in CL patients. We confirmed this by studying specific proteins related to stress in the lesion tissue. In conclusion, our study uncovered several potential immune and disease-related mechanisms in CL patients in Sri Lanka. However, more detailed investigations are needed to fully understand these processes.\nFANCJ promotes PARP1 activity during DNA replication that is essential in BRCA1 deficient cells\nAuthors: Cong, K.; MacGilvary, N.; Lee, S.; MacLeod, S.; Calvo, J.; Peng, M.; Nedergaard Kousholt, A.; Day, T.; Cantor, S.\nScore: 13.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574095\nSingle-stranded DNA gaps are postulated to be fundamental to the mechanism of anti-cancer drugs. Gaining insights into their induction could therefore be pivotal for advancing therapeutic strategies. For poly (ADP-ribose) polymerase inhibitors (PARPi) to be effective, the presence of FANCJ helicase is required. However, the relationship between FANCJ dependent gaps and PARP1 catalytic inhibition or trapping--both linked to PARPi toxicity in BRCA deficient cells--is yet to be elucidated. Here, we find that the efficacy of PARPi is contingent on S-phase PARP1 activity, which is compromised in FANCJ deficient cells because PARP1, along with MSH2, is \"sequestered\" by G-quadruplexes. PARP1s replication activity is also diminished in cells missing a FANCJ-MLH1 interaction, but in such cells, depleting MSH2 can release sequestered PARP1, restoring PARPi-induced gaps and sensitivity. Our observations indicate that sequestered and trapped PARP1 are different chromatin-bound forms, with FANCJ loss increasing PARPi resistance in cells susceptible to canonical PARP1 trapping. However, in BRCA1 null cells, the loss of FANCJ mirrors the effects of PARP1 loss or inhibition, with the common detrimental factor being the loss of PARP1 activity during DNA replication, not trapping. These insights underline the crucial role of PARP1 activity during DNA replication in BRCA deficient cells and emphasize the importance of understanding drug mechanisms for enhancing precision medicine.\n",
  "wordCount" : "2305",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574476">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574476" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574476">
        <p class="paperTitle">A universal molecular mechanism driving aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574476" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574476" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.</p>
        <p class="info">Score: 43.3, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574476' target='https://doi.org/10.1101/2024.01.06.574476'> 10.1101/2024.01.06.574476</a></p>
        <p class="abstract">How cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin. This creates a more permissive local environment for G4 formation in subsequent generations. As a result, G4s gradually accumulate on promoters throughout mitosis, driving clock-like DNA hypomethylation and chromatin opening. In patients and in vitro models, loss-of-function mutations in the G4-resolving enzymes WRN, BLM and ERCC8 accelerate the erosion of the epigenomic landscape around G4. G4-driven epigenomic aging is strongly correlated with biological age and is conserved in yeast, nematodes, insects, fish, rodents, and humans. Our results revealed a universal molecular mechanism of aging and provided mechanistic insight into how G-quadruplex processor mutations drive premature aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574462">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574462" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574462">
        <p class="paperTitle">CAP peptide artificially induces insect gall</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574462" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574462" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakamoto, T.; Kimura, S.; Nakayama, T.; Sato, M. P.; Shirasawa, K.; Sato, M. H.; Hirano, T.</p>
        <p class="info">Score: 14.3, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574462' target='https://doi.org/10.1101/2024.01.06.574462'> 10.1101/2024.01.06.574462</a></p>
        <p class="abstract">Galls caused by gall-inducing insects in their host plants clearly illustrate the concept of  extended phenotype, which refers to traits expressed in a host organism when manipulated by a parasite. Candidate effector molecules involved in gall formation, such as phytohormones, amino acids, and proteins, have been reported in numerous studies. However, to date, no attempts to artificially regenerate gall structures using effector candidates have been reported. In this study, we tested the peptide from Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 proteins, CAP peptide as a gall-inducing effector candidate obtained from transcripts isolated from the horned gall aphid, (Schlechtendalia chinensis) through in silico screening and the Arabidopsis-based gall-forming assay, which is a bioassay system for analysing the molecular mechanisms of gall formation. Furthermore, we succeeded in generating an artificial gall in the host plant Veronica peregrina, without any insect parasitism, using three minimal effector elements: CAP peptide, auxin, and cytokinin. Given the strong similarities observed in organ structure with a central cavity and three types of tissue and gene expression patterns between the native and artificial galls, we concluded that CAP peptide is a general gall-inducing effector peptide secreted by gall-inducing insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574825">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574825" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574825">
        <p class="paperTitle">Genome - nuclear lamina interactions are multivalent and cooperative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574825" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574825" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dauban, L.; Eder, M.; de Haas, M.; Franceschini-Santos, V. H.; Yanez-Cuna, J. O.; van Schaik, T.; Leemans, C.; Rademaker, K.; Martinez-Ara, M.; Martinovic, M.; de Wit, E.; van Steensel, B.</p>
        <p class="info">Score: 12.7, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574825' target='https://doi.org/10.1101/2024.01.10.574825'> 10.1101/2024.01.10.574825</a></p>
        <p class="abstract">Lamina-associated domains (LADs) are megabase-sized genomic regions that interact with the nuclear lamina (NL). It is not yet understood how their interactions with the NL are encoded in their DNA. Here, we designed an efficient LAD &#34;scrambling&#34; approach, based on transposon-mediated local hopping of loxP recombination sites, to generate series of large deletions and inversions that span LADs and flanking sequences. Mapping of NL interactions in these rearrangements revealed that a single LAD contacts the NL through multiple regions that act cooperatively or redundantly; some have more affinity for the NL than others and can pull neighbouring sequences to the NL. Genes drawn towards the NL showed often, but not always, reduced expression and increased H3K9me3 levels. Furthermore, neighbouring LADs can cooperatively interact with the NL when placed close enough to each other. These results elucidate principles that govern the positioning of megabase-sized genomic regions inside the cell nucleus.

HighlightsO_LIEfficient generation of LAD rearrangements by transposon hopping of loxP elements.
C_LIO_LINL interactions are multivalent, with some subregions being more potent tethers than others.
C_LIO_LILADs can cooperate to promote their association to the NL.
C_LIO_LIChanges in NL association are often accompanied by changes in gene activity and H3K9me3.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575070" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575070">
        <p class="paperTitle">Brain and cancer associated binding domain mutations provide insight into CTCF&#39;s relationship with chromatin and its ability to act as a chromatin organizer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.</p>
        <p class="info">Score: 12.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575070' target='https://doi.org/10.1101/2024.01.11.575070'> 10.1101/2024.01.11.575070</a></p>
        <p class="abstract">Although only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown. To investigate the relationship between CTCF and chromatin we applied a combination of imaging, structural and molecular approaches, using a series of brain and cancer associated CTCF mutations that act as CTCF perturbations. We demonstrate that binding and the functional impact of WT and mutant CTCF depend not only on the unique binding properties of each protein, but also on the genomic context of bound sites and enrichment of motifs for expressed TFs abutting these sites. Our studies also highlight the reciprocal relationship between CTCF and chromatin, demonstrating that the unique binding properties of WT and mutant proteins have a distinct impact on accessibility, TF binding, cohesin overlap, chromatin interactivity and gene expression programs, providing insight into their cancer and brain related effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575256">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575256" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575256">
        <p class="paperTitle">A dual histone code specifies the binding of heterochromatin protein Rhino to a subset of piRNA source loci</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575256" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575256" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akkouche, A.; Kneuss, E.; Bornelöv, S.; Renaud, Y.; Eastwood, E. L.; van Lopik, J.; Gueguen, N.; Jiang, M.; Creixell, P.; Maupetit-Mehouas, S.; Nicholson, B. C.; Brasset, E.; Hannon, G. J.</p>
        <p class="info">Score: 11.9, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575256' target='https://doi.org/10.1101/2024.01.11.575256'> 10.1101/2024.01.11.575256</a></p>
        <p class="abstract">Animal germ cells deploy a specialized small RNA-based silencing system, called the PIWI-interacting RNA (piRNA) pathway, to prevent unwanted expression of transposable elements and maintain genome integrity. In Drosophila germ cells, the majority of piRNA populations originate from dual-strand piRNA clusters, genomic regions highly enriched in transposon fragments, via an elaborate protein machinery centred on the heterochromatin protein 1 homolog, Rhino. Although Rhino binds to peptides carrying trimethylated H3K9 in vitro, it is not fully understood why in vivo only a fraction of H3K9me3-decorated heterochromatin is occupied by Rhino. Recent work uncovered that Rhino is recruited to a subset of piRNA clusters by the zinc finger protein Kipferl. Here we identify a Kipferl-independent mode of Rhino targeting that is dependent on the histone H3 lysine 27 methyltransferase Enhancer of Zeste and the presence of H3K9me3 and H3K27me3 marks. At Kipferl-independent sites, we find that Rhino, through its dimeric chromodomain, specifically binds to loci marked by both H3K9me3 and H3K27me3. These results expand our understanding of the characteristic binding profile of the heterochromatin protein Rhino. Our work reveals a role for dual histone modifications in defining the binding specificity of a chromatin protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574754" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574754">
        <p class="paperTitle">A CPC-shelterin-BTR axis regulates mitotic telomere deprotection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Romero-Zamora, D.; Rogers, S.; Low, R. R. J.; Robinson, A. B.; Page, S. G.; Lane, B. J. E.; Lamm, N.; Ishikawa, F.; Hayashi, M. T.; Cesare, A. J.</p>
        <p class="info">Score: 9.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574754' target='https://doi.org/10.1101/2024.01.09.574754'> 10.1101/2024.01.09.574754</a></p>
        <p class="abstract">Telomeres prevent ATM activation by sequestering chromosome termini within telomere loops (t-loops). Mitotic arrest promotes telomere linearity and a localized ATM-dependent telomere DNA damage response (DDR) through an unknown mechanism. Using unbiased interactomics, biochemical screening, molecular biology, and super-resolution imaging, we found that mitotic arrest-dependent (MAD) telomere deprotection requires the combined activities of the Chromosome passenger complex (CPC) on shelterin, and the BLM-TOP3A-RMI1/2 (BTR) complex on t-loops. During mitotic arrest, the CPC component Aurora Kinase B (AURKB) phosphorylated both the TRF1 hinge and TRF2 basic domains. The former enhanced CPC and TRF1 interaction through the CPC Survivin subunit, while the latter promoted telomere linearity, telomere DDR activation dependent upon BTR double Holliday junction dissolution activity, and mitotic death. We identify that the TRF2 basic domain functions in mitosis-specific telomere protection and reveal TRF1 regulation over a physiological ATM-dependent telomere DDR. The data demonstrate that MAD telomere deprotection is a sophisticated active mechanism that exposes telomere ends to signal mitotic stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575297" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575297">
        <p class="paperTitle">Optimization of Anticodon Edited Transfer RNAs (ACE-tRNAs) Function in Translation for Suppression of Nonsense Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porter, J. J.; Ko, W.; Sorensen, E. G.; Lueck, J. D.</p>
        <p class="info">Score: 5.3, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575297' target='https://doi.org/10.1101/2024.01.11.575297'> 10.1101/2024.01.11.575297</a></p>
        <p class="abstract">Nonsense suppressor tRNAs or AntiCodon-Edited tRNAs (ACE-tRNAs) have long been envisioned as a therapeutic approach to overcome genetic diseases resulting from the introduction of premature stop codons (PTCs). The ACE-tRNA approach for the rescue of PTCs has been hampered by ineffective delivery through available modalities for gene therapy. Here we have screened a series of ACE-tRNA expression cassette sequence libraries containing &gt;1,800 members to optimize ACE-tRNA function and provide a roadmap for optimization in the future. By optimizing PTC suppression efficiency of ACE-tRNAs, we have decreased the amount of ACE-tRNA required by ~16-fold for the most common cystic fibrosis-causing PTCs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572309" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572309">
        <p class="paperTitle">The V-ATPase/ATG16L1 axis is controlled by the V1H subunit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timimi, L.; Wrobel, A. G.; Chiduza, G. N.; Maslen, S. L.; Torres-Mendez, A.; Montaner, B.; Davis, C.; Skehel, M.; Rubinstein, J. L.; Schreiber, A. L.; Beale, R.</p>
        <p class="info">Score: 13.9, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572309' target='https://doi.org/10.1101/2023.12.19.572309'> 10.1101/2023.12.19.572309</a></p>
        <p class="abstract">Defects in organellar acidification indicate compromised or infected compartments. Recruitment of the autophagy-related ATG16L1 complex to pathologically de-acidified compartments targets ubiquitin-like ATG8 molecules to perturbed membranes. How this process is coupled to pH gradient disruption is unclear. Here, we reveal a direct role for the V1H subunit of the V-ATPase proton pump in recruiting ATG16L1. The interaction between V1H and ATG16L1 occurs within assembled V-ATPases, but not dissociated V1 complexes. This selectivity allows recruitment to be coupled to changes in V-ATPase assembly that follow pH dissipation. Cells lacking V1H undergo canonical macroautophagy but are unable to recruit ATG16L1 in response to influenza infection or ionophore drugs. We identify a loop within V1H that mediates ATG16L1 binding, which is absent in a neuronal isoform of V1H. Thus, V1H controls ATG16L1 recruitment in response to proton gradient dissipation, suggesting that the V-ATPase acts autonomously as a cell-intrinsic damage sensor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574579">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574579" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574579">
        <p class="paperTitle">Proteome profiling of cutaneous leishmaniasis lesions due to dermotropic Leishmania donovani  in Sri Lanka</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574579" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574579" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manamperi, N. H.; Edirisinghe, N. M.; Wijesinghe, H.; Pathiraja, L.; Pathirana, N.; Wanasinghe, V. S.; De Silva, C. G.; Abeyewickreme, W.; Karunaweera, N. D.</p>
        <p class="info">Score: 2.8, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574579' target='https://doi.org/10.1101/2024.01.07.574579'> 10.1101/2024.01.07.574579</a></p>
        <p class="abstract">Characterization of the host response in cutaneous leishmaniasis (CL) through proteome profiling has gained limited insights in leishmaniasis research, in comparison to that of the parasite. The primary objective of this study was to comprehensively analyze the proteomic profile of the skin lesions tissues in patients with CL, by mass spectrometry, and subsequent validation of these findings through immunohistochemical methods. Sixty-seven proteins exhibited significant differential expression between tissues of CL lesions and healthy controls (p&lt;0.01), representing numerous enriched biological processes within the lesion tissue, as evident by both the Kyoto Encyclopedia of Genes and Genomes (KEGG) and Reactome databases. Among these, the integrated endoplasmic reticulum stress response (IERSR) emerges as a pathway characterized by the up-regulated proteins in CL tissues compared to healthy skin. Expression of endoplasmic reticulum (ER) stress sensors, inositol-requiring enzyme-1 (IRE1), protein kinase RNA-like ER kinase (PERK), and activating transcription factor 6 (ATF6) in lesion tissue was validated by immunohistochemistry. In conclusion, proteomic profiling of skin lesions carried out as a discovery phase study revealed a multitude of probable immunological and pathological mechanisms operating in patients with CL in Sri Lanka, which needs to be further elaborated using more in-depth and targeted investigations.

Author SummaryCutaneous leishmaniasis (CL), is a skin infection caused by a type of single-celled parasite. These parasites are usually transmitted through the bite of infected sandflies. In Sri Lanka, CL is caused by a parasite type that usually causes a more severe disease form, known as visceral leishmaniasis. Interaction between the parasite and the human host is important in determining the disease outcome and hence, we conducted a study to look at the proteins in the skin lesions of people with CL using a technique called mass spectrometry. We found 67 proteins that were different between CL lesions and healthy skin. These proteins are involved in various processes in the body, and one specific process called the integrated endoplasmic reticulum stress response (IERSR) was more active in CL patients. We confirmed this by studying specific proteins related to stress in the lesion tissue. In conclusion, our study uncovered several potential immune and disease-related mechanisms in CL patients in Sri Lanka. However, more detailed investigations are needed to fully understand these processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574095" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574095">
        <p class="paperTitle">FANCJ promotes PARP1 activity during DNA replication that is essential in BRCA1 deficient cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cong, K.; MacGilvary, N.; Lee, S.; MacLeod, S.; Calvo, J.; Peng, M.; Nedergaard Kousholt, A.; Day, T.; Cantor, S.</p>
        <p class="info">Score: 13.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574095' target='https://doi.org/10.1101/2024.01.04.574095'> 10.1101/2024.01.04.574095</a></p>
        <p class="abstract">Single-stranded DNA gaps are postulated to be fundamental to the mechanism of anti-cancer drugs. Gaining insights into their induction could therefore be pivotal for advancing therapeutic strategies. For poly (ADP-ribose) polymerase inhibitors (PARPi) to be effective, the presence of FANCJ helicase is required. However, the relationship between FANCJ dependent gaps and PARP1 catalytic inhibition or trapping--both linked to PARPi toxicity in BRCA deficient cells--is yet to be elucidated. Here, we find that the efficacy of PARPi is contingent on S-phase PARP1 activity, which is compromised in FANCJ deficient cells because PARP1, along with MSH2, is &#34;sequestered&#34; by G-quadruplexes. PARP1s replication activity is also diminished in cells missing a FANCJ-MLH1 interaction, but in such cells, depleting MSH2 can release sequestered PARP1, restoring PARPi-induced gaps and sensitivity. Our observations indicate that sequestered and trapped PARP1 are different chromatin-bound forms, with FANCJ loss increasing PARPi resistance in cells susceptible to canonical PARP1 trapping. However, in BRCA1 null cells, the loss of FANCJ mirrors the effects of PARP1 loss or inhibition, with the common detrimental factor being the loss of PARP1 activity during DNA replication, not trapping. These insights underline the crucial role of PARP1 activity during DNA replication in BRCA deficient cells and emphasize the importance of understanding drug mechanisms for enhancing precision medicine.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
