<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="ATAC and SAGA histone acetyltransferase modules facilitate transcription factor binding to nucleosomes in an acetylation independent manner
Authors: Chesnutt, K.; Yayli, G.; Toelzer, C.; Cox, K.; Gautam, G.; Berger, I.; Tora, L.; Poirier, M. G.
Score: 22.9, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564358
Transcription initiation involves the coordination of multiple events, starting with activators binding specific DNA target sequences, which recruits transcription co-activators to open chromatin and enable binding of general transcription factors and RNA polymerase II to promoters.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="ATAC and SAGA histone acetyltransferase modules facilitate transcription factor binding to nucleosomes in an acetylation independent manner
Authors: Chesnutt, K.; Yayli, G.; Toelzer, C.; Cox, K.; Gautam, G.; Berger, I.; Tora, L.; Poirier, M. G.
Score: 22.9, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564358
Transcription initiation involves the coordination of multiple events, starting with activators binding specific DNA target sequences, which recruits transcription co-activators to open chromatin and enable binding of general transcription factors and RNA polymerase II to promoters." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="ATAC and SAGA histone acetyltransferase modules facilitate transcription factor binding to nucleosomes in an acetylation independent manner
Authors: Chesnutt, K.; Yayli, G.; Toelzer, C.; Cox, K.; Gautam, G.; Berger, I.; Tora, L.; Poirier, M. G.
Score: 22.9, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564358
Transcription initiation involves the coordination of multiple events, starting with activators binding specific DNA target sequences, which recruits transcription co-activators to open chromatin and enable binding of general transcription factors and RNA polymerase II to promoters."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "ATAC and SAGA histone acetyltransferase modules facilitate transcription factor binding to nucleosomes in an acetylation independent manner\nAuthors: Chesnutt, K.; Yayli, G.; Toelzer, C.; Cox, K.; Gautam, G.; Berger, I.; Tora, L.; Poirier, M. G.\nScore: 22.9, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564358\nTranscription initiation involves the coordination of multiple events, starting with activators binding specific DNA target sequences, which recruits transcription co-activators to open chromatin and enable binding of general transcription factors and RNA polymerase II to promoters.",
  "keywords": [
    
  ],
  "articleBody": " ATAC and SAGA histone acetyltransferase modules facilitate transcription factor binding to nucleosomes in an acetylation independent manner\nAuthors: Chesnutt, K.; Yayli, G.; Toelzer, C.; Cox, K.; Gautam, G.; Berger, I.; Tora, L.; Poirier, M. G.\nScore: 22.9, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564358\nTranscription initiation involves the coordination of multiple events, starting with activators binding specific DNA target sequences, which recruits transcription co-activators to open chromatin and enable binding of general transcription factors and RNA polymerase II to promoters. Two key human transcriptional coactivator complexes, ATAC (ADA-Two-A-Containing) and SAGA (Spt-Ada-Gcn5-acetyltransferase), target genomic loci to increase promoter accessibility. To better understand the function of ATAC and SAGA histone acetyltransferase (HAT) complexes, we used in vitro biochemical and biophysical assays to characterize human ATAC and SAGA HAT module interactions with nucleosomes and how a transcription factor (TF) coordinates these interactions. We found that ATAC and SAGA HAT modules bind nucleosomes with high affinity, independent of post-translational modifications (PTMs) and TFs. ATAC and SAGA HAT modules directly interact with the VP16 activator domain and a TF containing this domain enhances HAT module acetylation activity. Surprisingly, ATAC and SAGA HAT modules increase TF binding to its DNA target site within the nucleosome by an order of magnitude independent of histone acetylation. Altogether, our results reveal synergistic coordination between HAT modules and a TF, where ATAC and SAGA HAT modules: (i) acetylate histones to open chromatin, and (ii) facilitate TF targeting within nucleosomes independently of their acetylation activity.\nMapping RNA-Protein Interactions with Subcellular Resolution Using Colocalization CLIP\nAuthors: Yi, S.; Singh, S. S.; Rozen-Gagnon, K.; Luna, J. M.\nScore: 15.5, Published: 2023-10-26 DOI: 10.1101/2023.10.26.563984\nRNA binding proteins (RBPs) are essential for RNA metabolism and profoundly impact health and disease. The subcellular organization of RBP interaction networks with target RNAs remains largely unexplored. Here, we develop colocalization CLIP, a method that combines CrossLinking and ImmunoPrecipitation (CLIP) with proximity labeling, to explore in-depth the subcellular RNA interactions of the well-studied RNA-binding protein HuR. Using this method, we uncover HuRs dynamic and location-specific interactions with RNA, revealing alterations in sequence preferences and interactions in the nucleus, cytosol, or stress granule compartments. We uncover HuRs unique binding preferences within stress granules during arsenite stress, illuminating intricate interactions that conventional methodologies cannot capture. Overall, coCLIP provides a powerful method for revealing RBP:RNA interactions based on localization and lays the foundation for an advanced understanding of RBP models that incorporate subcellular location as a critical determinant of their functions.\nTelomere protein arrays stall DNA loop extrusion by condensin\nAuthors: Analikwu, B. T.; Deshayes, A.; van der Torre, J.; Guerin, T.; Katan, A. J.; Beneut, C.; Barth, R.; Phipps, J.; Scolari, V.; Veaute, X.; Barrington, C.; Busso, D.; Uhlmann, F.; Dubrana, K.; Mattarocci, S.; Dekker, C.; Marcand, S.\nScore: 11.6, Published: 2023-10-29 DOI: 10.1101/2023.10.29.564563\nDNA loop extrusion by SMC proteins is a key process underlying chromosomal organization. It is unknown how loop extruders interact with telomeres where chromosome ends are covered with a dense array of tens of neighboring DNA-binding proteins. Using complementary in vivo and in vitro single-molecule approaches, we study the interaction between loop-extruding condensin and arrays of Rap1, the double-stranded-DNA-binding telomeric protein of Saccharomyces cerevisiae. We show that dense linear Rap1 arrays can completely halt DNA loop extrusion, where the blocking efficiency depends on the array length and the DNA gap size between neighboring proteins. In cells, Rap1 arrays in the chromosome are found to act as contact insulators and to accumulate condensin at their borders, with direct implications for the resolution of dicentric chromosomes produced by telomere fusions. Our findings show that linear arrays of DNA-bound proteins can efficiently halt DNA loop extrusion by SMC proteins, which may impact a wide range of cellular processes from telomere functions to transcription and DNA repair.\nPervasive translational control of photosynthesis genes during photomorphogenesis is acquired by C4 genes\nAuthors: Llorens, I. R.; Lastovka, F.; Schreier, T. B.; Singh, P.; Chung, B. Y.; Hibberd, J. M.\nScore: 8.0, Published: 2023-10-27 DOI: 10.1101/2023.10.27.563924\nC4 photosynthesis allows increased efficiency and has evolved in more than sixty-six plant lineages. Underpinning this repeated appearance of the C4 pathway is a major transcriptional reprogramming of photosynthesis genes. Here we investigated whether evolution has also significantly modified translational control by defining the translational dynamics of C3 rice and C4 sorghum during photomorphogenesis. In the dark rice photosynthesis transcripts are low abundance but highly translated. After exposure to light translational efficiency declines. The same phenomena occur in sorghum but in addition C4 cycle genes show this response. We propose a model in which translational control of photosynthesis genes permits a rapid response to light and that this translational regulation is gained by C4 genes during the evolution of the C4 pathway.\nMeasuring mtDNA turnover, synthesis, and supercoiling via selective bromodeoxyuridine incorporation\nAuthors: Deng, J.; Mohan, A.; Shutt, T.\nScore: 7.1, Published: 2023-10-26 DOI: 10.1101/2023.10.24.563889\nIncreasing work on mitochondria is focused on the dynamic aspects of this organelle, a key part of which is regulation of the mitochondrial genome (mtDNA). Here we describe a protocol that selectively incorporates bromodeoxyuridine into mtDNA for the measurement of mtDNA turnover, synthesis or supercoiling via an adapted immunoblot and Southern blot protocol. For complete use and execution of this protocol see Al Khatib et al.1,2 and Lei et al.3\nMulti-modal skin atlas identifies a multicellular immune-stromal community associated with altered cornification and specific T cell expansion in atopic dermatitis\nAuthors: Fiskin, E.; Eraslan, G.; Alora-Palli, M. B.; Leyva-Castillo, J. M.; Kim, S.; Choe, H.; Lareau, C. A.; Lau, H.; Finan, E. P.; Teixeira-Soldano, I.; LaBere, B.; Chu, A.; Woods, B.; Chou, J.; Slyper, M.; Waldman, J.; Islam, S.; Schneider, L.; Phipatanakul, W.; Platt, C.; Rozenblatt-Rosen, O.; Delorey, T. M.; Deguine, J.; Smith, G. P.; Geha, R. S.; Regev, A.; Xavier, R. J.\nScore: 5.8, Published: 2023-10-31 DOI: 10.1101/2023.10.29.563503\nIn healthy skin, a cutaneous immune system maintains balance between tolerance towards innocuous environmental antigens and immune responses against pathological agents. In atopic dermatitis (AD), barrier and immune dysfunction result in chronic tissue inflammation. Our understanding of the skin tissue ecosystem in AD remains incomplete with regard to pathological barrier formation, and cellular state and clonal composition of disease-promoting cells. Here, we generated a multi-modal cell census of 310,691 cells from 86 cell subsets from whole skin tissue of 19 adult individuals, including non-lesional and lesional skin from 11 AD patients, and integrated it with 396,321 cells from 4 studies into a comprehensive atlas. Lesional skin comprised a unique immune and stromal multicellular community that included populations of MMP12+ dendritic cells (DCs), mature migratory DCs, cycling ILCs, NK cells, inflammatory CCL19+ IL4I1+ fibroblasts, and clonally expanded IL13+IL22+IL26+ T cells with overlapping type 2 and type 17 characteristics, connected by multiple inter-cellular positive feedback loops. Reconstruction of human keratinocyte differentiation from basal to cornified layers revealed a disrupted cornification trajectory in AD associated with signals from the disease-associated immune compartment. AD GWAS gene expression is enriched in cornified keratinocytes, IL13+IL22+IL26+ T cells, and ILCs, suggesting that epithelial or immune dysfunction can initiate and then converge towards AD. Our work highlights specific, disease-associated cell subsets and interactions as potential targets in progression and resolution of chronic inflammation.\nAccelerated drug resistant variant discovery with an enhanced, scalable mutagenic base editor platform\nAuthors: Dorighi, K.; Zhu, A.; Fortin, J.-P.; Lo, J. H.-H.; Sudhamsu, J.; Durinck, S.; Callow, M.; Foster, S.; Haley, B.\nScore: 5.8, Published: 2023-10-26 DOI: 10.1101/2023.10.25.564011\nPersonalized cancer therapeutics bring directed treatment options to patients based on the genetic signatures of their tumors. Unfortunately, tumor genomes are remarkably adaptable, and acquired resistance to these drugs through genetic means is an all-too-frequent occurrence. Identifying mutations that promote resistance within drug-treated patient populations can be cost, resource, and time intensive. Accordingly, base editing, enabled by Cas9-deaminase domain fusions, has emerged as a promising approach for rapid, large-scale resistance variant screening in situ. We adapted and optimized a conditional activation-induced cytidine deaminase (AID)-dCas9 system, which demonstrated greater heterogeneity of edits with an expanded footprint compared to the most commonly utilized cytosine base editor, BE4. When combined with a custom sgRNA library, we were able to identify both individual and complex, compound variants in EGFR and BRAF that confer resistance to established EGFR inhibitors. This system and the developed analytical pipeline provide a simple, highly-scalable platform for cis or trans drug-modifying variant discovery and for uncovering unique insights into protein structure-function relationships.\nThe Ribosome Assembly Factor Reh1 is Released from the Polypeptide Exit Tunnel in the Pioneer Round of Translation\nAuthors: Musalgaonkar, S.; Yelland, J.; Chitale, R.; Rao, S.; Ozadam, H.; Cenik, C.; Taylor, D.; Johnson, A. W.\nScore: 6.0, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563604\nAssembly of functional ribosomal subunits and successfully delivering them to the translating pool is a prerequisite for protein synthesis and cell growth. In S. cerevisiae, the ribosome assembly factor Reh1 binds to pre-60S subunits at a late stage during their cytoplasmic maturation. Previous work shows that the C-terminus of Reh1 inserts into the polypeptide exit tunnel (PET) of the pre-60S subunit. Unlike canonical assembly factors, which associate exclusively with pre-60S subunits, we observed that Reh1 sediments with polysomes in addition to free 60S subunits. We therefore investigated the intriguing possibility that Reh1 remains associated with 60S subunits after the release of the anti-association factor Tif6 and after subunit joining. Here, we show that Reh1-bound nascent 60S subunits associate with 40S subunits to form actively translating ribosomes. Using selective ribosome profiling, we found that Reh1-bound ribosomes populate open reading frames near start codons. Reh1-bound ribosomes are also strongly enriched for initiator tRNA, indicating they are associated with early elongation events. Using single particle cryo-electron microscopy to image cycloheximide-arrested Reh1-bound 80S ribosomes, we found that Reh1-bound 80S contain A site peptidyl tRNA, P site tRNA and eIF5A indicating that Reh1 does not dissociate from 60S until early stages of translation elongation. We propose that Reh1 is displaced by the elongating peptide chain. These results identify Reh1 as the last assembly factor released from the nascent 60S subunit during its pioneer round of translation.\nModularity of PRC1 Composition and Chromatin Interaction define Condensate Properties\nAuthors: Niekamp, S.; Marr, S. K.; Oei, T. A.; Subramanian, R.; Kingston, R. E.\nScore: 5.3, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564217\nPolycomb repressive complexes (PRC) play a key role in gene repression and are indispensable for proper development. Canonical PRC1 forms condensates in vitro and in cells and the ability of PRC1 to form condensates has been proposed to contribute to maintenance of repression. However, how chromatin and the various subunits of PRC1 contribute to condensation is largely unexplored. Using single-molecule imaging, we demonstrate that nucleosomal arrays and PRC1 act synergistically, reducing the critical concentration required for condensation by more than 20-fold. By reconstituting and imaging PRC1 with various subunit compositions, we find that the exact combination of PHC and CBX subunits determine the initiation, morphology, stability, and dynamics of condensates. In particular, the polymerization activity of PHC2 strongly influences condensate dynamics to promote formation of structures with distinct domains that adhere to each other but do not coalesce. Using live cell imaging, we confirmed that CBX properties are critical for condensate initiation and that PHC polymerization is important to maintain stable condensates. Together, we propose that PRC1 can fine-tune the degree and type of condensation by altering its composition which might offer important flexibility of regulatory function during different stages of development.\nP116 from Mycoplasma is a self-sufficient lipid uptake and delivery machinery\nAuthors: Manger, S.; Arghittu, S. M.; Sprankel, L.; Meier-Credo, J.; Wieland, K.; Schwalm, M. P.; Bublak, D.; Knapp, S.; Langer, J.; Covino, R.; Frangakis, A. S.\nScore: 4.6, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563710\nLipid acquisition and transport are fundamental processes in all organisms, but many of the key players remain unidentified. Here, we elucidate the lipid-cycling mechanism of the Mycoplasma pneumoniae membrane protein P116. We show that P116 not only extracts lipids from its environment but also self-sufficiently deposits them into both bacterial and eukaryotic cell membranes as well as liposomes. Our structures and molecular dynamics simulation show that the N-terminal region of P116, which resembles an SMP domain, is responsible for perturbing the membrane, while a hydrophobic pocket exploits the chemical gradient to collect the lipids and the proteins dorsal side acts as a mediator of membrane directionality. Furthermore, ligand binding and growth curve assays suggest the potential for designing small molecule inhibitors targeting this essential and immunodominant protein. We show that P116 is a versatile lipid acquisition and delivery machinery that shortcuts the multi-protein pathways used by more complex organisms. Thus, our work advances the understanding of common lipid transport strategies, which may aid research into the mechanisms of more complex lipid-handling machineries.\n",
  "wordCount" : "2099",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564358">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564358" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564358">
        <p class="paperTitle">ATAC and SAGA histone acetyltransferase modules facilitate transcription factor binding to nucleosomes in an acetylation independent manner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564358" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564358" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chesnutt, K.; Yayli, G.; Toelzer, C.; Cox, K.; Gautam, G.; Berger, I.; Tora, L.; Poirier, M. G.</p>
        <p class="info">Score: 22.9, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564358' target='https://doi.org/10.1101/2023.10.27.564358'> 10.1101/2023.10.27.564358</a></p>
        <p class="abstract">Transcription initiation involves the coordination of multiple events, starting with activators binding specific DNA target sequences, which recruits transcription co-activators to open chromatin and enable binding of general transcription factors and RNA polymerase II to promoters. Two key human transcriptional coactivator complexes, ATAC (ADA-Two-A-Containing) and SAGA (Spt-Ada-Gcn5-acetyltransferase), target genomic loci to increase promoter accessibility. To better understand the function of ATAC and SAGA histone acetyltransferase (HAT) complexes, we used in vitro biochemical and biophysical assays to characterize human ATAC and SAGA HAT module interactions with nucleosomes and how a transcription factor (TF) coordinates these interactions. We found that ATAC and SAGA HAT modules bind nucleosomes with high affinity, independent of post-translational modifications (PTMs) and TFs. ATAC and SAGA HAT modules directly interact with the VP16 activator domain and a TF containing this domain enhances HAT module acetylation activity. Surprisingly, ATAC and SAGA HAT modules increase TF binding to its DNA target site within the nucleosome by an order of magnitude independent of histone acetylation. Altogether, our results reveal synergistic coordination between HAT modules and a TF, where ATAC and SAGA HAT modules: (i) acetylate histones to open chromatin, and (ii) facilitate TF targeting within nucleosomes independently of their acetylation activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563984">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563984" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563984">
        <p class="paperTitle">Mapping RNA-Protein Interactions with Subcellular Resolution Using Colocalization CLIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563984" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563984" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yi, S.; Singh, S. S.; Rozen-Gagnon, K.; Luna, J. M.</p>
        <p class="info">Score: 15.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563984' target='https://doi.org/10.1101/2023.10.26.563984'> 10.1101/2023.10.26.563984</a></p>
        <p class="abstract">RNA binding proteins (RBPs) are essential for RNA metabolism and profoundly impact health and disease. The subcellular organization of RBP interaction networks with target RNAs remains largely unexplored. Here, we develop colocalization CLIP, a method that combines CrossLinking and ImmunoPrecipitation (CLIP) with proximity labeling, to explore in-depth the subcellular RNA interactions of the well-studied RNA-binding protein HuR. Using this method, we uncover HuRs dynamic and location-specific interactions with RNA, revealing alterations in sequence preferences and interactions in the nucleus, cytosol, or stress granule compartments. We uncover HuRs unique binding preferences within stress granules during arsenite stress, illuminating intricate interactions that conventional methodologies cannot capture. Overall, coCLIP provides a powerful method for revealing RBP:RNA interactions based on localization and lays the foundation for an advanced understanding of RBP models that incorporate subcellular location as a critical determinant of their functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564563">
        <p class="paperTitle">Telomere protein arrays stall DNA loop extrusion by condensin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Analikwu, B. T.; Deshayes, A.; van der Torre, J.; Guerin, T.; Katan, A. J.; Beneut, C.; Barth, R.; Phipps, J.; Scolari, V.; Veaute, X.; Barrington, C.; Busso, D.; Uhlmann, F.; Dubrana, K.; Mattarocci, S.; Dekker, C.; Marcand, S.</p>
        <p class="info">Score: 11.6, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564563' target='https://doi.org/10.1101/2023.10.29.564563'> 10.1101/2023.10.29.564563</a></p>
        <p class="abstract">DNA loop extrusion by SMC proteins is a key process underlying chromosomal organization. It is unknown how loop extruders interact with telomeres where chromosome ends are covered with a dense array of tens of neighboring DNA-binding proteins. Using complementary in vivo and in vitro single-molecule approaches, we study the interaction between loop-extruding condensin and arrays of Rap1, the double-stranded-DNA-binding telomeric protein of Saccharomyces cerevisiae. We show that dense linear Rap1 arrays can completely halt DNA loop extrusion, where the blocking efficiency depends on the array length and the DNA gap size between neighboring proteins. In cells, Rap1 arrays in the chromosome are found to act as contact insulators and to accumulate condensin at their borders, with direct implications for the resolution of dicentric chromosomes produced by telomere fusions. Our findings show that linear arrays of DNA-bound proteins can efficiently halt DNA loop extrusion by SMC proteins, which may impact a wide range of cellular processes from telomere functions to transcription and DNA repair.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.563924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.563924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.563924">
        <p class="paperTitle">Pervasive translational control of photosynthesis genes during photomorphogenesis is acquired by C4 genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.563924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.563924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Llorens, I. R.; Lastovka, F.; Schreier, T. B.; Singh, P.; Chung, B. Y.; Hibberd, J. M.</p>
        <p class="info">Score: 8.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.563924' target='https://doi.org/10.1101/2023.10.27.563924'> 10.1101/2023.10.27.563924</a></p>
        <p class="abstract">C4 photosynthesis allows increased efficiency and has evolved in more than sixty-six plant lineages. Underpinning this repeated appearance of the C4 pathway is a major transcriptional reprogramming of photosynthesis genes. Here we investigated whether evolution has also significantly modified translational control by defining the translational dynamics of C3 rice and C4 sorghum during photomorphogenesis. In the dark rice photosynthesis transcripts are low abundance but highly translated. After exposure to light translational efficiency declines. The same phenomena occur in sorghum but in addition C4 cycle genes show this response. We propose a model in which translational control of photosynthesis genes permits a rapid response to light and that this translational regulation is gained by C4 genes during the evolution of the C4 pathway.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563889">
        <p class="paperTitle">Measuring mtDNA turnover, synthesis, and supercoiling via selective bromodeoxyuridine incorporation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deng, J.; Mohan, A.; Shutt, T.</p>
        <p class="info">Score: 7.1, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563889' target='https://doi.org/10.1101/2023.10.24.563889'> 10.1101/2023.10.24.563889</a></p>
        <p class="abstract">Increasing work on mitochondria is focused on the dynamic aspects of this organelle, a key part of which is regulation of the mitochondrial genome (mtDNA). Here we describe a protocol that selectively incorporates bromodeoxyuridine into mtDNA for the measurement of mtDNA turnover, synthesis or supercoiling via an adapted immunoblot and Southern blot protocol. For complete use and execution of this protocol see Al Khatib et al.1,2 and Lei et al.3</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.563503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.563503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.563503">
        <p class="paperTitle">Multi-modal skin atlas identifies a multicellular immune-stromal community associated with altered cornification and specific T cell expansion in atopic dermatitis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.563503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.563503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fiskin, E.; Eraslan, G.; Alora-Palli, M. B.; Leyva-Castillo, J. M.; Kim, S.; Choe, H.; Lareau, C. A.; Lau, H.; Finan, E. P.; Teixeira-Soldano, I.; LaBere, B.; Chu, A.; Woods, B.; Chou, J.; Slyper, M.; Waldman, J.; Islam, S.; Schneider, L.; Phipatanakul, W.; Platt, C.; Rozenblatt-Rosen, O.; Delorey, T. M.; Deguine, J.; Smith, G. P.; Geha, R. S.; Regev, A.; Xavier, R. J.</p>
        <p class="info">Score: 5.8, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.563503' target='https://doi.org/10.1101/2023.10.29.563503'> 10.1101/2023.10.29.563503</a></p>
        <p class="abstract">In healthy skin, a cutaneous immune system maintains balance between tolerance towards innocuous environmental antigens and immune responses against pathological agents. In atopic dermatitis (AD), barrier and immune dysfunction result in chronic tissue inflammation. Our understanding of the skin tissue ecosystem in AD remains incomplete with regard to pathological barrier formation, and cellular state and clonal composition of disease-promoting cells. Here, we generated a multi-modal cell census of 310,691 cells from 86 cell subsets from whole skin tissue of 19 adult individuals, including non-lesional and lesional skin from 11 AD patients, and integrated it with 396,321 cells from 4 studies into a comprehensive atlas. Lesional skin comprised a unique immune and stromal multicellular community that included populations of MMP12&#43; dendritic cells (DCs), mature migratory DCs, cycling ILCs, NK cells, inflammatory CCL19&#43; IL4I1&#43; fibroblasts, and clonally expanded IL13&#43;IL22&#43;IL26&#43; T cells with overlapping type 2 and type 17 characteristics, connected by multiple inter-cellular positive feedback loops. Reconstruction of human keratinocyte differentiation from basal to cornified layers revealed a disrupted cornification trajectory in AD associated with signals from the disease-associated immune compartment. AD GWAS gene expression is enriched in cornified keratinocytes, IL13&#43;IL22&#43;IL26&#43; T cells, and ILCs, suggesting that epithelial or immune dysfunction can initiate and then converge towards AD. Our work highlights specific, disease-associated cell subsets and interactions as potential targets in progression and resolution of chronic inflammation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564011">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564011" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564011">
        <p class="paperTitle">Accelerated drug resistant variant discovery with an enhanced, scalable mutagenic base editor platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564011" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564011" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dorighi, K.; Zhu, A.; Fortin, J.-P.; Lo, J. H.-H.; Sudhamsu, J.; Durinck, S.; Callow, M.; Foster, S.; Haley, B.</p>
        <p class="info">Score: 5.8, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564011' target='https://doi.org/10.1101/2023.10.25.564011'> 10.1101/2023.10.25.564011</a></p>
        <p class="abstract">Personalized cancer therapeutics bring directed treatment options to patients based on the genetic signatures of their tumors. Unfortunately, tumor genomes are remarkably adaptable, and acquired resistance to these drugs through genetic means is an all-too-frequent occurrence. Identifying mutations that promote resistance within drug-treated patient populations can be cost, resource, and time intensive. Accordingly, base editing, enabled by Cas9-deaminase domain fusions, has emerged as a promising approach for rapid, large-scale resistance variant screening in situ. We adapted and optimized a conditional activation-induced cytidine deaminase (AID)-dCas9 system, which demonstrated greater heterogeneity of edits with an expanded footprint compared to the most commonly utilized cytosine base editor, BE4. When combined with a custom sgRNA library, we were able to identify both individual and complex, compound variants in EGFR and BRAF that confer resistance to established EGFR inhibitors. This system and the developed analytical pipeline provide a simple, highly-scalable platform for cis or trans drug-modifying variant discovery and for uncovering unique insights into protein structure-function relationships.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563604">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563604" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563604">
        <p class="paperTitle">The Ribosome Assembly Factor Reh1 is Released from the Polypeptide Exit Tunnel in the Pioneer Round of Translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563604" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563604" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Musalgaonkar, S.; Yelland, J.; Chitale, R.; Rao, S.; Ozadam, H.; Cenik, C.; Taylor, D.; Johnson, A. W.</p>
        <p class="info">Score: 6.0, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563604' target='https://doi.org/10.1101/2023.10.23.563604'> 10.1101/2023.10.23.563604</a></p>
        <p class="abstract">Assembly of functional ribosomal subunits and successfully delivering them to the translating pool is a prerequisite for protein synthesis and cell growth. In S. cerevisiae, the ribosome assembly factor Reh1 binds to pre-60S subunits at a late stage during their cytoplasmic maturation. Previous work shows that the C-terminus of Reh1 inserts into the polypeptide exit tunnel (PET) of the pre-60S subunit. Unlike canonical assembly factors, which associate exclusively with pre-60S subunits, we observed that Reh1 sediments with polysomes in addition to free 60S subunits. We therefore investigated the intriguing possibility that Reh1 remains associated with 60S subunits after the release of the anti-association factor Tif6 and after subunit joining. Here, we show that Reh1-bound nascent 60S subunits associate with 40S subunits to form actively translating ribosomes. Using selective ribosome profiling, we found that Reh1-bound ribosomes populate open reading frames near start codons. Reh1-bound ribosomes are also strongly enriched for initiator tRNA, indicating they are associated with early elongation events. Using single particle cryo-electron microscopy to image cycloheximide-arrested Reh1-bound 80S ribosomes, we found that Reh1-bound 80S contain A site peptidyl tRNA, P site tRNA and eIF5A indicating that Reh1 does not dissociate from 60S until early stages of translation elongation. We propose that Reh1 is displaced by the elongating peptide chain. These results identify Reh1 as the last assembly factor released from the nascent 60S subunit during its pioneer round of translation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564217">
        <p class="paperTitle">Modularity of PRC1 Composition and Chromatin Interaction define Condensate Properties</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Niekamp, S.; Marr, S. K.; Oei, T. A.; Subramanian, R.; Kingston, R. E.</p>
        <p class="info">Score: 5.3, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564217' target='https://doi.org/10.1101/2023.10.26.564217'> 10.1101/2023.10.26.564217</a></p>
        <p class="abstract">Polycomb repressive complexes (PRC) play a key role in gene repression and are indispensable for proper development. Canonical PRC1 forms condensates in vitro and in cells and the ability of PRC1 to form condensates has been proposed to contribute to maintenance of repression. However, how chromatin and the various subunits of PRC1 contribute to condensation is largely unexplored. Using single-molecule imaging, we demonstrate that nucleosomal arrays and PRC1 act synergistically, reducing the critical concentration required for condensation by more than 20-fold. By reconstituting and imaging PRC1 with various subunit compositions, we find that the exact combination of PHC and CBX subunits determine the initiation, morphology, stability, and dynamics of condensates. In particular, the polymerization activity of PHC2 strongly influences condensate dynamics to promote formation of structures with distinct domains that adhere to each other but do not coalesce. Using live cell imaging, we confirmed that CBX properties are critical for condensate initiation and that PHC polymerization is important to maintain stable condensates. Together, we propose that PRC1 can fine-tune the degree and type of condensation by altering its composition which might offer important flexibility of regulatory function during different stages of development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563710">
        <p class="paperTitle">P116 from Mycoplasma is a self-sufficient lipid uptake and delivery machinery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manger, S.; Arghittu, S. M.; Sprankel, L.; Meier-Credo, J.; Wieland, K.; Schwalm, M. P.; Bublak, D.; Knapp, S.; Langer, J.; Covino, R.; Frangakis, A. S.</p>
        <p class="info">Score: 4.6, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563710' target='https://doi.org/10.1101/2023.10.24.563710'> 10.1101/2023.10.24.563710</a></p>
        <p class="abstract">Lipid acquisition and transport are fundamental processes in all organisms, but many of the key players remain unidentified. Here, we elucidate the lipid-cycling mechanism of the Mycoplasma pneumoniae membrane protein P116. We show that P116 not only extracts lipids from its environment but also self-sufficiently deposits them into both bacterial and eukaryotic cell membranes as well as liposomes. Our structures and molecular dynamics simulation show that the N-terminal region of P116, which resembles an SMP domain, is responsible for perturbing the membrane, while a hydrophobic pocket exploits the chemical gradient to collect the lipids and the proteins dorsal side acts as a mediator of membrane directionality. Furthermore, ligand binding and growth curve assays suggest the potential for designing small molecule inhibitors targeting this essential and immunodominant protein. We show that P116 is a versatile lipid acquisition and delivery machinery that shortcuts the multi-protein pathways used by more complex organisms. Thus, our work advances the understanding of common lipid transport strategies, which may aid research into the mechanisms of more complex lipid-handling machineries.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
