<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity
Authors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O&#39;Riordan, M. X. D.; O&#39;Meara, T. R.
Score: 12.5, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560560
Infection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity
Authors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O&#39;Riordan, M. X. D.; O&#39;Meara, T. R.
Score: 12.5, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560560
Infection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity
Authors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O&#39;Riordan, M. X. D.; O&#39;Meara, T. R.
Score: 12.5, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560560
Infection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity\nAuthors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O\u0026#39;Riordan, M. X. D.; O\u0026#39;Meara, T. R.\nScore: 12.5, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560560\nInfection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated.",
  "keywords": [
    
  ],
  "articleBody": " Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity\nAuthors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O'Riordan, M. X. D.; O'Meara, T. R.\nScore: 12.5, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560560\nInfection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated. While recognition of microbe-associated molecular patterns can activate IRE1, it is unclear whether this depends on the canonical role of IRE1 in detecting misfolded proteins. Here, we report that Candida albicans infection of macrophages results in IRE1 activation through C-type lectin receptor signaling, reinforcing a role for IRE1 as a central regulator of host responses to infection by a broad range of pathogens. However, IRE1 activation was not preceded by protein misfolding in response to either C. albicans infection or lipopolysaccharide treatment, implicating a non-canonical mode of IRE1 activation after recognition of microbial patterns. Investigation of the phenotypic consequences of IRE1 activation in macrophage antimicrobial responses revealed that IRE1 activity enhances the fungicidal activity of macrophages. Macrophages lacking IRE1 activity displayed inefficient phagolysosomal fusion, enabling C. albicans to evade fungal killing and escape the phagosome. Together, these data provide mechanistic insight for the non-canonical activation of IRE1 during infection, and reveal central roles for IRE1 in macrophage antifungal responses.\nA conserved class of viral RNA structures regulate translation reinitiation through dynamic ribosome interactions\nAuthors: Sherlock, M.; Langeberg, C.; Segar, K.; Kieft, J. S.\nScore: 10.2, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560040\nCertain viral RNAs encode proteins downstream of the main protein coding region, expressed through \"termination-reinitiation\" events, dependent on RNA structure. RNA elements located upstream of the first stop codon within these viral mRNAs bind the ribosome, preventing ribosome recycling and inducing reinitiation. We used bioinformatic methods to identify new examples of viral reinitiation-stimulating RNAs and experimentally verified their secondary structure and function. We determined the structure of a representative viral RNA-ribosome complex using cryoEM. 3D classification and variability analyses reveal that the viral RNA structure can sample a range of conformations while remaining tethered to the ribosome, which enabling the ribosome to find a reinitiation start site within a limited range of mRNA sequence. Evaluating the conserved features and constraints of this entire RNA class in the context of the cryoEM reconstruction provides insight into mechanisms enabling reinitiation, a translation regulation strategy employed by many other viral and eukaryotic systems.\nLocalized synthesis of molecular chaperones sustains neuronal proteostasis\nAuthors: Alecki, C.; Rizwan, J.; Le, P.; Jacob-Tomas, S.; Xu, J. S. M.; Minotti, S.; Wu, T.; Durham, H.; Yeo, G. W.; Vera, M.\nScore: 9.0, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560761\nNeurons are challenged to maintain proteostasis in neuronal projections, particularly with the physiological stress at synapses to support intercellular communication underlying important functions such as memory and movement control. Proteostasis is maintained through regulated protein synthesis and degradation and chaperone-assisted protein folding. Using high-resolution fluorescent microscopy, we discovered that neurons localize a subset of chaperone mRNAs to their dendrites, particularly more proximal regions, and increase this asymmetric localization following proteotoxic stress through microtubule-based transport from the soma. The most abundant chaperone mRNA in dendrites encodes the constitutive heat shock protein 70, HSPA8. Proteotoxic stress in cultured neurons, induced by inhibiting proteasome activity or inducing oxidative stress, enhanced transport of Hspa8 mRNAs to dendrites and the percentage of mRNAs engaged in translation on mono and polyribosomes. Knocking down the ALS-related protein Fused in Sarcoma (FUS) and a dominant mutation in the heterogenous nuclear ribonucleoprotein A2/B1 (HNRNPA2B1) impaired stress-mediated localization of Hspa8 mRNA to dendrites in cultured murine motor neurons and human iPSC-derived neurons, respectively, revealing the importance of these RNA-binding proteins in maintaining proteostasis. These results reveal the increased dendritic localization and translation of the constitutive HSP70 Hspa8 mRNA as a crucial neuronal stress response to uphold proteostasis and prevent neurodegeneration.\nStability of Genomic Imprinting and X-Chromosome Inactivation in the Aging Brain\nAuthors: Mancino, S.; Seneviratne, J.; Mupo, A.; Krueger, F.; Oxley, D.; Eckersley-Maslin, M.; da Rocha, S. T.\nScore: 7.2, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560184\nEpigenetic drift is a hallmark of aging that contributes to the irreversible decline in organismal fitness ultimately leading to aging-related diseases. Epigenetic modifications regulate the cellular memory of the epigenetic processes of genomic imprinting and X-chromosome inactivation to ensure monoallelic expression of imprinted and X-linked genes. Whether epigenetic drift affects maintenance of genomic imprinting and X-chromosome inactivation has not been comprehensively studied. Here, we investigate the allele-specific transcriptional and epigenetic signatures of the aging brain, by comparing juvenile and old hybrid mice obtained from C57BL/6J (BL6) \u0026 CAST/EiJ (CAST) reciprocal crosses, with an emphasis on the hippocampus (HCP). We confirm that the aged HCP shows an increase of DNA hydroxymethylation, a sign of epigenetic drift, and a typical aging transcriptional signature. Genomic imprinting was found to be largely unaffected with stable parent-of-origin-specific DNA methylation in HCP, but also other brain regions such as the cerebellum (CB), nucleus accumbens, hypothalamus and prefrontal cortex. Consistently, transcriptomics analysis confirmed unaltered imprinting expression in the aged HCP. An exception are three novel non-coding transcripts (B230209E15Rik, Ube2nl and A330076H08Rik) at the Prader-Willi syndrome/Angelman syndrome (PWS/AS) imprinted locus which lose strict monoallelic expression during aging. Like imprinting, X-chromosome inactivation was remarkably stable with no signs of aging-driven skewing or relaxation of monoallelic expression of X-linked genes. Our study provides a valuable resource for evaluating monoallelic expression in the aging brain and reveals that, despite epigenetic drift during aging, genomic imprinting and X-chromosome inactivation remain predominantly stable throughout the process of physiological aging in the mouse brain.\nDeep mutational scanning reveals a tight correlation between protein degradation and toxicity of thousands of non-native aspartoacylase protein variants\nAuthors: Gronbaek-Thygesen, M.; Voutsinos, V.; Johansson, K. E.; Schulze, T. K.; Cagiada, M.; Pedersen, L.; Clausen, L.; Nariya, S.; Powell, R. L.; Stein, A.; Fowler, D. M.; Lindorff-Larsen, K.; Hartmann-Petersen, R.\nScore: 7.1, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560410\nWhen the structural stability of a protein is compromised, the protein may form non-native interactions with other cell proteins and thus becomes a hazard to the cell. To mitigate this danger, destabilized proteins are targeted by the cellular protein quality control (PQC) network, which either corrects the folding defect or targets the protein for degradation. However, the details of how the protein folding and degradation systems collaborate to combat potentially toxic non-native proteins are unknown. To address this issue, we performed systematic studies on destabilized variants of the cytosolic aspartoacylase, ASPA, where loss-of-function variants are linked to Canavans disease, an autosomal recessive and lethal neurological disorder, characterized by the spongy degeneration of the white matter in the brain. Using Variant Abundance by Massively Parallel sequencing (VAMP-seq), we determined the abundance of 6152 out of the 6260 ([~]98%) possible single-site missense and nonsense ASPA variants in cultured human cells. The majority of the low abundance ASPA variants are degraded through the ubiquitin-proteasome system (UPS) and become toxic upon prolonged expression. Variant cellular abundance data correlates with predicted thermodynamic stability, evolutionary conservation, and separates most known disease-linked variants from benign variants. Systematic mapping of degradation signals (degrons) shows that inherent primary degrons in ASPA are located in buried regions, and reveals that the wild-type ASPA C-terminal region functions as a degron. Collectively, our data can be used to interpret Canavans disease variants and also offer mechanistic insight into how ASPA missense variants are targeted by the PQC system. These are essential steps towards future implementation of precision medicine for Canavans disease.\nInterleukin 11 therapy causes acute heart failure and its use in patients should be reconsidered\nAuthors: Sweeney, M.; O'Fee, K.; Villanueva-Hayes, C.; Rahman, E.; Lee, M.; Maatz, H.; Lindberg, E. L.; Vanezis, K.; Andrew, I.; Jennings, E. R.; Lim, W.-W.; Widjaja, A. A.; Hübner, N.; Barton, P. J. R.; Cook, S. A.\nScore: 6.5, Published: 2023-10-01 DOI: 10.1101/2023.09.30.560259\nBackgroundInterleukin 11 (IL11) was initially thought important for platelet production, which led to recombinant IL11 being developed as a drug to treat thrombocytopenia. IL11 was later found to be redundant for haematopoiesis and its use in patients is associated with unexplained cardiac side effects. Here we identify previously unappreciated and direct cardiomyocyte toxicities associated with IL11 therapy. MethodsWe injected recombinant mouse lL11 (rmIL11) into mice and studied its molecular effects in the heart using immunoblotting, qRT-PCR, bulk RNA-seq, single nuclei RNA-seq (snRNA-seq) and ATAC-seq. The physiological impact of IL11 was assessed by echocardiography in vivo and using cardiomyocyte contractility assays in vitro. To determine the activity of IL11 specifically in cardiomyocytes we made two cardiomyocyte-specific Il11ra1 knockout mouse models using either AAV9-mediated and Tnnt2-restricted (vCMKO) or Myh6 (m6CMKO) Cre expression and an Il11ra1 floxed mouse strain. In pharmacologic studies, we studied the effects of JAK/STAT inhibition on rmIL11-induced cardiac toxicities. ResultsInjection of rmIL11 caused acute and dose-dependent impairment of left ventricular ejection fraction (saline (2 {micro}L/kg), 60.4%{+/-}3.1; rmIL11 (200 mcg/kg), 31.6%{+/-}2.0; p\u003c0.0001, n=5). Following rmIL11 injection, myocardial STAT3 and JNK phosphorylation were increased and bulk RNA-seq revealed upregulation of pro-inflammatory pathways (TNF, NF{kappa}B and JAK/STAT) and perturbed calcium handling. SnRNA-seq showed rmIL11-induced expression of stress factors (Ankrd1, Ankrd23, Xirp2), activator protein-1 (AP-1) transcription factor genes and Nppb in the cardiomyocyte compartment. Following rmIL11 injection, ATAC-seq identified epigenetic enrichment of the Ankrd1 and Nppb genes and stress-responsive, AP-1 transcription factor binding sites. Cardiomyocyte-specific effects were examined in vCMKO and m6CMKO mice, which were both protected from rmIL11-induced left ventricular impairment and molecular pathobiologies. In mechanistic studies, inhibition of JAK/STAT signalling with either ruxolitinib or tofacitinib prevented rmIL11-induced cardiac dysfunction. ConclusionsInjection of IL11 directly activates JAK/STAT3 in cardiomyocytes to cause acute heart failure. Our data overturn the earlier assumption that IL11 is cardioprotective and explain the serious cardiac side effects associated with IL11 therapy, which questions its continued use in patients. Clinical PerspectiveO_ST_ABSWhat is new?C_ST_ABSO_LIInjection of IL11 to mice causes acute and dose-dependent left ventricular impairment C_LIO_LIIL11 activates JAK/STAT3 in cardiomyocytes to cause cell stress, inflammation and impaired calcium handling C_LIO_LIThese data identify, for the first time, that IL11 is directly toxic in cardiomyocytes, overturning the earlier literature that suggested the opposite C_LI What are the clinical implications?O_LIRecombinant human IL11 (rhIL11) is used as a drug to increase platelets in patients with thrombocytopenia but this has severe and unexplained cardiac side effects C_LIO_LIWe show that IL11 injection causes cardiomyocyte dysfunction and heart failure, which explains its cardiac toxicities that were previously thought non-specific C_LIO_LIThese findings have immediate translational implications as they question the continued use of rhIL11 in patients around the world C_LI\nRNA structure modulates Cas13 activity and enables mismatch detection\nAuthors: Kimchi, O.; Larsen, B. B.; Dunkley, O. R.; te Velthuis, A. J. W.; Myhrvold, C.\nScore: 6.3, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560533\nThe RNA-targeting CRISPR nuclease Cas13 has emerged as a powerful tool for applications ranging from nucleic acid detection to transcriptome engineering and RNA imaging. Cas13 is activated by the hybridization of a CRISPR RNA (crRNA) to a complementary single-stranded RNA (ssRNA) protospacer in a target RNA. Though Cas13 is not activated by double-stranded RNA (dsRNA) in vitro, it paradoxically demonstrates robust RNA targeting in environments where the vast majority of RNAs are highly structured. Understanding Cas13's mechanism of binding and activation will be key to improving its ability to detect and perturb RNA; however, the mechanism by which Cas13 binds structured RNAs remains unknown. Here, we systematically probe the mechanism of LwaCas13a activation in response to RNA structure perturbations using a massively multiplexed screen. We find that there are two distinct sequence-independent modes by which secondary structure affects Cas13 activity: structure in the protospacer region competes with the crRNA and can be disrupted via a strand-displacement mechanism, while structure in the region 3' to the protospacer has an allosteric inhibitory effect. We leverage the kinetic nature of the strand displacement process to improve Cas13-based RNA detection and enhance mismatch discrimination by up to 50-fold, enabling sequence-agnostic mutation identification at low (\u003c1%) allele frequencies. Our work sets a new standard for CRISPR-based nucleic acid detection and will enable intelligent and secondary-structure-guided target selection while also expanding the range of RNAs available for targeting with Cas13.\nAltered chromatin occupancy of patient-associated H4 mutants misregulate neuronal differentiation\nAuthors: Feng, L.; Barrows, D.; Zhong, L.; Matlik, K.; Porter, E. G.; Djomo, A. M.; Yau, I.; Soshnev, A. A.; Carroll, T. S.; Wen, D.; Hatten, M. E.; Garcia, B. A.; Allis, C. D.\nScore: 6.7, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560141\nChromatin is a crucial regulator of gene expression and tightly controls development across species. Mutations in only one copy of multiple histone genes were identified in children with developmental disorders characterized by microcephaly, but their mechanistic roles in development remain unclear. Here we focus on dominant mutations affecting histone H4 lysine 91. These H4K91 mutants form aberrant nuclear puncta at specific heterochromatin regions. Mechanistically, H4K91 mutants demonstrate enhanced binding to the histone variant H3.3, and ablation of H3.3 or the H3.3-specific chaperone DAXX diminishes the mutant localization to chromatin. Our functional studies demonstrate that H4K91 mutant expression increases chromatin accessibility, alters developmental gene expression through accelerating pro-neural differentiation, and causes reduced mouse brain size in vivo, reminiscent of the microcephaly phenotypes of patients. Together, our studies unveil a distinct molecular pathogenic mechanism from other known histone mutants, where H4K91 mutants misregulate cell fate during development through abnormal genomic localization.\nPost-ischemic inactivation of HIF prolyl hydroxylases in endothelium promotes maladaptive kidney repair by inducing glycolysis\nAuthors: Tiwari, R.; Sharma, R.; Rajendran, G.; Borkowski, G. S.; An, S. Y.; Schonfeld, M. P.; O'Sullivan, J.; Schipma, M.; Zhou, Y.; Courbon, G.; David, V.; Quaggin, S. E.; Thorp, E. B.; Chandel, N. S.; Kapitsinou, P. P.\nScore: 6.0, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560700\nIschemic acute kidney injury (AKI) is common in hospitalized patients and increases the risk for chronic kidney disease (CKD). Impaired endothelial cell (EC) functions are thought to contribute in AKI to CKD transition, but the underlying mechanisms remain unclear. Here, we identify a critical role for endothelial oxygen sensing prolyl hydroxylase domain (PHD) enzymes 1-3 in regulating post-ischemic kidney repair. In renal endothelium, we observed compartment-specific differences in the expression of the three PHD isoforms in both mice and humans. We found that post-ischemic concurrent inactivation of endothelial PHD1, PHD2, and PHD3 but not PHD2 alone promoted maladaptive kidney repair characterized by exacerbated tissue injury, fibrosis, and inflammation. Single-cell RNA-seq analysis of the post-ischemic endothelial PHD1, PHD2 and PHD3 deficient (PHDTiEC) kidney revealed an endothelial glycolytic transcriptional signature, also observed in human kidneys with severe AKI. This metabolic program was coupled to upregulation of the SLC16A3 gene encoding the lactate exporter monocarboxylate transporter 4 (MCT4). Strikingly, treatment with the MCT4 inhibitor syrosingopine restored adaptive kidney repair in PHDTiEC mice. Mechanistically, MCT4 inhibition suppressed pro-inflammatory EC activation reducing monocyte-endothelial cell interaction. Our findings suggest avenues for halting AKI to CKD transition based on selectively targeting the endothelial hypoxia-driven glycolysis/MCT4 axis.\nLoss of alkyladenine DNA glycosylase alters gene expression in the developing mouse brain and leads to reduced anxiety and improved memory\nAuthors: Bordin, D. L.; Grooms, K.; Montaldo, N. P.; Martin, S. L. F.; Saetrom, P.; Samson, L. D.; Bjoras, M.; Van Loon, B.\nScore: 5.2, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561113\nNeurodevelopment is a tightly coordinated process, during which the genome is exposed to spectra of endogenous agents at different stages of differentiation. Emerging evidence indicates that DNA damage is an important feature of developing brain, tightly linked to gene expression and neuronal activity. Some of the most frequent DNA damage includes changes to DNA bases, which are recognized by DNA glycosylases and repaired through base excision repair (BER) pathway. The only mammalian DNA glycosylase able to remove frequent alkylated DNA based is alkyladenine DNA glycosylase (Aag, aka Mpg). We recently demonstrated that, besides its role in DNA repair, AAG affects expression of neurodevelopmental genes in human cells. Aag was further proposed to act as reader of epigenetic marks, including 5-hydroxymethylcytosine (5hmC), in the mouse brain. Despite the potential Aag involvement in the key brain processes, the impact of Aag loss on developing brain remains unknown. Here, by using Aag knockout (Aag-/-) mice, we show that Aag absence leads to reduced DNA break levels, evident in lowered number of {gamma}H2AX foci in postnatal day 5 (P5) hippocampi. This is accompanied by changes in 5hmC signal intensity in different hippocampal regions. Transcriptome analysis of hippocampi and prefrontal cortex, at different developmental stages, indicates that lack of Aag alters gene expression, primarily of genes involved in regulation of response to stress. Across all developmental stages tested aldehyde dehydrogenase 2 (Aldh2) emerged as one of the most prominent genes deregulated in Aag-dependent manner. In line with the changes in hippocampal DNA damage levels and the gene expression, adult Aag-/- mice exhibit altered behavior, evident in decreased anxiety levels determined in the Elevated Zero Maze and increased alternations in the Elevated T Maze tests. Taken together these results suggests that Aag has functions in modulation of genome dynamics during brain development, important for animal behavior.\n",
  "wordCount" : "2828",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560560">
        <p class="paperTitle">Non-canonical activation of IRE1α during Candida albicans infection enhances macrophage fungicidal activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McFadden, M. J.; Reynolds, M. B.; Michmerhuizen, B. C.; Anderson, F. M.; O&#39;Riordan, M. X. D.; O&#39;Meara, T. R.</p>
        <p class="info">Score: 12.5, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560560' target='https://doi.org/10.1101/2023.10.02.560560'> 10.1101/2023.10.02.560560</a></p>
        <p class="abstract">Infection by intracellular pathogens can trigger activation of the IRE1 branch of the unfolded protein response (UPR), which then modulates innate immunity and infection outcomes during bacterial or viral infection. However, the mechanisms by which infection activates IRE1 have not been fully elucidated. While recognition of microbe-associated molecular patterns can activate IRE1, it is unclear whether this depends on the canonical role of IRE1 in detecting misfolded proteins. Here, we report that Candida albicans infection of macrophages results in IRE1 activation through C-type lectin receptor signaling, reinforcing a role for IRE1 as a central regulator of host responses to infection by a broad range of pathogens. However, IRE1 activation was not preceded by protein misfolding in response to either C. albicans infection or lipopolysaccharide treatment, implicating a non-canonical mode of IRE1 activation after recognition of microbial patterns. Investigation of the phenotypic consequences of IRE1 activation in macrophage antimicrobial responses revealed that IRE1 activity enhances the fungicidal activity of macrophages. Macrophages lacking IRE1 activity displayed inefficient phagolysosomal fusion, enabling C. albicans to evade fungal killing and escape the phagosome. Together, these data provide mechanistic insight for the non-canonical activation of IRE1 during infection, and reveal central roles for IRE1 in macrophage antifungal responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560040">
        <p class="paperTitle">A conserved class of viral RNA structures regulate translation reinitiation through dynamic ribosome interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sherlock, M.; Langeberg, C.; Segar, K.; Kieft, J. S.</p>
        <p class="info">Score: 10.2, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560040' target='https://doi.org/10.1101/2023.09.29.560040'> 10.1101/2023.09.29.560040</a></p>
        <p class="abstract">Certain viral RNAs encode proteins downstream of the main protein coding region, expressed through &#34;termination-reinitiation&#34; events, dependent on RNA structure. RNA elements located upstream of the first stop codon within these viral mRNAs bind the ribosome, preventing ribosome recycling and inducing reinitiation. We used bioinformatic methods to identify new examples of viral reinitiation-stimulating RNAs and experimentally verified their secondary structure and function. We determined the structure of a representative viral RNA-ribosome complex using cryoEM. 3D classification and variability analyses reveal that the viral RNA structure can sample a range of conformations while remaining tethered to the ribosome, which enabling the ribosome to find a reinitiation start site within a limited range of mRNA sequence. Evaluating the conserved features and constraints of this entire RNA class in the context of the cryoEM reconstruction provides insight into mechanisms enabling reinitiation, a translation regulation strategy employed by many other viral and eukaryotic systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560761">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560761" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560761">
        <p class="paperTitle">Localized synthesis of molecular chaperones sustains neuronal proteostasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560761" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560761" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alecki, C.; Rizwan, J.; Le, P.; Jacob-Tomas, S.; Xu, J. S. M.; Minotti, S.; Wu, T.; Durham, H.; Yeo, G. W.; Vera, M.</p>
        <p class="info">Score: 9.0, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560761' target='https://doi.org/10.1101/2023.10.03.560761'> 10.1101/2023.10.03.560761</a></p>
        <p class="abstract">Neurons are challenged to maintain proteostasis in neuronal projections, particularly with the physiological stress at synapses to support intercellular communication underlying important functions such as memory and movement control. Proteostasis is maintained through regulated protein synthesis and degradation and chaperone-assisted protein folding. Using high-resolution fluorescent microscopy, we discovered that neurons localize a subset of chaperone mRNAs to their dendrites, particularly more proximal regions, and increase this asymmetric localization following proteotoxic stress through microtubule-based transport from the soma. The most abundant chaperone mRNA in dendrites encodes the constitutive heat shock protein 70, HSPA8. Proteotoxic stress in cultured neurons, induced by inhibiting proteasome activity or inducing oxidative stress, enhanced transport of Hspa8 mRNAs to dendrites and the percentage of mRNAs engaged in translation on mono and polyribosomes. Knocking down the ALS-related protein Fused in Sarcoma (FUS) and a dominant mutation in the heterogenous nuclear ribonucleoprotein A2/B1 (HNRNPA2B1) impaired stress-mediated localization of Hspa8 mRNA to dendrites in cultured murine motor neurons and human iPSC-derived neurons, respectively, revealing the importance of these RNA-binding proteins in maintaining proteostasis. These results reveal the increased dendritic localization and translation of the constitutive HSP70 Hspa8 mRNA as a crucial neuronal stress response to uphold proteostasis and prevent neurodegeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560184">
        <p class="paperTitle">Stability of Genomic Imprinting and X-Chromosome Inactivation in the Aging Brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mancino, S.; Seneviratne, J.; Mupo, A.; Krueger, F.; Oxley, D.; Eckersley-Maslin, M.; da Rocha, S. T.</p>
        <p class="info">Score: 7.2, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560184' target='https://doi.org/10.1101/2023.09.29.560184'> 10.1101/2023.09.29.560184</a></p>
        <p class="abstract">Epigenetic drift is a hallmark of aging that contributes to the irreversible decline in organismal fitness ultimately leading to aging-related diseases. Epigenetic modifications regulate the cellular memory of the epigenetic processes of genomic imprinting and X-chromosome inactivation to ensure monoallelic expression of imprinted and X-linked genes. Whether epigenetic drift affects maintenance of genomic imprinting and X-chromosome inactivation has not been comprehensively studied. Here, we investigate the allele-specific transcriptional and epigenetic signatures of the aging brain, by comparing juvenile and old hybrid mice obtained from C57BL/6J (BL6) &amp; CAST/EiJ (CAST) reciprocal crosses, with an emphasis on the hippocampus (HCP). We confirm that the aged HCP shows an increase of DNA hydroxymethylation, a sign of epigenetic drift, and a typical aging transcriptional signature. Genomic imprinting was found to be largely unaffected with stable parent-of-origin-specific DNA methylation in HCP, but also other brain regions such as the cerebellum (CB), nucleus accumbens, hypothalamus and prefrontal cortex. Consistently, transcriptomics analysis confirmed unaltered imprinting expression in the aged HCP. An exception are three novel non-coding transcripts (B230209E15Rik, Ube2nl and A330076H08Rik) at the Prader-Willi syndrome/Angelman syndrome (PWS/AS) imprinted locus which lose strict monoallelic expression during aging. Like imprinting, X-chromosome inactivation was remarkably stable with no signs of aging-driven skewing or relaxation of monoallelic expression of X-linked genes. Our study provides a valuable resource for evaluating monoallelic expression in the aging brain and reveals that, despite epigenetic drift during aging, genomic imprinting and X-chromosome inactivation remain predominantly stable throughout the process of physiological aging in the mouse brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560410">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560410" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560410">
        <p class="paperTitle">Deep mutational scanning reveals a tight correlation between protein degradation and toxicity of thousands of non-native aspartoacylase protein variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560410" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560410" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gronbaek-Thygesen, M.; Voutsinos, V.; Johansson, K. E.; Schulze, T. K.; Cagiada, M.; Pedersen, L.; Clausen, L.; Nariya, S.; Powell, R. L.; Stein, A.; Fowler, D. M.; Lindorff-Larsen, K.; Hartmann-Petersen, R.</p>
        <p class="info">Score: 7.1, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560410' target='https://doi.org/10.1101/2023.10.02.560410'> 10.1101/2023.10.02.560410</a></p>
        <p class="abstract">When the structural stability of a protein is compromised, the protein may form non-native interactions with other cell proteins and thus becomes a hazard to the cell. To mitigate this danger, destabilized proteins are targeted by the cellular protein quality control (PQC) network, which either corrects the folding defect or targets the protein for degradation. However, the details of how the protein folding and degradation systems collaborate to combat potentially toxic non-native proteins are unknown. To address this issue, we performed systematic studies on destabilized variants of the cytosolic aspartoacylase, ASPA, where loss-of-function variants are linked to Canavans disease, an autosomal recessive and lethal neurological disorder, characterized by the spongy degeneration of the white matter in the brain. Using Variant Abundance by Massively Parallel sequencing (VAMP-seq), we determined the abundance of 6152 out of the 6260 ([~]98%) possible single-site missense and nonsense ASPA variants in cultured human cells. The majority of the low abundance ASPA variants are degraded through the ubiquitin-proteasome system (UPS) and become toxic upon prolonged expression. Variant cellular abundance data correlates with predicted thermodynamic stability, evolutionary conservation, and separates most known disease-linked variants from benign variants. Systematic mapping of degradation signals (degrons) shows that inherent primary degrons in ASPA are located in buried regions, and reveals that the wild-type ASPA C-terminal region functions as a degron. Collectively, our data can be used to interpret Canavans disease variants and also offer mechanistic insight into how ASPA missense variants are targeted by the PQC system. These are essential steps towards future implementation of precision medicine for Canavans disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560259">
        <p class="paperTitle">Interleukin 11 therapy causes acute heart failure and its use in patients should be reconsidered</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sweeney, M.; O&#39;Fee, K.; Villanueva-Hayes, C.; Rahman, E.; Lee, M.; Maatz, H.; Lindberg, E. L.; Vanezis, K.; Andrew, I.; Jennings, E. R.; Lim, W.-W.; Widjaja, A. A.; Hübner, N.; Barton, P. J. R.; Cook, S. A.</p>
        <p class="info">Score: 6.5, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560259' target='https://doi.org/10.1101/2023.09.30.560259'> 10.1101/2023.09.30.560259</a></p>
        <p class="abstract">BackgroundInterleukin 11 (IL11) was initially thought important for platelet production, which led to recombinant IL11 being developed as a drug to treat thrombocytopenia. IL11 was later found to be redundant for haematopoiesis and its use in patients is associated with unexplained cardiac side effects. Here we identify previously unappreciated and direct cardiomyocyte toxicities associated with IL11 therapy.

MethodsWe injected recombinant mouse lL11 (rmIL11) into mice and studied its molecular effects in the heart using immunoblotting, qRT-PCR, bulk RNA-seq, single nuclei RNA-seq (snRNA-seq) and ATAC-seq. The physiological impact of IL11 was assessed by echocardiography in vivo and using cardiomyocyte contractility assays in vitro. To determine the activity of IL11 specifically in cardiomyocytes we made two cardiomyocyte-specific Il11ra1 knockout mouse models using either AAV9-mediated and Tnnt2-restricted (vCMKO) or Myh6 (m6CMKO) Cre expression and an Il11ra1 floxed mouse strain. In pharmacologic studies, we studied the effects of JAK/STAT inhibition on rmIL11-induced cardiac toxicities.

ResultsInjection of rmIL11 caused acute and dose-dependent impairment of left ventricular ejection fraction (saline (2 {micro}L/kg), 60.4%{&#43;/-}3.1; rmIL11 (200 mcg/kg), 31.6%{&#43;/-}2.0; p&lt;0.0001, n=5). Following rmIL11 injection, myocardial STAT3 and JNK phosphorylation were increased and bulk RNA-seq revealed upregulation of pro-inflammatory pathways (TNF, NF{kappa}B and JAK/STAT) and perturbed calcium handling. SnRNA-seq showed rmIL11-induced expression of stress factors (Ankrd1, Ankrd23, Xirp2), activator protein-1 (AP-1) transcription factor genes and Nppb in the cardiomyocyte compartment. Following rmIL11 injection, ATAC-seq identified epigenetic enrichment of the Ankrd1 and Nppb genes and stress-responsive, AP-1 transcription factor binding sites. Cardiomyocyte-specific effects were examined in vCMKO and m6CMKO mice, which were both protected from rmIL11-induced left ventricular impairment and molecular pathobiologies. In mechanistic studies, inhibition of JAK/STAT signalling with either ruxolitinib or tofacitinib prevented rmIL11-induced cardiac dysfunction.

ConclusionsInjection of IL11 directly activates JAK/STAT3 in cardiomyocytes to cause acute heart failure. Our data overturn the earlier assumption that IL11 is cardioprotective and explain the serious cardiac side effects associated with IL11 therapy, which questions its continued use in patients.

Clinical PerspectiveO_ST_ABSWhat is new?C_ST_ABSO_LIInjection of IL11 to mice causes acute and dose-dependent left ventricular impairment
C_LIO_LIIL11 activates JAK/STAT3 in cardiomyocytes to cause cell stress, inflammation and impaired calcium handling
C_LIO_LIThese data identify, for the first time, that IL11 is directly toxic in cardiomyocytes, overturning the earlier literature that suggested the opposite
C_LI

What are the clinical implications?O_LIRecombinant human IL11 (rhIL11) is used as a drug to increase platelets in patients with thrombocytopenia but this has severe and unexplained cardiac side effects
C_LIO_LIWe show that IL11 injection causes cardiomyocyte dysfunction and heart failure, which explains its cardiac toxicities that were previously thought non-specific
C_LIO_LIThese findings have immediate translational implications as they question the continued use of rhIL11 in patients around the world
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560533">
        <p class="paperTitle">RNA structure modulates Cas13 activity and enables mismatch detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kimchi, O.; Larsen, B. B.; Dunkley, O. R.; te Velthuis, A. J. W.; Myhrvold, C.</p>
        <p class="info">Score: 6.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560533' target='https://doi.org/10.1101/2023.10.05.560533'> 10.1101/2023.10.05.560533</a></p>
        <p class="abstract">The RNA-targeting CRISPR nuclease Cas13 has emerged as a powerful tool for applications ranging from nucleic acid detection to transcriptome engineering and RNA imaging. Cas13 is activated by the hybridization of a CRISPR RNA (crRNA) to a complementary single-stranded RNA (ssRNA) protospacer in a target RNA. Though Cas13 is not activated by double-stranded RNA (dsRNA) in vitro, it paradoxically demonstrates robust RNA targeting in environments where the vast majority of RNAs are highly structured. Understanding Cas13&#39;s mechanism of binding and activation will be key to improving its ability to detect and perturb RNA; however, the mechanism by which Cas13 binds structured RNAs remains unknown. Here, we systematically probe the mechanism of LwaCas13a activation in response to RNA structure perturbations using a massively multiplexed screen. We find that there are two distinct sequence-independent modes by which secondary structure affects Cas13 activity: structure in the protospacer region competes with the crRNA and can be disrupted via a strand-displacement mechanism, while structure in the region 3&#39; to the protospacer has an allosteric inhibitory effect. We leverage the kinetic nature of the strand displacement process to improve Cas13-based RNA detection and enhance mismatch discrimination by up to 50-fold, enabling sequence-agnostic mutation identification at low (&lt;1%) allele frequencies. Our work sets a new standard for CRISPR-based nucleic acid detection and will enable intelligent and secondary-structure-guided target selection while also expanding the range of RNAs available for targeting with Cas13.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560141">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560141" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560141">
        <p class="paperTitle">Altered chromatin occupancy of patient-associated H4 mutants misregulate neuronal differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560141" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560141" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, L.; Barrows, D.; Zhong, L.; Matlik, K.; Porter, E. G.; Djomo, A. M.; Yau, I.; Soshnev, A. A.; Carroll, T. S.; Wen, D.; Hatten, M. E.; Garcia, B. A.; Allis, C. D.</p>
        <p class="info">Score: 6.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560141' target='https://doi.org/10.1101/2023.09.29.560141'> 10.1101/2023.09.29.560141</a></p>
        <p class="abstract">Chromatin is a crucial regulator of gene expression and tightly controls development across species. Mutations in only one copy of multiple histone genes were identified in children with developmental disorders characterized by microcephaly, but their mechanistic roles in development remain unclear. Here we focus on dominant mutations affecting histone H4 lysine 91. These H4K91 mutants form aberrant nuclear puncta at specific heterochromatin regions. Mechanistically, H4K91 mutants demonstrate enhanced binding to the histone variant H3.3, and ablation of H3.3 or the H3.3-specific chaperone DAXX diminishes the mutant localization to chromatin. Our functional studies demonstrate that H4K91 mutant expression increases chromatin accessibility, alters developmental gene expression through accelerating pro-neural differentiation, and causes reduced mouse brain size in vivo, reminiscent of the microcephaly phenotypes of patients. Together, our studies unveil a distinct molecular pathogenic mechanism from other known histone mutants, where H4K91 mutants misregulate cell fate during development through abnormal genomic localization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560700">
        <p class="paperTitle">Post-ischemic inactivation of HIF prolyl hydroxylases in endothelium promotes maladaptive kidney repair by inducing glycolysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tiwari, R.; Sharma, R.; Rajendran, G.; Borkowski, G. S.; An, S. Y.; Schonfeld, M. P.; O&#39;Sullivan, J.; Schipma, M.; Zhou, Y.; Courbon, G.; David, V.; Quaggin, S. E.; Thorp, E. B.; Chandel, N. S.; Kapitsinou, P. P.</p>
        <p class="info">Score: 6.0, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560700' target='https://doi.org/10.1101/2023.10.03.560700'> 10.1101/2023.10.03.560700</a></p>
        <p class="abstract">Ischemic acute kidney injury (AKI) is common in hospitalized patients and increases the risk for chronic kidney disease (CKD). Impaired endothelial cell (EC) functions are thought to contribute in AKI to CKD transition, but the underlying mechanisms remain unclear. Here, we identify a critical role for endothelial oxygen sensing prolyl hydroxylase domain (PHD) enzymes 1-3 in regulating post-ischemic kidney repair. In renal endothelium, we observed compartment-specific differences in the expression of the three PHD isoforms in both mice and humans. We found that post-ischemic concurrent inactivation of endothelial PHD1, PHD2, and PHD3 but not PHD2 alone promoted maladaptive kidney repair characterized by exacerbated tissue injury, fibrosis, and inflammation. Single-cell RNA-seq analysis of the post-ischemic endothelial PHD1, PHD2 and PHD3 deficient (PHDTiEC) kidney revealed an endothelial glycolytic transcriptional signature, also observed in human kidneys with severe AKI. This metabolic program was coupled to upregulation of the SLC16A3 gene encoding the lactate exporter monocarboxylate transporter 4 (MCT4). Strikingly, treatment with the MCT4 inhibitor syrosingopine restored adaptive kidney repair in PHDTiEC mice. Mechanistically, MCT4 inhibition suppressed pro-inflammatory EC activation reducing monocyte-endothelial cell interaction. Our findings suggest avenues for halting AKI to CKD transition based on selectively targeting the endothelial hypoxia-driven glycolysis/MCT4 axis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561113">
        <p class="paperTitle">Loss of alkyladenine DNA glycosylase alters gene expression in the developing mouse brain and leads to reduced anxiety and improved memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bordin, D. L.; Grooms, K.; Montaldo, N. P.; Martin, S. L. F.; Saetrom, P.; Samson, L. D.; Bjoras, M.; Van Loon, B.</p>
        <p class="info">Score: 5.2, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561113' target='https://doi.org/10.1101/2023.10.05.561113'> 10.1101/2023.10.05.561113</a></p>
        <p class="abstract">Neurodevelopment is a tightly coordinated process, during which the genome is exposed to spectra of endogenous agents at different stages of differentiation. Emerging evidence indicates that DNA damage is an important feature of developing brain, tightly linked to gene expression and neuronal activity. Some of the most frequent DNA damage includes changes to DNA bases, which are recognized by DNA glycosylases and repaired through base excision repair (BER) pathway. The only mammalian DNA glycosylase able to remove frequent alkylated DNA based is alkyladenine DNA glycosylase (Aag, aka Mpg). We recently demonstrated that, besides its role in DNA repair, AAG affects expression of neurodevelopmental genes in human cells. Aag was further proposed to act as reader of epigenetic marks, including 5-hydroxymethylcytosine (5hmC), in the mouse brain. Despite the potential Aag involvement in the key brain processes, the impact of Aag loss on developing brain remains unknown. Here, by using Aag knockout (Aag-/-) mice, we show that Aag absence leads to reduced DNA break levels, evident in lowered number of {gamma}H2AX foci in postnatal day 5 (P5) hippocampi. This is accompanied by changes in 5hmC signal intensity in different hippocampal regions. Transcriptome analysis of hippocampi and prefrontal cortex, at different developmental stages, indicates that lack of Aag alters gene expression, primarily of genes involved in regulation of response to stress. Across all developmental stages tested aldehyde dehydrogenase 2 (Aldh2) emerged as one of the most prominent genes deregulated in Aag-dependent manner. In line with the changes in hippocampal DNA damage levels and the gene expression, adult Aag-/- mice exhibit altered behavior, evident in decreased anxiety levels determined in the Elevated Zero Maze and increased alternations in the Elevated T Maze tests. Taken together these results suggests that Aag has functions in modulation of genome dynamics during brain development, important for animal behavior.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
