<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.
Authors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.
Score: 22.0, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561763
This report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.
Authors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.
Score: 22.0, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561763
This report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.
Authors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.
Score: 22.0, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561763
This report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.\nAuthors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.\nScore: 22.0, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561763\nThis report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe.",
  "keywords": [
    
  ],
  "articleBody": " Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.\nAuthors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.\nScore: 22.0, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561763\nThis report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe. Streptavidin conjugated with horseradish peroxidase and enhanced chemiluminescence substrate are then used to detect the probe bound to the target RNA. Our results demonstrate the versatility of this method in detecting natural and engineered RNAs expressed in cells, including messenger and noncoding RNAs. We show that Chemi-Northern detection is sensitive and fast, detecting attomole amounts of RNA in as little as 1 second, with high signal intensity and low background. The dynamic response displays excellent linearity. Using Chemi-Northern, we measure the significant, reproducible reduction of mRNA levels by human sequence-specific RNA-binding proteins, PUM1 and PUM2. Additionally, we measure the interaction of endogenous poly(A) binding protein, PABPC1, with poly-adenylated mRNA. Thus, the Chemi-Northern method provides a versatile, simple, cost-effective method to enable researchers to detect and measure changes in RNA expression, processing, binding, and decay of RNAs.\nOpen-source, high-throughput targeted in-situ transcriptomics for developmental biologists\nAuthors: Lee, H.; Mattsson Langseth, C.; Marco Salas, S.; Metousis, A.; Rueda Alana, E.; Garcia Moreno, F.; Grillo, M.; Nilsson, M.\nScore: 20.2, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561689\nMultiplexed spatial profiling of mRNAs has recently gained traction as a tool to explore the cellular diversity and the architecture of tissues. We propose a sensitive, open-source, simple and flexible method for the generation of in-situ expression maps of hundreds of genes. We exploit direct ligation of padlock probes on mRNAs, coupled with rolling circle amplification and hybridization-based in situ combinatorial barcoding, to achieve high detection efficiency, high throughput and large multiplexing. We validate the method across a number of species, and show its use in combination with orthogonal methods such as antibody staining, highlighting its potential value for developmental biology studies. Finally, we provide an end-to-end computational workflow that covers the steps of probe design, image processing, data extraction, cell segmentation, clustering and annotation of cell types. By enabling easier access to highthroughput spatially resolved transcriptomics, we hope to encourage a diversity of applications and the exploration of a wide range of biological questions.\nHibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion\nAuthors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.\nScore: 113.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365\nCell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria. The molecular mechanism underlying ribosomal sequestration remains elusive. In this study, we performed time-lapse in situ cryo-electron tomography and cryo-electron microscopy complemented by biochemical experiments to elucidate the molecular details of this adaptive response. Our analysis indicate that upon glucose depletion protein synthesis is halted, causing ribosomes to enter an inactive state characterized by a conformational change that obstructs the peptidyl transferase center. Our in situ experiments reveal the presence of oligomeric arrays of hibernating ribosomes tethered to the mitochondrial surface. Surprisingly, ribosomes bind to the outer mitochondrial membrane via the small ribosomal subunit, an interaction facilitated by the ribosomal protein RACK1-orthologue Cpc2. Our experiments show that ribosome tethering is important for cell survival under glucose depletion conditions. This study broadens our understanding of the cellular adaptations triggered by nutrient scarcity and the underlying molecular mechanisms that regulate cell quiescence.\nSHIFTR enables the unbiased and multiplexed identification of proteins bound to specific RNA regions in live cells\nAuthors: Aydin, J.; Gabel, A.; Zielinski, S.; Ganskih, S.; Schmidt, N.; Hartigan, C. R.; Schenone, M.; Carr, S. A.; Munschauer, M.\nScore: 6.8, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561498\nRNA-protein interactions determine the cellular fate of RNA and are central to regulating gene expression outcomes in health and disease. To date, no method exists that is able to identify proteins that interact with specific regions within endogenous RNAs in live cells. Here, we develop SHIFTR (Selective RNase H-mediated interactome framing for target RNA regions), an efficient and scalable approach to identify proteins bound to selected regions within endogenous RNAs using mass spectrometry. Compared to state-of-the-art techniques, SHIFTR is superior in accuracy, captures close to zero background interactions and requires orders of magnitude lower input material. We establish SHIFTR workflows for targeting RNA classes of different length and abundance, including short and long non-coding RNAs, as well as mRNAs and demonstrate that SHIFTR is compatible with sequentially mapping interactomes for multiple target RNAs in a single experiment. Using SHIFTR, we comprehensively identify interactions of cis-regulatory elements located at the 5' and 3'- terminal regions of the authentic SARS-CoV-2 RNA genome in infected cells and accurately recover known and novel interactions linked to the function of these viral RNA elements. SHIFTR enables the systematic mapping of region-resolved RNA interactomes for any RNA in any cell type and has the potential to revolutionize our understanding of transcriptomes and their regulation.\nExtrachromosomal circular DNA is required for yeast stress granule assembly\nAuthors: Demeshkina, N. A.; Ferre-D'Amare, A. R.\nScore: 6.2, Published: 2023-10-13 DOI: 10.1101/2023.10.12.562116\nStress granules are a key cellular response, conserved across eukaryotes, to environmental insults. They arise upon stress-induced reversible stalling of protein synthesis and redistribution of mRNAs from polysomes into these large cytoplasmic ribonucleoprotein condensates. Many cellular pathways lead to formation and turnover of stress granules. Despite research spanning four decades, biochemical analysis of stress granules remains challenging because of their complexity and dynamic nature. Hitherto, stress granules have been studied cytologically, and their composition deduced from candidate protein approaches including immunoprecipitation and proximity labeling. Now, we have directly purified stress granule cores from yeast, relying on their density and size, finding that they are discrete particles with a size distribution peaking at 135 nm. Unexpectedly, we discovered that in addition to diverse proteins and RNAs, stress granule cores contain extrachromosomal circular DNA (eccDNA). These double-stranded DNAs have been found in all eukaryotes, but there is no precedent for a role in membraneless organelle biogenesis. By repurposing CRISPR technology to target DNA in the cytoplasm, we demonstrate that both presence and accessibility of eccDNA are required for stress granule formation in live yeast cells. Our finding reveals an unrecognized link between DNA metabolism and the cytoplasmic stress response, and suggests that sequence-specific eccDNA-mRNA interactions may mediate stress granule assembly.\nAnalysis the molecular similarity of least common amino acid sites in ACE2 receptor to predict the potential susceptible species for SARS-CoV-2\nAuthors: YeZhi, H.; Fan, X.; Villalan, A. K.; Zhang, S.; Joka, F. R.; Wu, X.; Wang, H.; Wang, X.\nScore: 5.0, Published: 2023-10-16 DOI: 10.1101/2023.10.13.562198\nThis research offers a bioinformatics approach to forecasting both domestic and wild animals' likelihood of being susceptible to SARS-CoV-2 infection. Genomic sequencing can resolve phylogenetic relationships between the virus and the susceptible host. The genome sequence of SARS-CoV-2 is highly interactive with the specific sequence region of the ACE2 receptor of the host species. We further evaluate this concept to identify the most important SARS-CoV-2 binding amino acid sites in the ACE2 receptor sequence through the common similarity of the last common amino acid sites (LCAS) in known susceptible host species. Therefore, the SARS-CoV-2 viral genomic interacting key amino acid region in the ACE2 receptor sequence of known susceptible human host was summarized and compared with other reported known SARS-CoV-2 susceptible host species. We identified the 10 most significant amino acid sites for interaction with SARS-CoV-2 infection from the ACE2 receptor sequence region based on the LCAS similarity pattern in known sensitive SARS-CoV-2 hosts. The most significant 10 LCAS were further compared with ACE2 receptor sequences of unknown species to evaluate the similarity of the last common amino acid pattern (LCAP). We predicted the probability of SARS-CoV-2 infection risk in unknown species through the LCAS similarity pattern. This method can be used as a screening tool to assess the risk of SARS-CoV-2 infection in domestic and wild animals to prevent outbreaks of infection.\nPredicting prime editing efficiency across diverse edit types and chromatin contexts with machine learning\nAuthors: Mathis, N.; Allam, A.; Talas, A.; Benvenuto, E.; Schep, R.; Damodharan, T.; Balazs, Z.; Janjuha, S.; Schmidheini, L.; Böck, D.; van Steensel, B.; Krauthammer, M.; Schwank, G.\nScore: 23.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561414\nPrime editing is a powerful genome editing technology, but its efficiency varies depending on the pegRNA design and target locus. Existing computational models for predicting prime editing rates are limited by their focus on specific edit types and by omitting the local chromatin environment. In our study, we developed machine learning models that predict prime editing efficiencies across a wide range of edit types up to 15 bp (PRIDICT2.0) and in different chromatin contexts (ePRIDICT). Both models can be accessed at www.pridict.it.\nECM-Focused Proteomic Analysis of Ear Punch Regeneration in Acomys cahirinus\nAuthors: McCabe, M. C.; Okamura, D. M.; Erickson, C. B.; Perry, B. W.; Brewer, C. M.; Nguyen, E. D.; Saviola, A. J.; Majesky, M. W.; Hansen, K. C.\nScore: 4.3, Published: 2023-10-17 DOI: 10.1101/2023.10.11.561940\nIn mammals, significant injury is generally followed by the formation of a fibrotic scar which provides structural integrity but fails to functionally restore damaged tissue. Spiny mice of the genus Acomys represent the first example of full skin autotomy in mammals. Acomys cahirinus has evolved extremely weak skin as a strategy to avoid predation and is able to repeatedly regenerate healthy tissue without scar after severe skin injury or full-thickness ear punches. Extracellular matrix (ECM) composition is a critical regulator of wound repair and scar formation and previous studies have suggested that alterations in its expression may be responsible for the differences in regenerative capacity observed between Mus musculus and A. cahirinus, yet analysis of this critical tissue component has been limited in previous studies by its insolubility and resistance to extraction. Here, we utilize a 3-step ECM-optimized extraction to perform proteomic analysis of tissue composition during wound repair after full-thickness ear punches in A. cahirinus and M. musculus from weeks 1 to 4 post-injury. We observe changes in a wide range of ECM proteins which have been previously implicated in wound regeneration and scar formation, including collagens, coagulation and provisional matrix proteins, and matricryptic signaling peptides. We additionally report differences in crosslinking enzyme activity and ECM protein solubility between Mus and Acomys. Furthermore, we observed rapid and sustained increases in CD206, a marker of pro-regenerative M2 macrophages, in Acomys, whereas little or no increase in CD206 was detected in Mus. Together, these findings contribute to a comprehensive understanding of tissue cues which drive the regenerative capacity of Acomys and identify a number of potential targets for future pro-regenerative therapies.\nAutomated model building and protein identification in cryo-EM maps\nAuthors: Jamali, K.; Kall, L.; Zhang, R.; Brown, A.; Kimanius, D.; Scheres, S.\nScore: 98.8, Published: 2023-10-17 DOI: 10.1101/2023.05.16.541002\nInterpreting electron cryo-microscopy (cryo-EM) maps with atomic models requires high levels of expertise and labour-intensive manual intervention. We present ModelAngelo, a machine-learning approach for automated atomic model building in cryo-EM maps. By combining information from the cryo-EM map with information from protein sequence and structure in a single graph neural network, ModelAngelo builds atomic models for proteins that are of similar quality as those generated by human experts. For nucleotides, ModelAngelo builds backbones with similar accuracy as humans. By using its predicted amino acid probabilities for each residue in hidden Markov model sequence searches, ModelAngelo outperforms human experts in the identification of proteins with unknown sequences. ModelAngelo will thus remove bottlenecks and increase objectivity in cryo-EM structure determination.\nNMD targets experience deadenylation during their maturation and endonucleolytic cleavage during their decay\nAuthors: Arribere, J. A.; Viscardi, M. J.\nScore: 9.4, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560204\nPremature stop codon-containing mRNAs can produce truncated and dominantly acting proteins that harm cells. Eukaryotic cells protect themselves by degrading such mRNAs via the Nonsense-Mediated mRNA Decay (NMD) pathway. The precise reactions by which cells attack NMD target mRNAs remain obscure, precluding a mechanistic understanding of NMD and hampering therapeutic efforts to control NMD. A key step in NMD is the decay of the mRNA, which is proposed to occur via several competing models including deadenylation, exonucleolytic decay, and/or endonucleolytic decay. We set out to clarify the relative contributions of these decay mechanisms to NMD, and to identify the role of key factors. Here, we modify and deploy single-molecule nanopore mRNA sequencing to capture full-length NMD targets and their degradation intermediates, and we obtain single-molecule measures of splicing isoform, cleavage state, and poly(A) tail length. We observe robust endonucleolytic cleavage of NMD targets in vivo that depends on the nuclease SMG-6 and we use the occurence of cleavages to identify several known NMD targets. We show that NMD target mRNAs experience deadenylation, but similar to the extent that normal mRNAs experience as they enter the translational pool. Furthermore, we show that a factor (SMG-5) that historically was ascribed a function in deadenylation, is in fact required for SMG-6-mediated cleavage. Our results support a model in which NMD factors act in concert to degrade NMD targets in animals via an endonucleolytic cleavage near the stop codon, and suggest that deadenylation is a normal part of mRNA (and NMD target) maturation rather than a facet unique to NMD. Our work clarifies the route by which NMD target mRNAs are attacked in an animal.\n",
  "wordCount" : "2285",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561763">
        <p class="paperTitle">Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.</p>
        <p class="info">Score: 22.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561763' target='https://doi.org/10.1101/2023.10.10.561763'> 10.1101/2023.10.10.561763</a></p>
        <p class="abstract">This report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe. Streptavidin conjugated with horseradish peroxidase and enhanced chemiluminescence substrate are then used to detect the probe bound to the target RNA. Our results demonstrate the versatility of this method in detecting natural and engineered RNAs expressed in cells, including messenger and noncoding RNAs. We show that Chemi-Northern detection is sensitive and fast, detecting attomole amounts of RNA in as little as 1 second, with high signal intensity and low background. The dynamic response displays excellent linearity. Using Chemi-Northern, we measure the significant, reproducible reduction of mRNA levels by human sequence-specific RNA-binding proteins, PUM1 and PUM2. Additionally, we measure the interaction of endogenous poly(A) binding protein, PABPC1, with poly-adenylated mRNA. Thus, the Chemi-Northern method provides a versatile, simple, cost-effective method to enable researchers to detect and measure changes in RNA expression, processing, binding, and decay of RNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561689">
        <p class="paperTitle">Open-source, high-throughput targeted in-situ transcriptomics for developmental biologists</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, H.; Mattsson Langseth, C.; Marco Salas, S.; Metousis, A.; Rueda Alana, E.; Garcia Moreno, F.; Grillo, M.; Nilsson, M.</p>
        <p class="info">Score: 20.2, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561689' target='https://doi.org/10.1101/2023.10.10.561689'> 10.1101/2023.10.10.561689</a></p>
        <p class="abstract">Multiplexed spatial profiling of mRNAs has recently gained traction as a tool to explore the cellular diversity and the architecture of tissues. We propose a sensitive, open-source, simple and flexible method for the generation of in-situ expression maps of hundreds of genes. We exploit direct ligation of padlock probes on mRNAs, coupled with rolling circle amplification and hybridization-based in situ combinatorial barcoding, to achieve high detection efficiency, high throughput and large multiplexing. We validate the method across a number of species, and show its use in combination with orthogonal methods such as antibody staining, highlighting its potential value for developmental biology studies. Finally, we provide an end-to-end computational workflow that covers the steps of probe design, image processing, data extraction, cell segmentation, clustering and annotation of cell types. By enabling easier access to highthroughput spatially resolved transcriptomics, we hope to encourage a diversity of applications and the exploration of a wide range of biological questions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561365">
        <p class="paperTitle">Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.</p>
        <p class="info">Score: 113.1, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561365' target='https://doi.org/10.1101/2023.10.08.561365'> 10.1101/2023.10.08.561365</a></p>
        <p class="abstract">Cell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria. The molecular mechanism underlying ribosomal sequestration remains elusive. In this study, we performed time-lapse in situ cryo-electron tomography and cryo-electron microscopy complemented by biochemical experiments to elucidate the molecular details of this adaptive response. Our analysis indicate that upon glucose depletion protein synthesis is halted, causing ribosomes to enter an inactive state characterized by a conformational change that obstructs the peptidyl transferase center. Our in situ experiments reveal the presence of oligomeric arrays of hibernating ribosomes tethered to the mitochondrial surface. Surprisingly, ribosomes bind to the outer mitochondrial membrane via the small ribosomal subunit, an interaction facilitated by the ribosomal protein RACK1-orthologue Cpc2. Our experiments show that ribosome tethering is important for cell survival under glucose depletion conditions. This study broadens our understanding of the cellular adaptations triggered by nutrient scarcity and the underlying molecular mechanisms that regulate cell quiescence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561498">
        <p class="paperTitle">SHIFTR enables the unbiased and multiplexed identification of proteins bound to specific RNA regions in live cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aydin, J.; Gabel, A.; Zielinski, S.; Ganskih, S.; Schmidt, N.; Hartigan, C. R.; Schenone, M.; Carr, S. A.; Munschauer, M.</p>
        <p class="info">Score: 6.8, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561498' target='https://doi.org/10.1101/2023.10.09.561498'> 10.1101/2023.10.09.561498</a></p>
        <p class="abstract">RNA-protein interactions determine the cellular fate of RNA and are central to regulating gene expression outcomes in health and disease. To date, no method exists that is able to identify proteins that interact with specific regions within endogenous RNAs in live cells. Here, we develop SHIFTR (Selective RNase H-mediated interactome framing for target RNA regions), an efficient and scalable approach to identify proteins bound to selected regions within endogenous RNAs using mass spectrometry. Compared to state-of-the-art techniques, SHIFTR is superior in accuracy, captures close to zero background interactions and requires orders of magnitude lower input material. We establish SHIFTR workflows for targeting RNA classes of different length and abundance, including short and long non-coding RNAs, as well as mRNAs and demonstrate that SHIFTR is compatible with sequentially mapping interactomes for multiple target RNAs in a single experiment. Using SHIFTR, we comprehensively identify interactions of cis-regulatory elements located at the 5&#39; and 3&#39;- terminal regions of the authentic SARS-CoV-2 RNA genome in infected cells and accurately recover known and novel interactions linked to the function of these viral RNA elements. SHIFTR enables the systematic mapping of region-resolved RNA interactomes for any RNA in any cell type and has the potential to revolutionize our understanding of transcriptomes and their regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562116" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562116">
        <p class="paperTitle">Extrachromosomal circular DNA is required for yeast stress granule assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demeshkina, N. A.; Ferre-D&#39;Amare, A. R.</p>
        <p class="info">Score: 6.2, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562116' target='https://doi.org/10.1101/2023.10.12.562116'> 10.1101/2023.10.12.562116</a></p>
        <p class="abstract">Stress granules are a key cellular response, conserved across eukaryotes, to environmental insults. They arise upon stress-induced reversible stalling of protein synthesis and redistribution of mRNAs from polysomes into these large cytoplasmic ribonucleoprotein condensates. Many cellular pathways lead to formation and turnover of stress granules. Despite research spanning four decades, biochemical analysis of stress granules remains challenging because of their complexity and dynamic nature. Hitherto, stress granules have been studied cytologically, and their composition deduced from candidate protein approaches including immunoprecipitation and proximity labeling. Now, we have directly purified stress granule cores from yeast, relying on their density and size, finding that they are discrete particles with a size distribution peaking at 135 nm. Unexpectedly, we discovered that in addition to diverse proteins and RNAs, stress granule cores contain extrachromosomal circular DNA (eccDNA). These double-stranded DNAs have been found in all eukaryotes, but there is no precedent for a role in membraneless organelle biogenesis. By repurposing CRISPR technology to target DNA in the cytoplasm, we demonstrate that both presence and accessibility of eccDNA are required for stress granule formation in live yeast cells. Our finding reveals an unrecognized link between DNA metabolism and the cytoplasmic stress response, and suggests that sequence-specific eccDNA-mRNA interactions may mediate stress granule assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562198">
        <p class="paperTitle">Analysis the molecular similarity of least common amino acid sites in ACE2 receptor to predict the potential susceptible species for SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: YeZhi, H.; Fan, X.; Villalan, A. K.; Zhang, S.; Joka, F. R.; Wu, X.; Wang, H.; Wang, X.</p>
        <p class="info">Score: 5.0, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562198' target='https://doi.org/10.1101/2023.10.13.562198'> 10.1101/2023.10.13.562198</a></p>
        <p class="abstract">This research offers a bioinformatics approach to forecasting both domestic and wild animals&#39; likelihood of being susceptible to SARS-CoV-2 infection. Genomic sequencing can resolve phylogenetic relationships between the virus and the susceptible host. The genome sequence of SARS-CoV-2 is highly interactive with the specific sequence region of the ACE2 receptor of the host species. We further evaluate this concept to identify the most important SARS-CoV-2 binding amino acid sites in the ACE2 receptor sequence through the common similarity of the last common amino acid sites (LCAS) in known susceptible host species. Therefore, the SARS-CoV-2 viral genomic interacting key amino acid region in the ACE2 receptor sequence of known susceptible human host was summarized and compared with other reported known SARS-CoV-2 susceptible host species. We identified the 10 most significant amino acid sites for interaction with SARS-CoV-2 infection from the ACE2 receptor sequence region based on the LCAS similarity pattern in known sensitive SARS-CoV-2 hosts. The most significant 10 LCAS were further compared with ACE2 receptor sequences of unknown species to evaluate the similarity of the last common amino acid pattern (LCAP). We predicted the probability of SARS-CoV-2 infection risk in unknown species through the LCAS similarity pattern. This method can be used as a screening tool to assess the risk of SARS-CoV-2 infection in domestic and wild animals to prevent outbreaks of infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561414">
        <p class="paperTitle">Predicting prime editing efficiency across diverse edit types and chromatin contexts with machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mathis, N.; Allam, A.; Talas, A.; Benvenuto, E.; Schep, R.; Damodharan, T.; Balazs, Z.; Janjuha, S.; Schmidheini, L.; Böck, D.; van Steensel, B.; Krauthammer, M.; Schwank, G.</p>
        <p class="info">Score: 23.7, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561414' target='https://doi.org/10.1101/2023.10.09.561414'> 10.1101/2023.10.09.561414</a></p>
        <p class="abstract">Prime editing is a powerful genome editing technology, but its efficiency varies depending on the pegRNA design and target locus. Existing computational models for predicting prime editing rates are limited by their focus on specific edit types and by omitting the local chromatin environment. In our study, we developed machine learning models that predict prime editing efficiencies across a wide range of edit types up to 15 bp (PRIDICT2.0) and in different chromatin contexts (ePRIDICT). Both models can be accessed at www.pridict.it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561940">
        <p class="paperTitle">ECM-Focused Proteomic Analysis of Ear Punch Regeneration in Acomys cahirinus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCabe, M. C.; Okamura, D. M.; Erickson, C. B.; Perry, B. W.; Brewer, C. M.; Nguyen, E. D.; Saviola, A. J.; Majesky, M. W.; Hansen, K. C.</p>
        <p class="info">Score: 4.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561940' target='https://doi.org/10.1101/2023.10.11.561940'> 10.1101/2023.10.11.561940</a></p>
        <p class="abstract">In mammals, significant injury is generally followed by the formation of a fibrotic scar which provides structural integrity but fails to functionally restore damaged tissue. Spiny mice of the genus Acomys represent the first example of full skin autotomy in mammals. Acomys cahirinus has evolved extremely weak skin as a strategy to avoid predation and is able to repeatedly regenerate healthy tissue without scar after severe skin injury or full-thickness ear punches. Extracellular matrix (ECM) composition is a critical regulator of wound repair and scar formation and previous studies have suggested that alterations in its expression may be responsible for the differences in regenerative capacity observed between Mus musculus and A. cahirinus, yet analysis of this critical tissue component has been limited in previous studies by its insolubility and resistance to extraction. Here, we utilize a 3-step ECM-optimized extraction to perform proteomic analysis of tissue composition during wound repair after full-thickness ear punches in A. cahirinus and M. musculus from weeks 1 to 4 post-injury. We observe changes in a wide range of ECM proteins which have been previously implicated in wound regeneration and scar formation, including collagens, coagulation and provisional matrix proteins, and matricryptic signaling peptides. We additionally report differences in crosslinking enzyme activity and ECM protein solubility between Mus and Acomys. Furthermore, we observed rapid and sustained increases in CD206, a marker of pro-regenerative M2 macrophages, in Acomys, whereas little or no increase in CD206 was detected in Mus. Together, these findings contribute to a comprehensive understanding of tissue cues which drive the regenerative capacity of Acomys and identify a number of potential targets for future pro-regenerative therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.16.541002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.16.541002" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.16.541002">
        <p class="paperTitle">Automated model building and protein identification in cryo-EM maps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.16.541002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.16.541002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jamali, K.; Kall, L.; Zhang, R.; Brown, A.; Kimanius, D.; Scheres, S.</p>
        <p class="info">Score: 98.8, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.16.541002' target='https://doi.org/10.1101/2023.05.16.541002'> 10.1101/2023.05.16.541002</a></p>
        <p class="abstract">Interpreting electron cryo-microscopy (cryo-EM) maps with atomic models requires high levels of expertise and labour-intensive manual intervention. We present ModelAngelo, a machine-learning approach for automated atomic model building in cryo-EM maps. By combining information from the cryo-EM map with information from protein sequence and structure in a single graph neural network, ModelAngelo builds atomic models for proteins that are of similar quality as those generated by human experts. For nucleotides, ModelAngelo builds backbones with similar accuracy as humans. By using its predicted amino acid probabilities for each residue in hidden Markov model sequence searches, ModelAngelo outperforms human experts in the identification of proteins with unknown sequences. ModelAngelo will thus remove bottlenecks and increase objectivity in cryo-EM structure determination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560204">
        <p class="paperTitle">NMD targets experience deadenylation during their maturation and endonucleolytic cleavage during their decay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arribere, J. A.; Viscardi, M. J.</p>
        <p class="info">Score: 9.4, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560204' target='https://doi.org/10.1101/2023.09.29.560204'> 10.1101/2023.09.29.560204</a></p>
        <p class="abstract">Premature stop codon-containing mRNAs can produce truncated and dominantly acting proteins that harm cells. Eukaryotic cells protect themselves by degrading such mRNAs via the Nonsense-Mediated mRNA Decay (NMD) pathway. The precise reactions by which cells attack NMD target mRNAs remain obscure, precluding a mechanistic understanding of NMD and hampering therapeutic efforts to control NMD. A key step in NMD is the decay of the mRNA, which is proposed to occur via several competing models including deadenylation, exonucleolytic decay, and/or endonucleolytic decay. We set out to clarify the relative contributions of these decay mechanisms to NMD, and to identify the role of key factors. Here, we modify and deploy single-molecule nanopore mRNA sequencing to capture full-length NMD targets and their degradation intermediates, and we obtain single-molecule measures of splicing isoform, cleavage state, and poly(A) tail length. We observe robust endonucleolytic cleavage of NMD targets in vivo that depends on the nuclease SMG-6 and we use the occurence of cleavages to identify several known NMD targets. We show that NMD target mRNAs experience deadenylation, but similar to the extent that normal mRNAs experience as they enter the translational pool. Furthermore, we show that a factor (SMG-5) that historically was ascribed a function in deadenylation, is in fact required for SMG-6-mediated cleavage. Our results support a model in which NMD factors act in concert to degrade NMD targets in animals via an endonucleolytic cleavage near the stop codon, and suggest that deadenylation is a normal part of mRNA (and NMD target) maturation rather than a facet unique to NMD. Our work clarifies the route by which NMD target mRNAs are attacked in an animal.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
