<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor
Authors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.
Score: 13.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516
The human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor
Authors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.
Score: 13.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516
The human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor
Authors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.
Score: 13.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516
The human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor\nAuthors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.\nScore: 13.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516\nThe human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin.",
  "keywords": [
    
  ],
  "articleBody": " DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor\nAuthors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.\nScore: 13.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545516\nThe human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin. However, how HUSH and MORC2 operate alongside DNA methylation, a central epigenetic regulator of repeat transcription, remains poorly understood. Here we interrogate this relationship in human neural progenitor cells (hNPCs), a somatic model of brain development that tolerates removal of DNA methyltransferase DNMT1. Upon loss of MORC2 or HUSH subunit TASOR in hNPCs, L1s remain silenced by robust promoter methylation. However, genome demethylation and activation of evolutionarily-young L1s attracts MORC2 binding. Simultaneous depletion of DNMT1 and MORC2 causes massive accumulation of L1 transcripts. We identify the same mechanistic hierarchy at pericentromeric -satellites and clustered protocadherin genes, repetitive elements important for chromosome structure and neurodevelopment respectively. Our data delineate the independent epigenetic control of repeats in somatic cells, with implications for understanding the vital functions of HUSH-MORC2 in hypomethylated contexts throughout human development.\nTidyTron: Reducing lab waste using validated wash-and-reuse protocols for common plasticware in Opentrons OT-2 lab robots\nAuthors: Bryant, J. A.; Longmire, C.; Sridhar, S.; Janousek, S.; Kellinger, M.; Wright, R. C.\nScore: 14.3, Published: 2023-06-09 DOI: 10.1101/2023.06.09.544400\nEvery year biotechnology labs generate a combined total of [~]5.5 million tons of plastic waste. As the global bioeconomy expands, biofoundries will inevitably increase plastic consumption in-step with synthetic biology scaling. Decontamination and reuse of single-use plastics could increase sustainability and reduce recurring costs of biological research. However, throughput and variable cleaning quality make manual decontamination impractical in most instances. Automating single-use plastic cleaning with liquid handling robots makes decontamination more practical by offering higher throughput and consistent cleaning quality. However, open-source, validated protocols using low-cost lab robotics for effective decontamination of plasticware--facilitating safe reuse--have not yet been developed. Here we introduce and validate TidyTron: a library of protocols for cleaning micropipette tips and microtiter plates that are contaminated with DNA, E. coli, and S. cerevisiae. We tested a variety of cleaning solutions, contact times, and agitation methods with the aim of minimizing time and cost, while maximizing cleaning stringency and sustainability. We tested and validated these cleaning procedures by comparing fresh versus cleaned tips and plates for contamination with cells, DNA, or cleaning solutions. We assessed contamination by measuring colony forming units by plating, PCR efficiency and DNA concentration by qPCR, and event counts and debris by flow cytometry. Open source cleaning protocols are available at https://github.com/PlantSynBioLab/TidyTron and hosted on a graphical user interface at https://jbryantvt.github.io/TidyTron/.\nLoss of endothelial cell heterogeneity in arteries after obesogenic diet\nAuthors: Dunaway, L. S.; Luse, M. A.; Nyshadham, S.; Bulut, G.; Alencar, G. F.; Chavkin, N. W.; Cortese-Krott, M.; Hirschi, K. K.; Isakson, B. E.\nScore: 9.6, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546320\nBackgroundIt is well recognized that obesity leads to arterial endothelial dysfunction and cardiovascular disease. However, the progression to endothelial dysfunction is not clear. Endothelial cells (ECs) adapt to the unique needs of their resident tissue and respond to systemic metabolic perturbations. We sought to better understand how obesity affects EC phenotypes in different tissues specifically focusing on mitochondrial gene expression. MethodsWe performed bulk RNA sequencing (RNA-seq) and single cell RNA-seq (scRNA-seq) on mesenteric and adipose ECs isolated from normal chow (NC) and high fat diet (HFD) fed mice. Differential gene expression, gene ontology pathway, and transcription factor analyses were performed. We further investigated our hypothesis in humans using published human adipose single nuclei RNA-seq (snRNA-seq) data. ResultsBulk RNA-seq revealed higher mitochondrial gene expression in adipose ECs compared to mesenteric ECs in both NC and HFD mice. We then performed scRNA-seq and categorized EC clusters as arterial, capillary, venous, or lymphatic. HFD decreased the number of differentially expressed genes between mesenteric and adipose ECs in all subtypes, but the largest effect was seen in arterial ECs. Further analysis of arterial ECs revealed genes coding for mitochondrial oxidative phosphorylation proteins were enriched in adipose compared to mesentery under NC conditions. In HFD mice, these genes were decreased in adipose ECs becoming similar to mesenteric ECs. Transcription factor analysis revealed C/EBP and PPAR{gamma}, both known to regulate lipid handling and metabolism, had high specificity scores in the NC adipose artery ECs. These findings were recapitulated in snRNA-seq data from human adipose. ConclusionsThese data suggest mesenteric and adipose arterial ECs metabolize lipids differently and the transcriptional phenotype of these two vascular beds converge in obesity, in part, due to downregulation of PPAR{gamma} and C/EBP in adipose artery ECs. This work lays the foundation for investigating vascular bed specific adaptations to obesity.\nLoop-extruding Smc5/6 organizes transcription-induced positive DNA supercoils\nAuthors: Jeppsson, K.; Pradhan, B.; Sutani, T.; Sakata, T.; Umeda Igarashi, M.; Berta, D. G.; Kanno, T.; Nakato, R.; Shirahige, K.; Kim, E.; Bjorkegren, C.\nScore: 9.4, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545053\nThe Structural Maintenance of Chromosome (SMC) protein complexes cohesin, condensin and the Smc5/6 complex (Smc5/6) are essential for chromosome function. At the molecular level, these complexes fold DNA by loop extrusion. Accordingly, cohesin creates chromosome loops in interphase, and condensin compacts mitotic chromosomes. However, the role of Smc5/6s recently discovered DNA loop extrusion activity is unknown. Here, we uncover that Smc5/6 associates with transcription-induced positively supercoiled chromosomal DNA at cohesin-dependent chromosome loop boundaries. Mechanistically, single-molecule imaging reveals that dimers of Smc5/6 specifically recognize the tip of positively supercoiled DNA plectonemes, and efficiently initiates loop extrusion to gather the supercoiled DNA into a large plectonemic loop. Finally, Hi-C analysis shows that Smc5/6 links chromosomal regions containing transcription-induced positive supercoiling in cis. Altogether, our findings indicate that Smc5/6 controls the three-dimensional organization of chromosomes by recognizing and initiating loop extrusion on positively supercoiled DNA.\nProgrammable RNA-guided endonucleases are widespread in eukaryotes and their viruses\nAuthors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.\nScore: 206.5, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871\nTnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.\nBRG1 establishes the neuroectodermal chromatin landscape to restrict dorsal cell fates\nAuthors: Hoffman, J. A.; Muse, G. W.; Langer, L. F.; Gandara, I.; Ward, J. M.; Archer, T. K.\nScore: 8.5, Published: 2023-06-24 DOI: 10.1101/2023.06.24.546365\nCell fate decisions are achieved with gene expression changes driven by lineage-specific transcription factors (TFs). These TFs depend on chromatin remodelers including the BAF complex to activate target genes. BAF complex subunits are essential for development and frequently mutated in cancer. Thus, interrogating how BAF complexes contribute to cell fate decisions is critical for human health. We examined the requirement for the catalytic BAF subunit BRG1 in neural progenitor cell (NPC) specification from human embryonic stem cells. During the earliest stages of differentiation, BRG1 was required to establish chromatin accessibility at neuroectoderm-specific enhancers. BRG1 depletion resulted in abnormal NPC populations that differentially expressed neuroectodermal TFs, were more prone to neuronal differentiation, and precociously formed neural crest lineages. These findings demonstrate that BRG1 mediates NPC specification by ensuring proper expression of lineage-specific TFs and appropriate activation of their transcriptional programs.\nMolecular pathway of mitochondrial preprotein import through the TOM-TIM23 supercomplex\nAuthors: Zhou, X.; Yang, Y.; Wang, G.; Wang, S.; Sun, D.; Ou, X.; Lian, Y.; Li, L.\nScore: 8.3, Published: 2023-06-23 DOI: 10.1101/2023.06.21.546012\nMost mitochondrial proteins need to be imported from the cytosol. Over half of mitochondrial proteins are imported through the pre-sequence pathway that is controlled by the TOM complex in the outer membrane and the TIM23 complex in the inner membrane. It is unclear on the molecular level how proteins cross the mitochondrial double membranes through the TOM and TIM23 complexes. Here, we report the assembly of the active TOM-TIM23 supercomplex with translocating polypeptide substrates captured in the import pathway. Electron cryo-microscopy (Cryo-EM) analyses reveal that during translocation across the outer membrane, the polypeptide substrates pass through the center of the Tom40 channel while interacting with a glutamine-rich patch in the inner wall of Tom40. Structural and biochemical analyses show that the TIM23 complex contains a heterotrimer of the subunits Tim23, Tim17, and Mgr2 in the inner membrane. Tim17 and Mgr2 shield the polypeptide substrates from the lipid environment. The import pathway consists of two highly conserved residue patches of Tim17, one negatively charged patch at the entrance and one hydrophobic patch in the middle of the pathway. These data reveal an unexpected pre-sequence pathway mediated by the TOM-TIM23 supercomplex for facilitating protein import across the double membranes of mitochondria.\nWolbachia-mediated resistance to Zika virus infection in Aedes aegypti is dominated by diverse transcriptional regulation and weak evolutionary pressures\nAuthors: Boehm, E. C.; Jaeger, A. S.; Ries, H. J.; Castaneda, D.; Weiler, A. M.; Valencia, C. C.; Weger-Lucarelli, J.; Ebel, G.; O'Connor, S. L.; Friedrich, T. C.; Zamanian, M.; Aliota, M. T.\nScore: 7.9, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546271\nA promising candidate for arbovirus control and prevention relies on replacing arbovirus-susceptible Aedes aegypti populations with mosquitoes that have been colonized by the intracellular bacterium Wolbachia and thus have a reduced capacity to transmit arboviruses. This reduced capacity to transmit arboviruses is mediated through a phenomenon referred to as pathogen blocking. Pathogen blocking has primarily been proposed as a tool to control dengue virus (DENV) transmission, however it works against a range of viruses, including Zika virus (ZIKV). Despite years of research, the molecular mechanisms underlying pathogen blocking still need to be better understood. Here, we used RNA-seq to characterize mosquito gene transcription dynamics in Ae. aegypti infected with the wMel strain of Wolbachia that are being released by the World Mosquito Program in Medellin, Colombia. Comparative analyses using ZIKV-infected, uninfected tissues, and mosquitoes without Wolbachia revealed that the influence of wMel on mosquito gene transcription is multifactorial. Importantly, because Wolbachia limits, but does not completely prevent, replication of ZIKV and other viruses in coinfected mosquitoes, there is a possibility that these viruses could evolve resistance to pathogen blocking. Therefore, to understand the influence of Wolbachia on within-host ZIKV evolution, we characterized the genetic diversity of molecularly barcoded ZIKV virus populations replicating in Wolbachia-infected mosquitoes and found that within-host ZIKV evolution was subject to weak purifying selection and, unexpectedly, loose anatomical bottlenecks in the presence and absence of Wolbachia. Together, these findings suggest that there is no clear transcriptional profile associated with Wolbachia-mediated ZIKV restriction, and that there is no evidence for ZIKV escape from this restriction in our system.\nSelf-reversal facilitates the resolution of HMCES-DNA protein crosslinks in cells\nAuthors: Rua-Fernandez, J.; Lovejoy, C. A.; Mehta, K. P. M.; Paulin, K. A.; Toudji, Y. T.; Eichman, B. F.; Cortez, D.\nScore: 8.4, Published: 2023-06-14 DOI: 10.1101/2023.06.14.544844\nAbasic sites are common DNA lesions that stall polymerases and threaten genome stability. When located in single-stranded DNA (ssDNA), they are shielded from aberrant processing by HMCES via a DNA-protein crosslink (DPC) that prevents double-strand breaks. Nevertheless, the HMCES-DPC must be removed to complete DNA repair. Here, we found that DNA polymerase inhibition generates ssDNA abasic sites and HMCES-DPCs. These DPCs are resolved with a half-life of approximately 1.5 hours. Resolution does not require the proteasome or SPRTN protease. Instead, HMCES-DPC self-reversal is important for resolution. Biochemically, self-reversal is favored when the ssDNA is converted to duplex DNA. When the self-reversal mechanism is inactivated, HMCES-DPC removal is delayed, cell proliferation is slowed, and cells become hypersensitive to DNA damage agents that increase AP site formation. Thus, HMCES-DPC formation followed by self-reversal is an important mechanism for ssDNA AP site management.\nN-6-methyladenosine (m6A) Promotes the Nuclear Retention of mRNAs with Intact 5' Splice Site Motifs\nAuthors: Lee, E. S.; Scalize de Olivera, L.; Jomphe, R. Y.; Ihn, S. S.; Smith, H. W.; Nabeel-Shah, S.; Pu, S.; Greenblatt, J. F.; Palazzo, A. F.\nScore: 7.5, Published: 2023-06-21 DOI: 10.1101/2023.06.20.545713\nIn eukaryotes, quality control of mRNA represents an important regulatory mechanism for gene expression. Misprocessed mRNAs that contain an intact 5 Splice Site (5SS) motif are retained in the nucleus and targeted for decay. Previously, we showed that the nuclear retention of these transcripts requires ZFC3H1, a component of the Poly(A) Exosome Targeting (PAXT) complex, and U1-70K, a component of the U1 snRNP. In S. pombe, the ZFC3H1 homolog, Red1, binds to the YTH-domain containing protein Mmi1 to target certain RNA transcripts for nuclear retention and decay. Here we show that ZFC3H1 and U1-70K interact with YTHDC1 and YTHDC2, two YTH domain-containing proteins that bind to N-6-methyladenosine (m6A) modified RNAs. We then show that YTHDC1 and YTHDC2 are required for the nuclear retention of mRNAs with intact 5SS motifs. Furthermore, disruption of m6A methyltransferase activity inhibits the nuclear retention of these transcripts. Using m6A-miCLIP analysis, we map m6A methylation marks to intronic polyadenylated (IPA) transcripts, which contain intact 5SS motifs and are nuclear retained and degraded in a ZFC3H1-dependent manner. We find that m6A is enriched near intact 5SS motifs and the poly(A)-tail. Overall, this work suggests that the m6A modification acts as part of an evolutionarily conserved quality control mechanism that targets misprocessed mRNAs for nuclear retention and decay.\n",
  "wordCount" : "2378",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545516">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545516" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545516">
        <p class="paperTitle">DNA methylation governs the sensitivity of repeats to restriction by the HUSH-MORC2 corepressor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545516" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545516" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandiloski, N.; Horvath, V.; Karlsson, O. E.; Christoforidou, G.; Dorazehi, F.; Koutounidou, S.; Matas, J.; Gerdes, P.; Garza, R.; Jönsson, M. E.; Adami, A.; Atacho, D.; Johansson, J. G.; Englund, E.; Kokaia, Z.; Jakobsson, J.; Douse, C. H.</p>
        <p class="info">Score: 13.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545516' target='https://doi.org/10.1101/2023.06.21.545516'> 10.1101/2023.06.21.545516</a></p>
        <p class="abstract">The human silencing hub (HUSH) complex binds to transcripts of LINE-1 retrotransposons (L1s) and other genomic repeats, recruiting MORC2 and other effectors to remodel chromatin. However, how HUSH and MORC2 operate alongside DNA methylation, a central epigenetic regulator of repeat transcription, remains poorly understood. Here we interrogate this relationship in human neural progenitor cells (hNPCs), a somatic model of brain development that tolerates removal of DNA methyltransferase DNMT1. Upon loss of MORC2 or HUSH subunit TASOR in hNPCs, L1s remain silenced by robust promoter methylation. However, genome demethylation and activation of evolutionarily-young L1s attracts MORC2 binding. Simultaneous depletion of DNMT1 and MORC2 causes massive accumulation of L1 transcripts. We identify the same mechanistic hierarchy at pericentromeric -satellites and clustered protocadherin genes, repetitive elements important for chromosome structure and neurodevelopment respectively. Our data delineate the independent epigenetic control of repeats in somatic cells, with implications for understanding the vital functions of HUSH-MORC2 in hypomethylated contexts throughout human development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.544400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.544400" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.544400">
        <p class="paperTitle">TidyTron: Reducing lab waste using validated wash-and-reuse protocols for common plasticware in Opentrons OT-2 lab robots</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.544400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.544400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, J. A.; Longmire, C.; Sridhar, S.; Janousek, S.; Kellinger, M.; Wright, R. C.</p>
        <p class="info">Score: 14.3, Published: 2023-06-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.544400' target='https://doi.org/10.1101/2023.06.09.544400'> 10.1101/2023.06.09.544400</a></p>
        <p class="abstract">Every year biotechnology labs generate a combined total of [~]5.5 million tons of plastic waste. As the global bioeconomy expands, biofoundries will inevitably increase plastic consumption in-step with synthetic biology scaling. Decontamination and reuse of single-use plastics could increase sustainability and reduce recurring costs of biological research. However, throughput and variable cleaning quality make manual decontamination impractical in most instances. Automating single-use plastic cleaning with liquid handling robots makes decontamination more practical by offering higher throughput and consistent cleaning quality. However, open-source, validated protocols using low-cost lab robotics for effective decontamination of plasticware--facilitating safe reuse--have not yet been developed. Here we introduce and validate TidyTron: a library of protocols for cleaning micropipette tips and microtiter plates that are contaminated with DNA, E. coli, and S. cerevisiae. We tested a variety of cleaning solutions, contact times, and agitation methods with the aim of minimizing time and cost, while maximizing cleaning stringency and sustainability. We tested and validated these cleaning procedures by comparing fresh versus cleaned tips and plates for contamination with cells, DNA, or cleaning solutions. We assessed contamination by measuring colony forming units by plating, PCR efficiency and DNA concentration by qPCR, and event counts and debris by flow cytometry. Open source cleaning protocols are available at https://github.com/PlantSynBioLab/TidyTron and hosted on a graphical user interface at https://jbryantvt.github.io/TidyTron/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546320">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546320" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546320">
        <p class="paperTitle">Loss of endothelial cell heterogeneity in arteries after obesogenic diet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546320" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546320" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dunaway, L. S.; Luse, M. A.; Nyshadham, S.; Bulut, G.; Alencar, G. F.; Chavkin, N. W.; Cortese-Krott, M.; Hirschi, K. K.; Isakson, B. E.</p>
        <p class="info">Score: 9.6, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546320' target='https://doi.org/10.1101/2023.06.23.546320'> 10.1101/2023.06.23.546320</a></p>
        <p class="abstract">BackgroundIt is well recognized that obesity leads to arterial endothelial dysfunction and cardiovascular disease. However, the progression to endothelial dysfunction is not clear. Endothelial cells (ECs) adapt to the unique needs of their resident tissue and respond to systemic metabolic perturbations. We sought to better understand how obesity affects EC phenotypes in different tissues specifically focusing on mitochondrial gene expression.

MethodsWe performed bulk RNA sequencing (RNA-seq) and single cell RNA-seq (scRNA-seq) on mesenteric and adipose ECs isolated from normal chow (NC) and high fat diet (HFD) fed mice. Differential gene expression, gene ontology pathway, and transcription factor analyses were performed. We further investigated our hypothesis in humans using published human adipose single nuclei RNA-seq (snRNA-seq) data.

ResultsBulk RNA-seq revealed higher mitochondrial gene expression in adipose ECs compared to mesenteric ECs in both NC and HFD mice. We then performed scRNA-seq and categorized EC clusters as arterial, capillary, venous, or lymphatic. HFD decreased the number of differentially expressed genes between mesenteric and adipose ECs in all subtypes, but the largest effect was seen in arterial ECs. Further analysis of arterial ECs revealed genes coding for mitochondrial oxidative phosphorylation proteins were enriched in adipose compared to mesentery under NC conditions. In HFD mice, these genes were decreased in adipose ECs becoming similar to mesenteric ECs. Transcription factor analysis revealed C/EBP and PPAR{gamma}, both known to regulate lipid handling and metabolism, had high specificity scores in the NC adipose artery ECs. These findings were recapitulated in snRNA-seq data from human adipose.

ConclusionsThese data suggest mesenteric and adipose arterial ECs metabolize lipids differently and the transcriptional phenotype of these two vascular beds converge in obesity, in part, due to downregulation of PPAR{gamma} and C/EBP in adipose artery ECs. This work lays the foundation for investigating vascular bed specific adaptations to obesity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545053">
        <p class="paperTitle">Loop-extruding Smc5/6 organizes transcription-induced positive DNA supercoils</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeppsson, K.; Pradhan, B.; Sutani, T.; Sakata, T.; Umeda Igarashi, M.; Berta, D. G.; Kanno, T.; Nakato, R.; Shirahige, K.; Kim, E.; Bjorkegren, C.</p>
        <p class="info">Score: 9.4, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545053' target='https://doi.org/10.1101/2023.06.20.545053'> 10.1101/2023.06.20.545053</a></p>
        <p class="abstract">The Structural Maintenance of Chromosome (SMC) protein complexes cohesin, condensin and the Smc5/6 complex (Smc5/6) are essential for chromosome function. At the molecular level, these complexes fold DNA by loop extrusion. Accordingly, cohesin creates chromosome loops in interphase, and condensin compacts mitotic chromosomes. However, the role of Smc5/6s recently discovered DNA loop extrusion activity is unknown. Here, we uncover that Smc5/6 associates with transcription-induced positively supercoiled chromosomal DNA at cohesin-dependent chromosome loop boundaries. Mechanistically, single-molecule imaging reveals that dimers of Smc5/6 specifically recognize the tip of positively supercoiled DNA plectonemes, and efficiently initiates loop extrusion to gather the supercoiled DNA into a large plectonemic loop. Finally, Hi-C analysis shows that Smc5/6 links chromosomal regions containing transcription-induced positive supercoiling in cis. Altogether, our findings indicate that Smc5/6 controls the three-dimensional organization of chromosomes by recognizing and initiating loop extrusion on positively supercoiled DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544871">
        <p class="paperTitle">Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.</p>
        <p class="info">Score: 206.5, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544871' target='https://doi.org/10.1101/2023.06.13.544871'> 10.1101/2023.06.13.544871</a></p>
        <p class="abstract">TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.546365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.546365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.546365">
        <p class="paperTitle">BRG1 establishes the neuroectodermal chromatin landscape to restrict dorsal cell fates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.546365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.546365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffman, J. A.; Muse, G. W.; Langer, L. F.; Gandara, I.; Ward, J. M.; Archer, T. K.</p>
        <p class="info">Score: 8.5, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.546365' target='https://doi.org/10.1101/2023.06.24.546365'> 10.1101/2023.06.24.546365</a></p>
        <p class="abstract">Cell fate decisions are achieved with gene expression changes driven by lineage-specific transcription factors (TFs). These TFs depend on chromatin remodelers including the BAF complex to activate target genes. BAF complex subunits are essential for development and frequently mutated in cancer. Thus, interrogating how BAF complexes contribute to cell fate decisions is critical for human health. We examined the requirement for the catalytic BAF subunit BRG1 in neural progenitor cell (NPC) specification from human embryonic stem cells. During the earliest stages of differentiation, BRG1 was required to establish chromatin accessibility at neuroectoderm-specific enhancers. BRG1 depletion resulted in abnormal NPC populations that differentially expressed neuroectodermal TFs, were more prone to neuronal differentiation, and precociously formed neural crest lineages. These findings demonstrate that BRG1 mediates NPC specification by ensuring proper expression of lineage-specific TFs and appropriate activation of their transcriptional programs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.546012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.546012" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.546012">
        <p class="paperTitle">Molecular pathway of mitochondrial preprotein import through the TOM-TIM23 supercomplex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.546012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.546012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, X.; Yang, Y.; Wang, G.; Wang, S.; Sun, D.; Ou, X.; Lian, Y.; Li, L.</p>
        <p class="info">Score: 8.3, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.546012' target='https://doi.org/10.1101/2023.06.21.546012'> 10.1101/2023.06.21.546012</a></p>
        <p class="abstract">Most mitochondrial proteins need to be imported from the cytosol. Over half of mitochondrial proteins are imported through the pre-sequence pathway that is controlled by the TOM complex in the outer membrane and the TIM23 complex in the inner membrane. It is unclear on the molecular level how proteins cross the mitochondrial double membranes through the TOM and TIM23 complexes. Here, we report the assembly of the active TOM-TIM23 supercomplex with translocating polypeptide substrates captured in the import pathway. Electron cryo-microscopy (Cryo-EM) analyses reveal that during translocation across the outer membrane, the polypeptide substrates pass through the center of the Tom40 channel while interacting with a glutamine-rich patch in the inner wall of Tom40. Structural and biochemical analyses show that the TIM23 complex contains a heterotrimer of the subunits Tim23, Tim17, and Mgr2 in the inner membrane. Tim17 and Mgr2 shield the polypeptide substrates from the lipid environment. The import pathway consists of two highly conserved residue patches of Tim17, one negatively charged patch at the entrance and one hydrophobic patch in the middle of the pathway. These data reveal an unexpected pre-sequence pathway mediated by the TOM-TIM23 supercomplex for facilitating protein import across the double membranes of mitochondria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546271" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546271">
        <p class="paperTitle">Wolbachia-mediated resistance to Zika virus infection in Aedes aegypti is dominated by diverse transcriptional regulation and weak evolutionary pressures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boehm, E. C.; Jaeger, A. S.; Ries, H. J.; Castaneda, D.; Weiler, A. M.; Valencia, C. C.; Weger-Lucarelli, J.; Ebel, G.; O&#39;Connor, S. L.; Friedrich, T. C.; Zamanian, M.; Aliota, M. T.</p>
        <p class="info">Score: 7.9, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546271' target='https://doi.org/10.1101/2023.06.26.546271'> 10.1101/2023.06.26.546271</a></p>
        <p class="abstract">A promising candidate for arbovirus control and prevention relies on replacing arbovirus-susceptible Aedes aegypti populations with mosquitoes that have been colonized by the intracellular bacterium Wolbachia and thus have a reduced capacity to transmit arboviruses. This reduced capacity to transmit arboviruses is mediated through a phenomenon referred to as pathogen blocking. Pathogen blocking has primarily been proposed as a tool to control dengue virus (DENV) transmission, however it works against a range of viruses, including Zika virus (ZIKV). Despite years of research, the molecular mechanisms underlying pathogen blocking still need to be better understood. Here, we used RNA-seq to characterize mosquito gene transcription dynamics in Ae. aegypti infected with the wMel strain of Wolbachia that are being released by the World Mosquito Program in Medellin, Colombia. Comparative analyses using ZIKV-infected, uninfected tissues, and mosquitoes without Wolbachia revealed that the influence of wMel on mosquito gene transcription is multifactorial. Importantly, because Wolbachia limits, but does not completely prevent, replication of ZIKV and other viruses in coinfected mosquitoes, there is a possibility that these viruses could evolve resistance to pathogen blocking. Therefore, to understand the influence of Wolbachia on within-host ZIKV evolution, we characterized the genetic diversity of molecularly barcoded ZIKV virus populations replicating in Wolbachia-infected mosquitoes and found that within-host ZIKV evolution was subject to weak purifying selection and, unexpectedly, loose anatomical bottlenecks in the presence and absence of Wolbachia. Together, these findings suggest that there is no clear transcriptional profile associated with Wolbachia-mediated ZIKV restriction, and that there is no evidence for ZIKV escape from this restriction in our system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.544844">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.544844" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.544844">
        <p class="paperTitle">Self-reversal facilitates the resolution of HMCES-DNA protein crosslinks in cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.544844" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.544844" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rua-Fernandez, J.; Lovejoy, C. A.; Mehta, K. P. M.; Paulin, K. A.; Toudji, Y. T.; Eichman, B. F.; Cortez, D.</p>
        <p class="info">Score: 8.4, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.544844' target='https://doi.org/10.1101/2023.06.14.544844'> 10.1101/2023.06.14.544844</a></p>
        <p class="abstract">Abasic sites are common DNA lesions that stall polymerases and threaten genome stability. When located in single-stranded DNA (ssDNA), they are shielded from aberrant processing by HMCES via a DNA-protein crosslink (DPC) that prevents double-strand breaks. Nevertheless, the HMCES-DPC must be removed to complete DNA repair. Here, we found that DNA polymerase  inhibition generates ssDNA abasic sites and HMCES-DPCs. These DPCs are resolved with a half-life of approximately 1.5 hours. Resolution does not require the proteasome or SPRTN protease. Instead, HMCES-DPC self-reversal is important for resolution. Biochemically, self-reversal is favored when the ssDNA is converted to duplex DNA. When the self-reversal mechanism is inactivated, HMCES-DPC removal is delayed, cell proliferation is slowed, and cells become hypersensitive to DNA damage agents that increase AP site formation. Thus, HMCES-DPC formation followed by self-reversal is an important mechanism for ssDNA AP site management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545713">
        <p class="paperTitle">N-6-methyladenosine (m6A) Promotes the Nuclear Retention of mRNAs with Intact 5&#39; Splice Site Motifs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, E. S.; Scalize de Olivera, L.; Jomphe, R. Y.; Ihn, S. S.; Smith, H. W.; Nabeel-Shah, S.; Pu, S.; Greenblatt, J. F.; Palazzo, A. F.</p>
        <p class="info">Score: 7.5, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545713' target='https://doi.org/10.1101/2023.06.20.545713'> 10.1101/2023.06.20.545713</a></p>
        <p class="abstract">In eukaryotes, quality control of mRNA represents an important regulatory mechanism for gene expression. Misprocessed mRNAs that contain an intact 5 Splice Site (5SS) motif are retained in the nucleus and targeted for decay. Previously, we showed that the nuclear retention of these transcripts requires ZFC3H1, a component of the Poly(A) Exosome Targeting (PAXT) complex, and U1-70K, a component of the U1 snRNP. In S. pombe, the ZFC3H1 homolog, Red1, binds to the YTH-domain containing protein Mmi1 to target certain RNA transcripts for nuclear retention and decay. Here we show that ZFC3H1 and U1-70K interact with YTHDC1 and YTHDC2, two YTH domain-containing proteins that bind to N-6-methyladenosine (m6A) modified RNAs. We then show that YTHDC1 and YTHDC2 are required for the nuclear retention of mRNAs with intact 5SS motifs. Furthermore, disruption of m6A methyltransferase activity inhibits the nuclear retention of these transcripts. Using m6A-miCLIP analysis, we map m6A methylation marks to intronic polyadenylated (IPA) transcripts, which contain intact 5SS motifs and are nuclear retained and degraded in a ZFC3H1-dependent manner. We find that m6A is enriched near intact 5SS motifs and the poly(A)-tail. Overall, this work suggests that the m6A modification acts as part of an evolutionarily conserved quality control mechanism that targets misprocessed mRNAs for nuclear retention and decay.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
