<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation
Authors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.
Score: 22.9, Published: 2024-02-10 DOI: 10.1101/2024.02.10.579772
Human cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation
Authors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.
Score: 22.9, Published: 2024-02-10 DOI: 10.1101/2024.02.10.579772
Human cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation
Authors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.
Score: 22.9, Published: 2024-02-10 DOI: 10.1101/2024.02.10.579772
Human cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation\nAuthors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.\nScore: 22.9, Published: 2024-02-10 DOI: 10.1101/2024.02.10.579772\nHuman cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry.",
  "keywords": [
    
  ],
  "articleBody": " Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation\nAuthors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.\nScore: 22.9, Published: 2024-02-10 DOI: 10.1101/2024.02.10.579772\nHuman cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry. HCMV vaccine candidates containing gB have demonstrated moderate clinical efficacy, but no HCMV vaccine has been FDA-approved. Here, we used structure-based design to identify and characterize amino acid substitutions that stabilize gB in its metastable prefusion conformation. One variant containing two engineered interprotomer disulfide bonds and two cavity-filling substitutions (gB-C7), displayed increased expression and thermostability. A 2.8 [A] resolution cryo-electron microscopy structure shows that gB-C7 adopts a prefusion-like conformation, revealing additional structural elements at the membrane-distal apex. Unlike previous observations for several class I viral fusion proteins, mice immunized with postfusion or prefusion-stabilized forms of soluble gB protein displayed similar neutralizing antibody titers, here specifically against an HCMV laboratory strain on fibroblasts. Collectively, these results identify initial strategies to stabilize class III viral fusion proteins and provide tools to probe gB-directed antibody responses. TeaserA structure-based design campaign leads to stabilization of the class III viral fusion protein from HCMV in a prefusion-like conformation.\nRewriting endogenous human transcripts with trans-splicing\nAuthors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.\nScore: 84.4, Published: 2024-01-30 DOI: 10.1101/2024.01.29.577779\nSplicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.\nThe N-terminal helix of MarA as a key element in the mechanism of DNA binding\nAuthors: Corbella, M.; Moreira, C.; Bello-Madruga, R.; Torrent, M.; Kamerlin, S. C. L.; Blair, J. M. A.; Sancho-Vaello, E.\nScore: 8.6, Published: 2024-02-13 DOI: 10.1101/2024.02.13.580091\nEfflux is one of the mechanisms employed by Gram-negative bacteria to become resistant to routinely used antibiotics. The inhibition of efflux by targeting their regulators is a promising strategy to re-sensitise bacterial pathogens to antibiotics. AcrAB-TolC is the main Resistance-Nodulation-Division efflux pump in Enterobacteriaceae. MarA is an AraC/XylS family global regulator that regulates more than 40 genes related to the antimicrobial resistance phenotype, including acrAB. The aim of this work was to understand the role of the N-terminal helix of MarA in the mechanism of DNA binding. An N-terminal deletion of MarA showed that the N-terminal helix has a role in the recognition of the functional marboxes. By engineering two double cysteine variants of MarA, and combining in vitro electrophoretic mobility assays and in vivo measurements of acrAB transcription with molecular dynamic simulations, it was shown that the immobilization of the N-terminal helix of MarA prevents binding to DNA. This new mechanism of inhibition seems to be universal for the monomeric members of the AraC/XylS family, as suggested by additional molecular dynamics simulations of the two-domain protein Rob. These results point to the N-terminal helix of the AraC/XylS family monomeric regulators as a promising target for the development of inhibitors.\nA single amino acid in the Salmonella effector SarA/SteE triggers supraphysiological activation of STAT3 for anti-inflammatory target gene expression\nAuthors: Gaggioli, M. R.; Jones, A. G.; Panagi, I.; Washington, E. J.; Loney, R. E.; Muench, J. H.; Brennan, R. G.; Thurston, T. L. M.; Ko, D. C.\nScore: 8.0, Published: 2024-02-14 DOI: 10.1101/2024.02.14.580367\nNon-typhoidal Salmonella enterica cause an estimated 1 million cases of gastroenteritis annually in the United States. These serovars use secreted protein effectors to mimic and reprogram host cellular functions. We previously discovered that the secreted effector SarA (Salmonella anti-inflammatory response activator; also known as SteE) was required for increased intracellular replication of S. Typhimurium and production of the anti-inflammatory cytokine interleukin-10 (IL-10). SarA facilitates phosphorylation of STAT3 through a region of homology with the host cytokine receptor gp130. Here, we demonstrate that a single amino acid difference between SarA and gp130 is critical for the anti-inflammatory bias of SarA-STAT3 signaling. An isoleucine at the pY+1 position of the YxxQ motif in SarA (which binds the SH2 domain in STAT3) causes increased STAT3 phosphorylation and expression of anti-inflammatory target genes. This isoleucine, completely conserved in [~]4000 Salmonella isolates, renders SarA a better substrate for tyrosine phosphorylation by GSK-3. GSK-3 is canonically a serine/threonine kinase that nonetheless undergoes tyrosine autophosphorylation at a motif that has an invariant isoleucine at the pY+1 position. Our results provide a molecular basis for how a Salmonella secreted effector achieves supraphysiological levels of STAT3 activation to control host genes during infection.\nUnlocking prophage potential: In silico and experimental analysis of a novel Mycobacterium fortuitum LysinB containing a peptidoglycan-binding domain\nAuthors: Das, R.; Nadar, K.; Arora, R.; Bajpai, U.\nScore: 7.9, Published: 2024-02-15 DOI: 10.1101/2024.02.15.580446\nEndolysins are highly evolved bacteriophage-encoded lytic enzymes produced to damage the bacterial cell wall for phage progeny release. They offer promising potential as highly specific lytic proteins with a low chance of bacterial resistance. The diversity in lysin sequences and domain organization can be staggering. In silico analysis of bacteriophage and prophage genomes can help identify endolysins exhibiting unique features and high antibacterial activity, hence feeding the pipeline of narrow-spectrum protein antibiotics. Mycobacteriophage lysis cassettes mostly have two lytic enzymes, LysinA and LysinB. The enzyme LysinA targets peptidoglycan in the cell wall and possesses a modular architecture. LysinB typically contains a single domain and acts upon the mycolyl ester linkages in mycolyl-arabinogalactan-peptidoglycan (Payne et al., 2010). This study aimed to find novel LysinBs against Mycobacterium fortuitum. After a detailed in silico characterization of lysis cassettes from three M. fortuitum prophages, we chose to work on a LysinB (hereafter described as LysinB_MF) found in an incomplete prophage (phiE1336, 9.4 kb in strain E1336). LysinB_MF showed low sequence similarity with any other endolysins in the database and formed a separate clade on phylogenetic analysis. LysinB_MFs structure, extracted from the AlphaFold Protein Structure Database, demonstrated a modular architecture with two structurally distinct domains: a peptidoglycan-binding domain (PGBD) at the N-terminal and the characteristic alpha/beta hydrolase domain connected via a linker peptide. We found the alpha/beta hydrolase domain, which is the enzyme-active domain (EAD), contains the conserved Ser-Asp-His catalytic triad with a tunnel-like topology and forms intermolecular hydrogen bonds. The PGBD shows structural similarity to the cell-wall binding domain of an amidase from Clostridium acetobutylicum, hinting at its acquisition due to domain mobility. Our in silico electrostatic potential analysis suggested that PGBD might be essential to the enzyme activity. This was experimentally validated by generating a truncated version of the enzyme, which demonstrated about six-fold decreased activity compared to its native form. The antimycobacterial activity of this enzyme was also compromised in its absence. Based on our analysis, PGBD emerged as an integral constituent of enzymes with diverse functional properties and is predicted to be a conserved cross-kingdom. Overall, this study highlights the importance of mining mycobacterial prophages as a novel endolysin source. It also provides unique insights into the diverse architecture of mycobacteriophage-encoded endolysins and the importance of functional domains for their catalytic activities.\nProgrammable RNA writing with trans-splicing\nAuthors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.\nScore: 128.3, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223\nRNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing \u0026 Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5' or 3' trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington's disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntingtons disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.\nLoop extrusion-mediated plasmid DNA cleavage by the bacterial SMC Wadjet complex\nAuthors: Pradhan, B.; Deep, A.; Koenig, J.; Baaske, M.; Corbett, K.; Kim, E.\nScore: 6.2, Published: 2024-02-17 DOI: 10.1101/2024.02.17.580791\nStructural maintenance of chromosomes (SMC) protein complexes play pivotal roles in genome organization and maintenance across all domains of life. In prokaryotes, SMC family Wadjet complexes structurally resemble the widespread MukBEF genome-organizing complexes but serve a defensive role by inhibiting plasmid transformation. We previously showed that Wadjet specifically cleaves circular DNA; however, the molecular mechanism underlying DNA substrate recognition remains unclear. Here, we use in vitro single-molecule imaging to directly visualize DNA loop extrusion and plasmid cleavage by Wadjet. We find that Wadjet is a symmetric DNA loop extruder that simultaneously reels in DNA from both sides of a growing loop and that this activity requires a dimeric JetABC supercomplex containing two dimers of the JetC motor subunit. On surface-anchored plasmid DNAs, Wadjet extrudes the full length of a 44 kilobase pair plasmid, stalls, and then cleaves DNA. Our findings reveal the role of loop extrusion in the specific recognition and elimination of plasmids by Wadjet, and establish loop extrusion as an evolutionarily conserved mechanism among SMC complexes across kingdoms of life.\nProtein UFMylation regulates early events of ribosomal DNA double-stranded break response\nAuthors: Panichnantakul, P.; Aguilar, L. C.; Daynard, E.; Guest, M.; Peters, C.; Vogel, J.; Oeffinger, M.\nScore: 5.2, Published: 2024-02-14 DOI: 10.1101/2024.02.14.580383\nThe highly repetitive and transcriptionally active ribosomal DNA (rDNA) genes are exceedingly susceptible to genotoxic stress. Induction of DNA double-strand breaks (DSBs) in rDNA repeats is associated with ATM-dependent rDNA silencing and nucleolar reorganization where rDNA is segregated into nucleolar caps. However, the regulatory events underlying this response remain elusive. Here, we identify protein UFMylation as essential for rDNA damage response in human cells. We further show the only UFM1-E3-ligase UFL1 and its binding partner DDRGK1 localize to nucleolar caps upon rDNA damage, and that UFL1 loss impairs ATM activation and rDNA transcriptional silencing, leading to reduced rDNA segregation. A first-ever analysis of nuclear and nucleolar UFMylation targets in response to DSBs induction further identified key DNA repair factors including ATM, in addition to chromatin and actin network regulators. Taken together, our data provides the first evidence of an essential role for UFMylation in orchestrating rDNA DSB repair.\nOGT prevents DNA demethylation and suppresses the expression of transposable elements in heterochromatin by restraining TET activity genome-wide\nAuthors: Sepulveda, H.; Li, X.; Yue, X.; Angel, J. C.; Arteaga-Vazquez, L. J.; Brown, C.; Brunelli, M.; Jansz, N.; Puddu, F.; Scotcher, J.; Creed, P.; Kennedy, P. H.; Manriquez, C.; Myers, S. A.; Crawford, R.; Faulkner, G. J.; Rao, A.\nScore: 5.1, Published: 2024-01-31 DOI: 10.1101/2024.01.31.578097\nThe O-GlcNAc transferase OGT interacts robustly with all three mammalian TET methylcytosine dioxygenases. We show here that deletion of the Ogt gene in mouse embryonic stem cells (mESC) results in a widespread increase in the TET product 5-hydroxymethylcytosine (5hmC) in both euchromatic and heterochromatic compartments, with concomitant reduction of the TET substrate 5-methylcytosine (5mC) at the same genomic regions. mESC engineered to abolish the TET1-OGT interaction likewise displayed a genome-wide decrease of 5mC. DNA hypomethylation in OGT-deficient cells was accompanied by de-repression of transposable elements (TEs) predominantly located in heterochromatin, and this increase in TE expression was sometimes accompanied by increased cis-expression of genes and exons located 3 of the expressed TE. Thus, the TET-OGT interaction prevents DNA demethylation and TE expression in heterochromatin by restraining TET activity genome-wide. We suggest that OGT protects the genome against DNA hypomethylation and impaired heterochromatin integrity, preventing the aberrant increase in TE expression observed in cancer, autoimmune-inflammatory diseases, cellular senescence and ageing.\nTranslational response to mitochondrial stresses is orchestrated by tRNA modifications.\nAuthors: Rashad, S.; Al-Mesitef, S.; Mousa, A.; Zhou, Y.; Ando, D.; Sun, G.; Fukuuchi, T.; Iwasaki, Y.; Xiang, J.; Byrne, S. R.; Sun, J.; Maekawa, M.; Saigusa, D.; Begley, T.; Dedon, P.; Niizuma, K.\nScore: 4.5, Published: 2024-02-14 DOI: 10.1101/2024.02.14.580389\nMitochondrial stress and dysfunction play important roles in many pathologies. However, how cells respond to mitochondrial stress is not fully understood. Here, we examined the translational response to electron transport chain (ETC) inhibition and arsenite induced mitochondrial stresses. Our analysis revealed that during mitochondrial stress, tRNA modifications (namely f5C, hm5C, queuosine and its derivatives, and mcm5U) dynamically change to fine tune codon decoding, usage, and optimality. These changes in codon optimality drive the translation of many pathways and gene sets, such as the ATF4 pathway and selenoproteins, involved in the cellular response to mitochondrial stress. We further examined several of these modifications using targeted approaches. ALKBH1 knockout (KO) abrogated f5C and hm5C levels and led to mitochondrial dysfunction, reduced proliferation, and impacted mRNA translation rates. Our analysis revealed that tRNA queuosine (tRNA-Q) is a master regulator of the mitochondrial stress response. KO of QTRT1 or QTRT2, the enzymes responsible for tRNA-Q synthesis, led to mitochondrial dysfunction, translational dysregulation, and metabolic alterations in mitochondria-related pathways, without altering cellular proliferation. In addition, our analysis revealed that tRNA-Q loss led to a domino effect on various tRNA modifications. Some of these changes could be explained by metabolic profiling. Our analysis also revealed that utilizing serum deprivation or alteration with Queuine supplementation to study tRNA-Q or stress response can introduce various confounding factors by altering many other tRNA modifications. In summary, our data show that tRNA modifications are master regulators of the mitochondrial stress response by driving changes in codon decoding.\n",
  "wordCount" : "2586",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579772" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579772">
        <p class="paperTitle">Structure-based design of a soluble human cytomegalovirus glycoprotein B antigen stabilized in a prefusion-like conformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sponholtz, M. R.; Byrne, P. O.; Lee, A. G.; Ramamohan, A. R.; Goldsmith, J. A.; McCool, R. S.; Zhou, L.; Johnson, N. V.; Hsieh, C.-L.; Connors, M.; Karthigeyan, K.; Campbell, J. D.; Permar, S. R.; Maynard, J. A.; Yu, D.; Bottomley, M. J.; McLellan, J. S.</p>
        <p class="info">Score: 22.9, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579772' target='https://doi.org/10.1101/2024.02.10.579772'> 10.1101/2024.02.10.579772</a></p>
        <p class="abstract">Human cytomegalovirus (HCMV) glycoprotein B (gB) is a class III membrane fusion protein required for viral entry. HCMV vaccine candidates containing gB have demonstrated moderate clinical efficacy, but no HCMV vaccine has been FDA-approved. Here, we used structure-based design to identify and characterize amino acid substitutions that stabilize gB in its metastable prefusion conformation. One variant containing two engineered interprotomer disulfide bonds and two cavity-filling substitutions (gB-C7), displayed increased expression and thermostability. A 2.8 [A] resolution cryo-electron microscopy structure shows that gB-C7 adopts a prefusion-like conformation, revealing additional structural elements at the membrane-distal apex. Unlike previous observations for several class I viral fusion proteins, mice immunized with postfusion or prefusion-stabilized forms of soluble gB protein displayed similar neutralizing antibody titers, here specifically against an HCMV laboratory strain on fibroblasts. Collectively, these results identify initial strategies to stabilize class III viral fusion proteins and provide tools to probe gB-directed antibody responses.

TeaserA structure-based design campaign leads to stabilization of the class III viral fusion protein from HCMV in a prefusion-like conformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577779">
        <p class="paperTitle">Rewriting endogenous human transcripts with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 84.4, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577779' target='https://doi.org/10.1101/2024.01.29.577779'> 10.1101/2024.01.29.577779</a></p>
        <p class="abstract">Splicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580091" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580091">
        <p class="paperTitle">The N-terminal helix of MarA as a key element in the mechanism of DNA binding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Corbella, M.; Moreira, C.; Bello-Madruga, R.; Torrent, M.; Kamerlin, S. C. L.; Blair, J. M. A.; Sancho-Vaello, E.</p>
        <p class="info">Score: 8.6, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580091' target='https://doi.org/10.1101/2024.02.13.580091'> 10.1101/2024.02.13.580091</a></p>
        <p class="abstract">Efflux is one of the mechanisms employed by Gram-negative bacteria to become resistant to routinely used antibiotics. The inhibition of efflux by targeting their regulators is a promising strategy to re-sensitise bacterial pathogens to antibiotics. AcrAB-TolC is the main Resistance-Nodulation-Division efflux pump in Enterobacteriaceae. MarA is an AraC/XylS family global regulator that regulates more than 40 genes related to the antimicrobial resistance phenotype, including acrAB. The aim of this work was to understand the role of the N-terminal helix of MarA in the mechanism of DNA binding. An N-terminal deletion of MarA showed that the N-terminal helix has a role in the recognition of the functional marboxes. By engineering two double cysteine variants of MarA, and combining in vitro electrophoretic mobility assays and in vivo measurements of acrAB transcription with molecular dynamic simulations, it was shown that the immobilization of the N-terminal helix of MarA prevents binding to DNA. This new mechanism of inhibition seems to be universal for the monomeric members of the AraC/XylS family, as suggested by additional molecular dynamics simulations of the two-domain protein Rob. These results point to the N-terminal helix of the AraC/XylS family monomeric regulators as a promising target for the development of inhibitors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580367">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580367" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580367">
        <p class="paperTitle">A single amino acid in the Salmonella effector SarA/SteE triggers supraphysiological activation of STAT3 for anti-inflammatory target gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580367" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580367" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gaggioli, M. R.; Jones, A. G.; Panagi, I.; Washington, E. J.; Loney, R. E.; Muench, J. H.; Brennan, R. G.; Thurston, T. L. M.; Ko, D. C.</p>
        <p class="info">Score: 8.0, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580367' target='https://doi.org/10.1101/2024.02.14.580367'> 10.1101/2024.02.14.580367</a></p>
        <p class="abstract">Non-typhoidal Salmonella enterica cause an estimated 1 million cases of gastroenteritis annually in the United States. These serovars use secreted protein effectors to mimic and reprogram host cellular functions. We previously discovered that the secreted effector SarA (Salmonella anti-inflammatory response activator; also known as SteE) was required for increased intracellular replication of S. Typhimurium and production of the anti-inflammatory cytokine interleukin-10 (IL-10). SarA facilitates phosphorylation of STAT3 through a region of homology with the host cytokine receptor gp130. Here, we demonstrate that a single amino acid difference between SarA and gp130 is critical for the anti-inflammatory bias of SarA-STAT3 signaling. An isoleucine at the pY&#43;1 position of the YxxQ motif in SarA (which binds the SH2 domain in STAT3) causes increased STAT3 phosphorylation and expression of anti-inflammatory target genes. This isoleucine, completely conserved in [~]4000 Salmonella isolates, renders SarA a better substrate for tyrosine phosphorylation by GSK-3. GSK-3 is canonically a serine/threonine kinase that nonetheless undergoes tyrosine autophosphorylation at a motif that has an invariant isoleucine at the pY&#43;1 position. Our results provide a molecular basis for how a Salmonella secreted effector achieves supraphysiological levels of STAT3 activation to control host genes during infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580446">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580446" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580446">
        <p class="paperTitle">Unlocking prophage potential: In silico and experimental analysis of a novel Mycobacterium fortuitum LysinB containing a peptidoglycan-binding domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580446" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580446" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Das, R.; Nadar, K.; Arora, R.; Bajpai, U.</p>
        <p class="info">Score: 7.9, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580446' target='https://doi.org/10.1101/2024.02.15.580446'> 10.1101/2024.02.15.580446</a></p>
        <p class="abstract">Endolysins are highly evolved bacteriophage-encoded lytic enzymes produced to damage the bacterial cell wall for phage progeny release. They offer promising potential as highly specific lytic proteins with a low chance of bacterial resistance. The diversity in lysin sequences and domain organization can be staggering. In silico analysis of bacteriophage and prophage genomes can help identify endolysins exhibiting unique features and high antibacterial activity, hence feeding the pipeline of narrow-spectrum protein antibiotics. Mycobacteriophage lysis cassettes mostly have two lytic enzymes, LysinA and LysinB. The enzyme LysinA targets peptidoglycan in the cell wall and possesses a modular architecture. LysinB typically contains a single domain and acts upon the mycolyl ester linkages in mycolyl-arabinogalactan-peptidoglycan (Payne et al., 2010). This study aimed to find novel LysinBs against Mycobacterium fortuitum. After a detailed in silico characterization of lysis cassettes from three M. fortuitum prophages, we chose to work on a LysinB (hereafter described as LysinB_MF) found in an incomplete prophage (phiE1336, 9.4 kb in strain E1336). LysinB_MF showed low sequence similarity with any other endolysins in the database and formed a separate clade on phylogenetic analysis. LysinB_MFs structure, extracted from the AlphaFold Protein Structure Database, demonstrated a modular architecture with two structurally distinct domains: a peptidoglycan-binding domain (PGBD) at the N-terminal and the characteristic alpha/beta hydrolase domain connected via a linker peptide. We found the alpha/beta hydrolase domain, which is the enzyme-active domain (EAD), contains the conserved Ser-Asp-His catalytic triad with a tunnel-like topology and forms intermolecular hydrogen bonds. The PGBD shows structural similarity to the cell-wall binding domain of an amidase from Clostridium acetobutylicum, hinting at its acquisition due to domain mobility. Our in silico electrostatic potential analysis suggested that PGBD might be essential to the enzyme activity. This was experimentally validated by generating a truncated version of the enzyme, which demonstrated about six-fold decreased activity compared to its native form. The antimycobacterial activity of this enzyme was also compromised in its absence. Based on our analysis, PGBD emerged as an integral constituent of enzymes with diverse functional properties and is predicted to be a conserved cross-kingdom. Overall, this study highlights the importance of mining mycobacterial prophages as a novel endolysin source. It also provides unique insights into the diverse architecture of mycobacteriophage-encoded endolysins and the importance of functional domains for their catalytic activities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578223" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578223">
        <p class="paperTitle">Programmable RNA writing with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.</p>
        <p class="info">Score: 128.3, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578223' target='https://doi.org/10.1101/2024.01.31.578223'> 10.1101/2024.01.31.578223</a></p>
        <p class="abstract">RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington&#39;s disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntingtons disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.17.580791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.17.580791" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.17.580791">
        <p class="paperTitle">Loop extrusion-mediated plasmid DNA cleavage by the bacterial SMC Wadjet complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.17.580791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.17.580791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pradhan, B.; Deep, A.; Koenig, J.; Baaske, M.; Corbett, K.; Kim, E.</p>
        <p class="info">Score: 6.2, Published: 2024-02-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.17.580791' target='https://doi.org/10.1101/2024.02.17.580791'> 10.1101/2024.02.17.580791</a></p>
        <p class="abstract">Structural maintenance of chromosomes (SMC) protein complexes play pivotal roles in genome organization and maintenance across all domains of life. In prokaryotes, SMC family Wadjet complexes structurally resemble the widespread MukBEF genome-organizing complexes but serve a defensive role by inhibiting plasmid transformation. We previously showed that Wadjet specifically cleaves circular DNA; however, the molecular mechanism underlying DNA substrate recognition remains unclear. Here, we use in vitro single-molecule imaging to directly visualize DNA loop extrusion and plasmid cleavage by Wadjet. We find that Wadjet is a symmetric DNA loop extruder that simultaneously reels in DNA from both sides of a growing loop and that this activity requires a dimeric JetABC supercomplex containing two dimers of the JetC motor subunit. On surface-anchored plasmid DNAs, Wadjet extrudes the full length of a 44 kilobase pair plasmid, stalls, and then cleaves DNA. Our findings reveal the role of loop extrusion in the specific recognition and elimination of plasmids by Wadjet, and establish loop extrusion as an evolutionarily conserved mechanism among SMC complexes across kingdoms of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580383">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580383" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580383">
        <p class="paperTitle">Protein UFMylation regulates early events of ribosomal DNA double-stranded break response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580383" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580383" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Panichnantakul, P.; Aguilar, L. C.; Daynard, E.; Guest, M.; Peters, C.; Vogel, J.; Oeffinger, M.</p>
        <p class="info">Score: 5.2, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580383' target='https://doi.org/10.1101/2024.02.14.580383'> 10.1101/2024.02.14.580383</a></p>
        <p class="abstract">The highly repetitive and transcriptionally active ribosomal DNA (rDNA) genes are exceedingly susceptible to genotoxic stress. Induction of DNA double-strand breaks (DSBs) in rDNA repeats is associated with ATM-dependent rDNA silencing and nucleolar reorganization where rDNA is segregated into nucleolar caps. However, the regulatory events underlying this response remain elusive. Here, we identify protein UFMylation as essential for rDNA damage response in human cells. We further show the only UFM1-E3-ligase UFL1 and its binding partner DDRGK1 localize to nucleolar caps upon rDNA damage, and that UFL1 loss impairs ATM activation and rDNA transcriptional silencing, leading to reduced rDNA segregation. A first-ever analysis of nuclear and nucleolar UFMylation targets in response to DSBs induction further identified key DNA repair factors including ATM, in addition to chromatin and actin network regulators. Taken together, our data provides the first evidence of an essential role for UFMylation in orchestrating rDNA DSB repair.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578097" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578097">
        <p class="paperTitle">OGT prevents DNA demethylation and suppresses the expression of transposable elements in heterochromatin by restraining TET activity genome-wide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sepulveda, H.; Li, X.; Yue, X.; Angel, J. C.; Arteaga-Vazquez, L. J.; Brown, C.; Brunelli, M.; Jansz, N.; Puddu, F.; Scotcher, J.; Creed, P.; Kennedy, P. H.; Manriquez, C.; Myers, S. A.; Crawford, R.; Faulkner, G. J.; Rao, A.</p>
        <p class="info">Score: 5.1, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578097' target='https://doi.org/10.1101/2024.01.31.578097'> 10.1101/2024.01.31.578097</a></p>
        <p class="abstract">The O-GlcNAc transferase OGT interacts robustly with all three mammalian TET methylcytosine dioxygenases. We show here that deletion of the Ogt gene in mouse embryonic stem cells (mESC) results in a widespread increase in the TET product 5-hydroxymethylcytosine (5hmC) in both euchromatic and heterochromatic compartments, with concomitant reduction of the TET substrate 5-methylcytosine (5mC) at the same genomic regions. mESC engineered to abolish the TET1-OGT interaction likewise displayed a genome-wide decrease of 5mC. DNA hypomethylation in OGT-deficient cells was accompanied by de-repression of transposable elements (TEs) predominantly located in heterochromatin, and this increase in TE expression was sometimes accompanied by increased cis-expression of genes and exons located 3 of the expressed TE. Thus, the TET-OGT interaction prevents DNA demethylation and TE expression in heterochromatin by restraining TET activity genome-wide. We suggest that OGT protects the genome against DNA hypomethylation and impaired heterochromatin integrity, preventing the aberrant increase in TE expression observed in cancer, autoimmune-inflammatory diseases, cellular senescence and ageing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580389" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580389">
        <p class="paperTitle">Translational response to mitochondrial stresses is orchestrated by tRNA modifications.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rashad, S.; Al-Mesitef, S.; Mousa, A.; Zhou, Y.; Ando, D.; Sun, G.; Fukuuchi, T.; Iwasaki, Y.; Xiang, J.; Byrne, S. R.; Sun, J.; Maekawa, M.; Saigusa, D.; Begley, T.; Dedon, P.; Niizuma, K.</p>
        <p class="info">Score: 4.5, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580389' target='https://doi.org/10.1101/2024.02.14.580389'> 10.1101/2024.02.14.580389</a></p>
        <p class="abstract">Mitochondrial stress and dysfunction play important roles in many pathologies. However, how cells respond to mitochondrial stress is not fully understood. Here, we examined the translational response to electron transport chain (ETC) inhibition and arsenite induced mitochondrial stresses. Our analysis revealed that during mitochondrial stress, tRNA modifications (namely f5C, hm5C, queuosine and its derivatives, and mcm5U) dynamically change to fine tune codon decoding, usage, and optimality. These changes in codon optimality drive the translation of many pathways and gene sets, such as the ATF4 pathway and selenoproteins, involved in the cellular response to mitochondrial stress. We further examined several of these modifications using targeted approaches. ALKBH1 knockout (KO) abrogated f5C and hm5C levels and led to mitochondrial dysfunction, reduced proliferation, and impacted mRNA translation rates. Our analysis revealed that tRNA queuosine (tRNA-Q) is a master regulator of the mitochondrial stress response. KO of QTRT1 or QTRT2, the enzymes responsible for tRNA-Q synthesis, led to mitochondrial dysfunction, translational dysregulation, and metabolic alterations in mitochondria-related pathways, without altering cellular proliferation. In addition, our analysis revealed that tRNA-Q loss led to a domino effect on various tRNA modifications. Some of these changes could be explained by metabolic profiling. Our analysis also revealed that utilizing serum deprivation or alteration with Queuine supplementation to study tRNA-Q or stress response can introduce various confounding factors by altering many other tRNA modifications. In summary, our data show that tRNA modifications are master regulators of the mitochondrial stress response by driving changes in codon decoding.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
