<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Antagonistic conflict between transposon-encoded introns and guide RNAs
Authors: Zedaveinyte, R.; Meers, C.; Le, H. C.; Mortman, E. E.; Tang, S.; Lampe, G. D.; Pesari, S. R.; Gelsinger, D. R.; Wiegand, T.; Sternberg, S. H.
Score: 20.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567912
TnpB nucleases represent the evolutionary precursors to CRISPR-Cas12 and are widespread in all domains of life, presumably due to the critical roles they play in transposon proliferation. IS605-family TnpB homologs function in bacteria as programmable homing endonucleases by exploiting transposon-encoded guide RNAs to cleave vacant genomic sites, thereby driving transposon maintenance through DSB-stimulated homologous recombination.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Antagonistic conflict between transposon-encoded introns and guide RNAs
Authors: Zedaveinyte, R.; Meers, C.; Le, H. C.; Mortman, E. E.; Tang, S.; Lampe, G. D.; Pesari, S. R.; Gelsinger, D. R.; Wiegand, T.; Sternberg, S. H.
Score: 20.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567912
TnpB nucleases represent the evolutionary precursors to CRISPR-Cas12 and are widespread in all domains of life, presumably due to the critical roles they play in transposon proliferation. IS605-family TnpB homologs function in bacteria as programmable homing endonucleases by exploiting transposon-encoded guide RNAs to cleave vacant genomic sites, thereby driving transposon maintenance through DSB-stimulated homologous recombination." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Antagonistic conflict between transposon-encoded introns and guide RNAs
Authors: Zedaveinyte, R.; Meers, C.; Le, H. C.; Mortman, E. E.; Tang, S.; Lampe, G. D.; Pesari, S. R.; Gelsinger, D. R.; Wiegand, T.; Sternberg, S. H.
Score: 20.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567912
TnpB nucleases represent the evolutionary precursors to CRISPR-Cas12 and are widespread in all domains of life, presumably due to the critical roles they play in transposon proliferation. IS605-family TnpB homologs function in bacteria as programmable homing endonucleases by exploiting transposon-encoded guide RNAs to cleave vacant genomic sites, thereby driving transposon maintenance through DSB-stimulated homologous recombination."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Antagonistic conflict between transposon-encoded introns and guide RNAs\nAuthors: Zedaveinyte, R.; Meers, C.; Le, H. C.; Mortman, E. E.; Tang, S.; Lampe, G. D.; Pesari, S. R.; Gelsinger, D. R.; Wiegand, T.; Sternberg, S. H.\nScore: 20.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567912\nTnpB nucleases represent the evolutionary precursors to CRISPR-Cas12 and are widespread in all domains of life, presumably due to the critical roles they play in transposon proliferation. IS605-family TnpB homologs function in bacteria as programmable homing endonucleases by exploiting transposon-encoded guide RNAs to cleave vacant genomic sites, thereby driving transposon maintenance through DSB-stimulated homologous recombination.",
  "keywords": [
    
  ],
  "articleBody": " Antagonistic conflict between transposon-encoded introns and guide RNAs\nAuthors: Zedaveinyte, R.; Meers, C.; Le, H. C.; Mortman, E. E.; Tang, S.; Lampe, G. D.; Pesari, S. R.; Gelsinger, D. R.; Wiegand, T.; Sternberg, S. H.\nScore: 20.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567912\nTnpB nucleases represent the evolutionary precursors to CRISPR-Cas12 and are widespread in all domains of life, presumably due to the critical roles they play in transposon proliferation. IS605-family TnpB homologs function in bacteria as programmable homing endonucleases by exploiting transposon-encoded guide RNAs to cleave vacant genomic sites, thereby driving transposon maintenance through DSB-stimulated homologous recombination. Whether this pathway is conserved in other genetic contexts, and in association with other transposases, is unknown. Here we uncover molecular mechanisms of transposition and RNA-guided DNA cleavage by IS607-family elements that, remarkably, also encode catalytic, self-splicing group I introns. After reconstituting and systematically investigating each of these biochemical activities for a candidate IStron derived from Clostridium botulinum, we discovered sequence and structural features of the transposon-encoded RNA that satisfy molecular requirements of a group I intron and TnpB guide RNA, while still retaining the ability to be faithfully mobilized at the DNA level by the TnpA transposase. Strikingly, intron splicing was strongly repressed not only by TnpB, but also by the secondary structure of {omega}RNA alone, allowing the element to carefully control the relative levels of spliced products versus functional guide RNAs. Our results suggest that IStron transcripts have evolved a sensitive equilibrium to balance competing and mutually exclusive activities that promote transposon maintenance while limiting adverse fitness costs on the host. Collectively, this work explains how diverse enzymatic activities emerged during the selfish spread of IS607-family elements and highlights molecular innovation in the multi-functional utility of transposon-encoded noncoding RNAs.\nSpatial transcriptomics defines injury-specific microenvironments in the adult mouse kidney and novel cellular interactions in regeneration and disease\nAuthors: Polonsky, M.; Gerhardt, L. M. S.; Yun, J.; Koppitch, K.; Colon, K.; Amrhein, H.; Zheng, S.; Yuan, G.-C.; Thomson, M.; Cai, L.; McMahon, A. P.\nScore: 16.9, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568217\nKidney injury disrupts the intricate renal architecture and triggers limited regeneration, and injury-invoked inflammation and fibrosis. Deciphering molecular pathways and cellular interactions driving these processes is challenging due to the complex renal architecture. Here, we applied single cell spatial transcriptomics to examine ischemia-reperfusion injury in the mouse kidney. Spatial transcriptomics revealed injury-specific and spatially-dependent gene expression patterns in distinct cellular microenvironments within the kidney and predicted Clcf1-Crfl1 in a molecular interplay between persistently injured proximal tubule cells and neighboring fibroblasts. Immune cell types play a critical role in organ repair. Spatial analysis revealed cellular microenvironments resembling early tertiary lymphoid structures and identified associated molecular pathways. Collectively, this study supports a focus on molecular interactions in cellular microenvironments to enhance understanding of injury, repair and disease.\nReversible histone deacetylase activity catalyzes lysine acylation\nAuthors: Tsusaka, T.; Najar, M. A.; Schwarz, B.; Bohrnsen, E.; Oses-Prieto, J. A.; Lee, C.; Burlingame, A. L.; Bosio, C. M.; Burslem, G. M.; Goldberg, E. L.\nScore: 33.3, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567549\nStarvation and low carbohydrate diets lead to the accumulation of the ketone body, {beta}-hydroxybutyrate (BHB), whose blood concentrations increase more than 10-fold into the millimolar range. In addition to providing a carbon source, BHB accumulation triggers lysine {beta}-hydroxybutyrylation (Kbhb) of proteins via unknown mechanisms. As with other lysine acylation events, Kbhb marks can be removed by histone deacetylases (HDACs). Here, we report that class I HDACs unexpectedly catalyze protein lysine modification with {beta}-hydroxybutyrate (BHB). Mutational analyses of the HDAC2 active site reveal a shared reliance on key amino acids for classical deacetylation and non-canonical HDAC-catalyzed {beta}-hydroxybutyrylation. Also consistent with reverse HDAC activity, Kbhb formation is driven by mass action and substrate availability. This reverse HDAC activity is not limited to BHB but also extends to multiple short-chain fatty acids. The reversible activity of class I HDACs described here represents a novel mechanism of PTM deposition relevant to metabolically-sensitive proteome modifications.\nA C-terminal HaloTag Impairs AGO2 Function\nAuthors: Shah, K. M.; Crozier, A. F. F.; Assaraf, A. A.; Arya, M.; Ribeiro, P. B.; Plevin, M. J.; Sharp, T. V.\nScore: 8.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567203\nA full understanding of RNA silencing requires appropriate molecular biology tools to explore the roles of Argonaute 2 (AGO2) and the RNA-induced silencing complex (RISC). Commonly used approaches to study RNA silencing and RISC, such as those relying on affinity tagging and antibodies, have important limitations that can lead to artificial results. Both the N- and C-terminal domains of AGO2 have been shown to be important for correct activity and yet the consequences of appending tags to either terminus have not been fully investigated. N-terminal tags are frequently used to study AGO(2) biology. Recently, an N-terminal HaloTag-Ago2 fusion was reported and examined in mice. While the versatile HaloTag provided new opportunities to study RISC biology, the tagged construct showed certain activity changes compared to unmodified Ago2. CRISPaint, a new CRISPR-Cas9 technique, offers a route to the accurate and efficient generation of endogenous C-terminal tag fusions. Here, we used CRISPaint to generate the first reported recombinant AGO2 construct with a C-terminal tag: an endogenous C-terminal HaloTag fusion to AGO2 (AGO2HALO) in human (A549) cells. We found that the AGO2HALO fusion protein retains the capacity to interact with the key protein binding partner TNRC6A and that the C-terminal HaloTag does not affect cell viability. However, the AGO2HALO fusion significantly impairs RNA cleavage and RNA silencing activity compared to control cells, and reduces nuclear localisation of the fusion protein. We conclude that the fusion of a C-terminal HaloTag to AGO2 is not appropriate for studying AGO2 and RISC. Our results stress the importance of fully validating recombinant tagging strategies to ensure that any results generated do not obscure critical functional defects.\nMsc1 is a nuclear envelope protein that reinforces DNA repair in late mitosis.\nAuthors: Medina-Suarez, S.; Ayra-Plasencia, J.; Perez-Martinez, L.; Butter, F.; Machin, F.\nScore: 8.3, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568106\nPrecise double-strand break (DSB) repair is paramount for genome stability. Homologous recombination (HR) repairs DSBs when cyclin-dependent kinase (CDK) activity is high, which correlates with the availability of the sister chromatid as a template. However, anaphase and telophase are paradoxical scenarios since high CDK favors HR despite sister chromatids being no longer aligned. To identify factors specifically involved in DSB repair in late mitosis, we have undertaken comparative proteomics in Saccharomyces cerevisiae and found that Msc1, a poorly characterized nuclear envelope protein, is significantly enriched upon both random and guided DSBs. We further show that {Delta}msc1 is more sensitive to DSBs in late mitosis, and has a delayed repair of DBSs, as indicated by increased Rad53 hyperphosphorylation, fewer Rad52 repair factories, and slower HR completion. We discuss how Msc1 may favor the formation of Rad52 factories and the timely completion of HR before cytokinesis.\nStructural basis for polyuridine tract recognition by SARS-CoV-2 Nsp15\nAuthors: Ito, F.; Yang, H.; Zhou, Z. H.; Chen, X. S.\nScore: 6.8, Published: 2023-11-20 DOI: 10.1101/2023.11.17.567629\nSARS-CoV-2 non-structural protein 15 (Nsp15) is critical for productive viral replication and evasion of host immunity. The uridine-specific endoribonuclease activity of Nsp15 mediates the cleavage of the polyuridine [poly(U)] tract of the negative-strand coronavirus genome to minimize the formation of dsRNA that activates the host antiviral interferon signaling. However, the molecular basis for the recognition and cleavage of the poly(U) tract by Nsp15 is incompletely understood. Here, we present cryogenic electron microscopy (cryoEM) structures of SARS-CoV-2 Nsp15 bound to viral replication intermediate dsRNA containing poly(U) tract at 2.7-3.3 [A] resolution. The structures reveal one copy of dsRNA binds to the sidewall of an Nsp15 homohexamer, spanning three subunits in two distinct binding states. The target uracil is dislodged from the base-pairing of the dsRNA by amino acid residues W332 and M330 of Nsp15, and the dislodged base is entrapped at the endonuclease active site center. Up to 20 A/U base pairs are anchored on the Nsp15 hexamer, which explains the basis for a substantially shortened poly(U) sequence in the negative strand coronavirus genome compared to the long poly(A) tail in its positive strand. Our results provide mechanistic insights into the unique immune evasion strategy employed by coronavirus Nsp15.\nEnhanced surface accessibility of SARS-CoV-2 Omicron spike protein due to an altered glycosylation profile\nAuthors: Wang, D.; Zhang, Z.; Baudys, J.; Haynes, C. A.; Osman, S. H.; Zhou, B.; Barr, J. R.; Gumbart, J. C.\nScore: 6.6, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568381\nSARS-CoV-2 spike (S) proteins undergo extensive glycosylation, aiding proper folding, enhancing stability, and evading host immune surveillance. In this study, we used mass spectrometric analysis to elucidate the N-glycosylation characteristics and disulfide bonding of recombinant spike proteins derived from the SARS-CoV-2 Omicron variant (B.1.1.529) in comparison with the D614G spike variant. Furthermore, we conducted microsecond-long molecular dynamics simulations on spike proteins to resolve how the different N-glycans impact spike conformational sampling in the two variants. Our findings reveal that the Omicron spike protein maintains an overall resemblance to the D614G spike variant in terms of site-specific glycan processing and disulfide bond formation. Nonetheless, alterations in glycans were observed at certain N-glycosylation sites. These changes, in synergy with mutations within the Omicron spike protein, result in increased surface accessibility of the macromolecule, including ectodomain, receptor-binding domain, and N-terminal domain. These insights contribute to our understanding of the interplay between structure and function, thereby advancing effective vaccination and therapeutic strategies.\nSymbiosis modulates gene expression of symbionts, but not hosts, under thermal challenge\nAuthors: Aichelman, H. E.; Huzar, A. K.; Wuitchik, D. M.; Atherton, K. F.; Wright, R. M.; Dixon, G. B.; Schlatter, E.; Haftel, N.; Davies, S. W.\nScore: 6.6, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568130\nIncreasing ocean temperatures are causing dysbiosis between coral hosts and their symbionts. Previous work suggests that coral host gene expression responds more strongly to environmental stress compared to their intracellular symbionts; however, the causes and consequences of this phenomenon remain untested. We hypothesized that symbionts are less responsive because hosts modulate symbiont environments to buffer stress. To test this hypothesis, we leveraged the facultative symbiosis between the scleractinian coral Oculina arbuscula and its symbiont Breviolum psygmophilum to characterize gene expression responses of both symbiotic partners in and ex hospite under thermal challenges. To characterize host and in hospite symbiont responses, symbiotic and aposymbiotic O. arbuscula were exposed to three treatments: 1) control (18{degrees}C), 2) heat (32{degrees}C), and 3) cold (6{degrees}C). This experiment was replicated with B. psygmophilum cultured from O. arbuscula to characterize ex hospite symbiont responses. Both thermal challenges elicited classic environmental stress responses (ESRs) in O. arbuscula regardless of symbiotic state, with hosts responding more strongly to cold challenge. Hosts also exhibited stronger responses than in hospite symbionts. In and ex hospite B. psygmophilum both downregulated genes associated with photosynthesis under thermal challenge; however, ex hospite symbionts exhibited greater gene expression plasticity and differential expression of genes associated with ESRs. Taken together, these findings suggest that O. arbuscula hosts may buffer environments of B. psygmophilum symbionts; however, we outline the future work needed to confirm this hypothesis.\nEfficient scar-free knock-ins of several kilobases by engineered CRISPR/Cas endonucleases\nAuthors: Schreiber, T.; Prange, A.; Schaefer, P.; Iwen, T.; Gruetzner, R.; Marillonnet, S.; Tissier, A. F.\nScore: 6.2, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568561\nIn plants and mammals, non-homologous end-joining is the dominant pathway to repair DNA double strand breaks, making it challenging to generate knock-in events. Using a transient assay in Nicotiana benthamiana we identified two groups of exonucleases, respectively from the Herpes Virus and from the bacteriophage T7 families, hat confer up to 38-fold increase in HDR frequencies when fused to Cas9. We achieved precise and scar-free insertion of several kilobases of DNA both in transient and stable transformation systems. In Arabidopsis thaliana, fusion of Cas9 to a Herpes Virus family exonuclease leads to 10-fold higher frequencies of knock-ins. Our results open perspectives for the routine production of knock-in and gene replacement events in plants.\nThe recombination initiation functions DprA and RecFOR suppress microindel mutations in Acinetobacter baylyi ADP1\nAuthors: Liljegren, M. M.; Gama, J. A.; Johnsen, P. J.; Harms, K.\nScore: 4.8, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568220\nShort-Patch Double Illegitimate Recombination (SPDIR) has been recently identified as a rare mutation mechanism. During SPDIR, ectopic DNA single-strands anneal with genomic DNA at microhomologies and get integrated in the course of DNA replication, presumably acting as Okazaki fragments. The resulting microindel mutations are highly variable in size and sequence. In the soil bacterium Acinetobacter baylyi, SPDIR mutations are tightly controlled by genome maintenance functions including RecA. In this study, we investigate the roles of DprA, RecFOR and RecBCD, which are cytoplasmic functions that load DNA single-strands with RecA. All three functions suppress SPDIR mutations in wildtype to levels below the detection limit. While SPDIR mutations are slightly elevated in the absence of DprA alone, they are strongly increased in the absence of both DprA and RecA. This SPDIR-avoiding function of DprA is not related to its role in natural transformation. These results suggest an antimutational function for DprA and offer an explanation for the ubiquity of dprA in the genomes of non-transformable bacteria.\n",
  "wordCount" : "2165",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567912">
        <p class="paperTitle">Antagonistic conflict between transposon-encoded introns and guide RNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zedaveinyte, R.; Meers, C.; Le, H. C.; Mortman, E. E.; Tang, S.; Lampe, G. D.; Pesari, S. R.; Gelsinger, D. R.; Wiegand, T.; Sternberg, S. H.</p>
        <p class="info">Score: 20.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567912' target='https://doi.org/10.1101/2023.11.20.567912'> 10.1101/2023.11.20.567912</a></p>
        <p class="abstract">TnpB nucleases represent the evolutionary precursors to CRISPR-Cas12 and are widespread in all domains of life, presumably due to the critical roles they play in transposon proliferation. IS605-family TnpB homologs function in bacteria as programmable homing endonucleases by exploiting transposon-encoded guide RNAs to cleave vacant genomic sites, thereby driving transposon maintenance through DSB-stimulated homologous recombination. Whether this pathway is conserved in other genetic contexts, and in association with other transposases, is unknown. Here we uncover molecular mechanisms of transposition and RNA-guided DNA cleavage by IS607-family elements that, remarkably, also encode catalytic, self-splicing group I introns. After reconstituting and systematically investigating each of these biochemical activities for a candidate  IStron derived from Clostridium botulinum, we discovered sequence and structural features of the transposon-encoded RNA that satisfy molecular requirements of a group I intron and TnpB guide RNA, while still retaining the ability to be faithfully mobilized at the DNA level by the TnpA transposase. Strikingly, intron splicing was strongly repressed not only by TnpB, but also by the secondary structure of {omega}RNA alone, allowing the element to carefully control the relative levels of spliced products versus functional guide RNAs. Our results suggest that IStron transcripts have evolved a sensitive equilibrium to balance competing and mutually exclusive activities that promote transposon maintenance while limiting adverse fitness costs on the host. Collectively, this work explains how diverse enzymatic activities emerged during the selfish spread of IS607-family elements and highlights molecular innovation in the multi-functional utility of transposon-encoded noncoding RNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568217">
        <p class="paperTitle">Spatial transcriptomics defines injury-specific microenvironments in the adult mouse kidney and novel cellular interactions in regeneration and disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Polonsky, M.; Gerhardt, L. M. S.; Yun, J.; Koppitch, K.; Colon, K.; Amrhein, H.; Zheng, S.; Yuan, G.-C.; Thomson, M.; Cai, L.; McMahon, A. P.</p>
        <p class="info">Score: 16.9, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568217' target='https://doi.org/10.1101/2023.11.22.568217'> 10.1101/2023.11.22.568217</a></p>
        <p class="abstract">Kidney injury disrupts the intricate renal architecture and triggers limited regeneration, and injury-invoked inflammation and fibrosis. Deciphering molecular pathways and cellular interactions driving these processes is challenging due to the complex renal architecture. Here, we applied single cell spatial transcriptomics to examine ischemia-reperfusion injury in the mouse kidney. Spatial transcriptomics revealed injury-specific and spatially-dependent gene expression patterns in distinct cellular microenvironments within the kidney and predicted Clcf1-Crfl1 in a molecular interplay between persistently injured proximal tubule cells and neighboring fibroblasts. Immune cell types play a critical role in organ repair. Spatial analysis revealed cellular microenvironments resembling early tertiary lymphoid structures and identified associated molecular pathways. Collectively, this study supports a focus on molecular interactions in cellular microenvironments to enhance understanding of injury, repair and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567549">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567549" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567549">
        <p class="paperTitle">Reversible histone deacetylase activity catalyzes lysine acylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567549" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567549" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsusaka, T.; Najar, M. A.; Schwarz, B.; Bohrnsen, E.; Oses-Prieto, J. A.; Lee, C.; Burlingame, A. L.; Bosio, C. M.; Burslem, G. M.; Goldberg, E. L.</p>
        <p class="info">Score: 33.3, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567549' target='https://doi.org/10.1101/2023.11.17.567549'> 10.1101/2023.11.17.567549</a></p>
        <p class="abstract">Starvation and low carbohydrate diets lead to the accumulation of the ketone body, {beta}-hydroxybutyrate (BHB), whose blood concentrations increase more than 10-fold into the millimolar range. In addition to providing a carbon source, BHB accumulation triggers lysine {beta}-hydroxybutyrylation (Kbhb) of proteins via unknown mechanisms. As with other lysine acylation events, Kbhb marks can be removed by histone deacetylases (HDACs). Here, we report that class I HDACs unexpectedly catalyze protein lysine modification with {beta}-hydroxybutyrate (BHB). Mutational analyses of the HDAC2 active site reveal a shared reliance on key amino acids for classical deacetylation and non-canonical HDAC-catalyzed {beta}-hydroxybutyrylation. Also consistent with reverse HDAC activity, Kbhb formation is driven by mass action and substrate availability. This reverse HDAC activity is not limited to BHB but also extends to multiple short-chain fatty acids. The reversible activity of class I HDACs described here represents a novel mechanism of PTM deposition relevant to metabolically-sensitive proteome modifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567203">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567203" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567203">
        <p class="paperTitle">A C-terminal HaloTag Impairs AGO2 Function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567203" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567203" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shah, K. M.; Crozier, A. F. F.; Assaraf, A. A.; Arya, M.; Ribeiro, P. B.; Plevin, M. J.; Sharp, T. V.</p>
        <p class="info">Score: 8.4, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567203' target='https://doi.org/10.1101/2023.11.20.567203'> 10.1101/2023.11.20.567203</a></p>
        <p class="abstract">A full understanding of RNA silencing requires appropriate molecular biology tools to explore the roles of Argonaute 2 (AGO2) and the RNA-induced silencing complex (RISC). Commonly used approaches to study RNA silencing and RISC, such as those relying on affinity tagging and antibodies, have important limitations that can lead to artificial results. Both the N- and C-terminal domains of AGO2 have been shown to be important for correct activity and yet the consequences of appending tags to either terminus have not been fully investigated. N-terminal tags are frequently used to study AGO(2) biology. Recently, an N-terminal HaloTag-Ago2 fusion was reported and examined in mice. While the versatile HaloTag provided new opportunities to study RISC biology, the tagged construct showed certain activity changes compared to unmodified Ago2. CRISPaint, a new CRISPR-Cas9 technique, offers a route to the accurate and efficient generation of endogenous C-terminal tag fusions. Here, we used CRISPaint to generate the first reported recombinant AGO2 construct with a C-terminal tag: an endogenous C-terminal HaloTag fusion to AGO2 (AGO2HALO) in human (A549) cells. We found that the AGO2HALO fusion protein retains the capacity to interact with the key protein binding partner TNRC6A and that the C-terminal HaloTag does not affect cell viability. However, the AGO2HALO fusion significantly impairs RNA cleavage and RNA silencing activity compared to control cells, and reduces nuclear localisation of the fusion protein. We conclude that the fusion of a C-terminal HaloTag to AGO2 is not appropriate for studying AGO2 and RISC. Our results stress the importance of fully validating recombinant tagging strategies to ensure that any results generated do not obscure critical functional defects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568106">
        <p class="paperTitle">Msc1 is a nuclear envelope protein that reinforces DNA repair in late mitosis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Medina-Suarez, S.; Ayra-Plasencia, J.; Perez-Martinez, L.; Butter, F.; Machin, F.</p>
        <p class="info">Score: 8.3, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568106' target='https://doi.org/10.1101/2023.11.21.568106'> 10.1101/2023.11.21.568106</a></p>
        <p class="abstract">Precise double-strand break (DSB) repair is paramount for genome stability. Homologous recombination (HR) repairs DSBs when cyclin-dependent kinase (CDK) activity is high, which correlates with the availability of the sister chromatid as a template. However, anaphase and telophase are paradoxical scenarios since high CDK favors HR despite sister chromatids being no longer aligned. To identify factors specifically involved in DSB repair in late mitosis, we have undertaken comparative proteomics in Saccharomyces cerevisiae and found that Msc1, a poorly characterized nuclear envelope protein, is significantly enriched upon both random and guided DSBs. We further show that {Delta}msc1 is more sensitive to DSBs in late mitosis, and has a delayed repair of DBSs, as indicated by increased Rad53 hyperphosphorylation, fewer Rad52 repair factories, and slower HR completion. We discuss how Msc1 may favor the formation of Rad52 factories and the timely completion of HR before cytokinesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567629">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567629" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567629">
        <p class="paperTitle">Structural basis for polyuridine tract recognition by SARS-CoV-2 Nsp15</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567629" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567629" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ito, F.; Yang, H.; Zhou, Z. H.; Chen, X. S.</p>
        <p class="info">Score: 6.8, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567629' target='https://doi.org/10.1101/2023.11.17.567629'> 10.1101/2023.11.17.567629</a></p>
        <p class="abstract">SARS-CoV-2 non-structural protein 15 (Nsp15) is critical for productive viral replication and evasion of host immunity. The uridine-specific endoribonuclease activity of Nsp15 mediates the cleavage of the polyuridine [poly(U)] tract of the negative-strand coronavirus genome to minimize the formation of dsRNA that activates the host antiviral interferon signaling. However, the molecular basis for the recognition and cleavage of the poly(U) tract by Nsp15 is incompletely understood. Here, we present cryogenic electron microscopy (cryoEM) structures of SARS-CoV-2 Nsp15 bound to viral replication intermediate dsRNA containing poly(U) tract at 2.7-3.3 [A] resolution. The structures reveal one copy of dsRNA binds to the sidewall of an Nsp15 homohexamer, spanning three subunits in two distinct binding states. The target uracil is dislodged from the base-pairing of the dsRNA by amino acid residues W332 and M330 of Nsp15, and the dislodged base is entrapped at the endonuclease active site center. Up to 20 A/U base pairs are anchored on the Nsp15 hexamer, which explains the basis for a substantially shortened poly(U) sequence in the negative strand coronavirus genome compared to the long poly(A) tail in its positive strand. Our results provide mechanistic insights into the unique immune evasion strategy employed by coronavirus Nsp15.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568381">
        <p class="paperTitle">Enhanced surface accessibility of SARS-CoV-2 Omicron spike protein due to an altered glycosylation profile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, D.; Zhang, Z.; Baudys, J.; Haynes, C. A.; Osman, S. H.; Zhou, B.; Barr, J. R.; Gumbart, J. C.</p>
        <p class="info">Score: 6.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568381' target='https://doi.org/10.1101/2023.11.22.568381'> 10.1101/2023.11.22.568381</a></p>
        <p class="abstract">SARS-CoV-2 spike (S) proteins undergo extensive glycosylation, aiding proper folding, enhancing stability, and evading host immune surveillance. In this study, we used mass spectrometric analysis to elucidate the N-glycosylation characteristics and disulfide bonding of recombinant spike proteins derived from the SARS-CoV-2 Omicron variant (B.1.1.529) in comparison with the D614G spike variant. Furthermore, we conducted microsecond-long molecular dynamics simulations on spike proteins to resolve how the different N-glycans impact spike conformational sampling in the two variants. Our findings reveal that the Omicron spike protein maintains an overall resemblance to the D614G spike variant in terms of site-specific glycan processing and disulfide bond formation. Nonetheless, alterations in glycans were observed at certain N-glycosylation sites. These changes, in synergy with mutations within the Omicron spike protein, result in increased surface accessibility of the macromolecule, including ectodomain, receptor-binding domain, and N-terminal domain. These insights contribute to our understanding of the interplay between structure and function, thereby advancing effective vaccination and therapeutic strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568130">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568130" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568130">
        <p class="paperTitle">Symbiosis modulates gene expression of symbionts, but not hosts, under thermal challenge</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568130" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568130" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aichelman, H. E.; Huzar, A. K.; Wuitchik, D. M.; Atherton, K. F.; Wright, R. M.; Dixon, G. B.; Schlatter, E.; Haftel, N.; Davies, S. W.</p>
        <p class="info">Score: 6.6, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568130' target='https://doi.org/10.1101/2023.11.21.568130'> 10.1101/2023.11.21.568130</a></p>
        <p class="abstract">Increasing ocean temperatures are causing dysbiosis between coral hosts and their symbionts. Previous work suggests that coral host gene expression responds more strongly to environmental stress compared to their intracellular symbionts; however, the causes and consequences of this phenomenon remain untested. We hypothesized that symbionts are less responsive because hosts modulate symbiont environments to buffer stress. To test this hypothesis, we leveraged the facultative symbiosis between the scleractinian coral Oculina arbuscula and its symbiont Breviolum psygmophilum to characterize gene expression responses of both symbiotic partners in and ex hospite under thermal challenges. To characterize host and in hospite symbiont responses, symbiotic and aposymbiotic O. arbuscula were exposed to three treatments: 1) control (18{degrees}C), 2) heat (32{degrees}C), and 3) cold (6{degrees}C). This experiment was replicated with B. psygmophilum cultured from O. arbuscula to characterize ex hospite symbiont responses. Both thermal challenges elicited classic environmental stress responses (ESRs) in O. arbuscula regardless of symbiotic state, with hosts responding more strongly to cold challenge. Hosts also exhibited stronger responses than in hospite symbionts. In and ex hospite B. psygmophilum both downregulated genes associated with photosynthesis under thermal challenge; however, ex hospite symbionts exhibited greater gene expression plasticity and differential expression of genes associated with ESRs. Taken together, these findings suggest that O. arbuscula hosts may buffer environments of B. psygmophilum symbionts; however, we outline the future work needed to confirm this hypothesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568561">
        <p class="paperTitle">Efficient scar-free knock-ins of several kilobases by engineered CRISPR/Cas endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schreiber, T.; Prange, A.; Schaefer, P.; Iwen, T.; Gruetzner, R.; Marillonnet, S.; Tissier, A. F.</p>
        <p class="info">Score: 6.2, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568561' target='https://doi.org/10.1101/2023.11.24.568561'> 10.1101/2023.11.24.568561</a></p>
        <p class="abstract">In plants and mammals, non-homologous end-joining is the dominant pathway to repair DNA double strand breaks, making it challenging to generate knock-in events. Using a transient assay in Nicotiana benthamiana we identified two groups of exonucleases, respectively from the Herpes Virus and from the bacteriophage T7 families, hat confer up to 38-fold increase in HDR frequencies when fused to Cas9. We achieved precise and scar-free insertion of several kilobases of DNA both in transient and stable transformation systems. In Arabidopsis thaliana, fusion of Cas9 to a Herpes Virus family exonuclease leads to 10-fold higher frequencies of knock-ins. Our results open perspectives for the routine production of knock-in and gene replacement events in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568220">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568220" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568220">
        <p class="paperTitle">The recombination initiation functions DprA and RecFOR suppress microindel mutations in Acinetobacter baylyi ADP1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568220" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568220" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liljegren, M. M.; Gama, J. A.; Johnsen, P. J.; Harms, K.</p>
        <p class="info">Score: 4.8, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568220' target='https://doi.org/10.1101/2023.11.22.568220'> 10.1101/2023.11.22.568220</a></p>
        <p class="abstract">Short-Patch Double Illegitimate Recombination (SPDIR) has been recently identified as a rare mutation mechanism. During SPDIR, ectopic DNA single-strands anneal with genomic DNA at microhomologies and get integrated in the course of DNA replication, presumably acting as Okazaki fragments. The resulting microindel mutations are highly variable in size and sequence. In the soil bacterium Acinetobacter baylyi, SPDIR mutations are tightly controlled by genome maintenance functions including RecA. In this study, we investigate the roles of DprA, RecFOR and RecBCD, which are cytoplasmic functions that load DNA single-strands with RecA. All three functions suppress SPDIR mutations in wildtype to levels below the detection limit. While SPDIR mutations are slightly elevated in the absence of DprA alone, they are strongly increased in the absence of both DprA and RecA. This SPDIR-avoiding function of DprA is not related to its role in natural transformation. These results suggest an antimutational function for DprA and offer an explanation for the ubiquity of dprA in the genomes of non-transformable bacteria.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
