<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evaluation of AlphaFold Antibody-Antigen Modeling with Implications for Improving Predictive Accuracy
Authors: Yin, R.; Pierce, B. G.
Score: 27.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547832
High resolution antibody-antigen structures provide critical insights into immune recognition and can inform therapeutic design. The challenges of experimental structural determination and the diversity of the immune repertoire underscore the necessity of accurate computational tools for modeling antibody-antigen complexes. Initial benchmarking showed that despite overall success in modeling protein-protein complexes, AlphaFold and AlphaFold-Multimer have limited success in modeling antibody-antigen interactions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Evaluation of AlphaFold Antibody-Antigen Modeling with Implications for Improving Predictive Accuracy
Authors: Yin, R.; Pierce, B. G.
Score: 27.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547832
High resolution antibody-antigen structures provide critical insights into immune recognition and can inform therapeutic design. The challenges of experimental structural determination and the diversity of the immune repertoire underscore the necessity of accurate computational tools for modeling antibody-antigen complexes. Initial benchmarking showed that despite overall success in modeling protein-protein complexes, AlphaFold and AlphaFold-Multimer have limited success in modeling antibody-antigen interactions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Evaluation of AlphaFold Antibody-Antigen Modeling with Implications for Improving Predictive Accuracy
Authors: Yin, R.; Pierce, B. G.
Score: 27.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547832
High resolution antibody-antigen structures provide critical insights into immune recognition and can inform therapeutic design. The challenges of experimental structural determination and the diversity of the immune repertoire underscore the necessity of accurate computational tools for modeling antibody-antigen complexes. Initial benchmarking showed that despite overall success in modeling protein-protein complexes, AlphaFold and AlphaFold-Multimer have limited success in modeling antibody-antigen interactions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Evaluation of AlphaFold Antibody-Antigen Modeling with Implications for Improving Predictive Accuracy\nAuthors: Yin, R.; Pierce, B. G.\nScore: 27.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547832\nHigh resolution antibody-antigen structures provide critical insights into immune recognition and can inform therapeutic design. The challenges of experimental structural determination and the diversity of the immune repertoire underscore the necessity of accurate computational tools for modeling antibody-antigen complexes. Initial benchmarking showed that despite overall success in modeling protein-protein complexes, AlphaFold and AlphaFold-Multimer have limited success in modeling antibody-antigen interactions.",
  "keywords": [
    
  ],
  "articleBody": " Evaluation of AlphaFold Antibody-Antigen Modeling with Implications for Improving Predictive Accuracy\nAuthors: Yin, R.; Pierce, B. G.\nScore: 27.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547832\nHigh resolution antibody-antigen structures provide critical insights into immune recognition and can inform therapeutic design. The challenges of experimental structural determination and the diversity of the immune repertoire underscore the necessity of accurate computational tools for modeling antibody-antigen complexes. Initial benchmarking showed that despite overall success in modeling protein-protein complexes, AlphaFold and AlphaFold-Multimer have limited success in modeling antibody-antigen interactions. In this study, we performed a thorough analysis of AlphaFold's antibody-antigen modeling performance on 429 nonredundant antibody-antigen complex structures, identifying useful confidence metrics for predicting model quality, and features of complexes associated with improved modeling success. We show the importance of bound-like component modeling in complex assembly accuracy, and that the current version of AlphaFold improves near-native modeling success to over 30%, versus approximately 20% for a previous version. With this improved success, AlphaFold can generate accurate antibody-antigen models in many cases, while additional training may further improve its performance.\nProgrammable RNA-guided endonucleases are widespread in eukaryotes and their viruses\nAuthors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.\nScore: 260.7, Published: 2023-06-14 DOI: 10.1101/2023.06.13.544871\nTnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.\nEfficient Formation of Single-copy Human Artificial Chromosomes\nAuthors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.\nScore: 29.5, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547284\nLarge DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells. One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.\nChemogenetic profiling of ubiquitin-like modifier pathways identifies NFATC2IP as a mediator of SUMO-dependent genome integrity\nAuthors: Cho, T.; Zhao, Y.; Olivieri, M.; Hoeg, L.; Setiaputra, D.; Durocher, D.\nScore: 18.8, Published: 2023-07-05 DOI: 10.1101/2023.06.30.547196\nThe post-translational modification of proteins by ubiquitin and ubiquitin-like polypeptides controls multiple cellular processes including the abundance of a large fraction of the proteome. We applied genome-scale CRISPR/Cas9 screens to elucidate the genetic architecture of the response to inhibition of ubiquitin, NEDD8 and SUMO conjugation pathways as well as inhibition of the p97/VCP segregase. This effort identified 395 genes whose disruption alters the fitness of human cells when faced with perturbations in these pathways. We validated that the TMED2 and TMED10 proteins, which are localized to the secretory pathway, promote resistance to p97/VCP inhibition and also characterized NFATC2IP, an evolutionarily conserved protein harboring SUMO-like domains as a major player in promoting genomic integrity when SUMOylation is inhibited. We propose that NFATC2IP acts in interphase cells to promote the SUMO-dependent E3 ligase activity of the SMC5/SMC6 complex, which is critical for SUMO-dependent genome integrity.\nThe CPSF phosphatase module links transcription termination to chromatin silencing\nAuthors: Mateo Bonmati, E.; Fang, X.; Maple, R.; Fiedler, M.; Passmore, L. A.; Dean, C. A.\nScore: 16.8, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547976\nRNA-mediated chromatin silencing plays important roles in development and environmental response in many organisms. However, it is still unclear how the RNA, and/or the transcriptional processes producing them, trigger chromatin silencing. Through the study of developmental timing in plants we have found co-transcriptional RNA processing modulates a chromatin regulatory mechanism controlling quantitative expression of Arabidopsis FLOWERING LOCUS C (FLC). Here, we show that the phosphatase module of the mRNA 3' processing CPSF complex is central to this mechanism. The phosphatase module components APRF1 (yeast Swd2/human WDR82), TOPP4 (Glc7/PP1) and LD (which we show is functionally equivalent to Ref2/PNUTS) all function genetically downstream of the CPSF-mediated cleavage and polyadenylation and are necessary for chromatin modification that results in FLC transcriptional silencing. LD has been previously shown to stoichiometrically co-associate in vivo with the histone demethylase FLD. We show both proteins result in removal of H3K4 monomethylation in the central section of the FLC locus. This links transcription termination to delivery of chromatin modifications that then influence subsequent transcription. Thus, physical association of the CPSF phosphatase module with chromatin modifiers generates a transcription-coupled silencing mechanism. The conservation of the factors involved suggest this mechanism may be generally relevant to transcription-mediated chromatin silencing.\nProteomics analysis reveals a role for E. coli polyphosphate kinase in membrane structure and polymyxin resistance during starvation\nAuthors: Baijal, K.; Abrumchuk, I.; Herrera, C.; Trent, M. S.; Lavallee-Adam, M.; Downey, M.\nScore: 11.0, Published: 2023-07-06 DOI: 10.1101/2023.07.06.546892\nPolyphosphates (polyP) are chains of inorganic phosphates that can reach over 1000 residues in length. In Escherichia coli, polyP is produced by the polyP kinase (PPK) and is thought to play a protective role during the response to cellular stress. However, the molecular pathways impacted by PPK activity and polyP accumulation remain poorly characterized. In this work we used label-free mass spectrometry to study the response of bacteria that cannot produce polyP ({triangleup}ppk) during starvation to identify novel pathways regulated by PPK. In response to starvation, we found 92 proteins significantly differentially expressed between wild-type and {triangleup}ppk mutant cells. Wild-type cells were enriched for proteins related to amino acid biosynthesis and transport, while {triangleup}ppk mutants were enriched for proteins related to translation and ribosome biogenesis, suggesting that without PPK, cells remain inappropriately primed for growth even in the absence of required building blocks. From our dataset, we were particularly interested in Arn and EptA proteins, which were downregulated in {triangleup}ppk mutants compared to wild-type controls, because they play a role in lipid A modifications linked to polymyxin resistance. Using western blotting, we confirm differential expression of these and related proteins, and provide evidence that this mis-regulation in {triangleup}ppk cells stems from a failure to induce the BasS/BasR two-component system during starvation. We also show that {triangleup}ppk mutants unable to upregulate Arn and EptA expression lack the respective L-Ara4N and pEtN modifications on lipid A. In line with this observation, loss of ppk restores polymyxin sensitivity in resistant strains carrying a constitutively active basR allele. Overall, we show a new role for PPK in lipid A modification during starvation and provide a rationale for targeting PPK to sensitize bacteria towards polymyxin treatment. We further anticipate that our proteomics work will provide an important resource for researchers interested in the diverse pathways impacted by PPK.\nA Spatiotemporal Proteomic Map of Human Adipogenesis\nAuthors: Klingelhuber, F.; Frendo-Cumbo, S.; Hmeadi, O.; Taylor, A. J.; Kakimoto, P.; Ribicic, S.; Couchet, M.; Wabitsch, M.; Mueller, T. D.; Ryden, M.; Mejhert, N.; Krahmer, N.\nScore: 10.9, Published: 2023-07-03 DOI: 10.1101/2023.07.01.547321\nWhite adipocytes function as the major energy reservoir in humans by storing substantial amounts of triglycerides and their dysfunction is associated with metabolic disorders. However, the mechanisms underlying cellular specialization during adipogenesis remain unknown. Here, using a high-sensitivity-high throughput workflow, we generated a spatiotemporal proteomic atlas of human adipogenesis encompassing information for [~]8.000 proteins. Our systematic approach gives insights into cellular remodeling and the spatial reorganization of metabolic pathways to optimize cells for lipid accumulation and highlights the coordinated regulation of protein localization and abundance during adipogenesis. More specifically, we identified a compartment-specific regulation of protein levels and localization changes of metabolic enzymes to reprogram branched chain amino acid and one-carbon metabolism to provide building blocks and reduction equivalents for lipid synthesis. Additionally, we identified C19orf12 as a differentiation induced and adipocyte-specific lipid droplet (LD) protein, which interacts with the translocase of the outer membrane (TOM) complex of LD associated mitochondria and regulates adipocyte lipid storage. Overall, our study provides a comprehensive resource for understanding human adipogenesis and for future discoveries in the field. KEY POINTSO_LIHuman adipogenesis induces distinct temporal changes in protein abundance C_LIO_LI20% of all detected proteins change organellar localization during adipogenesis C_LIO_LIBCAA and one-carbon metabolism enzyme levels and localizations are coordinately regulated to promote adipogenesis C_LIO_LIC19orf12 is an adipocyte specific LD protein and regulator of lipid storage C_LI\nProximal termination generates a transcriptional state that determines the rate of establishment of Polycomb silencing\nAuthors: Menon, G.; Mateo Bonmati, E.; Reeck, S.; Maple, R.; Wu, Z.; Ietswaart, R.; Dean, C.; Howard, M.\nScore: 10.1, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547969\nChromatin-mediated transcriptional states are central to gene regulation in development and environmental response, with co-transcriptional processes involved in their establishment. Quantitative regulation of Arabidopsis FLOWERING LOCUS C (FLC) is key to determining reproductive strategy. Low FLC expression underpins rapid-cycling and is established through a transcription-coupled chromatin mechanism. Proximal termination of antisense transcripts is linked to histone 3 lysine 4 demethylation of adjacent chromatin that leads to stable Polycomb Repressive Complex 2 (PRC2) silencing. However, how the termination-induced chromatin environment influences the switch to PRC2 silencing is still unclear. Here, we combine molecular approaches with theory to develop a dynamic mathematical model that incorporates sense/antisense transcription, alternative termination sites, and the interplay of these processes with varying levels of activating (H3K4me1)/silencing (H3K27me3) histone modifications. The model captures different feedback mechanisms between co-transcriptional 3' processing and chromatin modifications, detailing how proximal co-transcriptional polyadenylation/termination can set the subsequent level of productive transcription via removal of H3K4 monomethylation across the locus. Since transcription universally antagonizes Polycomb silencing, this dictates the degree of antagonism to H3K27me3, thus determining the rate at which Polycomb repression is established. These principles are likely to be central to regulating transcriptional output at many targets and generally relevant for Polycomb silencing in many genomes.\nAntisense transcription and PRC2 repression function in parallel during vernalization\nAuthors: Nielsen, M.; Menon, G.; Zhao, Y.; Mateo-Bonmati, E.; Wolff, P.; Zhou, S.; Howard, M.; Dean, C.\nScore: 8.3, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547987\nNon-coding transcription induces chromatin changes that can mediate environmental responsiveness, but the causes and consequences of these mechanisms are still unclear. Here, we investigate how antisense transcription interfaces with Polycomb Repressive Complex 2 silencing during winter-induced epigenetic regulation of Arabidopsis FLOWERING LOCUS C (FLC). We combine genetic and chromatin analyses on lines ineffective or hyperactive for the antisense pathway, with computational modelling to reconcile all the data on FLC repression. Our results show that FLC is silenced through pathways that function with different dynamics: an antisense transcription-mediated pathway capable of fast response; and in parallel a slow Polycomb Repressive Complex 2 (PRC2) switching mechanism that maintains each allele in an epigenetically silenced state. Components of both the antisense and PRC2 pathways are regulated by a common transcriptional regulator (NTL8), which accumulates by reduced dilution due to slow growth at low temperature. The parallel activities of the regulatory steps, and their control by temperature-dependent growth dynamics, creates a flexible system for registering widely fluctuating natural temperature conditions that change year on year, and yet ensure robust epigenetic silencing of FLC.\nIdentification of fish spawning based on size-selected sampling and diurnal concentration changes of environmental DNA resulting from sperm-release\nAuthors: Tsuji, S.; Shibata, N.\nScore: 9.5, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545639\nEnvironmental DNA (eDNA) analysis is a useful method for detecting spawning behaviour by observing elevated eDNA concentrations and/or nuclear/mitochondrial DNA ratios. However, little is known about the characteristics and dynamics of sperm-derived eDNA, which is key to detection. This study focused on changes in eDNA particle size distribution (PSD) and concentrations pre-, post-, and during spawning of Plecoglossus altivelis, to investigate the potential for semi-selective recovery of sperm-derived eDNA and the importance of sampling time for improving detection sensitivity. Firstly, PSD changes between pre- and post-spawning were investigated by comparing concentrations and proportions of eDNA obtained from filters with different pore sizes. Secondly, the diurnal changes in eDNA concentration were monitored at the peak of the spawning season by collecting river water every 1-3 hours. Results showed that eDNA related to sperm-head size increased at post-spawning, and eDNA concentrations had significant diurnal changes with a peak during the spawning time window. These findings suggest that semi-selective recovery of sperm-derived eDNA based on particle size and/or sampling during the spawning time window with increased concentrations can improve the detection sensitivity of eDNA-based spawning surveys. This study provides essential fundamental information for advancing eDNA-based spawning surveys and contributes to their further development. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=107 SRC=\"FIGDIR/small/545639v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@134432dorg.highwire.dtl.DTLVardef@267657org.highwire.dtl.DTLVardef@1236480org.highwire.dtl.DTLVardef@9b3503_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2293",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547832" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547832">
        <p class="paperTitle">Evaluation of AlphaFold Antibody-Antigen Modeling with Implications for Improving Predictive Accuracy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yin, R.; Pierce, B. G.</p>
        <p class="info">Score: 27.0, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547832' target='https://doi.org/10.1101/2023.07.05.547832'> 10.1101/2023.07.05.547832</a></p>
        <p class="abstract">High resolution antibody-antigen structures provide critical insights into immune recognition and can inform therapeutic design. The challenges of experimental structural determination and the diversity of the immune repertoire underscore the necessity of accurate computational tools for modeling antibody-antigen complexes. Initial benchmarking showed that despite overall success in modeling protein-protein complexes, AlphaFold and AlphaFold-Multimer have limited success in modeling antibody-antigen interactions. In this study, we performed a thorough analysis of AlphaFold&#39;s antibody-antigen modeling performance on 429 nonredundant antibody-antigen complex structures, identifying useful confidence metrics for predicting model quality, and features of complexes associated with improved modeling success. We show the importance of bound-like component modeling in complex assembly accuracy, and that the current version of AlphaFold improves near-native modeling success to over 30%, versus approximately 20% for a previous version. With this improved success, AlphaFold can generate accurate antibody-antigen models in many cases, while additional training may further improve its performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544871">
        <p class="paperTitle">Programmable RNA-guided endonucleases are widespread in eukaryotes and their viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, K.; Lim, J.; Sgrizzi, S.; Trinh, M.; Kayabolen, A.; Yutin, N.; Koonin, E. V.; Abudayyeh, O. O.; Gootenberg, J. S.</p>
        <p class="info">Score: 260.7, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544871' target='https://doi.org/10.1101/2023.06.13.544871'> 10.1101/2023.06.13.544871</a></p>
        <p class="abstract">TnpB proteins are RNA-guided nucleases that are broadly associated with IS200/605 family transposons in prokaryotes. TnpB homologs, named Fanzors, have been detected in genomes of some eukaryotes and large viruses, but their activity and functions in eukaryotes remain unknown. We searched genomes of diverse eukaryotes and their viruses for TnpB homologs and identified numerous putative RNA-guided nucleases that are often associated with various transposases, suggesting they are encoded in mobile genetic elements. Reconstruction of the evolution of these nucleases, which we rename Horizontally-transferred Eukaryotic RNA-guided Mobile Element Systems (HERMES), revealed multiple acquisitions of TnpBs by eukaryotes and subsequent diversification. In their adaptation and spread in eukaryotes, HERMES proteins acquired nuclear localization signals, and genes captured introns, indicating extensive, long term adaptation to functioning in eukaryotic cells. Biochemical and cellular evidence show that HERMES employ non-coding RNAs encoded adjacent to the nuclease for RNA-guided cleavage of double-stranded DNA. HERMES nucleases contain a re-arranged catalytic site of the RuvC domain, similar to a distinct subset of TnpBs, and lack collateral cleavage activity. We demonstrate that HERMES can be harnessed for genome editing in human cells, highlighting the potential of these widespread eukaryotic RNA-guided nucleases for biotechnology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547284">
        <p class="paperTitle">Efficient Formation of Single-copy Human Artificial Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.</p>
        <p class="info">Score: 29.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547284' target='https://doi.org/10.1101/2023.06.30.547284'> 10.1101/2023.06.30.547284</a></p>
        <p class="abstract">Large DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.

One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547196" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547196">
        <p class="paperTitle">Chemogenetic profiling of ubiquitin-like modifier pathways identifies NFATC2IP as a mediator of SUMO-dependent genome integrity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cho, T.; Zhao, Y.; Olivieri, M.; Hoeg, L.; Setiaputra, D.; Durocher, D.</p>
        <p class="info">Score: 18.8, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547196' target='https://doi.org/10.1101/2023.06.30.547196'> 10.1101/2023.06.30.547196</a></p>
        <p class="abstract">The post-translational modification of proteins by ubiquitin and ubiquitin-like polypeptides controls multiple cellular processes including the abundance of a large fraction of the proteome. We applied genome-scale CRISPR/Cas9 screens to elucidate the genetic architecture of the response to inhibition of ubiquitin, NEDD8 and SUMO conjugation pathways as well as inhibition of the p97/VCP segregase. This effort identified 395 genes whose disruption alters the fitness of human cells when faced with perturbations in these pathways. We validated that the TMED2 and TMED10 proteins, which are localized to the secretory pathway, promote resistance to p97/VCP inhibition and also characterized NFATC2IP, an evolutionarily conserved protein harboring SUMO-like domains as a major player in promoting genomic integrity when SUMOylation is inhibited. We propose that NFATC2IP acts in interphase cells to promote the SUMO-dependent E3 ligase activity of the SMC5/SMC6 complex, which is critical for SUMO-dependent genome integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547976">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547976" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547976">
        <p class="paperTitle">The CPSF phosphatase module links transcription termination to chromatin silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547976" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547976" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mateo Bonmati, E.; Fang, X.; Maple, R.; Fiedler, M.; Passmore, L. A.; Dean, C. A.</p>
        <p class="info">Score: 16.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547976' target='https://doi.org/10.1101/2023.07.06.547976'> 10.1101/2023.07.06.547976</a></p>
        <p class="abstract">RNA-mediated chromatin silencing plays important roles in development and environmental response in many organisms. However, it is still unclear how the RNA, and/or the transcriptional processes producing them, trigger chromatin silencing. Through the study of developmental timing in plants we have found co-transcriptional RNA processing modulates a chromatin regulatory mechanism controlling quantitative expression of Arabidopsis FLOWERING LOCUS C (FLC). Here, we show that the phosphatase module of the mRNA 3&#39; processing CPSF complex is central to this mechanism. The phosphatase module components APRF1 (yeast Swd2/human WDR82), TOPP4 (Glc7/PP1) and LD (which we show is functionally equivalent to Ref2/PNUTS) all function genetically downstream of the CPSF-mediated cleavage and polyadenylation and are necessary for chromatin modification that results in FLC transcriptional silencing. LD has been previously shown to stoichiometrically co-associate in vivo with the histone demethylase FLD. We show both proteins result in removal of H3K4 monomethylation in the central section of the FLC locus. This links transcription termination to delivery of chromatin modifications that then influence subsequent transcription. Thus, physical association of the CPSF phosphatase module with chromatin modifiers generates a transcription-coupled silencing mechanism. The conservation of the factors involved suggest this mechanism may be generally relevant to transcription-mediated chromatin silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.546892">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.546892" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.546892">
        <p class="paperTitle">Proteomics analysis reveals a role for E. coli polyphosphate kinase in membrane structure and polymyxin resistance during starvation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.546892" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.546892" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baijal, K.; Abrumchuk, I.; Herrera, C.; Trent, M. S.; Lavallee-Adam, M.; Downey, M.</p>
        <p class="info">Score: 11.0, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.546892' target='https://doi.org/10.1101/2023.07.06.546892'> 10.1101/2023.07.06.546892</a></p>
        <p class="abstract">Polyphosphates (polyP) are chains of inorganic phosphates that can reach over 1000 residues in length. In Escherichia coli, polyP is produced by the polyP kinase (PPK) and is thought to play a protective role during the response to cellular stress. However, the molecular pathways impacted by PPK activity and polyP accumulation remain poorly characterized. In this work we used label-free mass spectrometry to study the response of bacteria that cannot produce polyP ({triangleup}ppk) during starvation to identify novel pathways regulated by PPK. In response to starvation, we found 92 proteins significantly differentially expressed between wild-type and {triangleup}ppk mutant cells. Wild-type cells were enriched for proteins related to amino acid biosynthesis and transport, while {triangleup}ppk mutants were enriched for proteins related to translation and ribosome biogenesis, suggesting that without PPK, cells remain inappropriately primed for growth even in the absence of required building blocks. From our dataset, we were particularly interested in Arn and EptA proteins, which were downregulated in {triangleup}ppk mutants compared to wild-type controls, because they play a role in lipid A modifications linked to polymyxin resistance. Using western blotting, we confirm differential expression of these and related proteins, and provide evidence that this mis-regulation in {triangleup}ppk cells stems from a failure to induce the BasS/BasR two-component system during starvation. We also show that {triangleup}ppk mutants unable to upregulate Arn and EptA expression lack the respective L-Ara4N and pEtN modifications on lipid A. In line with this observation, loss of ppk restores polymyxin sensitivity in resistant strains carrying a constitutively active basR allele. Overall, we show a new role for PPK in lipid A modification during starvation and provide a rationale for targeting PPK to sensitize bacteria towards polymyxin treatment. We further anticipate that our proteomics work will provide an important resource for researchers interested in the diverse pathways impacted by PPK.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.547321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.547321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.547321">
        <p class="paperTitle">A Spatiotemporal Proteomic Map of Human Adipogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.547321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.547321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klingelhuber, F.; Frendo-Cumbo, S.; Hmeadi, O.; Taylor, A. J.; Kakimoto, P.; Ribicic, S.; Couchet, M.; Wabitsch, M.; Mueller, T. D.; Ryden, M.; Mejhert, N.; Krahmer, N.</p>
        <p class="info">Score: 10.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.547321' target='https://doi.org/10.1101/2023.07.01.547321'> 10.1101/2023.07.01.547321</a></p>
        <p class="abstract">White adipocytes function as the major energy reservoir in humans by storing substantial amounts of triglycerides and their dysfunction is associated with metabolic disorders. However, the mechanisms underlying cellular specialization during adipogenesis remain unknown. Here, using a high-sensitivity-high throughput workflow, we generated a spatiotemporal proteomic atlas of human adipogenesis encompassing information for [~]8.000 proteins. Our systematic approach gives insights into cellular remodeling and the spatial reorganization of metabolic pathways to optimize cells for lipid accumulation and highlights the coordinated regulation of protein localization and abundance during adipogenesis. More specifically, we identified a compartment-specific regulation of protein levels and localization changes of metabolic enzymes to reprogram branched chain amino acid and one-carbon metabolism to provide building blocks and reduction equivalents for lipid synthesis. Additionally, we identified C19orf12 as a differentiation induced and adipocyte-specific lipid droplet (LD) protein, which interacts with the translocase of the outer membrane (TOM) complex of LD associated mitochondria and regulates adipocyte lipid storage. Overall, our study provides a comprehensive resource for understanding human adipogenesis and for future discoveries in the field.

KEY POINTSO_LIHuman adipogenesis induces distinct temporal changes in protein abundance
C_LIO_LI20% of all detected proteins change organellar localization during adipogenesis
C_LIO_LIBCAA and one-carbon metabolism enzyme levels and localizations are coordinately regulated to promote adipogenesis
C_LIO_LIC19orf12 is an adipocyte specific LD protein and regulator of lipid storage
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547969">
        <p class="paperTitle">Proximal termination generates a transcriptional state that determines the rate of establishment of Polycomb silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Menon, G.; Mateo Bonmati, E.; Reeck, S.; Maple, R.; Wu, Z.; Ietswaart, R.; Dean, C.; Howard, M.</p>
        <p class="info">Score: 10.1, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547969' target='https://doi.org/10.1101/2023.07.06.547969'> 10.1101/2023.07.06.547969</a></p>
        <p class="abstract">Chromatin-mediated transcriptional states are central to gene regulation in development and environmental response, with co-transcriptional processes involved in their establishment. Quantitative regulation of Arabidopsis FLOWERING LOCUS C (FLC) is key to determining reproductive strategy. Low FLC expression underpins rapid-cycling and is established through a transcription-coupled chromatin mechanism. Proximal termination of antisense transcripts is linked to histone 3 lysine 4 demethylation of adjacent chromatin that leads to stable Polycomb Repressive Complex 2 (PRC2) silencing. However, how the termination-induced chromatin environment influences the switch to PRC2 silencing is still unclear. Here, we combine molecular approaches with theory to develop a dynamic mathematical model that incorporates sense/antisense transcription, alternative termination sites, and the interplay of these processes with varying levels of activating (H3K4me1)/silencing (H3K27me3) histone modifications. The model captures different feedback mechanisms between co-transcriptional 3&#39; processing and chromatin modifications, detailing how proximal co-transcriptional polyadenylation/termination can set the subsequent level of productive transcription via removal of H3K4 monomethylation across the locus. Since transcription universally antagonizes Polycomb silencing, this dictates the degree of antagonism to H3K27me3, thus determining the rate at which Polycomb repression is established. These principles are likely to be central to regulating transcriptional output at many targets and generally relevant for Polycomb silencing in many genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547987">
        <p class="paperTitle">Antisense transcription and PRC2 repression function in parallel during vernalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nielsen, M.; Menon, G.; Zhao, Y.; Mateo-Bonmati, E.; Wolff, P.; Zhou, S.; Howard, M.; Dean, C.</p>
        <p class="info">Score: 8.3, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547987' target='https://doi.org/10.1101/2023.07.07.547987'> 10.1101/2023.07.07.547987</a></p>
        <p class="abstract">Non-coding transcription induces chromatin changes that can mediate environmental responsiveness, but the causes and consequences of these mechanisms are still unclear. Here, we investigate how antisense transcription interfaces with Polycomb Repressive Complex 2 silencing during winter-induced epigenetic regulation of Arabidopsis FLOWERING LOCUS C (FLC). We combine genetic and chromatin analyses on lines ineffective or hyperactive for the antisense pathway, with computational modelling to reconcile all the data on FLC repression. Our results show that FLC is silenced through pathways that function with different dynamics: an antisense transcription-mediated pathway capable of fast response; and in parallel a slow Polycomb Repressive Complex 2 (PRC2) switching mechanism that maintains each allele in an epigenetically silenced state. Components of both the antisense and PRC2 pathways are regulated by a common transcriptional regulator (NTL8), which accumulates by reduced dilution due to slow growth at low temperature. The parallel activities of the regulatory steps, and their control by temperature-dependent growth dynamics, creates a flexible system for registering widely fluctuating natural temperature conditions that change year on year, and yet ensure robust epigenetic silencing of FLC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545639">
        <p class="paperTitle">Identification of fish spawning based on size-selected sampling and diurnal concentration changes of environmental DNA resulting from sperm-release</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsuji, S.; Shibata, N.</p>
        <p class="info">Score: 9.5, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545639' target='https://doi.org/10.1101/2023.06.19.545639'> 10.1101/2023.06.19.545639</a></p>
        <p class="abstract">Environmental DNA (eDNA) analysis is a useful method for detecting spawning behaviour by observing elevated eDNA concentrations and/or nuclear/mitochondrial DNA ratios. However, little is known about the characteristics and dynamics of sperm-derived eDNA, which is key to detection. This study focused on changes in eDNA particle size distribution (PSD) and concentrations pre-, post-, and during spawning of Plecoglossus altivelis, to investigate the potential for semi-selective recovery of sperm-derived eDNA and the importance of sampling time for improving detection sensitivity. Firstly, PSD changes between pre- and post-spawning were investigated by comparing concentrations and proportions of eDNA obtained from filters with different pore sizes. Secondly, the diurnal changes in eDNA concentration were monitored at the peak of the spawning season by collecting river water every 1-3 hours. Results showed that eDNA related to sperm-head size increased at post-spawning, and eDNA concentrations had significant diurnal changes with a peak during the spawning time window. These findings suggest that semi-selective recovery of sperm-derived eDNA based on particle size and/or sampling during the spawning time window with increased concentrations can improve the detection sensitivity of eDNA-based spawning surveys. This study provides essential fundamental information for advancing eDNA-based spawning surveys and contributes to their further development.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=107 SRC=&#34;FIGDIR/small/545639v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@134432dorg.highwire.dtl.DTLVardef@267657org.highwire.dtl.DTLVardef@1236480org.highwire.dtl.DTLVardef@9b3503_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
