<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="XIST dampens X chromosome activity in a SPEN-dependent manner during early human development
Authors: Alfeghaly, C.; Castel, G.; Cazottes, E.; Moscatelli, M.; Moinard, E.; Casanova, M.; Boni, J.; Mahadik, K.; Lammers, J.; Freour, T.; Chauviere, L.; Piqueras, C.; Boers, R.; Boers, J.; Gribnau, J.; David, L.; Ouimette, J.-F.; Rougeulle, C.
Score: 23.8, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563078
XIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="XIST dampens X chromosome activity in a SPEN-dependent manner during early human development
Authors: Alfeghaly, C.; Castel, G.; Cazottes, E.; Moscatelli, M.; Moinard, E.; Casanova, M.; Boni, J.; Mahadik, K.; Lammers, J.; Freour, T.; Chauviere, L.; Piqueras, C.; Boers, R.; Boers, J.; Gribnau, J.; David, L.; Ouimette, J.-F.; Rougeulle, C.
Score: 23.8, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563078
XIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="XIST dampens X chromosome activity in a SPEN-dependent manner during early human development
Authors: Alfeghaly, C.; Castel, G.; Cazottes, E.; Moscatelli, M.; Moinard, E.; Casanova, M.; Boni, J.; Mahadik, K.; Lammers, J.; Freour, T.; Chauviere, L.; Piqueras, C.; Boers, R.; Boers, J.; Gribnau, J.; David, L.; Ouimette, J.-F.; Rougeulle, C.
Score: 23.8, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563078
XIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "XIST dampens X chromosome activity in a SPEN-dependent manner during early human development\nAuthors: Alfeghaly, C.; Castel, G.; Cazottes, E.; Moscatelli, M.; Moinard, E.; Casanova, M.; Boni, J.; Mahadik, K.; Lammers, J.; Freour, T.; Chauviere, L.; Piqueras, C.; Boers, R.; Boers, J.; Gribnau, J.; David, L.; Ouimette, J.-F.; Rougeulle, C.\nScore: 23.8, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563078\nXIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing.",
  "keywords": [
    
  ],
  "articleBody": " XIST dampens X chromosome activity in a SPEN-dependent manner during early human development\nAuthors: Alfeghaly, C.; Castel, G.; Cazottes, E.; Moscatelli, M.; Moinard, E.; Casanova, M.; Boni, J.; Mahadik, K.; Lammers, J.; Freour, T.; Chauviere, L.; Piqueras, C.; Boers, R.; Boers, J.; Gribnau, J.; David, L.; Ouimette, J.-F.; Rougeulle, C.\nScore: 23.8, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563078\nXIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing. The long non-coding RNA XACT co-accumulates with XIST on active Xs and may antagonize XIST function. Here we used human ES cells in a naive state of pluripotency to assess the function of XIST and XACT in shaping the X chromosome chromatin and transcriptional landscapes during pre-implantation development. We show that XIST triggers the deposition of polycomb-mediated repressive histone modifications and attenuates transcription of most X-linked genes in a SPEN-dependent manner, while XACT deficiency does not significantly affect XIST activity or X-linked gene expression. Our study demonstrates that XIST is functional prior to XCI, confirms the existence of a transient process of X chromosome dosage compensation, and reveals that X chromosome inactivation and dampening rely on the same set of factors.\nAutomated model building and protein identification in cryo-EM maps\nAuthors: Jamali, K.; Kall, L.; Zhang, R.; Brown, A.; Kimanius, D.; Scheres, S.\nScore: 104.1, Published: 2023-10-17 DOI: 10.1101/2023.05.16.541002\nInterpreting electron cryo-microscopy (cryo-EM) maps with atomic models requires high levels of expertise and labour-intensive manual intervention. We present ModelAngelo, a machine-learning approach for automated atomic model building in cryo-EM maps. By combining information from the cryo-EM map with information from protein sequence and structure in a single graph neural network, ModelAngelo builds atomic models for proteins that are of similar quality as those generated by human experts. For nucleotides, ModelAngelo builds backbones with similar accuracy as humans. By using its predicted amino acid probabilities for each residue in hidden Markov model sequence searches, ModelAngelo outperforms human experts in the identification of proteins with unknown sequences. ModelAngelo will thus remove bottlenecks and increase objectivity in cryo-EM structure determination.\nExtrachromosomal circular DNA is required for yeast stress granule assembly\nAuthors: Demeshkina, N. A.; Ferre-D'Amare, A. R.\nScore: 7.7, Published: 2023-10-13 DOI: 10.1101/2023.10.12.562116\nStress granules are a key cellular response, conserved across eukaryotes, to environmental insults. They arise upon stress-induced reversible stalling of protein synthesis and redistribution of mRNAs from polysomes into these large cytoplasmic ribonucleoprotein condensates. Many cellular pathways lead to formation and turnover of stress granules. Despite research spanning four decades, biochemical analysis of stress granules remains challenging because of their complexity and dynamic nature. Hitherto, stress granules have been studied cytologically, and their composition deduced from candidate protein approaches including immunoprecipitation and proximity labeling. Now, we have directly purified stress granule cores from yeast, relying on their density and size, finding that they are discrete particles with a size distribution peaking at 135 nm. Unexpectedly, we discovered that in addition to diverse proteins and RNAs, stress granule cores contain extrachromosomal circular DNA (eccDNA). These double-stranded DNAs have been found in all eukaryotes, but there is no precedent for a role in membraneless organelle biogenesis. By repurposing CRISPR technology to target DNA in the cytoplasm, we demonstrate that both presence and accessibility of eccDNA are required for stress granule formation in live yeast cells. Our finding reveals an unrecognized link between DNA metabolism and the cytoplasmic stress response, and suggests that sequence-specific eccDNA-mRNA interactions may mediate stress granule assembly.\nChronically disrupted sleep induces senescence in the visceral adipose tissue of C57BL/6J mice\nAuthors: Timonina, D.; Hormazabal, G. V.; Heckenbach, I.; Anderton, E.; Haky, L.; Floro, A.; Riley, R.; Kwok, R.; Breslin, S.; Ingle, H.; Tiwari, R.; Bielska, O.; Scheibye-Knudsen, M.; Kasler, H. G.; Campisi, J.; Walter, M.; Verdin, E.\nScore: 6.6, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562803\nThe role of sleep in systemic aging remains poorly understood, despite sleeps essential function in preserving overall health and the prevalence of reduced sleep quality in modern society. Although reduced sleep correlates with an elevated risk of age-related diseases in humans, the mechanisms underlying this are unclear. In this study, we established a link between sleep and aging by demonstrating that disrupting sleep in C57BL/6 mice drives cellular senescence in the visceral adipose tissue. Sleep disruption also led to increased oxidative stress and DNA damage, both recognized triggers for senescence induction. Cellular senescence is implicated in numerous age-related conditions which are associated with insufficient sleep, such as cardiovascular disease, type 2 diabetes, and chronic inflammation. Our findings identify an accumulation of senescent cells in the adipose tissue, which serves as a potential target through which disturbed sleep accelerates the aging process and elevates the risk of age-related diseases.\nAnalysis the molecular similarity of least common amino acid sites in ACE2 receptor to predict the potential susceptible species for SARS-CoV-2\nAuthors: YeZhi, H.; Fan, X.; Villalan, A. K.; Zhang, S.; Joka, F. R.; Wu, X.; Wang, H.; Wang, X.\nScore: 6.0, Published: 2023-10-16 DOI: 10.1101/2023.10.13.562198\nThis research offers a bioinformatics approach to forecasting both domestic and wild animals likelihood of being susceptible to SARS-CoV-2 infection. Genomic sequencing can resolve phylogenetic relationships between the virus and the susceptible host. The genome sequence of SARS-CoV-2 is highly interactive with the specific sequence region of the ACE2 receptor of the host species. We further evaluate this concept to identify the most important SARS-CoV-2 binding amino acid sites in the ACE2 receptor sequence through the common similarity of the last common amino acid sites (LCAS) in known susceptible host species. Therefore, the SARS-CoV-2 viral genomic interacting key amino acid region in the ACE2 receptor sequence of known susceptible human host was summarized and compared with other reported known SARS-CoV-2 susceptible host species. We identified the 10 most significant amino acid sites for interaction with SARS-CoV-2 infection from the ACE2 receptor sequence region based on the LCAS similarity pattern in known sensitive SARS-CoV-2 hosts. The most significant 10 LCAS were further compared with ACE2 receptor sequences of unknown species to evaluate the similarity of the last common amino acid pattern (LCAP). We predicted the probability of SARS-CoV-2 infection risk in unknown species through the LCAS similarity pattern. This method can be used as a screening tool to assess the risk of SARS-CoV-2 infection in domestic and wild animals to prevent outbreaks of infection. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=98 SRC=\"FIGDIR/small/562198v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@15bb59aorg.highwire.dtl.DTLVardef@1efc572org.highwire.dtl.DTLVardef@8bf4adorg.highwire.dtl.DTLVardef@6ca1ba_HPS_FORMAT_FIGEXP M_FIG C_FIG\nControl of poly(A)-tail length and translation in vertebrate oocytes and early embryos\nAuthors: Xiang, K.; Ly, J.; Bartel, D.\nScore: 6.0, Published: 2023-10-19 DOI: 10.1101/2023.10.18.562922\nDuring oocyte maturation and early embryogenesis, changes in mRNA poly(A)-tail lengths strongly influence translation, but how these tail-length changes are orchestrated has been unclear. Here, we performed tail-length and translational profiling of mRNA reporter libraries (each with millions of 3'-UTR sequence variants) in frog oocytes and embryos, and fish embryos. We found that the UUUUA element, together with the polyadenylation signal (AAUAAA or AUUAAA), specifies cytoplasmic polyadenylation, and identified contextual features that modulate the activity of both elements. In maturing oocytes, this tail lengthening occurs against a backdrop of global deadenylation and the action of C-rich elements that specify tail-length-independent translational repression. In embryos, cytoplasmic polyadenylation becomes more permissive, and additional elements specify waves of stage-specific deadenylation. Together, these findings largely explain the complex tapestry of tail-length changes observed in early frog and fish development, with strong evidence of conservation in both mice and humans. HighlightsO_LIUUUUA, modulated by contextual features, specifies most cytoplasmic polyadenylation C_LIO_LIStage-specific sequence motifs drive waves of tail-length shortening in embryos C_LIO_LIUUUUA and C-rich motifs can direct tail-length-independent translational repression C_LIO_LITail-length control is conserved in oocytes of frogs, mice, and humans C_LI\nEnhanced Ca2+ Binding to EF-Hands through Phosphorylation of Conserved Serine Residues Activates MpRBOHB and Chitin-Triggered ROS Production\nAuthors: Hashimoto, T.; Hashimoto, K.; Shindo, H.; Tsuboyama, S.; Miyakawa, T.; Tanokura, M.; Kuchitsu, K.\nScore: 6.1, Published: 2023-10-05 DOI: 10.1101/2023.10.05.559649\nNADPH oxidases/RBOHs catalyze apoplastic ROS production and act as key signaling nodes, integrating multiple signal transduction pathways regulating plant development and stress responses. Although RBOHs have been suggested to be activated by Ca2+ binding and phosphorylation by various protein kinases, a mechanism linking Ca2+ binding and phosphorylation in the activity regulation remained elusive. Chitin-triggered ROS production required cytosolic Ca2+ elevation and Ca2+ binding to MpRBOHB in a liverwort Marchantia polymorpha. Heterologous expression analysis of truncated variants revealed that a segment of the N-terminal cytosolic region highly conserved among land plant RBOHs encompassing the two EF-hand motifs is essential for the activation of MpRBOHB. Within the conserved regulatory domain, we have identified two Ser residues whose phosphorylation is critical for the activation in planta. Isothermal titration calorimetry analyses revealed that phosphorylation of the two Ser residues increased the Ca2+ binding affinity of MpRBOHB, while Ca2+ binding is indispensable for the activation, even if the two Ser residues are phosphorylated. Our findings shed light on a mechanism through which phosphorylation potentiates the Ca2+-dependent activation of MpRBOHB, emphasizing the pivotal role of Ca2+ binding in mediating the Ca2+ and phosphorylation-driven activation of MpRBOHB, which is likely to represent a fundamental mechanism conserved among land plant RBOHs.\nThe sensor of the bacterial histidine kinase CpxA is a novel dimer of extracytoplasmic Per-ARNT-Sim (PAS) domains\nAuthors: Cho, T. H. S.; Murray, C.; Malpica, R.; Margain-Quevedo, R.; Thede, G. L.; Lu, J.; Edwards, R. A.; Glover, J. N. M.; Raivio, T. L.\nScore: 5.7, Published: 2023-10-18 DOI: 10.1101/2023.10.18.561931\nHistidine kinases are key bacterial sensors that recognize diverse environmental stimuli. While mechanisms of phosphorylation and phosphotransfer by cytoplasmic kinase domains are relatively well-characterized, the ways in which extracytoplasmic sensor domains regulate activation remain mysterious. The Cpx envelope stress response is a conserved Gram-negative two-component system which is controlled by the sensor kinase CpxA. We report the structure of the Escherichia coli CpxA sensor domain (CpxA-SD) as a globular Per-ARNT-Sim (PAS)-like fold highly similar to that of Vibrio parahaemolyticus CpxA as determined by X-ray crystallography. Because sensor kinase dimerization is important for signaling, we used AlphaFold2 to model CpxA-SD in the context of its connected transmembrane domains, which yielded a novel dimer of PAS domains possessing a distinct dimer organization compared to previously characterized sensor domains. Gain of function cpxA* alleles map to the dimer interface, and mutation of other residues in this region also leads to constitutive activation. CpxA activation can be suppressed by mutations that restore inter-monomer interactions, suggesting that inhibitory interactions between CpxA-SD monomers are the major point of control for CpxA activation and signaling. Searching through hundreds of structural homologues revealed the sensor domain of Pseudomonas aeruginosa sensor kinase PfeS as the only PAS structure in the same novel dimer orientation as CpxA, suggesting that our dimer orientation may be utilized by other extracytoplasmic PAS domains. Overall, our findings provide insight into the diversity of the organization of PAS sensory domains and how they regulate sensor kinase activation.\nAedes aegypti (Dipetera: Culicidae) odorant receptors respond to floral and plant derived volatiles.\nAuthors: Pullmann-Lindsley, H.; Huff, R.; Boyi, J.; Pitts, J.\nScore: 5.5, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562234\nAdult mosquitoes require regular sugar meals, usually in the form of floral nectar, to survive in natural habitats. Both males and females locate potential sugar sources using sensory proteins called odorant receptors, that are activated by plant volatiles and facilitate orientation toward flowers or honeydew. The yellow fever mosquito, Aedes aegypti (Linnaeus, 1762), possesses a large gene family of odorant receptors, many of which are likely to be involved in floral odor detection. Using a heterologous expression system and a technique called two-electrode voltage clamping, we are able to identify environmentally relevant chemical cues that produce responses in specific odorant receptors. In this study, we have uncovered ligand-receptor pairings for a suite Aedes aegypti receptors that are likely to mediate appetitive or aversive behavioral responses. This information can be used to assess mosquito foraging behavior and develop novel control strategies, especially those that incorporate mosquito bait-and-kill technologies.\nChronic Interferon Stimulated Gene Transcription Promotes Oncogene Induced Breast Cancer\nAuthors: Wang, H.; Canasto-Chibuque, C.; Kim, J. H.; Hohl, M.; Leslie, C.; Reis-Filho, J. S.; Petrini, J. H.\nScore: 5.5, Published: 2023-10-16 DOI: 10.1101/2023.10.16.562529\nThe Mre11 complex (comprising Mre11, Rad50, Nbs1) is integral to the maintenance of genome stability. We previously showed that a hypomorphic Mre11 mutant mouse strain (Mre11ATLD1/ATLD1) was highly susceptible to oncogene induced breast cancer. Here we used a mammary organoid system to examine which Mre11 dependent responses are tumor suppressive. We found that Mre11ATLD1/ATLD1 organoids exhibited an elevated interferon stimulated gene (ISG) signature and sustained changes in chromatin accessibility. This Mre11ATLD1/ATLD1 phenotype depended on DNA binding of a nuclear innate immune sensor, IFI205. Ablation of Ifi205 in Mre11ATLD1/ATLD1 organoids restored baseline and oncogene-induced chromatin accessibility patterns to those observed in WT. Implantation of Mre11ATLD1/ATLD1 organoids and activation of oncogene led to aggressive metastatic breast cancer. This outcome was reversed in implanted Ifi205-/- Mre11ATLD1/ATLD1 organoids. These data reveal a connection between innate immune signaling and tumor suppression in mammary epithelium. Given the abundance of aberrant DNA structures that arise in the context of genome instability syndromes, the data further suggest that cancer predisposition in those contexts may be partially attributable to tonic innate immune transcriptional programs.\n",
  "wordCount" : "2200",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563078" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563078">
        <p class="paperTitle">XIST dampens X chromosome activity in a SPEN-dependent manner during early human development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alfeghaly, C.; Castel, G.; Cazottes, E.; Moscatelli, M.; Moinard, E.; Casanova, M.; Boni, J.; Mahadik, K.; Lammers, J.; Freour, T.; Chauviere, L.; Piqueras, C.; Boers, R.; Boers, J.; Gribnau, J.; David, L.; Ouimette, J.-F.; Rougeulle, C.</p>
        <p class="info">Score: 23.8, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563078' target='https://doi.org/10.1101/2023.10.19.563078'> 10.1101/2023.10.19.563078</a></p>
        <p class="abstract">XIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing. The long non-coding RNA XACT co-accumulates with XIST on active Xs and may antagonize XIST function. Here we used human ES cells in a naive state of pluripotency to assess the function of XIST and XACT in shaping the X chromosome chromatin and transcriptional landscapes during pre-implantation development. We show that XIST triggers the deposition of polycomb-mediated repressive histone modifications and attenuates transcription of most X-linked genes in a SPEN-dependent manner, while XACT deficiency does not significantly affect XIST activity or X-linked gene expression. Our study demonstrates that XIST is functional prior to XCI, confirms the existence of a transient process of X chromosome dosage compensation, and reveals that X chromosome inactivation and dampening rely on the same set of factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.16.541002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.16.541002" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.16.541002">
        <p class="paperTitle">Automated model building and protein identification in cryo-EM maps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.16.541002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.16.541002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jamali, K.; Kall, L.; Zhang, R.; Brown, A.; Kimanius, D.; Scheres, S.</p>
        <p class="info">Score: 104.1, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.16.541002' target='https://doi.org/10.1101/2023.05.16.541002'> 10.1101/2023.05.16.541002</a></p>
        <p class="abstract">Interpreting electron cryo-microscopy (cryo-EM) maps with atomic models requires high levels of expertise and labour-intensive manual intervention. We present ModelAngelo, a machine-learning approach for automated atomic model building in cryo-EM maps. By combining information from the cryo-EM map with information from protein sequence and structure in a single graph neural network, ModelAngelo builds atomic models for proteins that are of similar quality as those generated by human experts. For nucleotides, ModelAngelo builds backbones with similar accuracy as humans. By using its predicted amino acid probabilities for each residue in hidden Markov model sequence searches, ModelAngelo outperforms human experts in the identification of proteins with unknown sequences. ModelAngelo will thus remove bottlenecks and increase objectivity in cryo-EM structure determination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562116" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562116">
        <p class="paperTitle">Extrachromosomal circular DNA is required for yeast stress granule assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demeshkina, N. A.; Ferre-D&#39;Amare, A. R.</p>
        <p class="info">Score: 7.7, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562116' target='https://doi.org/10.1101/2023.10.12.562116'> 10.1101/2023.10.12.562116</a></p>
        <p class="abstract">Stress granules are a key cellular response, conserved across eukaryotes, to environmental insults. They arise upon stress-induced reversible stalling of protein synthesis and redistribution of mRNAs from polysomes into these large cytoplasmic ribonucleoprotein condensates. Many cellular pathways lead to formation and turnover of stress granules. Despite research spanning four decades, biochemical analysis of stress granules remains challenging because of their complexity and dynamic nature. Hitherto, stress granules have been studied cytologically, and their composition deduced from candidate protein approaches including immunoprecipitation and proximity labeling. Now, we have directly purified stress granule cores from yeast, relying on their density and size, finding that they are discrete particles with a size distribution peaking at 135 nm. Unexpectedly, we discovered that in addition to diverse proteins and RNAs, stress granule cores contain extrachromosomal circular DNA (eccDNA). These double-stranded DNAs have been found in all eukaryotes, but there is no precedent for a role in membraneless organelle biogenesis. By repurposing CRISPR technology to target DNA in the cytoplasm, we demonstrate that both presence and accessibility of eccDNA are required for stress granule formation in live yeast cells. Our finding reveals an unrecognized link between DNA metabolism and the cytoplasmic stress response, and suggests that sequence-specific eccDNA-mRNA interactions may mediate stress granule assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562803">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562803" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562803">
        <p class="paperTitle">Chronically disrupted sleep induces senescence in the visceral adipose tissue of C57BL/6J mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562803" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562803" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timonina, D.; Hormazabal, G. V.; Heckenbach, I.; Anderton, E.; Haky, L.; Floro, A.; Riley, R.; Kwok, R.; Breslin, S.; Ingle, H.; Tiwari, R.; Bielska, O.; Scheibye-Knudsen, M.; Kasler, H. G.; Campisi, J.; Walter, M.; Verdin, E.</p>
        <p class="info">Score: 6.6, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562803' target='https://doi.org/10.1101/2023.10.17.562803'> 10.1101/2023.10.17.562803</a></p>
        <p class="abstract">The role of sleep in systemic aging remains poorly understood, despite sleeps essential function in preserving overall health and the prevalence of reduced sleep quality in modern society. Although reduced sleep correlates with an elevated risk of age-related diseases in humans, the mechanisms underlying this are unclear. In this study, we established a link between sleep and aging by demonstrating that disrupting sleep in C57BL/6 mice drives cellular senescence in the visceral adipose tissue. Sleep disruption also led to increased oxidative stress and DNA damage, both recognized triggers for senescence induction. Cellular senescence is implicated in numerous age-related conditions which are associated with insufficient sleep, such as cardiovascular disease, type 2 diabetes, and chronic inflammation. Our findings identify an accumulation of senescent cells in the adipose tissue, which serves as a potential target through which disturbed sleep accelerates the aging process and elevates the risk of age-related diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562198">
        <p class="paperTitle">Analysis the molecular similarity of least common amino acid sites in ACE2 receptor to predict the potential susceptible species for SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: YeZhi, H.; Fan, X.; Villalan, A. K.; Zhang, S.; Joka, F. R.; Wu, X.; Wang, H.; Wang, X.</p>
        <p class="info">Score: 6.0, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562198' target='https://doi.org/10.1101/2023.10.13.562198'> 10.1101/2023.10.13.562198</a></p>
        <p class="abstract">This research offers a bioinformatics approach to forecasting both domestic and wild animals likelihood of being susceptible to SARS-CoV-2 infection. Genomic sequencing can resolve phylogenetic relationships between the virus and the susceptible host. The genome sequence of SARS-CoV-2 is highly interactive with the specific sequence region of the ACE2 receptor of the host species. We further evaluate this concept to identify the most important SARS-CoV-2 binding amino acid sites in the ACE2 receptor sequence through the common similarity of the last common amino acid sites (LCAS) in known susceptible host species. Therefore, the SARS-CoV-2 viral genomic interacting key amino acid region in the ACE2 receptor sequence of known susceptible human host was summarized and compared with other reported known SARS-CoV-2 susceptible host species. We identified the 10 most significant amino acid sites for interaction with SARS-CoV-2 infection from the ACE2 receptor sequence region based on the LCAS similarity pattern in known sensitive SARS-CoV-2 hosts. The most significant 10 LCAS were further compared with ACE2 receptor sequences of unknown species to evaluate the similarity of the last common amino acid pattern (LCAP). We predicted the probability of SARS-CoV-2 infection risk in unknown species through the LCAS similarity pattern. This method can be used as a screening tool to assess the risk of SARS-CoV-2 infection in domestic and wild animals to prevent outbreaks of infection.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=98 SRC=&#34;FIGDIR/small/562198v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@15bb59aorg.highwire.dtl.DTLVardef@1efc572org.highwire.dtl.DTLVardef@8bf4adorg.highwire.dtl.DTLVardef@6ca1ba_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562922">
        <p class="paperTitle">Control of poly(A)-tail length and translation in vertebrate oocytes and early embryos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xiang, K.; Ly, J.; Bartel, D.</p>
        <p class="info">Score: 6.0, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562922' target='https://doi.org/10.1101/2023.10.18.562922'> 10.1101/2023.10.18.562922</a></p>
        <p class="abstract">During oocyte maturation and early embryogenesis, changes in mRNA poly(A)-tail lengths strongly influence translation, but how these tail-length changes are orchestrated has been unclear. Here, we performed tail-length and translational profiling of mRNA reporter libraries (each with millions of 3&#39;-UTR sequence variants) in frog oocytes and embryos, and fish embryos. We found that the UUUUA element, together with the polyadenylation signal (AAUAAA or AUUAAA), specifies cytoplasmic polyadenylation, and identified contextual features that modulate the activity of both elements. In maturing oocytes, this tail lengthening occurs against a backdrop of global deadenylation and the action of C-rich elements that specify tail-length-independent translational repression. In embryos, cytoplasmic polyadenylation becomes more permissive, and additional elements specify waves of stage-specific deadenylation. Together, these findings largely explain the complex tapestry of tail-length changes observed in early frog and fish development, with strong evidence of conservation in both mice and humans.

HighlightsO_LIUUUUA, modulated by contextual features, specifies most cytoplasmic polyadenylation
C_LIO_LIStage-specific sequence motifs drive waves of tail-length shortening in embryos
C_LIO_LIUUUUA and C-rich motifs can direct tail-length-independent translational repression
C_LIO_LITail-length control is conserved in oocytes of frogs, mice, and humans
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.559649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.559649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.559649">
        <p class="paperTitle">Enhanced Ca2&#43; Binding to EF-Hands through Phosphorylation of Conserved Serine Residues Activates MpRBOHB and Chitin-Triggered ROS Production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.559649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.559649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hashimoto, T.; Hashimoto, K.; Shindo, H.; Tsuboyama, S.; Miyakawa, T.; Tanokura, M.; Kuchitsu, K.</p>
        <p class="info">Score: 6.1, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.559649' target='https://doi.org/10.1101/2023.10.05.559649'> 10.1101/2023.10.05.559649</a></p>
        <p class="abstract">NADPH oxidases/RBOHs catalyze apoplastic ROS production and act as key signaling nodes, integrating multiple signal transduction pathways regulating plant development and stress responses. Although RBOHs have been suggested to be activated by Ca2&#43; binding and phosphorylation by various protein kinases, a mechanism linking Ca2&#43; binding and phosphorylation in the activity regulation remained elusive. Chitin-triggered ROS production required cytosolic Ca2&#43; elevation and Ca2&#43; binding to MpRBOHB in a liverwort Marchantia polymorpha. Heterologous expression analysis of truncated variants revealed that a segment of the N-terminal cytosolic region highly conserved among land plant RBOHs encompassing the two EF-hand motifs is essential for the activation of MpRBOHB. Within the conserved regulatory domain, we have identified two Ser residues whose phosphorylation is critical for the activation in planta. Isothermal titration calorimetry analyses revealed that phosphorylation of the two Ser residues increased the Ca2&#43; binding affinity of MpRBOHB, while Ca2&#43; binding is indispensable for the activation, even if the two Ser residues are phosphorylated. Our findings shed light on a mechanism through which phosphorylation potentiates the Ca2&#43;-dependent activation of MpRBOHB, emphasizing the pivotal role of Ca2&#43; binding in mediating the Ca2&#43; and phosphorylation-driven activation of MpRBOHB, which is likely to represent a fundamental mechanism conserved among land plant RBOHs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.561931">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.561931" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.561931">
        <p class="paperTitle">The sensor of the bacterial histidine kinase CpxA is a novel dimer of extracytoplasmic Per-ARNT-Sim (PAS) domains</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.561931" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.561931" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cho, T. H. S.; Murray, C.; Malpica, R.; Margain-Quevedo, R.; Thede, G. L.; Lu, J.; Edwards, R. A.; Glover, J. N. M.; Raivio, T. L.</p>
        <p class="info">Score: 5.7, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.561931' target='https://doi.org/10.1101/2023.10.18.561931'> 10.1101/2023.10.18.561931</a></p>
        <p class="abstract">Histidine kinases are key bacterial sensors that recognize diverse environmental stimuli. While mechanisms of phosphorylation and phosphotransfer by cytoplasmic kinase domains are relatively well-characterized, the ways in which extracytoplasmic sensor domains regulate activation remain mysterious. The Cpx envelope stress response is a conserved Gram-negative two-component system which is controlled by the sensor kinase CpxA. We report the structure of the Escherichia coli CpxA sensor domain (CpxA-SD) as a globular Per-ARNT-Sim (PAS)-like fold highly similar to that of Vibrio parahaemolyticus CpxA as determined by X-ray crystallography. Because sensor kinase dimerization is important for signaling, we used AlphaFold2 to model CpxA-SD in the context of its connected transmembrane domains, which yielded a novel dimer of PAS domains possessing a distinct dimer organization compared to previously characterized sensor domains. Gain of function cpxA* alleles map to the dimer interface, and mutation of other residues in this region also leads to constitutive activation. CpxA activation can be suppressed by mutations that restore inter-monomer interactions, suggesting that inhibitory interactions between CpxA-SD monomers are the major point of control for CpxA activation and signaling. Searching through hundreds of structural homologues revealed the sensor domain of Pseudomonas aeruginosa sensor kinase PfeS as the only PAS structure in the same novel dimer orientation as CpxA, suggesting that our dimer orientation may be utilized by other extracytoplasmic PAS domains. Overall, our findings provide insight into the diversity of the organization of PAS sensory domains and how they regulate sensor kinase activation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562234">
        <p class="paperTitle">Aedes aegypti (Dipetera: Culicidae) odorant receptors respond to floral and plant derived volatiles.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pullmann-Lindsley, H.; Huff, R.; Boyi, J.; Pitts, J.</p>
        <p class="info">Score: 5.5, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562234' target='https://doi.org/10.1101/2023.10.17.562234'> 10.1101/2023.10.17.562234</a></p>
        <p class="abstract">Adult mosquitoes require regular sugar meals, usually in the form of floral nectar, to survive in natural habitats. Both males and females locate potential sugar sources using sensory proteins called odorant receptors, that are activated by plant volatiles and facilitate orientation toward flowers or honeydew. The yellow fever mosquito, Aedes aegypti (Linnaeus, 1762), possesses a large gene family of odorant receptors, many of which are likely to be involved in floral odor detection. Using a heterologous expression system and a technique called two-electrode voltage clamping, we are able to identify environmentally relevant chemical cues that produce responses in specific odorant receptors. In this study, we have uncovered ligand-receptor pairings for a suite Aedes aegypti receptors that are likely to mediate appetitive or aversive behavioral responses. This information can be used to assess mosquito foraging behavior and develop novel control strategies, especially those that incorporate mosquito bait-and-kill technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562529">
        <p class="paperTitle">Chronic Interferon Stimulated Gene Transcription Promotes Oncogene Induced Breast Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Canasto-Chibuque, C.; Kim, J. H.; Hohl, M.; Leslie, C.; Reis-Filho, J. S.; Petrini, J. H.</p>
        <p class="info">Score: 5.5, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562529' target='https://doi.org/10.1101/2023.10.16.562529'> 10.1101/2023.10.16.562529</a></p>
        <p class="abstract">The Mre11 complex (comprising Mre11, Rad50, Nbs1) is integral to the maintenance of genome stability. We previously showed that a hypomorphic Mre11 mutant mouse strain (Mre11ATLD1/ATLD1) was highly susceptible to oncogene induced breast cancer. Here we used a mammary organoid system to examine which Mre11 dependent responses are tumor suppressive. We found that Mre11ATLD1/ATLD1 organoids exhibited an elevated interferon stimulated gene (ISG) signature and sustained changes in chromatin accessibility. This Mre11ATLD1/ATLD1 phenotype depended on DNA binding of a nuclear innate immune sensor, IFI205. Ablation of Ifi205 in Mre11ATLD1/ATLD1 organoids restored baseline and oncogene-induced chromatin accessibility patterns to those observed in WT. Implantation of Mre11ATLD1/ATLD1 organoids and activation of oncogene led to aggressive metastatic breast cancer. This outcome was reversed in implanted Ifi205-/- Mre11ATLD1/ATLD1 organoids. These data reveal a connection between innate immune signaling and tumor suppression in mammary epithelium. Given the abundance of aberrant DNA structures that arise in the context of genome instability syndromes, the data further suggest that cancer predisposition in those contexts may be partially attributable to tonic innate immune transcriptional programs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
