<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation
Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.
Score: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598
Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation
Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.
Score: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598
Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation
Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.
Score: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598
Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation\nAuthors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.\nScore: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598\nEpigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence.",
  "keywords": [
    
  ],
  "articleBody": " Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation\nAuthors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.\nScore: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598\nEpigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence. Nucleosomes block access to DNA methyltransferases, unless they are remodeled by DECREASE IN DNA METHYLATION1 (DDM1Lsh/HELLS), a Snf2-like master regulator of epigenetic inheritance. We show that DDM1 activity results in replacement of the transcriptional histone variant H3.3 for the replicative variant H3.1 during the cell cycle. In ddm1 mutants, DNA methylation can be restored by loss of the H3.3 chaperone HIRA, while the H3.1 chaperone CAF-1 becomes essential. The single-particle cryo-EM structure at 3.2 [A] of DDM1 with a variant nucleosome reveals direct engagement at SHL2 with histone H3.3 at or near variant residues required for assembly, as well as with the deacetylated H4 tail. An N-terminal autoinhibitory domain binds H2A variants to allow remodeling, while a disulfide bond in the helicase domain is essential for activity in vivo and in vitro. We show that differential remodeling of H3 and H2A variants in vitro reflects preferential deposition in vivo. DDM1 co-localizes with H3.1 and H3.3 during the cell cycle, and with the DNA methyltransferase MET1Dnmt1. DDM1 localization to the chromosome is blocked by H4K16 acetylation, which accumulates at DDM1 targets in ddm1 mutants, as does the sperm cell specific H3.3 variant MGH3 in pollen, which acts as a placeholder nucleosome in the germline and contributes to epigenetic inheritance.\nMulti-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation\nAuthors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.\nScore: 28.2, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546730\nPartial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.\nMolecular recognition of an aversive odorant by the murine trace amine-associated receptor TAAR7f\nAuthors: Gusach, A.; Lee, Y.; Khoshgrudi, A. N.; Mukhaleva, E.; Ma, N.; Koers, E. J.; Chen, Q.; Edwards, P.; Huang, F.; Kim, J.; Mancia, F.; Veprintsev, D. J.; Vaidehi, N.; Weyand, S.; Tate, C. G.\nScore: 17.3, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547762\nThere are two main families of G protein-coupled receptors that detect odours in humans, the odorant receptors (ORs) and the trace amine-associated receptors (TAARs). Their amino acid sequences are distinct, with the TAARs being most similar to the aminergic receptors such as those activated by adrenaline, serotonin and histamine. To elucidate the structural determinants of ligand recognition by TAARs, we have determined the cryo-EM structure of a murine receptor, mTAAR7f, coupled to the heterotrimeric G protein Gs and bound to the odorant N,N-dimethylcyclohexylamine (DMCH) to an overall resolution of 2.9 [A]. DMCH is bound in a hydrophobic orthosteric binding site primarily through van der Waals interactions and a strong charge-charge interaction between the tertiary amine of the ligand and an aspartic acid residue. This site is distinct and non-overlapping with the binding site for the odorant propionate in the odorant receptor OR51E2. The structure, in combination with mutagenesis data and molecular dynamics simulations suggests that the activation of the receptor follows a similar pathway to that of the {beta}-adrenoceptors, with the significant difference that DMCH interacts directly with one of the main activation microswitch residues.\nTrabectedin derails transcription-coupled nucleotide excision repair to induce DNA breaks in highly transcribed genes\nAuthors: Son, K.; Takhaveev, V.; Mor, V.; Yu, H.; Dillier, E.; Zilio, N.; Pullen, N. J. L.; Ivanov, D.; Ulrich, H. D.; Sturla, S. J.; Scharer, O. D.\nScore: 14.1, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548294\nMost genotoxic anticancer agents fail in tumors with intact DNA repair. Therefore, trabectedin, a unique agent more toxic to cells with active DNA repair, specifically transcription-coupled nucleotide excision repair (TC-NER), provides new therapeutic opportunities. To unlock the potential of trabectedin and inform its application in precision oncology, a full mechanistic understanding of the drugs TC-NER-dependent toxicity is needed. Here, we determined that abortive TC-NER of trabectedin-DNA adducts forms persistent single-strand breaks (SSBs) as the adducts block the second of the two sequential NER incisions. We mapped the 3-hydroxyl groups of SSBs originating from the first NER incision at trabectedin lesions, recording TC-NER on a genome-wide scale. We showed that trabectedin-induced SSBs primarily occur in transcribed strands of active genes and peak near transcription start sites. Frequent SSBs were also found outside gene bodies, connecting TC-NER to divergent transcription from promoters. This work advances the use trabectedin for precision oncology and for studying TC-NER and transcription.\nEfficient Formation of Single-copy Human Artificial Chromosomes\nAuthors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.\nScore: 43.5, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547284\nLarge DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells. One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.\nMolecular and structural basis of the heterochromatin-specific chromatin remodeling activity by Arabidopsis DDM1\nAuthors: Osakabe, A.; Takizawa, Y.; Horikoshi, N.; Hatazawa, S.; Negishi, L.; Berger, F.; Kakutani, T.; Kurumizaka, H.\nScore: 12.2, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548306\nThe chromatin remodeler DECREASE IN DNA METHYLATION 1 (DDM1) deposits the histone H2A variant H2A.W and silences transposons in Arabidopsis thaliana. However, the molecular mechanisms by which DDM1 specifically targets the nucleosome containing H2A.W and allows chromatin writers to access heterochromatin remained elusive. Here, we show that DDM1 promotes remodeling of the H2A.W nucleosome and requires interactions with the H2A.W-specific C-terminal tail. The cryo-EM structure of the DDM1-H2A.W nucleosome complex revealed that DDM1 binds to the N-terminal tail of H4 and the nucleosomal DNA. Comparison with the cryo-EM structure of the nucleosome containing H2A.W suggested that DDM1 increases the DNA end flexibility of nucleosomes. Based on these biochemical and structural results, we propose that the chromatin remodeling activity of DDM1 with the heterochromatin-specific H2A.W contributes to the maintenance of repressive epigenetic marks in heterochromatin by providing DNA methyltransferases with access to nucleosomal DNA.\nAntisense transcription and PRC2 repression function in parallel during vernalization\nAuthors: Nielsen, M.; Menon, G.; Zhao, Y.; Mateo-Bonmati, E.; Wolff, P.; Zhou, S.; Howard, M.; Dean, C.\nScore: 17.1, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547987\nNon-coding transcription induces chromatin changes that can mediate environmental responsiveness, but the causes and consequences of these mechanisms are still unclear. Here, we investigate how antisense transcription interfaces with Polycomb Repressive Complex 2 silencing during winter-induced epigenetic regulation of Arabidopsis FLOWERING LOCUS C (FLC). Through genetic, chromatin, and computational analyses, we show that FLC is silenced through pathways that function with different dynamics: an antisense transcription-mediated pathway capable of fast response; and in parallel a slow Polycomb Repressive Complex 2 (PRC2) switching mechanism that maintains each allele in an epigenetically silenced state. Components of both the antisense and PRC2 pathways are regulated by a common transcriptional regulator (NTL8), which accumulates slowly due to reduced growth at low temperatures. The parallel activities of the regulatory steps, which we encapsulate in a mathematical model, creates a flexible system for registering widely fluctuating natural temperature conditions that change year on year, and yet ensure robust epigenetic silencing of FLC. SignificanceThe role of non-coding transcription in establishing and maintaining chromatin states is controversial, mainly because of extensive feedbacks complicating analysis of the relationship between co-transcriptional processing, chromatin state and transcription. This controversy has extended to the role of antisense transcription in the Polycomb-mediated epigenetic silencing of Arabidopsis FLC, a key step in the process of vernalization. Here, we show that antisense transcription and PRC2 silence FLC in parallel pathways that are affected by growth dynamics and temperature fluctuations. These features explain the varied importance of antisense transcription in cold-induced FLC epigenetic silencing seen in various studies using different environmental and growth conditions. The parallel repressive inputs and extensive feedbacks make the mechanism counter-intuitive but provide great flexibility to the plant.\nTBP facilitates RNA Polymerase I transcription following mitosis\nAuthors: Kwan, J. Z. J.; Nguyen, T. F.; Teves, S. S.\nScore: 7.7, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548763\nThe TATA-box binding protein (TBP) is the sole transcription factor common in the initiation complexes of the three major eukaryotic RNA Polymerases (Pol I, II, and III). Decades of research have shown that TBP is essential for proper transcription by the three RNA Pols, though the emergence of TBP paralogs throughout evolution have expanded the complexity in RNA Pol initiation. We have previously shown that acute TBP depletion in mouse embryonic stem cells led to a global decrease in Pol III activity, consistent with the requirement of TBP in Pol III initiation. In contrast, Pol II transcription remained unaffected in the absence of TBP and its paralogs. In this report, we show that, in contrast to Pol II-transcribed genes, the TBP paralog TRF2 does not bind to Pol I promoters, and therefore cannot functionally replace TBP upon depletion. Importantly, acute TBP depletion has no major effect on Pol I occupancy or activity on ribosomal RNA genes, but TBP binding in mitosis leads to efficient Pol I reactivation following cell division. These findings provide a more nuanced role for TBP in Pol I transcription in murine embryonic stem cells.\nConserved and repetitive motifs in an intrinsically disordered protein drive α-carboxysome assembly\nAuthors: Turnsek, J. B.; Oltrogge, L. M.; Savage, D. F.\nScore: 6.5, Published: 2023-07-08 DOI: 10.1101/2023.07.08.548221\nAll photosynthetic bacteria and some chemoautotrophic bacteria fix CO2 into sugars in specialized proteinaceous compartments called carboxysomes. Carboxysomes enclose the enzymes Rubisco and carbonic anhydrase inside a layer of shell proteins to increase the CO2 concentration for efficient carbon fixation by Rubisco. In the -carboxysome lineage, a disordered and highly repetitive protein named CsoS2 is essential for carboxysome formation and function. Without it, the bacteria are unable to fix enough carbon to grow in air. How a protein lacking structure serves as the architectural scaffold for such a vital cellular compartment remains unknown. In this study, we identify key residues in CsoS2 that are necessary for building functional -carboxysomes in vivo. These highly conserved and repetitive residues, VTG and Y, contribute to the interaction between CsoS2 and shell proteins. We also demonstrate in vitro reconstitution of the -carboxysome into spherical condensates with CsoS2, Rubisco, and shell proteins, and show the utility of reconstitution as a biochemical tool to study carboxysome biogenesis. The precise self-assembly of thousands of proteins is crucial for carboxysome formation, and understanding this process could enable their use in alternative biological hosts or industrial processes as effective tools to fix carbon.\nThe environmentally-regulated interplay between local three-dimensional chromatin organisation and transcription of proVWX in E. coli\nAuthors: Rashid, F. Z. M.; Cramezy, F. G. E.; Hofmann, A.; Forrest, D.; Grainger, D. C.; Heermann, D. W.; Dame, R. T.\nScore: 6.5, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548347\nNucleoid associated proteins (NAPs) maintain the architecture of bacterial chromosomes and regulate gene expression. Thus, their role as transcription factors may involve three-dimensional chromosome re-organisation. While this model is supported by in vitro studies, direct in vivo evidence is lacking. Here, we use RT-qPCR and 3C-qPCR to study the transcriptional and architectural profiles of the H-NS-regulated, osmoresponsive proVWX operon of Escherichia coli at different osmolarities and provide in vivo evidence for transcription regulation by NAP-mediated chromosome re-modelling in bacteria. We show that activation of proVWX in response to a hyperosmotic shock involves the destabilization of H-NS-mediated bridges anchored between the proVWX downstream and upstream regulatory elements (DRE and URE), and between the DRE and ygaY that lies immediately downstream of proVWX. The re-establishment of these bridges upon adaptation to hyperosmolarity represses the operon. H-NS and H-NS-like proteins are wide-spread amongst bacteria, suggesting that chromosome re-modelling may be a typical feature of transcriptional control in bacteria.\n",
  "wordCount" : "2283",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548598" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548598">
        <p class="paperTitle">Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.</p>
        <p class="info">Score: 20.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548598' target='https://doi.org/10.1101/2023.07.11.548598'> 10.1101/2023.07.11.548598</a></p>
        <p class="abstract">Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence. Nucleosomes block access to DNA methyltransferases, unless they are remodeled by DECREASE IN DNA METHYLATION1 (DDM1Lsh/HELLS), a Snf2-like master regulator of epigenetic inheritance. We show that DDM1 activity results in replacement of the transcriptional histone variant H3.3 for the replicative variant H3.1 during the cell cycle. In ddm1 mutants, DNA methylation can be restored by loss of the H3.3 chaperone HIRA, while the H3.1 chaperone CAF-1 becomes essential. The single-particle cryo-EM structure at 3.2 [A] of DDM1 with a variant nucleosome reveals direct engagement at SHL2 with histone H3.3 at or near variant residues required for assembly, as well as with the deacetylated H4 tail. An N-terminal autoinhibitory domain binds H2A variants to allow remodeling, while a disulfide bond in the helicase domain is essential for activity in vivo and in vitro. We show that differential remodeling of H3 and H2A variants in vitro reflects preferential deposition in vivo. DDM1 co-localizes with H3.1 and H3.3 during the cell cycle, and with the DNA methyltransferase MET1Dnmt1. DDM1 localization to the chromosome is blocked by H4K16 acetylation, which accumulates at DDM1 targets in ddm1 mutants, as does the sperm cell specific H3.3 variant MGH3 in pollen, which acts as a placeholder nucleosome in the germline and contributes to epigenetic inheritance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546730">
        <p class="paperTitle">Multi-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.</p>
        <p class="info">Score: 28.2, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546730' target='https://doi.org/10.1101/2023.06.30.546730'> 10.1101/2023.06.30.546730</a></p>
        <p class="abstract">Partial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547762">
        <p class="paperTitle">Molecular recognition of an aversive odorant by the murine trace amine-associated receptor TAAR7f</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gusach, A.; Lee, Y.; Khoshgrudi, A. N.; Mukhaleva, E.; Ma, N.; Koers, E. J.; Chen, Q.; Edwards, P.; Huang, F.; Kim, J.; Mancia, F.; Veprintsev, D. J.; Vaidehi, N.; Weyand, S.; Tate, C. G.</p>
        <p class="info">Score: 17.3, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547762' target='https://doi.org/10.1101/2023.07.07.547762'> 10.1101/2023.07.07.547762</a></p>
        <p class="abstract">There are two main families of G protein-coupled receptors that detect odours in humans, the odorant receptors (ORs) and the trace amine-associated receptors (TAARs). Their amino acid sequences are distinct, with the TAARs being most similar to the aminergic receptors such as those activated by adrenaline, serotonin and histamine. To elucidate the structural determinants of ligand recognition by TAARs, we have determined the cryo-EM structure of a murine receptor, mTAAR7f, coupled to the heterotrimeric G protein Gs and bound to the odorant N,N-dimethylcyclohexylamine (DMCH) to an overall resolution of 2.9 [A]. DMCH is bound in a hydrophobic orthosteric binding site primarily through van der Waals interactions and a strong charge-charge interaction between the tertiary amine of the ligand and an aspartic acid residue. This site is distinct and non-overlapping with the binding site for the odorant propionate in the odorant receptor OR51E2. The structure, in combination with mutagenesis data and molecular dynamics simulations suggests that the activation of the receptor follows a similar pathway to that of the {beta}-adrenoceptors, with the significant difference that DMCH interacts directly with one of the main activation microswitch residues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548294">
        <p class="paperTitle">Trabectedin derails transcription-coupled nucleotide excision repair to induce DNA breaks in highly transcribed genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Son, K.; Takhaveev, V.; Mor, V.; Yu, H.; Dillier, E.; Zilio, N.; Pullen, N. J. L.; Ivanov, D.; Ulrich, H. D.; Sturla, S. J.; Scharer, O. D.</p>
        <p class="info">Score: 14.1, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548294' target='https://doi.org/10.1101/2023.07.10.548294'> 10.1101/2023.07.10.548294</a></p>
        <p class="abstract">Most genotoxic anticancer agents fail in tumors with intact DNA repair. Therefore, trabectedin, a unique agent more toxic to cells with active DNA repair, specifically transcription-coupled nucleotide excision repair (TC-NER), provides new therapeutic opportunities. To unlock the potential of trabectedin and inform its application in precision oncology, a full mechanistic understanding of the drugs TC-NER-dependent toxicity is needed. Here, we determined that abortive TC-NER of trabectedin-DNA adducts forms persistent single-strand breaks (SSBs) as the adducts block the second of the two sequential NER incisions. We mapped the 3-hydroxyl groups of SSBs originating from the first NER incision at trabectedin lesions, recording TC-NER on a genome-wide scale. We showed that trabectedin-induced SSBs primarily occur in transcribed strands of active genes and peak near transcription start sites. Frequent SSBs were also found outside gene bodies, connecting TC-NER to divergent transcription from promoters. This work advances the use trabectedin for precision oncology and for studying TC-NER and transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547284">
        <p class="paperTitle">Efficient Formation of Single-copy Human Artificial Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.</p>
        <p class="info">Score: 43.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547284' target='https://doi.org/10.1101/2023.06.30.547284'> 10.1101/2023.06.30.547284</a></p>
        <p class="abstract">Large DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.

One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548306">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548306" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548306">
        <p class="paperTitle">Molecular and structural basis of the heterochromatin-specific chromatin remodeling activity by Arabidopsis DDM1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548306" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548306" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Osakabe, A.; Takizawa, Y.; Horikoshi, N.; Hatazawa, S.; Negishi, L.; Berger, F.; Kakutani, T.; Kurumizaka, H.</p>
        <p class="info">Score: 12.2, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548306' target='https://doi.org/10.1101/2023.07.10.548306'> 10.1101/2023.07.10.548306</a></p>
        <p class="abstract">The chromatin remodeler DECREASE IN DNA METHYLATION 1 (DDM1) deposits the histone H2A variant H2A.W and silences transposons in Arabidopsis thaliana. However, the molecular mechanisms by which DDM1 specifically targets the nucleosome containing H2A.W and allows chromatin writers to access heterochromatin remained elusive. Here, we show that DDM1 promotes remodeling of the H2A.W nucleosome and requires interactions with the H2A.W-specific C-terminal tail. The cryo-EM structure of the DDM1-H2A.W nucleosome complex revealed that DDM1 binds to the N-terminal tail of H4 and the nucleosomal DNA. Comparison with the cryo-EM structure of the nucleosome containing H2A.W suggested that DDM1 increases the DNA end flexibility of nucleosomes. Based on these biochemical and structural results, we propose that the chromatin remodeling activity of DDM1 with the heterochromatin-specific H2A.W contributes to the maintenance of repressive epigenetic marks in heterochromatin by providing DNA methyltransferases with access to nucleosomal DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547987">
        <p class="paperTitle">Antisense transcription and PRC2 repression function in parallel during vernalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nielsen, M.; Menon, G.; Zhao, Y.; Mateo-Bonmati, E.; Wolff, P.; Zhou, S.; Howard, M.; Dean, C.</p>
        <p class="info">Score: 17.1, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547987' target='https://doi.org/10.1101/2023.07.07.547987'> 10.1101/2023.07.07.547987</a></p>
        <p class="abstract">Non-coding transcription induces chromatin changes that can mediate environmental responsiveness, but the causes and consequences of these mechanisms are still unclear. Here, we investigate how antisense transcription interfaces with Polycomb Repressive Complex 2 silencing during winter-induced epigenetic regulation of Arabidopsis FLOWERING LOCUS C (FLC). Through genetic, chromatin, and computational analyses, we show that FLC is silenced through pathways that function with different dynamics: an antisense transcription-mediated pathway capable of fast response; and in parallel a slow Polycomb Repressive Complex 2 (PRC2) switching mechanism that maintains each allele in an epigenetically silenced state. Components of both the antisense and PRC2 pathways are regulated by a common transcriptional regulator (NTL8), which accumulates slowly due to reduced growth at low temperatures. The parallel activities of the regulatory steps, which we encapsulate in a mathematical model, creates a flexible system for registering widely fluctuating natural temperature conditions that change year on year, and yet ensure robust epigenetic silencing of FLC.

SignificanceThe role of non-coding transcription in establishing and maintaining chromatin states is controversial, mainly because of extensive feedbacks complicating analysis of the relationship between co-transcriptional processing, chromatin state and transcription. This controversy has extended to the role of antisense transcription in the Polycomb-mediated epigenetic silencing of Arabidopsis FLC, a key step in the process of vernalization. Here, we show that antisense transcription and PRC2 silence FLC in parallel pathways that are affected by growth dynamics and temperature fluctuations. These features explain the varied importance of antisense transcription in cold-induced FLC epigenetic silencing seen in various studies using different environmental and growth conditions. The parallel repressive inputs and extensive feedbacks make the mechanism counter-intuitive but provide great flexibility to the plant.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548763">
        <p class="paperTitle">TBP facilitates RNA Polymerase I transcription following mitosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kwan, J. Z. J.; Nguyen, T. F.; Teves, S. S.</p>
        <p class="info">Score: 7.7, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548763' target='https://doi.org/10.1101/2023.07.13.548763'> 10.1101/2023.07.13.548763</a></p>
        <p class="abstract">The TATA-box binding protein (TBP) is the sole transcription factor common in the initiation complexes of the three major eukaryotic RNA Polymerases (Pol I, II, and III). Decades of research have shown that TBP is essential for proper transcription by the three RNA Pols, though the emergence of TBP paralogs throughout evolution have expanded the complexity in RNA Pol initiation. We have previously shown that acute TBP depletion in mouse embryonic stem cells led to a global decrease in Pol III activity, consistent with the requirement of TBP in Pol III initiation. In contrast, Pol II transcription remained unaffected in the absence of TBP and its paralogs. In this report, we show that, in contrast to Pol II-transcribed genes, the TBP paralog TRF2 does not bind to Pol I promoters, and therefore cannot functionally replace TBP upon depletion. Importantly, acute TBP depletion has no major effect on Pol I occupancy or activity on ribosomal RNA genes, but TBP binding in mitosis leads to efficient Pol I reactivation following cell division. These findings provide a more nuanced role for TBP in Pol I transcription in murine embryonic stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.548221">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.548221" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.548221">
        <p class="paperTitle">Conserved and repetitive motifs in an intrinsically disordered protein drive α-carboxysome assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.548221" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.548221" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Turnsek, J. B.; Oltrogge, L. M.; Savage, D. F.</p>
        <p class="info">Score: 6.5, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.548221' target='https://doi.org/10.1101/2023.07.08.548221'> 10.1101/2023.07.08.548221</a></p>
        <p class="abstract">All photosynthetic bacteria and some chemoautotrophic bacteria fix CO2 into sugars in specialized proteinaceous compartments called carboxysomes. Carboxysomes enclose the enzymes Rubisco and carbonic anhydrase inside a layer of shell proteins to increase the CO2 concentration for efficient carbon fixation by Rubisco. In the -carboxysome lineage, a disordered and highly repetitive protein named CsoS2 is essential for carboxysome formation and function. Without it, the bacteria are unable to fix enough carbon to grow in air. How a protein lacking structure serves as the architectural scaffold for such a vital cellular compartment remains unknown. In this study, we identify key residues in CsoS2 that are necessary for building functional -carboxysomes in vivo. These highly conserved and repetitive residues, VTG and Y, contribute to the interaction between CsoS2 and shell proteins. We also demonstrate in vitro reconstitution of the -carboxysome into spherical condensates with CsoS2, Rubisco, and shell proteins, and show the utility of reconstitution as a biochemical tool to study carboxysome biogenesis. The precise self-assembly of thousands of proteins is crucial for carboxysome formation, and understanding this process could enable their use in alternative biological hosts or industrial processes as effective tools to fix carbon.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548347">
        <p class="paperTitle">The environmentally-regulated interplay between local three-dimensional chromatin organisation and transcription of proVWX in E. coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rashid, F. Z. M.; Cramezy, F. G. E.; Hofmann, A.; Forrest, D.; Grainger, D. C.; Heermann, D. W.; Dame, R. T.</p>
        <p class="info">Score: 6.5, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548347' target='https://doi.org/10.1101/2023.07.10.548347'> 10.1101/2023.07.10.548347</a></p>
        <p class="abstract">Nucleoid associated proteins (NAPs) maintain the architecture of bacterial chromosomes and regulate gene expression. Thus, their role as transcription factors may involve three-dimensional chromosome re-organisation. While this model is supported by in vitro studies, direct in vivo evidence is lacking. Here, we use RT-qPCR and 3C-qPCR to study the transcriptional and architectural profiles of the H-NS-regulated, osmoresponsive proVWX operon of Escherichia coli at different osmolarities and provide in vivo evidence for transcription regulation by NAP-mediated chromosome re-modelling in bacteria. We show that activation of proVWX in response to a hyperosmotic shock involves the destabilization of H-NS-mediated bridges anchored between the proVWX downstream and upstream regulatory elements (DRE and URE), and between the DRE and ygaY that lies immediately downstream of proVWX. The re-establishment of these bridges upon adaptation to hyperosmolarity represses the operon. H-NS and H-NS-like proteins are wide-spread amongst bacteria, suggesting that chromosome re-modelling may be a typical feature of transcriptional control in bacteria.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
