<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion
Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.
Score: 113.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365
Cell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion
Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.
Score: 113.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365
Cell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion
Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.
Score: 113.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365
Cell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion\nAuthors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.\nScore: 113.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365\nCell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria.",
  "keywords": [
    
  ],
  "articleBody": " Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion\nAuthors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.\nScore: 113.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365\nCell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria. The molecular mechanism underlying ribosomal sequestration remains elusive. In this study, we performed time-lapse in situ cryo-electron tomography and cryo-electron microscopy complemented by biochemical experiments to elucidate the molecular details of this adaptive response. Our analysis indicate that upon glucose depletion protein synthesis is halted, causing ribosomes to enter an inactive state characterized by a conformational change that obstructs the peptidyl transferase center. Our in situ experiments reveal the presence of oligomeric arrays of hibernating ribosomes tethered to the mitochondrial surface. Surprisingly, ribosomes bind to the outer mitochondrial membrane via the small ribosomal subunit, an interaction facilitated by the ribosomal protein RACK1-orthologue Cpc2. Our experiments show that ribosome tethering is important for cell survival under glucose depletion conditions. This study broadens our understanding of the cellular adaptations triggered by nutrient scarcity and the underlying molecular mechanisms that regulate cell quiescence.\nChemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.\nAuthors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.\nScore: 24.5, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561763\nThis report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe. Streptavidin conjugated with horseradish peroxidase and enhanced chemiluminescence substrate are then used to detect the probe bound to the target RNA. Our results demonstrate the versatility of this method in detecting natural and engineered RNAs expressed in cells, including messenger and noncoding RNAs. We show that Chemi-Northern detection is sensitive and fast, detecting attomole amounts of RNA in as little as 1 second, with high signal intensity and low background. The dynamic response displays excellent linearity. Using Chemi-Northern, we measure the significant, reproducible reduction of mRNA levels by human sequence-specific RNA-binding proteins, PUM1 and PUM2. Additionally, we measure the interaction of endogenous poly(A) binding protein, PABPC1, with poly-adenylated mRNA. Thus, the Chemi-Northern method provides a versatile, simple, cost-effective method to enable researchers to detect and measure changes in RNA expression, processing, binding, and decay of RNAs.\nPredicting prime editing efficiency across diverse edit types and chromatin contexts with machine learning\nAuthors: Mathis, N.; Allam, A.; Talas, A.; Benvenuto, E.; Schep, R.; Damodharan, T.; Balazs, Z.; Janjuha, S.; Schmidheini, L.; Böck, D.; van Steensel, B.; Krauthammer, M.; Schwank, G.\nScore: 23.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561414\nPrime editing is a powerful genome editing technology, but its efficiency varies depending on the pegRNA design and target locus. Existing computational models for predicting prime editing rates are limited by their focus on specific edit types and by omitting the local chromatin environment. In our study, we developed machine learning models that predict prime editing efficiencies across a wide range of edit types up to 15 bp (PRIDICT2.0) and in different chromatin contexts (ePRIDICT). Both models can be accessed at www.pridict.it.\nOpen-source, high-throughput targeted in-situ transcriptomics for developmental biologists\nAuthors: Lee, H.; Mattsson Langseth, C.; Marco Salas, S.; Metousis, A.; Rueda Alana, E.; Garcia Moreno, F.; Grillo, M.; Nilsson, M.\nScore: 16.4, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561689\nMultiplexed spatial profiling of mRNAs has recently gained traction as a tool to explore the cellular diversity and the architecture of tissues. We propose a sensitive, open-source, simple and flexible method for the generation of in-situ expression maps of hundreds of genes. We exploit direct ligation of padlock probes on mRNAs, coupled with rolling circle amplification and hybridization-based in situ combinatorial barcoding, to achieve high detection efficiency, high throughput and large multiplexing. We validate the method across a number of species, and show its use in combination with orthogonal methods such as antibody staining, highlighting its potential value for developmental biology studies. Finally, we provide an end-to-end computational workflow that covers the steps of probe design, image processing, data extraction, cell segmentation, clustering and annotation of cell types. By enabling easier access to high-throughput spatially resolved transcriptomics, we hope to encourage a diversity of applications and the exploration of a wide range of biological questions.\nThe Rho GTPase Rac1 mediates exercise training adaptations\nAuthors: Raun, S. H.; Henriquez-Olguin, C.; Frank, E.; Knudsen, J. R.; Ali, M. S.; Andersen, N. R.; Moeller, L. L. V.; Davey, J.; Qian, H.; Coelho, A.; Carl, C. S.; Voldstedlund, C. T.; Kiens, B.; Holmdahl, R.; Gregorevic, P.; Jensen, T. E.; Richter, E. A.; Sylow, L.\nScore: 10.1, Published: 2023-10-09 DOI: 10.1101/2023.10.08.561442\nExercise training elicits tremendous health benefits; however, the molecular underpinnings are poorly understood. As one of the most regulated groups of proteins following acute exercise in human muscle, Rho GTPases are unexplored candidates for mediating the beneficial effects of exercise. The Rho GTPase Rac1 was activated during multiple exercise modalities and remained elevated hours after resistance exercise in human muscle. Inducible muscle-specific Rac1 knockout (Rac1 imKO) mice, displayed attenuated muscle protein synthesis, glycogen resynthesis and p38 MAPK signaling in recovery from contractions. Exercise training upregulated Rac1 protein content in human and mouse muscle. Overexpression of hyperactive Rac1 elevated reactive oxidant species production during exercise yet did not induce a trained muscle phenotype. In Rac1 imKO mice, the improvements in running capacity and muscle mass after exercise training were diminished. Using gain- and loss-of-function mouse models and human muscle biopsies, we identify Rac1 as a regulator of exercise training adaptions. HighlightsO_LIVarious exercise modalities activate Rac1 signaling in human skeletal muscle. C_LIO_LIHSP27, MNK1, and CREB are Rac1-dependent contraction-responsive targets in muscle. C_LIO_LIPost-contraction protein synthesis requires Rac1 but not NOX2. C_LIO_LIRac1-NOX2 signaling is necessary for post-contraction glycogen resynthesis. C_LIO_LIExercise training increases Rac1 protein content in human and mouse muscles. C_LIO_LIRac1 mediates critical adaptations to exercise training. C_LI\nOptical ice thickness measurement in the VitroJet for time-efficient single particle structure determination\nAuthors: Henderikx, R. J. M.; Shahzad, S.; Schotman, M. J. G.; Mann, D.; Heidler, T. V.; Leahy, E.; Ashtiani, D.; Jeurissen, R. J. M.; Peters, P. J.; Sachse, C.; Beulen, B. W. A. M. M.\nScore: 8.6, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561488\nIn cryo-electron microscopy structure determination, embedding biomolecules in vitreous ice of optimal thickness is critical. Ice thickness assessment and selection of suitable holes for data collection are currently part of time-consuming preparatory routines performed on expensive electron microscopes. For this reason, we developed a method to determine ice thickness during sample preparation using the optical camera integrated in the VitroJet. As a result, the ice thickness of buffer-suspended holes on an EM grid can be determined faithfully in the working range relevant for single particle cryo-EM. Dependent on the thickness, the error is below {+/-} 20 nm in the range between 0 - 70 nm down to {+/-} 10 nm in the thinnest ice regions (10 - 40 nm). Single particle structures of apoferritin were determined at two distinct thicknesses of 30 nm and 70 nm to demonstrate the suitability of optical layer thickness measurements for time-efficient cryo-EM structure determination.\nBroad variation in response of individual introns to splicing inhibitors in a humanized yeast strain\nAuthors: Hunter, O.; Talkish, J.; Quick-Cleveland, J.; Igel, H.; Tan, A.; Kuersten, S.; Katzman, S.; Donohue, J. P.; Jurica, M. S.; Ares, M.\nScore: 8.8, Published: 2023-10-05 DOI: 10.1101/2023.10.05.560965\nIntron branch point (BP) recognition by the U2 snRNP is a critical step of splicing, vulnerable to recurrent cancer mutations and bacterial natural product inhibitors. The BP binds a conserved pocket in the SF3B1 (human) or Hsh155 (yeast) U2 snRNP protein. Amino acids that line this pocket affect binding of splicing inhibitors like Pladienolide-B (Plad-B), such that organisms differ in their sensitivity. To study the mechanism of splicing inhibitor action in a simplified system, we modified the naturally Plad-B resistant yeast Saccharomyces cerevisiae by changing 14 amino acids in the Hsh155 BP pocket to those from human. This humanized yeast grows normally, and splicing is largely unaffected by the mutation. Minutes after addition of Plad-B splicing is inhibited, however different introns appear inhibited to different extents. Intron-specific inhibition differences are also observed when we measure co-transcriptional splicing in Plad-B using single-molecule intron tracking (SMIT) to minimize gene-specific transcription and decay rates that cloud estimates of inhibition by standard RNA-seq. Comparison of Plad-B intron sensitivities to those of the structurally distinct inhibitor Thailanstatin-A reveals intron-specific differences in sensitivity to different compounds. This work exposes a complex relationship between binding of different members of this class of inhibitors to the spliceosome and intron-specific rates of BP recognition and catalysis. The results echo and extend observations from mammalian cells, where inhibitor studies are complicated by multi-intron gene architecture and alternative splicing. The more compact yeast system may hasten characterization of splicing inhibitors, accelerating improvements in selectivity and therapeutic efficacy.\nSHIFTR enables the unbiased and multiplexed identification of proteins bound to specific RNA regions in live cells\nAuthors: Aydin, J.; Gabel, A.; Zielinski, S.; Ganskih, S.; Schmidt, N.; Hartigan, C. R.; Schenone, M.; Carr, S. A.; Munschauer, M.\nScore: 6.8, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561498\nRNA-protein interactions determine the cellular fate of RNA and are central to regulating gene expression outcomes in health and disease. To date, no method exists that is able to identify proteins that interact with specific regions within endogenous RNAs in live cells. Here, we develop SHIFTR (Selective RNase H-mediated interactome framing for target RNA regions), an efficient and scalable approach to identify proteins bound to selected regions within endogenous RNAs using mass spectrometry. Compared to state-of-the-art techniques, SHIFTR is superior in accuracy, captures close to zero background interactions and requires orders of magnitude lower input material. We establish SHIFTR workflows for targeting RNA classes of different length and abundance, including short and long non-coding RNAs, as well as mRNAs and demonstrate that SHIFTR is compatible with sequentially mapping interactomes for multiple target RNAs in a single experiment. Using SHIFTR, we comprehensively identify interactions of cis-regulatory elements located at the 5' and 3'- terminal regions of the authentic SARS-CoV-2 RNA genome in infected cells and accurately recover known and novel interactions linked to the function of these viral RNA elements. SHIFTR enables the systematic mapping of region-resolved RNA interactomes for any RNA in any cell type and has the potential to revolutionize our understanding of transcriptomes and their regulation.\nInter-organellar nucleic acid communication by a mitochondrial tRNA regulates nuclear metabolic transcription\nAuthors: Rouya, C.; Yambire, K. F.; Derbyshire, M. L.; Alwaseem, H.; Tavazoie, S. F.\nScore: 5.3, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558912\nEfficient communication between mitochondria and the nucleus underlies homoeostatic metabolic control, though the involved mitochondrial factors and their mechanisms are poorly defined. Here, we report the surprising detection of multiple mitochondrial-derived transfer RNAs (mito-tRNAs) within the nuclei of human cells. Focused studies of nuclear-transported mito-tRNA-asparagine (mtAsn) revealed that its cognate charging enzyme (NARS2) is also present in the nucleus. MtAsn promoted interaction of NARS2 with histone deacetylase 2 (HDAC2), and repressed HDAC2 association with specific chromatin loci. Perturbation of this axis using antisense oligonucleotides promoted nucleotide biogenesis and enhanced breast cancer growth, and RNA and nascent transcript sequencing demonstrated specific alterations in the transcription of nuclear genes. These findings uncover nucleic-acid mediated communication between two organelles and the existence of a machinery for nuclear gene regulation by a mito-tRNA that restricts tumor growth through metabolic control. HighlightsO_LIMultiple mitochondrial-derived tRNAs are detected in human cell nuclei C_LIO_LIMtAsn promotes binding between NARS2 and HDAC2 C_LIO_LIMetabolic alterations driven by mtAsn impact cell proliferation C_LIO_LIMtAsn inhibition releases HDAC2 to bind and transcriptionally regulate multiple nuclear genes C_LI\nECM-Focused Proteomic Analysis of Ear Punch Regeneration in Acomys cahirinus\nAuthors: McCabe, M. C.; Okamura, D. M.; Erickson, C. B.; Perry, B. W.; Brewer, C. M.; Nguyen, E. D.; Saviola, A. J.; Majesky, M. W.; Hansen, K. C.\nScore: 4.1, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561940\nIn mammals, significant injury is generally followed by the formation of a fibrotic scar which provides structural integrity but fails to functionally restore damaged tissue. Spiny mice of the genus Acomys represent the first example of full skin autotomy in mammals. Acomys cahirinus has evolved extremely weak skin as a strategy to avoid predation and is able to repeatedly regenerate healthy tissue without scar after severe skin injury or full-thickness ear punches. Extracellular matrix (ECM) composition is a critical regulator of wound repair and scar formation and previous studies have suggested that alterations in its expression may be responsible for the differences in regenerative capacity observed between Mus musculus and A. cahirinus, yet analysis of this critical tissue component has been limited in previous studies by its insolubility and resistance to extraction. Here, we utilize a 3-step ECM-optimized extraction to perform proteomic analysis of tissue composition during wound repair after full-thickness ear punches in A. cahirinus and M. musculus from weeks 1 to 4 post-injury. We observe changes in a wide range of ECM proteins which have been previously implicated in wound regeneration and scar formation, including collagens, coagulation and provisional matrix proteins, and matricryptic signaling peptides. We additionally report differences in crosslinking enzyme activity and ECM protein solubility between Mus and Acomys. Furthermore, we observed rapid and sustained increases in CD206, a marker of pro-regenerative M2 macrophages, in Acomys, whereas little or no increase in CD206 was detected in Mus. Together, these findings contribute to a comprehensive understanding of tissue cues which drive the regenerative capacity of Acomys and identify a number of potential targets for future pro-regenerative therapies.\n",
  "wordCount" : "2306",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561365">
        <p class="paperTitle">Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.</p>
        <p class="info">Score: 113.1, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561365' target='https://doi.org/10.1101/2023.10.08.561365'> 10.1101/2023.10.08.561365</a></p>
        <p class="abstract">Cell survival under nutrient-deprived conditions relies on cells ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria. The molecular mechanism underlying ribosomal sequestration remains elusive. In this study, we performed time-lapse in situ cryo-electron tomography and cryo-electron microscopy complemented by biochemical experiments to elucidate the molecular details of this adaptive response. Our analysis indicate that upon glucose depletion protein synthesis is halted, causing ribosomes to enter an inactive state characterized by a conformational change that obstructs the peptidyl transferase center. Our in situ experiments reveal the presence of oligomeric arrays of hibernating ribosomes tethered to the mitochondrial surface. Surprisingly, ribosomes bind to the outer mitochondrial membrane via the small ribosomal subunit, an interaction facilitated by the ribosomal protein RACK1-orthologue Cpc2. Our experiments show that ribosome tethering is important for cell survival under glucose depletion conditions. This study broadens our understanding of the cellular adaptations triggered by nutrient scarcity and the underlying molecular mechanisms that regulate cell quiescence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561763">
        <p class="paperTitle">Chemi-Northern: a versatile chemiluminescent northern blot method for analysis and quantitation of RNA molecules.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McKenney, K. M.; Connacher, R. P.; Dunshee, E. B.; Goldstrohm, A. C.</p>
        <p class="info">Score: 24.5, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561763' target='https://doi.org/10.1101/2023.10.10.561763'> 10.1101/2023.10.10.561763</a></p>
        <p class="abstract">This report describes a chemiluminescence-based detection method for RNAs on northern blots, designated Chemi-Northern. This approach builds on the simplicity and versatility of northern blotting, while dispensing of the need for expensive and cumbersome radioactivity. RNAs are first separated on denaturing gel electrophoresis, transferred to a nylon membrane, and then hybridized to a biotinylated RNA or DNA antisense probe. Streptavidin conjugated with horseradish peroxidase and enhanced chemiluminescence substrate are then used to detect the probe bound to the target RNA. Our results demonstrate the versatility of this method in detecting natural and engineered RNAs expressed in cells, including messenger and noncoding RNAs. We show that Chemi-Northern detection is sensitive and fast, detecting attomole amounts of RNA in as little as 1 second, with high signal intensity and low background. The dynamic response displays excellent linearity. Using Chemi-Northern, we measure the significant, reproducible reduction of mRNA levels by human sequence-specific RNA-binding proteins, PUM1 and PUM2. Additionally, we measure the interaction of endogenous poly(A) binding protein, PABPC1, with poly-adenylated mRNA. Thus, the Chemi-Northern method provides a versatile, simple, cost-effective method to enable researchers to detect and measure changes in RNA expression, processing, binding, and decay of RNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561414">
        <p class="paperTitle">Predicting prime editing efficiency across diverse edit types and chromatin contexts with machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mathis, N.; Allam, A.; Talas, A.; Benvenuto, E.; Schep, R.; Damodharan, T.; Balazs, Z.; Janjuha, S.; Schmidheini, L.; Böck, D.; van Steensel, B.; Krauthammer, M.; Schwank, G.</p>
        <p class="info">Score: 23.7, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561414' target='https://doi.org/10.1101/2023.10.09.561414'> 10.1101/2023.10.09.561414</a></p>
        <p class="abstract">Prime editing is a powerful genome editing technology, but its efficiency varies depending on the pegRNA design and target locus. Existing computational models for predicting prime editing rates are limited by their focus on specific edit types and by omitting the local chromatin environment. In our study, we developed machine learning models that predict prime editing efficiencies across a wide range of edit types up to 15 bp (PRIDICT2.0) and in different chromatin contexts (ePRIDICT). Both models can be accessed at www.pridict.it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561689">
        <p class="paperTitle">Open-source, high-throughput targeted in-situ transcriptomics for developmental biologists</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, H.; Mattsson Langseth, C.; Marco Salas, S.; Metousis, A.; Rueda Alana, E.; Garcia Moreno, F.; Grillo, M.; Nilsson, M.</p>
        <p class="info">Score: 16.4, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561689' target='https://doi.org/10.1101/2023.10.10.561689'> 10.1101/2023.10.10.561689</a></p>
        <p class="abstract">Multiplexed spatial profiling of mRNAs has recently gained traction as a tool to explore the cellular diversity and the architecture of tissues. We propose a sensitive, open-source, simple and flexible method for the generation of in-situ expression maps of hundreds of genes. We exploit direct ligation of padlock probes on mRNAs, coupled with rolling circle amplification and hybridization-based in situ combinatorial barcoding, to achieve high detection efficiency, high throughput and large multiplexing. We validate the method across a number of species, and show its use in combination with orthogonal methods such as antibody staining, highlighting its potential value for developmental biology studies. Finally, we provide an end-to-end computational workflow that covers the steps of probe design, image processing, data extraction, cell segmentation, clustering and annotation of cell types. By enabling easier access to high-throughput spatially resolved transcriptomics, we hope to encourage a diversity of applications and the exploration of a wide range of biological questions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561442" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561442">
        <p class="paperTitle">The Rho GTPase Rac1 mediates exercise training adaptations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raun, S. H.; Henriquez-Olguin, C.; Frank, E.; Knudsen, J. R.; Ali, M. S.; Andersen, N. R.; Moeller, L. L. V.; Davey, J.; Qian, H.; Coelho, A.; Carl, C. S.; Voldstedlund, C. T.; Kiens, B.; Holmdahl, R.; Gregorevic, P.; Jensen, T. E.; Richter, E. A.; Sylow, L.</p>
        <p class="info">Score: 10.1, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561442' target='https://doi.org/10.1101/2023.10.08.561442'> 10.1101/2023.10.08.561442</a></p>
        <p class="abstract">Exercise training elicits tremendous health benefits; however, the molecular underpinnings are poorly understood. As one of the most regulated groups of proteins following acute exercise in human muscle, Rho GTPases are unexplored candidates for mediating the beneficial effects of exercise. The Rho GTPase Rac1 was activated during multiple exercise modalities and remained elevated hours after resistance exercise in human muscle. Inducible muscle-specific Rac1 knockout (Rac1 imKO) mice, displayed attenuated muscle protein synthesis, glycogen resynthesis and p38 MAPK signaling in recovery from contractions. Exercise training upregulated Rac1 protein content in human and mouse muscle. Overexpression of hyperactive Rac1 elevated reactive oxidant species production during exercise yet did not induce a trained muscle phenotype. In Rac1 imKO mice, the improvements in running capacity and muscle mass after exercise training were diminished. Using gain- and loss-of-function mouse models and human muscle biopsies, we identify Rac1 as a regulator of exercise training adaptions.

HighlightsO_LIVarious exercise modalities activate Rac1 signaling in human skeletal muscle.
C_LIO_LIHSP27, MNK1, and CREB are Rac1-dependent contraction-responsive targets in muscle.
C_LIO_LIPost-contraction protein synthesis requires Rac1 but not NOX2.
C_LIO_LIRac1-NOX2 signaling is necessary for post-contraction glycogen resynthesis.
C_LIO_LIExercise training increases Rac1 protein content in human and mouse muscles.
C_LIO_LIRac1 mediates critical adaptations to exercise training.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561488">
        <p class="paperTitle">Optical ice thickness measurement in the VitroJet for time-efficient single particle structure determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henderikx, R. J. M.; Shahzad, S.; Schotman, M. J. G.; Mann, D.; Heidler, T. V.; Leahy, E.; Ashtiani, D.; Jeurissen, R. J. M.; Peters, P. J.; Sachse, C.; Beulen, B. W. A. M. M.</p>
        <p class="info">Score: 8.6, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561488' target='https://doi.org/10.1101/2023.10.09.561488'> 10.1101/2023.10.09.561488</a></p>
        <p class="abstract">In cryo-electron microscopy structure determination, embedding biomolecules in vitreous ice of optimal thickness is critical. Ice thickness assessment and selection of suitable holes for data collection are currently part of time-consuming preparatory routines performed on expensive electron microscopes. For this reason, we developed a method to determine ice thickness during sample preparation using the optical camera integrated in the VitroJet. As a result, the ice thickness of buffer-suspended holes on an EM grid can be determined faithfully in the working range relevant for single particle cryo-EM. Dependent on the thickness, the error is below {&#43;/-} 20 nm in the range between 0 - 70 nm down to {&#43;/-} 10 nm in the thinnest ice regions (10 - 40 nm). Single particle structures of apoferritin were determined at two distinct thicknesses of 30 nm and 70 nm to demonstrate the suitability of optical layer thickness measurements for time-efficient cryo-EM structure determination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560965">
        <p class="paperTitle">Broad variation in response of individual introns to splicing inhibitors in a humanized yeast strain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hunter, O.; Talkish, J.; Quick-Cleveland, J.; Igel, H.; Tan, A.; Kuersten, S.; Katzman, S.; Donohue, J. P.; Jurica, M. S.; Ares, M.</p>
        <p class="info">Score: 8.8, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560965' target='https://doi.org/10.1101/2023.10.05.560965'> 10.1101/2023.10.05.560965</a></p>
        <p class="abstract">Intron branch point (BP) recognition by the U2 snRNP is a critical step of splicing, vulnerable to recurrent cancer mutations and bacterial natural product inhibitors. The BP binds a conserved pocket in the SF3B1 (human) or Hsh155 (yeast) U2 snRNP protein. Amino acids that line this pocket affect binding of splicing inhibitors like Pladienolide-B (Plad-B), such that organisms differ in their sensitivity. To study the mechanism of splicing inhibitor action in a simplified system, we modified the naturally Plad-B resistant yeast Saccharomyces cerevisiae by changing 14 amino acids in the Hsh155 BP pocket to those from human. This humanized yeast grows normally, and splicing is largely unaffected by the mutation. Minutes after addition of Plad-B splicing is inhibited, however different introns appear inhibited to different extents. Intron-specific inhibition differences are also observed when we measure co-transcriptional splicing in Plad-B using single-molecule intron tracking (SMIT) to minimize gene-specific transcription and decay rates that cloud estimates of inhibition by standard RNA-seq. Comparison of Plad-B intron sensitivities to those of the structurally distinct inhibitor Thailanstatin-A reveals intron-specific differences in sensitivity to different compounds. This work exposes a complex relationship between binding of different members of this class of inhibitors to the spliceosome and intron-specific rates of BP recognition and catalysis. The results echo and extend observations from mammalian cells, where inhibitor studies are complicated by multi-intron gene architecture and alternative splicing. The more compact yeast system may hasten characterization of splicing inhibitors, accelerating improvements in selectivity and therapeutic efficacy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561498">
        <p class="paperTitle">SHIFTR enables the unbiased and multiplexed identification of proteins bound to specific RNA regions in live cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aydin, J.; Gabel, A.; Zielinski, S.; Ganskih, S.; Schmidt, N.; Hartigan, C. R.; Schenone, M.; Carr, S. A.; Munschauer, M.</p>
        <p class="info">Score: 6.8, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561498' target='https://doi.org/10.1101/2023.10.09.561498'> 10.1101/2023.10.09.561498</a></p>
        <p class="abstract">RNA-protein interactions determine the cellular fate of RNA and are central to regulating gene expression outcomes in health and disease. To date, no method exists that is able to identify proteins that interact with specific regions within endogenous RNAs in live cells. Here, we develop SHIFTR (Selective RNase H-mediated interactome framing for target RNA regions), an efficient and scalable approach to identify proteins bound to selected regions within endogenous RNAs using mass spectrometry. Compared to state-of-the-art techniques, SHIFTR is superior in accuracy, captures close to zero background interactions and requires orders of magnitude lower input material. We establish SHIFTR workflows for targeting RNA classes of different length and abundance, including short and long non-coding RNAs, as well as mRNAs and demonstrate that SHIFTR is compatible with sequentially mapping interactomes for multiple target RNAs in a single experiment. Using SHIFTR, we comprehensively identify interactions of cis-regulatory elements located at the 5&#39; and 3&#39;- terminal regions of the authentic SARS-CoV-2 RNA genome in infected cells and accurately recover known and novel interactions linked to the function of these viral RNA elements. SHIFTR enables the systematic mapping of region-resolved RNA interactomes for any RNA in any cell type and has the potential to revolutionize our understanding of transcriptomes and their regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558912">
        <p class="paperTitle">Inter-organellar nucleic acid communication by a mitochondrial tRNA regulates nuclear metabolic transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rouya, C.; Yambire, K. F.; Derbyshire, M. L.; Alwaseem, H.; Tavazoie, S. F.</p>
        <p class="info">Score: 5.3, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558912' target='https://doi.org/10.1101/2023.09.21.558912'> 10.1101/2023.09.21.558912</a></p>
        <p class="abstract">Efficient communication between mitochondria and the nucleus underlies homoeostatic metabolic control, though the involved mitochondrial factors and their mechanisms are poorly defined. Here, we report the surprising detection of multiple mitochondrial-derived transfer RNAs (mito-tRNAs) within the nuclei of human cells. Focused studies of nuclear-transported mito-tRNA-asparagine (mtAsn) revealed that its cognate charging enzyme (NARS2) is also present in the nucleus. MtAsn promoted interaction of NARS2 with histone deacetylase 2 (HDAC2), and repressed HDAC2 association with specific chromatin loci. Perturbation of this axis using antisense oligonucleotides promoted nucleotide biogenesis and enhanced breast cancer growth, and RNA and nascent transcript sequencing demonstrated specific alterations in the transcription of nuclear genes. These findings uncover nucleic-acid mediated communication between two organelles and the existence of a machinery for nuclear gene regulation by a mito-tRNA that restricts tumor growth through metabolic control.

HighlightsO_LIMultiple mitochondrial-derived tRNAs are detected in human cell nuclei
C_LIO_LIMtAsn promotes binding between NARS2 and HDAC2
C_LIO_LIMetabolic alterations driven by mtAsn impact cell proliferation
C_LIO_LIMtAsn inhibition releases HDAC2 to bind and transcriptionally regulate multiple nuclear genes
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561940">
        <p class="paperTitle">ECM-Focused Proteomic Analysis of Ear Punch Regeneration in Acomys cahirinus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCabe, M. C.; Okamura, D. M.; Erickson, C. B.; Perry, B. W.; Brewer, C. M.; Nguyen, E. D.; Saviola, A. J.; Majesky, M. W.; Hansen, K. C.</p>
        <p class="info">Score: 4.1, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561940' target='https://doi.org/10.1101/2023.10.11.561940'> 10.1101/2023.10.11.561940</a></p>
        <p class="abstract">In mammals, significant injury is generally followed by the formation of a fibrotic scar which provides structural integrity but fails to functionally restore damaged tissue. Spiny mice of the genus Acomys represent the first example of full skin autotomy in mammals. Acomys cahirinus has evolved extremely weak skin as a strategy to avoid predation and is able to repeatedly regenerate healthy tissue without scar after severe skin injury or full-thickness ear punches. Extracellular matrix (ECM) composition is a critical regulator of wound repair and scar formation and previous studies have suggested that alterations in its expression may be responsible for the differences in regenerative capacity observed between Mus musculus and A. cahirinus, yet analysis of this critical tissue component has been limited in previous studies by its insolubility and resistance to extraction. Here, we utilize a 3-step ECM-optimized extraction to perform proteomic analysis of tissue composition during wound repair after full-thickness ear punches in A. cahirinus and M. musculus from weeks 1 to 4 post-injury. We observe changes in a wide range of ECM proteins which have been previously implicated in wound regeneration and scar formation, including collagens, coagulation and provisional matrix proteins, and matricryptic signaling peptides. We additionally report differences in crosslinking enzyme activity and ECM protein solubility between Mus and Acomys. Furthermore, we observed rapid and sustained increases in CD206, a marker of pro-regenerative M2 macrophages, in Acomys, whereas little or no increase in CD206 was detected in Mus. Together, these findings contribute to a comprehensive understanding of tissue cues which drive the regenerative capacity of Acomys and identify a number of potential targets for future pro-regenerative therapies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
