<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody
Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody
Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody
Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody\nAuthors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.",
  "keywords": [
    
  ],
  "articleBody": " Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody\nAuthors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.\nScore: 321.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575741\nSUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.\nA universal molecular mechanism driving aging\nAuthors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.\nScore: 57.5, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574476\nHow cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin. This creates a more permissive local environment for G4 formation in subsequent generations. As a result, G4s gradually accumulate on promoters throughout mitosis, driving clock-like DNA hypomethylation and chromatin opening. In patients and in vitro models, loss-of-function mutations in the G4-resolving enzymes WRN, BLM and ERCC8 accelerate the erosion of the epigenomic landscape around G4. G4-driven epigenomic aging is strongly correlated with biological age and is conserved in yeast, nematodes, insects, fish, rodents, and humans. Our results revealed a universal molecular mechanism of aging and provided mechanistic insight into how G-quadruplex processor mutations drive premature aging.\nThe mitochondrial mRNA stabilizing protein, SLIRP, regulates skeletal muscle mitochondrial structure and respiration by exercise-recoverable mechanisms\nAuthors: Pham, T. C. P.; Raun, S. H.; Havula, E.; Henriquez-Olguin, C.; Rubalcava-Gracia, D.; Frank, E.; Fritzen, A. M.; Jannig, P. R.; Andersen, N. R.; Kruse, R.; Ali, M.; Halling, J. F.; Ringholm, S.; Needham, E. J.; Hansen, S.; Lemminger, A. K.; Schjerling, P.; Petersen, M. H.; de Almeida, M. E.; Jensen, T. E.; Kiens, B.; Hostrup, M.; Larsen, S.; Ortenblad, N.; Hojlund, K.; Kjaer, M.; Ruas, J. L.; Trifunovic, A.; Wojtaszewski, J. F. P.; Nielsen, J.; Qvortrup, K.; Pilegaard, H.; Richter, E. A.; Sylow, L.\nScore: 16.0, Published: 2024-01-15 DOI: 10.1101/2023.10.30.564600\nSummary and graphical abstractDecline in mitochondrial function associates with decreased muscle mass and strength in multiple conditions, including sarcopenia and type 2 diabetes. Optimal treatment could include improving mitochondrial function, however, there are limited and equivocal data regarding the molecular cues controlling muscle mitochondrial plasticity. Here we uncover the mitochondrial-mRNA-stabilizing protein SLIRP, in complex with LRPPRC, as a PGC-1 target that regulates mitochondrial structure, respiration, and mitochondrially-encoded-mRNA pools in skeletal muscle. Exercise training effectively counteracted mitochondrial defects induced by loss of LRPPRC/SLIRP, despite sustained low mitochondrially-encoded-mRNA pools, via increased mitoribosome translation capacity. In humans, exercise training robustly increased muscle SLIRP and LRPPRC protein content across exercise modalities and sexes, yet this increase was less prominent in subjects with type 2 diabetes. Our work identifies a mechanism of post-transcriptional mitochondrial regulation in skeletal muscle through mitochondrial mRNA stabilization. It emphasizes exercise as an effective approach to alleviate mitochondrial defects by possibly increasing mitoribosome capacity. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=142 SRC=\"FIGDIR/small/564600v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (61K): org.highwire.dtl.DTLVardef@88b3e7org.highwire.dtl.DTLVardef@db21bdorg.highwire.dtl.DTLVardef@841960org.highwire.dtl.DTLVardef@9e16ec_HPS_FORMAT_FIGEXP M_FIG C_FIG\nHeterodimerization of Endolysin Isoforms During Bacterial Infection by Staphylococcal Phage {varphi}2638A\nAuthors: Zinsli, L. V.; Sobieraj, A. M.; Ernst, P.; Meile, S.; Kilcher, S.; Iseli, C.; Keller, A.; Dreier, B.; Mittl, P. R. E.; Plueckthun, A.; Loessner, M. J.; Schmelcher, M.; Dunne, M.\nScore: 11.0, Published: 2024-01-16 DOI: 10.1101/2024.01.16.575832\nAO_SCPLOWBSTRACTC_SCPLOWBacteriophage endolysins targeting Gram-positive bacteria typically feature a modular architecture of one or more enzymatically active domains (EADs) and cell wall binding domains (CBDs). Several endolysins also feature internal translational start sites (iTSSs) that produce short variant (SV) isoforms alongside the full-length (FL) endolysin. While the lytic activity of endolysins and their isoforms has been extensively studied as exogenous agents, the purpose behind producing the SV isoform during the phage infection cycle remains to be explored. In this study, we used staphylococcal phage {varphi}2638A as a model to determine the interplay between its full-length endolysin, Ply2638A, and its SV isoform during phage infection. X-ray crystallography structures and AlphaFold-generated models enabled elucidation of individual functions of the M23 endopeptidase, central amidase, and SH3b domains of Ply2638A. Production of the SV isoform (amidase and SH3b) was confirmed during phage infection and shown to form a heterodimer complex with Ply2638A via inter-amidase domain interactions. Using genetically engineered phage variants, we show that production of both isoforms provides an advantage during phage infection as phages producing only one isoform presented impaired lytic activity, which could be partly restored through recombinant protein complementation of the missing isoform. Importantly, when applied as an antimicrobial protein against Staphylococcus aureus in culture, the activity of Ply2638A remained constant regardless of SV isoform complementation. Drawing from our findings, we propose that SV isoform production provides its biological advantage upon endolysin entry to the periplasmic space to ensure optimal peptidoglycan degradation prior to cell wall lysis and progeny phage release.\nBrain and cancer associated binding domain mutations provide insight into CTCF's relationship with chromatin and its ability to act as a chromatin organizer\nAuthors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.\nScore: 18.2, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575070\nAlthough only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown. To investigate the relationship between CTCF and chromatin we applied a combination of imaging, structural and molecular approaches, using a series of brain and cancer associated CTCF mutations that act as CTCF perturbations. We demonstrate that binding and the functional impact of WT and mutant CTCF depend not only on the unique binding properties of each protein, but also on the genomic context of bound sites and enrichment of motifs for expressed TFs abutting these sites. Our studies also highlight the reciprocal relationship between CTCF and chromatin, demonstrating that the unique binding properties of WT and mutant proteins have a distinct impact on accessibility, TF binding, cohesin overlap, chromatin interactivity and gene expression programs, providing insight into their cancer and brain related effects.\nKAT5 Acetylates Human Chromatin Protein PC4 to promote DNA Repair\nAuthors: kundu, T. K.; Agrawal, A.; Sikder, S.; Singh, S.; Peraman, R.; Ravichandran, V.\nScore: 6.2, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575390\nHuman positive coactivator 4 (PC4), is a highly abundant non-histone chromatin protein involved in diverse cellular processes, including transcription regulation, genome organization, DNA repair, etc. The majority of PC4 exists in a phosphorylated state in cells, which impinges its acetylation by p300 and thereby inhibits its double-stranded DNA binding ability and transcriptional co-activator function. Recently, we have shown that PC4 interacts with linker histone H1 in its phosphorylated state, and this interaction is important for PC4- mediated chromatin compaction. PC4 was also found to be an activator of non-homologous end joining and DSB repair activity. Knockdown of PC4 causes drastic decompaction and enhanced autophagy in the cells. Mechanistically, in the absence of PC4, the genome becomes highly vulnerable to DNA damage with an altered epigenetic landscape. Here, we report that other than p300, PC4 also gets acetylated by DNA repair facilitating lysine acetyltransferase KAT5 (Tip60), at K80 residue when the cells are subjected to DNA damage. The vulnerability of DNA in PC4 devoid cells could be substantially reduced by reintroducing wild-type PC4 to the cells but not the mutant PC4 (K80R PC4), defective in KAT5-mediated acetylation. Presumably, KAT5-mediated acetylation of PC4 at K80 residue facilitates the DNA repair machinery at the damage site and thus contributes to DNA damage repair, a process that could be of high significance both in cancer and in aging.\nMobile-CRISPRi as a powerful tool for modulating Vibrio gene expression\nAuthors: Geyman, L. J.; Tanner, M.; Rosario-Melendez, N.; Peters, J.; Mandel, M. J.; van Kessel, J. C.\nScore: 5.6, Published: 2024-01-18 DOI: 10.1101/2024.01.17.575898\nCRISPRi (Clustered Regularly Interspaced Palindromic Repeats interference) is a gene knockdown method that uses a deactivated Cas9 protein (dCas9) that binds a specific gene target locus dictated by an encoded guide RNA (sgRNA) to block transcription. Mobile-CRISPRi is a suite of modular vectors that enable CRISPRi knockdowns in diverse bacteria by integrating IPTG-inducible dcas9 and sgRNA genes into the genome using Tn7 transposition. Here, we show that the Mobile-CRISPRi system functions robustly and specifically in multiple Vibrio species: Vibrio cholerae, Vibrio fischeri, Vibrio vulnificus, Vibrio parahaemolyticus, and Vibrio campbellii. We demonstrate efficacy by targeting both essential and non-essential genes that function to produce defined, measurable phenotypes: bioluminescence, quorum sensing, cell division, and growth arrest. We anticipate that Mobile-CRISPRi will be used in Vibrio species to systematically probe gene function and essentiality in various behaviors and native environments.\nChromatin remodeling initiates transcriptional and post-transcriptional silencing in Neurospora\nAuthors: Carlier, F.; Castro Ramirez, S.; Kilani, J.; Chehboub, S.; Gladyshev, E.\nScore: 4.8, Published: 2024-01-15 DOI: 10.1101/2024.01.15.575522\nIn many eukaryotes, including mammals, transcriptional (heterochromatin-mediated) and post-transcriptional (RNAi-mediated) silencing of repetitive DNA can be initiated by poorly understood mechanisms independent of strong sequence-specific cues. We found that in the fungus Neurospora crassa both types of silencing were induced by the aberrant binding of transcription factors to a synthetic repetitive locus represented by the tetO operator array. Transcriptional silencing of the array was mediated by canonical constitutive heterochromatin. On the other hand, post-transcriptional silencing resembled quelling, but occurred normally in the absence of RAD51 and RAD52, two principal recombination factors. Therefore, this process was named recombination-independent quelling (RIQ). Initiation of RIQ and heterochromatin was associated with perturbed chromatin, and both processes required SAD-6, a SWI/SNF chromatin remodeler orthologous to ATRX. Without SAD-6, and while still occupied by transcription factors, the locus featured an extreme state that was associated with strong nucleosome depletion and high MNase sensitivity, however, this state was unable to promote silencing. We also found that native AT-rich DNA (which comprises relics of transposable elements in N. crassa) could undergo RIQ according to the same principles. These results suggested a general model in which the de novo silencing can be induced by the active remodeling of a perturbed (nucleosome-depleted) chromatin state.\nEasyGrid: A versatile platform for automated cryo-EM sample preparation and quality control\nAuthors: Gemin, O.; Armijo, V.; Hons, M.; Bissardon, C.; Linares, R.; Bowler, M. W.; Wolff, G.; Kovalev, K.; Babenko, A.; Salo, V. T.; Schneider, S.; Rossi, C.; Lecomte, L.; Deckers, T.; Lauzier, K.; Janocha, R.; Felisaz, F.; Sinoir, J.; Galej, W. P.; Mahamid, J.; Muller, C. W.; Eustermann, S.; Mattei, S.; Cipriani, F.; Papp, G.\nScore: 4.6, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576170\nImaging biological macromolecules in their native state with single-particle cryo-electron microscopy (cryo-EM) or in situ cryo-electron tomography (cryo-ET) requires optimized approaches for the preparation and vitrification of biological samples. Here, we describe EasyGrid, a versatile technology enabling systematic, tailored and advanced sample preparation for cellular and structural biology. This automated, standalone platform combines in-line plasma treatment, microfluidic dispensing, blot-less sample spreading, jet-based vitrification and on-the-fly grid quality control using light interferometry to streamline cryo-EM sample optimization. With EasyGrid, we optimized grid preparation for different purified macromolecular complexes and subsequently determined their structure with cryo- EM. We also demonstrated how the platform allows better vitrification of large, mammalian cells compared to standard plunge-freezing. Automated sample preparation with EasyGrid establishes an advanced, high-throughput platform for both single-particle cryo-EM and cellular cryo-ET sample preparation.\nGenome - nuclear lamina interactions are multivalent and cooperative\nAuthors: Dauban, L.; Eder, M.; de Haas, M.; Franceschini-Santos, V. H.; Yanez-Cuna, J. O.; van Schaik, T.; Leemans, C.; Rademaker, K.; Martinez-Ara, M.; Martinovic, M.; de Wit, E.; van Steensel, B.\nScore: 16.4, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574825\nLamina-associated domains (LADs) are megabase-sized genomic regions that interact with the nuclear lamina (NL). It is not yet understood how their interactions with the NL are encoded in their DNA. Here, we designed an efficient LAD \"scrambling\" approach, based on transposon-mediated local hopping of loxP recombination sites, to generate series of large deletions and inversions that span LADs and flanking sequences. Mapping of NL interactions in these rearrangements revealed that a single LAD contacts the NL through multiple regions that act cooperatively or redundantly; some have more affinity for the NL than others and can pull neighbouring sequences to the NL. Genes drawn towards the NL showed often, but not always, reduced expression and increased H3K9me3 levels. Furthermore, neighbouring LADs can cooperatively interact with the NL when placed close enough to each other. These results elucidate principles that govern the positioning of megabase-sized genomic regions inside the cell nucleus. HighlightsO_LIEfficient generation of LAD rearrangements by transposon hopping of loxP elements. C_LIO_LINL interactions are multivalent, with some subregions being more potent tethers than others. C_LIO_LILADs can cooperate to promote their association to the NL. C_LIO_LIChanges in NL association are often accompanied by changes in gene activity and H3K9me3. C_LI\n",
  "wordCount" : "2315",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575741" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575741">
        <p class="paperTitle">Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.</p>
        <p class="info">Score: 321.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575741' target='https://doi.org/10.1101/2024.01.15.575741'> 10.1101/2024.01.15.575741</a></p>
        <p class="abstract">SUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574476">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574476" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574476">
        <p class="paperTitle">A universal molecular mechanism driving aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574476" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574476" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, W.; Zheng, J.; Xiao, Y.; Ju, L.; Chen, F.; Fu, J.; Jiang, H.; Zhang, Y.</p>
        <p class="info">Score: 57.5, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574476' target='https://doi.org/10.1101/2024.01.06.574476'> 10.1101/2024.01.06.574476</a></p>
        <p class="abstract">How cell replication ultimately results in aging and the Hayflick limit are not fully understood. Here we show that clock-like accumulation of DNA G-quadruplexes (G4s) throughout cell replication drives conserved aging mechanisms. G4 stimulates transcription-replication interactions to delay genome replication and impairs DNA re-methylation and histone modification recovery, leading to loss of heterochromatin. This creates a more permissive local environment for G4 formation in subsequent generations. As a result, G4s gradually accumulate on promoters throughout mitosis, driving clock-like DNA hypomethylation and chromatin opening. In patients and in vitro models, loss-of-function mutations in the G4-resolving enzymes WRN, BLM and ERCC8 accelerate the erosion of the epigenomic landscape around G4. G4-driven epigenomic aging is strongly correlated with biological age and is conserved in yeast, nematodes, insects, fish, rodents, and humans. Our results revealed a universal molecular mechanism of aging and provided mechanistic insight into how G-quadruplex processor mutations drive premature aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564600">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564600" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564600">
        <p class="paperTitle">The mitochondrial mRNA stabilizing protein, SLIRP, regulates skeletal muscle mitochondrial structure and respiration by exercise-recoverable mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564600" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564600" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pham, T. C. P.; Raun, S. H.; Havula, E.; Henriquez-Olguin, C.; Rubalcava-Gracia, D.; Frank, E.; Fritzen, A. M.; Jannig, P. R.; Andersen, N. R.; Kruse, R.; Ali, M.; Halling, J. F.; Ringholm, S.; Needham, E. J.; Hansen, S.; Lemminger, A. K.; Schjerling, P.; Petersen, M. H.; de Almeida, M. E.; Jensen, T. E.; Kiens, B.; Hostrup, M.; Larsen, S.; Ortenblad, N.; Hojlund, K.; Kjaer, M.; Ruas, J. L.; Trifunovic, A.; Wojtaszewski, J. F. P.; Nielsen, J.; Qvortrup, K.; Pilegaard, H.; Richter, E. A.; Sylow, L.</p>
        <p class="info">Score: 16.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564600' target='https://doi.org/10.1101/2023.10.30.564600'> 10.1101/2023.10.30.564600</a></p>
        <p class="abstract">Summary and graphical abstractDecline in mitochondrial function associates with decreased muscle mass and strength in multiple conditions, including sarcopenia and type 2 diabetes. Optimal treatment could include improving mitochondrial function, however, there are limited and equivocal data regarding the molecular cues controlling muscle mitochondrial plasticity. Here we uncover the mitochondrial-mRNA-stabilizing protein SLIRP, in complex with LRPPRC, as a PGC-1 target that regulates mitochondrial structure, respiration, and mitochondrially-encoded-mRNA pools in skeletal muscle. Exercise training effectively counteracted mitochondrial defects induced by loss of LRPPRC/SLIRP, despite sustained low mitochondrially-encoded-mRNA pools, via increased mitoribosome translation capacity. In humans, exercise training robustly increased muscle SLIRP and LRPPRC protein content across exercise modalities and sexes, yet this increase was less prominent in subjects with type 2 diabetes. Our work identifies a mechanism of post-transcriptional mitochondrial regulation in skeletal muscle through mitochondrial mRNA stabilization. It emphasizes exercise as an effective approach to alleviate mitochondrial defects by possibly increasing mitoribosome capacity.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=142 SRC=&#34;FIGDIR/small/564600v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (61K):
org.highwire.dtl.DTLVardef@88b3e7org.highwire.dtl.DTLVardef@db21bdorg.highwire.dtl.DTLVardef@841960org.highwire.dtl.DTLVardef@9e16ec_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575832" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575832">
        <p class="paperTitle">Heterodimerization of Endolysin Isoforms During Bacterial Infection by Staphylococcal Phage {varphi}2638A</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zinsli, L. V.; Sobieraj, A. M.; Ernst, P.; Meile, S.; Kilcher, S.; Iseli, C.; Keller, A.; Dreier, B.; Mittl, P. R. E.; Plueckthun, A.; Loessner, M. J.; Schmelcher, M.; Dunne, M.</p>
        <p class="info">Score: 11.0, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575832' target='https://doi.org/10.1101/2024.01.16.575832'> 10.1101/2024.01.16.575832</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWBacteriophage endolysins targeting Gram-positive bacteria typically feature a modular architecture of one or more enzymatically active domains (EADs) and cell wall binding domains (CBDs). Several endolysins also feature internal translational start sites (iTSSs) that produce short variant (SV) isoforms alongside the full-length (FL) endolysin. While the lytic activity of endolysins and their isoforms has been extensively studied as exogenous agents, the purpose behind producing the SV isoform during the phage infection cycle remains to be explored. In this study, we used staphylococcal phage {varphi}2638A as a model to determine the interplay between its full-length endolysin, Ply2638A, and its SV isoform during phage infection. X-ray crystallography structures and AlphaFold-generated models enabled elucidation of individual functions of the M23 endopeptidase, central amidase, and SH3b domains of Ply2638A. Production of the SV isoform (amidase and SH3b) was confirmed during phage infection and shown to form a heterodimer complex with Ply2638A via inter-amidase domain interactions. Using genetically engineered phage variants, we show that production of both isoforms provides an advantage during phage infection as phages producing only one isoform presented impaired lytic activity, which could be partly restored through recombinant protein complementation of the missing isoform. Importantly, when applied as an antimicrobial protein against Staphylococcus aureus in culture, the activity of Ply2638A remained constant regardless of SV isoform complementation. Drawing from our findings, we propose that SV isoform production provides its biological advantage upon endolysin entry to the periplasmic space to ensure optimal peptidoglycan degradation prior to cell wall lysis and progeny phage release.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575070" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575070">
        <p class="paperTitle">Brain and cancer associated binding domain mutations provide insight into CTCF&#39;s relationship with chromatin and its ability to act as a chromatin organizer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Do, C.; Jiang, G.; Katsifis, C. C.; Narducci, D. N.; Yang, J.; Cova, G.; Sakellaropoulos, T.; Vidal, R.; Lhoumaud, P.; Fara Regis, F.; Kakabadze, N.; Nora, E. P.; Noyes, M.; Chen, X.; Hansen, A. S.; Skok, J. A.</p>
        <p class="info">Score: 18.2, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575070' target='https://doi.org/10.1101/2024.01.11.575070'> 10.1101/2024.01.11.575070</a></p>
        <p class="abstract">Although only a fraction of CTCF motifs are bound in any cell type, and few occupied sites overlap cohesin, the mechanisms underlying cell-type specific attachment and ability to function as a chromatin organizer remain unknown. To investigate the relationship between CTCF and chromatin we applied a combination of imaging, structural and molecular approaches, using a series of brain and cancer associated CTCF mutations that act as CTCF perturbations. We demonstrate that binding and the functional impact of WT and mutant CTCF depend not only on the unique binding properties of each protein, but also on the genomic context of bound sites and enrichment of motifs for expressed TFs abutting these sites. Our studies also highlight the reciprocal relationship between CTCF and chromatin, demonstrating that the unique binding properties of WT and mutant proteins have a distinct impact on accessibility, TF binding, cohesin overlap, chromatin interactivity and gene expression programs, providing insight into their cancer and brain related effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575390" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575390">
        <p class="paperTitle">KAT5 Acetylates Human Chromatin Protein PC4 to promote DNA Repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: kundu, T. K.; Agrawal, A.; Sikder, S.; Singh, S.; Peraman, R.; Ravichandran, V.</p>
        <p class="info">Score: 6.2, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575390' target='https://doi.org/10.1101/2024.01.12.575390'> 10.1101/2024.01.12.575390</a></p>
        <p class="abstract">Human positive coactivator 4 (PC4), is a highly abundant non-histone chromatin protein involved in diverse cellular processes, including transcription regulation, genome organization, DNA repair, etc. The majority of PC4 exists in a phosphorylated state in cells, which impinges its acetylation by p300 and thereby inhibits its double-stranded DNA binding ability and transcriptional co-activator function. Recently, we have shown that PC4 interacts with linker histone H1 in its phosphorylated state, and this interaction is important for PC4- mediated chromatin compaction. PC4 was also found to be an activator of non-homologous end joining and DSB repair activity. Knockdown of PC4 causes drastic decompaction and enhanced autophagy in the cells. Mechanistically, in the absence of PC4, the genome becomes highly vulnerable to DNA damage with an altered epigenetic landscape. Here, we report that other than p300, PC4 also gets acetylated by DNA repair facilitating lysine acetyltransferase KAT5 (Tip60), at K80 residue when the cells are subjected to DNA damage. The vulnerability of DNA in PC4 devoid cells could be substantially reduced by reintroducing wild-type PC4 to the cells but not the mutant PC4 (K80R PC4), defective in KAT5-mediated acetylation. Presumably, KAT5-mediated acetylation of PC4 at K80 residue facilitates the DNA repair machinery at the damage site and thus contributes to DNA damage repair, a process that could be of high significance both in cancer and in aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575898">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575898" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575898">
        <p class="paperTitle">Mobile-CRISPRi as a powerful tool for modulating Vibrio gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575898" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575898" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Geyman, L. J.; Tanner, M.; Rosario-Melendez, N.; Peters, J.; Mandel, M. J.; van Kessel, J. C.</p>
        <p class="info">Score: 5.6, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575898' target='https://doi.org/10.1101/2024.01.17.575898'> 10.1101/2024.01.17.575898</a></p>
        <p class="abstract">CRISPRi (Clustered Regularly Interspaced Palindromic Repeats interference) is a gene knockdown method that uses a deactivated Cas9 protein (dCas9) that binds a specific gene target locus dictated by an encoded guide RNA (sgRNA) to block transcription. Mobile-CRISPRi is a suite of modular vectors that enable CRISPRi knockdowns in diverse bacteria by integrating IPTG-inducible dcas9 and sgRNA genes into the genome using Tn7 transposition. Here, we show that the Mobile-CRISPRi system functions robustly and specifically in multiple Vibrio species: Vibrio cholerae, Vibrio fischeri, Vibrio vulnificus, Vibrio parahaemolyticus, and Vibrio campbellii. We demonstrate efficacy by targeting both essential and non-essential genes that function to produce defined, measurable phenotypes: bioluminescence, quorum sensing, cell division, and growth arrest. We anticipate that Mobile-CRISPRi will be used in Vibrio species to systematically probe gene function and essentiality in various behaviors and native environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575522" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575522">
        <p class="paperTitle">Chromatin remodeling initiates transcriptional and post-transcriptional silencing in Neurospora</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carlier, F.; Castro Ramirez, S.; Kilani, J.; Chehboub, S.; Gladyshev, E.</p>
        <p class="info">Score: 4.8, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575522' target='https://doi.org/10.1101/2024.01.15.575522'> 10.1101/2024.01.15.575522</a></p>
        <p class="abstract">In many eukaryotes, including mammals, transcriptional (heterochromatin-mediated) and post-transcriptional (RNAi-mediated) silencing of repetitive DNA can be initiated by poorly understood mechanisms independent of strong sequence-specific cues. We found that in the fungus Neurospora crassa both types of silencing were induced by the aberrant binding of transcription factors to a synthetic repetitive locus represented by the tetO operator array. Transcriptional silencing of the array was mediated by canonical constitutive heterochromatin. On the other hand, post-transcriptional silencing resembled quelling, but occurred normally in the absence of RAD51 and RAD52, two principal recombination factors. Therefore, this process was named recombination-independent quelling (RIQ). Initiation of RIQ and heterochromatin was associated with perturbed chromatin, and both processes required SAD-6, a SWI/SNF chromatin remodeler orthologous to ATRX. Without SAD-6, and while still occupied by transcription factors, the locus featured an extreme state that was associated with strong nucleosome depletion and high MNase sensitivity, however, this state was unable to promote silencing. We also found that native AT-rich DNA (which comprises relics of transposable elements in N. crassa) could undergo RIQ according to the same principles. These results suggested a general model in which the de novo silencing can be induced by the active remodeling of a perturbed (nucleosome-depleted) chromatin state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576170">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576170" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576170">
        <p class="paperTitle">EasyGrid: A versatile platform for automated cryo-EM sample preparation and quality control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576170" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576170" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gemin, O.; Armijo, V.; Hons, M.; Bissardon, C.; Linares, R.; Bowler, M. W.; Wolff, G.; Kovalev, K.; Babenko, A.; Salo, V. T.; Schneider, S.; Rossi, C.; Lecomte, L.; Deckers, T.; Lauzier, K.; Janocha, R.; Felisaz, F.; Sinoir, J.; Galej, W. P.; Mahamid, J.; Muller, C. W.; Eustermann, S.; Mattei, S.; Cipriani, F.; Papp, G.</p>
        <p class="info">Score: 4.6, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576170' target='https://doi.org/10.1101/2024.01.18.576170'> 10.1101/2024.01.18.576170</a></p>
        <p class="abstract">Imaging biological macromolecules in their native state with single-particle cryo-electron microscopy (cryo-EM) or in situ cryo-electron tomography (cryo-ET) requires optimized approaches for the preparation and vitrification of biological samples. Here, we describe EasyGrid, a versatile technology enabling systematic, tailored and advanced sample preparation for cellular and structural biology. This automated, standalone platform combines in-line plasma treatment, microfluidic dispensing, blot-less sample spreading, jet-based vitrification and on-the-fly grid quality control using light interferometry to streamline cryo-EM sample optimization. With EasyGrid, we optimized grid preparation for different purified macromolecular complexes and subsequently determined their structure with cryo- EM. We also demonstrated how the platform allows better vitrification of large, mammalian cells compared to standard plunge-freezing. Automated sample preparation with EasyGrid establishes an advanced, high-throughput platform for both single-particle cryo-EM and cellular cryo-ET sample preparation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574825">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574825" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574825">
        <p class="paperTitle">Genome - nuclear lamina interactions are multivalent and cooperative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574825" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574825" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dauban, L.; Eder, M.; de Haas, M.; Franceschini-Santos, V. H.; Yanez-Cuna, J. O.; van Schaik, T.; Leemans, C.; Rademaker, K.; Martinez-Ara, M.; Martinovic, M.; de Wit, E.; van Steensel, B.</p>
        <p class="info">Score: 16.4, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574825' target='https://doi.org/10.1101/2024.01.10.574825'> 10.1101/2024.01.10.574825</a></p>
        <p class="abstract">Lamina-associated domains (LADs) are megabase-sized genomic regions that interact with the nuclear lamina (NL). It is not yet understood how their interactions with the NL are encoded in their DNA. Here, we designed an efficient LAD &#34;scrambling&#34; approach, based on transposon-mediated local hopping of loxP recombination sites, to generate series of large deletions and inversions that span LADs and flanking sequences. Mapping of NL interactions in these rearrangements revealed that a single LAD contacts the NL through multiple regions that act cooperatively or redundantly; some have more affinity for the NL than others and can pull neighbouring sequences to the NL. Genes drawn towards the NL showed often, but not always, reduced expression and increased H3K9me3 levels. Furthermore, neighbouring LADs can cooperatively interact with the NL when placed close enough to each other. These results elucidate principles that govern the positioning of megabase-sized genomic regions inside the cell nucleus.

HighlightsO_LIEfficient generation of LAD rearrangements by transposon hopping of loxP elements.
C_LIO_LINL interactions are multivalent, with some subregions being more potent tethers than others.
C_LIO_LILADs can cooperate to promote their association to the NL.
C_LIO_LIChanges in NL association are often accompanied by changes in gene activity and H3K9me3.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
