<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-confidence 3D template matching for cryo-electron tomography
Authors: Cruz-Leon, S.; Majtner, T.; Hoffmann, P. C.; Kreysing, J. P.; Tuijtel, M. W.; Schaefer, S. L.; Geissler, K.; Beck, M.; Turonova, B.; Hummer, G.
Score: 40.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556310
Cryo-electron tomography (CryoET) resolves individual macromolecules inside living cells. However, the complex composition and high density of cells challenge the faithful identification of features in tomograms. Here, we capitalize on recent advances in electron tomography and demonstrate that 3D template matching (TM) localizes a wide range of structures inside crowded eukaryotic cells with confidence 10 to 100-fold above the noise level.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="High-confidence 3D template matching for cryo-electron tomography
Authors: Cruz-Leon, S.; Majtner, T.; Hoffmann, P. C.; Kreysing, J. P.; Tuijtel, M. W.; Schaefer, S. L.; Geissler, K.; Beck, M.; Turonova, B.; Hummer, G.
Score: 40.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556310
Cryo-electron tomography (CryoET) resolves individual macromolecules inside living cells. However, the complex composition and high density of cells challenge the faithful identification of features in tomograms. Here, we capitalize on recent advances in electron tomography and demonstrate that 3D template matching (TM) localizes a wide range of structures inside crowded eukaryotic cells with confidence 10 to 100-fold above the noise level." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="High-confidence 3D template matching for cryo-electron tomography
Authors: Cruz-Leon, S.; Majtner, T.; Hoffmann, P. C.; Kreysing, J. P.; Tuijtel, M. W.; Schaefer, S. L.; Geissler, K.; Beck, M.; Turonova, B.; Hummer, G.
Score: 40.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556310
Cryo-electron tomography (CryoET) resolves individual macromolecules inside living cells. However, the complex composition and high density of cells challenge the faithful identification of features in tomograms. Here, we capitalize on recent advances in electron tomography and demonstrate that 3D template matching (TM) localizes a wide range of structures inside crowded eukaryotic cells with confidence 10 to 100-fold above the noise level."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "High-confidence 3D template matching for cryo-electron tomography\nAuthors: Cruz-Leon, S.; Majtner, T.; Hoffmann, P. C.; Kreysing, J. P.; Tuijtel, M. W.; Schaefer, S. L.; Geissler, K.; Beck, M.; Turonova, B.; Hummer, G.\nScore: 40.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556310\nCryo-electron tomography (CryoET) resolves individual macromolecules inside living cells. However, the complex composition and high density of cells challenge the faithful identification of features in tomograms. Here, we capitalize on recent advances in electron tomography and demonstrate that 3D template matching (TM) localizes a wide range of structures inside crowded eukaryotic cells with confidence 10 to 100-fold above the noise level.",
  "keywords": [
    
  ],
  "articleBody": " High-confidence 3D template matching for cryo-electron tomography\nAuthors: Cruz-Leon, S.; Majtner, T.; Hoffmann, P. C.; Kreysing, J. P.; Tuijtel, M. W.; Schaefer, S. L.; Geissler, K.; Beck, M.; Turonova, B.; Hummer, G.\nScore: 40.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556310\nCryo-electron tomography (CryoET) resolves individual macromolecules inside living cells. However, the complex composition and high density of cells challenge the faithful identification of features in tomograms. Here, we capitalize on recent advances in electron tomography and demonstrate that 3D template matching (TM) localizes a wide range of structures inside crowded eukaryotic cells with confidence 10 to 100-fold above the noise level. We establish a TM pipeline with systematically tuned parameters for automated, objective and comprehensive feature identification. High-fidelity and high-confidence localizations of nuclear pore complexes, vaults, ribosomes, proteasomes, lipid membranes and microtubules, and individual subunits, demonstrate that TM is generic. We resolve [~]100-kDa proteins, connect the functional states of complexes to their cellular localization, and capture vaults carrying ribosomal cargo in situ. By capturing individual molecular events inside living cells with defined statistical confidence, high-confidence TM greatly speeds up the CryoET workflow and sets the stage for visual proteomics.\nA cryo-ET study of ciliary rootlet organization\nAuthors: van Hoorn, C.; Carter, A. P.\nScore: 33.3, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556114\nCiliary rootlets are striated bundles of filaments that connect the base of cilia to internal cellular structures. Rootlets are critical for the sensory and motile functions of cilia. However, the mechanisms underlying these functions remain unknown, in part due to a lack of structural information of rootlet organization. In this study, we obtain 3D reconstructions of membrane-associated and purified rootlets using cryo-electron tomography (cryo-ET). We show that flexible protrusions on the rootlet surface, which emanate from the cross-striations, connect to intracellular membranes. In purified rootlets, the striations were classified into amorphous (A)-bands, associated with accumulations on the rootlet surface, and discrete (D)-bands corresponding to punctate lines of density that run through the rootlet. These striations connect a flexible network of longitudinal filaments. Subtomogram averaging suggests the filaments consist of two intertwined coiled coils. The rootlets filamentous architecture, with frequent membrane-connecting cross-striations, lends itself well for anchoring large membranes in the cell.\nTardigrades dramatically upregulate DNA repair pathway genes in response to ionizing radiation\nAuthors: Clark-Hachtel, C. M.; Hibshman, J. D.; De Buysscher, T.; Goldstein, B.\nScore: 26.2, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556677\nTardigrades can survive remarkable doses of ionizing radiation, up to about 1000 times the lethal dose for humans. How they do so is incompletely understood. We found that the tardigrade Hypsibius exemplaris suffers DNA damage upon gamma irradiation, but damage is repaired. We show that tardigrades have a specific and robust response to ionizing radiation: irradiation induces a rapid, dramatic upregulation of many DNA repair genes. By expressing tardigrade genes in bacteria, we validate that increased expression of some repair genes can suffice to increase radiation tolerance. We show that at least one such gene is necessary for tardigrade radiation tolerance. Tardigrades' ability to sense ionizing radiation and massively upregulate specific DNA repair pathway genes may represent an evolved solution for maintaining DNA integrity.\nThe Fanconi anemia core complex promotes CtIP-dependent end-resection to drive homologous recombination at DNA double-strand breaks\nAuthors: van de Kooij, B.; van der Wal, F. J.; Rother, M. B.; Creixell, P.; Stout, M.; Wiegant, W.; Joughin, B. A.; Vornberger, J.; van Vugt, M. A. T. M.; Altmeyer, M.; Yaffe, M. B.; van Attikum, H.\nScore: 16.2, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556391\nHomologous Recombination (HR) is a high-fidelity repair mechanism of DNA Double-Strand Breaks (DSBs), which are induced by irradiation, genotoxic chemicals or physiological DNA damaging processes. DSBs are also generated as intermediates during the repair of interstrand crosslinks (ICLs). In this context, the Fanconi anemia (FA) core complex, which is effectively recruited to ICLs, promotes HR-mediated DSB-repair. However, whether the FA core complex also promotes HR at ICL-independent DSBs remains controversial. Here, we identified the FA core complex members FANCL and Ube2T as HR-promoting factors in a CRISPR/Cas9-based screen with cells carrying the DSB-repair reporter DSB-Spectrum. Using isogenic cell-line models, we validated the HR-function of FANCL and Ube2T, and demonstrated a similar function for their ubiquitination-substrate FANCD2. We further show that FANCL and Ube2T are directly recruited to DSBs and are required for the accumulation of FANCD2 at these break sites. Mechanistically, we demonstrate that FANCL ubiquitin ligase activity is required for the accumulation of the nuclease CtIP at DSBs, and consequently for optimal end-resection and Rad51 loading. CtIP overexpression rescues HR in FANCL-deficient cells, validating that FANCL primarily regulates HR by promoting CtIP recruitment. Together, these data demonstrate that the FA core complex and FANCD2 have a dual genome maintenance function by promoting repair of DSBs as well as the repair of ICLs.\nDysregulation of extracellular vesicle protein cargo in female ME/CFS cases and sedentary controls in response to maximal exercise\nAuthors: Giloteaux, L.; Glass, K. A.; Germain, A.; Zhang, S.; Hanson, M. R.\nScore: 91.6, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555033\nIn healthy individuals, physical exercise improves cardiovascular health and muscle strength, alleviates fatigue, and reduces risk of chronic diseases. Although exercise is suggested as a lifestyle intervention to manage various chronic illnesses, it negatively affects people with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), who suffer from exercise intolerance. We hypothesized that altered extracellular vesicle (EV) signaling in ME/CFS patients after an exercise challenge may contribute to their prolonged and exacerbated negative response to exertion (post-exertional malaise). EVs were isolated by size exclusion chromatography from the plasma of 18 female ME/CFS patients and 17 age- and BMI-matched female sedentary controls at three time points: before, 15 minutes, and 24 hours after a maximal cardiopulmonary exercise test. EVs were characterized using nanoparticle tracking analysis and their protein cargo was quantified using Tandem Mass Tag-based (TMT) proteomics. The results show that exercise affects the EV proteome in ME/CFS patients differently than in healthy individuals and that changes in EV proteins after exercise are strongly correlated with symptom severity in ME/CFS. Differentially abundant proteins in ME/CFS patients vs. controls were involved in many pathways and systems, including coagulation processes, muscle contraction (both smooth and skeletal muscle), cytoskeletal proteins, the immune system, and brain signaling.\nCoSMIC - A hybrid approach for large-scale, high-resolution microbial profiling of novel niches\nAuthors: Knafo, M.; Rezenman, S.; Idan, T.; Elgart, M.; Dagan, S.; Reich, Z.; Kapon, R.; Sade, D.; Shental, N.\nScore: 7.6, Published: 2023-09-03 DOI: 10.1101/2023.09.03.556087\nMicrobiome studies have become integral to a wide range of biological research from ecology to oncology, mainly relying on short-read sequencing of variable regions along the 16S rRNA gene. However, the comprehensiveness of 16S rRNA studies has been persistently challenged due to a lack of primer universality and primer biases, causing differences between a studys results and the underlying bacterial community. Moreover, relying on a small part of the gene often provides low phylogenetic resolution, hampering downstream taxonomy-based analysis and the ability to harmonize results from studies performed using different variable regions. Here, we introduce a framework called Comprehensive Small Ribosomal Subunit Mapping and Identification of Communities (CoSMIC), effectively addressing these challenges. CoSMIC begins with long-read full-length sequencing of the 16S rRNA gene, using Locked Nucleic Acid primers over pooled samples. This step augments the Small Subunit (SSU) reference database with novel niche-specific SSUs. Subsequently, CoSMIC optimizes a set of primer pairs targeting multiple non-consecutive variable regions along the gene, followed by standard short-read sequencing of each sample. Data from different regions are seamlessly integrated using the SMURF framework, thus alleviating primer-based biases and providing extremely high phylogenetic resolution. We evaluated CoSMIC across plant, root, soil and marine sponge samples, yielding higher profiling accuracy and unprecedented phylogenetic resolution compared to standard methods while detecting [~]40,000 novel SSUs. CoSMIC provides researchers with a robust, customizable, and cost-effective framework addressing the challenges of 16S rRNA sequencing, especially in underexplored habitats, paving the way for data harmonization across microbiome studies.\nCohesin still drives homologous recombination repair of DNA double-strand breaks in late mitosis.\nAuthors: Ayra Plasencia, J.; Symington, L. S.; Machin, F.\nScore: 7.2, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556828\nThe cohesin complex maintains sister chromatid cohesion from S phase to anaphase onset. Cohesin also plays roles in chromosome structure and DNA repair. At anaphase onset, the cohesin subunit Scc1 is cleaved to allow segregation in an orderly manner, although some residual cohesin subunits remain to maintain chromosome structure. Efficient DNA double-strand break (DSB) repair by homologous recombination (HR) with the sister chromatid also depends on cohesin. Here, we have tested whether residual cohesin is important during DSB repair in anaphase/telophase (late mitosis). Using the well-established MAT switching system, we first show that HR is molecularly functional in late mitosis, and then show that cohesin is required for its efficiency. During DSBs in late mitosis, the segregated sister loci get closer and have episodes of coalescence, which may favour repair by HR. Here, we also show that these cytological phenotypes also depend on cohesin. Finally, full-length Scc1 returns after DSBs in late mitosis, suggesting that de novo widespread cohesin complexes can support these retrograde behaviours.\nAlternative splicing coupled to nonsense-mediated decay coordinates downregulation of non-neuronal genes in developing neurons\nAuthors: Zhuravskaya, A.; Yap, K.; Hamid, F.; Makeyev, E. V.\nScore: 6.3, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556212\nThe functional coupling between alternative pre-mRNA splicing (AS) and the mRNA quality control mechanism called nonsense-mediated decay (NMD) can modulate transcript abundance. Previous studies have identified several examples of such a regulation in developing neurons. However, the systems-level effects of AS-NMD in this context are poorly understood. We developed an R package, factR2, which offers a comprehensive suite of AS-NMD analysis functions. Using this tool, we conducted a longitudinal analysis of gene expression in pluripotent stem cells undergoing induced neuronal differentiation. Our analysis uncovered hundreds of AS-NMD events with significant potential to regulate gene expression. Notably, this regulation was significantly overrepresented in specific functional groups of developmentally downregulated genes. Particularly strong association with gene downregulation was detected for alternative cassette exons stimulating NMD (NS-CEs) upon their inclusion into mature mRNA. By combining bioinformatics analyses with CRISPR/Cas9 genome editing and other experimental approaches we show that NS-CEs regulated by the RNA-binding protein PTBP1 dampen the expression of their genes in developing neurons. We also provide evidence that the NS-CE activity is temporally coordinated with NMD-independent gene repression mechanisms. Our study provides an accessible workflow for the discovery and prioritization of AS-NMD targets. It further argues that the AS-NMD pathway plays a widespread role in developing neurons by facilitating the downregulation of functionally related non-neuronal genes.\nThe characteristics of CTCF binding sequences contribute to enhancer blocking activity\nAuthors: Tsang, F. H.; Stolper, R. J.; Hanifi, M.; Cornell, L. J.; Francis, H. S.; Davies, B.; Higgs, D. R.; Kassouf, M. T.\nScore: 6.0, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556325\nWhile the elements encoding enhancers and promoters have been relatively well studied, the full spectrum of insulator elements which bind the CCCTC binding factor (CTCF), is relatively poorly characterised. This is partly due to the genomic context of CTCF sites greatly influencing their roles and activity. Here we have developed an experimental system to determine the ability of consistently sized, individual CTCF elements to interpose between enhancers and promoters and thereby reduce gene expression during differentiation. Importantly, each element is tested in the identical location thereby minimising the effect of genomic context. We found no correlation between the ability of CTCF elements to block enhancer-promoter activity with the amount of CTCF or cohesin bound at the natural genomic sites of these elements; the degree of evolutionary conservation; or their resemblance to the consensus core sequences. Nevertheless, we have shown that the strongest enhancer-promoter blockers include a previously described bound element lying upstream of the CTCF core motif. In addition, we found other uncharacterised DNaseI footprints located close to the core motif that may affect function. We have developed an assay of CTCF sequences which will enable researchers to sub-classify CTCF elements in a uniform and unbiased way.\nA Toxoplasma gondii putative arginine transporter localizes to the plant-like vacuolar compartment and controls parasite extracellular survival and stage differentiation\nAuthors: Piro, F.; Masci, S.; Kannan, G.; Focaia, R.; Schultz, T. L.; Carruthers, V. B.; Di Cristina, M.\nScore: 6.3, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555807\nToxoplasma gondii is a protozoan parasite that infects a broad spectrum of hosts and can colonize many organs and cell types. The ability to reside within a wide range of different niches requires substantial adaptability to diverse microenvironments. Very little is known about how this parasite senses various milieus and adapts its metabolism to survive, replicate during the acute stage, and then differentiate to the chronic stage. Most eukaryotes, from yeast to mammals, rely on a nutrient sensing machinery involving the TORC complex as master regulator of cell growth and cell cycle progression. The lysosome functions as a signaling hub where TORC complex assembles and is activated by transceptors, which both sense and transport amino acids, including the arginine transceptor SLC38A9. While most of the TORC components are lost in T. gondii, indicating the evolution of a distinct nutrient sensing mechanism, the parasites lysosomal plant-like vacuolar compartment (PLVAC) may still serve as a sensory platform for controlling parasite growth and differentiation. Using SLC38A9 to query the T. gondii proteome, we identified four putative amino acid transporters, termed TgAAT1-4, that structurally resemble the SLC38A9 arginine transceptor. Assessing their expression and sub-cellular localization, we found that one of them, TgAAT1, localized to the PLVAC and is necessary for normal parasite extracellular survival and bradyzoite differentiation. Moreover, we show that TgAAT1 is involved in the PLVAC efflux of arginine, an amino acid playing a key role in T. gondii differentiation, further supporting the hypothesis that TgAAT1 might play a role in nutrient sensing. IMPORTANCET. gondii is a highly successful parasite infecting a broad range of warm-blood organisms including about one third of all humans. Although Toxoplasma infections rarely result in symptomatic disease in individuals with a healthy immune system, the incredibly high number of persons infected along with the risk of severe infection in immunocompromised patients and the potential link of chronic infection to mental disorders make this infection a significant public health concern. As a result, there is a pressing need for new treatment approaches that are both effective and well-tolerated. The limitations in understanding how Toxoplasma gondii manages its metabolism to adapt to changing environments and triggers its transformation into bradyzoites have hindered the discovery of vulnerabilities in its metabolic pathways or nutrient acquisition mechanisms to identify new therapeutic targets. In this work, we have shown that the lysosome-like organelle PLVAC, acting through the putative arginine transporter TgAAT1, plays a pivotal role in regulating the parasites extracellular survival and differentiation into bradyzoites.\n",
  "wordCount" : "2435",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556310">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556310" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556310">
        <p class="paperTitle">High-confidence 3D template matching for cryo-electron tomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556310" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556310" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cruz-Leon, S.; Majtner, T.; Hoffmann, P. C.; Kreysing, J. P.; Tuijtel, M. W.; Schaefer, S. L.; Geissler, K.; Beck, M.; Turonova, B.; Hummer, G.</p>
        <p class="info">Score: 40.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556310' target='https://doi.org/10.1101/2023.09.05.556310'> 10.1101/2023.09.05.556310</a></p>
        <p class="abstract">Cryo-electron tomography (CryoET) resolves individual macromolecules inside living cells. However, the complex composition and high density of cells challenge the faithful identification of features in tomograms. Here, we capitalize on recent advances in electron tomography and demonstrate that 3D template matching (TM) localizes a wide range of structures inside crowded eukaryotic cells with confidence 10 to 100-fold above the noise level. We establish a TM pipeline with systematically tuned parameters for automated, objective and comprehensive feature identification. High-fidelity and high-confidence localizations of nuclear pore complexes, vaults, ribosomes, proteasomes, lipid membranes and microtubules, and individual subunits, demonstrate that TM is generic. We resolve [~]100-kDa proteins, connect the functional states of complexes to their cellular localization, and capture vaults carrying ribosomal cargo in situ. By capturing individual molecular events inside living cells with defined statistical confidence, high-confidence TM greatly speeds up the CryoET workflow and sets the stage for visual proteomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556114">
        <p class="paperTitle">A cryo-ET study of ciliary rootlet organization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Hoorn, C.; Carter, A. P.</p>
        <p class="info">Score: 33.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556114' target='https://doi.org/10.1101/2023.09.03.556114'> 10.1101/2023.09.03.556114</a></p>
        <p class="abstract">Ciliary rootlets are striated bundles of filaments that connect the base of cilia to internal cellular structures. Rootlets are critical for the sensory and motile functions of cilia. However, the mechanisms underlying these functions remain unknown, in part due to a lack of structural information of rootlet organization. In this study, we obtain 3D reconstructions of membrane-associated and purified rootlets using cryo-electron tomography (cryo-ET). We show that flexible protrusions on the rootlet surface, which emanate from the cross-striations, connect to intracellular membranes. In purified rootlets, the striations were classified into amorphous (A)-bands, associated with accumulations on the rootlet surface, and discrete (D)-bands corresponding to punctate lines of density that run through the rootlet. These striations connect a flexible network of longitudinal filaments. Subtomogram averaging suggests the filaments consist of two intertwined coiled coils. The rootlets filamentous architecture, with frequent membrane-connecting cross-striations, lends itself well for anchoring large membranes in the cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556677">
        <p class="paperTitle">Tardigrades dramatically upregulate DNA repair pathway genes in response to ionizing radiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clark-Hachtel, C. M.; Hibshman, J. D.; De Buysscher, T.; Goldstein, B.</p>
        <p class="info">Score: 26.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556677' target='https://doi.org/10.1101/2023.09.07.556677'> 10.1101/2023.09.07.556677</a></p>
        <p class="abstract">Tardigrades can survive remarkable doses of ionizing radiation, up to about 1000 times the lethal dose for humans. How they do so is incompletely understood. We found that the tardigrade Hypsibius exemplaris suffers DNA damage upon gamma irradiation, but damage is repaired. We show that tardigrades have a specific and robust response to ionizing radiation: irradiation induces a rapid, dramatic upregulation of many DNA repair genes. By expressing tardigrade genes in bacteria, we validate that increased expression of some repair genes can suffice to increase radiation tolerance. We show that at least one such gene is necessary for tardigrade radiation tolerance. Tardigrades&#39; ability to sense ionizing radiation and massively upregulate specific DNA repair pathway genes may represent an evolved solution for maintaining DNA integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556391">
        <p class="paperTitle">The Fanconi anemia core complex promotes CtIP-dependent end-resection to drive homologous recombination at DNA double-strand breaks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van de Kooij, B.; van der Wal, F. J.; Rother, M. B.; Creixell, P.; Stout, M.; Wiegant, W.; Joughin, B. A.; Vornberger, J.; van Vugt, M. A. T. M.; Altmeyer, M.; Yaffe, M. B.; van Attikum, H.</p>
        <p class="info">Score: 16.2, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556391' target='https://doi.org/10.1101/2023.09.05.556391'> 10.1101/2023.09.05.556391</a></p>
        <p class="abstract">Homologous Recombination (HR) is a high-fidelity repair mechanism of DNA Double-Strand Breaks (DSBs), which are induced by irradiation, genotoxic chemicals or physiological DNA damaging processes. DSBs are also generated as intermediates during the repair of interstrand crosslinks (ICLs). In this context, the Fanconi anemia (FA) core complex, which is effectively recruited to ICLs, promotes HR-mediated DSB-repair. However, whether the FA core complex also promotes HR at ICL-independent DSBs remains controversial. Here, we identified the FA core complex members FANCL and Ube2T as HR-promoting factors in a CRISPR/Cas9-based screen with cells carrying the DSB-repair reporter DSB-Spectrum. Using isogenic cell-line models, we validated the HR-function of FANCL and Ube2T, and demonstrated a similar function for their ubiquitination-substrate FANCD2. We further show that FANCL and Ube2T are directly recruited to DSBs and are required for the accumulation of FANCD2 at these break sites. Mechanistically, we demonstrate that FANCL ubiquitin ligase activity is required for the accumulation of the nuclease CtIP at DSBs, and consequently for optimal end-resection and Rad51 loading. CtIP overexpression rescues HR in FANCL-deficient cells, validating that FANCL primarily regulates HR by promoting CtIP recruitment. Together, these data demonstrate that the FA core complex and FANCD2 have a dual genome maintenance function by promoting repair of DSBs as well as the repair of ICLs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555033">
        <p class="paperTitle">Dysregulation of extracellular vesicle protein cargo in female ME/CFS cases and sedentary controls in response to maximal exercise</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giloteaux, L.; Glass, K. A.; Germain, A.; Zhang, S.; Hanson, M. R.</p>
        <p class="info">Score: 91.6, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555033' target='https://doi.org/10.1101/2023.08.28.555033'> 10.1101/2023.08.28.555033</a></p>
        <p class="abstract">In healthy individuals, physical exercise improves cardiovascular health and muscle strength, alleviates fatigue, and reduces risk of chronic diseases. Although exercise is suggested as a lifestyle intervention to manage various chronic illnesses, it negatively affects people with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), who suffer from exercise intolerance. We hypothesized that altered extracellular vesicle (EV) signaling in ME/CFS patients after an exercise challenge may contribute to their prolonged and exacerbated negative response to exertion (post-exertional malaise). EVs were isolated by size exclusion chromatography from the plasma of 18 female ME/CFS patients and 17 age- and BMI-matched female sedentary controls at three time points: before, 15 minutes, and 24 hours after a maximal cardiopulmonary exercise test. EVs were characterized using nanoparticle tracking analysis and their protein cargo was quantified using Tandem Mass Tag-based (TMT) proteomics. The results show that exercise affects the EV proteome in ME/CFS patients differently than in healthy individuals and that changes in EV proteins after exercise are strongly correlated with symptom severity in ME/CFS. Differentially abundant proteins in ME/CFS patients vs. controls were involved in many pathways and systems, including coagulation processes, muscle contraction (both smooth and skeletal muscle), cytoskeletal proteins, the immune system, and brain signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556087" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556087">
        <p class="paperTitle">CoSMIC - A hybrid approach for large-scale, high-resolution microbial profiling of novel niches</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knafo, M.; Rezenman, S.; Idan, T.; Elgart, M.; Dagan, S.; Reich, Z.; Kapon, R.; Sade, D.; Shental, N.</p>
        <p class="info">Score: 7.6, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556087' target='https://doi.org/10.1101/2023.09.03.556087'> 10.1101/2023.09.03.556087</a></p>
        <p class="abstract">Microbiome studies have become integral to a wide range of biological research from ecology to oncology, mainly relying on short-read sequencing of variable regions along the 16S rRNA gene. However, the comprehensiveness of 16S rRNA studies has been persistently challenged due to a lack of primer universality and primer biases, causing differences between a studys results and the underlying bacterial community. Moreover, relying on a small part of the gene often provides low phylogenetic resolution, hampering downstream taxonomy-based analysis and the ability to harmonize results from studies performed using different variable regions.

Here, we introduce a framework called  Comprehensive Small Ribosomal Subunit Mapping and Identification of Communities (CoSMIC), effectively addressing these challenges. CoSMIC begins with long-read full-length sequencing of the 16S rRNA gene, using Locked Nucleic Acid primers over pooled samples. This step augments the Small Subunit (SSU) reference database with novel niche-specific SSUs. Subsequently, CoSMIC optimizes a set of primer pairs targeting multiple non-consecutive variable regions along the gene, followed by standard short-read sequencing of each sample. Data from different regions are seamlessly integrated using the SMURF framework, thus alleviating primer-based biases and providing extremely high phylogenetic resolution. We evaluated CoSMIC across plant, root, soil and marine sponge samples, yielding higher profiling accuracy and unprecedented phylogenetic resolution compared to standard methods while detecting [~]40,000 novel SSUs.

CoSMIC provides researchers with a robust, customizable, and cost-effective framework addressing the challenges of 16S rRNA sequencing, especially in underexplored habitats, paving the way for data harmonization across microbiome studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556828">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556828" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556828">
        <p class="paperTitle">Cohesin still drives homologous recombination repair of DNA double-strand breaks in late mitosis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556828" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556828" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ayra Plasencia, J.; Symington, L. S.; Machin, F.</p>
        <p class="info">Score: 7.2, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556828' target='https://doi.org/10.1101/2023.09.08.556828'> 10.1101/2023.09.08.556828</a></p>
        <p class="abstract">The cohesin complex maintains sister chromatid cohesion from S phase to anaphase onset. Cohesin also plays roles in chromosome structure and DNA repair. At anaphase onset, the cohesin subunit Scc1 is cleaved to allow segregation in an orderly manner, although some residual cohesin subunits remain to maintain chromosome structure. Efficient DNA double-strand break (DSB) repair by homologous recombination (HR) with the sister chromatid also depends on cohesin. Here, we have tested whether residual cohesin is important during DSB repair in anaphase/telophase (late mitosis). Using the well-established MAT switching system, we first show that HR is molecularly functional in late mitosis, and then show that cohesin is required for its efficiency. During DSBs in late mitosis, the segregated sister loci get closer and have episodes of coalescence, which may favour repair by HR. Here, we also show that these cytological phenotypes also depend on cohesin. Finally, full-length Scc1 returns after DSBs in late mitosis, suggesting that de novo widespread cohesin complexes can support these retrograde behaviours.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556212">
        <p class="paperTitle">Alternative splicing coupled to nonsense-mediated decay coordinates downregulation of non-neuronal genes in developing neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhuravskaya, A.; Yap, K.; Hamid, F.; Makeyev, E. V.</p>
        <p class="info">Score: 6.3, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556212' target='https://doi.org/10.1101/2023.09.04.556212'> 10.1101/2023.09.04.556212</a></p>
        <p class="abstract">The functional coupling between alternative pre-mRNA splicing (AS) and the mRNA quality control mechanism called nonsense-mediated decay (NMD) can modulate transcript abundance. Previous studies have identified several examples of such a regulation in developing neurons. However, the systems-level effects of AS-NMD in this context are poorly understood. We developed an R package, factR2, which offers a comprehensive suite of AS-NMD analysis functions. Using this tool, we conducted a longitudinal analysis of gene expression in pluripotent stem cells undergoing induced neuronal differentiation. Our analysis uncovered hundreds of AS-NMD events with significant potential to regulate gene expression. Notably, this regulation was significantly overrepresented in specific functional groups of developmentally downregulated genes. Particularly strong association with gene downregulation was detected for alternative cassette exons stimulating NMD (NS-CEs) upon their inclusion into mature mRNA. By combining bioinformatics analyses with CRISPR/Cas9 genome editing and other experimental approaches we show that NS-CEs regulated by the RNA-binding protein PTBP1 dampen the expression of their genes in developing neurons. We also provide evidence that the NS-CE activity is temporally coordinated with NMD-independent gene repression mechanisms. Our study provides an accessible workflow for the discovery and prioritization of AS-NMD targets. It further argues that the AS-NMD pathway plays a widespread role in developing neurons by facilitating the downregulation of functionally related non-neuronal genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556325">
        <p class="paperTitle">The characteristics of CTCF binding sequences contribute to enhancer blocking activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsang, F. H.; Stolper, R. J.; Hanifi, M.; Cornell, L. J.; Francis, H. S.; Davies, B.; Higgs, D. R.; Kassouf, M. T.</p>
        <p class="info">Score: 6.0, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556325' target='https://doi.org/10.1101/2023.09.06.556325'> 10.1101/2023.09.06.556325</a></p>
        <p class="abstract">While the elements encoding enhancers and promoters have been relatively well studied, the full spectrum of insulator elements which bind the CCCTC binding factor (CTCF), is relatively poorly characterised. This is partly due to the genomic context of CTCF sites greatly influencing their roles and activity. Here we have developed an experimental system to determine the ability of consistently sized, individual CTCF elements to interpose between enhancers and promoters and thereby reduce gene expression during differentiation. Importantly, each element is tested in the identical location thereby minimising the effect of genomic context. We found no correlation between the ability of CTCF elements to block enhancer-promoter activity with the amount of CTCF or cohesin bound at the natural genomic sites of these elements; the degree of evolutionary conservation; or their resemblance to the consensus core sequences. Nevertheless, we have shown that the strongest enhancer-promoter blockers include a previously described bound element lying upstream of the CTCF core motif. In addition, we found other uncharacterised DNaseI footprints located close to the core motif that may affect function. We have developed an assay of CTCF sequences which will enable researchers to sub-classify CTCF elements in a uniform and unbiased way.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555807">
        <p class="paperTitle">A Toxoplasma gondii putative arginine transporter localizes to the plant-like vacuolar compartment and controls parasite extracellular survival and stage differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piro, F.; Masci, S.; Kannan, G.; Focaia, R.; Schultz, T. L.; Carruthers, V. B.; Di Cristina, M.</p>
        <p class="info">Score: 6.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555807' target='https://doi.org/10.1101/2023.08.31.555807'> 10.1101/2023.08.31.555807</a></p>
        <p class="abstract">Toxoplasma gondii is a protozoan parasite that infects a broad spectrum of hosts and can colonize many organs and cell types. The ability to reside within a wide range of different niches requires substantial adaptability to diverse microenvironments. Very little is known about how this parasite senses various milieus and adapts its metabolism to survive, replicate during the acute stage, and then differentiate to the chronic stage. Most eukaryotes, from yeast to mammals, rely on a nutrient sensing machinery involving the TORC complex as master regulator of cell growth and cell cycle progression. The lysosome functions as a signaling hub where TORC complex assembles and is activated by transceptors, which both sense and transport amino acids, including the arginine transceptor SLC38A9. While most of the TORC components are lost in T. gondii, indicating the evolution of a distinct nutrient sensing mechanism, the parasites lysosomal plant-like vacuolar compartment (PLVAC) may still serve as a sensory platform for controlling parasite growth and differentiation. Using SLC38A9 to query the T. gondii proteome, we identified four putative amino acid transporters, termed TgAAT1-4, that structurally resemble the SLC38A9 arginine transceptor. Assessing their expression and sub-cellular localization, we found that one of them, TgAAT1, localized to the PLVAC and is necessary for normal parasite extracellular survival and bradyzoite differentiation. Moreover, we show that TgAAT1 is involved in the PLVAC efflux of arginine, an amino acid playing a key role in T. gondii differentiation, further supporting the hypothesis that TgAAT1 might play a role in nutrient sensing.

IMPORTANCET. gondii is a highly successful parasite infecting a broad range of warm-blood organisms including about one third of all humans. Although Toxoplasma infections rarely result in symptomatic disease in individuals with a healthy immune system, the incredibly high number of persons infected along with the risk of severe infection in immunocompromised patients and the potential link of chronic infection to mental disorders make this infection a significant public health concern. As a result, there is a pressing need for new treatment approaches that are both effective and well-tolerated. The limitations in understanding how Toxoplasma gondii manages its metabolism to adapt to changing environments and triggers its transformation into bradyzoites have hindered the discovery of vulnerabilities in its metabolic pathways or nutrient acquisition mechanisms to identify new therapeutic targets. In this work, we have shown that the lysosome-like organelle PLVAC, acting through the putative arginine transporter TgAAT1, plays a pivotal role in regulating the parasites extracellular survival and differentiation into bradyzoites.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
