<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Barcode 100K Specimens: In a Single Nanopore Run
Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.
Score: 61.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282
It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Barcode 100K Specimens: In a Single Nanopore Run
Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.
Score: 61.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282
It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Barcode 100K Specimens: In a Single Nanopore Run
Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.
Score: 61.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282
It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Barcode 100K Specimens: In a Single Nanopore Run\nAuthors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.\nScore: 61.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282\nIt is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity.",
  "keywords": [
    
  ],
  "articleBody": " Barcode 100K Specimens: In a Single Nanopore Run\nAuthors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.\nScore: 61.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569282\nIt is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.\nCodon optimality modulates protein output by tuning translation initiation\nAuthors: Lyons, E. F.; Devanneaux, L. C.; Muller, R. Y.; Freitas, A. V.; Meacham, Z. A.; McSharry, M. V.; Trinh, V. N.; Rogers, A. J.; Ingolia, N. T.; Lareau, L. F.\nScore: 53.8, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568910\nThe impact of synonymous codon choice on protein output has important implications for understanding endogenous gene expression and design of synthetic mRNAs. Previously, we used a neural network model to design a series of synonymous fluorescent reporters whose protein output in yeast spanned a seven-fold range corresponding to their predicted translation speed. Here, we show that this effect is not due primarily to the established impact of slow elongation on mRNA stability, but rather, that an active mechanism further decreases the number of proteins made per mRNA. We combine simulations and careful experiments on fluorescent reporters to argue that translation initiation is limited on non-optimally encoded transcripts. Using a genome-wide CRISPRi screen to discover factors modulating the output from non-optimal transcripts, we identify a set of translation initiation factors including multiple subunits of eIF3 whose depletion restored protein output of a non-optimal reporter. Our results show that codon usage can directly limit protein production, across the full range of endogenous variability in codon usage, by limiting translation initiation.\nIdentification of key residues in MERS-CoV and SARS-CoV-2 main proteases for resistance against clinically applied inhibitors nirmatrelvir and ensitrelvir\nAuthors: Krismer, L.; Schoeppe, H.; Rauch, S.; Bante, D.; Sprenger, B.; Naschberger, A.; Costacurta, F.; Fuerst, A.; Sauerwein, A.; Rupp, B.; Kaserer, T.; von Laer, D.; Heilmann, E.\nScore: 29.2, Published: 2023-12-04 DOI: 10.1101/2023.12.04.569917\nThe Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an epidemic, zoonotically emerging pathogen initially reported in Saudi Arabia in 2012. MERS-CoV has the potential to mutate or recombine with other coronaviruses, thus acquiring the ability to efficiently spread among humans and become pandemic. Its high mortality rate of up to 35 % and the absence of effective targeted therapies call for the development of antiviral drugs for this pathogen. Since the beginning of the SARS-CoV-2 pandemic, extensive research has focused on identifying protease inhibitors for the treatment of SARS-CoV-2. Our intention was therefore to assess whether these protease inhibitors are viable options for combating MERS-CoV. To that end, we used previously established protease assays to quantify inhibition of the SARS-CoV-2 and MERS-CoV main proteases. Furthermore, we selected MERS-CoV-Mpro mutants resistant against nirmatrelvir, the most effective inhibitor of this protease, with a safe, surrogate virus-based system, and suggest putative resistance mechanisms. Notably, nirmatrelvir demonstrated effectiveness against various viral proteases, illustrating its potential as a broad-spectrum coronavirus inhibitor. To adress the inherent resistance of MERS-CoV-Mpro to ensitrelvir, we applied directed mutagenesis to a key ensitrelvir-interacting residue and provided structural models.\nTranscription-coupled changes in higher-order genomic structure and transcription hub viscosity prolong enhancer-promoter connectivity\nAuthors: Ohishi, H.; Shinkai, S.; Owada, H.; Fujii, T.; Hosoda, K.; Onami, S.; Yamamoto, T.; Ohkawa, Y.; Ochiai, H.\nScore: 35.8, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568629\nThe orchestration of our genes heavily relies on a coordinated communication between enhancers and promoters, yet how this dynamic interplay remains elusive while transcription is active. Here, we investigated enhancer-promoter (E-P) interactions and relative to transcriptional bursting in mouse embryonic stem cells using sequential DNA/RNA/immunofluorescence (IF)-FISH analyses and computational simulations. Our data reveal that the active state of specific genes is characterized by higher-order genomic structures and local condensates of transcriptional regulatory factors, leading to an elevation in local viscosity that highly stabilizes the duration of E-P interactions. Our study underscores the pivotal role of viscosity in transcriptional dynamics and paves the way for a more nuanced understanding of gene-specific regulatory mechanisms.\nH3K27me3 spreading organizes canonical PRC1 chromatin architecture to regulate developmental programs\nAuthors: Krug, B.; Hu, B.; Chen, H.; Ptack, A.; Chen, X.; Gretarsson, K. H.; Deshmukh, S.; Kabir, N.; Faria Andrade, A.; Jabbour, E.; Harutyunyan, A. S.; Lee, J. J. Y.; Hulswit, M.; Faury, D.; Russo, C.; Xu, X.; Johnston, M. J.; Baguette, A.; Dahl, N. A.; Weil, A. G.; Ellezam, B.; Dali, R.; Blanchette, M.; Wilson, K.; Garcia, B. A.; Soni, R. K.; Gallo, M.; Taylor, M. D.; Kleinman, C. L.; Majewski, J.; Jabado, N.; Lu, C.\nScore: 17.6, Published: 2023-11-28 DOI: 10.1101/2023.11.28.567931\nPolycomb Repressive Complex 2 (PRC2)-mediated histone H3K27 tri-methylation (H3K27me3) recruits canonical PRC1 (cPRC1) to maintain heterochromatin. In early development, polycomb-regulated genes are connected through long-range 3D interactions which resolve upon differentiation. Here, we report that polycomb looping is controlled by H3K27me3 spreading and regulates target gene silencing and cell fate specification. Using glioma-derived H3 Lys-27-Met (H3K27M) mutations as tools to restrict H3K27me3 deposition, we show that H3K27me3 confinement concentrates the chromatin pool of cPRC1, resulting in heightened 3D interactions mirroring chromatin architecture of pluripotency, and stringent gene repression that maintains cells in progenitor states to facilitate tumor development. Conversely, H3K27me3 spread in pluripotent stem cells, following neural differentiation or loss of the H3K36 methyltransferase NSD1, dilutes cPRC1 concentration and dissolves polycomb loops. These results identify the regulatory principles and disease implications of polycomb looping and nominate histone modification-guided distribution of reader complexes as an important mechanism for nuclear compartment organization. Highlights{square} The confinement of H3K27me3 at PRC2 nucleation sites without its spreading correlates with increased 3D chromatin interactions. {square}The H3K27M oncohistone concentrates canonical PRC1 that anchors chromatin loop interactions in gliomas, silencing developmental programs. {square}Stem and progenitor cells require factors promoting H3K27me3 confinement, including H3K36me2, to maintain cPRC1 loop architecture. {square}The cPRC1-H3K27me3 interaction is a targetable driver of aberrant self-renewal in tumor cells.\nDouble-strand breaks in facultative heterochromatin require specific movements and chromatin changes for efficient repair\nAuthors: Wensveen, M. R.; Dixit, A. A.; Colmenares, S. U.; Janssen, A.\nScore: 13.7, Published: 2023-12-01 DOI: 10.1101/2023.12.01.569547\nDNA double-strand breaks (DSBs) must be properly repaired within diverse chromatin domains in order to maintain genome stability. Each chromatin domain has specific properties that can influence the DNA repair response. Whereas euchromatin has an open structure and is associated with active transcription, facultative heterochromatin is more compact and essential to silence developmental genes. Facultative heterochromatin is enriched for Histone H3 Lysine 27 tri-methylation (H3K27me3) and polycomb group proteins, and accumulates in sub-nuclear foci, called polycomb bodies. Whether the specific chromatin properties of facultative heterochromatin require distinct DSB repair dynamics remains unknown. Here we integrate inducible single DSB systems in eu- and facultative hetero-chromatin in Drosophila animals and find that facultative heterochromatic DSBs rapidly move outside polycomb bodies within minutes after their appearance. In line with these specific DSB dynamics, we find a local loss in the canonical heterochromatin marker H3K27me3 at break sites. We demonstrate that DSB movement and loss of H3K27me3 at heterochromatic DSBs both depend on the histone demethylase dUtx. Importantly, this dUtx-mediated loss of H3K27me3 is required for completion of homologous recombination at facultative heterochromatic DSBs. We conclude that DSBs in polycomb bodies are subjected to dUtx-mediated loss of the silencing mark H3K27me3, which promotes DSB movement and timely repair by homologous recombination. Thus, facultative heterochromatin requires distinct chromatin changes to promote safe DSB repair.\nEfficient scar-free knock-ins of several kilobases by engineered CRISPR/Cas endonucleases\nAuthors: Schreiber, T.; Prange, A.; Schaefer, P.; Iwen, T.; Gruetzner, R.; Marillonnet, S.; Tissier, A. F.\nScore: 61.7, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568561\nIn plants and mammals, non-homologous end-joining is the dominant pathway to repair DNA double strand breaks, making it challenging to generate knock-in events. Using a transient assay in Nicotiana benthamiana we identified two groups of exonucleases, respectively from the Herpes Virus and from the bacteriophage T7 families, hat confer up to 38-fold increase in HDR frequencies when fused to Cas9. We achieved precise and scar-free insertion of several kilobases of DNA both in transient and stable transformation systems. In Arabidopsis thaliana, fusion of Cas9 to a Herpes Virus family exonuclease leads to 10-fold higher frequencies of knock-ins. Our results open perspectives for the routine production of knock-in and gene replacement events in plants. One-Sentence SummaryFusions of CRISPR endonucleases to specific 5'-exonucleases leads to significant increase in scar-free multikilobase knock-ins.\nRNA polymerase II transcription with partially assembled TFIID complexes\nAuthors: Hisler, V.; Bardot, P.; Detilleux, D.; Stierle, M.; Garcia Sanchez, E.; Richard, C.; Hadj Arab, L.; Ehrhard, C.; Morlet, B.; Hadziev, Y.; Jung, M.; Le Gras, S.; Negroni, L.; Mueller, F.; Tora, L.; Vincent, S. D.\nScore: 11.6, Published: 2023-11-27 DOI: 10.1101/2023.11.27.567046\nThe recognition of core promoter sequences by the general transcription factor TFIID is the first step in the process of RNA polymerase II (Pol II) transcription initiation. Metazoan holo-TFIID is composed of the TATA binding protein (TBP) and of 13 TBP associated factors (TAFs). Inducible Taf7 knock out (KO) results in the formation of a Taf7-less TFIID complex, while Taf10 KO leads to serious defects within the TFIID assembly pathway. Either TAF7 or TAF10 depletions correlate with the detected TAF occupancy changes at promoters, and with the distinct phenotype severities observed in mouse embryonic stem cells or mouse embryos. Surprisingly however, under either Taf7 or Taf10 deletion conditions, TBP is still associated to the chromatin, and no major changes are observed in nascent Pol II transcription. Thus, partially assembled TFIID complexes can sustain Pol II transcription initiation, but cannot replace holo-TFIID over several cell divisions and/or development.\nValidation of a heat-inducible Ixodes scapularis HSP70 promoter and developing a tick-specific 3xP3 promoter sequence in ISE6 cells\nAuthors: Pham, M.; Hoffmann, H.-h.; Kurtti, T.; Chana, R.; Garcia-Cruz, O.; Aliabadi, S.; Gulia-Nuss, M.\nScore: 6.8, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569248\nIxodes scapularis is an important vector of many pathogens, including the causative agent of Lyme disease, tick-borne encephalitis, and anaplasmosis. The study of gene function in I. scapularis and other ticks has been hampered by the lack of genetic tools, such as an inducible promoter to permit temporal control over transgenes encoding protein or double-stranded RNA expression. Studies of vector-pathogen relationships would also benefit from the capability to activate anti-pathogen genes at different times during pathogen infection and dissemination. We have characterized an intergenic sequence upstream of the heat shock protein 70 (HSP70) gene that can drive Renilla luciferase expression and mCherry fluorescence in the I. scapularis cell line ISE6. In another construct, we replaced the Drosophila melanogaster minimal HSP70 promoter in the synthetic 3xP3 promoter with a minimal portion of the I. scapularis HSP70 promoter and generated an I. scapularis specific 3xP3 (Is3xP3) promoter. Both promoter constructs, IsHSP70 and Is3xP3, allow for heat-inducible expression of mCherry fluorescence in ISE6 cells with an approximately 10-fold increase in the percentage of fluorescent positive cells upon exposure to a 2 h heat shock. These promoters described here will be valuable tools for gene function studies and temporal control of gene expression, including anti-pathogen genes. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=142 SRC=\"FIGDIR/small/569248v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@b2b928org.highwire.dtl.DTLVardef@199f173org.highwire.dtl.DTLVardef@1bae0ecorg.highwire.dtl.DTLVardef@4de3c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nThe AccelerAge framework: A new statistical approach to predict biological age based on time-to-event data\nAuthors: Sluiskes, M. H.; Goeman, J. J.; Beekman, M.; Slagboom, P. E.; van den Akker, E. B.; Putter, H.; Rodriguez-Girondo, M.\nScore: 8.1, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566803\nAging is a multifaceted and intricate physiological process characterized by a gradual decline in functional capacity, leading to increased susceptibility to diseases and mortality. While chronological age serves as a strong risk factor for age-related health conditions, considerable heterogeneity exists in the aging trajectories of individuals, suggesting that biological age may provide a more nuanced understanding of the aging process. However, the concept of biological age lacks a clear operationalization, leading to the development of various biological age predictors without a solid statistical foundation. This paper addresses these limitations by proposing a comprehensive operationalization of biological age, introducing the \"AccelerAge\" framework for predicting biological age, and introducing previously underutilized evaluation measures for assessing the performance of biological age predictors. The AccelerAge framework, based on Accelerated Failure Time (AFT) models, directly models the effect of candidate predictors of aging on an individuals survival time, aligning with the prevalent metaphor of aging as a clock. We compare predictors based on the AccelerAge framework to a predictor based on the GrimAge predictor, which is considered one of the best-performing biological age predictors, using simulated data as well as data from the UK Biobank and the Leiden Longevity Study. Our approach seeks to establish a robust statistical foundation for biological age clocks, enabling a more accurate and interpretable assessment of an individuals aging status.\n",
  "wordCount" : "2345",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569282">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569282" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569282">
        <p class="paperTitle">Barcode 100K Specimens: In a Single Nanopore Run</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569282" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569282" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hebert, P. D. N.; Floyd, R.; Jafarpour, S.; Prosser, S. W. J.</p>
        <p class="info">Score: 61.3, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569282' target='https://doi.org/10.1101/2023.11.29.569282'> 10.1101/2023.11.29.569282</a></p>
        <p class="abstract">It is a global priority to better manage the biosphere, but action needs to be informed by monitoring shifts in the abundance and distribution of species across the domains of life. The acquisition of such information is currently constrained by the limited knowledge of biodiversity. Among the 20 million or more species of eukaryotes, just a tenth have scientific names. DNA barcoding can speed the registration of unknown animal species, the most diverse kingdom of eukaryotes, as the BIN system automates their recognition. However, inexpensive analytical protocols are critical as the census of all animal species will require processing a billion or more specimens. Barcoding involves DNA extraction followed by PCR and sequencing with the last step dominating costs until 2017. By recovering barcodes from highly multiplexed samples, the Sequel platforms from Pacific BioSciences slashed costs by 90%, but these instruments are only deployed in core facilities because of their expense. Sequencers from Oxford Nanopore Technologies provide an escape from high capital and service costs, but their low sequence fidelity has, until now, kept analytical cost above Sequel. However, the improved performance of its latest flow cells (R10.4.1) might erase this differential. This study demonstrates that a regular MinION flow cell can characterize an amplicon pool derived from 100,000 specimens while a Flongle flow cell can process one derived from several thousand. At $0.01 per specimen, DNA sequencing is now the least expensive step in the barcode workflow. By coupling simplified protocols for DNA extraction with ultra-low volume PCRs, it will be possible to move from specimen to DNA barcode for $0.10, a price point that will enable the census of all species within two decades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568910">
        <p class="paperTitle">Codon optimality modulates protein output by tuning translation initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lyons, E. F.; Devanneaux, L. C.; Muller, R. Y.; Freitas, A. V.; Meacham, Z. A.; McSharry, M. V.; Trinh, V. N.; Rogers, A. J.; Ingolia, N. T.; Lareau, L. F.</p>
        <p class="info">Score: 53.8, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568910' target='https://doi.org/10.1101/2023.11.27.568910'> 10.1101/2023.11.27.568910</a></p>
        <p class="abstract">The impact of synonymous codon choice on protein output has important implications for understanding endogenous gene expression and design of synthetic mRNAs. Previously, we used a neural network model to design a series of synonymous fluorescent reporters whose protein output in yeast spanned a seven-fold range corresponding to their predicted translation speed. Here, we show that this effect is not due primarily to the established impact of slow elongation on mRNA stability, but rather, that an active mechanism further decreases the number of proteins made per mRNA. We combine simulations and careful experiments on fluorescent reporters to argue that translation initiation is limited on non-optimally encoded transcripts. Using a genome-wide CRISPRi screen to discover factors modulating the output from non-optimal transcripts, we identify a set of translation initiation factors including multiple subunits of eIF3 whose depletion restored protein output of a non-optimal reporter. Our results show that codon usage can directly limit protein production, across the full range of endogenous variability in codon usage, by limiting translation initiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569917">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569917" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569917">
        <p class="paperTitle">Identification of key residues in MERS-CoV and SARS-CoV-2 main proteases for resistance against clinically applied inhibitors nirmatrelvir and ensitrelvir</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569917" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569917" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krismer, L.; Schoeppe, H.; Rauch, S.; Bante, D.; Sprenger, B.; Naschberger, A.; Costacurta, F.; Fuerst, A.; Sauerwein, A.; Rupp, B.; Kaserer, T.; von Laer, D.; Heilmann, E.</p>
        <p class="info">Score: 29.2, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569917' target='https://doi.org/10.1101/2023.12.04.569917'> 10.1101/2023.12.04.569917</a></p>
        <p class="abstract">The Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an epidemic, zoonotically emerging pathogen initially reported in Saudi Arabia in 2012. MERS-CoV has the potential to mutate or recombine with other coronaviruses, thus acquiring the ability to efficiently spread among humans and become pandemic. Its high mortality rate of up to 35 % and the absence of effective targeted therapies call for the development of antiviral drugs for this pathogen. Since the beginning of the SARS-CoV-2 pandemic, extensive research has focused on identifying protease inhibitors for the treatment of SARS-CoV-2. Our intention was therefore to assess whether these protease inhibitors are viable options for combating MERS-CoV. To that end, we used previously established protease assays to quantify inhibition of the SARS-CoV-2 and MERS-CoV main proteases. Furthermore, we selected MERS-CoV-Mpro mutants resistant against nirmatrelvir, the most effective inhibitor of this protease, with a safe, surrogate virus-based system, and suggest putative resistance mechanisms. Notably, nirmatrelvir demonstrated effectiveness against various viral proteases, illustrating its potential as a broad-spectrum coronavirus inhibitor. To adress the inherent resistance of MERS-CoV-Mpro to ensitrelvir, we applied directed mutagenesis to a key ensitrelvir-interacting residue and provided structural models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568629">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568629" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568629">
        <p class="paperTitle">Transcription-coupled changes in higher-order genomic structure and transcription hub viscosity prolong enhancer-promoter connectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568629" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568629" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ohishi, H.; Shinkai, S.; Owada, H.; Fujii, T.; Hosoda, K.; Onami, S.; Yamamoto, T.; Ohkawa, Y.; Ochiai, H.</p>
        <p class="info">Score: 35.8, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568629' target='https://doi.org/10.1101/2023.11.27.568629'> 10.1101/2023.11.27.568629</a></p>
        <p class="abstract">The orchestration of our genes heavily relies on a coordinated communication between enhancers and promoters, yet how this dynamic interplay remains elusive while transcription is active. Here, we investigated enhancer-promoter (E-P) interactions and relative to transcriptional bursting in mouse embryonic stem cells using sequential DNA/RNA/immunofluorescence (IF)-FISH analyses and computational simulations. Our data reveal that the active state of specific genes is characterized by higher-order genomic structures and local condensates of transcriptional regulatory factors, leading to an elevation in local viscosity that highly stabilizes the duration of E-P interactions. Our study underscores the pivotal role of viscosity in transcriptional dynamics and paves the way for a more nuanced understanding of gene-specific regulatory mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.567931">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.567931" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.567931">
        <p class="paperTitle">H3K27me3 spreading organizes canonical PRC1 chromatin architecture to regulate developmental programs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.567931" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.567931" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krug, B.; Hu, B.; Chen, H.; Ptack, A.; Chen, X.; Gretarsson, K. H.; Deshmukh, S.; Kabir, N.; Faria Andrade, A.; Jabbour, E.; Harutyunyan, A. S.; Lee, J. J. Y.; Hulswit, M.; Faury, D.; Russo, C.; Xu, X.; Johnston, M. J.; Baguette, A.; Dahl, N. A.; Weil, A. G.; Ellezam, B.; Dali, R.; Blanchette, M.; Wilson, K.; Garcia, B. A.; Soni, R. K.; Gallo, M.; Taylor, M. D.; Kleinman, C. L.; Majewski, J.; Jabado, N.; Lu, C.</p>
        <p class="info">Score: 17.6, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.567931' target='https://doi.org/10.1101/2023.11.28.567931'> 10.1101/2023.11.28.567931</a></p>
        <p class="abstract">Polycomb Repressive Complex 2 (PRC2)-mediated histone H3K27 tri-methylation (H3K27me3) recruits canonical PRC1 (cPRC1) to maintain heterochromatin. In early development, polycomb-regulated genes are connected through long-range 3D interactions which resolve upon differentiation. Here, we report that polycomb looping is controlled by H3K27me3 spreading and regulates target gene silencing and cell fate specification. Using glioma-derived H3 Lys-27-Met (H3K27M) mutations as tools to restrict H3K27me3 deposition, we show that H3K27me3 confinement concentrates the chromatin pool of cPRC1, resulting in heightened 3D interactions mirroring chromatin architecture of pluripotency, and stringent gene repression that maintains cells in progenitor states to facilitate tumor development. Conversely, H3K27me3 spread in pluripotent stem cells, following neural differentiation or loss of the H3K36 methyltransferase NSD1, dilutes cPRC1 concentration and dissolves polycomb loops. These results identify the regulatory principles and disease implications of polycomb looping and nominate histone modification-guided distribution of reader complexes as an important mechanism for nuclear compartment organization.

Highlights{square} The confinement of H3K27me3 at PRC2 nucleation sites without its spreading correlates with increased 3D chromatin interactions.
{square}The H3K27M oncohistone concentrates canonical PRC1 that anchors chromatin loop interactions in gliomas, silencing developmental programs.
{square}Stem and progenitor cells require factors promoting H3K27me3 confinement, including H3K36me2, to maintain cPRC1 loop architecture.
{square}The cPRC1-H3K27me3 interaction is a targetable driver of aberrant self-renewal in tumor cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569547" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569547">
        <p class="paperTitle">Double-strand breaks in facultative heterochromatin require specific movements and chromatin changes for efficient repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wensveen, M. R.; Dixit, A. A.; Colmenares, S. U.; Janssen, A.</p>
        <p class="info">Score: 13.7, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569547' target='https://doi.org/10.1101/2023.12.01.569547'> 10.1101/2023.12.01.569547</a></p>
        <p class="abstract">DNA double-strand breaks (DSBs) must be properly repaired within diverse chromatin domains in order to maintain genome stability. Each chromatin domain has specific properties that can influence the DNA repair response. Whereas euchromatin has an open structure and is associated with active transcription, facultative heterochromatin is more compact and essential to silence developmental genes. Facultative heterochromatin is enriched for Histone H3 Lysine 27 tri-methylation (H3K27me3) and polycomb group proteins, and accumulates in sub-nuclear foci, called polycomb bodies. Whether the specific chromatin properties of facultative heterochromatin require distinct DSB repair dynamics remains unknown. Here we integrate inducible single DSB systems in eu- and facultative hetero-chromatin in Drosophila animals and find that facultative heterochromatic DSBs rapidly move outside polycomb bodies within minutes after their appearance. In line with these specific DSB dynamics, we find a local loss in the canonical heterochromatin marker H3K27me3 at break sites. We demonstrate that DSB movement and loss of H3K27me3 at heterochromatic DSBs both depend on the histone demethylase dUtx. Importantly, this dUtx-mediated loss of H3K27me3 is required for completion of homologous recombination at facultative heterochromatic DSBs. We conclude that DSBs in polycomb bodies are subjected to dUtx-mediated loss of the silencing mark H3K27me3, which promotes DSB movement and timely repair by homologous recombination. Thus, facultative heterochromatin requires distinct chromatin changes to promote safe DSB repair.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568561">
        <p class="paperTitle">Efficient scar-free knock-ins of several kilobases by engineered CRISPR/Cas endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schreiber, T.; Prange, A.; Schaefer, P.; Iwen, T.; Gruetzner, R.; Marillonnet, S.; Tissier, A. F.</p>
        <p class="info">Score: 61.7, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568561' target='https://doi.org/10.1101/2023.11.24.568561'> 10.1101/2023.11.24.568561</a></p>
        <p class="abstract">In plants and mammals, non-homologous end-joining is the dominant pathway to repair DNA double strand breaks, making it challenging to generate knock-in events. Using a transient assay in Nicotiana benthamiana we identified two groups of exonucleases, respectively from the Herpes Virus and from the bacteriophage T7 families, hat confer up to 38-fold increase in HDR frequencies when fused to Cas9. We achieved precise and scar-free insertion of several kilobases of DNA both in transient and stable transformation systems. In Arabidopsis thaliana, fusion of Cas9 to a Herpes Virus family exonuclease leads to 10-fold higher frequencies of knock-ins. Our results open perspectives for the routine production of knock-in and gene replacement events in plants.

One-Sentence SummaryFusions of CRISPR endonucleases to specific 5&#39;-exonucleases leads to significant increase in scar-free multikilobase knock-ins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.567046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.567046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.567046">
        <p class="paperTitle">RNA polymerase II transcription with partially assembled TFIID complexes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.567046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.567046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hisler, V.; Bardot, P.; Detilleux, D.; Stierle, M.; Garcia Sanchez, E.; Richard, C.; Hadj Arab, L.; Ehrhard, C.; Morlet, B.; Hadziev, Y.; Jung, M.; Le Gras, S.; Negroni, L.; Mueller, F.; Tora, L.; Vincent, S. D.</p>
        <p class="info">Score: 11.6, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.567046' target='https://doi.org/10.1101/2023.11.27.567046'> 10.1101/2023.11.27.567046</a></p>
        <p class="abstract">The recognition of core promoter sequences by the general transcription factor TFIID is the first step in the process of RNA polymerase II (Pol II) transcription initiation. Metazoan holo-TFIID is composed of the TATA binding protein (TBP) and of 13 TBP associated factors (TAFs). Inducible Taf7 knock out (KO) results in the formation of a Taf7-less TFIID complex, while Taf10 KO leads to serious defects within the TFIID assembly pathway. Either TAF7 or TAF10 depletions correlate with the detected TAF occupancy changes at promoters, and with the distinct phenotype severities observed in mouse embryonic stem cells or mouse embryos. Surprisingly however, under either Taf7 or Taf10 deletion conditions, TBP is still associated to the chromatin, and no major changes are observed in nascent Pol II transcription. Thus, partially assembled TFIID complexes can sustain Pol II transcription initiation, but cannot replace holo-TFIID over several cell divisions and/or development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569248">
        <p class="paperTitle">Validation of a heat-inducible Ixodes scapularis HSP70 promoter and developing a tick-specific 3xP3 promoter sequence in ISE6 cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pham, M.; Hoffmann, H.-h.; Kurtti, T.; Chana, R.; Garcia-Cruz, O.; Aliabadi, S.; Gulia-Nuss, M.</p>
        <p class="info">Score: 6.8, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569248' target='https://doi.org/10.1101/2023.11.29.569248'> 10.1101/2023.11.29.569248</a></p>
        <p class="abstract">Ixodes scapularis is an important vector of many pathogens, including the causative agent of Lyme disease, tick-borne encephalitis, and anaplasmosis. The study of gene function in I. scapularis and other ticks has been hampered by the lack of genetic tools, such as an inducible promoter to permit temporal control over transgenes encoding protein or double-stranded RNA expression. Studies of vector-pathogen relationships would also benefit from the capability to activate anti-pathogen genes at different times during pathogen infection and dissemination. We have characterized an intergenic sequence upstream of the heat shock protein 70 (HSP70) gene that can drive Renilla luciferase expression and mCherry fluorescence in the I. scapularis cell line ISE6. In another construct, we replaced the Drosophila melanogaster minimal HSP70 promoter in the synthetic 3xP3 promoter with a minimal portion of the I. scapularis HSP70 promoter and generated an I. scapularis specific 3xP3 (Is3xP3) promoter. Both promoter constructs, IsHSP70 and Is3xP3, allow for heat-inducible expression of mCherry fluorescence in ISE6 cells with an approximately 10-fold increase in the percentage of fluorescent positive cells upon exposure to a 2 h heat shock. These promoters described here will be valuable tools for gene function studies and temporal control of gene expression, including anti-pathogen genes.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=142 SRC=&#34;FIGDIR/small/569248v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@b2b928org.highwire.dtl.DTLVardef@199f173org.highwire.dtl.DTLVardef@1bae0ecorg.highwire.dtl.DTLVardef@4de3c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566803">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566803" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566803">
        <p class="paperTitle">The AccelerAge framework: A new statistical approach to predict biological age based on time-to-event data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566803" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566803" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sluiskes, M. H.; Goeman, J. J.; Beekman, M.; Slagboom, P. E.; van den Akker, E. B.; Putter, H.; Rodriguez-Girondo, M.</p>
        <p class="info">Score: 8.1, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566803' target='https://doi.org/10.1101/2023.11.13.566803'> 10.1101/2023.11.13.566803</a></p>
        <p class="abstract">Aging is a multifaceted and intricate physiological process characterized by a gradual decline in functional capacity, leading to increased susceptibility to diseases and mortality. While chronological age serves as a strong risk factor for age-related health conditions, considerable heterogeneity exists in the aging trajectories of individuals, suggesting that biological age may provide a more nuanced understanding of the aging process. However, the concept of biological age lacks a clear operationalization, leading to the development of various biological age predictors without a solid statistical foundation. This paper addresses these limitations by proposing a comprehensive operationalization of biological age, introducing the &#34;AccelerAge&#34; framework for predicting biological age, and introducing previously underutilized evaluation measures for assessing the performance of biological age predictors. The AccelerAge framework, based on Accelerated Failure Time (AFT) models, directly models the effect of candidate predictors of aging on an individuals survival time, aligning with the prevalent metaphor of aging as a clock. We compare predictors based on the AccelerAge framework to a predictor based on the GrimAge predictor, which is considered one of the best-performing biological age predictors, using simulated data as well as data from the UK Biobank and the Leiden Longevity Study. Our approach seeks to establish a robust statistical foundation for biological age clocks, enabling a more accurate and interpretable assessment of an individuals aging status.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
