<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion
Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.
Score: 106.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365
Cell survival under nutrient-deprived conditions relies on cells&#39; ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion
Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.
Score: 106.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365
Cell survival under nutrient-deprived conditions relies on cells&#39; ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion
Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.
Score: 106.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365
Cell survival under nutrient-deprived conditions relies on cells&#39; ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion\nAuthors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.\nScore: 106.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365\nCell survival under nutrient-deprived conditions relies on cells\u0026#39; ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria.",
  "keywords": [
    
  ],
  "articleBody": " Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion\nAuthors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.\nScore: 106.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.561365\nCell survival under nutrient-deprived conditions relies on cells' ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria. The molecular mechanism underlying ribosomal sequestration remains elusive. In this study, we performed time-lapse in situ cryo-electron tomography and cryo-electron microscopy complemented by biochemical experiments to elucidate the molecular details of this adaptive response. Our analysis indicate that upon glucose depletion protein synthesis is halted, causing ribosomes to enter an inactive state characterized by a conformational change that obstructs the peptidyl transferase center. Our in situ experiments reveal the presence of oligomeric arrays of hibernating ribosomes tethered to the mitochondrial surface. Surprisingly, ribosomes bind to the outer mitochondrial membrane via the small ribosomal subunit, an interaction facilitated by the ribosomal protein RACK1-orthologue Cpc2. Our experiments show that ribosome tethering is important for cell survival under glucose depletion conditions. This study broadens our understanding of the cellular adaptations triggered by nutrient scarcity and the underlying molecular mechanisms that regulate cell quiescence.\nPredicting prime editing efficiency across diverse edit types and chromatin contexts with machine learning\nAuthors: Mathis, N.; Allam, A.; Talas, A.; Benvenuto, E.; Schep, R.; Damodharan, T.; Balazs, Z.; Janjuha, S.; Schmidheini, L.; Böck, D.; van Steensel, B.; Krauthammer, M.; Schwank, G.\nScore: 22.9, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561414\nPrime editing is a powerful genome editing technology, but its efficiency varies depending on the pegRNA design and target locus. Existing computational models for predicting prime editing rates are limited by their focus on specific edit types and by omitting the local chromatin environment. In our study, we developed machine learning models that predict prime editing efficiencies across a wide range of edit types up to 15 bp ('PRIDICT2.0') and in different chromatin contexts ('ePRIDICT'). Both models can be accessed at www.pridict.it.\nLocalized synthesis of molecular chaperones sustains neuronal proteostasis\nAuthors: Alecki, C.; Rizwan, J.; Le, P.; Jacob-Tomas, S.; Xu, J. S. M.; Minotti, S.; Wu, T.; Durham, H.; Yeo, G. W.; Vera, M.\nScore: 11.6, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560761\nNeurons are challenged to maintain proteostasis in neuronal projections, particularly with the physiological stress at synapses to support intercellular communication underlying important functions such as memory and movement control. Proteostasis is maintained through regulated protein synthesis and degradation and chaperone-assisted protein folding. Using high-resolution fluorescent microscopy, we discovered that neurons localize a subset of chaperone mRNAs to their dendrites, particularly more proximal regions, and increase this asymmetric localization following proteotoxic stress through microtubule-based transport from the soma. The most abundant chaperone mRNA in dendrites encodes the constitutive heat shock protein 70, HSPA8. Proteotoxic stress in cultured neurons, induced by inhibiting proteasome activity or inducing oxidative stress, enhanced transport of Hspa8 mRNAs to dendrites and the percentage of mRNAs engaged in translation on mono and polyribosomes. Knocking down the ALS-related protein Fused in Sarcoma (FUS) and a dominant mutation in the heterogenous nuclear ribonucleoprotein A2/B1 (HNRNPA2B1) impaired stress-mediated localization of Hspa8 mRNA to dendrites in cultured murine motor neurons and human iPSC-derived neurons, respectively, revealing the importance of these RNA-binding proteins in maintaining proteostasis. These results reveal the increased dendritic localization and translation of the constitutive HSP70 Hspa8 mRNA as a crucial neuronal stress response to uphold proteostasis and prevent neurodegeneration. SUMMARYLocalizing chaperones mRNAs in neuronal dendrites is a novel on-demand system to uphold proteostasis upon stress.\nA conserved class of viral RNA structures regulate translation reinitiation through dynamic ribosome interactions\nAuthors: Sherlock, M.; Langeberg, C.; Segar, K.; Kieft, J. S.\nScore: 10.7, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560040\nCertain viral RNAs encode proteins downstream of the main protein coding region, expressed through \"termination-reinitiation\" events, dependent on RNA structure. RNA elements located upstream of the first stop codon within these viral mRNAs bind the ribosome, preventing ribosome recycling and inducing reinitiation. We used bioinformatic methods to identify new examples of viral reinitiation-stimulating RNAs and experimentally verified their secondary structure and function. We determined the structure of a representative viral RNA-ribosome complex using cryoEM. 3D classification and variability analyses reveal that the viral RNA structure can sample a range of conformations while remaining tethered to the ribosome, which enabling the ribosome to find a reinitiation start site within a limited range of mRNA sequence. Evaluating the conserved features and constraints of this entire RNA class in the context of the cryoEM reconstruction provides insight into mechanisms enabling reinitiation, a translation regulation strategy employed by many other viral and eukaryotic systems.\nBroad variation in response of individual introns to splicing inhibitors in a humanized yeast strain\nAuthors: Hunter, O.; Talkish, J.; Quick-Cleveland, J.; Igel, H.; Tan, A.; Kuersten, S.; Katzman, S.; Donohue, J. P.; Jurica, M. S.; Ares, M.\nScore: 9.0, Published: 2023-10-05 DOI: 10.1101/2023.10.05.560965\nIntron branch point (BP) recognition by the U2 snRNP is a critical step of splicing, vulnerable to recurrent cancer mutations and bacterial natural product inhibitors. The BP binds a conserved pocket in the SF3B1 (human) or Hsh155 (yeast) U2 snRNP protein. Amino acids that line this pocket affect binding of splicing inhibitors like Pladienolide-B (Plad-B), such that organisms differ in their sensitivity. To study the mechanism of splicing inhibitor action in a simplified system, we modified the naturally Plad-B resistant yeast Saccharomyces cerevisiae by changing 14 amino acids in the Hsh155 BP pocket to those from human. This humanized yeast grows normally, and splicing is largely unaffected by the mutation. Minutes after addition of Plad-B splicing is inhibited, however different introns appear inhibited to different extents. Intron-specific inhibition differences are also observed when we measure co-transcriptional splicing in Plad-B using single-molecule intron tracking (SMIT) to minimize gene-specific transcription and decay rates that cloud estimates of inhibition by standard RNA-seq. Comparison of Plad-B intron sensitivities to those of the structurally distinct inhibitor Thailanstatin-A reveals intron-specific differences in sensitivity to different compounds. This work exposes a complex relationship between binding of different members of this class of inhibitors to the spliceosome and intron-specific rates of BP recognition and catalysis. The results echo and extend observations from mammalian cells, where inhibitor studies are complicated by multi-intron gene architecture and alternative splicing. The more compact yeast system may hasten characterization of splicing inhibitors, accelerating improvements in selectivity and therapeutic efficacy.\nThe Rho GTPase Rac1 mediates exercise training adaptations\nAuthors: Raun, S. H.; Henriquez-Olguin, C.; Frank, E.; Knudsen, J. R.; Ali, M. S.; Andersen, N. R.; Moeller, L. L. V.; Davey, J.; Qian, H.; Coelho, A.; Carl, C. S.; Voldstedlund, C. T.; Kiens, B.; Holmdahl, R.; Gregorevic, P.; Jensen, T. E.; Richter, E. A.; Sylow, L.\nScore: 8.4, Published: 2023-10-09 DOI: 10.1101/2023.10.08.561442\nExercise training elicits tremendous health benefits; however, the molecular underpinnings are poorly understood. As one of the most regulated groups of proteins following acute exercise in human muscle, Rho GTPases are unexplored candidates for mediating the beneficial effects of exercise. The Rho GTPase Rac1 was activated during multiple exercise modalities and remained elevated hours after resistance exercise in human muscle. Inducible muscle-specific Rac1 knockout (Rac1 imKO) mice, displayed attenuated muscle protein synthesis, glycogen resynthesis and p38 MAPK signaling in recovery from contractions. Exercise training upregulated Rac1 protein content in human and mouse muscle. Overexpression of hyperactive Rac1 elevated reactive oxidant species production during exercise yet did not induce a trained muscle phenotype. In Rac1 imKO mice, the improvements in running capacity and muscle mass after exercise training were diminished. Using gain- and loss-of-function mouse models and human muscle biopsies, we identify Rac1 as a regulator of exercise training adaptions.\nOptical ice thickness measurement in the VitroJet for time-efficient single particle structure determination\nAuthors: Henderikx, R. J. M.; Shahzad, S.; Schotman, M. J. G.; Mann, D.; Heidler, T. V.; Leahy, E.; Ashtiani, D.; Jeurissen, R. J. M.; Peters, P. J.; Sachse, C.; Beulen, B. W. A. M. M.\nScore: 7.8, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561488\nIn cryo-electron microscopy structure determination, embedding biomolecules in vitreous ice of optimal thickness is critical. Ice thickness assessment and selection of suitable holes for data collection are currently part of time-consuming preparatory routines performed on expensive electron microscopes. For this reason, we developed a method to determine ice thickness during sample preparation using the optical camera integrated in the VitroJet. As a result, the ice thickness of buffer-suspended holes on an EM grid can be determined faithfully in the working range relevant for single particle cryo-EM. Dependent on the thickness, the error is below +- 20 nm in the range between 0 - 70 nm down to +- 10 nm in the thinnest ice regions (10 - 40 nm). Single particle structures of apoferritin were determined at two distinct thicknesses of 30 nm and 70 nm to demonstrate the suitability of optical layer thickness measurements for time-efficient cryo-EM structure determination.\nPost-ischemic inactivation of HIF prolyl hydroxylases in endothelium promotes maladaptive kidney repair by inducing glycolysis\nAuthors: Tiwari, R.; Sharma, R.; Rajendran, G.; Borkowski, G. S.; An, S. Y.; Schonfeld, M. P.; O'Sullivan, J.; Schipma, M.; Zhou, Y.; Courbon, G.; David, V.; Quaggin, S. E.; Thorp, E. B.; Chandel, N. S.; Kapitsinou, P. P.\nScore: 6.5, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560700\nIschemic acute kidney injury (AKI) is common in hospitalized patients and increases the risk for chronic kidney disease (CKD). Impaired endothelial cell (EC) functions are thought to contribute in AKI to CKD transition, but the underlying mechanisms remain unclear. Here, we identify a critical role for endothelial oxygen sensing prolyl hydroxylase domain (PHD) enzymes 1-3 in regulating post-ischemic kidney repair. In renal endothelium, we observed compartment-specific differences in the expression of the three PHD isoforms in both mice and humans. We found that post-ischemic concurrent inactivation of endothelial PHD1, PHD2, and PHD3 but not PHD2 alone promoted maladaptive kidney repair characterized by exacerbated tissue injury, fibrosis, and inflammation. Single-cell RNA-seq analysis of the post-ischemic endothelial PHD1, PHD2 and PHD3 deficient (PHDTiEC) kidney revealed an endothelial glycolytic transcriptional signature, also observed in human kidneys with severe AKI. This metabolic program was coupled to upregulation of the SLC16A3 gene encoding the lactate exporter monocarboxylate transporter 4 (MCT4). Strikingly, treatment with the MCT4 inhibitor syrosingopine restored adaptive kidney repair in PHDTiEC mice. Mechanistically, MCT4 inhibition suppressed pro-inflammatory EC activation reducing monocyte-endothelial cell interaction. Our findings suggest avenues for halting AKI to CKD transition based on selectively targeting the endothelial hypoxia-driven glycolysis/MCT4 axis.\nRNA structure modulates Cas13 activity and enables mismatch detection\nAuthors: Kimchi, O.; Larsen, B. B.; Dunkley, O. R.; te Velthuis, A. J. W.; Myhrvold, C.\nScore: 6.3, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560533\nThe RNA-targeting CRISPR nuclease Cas13 has emerged as a powerful tool for applications ranging from nucleic acid detection to transcriptome engineering and RNA imaging. Cas13 is activated by the hybridization of a CRISPR RNA (crRNA) to a complementary single-stranded RNA (ssRNA) protospacer in a target RNA. Though Cas13 is not activated by double-stranded RNA (dsRNA) in vitro, it paradoxically demonstrates robust RNA targeting in environments where the vast majority of RNAs are highly structured. Understanding Cas13's mechanism of binding and activation will be key to improving its ability to detect and perturb RNA; however, the mechanism by which Cas13 binds structured RNAs remains unknown. Here, we systematically probe the mechanism of LwaCas13a activation in response to RNA structure perturbations using a massively multiplexed screen. We find that there are two distinct sequence-independent modes by which secondary structure affects Cas13 activity: structure in the protospacer region competes with the crRNA and can be disrupted via a strand-displacement mechanism, while structure in the region 3' to the protospacer has an allosteric inhibitory effect. We leverage the kinetic nature of the strand displacement process to improve Cas13-based RNA detection and enhance mismatch discrimination by up to 50-fold, enabling sequence-agnostic mutation identification at low (\u003c1%) allele frequencies. Our work sets a new standard for CRISPR-based nucleic acid detection and will enable intelligent and secondary-structure-guided target selection while also expanding the range of RNAs available for targeting with Cas13.\nLoss of alkyladenine DNA glycosylase alters gene expression in the developing mouse brain and leads to reduced anxiety and improved memory\nAuthors: Bordin, D. L.; Grooms, K.; Montaldo, N. P.; Martin, S. L. F.; Saetrom, P.; Samson, L. D.; Bjoras, M.; Van Loon, B.\nScore: 5.2, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561113\nNeurodevelopment is a tightly coordinated process, during which the genome is exposed to spectra of endogenous agents at different stages of differentiation. Emerging evidence indicates that DNA damage is an important feature of developing brain, tightly linked to gene expression and neuronal activity. Some of the most frequent DNA damage includes changes to DNA bases, which are recognized by DNA glycosylases and repaired through base excision repair (BER) pathway. The only mammalian DNA glycosylase able to remove frequent alkylated DNA based is alkyladenine DNA glycosylase (Aag, aka Mpg). We recently demonstrated that, besides its role in DNA repair, AAG affects expression of neurodevelopmental genes in human cells. Aag was further proposed to act as reader of epigenetic marks, including 5-hydroxymethylcytosine (5hmC), in the mouse brain. Despite the potential Aag involvement in the key brain processes, the impact of Aag loss on developing brain remains unknown. Here, by using Aag knockout (Aag-/-) mice, we show that Aag absence leads to reduced DNA break levels, evident in lowered number of {gamma}H2AX foci in postnatal day 5 (P5) hippocampi. This is accompanied by changes in 5hmC signal intensity in different hippocampal regions. Transcriptome analysis of hippocampi and prefrontal cortex, at different developmental stages, indicates that lack of Aag alters gene expression, primarily of genes involved in regulation of response to stress. Across all developmental stages tested aldehyde dehydrogenase 2 (Aldh2) emerged as one of the most prominent genes deregulated in Aag-dependent manner. In line with the changes in hippocampal DNA damage levels and the gene expression, adult Aag-/- mice exhibit altered behavior, evident in decreased anxiety levels determined in the Elevated Zero Maze and increased alternations in the Elevated T Maze tests. Taken together these results suggests that Aag has functions in modulation of genome dynamics during brain development, important for animal behavior. HighlightsO_LIAag loss results in reduced DNA damage signal in developing hippocampus; C_LIO_LI5hmC signal intensity is perturbed in hippocampal regions of Aag-/- mice; C_LIO_LIGene expression is altered in Aag-/- hippocampus and prefrontal cortex; C_LIO_LIAag represses Aldh2 expression; C_LIO_LIAag-/- mice have reduced anxiety and improved memory. C_LI\n",
  "wordCount" : "2414",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561365">
        <p class="paperTitle">Hibernating ribosomes tether to mitochondria as an adaptive response to cellular stress during glucose depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gemin, O.; Gluc, M.; Purdy, M.; Rosa, H.; Niemann, M.; Peskova, Y.; Mattei, S.; Jomaa, A.</p>
        <p class="info">Score: 106.1, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561365' target='https://doi.org/10.1101/2023.10.08.561365'> 10.1101/2023.10.08.561365</a></p>
        <p class="abstract">Cell survival under nutrient-deprived conditions relies on cells&#39; ability to adapt their organelles and to rewire their metabolic pathways. In the fission yeast Schizosaccharomyces pombe, nutrient depletion is an unfavorable condition for protein synthesis and triggers a response characterized by mitochondrial fragmentation and the sequestration of cytosolic ribosomes on mitochondria. The molecular mechanism underlying ribosomal sequestration remains elusive. In this study, we performed time-lapse in situ cryo-electron tomography and cryo-electron microscopy complemented by biochemical experiments to elucidate the molecular details of this adaptive response. Our analysis indicate that upon glucose depletion protein synthesis is halted, causing ribosomes to enter an inactive state characterized by a conformational change that obstructs the peptidyl transferase center. Our in situ experiments reveal the presence of oligomeric arrays of hibernating ribosomes tethered to the mitochondrial surface. Surprisingly, ribosomes bind to the outer mitochondrial membrane via the small ribosomal subunit, an interaction facilitated by the ribosomal protein RACK1-orthologue Cpc2. Our experiments show that ribosome tethering is important for cell survival under glucose depletion conditions. This study broadens our understanding of the cellular adaptations triggered by nutrient scarcity and the underlying molecular mechanisms that regulate cell quiescence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561414">
        <p class="paperTitle">Predicting prime editing efficiency across diverse edit types and chromatin contexts with machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mathis, N.; Allam, A.; Talas, A.; Benvenuto, E.; Schep, R.; Damodharan, T.; Balazs, Z.; Janjuha, S.; Schmidheini, L.; Böck, D.; van Steensel, B.; Krauthammer, M.; Schwank, G.</p>
        <p class="info">Score: 22.9, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561414' target='https://doi.org/10.1101/2023.10.09.561414'> 10.1101/2023.10.09.561414</a></p>
        <p class="abstract">Prime editing is a powerful genome editing technology, but its efficiency varies depending on the pegRNA design and target locus. Existing computational models for predicting prime editing rates are limited by their focus on specific edit types and by omitting the local chromatin environment. In our study, we developed machine learning models that predict prime editing efficiencies across a wide range of edit types up to 15 bp (&#39;PRIDICT2.0&#39;) and in different chromatin contexts (&#39;ePRIDICT&#39;). Both models can be accessed at www.pridict.it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560761">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560761" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560761">
        <p class="paperTitle">Localized synthesis of molecular chaperones sustains neuronal proteostasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560761" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560761" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alecki, C.; Rizwan, J.; Le, P.; Jacob-Tomas, S.; Xu, J. S. M.; Minotti, S.; Wu, T.; Durham, H.; Yeo, G. W.; Vera, M.</p>
        <p class="info">Score: 11.6, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560761' target='https://doi.org/10.1101/2023.10.03.560761'> 10.1101/2023.10.03.560761</a></p>
        <p class="abstract">Neurons are challenged to maintain proteostasis in neuronal projections, particularly with the physiological stress at synapses to support intercellular communication underlying important functions such as memory and movement control. Proteostasis is maintained through regulated protein synthesis and degradation and chaperone-assisted protein folding. Using high-resolution fluorescent microscopy, we discovered that neurons localize a subset of chaperone mRNAs to their dendrites, particularly more proximal regions, and increase this asymmetric localization following proteotoxic stress through microtubule-based transport from the soma. The most abundant chaperone mRNA in dendrites encodes the constitutive heat shock protein 70, HSPA8. Proteotoxic stress in cultured neurons, induced by inhibiting proteasome activity or inducing oxidative stress, enhanced transport of Hspa8 mRNAs to dendrites and the percentage of mRNAs engaged in translation on mono and polyribosomes. Knocking down the ALS-related protein Fused in Sarcoma (FUS) and a dominant mutation in the heterogenous nuclear ribonucleoprotein A2/B1 (HNRNPA2B1) impaired stress-mediated localization of Hspa8 mRNA to dendrites in cultured murine motor neurons and human iPSC-derived neurons, respectively, revealing the importance of these RNA-binding proteins in maintaining proteostasis. These results reveal the increased dendritic localization and translation of the constitutive HSP70 Hspa8 mRNA as a crucial neuronal stress response to uphold proteostasis and prevent neurodegeneration.

SUMMARYLocalizing chaperones mRNAs in neuronal dendrites is a novel on-demand system to uphold proteostasis upon stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560040">
        <p class="paperTitle">A conserved class of viral RNA structures regulate translation reinitiation through dynamic ribosome interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sherlock, M.; Langeberg, C.; Segar, K.; Kieft, J. S.</p>
        <p class="info">Score: 10.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560040' target='https://doi.org/10.1101/2023.09.29.560040'> 10.1101/2023.09.29.560040</a></p>
        <p class="abstract">Certain viral RNAs encode proteins downstream of the main protein coding region, expressed through &#34;termination-reinitiation&#34; events, dependent on RNA structure. RNA elements located upstream of the first stop codon within these viral mRNAs bind the ribosome, preventing ribosome recycling and inducing reinitiation. We used bioinformatic methods to identify new examples of viral reinitiation-stimulating RNAs and experimentally verified their secondary structure and function. We determined the structure of a representative viral RNA-ribosome complex using cryoEM. 3D classification and variability analyses reveal that the viral RNA structure can sample a range of conformations while remaining tethered to the ribosome, which enabling the ribosome to find a reinitiation start site within a limited range of mRNA sequence. Evaluating the conserved features and constraints of this entire RNA class in the context of the cryoEM reconstruction provides insight into mechanisms enabling reinitiation, a translation regulation strategy employed by many other viral and eukaryotic systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560965">
        <p class="paperTitle">Broad variation in response of individual introns to splicing inhibitors in a humanized yeast strain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hunter, O.; Talkish, J.; Quick-Cleveland, J.; Igel, H.; Tan, A.; Kuersten, S.; Katzman, S.; Donohue, J. P.; Jurica, M. S.; Ares, M.</p>
        <p class="info">Score: 9.0, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560965' target='https://doi.org/10.1101/2023.10.05.560965'> 10.1101/2023.10.05.560965</a></p>
        <p class="abstract">Intron branch point (BP) recognition by the U2 snRNP is a critical step of splicing, vulnerable to recurrent cancer mutations and bacterial natural product inhibitors. The BP binds a conserved pocket in the SF3B1 (human) or Hsh155 (yeast) U2 snRNP protein. Amino acids that line this pocket affect binding of splicing inhibitors like Pladienolide-B (Plad-B), such that organisms differ in their sensitivity. To study the mechanism of splicing inhibitor action in a simplified system, we modified the naturally Plad-B resistant yeast Saccharomyces cerevisiae by changing 14 amino acids in the Hsh155 BP pocket to those from human. This humanized yeast grows normally, and splicing is largely unaffected by the mutation. Minutes after addition of Plad-B splicing is inhibited, however different introns appear inhibited to different extents. Intron-specific inhibition differences are also observed when we measure co-transcriptional splicing in Plad-B using single-molecule intron tracking (SMIT) to minimize gene-specific transcription and decay rates that cloud estimates of inhibition by standard RNA-seq. Comparison of Plad-B intron sensitivities to those of the structurally distinct inhibitor Thailanstatin-A reveals intron-specific differences in sensitivity to different compounds. This work exposes a complex relationship between binding of different members of this class of inhibitors to the spliceosome and intron-specific rates of BP recognition and catalysis. The results echo and extend observations from mammalian cells, where inhibitor studies are complicated by multi-intron gene architecture and alternative splicing. The more compact yeast system may hasten characterization of splicing inhibitors, accelerating improvements in selectivity and therapeutic efficacy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561442" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561442">
        <p class="paperTitle">The Rho GTPase Rac1 mediates exercise training adaptations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raun, S. H.; Henriquez-Olguin, C.; Frank, E.; Knudsen, J. R.; Ali, M. S.; Andersen, N. R.; Moeller, L. L. V.; Davey, J.; Qian, H.; Coelho, A.; Carl, C. S.; Voldstedlund, C. T.; Kiens, B.; Holmdahl, R.; Gregorevic, P.; Jensen, T. E.; Richter, E. A.; Sylow, L.</p>
        <p class="info">Score: 8.4, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561442' target='https://doi.org/10.1101/2023.10.08.561442'> 10.1101/2023.10.08.561442</a></p>
        <p class="abstract">Exercise training elicits tremendous health benefits; however, the molecular underpinnings are poorly understood. As one of the most regulated groups of proteins following acute exercise in human muscle, Rho GTPases are unexplored candidates for mediating the beneficial effects of exercise. The Rho GTPase Rac1 was activated during multiple exercise modalities and remained elevated hours after resistance exercise in human muscle. Inducible muscle-specific Rac1 knockout (Rac1 imKO) mice, displayed attenuated muscle protein synthesis, glycogen resynthesis and p38 MAPK signaling in recovery from contractions. Exercise training upregulated Rac1 protein content in human and mouse muscle. Overexpression of hyperactive Rac1 elevated reactive oxidant species production during exercise yet did not induce a trained muscle phenotype. In Rac1 imKO mice, the improvements in running capacity and muscle mass after exercise training were diminished. Using gain- and loss-of-function mouse models and human muscle biopsies, we identify Rac1 as a regulator of exercise training adaptions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561488">
        <p class="paperTitle">Optical ice thickness measurement in the VitroJet for time-efficient single particle structure determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henderikx, R. J. M.; Shahzad, S.; Schotman, M. J. G.; Mann, D.; Heidler, T. V.; Leahy, E.; Ashtiani, D.; Jeurissen, R. J. M.; Peters, P. J.; Sachse, C.; Beulen, B. W. A. M. M.</p>
        <p class="info">Score: 7.8, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561488' target='https://doi.org/10.1101/2023.10.09.561488'> 10.1101/2023.10.09.561488</a></p>
        <p class="abstract">In cryo-electron microscopy structure determination, embedding biomolecules in vitreous ice of optimal thickness is critical. Ice thickness assessment and selection of suitable holes for data collection are currently part of time-consuming preparatory routines performed on expensive electron microscopes. For this reason, we developed a method to determine ice thickness during sample preparation using the optical camera integrated in the VitroJet. As a result, the ice thickness of buffer-suspended holes on an EM grid can be determined faithfully in the working range relevant for single particle cryo-EM. Dependent on the thickness, the error is below &#43;- 20 nm in the range between 0 - 70 nm down to &#43;- 10 nm in the thinnest ice regions (10 - 40 nm). Single particle structures of apoferritin were determined at two distinct thicknesses of 30 nm and 70 nm to demonstrate the suitability of optical layer thickness measurements for time-efficient cryo-EM structure determination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560700">
        <p class="paperTitle">Post-ischemic inactivation of HIF prolyl hydroxylases in endothelium promotes maladaptive kidney repair by inducing glycolysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tiwari, R.; Sharma, R.; Rajendran, G.; Borkowski, G. S.; An, S. Y.; Schonfeld, M. P.; O&#39;Sullivan, J.; Schipma, M.; Zhou, Y.; Courbon, G.; David, V.; Quaggin, S. E.; Thorp, E. B.; Chandel, N. S.; Kapitsinou, P. P.</p>
        <p class="info">Score: 6.5, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560700' target='https://doi.org/10.1101/2023.10.03.560700'> 10.1101/2023.10.03.560700</a></p>
        <p class="abstract">Ischemic acute kidney injury (AKI) is common in hospitalized patients and increases the risk for chronic kidney disease (CKD). Impaired endothelial cell (EC) functions are thought to contribute in AKI to CKD transition, but the underlying mechanisms remain unclear. Here, we identify a critical role for endothelial oxygen sensing prolyl hydroxylase domain (PHD) enzymes 1-3 in regulating post-ischemic kidney repair. In renal endothelium, we observed compartment-specific differences in the expression of the three PHD isoforms in both mice and humans. We found that post-ischemic concurrent inactivation of endothelial PHD1, PHD2, and PHD3 but not PHD2 alone promoted maladaptive kidney repair characterized by exacerbated tissue injury, fibrosis, and inflammation. Single-cell RNA-seq analysis of the post-ischemic endothelial PHD1, PHD2 and PHD3 deficient (PHDTiEC) kidney revealed an endothelial glycolytic transcriptional signature, also observed in human kidneys with severe AKI. This metabolic program was coupled to upregulation of the SLC16A3 gene encoding the lactate exporter monocarboxylate transporter 4 (MCT4). Strikingly, treatment with the MCT4 inhibitor syrosingopine restored adaptive kidney repair in PHDTiEC mice. Mechanistically, MCT4 inhibition suppressed pro-inflammatory EC activation reducing monocyte-endothelial cell interaction. Our findings suggest avenues for halting AKI to CKD transition based on selectively targeting the endothelial hypoxia-driven glycolysis/MCT4 axis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560533">
        <p class="paperTitle">RNA structure modulates Cas13 activity and enables mismatch detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kimchi, O.; Larsen, B. B.; Dunkley, O. R.; te Velthuis, A. J. W.; Myhrvold, C.</p>
        <p class="info">Score: 6.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560533' target='https://doi.org/10.1101/2023.10.05.560533'> 10.1101/2023.10.05.560533</a></p>
        <p class="abstract">The RNA-targeting CRISPR nuclease Cas13 has emerged as a powerful tool for applications ranging from nucleic acid detection to transcriptome engineering and RNA imaging. Cas13 is activated by the hybridization of a CRISPR RNA (crRNA) to a complementary single-stranded RNA (ssRNA) protospacer in a target RNA. Though Cas13 is not activated by double-stranded RNA (dsRNA) in vitro, it paradoxically demonstrates robust RNA targeting in environments where the vast majority of RNAs are highly structured. Understanding Cas13&#39;s mechanism of binding and activation will be key to improving its ability to detect and perturb RNA; however, the mechanism by which Cas13 binds structured RNAs remains unknown. Here, we systematically probe the mechanism of LwaCas13a activation in response to RNA structure perturbations using a massively multiplexed screen. We find that there are two distinct sequence-independent modes by which secondary structure affects Cas13 activity: structure in the protospacer region competes with the crRNA and can be disrupted via a strand-displacement mechanism, while structure in the region 3&#39; to the protospacer has an allosteric inhibitory effect. We leverage the kinetic nature of the strand displacement process to improve Cas13-based RNA detection and enhance mismatch discrimination by up to 50-fold, enabling sequence-agnostic mutation identification at low (&lt;1%) allele frequencies. Our work sets a new standard for CRISPR-based nucleic acid detection and will enable intelligent and secondary-structure-guided target selection while also expanding the range of RNAs available for targeting with Cas13.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561113">
        <p class="paperTitle">Loss of alkyladenine DNA glycosylase alters gene expression in the developing mouse brain and leads to reduced anxiety and improved memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bordin, D. L.; Grooms, K.; Montaldo, N. P.; Martin, S. L. F.; Saetrom, P.; Samson, L. D.; Bjoras, M.; Van Loon, B.</p>
        <p class="info">Score: 5.2, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561113' target='https://doi.org/10.1101/2023.10.05.561113'> 10.1101/2023.10.05.561113</a></p>
        <p class="abstract">Neurodevelopment is a tightly coordinated process, during which the genome is exposed to spectra of endogenous agents at different stages of differentiation. Emerging evidence indicates that DNA damage is an important feature of developing brain, tightly linked to gene expression and neuronal activity. Some of the most frequent DNA damage includes changes to DNA bases, which are recognized by DNA glycosylases and repaired through base excision repair (BER) pathway. The only mammalian DNA glycosylase able to remove frequent alkylated DNA based is alkyladenine DNA glycosylase (Aag, aka Mpg). We recently demonstrated that, besides its role in DNA repair, AAG affects expression of neurodevelopmental genes in human cells. Aag was further proposed to act as reader of epigenetic marks, including 5-hydroxymethylcytosine (5hmC), in the mouse brain. Despite the potential Aag involvement in the key brain processes, the impact of Aag loss on developing brain remains unknown. Here, by using Aag knockout (Aag-/-) mice, we show that Aag absence leads to reduced DNA break levels, evident in lowered number of {gamma}H2AX foci in postnatal day 5 (P5) hippocampi. This is accompanied by changes in 5hmC signal intensity in different hippocampal regions. Transcriptome analysis of hippocampi and prefrontal cortex, at different developmental stages, indicates that lack of Aag alters gene expression, primarily of genes involved in regulation of response to stress. Across all developmental stages tested aldehyde dehydrogenase 2 (Aldh2) emerged as one of the most prominent genes deregulated in Aag-dependent manner. In line with the changes in hippocampal DNA damage levels and the gene expression, adult Aag-/- mice exhibit altered behavior, evident in decreased anxiety levels determined in the Elevated Zero Maze and increased alternations in the Elevated T Maze tests. Taken together these results suggests that Aag has functions in modulation of genome dynamics during brain development, important for animal behavior.

HighlightsO_LIAag loss results in reduced DNA damage signal in developing hippocampus;
C_LIO_LI5hmC signal intensity is perturbed in hippocampal regions of Aag-/- mice;
C_LIO_LIGene expression is altered in Aag-/- hippocampus and prefrontal cortex;
C_LIO_LIAag represses Aldh2 expression;
C_LIO_LIAag-/- mice have reduced anxiety and improved memory.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
