<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation
Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.
Score: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598
Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation
Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.
Score: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598
Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation
Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.
Score: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598
Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation\nAuthors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.\nScore: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598\nEpigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence.",
  "keywords": [
    
  ],
  "articleBody": " Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation\nAuthors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.\nScore: 20.5, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548598\nEpigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence. Nucleosomes block access to DNA methyltransferases, unless they are remodeled by DECREASE IN DNA METHYLATION1 (DDM1Lsh/HELLS), a Snf2-like master regulator of epigenetic inheritance. We show that DDM1 activity results in replacement of the transcriptional histone variant H3.3 for the replicative variant H3.1 during the cell cycle. In ddm1 mutants, DNA methylation can be restored by loss of the H3.3 chaperone HIRA, while the H3.1 chaperone CAF-1 becomes essential. The single-particle cryo-EM structure at 3.2 [A] of DDM1 with a variant nucleosome reveals direct engagement at SHL2 with histone H3.3 at or near variant residues required for assembly, as well as with the deacetylated H4 tail. An N-terminal autoinhibitory domain binds H2A variants to allow remodeling, while a disulfide bond in the helicase domain is essential for activity in vivo and in vitro. We show that differential remodeling of H3 and H2A variants in vitro reflects preferential deposition in vivo. DDM1 co-localizes with H3.1 and H3.3 during the cell cycle, and with the DNA methyltransferase MET1Dnmt1. DDM1 localization to the chromosome is blocked by H4K16 acetylation, which accumulates at DDM1 targets in ddm1 mutants, as does the sperm cell specific H3.3 variant MGH3 in pollen, which acts as a placeholder nucleosome in the germline and contributes to epigenetic inheritance.\nMechanism of target site selection by type V-K CRISPR-associated transposases\nAuthors: George, J. T.; Acree, C.; Park, J.-U.; Kong, M.; Wiegand, T.; Pignot, Y. L.; Kellogg, E. H.; Greene, E. C.; Sternberg, S. H.\nScore: 19.1, Published: 2023-07-14 DOI: 10.1101/2023.07.14.548620\nUnlike canonical CRISPR-Cas systems that rely on RNA-guided nucleases for target cleavage, CRISPR-associated transposases (CASTs) repurpose nuclease-deficient CRISPR effectors to facilitate RNA-guided transposition of large genetic payloads. Type V-K CASTs offer several potential upsides for genome engineering, due to their compact size, easy programmability, and unidirectional integration. However, these systems are substantially less accurate than type I-F CASTs, and the molecular basis for this difference has remained elusive. Here we reveal that type V-K CASTs undergo two distinct mobilization pathways with remarkably different specificities: RNA-dependent and RNA-independent transposition. Whereas RNA-dependent transposition relies on Cas12k for accurate target selection, RNA-independent integration events are untargeted and primarily driven by the local availability of TnsC filaments. The cryo-EM structure of the untargeted complex reveals a TnsB-TnsC-TniQ transpososome that encompasses two turns of a TnsC filament and otherwise resembles major architectural aspects of the Cas12k-containing transpososome. Using single-molecule experiments and genome-wide meta-analyses, we found that AT-rich sites are preferred substrates for untargeted transposition and that the TnsB transposase also imparts local specificity, which collectively determine the precise insertion site. Knowledge of these motifs allowed us to direct untargeted transposition events to specific hotspot regions of a plasmid. Finally, by exploiting TnsBs preference for on-target integration and modulating the availability of TnsC, we suppressed RNA-independent transposition events and increased type V-K CAST specificity up to 98.1%, without compromising the efficiency of on-target integration. Collectively, our results reveal the importance of dissecting target site selection mechanisms and highlight new opportunities to leverage CAST systems for accurate, kilobase-scale genome engineering applications.\nMulti-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation\nAuthors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.\nScore: 29.2, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546730\nPartial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.\nExtrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation\nAuthors: Galitsyna, A.; Bykov, N.; Ulianov, S.; Veil, M.; Gao, M.; Perevoshchikova, K. Y.; Gelfand, M. S.; Razin, S. V.; Mirny, L. A.; Onichtchouk, D.\nScore: 12.7, Published: 2023-07-15 DOI: 10.1101/2023.07.15.549120\nThe first activation of gene expression during development (zygotic genome activation, ZGA) is accompanied by massive changes in chromosome organization. The connection between these two processes remains unknown. Using Hi-C for zebrafish embryos, we found that chromosome folding starts by establishing \"fountains\", novel elements of chromosome organization, emerging selectively at enhancers upon ZGA. Using polymer simulations, we demonstrate that fountains can emerge as sites of targeted cohesin loading and require two-sided, yet desynchronized, loop extrusion. Specific loss of fountains upon loss of pioneer transcription factors that drive ZGA reveals a causal connection between enhancer activity and fountain formation. Finally, we show that fountains emerge in early Medaka and Xenopus embryos; moreover, we found cohesin-dependent fountain pattern on enhancers of mouse embryonic stem cells. Taken together, fountains are the first enhancer-specific elements of chromosome organization; they constitute starting points of chromosome folding during early development, likely serving as sites of targeted cohesin loading.\nTBP facilitates RNA Polymerase I transcription following mitosis\nAuthors: Kwan, J. Z. J.; Nguyen, T. F.; Teves, S. S.\nScore: 7.7, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548763\nThe TATA-box binding protein (TBP) is the sole transcription factor common in the initiation complexes of the three major eukaryotic RNA Polymerases (Pol I, II, and III). Decades of research have shown that TBP is essential for proper transcription by the three RNA Pols, though the emergence of TBP paralogs throughout evolution have expanded the complexity in RNA Pol initiation. We have previously shown that acute TBP depletion in mouse embryonic stem cells led to a global decrease in Pol III activity, consistent with the requirement of TBP in Pol III inititation. In contrast, Pol II transcription remained unaffected in the absence of TBP and its paralogs. In this report, we show that, in contrast to Pol II-transcribed genes, the TBP paralog TRF2 does not bind to Pol I promoters, and therefore cannot functionally replace TBP upon depletion. Importantly, acute TBP depletion has no major effect on Pol I occupancy or activity on ribosomal RNA genes, but TBP binding in mitosis leads to efficient Pol I reactivation following cell division. These findings provide a more nuanced role for TBP in Pol I transcription in murine embryonic stem cells. SignificanceThe TATA-box binding protein (TBP) is a highly conserved and essential protein in eukaryotes. Decades of in vitro and yeast research have established its role in the initiation of the three main eukaryotic RNA polymerases. However, the ability to rapidly deplete proteins in vivo is revealing more nuance in the function of TBP in mammalian cells. Using this technology, we reassess the role of TBP in RNA Polymerase I (Pol I) transcription in mouse embryonic stem cells. We find that neither TBP nor its paralog TRF2 are required for Pol I recruitment or activity, but TBP binding during mitosis promotes efficient reactivation after cell division. Overall, these findings provide new evidence into the complex function of TBP in eukaryotic transcription.\nEnhancing sensitivity and versatility of Tn5 based single cell omics\nAuthors: Seufert, I.; Sant, P.; Bauer, K.; Syed, A. P.; Rippe, K.; Mallm, J.-P.\nScore: 7.3, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548833\nThe analysis of chromatin features in single cells centers around Tn5 transposase and exploits its activity to simultaneously fragment target DNA and integrate adapter sequences of choice. This reaction provides a direct readout in the assay for transposase-accessible chromatin in single cells (scATAC-seq) to map open chromatin regions. However, a current limitation is the sparse coverage of open chromatin loci that are detected in a given single cell by droplet-based methods. Thus, enhancing Tn5 activity to improve genomic coverage of scATAC-seq or facilitating multi-omics readouts of chromatin features via Tn5 together with the transcriptome is of great interest. Here, we address these issues by optimizing scATAC-seq for an increased number of integrations per cell. In addition, we provide a protocol that combines mapping of histone modification with scRNA-seq from the same cell by targeting Tn5 to antibody-bound chromatin epitopes. Our experimental workflows improve the results obtained from the downstream data analysis and serve to better resolve epigenetic heterogeneity and transcription regulation in single cells.\nRelationship Between Domain and Function of the Yeast RNase T2, Rny1p, Which Mediates rRNA Degradation upon Starvation\nAuthors: Minami, A.; Nishi, K.; Yamada, R.; Jinnai, G.; Shima, H.; Oishi, S.; Akagawa, H.; Aono, T.; Hidaka, M.; Masaki, H.; Kuzuyama, T.; Noda, Y.; Ogawa, T.\nScore: 4.1, Published: 2023-07-15 DOI: 10.1101/2023.07.14.548783\nRNase T2 is found in almost all organisms and non-living viruses. Although enzymatic activity is simple, RNase T2 involves diverse and essential roles in each species. Saccharomyces cerevisiae RNase T2, known as Rny1p, is localized in vacuoles and mediates rRNA degradation during autophagy of ribosomes. However, whether the ribosomal degradation is selective or non-selective or whether the two pathways coexist is still being determined. We found that most ribosomes are selectively degraded, and Rsa1p involves the degradation. As described, Rny1p is vacuolar-localized; however, some are secreted outside the cell and anchored to the host cell wall. Rny1p is characteristic of having a C-terminal extension with an unknown biological role. Our biochemical analyses indicated that the C-terminal extension is not required for the rRNA degradation but for anchoring Rny1p to the cell wall. Moreover, the predicted structure of the C-terminal extension of Rny1p superimposed well with sugar-binding domain-containing proteins, suggesting that Rny1p binds the sugar chain of the cell wall using the C-terminal extension. Phylogenetic analysis indicated that fungal RNase T2 evolved independently of RNase T2 from other species. Besides, structures of C-terminal extensions of other fungal RNase T2 proteins are almost identical to that of Rny1p. RNase T2 proteins from other than fungi also have C-terminal extensions. However, their sequences, structures, and biological functions differ from fungi. As the structures of the RNase domain of eukaryotic RNase T2 proteins are almost identical, the C-terminal extension confers species-specific functions to RNase T2.\nEfficient Formation of Single-copy Human Artificial Chromosomes\nAuthors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.\nScore: 43.5, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547284\nLarge DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells. One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.\nTur1 regulates alternative TSS usage in Cryptococcus\nAuthors: Dang, T. T. V.; Maufrais, C.; Colin, J.; Moyrand, F.; Mouyna, I.; Coppee, J.-Y.; Onyishi, C. U.; Lipecka, J.; Guerrera, I. C.; May, R. C.; Janbon, G.\nScore: 2.8, Published: 2023-07-18 DOI: 10.1101/2023.07.18.549460\nAlternative transcription start site (TSS) usage regulation has been identified as a major means of gene expression regulation in metazoans. However, in fungi, its impact remains elusive as its study has thus far been restricted to model yeasts. Here, we first re-analysed TSS-seq data to define genuine TSS clusters in two species of pathogenic Cryptococcus. We identified two types of TSS clusters associated with specific DNA sequence motifs. Our analysis also revealed that alternative TSS usage regulation in response to environmental cues is widespread in Cryptococcus, altering gene expression and protein targeting. Importantly, we performed a forward genetic screen to identify a unique transcription factor (TF) named Tur1, which regulates alternative TSS (altTSS) usage genome-wide when cells switch from exponential phase to stationary phase. Tur1 has been previously shown to be essential for virulence in C. neoformans. We demonstrated here that a tur1{Delta} mutant strain is more sensitive to superoxide stress and phagocytosed more efficiently by macrophages than the wild-type (WT) strain.\nThe substrate-binding domains of the osmoregulatory ABC importer OpuA physically interact\nAuthors: Noort, van den, M.; Drougkas, P.; Paulino, C.; Poolman, B.\nScore: 2.5, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548808\nBacteria utilize various strategies to prevent internal dehydration during hypertonic stress. A common approach to countering the effects of the stress is to import compatible solutes such as glycine betaine, leading to simultaneous passive water fluxes following the osmotic gradient. OpuA from Lactococcus lactis is a type I ABC-importer that uses two substrate-binding domains (SBDs) to capture extracellular glycine betaine and deliver the substrate to the transmembrane domains for subsequent transport. OpuA senses osmotic stress via changes in the internal ionic strength and is furthermore regulated by the 2nd messenger cyclic-di-AMP. We now show, by means of solution-based single-molecule FRET and analysis with multi- parameter photon-by-photon hidden Markov modeling, that the SBDs interact in an ionic strength-dependent manner. The smFRET data are in accordance with the apparent cooperativity in transport and supported by new cryo-EM data of OpuA. We propose that physical interactions between SBDs and cooperativity in substrate delivery could be more widespread than recognized thus far.\n",
  "wordCount" : "2416",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548598" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548598">
        <p class="paperTitle">Chromatin remodeling of histone H3 variants underlies epigenetic inheritance of DNA methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, S. C.; Adams, D. W.; Ipsaro, J. J.; Cahn, J.; Lynn, J.; Kim, H.-S.; Berube, B.; Major, V.; Calarco, J. P.; LeBlanc, C.; Bhattacharjee, S.; Ramu, U.; Grimanelli, D.; Jacob, Y.; Voigt, P.; Joshua-Tor, L.; Martienssen, R. A.</p>
        <p class="info">Score: 20.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548598' target='https://doi.org/10.1101/2023.07.11.548598'> 10.1101/2023.07.11.548598</a></p>
        <p class="abstract">Epigenetic inheritance refers to the faithful replication of DNA methylation and histone modification independent of DNA sequence. Nucleosomes block access to DNA methyltransferases, unless they are remodeled by DECREASE IN DNA METHYLATION1 (DDM1Lsh/HELLS), a Snf2-like master regulator of epigenetic inheritance. We show that DDM1 activity results in replacement of the transcriptional histone variant H3.3 for the replicative variant H3.1 during the cell cycle. In ddm1 mutants, DNA methylation can be restored by loss of the H3.3 chaperone HIRA, while the H3.1 chaperone CAF-1 becomes essential. The single-particle cryo-EM structure at 3.2 [A] of DDM1 with a variant nucleosome reveals direct engagement at SHL2 with histone H3.3 at or near variant residues required for assembly, as well as with the deacetylated H4 tail. An N-terminal autoinhibitory domain binds H2A variants to allow remodeling, while a disulfide bond in the helicase domain is essential for activity in vivo and in vitro. We show that differential remodeling of H3 and H2A variants in vitro reflects preferential deposition in vivo. DDM1 co-localizes with H3.1 and H3.3 during the cell cycle, and with the DNA methyltransferase MET1Dnmt1. DDM1 localization to the chromosome is blocked by H4K16 acetylation, which accumulates at DDM1 targets in ddm1 mutants, as does the sperm cell specific H3.3 variant MGH3 in pollen, which acts as a placeholder nucleosome in the germline and contributes to epigenetic inheritance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548620">
        <p class="paperTitle">Mechanism of target site selection by type V-K CRISPR-associated transposases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: George, J. T.; Acree, C.; Park, J.-U.; Kong, M.; Wiegand, T.; Pignot, Y. L.; Kellogg, E. H.; Greene, E. C.; Sternberg, S. H.</p>
        <p class="info">Score: 19.1, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548620' target='https://doi.org/10.1101/2023.07.14.548620'> 10.1101/2023.07.14.548620</a></p>
        <p class="abstract">Unlike canonical CRISPR-Cas systems that rely on RNA-guided nucleases for target cleavage, CRISPR-associated transposases (CASTs) repurpose nuclease-deficient CRISPR effectors to facilitate RNA-guided transposition of large genetic payloads. Type V-K CASTs offer several potential upsides for genome engineering, due to their compact size, easy programmability, and unidirectional integration. However, these systems are substantially less accurate than type I-F CASTs, and the molecular basis for this difference has remained elusive. Here we reveal that type V-K CASTs undergo two distinct mobilization pathways with remarkably different specificities: RNA-dependent and RNA-independent transposition. Whereas RNA-dependent transposition relies on Cas12k for accurate target selection, RNA-independent integration events are untargeted and primarily driven by the local availability of TnsC filaments. The cryo-EM structure of the untargeted complex reveals a TnsB-TnsC-TniQ transpososome that encompasses two turns of a TnsC filament and otherwise resembles major architectural aspects of the Cas12k-containing transpososome. Using single-molecule experiments and genome-wide meta-analyses, we found that AT-rich sites are preferred substrates for untargeted transposition and that the TnsB transposase also imparts local specificity, which collectively determine the precise insertion site. Knowledge of these motifs allowed us to direct untargeted transposition events to specific hotspot regions of a plasmid. Finally, by exploiting TnsBs preference for on-target integration and modulating the availability of TnsC, we suppressed RNA-independent transposition events and increased type V-K CAST specificity up to 98.1%, without compromising the efficiency of on-target integration. Collectively, our results reveal the importance of dissecting target site selection mechanisms and highlight new opportunities to leverage CAST systems for accurate, kilobase-scale genome engineering applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546730">
        <p class="paperTitle">Multi-omics characterization of partial chemical reprogramming reveals evidence of cell rejuvenation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitchell, W.; Goeminne, L. J. E.; Tyshkovskiy, A.; Zhang, S.; Paulo, J. A.; Pierce, K. A.; Choy, A. H.; Clish, C. B.; Gygi, S. P.; Gladyshev, V. N.</p>
        <p class="info">Score: 29.2, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546730' target='https://doi.org/10.1101/2023.06.30.546730'> 10.1101/2023.06.30.546730</a></p>
        <p class="abstract">Partial reprogramming by cyclic short-term expression of Yamanaka factors holds promise for shifting cells to younger states and consequently delaying the onset of many diseases of aging. However, the delivery of transgenes and potential risk of teratoma formation present challenges for in vivo applications. Recent advances include the use of cocktails of compounds to reprogram somatic cells, but the characteristics and mechanisms of partial cellular reprogramming by chemicals remain unclear. Here, we report a multi-omics characterization of partial chemical reprogramming in fibroblasts from young and aged mice. We measured the effects of partial chemical reprogramming on the epigenome, transcriptome, proteome, phosphoproteome, and metabolome. At the transcriptome, proteome, and phosphoproteome levels, we saw widescale changes induced by this treatment, with the most notable signature being an upregulation of mitochondrial oxidative phosphorylation. Furthermore, at the metabolome level, we observed a reduction in the accumulation of aging-related metabolites. Using both transcriptomic and epigenetic clock-based analyses, we show that partial chemical reprogramming reduces the biological age of mouse fibroblasts. We demonstrate that these changes have functional impacts, as evidenced by changes in cellular respiration and mitochondrial membrane potential. Taken together, these results illuminate the potential for chemical reprogramming reagents to rejuvenate aged biological systems, and warrant further investigation into adapting these approaches for in vivo age reversal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.549120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.549120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.549120">
        <p class="paperTitle">Extrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.549120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.549120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Galitsyna, A.; Bykov, N.; Ulianov, S.; Veil, M.; Gao, M.; Perevoshchikova, K. Y.; Gelfand, M. S.; Razin, S. V.; Mirny, L. A.; Onichtchouk, D.</p>
        <p class="info">Score: 12.7, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.549120' target='https://doi.org/10.1101/2023.07.15.549120'> 10.1101/2023.07.15.549120</a></p>
        <p class="abstract">The first activation of gene expression during development (zygotic genome activation, ZGA) is accompanied by massive changes in chromosome organization. The connection between these two processes remains unknown. Using Hi-C for zebrafish embryos, we found that chromosome folding starts by establishing &#34;fountains&#34;, novel elements of chromosome organization, emerging selectively at enhancers upon ZGA. Using polymer simulations, we demonstrate that fountains can emerge as sites of targeted cohesin loading and require two-sided, yet desynchronized, loop extrusion. Specific loss of fountains upon loss of pioneer transcription factors that drive ZGA reveals a causal connection between enhancer activity and fountain formation. Finally, we show that fountains emerge in early Medaka and Xenopus embryos; moreover, we found cohesin-dependent fountain pattern on enhancers of mouse embryonic stem cells. Taken together, fountains are the first enhancer-specific elements of chromosome organization; they constitute starting points of chromosome folding during early development, likely serving as sites of targeted cohesin loading.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548763">
        <p class="paperTitle">TBP facilitates RNA Polymerase I transcription following mitosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kwan, J. Z. J.; Nguyen, T. F.; Teves, S. S.</p>
        <p class="info">Score: 7.7, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548763' target='https://doi.org/10.1101/2023.07.13.548763'> 10.1101/2023.07.13.548763</a></p>
        <p class="abstract">The TATA-box binding protein (TBP) is the sole transcription factor common in the initiation complexes of the three major eukaryotic RNA Polymerases (Pol I, II, and III). Decades of research have shown that TBP is essential for proper transcription by the three RNA Pols, though the emergence of TBP paralogs throughout evolution have expanded the complexity in RNA Pol initiation. We have previously shown that acute TBP depletion in mouse embryonic stem cells led to a global decrease in Pol III activity, consistent with the requirement of TBP in Pol III inititation. In contrast, Pol II transcription remained unaffected in the absence of TBP and its paralogs. In this report, we show that, in contrast to Pol II-transcribed genes, the TBP paralog TRF2 does not bind to Pol I promoters, and therefore cannot functionally replace TBP upon depletion. Importantly, acute TBP depletion has no major effect on Pol I occupancy or activity on ribosomal RNA genes, but TBP binding in mitosis leads to efficient Pol I reactivation following cell division. These findings provide a more nuanced role for TBP in Pol I transcription in murine embryonic stem cells.

SignificanceThe TATA-box binding protein (TBP) is a highly conserved and essential protein in eukaryotes. Decades of in vitro and yeast research have established its role in the initiation of the three main eukaryotic RNA polymerases. However, the ability to rapidly deplete proteins in vivo is revealing more nuance in the function of TBP in mammalian cells. Using this technology, we reassess the role of TBP in RNA Polymerase I (Pol I) transcription in mouse embryonic stem cells. We find that neither TBP nor its paralog TRF2 are required for Pol I recruitment or activity, but TBP binding during mitosis promotes efficient reactivation after cell division. Overall, these findings provide new evidence into the complex function of TBP in eukaryotic transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548833">
        <p class="paperTitle">Enhancing sensitivity and versatility of Tn5 based single cell omics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seufert, I.; Sant, P.; Bauer, K.; Syed, A. P.; Rippe, K.; Mallm, J.-P.</p>
        <p class="info">Score: 7.3, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548833' target='https://doi.org/10.1101/2023.07.13.548833'> 10.1101/2023.07.13.548833</a></p>
        <p class="abstract">The analysis of chromatin features in single cells centers around Tn5 transposase and exploits its activity to simultaneously fragment target DNA and integrate adapter sequences of choice. This reaction provides a direct readout in the assay for transposase-accessible chromatin in single cells (scATAC-seq) to map open chromatin regions. However, a current limitation is the sparse coverage of open chromatin loci that are detected in a given single cell by droplet-based methods. Thus, enhancing Tn5 activity to improve genomic coverage of scATAC-seq or facilitating multi-omics readouts of chromatin features via Tn5 together with the transcriptome is of great interest. Here, we address these issues by optimizing scATAC-seq for an increased number of integrations per cell. In addition, we provide a protocol that combines mapping of histone modification with scRNA-seq from the same cell by targeting Tn5 to antibody-bound chromatin epitopes. Our experimental workflows improve the results obtained from the downstream data analysis and serve to better resolve epigenetic heterogeneity and transcription regulation in single cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548783">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548783" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548783">
        <p class="paperTitle">Relationship Between Domain and Function of the Yeast RNase T2, Rny1p, Which Mediates rRNA Degradation upon Starvation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548783" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548783" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minami, A.; Nishi, K.; Yamada, R.; Jinnai, G.; Shima, H.; Oishi, S.; Akagawa, H.; Aono, T.; Hidaka, M.; Masaki, H.; Kuzuyama, T.; Noda, Y.; Ogawa, T.</p>
        <p class="info">Score: 4.1, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548783' target='https://doi.org/10.1101/2023.07.14.548783'> 10.1101/2023.07.14.548783</a></p>
        <p class="abstract">RNase T2 is found in almost all organisms and non-living viruses. Although enzymatic activity is simple, RNase T2 involves diverse and essential roles in each species. Saccharomyces cerevisiae RNase T2, known as Rny1p, is localized in vacuoles and mediates rRNA degradation during autophagy of ribosomes. However, whether the ribosomal degradation is selective or non-selective or whether the two pathways coexist is still being determined. We found that most ribosomes are selectively degraded, and Rsa1p involves the degradation. As described, Rny1p is vacuolar-localized; however, some are secreted outside the cell and anchored to the host cell wall. Rny1p is characteristic of having a C-terminal extension with an unknown biological role. Our biochemical analyses indicated that the C-terminal extension is not required for the rRNA degradation but for anchoring Rny1p to the cell wall. Moreover, the predicted structure of the C-terminal extension of Rny1p superimposed well with sugar-binding domain-containing proteins, suggesting that Rny1p binds the sugar chain of the cell wall using the C-terminal extension. Phylogenetic analysis indicated that fungal RNase T2 evolved independently of RNase T2 from other species. Besides, structures of C-terminal extensions of other fungal RNase T2 proteins are almost identical to that of Rny1p. RNase T2 proteins from other than fungi also have C-terminal extensions. However, their sequences, structures, and biological functions differ from fungi. As the structures of the RNase domain of eukaryotic RNase T2 proteins are almost identical, the C-terminal extension confers species-specific functions to RNase T2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547284">
        <p class="paperTitle">Efficient Formation of Single-copy Human Artificial Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gambogi, C. W.; Mer, E.; Brown, D. M.; Yankson, G.; Gavade, J. N.; Logsdon, G. A.; Heun, P.; Glass, J. I.; Black, B. E.</p>
        <p class="info">Score: 43.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547284' target='https://doi.org/10.1101/2023.06.30.547284'> 10.1101/2023.06.30.547284</a></p>
        <p class="abstract">Large DNA assembly methodologies underlie milestone achievements in synthetic prokaryotic and budding yeast chromosomes. While budding yeast control chromosome inheritance through [~]125 bp DNA sequence-defined centromeres, mammals and many other eukaryotes use large, epigenetic centromeres. Harnessing centromere epigenetics permits human artificial chromosome (HAC) formation but is not sufficient to avoid rampant multimerization of the initial DNA molecule upon introduction to cells. Here, we describe an approach that efficiently forms single-copy HACs. It employs a [~]750 kb construct that is sufficiently large to house the distinct chromatin types present at the inner and outer centromere, obviating the need to multimerize. Delivery to mammalian cells is streamlined by employing yeast spheroplast fusion. These developments permit faithful chromosome engineering in the context of metazoan cells.

One-Sentence SummaryA quarter century after the first human artificial chromosomes, a solution to their uncontrolled multimerization is achieved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549460" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549460">
        <p class="paperTitle">Tur1 regulates alternative TSS usage in Cryptococcus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dang, T. T. V.; Maufrais, C.; Colin, J.; Moyrand, F.; Mouyna, I.; Coppee, J.-Y.; Onyishi, C. U.; Lipecka, J.; Guerrera, I. C.; May, R. C.; Janbon, G.</p>
        <p class="info">Score: 2.8, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549460' target='https://doi.org/10.1101/2023.07.18.549460'> 10.1101/2023.07.18.549460</a></p>
        <p class="abstract">Alternative transcription start site (TSS) usage regulation has been identified as a major means of gene expression regulation in metazoans. However, in fungi, its impact remains elusive as its study has thus far been restricted to model yeasts. Here, we first re-analysed TSS-seq data to define genuine TSS clusters in two species of pathogenic Cryptococcus. We identified two types of TSS clusters associated with specific DNA sequence motifs. Our analysis also revealed that alternative TSS usage regulation in response to environmental cues is widespread in Cryptococcus, altering gene expression and protein targeting. Importantly, we performed a forward genetic screen to identify a unique transcription factor (TF) named Tur1, which regulates alternative TSS (altTSS) usage genome-wide when cells switch from exponential phase to stationary phase. Tur1 has been previously shown to be essential for virulence in C. neoformans. We demonstrated here that a tur1{Delta} mutant strain is more sensitive to superoxide stress and phagocytosed more efficiently by macrophages than the wild-type (WT) strain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548808" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548808">
        <p class="paperTitle">The substrate-binding domains of the osmoregulatory ABC importer OpuA physically interact</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noort, van den, M.; Drougkas, P.; Paulino, C.; Poolman, B.</p>
        <p class="info">Score: 2.5, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548808' target='https://doi.org/10.1101/2023.07.13.548808'> 10.1101/2023.07.13.548808</a></p>
        <p class="abstract">Bacteria utilize various strategies to prevent internal dehydration during hypertonic stress. A common approach to countering the effects of the stress is to import compatible solutes such as glycine betaine, leading to simultaneous passive water fluxes following the osmotic gradient. OpuA from Lactococcus lactis is a type I ABC-importer that uses two substrate-binding domains (SBDs) to capture extracellular glycine betaine and deliver the substrate to the transmembrane domains for subsequent transport. OpuA senses osmotic stress via changes in the internal ionic strength and is furthermore regulated by the 2nd messenger cyclic-di-AMP. We now show, by means of solution-based single-molecule FRET and analysis with multi- parameter photon-by-photon hidden Markov modeling, that the SBDs interact in an ionic strength-dependent manner. The smFRET data are in accordance with the apparent cooperativity in transport and supported by new cryo-EM data of OpuA. We propose that physical interactions between SBDs and cooperativity in substrate delivery could be more widespread than recognized thus far.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
