<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multi-locus CRISPRi targeting with a single truncated guide RNA
Authors: Moore, M. M.; Wekhande, S.; Issner, R.; Collins, A.; Liu, Y.; Javed, N.; Buenrostro, J. D.; Epstein, C. B.; Mattei, E.; Doench, J. G.; Bernstein, B. E.; Shoresh, N.; Najm, F. J.
Score: 18.9, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563306
A critical goal in functional genomics is evaluating which non-coding elements contribute to gene expression, cellular function, and disease. Functional characterization remains a challenge due to the abundance and complexity of candidate elements.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Multi-locus CRISPRi targeting with a single truncated guide RNA
Authors: Moore, M. M.; Wekhande, S.; Issner, R.; Collins, A.; Liu, Y.; Javed, N.; Buenrostro, J. D.; Epstein, C. B.; Mattei, E.; Doench, J. G.; Bernstein, B. E.; Shoresh, N.; Najm, F. J.
Score: 18.9, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563306
A critical goal in functional genomics is evaluating which non-coding elements contribute to gene expression, cellular function, and disease. Functional characterization remains a challenge due to the abundance and complexity of candidate elements." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Multi-locus CRISPRi targeting with a single truncated guide RNA
Authors: Moore, M. M.; Wekhande, S.; Issner, R.; Collins, A.; Liu, Y.; Javed, N.; Buenrostro, J. D.; Epstein, C. B.; Mattei, E.; Doench, J. G.; Bernstein, B. E.; Shoresh, N.; Najm, F. J.
Score: 18.9, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563306
A critical goal in functional genomics is evaluating which non-coding elements contribute to gene expression, cellular function, and disease. Functional characterization remains a challenge due to the abundance and complexity of candidate elements."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Multi-locus CRISPRi targeting with a single truncated guide RNA\nAuthors: Moore, M. M.; Wekhande, S.; Issner, R.; Collins, A.; Liu, Y.; Javed, N.; Buenrostro, J. D.; Epstein, C. B.; Mattei, E.; Doench, J. G.; Bernstein, B. E.; Shoresh, N.; Najm, F. J.\nScore: 18.9, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563306\nA critical goal in functional genomics is evaluating which non-coding elements contribute to gene expression, cellular function, and disease. Functional characterization remains a challenge due to the abundance and complexity of candidate elements.",
  "keywords": [
    
  ],
  "articleBody": " Multi-locus CRISPRi targeting with a single truncated guide RNA\nAuthors: Moore, M. M.; Wekhande, S.; Issner, R.; Collins, A.; Liu, Y.; Javed, N.; Buenrostro, J. D.; Epstein, C. B.; Mattei, E.; Doench, J. G.; Bernstein, B. E.; Shoresh, N.; Najm, F. J.\nScore: 18.9, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563306\nA critical goal in functional genomics is evaluating which non-coding elements contribute to gene expression, cellular function, and disease. Functional characterization remains a challenge due to the abundance and complexity of candidate elements. Here, we develop a CRISPRi- based approach for multi-locus screening of putative transcription factor binding sites with a single truncated guide. A truncated guide with hundreds of sequence match sites can reliably disrupt enhancer activity, which expands the targeting scope of CRISPRi while maintaining repressive efficacy. We screen over 13,000 possible CTCF binding sites with 24 guides at 10 nucleotides in spacer length. These truncated guides direct CRISPRi-mediated deposition of repressive H3K9me3 marks and disrupt transcription factor binding at most sequence match target sites. This approach is valuable for elucidating functional transcription factor binding motifs or other repeated genomic sequences and is easily implementable with existing tools.\nMapping RNA-Protein Interactions with Subcellular Resolution Using Colocalization CLIP\nAuthors: Yi, S.; Singh, S. S.; Rozen-Gagnon, K.; Luna, J. M.\nScore: 15.0, Published: 2023-10-26 DOI: 10.1101/2023.10.26.563984\nRNA binding proteins (RBPs) are essential for RNA metabolism and profoundly impact health and disease. The subcellular organization of RBP interaction networks with target RNAs remains largely unexplored. Here, we develop colocalization CLIP, a method that combines CrossLinking and ImmunoPrecipitation (CLIP) with proximity labeling, to explore in-depth the subcellular RNA interactions of the well-studied RNA-binding protein HuR. Using this method, we uncover HuR's dynamic and location-specific interactions with RNA, revealing alterations in sequence preferences and interactions in the nucleus, cytosol, or stress granule compartments. We uncover HuR's unique binding preferences within stress granules during arsenite stress, illuminating intricate interactions that conventional methodologies cannot capture. Overall, coCLIP provides a powerful method for revealing RBP:RNA interactions based on localization and lays the foundation for an advanced understanding of RBP models that incorporate subcellular location as a critical determinant of their functions.\nChronically disrupted sleep induces senescence in the visceral adipose tissue of C57BL/6J mice\nAuthors: Timonina, D.; Hormazabal, G. V.; Heckenbach, I.; Anderton, E.; Haky, L.; Floro, A.; Riley, R.; Kwok, R.; Breslin, S.; Ingle, H.; Tiwari, R.; Bielska, O.; Scheibye-Knudsen, M.; Kasler, H. G.; Campisi, J.; Walter, M.; Verdin, E.\nScore: 20.2, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562803\nThe role of sleep in systemic aging remains poorly understood, despite sleeps essential function in preserving overall health and the prevalence of reduced sleep quality in modern society. Although reduced sleep correlates with an elevated risk of age-related diseases in humans, the mechanisms underlying this are unclear. In this study, we established a link between sleep and aging by demonstrating that disrupting sleep in C57BL/6 mice drives cellular senescence in the visceral adipose tissue. Sleep disruption also led to increased oxidative stress and DNA damage, both recognized triggers for senescence induction. Cellular senescence is implicated in numerous age-related conditions which are associated with insufficient sleep, such as cardiovascular disease, type 2 diabetes, and chronic inflammation. Our findings identify an accumulation of senescent cells in the adipose tissue, which serves as a potential target through which disturbed sleep accelerates the aging process and elevates the risk of age-related diseases.\nExtrachromosomal circular DNA is required for yeast stress granule assembly\nAuthors: Demeshkina, N. A.; Ferre-D'Amare, A. R.\nScore: 16.9, Published: 2023-10-13 DOI: 10.1101/2023.10.12.562116\nStress granules are a key cellular response, conserved across eukaryotes, to environmental insults. They arise upon stress-induced reversible stalling of protein synthesis and redistribution of mRNAs from polysomes into these large cytoplasmic ribonucleoprotein condensates. Many cellular pathways lead to formation and turnover of stress granules. Despite research spanning four decades, biochemical analysis of stress granules remains challenging because of their complexity and dynamic nature. Hitherto, stress granules have been studied cytologically, and their composition deduced from candidate protein approaches including immunoprecipitation and proximity labeling. Now, we have directly purified stress granule cores from yeast, relying on their density and size, finding that they are discrete particles with a size distribution peaking at 135 nm. Unexpectedly, we discovered that in addition to diverse proteins and RNAs, stress granule cores contain extrachromosomal circular DNA (eccDNA). These double-stranded DNAs have been found in all eukaryotes, but there is no precedent for a role in membraneless organelle biogenesis. By repurposing CRISPR technology to target DNA in the cytoplasm, we demonstrate that both presence and accessibility of eccDNA are required for stress granule formation in live yeast cells. Our finding reveals an unrecognized link between DNA metabolism and the cytoplasmic stress response, and suggests that sequence-specific eccDNA-mRNA interactions may mediate stress granule assembly.\nThe widely used Ucp1-CreEvdr transgene elicits complex developmental and metabolic phenotypes.\nAuthors: Halurkar, M. S.; Inoue, O.; Mukherjee, R.; Bonatto Paese, C. L.; Duszynski, M.; Brugmann, S. C.; Lim, H.-W.; Sanchez-Gurmaches, J.\nScore: 7.4, Published: 2023-10-20 DOI: 10.1101/2023.10.20.563165\nBacterial artificial chromosome transgenic models, including most Cre-recombinases, enable potent interrogation of gene function in vivo but require rigorous validation as limitations emerge. Due to its high relevance to metabolic studies, we performed comprehensive analysis of the Ucp1-CreEvdr line which is widely used for brown fat research. Hemizygotes exhibited major brown and white fat transcriptomic dysregulation, indicating potential altered tissue function. Ucp1-CreEvdrhomozygotes also show high mortality, growth defects, and craniofacial abnormalities. Mapping the transgene insertion site revealed insertion in chromosome 1 accompanied by large genomic alterations disrupting several genes expressed in a range of tissues. Notably, Ucp1-CreEvdr transgene retains an extra Ucp1 gene copy that may be highly expressed under high thermogenic burden. Our multi-faceted analysis highlights a complex phenotype arising from the presence of the Ucp1-CreEvdr transgene independently of the intended genetic manipulations. Overall, comprehensive validation of transgenic mice is imperative to maximize discovery while mitigating unexpected, off-target effects. HighlightsO_LIHemizygous Ucp1-CreEvdr mice exhibit substantial brown and white fat tissue dysregulation. C_LIO_LIHomozygous Ucp1-CreEvdr mice display high mortality, growth defects, and craniofacial abnormalities. C_LIO_LIThe Ucp1-CreEvdr transgene integration resulted in major genomic disruptions affecting multiple genes. C_LIO_LIThe Ucp1-CreEvdr transgene retains a possibly functional extra Ucp1 copy. C_LI\nThe Ribosome Assembly Factor Reh1 is Released from the Polypeptide Exit Tunnel in the Pioneer Round of Translation\nAuthors: Musalgaonkar, S.; Yelland, J.; Chitale, R.; Rao, S.; Ozadam, H.; Cenik, C.; Taylor, D.; Johnson, A. W.\nScore: 6.0, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563604\nAssembly of functional ribosomal subunits and successfully delivering them to the translating pool is a prerequisite for protein synthesis and cell growth. In S. cerevisiae, the ribosome assembly factor Reh1 binds to pre-60S subunits at a late stage during their cytoplasmic maturation. Previous work shows that the C-terminus of Reh1 inserts into the polypeptide exit tunnel (PET) of the pre-60S subunit. Unlike canonical assembly factors, which associate exclusively with pre-60S subunits, we observed that Reh1 sediments with polysomes in addition to free 60S subunits. We therefore investigated the intriguing possibility that Reh1 remains associated with 60S subunits after the release of the anti-association factor Tif6 and after subunit joining. Here, we show that Reh1-bound nascent 60S subunits associate with 40S subunits to form actively translating ribosomes. Using selective ribosome profiling, we found that Reh1-bound ribosomes populate open reading frames near start codons. Reh1-bound ribosomes are also strongly enriched for initiator tRNA, indicating they are associated with early elongation events. Using single particle cryo-electron microscopy to image cycloheximide-arrested Reh1-bound 80S ribosomes, we found that Reh1-bound 80S contain A site peptidyl tRNA, P site tRNA and eIF5A indicating that Reh1 does not dissociate from 60S until early stages of translation elongation. We propose that Reh1 is displaced by the elongating peptide chain. These results identify Reh1 as the last assembly factor released from the nascent 60S subunit during its pioneer round of translation.\nAccelerated drug resistant variant discovery with an enhanced, scalable mutagenic base editor platform\nAuthors: Dorighi, K.; Zhu, A.; Fortin, J.-P.; Lo, J. H.-H.; Sudhamsu, J.; Durinck, S.; Callow, M.; Foster, S.; Haley, B.\nScore: 5.8, Published: 2023-10-26 DOI: 10.1101/2023.10.25.564011\nPersonalized cancer therapeutics bring directed treatment options to patients based on the genetic signatures of their tumors. Unfortunately, tumor genomes are remarkably adaptable, and acquired resistance to these drugs through genetic means is an all-too-frequent occurrence. Identifying mutations that promote resistance within drug-treated patient populations can be cost, resource, and time intensive. Accordingly, base editing, enabled by Cas9-deaminase domain fusions, has emerged as a promising approach for rapid, large-scale resistance variant screening in situ. We adapted and optimized a conditional activation-induced cytidine deaminase (AID)-dCas9 system, which demonstrated greater heterogeneity of edits with an expanded footprint compared to the most commonly utilized cytosine base editor, BE4. When combined with a custom sgRNA library, we were able to identify both individual and complex, compound variants in EGFR and BRAF that confer resistance to established EGFR inhibitors. This system and the developed analytical pipeline provide a simple, highly-scalable platform for cis or trans drug-modifying variant discovery and for uncovering unique insights into protein structure-function relationships.\nP116 from Mycoplasma is a self-sufficient lipid uptake and delivery machinery\nAuthors: Manger, S.; Arghittu, S. M.; Sprankel, L.; Meier-Credo, J.; Wieland, K.; Schwalm, M. P.; Bublak, D.; Knapp, S.; Langer, J.; Covino, R.; Frangakis, A. S.\nScore: 4.6, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563710\nLipid acquisition and transport are fundamental processes in all organisms, but many of the key players remain unidentified. Here, we elucidate the lipid-cycling mechanism of the Mycoplasma pneumoniae membrane protein P116. We show that P116 not only extracts lipids from its environment but also self-sufficiently deposits them into both bacterial and eukaryotic cell membranes as well as liposomes. Our structures and molecular dynamics simulation show that the N-terminal region of P116, which resembles an SMP domain, is responsible for perturbing the membrane, while a hydrophobic pocket exploits the chemical gradient to collect the lipids and the proteins dorsal side acts as a mediator of membrane directionality. Furthermore, ligand binding and growth curve assays suggest the potential for designing small molecule inhibitors targeting this essential and immunodominant protein. We show that P116 is a versatile lipid acquisition and delivery machinery that shortcuts the multi-protein pathways used by more complex organisms. Thus, our work advances the understanding of common lipid transport strategies, which may aid research into the mechanisms of more complex lipid-handling machineries.\nModularity of PRC1 Composition and Chromatin Interaction define Condensate Properties\nAuthors: Niekamp, S.; Marr, S. K.; Oei, T. A.; Subramanian, R.; Kingston, R. E.\nScore: 4.5, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564217\nPolycomb repressive complexes (PRC) play a key role in gene repression and are indispensable for proper development. Canonical PRC1 forms condensates in vitro and in cells and the ability of PRC1 to form condensates has been proposed to contribute to maintenance of repression. However, how chromatin and the various subunits of PRC1 contribute to condensation is largely unexplored. Using single-molecule imaging, we demonstrate that nucleosomal arrays and PRC1 act synergistically, reducing the critical concentration required for condensation by more than 20-fold. By reconstituting and imaging PRC1 with various subunit compositions, we find that the exact combination of PHC and CBX subunits determine the initiation, morphology, stability, and dynamics of condensates. In particular, the polymerization activity of PHC2 strongly influences condensate dynamics to promote formation of structures with distinct domains that adhere to each other but do not coalesce. Using live cell imaging, we confirmed that CBX properties are critical for condensate initiation and that PHC polymerization is important to maintain stable condensates. Together, we propose that PRC1 can fine-tune the degree and type of condensation by altering its composition which might offer important flexibility of regulatory function during different stages of development.\nMicrobiome-Derived Prion-Like Proteins and Their Potential to Trigger Cognitive Dysfunction\nAuthors: Seira Curto, J.; Dominguez Martinez, A.; Sotillo Sotillo, P.; Serrat Garcia, M.; Fernandez Gallegos, M. R.; Sanchez de Groot, N.\nScore: 4.5, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563052\nOur life is intricately connected to microorganisms through infection or symbiotic relationships. While the inter-species propagation of prion-like proteins is well-established, their presence in the microbiome and impact on the host remains largely unexplored. To address this, we conducted a systematic study integrating in silico, in vitro, and in vivo analyses, showing that 63% of the gastrointestinal tract microbiome encodes prion-like sequences. These sequences can form amyloid fibrils capable of interfering with the aggregation of the Amyloid-beta-peptide and promoting the aggregation and propagation of the Sup35 prion. Finally, when C. elegans were fed with bacteria expressing chimeras of our prion candidates, it resulted in the loss of sensory memory, reproducing the Alzheimers model phenotype. In our model, memory impairment is linked to aggregate fragmentation and its susceptibility to degradation. Taken together, these findings show that the gut microbiota serves as a potential reservoir of prion-like sequences, supporting the idea that microbial products may influence the pathogenesis of neurodegenerative diseases.\n",
  "wordCount" : "2128",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563306">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563306" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563306">
        <p class="paperTitle">Multi-locus CRISPRi targeting with a single truncated guide RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563306" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563306" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moore, M. M.; Wekhande, S.; Issner, R.; Collins, A.; Liu, Y.; Javed, N.; Buenrostro, J. D.; Epstein, C. B.; Mattei, E.; Doench, J. G.; Bernstein, B. E.; Shoresh, N.; Najm, F. J.</p>
        <p class="info">Score: 18.9, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563306' target='https://doi.org/10.1101/2023.10.20.563306'> 10.1101/2023.10.20.563306</a></p>
        <p class="abstract">A critical goal in functional genomics is evaluating which non-coding elements contribute to gene expression, cellular function, and disease. Functional characterization remains a challenge due to the abundance and complexity of candidate elements. Here, we develop a CRISPRi- based approach for multi-locus screening of putative transcription factor binding sites with a single truncated guide. A truncated guide with hundreds of sequence match sites can reliably disrupt enhancer activity, which expands the targeting scope of CRISPRi while maintaining repressive efficacy. We screen over 13,000 possible CTCF binding sites with 24 guides at 10 nucleotides in spacer length. These truncated guides direct CRISPRi-mediated deposition of repressive H3K9me3 marks and disrupt transcription factor binding at most sequence match target sites. This approach is valuable for elucidating functional transcription factor binding motifs or other repeated genomic sequences and is easily implementable with existing tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563984">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563984" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563984">
        <p class="paperTitle">Mapping RNA-Protein Interactions with Subcellular Resolution Using Colocalization CLIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563984" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563984" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yi, S.; Singh, S. S.; Rozen-Gagnon, K.; Luna, J. M.</p>
        <p class="info">Score: 15.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563984' target='https://doi.org/10.1101/2023.10.26.563984'> 10.1101/2023.10.26.563984</a></p>
        <p class="abstract">RNA binding proteins (RBPs) are essential for RNA metabolism and profoundly impact health and disease. The subcellular organization of RBP interaction networks with target RNAs remains largely unexplored. Here, we develop colocalization CLIP, a method that combines CrossLinking and ImmunoPrecipitation (CLIP) with proximity labeling, to explore in-depth the subcellular RNA interactions of the well-studied RNA-binding protein HuR. Using this method, we uncover HuR&#39;s dynamic and location-specific interactions with RNA, revealing alterations in sequence preferences and interactions in the nucleus, cytosol, or stress granule compartments. We uncover HuR&#39;s unique binding preferences within stress granules during arsenite stress, illuminating intricate interactions that conventional methodologies cannot capture. Overall, coCLIP provides a powerful method for revealing RBP:RNA interactions based on localization and lays the foundation for an advanced understanding of RBP models that incorporate subcellular location as a critical determinant of their functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562803">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562803" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562803">
        <p class="paperTitle">Chronically disrupted sleep induces senescence in the visceral adipose tissue of C57BL/6J mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562803" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562803" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timonina, D.; Hormazabal, G. V.; Heckenbach, I.; Anderton, E.; Haky, L.; Floro, A.; Riley, R.; Kwok, R.; Breslin, S.; Ingle, H.; Tiwari, R.; Bielska, O.; Scheibye-Knudsen, M.; Kasler, H. G.; Campisi, J.; Walter, M.; Verdin, E.</p>
        <p class="info">Score: 20.2, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562803' target='https://doi.org/10.1101/2023.10.17.562803'> 10.1101/2023.10.17.562803</a></p>
        <p class="abstract">The role of sleep in systemic aging remains poorly understood, despite sleeps essential function in preserving overall health and the prevalence of reduced sleep quality in modern society. Although reduced sleep correlates with an elevated risk of age-related diseases in humans, the mechanisms underlying this are unclear. In this study, we established a link between sleep and aging by demonstrating that disrupting sleep in C57BL/6 mice drives cellular senescence in the visceral adipose tissue. Sleep disruption also led to increased oxidative stress and DNA damage, both recognized triggers for senescence induction. Cellular senescence is implicated in numerous age-related conditions which are associated with insufficient sleep, such as cardiovascular disease, type 2 diabetes, and chronic inflammation. Our findings identify an accumulation of senescent cells in the adipose tissue, which serves as a potential target through which disturbed sleep accelerates the aging process and elevates the risk of age-related diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562116" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562116">
        <p class="paperTitle">Extrachromosomal circular DNA is required for yeast stress granule assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demeshkina, N. A.; Ferre-D&#39;Amare, A. R.</p>
        <p class="info">Score: 16.9, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562116' target='https://doi.org/10.1101/2023.10.12.562116'> 10.1101/2023.10.12.562116</a></p>
        <p class="abstract">Stress granules are a key cellular response, conserved across eukaryotes, to environmental insults. They arise upon stress-induced reversible stalling of protein synthesis and redistribution of mRNAs from polysomes into these large cytoplasmic ribonucleoprotein condensates. Many cellular pathways lead to formation and turnover of stress granules. Despite research spanning four decades, biochemical analysis of stress granules remains challenging because of their complexity and dynamic nature. Hitherto, stress granules have been studied cytologically, and their composition deduced from candidate protein approaches including immunoprecipitation and proximity labeling. Now, we have directly purified stress granule cores from yeast, relying on their density and size, finding that they are discrete particles with a size distribution peaking at 135 nm. Unexpectedly, we discovered that in addition to diverse proteins and RNAs, stress granule cores contain extrachromosomal circular DNA (eccDNA). These double-stranded DNAs have been found in all eukaryotes, but there is no precedent for a role in membraneless organelle biogenesis. By repurposing CRISPR technology to target DNA in the cytoplasm, we demonstrate that both presence and accessibility of eccDNA are required for stress granule formation in live yeast cells. Our finding reveals an unrecognized link between DNA metabolism and the cytoplasmic stress response, and suggests that sequence-specific eccDNA-mRNA interactions may mediate stress granule assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563165">
        <p class="paperTitle">The widely used Ucp1-CreEvdr transgene elicits complex developmental and metabolic phenotypes.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Halurkar, M. S.; Inoue, O.; Mukherjee, R.; Bonatto Paese, C. L.; Duszynski, M.; Brugmann, S. C.; Lim, H.-W.; Sanchez-Gurmaches, J.</p>
        <p class="info">Score: 7.4, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563165' target='https://doi.org/10.1101/2023.10.20.563165'> 10.1101/2023.10.20.563165</a></p>
        <p class="abstract">Bacterial artificial chromosome transgenic models, including most Cre-recombinases, enable potent interrogation of gene function in vivo but require rigorous validation as limitations emerge. Due to its high relevance to metabolic studies, we performed comprehensive analysis of the Ucp1-CreEvdr line which is widely used for brown fat research. Hemizygotes exhibited major brown and white fat transcriptomic dysregulation, indicating potential altered tissue function. Ucp1-CreEvdrhomozygotes also show high mortality, growth defects, and craniofacial abnormalities. Mapping the transgene insertion site revealed insertion in chromosome 1 accompanied by large genomic alterations disrupting several genes expressed in a range of tissues. Notably, Ucp1-CreEvdr transgene retains an extra Ucp1 gene copy that may be highly expressed under high thermogenic burden. Our multi-faceted analysis highlights a complex phenotype arising from the presence of the Ucp1-CreEvdr transgene independently of the intended genetic manipulations. Overall, comprehensive validation of transgenic mice is imperative to maximize discovery while mitigating unexpected, off-target effects.

HighlightsO_LIHemizygous Ucp1-CreEvdr mice exhibit substantial brown and white fat tissue dysregulation.
C_LIO_LIHomozygous Ucp1-CreEvdr mice display high mortality, growth defects, and craniofacial abnormalities.
C_LIO_LIThe Ucp1-CreEvdr transgene integration resulted in major genomic disruptions affecting multiple genes.
C_LIO_LIThe Ucp1-CreEvdr transgene retains a possibly functional extra Ucp1 copy.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563604">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563604" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563604">
        <p class="paperTitle">The Ribosome Assembly Factor Reh1 is Released from the Polypeptide Exit Tunnel in the Pioneer Round of Translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563604" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563604" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Musalgaonkar, S.; Yelland, J.; Chitale, R.; Rao, S.; Ozadam, H.; Cenik, C.; Taylor, D.; Johnson, A. W.</p>
        <p class="info">Score: 6.0, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563604' target='https://doi.org/10.1101/2023.10.23.563604'> 10.1101/2023.10.23.563604</a></p>
        <p class="abstract">Assembly of functional ribosomal subunits and successfully delivering them to the translating pool is a prerequisite for protein synthesis and cell growth. In S. cerevisiae, the ribosome assembly factor Reh1 binds to pre-60S subunits at a late stage during their cytoplasmic maturation. Previous work shows that the C-terminus of Reh1 inserts into the polypeptide exit tunnel (PET) of the pre-60S subunit. Unlike canonical assembly factors, which associate exclusively with pre-60S subunits, we observed that Reh1 sediments with polysomes in addition to free 60S subunits. We therefore investigated the intriguing possibility that Reh1 remains associated with 60S subunits after the release of the anti-association factor Tif6 and after subunit joining. Here, we show that Reh1-bound nascent 60S subunits associate with 40S subunits to form actively translating ribosomes. Using selective ribosome profiling, we found that Reh1-bound ribosomes populate open reading frames near start codons. Reh1-bound ribosomes are also strongly enriched for initiator tRNA, indicating they are associated with early elongation events. Using single particle cryo-electron microscopy to image cycloheximide-arrested Reh1-bound 80S ribosomes, we found that Reh1-bound 80S contain A site peptidyl tRNA, P site tRNA and eIF5A indicating that Reh1 does not dissociate from 60S until early stages of translation elongation. We propose that Reh1 is displaced by the elongating peptide chain. These results identify Reh1 as the last assembly factor released from the nascent 60S subunit during its pioneer round of translation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564011">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564011" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564011">
        <p class="paperTitle">Accelerated drug resistant variant discovery with an enhanced, scalable mutagenic base editor platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564011" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564011" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dorighi, K.; Zhu, A.; Fortin, J.-P.; Lo, J. H.-H.; Sudhamsu, J.; Durinck, S.; Callow, M.; Foster, S.; Haley, B.</p>
        <p class="info">Score: 5.8, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564011' target='https://doi.org/10.1101/2023.10.25.564011'> 10.1101/2023.10.25.564011</a></p>
        <p class="abstract">Personalized cancer therapeutics bring directed treatment options to patients based on the genetic signatures of their tumors. Unfortunately, tumor genomes are remarkably adaptable, and acquired resistance to these drugs through genetic means is an all-too-frequent occurrence. Identifying mutations that promote resistance within drug-treated patient populations can be cost, resource, and time intensive. Accordingly, base editing, enabled by Cas9-deaminase domain fusions, has emerged as a promising approach for rapid, large-scale resistance variant screening in situ. We adapted and optimized a conditional activation-induced cytidine deaminase (AID)-dCas9 system, which demonstrated greater heterogeneity of edits with an expanded footprint compared to the most commonly utilized cytosine base editor, BE4. When combined with a custom sgRNA library, we were able to identify both individual and complex, compound variants in EGFR and BRAF that confer resistance to established EGFR inhibitors. This system and the developed analytical pipeline provide a simple, highly-scalable platform for cis or trans drug-modifying variant discovery and for uncovering unique insights into protein structure-function relationships.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563710">
        <p class="paperTitle">P116 from Mycoplasma is a self-sufficient lipid uptake and delivery machinery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manger, S.; Arghittu, S. M.; Sprankel, L.; Meier-Credo, J.; Wieland, K.; Schwalm, M. P.; Bublak, D.; Knapp, S.; Langer, J.; Covino, R.; Frangakis, A. S.</p>
        <p class="info">Score: 4.6, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563710' target='https://doi.org/10.1101/2023.10.24.563710'> 10.1101/2023.10.24.563710</a></p>
        <p class="abstract">Lipid acquisition and transport are fundamental processes in all organisms, but many of the key players remain unidentified. Here, we elucidate the lipid-cycling mechanism of the Mycoplasma pneumoniae membrane protein P116. We show that P116 not only extracts lipids from its environment but also self-sufficiently deposits them into both bacterial and eukaryotic cell membranes as well as liposomes. Our structures and molecular dynamics simulation show that the N-terminal region of P116, which resembles an SMP domain, is responsible for perturbing the membrane, while a hydrophobic pocket exploits the chemical gradient to collect the lipids and the proteins dorsal side acts as a mediator of membrane directionality. Furthermore, ligand binding and growth curve assays suggest the potential for designing small molecule inhibitors targeting this essential and immunodominant protein. We show that P116 is a versatile lipid acquisition and delivery machinery that shortcuts the multi-protein pathways used by more complex organisms. Thus, our work advances the understanding of common lipid transport strategies, which may aid research into the mechanisms of more complex lipid-handling machineries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564217">
        <p class="paperTitle">Modularity of PRC1 Composition and Chromatin Interaction define Condensate Properties</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Niekamp, S.; Marr, S. K.; Oei, T. A.; Subramanian, R.; Kingston, R. E.</p>
        <p class="info">Score: 4.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564217' target='https://doi.org/10.1101/2023.10.26.564217'> 10.1101/2023.10.26.564217</a></p>
        <p class="abstract">Polycomb repressive complexes (PRC) play a key role in gene repression and are indispensable for proper development. Canonical PRC1 forms condensates in vitro and in cells and the ability of PRC1 to form condensates has been proposed to contribute to maintenance of repression. However, how chromatin and the various subunits of PRC1 contribute to condensation is largely unexplored. Using single-molecule imaging, we demonstrate that nucleosomal arrays and PRC1 act synergistically, reducing the critical concentration required for condensation by more than 20-fold. By reconstituting and imaging PRC1 with various subunit compositions, we find that the exact combination of PHC and CBX subunits determine the initiation, morphology, stability, and dynamics of condensates. In particular, the polymerization activity of PHC2 strongly influences condensate dynamics to promote formation of structures with distinct domains that adhere to each other but do not coalesce. Using live cell imaging, we confirmed that CBX properties are critical for condensate initiation and that PHC polymerization is important to maintain stable condensates. Together, we propose that PRC1 can fine-tune the degree and type of condensation by altering its composition which might offer important flexibility of regulatory function during different stages of development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563052">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563052" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563052">
        <p class="paperTitle">Microbiome-Derived Prion-Like Proteins and Their Potential to Trigger Cognitive Dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563052" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563052" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seira Curto, J.; Dominguez Martinez, A.; Sotillo Sotillo, P.; Serrat Garcia, M.; Fernandez Gallegos, M. R.; Sanchez de Groot, N.</p>
        <p class="info">Score: 4.5, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563052' target='https://doi.org/10.1101/2023.10.19.563052'> 10.1101/2023.10.19.563052</a></p>
        <p class="abstract">Our life is intricately connected to microorganisms through infection or symbiotic relationships. While the inter-species propagation of prion-like proteins is well-established, their presence in the microbiome and impact on the host remains largely unexplored. To address this, we conducted a systematic study integrating in silico, in vitro, and in vivo analyses, showing that 63% of the gastrointestinal tract microbiome encodes prion-like sequences. These sequences can form amyloid fibrils capable of interfering with the aggregation of the Amyloid-beta-peptide and promoting the aggregation and propagation of the Sup35 prion. Finally, when C. elegans were fed with bacteria expressing chimeras of our prion candidates, it resulted in the loss of sensory memory, reproducing the Alzheimers model phenotype. In our model, memory impairment is linked to aggregate fragmentation and its susceptibility to degradation. Taken together, these findings show that the gut microbiota serves as a potential reservoir of prion-like sequences, supporting the idea that microbial products may influence the pathogenesis of neurodegenerative diseases.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
