<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency
Authors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.
Score: 53.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557729
In mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)- deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency
Authors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.
Score: 53.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557729
In mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)- deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency
Authors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.
Score: 53.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557729
In mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)- deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency\nAuthors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.\nScore: 53.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557729\nIn mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)- deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions.",
  "keywords": [
    
  ],
  "articleBody": " DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency\nAuthors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.\nScore: 53.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557729\nIn mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)- deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions. To formally assess how 5mC shapes the H3K27me3 landscape, we profiled the epigenome of naive and differentiated cells in the presence and absence of the DNA methylation machinery. Surprisingly, we found that 5mC accumulation is not required to restrict most H3K27me3 domains. We went on to show that this 5mC-independent H3K27me3 restriction is mediated by aberrant expression of the PRC2 antagonist Ezhip. At the regions where 5mC appears to genuinely supplant H3K27me3, we identified 68 candidate genes that appeared to require 5mC deposition and/or H3K27me3 depletion for their activation in differentiated cells. Employing site-directed epigenome editing to directly modulate 5mC levels, we demonstrated that 5mC deposition is sufficient to antagonize H3K27me3 deposition and confer gene activation at individual candidates. Altogether, we systematically measured the antagonistic interplay between 5mC and H3K27me3 in a system that recapitulates early embryonic dynamics. Our results suggest that H3K27me3 restraint depends on 5mC, both directly and indirectly. This study also reveals a non-canonical role of 5mC in gene activation, which may be important not only for normal development but also for cancer progression, as oncogenic cells frequently exhibit dynamic replacement of 5mC for H3K27me3 and vice versa.\nDeep quantitative glycoproteomics reveals gut microbiome induced remodeling of the brain glycoproteome\nAuthors: Potel, C. M.; Burtscher, M. L.; Garrido-Rodriguez, M.; Brauer-Nikonow, A.; Becher, I.; Typas, A.; Zimmermann, M.; Savitski, M. M.\nScore: 24.6, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557529\nHighlightsO_LIHigh throughput glycoproteomics method with multiplexed quantification C_LIO_LI25-fold improvement of the mouse brain glycoproteome coverage C_LIO_LIStructural features dictate level of glycosite micro-heterogeneity C_LIO_LIGut microbiome composition extensively impacts the brain glycoproteome C_LIO_LIModulation of glycosylation is site-specific C_LI Protein glycosylation is a highly diverse post-translational modification, modulating key cellular processes such as cell signaling, adhesion and cell-cell interactions. Its deregulation has been associated with various pathologies, including cancer and neurological diseases. Methods capable of quantifying glycosylation dynamics are essential to start unraveling the biological functions of protein glycosylation. Here we present Deep Quantitative Glycoprofiling (DQGlyco), a method that combines high-throughput sample preparation, high-sensitivity detection, and precise multiplexed quantification of protein glycosylation. We used DQGlyco to profile the mouse brain glycoproteome, in which we identify 158,972 and 15,056 unique N- and O-glycopeptides localized on 3,199 and 2,365 glycoproteins, respectively - this amounts to 25-fold more glycopeptides identified compared to previous studies. We observed extensive heterogeneity of glycoforms and determined their functional and structural preferences. The presence of a defined gut microbiota resulted in extensive remodeling of the brain glycoproteome when compared to that of germ-free animals, exemplifying how the gut microbiome may affect brain protein functions.\nStructure of γ-secretase (PSEN1/APH-1B) in complex with Aβ46 provides insights into amyloid-β processing and modulation by the APH-1B isoform\nAuthors: Odorcic, I.; Soliman, M. B. H.; Lismont, S.; Chavez-Gutierrez, L.; Efremov, R. G.\nScore: 15.0, Published: 2023-09-09 DOI: 10.1101/2023.09.09.556900\nDeposition of amyloid-{beta} (A{beta}) peptides in the brain is a hallmark of Alzheimers disease. A{beta}s are generated through sequential proteolysis of the amyloid precursor protein by the {gamma}-secretase complexes (GSECs). A{beta} peptide length, which is modulated by the Presenilin (PSEN) and APH-1 subunits of GSEC, is critical for Alzheimers pathogenesis. Despite high relevance, mechanistic understanding of the proteolysis of A{beta}, and its modulation by APH-1, remain incomplete. Here, we report cryo-EM structures of human GSEC (PSEN1/APH-1B) reconstituted into lipid nanodiscs in apo form and in complex with the intermediate A{beta}46 substrate. We found a divergent APH-1 loop to be involved with PSEN1 in substrate-binding-induced concerted rearrangements. Upstream the catalytic site, A{beta}46 structure is similar to the endopeptidase substrates and is stabilised by polar interactions including a previously unseen interaction with PSEN1 loop1. The hybrid {beta}-sheet was not observed downstream the catalytic site.\nTAX1BP1 recruits ATG9 vesicles through SCAMP3 binding\nAuthors: Hama, Y.; Kurikawa, Y.; Matsui, T.; Mizushima, N.; Yamamoto, H.\nScore: 13.0, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553817\nMacroautophagy is a cellular process that delivers cytoplasmic material to lysosomes for degradation via autophagosomes. It often involves the selective degradation of ubiquitinated proteins. During selective macroautophagy, five ubiquitin-binding adaptors, p62, NBR1, OPTN, NDP52, and TAX1BP1, form biomolecular condensates with ubiquitinated proteins and recruit ATG9 vesicles, which serve as the initial membrane source required for autophagosome formation. However, the molecular details underlying the cargo/adaptor-dependent recruitment of ATG9 vesicles remain unclear. Here, we show that ATG9 vesicles are recruited by three cargo adaptors: TAX1BP1, NBR1, and OPTN. We also find that ATG9A itself is not the determinant for recruitment by these cargo adaptors, and that TAX1BP1-dependent ATG9 vesicle recruitment is mediated by SCAMP3, a transmembrane protein on the ATG9 vesicles, through binding to the coiled-coil 1 domain of TAX1BP1. These findings provide mechanistic insights into the cargo/adaptor-dependent assembly of ATG9 vesicles in mammals.\nMolecular model of a bacterial flagellar motor in situ reveals a \"parts-list\" of protein adaptations to increase torque\nAuthors: Drobnic, T.; Cohen, E. J.; Alzheimer, M.; Froschauer, K.; Svensson, S.; Singh, N.; Garg, S. G.; Henderson, L.; Umrekar, T.; Nans, A.; Ribardo, D.; Hochberg, G.; Hendrixson, D. R.; Sharma, C. M.; Rosenthal, P.; Beeby, M.\nScore: 11.8, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556779\nOne hurdle to understanding how molecular machines function and evolve is our inability to see their structures in situ. Here we describe a minicell system that enables in situ cryogenic electron microscopy imaging and single particle analysis to probe the mechanisms and evolution of an iconic molecular machine, the bacterial flagellar motor, which spins a helical propeller for bacterial propulsion. Innovations in sample preparation and imaging enabled resolutions sufficient to build an in situ molecular model of the C. jejuni flagellar motor. Our results provide unprecedented insights into the in situ context of flagellar motors, highlight origins of recruited components involved in the unusually high torque of the C. jejuni motor, identify previously unknown components, and reveal corresponding modifications of core components. We also visualise structures involved in torque generation and secretion previously recalcitrant to structure determination. This technique will be of broad applicability to other large membrane-residing protein complexes. Note that this manuscript has a sibling manuscript titled \"Evolution of a large periplasmic disk in Campylobacterota flagella facilitated efficient motility alongside autoagglutination\" that dissects the function of the large disk described in this manuscript.\nSpatiotemporal Proteomics Deciphers Functional Selectivity of EGFR ligands\nAuthors: Eguchi, A.; Emdal, K. B.; Ye, Z.; Guzman, U. H.; Olsen, J. V.\nScore: 9.3, Published: 2023-09-14 DOI: 10.1101/2023.09.13.557663\nThe epidermal growth factor receptor (EGFR) induces different signaling outputs depending on ligand identity and biological context. This phenomenon is known as functional selectivity, but the underlying molecular mechanisms remain elusive. Here, we investigated this on a global scale and time-resolved by high-throughput multilayered proteomics integrating dynamic changes in the EGFR interaction network by proximity biotinylation using EGFR-TurboID, phosphoproteome, and proteome in response to stimulation with the six highest-affinity EGFR ligands. We obtained comprehensive temporal profiles of protein recruitment and phosphosite changes pinpointing signaling proteins differentially regulated by the six ligands with key impact on EGFR endocytic fate, e.g. degradation or recycling. Specifically, the Epsin family protein, Clint1 was identified to control the endocytic trafficking of EGFR towards degradation. Moreover, we characterized the protein interaction selectivity of EGFR C-terminally phosphorylated tyrosine residues using a panel of tyrosine mutated constructs showing STAT5 specificity for EGFR Y1173. These data provide a comprehensive resource deciphering functional selectivity of EGFR signaling to support discovery of novel drug targets.\nA brown fat-enriched adipokine, ASRA, is a leptin receptor antagonist that stimulates appetite\nAuthors: Huang, L.; Liu, P.; Du, Y.; Pan, D.; Lee, A.; Wolfe, S.; Wang, Y.-X.\nScore: 8.2, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557454\nThe endocrine control of food intake remains incompletely understood, and whether the leptin receptor-mediated anorexigenic pathway in the hypothalamus is negatively regulated by a humoral factor is unknown. Here we identify an appetite-stimulating factor - ASRA - that acts as a leptin receptor antagonist. ASRA encodes an 8 kD protein that is abundantly and selectively expressed in adipose tissue and to a lesser extent, in liver, and is upregulated during fasting and cold. ASRA protein associates with autophagosomes and its secretion is induced by energy deficiency. Overexpression of ASRA in mice attenuates leptin receptor signaling leading to elevated blood glucose and development of severe hyperphagic obesity, whereas either adipose- or liver-specific ASRA knockout mice display increased leptin sensitivity, improved glucose homeostasis, reduced food intake, and resistance to high fat diet-induced obesity. Furthermore, ASRA is indispensable for cold-evoked feeding response. Recombinant ASRA (rASRA) protein binds to leptin receptor and suppresses leptin receptor signaling in cultured cells. In vivo, rASRA promotes food intake and increases blood glucose in a leptin receptor signaling-dependent manner. Our studies collectively show that ASRA, acting as a peripheral signal of energy deficit, stimulates appetite and regulates glucose metabolism by antagonizing leptin receptor signaling, thus revealing a previously unknown endocrine mechanism that has important implications for our understanding of leptin resistance.\nThe intrinsically disordered region of the E3 ubiquitin ligase TRIP12 induces the formation of chromatin condensates and interferes with DNA damage response.\nAuthors: Vargas, C.; Brunet, M.; Ricard, A.; varry, d.; Muzzopappa, F.; Erdel, F.; Labrousee, G.; Hanoun, N.; Ligat, L.; Farce, M.; Stella, A.; Burlet-Schiltz, O.; Laurell, H.; Bery, N.; Cordelier, P.; dufresne, m.; torrisani, j.\nScore: 7.0, Published: 2023-09-13 DOI: 10.1101/2023.09.13.556486\nChromatin compaction is crucial for the faithful expression and integrity of the genome. Although largely studied, proteins and mechanisms that control the chromatin compaction are not entirely discovered. We previously showed that the nuclear HECT-type E3 ubiquitin ligase Thyroid hormone Receptor Interacting Protein 12 (TRIP12) is tightly associated to chromatin. As TRIP12 is overexpressed in several types of cancers, we explored herein the consequences of a TRIP12 overexpression on chromatin homeostasis. First, we established the TRIP12 proxisome and unveiled its pleiotropic role in chromatin regulation. Second, we demonstrated that TRIP12 overexpression leads to the formation of chromatin condensates enriched in heterochromatin marks via its intrinsically disordered region (IDR). We further discovered that the formation of TRIP12-mediated chromatin condensates is highly dynamic and driven by a mechanism of phase separation. Chromatin condensate formation depends on the TRIP12 concentration, the length of the TRIP12-IDR and relies on electrostatic interactions. We found that the formation of TRIP12 mediated-condensates alters cell cycle progression, genome accessibility, transcription as well as DNA damage response by inhibiting the accumulation of Mediator of DNA Damage Checkpoint 1 (MDC1). Altogether, this study reveals a novel dynamic role for TRIP12 in chromatin compaction independently of its ubiquitin ligase activity with important consequences on cellular homeostasis. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=155 SRC=\"FIGDIR/small/556486v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@ba5d55org.highwire.dtl.DTLVardef@14578d9org.highwire.dtl.DTLVardef@fb85org.highwire.dtl.DTLVardef@80e92d_HPS_FORMAT_FIGEXP M_FIG C_FIG\nParallel G-quadruplex folds via multiple paths involving G-tract stacking and structuring from coil ensemble\nAuthors: Pokorna, P.; Mlynsky, V.; Bussi, G.; Sponer, J.; Stadlbauer, P.\nScore: 6.6, Published: 2023-09-09 DOI: 10.1101/2023.09.09.556957\nG-quadruplexes (G4s) are non-canonical nucleic acid structures that fold through complex processes. Characterization of G4 folding landscape contributes to comprehending G4 roles in gene regulation but is challenging for experiments and computations. Here we investigate the folding of a three-quartet parallel DNA G4 with (GGGA)3GGG sequence using all-atom explicit-solvent enhanced-sampling molecular dynamics (MD) simulations. We suggest an early formation of guanine stacks in the G-tracts, which behave as semi-rigid blocks in the folding process. The parallel G4 folding is initiated by the formation of a collapsed compact coil-like ensemble. Structuring of the G4 from the coil then proceeds via various cross-like, hairpin, slip-stranded, and two-quartet ensembles and can bypass the G-triplex structure. While parallel G-hairpins are extremely unstable when isolated, they are more stable inside the coil structure. Folding of parallel G4 does not appear to involve any salient intermediates and, instead, it is an extremely multiple-pathway process. On the methodology side, we show that the AMBER DNA force field predicts the folded G4 to be less stable than the unfolded ensemble, uncovering substantial force-field issues. Overall, we provide unique atomistic insights into the folding landscape of parallel-stranded G4 but also reveal limitations of the state-of-the-art MD techniques.\nUncoupling the TFIIH Core and Kinase Modules Leads To Unregulated RNA Polymerase II CTD Serine 5 Phosphorylation\nAuthors: Giordano, G.; Buratowski, R.; Jeronimo, C.; Poitras, C.; Robert, F.; Buratowski, S.\nScore: 6.2, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557269\nTFIIH is an essential transcription initiation factor for RNA polymerase II (RNApII). This multi-subunit complex comprises two modules that are physically linked by the subunit Tfb3 (MAT1 in metazoans). The TFIIH Core Module, with two DNA-dependent ATPases and several additional subunits, promotes DNA unwinding. The TFIIH Kinase Module phosphorylates Serine 5 of the C-terminal domain (CTD) of RNApII subunit Rpb1, a modification that coordinates exchange of initiation and early elongation factors. While it is not obvious why these two disparate activities are bundled into one factor, the connection may provide temporal coordination during early initiation. Here we show that Tfb3 can be split into two parts to uncouple the TFIIH modules. The resulting cells grow slower than normal, but are viable. Chromatin immunoprecipitation of the split TFIIH shows that the Core Module, but not the Kinase, is properly recruited to promoters. Instead of the normal promoter-proximal peak, high CTD Serine 5 phosphorylation is seen throughout transcribed regions. Therefore, coupling the TFIIH modules is necessary to localize and limit CTD kinase activity to early stages of transcription. These results are consistent with the idea that the two TFIIH modules began as independent functional entities that became connected by Tfb3 during early eukaryotic evolution.\n",
  "wordCount" : "2297",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557729" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557729">
        <p class="paperTitle">DNA methylation shapes the Polycomb landscape during the exit from naïve pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Albert, J. R.; Urli, T.; Monteagudo-Sanchez, A.; Le Breton, A.; Sultanova, A.; David, A.; Schulz, M.; Greenberg, M. V. C.</p>
        <p class="info">Score: 53.8, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557729' target='https://doi.org/10.1101/2023.09.14.557729'> 10.1101/2023.09.14.557729</a></p>
        <p class="abstract">In mammals, 5 methyl-cytosine (5mC) and Polycomb Repressive Complex 2 (PRC2)- deposited histone 3 lysine 27 trimethylation (H3K27me3) are generally mutually exclusive at CpG-rich regions. As mouse embryonic stem cells exit the naive pluripotent state, there is a massive gain of 5mC coincident with a restriction of broad H3K27me3 to 5mC-free, CpG-rich regions. To formally assess how 5mC shapes the H3K27me3 landscape, we profiled the epigenome of naive and differentiated cells in the presence and absence of the DNA methylation machinery. Surprisingly, we found that 5mC accumulation is not required to restrict most H3K27me3 domains. We went on to show that this 5mC-independent H3K27me3 restriction is mediated by aberrant expression of the PRC2 antagonist Ezhip. At the regions where 5mC appears to genuinely supplant H3K27me3, we identified 68 candidate genes that appeared to require 5mC deposition and/or H3K27me3 depletion for their activation in differentiated cells. Employing site-directed epigenome editing to directly modulate 5mC levels, we demonstrated that 5mC deposition is sufficient to antagonize H3K27me3 deposition and confer gene activation at individual candidates. Altogether, we systematically measured the antagonistic interplay between 5mC and H3K27me3 in a system that recapitulates early embryonic dynamics. Our results suggest that H3K27me3 restraint depends on 5mC, both directly and indirectly. This study also reveals a non-canonical role of 5mC in gene activation, which may be important not only for normal development but also for cancer progression, as oncogenic cells frequently exhibit dynamic replacement of 5mC for H3K27me3 and vice versa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557529">
        <p class="paperTitle">Deep quantitative glycoproteomics reveals gut microbiome induced remodeling of the brain glycoproteome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Potel, C. M.; Burtscher, M. L.; Garrido-Rodriguez, M.; Brauer-Nikonow, A.; Becher, I.; Typas, A.; Zimmermann, M.; Savitski, M. M.</p>
        <p class="info">Score: 24.6, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557529' target='https://doi.org/10.1101/2023.09.13.557529'> 10.1101/2023.09.13.557529</a></p>
        <p class="abstract">HighlightsO_LIHigh throughput glycoproteomics method with multiplexed quantification
C_LIO_LI25-fold improvement of the mouse brain glycoproteome coverage
C_LIO_LIStructural features dictate level of glycosite micro-heterogeneity
C_LIO_LIGut microbiome composition extensively impacts the brain glycoproteome
C_LIO_LIModulation of glycosylation is site-specific
C_LI

Protein glycosylation is a highly diverse post-translational modification, modulating key cellular processes such as cell signaling, adhesion and cell-cell interactions. Its deregulation has been associated with various pathologies, including cancer and neurological diseases. Methods capable of quantifying glycosylation dynamics are essential to start unraveling the biological functions of protein glycosylation. Here we present Deep Quantitative Glycoprofiling (DQGlyco), a method that combines high-throughput sample preparation, high-sensitivity detection, and precise multiplexed quantification of protein glycosylation. We used DQGlyco to profile the mouse brain glycoproteome, in which we identify 158,972 and 15,056 unique N- and O-glycopeptides localized on 3,199 and 2,365 glycoproteins, respectively - this amounts to 25-fold more glycopeptides identified compared to previous studies. We observed extensive heterogeneity of glycoforms and determined their functional and structural preferences. The presence of a defined gut microbiota resulted in extensive remodeling of the brain glycoproteome when compared to that of germ-free animals, exemplifying how the gut microbiome may affect brain protein functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556900">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556900" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556900">
        <p class="paperTitle">Structure of γ-secretase (PSEN1/APH-1B) in complex with Aβ46 provides insights into amyloid-β processing and modulation by the APH-1B isoform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556900" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556900" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Odorcic, I.; Soliman, M. B. H.; Lismont, S.; Chavez-Gutierrez, L.; Efremov, R. G.</p>
        <p class="info">Score: 15.0, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556900' target='https://doi.org/10.1101/2023.09.09.556900'> 10.1101/2023.09.09.556900</a></p>
        <p class="abstract">Deposition of amyloid-{beta} (A{beta}) peptides in the brain is a hallmark of Alzheimers disease. A{beta}s are generated through sequential proteolysis of the amyloid precursor protein by the {gamma}-secretase complexes (GSECs). A{beta} peptide length, which is modulated by the Presenilin (PSEN) and APH-1 subunits of GSEC, is critical for Alzheimers pathogenesis. Despite high relevance, mechanistic understanding of the proteolysis of A{beta}, and its modulation by APH-1, remain incomplete. Here, we report cryo-EM structures of human GSEC (PSEN1/APH-1B) reconstituted into lipid nanodiscs in apo form and in complex with the intermediate A{beta}46 substrate. We found a divergent APH-1 loop to be involved with PSEN1 in substrate-binding-induced concerted rearrangements. Upstream the catalytic site, A{beta}46 structure is similar to the endopeptidase substrates and is stabilised by polar interactions including a previously unseen interaction with PSEN1 loop1. The hybrid {beta}-sheet was not observed downstream the catalytic site.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553817">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553817" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553817">
        <p class="paperTitle">TAX1BP1 recruits ATG9 vesicles through SCAMP3 binding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553817" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553817" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hama, Y.; Kurikawa, Y.; Matsui, T.; Mizushima, N.; Yamamoto, H.</p>
        <p class="info">Score: 13.0, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553817' target='https://doi.org/10.1101/2023.08.18.553817'> 10.1101/2023.08.18.553817</a></p>
        <p class="abstract">Macroautophagy is a cellular process that delivers cytoplasmic material to lysosomes for degradation via autophagosomes. It often involves the selective degradation of ubiquitinated proteins. During selective macroautophagy, five ubiquitin-binding adaptors, p62, NBR1, OPTN, NDP52, and TAX1BP1, form biomolecular condensates with ubiquitinated proteins and recruit ATG9 vesicles, which serve as the initial membrane source required for autophagosome formation. However, the molecular details underlying the cargo/adaptor-dependent recruitment of ATG9 vesicles remain unclear. Here, we show that ATG9 vesicles are recruited by three cargo adaptors: TAX1BP1, NBR1, and OPTN. We also find that ATG9A itself is not the determinant for recruitment by these cargo adaptors, and that TAX1BP1-dependent ATG9 vesicle recruitment is mediated by SCAMP3, a transmembrane protein on the ATG9 vesicles, through binding to the coiled-coil 1 domain of TAX1BP1. These findings provide mechanistic insights into the cargo/adaptor-dependent assembly of ATG9 vesicles in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556779" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556779">
        <p class="paperTitle">Molecular model of a bacterial flagellar motor in situ reveals a &#34;parts-list&#34; of protein adaptations to increase torque</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drobnic, T.; Cohen, E. J.; Alzheimer, M.; Froschauer, K.; Svensson, S.; Singh, N.; Garg, S. G.; Henderson, L.; Umrekar, T.; Nans, A.; Ribardo, D.; Hochberg, G.; Hendrixson, D. R.; Sharma, C. M.; Rosenthal, P.; Beeby, M.</p>
        <p class="info">Score: 11.8, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556779' target='https://doi.org/10.1101/2023.09.08.556779'> 10.1101/2023.09.08.556779</a></p>
        <p class="abstract">One hurdle to understanding how molecular machines function and evolve is our inability to see their structures in situ. Here we describe a minicell system that enables in situ cryogenic electron microscopy imaging and single particle analysis to probe the mechanisms and evolution of an iconic molecular machine, the bacterial flagellar motor, which spins a helical propeller for bacterial propulsion. Innovations in sample preparation and imaging enabled resolutions sufficient to build an in situ molecular model of the C. jejuni flagellar motor. Our results provide unprecedented insights into the in situ context of flagellar motors, highlight origins of recruited components involved in the unusually high torque of the C. jejuni motor, identify previously unknown components, and reveal corresponding modifications of core components. We also visualise structures involved in torque generation and secretion previously recalcitrant to structure determination. This technique will be of broad applicability to other large membrane-residing protein complexes. Note that this manuscript has a sibling manuscript titled &#34;Evolution of a large periplasmic disk in Campylobacterota flagella facilitated efficient motility alongside autoagglutination&#34; that dissects the function of the large disk described in this manuscript.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557663">
        <p class="paperTitle">Spatiotemporal Proteomics Deciphers Functional Selectivity of EGFR ligands</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eguchi, A.; Emdal, K. B.; Ye, Z.; Guzman, U. H.; Olsen, J. V.</p>
        <p class="info">Score: 9.3, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557663' target='https://doi.org/10.1101/2023.09.13.557663'> 10.1101/2023.09.13.557663</a></p>
        <p class="abstract">The epidermal growth factor receptor (EGFR) induces different signaling outputs depending on ligand identity and biological context. This phenomenon is known as functional selectivity, but the underlying molecular mechanisms remain elusive. Here, we investigated this on a global scale and time-resolved by high-throughput multilayered proteomics integrating dynamic changes in the EGFR interaction network by proximity biotinylation using EGFR-TurboID, phosphoproteome, and proteome in response to stimulation with the six highest-affinity EGFR ligands. We obtained comprehensive temporal profiles of protein recruitment and phosphosite changes pinpointing signaling proteins differentially regulated by the six ligands with key impact on EGFR endocytic fate, e.g. degradation or recycling. Specifically, the Epsin family protein, Clint1 was identified to control the endocytic trafficking of EGFR towards degradation. Moreover, we characterized the protein interaction selectivity of EGFR C-terminally phosphorylated tyrosine residues using a panel of tyrosine mutated constructs showing STAT5 specificity for EGFR Y1173. These data provide a comprehensive resource deciphering functional selectivity of EGFR signaling to support discovery of novel drug targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557454">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557454" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557454">
        <p class="paperTitle">A brown fat-enriched adipokine, ASRA, is a leptin receptor antagonist that stimulates appetite</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557454" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557454" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, L.; Liu, P.; Du, Y.; Pan, D.; Lee, A.; Wolfe, S.; Wang, Y.-X.</p>
        <p class="info">Score: 8.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557454' target='https://doi.org/10.1101/2023.09.12.557454'> 10.1101/2023.09.12.557454</a></p>
        <p class="abstract">The endocrine control of food intake remains incompletely understood, and whether the leptin receptor-mediated anorexigenic pathway in the hypothalamus is negatively regulated by a humoral factor is unknown. Here we identify an appetite-stimulating factor - ASRA - that acts as a leptin receptor antagonist. ASRA encodes an 8 kD protein that is abundantly and selectively expressed in adipose tissue and to a lesser extent, in liver, and is upregulated during fasting and cold. ASRA protein associates with autophagosomes and its secretion is induced by energy deficiency. Overexpression of ASRA in mice attenuates leptin receptor signaling leading to elevated blood glucose and development of severe hyperphagic obesity, whereas either adipose- or liver-specific ASRA knockout mice display increased leptin sensitivity, improved glucose homeostasis, reduced food intake, and resistance to high fat diet-induced obesity. Furthermore, ASRA is indispensable for cold-evoked feeding response. Recombinant ASRA (rASRA) protein binds to leptin receptor and suppresses leptin receptor signaling in cultured cells. In vivo, rASRA promotes food intake and increases blood glucose in a leptin receptor signaling-dependent manner. Our studies collectively show that ASRA, acting as a peripheral signal of energy deficit, stimulates appetite and regulates glucose metabolism by antagonizing leptin receptor signaling, thus revealing a previously unknown endocrine mechanism that has important implications for our understanding of leptin resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.556486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.556486" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.556486">
        <p class="paperTitle">The intrinsically disordered region of the E3 ubiquitin ligase TRIP12 induces the formation of chromatin condensates and interferes with DNA damage response.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.556486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.556486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vargas, C.; Brunet, M.; Ricard, A.; varry, d.; Muzzopappa, F.; Erdel, F.; Labrousee, G.; Hanoun, N.; Ligat, L.; Farce, M.; Stella, A.; Burlet-Schiltz, O.; Laurell, H.; Bery, N.; Cordelier, P.; dufresne, m.; torrisani, j.</p>
        <p class="info">Score: 7.0, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.556486' target='https://doi.org/10.1101/2023.09.13.556486'> 10.1101/2023.09.13.556486</a></p>
        <p class="abstract">Chromatin compaction is crucial for the faithful expression and integrity of the genome. Although largely studied, proteins and mechanisms that control the chromatin compaction are not entirely discovered. We previously showed that the nuclear HECT-type E3 ubiquitin ligase Thyroid hormone Receptor Interacting Protein 12 (TRIP12) is tightly associated to chromatin. As TRIP12 is overexpressed in several types of cancers, we explored herein the consequences of a TRIP12 overexpression on chromatin homeostasis. First, we established the TRIP12 proxisome and unveiled its pleiotropic role in chromatin regulation. Second, we demonstrated that TRIP12 overexpression leads to the formation of chromatin condensates enriched in heterochromatin marks via its intrinsically disordered region (IDR). We further discovered that the formation of TRIP12-mediated chromatin condensates is highly dynamic and driven by a mechanism of phase separation. Chromatin condensate formation depends on the TRIP12 concentration, the length of the TRIP12-IDR and relies on electrostatic interactions. We found that the formation of TRIP12 mediated-condensates alters cell cycle progression, genome accessibility, transcription as well as DNA damage response by inhibiting the accumulation of Mediator of DNA Damage Checkpoint 1 (MDC1). Altogether, this study reveals a novel dynamic role for TRIP12 in chromatin compaction independently of its ubiquitin ligase activity with important consequences on cellular homeostasis.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=155 SRC=&#34;FIGDIR/small/556486v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@ba5d55org.highwire.dtl.DTLVardef@14578d9org.highwire.dtl.DTLVardef@fb85org.highwire.dtl.DTLVardef@80e92d_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556957">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556957" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556957">
        <p class="paperTitle">Parallel G-quadruplex folds via multiple paths involving G-tract stacking and structuring from coil ensemble</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556957" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556957" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pokorna, P.; Mlynsky, V.; Bussi, G.; Sponer, J.; Stadlbauer, P.</p>
        <p class="info">Score: 6.6, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556957' target='https://doi.org/10.1101/2023.09.09.556957'> 10.1101/2023.09.09.556957</a></p>
        <p class="abstract">G-quadruplexes (G4s) are non-canonical nucleic acid structures that fold through complex processes. Characterization of G4 folding landscape contributes to comprehending G4 roles in gene regulation but is challenging for experiments and computations. Here we investigate the folding of a three-quartet parallel DNA G4 with (GGGA)3GGG sequence using all-atom explicit-solvent enhanced-sampling molecular dynamics (MD) simulations. We suggest an early formation of guanine stacks in the G-tracts, which behave as semi-rigid blocks in the folding process. The parallel G4 folding is initiated by the formation of a collapsed compact coil-like ensemble. Structuring of the G4 from the coil then proceeds via various cross-like, hairpin, slip-stranded, and two-quartet ensembles and can bypass the G-triplex structure. While parallel G-hairpins are extremely unstable when isolated, they are more stable inside the coil structure. Folding of parallel G4 does not appear to involve any salient intermediates and, instead, it is an extremely multiple-pathway process. On the methodology side, we show that the AMBER DNA force field predicts the folded G4 to be less stable than the unfolded ensemble, uncovering substantial force-field issues. Overall, we provide unique atomistic insights into the folding landscape of parallel-stranded G4 but also reveal limitations of the state-of-the-art MD techniques.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557269">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557269" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557269">
        <p class="paperTitle">Uncoupling the TFIIH Core and Kinase Modules Leads To Unregulated RNA Polymerase II CTD Serine 5 Phosphorylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557269" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557269" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giordano, G.; Buratowski, R.; Jeronimo, C.; Poitras, C.; Robert, F.; Buratowski, S.</p>
        <p class="info">Score: 6.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557269' target='https://doi.org/10.1101/2023.09.11.557269'> 10.1101/2023.09.11.557269</a></p>
        <p class="abstract">TFIIH is an essential transcription initiation factor for RNA polymerase II (RNApII). This multi-subunit complex comprises two modules that are physically linked by the subunit Tfb3 (MAT1 in metazoans). The TFIIH Core Module, with two DNA-dependent ATPases and several additional subunits, promotes DNA unwinding. The TFIIH Kinase Module phosphorylates Serine 5 of the C-terminal domain (CTD) of RNApII subunit Rpb1, a modification that coordinates exchange of initiation and early elongation factors. While it is not obvious why these two disparate activities are bundled into one factor, the connection may provide temporal coordination during early initiation. Here we show that Tfb3 can be split into two parts to uncouple the TFIIH modules. The resulting cells grow slower than normal, but are viable. Chromatin immunoprecipitation of the split TFIIH shows that the Core Module, but not the Kinase, is properly recruited to promoters. Instead of the normal promoter-proximal peak, high CTD Serine 5 phosphorylation is seen throughout transcribed regions. Therefore, coupling the TFIIH modules is necessary to localize and limit CTD kinase activity to early stages of transcription. These results are consistent with the idea that the two TFIIH modules began as independent functional entities that became connected by Tfb3 during early eukaryotic evolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
