<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples
Authors: Arimura, Y.; Konishi, H. A.; Funabiki, H.
Score: 53.3, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576499
Here, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to &lt; 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples
Authors: Arimura, Y.; Konishi, H. A.; Funabiki, H.
Score: 53.3, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576499
Here, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to &lt; 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples
Authors: Arimura, Y.; Konishi, H. A.; Funabiki, H.
Score: 53.3, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576499
Here, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to &lt; 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples\nAuthors: Arimura, Y.; Konishi, H. A.; Funabiki, H.\nScore: 53.3, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576499\nHere, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to \u0026lt; 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1.",
  "keywords": [
    
  ],
  "articleBody": " Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples\nAuthors: Arimura, Y.; Konishi, H. A.; Funabiki, H.\nScore: 53.3, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576499\nHere, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to \u003c 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1.8 associated nucleosomes isolated from interphase and metaphase chromosomes in Xenopus egg extract. In combination with Duplicated Selection To Exclude Rubbish particles (DuSTER), a particle curation method that removes low signal-to-noise ratio particles, we also resolved the 3D cryo-EM structures of H1.8-bound nucleoplasmin NPM2, revealing distinct open and closed structural variants. Our approaches offer structural insights into how the chaperone NPM2 interacts with its targets and the cell cycle-dependent regulation of H1.8 association to nucleosomes.\nCryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization\nAuthors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.\nScore: 43.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576763\nCryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids. We demonstrate that vitreous ice may be produced by plunging clipped grids with purified proteins into liquid ethane and that clipped grids may be reused several times for different protein samples. Furthermore, we demonstrate the vitrification of thin areas of cells prepared on gold-coated, pre-clipped grids.\nEasyGrid: A versatile platform for automated cryo-EM sample preparation and quality control\nAuthors: Gemin, O.; Armijo, V.; Hons, M.; Bissardon, C.; Linares, R.; Bowler, M. W.; Wolff, G.; Kovalev, K.; Babenko, A.; Salo, V. T.; Schneider, S.; Rossi, C.; Lecomte, L.; Deckers, T.; Lauzier, K.; Janocha, R.; Felisaz, F.; Sinoir, J.; Galej, W. P.; Mahamid, J.; Muller, C. W.; Eustermann, S.; Mattei, S.; Cipriani, F.; Papp, G.\nScore: 34.2, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576170\nImaging biological macromolecules in their native state with single-particle cryo-electron microscopy (cryo-EM) or in situ cryo-electron tomography (cryo-ET) requires optimized approaches for the preparation and vitrification of biological samples. Here, we describe EasyGrid, a versatile technology enabling systematic, tailored and advanced sample preparation for cellular and structural biology. This automated, standalone platform combines in-line plasma treatment, microfluidic dispensing, blot-less sample spreading, jet-based vitrification and on-the-fly grid quality control using light interferometry to streamline cryo-EM sample optimization. With EasyGrid, we optimized grid preparation for different purified macromolecular complexes and subsequently determined their structure with cryo-EM. We also demonstrated how the platform allows better vitrification of large, mammalian cells compared to standard plunge-freezing. Automated sample preparation with EasyGrid establishes an advanced, high-throughput platform for both single-particle cryo-EM and cellular cryo-ET sample preparation.\nThe conformational landscape of human transthyretin revealed by cryo-EM\nAuthors: Basanta, B.; Nugroho, K.; Yan, N. Y.; Kline, G. M.; Powers, E. T.; Tsai, F. J.; Wu, M.; Hansel-Harris, A.; Chen, J. S.; Forli, S.; Kelly, J. W.; Lander, G. C.\nScore: 25.0, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576879\nTransthyretin (TTR) is a natively tetrameric thyroxine transporter found in blood and cerebrospinal fluid whose misfolding and aggregation causes transthyretin amyloidosis. A rational drug design campaign identified the small molecule tafamidis (Vyndaqel/Vyndamax) as an effective stabilizer of the native TTR fold, and this aggregation inhibitor is regulatory agency-approved for the treatment of TTR amyloidosis. Despite 50 years of structural studies on TTR and this triumph of structure-based drug design, there remains a notable dearth of structural information available to understand ligand binding allostery and amyloidogenic TTR unfolding intermediates. We used single-particle cryo-electron microscopy (cryo-EM) to investigate the conformational landscape of this 55 kiloDalton tetramer in the absence and presence of one or two ligands, revealing inherent asymmetries in the tetrameric architecture and previously unobserved conformational states. These findings provide critical mechanistic insights into negatively cooperative ligand binding and the structural pathways responsible for TTR amyloidogenesis. This study underscores the capacity of cryo-EM to provide new insights into protein structures that have been historically considered too small to visualize and to identify pharmacological targets suppressed by the confines of the crystal lattice, opening uncharted territory in structure-based drug design.\nDiscovery and Characterization of a Pan-betacoronavirus S2-binding antibody\nAuthors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.\nScore: 324.2, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575741\nSUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.\nTrends of Plasmodium falciparum molecular markers associated with resistance to artemisinins and reduced susceptibility to lumefantrine in Mainland Tanzania from 2016 to 2021\nAuthors: Bakari, C.; Mandara, C. I.; Madebe, R. A.; Seth, M. D.; Ngasala, B.; Kamugisha, E.; Ahmed, M.; Francis, F.; Bushukatale, S.; Chiduo, M.; Makene, T.; Kabanywanyi, A.; Mahende, M. K.; Muro, F.; Mkude, S.; Mandike, R.; Molteni, F.; Chacky, F.; Bishanga, D.; Njau, R.; Warsame, M.; Kabula, B.; Nyinondi, S.; Talundzic, E.; Lucchi, N. W.; Venkatesan, M.; Moriarty, L. F.; Serbantez, N.; Kitojo, C.; Reaves, E. J.; Halsey, E. S.; Mohamed, A.; Udhayakumar, V.; Ishengoma, D. S.\nScore: 11.7, Published: 2024-01-21 DOI: 10.1101/2024.01.18.576107\nBackgroundTherapeutic efficacy studies (TESs) and detection of molecular markers of drug resistance are recommended by the World Health Organization (WHO) to monitor the efficacy of artemis inin combination therapy (ACT). This study assessed the trends of molecular markers of artemis inin resistance and/or reduced susceptibility to lumefantrine using samples collected in TES conducted in Mainland Tanzania from 2016 to 2021. MethodsA total of 2,015 samples were collected during TES of artemether-lumefantrine at eight sentinel sites (in Kigoma, Mbeya, Morogoro, Mtwara, Mwanza, Pwani, Tabora, and Tanga regions) between 2016 and 2021. Photo-induced electron transfer polymerase chain reaction (PET-PCR) was used to confirm presence of malaria parasites before capillary sequencing, which targeted two genes: Plasmodium falciparum kelch 13 propeller domain (k13) and P. falciparum multidrug resistance 1 (pfmdr1). ResultsSequencing success was [\u0026ge;]87.8%, and 1,724/1,769 (97.5%) k13 wild-type samples were detected. Thirty-seven (2.1%) samples had synonymous mutations and only eight (0.4%) had non-synonymous mutations in the k13 gene; seven of these were not validated by WHO as molecular markers of resistance (I416V, E433D, R471S, P475S, A578S, and Q613E). One sample from Morogoro in 2020 had a k13 R622I mutation, which is a validated marker of artemisinin partial resistance. For pfmdr1, all except two samples carried N86 (wild-type), while mutations at Y184F increased from 33.9% in 2016 to about 60.5% in 2021, and only four samples (0.2%) had D1246Y mutations. pfmdr1 haplotypes were reported in 1,711 samples, with 985 (57.6%) NYD, 720 (42.1%) NFD, and six (0.4%) carrying minor haplotypes (three with NYY, 0.2%; YFD in two, 0.1%; and NFY in one sample, 0.1%). Between 2016 and 2021, NYD decreased from 66.1% to 45.2%, while NFD increased from 38.5% to 54.7%. ConclusionThis is the first report of the R622I (k13 validated mutation) in Tanzania. N86 and D1246 were nearly fixed, while increases in Y184F mutations and NFD haplotype were observed between 2016 and 2021. Despite the reports of ART-R in Rwanda and Uganda, this study did not report any other validated mutations in these study sites in Tanzania apart from R622I suggesting that intensified surveillance is urgently needed to monitor trends of drug resistance markers and their impact on the performance of ACTs.\nPeptidyl-tRNA hydrolysis rate influences the efficiency of nonsense-mediated mRNA decay\nAuthors: Kolakada, D.; Fu, R.; Biziaev, N.; Shuvalov, A.; Lore, M.; Campbell, A. E.; Cortazar, M. A.; Sajek, M. P.; Hesselberth, J. R.; Mukherjee, N.; Alkalaeva, E.; Jagannathan, S.\nScore: 11.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575080\nNonsense-mediated mRNA decay (NMD) is a quality control mechanism that prevents the accumulation of harmful truncated proteins by degrading transcripts with premature termination codons (PTCs). NMD efficiency varies across many contexts, but the factors that influence this variability remain poorly understood. Here, we find an enrichment of glycine (Gly) codons preceding a PTC in common nonsense variants in contrast with a depletion of Gly codons preceding a normal termination codon (NTC). Gly-PTC contexts have higher NMD activity compared to an alanine-PTC context, and this effect is stronger on NMD substrates with long 3UTRs. We used a massively parallel reporter assay to test all possible combinations of -2 and -1 codons, the PTC, and the +4 nucleotide to assess comprehensively how PTC sequence context affects NMD efficiency. A random forest classifier revealed that peptidyl-tRNA hydrolysis rate during translation termination was the most important feature in discriminating high and low NMD activity. We show with in vitro biochemical assays that Gly-TC contexts have the slowest termination rate compared to other codons. Furthermore, Gly-PTC enrichment is most pronounced in genes that tolerate loss-of-function variants, suggesting that enhanced NMD of Gly-PTC context has shaped the evolution of PTCs. Based on these findings, we propose that NMD efficiency is modulated by the \"window of opportunity\" offered by peptidyl tRNA hydrolysis rate and thus, translation termination kinetics.\nSterol O-acyltransferase (SOAT/ACAT) activity is required to form cholesterol crystals in hepatocyte lipid droplets\nAuthors: Bairos, J. A.; Njoku, U.; Zafar, M.; Akl, M. G.; Li, L.; Parlakgul, G.; Arruda, A. P.; Widenmaier, S. B.\nScore: 6.2, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576345\nObjectiveExcess unesterified (free) cholesterol can induce formation of cholesterol crystals in hepatocyte lipid droplets. Presence of such crystal distinguishes metabolic dysfunction associated steatohepatitis (MASH) from simple steatosis and may underlie its pathogenesis by causing cell damage that triggers liver inflammation. The mechanism linking cholesterol excess to its crystallization in lipid droplets is unclear. As cholesteryl esters localize to and accumulate in lipid droplets much more readily than free cholesterol, we investigated whether cholesterol esterification by sterol O-acyltransferase (SOAT), also known as acyl co-A cholesterol acyltransferase (ACAT) is required for hepatocyte lipid droplet crystal formation. MethodCholesterol crystals were measured in cholesterol loaded Hep3B hepatocytes, RAW264.7 macrophages and mouse liver using polarizing light microscopy. We examined the effect of blocking SOAT activity on crystal formation and compared these results to cholesterol metabolism and the progression to intracellular crystal deposits. ResultsCholesterol loading of Hep3B cells caused robust levels of lipid droplet localized crystal formation in a dose- and time-dependent manner. Co-treatment with SOAT inhibitors and genetic ablation of SOAT1 blocked crystal formation. SOAT inhibitor also blocked crystal formation in low density lipoprotein (LDL) treated Hep3B cells, acetylated LDL treated RAW 264.7 macrophages, and in the liver of mice genetically predisposed to hepatic cholesterol overload and in mice fed a cholesterol enriched, MASH-promoting diet for 24 weeks. ConclusionSOAT1-mediated esterification may underlie cholesterol crystals associated with MASH by concentrating it in lipid droplets. These findings imply that inhibiting hepatocyte SOAT1 may alleviate cholesterol associated MASH. Moreover, that a lipid droplet localized cholesteryl ester hydrolase may be required for cholesterol crystal formation or, instead, that the crystals are composed of cholesteryl ester. Funding SourcesGrants supporting this research were awarded to SBW from the Natural Sciences and Engineering Research Council of Canada (NSERC). SBW was supported by a National New Investigator Award and McDonald Scholarship from the Heart and Stroke Foundation of Canada. UN and MA were supported by a James Regan Cardiology Research scholarship from University of Saskatchewans College of Medicine.\nSubcellular localization of truncated MAGEL2 proteins: insight into the molecular pathology of Schaaf-Yang syndrome\nAuthors: Centeno-Pla, M.; Alcaide-Consuegra, E.; Gibson, S.; Prat-Planas, A.; Gutierrez-Avila, J. D.; Grinberg, D.; Urreizti, R.; Rabionet, R.; Balcells, S.\nScore: 5.9, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576607\nSchaaf-Yang syndrome (SYS) is an ultra-rare neurodevelopmental disorder caused by truncating mutations in MAGEL2. Heterologous expression of wild-type (WT) or a truncated (p.Gln638*) C-terminal HA-tagged MAGEL2 revealed a shift from a primarily cytoplasmic to a more nuclear localization for the truncated protein variant. We now extend this analysis to six additional SYS mutations on a N-terminal FLAG-tagged MAGEL2. Our results replicate and extend our previous findings, showing that all the truncated MAGEL2 proteins consistently display a predominant nuclear localization, irrespective of the C-terminal or N-terminal position and the chemistry of the tag. The variants associated with arthrogryposis multiplex congenita (AMC) display a more pronounced nuclear retention phenotype, suggesting a correlation between clinical severity and the degree of nuclear mis-localization. These results point to a neomorphic effect of truncated MAGEL2, which might contribute to the pathogenesis of SYS.\nGeometric changes in the nucleoids of Deinococcus radiodurans reveal involvement of new proteins in recovery from ionizing radiation.\nAuthors: Cordova, A.; Niese, B.; Sweet, P.; Kamat, P.; Phillip, J. M.; Gordon, V.; Contreras, L. M.\nScore: 5.4, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576117\nThe extremophile Deinococcus radiodurans maintains a highly-organized and condensed nucleoid as its default state, possibly contributing to high tolerance of ionizing radiation (IR). Previous studies of the D. radiodurans nucleoid were limited by reliance on manual image annotation and qualitative metrics. Here, we introduce a high-throughput approach to quantify the geometric properties of cells and nucleoids, using confocal microscopy, digital reconstructions of cells, and computational modeling. We utilize this novel approach to investigate the dynamic process of nucleoid condensation in response to IR stress. Our quantitative analysis reveals that at the population level, exposure to IR induced nucleoid compaction and decreased size of D. radiodurans cells. Morphological analysis and clustering identified six distinct sub-populations across all tested experimental conditions. Results indicate that exposure to IR induces fractional redistributions of cells across sub-populations to exhibit morphologies that associate with greater nucleoid condensation, and decreased abundance of sub-populations associated with cell division. Nucleoid associated proteins (NAPs) may link nucleoid compaction and stress tolerance, but their roles in regulating compaction in D. radiodurans is unknown. Imaging of genomic mutants of known and suspected NAPs that contribute to nucleoid condensation found that deletion of nucleic acid binding proteins, not previously described as NAPs, can remodel the nucleoid by driving condensation or decondensation in the absence of stress and that IR increases the abundance of these morphological states. Thus, our integrated analysis introduces a new methodology for studying environmental influences on bacterial nucleoids and provides an opportunity to further investigate potential regulators of nucleoid condensation. ImportanceD. radiodurans, an extremophile known for its stress tolerance, constitutively maintains a highly-condensed nucleoid. Qualitative studies have described nucleoid behavior under a variety of conditions. However, a lack of quantitative data regarding nucleoid organization and dynamics have limited our understanding of regulatory mechanisms controlling nucleoid organization in D. radiodurans. Here, we introduce a quantitative approach that enables high-throughput quantitative measurements of subcellular spatial characteristics in bacterial cells. Applying this to wild-type or single-protein-deficient populations of D. radiodurans subjected to ionizing radiation, we identified significant stress-responsive changes in cell shape, nucleoid organization, and morphology. These findings highlight this methodologys adaptability and capacity for quantitatively analyzing the cellular response to stressors for screening cellular proteins involved in bacterial nucleoid organization.\n",
  "wordCount" : "2631",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576499" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576499">
        <p class="paperTitle">Cryo-EM analysis on magnetic beads for scarce macromolecules in heterogeneous samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arimura, Y.; Konishi, H. A.; Funabiki, H.</p>
        <p class="info">Score: 53.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576499' target='https://doi.org/10.1101/2024.01.21.576499'> 10.1101/2024.01.21.576499</a></p>
        <p class="abstract">Here, we introduce Magnetic Isolation and Concentration (MagIC)-cryo-EM, a technique enabling direct structural analysis of targets captured on nanomagnetic beads, thereby reducing the targets concentration requirement to &lt; 0.0005 mg/ml, which is 100 to 1000-fold lower than conventional cryo-EM methods. Using MagIC-cryo-EM, we characterized structural variations of oocyte-specific linker histone H1.8 associated nucleosomes isolated from interphase and metaphase chromosomes in Xenopus egg extract. In combination with Duplicated Selection To Exclude Rubbish particles (DuSTER), a particle curation method that removes low signal-to-noise ratio particles, we also resolved the 3D cryo-EM structures of H1.8-bound nucleoplasmin NPM2, revealing distinct open and closed structural variants. Our approaches offer structural insights into how the chaperone NPM2 interacts with its targets and the cell cycle-dependent regulation of H1.8 association to nucleosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576763" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576763">
        <p class="paperTitle">CryoCycle your grids: Plunge vitrifying and reusing clipped grids to advance cryoEM democratization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serbynovskyi, V.; Wang, J.; Chua, E. Y. D.; Ishemgulova, A.; Alink, L. M.; Budell, W. C.; Johnston, J. D.; Dubbeldam, C.; Gonzalez, F. A.; Rozovsky, S.; Eng, E. T.; de Marco, A.; Noble, A. J.</p>
        <p class="info">Score: 43.1, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576763' target='https://doi.org/10.1101/2024.01.23.576763'> 10.1101/2024.01.23.576763</a></p>
        <p class="abstract">CryoEM democratization is hampered by access to costly plunge-freezing supplies. We introduce methods called CryoCycle, for reliably blotting, vitrifying, and reusing clipped cryoEM grids. We demonstrate that vitreous ice may be produced by plunging clipped grids with purified proteins into liquid ethane and that clipped grids may be reused several times for different protein samples. Furthermore, we demonstrate the vitrification of thin areas of cells prepared on gold-coated, pre-clipped grids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576170">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576170" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576170">
        <p class="paperTitle">EasyGrid: A versatile platform for automated cryo-EM sample preparation and quality control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576170" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576170" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gemin, O.; Armijo, V.; Hons, M.; Bissardon, C.; Linares, R.; Bowler, M. W.; Wolff, G.; Kovalev, K.; Babenko, A.; Salo, V. T.; Schneider, S.; Rossi, C.; Lecomte, L.; Deckers, T.; Lauzier, K.; Janocha, R.; Felisaz, F.; Sinoir, J.; Galej, W. P.; Mahamid, J.; Muller, C. W.; Eustermann, S.; Mattei, S.; Cipriani, F.; Papp, G.</p>
        <p class="info">Score: 34.2, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576170' target='https://doi.org/10.1101/2024.01.18.576170'> 10.1101/2024.01.18.576170</a></p>
        <p class="abstract">Imaging biological macromolecules in their native state with single-particle cryo-electron microscopy (cryo-EM) or in situ cryo-electron tomography (cryo-ET) requires optimized approaches for the preparation and vitrification of biological samples. Here, we describe EasyGrid, a versatile technology enabling systematic, tailored and advanced sample preparation for cellular and structural biology. This automated, standalone platform combines in-line plasma treatment, microfluidic dispensing, blot-less sample spreading, jet-based vitrification and on-the-fly grid quality control using light interferometry to streamline cryo-EM sample optimization. With EasyGrid, we optimized grid preparation for different purified macromolecular complexes and subsequently determined their structure with cryo-EM. We also demonstrated how the platform allows better vitrification of large, mammalian cells compared to standard plunge-freezing. Automated sample preparation with EasyGrid establishes an advanced, high-throughput platform for both single-particle cryo-EM and cellular cryo-ET sample preparation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576879" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576879">
        <p class="paperTitle">The conformational landscape of human transthyretin revealed by cryo-EM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Basanta, B.; Nugroho, K.; Yan, N. Y.; Kline, G. M.; Powers, E. T.; Tsai, F. J.; Wu, M.; Hansel-Harris, A.; Chen, J. S.; Forli, S.; Kelly, J. W.; Lander, G. C.</p>
        <p class="info">Score: 25.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576879' target='https://doi.org/10.1101/2024.01.23.576879'> 10.1101/2024.01.23.576879</a></p>
        <p class="abstract">Transthyretin (TTR) is a natively tetrameric thyroxine transporter found in blood and cerebrospinal fluid whose misfolding and aggregation causes transthyretin amyloidosis. A rational drug design campaign identified the small molecule tafamidis (Vyndaqel/Vyndamax) as an effective stabilizer of the native TTR fold, and this aggregation inhibitor is regulatory agency-approved for the treatment of TTR amyloidosis. Despite 50 years of structural studies on TTR and this triumph of structure-based drug design, there remains a notable dearth of structural information available to understand ligand binding allostery and amyloidogenic TTR unfolding intermediates. We used single-particle cryo-electron microscopy (cryo-EM) to investigate the conformational landscape of this 55 kiloDalton tetramer in the absence and presence of one or two ligands, revealing inherent asymmetries in the tetrameric architecture and previously unobserved conformational states. These findings provide critical mechanistic insights into negatively cooperative ligand binding and the structural pathways responsible for TTR amyloidogenesis. This study underscores the capacity of cryo-EM to provide new insights into protein structures that have been historically considered too small to visualize and to identify pharmacological targets suppressed by the confines of the crystal lattice, opening uncharted territory in structure-based drug design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575741" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575741">
        <p class="paperTitle">Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.</p>
        <p class="info">Score: 324.2, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575741' target='https://doi.org/10.1101/2024.01.15.575741'> 10.1101/2024.01.15.575741</a></p>
        <p class="abstract">SUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576107">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576107" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576107">
        <p class="paperTitle">Trends of Plasmodium falciparum molecular markers associated with resistance to artemisinins and reduced susceptibility to lumefantrine in Mainland Tanzania from 2016 to 2021</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576107" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576107" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bakari, C.; Mandara, C. I.; Madebe, R. A.; Seth, M. D.; Ngasala, B.; Kamugisha, E.; Ahmed, M.; Francis, F.; Bushukatale, S.; Chiduo, M.; Makene, T.; Kabanywanyi, A.; Mahende, M. K.; Muro, F.; Mkude, S.; Mandike, R.; Molteni, F.; Chacky, F.; Bishanga, D.; Njau, R.; Warsame, M.; Kabula, B.; Nyinondi, S.; Talundzic, E.; Lucchi, N. W.; Venkatesan, M.; Moriarty, L. F.; Serbantez, N.; Kitojo, C.; Reaves, E. J.; Halsey, E. S.; Mohamed, A.; Udhayakumar, V.; Ishengoma, D. S.</p>
        <p class="info">Score: 11.7, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576107' target='https://doi.org/10.1101/2024.01.18.576107'> 10.1101/2024.01.18.576107</a></p>
        <p class="abstract">BackgroundTherapeutic efficacy studies (TESs) and detection of molecular markers of drug resistance are recommended by the World Health Organization (WHO) to monitor the efficacy of artemis inin combination therapy (ACT). This study assessed the trends of molecular markers of artemis inin resistance and/or reduced susceptibility to lumefantrine using samples collected in TES conducted in Mainland Tanzania from 2016 to 2021.

MethodsA total of 2,015 samples were collected during TES of artemether-lumefantrine at eight sentinel sites (in Kigoma, Mbeya, Morogoro, Mtwara, Mwanza, Pwani, Tabora, and Tanga regions) between 2016 and 2021. Photo-induced electron transfer polymerase chain reaction (PET-PCR) was used to confirm presence of malaria parasites before capillary sequencing, which targeted two genes: Plasmodium falciparum kelch 13 propeller domain (k13) and P. falciparum multidrug resistance 1 (pfmdr1).

ResultsSequencing success was [&amp;ge;]87.8%, and 1,724/1,769 (97.5%) k13 wild-type samples were detected. Thirty-seven (2.1%) samples had synonymous mutations and only eight (0.4%) had non-synonymous mutations in the k13 gene; seven of these were not validated by WHO as molecular markers of resistance (I416V, E433D, R471S, P475S, A578S, and Q613E). One sample from Morogoro in 2020 had a k13 R622I mutation, which is a validated marker of artemisinin partial resistance. For pfmdr1, all except two samples carried N86 (wild-type), while mutations at Y184F increased from 33.9% in 2016 to about 60.5% in 2021, and only four samples (0.2%) had D1246Y mutations. pfmdr1 haplotypes were reported in 1,711 samples, with 985 (57.6%) NYD, 720 (42.1%) NFD, and six (0.4%) carrying minor haplotypes (three with NYY, 0.2%; YFD in two, 0.1%; and NFY in one sample, 0.1%). Between 2016 and 2021, NYD decreased from 66.1% to 45.2%, while NFD increased from 38.5% to 54.7%.

ConclusionThis is the first report of the R622I (k13 validated mutation) in Tanzania. N86 and D1246 were nearly fixed, while increases in Y184F mutations and NFD haplotype were observed between 2016 and 2021. Despite the reports of ART-R in Rwanda and Uganda, this study did not report any other validated mutations in these study sites in Tanzania apart from R622I suggesting that intensified surveillance is urgently needed to monitor trends of drug resistance markers and their impact on the performance of ACTs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575080" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575080">
        <p class="paperTitle">Peptidyl-tRNA hydrolysis rate influences the efficiency of nonsense-mediated mRNA decay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolakada, D.; Fu, R.; Biziaev, N.; Shuvalov, A.; Lore, M.; Campbell, A. E.; Cortazar, M. A.; Sajek, M. P.; Hesselberth, J. R.; Mukherjee, N.; Alkalaeva, E.; Jagannathan, S.</p>
        <p class="info">Score: 11.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575080' target='https://doi.org/10.1101/2024.01.10.575080'> 10.1101/2024.01.10.575080</a></p>
        <p class="abstract">Nonsense-mediated mRNA decay (NMD) is a quality control mechanism that prevents the accumulation of harmful truncated proteins by degrading transcripts with premature termination codons (PTCs). NMD efficiency varies across many contexts, but the factors that influence this variability remain poorly understood. Here, we find an enrichment of glycine (Gly) codons preceding a PTC in common nonsense variants in contrast with a depletion of Gly codons preceding a normal termination codon (NTC). Gly-PTC contexts have higher NMD activity compared to an alanine-PTC context, and this effect is stronger on NMD substrates with long 3UTRs. We used a massively parallel reporter assay to test all possible combinations of -2 and -1 codons, the PTC, and the &#43;4 nucleotide to assess comprehensively how PTC sequence context affects NMD efficiency. A random forest classifier revealed that peptidyl-tRNA hydrolysis rate during translation termination was the most important feature in discriminating high and low NMD activity. We show with in vitro biochemical assays that Gly-TC contexts have the slowest termination rate compared to other codons. Furthermore, Gly-PTC enrichment is most pronounced in genes that tolerate loss-of-function variants, suggesting that enhanced NMD of Gly-PTC context has shaped the evolution of PTCs. Based on these findings, we propose that NMD efficiency is modulated by the &#34;window of opportunity&#34; offered by peptidyl tRNA hydrolysis rate and thus, translation termination kinetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576345">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576345" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576345">
        <p class="paperTitle">Sterol O-acyltransferase (SOAT/ACAT) activity is required to form cholesterol crystals in hepatocyte lipid droplets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576345" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576345" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bairos, J. A.; Njoku, U.; Zafar, M.; Akl, M. G.; Li, L.; Parlakgul, G.; Arruda, A. P.; Widenmaier, S. B.</p>
        <p class="info">Score: 6.2, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576345' target='https://doi.org/10.1101/2024.01.20.576345'> 10.1101/2024.01.20.576345</a></p>
        <p class="abstract">ObjectiveExcess unesterified (free) cholesterol can induce formation of cholesterol crystals in hepatocyte lipid droplets. Presence of such crystal distinguishes metabolic dysfunction associated steatohepatitis (MASH) from simple steatosis and may underlie its pathogenesis by causing cell damage that triggers liver inflammation. The mechanism linking cholesterol excess to its crystallization in lipid droplets is unclear. As cholesteryl esters localize to and accumulate in lipid droplets much more readily than free cholesterol, we investigated whether cholesterol esterification by sterol O-acyltransferase (SOAT), also known as acyl co-A cholesterol acyltransferase (ACAT) is required for hepatocyte lipid droplet crystal formation.

MethodCholesterol crystals were measured in cholesterol loaded Hep3B hepatocytes, RAW264.7 macrophages and mouse liver using polarizing light microscopy. We examined the effect of blocking SOAT activity on crystal formation and compared these results to cholesterol metabolism and the progression to intracellular crystal deposits.

ResultsCholesterol loading of Hep3B cells caused robust levels of lipid droplet localized crystal formation in a dose- and time-dependent manner. Co-treatment with SOAT inhibitors and genetic ablation of SOAT1 blocked crystal formation. SOAT inhibitor also blocked crystal formation in low density lipoprotein (LDL) treated Hep3B cells, acetylated LDL treated RAW 264.7 macrophages, and in the liver of mice genetically predisposed to hepatic cholesterol overload and in mice fed a cholesterol enriched, MASH-promoting diet for 24 weeks.

ConclusionSOAT1-mediated esterification may underlie cholesterol crystals associated with MASH by concentrating it in lipid droplets. These findings imply that inhibiting hepatocyte SOAT1 may alleviate cholesterol associated MASH. Moreover, that a lipid droplet localized cholesteryl ester hydrolase may be required for cholesterol crystal formation or, instead, that the crystals are composed of cholesteryl ester.

Funding SourcesGrants supporting this research were awarded to SBW from the Natural Sciences and Engineering Research Council of Canada (NSERC). SBW was supported by a National New Investigator Award and McDonald Scholarship from the Heart and Stroke Foundation of Canada. UN and MA were supported by a James Regan Cardiology Research scholarship from University of Saskatchewans College of Medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576607" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576607">
        <p class="paperTitle">Subcellular localization of truncated MAGEL2 proteins: insight into the molecular pathology of Schaaf-Yang syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Centeno-Pla, M.; Alcaide-Consuegra, E.; Gibson, S.; Prat-Planas, A.; Gutierrez-Avila, J. D.; Grinberg, D.; Urreizti, R.; Rabionet, R.; Balcells, S.</p>
        <p class="info">Score: 5.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576607' target='https://doi.org/10.1101/2024.01.22.576607'> 10.1101/2024.01.22.576607</a></p>
        <p class="abstract">Schaaf-Yang syndrome (SYS) is an ultra-rare neurodevelopmental disorder caused by truncating mutations in MAGEL2. Heterologous expression of wild-type (WT) or a truncated (p.Gln638*) C-terminal HA-tagged MAGEL2 revealed a shift from a primarily cytoplasmic to a more nuclear localization for the truncated protein variant. We now extend this analysis to six additional SYS mutations on a N-terminal FLAG-tagged MAGEL2. Our results replicate and extend our previous findings, showing that all the truncated MAGEL2 proteins consistently display a predominant nuclear localization, irrespective of the C-terminal or N-terminal position and the chemistry of the tag. The variants associated with arthrogryposis multiplex congenita (AMC) display a more pronounced nuclear retention phenotype, suggesting a correlation between clinical severity and the degree of nuclear mis-localization. These results point to a neomorphic effect of truncated MAGEL2, which might contribute to the pathogenesis of SYS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576117" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576117">
        <p class="paperTitle">Geometric changes in the nucleoids of Deinococcus radiodurans reveal involvement of new proteins in recovery from ionizing radiation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cordova, A.; Niese, B.; Sweet, P.; Kamat, P.; Phillip, J. M.; Gordon, V.; Contreras, L. M.</p>
        <p class="info">Score: 5.4, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576117' target='https://doi.org/10.1101/2024.01.17.576117'> 10.1101/2024.01.17.576117</a></p>
        <p class="abstract">The extremophile Deinococcus radiodurans maintains a highly-organized and condensed nucleoid as its default state, possibly contributing to high tolerance of ionizing radiation (IR). Previous studies of the D. radiodurans nucleoid were limited by reliance on manual image annotation and qualitative metrics. Here, we introduce a high-throughput approach to quantify the geometric properties of cells and nucleoids, using confocal microscopy, digital reconstructions of cells, and computational modeling. We utilize this novel approach to investigate the dynamic process of nucleoid condensation in response to IR stress. Our quantitative analysis reveals that at the population level, exposure to IR induced nucleoid compaction and decreased size of D. radiodurans cells. Morphological analysis and clustering identified six distinct sub-populations across all tested experimental conditions. Results indicate that exposure to IR induces fractional redistributions of cells across sub-populations to exhibit morphologies that associate with greater nucleoid condensation, and decreased abundance of sub-populations associated with cell division. Nucleoid associated proteins (NAPs) may link nucleoid compaction and stress tolerance, but their roles in regulating compaction in D. radiodurans is unknown. Imaging of genomic mutants of known and suspected NAPs that contribute to nucleoid condensation found that deletion of nucleic acid binding proteins, not previously described as NAPs, can remodel the nucleoid by driving condensation or decondensation in the absence of stress and that IR increases the abundance of these morphological states. Thus, our integrated analysis introduces a new methodology for studying environmental influences on bacterial nucleoids and provides an opportunity to further investigate potential regulators of nucleoid condensation.

ImportanceD. radiodurans, an extremophile known for its stress tolerance, constitutively maintains a highly-condensed nucleoid. Qualitative studies have described nucleoid behavior under a variety of conditions. However, a lack of quantitative data regarding nucleoid organization and dynamics have limited our understanding of regulatory mechanisms controlling nucleoid organization in D. radiodurans. Here, we introduce a quantitative approach that enables high-throughput quantitative measurements of subcellular spatial characteristics in bacterial cells. Applying this to wild-type or single-protein-deficient populations of D. radiodurans subjected to ionizing radiation, we identified significant stress-responsive changes in cell shape, nucleoid organization, and morphology. These findings highlight this methodologys adaptability and capacity for quantitatively analyzing the cellular response to stressors for screening cellular proteins involved in bacterial nucleoid organization.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
