<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A bacterial expression cloning screen reveals tardigrade single-stranded DNA-binding proteins as potent desicco-protectants
Authors: Hibshman, J. D.; Clark-Hachtel, C. M.; Bloom, K. S.; Goldstein, B.
Score: 11.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554171
Tardigrades are known for their ability to survive remarkable extremes, but relatively little is known about the molecular underpinnings that allow for their extraordinary stress tolerance. We devised an expression cloning approach to screen for novel protectants from desiccation-tolerant tardigrades.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="A bacterial expression cloning screen reveals tardigrade single-stranded DNA-binding proteins as potent desicco-protectants
Authors: Hibshman, J. D.; Clark-Hachtel, C. M.; Bloom, K. S.; Goldstein, B.
Score: 11.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554171
Tardigrades are known for their ability to survive remarkable extremes, but relatively little is known about the molecular underpinnings that allow for their extraordinary stress tolerance. We devised an expression cloning approach to screen for novel protectants from desiccation-tolerant tardigrades." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="A bacterial expression cloning screen reveals tardigrade single-stranded DNA-binding proteins as potent desicco-protectants
Authors: Hibshman, J. D.; Clark-Hachtel, C. M.; Bloom, K. S.; Goldstein, B.
Score: 11.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554171
Tardigrades are known for their ability to survive remarkable extremes, but relatively little is known about the molecular underpinnings that allow for their extraordinary stress tolerance. We devised an expression cloning approach to screen for novel protectants from desiccation-tolerant tardigrades."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "A bacterial expression cloning screen reveals tardigrade single-stranded DNA-binding proteins as potent desicco-protectants\nAuthors: Hibshman, J. D.; Clark-Hachtel, C. M.; Bloom, K. S.; Goldstein, B.\nScore: 11.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554171\nTardigrades are known for their ability to survive remarkable extremes, but relatively little is known about the molecular underpinnings that allow for their extraordinary stress tolerance. We devised an expression cloning approach to screen for novel protectants from desiccation-tolerant tardigrades.",
  "keywords": [
    
  ],
  "articleBody": " A bacterial expression cloning screen reveals tardigrade single-stranded DNA-binding proteins as potent desicco-protectants\nAuthors: Hibshman, J. D.; Clark-Hachtel, C. M.; Bloom, K. S.; Goldstein, B.\nScore: 11.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554171\nTardigrades are known for their ability to survive remarkable extremes, but relatively little is known about the molecular underpinnings that allow for their extraordinary stress tolerance. We devised an expression cloning approach to screen for novel protectants from desiccation-tolerant tardigrades. We expressed cDNA libraries from the tardigrades Hypsibius exemplaris and Ramazzottius varieornatus in E. coli and subjected the bacteria to desiccation. Sequencing the populations of surviving bacteria revealed enrichment of mitochondrial single-stranded DNA-binding proteins (mtSSBs) from both tardigrade species. Expression of mtSSBs in bacteria improved desiccation survival to similar levels as some of the best protectants known to date. The DNA-binding activity of the oligonucleotide/oligosaccharide binding (OB) fold domain of mtSSB was necessary and sufficient to improve desiccation tolerance. By comparing the efficacy of mtSSBs from multiple organisms, we provide evidence that desicco-protection is a conserved feature of mtSSBs. More broadly, any OB-fold containing protein may act similarly to promote desiccation tolerance, likely by coating ssDNA to limit damage during desiccation. These results identify single-stranded DNA-binding activity as a potent contributor to extreme desiccation tolerance.\nConserved and species-specific transcriptional responses to daily programmed resistance exercise in rat and mouse.\nAuthors: Viggars, M. R.; Sutherland, H.; Cardozo, C. P.; Jarvis, J. C.\nScore: 10.6, Published: 2023-08-23 DOI: 10.1101/2023.08.23.552047\nMice are often used in gain or loss of function studies to understand how genes regulate metabolism and adaptation to exercise in skeletal muscle. Once-daily resistance training with electrical nerve stimulation produces hypertrophy of the dorsiflexors in rat, but not in mouse. Using implantable pulse generators, we assessed the acute transcriptional response (1-hour post exercise) after 2, 10 and 20 days of training in free living mice and rats using identical nerve stimulation paradigms. RNA-sequencing revealed strong concordance in the timecourse of many transcriptional responses in the tibialis anterior muscles of both species including responses related to stress responses/immediate-early genes, and collagen homeostasis, ribosomal subunits, autophagy and focal adhesion. However, pathways associated with energy metabolism including carbon metabolism, oxidative phosphorylation, mitochondrial translation, propanoate metabolism and valine, leucine and isoleucine degradation were oppositely regulated between species. These pathways were suppressed in the rat but upregulated in the mouse. Our transcriptional analysis suggests that although many pathways associated with growth show remarkable similarities between species, absence of an actual growth response in the mouse may be because the mouse prioritises energy metabolism, specifically replenishment of fuel stores and intermediate metabolites.\nDeep screening of proximal and distal splicing-regulatory elements in a native sequence context\nAuthors: Recinos, Y.; Ustianenko, D.; Yeh, Y.-T.; Wang, X.; Jacko, M.; Yesantharao, L. V.; Wu, Q.; Zhang, C.\nScore: 17.0, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554109\nPre-mRNA splicing, a key process in gene expression, can be therapeutically modulated using various drug modalities, including antisense oligonucleotides (ASOs). However, determining promising targets is impeded by the challenge of systematically mapping splicing-regulatory elements (SREs) in their native sequence context. Here, we use the catalytically dead CRISPR-RfxCas13d RNA-targeting system (dCas13d/gRNA) as a programmable platform to bind SREs and modulate splicing by competing against endogenous splicing factors. SpliceRUSH, a high-throughput screening method, was developed to map SREs in any gene of interest using a lentivirus gRNA library that tiles the genetic region, including distal intronic sequences. When applied to SMN2, a therapeutic target for spinal muscular atrophy, SpliceRUSH robustly identified not only known SREs, but also a novel distal intronic splicing enhancer, which can be targeted to alter exon 7 splicing using either dCas13d/gRNA or ASOs. This technology enables a deeper understanding of splicing regulation with applications for RNA-based drug discovery.\nDysregulation of extracellular vesicle protein cargo in female ME/CFS cases and sedentary controls in response to maximal exercise\nAuthors: Giloteaux, L.; Glass, K. A.; Germain, A.; Zhang, S.; Hanson, M. R.\nScore: 8.2, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555033\nIn healthy individuals, physical exercise improves cardiovascular health and muscle strength, alleviates fatigue, and reduces risk of chronic diseases. Although exercise is suggested as a lifestyle intervention to manage various chronic illnesses, it negatively affects people with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), who suffer from exercise intolerance. We hypothesized that altered extracellular vesicle (EV) signaling in ME/CFS patients after an exercise challenge may contribute to their prolonged and exacerbated negative response to exertion (post-exertional malaise). EVs were isolated by size exclusion chromatography from the plasma of 18 female ME/CFS patients and 17 age- and BMI-matched female sedentary controls at three time points: before, 15 minutes, and 24 hours after a maximal cardiopulmonary exercise test. EVs were characterized using nanoparticle tracking analysis and their protein cargo was quantified using Tandem Mass Tag-based (TMT) proteomics. The results show that exercise affects the EV proteome in ME/CFS patients differently than in healthy individuals and that changes in EV proteins after exercise are strongly correlated with symptom severity in ME/CFS. Differentially abundant proteins in ME/CFS patients vs. controls were involved in many pathways and systems, including coagulation processes, muscle contraction (both smooth and skeletal muscle), cytoskeletal proteins, the immune system, and brain signaling.\nCanonical and non-canonical contributions of human Dicer helicase domain in antiviral defense\nAuthors: Baldaccini, M.; Gaucherand, L.; Chane-Woon-Ming, B.; Messmer, M.; Gucciardi, F.; Pfeffer, S.\nScore: 7.4, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554407\nIn mammals, the co-existence of RNAi and the type I interferon response in somatic cells begs the question of their compatibility and relative contribution during viral infection. Previous studies provided hints that both mitigating co-factors and self-limiting properties of key proteins such as Dicer could explain the apparent inefficiency of antiviral RNAi. Indeed, the helicase domain of human Dicer limits its processing activity and acts as an interaction platform for co-factors that could hinder its function. We studied the involvement of several helicase-truncated mutants of human Dicer in the antiviral response. We show that all deletion mutants display an antiviral phenotype against alphaviruses and an enterovirus. While only one of them, Dicer N1, is antiviral in an RNAi-independent manner, they all require the expression of PKR to be active. To elucidate the mechanism underlying the antiviral phenotype of Dicer N1 expressing cells, we analyzed their transcriptome and found that many genes from the interferon and inflammatory response were upregulated. We could show that these genes appear to be controlled by transcription factors such as STAT-1, STAT-2, and NF-kB. Finally, we demonstrated that blocking the NF-kB pathway in Dicer N1 cells abrogated their antiviral phenotype. Our findings highlight the crosstalk between Dicer, PKR, and the IFN-I pathway, and suggest that human Dicer may have repurposed its helicase domain to prevent basal activation of antiviral and inflammatory pathways.\nThe ClpX chaperone and a hypermorphic FtsA variant with impaired self-interaction are mutually compensatory for coordinating Staphylococcus aureus cell division\nAuthors: Henriksen, C.; Baek, K.; Wacnik, K.; Gallay, C.; Veening, J.-W.; Foster, S. J.; Frees, D.\nScore: 7.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.554348\nBacterial cell division requires the coordinated assembly and disassembly of a large protein complex called the divisome, however, the exact role of molecular chaperones in this critical process remains unclear. In the important pathogenic bacterium Staphylococcus aureus, the ClpX chaperone is essential for growth at 30{degrees}C and microscopic analyses suggested that ClpX plays a temperature-dependent role in cell division. We here provide genetic evidence that ClpX unfoldase activity is a determinant for proper coordination of cell division by showing that a spontaneous G325V substitution in the ATP-binding domain of the essential FtsA cell division protein rescues growth and septum synthesis in a Staphylococcus aureus clpX mutant. The polymerization state of FtsA is thought to control initiation of bacterial septum synthesis and, while restoring the aberrant FtsA dynamics in clpX cells, the FtsAG325V variant displayed reduced ability to interact with itself and other cell division proteins. In wild-type cells, the ftsAG325V allele shared phenotypes with E. coli superfission ftsA mutants and accelerated the cell cycle, increased the risk of daughter cell lysis, and conferred sensitivity to heat and antibiotics inhibiting cell wall synthesis. Strikingly, lethality was mitigated by spontaneous mutations that inactivate ClpX. Taken together, our results suggest that ClpX promotes septum synthesis by antagonizing FtsA interactions and illuminates the critical role of a protein unfoldase in coordinating bacterial cell division. IMPORTANCEEssential biological processes, such as cell division, are performed by multiple proteins working together in dynamic functional complexes. In eukaryotic cells, the disassembly of such molecular machines is often assisted by molecular chaperones capable of unfolding proteins. The ClpX unfoldase is conserved from bacteria to humans, however, the roles of ClpX in bacterial cell biology remain relatively unexplored. By combining genetic methods with super-resolution microscopy techniques, we show here that ClpX and a mutant variant of the essential cell division protein FtsA mutually compensate for each other in controlling cell division of the pathogenic bacterium Staphylococcus aureus. The selected FtsAG325V variant has diminished self-interactions and restored the aberrant FtsA dynamics in clpX cells suggesting that ClpX promotes cell division by antagonizing FtsA protein interactions. This study, for the first time, illuminates the important role of protein unfoldases in the functioning of the divisome, a multienzyme complex fundamental to bacterial reproduction.\nNovel function of U7 snRNA in the repression of HERV1/LTR12s and lincRNAs in human cells\nAuthors: Plewka, P.; Szczesniak, M. W.; Pasieka, R.; Stepien, A.; Wanowska, E.; Makalowska, I.; Raczynska, K. D.\nScore: 6.9, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554787\nU7 snRNA is part of U7 snRNP, a complex required for the 3'end processing of replication-dependent histone pre-mRNAs in the S phase of the cell cycle. During this maturation event, the 5' region of U7 snRNA hybridizes with the highly complementary sequence present in the 3'UTR of histone pre-mRNAs, called histone downstream element, HDE. This base-pair interaction triggers subsequent reactions that eventually result in cleavage and release of mature histone transcripts. Intriguingly, U7 snRNP is constitutively expressed throughout the cell cycle and in nondividing cells, suggesting another function of U7 snRNA/snRNP in cells. Here, we show that several human endogenous retroviruses (HERVs) are significantly upregulated in HEK293T cells with U7 snRNA knockdown. They predominantly belong to the LTR12 class. Interestingly, some of them are located within long intergenic noncoding RNAs (lincRNAs), which in turn are upregulated in U7 snRNA knockdown cells as well. Significantly, both these HERV1/LTR12s and lincRNAs contain two or more sequence motifs that perfectly match the 5' end of U7 snRNA, which we called HDE-like motifs. We confirmed that mutations within the HDE-like motifs abrogate U7 snRNA regulatory function and stimulate the expression of selected lincRNAs. Furthermore, we demonstrate that U7 snRNA inhibits HERV1/LTR12 and lincRNA expression at the transcription level. We propose a mechanism in which U7 snRNA hampers binding/activity of NF-Y transcription factor to CCAAT motifs that are frequently found in LTRs as well as in a close proximity to HDE-like motifs. The expression of many HERV1/LTR12s and lincRNAs regulated by U7 snRNA seems to be tissue specific, therefore, we suggest that U7 snRNA plays a protective role in keeping deleterious genetic elements in silence in selected types of cells.\nNetwork-based prioritisation and validation of novel regulators of vascular smooth muscle cell proliferation in disease\nAuthors: Lambert, J.; Oc, S.; Worssam, M. D.; Häussler, D.; Figg, N. L.; Baxter, R.; Foote, K.; Finigan, A.; Mahbubani, K. T.; Bennett, M. R.; Krüger, A.; Spivakov, M.; Jorgensen, H. F.\nScore: 5.5, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554834\nAberrant vascular smooth muscle cell (VSMC) homeostasis and proliferation are hallmarks of vascular diseases causing heart attack and stroke. To elucidate molecular determinants governing VSMC proliferation, we reconstructed gene regulatory networks from single cell transcriptomics and epigenetic profiling. We find progressive activation of enhancers at disease-relevant loci in VSMCs that do not respond to injury and proliferation-predisposed cells. Our analysis suggests that while many transcription factors are shared, their target loci differ between VSMC states. Comparison of rewiring between VSMC subsets and in silico perturbation analysis prioritised novel regulators, including RUNX1 and the network target TIMP1. We experimentally validated that the pioneer factor RUNX1 increased VSMC responsiveness and show that TIMP1 feeds back to promote VSMC proliferation through CD74-mediated STAT3 signalling. Both RUNX1 and the TIMP1-CD74 axis were expressed in human VSMCs, at low frequency in normal arteries and increased in disease, suggesting clinical significance and potential as novel vascular disease targets.\nMutual homeostasis of charged proteins\nAuthors: Faraway, R.; Heaven, N. C.; Digby, H.; Wilkins, O. G.; Chakrabarti, A. M.; Iosub, I. A.; Knez, L.; Ameres, S. L.; Plaschka, C.; Ule, J.\nScore: 14.3, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554177\nProtein dosage is regulated to maintain cellular homeostasis and health. The dosage of proteins containing disordered low complexity domains (LCDs) must be particularly well-controlled to prevent aberrant disease, yet no mechanism to maintain homeostasis has been identified1, 2. Here we report a mutual homeostatic mechanism that controls the concentration of such proteins, termed interstasis, in which proteins with similar LCDs co-regulate their combined dosage through collective negative feedback. We focused on the mechanism that exploits the fundamental multivalency of GA-rich RNA regions that encode charged LCDs, including those with arginine-enriched mixed charge domains (R-MCDs). Modest variations in the abundance of an R-MCD protein change the properties of nuclear speckles, a protein-RNA condensate, selectively trapping multivalent GA-rich mRNAs to promote their nuclear retention. This interstasis depends on conserved codon biases, shared by amniotes, which enhance the multivalency of GA-rich regions encoding charged LCDs. The threshold of interstasis is modulated by CLK kinases, which affect the nuclear speckle localisation of proteins such as TRA2B, key binder of GA-rich RNAs. Notably, many classes of LCDs are encoded by RNA regions containing multivalency-enhancing codon biases, each preferentially bound by specific proteins, suggesting that interstasis might co-regulate many classes of functionally related LCD-containing proteins through dose-sensitivity of various types of protein-RNA condensates.\nNRF2 Dysregulation in Mice Leads to Inadequate Beta-Cell Mass Expansion during Pregnancy and Gestational Diabetes\nAuthors: Baumel-Alterzon, S.; Tse, I.; Heidery, F.; Garcia-Ocana, A.; Scott, D. K.\nScore: 3.4, Published: 2023-08-28 DOI: 10.1101/2023.08.28.555207\nThe late stages of the mammalian pregnancy are accompanied with increased insulin resistance due to the increased glucose demand of the growing fetus. Therefore, as a compensatory response to maintain the maternal normal blood glucose levels, maternal beta-cell mass expands leading to increased insulin release. Defects in beta-cell adaptive expansion during pregnancy can lead to gestational diabetes mellitus (GDM). Although the exact mechanisms that promote GDM are poorly understood, GDM strongly associates with impaired beta-cell proliferation and with increased levels of reactive oxygen species (ROS). Here, we show that NRF2 levels are upregulated in mouse beta-cells at gestation day 15 (GD15) concomitant with increased beta-cell proliferation. Importantly, mice with tamoxifen-induced beta-cell-specific NRF2 deletion display inhibition of beta-cell proliferation, increased beta-cell oxidative stress and elevated levels of beta-cell death at GD15. This results in attenuated beta-cell mass expansion and disturbed glucose homeostasis towards the end of pregnancy. Collectively, these results highlight the importance of NRF2-oxidative stress regulation in beta-cell mass adaptation to pregnancy and suggest NRF2 as a potential therapeutic target for treating GDM.\n",
  "wordCount" : "2486",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554171">
        <p class="paperTitle">A bacterial expression cloning screen reveals tardigrade single-stranded DNA-binding proteins as potent desicco-protectants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hibshman, J. D.; Clark-Hachtel, C. M.; Bloom, K. S.; Goldstein, B.</p>
        <p class="info">Score: 11.9, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554171' target='https://doi.org/10.1101/2023.08.21.554171'> 10.1101/2023.08.21.554171</a></p>
        <p class="abstract">Tardigrades are known for their ability to survive remarkable extremes, but relatively little is known about the molecular underpinnings that allow for their extraordinary stress tolerance. We devised an expression cloning approach to screen for novel protectants from desiccation-tolerant tardigrades. We expressed cDNA libraries from the tardigrades Hypsibius exemplaris and Ramazzottius varieornatus in E. coli and subjected the bacteria to desiccation. Sequencing the populations of surviving bacteria revealed enrichment of mitochondrial single-stranded DNA-binding proteins (mtSSBs) from both tardigrade species. Expression of mtSSBs in bacteria improved desiccation survival to similar levels as some of the best protectants known to date. The DNA-binding activity of the oligonucleotide/oligosaccharide binding (OB) fold domain of mtSSB was necessary and sufficient to improve desiccation tolerance. By comparing the efficacy of mtSSBs from multiple organisms, we provide evidence that desicco-protection is a conserved feature of mtSSBs. More broadly, any OB-fold containing protein may act similarly to promote desiccation tolerance, likely by coating ssDNA to limit damage during desiccation. These results identify single-stranded DNA-binding activity as a potent contributor to extreme desiccation tolerance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.552047">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.552047" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.552047">
        <p class="paperTitle">Conserved and species-specific transcriptional responses to daily programmed resistance exercise in rat and mouse.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.552047" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.552047" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Viggars, M. R.; Sutherland, H.; Cardozo, C. P.; Jarvis, J. C.</p>
        <p class="info">Score: 10.6, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.552047' target='https://doi.org/10.1101/2023.08.23.552047'> 10.1101/2023.08.23.552047</a></p>
        <p class="abstract">Mice are often used in gain or loss of function studies to understand how genes regulate metabolism and adaptation to exercise in skeletal muscle. Once-daily resistance training with electrical nerve stimulation produces hypertrophy of the dorsiflexors in rat, but not in mouse. Using implantable pulse generators, we assessed the acute transcriptional response (1-hour post exercise) after 2, 10 and 20 days of training in free living mice and rats using identical nerve stimulation paradigms. RNA-sequencing revealed strong concordance in the timecourse of many transcriptional responses in the tibialis anterior muscles of both species including responses related to  stress responses/immediate-early genes, and  collagen homeostasis,  ribosomal subunits,  autophagy and  focal adhesion. However, pathways associated with energy metabolism including  carbon metabolism,  oxidative phosphorylation,  mitochondrial translation,  propanoate metabolism and  valine, leucine and isoleucine degradation were oppositely regulated between species. These pathways were suppressed in the rat but upregulated in the mouse. Our transcriptional analysis suggests that although many pathways associated with growth show remarkable similarities between species, absence of an actual growth response in the mouse may be because the mouse prioritises energy metabolism, specifically replenishment of fuel stores and intermediate metabolites.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554109" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554109">
        <p class="paperTitle">Deep screening of proximal and distal splicing-regulatory elements in a native sequence context</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Recinos, Y.; Ustianenko, D.; Yeh, Y.-T.; Wang, X.; Jacko, M.; Yesantharao, L. V.; Wu, Q.; Zhang, C.</p>
        <p class="info">Score: 17.0, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554109' target='https://doi.org/10.1101/2023.08.21.554109'> 10.1101/2023.08.21.554109</a></p>
        <p class="abstract">Pre-mRNA splicing, a key process in gene expression, can be therapeutically modulated using various drug modalities, including antisense oligonucleotides (ASOs). However, determining promising targets is impeded by the challenge of systematically mapping splicing-regulatory elements (SREs) in their native sequence context. Here, we use the catalytically dead CRISPR-RfxCas13d RNA-targeting system (dCas13d/gRNA) as a programmable platform to bind SREs and modulate splicing by competing against endogenous splicing factors. SpliceRUSH, a high-throughput screening method, was developed to map SREs in any gene of interest using a lentivirus gRNA library that tiles the genetic region, including distal intronic sequences. When applied to SMN2, a therapeutic target for spinal muscular atrophy, SpliceRUSH robustly identified not only known SREs, but also a novel distal intronic splicing enhancer, which can be targeted to alter exon 7 splicing using either dCas13d/gRNA or ASOs. This technology enables a deeper understanding of splicing regulation with applications for RNA-based drug discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555033">
        <p class="paperTitle">Dysregulation of extracellular vesicle protein cargo in female ME/CFS cases and sedentary controls in response to maximal exercise</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giloteaux, L.; Glass, K. A.; Germain, A.; Zhang, S.; Hanson, M. R.</p>
        <p class="info">Score: 8.2, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555033' target='https://doi.org/10.1101/2023.08.28.555033'> 10.1101/2023.08.28.555033</a></p>
        <p class="abstract">In healthy individuals, physical exercise improves cardiovascular health and muscle strength, alleviates fatigue, and reduces risk of chronic diseases. Although exercise is suggested as a lifestyle intervention to manage various chronic illnesses, it negatively affects people with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), who suffer from exercise intolerance. We hypothesized that altered extracellular vesicle (EV) signaling in ME/CFS patients after an exercise challenge may contribute to their prolonged and exacerbated negative response to exertion (post-exertional malaise). EVs were isolated by size exclusion chromatography from the plasma of 18 female ME/CFS patients and 17 age- and BMI-matched female sedentary controls at three time points: before, 15 minutes, and 24 hours after a maximal cardiopulmonary exercise test. EVs were characterized using nanoparticle tracking analysis and their protein cargo was quantified using Tandem Mass Tag-based (TMT) proteomics. The results show that exercise affects the EV proteome in ME/CFS patients differently than in healthy individuals and that changes in EV proteins after exercise are strongly correlated with symptom severity in ME/CFS. Differentially abundant proteins in ME/CFS patients vs. controls were involved in many pathways and systems, including coagulation processes, muscle contraction (both smooth and skeletal muscle), cytoskeletal proteins, the immune system, and brain signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554407">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554407" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554407">
        <p class="paperTitle">Canonical and non-canonical contributions of human Dicer helicase domain in antiviral defense</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554407" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554407" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baldaccini, M.; Gaucherand, L.; Chane-Woon-Ming, B.; Messmer, M.; Gucciardi, F.; Pfeffer, S.</p>
        <p class="info">Score: 7.4, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554407' target='https://doi.org/10.1101/2023.08.23.554407'> 10.1101/2023.08.23.554407</a></p>
        <p class="abstract">In mammals, the co-existence of RNAi and the type I interferon response in somatic cells begs the question of their compatibility and relative contribution during viral infection. Previous studies provided hints that both mitigating co-factors and self-limiting properties of key proteins such as Dicer could explain the apparent inefficiency of antiviral RNAi. Indeed, the helicase domain of human Dicer limits its processing activity and acts as an interaction platform for co-factors that could hinder its function. We studied the involvement of several helicase-truncated mutants of human Dicer in the antiviral response. We show that all deletion mutants display an antiviral phenotype against alphaviruses and an enterovirus. While only one of them, Dicer N1, is antiviral in an RNAi-independent manner, they all require the expression of PKR to be active. To elucidate the mechanism underlying the antiviral phenotype of Dicer N1 expressing cells, we analyzed their transcriptome and found that many genes from the interferon and inflammatory response were upregulated. We could show that these genes appear to be controlled by transcription factors such as STAT-1, STAT-2, and NF-kB. Finally, we demonstrated that blocking the NF-kB pathway in Dicer N1 cells abrogated their antiviral phenotype. Our findings highlight the crosstalk between Dicer, PKR, and the IFN-I pathway, and suggest that human Dicer may have repurposed its helicase domain to prevent basal activation of antiviral and inflammatory pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554348">
        <p class="paperTitle">The ClpX chaperone and a hypermorphic FtsA variant with impaired self-interaction are mutually compensatory for coordinating Staphylococcus aureus cell division</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henriksen, C.; Baek, K.; Wacnik, K.; Gallay, C.; Veening, J.-W.; Foster, S. J.; Frees, D.</p>
        <p class="info">Score: 7.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554348' target='https://doi.org/10.1101/2023.08.22.554348'> 10.1101/2023.08.22.554348</a></p>
        <p class="abstract">Bacterial cell division requires the coordinated assembly and disassembly of a large protein complex called the divisome, however, the exact role of molecular chaperones in this critical process remains unclear. In the important pathogenic bacterium Staphylococcus aureus, the ClpX chaperone is essential for growth at 30{degrees}C and microscopic analyses suggested that ClpX plays a temperature-dependent role in cell division. We here provide genetic evidence that ClpX unfoldase activity is a determinant for proper coordination of cell division by showing that a spontaneous G325V substitution in the ATP-binding domain of the essential FtsA cell division protein rescues growth and septum synthesis in a Staphylococcus aureus clpX mutant. The polymerization state of FtsA is thought to control initiation of bacterial septum synthesis and, while restoring the aberrant FtsA dynamics in clpX cells, the FtsAG325V variant displayed reduced ability to interact with itself and other cell division proteins. In wild-type cells, the ftsAG325V allele shared phenotypes with E. coli superfission ftsA mutants and accelerated the cell cycle, increased the risk of daughter cell lysis, and conferred sensitivity to heat and antibiotics inhibiting cell wall synthesis. Strikingly, lethality was mitigated by spontaneous mutations that inactivate ClpX. Taken together, our results suggest that ClpX promotes septum synthesis by antagonizing FtsA interactions and illuminates the critical role of a protein unfoldase in coordinating bacterial cell division.

IMPORTANCEEssential biological processes, such as cell division, are performed by multiple proteins working together in dynamic functional complexes. In eukaryotic cells, the disassembly of such molecular machines is often assisted by molecular chaperones capable of unfolding proteins. The ClpX unfoldase is conserved from bacteria to humans, however, the roles of ClpX in bacterial cell biology remain relatively unexplored. By combining genetic methods with super-resolution microscopy techniques, we show here that ClpX and a mutant variant of the essential cell division protein FtsA mutually compensate for each other in controlling cell division of the pathogenic bacterium Staphylococcus aureus. The selected FtsAG325V variant has diminished self-interactions and restored the aberrant FtsA dynamics in clpX cells suggesting that ClpX promotes cell division by antagonizing FtsA protein interactions. This study, for the first time, illuminates the important role of protein unfoldases in the functioning of the divisome, a multienzyme complex fundamental to bacterial reproduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554787">
        <p class="paperTitle">Novel function of U7 snRNA in the repression of HERV1/LTR12s and lincRNAs in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plewka, P.; Szczesniak, M. W.; Pasieka, R.; Stepien, A.; Wanowska, E.; Makalowska, I.; Raczynska, K. D.</p>
        <p class="info">Score: 6.9, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554787' target='https://doi.org/10.1101/2023.08.25.554787'> 10.1101/2023.08.25.554787</a></p>
        <p class="abstract">U7 snRNA is part of U7 snRNP, a complex required for the 3&#39;end processing of replication-dependent histone pre-mRNAs in the S phase of the cell cycle. During this maturation event, the 5&#39; region of U7 snRNA hybridizes with the highly complementary sequence present in the 3&#39;UTR of histone pre-mRNAs, called histone downstream element, HDE. This base-pair interaction triggers subsequent reactions that eventually result in cleavage and release of mature histone transcripts. Intriguingly, U7 snRNP is constitutively expressed throughout the cell cycle and in nondividing cells, suggesting another function of U7 snRNA/snRNP in cells. Here, we show that several human endogenous retroviruses (HERVs) are significantly upregulated in HEK293T cells with U7 snRNA knockdown. They predominantly belong to the LTR12 class. Interestingly, some of them are located within long intergenic noncoding RNAs (lincRNAs), which in turn are upregulated in U7 snRNA knockdown cells as well. Significantly, both these HERV1/LTR12s and lincRNAs contain two or more sequence motifs that perfectly match the 5&#39; end of U7 snRNA, which we called HDE-like motifs. We confirmed that mutations within the HDE-like motifs abrogate U7 snRNA regulatory function and stimulate the expression of selected lincRNAs. Furthermore, we demonstrate that U7 snRNA inhibits HERV1/LTR12 and lincRNA expression at the transcription level. We propose a mechanism in which U7 snRNA hampers binding/activity of NF-Y transcription factor to CCAAT motifs that are frequently found in LTRs as well as in a close proximity to HDE-like motifs. The expression of many HERV1/LTR12s and lincRNAs regulated by U7 snRNA seems to be tissue specific, therefore, we suggest that U7 snRNA plays a protective role in keeping deleterious genetic elements in silence in selected types of cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554834">
        <p class="paperTitle">Network-based prioritisation and validation of novel regulators of vascular smooth muscle cell proliferation in disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lambert, J.; Oc, S.; Worssam, M. D.; Häussler, D.; Figg, N. L.; Baxter, R.; Foote, K.; Finigan, A.; Mahbubani, K. T.; Bennett, M. R.; Krüger, A.; Spivakov, M.; Jorgensen, H. F.</p>
        <p class="info">Score: 5.5, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554834' target='https://doi.org/10.1101/2023.08.25.554834'> 10.1101/2023.08.25.554834</a></p>
        <p class="abstract">Aberrant vascular smooth muscle cell (VSMC) homeostasis and proliferation are hallmarks of vascular diseases causing heart attack and stroke. To elucidate molecular determinants governing VSMC proliferation, we reconstructed gene regulatory networks from single cell transcriptomics and epigenetic profiling. We find progressive activation of enhancers at disease-relevant loci in VSMCs that do not respond to injury and proliferation-predisposed cells. Our analysis suggests that while many transcription factors are shared, their target loci differ between VSMC states. Comparison of rewiring between VSMC subsets and in silico perturbation analysis prioritised novel regulators, including RUNX1 and the network target TIMP1. We experimentally validated that the pioneer factor RUNX1 increased VSMC responsiveness and show that TIMP1 feeds back to promote VSMC proliferation through CD74-mediated STAT3 signalling. Both RUNX1 and the TIMP1-CD74 axis were expressed in human VSMCs, at low frequency in normal arteries and increased in disease, suggesting clinical significance and potential as novel vascular disease targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554177">
        <p class="paperTitle">Mutual homeostasis of charged proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faraway, R.; Heaven, N. C.; Digby, H.; Wilkins, O. G.; Chakrabarti, A. M.; Iosub, I. A.; Knez, L.; Ameres, S. L.; Plaschka, C.; Ule, J.</p>
        <p class="info">Score: 14.3, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554177' target='https://doi.org/10.1101/2023.08.21.554177'> 10.1101/2023.08.21.554177</a></p>
        <p class="abstract">Protein dosage is regulated to maintain cellular homeostasis and health. The dosage of proteins containing disordered low complexity domains (LCDs) must be particularly well-controlled to prevent aberrant disease, yet no mechanism to maintain homeostasis has been identified1, 2. Here we report a mutual homeostatic mechanism that controls the concentration of such proteins, termed interstasis, in which proteins with similar LCDs co-regulate their combined dosage through collective negative feedback. We focused on the mechanism that exploits the fundamental multivalency of GA-rich RNA regions that encode charged LCDs, including those with arginine-enriched mixed charge domains (R-MCDs). Modest variations in the abundance of an R-MCD protein change the properties of nuclear speckles, a protein-RNA condensate, selectively trapping multivalent GA-rich mRNAs to promote their nuclear retention. This interstasis depends on conserved codon biases, shared by amniotes, which enhance the multivalency of GA-rich regions encoding charged LCDs. The threshold of interstasis is modulated by CLK kinases, which affect the nuclear speckle localisation of proteins such as TRA2B, key binder of GA-rich RNAs. Notably, many classes of LCDs are encoded by RNA regions containing multivalency-enhancing codon biases, each preferentially bound by specific proteins, suggesting that interstasis might co-regulate many classes of functionally related LCD-containing proteins through dose-sensitivity of various types of protein-RNA condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555207">
        <p class="paperTitle">NRF2 Dysregulation in Mice Leads to Inadequate Beta-Cell Mass Expansion during Pregnancy and Gestational Diabetes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumel-Alterzon, S.; Tse, I.; Heidery, F.; Garcia-Ocana, A.; Scott, D. K.</p>
        <p class="info">Score: 3.4, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555207' target='https://doi.org/10.1101/2023.08.28.555207'> 10.1101/2023.08.28.555207</a></p>
        <p class="abstract">The late stages of the mammalian pregnancy are accompanied with increased insulin resistance due to the increased glucose demand of the growing fetus. Therefore, as a compensatory response to maintain the maternal normal blood glucose levels, maternal beta-cell mass expands leading to increased insulin release. Defects in beta-cell adaptive expansion during pregnancy can lead to gestational diabetes mellitus (GDM). Although the exact mechanisms that promote GDM are poorly understood, GDM strongly associates with impaired beta-cell proliferation and with increased levels of reactive oxygen species (ROS). Here, we show that NRF2 levels are upregulated in mouse beta-cells at gestation day 15 (GD15) concomitant with increased beta-cell proliferation. Importantly, mice with tamoxifen-induced beta-cell-specific NRF2 deletion display inhibition of beta-cell proliferation, increased beta-cell oxidative stress and elevated levels of beta-cell death at GD15. This results in attenuated beta-cell mass expansion and disturbed glucose homeostasis towards the end of pregnancy. Collectively, these results highlight the importance of NRF2-oxidative stress regulation in beta-cell mass adaptation to pregnancy and suggest NRF2 as a potential therapeutic target for treating GDM.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
