<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Programmable RNA writing with trans-splicing
Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.
Score: 112.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223
RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Programmable RNA writing with trans-splicing
Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.
Score: 112.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223
RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Programmable RNA writing with trans-splicing
Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.
Score: 112.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223
RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Programmable RNA writing with trans-splicing\nAuthors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.\nScore: 112.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223\nRNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing \u0026amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5\u0026#39; or 3\u0026#39; trans-splicing.",
  "keywords": [
    
  ],
  "articleBody": " Programmable RNA writing with trans-splicing\nAuthors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.\nScore: 112.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578223\nRNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing \u0026 Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5' or 3' trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington's disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntington's disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.\nRewriting endogenous human transcripts with trans-splicing\nAuthors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.\nScore: 70.3, Published: 2024-01-30 DOI: 10.1101/2024.01.29.577779\nSplicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.\nSingle-cell Rapid Capture Hybridization sequencing (scRaCH-seq) to reliably detect isoform usage and coding mutations in targeted genes at a single-cell level\nAuthors: Peng, H.; Jabbari, J. S.; Tian, L.; Chua, C. C.; Anstee, N. S.; Amin, N.; Wei, A. H.; Davidson, N. M.; Roberts, A. W.; Huang, D. C. S.; Ritchie, M. E.; Thijssen, R.\nScore: 14.1, Published: 2024-01-30 DOI: 10.1101/2024.01.30.577942\nSingle-cell long-read sequencing has transformed our understanding of isoform usage and the mutation heterogeneity between cells. Despite unbiased in-depth analysis, the low sequencing throughput often results in insufficient read coverage thereby limiting our ability to perform mutation calling for specific genes. Here, we developed a single-cell Rapid Capture Hybridization sequencing (scRaCH-seq) method that demonstrated high specificity and efficiency in capturing targeted transcripts using long-read sequencing, allowing an in-depth analysis of mutation status and transcript usage for genes of interest. The method includes creating a probe panel for transcript capture, using barcoded primers for pooling and efficient sequencing via Oxford Nanopore Technologies platforms. scRaCH-seq is applicable to stored and indexed single-cell cDNA which allows analysis to be combined with existing short-read RNA-seq datasets. In our investigation of BTK and SF3B1 genes in samples from patients with chronic lymphocytic leukaemia (CLL), we detected SF3B1 isoforms and mutations with high sensitivity. Integration with short-read scRNA-seq data revealed significant gene expression differences in SF3B1-mutated CLL cells, though it did not impact the sensitivity of the anti-cancer drug venetoclax. scRaCH-seqs capability to study long-read transcripts of multiple genes makes it a powerful tool for single-cell genomics.\nNELF coordinates Pol II transcription termination and DNA replication initiation\nAuthors: Nakayama, C.; Daigaku, Y.; Aoi, Y.; Fang, Q.; Kimura, H.; Shilatifard, A.; Tellier, M.; Nojima, T.\nScore: 11.6, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578294\nRegulation of RNA polymerase II (Pol II) transcription is closely associated with cell proliferation. However, it remains unclear how the Pol II transcription program is altered in cancer to favour cell growth. Here, we find that gene expression of NELFCD, a known negative elongation factor, is up-regulated in colorectal tumours. To dissect the direct role of NELF-C on Pol II transcription in such cancer, we employed an auxin-dependent protein degradation system for NELF-C in combination with nascent transcript sequencing technologies. Strikingly, we demonstrated that the acute loss of NELF-C protein globally perturbs Pol II transcription termination and also increases transcription elongation rate, independently of promoter-proximal Pol II pausing. This results in Pol II transcription into DNA replication initiation zones, and may link to failure of the cell cycle transition into S phase. We anticipate that NELF will be a potential therapeutic target to restrict colorectal cancers by promoting transcription-replication conflict. HIGHLIGHTSO_LIExpression of NELFCD transcript is up-regulated in colorectal tumors C_LIO_LINELF-C protein is mandatory for the transition between G1-S phases during cell cycle C_LIO_LINELF-C loss impairs transcription termination independently of Pol II promoter-proximal pausing C_LIO_LINELF-C loss leads Pol II to invade DNA replication initiation zones C_LI\nAn endothelial SOX18-mevalonate pathway axis enables repurposing of statins for infantile hemangioma\nAuthors: Holm, A.; Graus, M.; Wylie-Sears, J.; Borgelt, L.; Tan, J. W. H.; Nasim, S.; Chung, L.; Jain, A.; Sun, M.; Sun, L.; Brouillard, P.; Lekwuttikarn, R.; Kozakewich, H.; Qi, Y. J.; Teng, J.; Mulliken, J.; Vikkula, M.; Francois, M.; Bischoff, J.\nScore: 8.1, Published: 2024-02-02 DOI: 10.1101/2024.01.29.577829\nInfantile hemangioma (IH) is the most common tumor in children and a paradigm for pathological vasculogenesis, angiogenesis and regression. Propranolol is the mainstay of treatment for IH. It inhibits hemangioma vessel formation via a {beta}-adrenergic receptor independent off-target effect of its R(+) enantiomer on the endothelial specific transcription factor sex-determining region Y (SRY) box transcription factor 18 (SOX18). Transcriptomic profiling of patient-derived hemangioma stem cells uncovered the mevalonate pathway (MVP) as a target of R(+) propranolol. Loss of SOX18 function confirmed R(+) propranolol mode of action on the MVP. Functional validation in preclinical IH models revealed that statins - targeting the MVP - are potent inhibitors of hemangioma vessel formation. We propose a novel SOX18-MVP-axis as a central regulator of IH pathogenesis and suggest statin repurposing to treat IH. Our findings reveal novel pleiotropic effects of beta-blockers and statins acting on the SOX18-MVP axis to disable an endothelial specific program in IH, which may impact other scenarios involving pathological vasculogenesis and angiogenesis.\nAnalyzing the implications of protein folding delay caused by translation\nAuthors: Houben, B.; Duran-Romana, R.; Fernandez Migens, P.; Rousseau, F.; Schymkowitz, J.\nScore: 7.4, Published: 2024-01-27 DOI: 10.1101/2024.01.27.577370\nBecause of vectorial protein production, residues that interact in the native protein structure but are distantly separated in the primary sequence are unavailable simultaneously. Instead, there is a temporal delay during which the N-terminal interaction partner is vulnerable to off-pathway, non-native interactions. In this analysis, we introduce \"FoldDelay\" (FD), a metric that integrates the topological pattern of atomic interactions of the native structure with translation kinetics to quantify such time delays. The FD metric reveals that many proteins, particularly at eukaryotic translation rates, exhibit residues with FDs in the range of tens of seconds. These residues, predominantly in well-structured, buried regions, often coincide with predicted aggregation-prone regions. We show a correlation between FD and co-translational engagement by the yeast Hsp70 chaperone Ssb, suggesting that fold-delayed regions have a propensity to misfold. In support of this, we show that proteins with high FDs are more frequently co-translationally ubiquitinated and prone to aggregate upon Ssb deletion. Finally, we find that FD cannot be adequately reduced through codon optimization, highlighting the importance of co-translational chaperones to shield these vulnerable regions. This work offers insights into co-translational proteostasis and the delicate balance between efficient folding and potential misfolding and aggregation during translation.\nThe Impact of Mismatches Within the RNA: DNA Hybrid in the Transposon-Encoded Type I-F CRISPR-Cas System\nAuthors: Alalmaie, A.; Khashan, R.\nScore: 3.0, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578492\nRecent research has revealed a collaboration between type I-F CRISPR systems and Tn7 transposons in certain bacteria, leading to the discovery of a new gene-editing tool called INTEGRATE. This system integrates transposons into the target strand without introducing a double-strand break or repair mechanism, making it highly promising. The published results showed that a full match between the spacer region of crRNA and the target DNA is necessary to form a stable and complete R-loop, which is critical for accurate integration. PAM-distal mismatches affected RNA-guided DNA transposition differently, with mismatches in positions 25-28 completely blocking the process, while mismatches in positions 29-32 were tolerated. To understand the impact of PAM-distal mismatches on the R-loop stabilization and transposition process, classical all-atom molecular dynamics simulations were conducted using three independent models, including one with a complete RNA-DNA match and two with PAM-distal mismatches located at positions 25-32. Our results suggest that the stable rotation of Cas8-HB is a critical step in the interaction with TniQ and initiation of the DNA transposition process. Network analysis techniques were employed to investigate the communication pathways within Cas8 and TniQ dimers, including eigenvector centrality and correlation analysis. These techniques revealed that certain amino acids in TniQ and Cas8 were highly central to the communication pathways, as they exhibited significant changes in centrality under both mismatches. The findings discussed in this study provide valuable insights into the mechanisms involved in the DNA transposition process and shed light on how PAM-distal mismatches can affect these mechanisms.\nMechanistic Insight into the Conformational Changes of Cas8 upon Binding to Different PAM Sequences in the Transposon-Encoded Type I-F CRISPR-Cas System\nAuthors: Alalmaie, A.; Khashan, R.\nScore: 2.8, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578494\nThe INTEGRATE system is a gene-editing approach that offers advantages over the widely used CRISPR-Cas9 system. It does not introduce double strand breaks in the target DNA but rather integrates the desired DNA sequence directly into it. The first step in the integration process is PAM recognition, which is critical to understanding and optimizing the system. Experimental testing revealed varying integration efficiencies of different PAM mutants, and computational simulations were carried out to gain mechanistic insight into the conformational changes of Cas8 during PAM recognition. Our results showed that the interaction between Arg246 and Guanine at position (-1) of the target strand is critical for PAM recognition. We found that unfavorable interactions in the 5'-AC-3' PAM mutant disrupted this interaction and may be responsible for its 0% integration efficiency. Additionally, we discovered that PAM sequences not only initiate the integration process but also regulate it through an allosteric mechanism that connects the N-terminal domain and the helical bundle of Cas8. This allosteric regulation was present in all PAMs tested, even those with lower integration efficiencies, such as 5'-TC-3' and 5'-AC-3'. We identified the Cas8 residues that are involved in this regulation. Our findings provide valuable insights into PAM recognition mechanisms in the INTEGRATE system and can help improve the gene-editing technology.\nBeyond static structures: quantitative dynamics in the eukaryotic RNA exosome complex\nAuthors: Liebau, J.; Lazzaretti, D.; Bichler, A.; Pilsl, M.; Sprangers, R.\nScore: 2.6, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577622\nMolecular machines play pivotal roles in all biological processes. Most structural methods, however, are unable to directly probe molecular motions. We demonstrate that dedicated NMR experiments can provide quantitative insights into functionally important dynamic regions in very large asymmetric protein complexes. We establish this for the 410 kDa eukaryotic RNA exosome complex that contains ten distinct protein chains. NMR data reveal site-specific interactions and conformational changes in regions that are invisible in static cryo-EM and crystal structures. In particular, we identify a flexible plug region that blocks an aberrant route of the RNA to the active site. Our work thus demonstrates that a combination of state-of-the-art structural biology methods can provide quantitative insights into molecular machines that go well beyond static images.\nDiscovery and Characterization of a Pan-betacoronavirus S2-binding antibody\nAuthors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.\nScore: 323.8, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575741\nSUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.\n",
  "wordCount" : "2374",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578223" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578223">
        <p class="paperTitle">Programmable RNA writing with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitt-Ulms, C.; Kayabolen, A.; Manero-Carranza, M.; Zhou, N.; Donnelly, K.; Nuccio, S. P.; Kato, K.; Nishimasu, H.; Gootenberg, J. S.; Abudayyeh, O. O.</p>
        <p class="info">Score: 112.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578223' target='https://doi.org/10.1101/2024.01.31.578223'> 10.1101/2024.01.31.578223</a></p>
        <p class="abstract">RNA editing offers the opportunity to introduce either stable or transient modifications to nucleic acid sequence without permanent off-target effects, but installation of arbitrary edits into the transcriptome is currently infeasible. Here, we describe Programmable RNA Editing &amp; Cleavage for Insertion, Substitution, and Erasure (PRECISE), a versatile RNA editing method for writing RNA of arbitrary length and sequence into existing pre-mRNAs via 5&#39; or 3&#39; trans-splicing. In trans-splicing, an exogenous template is introduced to compete with the endogenous pre-mRNA, allowing for replacement of upstream or downstream exon sequence. Using Cas7-11 cleavage of pre-mRNAs to bias towards editing outcomes, we boost the efficiency of RNA trans-splicing by 10-100 fold, achieving editing rates between 5-50% and 85% on endogenous and reporter transcripts, respectively, while maintaining high-fidelity. We demonstrate PRECISE editing across 11 distinct endogenous transcripts of widely varying expression levels, showcasing more than 50 types of edits, including all 12 possible transversions and transitions, insertions ranging from 1 to 1,863 nucleotides, and deletions. We show high efficiency replacement of exon 4 of MECP2, addressing most mutations that drive the Rett Syndrome; editing of SHANK3 transcripts, a gene involved in Autism; and replacement of exon 1 of HTT, removing the hallmark repeat expansions of Huntington&#39;s disease. Whole transcriptome sequencing reveals the high precision of PRECISE editing and lack of off-target trans-splicing activity. Furthermore, we combine payload engineering and ribozymes for protein-free, high-efficiency trans-splicing, with demonstrated efficiency in editing HTT exon 1 via AAV delivery. We show that the high activity of PRECISE editing enables editing in non-dividing neurons and patient-derived Huntington&#39;s disease fibroblasts. PRECISE editing markedly broadens the scope of genetic editing, is straightforward to deliver over existing gene editing tools like prime editing, lacks permanent off-targets, and can enable any type of genetic edit large or small, including edits not otherwise possible with existing RNA base editors, widening the spectrum of addressable diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577779">
        <p class="paperTitle">Rewriting endogenous human transcripts with trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chandrasekaran, S. S.; Tau, C.; Nemeth, M.; Pawluk, A.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 70.3, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577779' target='https://doi.org/10.1101/2024.01.29.577779'> 10.1101/2024.01.29.577779</a></p>
        <p class="abstract">Splicing bridges the gap between static DNA sequence and the diverse and dynamic set of protein products that execute a genes biological functions. While exon skipping technologies enable influence over splice site selection, many desired perturbations to the transcriptome require replacement or addition of exogenous exons to target mRNAs: for example, to replace disease-causing exons, repair truncated proteins, or engineer protein fusions. Here, we report the development of RNA-guided trans-splicing with Cas editor (RESPLICE), inspired by the rare, natural process of trans-splicing that joins exons from two distinct primary transcripts. RESPLICE uses two orthogonal RNA-targeting CRISPR effectors to co-localize a trans-splicing pre-mRNA and to inhibit the cis-splicing reaction, respectively. We demonstrate efficient, specific, and programmable trans-splicing of multi-kilobase RNA cargo into nine endogenous transcripts across two human cell types, achieving up to 45% trans-splicing efficiency in bulk, or 90% when sorting for high effector expression. Our results present RESPLICE as a new mode of RNA editing for fine-tuned and transient control of cellular programs without permanent alterations to the genetic code.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577942" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577942">
        <p class="paperTitle">Single-cell Rapid Capture Hybridization sequencing (scRaCH-seq) to reliably detect isoform usage and coding mutations in targeted genes at a single-cell level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, H.; Jabbari, J. S.; Tian, L.; Chua, C. C.; Anstee, N. S.; Amin, N.; Wei, A. H.; Davidson, N. M.; Roberts, A. W.; Huang, D. C. S.; Ritchie, M. E.; Thijssen, R.</p>
        <p class="info">Score: 14.1, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577942' target='https://doi.org/10.1101/2024.01.30.577942'> 10.1101/2024.01.30.577942</a></p>
        <p class="abstract">Single-cell long-read sequencing has transformed our understanding of isoform usage and the mutation heterogeneity between cells. Despite unbiased in-depth analysis, the low sequencing throughput often results in insufficient read coverage thereby limiting our ability to perform mutation calling for specific genes. Here, we developed a single-cell Rapid Capture Hybridization sequencing (scRaCH-seq) method that demonstrated high specificity and efficiency in capturing targeted transcripts using long-read sequencing, allowing an in-depth analysis of mutation status and transcript usage for genes of interest. The method includes creating a probe panel for transcript capture, using barcoded primers for pooling and efficient sequencing via Oxford Nanopore Technologies platforms. scRaCH-seq is applicable to stored and indexed single-cell cDNA which allows analysis to be combined with existing short-read RNA-seq datasets. In our investigation of BTK and SF3B1 genes in samples from patients with chronic lymphocytic leukaemia (CLL), we detected SF3B1 isoforms and mutations with high sensitivity. Integration with short-read scRNA-seq data revealed significant gene expression differences in SF3B1-mutated CLL cells, though it did not impact the sensitivity of the anti-cancer drug venetoclax. scRaCH-seqs capability to study long-read transcripts of multiple genes makes it a powerful tool for single-cell genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578294" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578294">
        <p class="paperTitle">NELF coordinates Pol II transcription termination and DNA replication initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakayama, C.; Daigaku, Y.; Aoi, Y.; Fang, Q.; Kimura, H.; Shilatifard, A.; Tellier, M.; Nojima, T.</p>
        <p class="info">Score: 11.6, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578294' target='https://doi.org/10.1101/2024.01.31.578294'> 10.1101/2024.01.31.578294</a></p>
        <p class="abstract">Regulation of RNA polymerase II (Pol II) transcription is closely associated with cell proliferation. However, it remains unclear how the Pol II transcription program is altered in cancer to favour cell growth. Here, we find that gene expression of NELFCD, a known negative elongation factor, is up-regulated in colorectal tumours. To dissect the direct role of NELF-C on Pol II transcription in such cancer, we employed an auxin-dependent protein degradation system for NELF-C in combination with nascent transcript sequencing technologies. Strikingly, we demonstrated that the acute loss of NELF-C protein globally perturbs Pol II transcription termination and also increases transcription elongation rate, independently of promoter-proximal Pol II pausing. This results in Pol II transcription into DNA replication initiation zones, and may link to failure of the cell cycle transition into S phase. We anticipate that NELF will be a potential therapeutic target to restrict colorectal cancers by promoting transcription-replication conflict.

HIGHLIGHTSO_LIExpression of NELFCD transcript is up-regulated in colorectal tumors
C_LIO_LINELF-C protein is mandatory for the transition between G1-S phases during cell cycle
C_LIO_LINELF-C loss impairs transcription termination independently of Pol II promoter-proximal pausing
C_LIO_LINELF-C loss leads Pol II to invade DNA replication initiation zones
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577829">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577829" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577829">
        <p class="paperTitle">An endothelial SOX18-mevalonate pathway axis enables repurposing of statins for infantile hemangioma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577829" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577829" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holm, A.; Graus, M.; Wylie-Sears, J.; Borgelt, L.; Tan, J. W. H.; Nasim, S.; Chung, L.; Jain, A.; Sun, M.; Sun, L.; Brouillard, P.; Lekwuttikarn, R.; Kozakewich, H.; Qi, Y. J.; Teng, J.; Mulliken, J.; Vikkula, M.; Francois, M.; Bischoff, J.</p>
        <p class="info">Score: 8.1, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577829' target='https://doi.org/10.1101/2024.01.29.577829'> 10.1101/2024.01.29.577829</a></p>
        <p class="abstract">Infantile hemangioma (IH) is the most common tumor in children and a paradigm for pathological vasculogenesis, angiogenesis and regression. Propranolol is the mainstay of treatment for IH. It inhibits hemangioma vessel formation via a {beta}-adrenergic receptor independent off-target effect of its R(&#43;) enantiomer on the endothelial specific transcription factor sex-determining region Y (SRY) box transcription factor 18 (SOX18). Transcriptomic profiling of patient-derived hemangioma stem cells uncovered the mevalonate pathway (MVP) as a target of R(&#43;) propranolol. Loss of SOX18 function confirmed R(&#43;) propranolol mode of action on the MVP. Functional validation in preclinical IH models revealed that statins - targeting the MVP - are potent inhibitors of hemangioma vessel formation. We propose a novel SOX18-MVP-axis as a central regulator of IH pathogenesis and suggest statin repurposing to treat IH. Our findings reveal novel pleiotropic effects of beta-blockers and statins acting on the SOX18-MVP axis to disable an endothelial specific program in IH, which may impact other scenarios involving pathological vasculogenesis and angiogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577370">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577370" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577370">
        <p class="paperTitle">Analyzing the implications of protein folding delay caused by translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577370" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577370" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Houben, B.; Duran-Romana, R.; Fernandez Migens, P.; Rousseau, F.; Schymkowitz, J.</p>
        <p class="info">Score: 7.4, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577370' target='https://doi.org/10.1101/2024.01.27.577370'> 10.1101/2024.01.27.577370</a></p>
        <p class="abstract">Because of vectorial protein production, residues that interact in the native protein structure but are distantly separated in the primary sequence are unavailable simultaneously. Instead, there is a temporal delay during which the N-terminal interaction partner is vulnerable to off-pathway, non-native interactions. In this analysis, we introduce &#34;FoldDelay&#34; (FD), a metric that integrates the topological pattern of atomic interactions of the native structure with translation kinetics to quantify such time delays. The FD metric reveals that many proteins, particularly at eukaryotic translation rates, exhibit residues with FDs in the range of tens of seconds. These residues, predominantly in well-structured, buried regions, often coincide with predicted aggregation-prone regions. We show a correlation between FD and co-translational engagement by the yeast Hsp70 chaperone Ssb, suggesting that fold-delayed regions have a propensity to misfold. In support of this, we show that proteins with high FDs are more frequently co-translationally ubiquitinated and prone to aggregate upon Ssb deletion. Finally, we find that FD cannot be adequately reduced through codon optimization, highlighting the importance of co-translational chaperones to shield these vulnerable regions. This work offers insights into co-translational proteostasis and the delicate balance between efficient folding and potential misfolding and aggregation during translation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578492">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578492" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578492">
        <p class="paperTitle">The Impact of Mismatches Within the RNA: DNA Hybrid in the Transposon-Encoded Type I-F CRISPR-Cas System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578492" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578492" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alalmaie, A.; Khashan, R.</p>
        <p class="info">Score: 3.0, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578492' target='https://doi.org/10.1101/2024.02.01.578492'> 10.1101/2024.02.01.578492</a></p>
        <p class="abstract">Recent research has revealed a collaboration between type I-F CRISPR systems and Tn7 transposons in certain bacteria, leading to the discovery of a new gene-editing tool called INTEGRATE. This system integrates transposons into the target strand without introducing a double-strand break or repair mechanism, making it highly promising. The published results showed that a full match between the spacer region of crRNA and the target DNA is necessary to form a stable and complete R-loop, which is critical for accurate integration. PAM-distal mismatches affected RNA-guided DNA transposition differently, with mismatches in positions 25-28 completely blocking the process, while mismatches in positions 29-32 were tolerated. To understand the impact of PAM-distal mismatches on the R-loop stabilization and transposition process, classical all-atom molecular dynamics simulations were conducted using three independent models, including one with a complete RNA-DNA match and two with PAM-distal mismatches located at positions 25-32. Our results suggest that the stable rotation of Cas8-HB is a critical step in the interaction with TniQ and initiation of the DNA transposition process. Network analysis techniques were employed to investigate the communication pathways within Cas8 and TniQ dimers, including eigenvector centrality and correlation analysis. These techniques revealed that certain amino acids in TniQ and Cas8 were highly central to the communication pathways, as they exhibited significant changes in centrality under both mismatches. The findings discussed in this study provide valuable insights into the mechanisms involved in the DNA transposition process and shed light on how PAM-distal mismatches can affect these mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578494">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578494" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578494">
        <p class="paperTitle">Mechanistic Insight into the Conformational Changes of Cas8 upon Binding to Different PAM Sequences in the Transposon-Encoded Type I-F CRISPR-Cas System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578494" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578494" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alalmaie, A.; Khashan, R.</p>
        <p class="info">Score: 2.8, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578494' target='https://doi.org/10.1101/2024.02.01.578494'> 10.1101/2024.02.01.578494</a></p>
        <p class="abstract">The INTEGRATE system is a gene-editing approach that offers advantages over the widely used CRISPR-Cas9 system. It does not introduce double strand breaks in the target DNA but rather integrates the desired DNA sequence directly into it. The first step in the integration process is PAM recognition, which is critical to understanding and optimizing the system. Experimental testing revealed varying integration efficiencies of different PAM mutants, and computational simulations were carried out to gain mechanistic insight into the conformational changes of Cas8 during PAM recognition. Our results showed that the interaction between Arg246 and Guanine at position (-1) of the target strand is critical for PAM recognition. We found that unfavorable interactions in the 5&#39;-AC-3&#39; PAM mutant disrupted this interaction and may be responsible for its 0% integration efficiency. Additionally, we discovered that PAM sequences not only initiate the integration process but also regulate it through an allosteric mechanism that connects the N-terminal domain and the helical bundle of Cas8. This allosteric regulation was present in all PAMs tested, even those with lower integration efficiencies, such as 5&#39;-TC-3&#39; and 5&#39;-AC-3&#39;. We identified the Cas8 residues that are involved in this regulation. Our findings provide valuable insights into PAM recognition mechanisms in the INTEGRATE system and can help improve the gene-editing technology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577622" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577622">
        <p class="paperTitle">Beyond static structures: quantitative dynamics in the eukaryotic RNA exosome complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liebau, J.; Lazzaretti, D.; Bichler, A.; Pilsl, M.; Sprangers, R.</p>
        <p class="info">Score: 2.6, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577622' target='https://doi.org/10.1101/2024.01.28.577622'> 10.1101/2024.01.28.577622</a></p>
        <p class="abstract">Molecular machines play pivotal roles in all biological processes. Most structural methods, however, are unable to directly probe molecular motions. We demonstrate that dedicated NMR experiments can provide quantitative insights into functionally important dynamic regions in very large asymmetric protein complexes. We establish this for the 410 kDa eukaryotic RNA exosome complex that contains ten distinct protein chains. NMR data reveal site-specific interactions and conformational changes in regions that are invisible in static cryo-EM and crystal structures. In particular, we identify a flexible plug region that blocks an aberrant route of the RNA to the active site. Our work thus demonstrates that a combination of state-of-the-art structural biology methods can provide quantitative insights into molecular machines that go well beyond static images.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575741" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575741">
        <p class="paperTitle">Discovery and Characterization of a Pan-betacoronavirus S2-binding antibody</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, N. V.; Wall, S.; Kramer, K.; Holt, C.; Periasamy, S.; Richardson, S.; Suryadevara, N.; Andreano, E.; Paciello, I.; Pierleoni, G.; Piccini, G.; Huang, Y.; Ge, P.; Allen, J.; Uno, N.; Shiakolas, A.; Pilewski, K.; Nargi, R.; Abu-Shmais, A.; Sutton, R.; Parks, R.; Haynes, B.; Carnahan, R.; Crowe, J.; Montomoli, E.; Rappuoli, R.; Bukreyev, A.; Ross, T.; Sautto, G.; Georgiev, I.; McLellan, J.</p>
        <p class="info">Score: 323.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575741' target='https://doi.org/10.1101/2024.01.15.575741'> 10.1101/2024.01.15.575741</a></p>
        <p class="abstract">SUMMARY/ABSTRACTThree coronaviruses have spilled over from animal reservoirs into the human population and caused deadly epidemics or pandemics. The continued emergence of coronaviruses highlights the need for pan-coronavirus interventions for effective pandemic preparedness. Here, using LIBRA-seq, we report a panel of 50 coronavirus antibodies isolated from human B cells. Of these antibodies, 54043-5 was shown to bind the S2 subunit of spike proteins from alpha-, beta-, and deltacoronaviruses. A cryo-EM structure of 54043-5 bound to the pre-fusion S2 subunit of the SARS-CoV-2 spike defined an epitope at the apex of S2 that is highly conserved among betacoronaviruses. Although non-neutralizing, 54043-5 induced Fc-dependent antiviral responses, including ADCC and ADCP. In murine SARS-CoV-2 challenge studies, protection against disease was observed after introduction of Leu234Ala, Leu235Ala, and Pro329Gly (LALA-PG) substitutions in the Fc region of 54043-5. Together, these data provide new insights into the protective mechanisms of non-neutralizing antibodies and define a broadly conserved epitope within the S2 subunit.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
