<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Genome-Wide CRISPR Screen Identifies Sortilin as the Receptor Responsible for Galectin-1 Lysosomal Trafficking
Authors: Donnelly, J.; Kamber, R. A.; Wisnovsky, S.; Roberts, D. S.; Peltan, E. L.; Bassik, M. C.; Bertozzi, C. R.
Score: 26.3, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574113
Galectins are a family of mammalian glycan-binding proteins that have been implicated as regulators of myriad cellular processes including cell migration, apoptosis, and immune modulation. Several members of this family, such as galectin-1, exhibit both cell-surface and intracellular functions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="A Genome-Wide CRISPR Screen Identifies Sortilin as the Receptor Responsible for Galectin-1 Lysosomal Trafficking
Authors: Donnelly, J.; Kamber, R. A.; Wisnovsky, S.; Roberts, D. S.; Peltan, E. L.; Bassik, M. C.; Bertozzi, C. R.
Score: 26.3, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574113
Galectins are a family of mammalian glycan-binding proteins that have been implicated as regulators of myriad cellular processes including cell migration, apoptosis, and immune modulation. Several members of this family, such as galectin-1, exhibit both cell-surface and intracellular functions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="A Genome-Wide CRISPR Screen Identifies Sortilin as the Receptor Responsible for Galectin-1 Lysosomal Trafficking
Authors: Donnelly, J.; Kamber, R. A.; Wisnovsky, S.; Roberts, D. S.; Peltan, E. L.; Bassik, M. C.; Bertozzi, C. R.
Score: 26.3, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574113
Galectins are a family of mammalian glycan-binding proteins that have been implicated as regulators of myriad cellular processes including cell migration, apoptosis, and immune modulation. Several members of this family, such as galectin-1, exhibit both cell-surface and intracellular functions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "A Genome-Wide CRISPR Screen Identifies Sortilin as the Receptor Responsible for Galectin-1 Lysosomal Trafficking\nAuthors: Donnelly, J.; Kamber, R. A.; Wisnovsky, S.; Roberts, D. S.; Peltan, E. L.; Bassik, M. C.; Bertozzi, C. R.\nScore: 26.3, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574113\nGalectins are a family of mammalian glycan-binding proteins that have been implicated as regulators of myriad cellular processes including cell migration, apoptosis, and immune modulation. Several members of this family, such as galectin-1, exhibit both cell-surface and intracellular functions.",
  "keywords": [
    
  ],
  "articleBody": " A Genome-Wide CRISPR Screen Identifies Sortilin as the Receptor Responsible for Galectin-1 Lysosomal Trafficking\nAuthors: Donnelly, J.; Kamber, R. A.; Wisnovsky, S.; Roberts, D. S.; Peltan, E. L.; Bassik, M. C.; Bertozzi, C. R.\nScore: 26.3, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574113\nGalectins are a family of mammalian glycan-binding proteins that have been implicated as regulators of myriad cellular processes including cell migration, apoptosis, and immune modulation. Several members of this family, such as galectin-1, exhibit both cell-surface and intracellular functions. Interestingly, galectin-1 can be found in the endomembrane system, nucleus, or cytosol, as well as on the cell surface. The mechanisms by which galectin-1 traffics between cellular compartments, including its unconventional secretion and internalization processes, are poorly understood. Here, we determined the pathways by which exogenous galectin-1 enters cells and explored its capacity as a delivery vehicle for protein and siRNA therapeutics. We used a galectin-1-toxin conjugate, modelled on antibody-drug conjugates, as a selection tool in a genome-wide CRISPR screen. We discovered that galectin-1 interacts with the endosome-lysosome trafficking receptor sortilin in a glycan-dependent manner, which regulates galectin-1 trafficking to the lysosome. Further, we show that this pathway can be exploited for delivery of a functional siRNA. This study sheds light on the mechanisms by which galectin-1 is internalized by cells and suggests a new strategy for intracellular drug delivery via galectin-1 conjugation.\nShieldin and CST co-orchestrate DNA polymerase-dependent tailed-end joining reactions independently of 53BP1-governed repair pathway choice\nAuthors: King, A.; Reichl, P.; Metson, J. S.; Parker, R.; Munro, D.; Oliveira, C.; Becker, J. R.; Biggs, D.; Preece, C.; Davies, B.; Chapman, J. R.\nScore: 20.1, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572534\n53BP1 regulates DNA end-joining in lymphocytes, diversifying immune antigen receptors. This involves nucleosome-bound 53BP1 at DNA double-stranded breaks (DSBs) recruiting RIF1 and shieldin, a poorly understood DNA-binding complex. The 53BP1-RIF1-shieldin axis is pathological in BRCA1-mutated cancers, blocking homologous recombination (HR) and driving illegitimate non-homologous end-joining (NHEJ). However, how this axis regulates DNA end-joining and HR suppression remains unresolved. We investigated shieldin and its interplay with CST, a complex recently implicated in 53BP1-dependent activities. Immunophenotypically, mice lacking shieldin or CST are equivalent, with class-switch recombination co-reliant on both complexes. ATM-dependent DNA damage signalling underpins this cooperation, inducing physical interactions between these complexes that reveal shieldin as a DSB-responsive CST adaptor. Furthermore, DNA polymerase {zeta} functions downstream of shieldin, establishing DNA fill-in synthesis as the physiological function of shieldin-CST. Lastly, 53BP1 suppresses HR and promotes NHEJ in BRCA1-deficient mice and cells independently of shieldin. These findings showcase the resilience of the 53BP1 pathway, achieved through the collaboration of chromatin-bound 53BP1 complexes and DNA end-processing effector proteins.\nUnraveling the mechanism of centromere inactivation in human aging\nAuthors: Sikder, S.; Baek, S.; McNeil, T.; Dalal, Y.\nScore: 14.9, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573721\nAging involves a range of genetic, epigenetic, and physiological alterations. A key characteristic of aged cells is the loss of global heterochromatin, accompanied by a reduction in canonical histone levels. In this study, we track the fate of centromeres during aging in human cells. Our findings reveal that the centromeric histone H3 variant CENP-A is downregulated in aged cells, in a p53-dependent manner. We observe repression of centromeric noncoding transcription through an epigenetic mechanism via recruitment of a lysine-specific demethylase 1 (LSD1/KDM1A) to centromeres. This suppression results in defective de novo CENP-A loading at aging centromeres. By dual inhibition of p53 and LSD1/KDM1A in aged cells, we mitigate the reduction in centromeric proteins and centromeric transcripts, leading to mitotic rejuvenation of these cells. These results offer insights into a novel mechanism for centromeric inactivation during aging and provide potential strategies to reactivate centromeres.\nMitochondrial morphology and function in mitochondrial disease\nAuthors: Faitg, J.; Davey, T.; Laws, R.; Lawless, C.; Tuppen, H.; Fitton, E.; Turnbull, D. E.; Vincent, A. E.\nScore: 15.3, Published: 2023-12-20 DOI: 10.1101/2023.12.20.571319\nMitochondria play a crucial role in maintaining cellular health. It is interesting that the shape of mitochondria can vary depending on the type of cell, mitochondrial function, and other cellular conditions. The morphology of this network is maintained by the balance between mitochondrial fusion and fission, necessary for the proper functioning of the mitochondrial energetics as well as the maintenance of mitochondria DNA (mtDNA) (Chan, 2006; Friedman and Nunnari, 2014; Glancy et al., 2015). If the balance between the two processes is slightly perturbed, dramatic changes in mitochondrial morphology can occur. However, there are limited studies that link functional assessment with mitochondrial morphology evaluation at high magnification, even fewer that do so in situ and none in human muscle biopsies. Therefore, we have developed a method which combines functional assessment of mitochondria through Cytochrome c Oxidase (COX) histochemistry, with a 3D electron microscopy (EM) technique, serial block-face scanning electron microscopy (SBFSEM). Using this technique, we examine the relationship between 3D mitochondrial morphology and mitochondrial activity in human skeletal muscle. Here we apply COX-SBFSEM to muscle samples from patients with single, large-scale mtDNA deletions, a cause of mitochondrial disease. These deletions cause oxidative phosphorylation deficiency, which can be observed through changes in COX activity, and a typical mosaic pattern of enzyme deficiency with both COX normal and deficient fibres present in the same biopsy. Using COX-SBFSEM, we can distinguish three distinct mitochondrial populations within muscle fibres: COX normal (+), COX intermediate ({+/-}) and COX deficient (-) mitochondria. COX+ mitochondria were larger and more complex than the COX- mitochondria, However, in fibres that were generally COX-, there was no apparent difference between the three types of mitochondria and all mitochondria in fibres that are COX- overall are fragmented and spherical. One of the main advantages of combining 3D-EM with the COX reaction is the ability to look at how per-mitochondrion oxidative phosphorylation status is spatially distributed within muscle fibres. Here we show a robust spatial pattern in fibres that are generally COX+ and that the spatial pattern is less clear in fibres that are predominantly COX{+/-} and COX- .\nFANCJ promotes PARP1 activity during DNA replication that is essential in BRCA1 deficient cells\nAuthors: Cong, K.; MacGilvary, N.; Lee, S.; MacLeod, S.; Calvo, J.; Peng, M.; Nedergaard Kousholt, A.; Day, T.; Cantor, S.\nScore: 11.3, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574095\nSingle-stranded DNA gaps are postulated to be fundamental to the mechanism of anti-cancer drugs. Gaining insights into their induction could therefore be pivotal for advancing therapeutic strategies. For poly (ADP-ribose) polymerase inhibitors (PARPi) to be effective, the presence of FANCJ helicase is required. However, the relationship between FANCJ dependent gaps and PARP1 catalytic inhibition or trapping--both linked to PARPi toxicity in BRCA deficient cells--is yet to be elucidated. Here, we find that the efficacy of PARPi is contingent on S-phase PARP1 activity, which is compromised in FANCJ deficient cells because PARP1, along with MSH2, is \"sequestered\" by G-quadruplexes. PARP1s replication activity is also diminished in cells missing a FANCJ-MLH1 interaction, but in such cells, depleting MSH2 can release sequestered PARP1, restoring PARPi-induced gaps and sensitivity. Our observations indicate that sequestered and trapped PARP1 are different chromatin-bound forms, with FANCJ loss increasing PARPi resistance in cells susceptible to canonical PARP1 trapping. However, in BRCA1 null cells, the loss of FANCJ mirrors the effects of PARP1 loss or inhibition, with the common detrimental factor being the loss of PARP1 activity during DNA replication, not trapping. These insights underline the crucial role of PARP1 activity during DNA replication in BRCA deficient cells and emphasize the importance of understanding drug mechanisms for enhancing precision medicine.\nROSALIND protects the mitochondrial translational machinery from oxidative damage\nAuthors: Katopodi, V.; Verheyden, Y.; Cinque, S.; Jimenez, E. L.; Demesmaeker, E.; Marino, A.; Derua, R.; Groaz, E.; Leucci, E.\nScore: 8.2, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574152\nUpregulation of mitochondrial respiration coupled with high ROS-scavenging capacity is a characteristic shared by drug-tolerant cells in several cancers. As translational fidelity is essential for cell fitness, protection of the mitochondrial and cytosolic ribosomes from oxidative damage is pivotal. While mechanisms for recognizing and repairing such damage exist in the cytoplasm, the corresponding process in the mitochondria remains unclear. By performing Ascorbate PEroXidase (APEX)-proximity ligation assay directed to the mitochondrial matrix followed by isolation and sequencing of RNA associated to mitochondrial proteins, we identified the nuclear-encoded lncRNA ROSALIND as an interacting partner of ribosomes. ROSALIND is upregulated in recurrent tumours and its expression can discriminate between responders and non-responders to immune checkpoint blockade in a melanoma cohort of patients. Featuring an unusually high G content, ROSALIND serves as a substrate for oxidation. Consequently, inhibiting ROSALIND leads to an increase in ROS and protein oxidation, resulting in severe mitochondrial respiration defects. This, in turn, impairs melanoma cell viability and increases immunogenicity both in vitro and ex vivo in preclinical humanized cancer models. These findings underscore the role of ROSALIND as a novel ROS buffering system, safeguarding mitochondrial translation from oxidative stress, and shed light on potential therapeutic strategies for overcoming cancer therapy resistance.\nInterplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing\nAuthors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.\nScore: 17.4, Published: 2023-12-31 DOI: 10.1101/2023.12.28.573526\nThe Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood. In this study, we have identified a second HuSH complex, termed HuSH2, centered around TASOR2, a paralog of the core TASOR protein in HuSH. Our findings indicate that every subunit in both HuSH and HuSH2 has an important role in achieving precise genomic localization to distinct, non-overlapping genomic loci. We utilized in silico protein structure prediction to simulate the interactions between MPP8 and both TASOR paralogs. Drawing on the insights gained from these predictions, we implemented amino acid substitutions that interfered with the binding of MPP8 to each HuSH complex. Leveraging these MPP8 transgenes and other constructs, we identified an important role played by the relative quantities of HuSH complexes in controlling the activity of LINE-1 elements. Furthermore, our results suggest that dynamic changes in TASOR and TASOR2 expression enable cells to finely tune the extent of HuSH-mediated silencing. Our study provides insights into the intricate interplay between HuSH complexes, illuminating their important role in the regulation of retrotransposon silencing. Key PointsO_LIThe identification of a previously unknown HuSH2 complex, with TASOR2 as its central component. C_LIO_LIHuSH and HuSH2 complexes exhibit unique genomic localization patterns within the human genome. C_LIO_LIDisruption of the delicate balance between the two HuSH complexes results in the desilencing of LINE-1 and other transposable elements. C_LIO_LITASOR and TASOR2 engage in a competitive interaction for the HuSH subunit MPP8. C_LIO_LIThe localization of MPP8 to either HuSH or HuSH2 sites is intricately regulated by its interaction with TASOR and TASOR2. C_LI\nIdentification of the main barriers to Ku accumulation in chromatin\nAuthors: Bossaert, M.; Moreno, A.; Peixoto, A.; Pillaire, M.-J.; Chanut, P.; Frit, P.; Calsou, P.; Loparo, J. J.; Britton, S.\nScore: 6.0, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574002\nRepair of DNA double strand breaks by the non-homologous end-joining pathway is initiated by the binding of Ku to DNA ends. Given its high affinity for ends, multiple Ku proteins load onto linear DNAs in vitro. However, in cells, Ku loading is limited to [~]1-2 molecules per DNA end. The mechanisms enforcing this limit are currently unknown. Here we show that the catalytic subunit of the DNA-dependent protein kinase (DNA-PKcs), but not its protein kinase activity, is required to prevent excessive Ku entry into chromatin. Ku accumulation is further restricted by two mechanisms: a neddylation/FBXL12-dependent process which actively removes loaded Ku molecules throughout the cell cycle and a CtIP/ATM-dependent mechanism which operates in S-phase. Finally, we demonstrate that the misregulation of Ku loading leads to impaired transcription in the vicinity of DNA ends. Together our data shed light on the multiple layers of coordinated mechanisms operating to prevent Ku from invading chromatin and interfering with other DNA transactions. HighlightsO_LIDNA-PKcs structurally blocks Ku sliding into chromatin in human \u0026 Xenopus C_LIO_LIA neddylation/FBXL12-dependent mechanism limits Ku accumulation on chromatin C_LIO_LIIn S-phase, ATM/CtIP overcomes Ku accumulation C_LIO_LIIn absence of DNA-PKcs, transcription at the DNA end vicinity is inhibited C_LI eTOC blurbThe DNA end binding protein Ku can slide onto naked DNA but this is limited in cells. Using human cells and Xenopus egg extracts, DNA-PKcs is identified as the main structural barrier to Ku entry into chromatin, along with two active mechanisms which limit Ku accumulation in absence of DNA-PKcs. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=194 SRC=\"FIGDIR/small/574002v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (67K): org.highwire.dtl.DTLVardef@a62325org.highwire.dtl.DTLVardef@14d7973org.highwire.dtl.DTLVardef@1608548org.highwire.dtl.DTLVardef@1ce3dc0_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPlasticity of the MFS1 promotor is not the only driver of Multidrug resistance in Zymoseptoria tritici\nAuthors: Patry-Leclaire, S.; Pitarch, A.; Pitarch, A.; Walker, A.-S.; Filinger, S.\nScore: 5.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573052\nBackgroundMultidrug resistance has been identified in the fungal pathogen responsible for Septoria leaf blotch, Zymoseptoria tritici, since 2011. It has been linked to the overexpression of the gene encoding the MFS1 transporter due to inserts in the promoter region of MFS1 (PMFS1), namely types I-III. Recently, two new inserts were discovered in PMFS1 that were not linked to MDR, interrogating about whether PMFS1 inserts are the only drivers of MDR in Z. tritici. The goal of our study was to gain a more complete view of MDR in Z. tritici by examining the genotypic diversity associated with the MDR phenotype in a large sample of the modern population. ResultsWe isolated 384 potential MDR strains between 2020 and 2021 in northern Europe for PMFS1 genotype and MDR assessment. We discovered six new inserts in PMFS1, bringing the total count to 13 including one insertion-deletion in the 5 UTR region. Of these, 11 display similarities with transposable elements, and 3 are not linked to MDR. Some field strains were significantly more resistant than their respective reference of the same PMFS1 genotype and some strains without insert displayed MDR phenotype. ConclusionWe described the landscape of the MDR in modern Z. tritici population and postulate that PMFS1 is a hot-spot for insertions involving transposition events. Our study shows that MDR cannot be solely explained by inserts found in PMFS1, and that additional mechanisms might be at work.\nIn vivo single-molecule imaging of RecB reveals efficient repair of DNA damage in Escherichia coli\nAuthors: Lepore, A.; Thedie, D.; McLaren, L.; Azeroglu, B.; Pambos, O. J.; Kapanidis, A. N.; El Karoui, M.\nScore: 5.0, Published: 2023-12-22 DOI: 10.1101/2023.12.22.573010\nEfficient DNA repair is crucial for maintaining genome integrity and ensuring cell survival. In Escherichia coli, RecBCD plays a crucial role in processing DNA ends following a DNA double-strand break (DSB) to initiate repair by homologous recombination. While RecBCD has been extensively studied in vitro, less is known about how it contributes to rapid and efficient repair in living bacteria. Here, we perform single-molecule microscopy to investigate DNA repair in real-time in E. coli. We quantify RecB single-molecule mobility and monitor the induction of the DNA damage response (SOS response) in individual cells. We show that RecB binding to broken DNA ends leads to efficient repair without SOS induction. In contrast, in a RecB mutant with modified activities leading to the activation of an alternative repair pathway, repair is less efficient and leads to high SOS induction. Our findings reveal how subtle alterations in RecB activity profoundly impact the efficiency of DNA repair in E. coli.\n",
  "wordCount" : "2546",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574113" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574113">
        <p class="paperTitle">A Genome-Wide CRISPR Screen Identifies Sortilin as the Receptor Responsible for Galectin-1 Lysosomal Trafficking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Donnelly, J.; Kamber, R. A.; Wisnovsky, S.; Roberts, D. S.; Peltan, E. L.; Bassik, M. C.; Bertozzi, C. R.</p>
        <p class="info">Score: 26.3, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574113' target='https://doi.org/10.1101/2024.01.03.574113'> 10.1101/2024.01.03.574113</a></p>
        <p class="abstract">Galectins are a family of mammalian glycan-binding proteins that have been implicated as regulators of myriad cellular processes including cell migration, apoptosis, and immune modulation. Several members of this family, such as galectin-1, exhibit both cell-surface and intracellular functions. Interestingly, galectin-1 can be found in the endomembrane system, nucleus, or cytosol, as well as on the cell surface. The mechanisms by which galectin-1 traffics between cellular compartments, including its unconventional secretion and internalization processes, are poorly understood. Here, we determined the pathways by which exogenous galectin-1 enters cells and explored its capacity as a delivery vehicle for protein and siRNA therapeutics. We used a galectin-1-toxin conjugate, modelled on antibody-drug conjugates, as a selection tool in a genome-wide CRISPR screen. We discovered that galectin-1 interacts with the endosome-lysosome trafficking receptor sortilin in a glycan-dependent manner, which regulates galectin-1 trafficking to the lysosome. Further, we show that this pathway can be exploited for delivery of a functional siRNA. This study sheds light on the mechanisms by which galectin-1 is internalized by cells and suggests a new strategy for intracellular drug delivery via galectin-1 conjugation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572534" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572534">
        <p class="paperTitle">Shieldin and CST co-orchestrate DNA polymerase-dependent tailed-end joining reactions independently of 53BP1-governed repair pathway choice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: King, A.; Reichl, P.; Metson, J. S.; Parker, R.; Munro, D.; Oliveira, C.; Becker, J. R.; Biggs, D.; Preece, C.; Davies, B.; Chapman, J. R.</p>
        <p class="info">Score: 20.1, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572534' target='https://doi.org/10.1101/2023.12.20.572534'> 10.1101/2023.12.20.572534</a></p>
        <p class="abstract">53BP1 regulates DNA end-joining in lymphocytes, diversifying immune antigen receptors. This involves nucleosome-bound 53BP1 at DNA double-stranded breaks (DSBs) recruiting RIF1 and shieldin, a poorly understood DNA-binding complex. The 53BP1-RIF1-shieldin axis is pathological in BRCA1-mutated cancers, blocking homologous recombination (HR) and driving illegitimate non-homologous end-joining (NHEJ). However, how this axis regulates DNA end-joining and HR suppression remains unresolved.

We investigated shieldin and its interplay with CST, a complex recently implicated in 53BP1-dependent activities. Immunophenotypically, mice lacking shieldin or CST are equivalent, with class-switch recombination co-reliant on both complexes. ATM-dependent DNA damage signalling underpins this cooperation, inducing physical interactions between these complexes that reveal shieldin as a DSB-responsive CST adaptor. Furthermore, DNA polymerase {zeta} functions downstream of shieldin, establishing DNA fill-in synthesis as the physiological function of shieldin-CST. Lastly, 53BP1 suppresses HR and promotes NHEJ in BRCA1-deficient mice and cells independently of shieldin. These findings showcase the resilience of the 53BP1 pathway, achieved through the collaboration of chromatin-bound 53BP1 complexes and DNA end-processing effector proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573721">
        <p class="paperTitle">Unraveling the mechanism of centromere inactivation in human aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikder, S.; Baek, S.; McNeil, T.; Dalal, Y.</p>
        <p class="info">Score: 14.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573721' target='https://doi.org/10.1101/2023.12.30.573721'> 10.1101/2023.12.30.573721</a></p>
        <p class="abstract">Aging involves a range of genetic, epigenetic, and physiological alterations. A key characteristic of aged cells is the loss of global heterochromatin, accompanied by a reduction in canonical histone levels. In this study, we track the fate of centromeres during aging in human cells. Our findings reveal that the centromeric histone H3 variant CENP-A is downregulated in aged cells, in a p53-dependent manner. We observe repression of centromeric noncoding transcription through an epigenetic mechanism via recruitment of a lysine-specific demethylase 1 (LSD1/KDM1A) to centromeres. This suppression results in defective de novo CENP-A loading at aging centromeres. By dual inhibition of p53 and LSD1/KDM1A in aged cells, we mitigate the reduction in centromeric proteins and centromeric transcripts, leading to mitotic rejuvenation of these cells. These results offer insights into a novel mechanism for centromeric inactivation during aging and provide potential strategies to reactivate centromeres.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.571319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.571319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.571319">
        <p class="paperTitle">Mitochondrial morphology and function in mitochondrial disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.571319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.571319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faitg, J.; Davey, T.; Laws, R.; Lawless, C.; Tuppen, H.; Fitton, E.; Turnbull, D. E.; Vincent, A. E.</p>
        <p class="info">Score: 15.3, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.571319' target='https://doi.org/10.1101/2023.12.20.571319'> 10.1101/2023.12.20.571319</a></p>
        <p class="abstract">Mitochondria play a crucial role in maintaining cellular health. It is interesting that the shape of mitochondria can vary depending on the type of cell, mitochondrial function, and other cellular conditions. The morphology of this network is maintained by the balance between mitochondrial fusion and fission, necessary for the proper functioning of the mitochondrial energetics as well as the maintenance of mitochondria DNA (mtDNA) (Chan, 2006; Friedman and Nunnari, 2014; Glancy et al., 2015). If the balance between the two processes is slightly perturbed, dramatic changes in mitochondrial morphology can occur.

However, there are limited studies that link functional assessment with mitochondrial morphology evaluation at high magnification, even fewer that do so in situ and none in human muscle biopsies. Therefore, we have developed a method which combines functional assessment of mitochondria through Cytochrome c Oxidase (COX) histochemistry, with a 3D electron microscopy (EM) technique, serial block-face scanning electron microscopy (SBFSEM). Using this technique, we examine the relationship between 3D mitochondrial morphology and mitochondrial activity in human skeletal muscle.

Here we apply COX-SBFSEM to muscle samples from patients with single, large-scale mtDNA deletions, a cause of mitochondrial disease. These deletions cause oxidative phosphorylation deficiency, which can be observed through changes in COX activity, and a typical mosaic pattern of enzyme deficiency with both COX normal and deficient fibres present in the same biopsy.

Using COX-SBFSEM, we can distinguish three distinct mitochondrial populations within muscle fibres: COX normal (&#43;), COX intermediate ({&#43;/-}) and COX deficient (-) mitochondria. COX&#43; mitochondria were larger and more complex than the COX- mitochondria, However, in fibres that were generally COX-, there was no apparent difference between the three types of mitochondria and all mitochondria in fibres that are COX- overall are fragmented and spherical. One of the main advantages of combining 3D-EM with the COX reaction is the ability to look at how per-mitochondrion oxidative phosphorylation status is spatially distributed within muscle fibres. Here we show a robust spatial pattern in fibres that are generally COX&#43; and that the spatial pattern is less clear in fibres that are predominantly COX{&#43;/-} and COX- .</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574095" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574095">
        <p class="paperTitle">FANCJ promotes PARP1 activity during DNA replication that is essential in BRCA1 deficient cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cong, K.; MacGilvary, N.; Lee, S.; MacLeod, S.; Calvo, J.; Peng, M.; Nedergaard Kousholt, A.; Day, T.; Cantor, S.</p>
        <p class="info">Score: 11.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574095' target='https://doi.org/10.1101/2024.01.04.574095'> 10.1101/2024.01.04.574095</a></p>
        <p class="abstract">Single-stranded DNA gaps are postulated to be fundamental to the mechanism of anti-cancer drugs. Gaining insights into their induction could therefore be pivotal for advancing therapeutic strategies. For poly (ADP-ribose) polymerase inhibitors (PARPi) to be effective, the presence of FANCJ helicase is required. However, the relationship between FANCJ dependent gaps and PARP1 catalytic inhibition or trapping--both linked to PARPi toxicity in BRCA deficient cells--is yet to be elucidated. Here, we find that the efficacy of PARPi is contingent on S-phase PARP1 activity, which is compromised in FANCJ deficient cells because PARP1, along with MSH2, is &#34;sequestered&#34; by G-quadruplexes. PARP1s replication activity is also diminished in cells missing a FANCJ-MLH1 interaction, but in such cells, depleting MSH2 can release sequestered PARP1, restoring PARPi-induced gaps and sensitivity. Our observations indicate that sequestered and trapped PARP1 are different chromatin-bound forms, with FANCJ loss increasing PARPi resistance in cells susceptible to canonical PARP1 trapping. However, in BRCA1 null cells, the loss of FANCJ mirrors the effects of PARP1 loss or inhibition, with the common detrimental factor being the loss of PARP1 activity during DNA replication, not trapping. These insights underline the crucial role of PARP1 activity during DNA replication in BRCA deficient cells and emphasize the importance of understanding drug mechanisms for enhancing precision medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574152">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574152" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574152">
        <p class="paperTitle">ROSALIND protects the mitochondrial translational machinery from oxidative damage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574152" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574152" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Katopodi, V.; Verheyden, Y.; Cinque, S.; Jimenez, E. L.; Demesmaeker, E.; Marino, A.; Derua, R.; Groaz, E.; Leucci, E.</p>
        <p class="info">Score: 8.2, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574152' target='https://doi.org/10.1101/2024.01.04.574152'> 10.1101/2024.01.04.574152</a></p>
        <p class="abstract">Upregulation of mitochondrial respiration coupled with high ROS-scavenging capacity is a characteristic shared by drug-tolerant cells in several cancers. As translational fidelity is essential for cell fitness, protection of the mitochondrial and cytosolic ribosomes from oxidative damage is pivotal. While mechanisms for recognizing and repairing such damage exist in the cytoplasm, the corresponding process in the mitochondria remains unclear.

By performing Ascorbate PEroXidase (APEX)-proximity ligation assay directed to the mitochondrial matrix followed by isolation and sequencing of RNA associated to mitochondrial proteins, we identified the nuclear-encoded lncRNA ROSALIND as an interacting partner of ribosomes. ROSALIND is upregulated in recurrent tumours and its expression can discriminate between responders and non-responders to immune checkpoint blockade in a melanoma cohort of patients. Featuring an unusually high G content, ROSALIND serves as a substrate for oxidation. Consequently, inhibiting ROSALIND leads to an increase in ROS and protein oxidation, resulting in severe mitochondrial respiration defects. This, in turn, impairs melanoma cell viability and increases immunogenicity both in vitro and ex vivo in preclinical humanized cancer models. These findings underscore the role of ROSALIND as a novel ROS buffering system, safeguarding mitochondrial translation from oxidative stress, and shed light on potential therapeutic strategies for overcoming cancer therapy resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573526">
        <p class="paperTitle">Interplay between Two Paralogous Human Silencing Hub (HuSH) Complexes in Regulating LINE-1 Element Silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jensvold, Z. D.; Christenson, A. E.; Flood, J. R.; Lewis, P. W.</p>
        <p class="info">Score: 17.4, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573526' target='https://doi.org/10.1101/2023.12.28.573526'> 10.1101/2023.12.28.573526</a></p>
        <p class="abstract">The Human Silencing Hub (HuSH) complex is composed of TASOR, MPP8, and PPHLN1 subunits and serves as a conserved protein complex responsible for silencing transposable elements in vertebrate animals. Despite its importance, the regulatory mechanisms and recruitment dynamics governing this complex remain poorly understood. In this study, we have identified a second HuSH complex, termed HuSH2, centered around TASOR2, a paralog of the core TASOR protein in HuSH. Our findings indicate that every subunit in both HuSH and HuSH2 has an important role in achieving precise genomic localization to distinct, non-overlapping genomic loci. We utilized in silico protein structure prediction to simulate the interactions between MPP8 and both TASOR paralogs. Drawing on the insights gained from these predictions, we implemented amino acid substitutions that interfered with the binding of MPP8 to each HuSH complex. Leveraging these MPP8 transgenes and other constructs, we identified an important role played by the relative quantities of HuSH complexes in controlling the activity of LINE-1 elements. Furthermore, our results suggest that dynamic changes in TASOR and TASOR2 expression enable cells to finely tune the extent of HuSH-mediated silencing. Our study provides insights into the intricate interplay between HuSH complexes, illuminating their important role in the regulation of retrotransposon silencing.

Key PointsO_LIThe identification of a previously unknown HuSH2 complex, with TASOR2 as its central component.
C_LIO_LIHuSH and HuSH2 complexes exhibit unique genomic localization patterns within the human genome.
C_LIO_LIDisruption of the delicate balance between the two HuSH complexes results in the desilencing of LINE-1 and other transposable elements.
C_LIO_LITASOR and TASOR2 engage in a competitive interaction for the HuSH subunit MPP8.
C_LIO_LIThe localization of MPP8 to either HuSH or HuSH2 sites is intricately regulated by its interaction with TASOR and TASOR2.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574002" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574002">
        <p class="paperTitle">Identification of the main barriers to Ku accumulation in chromatin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bossaert, M.; Moreno, A.; Peixoto, A.; Pillaire, M.-J.; Chanut, P.; Frit, P.; Calsou, P.; Loparo, J. J.; Britton, S.</p>
        <p class="info">Score: 6.0, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574002' target='https://doi.org/10.1101/2024.01.03.574002'> 10.1101/2024.01.03.574002</a></p>
        <p class="abstract">Repair of DNA double strand breaks by the non-homologous end-joining pathway is initiated by the binding of Ku to DNA ends. Given its high affinity for ends, multiple Ku proteins load onto linear DNAs in vitro. However, in cells, Ku loading is limited to [~]1-2 molecules per DNA end. The mechanisms enforcing this limit are currently unknown. Here we show that the catalytic subunit of the DNA-dependent protein kinase (DNA-PKcs), but not its protein kinase activity, is required to prevent excessive Ku entry into chromatin. Ku accumulation is further restricted by two mechanisms: a neddylation/FBXL12-dependent process which actively removes loaded Ku molecules throughout the cell cycle and a CtIP/ATM-dependent mechanism which operates in S-phase. Finally, we demonstrate that the misregulation of Ku loading leads to impaired transcription in the vicinity of DNA ends. Together our data shed light on the multiple layers of coordinated mechanisms operating to prevent Ku from invading chromatin and interfering with other DNA transactions.

HighlightsO_LIDNA-PKcs structurally blocks Ku sliding into chromatin in human &amp; Xenopus
C_LIO_LIA neddylation/FBXL12-dependent mechanism limits Ku accumulation on chromatin
C_LIO_LIIn S-phase, ATM/CtIP overcomes Ku accumulation
C_LIO_LIIn absence of DNA-PKcs, transcription at the DNA end vicinity is inhibited
C_LI

eTOC blurbThe DNA end binding protein Ku can slide onto naked DNA but this is limited in cells. Using human cells and Xenopus egg extracts, DNA-PKcs is identified as the main structural barrier to Ku entry into chromatin, along with two active mechanisms which limit Ku accumulation in absence of DNA-PKcs.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=194 SRC=&#34;FIGDIR/small/574002v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (67K):
org.highwire.dtl.DTLVardef@a62325org.highwire.dtl.DTLVardef@14d7973org.highwire.dtl.DTLVardef@1608548org.highwire.dtl.DTLVardef@1ce3dc0_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573052">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573052" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573052">
        <p class="paperTitle">Plasticity of the MFS1 promotor is not the only driver of Multidrug resistance in Zymoseptoria tritici</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573052" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573052" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patry-Leclaire, S.; Pitarch, A.; Pitarch, A.; Walker, A.-S.; Filinger, S.</p>
        <p class="info">Score: 5.8, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573052' target='https://doi.org/10.1101/2023.12.27.573052'> 10.1101/2023.12.27.573052</a></p>
        <p class="abstract">BackgroundMultidrug resistance has been identified in the fungal pathogen responsible for Septoria leaf blotch, Zymoseptoria tritici, since 2011. It has been linked to the overexpression of the gene encoding the MFS1 transporter due to inserts in the promoter region of MFS1 (PMFS1), namely types I-III. Recently, two new inserts were discovered in PMFS1 that were not linked to MDR, interrogating about whether PMFS1 inserts are the only drivers of MDR in Z. tritici.

The goal of our study was to gain a more complete view of MDR in Z. tritici by examining the genotypic diversity associated with the MDR phenotype in a large sample of the modern population.

ResultsWe isolated 384 potential MDR strains between 2020 and 2021 in northern Europe for PMFS1 genotype and MDR assessment. We discovered six new inserts in PMFS1, bringing the total count to 13 including one insertion-deletion in the 5 UTR region. Of these, 11 display similarities with transposable elements, and 3 are not linked to MDR. Some field strains were significantly more resistant than their respective reference of the same PMFS1 genotype and some strains without insert displayed MDR phenotype.

ConclusionWe described the landscape of the MDR in modern Z. tritici population and postulate that PMFS1 is a hot-spot for insertions involving transposition events. Our study shows that MDR cannot be solely explained by inserts found in PMFS1, and that additional mechanisms might be at work.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573010" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573010">
        <p class="paperTitle">In vivo single-molecule imaging of RecB reveals efficient repair of DNA damage in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lepore, A.; Thedie, D.; McLaren, L.; Azeroglu, B.; Pambos, O. J.; Kapanidis, A. N.; El Karoui, M.</p>
        <p class="info">Score: 5.0, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573010' target='https://doi.org/10.1101/2023.12.22.573010'> 10.1101/2023.12.22.573010</a></p>
        <p class="abstract">Efficient DNA repair is crucial for maintaining genome integrity and ensuring cell survival. In Escherichia coli, RecBCD plays a crucial role in processing DNA ends following a DNA double-strand break (DSB) to initiate repair by homologous recombination. While RecBCD has been extensively studied in vitro, less is known about how it contributes to rapid and efficient repair in living bacteria. Here, we perform single-molecule microscopy to investigate DNA repair in real-time in E. coli. We quantify RecB single-molecule mobility and monitor the induction of the DNA damage response (SOS response) in individual cells. We show that RecB binding to broken DNA ends leads to efficient repair without SOS induction. In contrast, in a RecB mutant with modified activities leading to the activation of an alternative repair pathway, repair is less efficient and leads to high SOS induction. Our findings reveal how subtle alterations in RecB activity profoundly impact the efficiency of DNA repair in E. coli.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
