<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>molecular biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>molecular biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes
Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.
Score: 51.5, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727
RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/molecular-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="molecular biology" />
<meta property="og:description" content="Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes
Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.
Score: 51.5, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727
RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/molecular-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="molecular biology"/>
<meta name="twitter:description" content="Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes
Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.
Score: 51.5, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727
RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "molecular biology",
      "item": "https://trxiv.yorks0n.com/posts/molecular-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "molecular biology",
  "name": "molecular biology",
  "description": "Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes\nAuthors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.\nScore: 51.5, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727\nRNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing.",
  "keywords": [
    
  ],
  "articleBody": " Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes\nAuthors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.\nScore: 51.5, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552727\nRNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing. Some of these enzymes are components of insertion sequences (IS) in the IS200/IS605 and IS607 transposon families. Both IS families encode a TnpA transposase and TnpB nuclease, an RNA-guided enzyme ancestral to CRISPR-Cas12. In eukaryotes and their viruses, TnpB homologs occur as two distinct types, Fanzor1 and Fanzor2. We analyzed the evolutionary relationships between prokaryotic TnpBs and eukaryotic Fanzors, revealing that a clade of IS607 TnpBs with unusual active site arrangement found primarily in Cyanobacteriota likely gave rise to both types of Fanzors. The wide-spread nature of Fanzors imply that the properties of this particular group of IS607 TnpBs were particularly suited to adaptation and evolution in eukaryotes and their viruses. Experimental characterization of a prokaryotic IS607 TnpB and virally encoded Fanzor1s uncovered features that may have fostered coevolution between TnpBs/Fanzors and their cognate transposases. Our results provide insight into the evolutionary origins of a ubiquitous family of RNA-guided proteins that shows remarkable conservation across domains of life.\nOrchestration of pluripotent stem cell genome reactivation during mitotic exit\nAuthors: Placzek, S.; Vanzan, L.; Suter, D. M.\nScore: 19.0, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552605\nThe maintenance of cell identity faces many challenges during mitosis, as most DNA-binding proteins are evicted from DNA and transcription is virtually abolished. How cells maintain their identity through cell division and faithfully re-initiate gene expression during mitotic exit is unclear. Here, we developed a novel reporter system enabling cell cycle synchronization-free separation of pluripotent stem cells in temporal bins of \u003c 30 minutes during mitotic exit. This allowed us to quantify genome-wide reactivation of transcription, sequential changes in chromatin accessibility, and re-binding of the pluripotency transcription factors OCT4, SOX2, and NANOG (OSN). We found that transcriptional activity progressively ramped up after mitosis and that OSN rapidly reoccupied the genome during the anaphase-telophase transition. We also demonstrate transcription factor-specific, dynamic relocation patterns and a hierarchical reorganization of the OSN binding landscape governed by OCT4 and SOX2. Our study sheds light on the dynamic orchestration of transcriptional reactivation after mitosis.\nInsights into human outer kinetochore assembly and force transmission from a structure-function analysis of the KMN network\nAuthors: Polley, S.; Raisch, T.; Koerner, M.; Terbeck, M.; Graeter, F.; Raunser, S.; Aponte-Santamaria, C.; Vetter, I. R.; Musacchio, A.\nScore: 16.0, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552315\nThe biorientation of chromosomes during cell division is necessary for precise dispatching of a mother cells chromosomes into its two daughters. Kinetochores, large layered structures built on specialized chromosome loci named centromeres, promote biorientation by binding and sensing spindle microtubules. The kinetochore outer layer consists of a 10-subunit apparatus comprising Knl1C, Mis12C, and Ndc80C subcomplexes (KMN network). The KMN network is highly elongated and docks on kinetochores and microtubules using interfaces at its opposite extremes. Here, we combine cryo-EM reconstructions and AlphaFold2 predictions to generate a model of the KMN network that reveals all intra-KMN interfaces. We identify and functionally validate two interaction interfaces that link Mis12C to Ndc80C and Knl1C. Through targeted interference experiments and molecular dynamics simulations we demonstrate this mutual organization stabilizes the KMN network. Our work reports the first comprehensive structural and functional analysis of the microtubule binding machinery of kinetochores and elucidates a path of microtubule-generated force transmission\nThe SARM1 TIR domain produces glycocyclic ADPR molecules as minor products\nAuthors: Garb, J.; Amitai, G.; Lu, A.; Ofir, G.; Brandis, A.; Mehlman, T.; Kranzusch, P.; Sorek, R.\nScore: 15.3, Published: 2023-08-10 DOI: 10.1101/2023.08.10.552750\nSterile alpha and TIR motif-containing 1 (SARM1) is a protein involved in programmed death of injured axons. Following axon injury or a drug-induced insult, the TIR domain of SARM1 degrades the essential molecule nicotinamide adenine dinucleotide (NAD+), leading to a form of axonal death called Wallerian degeneration. Degradation of NAD+ by SARM1 is essential for the Wallerian degeneration process, but accumulating evidence suggest that other activities of SARM1, beyond the mere degradation of NAD+, may be necessary for programmed axonal death. In this study we show that the TIR domains of both human and fruit fly SARM1 produce 1''-2' and 1''-3' glycocyclic ADP-ribose (gcADPR) molecules as minor products. As previously reported, we observed that SARM1 TIR domains mostly convert NAD+ to ADPR (for human SARM1) or cADPR (in the case of SARM1 from Drosophila melanogaster). However, we now show that human and Drosophila SARM1 additionally convert ~0.1-0.5% of NAD+ into gcADPR molecules. We find that SARM1 TIR domains produce gcADPR molecules both when purified in vitro and when expressed in bacterial cells. Given that gcADPR is a second messenger involved in programmed cell death in bacteria and likely in plants, we propose that gcADPR may play a role in SARM1-induced programmed axonal death in animals.\nGerm granule association drives small RNA specificity for a nuclear Argonaute protein\nAuthors: Chen, S.; Phillips, C. M.\nScore: 12.3, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551545\nRNA interference (RNAi) is a conserved gene silencing process that exists in diverse organisms to protect genome integrity and regulate gene expression. In C. elegans, the majority of RNAi pathway proteins localize to perinuclear, phase-separated germ granules, which are comprised of sub-domains referred to as P granules, Mutator foci, Z granules, and SIMR foci. However, the protein components and function of the newly discovered SIMR foci are unknown. Here we demonstrate that HRDE-2 localizes to SIMR foci and interacts with the germline nuclear RNAi Argonaute HRDE-1. Furthermore, HRDE-1 also localizes to SIMR foci, dependent on HRDE-2, but only in its small RNA unbound state. This germ granule localization is critical to promote the small RNA binding specificity of HRDE-1 and, in the absence of HRDE-2, HRDE-1 exclusively loads CSR-class 22G-RNAs rather than WAGO-class 22G-RNAs, resulting in inappropriate H3K9me3 deposition on CSR-target genes. Thus, our study demonstrates that the recruitment of unloaded HRDE-1 to germ granules, mediated by HRDE-2, is critical to ensure that the correct small RNAs are used to guide nuclear RNA silencing in the C. elegans germline.\nN-glycosylation as a eukaryotic protective mechanism against protein aggregation\nAuthors: Duran-Romana, R.; Houben, B.; De Vleeschouwer, M.; Louros, N.; Wilson, M. P.; Matthijs, G.; Schymkowitz, J.; Rousseau, F.\nScore: 7.2, Published: 2023-08-11 DOI: 10.1101/2023.08.11.552904\nThe tendency for proteins to form aggregates is an inherent part of every proteome and arises from the self-assembly of short protein segments called aggregation-prone regions (APRs). While post-translational modifications (PTMs) have been implicated in modulating protein aggregation, their direct role in APRs remains poorly understood. In this study, we used a combination of proteome-wide computational analyses and biochemical techniques to investigate the potential involvement of PTMs in aggregation regulation. Our findings reveal that while most PTM types are disfavored near APRs, N-glycosylation is enriched and evolutionarily selected, especially in proteins prone to misfolding. Experimentally, we show that N-glycosylation inhibits the aggregation of peptides in vitro through steric hindrance. Moreover, mining existing proteomics data, we find that the loss of N-glycans at the flanks of APRs leads to specific protein aggregation in Neuro2a cells. Our results point towards a novel intrinsic role for N-glycosylation, directly preventing protein aggregation in eukaryotes.\nIntra-host variation in the spike S1/S2 region of a feline coronavirus type-1 in a cat with persistent infection\nAuthors: Olarte-Castillo, X. A.; Licitra, B. N.; Andre, N. M.; Sierra, M. A.; Mason, C. E.; Goodman, L. B.; Whittaker, G.\nScore: 45.6, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551356\nFeline coronavirus type 1 (FCoV-1) is widely known for causing feline infectious peritonitis (FIP), a systemic infection that is often fatal, with the virus known as the FIPV biotype. However, subclinical disease also occurs, in which cats may not show signs and intermittently shed the virus, including in feces, possibly for long periods of time. This virus is known as the FECV biotype. Progression of FECV to FIPV has been linked to several genomic changes, however a specific region of the viral spike protein at the interface of the spike S1 and S2 domains has been especially implicated. In this study, we followed a cat (#576) for six years from 2017, at which time FCoV-1 was detected in feces and conjunctival swabs, until 2022, when the animal was euthanized based on a diagnosis of alimentary small cell lymphoma. Over this time period, the cat was clinically diagnosed with inflammatory bowel disease and chronic rhinitis, and cardiac problems were also suspected. Using hybridization capture targeting the spike (S) gene of FCoV followed by next-generation sequencing, we screened 27 clinical samples. We detected FCoV-1 in 4 samples taken in 2017 (intestine and nasal tissue, feces, and conjunctiva), and 3 samples taken in 2022 (feces, and intestinal and heart tissue), but not in fecal samples taken in 2019 and 2020. Next, we focused on the S1/S2 region within S, which contains the furin cleavage site (FCS), a key regulator of viral transmission and pathogenesis. We show that the FCoV-1 variants obtained from feces in 2017 and 2022 were identical, while the ones from conjunctiva (2017), heart (2022), and intestine (2017 and 2022) were distinct. Sequence comparison of all the variants obtained showed that most of the non-synonymous changes in the S1/S2 region occur within the FCS. In the heart, we found two variants that differed by a single nucleotide, resulting in distinct FCS motifs that differ in one amino acid. It is predicted that one of these FCS motifs will down-regulate spike cleavability. The variant from the conjunctiva (2017) had a 6-nucleotide in-frame insertion that resulted in a longer and more exposed S1/S2 loop, which is predicted to be more accessible to the furin protease. Our studies indicate that FCoV-1 can independently persist in the gastrointestinal tract and heart of a cat over a long period of time without evidence of typical FIP signs, with intermittent viral shedding from the gastrointestinal and respiratory tracts.\nJointly modeling deep mutational scans identifies shifted mutational effects among SARS-CoV-2 spike homologs\nAuthors: Haddox, H. K.; Galloway, J. G.; Dadonaite, B.; Bloom, J. D.; Matsen, F. A.; DeWitt, W. S.\nScore: 24.1, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551037\nDeep mutational scanning (DMS) is a high-throughput experimental technique that measures the effects of thousands of mutations to a protein. These experiments can be performed on multiple homologs of a protein or on the same protein selected under multiple conditions. It is often of biological interest to identify mutations with shifted effects across homologs or conditions. However, it is challenging to determine if observed shifts arise from biological signal or experimental noise. Here, we describe a method for jointly inferring mutational effects across multiple DMS experiments while also identifying mutations that have shifted in their effects among experiments. A key aspect of our method is to regularize the inferred shifts, so that they are nonzero only when strongly supported by the data. We apply this method to DMS experiments that measure how mutations to spike proteins from SARS-CoV-2 variants (Delta, Omicron BA.1, and Omicron BA.2) affect cell entry. Most mutational effects are conserved between these spike homologs, but a fraction have markedly shifted. We experimentally validate a subset of the mutations inferred to have shifted effects, and confirm differences of \u003e1,000-fold in the impact of the same mutation on spike-mediated viral infection across spikes from different SARS-CoV-2 variants. Overall, our work establishes a general approach for comparing sets of DMS experiments to identify biologically important shifts in mutational effects. Significance StatementAmino-acid mutations to a protein have effects that can shift as the protein evolves or is put under new selective pressure. The effects of amino-acid mutations to a specific protein under a defined selective pressure can be measured by deep mutational scanning experiments. Here, we devise an approach to quantify shifts in mutational effects between experiments performed on different homologs (i.e. variants) of the same protein, or on the same protein selected under different conditions. We use this approach to compare experiments performed on three homologs of SARS-CoV-2 spike, identifying mutations that have shifted in their effect on spike-mediated viral infection by \u003e1,000 fold across SARS-CoV-2 variants.\nRNA binding limits the ligand induced transcriptional potential of estrogen receptor-alpha\nAuthors: Soota, D.; Saravanan, B.; Mann, R.; Kharbanda, T.; Notani, D.\nScore: 5.7, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552751\nTranscription factors (TFs) primarily regulate gene expression by binding to DNA through their DNA binding domains (DBD). Additionally, approximately half of these TFs also interact with RNA. However, the role of RNA in enabling TF binding on chromatin and subsequent transcription is poorly understood. Estrogen receptor-alpha (ERalpha) is one such TF that activates genes in response to estrogen stimulation. Here, we report that ER interacts with various types of RNAs in ligand dependent manner via its RNA binding motif in hinge region. RNA binding defects lead to a global loss of ERalpha binding in the genome, particularly at weaker ERalpha motifs. In the absence of RNA binding, the ERalpha exhibits dynamic behavior in the nucleus and unexpectedly, the dynamic binding coincides with robust polymerase loading on ERalpha bound chromatin regions. The higher occupancy of PolII was recapitulated by robust ligand induced transcription of ERalpha-regulated genes. Collectively, our results suggest that RNA interactions strengthen ERalpha binding to chromatin limiting the ligand-dependent transcriptional upregulation of estrogen-induced genes.\nStructural basis of archaeal FttA-dependent transcription termination\nAuthors: Wang, C.; Molodtsov, V.; Sanders, T. J.; Marshall, C. J.; Firlar, E.; Kaelber, J. T.; Santangelo, T. J.; Ebright, R. H.\nScore: 5.3, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552649\nThe ribonuclease FttA mediates factor-dependent transcription termination in archaea. Here, we report the structure of a Thermococcus kodakarensis transcription pre-termination complex comprising FttA, Spt4, Spt5, and a transcription elongation complex (TEC). The structure shows that FttA interacts with the TEC in a manner that enables RNA to proceed directly from the TEC RNA-exit channel to the FttA catalytic center and that enables endonucleolytic cleavage of RNA by FttA, followed by 5'-3' exonucleolytic cleavage of RNA by FttA and concomitant 5'-3' translocation of FttA on RNA, to apply mechanical force to the TEC and trigger termination. The structure further reveals that Spt5 bridges FttA and the TEC, explaining how Spt5 stimulates FttA-dependent termination. The results reveal functional analogy between bacterial and archaeal factor-dependent termination, reveal functional homology between archaeal and eukaryotic factor-dependent termination, and reveal fundamental mechanistic similarities in factor-dependent termination in the three domains of life: bacterial, archaeal, and eukaryotic.\n",
  "wordCount" : "2416",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/molecular-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      molecular biology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552727">
        <p class="paperTitle">Eukaryotic RNA-guided endonucleases evolved from a unique clade of bacterial enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoon, P. H.; Skopintsev, P.; Shi, H.; Chen, L.-X.; Adler, B. A.; Al-Shimary, M.; Craig, R. J.; Li, Z.; Amerasekera, J.; Trinidad, M.; Nisonoff, H.; Chen, K.; Lahiri, A.; Boger, R.; Jacobsen, S.; Banfield, J. F.; Doudna, J. A.</p>
        <p class="info">Score: 51.5, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552727' target='https://doi.org/10.1101/2023.08.09.552727'> 10.1101/2023.08.09.552727</a></p>
        <p class="abstract">RNA-guided endonucleases form the crux of diverse biological processes and technologies, including adaptive immunity, transposition, and genome editing. Some of these enzymes are components of insertion sequences (IS) in the IS200/IS605 and IS607 transposon families. Both IS families encode a TnpA transposase and TnpB nuclease, an RNA-guided enzyme ancestral to CRISPR-Cas12. In eukaryotes and their viruses, TnpB homologs occur as two distinct types, Fanzor1 and Fanzor2. We analyzed the evolutionary relationships between prokaryotic TnpBs and eukaryotic Fanzors, revealing that a clade of IS607 TnpBs with unusual active site arrangement found primarily in Cyanobacteriota likely gave rise to both types of Fanzors. The wide-spread nature of Fanzors imply that the properties of this particular group of IS607 TnpBs were particularly suited to adaptation and evolution in eukaryotes and their viruses. Experimental characterization of a prokaryotic IS607 TnpB and virally encoded Fanzor1s uncovered features that may have fostered coevolution between TnpBs/Fanzors and their cognate transposases. Our results provide insight into the evolutionary origins of a ubiquitous family of RNA-guided proteins that shows remarkable conservation across domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552605">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552605" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552605">
        <p class="paperTitle">Orchestration of pluripotent stem cell genome reactivation during mitotic exit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552605" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552605" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Placzek, S.; Vanzan, L.; Suter, D. M.</p>
        <p class="info">Score: 19.0, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552605' target='https://doi.org/10.1101/2023.08.09.552605'> 10.1101/2023.08.09.552605</a></p>
        <p class="abstract">The maintenance of cell identity faces many challenges during mitosis, as most DNA-binding proteins are evicted from DNA and transcription is virtually abolished. How cells maintain their identity through cell division and faithfully re-initiate gene expression during mitotic exit is unclear. Here, we developed a novel reporter system enabling cell cycle synchronization-free separation of pluripotent stem cells in temporal bins of &lt; 30 minutes during mitotic exit. This allowed us to quantify genome-wide reactivation of transcription, sequential changes in chromatin accessibility, and re-binding of the pluripotency transcription factors OCT4, SOX2, and NANOG (OSN). We found that transcriptional activity progressively ramped up after mitosis and that OSN rapidly reoccupied the genome during the anaphase-telophase transition. We also demonstrate transcription factor-specific, dynamic relocation patterns and a hierarchical reorganization of the OSN binding landscape governed by OCT4 and SOX2. Our study sheds light on the dynamic orchestration of transcriptional reactivation after mitosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552315">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552315" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552315">
        <p class="paperTitle">Insights into human outer kinetochore assembly and force transmission from a structure-function analysis of the KMN network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552315" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552315" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Polley, S.; Raisch, T.; Koerner, M.; Terbeck, M.; Graeter, F.; Raunser, S.; Aponte-Santamaria, C.; Vetter, I. R.; Musacchio, A.</p>
        <p class="info">Score: 16.0, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552315' target='https://doi.org/10.1101/2023.08.07.552315'> 10.1101/2023.08.07.552315</a></p>
        <p class="abstract">The biorientation of chromosomes during cell division is necessary for precise dispatching of a mother cells chromosomes into its two daughters. Kinetochores, large layered structures built on specialized chromosome loci named centromeres, promote biorientation by binding and sensing spindle microtubules. The kinetochore outer layer consists of a 10-subunit apparatus comprising Knl1C, Mis12C, and Ndc80C subcomplexes (KMN network). The KMN network is highly elongated and docks on kinetochores and microtubules using interfaces at its opposite extremes. Here, we combine cryo-EM reconstructions and AlphaFold2 predictions to generate a model of the KMN network that reveals all intra-KMN interfaces. We identify and functionally validate two interaction interfaces that link Mis12C to Ndc80C and Knl1C. Through targeted interference experiments and molecular dynamics simulations we demonstrate this mutual organization stabilizes the KMN network. Our work reports the first comprehensive structural and functional analysis of the microtubule binding machinery of kinetochores and elucidates a path of microtubule-generated force transmission</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552750">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552750" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552750">
        <p class="paperTitle">The SARM1 TIR domain produces glycocyclic ADPR molecules as minor products</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552750" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552750" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garb, J.; Amitai, G.; Lu, A.; Ofir, G.; Brandis, A.; Mehlman, T.; Kranzusch, P.; Sorek, R.</p>
        <p class="info">Score: 15.3, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552750' target='https://doi.org/10.1101/2023.08.10.552750'> 10.1101/2023.08.10.552750</a></p>
        <p class="abstract">Sterile alpha and TIR motif-containing 1 (SARM1) is a protein involved in programmed death of injured axons. Following axon injury or a drug-induced insult, the TIR domain of SARM1 degrades the essential molecule nicotinamide adenine dinucleotide (NAD&#43;), leading to a form of axonal death called Wallerian degeneration. Degradation of NAD&#43; by SARM1 is essential for the Wallerian degeneration process, but accumulating evidence suggest that other activities of SARM1, beyond the mere degradation of NAD&#43;, may be necessary for programmed axonal death. In this study we show that the TIR domains of both human and fruit fly SARM1 produce 1&#39;&#39;-2&#39; and 1&#39;&#39;-3&#39; glycocyclic ADP-ribose (gcADPR) molecules as minor products. As previously reported, we observed that SARM1 TIR domains mostly convert NAD&#43; to ADPR (for human SARM1) or cADPR (in the case of SARM1 from Drosophila melanogaster). However, we now show that human and Drosophila SARM1 additionally convert ~0.1-0.5% of NAD&#43; into gcADPR molecules. We find that SARM1 TIR domains produce gcADPR molecules both when purified in vitro and when expressed in bacterial cells. Given that gcADPR is a second messenger involved in programmed cell death in bacteria and likely in plants, we propose that gcADPR may play a role in SARM1-induced programmed axonal death in animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551545">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551545" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551545">
        <p class="paperTitle">Germ granule association drives small RNA specificity for a nuclear Argonaute protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551545" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551545" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, S.; Phillips, C. M.</p>
        <p class="info">Score: 12.3, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551545' target='https://doi.org/10.1101/2023.08.01.551545'> 10.1101/2023.08.01.551545</a></p>
        <p class="abstract">RNA interference (RNAi) is a conserved gene silencing process that exists in diverse organisms to protect genome integrity and regulate gene expression. In C. elegans, the majority of RNAi pathway proteins localize to perinuclear, phase-separated germ granules, which are comprised of sub-domains referred to as P granules, Mutator foci, Z granules, and SIMR foci. However, the protein components and function of the newly discovered SIMR foci are unknown. Here we demonstrate that HRDE-2 localizes to SIMR foci and interacts with the germline nuclear RNAi Argonaute HRDE-1. Furthermore, HRDE-1 also localizes to SIMR foci, dependent on HRDE-2, but only in its small RNA unbound state. This germ granule localization is critical to promote the small RNA binding specificity of HRDE-1 and, in the absence of HRDE-2, HRDE-1 exclusively loads CSR-class 22G-RNAs rather than WAGO-class 22G-RNAs, resulting in inappropriate H3K9me3 deposition on CSR-target genes. Thus, our study demonstrates that the recruitment of unloaded HRDE-1 to germ granules, mediated by HRDE-2, is critical to ensure that the correct small RNAs are used to guide nuclear RNA silencing in the C. elegans germline.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552904">
        <p class="paperTitle">N-glycosylation as a eukaryotic protective mechanism against protein aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duran-Romana, R.; Houben, B.; De Vleeschouwer, M.; Louros, N.; Wilson, M. P.; Matthijs, G.; Schymkowitz, J.; Rousseau, F.</p>
        <p class="info">Score: 7.2, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552904' target='https://doi.org/10.1101/2023.08.11.552904'> 10.1101/2023.08.11.552904</a></p>
        <p class="abstract">The tendency for proteins to form aggregates is an inherent part of every proteome and arises from the self-assembly of short protein segments called aggregation-prone regions (APRs). While post-translational modifications (PTMs) have been implicated in modulating protein aggregation, their direct role in APRs remains poorly understood. In this study, we used a combination of proteome-wide computational analyses and biochemical techniques to investigate the potential involvement of PTMs in aggregation regulation. Our findings reveal that while most PTM types are disfavored near APRs, N-glycosylation is enriched and evolutionarily selected, especially in proteins prone to misfolding. Experimentally, we show that N-glycosylation inhibits the aggregation of peptides in vitro through steric hindrance. Moreover, mining existing proteomics data, we find that the loss of N-glycans at the flanks of APRs leads to specific protein aggregation in Neuro2a cells. Our results point towards a novel intrinsic role for N-glycosylation, directly preventing protein aggregation in eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551356">
        <p class="paperTitle">Intra-host variation in the spike S1/S2 region of a feline coronavirus type-1 in a cat with persistent infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Olarte-Castillo, X. A.; Licitra, B. N.; Andre, N. M.; Sierra, M. A.; Mason, C. E.; Goodman, L. B.; Whittaker, G.</p>
        <p class="info">Score: 45.6, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551356' target='https://doi.org/10.1101/2023.07.31.551356'> 10.1101/2023.07.31.551356</a></p>
        <p class="abstract">Feline coronavirus type 1 (FCoV-1) is widely known for causing feline infectious peritonitis (FIP), a systemic infection that is often fatal, with the virus known as the FIPV biotype. However, subclinical disease also occurs, in which cats may not show signs and intermittently shed the virus, including in feces, possibly for long periods of time. This virus is known as the FECV biotype. Progression of FECV to FIPV has been linked to several genomic changes, however a specific region of the viral spike protein at the interface of the spike S1 and S2 domains has been especially implicated. In this study, we followed a cat (#576) for six years from 2017, at which time FCoV-1 was detected in feces and conjunctival swabs, until 2022, when the animal was euthanized based on a diagnosis of alimentary small cell lymphoma. Over this time period, the cat was clinically diagnosed with inflammatory bowel disease and chronic rhinitis, and cardiac problems were also suspected. Using hybridization capture targeting the spike (S) gene of FCoV followed by next-generation sequencing, we screened 27 clinical samples. We detected FCoV-1 in 4 samples taken in 2017 (intestine and nasal tissue, feces, and conjunctiva), and 3 samples taken in 2022 (feces, and intestinal and heart tissue), but not in fecal samples taken in 2019 and 2020. Next, we focused on the S1/S2 region within S, which contains the furin cleavage site (FCS), a key regulator of viral transmission and pathogenesis. We show that the FCoV-1 variants obtained from feces in 2017 and 2022 were identical, while the ones from conjunctiva (2017), heart (2022), and intestine (2017 and 2022) were distinct. Sequence comparison of all the variants obtained showed that most of the non-synonymous changes in the S1/S2 region occur within the FCS. In the heart, we found two variants that differed by a single nucleotide, resulting in distinct FCS motifs that differ in one amino acid. It is predicted that one of these FCS motifs will down-regulate spike cleavability. The variant from the conjunctiva (2017) had a 6-nucleotide in-frame insertion that resulted in a longer and more exposed S1/S2 loop, which is predicted to be more accessible to the furin protease. Our studies indicate that FCoV-1 can independently persist in the gastrointestinal tract and heart of a cat over a long period of time without evidence of typical FIP signs, with intermittent viral shedding from the gastrointestinal and respiratory tracts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551037">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551037" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551037">
        <p class="paperTitle">Jointly modeling deep mutational scans identifies shifted mutational effects among SARS-CoV-2 spike homologs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551037" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551037" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haddox, H. K.; Galloway, J. G.; Dadonaite, B.; Bloom, J. D.; Matsen, F. A.; DeWitt, W. S.</p>
        <p class="info">Score: 24.1, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551037' target='https://doi.org/10.1101/2023.07.31.551037'> 10.1101/2023.07.31.551037</a></p>
        <p class="abstract">Deep mutational scanning (DMS) is a high-throughput experimental technique that measures the effects of thousands of mutations to a protein. These experiments can be performed on multiple homologs of a protein or on the same protein selected under multiple conditions. It is often of biological interest to identify mutations with shifted effects across homologs or conditions. However, it is challenging to determine if observed shifts arise from biological signal or experimental noise. Here, we describe a method for jointly inferring mutational effects across multiple DMS experiments while also identifying mutations that have shifted in their effects among experiments. A key aspect of our method is to regularize the inferred shifts, so that they are nonzero only when strongly supported by the data. We apply this method to DMS experiments that measure how mutations to spike proteins from SARS-CoV-2 variants (Delta, Omicron BA.1, and Omicron BA.2) affect cell entry. Most mutational effects are conserved between these spike homologs, but a fraction have markedly shifted. We experimentally validate a subset of the mutations inferred to have shifted effects, and confirm differences of &gt;1,000-fold in the impact of the same mutation on spike-mediated viral infection across spikes from different SARS-CoV-2 variants. Overall, our work establishes a general approach for comparing sets of DMS experiments to identify biologically important shifts in mutational effects.

Significance StatementAmino-acid mutations to a protein have effects that can shift as the protein evolves or is put under new selective pressure. The effects of amino-acid mutations to a specific protein under a defined selective pressure can be measured by deep mutational scanning experiments. Here, we devise an approach to quantify shifts in mutational effects between experiments performed on different homologs (i.e. variants) of the same protein, or on the same protein selected under different conditions. We use this approach to compare experiments performed on three homologs of SARS-CoV-2 spike, identifying mutations that have shifted in their effect on spike-mediated viral infection by &gt;1,000 fold across SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552751">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552751" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552751">
        <p class="paperTitle">RNA binding limits the ligand induced transcriptional potential of estrogen receptor-alpha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552751" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552751" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soota, D.; Saravanan, B.; Mann, R.; Kharbanda, T.; Notani, D.</p>
        <p class="info">Score: 5.7, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552751' target='https://doi.org/10.1101/2023.08.10.552751'> 10.1101/2023.08.10.552751</a></p>
        <p class="abstract">Transcription factors (TFs) primarily regulate gene expression by binding to DNA through their DNA binding domains (DBD). Additionally, approximately half of these TFs also interact with RNA. However, the role of RNA in enabling TF binding on chromatin and subsequent transcription is poorly understood. Estrogen receptor-alpha (ERalpha) is one such TF that activates genes in response to estrogen stimulation. Here, we report that ER interacts with various types of RNAs in ligand dependent manner via its RNA binding motif in hinge region. RNA binding defects lead to a global loss of ERalpha binding in the genome, particularly at weaker ERalpha motifs. In the absence of RNA binding, the ERalpha exhibits dynamic behavior in the nucleus and unexpectedly, the dynamic binding coincides with robust polymerase loading on ERalpha bound chromatin regions. The higher occupancy of PolII was recapitulated by robust ligand induced transcription of ERalpha-regulated genes. Collectively, our results suggest that RNA interactions strengthen ERalpha binding to chromatin limiting the ligand-dependent transcriptional upregulation of estrogen-induced genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552649">
        <p class="paperTitle">Structural basis of archaeal FttA-dependent transcription termination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, C.; Molodtsov, V.; Sanders, T. J.; Marshall, C. J.; Firlar, E.; Kaelber, J. T.; Santangelo, T. J.; Ebright, R. H.</p>
        <p class="info">Score: 5.3, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552649' target='https://doi.org/10.1101/2023.08.09.552649'> 10.1101/2023.08.09.552649</a></p>
        <p class="abstract">The ribonuclease FttA mediates factor-dependent transcription termination in archaea. Here, we report the structure of a Thermococcus kodakarensis transcription pre-termination complex comprising FttA, Spt4, Spt5, and a transcription elongation complex (TEC). The structure shows that FttA interacts with the TEC in a manner that enables RNA to proceed directly from the TEC RNA-exit channel to the FttA catalytic center and that enables endonucleolytic cleavage of RNA by FttA, followed by 5&#39;-3&#39; exonucleolytic cleavage of RNA by FttA and concomitant 5&#39;-3&#39; translocation of FttA on RNA, to apply mechanical force to the TEC and trigger termination. The structure further reveals that Spt5 bridges FttA and the TEC, explaining how Spt5 stimulates FttA-dependent termination. The results reveal functional analogy between bacterial and archaeal factor-dependent termination, reveal functional homology between archaeal and eukaryotic factor-dependent termination, and reveal fundamental mechanistic similarities in factor-dependent termination in the three domains of life: bacterial, archaeal, and eukaryotic.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
