<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>pharmacology and toxicology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>pharmacology and toxicology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Targeting cancer with small molecule pan-KRAS degraders
Authors: Popow, J.; Farnaby, W.; Gollner, A.; Kofink, C.; Fischer, G.; Wurm, M.; Zollman, D.; Wijaya, A.; Mischerikow, N.; Hasenoehrl, C.; Prokofeva, P.; Arnhof, H.; Arce-Solano, S.; Bell, S.; Boeck, G.; Diers, E.; Frost, A. B.; Goodwin-Tindall, J.; Karolyi-Oezguer, J.; Khan, S.; Klawatsch, T.; Koegl, M.; Kousek, R.; Kratochvil, B.; Kropatsch, K.; Lauber, A. A.; McLennan, R.; Olt, S.; Peter, D.; Petermann, O.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/pharmacology-and-toxicology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="pharmacology and toxicology" />
<meta property="og:description" content="Targeting cancer with small molecule pan-KRAS degraders
Authors: Popow, J.; Farnaby, W.; Gollner, A.; Kofink, C.; Fischer, G.; Wurm, M.; Zollman, D.; Wijaya, A.; Mischerikow, N.; Hasenoehrl, C.; Prokofeva, P.; Arnhof, H.; Arce-Solano, S.; Bell, S.; Boeck, G.; Diers, E.; Frost, A. B.; Goodwin-Tindall, J.; Karolyi-Oezguer, J.; Khan, S.; Klawatsch, T.; Koegl, M.; Kousek, R.; Kratochvil, B.; Kropatsch, K.; Lauber, A. A.; McLennan, R.; Olt, S.; Peter, D.; Petermann, O." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/pharmacology-and-toxicology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="pharmacology and toxicology"/>
<meta name="twitter:description" content="Targeting cancer with small molecule pan-KRAS degraders
Authors: Popow, J.; Farnaby, W.; Gollner, A.; Kofink, C.; Fischer, G.; Wurm, M.; Zollman, D.; Wijaya, A.; Mischerikow, N.; Hasenoehrl, C.; Prokofeva, P.; Arnhof, H.; Arce-Solano, S.; Bell, S.; Boeck, G.; Diers, E.; Frost, A. B.; Goodwin-Tindall, J.; Karolyi-Oezguer, J.; Khan, S.; Klawatsch, T.; Koegl, M.; Kousek, R.; Kratochvil, B.; Kropatsch, K.; Lauber, A. A.; McLennan, R.; Olt, S.; Peter, D.; Petermann, O."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "pharmacology and toxicology",
      "item": "https://trxiv.yorks0n.com/posts/pharmacology-and-toxicology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "pharmacology and toxicology",
  "name": "pharmacology and toxicology",
  "description": "Targeting cancer with small molecule pan-KRAS degraders\nAuthors: Popow, J.; Farnaby, W.; Gollner, A.; Kofink, C.; Fischer, G.; Wurm, M.; Zollman, D.; Wijaya, A.; Mischerikow, N.; Hasenoehrl, C.; Prokofeva, P.; Arnhof, H.; Arce-Solano, S.; Bell, S.; Boeck, G.; Diers, E.; Frost, A. B.; Goodwin-Tindall, J.; Karolyi-Oezguer, J.; Khan, S.; Klawatsch, T.; Koegl, M.; Kousek, R.; Kratochvil, B.; Kropatsch, K.; Lauber, A. A.; McLennan, R.; Olt, S.; Peter, D.; Petermann, O.",
  "keywords": [
    
  ],
  "articleBody": " Targeting cancer with small molecule pan-KRAS degraders\nAuthors: Popow, J.; Farnaby, W.; Gollner, A.; Kofink, C.; Fischer, G.; Wurm, M.; Zollman, D.; Wijaya, A.; Mischerikow, N.; Hasenoehrl, C.; Prokofeva, P.; Arnhof, H.; Arce-Solano, S.; Bell, S.; Boeck, G.; Diers, E.; Frost, A. B.; Goodwin-Tindall, J.; Karolyi-Oezguer, J.; Khan, S.; Klawatsch, T.; Koegl, M.; Kousek, R.; Kratochvil, B.; Kropatsch, K.; Lauber, A. A.; McLennan, R.; Olt, S.; Peter, D.; Petermann, O.; Roessler, V.; Stolt-Bergner, P.; Strack, P.; Strauss, E.; Trainor, N.; Vetma, V.; Whitworth, C.; Zhong, S.; Quant, J.; Weinstabl, H.; KuÌˆster, B.; Ettmayer, P.; Ciulli, A.\nScore: 44.6, Published: 2023-10-26 DOI: 10.1101/2023.10.24.563163\nDespite the high prevalence of cancers driven by KRAS mutations, to date only the G12C mutation has been clinically proven to be druggable via covalent targeting of the mutated cysteine amino acid residue1. However, in many cancer indications other KRAS mutations, such as G12D and -V, are far more prevalent and small molecule concepts that can address a wider variety of oncogenic KRAS alleles are in high clinical demand2. Here we show that a single small molecule can be used to simultaneously and potently degrade 13 out of 17 of the most prevalent oncogenic KRAS alleles, including those not yet tractable by inhibitors. Compared with inhibition, degradation of oncogenic KRAS results in more profound and sustained pathway modulation across a broad range of KRAS mutant cell lines. As a result, KRAS degraders inhibit growth of the majority of cancer cell lines driven by KRAS mutations while sparing models without genetic KRAS aberrations. Finally, we demonstrate that pharmacological degradation of oncogenic KRAS leads to tumour regression in vivo. Together, these findings unveil a new path towards addressing KRAS driven cancers with small molecule degraders.\nStructure-Based Discovery of a NPFF1R Antagonist with Analgesic Activity\nAuthors: Bender, B. J.; Pickett, J. E.; Braz, J.; Kang, H. J.; Gahbauer, S.; Bhardwaj, K.; Rodriguez-Rosado, S.; Liu, Y.; Jain, M.; Basbaum, A.; Roth, B. L.; Shoichet, B. K.\nScore: 5.0, Published: 2023-10-29 DOI: 10.1101/2023.10.25.564029\nWhile opioid drugs remain among the most effective analgesics for pain management, adverse effects limit their use. Molecules that synergize with opioids, increasing analgesia without increasing side effects, could prove beneficial. A potential way to do so is via the RF-amide receptor system, as NPFFR1 agonists reduce -opioid receptor (OR)-based analgesia while antagonists increase it. These inferences are, however, clouded by the lack of selectivity of most NPFF1R ligands. Seeking selective antagonists of the NPFF1R, we screened a large virtual library against a homology model of NPFF1R. From 26 high-ranking molecules that were synthesized and tested, one antagonized NPFF1R with a Ki of 319 nM. Structure-based optimization led to a 22 nM antagonist of NPFF1R, compound 56, with selectivity against a large panel of GPCRs. When administered alone, 56 has no activity in mouse tail-flick nociception assays. However, coadministration of compound 56 and morphine produced significantly greater antinociception than did morphine alone, consistent with the notion that NPFF1R nociceptive activity occurs via modulation of OR signaling. Surprisingly, in the hot-plate assays 56 was analgesic by itself, suggesting that NPFF1R alone can also confer analgesia. At equi-analgesic doses, combinations of 56 with morphine reduced the common constipation side effect of morphine versus using morphine alone. The high selectivity of 56 and its activity in cooperation with morphine supports further analgesic development against NPFF1R and against the RF-amide family of receptors more generally.\nHigh-content fluorescence bioassay to investigate pore formation, ion channel modulation and cell membrane lysis induced by venoms\nAuthors: Kramer, S.; Kotapati, C.; Cao, Y.; Fry, B. G.; Palpant, N. J.; King, G. F.; Cardoso, F. C.\nScore: 4.8, Published: 2023-10-29 DOI: 10.1101/2023.10.25.563942\nVenoms comprise highly evolved bioactive molecules modulating ion channels, receptors, coagulation factors, and the cellular membrane. This array of targets and bioactivity requires high-content bioassays to aid the development of novel envenomation treatments and biotechnological and pharmacological agents. To address this gap in venom research, we developed a fluorescence-based high-throughput and high-content cellular assay to simultaneously identify common cellular activities produced by venoms: membrane lysis, pore-formation, and ion channel modulation. By combining intracellular calcium with extracellular nucleic acid measurements, we distinguished these venom mechanisms using one cellular assay. We applied our high-content bioassay in three cell types exposed to venom components representing lytic, ion pore-forming or ion channel modulator toxins. Beyond the distinct profiles produced by these three types of action mechanisms, we found that the pore-forming latrotoxin alpha-Lt1a prefers human neuroblastoma to kidney cells and cardiomyocytes, while the lytic bee peptide melittin is not selective. Furthermore, evaluation of snake venoms showed that Elapid species induced rapid membrane lysis, while Viper species showed variable to no activity on neuroblastoma cells. These findings demonstrate that our high-content bioassay distinguish clades and interspecific traits and capture the clinical observations at venom level and is capable of differentiate ion pore-forming from membrane lysis and ion channel modulation. We hope our research will accelerate the understanding of venom biology and the diversity of toxins inducing cytotoxic, cardiotoxic and neurotoxic effects and assist in identifying venom components whose properties could benefit humankind.\nDiscovery of a novel inhibitor of macropinocytosis with antiviral activity\nAuthors: Porebski, B.; Christ, W.; Corman, A.; Haraldsson, M.; Barz, M.; Lidemalm, L.; Haggblad, M.; Illmain, J.; Wright, S.; Murga, M.; Schlegel, J.; Sezgin, E.; Bhabha, G.; Lauschke, V. M.; Lafarga, M.; Klingstrom, J.; Huhn, D.; Fernandez-Capetillo, O.\nScore: 3.5, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563967\nSeveral viruses hijack various forms of endocytosis in order to infect host cells. Here, we report the discovery of a new molecule with antiviral properties that we named virapinib, which limits viral entry by macropinocytosis. The identification of virapinib derives from a chemical screen using High-Throughput Microscopy, where we identified new chemical entities capable of preventing infection with a pseudotype virus expressing the spike (S) protein from SARS-CoV-2. Subsequent experiments confirmed the capacity of virapinib to inhibit infection by SARS-CoV-2, as well as by additional viruses, such as Monkeypox virus and TBEV. Mechanistic analyses revealed that the compound inhibited macropinocytosis, limiting this entry route for the viruses. Importantly, virapinib has no significant toxicity to host cells. In summary, we present a new molecule that inhibits viral entry via the endocytic route, offering a new alternative to prevent viral infection.\nPharmacological and behavioral effects of tryptamines present in psilocybin-containing mushrooms\nAuthors: Rakoczy, R. J.; Runge, G. N.; Sen, A. K.; Sandoval, O.; Nguyen, Q.; Roberts, B. R.; Sciortino, J. H.; Gibbons, W. J.; Friedberg, L. M.; Jones, J. A.; McMurray, M. S.\nScore: 3.2, Published: 2023-10-23 DOI: 10.1101/2023.10.19.563138\nDemand for more efficacious antidepressants, particularly those with a rapid onset of action, has resulted in a reevaluation of psychedelic drugs for their therapeutic potential. Several tryptamines found in psilocybin-containing magic mushrooms share chemical similarities with psilocybin, and early work suggests they may also share receptor targets. However, few studies have explored their pharmacological and behavioral effects. To accomplish this, we compared baeocystin, norbaeocystin, and aeruginascin with psilocybin to determine if they are metabolized by the same enzymes, penetrate the blood brain barrier, serve as ligands for similar centrally located receptors, and modulate behavior in rodents similarly. We first assessed the stability and optimal storage and handling conditions for each compound. In vitro enzyme kinetics assays then found that all compounds shared nearly identical rates of dephosphorylation via alkaline phosphatase and metabolism by monoamine oxidase. Further, we found that only the dephosphorylated products of baeocystin and norbaeocystin could cross a blood brain barrier mimetic to a similar degree as the dephosphorylated form of psilocybin, psilocin. Behaviorally, only psilocybin was found to induce head twitch responses in rats, a marker of 5HT2A agonism and indicator of the compounds hallucinogenic potential. However, like psilocybin, norbaeocystin was also found to improve outcomes in the forced swim test. All compounds were found to cause minimal changes to metrics of renal and hepatic health, suggesting innocuous safety profiles. Collectively, this work suggests that other naturally-occurring tryptamines, especially norbaeocystin, may share the same therapeutic potential as psilocybin, but without causing hallucinations. HIGHLIGHTSO_LIBaeocystin, norbaeocystin, and aeruginascin may have similar therapeutic value to psilocybin, but are understudied C_LIO_LICompound stability varied widely, with dephosphorylated forms showing lowest stability C_LIO_LIRates of metabolism by alkaline phosphatase and monoamine oxidase were similar across compounds C_LIO_LIBlood brain barrier penetration was limited to dephosphorylated forms of psilocybin, baeocystin, and norbaeocystin C_LIO_LIRat behavioral testing suggested norbaeocystin may have therapeutic utility similar to psilocybin, without causing hallucinations C_LI\nSupport Vector Machine based prediction models for drug repurposing and designing novel drugs for colorectal cancer\nAuthors: Sengupta, A.; Kumar, R.\nScore: 1.2, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563215\nColorectal cancer (CRC) has witnessed a concerning increase in incidence and poses a significant therapeutic challenge due to its poor prognosis. There is a pressing demand to identify novel drug therapies to combat CRC. In this study, we addressed this need by utilizing the library of CRC pharmacological profiles of anticancer drugs and developed QSAR models for prediction of alternative and promiscuous anti-cancer compounds for CRC treatment. Our QSAR models demonstrated their robustness by achieving high correlation of determination (R2) after 10-fold cross validation. For 12 CRC cell lines, R2 ranges from 0.609-0.827. Highest performance was achieved for SW1417 and GP5d cell lines with R2, 0.827 and 0.786, respectively. Further, we listed the most common descriptors in the drug profiles of the CRC cell lines and we also checked the correlation of the descriptors with the drug activity. The KRFP314 fingerprint was the predominantly occurring descriptor, with the KRFPC314 fingerprint following closely in prevalence within the drug profiles of the CRC cell lines. Beyond predictive modeling, we also successfully validated drug-to-oncogene relationship via in-silico methods and identified FDA-approved drugs which could be used as potential anti-CRC drugs, paving the way for subsequent in vitro or in vivo experiments to validate their efficacy. To provide easy accessibility and utility of our research findings, we have incorporated these models into a free-to-use user-friendly prediction webserver named, \"ColoRecPred\" hosted on project.iith.ac.in/cgntlab/colorecpred. This web-based tool can be used to screen for potential anti-cancer compounds for CRC.\n",
  "wordCount" : "1662",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/pharmacology-and-toxicology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      pharmacology and toxicology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563163">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563163" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563163">
        <p class="paperTitle">Targeting cancer with small molecule pan-KRAS degraders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563163" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563163" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Popow, J.; Farnaby, W.; Gollner, A.; Kofink, C.; Fischer, G.; Wurm, M.; Zollman, D.; Wijaya, A.; Mischerikow, N.; Hasenoehrl, C.; Prokofeva, P.; Arnhof, H.; Arce-Solano, S.; Bell, S.; Boeck, G.; Diers, E.; Frost, A. B.; Goodwin-Tindall, J.; Karolyi-Oezguer, J.; Khan, S.; Klawatsch, T.; Koegl, M.; Kousek, R.; Kratochvil, B.; Kropatsch, K.; Lauber, A. A.; McLennan, R.; Olt, S.; Peter, D.; Petermann, O.; Roessler, V.; Stolt-Bergner, P.; Strack, P.; Strauss, E.; Trainor, N.; Vetma, V.; Whitworth, C.; Zhong, S.; Quant, J.; Weinstabl, H.; KuÌˆster, B.; Ettmayer, P.; Ciulli, A.</p>
        <p class="info">Score: 44.6, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563163' target='https://doi.org/10.1101/2023.10.24.563163'> 10.1101/2023.10.24.563163</a></p>
        <p class="abstract">Despite the high prevalence of cancers driven by KRAS mutations, to date only the G12C mutation has been clinically proven to be druggable via covalent targeting of the mutated cysteine amino acid residue1. However, in many cancer indications other KRAS mutations, such as G12D and -V, are far more prevalent and small molecule concepts that can address a wider variety of oncogenic KRAS alleles are in high clinical demand2. Here we show that a single small molecule can be used to simultaneously and potently degrade 13 out of 17 of the most prevalent oncogenic KRAS alleles, including those not yet tractable by inhibitors. Compared with inhibition, degradation of oncogenic KRAS results in more profound and sustained pathway modulation across a broad range of KRAS mutant cell lines. As a result, KRAS degraders inhibit growth of the majority of cancer cell lines driven by KRAS mutations while sparing models without genetic KRAS aberrations. Finally, we demonstrate that pharmacological degradation of oncogenic KRAS leads to tumour regression in vivo. Together, these findings unveil a new path towards addressing KRAS driven cancers with small molecule degraders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564029">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564029" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564029">
        <p class="paperTitle">Structure-Based Discovery of a NPFF1R Antagonist with Analgesic Activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564029" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564029" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bender, B. J.; Pickett, J. E.; Braz, J.; Kang, H. J.; Gahbauer, S.; Bhardwaj, K.; Rodriguez-Rosado, S.; Liu, Y.; Jain, M.; Basbaum, A.; Roth, B. L.; Shoichet, B. K.</p>
        <p class="info">Score: 5.0, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564029' target='https://doi.org/10.1101/2023.10.25.564029'> 10.1101/2023.10.25.564029</a></p>
        <p class="abstract">While opioid drugs remain among the most effective analgesics for pain management, adverse effects limit their use. Molecules that synergize with opioids, increasing analgesia without increasing side effects, could prove beneficial. A potential way to do so is via the RF-amide receptor system, as NPFFR1 agonists reduce -opioid receptor (OR)-based analgesia while antagonists increase it. These inferences are, however, clouded by the lack of selectivity of most NPFF1R ligands. Seeking selective antagonists of the NPFF1R, we screened a large virtual library against a homology model of NPFF1R. From 26 high-ranking molecules that were synthesized and tested, one antagonized NPFF1R with a Ki of 319 nM. Structure-based optimization led to a 22 nM antagonist of NPFF1R, compound 56, with selectivity against a large panel of GPCRs. When administered alone, 56 has no activity in mouse tail-flick nociception assays. However, coadministration of compound 56 and morphine produced significantly greater antinociception than did morphine alone, consistent with the notion that NPFF1R nociceptive activity occurs via modulation of OR signaling. Surprisingly, in the hot-plate assays 56 was analgesic by itself, suggesting that NPFF1R alone can also confer analgesia. At equi-analgesic doses, combinations of 56 with morphine reduced the common constipation side effect of morphine versus using morphine alone. The high selectivity of 56 and its activity in cooperation with morphine supports further analgesic development against NPFF1R and against the RF-amide family of receptors more generally.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563942">
        <p class="paperTitle">High-content fluorescence bioassay to investigate pore formation, ion channel modulation and cell membrane lysis induced by venoms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kramer, S.; Kotapati, C.; Cao, Y.; Fry, B. G.; Palpant, N. J.; King, G. F.; Cardoso, F. C.</p>
        <p class="info">Score: 4.8, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563942' target='https://doi.org/10.1101/2023.10.25.563942'> 10.1101/2023.10.25.563942</a></p>
        <p class="abstract">Venoms comprise highly evolved bioactive molecules modulating ion channels, receptors, coagulation factors, and the cellular membrane. This array of targets and bioactivity requires high-content bioassays to aid the development of novel envenomation treatments and biotechnological and pharmacological agents. To address this gap in venom research, we developed a fluorescence-based high-throughput and high-content cellular assay to simultaneously identify common cellular activities produced by venoms: membrane lysis, pore-formation, and ion channel modulation. By combining intracellular calcium with extracellular nucleic acid measurements, we distinguished these venom mechanisms using one cellular assay. We applied our high-content bioassay in three cell types exposed to venom components representing lytic, ion pore-forming or ion channel modulator toxins. Beyond the distinct profiles produced by these three types of action mechanisms, we found that the pore-forming latrotoxin alpha-Lt1a prefers human neuroblastoma to kidney cells and cardiomyocytes, while the lytic bee peptide melittin is not selective. Furthermore, evaluation of snake venoms showed that Elapid species induced rapid membrane lysis, while Viper species showed variable to no activity on neuroblastoma cells. These findings demonstrate that our high-content bioassay distinguish clades and interspecific traits and capture the clinical observations at venom level and is capable of differentiate ion pore-forming from membrane lysis and ion channel modulation. We hope our research will accelerate the understanding of venom biology and the diversity of toxins inducing cytotoxic, cardiotoxic and neurotoxic effects and assist in identifying venom components whose properties could benefit humankind.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563967">
        <p class="paperTitle">Discovery of a novel inhibitor of macropinocytosis with antiviral activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porebski, B.; Christ, W.; Corman, A.; Haraldsson, M.; Barz, M.; Lidemalm, L.; Haggblad, M.; Illmain, J.; Wright, S.; Murga, M.; Schlegel, J.; Sezgin, E.; Bhabha, G.; Lauschke, V. M.; Lafarga, M.; Klingstrom, J.; Huhn, D.; Fernandez-Capetillo, O.</p>
        <p class="info">Score: 3.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563967' target='https://doi.org/10.1101/2023.10.25.563967'> 10.1101/2023.10.25.563967</a></p>
        <p class="abstract">Several viruses hijack various forms of endocytosis in order to infect host cells. Here, we report the discovery of a new molecule with antiviral properties that we named virapinib, which limits viral entry by macropinocytosis. The identification of virapinib derives from a chemical screen using High-Throughput Microscopy, where we identified new chemical entities capable of preventing infection with a pseudotype virus expressing the spike (S) protein from SARS-CoV-2. Subsequent experiments confirmed the capacity of virapinib to inhibit infection by SARS-CoV-2, as well as by additional viruses, such as Monkeypox virus and TBEV. Mechanistic analyses revealed that the compound inhibited macropinocytosis, limiting this entry route for the viruses. Importantly, virapinib has no significant toxicity to host cells. In summary, we present a new molecule that inhibits viral entry via the endocytic route, offering a new alternative to prevent viral infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563138">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563138" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563138">
        <p class="paperTitle">Pharmacological and behavioral effects of tryptamines present in psilocybin-containing mushrooms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563138" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563138" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rakoczy, R. J.; Runge, G. N.; Sen, A. K.; Sandoval, O.; Nguyen, Q.; Roberts, B. R.; Sciortino, J. H.; Gibbons, W. J.; Friedberg, L. M.; Jones, J. A.; McMurray, M. S.</p>
        <p class="info">Score: 3.2, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563138' target='https://doi.org/10.1101/2023.10.19.563138'> 10.1101/2023.10.19.563138</a></p>
        <p class="abstract">Demand for more efficacious antidepressants, particularly those with a rapid onset of action, has resulted in a reevaluation of psychedelic drugs for their therapeutic potential. Several tryptamines found in psilocybin-containing  magic mushrooms share chemical similarities with psilocybin, and early work suggests they may also share receptor targets. However, few studies have explored their pharmacological and behavioral effects. To accomplish this, we compared baeocystin, norbaeocystin, and aeruginascin with psilocybin to determine if they are metabolized by the same enzymes, penetrate the blood brain barrier, serve as ligands for similar centrally located receptors, and modulate behavior in rodents similarly. We first assessed the stability and optimal storage and handling conditions for each compound. In vitro enzyme kinetics assays then found that all compounds shared nearly identical rates of dephosphorylation via alkaline phosphatase and metabolism by monoamine oxidase. Further, we found that only the dephosphorylated products of baeocystin and norbaeocystin could cross a blood brain barrier mimetic to a similar degree as the dephosphorylated form of psilocybin, psilocin. Behaviorally, only psilocybin was found to induce head twitch responses in rats, a marker of 5HT2A agonism and indicator of the compounds hallucinogenic potential. However, like psilocybin, norbaeocystin was also found to improve outcomes in the forced swim test. All compounds were found to cause minimal changes to metrics of renal and hepatic health, suggesting innocuous safety profiles. Collectively, this work suggests that other naturally-occurring tryptamines, especially norbaeocystin, may share the same therapeutic potential as psilocybin, but without causing hallucinations.

HIGHLIGHTSO_LIBaeocystin, norbaeocystin, and aeruginascin may have similar therapeutic value to psilocybin, but are understudied
C_LIO_LICompound stability varied widely, with dephosphorylated forms showing lowest stability
C_LIO_LIRates of metabolism by alkaline phosphatase and monoamine oxidase were similar across compounds
C_LIO_LIBlood brain barrier penetration was limited to dephosphorylated forms of psilocybin, baeocystin, and norbaeocystin
C_LIO_LIRat behavioral testing suggested norbaeocystin may have therapeutic utility similar to psilocybin, without causing hallucinations
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563215">
        <p class="paperTitle">Support Vector Machine based prediction models for drug repurposing and designing novel drugs for colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, A.; Kumar, R.</p>
        <p class="info">Score: 1.2, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563215' target='https://doi.org/10.1101/2023.10.20.563215'> 10.1101/2023.10.20.563215</a></p>
        <p class="abstract">Colorectal cancer (CRC) has witnessed a concerning increase in incidence and poses a significant therapeutic challenge due to its poor prognosis. There is a pressing demand to identify novel drug therapies to combat CRC. In this study, we addressed this need by utilizing the library of CRC pharmacological profiles of anticancer drugs and developed QSAR models for prediction of alternative and promiscuous anti-cancer compounds for CRC treatment. Our QSAR models demonstrated their robustness by achieving high correlation of determination (R2) after 10-fold cross validation. For 12 CRC cell lines, R2 ranges from 0.609-0.827. Highest performance was achieved for SW1417 and GP5d cell lines with R2, 0.827 and 0.786, respectively. Further, we listed the most common descriptors in the drug profiles of the CRC cell lines and we also checked the correlation of the descriptors with the drug activity. The KRFP314 fingerprint was the predominantly occurring descriptor, with the KRFPC314 fingerprint following closely in prevalence within the drug profiles of the CRC cell lines. Beyond predictive modeling, we also successfully validated drug-to-oncogene relationship via in-silico methods and identified FDA-approved drugs which could be used as potential anti-CRC drugs, paving the way for subsequent in vitro or in vivo experiments to validate their efficacy. To provide easy accessibility and utility of our research findings, we have incorporated these models into a free-to-use user-friendly prediction webserver named, &#34;ColoRecPred&#34; hosted on project.iith.ac.in/cgntlab/colorecpred. This web-based tool can be used to screen for potential anti-cancer compounds for CRC.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
