<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="How to barcode (almost all) freshwater biodiversity
Authors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.
Score: 16.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571596
Freshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="How to barcode (almost all) freshwater biodiversity
Authors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.
Score: 16.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571596
Freshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="How to barcode (almost all) freshwater biodiversity
Authors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.
Score: 16.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571596
Freshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "How to barcode (almost all) freshwater biodiversity\nAuthors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.\nScore: 16.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571596\nFreshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation.",
  "keywords": [
    
  ],
  "articleBody": " How to barcode (almost all) freshwater biodiversity\nAuthors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.\nScore: 16.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571596\nFreshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation. Environmental DNA surveys offer a means to inventory and monitor aquatic diversity, yet most studies focus on one or a few taxonomic groups only. In this study, we sought to 1) identify thoroughly validated, cost-efficient primer pair combinations that maximize detection of broad swaths of freshwater diversity, and 2) facilitate future primer pair selection by creating a free online and user-friendly tool. We first evaluated the completeness of public reference sequence databases and the efficiency of 14 primer pairs using an in silico approach, and then performed eDNA surveys using five mock communities (mix of DNA from tissues), water samples from aquarium samples with known taxonomic composition, and finally water samples from freshwater systems in Eastern Canada. We highlight the power of eDNA-based metabarcoding for reconstructing freshwater communities, including prey, parasite, pathogen, invasive, and declining species. Our work reveals the importance of the marker choice on species resolution, as well as the importance of degenerate primers, the length of the target fragment and the filtering parameters on detection success in water eDNA samples. Our new online tool SNIPe revealed that 13 to 14 primer pairs are necessary to recover 100% of the species in water samples (aquarium and natural systems), but four primer pairs are sufficient to recover almost 75% of taxa with little overlap. These results highlight the usefulness of eDNA for freshwater monitoring and should prompt more studies on tools to survey all-inclusive communities.\nConnectivity of marine predators over the Patagonian Shelf during the highly pathogenic avian influenza (HPAI) outbreak\nAuthors: Riaz, J.; Orben, R. A.; Gamble, A.; Tierney, M.; Catry, P.; Campioni, L.; Granadeiro, J. P.; Baylis, A. M. M.\nScore: 15.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.570574\nAnimal movement and population connectivity are key areas of uncertainty in efforts to understand and predict the spread of infectious disease. The emergence of highly pathogenic avian influenza (HPAI) in South America poses a significant threat to globally significant populations of colonial breeding marine predators in the South Atlantic. Yet, there is a poor understanding of which species or migratory pathways may facilitate disease spread. Compiling one of the largest available animal tracking datasets in the South Atlantic, we examine connectivity and inter-population mixing for colonial breeding marine predators tagged at the Falkland Islands. We reveal extensive connectivity for three regionally dominant and gregarious species over the Patagonian Shelf. Black browed albatrosses (BBA), South American fur seals (SAFS) and Magellanic penguins (MAG) used coastal waters along the Atlantic coast of South America (Argentina and Uruguay). These behaviours were recorded at or in close proximity to breeding colonies and haul-out areas with dense aggregations of marine predators. Transit times to and from the Falkland Islands to the continental coast ranged from 0.2 - 70 days, with 84% of animals making this transit within 4 days - a conservative estimate for HPAI infectious period. Our findings show the incursion of HPAI to the Falkland Islands marine predator community is a highly credible threat, which may be facilitated by BBA, SAFS and MAG connectivity with South America. This information is vital in supporting HPAI disease surveillance, risk assessment and marine management efforts across the region. SignificanceThe recent emergence of highly pathogenic avian influenza (HPAI) in South America poses a major threat to globally significant marine predator populations in the South Atlantic. There is extensive connectivity over the southern Patagonian Shelf between regionally dominant seal and seabird populations, with potential for large-scale pathogen spread. Despite this connectivity, outbreaks of HPAI are unevenly distributed across the region. Connectivity information is integral for regional disease surveillance, predictive modelling and population viability assessments.\nClimate change has desynchronized insect and vegetation phenologies across Europe\nAuthors: Huang, Y.; Yu, C.; Huang, W.; Liu, Y.; Qi, M.; Bai, J.; Dong, Y.; Gascoigne, S.; Ciais, P.; Penuelas, J.; Salguero-Gomez, R.\nScore: 14.4, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571152\nClimate change is drastically affecting the phenologies of species worldwide, including shifts in development1-3. The potential impact of climate change on the synchronicity of plant-insect phenology is particularly concerning since the stability of pollination networks and food chains depends on their tight temporal dependencies4-6. Furthermore, the recently reported \"insect armageddon\"7 makes it urgent to establish trends and identify primary drivers of plant-insect phenological synchrony. Here, coupling high-resolution remote sensing and citizen science data across Europe over 34 years, we examine the differences in occurrence dates of 1,584 herbivorous insects across four orders and the corresponding dates of leaf unfolding. We find that 61.2% of the vegetation and insect phenologies have become highly asynchronous, with vegetation phenology advancing four-fold faster than insect phenology. These trends were modulated by insects life-cycles and taxonomic order. A primary driver of this phenological mismatch is the higher sensitivity of vegetation phenology than insect phenology to climate, which has prevented insects from matching the pace of plant phenological advance in the growing season. Our analyses provide a unique continental overview and underlying mechanisms of the asynchronicity between vegetation and insect phenologies, thus enhancing our ability to predict and manage its potential cascading ecological effects.\nEnvironmental gradients mediate dispersal evolution during biological invasions\nAuthors: Benning, J. W.; Clark, E. I.; Hufbauer, R. A.; Weiss-Lehman, C.\nScore: 9.3, Published: 2023-12-15 DOI: 10.1101/2023.12.08.570855\nRapid dispersal evolution at the edge of a range expansion can accelerate invasions. However, expanding populations will often encounter environmental gradients that entail a fitness cost of dispersal. We used an eco-evolutionary model to explore how environmental heterogeneity influences adaptation and dispersal evolution during range expansion and, in turn, modulates the speed and predictability of invasion. Environmental gradients opposed the evolution of increased dispersal during invasion, even leading to the evolution of reduced dispersal along steep gradients. Counterintuitively, reduced dispersal allowed for faster expansion by minimizing maladaptation. While evolution across homogenous landscapes caused invasions to be highly unpredictable, even shallow environmental gradients greatly increased invasion predictability. We illustrate our models potential application to prediction and management of invasions by parameterizing it with data from a recent invertebrate range expansion. Overall, we find that environmental heterogeneity and local adaptation strongly modulate the effect of dispersal evolution on invasion trajectories.\nMetaproteogenomics resolution of a high-CO2 aquifer community suggests an active symbiotic lifestyle of groundwater Gracilibacteria\nAuthors: Figueroa-Gonzalez, P. A.; Bornemann, T. L. V.; Hinzke, T.; Maass, S.; Trautwein-Schult, A.; Starke, J.; Moore, C. J.; Esser, S. P.; Plewka, J.; Hesse, T.; Schmidt, T. C.; Schreiber, U.; Bor, B.; Becher, D.; Probst, A. J.\nScore: 7.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572140\nBackground. Bacteria of the Candidate Phyla Radiation (CPR), constituting about 25% of the bacterial biodiversity, are characterized by small cell size and patchy genomes without complete key metabolic pathways, suggesting a symbiotic lifestyle. Gracilibacteria (BD1-5), which are part of the CPR branch, possess alternate coded genomes and have not yet been cultivated. However, besides genomic evidence, little is known about the lifestyle of Gracilibacteria, their temporal dynamics, and activity in natural ecosystems, particularly in groundwater, where they were initially been genomically resolved. Therefore, we here aimed to investigate Gracilibacteria activity in situ and to discern expressed genes involved in their lifestyle, using the metaproteogenome of Gracilibacteria as a function of time in the cold-water geyser Wallender Born in the Volcanic Eifel region in Germany. Results. We coupled genome-resolved metagenomics and metaproteomics to investigate a cold-water geyser microbial community enriched in Gracilibacteria across a 12-day time-series. Groundwater was collected and sequentially filtered to fraction CPR and other bacteria. Based on 670 Gbps of metagenomic data, 1129 different ribosomal protein S3 marker genes and 751 high-quality genomes (123 population genomes after dereplication), we identified dominant bacteria belonging to Galionellales and Gracilibacteria along with keystone microbes, which were low in genomic abundance but substantially contributing to proteomic abundance. Seven high-quality Gracilibacteria genomes showed typical limitations, such as limited amino acid or nucleotide synthesis, in their central metabolism but no co-occurrence with potential hosts. The genomes of these Gracilibacteria encoded for a high number of proteins related to a symbiotic or even predatory lifestyle, e.g., type IV and type II secretion system subunits and features related to cell-cell interactions and cell motility, which were also detected on protein level. Conclusions. Coupling metagenomics to metaproteomics enabled us to identify microbial keystone taxa in a high-CO2 aquifer, and to reveal microbial dynamics of Gracilibacteria. We posit that Gracilibacteria might be successful microbial predators in this ecosystem, potentially aiding in population control of this highly perturbed microbial geyser community from the deep biosphere.\nSeasonal and environmental factors contribute to the variation in the gut microbiome: a large-scale study of a small bird\nAuthors: Liukkonen, M.; Muriel, J.; Martinez-Padilla, J.; Nord, A.; Pakanen, V.-M.; Rosivall, B.; Tilgar, V.; van Oers, K.; Grond, K.; Ruuskanen, S.\nScore: 7.6, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571395\nEnvironmental variation can shape the gut microbiome, but majority of studies use captive-bred species, while data on large-scale variation in the gut microbiome and the associated environmental factors is lacking. Furthermore, previous studies have limited taxonomical coverage, and for example knowledge about avian gut microbiomes is still scarce. We investigated large-scale environmental variation in the gut microbiome of wild adult great tits across the species European distribution range. Our results show that gut microbiome diversity is higher during winter and that there are compositional differences between winter and summer gut microbiomes. During winter, individuals inhabiting mixed forest habitat show higher gut microbiome diversity, whereas there was no similar association during summer. Also, temperature was found to be a small contributor to compositional differences in the gut microbiome. We did not find significant differences in the gut microbiome among populations, nor any association between latitude, rainfall, and the gut microbiome. The results suggest that there is a seasonal change in wild avian gut microbiomes, but that there are still many unknown factors that shape the gut microbiome of wild bird populations.\nThermal tolerance and survival are modulated by a natural gradient of infection in differentially acclimated hosts\nAuthors: De Bonville, J.; Cote, A.; Binning, S. A.\nScore: 7.5, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571921\nWild ectotherms are exposed to multiple stressors, including parasites, which can affect their responses to environmental change. Simultaneously, unprecedented warm temperatures are being recorded worldwide, increasing both the average and maximum temperatures experienced in nature. Understanding how ectotherms, such as fishes, will react to the combined stress of parasites and higher average temperatures can help predict the impact of extreme events such as heat waves on populations. The critical thermal method (CTM), which assesses upper (CTmax) and lower (CTmin) thermal tolerance, is often used in acclimated ectotherms to help predict their tolerance to various temperature scenarios. Yet, few studies have characterized the response of naturally infected fish to extreme temperature events or how acute thermal stress affects subsequent survival. We acclimated naturally infected pumpkinseed sunfish (Lepomis gibbosus), to four ecologically relevant temperatures (10, 15, 20 and 25{degrees}C) and one future warming scenario (30{degrees}C) for three weeks, before measuring CTmax and CTmin. We also assessed individual survival the week following CTmax. Interestingly, trematode parasites causing black spot disease were negatively related to CTmax, suggesting that heavily infected fish are less tolerant to acute warming. Moreover, fish infected with yellow grub parasites showed decreased survival in the days following CTmax implying that the infection load has negative survival consequences on sunfish during extreme warming events. Our findings indicate that parasite infection and high prolonged average temperatures can affect fish thermal tolerance and survival, emphasizing the need to better understand the concomitant effects of stressors on health outcomes in wild populations. Summary statementThis study shows that parasites influence thermal tolerance and survival of fish, suggesting that such stressors are important to increase the ecological relevance of thermal tolerance studies of wild animals.\nEnvironmental DNA captures the internal structure of a pond metacommunity\nAuthors: Cai, W.; Pichler, M.; Biggs, J.; Nicolet, P.; Ewald, N.; Griffiths, R. A.; Bush, A.; Leibold, M. A.; Hartig, F.; Yu, D. W.\nScore: 3.7, Published: 2023-12-12 DOI: 10.1101/2023.12.12.571176\nMetacommunity theory unites community and spatial ecology. Recent innovations in sequencing technology may now allow us, for the first time, to confront this theory with entire trophic community data at large scales. We metabarcoded vertebrate eDNA from 320 ponds in England. Using this novel community data, we calculated the internal structure of this metacommunity, which is the contributions of environmental, biotic, and spatial processes, as well as stochasticity, to the distribution of each species and the composition of each site. Our results suggest that community assembly processes in this pondscape do not conform to a single metacommunity paradigm, but rather vary among species and over space. Combining this variation with external knowledge of species biologies allows us to better understand the differential influences of environmental filtering, species interactions, and dispersal limitation on several of the vertebrate species, paving the way for the routine empirical study of metacommunities.\nIncreasing marine trophic web knowledge through DNA analyses of fish stomach contents: a step towards an Ecosystem Based Approach to fisheries research\nAuthors: Canals, O.; Lanzen, A.; Mendibil, I.; Bachiller, E.; Corrales, X.; Andonegi, E.; Cotano, U.; Rodriguez-Ezpeleta, N.\nScore: 2.9, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572137\nMultispecies and ecosystem models, which are key for the implementation of ecosystem-based approaches to fisheries management, require extensive data on the trophic interactions between marine organisms, including changes over time. DNA metabarcoding (the simultaneous identification of all taxa in a sample by sequencing a short DNA region) could be used for speeding up large-scaled stomach content data collection. Yet, for DNA metabarcoding to be routinely implemented, technical challenges should be addressed, such as the potentially complicated sampling logistics, the detection of a high proportion of predator DNA, and the inability to provide reliable abundance estimations. Here, we present a DNA metabarcoding assay developed to examine the diet of five commercially important fish, which is feasible to be incorporated into routinary samplings. The method is devised to speed up the analysis process by avoiding the stomach dissection and content extraction steps, while preventing the amplification of predator DNA by using blocking primers. Tested in mock samples and in real stomachs, the method has proven effective and shows great effectiveness discerning diet variations due to predator ecology or prey availability. Additionally, by applying our protocol to mackerel stomachs previously analysed by visual inspection, we showcase how DNA metabarcoding could complement visually based data by detecting overlooked prey by the visual approach. We finally discuss how DNA metabarcoding-based data can contribute to trophic data collection. Our work reinforces the potential of DNA metabarcoding for the study and monitoring of fish trophic interactions and provides a basis for its incorporation into routine monitoring programs, which will be critical for the implementation of ecosystem-based approaches to fisheries management.\nImpact of climate change and oligotrophication on quality and quantity of lake primary production: A case study in Lake Biwa\nAuthors: Kazama, T.; Hayakawa, K.; Nagata, T.; Shimotori, K.; Imai, A.\nScore: 2.9, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571786\nGlobal climate change and anthropogenic oligotrophication are expected to reshape the dynamics of primary production (PP) in aquatic ecosystems; however, few studies have explored their long-term effects. In theory, the PP of phytoplankton in Lake Biwa may decline over decades due to warming, heightened stratification, and anthropogenic oligotrophication. Furthermore, the PP of large phytoplankton, which are inedible to zooplankton, along with biomass-specific productivity (PBc), could decrease. In this study, data from 1976-2021 and active fluorometry measurements taken in 2020 and 2021 were evaluated. Quantitatively, the temporal dynamics of mean seasonal PP during 1971-2021 were assessed according to the carbon fixation rate to investigate relationships among environmental factors. Qualitatively, phytoplankton biomass, PP, and PBc were measured in two size fractions [edible (S) or inedible (L) for zooplankton] in 2020 and 2021, and the L:S balance for these three measures was compared between 1992 (low-temperature/high-nutrient conditions) and 2020-2021 (high-temperature/low-nutrient conditions) to assess seasonal dynamics. The results indicated that climate change and anthropogenic oligotrophication over the past 50 years have diminished Lake Biwas PP since the 1990s, impacting the phenology of PP dynamics. However, the L:S balance in PP and PBc has exhibited minimal change since 1992. These findings suggest that, although climate change and oligotrophication may reduce overall PP, they do not markedly alter the inedible/edible phytoplankton balance in terms of PP and PBc. Instead, as total PP declines, the production of small edible phytoplankton decreases proportionally, potentially affecting trophic transfer efficiency and material cycling in Lake Biwa.\n",
  "wordCount" : "2801",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571596">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571596" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571596">
        <p class="paperTitle">How to barcode (almost all) freshwater biodiversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571596" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571596" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.</p>
        <p class="info">Score: 16.6, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571596' target='https://doi.org/10.1101/2023.12.13.571596'> 10.1101/2023.12.13.571596</a></p>
        <p class="abstract">Freshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation. Environmental DNA surveys offer a means to inventory and monitor aquatic diversity, yet most studies focus on one or a few taxonomic groups only. In this study, we sought to 1) identify thoroughly validated, cost-efficient primer pair combinations that maximize detection of broad swaths of freshwater diversity, and 2) facilitate future primer pair selection by creating a free online and user-friendly tool. We first evaluated the completeness of public reference sequence databases and the efficiency of 14 primer pairs using an in silico approach, and then performed eDNA surveys using five mock communities (mix of DNA from tissues), water samples from aquarium samples with known taxonomic composition, and finally water samples from freshwater systems in Eastern Canada. We highlight the power of eDNA-based metabarcoding for reconstructing freshwater communities, including prey, parasite, pathogen, invasive, and declining species. Our work reveals the importance of the marker choice on species resolution, as well as the importance of degenerate primers, the length of the target fragment and the filtering parameters on detection success in water eDNA samples. Our new online tool SNIPe revealed that 13 to 14 primer pairs are necessary to recover 100% of the species in water samples (aquarium and natural systems), but four primer pairs are sufficient to recover almost 75% of taxa with little overlap. These results highlight the usefulness of eDNA for freshwater monitoring and should prompt more studies on tools to survey all-inclusive communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.570574">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.570574" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.570574">
        <p class="paperTitle">Connectivity of marine predators over the Patagonian Shelf during the highly pathogenic avian influenza (HPAI) outbreak</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.570574" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.570574" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Riaz, J.; Orben, R. A.; Gamble, A.; Tierney, M.; Catry, P.; Campioni, L.; Granadeiro, J. P.; Baylis, A. M. M.</p>
        <p class="info">Score: 15.0, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.570574' target='https://doi.org/10.1101/2023.12.12.570574'> 10.1101/2023.12.12.570574</a></p>
        <p class="abstract">Animal movement and population connectivity are key areas of uncertainty in efforts to understand and predict the spread of infectious disease. The emergence of highly pathogenic avian influenza (HPAI) in South America poses a significant threat to globally significant populations of colonial breeding marine predators in the South Atlantic. Yet, there is a poor understanding of which species or migratory pathways may facilitate disease spread. Compiling one of the largest available animal tracking datasets in the South Atlantic, we examine connectivity and inter-population mixing for colonial breeding marine predators tagged at the Falkland Islands. We reveal extensive connectivity for three regionally dominant and gregarious species over the Patagonian Shelf. Black browed albatrosses (BBA), South American fur seals (SAFS) and Magellanic penguins (MAG) used coastal waters along the Atlantic coast of South America (Argentina and Uruguay). These behaviours were recorded at or in close proximity to breeding colonies and haul-out areas with dense aggregations of marine predators. Transit times to and from the Falkland Islands to the continental coast ranged from 0.2 - 70 days, with 84% of animals making this transit within 4 days - a conservative estimate for HPAI infectious period. Our findings show the incursion of HPAI to the Falkland Islands marine predator community is a highly credible threat, which may be facilitated by BBA, SAFS and MAG connectivity with South America. This information is vital in supporting HPAI disease surveillance, risk assessment and marine management efforts across the region.

SignificanceThe recent emergence of highly pathogenic avian influenza (HPAI) in South America poses a major threat to globally significant marine predator populations in the South Atlantic. There is extensive connectivity over the southern Patagonian Shelf between regionally dominant seal and seabird populations, with potential for large-scale pathogen spread. Despite this connectivity, outbreaks of HPAI are unevenly distributed across the region. Connectivity information is integral for regional disease surveillance, predictive modelling and population viability assessments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571152">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571152" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571152">
        <p class="paperTitle">Climate change has desynchronized insect and vegetation phenologies across Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571152" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571152" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, Y.; Yu, C.; Huang, W.; Liu, Y.; Qi, M.; Bai, J.; Dong, Y.; Gascoigne, S.; Ciais, P.; Penuelas, J.; Salguero-Gomez, R.</p>
        <p class="info">Score: 14.4, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571152' target='https://doi.org/10.1101/2023.12.11.571152'> 10.1101/2023.12.11.571152</a></p>
        <p class="abstract">Climate change is drastically affecting the phenologies of species worldwide, including shifts in development1-3. The potential impact of climate change on the synchronicity of plant-insect phenology is particularly concerning since the stability of pollination networks and food chains depends on their tight temporal dependencies4-6. Furthermore, the recently reported &#34;insect armageddon&#34;7 makes it urgent to establish trends and identify primary drivers of plant-insect phenological synchrony. Here, coupling high-resolution remote sensing and citizen science data across Europe over 34 years, we examine the differences in occurrence dates of 1,584 herbivorous insects across four orders and the corresponding dates of leaf unfolding. We find that 61.2% of the vegetation and insect phenologies have become highly asynchronous, with vegetation phenology advancing four-fold faster than insect phenology. These trends were modulated by insects life-cycles and taxonomic order. A primary driver of this phenological mismatch is the higher sensitivity of vegetation phenology than insect phenology to climate, which has prevented insects from matching the pace of plant phenological advance in the growing season. Our analyses provide a unique continental overview and underlying mechanisms of the asynchronicity between vegetation and insect phenologies, thus enhancing our ability to predict and manage its potential cascading ecological effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570855">
        <p class="paperTitle">Environmental gradients mediate dispersal evolution during biological invasions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benning, J. W.; Clark, E. I.; Hufbauer, R. A.; Weiss-Lehman, C.</p>
        <p class="info">Score: 9.3, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570855' target='https://doi.org/10.1101/2023.12.08.570855'> 10.1101/2023.12.08.570855</a></p>
        <p class="abstract">Rapid dispersal evolution at the edge of a range expansion can accelerate invasions. However, expanding populations will often encounter environmental gradients that entail a fitness cost of dispersal. We used an eco-evolutionary model to explore how environmental heterogeneity influences adaptation and dispersal evolution during range expansion and, in turn, modulates the speed and predictability of invasion. Environmental gradients opposed the evolution of increased dispersal during invasion, even leading to the evolution of reduced dispersal along steep gradients. Counterintuitively, reduced dispersal allowed for faster expansion by minimizing maladaptation. While evolution across homogenous landscapes caused invasions to be highly unpredictable, even shallow environmental gradients greatly increased invasion predictability. We illustrate our models potential application to prediction and management of invasions by parameterizing it with data from a recent invertebrate range expansion. Overall, we find that environmental heterogeneity and local adaptation strongly modulate the effect of dispersal evolution on invasion trajectories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572140">
        <p class="paperTitle">Metaproteogenomics resolution of a high-CO2 aquifer community suggests an active symbiotic lifestyle of groundwater Gracilibacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa-Gonzalez, P. A.; Bornemann, T. L. V.; Hinzke, T.; Maass, S.; Trautwein-Schult, A.; Starke, J.; Moore, C. J.; Esser, S. P.; Plewka, J.; Hesse, T.; Schmidt, T. C.; Schreiber, U.; Bor, B.; Becher, D.; Probst, A. J.</p>
        <p class="info">Score: 7.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572140' target='https://doi.org/10.1101/2023.12.18.572140'> 10.1101/2023.12.18.572140</a></p>
        <p class="abstract">Background. Bacteria of the Candidate Phyla Radiation (CPR), constituting about 25% of the bacterial biodiversity, are characterized by small cell size and patchy genomes without complete key metabolic pathways, suggesting a symbiotic lifestyle. Gracilibacteria (BD1-5), which are part of the CPR branch, possess alternate coded genomes and have not yet been cultivated. However, besides genomic evidence, little is known about the lifestyle of Gracilibacteria, their temporal dynamics, and activity in natural ecosystems, particularly in groundwater, where they were initially been genomically resolved. Therefore, we here aimed to investigate Gracilibacteria activity in situ and to discern expressed genes involved in their lifestyle, using the metaproteogenome of Gracilibacteria as a function of time in the cold-water geyser Wallender Born in the Volcanic Eifel region in Germany. Results. We coupled genome-resolved metagenomics and metaproteomics to investigate a cold-water geyser microbial community enriched in Gracilibacteria across a 12-day time-series. Groundwater was collected and sequentially filtered to fraction CPR and other bacteria. Based on 670 Gbps of metagenomic data, 1129 different ribosomal protein S3 marker genes and 751 high-quality genomes (123 population genomes after dereplication), we identified dominant bacteria belonging to Galionellales and Gracilibacteria along with keystone microbes, which were low in genomic abundance but substantially contributing to proteomic abundance. Seven high-quality Gracilibacteria genomes showed typical limitations, such as limited amino acid or nucleotide synthesis, in their central metabolism but no co-occurrence with potential hosts. The genomes of these Gracilibacteria encoded for a high number of proteins related to a symbiotic or even predatory lifestyle, e.g., type IV and type II secretion system subunits and features related to cell-cell interactions and cell motility, which were also detected on protein level. Conclusions. Coupling metagenomics to metaproteomics enabled us to identify microbial keystone taxa in a high-CO2 aquifer, and to reveal microbial dynamics of Gracilibacteria. We posit that Gracilibacteria might be successful microbial predators in this ecosystem, potentially aiding in population control of this highly perturbed microbial geyser community from the deep biosphere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571395">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571395" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571395">
        <p class="paperTitle">Seasonal and environmental factors contribute to the variation in the gut microbiome: a large-scale study of a small bird</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571395" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571395" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liukkonen, M.; Muriel, J.; Martinez-Padilla, J.; Nord, A.; Pakanen, V.-M.; Rosivall, B.; Tilgar, V.; van Oers, K.; Grond, K.; Ruuskanen, S.</p>
        <p class="info">Score: 7.6, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571395' target='https://doi.org/10.1101/2023.12.12.571395'> 10.1101/2023.12.12.571395</a></p>
        <p class="abstract">Environmental variation can shape the gut microbiome, but majority of studies use captive-bred species, while data on large-scale variation in the gut microbiome and the associated environmental factors is lacking. Furthermore, previous studies have limited taxonomical coverage, and for example knowledge about avian gut microbiomes is still scarce. We investigated large-scale environmental variation in the gut microbiome of wild adult great tits across the species European distribution range. Our results show that gut microbiome diversity is higher during winter and that there are compositional differences between winter and summer gut microbiomes. During winter, individuals inhabiting mixed forest habitat show higher gut microbiome diversity, whereas there was no similar association during summer. Also, temperature was found to be a small contributor to compositional differences in the gut microbiome. We did not find significant differences in the gut microbiome among populations, nor any association between latitude, rainfall, and the gut microbiome. The results suggest that there is a seasonal change in wild avian gut microbiomes, but that there are still many unknown factors that shape the gut microbiome of wild bird populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571921">
        <p class="paperTitle">Thermal tolerance and survival are modulated by a natural gradient of infection in differentially acclimated hosts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Bonville, J.; Cote, A.; Binning, S. A.</p>
        <p class="info">Score: 7.5, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571921' target='https://doi.org/10.1101/2023.12.15.571921'> 10.1101/2023.12.15.571921</a></p>
        <p class="abstract">Wild ectotherms are exposed to multiple stressors, including parasites, which can affect their responses to environmental change. Simultaneously, unprecedented warm temperatures are being recorded worldwide, increasing both the average and maximum temperatures experienced in nature. Understanding how ectotherms, such as fishes, will react to the combined stress of parasites and higher average temperatures can help predict the impact of extreme events such as heat waves on populations. The critical thermal method (CTM), which assesses upper (CTmax) and lower (CTmin) thermal tolerance, is often used in acclimated ectotherms to help predict their tolerance to various temperature scenarios. Yet, few studies have characterized the response of naturally infected fish to extreme temperature events or how acute thermal stress affects subsequent survival. We acclimated naturally infected pumpkinseed sunfish (Lepomis gibbosus), to four ecologically relevant temperatures (10, 15, 20 and 25{degrees}C) and one future warming scenario (30{degrees}C) for three weeks, before measuring CTmax and CTmin. We also assessed individual survival the week following CTmax. Interestingly, trematode parasites causing black spot disease were negatively related to CTmax, suggesting that heavily infected fish are less tolerant to acute warming. Moreover, fish infected with yellow grub parasites showed decreased survival in the days following CTmax implying that the infection load has negative survival consequences on sunfish during extreme warming events. Our findings indicate that parasite infection and high prolonged average temperatures can affect fish thermal tolerance and survival, emphasizing the need to better understand the concomitant effects of stressors on health outcomes in wild populations.

Summary statementThis study shows that parasites influence thermal tolerance and survival of fish, suggesting that such stressors are important to increase the ecological relevance of thermal tolerance studies of wild animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571176">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571176" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571176">
        <p class="paperTitle">Environmental DNA captures the internal structure of a pond metacommunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571176" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571176" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cai, W.; Pichler, M.; Biggs, J.; Nicolet, P.; Ewald, N.; Griffiths, R. A.; Bush, A.; Leibold, M. A.; Hartig, F.; Yu, D. W.</p>
        <p class="info">Score: 3.7, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571176' target='https://doi.org/10.1101/2023.12.12.571176'> 10.1101/2023.12.12.571176</a></p>
        <p class="abstract">Metacommunity theory unites community and spatial ecology. Recent innovations in sequencing technology may now allow us, for the first time, to confront this theory with entire trophic community data at large scales. We metabarcoded vertebrate eDNA from 320 ponds in England. Using this  novel community data, we calculated the  internal structure of this metacommunity, which is the contributions of environmental, biotic, and spatial processes, as well as stochasticity, to the distribution of each species and the composition of each site. Our results suggest that community assembly processes in this pondscape do not conform to a single metacommunity paradigm, but rather vary among species and over space. Combining this variation with external knowledge of species biologies allows us to better understand the differential influences of environmental filtering, species interactions, and dispersal limitation on several of the vertebrate species, paving the way for the routine empirical study of metacommunities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572137">
        <p class="paperTitle">Increasing marine trophic web knowledge through DNA analyses of fish stomach contents: a step towards an Ecosystem Based Approach to fisheries research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Canals, O.; Lanzen, A.; Mendibil, I.; Bachiller, E.; Corrales, X.; Andonegi, E.; Cotano, U.; Rodriguez-Ezpeleta, N.</p>
        <p class="info">Score: 2.9, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572137' target='https://doi.org/10.1101/2023.12.18.572137'> 10.1101/2023.12.18.572137</a></p>
        <p class="abstract">Multispecies and ecosystem models, which are key for the implementation of ecosystem-based approaches to fisheries management, require extensive data on the trophic interactions between marine organisms, including changes over time. DNA metabarcoding (the simultaneous identification of all taxa in a sample by sequencing a short DNA region) could be used for speeding up large-scaled stomach content data collection. Yet, for DNA metabarcoding to be routinely implemented, technical challenges should be addressed, such as the potentially complicated sampling logistics, the detection of a high proportion of predator DNA, and the inability to provide reliable abundance estimations. Here, we present a DNA metabarcoding assay developed to examine the diet of five commercially important fish, which is feasible to be incorporated into routinary samplings. The method is devised to speed up the analysis process by avoiding the stomach dissection and content extraction steps, while preventing the amplification of predator DNA by using blocking primers. Tested in mock samples and in real stomachs, the method has proven effective and shows great effectiveness discerning diet variations due to predator ecology or prey availability. Additionally, by applying our protocol to mackerel stomachs previously analysed by visual inspection, we showcase how DNA metabarcoding could complement visually based data by detecting overlooked prey by the visual approach. We finally discuss how DNA metabarcoding-based data can contribute to trophic data collection. Our work reinforces the potential of DNA metabarcoding for the study and monitoring of fish trophic interactions and provides a basis for its incorporation into routine monitoring programs, which will be critical for the implementation of ecosystem-based approaches to fisheries management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571786">
        <p class="paperTitle">Impact of climate change and oligotrophication on quality and quantity of lake primary production: A case study in Lake Biwa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kazama, T.; Hayakawa, K.; Nagata, T.; Shimotori, K.; Imai, A.</p>
        <p class="info">Score: 2.9, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571786' target='https://doi.org/10.1101/2023.12.15.571786'> 10.1101/2023.12.15.571786</a></p>
        <p class="abstract">Global climate change and anthropogenic oligotrophication are expected to reshape the dynamics of primary production (PP) in aquatic ecosystems; however, few studies have explored their long-term effects. In theory, the PP of phytoplankton in Lake Biwa may decline over decades due to warming, heightened stratification, and anthropogenic oligotrophication. Furthermore, the PP of large phytoplankton, which are inedible to zooplankton, along with biomass-specific productivity (PBc), could decrease. In this study, data from 1976-2021 and active fluorometry measurements taken in 2020 and 2021 were evaluated. Quantitatively, the temporal dynamics of mean seasonal PP during 1971-2021 were assessed according to the carbon fixation rate to investigate relationships among environmental factors. Qualitatively, phytoplankton biomass, PP, and PBc were measured in two size fractions [edible (S) or inedible (L) for zooplankton] in 2020 and 2021, and the L:S balance for these three measures was compared between 1992 (low-temperature/high-nutrient conditions) and 2020-2021 (high-temperature/low-nutrient conditions) to assess seasonal dynamics. The results indicated that climate change and anthropogenic oligotrophication over the past 50 years have diminished Lake Biwas PP since the 1990s, impacting the phenology of PP dynamics. However, the L:S balance in PP and PBc has exhibited minimal change since 1992. These findings suggest that, although climate change and oligotrophication may reduce overall PP, they do not markedly alter the inedible/edible phytoplankton balance in terms of PP and PBc. Instead, as total PP declines, the production of small edible phytoplankton decreases proportionally, potentially affecting trophic transfer efficiency and material cycling in Lake Biwa.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
