<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Statistical prediction of microbial metabolic traits from genomes
Authors: Li, Z.; Selim, A.; Kuehn, S.
Score: 32.8, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547261
The metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="Statistical prediction of microbial metabolic traits from genomes
Authors: Li, Z.; Selim, A.; Kuehn, S.
Score: 32.8, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547261
The metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="Statistical prediction of microbial metabolic traits from genomes
Authors: Li, Z.; Selim, A.; Kuehn, S.
Score: 32.8, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547261
The metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "Statistical prediction of microbial metabolic traits from genomes\nAuthors: Li, Z.; Selim, A.; Kuehn, S.\nScore: 32.8, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547261\nThe metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source.",
  "keywords": [
    
  ],
  "articleBody": " Statistical prediction of microbial metabolic traits from genomes\nAuthors: Li, Z.; Selim, A.; Kuehn, S.\nScore: 32.8, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547261\nThe metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source. Using these data as well as two existing datasets, we show that statistical approaches can accurately predict bacterial carbon utilization traits from genomes. First, we show that classifiers trained on gene content can accurately predict bacterial carbon utilization phenotypes by encoding phylogenetic information. These models substantially outperform predictions made by constraint-based metabolic models automatically constructed from genomes. However, phylogeny-based predictions fail to predict traits for taxa that are phyloge-netically distant from any strains in the training set. To overcome this we train improved models on gene presence/absence to predict carbon utilization traits from gene content. We show that models that predict carbon utilization traits from gene presence/absence can generalize to taxa that are phylogenetically distant from the training set either by exploiting biochemical information for feature selection or by having sufficiently large datasets. In the latter case, we provide evidence that a statistical approach can identify putatively mechanistic genes involved in metabolic traits. Our study demonstrates the potential power for predicting microbial phenotypes from genotypes using statistical approaches.\nRapid species discovery and identification with real-time barcoding facilitated by ONTbarcoder 2.0 and Oxford Nanopore R10.4\nAuthors: Srivathsan, A.; Feng, V.; Ramos, D.; Meier, R.\nScore: 65.7, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546538\nMost animal species are undescribed and hidden in biodiversity samples consisting of thousands of specimens that are difficult to sort to species using morphological characters. Sorting with DNA barcodes is an attractive alternative, but needs cost-effective techniques that are suitable for use in many laboratories around the world so that barcoding can be made widely available. Oxford Nanopores portable and inexpensive MinION sequencers are a good option because they allow for reducing the sequencing and capital-cost of a fully functional barcoding laboratory. We here provide an update to the MinION barcoding software ONTbarcoder that is available for commonly used operating systems and includes a GUI. We show that the read quality obtained with ONTs latest basecallers and chemistry (R10.4 flowcells, Q20+ kits) allow for three key improvements. Firstly, ONTbarcoder can now deliver real-time barcoding to complement ONTs real-time sequencing. This means that the first barcodes are obtained within minutes of starting a sequencing run and a significant proportion of barcodes are available before sequencing completes. This real-time feature of ONTbarcoder can be configured to handle both Mk1B and Mk1C. The input is a demultiplexing sheet and base-called or raw data. Secondly, the improved read quality of ONTs latest flow cells (R10.4) allows for the use of primers with shorter indices than what was previously needed (9bp vs. 12-13bp). This decreases primer cost and improves PCR success rates. Thirdly, we demonstrate that the availability of R10.4 chemistry in the low-cost Flongle flowcell is an attractive option for users who require only 200-250 barcodes at a time.\nMegafauna diversity and functional declines in Europe from the Last Interglacial (LIG) to the present\nAuthors: Davoli, M.; Monsarrat, S.; Pedersen, R.; Scussolini, P.; Karger, D. N.; Normand, S.; Svenning, J.-C.\nScore: 23.1, Published: 2023-06-13 DOI: 10.1101/2023.06.12.544580\nAim: to estimate European megafauna diversity and functional declines in the present compared to the nearest in time modern-analogue climate period prior to the worldwide diffusion of Homo sapiens. Location: Europe. Time period: Last Interglacial (LIG; ca. 127,000 years ago) to present. Major taxa studied: wild, large ([\u0026ge;]10 kg) terrestrial mammals. Methods: we assessed the distribution of 48 European megafauna species during the LIG using hindcasting modelling and fossil records. Then, we estimated the decline in megafauna community diversity and potential trait-based functional effects from the LIG to the present, accounting for climate differences between the two periods. Results: from the LIG to the present, species richness and community biomass across Europe dropped by 74.3% ({+/-} 9.9% SD) and 96.7% ({+/-} 4.1% SD), respectively. Functional diversity dropped by 59.1% ({+/-} 11.8% SD) for herbivores and by 48.2% ({+/-} 25.0% SD) for carnivores, while trait-informed potential vegetation and meat consumptions dropped by 91.1% ({+/-} 7.4% SD) and 61.2% ({+/-} 17.2% SD), respectively. The loss in megafauna diversity and associated ecological processes were high everywhere, but particularly in western Europe for carnivores and in the East European Plain for herbivores. We found that potential megafauna richness and functional patterns in the two periods were near identical if only climate-driven differences were considered. Main conclusions: severe, size-biased defaunation has degraded megafauna assemblages and megafauna-mediated ecological processes across Europe from the LIG to the present. These patterns cannot be explained by climate differences between the two periods, thus were likely driven by the impact of prehistoric Homo sapiens. The results suggest that the structure of wild ecosystems of the present strongly deviates from the evolutionary norm, notably with decreased functional heterogeneity and decreased fluxes of biogeochemical compounds across the trophic networks, highlighting the importance of ambitious policies of megafauna community restoration to support ecosystems functioning.\nAbsence of Coronavirus in terns in the Western Indian Ocean?\nAuthors: Lebarbenchon, C.; Feare, C.; Larose, C.; Boucher, S.; Jaeger, A.; Le Corre, M.\nScore: 14.6, Published: 2023-07-03 DOI: 10.1101/2023.06.30.547193\nWe investigated coronavirus circulation in three tern species, in four islands of the Western Indian Ocean (Bird, Reunion, Europa, Juan de Nova). None of the 1513 samples tested positive by RT-PCR. We discuss the implication in term of host species range, ecological drivers of virus transmission, and diagnostic tools.\nGIFT an R package to access the Global Inventory of Floras and Traits\nAuthors: Denelle, P.; Weigelt, P.; Kreft, H.\nScore: 179.1, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704\nO_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale. C_LIO_LIThe Global Inventory of Floras and Traits (GIFT) is a global database of regional plant checklists that has proven successful in documenting biogeographical and geographical patterns of plants. Since the release of the first version of GIFT, the database kept on expanding. We introduce GIFT version 3.0, which contains 5,169 checklists referring to 3,400 regions. These checklists include a total of 371,148 land plant species, mostly vascular plants, of which 354,848 are accepted species names, and 109 functional traits. This new version uses new resources for taxonomic name standardization, is matched to a new plant phylogeny, comes with a new trait aggregation workflow, and includes additional environmental variables. C_LIO_LIWe also present the GIFT R-package, which contains all necessary functions to retrieve distributional, functional, phylogenetic, and environmental data from the GIFT database. The package comes with a dedicated website, https://biogeomacro.github.io/GIFT/, which includes three rich vignettes to guide users in retrieving data from GIFT. C_LIO_LIThe recent development of GIFT and its associated R-package provide ecologists with access to one of the largest plant databases. This will foster research into regional to global patterns of plant diversity and their underlying mechanisms. Proper versioning of the database and the ability to retrieve and cite data from any previous and current instance of the GIFT database will ensure the reproducibility of studies that utilize it. C_LI\nTreated eave screens in combination with screened doors and windows, are more effective than untreated eave screens in a similar combination in reducing indoor and outdoor Anopheles populations under semi-filed conditions in western Kenya\nAuthors: Abong'o, B.; Agumba, S.; Moshi, V.; Simwero, J.; Otima, J.; Ochomo, E.\nScore: 8.6, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547882\nIntroduction: Human habitats remain the main point of human-vector interaction leading to malaria transmission despite sustained use of insecticide treated nets (ITNs) and indoor residual spraying (IRS). Simple structural modifications involving screening of doors, windows and eaves have great potential for reducing indoor entry of mosquitoes and hence malaria transmission. Methods: Four huts, each constructed inside a semi-field structure, allowing the experimental release of mosquitoes at the Kenya Medical Research Institute (KEMRI), Kisumu research station were used in the study. Two huts had untreated eave and door screens and screened air cavities in place of windows in experiment 1 with the eave screen treated using Actellic(R) insecticide in experiment 2. The other two huts remained unscreened throughout the study. First filial (F1) generation of Anopheles funestus from Siaya, F0 reared from An. arabiensis larvae collected from Ahero and An. arabiensis Dongola strain from the insectary were raised to 3-day old adults and used in experiments. Two hundred, 3-day old adults of each species were released in each semi-field structure at dusk and recaptured the following day at 0700hrs and at 0900 hours. A single volunteer slept in each hut under untreated bed net each night of the study. Recaptured mosquitoes were counted and recorded by collection location, either indoor or outdoor of each hut in the different semi-field structures. Results: Significantly fewer An. arabiensis from Ahero [RR=0.10; (95%CI: 0.02-0.63); P\u003c0.0145], An. arabiensis Dongola strain [RR=0.11; (95%CI: 0.04 - 0.19); P\u003c0.0001 and An. funestus from Siaya [RR=0.10; (95%CI: 0.06-0.17); P\u003c0.0001] were observed inside modified huts compared to unmodified ones. Treating of eave screen material with Actellic(R) 300CS significantly reduced the numbers An. arabiensis from Ahero [RR=0.003; (95%CI: 0.00-0.03); P\u003c0.0001] and An. arabiensis Dongola strain [RR=0.03; (95%CI: 0.02-0.05); P\u003c0.0001] indoors of huts with treated eave screen compared to huts with untreated eave screens, while totally preventing entry of An. funestus indoors. These modifications cost \u003c250usd/structure. Discussion and Conclusion: This article describes affordable and effective ways of reducing mosquito entry into the house by modifying the eaves, doors and windows. These modifications were highly effective in reducing indoor entry of mosquitoes. Additionally, treatment of eave screen material with an effective insecticide further reduces the Anopheles population in and around the screened huts under semi-field conditions and could greatly complement existing vector control efforts.\nCompetition for light can drive adverse species-composition shifts in the Amazon Forest under elevated CO2\nAuthors: Joshi, J.; Hofhansl, F.; Singh, S.; Stocker, B.; Brännström, A.; Franklin, O.; Blanco, C. C.; Aleixo, I. F.; Lapola, D. M.; Prentice, I. C.; Dieckmann, U.\nScore: 7.4, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547575\nThe resilience of biodiverse forests to climate change depends on an interplay of adaptive processes operating at multiple temporal and organizational scales. These include short-term acclimation of physiological processes like photosynthesis and respiration, mid-term changes in forest structure due to competition, and long-term changes in community composition arising from competitive exclusion and genetic trait evolution. To investigate the roles of diversity and adaptation for forest resilience, we present Plant-FATE, a parsimonious eco-evolutionary vegetation model. Tested with data from a hyperdiverse Amazonian terra-firme forest, our model accurately predicts multiple emergent ecosystem properties characterizing forest structure and function. Under elevated CO2 conditions, we predict an increase in productivity, leaf area, and aboveground biomass, with the magnitude of this increase declining in nutrient-deprived soils if trees allocate more carbon to the rhizosphere to overcome nutrient limitation. Furthermore, increased aboveground productivity leads to greater competition for light and drives a shift in community composition towards fast-growing but short-lived species characterized by lower wood densities. Such a transition reduces the carbon residence time of woody biomass, dampening carbon-sink strength and potentially rendering the Amazon Forest more vulnerable to future climatic extreme events.\nNew record of Dendronephthya hemprichi (Family: Nephtheidae) fromMediterranean, Israel- an evidence for tropicalization?\nAuthors: Nativ, H.; Galili, O.; Almuly, R.; Einbinder, S.; Tchernov, D.; Mass, T.\nScore: 5.8, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547739\nBioinvasions have the potential to provoke cascade effects that can disrupt natural ecosystems, and cause ecological regime shifts. The Mediterranean Sea is particularly prone to bioinvasions as the changing water conditions, evoked by climate change, are creating advantageous conditions for Lessepsian migrants from the Red Sea. Recently, in May 2023, a new alien species was documented in the Mediterranean Sea - a soft coral of the genus Dendronephthya. This discovery was made by divers conducting Long Term Ecological Research surveys, along the coast of Israel, at a depth of 42 m. Genetic and morphological testing were utilized to confirm the species identity as Dendronepthya hemprichi, an Indo-Pacific coral, common in the Red Sea. According to life history traits of this species such as accelerated attachment to available surfaces and fast growth, we expect it to rapidly expand its distribution and abundance across the Mediterranean.\nHierarchical Classification of Insects with Multitask Learning and Anomaly Detection\nAuthors: Bjerge, K.; Geissmann, Q.; Alison, J.; Mann, H. M. R.; Hoye, T. T.; Dyrmann, M.; Karstoft, H.\nScore: 6.5, Published: 2023-06-30 DOI: 10.1101/2023.06.29.546989\nCameras and computer vision are revolutionising the study of insects, creating new research opportunities within agriculture, epidemiology, evolution, ecology and monitoring of biodiversity. However, a major challenge is the diversity of insects and close resemblances of many species combined with computer vision are often not sufficient to classify large numbers of insect species, which sometimes cannot be identified at the species level. Here, we present an algorithm to hierarchically classify insects from images, leveraging a simple taxonomy to (1) classify specimens across multiple taxonomic ranks simultaneously, and (2) highlight the lowest rank at which a reliable classification can be reached. Specifically, we propose multitask learning, a loss function incorporating class dependency at each taxonomic rank, and anomaly detection based on outlier analysis for quantification of uncertainty. First, we compile a dataset of 41,731 images of insects, combining images from time-lapse monitoring of floral scenes with images from the Global Biodiversity Information Facility (GBIF). Second, we adapt state-of-the-art convolutional neural networks, ResNet and EfficientNet, for the hierarchical classification of insects belonging to three orders, five families and nine species. Third, we assess model generalization for 11 species unseen by the trained models. Here, anomaly detection is used to predict the higher rank of the species not present in the training set. We found that incorporating a simple taxonomy into our model increased accuracy at higher taxonomic ranks. As expected, our algorithm correctly classified new insect species at higher taxonomic ranks, while classification was uncertain at lower taxonomic ranks. Anomaly detection can effectively flag novel taxa that are visually distinct from species in the training data. However, five novel taxa were consistently mistaken for visually similar species in the training data. Above all, we have demonstrated a practical approach to hierarchical classification based on species taxonomy and uncertainty during automated in situ monitoring of live insects. Our method is simple and versatile and could be implemented to classify a wide range of insects as well as other organisms.\nPiroplasm parasites (Apicomplexa: Piroplasmida) in northeastern populations of the invasive Asian longhorned tick, Haemaphysalis longicornis Neumann (Ixodida: Ixodidae), in the United States\nAuthors: Herb, H. E.; Ferreira, F. C.; Gonzalez, J.; Fonseca, D. M.\nScore: 6.6, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546593\nPiroplasms, which include the agents of cattle fever and human and dog babesiosis, are a diverse group of blood parasites of significant veterinary and medical importance. The invasive Asian longhorned tick, Haemaphysalis longicornis, is a known vector of piroplasms in its native range in east Asia and invasive range in Australasia. In the US state of Virginia, H. longicornis has been associated with Theileria orientalis Ikeda outbreaks that caused cattle mortality. We examined 667 H. longicornis collected in 2021 from three sites in New Brunswick, New Jersey, the US state where established populations of this species were first detected in 2017. We used primers targeting the 18S small subunit rRNA and the cytochrome b oxidase loci and unveiled the presence of DNA from an unidentified Theileria species (in 1 nymph) and Theileria cervi type F (1 adult, 5 nymphs). In addition, we sequenced a 130 bp fragment of the cytochrome oxidase b locus from Odocoileus virginianus, the white-tailed deer, in a partially engorged questing H. longicornis, supporting the association of this tick species with deer. We also detected DNA from an undescribed Babesia sensu stricto ( true Babesia, 2 adults, 2 nymphs) as well as Babesia sp. Coco (1 adult, 1 nymph). Finally, we detected DNA from Babesia microti S837 (1 adult, 4 nymphs). Babesia microti S837 has been sequenced from striped skunks, Mephitis mephitis, and is closely related to the human pathogen B. microti US-type. The five parasites we are associating with H. longicornis represent a diverse assemblage spanning three clades in the piroplasm phylogeny, two undescribed, raising concerns of transmission amplification of veterinary pathogens as well as spillover of pathogens from wildlife to humans.\n",
  "wordCount" : "2782",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547261">
        <p class="paperTitle">Statistical prediction of microbial metabolic traits from genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Z.; Selim, A.; Kuehn, S.</p>
        <p class="info">Score: 32.8, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547261' target='https://doi.org/10.1101/2023.06.30.547261'> 10.1101/2023.06.30.547261</a></p>
        <p class="abstract">The metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source. Using these data as well as two existing datasets, we show that statistical approaches can accurately predict bacterial carbon utilization traits from genomes. First, we show that classifiers trained on gene content can accurately predict bacterial carbon utilization phenotypes by encoding phylogenetic information. These models substantially outperform predictions made by constraint-based metabolic models automatically constructed from genomes. However, phylogeny-based predictions fail to predict traits for taxa that are phyloge-netically distant from any strains in the training set. To overcome this we train improved models on gene presence/absence to predict carbon utilization traits from gene content. We show that models that predict carbon utilization traits from gene presence/absence can generalize to taxa that are phylogenetically distant from the training set either by exploiting biochemical information for feature selection or by having sufficiently large datasets. In the latter case, we provide evidence that a statistical approach can identify putatively mechanistic genes involved in metabolic traits. Our study demonstrates the potential power for predicting microbial phenotypes from genotypes using statistical approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546538">
        <p class="paperTitle">Rapid species discovery and identification with real-time barcoding facilitated by ONTbarcoder 2.0 and Oxford Nanopore R10.4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivathsan, A.; Feng, V.; Ramos, D.; Meier, R.</p>
        <p class="info">Score: 65.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546538' target='https://doi.org/10.1101/2023.06.26.546538'> 10.1101/2023.06.26.546538</a></p>
        <p class="abstract">Most animal species are undescribed and hidden in biodiversity samples consisting of thousands of specimens that are difficult to sort to species using morphological characters. Sorting with DNA barcodes is an attractive alternative, but needs cost-effective techniques that are suitable for use in many laboratories around the world so that barcoding can be made widely available. Oxford Nanopores portable and inexpensive MinION sequencers are a good option because they allow for reducing the sequencing and capital-cost of a fully functional barcoding laboratory. We here provide an update to the MinION barcoding software ONTbarcoder that is available for commonly used operating systems and includes a GUI. We show that the read quality obtained with ONTs latest basecallers and chemistry (R10.4 flowcells, Q20&#43; kits) allow for three key improvements. Firstly, ONTbarcoder can now deliver real-time barcoding to complement ONTs real-time sequencing. This means that the first barcodes are obtained within minutes of starting a sequencing run and a significant proportion of barcodes are available before sequencing completes. This real-time feature of ONTbarcoder can be configured to handle both Mk1B and Mk1C. The input is a demultiplexing sheet and base-called or raw data. Secondly, the improved read quality of ONTs latest flow cells (R10.4) allows for the use of primers with shorter indices than what was previously needed (9bp vs. 12-13bp). This decreases primer cost and improves PCR success rates. Thirdly, we demonstrate that the availability of R10.4 chemistry in the low-cost Flongle flowcell is an attractive option for users who require only 200-250 barcodes at a time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544580">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544580" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544580">
        <p class="paperTitle">Megafauna diversity and functional declines in Europe from the Last Interglacial (LIG) to the present</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544580" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544580" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davoli, M.; Monsarrat, S.; Pedersen, R.; Scussolini, P.; Karger, D. N.; Normand, S.; Svenning, J.-C.</p>
        <p class="info">Score: 23.1, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544580' target='https://doi.org/10.1101/2023.06.12.544580'> 10.1101/2023.06.12.544580</a></p>
        <p class="abstract">Aim: to estimate European megafauna diversity and functional declines in the present compared to the nearest in time modern-analogue climate period prior to the worldwide diffusion of Homo sapiens.

Location: Europe.

Time period: Last Interglacial (LIG; ca. 127,000 years ago) to present.

Major taxa studied: wild, large ([&amp;ge;]10 kg) terrestrial mammals.

Methods: we assessed the distribution of 48 European megafauna species during the LIG using hindcasting modelling and fossil records. Then, we estimated the decline in megafauna community diversity and potential trait-based functional effects from the LIG to the present, accounting for climate differences between the two periods.

Results: from the LIG to the present, species richness and community biomass across Europe dropped by 74.3% ({&#43;/-} 9.9% SD) and 96.7% ({&#43;/-} 4.1% SD), respectively. Functional diversity dropped by 59.1% ({&#43;/-} 11.8% SD) for herbivores and by 48.2% ({&#43;/-} 25.0% SD) for carnivores, while trait-informed potential vegetation and meat consumptions dropped by 91.1% ({&#43;/-} 7.4% SD) and 61.2% ({&#43;/-} 17.2% SD), respectively. The loss in megafauna diversity and associated ecological processes were high everywhere, but particularly in western Europe for carnivores and in the East European Plain for herbivores. We found that potential megafauna richness and functional patterns in the two periods were near identical if only climate-driven differences were considered.

Main conclusions: severe, size-biased defaunation has degraded megafauna assemblages and megafauna-mediated ecological processes across Europe from the LIG to the present. These patterns cannot be explained by climate differences between the two periods, thus were likely driven by the impact of prehistoric Homo sapiens. The results suggest that the structure of wild ecosystems of the present strongly deviates from the evolutionary norm, notably with decreased functional heterogeneity and decreased fluxes of biogeochemical compounds across the trophic networks, highlighting the importance of ambitious policies of megafauna community restoration to support ecosystems functioning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547193">
        <p class="paperTitle">Absence of Coronavirus in terns in the Western Indian Ocean?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lebarbenchon, C.; Feare, C.; Larose, C.; Boucher, S.; Jaeger, A.; Le Corre, M.</p>
        <p class="info">Score: 14.6, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547193' target='https://doi.org/10.1101/2023.06.30.547193'> 10.1101/2023.06.30.547193</a></p>
        <p class="abstract">We investigated coronavirus circulation in three tern species, in four islands of the Western Indian Ocean (Bird, Reunion, Europa, Juan de Nova). None of the 1513 samples tested positive by RT-PCR. We discuss the implication in term of host species range, ecological drivers of virus transmission, and diagnostic tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546704">
        <p class="paperTitle">GIFT an R package to access the Global Inventory of Floras and Traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Denelle, P.; Weigelt, P.; Kreft, H.</p>
        <p class="info">Score: 179.1, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546704' target='https://doi.org/10.1101/2023.06.27.546704'> 10.1101/2023.06.27.546704</a></p>
        <p class="abstract">O_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale.
C_LIO_LIThe Global Inventory of Floras and Traits (GIFT) is a global database of regional plant checklists that has proven successful in documenting biogeographical and geographical patterns of plants. Since the release of the first version of GIFT, the database kept on expanding. We introduce GIFT version 3.0, which contains 5,169 checklists referring to 3,400 regions. These checklists include a total of 371,148 land plant species, mostly vascular plants, of which 354,848 are accepted species names, and 109 functional traits. This new version uses new resources for taxonomic name standardization, is matched to a new plant phylogeny, comes with a new trait aggregation workflow, and includes additional environmental variables.
C_LIO_LIWe also present the GIFT R-package, which contains all necessary functions to retrieve distributional, functional, phylogenetic, and environmental data from the GIFT database. The package comes with a dedicated website, https://biogeomacro.github.io/GIFT/, which includes three rich vignettes to guide users in retrieving data from GIFT.
C_LIO_LIThe recent development of GIFT and its associated R-package provide ecologists with access to one of the largest plant databases. This will foster research into regional to global patterns of plant diversity and their underlying mechanisms. Proper versioning of the database and the ability to retrieve and cite data from any previous and current instance of the GIFT database will ensure the reproducibility of studies that utilize it.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547882">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547882" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547882">
        <p class="paperTitle">Treated eave screens in combination with screened doors and windows, are more effective than untreated eave screens in a similar combination in reducing indoor and outdoor Anopheles populations under semi-filed conditions in western Kenya</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547882" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547882" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abong&#39;o, B.; Agumba, S.; Moshi, V.; Simwero, J.; Otima, J.; Ochomo, E.</p>
        <p class="info">Score: 8.6, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547882' target='https://doi.org/10.1101/2023.07.05.547882'> 10.1101/2023.07.05.547882</a></p>
        <p class="abstract">Introduction: Human habitats remain the main point of human-vector interaction leading to malaria transmission despite sustained use of insecticide treated nets (ITNs) and indoor residual spraying (IRS). Simple structural modifications involving screening of doors, windows and eaves have great potential for reducing indoor entry of mosquitoes and hence malaria transmission. Methods: Four huts, each constructed inside a semi-field structure, allowing the experimental release of mosquitoes at the Kenya Medical Research Institute (KEMRI), Kisumu research station were used in the study. Two huts had untreated eave and door screens and screened air cavities in place of windows in experiment 1 with the eave screen treated using Actellic(R) insecticide in experiment 2. The other two huts remained unscreened throughout the study. First filial (F1) generation of Anopheles funestus from Siaya, F0 reared from An. arabiensis larvae collected from Ahero and An. arabiensis Dongola strain from the insectary were raised to 3-day old adults and used in experiments. Two hundred, 3-day old adults of each species were released in each semi-field structure at dusk and recaptured the following day at 0700hrs and at 0900 hours. A single volunteer slept in each hut under untreated bed net each night of the study. Recaptured mosquitoes were counted and recorded by collection location, either indoor or outdoor of each hut in the different semi-field structures. Results: Significantly fewer An. arabiensis from Ahero [RR=0.10; (95%CI: 0.02-0.63); P&lt;0.0145], An. arabiensis Dongola strain [RR=0.11; (95%CI: 0.04 - 0.19); P&lt;0.0001 and An. funestus from Siaya [RR=0.10; (95%CI: 0.06-0.17); P&lt;0.0001] were observed inside modified huts compared to unmodified ones. Treating of eave screen material with Actellic(R) 300CS significantly reduced the numbers An. arabiensis from Ahero [RR=0.003; (95%CI: 0.00-0.03); P&lt;0.0001] and An. arabiensis Dongola strain [RR=0.03; (95%CI: 0.02-0.05); P&lt;0.0001] indoors of huts with treated eave screen compared to huts with untreated eave screens, while totally preventing entry of An. funestus indoors. These modifications cost &lt;250usd/structure. Discussion and Conclusion: This article describes affordable and effective ways of reducing mosquito entry into the house by modifying the eaves, doors and windows. These modifications were highly effective in reducing indoor entry of mosquitoes. Additionally, treatment of eave screen material with an effective insecticide further reduces the Anopheles population in and around the screened huts under semi-field conditions and could greatly complement existing vector control efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547575" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547575">
        <p class="paperTitle">Competition for light can drive adverse species-composition shifts in the Amazon Forest under elevated CO2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, J.; Hofhansl, F.; Singh, S.; Stocker, B.; Brännström, A.; Franklin, O.; Blanco, C. C.; Aleixo, I. F.; Lapola, D. M.; Prentice, I. C.; Dieckmann, U.</p>
        <p class="info">Score: 7.4, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547575' target='https://doi.org/10.1101/2023.07.03.547575'> 10.1101/2023.07.03.547575</a></p>
        <p class="abstract">The resilience of biodiverse forests to climate change depends on an interplay of adaptive processes operating at multiple temporal and organizational scales. These include short-term acclimation of physiological processes like photosynthesis and respiration, mid-term changes in forest structure due to competition, and long-term changes in community composition arising from competitive exclusion and genetic trait evolution. To investigate the roles of diversity and adaptation for forest resilience, we present Plant-FATE, a parsimonious eco-evolutionary vegetation model. Tested with data from a hyperdiverse Amazonian terra-firme forest, our model accurately predicts multiple emergent ecosystem properties characterizing forest structure and function. Under elevated CO2 conditions, we predict an increase in productivity, leaf area, and aboveground biomass, with the magnitude of this increase declining in nutrient-deprived soils if trees allocate more carbon to the rhizosphere to overcome nutrient limitation. Furthermore, increased aboveground productivity leads to greater competition for light and drives a shift in community composition towards fast-growing but short-lived species characterized by lower wood densities. Such a transition reduces the carbon residence time of woody biomass, dampening carbon-sink strength and potentially rendering the Amazon Forest more vulnerable to future climatic extreme events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547739">
        <p class="paperTitle">New record of Dendronephthya hemprichi (Family: Nephtheidae) fromMediterranean, Israel- an evidence for tropicalization?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nativ, H.; Galili, O.; Almuly, R.; Einbinder, S.; Tchernov, D.; Mass, T.</p>
        <p class="info">Score: 5.8, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547739' target='https://doi.org/10.1101/2023.07.04.547739'> 10.1101/2023.07.04.547739</a></p>
        <p class="abstract">Bioinvasions have the potential to provoke cascade effects that can disrupt natural ecosystems, and cause ecological regime shifts. The Mediterranean Sea is particularly prone to bioinvasions as the changing water conditions, evoked by climate change, are creating advantageous conditions for Lessepsian migrants from the Red Sea. Recently, in May 2023, a new alien species was documented in the Mediterranean Sea - a soft coral of the genus Dendronephthya. This discovery was made by divers conducting Long Term Ecological Research surveys, along the coast of Israel, at a depth of 42 m. Genetic and morphological testing were utilized to confirm the species identity as Dendronepthya hemprichi, an Indo-Pacific coral, common in the Red Sea. According to life history traits of this species such as accelerated attachment to available surfaces and fast growth, we expect it to rapidly expand its distribution and abundance across the Mediterranean.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546989">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546989" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546989">
        <p class="paperTitle">Hierarchical Classification of Insects with Multitask Learning and Anomaly Detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546989" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546989" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bjerge, K.; Geissmann, Q.; Alison, J.; Mann, H. M. R.; Hoye, T. T.; Dyrmann, M.; Karstoft, H.</p>
        <p class="info">Score: 6.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546989' target='https://doi.org/10.1101/2023.06.29.546989'> 10.1101/2023.06.29.546989</a></p>
        <p class="abstract">Cameras and computer vision are revolutionising the study of insects, creating new research opportunities within agriculture, epidemiology, evolution, ecology and monitoring of biodiversity. However, a major challenge is the diversity of insects and close resemblances of many species combined with computer vision are often not sufficient to classify large numbers of insect species, which sometimes cannot be identified at the species level. Here, we present an algorithm to hierarchically classify insects from images, leveraging a simple taxonomy to (1) classify specimens across multiple taxonomic ranks simultaneously, and (2) highlight the lowest rank at which a reliable classification can be reached. Specifically, we propose multitask learning, a loss function incorporating class dependency at each taxonomic rank, and anomaly detection based on outlier analysis for quantification of uncertainty. First, we compile a dataset of 41,731 images of insects, combining images from time-lapse monitoring of floral scenes with images from the Global Biodiversity Information Facility (GBIF). Second, we adapt state-of-the-art convolutional neural networks, ResNet and EfficientNet, for the hierarchical classification of insects belonging to three orders, five families and nine species. Third, we assess model generalization for 11 species unseen by the trained models. Here, anomaly detection is used to predict the higher rank of the species not present in the training set. We found that incorporating a simple taxonomy into our model increased accuracy at higher taxonomic ranks. As expected, our algorithm correctly classified new insect species at higher taxonomic ranks, while classification was uncertain at lower taxonomic ranks. Anomaly detection can effectively flag novel taxa that are visually distinct from species in the training data. However, five novel taxa were consistently mistaken for visually similar species in the training data. Above all, we have demonstrated a practical approach to hierarchical classification based on species taxonomy and uncertainty during automated in situ monitoring of live insects. Our method is simple and versatile and could be implemented to classify a wide range of insects as well as other organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546593">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546593" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546593">
        <p class="paperTitle">Piroplasm parasites (Apicomplexa: Piroplasmida) in northeastern populations of the invasive Asian longhorned tick, Haemaphysalis longicornis Neumann (Ixodida: Ixodidae), in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546593" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546593" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herb, H. E.; Ferreira, F. C.; Gonzalez, J.; Fonseca, D. M.</p>
        <p class="info">Score: 6.6, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546593' target='https://doi.org/10.1101/2023.06.26.546593'> 10.1101/2023.06.26.546593</a></p>
        <p class="abstract">Piroplasms, which include the agents of cattle fever and human and dog babesiosis, are a diverse group of blood parasites of significant veterinary and medical importance. The invasive Asian longhorned tick, Haemaphysalis longicornis, is a known vector of piroplasms in its native range in east Asia and invasive range in Australasia. In the US state of Virginia, H. longicornis has been associated with Theileria orientalis Ikeda outbreaks that caused cattle mortality. We examined 667 H. longicornis collected in 2021 from three sites in New Brunswick, New Jersey, the US state where established populations of this species were first detected in 2017. We used primers targeting the 18S small subunit rRNA and the cytochrome b oxidase loci and unveiled the presence of DNA from an unidentified Theileria species (in 1 nymph) and Theileria cervi type F (1 adult, 5 nymphs). In addition, we sequenced a 130 bp fragment of the cytochrome oxidase b locus from Odocoileus virginianus, the white-tailed deer, in a partially engorged questing H. longicornis, supporting the association of this tick species with deer. We also detected DNA from an undescribed Babesia sensu stricto ( true Babesia, 2 adults, 2 nymphs) as well as Babesia sp. Coco (1 adult, 1 nymph). Finally, we detected DNA from Babesia microti S837 (1 adult, 4 nymphs). Babesia microti S837 has been sequenced from striped skunks, Mephitis mephitis, and is closely related to the human pathogen B. microti US-type. The five parasites we are associating with H. longicornis represent a diverse assemblage spanning three clades in the piroplasm phylogeny, two undescribed, raising concerns of transmission amplification of veterinary pathogens as well as spillover of pathogens from wildlife to humans.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
