<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Geographically widespread and novel hemotropic mycoplasmas and bartonellae in Mexican free-tailed bats and sympatric North American bat species
Authors: Becker, D.; Dyer, K.; Lock, L.; Pladas, S.; Sukhadia, A.; Demory, B.; Nunes Batista, J.; Pineda, M.; Simmons, N.; Adams, A.; Frick, W.; O&#39;Mara, T.; Volokhov, D.
Score: 13.5, Published: 2024-02-09 DOI: 10.1101/2024.02.08.577874
Bacterial pathogens remain poorly characterized in bats, especially in North America. We describe novel (and in some cases panmictic) hemoplasmas (12.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="Geographically widespread and novel hemotropic mycoplasmas and bartonellae in Mexican free-tailed bats and sympatric North American bat species
Authors: Becker, D.; Dyer, K.; Lock, L.; Pladas, S.; Sukhadia, A.; Demory, B.; Nunes Batista, J.; Pineda, M.; Simmons, N.; Adams, A.; Frick, W.; O&#39;Mara, T.; Volokhov, D.
Score: 13.5, Published: 2024-02-09 DOI: 10.1101/2024.02.08.577874
Bacterial pathogens remain poorly characterized in bats, especially in North America. We describe novel (and in some cases panmictic) hemoplasmas (12." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="Geographically widespread and novel hemotropic mycoplasmas and bartonellae in Mexican free-tailed bats and sympatric North American bat species
Authors: Becker, D.; Dyer, K.; Lock, L.; Pladas, S.; Sukhadia, A.; Demory, B.; Nunes Batista, J.; Pineda, M.; Simmons, N.; Adams, A.; Frick, W.; O&#39;Mara, T.; Volokhov, D.
Score: 13.5, Published: 2024-02-09 DOI: 10.1101/2024.02.08.577874
Bacterial pathogens remain poorly characterized in bats, especially in North America. We describe novel (and in some cases panmictic) hemoplasmas (12."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "Geographically widespread and novel hemotropic mycoplasmas and bartonellae in Mexican free-tailed bats and sympatric North American bat species\nAuthors: Becker, D.; Dyer, K.; Lock, L.; Pladas, S.; Sukhadia, A.; Demory, B.; Nunes Batista, J.; Pineda, M.; Simmons, N.; Adams, A.; Frick, W.; O\u0026#39;Mara, T.; Volokhov, D.\nScore: 13.5, Published: 2024-02-09 DOI: 10.1101/2024.02.08.577874\nBacterial pathogens remain poorly characterized in bats, especially in North America. We describe novel (and in some cases panmictic) hemoplasmas (12.",
  "keywords": [
    
  ],
  "articleBody": " Geographically widespread and novel hemotropic mycoplasmas and bartonellae in Mexican free-tailed bats and sympatric North American bat species\nAuthors: Becker, D.; Dyer, K.; Lock, L.; Pladas, S.; Sukhadia, A.; Demory, B.; Nunes Batista, J.; Pineda, M.; Simmons, N.; Adams, A.; Frick, W.; O'Mara, T.; Volokhov, D.\nScore: 13.5, Published: 2024-02-09 DOI: 10.1101/2024.02.08.577874\nBacterial pathogens remain poorly characterized in bats, especially in North America. We describe novel (and in some cases panmictic) hemoplasmas (12.9% positivity) and bartonellae (16.7% positivity) across three colonies of Mexican free-tailed bats (Tadarida brasiliensis), a partially migratory species that can seasonally travel hundreds of kilometers. Molecular analyses identified three novel Candidatus hemoplasma species most similar to another novel Candidatus species in Neotropical molossid bats. We also detected novel hemoplasmas in sympatric cave myotis (Myotis velifer) and pallid bats (Antrozous pallidus), with sequences in the latter 96.5% related to C. Mycoplasma haemohominis. We identified eight Bartonella genotypes, including those in cave myotis, with 96.7% similarity to C. Bartonella mayotimonensis. We also detected Bartonella rochalimae in migratory Tadarida brasiliensis, representing the first report of this human pathogen in bats. The seasonality and diversity of these bacteria observed here suggest that additional longitudinal, genomic, and immunological studies in bats are warranted.\nDoes method of field preparation affect survival and growth of juvenile Chinook salmon in agricultural floodplains?\nAuthors: Tallman, R.; Wampler, A.; Singer, G.; Jeffres, C.; Cocherell, D.; Colby, J.; Fangue, N.; Lusardi, R.; Rypel, A.\nScore: 11.3, Published: 2024-02-06 DOI: 10.1101/2024.02.02.578246\nThere is growing interest to integrate conservation initiatives into agricultural practices using a reconciliation ecology framework. In Californias Central Valley, one approach to improving crucial nursery habitat for threatened and endangered fish species is by re-creating floodplain habitats through the inundation of agricultural fields during the non-growing season. We conducted a series of field experiments in agricultural floodplains (winter-flooded rice fields that were once natural floodplains) to explore whether different field preparation methods enhanced growth and survival of floodplain-dependent fish species. Approximately 8,000 juvenile fall-run Chinook Salmon (Oncorhynchus tshawytscha) were reared for at least 28 days on eight, 0.2 ha experimental fields. Each experimental field represented one of four treatments (two replicates per treatment): addition of large wood, addition of in-field canals, addition of both large wood and in-field canals, and control-where no additional field preparations were implemented beyond standard post-growing practices: i.e., chopping rice into stubble, baling and removing excess rice straw, and discing the field with a single pass by plowing rice straw into the ground to promote decomposition. We found no significant difference in water temperature, fish growth or fish survival among habitat treatments. Across treatments, survival ranged from 50.1%-78.3% and averaged 65.75% (/- 7.89% SE). These findings suggest agricultural floodplains require no additional modifications to promote fish growth and survival. Results illustrate the benefits of integrating working landscapes with conservation initiatives by creating more accessible and beneficial habitat for native fishes.\nGlobal niche partitioning of purine and pyrimidine cross-feeding among ocean microbes\nAuthors: Braakman, R.; Satinsky, B.; O'Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.\nScore: 10.2, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579562\nCharacterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups. The cyanobacterium Prochlorococcus exudes both compound classes, which metabolic reconstructions suggest follows synchronous daily genome replication. Co-occurring heterotrophs differentiate into purine or pyrimidine specialists, or generalists, and use compounds for different purposes. The most abundant heterotroph, SAR11, is a specialist that uses purines as sources of energy, carbon and/or nitrogen, with subgroups differentiating along ocean-scale gradients in the supply of energy and nitrogen, in turn leading to putative cryptic nitrogen cycles that link many microbes. Finally, in a SAR11 subgroup that dominates where Prochlorococcus is abundant, adenine additions to cultures inhibit DNA synthesis, poising cells for replication. We argue this subgroup uses inferred daily pulses of adenine from Prochlorococcus to metabolically synchronize to the daily supply of photosynthate from surrounding phytoplankton.\nWhat unexplored means: Mapping undersampled regions in natural history collections\nAuthors: Chakrabarty, P.; Ball, L.; Bedoya, A.; Rodriguez Machado, S.; Paredes-Burneo, D.; Rutledge, S.; Boyd, D.; Vander Pluym, D.; Babb-Biernacki, S.; Chipps, A.; Ozturk, R.; Terzi, Y.\nScore: 9.0, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579602\nWe examined global records of accessible natural history voucher collections (with publicly available data) for terrestrial and freshwater vascular plants, fungi, freshwater fishes, birds, mammals, and herpetofauna (amphibians and reptiles) and highlight areas of the world that would be considered undersampled and sometimes called \"unexplored\" (i.e., have relatively low, or no evidence of, past sampling efforts) under typical Western-scientific descriptions. We also question what \"unexplored\" may actually mean in these contexts and explain how retiring the term in favor of more nuanced phrasing can mitigate future misunderstandings of natural history science.\nModelling the spatiotemporal dynamics of multispecies population interactions in the context of climate change\nAuthors: Christou, K.; Schmucki, R.; Audusseau, H.\nScore: 4.3, Published: 2024-02-09 DOI: 10.1101/2024.02.07.579316\nO_LISpecies interactions are fundamental to the stability and productivity of ecosystems. To improve our capacity to predict and understand how climate change shapes the distribution of species and the dynamics of biotic interactions, we need to develop spatially explicit multi-species models that are built upon species-specific responses to changing conditions. C_LIO_LIWe developed a two-dimensional diffusion-advection-reaction predator-prey model that integrates species-specific responses to heterogeneous landscapes, affecting species dispersal, reproduction and survival rates. We derived conditions for the stability and feasibility of the coexistence steady-state and observed how temperature variation stabilises or destabilises the system. C_LIO_LIWe conducted numerical simulations to explore the effect of predicted extreme temperatures on the spatial dynamics of a parasitoid-butterfly system and their interactions. Applied to four different climatic environments, the numerical approximations demonstrate the asymmetric impact of a warming climate on interacting species. The output density distribution maps highlight the capability of our model to produce interpretable multiscale predictions which can be used to identify and evaluate species vulnerability locally and across their range. C_LIO_LIBy building upon a solid mechanistic understanding of species-specific responses to environmental change, our model can be extended to other species and variables, including environments where the availability of empirical data is limited, and explore the dynamics and distribution of interacting species under different scenarios of environmental change. C_LI\nDon't Leave the Past Behind: How Larval Experience Shapes Pupal Antipredator Response in Aedes aegypti\nAuthors: Rawat, K.; Bhambore, A. A.; Isvaran, K.\nScore: 3.6, Published: 2024-02-07 DOI: 10.1101/2024.02.02.578532\nAnimals use predation encounters or risk experiences to influence their future antipredator responses. Such carryover effects of predation can benefit them by enhancing their antipredator behaviour and thereby decreasing their risk of mortality through predation. Despite these fitness benefits, behavioural carryovers of predation past may not be a common phenomenon in complex life cycles. Complex life cycles pose a unique evolutionary and physiological challenge to behavioural carryovers since every life stage is distinct in morphology, physiology, and function. Each life stage of a complex life cycle is expected to evolve its independent response based on the real-time threat level and manage the trade-offs accordingly. Along with the evolutionary challenge, physiological barriers can hamper behavioural carryovers, especially in holometabolous insects, where we observe extensive tissue remodelling and developmental compartmentalisation. We investigated behavioural carryover in the holometabolous mosquito model system, Aedes aegypti. We asked whether predation risk during a life stage carries over to the subsequent stage, influencing its behaviour, or if the next life stage responds according to its threat environment. Aedes aegypti has four major stages- egg, larva, pupa, and adult. We examined the effect of predation-risk experience across larval and pupal stages. Larval and pupal stages differ in morphology, physiology and function. They share the same habitat and, therefore, similar threats. We manipulated the threat of predation experienced by larvae and investigated its influence on pupal behaviour. We found behavioural carryover in the pupal stage for the first time and discovered exciting interactions between past experiences and the current threat environment. Our study underscores the crucial role of predation pressure in shaping the evolution of complex life cycles, emphasising the significance of early experiences with predators in influencing behavioural traits across distinct life stages.\nTime-series metagenomics reveals changing protistan ecology of a temperate dimictic lake\nAuthors: Krinos, A. I.; Bowers, R. M.; Rohwer, R. R.; McMahon, K.; Woyke, T.; Schulz, F.\nScore: 3.2, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579653\nProtists, single-celled eukaryotic organisms, are critical to food web ecology, contributing to primary productivity and connecting small bacteria and archaea to higher trophic levels. Lake Mendota is a large, eutrophic natural lake that is a Long-Term Ecological Research site and among the world's best-studied freshwater systems. Metagenomic samples have been collected and shotgun sequenced from Lake Mendota for the last twenty years. Here, we analyze this comprehensive time series to infer changes to the structure and function of the protistan community, and to hypothesize about their interactions with bacteria. Based on small subunit rRNA genes extracted from the metagenomes and metagenome-assembled genomes of microeukaryotes, we identify shifts in the eukaryotic phytoplankton community over time, which we predict to be a consequence of reduced zooplankton grazing pressures after the invasion of a invasive predator (the spiny water flea) to the lake. The metagenomic data also reveal the presence of the spiny water flea and the zebra mussel, a second invasive species to Lake Mendota, prior to their visual identification during routine monitoring. Further, we use species co-occurrence and co-abundance analysis to connect the protistan community with bacterial taxa. Correlation analysis suggests that protists and bacteria may interact or respond similarly to environmental conditions. Cryptophytes declined in the second decade of the timeseries, while many alveolate groups (e.g. ciliates and dinoflagellates) and diatoms increased in abundance, changes that have implications for food web efficiency in Lake Mendota. We demonstrate that metagenomic sequence-based community analysis can complement existing efforts to monitor protists in Lake Mendota based on microscopy-based count surveys. We observed patterns of seasonal abundance in microeukaryotes in Lake Mendota that corroborated expectations from other systems, including high abundance of cryptophytes in winter and diatoms in fall and spring, but with much higher resolution than previous surveys. Our study identified long-term changes in the abundance of eukaryotic microbes, and provided context for the known establishment of an invasive species that catalyzes a trophic cascade involving protists. Our findings are important for decoding potential long-term consequences of human interventions, including invasive species introduction.\nQuantifying the IUCN Red List: Using historical assessments to calculate future extinction risk\nAuthors: Bates, R.; Taylor, E.; Sun, Y.; Gumbs, R.; Böhm, M.; Gray, C.; Rosindell, J.\nScore: 3.0, Published: 2024-02-08 DOI: 10.1101/2024.02.04.578808\nThe IUCN Red List of Threatened Species assigns species to discrete categories based on a common set of criteria to capture relative extinction risk. Some quantitative analyses require a continuous probability of extinction not provided by discrete categories. Furthermore, the criteria may not translate to extinction risk in a universal way across species, because some species attributes are not incorporated into the criteria. Here we calculate extinction risk using Markov Models parameterised with historical IUCN Red List assessment data, and we provide quantitative extinction risk for use in downstream analyses. We find that taxonomy, body size, and habitat specialism interact with extinction risk in ways not fully captured by the existing categories. For example, an Endangered habitat specialist appears equally at risk compared to a Critically Endangered habitat generalist. We hope our findings will support more accurate quantitative biodiversity analyses of species extinctions and declines using Red List data.\nVariable responses of individual species to tropical forest degradation\nAuthors: Ewers, R. M.; Pearse, W. D.; Orme, C. D. L.; Amarasekare, P.; De Lorm, T.; Granville, N.; Adzhar, R.; Aldridge, D. C.; Ancrenaz, M.; Atton, G.; Barclay, H.; Barclay, M. L.; Bernard, H.; Bicknell, J. E.; Bishop, T. R.; Blackman, J.; Both, S.; Boyle, M. W.; Brant, H.; Brasington, E.; Burslem, D.; Bush, E. R.; Calloway, K.; Carbone, C.; Cator, L.; Chapman, P. M.; Chey, V.; Chung, A.; Clare, E. L.; Cusack, J.; Dancak, M.; Davies, Z. G.; Davison, C. W.; Dawood, M. M.; Deere, N. J.; Dickinson, K. M.; Didham, R. K.; Dobert, T. F.; Dow, R. A.; Drinkwater, R.; Edwards, D. P.; Eggleton, P.; Faruk, A.; F\nScore: 2.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.576668\nThe functional stability of ecosystems depends greatly on interspecific differences in responses to environmental perturbation. However, responses to perturbation are not necessarily invariant among populations of the same species, so intraspecific variation in responses might also contribute. Such inter-population response diversity has recently been shown to occur spatially across species ranges, but we lack estimates of the extent to which individual populations across an entire community might have perturbation responses that vary through time. We assess this using 524 taxa that have been repeatedly surveyed for the effects of tropical forest logging at a focal landscape in Sabah, Malaysia. Just 39 % of taxa - all with non-significant responses to forest degradation - had invariant responses. All other taxa (61 %) showed significantly different responses to the same forest degradation gradient across surveys, with 6 % of taxa responding to forest degradation in opposite directions across multiple surveys. Individual surveys had low power (\u003c 80 %) to determine the correct direction of response to forest degradation for one-fifth of all taxa. Recurrent rounds of logging disturbance increased the prevalence of intra-population response diversity, while uncontrollable environmental variation and/or turnover of intraspecific phenotypes generated variable responses in at least 44 % of taxa. Our results show that the responses of individual species to local environmental perturbations are remarkably flexible, likely providing an unrealised boost to the stability of disturbed habitats such as logged tropical forests.\nThe global root exudate carbon flux\nAuthors: Chari, N. R.; Tumber-Davila, S. J.; Phillips, R. P.; Bauerle, T. L.; Brunn, M.; Hafner, B. D.; Klein, T.; Obersteiner, S.; Reay, M. K.; Ullah, S.; Taylor, B. N.\nScore: 2.4, Published: 2024-02-06 DOI: 10.1101/2024.02.01.578470\nRoot exudation, the export of low-molecular weight organic carbon (C) from living plant roots to soil, influences microbial activity, nutrient availability, and ecosystem feedbacks to climate change, but the magnitude of this C flux at ecosystem and global scales is largely unknown. Here, we synthesize in situ measurements of root exudation rates and couple those to estimates of fine root biomass to estimate global and biome-level root exudate C fluxes. We estimate a global root exudate flux of 15.2 PgC y-1, or about 10% of global annual gross primary productivity. We found no differences in root mass-specific exudation rates among biomes, though total exudate fluxes are estimated to be greatest in grasslands owing to their high density of absorptive root biomass. Our synthesis highlights the global importance of root exudates in the terrestrial C cycle and identifies regions where more in situ measurements are needed to improve future estimates of root exudate C fluxes.\n",
  "wordCount" : "2478",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.577874">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.577874" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.577874">
        <p class="paperTitle">Geographically widespread and novel hemotropic mycoplasmas and bartonellae in Mexican free-tailed bats and sympatric North American bat species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.577874" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.577874" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Becker, D.; Dyer, K.; Lock, L.; Pladas, S.; Sukhadia, A.; Demory, B.; Nunes Batista, J.; Pineda, M.; Simmons, N.; Adams, A.; Frick, W.; O&#39;Mara, T.; Volokhov, D.</p>
        <p class="info">Score: 13.5, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.577874' target='https://doi.org/10.1101/2024.02.08.577874'> 10.1101/2024.02.08.577874</a></p>
        <p class="abstract">Bacterial pathogens remain poorly characterized in bats, especially in North America. We describe novel (and in some cases panmictic) hemoplasmas (12.9% positivity) and bartonellae (16.7% positivity) across three colonies of Mexican free-tailed bats (Tadarida brasiliensis), a partially migratory species that can seasonally travel hundreds of kilometers. Molecular analyses identified three novel Candidatus hemoplasma species most similar to another novel Candidatus species in Neotropical molossid bats. We also detected novel hemoplasmas in sympatric cave myotis (Myotis velifer) and pallid bats (Antrozous pallidus), with sequences in the latter 96.5% related to C. Mycoplasma haemohominis. We identified eight Bartonella genotypes, including those in cave myotis, with 96.7% similarity to C. Bartonella mayotimonensis. We also detected Bartonella rochalimae in migratory Tadarida brasiliensis, representing the first report of this human pathogen in bats. The seasonality and diversity of these bacteria observed here suggest that additional longitudinal, genomic, and immunological studies in bats are warranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578246">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578246" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578246">
        <p class="paperTitle">Does method of field preparation affect survival and growth of juvenile Chinook salmon in agricultural floodplains?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578246" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578246" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tallman, R.; Wampler, A.; Singer, G.; Jeffres, C.; Cocherell, D.; Colby, J.; Fangue, N.; Lusardi, R.; Rypel, A.</p>
        <p class="info">Score: 11.3, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578246' target='https://doi.org/10.1101/2024.02.02.578246'> 10.1101/2024.02.02.578246</a></p>
        <p class="abstract">There is growing interest to integrate conservation initiatives into agricultural practices using a reconciliation ecology framework. In Californias Central Valley, one approach to improving crucial nursery habitat for threatened and endangered fish species is by re-creating floodplain habitats through the inundation of agricultural fields during the non-growing season. We conducted a series of field experiments in agricultural floodplains (winter-flooded rice fields that were once natural floodplains) to explore whether different field preparation methods enhanced growth and survival of floodplain-dependent fish species. Approximately 8,000 juvenile fall-run Chinook Salmon (Oncorhynchus tshawytscha) were reared for at least 28 days on eight, 0.2 ha experimental fields. Each experimental field represented one of four treatments (two replicates per treatment): addition of large wood, addition of in-field canals, addition of both large wood and in-field canals, and control-where no additional field preparations were implemented beyond standard post-growing practices: i.e., chopping rice into stubble, baling and removing excess rice straw, and discing the field with a single pass by plowing rice straw into the ground to promote decomposition. We found no significant difference in water temperature, fish growth or fish survival among habitat treatments. Across treatments, survival ranged from 50.1%-78.3% and averaged 65.75% (/- 7.89% SE). These findings suggest agricultural floodplains require no additional modifications to promote fish growth and survival. Results illustrate the benefits of integrating working landscapes with conservation initiatives by creating more accessible and beneficial habitat for native fishes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579562">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579562" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579562">
        <p class="paperTitle">Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579562" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579562" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Braakman, R.; Satinsky, B.; O&#39;Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.</p>
        <p class="info">Score: 10.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579562' target='https://doi.org/10.1101/2024.02.09.579562'> 10.1101/2024.02.09.579562</a></p>
        <p class="abstract">Characterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups. The cyanobacterium Prochlorococcus exudes both compound classes, which metabolic reconstructions suggest follows synchronous daily genome replication. Co-occurring heterotrophs differentiate into purine or pyrimidine specialists, or generalists, and use compounds for different purposes. The most abundant heterotroph, SAR11, is a specialist that uses purines as sources of energy, carbon and/or nitrogen, with subgroups differentiating along ocean-scale gradients in the supply of energy and nitrogen, in turn leading to putative cryptic nitrogen cycles that link many microbes. Finally, in a SAR11 subgroup that dominates where Prochlorococcus is abundant, adenine additions to cultures inhibit DNA synthesis, poising cells for replication. We argue this subgroup uses inferred daily pulses of adenine from Prochlorococcus to metabolically synchronize to the daily supply of photosynthate from surrounding phytoplankton.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579602" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579602">
        <p class="paperTitle">What unexplored means: Mapping undersampled regions in natural history collections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakrabarty, P.; Ball, L.; Bedoya, A.; Rodriguez Machado, S.; Paredes-Burneo, D.; Rutledge, S.; Boyd, D.; Vander Pluym, D.; Babb-Biernacki, S.; Chipps, A.; Ozturk, R.; Terzi, Y.</p>
        <p class="info">Score: 9.0, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579602' target='https://doi.org/10.1101/2024.02.09.579602'> 10.1101/2024.02.09.579602</a></p>
        <p class="abstract">We examined global records of accessible natural history voucher collections (with publicly available data) for terrestrial and freshwater vascular plants, fungi, freshwater fishes, birds, mammals, and herpetofauna (amphibians and reptiles) and highlight areas of the world that would be considered undersampled and sometimes called &#34;unexplored&#34; (i.e., have relatively low, or no evidence of, past sampling efforts) under typical Western-scientific descriptions. We also question what &#34;unexplored&#34; may actually mean in these contexts and explain how retiring the term in favor of more nuanced phrasing can mitigate future misunderstandings of natural history science.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579316" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579316">
        <p class="paperTitle">Modelling the spatiotemporal dynamics of multispecies population interactions in the context of climate change</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Christou, K.; Schmucki, R.; Audusseau, H.</p>
        <p class="info">Score: 4.3, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579316' target='https://doi.org/10.1101/2024.02.07.579316'> 10.1101/2024.02.07.579316</a></p>
        <p class="abstract">O_LISpecies interactions are fundamental to the stability and productivity of ecosystems. To improve our capacity to predict and understand how climate change shapes the distribution of species and the dynamics of biotic interactions, we need to develop spatially explicit multi-species models that are built upon species-specific responses to changing conditions.
C_LIO_LIWe developed a two-dimensional diffusion-advection-reaction predator-prey model that integrates species-specific responses to heterogeneous landscapes, affecting species dispersal, reproduction and survival rates. We derived conditions for the stability and feasibility of the coexistence steady-state and observed how temperature variation stabilises or destabilises the system.
C_LIO_LIWe conducted numerical simulations to explore the effect of predicted extreme temperatures on the spatial dynamics of a parasitoid-butterfly system and their interactions. Applied to four different climatic environments, the numerical approximations demonstrate the asymmetric impact of a warming climate on interacting species. The output density distribution maps highlight the capability of our model to produce interpretable multiscale predictions which can be used to identify and evaluate species vulnerability locally and across their range.
C_LIO_LIBy building upon a solid mechanistic understanding of species-specific responses to environmental change, our model can be extended to other species and variables, including environments where the availability of empirical data is limited, and explore the dynamics and distribution of interacting species under different scenarios of environmental change.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578532">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578532" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578532">
        <p class="paperTitle">Don&#39;t Leave the Past Behind: How Larval Experience Shapes Pupal Antipredator Response in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578532" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578532" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rawat, K.; Bhambore, A. A.; Isvaran, K.</p>
        <p class="info">Score: 3.6, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578532' target='https://doi.org/10.1101/2024.02.02.578532'> 10.1101/2024.02.02.578532</a></p>
        <p class="abstract">Animals use predation encounters or risk experiences to influence their future antipredator responses. Such carryover effects of predation can benefit them by enhancing their antipredator behaviour and thereby decreasing their risk of mortality through predation. Despite these fitness benefits, behavioural carryovers of predation past may not be a common phenomenon in complex life cycles. Complex life cycles pose a unique evolutionary and physiological challenge to behavioural carryovers since every life stage is distinct in morphology, physiology, and function. Each life stage of a complex life cycle is expected to evolve its independent response based on the real-time threat level and manage the trade-offs accordingly. Along with the evolutionary challenge, physiological barriers can hamper behavioural carryovers, especially in holometabolous insects, where we observe extensive tissue remodelling and developmental compartmentalisation. We investigated behavioural carryover in the holometabolous mosquito model system, Aedes aegypti. We asked whether predation risk during a life stage carries over to the subsequent stage, influencing its behaviour, or if the next life stage responds according to its threat environment. Aedes aegypti has four major stages- egg, larva, pupa, and adult. We examined the effect of predation-risk experience across larval and pupal stages. Larval and pupal stages differ in morphology, physiology and function. They share the same habitat and, therefore, similar threats. We manipulated the threat of predation experienced by larvae and investigated its influence on pupal behaviour. We found behavioural carryover in the pupal stage for the first time and discovered exciting interactions between past experiences and the current threat environment. Our study underscores the crucial role of predation pressure in shaping the evolution of complex life cycles, emphasising the significance of early experiences with predators in influencing behavioural traits across distinct life stages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579653" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579653">
        <p class="paperTitle">Time-series metagenomics reveals changing protistan ecology of a temperate dimictic lake</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krinos, A. I.; Bowers, R. M.; Rohwer, R. R.; McMahon, K.; Woyke, T.; Schulz, F.</p>
        <p class="info">Score: 3.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579653' target='https://doi.org/10.1101/2024.02.09.579653'> 10.1101/2024.02.09.579653</a></p>
        <p class="abstract">Protists, single-celled eukaryotic organisms, are critical to food web ecology, contributing to primary productivity and connecting small bacteria and archaea to higher trophic levels. Lake Mendota is a large, eutrophic natural lake that is a Long-Term Ecological Research site and among the world&#39;s best-studied freshwater systems. Metagenomic samples have been collected and shotgun sequenced from Lake Mendota for the last twenty years. Here, we analyze this comprehensive time series to infer changes to the structure and function of the protistan community, and to hypothesize about their interactions with bacteria. Based on small subunit rRNA genes extracted from the metagenomes and metagenome-assembled genomes of microeukaryotes, we identify shifts in the eukaryotic phytoplankton community over time, which we predict to be a consequence of reduced zooplankton grazing pressures after the invasion of a invasive predator (the spiny water flea) to the lake. The metagenomic data also reveal the presence of the spiny water flea and the zebra mussel, a second invasive species to Lake Mendota, prior to their visual identification during routine monitoring. Further, we use species co-occurrence and co-abundance analysis to connect the protistan community with bacterial taxa. Correlation analysis suggests that protists and bacteria may interact or respond similarly to environmental conditions. Cryptophytes declined in the second decade of the timeseries, while many alveolate groups (e.g. ciliates and dinoflagellates) and diatoms increased in abundance, changes that have implications for food web efficiency in Lake Mendota. We demonstrate that metagenomic sequence-based community analysis can complement existing efforts to monitor protists in Lake Mendota based on microscopy-based count surveys. We observed patterns of seasonal abundance in microeukaryotes in Lake Mendota that corroborated expectations from other systems, including high abundance of cryptophytes in winter and diatoms in fall and spring, but with much higher resolution than previous surveys. Our study identified long-term changes in the abundance of eukaryotic microbes, and provided context for the known establishment of an invasive species that catalyzes a trophic cascade involving protists. Our findings are important for decoding potential long-term consequences of human interventions, including invasive species introduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578808" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578808">
        <p class="paperTitle">Quantifying the IUCN Red List: Using historical assessments to calculate future extinction risk</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bates, R.; Taylor, E.; Sun, Y.; Gumbs, R.; Böhm, M.; Gray, C.; Rosindell, J.</p>
        <p class="info">Score: 3.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578808' target='https://doi.org/10.1101/2024.02.04.578808'> 10.1101/2024.02.04.578808</a></p>
        <p class="abstract">The IUCN Red List of Threatened Species assigns species to discrete categories based on a common set of criteria to capture relative extinction risk. Some quantitative analyses require a continuous probability of extinction not provided by discrete categories. Furthermore, the criteria may not translate to extinction risk in a universal way across species, because some species attributes are not incorporated into the criteria. Here we calculate extinction risk using Markov Models parameterised with historical IUCN Red List assessment data, and we provide quantitative extinction risk for use in downstream analyses. We find that taxonomy, body size, and habitat specialism interact with extinction risk in ways not fully captured by the existing categories. For example, an Endangered habitat specialist appears equally at risk compared to a Critically Endangered habitat generalist. We hope our findings will support more accurate quantitative biodiversity analyses of species extinctions and declines using Red List data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.576668">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.576668" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.576668">
        <p class="paperTitle">Variable responses of individual species to tropical forest degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.576668" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.576668" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ewers, R. M.; Pearse, W. D.; Orme, C. D. L.; Amarasekare, P.; De Lorm, T.; Granville, N.; Adzhar, R.; Aldridge, D. C.; Ancrenaz, M.; Atton, G.; Barclay, H.; Barclay, M. L.; Bernard, H.; Bicknell, J. E.; Bishop, T. R.; Blackman, J.; Both, S.; Boyle, M. W.; Brant, H.; Brasington, E.; Burslem, D.; Bush, E. R.; Calloway, K.; Carbone, C.; Cator, L.; Chapman, P. M.; Chey, V.; Chung, A.; Clare, E. L.; Cusack, J.; Dancak, M.; Davies, Z. G.; Davison, C. W.; Dawood, M. M.; Deere, N. J.; Dickinson, K. M.; Didham, R. K.; Dobert, T. F.; Dow, R. A.; Drinkwater, R.; Edwards, D. P.; Eggleton, P.; Faruk, A.; F</p>
        <p class="info">Score: 2.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.576668' target='https://doi.org/10.1101/2024.02.09.576668'> 10.1101/2024.02.09.576668</a></p>
        <p class="abstract">The functional stability of ecosystems depends greatly on interspecific differences in responses to environmental perturbation. However, responses to perturbation are not necessarily invariant among populations of the same species, so intraspecific variation in responses might also contribute. Such inter-population response diversity has recently been shown to occur spatially across species ranges, but we lack estimates of the extent to which individual populations across an entire community might have perturbation responses that vary through time. We assess this using 524 taxa that have been repeatedly surveyed for the effects of tropical forest logging at a focal landscape in Sabah, Malaysia. Just 39 % of taxa - all with non-significant responses to forest degradation - had invariant responses. All other taxa (61 %) showed significantly different responses to the same forest degradation gradient across surveys, with 6 % of taxa responding to forest degradation in opposite directions across multiple surveys. Individual surveys had low power (&lt; 80 %) to determine the correct direction of response to forest degradation for one-fifth of all taxa. Recurrent rounds of logging disturbance increased the prevalence of intra-population response diversity, while uncontrollable environmental variation and/or turnover of intraspecific phenotypes generated variable responses in at least 44 % of taxa. Our results show that the responses of individual species to local environmental perturbations are remarkably flexible, likely providing an unrealised boost to the stability of disturbed habitats such as logged tropical forests.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578470" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578470">
        <p class="paperTitle">The global root exudate carbon flux</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chari, N. R.; Tumber-Davila, S. J.; Phillips, R. P.; Bauerle, T. L.; Brunn, M.; Hafner, B. D.; Klein, T.; Obersteiner, S.; Reay, M. K.; Ullah, S.; Taylor, B. N.</p>
        <p class="info">Score: 2.4, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578470' target='https://doi.org/10.1101/2024.02.01.578470'> 10.1101/2024.02.01.578470</a></p>
        <p class="abstract">Root exudation, the export of low-molecular weight organic carbon (C) from living plant roots to soil, influences microbial activity, nutrient availability, and ecosystem feedbacks to climate change, but the magnitude of this C flux at ecosystem and global scales is largely unknown. Here, we synthesize in situ measurements of root exudation rates and couple those to estimates of fine root biomass to estimate global and biome-level root exudate C fluxes. We estimate a global root exudate flux of 15.2 PgC y-1, or about 10% of global annual gross primary productivity. We found no differences in root mass-specific exudation rates among biomes, though total exudate fluxes are estimated to be greatest in grasslands owing to their high density of absorptive root biomass. Our synthesis highlights the global importance of root exudates in the terrestrial C cycle and identifies regions where more in situ measurements are needed to improve future estimates of root exudate C fluxes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
