<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland
Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.
Score: 9.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570
Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland
Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.
Score: 9.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570
Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland
Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.
Score: 9.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570
Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland\nAuthors: Curran, T. G.; O\u0026#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O\u0026#39;Neill, D.; McDevitt, A.; O\u0026#39;Reilly, C.; O\u0026#39;Meara, D. B.\nScore: 9.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570\nArthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase.",
  "keywords": [
    
  ],
  "articleBody": " DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland\nAuthors: Curran, T. G.; O'Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O'Neill, D.; McDevitt, A.; O'Reilly, C.; O'Meara, D. B.\nScore: 9.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570\nArthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase. Such dynamics lead to weakened ecosystem resilience, which can facilitate the establishment and spread of species with vector capabilities, such as mosquitoes. While many of the mosquito species present in Ireland possess vector capabilities, populations have not been nationally surveyed since 1991. At that time, species identification relied on morphological examination, a method that was even then recognised as inadequate for differentiating certain species within complexes. In this study, we present the first DNA barcoding investigation of Irish mosquito populations and reveal species and hidden diversity not previously recognised in Ireland. These include two potential new species/ecotypes, Culex torrentium and Culex pipiens form pipiens, which were recorded for the first time in Ireland. We demonstrate that barcoding with Cytochrome C Oxidase Subunit 1 (COI) based primers is suitable for the majority of species-level identifications. However, additional primer sets targeting the Internal Transcribed Spacer regions 1 and 2 (ITS1 and ITS2) were necessary to differentiate species within the Culex pipiens complex. The multi-locus approach employed in this study can enhance national surveillance efforts, especially in monitoring mosquitoes that may transmit vector-borne diseases, and in recognising increased species diversity from a biodiversity perspective.\nEnvironmental gradients mediate dispersal evolution during biological invasions\nAuthors: Benning, J. W.; Clark, E. I.; Hufbauer, R. A.; Weiss-Lehman, C.\nScore: 9.3, Published: 2023-12-15 DOI: 10.1101/2023.12.08.570855\nRapid dispersal evolution at the edge of a range expansion can accelerate invasions. However, expanding populations will often encounter environmental gradients that entail a fitness cost of dispersal. We used an eco-evolutionary model to explore how environmental heterogeneity influences adaptation and dispersal evolution during range expansion and, in turn, modulates the speed and predictability of invasion. Environmental gradients opposed the evolution of increased dispersal during invasion, even leading to the evolution of reduced dispersal along steep gradients. Counterintuitively, reduced dispersal allowed for faster expansion by minimizing maladaptation. While evolution across homogenous landscapes caused invasions to be highly unpredictable, even shallow environmental gradients greatly increased invasion predictability. We illustrate our models potential application to prediction and management of invasions by parameterizing it with data from a recent invertebrate range expansion. Overall, we find that environmental heterogeneity and local adaptation strongly modulate the effect of dispersal evolution on invasion trajectories.\nMetaproteogenomics resolution of a high-CO2 aquifer community suggests an active symbiotic lifestyle of groundwater Gracilibacteria\nAuthors: Figueroa-Gonzalez, P. A.; Bornemann, T. L. V.; Hinzke, T.; Maass, S.; Trautwein-Schult, A.; Starke, J.; Moore, C. J.; Esser, S. P.; Plewka, J.; Hesse, T.; Schmidt, T. C.; Schreiber, U.; Bor, B.; Becher, D.; Probst, A. J.\nScore: 7.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572140\nBackgroundBacteria of the Candidate Phyla Radiation (CPR), constituting about 25% of the bacterial biodiversity, are characterized by small cell size and patchy genomes without complete key metabolic pathways, suggesting a symbiotic lifestyle. Gracilibacteria (BD1-5), which are part of the CPR branch, possess alternate coded genomes and have not yet been cultivated. However, besides genomic evidence, little is known about the lifestyle of Gracilibacteria, their temporal dynamics, and activity in natural ecosystems, particularly in groundwater, where they were initially been genomically resolved. Therefore, we here aimed to investigate Gracilibacteria activity in situ and to discern expressed genes involved in their lifestyle, using the metaproteogenome of Gracilibacteria as a function of time in the cold-water geyser Wallender Born in the Volcanic Eifel region in Germany. ResultsWe coupled genome-resolved metagenomics and metaproteomics to investigate a cold-water geyser microbial community enriched in Gracilibacteria across a 12-day time-series. Groundwater was collected and sequentially filtered to fraction CPR and other bacteria. Based on 670 Gbps of metagenomic data, 1129 different ribosomal protein S3 marker genes and 751 high-quality genomes (123 population genomes after dereplication), we identified dominant bacteria belonging to Galionellales and Gracilibacteria along with keystone microbes, which were low in genomic abundance but substantially contributing to proteomic abundance. Seven high-quality Gracilibacteria genomes showed typical limitations, such as limited amino acid or nucleotide synthesis, in their central metabolism but no co-occurrence with potential hosts. The genomes of these Gracilibacteria encoded for a high number of proteins related to a symbiotic or even predatory lifestyle, e.g., type IV and type II secretion system subunits and features related to cell-cell interactions and cell motility, which were also detected on protein level. ConclusionsCoupling metagenomics to metaproteomics enabled us to identify microbial keystone taxa in a high-CO2 aquifer, and to reveal microbial dynamics of Gracilibacteria. We posit that Gracilibacteria might be successful microbial predators in this ecosystem, potentially aiding in population control of this highly perturbed microbial geyser community from the deep biosphere.\nThermal tolerance and survival are modulated by a natural gradient of infection in differentially acclimated hosts\nAuthors: De Bonville, J.; Cote, A.; Binning, S. A.\nScore: 8.3, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571921\nWild ectotherms are exposed to multiple stressors, including parasites, which can affect their responses to environmental change. Simultaneously, unprecedented warm temperatures are being recorded worldwide, increasing both the average and maximum temperatures experienced in nature. Understanding how ectotherms, such as fishes, will react to the combined stress of parasites and higher average temperatures can help predict the impact of extreme events such as heat waves on populations. The critical thermal method (CTM), which assesses upper (CTmax) and lower (CTmin) thermal tolerance, is often used in acclimated ectotherms to help predict their tolerance to various temperature scenarios. Yet, few studies have characterized the response of naturally infected fish to extreme temperature events or how acute thermal stress affects subsequent survival. We acclimated naturally infected pumpkinseed sunfish (Lepomis gibbosus), to four ecologically relevant temperatures (10, 15, 20 and 25{degrees}C) and one future warming scenario (30{degrees}C) for three weeks, before measuring CTmax and CTmin. We also assessed individual survival the week following CTmax. Interestingly, trematode parasites causing black spot disease were negatively related to CTmax, suggesting that heavily infected fish are less tolerant to acute warming. Moreover, fish infected with yellow grub parasites showed decreased survival in the days following CTmax implying that the infection load has negative survival consequences on sunfish during extreme warming events. Our findings indicate that parasite infection and high prolonged average temperatures can affect fish thermal tolerance and survival, emphasizing the need to better understand the concomitant effects of stressors on health outcomes in wild populations. Summary statementThis study shows that parasites influence thermal tolerance and survival of fish, suggesting that such stressors are important to increase the ecological relevance of thermal tolerance studies of wild animals.\nUnderstanding interactions in a complex world: Design and analysis of multi-species functional response experiments\nAuthors: Rosenbaum, B.; Li, J.; Hirt, M. R.; Ryser, R.; Brose, U.\nScore: 5.3, Published: 2023-12-19 DOI: 10.1101/2023.12.19.571428\nO_LIThe functional response describes resource-dependent feeding rates of consumers. While models for feeding on a single resource species are well studied and supported by a large body of empirical research, consumers feeding on multiple resource species are ubiquitous in nature. However, feeding experiments designed for parameterizing multi-species functional responses (MSFR) are extremely rare, mainly due to logistical challenges and the non-trivial nature of their statistical analysis. C_LIO_LIHere, we describe how these models can be fitted to empirical data in a Bayesian framework. Specifically, we address the problem of prey depletion during experiments, which can be accounted for through dynamical modeling. In a comprehensive simulation study, we test the effects of experimental design, sample size and noise level on the identifiability of four distinct MSFR models. Additionally, we demonstrate the methods versatility by applying it to a list of empirical datasets. C_LIO_LIWe identify experimental designs for feeding trials that produce the most accurate parameter estimates in two- and three-prey scenarios. Although noise can introduce systematic bias in parameter estimates, model selection performs surprisingly well, almost always identifying the correct model even for small datasets. C_LIO_LIOur flexible method allows the simultaneous analysis of feeding experiments from both single- and multi-prey scenarios, either with or without prey depletion. This will help to improve our understanding of prey selectivity, prey switching and hence food web stability and biodiversity. Our approach equips researchers with the right statistical tools to improve the mechanistic understanding of feeding interactions in complex ecosystems. C_LI\nIncreasing marine trophic web knowledge through DNA analyses of fish stomach contents: a step towards an Ecosystem Based Approach to fisheries research\nAuthors: Canals, O.; Lanzen, A.; Mendibil, I.; Bachiller, E.; Corrales, X.; Andonegi, E.; Cotano, U.; Rodriguez-Ezpeleta, N.\nScore: 5.3, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572137\nMultispecies and ecosystem models, which are key for the implementation of ecosystem-based approaches to fisheries management, require extensive data on the trophic interactions between marine organisms, including changes over time. DNA metabarcoding (the simultaneous identification of all taxa in a sample by sequencing a short DNA region) could be used for speeding up large-scaled stomach content data collection. Yet, for DNA metabarcoding to be routinely implemented, technical challenges should be addressed, such as the potentially complicated sampling logistics, the detection of a high proportion of predator DNA, and the inability to provide reliable abundance estimations. Here, we present a DNA metabarcoding assay developed to examine the diet of five commercially important fish, which is feasible to be incorporated into routinary samplings. The method is devised to speed up the analysis process by avoiding the stomach dissection and content extraction steps, while preventing the amplification of predator DNA by using blocking primers. Tested in mock samples and in real stomachs, the method has proven effective and shows great effectiveness discerning diet variations due to predator ecology or prey availability. Additionally, by applying our protocol to mackerel stomachs previously analysed by visual inspection, we showcase how DNA metabarcoding could complement visually based data by detecting overlooked prey by the visual approach. We finally discuss how DNA metabarcoding-based data can contribute to trophic data collection. Our work reinforces the potential of DNA metabarcoding for the study and monitoring of fish trophic interactions and provides a basis for its incorporation into routine monitoring programs, which will be critical for the implementation of ecosystem-based approaches to fisheries management.\nSpecies interactions drive continuous assembly of freshwater communities in stochastic environments\nAuthors: Tabi, A.; Siqueira, T.; Tonkin, J. D.\nScore: 2.5, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572946\nUnderstanding the factors driving the maintenance of long-term biodiversity in changing environments is essential for improving restoration and sustainability strategies in the face of global environmental change. Biodiversity is shaped by both niche and stochastic processes, however the strength of deterministic processes in unpredictable environmental regimes is highly debated. Since communities continuously change over time and space - species persist, disappear or (re)appear - understanding the drivers of species gains and losses from communities should inform us about whether niche or stochastic processes dominate community dynamics. Applying a nonparametric causal discovery approach to a 30-year time series containing annual abundances of benthic invertebrates across 66 locations in New Zealand rivers, we found a strong asynchronous causal relationship between species gains and losses directly driven by predation indicating that niche processes dominate community dynamics. Despite the unpredictable nature of these system, environmental noise was only indirectly related to species gains and losses through altering life history trait distribution. Using a stochastic birth-death framework, we demonstrate that the negative relationship between species gains and losses can not emerge without strong niche processes. Our results showed that even in systems that are dominated by unpredictable environmental variability, species interactions drive continuous community assembly.\nMaternal body condition affects the response of the gut microbiome to a widespread contaminant in larval spined toads\nAuthors: Tartu, S.; Pollet, N.; Clavereau, I.; Bouchard, G.; Brischoux, F.\nScore: 2.1, Published: 2023-12-20 DOI: 10.1101/2023.12.18.572122\nGlyphosates primary metabolite, aminomethylphosphonic acid (AMPA) is the most detected pollutant in surface waters. Recent studies have raised concerns regarding its toxicity, yet the involved underlying mechanisms remain poorly understood. A disruption of the gut microbiome, which plays a crucial role in host health, could be the mediator of most adverse effects. We investigated the impact of AMPA exposure on the gut microbiome of spined toad tadpoles (Bufo spinosus). We hypothesized that AMPA could alter the gut microbiota composition and that these effects could depend on the microbiota source. We exposed tadpoles to minute concentrations of AMPA and analyzed their faecal microbiota using 16S rRNA gene sequencing as a proxy of the gut microbiota. AMPA exposure decreased the gut bacterial biomass and affected bacterial community composition of tadpoles faeces. Furthermore, we observed interactions between AMPA exposure and maternal body condition on the Bacteroidota and Actinobacteriota phyla abundances. This suggests a maternal effect on early-life microbial colonizers that could influence the response of the gut microbiome to AMPA. These findings highlight the importance of considering the gut microbiome when studying the effects of environmental contaminants. Further research is needed to elucidate the long-term implications of this microbiome alteration for amphibian health.\nHow to barcode (almost all) freshwater biodiversity\nAuthors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.\nScore: 16.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571596\nFreshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation. Environmental DNA surveys offer a means to inventory and monitor aquatic diversity, yet most studies focus on one or a few taxonomic groups only. In this study, we sought to 1) identify thoroughly validated, cost-efficient primer pair combinations that maximize detection of broad swaths of freshwater diversity, and 2) facilitate future primer pair selection by creating a free online and user-friendly tool. We first evaluated the completeness of public reference sequence databases and the efficiency of 14 primer pairs using an in silico approach, and then performed eDNA surveys using five mock communities (mix of DNA from tissues), water samples from aquarium samples with known taxonomic composition, and finally water samples from freshwater systems in Eastern Canada. We highlight the power of eDNA-based metabarcoding for reconstructing freshwater communities, including prey, parasite, pathogen, invasive, and declining species. Our work reveals the importance of the marker choice on species resolution, as well as the importance of degenerate primers, the length of the target fragment and the filtering parameters on detection success in water eDNA samples. Our new online tool SNIPe revealed that 13 to 14 primer pairs are necessary to recover 100% of the species in water samples (aquarium and natural systems), but four primer pairs are sufficient to recover almost 75% of taxa with little overlap. These results highlight the usefulness of eDNA for freshwater monitoring and should prompt more studies on tools to survey all-inclusive communities.\nHow Clustering Promotes Biodiversity: A Solution to the Plankton Paradox\nAuthors: Huruntz, K. K.; Ghukasyan, L. E.; Gevorgyan, G. A.\nScore: 1.6, Published: 2023-12-19 DOI: 10.1101/2023.12.17.572061\nIt is known that a species-saturated ecological community driven by competitive Lotka-Volterra equations, following a path of competitive exclusion, lingers in a long-term state with species congregated in niche space into well-separated clusters. Characterizing the topology of the niche structure of this state as a hierarchical one, we put forward a generalization of the Lotka-Volterra multi-resource competition model, which recognizes this topology at the basis of its own network structure of interrelated differential equations. This kind of generalization substantially reduces the competitive constraints on species coexistence, offering promising potential for a conceptual resolution of the plankton paradox.\n",
  "wordCount" : "2648",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572570">
        <p class="paperTitle">DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.</p>
        <p class="info">Score: 9.3, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572570' target='https://doi.org/10.1101/2023.12.20.572570'> 10.1101/2023.12.20.572570</a></p>
        <p class="abstract">Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase. Such dynamics lead to weakened ecosystem resilience, which can facilitate the establishment and spread of species with vector capabilities, such as mosquitoes. While many of the mosquito species present in Ireland possess vector capabilities, populations have not been nationally surveyed since 1991. At that time, species identification relied on morphological examination, a method that was even then recognised as inadequate for differentiating certain species within complexes. In this study, we present the first DNA barcoding investigation of Irish mosquito populations and reveal species and hidden diversity not previously recognised in Ireland. These include two potential new species/ecotypes, Culex torrentium and Culex pipiens form pipiens, which were recorded for the first time in Ireland. We demonstrate that barcoding with Cytochrome C Oxidase Subunit 1 (COI) based primers is suitable for the majority of species-level identifications. However, additional primer sets targeting the Internal Transcribed Spacer regions 1 and 2 (ITS1 and ITS2) were necessary to differentiate species within the Culex pipiens complex. The multi-locus approach employed in this study can enhance national surveillance efforts, especially in monitoring mosquitoes that may transmit vector-borne diseases, and in recognising increased species diversity from a biodiversity perspective.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570855">
        <p class="paperTitle">Environmental gradients mediate dispersal evolution during biological invasions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benning, J. W.; Clark, E. I.; Hufbauer, R. A.; Weiss-Lehman, C.</p>
        <p class="info">Score: 9.3, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570855' target='https://doi.org/10.1101/2023.12.08.570855'> 10.1101/2023.12.08.570855</a></p>
        <p class="abstract">Rapid dispersal evolution at the edge of a range expansion can accelerate invasions. However, expanding populations will often encounter environmental gradients that entail a fitness cost of dispersal. We used an eco-evolutionary model to explore how environmental heterogeneity influences adaptation and dispersal evolution during range expansion and, in turn, modulates the speed and predictability of invasion. Environmental gradients opposed the evolution of increased dispersal during invasion, even leading to the evolution of reduced dispersal along steep gradients. Counterintuitively, reduced dispersal allowed for faster expansion by minimizing maladaptation. While evolution across homogenous landscapes caused invasions to be highly unpredictable, even shallow environmental gradients greatly increased invasion predictability. We illustrate our models potential application to prediction and management of invasions by parameterizing it with data from a recent invertebrate range expansion. Overall, we find that environmental heterogeneity and local adaptation strongly modulate the effect of dispersal evolution on invasion trajectories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572140">
        <p class="paperTitle">Metaproteogenomics resolution of a high-CO2 aquifer community suggests an active symbiotic lifestyle of groundwater Gracilibacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa-Gonzalez, P. A.; Bornemann, T. L. V.; Hinzke, T.; Maass, S.; Trautwein-Schult, A.; Starke, J.; Moore, C. J.; Esser, S. P.; Plewka, J.; Hesse, T.; Schmidt, T. C.; Schreiber, U.; Bor, B.; Becher, D.; Probst, A. J.</p>
        <p class="info">Score: 7.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572140' target='https://doi.org/10.1101/2023.12.18.572140'> 10.1101/2023.12.18.572140</a></p>
        <p class="abstract">BackgroundBacteria of the Candidate Phyla Radiation (CPR), constituting about 25% of the bacterial biodiversity, are characterized by small cell size and patchy genomes without complete key metabolic pathways, suggesting a symbiotic lifestyle. Gracilibacteria (BD1-5), which are part of the CPR branch, possess alternate coded genomes and have not yet been cultivated. However, besides genomic evidence, little is known about the lifestyle of Gracilibacteria, their temporal dynamics, and activity in natural ecosystems, particularly in groundwater, where they were initially been genomically resolved. Therefore, we here aimed to investigate Gracilibacteria activity in situ and to discern expressed genes involved in their lifestyle, using the metaproteogenome of Gracilibacteria as a function of time in the cold-water geyser Wallender Born in the Volcanic Eifel region in Germany.

ResultsWe coupled genome-resolved metagenomics and metaproteomics to investigate a cold-water geyser microbial community enriched in Gracilibacteria across a 12-day time-series. Groundwater was collected and sequentially filtered to fraction CPR and other bacteria. Based on 670 Gbps of metagenomic data, 1129 different ribosomal protein S3 marker genes and 751 high-quality genomes (123 population genomes after dereplication), we identified dominant bacteria belonging to Galionellales and Gracilibacteria along with keystone microbes, which were low in genomic abundance but substantially contributing to proteomic abundance. Seven high-quality Gracilibacteria genomes showed typical limitations, such as limited amino acid or nucleotide synthesis, in their central metabolism but no co-occurrence with potential hosts. The genomes of these Gracilibacteria encoded for a high number of proteins related to a symbiotic or even predatory lifestyle, e.g., type IV and type II secretion system subunits and features related to cell-cell interactions and cell motility, which were also detected on protein level.

ConclusionsCoupling metagenomics to metaproteomics enabled us to identify microbial keystone taxa in a high-CO2 aquifer, and to reveal microbial dynamics of Gracilibacteria. We posit that Gracilibacteria might be successful microbial predators in this ecosystem, potentially aiding in population control of this highly perturbed microbial geyser community from the deep biosphere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571921">
        <p class="paperTitle">Thermal tolerance and survival are modulated by a natural gradient of infection in differentially acclimated hosts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Bonville, J.; Cote, A.; Binning, S. A.</p>
        <p class="info">Score: 8.3, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571921' target='https://doi.org/10.1101/2023.12.15.571921'> 10.1101/2023.12.15.571921</a></p>
        <p class="abstract">Wild ectotherms are exposed to multiple stressors, including parasites, which can affect their responses to environmental change. Simultaneously, unprecedented warm temperatures are being recorded worldwide, increasing both the average and maximum temperatures experienced in nature. Understanding how ectotherms, such as fishes, will react to the combined stress of parasites and higher average temperatures can help predict the impact of extreme events such as heat waves on populations. The critical thermal method (CTM), which assesses upper (CTmax) and lower (CTmin) thermal tolerance, is often used in acclimated ectotherms to help predict their tolerance to various temperature scenarios. Yet, few studies have characterized the response of naturally infected fish to extreme temperature events or how acute thermal stress affects subsequent survival. We acclimated naturally infected pumpkinseed sunfish (Lepomis gibbosus), to four ecologically relevant temperatures (10, 15, 20 and 25{degrees}C) and one future warming scenario (30{degrees}C) for three weeks, before measuring CTmax and CTmin. We also assessed individual survival the week following CTmax. Interestingly, trematode parasites causing black spot disease were negatively related to CTmax, suggesting that heavily infected fish are less tolerant to acute warming. Moreover, fish infected with yellow grub parasites showed decreased survival in the days following CTmax implying that the infection load has negative survival consequences on sunfish during extreme warming events. Our findings indicate that parasite infection and high prolonged average temperatures can affect fish thermal tolerance and survival, emphasizing the need to better understand the concomitant effects of stressors on health outcomes in wild populations.

Summary statementThis study shows that parasites influence thermal tolerance and survival of fish, suggesting that such stressors are important to increase the ecological relevance of thermal tolerance studies of wild animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.571428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.571428" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.571428">
        <p class="paperTitle">Understanding interactions in a complex world: Design and analysis of multi-species functional response experiments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.571428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.571428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosenbaum, B.; Li, J.; Hirt, M. R.; Ryser, R.; Brose, U.</p>
        <p class="info">Score: 5.3, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.571428' target='https://doi.org/10.1101/2023.12.19.571428'> 10.1101/2023.12.19.571428</a></p>
        <p class="abstract">O_LIThe functional response describes resource-dependent feeding rates of consumers. While models for feeding on a single resource species are well studied and supported by a large body of empirical research, consumers feeding on multiple resource species are ubiquitous in nature. However, feeding experiments designed for parameterizing multi-species functional responses (MSFR) are extremely rare, mainly due to logistical challenges and the non-trivial nature of their statistical analysis.
C_LIO_LIHere, we describe how these models can be fitted to empirical data in a Bayesian framework. Specifically, we address the problem of prey depletion during experiments, which can be accounted for through dynamical modeling. In a comprehensive simulation study, we test the effects of experimental design, sample size and noise level on the identifiability of four distinct MSFR models. Additionally, we demonstrate the methods versatility by applying it to a list of empirical datasets.
C_LIO_LIWe identify experimental designs for feeding trials that produce the most accurate parameter estimates in two- and three-prey scenarios. Although noise can introduce systematic bias in parameter estimates, model selection performs surprisingly well, almost always identifying the correct model even for small datasets.
C_LIO_LIOur flexible method allows the simultaneous analysis of feeding experiments from both single- and multi-prey scenarios, either with or without prey depletion. This will help to improve our understanding of prey selectivity, prey switching and hence food web stability and biodiversity. Our approach equips researchers with the right statistical tools to improve the mechanistic understanding of feeding interactions in complex ecosystems.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572137">
        <p class="paperTitle">Increasing marine trophic web knowledge through DNA analyses of fish stomach contents: a step towards an Ecosystem Based Approach to fisheries research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Canals, O.; Lanzen, A.; Mendibil, I.; Bachiller, E.; Corrales, X.; Andonegi, E.; Cotano, U.; Rodriguez-Ezpeleta, N.</p>
        <p class="info">Score: 5.3, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572137' target='https://doi.org/10.1101/2023.12.18.572137'> 10.1101/2023.12.18.572137</a></p>
        <p class="abstract">Multispecies and ecosystem models, which are key for the implementation of ecosystem-based approaches to fisheries management, require extensive data on the trophic interactions between marine organisms, including changes over time. DNA metabarcoding (the simultaneous identification of all taxa in a sample by sequencing a short DNA region) could be used for speeding up large-scaled stomach content data collection. Yet, for DNA metabarcoding to be routinely implemented, technical challenges should be addressed, such as the potentially complicated sampling logistics, the detection of a high proportion of predator DNA, and the inability to provide reliable abundance estimations. Here, we present a DNA metabarcoding assay developed to examine the diet of five commercially important fish, which is feasible to be incorporated into routinary samplings. The method is devised to speed up the analysis process by avoiding the stomach dissection and content extraction steps, while preventing the amplification of predator DNA by using blocking primers. Tested in mock samples and in real stomachs, the method has proven effective and shows great effectiveness discerning diet variations due to predator ecology or prey availability. Additionally, by applying our protocol to mackerel stomachs previously analysed by visual inspection, we showcase how DNA metabarcoding could complement visually based data by detecting overlooked prey by the visual approach. We finally discuss how DNA metabarcoding-based data can contribute to trophic data collection. Our work reinforces the potential of DNA metabarcoding for the study and monitoring of fish trophic interactions and provides a basis for its incorporation into routine monitoring programs, which will be critical for the implementation of ecosystem-based approaches to fisheries management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572946">
        <p class="paperTitle">Species interactions drive continuous assembly of freshwater communities in stochastic environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tabi, A.; Siqueira, T.; Tonkin, J. D.</p>
        <p class="info">Score: 2.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572946' target='https://doi.org/10.1101/2023.12.21.572946'> 10.1101/2023.12.21.572946</a></p>
        <p class="abstract">Understanding the factors driving the maintenance of long-term biodiversity in changing environments is essential for improving restoration and sustainability strategies in the face of global environmental change. Biodiversity is shaped by both niche and stochastic processes, however the strength of deterministic processes in unpredictable environmental regimes is highly debated. Since communities continuously change over time and space - species persist, disappear or (re)appear - understanding the drivers of species gains and losses from communities should inform us about whether niche or stochastic processes dominate community dynamics. Applying a nonparametric causal discovery approach to a 30-year time series containing annual abundances of benthic invertebrates across 66 locations in New Zealand rivers, we found a strong asynchronous causal relationship between species gains and losses directly driven by predation indicating that niche processes dominate community dynamics. Despite the unpredictable nature of these system, environmental noise was only indirectly related to species gains and losses through altering life history trait distribution. Using a stochastic birth-death framework, we demonstrate that the negative relationship between species gains and losses can not emerge without strong niche processes. Our results showed that even in systems that are dominated by unpredictable environmental variability, species interactions drive continuous community assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572122">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572122" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572122">
        <p class="paperTitle">Maternal body condition affects the response of the gut microbiome to a widespread contaminant in larval spined toads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572122" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572122" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tartu, S.; Pollet, N.; Clavereau, I.; Bouchard, G.; Brischoux, F.</p>
        <p class="info">Score: 2.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572122' target='https://doi.org/10.1101/2023.12.18.572122'> 10.1101/2023.12.18.572122</a></p>
        <p class="abstract">Glyphosates primary metabolite, aminomethylphosphonic acid (AMPA) is the most detected pollutant in surface waters. Recent studies have raised concerns regarding its toxicity, yet the involved underlying mechanisms remain poorly understood. A disruption of the gut microbiome, which plays a crucial role in host health, could be the mediator of most adverse effects. We investigated the impact of AMPA exposure on the gut microbiome of spined toad tadpoles (Bufo spinosus). We hypothesized that AMPA could alter the gut microbiota composition and that these effects could depend on the microbiota source. We exposed tadpoles to minute concentrations of AMPA and analyzed their faecal microbiota using 16S rRNA gene sequencing as a proxy of the gut microbiota. AMPA exposure decreased the gut bacterial biomass and affected bacterial community composition of tadpoles faeces. Furthermore, we observed interactions between AMPA exposure and maternal body condition on the Bacteroidota and Actinobacteriota phyla abundances. This suggests a maternal effect on early-life microbial colonizers that could influence the response of the gut microbiome to AMPA. These findings highlight the importance of considering the gut microbiome when studying the effects of environmental contaminants. Further research is needed to elucidate the long-term implications of this microbiome alteration for amphibian health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571596">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571596" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571596">
        <p class="paperTitle">How to barcode (almost all) freshwater biodiversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571596" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571596" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tournayre, O.; Tian, H.; Lougheed, D. R.; Windle, M. J. S.; Lambert, S.; Carter, J.; Sun, Z.; Ridal, J.; Wang, Y.; Cumming, B. F.; Arnott, S. E.; Lougheed, S. C.</p>
        <p class="info">Score: 16.6, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571596' target='https://doi.org/10.1101/2023.12.13.571596'> 10.1101/2023.12.13.571596</a></p>
        <p class="abstract">Freshwater ecosystems are complex, diverse and face a variety of imminent threats that have led to changes in both ecosystem structure and function. It is urgent that we develop and standardize monitoring tools allowing for rapid and comprehensive assessment of freshwater communities to understand their changing dynamics and to inform conservation. Environmental DNA surveys offer a means to inventory and monitor aquatic diversity, yet most studies focus on one or a few taxonomic groups only. In this study, we sought to 1) identify thoroughly validated, cost-efficient primer pair combinations that maximize detection of broad swaths of freshwater diversity, and 2) facilitate future primer pair selection by creating a free online and user-friendly tool. We first evaluated the completeness of public reference sequence databases and the efficiency of 14 primer pairs using an in silico approach, and then performed eDNA surveys using five mock communities (mix of DNA from tissues), water samples from aquarium samples with known taxonomic composition, and finally water samples from freshwater systems in Eastern Canada. We highlight the power of eDNA-based metabarcoding for reconstructing freshwater communities, including prey, parasite, pathogen, invasive, and declining species. Our work reveals the importance of the marker choice on species resolution, as well as the importance of degenerate primers, the length of the target fragment and the filtering parameters on detection success in water eDNA samples. Our new online tool SNIPe revealed that 13 to 14 primer pairs are necessary to recover 100% of the species in water samples (aquarium and natural systems), but four primer pairs are sufficient to recover almost 75% of taxa with little overlap. These results highlight the usefulness of eDNA for freshwater monitoring and should prompt more studies on tools to survey all-inclusive communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.17.572061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.17.572061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.17.572061">
        <p class="paperTitle">How Clustering Promotes Biodiversity: A Solution to the Plankton Paradox</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.17.572061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.17.572061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huruntz, K. K.; Ghukasyan, L. E.; Gevorgyan, G. A.</p>
        <p class="info">Score: 1.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.17.572061' target='https://doi.org/10.1101/2023.12.17.572061'> 10.1101/2023.12.17.572061</a></p>
        <p class="abstract">It is known that a species-saturated ecological community driven by competitive Lotka-Volterra equations, following a path of competitive exclusion, lingers in a long-term state with species congregated in niche space into well-separated clusters. Characterizing the topology of the niche structure of this state as a hierarchical one, we put forward a generalization of the Lotka-Volterra multi-resource competition model, which recognizes this topology at the basis of its own network structure of interrelated differential equations. This kind of generalization substantially reduces the competitive constraints on species coexistence, offering promising potential for a conceptual resolution of the plankton paradox.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
