<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes
Authors: Braakman, R.; Satinsky, B.; O&#39;Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.
Score: 16.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579562
Characterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes
Authors: Braakman, R.; Satinsky, B.; O&#39;Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.
Score: 16.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579562
Characterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes
Authors: Braakman, R.; Satinsky, B.; O&#39;Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.
Score: 16.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579562
Characterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes\nAuthors: Braakman, R.; Satinsky, B.; O\u0026#39;Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.\nScore: 16.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579562\nCharacterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups.",
  "keywords": [
    
  ],
  "articleBody": " Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes\nAuthors: Braakman, R.; Satinsky, B.; O'Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.\nScore: 16.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579562\nCharacterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups. The cyanobacterium Prochlorococcus exudes both compound classes, which metabolic reconstructions suggest follows synchronous daily genome replication. Co-occurring heterotrophs differentiate into purine or pyrimidine specialists, or generalists, and use compounds for different purposes. The most abundant heterotroph, SAR11, is a specialist that uses purines as sources of energy, carbon and/or nitrogen, with subgroups differentiating along ocean-scale gradients in the supply of energy and nitrogen, in turn leading to putative cryptic nitrogen cycles that link many microbes. Finally, in a SAR11 subgroup that dominates where Prochlorococcus is abundant, adenine additions to cultures inhibit DNA synthesis, poising cells for replication. We argue this subgroup uses inferred daily pulses of adenine from Prochlorococcus to metabolically synchronize to the daily supply of photosynthate from surrounding phytoplankton.\nWhat unexplored means: Mapping undersampled regions in natural history collections\nAuthors: Chakrabarty, P.; Ball, L.; Bedoya, A.; Rodriguez Machado, S.; Paredes-Burneo, D.; Rutledge, S.; Boyd, D.; Vander Pluym, D.; Babb-Biernacki, S.; Chipps, A.; Ozturk, R.; Terzi, Y.\nScore: 12.0, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579602\nWe examined global records of accessible natural history voucher collections (with publicly available data) for terrestrial and freshwater vascular plants, fungi, freshwater fishes, birds, mammals, and herpetofauna (amphibians and reptiles) and highlight areas of the world that would be considered undersampled and sometimes called \"unexplored\" (i.e., have relatively low, or no evidence of, past sampling efforts) under typical Western-scientific descriptions. We also question what \"unexplored\" may actually mean in these contexts and explain how retiring the term in favor of more nuanced phrasing can mitigate future misunderstandings of natural history science.\nPredictable functional biogeography of marine microbial heterotrophs\nAuthors: Zakem, E. J.; McNichol, J.; Weissman, J. L.; Raut, Y.; Xu, L.; Halewood, E. R.; Carlson, C. A.; Dutkiewicz, S.; Fuhrman, J. A.; Levine, N. M.\nScore: 6.2, Published: 2024-02-15 DOI: 10.1101/2024.02.14.580411\nMicrobial heterotrophs (`picoheterotrophs') drive global carbon cycling, but how to quantitatively organize their functional complexity remains unclear. Here, we generate a global-scale, mechanistic understanding of marine picoheterotrophic functional biogeography with a novel model-data synthesis. We build picoheterotrophic diversity into a trait-based marine ecosystem model along two axes: substrate lability and optimization for growth rate (copiotrophy) vs. substrate affinity (oligotrophy). Using genetic sequences along an Alaska-to-Antarctica Pacific Ocean transect, we compile 21 picoheterotrophic guilds and estimate their degree of copiotrophy. Data and model agreement suggests that gradients in predation and substrate lability predominantly set biogeographical patterns, and identifies `slow copiotrophs' subsisting at depth. Results demonstrate the predictability of the marine microbiome and connect ecological dynamics with carbon storage, crucial for projecting changes in a warming ocean.\nTime-series metagenomics reveals changing protistan ecology of a temperate dimictic lake\nAuthors: Krinos, A. I.; Bowers, R. M.; Rohwer, R. R.; McMahon, K.; Woyke, T.; Schulz, F.\nScore: 3.8, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579653\nBackgroundProtists, single-celled eukaryotic organisms, are critical to food web ecology, contributing to primary productivity and connecting small bacteria and archaea to higher trophic levels. Lake Mendota is a large, eutrophic natural lake that is a Long-Term Ecological Research site and among the worlds best-studied freshwater systems. Metagenomic samples have been collected and shotgun sequenced from Lake Mendota for the last twenty years. Here, we analyze this comprehensive time series to infer changes to the structure and function of the protistan community, and to hypothesize about their interactions with bacteria. ResultsBased on small subunit rRNA genes extracted from the metagenomes and metagenome-assembled genomes of microeukaryotes, we identify shifts in the eukaryotic phytoplankton community over time, which we predict to be a consequence of reduced zooplankton grazing pressures after the invasion of a invasive predator (the spiny water flea) to the lake. The metagenomic data also reveal the presence of the spiny water flea and the zebra mussel, a second invasive species to Lake Mendota, prior to their visual identification during routine monitoring. Further, we use species co-occurrence and co-abundance analysis to connect the protistan community with bacterial taxa. Correlation analysis suggests that protists and bacteria may interact or respond similarly to environmental conditions. Cryptophytes declined in the second decade of the timeseries, while many alveolate groups (e.g. ciliates and dinoflagellates) and diatoms increased in abundance, changes that have implications for food web efficiency in Lake Mendota. ConclusionsWe demonstrate that metagenomic sequence-based community analysis can complement existing eorts to monitor protists in Lake Mendota based on microscopy-based count surveys. We observed patterns of seasonal abundance in microeukaryotes in Lake Mendota that corroborated expectations from other systems, including high abundance of cryptophytes in winter and diatoms in fall and spring, but with much higher resolution than previous surveys. Our study identified long-term changes in the abundance of eukaryotic microbes, and provided context for the known establishment of an invasive species that catalyzes a trophic cascade involving protists. Our findings are important for decoding potential long-term consequences of human interventions, including invasive species introduction.\nTransinfection of Wolbachia wAlbB into Culex quinquefasciatus mosquitoes does not alter vector competence for Hawaiian avian malaria (Plasmodium relictum GRW4)\nAuthors: Kilpatrick, A. M.; Seidl, C. M.; Ipsaro, I. J.; Garrison, C. E.; Fabbri, G.; Howell, P. I.; McGowan, A. G.; White, B. J.; Mitchell, S. N.\nScore: 3.5, Published: 2024-02-16 DOI: 10.1101/2024.02.16.580617\nAvian malaria is expanding upslope with warmer temperatures and driving multiple species of Hawaiian birds towards extinction. Methods to reduce malaria transmission are urgently needed to prevent further declines. Releasing Wolbachia-infected incompatible male mosquitoes suppress mosquito populations and releasing Wolbachia-infected female mosquitoes could reduce pathogen transmission if the Wolbachia strain reduced vector competence. We cleared Culex quinquefasciatus of their natural Wolbachia pipientis wPip infection and transinfected them with Wolbachia wAlbB isolated from Aedes albopictus. We show that wAlbB infection was transmitted transovarially, and demonstrate cytoplasmic incompatibility with wild-type mosquitoes infected with wPip from Oahu and Maui, Hawaii. We measured vector competence for avian malaria, Plasmodium relictum, lineage GRW4, of seven mosquito lines (two with wAlbB; three with natural wPip infection, and two cleared of Wolbachia infection) by allowing them to feed on canaries infected with recently collected field isolates of Hawaiian P. relictum. We tested 73 groups (Ntotal = 1176) of mosquitoes for P. relictum infection in abdomens and disseminated (thorax) infections 6-14 days after feeding across a range of parasitemias from 0.028% to 2.49%, and a smaller subset of salivary glands. We found no measurable effect of Wolbachia on any endpoint, but strong effects of parasitemia, days post feeding, and mosquito strain on both abdomen infection prevalence and disseminated infection prevalence. These results suggest that releasing male wAlbB-infected C. quinquefasciatus mosquitoes could suppress wPip-infected mosquito populations, but would have little positive or negative impact on mosquito vector competence for P. relictum if wAlbB became established in local mosquito populations. More broadly, the lack of Wolbachia effects on vector competence we observed highlights the variable impacts of both native and transfected Wolbachia infections in mosquitoes.\nAn increasing Arctic-Boreal CO2 sink despite strong regional sources\nAuthors: Virkkala, A.-M.\nScore: 2.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579581\nThe Arctic-Boreal Zone (ABZ) is rapidly warming, impacting its large soil carbon stocks. We use a new compilation of terrestrial ecosystem CO2 fluxes, geospatial datasets and random forest models to show that although the ABZ was an increasing terrestrial CO2 sink from 2001 to 2020 (mean {+/-} standard deviation in net ecosystem exchange: -548 {+/-} 140 Tg C yr-1; trend: -14 Tg C yr-1, p\u003c0.001), more than 30% of the region was a net CO2 source. Tundra regions may have already started to function on average as CO2 sources, demonstrating a critical shift in carbon dynamics. After factoring in fire emissions, the increasing ABZ sink was no longer statistically significant (budget: -319 {+/-} 140 Tg C yr-1; trend: -9 Tg C yr-1), with the permafrost region becoming CO2 neutral (budget: -24 {+/-} 123 Tg C yr-1; trend: -3 Tg C yr-1), underscoring the importance of fire in this region.\nVariable responses of individual species to tropical forest degradation\nAuthors: Ewers, R. M.; Pearse, W. D.; Orme, C. D. L.; Amarasekare, P.; De Lorm, T.; Granville, N.; Adzhar, R.; Aldridge, D. C.; Ancrenaz, M.; Atton, G.; Barclay, H.; Barclay, M. L.; Bernard, H.; Bicknell, J. E.; Bishop, T. R.; Blackman, J.; Both, S.; Boyle, M. W.; Brant, H.; Brasington, E.; Burslem, D.; Bush, E. R.; Calloway, K.; Carbone, C.; Cator, L.; Chapman, P. M.; Chey, V.; Chung, A.; Clare, E. L.; Cusack, J.; Dancak, M.; Davies, Z. G.; Davison, C. W.; Dawood, M. M.; Deere, N. J.; Dickinson, K. M.; Didham, R. K.; Dobert, T. F.; Dow, R. A.; Drinkwater, R.; Edwards, D. P.; Eggleton, P.; Faruk, A.; F\nScore: 2.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.576668\nThe functional stability of ecosystems depends greatly on interspecific differences in responses to environmental perturbation. However, responses to perturbation are not necessarily invariant among populations of the same species, so intraspecific variation in responses might also contribute. Such inter-population response diversity has recently been shown to occur spatially across species ranges, but we lack estimates of the extent to which individual populations across an entire community might have perturbation responses that vary through time. We assess this using 524 taxa that have been repeatedly surveyed for the effects of tropical forest logging at a focal landscape in Sabah, Malaysia. Just 39 % of taxa - all with non-significant responses to forest degradation - had invariant responses. All other taxa (61 %) showed significantly different responses to the same forest degradation gradient across surveys, with 6 % of taxa responding to forest degradation in opposite directions across multiple surveys. Individual surveys had low power (\u003c 80 %) to determine the correct direction of response to forest degradation for one-fifth of all taxa. Recurrent rounds of logging disturbance increased the prevalence of intra-population response diversity, while uncontrollable environmental variation and/or turnover of intraspecific phenotypes generated variable responses in at least 44 % of taxa. Our results show that the responses of individual species to local environmental perturbations are remarkably flexible, likely providing an unrealised boost to the stability of disturbed habitats such as logged tropical forests.\nA Bayesian overhaul of thermal tolerance landscape models: Predicting ectotherm lethality buildup and survival amid heatwaves\nAuthors: Vajedsamiei, J.; Warlo, N.; Meier, H. E. M.; Melzner, F.\nScore: 3.1, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576827\n1. In the face of escalating heatwaves, accurately forecasting ectotherm population mortality is a pressing ecological challenge. Current Thermal Tolerance Landscape (TTL) models, while surpassing single-threshold metrics by incorporating individual survival times, are constrained by frequentist regression parametrization reliant on constant-temperature experiments, omitting probabilistic outcomes. 2. This study addresses these limitations by pioneering the application of Approximate Bayesian Computation-Sequential Monte Carlo (ABC-SMC) to analyze survival data from Baltic Mytilus mussels subjected to both microcosm (constant temperature) and mesocosm (dynamic temperature) heatwave regimes. 3. The ABC-SMC yields probabilistic predictions of individual lethality buildup and population survival trajectories, closely aligned with observed survival data across both experimental conditions. Informed by more realistic dynamic data, the TTL model predicts local mussel resilience against the most extreme summer heatwaves projected for this century, albeit with considerations for sublethal impacts and potential recruitment declines. 4. Our approach can enhance the predictive accuracy concerning the sensitivity of key marine populations amidst intensifying heatwaves, addressing the urgent need for accurate modeling tools to inform conservation practices and ecosystem management, ultimately aiding in the preservation of marine biodiversity.\nA comprehensive evaluation of taxonomic classifiers in marine vertebrate eDNA studies\nAuthors: Bayer, P. E.; Bennett, A.; Nester, G.; Corrigan, S.; Raes, E. J.; McInnes, A. S.; Cooper, M.; Ayad, M. E.; McVey, P.; Kardailsky, A.; Pearce, J.; Fraser, M. W.; Goncalves, P.; Burnell, S.; Rauschert, S.\nScore: 2.5, Published: 2024-02-17 DOI: 10.1101/2024.02.15.580601\nEnvironmental DNA (eDNA) metabarcoding is a widely used tool for surveying marine vertebrate biodiversity. To this end, many computational tools have been released and a plethora of bioinformatic approaches are used for eDNA-based community composition analysis. Simulation studies and careful evaluation of taxonomic classifiers are essential to establish reliable benchmarks to improve accuracy and reproducibility of eDNA-based findings. Here we present a comprehensive evaluation of nine taxonomic classifiers exploring three widely used mitochondrial markers (12S rDNA, 16S rDNA, and COI) in Australian marine vertebrates. Curated reference databases and exclusion database tests were used to simulate diverse species compositions, including three positive control and two negative control datasets. Using these simulated datasets, we were able to identify between 19% to 85% of marine vertebrate species using mitochondrial markers. We show that MMSeqs2 and Metabuli generally outperform BLAST with 10% and 11% higher F1 scores for 12S and 16S rDNA markers, respectively, and that Naive Bayes Classifiers such as Mothur outperform sequence-based classifiers except MMSeqs2 for COI markers by 11%. Database exclusion tests reveal that MMSeqs2 and BLAST are less susceptible to false positives compared to Kraken2 with default parameters. Based on these findings, we recommend that MMSeqs2 is used for taxonomic classification of marine vertebrates given its ability to improve species-level assignments while reducing the number of false positives. Our work contributes to the establishment of best practices in eDNA-based biodiversity analysis to ultimately increase the reliability of this monitoring tool in the context of marine vertebrate conservation.\nInsights into the nature of the microalgal toxins from the Chrysochromulina leadbeateri blooms in Northern Norwegian fjords\nAuthors: Wang, X.; Fon, M.; Andersen, A. J. C.; Solhaug, A.; Ingebrigtsen, R. A.; Samdal, I. A.; Uhlig, S.; Miles, C. O.; Edvardsen, B.; Larsen, T. O.\nScore: 2.5, Published: 2024-02-12 DOI: 10.1101/2024.02.08.576530\nIn May-June 2019, the microalga Chrysochromulina leadbeateri caused a massive fish-killing event in several fjords in Northern Norway, resulting in the largest direct impact ever on aquaculture in northern Europe due to toxic algae. Motivated by the fact that no algal toxins have previously been described from C. leadbeateri, we set out to investigate the chemical nature and toxicity of secondary metabolites in extracts of two strains (UIO 393, UIO 394) isolated from the 2019 bloom, as well as one older strain (UIO 035) isolated during a bloom in Northern Norway in 1991. Initial LC-DAD-MS/MS-based molecular networking analysis of the crude MeOH extracts of the cultivated strains showed that their profiles of small organic molecules, including a large number of known lipids, were very similar, suggesting that the same class of toxin(s) were likely the causative agents of the two harmful algal bloom (HAB) events. Next, bioassay-guided fractionation using the RTgill-W1 cell line and metabolomics analysis pointed to a major compound affording [M+H]+ ions at m/z 1399.8333 as a possible toxin, corresponding to a compound with the formula C67H127ClO27. Moreover, our study unveiled a series of minor analogues exhibiting distinct patterns of chlorination and sulfation, together defining a new family of compounds, which we propose to name leadbeaterins. Remarkably, these suspected toxins were detected in situ in samples collected during the 2019 bloom close to Tromso, thereby substantiating their likely role in fish kills. The elemental compositions of the putative C. leadbeateri ichthyotoxins strongly indicate them to be long linear polyhydroxylated polyketides, structurally similar to sterolysins, reported from a number of dinoflagellates.\n",
  "wordCount" : "2550",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579562">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579562" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579562">
        <p class="paperTitle">Global niche partitioning of purine and pyrimidine cross-feeding among ocean microbes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579562" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579562" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Braakman, R.; Satinsky, B.; O&#39;Keefe, T. J.; Longnecker, K.; Becker, J. W.; Dooley, K.; Hogle, S. L.; Arellano, A.; Kido Soule, M. C.; Kujawinski, E. B.; Chisholm, S. W.</p>
        <p class="info">Score: 16.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579562' target='https://doi.org/10.1101/2024.02.09.579562'> 10.1101/2024.02.09.579562</a></p>
        <p class="abstract">Characterizing the diversity of cross-feeding pathways in ocean microbes illuminates forces shaping co-evolution, ecosystem self-assembly and carbon cycling. Here we uncover a purine and pyrimidine cross-feeding network in globally abundant groups. The cyanobacterium Prochlorococcus exudes both compound classes, which metabolic reconstructions suggest follows synchronous daily genome replication. Co-occurring heterotrophs differentiate into purine or pyrimidine specialists, or generalists, and use compounds for different purposes. The most abundant heterotroph, SAR11, is a specialist that uses purines as sources of energy, carbon and/or nitrogen, with subgroups differentiating along ocean-scale gradients in the supply of energy and nitrogen, in turn leading to putative cryptic nitrogen cycles that link many microbes. Finally, in a SAR11 subgroup that dominates where Prochlorococcus is abundant, adenine additions to cultures inhibit DNA synthesis, poising cells for replication. We argue this subgroup uses inferred daily pulses of adenine from Prochlorococcus to metabolically synchronize to the daily supply of photosynthate from surrounding phytoplankton.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579602" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579602">
        <p class="paperTitle">What unexplored means: Mapping undersampled regions in natural history collections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakrabarty, P.; Ball, L.; Bedoya, A.; Rodriguez Machado, S.; Paredes-Burneo, D.; Rutledge, S.; Boyd, D.; Vander Pluym, D.; Babb-Biernacki, S.; Chipps, A.; Ozturk, R.; Terzi, Y.</p>
        <p class="info">Score: 12.0, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579602' target='https://doi.org/10.1101/2024.02.09.579602'> 10.1101/2024.02.09.579602</a></p>
        <p class="abstract">We examined global records of accessible natural history voucher collections (with publicly available data) for terrestrial and freshwater vascular plants, fungi, freshwater fishes, birds, mammals, and herpetofauna (amphibians and reptiles) and highlight areas of the world that would be considered undersampled and sometimes called &#34;unexplored&#34; (i.e., have relatively low, or no evidence of, past sampling efforts) under typical Western-scientific descriptions. We also question what &#34;unexplored&#34; may actually mean in these contexts and explain how retiring the term in favor of more nuanced phrasing can mitigate future misunderstandings of natural history science.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580411">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580411" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580411">
        <p class="paperTitle">Predictable functional biogeography of marine microbial heterotrophs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580411" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580411" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zakem, E. J.; McNichol, J.; Weissman, J. L.; Raut, Y.; Xu, L.; Halewood, E. R.; Carlson, C. A.; Dutkiewicz, S.; Fuhrman, J. A.; Levine, N. M.</p>
        <p class="info">Score: 6.2, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580411' target='https://doi.org/10.1101/2024.02.14.580411'> 10.1101/2024.02.14.580411</a></p>
        <p class="abstract">Microbial heterotrophs (`picoheterotrophs&#39;) drive global carbon cycling, but how to quantitatively organize their functional complexity remains unclear. Here, we generate a global-scale, mechanistic understanding of marine picoheterotrophic functional biogeography with a novel model-data synthesis. We build picoheterotrophic diversity into a trait-based marine ecosystem model along two axes: substrate lability and optimization for growth rate (copiotrophy) vs. substrate affinity (oligotrophy). Using genetic sequences along an Alaska-to-Antarctica Pacific Ocean transect, we compile 21 picoheterotrophic guilds and estimate their degree of copiotrophy. Data and model agreement suggests that gradients in predation and substrate lability predominantly set biogeographical patterns, and identifies `slow copiotrophs&#39; subsisting at depth. Results demonstrate the predictability of the marine microbiome and connect ecological dynamics with carbon storage, crucial for projecting changes in a warming ocean.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579653" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579653">
        <p class="paperTitle">Time-series metagenomics reveals changing protistan ecology of a temperate dimictic lake</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krinos, A. I.; Bowers, R. M.; Rohwer, R. R.; McMahon, K.; Woyke, T.; Schulz, F.</p>
        <p class="info">Score: 3.8, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579653' target='https://doi.org/10.1101/2024.02.09.579653'> 10.1101/2024.02.09.579653</a></p>
        <p class="abstract">BackgroundProtists, single-celled eukaryotic organisms, are critical to food web ecology, contributing to primary productivity and connecting small bacteria and archaea to higher trophic levels. Lake Mendota is a large, eutrophic natural lake that is a Long-Term Ecological Research site and among the worlds best-studied freshwater systems. Metagenomic samples have been collected and shotgun sequenced from Lake Mendota for the last twenty years. Here, we analyze this comprehensive time series to infer changes to the structure and function of the protistan community, and to hypothesize about their interactions with bacteria.

ResultsBased on small subunit rRNA genes extracted from the metagenomes and metagenome-assembled genomes of microeukaryotes, we identify shifts in the eukaryotic phytoplankton community over time, which we predict to be a consequence of reduced zooplankton grazing pressures after the invasion of a invasive predator (the spiny water flea) to the lake. The metagenomic data also reveal the presence of the spiny water flea and the zebra mussel, a second invasive species to Lake Mendota, prior to their visual identification during routine monitoring. Further, we use species co-occurrence and co-abundance analysis to connect the protistan community with bacterial taxa. Correlation analysis suggests that protists and bacteria may interact or respond similarly to environmental conditions. Cryptophytes declined in the second decade of the timeseries, while many alveolate groups (e.g. ciliates and dinoflagellates) and diatoms increased in abundance, changes that have implications for food web efficiency in Lake Mendota.

ConclusionsWe demonstrate that metagenomic sequence-based community analysis can complement existing eorts to monitor protists in Lake Mendota based on microscopy-based count surveys. We observed patterns of seasonal abundance in microeukaryotes in Lake Mendota that corroborated expectations from other systems, including high abundance of cryptophytes in winter and diatoms in fall and spring, but with much higher resolution than previous surveys. Our study identified long-term changes in the abundance of eukaryotic microbes, and provided context for the known establishment of an invasive species that catalyzes a trophic cascade involving protists. Our findings are important for decoding potential long-term consequences of human interventions, including invasive species introduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.16.580617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.16.580617" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.16.580617">
        <p class="paperTitle">Transinfection of Wolbachia wAlbB into Culex quinquefasciatus mosquitoes does not alter vector competence for Hawaiian avian malaria (Plasmodium relictum GRW4)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.16.580617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.16.580617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kilpatrick, A. M.; Seidl, C. M.; Ipsaro, I. J.; Garrison, C. E.; Fabbri, G.; Howell, P. I.; McGowan, A. G.; White, B. J.; Mitchell, S. N.</p>
        <p class="info">Score: 3.5, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.16.580617' target='https://doi.org/10.1101/2024.02.16.580617'> 10.1101/2024.02.16.580617</a></p>
        <p class="abstract">Avian malaria is expanding upslope with warmer temperatures and driving multiple species of Hawaiian birds towards extinction. Methods to reduce malaria transmission are urgently needed to prevent further declines. Releasing Wolbachia-infected incompatible male mosquitoes suppress mosquito populations and releasing Wolbachia-infected female mosquitoes could reduce pathogen transmission if the Wolbachia strain reduced vector competence. We cleared Culex quinquefasciatus of their natural Wolbachia pipientis wPip infection and transinfected them with Wolbachia wAlbB isolated from Aedes albopictus. We show that wAlbB infection was transmitted transovarially, and demonstrate cytoplasmic incompatibility with wild-type mosquitoes infected with wPip from Oahu and Maui, Hawaii. We measured vector competence for avian malaria, Plasmodium relictum, lineage GRW4, of seven mosquito lines (two with wAlbB; three with natural wPip infection, and two cleared of Wolbachia infection) by allowing them to feed on canaries infected with recently collected field isolates of Hawaiian P. relictum. We tested 73 groups (Ntotal = 1176) of mosquitoes for P. relictum infection in abdomens and disseminated (thorax) infections 6-14 days after feeding across a range of parasitemias from 0.028% to 2.49%, and a smaller subset of salivary glands. We found no measurable effect of Wolbachia on any endpoint, but strong effects of parasitemia, days post feeding, and mosquito strain on both abdomen infection prevalence and disseminated infection prevalence. These results suggest that releasing male wAlbB-infected C. quinquefasciatus mosquitoes could suppress wPip-infected mosquito populations, but would have little positive or negative impact on mosquito vector competence for P. relictum if wAlbB became established in local mosquito populations. More broadly, the lack of Wolbachia effects on vector competence we observed highlights the variable impacts of both native and transfected Wolbachia infections in mosquitoes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579581">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579581" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579581">
        <p class="paperTitle">An increasing Arctic-Boreal CO2 sink despite strong regional sources</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579581" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579581" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Virkkala, A.-M.</p>
        <p class="info">Score: 2.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579581' target='https://doi.org/10.1101/2024.02.09.579581'> 10.1101/2024.02.09.579581</a></p>
        <p class="abstract">The Arctic-Boreal Zone (ABZ) is rapidly warming, impacting its large soil carbon stocks. We use a new compilation of terrestrial ecosystem CO2 fluxes, geospatial datasets and random forest models to show that although the ABZ was an increasing terrestrial CO2 sink from 2001 to 2020 (mean {&#43;/-} standard deviation in net ecosystem exchange: -548 {&#43;/-} 140 Tg C yr-1; trend: -14 Tg C yr-1, p&lt;0.001), more than 30% of the region was a net CO2 source. Tundra regions may have already started to function on average as CO2 sources, demonstrating a critical shift in carbon dynamics. After factoring in fire emissions, the increasing ABZ sink was no longer statistically significant (budget: -319 {&#43;/-} 140 Tg C yr-1; trend: -9 Tg C yr-1), with the permafrost region becoming CO2 neutral (budget: -24 {&#43;/-} 123 Tg C yr-1; trend: -3 Tg C yr-1), underscoring the importance of fire in this region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.576668">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.576668" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.576668">
        <p class="paperTitle">Variable responses of individual species to tropical forest degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.576668" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.576668" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ewers, R. M.; Pearse, W. D.; Orme, C. D. L.; Amarasekare, P.; De Lorm, T.; Granville, N.; Adzhar, R.; Aldridge, D. C.; Ancrenaz, M.; Atton, G.; Barclay, H.; Barclay, M. L.; Bernard, H.; Bicknell, J. E.; Bishop, T. R.; Blackman, J.; Both, S.; Boyle, M. W.; Brant, H.; Brasington, E.; Burslem, D.; Bush, E. R.; Calloway, K.; Carbone, C.; Cator, L.; Chapman, P. M.; Chey, V.; Chung, A.; Clare, E. L.; Cusack, J.; Dancak, M.; Davies, Z. G.; Davison, C. W.; Dawood, M. M.; Deere, N. J.; Dickinson, K. M.; Didham, R. K.; Dobert, T. F.; Dow, R. A.; Drinkwater, R.; Edwards, D. P.; Eggleton, P.; Faruk, A.; F</p>
        <p class="info">Score: 2.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.576668' target='https://doi.org/10.1101/2024.02.09.576668'> 10.1101/2024.02.09.576668</a></p>
        <p class="abstract">The functional stability of ecosystems depends greatly on interspecific differences in responses to environmental perturbation. However, responses to perturbation are not necessarily invariant among populations of the same species, so intraspecific variation in responses might also contribute. Such inter-population response diversity has recently been shown to occur spatially across species ranges, but we lack estimates of the extent to which individual populations across an entire community might have perturbation responses that vary through time. We assess this using 524 taxa that have been repeatedly surveyed for the effects of tropical forest logging at a focal landscape in Sabah, Malaysia. Just 39 % of taxa - all with non-significant responses to forest degradation - had invariant responses. All other taxa (61 %) showed significantly different responses to the same forest degradation gradient across surveys, with 6 % of taxa responding to forest degradation in opposite directions across multiple surveys. Individual surveys had low power (&lt; 80 %) to determine the correct direction of response to forest degradation for one-fifth of all taxa. Recurrent rounds of logging disturbance increased the prevalence of intra-population response diversity, while uncontrollable environmental variation and/or turnover of intraspecific phenotypes generated variable responses in at least 44 % of taxa. Our results show that the responses of individual species to local environmental perturbations are remarkably flexible, likely providing an unrealised boost to the stability of disturbed habitats such as logged tropical forests.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576827" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576827">
        <p class="paperTitle">A Bayesian overhaul of thermal tolerance landscape models: Predicting ectotherm lethality buildup and survival amid heatwaves</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vajedsamiei, J.; Warlo, N.; Meier, H. E. M.; Melzner, F.</p>
        <p class="info">Score: 3.1, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576827' target='https://doi.org/10.1101/2024.01.23.576827'> 10.1101/2024.01.23.576827</a></p>
        <p class="abstract">1. In the face of escalating heatwaves, accurately forecasting ectotherm population mortality is a pressing ecological challenge. Current Thermal Tolerance Landscape (TTL) models, while surpassing single-threshold metrics by incorporating individual survival times, are constrained by frequentist regression parametrization reliant on constant-temperature experiments, omitting probabilistic outcomes.

2. This study addresses these limitations by pioneering the application of Approximate Bayesian Computation-Sequential Monte Carlo (ABC-SMC) to analyze survival data from Baltic Mytilus mussels subjected to both microcosm (constant temperature) and mesocosm (dynamic temperature) heatwave regimes.

3. The ABC-SMC yields probabilistic predictions of individual lethality buildup and population survival trajectories, closely aligned with observed survival data across both experimental conditions. Informed by more realistic dynamic data, the TTL model predicts local mussel resilience against the most extreme summer heatwaves projected for this century, albeit with considerations for sublethal impacts and potential recruitment declines.

4. Our approach can enhance the predictive accuracy concerning the sensitivity of key marine populations amidst intensifying heatwaves, addressing the urgent need for accurate modeling tools to inform conservation practices and ecosystem management, ultimately aiding in the preservation of marine biodiversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580601">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580601" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580601">
        <p class="paperTitle">A comprehensive evaluation of taxonomic classifiers in marine vertebrate eDNA studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580601" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580601" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bayer, P. E.; Bennett, A.; Nester, G.; Corrigan, S.; Raes, E. J.; McInnes, A. S.; Cooper, M.; Ayad, M. E.; McVey, P.; Kardailsky, A.; Pearce, J.; Fraser, M. W.; Goncalves, P.; Burnell, S.; Rauschert, S.</p>
        <p class="info">Score: 2.5, Published: 2024-02-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580601' target='https://doi.org/10.1101/2024.02.15.580601'> 10.1101/2024.02.15.580601</a></p>
        <p class="abstract">Environmental DNA (eDNA) metabarcoding is a widely used tool for surveying marine vertebrate biodiversity. To this end, many computational tools have been released and a plethora of bioinformatic approaches are used for eDNA-based community composition analysis. Simulation studies and careful evaluation of taxonomic classifiers are essential to establish reliable benchmarks to improve accuracy and reproducibility of eDNA-based findings. Here we present a comprehensive evaluation of nine taxonomic classifiers exploring three widely used mitochondrial markers (12S rDNA, 16S rDNA, and COI) in Australian marine vertebrates. Curated reference databases and exclusion database tests were used to simulate diverse species compositions, including three positive control and two negative control datasets. Using these simulated datasets, we were able to identify between 19% to 85% of marine vertebrate species using mitochondrial markers. We show that MMSeqs2 and Metabuli generally outperform BLAST with 10% and 11% higher F1 scores for 12S and 16S rDNA markers, respectively, and that Naive Bayes Classifiers such as Mothur outperform sequence-based classifiers except MMSeqs2 for COI markers by 11%. Database exclusion tests reveal that MMSeqs2 and BLAST are less susceptible to false positives compared to Kraken2 with default parameters. Based on these findings, we recommend that MMSeqs2 is used for taxonomic classification of marine vertebrates given its ability to improve species-level assignments while reducing the number of false positives. Our work contributes to the establishment of best practices in eDNA-based biodiversity analysis to ultimately increase the reliability of this monitoring tool in the context of marine vertebrate conservation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.576530">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.576530" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.576530">
        <p class="paperTitle">Insights into the nature of the microalgal toxins from the Chrysochromulina leadbeateri blooms in Northern Norwegian fjords</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.576530" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.576530" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, X.; Fon, M.; Andersen, A. J. C.; Solhaug, A.; Ingebrigtsen, R. A.; Samdal, I. A.; Uhlig, S.; Miles, C. O.; Edvardsen, B.; Larsen, T. O.</p>
        <p class="info">Score: 2.5, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.576530' target='https://doi.org/10.1101/2024.02.08.576530'> 10.1101/2024.02.08.576530</a></p>
        <p class="abstract">In May-June 2019, the microalga Chrysochromulina leadbeateri caused a massive fish-killing event in several fjords in Northern Norway, resulting in the largest direct impact ever on aquaculture in northern Europe due to toxic algae. Motivated by the fact that no algal toxins have previously been described from C. leadbeateri, we set out to investigate the chemical nature and toxicity of secondary metabolites in extracts of two strains (UIO 393, UIO 394) isolated from the 2019 bloom, as well as one older strain (UIO 035) isolated during a bloom in Northern Norway in 1991. Initial LC-DAD-MS/MS-based molecular networking analysis of the crude MeOH extracts of the cultivated strains showed that their profiles of small organic molecules, including a large number of known lipids, were very similar, suggesting that the same class of toxin(s) were likely the causative agents of the two harmful algal bloom (HAB) events. Next, bioassay-guided fractionation using the RTgill-W1 cell line and metabolomics analysis pointed to a major compound affording [M&#43;H]&#43; ions at m/z 1399.8333 as a possible toxin, corresponding to a compound with the formula C67H127ClO27. Moreover, our study unveiled a series of minor analogues exhibiting distinct patterns of chlorination and sulfation, together defining a new family of compounds, which we propose to name leadbeaterins. Remarkably, these suspected toxins were detected in situ in samples collected during the 2019 bloom close to Tromso, thereby substantiating their likely role in fish kills. The elemental compositions of the putative C. leadbeateri ichthyotoxins strongly indicate them to be long linear polyhydroxylated polyketides, structurally similar to sterolysins, reported from a number of dinoflagellates.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
