<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland
Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.
Score: 10.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570
Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland
Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.
Score: 10.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570
Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland
Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.
Score: 10.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570
Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland\nAuthors: Curran, T. G.; O\u0026#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O\u0026#39;Neill, D.; McDevitt, A.; O\u0026#39;Reilly, C.; O\u0026#39;Meara, D. B.\nScore: 10.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570\nArthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase.",
  "keywords": [
    
  ],
  "articleBody": " DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland\nAuthors: Curran, T. G.; O'Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O'Neill, D.; McDevitt, A.; O'Reilly, C.; O'Meara, D. B.\nScore: 10.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572570\nArthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase. Such dynamics lead to weakened ecosystem resilience, which can facilitate the establishment and spread of species with vector capabilities, such as mosquitoes. While many of the mosquito species present in Ireland possess vector capabilities, populations have not been nationally surveyed since 1991. At that time, species identification relied on morphological examination, a method that was even then recognised as inadequate for differentiating certain species within complexes. In this study, we present the first DNA barcoding investigation of Irish mosquito populations and reveal species and hidden diversity not previously recognised in Ireland. These include two potential new species/ecotypes, Culex torrentium and Culex pipiens form pipiens, which were recorded for the first time in Ireland. We demonstrate that barcoding with Cytochrome C Oxidase Subunit 1 (COI) based primers is suitable for the majority of species-level identifications. However, additional primer sets targeting the Internal Transcribed Spacer regions 1 and 2 (ITS1 and ITS2) were necessary to differentiate species within the Culex pipiens complex. The multi-locus approach employed in this study can enhance national surveillance efforts, especially in monitoring mosquitoes that may transmit vector-borne diseases, and in recognising increased species diversity from a biodiversity perspective.\nUnderstanding interactions in a complex world: Design and analysis of multi-species functional response experiments\nAuthors: Rosenbaum, B.; Li, J.; Hirt, M. R.; Ryser, R.; Brose, U.\nScore: 5.3, Published: 2023-12-19 DOI: 10.1101/2023.12.19.571428\nO_LIThe functional response describes resource-dependent feeding rates of consumers. While models for feeding on a single resource species are well studied and supported by a large body of empirical research, consumers feeding on multiple resource species are ubiquitous in nature. However, feeding experiments designed for parameterizing multi-species functional responses (MSFR) are extremely rare, mainly due to logistical challenges and the non-trivial nature of their statistical analysis. C_LIO_LIHere, we describe how these models can be fitted to empirical data in a Bayesian framework. Specifically, we address the problem of prey depletion during experiments, which can be accounted for through dynamical modeling. In a comprehensive simulation study, we test the effects of experimental design, sample size and noise level on the identifiability of four distinct MSFR models. Additionally, we demonstrate the methods versatility by applying it to a list of empirical datasets. C_LIO_LIWe identify experimental designs for feeding trials that produce the most accurate parameter estimates in two- and three-prey scenarios. Although noise can introduce systematic bias in parameter estimates, model selection performs surprisingly well, almost always identifying the correct model even for small datasets. C_LIO_LIOur flexible method allows the simultaneous analysis of feeding experiments from both single- and multi-prey scenarios, either with or without prey depletion. This will help to improve our understanding of prey selectivity, prey switching and hence food web stability and biodiversity. Our approach equips researchers with the right statistical tools to improve the mechanistic understanding of feeding interactions in complex ecosystems. C_LI\nIncreasing marine trophic web knowledge through DNA analyses of fish stomach contents: a step towards an Ecosystem Based Approach to fisheries research\nAuthors: Canals, O.; Lanzen, A.; Mendibil, I.; Bachiller, E.; Corrales, X.; Andonegi, E.; Cotano, U.; Rodriguez-Ezpeleta, N.\nScore: 5.3, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572137\nMultispecies and ecosystem models, which are key for the implementation of ecosystem-based approaches to fisheries management, require extensive data on the trophic interactions between marine organisms, including changes over time. DNA metabarcoding (the simultaneous identification of all taxa in a sample by sequencing a short DNA region) could be used for speeding up large-scaled stomach content data collection. Yet, for DNA metabarcoding to be routinely implemented, technical challenges should be addressed, such as the potentially complicated sampling logistics, the detection of a high proportion of predator DNA, and the inability to provide reliable abundance estimations. Here, we present a DNA metabarcoding assay developed to examine the diet of five commercially important fish, which is feasible to be incorporated into routinary samplings. The method is devised to speed up the analysis process by avoiding the stomach dissection and content extraction steps, while preventing the amplification of predator DNA by using blocking primers. Tested in mock samples and in real stomachs, the method has proven effective and shows great effectiveness discerning diet variations due to predator ecology or prey availability. Additionally, by applying our protocol to mackerel stomachs previously analysed by visual inspection, we showcase how DNA metabarcoding could complement visually based data by detecting overlooked prey by the visual approach. We finally discuss how DNA metabarcoding-based data can contribute to trophic data collection. Our work reinforces the potential of DNA metabarcoding for the study and monitoring of fish trophic interactions and provides a basis for its incorporation into routine monitoring programs, which will be critical for the implementation of ecosystem-based approaches to fisheries management.\nDining in the daylight: Nocturnal rhinoceros beetles extend feeding periods on host trees with reduced sap exudation\nAuthors: Kanda, A.; Shibata, R.; Ueno, T.; Yamashita, N.; Kojima, W.\nScore: 3.5, Published: 2023-12-25 DOI: 10.1101/2023.12.25.573284\nWhile the Japanese rhinoceros beetle Trypoxylus dichotomus typically feeds on the sap of the oak Quercus acutissima and the crape myrtle Lagerstroemia subcostata during the night, it exhibits feeding activity both during the day and night when it utilizes the ash tree Fraxinus griffithii. However, the mechanisms underlying the variations in temporal activity patterns remain unknown. We compared feeding rates (measured as body mass increments) and sap exudation rates among F. griffithii, Q. acutissima, and L. subcostata. We found that beetles feeding on L. subcostata and Q. acutissima exhibited significantly higher feeding rates than those feeding on F. griffithii. No significant differences in feeding rates were observed between L. subcostata and Q. acutissima. The sap exudation rate was significantly higher for Q. acutissima than for F. griffithii. However, there were no significant differences in the sap exudation rates between F. griffithii and L. subcostata or between Q. acutissima and L. subcostata. These findings suggest that lower feeding rates on F. griffithii prolong the feeding duration, resulting in daytime activity. While the low sap exudation in L. subcostata seems inconsistent with high feeding rates on this host, this apparent contradiction could be related to the extended duration of sap exudation.\nSpecies interactions drive continuous assembly of freshwater communities in stochastic environments\nAuthors: Tabi, A.; Siqueira, T.; Tonkin, J. D.\nScore: 3.5, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572946\nUnderstanding the factors driving the maintenance of long-term biodiversity in changing environments is essential for improving restoration and sustainability strategies in the face of global environmental change. Biodiversity is shaped by both niche and stochastic processes, however the strength of deterministic processes in unpredictable environmental regimes is highly debated. Since communities continuously change over time and space - species persist, disappear or (re)appear - understanding the drivers of species gains and losses from communities should inform us about whether niche or stochastic processes dominate community dynamics. Applying a nonparametric causal discovery approach to a 30-year time series containing annual abundances of benthic invertebrates across 66 locations in New Zealand rivers, we found a strong asynchronous causal relationship between species gains and losses directly driven by predation indicating that niche processes dominate community dynamics. Despite the unpredictable nature of these system, environmental noise was only indirectly related to species gains and losses through altering life history trait distribution. Using a stochastic birth-death framework, we demonstrate that the negative relationship between species gains and losses can not emerge without strong niche processes. Our results showed that even in systems that are dominated by unpredictable environmental variability, species interactions drive continuous community assembly.\nTrophic microRNA: Precursor and mature microRNA ingestion downregulates the target transcripts and hampers larval development in Plutella xylostella\nAuthors: Bardapurkar, R.; Pandit, S.\nScore: 3.0, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572757\nMicroRNAs (miRNAs) are post-transcriptional gene regulators. In the miRNA pathway's cytoplasmic part, the miRNA is processed from a hairpin-structured precursor (pre-miRNA) to a double-stranded (ds) mature RNA and ultimately to a single-stranded mature miRNA. In insects, ingesting these two ds forms can regulate the target gene expression; this inspired the trophic miRNA's use as a functional genomics and pest management tool. However, systematic studies enabling comparisons of pre- and mature forms, dosages, administration times, and instar-wise effects on target transcripts and phenotypes, which can help develop a miRNA administration method, are unavailable due to the different focuses of the previous investigations. We investigated the impact of trophically delivered Px-let-7 miRNA on the Brassicaceae pest, Plutella xylostella, to compare the efficacies of its pre- and ds-mature forms. Continuous feeding on the miRNA-supplemented diet suppressed expressions of FTZ-F1 and E74, the target ecdysone pathway genes. Both the pre-let-7 and mature let-7 miRNA forms similarly downregulated the target transcripts in all four larval instars. Pre-let-7 and let-7 ingestions decreased P. xylostella's larval mass and instar duration and increased mortality in all instars, exhibiting adverse effects on larval growth and development. Pre-miRNA processing Dicer-1 was upregulated upon pre-let-7 ingestion, indicating the miRNA uptake by the midgut cells. The scrambled sequence controls did not affect the target transcripts, suggesting the sequence-specific targeting by the mature miRNA and hairpin cassette's non-involvement in the target downregulation. This work provides a framework for miRNA and target gene function analyses and potentiates the trophic miRNA's utility in pest management.\nEnvironmental DNA reveals fine scale spatial and temporal variation of prey species for marine mammals in a Scottish marine protected area\nAuthors: Boyse, E.; Robinson, K. P.; Beger, M.; Carr, I. M.; Taylor, M.; Valsecchi, E.; Goodman, S. J.\nScore: 2.8, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572838\nMarine mammal foraging grounds are popular focal points for marine protected area (MPA) implementation, but may be temporally dynamic, requiring continuous monitoring to infer prey availability and abundance. Marine mammal distributions are assumed to be driven by their prey in foraging areas, but limited understanding of prey distributions often prevents us from exploring how shifting prey availability impacts both seasonal and long-term marine mammal distributions. Environmental DNA (eDNA) metabarcoding could enhance understanding of marine mammal habitat use in relation to their prey through simultaneous monitoring of both. However, eDNA applications focused on marine mammals or predator-prey dynamics have been limited to date. In this study, we assess spatiotemporal changes in the availability and abundance of minke whale (Balaenoptera acutorostrata) prey species in a newly established MPA, employing eDNA metabarcoding. We recovered 105 molecular operational taxonomic units (OTUs) from marine vertebrates using two primer sets targeting 12S and 16S genes, along with 112 OTUs from a broader eukaryotic primer set targeting 18S rRNA. Overall, key forage fish prey species, sandeels and clupeids, were the most abundant teleost fishes detected, although their availability varied temporally and with distance from shore. We also found clear spatial partitioning between coastal bottlenose dolphins and the more pelagic minke whales and harbour porpoises, paralleling availability of their main prey species. Other species of conservation interest were also detected including the critically endangered European eel (Anguilla anguilla), blue fin tuna (Thunnus thynnus), and the invasive pink salmon (Oncorhynchus gorbuscha). This study demonstrates the application of eDNA to detect spatiotemporal trends in the occurrence and abundance of cetacean predators and their prey, furthering our understanding of fine-scale habitat use within MPAs. Future, long-term monitoring of predator-prey dynamics with eDNA could improve our ability to predict climate-induced shifts in foraging grounds and enhance rapid responses with appropriate management actions.\nConcerns on research history overview, distribution patterns, and potentially neglected ecological consequences of the invasive apple snail (Pomacea sp.): A review\nAuthors: Luo, D.; Li, Y.; Li, J.\nScore: 2.0, Published: 2023-12-24 DOI: 10.1101/2023.12.23.573227\nInvasive species pose a threat to global ecosystems and biodiversity, with the apple snail standing out as one kind of the worst invasive species due to its exclusive expansion. In order to draw a conceptual framework of its invasion pattern from an overview, we endeavor to conduct a bibliometric analysis and systematic review to explore the existing knowledge on research history and subjective categorization of Pomacea canaliculata and P. maculata, and to uncover emerging trends of distribution and possible neglected impact on food webs. The research history reveals that the first international research publication was documented in 1958. However, a substantial increase in international publications occurred only after 2000. The adoption and effective implementation of a universally agreed upon management strategy may greatly enhance the progress of research on apple snail invasion. Notably, co-word analysis revealed that the content of each thematic cluster has a shift from basic biology to the monitoring of occurrences and environmental adaptation. Currently, the global distribution of apple snails has expanded to 39 countries primarily as a result of anthropogenic translocation. All infested countries, with the exception of Israel and Singapore, are major paddy rice producers. The northernmost distribution limit is Tyumen in Russia, while the southernmost limit is the Buenos Aires Province in Argentina. Biological control has emerged as a prominent technology for the prevention of apple snail infestations, leading to the identification of 98 non-target enemy species as potential control agents. A taxonomic diversity analysis revealed that these species encompassed a wide range of phylogenetic classifications, including 6 Phyla, 12 Classes, 33 Orders, 55 Families, and 82 Genera. This observation suggests that there is a substantial pool of animals from diverse phylogenetic levels that can prey upon apple snails. Given the high population density and extensive distribution of apple snails, there is a pressing need for further research to explore their potential ecological ramifications. The current understanding of the upward ecological impacts of apple snails remains inadequate. These findings provide valuable insights for future research endeavors and contribute to the effective management of invasive apple snail populations.\nMetaproteogenomics resolution of a high-CO2 aquifer community suggests an active symbiotic lifestyle of groundwater Gracilibacteria\nAuthors: Figueroa-Gonzalez, P. A.; Bornemann, T. L. V.; Hinzke, T.; Maass, S.; Trautwein-Schult, A.; Starke, J.; Moore, C. J.; Esser, S. P.; Plewka, J.; Hesse, T.; Schmidt, T. C.; Schreiber, U.; Bor, B.; Becher, D.; Probst, A. J.\nScore: 7.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572140\nBackgroundBacteria of the Candidate Phyla Radiation (CPR), constituting about 25% of the bacterial biodiversity, are characterized by small cell size and patchy genomes without complete key metabolic pathways, suggesting a symbiotic lifestyle. Gracilibacteria (BD1-5), which are part of the CPR branch, possess alternate coded genomes and have not yet been cultivated. However, besides genomic evidence, little is known about the lifestyle of Gracilibacteria, their temporal dynamics, and activity in natural ecosystems, particularly in groundwater, where they were initially been genomically resolved. Therefore, we here aimed to investigate Gracilibacteria activity in situ and to discern expressed genes involved in their lifestyle, using the metaproteogenome of Gracilibacteria as a function of time in the cold-water geyser Wallender Born in the Volcanic Eifel region in Germany. ResultsWe coupled genome-resolved metagenomics and metaproteomics to investigate a cold-water geyser microbial community enriched in Gracilibacteria across a 12-day time-series. Groundwater was collected and sequentially filtered to fraction CPR and other bacteria. Based on 670 Gbps of metagenomic data, 1129 different ribosomal protein S3 marker genes and 751 high-quality genomes (123 population genomes after dereplication), we identified dominant bacteria belonging to Galionellales and Gracilibacteria along with keystone microbes, which were low in genomic abundance but substantially contributing to proteomic abundance. Seven high-quality Gracilibacteria genomes showed typical limitations, such as limited amino acid or nucleotide synthesis, in their central metabolism but no co-occurrence with potential hosts. The genomes of these Gracilibacteria encoded for a high number of proteins related to a symbiotic or even predatory lifestyle, e.g., type IV and type II secretion system subunits and features related to cell-cell interactions and cell motility, which were also detected on protein level. ConclusionsCoupling metagenomics to metaproteomics enabled us to identify microbial keystone taxa in a high-CO2 aquifer, and to reveal microbial dynamics of Gracilibacteria. We posit that Gracilibacteria might be successful microbial predators in this ecosystem, potentially aiding in population control of this highly perturbed microbial geyser community from the deep biosphere.\nWinners in good times and bad times: Aerobic anoxygenic phototrophic bacteria profit from photoheterotrophy under carbon-rich and poor conditions\nAuthors: Piwosz, K.; Villena-Alemany, C.; Calkiewicz, J.; Mujakic, I.; Nahlik, V.; Dean, J.; Koblizek, M.\nScore: 1.2, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572764\nAerobic Anoxygenic Phototrophic (AAP) bacteria are an important component of freshwater bacterioplankton. They can support their heterotrophic metabolism with energy from light, and by that enhance their growth efficiency. Based on results from cultures, it was hypothesized that photoheterotrophy provides an advantage under carbon limitation and facilitates access to recalcitrant or low-energy carbon sources. However, verification of these hypotheses for natural AAP communities has been lacking. Here, we conducted whole community manipulation experiments and compared the growth of AAP bacteria under carbon limited and with recalcitrant or low-energy carbon sources under dark and light conditions to elucidate how they profit from photoheterotrophy. We found that it depends on the season. In spring, AAP bacteria induce photoheterotrophic metabolism under carbon limitation but they outperform heterotrophic bacteria when carbon is available. This effect seems to be driven by physiological responses rather than changes at the community level. In autumn photoheterotrophy is less beneficial. In both seasons, AAP bacteria responded negatively to recalcitrant or low-energy carbon sources in light. This unexpected observation may have ecosystem-level consequences as lake browning continues. In general, our findings contribute to the understanding of the dynamics of AAP bacteria observed in pelagic environments.\n",
  "wordCount" : "2945",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572570">
        <p class="paperTitle">DNA barcoding reveals an increased diversity within the genus Culex (Diptera: Culicidae) in Ireland</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Curran, T. G.; O&#39;Hanlon, A.; Gallagher, F.; Pedersen, R. S.; Zintl, A.; Isiye, E.; Olmeda, A. V.; O&#39;Neill, D.; McDevitt, A.; O&#39;Reilly, C.; O&#39;Meara, D. B.</p>
        <p class="info">Score: 10.2, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572570' target='https://doi.org/10.1101/2023.12.20.572570'> 10.1101/2023.12.20.572570</a></p>
        <p class="abstract">Arthropod populations are changing globally due to the combined impacts of climate change, land-use transformation, globalisation and trade. Changes in arthropod populations can potentially result in the decline and extinction of some native and/or endemic species, while other species that potentially pose threats may increase. Such dynamics lead to weakened ecosystem resilience, which can facilitate the establishment and spread of species with vector capabilities, such as mosquitoes. While many of the mosquito species present in Ireland possess vector capabilities, populations have not been nationally surveyed since 1991. At that time, species identification relied on morphological examination, a method that was even then recognised as inadequate for differentiating certain species within complexes. In this study, we present the first DNA barcoding investigation of Irish mosquito populations and reveal species and hidden diversity not previously recognised in Ireland. These include two potential new species/ecotypes, Culex torrentium and Culex pipiens form pipiens, which were recorded for the first time in Ireland. We demonstrate that barcoding with Cytochrome C Oxidase Subunit 1 (COI) based primers is suitable for the majority of species-level identifications. However, additional primer sets targeting the Internal Transcribed Spacer regions 1 and 2 (ITS1 and ITS2) were necessary to differentiate species within the Culex pipiens complex. The multi-locus approach employed in this study can enhance national surveillance efforts, especially in monitoring mosquitoes that may transmit vector-borne diseases, and in recognising increased species diversity from a biodiversity perspective.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.571428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.571428" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.571428">
        <p class="paperTitle">Understanding interactions in a complex world: Design and analysis of multi-species functional response experiments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.571428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.571428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosenbaum, B.; Li, J.; Hirt, M. R.; Ryser, R.; Brose, U.</p>
        <p class="info">Score: 5.3, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.571428' target='https://doi.org/10.1101/2023.12.19.571428'> 10.1101/2023.12.19.571428</a></p>
        <p class="abstract">O_LIThe functional response describes resource-dependent feeding rates of consumers. While models for feeding on a single resource species are well studied and supported by a large body of empirical research, consumers feeding on multiple resource species are ubiquitous in nature. However, feeding experiments designed for parameterizing multi-species functional responses (MSFR) are extremely rare, mainly due to logistical challenges and the non-trivial nature of their statistical analysis.
C_LIO_LIHere, we describe how these models can be fitted to empirical data in a Bayesian framework. Specifically, we address the problem of prey depletion during experiments, which can be accounted for through dynamical modeling. In a comprehensive simulation study, we test the effects of experimental design, sample size and noise level on the identifiability of four distinct MSFR models. Additionally, we demonstrate the methods versatility by applying it to a list of empirical datasets.
C_LIO_LIWe identify experimental designs for feeding trials that produce the most accurate parameter estimates in two- and three-prey scenarios. Although noise can introduce systematic bias in parameter estimates, model selection performs surprisingly well, almost always identifying the correct model even for small datasets.
C_LIO_LIOur flexible method allows the simultaneous analysis of feeding experiments from both single- and multi-prey scenarios, either with or without prey depletion. This will help to improve our understanding of prey selectivity, prey switching and hence food web stability and biodiversity. Our approach equips researchers with the right statistical tools to improve the mechanistic understanding of feeding interactions in complex ecosystems.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572137">
        <p class="paperTitle">Increasing marine trophic web knowledge through DNA analyses of fish stomach contents: a step towards an Ecosystem Based Approach to fisheries research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Canals, O.; Lanzen, A.; Mendibil, I.; Bachiller, E.; Corrales, X.; Andonegi, E.; Cotano, U.; Rodriguez-Ezpeleta, N.</p>
        <p class="info">Score: 5.3, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572137' target='https://doi.org/10.1101/2023.12.18.572137'> 10.1101/2023.12.18.572137</a></p>
        <p class="abstract">Multispecies and ecosystem models, which are key for the implementation of ecosystem-based approaches to fisheries management, require extensive data on the trophic interactions between marine organisms, including changes over time. DNA metabarcoding (the simultaneous identification of all taxa in a sample by sequencing a short DNA region) could be used for speeding up large-scaled stomach content data collection. Yet, for DNA metabarcoding to be routinely implemented, technical challenges should be addressed, such as the potentially complicated sampling logistics, the detection of a high proportion of predator DNA, and the inability to provide reliable abundance estimations. Here, we present a DNA metabarcoding assay developed to examine the diet of five commercially important fish, which is feasible to be incorporated into routinary samplings. The method is devised to speed up the analysis process by avoiding the stomach dissection and content extraction steps, while preventing the amplification of predator DNA by using blocking primers. Tested in mock samples and in real stomachs, the method has proven effective and shows great effectiveness discerning diet variations due to predator ecology or prey availability. Additionally, by applying our protocol to mackerel stomachs previously analysed by visual inspection, we showcase how DNA metabarcoding could complement visually based data by detecting overlooked prey by the visual approach. We finally discuss how DNA metabarcoding-based data can contribute to trophic data collection. Our work reinforces the potential of DNA metabarcoding for the study and monitoring of fish trophic interactions and provides a basis for its incorporation into routine monitoring programs, which will be critical for the implementation of ecosystem-based approaches to fisheries management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.25.573284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.25.573284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.25.573284">
        <p class="paperTitle">Dining in the daylight: Nocturnal rhinoceros beetles extend feeding periods on host trees with reduced sap exudation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.25.573284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.25.573284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kanda, A.; Shibata, R.; Ueno, T.; Yamashita, N.; Kojima, W.</p>
        <p class="info">Score: 3.5, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.25.573284' target='https://doi.org/10.1101/2023.12.25.573284'> 10.1101/2023.12.25.573284</a></p>
        <p class="abstract">While the Japanese rhinoceros beetle Trypoxylus dichotomus typically feeds on the sap of the oak Quercus acutissima and the crape myrtle Lagerstroemia subcostata during the night, it exhibits feeding activity both during the day and night when it utilizes the ash tree Fraxinus griffithii. However, the mechanisms underlying the variations in temporal activity patterns remain unknown. We compared feeding rates (measured as body mass increments) and sap exudation rates among F. griffithii, Q. acutissima, and L. subcostata. We found that beetles feeding on L. subcostata and Q. acutissima exhibited significantly higher feeding rates than those feeding on F. griffithii. No significant differences in feeding rates were observed between L. subcostata and Q. acutissima. The sap exudation rate was significantly higher for Q. acutissima than for F. griffithii. However, there were no significant differences in the sap exudation rates between F. griffithii and L. subcostata or between Q. acutissima and L. subcostata. These findings suggest that lower feeding rates on F. griffithii prolong the feeding duration, resulting in daytime activity. While the low sap exudation in L. subcostata seems inconsistent with high feeding rates on this host, this apparent contradiction could be related to the extended duration of sap exudation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572946">
        <p class="paperTitle">Species interactions drive continuous assembly of freshwater communities in stochastic environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tabi, A.; Siqueira, T.; Tonkin, J. D.</p>
        <p class="info">Score: 3.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572946' target='https://doi.org/10.1101/2023.12.21.572946'> 10.1101/2023.12.21.572946</a></p>
        <p class="abstract">Understanding the factors driving the maintenance of long-term biodiversity in changing environments is essential for improving restoration and sustainability strategies in the face of global environmental change. Biodiversity is shaped by both niche and stochastic processes, however the strength of deterministic processes in unpredictable environmental regimes is highly debated. Since communities continuously change over time and space - species persist, disappear or (re)appear - understanding the drivers of species gains and losses from communities should inform us about whether niche or stochastic processes dominate community dynamics. Applying a nonparametric causal discovery approach to a 30-year time series containing annual abundances of benthic invertebrates across 66 locations in New Zealand rivers, we found a strong asynchronous causal relationship between species gains and losses directly driven by predation indicating that niche processes dominate community dynamics. Despite the unpredictable nature of these system, environmental noise was only indirectly related to species gains and losses through altering life history trait distribution. Using a stochastic birth-death framework, we demonstrate that the negative relationship between species gains and losses can not emerge without strong niche processes. Our results showed that even in systems that are dominated by unpredictable environmental variability, species interactions drive continuous community assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572757">
        <p class="paperTitle">Trophic microRNA: Precursor and mature microRNA ingestion downregulates the target transcripts and hampers larval development in Plutella xylostella</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bardapurkar, R.; Pandit, S.</p>
        <p class="info">Score: 3.0, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572757' target='https://doi.org/10.1101/2023.12.21.572757'> 10.1101/2023.12.21.572757</a></p>
        <p class="abstract">MicroRNAs (miRNAs) are post-transcriptional gene regulators. In the miRNA pathway&#39;s cytoplasmic part, the miRNA is processed from a hairpin-structured precursor (pre-miRNA) to a double-stranded (ds) mature RNA and ultimately to a single-stranded mature miRNA. In insects, ingesting these two ds forms can regulate the target gene expression; this inspired the trophic miRNA&#39;s use as a functional genomics and pest management tool. However, systematic studies enabling comparisons of pre- and mature forms, dosages, administration times, and instar-wise effects on target transcripts and phenotypes, which can help develop a miRNA administration method, are unavailable due to the different focuses of the previous investigations. We investigated the impact of trophically delivered Px-let-7 miRNA on the Brassicaceae pest, Plutella xylostella, to compare the efficacies of its pre- and ds-mature forms. Continuous feeding on the miRNA-supplemented diet suppressed expressions of FTZ-F1 and E74, the target ecdysone pathway genes. Both the pre-let-7 and mature let-7 miRNA forms similarly downregulated the target transcripts in all four larval instars. Pre-let-7 and let-7 ingestions decreased P. xylostella&#39;s larval mass and instar duration and increased mortality in all instars, exhibiting adverse effects on larval growth and development. Pre-miRNA processing Dicer-1 was upregulated upon pre-let-7 ingestion, indicating the miRNA uptake by the midgut cells. The scrambled sequence controls did not affect the target transcripts, suggesting the sequence-specific targeting by the mature miRNA and hairpin cassette&#39;s non-involvement in the target downregulation. This work provides a framework for miRNA and target gene function analyses and potentiates the trophic miRNA&#39;s utility in pest management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572838">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572838" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572838">
        <p class="paperTitle">Environmental DNA reveals fine scale spatial and temporal variation of prey species for marine mammals in a Scottish marine protected area</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572838" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572838" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boyse, E.; Robinson, K. P.; Beger, M.; Carr, I. M.; Taylor, M.; Valsecchi, E.; Goodman, S. J.</p>
        <p class="info">Score: 2.8, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572838' target='https://doi.org/10.1101/2023.12.21.572838'> 10.1101/2023.12.21.572838</a></p>
        <p class="abstract">Marine mammal foraging grounds are popular focal points for marine protected area (MPA) implementation, but may be temporally dynamic, requiring continuous monitoring to infer prey availability and abundance. Marine mammal distributions are assumed to be driven by their prey in foraging areas, but limited understanding of prey distributions often prevents us from exploring how shifting prey availability impacts both seasonal and long-term marine mammal distributions. Environmental DNA (eDNA) metabarcoding could enhance understanding of marine mammal habitat use in relation to their prey through simultaneous monitoring of both. However, eDNA applications focused on marine mammals or predator-prey dynamics have been limited to date. In this study, we assess spatiotemporal changes in the availability and abundance of minke whale (Balaenoptera acutorostrata) prey species in a newly established MPA, employing eDNA metabarcoding. We recovered 105 molecular operational taxonomic units (OTUs) from marine vertebrates using two primer sets targeting 12S and 16S genes, along with 112 OTUs from a broader eukaryotic primer set targeting 18S rRNA. Overall, key forage fish prey species, sandeels and clupeids, were the most abundant teleost fishes detected, although their availability varied temporally and with distance from shore. We also found clear spatial partitioning between coastal bottlenose dolphins and the more pelagic minke whales and harbour porpoises, paralleling availability of their main prey species. Other species of conservation interest were also detected including the critically endangered European eel (Anguilla anguilla), blue fin tuna (Thunnus thynnus), and the invasive pink salmon (Oncorhynchus gorbuscha). This study demonstrates the application of eDNA to detect spatiotemporal trends in the occurrence and abundance of cetacean predators and their prey, furthering our understanding of fine-scale habitat use within MPAs. Future, long-term monitoring of predator-prey dynamics with eDNA could improve our ability to predict climate-induced shifts in foraging grounds and enhance rapid responses with appropriate management actions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573227">
        <p class="paperTitle">Concerns on research history overview, distribution patterns, and potentially neglected ecological consequences of the invasive apple snail (Pomacea sp.): A review</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, D.; Li, Y.; Li, J.</p>
        <p class="info">Score: 2.0, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573227' target='https://doi.org/10.1101/2023.12.23.573227'> 10.1101/2023.12.23.573227</a></p>
        <p class="abstract">Invasive species pose a threat to global ecosystems and biodiversity, with the apple snail standing out as one kind of the worst invasive species due to its exclusive expansion. In order to draw a conceptual framework of its invasion pattern from an overview, we endeavor to conduct a bibliometric analysis and systematic review to explore the existing knowledge on research history and subjective categorization of Pomacea canaliculata and P. maculata, and to uncover emerging trends of distribution and possible neglected impact on food webs. The research history reveals that the first international research publication was documented in 1958. However, a substantial increase in international publications occurred only after 2000. The adoption and effective implementation of a universally agreed upon management strategy may greatly enhance the progress of research on apple snail invasion. Notably, co-word analysis revealed that the content of each thematic cluster has a shift from basic biology to the monitoring of occurrences and environmental adaptation. Currently, the global distribution of apple snails has expanded to 39 countries primarily as a result of anthropogenic translocation. All infested countries, with the exception of Israel and Singapore, are major paddy rice producers. The northernmost distribution limit is Tyumen in Russia, while the southernmost limit is the Buenos Aires Province in Argentina. Biological control has emerged as a prominent technology for the prevention of apple snail infestations, leading to the identification of 98 non-target enemy species as potential control agents. A taxonomic diversity analysis revealed that these species encompassed a wide range of phylogenetic classifications, including 6 Phyla, 12 Classes, 33 Orders, 55 Families, and 82 Genera. This observation suggests that there is a substantial pool of animals from diverse phylogenetic levels that can prey upon apple snails. Given the high population density and extensive distribution of apple snails, there is a pressing need for further research to explore their potential ecological ramifications. The current understanding of the upward ecological impacts of apple snails remains inadequate. These findings provide valuable insights for future research endeavors and contribute to the effective management of invasive apple snail populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572140">
        <p class="paperTitle">Metaproteogenomics resolution of a high-CO2 aquifer community suggests an active symbiotic lifestyle of groundwater Gracilibacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa-Gonzalez, P. A.; Bornemann, T. L. V.; Hinzke, T.; Maass, S.; Trautwein-Schult, A.; Starke, J.; Moore, C. J.; Esser, S. P.; Plewka, J.; Hesse, T.; Schmidt, T. C.; Schreiber, U.; Bor, B.; Becher, D.; Probst, A. J.</p>
        <p class="info">Score: 7.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572140' target='https://doi.org/10.1101/2023.12.18.572140'> 10.1101/2023.12.18.572140</a></p>
        <p class="abstract">BackgroundBacteria of the Candidate Phyla Radiation (CPR), constituting about 25% of the bacterial biodiversity, are characterized by small cell size and patchy genomes without complete key metabolic pathways, suggesting a symbiotic lifestyle. Gracilibacteria (BD1-5), which are part of the CPR branch, possess alternate coded genomes and have not yet been cultivated. However, besides genomic evidence, little is known about the lifestyle of Gracilibacteria, their temporal dynamics, and activity in natural ecosystems, particularly in groundwater, where they were initially been genomically resolved. Therefore, we here aimed to investigate Gracilibacteria activity in situ and to discern expressed genes involved in their lifestyle, using the metaproteogenome of Gracilibacteria as a function of time in the cold-water geyser Wallender Born in the Volcanic Eifel region in Germany.

ResultsWe coupled genome-resolved metagenomics and metaproteomics to investigate a cold-water geyser microbial community enriched in Gracilibacteria across a 12-day time-series. Groundwater was collected and sequentially filtered to fraction CPR and other bacteria. Based on 670 Gbps of metagenomic data, 1129 different ribosomal protein S3 marker genes and 751 high-quality genomes (123 population genomes after dereplication), we identified dominant bacteria belonging to Galionellales and Gracilibacteria along with keystone microbes, which were low in genomic abundance but substantially contributing to proteomic abundance. Seven high-quality Gracilibacteria genomes showed typical limitations, such as limited amino acid or nucleotide synthesis, in their central metabolism but no co-occurrence with potential hosts. The genomes of these Gracilibacteria encoded for a high number of proteins related to a symbiotic or even predatory lifestyle, e.g., type IV and type II secretion system subunits and features related to cell-cell interactions and cell motility, which were also detected on protein level.

ConclusionsCoupling metagenomics to metaproteomics enabled us to identify microbial keystone taxa in a high-CO2 aquifer, and to reveal microbial dynamics of Gracilibacteria. We posit that Gracilibacteria might be successful microbial predators in this ecosystem, potentially aiding in population control of this highly perturbed microbial geyser community from the deep biosphere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572764">
        <p class="paperTitle">Winners in good times and bad times: Aerobic anoxygenic phototrophic bacteria profit from photoheterotrophy under carbon-rich and poor conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piwosz, K.; Villena-Alemany, C.; Calkiewicz, J.; Mujakic, I.; Nahlik, V.; Dean, J.; Koblizek, M.</p>
        <p class="info">Score: 1.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572764' target='https://doi.org/10.1101/2023.12.21.572764'> 10.1101/2023.12.21.572764</a></p>
        <p class="abstract">Aerobic Anoxygenic Phototrophic (AAP) bacteria are an important component of freshwater bacterioplankton. They can support their heterotrophic metabolism with energy from light, and by that enhance their growth efficiency. Based on results from cultures, it was hypothesized that photoheterotrophy provides an advantage under carbon limitation and facilitates access to recalcitrant or low-energy carbon sources. However, verification of these hypotheses for natural AAP communities has been lacking. Here, we conducted whole community manipulation experiments and compared the growth of AAP bacteria under carbon limited and with recalcitrant or low-energy carbon sources under dark and light conditions to elucidate how they profit from photoheterotrophy. We found that it depends on the season. In spring, AAP bacteria induce photoheterotrophic metabolism under carbon limitation but they outperform heterotrophic bacteria when carbon is available. This effect seems to be driven by physiological responses rather than changes at the community level. In autumn photoheterotrophy is less beneficial. In both seasons, AAP bacteria responded negatively to recalcitrant or low-energy carbon sources in light. This unexpected observation may have ecosystem-level consequences as lake browning continues. In general, our findings contribute to the understanding of the dynamics of AAP bacteria observed in pelagic environments.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
