<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Drab and distant birds are studied less than their fancy-feathered friends
Authors: Fischer, S. E.; Lindsay, A. M.; Otten, J. G.; Miles, D. B.; Streby, H. M.
Score: 98.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.560707
Human decisions are influenced by implicit biases, and scientists do not exist in an objectivity vacuum. Subconscious biases in scientists choices about which species to study may beget distorted knowledge bases and stagnant paradigms. Disparities in biological knowledge can result from bias in study species selection within a cycle of policymaking, funding, and publication, all subject to implicit biases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="Drab and distant birds are studied less than their fancy-feathered friends
Authors: Fischer, S. E.; Lindsay, A. M.; Otten, J. G.; Miles, D. B.; Streby, H. M.
Score: 98.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.560707
Human decisions are influenced by implicit biases, and scientists do not exist in an objectivity vacuum. Subconscious biases in scientists choices about which species to study may beget distorted knowledge bases and stagnant paradigms. Disparities in biological knowledge can result from bias in study species selection within a cycle of policymaking, funding, and publication, all subject to implicit biases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="Drab and distant birds are studied less than their fancy-feathered friends
Authors: Fischer, S. E.; Lindsay, A. M.; Otten, J. G.; Miles, D. B.; Streby, H. M.
Score: 98.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.560707
Human decisions are influenced by implicit biases, and scientists do not exist in an objectivity vacuum. Subconscious biases in scientists choices about which species to study may beget distorted knowledge bases and stagnant paradigms. Disparities in biological knowledge can result from bias in study species selection within a cycle of policymaking, funding, and publication, all subject to implicit biases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "Drab and distant birds are studied less than their fancy-feathered friends\nAuthors: Fischer, S. E.; Lindsay, A. M.; Otten, J. G.; Miles, D. B.; Streby, H. M.\nScore: 98.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.560707\nHuman decisions are influenced by implicit biases, and scientists do not exist in an objectivity vacuum. Subconscious biases in scientists choices about which species to study may beget distorted knowledge bases and stagnant paradigms. Disparities in biological knowledge can result from bias in study species selection within a cycle of policymaking, funding, and publication, all subject to implicit biases.",
  "keywords": [
    
  ],
  "articleBody": " Drab and distant birds are studied less than their fancy-feathered friends\nAuthors: Fischer, S. E.; Lindsay, A. M.; Otten, J. G.; Miles, D. B.; Streby, H. M.\nScore: 98.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.560707\nHuman decisions are influenced by implicit biases, and scientists do not exist in an objectivity vacuum. Subconscious biases in scientists choices about which species to study may beget distorted knowledge bases and stagnant paradigms. Disparities in biological knowledge can result from bias in study species selection within a cycle of policymaking, funding, and publication, all subject to implicit biases. Here, we show that ornithological research in the USA and Canada is biased toward birds with greater aesthetic salience and those with larger breeding ranges and ranges that encompass more universities. We quantified components of aesthetic salience (e.g., color, pattern/contrast, body size) of 293 passerines and near-passerines based on empirically documented human visual preferences and investigated whether these components were associated with research effort. We also quantified each species breeding range size and the number of universities within that range. Accounting for phylogenetic relatedness, we found that these metrics of aesthetics, familiarity, and accessibility combined to explain 45% of the variation in the number of published papers about each species from 1965-2020. On average, birds in the top 10% of aesthetic salience were studied 3.0X more than birds in the bottom 10%, and publication numbers were predicted most strongly by color and pattern components of aesthetic salience. Birds in the top 10% of breeding range size and university abundance were studied 3.8X and 3.5X more often than species in the bottom 10% of those categories, respectively. Species listed as Endangered and those featured on journal covers have greater aesthetic salience scores than other species. We discuss how these biases may influence perceived relative value of species with respect to culture and conservation. The disparities in empirical knowledge we describe here perpetuate a positive feedback loop, thus widening the gap between the avian \"haves\" and \"have-nots\", with some questions answered repeatedly while potentially critical discoveries are left undiscovered. O_QD\"All animals are equal, but some animals are more equal than others.\" --George Orwell, Animal Farm (1945) C_QD\nGlobal abundance patterns, diversity, and ecology of Patescibacteria in wastewater treatment plants\nAuthors: Wagner, M.; Hu, H.; Kristensen, J. M.; Pjevac, P.; Kitzinger, K.; Herbold, C. W.; Hausmann, B.; Dueholm, M. K. D.; Nielsen, P. H.\nScore: 58.4, Published: 2023-10-29 DOI: 10.1101/2023.10.25.562895\nBackgroundMicroorganisms are responsible for nutrient removal and resource recovery in wastewater treatment plants (WWTPs), and their diversity is often studied by 16S rRNA gene amplicon sequencing. However, this approach underestimates the abundance and diversity of Patescibacteria due to the low coverage of commonly used PCR primers for this highly divergent bacterial phylum. Therefore, our current understanding of the global diversity, distribution, and ecological role of Patescibacteria in WWTPs is very incomplete. This is particularly relevant as Patescibacteria are considered to be associated with microbial host cells and can therefore influence the abundance and temporal variability of other microbial groups that are important for WWTP functioning. ResultsHere, we evaluated the in silico coverage of widely used 16S rRNA gene-targeted primer pairs and redesigned a primer pair targeting the V4 region of bacterial and archaeal 16S rRNA genes to expand its coverage for Patescibacteria. We then experimentally evaluated and compared the performance of the original and modified V4-targeted primers on 565 WWTP samples from the MiDAS global sample collection. Using the modified primer pair, the percentage of ASVs classified as Patescibacteria increased from 5.9% to 23.8%, and the number of detected patescibacterial genera increased from 560 to 1,576, while the detected diversity of the remaining microbial community remained similar. Due to this significantly improved coverage of Patescibacteria, we identified 23 core genera of Patescibacteria in WWTPs and described the global distribution pattern of these unusual microbes in these systems. Finally, correlation network analysis revealed potential host organisms that might be associated with Patescibacteria in WWTPs. Interestingly, strong indications were found for an association between Patescibacteria of the Saccharimonadia and globally abundant polyphosphate-accumulating organisms of the genus Ca. Phosporibacter. ConclusionsOur study (i) provides an improved 16S rRNA gene V4 region-targeted amplicon primer pair inclusive of Patescibacteria with little impact on the detection of other taxa, (ii) reveals the diversity and distribution patterns of Patescibacteria in WWTPs on a global scale, and (iii) provides new insights into the ecological role and potential hosts of Patescibacteria in WWTPs.\nMetabolic similarity and the predictability of microbial community assembly\nAuthors: Vila, J. C. C.; Goldford, J.; Estrela, S.; Bajic, D.; Sanchez-Gorostiaga, A.; Damian-Serrano, A.; Lu, N.; Marsland, R.; Rebolleda-Gomez, M.; Mehta, P.; Sanchez, A.\nScore: 53.9, Published: 2023-10-28 DOI: 10.1101/2023.10.25.564019\nAbstractWhen microbial communities form, their composition is shaped by selective pressures imposed by the environment. Can we predict which communities will assemble under different environmental conditions? Here, we hypothesize that quantitative similarities in metabolic traits across metabolically similar environments lead to predictable similarities in community composition. To that end, we measured the growth rate and by-product profile of a library of proteobacterial strains in a large number of single nutrient environments. We found that growth rates and secretion profiles were positively correlated across environments when the supplied substrate was metabolically similar. By analyzing hundreds of in-vitro communities experimentally assembled in an array of different synthetic environments, we then show that metabolically similar substrates select for taxonomically similar communities. These findings lead us to propose and then validate a comparative approach for quantitatively predicting the effects of novel substrates on the composition of complex microbial consortia.\nA globally synthesised and flagged bee occurrence dataset and cleaning workflow\nAuthors: Dorey, J. B.; Fischer, E. E.; Chesshire, P. R.; Nava-Bolanos, A.; OReilly, R. L.; Bossert, S.; Collins, S. M.; Lichtenberg, E. M.; Tucker, E. M.; Smith-Pardo, A.; Falcon-Brindis, A.; Guevara, D. A.; Ribeiro, B.; de Pedro, D.; Pickering, J.; James, K.-L.; Parys, K. A.; McCabe, L. M.; Rogan, M. S.; Minckley, R. L.; Velazco, S. J. E.; Griswold, T.; Zarrillo, T. A.; Jetz, W.; Sica, Y. V.; Orr, M. C.; Guzman, L. M.; Ascher, J. A.; Hughes, A. C.; Cobb, N. S.\nScore: 36.5, Published: 2023-10-09 DOI: 10.1101/2023.06.30.547152\nSpecies occurrence data are foundational for research, conservation, and science communication, but the limited availability and accessibility of reliable data represents a major obstacle, particularly for insects, which face mounting pressures. We present BeeBDC, a new R package, and a global bee occurrence dataset to address this issue. We combined \u003e18.3 million bee occurrence records from multiple public repositories (GBIF, SCAN, iDigBio, USGS, ALA) and smaller datasets, then standardised, flagged, deduplicated, and cleaned the data using the reproducible BeeBDC R-workflow. Specifically, we harmonised species names (following established global taxonomy), country names, and collection dates and we added record-level flags for a series of potential quality issues. These data are provided in two formats, \"cleaned\" and \"flagged-but-uncleaned\". The BeeBDC package with online documentation provides end users the ability to modify filtering parameters to address their research questions. By publishing reproducible R workflows and globally cleaned datasets, we can increase the accessibility and reliability of downstream analyses. This workflow can be implemented for other taxa to support research and conservation.\nConspicuous shell shape plasticity across lake-stream habitats in a freshwater mussel (Pyganodon grandis)\nAuthors: Keogh, S. M.; Minerich, B. J.; Ohlman, L. M.; Pletta, M. E.; Scheunemann, A. E.; Schroeder, Z. K.; Secrist, Z. A.; Franzen, A. J.; Sietman, B. E.; Simons, A. M.\nScore: 21.1, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564758\nHydrodynamic forces and their absence appears to exert differential selection pressure between lake and stream populations, creating a tight fit between organismal phenotypes and their environments. Ecophenotypic variants may be the result from isolated genetic evolution or phenotypic plasticity, where a widespread genotype can produce multiple phenotypes dependent on the environment. Freshwater mussels possess a wide degree of morphological variation that frequently covaries with the environment, making them a good system to understand the mechanisms of ecophenotypic variation across hydrological conditions. We designed a two-year experiment where individuals from the same Pyganodon grandis maternal brood (half-full siblings) were reared at a controlled site and four natural sites involving one lake and three streams. At the end of the experiment, shell shape was quantified for recaptured (N=70), wild (N=206), and zoo-reared (N=305) mussels. Analysis of covariance found significant differences in shell shape between rearing sites, particularly between stream and lake habitats, but no shape differences were detected across the three stream sites. At two of the four sites, shell shape of recaptured individuals matched the morphology of wild populations. Genomic sequencing and parentage analysis identified 27 different fathers among recaptured individuals. Yet, no genetic differences were present between stream and lake habitats and there was no effect of parentage on shell shape. Taken together, phenotypic plasticity, over genetic differentiation, is identified as the primary mechanism of shell shape ecophenotypy. Plasticity may be a key adaptation for freshwater mussels and possibly provide a buffer against their imperilment in degraded habitats.\nNuclear eDNA Metabarcoding Primers for Anthozoan Coral Biodiversity Assessment\nAuthors: McCartin, L. J.; Saso, E.; Vohsen, S. A.; Pittoors, N. C.; Demetriades, P.; McFadden, C. S.; Quattrini, A. M.; Herrera, S.\nScore: 18.4, Published: 2023-10-31 DOI: 10.1101/2023.10.26.564240\nThe distributions of anthozoan corals are under-characterized due to their wide bathymetric range, occurrences in remote locales, and difficulties of identification from morphology alone. Environmental DNA (eDNA) sequencing promises to be a non-invasive strategy to complement conventional approaches for mapping and monitoring coral communities. Primers for eDNA meta-barcoding have been designed to amplify nuclear and mitochondrial DNA barcodes in shallow scleractinians and mitochondrial MutS in deep-sea octocorals. However, a comprehensive method for eDNA meta-barcoding from all anthozoan corals, including black corals, has not been developed. We leveraged a sequence database of global coral collections, from shallow water to the deep sea, to design new PCR primers for coral eDNA sequencing that target the 28S rRNA gene. We tested the performance of these primers by amplifying and sequencing eDNA from water samples collected in the Gulf of Mexico near mesophotic and deep-sea corals that were also imaged, sampled, and sequenced. Sequencing libraries produced using the primers were highly enriched in coral eDNA, with up to 99.8% of the reads originating from corals. Further, the 28S barcode amplified using the primers distinguished coral genera. We recovered amplicon sequencing variants (ASVs) identical to DNA barcodes derived from Sanger sequencing and genome skimming of corals sampled at the same field sites. This new eDNA meta-barcoding strategy permits targeted eDNA sequencing of black corals, octocorals, and scleractinians at sites where they co-occur and expands our current toolkit for mapping and monitoring coral communities in shallow coral reefs and the deep sea.\nAge-related spatial ecology of Audouin's gull during the non-breeding season\nAuthors: Ponti, R.; Morera-Pujol, V.; Sallent, A.; Gonzalez-Solis, J.; Ramos, R.\nScore: 9.0, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563411\nRelationships between individuals age and the movement ecology and habitat preference of long-lived migratory birds still remain understudied. According to the exploration-refinement hypothesis it is thought that adults would select better and more productive areas for foraging than inexperienced juvenile birds would do. Additionally, age-related differences in migratory patterns and exploited habitats could be explained by the attempt to avoid competition between juveniles and adults. Here, we explored the differences in the migratory patterns, habitat selection and foraging behaviour between juvenile and adult Audouins gulls (Ichthyaetus audouinii), a species listed as vulnerable by the IUCN. We captured 9 juveniles and 8 adults in the colony of San Pedro (SE Spain) and equipped them with high-resolution 5-min programmed GPS to track their postnuptial/first migration and non-breeding destinations. First, juveniles tended to migrate longer distances than adults did. Second, the time spent foraging between age groups did not differ. Third, freshwater masses constituted an essential habitat during the non-breeding season for both juveniles and adults. Fourth, we found that adults used a greater variety of habitats than juveniles did, but adults positively select foraging habitats despite the low availability while juveniles do not. Finally, repeatability in habitat use of individuals of the same age was rather low. We provided evidence of age-related differences in migratory patterns and habitat exploitation during the non-breeding period in a migratory seabird which can be explained by the avoidance of competition between adults and juveniles and the greater experience in foraging performance that adults have in comparison with juveniles.\nGains in soil C storage under anthropogenic N deposition are rapidly lost following its cessation\nAuthors: Propson, B. E.; Zak, D. R.; Classen, A. T.; Burton, A. J.; Freedman, Z. B.\nScore: 8.2, Published: 2023-11-02 DOI: 10.1101/2023.10.31.562515\nHistorically high anthropogenic nitrogen (N) deposition enhanced the global terrestrial carbon (C) sink that has partially offset anthropogenic CO2 emissions. This was driven by the suppression of microbial activity associated with the breakdown of biochemically recalcitrant soil organic matter (SOM), which resulted in the consequential accumulation of soil C. However, since the implementation of anthropogenic emission abatement policies, atmospheric N deposition has declined globally since the 1990s. The objective of our study was to assess the initial recovery trajectory of the enhanced terrestrial C sink, and associated microbial activities, that was supported under high rates of anthropogenic N deposition when N deposition is reduced. We used a long-term field experiment that received experimental N deposition for 24-years across the Great Lakes region of North America. We measured soil C and N content, microbial respiration, and extracellular enzyme activity potential in the organic and mineral soil horizons and compared standardized effect sizes of soil C in response to the N deposition treatment across the history of our experiment (1994-2022). Our results demonstrate that the accumulated C in the organic horizon from experimental N deposition has been lost and exhibit additional deficits 5-years following the return to ambient N deposition and is, in part, the result of mechanistic changes in microbial activity. Whereas, soil C in the mineral soil is being lost to a lesser extent. We suspect the organic horizon dynamics that we document may occur in other temperate forests, potentially reducing the Northern Hemisphere C sink and enhancing climate warming.\nTerrestrial land cover shapes fish diversity in major subtropical rivers\nAuthors: Zhang, H.; Blackman, R. C.; Furrer, R.; Osathanunkul, M.; Brantschen, J.; Di Muri, C.; Harper, L. R.; Haenfling, B.; Niklaus, P. A.; Pellissier, L.; Schaepman, M. E.; Zong, S.; Altermatt, F.\nScore: 7.4, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564688\nFreshwater biodiversity is critically affected by human modifications of terrestrial land use and land cover (LULC)1,2. Yet, knowledge of the spatial extent and magnitude of LULC-aquatic biodiversity linkages is still surprisingly limited, impeding the implementation of optimal management strategies3. Here, we compiled fish diversity data across a 160,000-km2 subtropical river catchment in Thailand characterized by exceptional biodiversity4 yet intense anthropogenic alterations5, and attributed fish species richness and community composition to contemporary terrestrial LULC across the catchment. We estimated a spatial range of LULC effects extending up to about 20 km upstream from sampling sites, and explained nearly 60 % of the variance in the observed species richness, associated with major LULC categories including croplands, forest, and urban areas. We find that integrating both spatial range and magnitudes of LULC effects is needed to accurately predict fish species richness. Further, projected LULC changes showcase future gains and losses of fish species richness across the river network and offer a scalable basis for riverine biodiversity conservation and land management, allowing for potential mitigation of biodiversity loss in highly diverse yet data-deficient tropical to sub-tropical riverine habitats.\nUnveiling Urban Complexity: Integrating OpenStreetMap to enhance representation of fine-scale landscape heterogeneity\nAuthors: Gelmi-Candusso, T. A.; Rodriguez, P.; Fortin, M.-J.\nScore: 6.9, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564785\nLandscape heterogeneity has an impact on wildlife behavior, their interactions, and their persistence. Urban landscapes are among the world's most heterogeneous landscapes, yet current global landcover maps classify developed land in a single landcover type. This limits the spatial scale at which urban ecologists can approach research questions. OpenStreetMap (OSM), an open-source mapping platform, can be leveraged to enhance the representation of landscape heterogeneity in developed areas. For this, we extracted OSM features with attributes representing infrastructure, land use and green cover, integrating these into a continental landcover map through a globally applicable computational framework. We validated our OSM-enhanced landcover layer against existing remote sensing, aerial photography, and local governmental maps for 33 cities in North America. Our framework's output provides an 89% accurate representation of landscape heterogeneity. We discuss caveats, potential improvements, and ecological applications. Our OSM-based landcover enhancement framework will facilitate the use of open-source landscape information for improved ecological modeling and urban planning.\n",
  "wordCount" : "2746",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.560707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.560707" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.560707">
        <p class="paperTitle">Drab and distant birds are studied less than their fancy-feathered friends</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.560707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.560707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, S. E.; Lindsay, A. M.; Otten, J. G.; Miles, D. B.; Streby, H. M.</p>
        <p class="info">Score: 98.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.560707' target='https://doi.org/10.1101/2023.10.26.560707'> 10.1101/2023.10.26.560707</a></p>
        <p class="abstract">Human decisions are influenced by implicit biases, and scientists do not exist in an objectivity vacuum. Subconscious biases in scientists choices about which species to study may beget distorted knowledge bases and stagnant paradigms. Disparities in biological knowledge can result from bias in study species selection within a cycle of policymaking, funding, and publication, all subject to implicit biases. Here, we show that ornithological research in the USA and Canada is biased toward birds with greater aesthetic salience and those with larger breeding ranges and ranges that encompass more universities. We quantified components of aesthetic salience (e.g., color, pattern/contrast, body size) of 293 passerines and near-passerines based on empirically documented human visual preferences and investigated whether these components were associated with research effort. We also quantified each species breeding range size and the number of universities within that range. Accounting for phylogenetic relatedness, we found that these metrics of aesthetics, familiarity, and accessibility combined to explain 45% of the variation in the number of published papers about each species from 1965-2020. On average, birds in the top 10% of aesthetic salience were studied 3.0X more than birds in the bottom 10%, and publication numbers were predicted most strongly by color and pattern components of aesthetic salience. Birds in the top 10% of breeding range size and university abundance were studied 3.8X and 3.5X more often than species in the bottom 10% of those categories, respectively. Species listed as Endangered and those featured on journal covers have greater aesthetic salience scores than other species. We discuss how these biases may influence perceived relative value of species with respect to culture and conservation. The disparities in empirical knowledge we describe here perpetuate a positive feedback loop, thus widening the gap between the avian &#34;haves&#34; and &#34;have-nots&#34;, with some questions answered repeatedly while potentially critical discoveries are left undiscovered.

O_QD&#34;All animals are equal, but some animals are more equal than others.&#34; --George Orwell, Animal Farm (1945)

C_QD</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.562895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.562895" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.562895">
        <p class="paperTitle">Global abundance patterns, diversity, and ecology of Patescibacteria in wastewater treatment plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.562895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.562895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wagner, M.; Hu, H.; Kristensen, J. M.; Pjevac, P.; Kitzinger, K.; Herbold, C. W.; Hausmann, B.; Dueholm, M. K. D.; Nielsen, P. H.</p>
        <p class="info">Score: 58.4, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.562895' target='https://doi.org/10.1101/2023.10.25.562895'> 10.1101/2023.10.25.562895</a></p>
        <p class="abstract">BackgroundMicroorganisms are responsible for nutrient removal and resource recovery in wastewater treatment plants (WWTPs), and their diversity is often studied by 16S rRNA gene amplicon sequencing. However, this approach underestimates the abundance and diversity of Patescibacteria due to the low coverage of commonly used PCR primers for this highly divergent bacterial phylum. Therefore, our current understanding of the global diversity, distribution, and ecological role of Patescibacteria in WWTPs is very incomplete. This is particularly relevant as Patescibacteria are considered to be associated with microbial host cells and can therefore influence the abundance and temporal variability of other microbial groups that are important for WWTP functioning.

ResultsHere, we evaluated the in silico coverage of widely used 16S rRNA gene-targeted primer pairs and redesigned a primer pair targeting the V4 region of bacterial and archaeal 16S rRNA genes to expand its coverage for Patescibacteria. We then experimentally evaluated and compared the performance of the original and modified V4-targeted primers on 565 WWTP samples from the MiDAS global sample collection. Using the modified primer pair, the percentage of ASVs classified as Patescibacteria increased from 5.9% to 23.8%, and the number of detected patescibacterial genera increased from 560 to 1,576, while the detected diversity of the remaining microbial community remained similar. Due to this significantly improved coverage of Patescibacteria, we identified 23 core genera of Patescibacteria in WWTPs and described the global distribution pattern of these unusual microbes in these systems. Finally, correlation network analysis revealed potential host organisms that might be associated with Patescibacteria in WWTPs. Interestingly, strong indications were found for an association between Patescibacteria of the Saccharimonadia and globally abundant polyphosphate-accumulating organisms of the genus Ca. Phosporibacter.

ConclusionsOur study (i) provides an improved 16S rRNA gene V4 region-targeted amplicon primer pair inclusive of Patescibacteria with little impact on the detection of other taxa, (ii) reveals the diversity and distribution patterns of Patescibacteria in WWTPs on a global scale, and (iii) provides new insights into the ecological role and potential hosts of Patescibacteria in WWTPs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564019">
        <p class="paperTitle">Metabolic similarity and the predictability of microbial community assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vila, J. C. C.; Goldford, J.; Estrela, S.; Bajic, D.; Sanchez-Gorostiaga, A.; Damian-Serrano, A.; Lu, N.; Marsland, R.; Rebolleda-Gomez, M.; Mehta, P.; Sanchez, A.</p>
        <p class="info">Score: 53.9, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564019' target='https://doi.org/10.1101/2023.10.25.564019'> 10.1101/2023.10.25.564019</a></p>
        <p class="abstract">AbstractWhen microbial communities form, their composition is shaped by selective pressures imposed by the environment. Can we predict which communities will assemble under different environmental conditions? Here, we hypothesize that quantitative similarities in metabolic traits across metabolically similar environments lead to predictable similarities in community composition. To that end, we measured the growth rate and by-product profile of a library of proteobacterial strains in a large number of single nutrient environments. We found that growth rates and secretion profiles were positively correlated across environments when the supplied substrate was metabolically similar. By analyzing hundreds of in-vitro communities experimentally assembled in an array of different synthetic environments, we then show that metabolically similar substrates select for taxonomically similar communities. These findings lead us to propose and then validate a comparative approach for quantitatively predicting the effects of novel substrates on the composition of complex microbial consortia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547152">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547152" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547152">
        <p class="paperTitle">A globally synthesised and flagged bee occurrence dataset and cleaning workflow</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547152" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547152" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dorey, J. B.; Fischer, E. E.; Chesshire, P. R.; Nava-Bolanos, A.; OReilly, R. L.; Bossert, S.; Collins, S. M.; Lichtenberg, E. M.; Tucker, E. M.; Smith-Pardo, A.; Falcon-Brindis, A.; Guevara, D. A.; Ribeiro, B.; de Pedro, D.; Pickering, J.; James, K.-L.; Parys, K. A.; McCabe, L. M.; Rogan, M. S.; Minckley, R. L.; Velazco, S. J. E.; Griswold, T.; Zarrillo, T. A.; Jetz, W.; Sica, Y. V.; Orr, M. C.; Guzman, L. M.; Ascher, J. A.; Hughes, A. C.; Cobb, N. S.</p>
        <p class="info">Score: 36.5, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547152' target='https://doi.org/10.1101/2023.06.30.547152'> 10.1101/2023.06.30.547152</a></p>
        <p class="abstract">Species occurrence data are foundational for research, conservation, and science communication, but the limited availability and accessibility of reliable data represents a major obstacle, particularly for insects, which face mounting pressures. We present BeeBDC, a new R package, and a global bee occurrence dataset to address this issue. We combined &gt;18.3 million bee occurrence records from multiple public repositories (GBIF, SCAN, iDigBio, USGS, ALA) and smaller datasets, then standardised, flagged, deduplicated, and cleaned the data using the reproducible BeeBDC R-workflow. Specifically, we harmonised species names (following established global taxonomy), country names, and collection dates and we added record-level flags for a series of potential quality issues. These data are provided in two formats, &#34;cleaned&#34; and &#34;flagged-but-uncleaned&#34;. The BeeBDC package with online documentation provides end users the ability to modify filtering parameters to address their research questions. By publishing reproducible R workflows and globally cleaned datasets, we can increase the accessibility and reliability of downstream analyses. This workflow can be implemented for other taxa to support research and conservation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564758">
        <p class="paperTitle">Conspicuous shell shape plasticity across lake-stream habitats in a freshwater mussel (Pyganodon grandis)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keogh, S. M.; Minerich, B. J.; Ohlman, L. M.; Pletta, M. E.; Scheunemann, A. E.; Schroeder, Z. K.; Secrist, Z. A.; Franzen, A. J.; Sietman, B. E.; Simons, A. M.</p>
        <p class="info">Score: 21.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564758' target='https://doi.org/10.1101/2023.10.30.564758'> 10.1101/2023.10.30.564758</a></p>
        <p class="abstract">Hydrodynamic forces and their absence appears to exert differential selection pressure between lake and stream populations, creating a tight fit between organismal phenotypes and their environments. Ecophenotypic variants may be the result from isolated genetic evolution or phenotypic plasticity, where a widespread genotype can produce multiple phenotypes dependent on the environment. Freshwater mussels possess a wide degree of morphological variation that frequently covaries with the environment, making them a good system to understand the mechanisms of ecophenotypic variation across hydrological conditions. We designed a two-year experiment where individuals from the same Pyganodon grandis maternal brood (half-full siblings) were reared at a controlled site and four natural sites involving one lake and three streams. At the end of the experiment, shell shape was quantified for recaptured (N=70), wild (N=206), and zoo-reared (N=305) mussels. Analysis of covariance found significant differences in shell shape between rearing sites, particularly between stream and lake habitats, but no shape differences were detected across the three stream sites. At two of the four sites, shell shape of recaptured individuals matched the morphology of wild populations. Genomic sequencing and parentage analysis identified 27 different fathers among recaptured individuals. Yet, no genetic differences were present between stream and lake habitats and there was no effect of parentage on shell shape. Taken together, phenotypic plasticity, over genetic differentiation, is identified as the primary mechanism of shell shape ecophenotypy. Plasticity may be a key adaptation for freshwater mussels and possibly provide a buffer against their imperilment in degraded habitats.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564240">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564240" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564240">
        <p class="paperTitle">Nuclear eDNA Metabarcoding Primers for Anthozoan Coral Biodiversity Assessment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564240" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564240" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCartin, L. J.; Saso, E.; Vohsen, S. A.; Pittoors, N. C.; Demetriades, P.; McFadden, C. S.; Quattrini, A. M.; Herrera, S.</p>
        <p class="info">Score: 18.4, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564240' target='https://doi.org/10.1101/2023.10.26.564240'> 10.1101/2023.10.26.564240</a></p>
        <p class="abstract">The distributions of anthozoan corals are under-characterized due to their wide bathymetric range, occurrences in remote locales, and difficulties of identification from morphology alone. Environmental DNA (eDNA) sequencing promises to be a non-invasive strategy to complement conventional approaches for mapping and monitoring coral communities. Primers for eDNA meta-barcoding have been designed to amplify nuclear and mitochondrial DNA barcodes in shallow scleractinians and mitochondrial MutS in deep-sea octocorals. However, a comprehensive method for eDNA meta-barcoding from all anthozoan corals, including black corals, has not been developed. We leveraged a sequence database of global coral collections, from shallow water to the deep sea, to design new PCR primers for coral eDNA sequencing that target the 28S rRNA gene. We tested the performance of these primers by amplifying and sequencing eDNA from water samples collected in the Gulf of Mexico near mesophotic and deep-sea corals that were also imaged, sampled, and sequenced. Sequencing libraries produced using the primers were highly enriched in coral eDNA, with up to 99.8% of the reads originating from corals. Further, the 28S barcode amplified using the primers distinguished coral genera. We recovered amplicon sequencing variants (ASVs) identical to DNA barcodes derived from Sanger sequencing and genome skimming of corals sampled at the same field sites. This new eDNA meta-barcoding strategy permits targeted eDNA sequencing of black corals, octocorals, and scleractinians at sites where they co-occur and expands our current toolkit for mapping and monitoring coral communities in shallow coral reefs and the deep sea.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563411">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563411" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563411">
        <p class="paperTitle">Age-related spatial ecology of Audouin&#39;s gull during the non-breeding season</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563411" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563411" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ponti, R.; Morera-Pujol, V.; Sallent, A.; Gonzalez-Solis, J.; Ramos, R.</p>
        <p class="info">Score: 9.0, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563411' target='https://doi.org/10.1101/2023.10.23.563411'> 10.1101/2023.10.23.563411</a></p>
        <p class="abstract">Relationships between individuals age and the movement ecology and habitat preference of long-lived migratory birds still remain understudied. According to the exploration-refinement hypothesis it is thought that adults would select better and more productive areas for foraging than inexperienced juvenile birds would do. Additionally, age-related differences in migratory patterns and exploited habitats could be explained by the attempt to avoid competition between juveniles and adults. Here, we explored the differences in the migratory patterns, habitat selection and foraging behaviour between juvenile and adult Audouins gulls (Ichthyaetus audouinii), a species listed as vulnerable by the IUCN. We captured 9 juveniles and 8 adults in the colony of San Pedro (SE Spain) and equipped them with high-resolution 5-min programmed GPS to track their postnuptial/first migration and non-breeding destinations. First, juveniles tended to migrate longer distances than adults did. Second, the time spent foraging between age groups did not differ. Third, freshwater masses constituted an essential habitat during the non-breeding season for both juveniles and adults. Fourth, we found that adults used a greater variety of habitats than juveniles did, but adults positively select foraging habitats despite the low availability while juveniles do not. Finally, repeatability in habitat use of individuals of the same age was rather low. We provided evidence of age-related differences in migratory patterns and habitat exploitation during the non-breeding period in a migratory seabird which can be explained by the avoidance of competition between adults and juveniles and the greater experience in foraging performance that adults have in comparison with juveniles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.562515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.562515" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.562515">
        <p class="paperTitle">Gains in soil C storage under anthropogenic N deposition are rapidly lost following its cessation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.562515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.562515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Propson, B. E.; Zak, D. R.; Classen, A. T.; Burton, A. J.; Freedman, Z. B.</p>
        <p class="info">Score: 8.2, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.562515' target='https://doi.org/10.1101/2023.10.31.562515'> 10.1101/2023.10.31.562515</a></p>
        <p class="abstract">Historically high anthropogenic nitrogen (N) deposition enhanced the global terrestrial carbon (C) sink that has partially offset anthropogenic CO2 emissions. This was driven by the suppression of microbial activity associated with the breakdown of biochemically recalcitrant soil organic matter (SOM), which resulted in the consequential accumulation of soil C. However, since the implementation of anthropogenic emission abatement policies, atmospheric N deposition has declined globally since the 1990s. The objective of our study was to assess the initial recovery trajectory of the enhanced terrestrial C sink, and associated microbial activities, that was supported under high rates of anthropogenic N deposition when N deposition is reduced. We used a long-term field experiment that received experimental N deposition for 24-years across the Great Lakes region of North America. We measured soil C and N content, microbial respiration, and extracellular enzyme activity potential in the organic and mineral soil horizons and compared standardized effect sizes of soil C in response to the N deposition treatment across the history of our experiment (1994-2022). Our results demonstrate that the accumulated C in the organic horizon from experimental N deposition has been lost and exhibit additional deficits 5-years following the return to ambient N deposition and is, in part, the result of mechanistic changes in microbial activity. Whereas, soil C in the mineral soil is being lost to a lesser extent. We suspect the organic horizon dynamics that we document may occur in other temperate forests, potentially reducing the Northern Hemisphere C sink and enhancing climate warming.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564688">
        <p class="paperTitle">Terrestrial land cover shapes fish diversity in major subtropical rivers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, H.; Blackman, R. C.; Furrer, R.; Osathanunkul, M.; Brantschen, J.; Di Muri, C.; Harper, L. R.; Haenfling, B.; Niklaus, P. A.; Pellissier, L.; Schaepman, M. E.; Zong, S.; Altermatt, F.</p>
        <p class="info">Score: 7.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564688' target='https://doi.org/10.1101/2023.10.30.564688'> 10.1101/2023.10.30.564688</a></p>
        <p class="abstract">Freshwater biodiversity is critically affected by human modifications of terrestrial land use and land cover (LULC)1,2. Yet, knowledge of the spatial extent and magnitude of LULC-aquatic biodiversity linkages is still surprisingly limited, impeding the implementation of optimal management strategies3. Here, we compiled fish diversity data across a 160,000-km2 subtropical river catchment in Thailand characterized by exceptional biodiversity4 yet intense anthropogenic alterations5, and attributed fish species richness and community composition to contemporary terrestrial LULC across the catchment. We estimated a spatial range of LULC effects extending up to about 20 km upstream from sampling sites, and explained nearly 60 % of the variance in the observed species richness, associated with major LULC categories including croplands, forest, and urban areas. We find that integrating both spatial range and magnitudes of LULC effects is needed to accurately predict fish species richness. Further, projected LULC changes showcase future gains and losses of fish species richness across the river network and offer a scalable basis for riverine biodiversity conservation and land management, allowing for potential mitigation of biodiversity loss in highly diverse yet data-deficient tropical to sub-tropical riverine habitats.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564785">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564785" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564785">
        <p class="paperTitle">Unveiling Urban Complexity: Integrating OpenStreetMap to enhance representation of fine-scale landscape heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564785" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564785" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gelmi-Candusso, T. A.; Rodriguez, P.; Fortin, M.-J.</p>
        <p class="info">Score: 6.9, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564785' target='https://doi.org/10.1101/2023.10.31.564785'> 10.1101/2023.10.31.564785</a></p>
        <p class="abstract">Landscape heterogeneity has an impact on wildlife behavior, their interactions, and their persistence. Urban landscapes are among the world&#39;s most heterogeneous landscapes, yet current global landcover maps classify developed land in a single landcover type. This limits the spatial scale at which urban ecologists can approach research questions. OpenStreetMap (OSM), an open-source mapping platform, can be leveraged to enhance the representation of landscape heterogeneity in developed areas. For this, we extracted OSM features with attributes representing infrastructure, land use and green cover, integrating these into a continental landcover map through a globally applicable computational framework. We validated our OSM-enhanced landcover layer against existing remote sensing, aerial photography, and local governmental maps for 33 cities in North America. Our framework&#39;s output provides an 89% accurate representation of landscape heterogeneity. We discuss caveats, potential improvements, and ecological applications. Our OSM-based landcover enhancement framework will facilitate the use of open-source landscape information for improved ecological modeling and urban planning.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
