<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="GIFT an R package to access the Global Inventory of Floras and Traits
Authors: Denelle, P.; Weigelt, P.; Kreft, H.
Score: 168.0, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704
1. Advancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="GIFT an R package to access the Global Inventory of Floras and Traits
Authors: Denelle, P.; Weigelt, P.; Kreft, H.
Score: 168.0, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704
1. Advancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="GIFT an R package to access the Global Inventory of Floras and Traits
Authors: Denelle, P.; Weigelt, P.; Kreft, H.
Score: 168.0, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704
1. Advancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "GIFT an R package to access the Global Inventory of Floras and Traits\nAuthors: Denelle, P.; Weigelt, P.; Kreft, H.\nScore: 168.0, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704\n1. Advancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale.",
  "keywords": [
    
  ],
  "articleBody": " GIFT an R package to access the Global Inventory of Floras and Traits\nAuthors: Denelle, P.; Weigelt, P.; Kreft, H.\nScore: 168.0, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704\n1. Advancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale. 2. The Global Inventory of Floras and Traits (GIFT) is a global database of regional plant checklists that has proven successful in documenting biogeographical and geographical patterns of plants. Since the release of the first version of GIFT, the database kept on expanding. We introduce GIFT version 3.0, which contains 5,169 checklists referring to 3,400 regions. These checklists include a total of 371,148 land plant species, mostly vascular plants, of which 354,848 are accepted species names, and 109 functional traits. This new version uses new resources for taxonomic name standardization, is matched to a new plant phylogeny, comes with a new trait aggregation workflow, and includes additional environmental variables. 3. We also present the GIFT R-package, which contains all necessary functions to retrieve distributional, functional, phylogenetic, and environmental data from the GIFT database. The package comes with a dedicated website, https://biogeomacro.github.io/GIFT/, which includes three rich vignettes to guide users in retrieving data from GIFT. 4. The recent development of GIFT and its associated R-package provide ecologists with access to one of the largest plant databases. This will foster research into regional to global patterns of plant diversity and their underlying mechanisms. Proper versioning of the database and the ability to retrieve and cite data from any previous and current instance of the GIFT database will ensure the reproducibility of studies that utilize it.\nRapid species discovery and identification with real-time barcoding facilitated by ONTbarcoder 2.0 and Oxford Nanopore R10.4\nAuthors: Srivathsan, A.; Feng, V.; Ramos, D.; Meier, R.\nScore: 47.5, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546538\nMost animal species are undescribed and hidden in biodiversity samples consisting of thousands of specimens that are difficult to sort to species using morphological characters. Sorting with DNA barcodes is an attractive alternative, but needs cost-effective techniques that are suitable for use in many laboratories around the world so that barcoding can be made widely available. Oxford Nanopore's portable and inexpensive MinION sequencers are a good option because they allow for reducing the sequencing and capital-cost of a fully functional barcoding laboratory. We here provide an update to the MinION barcoding software ONTbarcoder that is available for commonly used operating systems and includes a GUI. We show that the read quality obtained with ONT's latest basecallers and chemistry (R10.4 flowcells, Q20+ kits) allow for three key improvements. Firstly, ONTbarcoder can now deliver real-time barcoding to complement ONT's real-time sequencing. This means that the first barcodes are obtained within minutes of starting a sequencing run and a significant proportion of barcodes are available before sequencing completes. This real-time feature of ONTbarcoder can be configured to handle both Mk1B and Mk1C. The input is a demultiplexing sheet and base-called or raw data. Secondly, the improved read quality of ONT's latest flow cells (R10.4) allows for the use of primers with shorter indices than what was previously needed (9bp vs. 12-13bp). This decreases primer cost and improves PCR success rates. Thirdly, we demonstrate that the availability of R10.4 chemistry in the low-cost Flongle flowcell is an attractive option for users who require only 200-250 barcodes at a time.\nCan higher-order interactions resolve the species coexistence paradox?\nAuthors: Gibbs, T. L.; Gellner, G.; Levin, S. A.; McCann, K. S.; Hastings, A.; Levine, J. M.\nScore: 14.5, Published: 2023-06-23 DOI: 10.1101/2023.06.19.545649\nMost ecological models are based on the assumption that species interact in pairs. Emerging in diverse communities, however, are higher-order interactions, in which two or more species jointly impact the growth of a third species. A pitfall of the pairwise approach is that it misses the higher-order interactions potentially responsible for maintaining diversity in nature. Here, we explore how well higher-order interactions enable coexistence when pairwise interactions are insufficient to do so. Specifically, we explore the stability properties of systems where higher-order interactions guarantee that a specified set of abundances are a feasible equilibrium of the dynamics. Even these higher-order interactions do not necessarily produce robust coexistence. We find that facilitative higher-order interactions that counter pairwise competitive interactions are less likely to generate stable equilibria than competitive interactions countering pairwise facilitation. In both cases the system is more likely to be stable when the pairwise interactions are weak, and when the equilibrium abundances are near those produced by the pairwise interactions. Last, we show that correlations between the pairwise and higher-order interactions permit robust coexistence in diverse systems. Our work reveals the challenges in generating stable coexistence through higher-order interactions, but also uncovers higher-order interaction patterns that can enable diversity.\nGenome-scale community modelling reveals conserved metabolic cross-feedings in epipelagic bacterioplankton communities\nAuthors: Giordano, N.; Gaudin, M.; Trottier, C.; Delage, E.; Nef, C.; Bowler, C.; Chaffron, S.\nScore: 13.9, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545869\nMarine microorganisms form complex communities of interacting organisms that influence central ecosystem functions in the ocean such as primary production and nutrient cycling. Identifying the mechanisms controlling their assembly and activities is a major challenge in microbial ecology. Here, we integrated Tara Oceans meta-omics data to predict genome-scale community interactions within prokaryotic assemblages in the euphotic ocean. A global genome-resolved co-activity network revealed a significant number of inter-lineage associations across large phylogenetic distances. Identified co-active communities included species displaying smaller genomes but encoding a higher potential for quorum sensing, biofilm formation, and secondary metabolism. Community metabolic modelling revealed a higher potential for interaction within co-active communities and pointed towards conserved metabolic cross-feedings, in particular of specific amino acids and group B vitamins. Our integrated ecological and metabolic modelling approach indicates genome streamlining and metabolic auxotrophies as central joint mechanisms shaping bacterioplankton community assembly in the surface global ocean.\nMicrobial Associates of the Elm Leaf Beetle: Uncovering the Absence of Resident Bacteria and the Influence of Fungi on Insect Performance\nAuthors: Schott, J.; Rakei, J.; Remus-Emsermann, M. N.; Johnston, P. R.; Mbedi, S.; Sparmann, S.; Hilker, M.; Paniagua Voirol, L. R.\nScore: 6.5, Published: 2023-06-28 DOI: 10.1101/2023.06.23.546294\nMicrobial symbionts play crucial roles in the biology of many insects. While bacteria have been the primary focus of research on insect-microbe symbiosis, recent studies suggest that fungal symbionts may be just as important. The elm leaf beetle (ELB, Xanthogaleruca luteola) is a serious pest species of elm (Ulmus minor). Using culture-dependent and independent methods, we investigated the abundance and species richness of bacteria and fungi throughout various ELB life stages and generations, while concurrently analysing microbial communities on elm leaves. No persistent bacterial community was found to be associated with the ELB or elm leaves. By contrast, fungi were persistently present in the beetles feeding life stages and on elm leaves. Fungal community sequencing revealed a predominance of the genera Penicillium and Aspergillus in insects and on leaves. Culture-dependent surveys showed a high prevalence of two fungal colony morphotypes closely related to Penicillium lanosocoeruleum and Aspergillus flavus. Among these, the Penicillium morphotype was significantly more abundant on feeding-damaged compared to intact leaves, suggesting that the fungus thrives in the presence of the ELB. We assessed whether the detected prevalent fungal morphotypes influenced ELBs performance by rearing insects on i) surface-sterilised leaves, ii) leaves inoculated with Penicillium spores, and iii) leaves inoculated with Aspergillus spores. Insects feeding on Penicillium-inoculated leaves gained more biomass and tended to lay larger egg clutches than those consuming surface-sterilised leaves or Aspergillus-inoculated leaves. Our results demonstrate that the ELB does not harbour resident bacteria and that it might benefit from associating with Penicillium fungi. ImportanceOur study provides insights into the still understudied role of microbial symbionts in the biology of the ELB, a major pest of elms. Contrary to expectations, we found no persistent bacterial symbionts associated with the ELB or elm leaves. Our research thus contributes to the growing body of knowledge that not all insects rely on bacterial symbionts. While no persistent bacterial symbionts were detectable in the ELB and elm leaf samples, our analyses revealed the persistent presence of fungi, particularly Penicillium and Aspergillus on both elm leaves and in the feeding ELB stages. Moreover, when ELB were fed with fungus-treated elm leaves, we detected a potentially beneficial effect of Penicillium on the ELBs development and fecundity. Our results highlight the significance of fungal symbionts in the biology of this insect.\nNew resources for environmental genomics in the wild Mediterranean grass B. distachyon\nAuthors: Minadakis, N.; Williams, H.; Horvath, R.; Cakovic, D.; Stritt, C.; Thieme, M.; Bourgeois, Y. X. C.; Roulin, A.\nScore: 18.4, Published: 2023-06-13 DOI: 10.1101/2023.06.01.543285\nClosely related to economically important crops, the grass Brachypodium distachyon has been originally established as a pivotal species for grass genomics but more recently flourished as a model for developmental biology. Grasses encompass more than 10,000 species and cover more than 40% of the world land area from tropical to temperate regions. Unlocking the sources of phenotypic variation in B. distachyon is hence of prime interest in fundamental and applied research in agronomy, ecology and evolution. We present here the B. distachyon diversity panel, which encompasses 332 fully sequenced accessions covering the whole species distribution from Spain to Iraq. By combining population genetics, niche modeling and landscape genomics, we suggest that B. distachyon recolonized Europe and the Middle East following the last glacial maximum. Consequently, the species faced new environmental conditions which led to clear associations between bioclimatic variables and genetic factors as well as footprints of positive selection in the genome. Altogether, this genomic resource offers a powerful alternative to Arabidopsis thaliana to investigate the genetic bases of adaptation and phenotypic plasticity in plants and more specifically in monocots.\nMammal life history strategies are strongly associated with environmental realms\nAuthors: Beccari, E.; Capdevila Lanzaco, P.; Salguero-Gomez, R.; Perez Carmona, C.\nScore: 5.7, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546851\nMammalian investments in survival, development, and reproduction are constrained by physiological and evolutionary factors, so their life history strategies can be characterized by a few axes of variation. These axes conform a space in which species are positioned according to which life history strategies are favoured in the environment they exploit. Yet, a global synthesis quantifying the realized mammalian life history diversity is lacking, and exploring how the environment shapes mammalian life history strategies is missing. Here, we used six life history traits to build a global life history space and explored how major environmental realms (land, air, water) influence mammalian life history strategies. We demonstrate that realms are tightly linked to distinct life history strategies. Predominantly, aquatic and aerial species adhere to slower life history strategies, while terrestrial species tend to exhibit faster life histories. Highly encephalized terrestrial species, like monkeys, are a notable exception to these patterns. In addition, species transitioning between the terrestrial and aquatic realms, such as seals and otters, show intermediate life history strategies. Furthermore, time since evolutionary divergence from land is a strong predictor of aquatic life history strategies. Our results provide compelling evidence linking environmental realms to the diversity of life history strategies among mammals, potentially informing assessments of species viability.\nWildlife vaccination strategies for eliminating bovine tuberculosis at the wildlife-livestock interface\nAuthors: Pandey, A.; Feuka, A. B.; Cosgrove, M.; Moriarty, M.; Duffiney, A.; VerCauteren, K. C.; Campa, H.; Pepin, K. M.\nScore: 2.9, Published: 2023-06-25 DOI: 10.1101/2023.06.22.546040\nMany pathogens of humans and livestock also infect wildlife that can act as a reservoir and challenge disease control or elimination. Efficient and effective prioritization of research and management actions requires an understanding of the potential for new tools to improve elimination probability with feasible deployment strategies that can be implemented at scale. Wildlife vaccination is gaining interest as a tool for managing several wildlife diseases. To evaluate the effect of vaccinating white-tailed deer (Odocoileus virginianus), in combination with harvest, in reducing and eliminating bovine tuberculosis from deer populations in Michigan, we developed a mechanistic age-structured disease transmission model for bovine tuberculosis with integrated disease management. We evaluated the impact of pulse vaccination across a range of vaccine properties. Pulse vaccination was effective for reducing disease prevalence rapidly with even low (30%) to moderate (60%) vaccine coverage of the susceptible and exposed deer population and was further improved when combined with increased harvest. The impact of increased harvest depended on the relative strength of transmission modes, i.e., direct vs indirect transmission. Vaccine coverage and efficacy were the most important vaccine properties for reducing and eliminating disease from the local population. By fitting the model to the core endemic area of bovine tuberculosis in Michigan, USA, we identified feasible integrated management strategies involving vaccination and increased harvest that reduced disease prevalence in free-ranging deer. Few scenarios led to disease elimination due to the chronic nature of bovine tuberculosis. A long-term commitment to regular vaccination campaigns, and further research on increasing vaccines efficacy and uptake rate in free-ranging deer are important for disease management.\nRelationship between the colors of the rivers in the Amazon and the incidence of malaria\nAuthors: Fonseca, F.; Martinez, J.-M.; Balieiro, A.; Orellana, J.; Filizola, N.\nScore: 2.2, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547004\nBackground: Malaria is an infection caused by parasites of the genus Plasmodium, which are transmitted to humans via the bite of the female Anopheles mosquito. In Brazil, approximately 99% of malaria cases are concentrated in the Amazon region. Os rios desempenham um papel importante no ciclo de vida da malaria, uma vez que o vetor se reproduz em ambiente aquatico. The waters of the rivers in the Amazon have distinct chemical characteristics and this prompted us to analyze the influence of the color of the waters of the rivers in the Amazon on the distribution of malaria. Methodology: This study was conducted for a period of seventeen years (2003-2019) in 50 municipalities in the state of Amazonas, Brazil. A generalized linear mixed model was developed to analyze the association of malaria incidence and three types of river color: white, black and mixed. Principal Findings: The results suggest that there is a trend towards a decrease in malaria cases until 2015, with a possible resumption of the incidence of the disease from 2017 onwards, in all types of river color. In addition, the research indicates that places located near black- or mixed-water rivers have a higher incidence of malaria when compared to places on the banks of white-water rivers. Conclusions: Historically, the hydrological regime has played an important role in the dynamics of malaria in the Amazon, but little is known about the relationship between river colors and the incidence of the disease. In this sense, our results, by showing a significant association between the colors of the rivers and the incidence of malaria over time, seem to expand the understanding between physical-chemical characteristics of the rivers and the occurrence of malaria.\nIs it selfish to be filamentous in biofilms? Individual-based modeling links microbial growth strategies with morphology using the new and modular iDynoMiCS 2.0\nAuthors: Cockx, B. J. R.; Foster, T.; Clegg, R. J.; Alden, K.; Arya, S.; Stekel, D. J.; Smets, B. F.; Kreft, J.-U.\nScore: 1.8, Published: 2023-06-30 DOI: 10.1101/2023.06.27.546816\nMicrobial communities are found in all habitable environments and often occur in assemblages with self-organized spatial structures developing over time. This complexity can only be understood, predicted, and managed by combining experiments with mathematical modeling. Individual-based models are particularly suited if individual heterogeneity, local interactions, and adaptive behavior are of interest. Here we present the completely overhauled software platform, the individual-based Dynamics of Microbial Communities Simulator, iDynoMiCS 2.0, which enables researchers to specify a range of different models without having to program. Key new features and improvements are: (1) Substantially enhanced ease of use (graphical user interface, editor for model specification, unit conversions, data analysis and visualization and more). (2) Increased performance and scalability enabling simulations of up to 10 million agents in 3D biofilms. (3) Kinetics can be specified with any arithmetic function. (4) Agent properties can be assembled from orthogonal modules for pick and mix flexibility. (5) Force-based mechanical interaction framework enabling attractive forces and non-spherical agent morphologies as an alternative to the shoving algorithm. The new iDynoMiCS 2.0 has undergone intensive testing, from unit tests to a suite of increasingly complex numerical tests and the standard Benchmark 3 based on nitrifying biofilms. A second test case was based on the \"biofilms promote altruism\" study previously implemented in BacSim because competition outcomes are highly sensitive to the developing spatial structures due to positive feedback between cooperative individuals. We extended this case study by adding morphology to find that (i) filamentous bacteria outcompete spherical bacteria regardless of growth strategy and (ii) non-cooperating filaments outcompete cooperating filaments because filaments can escape the stronger competition between themselves. In conclusion, the new substantially improved iDynoMiCS 2.0 joins a growing number of platforms for individual-based modeling of microbial communities with specific advantages and disadvantages that we discuss, giving users a wider choice.\n",
  "wordCount" : "2848",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546704">
        <p class="paperTitle">GIFT an R package to access the Global Inventory of Floras and Traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Denelle, P.; Weigelt, P.; Kreft, H.</p>
        <p class="info">Score: 168.0, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546704' target='https://doi.org/10.1101/2023.06.27.546704'> 10.1101/2023.06.27.546704</a></p>
        <p class="abstract">1. Advancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale. 2. The Global Inventory of Floras and Traits (GIFT) is a global database of regional plant checklists that has proven successful in documenting biogeographical and geographical patterns of plants. Since the release of the first version of GIFT, the database kept on expanding. We introduce GIFT version 3.0, which contains 5,169 checklists referring to 3,400 regions. These checklists include a total of 371,148 land plant species, mostly vascular plants, of which 354,848 are accepted species names, and 109 functional traits. This new version uses new resources for taxonomic name standardization, is matched to a new plant phylogeny, comes with a new trait aggregation workflow, and includes additional environmental variables. 3. We also present the GIFT R-package, which contains all necessary functions to retrieve distributional, functional, phylogenetic, and environmental data from the GIFT database. The package comes with a dedicated website, https://biogeomacro.github.io/GIFT/, which includes three rich vignettes to guide users in retrieving data from GIFT. 4. The recent development of GIFT and its associated R-package provide ecologists with access to one of the largest plant databases. This will foster research into regional to global patterns of plant diversity and their underlying mechanisms. Proper versioning of the database and the ability to retrieve and cite data from any previous and current instance of the GIFT database will ensure the reproducibility of studies that utilize it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546538">
        <p class="paperTitle">Rapid species discovery and identification with real-time barcoding facilitated by ONTbarcoder 2.0 and Oxford Nanopore R10.4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivathsan, A.; Feng, V.; Ramos, D.; Meier, R.</p>
        <p class="info">Score: 47.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546538' target='https://doi.org/10.1101/2023.06.26.546538'> 10.1101/2023.06.26.546538</a></p>
        <p class="abstract">Most animal species are undescribed and hidden in biodiversity samples consisting of thousands of specimens that are difficult to sort to species using morphological characters. Sorting with DNA barcodes is an attractive alternative, but needs cost-effective techniques that are suitable for use in many laboratories around the world so that barcoding can be made widely available. Oxford Nanopore&#39;s portable and inexpensive MinION sequencers are a good option because they allow for reducing the sequencing and capital-cost of a fully functional barcoding laboratory. We here provide an update to the MinION barcoding software ONTbarcoder that is available for commonly used operating systems and includes a GUI. We show that the read quality obtained with ONT&#39;s latest basecallers and chemistry (R10.4 flowcells, Q20&#43; kits) allow for three key improvements. Firstly, ONTbarcoder can now deliver real-time barcoding to complement ONT&#39;s real-time sequencing. This means that the first barcodes are obtained within minutes of starting a sequencing run and a significant proportion of barcodes are available before sequencing completes. This real-time feature of ONTbarcoder can be configured to handle both Mk1B and Mk1C. The input is a demultiplexing sheet and base-called or raw data. Secondly, the improved read quality of ONT&#39;s latest flow cells (R10.4) allows for the use of primers with shorter indices than what was previously needed (9bp vs. 12-13bp). This decreases primer cost and improves PCR success rates. Thirdly, we demonstrate that the availability of R10.4 chemistry in the low-cost Flongle flowcell is an attractive option for users who require only 200-250 barcodes at a time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545649">
        <p class="paperTitle">Can higher-order interactions resolve the species coexistence paradox?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibbs, T. L.; Gellner, G.; Levin, S. A.; McCann, K. S.; Hastings, A.; Levine, J. M.</p>
        <p class="info">Score: 14.5, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545649' target='https://doi.org/10.1101/2023.06.19.545649'> 10.1101/2023.06.19.545649</a></p>
        <p class="abstract">Most ecological models are based on the assumption that species interact in pairs. Emerging in diverse communities, however, are higher-order interactions, in which two or more species jointly impact the growth of a third species. A pitfall of the pairwise approach is that it misses the higher-order interactions potentially responsible for maintaining diversity in nature. Here, we explore how well higher-order interactions enable coexistence when pairwise interactions are insufficient to do so. Specifically, we explore the stability properties of systems where higher-order interactions guarantee that a specified set of abundances are a feasible equilibrium of the dynamics. Even these higher-order interactions do not necessarily produce robust coexistence. We find that facilitative higher-order interactions that counter pairwise competitive interactions are less likely to generate stable equilibria than competitive interactions countering pairwise facilitation. In both cases the system is more likely to be stable when the pairwise interactions are weak, and when the equilibrium abundances are near those produced by the pairwise interactions. Last, we show that correlations between the pairwise and higher-order interactions permit robust coexistence in diverse systems. Our work reveals the challenges in generating stable coexistence through higher-order interactions, but also uncovers higher-order interaction patterns that can enable diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545869">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545869" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545869">
        <p class="paperTitle">Genome-scale community modelling reveals conserved metabolic cross-feedings in epipelagic bacterioplankton communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545869" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545869" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giordano, N.; Gaudin, M.; Trottier, C.; Delage, E.; Nef, C.; Bowler, C.; Chaffron, S.</p>
        <p class="info">Score: 13.9, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545869' target='https://doi.org/10.1101/2023.06.21.545869'> 10.1101/2023.06.21.545869</a></p>
        <p class="abstract">Marine microorganisms form complex communities of interacting organisms that influence central ecosystem functions in the ocean such as primary production and nutrient cycling. Identifying the mechanisms controlling their assembly and activities is a major challenge in microbial ecology. Here, we integrated Tara Oceans meta-omics data to predict genome-scale community interactions within prokaryotic assemblages in the euphotic ocean. A global genome-resolved co-activity network revealed a significant number of inter-lineage associations across large phylogenetic distances. Identified co-active communities included species displaying smaller genomes but encoding a higher potential for quorum sensing, biofilm formation, and secondary metabolism. Community metabolic modelling revealed a higher potential for interaction within co-active communities and pointed towards conserved metabolic cross-feedings, in particular of specific amino acids and group B vitamins. Our integrated ecological and metabolic modelling approach indicates genome streamlining and metabolic auxotrophies as central joint mechanisms shaping bacterioplankton community assembly in the surface global ocean.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546294">
        <p class="paperTitle">Microbial Associates of the Elm Leaf Beetle: Uncovering the Absence of Resident Bacteria and the Influence of Fungi on Insect Performance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, J.; Rakei, J.; Remus-Emsermann, M. N.; Johnston, P. R.; Mbedi, S.; Sparmann, S.; Hilker, M.; Paniagua Voirol, L. R.</p>
        <p class="info">Score: 6.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546294' target='https://doi.org/10.1101/2023.06.23.546294'> 10.1101/2023.06.23.546294</a></p>
        <p class="abstract">Microbial symbionts play crucial roles in the biology of many insects. While bacteria have been the primary focus of research on insect-microbe symbiosis, recent studies suggest that fungal symbionts may be just as important. The elm leaf beetle (ELB, Xanthogaleruca luteola) is a serious pest species of elm (Ulmus minor). Using culture-dependent and independent methods, we investigated the abundance and species richness of bacteria and fungi throughout various ELB life stages and generations, while concurrently analysing microbial communities on elm leaves. No persistent bacterial community was found to be associated with the ELB or elm leaves. By contrast, fungi were persistently present in the beetles feeding life stages and on elm leaves. Fungal community sequencing revealed a predominance of the genera Penicillium and Aspergillus in insects and on leaves. Culture-dependent surveys showed a high prevalence of two fungal colony morphotypes closely related to Penicillium lanosocoeruleum and Aspergillus flavus. Among these, the Penicillium morphotype was significantly more abundant on feeding-damaged compared to intact leaves, suggesting that the fungus thrives in the presence of the ELB. We assessed whether the detected prevalent fungal morphotypes influenced ELBs performance by rearing insects on i) surface-sterilised leaves, ii) leaves inoculated with Penicillium spores, and iii) leaves inoculated with Aspergillus spores. Insects feeding on Penicillium-inoculated leaves gained more biomass and tended to lay larger egg clutches than those consuming surface-sterilised leaves or Aspergillus-inoculated leaves. Our results demonstrate that the ELB does not harbour resident bacteria and that it might benefit from associating with Penicillium fungi.

ImportanceOur study provides insights into the still understudied role of microbial symbionts in the biology of the ELB, a major pest of elms. Contrary to expectations, we found no persistent bacterial symbionts associated with the ELB or elm leaves. Our research thus contributes to the growing body of knowledge that not all insects rely on bacterial symbionts. While no persistent bacterial symbionts were detectable in the ELB and elm leaf samples, our analyses revealed the persistent presence of fungi, particularly Penicillium and Aspergillus on both elm leaves and in the feeding ELB stages. Moreover, when ELB were fed with fungus-treated elm leaves, we detected a potentially beneficial effect of Penicillium on the ELBs development and fecundity. Our results highlight the significance of fungal symbionts in the biology of this insect.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543285">
        <p class="paperTitle">New resources for environmental genomics in the wild Mediterranean grass B. distachyon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minadakis, N.; Williams, H.; Horvath, R.; Cakovic, D.; Stritt, C.; Thieme, M.; Bourgeois, Y. X. C.; Roulin, A.</p>
        <p class="info">Score: 18.4, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543285' target='https://doi.org/10.1101/2023.06.01.543285'> 10.1101/2023.06.01.543285</a></p>
        <p class="abstract">Closely related to economically important crops, the grass Brachypodium distachyon has been originally established as a pivotal species for grass genomics but more recently flourished as a model for developmental biology. Grasses encompass more than 10,000 species and cover more than 40% of the world land area from tropical to temperate regions. Unlocking the sources of phenotypic variation in B. distachyon is hence of prime interest in fundamental and applied research in agronomy, ecology and evolution. We present here the B. distachyon diversity panel, which encompasses 332 fully sequenced accessions covering the whole species distribution from Spain to Iraq. By combining population genetics, niche modeling and landscape genomics, we suggest that B. distachyon recolonized Europe and the Middle East following the last glacial maximum. Consequently, the species faced new environmental conditions which led to clear associations between bioclimatic variables and genetic factors as well as footprints of positive selection in the genome. Altogether, this genomic resource offers a powerful alternative to Arabidopsis thaliana to investigate the genetic bases of adaptation and phenotypic plasticity in plants and more specifically in monocots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546851">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546851" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546851">
        <p class="paperTitle">Mammal life history strategies are strongly associated with environmental realms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546851" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546851" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beccari, E.; Capdevila Lanzaco, P.; Salguero-Gomez, R.; Perez Carmona, C.</p>
        <p class="info">Score: 5.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546851' target='https://doi.org/10.1101/2023.06.29.546851'> 10.1101/2023.06.29.546851</a></p>
        <p class="abstract">Mammalian investments in survival, development, and reproduction are constrained by physiological and evolutionary factors, so their life history strategies can be characterized by a few axes of variation. These axes conform a space in which species are positioned according to which life history strategies are favoured in the environment they exploit. Yet, a global synthesis quantifying the realized mammalian life history diversity is lacking, and exploring how the environment shapes mammalian life history strategies is missing. Here, we used six life history traits to build a global life history space and explored how major environmental realms (land, air, water) influence mammalian life history strategies. We demonstrate that realms are tightly linked to distinct life history strategies. Predominantly, aquatic and aerial species adhere to slower life history strategies, while terrestrial species tend to exhibit faster life histories. Highly encephalized terrestrial species, like monkeys, are a notable exception to these patterns. In addition, species transitioning between the terrestrial and aquatic realms, such as seals and otters, show intermediate life history strategies. Furthermore, time since evolutionary divergence from land is a strong predictor of aquatic life history strategies. Our results provide compelling evidence linking environmental realms to the diversity of life history strategies among mammals, potentially informing assessments of species viability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546040">
        <p class="paperTitle">Wildlife vaccination strategies for eliminating bovine tuberculosis at the wildlife-livestock interface</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandey, A.; Feuka, A. B.; Cosgrove, M.; Moriarty, M.; Duffiney, A.; VerCauteren, K. C.; Campa, H.; Pepin, K. M.</p>
        <p class="info">Score: 2.9, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546040' target='https://doi.org/10.1101/2023.06.22.546040'> 10.1101/2023.06.22.546040</a></p>
        <p class="abstract">Many pathogens of humans and livestock also infect wildlife that can act as a reservoir and challenge disease control or elimination. Efficient and effective prioritization of research and management actions requires an understanding of the potential for new tools to improve elimination probability with feasible deployment strategies that can be implemented at scale. Wildlife vaccination is gaining interest as a tool for managing several wildlife diseases. To evaluate the effect of vaccinating white-tailed deer (Odocoileus virginianus), in combination with harvest, in reducing and eliminating bovine tuberculosis from deer populations in Michigan, we developed a mechanistic age-structured disease transmission model for bovine tuberculosis with integrated disease management. We evaluated the impact of pulse vaccination across a range of vaccine properties. Pulse vaccination was effective for reducing disease prevalence rapidly with even low (30%) to moderate (60%) vaccine coverage of the susceptible and exposed deer population and was further improved when combined with increased harvest. The impact of increased harvest depended on the relative strength of transmission modes, i.e., direct vs indirect transmission. Vaccine coverage and efficacy were the most important vaccine properties for reducing and eliminating disease from the local population. By fitting the model to the core endemic area of bovine tuberculosis in Michigan, USA, we identified feasible integrated management strategies involving vaccination and increased harvest that reduced disease prevalence in free-ranging deer. Few scenarios led to disease elimination due to the chronic nature of bovine tuberculosis. A long-term commitment to regular vaccination campaigns, and further research on increasing vaccines efficacy and uptake rate in free-ranging deer are important for disease management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547004" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547004">
        <p class="paperTitle">Relationship between the colors of the rivers in the Amazon and the incidence of malaria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fonseca, F.; Martinez, J.-M.; Balieiro, A.; Orellana, J.; Filizola, N.</p>
        <p class="info">Score: 2.2, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547004' target='https://doi.org/10.1101/2023.06.29.547004'> 10.1101/2023.06.29.547004</a></p>
        <p class="abstract">Background: Malaria is an infection caused by parasites of the genus Plasmodium, which are transmitted to humans via the bite of the female Anopheles mosquito. In Brazil, approximately 99% of malaria cases are concentrated in the Amazon region. Os rios desempenham um papel importante no ciclo de vida da malaria, uma vez que o vetor se reproduz em ambiente aquatico. The waters of the rivers in the Amazon have distinct chemical characteristics and this prompted us to analyze the influence of the color of the waters of the rivers in the Amazon on the distribution of malaria. Methodology: This study was conducted for a period of seventeen years (2003-2019) in 50 municipalities in the state of Amazonas, Brazil. A generalized linear mixed model was developed to analyze the association of malaria incidence and three types of river color: white, black and mixed. Principal Findings: The results suggest that there is a trend towards a decrease in malaria cases until 2015, with a possible resumption of the incidence of the disease from 2017 onwards, in all types of river color. In addition, the research indicates that places located near black- or mixed-water rivers have a higher incidence of malaria when compared to places on the banks of white-water rivers. Conclusions: Historically, the hydrological regime has played an important role in the dynamics of malaria in the Amazon, but little is known about the relationship between river colors and the incidence of the disease. In this sense, our results, by showing a significant association between the colors of the rivers and the incidence of malaria over time, seem to expand the understanding between physical-chemical characteristics of the rivers and the occurrence of malaria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546816">
        <p class="paperTitle">Is it selfish to be filamentous in biofilms? Individual-based modeling links microbial growth strategies with morphology using the new and modular iDynoMiCS 2.0</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cockx, B. J. R.; Foster, T.; Clegg, R. J.; Alden, K.; Arya, S.; Stekel, D. J.; Smets, B. F.; Kreft, J.-U.</p>
        <p class="info">Score: 1.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546816' target='https://doi.org/10.1101/2023.06.27.546816'> 10.1101/2023.06.27.546816</a></p>
        <p class="abstract">Microbial communities are found in all habitable environments and often occur in assemblages with self-organized spatial structures developing over time. This complexity can only be understood, predicted, and managed by combining experiments with mathematical modeling. Individual-based models are particularly suited if individual heterogeneity, local interactions, and adaptive behavior are of interest. Here we present the completely overhauled software platform, the individual-based Dynamics of Microbial Communities Simulator, iDynoMiCS 2.0, which enables researchers to specify a range of different models without having to program. Key new features and improvements are: (1) Substantially enhanced ease of use (graphical user interface, editor for model specification, unit conversions, data analysis and visualization and more). (2) Increased performance and scalability enabling simulations of up to 10 million agents in 3D biofilms. (3) Kinetics can be specified with any arithmetic function. (4) Agent properties can be assembled from orthogonal modules for pick and mix flexibility. (5) Force-based mechanical interaction framework enabling attractive forces and non-spherical agent morphologies as an alternative to the shoving algorithm. The new iDynoMiCS 2.0 has undergone intensive testing, from unit tests to a suite of increasingly complex numerical tests and the standard Benchmark 3 based on nitrifying biofilms. A second test case was based on the &#34;biofilms promote altruism&#34; study previously implemented in BacSim because competition outcomes are highly sensitive to the developing spatial structures due to positive feedback between cooperative individuals. We extended this case study by adding morphology to find that (i) filamentous bacteria outcompete spherical bacteria regardless of growth strategy and (ii) non-cooperating filaments outcompete cooperating filaments because filaments can escape the stronger competition between themselves. In conclusion, the new substantially improved iDynoMiCS 2.0 joins a growing number of platforms for individual-based modeling of microbial communities with specific advantages and disadvantages that we discuss, giving users a wider choice.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
