<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>ecology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>ecology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="GIFT an R package to access the Global Inventory of Floras and Traits
Authors: Denelle, P.; Weigelt, P.; Kreft, H.
Score: 178.6, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704
O_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/ecology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="ecology" />
<meta property="og:description" content="GIFT an R package to access the Global Inventory of Floras and Traits
Authors: Denelle, P.; Weigelt, P.; Kreft, H.
Score: 178.6, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704
O_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/ecology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ecology"/>
<meta name="twitter:description" content="GIFT an R package to access the Global Inventory of Floras and Traits
Authors: Denelle, P.; Weigelt, P.; Kreft, H.
Score: 178.6, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704
O_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "ecology",
      "item": "https://trxiv.yorks0n.com/posts/ecology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "ecology",
  "name": "ecology",
  "description": "GIFT an R package to access the Global Inventory of Floras and Traits\nAuthors: Denelle, P.; Weigelt, P.; Kreft, H.\nScore: 178.6, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704\nO_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale.",
  "keywords": [
    
  ],
  "articleBody": " GIFT an R package to access the Global Inventory of Floras and Traits\nAuthors: Denelle, P.; Weigelt, P.; Kreft, H.\nScore: 178.6, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546704\nO_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale. C_LIO_LIThe Global Inventory of Floras and Traits (GIFT) is a global database of regional plant checklists that has proven successful in documenting biogeographical and geographical patterns of plants. Since the release of the first version of GIFT, the database kept on expanding. We introduce GIFT version 3.0, which contains 5,169 checklists referring to 3,400 regions. These checklists include a total of 371,148 land plant species, mostly vascular plants, of which 354,848 are accepted species names, and 109 functional traits. This new version uses new resources for taxonomic name standardization, is matched to a new plant phylogeny, comes with a new trait aggregation workflow, and includes additional environmental variables. C_LIO_LIWe also present the GIFT R-package, which contains all necessary functions to retrieve distributional, functional, phylogenetic, and environmental data from the GIFT database. The package comes with a dedicated website, https://biogeomacro.github.io/GIFT/, which includes three rich vignettes to guide users in retrieving data from GIFT. C_LIO_LIThe recent development of GIFT and its associated R-package provide ecologists with access to one of the largest plant databases. This will foster research into regional to global patterns of plant diversity and their underlying mechanisms. Proper versioning of the database and the ability to retrieve and cite data from any previous and current instance of the GIFT database will ensure the reproducibility of studies that utilize it. C_LI\nRapid species discovery and identification with real-time barcoding facilitated by ONTbarcoder 2.0 and Oxford Nanopore R10.4\nAuthors: Srivathsan, A.; Feng, V.; Ramos, D.; Meier, R.\nScore: 63.4, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546538\nMost animal species are undescribed and hidden in biodiversity samples consisting of thousands of specimens that are difficult to sort to species using morphological characters. Sorting with DNA barcodes is an attractive alternative, but needs cost-effective techniques that are suitable for use in many laboratories around the world so that barcoding can be made widely available. Oxford Nanopores portable and inexpensive MinION sequencers are a good option because they allow for reducing the sequencing and capital-cost of a fully functional barcoding laboratory. We here provide an update to the MinION barcoding software ONTbarcoder that is available for commonly used operating systems and includes a GUI. We show that the read quality obtained with ONTs latest basecallers and chemistry (R10.4 flowcells, Q20+ kits) allow for three key improvements. Firstly, ONTbarcoder can now deliver real-time barcoding to complement ONTs real-time sequencing. This means that the first barcodes are obtained within minutes of starting a sequencing run and a significant proportion of barcodes are available before sequencing completes. This real-time feature of ONTbarcoder can be configured to handle both Mk1B and Mk1C. The input is a demultiplexing sheet and base-called or raw data. Secondly, the improved read quality of ONTs latest flow cells (R10.4) allows for the use of primers with shorter indices than what was previously needed (9bp vs. 12-13bp). This decreases primer cost and improves PCR success rates. Thirdly, we demonstrate that the availability of R10.4 chemistry in the low-cost Flongle flowcell is an attractive option for users who require only 200-250 barcodes at a time.\nStatistical prediction of microbial metabolic traits from genomes\nAuthors: Li, Z.; Selim, A.; Kuehn, S.\nScore: 27.9, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547261\nThe metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source. Using these data as well as two existing datasets, we show that statistical approaches can accurately predict bacterial carbon utilization traits from genomes. First, we show that classifiers trained on gene content can accurately predict bacterial carbon utilization phenotypes by encoding phylogenetic information. These models substantially outperform predictions made by constraint-based metabolic models automatically constructed from genomes. However, phylogeny-based predictions fail to predict traits for taxa that are phylogenetically distant from any strains in the training set. To overcome this we train improved models on gene presence/absence to predict carbon utilization traits from gene content. We show that models that predict carbon utilization traits from gene presence/absence can generalize to taxa that are phylogenetically distant from the training set either by exploiting biochemical information for feature selection or by having sufficiently large datasets. In the latter case, we provide evidence that a statistical approach can identify putatively mechanistic genes involved in metabolic traits. Our study demonstrates the potential power for predicting microbial phenotypes from genotypes using statistical approaches.\nAbsence of Coronavirus in terns in the Western Indian Ocean?\nAuthors: Lebarbenchon, C.; Feare, C.; Larose, C.; Boucher, S.; Jaeger, A.; Le Corre, M.\nScore: 12.4, Published: 2023-07-03 DOI: 10.1101/2023.06.30.547193\nWe investigated coronavirus circulation in three tern species, in four islands of the Western Indian Ocean (Bird, Reunion, Europa, Juan de Nova). None of the 1513 samples tested positive by RT-PCR. We discuss the implication in term of host species range, ecological drivers of virus transmission, and diagnostic tools.\nMicrobial Associates of the Elm Leaf Beetle: Uncovering the Absence of Resident Bacteria and the Influence of Fungi on Insect Performance\nAuthors: Schott, J.; Rakei, J.; Remus-Emsermann, M. N.; Johnston, P. R.; Mbedi, S.; Sparmann, S.; Hilker, M.; Paniagua Voirol, L. R.\nScore: 6.5, Published: 2023-06-28 DOI: 10.1101/2023.06.23.546294\nMicrobial symbionts play crucial roles in the biology of many insects. While bacteria have been the primary focus of research on insect-microbe symbiosis, recent studies suggest that fungal symbionts may be just as important. The elm leaf beetle (ELB, Xanthogaleruca luteola) is a serious pest species of elm (Ulmus minor). Using culture-dependent and independent methods, we investigated the abundance and species richness of bacteria and fungi throughout various ELB life stages and generations, while concurrently analysing microbial communities on elm leaves. No persistent bacterial community was found to be associated with the ELB or elm leaves. By contrast, fungi were persistently present in the beetles feeding life stages and on elm leaves. Fungal community sequencing revealed a predominance of the genera Penicillium and Aspergillus in insects and on leaves. Culture-dependent surveys showed a high prevalence of two fungal colony morphotypes closely related to Penicillium lanosocoeruleum and Aspergillus flavus. Among these, the Penicillium morphotype was significantly more abundant on feeding-damaged compared to intact leaves, suggesting that the fungus thrives in the presence of the ELB. We assessed whether the detected prevalent fungal morphotypes influenced ELBs performance by rearing insects on i) surface-sterilised leaves, ii) leaves inoculated with Penicillium spores, and iii) leaves inoculated with Aspergillus spores. Insects feeding on Penicillium-inoculated leaves gained more biomass and tended to lay larger egg clutches than those consuming surface-sterilised leaves or Aspergillus-inoculated leaves. Our results demonstrate that the ELB does not harbour resident bacteria and that it might benefit from associating with Penicillium fungi. ImportanceOur study provides insights into the still understudied role of microbial symbionts in the biology of the ELB, a major pest of elms. Contrary to expectations, we found no persistent bacterial symbionts associated with the ELB or elm leaves. Our research thus contributes to the growing body of knowledge that not all insects rely on bacterial symbionts. While no persistent bacterial symbionts were detectable in the ELB and elm leaf samples, our analyses revealed the persistent presence of fungi, particularly Penicillium and Aspergillus on both elm leaves and in the feeding ELB stages. Moreover, when ELB were fed with fungus-treated elm leaves, we detected a potentially beneficial effect of Penicillium on the ELBs development and fecundity. Our results highlight the significance of fungal symbionts in the biology of this insect.\nCompetition for light can drive adverse species-composition shifts in the Amazon Forest under elevated CO2\nAuthors: Joshi, J.; Hofhansl, F.; Singh, S.; Stocker, B.; Brännström, A.; Franklin, O.; Blanco, C. C.; Aleixo, I. F.; Lapola, D. M.; Prentice, I. C.; Dieckmann, U.\nScore: 6.5, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547575\nThe resilience of biodiverse forests to climate change depends on an interplay of adaptive processes operating at multiple temporal and organizational scales. These include short-term acclimation of physiological processes like photosynthesis and respiration, mid-term changes in forest structure due to competition, and long-term changes in community composition arising from competitive exclusion and genetic trait evolution. To investigate the roles of diversity and adaptation for forest resilience, we present Plant-FATE, a parsimonious eco-evolutionary vegetation model. Tested with data from a hyperdiverse Amazonian terra-firme forest, our model accurately predicts multiple emergent ecosystem properties characterizing forest structure and function. Under elevated CO2 conditions, we predict an increase in productivity, leaf area, and aboveground biomass, with the magnitude of this increase declining in nutrient-deprived soils if trees allocate more carbon to the rhizosphere to overcome nutrient limitation. Furthermore, increased aboveground productivity leads to greater competition for light and drives a shift in community composition towards fast-growing but short-lived species characterized by lower wood densities. Such a transition reduces the carbon residence time of woody biomass, dampening carbon-sink strength and potentially rendering the Amazon Forest more vulnerable to future climatic extreme events.\nMammal life history strategies are strongly associated with environmental realms\nAuthors: Beccari, E.; Capdevila Lanzaco, P.; Salguero-Gomez, R.; Perez Carmona, C.\nScore: 5.7, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546851\nMammalian investments in survival, development, and reproduction are constrained by physiological and evolutionary factors1, so their life history strategies can be characterized by a few axes of variation2-4. These axes conform a space in which species are positioned according to which life history strategies are favoured in the environment they exploit1,5,6. Yet, a global synthesis quantifying the realized mammalian life history diversity is lacking and exploring how the environment shapes mammalian life history strategies is missing. Here, we used six life history traits to build a global life history space and explored how major environmental realms (land, air, water) influence mammalian life history strategies. We demonstrate that realms are tightly linked to distinct life history strategies. Predominantly, aquatic and aerial species adhere to slower life history strategies, while terrestrial species tend to exhibit faster life histories. Highly encephalized terrestrial species, like monkeys, are a notable exception to these patterns. In addition, species transitioning between the terrestrial and aquatic realms, such as seals and otters, show intermediate life history strategies. Furthermore, time since evolutionary divergence from land is a strong predictor of aquatic life history strategies. Our results provide compelling evidence linking environmental realms to the diversity of life history strategies among mammals, potentially informing assessments of species viability.\nNew resources for environmental genomics in the wild Mediterranean grass B. distachyon\nAuthors: Minadakis, N.; Williams, H.; Horvath, R.; Cakovic, D.; Stritt, C.; Thieme, M.; Bourgeois, Y. X. C.; Roulin, A.\nScore: 18.4, Published: 2023-06-13 DOI: 10.1101/2023.06.01.543285\nClosely related to economically important crops, the grass Brachypodium distachyon has been originally established as a pivotal species for grass genomics but more recently flourished as a model for developmental biology. Grasses encompass more than 10,000 species and cover more than 40% of the world land area from tropical to temperate regions. Unlocking the sources of phenotypic variation in B. distachyon is hence of prime interest in fundamental and applied research in agronomy, ecology and evolution. We present here the B. distachyon diversity panel, which encompasses 332 fully sequenced accessions covering the whole species distribution from Spain to Iraq. By combining population genetics, niche modeling and landscape genomics, we suggest that B. distachyon recolonized Europe and the Middle East following the last glacial maximum. Consequently, the species faced new environmental conditions which led to clear associations between bioclimatic variables and genetic factors as well as footprints of positive selection in the genome. Altogether, this genomic resource offers a powerful alternative to Arabidopsis thaliana to investigate the genetic bases of adaptation and phenotypic plasticity in plants and more specifically in monocots.\nPiroplasm parasites (Apicomplexa: Piroplasmida) in northeastern populations of the invasive Asian longhorned tick, Haemaphysalis longicornis Neumann (Ixodida: Ixodidae), in the United States\nAuthors: Herb, H. E.; Ferreira, F. C.; Gonzalez, J.; Fonseca, D. M.\nScore: 4.5, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546593\nPiroplasms, which include the agents of cattle fever and human and dog babesiosis, are a diverse group of blood parasites of significant veterinary and medical importance. The invasive Asian longhorned tick, Haemaphysalis longicornis, is a known vector of piroplasms in its native range in east Asia and invasive range in Australasia. In the US state of Virginia, H. longicornis has been associated with Theileria orientalis Ikeda outbreaks that caused cattle mortality. We examined 667 H. longicornis collected in 2021 from three sites in New Brunswick, New Jersey, the US state where established populations of this species were first detected in 2017. We used primers targeting the 18S small subunit rRNA and the cytochrome b oxidase loci and unveiled the presence of DNA from an unidentified Theileria species (in 1 nymph) and Theileria cervi type F (1 adult, 5 nymphs). In addition, we sequenced a 130 bp fragment of the cytochrome oxidase b locus from Odocoileus virginianus, the white-tailed deer, in a partially engorged questing H. longicornis, supporting the association of this tick species with deer. We also detected DNA from an undescribed Babesia sensu stricto ( true Babesia, 2 adults, 2 nymphs) as well as Babesia sp. Coco (1 adult, 1 nymph). Finally, we detected DNA from Babesia microti S837 (1 adult, 4 nymphs). Babesia microti S837 has been sequenced from striped skunks, Mephitis mephitis, and is closely related to the human pathogen B. microti US-type. The five parasites we are associating with H. longicornis represent a diverse assemblage spanning three clades in the piroplasm phylogeny, two undescribed, raising concerns of transmission amplification of veterinary pathogens as well as spillover of pathogens from wildlife to humans.\nWidespread analytical pitfalls in empirical coexistence studies and a checklist for improving their statistical robustness\nAuthors: Terry, C. D.; Armitage, D. W.\nScore: 3.2, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547661\n1. Modern Coexistence Theory (MCT) offers a conceptually straightforward approach for connecting empirical observations with an elegant theoretical framework, gaining popularity rapidly over the past decade. However, beneath this surface-level simplicity lie various assumptions and subjective choices made during data analysis. These can lead researchers to draw qualitatively different conclusions from the same set of experiments. As the predictions of MCT studies are often treated as outcomes, and many readers and reviewers may not be familiar with the framework's assumptions, there is a particular risk of 'researcher degrees of freedom' inflating the confidence in results, thereby affecting reproducibility and predictive power. 2. To tackle these concerns, we introduce a checklist consisting of statistical best-practices to promote more robust empirical applications of MCT. Our recommendations are organised into four categories: presentation and sharing of raw data, testing model assumptions and fits, managing uncertainty associated with model coefficients, and incorporating this uncertainty into coexistence predictions. 3. We surveyed empirical MCT studies published over the past 15 years and discovered a high degree of variation in the level of statistical rigour and adherence to best practices. We present case studies to illustrate the dependence of results on seemingly innocuous choices among competition model structure and error distributions, which in some cases reversed the predicted coexistence outcomes. These results demonstrate how different analytical approaches can profoundly alter the interpretation of experimental results, underscoring the importance of carefully considering and thoroughly justifying each step taken in the analysis pathway. 4. Our checklist serves as a resource for authors and reviewers alike, providing guidance to strengthen the empirical foundation of empirical coexistence analyses. As the field of empirical MCT shifts from a descriptive, trailblazing phase to a stage of consolidation, we emphasise the need for caution when building upon the findings of earlier studies. To ensure that progress made in the field of ecological coexistence is based on robust and reliable evidence, it is crucial to subject our predictions, conclusions, and generalizability to a more rigorous assessment than is currently the trend.\n",
  "wordCount" : "2718",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/ecology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      ecology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546704">
        <p class="paperTitle">GIFT an R package to access the Global Inventory of Floras and Traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Denelle, P.; Weigelt, P.; Kreft, H.</p>
        <p class="info">Score: 178.6, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546704' target='https://doi.org/10.1101/2023.06.27.546704'> 10.1101/2023.06.27.546704</a></p>
        <p class="abstract">O_LIAdvancing knowledge of biodiversity requires open-access global databases and workflows. This appears particularly crucial for plants, as ongoing changes pose major threats to this central group of organisms. Having large-scale information on plant distributions, functional traits and evolutionary history will enable the scientific community to improve its understanding of the patterns and drivers of plant diversity on a global scale.
C_LIO_LIThe Global Inventory of Floras and Traits (GIFT) is a global database of regional plant checklists that has proven successful in documenting biogeographical and geographical patterns of plants. Since the release of the first version of GIFT, the database kept on expanding. We introduce GIFT version 3.0, which contains 5,169 checklists referring to 3,400 regions. These checklists include a total of 371,148 land plant species, mostly vascular plants, of which 354,848 are accepted species names, and 109 functional traits. This new version uses new resources for taxonomic name standardization, is matched to a new plant phylogeny, comes with a new trait aggregation workflow, and includes additional environmental variables.
C_LIO_LIWe also present the GIFT R-package, which contains all necessary functions to retrieve distributional, functional, phylogenetic, and environmental data from the GIFT database. The package comes with a dedicated website, https://biogeomacro.github.io/GIFT/, which includes three rich vignettes to guide users in retrieving data from GIFT.
C_LIO_LIThe recent development of GIFT and its associated R-package provide ecologists with access to one of the largest plant databases. This will foster research into regional to global patterns of plant diversity and their underlying mechanisms. Proper versioning of the database and the ability to retrieve and cite data from any previous and current instance of the GIFT database will ensure the reproducibility of studies that utilize it.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546538">
        <p class="paperTitle">Rapid species discovery and identification with real-time barcoding facilitated by ONTbarcoder 2.0 and Oxford Nanopore R10.4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivathsan, A.; Feng, V.; Ramos, D.; Meier, R.</p>
        <p class="info">Score: 63.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546538' target='https://doi.org/10.1101/2023.06.26.546538'> 10.1101/2023.06.26.546538</a></p>
        <p class="abstract">Most animal species are undescribed and hidden in biodiversity samples consisting of thousands of specimens that are difficult to sort to species using morphological characters. Sorting with DNA barcodes is an attractive alternative, but needs cost-effective techniques that are suitable for use in many laboratories around the world so that barcoding can be made widely available. Oxford Nanopores portable and inexpensive MinION sequencers are a good option because they allow for reducing the sequencing and capital-cost of a fully functional barcoding laboratory. We here provide an update to the MinION barcoding software ONTbarcoder that is available for commonly used operating systems and includes a GUI. We show that the read quality obtained with ONTs latest basecallers and chemistry (R10.4 flowcells, Q20&#43; kits) allow for three key improvements. Firstly, ONTbarcoder can now deliver real-time barcoding to complement ONTs real-time sequencing. This means that the first barcodes are obtained within minutes of starting a sequencing run and a significant proportion of barcodes are available before sequencing completes. This real-time feature of ONTbarcoder can be configured to handle both Mk1B and Mk1C. The input is a demultiplexing sheet and base-called or raw data. Secondly, the improved read quality of ONTs latest flow cells (R10.4) allows for the use of primers with shorter indices than what was previously needed (9bp vs. 12-13bp). This decreases primer cost and improves PCR success rates. Thirdly, we demonstrate that the availability of R10.4 chemistry in the low-cost Flongle flowcell is an attractive option for users who require only 200-250 barcodes at a time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547261">
        <p class="paperTitle">Statistical prediction of microbial metabolic traits from genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Z.; Selim, A.; Kuehn, S.</p>
        <p class="info">Score: 27.9, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547261' target='https://doi.org/10.1101/2023.06.30.547261'> 10.1101/2023.06.30.547261</a></p>
        <p class="abstract">The metabolic activity of microbial communities is central to their role in biogeochemical cycles, human health, and biotechnology. Despite the abundance of sequencing data characterizing these consortia, it remains a serious challenge to predict microbial metabolic traits from sequencing data alone. Here we culture 96 bacterial isolates individually and assay their ability to grow on 10 distinct compounds as a sole carbon source. Using these data as well as two existing datasets, we show that statistical approaches can accurately predict bacterial carbon utilization traits from genomes. First, we show that classifiers trained on gene content can accurately predict bacterial carbon utilization phenotypes by encoding phylogenetic information. These models substantially outperform predictions made by constraint-based metabolic models automatically constructed from genomes. However, phylogeny-based predictions fail to predict traits for taxa that are phylogenetically distant from any strains in the training set. To overcome this we train improved models on gene presence/absence to predict carbon utilization traits from gene content. We show that models that predict carbon utilization traits from gene presence/absence can generalize to taxa that are phylogenetically distant from the training set either by exploiting biochemical information for feature selection or by having sufficiently large datasets. In the latter case, we provide evidence that a statistical approach can identify putatively mechanistic genes involved in metabolic traits. Our study demonstrates the potential power for predicting microbial phenotypes from genotypes using statistical approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547193">
        <p class="paperTitle">Absence of Coronavirus in terns in the Western Indian Ocean?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lebarbenchon, C.; Feare, C.; Larose, C.; Boucher, S.; Jaeger, A.; Le Corre, M.</p>
        <p class="info">Score: 12.4, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547193' target='https://doi.org/10.1101/2023.06.30.547193'> 10.1101/2023.06.30.547193</a></p>
        <p class="abstract">We investigated coronavirus circulation in three tern species, in four islands of the Western Indian Ocean (Bird, Reunion, Europa, Juan de Nova). None of the 1513 samples tested positive by RT-PCR. We discuss the implication in term of host species range, ecological drivers of virus transmission, and diagnostic tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546294">
        <p class="paperTitle">Microbial Associates of the Elm Leaf Beetle: Uncovering the Absence of Resident Bacteria and the Influence of Fungi on Insect Performance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, J.; Rakei, J.; Remus-Emsermann, M. N.; Johnston, P. R.; Mbedi, S.; Sparmann, S.; Hilker, M.; Paniagua Voirol, L. R.</p>
        <p class="info">Score: 6.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546294' target='https://doi.org/10.1101/2023.06.23.546294'> 10.1101/2023.06.23.546294</a></p>
        <p class="abstract">Microbial symbionts play crucial roles in the biology of many insects. While bacteria have been the primary focus of research on insect-microbe symbiosis, recent studies suggest that fungal symbionts may be just as important. The elm leaf beetle (ELB, Xanthogaleruca luteola) is a serious pest species of elm (Ulmus minor). Using culture-dependent and independent methods, we investigated the abundance and species richness of bacteria and fungi throughout various ELB life stages and generations, while concurrently analysing microbial communities on elm leaves. No persistent bacterial community was found to be associated with the ELB or elm leaves. By contrast, fungi were persistently present in the beetles feeding life stages and on elm leaves. Fungal community sequencing revealed a predominance of the genera Penicillium and Aspergillus in insects and on leaves. Culture-dependent surveys showed a high prevalence of two fungal colony morphotypes closely related to Penicillium lanosocoeruleum and Aspergillus flavus. Among these, the Penicillium morphotype was significantly more abundant on feeding-damaged compared to intact leaves, suggesting that the fungus thrives in the presence of the ELB. We assessed whether the detected prevalent fungal morphotypes influenced ELBs performance by rearing insects on i) surface-sterilised leaves, ii) leaves inoculated with Penicillium spores, and iii) leaves inoculated with Aspergillus spores. Insects feeding on Penicillium-inoculated leaves gained more biomass and tended to lay larger egg clutches than those consuming surface-sterilised leaves or Aspergillus-inoculated leaves. Our results demonstrate that the ELB does not harbour resident bacteria and that it might benefit from associating with Penicillium fungi.

ImportanceOur study provides insights into the still understudied role of microbial symbionts in the biology of the ELB, a major pest of elms. Contrary to expectations, we found no persistent bacterial symbionts associated with the ELB or elm leaves. Our research thus contributes to the growing body of knowledge that not all insects rely on bacterial symbionts. While no persistent bacterial symbionts were detectable in the ELB and elm leaf samples, our analyses revealed the persistent presence of fungi, particularly Penicillium and Aspergillus on both elm leaves and in the feeding ELB stages. Moreover, when ELB were fed with fungus-treated elm leaves, we detected a potentially beneficial effect of Penicillium on the ELBs development and fecundity. Our results highlight the significance of fungal symbionts in the biology of this insect.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547575" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547575">
        <p class="paperTitle">Competition for light can drive adverse species-composition shifts in the Amazon Forest under elevated CO2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, J.; Hofhansl, F.; Singh, S.; Stocker, B.; Brännström, A.; Franklin, O.; Blanco, C. C.; Aleixo, I. F.; Lapola, D. M.; Prentice, I. C.; Dieckmann, U.</p>
        <p class="info">Score: 6.5, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547575' target='https://doi.org/10.1101/2023.07.03.547575'> 10.1101/2023.07.03.547575</a></p>
        <p class="abstract">The resilience of biodiverse forests to climate change depends on an interplay of adaptive processes operating at multiple temporal and organizational scales. These include short-term acclimation of physiological processes like photosynthesis and respiration, mid-term changes in forest structure due to competition, and long-term changes in community composition arising from competitive exclusion and genetic trait evolution. To investigate the roles of diversity and adaptation for forest resilience, we present Plant-FATE, a parsimonious eco-evolutionary vegetation model. Tested with data from a hyperdiverse Amazonian terra-firme forest, our model accurately predicts multiple emergent ecosystem properties characterizing forest structure and function. Under elevated CO2 conditions, we predict an increase in productivity, leaf area, and aboveground biomass, with the magnitude of this increase declining in nutrient-deprived soils if trees allocate more carbon to the rhizosphere to overcome nutrient limitation. Furthermore, increased aboveground productivity leads to greater competition for light and drives a shift in community composition towards fast-growing but short-lived species characterized by lower wood densities. Such a transition reduces the carbon residence time of woody biomass, dampening carbon-sink strength and potentially rendering the Amazon Forest more vulnerable to future climatic extreme events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546851">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546851" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546851">
        <p class="paperTitle">Mammal life history strategies are strongly associated with environmental realms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546851" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546851" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beccari, E.; Capdevila Lanzaco, P.; Salguero-Gomez, R.; Perez Carmona, C.</p>
        <p class="info">Score: 5.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546851' target='https://doi.org/10.1101/2023.06.29.546851'> 10.1101/2023.06.29.546851</a></p>
        <p class="abstract">Mammalian investments in survival, development, and reproduction are constrained by physiological and evolutionary factors1, so their life history strategies can be characterized by a few axes of variation2-4. These axes conform a space in which species are positioned according to which life history strategies are favoured in the environment they exploit1,5,6. Yet, a global synthesis quantifying the realized mammalian life history diversity is lacking and exploring how the environment shapes mammalian life history strategies is missing. Here, we used six life history traits to build a global life history space and explored how major environmental realms (land, air, water) influence mammalian life history strategies. We demonstrate that realms are tightly linked to distinct life history strategies. Predominantly, aquatic and aerial species adhere to slower life history strategies, while terrestrial species tend to exhibit faster life histories. Highly encephalized terrestrial species, like monkeys, are a notable exception to these patterns. In addition, species transitioning between the terrestrial and aquatic realms, such as seals and otters, show intermediate life history strategies. Furthermore, time since evolutionary divergence from land is a strong predictor of aquatic life history strategies. Our results provide compelling evidence linking environmental realms to the diversity of life history strategies among mammals, potentially informing assessments of species viability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543285">
        <p class="paperTitle">New resources for environmental genomics in the wild Mediterranean grass B. distachyon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minadakis, N.; Williams, H.; Horvath, R.; Cakovic, D.; Stritt, C.; Thieme, M.; Bourgeois, Y. X. C.; Roulin, A.</p>
        <p class="info">Score: 18.4, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543285' target='https://doi.org/10.1101/2023.06.01.543285'> 10.1101/2023.06.01.543285</a></p>
        <p class="abstract">Closely related to economically important crops, the grass Brachypodium distachyon has been originally established as a pivotal species for grass genomics but more recently flourished as a model for developmental biology. Grasses encompass more than 10,000 species and cover more than 40% of the world land area from tropical to temperate regions. Unlocking the sources of phenotypic variation in B. distachyon is hence of prime interest in fundamental and applied research in agronomy, ecology and evolution. We present here the B. distachyon diversity panel, which encompasses 332 fully sequenced accessions covering the whole species distribution from Spain to Iraq. By combining population genetics, niche modeling and landscape genomics, we suggest that B. distachyon recolonized Europe and the Middle East following the last glacial maximum. Consequently, the species faced new environmental conditions which led to clear associations between bioclimatic variables and genetic factors as well as footprints of positive selection in the genome. Altogether, this genomic resource offers a powerful alternative to Arabidopsis thaliana to investigate the genetic bases of adaptation and phenotypic plasticity in plants and more specifically in monocots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546593">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546593" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546593">
        <p class="paperTitle">Piroplasm parasites (Apicomplexa: Piroplasmida) in northeastern populations of the invasive Asian longhorned tick, Haemaphysalis longicornis Neumann (Ixodida: Ixodidae), in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546593" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546593" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herb, H. E.; Ferreira, F. C.; Gonzalez, J.; Fonseca, D. M.</p>
        <p class="info">Score: 4.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546593' target='https://doi.org/10.1101/2023.06.26.546593'> 10.1101/2023.06.26.546593</a></p>
        <p class="abstract">Piroplasms, which include the agents of cattle fever and human and dog babesiosis, are a diverse group of blood parasites of significant veterinary and medical importance. The invasive Asian longhorned tick, Haemaphysalis longicornis, is a known vector of piroplasms in its native range in east Asia and invasive range in Australasia. In the US state of Virginia, H. longicornis has been associated with Theileria orientalis Ikeda outbreaks that caused cattle mortality. We examined 667 H. longicornis collected in 2021 from three sites in New Brunswick, New Jersey, the US state where established populations of this species were first detected in 2017. We used primers targeting the 18S small subunit rRNA and the cytochrome b oxidase loci and unveiled the presence of DNA from an unidentified Theileria species (in 1 nymph) and Theileria cervi type F (1 adult, 5 nymphs). In addition, we sequenced a 130 bp fragment of the cytochrome oxidase b locus from Odocoileus virginianus, the white-tailed deer, in a partially engorged questing H. longicornis, supporting the association of this tick species with deer. We also detected DNA from an undescribed Babesia sensu stricto ( true Babesia, 2 adults, 2 nymphs) as well as Babesia sp. Coco (1 adult, 1 nymph). Finally, we detected DNA from Babesia microti S837 (1 adult, 4 nymphs). Babesia microti S837 has been sequenced from striped skunks, Mephitis mephitis, and is closely related to the human pathogen B. microti US-type. The five parasites we are associating with H. longicornis represent a diverse assemblage spanning three clades in the piroplasm phylogeny, two undescribed, raising concerns of transmission amplification of veterinary pathogens as well as spillover of pathogens from wildlife to humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547661">
        <p class="paperTitle">Widespread analytical pitfalls in empirical coexistence studies and a checklist for improving their statistical robustness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terry, C. D.; Armitage, D. W.</p>
        <p class="info">Score: 3.2, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547661' target='https://doi.org/10.1101/2023.07.04.547661'> 10.1101/2023.07.04.547661</a></p>
        <p class="abstract">1. Modern Coexistence Theory (MCT) offers a conceptually straightforward approach for connecting empirical observations with an elegant theoretical framework, gaining popularity rapidly over the past decade. However, beneath this surface-level simplicity lie various assumptions and subjective choices made during data analysis. These can lead researchers to draw qualitatively different conclusions from the same set of experiments. As the predictions of MCT studies are often treated as outcomes, and many readers and reviewers may not be familiar with the framework&#39;s assumptions, there is a particular risk of &#39;researcher degrees of freedom&#39; inflating the confidence in results, thereby affecting reproducibility and predictive power. 2. To tackle these concerns, we introduce a checklist consisting of statistical best-practices to promote more robust empirical applications of MCT. Our recommendations are organised into four categories: presentation and sharing of raw data, testing model assumptions and fits, managing uncertainty associated with model coefficients, and incorporating this uncertainty into coexistence predictions. 3. We surveyed empirical MCT studies published over the past 15 years and discovered a high degree of variation in the level of statistical rigour and adherence to best practices. We present case studies to illustrate the dependence of results on seemingly innocuous choices among competition model structure and error distributions, which in some cases reversed the predicted coexistence outcomes. These results demonstrate how different analytical approaches can profoundly alter the interpretation of experimental results, underscoring the importance of carefully considering and thoroughly justifying each step taken in the analysis pathway. 4. Our checklist serves as a resource for authors and reviewers alike, providing guidance to strengthen the empirical foundation of empirical coexistence analyses. As the field of empirical MCT shifts from a descriptive, trailblazing phase to a stage of consolidation, we emphasise the need for caution when building upon the findings of earlier studies. To ensure that progress made in the field of ecological coexistence is based on robust and reliable evidence, it is crucial to subject our predictions, conclusions, and generalizability to a more rigorous assessment than is currently the trend.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
