<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 57.5, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 57.5, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 57.5, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 57.5, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations.",
  "keywords": [
    
  ],
  "articleBody": " Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 57.5, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at pop.evemodel.org.\nGenetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes\nAuthors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.\nScore: 28.5, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23299014\nThe biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).\nGenetic analysis of 104 pregnancy phenotypes in 39,194 Chinese women\nAuthors: Xiao, H.; Li, L.; Yang, M.; Zeng, J.; Zhou, J.; Tao, Y.; Zhou, Y.; Cai, M.; Liu, J.; Huang, Y.; Zhong, Y.; Liu, P.; Cao, Z.; Mei, H.; Cai, X.; Hu, L.; Zhou, R.; Xu, X.; Yang, H.; Wang, J.; Zhu, H.; Zhou, A.; Jin, X.\nScore: 10.1, Published: 2023-11-24 DOI: 10.1101/2023.11.23.23298979\nMaternity is a special period in a womans life that involves substantial physiological, psychological, and hormonal changes. These changes may cause alterations in many clinical measurements during pregnancy, which can be used to monitor and diagnose maternal disorders and adverse postnatal outcomes. Exploring the genetic background of these phenotypes is key to elucidating the pathogenesis of pregnancy disorders. In this study, we conducted a large-scale molecular biology analysis of 104 pregnancy phenotypes based on genotype data from 39,194 Chinses women. Genome- wide association analysis identified a total of 407 trait-locus associations, of which 75.18% were previously reported. Among the 101 novel associations for 37 phenotypes, some were potentially pregnancy-specific and worth further experimental investigation. For example, ESR1 with fasting glucose, hemoglobin, hematocrit, and several leukocytoses; ZSCAN31 with blood urea nitrogen. We further performed pathway- based analysis and uncovered at least one significant pathway for 24 traits, in addition to previously known functional pathways, novel findings included birthweight with \"Reactome signaling by NODAL\", twin pregnancy with \"Reactome mitotic G1-G1/S phases\". The partitioning heritability analysis recapitulated known trait-relevant tissue/cell types, and also discovered interesting results including twin pregnancy with \"embryoid bodies\" cell-type enrichment, the delivery type cesarean section with \"fallopian tube\", and birth weight with \"ovary and embryonic stem cells\". In terms of both sample size and the variety of phenotypes, our work is one of the largest genetic studies of pregnancy phenotypes across all populations. We believe that this study will provide a valuable resource for exploring the genetic background of pregnancy phenotypes and also for further research on pregnancy-related diseases and adverse neonatal outcomes.\nA validated heart-specific model for splice-disrupting variants in childhood heart disease\nAuthors: Lesurf, R.; Breckpot, J.; Bouwmeester, J.; Hanafi, N.; Jain, A.; Liang, Y.; Papaz, T.; Lougheed, J.; Mondal, T.; Alsalehi, M.; Altamirano-Diaz, L.; Oechslin, E.; Audain, E.; Dombrowsky, G.; Postma, A. V.; Woudstra, O. I.; Bouma, B. J.; Hitz, M.-P.; Bezzina, C. R.; Blue, G.; Winlaw, D. S.; Mital, S.\nScore: 3.5, Published: 2023-11-27 DOI: 10.1101/2023.11.23.23298903\nCongenital heart disease (CHD) is the most common congenital anomaly. Non-canonical splice-disrupting variants are not routinely evaluated by clinical tests. Algorithms including SpliceAI predict such variants, but are not specific to cardiac-expressed genes. Whole genome (WGS) (n=1083) and myocardial RNA-Sequencing (RNA-Seq) (n=114) of CHD cases was used to identify splice-disrupting variants. Using features of variants confirmed to affect splicing in myocardial RNA, we trained a machine learning model that outperformed SpliceAI for predicting cardiac-specific splice-disrupting variants (AUC 0.92 vs 0.66), and was independently validated in 43 cardiomyopathy probands (AUC 0.88 vs 0.64). Application of this model to 971 CHD WGS samples identified 9% patients with splice-disrupting variants in CHD genes. Forty-one% of predicted splice-disrupting variants were deeply intronic. The burden of variants in CHD genes was higher in cases compared with 2,570 controls. Our model improved genetic yield by identifying splice-disrupting variants that are not evaluated by routine tests.\nGenomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in AR-altered lethal prostate cancer\nAuthors: Chauhan, P. S.; Alahi, I.; Sinha, S.; Shiang, A. L.; Mueller, R.; Webster, J.; Dang, H. X.; Saha, D.; Greiner, L.; Yang, B.; Ni, G.; Ledet, E. M.; Babbra, R. K.; Feng, W.; Harris, P. K.; Qaium, F.; Jaeger, E. B.; Miller, P. J.; Caputo, S. A.; Sartor, O.; Pachynski, R. K.; Maher, C. A.; Chaudhuri, A. A.\nScore: 2.9, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23299215\nMetastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor (AR)-targeted agents is often lethal. Unfortunately, biomarkers for this deadly disease remain under investigation, and underpinning mechanisms are ill-understood. Here, we applied deep sequencing to ~100 mCRPC patients prior to the initiation of first-line AR-targeted therapy, which detected AR/enhancer alterations in over a third of patients, which correlated with lethality. To delve into the mechanism underlying why these patients with cell-free AR/enhancer alterations developed more lethal prostate cancer, we next performed genome-wide cell-free DNA epigenomics. Strikingly, we found that binding sites for transcription factors associated with developmental stemness were nucleosomally more accessible. These results were corroborated using cell-free DNA methylation data, as well as tumor RNA sequencing from a held-out cohort of mCRPC patients. Thus, we validated the importance of AR/enhancer alterations as a prognostic biomarker in lethal mCRPC, and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.\nCopy-number variants and polygenic risk for intelligence confer risk for autism spectrum disorder irrespective of their effects on cognitive ability\nAuthors: Schmilovich, Z.; Bourque, V.-R.; Douard, E.; Huguet, G.; Poulain, C.; Ross, J. P.; Alipour, P.; Castonguay, C.-E.; Younis, N.; Jean-Louis, M.; Saci, Z.; Pausova, Z.; Paus, T.; Schuman, G.; Porteous, D.; Davies, G.; Redmond, P.; Harris, S. E.; Deary, I. J.; Whalley, H.; Hayward, C.; Dion, P. A.; Jacquemont, S.; Rouleau, G. A.\nScore: 2.6, Published: 2023-11-29 DOI: 10.1101/2023.11.29.23299190\nRare copy number variants (CNVs) and polygenic risk for intelligence (PRS-IQ) both confer risk for autism spectrum disorder (ASD) but have opposing effects on cognitive ability. The field has struggled to disentangle the effects of these two classes of genomic variants on cognitive ability from their effects on ASD risk, in part because previous studies did not include controls with cognitive measures. We aim to investigate the impact of these genomic variants on ASD risk while adjusting for their known effects on cognitive ability. In a cohort of 8,426 subjects with ASD and 169,804 controls with cognitive assessments, we found that rare coding CNVs and PRS-IQ increased ASD risk, even after adjusting for their effects on cognitive ability. Bottom decile PRS-IQ and CNVs both decreased cognitive ability but had opposing effects on ASD risk. Models combining both classes of variants showed that the effects of rare CNVs and PRS-IQ on ASD risk and cognitive ability were largely additive, further suggesting that risk for ASD is conferred independently from its effects on cognitive ability. Despite imparting mostly additive effects on ASD risk, rare CNVs and PRS-IQ showed opposing effects on core and associated features and developmental history among subjects with ASD. Our findings suggest that cognitive ability itself may not be the factor driving the underlying risk for ASD conferred by these two classes of genomic variants. In other words, ASD risk and cognitive ability may be two distinct manifestations of CNVs and PRS-IQ. This study also highlights the challenge of understanding how genetic risk for ASD maps onto its dimensional traits.\nMulti-trait genome-wide association study in 34,394 Chinese women reveals the genetic architecture of plasma metabolites during pregnancy\nAuthors: Liu, S.; Yao, J.; Lin, L.; Lan, X.; Wu, L.; Kong, N.; Deng, Y.; Li, Y.; Xie, J.; Zhu, H.; Wu, X.; Li, Z.; Xiong, L.; Wang, Y.; Ren, J.; Qiu, X.; Zhao, W.; Gao, Y.; Chen, Y.; Su, F.; Zhou, Y.; Rao, W.; Zhang, J.; Hou, G.; Huang, L.; Li, L.; Liu, X.; Nie, C.; Luo, L.; Liu, Z.; Chen, F.; Lin, S.; Zhao, L.; Fu, Q.; Jiang, D.; Yin, Y.; Xu, X.; Wang, J.; Yang, H.; Wang, R.; Niu, J.; Wei, F.; Jin, X.; Liu, S.\nScore: 1.8, Published: 2023-11-27 DOI: 10.1101/2023.11.26.23299022\nMetabolites are important indicators of individual health and can serve as crucial targets for therapy. However, the genetic basis of many metabolites remains largely unexplored, especially among underrepresented East Asians and during critical periods such as pregnancy. In this study, we utilized genetic information obtained from non-invasive prenatal testing to conduct a genome-wide association analysis of 84 metabolites, including 37 amino acids, 10 vitamins, 24 metal elements, and 13 hormones, among 34,394 Chinese pregnant women. Of these metabolites, 52 and 11 had not previously been studied in East Asians or globally. We identified 30 novel metabolite-gene associations. We also observed substantial differences in the genetic effects on hormones between pregnancy and non-pregnancy periods, suggesting effect modifications in response to physiological changes. Furthermore, we uncovered pervasive pleiotropic effects for 50.94% of the genetic associations among metabolites, as well as between six metabolites and eight pregnancy biomarkers. Using mendelian randomization, we identified potential causal relationships between plasma folate and ischemic stroke, vitamin D3 and Graves disease, copper and open-angle glaucoma, and androstenedione and rheumatoid arthritis. These discoveries provide invaluable genetic insights into human metabolism, laying the foundation for future mechanistic studies and the development of new therapeutic targets, particularly for underrepresented East Asians.\nGenetic association analysis of age of ischemic stroke onset: a caveat\nAuthors: von Berg, J.; McArdle, P.; Happola, P. J.; Haessler, J.; Kooperberg, C.; Lemmens, R.; Pezzini, A.; Thijs, V.; Pulit, S. L.; Kittner, S.; Mitchell, B. D.; de Ridder, J.; van der Laan, S. W.\nScore: 1.2, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23294385\nLarge genome-wide association studies (GWAS) employing case-control study designs have now identified tens of loci associated with ischemic stroke (IS). As a complement to these studies, we performed GWAS in a case-only design to identify loci influencing age at onset (AAO) of ischemic stroke. Analyses were conducted in a Discovery cohort of 10,857 ischemic stroke cases using a linear regression framework. We meta-analyzed all SNPs with p-value \u003c 1x10-5 in a sex-combined or sex-stratified analysis using summary data from two additional replication cohorts. In the women-only meta-analysis, we detected significant evidence for association of AAO with rs429358, an exonic variant in APOE that encodes for the APOE-{Jukcy}4 allele. Each copy of the rs429358:T\u003eC allele was associated with a 1.29 years earlier stroke AOO (meta p-value = 2.48x10-11). This APOE variant has previously been associated with increased mortality and ischemic stroke AAO. We hypothesized that the association with AAO may reflect a survival bias attributable to an age-related decline in mortality among APOE-{Jukcy}4 carriers and have no association to stroke AAO per se. Using a simulation study, we found that a variant associated with overall mortality might indeed be detected with an AAO analysis. A variant with a two-fold increase on mortality risk would lead to an observed effect of AAO that is comparable to what we found. In conclusion, we detected a robust association of the APOE locus with stroke AAO and provided simulations to suggest that this association may be unrelated to ischemic stroke per se but related to a general survival bias.\nEvaluating the performance of polygenic risk profiling across diverse ancestry populations in Parkinson's disease\nAuthors: Saffie Awad, P.; Elsayed, I.; Sanyaolu, A.; Wild Crea, P.; Schumacher Schuh, A. F.; Levine, K. S.; Vitale, D.; Korestky, M. J.; Kim, J.; Peixoto Leal, T.; Perinan, M. T.; Dey, S.; Noyce, A. J.; Reyes-Palomares, A.; Rodriguez-Losada, N.; Foo, J. N.; Mohamed, W.; Heilbron, K.; Norcliffe-Kaufmann, L.; the 23andMe Research Team, ; Rizig, M.; Okubadejo, N.; Nalls, M.; Blauwendraat, C.; Singleton, A.; Leonard, H.; Makarious, M. B.; Mata, I. F.; Bandres Ciga, S.; Global Parkinsons Genetics Program (GP2), Score: 1.2, Published: 2023-11-29 DOI: 10.1101/2023.11.28.23299090\nObjectiveThis study aims to address disparities in risk prediction by evaluating the performance of polygenic risk score (PRS) models using the 90 risk variants across 78 independent loci previously linked to Parkinsons disease (PD) risk across seven diverse ancestry populations. MethodsWe conducted a multi-stage study, testing PRS models in predicting PD status across seven different ancestries applying three approaches: 1) PRS adjusted by gender and age; 2) PRS adjusted by gender, age and principal components (PCs); and 3) PRS adjusted by gender, age and percentage of population admixture. These models were built using the largest four population-specific summary statistics of PD risk to date (base data) and individual level data obtained from the Global Parkinsons Genetics Program (target data). We performed power calculations to estimate the minimum sample size required to conduct these analyses. A total of 91 PRS models were developed to investigate cumulative known genetic variation associated with PD risk and age of onset in a global context. ResultsWe observed marked heterogeneity in risk estimates across non-European ancestries, including East Asians, Central Asians, Latino/Admixed Americans, Africans, African admixed, and Ashkenazi Jewish populations. Risk allele patterns for the 90 risk variants yielded significant differences in directionality, frequency, and magnitude of effect. PRS did not improve in performance when predicting disease status using similar base and target data across multiple ancestries, demonstrating that cumulative PRS models based on current known risk are inherently biased towards European populations. We found that PRS models adjusted by percentage of admixture outperformed models that adjusted for conventional PCs in highly admixed populations. Overall, the clinical utility of our models in individually predicting PD status is limited in concordance with the estimates observed in European populations. InterpretationThis study represents the first comprehensive assessment of how PRS models predict PD risk and age at onset in a multi-ancestry fashion. Given the heterogeneity and distinct genetic architecture of PD across different populations, our assessment emphasizes the need for larger and diverse study cohorts of individual-level target data and well-powered ancestry-specific summary statistics. Our current understanding of PD status unraveled through GWAS in European populations is not generally applicable to other ancestries. Future studies should integrate clinical and *omics level data to enhance the accuracy and predictive power of PRS across diverse populations.\nGenomic epidemiology of SARS-CoV-2 in Sudan: A retrospective analysis 2020-2022\nAuthors: Mburu, M. W.; Lambisia, A. W.; Elnegoumi, M.; Nour, E. O. M.; Khair, O. M. M.; Abualas, E.; Elhassan, R. M.; Adam, M. A. M.; Mohammed, K. S.; Morobe, J. M.; Ndwiga, L.; Moraa, E. O.; Makori, T. O.; Murunga, N.; Katama, E. N.; Waruguru, D. W.; Githinji, G.; Mwangi, S.; Tessema, S. K.; Adam, M. H. E.; Mahmoud, I.; Mohamed, H. S.; Mustafa, Y. A. A.; Mohamed, M. M. A.; Ahmed, R. H. A.; Mohamed, N. Y. I.; Ali, R. H.; Ebraheem, R. S. M.; Ahmed, H. H. H.; Abdalla, H. E.; Mukhtar, I.; Alla, A. E. M. W.; Elsheikh, O. H. A.; Musa, A. O.; Faki, M. T.; Ahmed, G. M.; Ahmed, T. E. R.; Abdalla, M. A. O.; Moh\nScore: 1.2, Published: 2023-11-27 DOI: 10.1101/2023.11.27.23299044\nSARS-CoV-2 was first detected in Sudan on 13th March 2020. Here, we describe the genomic epidemiology of SARS-CoV-2 in Sudan between May 2020 and April 2022 to understand the introduction and transmission of SARS-CoV-2 variants in the country. A total of 667 SARS-CoV-2 positive samples were successfully sequenced using the nCoV-19 Artic protocol on the Oxford Nanopore Technology ([\u0026ge;]70% genome completeness). The genomes were compared with a select contemporaneous global dataset to determine genetic relatedness and estimate import/export events. The genomes were classified into 37 Pango lineages within the ancestral strain (107 isolates across 13 Pango lineages), Eta variant of interest (VOI) (78 isolates in 1 lineage), Alpha variant of concern (VOC) (10 isolates in 2 lineages), Beta VOC (26 isolates in 1 lineage), Delta VOC (171 isolates across 8 lineages) and Omicron VOC (242 isolates across 12 lineages). We estimated a total of 144 introductions of the observed variants from different countries across the globe. Multiple introductions of the Eta VOI, Beta VOC and Omicron VOC were observed in Sudan mainly from Europe and Africa. These findings suggest a need for continuous genomic surveillance of SARS-CoV-2 to monitor their introduction and spread consequently inform public health measures to combat SARS-CoV-2 transmission.\n",
  "wordCount" : "3070",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23299062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23299062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23299062">
        <p class="paperTitle">Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23299062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23299062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.</p>
        <p class="info">Score: 57.5, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23299062' target='https://doi.org/10.1101/2023.11.27.23299062'> 10.1101/2023.11.27.23299062</a></p>
        <p class="abstract">Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at pop.evemodel.org.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.23299014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.23299014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.23299014">
        <p class="paperTitle">Genetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.23299014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.23299014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.</p>
        <p class="info">Score: 28.5, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.23299014' target='https://doi.org/10.1101/2023.11.25.23299014'> 10.1101/2023.11.25.23299014</a></p>
        <p class="abstract">The biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.23298979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.23298979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.23298979">
        <p class="paperTitle">Genetic analysis of 104 pregnancy phenotypes in 39,194 Chinese women</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.23298979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.23298979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xiao, H.; Li, L.; Yang, M.; Zeng, J.; Zhou, J.; Tao, Y.; Zhou, Y.; Cai, M.; Liu, J.; Huang, Y.; Zhong, Y.; Liu, P.; Cao, Z.; Mei, H.; Cai, X.; Hu, L.; Zhou, R.; Xu, X.; Yang, H.; Wang, J.; Zhu, H.; Zhou, A.; Jin, X.</p>
        <p class="info">Score: 10.1, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.23298979' target='https://doi.org/10.1101/2023.11.23.23298979'> 10.1101/2023.11.23.23298979</a></p>
        <p class="abstract">Maternity is a special period in a womans life that involves substantial physiological, psychological, and hormonal changes. These changes may cause alterations in many clinical measurements during pregnancy, which can be used to monitor and diagnose maternal disorders and adverse postnatal outcomes. Exploring the genetic background of these phenotypes is key to elucidating the pathogenesis of pregnancy disorders. In this study, we conducted a large-scale molecular biology analysis of 104 pregnancy phenotypes based on genotype data from 39,194 Chinses women. Genome- wide association analysis identified a total of 407 trait-locus associations, of which 75.18% were previously reported. Among the 101 novel associations for 37 phenotypes, some were potentially pregnancy-specific and worth further experimental investigation. For example, ESR1 with fasting glucose, hemoglobin, hematocrit, and several leukocytoses; ZSCAN31 with blood urea nitrogen. We further performed pathway- based analysis and uncovered at least one significant pathway for 24 traits, in addition to previously known functional pathways, novel findings included birthweight with &#34;Reactome signaling by NODAL&#34;, twin pregnancy with &#34;Reactome mitotic G1-G1/S phases&#34;. The partitioning heritability analysis recapitulated known trait-relevant tissue/cell types, and also discovered interesting results including twin pregnancy with &#34;embryoid bodies&#34; cell-type enrichment, the delivery type cesarean section with &#34;fallopian tube&#34;, and birth weight with &#34;ovary and embryonic stem cells&#34;. In terms of both sample size and the variety of phenotypes, our work is one of the largest genetic studies of pregnancy phenotypes across all populations. We believe that this study will provide a valuable resource for exploring the genetic background of pregnancy phenotypes and also for further research on pregnancy-related diseases and adverse neonatal outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.23298903">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.23298903" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.23298903">
        <p class="paperTitle">A validated heart-specific model for splice-disrupting variants in childhood heart disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.23298903" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.23298903" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lesurf, R.; Breckpot, J.; Bouwmeester, J.; Hanafi, N.; Jain, A.; Liang, Y.; Papaz, T.; Lougheed, J.; Mondal, T.; Alsalehi, M.; Altamirano-Diaz, L.; Oechslin, E.; Audain, E.; Dombrowsky, G.; Postma, A. V.; Woudstra, O. I.; Bouma, B. J.; Hitz, M.-P.; Bezzina, C. R.; Blue, G.; Winlaw, D. S.; Mital, S.</p>
        <p class="info">Score: 3.5, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.23298903' target='https://doi.org/10.1101/2023.11.23.23298903'> 10.1101/2023.11.23.23298903</a></p>
        <p class="abstract">Congenital heart disease (CHD) is the most common congenital anomaly. Non-canonical splice-disrupting variants are not routinely evaluated by clinical tests. Algorithms including SpliceAI predict such variants, but are not specific to cardiac-expressed genes. Whole genome (WGS) (n=1083) and myocardial RNA-Sequencing (RNA-Seq) (n=114) of CHD cases was used to identify splice-disrupting variants. Using features of variants confirmed to affect splicing in myocardial RNA, we trained a machine learning model that outperformed SpliceAI for predicting cardiac-specific splice-disrupting variants (AUC 0.92 vs 0.66), and was independently validated in 43 cardiomyopathy probands (AUC 0.88 vs 0.64). Application of this model to 971 CHD WGS samples identified 9% patients with splice-disrupting variants in CHD genes. Forty-one% of predicted splice-disrupting variants were deeply intronic. The burden of variants in CHD genes was higher in cases compared with 2,570 controls. Our model improved genetic yield by identifying splice-disrupting variants that are not evaluated by routine tests.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23299215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23299215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23299215">
        <p class="paperTitle">Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in AR-altered lethal prostate cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23299215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23299215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chauhan, P. S.; Alahi, I.; Sinha, S.; Shiang, A. L.; Mueller, R.; Webster, J.; Dang, H. X.; Saha, D.; Greiner, L.; Yang, B.; Ni, G.; Ledet, E. M.; Babbra, R. K.; Feng, W.; Harris, P. K.; Qaium, F.; Jaeger, E. B.; Miller, P. J.; Caputo, S. A.; Sartor, O.; Pachynski, R. K.; Maher, C. A.; Chaudhuri, A. A.</p>
        <p class="info">Score: 2.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23299215' target='https://doi.org/10.1101/2023.12.01.23299215'> 10.1101/2023.12.01.23299215</a></p>
        <p class="abstract">Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor (AR)-targeted agents is often lethal. Unfortunately, biomarkers for this deadly disease remain under investigation, and underpinning mechanisms are ill-understood. Here, we applied deep sequencing to ~100 mCRPC patients prior to the initiation of first-line AR-targeted therapy, which detected AR/enhancer alterations in over a third of patients, which correlated with lethality. To delve into the mechanism underlying why these patients with cell-free AR/enhancer alterations developed more lethal prostate cancer, we next performed genome-wide cell-free DNA epigenomics. Strikingly, we found that binding sites for transcription factors associated with developmental stemness were nucleosomally more accessible. These results were corroborated using cell-free DNA methylation data, as well as tumor RNA sequencing from a held-out cohort of mCRPC patients. Thus, we validated the importance of AR/enhancer alterations as a prognostic biomarker in lethal mCRPC, and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.23299190">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.23299190" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.23299190">
        <p class="paperTitle">Copy-number variants and polygenic risk for intelligence confer risk for autism spectrum disorder irrespective of their effects on cognitive ability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.23299190" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.23299190" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmilovich, Z.; Bourque, V.-R.; Douard, E.; Huguet, G.; Poulain, C.; Ross, J. P.; Alipour, P.; Castonguay, C.-E.; Younis, N.; Jean-Louis, M.; Saci, Z.; Pausova, Z.; Paus, T.; Schuman, G.; Porteous, D.; Davies, G.; Redmond, P.; Harris, S. E.; Deary, I. J.; Whalley, H.; Hayward, C.; Dion, P. A.; Jacquemont, S.; Rouleau, G. A.</p>
        <p class="info">Score: 2.6, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.23299190' target='https://doi.org/10.1101/2023.11.29.23299190'> 10.1101/2023.11.29.23299190</a></p>
        <p class="abstract">Rare copy number variants (CNVs) and polygenic risk for intelligence (PRS-IQ) both confer risk for autism spectrum disorder (ASD) but have opposing effects on cognitive ability. The field has struggled to disentangle the effects of these two classes of genomic variants on cognitive ability from their effects on ASD risk, in part because previous studies did not include controls with cognitive measures. We aim to investigate the impact of these genomic variants on ASD risk while adjusting for their known effects on cognitive ability. In a cohort of 8,426 subjects with ASD and 169,804 controls with cognitive assessments, we found that rare coding CNVs and PRS-IQ increased ASD risk, even after adjusting for their effects on cognitive ability. Bottom decile PRS-IQ and CNVs both decreased cognitive ability but had opposing effects on ASD risk. Models combining both classes of variants showed that the effects of rare CNVs and PRS-IQ on ASD risk and cognitive ability were largely additive, further suggesting that risk for ASD is conferred independently from its effects on cognitive ability. Despite imparting mostly additive effects on ASD risk, rare CNVs and PRS-IQ showed opposing effects on core and associated features and developmental history among subjects with ASD. Our findings suggest that cognitive ability itself may not be the factor driving the underlying risk for ASD conferred by these two classes of genomic variants. In other words, ASD risk and cognitive ability may be two distinct manifestations of CNVs and PRS-IQ. This study also highlights the challenge of understanding how genetic risk for ASD maps onto its dimensional traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.23299022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.23299022" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.23299022">
        <p class="paperTitle">Multi-trait genome-wide association study in 34,394 Chinese women reveals the genetic architecture of plasma metabolites during pregnancy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.23299022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.23299022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S.; Yao, J.; Lin, L.; Lan, X.; Wu, L.; Kong, N.; Deng, Y.; Li, Y.; Xie, J.; Zhu, H.; Wu, X.; Li, Z.; Xiong, L.; Wang, Y.; Ren, J.; Qiu, X.; Zhao, W.; Gao, Y.; Chen, Y.; Su, F.; Zhou, Y.; Rao, W.; Zhang, J.; Hou, G.; Huang, L.; Li, L.; Liu, X.; Nie, C.; Luo, L.; Liu, Z.; Chen, F.; Lin, S.; Zhao, L.; Fu, Q.; Jiang, D.; Yin, Y.; Xu, X.; Wang, J.; Yang, H.; Wang, R.; Niu, J.; Wei, F.; Jin, X.; Liu, S.</p>
        <p class="info">Score: 1.8, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.23299022' target='https://doi.org/10.1101/2023.11.26.23299022'> 10.1101/2023.11.26.23299022</a></p>
        <p class="abstract">Metabolites are important indicators of individual health and can serve as crucial targets for therapy. However, the genetic basis of many metabolites remains largely unexplored, especially among underrepresented East Asians and during critical periods such as pregnancy. In this study, we utilized genetic information obtained from non-invasive prenatal testing to conduct a genome-wide association analysis of 84 metabolites, including 37 amino acids, 10 vitamins, 24 metal elements, and 13 hormones, among 34,394 Chinese pregnant women. Of these metabolites, 52 and 11 had not previously been studied in East Asians or globally. We identified 30 novel metabolite-gene associations. We also observed substantial differences in the genetic effects on hormones between pregnancy and non-pregnancy periods, suggesting effect modifications in response to physiological changes. Furthermore, we uncovered pervasive pleiotropic effects for 50.94% of the genetic associations among metabolites, as well as between six metabolites and eight pregnancy biomarkers. Using mendelian randomization, we identified potential causal relationships between plasma folate and ischemic stroke, vitamin D3 and Graves disease, copper and open-angle glaucoma, and androstenedione and rheumatoid arthritis. These discoveries provide invaluable genetic insights into human metabolism, laying the foundation for future mechanistic studies and the development of new therapeutic targets, particularly for underrepresented East Asians.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23294385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23294385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23294385">
        <p class="paperTitle">Genetic association analysis of age of ischemic stroke onset: a caveat</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23294385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23294385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: von Berg, J.; McArdle, P.; Happola, P. J.; Haessler, J.; Kooperberg, C.; Lemmens, R.; Pezzini, A.; Thijs, V.; Pulit, S. L.; Kittner, S.; Mitchell, B. D.; de Ridder, J.; van der Laan, S. W.</p>
        <p class="info">Score: 1.2, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23294385' target='https://doi.org/10.1101/2023.12.01.23294385'> 10.1101/2023.12.01.23294385</a></p>
        <p class="abstract">Large genome-wide association studies (GWAS) employing case-control study designs have now identified tens of loci associated with ischemic stroke (IS). As a complement to these studies, we performed GWAS in a case-only design to identify loci influencing age at onset (AAO) of ischemic stroke. Analyses were conducted in a Discovery cohort of 10,857 ischemic stroke cases using a linear regression framework. We meta-analyzed all SNPs with p-value &lt; 1x10-5 in a sex-combined or sex-stratified analysis using summary data from two additional replication cohorts. In the women-only meta-analysis, we detected significant evidence for association of AAO with rs429358, an exonic variant in APOE that encodes for the APOE-{Jukcy}4 allele. Each copy of the rs429358:T&gt;C allele was associated with a 1.29 years earlier stroke AOO (meta p-value = 2.48x10-11). This APOE variant has previously been associated with increased mortality and ischemic stroke AAO. We hypothesized that the association with AAO may reflect a survival bias attributable to an age-related decline in mortality among APOE-{Jukcy}4 carriers and have no association to stroke AAO per se. Using a simulation study, we found that a variant associated with overall mortality might indeed be detected with an AAO analysis. A variant with a two-fold increase on mortality risk would lead to an observed effect of AAO that is comparable to what we found. In conclusion, we detected a robust association of the APOE locus with stroke AAO and provided simulations to suggest that this association may be unrelated to ischemic stroke per se but related to a general survival bias.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.23299090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.23299090" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.23299090">
        <p class="paperTitle">Evaluating the performance of polygenic risk profiling across diverse ancestry populations in Parkinson&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.23299090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.23299090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saffie Awad, P.; Elsayed, I.; Sanyaolu, A.; Wild Crea, P.; Schumacher Schuh, A. F.; Levine, K. S.; Vitale, D.; Korestky, M. J.; Kim, J.; Peixoto Leal, T.; Perinan, M. T.; Dey, S.; Noyce, A. J.; Reyes-Palomares, A.; Rodriguez-Losada, N.; Foo, J. N.; Mohamed, W.; Heilbron, K.; Norcliffe-Kaufmann, L.; the 23andMe Research Team,  ; Rizig, M.; Okubadejo, N.; Nalls, M.; Blauwendraat, C.; Singleton, A.; Leonard, H.; Makarious, M. B.; Mata, I. F.; Bandres Ciga, S.; Global Parkinsons Genetics Program (GP2),  </p>
        <p class="info">Score: 1.2, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.23299090' target='https://doi.org/10.1101/2023.11.28.23299090'> 10.1101/2023.11.28.23299090</a></p>
        <p class="abstract">ObjectiveThis study aims to address disparities in risk prediction by evaluating the performance of polygenic risk score (PRS) models using the 90 risk variants across 78 independent loci previously linked to Parkinsons disease (PD) risk across seven diverse ancestry populations.

MethodsWe conducted a multi-stage study, testing PRS models in predicting PD status across seven different ancestries applying three approaches: 1) PRS adjusted by gender and age; 2) PRS adjusted by gender, age and principal components (PCs); and 3) PRS adjusted by gender, age and percentage of population admixture. These models were built using the largest four population-specific summary statistics of PD risk to date (base data) and individual level data obtained from the Global Parkinsons Genetics Program (target data). We performed power calculations to estimate the minimum sample size required to conduct these analyses. A total of 91 PRS models were developed to investigate cumulative known genetic variation associated with PD risk and age of onset in a global context.

ResultsWe observed marked heterogeneity in risk estimates across non-European ancestries, including East Asians, Central Asians, Latino/Admixed Americans, Africans, African admixed, and Ashkenazi Jewish populations. Risk allele patterns for the 90 risk variants yielded significant differences in directionality, frequency, and magnitude of effect. PRS did not improve in performance when predicting disease status using similar base and target data across multiple ancestries, demonstrating that cumulative PRS models based on current known risk are inherently biased towards European populations. We found that PRS models adjusted by percentage of admixture outperformed models that adjusted for conventional PCs in highly admixed populations. Overall, the clinical utility of our models in individually predicting PD status is limited in concordance with the estimates observed in European populations.

InterpretationThis study represents the first comprehensive assessment of how PRS models predict PD risk and age at onset in a multi-ancestry fashion. Given the heterogeneity and distinct genetic architecture of PD across different populations, our assessment emphasizes the need for larger and diverse study cohorts of individual-level target data and well-powered ancestry-specific summary statistics. Our current understanding of PD status unraveled through GWAS in European populations is not generally applicable to other ancestries. Future studies should integrate clinical and *omics level data to enhance the accuracy and predictive power of PRS across diverse populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23299044">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23299044" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23299044">
        <p class="paperTitle">Genomic epidemiology of SARS-CoV-2 in Sudan: A retrospective analysis 2020-2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23299044" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23299044" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mburu, M. W.; Lambisia, A. W.; Elnegoumi, M.; Nour, E. O. M.; Khair, O. M. M.; Abualas, E.; Elhassan, R. M.; Adam, M. A. M.; Mohammed, K. S.; Morobe, J. M.; Ndwiga, L.; Moraa, E. O.; Makori, T. O.; Murunga, N.; Katama, E. N.; Waruguru, D. W.; Githinji, G.; Mwangi, S.; Tessema, S. K.; Adam, M. H. E.; Mahmoud, I.; Mohamed, H. S.; Mustafa, Y. A. A.; Mohamed, M. M. A.; Ahmed, R. H. A.; Mohamed, N. Y. I.; Ali, R. H.; Ebraheem, R. S. M.; Ahmed, H. H. H.; Abdalla, H. E.; Mukhtar, I.; Alla, A. E. M. W.; Elsheikh, O. H. A.; Musa, A. O.; Faki, M. T.; Ahmed, G. M.; Ahmed, T. E. R.; Abdalla, M. A. O.; Moh</p>
        <p class="info">Score: 1.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23299044' target='https://doi.org/10.1101/2023.11.27.23299044'> 10.1101/2023.11.27.23299044</a></p>
        <p class="abstract">SARS-CoV-2 was first detected in Sudan on 13th March 2020. Here, we describe the genomic epidemiology of SARS-CoV-2 in Sudan between May 2020 and April 2022 to understand the introduction and transmission of SARS-CoV-2 variants in the country. A total of 667 SARS-CoV-2 positive samples were successfully sequenced using the nCoV-19 Artic protocol on the Oxford Nanopore Technology ([&amp;ge;]70% genome completeness). The genomes were compared with a select contemporaneous global dataset to determine genetic relatedness and estimate import/export events. The genomes were classified into 37 Pango lineages within the ancestral strain (107 isolates across 13 Pango lineages), Eta variant of interest (VOI) (78 isolates in 1 lineage), Alpha variant of concern (VOC) (10 isolates in 2 lineages), Beta VOC (26 isolates in 1 lineage), Delta VOC (171 isolates across 8 lineages) and Omicron VOC (242 isolates across 12 lineages). We estimated a total of 144 introductions of the observed variants from different countries across the globe. Multiple introductions of the Eta VOI, Beta VOC and Omicron VOC were observed in Sudan mainly from Europe and Africa. These findings suggest a need for continuous genomic surveillance of SARS-CoV-2 to monitor their introduction and spread consequently inform public health measures to combat SARS-CoV-2 transmission.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
