<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide association analysis of composite sleep health scores in 413,904 individuals
Authors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide association analysis of composite sleep health scores in 413,904 individuals
Authors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide association analysis of composite sleep health scores in 413,904 individuals
Authors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide association analysis of composite sleep health scores in 413,904 individuals\nAuthors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide association analysis of composite sleep health scores in 413,904 individuals\nAuthors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.; Wang, H.\nScore: 5.2, Published: 2024-02-03 DOI: 10.1101/2024.02.02.24302211\nRecent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, and together may provide a more complete picture of sleep health, while also illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p\u003c8.3e-9), along with 341 previously reported loci (p\u003c5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2=0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections to behavioral, psychological, and cardiometabolic traits.\nA genome-wide investigation into the underlying genetic architecture of personality traits and overlap with psychopathology.\nAuthors: Gupta, P.; Galimberti, M.; Liu, Y.; Beck, S.; Wingo, A.; Wingo, T.; Adhikari, K.; VA Million Veteran Program, ; Stein, M.; Gelernter, J.; Levey, D. F.\nScore: 5.6, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301428\nPersonality is influenced by both genetic and environmental factors and is associated with other psychiatric traits such as anxiety and depression. The \"Big Five\" personality traits, which include neuroticism, extraversion, agreeableness, conscientiousness, and openness, are a widely accepted and influential framework for understanding and describing human personality. Of the big five personality traits, neuroticism has most often been the focus of genetic studies and is linked to various mental illnesses including depression, anxiety, and schizophrenia. Our knowledge of the genetic architecture of the other four personality traits is more limited. Utilizing the Million Veteran Program (MVP) cohort we conducted a genome-wide association study (GWAS) in individuals of European and African ancestry. Adding other published data, we performed GWAS meta-analysis for each of the five personality traits with sample sizes ranging from 237,390 to 682,688. We identified 158, 14, 3, 2, and 7 independent genome-wide significant (GWS) loci associated with neuroticism, extraversion, agreeableness, conscientiousness, and openness, respectively. These findings represent 55 novel loci for neuroticism, as well as the first GWS loci discovered for extraversion and agreeableness. Gene-based association testing revealed 254 genes showing significant association with at least one of the five personality traits. Transcriptome-wide and proteome-wide analysis identified altered expression of genes and proteins such as CRHR1, SLC12A5, MAPT, and STX4. Pathway enrichment and drug perturbation analyses identified complex biology underlying human personality traits. We also studied the inter-relationship of personality traits with 1,437 other traits in a phenome-wide genetic correlation analysis, identifying new associations. Mendelian randomization showed positive bidirectional effects between neuroticism and depression and anxiety while a negative bidirectional effect was observed for agreeableness and these psychiatric traits. This study improves our comprehensive understanding of the genetic architecture underlying personality traits and their relationship to other complex human traits.\nPRESCOTT: a population aware, epistatic and structural model accurately predicts missense effect\nAuthors: Tekpinar, M.; David, L.; Henry, T.; Carbone, A.\nScore: 4.5, Published: 2024-02-06 DOI: 10.1101/2024.02.03.24302219\nPredicting the functional impact of point mutations is a complex yet vital task in genomics. PRESCOTT stands at the forefront of this challenge and reconstructs complete mutational landscapes of proteins, enables the identification of protein regions most vulnerable to mutations and assigns scores to individual mutations, assisting pathologists in evaluating the pathogenic potential of missense variants. PRESCOTT categorizes these variants into three distinct classes: Benign, Pathogenic, or Variants of Uncertain Significance (VUS). The model leverages protein sequences across millions of species, advanced protein structural models, and extensive genomic and exomic data from diverse human populations. By using only sequence and structural information, it significantly improves on current standards for predicting mutations in human proteins and matches AlphaMissense performance, which incorporates allele frequency data in its analysis. By including population-specific allele frequencies, PRESCOTT excels in genome-scale score separation of ClinVar benign and pathogenic variants and surpasses AlphaMissense in analyzing the ACMG reference human dataset and the over 1800 proteins from the Human Protein Dataset. Its efficacy is particularly notable in autoinflammatory diseases, accurately predicting pathogenic gain-of-function missense mutations, a task known for its difficulty. Efficiency and accessibility are key aspects of PRESCOTT. The user-friendly PRESCOTT webserver facilitates mutation effect calculations on any protein and protein variants. The server hosts a Comprehensive Human Protein Database for over 19,000 human proteins, based on sequences and structures, ready for a customized allele population analysis. Additionally, the tool provides open access to all intermediate scores, ensuring interpretability and transparency in variant analysis. PRESCOTT is a significant stride forward in the field of genomic medicine, offering unparalleled insights into protein mutational impacts.\nAn Oxford Nanopore Technology-Based Hepatitis B Virus Sequencing Protocol Suitable For Genomic Surveillance Within Clinical Diagnostic Settings.\nAuthors: Tshiabuila, D.; Choga, W.; San, J. E.; Maponga, T.; Preiser, W.; van Zyl, G.; Moir, M.; van Wyk, S.; Giandhari, J.; Pillay, S.; Anyaneji, U. J.; Lessells, R.; Naidoo, Y.; Sanko, T. J.; Wilkinson, E.; Tegally, H.; Baxter, C.; Martin, D.; de Oliveira, T.\nScore: 13.9, Published: 2024-01-25 DOI: 10.1101/2024.01.19.24301519\nChronic hepatitis B virus (HBV) infection remains a significant public health concern, particularly in Africa, where there is a substantial. HBV is an enveloped virus, with isolates being classified into ten phylogenetically distinct genotypes (A - J) determined based on full-genome sequence data or reverse hybridization-based diagnostic tests. In practice, limitations are noted in that diagnostic sequencing, generally using Sanger sequencing, tends to focus only on the S-gene, yielding little or no information on intra-patient HBV genetic diversity with very low-frequency variants and reverse hybridization detects only known genotype-specific mutations. To resolve these limitations, we developed an Oxford Nanopore Technology (ONT)-based HBV genotyping protocol suitable for clinical virology, yielding complete HBV genome sequences and extensive data on intra-patient HBV diversity. Specifically, the protocol involves tiling-based PCR amplification of HBV sequences, library preparation using the ONT Rapid Barcoding Kit, ONT GridION sequencing, genotyping using Genome Detective software, recombination analysis using jpHMM and RDP5 software, and drug resistance profiling using Geno2pheno software. We prove the utility of our protocol by efficiently generating and characterizing high-quality near full-length HBV genomes from 148 left-over diagnostic Hepatitis B patient samples obtained in the Western Cape province of South Africa, providing valuable insights into the genetic diversity and epidemiology of HBV in this region of the world.\nWhole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy\nAuthors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Stamelou, M.; Arzberger, T.; Roeber, S.; Muller, U.; Hopfner, F.; Hopfner, F.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L\nScore: 9.9, Published: 2024-01-30 DOI: 10.1101/2023.12.28.23300612\nBackgroundProgressive supranuclear palsy (PSP) is a rare neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. Previous genome-wide association studies for PSP were based on genotype array, therefore, were inadequate for the analysis of rare variants as well as larger mutations, such as small insertions/deletions (indels) and structural variants (SVs). MethodIn this study, we performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), indels, and SVs, in a cohort of 1,718 cases and 2,944 controls of European ancestry. Of the 1,718 PSP individuals, 1,441 were autopsy-confirmed and 277 were clinically diagnosed. ResultsOur analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and further uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. Notably, in contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele in PSP. Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. Moreover, seven common SVs associated with PSP were observed in the H1/H2 haplotype region (17q21.31) and other loci, including IGH, PCMT1, CYP2A13, and SMCP. In the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP. ConclusionsThrough WGS, we significantly enhanced our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.\nProteome-Wide Assessment of Clustering of Missense Variants in Neurodevelopmental Disorders Versus Cancer\nAuthors: Ng, J. K.; Chen, Y.; Akinwe, T. M.; Heins, H. B.; Mehinovic, E.; Chang, Y.; Payne, Z. L.; Manuel, J. G.; Karchin, R.; Turner, T. N.\nScore: 3.0, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302238\nMissense de novo variants (DNVs) and missense somatic variants contribute to neurodevelopmental disorders (NDDs) and cancer, respectively. Proteins with statistical enrichment based on analyses of these variants exhibit convergence in the differing NDD and cancer phenotypes. Herein, the question of why some of the same proteins are identified in both phenotypes is examined through investigation of clustering of missense variation at the protein level. Our hypothesis is that missense variation is present in different protein locations in the two phenotypes leading to the distinct phenotypic outcomes. We tested this hypothesis in 1D protein space using our software CLUMP. Furthermore, we newly developed 3D-CLUMP that uses 3D protein structures to spatially test clustering of missense variation for proteome-wide significance. We examined missense DNVs in 39,883 parent-child sequenced trios with NDDs and missense somatic variants from 10,543 sequenced tumors covering five TCGA cancer types and two COSMIC pan-cancer aggregates of tissue types. There were 57 proteins with proteome-wide significant missense variation clustering in NDDs when compared to cancers and 79 proteins with proteome-wide significant missense clustering in cancers compared to NDDs. While our main objective was to identify differences in patterns of missense variation, we also identified a novel NDD protein BLTP2. Overall, our study is innovative, provides new insights into differential missense variation in NDDs and cancer at the protein-level, and contributes necessary information toward building a framework for thinking about prognostic and therapeutic aspects of these proteins.\nVariant-specific pathophysiological mechanisms of AFF3 differently influence transcriptome profiles\nAuthors: Bassani, S.; Chrast, J.; Ambrosini, G.; Voisin, N.; Schütz, F.; Brusco, A.; Sirchia, F.; Turban, L.; Schubert, S.; Jamra, R. A.; Schlump, J.-U.; DeMille, D.; Bayrak-Toydemir, P.; Nelson, G. R.; Wong, K. N.; Duncan, L.; Mosera, M.; Gilissen, C.; Vissers, L. E. L. M.; Pfundt, R.; Kersseboom, R.; Yttervik, H.; Hansen, G. A. M.; Falkenberg Smeland, M.; Butler, K. M.; Lyons, M. J.; Carvalho, C. M. B.; Zhang, C.; Lupski, J. R.; Potocki, L.; Flores-Gallegos, L.; Morales-Toquero, R.; Petit, F.; Yalcin, B.; Tuttle, A.; Zghal Elloumi, H.; Mccormick, L.; Kukolich, M.; Klaas, O.; Horvath, J.; Scala, M.; Score: 7.0, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301100\nBackgroundWe previously described the KINSSHIP syndrome, an autosomal dominant disorder associated with intellectual disability (ID), mesomelic dysplasia and horseshoe kidney,caused by de novo variants in the degron of AFF3. Mouse knock-ins and overexpression in zebrafish provided evidence for a dominant-negative (DN) mode-of-action, wherein an increased level of AFF3 resulted in pathological effects. MethodsEvolutionary constraints suggest that other mode-of-inheritance could be at play. We challenged this hypothesis by screening ID cohorts for individuals with predicted-to-be deleterious variants in AFF3. We used both animal and cellular models to assess the deleteriousness of the identified variants. ResultsWe identified an individual with a KINSSHIP-like phenotype carrying a de novo partial duplication of AFF3 further strengthening the hypothesis that an increased level of AFF3 is pathological. We also detected seventeen individuals displaying a milder syndrome with either heterozygous LoF or biallelic missense variants in AFF3. Consistent with semi-dominance, we discovered three patients with homozygous LoF and one compound heterozygote for a LoF and a missense variant, who presented more severe phenotypes than their heterozygous parents. Matching zebrafish knockdowns exhibit neurological defects that could be rescued by expressing human AFF3 mRNA, confirming their association with the ablation of aff3. Conversely, some of the human AFF3 mRNAs carrying missense variants identified in affected individuals did not complement. Overexpression of mutated AFF3 mRNAs in zebrafish embryos produced a significant increase of abnormal larvae compared to wild-type overexpression further demonstrating deleteriousness. To further assess the effect of AFF3 variation, we profiled the transcriptome of fibroblasts from affected individuals and engineered isogenic cells harboring +/+, DN/DN, LoF/+, LoF/LoF or DN/LoF AFF3 genotypes. The expression of more than a third of the AFF3 bound loci is modified in either the DN/DN or the LoF/LoF lines. While the same pathways are affected, only about one-third of the differentially expressed genes are common to these homozygote datasets, indicating that AFF3 LoF and DN variants largely modulate transcriptomes differently, e.g. the DNA repair pathway displayed opposite modulation. ConclusionsOur results and the high pleiotropy shown by variation at this locus suggest that minute changes in AFF3 function are deleterious.\nConcordance of whole-genome amplified embryonic DNA with the subsequently born child\nAuthors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.\nScore: 21.5, Published: 2024-01-15 DOI: 10.1101/2024.01.12.24301086\nBefore implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGSs ability to identify de novo variants we compared the childs genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimers Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGSs capability to identify genetic variants not explicitly targeted for monogenic screening.\n",
  "wordCount" : "2524",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.24302211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.24302211" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.24302211">
        <p class="paperTitle">Genome-wide association analysis of composite sleep health scores in 413,904 individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.24302211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.24302211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.; Wang, H.</p>
        <p class="info">Score: 5.2, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.24302211' target='https://doi.org/10.1101/2024.02.02.24302211'> 10.1101/2024.02.02.24302211</a></p>
        <p class="abstract">Recent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, and together may provide a more complete picture of sleep health, while also illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p&lt;8.3e-9), along with 341 previously reported loci (p&lt;5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2=0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections to behavioral, psychological, and cardiometabolic traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301428" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301428">
        <p class="paperTitle">A genome-wide investigation into the underlying genetic architecture of personality traits and overlap with psychopathology.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, P.; Galimberti, M.; Liu, Y.; Beck, S.; Wingo, A.; Wingo, T.; Adhikari, K.; VA Million Veteran Program,  ; Stein, M.; Gelernter, J.; Levey, D. F.</p>
        <p class="info">Score: 5.6, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301428' target='https://doi.org/10.1101/2024.01.17.24301428'> 10.1101/2024.01.17.24301428</a></p>
        <p class="abstract">Personality is influenced by both genetic and environmental factors and is associated with other psychiatric traits such as anxiety and depression. The &#34;Big Five&#34; personality traits, which include neuroticism, extraversion, agreeableness, conscientiousness, and openness, are a widely accepted and influential framework for understanding and describing human personality. Of the big five personality traits, neuroticism has most often been the focus of genetic studies and is linked to various mental illnesses including depression, anxiety, and schizophrenia. Our knowledge of the genetic architecture of the other four personality traits is more limited. Utilizing the Million Veteran Program (MVP) cohort we conducted a genome-wide association study (GWAS) in individuals of European and African ancestry. Adding other published data, we performed GWAS meta-analysis for each of the five personality traits with sample sizes ranging from 237,390 to 682,688. We identified 158, 14, 3, 2, and 7 independent genome-wide significant (GWS) loci associated with neuroticism, extraversion, agreeableness, conscientiousness, and openness, respectively. These findings represent 55 novel loci for neuroticism, as well as the first GWS loci discovered for extraversion and agreeableness. Gene-based association testing revealed 254 genes showing significant association with at least one of the five personality traits. Transcriptome-wide and proteome-wide analysis identified altered expression of genes and proteins such as CRHR1, SLC12A5, MAPT, and STX4. Pathway enrichment and drug perturbation analyses identified complex biology underlying human personality traits. We also studied the inter-relationship of personality traits with 1,437 other traits in a phenome-wide genetic correlation analysis, identifying new associations. Mendelian randomization showed positive bidirectional effects between neuroticism and depression and anxiety while a negative bidirectional effect was observed for agreeableness and these psychiatric traits. This study improves our comprehensive understanding of the genetic architecture underlying personality traits and their relationship to other complex human traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.24302219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.24302219" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.24302219">
        <p class="paperTitle">PRESCOTT: a population aware, epistatic and structural model accurately predicts missense effect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.24302219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.24302219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tekpinar, M.; David, L.; Henry, T.; Carbone, A.</p>
        <p class="info">Score: 4.5, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.24302219' target='https://doi.org/10.1101/2024.02.03.24302219'> 10.1101/2024.02.03.24302219</a></p>
        <p class="abstract">Predicting the functional impact of point mutations is a complex yet vital task in genomics. PRESCOTT stands at the forefront of this challenge and reconstructs complete mutational landscapes of proteins, enables the identification of protein regions most vulnerable to mutations and assigns scores to individual mutations, assisting pathologists in evaluating the pathogenic potential of missense variants. PRESCOTT categorizes these variants into three distinct classes: Benign, Pathogenic, or Variants of Uncertain Significance (VUS). The model leverages protein sequences across millions of species, advanced protein structural models, and extensive genomic and exomic data from diverse human populations. By using only sequence and structural information, it significantly improves on current standards for predicting mutations in human proteins and matches AlphaMissense performance, which incorporates allele frequency data in its analysis. By including population-specific allele frequencies, PRESCOTT excels in genome-scale score separation of ClinVar benign and pathogenic variants and surpasses AlphaMissense in analyzing the ACMG reference human dataset and the over 1800 proteins from the Human Protein Dataset. Its efficacy is particularly notable in autoinflammatory diseases, accurately predicting pathogenic gain-of-function missense mutations, a task known for its difficulty. Efficiency and accessibility are key aspects of PRESCOTT. The user-friendly PRESCOTT webserver facilitates mutation effect calculations on any protein and protein variants. The server hosts a Comprehensive Human Protein Database for over 19,000 human proteins, based on sequences and structures, ready for a customized allele population analysis. Additionally, the tool provides open access to all intermediate scores, ensuring interpretability and transparency in variant analysis. PRESCOTT is a significant stride forward in the field of genomic medicine, offering unparalleled insights into protein mutational impacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.19.24301519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.19.24301519" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.19.24301519">
        <p class="paperTitle">An Oxford Nanopore Technology-Based Hepatitis B Virus Sequencing Protocol Suitable For Genomic Surveillance Within Clinical Diagnostic Settings.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.19.24301519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.19.24301519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tshiabuila, D.; Choga, W.; San, J. E.; Maponga, T.; Preiser, W.; van Zyl, G.; Moir, M.; van Wyk, S.; Giandhari, J.; Pillay, S.; Anyaneji, U. J.; Lessells, R.; Naidoo, Y.; Sanko, T. J.; Wilkinson, E.; Tegally, H.; Baxter, C.; Martin, D.; de Oliveira, T.</p>
        <p class="info">Score: 13.9, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.19.24301519' target='https://doi.org/10.1101/2024.01.19.24301519'> 10.1101/2024.01.19.24301519</a></p>
        <p class="abstract">Chronic hepatitis B virus (HBV) infection remains a significant public health concern, particularly in Africa, where there is a substantial. HBV is an enveloped virus, with isolates being classified into ten phylogenetically distinct genotypes (A - J) determined based on full-genome sequence data or reverse hybridization-based diagnostic tests. In practice, limitations are noted in that diagnostic sequencing, generally using Sanger sequencing, tends to focus only on the S-gene, yielding little or no information on intra-patient HBV genetic diversity with very low-frequency variants and reverse hybridization detects only known genotype-specific mutations. To resolve these limitations, we developed an Oxford Nanopore Technology (ONT)-based HBV genotyping protocol suitable for clinical virology, yielding complete HBV genome sequences and extensive data on intra-patient HBV diversity. Specifically, the protocol involves tiling-based PCR amplification of HBV sequences, library preparation using the ONT Rapid Barcoding Kit, ONT GridION sequencing, genotyping using Genome Detective software, recombination analysis using jpHMM and RDP5 software, and drug resistance profiling using Geno2pheno software. We prove the utility of our protocol by efficiently generating and characterizing high-quality near full-length HBV genomes from 148 left-over diagnostic Hepatitis B patient samples obtained in the Western Cape province of South Africa, providing valuable insights into the genetic diversity and epidemiology of HBV in this region of the world.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.23300612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.23300612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.23300612">
        <p class="paperTitle">Whole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.23300612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.23300612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Stamelou, M.; Arzberger, T.; Roeber, S.; Muller, U.; Hopfner, F.; Hopfner, F.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L</p>
        <p class="info">Score: 9.9, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.23300612' target='https://doi.org/10.1101/2023.12.28.23300612'> 10.1101/2023.12.28.23300612</a></p>
        <p class="abstract">BackgroundProgressive supranuclear palsy (PSP) is a rare neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. Previous genome-wide association studies for PSP were based on genotype array, therefore, were inadequate for the analysis of rare variants as well as larger mutations, such as small insertions/deletions (indels) and structural variants (SVs).

MethodIn this study, we performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), indels, and SVs, in a cohort of 1,718 cases and 2,944 controls of European ancestry. Of the 1,718 PSP individuals, 1,441 were autopsy-confirmed and 277 were clinically diagnosed.

ResultsOur analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and further uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. Notably, in contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele in PSP. Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. Moreover, seven common SVs associated with PSP were observed in the H1/H2 haplotype region (17q21.31) and other loci, including IGH, PCMT1, CYP2A13, and SMCP. In the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP.

ConclusionsThrough WGS, we significantly enhanced our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.24302238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.24302238" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.24302238">
        <p class="paperTitle">Proteome-Wide Assessment of Clustering of Missense Variants in Neurodevelopmental Disorders Versus Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.24302238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.24302238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ng, J. K.; Chen, Y.; Akinwe, T. M.; Heins, H. B.; Mehinovic, E.; Chang, Y.; Payne, Z. L.; Manuel, J. G.; Karchin, R.; Turner, T. N.</p>
        <p class="info">Score: 3.0, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.24302238' target='https://doi.org/10.1101/2024.02.02.24302238'> 10.1101/2024.02.02.24302238</a></p>
        <p class="abstract">Missense de novo variants (DNVs) and missense somatic variants contribute to neurodevelopmental disorders (NDDs) and cancer, respectively. Proteins with statistical enrichment based on analyses of these variants exhibit convergence in the differing NDD and cancer phenotypes. Herein, the question of why some of the same proteins are identified in both phenotypes is examined through investigation of clustering of missense variation at the protein level. Our hypothesis is that missense variation is present in different protein locations in the two phenotypes leading to the distinct phenotypic outcomes. We tested this hypothesis in 1D protein space using our software CLUMP. Furthermore, we newly developed 3D-CLUMP that uses 3D protein structures to spatially test clustering of missense variation for proteome-wide significance. We examined missense DNVs in 39,883 parent-child sequenced trios with NDDs and missense somatic variants from 10,543 sequenced tumors covering five TCGA cancer types and two COSMIC pan-cancer aggregates of tissue types. There were 57 proteins with proteome-wide significant missense variation clustering in NDDs when compared to cancers and 79 proteins with proteome-wide significant missense clustering in cancers compared to NDDs. While our main objective was to identify differences in patterns of missense variation, we also identified a novel NDD protein BLTP2. Overall, our study is innovative, provides new insights into differential missense variation in NDDs and cancer at the protein-level, and contributes necessary information toward building a framework for thinking about prognostic and therapeutic aspects of these proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.24301100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.24301100" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.24301100">
        <p class="paperTitle">Variant-specific pathophysiological mechanisms of AFF3 differently influence transcriptome profiles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.24301100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.24301100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bassani, S.; Chrast, J.; Ambrosini, G.; Voisin, N.; Schütz, F.; Brusco, A.; Sirchia, F.; Turban, L.; Schubert, S.; Jamra, R. A.; Schlump, J.-U.; DeMille, D.; Bayrak-Toydemir, P.; Nelson, G. R.; Wong, K. N.; Duncan, L.; Mosera, M.; Gilissen, C.; Vissers, L. E. L. M.; Pfundt, R.; Kersseboom, R.; Yttervik, H.; Hansen, G. A. M.; Falkenberg Smeland, M.; Butler, K. M.; Lyons, M. J.; Carvalho, C. M. B.; Zhang, C.; Lupski, J. R.; Potocki, L.; Flores-Gallegos, L.; Morales-Toquero, R.; Petit, F.; Yalcin, B.; Tuttle, A.; Zghal Elloumi, H.; Mccormick, L.; Kukolich, M.; Klaas, O.; Horvath, J.; Scala, M.; </p>
        <p class="info">Score: 7.0, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.24301100' target='https://doi.org/10.1101/2024.01.14.24301100'> 10.1101/2024.01.14.24301100</a></p>
        <p class="abstract">BackgroundWe previously described the KINSSHIP syndrome, an autosomal dominant disorder associated with intellectual disability (ID), mesomelic dysplasia and horseshoe kidney,caused by de novo variants in the degron of AFF3. Mouse knock-ins and overexpression in zebrafish provided evidence for a dominant-negative (DN) mode-of-action, wherein an increased level of AFF3 resulted in pathological effects.

MethodsEvolutionary constraints suggest that other mode-of-inheritance could be at play. We challenged this hypothesis by screening ID cohorts for individuals with predicted-to-be deleterious variants in AFF3. We used both animal and cellular models to assess the deleteriousness of the identified variants.

ResultsWe identified an individual with a KINSSHIP-like phenotype carrying a de novo partial duplication of AFF3 further strengthening the hypothesis that an increased level of AFF3 is pathological. We also detected seventeen individuals displaying a milder syndrome with either heterozygous LoF or biallelic missense variants in AFF3. Consistent with semi-dominance, we discovered three patients with homozygous LoF and one compound heterozygote for a LoF and a missense variant, who presented more severe phenotypes than their heterozygous parents. Matching zebrafish knockdowns exhibit neurological defects that could be rescued by expressing human AFF3 mRNA, confirming their association with the ablation of aff3. Conversely, some of the human AFF3 mRNAs carrying missense variants identified in affected individuals did not complement. Overexpression of mutated AFF3 mRNAs in zebrafish embryos produced a significant increase of abnormal larvae compared to wild-type overexpression further demonstrating deleteriousness. To further assess the effect of AFF3 variation, we profiled the transcriptome of fibroblasts from affected individuals and engineered isogenic cells harboring &#43;/&#43;, DN/DN, LoF/&#43;, LoF/LoF or DN/LoF AFF3 genotypes. The expression of more than a third of the AFF3 bound loci is modified in either the DN/DN or the LoF/LoF lines. While the same pathways are affected, only about one-third of the differentially expressed genes are common to these homozygote datasets, indicating that AFF3 LoF and DN variants largely modulate transcriptomes differently, e.g. the DNA repair pathway displayed opposite modulation.

ConclusionsOur results and the high pleiotropy shown by variation at this locus suggest that minute changes in AFF3 function are deleterious.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301086" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301086">
        <p class="paperTitle">Concordance of whole-genome amplified embryonic DNA with the subsequently born child</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.</p>
        <p class="info">Score: 21.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301086' target='https://doi.org/10.1101/2024.01.12.24301086'> 10.1101/2024.01.12.24301086</a></p>
        <p class="abstract">Before implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGSs ability to identify de novo variants we compared the childs genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimers Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGSs capability to identify genetic variants not explicitly targeted for monogenic screening.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
