<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease
Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.
Score: 27.2, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623
Studies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease
Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.
Score: 27.2, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623
Studies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease
Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.
Score: 27.2, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623
Studies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Polygenic burden of short tandem repeat expansions promote risk for Alzheimer\u0026#39;s disease\nAuthors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.\nScore: 27.2, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623\nStudies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation.",
  "keywords": [
    
  ],
  "articleBody": " Polygenic burden of short tandem repeat expansions promote risk for Alzheimer's disease\nAuthors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.\nScore: 27.2, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623\nStudies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation. There are over one million short tandem repeats (STRs) in the genome, and their link to AD risk has not been assessed. As pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk. Here, we genotyped 321,742 polymorphic STR tracts genome-wide using PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals). We implemented an approach to identify STR expansions as STRs with tract lengths that are outliers from the population. We then tested for differences in aggregate burden of expansions in case versus control individuals. AD patients had a 1.19-fold increase of STR expansions compared to healthy elderly controls (p=8.27x10-3, two-sided Mann Whitney test). Individuals carrying \u003e 30 STR expansions had 3.62-fold higher odds of having AD and had more severe AD neuropathology. AD STR expansions were highly enriched within active promoters in post-mortem hippocampal brain tissues and particularly within SINE-VNTR-Alu (SVA) retrotransposons. Together, these results demonstrate that expanded STRs within active promoter regions of the genome promote risk of AD.\nCR1 variants contribute to FSGS susceptibility across multiple populations.\nAuthors: Skitchenko, R.; Modrusan, Z.; Loboda, A.; Kopp, J. B.; Winkler, C. A.; Sergushichev, A.; Gupta, N.; Stevens, C.; Daly, M. J.; Shaw, A.; Artomov, M.\nScore: 3.0, Published: 2023-11-20 DOI: 10.1101/2023.11.20.23298462\nFocal segmental glomerulosclerosis (FSGS) is a common cause of nephrotic syndrome with an annual incidence in the United States in African-Americans compared to European-Americans of 24 cases and 5 cases per million, respectively. Among glomerular diseases in Europe and Latin-America, FSGS was the second most frequent diagnosis, and in Asia the fifth. We expand previous efforts in understanding genetics of FSGS by performing a case-control study involving ethnically-diverse groups FSGS cases (726) and a pool of controls (13,994), using panel sequencing of approximately 2,500 podocyte-expressed genes. Through rare variant association tests, we replicated known risk genes - KANK1, COL4A4, and APOL1. A novel significant association was observed for the gene encoding complement receptor 1 (CR1). High-risk rare variants in CR1 in the European-American cohort were commonly observed in Latin- and African-Americans. Therefore, a combined rare and common variant analysis was used to replicate the CR1 association in non-European populations. The CR1 risk variant, rs17047661, gives rise to the Sl1/Sl2 (R1601G) allele that was previously associated with protection against cerebral malaria. Pleiotropic effects of rs17047661 may explain the difference in allele frequencies across continental ancestries and suggest a possible role for genetically-driven alterations of adaptive immunity in the pathogenesis of FSGS.\nEvaluation of polygenic scoring methods in five biobanks reveals greater variability between biobanks than between methods and highlights benefits of ensemble learning\nAuthors: Monti, R.; Eick, L.; Hudjashov, G.; Läll, K.; Kanoni, S.; Wolford, B. N.; Wingfield, B.; Pain, O.; Wharrie, S.; Jermy, B.; McMahon, A.; Hartonen, T.; Heyne, H. O.; Mars, N.; Genes \u0026 Health Research Team, ; Hveem, K.; Inouye, M.; van Heel, D. A.; Mägi, R.; Marttinen, P.; Ripatti, S.; Ganna, A.; Lippert, C.\nScore: 2.5, Published: 2023-11-20 DOI: 10.1101/2023.11.20.23298215\nMethods to estimate polygenic scores (PGS) from genome-wide association studies are increasingly utilized. However, independent method evaluation is lacking, and method 15 comparisons are often limited. Here, we evaluate polygenic scores derived using seven methods in five biobank studies (totaling about 1.2 million participants) across 16 diseases and quantitative traits, building on a reference-standardized framework. We conducted meta- analyses to quantify the effects of method choice, hyperparameter tuning, method ensembling and target biobank on PGS performance. We found that no single method consistently 20 outperformed all others. PGS effect sizes were more variable between biobanks than between methods within biobanks when methods were well-tuned. Differences between methods were largest for the two investigated autoimmune diseases, seropositive rheumatoid arthritis and type 1 diabetes. For most methods, cross-validation was more reliable for tuning hyperparameters than automatic tuning (without the use of target data). For a given target 25 phenotype, elastic net models combining PGS across methods (ensemble PGS) tuned in the UK Biobank provided consistent, high, and cross-biobank transferable performance, increasing PGS effect sizes ({beta}-coefficients) by a median of 5.0% relative to LDpred2 and MegaPRS (the two best performing single methods when tuned with cross-validation). Our interactively browsable online-results 30 (https://methodscomparison.intervenegeneticscores.org/) and open-source workflow prspipe (https://github.com/intervene-EU-H2020/prspipe) provide a rich resource and reference for the analysis of polygenic scoring methods across biobanks.\nA comprehensive characterization of blood group antigen variants in the Middle Eastern population genomes - Insights into genetic epidemiology\nAuthors: Rophina, M.; Pandhare, K. R.; Scaria, V.\nScore: 1.8, Published: 2023-11-20 DOI: 10.1101/2023.11.20.23298777\nBackground The Middle Eastern population is characterized by increased prevalence of various Mendelian disorders owing to increased rates of consanguinity. Especially in disease conditions which require chronic transfusion support, it becomes important to know the blood group characteristics of potential donors to increase the likelihood of transfusion success. As there prevails a gap in knowledge about the population specific overall blood antigen profiles, this study seeks to utilize next generation sequencing datasets to unravel the comprehensive landscape of clinically significant minor blood group alleles in the middle eastern population. Methods This study utilizes the genetic variation data from a range of public datasets including the Greater Middle East Variome, the Qatar genome and exomes and the Iranome datasets to estimate the genotypic and phenotypic frequencies of blood group alleles in the Middle Easterners. The estimated frequencies were duly compared with major global populations to identify significant similarities or differences if any. Results A total of 77 unique ISBT approved blood group alleles were found commonly in all datasets. 8 variants (rs8176058, rs1058396, rs565898944, rs28362692, rs2071699, rs34783571, rs60322991 and rs57467915) belonging to KELL, KIDD, COLTON, H, JUNIOR and LANGEREIS blood groups were found clinically significant with previously reported evidence on transfusion complications. 730 variants were found to span exonic or splicing regions out of which 70 were predicted to be potentially deleterious by at least four computational tools. Conclusions This study serves first of its kind to extensively characterize the known and novel blood alleles in the Middle Easterners. A comprehensive user-friendly online resource named alnasab - Alleles and antigens in Arab and Persian populations associated with blood groups was also developed as a dependable reference for future transfusion research. The resource is accessible at https://clingen.igib.res.in/alnasab/\nEstimating cancer risk in carriers of Lynch syndrome variants in UK Biobank\nAuthors: Fummey, E.; Navarro, P.; Plazzer, J.-P.; Frayling, I. M.; Knott, S.; Tenesa, A.\nScore: 2.5, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298308\nBackgroundLynch syndrome (LS) is an inherited cancer predisposition syndrome caused by genetic variants affecting DNA mismatch repair (MMR) genes MLH1, MSH2, MSH6, and PMS2. Cancer risk in LS is estimated from cohorts of individuals ascertained by family history of cancer, which is known to upwardly bias estimates. MethodsThe InSiGHT Database classifies MMR gene variants by pathogenicity through expert panel review of published evidence. 830 carriers of pathogenic or likely pathogenic (path_MMR) MMR gene variants from InSiGHT were identified in 454,756 UK Biobank participants using whole exome sequence. Nelson-Aalen survival analysis was used to estimate cumulative incidence of colorectal, endometrial, and breast cancer. ResultsCumulative incidence of colorectal and endometrial cancer by age 70 was elevated in path_MMR carriers compared to non-carriers (colorectal: 11.8% (95% CI: 9.5 - 14.6) vs. 1.7% (1.6 - 1.7), endometrial: 13.4% (10.2 - 17.6) vs. 1.0% (0.9 - 1.0)), but the magnitude of this increase differed between genes. Cumulative breast cancer incidence by age 70 was not elevated in path_MMR carriers compared to non-carriers (8.9% (6.3 - 12.4) vs. 7.5% (7.4 - 7.6)). Cumulative cancer incidence estimates in UK Biobank were similar to estimates from the Prospective Lynch Syndrome Database for all genes and cancers, except there was no evidence for elevated endometrial cancer risk in carriers of pathogenic PMS2 variants in UK Biobank. ConclusionThese results can be used to inform the management of incidentally identified cases of LS. For example, they support the application of existing colorectal cancer surveillance strategies for LS in incidentally identified cases.\nA Massively Parallel Screen of 5'UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes\nAuthors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.\nScore: 18.9, Published: 2023-11-03 DOI: 10.1101/2023.11.02.23297961\nDe novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5'UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5'UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5'UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.\nMapping the landscape of lineage-specific dynamic regulation of gene expression using single-cell transcriptomics and application to genetics of complex disease\nAuthors: Abe, H.; Lin, P.; Zhou, D.; Ruderfer, D. M.; Gamazon, E. R.\nScore: 1.2, Published: 2023-10-25 DOI: 10.1101/2023.10.24.23297476\nSingle-cell transcriptome data can provide insights into how genetic variation influences biological processes involved in human biology and disease. However, the identification of gene-level associations in distinct cell types faces several challenges, including the limited reference resource from population scale studies, data sparsity in single-cell RNA sequencing, and the complex cell-state pattern of expression within individual cell types. Here we develop genetic models of cell type specific and cell state adjusted gene expression in dopaminergic neurons in the process of specializing from induced pluripotent stem cells. The resulting framework quantifies the dynamics of the genetic regulation of gene expression and estimates its cell type specificity. As an application, we show that the approach detects known and new genes associated with schizophrenia and enables insights into context-dependent disease mechanisms. We provide a genomic resource from a phenome-wide application of our models to more than 1500 phenotypes from the UK Biobank. Using longitudinal genetically determined expression, we implement a predictive causality framework, evaluating the prediction of future values of a target gene expression using prior values of a putative regulatory gene. Collectively, this work demonstrates the insights that can be gained into the molecular underpinnings of diseases by quantifying the genetic control of gene expression at single-cell resolution.\n",
  "wordCount" : "1934",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.23298623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.23298623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.23298623">
        <p class="paperTitle">Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.23298623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.23298623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.</p>
        <p class="info">Score: 27.2, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.23298623' target='https://doi.org/10.1101/2023.11.16.23298623'> 10.1101/2023.11.16.23298623</a></p>
        <p class="abstract">Studies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation. There are over one million short tandem repeats (STRs) in the genome, and their link to AD risk has not been assessed. As pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk. Here, we genotyped 321,742 polymorphic STR tracts genome-wide using PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals). We implemented an approach to identify STR expansions as STRs with tract lengths that are outliers from the population. We then tested for differences in aggregate burden of expansions in case versus control individuals. AD patients had a 1.19-fold increase of STR expansions compared to healthy elderly controls (p=8.27x10-3, two-sided Mann Whitney test). Individuals carrying &gt; 30 STR expansions had 3.62-fold higher odds of having AD and had more severe AD neuropathology. AD STR expansions were highly enriched within active promoters in post-mortem hippocampal brain tissues and particularly within SINE-VNTR-Alu (SVA) retrotransposons. Together, these results demonstrate that expanded STRs within active promoter regions of the genome promote risk of AD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298462">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298462" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298462">
        <p class="paperTitle">CR1 variants contribute to FSGS susceptibility across multiple populations.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298462" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298462" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Skitchenko, R.; Modrusan, Z.; Loboda, A.; Kopp, J. B.; Winkler, C. A.; Sergushichev, A.; Gupta, N.; Stevens, C.; Daly, M. J.; Shaw, A.; Artomov, M.</p>
        <p class="info">Score: 3.0, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298462' target='https://doi.org/10.1101/2023.11.20.23298462'> 10.1101/2023.11.20.23298462</a></p>
        <p class="abstract">Focal segmental glomerulosclerosis (FSGS) is a common cause of nephrotic syndrome with an annual incidence in the United States in African-Americans compared to European-Americans of 24 cases and 5 cases per million, respectively. Among glomerular diseases in Europe and Latin-America, FSGS was the second most frequent diagnosis, and in Asia the fifth. We expand previous efforts in understanding genetics of FSGS by performing a case-control study involving ethnically-diverse groups FSGS cases (726) and a pool of controls (13,994), using panel sequencing of approximately 2,500 podocyte-expressed genes. Through rare variant association tests, we replicated known risk genes - KANK1, COL4A4, and APOL1. A novel significant association was observed for the gene encoding complement receptor 1 (CR1). High-risk rare variants in CR1 in the European-American cohort were commonly observed in Latin- and African-Americans. Therefore, a combined rare and common variant analysis was used to replicate the CR1 association in non-European populations. The CR1 risk variant, rs17047661, gives rise to the Sl1/Sl2 (R1601G) allele that was previously associated with protection against cerebral malaria. Pleiotropic effects of rs17047661 may explain the difference in allele frequencies across continental ancestries and suggest a possible role for genetically-driven alterations of adaptive immunity in the pathogenesis of FSGS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298215">
        <p class="paperTitle">Evaluation of polygenic scoring methods in five biobanks reveals greater variability between biobanks than between methods and highlights benefits of ensemble learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monti, R.; Eick, L.; Hudjashov, G.; Läll, K.; Kanoni, S.; Wolford, B. N.; Wingfield, B.; Pain, O.; Wharrie, S.; Jermy, B.; McMahon, A.; Hartonen, T.; Heyne, H. O.; Mars, N.; Genes &amp; Health Research Team,  ; Hveem, K.; Inouye, M.; van Heel, D. A.; Mägi, R.; Marttinen, P.; Ripatti, S.; Ganna, A.; Lippert, C.</p>
        <p class="info">Score: 2.5, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298215' target='https://doi.org/10.1101/2023.11.20.23298215'> 10.1101/2023.11.20.23298215</a></p>
        <p class="abstract">Methods to estimate polygenic scores (PGS) from genome-wide association studies are increasingly utilized. However, independent method evaluation is lacking, and method 15 comparisons are often limited. Here, we evaluate polygenic scores derived using seven methods in five biobank studies (totaling about 1.2 million participants) across 16 diseases and quantitative traits, building on a reference-standardized framework. We conducted meta- analyses to quantify the effects of method choice, hyperparameter tuning, method ensembling and target biobank on PGS performance. We found that no single method consistently 20 outperformed all others. PGS effect sizes were more variable between biobanks than between methods within biobanks when methods were well-tuned. Differences between methods were largest for the two investigated autoimmune diseases, seropositive rheumatoid arthritis and type 1 diabetes. For most methods, cross-validation was more reliable for tuning hyperparameters than automatic tuning (without the use of target data). For a given target 25 phenotype, elastic net models combining PGS across methods (ensemble PGS) tuned in the UK Biobank provided consistent, high, and cross-biobank transferable performance, increasing PGS effect sizes ({beta}-coefficients) by a median of 5.0% relative to LDpred2 and MegaPRS (the two best performing single methods when tuned with cross-validation). Our interactively browsable online-results 30 (https://methodscomparison.intervenegeneticscores.org/) and open-source workflow prspipe (https://github.com/intervene-EU-H2020/prspipe) provide a rich resource and reference for the analysis of polygenic scoring methods across biobanks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298777">
        <p class="paperTitle">A comprehensive characterization of blood group antigen variants in the Middle Eastern population genomes - Insights into genetic epidemiology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rophina, M.; Pandhare, K. R.; Scaria, V.</p>
        <p class="info">Score: 1.8, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298777' target='https://doi.org/10.1101/2023.11.20.23298777'> 10.1101/2023.11.20.23298777</a></p>
        <p class="abstract">Background The Middle Eastern population is characterized by increased prevalence of various Mendelian disorders owing to increased rates of consanguinity. Especially in disease conditions which require chronic transfusion support, it becomes important to know the blood group characteristics of potential donors to increase the likelihood of transfusion success. As there prevails a gap in knowledge about the population specific overall blood antigen profiles, this study seeks to utilize next generation sequencing datasets to unravel the comprehensive landscape of clinically significant minor blood group alleles in the middle eastern population. Methods This study utilizes the genetic variation data from a range of public datasets including the Greater Middle East Variome, the Qatar genome and exomes and the Iranome datasets to estimate the genotypic and phenotypic frequencies of blood group alleles in the Middle Easterners. The estimated frequencies were duly compared with major global populations to identify significant similarities or differences if any. Results A total of 77 unique ISBT approved blood group alleles were found commonly in all datasets. 8 variants (rs8176058, rs1058396, rs565898944, rs28362692, rs2071699, rs34783571, rs60322991 and rs57467915) belonging to KELL, KIDD, COLTON, H, JUNIOR and LANGEREIS blood groups were found clinically significant with previously reported evidence on transfusion complications. 730 variants were found to span exonic or splicing regions out of which 70 were predicted to be potentially deleterious by at least four computational tools. Conclusions This study serves first of its kind to extensively characterize the known and novel blood alleles in the Middle Easterners. A comprehensive user-friendly online resource named alnasab - Alleles and antigens in Arab and Persian populations associated with blood groups was also developed as a dependable reference for future transfusion research. The resource is accessible at https://clingen.igib.res.in/alnasab/</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298308">
        <p class="paperTitle">Estimating cancer risk in carriers of Lynch syndrome variants in UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fummey, E.; Navarro, P.; Plazzer, J.-P.; Frayling, I. M.; Knott, S.; Tenesa, A.</p>
        <p class="info">Score: 2.5, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298308' target='https://doi.org/10.1101/2023.11.10.23298308'> 10.1101/2023.11.10.23298308</a></p>
        <p class="abstract">BackgroundLynch syndrome (LS) is an inherited cancer predisposition syndrome caused by genetic variants affecting DNA mismatch repair (MMR) genes MLH1, MSH2, MSH6, and PMS2. Cancer risk in LS is estimated from cohorts of individuals ascertained by family history of cancer, which is known to upwardly bias estimates.

MethodsThe InSiGHT Database classifies MMR gene variants by pathogenicity through expert panel review of published evidence. 830 carriers of pathogenic or likely pathogenic (path_MMR) MMR gene variants from InSiGHT were identified in 454,756 UK Biobank participants using whole exome sequence. Nelson-Aalen survival analysis was used to estimate cumulative incidence of colorectal, endometrial, and breast cancer.

ResultsCumulative incidence of colorectal and endometrial cancer by age 70 was elevated in path_MMR carriers compared to non-carriers (colorectal: 11.8% (95% CI: 9.5 - 14.6) vs. 1.7% (1.6 - 1.7), endometrial: 13.4% (10.2 - 17.6) vs. 1.0% (0.9 - 1.0)), but the magnitude of this increase differed between genes. Cumulative breast cancer incidence by age 70 was not elevated in path_MMR carriers compared to non-carriers (8.9% (6.3 - 12.4) vs. 7.5% (7.4 - 7.6)). Cumulative cancer incidence estimates in UK Biobank were similar to estimates from the Prospective Lynch Syndrome Database for all genes and cancers, except there was no evidence for elevated endometrial cancer risk in carriers of pathogenic PMS2 variants in UK Biobank.

ConclusionThese results can be used to inform the management of incidentally identified cases of LS. For example, they support the application of existing colorectal cancer surveillance strategies for LS in incidentally identified cases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297961">
        <p class="paperTitle">A Massively Parallel Screen of 5&#39;UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.</p>
        <p class="info">Score: 18.9, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297961' target='https://doi.org/10.1101/2023.11.02.23297961'> 10.1101/2023.11.02.23297961</a></p>
        <p class="abstract">De novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5&#39;UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5&#39;UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5&#39;UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.23297476">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.23297476" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.23297476">
        <p class="paperTitle">Mapping the landscape of lineage-specific dynamic regulation of gene expression using single-cell transcriptomics and application to genetics of complex disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.23297476" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.23297476" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abe, H.; Lin, P.; Zhou, D.; Ruderfer, D. M.; Gamazon, E. R.</p>
        <p class="info">Score: 1.2, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.23297476' target='https://doi.org/10.1101/2023.10.24.23297476'> 10.1101/2023.10.24.23297476</a></p>
        <p class="abstract">Single-cell transcriptome data can provide insights into how genetic variation influences biological processes involved in human biology and disease. However, the identification of gene-level associations in distinct cell types faces several challenges, including the limited reference resource from population scale studies, data sparsity in single-cell RNA sequencing, and the complex cell-state pattern of expression within individual cell types. Here we develop genetic models of cell type specific and cell state adjusted gene expression in dopaminergic neurons in the process of specializing from induced pluripotent stem cells. The resulting framework quantifies the dynamics of the genetic regulation of gene expression and estimates its cell type specificity. As an application, we show that the approach detects known and new genes associated with schizophrenia and enables insights into context-dependent disease mechanisms. We provide a genomic resource from a phenome-wide application of our models to more than 1500 phenotypes from the UK Biobank. Using longitudinal genetically determined expression, we implement a predictive causality framework, evaluating the prediction of future values of a target gene expression using prior values of a putative regulatory gene. Collectively, this work demonstrates the insights that can be gained into the molecular underpinnings of diseases by quantifying the genetic control of gene expression at single-cell resolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
