<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors
Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.
Score: 34.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414
BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors
Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.
Score: 34.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414
BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors
Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.
Score: 34.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414
BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors\nAuthors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.\nScore: 34.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414\nBackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1.",
  "keywords": [
    
  ],
  "articleBody": " Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors\nAuthors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.\nScore: 34.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414\nBackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1.30M Latine, and 0.45M African American individuals). Using cross-sectional data for PRS construction and a prospective cohort for evaluation, we estimated PRS-associated cumulative incidences after one year of follow-up for 12 clinical endpoints. ResultsThe cumulative incidence of disease at one year was consistently higher among individuals in the top 10% of each PRS. Relative risks (RRs) comparing top to bottom 10% of PRS differed across diseases (e.g. European RR 2.12 for eczema vs 12.53 for T2D). Estimates were similar between Europeans and Latines however were more modest for African Americans (e.g. T2D RR 10.92 for Latines vs. 4.00 for African Americans). Clinical manifestation occurred earlier among those in top vs bottom 10% of polygenic risk: 16yrs for hypertension, and 9.5yrs for T2D. Among participants at elevated conventional risk of CHD or T2D, those in the top 10% PRS had a 10-20 fold higher RR of disease incidence vs those not at conventional risk. Among individuals at high polygenic risk of CHD or T2D, favorable lifestyle characteristics associated with 64-73% lower RR of developing disease over 1-year, with cumulative incidence equivalent to the population average. ConclusionIn an ancestrally-diverse cohort, individuals in the top 10% PRS had higher 1-year disease incidence and earlier age of clinical manifestation. PRS provided risk stratification beyond conventional risk factors. Lifestyle characteristics markedly lowered disease incidence among those at elevated polygenic risk.\nDecoding Genetics, Ancestry, and Geospatial Context for Precision Health\nAuthors: Koyama, S.; Wang, Y.; Paruchuri, K.; Uddin, M. M.; Cho, S. M. J.; Urbut, S. M.; Haidermota, S.; Hornsby, W. E.; Green, R. C.; Daly, M. J.; Neale, B. M.; Ellinor, P. T.; Smoller, J. W.; Lebo, M. S.; Karlson, E. W.; Martin, A. R.; Natarajan, P.\nScore: 32.2, Published: 2023-10-25 DOI: 10.1101/2023.10.24.23297096\nMass General Brigham, an integrated healthcare system based in the Greater Boston area of Massachusetts, annually serves 1.5 million patients. We established the Mass General Brigham Biobank (MGBB), encompassing 142,238 participants, to unravel the intricate relationships among genomic profiles, environmental context, and disease manifestations within clinical practice. In this study, we highlight the impact of ancestral diversity in the MGBB by employing population genetics, geospatial assessment, and association analyses of rare and common genetic variants. The population structures captured by the genetics mirror the sequential immigration to the Greater Boston area throughout American history, highlighting communities tied to shared genetic and environmental factors. Our investigation underscores the potency of unbiased, large-scale analyses in a healthcare-affiliated biobank, elucidating the dynamic interplay across genetics, immigration, structural geospatial factors, and health outcomes in one of the earliest American sites of European colonization.\nDeterminants of mosaic chromosomal alteration fitness\nAuthors: Pershad, Y.; Mack, T.; Poisner, H. M.; Jakubek, Y. A.; Stilp, A. M.; Mitchell, B. D.; Lewis, J. P.; Boerwinkle, E.; Loos, R. J.; Chami, N.; Wang, Z.; Barnes, K.; Pankratz, N.; Fornage, M.; Redline, S.; Psaty, B. M.; Bis, J. C.; Shojaie, A.; Silverman, E. K.; Cho, M. H.; Yun, J.; DeMeo, D.; Levy, D.; Johnson, A.; Mathias, R.; Taub, M.; Arnett, D.; North, K.; Raffield, L. M.; Carlson, A.; Doyle, M. F.; Rich, S. S.; Rotter, J. I.; Guo, X.; Cox, N.; Roden, D. M.; Franceschini, N.; Desai, P.; Reiner, A.; Auer, P. L.; Scheet, P.; Jaiswal, S.; Weinstock, J. S.; Bick, A. G.\nScore: 14.8, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297280\nClonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal Expansion Rate (PACER) method to estimate clonal expansion rate for 6,381 individuals in the NHLBI TOPMed cohort with gain, loss, and copy-neutral loss of heterozygosity mCAs. Our estimates of mCA fitness were correlated (R2 = 0.49) with an alternative approach that estimated fitness of mCAs in the UK Biobank using a theoretical probability distribution. Individuals with lymphoid-associated mCAs had a significantly higher white blood cell count and faster clonal expansion rate. In a cross-sectional analysis, genome-wide association study of estimates of mCA expansion rate identified TCL1A, NRIP1, and TERT locus variants as modulators of mCA clonal expansion rate.\nPlasticity of Human Microglia and Brain Perivascular Macrophages in Aging and Alzheimer's Disease\nAuthors: Lee, D.; Porras, C.; Spencer, C.; Pjanic, M.; Weiler, P.; Kosoy, R.; Bendl, J.; N.M., P.; Wang, X.; Zheng, S.; Therrien, K.; Mathur, D.; Kleopoulos, S. P.; Shao, Z.; Argyriou, S.; Alvia, M.; Casey, C.; Hong, A.; Beaumont, K. G.; Sebra, R.; Kellner, C. P.; Bennett, D. A.; Yuan, G.-C.; Voloudakis, G.; Theis, F. J.; Haroutunian, V.; Hoffman, G. E.; Fullard, J. F.; Roussos, P.\nScore: 12.5, Published: 2023-10-26 DOI: 10.1101/2023.10.25.23297558\nMicroglia and perivascular macrophages, myeloid-origin resident immune cells in the human brain, play crucial roles in Alzheimer's disease (AD)1-4. However, the field lacks a unified taxonomy describing their heterogeneity and plasticity5. To address this, we applied single-cell profiling to two independent, demographically diverse cohorts. The first comprises 543,012 viable myeloid cells from 137 unique postmortem brain specimens, while the second consists of 289,493 myeloid nuclei from 1,470 donors. Collectively, they cover the human lifespan and varying degrees of AD neuropathology. We identify 13 transcriptionally distinct myeloid subtypes, including the \"GPNMB\" subtype that proliferates with AD. We distinguish two contrasting homeostatic microglial states in AD and with aging: the first (\"FRMD4A\") wanes over time, while the second (\"PICALM\") becomes more prevalent. By prioritizing AD-risk genes, including PTPRG, DPYD, and IL15, and placing them into a regulatory hierarchy, we identify common upstream transcriptional regulators, namely MITF and KLF12, that regulate the expression of AD-risk genes in the opposite directions. Through the construction of cell-to-cell interaction networks, we identify candidate ligand-receptor pairs, including APOE:SORL1 and APOE:TREM2, associated with AD progression. We show polygenic risk for AD predisposes and prioritize the GPNMB subtype as a therapeutic target of early intervention. Our findings delineate the relationship between distinct functional states of myeloid cells and their pathophysiological response to aging and AD, providing a significant step toward the mechanistic understanding of the roles of microglia in AD and the identification of novel therapeutics.\nDNAm scores for serum GDF15 and NT-proBNP levels associate with a range of traits affecting the body and brain\nAuthors: Gadd, D. A.; Smith, H.; Mullin, D.; Chybowska, A.; Hillary, R. F.; Kimenai, D. M.; Bernabeu, E.; Cheng, Y.; Fawns-Ritchie, C.; Campbell, A.; Page, D.; Taylor, A.; Corley, J.; Valdes Hernandez, M. D. C.; Munoz Maniega, S.; Bastin, M. E.; Wardlaw, J. M.; Walker, R. M.; Evans, K. L.; McIntosh, A. M.; Hayward, C.; Russ, T. C.; Harris, S. E.; Welsh, P.; Sattar, N.; Cox, S. R.; McCartney, D. L.; Marioni, R. E.\nScore: 9.3, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297200\nBackgroundPlasma growth differentiation factor 15 (GDF15) and N-terminal pro-B-type natriuretic peptide (NT-proBNP) are cardiovascular biomarkers that associate with a range of diseases. Epigenetic scores (EpiScores) for GDF15 and NT-proBNP may provide new routes for risk stratification. ResultsIn the Generation Scotland cohort (N [\u0026ge;] 16,963), GDF15 levels were associated with incident dementia, ischaemic stroke and type 2 diabetes, whereas NT-proBNP levels were associated with incident ischaemic heart disease, ischaemic stroke and type 2 diabetes (all PFDR \u003c 0.05). Bayesian Epigenome-wide association studies (EWAS) identified 12 and 4 DNA methylation (DNAm) CpG sites associated (Posterior Inclusion Probability [PIP] \u003e 95%) with levels of GDF15 and NT-proBNP, respectively. EpiScores for GDF15 and NT-proBNP that were trained in a subset of the population. The GDF15 EpiScore replicated protein associations with incident dementia, type 2 diabetes and ischaemic stroke in the Generation Scotland test set (Hazard Ratios (HR) range 1.36 - 1.41, PFDR \u003c0.03). The EpiScore for NT-proBNP replicated the protein association with type 2 diabetes, but failed to replicate an association with ischaemic stroke. EpiScores explained comparable variance in protein levels across both the Generation Scotland test set and the external LBC1936 test cohort (R2 range of 5.7-12.2%). In LBC1936, both EpiScores were associated with indicators of poorer brain health. Neither EpiScore was associated with incident dementia in the LBC1936 population. ConclusionsEpiScores for serum levels of GDF15 and Nt-proBNP associate with body and brain health traits. These EpiScores are provided as potential tools for disease risk stratification.\nUnderstanding and Predicting Polycystic Ovary Syndrome through Shared Genetics with Testosterone, SHBG, and Chronic Inflammation\nAuthors: Petersen, L. K.; Brixi, G.; Li, J.; Hu, J.; Wang, Z.; Han, X.; Meir, A. Y.; Tyrmi, J.; Mahalingaiah, S.; Piltonen, T.; Liang, L.\nScore: 5.5, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297115\nPolycystic ovary syndrome (PCOS) is a common hormonal disorder that affects one out of eight women and has high metabolic and psychological comorbidities. PCOS is thought to be associated with increased systemic low-grade inflammation, but the underlying mechanisms remain unclear. Here we study the genetic relationship between PCOS and obesity, testosterone, sex hormone binding globulin (SHBG), and chronic inflammation. First we create a large meta-analysis of PCOS (7,747 PCOS cases and 498,227 controls) and identify four novel genetic loci associated with PCOS. These novel loci have been previously associated with changes in gene expression in multiple tissues including the thyroid and ovary. We then analyze the PCOS meta analysis alongside GWASs for obesity (n=681,275), SHBG (n=190,366), testosterone (n=176,687), and meta-analyses of 130 inflammatory biomarkers (average n=30,000). We replicate potential causal relationships (via Mendelian randomization) from obesity and SHBG to PCOS and find extensive genetic correlations and causality between these traits and inflammatory biomarkers. We identify significant genetic correlations between PCOS and eight inflammatory biomarkers, including new relationships such as a strong correlation to death receptor 5 (LDSC rg = 0.54, FDR = 0.043). Although these results point to a shared genetic architecture between PCOS and inflammation, we did not find statistically significant causality between them; however, we did find 27 inflammatory biomarkers with significant causal effects on SHBG and four biomarkers with significant causal effects on testosterone. These results support the hypothesis that chronic inflammation can influence androgen and SHBG levels, though more research is needed to verify these results. Finally, we show that combining the polygenic risk scores of PCOS-related traits improves genetic prediction of PCOS cases in the UK Biobank and MGB Biobank compared to using only PCOS risk scores (0.72 AUC from 0.59 AUC and 0.61 AUC from 0.59 AUC in each biobank respectively).\nDisentangling the link between maternal influences on birth weight and disease risk in 36,211 genotyped mother-child pairs\nAuthors: Leinonen, J. T.; FinnGen, ; Pirinen, M.; Tukiainen, T.\nScore: 12.6, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297154\nEpidemiological studies have robustly linked lower birth weight to later-life disease risks. These observations may reflect the adverse impact of intrauterine growth restriction on a childs health. However, causal evidence supporting such a mechanism in humans is largely lacking. Using Mendelian Randomization and 36,211 genotyped mother-child pairs from the FinnGen study, we assessed the relationship between intrauterine growth restriction and five common health outcomes (coronary heart disease (CHD), hypertension, statin use, type 2 diabetes and cancer). We proxied intrauterine growth restriction with polygenic scores for maternal effects on birth weight and took into account transmission of genetic variants between a mother and a child in the analyses. We find limited evidence for contribution of normal variation in maternally influenced intrauterine growth on later-life disease. Instead, we find support for genetic pleiotropy in the child genome linking birth weight to CHD and hypertension. Our study illustrates the opportunities that data from genotyped parent-child pairs from a population-based biobank provides for addressing causality of maternal influences.\nMendelian randomization study of birth weight and risk of psychiatric disorders later in life\nAuthors: Wu, X.; Geller, F.; Helenius, D.; Grove, J.; Albinana, C.; Petersen, L. V.; Bulik, C.; Borglum, A.; Werge, T.; Vilhjalmsson, B. J.; Feenstra, B.\nScore: 5.0, Published: 2023-10-26 DOI: 10.1101/2023.10.26.23297618\nLow birth weight has been associated with a higher risk of psychiatric disorders later in life. The underlying causal mechanisms of this relationship are however not clear. In this study, we investigate whether variation in fetal growth has a direct causal effect on mental health. Using birth weight as a proxy measure for fetal growth, we first assessed associations between observed birth weight and later diagnosis of psychiatric disorders in the Danish iPSYCH and ANGI-DK cohorts. Next, we constructed a polygenic score for birth weight based on \u003e1 million variants and tested for association with psychiatric disorders. Finally, using 86 single-nucleotide polymorphisms with robust fetal-only genetic associations with birth weight, we assessed the causal relationship of genetically mediated fetal growth and psychiatric disorders using Mendelian randomization analyses. We found that higher observed birth weight was associated with lower risk of several psychiatric disorders. Polygenic score analyses supported this pattern for attention deficit/hyperactivity disorder, where an increase of one standard deviation in the score for birth weight corresponded to an odds ratio of 0.85 (95% confidence interval 0.79-0.92, P=6.27x10-5). However, one- and two-sample Mendelian randomization analyses did not indicate a direct causal relationship between the birth weight of children and their risk of psychiatric disorders. In conclusion, our study does not support a direct causal effect of fetal growth (as proxied by birth weight) on the risk of psychiatric disorders later in life, suggesting that the observed association between birth weight and mental health is likely to be caused by other factors.\nComprehensive reanalysis for CNVs in exome sequencing data from unsolved rare disease cases results in new diagnoses\nAuthors: Demidov, G.; Yaldiz, B.; Garcia-Pelaez, J.; de Boer, E.; Schuermans, N.; Van de Vondel, L.; Johansson, L. F.; Musacchia, F.; Paramonov, I.; Benetti, E.; Bullich, G.; Sablauskas, K.; Beltran, S.; Gilissen, C.; Hoischen, A.; Ossowski, S.; de Voer, R.; Lohmann, K.; Oliveira, C.; Topf, A.; Vissers, L. E. L. M.; Solve-RD Consortium, ; Laurie, S.\nScore: 4.5, Published: 2023-10-23 DOI: 10.1101/2023.10.22.23296993\nWe report the diagnostic results of a comprehensive copy number variant (CNV) reanalysis of 9,171 exome sequencing (ES) datasets from 5,757 families, including 6,143 individuals affected by a rare disease (RD). The data analysed was extremely heterogeneous, having been generated using 28 different exome enrichment kits, and sequenced on multiple short-read sequencing platforms, by 42 different research groups across Europe partnering in the Solve-RD project. Each of these research groups had previously undertaken their own analysis of the ES data but had failed to identify disease-causing variants. We applied three CNV calling algorithms to maximise sensitivity: ClinCNV, Conifer, and ExomeDepth. Rare CNVs overlapping genes of interest in custom lists provided by one of four partner European Reference Networks (ERN) were identified and taken forward for interpretation by clinical experts in RD. To facilitate interpretation, Integrative Genomics Viewer (IGV) screenshots incorporating a variety of custom-made tracks were generated for all prioritised CNVs. These analyses have resulted in a molecular diagnosis being provided for 51 families in this sample, with ClinCNV performing the best of the three algorithms in identifying disease-causing CNVs. We also identified pathogenic CNVs that are partially explanatory of the probands phenotype in a further 34 individuals. This work illustrates the value of reanalysing ES cold cases for CNVs even where analyses had been undertaken previously. Crucially, identification of these previously undetected CNVs has resulted in the conclusion of the diagnostic odyssey for these RD families, some of which had endured decades.\nLarge-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders\nAuthors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.; Gardham, A.; Ghali, N.; Green, A.; Hanna, W.; Harrison, R.; Hegarty, M.; Higgs, J.; Holder, M.; Irving, R.; Jain, V.; Johnson, K.; Jolley, R.; Jones, W.; Jones, G.; Joss, S.; Kalinauskiene, R.; Kanini, F.; Kavanagh, K.; Khan, M.; Khan, N.; Kivuva, E.; Lahiri, N.; L\nScore: 21.0, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297202\nObjectiveWe sought to evaluate outcomes for clinical management following a genetic diagnosis from the Deciphering Developmental Disorders (DDD) Study. DesignIndividuals in the DDD study who had a pathogenic/likely pathogenic genotype in the DECIPHER database were selected for inclusion (n=5010). Clinical notes from regional clinical genetics services notes were reviewed to assess pre-defined clinical outcomes relating to interventions, prenatal choices, and information provision. ResultsOutcomes were recorded for 4237 diagnosed probands (85% of those eligible) from all 24 recruiting centres across the UK and Ireland. Additional diagnostic or screening tests were performed in 903 (21%) probands through referral to a range of different clinical specialties, and stopped or avoided in a further 26 (0.6%). Disease-specific treatment was started in 85 (2%) probands, including seizure-control medications and dietary supplements, and contra-indicated medications were stopped/avoided as no longer necessary in a further 20 (0.5%). The option of prenatal/preimplantation genetic testing was discussed with 1204 (28%) families, despite the relatively advanced age of the parents at the time of diagnosis. Importantly, condition-specific information or literature was given to 3214 (76%) families, and 880 (21%) were involved in family support groups. In the most common condition (KBG syndrome; 79 (2%) probands), clinical interventions only partially reflected the temporal development of phenotypes, highlighting the importance of consensus management guidelines and patient support groups. ConclusionsOur results underscore the importance of achieving a clinico-molecular diagnosis to ensure timely onward referral of patients, enabling appropriate care and anticipatory surveillance, and for accessing relevant patient support groups.\n",
  "wordCount" : "2971",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.23297414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.23297414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.23297414">
        <p class="paperTitle">Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.23297414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.23297414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.</p>
        <p class="info">Score: 34.2, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.23297414' target='https://doi.org/10.1101/2023.10.23.23297414'> 10.1101/2023.10.23.23297414</a></p>
        <p class="abstract">BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS).

MethodWe used a dataset from 23andMe (6.77M European, 1.30M Latine, and 0.45M African American individuals). Using cross-sectional data for PRS construction and a prospective cohort for evaluation, we estimated PRS-associated cumulative incidences after one year of follow-up for 12 clinical endpoints.

ResultsThe cumulative incidence of disease at one year was consistently higher among individuals in the top 10% of each PRS. Relative risks (RRs) comparing top to bottom 10% of PRS differed across diseases (e.g. European RR 2.12 for eczema vs 12.53 for T2D). Estimates were similar between Europeans and Latines however were more modest for African Americans (e.g. T2D RR 10.92 for Latines vs. 4.00 for African Americans). Clinical manifestation occurred earlier among those in top vs bottom 10% of polygenic risk: 16yrs for hypertension, and 9.5yrs for T2D. Among participants at elevated conventional risk of CHD or T2D, those in the top 10% PRS had a 10-20 fold higher RR of disease incidence vs those not at conventional risk. Among individuals at high polygenic risk of CHD or T2D, favorable lifestyle characteristics associated with 64-73% lower RR of developing disease over 1-year, with cumulative incidence equivalent to the population average.

ConclusionIn an ancestrally-diverse cohort, individuals in the top 10% PRS had higher 1-year disease incidence and earlier age of clinical manifestation. PRS provided risk stratification beyond conventional risk factors. Lifestyle characteristics markedly lowered disease incidence among those at elevated polygenic risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.23297096">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.23297096" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.23297096">
        <p class="paperTitle">Decoding Genetics, Ancestry, and Geospatial Context for Precision Health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.23297096" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.23297096" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koyama, S.; Wang, Y.; Paruchuri, K.; Uddin, M. M.; Cho, S. M. J.; Urbut, S. M.; Haidermota, S.; Hornsby, W. E.; Green, R. C.; Daly, M. J.; Neale, B. M.; Ellinor, P. T.; Smoller, J. W.; Lebo, M. S.; Karlson, E. W.; Martin, A. R.; Natarajan, P.</p>
        <p class="info">Score: 32.2, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.23297096' target='https://doi.org/10.1101/2023.10.24.23297096'> 10.1101/2023.10.24.23297096</a></p>
        <p class="abstract">Mass General Brigham, an integrated healthcare system based in the Greater Boston area of Massachusetts, annually serves 1.5 million patients. We established the Mass General Brigham Biobank (MGBB), encompassing 142,238 participants, to unravel the intricate relationships among genomic profiles, environmental context, and disease manifestations within clinical practice. In this study, we highlight the impact of ancestral diversity in the MGBB by employing population genetics, geospatial assessment, and association analyses of rare and common genetic variants. The population structures captured by the genetics mirror the sequential immigration to the Greater Boston area throughout American history, highlighting communities tied to shared genetic and environmental factors. Our investigation underscores the potency of unbiased, large-scale analyses in a healthcare-affiliated biobank, elucidating the dynamic interplay across genetics, immigration, structural geospatial factors, and health outcomes in one of the earliest American sites of European colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297280">
        <p class="paperTitle">Determinants of mosaic chromosomal alteration fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pershad, Y.; Mack, T.; Poisner, H. M.; Jakubek, Y. A.; Stilp, A. M.; Mitchell, B. D.; Lewis, J. P.; Boerwinkle, E.; Loos, R. J.; Chami, N.; Wang, Z.; Barnes, K.; Pankratz, N.; Fornage, M.; Redline, S.; Psaty, B. M.; Bis, J. C.; Shojaie, A.; Silverman, E. K.; Cho, M. H.; Yun, J.; DeMeo, D.; Levy, D.; Johnson, A.; Mathias, R.; Taub, M.; Arnett, D.; North, K.; Raffield, L. M.; Carlson, A.; Doyle, M. F.; Rich, S. S.; Rotter, J. I.; Guo, X.; Cox, N.; Roden, D. M.; Franceschini, N.; Desai, P.; Reiner, A.; Auer, P. L.; Scheet, P.; Jaiswal, S.; Weinstock, J. S.; Bick, A. G.</p>
        <p class="info">Score: 14.8, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297280' target='https://doi.org/10.1101/2023.10.20.23297280'> 10.1101/2023.10.20.23297280</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal Expansion Rate (PACER) method to estimate clonal expansion rate for 6,381 individuals in the NHLBI TOPMed cohort with gain, loss, and copy-neutral loss of heterozygosity mCAs. Our estimates of mCA fitness were correlated (R2 = 0.49) with an alternative approach that estimated fitness of mCAs in the UK Biobank using a theoretical probability distribution. Individuals with lymphoid-associated mCAs had a significantly higher white blood cell count and faster clonal expansion rate. In a cross-sectional analysis, genome-wide association study of estimates of mCA expansion rate identified TCL1A, NRIP1, and TERT locus variants as modulators of mCA clonal expansion rate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.23297558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.23297558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.23297558">
        <p class="paperTitle">Plasticity of Human Microglia and Brain Perivascular Macrophages in Aging and Alzheimer&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.23297558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.23297558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, D.; Porras, C.; Spencer, C.; Pjanic, M.; Weiler, P.; Kosoy, R.; Bendl, J.; N.M., P.; Wang, X.; Zheng, S.; Therrien, K.; Mathur, D.; Kleopoulos, S. P.; Shao, Z.; Argyriou, S.; Alvia, M.; Casey, C.; Hong, A.; Beaumont, K. G.; Sebra, R.; Kellner, C. P.; Bennett, D. A.; Yuan, G.-C.; Voloudakis, G.; Theis, F. J.; Haroutunian, V.; Hoffman, G. E.; Fullard, J. F.; Roussos, P.</p>
        <p class="info">Score: 12.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.23297558' target='https://doi.org/10.1101/2023.10.25.23297558'> 10.1101/2023.10.25.23297558</a></p>
        <p class="abstract">Microglia and perivascular macrophages, myeloid-origin resident immune cells in the human brain, play crucial roles in Alzheimer&#39;s disease (AD)1-4. However, the field lacks a unified taxonomy describing their heterogeneity and plasticity5. To address this, we applied single-cell profiling to two independent, demographically diverse cohorts. The first comprises 543,012 viable myeloid cells from 137 unique postmortem brain specimens, while the second consists of 289,493 myeloid nuclei from 1,470 donors. Collectively, they cover the human lifespan and varying degrees of AD neuropathology. We identify 13 transcriptionally distinct myeloid subtypes, including the &#34;GPNMB&#34; subtype that proliferates with AD. We distinguish two contrasting homeostatic microglial states in AD and with aging: the first (&#34;FRMD4A&#34;) wanes over time, while the second (&#34;PICALM&#34;) becomes more prevalent. By prioritizing AD-risk genes, including PTPRG, DPYD, and IL15, and placing them into a regulatory hierarchy, we identify common upstream transcriptional regulators, namely MITF and KLF12, that regulate the expression of AD-risk genes in the opposite directions. Through the construction of cell-to-cell interaction networks, we identify candidate ligand-receptor pairs, including APOE:SORL1 and APOE:TREM2, associated with AD progression. We show polygenic risk for AD predisposes and prioritize the GPNMB subtype as a therapeutic target of early intervention. Our findings delineate the relationship between distinct functional states of myeloid cells and their pathophysiological response to aging and AD, providing a significant step toward the mechanistic understanding of the roles of microglia in AD and the identification of novel therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297200">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297200" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297200">
        <p class="paperTitle">DNAm scores for serum GDF15 and NT-proBNP levels associate with a range of traits affecting the body and brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297200" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297200" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gadd, D. A.; Smith, H.; Mullin, D.; Chybowska, A.; Hillary, R. F.; Kimenai, D. M.; Bernabeu, E.; Cheng, Y.; Fawns-Ritchie, C.; Campbell, A.; Page, D.; Taylor, A.; Corley, J.; Valdes Hernandez, M. D. C.; Munoz Maniega, S.; Bastin, M. E.; Wardlaw, J. M.; Walker, R. M.; Evans, K. L.; McIntosh, A. M.; Hayward, C.; Russ, T. C.; Harris, S. E.; Welsh, P.; Sattar, N.; Cox, S. R.; McCartney, D. L.; Marioni, R. E.</p>
        <p class="info">Score: 9.3, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297200' target='https://doi.org/10.1101/2023.10.18.23297200'> 10.1101/2023.10.18.23297200</a></p>
        <p class="abstract">BackgroundPlasma growth differentiation factor 15 (GDF15) and N-terminal pro-B-type natriuretic peptide (NT-proBNP) are cardiovascular biomarkers that associate with a range of diseases. Epigenetic scores (EpiScores) for GDF15 and NT-proBNP may provide new routes for risk stratification.

ResultsIn the Generation Scotland cohort (N [&amp;ge;] 16,963), GDF15 levels were associated with incident dementia, ischaemic stroke and type 2 diabetes, whereas NT-proBNP levels were associated with incident ischaemic heart disease, ischaemic stroke and type 2 diabetes (all PFDR &lt; 0.05). Bayesian Epigenome-wide association studies (EWAS) identified 12 and 4 DNA methylation (DNAm) CpG sites associated (Posterior Inclusion Probability [PIP] &gt; 95%) with levels of GDF15 and NT-proBNP, respectively. EpiScores for GDF15 and NT-proBNP that were trained in a subset of the population. The GDF15 EpiScore replicated protein associations with incident dementia, type 2 diabetes and ischaemic stroke in the Generation Scotland test set (Hazard Ratios (HR) range 1.36 - 1.41, PFDR &lt;0.03). The EpiScore for NT-proBNP replicated the protein association with type 2 diabetes, but failed to replicate an association with ischaemic stroke. EpiScores explained comparable variance in protein levels across both the Generation Scotland test set and the external LBC1936 test cohort (R2 range of 5.7-12.2%). In LBC1936, both EpiScores were associated with indicators of poorer brain health. Neither EpiScore was associated with incident dementia in the LBC1936 population.

ConclusionsEpiScores for serum levels of GDF15 and Nt-proBNP associate with body and brain health traits. These EpiScores are provided as potential tools for disease risk stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297115" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297115">
        <p class="paperTitle">Understanding and Predicting Polycystic Ovary Syndrome through Shared Genetics with Testosterone, SHBG, and Chronic Inflammation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petersen, L. K.; Brixi, G.; Li, J.; Hu, J.; Wang, Z.; Han, X.; Meir, A. Y.; Tyrmi, J.; Mahalingaiah, S.; Piltonen, T.; Liang, L.</p>
        <p class="info">Score: 5.5, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297115' target='https://doi.org/10.1101/2023.10.17.23297115'> 10.1101/2023.10.17.23297115</a></p>
        <p class="abstract">Polycystic ovary syndrome (PCOS) is a common hormonal disorder that affects one out of eight women and has high metabolic and psychological comorbidities. PCOS is thought to be associated with increased systemic low-grade inflammation, but the underlying mechanisms remain unclear. Here we study the genetic relationship between PCOS and obesity, testosterone, sex hormone binding globulin (SHBG), and chronic inflammation. First we create a large meta-analysis of PCOS (7,747 PCOS cases and 498,227 controls) and identify four novel genetic loci associated with PCOS. These novel loci have been previously associated with changes in gene expression in multiple tissues including the thyroid and ovary. We then analyze the PCOS meta analysis alongside GWASs for obesity (n=681,275), SHBG (n=190,366), testosterone (n=176,687), and meta-analyses of 130 inflammatory biomarkers (average n=30,000). We replicate potential causal relationships (via Mendelian randomization) from obesity and SHBG to PCOS and find extensive genetic correlations and causality between these traits and inflammatory biomarkers. We identify significant genetic correlations between PCOS and eight inflammatory biomarkers, including new relationships such as a strong correlation to death receptor 5 (LDSC rg = 0.54, FDR = 0.043). Although these results point to a shared genetic architecture between PCOS and inflammation, we did not find statistically significant causality between them; however, we did find 27 inflammatory biomarkers with significant causal effects on SHBG and four biomarkers with significant causal effects on testosterone. These results support the hypothesis that chronic inflammation can influence androgen and SHBG levels, though more research is needed to verify these results. Finally, we show that combining the polygenic risk scores of PCOS-related traits improves genetic prediction of PCOS cases in the UK Biobank and MGB Biobank compared to using only PCOS risk scores (0.72 AUC from 0.59 AUC and 0.61 AUC from 0.59 AUC in each biobank respectively).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297154">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297154" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297154">
        <p class="paperTitle">Disentangling the link between maternal influences on birth weight and disease risk in 36,211 genotyped mother-child pairs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297154" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297154" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leinonen, J. T.; FinnGen,  ; Pirinen, M.; Tukiainen, T.</p>
        <p class="info">Score: 12.6, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297154' target='https://doi.org/10.1101/2023.10.17.23297154'> 10.1101/2023.10.17.23297154</a></p>
        <p class="abstract">Epidemiological studies have robustly linked lower birth weight to later-life disease risks. These observations may reflect the adverse impact of intrauterine growth restriction on a childs health. However, causal evidence supporting such a mechanism in humans is largely lacking.

Using Mendelian Randomization and 36,211 genotyped mother-child pairs from the FinnGen study, we assessed the relationship between intrauterine growth restriction and five common health outcomes (coronary heart disease (CHD), hypertension, statin use, type 2 diabetes and cancer). We proxied intrauterine growth restriction with polygenic scores for maternal effects on birth weight and took into account transmission of genetic variants between a mother and a child in the analyses.

We find limited evidence for contribution of normal variation in maternally influenced intrauterine growth on later-life disease. Instead, we find support for genetic pleiotropy in the child genome linking birth weight to CHD and hypertension. Our study illustrates the opportunities that data from genotyped parent-child pairs from a population-based biobank provides for addressing causality of maternal influences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.23297618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.23297618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.23297618">
        <p class="paperTitle">Mendelian randomization study of birth weight and risk of psychiatric disorders later in life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.23297618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.23297618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, X.; Geller, F.; Helenius, D.; Grove, J.; Albinana, C.; Petersen, L. V.; Bulik, C.; Borglum, A.; Werge, T.; Vilhjalmsson, B. J.; Feenstra, B.</p>
        <p class="info">Score: 5.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.23297618' target='https://doi.org/10.1101/2023.10.26.23297618'> 10.1101/2023.10.26.23297618</a></p>
        <p class="abstract">Low birth weight has been associated with a higher risk of psychiatric disorders later in life. The underlying causal mechanisms of this relationship are however not clear. In this study, we investigate whether variation in fetal growth has a direct causal effect on mental health. Using birth weight as a proxy measure for fetal growth, we first assessed associations between observed birth weight and later diagnosis of psychiatric disorders in the Danish iPSYCH and ANGI-DK cohorts. Next, we constructed a polygenic score for birth weight based on &gt;1 million variants and tested for association with psychiatric disorders. Finally, using 86 single-nucleotide polymorphisms with robust fetal-only genetic associations with birth weight, we assessed the causal relationship of genetically mediated fetal growth and psychiatric disorders using Mendelian randomization analyses. We found that higher observed birth weight was associated with lower risk of several psychiatric disorders. Polygenic score analyses supported this pattern for attention deficit/hyperactivity disorder, where an increase of one standard deviation in the score for birth weight corresponded to an odds ratio of 0.85 (95% confidence interval 0.79-0.92, P=6.27x10-5). However, one- and two-sample Mendelian randomization analyses did not indicate a direct causal relationship between the birth weight of children and their risk of psychiatric disorders. In conclusion, our study does not support a direct causal effect of fetal growth (as proxied by birth weight) on the risk of psychiatric disorders later in life, suggesting that the observed association between birth weight and mental health is likely to be caused by other factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.23296993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.23296993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.23296993">
        <p class="paperTitle">Comprehensive reanalysis for CNVs in exome sequencing data from unsolved rare disease cases results in new diagnoses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.23296993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.23296993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demidov, G.; Yaldiz, B.; Garcia-Pelaez, J.; de Boer, E.; Schuermans, N.; Van de Vondel, L.; Johansson, L. F.; Musacchia, F.; Paramonov, I.; Benetti, E.; Bullich, G.; Sablauskas, K.; Beltran, S.; Gilissen, C.; Hoischen, A.; Ossowski, S.; de Voer, R.; Lohmann, K.; Oliveira, C.; Topf, A.; Vissers, L. E. L. M.; Solve-RD Consortium,  ; Laurie, S.</p>
        <p class="info">Score: 4.5, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.23296993' target='https://doi.org/10.1101/2023.10.22.23296993'> 10.1101/2023.10.22.23296993</a></p>
        <p class="abstract">We report the diagnostic results of a comprehensive copy number variant (CNV) reanalysis of 9,171 exome sequencing (ES) datasets from 5,757 families, including 6,143 individuals affected by a rare disease (RD). The data analysed was extremely heterogeneous, having been generated using 28 different exome enrichment kits, and sequenced on multiple short-read sequencing platforms, by 42 different research groups across Europe partnering in the Solve-RD project. Each of these research groups had previously undertaken their own analysis of the ES data but had failed to identify disease-causing variants.

We applied three CNV calling algorithms to maximise sensitivity: ClinCNV, Conifer, and ExomeDepth. Rare CNVs overlapping genes of interest in custom lists provided by one of four partner European Reference Networks (ERN) were identified and taken forward for interpretation by clinical experts in RD. To facilitate interpretation, Integrative Genomics Viewer (IGV) screenshots incorporating a variety of custom-made tracks were generated for all prioritised CNVs.

These analyses have resulted in a molecular diagnosis being provided for 51 families in this sample, with ClinCNV performing the best of the three algorithms in identifying disease-causing CNVs. We also identified pathogenic CNVs that are partially explanatory of the probands phenotype in a further 34 individuals. This work illustrates the value of reanalysing ES cold cases for CNVs even where analyses had been undertaken previously. Crucially, identification of these previously undetected CNVs has resulted in the conclusion of the diagnostic odyssey for these RD families, some of which had endured decades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297202">
        <p class="paperTitle">Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.; Gardham, A.; Ghali, N.; Green, A.; Hanna, W.; Harrison, R.; Hegarty, M.; Higgs, J.; Holder, M.; Irving, R.; Jain, V.; Johnson, K.; Jolley, R.; Jones, W.; Jones, G.; Joss, S.; Kalinauskiene, R.; Kanini, F.; Kavanagh, K.; Khan, M.; Khan, N.; Kivuva, E.; Lahiri, N.; L</p>
        <p class="info">Score: 21.0, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297202' target='https://doi.org/10.1101/2023.10.18.23297202'> 10.1101/2023.10.18.23297202</a></p>
        <p class="abstract">ObjectiveWe sought to evaluate outcomes for clinical management following a genetic diagnosis from the Deciphering Developmental Disorders (DDD) Study.

DesignIndividuals in the DDD study who had a pathogenic/likely pathogenic genotype in the DECIPHER database were selected for inclusion (n=5010). Clinical notes from regional clinical genetics services notes were reviewed to assess pre-defined clinical outcomes relating to interventions, prenatal choices, and information provision.

ResultsOutcomes were recorded for 4237 diagnosed probands (85% of those eligible) from all 24 recruiting centres across the UK and Ireland. Additional diagnostic or screening tests were performed in 903 (21%) probands through referral to a range of different clinical specialties, and stopped or avoided in a further 26 (0.6%). Disease-specific treatment was started in 85 (2%) probands, including seizure-control medications and dietary supplements, and contra-indicated medications were stopped/avoided as no longer necessary in a further 20 (0.5%).

The option of prenatal/preimplantation genetic testing was discussed with 1204 (28%) families, despite the relatively advanced age of the parents at the time of diagnosis.

Importantly, condition-specific information or literature was given to 3214 (76%) families, and 880 (21%) were involved in family support groups. In the most common condition (KBG syndrome; 79 (2%) probands), clinical interventions only partially reflected the temporal development of phenotypes, highlighting the importance of consensus management guidelines and patient support groups.

ConclusionsOur results underscore the importance of achieving a clinico-molecular diagnosis to ensure timely onward referral of patients, enabling appropriate care and anticipatory surveillance, and for accessing relevant patient support groups.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
