<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CXCL12 regulates coronary artery dominance in diverse populations and links development to disease
Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.
Score: 28.7, Published: 2023-10-27 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="CXCL12 regulates coronary artery dominance in diverse populations and links development to disease
Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.
Score: 28.7, Published: 2023-10-27 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="CXCL12 regulates coronary artery dominance in diverse populations and links development to disease
Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.
Score: 28.7, Published: 2023-10-27 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "CXCL12 regulates coronary artery dominance in diverse populations and links development to disease\nAuthors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.\nScore: 28.7, Published: 2023-10-27 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " CXCL12 regulates coronary artery dominance in diverse populations and links development to disease\nAuthors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.\nScore: 28.7, Published: 2023-10-27 DOI: 10.1101/2023.10.27.23297507\nMammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.\nClustering of predicted loss-of-function variants in genes linked with monogenic disease can explain incomplete penetrance\nAuthors: Beaumont, R. N.; Hawkes, G.; Gunning, A. C.; Wright, C.\nScore: 20.8, Published: 2023-10-12 DOI: 10.1101/2023.10.11.23296535\nPredicted loss-of-function variants (pLoFs) are often associated with disease. For genes linked with monogenic diseases, we hypothesised that pLoFs present in apparently unaffected individuals may cluster in LoF-tolerant regions. We compared the distribution of pLoFs in ClinVar versus 454,773 individuals in UK Biobank and clustered the variants using Gaussian mixture models. We found that genes in which haploinsufficiency causes developmental disorders with incomplete penetrance were less likely to have a uniform pLoF distribution than other genes (P\u003c2.2x10-6). In some cases (e.g., ARID1B and GATA6), pLoF variants in the first quarter of the gene could be rescued by an alternative translation start site and should not be reported as pathogenic. In other cases (e.g., ODC1), pathogenic pLoFs were clustered only at the end of the gene, consistent with a gain-of-function disease mechanism. Our results support the use of localised constraint metrics when interpreting variants.\nFine-mapping causal tissues and genes at disease-associated loci\nAuthors: Strober, B. J.; Zhang, M. J.; Amariuta, T.; Rossen, J.; Price, A. L.\nScore: 15.4, Published: 2023-11-01 DOI: 10.1101/2023.11.01.23297909\nHeritable diseases often manifest in a highly tissue-specific manner, with different disease loci mediated by genes in distinct tissues or cell types. We propose Tissue-Gene Fine-Mapping (TGFM), a fine-mapping method that infers the posterior probability (PIP) for each gene-tissue pair to mediate a disease locus by analyzing GWAS summary statistics (and in-sample LD) and leveraging eQTL data from diverse tissues to build cis-predicted expression models; TGFM also assigns PIPs to causal variants that are not mediated by gene expression in assayed genes and tissues. TGFM accounts for both co-regulation across genes and tissues and LD between SNPs (generalizing existing fine-mapping methods), and incorporates genome-wide estimates of each tissues contribution to disease as tissue-level priors. TGFM was well-calibrated and moderately well-powered in simulations; unlike previous methods, TGFM was able to attain correct calibration by modeling uncertainty in cis-predicted expression models. We applied TGFM to 45 UK Biobank diseases/traits (average N = 316K) using eQTL data from 38 GTEx tissues. TGFM identified an average of 147 PIP \u003e 0.5 causal genetic elements per disease/trait, of which 11% were gene-tissue pairs. Implicated gene-tissue pairs were concentrated in known disease-critical tissues, and causal genes were strongly enriched in disease-relevant gene sets. Causal gene-tissue pairs identified by TGFM recapitulated known biology (e.g., TPO-thyroid for Hypothyroidism), but also included biologically plausible novel findings (e.g., SLC20A2-artery aorta for Diastolic blood pressure). Further application of TGFM to single-cell eQTL data from 9 cell types in peripheral blood mononuclear cells (PBMC), analyzed jointly with GTEx tissues, identified 30 additional causal gene-PBMC cell type pairs at PIP \u003e 0.5--primarily for autoimmune disease and blood cell traits, including the well-established role of CTLA4 in CD8+ T cells for All autoimmune disease. In conclusion, TGFM is a robust and powerful method for fine-mapping causal tissues and genes at disease-associated loci.\nA Massively Parallel Screen of 5'UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes\nAuthors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.\nScore: 12.7, Published: 2023-11-03 DOI: 10.1101/2023.11.02.23297961\nDe novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5'UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5'UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5'UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.\nEvaluating the cost-effectiveness of polygenic risk score-stratified screening for abdominal aortic aneurysm\nAuthors: Kelemen, M.; Danesh, J.; Di Angelantonio, E.; Inouye, M.; O'Sullivan, J. W.; Pennells, L.; Wood, A.; Sweeting, M.; Harrison, S.; Kim, L.\nScore: 8.2, Published: 2023-11-06 DOI: 10.1101/2023.11.02.23297906\nAs the heritability of abdominal aortic aneurysm (AAA) is high and AAA partially shares genetic architecture with other cardiovascular diseases, there is interest in whether genetic information could inform AAA screening strategies. Exploiting pleiotropy and meta-analysing summary data from large AAA studies, we constructed a polygenic risk score (PRS) for AAA. Compared with the low PRS tertile, the intermediate and high PRS tertiles had hazard ratios for AAA of 2.13 (95%CI 1.61, 2.82) and 3.70 (95%CI 2.86, 4.80) respectively, after adjusting for known clinical risk factors. Using simulation modelling, we compared PRS- and smoking-stratified screening with inviting men at age 65 and not inviting women (current UK strategy). In a futuristic scenario where genomic information is available on the population, our modelling suggests inviting male current smokers with high PRS earlier than 65 and targeting screening in female smokers with high/intermediate PRS at 65 and 70 respectively, may improve cost-effectiveness.\nEvaluating the impact of modeling choices on the performance of integrated genetic and clinical models\nAuthors: Morley, T. J.; Willimitis, D. J.; Ripperger, M. A.; Lee, H.; Han, L.; Zhou, Y.; Kang, J.; Davis, L. K.; Smoller, J. W.; Choi, K. W.; Walsh, C. G.; Ruderfer, D. M.\nScore: 5.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.23297927\nThe value of genetic information for improving the performance of clinical risk prediction models has yielded variable conclusions. Many methodological decisions have the potential to contribute to differential results across studies. Here, we performed multiple modeling experiments integrating clinical and demographic data from electronic health records (EHR) and genetic data to understand which decision points may affect performance. Clinical data in the form of structured diagnostic codes, medications, procedural codes, and demographics were extracted from two large independent health systems and polygenic risk scores (PRS) were generated across all patients with genetic data in the corresponding biobanks. Crohns disease was used as the model phenotype based on its substantial genetic component, established EHR-based definition, and sufficient prevalence for model training and testing. We investigated the impact of PRS integration method, as well as choices regarding training sample, model complexity, and performance metrics. Overall, our results show that including PRS resulted in higher performance by some metrics but the gain in performance was only robust when combined with demographic data alone. Improvements were inconsistent or negligible after including additional clinical information. The impact of genetic information on performance also varied by PRS integration method, with a small improvement in some cases from combining PRS with the output of a clinical model (late-fusion) compared to its inclusion an additional feature (early-fusion). The effects of other modeling decisions varied between institutions though performance increased with more compute-intensive models such as random forest. This work highlights the importance of considering methodological decision points in interpreting the impact on prediction performance when including PRS information in clinical models.\nA polygenic risk score identifies undiagnosed cases of diabetes\nAuthors: German, C.; Ashenhurst, J.; Wang, W.; 23andMe Research Team, ; Granka, J. M.; Koelsch, B. L.; Abul-Husn, N. S.; Aslibekyan, S.; Auton, A.; Tung, J.; Shringarpure, S. S.; Holmes, M. V.\nScore: 3.9, Published: 2023-11-01 DOI: 10.1101/2023.10.31.23297847\nImportanceTwenty-three percent of 37.3M adults in the USA with diabetes are estimated to be undiagnosed, leading to potentially avoidable sequelae and morbidity. ObjectiveTo explore the utility of a polygenic risk score (PRS) at identifying individuals with undiagnosed diabetes and prediabetes. Design, Setting and ParticipantsIndividuals without doctor-diagnosed diabetes at study baseline in the UK Biobank (UKB) with HbA1c and BMI measurements. Participants were restricted to white individuals to use an ancestry-appropriate PRS. Undiagnosed diabetes and prediabetes were defined using HbA1c ([\u0026ge;]6.5% and [\u0026ge;]5.7 - \u003c6.5%, respectively). ExposuresA diabetes PRS comprising 13,863 SNPs derived from the 23andMe Research Cohort, and measured BMI among UKB participants. ResultsOf 412,439 individuals self-reporting an absence of diagnosed diabetes and who had BMI and HbA1c measurements at baseline, 2,934 (0.7%) had undiagnosed diabetes, representing 11.9% of all (diagnosed and undiagnosed) diabetes. Nearly half (1,362, 46%) of undiagnosed diabetes cases were among individuals in the top 25% of the PRS distribution. Overweight individuals (BMI [\u0026ge;]25 - \u003c30 kg/m2) who were in the top 12.5% of the PRS distribution had a similar frequency of undiagnosed diabetes (0.8-1.6% frequency) as individuals with obesity (BMI [\u0026ge;]30kg/m2) in the lowest 12.5% of the PRS distribution (0.7-1.7% frequency). Combining overweight and obesity with the PRS identified nearly all cases of undiagnosed diabetes: individuals with a BMI [\u0026ge;]25 kg/m2 (66% of the study population) or those in the top 54-69% of the PRS identified 98-99% of undiagnosed cases. Of the 199 undiagnosed diabetes cases occurring among individuals with a normal BMI (\u003c25kg/m2), two-thirds were among individuals in the top 50% of the PRS. Prediabetes was common (14%), with measured BMI and PRS providing additive risk. Among those in the top 12.5% PRS with BMI [\u0026ge;]35kg/m2, 6.3% developed incident diabetes over 4 years follow-up, as compared to 0% among the bottom 12.5% PRS with BMI\u003c25kg/m2. ConclusionsA diabetes PRS is informative at identifying undiagnosed cases. PRS may have broader utility in detecting individuals with asymptomatic disease. Key PointsO_ST_ABSQuestionC_ST_ABSDoes a polygenic risk score (PRS) have utility in identifying individuals with undiagnosed type 2 diabetes (T2D)? FindingsIn this analysis of 412,439 individuals without doctor-diagnosed diabetes, a T2D PRS performed additively to body mass index (BMI) at identifying individuals with undiagnosed diabetes. Selecting individuals on the basis of overweight/obesity or a T2D PRS identified almost all cases of undiagnosed diabetes. The majority of undiagnosed diabetes cases among individuals with normal weight occurred among those at elevated polygenic risk. MeaningA T2D PRS identifies cases of undiagnosed diabetes among individuals with and without overweight or obesity.\nDiagnostic Utility of Genome-wide DNA Methylation Analysis in Genetically Unsolved Developmental and Epileptic Encephalopathies and Refinement of a CHD2 Episignature\nAuthors: LaFlamme, C. W.; Rastin, C.; Sengupta, S.; Pennington, H. E.; Russ-Hall, S. J.; Schneider, A. L.; Bonkowski, E. S.; Almanza Fuerte, E. P.; Galey, M.; Goffena, J.; Gibson, S. B.; Allan, T. J.; Nyaga, D. M.; Lieffering, N.; Hebbar, M.; Walker, E. V.; Darnell, D.; Olsen, S. R.; Kolekar, P.; Djekidel, M. N.; Rosikiewicz, W.; McConkey, H.; Kerkhof, J.; Levy, M. A.; Relator, R.; Lev, D.; Lerman-Sagie, T.; Park, K. L.; Alders, M.; Cappuccio, G.; Chatron, N.; Demain, L.; Genevieve, D.; Lesca, G.; Roscioli, T.; Sanlaville, D.; Tedder, M. L.; Weisz-Hubshman, M.; Ketkar, S.; Dai, H.; Carlyle Worley, K.; Score: 13.0, Published: 2023-10-13 DOI: 10.1101/2023.10.11.23296741\nSequence-based genetic testing currently identifies causative genetic variants in [~]50% of individuals with developmental and epileptic encephalopathies (DEEs). Aberrant changes in DNA methylation are implicated in various neurodevelopmental disorders but remain unstudied in DEEs. Rare epigenetic variations (\"epivariants\") can drive disease by modulating gene expression at single loci, whereas genome-wide DNA methylation changes can result in distinct \"episignature\" biomarkers for monogenic disorders in a growing number of rare diseases. Here, we interrogate the diagnostic utility of genome-wide DNA methylation array analysis on peripheral blood samples from 516 individuals with genetically unsolved DEEs who had previously undergone extensive genetic testing. We identified rare differentially methylated regions (DMRs) and explanatory episignatures to discover causative and candidate genetic etiologies in 10 individuals. We then used long-read sequencing to identify DNA variants underlying rare DMRs, including one balanced translocation, three CG-rich repeat expansions, and two copy number variants. We also identify pathogenic sequence variants associated with episignatures; some had been missed by previous exome sequencing. Although most DEE genes lack known episignatures, the increase in diagnostic yield for DNA methylation analysis in DEEs is comparable to the added yield of genome sequencing. Finally, we refine an episignature for CHD2 using an 850K methylation array which was further refined at higher CpG resolution using bisulfite sequencing to investigate potential insights into CHD2 pathophysiology. Our study demonstrates the diagnostic yield of genome-wide DNA methylation analysis to identify causal and candidate genetic causes as [~]2% (10/516) for unsolved DEE cases.\nGenome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves cross-ancestry prediction\nAuthors: Hoffmann, T. J.; Graff, R. E.; Madduri, R. K.; Rodriguez, A. A.; Cario, C. L.; Feng, K.; Jiang, Y.; Wang, A.; Klein, R. J.; Pierce, B. L.; Eggener, S.; Tong, L.; Blot, W.; Long, J.; Rebbeck, T.; Lachance, J.; Andrews, C.; Adebiyi, A. O.; Adusei, B.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Jalloh, M.; Janivara, R.; Chen, W. C.; Mensah, J. E.; Agalliu, I.; Berndt, S. I.; Shelley, J. P.; Schaffer, K.; Machiela, M. J.; Freedman, N. D.; Huang, W.-Y.; Li, S. A.; Goodman, P. J.; Till, C.; Thompson, I.; Lilja, H.; Van Den Eeden, S. K.; Chanock, S. J.; Mosley, J. D.; Conti, D. V.; Haiman, C. A.;\nScore: 7.0, Published: 2023-10-30 DOI: 10.1101/2023.10.27.23297676\nWe conducted a multi-ancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry; 58,236 African ancestry; 23,546 Hispanic/Latino; 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (p[\u0026le;]5e-8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n=95,768). Meta-analyzing discovery and replication (n=392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our new polygenic risk score reached 16.9% (95% CI=16.1%-17.8%) in European ancestry, 9.5% (95% CI=7.0%-12.2%) in African ancestry, 18.6% (95% CI=15.8%-21.4%) in Hispanic/Latino, and 15.3% (95% CI=12.7%-18.1%) in Asian ancestry, and lower for higher age. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, with potential to personalize prostate cancer screening.\nThe qMini assay identifies an overlooked class of splice variants\nAuthors: Guan, B.; Bender, C.; Pantrangi, M.; Moore, N.; Reeves, M.; Naik, A.; Li, H.; Goetz, K.; Blain, D.; Agather, A.; Cukras, C.; Zein, W. M.; Huryn, L. A.; Brooks, B. P.; Hufnagel, R. B.\nScore: 2.2, Published: 2023-11-03 DOI: 10.1101/2023.11.02.23297963\nSplice variants are known to cause diseases by utilizing alternative splice sites, potentially resulting in protein truncation or mRNA degradation by nonsense-mediated decay. Splice variants are verified when altered mature mRNA sequences are identified in RNA analyses or minigene assays. Using a quantitative minigene assay, qMini, we uncovered a previously overlooked class of disease-associated splice variants that did not alter mRNA sequence but decreased mature mRNA level, suggesting a potentially new pathogenic mechanism.\n",
  "wordCount" : "2803",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297507">
        <p class="paperTitle">CXCL12 regulates coronary artery dominance in diverse populations and links development to disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.</p>
        <p class="info">Score: 28.7, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297507' target='https://doi.org/10.1101/2023.10.27.23297507'> 10.1101/2023.10.27.23297507</a></p>
        <p class="abstract">Mammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.23296535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.23296535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.23296535">
        <p class="paperTitle">Clustering of predicted loss-of-function variants in genes linked with monogenic disease can explain incomplete penetrance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.23296535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.23296535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beaumont, R. N.; Hawkes, G.; Gunning, A. C.; Wright, C.</p>
        <p class="info">Score: 20.8, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.23296535' target='https://doi.org/10.1101/2023.10.11.23296535'> 10.1101/2023.10.11.23296535</a></p>
        <p class="abstract">Predicted loss-of-function variants (pLoFs) are often associated with disease. For genes linked with monogenic diseases, we hypothesised that pLoFs present in apparently unaffected individuals may cluster in LoF-tolerant regions. We compared the distribution of pLoFs in ClinVar versus 454,773 individuals in UK Biobank and clustered the variants using Gaussian mixture models. We found that genes in which haploinsufficiency causes developmental disorders with incomplete penetrance were less likely to have a uniform pLoF distribution than other genes (P&lt;2.2x10-6). In some cases (e.g., ARID1B and GATA6), pLoF variants in the first quarter of the gene could be rescued by an alternative translation start site and should not be reported as pathogenic. In other cases (e.g., ODC1), pathogenic pLoFs were clustered only at the end of the gene, consistent with a gain-of-function disease mechanism. Our results support the use of localised constraint metrics when interpreting variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.23297909">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.23297909" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.23297909">
        <p class="paperTitle">Fine-mapping causal tissues and genes at disease-associated loci</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.23297909" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.23297909" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strober, B. J.; Zhang, M. J.; Amariuta, T.; Rossen, J.; Price, A. L.</p>
        <p class="info">Score: 15.4, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.23297909' target='https://doi.org/10.1101/2023.11.01.23297909'> 10.1101/2023.11.01.23297909</a></p>
        <p class="abstract">Heritable diseases often manifest in a highly tissue-specific manner, with different disease loci mediated by genes in distinct tissues or cell types. We propose Tissue-Gene Fine-Mapping (TGFM), a fine-mapping method that infers the posterior probability (PIP) for each gene-tissue pair to mediate a disease locus by analyzing GWAS summary statistics (and in-sample LD) and leveraging eQTL data from diverse tissues to build cis-predicted expression models; TGFM also assigns PIPs to causal variants that are not mediated by gene expression in assayed genes and tissues. TGFM accounts for both co-regulation across genes and tissues and LD between SNPs (generalizing existing fine-mapping methods), and incorporates genome-wide estimates of each tissues contribution to disease as tissue-level priors. TGFM was well-calibrated and moderately well-powered in simulations; unlike previous methods, TGFM was able to attain correct calibration by modeling uncertainty in cis-predicted expression models. We applied TGFM to 45 UK Biobank diseases/traits (average N = 316K) using eQTL data from 38 GTEx tissues. TGFM identified an average of 147 PIP &gt; 0.5 causal genetic elements per disease/trait, of which 11% were gene-tissue pairs. Implicated gene-tissue pairs were concentrated in known disease-critical tissues, and causal genes were strongly enriched in disease-relevant gene sets. Causal gene-tissue pairs identified by TGFM recapitulated known biology (e.g., TPO-thyroid for Hypothyroidism), but also included biologically plausible novel findings (e.g., SLC20A2-artery aorta for Diastolic blood pressure). Further application of TGFM to single-cell eQTL data from 9 cell types in peripheral blood mononuclear cells (PBMC), analyzed jointly with GTEx tissues, identified 30 additional causal gene-PBMC cell type pairs at PIP &gt; 0.5--primarily for autoimmune disease and blood cell traits, including the well-established role of CTLA4 in CD8&#43; T cells for All autoimmune disease. In conclusion, TGFM is a robust and powerful method for fine-mapping causal tissues and genes at disease-associated loci.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297961">
        <p class="paperTitle">A Massively Parallel Screen of 5&#39;UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.</p>
        <p class="info">Score: 12.7, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297961' target='https://doi.org/10.1101/2023.11.02.23297961'> 10.1101/2023.11.02.23297961</a></p>
        <p class="abstract">De novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5&#39;UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5&#39;UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5&#39;UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297906">
        <p class="paperTitle">Evaluating the cost-effectiveness of polygenic risk score-stratified screening for abdominal aortic aneurysm</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelemen, M.; Danesh, J.; Di Angelantonio, E.; Inouye, M.; O&#39;Sullivan, J. W.; Pennells, L.; Wood, A.; Sweeting, M.; Harrison, S.; Kim, L.</p>
        <p class="info">Score: 8.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297906' target='https://doi.org/10.1101/2023.11.02.23297906'> 10.1101/2023.11.02.23297906</a></p>
        <p class="abstract">As the heritability of abdominal aortic aneurysm (AAA) is high and AAA partially shares genetic architecture with other cardiovascular diseases, there is interest in whether genetic information could inform AAA screening strategies. Exploiting pleiotropy and meta-analysing summary data from large AAA studies, we constructed a polygenic risk score (PRS) for AAA. Compared with the low PRS tertile, the intermediate and high PRS tertiles had hazard ratios for AAA of 2.13 (95%CI 1.61, 2.82) and 3.70 (95%CI 2.86, 4.80) respectively, after adjusting for known clinical risk factors. Using simulation modelling, we compared PRS- and smoking-stratified screening with inviting men at age 65 and not inviting women (current UK strategy). In a futuristic scenario where genomic information is available on the population, our modelling suggests inviting male current smokers with high PRS earlier than 65 and targeting screening in female smokers with high/intermediate PRS at 65 and 70 respectively, may improve cost-effectiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.23297927">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.23297927" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.23297927">
        <p class="paperTitle">Evaluating the impact of modeling choices on the performance of integrated genetic and clinical models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.23297927" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.23297927" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morley, T. J.; Willimitis, D. J.; Ripperger, M. A.; Lee, H.; Han, L.; Zhou, Y.; Kang, J.; Davis, L. K.; Smoller, J. W.; Choi, K. W.; Walsh, C. G.; Ruderfer, D. M.</p>
        <p class="info">Score: 5.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.23297927' target='https://doi.org/10.1101/2023.11.01.23297927'> 10.1101/2023.11.01.23297927</a></p>
        <p class="abstract">The value of genetic information for improving the performance of clinical risk prediction models has yielded variable conclusions. Many methodological decisions have the potential to contribute to differential results across studies. Here, we performed multiple modeling experiments integrating clinical and demographic data from electronic health records (EHR) and genetic data to understand which decision points may affect performance. Clinical data in the form of structured diagnostic codes, medications, procedural codes, and demographics were extracted from two large independent health systems and polygenic risk scores (PRS) were generated across all patients with genetic data in the corresponding biobanks. Crohns disease was used as the model phenotype based on its substantial genetic component, established EHR-based definition, and sufficient prevalence for model training and testing. We investigated the impact of PRS integration method, as well as choices regarding training sample, model complexity, and performance metrics. Overall, our results show that including PRS resulted in higher performance by some metrics but the gain in performance was only robust when combined with demographic data alone. Improvements were inconsistent or negligible after including additional clinical information. The impact of genetic information on performance also varied by PRS integration method, with a small improvement in some cases from combining PRS with the output of a clinical model (late-fusion) compared to its inclusion an additional feature (early-fusion). The effects of other modeling decisions varied between institutions though performance increased with more compute-intensive models such as random forest. This work highlights the importance of considering methodological decision points in interpreting the impact on prediction performance when including PRS information in clinical models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.23297847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.23297847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.23297847">
        <p class="paperTitle">A polygenic risk score identifies undiagnosed cases of diabetes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.23297847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.23297847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: German, C.; Ashenhurst, J.; Wang, W.; 23andMe Research Team,  ; Granka, J. M.; Koelsch, B. L.; Abul-Husn, N. S.; Aslibekyan, S.; Auton, A.; Tung, J.; Shringarpure, S. S.; Holmes, M. V.</p>
        <p class="info">Score: 3.9, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.23297847' target='https://doi.org/10.1101/2023.10.31.23297847'> 10.1101/2023.10.31.23297847</a></p>
        <p class="abstract">ImportanceTwenty-three percent of 37.3M adults in the USA with diabetes are estimated to be undiagnosed, leading to potentially avoidable sequelae and morbidity.

ObjectiveTo explore the utility of a polygenic risk score (PRS) at identifying individuals with undiagnosed diabetes and prediabetes.

Design, Setting and ParticipantsIndividuals without doctor-diagnosed diabetes at study baseline in the UK Biobank (UKB) with HbA1c and BMI measurements. Participants were restricted to white individuals to use an ancestry-appropriate PRS. Undiagnosed diabetes and prediabetes were defined using HbA1c ([&amp;ge;]6.5% and [&amp;ge;]5.7 - &lt;6.5%, respectively).

ExposuresA diabetes PRS comprising 13,863 SNPs derived from the 23andMe Research Cohort, and measured BMI among UKB participants.

ResultsOf 412,439 individuals self-reporting an absence of diagnosed diabetes and who had BMI and HbA1c measurements at baseline, 2,934 (0.7%) had undiagnosed diabetes, representing 11.9% of all (diagnosed and undiagnosed) diabetes. Nearly half (1,362, 46%) of undiagnosed diabetes cases were among individuals in the top 25% of the PRS distribution. Overweight individuals (BMI [&amp;ge;]25 - &lt;30 kg/m2) who were in the top 12.5% of the PRS distribution had a similar frequency of undiagnosed diabetes (0.8-1.6% frequency) as individuals with obesity (BMI [&amp;ge;]30kg/m2) in the lowest 12.5% of the PRS distribution (0.7-1.7% frequency). Combining overweight and obesity with the PRS identified nearly all cases of undiagnosed diabetes: individuals with a BMI [&amp;ge;]25 kg/m2 (66% of the study population) or those in the top 54-69% of the PRS identified 98-99% of undiagnosed cases. Of the 199 undiagnosed diabetes cases occurring among individuals with a normal BMI (&lt;25kg/m2), two-thirds were among individuals in the top 50% of the PRS. Prediabetes was common (14%), with measured BMI and PRS providing additive risk. Among those in the top 12.5% PRS with BMI [&amp;ge;]35kg/m2, 6.3% developed incident diabetes over 4 years follow-up, as compared to 0% among the bottom 12.5% PRS with BMI&lt;25kg/m2.

ConclusionsA diabetes PRS is informative at identifying undiagnosed cases. PRS may have broader utility in detecting individuals with asymptomatic disease.

Key PointsO_ST_ABSQuestionC_ST_ABSDoes a polygenic risk score (PRS) have utility in identifying individuals with undiagnosed type 2 diabetes (T2D)?

FindingsIn this analysis of 412,439 individuals without doctor-diagnosed diabetes, a T2D PRS performed additively to body mass index (BMI) at identifying individuals with undiagnosed diabetes. Selecting individuals on the basis of overweight/obesity or a T2D PRS identified almost all cases of undiagnosed diabetes. The majority of undiagnosed diabetes cases among individuals with normal weight occurred among those at elevated polygenic risk.

MeaningA T2D PRS identifies cases of undiagnosed diabetes among individuals with and without overweight or obesity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.23296741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.23296741" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.23296741">
        <p class="paperTitle">Diagnostic Utility of Genome-wide DNA Methylation Analysis in Genetically Unsolved Developmental and Epileptic Encephalopathies and Refinement of a CHD2 Episignature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.23296741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.23296741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LaFlamme, C. W.; Rastin, C.; Sengupta, S.; Pennington, H. E.; Russ-Hall, S. J.; Schneider, A. L.; Bonkowski, E. S.; Almanza Fuerte, E. P.; Galey, M.; Goffena, J.; Gibson, S. B.; Allan, T. J.; Nyaga, D. M.; Lieffering, N.; Hebbar, M.; Walker, E. V.; Darnell, D.; Olsen, S. R.; Kolekar, P.; Djekidel, M. N.; Rosikiewicz, W.; McConkey, H.; Kerkhof, J.; Levy, M. A.; Relator, R.; Lev, D.; Lerman-Sagie, T.; Park, K. L.; Alders, M.; Cappuccio, G.; Chatron, N.; Demain, L.; Genevieve, D.; Lesca, G.; Roscioli, T.; Sanlaville, D.; Tedder, M. L.; Weisz-Hubshman, M.; Ketkar, S.; Dai, H.; Carlyle Worley, K.; </p>
        <p class="info">Score: 13.0, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.23296741' target='https://doi.org/10.1101/2023.10.11.23296741'> 10.1101/2023.10.11.23296741</a></p>
        <p class="abstract">Sequence-based genetic testing currently identifies causative genetic variants in [~]50% of individuals with developmental and epileptic encephalopathies (DEEs). Aberrant changes in DNA methylation are implicated in various neurodevelopmental disorders but remain unstudied in DEEs. Rare epigenetic variations (&#34;epivariants&#34;) can drive disease by modulating gene expression at single loci, whereas genome-wide DNA methylation changes can result in distinct &#34;episignature&#34; biomarkers for monogenic disorders in a growing number of rare diseases. Here, we interrogate the diagnostic utility of genome-wide DNA methylation array analysis on peripheral blood samples from 516 individuals with genetically unsolved DEEs who had previously undergone extensive genetic testing. We identified rare differentially methylated regions (DMRs) and explanatory episignatures to discover causative and candidate genetic etiologies in 10 individuals. We then used long-read sequencing to identify DNA variants underlying rare DMRs, including one balanced translocation, three CG-rich repeat expansions, and two copy number variants. We also identify pathogenic sequence variants associated with episignatures; some had been missed by previous exome sequencing. Although most DEE genes lack known episignatures, the increase in diagnostic yield for DNA methylation analysis in DEEs is comparable to the added yield of genome sequencing. Finally, we refine an episignature for CHD2 using an 850K methylation array which was further refined at higher CpG resolution using bisulfite sequencing to investigate potential insights into CHD2 pathophysiology. Our study demonstrates the diagnostic yield of genome-wide DNA methylation analysis to identify causal and candidate genetic causes as [~]2% (10/516) for unsolved DEE cases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297676">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297676" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297676">
        <p class="paperTitle">Genome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves cross-ancestry prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297676" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297676" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, T. J.; Graff, R. E.; Madduri, R. K.; Rodriguez, A. A.; Cario, C. L.; Feng, K.; Jiang, Y.; Wang, A.; Klein, R. J.; Pierce, B. L.; Eggener, S.; Tong, L.; Blot, W.; Long, J.; Rebbeck, T.; Lachance, J.; Andrews, C.; Adebiyi, A. O.; Adusei, B.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Jalloh, M.; Janivara, R.; Chen, W. C.; Mensah, J. E.; Agalliu, I.; Berndt, S. I.; Shelley, J. P.; Schaffer, K.; Machiela, M. J.; Freedman, N. D.; Huang, W.-Y.; Li, S. A.; Goodman, P. J.; Till, C.; Thompson, I.; Lilja, H.; Van Den Eeden, S. K.; Chanock, S. J.; Mosley, J. D.; Conti, D. V.; Haiman, C. A.;</p>
        <p class="info">Score: 7.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297676' target='https://doi.org/10.1101/2023.10.27.23297676'> 10.1101/2023.10.27.23297676</a></p>
        <p class="abstract">We conducted a multi-ancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry; 58,236 African ancestry; 23,546 Hispanic/Latino; 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (p[&amp;le;]5e-8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n=95,768). Meta-analyzing discovery and replication (n=392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our new polygenic risk score reached 16.9% (95% CI=16.1%-17.8%) in European ancestry, 9.5% (95% CI=7.0%-12.2%) in African ancestry, 18.6% (95% CI=15.8%-21.4%) in Hispanic/Latino, and 15.3% (95% CI=12.7%-18.1%) in Asian ancestry, and lower for higher age. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, with potential to personalize prostate cancer screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297963">
        <p class="paperTitle">The qMini assay identifies an overlooked class of splice variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guan, B.; Bender, C.; Pantrangi, M.; Moore, N.; Reeves, M.; Naik, A.; Li, H.; Goetz, K.; Blain, D.; Agather, A.; Cukras, C.; Zein, W. M.; Huryn, L. A.; Brooks, B. P.; Hufnagel, R. B.</p>
        <p class="info">Score: 2.2, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297963' target='https://doi.org/10.1101/2023.11.02.23297963'> 10.1101/2023.11.02.23297963</a></p>
        <p class="abstract">Splice variants are known to cause diseases by utilizing alternative splice sites, potentially resulting in protein truncation or mRNA degradation by nonsense-mediated decay. Splice variants are verified when altered mature mRNA sequences are identified in RNA analyses or minigene assays. Using a quantitative minigene assay, qMini, we uncovered a previously overlooked class of disease-associated splice variants that did not alter mRNA sequence but decreased mature mRNA level, suggesting a potentially new pathogenic mechanism.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
