<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs
Authors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.
Score: 18.4, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296230
BackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs
Authors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.
Score: 18.4, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296230
BackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs
Authors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.
Score: 18.4, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296230
BackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs\nAuthors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.\nScore: 18.4, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296230\nBackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders.",
  "keywords": [
    
  ],
  "articleBody": " Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs\nAuthors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.\nScore: 18.4, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296230\nBackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders. Spinocerebellar ataxia 4 (SCA4), one of the rarest SCAs, is characterized by adult-onset ataxia, polyneuropathy and linked to chromosome 16q22.1, the underlying mutation remains to be discovered. MethodsThree Swedish families affected by undiagnosed SCA went through detailed examinations, neurophysiological tests, neuroimaging studies and genetic testing. Imaging included MRI of the neuroaxis and brain PET. In 3 cases neuropathological assessments were performed. Genetic testing included SR WGS, STR analysis with Expansion Hunter de novo and long read (LR) WGS. ResultsNovel features such as dysautonomia, motor neuron affection, and abnormal eye movements, were found. Anticipation was documented as well. Atrophy in the cerebellum, brainstem and spinal cord was found in patients studied with MRI. [18F]FDG-PET demonstrated brain hypometabolism whereas [11C]Flumazenil-PET yielded reduced binding in several brain lobes, insula, thalamus, hypothalamus and cerebellum. Neuropathological assessment revealed moderate to severe loss of Purkinje cells in the cerebellum. In the spinal cord a marked loss of motor neurons in the anterior horns was seen as well as pronounced degeneration of posterior tracts. Intranuclear, mainly neuronal, inclusions positive for p62 and ubiquitin were sparse but widespread in the CNS. This finding prompted assessment for nucleotide expansions. A poly-glycine stretch encoding GGC expansions in the last exon of the zink finger homeobox 3 (ZFHX3) gene was identified segregating with disease, and not found in 1000 controls. ConclusionsSCA4 is a neurodegenerative disease, caused by a novel GGC expansion in ZFHX3, arguably the first polyglycine disorder in humans.\nThe alpha-2A-adrenergic receptor (ADRA2A) modulates susceptibility to Raynaud's syndrome\nAuthors: Tervi, A.; Ramste, M.; Abner, E.; Cheng, P.; Lane, J. M.; Maher, M.; Lammi, V.; Strausz, S.; Nguyen, T.; Docampo, M. L.; Gu, W.; FinnGen, ; Estonian biobank research team, ; Esko, T.; Saxena, R.; Palotie, A.; Ripatti, S.; Sinnott-Armstrong, N.; Daly, M. J.; Rabinovitch, M.; Heckman, C. A.; Quertermous, T.; Jones, S. E.; Ollila, H. M.\nScore: 11.8, Published: 2023-10-05 DOI: 10.1101/2023.10.04.23296526\nRaynaud's syndrome is a common dysautonomia where exposure to cold increases the vascular tone of distal arteries causing vasoconstriction and hypoxia particularly in the extremities. Current treatment options are limited and unspecific. Biological mechanisms leading to the phenotype remain uncharacterized. Using genetic and electronic health record data from the UK Biobank, the Mass-General Brigham Biobank, the Estonian Biobank, and the FinnGen study, we identified 11,605 individuals with a diagnosis for Raynaud's syndrome and 1,116,172 population controls. We found eight loci including endothelial nitric oxide synthase (NOS3), HLA, and a notable association at the alpha-2A-adrenergic receptor (ADRA2A) locus (rs7090046, P = 3.93x10-47), implicating adrenergic signaling as a major risk factor with Raynaud's syndrome. We further investigate the role of the variants and ADRA2A expression in functional and physiological models. In silico follow-up analysis revealed an expression quantitative trait locus (eQTL) that co-localized and increased ADRA2A gene expression in tissue specific manner in the distal arteries. Staining with RNA scope further clarified the specificity of ADRA2A expression in small vessels. We show by CRISPR gene editing that the SNP region modifies ADRA2A gene expression in pulmonary artery smooth muscle cells. Finally, we performed a functional contraction assay on smooth muscle cells in cold conditions and showed lower contraction in ADRA2A-deficient and higher contraction in ADRA2A-overexpressing smooth muscle cells. Our results indicate that Raynaud's syndrome is related to vascular function mediated by adrenergic signaling through ADRA2A. Our study highlights the power of genome-wide association testing as a discovery tool for poorly understood clinical endpoints and further clarifies the role of adrenergic signaling in Raynaud's syndrome by fine-mapping, using in vitro genomic manipulations and functional validation in distal smooth muscle cell populations located in arterioles.\nAn AluYb8 retrotransposon characterises a risk haplotype of TMEM106B associated in neurodegeneration\nAuthors: Salazar, A.; Tesi, N.; Knoop, L.; Pijnenburg, Y.; van der Lee, S.; Wijesekera, S.; Krizova, J.; Hiltunen, M.; Damme, M.; Petrucelli, L.; Reinders, M.; Hulsman, M.; Holstege, H.\nScore: 8.8, Published: 2023-10-03 DOI: 10.1101/2023.07.16.23292721\nGenome-wide association studies identified a role for TMEM106B in various neurodegenerative diseases. Based on long-read whole-genome sequencing data of 256 individuals, we identified an AluYb8 retrotransposon in the 3 UTR of the risk haplotype of TMEM106B. When transcriptionally active, Alu-elements can propogate throughout the genome, and mediate (post-)transcriptional dysregulation of nearby genes. We found that TMEM106B haplotypes carrying the AluYb8 element are more methylated than those without, likely reflecting an evolutionary selection to suppress propagation. AluYb8 activation can be further suppressed by TDP-43, in its role in post-transcriptional RNA-processing. However, age-related loss of TDP-43, by reduced methylation in the 3 UTR of TARDBP, may release AluYb8 suppression. Together, our findings suggest that in the aging brain, the AluYb8 insertion may mediate dysregulation of TMEM106B, impacting the endolysosomal system via a negative-feedback loop, ultimately leading to neurodegenerative disease. Notably, TMEM106B haplotype sequences are different between African and European genomes, which likely explains the different effects on disease-risk between both populations. Overall, our research advances the understanding of the roles of TDP-43 and TMEM106B in neurodegenerative diseases, and provides a novel connection between genetic variation and age-related changes in genomic and cellular regulation.\nShared genetic basis informs the roles of polyunsaturated fatty acids in brain disorders\nAuthors: Xu, H.; Sun, Y.; Francis, M.; Cheng, C. F.; Modulla, N. T. R.; Brenna, J. T.; Chiang, C. W. K.; Ye, K.\nScore: 6.0, Published: 2023-10-04 DOI: 10.1101/2023.10.03.23296500\nThe neural tissue is rich in polyunsaturated fatty acids (PUFAs), components that are indispensable for the proper functioning of neurons, such as neurotransmission. PUFA nutritional deficiency and imbalance have been linked to a variety of chronic brain disorders, including major depressive disorder (MDD), anxiety, and anorexia. However, the effects of PUFAs on brain disorders remain inconclusive, and the extent of their shared genetic determinants is largely unknown. Here, we used genome-wide association summary statistics to systematically examine the shared genetic basis between six phenotypes of circulating PUFAs (N = 114,999) and 20 brain disorders (N = 9,725-762,917), infer their potential causal relationships, identify colocalized regions, and pinpoint shared genetic variants. Genetic correlation and polygenic overlap analyses revealed a widespread shared genetic basis for 77 trait pairs between six PUFA phenotypes and 16 brain disorders. Two-sample Mendelian randomization analysis indicated potential causal relationships for 16 pairs of PUFAs and brain disorders, including alcohol consumption, bipolar disorder (BIP), and MDD. Colocalization analysis identified 40 shared loci (13 unique) among six PUFAs and ten brain disorders. Twenty-two unique variants were statistically inferred as candidate shared causal variants, including rs1260326 (GCKR), rs174564 (FADS2) and rs4818766 (ADARB1). These findings reveal a widespread shared genetic basis between PUFAs and brain disorders, pinpoint specific shared variants, and provide support for the potential effects of PUFAs on certain brain disorders, especially MDD, BIP, and alcohol consumption.\nGenome-wide association analysis reveals insights into the molecular aetiology underlying dilated cardiomyopathy\nAuthors: Zheng, S. L.; Henry, A.; Cannie, D.; Lee, M.; Miller, D.; McGurk, K. A.; Bond, I.; Xu, X.; Issa, H.; Francis, C.; De Marvao, A.; Theotokis, P. I.; Buchan, R. J.; Speed, D.; Abner, E.; Adams, L.; Aragam, K. G.; Arnlov, J.; Axelsson Raja, A.; Backman, J. D.; Baksi, J.; Barton, P. J.; Biddinger, K. J.; Boersma, E.; Brandimarto, J.; Brunak, S.; Bundgaard, H.; Carey, D. J.; Charron, P.; Cook, J. P.; Cook, S. A.; Denaxas, S.; Deleuze, J.-F.; Doney, A. S.; Elliot, P.; Erikstrup, C.; Esko, T.; Farber-Eger, E. H.; Finan, C.; Garnier, S.; Ghouse, J.; Giedraitis, V.; Gudbjartsson, D. F.; Haggerty, C. M.;\nScore: 15.4, Published: 2023-09-29 DOI: 10.1101/2023.09.28.23295408\nDilated cardiomyopathy (DCM) is a clinical disorder characterised by reduced contractility of the heart muscle that is not explained by coronary artery disease or abnormal haemodynamic loading. Although Mendelian disease is well described, clinical testing yields a genetic cause in a minority of patients. The role of complex inheritance is emerging, however the common genetic architecture is relatively unexplored. To improve our understanding of the genetic basis of DCM, we perform a genome-wide association study (GWAS) meta-analysis comprising 14,255 DCM cases and 1,199,156 controls, and a multi-trait GWAS incorporating correlated cardiac magnetic resonance imaging traits of 36,203 participants. We identify 80 genetic susceptibility loci and prioritize 61 putative effector genes for DCM by synthesizing evidence from 8 gene prioritization strategies. Rare variant association testing identifies genes associated with DCM, including MAP3K7, NEDD4L, and SSPN. Through integration with single-nuclei transcriptomics from 52 end-stage DCM patients and 18 controls, we identify cellular states, biological pathways, and intercellular communications driving DCM pathogenesis. Finally, we demonstrate that a polygenic score predicts DCM in the general population and modulates the penetrance of rare pathogenic and likely pathogenic variants in DCM-causing genes. Our findings may inform the design of novel clinical genetic testing strategies incorporating polygenic background and the genes and pathways identified may inform the development of targeted therapeutics.\nMulti-ancestry Polygenic Mechanisms of Type 2 Diabetes Elucidate Disease Processes and Clinical Heterogeneity\nAuthors: Smith, K.; Deutsch, A.; McGrail, C.; Kim, H.; Hsu, S.; Mandla, R.; Schroeder, P. H.; Westerman, K. E.; Szczerbinski, L.; Majarian, T. D.; Kaur, V.; Williamson, A.; Claussnitzer, M.; Florez, J. C.; Manning, A. K.; Mercader, J. M.; Gaulton, K. J.; Udler, M. S.\nScore: 16.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.23296294\nWe identified genetic subtypes of type 2 diabetes (T2D) by analyzing genetic data from diverse groups, including non-European populations. We implemented soft clustering with 650 T2D-associated genetic variants, capturing known and novel T2D subtypes with distinct cardiometabolic trait associations. The twelve genetic clusters were distinctively enriched for single-cell regulatory regions. Polygenic scores derived from the clusters differed in distribution between ancestry groups, including a significantly higher proportion of lipodystrophy-related polygenic risk in East Asian ancestry. T2D risk was equivalent at a BMI of 30 kg/m2 in the European subpopulation and 24.2 (22.9- 25.5) kg/m2 in the East Asian subpopulation; after adjusting for cluster-specific genetic risk, the equivalent BMI threshold increased to 28.5 (27.1-30.0) kg/m2 in the East Asian group, explaining about 75% of the difference in BMI thresholds. Thus, these multi-ancestry T2D genetic subtypes encompass a broader range of biological mechanisms and help explain ancestry-associated differences in T2D risk profiles.\nGenome sequencing as a generic diagnostic strategy for rare disease\nAuthors: Schobers, G.; Derks, R.; Ouden, A. D.; Swinkels, H.; Reeuwijk, J. V.; Bosgoed, E.; Lugtenberg, D.; Sun, S. M.; Corominas Galbani, J.; Weiss, J.; Blok, R.; Olde Keizer, R.; Hofste, T.; Hellebrekers, D.; Leeuw, N. D.; Stegmann, A.; Kamsteeg, E.-J.; Paulussen, A.; Ligtenberg, M.; Bradley, X. Z.; Peden, J.; Gutierrez, A.; Pullen, A.; Payne, T.; Gilissen, C.; Wijngaard, A. V.; Brunner, H.; Nelen, M.; Yntema, H.; Vissers, L.\nScore: 4.8, Published: 2023-09-28 DOI: 10.1101/2023.09.28.23296271\nBackgroundTo diagnose the full spectrum of hereditary and congenital diseases, genetic laboratories use many different workflows, ranging from karyotyping to exome sequencing. A single generic high-throughput workflow would greatly increase efficiency. We assessed whether genome sequencing (GS) can replace these existing workflows aimed at germline genetic diagnosis for rare disease. MethodsWe performed GS (NovaSeq6000; 37x mean coverage) on 1,000 cases with 1,271 known clinically relevant variants, identified across different workflows, representative of our tertiary diagnostic centers. Variants were categorized into small variants (single nucleotide variants and indels \u003c50 bp), large variants (copy number variants and short tandem repeats) and other variants (structural variants and aneuploidies). Variant calling format files were queried per variant, from which workflow-specific true positive rates (TPRs) for detection were determined. A TPR of [\u0026ge;]98% was considered the lower threshold for transition to GS. A GS-first scenario was generated for our laboratory, using diagnostic efficacy and predicted false negative as primary outcome measures. As input, we modeled the diagnostic path for all 24,570 individuals referred in 2022, combining the clinical referral, the transition of the underlying workflow(s) to GS, and the variant type(s) to be detected. ResultsOverall, 95% (1,206/1,271) of variants were detected. Detection rates differed per variant category: small variants in 96% (826/860), large variants in 93% (341/366), and other variants in 87% (39/45). TPRs varied between workflows (79-100%), with 7/10 being replaceable by GS. Models for our laboratory indicate that a GS-first strategy would be feasible for 84.9% of clinical referrals (750/883), translating to 71% of all individuals (17,444/24,570) receiving GS as their primary test. An estimated false negative rate of 0.3% could be expected. ConclusionGS can capture clinically relevant germline variants in a GS-first strategy for the majority of clinical indications in a genetics diagnostic lab.\nBlood-based multivariate methylation risk score for cognitive impairment and dementia\nAuthors: Koetsier, J.; Cavill, R.; Reijnders, R. A.; Harvey, J.; Deckers, K.; Köhler, S.; Eijssen, L.; Smith, R. G.; Smith, A. R.; Burrage, J.; Walker, E. M.; Shireby, G.; Hannon, E.; Dempster, E.; Frayling, T.; Mill, J.; Dobricic, V.; Sommerer, Y.; Johannsen, P.; Wittig, M.; Franke, A.; Vandenberghe, R.; Schaeverbeke, J.; Freund-Levi, Y.; Frölich, L.; Scheltens, P.; Teunissen, C.; Frisoni, G.; Blin, O.; Richardson, J.; Bordet, R.; Engelborghs, S.; de Roeck, E.; Martinez-Lage, P.; Tainta, M.; Lleo, A.; Sala, I.; Popp, J.; Peyratout, G.; Verhey, F.; Tsolaki, M.; Andreasson, U.; Blennow, K.; Zetterberg\nScore: 3.5, Published: 2023-09-28 DOI: 10.1101/2023.09.27.23296143\nINTRODUCTIONGiven the established association between DNA methylation and the pathophysiology of dementia and its plausible role as a molecular mediator of lifestyle and environment, blood-derived DNA methylation data could enable early detection of dementia risk. METHODSIn conjunction with an extensive array of machine learning techniques, we employed whole blood genome-wide DNA methylation data as a surrogate for 14 modifiable and non-modifiable factors in the assessment of dementia risk in two independent cohorts of Alzheimers disease (AD) and Parkinsons disease (PD). RESULTSWe established a multivariate methylation risk score (MMRS) to identify the status of mild cognitive impairment (MCI) cross-sectionally, independent of age and sex. We further demonstrated significant predictive capability of this score for the prospective onset of cognitive decline in AD and PD. DISCUSSIONOur work shows the potential of employing blood-derived DNA methylation data in the assessment of dementia risk.\nAssociation Between Psychiatric Polygenic Scores, Healthcare Utilization and Comorbidity Burden\nAuthors: Kirchner, H. L.; Rocha, D.; Linner, R. K.; Wilimitis, D.; Walsh, C. G.; Ripperger, M. A.; Lee, H.; Liu, Z.; Davis, L.; Hu, Y.; Chabris, C. F.; Smoller, J. W.\nScore: 2.1, Published: 2023-09-30 DOI: 10.1101/2023.09.29.23296345\nPurposeTo estimate the association of psychiatric polygenic scores with healthcare utilization and comorbidity burden. MethodsObservational cohort study (N = 118,882) of adolescent and adult biobank participants with linked electronic health records (EHRs) from three diverse study sites; (Massachusetts General Brigham, Vanderbilt University Medical Center, Geisinger). Polygenic scores (PGS) were derived from the largest available GWAS of major depressive depression, bipolar disorder, and schizophrenia at the time of analysis. Negative binomial regression models were used to estimate the association between each psychiatric PGS and healthcare utilization and comorbidity burden. Healthcare utilization was measured as frequency of emergency department (ED), inpatient (IP), and outpatient (OP) visits. Comorbidity burden was defined by the Elixhauser Comorbidity Index and the Charlson Comorbidity Index. ResultsParticipants had a median follow-up duration of 12 years in the EHR. Individuals in the top decile of polygenic score for major depressive disorder had significantly more ED visits (RR=1.22, 95% CI; 1.17, 1.29) compared to those the lowest decile. Increases were also observed with IP and comorbidity burden. Among those diagnosed with depression and in the highest decile of the PGS, there was an increase in all utilization types (ED: RR=1.56, 95% CI 1.41, 1.72; OP: RR=1.16, 95% CI 1.08, 1.24; IP: RR=1.23, 95% CI 1.12, 1.36) post-diagnosis. No clinically significant results were observed with bipolar and schizophrenia polygenic scores. ConclusionsPolygenic score for depression is modestly associated with increased healthcare resource utilization and comorbidity burden, in the absence of diagnosis. Following a diagnosis of depression, the PGS was associated with further increases in healthcare utilization. These findings suggest that depression genetic risk is associated with utilization and burden of chronic disease in real-world settings.\nLandscape of blood group antigens and alleles in the Indian population from whole genome sequences\nAuthors: Rophina, M.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.\nScore: 4.3, Published: 2023-09-26 DOI: 10.1101/2023.09.26.23296145\nBlood group antigens are genetically inherited macromolecular structures which form the underlying factor for inter individual variations in human blood. Currently there exists over 390 human blood group antigens corresponding to 44 blood group systems and 2 erythroid specific transcription factors. Distribution of these blood group antigens have been found to differ significantly among various ethnic populations. To date, there is a lack of comprehensive research that offers extensive blood group profiles for the Indian population. Whole genome sequence data (hg38) of 1029 self-declared healthy Indian individuals generated as a part of the pilot phase IndiGen programme were used for the analysis. Variants spanning the genes of 44 blood group systems and two transcription factors KLF1, GATA1 were fetched and annotated for their functional consequences. Our study reports a total of 40712 blood group related variants of which 695 were identified as non-synonymous variants in the coding region. Of the total non-synonymous variants, 105 were found to have a known blood phenotype. A total of 24 variants belonging to 12 blood groups were predicted to be deleterious by more than three computational tools. Our study was also able to identify a few rare blood phenotypes including Au(a-b+), Js(a+b+), Di(a+b-), In(a+b-) and KANNO-. This study is the first to use genomic data to understand the blood group antigen profiles of the Indian population, and it also systematically compares these profiles with those of other global populations. Key pointsO_LIAccurate characterization of the genomic landscape of known and rare blood group alleles and antigens in the Indian population using the whole genome sequencing data of 1029 self-declared healthy individuals C_LIO_LIUnderstanding the distinct similarities and differences in blood group genotypes and phenotypes across diverse global populations through systematic comparison of genomic datasets. C_LI Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=151 SRC=\"FIGDIR/small/23296145v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@1dc3400org.highwire.dtl.DTLVardef@1da07a0org.highwire.dtl.DTLVardef@12a9f74org.highwire.dtl.DTLVardef@14a848d_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "3020",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.23296230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.23296230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.23296230">
        <p class="paperTitle">Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.23296230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.23296230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.</p>
        <p class="info">Score: 18.4, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.23296230' target='https://doi.org/10.1101/2023.10.03.23296230'> 10.1101/2023.10.03.23296230</a></p>
        <p class="abstract">BackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders. Spinocerebellar ataxia 4 (SCA4), one of the rarest SCAs, is characterized by adult-onset ataxia, polyneuropathy and linked to chromosome 16q22.1, the underlying mutation remains to be discovered.

MethodsThree Swedish families affected by undiagnosed SCA went through detailed examinations, neurophysiological tests, neuroimaging studies and genetic testing. Imaging included MRI of the neuroaxis and brain PET. In 3 cases neuropathological assessments were performed. Genetic testing included SR WGS, STR analysis with Expansion Hunter de novo and long read (LR) WGS.

ResultsNovel features such as dysautonomia, motor neuron affection, and abnormal eye movements, were found. Anticipation was documented as well. Atrophy in the cerebellum, brainstem and spinal cord was found in patients studied with MRI. [18F]FDG-PET demonstrated brain hypometabolism whereas [11C]Flumazenil-PET yielded reduced binding in several brain lobes, insula, thalamus, hypothalamus and cerebellum. Neuropathological assessment revealed moderate to severe loss of Purkinje cells in the cerebellum. In the spinal cord a marked loss of motor neurons in the anterior horns was seen as well as pronounced degeneration of posterior tracts. Intranuclear, mainly neuronal, inclusions positive for p62 and ubiquitin were sparse but widespread in the CNS. This finding prompted assessment for nucleotide expansions. A poly-glycine stretch encoding GGC expansions in the last exon of the zink finger homeobox 3 (ZFHX3) gene was identified segregating with disease, and not found in 1000 controls.

ConclusionsSCA4 is a neurodegenerative disease, caused by a novel GGC expansion in ZFHX3, arguably the first polyglycine disorder in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.23296526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.23296526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.23296526">
        <p class="paperTitle">The alpha-2A-adrenergic receptor (ADRA2A) modulates susceptibility to Raynaud&#39;s syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.23296526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.23296526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tervi, A.; Ramste, M.; Abner, E.; Cheng, P.; Lane, J. M.; Maher, M.; Lammi, V.; Strausz, S.; Nguyen, T.; Docampo, M. L.; Gu, W.; FinnGen,  ; Estonian biobank research team,  ; Esko, T.; Saxena, R.; Palotie, A.; Ripatti, S.; Sinnott-Armstrong, N.; Daly, M. J.; Rabinovitch, M.; Heckman, C. A.; Quertermous, T.; Jones, S. E.; Ollila, H. M.</p>
        <p class="info">Score: 11.8, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.23296526' target='https://doi.org/10.1101/2023.10.04.23296526'> 10.1101/2023.10.04.23296526</a></p>
        <p class="abstract">Raynaud&#39;s syndrome is a common dysautonomia where exposure to cold increases the vascular tone of distal arteries causing vasoconstriction and hypoxia particularly in the extremities. Current treatment options are limited and unspecific. Biological mechanisms leading to the phenotype remain uncharacterized. Using genetic and electronic health record data from the UK Biobank, the Mass-General Brigham Biobank, the Estonian Biobank, and the FinnGen study, we identified 11,605 individuals with a diagnosis for Raynaud&#39;s syndrome and 1,116,172 population controls. We found eight loci including endothelial nitric oxide synthase (NOS3), HLA, and a notable association at the alpha-2A-adrenergic receptor (ADRA2A) locus (rs7090046, P = 3.93x10-47), implicating adrenergic signaling as a major risk factor with Raynaud&#39;s syndrome. We further investigate the role of the variants and ADRA2A expression in functional and physiological models. In silico follow-up analysis revealed an expression quantitative trait locus (eQTL) that co-localized and increased ADRA2A gene expression in tissue specific manner in the distal arteries. Staining with RNA scope further clarified the specificity of ADRA2A expression in small vessels. We show by CRISPR gene editing that the SNP region modifies ADRA2A gene expression in pulmonary artery smooth muscle cells. Finally, we performed a functional contraction assay on smooth muscle cells in cold conditions and showed lower contraction in ADRA2A-deficient and higher contraction in ADRA2A-overexpressing smooth muscle cells. Our results indicate that Raynaud&#39;s syndrome is related to vascular function mediated by adrenergic signaling through ADRA2A. Our study highlights the power of genome-wide association testing as a discovery tool for poorly understood clinical endpoints and further clarifies the role of adrenergic signaling in Raynaud&#39;s syndrome by fine-mapping, using in vitro genomic manipulations and functional validation in distal smooth muscle cell populations located in arterioles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292721">
        <p class="paperTitle">An AluYb8 retrotransposon characterises a risk haplotype of TMEM106B associated in neurodegeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salazar, A.; Tesi, N.; Knoop, L.; Pijnenburg, Y.; van der Lee, S.; Wijesekera, S.; Krizova, J.; Hiltunen, M.; Damme, M.; Petrucelli, L.; Reinders, M.; Hulsman, M.; Holstege, H.</p>
        <p class="info">Score: 8.8, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292721' target='https://doi.org/10.1101/2023.07.16.23292721'> 10.1101/2023.07.16.23292721</a></p>
        <p class="abstract">Genome-wide association studies identified a role for TMEM106B in various neurodegenerative diseases. Based on long-read whole-genome sequencing data of 256 individuals, we identified an AluYb8 retrotransposon in the 3 UTR of the risk haplotype of TMEM106B. When transcriptionally active, Alu-elements can propogate throughout the genome, and mediate (post-)transcriptional dysregulation of nearby genes. We found that TMEM106B haplotypes carrying the AluYb8 element are more methylated than those without, likely reflecting an evolutionary selection to suppress propagation. AluYb8 activation can be further suppressed by TDP-43, in its role in post-transcriptional RNA-processing. However, age-related loss of TDP-43, by reduced methylation in the 3 UTR of TARDBP, may release AluYb8 suppression. Together, our findings suggest that in the aging brain, the AluYb8 insertion may mediate dysregulation of TMEM106B, impacting the endolysosomal system via a negative-feedback loop, ultimately leading to neurodegenerative disease. Notably, TMEM106B haplotype sequences are different between African and European genomes, which likely explains the different effects on disease-risk between both populations. Overall, our research advances the understanding of the roles of TDP-43 and TMEM106B in neurodegenerative diseases, and provides a novel connection between genetic variation and age-related changes in genomic and cellular regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.23296500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.23296500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.23296500">
        <p class="paperTitle">Shared genetic basis informs the roles of polyunsaturated fatty acids in brain disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.23296500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.23296500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, H.; Sun, Y.; Francis, M.; Cheng, C. F.; Modulla, N. T. R.; Brenna, J. T.; Chiang, C. W. K.; Ye, K.</p>
        <p class="info">Score: 6.0, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.23296500' target='https://doi.org/10.1101/2023.10.03.23296500'> 10.1101/2023.10.03.23296500</a></p>
        <p class="abstract">The neural tissue is rich in polyunsaturated fatty acids (PUFAs), components that are indispensable for the proper functioning of neurons, such as neurotransmission. PUFA nutritional deficiency and imbalance have been linked to a variety of chronic brain disorders, including major depressive disorder (MDD), anxiety, and anorexia. However, the effects of PUFAs on brain disorders remain inconclusive, and the extent of their shared genetic determinants is largely unknown. Here, we used genome-wide association summary statistics to systematically examine the shared genetic basis between six phenotypes of circulating PUFAs (N = 114,999) and 20 brain disorders (N = 9,725-762,917), infer their potential causal relationships, identify colocalized regions, and pinpoint shared genetic variants. Genetic correlation and polygenic overlap analyses revealed a widespread shared genetic basis for 77 trait pairs between six PUFA phenotypes and 16 brain disorders. Two-sample Mendelian randomization analysis indicated potential causal relationships for 16 pairs of PUFAs and brain disorders, including alcohol consumption, bipolar disorder (BIP), and MDD. Colocalization analysis identified 40 shared loci (13 unique) among six PUFAs and ten brain disorders. Twenty-two unique variants were statistically inferred as candidate shared causal variants, including rs1260326 (GCKR), rs174564 (FADS2) and rs4818766 (ADARB1). These findings reveal a widespread shared genetic basis between PUFAs and brain disorders, pinpoint specific shared variants, and provide support for the potential effects of PUFAs on certain brain disorders, especially MDD, BIP, and alcohol consumption.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.23295408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.23295408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.23295408">
        <p class="paperTitle">Genome-wide association analysis reveals insights into the molecular aetiology underlying dilated cardiomyopathy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.23295408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.23295408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, S. L.; Henry, A.; Cannie, D.; Lee, M.; Miller, D.; McGurk, K. A.; Bond, I.; Xu, X.; Issa, H.; Francis, C.; De Marvao, A.; Theotokis, P. I.; Buchan, R. J.; Speed, D.; Abner, E.; Adams, L.; Aragam, K. G.; Arnlov, J.; Axelsson Raja, A.; Backman, J. D.; Baksi, J.; Barton, P. J.; Biddinger, K. J.; Boersma, E.; Brandimarto, J.; Brunak, S.; Bundgaard, H.; Carey, D. J.; Charron, P.; Cook, J. P.; Cook, S. A.; Denaxas, S.; Deleuze, J.-F.; Doney, A. S.; Elliot, P.; Erikstrup, C.; Esko, T.; Farber-Eger, E. H.; Finan, C.; Garnier, S.; Ghouse, J.; Giedraitis, V.; Gudbjartsson, D. F.; Haggerty, C. M.;</p>
        <p class="info">Score: 15.4, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.23295408' target='https://doi.org/10.1101/2023.09.28.23295408'> 10.1101/2023.09.28.23295408</a></p>
        <p class="abstract">Dilated cardiomyopathy (DCM) is a clinical disorder characterised by reduced contractility of the heart muscle that is not explained by coronary artery disease or abnormal haemodynamic loading. Although Mendelian disease is well described, clinical testing yields a genetic cause in a minority of patients. The role of complex inheritance is emerging, however the common genetic architecture is relatively unexplored. To improve our understanding of the genetic basis of DCM, we perform a genome-wide association study (GWAS) meta-analysis comprising 14,255 DCM cases and 1,199,156 controls, and a multi-trait GWAS incorporating correlated cardiac magnetic resonance imaging traits of 36,203 participants. We identify 80 genetic susceptibility loci and prioritize 61 putative effector genes for DCM by synthesizing evidence from 8 gene prioritization strategies. Rare variant association testing identifies genes associated with DCM, including MAP3K7, NEDD4L, and SSPN. Through integration with single-nuclei transcriptomics from 52 end-stage DCM patients and 18 controls, we identify cellular states, biological pathways, and intercellular communications driving DCM pathogenesis. Finally, we demonstrate that a polygenic score predicts DCM in the general population and modulates the penetrance of rare pathogenic and likely pathogenic variants in DCM-causing genes. Our findings may inform the design of novel clinical genetic testing strategies incorporating polygenic background and the genes and pathways identified may inform the development of targeted therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.23296294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.23296294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.23296294">
        <p class="paperTitle">Multi-ancestry Polygenic Mechanisms of Type 2 Diabetes Elucidate Disease Processes and Clinical Heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.23296294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.23296294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, K.; Deutsch, A.; McGrail, C.; Kim, H.; Hsu, S.; Mandla, R.; Schroeder, P. H.; Westerman, K. E.; Szczerbinski, L.; Majarian, T. D.; Kaur, V.; Williamson, A.; Claussnitzer, M.; Florez, J. C.; Manning, A. K.; Mercader, J. M.; Gaulton, K. J.; Udler, M. S.</p>
        <p class="info">Score: 16.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.23296294' target='https://doi.org/10.1101/2023.09.28.23296294'> 10.1101/2023.09.28.23296294</a></p>
        <p class="abstract">We identified genetic subtypes of type 2 diabetes (T2D) by analyzing genetic data from diverse groups, including non-European populations. We implemented soft clustering with 650 T2D-associated genetic variants, capturing known and novel T2D subtypes with distinct cardiometabolic trait associations. The twelve genetic clusters were distinctively enriched for single-cell regulatory regions. Polygenic scores derived from the clusters differed in distribution between ancestry groups, including a significantly higher proportion of lipodystrophy-related polygenic risk in East Asian ancestry. T2D risk was equivalent at a BMI of 30 kg/m2 in the European subpopulation and 24.2 (22.9- 25.5) kg/m2 in the East Asian subpopulation; after adjusting for cluster-specific genetic risk, the equivalent BMI threshold increased to 28.5 (27.1-30.0) kg/m2 in the East Asian group, explaining about 75% of the difference in BMI thresholds. Thus, these multi-ancestry T2D genetic subtypes encompass a broader range of biological mechanisms and help explain ancestry-associated differences in T2D risk profiles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.23296271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.23296271" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.23296271">
        <p class="paperTitle">Genome sequencing as a generic diagnostic strategy for rare disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.23296271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.23296271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schobers, G.; Derks, R.; Ouden, A. D.; Swinkels, H.; Reeuwijk, J. V.; Bosgoed, E.; Lugtenberg, D.; Sun, S. M.; Corominas Galbani, J.; Weiss, J.; Blok, R.; Olde Keizer, R.; Hofste, T.; Hellebrekers, D.; Leeuw, N. D.; Stegmann, A.; Kamsteeg, E.-J.; Paulussen, A.; Ligtenberg, M.; Bradley, X. Z.; Peden, J.; Gutierrez, A.; Pullen, A.; Payne, T.; Gilissen, C.; Wijngaard, A. V.; Brunner, H.; Nelen, M.; Yntema, H.; Vissers, L.</p>
        <p class="info">Score: 4.8, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.23296271' target='https://doi.org/10.1101/2023.09.28.23296271'> 10.1101/2023.09.28.23296271</a></p>
        <p class="abstract">BackgroundTo diagnose the full spectrum of hereditary and congenital diseases, genetic laboratories use many different workflows, ranging from karyotyping to exome sequencing. A single generic high-throughput workflow would greatly increase efficiency. We assessed whether genome sequencing (GS) can replace these existing workflows aimed at germline genetic diagnosis for rare disease.

MethodsWe performed GS (NovaSeq6000; 37x mean coverage) on 1,000 cases with 1,271 known clinically relevant variants, identified across different workflows, representative of our tertiary diagnostic centers. Variants were categorized into small variants (single nucleotide variants and indels &lt;50 bp), large variants (copy number variants and short tandem repeats) and other variants (structural variants and aneuploidies). Variant calling format files were queried per variant, from which workflow-specific true positive rates (TPRs) for detection were determined. A TPR of [&amp;ge;]98% was considered the lower threshold for transition to GS. A GS-first scenario was generated for our laboratory, using diagnostic efficacy and predicted false negative as primary outcome measures. As input, we modeled the diagnostic path for all 24,570 individuals referred in 2022, combining the clinical referral, the transition of the underlying workflow(s) to GS, and the variant type(s) to be detected.

ResultsOverall, 95% (1,206/1,271) of variants were detected. Detection rates differed per variant category: small variants in 96% (826/860), large variants in 93% (341/366), and other variants in 87% (39/45). TPRs varied between workflows (79-100%), with 7/10 being replaceable by GS. Models for our laboratory indicate that a GS-first strategy would be feasible for 84.9% of clinical referrals (750/883), translating to 71% of all individuals (17,444/24,570) receiving GS as their primary test. An estimated false negative rate of 0.3% could be expected.

ConclusionGS can capture clinically relevant germline variants in a  GS-first strategy for the majority of clinical indications in a genetics diagnostic lab.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.23296143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.23296143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.23296143">
        <p class="paperTitle">Blood-based multivariate methylation risk score for cognitive impairment and dementia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.23296143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.23296143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koetsier, J.; Cavill, R.; Reijnders, R. A.; Harvey, J.; Deckers, K.; Köhler, S.; Eijssen, L.; Smith, R. G.; Smith, A. R.; Burrage, J.; Walker, E. M.; Shireby, G.; Hannon, E.; Dempster, E.; Frayling, T.; Mill, J.; Dobricic, V.; Sommerer, Y.; Johannsen, P.; Wittig, M.; Franke, A.; Vandenberghe, R.; Schaeverbeke, J.; Freund-Levi, Y.; Frölich, L.; Scheltens, P.; Teunissen, C.; Frisoni, G.; Blin, O.; Richardson, J.; Bordet, R.; Engelborghs, S.; de Roeck, E.; Martinez-Lage, P.; Tainta, M.; Lleo, A.; Sala, I.; Popp, J.; Peyratout, G.; Verhey, F.; Tsolaki, M.; Andreasson, U.; Blennow, K.; Zetterberg</p>
        <p class="info">Score: 3.5, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.23296143' target='https://doi.org/10.1101/2023.09.27.23296143'> 10.1101/2023.09.27.23296143</a></p>
        <p class="abstract">INTRODUCTIONGiven the established association between DNA methylation and the pathophysiology of dementia and its plausible role as a molecular mediator of lifestyle and environment, blood-derived DNA methylation data could enable early detection of dementia risk.

METHODSIn conjunction with an extensive array of machine learning techniques, we employed whole blood genome-wide DNA methylation data as a surrogate for 14 modifiable and non-modifiable factors in the assessment of dementia risk in two independent cohorts of Alzheimers disease (AD) and Parkinsons disease (PD).

RESULTSWe established a multivariate methylation risk score (MMRS) to identify the status of mild cognitive impairment (MCI) cross-sectionally, independent of age and sex. We further demonstrated significant predictive capability of this score for the prospective onset of cognitive decline in AD and PD.

DISCUSSIONOur work shows the potential of employing blood-derived DNA methylation data in the assessment of dementia risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.23296345">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.23296345" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.23296345">
        <p class="paperTitle">Association Between Psychiatric Polygenic Scores, Healthcare Utilization and Comorbidity Burden</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.23296345" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.23296345" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kirchner, H. L.; Rocha, D.; Linner, R. K.; Wilimitis, D.; Walsh, C. G.; Ripperger, M. A.; Lee, H.; Liu, Z.; Davis, L.; Hu, Y.; Chabris, C. F.; Smoller, J. W.</p>
        <p class="info">Score: 2.1, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.23296345' target='https://doi.org/10.1101/2023.09.29.23296345'> 10.1101/2023.09.29.23296345</a></p>
        <p class="abstract">PurposeTo estimate the association of psychiatric polygenic scores with healthcare utilization and comorbidity burden.

MethodsObservational cohort study (N = 118,882) of adolescent and adult biobank participants with linked electronic health records (EHRs) from three diverse study sites; (Massachusetts General Brigham, Vanderbilt University Medical Center, Geisinger). Polygenic scores (PGS) were derived from the largest available GWAS of major depressive depression, bipolar disorder, and schizophrenia at the time of analysis. Negative binomial regression models were used to estimate the association between each psychiatric PGS and healthcare utilization and comorbidity burden. Healthcare utilization was measured as frequency of emergency department (ED), inpatient (IP), and outpatient (OP) visits. Comorbidity burden was defined by the Elixhauser Comorbidity Index and the Charlson Comorbidity Index.

ResultsParticipants had a median follow-up duration of 12 years in the EHR. Individuals in the top decile of polygenic score for major depressive disorder had significantly more ED visits (RR=1.22, 95% CI; 1.17, 1.29) compared to those the lowest decile. Increases were also observed with IP and comorbidity burden. Among those diagnosed with depression and in the highest decile of the PGS, there was an increase in all utilization types (ED: RR=1.56, 95% CI 1.41, 1.72; OP: RR=1.16, 95% CI 1.08, 1.24; IP: RR=1.23, 95% CI 1.12, 1.36) post-diagnosis. No clinically significant results were observed with bipolar and schizophrenia polygenic scores.

ConclusionsPolygenic score for depression is modestly associated with increased healthcare resource utilization and comorbidity burden, in the absence of diagnosis. Following a diagnosis of depression, the PGS was associated with further increases in healthcare utilization. These findings suggest that depression genetic risk is associated with utilization and burden of chronic disease in real-world settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23296145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23296145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23296145">
        <p class="paperTitle">Landscape of blood group antigens and alleles in the Indian population from whole genome sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23296145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23296145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rophina, M.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.</p>
        <p class="info">Score: 4.3, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23296145' target='https://doi.org/10.1101/2023.09.26.23296145'> 10.1101/2023.09.26.23296145</a></p>
        <p class="abstract">Blood group antigens are genetically inherited macromolecular structures which form the underlying factor for inter individual variations in human blood. Currently there exists over 390 human blood group antigens corresponding to 44 blood group systems and 2 erythroid specific transcription factors. Distribution of these blood group antigens have been found to differ significantly among various ethnic populations. To date, there is a lack of comprehensive research that offers extensive blood group profiles for the Indian population. Whole genome sequence data (hg38) of 1029 self-declared healthy Indian individuals generated as a part of the pilot phase IndiGen programme were used for the analysis. Variants spanning the genes of 44 blood group systems and two transcription factors KLF1, GATA1 were fetched and annotated for their functional consequences. Our study reports a total of 40712 blood group related variants of which 695 were identified as non-synonymous variants in the coding region. Of the total non-synonymous variants, 105 were found to have a known blood phenotype. A total of 24 variants belonging to 12 blood groups were predicted to be deleterious by more than three computational tools. Our study was also able to identify a few rare blood phenotypes including Au(a-b&#43;), Js(a&#43;b&#43;), Di(a&#43;b-), In(a&#43;b-) and KANNO-. This study is the first to use genomic data to understand the blood group antigen profiles of the Indian population, and it also systematically compares these profiles with those of other global populations.

Key pointsO_LIAccurate characterization of the genomic landscape of known and rare blood group alleles and antigens in the Indian population using the whole genome sequencing data of 1029 self-declared healthy individuals
C_LIO_LIUnderstanding the distinct similarities and differences in blood group genotypes and phenotypes across diverse global populations through systematic comparison of genomic datasets.
C_LI

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=151 SRC=&#34;FIGDIR/small/23296145v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@1dc3400org.highwire.dtl.DTLVardef@1da07a0org.highwire.dtl.DTLVardef@12a9f74org.highwire.dtl.DTLVardef@14a848d_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
