<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Impact of genome build on RNA-seq interpretation and diagnostics
Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.
Score: 33.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165
Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Impact of genome build on RNA-seq interpretation and diagnostics
Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.
Score: 33.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165
Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Impact of genome build on RNA-seq interpretation and diagnostics
Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.
Score: 33.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165
Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Impact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 33.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses.",
  "keywords": [
    
  ],
  "articleBody": " Impact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 33.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.\nEstimating Disorder Probability Based on Polygenic Prediction Using the BPC Approach\nAuthors: Uffelmann, E.; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, ; Schizophrenia Working Group of the Psychiatric Genomics Consortium, ; Price, A. L.; Posthuma, D.; Peyrot, W. J.\nScore: 20.0, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301157\nPolygenic Scores (PGSs) summarize an individual's genetic propensity for a given trait in a single value, based on SNP effect sizes derived from Genome-Wide Association Study (GWAS) results. Methods have been developed that apply Bayesian approaches to improve the prediction accuracy of PGSs through optimization of estimated effect sizes. While these methods are generally well-calibrated for continuous traits (implying the predicted values are on average equal to the true trait values), they are not well-calibrated for binary disorder traits in ascertained samples. This is a problem because well-calibrated PGSs are needed to reliably compute the absolute disorder probability for an individual to facilitate future clinical implementation. Here we introduce the Bayesian polygenic score Probability Conversion (BPC) approach, which computes an individual's predicted disorder probability using GWAS summary statistics, an existing Bayesian PGS method (e.g. PRScs, SBayesR), the individual's genotype data, and a prior disorder probability. The BPC approach transforms the PGS to its underlying liability scale, computes the variances of the PGS in cases and controls, and applies Bayes' Theorem to compute the absolute disorder probability; it is practical in its application as it does not require a tuning dataset with both genotype and phenotype data. We applied the BPC approach to extensive simulated data and empirical data of nine disorders. The BPC approach yielded well-calibrated results that were consistently better than the results of another recently published approach.\nThe Metabolic Role of MAP3K15: Genetic and Phenotypic Insights from the 23andMe Research Database and Genetics-Driven Recruitment\nAuthors: Brady, J. J.; Kalkus, K.; Nguyen, D. T.; Wen, J.; Fuller, Z. L.; Qi, Y.; Deering, S. S.; Selcer, L.; Shringarpure, S. S.; The 23andMe Research Team, ; Holmes, M. V.; Wang, X.\nScore: 13.0, Published: 2024-01-10 DOI: 10.1101/2024.01.09.24301012\nMAP3K15 has been previously associated with protection from type 2 diabetes (T2D), prompting interest in the development of MAP3K15 inhibitors as a potential therapeutic option for diabetes. The trans-ancestry genome-wide association study (GWAS) meta-analysis and loss-of-function (LoF) burden testing methods that implicate association with T2D greatly benefit from large sample size. The direct-to-consumer genetic testing company, 23andMe, Inc., is the worlds largest research consented genetic database. We leveraged the 23andMe database to further inform the metabolic role of MAP3K15, using a variety of genetic analysis methods. We find that MAP3K15 LoF carriers show a significant delay of 4.5 years in the median age of T2D diagnosis among individuals at high polygenic risk and uncover a novel burden association of MAP3K15 LoF with protection against high cholesterol. We expanded these findings by establishing a capability to recruit consented participants on the basis of genetics unknown to them (specifically, a single LoF variant in MAP3K15, rs148312150), and obtained clinical laboratory evidence of a modest reduction in median cholesterol and LDL/HDL ratio in MAP3K15 LoF carriers. Our findings demonstrate the discovery power of the 23andMe database, including the feasibility of consented participant recruitment to inform therapeutic discovery and development.\nBreaking down causes, consequences, and mediating effects of age-related telomere shortening on human health\nAuthors: Moix, S.; Sadler, M. C.; Kutalik, Z.; Auwerx, C.\nScore: 12.8, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301196\nTelomeres represent repeated DNA sequences at the ends of chromosomes, which shorten with each cell division. Factors modulating telomere attrition and the health consequences thereof are not fully understood. To address this, we leveraged data from 326,363 unrelated UK Biobank participants of European ancestry and used linear regression and bidirectional univariable and multivariable Mendelian randomization (MR and MVMR) to elucidate the relationships between leukocyte telomere length (LTL) and 141 complex traits, including diseases, biomarkers, and lifestyle factors. We confirm that telomeres shorten with age and show a stronger decline in males than in females, which cannot be explained by hormonal or lifestyle differences. MR revealed 23 traits modulating LTL; e.g., smoking cessation and high educational attainment associated with longer LTL, while weekly alcohol intake, body mass index, urate levels, and female reproductive events, such as childbirth, associated with shorter LTL. We also identified 26 traits affected by LTL, with risk for cardiovascular, pulmonary, renal, and some autoimmune diseases being increased by short LTL, while longer LTL increased risk for other autoimmune conditions and cancers. Through MVMR, we show that LTL partially mediates the impact of educational attainment, body mass index, and female age at childbirth on lifespan. These results provide new insights into the biology of telomere regulation by shedding light on the modulators, consequences, and the mediatory role of telomere shortening, portraying an intricate relationship between LTL, diseases, lifestyle, and socio-economic factors.\nConcordance of whole-genome amplified embryonic DNA with the subsequently born child\nAuthors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.\nScore: 7.5, Published: 2024-01-15 DOI: 10.1101/2024.01.12.24301086\nBefore implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here for the first time, we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGS's ability to identify de novo variants we compared the child's genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimer's Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGS's capability to identify genetic variants not explicitly targeted for monogenic screening.\nRNA methyltransferase SPOUT1/CENP-32 links mitotic spindle organization with the neurodevelopmental disorder SpADMiSS\nAuthors: Dharmadhikari, A. V.; Abad, M. A.; Khan, S.; Maroofian, R.; Sands, T. T.; Ullah, F.; Samejima, I.; Wear, M. A.; Moore, K. E.; Kondakova, E.; Mitina, N.; Schaub, T.; Lee, G. K.; Umandap, C. H.; Berger, S. M.; Iglesias, A. D.; Popp, B.; Jamra, R. A.; Gabriel, H.; Rentas, S.; Rippert, A. L.; Izumi, K.; Conlin, L. K.; Koboldt, D. C.; Mihalic Mosher, T.; Hickey, S. E.; Albert, D. V.; Norwood, H.; Lewanda, A. F.; Dai, H.; Liu, P.; Mitani, T.; Marafi, D.; Pehlivan, D.; Posey, J. E.; Lippa, N.; Vena, N.; Heinzen, E. L.; Goldstein, D. B.; Mignot, C.; Agathe, J.-M. d. S.; Al-Sannaa, N. A.; Zamani, M.; S\nScore: 7.1, Published: 2024-01-09 DOI: 10.1101/2024.01.09.23300329\nSPOUT1/CENP-32 encodes a putative SPOUT RNA methyltransferase previously identified as a mitotic chromosome associated protein. SPOUT1/CENP-32 depletion leads to centrosome detachment from the spindle poles and chromosome misalignment. Aided by gene matching platforms, we identified 24 individuals with neurodevelopmental delays from 18 families with bi-allelic variants in SPOUT1/CENP-32 detected by exome/genome sequencing. Zebrafish spout1/cenp-32 mutants showed reduction in larval head size with concomitant apoptosis likely associated with altered cell cycle progression. In vivo complementation assays in zebrafish indicated that SPOUT1/CENP-32 missense variants identified in humans are pathogenic. Crystal structure analysis of SPOUT1/CENP-32 revealed that most disease-associated missense variants mapped to the catalytic domain. Additionally, SPOUT1/CENP-32 recurrent missense variants had reduced methyltransferase activity in vitro and compromised centrosome tethering to the spindle poles in human cells. Thus, SPOUT1/CENP-32 pathogenic variants cause an autosomal recessive neurodevelopmental disorder: SpADMiSS (SPOUT1 Associated Development delay Microcephaly Seizures Short stature) underpinned by mitotic spindle organization defects and consequent chromosome segregation errors.\nEHMT2 LOSS-OF-FUNCTION ALTERATIONS CAUSE A KLEEFSTRA-LIKE SYNDROME\nAuthors: Martinez-Delgado, B.; Lopez-Martin, E.; Kerkhof, J.; Baladron, B.; Mielu, L. M.; Sanchez-Ponce, D.; Bada-Navarro, A.; Herrero-Matesanz, M.; Lopez-Jimenez, L.; Rzasa, J.; Rots, D.; Fernandez, M.; Hernandez-San Miguel, E.; Gomez-Mariano, G.; Marin-Reina, P.; Cazorla-Calleja, R.; Alonso, J.; Kleefstra, T.; Posada, M.; Bermejo-Sanchez, E.; Sadikovic, B.; Barrero, M. J.\nScore: 5.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24300997\nDysregulation of the epigenetic machinery is associated with neurodevelopmental defects in humans. Kleefstra syndrome (KS) is a neurodevelopmental syndrome caused by heterozygous alterations in the gene EHMT1 that cause loss-of-function. EHTM1 and EHMT2 are highly similar histone methyltransferases that play relevant roles in development. Despite their similarity, individuals with alterations in EHMT2 have never been described. Here, we describe a pediatric patient with a KS-overlapping phenotype and a single base de novo substitution in EHMT2 that causes the amino acid change p.Ala1077Ser in the catalytic SET domain. This change causes a reduction in the affinity of the catalytic domain for the H3 tail and in the activity of the enzyme by three- to five-fold. DNA methylation, histone methylation and gene expression profiles suggest a significant overlap between the EHMT2 p.Ala1077Ser variant and KS. Based on this evidence we suggest that EHMT2 haploinsufficiency causes a Kleefstra-like syndrome. Although we cannot rule out dominant negative effects caused by the EHMT2 p.Ala1077Ser variant, our data and previously published data suggest that loss of EHMT2 function is probably more detrimental to cells than loss of EHMT1, explaining why individuals with alterations in EHMT2 are very rare.\nPrimateAI-3D outperforms AlphaMissense in real-world cohorts\nAuthors: Parry, D. A.; Bosc, T.; Hamp, T.; Fiziev, P.; Sharma, A.; Kassam, I.; McRae, J.; Farh, K.\nScore: 5.6, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301193\nAccurately predicting the impact of genetic variants is essential for interpreting genomic data, yet no consensus exists on how to measure classifier performance. We prepared the most comprehensive set of benchmarks to date and applied them to the recently published models PrimateAI-3D and AlphaMissense. PrimateAI-3D outperforms AlphaMissense on rare-disease cohort and biobank benchmarks, indicating that performance on clinical databases or in vitro assays does not reliably generalize to real-world cohorts.\nLethal phenotypes in Mendelian disorders\nAuthors: Cacheiro, P.; Lawson, S.; Van den Veyver, I. B.; Marengo, G.; Zocche, D.; Murray, S. A.; Duyzend, M.; Robinson, P. N.; Smedley, D.\nScore: 5.1, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301168\nEssential genes are those whose function is required for cell proliferation and/or organism survival. A genes intolerance to loss-of-function can be allocated within a spectrum, as opposed to being considered a binary feature, since this function might be essential at different stages of development, genetic backgrounds or other contexts. Existing resources that collect and characterise the essentiality status of genes are based on either proliferation assessment in human cell lines, embryonic and postnatal viability evaluation in different model organisms, and gene metrics such as intolerance to variation scores derived from human population sequencing studies. There are also several repositories available that document phenotypic annotations for rare disorders in humans such as the Online Mendelian Inheritance in Man (OMIM) and the Human Phenotype Ontology (HPO) knowledgebases. This raises the prospect of being able to use clinical data, including lethality as the most severe phenotypic manifestation, to further our characterisation of gene essentiality. Here we queried OMIM for terms related to lethality and classified all Mendelian genes into categories, according to the earliest age of death recorded for the associated disorders, from prenatal death to no reports of premature death. To showcase this curated catalogue of human essential genes, we developed the Lethal Phenotypes Portal (https://lethalphenotypes.research.its.qmul.ac.uk), where we also explore the relationships between these lethality categories, constraint metrics and viability in cell lines and mouse. Further analysis of the genes in these categories reveals differences in the mode of inheritance of the associated disorders, physiological systems affected and disease class. We highlight how the phenotypic similarity between genes in the same lethality category combined with gene family/group information can be used for novel disease gene discovery. Finally, we explore the overlaps and discrepancies between the lethal phenotypes observed in mouse and human and discuss potential explanations that include differences in transcriptional regulation, functional compensation and molecular disease mechanisms. We anticipate that this resource will aid clinicians in the diagnosis of early lethal conditions and assist researchers in investigating the properties that make these genes essential for human development.\nComplex trait associations in rare diseases and impacts on Mendelian variant interpretation\nAuthors: Smail, C.; Ge, B.; Keever-Keigher, M. R.; Schwendinger-Schreck, C.; Cheung, W. A.; Johnston, J. J.; Barrett, C.; Genomic Answers for Kids Consortium, ; Feldman, K.; Cohen, A. S.; Farrow, E. G.; Thiffault, I.; Grundberg, E.; Pastinen, T.\nScore: 5.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24301111\nEmerging evidence implicates common genetic variation - aggregated into polygenic scores (PGS) - impacting the onset and phenotypic presentation of rare diseases. In this study, we quantified individual polygenic liability for 1,151 previously published PGS in a cohort of 2,374 probands enrolled in the Genomic Answers for Kids (GA4K) rare disease study, revealing widespread associations between rare disease phenotypes and PGSs for common complex diseases and traits, blood protein levels, and brain and other organ morphological measurements. We observed increased polygenic burden in probands with variants of unknown significance (VUS) compared to unaffected carrier parents. We further observed an enrichment in overlap between diagnostic and candidate rare disease genes and large-effect PGS genes. Overall, our study supports and expands on previous findings of complex trait associations in rare disease phenotypes and provides a framework for identifying novel candidate rare disease genes and in understanding variable penetrance of candidate Mendelian disease variants.\n",
  "wordCount" : "2503",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301165">
        <p class="paperTitle">Impact of genome build on RNA-seq interpretation and diagnostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network,  ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.</p>
        <p class="info">Score: 33.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301165' target='https://doi.org/10.1101/2024.01.11.24301165'> 10.1101/2024.01.11.24301165</a></p>
        <p class="abstract">Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301157" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301157">
        <p class="paperTitle">Estimating Disorder Probability Based on Polygenic Prediction Using the BPC Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uffelmann, E.; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium,  ; Schizophrenia Working Group of the Psychiatric Genomics Consortium,  ; Price, A. L.; Posthuma, D.; Peyrot, W. J.</p>
        <p class="info">Score: 20.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301157' target='https://doi.org/10.1101/2024.01.12.24301157'> 10.1101/2024.01.12.24301157</a></p>
        <p class="abstract">Polygenic Scores (PGSs) summarize an individual&#39;s genetic propensity for a given trait in a single value, based on SNP effect sizes derived from Genome-Wide Association Study (GWAS) results. Methods have been developed that apply Bayesian approaches to improve the prediction accuracy of PGSs through optimization of estimated effect sizes. While these methods are generally well-calibrated for continuous traits (implying the predicted values are on average equal to the true trait values), they are not well-calibrated for binary disorder traits in ascertained samples. This is a problem because well-calibrated PGSs are needed to reliably compute the absolute disorder probability for an individual to facilitate future clinical implementation. Here we introduce the Bayesian polygenic score Probability Conversion (BPC) approach, which computes an individual&#39;s predicted disorder probability using GWAS summary statistics, an existing Bayesian PGS method (e.g. PRScs, SBayesR), the individual&#39;s genotype data, and a prior disorder probability. The BPC approach transforms the PGS to its underlying liability scale, computes the variances of the PGS in cases and controls, and applies Bayes&#39; Theorem to compute the absolute disorder probability; it is practical in its application as it does not require a tuning dataset with both genotype and phenotype data. We applied the BPC approach to extensive simulated data and empirical data of nine disorders. The BPC approach yielded well-calibrated results that were consistently better than the results of another recently published approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.24301012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.24301012" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.24301012">
        <p class="paperTitle">The Metabolic Role of MAP3K15: Genetic and Phenotypic Insights from the 23andMe Research Database and Genetics-Driven Recruitment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.24301012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.24301012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brady, J. J.; Kalkus, K.; Nguyen, D. T.; Wen, J.; Fuller, Z. L.; Qi, Y.; Deering, S. S.; Selcer, L.; Shringarpure, S. S.; The 23andMe Research Team,  ; Holmes, M. V.; Wang, X.</p>
        <p class="info">Score: 13.0, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.24301012' target='https://doi.org/10.1101/2024.01.09.24301012'> 10.1101/2024.01.09.24301012</a></p>
        <p class="abstract">MAP3K15 has been previously associated with protection from type 2 diabetes (T2D), prompting interest in the development of MAP3K15 inhibitors as a potential therapeutic option for diabetes. The trans-ancestry genome-wide association study (GWAS) meta-analysis and loss-of-function (LoF) burden testing methods that implicate association with T2D greatly benefit from large sample size. The direct-to-consumer genetic testing company, 23andMe, Inc., is the worlds largest research consented genetic database. We leveraged the 23andMe database to further inform the metabolic role of MAP3K15, using a variety of genetic analysis methods. We find that MAP3K15 LoF carriers show a significant delay of 4.5 years in the median age of T2D diagnosis among individuals at high polygenic risk and uncover a novel burden association of MAP3K15 LoF with protection against high cholesterol. We expanded these findings by establishing a capability to recruit consented participants on the basis of genetics unknown to them (specifically, a single LoF variant in MAP3K15, rs148312150), and obtained clinical laboratory evidence of a modest reduction in median cholesterol and LDL/HDL ratio in MAP3K15 LoF carriers. Our findings demonstrate the discovery power of the 23andMe database, including the feasibility of consented participant recruitment to inform therapeutic discovery and development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301196" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301196">
        <p class="paperTitle">Breaking down causes, consequences, and mediating effects of age-related telomere shortening on human health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moix, S.; Sadler, M. C.; Kutalik, Z.; Auwerx, C.</p>
        <p class="info">Score: 12.8, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301196' target='https://doi.org/10.1101/2024.01.12.24301196'> 10.1101/2024.01.12.24301196</a></p>
        <p class="abstract">Telomeres represent repeated DNA sequences at the ends of chromosomes, which shorten with each cell division. Factors modulating telomere attrition and the health consequences thereof are not fully understood. To address this, we leveraged data from 326,363 unrelated UK Biobank participants of European ancestry and used linear regression and bidirectional univariable and multivariable Mendelian randomization (MR and MVMR) to elucidate the relationships between leukocyte telomere length (LTL) and 141 complex traits, including diseases, biomarkers, and lifestyle factors. We confirm that telomeres shorten with age and show a stronger decline in males than in females, which cannot be explained by hormonal or lifestyle differences. MR revealed 23 traits modulating LTL; e.g., smoking cessation and high educational attainment associated with longer LTL, while weekly alcohol intake, body mass index, urate levels, and female reproductive events, such as childbirth, associated with shorter LTL. We also identified 26 traits affected by LTL, with risk for cardiovascular, pulmonary, renal, and some autoimmune diseases being increased by short LTL, while longer LTL increased risk for other autoimmune conditions and cancers. Through MVMR, we show that LTL partially mediates the impact of educational attainment, body mass index, and female age at childbirth on lifespan. These results provide new insights into the biology of telomere regulation by shedding light on the modulators, consequences, and the mediatory role of telomere shortening, portraying an intricate relationship between LTL, diseases, lifestyle, and socio-economic factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301086" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301086">
        <p class="paperTitle">Concordance of whole-genome amplified embryonic DNA with the subsequently born child</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.</p>
        <p class="info">Score: 7.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301086' target='https://doi.org/10.1101/2024.01.12.24301086'> 10.1101/2024.01.12.24301086</a></p>
        <p class="abstract">Before implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here for the first time, we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGS&#39;s ability to identify de novo variants we compared the child&#39;s genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimer&#39;s Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGS&#39;s capability to identify genetic variants not explicitly targeted for monogenic screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.23300329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.23300329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.23300329">
        <p class="paperTitle">RNA methyltransferase SPOUT1/CENP-32 links mitotic spindle organization with the neurodevelopmental disorder SpADMiSS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.23300329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.23300329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dharmadhikari, A. V.; Abad, M. A.; Khan, S.; Maroofian, R.; Sands, T. T.; Ullah, F.; Samejima, I.; Wear, M. A.; Moore, K. E.; Kondakova, E.; Mitina, N.; Schaub, T.; Lee, G. K.; Umandap, C. H.; Berger, S. M.; Iglesias, A. D.; Popp, B.; Jamra, R. A.; Gabriel, H.; Rentas, S.; Rippert, A. L.; Izumi, K.; Conlin, L. K.; Koboldt, D. C.; Mihalic Mosher, T.; Hickey, S. E.; Albert, D. V.; Norwood, H.; Lewanda, A. F.; Dai, H.; Liu, P.; Mitani, T.; Marafi, D.; Pehlivan, D.; Posey, J. E.; Lippa, N.; Vena, N.; Heinzen, E. L.; Goldstein, D. B.; Mignot, C.; Agathe, J.-M. d. S.; Al-Sannaa, N. A.; Zamani, M.; S</p>
        <p class="info">Score: 7.1, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.23300329' target='https://doi.org/10.1101/2024.01.09.23300329'> 10.1101/2024.01.09.23300329</a></p>
        <p class="abstract">SPOUT1/CENP-32 encodes a putative SPOUT RNA methyltransferase previously identified as a mitotic chromosome associated protein. SPOUT1/CENP-32 depletion leads to centrosome detachment from the spindle poles and chromosome misalignment. Aided by gene matching platforms, we identified 24 individuals with neurodevelopmental delays from 18 families with bi-allelic variants in SPOUT1/CENP-32 detected by exome/genome sequencing. Zebrafish spout1/cenp-32 mutants showed reduction in larval head size with concomitant apoptosis likely associated with altered cell cycle progression. In vivo complementation assays in zebrafish indicated that SPOUT1/CENP-32 missense variants identified in humans are pathogenic. Crystal structure analysis of SPOUT1/CENP-32 revealed that most disease-associated missense variants mapped to the catalytic domain. Additionally, SPOUT1/CENP-32 recurrent missense variants had reduced methyltransferase activity in vitro and compromised centrosome tethering to the spindle poles in human cells. Thus, SPOUT1/CENP-32 pathogenic variants cause an autosomal recessive neurodevelopmental disorder: SpADMiSS (SPOUT1 Associated Development delay Microcephaly Seizures Short stature) underpinned by mitotic spindle organization defects and consequent chromosome segregation errors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24300997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24300997" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24300997">
        <p class="paperTitle">EHMT2 LOSS-OF-FUNCTION ALTERATIONS CAUSE A KLEEFSTRA-LIKE SYNDROME</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24300997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24300997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Delgado, B.; Lopez-Martin, E.; Kerkhof, J.; Baladron, B.; Mielu, L. M.; Sanchez-Ponce, D.; Bada-Navarro, A.; Herrero-Matesanz, M.; Lopez-Jimenez, L.; Rzasa, J.; Rots, D.; Fernandez, M.; Hernandez-San Miguel, E.; Gomez-Mariano, G.; Marin-Reina, P.; Cazorla-Calleja, R.; Alonso, J.; Kleefstra, T.; Posada, M.; Bermejo-Sanchez, E.; Sadikovic, B.; Barrero, M. J.</p>
        <p class="info">Score: 5.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24300997' target='https://doi.org/10.1101/2024.01.10.24300997'> 10.1101/2024.01.10.24300997</a></p>
        <p class="abstract">Dysregulation of the epigenetic machinery is associated with neurodevelopmental defects in humans. Kleefstra syndrome (KS) is a neurodevelopmental syndrome caused by heterozygous alterations in the gene EHMT1 that cause loss-of-function. EHTM1 and EHMT2 are highly similar histone methyltransferases that play relevant roles in development. Despite their similarity, individuals with alterations in EHMT2 have never been described. Here, we describe a pediatric patient with a KS-overlapping phenotype and a single base de novo substitution in EHMT2 that causes the amino acid change p.Ala1077Ser in the catalytic SET domain. This change causes a reduction in the affinity of the catalytic domain for the H3 tail and in the activity of the enzyme by three- to five-fold. DNA methylation, histone methylation and gene expression profiles suggest a significant overlap between the EHMT2 p.Ala1077Ser variant and KS. Based on this evidence we suggest that EHMT2 haploinsufficiency causes a Kleefstra-like syndrome. Although we cannot rule out dominant negative effects caused by the EHMT2 p.Ala1077Ser variant, our data and previously published data suggest that loss of EHMT2 function is probably more detrimental to cells than loss of EHMT1, explaining why individuals with alterations in EHMT2 are very rare.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301193" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301193">
        <p class="paperTitle">PrimateAI-3D outperforms AlphaMissense in real-world cohorts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parry, D. A.; Bosc, T.; Hamp, T.; Fiziev, P.; Sharma, A.; Kassam, I.; McRae, J.; Farh, K.</p>
        <p class="info">Score: 5.6, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301193' target='https://doi.org/10.1101/2024.01.12.24301193'> 10.1101/2024.01.12.24301193</a></p>
        <p class="abstract">Accurately predicting the impact of genetic variants is essential for interpreting genomic data, yet no consensus exists on how to measure classifier performance. We prepared the most comprehensive set of benchmarks to date and applied them to the recently published models PrimateAI-3D and AlphaMissense. PrimateAI-3D outperforms AlphaMissense on rare-disease cohort and biobank benchmarks, indicating that performance on clinical databases or in vitro assays does not reliably generalize to real-world cohorts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301168" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301168">
        <p class="paperTitle">Lethal phenotypes in Mendelian disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cacheiro, P.; Lawson, S.; Van den Veyver, I. B.; Marengo, G.; Zocche, D.; Murray, S. A.; Duyzend, M.; Robinson, P. N.; Smedley, D.</p>
        <p class="info">Score: 5.1, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301168' target='https://doi.org/10.1101/2024.01.12.24301168'> 10.1101/2024.01.12.24301168</a></p>
        <p class="abstract">Essential genes are those whose function is required for cell proliferation and/or organism survival. A genes intolerance to loss-of-function can be allocated within a spectrum, as opposed to being considered a binary feature, since this function might be essential at different stages of development, genetic backgrounds or other contexts. Existing resources that collect and characterise the essentiality status of genes are based on either proliferation assessment in human cell lines, embryonic and postnatal viability evaluation in different model organisms, and gene metrics such as intolerance to variation scores derived from human population sequencing studies. There are also several repositories available that document phenotypic annotations for rare disorders in humans such as the Online Mendelian Inheritance in Man (OMIM) and the Human Phenotype Ontology (HPO) knowledgebases. This raises the prospect of being able to use clinical data, including lethality as the most severe phenotypic manifestation, to further our characterisation of gene essentiality. Here we queried OMIM for terms related to lethality and classified all Mendelian genes into categories, according to the earliest age of death recorded for the associated disorders, from prenatal death to no reports of premature death. To showcase this curated catalogue of human essential genes, we developed the Lethal Phenotypes Portal (https://lethalphenotypes.research.its.qmul.ac.uk), where we also explore the relationships between these lethality categories, constraint metrics and viability in cell lines and mouse. Further analysis of the genes in these categories reveals differences in the mode of inheritance of the associated disorders, physiological systems affected and disease class. We highlight how the phenotypic similarity between genes in the same lethality category combined with gene family/group information can be used for novel disease gene discovery. Finally, we explore the overlaps and discrepancies between the lethal phenotypes observed in mouse and human and discuss potential explanations that include differences in transcriptional regulation, functional compensation and molecular disease mechanisms. We anticipate that this resource will aid clinicians in the diagnosis of early lethal conditions and assist researchers in investigating the properties that make these genes essential for human development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24301111">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24301111" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24301111">
        <p class="paperTitle">Complex trait associations in rare diseases and impacts on Mendelian variant interpretation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24301111" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24301111" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smail, C.; Ge, B.; Keever-Keigher, M. R.; Schwendinger-Schreck, C.; Cheung, W. A.; Johnston, J. J.; Barrett, C.; Genomic Answers for Kids Consortium,  ; Feldman, K.; Cohen, A. S.; Farrow, E. G.; Thiffault, I.; Grundberg, E.; Pastinen, T.</p>
        <p class="info">Score: 5.0, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24301111' target='https://doi.org/10.1101/2024.01.10.24301111'> 10.1101/2024.01.10.24301111</a></p>
        <p class="abstract">Emerging evidence implicates common genetic variation - aggregated into polygenic scores (PGS) - impacting the onset and phenotypic presentation of rare diseases. In this study, we quantified individual polygenic liability for 1,151 previously published PGS in a cohort of 2,374 probands enrolled in the Genomic Answers for Kids (GA4K) rare disease study, revealing widespread associations between rare disease phenotypes and PGSs for common complex diseases and traits, blood protein levels, and brain and other organ morphological measurements. We observed increased polygenic burden in probands with variants of unknown significance (VUS) compared to unaffected carrier parents. We further observed an enrichment in overlap between diagnostic and candidate rare disease genes and large-effect PGS genes. Overall, our study supports and expands on previous findings of complex trait associations in rare disease phenotypes and provides a framework for identifying novel candidate rare disease genes and in understanding variable penetrance of candidate Mendelian disease variants.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
