<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 27.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 27.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 27.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 27.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations.",
  "keywords": [
    
  ],
  "articleBody": " Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 27.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at v2.evemodel.org.\nGenetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes\nAuthors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.\nScore: 23.2, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23299014\nThe biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).\nEvaluation of polygenic scoring methods in five biobanks reveals greater variability between biobanks than between methods and highlights benefits of ensemble learning\nAuthors: Monti, R.; Eick, L.; Hudjashov, G.; Läll, K.; Kanoni, S.; Wolford, B. N.; Wingfield, B.; Pain, O.; Wharrie, S.; Jermy, B.; McMahon, A.; Hartonen, T.; Heyne, H. O.; Mars, N.; Genes \u0026 Health Research Team, ; Hveem, K.; Inouye, M.; van Heel, D. A.; Mägi, R.; Marttinen, P.; Ripatti, S.; Ganna, A.; Lippert, C.\nScore: 22.0, Published: 2023-11-20 DOI: 10.1101/2023.11.20.23298215\nMethods to estimate polygenic scores (PGS) from genome-wide association studies are increasingly utilized. However, independent method evaluation is lacking, and method comparisons are often limited. Here, we evaluate polygenic scores derived using seven methods in five biobank studies (totaling about 1.2 million participants) across 16 diseases and quantitative traits, building on a reference-standardized framework. We conducted meta-analyses to quantify the effects of method choice, hyperparameter tuning, method ensembling and target biobank on PGS performance. We found that no single method consistently outperformed all others. PGS effect sizes were more variable between biobanks than between methods within biobanks when methods were well-tuned. Differences between methods were largest for the two investigated autoimmune diseases, seropositive rheumatoid arthritis and type 1 diabetes. For most methods, cross-validation was more reliable for tuning hyperparameters than automatic tuning (without the use of target data). For a given target phenotype, elastic net models combining PGS across methods (ensemble PGS) tuned in the UK Biobank provided consistent, high, and cross-biobank transferable performance, increasing PGS effect sizes ({beta}-coefficients) by a median of 5.0% relative to LDpred2 and MegaPRS (the two best performing single methods when tuned with cross-validation). Our interactively browsable online-results (https://methodscomparison.intervenegeneticscores.org/) and open-source workflow prspipe (https://github.com/intervene-EU-H2020/prspipe) provide a rich resource and reference for the analysis of polygenic scoring methods across biobanks.\nGenetic analysis of 104 pregnancy phenotypes in 39,194 Chinese women\nAuthors: Xiao, H.; Li, L.; Yang, M.; Zeng, J.; Zhou, J.; Tao, Y.; Zhou, Y.; Cai, M.; Liu, J.; Huang, Y.; Zhong, Y.; Liu, P.; Cao, Z.; Mei, H.; Cai, X.; Hu, L.; Zhou, R.; Xu, X.; Yang, H.; Wang, J.; Zhu, H.; Zhou, A.; Jin, X.\nScore: 8.8, Published: 2023-11-24 DOI: 10.1101/2023.11.23.23298979\nMaternity is a special period in a womans life that involves substantial physiological, psychological, and hormonal changes. These changes may cause alterations in many clinical measurements during pregnancy, which can be used to monitor and diagnose maternal disorders and adverse postnatal outcomes. Exploring the genetic background of these phenotypes is key to elucidating the pathogenesis of pregnancy disorders. In this study, we conducted a large-scale molecular biology analysis of 104 pregnancy phenotypes based on genotype data from 39,194 Chinses women. Genome- wide association analysis identified a total of 407 trait-locus associations, of which 75.18% were previously reported. Among the 101 novel associations for 37 phenotypes, some were potentially pregnancy-specific and worth further experimental investigation. For example, ESR1 with fasting glucose, hemoglobin, hematocrit, and several leukocytoses; ZSCAN31 with blood urea nitrogen. We further performed pathway- based analysis and uncovered at least one significant pathway for 24 traits, in addition to previously known functional pathways, novel findings included birthweight with \"Reactome signaling by NODAL\", twin pregnancy with \"Reactome mitotic G1-G1/S phases\". The partitioning heritability analysis recapitulated known trait-relevant tissue/cell types, and also discovered interesting results including twin pregnancy with \"embryoid bodies\" cell-type enrichment, the delivery type cesarean section with \"fallopian tube\", and birth weight with \"ovary and embryonic stem cells\". In terms of both sample size and the variety of phenotypes, our work is one of the largest genetic studies of pregnancy phenotypes across all populations. We believe that this study will provide a valuable resource for exploring the genetic background of pregnancy phenotypes and also for further research on pregnancy-related diseases and adverse neonatal outcomes.\nPolygenic burden of short tandem repeat expansions promote risk for Alzheimer's disease\nAuthors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.\nScore: 29.5, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623\nStudies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation. There are over one million short tandem repeats (STRs) in the genome, and their link to AD risk has not been assessed. As pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk. Here, we genotyped 321,742 polymorphic STR tracts genome-wide using PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals). We implemented an approach to identify STR expansions as STRs with tract lengths that are outliers from the population. We then tested for differences in aggregate burden of expansions in case versus control individuals. AD patients had a 1.19-fold increase of STR expansions compared to healthy elderly controls (p=8.27x10-3, two-sided Mann Whitney test). Individuals carrying \u003e 30 STR expansions had 3.62-fold higher odds of having AD and had more severe AD neuropathology. AD STR expansions were highly enriched within active promoters in post-mortem hippocampal brain tissues and particularly within SINE-VNTR-Alu (SVA) retrotransposons. Together, these results demonstrate that expanded STRs within active promoter regions of the genome promote risk of AD.\nA validated heart-specific model for splice-disrupting variants in childhood heart disease\nAuthors: Lesurf, R.; Breckpot, J.; Bouwmeester, J.; Hanafi, N.; Jain, A.; Liang, Y.; Papaz, T.; Lougheed, J.; Mondal, T.; Alsalehi, M.; Altamirano-Diaz, L.; Oechslin, E.; Audain, E.; Dombrowsky, G.; Postma, A. V.; Woudstra, O. I.; Bouma, B. J.; Hitz, M.-P.; Bezzina, C. R.; Blue, G.; Winlaw, D. S.; Mital, S.\nScore: 3.5, Published: 2023-11-27 DOI: 10.1101/2023.11.23.23298903\nCongenital heart disease (CHD) is the most common congenital anomaly. Non-canonical splice-disrupting variants are not routinely evaluated by clinical tests. Algorithms including SpliceAI predict such variants, but are not specific to cardiac-expressed genes. Whole genome (WGS) (n=1083) and myocardial RNA-Sequencing (RNA-Seq) (n=114) of CHD cases was used to identify splice-disrupting variants. Using features of variants confirmed to affect splicing in myocardial RNA, we trained a machine learning model that outperformed SpliceAI for predicting cardiac-specific splice-disrupting variants (AUC 0.92 vs 0.66), and was independently validated in 43 cardiomyopathy probands (AUC 0.88 vs 0.64). Application of this model to 971 CHD WGS samples identified 9% patients with splice-disrupting variants in CHD genes. Forty-one% of predicted splice-disrupting variants were deeply intronic. The burden of variants in CHD genes was higher in cases compared with 2,570 controls. Our model improved genetic yield by identifying splice-disrupting variants that are not evaluated by routine tests.\nA transgenerational mutational signature from ionizing radiation exposure.\nAuthors: Brand, F.; Klinkhammer, H.; Knaus, A.; Holtgrewe, M.; Weinhold, L.; Beule, D.; Ludwig, K.; Kothiyal, P.; Maxwell, G.; Noethen, M.; Schmid, M.; Sperling, K.; Krawitz, P.\nScore: 2.2, Published: 2023-11-20 DOI: 10.1101/2023.11.20.23298689\nEffects of accidental radiation exposure on the human germline remain a topic of medical interest. Sequencing technology now allows the accurate analysis of the entire human genome in potentially exposed cohorts to search for possible signatures of ionizing radiation exposure. Clustered de novo mutations (cDNMs), that are multiple DNA lesions within 20 bp of each other, have been hypothesized to be a signature for paternal exposure to ionizing radiation. We analyzed whole genome trio data of 240 children and their likely exposed parents as well as 1,275 offspring from unexposed families. The cohort of exposed children consists of 130 offspring of Chernobyl cleanup workers (CRU, exposure range = 0-4,080 mSv) and 110 offspring of former soldiers from both german armies which have likely been irradiated during their service (Radar cohort, exposure range = 0-353 mSv). For the Radar cohort, a retrospective dosage estimation was conducted based on the service records from each soldier, but it remains a challenging task to retrospectively estimate received dose accurately. In agreement with previous epidemiological data, no transgenerational effects could be detected for the class of isolated de novo mutations, which cause the majority of genetic disorders. We found that the number of isolated de novo mutations increases by 2% per year of age of the father at conception, which is also in line with previous works. On average, we observed 2.65 cDNMs per offspring in the CRU cohort, 1.48 in the Radar cohort and 0.88 in the control cohort, with a median of two clustered mutations per genome in offspring of irradiated fathers. This is a significant increase (p \u003c 0.005) in the number of cDNMs compared to a set of age-matched controls. Furthermore, we show that the cDNM rates scale with paternal exposure to ionizing radiation (p \u003c 0.001). Our findings corroborate that clustered de novo mutations represent a transgenerational biomarker for paternal exposure to ionizing radiation.\nNetwork medicine-based epistasis detection in complex diseases: ready for quantum computing\nAuthors: Hoffmann, M.; Incudini, M.; Poschenrieder, J. M.; Baier, S.; Fitz, A.; Maier, A.; Hartung, M.; Hoffmann, C. M.; Trummer, N.; Adamowicz, K.; Picciani, M.; Scheibling, E.; Harl, M.; Lesch, I.; Frey, H.; Kayser, S.; Wissenberg, P.; Schwartz, L.; Hafner, L.; Acharya, A.; Hackl, L.; Grabert, G.; Lee, S.-G.; Cho, G.; Cloward, M. E.; Jankowski, J.; Lee, H. K. D.; Tsoy, O.; Wenke, N.; Pedersen, A. G.; Bonnelykke, K.; Mandarino, A.; Melograna, F.; Schulz, L.; Climente-Gonzalez, H.; Wilhelm, M.; Iapichino, L.; Wienbrandt, L.; Ellinghaus, D.; Van Steen, K.; Grossi, M.; Furth, P. A.; Hennighausen, L.; Di Score: 7.6, Published: 2023-11-09 DOI: 10.1101/2023.11.07.23298205\nMost heritable diseases are polygenic. To comprehend the underlying genetic architecture, it is crucial to discover the clinically relevant epistatic interactions (EIs) between genomic single nucleotide polymorphisms (SNPs)1-3. Existing statistical computational methods for EI detection are mostly limited to pairs of SNPs due to the combinatorial explosion of higher-order EIs. With NeEDL (network-based epistasis detection via local search), we leverage network medicine to inform the selection of EIs that are an order of magnitude more statistically significant compared to existing tools and consist, on average, of five SNPs. We further show that this computationally demanding task can be substantially accelerated once quantum computing hardware becomes available. We apply NeEDL to eight different diseases and discover genes (affected by EIs of SNPs) that are partly known to affect the disease, additionally, these results are reproducible across independent cohorts. EIs for these eight diseases can be interactively explored in the Epistasis Disease Atlas (https://epistasis-disease-atlas.com). In summary, NeEDL is the first application that demonstrates the potential of seamlessly integrated quantum computing techniques to accelerate biomedical research. Our network medicine approach detects higher-order EIs with unprecedented statistical and biological evidence, yielding unique insights into polygenic diseases and providing a basis for the development of improved risk scores and combination therapies.\nMulti-trait genome-wide association study in 34,394 Chinese women reveals the genetic architecture of plasma metabolites during pregnancy\nAuthors: Liu, S.; Yao, J.; Lin, L.; Lan, X.; Wu, L.; Kong, N.; Deng, Y.; Li, Y.; Xie, J.; Zhu, H.; Wu, X.; Li, Z.; Xiong, L.; Wang, Y.; Ren, J.; Qiu, X.; Zhao, W.; Gao, Y.; Chen, Y.; Su, F.; Zhou, Y.; Rao, W.; Zhang, J.; Hou, G.; Huang, L.; Li, L.; Liu, X.; Nie, C.; Luo, L.; Liu, Z.; Chen, F.; Lin, S.; Zhao, L.; Fu, Q.; Jiang, D.; Yin, Y.; Xu, X.; Wang, J.; Yang, H.; Wang, R.; Niu, J.; Wei, F.; Jin, X.; Liu, S.\nScore: 1.8, Published: 2023-11-27 DOI: 10.1101/2023.11.26.23299022\nMetabolites are important indicators of individual health and can serve as crucial targets for therapy. However, the genetic basis of many metabolites remains largely unexplored, especially among underrepresented East Asians and during critical periods such as pregnancy. In this study, we utilized genetic information obtained from non-invasive prenatal testing to conduct a genome-wide association analysis of 84 metabolites, including 37 amino acids, 10 vitamins, 24 metal elements, and 13 hormones, among 34,394 Chinese pregnant women. Of these metabolites, 52 and 11 had not previously been studied in East Asians or globally. We identified 30 novel metabolite-gene associations. We also observed substantial differences in the genetic effects on hormones between pregnancy and non-pregnancy periods, suggesting effect modifications in response to physiological changes. Furthermore, we uncovered pervasive pleiotropic effects for 50.94% of the genetic associations among metabolites, as well as between six metabolites and eight pregnancy biomarkers. Using mendelian randomization, we identified potential causal relationships between plasma folate and ischemic stroke, vitamin D3 and Graves' disease, copper and open-angle glaucoma, and androstenedione and rheumatoid arthritis. These discoveries provide invaluable genetic insights into human metabolism, laying the foundation for future mechanistic studies and the development of new therapeutic targets, particularly for underrepresented East Asians.\nA comprehensive characterization of blood group antigen variants in the Middle Eastern population genomes - Insights into genetic epidemiology\nAuthors: Rophina, M.; Pandhare, K. R.; Scaria, V.\nScore: 1.8, Published: 2023-11-20 DOI: 10.1101/2023.11.20.23298777\nBackgroundThe Middle Eastern population is characterized by increased prevalence of various Mendelian disorders owing to increased rates of consanguinity. Especially in disease conditions which require chronic transfusion support, it becomes important to know the blood group characteristics of potential donors to increase the likelihood of transfusion success. As there prevails a gap in knowledge about the population specific overall blood antigen profiles, this study seeks to utilize next generation sequencing datasets to unravel the comprehensive landscape of clinically significant minor blood group alleles in the middle eastern population. MethodsThis study utilizes the genetic variation data from a range of public datasets including the Greater Middle East Variome, the Qatar genome and exomes and the Iranome datasets to estimate the genotypic and phenotypic frequencies of blood group alleles in the Middle Easterners. The estimated frequencies were duly compared with major global populations to identify significant similarities or differences if any. ResultsA total of 77 unique ISBT approved blood group alleles were found commonly in all datasets. 8 variants (rs8176058, rs1058396, rs565898944, rs28362692, rs2071699, rs34783571, rs60322991 and rs57467915) belonging to KELL, KIDD, COLTON, H, JUNIOR and LANGEREIS blood groups were found clinically significant with previously reported evidence on transfusion complications. 730 variants were found to span exonic or splicing regions out of which 70 were predicted to be potentially deleterious by at least four computational tools. ConclusionsThis study serves first of its kind to extensively characterize the known and novel blood alleles in the Middle Easterners. A comprehensive user-friendly online resource named alnasab - Alleles and antigens in Arab and Persian populations associated with blood groups was also developed as a dependable reference for future transfusion research. The resource is accessible at https://clingen.igib.res.in/alnasab/ Key pointsO_LILarge scale Middle Eastern population sequencing datasets including The Greater Middle Eastern Variome, Genomes and exomes from Qatar and the Iranome datasets were used in the study. C_LIO_LIA total of 2828 exomes and 88 genomes were analyzed accounting for a total of C_LIO_LI18717 unique human blood group related variants. C_LIO_LI2443 exonic variants were extracted which systematically included 1505 non-synonymous variants, 766 synonymous variants, 50 stopgain variants and 3 stop loss variants. C_LIO_LIBlood group associated variants identified in the study are provided as a comprehensive online repository - alnasab, Alleles and antigens in Arab and Persian populations associated with blood groups. The resource is accessible at https://clingen.igib.res.in/alnasab/ C_LI Visual abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=104 SRC=\"FIGDIR/small/23298777v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (21K): org.highwire.dtl.DTLVardef@51dcb0org.highwire.dtl.DTLVardef@6fc1a5org.highwire.dtl.DTLVardef@1126e3aorg.highwire.dtl.DTLVardef@102ca64_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "3034",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23299062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23299062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23299062">
        <p class="paperTitle">Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23299062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23299062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.</p>
        <p class="info">Score: 27.0, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23299062' target='https://doi.org/10.1101/2023.11.27.23299062'> 10.1101/2023.11.27.23299062</a></p>
        <p class="abstract">Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at v2.evemodel.org.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.23299014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.23299014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.23299014">
        <p class="paperTitle">Genetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.23299014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.23299014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.</p>
        <p class="info">Score: 23.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.23299014' target='https://doi.org/10.1101/2023.11.25.23299014'> 10.1101/2023.11.25.23299014</a></p>
        <p class="abstract">The biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298215">
        <p class="paperTitle">Evaluation of polygenic scoring methods in five biobanks reveals greater variability between biobanks than between methods and highlights benefits of ensemble learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monti, R.; Eick, L.; Hudjashov, G.; Läll, K.; Kanoni, S.; Wolford, B. N.; Wingfield, B.; Pain, O.; Wharrie, S.; Jermy, B.; McMahon, A.; Hartonen, T.; Heyne, H. O.; Mars, N.; Genes &amp; Health Research Team,  ; Hveem, K.; Inouye, M.; van Heel, D. A.; Mägi, R.; Marttinen, P.; Ripatti, S.; Ganna, A.; Lippert, C.</p>
        <p class="info">Score: 22.0, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298215' target='https://doi.org/10.1101/2023.11.20.23298215'> 10.1101/2023.11.20.23298215</a></p>
        <p class="abstract">Methods to estimate polygenic scores (PGS) from genome-wide association studies are increasingly utilized. However, independent method evaluation is lacking, and method comparisons are often limited. Here, we evaluate polygenic scores derived using seven methods in five biobank studies (totaling about 1.2 million participants) across 16 diseases and quantitative traits, building on a reference-standardized framework. We conducted meta-analyses to quantify the effects of method choice, hyperparameter tuning, method ensembling and target biobank on PGS performance. We found that no single method consistently outperformed all others. PGS effect sizes were more variable between biobanks than between methods within biobanks when methods were well-tuned. Differences between methods were largest for the two investigated autoimmune diseases, seropositive rheumatoid arthritis and type 1 diabetes. For most methods, cross-validation was more reliable for tuning hyperparameters than automatic tuning (without the use of target data). For a given target phenotype, elastic net models combining PGS across methods (ensemble PGS) tuned in the UK Biobank provided consistent, high, and cross-biobank transferable performance, increasing PGS effect sizes ({beta}-coefficients) by a median of 5.0% relative to LDpred2 and MegaPRS (the two best performing single methods when tuned with cross-validation). Our interactively browsable online-results (https://methodscomparison.intervenegeneticscores.org/) and open-source workflow prspipe (https://github.com/intervene-EU-H2020/prspipe) provide a rich resource and reference for the analysis of polygenic scoring methods across biobanks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.23298979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.23298979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.23298979">
        <p class="paperTitle">Genetic analysis of 104 pregnancy phenotypes in 39,194 Chinese women</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.23298979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.23298979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xiao, H.; Li, L.; Yang, M.; Zeng, J.; Zhou, J.; Tao, Y.; Zhou, Y.; Cai, M.; Liu, J.; Huang, Y.; Zhong, Y.; Liu, P.; Cao, Z.; Mei, H.; Cai, X.; Hu, L.; Zhou, R.; Xu, X.; Yang, H.; Wang, J.; Zhu, H.; Zhou, A.; Jin, X.</p>
        <p class="info">Score: 8.8, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.23298979' target='https://doi.org/10.1101/2023.11.23.23298979'> 10.1101/2023.11.23.23298979</a></p>
        <p class="abstract">Maternity is a special period in a womans life that involves substantial physiological, psychological, and hormonal changes. These changes may cause alterations in many clinical measurements during pregnancy, which can be used to monitor and diagnose maternal disorders and adverse postnatal outcomes. Exploring the genetic background of these phenotypes is key to elucidating the pathogenesis of pregnancy disorders. In this study, we conducted a large-scale molecular biology analysis of 104 pregnancy phenotypes based on genotype data from 39,194 Chinses women. Genome- wide association analysis identified a total of 407 trait-locus associations, of which 75.18% were previously reported. Among the 101 novel associations for 37 phenotypes, some were potentially pregnancy-specific and worth further experimental investigation. For example, ESR1 with fasting glucose, hemoglobin, hematocrit, and several leukocytoses; ZSCAN31 with blood urea nitrogen. We further performed pathway- based analysis and uncovered at least one significant pathway for 24 traits, in addition to previously known functional pathways, novel findings included birthweight with &#34;Reactome signaling by NODAL&#34;, twin pregnancy with &#34;Reactome mitotic G1-G1/S phases&#34;. The partitioning heritability analysis recapitulated known trait-relevant tissue/cell types, and also discovered interesting results including twin pregnancy with &#34;embryoid bodies&#34; cell-type enrichment, the delivery type cesarean section with &#34;fallopian tube&#34;, and birth weight with &#34;ovary and embryonic stem cells&#34;. In terms of both sample size and the variety of phenotypes, our work is one of the largest genetic studies of pregnancy phenotypes across all populations. We believe that this study will provide a valuable resource for exploring the genetic background of pregnancy phenotypes and also for further research on pregnancy-related diseases and adverse neonatal outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.23298623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.23298623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.23298623">
        <p class="paperTitle">Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.23298623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.23298623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.</p>
        <p class="info">Score: 29.5, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.23298623' target='https://doi.org/10.1101/2023.11.16.23298623'> 10.1101/2023.11.16.23298623</a></p>
        <p class="abstract">Studies of the genetics of Alzheimers disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation. There are over one million short tandem repeats (STRs) in the genome, and their link to AD risk has not been assessed. As pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk. Here, we genotyped 321,742 polymorphic STR tracts genome-wide using PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals). We implemented an approach to identify STR expansions as STRs with tract lengths that are outliers from the population. We then tested for differences in aggregate burden of expansions in case versus control individuals. AD patients had a 1.19-fold increase of STR expansions compared to healthy elderly controls (p=8.27x10-3, two-sided Mann Whitney test). Individuals carrying &gt; 30 STR expansions had 3.62-fold higher odds of having AD and had more severe AD neuropathology. AD STR expansions were highly enriched within active promoters in post-mortem hippocampal brain tissues and particularly within SINE-VNTR-Alu (SVA) retrotransposons. Together, these results demonstrate that expanded STRs within active promoter regions of the genome promote risk of AD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.23298903">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.23298903" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.23298903">
        <p class="paperTitle">A validated heart-specific model for splice-disrupting variants in childhood heart disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.23298903" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.23298903" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lesurf, R.; Breckpot, J.; Bouwmeester, J.; Hanafi, N.; Jain, A.; Liang, Y.; Papaz, T.; Lougheed, J.; Mondal, T.; Alsalehi, M.; Altamirano-Diaz, L.; Oechslin, E.; Audain, E.; Dombrowsky, G.; Postma, A. V.; Woudstra, O. I.; Bouma, B. J.; Hitz, M.-P.; Bezzina, C. R.; Blue, G.; Winlaw, D. S.; Mital, S.</p>
        <p class="info">Score: 3.5, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.23298903' target='https://doi.org/10.1101/2023.11.23.23298903'> 10.1101/2023.11.23.23298903</a></p>
        <p class="abstract">Congenital heart disease (CHD) is the most common congenital anomaly. Non-canonical splice-disrupting variants are not routinely evaluated by clinical tests. Algorithms including SpliceAI predict such variants, but are not specific to cardiac-expressed genes. Whole genome (WGS) (n=1083) and myocardial RNA-Sequencing (RNA-Seq) (n=114) of CHD cases was used to identify splice-disrupting variants. Using features of variants confirmed to affect splicing in myocardial RNA, we trained a machine learning model that outperformed SpliceAI for predicting cardiac-specific splice-disrupting variants (AUC 0.92 vs 0.66), and was independently validated in 43 cardiomyopathy probands (AUC 0.88 vs 0.64). Application of this model to 971 CHD WGS samples identified 9% patients with splice-disrupting variants in CHD genes. Forty-one% of predicted splice-disrupting variants were deeply intronic. The burden of variants in CHD genes was higher in cases compared with 2,570 controls. Our model improved genetic yield by identifying splice-disrupting variants that are not evaluated by routine tests.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298689">
        <p class="paperTitle">A transgenerational mutational signature from ionizing radiation exposure.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brand, F.; Klinkhammer, H.; Knaus, A.; Holtgrewe, M.; Weinhold, L.; Beule, D.; Ludwig, K.; Kothiyal, P.; Maxwell, G.; Noethen, M.; Schmid, M.; Sperling, K.; Krawitz, P.</p>
        <p class="info">Score: 2.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298689' target='https://doi.org/10.1101/2023.11.20.23298689'> 10.1101/2023.11.20.23298689</a></p>
        <p class="abstract">Effects of accidental radiation exposure on the human germline remain a topic of medical interest. Sequencing technology now allows the accurate analysis of the entire human genome in potentially exposed cohorts to search for possible signatures of ionizing radiation exposure. Clustered de novo mutations (cDNMs), that are multiple DNA lesions within 20 bp of each other, have been hypothesized to be a signature for paternal exposure to ionizing radiation. We analyzed whole genome trio data of 240 children and their likely exposed parents as well as 1,275 offspring from unexposed families. The cohort of exposed children consists of 130 offspring of Chernobyl cleanup workers (CRU, exposure range = 0-4,080 mSv) and 110 offspring of former soldiers from both german armies which have likely been irradiated during their service (Radar cohort, exposure range = 0-353 mSv). For the Radar cohort, a retrospective dosage estimation was conducted based on the service records from each soldier, but it remains a challenging task to retrospectively estimate received dose accurately. In agreement with previous epidemiological data, no transgenerational effects could be detected for the class of isolated de novo mutations, which cause the majority of genetic disorders. We found that the number of isolated de novo mutations increases by 2% per year of age of the father at conception, which is also in line with previous works.

On average, we observed 2.65 cDNMs per offspring in the CRU cohort, 1.48 in the Radar cohort and 0.88 in the control cohort, with a median of two clustered mutations per genome in offspring of irradiated fathers. This is a significant increase (p &lt; 0.005) in the number of cDNMs compared to a set of age-matched controls. Furthermore, we show that the cDNM rates scale with paternal exposure to ionizing radiation (p &lt; 0.001). Our findings corroborate that clustered de novo mutations represent a transgenerational biomarker for paternal exposure to ionizing radiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.23298205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.23298205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.23298205">
        <p class="paperTitle">Network medicine-based epistasis detection in complex diseases: ready for quantum computing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.23298205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.23298205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, M.; Incudini, M.; Poschenrieder, J. M.; Baier, S.; Fitz, A.; Maier, A.; Hartung, M.; Hoffmann, C. M.; Trummer, N.; Adamowicz, K.; Picciani, M.; Scheibling, E.; Harl, M.; Lesch, I.; Frey, H.; Kayser, S.; Wissenberg, P.; Schwartz, L.; Hafner, L.; Acharya, A.; Hackl, L.; Grabert, G.; Lee, S.-G.; Cho, G.; Cloward, M. E.; Jankowski, J.; Lee, H. K. D.; Tsoy, O.; Wenke, N.; Pedersen, A. G.; Bonnelykke, K.; Mandarino, A.; Melograna, F.; Schulz, L.; Climente-Gonzalez, H.; Wilhelm, M.; Iapichino, L.; Wienbrandt, L.; Ellinghaus, D.; Van Steen, K.; Grossi, M.; Furth, P. A.; Hennighausen, L.; Di </p>
        <p class="info">Score: 7.6, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.23298205' target='https://doi.org/10.1101/2023.11.07.23298205'> 10.1101/2023.11.07.23298205</a></p>
        <p class="abstract">Most heritable diseases are polygenic. To comprehend the underlying genetic architecture, it is crucial to discover the clinically relevant epistatic interactions (EIs) between genomic single nucleotide polymorphisms (SNPs)1-3. Existing statistical computational methods for EI detection are mostly limited to pairs of SNPs due to the combinatorial explosion of higher-order EIs. With NeEDL (network-based epistasis detection via local search), we leverage network medicine to inform the selection of EIs that are an order of magnitude more statistically significant compared to existing tools and consist, on average, of five SNPs. We further show that this computationally demanding task can be substantially accelerated once quantum computing hardware becomes available. We apply NeEDL to eight different diseases and discover genes (affected by EIs of SNPs) that are partly known to affect the disease, additionally, these results are reproducible across independent cohorts. EIs for these eight diseases can be interactively explored in the Epistasis Disease Atlas (https://epistasis-disease-atlas.com). In summary, NeEDL is the first application that demonstrates the potential of seamlessly integrated quantum computing techniques to accelerate biomedical research. Our network medicine approach detects higher-order EIs with unprecedented statistical and biological evidence, yielding unique insights into polygenic diseases and providing a basis for the development of improved risk scores and combination therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.23299022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.23299022" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.23299022">
        <p class="paperTitle">Multi-trait genome-wide association study in 34,394 Chinese women reveals the genetic architecture of plasma metabolites during pregnancy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.23299022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.23299022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S.; Yao, J.; Lin, L.; Lan, X.; Wu, L.; Kong, N.; Deng, Y.; Li, Y.; Xie, J.; Zhu, H.; Wu, X.; Li, Z.; Xiong, L.; Wang, Y.; Ren, J.; Qiu, X.; Zhao, W.; Gao, Y.; Chen, Y.; Su, F.; Zhou, Y.; Rao, W.; Zhang, J.; Hou, G.; Huang, L.; Li, L.; Liu, X.; Nie, C.; Luo, L.; Liu, Z.; Chen, F.; Lin, S.; Zhao, L.; Fu, Q.; Jiang, D.; Yin, Y.; Xu, X.; Wang, J.; Yang, H.; Wang, R.; Niu, J.; Wei, F.; Jin, X.; Liu, S.</p>
        <p class="info">Score: 1.8, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.23299022' target='https://doi.org/10.1101/2023.11.26.23299022'> 10.1101/2023.11.26.23299022</a></p>
        <p class="abstract">Metabolites are important indicators of individual health and can serve as crucial targets for therapy. However, the genetic basis of many metabolites remains largely unexplored, especially among underrepresented East Asians and during critical periods such as pregnancy. In this study, we utilized genetic information obtained from non-invasive prenatal testing to conduct a genome-wide association analysis of 84 metabolites, including 37 amino acids, 10 vitamins, 24 metal elements, and 13 hormones, among 34,394 Chinese pregnant women. Of these metabolites, 52 and 11 had not previously been studied in East Asians or globally. We identified 30 novel metabolite-gene associations. We also observed substantial differences in the genetic effects on hormones between pregnancy and non-pregnancy periods, suggesting effect modifications in response to physiological changes. Furthermore, we uncovered pervasive pleiotropic effects for 50.94% of the genetic associations among metabolites, as well as between six metabolites and eight pregnancy biomarkers. Using mendelian randomization, we identified potential causal relationships between plasma folate and ischemic stroke, vitamin D3 and Graves&#39; disease, copper and open-angle glaucoma, and androstenedione and rheumatoid arthritis. These discoveries provide invaluable genetic insights into human metabolism, laying the foundation for future mechanistic studies and the development of new therapeutic targets, particularly for underrepresented East Asians.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298777">
        <p class="paperTitle">A comprehensive characterization of blood group antigen variants in the Middle Eastern population genomes - Insights into genetic epidemiology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rophina, M.; Pandhare, K. R.; Scaria, V.</p>
        <p class="info">Score: 1.8, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298777' target='https://doi.org/10.1101/2023.11.20.23298777'> 10.1101/2023.11.20.23298777</a></p>
        <p class="abstract">BackgroundThe Middle Eastern population is characterized by increased prevalence of various Mendelian disorders owing to increased rates of consanguinity. Especially in disease conditions which require chronic transfusion support, it becomes important to know the blood group characteristics of potential donors to increase the likelihood of transfusion success. As there prevails a gap in knowledge about the population specific overall blood antigen profiles, this study seeks to utilize next generation sequencing datasets to unravel the comprehensive landscape of clinically significant minor blood group alleles in the middle eastern population.

MethodsThis study utilizes the genetic variation data from a range of public datasets including the Greater Middle East Variome, the Qatar genome and exomes and the Iranome datasets to estimate the genotypic and phenotypic frequencies of blood group alleles in the Middle Easterners. The estimated frequencies were duly compared with major global populations to identify significant similarities or differences if any.

ResultsA total of 77 unique ISBT approved blood group alleles were found commonly in all datasets. 8 variants (rs8176058, rs1058396, rs565898944, rs28362692, rs2071699, rs34783571, rs60322991 and rs57467915) belonging to KELL, KIDD, COLTON, H, JUNIOR and LANGEREIS blood groups were found clinically significant with previously reported evidence on transfusion complications. 730 variants were found to span exonic or splicing regions out of which 70 were predicted to be potentially deleterious by at least four computational tools.

ConclusionsThis study serves first of its kind to extensively characterize the known and novel blood alleles in the Middle Easterners. A comprehensive user-friendly online resource named alnasab - Alleles and antigens in Arab and Persian populations associated with blood groups was also developed as a dependable reference for future transfusion research. The resource is accessible at https://clingen.igib.res.in/alnasab/

Key pointsO_LILarge scale Middle Eastern population sequencing datasets including The Greater Middle Eastern Variome, Genomes and exomes from Qatar and the Iranome datasets were used in the study.
C_LIO_LIA total of 2828 exomes and 88 genomes were analyzed accounting for a total of
C_LIO_LI18717 unique human blood group related variants.
C_LIO_LI2443 exonic variants were extracted which systematically included 1505 non-synonymous variants, 766 synonymous variants, 50 stopgain variants and 3 stop loss variants.
C_LIO_LIBlood group associated variants identified in the study are provided as a comprehensive online repository - alnasab, Alleles and antigens in Arab and Persian populations associated with blood groups. The resource is accessible at https://clingen.igib.res.in/alnasab/
C_LI

Visual abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=104 SRC=&#34;FIGDIR/small/23298777v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (21K):
org.highwire.dtl.DTLVardef@51dcb0org.highwire.dtl.DTLVardef@6fc1a5org.highwire.dtl.DTLVardef@1126e3aorg.highwire.dtl.DTLVardef@102ca64_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
