<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genetic influences on circulating retinol and its relationship to human health
Authors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.
Score: 20.6, Published: 2023-08-10 DOI: 10.1101/2023.08.07.23293796
Retinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genetic influences on circulating retinol and its relationship to human health
Authors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.
Score: 20.6, Published: 2023-08-10 DOI: 10.1101/2023.08.07.23293796
Retinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genetic influences on circulating retinol and its relationship to human health
Authors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.
Score: 20.6, Published: 2023-08-10 DOI: 10.1101/2023.08.07.23293796
Retinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genetic influences on circulating retinol and its relationship to human health\nAuthors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.\nScore: 20.6, Published: 2023-08-10 DOI: 10.1101/2023.08.07.23293796\nRetinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan.",
  "keywords": [
    
  ],
  "articleBody": " Genetic influences on circulating retinol and its relationship to human health\nAuthors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.\nScore: 20.6, Published: 2023-08-10 DOI: 10.1101/2023.08.07.23293796\nRetinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan. Previous work has characterised genetic influences on circulating retinol; however, small sample sizes have limited our ability to fully appreciate the genetic architecture of this trait. In this study, we performed the largest genome-wide association study (GWAS) of retinol to date in up to 22,274 participants. We identified eight common variant loci associated with retinol, as well as a rare-variant signal. An integrative gene prioritisation pipeline supported novel retinol-associated genes outside of the main retinol transport complex (RBP4:TTR) related to lipid biology, energy homeostasis, and endocrine signalling. Genetic proxies of circulating retinol were then used to estimate causal relationships with almost 20,000 clinical phenotypes via a phenome-wide Mendelian randomisation study (MR-pheWAS). The MR-pheWAS suggested that retinol may exert causal effects on inflammation, adiposity, ocular measures, the microbiome, and MRI-derived brain phenotypes, amongst several others. Conversely, circulating retinol may be causally influenced by factors including lipids and renal function. Finally, we demonstrated how a retinol polygenic score could identify individuals who are more likely to fall outside of the normative range of circulating retinol for a given age. In summary, this study provides a comprehensive evaluation of the genetics of circulating retinol, as well as revealing traits which should be prioritised for further clinical investigation with respect to retinol related therapies or nutritional intervention.\nShared genetic etiology between chronic diseases and heart failure risk: the dual role of leukocyte telomere length\nAuthors: Wong, J.; Blechter, B.; Liu, Z.; Shi, J.; Roger, V. L.\nScore: 17.1, Published: 2023-08-08 DOI: 10.1101/2023.08.03.23293621\nIntroductionGenetic susceptibility to various chronic diseases has been shown to influence heart failure (HF) risk. However, the pathogenic mechanisms underlying these associations, particularly the role of phenotypic leukocyte telomere length (LTL), is unknown. We investigated the shared genetic etiology between chronic diseases, various traits, and HF risk, and whether LTL mediates or modifies these relationships. MethodsWe conducted prospective cohort analyses on 404,883 European participants from the UK Biobank, including 9,989 incident HF cases. Multivariable Cox regression was used to estimate associations between HF risk and 24 polygenic risk scores (PRSs) for various diseases or traits previously generated in the UK Biobank using a Bayesian approach. We assessed multiplicative interactions between the PRSs and LTL previously measured in the UK Biobank using quantitative PCR. Mediation analyses were conducted to estimate the proportion of the total effect of PRSs acting indirectly through LTL, an integrative marker of biological aging. ResultsWe identified 9 PRSs associated with HF risk, including those for various cardiovascular diseases or traits, rheumatoid arthritis (P=1.3E-04), and asthma (P=1.8E-08). Additionally, longer LTL was strongly associated with decreased HF risk (P-trend=1.7E-08). Notably, the asthma PRS had a super-multiplicative interaction with LTL (P-interaction=2.8E-03). However, LTL mediated only 1.13% (P\u003c0.001) of the total effect of the asthma PRS on HF risk. ConclusionsOur findings shed light onto the shared genetic etiology between HF risk, asthma, rheumatoid arthritis, and other traits. Longer LTL strengthened the genetic effect of asthma on HF, supporting their utility in risk stratification analyses.\nUnique Capabilities of Genome Sequencing for Rare Disease Diagnosis\nAuthors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G.; Pierce, E. A.; Place, E. M.; Pajusalu, S.; Pais, L.; Ounap, K.; Osei-Owusu, I.; Okur, V.; Oja, K. T.; OLeary, M.; OHeir, E.; Morel, C.; Marchant, R. G.; Mangilog, B. E.; Madden, J. A.; MacArthur, D.; Lovgren, A.; Lerner-Ellis, J. P.; Lin, J.; Laing, N.; H\nScore: 11.6, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293829\nBackground: Causal variants underlying rare disorders may remain elusive even after expansive gene panels or exome sequencing (ES). Clinicians and researchers may then turn to genome sequencing (GS), though the added value of this technique and its optimal use remain poorly defined. We therefore investigated the advantages of GS within a phenotypically diverse cohort. Methods: GS was performed for 744 individuals with rare disease who were genetically undiagnosed. Analysis included review of single nucleotide, indel, structural, and mitochondrial variants. Results: We successfully solved 218/744 (29.3%) cases using GS, with most solves involving established disease genes (157/218, 72.0%). Of all solved cases, 148 (67.9%) had previously had non-diagnostic ES. We systematically evaluated the 218 causal variants for features requiring GS to identify and 61/218 (28.0%) met these criteria, representing 8.2% of the entire cohort. These included small structural variants (13), copy neutral inversions and complex rearrangements (8), tandem repeat expansions (6), deep intronic variants (15), and coding variants that may be more easily found using GS related to uniformity of coverage (19). Conclusion: We describe the diagnostic yield of GS in a large and diverse cohort, illustrating several types of pathogenic variation eluding ES or other techniques. Our results reveal a higher diagnostic yield of GS, supporting the utility of a genome-first approach, with consideration of GS as a secondary or tertiary test when higher-resolution structural variant analysis is needed or there is a strong clinical suspicion for a condition and prior targeted genetic testing has been negative.\nSevere obesity may be an oligogenic condition: evidence from 1,714 adults seeking treatment in the UK National Health Service\nAuthors: Almansoori, S.; Amin, H.; Alsters, S.; Handley, D. K.; Yiorkas, A. M.; Hashim, N. A. N.; Ramzi, N. H.; Bonaomi, G.; Ahmed, J.; Agrawal, S.; Small, P.; Purkayastha, S.; Haelst, M. V.; Walters, R. G.; Roux, C. W. l.; Chahal, H. S.; Drenos, F.; Blakemore, A. I.\nScore: 11.6, Published: 2023-08-08 DOI: 10.1101/2023.08.04.23293229\nSevere (class III) obesity is a chronic, relapsing condition, with a high burden of co-morbidity and mortality. Previous evidence has established genetics as an important contributing factor. We therefore designed a custom genotyping array to screen a cohort of UK patients seeking treatment for severe obesity. 1,714 participants were genotyped using a custom AXIOM array, focusing on rare (minor allele frequency \u003c0.01) variants, with CADD-PHRED [\u0026ge;]15 in 78 genes known/suspected to cause Mendelian forms of obesity. Concordance analyses of 22 duplicate samples and 66 samples with whole exome sequence data revealed good genotyping reliability. We identified the proportion of study participants who carried, or were homozygous for, rare, predicted-deleterious variants in genes with dominant and recessive modes of inheritance (MOI), respectively. 27% of patients carried relevant mutations consistent with the expected MOI, which was very similar to the rate observed in the UKB 50K whole exome sequencing dataset. However, the clinical obesity cohort were more likely to carry two or more such variants, in separate genes, than UK Biobank participants (p = 0.018). In conclusion, our results provide evidence: that (i) custom genotyping arrays, used with improved algorithms can allow reliable, cost-effective screening for rare genetic variants; (ii) rare mutations in \"obesity genes\" may be at high prevalence among bariatric patients, as well as in the general population; and (iii) that severe obesity may have an oligogenic pattern of inheritance in some cases.\nStrong protective effect of the APOL1 p.N264K variant against G2-associated FSGS and kidney disease\nAuthors: Gupta, Y.; Friedman, D. J.; McNulty, M.; Khan, A.; Lane, B.; Wang, C.; Ke, J.; Jin, G.; Wooden, B.; Knob, A. L.; Lim, T. Y.; Appel, G. B.; Huggins, K.; Liu, L.; Mitrotti, A.; Stangl, M. C.; Bomback, A.; Westland, R.; Bodria, M.; Marasa, M.; Shang, N.; Cohen, D. J.; Crew, R. J.; Morello, W.; Canetta, P.; Radhakrishnan, J.; Martino, J.; Liu, Q.; Chung, W. K.; Espinoza, A.; Luo, Y.; Wei, W.-Q.; Feng, Q.; Weng, C.; Fang, Y.; Kullo, I. J.; Naderian, M.; Limdi, N.; Irvin, M. R.; Tiwari, H.; Mohan, S.; Rao, M.; Dube, G.; Chaudhary, N. S.; Gutierrez, O. M.; Judd, S. E.; Cushman, M.; Lange, L. A.; Lang\nScore: 8.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293554\nBlack Americans have a significantly higher risk of developing chronic kidney disease (CKD), especially focal segmental glomerulosclerosis (FSGS), than European Americans. Two coding variants (G1 and G2) in the APOL1 gene play a major role in this disparity. While 13% of Black Americans carry the high-risk recessive genotypes, only a fraction of these individuals develops FSGS or kidney failure, indicating the involvement of additional disease modifiers. Here, we show that the presence of the APOL1 p.N264K missense variant, when co-inherited with the G2 APOL1 risk allele, substantially reduces the penetrance of the G1G2 and G2G2 high-risk genotypes by rendering these genotypes low-risk. These results align with prior functional evidence showing that the p.N264K variant reduces the toxicity of the APOL1 high-risk alleles. These findings have important implications for our understanding of the mechanisms of APOL1-associated nephropathy, as well as for the clinical management of individuals with high-risk genotypes that include the G2 allele.\nGenetic Elucidation of Ultrasonography Fetal Anomalies in Children with Autism Spectrum Disorder\nAuthors: Regev, O.; Shil, A.; Bronshtein, T.; Hadar, A.; Meiri, G.; Zigdon, D.; Michaelovski, A.; Dinstein, I.; Hershkovitz, R.; Menashe, I.\nScore: 3.0, Published: 2023-08-09 DOI: 10.1101/2023.08.05.23293591\nRecent evidence suggests that certain fetal anomalies detected upon prenatal ultrasound screenings may be associated with autism spectrum disorder (ASD). In this cross-sectional study, we aimed to identify genetic variants associated with fetal ultrasound anomalies (UFAs) in children with ASD. The study included all children with ASD who are registered in the database of the Azrieli National Center of Autism and Neurodevelopment and for whom both prenatal ultrasound and whole exome sequencing (WES) data were available. We applied our in-house integrative bioinformatics pipeline, AutScore, to these WES data to prioritize gene-disrupting variants (GDVs) probably contributing to ASD susceptibily. Univariate statistics and multivariable regression were used to assess the associations between UFAs and GDVs identified in these children. The study sample comprised 126 children, of whom 43 (34.1%) had at least one UFA detected in the prenatal ultrasound scan. A total of 87 candidate ASD genetic variants were detected in 60 children, with 24 (40%) children carrying multiple variants. There was a weak, but significant, correlation between the number of mutations and the number of abnormalities detected in the same children (r = 0.21, P = 0.016). Children with UFAs were more likely to have loss-of-function (LoF) mutations (aOR=2.55, 95%CI: 1.13-5.80). This association was particularly noticeable when children with structural anomalies or children with UFAs in their head and brain scans were compared to children without UFAs (any mutation: aOR=8.28, 95%CI: 2.29-30.01; LoF: aOR=5.72, 95%CI: 2.08-15.71 and any mutation: aOR=6.39, 95%CI: 1.34-30.47; LoF: aOR=4.50, 95%CI: 1.32-15.35, respectively). GDVs associated with UFAs were enriched in genes highly expressed across all tissues (aOR=2.76, 95%CI: 1.14-6.68). The results provide valuable insights into the potential genetic basis of prenatal organogenesis abnormalities associated with ASD and shed light on the complex interplay between genetic factors and fetal development.\nA genotype-first approach identifies high incidence of NF1 pathogenic variants with distinct disease associations\nAuthors: Safonov, A.; Nomakuchi, T. T.; Chao, E.; Horton, C.; Dolinsky, J. S.; Yussuf, A.; Richardson, M.; Speare, V.; Li, S.; Bogus, Z. C.; Bonanni, M.; Raper, A.; Kallish, S.; Ritchie, M. D.; Regeneron Genetics Center, ; Penn Medicine BioBank, ; Nathanson, K. L.; Drivas, T. G.\nScore: 2.6, Published: 2023-08-10 DOI: 10.1101/2023.08.08.23293676\nLoss of function variants in the NF1 gene cause neurofibromatosis type 1 (NF1), a genetic disorder characterized by complete penetrance, prevalence of 1 in 3,000, characteristic physical exam findings, and a substantially increased risk for malignancy. However, our understanding of the disorder is entirely based on patients ascertained through phenotype-first approaches. Leveraging a genotype-first approach in two large patient cohorts, we demonstrate unexpectedly high prevalence (1 in 450-750) of NF1 pathogenic variants. Half were identified in individuals lacking clinical features of NF1, with many appearing to have post-zygotic mosaicism for the identified variant. Incidentally discovered variants were not associated with classic NF1 features but were associated with an increased incidence of malignancy compared to a control population. Our findings suggest that NF1 pathogenic variants are substantially more common than previously thought, often characterized by somatic mosaicism and reduced penetrance, and are important contributors to cancer risk in the general population.\nCopy-number variants as modulators of common disease susceptibility\nAuthors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; MaÌˆgi, R.; Estonian Biobank Research Team, ; Reymond, A.; Kutalik, Z.\nScore: 16.5, Published: 2023-08-05 DOI: 10.1101/2023.07.31.23293408\nBackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described. MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank. ResultsWe identified 73 signals involving 40 diseases, all of which indicating that CNVs increased disease risk and caused earlier onset. Even after correcting for these signals, a higher CNV burden increased risk for 18 disorders, mainly through the number of deleted genes, suggesting a polygenic CNV architecture. Number and identity of genes disturbed by CNVs affected their pathogenicity, with many associations being supported by colocalization with both common and rare single nucleotide variant association signals. Dissection of association signals provided insights into the epidemiology of known gene-disease pairs (e.g., deletions in BRCA1 and LDLR increased risk for ovarian cancer and ischemic heart disease, respectively), clarified dosage mechanisms of action (e.g., both increased and decreased dosage of 17q12 impacts renal health), and identified putative causal genes (e.g., ABCC6 for kidney stones). Characterization of the pleiotropic pathological consequences of recurrent CNVs at 15q13, 16p13.11, 16p12.2, and 22q11.2 in adulthood indicated variable expressivity of these regions and the involvement of multiple genes. ConclusionsOur results shed light on the prominent role of CNVs in determining common disease susceptibility within the general population and provide actionable insights allowing to anticipate later-onset comorbidities in carriers of recurrent CNVs.\nCritical assessment of variant prioritization methods for rare disease diagnosis within the Rare Genomes Project\nAuthors: Stenton, S. L.; O'Leary, M.; Lemire, G.; VanNoy, G. E.; DiTroia, S.; Ganesh, V. S.; Groopman, E.; O'Heir, E.; Mangilog, B.; Osei-Owusu, I.; Pais, L. S.; Serrano, J.; Singer-Berk, M.; Weisburd, B.; Wilson, M.; Austin-Tse, C.; Abdelhakim, M.; Althagafi, A.; Babbi, G.; Bellazzi, R.; Bovo, S.; Carta, M. G.; Casadio, R.; Coenen, P.-J.; De Paoli, F.; Floris, M.; Gajapathy, M.; Hoehndorf, R.; Jacobsen, J. O. B.; Joseph, T.; Kamandula, A.; Katsonis, P.; Kint, C.; Lichtarge, O.; Limongelli, I.; Lu, Y.; Magni, P.; Mamidi, T. K. K.; Martelli, P. L.; Mulargia, M.; Nicora, G.; Nykamp, K.; Pejaver, V.; Peng\nScore: 8.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293212\nBackgroundA major obstacle faced by rare disease families is obtaining a genetic diagnosis. The average \"diagnostic odyssey\" lasts over five years, and causal variants are identified in under 50%. The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing (GS) for diagnosis and gene discovery. Families are consented for sharing of sequence and phenotype data with researchers, allowing development of a Critical Assessment of Genome Interpretation (CAGI) community challenge, placing variant prioritization models head-to-head in a real-life clinical diagnostic setting. MethodsPredictors were provided a dataset of phenotype terms and variant calls from GS of 175 RGP individuals (65 families), including 35 solved training set families, with causal variants specified, and 30 test set families (14 solved, 16 unsolved). The challenge tasked teams with identifying the causal variants in as many test set families as possible. Ranked variant predictions were submitted with estimated probability of causal relationship (EPCR) values. Model performance was determined by two metrics, a weighted score based on rank position of true positive causal variants and maximum F-measure, based on precision and recall of causal variants across EPCR thresholds. ResultsSixteen teams submitted predictions from 52 models, some with manual review incorporated. Top performing teams recalled the causal variants in up to 13 of 14 solved families by prioritizing high quality variant calls that were rare, predicted deleterious, segregating correctly, and consistent with reported phenotype. In unsolved families, newly discovered diagnostic variants were returned to two families following confirmatory RNA sequencing, and two prioritized novel disease gene candidates were entered into Matchmaker Exchange. In one example, RNA sequencing demonstrated aberrant splicing due to a deep intronic indel in ASNS, identified in trans with a frameshift variant, in an unsolved proband with phenotype overlap with asparagine synthetase deficiency. ConclusionsBy objective assessment of variant predictions, we provide insights into current state-of-the-art algorithms and platforms for genome sequencing analysis for rare disease diagnosis and explore areas for future optimization. Identification of diagnostic variants in unsolved families promotes synergy between researchers with clinical and computational expertise as a means of advancing the field of clinical genome interpretation.\nDifferences in pathogenetic mechanism between Tibetan and Han high-altitude polycythemia based on a whole genome-wide association study\nAuthors: Zhuoma, B.; Shixuan, Z.; Xianwei, K.; Zhuoma, D.; Danzeng, Q.; La, Y.; Yang, D.; Meng, H.; Zixin, H.; Yanyun, M.; Zeshan, L.; Yi, L.; Jiucun, W.; Juan, W.\nScore: 1.2, Published: 2023-08-09 DOI: 10.1101/2023.08.05.23293524\nHigh-altitude polycythemia (HAPC) is a common and serious chronic disease affecting people of highland and plains ancestry living at high altitudes. This study investigated genetic susceptibility differences for HAPC among ethnic groups, with 2,248 volunteers participating, including 898 HAPC patients (nHan = 450, nTibetan = 448). The study included a GWAS of 198 cases (nHan = 100, nTibetan = 98), which revealed eight Tibetan HAPC-susceptibility single-nucleotide polymorphisms and four Han HAPC-susceptibility VARIANTs. Among them, the common polymorphism locus rs7618658 (SNX4, Pcombine\u003c5e-8) was found and verified in both Tibetans and Han Chinese. Furthermore, the exploration of Tibetan EPAS1 suggested that the rs1374749 locus of EPAS1, along with other locus-linked status changes, may be a prevalence factor for HAPC, and the GGTAC haplotype in which it is located is a risk haplotype for HAPC (P = 5.14E-9, OR = 1.85).The study showed significant genetic susceptibility to HAPC between Tibetans and Han Chinese, with Tibetans mainly susceptible in oxygen-sensing pathways such as EPAS1, which are associated with several phenotypes (Hemoglobin, Hematocrit, and Platelets, etc), while Han Chinese patients exhibited susceptibilities mainly in cell differentiation and angiogenesis, such as in SNX4 and LPAR1, which are similarly significantly associated with hemoglobin, hematocrit, and platelets.\n",
  "wordCount" : "3088",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.23293796">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.23293796" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.23293796">
        <p class="paperTitle">Genetic influences on circulating retinol and its relationship to human health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.23293796" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.23293796" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.</p>
        <p class="info">Score: 20.6, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.23293796' target='https://doi.org/10.1101/2023.08.07.23293796'> 10.1101/2023.08.07.23293796</a></p>
        <p class="abstract">Retinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan. Previous work has characterised genetic influences on circulating retinol; however, small sample sizes have limited our ability to fully appreciate the genetic architecture of this trait. In this study, we performed the largest genome-wide association study (GWAS) of retinol to date in up to 22,274 participants. We identified eight common variant loci associated with retinol, as well as a rare-variant signal. An integrative gene prioritisation pipeline supported novel retinol-associated genes outside of the main retinol transport complex (RBP4:TTR) related to lipid biology, energy homeostasis, and endocrine signalling. Genetic proxies of circulating retinol were then used to estimate causal relationships with almost 20,000 clinical phenotypes via a phenome-wide Mendelian randomisation study (MR-pheWAS). The MR-pheWAS suggested that retinol may exert causal effects on inflammation, adiposity, ocular measures, the microbiome, and MRI-derived brain phenotypes, amongst several others. Conversely, circulating retinol may be causally influenced by factors including lipids and renal function. Finally, we demonstrated how a retinol polygenic score could identify individuals who are more likely to fall outside of the normative range of circulating retinol for a given age. In summary, this study provides a comprehensive evaluation of the genetics of circulating retinol, as well as revealing traits which should be prioritised for further clinical investigation with respect to retinol related therapies or nutritional intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.23293621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.23293621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.23293621">
        <p class="paperTitle">Shared genetic etiology between chronic diseases and heart failure risk: the dual role of leukocyte telomere length</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.23293621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.23293621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, J.; Blechter, B.; Liu, Z.; Shi, J.; Roger, V. L.</p>
        <p class="info">Score: 17.1, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.23293621' target='https://doi.org/10.1101/2023.08.03.23293621'> 10.1101/2023.08.03.23293621</a></p>
        <p class="abstract">IntroductionGenetic susceptibility to various chronic diseases has been shown to influence heart failure (HF) risk. However, the pathogenic mechanisms underlying these associations, particularly the role of phenotypic leukocyte telomere length (LTL), is unknown. We investigated the shared genetic etiology between chronic diseases, various traits, and HF risk, and whether LTL mediates or modifies these relationships.

MethodsWe conducted prospective cohort analyses on 404,883 European participants from the UK Biobank, including 9,989 incident HF cases. Multivariable Cox regression was used to estimate associations between HF risk and 24 polygenic risk scores (PRSs) for various diseases or traits previously generated in the UK Biobank using a Bayesian approach. We assessed multiplicative interactions between the PRSs and LTL previously measured in the UK Biobank using quantitative PCR. Mediation analyses were conducted to estimate the proportion of the total effect of PRSs acting indirectly through LTL, an integrative marker of biological aging.

ResultsWe identified 9 PRSs associated with HF risk, including those for various cardiovascular diseases or traits, rheumatoid arthritis (P=1.3E-04), and asthma (P=1.8E-08). Additionally, longer LTL was strongly associated with decreased HF risk (P-trend=1.7E-08). Notably, the asthma PRS had a super-multiplicative interaction with LTL (P-interaction=2.8E-03). However, LTL mediated only 1.13% (P&lt;0.001) of the total effect of the asthma PRS on HF risk.

ConclusionsOur findings shed light onto the shared genetic etiology between HF risk, asthma, rheumatoid arthritis, and other traits. Longer LTL strengthened the genetic effect of asthma on HF, supporting their utility in risk stratification analyses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293829">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293829" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293829">
        <p class="paperTitle">Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293829" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293829" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G.; Pierce, E. A.; Place, E. M.; Pajusalu, S.; Pais, L.; Ounap, K.; Osei-Owusu, I.; Okur, V.; Oja, K. T.; OLeary, M.; OHeir, E.; Morel, C.; Marchant, R. G.; Mangilog, B. E.; Madden, J. A.; MacArthur, D.; Lovgren, A.; Lerner-Ellis, J. P.; Lin, J.; Laing, N.; H</p>
        <p class="info">Score: 11.6, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293829' target='https://doi.org/10.1101/2023.08.08.23293829'> 10.1101/2023.08.08.23293829</a></p>
        <p class="abstract">Background: Causal variants underlying rare disorders may remain elusive even after expansive gene panels or exome sequencing (ES). Clinicians and researchers may then turn to genome sequencing (GS), though the added value of this technique and its optimal use remain poorly defined. We therefore investigated the advantages of GS within a phenotypically diverse cohort. Methods: GS was performed for 744 individuals with rare disease who were genetically undiagnosed. Analysis included review of single nucleotide, indel, structural, and mitochondrial variants. Results: We successfully solved 218/744 (29.3%) cases using GS, with most solves involving established disease genes (157/218, 72.0%). Of all solved cases, 148 (67.9%) had previously had non-diagnostic ES. We systematically evaluated the 218 causal variants for features requiring GS to identify and 61/218 (28.0%) met these criteria, representing 8.2% of the entire cohort. These included small structural variants (13), copy neutral inversions and complex rearrangements (8), tandem repeat expansions (6), deep intronic variants (15), and coding variants that may be more easily found using GS related to uniformity of coverage (19). Conclusion: We describe the diagnostic yield of GS in a large and diverse cohort, illustrating several types of pathogenic variation eluding ES or other techniques. Our results reveal a higher diagnostic yield of GS, supporting the utility of a genome-first approach, with consideration of GS as a secondary or tertiary test when higher-resolution structural variant analysis is needed or there is a strong clinical suspicion for a condition and prior targeted genetic testing has been negative.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.23293229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.23293229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.23293229">
        <p class="paperTitle">Severe obesity may be an oligogenic condition: evidence from 1,714 adults seeking treatment in the UK National Health Service</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.23293229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.23293229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Almansoori, S.; Amin, H.; Alsters, S.; Handley, D. K.; Yiorkas, A. M.; Hashim, N. A. N.; Ramzi, N. H.; Bonaomi, G.; Ahmed, J.; Agrawal, S.; Small, P.; Purkayastha, S.; Haelst, M. V.; Walters, R. G.; Roux, C. W. l.; Chahal, H. S.; Drenos, F.; Blakemore, A. I.</p>
        <p class="info">Score: 11.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.23293229' target='https://doi.org/10.1101/2023.08.04.23293229'> 10.1101/2023.08.04.23293229</a></p>
        <p class="abstract">Severe (class III) obesity is a chronic, relapsing condition, with a high burden of co-morbidity and mortality. Previous evidence has established genetics as an important contributing factor. We therefore designed a custom genotyping array to screen a cohort of UK patients seeking treatment for severe obesity. 1,714 participants were genotyped using a custom AXIOM array, focusing on rare (minor allele frequency &lt;0.01) variants, with CADD-PHRED [&amp;ge;]15 in 78 genes known/suspected to cause Mendelian forms of obesity. Concordance analyses of 22 duplicate samples and 66 samples with whole exome sequence data revealed good genotyping reliability. We identified the proportion of study participants who carried, or were homozygous for, rare, predicted-deleterious variants in genes with dominant and recessive modes of inheritance (MOI), respectively. 27% of patients carried relevant mutations consistent with the expected MOI, which was very similar to the rate observed in the UKB 50K whole exome sequencing dataset. However, the clinical obesity cohort were more likely to carry two or more such variants, in separate genes, than UK Biobank participants (p = 0.018). In conclusion, our results provide evidence: that (i) custom genotyping arrays, used with improved algorithms can allow reliable, cost-effective screening for rare genetic variants; (ii) rare mutations in &#34;obesity genes&#34; may be at high prevalence among bariatric patients, as well as in the general population; and (iii) that severe obesity may have an oligogenic pattern of inheritance in some cases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293554">
        <p class="paperTitle">Strong protective effect of the APOL1 p.N264K variant against G2-associated FSGS and kidney disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, Y.; Friedman, D. J.; McNulty, M.; Khan, A.; Lane, B.; Wang, C.; Ke, J.; Jin, G.; Wooden, B.; Knob, A. L.; Lim, T. Y.; Appel, G. B.; Huggins, K.; Liu, L.; Mitrotti, A.; Stangl, M. C.; Bomback, A.; Westland, R.; Bodria, M.; Marasa, M.; Shang, N.; Cohen, D. J.; Crew, R. J.; Morello, W.; Canetta, P.; Radhakrishnan, J.; Martino, J.; Liu, Q.; Chung, W. K.; Espinoza, A.; Luo, Y.; Wei, W.-Q.; Feng, Q.; Weng, C.; Fang, Y.; Kullo, I. J.; Naderian, M.; Limdi, N.; Irvin, M. R.; Tiwari, H.; Mohan, S.; Rao, M.; Dube, G.; Chaudhary, N. S.; Gutierrez, O. M.; Judd, S. E.; Cushman, M.; Lange, L. A.; Lang</p>
        <p class="info">Score: 8.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293554' target='https://doi.org/10.1101/2023.08.02.23293554'> 10.1101/2023.08.02.23293554</a></p>
        <p class="abstract">Black Americans have a significantly higher risk of developing chronic kidney disease (CKD), especially focal segmental glomerulosclerosis (FSGS), than European Americans. Two coding variants (G1 and G2) in the APOL1 gene play a major role in this disparity. While 13% of Black Americans carry the high-risk recessive genotypes, only a fraction of these individuals develops FSGS or kidney failure, indicating the involvement of additional disease modifiers.

Here, we show that the presence of the APOL1 p.N264K missense variant, when co-inherited with the G2 APOL1 risk allele, substantially reduces the penetrance of the G1G2 and G2G2 high-risk genotypes by rendering these genotypes low-risk. These results align with prior functional evidence showing that the p.N264K variant reduces the toxicity of the APOL1 high-risk alleles. These findings have important implications for our understanding of the mechanisms of APOL1-associated nephropathy, as well as for the clinical management of individuals with high-risk genotypes that include the G2 allele.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.23293591">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.23293591" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.23293591">
        <p class="paperTitle">Genetic Elucidation of Ultrasonography Fetal Anomalies in Children with Autism Spectrum Disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.23293591" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.23293591" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Regev, O.; Shil, A.; Bronshtein, T.; Hadar, A.; Meiri, G.; Zigdon, D.; Michaelovski, A.; Dinstein, I.; Hershkovitz, R.; Menashe, I.</p>
        <p class="info">Score: 3.0, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.23293591' target='https://doi.org/10.1101/2023.08.05.23293591'> 10.1101/2023.08.05.23293591</a></p>
        <p class="abstract">Recent evidence suggests that certain fetal anomalies detected upon prenatal ultrasound screenings may be associated with autism spectrum disorder (ASD). In this cross-sectional study, we aimed to identify genetic variants associated with fetal ultrasound anomalies (UFAs) in children with ASD. The study included all children with ASD who are registered in the database of the Azrieli National Center of Autism and Neurodevelopment and for whom both prenatal ultrasound and whole exome sequencing (WES) data were available. We applied our in-house integrative bioinformatics pipeline, AutScore, to these WES data to prioritize gene-disrupting variants (GDVs) probably contributing to ASD susceptibily. Univariate statistics and multivariable regression were used to assess the associations between UFAs and GDVs identified in these children. The study sample comprised 126 children, of whom 43 (34.1%) had at least one UFA detected in the prenatal ultrasound scan. A total of 87 candidate ASD genetic variants were detected in 60 children, with 24 (40%) children carrying multiple variants. There was a weak, but significant, correlation between the number of mutations and the number of abnormalities detected in the same children (r = 0.21, P = 0.016). Children with UFAs were more likely to have loss-of-function (LoF) mutations (aOR=2.55, 95%CI: 1.13-5.80). This association was particularly noticeable when children with structural anomalies or children with UFAs in their head and brain scans were compared to children without UFAs (any mutation: aOR=8.28, 95%CI: 2.29-30.01; LoF: aOR=5.72, 95%CI: 2.08-15.71 and any mutation: aOR=6.39, 95%CI: 1.34-30.47; LoF: aOR=4.50, 95%CI: 1.32-15.35, respectively). GDVs associated with UFAs were enriched in genes highly expressed across all tissues (aOR=2.76, 95%CI: 1.14-6.68). The results provide valuable insights into the potential genetic basis of prenatal organogenesis abnormalities associated with ASD and shed light on the complex interplay between genetic factors and fetal development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293676">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293676" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293676">
        <p class="paperTitle">A genotype-first approach identifies high incidence of NF1 pathogenic variants with distinct disease associations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293676" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293676" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Safonov, A.; Nomakuchi, T. T.; Chao, E.; Horton, C.; Dolinsky, J. S.; Yussuf, A.; Richardson, M.; Speare, V.; Li, S.; Bogus, Z. C.; Bonanni, M.; Raper, A.; Kallish, S.; Ritchie, M. D.; Regeneron Genetics Center,  ; Penn Medicine BioBank,  ; Nathanson, K. L.; Drivas, T. G.</p>
        <p class="info">Score: 2.6, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293676' target='https://doi.org/10.1101/2023.08.08.23293676'> 10.1101/2023.08.08.23293676</a></p>
        <p class="abstract">Loss of function variants in the NF1 gene cause neurofibromatosis type 1 (NF1), a genetic disorder characterized by complete penetrance, prevalence of 1 in 3,000, characteristic physical exam findings, and a substantially increased risk for malignancy. However, our understanding of the disorder is entirely based on patients ascertained through phenotype-first approaches. Leveraging a genotype-first approach in two large patient cohorts, we demonstrate unexpectedly high prevalence (1 in 450-750) of NF1 pathogenic variants. Half were identified in individuals lacking clinical features of NF1, with many appearing to have post-zygotic mosaicism for the identified variant. Incidentally discovered variants were not associated with classic NF1 features but were associated with an increased incidence of malignancy compared to a control population. Our findings suggest that NF1 pathogenic variants are substantially more common than previously thought, often characterized by somatic mosaicism and reduced penetrance, and are important contributors to cancer risk in the general population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.23293408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.23293408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.23293408">
        <p class="paperTitle">Copy-number variants as modulators of common disease susceptibility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.23293408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.23293408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; MaÌˆgi, R.; Estonian Biobank Research Team,  ; Reymond, A.; Kutalik, Z.</p>
        <p class="info">Score: 16.5, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.23293408' target='https://doi.org/10.1101/2023.07.31.23293408'> 10.1101/2023.07.31.23293408</a></p>
        <p class="abstract">BackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described.

MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank.

ResultsWe identified 73 signals involving 40 diseases, all of which indicating that CNVs increased disease risk and caused earlier onset. Even after correcting for these signals, a higher CNV burden increased risk for 18 disorders, mainly through the number of deleted genes, suggesting a polygenic CNV architecture. Number and identity of genes disturbed by CNVs affected their pathogenicity, with many associations being supported by colocalization with both common and rare single nucleotide variant association signals. Dissection of association signals provided insights into the epidemiology of known gene-disease pairs (e.g., deletions in BRCA1 and LDLR increased risk for ovarian cancer and ischemic heart disease, respectively), clarified dosage mechanisms of action (e.g., both increased and decreased dosage of 17q12 impacts renal health), and identified putative causal genes (e.g., ABCC6 for kidney stones). Characterization of the pleiotropic pathological consequences of recurrent CNVs at 15q13, 16p13.11, 16p12.2, and 22q11.2 in adulthood indicated variable expressivity of these regions and the involvement of multiple genes.

ConclusionsOur results shed light on the prominent role of CNVs in determining common disease susceptibility within the general population and provide actionable insights allowing to anticipate later-onset comorbidities in carriers of recurrent CNVs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293212">
        <p class="paperTitle">Critical assessment of variant prioritization methods for rare disease diagnosis within the Rare Genomes Project</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stenton, S. L.; O&#39;Leary, M.; Lemire, G.; VanNoy, G. E.; DiTroia, S.; Ganesh, V. S.; Groopman, E.; O&#39;Heir, E.; Mangilog, B.; Osei-Owusu, I.; Pais, L. S.; Serrano, J.; Singer-Berk, M.; Weisburd, B.; Wilson, M.; Austin-Tse, C.; Abdelhakim, M.; Althagafi, A.; Babbi, G.; Bellazzi, R.; Bovo, S.; Carta, M. G.; Casadio, R.; Coenen, P.-J.; De Paoli, F.; Floris, M.; Gajapathy, M.; Hoehndorf, R.; Jacobsen, J. O. B.; Joseph, T.; Kamandula, A.; Katsonis, P.; Kint, C.; Lichtarge, O.; Limongelli, I.; Lu, Y.; Magni, P.; Mamidi, T. K. K.; Martelli, P. L.; Mulargia, M.; Nicora, G.; Nykamp, K.; Pejaver, V.; Peng</p>
        <p class="info">Score: 8.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293212' target='https://doi.org/10.1101/2023.08.02.23293212'> 10.1101/2023.08.02.23293212</a></p>
        <p class="abstract">BackgroundA major obstacle faced by rare disease families is obtaining a genetic diagnosis. The average &#34;diagnostic odyssey&#34; lasts over five years, and causal variants are identified in under 50%. The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing (GS) for diagnosis and gene discovery. Families are consented for sharing of sequence and phenotype data with researchers, allowing development of a Critical Assessment of Genome Interpretation (CAGI) community challenge, placing variant prioritization models head-to-head in a real-life clinical diagnostic setting.

MethodsPredictors were provided a dataset of phenotype terms and variant calls from GS of 175 RGP individuals (65 families), including 35 solved training set families, with causal variants specified, and 30 test set families (14 solved, 16 unsolved). The challenge tasked teams with identifying the causal variants in as many test set families as possible. Ranked variant predictions were submitted with estimated probability of causal relationship (EPCR) values. Model performance was determined by two metrics, a weighted score based on rank position of true positive causal variants and maximum F-measure, based on precision and recall of causal variants across EPCR thresholds.

ResultsSixteen teams submitted predictions from 52 models, some with manual review incorporated. Top performing teams recalled the causal variants in up to 13 of 14 solved families by prioritizing high quality variant calls that were rare, predicted deleterious, segregating correctly, and consistent with reported phenotype. In unsolved families, newly discovered diagnostic variants were returned to two families following confirmatory RNA sequencing, and two prioritized novel disease gene candidates were entered into Matchmaker Exchange. In one example, RNA sequencing demonstrated aberrant splicing due to a deep intronic indel in ASNS, identified in trans with a frameshift variant, in an unsolved proband with phenotype overlap with asparagine synthetase deficiency.

ConclusionsBy objective assessment of variant predictions, we provide insights into current state-of-the-art algorithms and platforms for genome sequencing analysis for rare disease diagnosis and explore areas for future optimization. Identification of diagnostic variants in unsolved families promotes synergy between researchers with clinical and computational expertise as a means of advancing the field of clinical genome interpretation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.23293524">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.23293524" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.23293524">
        <p class="paperTitle">Differences in pathogenetic mechanism between Tibetan and Han high-altitude polycythemia based on a whole genome-wide association study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.23293524" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.23293524" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhuoma, B.; Shixuan, Z.; Xianwei, K.; Zhuoma, D.; Danzeng, Q.; La, Y.; Yang, D.; Meng, H.; Zixin, H.; Yanyun, M.; Zeshan, L.; Yi, L.; Jiucun, W.; Juan, W.</p>
        <p class="info">Score: 1.2, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.23293524' target='https://doi.org/10.1101/2023.08.05.23293524'> 10.1101/2023.08.05.23293524</a></p>
        <p class="abstract">High-altitude polycythemia (HAPC) is a common and serious chronic disease affecting people of highland and plains ancestry living at high altitudes. This study investigated genetic susceptibility differences for HAPC among ethnic groups, with 2,248 volunteers participating, including 898 HAPC patients (nHan = 450, nTibetan = 448). The study included a GWAS of 198 cases (nHan = 100, nTibetan = 98), which revealed eight Tibetan HAPC-susceptibility single-nucleotide polymorphisms and four Han HAPC-susceptibility VARIANTs. Among them, the common polymorphism locus rs7618658 (SNX4, Pcombine&lt;5e-8) was found and verified in both Tibetans and Han Chinese. Furthermore, the exploration of Tibetan EPAS1 suggested that the rs1374749 locus of EPAS1, along with other locus-linked status changes, may be a prevalence factor for HAPC, and the GGTAC haplotype in which it is located is a risk haplotype for HAPC (P = 5.14E-9, OR = 1.85).The study showed significant genetic susceptibility to HAPC between Tibetans and Han Chinese, with Tibetans mainly susceptible in oxygen-sensing pathways such as EPAS1, which are associated with several phenotypes (Hemoglobin, Hematocrit, and Platelets, etc), while Han Chinese patients exhibited susceptibilities mainly in cell differentiation and angiogenesis, such as in SNX4 and LPAR1, which are similarly significantly associated with hemoglobin, hematocrit, and platelets.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
