<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 187.1, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nExome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank\nAuthors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.\nScore: 20.6, Published: 2023-07-03 DOI: 10.1101/2023.06.29.23291992\nExome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF \u003c 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( \u003c 1.68 x 10-7) and 17 nominally significant ( \u003c 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR \u003c 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.\nPOPULATION FREQUENCY OF REPEAT EXPANSIONS INDICATES INCREASED DISEASE PREVALENCE ESTIMATES ACROSS DIFFERENT POPULATIONS\nAuthors: Ibanez Garikano, K.; Jadhav, B.; Facchini, S.; Garg, P.; Zanovello, M.; Martin Trujillo, A.; Gies, S.; Galassi Deforie, V.; Gagliardi, D.; Hensman Moss, D.; Moutsianas, L.; EUROSCA, n.; Genomics England Research Consortium, ; Shoai, M.; Caulifield, M. J.; Cortese, A.; Escott Price, V.; Hardy, J.; Houlden, H.; Sharp, A. J.; Tucci, A.\nScore: 15.8, Published: 2023-07-06 DOI: 10.1101/2023.07.03.23292162\nRepeat expansion disorders (REDs) are a devastating group of predominantly neurological diseases. Together they are common, affecting 1 in 3,000 people worldwide with population-specific differences. However, prevalence estimates of REDs are hampered by heterogeneous clinical presentation, variable geographic distributions, and technological limitations leading to under-ascertainment. Here, leveraging whole genome sequencing data from 82,176 individuals from different populations we found an overall carrier frequency of REDs of 1 in 340 individuals. Modelling disease prevalence using genetic data, age at onset and survival, we show that REDs are up to 3-fold more prevalent than currently reported figures. While some REDs are population-specific, e.g. Huntington disease type 2, most REDs are represented in all broad genetic ancestries, including Africans and Asians, challenging the notion that some REDs are found only in European populations. These results have worldwide implications for local and global health communities in the diagnosis and management of REDs both at local and global levels.\nGenome-wide meta-analysis of CSF biomarkers in Alzheimer's disease and Parkinson's disease cohorts\nAuthors: Ta, M.; Blauwendraat, C.; Antar, T.; Leonard, H. L.; Singleton, A. B.; Nalls, M.; Iwaki, H.; Alzheimers Disease Neuroimaging Initiative (ADNI), ; the Fox Investigation for New Discovery of Biomarkers, Score: 9.2, Published: 2023-06-19 DOI: 10.1101/2023.06.13.23291354\nBackgroundAmyloid beta (A{beta}), phosphorylated tau (p-tau), and total tau (t-tau) in cerebrospinal fluid are established biomarkers for Alzheimers disease (AD). In other neurodegenerative diseases, such as Parkinsons disease (PD), these biomarkers have also been found to be altered, and the molecular mechanisms responsible for these alterations are still under investigation. Moreover, the interplay between these mechanisms and the diverse underlying disease states remains to be elucidated. ObjectivesTo investigate genetic contributions to the AD biomarkers and assess the commonality and heterogeneity of the associations per underlying disease status. MethodsWe conducted GWAS for the AD biomarkers on subjects from the Parkinsons Progression Markers Initiative (PPMI), the Fox Investigation for New Discovery of Biomarkers (BioFIND), and the Alzheimers Disease Neuroimaging Initiative (ADNI) and meta-analyzed with the largest AD GWAS.[7] We tested heterogeneity of associations of interest between different disease statuses (AD, PD, and control). ResultsWe observed three GWAS signals: the APOE locus for A{beta}, the 3q28 locus between GEMC1 and OSTN for p-tau and t-tau, and the 7p22 locus (top hit: rs60871478, an intronic variant for DNAAF5, also known as HEATR2) for p-tau. The 7p22 locus is novel and co-localized with the brain DNAAF5 expression. While no heterogeneity from underlying disease status was observed for the above GWAS signals, some disease risk loci suggested disease specific associations with these biomarkers. ConclusionsOur study identified a novel association at the intronic region of DNAAF5 associated with increased levels of p-tau across all diseases. We also observed some disease specific genetic associations with these biomarkers.\nDiversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program\nAuthors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.; Shi, Y.; Nandi, T. N.; Assimes, T. L.; Brunette, C. A.; Carroll, R. J.; Clifford, R.; Duvall, S.; Gelernter, J.; Hung, A.; Iyengar, S. K.; Joseph, J.; Kember, R.; Kranzler, H.; Levey, D.; Luoh, S.-W.; Merritt, V. C.; Overstreet, C.; Deak, J. D.; Grant, S. F. A.; Poli\nScore: 29.8, Published: 2023-06-29 DOI: 10.1101/2023.06.28.23291975\nGenome-wide association studies (GWAS) have underrepresented individuals from non-European populations, impeding progress in characterizing the genetic architecture and consequences of health and disease traits. To address this, we present a population-stratified phenome-wide GWAS followed by a multi-population meta-analysis for 2,068 traits derived from electronic health records of 635,969 participants in the Million Veteran Program (MVP), a longitudinal cohort study of diverse U.S. Veterans genetically similar to the respective African (121,177), Admixed American (59,048), East Asian (6,702), and European (449,042) superpopulations defined by the 1000 Genomes Project. We identified 38,270 independent variants associating with one or more traits at experiment-wide (P \u003c 4.6x10-11) significance; fine-mapping 6,318 signals identified from 613 traits to single-variant resolution. Among these, a third (2,069) of the associations were found only among participants genetically similar to non-European reference populations, demonstrating the importance of expanding diversity in genetic studies. Our work provides a comprehensive atlas of phenome-wide genetic associations for future studies dissecting the architecture of complex traits in diverse populations. One Sentence SummaryTo address the underrepresentation of non-European individuals in genome-wide association studies (GWAS), we conducted a population-stratified phenome-wide GWAS across 2,068 traits in 635,969 participants from the diverse U.S. Department of Veterans Affairs Million Veteran Program, with results expanding our knowledge of variant-trait associations and highlighting the importance of genetic diversity in understanding the architecture of complex health and disease traits.\nA systematic assessment of the impact of rare canonical splice site variants on splicing using functional and in silico methods\nAuthors: Oh, R. Y.; AlMail, A.; Cheerie, D.; Guirguis, G.; Hou, H.; Yuki, K. E.; Haque, B.; Thiruvahindrapuram, B.; Marshall, C. R.; Mendoza-Londono, R.; Shlien, A.; Kyriakopoulou, L. G.; Walker, S.; Dowling, J. J.; Wilson, M. D.; Costain, G.\nScore: 5.3, Published: 2023-07-06 DOI: 10.1101/2023.06.29.23292012\nBackground/Objectives: Canonical splice site variants (CSSVs) are often presumed to cause loss-of-function (LoF) and are assigned very strong evidence of pathogenicity (according to ACMG criterion PVS1). However, the exact nature and predictability of splicing effects of unselected rare CSSVs in blood-expressed genes is poorly understood. Methods: A total of 184 rare CSSVs in unselected blood-expressed genes were identified by genome sequencing in 121 individuals, and their impact on splicing was interrogated manually in RNA sequencing (RNA-seq) data. Blind to these RNA-seq data, we attempted to predict the precise impact of CSSVs by applying in silico tools and the ClinGen Sequence Variant Interpretation Working Group 2018 guidelines for applying PVS1 criterion. Results: There was no evidence of a frameshift nor of reduced expression consistent with nonsense-mediated decay for 24% of CSSVs: 17% had wildtype splicing only and normal junction depths, 3.25% resulted in cryptic splice site usage and in-frame indels, 3.25% resulted in full exon skipping (in-frame), and 0.5% resulted in full intron inclusion (in-frame). The predicted impact on splicing using (i) SpliceAI, (ii) MaxEntScan, and (iii) AutoPVS1, an automatic classification tool for PVS1 interpretation of null variants that utilizes Ensembl Variant Effect Predictor and MaxEntScan, was concordant with the RNA-seq analyses for 55%, 55% and 58% of CSSVs, respectively. Conclusion: Nearly 1 in 4 CSSVs may not cause LoF based on analysis of RNA-seq data. Predictions from in silico methods were often discordant with findings from RNA-seq. More caution may be warranted in applying PVS1-level evidence to CSSVs in the absence of functional data.\nNon-coding variants are a rare cause of recessive developmental disorders in trans with coding variants\nAuthors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium, ; Baralle, D.; Martin, H. C.; Whiffin, N.\nScore: 25.7, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291805\nPurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown. MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing. ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA). ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.\nLongitudinal multi-omics study reveals common etiology underlying association between plasma proteome and BMI trajectories in adolescent and young adult twins\nAuthors: Drouard, G.; Hagenbeek, F. A.; Whipp, A.; Pool, R.; Hottenga, J.-J.; Jansen, R.; Hubers, N.; Afonin, A.; BIOS Consortium, ; BBMRI-NL Metabolomics Consortium, ; Willemsen, G.; de Geus, E. J. C.; Ripatti, S.; Pirinen, M.; Kanninen, K. M.; Boomsma, D. I.; van Dongen, J.; Kaprio, J.\nScore: 4.3, Published: 2023-07-01 DOI: 10.1101/2023.06.28.23291995\nBackgroundThe influence of genetics and environment on the association of the plasma proteome with body mass index (BMI) and changes in BMI remain underexplored, and the links to other omics in these associations remain to be investigated. We characterized protein-BMI trajectory associations in adolescents and adults and how these connect to other omics layers. MethodsOur study included two cohorts of longitudinally followed twins: FinnTwin12 (N=651) and the Netherlands Twin Register (NTR) (N=665). Follow-up comprised four BMI measurements over approximately 6 (NTR: 23-27 years old) to 10 years (FinnTwin12: 12-22 years old), with omics data collected at the last BMI measurement. BMI changes were calculated using latent growth curve models. Mixed-effects models were used to quantify the associations between the abundance of 439 plasma proteins with BMI at blood sampling and changes in BMI. The sources of genetic and environmental variation underlying the protein abundances were quantified using twin models, as were the associations of proteins with BMI and BMI changes. In NTR, we investigated the association of gene expression of genes encoding proteins identified in FinnTwin12 with BMI and changes in BMI. We linked identified proteins and their coding genes to plasma metabolites and polygenic risk scores (PRS) using mixed-effect models and correlation networks. ResultsWe identified 66 and 14 proteins associated with BMI at blood sampling and changes in BMI, respectively. The average heritability of these proteins was 35%. Of the 66 BMI-protein associations, 43 and 12 showed genetic and environmental correlations, respectively, including 8 proteins showing both. Similarly, we observed 6 and 4 genetic and environmental correlations between changes in BMI and protein abundance, respectively. S100A8 gene expression was associated with BMI at blood sampling, and the PRG4 and CFI genes were associated with BMI changes. Proteins showed strong connections with many metabolites and PRSs, but we observed no multi-omics connections among gene expression and other omics layers. ConclusionsAssociations between the proteome and BMI trajectories are characterized by shared genetic, environmental, and metabolic etiologies. We observed few gene-protein pairs associated with BMI or changes in BMI at the proteome and transcriptome levels.\nThe broad impact of cell death genes on the human disease phenome\nAuthors: Rich, A. L.; Lin, P.; Gamazon, E. R.; Zinkel, S.\nScore: 4.5, Published: 2023-06-12 DOI: 10.1101/2023.06.11.23291256\nApoptotic, necroptotic, and pyroptotic cell death pathways are attractive and druggable targets for many human diseases, however the tissue specificity of these pathways and the relationship between these pathways and human disease is poorly characterized. Understanding the impact of modulating cell death gene expression on the human phenome could inform clinical investigation of cell death pathway-modulating therapeutics in human disorders by identifying novel trait associations and by detecting tissue-specific side effect profiles. We analyzed the expression profiles of an array of 44 cell death genes across somatic tissues in GTEx v8 and investigated the relationship between tissue-specific genetically determined expression of 44 cell death genes and the human phenome using summary statistics-based transcriptome wide association studies (TWAS) on human traits in the UK Biobank V3 (n [~]500,000). We evaluated 513 traits encompassing ICD-10 defined diagnoses and hematologic traits (blood count labs). Our analysis revealed hundreds of significant (FDR\u003c0.05) associations between cell death gene expression and diverse human phenotypes, which were independently validated in another large-scale biobank. Cell death genes were highly enriched for significant associations with blood traits versus non-cell-death genes, with apoptosis-associated genes enriched for leukocyte and platelet traits and necroptosis gene associations enriched for erythroid traits (e.g., Reticulocyte count, FDR=0.004). This suggests that immunogenic cell death pathways play an important role in regulating erythropoiesis and reinforces the paradigm that apoptosis pathway genes are critical for white blood cell and platelet development. Of functionally analogous genes, for instance pro-survival BCL2 family members, trait/direction-of-effect relationships were heterogeneous across blood traits. Overall, these results suggest that even functionally similar and/or orthologous cell death genes play distinct roles in their contribution to human phenotypes, and that cell death genes influence a diverse array of human traits.\nImproved Type 2 Diabetes Risk Stratification in the Qatar Biobank Cohort by Ensemble Learning Classifier Incorporating Multi-Trait, Population-Specific, Polygenic Risk Scores\nAuthors: Ahmed, I.; Ziab, M.; Mbarek, H.; Taheri, S.; Chagoury, O. L.; Hussain, S. A.; Lakshmi, J.; Bhat, A.; Fakhro, K. A.; Al-Shabeeb AKIL, A. S.\nScore: 12.3, Published: 2023-06-25 DOI: 10.1101/2023.06.23.23291830\nBackgroundType 2 Diabetes (T2D) is a pervasive chronic disease influenced by a complex interplay of environmental and genetic factors. To enhance T2D risk prediction, leveraging genetic information is essential, with polygenic risk scores (PRS) offering a promising tool for assessing individual genetic risk. Our study focuses on the comparison between multi-trait and single-trait PRS models and demonstrates how the incorporation of multi-trait PRS into risk prediction models can significantly augment T2D risk assessment accuracy and effectiveness. MethodsWe conducted genome-wide association studies (GWAS) on 12 distinct T2D-related traits within a cohort of 14,278 individuals, all sequenced under the Qatar Genome Programme (QGP). This in-depth genetic analysis yielded several novel genetic variants associated with T2D, which served as the foundation for constructing multiple weighted PRS models. To assess the cumulative risk from these predictors, we applied machine learning (ML) techniques, which allowed for a thorough risk assessment. ResultsOur research identified genetic variations tied to T2D risk and facilitated the construction of ML models integrating PRS predictors for an exhaustive risk evaluation. The top-performing ML model demonstrated a robust performance with an accuracy of 0.8549, AUC of 0.92, AUC-PR of 0.8522, and an F1 score of 0.757, reflecting its strong capacity to differentiate cases from controls. We are currently working on acquiring independent T2D cohorts to validate the efficacy of our final model. ConclusionOur research underscores the potential of PRS models in identifying individuals within the population who are at elevated risk of developing T2D and its associated complications. The use of multi-trait PRS and ML models for risk prediction could inform early interventions, potentially identifying T2D patients who stand to benefit most based on their individual genetic risk profile. This combined approach signifies a stride forward in the field of precision medicine, potentially enhancing T2D risk prediction, prevention, and management.\n",
  "wordCount" : "3115",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 187.1, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23291992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23291992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23291992">
        <p class="paperTitle">Exome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23291992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23291992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.</p>
        <p class="info">Score: 20.6, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23291992' target='https://doi.org/10.1101/2023.06.29.23291992'> 10.1101/2023.06.29.23291992</a></p>
        <p class="abstract">Exome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF &lt; 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( &lt; 1.68 x 10-7) and 17 nominally significant ( &lt; 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR &lt; 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.23292162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.23292162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.23292162">
        <p class="paperTitle">POPULATION FREQUENCY OF REPEAT EXPANSIONS INDICATES INCREASED DISEASE PREVALENCE ESTIMATES ACROSS DIFFERENT POPULATIONS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.23292162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.23292162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ibanez Garikano, K.; Jadhav, B.; Facchini, S.; Garg, P.; Zanovello, M.; Martin Trujillo, A.; Gies, S.; Galassi Deforie, V.; Gagliardi, D.; Hensman Moss, D.; Moutsianas, L.; EUROSCA, n.; Genomics England Research Consortium,  ; Shoai, M.; Caulifield, M. J.; Cortese, A.; Escott Price, V.; Hardy, J.; Houlden, H.; Sharp, A. J.; Tucci, A.</p>
        <p class="info">Score: 15.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.23292162' target='https://doi.org/10.1101/2023.07.03.23292162'> 10.1101/2023.07.03.23292162</a></p>
        <p class="abstract">Repeat expansion disorders (REDs) are a devastating group of predominantly neurological diseases. Together they are common, affecting 1 in 3,000 people worldwide with population-specific differences. However, prevalence estimates of REDs are hampered by heterogeneous clinical presentation, variable geographic distributions, and technological limitations leading to under-ascertainment. Here, leveraging whole genome sequencing data from 82,176 individuals from different populations we found an overall carrier frequency of REDs of 1 in 340 individuals. Modelling disease prevalence using genetic data, age at onset and survival, we show that REDs are up to 3-fold more prevalent than currently reported figures. While some REDs are population-specific, e.g. Huntington disease type 2, most REDs are represented in all broad genetic ancestries, including Africans and Asians, challenging the notion that some REDs are found only in European populations. These results have worldwide implications for local and global health communities in the diagnosis and management of REDs both at local and global levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.23291354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.23291354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.23291354">
        <p class="paperTitle">Genome-wide meta-analysis of CSF biomarkers in Alzheimer&#39;s disease and Parkinson&#39;s disease cohorts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.23291354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.23291354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ta, M.; Blauwendraat, C.; Antar, T.; Leonard, H. L.; Singleton, A. B.; Nalls, M.; Iwaki, H.; Alzheimers Disease Neuroimaging Initiative (ADNI),  ; the Fox Investigation for New Discovery of Biomarkers,  </p>
        <p class="info">Score: 9.2, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.23291354' target='https://doi.org/10.1101/2023.06.13.23291354'> 10.1101/2023.06.13.23291354</a></p>
        <p class="abstract">BackgroundAmyloid beta (A{beta}), phosphorylated tau (p-tau), and total tau (t-tau) in cerebrospinal fluid are established biomarkers for Alzheimers disease (AD). In other neurodegenerative diseases, such as Parkinsons disease (PD), these biomarkers have also been found to be altered, and the molecular mechanisms responsible for these alterations are still under investigation. Moreover, the interplay between these mechanisms and the diverse underlying disease states remains to be elucidated.

ObjectivesTo investigate genetic contributions to the AD biomarkers and assess the commonality and heterogeneity of the associations per underlying disease status.

MethodsWe conducted GWAS for the AD biomarkers on subjects from the Parkinsons Progression Markers Initiative (PPMI), the Fox Investigation for New Discovery of Biomarkers (BioFIND), and the Alzheimers Disease Neuroimaging Initiative (ADNI) and meta-analyzed with the largest AD GWAS.[7] We tested heterogeneity of associations of interest between different disease statuses (AD, PD, and control).

ResultsWe observed three GWAS signals: the APOE locus for A{beta}, the 3q28 locus between GEMC1 and OSTN for p-tau and t-tau, and the 7p22 locus (top hit: rs60871478, an intronic variant for DNAAF5, also known as HEATR2) for p-tau. The 7p22 locus is novel and co-localized with the brain DNAAF5 expression. While no heterogeneity from underlying disease status was observed for the above GWAS signals, some disease risk loci suggested disease specific associations with these biomarkers.

ConclusionsOur study identified a novel association at the intronic region of DNAAF5 associated with increased levels of p-tau across all diseases. We also observed some disease specific genetic associations with these biomarkers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.23291975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.23291975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.23291975">
        <p class="paperTitle">Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.23291975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.23291975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.; Shi, Y.; Nandi, T. N.; Assimes, T. L.; Brunette, C. A.; Carroll, R. J.; Clifford, R.; Duvall, S.; Gelernter, J.; Hung, A.; Iyengar, S. K.; Joseph, J.; Kember, R.; Kranzler, H.; Levey, D.; Luoh, S.-W.; Merritt, V. C.; Overstreet, C.; Deak, J. D.; Grant, S. F. A.; Poli</p>
        <p class="info">Score: 29.8, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.23291975' target='https://doi.org/10.1101/2023.06.28.23291975'> 10.1101/2023.06.28.23291975</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have underrepresented individuals from non-European populations, impeding progress in characterizing the genetic architecture and consequences of health and disease traits. To address this, we present a population-stratified phenome-wide GWAS followed by a multi-population meta-analysis for 2,068 traits derived from electronic health records of 635,969 participants in the Million Veteran Program (MVP), a longitudinal cohort study of diverse U.S. Veterans genetically similar to the respective African (121,177), Admixed American (59,048), East Asian (6,702), and European (449,042) superpopulations defined by the 1000 Genomes Project. We identified 38,270 independent variants associating with one or more traits at experiment-wide (P &lt; 4.6x10-11) significance; fine-mapping 6,318 signals identified from 613 traits to single-variant resolution. Among these, a third (2,069) of the associations were found only among participants genetically similar to non-European reference populations, demonstrating the importance of expanding diversity in genetic studies. Our work provides a comprehensive atlas of phenome-wide genetic associations for future studies dissecting the architecture of complex traits in diverse populations.

One Sentence SummaryTo address the underrepresentation of non-European individuals in genome-wide association studies (GWAS), we conducted a population-stratified phenome-wide GWAS across 2,068 traits in 635,969 participants from the diverse U.S. Department of Veterans Affairs Million Veteran Program, with results expanding our knowledge of variant-trait associations and highlighting the importance of genetic diversity in understanding the architecture of complex health and disease traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292012" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292012">
        <p class="paperTitle">A systematic assessment of the impact of rare canonical splice site variants on splicing using functional and in silico methods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oh, R. Y.; AlMail, A.; Cheerie, D.; Guirguis, G.; Hou, H.; Yuki, K. E.; Haque, B.; Thiruvahindrapuram, B.; Marshall, C. R.; Mendoza-Londono, R.; Shlien, A.; Kyriakopoulou, L. G.; Walker, S.; Dowling, J. J.; Wilson, M. D.; Costain, G.</p>
        <p class="info">Score: 5.3, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292012' target='https://doi.org/10.1101/2023.06.29.23292012'> 10.1101/2023.06.29.23292012</a></p>
        <p class="abstract">Background/Objectives: Canonical splice site variants (CSSVs) are often presumed to cause loss-of-function (LoF) and are assigned very strong evidence of pathogenicity (according to ACMG criterion PVS1). However, the exact nature and predictability of splicing effects of unselected rare CSSVs in blood-expressed genes is poorly understood. Methods: A total of 184 rare CSSVs in unselected blood-expressed genes were identified by genome sequencing in 121 individuals, and their impact on splicing was interrogated manually in RNA sequencing (RNA-seq) data. Blind to these RNA-seq data, we attempted to predict the precise impact of CSSVs by applying in silico tools and the ClinGen Sequence Variant Interpretation Working Group 2018 guidelines for applying PVS1 criterion. Results: There was no evidence of a frameshift nor of reduced expression consistent with nonsense-mediated decay for 24% of CSSVs: 17% had wildtype splicing only and normal junction depths, 3.25% resulted in cryptic splice site usage and in-frame indels, 3.25% resulted in full exon skipping (in-frame), and 0.5% resulted in full intron inclusion (in-frame). The predicted impact on splicing using (i) SpliceAI, (ii) MaxEntScan, and (iii) AutoPVS1, an automatic classification tool for PVS1 interpretation of null variants that utilizes Ensembl Variant Effect Predictor and MaxEntScan, was concordant with the RNA-seq analyses for 55%, 55% and 58% of CSSVs, respectively. Conclusion: Nearly 1 in 4 CSSVs may not cause LoF based on analysis of RNA-seq data. Predictions from in silico methods were often discordant with findings from RNA-seq. More caution may be warranted in applying PVS1-level evidence to CSSVs in the absence of functional data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291805">
        <p class="paperTitle">Non-coding variants are a rare cause of recessive developmental disorders in trans with coding variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium,  ; Baralle, D.; Martin, H. C.; Whiffin, N.</p>
        <p class="info">Score: 25.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291805' target='https://doi.org/10.1101/2023.06.23.23291805'> 10.1101/2023.06.23.23291805</a></p>
        <p class="abstract">PurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown.

MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing.

ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA).

ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.23291995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.23291995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.23291995">
        <p class="paperTitle">Longitudinal multi-omics study reveals common etiology underlying association between plasma proteome and BMI trajectories in adolescent and young adult twins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.23291995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.23291995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drouard, G.; Hagenbeek, F. A.; Whipp, A.; Pool, R.; Hottenga, J.-J.; Jansen, R.; Hubers, N.; Afonin, A.; BIOS Consortium,  ; BBMRI-NL Metabolomics Consortium,  ; Willemsen, G.; de Geus, E. J. C.; Ripatti, S.; Pirinen, M.; Kanninen, K. M.; Boomsma, D. I.; van Dongen, J.; Kaprio, J.</p>
        <p class="info">Score: 4.3, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.23291995' target='https://doi.org/10.1101/2023.06.28.23291995'> 10.1101/2023.06.28.23291995</a></p>
        <p class="abstract">BackgroundThe influence of genetics and environment on the association of the plasma proteome with body mass index (BMI) and changes in BMI remain underexplored, and the links to other omics in these associations remain to be investigated. We characterized protein-BMI trajectory associations in adolescents and adults and how these connect to other omics layers.

MethodsOur study included two cohorts of longitudinally followed twins: FinnTwin12 (N=651) and the Netherlands Twin Register (NTR) (N=665). Follow-up comprised four BMI measurements over approximately 6 (NTR: 23-27 years old) to 10 years (FinnTwin12: 12-22 years old), with omics data collected at the last BMI measurement. BMI changes were calculated using latent growth curve models. Mixed-effects models were used to quantify the associations between the abundance of 439 plasma proteins with BMI at blood sampling and changes in BMI. The sources of genetic and environmental variation underlying the protein abundances were quantified using twin models, as were the associations of proteins with BMI and BMI changes. In NTR, we investigated the association of gene expression of genes encoding proteins identified in FinnTwin12 with BMI and changes in BMI. We linked identified proteins and their coding genes to plasma metabolites and polygenic risk scores (PRS) using mixed-effect models and correlation networks.

ResultsWe identified 66 and 14 proteins associated with BMI at blood sampling and changes in BMI, respectively. The average heritability of these proteins was 35%. Of the 66 BMI-protein associations, 43 and 12 showed genetic and environmental correlations, respectively, including 8 proteins showing both. Similarly, we observed 6 and 4 genetic and environmental correlations between changes in BMI and protein abundance, respectively. S100A8 gene expression was associated with BMI at blood sampling, and the PRG4 and CFI genes were associated with BMI changes. Proteins showed strong connections with many metabolites and PRSs, but we observed no multi-omics connections among gene expression and other omics layers.

ConclusionsAssociations between the proteome and BMI trajectories are characterized by shared genetic, environmental, and metabolic etiologies. We observed few gene-protein pairs associated with BMI or changes in BMI at the proteome and transcriptome levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.11.23291256">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.11.23291256" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.11.23291256">
        <p class="paperTitle">The broad impact of cell death genes on the human disease phenome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.11.23291256" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.11.23291256" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rich, A. L.; Lin, P.; Gamazon, E. R.; Zinkel, S.</p>
        <p class="info">Score: 4.5, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.11.23291256' target='https://doi.org/10.1101/2023.06.11.23291256'> 10.1101/2023.06.11.23291256</a></p>
        <p class="abstract">Apoptotic, necroptotic, and pyroptotic cell death pathways are attractive and druggable targets for many human diseases, however the tissue specificity of these pathways and the relationship between these pathways and human disease is poorly characterized. Understanding the impact of modulating cell death gene expression on the human phenome could inform clinical investigation of cell death pathway-modulating therapeutics in human disorders by identifying novel trait associations and by detecting tissue-specific side effect profiles. We analyzed the expression profiles of an array of 44 cell death genes across somatic tissues in GTEx v8 and investigated the relationship between tissue-specific genetically determined expression of 44 cell death genes and the human phenome using summary statistics-based transcriptome wide association studies (TWAS) on human traits in the UK Biobank V3 (n [~]500,000). We evaluated 513 traits encompassing ICD-10 defined diagnoses and hematologic traits (blood count labs). Our analysis revealed hundreds of significant (FDR&lt;0.05) associations between cell death gene expression and diverse human phenotypes, which were independently validated in another large-scale biobank. Cell death genes were highly enriched for significant associations with blood traits versus non-cell-death genes, with apoptosis-associated genes enriched for leukocyte and platelet traits and necroptosis gene associations enriched for erythroid traits (e.g., Reticulocyte count, FDR=0.004). This suggests that immunogenic cell death pathways play an important role in regulating erythropoiesis and reinforces the paradigm that apoptosis pathway genes are critical for white blood cell and platelet development. Of functionally analogous genes, for instance pro-survival BCL2 family members, trait/direction-of-effect relationships were heterogeneous across blood traits. Overall, these results suggest that even functionally similar and/or orthologous cell death genes play distinct roles in their contribution to human phenotypes, and that cell death genes influence a diverse array of human traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291830">
        <p class="paperTitle">Improved Type 2 Diabetes Risk Stratification in the Qatar Biobank Cohort by Ensemble Learning Classifier Incorporating Multi-Trait, Population-Specific, Polygenic Risk Scores</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmed, I.; Ziab, M.; Mbarek, H.; Taheri, S.; Chagoury, O. L.; Hussain, S. A.; Lakshmi, J.; Bhat, A.; Fakhro, K. A.; Al-Shabeeb AKIL, A. S.</p>
        <p class="info">Score: 12.3, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291830' target='https://doi.org/10.1101/2023.06.23.23291830'> 10.1101/2023.06.23.23291830</a></p>
        <p class="abstract">BackgroundType 2 Diabetes (T2D) is a pervasive chronic disease influenced by a complex interplay of environmental and genetic factors. To enhance T2D risk prediction, leveraging genetic information is essential, with polygenic risk scores (PRS) offering a promising tool for assessing individual genetic risk. Our study focuses on the comparison between multi-trait and single-trait PRS models and demonstrates how the incorporation of multi-trait PRS into risk prediction models can significantly augment T2D risk assessment accuracy and effectiveness.

MethodsWe conducted genome-wide association studies (GWAS) on 12 distinct T2D-related traits within a cohort of 14,278 individuals, all sequenced under the Qatar Genome Programme (QGP). This in-depth genetic analysis yielded several novel genetic variants associated with T2D, which served as the foundation for constructing multiple weighted PRS models. To assess the cumulative risk from these predictors, we applied machine learning (ML) techniques, which allowed for a thorough risk assessment.

ResultsOur research identified genetic variations tied to T2D risk and facilitated the construction of ML models integrating PRS predictors for an exhaustive risk evaluation. The top-performing ML model demonstrated a robust performance with an accuracy of 0.8549, AUC of 0.92, AUC-PR of 0.8522, and an F1 score of 0.757, reflecting its strong capacity to differentiate cases from controls. We are currently working on acquiring independent T2D cohorts to validate the efficacy of our final model.

ConclusionOur research underscores the potential of PRS models in identifying individuals within the population who are at elevated risk of developing T2D and its associated complications. The use of multi-trait PRS and ML models for risk prediction could inform early interventions, potentially identifying T2D patients who stand to benefit most based on their individual genetic risk profile. This combined approach signifies a stride forward in the field of precision medicine, potentially enhancing T2D risk prediction, prevention, and management.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
