<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-cell RNA sequencing reveals dysregulated cellular programmes in the inflamed epithelium of Crohn&#39;s disease patients.
Authors: Krzak, M.; Alegbe, T.; Taylor, D. L.; Ghouraba, M. H.; Strickland, M.; Satti, R.; Thompson, T.; Arestang, K.; Przybilla, M. J.; Ramirez-Navarro, L.; Harris, B. T.; Cheam, K. A. X.; Noell, G.; Leonard, S.; Petrova, V.; Jones-Bell, C.; James, K. R.; Wana, N.; Hu, M. X.; Skelton, J.; Ostermayer, J.; Gu, Y.; Dawson, C.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Single-cell RNA sequencing reveals dysregulated cellular programmes in the inflamed epithelium of Crohn&#39;s disease patients.
Authors: Krzak, M.; Alegbe, T.; Taylor, D. L.; Ghouraba, M. H.; Strickland, M.; Satti, R.; Thompson, T.; Arestang, K.; Przybilla, M. J.; Ramirez-Navarro, L.; Harris, B. T.; Cheam, K. A. X.; Noell, G.; Leonard, S.; Petrova, V.; Jones-Bell, C.; James, K. R.; Wana, N.; Hu, M. X.; Skelton, J.; Ostermayer, J.; Gu, Y.; Dawson, C." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Single-cell RNA sequencing reveals dysregulated cellular programmes in the inflamed epithelium of Crohn&#39;s disease patients.
Authors: Krzak, M.; Alegbe, T.; Taylor, D. L.; Ghouraba, M. H.; Strickland, M.; Satti, R.; Thompson, T.; Arestang, K.; Przybilla, M. J.; Ramirez-Navarro, L.; Harris, B. T.; Cheam, K. A. X.; Noell, G.; Leonard, S.; Petrova, V.; Jones-Bell, C.; James, K. R.; Wana, N.; Hu, M. X.; Skelton, J.; Ostermayer, J.; Gu, Y.; Dawson, C."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Single-cell RNA sequencing reveals dysregulated cellular programmes in the inflamed epithelium of Crohn\u0026#39;s disease patients.\nAuthors: Krzak, M.; Alegbe, T.; Taylor, D. L.; Ghouraba, M. H.; Strickland, M.; Satti, R.; Thompson, T.; Arestang, K.; Przybilla, M. J.; Ramirez-Navarro, L.; Harris, B. T.; Cheam, K. A. X.; Noell, G.; Leonard, S.; Petrova, V.; Jones-Bell, C.; James, K. R.; Wana, N.; Hu, M. X.; Skelton, J.; Ostermayer, J.; Gu, Y.; Dawson, C.",
  "keywords": [
    
  ],
  "articleBody": " Single-cell RNA sequencing reveals dysregulated cellular programmes in the inflamed epithelium of Crohn's disease patients.\nAuthors: Krzak, M.; Alegbe, T.; Taylor, D. L.; Ghouraba, M. H.; Strickland, M.; Satti, R.; Thompson, T.; Arestang, K.; Przybilla, M. J.; Ramirez-Navarro, L.; Harris, B. T.; Cheam, K. A. X.; Noell, G.; Leonard, S.; Petrova, V.; Jones-Bell, C.; James, K. R.; Wana, N.; Hu, M. X.; Skelton, J.; Ostermayer, J.; Gu, Y.; Dawson, C.; Corridoni, D.; Cotobal, C.; Parkes, M.; Iyer, V.; Jones, G.-R.; McIntyre, R. E.; Raine, T.; Anderson, C. A.\nScore: 30.6, Published: 2023-09-08 DOI: 10.1101/2023.09.06.23295056\nCrohns disease (CD) is a complex inflammatory disorder of incompletely understood molecular aetiology. We generated a large single-cell RNA sequencing dataset from the terminal ileal biopsies of two independent cohorts comprising a total of 50 CD patients and 71 healthy controls. We performed transcriptomic analyses to reveal genes, cell types and mechanisms perturbed in CD, leveraging the power of the two cohorts to confirm our findings and assess replicability. In addition to mapping widespread alterations in cytokine signalling, we provide evidence of pan-epithelial upregulation of MHC class I genes and pathways in CD. Using non-negative matrix factorization we revealed intra- and inter-cellular upregulation of expression programmes such as G-protein coupled receptor signalling and interferon signalling, respectively, in CD. We observed an enrichment of CD heritability among marker genes for various activated T cell types and myeloid cells, supporting a causal role for these cell types in CD aetiology. Comparisons between our discovery and replication cohort revealed significant variation in differential gene-expression replicability across cell types. B, T and myeloid cells showed particularly poor replicability, suggesting caution should be exercised when interpreting unreplicated differential gene-expression results in these cell types. Overall, our results provide a rich resource for identifying cell type specific biomarkers of Crohns disease and identifying genes, cell types and pathways that are causally and replicably associated with disease.\nSystematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease\nAuthors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D'Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.\nScore: 28.4, Published: 2023-09-12 DOI: 10.1101/2023.09.12.23295416\nBackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown. MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls. ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations. ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.\nGenome-Wide Association Studies of Coffee Intake in UK/US Participants of European Ancestry Uncover Gene-Cohort Influences\nAuthors: Thorpe, H. H.; Fontanillas, P.; Pham, B.; Meredith, J. J.; Jennings, M. V.; Courchesne-Krak, N. S.; Vilar-Ribo, L.; Bianchi, S. B.; Mutz, J.; 23andMe Research Team, ; Elson, S. L.; Khokhar, J. Y.; Abdellaoui, A.; Davis, L. K.; Palmer, A. A.; Sanchez-Roige, S.\nScore: 25.9, Published: 2023-09-11 DOI: 10.1101/2023.09.09.23295284\nCoffee is one of the most widely consumed beverages. We performed a genome-wide association study (GWAS) of coffee intake in US-based 23andMe participants (N=130,153) and identified 7 significant loci, with many replicating in three multi-ancestral cohorts. We examined genetic correlations and performed a phenome-wide association study across thousands of biomarkers and health and lifestyle traits, then compared our results to the largest available GWAS of coffee intake from UK Biobank (UKB; N=334,659). The results of these two GWAS were highly discrepant. We observed positive genetic correlations between coffee intake and psychiatric illnesses, pain, and gastrointestinal traits in 23andMe that were absent or negative in UKB. Genetic correlations with cognition were negative in 23andMe but positive in UKB. The only consistent observations were positive genetic correlations with substance use and obesity. Our study shows that GWAS in different cohorts could capture cultural differences in the relationship between behavior and genetics.\nCo-expression in tissue-specific gene networks links genes in cancer-susceptibility loci to known somatic driver genes\nAuthors: Urzua-Traslavina, C. G.; van Lieshout, T.; Boulogne, F.; Domanegg, K.; Zidan, M.; Bakker, O. B.; Claringbould, A.; de Ridder, J.; Zwart, W.; Westra, H.-J.; Deelen, P.; Franke, L.\nScore: 14.9, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295254\nBackgroundThe genetic background of cancer remains complex and challenging to integrate. Many somatic mutations in genes are known to cause and drive cancer, while genome-wide association studies (GWAS) of cancer have revealed many germline risk factors associated with cancer. However, the overlap between known somatic driver genes and positional candidate genes from GWAS loci is surprisingly small. We hypothesised that genes from multiple independent cancer GWAS loci should show tissue-specific co-regulation patterns that converge on cancer-specific driver genes. ResultsWe studied recent well powered GWAS of breast, prostate, colorectal and skin cancer by estimating co-expression between genes and subsequently prioritising genes that show co- expression with genes mapping within susceptibility loci from cancer GWAS. We observed that the prioritised genes were strongly enriched for cancer drivers defined by COSMIC, intOGen and Dietlein et al. The enrichment of known cancer driver genes was most significant when using co-expression networks derived from non-cancer samples from the relevant tissue of origin. ConclusionWe show how genes in risk loci identified by cancer GWAS can be linked to known cancer driver genes through tissue-specific co-expression networks. This provides an important explanation for why seemingly unrelated sets of genes that harbour either germline risk factors or somatic mutations can eventually cause the same type of disease.\nRelationship between sex biases in gene expression and sex biases in autism and Alzheimer's disease\nAuthors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.\nScore: 14.2, Published: 2023-09-01 DOI: 10.1101/2023.08.29.23294773\nSex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimers disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=137 SRC=\"FIGDIR/small/23294773v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@7c198eorg.highwire.dtl.DTLVardef@171ea18org.highwire.dtl.DTLVardef@1bfe0dcorg.highwire.dtl.DTLVardef@43b98c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nCirculating anti-Müllerian hormone levels in pre-menopausal women: novel genetic insights from a GWAS meta-analysis\nAuthors: Pujol Gualdo, N.; Karjalainen, M. K.; Vosa, U.; Arffman, R. K.; Mägi, R.; Ronkainen, J.; Laisk, T.; Piltonen, T. T.\nScore: 5.0, Published: 2023-09-08 DOI: 10.1101/2023.09.07.23295182\nStudy questionCan a genome-wide association study (GWAS) meta-analysis, including a large sample of young premenopausal women from a founder population from Northern Finland, identify novel genetic variants for circulating anti-Mullerian hormone (AMH) levels and provide insights into biological pathways and tissues involved in AMH regulation? Summary answerWe identified six loci associated with AMH levels at P \u003c 5 x 10-8, including the previously reported MCM8, AMH and TEX41 loci, and three novel signals in or near CHEK2, BMP4 and EIF4EBP1. Gene set enrichment analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranks the pituitary gland as a top associated tissue. What is known alreadyAMH is expressed by preantral and small antral stage ovarian follicles in women, and variation in age-specific circulating AMH levels has been associated with several health conditions. However, the biological mechanisms underlying the association between health conditions and AMH levels are not yet fully understood. Previous GWAS have identified loci associated with AMH levels in pre-menopausal women, but they were limited by small sample sizes or focused mostly on older pre-menopausal women. Study design, size, durationWe performed a GWAS meta-analysis for AMH level measurements in 9,668 pre-menopausal women. Participants/materials, setting, methodsWe performed a GWAS meta-analysis in which we combined 2,619 AMH measurements (at age 31 years old) from a prospective founder population cohort (Northern Finland Birth Cohort 1966, NFBC1966) with a previous GWAS meta-analysis that included 7,049 pre-menopausal women (spanning age range 15-48). NFBC1966 AMH measurements were quantified using an automated assay (Elecsys(R) AMH Plus (Roche)). We annotated the genetic variants, combined different data layers to prioritise potential candidate genes, described significant pathways and tissues enriched by the GWAS signals, identified plausible regulatory roles using colocalization analysis and leveraged publicly available summary statistics to assess genetic and phenotypic correlations with multiple traits. Main results and the role of chanceThree novel genome-wide significant loci were identified. One of these is in complete linkage disequilibrium with c.1100delC in CHEK2, which is found to be 4-fold enriched in the Finnish population compared to other European populations. We propose a plausible regulatory effect of some of the GWAS variants linked to AMH, as they colocalise with GWAS signals associated with gene expression levels of BMP4, TEX41 and EIFBP41. Gene set analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranked the pituitary gland as the top association. Large scale dataThe GWAS meta-analysis summary statistics will be available for download from the GWAS Catalog. Accession numbers will be provided upon publication. Limitations, reasons for cautionThis study only included women of European ancestry and the unavailability of sufficiently sized relevant tissue data in gene expression datasets hinders the assessment of potential regulatory effects in reproductive tissues. Wider implications of the findingsOur results highlight the increased power of founder populations and larger sample sizes to boost the discovery of novel trait-associated variants underlying variation in AMH levels, which aided to characterise novel biological pathways and plausible genetic regulatory effects linked with AMH levels variation for the first time. Study funding / competing interest(s)This work has received funding from the European Unions Horizon 2020 research and innovation programme under the MATER Marie Sklodowska-Curie grant agreement No. 813707 and Oulu university scholarship foundation (N.P.-G.), Academy of Finland, Sigrid Juselius Foundation, Novo Nordisk, University of Oulu, Roche Diagnostics (T.T.P). This work was supported by the Estonian Research Council grant 1911 (R.M.). J.R. was supported by the European Unions Horizon 2020 research and innovation program under grant agreements No. 874739 (LongITools), 824989 (EUCAN- Connect), 848158 (EarlyCause) and 733206 (LifeCycle). U.V. was supported by the Estonian Research Council grant PRG (PRG1291). The NFBC1966 received financial support from University of Oulu Grant no. 24000692, Oulu University Hospital Grant no. 24301140, ERDF European Regional Development Fund Grant no. 539/2010 A31592.\nLong-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis\nAuthors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.\nScore: 11.8, Published: 2023-09-07 DOI: 10.1101/2023.09.05.23295093\nClonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in Communities Study (ARIC) participants over a median follow-up of 21 years. During this period, 735 participants developed incident CH. We found that age at baseline, sex, and dyslipidemia significantly influence the incidence of CH, while ASCVD and other traditional risk factors for ASCVD did not exhibit such associations. Our study also revealed associations between germline genetic variants and incident CH, prioritizing genes in CH development. Our comprehensive longitudinal assessment yields novel insights into the factors contributing to incident CH in older adults.\nThe pivotal role of the X-chromosome in the genetic architecture of the human brain\nAuthors: Jiang, Z.; Sullivan, P. F.; Li, T.; Zhao, B.; Wang, X.; Luo, T.; Huang, S.; Guan, P. Y.; Chen, J.; Yang, Y.; Stein, J. L.; Li, Y.; Liu, D.; Sun, L.; Zhu, H.\nScore: 8.4, Published: 2023-09-01 DOI: 10.1101/2023.08.30.23294848\nGenes on the X-chromosome are extensively expressed in the human brain, resulting in substantial influences on brain development, intellectual disability, and other brain-related disorders. To comprehensively investigate the X-chromosomes impact on the cerebral cortex, white matter tract microstructures, and intrinsic and extrinsic brain functions, we examined 2,822 complex brain imaging traits obtained from n = 34,000 subjects in the UK Biobank. We unveiled potential autosome-X-chromosome interaction, while proposing an atlas of dosage compensation (DC) for each set of traits. We observed a pronounced X-chromosome impact on the corticospinal tract and the functional amplitude and connectivity of visual networks. In association studies, we identified 50 genome-wide significant trait-locus pairs enriched in Xq28, 22 of which replicated in independent datasets (n = 4,900). Notably, 13 newly identified pairs were in the X-chromosomes non-pseudo-autosomal regions (NPR). The volume of the right ventral diencephalon shared genetic architecture with schizophrenia and educational attainment in a locus indexed by rs2361468 (located [~]3kb upstream of PJA1, a conserved and ubiquitously expressed gene implicated in multiple psychiatric disorders). No significant associations were identified in the pseudo-autosomal regions (PAR) or the Y-chromosome. Finally, we explored sex-specific associations on the X-chromosome and compared differing genetic effects between sexes. We found much more associations can be identified in males (33 versus 9) given a similar sample size. In conclusion, our research provides invaluable insights into the X-chromosomes role in the human brain, contributing to the observed sex differences in brain structure and function.\nStructural Variation Detection and Association Analysis of Whole-Genome-Sequence Data from 16,905 Alzheimer's Diseases Sequencing Project Subjects\nAuthors: Wang, H.; Dombroski, B. A.; Cheng, P.-L.; Tucci, A.; Si, Y.-q.; Farrell, J. J.; Tzeng, J.-Y.; Leung, Y. Y.; Malamon, J. S.; Wang, L.-S.; Vardarajan, B. N.; Farrer, L. A.; Schellenberg, G. D.; Lee, W.-P.\nScore: 1.2, Published: 2023-09-13 DOI: 10.1101/2023.09.13.23295505\nStructural variations (SVs) are important contributors to the genetics of numerous human diseases. However, their role in Alzheimers disease (AD) remains largely unstudied due to challenges in accurately detecting SVs. Here, we analyzed whole-genome sequencing data from the Alzheimers Disease Sequencing Project (ADSP, N=16,905 subjects) and identified 400,234 (168,223 high-quality) SVs. We found a significant burden of deletions and duplications in AD cases (OR=1.05, P=0.03), particularly for singletons (OR=1.12, P=0.0002) and homozygous events (OR=1.10, P\u003c0.0004). On AD genes, the ultra-rare SVs, including protein-altering SVs in ABCA7, APP, PLCG2, and SORL1, were associated with AD (SKAT-O P=0.004). Twenty-one SVs are in linkage disequilibrium (LD) with known AD-risk variants, e.g., a deletion (chr2:105731359-105736864) in complete LD (R2=0.99) with rs143080277 (chr2:105749599) in NCK2. We also identified 16 SVs associated with AD and 13 SVs associated with AD-related pathological/cognitive endophenotypes. Our findings demonstrate the broad impact of SVs on AD genetics. Search TermsAlzheimers disease, Structural variation, Copy number variation\nPrioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach\nAuthors: Pena-Martinez, E. G.; Pomales-Matos, D. A.; Rivera-Madera, A.; Messon-Bird, J. L.; Medina-Feliciano, J. G.; Sanabria-Alberto, L.; Barreiro-Rosario, A. C.; Rodriguez-Rios, J. M.; Rodriguez-Martinez, J. A.\nScore: 3.0, Published: 2023-09-02 DOI: 10.1101/2023.09.01.23294951\nCardiovascular diseases (CVDs) are the leading cause of death worldwide and are heavily influenced by genetic factors. Genome-wide association studies (GWAS) have mapped \u003e 90% of CVD-associated variants within the non-coding genome, which can alter the function of regulatory proteins, like transcription factors (TFs). However, due to the overwhelming number of GWAS single nucleotide polymorphisms (SNPs) (\u003e500,000), prioritizing variants for in vitro analysis remains challenging. In this work, we implemented a computational approach that considers support vector machine (SVM)-based TF binding site classification and cardiac expression quantitative trait loci (eQTL) analysis to identify and prioritize potential CVD-causing SNPs. We identified 1,535 CVD-associated SNPs that occur within human heart footprints/enhancers and 9,309 variants in linkage disequilibrium (LD) with differential gene expression profiles in cardiac tissue. Using hiPSC-CM ChIP-seq data from NKX2-5 and TBX5, two cardiac TFs essential for proper heart development, we trained a large-scale gapped k-mer SVM (LS-GKM-SVM) predictive model that can identify binding sites altered by CVD-associated SNPs. The computational predictive model was tested by scoring human heart footprints and enhancers in vitro through electrophoretic mobility shift assay (EMSA). Three variants (rs59310144, rs6715570, and rs61872084) were prioritized for in vitro validation based on their eQTL in cardiac tissue and LS-GKM-SVM prediction to alter NKX2-5 DNA binding. All three variants altered NKX2-5 DNA binding. In summary, we present a bioinformatic approach that considers tissue-specific eQTL analysis and SVM-based TF binding site classification to prioritize CVD-associated variants for in vitro experimental analysis. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=55 SRC=\"FIGDIR/small/23294951v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (18K): org.highwire.dtl.DTLVardef@3c03dforg.highwire.dtl.DTLVardef@224878org.highwire.dtl.DTLVardef@c3df2dorg.highwire.dtl.DTLVardef@17108e0_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "3001",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.23295056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.23295056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.23295056">
        <p class="paperTitle">Single-cell RNA sequencing reveals dysregulated cellular programmes in the inflamed epithelium of Crohn&#39;s disease patients.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.23295056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.23295056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krzak, M.; Alegbe, T.; Taylor, D. L.; Ghouraba, M. H.; Strickland, M.; Satti, R.; Thompson, T.; Arestang, K.; Przybilla, M. J.; Ramirez-Navarro, L.; Harris, B. T.; Cheam, K. A. X.; Noell, G.; Leonard, S.; Petrova, V.; Jones-Bell, C.; James, K. R.; Wana, N.; Hu, M. X.; Skelton, J.; Ostermayer, J.; Gu, Y.; Dawson, C.; Corridoni, D.; Cotobal, C.; Parkes, M.; Iyer, V.; Jones, G.-R.; McIntyre, R. E.; Raine, T.; Anderson, C. A.</p>
        <p class="info">Score: 30.6, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.23295056' target='https://doi.org/10.1101/2023.09.06.23295056'> 10.1101/2023.09.06.23295056</a></p>
        <p class="abstract">Crohns disease (CD) is a complex inflammatory disorder of incompletely understood molecular aetiology. We generated a large single-cell RNA sequencing dataset from the terminal ileal biopsies of two independent cohorts comprising a total of 50 CD patients and 71 healthy controls. We performed transcriptomic analyses to reveal genes, cell types and mechanisms perturbed in CD, leveraging the power of the two cohorts to confirm our findings and assess replicability. In addition to mapping widespread alterations in cytokine signalling, we provide evidence of pan-epithelial upregulation of MHC class I genes and pathways in CD. Using non-negative matrix factorization we revealed intra- and inter-cellular upregulation of expression programmes such as G-protein coupled receptor signalling and interferon signalling, respectively, in CD. We observed an enrichment of CD heritability among marker genes for various activated T cell types and myeloid cells, supporting a causal role for these cell types in CD aetiology. Comparisons between our discovery and replication cohort revealed significant variation in differential gene-expression replicability across cell types. B, T and myeloid cells showed particularly poor replicability, suggesting caution should be exercised when interpreting unreplicated differential gene-expression results in these cell types. Overall, our results provide a rich resource for identifying cell type specific biomarkers of Crohns disease and identifying genes, cell types and pathways that are causally and replicably associated with disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295416">
        <p class="paperTitle">Systematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D&#39;Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.</p>
        <p class="info">Score: 28.4, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295416' target='https://doi.org/10.1101/2023.09.12.23295416'> 10.1101/2023.09.12.23295416</a></p>
        <p class="abstract">BackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown.

MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls.

ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations.

ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.23295284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.23295284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.23295284">
        <p class="paperTitle">Genome-Wide Association Studies of Coffee Intake in UK/US Participants of European Ancestry Uncover Gene-Cohort Influences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.23295284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.23295284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thorpe, H. H.; Fontanillas, P.; Pham, B.; Meredith, J. J.; Jennings, M. V.; Courchesne-Krak, N. S.; Vilar-Ribo, L.; Bianchi, S. B.; Mutz, J.; 23andMe Research Team,  ; Elson, S. L.; Khokhar, J. Y.; Abdellaoui, A.; Davis, L. K.; Palmer, A. A.; Sanchez-Roige, S.</p>
        <p class="info">Score: 25.9, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.23295284' target='https://doi.org/10.1101/2023.09.09.23295284'> 10.1101/2023.09.09.23295284</a></p>
        <p class="abstract">Coffee is one of the most widely consumed beverages. We performed a genome-wide association study (GWAS) of coffee intake in US-based 23andMe participants (N=130,153) and identified 7 significant loci, with many replicating in three multi-ancestral cohorts. We examined genetic correlations and performed a phenome-wide association study across thousands of biomarkers and health and lifestyle traits, then compared our results to the largest available GWAS of coffee intake from UK Biobank (UKB; N=334,659). The results of these two GWAS were highly discrepant. We observed positive genetic correlations between coffee intake and psychiatric illnesses, pain, and gastrointestinal traits in 23andMe that were absent or negative in UKB. Genetic correlations with cognition were negative in 23andMe but positive in UKB. The only consistent observations were positive genetic correlations with substance use and obesity. Our study shows that GWAS in different cohorts could capture cultural differences in the relationship between behavior and genetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295254" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295254">
        <p class="paperTitle">Co-expression in tissue-specific gene networks links genes in cancer-susceptibility loci to known somatic driver genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Urzua-Traslavina, C. G.; van Lieshout, T.; Boulogne, F.; Domanegg, K.; Zidan, M.; Bakker, O. B.; Claringbould, A.; de Ridder, J.; Zwart, W.; Westra, H.-J.; Deelen, P.; Franke, L.</p>
        <p class="info">Score: 14.9, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295254' target='https://doi.org/10.1101/2023.09.08.23295254'> 10.1101/2023.09.08.23295254</a></p>
        <p class="abstract">BackgroundThe genetic background of cancer remains complex and challenging to integrate. Many somatic mutations in genes are known to cause and drive cancer, while genome-wide association studies (GWAS) of cancer have revealed many germline risk factors associated with cancer. However, the overlap between known somatic driver genes and positional candidate genes from GWAS loci is surprisingly small. We hypothesised that genes from multiple independent cancer GWAS loci should show tissue-specific co-regulation patterns that converge on cancer-specific driver genes.

ResultsWe studied recent well powered GWAS of breast, prostate, colorectal and skin cancer by estimating co-expression between genes and subsequently prioritising genes that show co- expression with genes mapping within susceptibility loci from cancer GWAS. We observed that the prioritised genes were strongly enriched for cancer drivers defined by COSMIC, intOGen and Dietlein et al. The enrichment of known cancer driver genes was most significant when using co-expression networks derived from non-cancer samples from the relevant tissue of origin.

ConclusionWe show how genes in risk loci identified by cancer GWAS can be linked to known cancer driver genes through tissue-specific co-expression networks. This provides an important explanation for why seemingly unrelated sets of genes that harbour either germline risk factors or somatic mutations can eventually cause the same type of disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.23294773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.23294773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.23294773">
        <p class="paperTitle">Relationship between sex biases in gene expression and sex biases in autism and Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.23294773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.23294773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.</p>
        <p class="info">Score: 14.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.23294773' target='https://doi.org/10.1101/2023.08.29.23294773'> 10.1101/2023.08.29.23294773</a></p>
        <p class="abstract">Sex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimers disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=137 SRC=&#34;FIGDIR/small/23294773v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@7c198eorg.highwire.dtl.DTLVardef@171ea18org.highwire.dtl.DTLVardef@1bfe0dcorg.highwire.dtl.DTLVardef@43b98c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.23295182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.23295182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.23295182">
        <p class="paperTitle">Circulating anti-Müllerian hormone levels in pre-menopausal women: novel genetic insights from a GWAS meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.23295182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.23295182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pujol Gualdo, N.; Karjalainen, M. K.; Vosa, U.; Arffman, R. K.; Mägi, R.; Ronkainen, J.; Laisk, T.; Piltonen, T. T.</p>
        <p class="info">Score: 5.0, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.23295182' target='https://doi.org/10.1101/2023.09.07.23295182'> 10.1101/2023.09.07.23295182</a></p>
        <p class="abstract">Study questionCan a genome-wide association study (GWAS) meta-analysis, including a large sample of young premenopausal women from a founder population from Northern Finland, identify novel genetic variants for circulating anti-Mullerian hormone (AMH) levels and provide insights into biological pathways and tissues involved in AMH regulation?

Summary answerWe identified six loci associated with AMH levels at P &lt; 5 x 10-8, including the previously reported MCM8, AMH and TEX41 loci, and three novel signals in or near CHEK2, BMP4 and EIF4EBP1. Gene set enrichment analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranks the pituitary gland as a top associated tissue.

What is known alreadyAMH is expressed by preantral and small antral stage ovarian follicles in women, and variation in age-specific circulating AMH levels has been associated with several health conditions. However, the biological mechanisms underlying the association between health conditions and AMH levels are not yet fully understood. Previous GWAS have identified loci associated with AMH levels in pre-menopausal women, but they were limited by small sample sizes or focused mostly on older pre-menopausal women.

Study design, size, durationWe performed a GWAS meta-analysis for AMH level measurements in 9,668 pre-menopausal women.

Participants/materials, setting, methodsWe performed a GWAS meta-analysis in which we combined 2,619 AMH measurements (at age 31 years old) from a prospective founder population cohort (Northern Finland Birth Cohort 1966, NFBC1966) with a previous GWAS meta-analysis that included 7,049 pre-menopausal women (spanning age range 15-48). NFBC1966 AMH measurements were quantified using an automated assay (Elecsys(R) AMH Plus (Roche)). We annotated the genetic variants, combined different data layers to prioritise potential candidate genes, described significant pathways and tissues enriched by the GWAS signals, identified plausible regulatory roles using colocalization analysis and leveraged publicly available summary statistics to assess genetic and phenotypic correlations with multiple traits.

Main results and the role of chanceThree novel genome-wide significant loci were identified. One of these is in complete linkage disequilibrium with c.1100delC in CHEK2, which is found to be 4-fold enriched in the Finnish population compared to other European populations. We propose a plausible regulatory effect of some of the GWAS variants linked to AMH, as they colocalise with GWAS signals associated with gene expression levels of BMP4, TEX41 and EIFBP41. Gene set analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranked the pituitary gland as the top association.

Large scale dataThe GWAS meta-analysis summary statistics will be available for download from the GWAS Catalog. Accession numbers will be provided upon publication.

Limitations, reasons for cautionThis study only included women of European ancestry and the unavailability of sufficiently sized relevant tissue data in gene expression datasets hinders the assessment of potential regulatory effects in reproductive tissues.

Wider implications of the findingsOur results highlight the increased power of founder populations and larger sample sizes to boost the discovery of novel trait-associated variants underlying variation in AMH levels, which aided to characterise novel biological pathways and plausible genetic regulatory effects linked with AMH levels variation for the first time.

Study funding / competing interest(s)This work has received funding from the European Unions Horizon 2020 research and innovation programme under the MATER Marie Sklodowska-Curie grant agreement No. 813707 and Oulu university scholarship foundation (N.P.-G.), Academy of Finland, Sigrid Juselius Foundation, Novo Nordisk, University of Oulu, Roche Diagnostics (T.T.P). This work was supported by the Estonian Research Council grant 1911 (R.M.). J.R. was supported by the European Unions Horizon 2020 research and innovation program under grant agreements No. 874739 (LongITools), 824989 (EUCAN- Connect), 848158 (EarlyCause) and 733206 (LifeCycle). U.V. was supported by the Estonian Research Council grant PRG (PRG1291). The NFBC1966 received financial support from University of Oulu Grant no. 24000692, Oulu University Hospital Grant no. 24301140, ERDF European Regional Development Fund Grant no. 539/2010 A31592.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.23295093">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.23295093" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.23295093">
        <p class="paperTitle">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.23295093" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.23295093" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.</p>
        <p class="info">Score: 11.8, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.23295093' target='https://doi.org/10.1101/2023.09.05.23295093'> 10.1101/2023.09.05.23295093</a></p>
        <p class="abstract">Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in Communities Study (ARIC) participants over a median follow-up of 21 years. During this period, 735 participants developed incident CH. We found that age at baseline, sex, and dyslipidemia significantly influence the incidence of CH, while ASCVD and other traditional risk factors for ASCVD did not exhibit such associations. Our study also revealed associations between germline genetic variants and incident CH, prioritizing genes in CH development. Our comprehensive longitudinal assessment yields novel insights into the factors contributing to incident CH in older adults.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.23294848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.23294848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.23294848">
        <p class="paperTitle">The pivotal role of the X-chromosome in the genetic architecture of the human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.23294848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.23294848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Z.; Sullivan, P. F.; Li, T.; Zhao, B.; Wang, X.; Luo, T.; Huang, S.; Guan, P. Y.; Chen, J.; Yang, Y.; Stein, J. L.; Li, Y.; Liu, D.; Sun, L.; Zhu, H.</p>
        <p class="info">Score: 8.4, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.23294848' target='https://doi.org/10.1101/2023.08.30.23294848'> 10.1101/2023.08.30.23294848</a></p>
        <p class="abstract">Genes on the X-chromosome are extensively expressed in the human brain, resulting in substantial influences on brain development, intellectual disability, and other brain-related disorders. To comprehensively investigate the X-chromosomes impact on the cerebral cortex, white matter tract microstructures, and intrinsic and extrinsic brain functions, we examined 2,822 complex brain imaging traits obtained from n = 34,000 subjects in the UK Biobank. We unveiled potential autosome-X-chromosome interaction, while proposing an atlas of dosage compensation (DC) for each set of traits. We observed a pronounced X-chromosome impact on the corticospinal tract and the functional amplitude and connectivity of visual networks. In association studies, we identified 50 genome-wide significant trait-locus pairs enriched in Xq28, 22 of which replicated in independent datasets (n = 4,900). Notably, 13 newly identified pairs were in the X-chromosomes non-pseudo-autosomal regions (NPR). The volume of the right ventral diencephalon shared genetic architecture with schizophrenia and educational attainment in a locus indexed by rs2361468 (located [~]3kb upstream of PJA1, a conserved and ubiquitously expressed gene implicated in multiple psychiatric disorders). No significant associations were identified in the pseudo-autosomal regions (PAR) or the Y-chromosome. Finally, we explored sex-specific associations on the X-chromosome and compared differing genetic effects between sexes. We found much more associations can be identified in males (33 versus 9) given a similar sample size. In conclusion, our research provides invaluable insights into the X-chromosomes role in the human brain, contributing to the observed sex differences in brain structure and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.23295505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.23295505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.23295505">
        <p class="paperTitle">Structural Variation Detection and Association Analysis of Whole-Genome-Sequence Data from 16,905 Alzheimer&#39;s Diseases Sequencing Project Subjects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.23295505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.23295505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Dombroski, B. A.; Cheng, P.-L.; Tucci, A.; Si, Y.-q.; Farrell, J. J.; Tzeng, J.-Y.; Leung, Y. Y.; Malamon, J. S.; Wang, L.-S.; Vardarajan, B. N.; Farrer, L. A.; Schellenberg, G. D.; Lee, W.-P.</p>
        <p class="info">Score: 1.2, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.23295505' target='https://doi.org/10.1101/2023.09.13.23295505'> 10.1101/2023.09.13.23295505</a></p>
        <p class="abstract">Structural variations (SVs) are important contributors to the genetics of numerous human diseases. However, their role in Alzheimers disease (AD) remains largely unstudied due to challenges in accurately detecting SVs. Here, we analyzed whole-genome sequencing data from the Alzheimers Disease Sequencing Project (ADSP, N=16,905 subjects) and identified 400,234 (168,223 high-quality) SVs. We found a significant burden of deletions and duplications in AD cases (OR=1.05, P=0.03), particularly for singletons (OR=1.12, P=0.0002) and homozygous events (OR=1.10, P&lt;0.0004). On AD genes, the ultra-rare SVs, including protein-altering SVs in ABCA7, APP, PLCG2, and SORL1, were associated with AD (SKAT-O P=0.004). Twenty-one SVs are in linkage disequilibrium (LD) with known AD-risk variants, e.g., a deletion (chr2:105731359-105736864) in complete LD (R2=0.99) with rs143080277 (chr2:105749599) in NCK2. We also identified 16 SVs associated with AD and 13 SVs associated with AD-related pathological/cognitive endophenotypes. Our findings demonstrate the broad impact of SVs on AD genetics.

Search TermsAlzheimers disease, Structural variation, Copy number variation</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.23294951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.23294951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.23294951">
        <p class="paperTitle">Prioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.23294951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.23294951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pena-Martinez, E. G.; Pomales-Matos, D. A.; Rivera-Madera, A.; Messon-Bird, J. L.; Medina-Feliciano, J. G.; Sanabria-Alberto, L.; Barreiro-Rosario, A. C.; Rodriguez-Rios, J. M.; Rodriguez-Martinez, J. A.</p>
        <p class="info">Score: 3.0, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.23294951' target='https://doi.org/10.1101/2023.09.01.23294951'> 10.1101/2023.09.01.23294951</a></p>
        <p class="abstract">Cardiovascular diseases (CVDs) are the leading cause of death worldwide and are heavily influenced by genetic factors. Genome-wide association studies (GWAS) have mapped &gt; 90% of CVD-associated variants within the non-coding genome, which can alter the function of regulatory proteins, like transcription factors (TFs). However, due to the overwhelming number of GWAS single nucleotide polymorphisms (SNPs) (&gt;500,000), prioritizing variants for in vitro analysis remains challenging. In this work, we implemented a computational approach that considers support vector machine (SVM)-based TF binding site classification and cardiac expression quantitative trait loci (eQTL) analysis to identify and prioritize potential CVD-causing SNPs. We identified 1,535 CVD-associated SNPs that occur within human heart footprints/enhancers and 9,309 variants in linkage disequilibrium (LD) with differential gene expression profiles in cardiac tissue. Using hiPSC-CM ChIP-seq data from NKX2-5 and TBX5, two cardiac TFs essential for proper heart development, we trained a large-scale gapped k-mer SVM

(LS-GKM-SVM) predictive model that can identify binding sites altered by CVD-associated SNPs. The computational predictive model was tested by scoring human heart footprints and enhancers in vitro through electrophoretic mobility shift assay (EMSA). Three variants (rs59310144, rs6715570, and rs61872084) were prioritized for in vitro validation based on their eQTL in cardiac tissue and LS-GKM-SVM prediction to alter NKX2-5 DNA binding. All three variants altered NKX2-5 DNA binding. In summary, we present a bioinformatic approach that considers tissue-specific eQTL analysis and SVM-based TF binding site classification to prioritize CVD-associated variants for in vitro experimental analysis.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=55 SRC=&#34;FIGDIR/small/23294951v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (18K):
org.highwire.dtl.DTLVardef@3c03dforg.highwire.dtl.DTLVardef@224878org.highwire.dtl.DTLVardef@c3df2dorg.highwire.dtl.DTLVardef@17108e0_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
