<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Applying Machine Learning on UK Biobank biomarker data empowers case-control discovery yield
Authors: Garg, M.; Karpinski, M.; Matelska, D.; Middleton, L.; Mitchell, J.; O&#39;Neill, A.; Wang, Q.; Harper, A. R.; Dhindsa, R. S.; Petrovski, S.; Vitsios, D.
Score: 9.5, Published: 2023-10-10 DOI: 10.1101/2023.10.10.23296832
Missing or inaccurate diagnoses in biobank datasets can reduce the power of human genetic association studies. We present a machine-learning framework (MILTON) that utilizes the wealth of phenotypic information available in a biobank dataset to identify undiagnosed individuals within the cohort who have biomarker profiles similar to those of positively diagnosed cases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Applying Machine Learning on UK Biobank biomarker data empowers case-control discovery yield
Authors: Garg, M.; Karpinski, M.; Matelska, D.; Middleton, L.; Mitchell, J.; O&#39;Neill, A.; Wang, Q.; Harper, A. R.; Dhindsa, R. S.; Petrovski, S.; Vitsios, D.
Score: 9.5, Published: 2023-10-10 DOI: 10.1101/2023.10.10.23296832
Missing or inaccurate diagnoses in biobank datasets can reduce the power of human genetic association studies. We present a machine-learning framework (MILTON) that utilizes the wealth of phenotypic information available in a biobank dataset to identify undiagnosed individuals within the cohort who have biomarker profiles similar to those of positively diagnosed cases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Applying Machine Learning on UK Biobank biomarker data empowers case-control discovery yield
Authors: Garg, M.; Karpinski, M.; Matelska, D.; Middleton, L.; Mitchell, J.; O&#39;Neill, A.; Wang, Q.; Harper, A. R.; Dhindsa, R. S.; Petrovski, S.; Vitsios, D.
Score: 9.5, Published: 2023-10-10 DOI: 10.1101/2023.10.10.23296832
Missing or inaccurate diagnoses in biobank datasets can reduce the power of human genetic association studies. We present a machine-learning framework (MILTON) that utilizes the wealth of phenotypic information available in a biobank dataset to identify undiagnosed individuals within the cohort who have biomarker profiles similar to those of positively diagnosed cases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Applying Machine Learning on UK Biobank biomarker data empowers case-control discovery yield\nAuthors: Garg, M.; Karpinski, M.; Matelska, D.; Middleton, L.; Mitchell, J.; O\u0026#39;Neill, A.; Wang, Q.; Harper, A. R.; Dhindsa, R. S.; Petrovski, S.; Vitsios, D.\nScore: 9.5, Published: 2023-10-10 DOI: 10.1101/2023.10.10.23296832\nMissing or inaccurate diagnoses in biobank datasets can reduce the power of human genetic association studies. We present a machine-learning framework (MILTON) that utilizes the wealth of phenotypic information available in a biobank dataset to identify undiagnosed individuals within the cohort who have biomarker profiles similar to those of positively diagnosed cases.",
  "keywords": [
    
  ],
  "articleBody": " Applying Machine Learning on UK Biobank biomarker data empowers case-control discovery yield\nAuthors: Garg, M.; Karpinski, M.; Matelska, D.; Middleton, L.; Mitchell, J.; O'Neill, A.; Wang, Q.; Harper, A. R.; Dhindsa, R. S.; Petrovski, S.; Vitsios, D.\nScore: 9.5, Published: 2023-10-10 DOI: 10.1101/2023.10.10.23296832\nMissing or inaccurate diagnoses in biobank datasets can reduce the power of human genetic association studies. We present a machine-learning framework (MILTON) that utilizes the wealth of phenotypic information available in a biobank dataset to identify undiagnosed individuals within the cohort who have biomarker profiles similar to those of positively diagnosed cases. We applied MILTON to perform an augmented phenome-wide association study (PheWAS) based on 405,703 whole exome sequencing samples from UK Biobank, resulting in improved signals for known (p\u003c1x10-8) gene-disease relationships alongside 206 novel gene-disease relationships that only achieved genome-wide significance upon using MILTON. To further validate these putatively novel discoveries, we adopt two orthogonal machine learning methods that prioritise gene-disease relationships using comprehensive publicly available datasets alongside a biological insights knowledge graph. For additional clinical translation utility, MILTON outputs a disease-specific biomarker set per disease as well as comorbidity clusters across ICD10 disease codes based on shared biomarker profiles of positively labelled cases. All the extracted associations and biomarker importance results for the 3,308 studied binary traits will be made available via an interactive web-portal.\nDiagnostic Utility of Genome-wide DNA Methylation Analysis in Genetically Unsolved Developmental and Epileptic Encephalopathies and Refinement of a CHD2 Episignature\nAuthors: LaFlamme, C. W.; Rastin, C.; Sengupta, S.; Pennington, H. E.; Russ-Hall, S. J.; Schneider, A. L.; Bonkowski, E. S.; Almanza Fuerte, E. P.; Galey, M.; Goffena, J.; Gibson, S. B.; Allan, T. J.; Nyaga, D. M.; Lieffering, N.; Hebbar, M.; Walker, E. V.; Darnell, D.; Olsen, S. R.; Kolekar, P.; Djekidel, M. N.; Rosikiewicz, W.; McConkey, H.; Kerkhof, J.; Levy, M. A.; Relator, R.; Lev, D.; Lerman-Sagie, T.; Park, K. L.; Alders, M.; Cappuccio, G.; Chatron, N.; Demain, L.; Genevieve, D.; Lesca, G.; Roscioli, T.; Sanlaville, D.; Tedder, M. L.; Weisz-Hubshman, M.; Ketkar, S.; Dai, H.; Carlyle Worley, K.; Score: 7.8, Published: 2023-10-13 DOI: 10.1101/2023.10.11.23296741\nSequence-based genetic testing currently identifies causative genetic variants in ~50% of individuals with developmental and epileptic encephalopathies (DEEs). Aberrant changes in DNA methylation are implicated in various neurodevelopmental disorders but remain unstudied in DEEs. Rare epigenetic variations (epivariants) can drive disease by modulating gene expression at single loci, whereas genome-wide DNA methylation changes can result in distinct episignature biomarkers for monogenic disorders in a growing number of rare diseases. Here, we interrogate the diagnostic utility of genome-wide DNA methylation array analysis on peripheral blood samples from 516 individuals with genetically unsolved DEEs who had previously undergone extensive genetic testing. We identified rare differentially methylated regions (DMRs) and explanatory episignatures to discover causative and candidate genetic etiologies in 10 individuals. We then used long-read sequencing to identify DNA variants underlying rare DMRs, including one balanced translocation, three CG-rich repeat expansions, and two copy number variants. We also identify pathogenic sequence variants associated with episignatures; some had been missed by previous exome sequencing. Although most DEE genes lack known episignatures, the increase in diagnostic yield for DNA methylation analysis in DEEs is comparable to the added yield of genome sequencing. Finally, we refine an episignature for CHD2 using an 850K methylation array which was further refined at higher CpG resolution using bisulfite sequencing to investigate potential insights into CHD2 pathophysiology. Our study demonstrates the diagnostic yield of genome-wide DNA methylation analysis to identify causal and candidate genetic causes as ~2% (10/516) for unsolved DEE cases.\nPKHD1L1, A Gene Involved in the Stereociliary Coat, Causes Autosomal Recessive Nonsyndromic Hearing Loss\nAuthors: Redfield, S. E.; De-la-Torre, P.; Zamani, M.; Khan, H.; Morris, T.; Shariati, G.; Karimi, M.; Kenna, M. A.; Seo, G. H.; Naz, S.; Galehdari, H.; Indzhykulian, A. A.; Shearer, A. E.; Vona, B.\nScore: 6.6, Published: 2023-10-09 DOI: 10.1101/2023.10.08.23296081\nIdentification of genes associated with nonsyndromic hearing loss is a crucial endeavor, given the substantial number of individuals who remain without a diagnosis after even the most advanced genetic testing. PKHD1L1 was established as necessary for the formation of the cochlear hair-cell stereociliary coat and causes hearing loss in mice and zebrafish when mutated. We sought to determine if biallelic variants in PKHD1L1 also cause hearing loss in humans. Exome sequencing was performed on DNA of three families segregating autosomal recessive moderate to severe nonsyndromic sensorineural hearing loss. Compound heterozygous missense p.[(Gly129Ser)];p.[(Gly1314Val)], homozygous missense p.(His2479Gln) and nonsense p.(Arg3381Ter) variants were identified in PKHD1L1 that were predicted to be damaging using in silico pathogenicity prediction methods. In vitro functional analysis of two missense variants was performed using purified recombinant PKHD1L1 protein fragments. We then evaluated protein thermodynamic stability with and without the missense variants found in one of the families. In vitro functional assessment indicated that both engineered PKHD1L1 mutant p.(Gly129Ser) and p.(Gly1314Val) constructs significantly reduced the folding and structural stabilities of the expressed protein fragments, providing further evidence to support pathogenicity of these variants. In silico molecular modelling using AlphaFold2 and protein sequence alignment analysis were carried out to further explore potential variant effects on protein folding and stability and exposed key structural features that might suggest PKHD1L1 protein destabilization. Multiple lines of evidence collectively associate PKHD1L1 with nonsyndromic mild-moderate to severe sensorineural hearing loss. PKHD1L1 testing in individuals with mild-moderate hearing loss may identify further affected families.\nAn AluYb8 retrotransposon characterises a risk haplotype of TMEM106B associated in neurodegeneration\nAuthors: Salazar, A.; Tesi, N.; Knoop, L.; Pijnenburg, Y.; van der Lee, S.; Wijesekera, S.; Krizova, J.; Hiltunen, M.; Damme, M.; Petrucelli, L.; Reinders, M.; Hulsman, M.; Holstege, H.\nScore: 7.5, Published: 2023-10-03 DOI: 10.1101/2023.07.16.23292721\nGenome-wide association studies identified a role for TMEM106B in various neurodegenerative diseases. Based on long-read whole-genome sequencing data of 256 individuals, we identified an AluYb8 retrotransposon in the 3 UTR of the risk haplotype of TMEM106B. When transcriptionally active, Alu-elements can propogate throughout the genome, and mediate (post-)transcriptional dysregulation of nearby genes. We found that TMEM106B haplotypes carrying the AluYb8 element are more methylated than those without, likely reflecting an evolutionary selection to suppress propagation. AluYb8 activation can be further suppressed by TDP-43, in its role in post-transcriptional RNA-processing. However, age-related loss of TDP-43, by reduced methylation in the 3 UTR of TARDBP, may release AluYb8 suppression. Together, our findings suggest that in the aging brain, the AluYb8 insertion may mediate dysregulation of TMEM106B, impacting the endolysosomal system via a negative-feedback loop, ultimately leading to neurodegenerative disease. Notably, TMEM106B haplotype sequences are different between African and European genomes, which likely explains the different effects on disease-risk between both populations. Overall, our research advances the understanding of the roles of TDP-43 and TMEM106B in neurodegenerative diseases, and provides a novel connection between genetic variation and age-related changes in genomic and cellular regulation.\nLimited overlap between genetic effects on disease susceptibility and disease survival\nAuthors: Yang, Z.; Pajuste, F.-D.; Zguro, K.; Cheng, Y.; Kurant, D. E.; Eoli, A.; Wanner, J.; Jermy, B.; FinnGen, ; Estonian Biobank research team, ; Kanoni, S.; Heel, D. A. v.; Genes \u0026 Health Research Team, ; Hayward, C.; Marioni, R. E.; McCartney, D. L.; Renieri, A.; Furini, S.; Genomics England Research Consortium, ; Magi, R.; Gusev, A.; Drineas, P.; Paschou, P.; Heyne, H.; Ripatti, S.; Mars, N.; Ganna, A.\nScore: 3.6, Published: 2023-10-12 DOI: 10.1101/2023.10.10.23296544\nUnderstanding disease progression is of a high biological and clinical interest. Unlike disease susceptibility whose genetic basis has been abundantly studied, less is known about the genetics of disease progression and its overlap with disease susceptibility. Considering ten common diseases (N cases ranging from 17,152 to 99,666) across seven biobanks, we systematically compared the genetic architecture of susceptibility and progression, defined as disease-specific mortality. We identified only one locus significantly associated with disease-specific mortality and show that, at a similar sample size, more genome-wide significant loci can be identified in a GWAS of disease susceptibility. Variants that were significantly affecting disease susceptibility were weakly or not associated with disease-specific mortality. Moreover, susceptibility polygenic scores (PGSs) were weak predictor of disease-specific mortality while a PGS for general lifespan was significantly associated with disease-specific mortality for five out of ten diseases. We used theoretical derivation and simulation to propose plausible explanations for our empirical observations and account for potential index-event bias. Overall, our findings point to little similarity in genetic effects between disease susceptibility and disease-specific mortality and suggest that either larger sample sizes or different measures of progression are needed to identify the genetic underpinning of disease progression.\nClinical interpretation of KCNH2 variants using a robust PS3/BS3 functional patch clamp assay\nAuthors: Thomson, K. L.; Jiang, C.; Richardson, E.; Westphal, D. S.; Burkard, T.; Wolf, C. M.; Vatta, M.; Harrison, S. M.; Ingles, J.; Bezzina, C. R.; Kroncke, B. M.; Vandenberg, J. I.; Ng, C.-A.\nScore: 3.6, Published: 2023-10-09 DOI: 10.1101/2023.10.08.23296707\nLong QT syndrome (LQTS), caused by the dysfunction of cardiac ion channels, increases the risk of sudden death in otherwise healthy young people. For many variants in LQTS genes there is insufficient evidence to make a definitive genetic diagnosis. We have established a robust functional patch clamp assay to facilitate classification of missense variants in KCNH2, one of the key LQTS genes. A curated set of 30 benign and 30 pathogenic missense variants were used to establish the range of normal and abnormal function. The extent to which variants reduced protein function was quantified using Z-scores; the number of standard deviations from the mean of the normalised current density of the set of benign variant controls. A Z-score of -2 defined the threshold for abnormal loss-of-function, which corresponds to 55% wild-type function. More extreme Z-scores were observed for variants with a greater loss-of-function effect. We propose that the Z-score for each variant can be used to inform the application and weighting of abnormal and normal functional evidence criteria (PS3 and BS3) within the American College of Medical Genetics and Genomics (ACMG) variant classification framework. The validity of this approach was demonstrated using a series of 18 KCNH2 missense variants detected in a childhood onset LQTS cohort, where the level of function assessed using our assay correlated to the Schwartz score (a scoring system used to quantify the probability of a clinical diagnosis of LQTS1) and the length of the QTc interval.\nDivergence Between Clinician and Patient Perspectives on Polygenic Embryo Screening: A Qualitative Study\nAuthors: Barlevy, D.; Cenolli, I.; Campbell, T.; Furrer, R.; Mukherjee, M.; Kostick-Quenet, K. M.; Carmi, S.; Lencz, T.; Lazaro-Munoz, G.; Pereira, S.\nScore: 3.2, Published: 2023-10-13 DOI: 10.1101/2023.10.12.23296961\nObjective: To explore and compare the perspectives of clinicians and patients on polygenic embryo screening. Design: Qualitative. Subjects: Fifty-three participants: 27 reproductive endocrinology and infertility specialists and 26 patients currently undergoing in vitro fertilization or had done so within the last five years. Main Outcome Measures: Qualitative thematic analysis of interview transcripts. Results: Both clinicians and patients often held favorable views of screening embryos for physical or psychiatric conditions, though clinicians tended to temper their positive attitudes with specific caveats. Clinicians also expressed negative views about screening embryos for traits more often than patients, who generally held more positive views. Most clinicians were either unwilling to discuss or offer polygenic embryo screening to patients or were willing to do so only under certain circumstances, while many patients expressed interest in polygenic embryo screening. Both sets of stakeholders envisioned multiple potential benefits or uses of polygenic embryo screening; the most common included selection and/or prioritization of embryos, receipt of more information about embryos, and preparation for the birth of a predisposed or affected child. Both sets of stakeholders also raised multiple potential, interrelated concerns about polygenic embryo screening. The most common concerns among both sets of stakeholders included the potential for different types of biases, most often in relation to selection of embryos with preferred genetic chances of traits, the probabilistic nature of polygenic embryo screening that can complicate patient counseling and/or lead to excessive cycles of in vitro fertilization, and a lack of data from long-term prospective studies supporting the clinical use of polygenic embryo screening. Conclusion: Despite patient interest in polygenic embryo screening, clinicians feel such screening is premature for clinical application. Though now embryos can be screened for their genetic chances of developing polygenic conditions and traits, many clinicians and patients maintain different attitudes depending on what is specifically screened, despite the blurry distinction between conditions and traits. Considerations raised by these stakeholders may help guide professional societies as they consider developing guidelines to navigate the uncertain terrain of polygenic embryo screening, which is already commercially available.\nDistinct explanations underlie gene-environment interactions in the UK Biobank\nAuthors: Durvasula, A.; Price, A.\nScore: 26.3, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969\nThe role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) \u003c 1 across E bins. Second, we detect genome-wide effects of the E variable on genetic variance by leveraging polygenic risk scores (PRS) to test for significant PRSxE in a regression of phenotypes on PRS, E, and PRSxE, together with differences in SNP-heritability across E bins. Third, we detect genome-wide proportional amplification of genetic and environmental effects as a function of the E variable by testing for significant PRSxE with no differences in SNP-heritability across E bins. Simulations show that these approaches achieve high sensitivity and specificity in distinguishing these three GxE scenarios. We applied our framework to 33 UK Biobank diseases/traits (average N=325K) and 10 E variables spanning lifestyle, diet, and other environmental exposures. First, we identified 19 trait-E pairs with rg significantly \u003c 1 (FDR\u003c5%) (average rg=0.95); for example, white blood cell count had rg=0.95 (s.e. 0.01) between smokers and non-smokers. Second, we identified 28 trait-E pairs with significant PRSxE and significant SNP-heritability differences across E bins; for example, type 2 diabetes had a significant PRSxE for alcohol consumption (P=1e-13) with 4.2x larger SNP-heritability in the largest versus smallest quintiles of alcohol consumption (P\u003c1e-16). Third, we identified 15 trait-E pairs with significant PRSxE with no SNP-heritability differences across E bins; for example, triglyceride levels had a significant PRSxE effect for composite diet score (P=4e-5) with no SNP-heritability differences. Analyses using biological sex as the E variable produced additional significant findings in each of the three scenarios. Overall, we infer a substantial contribution of GxE and GxSex effects to disease and complex trait variance.\nMendelian randomization revealing the protective effect of sodium-glucose cotransporter 2 inhibition on prostate cancer with verified evidence from electronic healthcare and biological data\nAuthors: Zheng, J.; Lu, J.; Qi, J.; Yang, Q.; Zhao, H.; Ye, Y.; Xu, M.; Xu, Y.; Wang, T.; Li, M.; Zhao, Z.; Zheng, R.; Wang, S.; Lin, H.; Hu, C.; Sze Ling Chui, C.; Au Yeung, S. L.; Luo, S.; Dimopoulou, O.; Dixon, P.; Harrison, S.; Liu, Y.; Robinson, J.; Yarmolinsky, J.; Haycock, P.; Yuan, J.; Lewis, S.; Gaunt, T. R.; Davey Smith, G.; Ning, G.; Martin, R. M.; Cui, B.; Wang, W.; Bi, Y.\nScore: 1.6, Published: 2023-10-13 DOI: 10.1101/2023.10.10.23296790\nBackground Observational studies indicated a decreased risk of prostate cancer by SGLT2 inhibitors, but high-quality evidence is lacking to make a clear conclusion. We evaluated the effect of SGLT2 inhibition on prostate cancer risk by triangulating evidence from three methods. Methods Genetic variants associated with HbA1c levels (P\u003c5e-8) in the genomic region of the target gene, SLC5A2, were used to proxy SGLT2 inhibition. In discovery, Mendelian randomization (MR) was applied to estimate effects of genetically proxied SGLT2 inhibition on risk of prostate cancer and its subtypes (79,148 cases and 61,106 controls). In a validation using electronic healthcare data, the association of incidence of prostate cancer between 24,155 new users of SGLT2 inhibitors and 24,155 new users of the active comparator, dipeptidyl peptidase 4 inhibitors, was estimated using electronic health-care data. In a biological validation, the differential gene expression of SLC5A2 between normal prostate tissue and tumour tissue were estimated in 691 prostate cancer patients. To validate the influence of glucose, the association between HbA1c levels and incident prostate cancer during 10-years of follow-up were estimated. Findings For genetic evidence, genetically proxied SGLT2 inhibition reduced the risk of overall (odds ratio=0.56, 95%CI=0.38 to 0.82), advanced (OR=0.52, 95%CI=0.27 to 0.99) and early-onset (OR=0.27, 95%CI=0.11 to 0.72) prostate cancer. For electronic healthcare evidence, usage of SGLT2 inhibitor was associated with a 23% reduced risk of prostate cancer (hazard ratio=0.77, 95%CI=0.61 to 0.99) in males with diabetes. For biological evidence, expression levels of the SLC5A2 gene in tumour prostate tissue was 2.02-fold higher than that in normal tissue (P=0.006). Genetically proxied HbA1c and observed HbA1c provided little evidence to support an association with total/incident prostate cancer, implies a non-glycemic effect of SGLT2 inhibition on prostate cancer. Interpretation This study provides genetic, electronic healthcare, and biological evidence to support a beneficial effect of SGLT2 inhibition on reducing prostate cancer risk. Future trials are warranted to investigate whether SLGT2 inhibitors can been recommended for diabetic individuals with high risk of prostate cancer or considered as an anti-prostate cancer therapy.\nInvestigation of BRCA1 exon 11 genetic variations in breast cancer among Libyan women\nAuthors: Elshwekh, H.; Alhudiri, I.; Abdul Jalil, H. M.; Mohamed, A. A.; Elkikli, A.; Jornaz, N.; Enattah, N.\nScore: 1.2, Published: 2023-10-14 DOI: 10.1101/2023.10.13.23296599\nAbstract Introduction The majority of hereditary breast and ovarian cancers are associated with mutations in two genes, breast cancer type 1 and 2 susceptibility genes (BRCA1 and BRCA2). Here, we describe for the first time analysis of BRCA1 exon 11 in 48 Libyan breast cancer patients with a family history of cancer. Methods All patients had a family history of cancer and were included in the study only if they have a family history of breast or ovarian cancer, male breast cancer, or triple negative tumors. PCR was performed using specific primer pairs spanning BRCA1 exon 11 followed by Sanger sequencing. Genetic analysis was done using Sequencher 5.1. Results We identified 12 genetic variants in BRCA1 exon 11. Three variants were novel (c.1019T\u003eC, c.2363T\u003eG, and c.3192T\u003eC). c.2363T\u003eG was predicted by SIFT as damaging. Six variants were of unknown significance (c.918T\u003eC, c.1853G\u003eC, c.1886G\u003eA, c.2215A\u003eG, c.2612C\u003eT, c.3113A\u003eC and c.3784T\u003eC), and 3 were classified as benign in ClinVar database (c.918T\u003eC, c.2082C\u003eT and c.2311T\u003eC). Conclusions Scanning of the entire BRCA1 is needed to identify any associated deleterious mutations. Although the clinical importance of unclassified variants is unknown, the association of certain variants with deleterious mutations and contralateral breast cancer warrants genetic testing and counseling in the Libyan population.\n",
  "wordCount" : "3094",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.23296832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.23296832" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.23296832">
        <p class="paperTitle">Applying Machine Learning on UK Biobank biomarker data empowers case-control discovery yield</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.23296832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.23296832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garg, M.; Karpinski, M.; Matelska, D.; Middleton, L.; Mitchell, J.; O&#39;Neill, A.; Wang, Q.; Harper, A. R.; Dhindsa, R. S.; Petrovski, S.; Vitsios, D.</p>
        <p class="info">Score: 9.5, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.23296832' target='https://doi.org/10.1101/2023.10.10.23296832'> 10.1101/2023.10.10.23296832</a></p>
        <p class="abstract">Missing or inaccurate diagnoses in biobank datasets can reduce the power of human genetic association studies. We present a machine-learning framework (MILTON) that utilizes the wealth of phenotypic information available in a biobank dataset to identify undiagnosed individuals within the cohort who have biomarker profiles similar to those of positively diagnosed cases. We applied MILTON to perform an augmented phenome-wide association study (PheWAS) based on 405,703 whole exome sequencing samples from UK Biobank, resulting in improved signals for known (p&lt;1x10-8) gene-disease relationships alongside 206 novel gene-disease relationships that only achieved genome-wide significance upon using MILTON. To further validate these putatively novel discoveries, we adopt two orthogonal machine learning methods that prioritise gene-disease relationships using comprehensive publicly available datasets alongside a biological insights knowledge graph. For additional clinical translation utility, MILTON outputs a disease-specific biomarker set per disease as well as comorbidity clusters across ICD10 disease codes based on shared biomarker profiles of positively labelled cases. All the extracted associations and biomarker importance results for the 3,308 studied binary traits will be made available via an interactive web-portal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.23296741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.23296741" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.23296741">
        <p class="paperTitle">Diagnostic Utility of Genome-wide DNA Methylation Analysis in Genetically Unsolved Developmental and Epileptic Encephalopathies and Refinement of a CHD2 Episignature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.23296741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.23296741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LaFlamme, C. W.; Rastin, C.; Sengupta, S.; Pennington, H. E.; Russ-Hall, S. J.; Schneider, A. L.; Bonkowski, E. S.; Almanza Fuerte, E. P.; Galey, M.; Goffena, J.; Gibson, S. B.; Allan, T. J.; Nyaga, D. M.; Lieffering, N.; Hebbar, M.; Walker, E. V.; Darnell, D.; Olsen, S. R.; Kolekar, P.; Djekidel, M. N.; Rosikiewicz, W.; McConkey, H.; Kerkhof, J.; Levy, M. A.; Relator, R.; Lev, D.; Lerman-Sagie, T.; Park, K. L.; Alders, M.; Cappuccio, G.; Chatron, N.; Demain, L.; Genevieve, D.; Lesca, G.; Roscioli, T.; Sanlaville, D.; Tedder, M. L.; Weisz-Hubshman, M.; Ketkar, S.; Dai, H.; Carlyle Worley, K.; </p>
        <p class="info">Score: 7.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.23296741' target='https://doi.org/10.1101/2023.10.11.23296741'> 10.1101/2023.10.11.23296741</a></p>
        <p class="abstract">Sequence-based genetic testing currently identifies causative genetic variants in ~50% of individuals with developmental and epileptic encephalopathies (DEEs). Aberrant changes in DNA methylation are implicated in various neurodevelopmental disorders but remain unstudied in DEEs. Rare epigenetic variations (epivariants) can drive disease by modulating gene expression at single loci, whereas genome-wide DNA methylation changes can result in distinct episignature biomarkers for monogenic disorders in a growing number of rare diseases. Here, we interrogate the diagnostic utility of genome-wide DNA methylation array analysis on peripheral blood samples from 516 individuals with genetically unsolved DEEs who had previously undergone extensive genetic testing. We identified rare differentially methylated regions (DMRs) and explanatory episignatures to discover causative and candidate genetic etiologies in 10 individuals. We then used long-read sequencing to identify DNA variants underlying rare DMRs, including one balanced translocation, three CG-rich repeat expansions, and two copy number variants. We also identify pathogenic sequence variants associated with episignatures; some had been missed by previous exome sequencing. Although most DEE genes lack known episignatures, the increase in diagnostic yield for DNA methylation analysis in DEEs is comparable to the added yield of genome sequencing. Finally, we refine an episignature for CHD2 using an 850K methylation array which was further refined at higher CpG resolution using bisulfite sequencing to investigate potential insights into CHD2 pathophysiology. Our study demonstrates the diagnostic yield of genome-wide DNA methylation analysis to identify causal and candidate genetic causes as ~2% (10/516) for unsolved DEE cases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.23296081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.23296081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.23296081">
        <p class="paperTitle">PKHD1L1, A Gene Involved in the Stereociliary Coat, Causes Autosomal Recessive Nonsyndromic Hearing Loss</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.23296081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.23296081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redfield, S. E.; De-la-Torre, P.; Zamani, M.; Khan, H.; Morris, T.; Shariati, G.; Karimi, M.; Kenna, M. A.; Seo, G. H.; Naz, S.; Galehdari, H.; Indzhykulian, A. A.; Shearer, A. E.; Vona, B.</p>
        <p class="info">Score: 6.6, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.23296081' target='https://doi.org/10.1101/2023.10.08.23296081'> 10.1101/2023.10.08.23296081</a></p>
        <p class="abstract">Identification of genes associated with nonsyndromic hearing loss is a crucial endeavor, given the substantial number of individuals who remain without a diagnosis after even the most advanced genetic testing. PKHD1L1 was established as necessary for the formation of the cochlear hair-cell stereociliary coat and causes hearing loss in mice and zebrafish when mutated. We sought to determine if biallelic variants in PKHD1L1 also cause hearing loss in humans.

Exome sequencing was performed on DNA of three families segregating autosomal recessive moderate to severe nonsyndromic sensorineural hearing loss. Compound heterozygous missense p.[(Gly129Ser)];p.[(Gly1314Val)], homozygous missense p.(His2479Gln) and nonsense p.(Arg3381Ter) variants were identified in PKHD1L1 that were predicted to be damaging using in silico pathogenicity prediction methods. In vitro functional analysis of two missense variants was performed using purified recombinant PKHD1L1 protein fragments. We then evaluated protein thermodynamic stability with and without the missense variants found in one of the families. In vitro functional assessment indicated that both engineered PKHD1L1 mutant p.(Gly129Ser) and p.(Gly1314Val) constructs significantly reduced the folding and structural stabilities of the expressed protein fragments, providing further evidence to support pathogenicity of these variants. In silico molecular modelling using AlphaFold2 and protein sequence alignment analysis were carried out to further explore potential variant effects on protein folding and stability and exposed key structural features that might suggest PKHD1L1 protein destabilization.

Multiple lines of evidence collectively associate PKHD1L1 with nonsyndromic mild-moderate to severe sensorineural hearing loss. PKHD1L1 testing in individuals with mild-moderate hearing loss may identify further affected families.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292721">
        <p class="paperTitle">An AluYb8 retrotransposon characterises a risk haplotype of TMEM106B associated in neurodegeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salazar, A.; Tesi, N.; Knoop, L.; Pijnenburg, Y.; van der Lee, S.; Wijesekera, S.; Krizova, J.; Hiltunen, M.; Damme, M.; Petrucelli, L.; Reinders, M.; Hulsman, M.; Holstege, H.</p>
        <p class="info">Score: 7.5, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292721' target='https://doi.org/10.1101/2023.07.16.23292721'> 10.1101/2023.07.16.23292721</a></p>
        <p class="abstract">Genome-wide association studies identified a role for TMEM106B in various neurodegenerative diseases. Based on long-read whole-genome sequencing data of 256 individuals, we identified an AluYb8 retrotransposon in the 3 UTR of the risk haplotype of TMEM106B. When transcriptionally active, Alu-elements can propogate throughout the genome, and mediate (post-)transcriptional dysregulation of nearby genes. We found that TMEM106B haplotypes carrying the AluYb8 element are more methylated than those without, likely reflecting an evolutionary selection to suppress propagation. AluYb8 activation can be further suppressed by TDP-43, in its role in post-transcriptional RNA-processing. However, age-related loss of TDP-43, by reduced methylation in the 3 UTR of TARDBP, may release AluYb8 suppression. Together, our findings suggest that in the aging brain, the AluYb8 insertion may mediate dysregulation of TMEM106B, impacting the endolysosomal system via a negative-feedback loop, ultimately leading to neurodegenerative disease. Notably, TMEM106B haplotype sequences are different between African and European genomes, which likely explains the different effects on disease-risk between both populations. Overall, our research advances the understanding of the roles of TDP-43 and TMEM106B in neurodegenerative diseases, and provides a novel connection between genetic variation and age-related changes in genomic and cellular regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.23296544">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.23296544" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.23296544">
        <p class="paperTitle">Limited overlap between genetic effects on disease susceptibility and disease survival</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.23296544" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.23296544" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, Z.; Pajuste, F.-D.; Zguro, K.; Cheng, Y.; Kurant, D. E.; Eoli, A.; Wanner, J.; Jermy, B.; FinnGen,  ; Estonian Biobank research team,  ; Kanoni, S.; Heel, D. A. v.; Genes &amp; Health Research Team,  ; Hayward, C.; Marioni, R. E.; McCartney, D. L.; Renieri, A.; Furini, S.; Genomics England Research Consortium,  ; Magi, R.; Gusev, A.; Drineas, P.; Paschou, P.; Heyne, H.; Ripatti, S.; Mars, N.; Ganna, A.</p>
        <p class="info">Score: 3.6, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.23296544' target='https://doi.org/10.1101/2023.10.10.23296544'> 10.1101/2023.10.10.23296544</a></p>
        <p class="abstract">Understanding disease progression is of a high biological and clinical interest. Unlike disease susceptibility whose genetic basis has been abundantly studied, less is known about the genetics of disease progression and its overlap with disease susceptibility. Considering ten common diseases (N cases ranging from 17,152 to 99,666) across seven biobanks, we systematically compared the genetic architecture of susceptibility and progression, defined as disease-specific mortality. We identified only one locus significantly associated with disease-specific mortality and show that, at a similar sample size, more genome-wide significant loci can be identified in a GWAS of disease susceptibility. Variants that were significantly affecting disease susceptibility were weakly or not associated with disease-specific mortality. Moreover, susceptibility polygenic scores (PGSs) were weak predictor of disease-specific mortality while a PGS for general lifespan was significantly associated with disease-specific mortality for five out of ten diseases. We used theoretical derivation and simulation to propose plausible explanations for our empirical observations and account for potential index-event bias. Overall, our findings point to little similarity in genetic effects between disease susceptibility and disease-specific mortality and suggest that either larger sample sizes or different measures of progression are needed to identify the genetic underpinning of disease progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.23296707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.23296707" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.23296707">
        <p class="paperTitle">Clinical interpretation of KCNH2 variants using a robust PS3/BS3 functional patch clamp assay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.23296707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.23296707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomson, K. L.; Jiang, C.; Richardson, E.; Westphal, D. S.; Burkard, T.; Wolf, C. M.; Vatta, M.; Harrison, S. M.; Ingles, J.; Bezzina, C. R.; Kroncke, B. M.; Vandenberg, J. I.; Ng, C.-A.</p>
        <p class="info">Score: 3.6, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.23296707' target='https://doi.org/10.1101/2023.10.08.23296707'> 10.1101/2023.10.08.23296707</a></p>
        <p class="abstract">Long QT syndrome (LQTS), caused by the dysfunction of cardiac ion channels, increases the risk of sudden death in otherwise healthy young people. For many variants in LQTS genes there is insufficient evidence to make a definitive genetic diagnosis. We have established a robust functional patch clamp assay to facilitate classification of missense variants in KCNH2, one of the key LQTS genes. A curated set of 30 benign and 30 pathogenic missense variants were used to establish the range of normal and abnormal function. The extent to which variants reduced protein function was quantified using Z-scores; the number of standard deviations from the mean of the normalised current density of the set of benign variant controls. A Z-score of -2 defined the threshold for abnormal loss-of-function, which corresponds to 55% wild-type function. More extreme Z-scores were observed for variants with a greater loss-of-function effect. We propose that the Z-score for each variant can be used to inform the application and weighting of abnormal and normal functional evidence criteria (PS3 and BS3) within the American College of Medical Genetics and Genomics (ACMG) variant classification framework. The validity of this approach was demonstrated using a series of 18 KCNH2 missense variants detected in a childhood onset LQTS cohort, where the level of function assessed using our assay correlated to the Schwartz score (a scoring system used to quantify the probability of a clinical diagnosis of LQTS1) and the length of the QTc interval.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.23296961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.23296961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.23296961">
        <p class="paperTitle">Divergence Between Clinician and Patient Perspectives on Polygenic Embryo Screening: A Qualitative Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.23296961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.23296961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barlevy, D.; Cenolli, I.; Campbell, T.; Furrer, R.; Mukherjee, M.; Kostick-Quenet, K. M.; Carmi, S.; Lencz, T.; Lazaro-Munoz, G.; Pereira, S.</p>
        <p class="info">Score: 3.2, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.23296961' target='https://doi.org/10.1101/2023.10.12.23296961'> 10.1101/2023.10.12.23296961</a></p>
        <p class="abstract">Objective: To explore and compare the perspectives of clinicians and patients on polygenic embryo screening. Design: Qualitative. Subjects: Fifty-three participants: 27 reproductive endocrinology and infertility specialists and 26 patients currently undergoing in vitro fertilization or had done so within the last five years. Main Outcome Measures: Qualitative thematic analysis of interview transcripts. Results: Both clinicians and patients often held favorable views of screening embryos for physical or psychiatric conditions, though clinicians tended to temper their positive attitudes with specific caveats. Clinicians also expressed negative views about screening embryos for traits more often than patients, who generally held more positive views. Most clinicians were either unwilling to discuss or offer polygenic embryo screening to patients or were willing to do so only under certain circumstances, while many patients expressed interest in polygenic embryo screening. Both sets of stakeholders envisioned multiple potential benefits or uses of polygenic embryo screening; the most common included selection and/or prioritization of embryos, receipt of more information about embryos, and preparation for the birth of a predisposed or affected child. Both sets of stakeholders also raised multiple potential, interrelated concerns about polygenic embryo screening. The most common concerns among both sets of stakeholders included the potential for different types of biases, most often in relation to selection of embryos with preferred genetic chances of traits, the probabilistic nature of polygenic embryo screening that can complicate patient counseling and/or lead to excessive cycles of in vitro fertilization, and a lack of data from long-term prospective studies supporting the clinical use of polygenic embryo screening. Conclusion: Despite patient interest in polygenic embryo screening, clinicians feel such screening is premature for clinical application. Though now embryos can be screened for their genetic chances of developing polygenic conditions and traits, many clinicians and patients maintain different attitudes depending on what is specifically screened, despite the blurry distinction between conditions and traits. Considerations raised by these stakeholders may help guide professional societies as they consider developing guidelines to navigate the uncertain terrain of polygenic embryo screening, which is already commercially available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.23295969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.23295969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.23295969">
        <p class="paperTitle">Distinct explanations underlie gene-environment interactions in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.23295969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.23295969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durvasula, A.; Price, A.</p>
        <p class="info">Score: 26.3, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.23295969' target='https://doi.org/10.1101/2023.09.22.23295969'> 10.1101/2023.09.22.23295969</a></p>
        <p class="abstract">The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins. Second, we detect genome-wide effects of the E variable on genetic variance by leveraging polygenic risk scores (PRS) to test for significant PRSxE in a regression of phenotypes on PRS, E, and PRSxE, together with differences in SNP-heritability across E bins. Third, we detect genome-wide proportional amplification of genetic and environmental effects as a function of the E variable by testing for significant PRSxE with no differences in SNP-heritability across E bins. Simulations show that these approaches achieve high sensitivity and specificity in distinguishing these three GxE scenarios. We applied our framework to 33 UK Biobank diseases/traits (average N=325K) and 10 E variables spanning lifestyle, diet, and other environmental exposures. First, we identified 19 trait-E pairs with rg significantly &lt; 1 (FDR&lt;5%) (average rg=0.95); for example, white blood cell count had rg=0.95 (s.e. 0.01) between smokers and non-smokers. Second, we identified 28 trait-E pairs with significant PRSxE and significant SNP-heritability differences across E bins; for example, type 2 diabetes had a significant PRSxE for alcohol consumption (P=1e-13) with 4.2x larger SNP-heritability in the largest versus smallest quintiles of alcohol consumption (P&lt;1e-16). Third, we identified 15 trait-E pairs with significant PRSxE with no SNP-heritability differences across E bins; for example, triglyceride levels had a significant PRSxE effect for composite diet score (P=4e-5) with no SNP-heritability differences. Analyses using biological sex as the E variable produced additional significant findings in each of the three scenarios. Overall, we infer a substantial contribution of GxE and GxSex effects to disease and complex trait variance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.23296790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.23296790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.23296790">
        <p class="paperTitle">Mendelian randomization revealing the protective effect of sodium-glucose cotransporter 2 inhibition on prostate cancer with verified evidence from electronic healthcare and biological data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.23296790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.23296790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, J.; Lu, J.; Qi, J.; Yang, Q.; Zhao, H.; Ye, Y.; Xu, M.; Xu, Y.; Wang, T.; Li, M.; Zhao, Z.; Zheng, R.; Wang, S.; Lin, H.; Hu, C.; Sze Ling Chui, C.; Au Yeung, S. L.; Luo, S.; Dimopoulou, O.; Dixon, P.; Harrison, S.; Liu, Y.; Robinson, J.; Yarmolinsky, J.; Haycock, P.; Yuan, J.; Lewis, S.; Gaunt, T. R.; Davey Smith, G.; Ning, G.; Martin, R. M.; Cui, B.; Wang, W.; Bi, Y.</p>
        <p class="info">Score: 1.6, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.23296790' target='https://doi.org/10.1101/2023.10.10.23296790'> 10.1101/2023.10.10.23296790</a></p>
        <p class="abstract">Background Observational studies indicated a decreased risk of prostate cancer by SGLT2 inhibitors, but high-quality evidence is lacking to make a clear conclusion. We evaluated the effect of SGLT2 inhibition on prostate cancer risk by triangulating evidence from three methods. Methods Genetic variants associated with HbA1c levels (P&lt;5e-8) in the genomic region of the target gene, SLC5A2, were used to proxy SGLT2 inhibition. In discovery, Mendelian randomization (MR) was applied to estimate effects of genetically proxied SGLT2 inhibition on risk of prostate cancer and its subtypes (79,148 cases and 61,106 controls). In a validation using electronic healthcare data, the association of incidence of prostate cancer between 24,155 new users of SGLT2 inhibitors and 24,155 new users of the active comparator, dipeptidyl peptidase 4 inhibitors, was estimated using electronic health-care data. In a biological validation, the differential gene expression of SLC5A2 between normal prostate tissue and tumour tissue were estimated in 691 prostate cancer patients. To validate the influence of glucose, the association between HbA1c levels and incident prostate cancer during 10-years of follow-up were estimated. Findings For genetic evidence, genetically proxied SGLT2 inhibition reduced the risk of overall (odds ratio=0.56, 95%CI=0.38 to 0.82), advanced (OR=0.52, 95%CI=0.27 to 0.99) and early-onset (OR=0.27, 95%CI=0.11 to 0.72) prostate cancer. For electronic healthcare evidence, usage of SGLT2 inhibitor was associated with a 23% reduced risk of prostate cancer (hazard ratio=0.77, 95%CI=0.61 to 0.99) in males with diabetes. For biological evidence, expression levels of the SLC5A2 gene in tumour prostate tissue was 2.02-fold higher than that in normal tissue (P=0.006). Genetically proxied HbA1c and observed HbA1c provided little evidence to support an association with total/incident prostate cancer, implies a non-glycemic effect of SGLT2 inhibition on prostate cancer. Interpretation This study provides genetic, electronic healthcare, and biological evidence to support a beneficial effect of SGLT2 inhibition on reducing prostate cancer risk. Future trials are warranted to investigate whether SLGT2 inhibitors can been recommended for diabetic individuals with high risk of prostate cancer or considered as an anti-prostate cancer therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.23296599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.23296599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.23296599">
        <p class="paperTitle">Investigation of BRCA1 exon 11 genetic variations in breast cancer among Libyan women</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.23296599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.23296599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elshwekh, H.; Alhudiri, I.; Abdul Jalil, H. M.; Mohamed, A. A.; Elkikli, A.; Jornaz, N.; Enattah, N.</p>
        <p class="info">Score: 1.2, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.23296599' target='https://doi.org/10.1101/2023.10.13.23296599'> 10.1101/2023.10.13.23296599</a></p>
        <p class="abstract">Abstract Introduction The majority of hereditary breast and ovarian cancers are associated with mutations in two genes, breast cancer type 1 and 2 susceptibility genes (BRCA1 and BRCA2). Here, we describe for the first time analysis of BRCA1 exon 11 in 48 Libyan breast cancer patients with a family history of cancer. Methods All patients had a family history of cancer and were included in the study only if they have a family history of breast or ovarian cancer, male breast cancer, or triple negative tumors. PCR was performed using specific primer pairs spanning BRCA1 exon 11 followed by Sanger sequencing. Genetic analysis was done using Sequencher 5.1. Results We identified 12 genetic variants in BRCA1 exon 11. Three variants were novel (c.1019T&gt;C, c.2363T&gt;G, and c.3192T&gt;C). c.2363T&gt;G was predicted by SIFT as damaging. Six variants were of unknown significance (c.918T&gt;C, c.1853G&gt;C, c.1886G&gt;A, c.2215A&gt;G, c.2612C&gt;T, c.3113A&gt;C and c.3784T&gt;C), and 3 were classified as benign in ClinVar database (c.918T&gt;C, c.2082C&gt;T and c.2311T&gt;C). Conclusions Scanning of the entire BRCA1 is needed to identify any associated deleterious mutations. Although the clinical importance of unclassified variants is unknown, the association of certain variants with deleterious mutations and contralateral breast cancer warrants genetic testing and counseling in the Libyan population.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
