<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms
Authors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.
Score: 22.3, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23293001
BackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes. MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms
Authors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.
Score: 22.3, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23293001
BackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes. MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms
Authors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.
Score: 22.3, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23293001
BackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes. MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms\nAuthors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.\nScore: 22.3, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23293001\nBackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes. MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms.",
  "keywords": [
    
  ],
  "articleBody": " Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms\nAuthors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.\nScore: 22.3, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23293001\nBackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes. MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms. ResultsWe observed a strong genetic correlation of anxiety disorder with COVID-19 positive status (rg=0.35, p=2 x 10-4) and COVID-19 hospitalization (rg=0.31, p=7.2 x 10-4). Among anxiety symptoms, \"Tense, sore, or aching muscles during worst period of anxiety\" was genetically correlated with COVID-19 positive status (rg=0.33, p=0.001), while \"Frequent trouble falling or staying asleep during worst period of anxiety\" was genetically correlated with COVID-19 hospitalization (rg=0.24, p=0.004). Through a latent causal variable analysis, we observed that COVID-19 outcomes have statistically significant genetic causality proportion (gcp) on anxiety symptoms (e.g., COVID-19 positive status[-\u0026gt;]\"Recent easy annoyance or irritability\" [boxv]gcp[boxv]=0.18, p=6.72 x 10-17). Conversely, anxiety disorders appear to have a possible causal effect on COVID-19 ([boxv]gcp[boxv]=0.38, p=3.17 x 10-9). Additionally, we also identified multiple loci with evidence of local genetic correlation between anxiety and COVID-19. These appear to be related to genetic effects shared with lung function, brain morphology, alcohol and tobacco use, and hematologic parameters. ConclusionsThis study provided important insights into the relationship between COVID-19 and mental health, differentiating the dynamics linking anxiety disorders to COVID-19 from the effect of COVID-19 on anxiety symptoms.\nGenome-wide analyses reveal widespread genetic overlap between neurological and psychiatric disorders and a convergence of biological associations related to the brain\nAuthors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC), ; The International Headache Genetics Consortium (IHGC), ; Steen, N. E.; van der Meer, D.; O'Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.\nScore: 30.7, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23292993\nNeurological and psychiatric disorders are considered to reflect distinct underlying pathogenic entities. However, the extent to which they share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate extensive genetic overlap across the disorders, with varying degrees of genetic correlations. In particular, migraine, essential tremor, stroke and multiple sclerosis were genetically correlated with several psychiatric disorders. Biological interrogation indicated heterogenous biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study demonstrates that neurological and psychiatric disorders are not genetically disparate, but share key etiological aspects, which have important implications for disease classification, clinical practice, and genomic precision medicine.\nCalibrated prediction intervals for polygenic scores across diverse contexts\nAuthors: Hou, K.; Xu, Z.; Ding, Y.; Harpak, A.; Pasaniuc, B.\nScore: 14.6, Published: 2023-07-27 DOI: 10.1101/2023.07.24.23293056\nPolygenic scores (PGS) have emerged as the tool of choice for genomic prediction in a wide range of fields from agriculture to personalized medicine. We analyze data from two large biobanks in the US (All of Us) and the UK (UK Biobank) to find widespread variability in PGS performance across contexts. Many contexts, including age, sex, and income, impact PGS accuracies with similar magnitudes as genetic ancestry. PGSs trained in single versus multi-ancestry cohorts show similar context-specificity in their accuracies. We introduce trait prediction intervals that are allowed to vary across contexts as a principled approach to account for context-specific PGS accuracy in genomic prediction. We model the impact of all contexts in a joint framework to enable PGS-based trait predictions that are well-calibrated (contain the trait value with 90% probability in all contexts), whereas methods that ignore context are mis-calibrated. We show that prediction intervals need to be adjusted for all considered traits ranging from 10% for diastolic blood pressure to 80% for waist circumference. Adjustment of prediction intervals depends on the dataset; for example, prediction intervals for education years need to be adjusted by 90% in All of Us versus 8% in UK Biobank. Our results provide a path forward towards utilization of PGS as a prediction tool across all individuals regardless of their contexts while highlighting the importance of comprehensive profile of context information in study design and data collection.\nIdentifying risk genes for embryo aneuploidy using ultra-low coverage whole-genome sequencing\nAuthors: Sun, S.; Aboelenain, M.; Ariad, D.; Haywood, M.; Wageman, C.; Duke, M.; Bag, A.; Viotti, M.; Katz-Jaffe, M.; McCoy, R.; Schindler, K.; Xing, J.\nScore: 6.5, Published: 2023-07-24 DOI: 10.1101/2023.07.22.23292618\nBackgroundAneuploidy, the state of a cell containing extra or missing chromosomes, frequently arises during human meiosis and is the primary cause of early miscarriage and maternal age-related in vitro fertilization (IVF) failure. IVF patients exhibit significant variability in aneuploidy rates, although the exact genetic causes of the variability in aneuploid egg production remain unclear. Preimplantation genetic testing for aneuploidy (PGT-A) using ultra-low coverage whole-genome sequencing (ulc-WGS) is a standard test for identifying and selecting IVF-derived embryos with a normal chromosome complement. The wealth of embryo aneuploidy data and ulc-WGS data from PGT-A has potential for discovering variants in paternal genomes that are associated with aneuploidy risk in their embryos. MethodsUsing ulc-WGS data from [~]10,000 PGT-A biopsies, we imputed genotype likelihoods of genetic variants in parental genomes. We then used the imputed variants and aneuploidy calls from the embryos to perform a genome-wide association study of aneuploidy incidence. Finally, we carried out functional evaluation of the identified candidate gene in a mouse oocyte system. ResultsWe identified one locus on chromosome 3 that is significantly associated with maternal meiotic aneuploidy risk. One candidate gene, CCDC66, encompassed by this locus, is involved in chromosome segregation during meiosis. Using mouse oocytes, we showed that CCDC66 regulates meiotic progression and chromosome segregation fidelity, especially in older mice. ConclusionsOur work extended the research utility of PGT-A ulc-WGS data by allowing robust association testing and improved the understanding of the genetic contribution to maternal meiotic aneuploidy risk. Importantly, we introduce a generalizable method that can be leveraged for similar association studies using ulc-WGS data.\nGenetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis\nAuthors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.\nScore: 48.3, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292611\nBackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.\nIntegration of polygenic and gut metagenomic risk prediction for common diseases\nAuthors: Liu, Y.; Ritchie, S. C.; Teo, S. M.; Ruuskanen, M. O.; Kambur, O.; Zhu, Q.; Sanders, J.; Vazquez-Baeza, Y.; Verspoor, K. M.; Jousilahti, P.; Lahti, L.; Niiranen, T.; Salomaa, V.; Havulinna, A.; Knight, R.; Meric, G.; Inouye, M.\nScore: 2.6, Published: 2023-08-05 DOI: 10.1101/2023.07.30.23293396\nMulti-omics has opened new avenues for non-invasive risk profiling and early detection of complex diseases. Both polygenic risk scores (PRSs) and the human microbiome have shown promise in improving risk assessment of various common diseases. Here, in a prospective population-based cohort (FINRISK 2002; n=5,676) with ~18 years of e-health record follow-up, we assess the incremental and combined value of PRSs and gut metagenomic sequencing as compared to conventional risk factors for predicting incident coronary artery disease (CAD), type 2 diabetes (T2D), Alzheimer's disease (AD) and prostate cancer. We found that PRSs improved predictive capacity over conventional risk factors for all diseases ({triangleup}C-indices between 0.010-0.027). In sex-stratified analyses, gut metagenomics improved predictive capacity over baseline age for CAD, T2D and prostate cancer; however, improvement over all conventional risk factors was only observed for T2D ({triangleup}C-index 0.004) and prostate cancer ({triangleup}C-index 0.005). Integrated risk models of PRSs, gut metagenomic scores and conventional risk factors achieved the highest predictive performance for all diseases studied as compared to models based on conventional risk factors alone. We make our integrated risk models available for the wider research community. This study demonstrates that integrated PRS and gut metagenomic risk models improve the predictive value over conventional risk factors for common chronic diseases.\nAssessing feasibility and risk to translate, de-identify and summarize medical letters using deep learning\nAuthors: Gauthier, L. W.; Willems, M.; Chatron, N.; Cenni, C.; Meyer, P.; Ruault, V.; Wells, C.; Sabbagh, Q.; Genevieve, D.; Yauy, K.\nScore: 2.6, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293234\nBackgroundPrecision medicine requires accurate phenotyping and data sharing, particularly for rare diseases. However, sharing medical letters across language barriers is challenging, as inconsistent and incomplete Human Phenotype Ontology (HPO) terms provided by physicians can lead to a loss of clinical information. MethodsTo assess the feasibility and the risk of using deep learning methods to translate, de-identify and summarize medical letters, we developed an open-source deep learning multi-language software in line with health data privacy. We conducted a non-inferiority clinical trial using deep learning methods versus a physician to de-identify protected health information (PHI) targeting a minimum sensitivity of 90% and specificity of 75%, and summarize non-English medical letters in HPO format, aiming a sensitivity of 75% and specificity of 90%. ResultsFrom March to April 2023, we evaluated 50 non-English medical letters from 8 physicians coming from 12 different indications of which neurodevelopmental disorders, congenital disorders, fetal pathology and oncology. Letter contains in median 15 PHI and 7 HPO terms. Deep learning method achieved a sensitivity of 99% and a specificity of 87% in de-identification, and a specificity of 92% in summarizing medical letters, reporting a median number of 6,6 HPO terms per letter, which is equivalent to the number of HPO terms provided usually by physicians in databases (6,8 in PhenoDB). ConclusionsDe-identification and summarization of non-English medical letters using deep learning methods, compared with the current manual physicians method reports non-inferior performance, providing insights on AI usage to facilitate precision medicine.\nA novel gene-screening approach reveals QSOX1/IL1RAP as promising biomarkers for the severity of non-alcoholic fatty liver disease.\nAuthors: Ma, W.; Huang, J.; Cai, B.; Shao, M.; Yu, X.; Kjaer, M. B.; Lv, M.; Zhong, X.; Xu, S.; Zhan, B.; Li, Q.; Huang, Q.; Ma, M.; Cheng, L.; Luo, Y.; Gronbaek, H.; Zhou, X.; Lin, L.\nScore: 1.1, Published: 2023-07-28 DOI: 10.1101/2023.07.26.23293038\nBackground and AimsNon-alcoholic fatty liver disease (NAFLD) is a progressive liver disease that ranges from simple steatosis to inflammation, fibrosis, and cirrhosis. To address the unmet need for new NAFLD biomarkers, we aimed to identify candidate biomarkers using publicly available RNA sequencing (RNA-seq) and proteomics data. MethodsA novel approach involving unsupervised gene clustering was performed using homogeneously processed and integrated RNA-seq data of 625 liver specimens to screen for NAFLD biomarkers in combination with public proteomics data from healthy controls and NAFLD patients. Additionally, we validated the results in the NAFLD and healthy cohorts using enzyme-linked immunosorbent assay (ELISA) of plasma and immunohistochemical staining (IHC) of liver samples. ResultsWe generated a database (https://dreamapp.biomed.au.dk/NAFLD/) for exploring gene expression changes along NAFLD progression to facilitate the identification of genes and pathways involved in the diseases progression. Through cross-analysis of the gene and protein clusters, we identified 38 genes as potential biomarkers for NAFLD severity. Up-regulation of Quiescin sulfhydryl oxidase 1 (QSOX1) and down-regulation of Interleukin-1 receptor accessory protein (IL1RAP) were associated with increasing NAFLD severity in RNA-seq and proteomics data. Remarkably, the QSOX1/IL1RAP ratio in plasma demonstrated effectiveness in diagnosing NAFLD, with an area under the receiver operating characteristic (AUROC) of up to 0.95 as quantified by proteomics profiling and an AUROC of 0.82 with ELISA. ConclusionsWe discovered a significant association between the levels of QSOX1/IL1RAP and NAFLD severity. Furthermore, the QSOX1/IL1RAP ratio shows promise as a non-invasive biomarker for diagnosing NAFLD and assessing its severity. (ChiCTR2300073940). Lay SummaryThis study aimed to find non-invasive biomarkers for non-alcoholic fatty liver disease (NAFLD). Researchers utilized a new gene clustering method to analyze RNA-seq data from 625 liver samples. The identified biomarkers were further validated using plasma proteomics profiling, enzyme-linked immunosorbent assay (ELISA), and liver immunohistochemical staining (IHC) in three separate groups of healthy controls and NAFLD patients. The study revealed that the levels of QSOX1 were elevated while IL1RAP levels were reduced with increasing severity of NAFLD. Notably, the ratio of QSOX1 to IL1RAP expression in plasma showed promise as a non-invasive diagnostic tool for assessing the severity of NAFLD, eliminating the reliance on liver biopsy. HighlightsO_LIRNA-seq data from 625 liver specimens comprising healthy controls and NAFLD patients with increasing severity were utilized for screening NAFLD biomarkers. C_LIO_LIA novel unsupervised method for clustering genes based on the similarity of gene expression across each sample enhanced the discovery of novel effective non-invasive NAFLD biomarkers. C_LIO_LIQSOX1, IL1RAP, and especially the QSOX1/IL1RAP ratio, were found to be associated with NAFLD severity. C_LIO_LIThe high sensitivity of the QSOX1/IL1RAP ratio in predicting NAFLD severity was validated with plasma proteomics quantification (AUROC = 0.95) and ELISA (AUROC = 0.82) in two independent patient cohorts. C_LI Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=\"FIGDIR/small/23293038v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (49K): org.highwire.dtl.DTLVardef@6d2fd6org.highwire.dtl.DTLVardef@5fa998org.highwire.dtl.DTLVardef@774cd0org.highwire.dtl.DTLVardef@53edfa_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA SYSTEMATIC RE-ANALYSIS OF COPY NUMBER LOSSES OF UNCERTAIN CLINICAL SIGNIFICANCE\nAuthors: Burghel, G. J.; Ellingford, J. M.; Wright, R.; Bradford, L.; Miller, J.; Watt, C.; Edgerley, J.; Naeem, F.; Banka, S.\nScore: 14.6, Published: 2023-07-19 DOI: 10.1101/2023.07.02.23292123\nBackgroundRe-analysis of whole exome/genome data improves diagnostic yield. However, the value of re-analysis of clinical array comparative genomic hybridisation (aCGH) data has never been investigated. Case-by-case re-analysis is impractical in busy diagnostic laboratories. Methods and ResultsWe harmonised historical post-natal clinical aCGH results from [~]16,000 patients tested via our diagnostic laboratory over [~]7 years with current clinical guidance. This led to identification of 33,857 benign, 2,173 class 3, and 979 pathogenic copy number losses (CNLs). We found benign CNLs to be significantly less likely to encompass haploinsufficient genes compared to the pathogenic or class 3 CNLs in our database. Using this observation, we developed a re-analysis pipeline (using up-to-date disease association data and haploinsufficiency scores) and shortlisted 207 class 3 CNLs encompassing at least one autosomal dominant disease-gene associated with haploinsufficiency or loss-of-function mechanism. Clinical scientist review led to reclassification of 7.2% shortlisted class 3 CNLs as pathogenic or likely pathogenic. This included first cases of CNV-mediated disease for some genes where all previously described cases involved only point variants. Interestingly, some CNLs could not be re-classified because the phenotypes of patients with CNLs seemed distinct from the known clinical features resulting from point variants, thus raising questions about accepted underlying disease mechanisms. Several potential novel disease-genes were identified that would need further validation. ConclusionsRe-analysis of clinical aCGH data increases diagnostic yield and demonstrates their research value. In future, the aCGH reanalysis program should be expanded to include other copy number variant types.\n",
  "wordCount" : "2995",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23293001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23293001" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23293001">
        <p class="paperTitle">Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23293001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23293001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.</p>
        <p class="info">Score: 22.3, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23293001' target='https://doi.org/10.1101/2023.07.21.23293001'> 10.1101/2023.07.21.23293001</a></p>
        <p class="abstract">BackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes.

MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms.

ResultsWe observed a strong genetic correlation of anxiety disorder with COVID-19 positive status (rg=0.35, p=2 x 10-4) and COVID-19 hospitalization (rg=0.31, p=7.2 x 10-4). Among anxiety symptoms, &#34;Tense, sore, or aching muscles during worst period of anxiety&#34; was genetically correlated with COVID-19 positive status (rg=0.33, p=0.001), while &#34;Frequent trouble falling or staying asleep during worst period of anxiety&#34; was genetically correlated with COVID-19 hospitalization (rg=0.24, p=0.004). Through a latent causal variable analysis, we observed that COVID-19 outcomes have statistically significant genetic causality proportion (gcp) on anxiety symptoms (e.g., COVID-19 positive status[-&amp;gt;]&#34;Recent easy annoyance or irritability&#34; [boxv]gcp[boxv]=0.18, p=6.72 x 10-17). Conversely, anxiety disorders appear to have a possible causal effect on COVID-19 ([boxv]gcp[boxv]=0.38, p=3.17 x 10-9). Additionally, we also identified multiple loci with evidence of local genetic correlation between anxiety and COVID-19. These appear to be related to genetic effects shared with lung function, brain morphology, alcohol and tobacco use, and hematologic parameters.

ConclusionsThis study provided important insights into the relationship between COVID-19 and mental health, differentiating the dynamics linking anxiety disorders to COVID-19 from the effect of COVID-19 on anxiety symptoms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292993">
        <p class="paperTitle">Genome-wide analyses reveal widespread genetic overlap between neurological and psychiatric disorders and a convergence of biological associations related to the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC),  ; The International Headache Genetics Consortium (IHGC),  ; Steen, N. E.; van der Meer, D.; O&#39;Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.</p>
        <p class="info">Score: 30.7, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292993' target='https://doi.org/10.1101/2023.07.21.23292993'> 10.1101/2023.07.21.23292993</a></p>
        <p class="abstract">Neurological and psychiatric disorders are considered to reflect distinct underlying pathogenic entities. However, the extent to which they share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate extensive genetic overlap across the disorders, with varying degrees of genetic correlations. In particular, migraine, essential tremor, stroke and multiple sclerosis were genetically correlated with several psychiatric disorders. Biological interrogation indicated heterogenous biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study demonstrates that neurological and psychiatric disorders are not genetically disparate, but share key etiological aspects, which have important implications for disease classification, clinical practice, and genomic precision medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.23293056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.23293056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.23293056">
        <p class="paperTitle">Calibrated prediction intervals for polygenic scores across diverse contexts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.23293056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.23293056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hou, K.; Xu, Z.; Ding, Y.; Harpak, A.; Pasaniuc, B.</p>
        <p class="info">Score: 14.6, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.23293056' target='https://doi.org/10.1101/2023.07.24.23293056'> 10.1101/2023.07.24.23293056</a></p>
        <p class="abstract">Polygenic scores (PGS) have emerged as the tool of choice for genomic prediction in a wide range of fields from agriculture to personalized medicine. We analyze data from two large biobanks in the US (All of Us) and the UK (UK Biobank) to find widespread variability in PGS performance across contexts. Many contexts, including age, sex, and income, impact PGS accuracies with similar magnitudes as genetic ancestry. PGSs trained in single versus multi-ancestry cohorts show similar context-specificity in their accuracies. We introduce trait prediction intervals that are allowed to vary across contexts as a principled approach to account for context-specific PGS accuracy in genomic prediction. We model the impact of all contexts in a joint framework to enable PGS-based trait predictions that are well-calibrated (contain the trait value with 90% probability in all contexts), whereas methods that ignore context are mis-calibrated. We show that prediction intervals need to be adjusted for all considered traits ranging from 10% for diastolic blood pressure to 80% for waist circumference. Adjustment of prediction intervals depends on the dataset; for example, prediction intervals for education years need to be adjusted by 90% in All of Us versus 8% in UK Biobank. Our results provide a path forward towards utilization of PGS as a prediction tool across all individuals regardless of their contexts while highlighting the importance of comprehensive profile of context information in study design and data collection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.23292618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.23292618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.23292618">
        <p class="paperTitle">Identifying risk genes for embryo aneuploidy using ultra-low coverage whole-genome sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.23292618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.23292618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, S.; Aboelenain, M.; Ariad, D.; Haywood, M.; Wageman, C.; Duke, M.; Bag, A.; Viotti, M.; Katz-Jaffe, M.; McCoy, R.; Schindler, K.; Xing, J.</p>
        <p class="info">Score: 6.5, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.23292618' target='https://doi.org/10.1101/2023.07.22.23292618'> 10.1101/2023.07.22.23292618</a></p>
        <p class="abstract">BackgroundAneuploidy, the state of a cell containing extra or missing chromosomes, frequently arises during human meiosis and is the primary cause of early miscarriage and maternal age-related in vitro fertilization (IVF) failure. IVF patients exhibit significant variability in aneuploidy rates, although the exact genetic causes of the variability in aneuploid egg production remain unclear. Preimplantation genetic testing for aneuploidy (PGT-A) using ultra-low coverage whole-genome sequencing (ulc-WGS) is a standard test for identifying and selecting IVF-derived embryos with a normal chromosome complement. The wealth of embryo aneuploidy data and ulc-WGS data from PGT-A has potential for discovering variants in paternal genomes that are associated with aneuploidy risk in their embryos.

MethodsUsing ulc-WGS data from [~]10,000 PGT-A biopsies, we imputed genotype likelihoods of genetic variants in parental genomes. We then used the imputed variants and aneuploidy calls from the embryos to perform a genome-wide association study of aneuploidy incidence. Finally, we carried out functional evaluation of the identified candidate gene in a mouse oocyte system.

ResultsWe identified one locus on chromosome 3 that is significantly associated with maternal meiotic aneuploidy risk. One candidate gene, CCDC66, encompassed by this locus, is involved in chromosome segregation during meiosis. Using mouse oocytes, we showed that CCDC66 regulates meiotic progression and chromosome segregation fidelity, especially in older mice.

ConclusionsOur work extended the research utility of PGT-A ulc-WGS data by allowing robust association testing and improved the understanding of the genetic contribution to maternal meiotic aneuploidy risk. Importantly, we introduce a generalizable method that can be leveraged for similar association studies using ulc-WGS data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292611">
        <p class="paperTitle">Genetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.</p>
        <p class="info">Score: 48.3, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292611' target='https://doi.org/10.1101/2023.07.13.23292611'> 10.1101/2023.07.13.23292611</a></p>
        <p class="abstract">BackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms.

MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms.

ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases.

Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3.

Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank.

Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS.

ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.23293396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.23293396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.23293396">
        <p class="paperTitle">Integration of polygenic and gut metagenomic risk prediction for common diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.23293396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.23293396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Ritchie, S. C.; Teo, S. M.; Ruuskanen, M. O.; Kambur, O.; Zhu, Q.; Sanders, J.; Vazquez-Baeza, Y.; Verspoor, K. M.; Jousilahti, P.; Lahti, L.; Niiranen, T.; Salomaa, V.; Havulinna, A.; Knight, R.; Meric, G.; Inouye, M.</p>
        <p class="info">Score: 2.6, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.23293396' target='https://doi.org/10.1101/2023.07.30.23293396'> 10.1101/2023.07.30.23293396</a></p>
        <p class="abstract">Multi-omics has opened new avenues for non-invasive risk profiling and early detection of complex diseases. Both polygenic risk scores (PRSs) and the human microbiome have shown promise in improving risk assessment of various common diseases. Here, in a prospective population-based cohort (FINRISK 2002; n=5,676) with ~18 years of e-health record follow-up, we assess the incremental and combined value of PRSs and gut metagenomic sequencing as compared to conventional risk factors for predicting incident coronary artery disease (CAD), type 2 diabetes (T2D), Alzheimer&#39;s disease (AD) and prostate cancer. We found that PRSs improved predictive capacity over conventional risk factors for all diseases ({triangleup}C-indices between 0.010-0.027). In sex-stratified analyses, gut metagenomics improved predictive capacity over baseline age for CAD, T2D and prostate cancer; however, improvement over all conventional risk factors was only observed for T2D ({triangleup}C-index 0.004) and prostate cancer ({triangleup}C-index 0.005). Integrated risk models of PRSs, gut metagenomic scores and conventional risk factors achieved the highest predictive performance for all diseases studied as compared to models based on conventional risk factors alone. We make our integrated risk models available for the wider research community. This study demonstrates that integrated PRS and gut metagenomic risk models improve the predictive value over conventional risk factors for common chronic diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293234">
        <p class="paperTitle">Assessing feasibility and risk to translate, de-identify and summarize medical letters using deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gauthier, L. W.; Willems, M.; Chatron, N.; Cenni, C.; Meyer, P.; Ruault, V.; Wells, C.; Sabbagh, Q.; Genevieve, D.; Yauy, K.</p>
        <p class="info">Score: 2.6, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293234' target='https://doi.org/10.1101/2023.07.27.23293234'> 10.1101/2023.07.27.23293234</a></p>
        <p class="abstract">BackgroundPrecision medicine requires accurate phenotyping and data sharing, particularly for rare diseases. However, sharing medical letters across language barriers is challenging, as inconsistent and incomplete Human Phenotype Ontology (HPO) terms provided by physicians can lead to a loss of clinical information.

MethodsTo assess the feasibility and the risk of using deep learning methods to translate, de-identify and summarize medical letters, we developed an open-source deep learning multi-language software in line with health data privacy. We conducted a non-inferiority clinical trial using deep learning methods versus a physician to de-identify protected health information (PHI) targeting a minimum sensitivity of 90% and specificity of 75%, and summarize non-English medical letters in HPO format, aiming a sensitivity of 75% and specificity of 90%.

ResultsFrom March to April 2023, we evaluated 50 non-English medical letters from 8 physicians coming from 12 different indications of which neurodevelopmental disorders, congenital disorders, fetal pathology and oncology. Letter contains in median 15 PHI and 7 HPO terms. Deep learning method achieved a sensitivity of 99% and a specificity of 87% in de-identification, and a specificity of 92% in summarizing medical letters, reporting a median number of 6,6 HPO terms per letter, which is equivalent to the number of HPO terms provided usually by physicians in databases (6,8 in PhenoDB).

ConclusionsDe-identification and summarization of non-English medical letters using deep learning methods, compared with the current manual physicians method reports non-inferior performance, providing insights on AI usage to facilitate precision medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.23293038">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.23293038" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.23293038">
        <p class="paperTitle">A novel gene-screening approach reveals QSOX1/IL1RAP as promising biomarkers for the severity of non-alcoholic fatty liver disease.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.23293038" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.23293038" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ma, W.; Huang, J.; Cai, B.; Shao, M.; Yu, X.; Kjaer, M. B.; Lv, M.; Zhong, X.; Xu, S.; Zhan, B.; Li, Q.; Huang, Q.; Ma, M.; Cheng, L.; Luo, Y.; Gronbaek, H.; Zhou, X.; Lin, L.</p>
        <p class="info">Score: 1.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.23293038' target='https://doi.org/10.1101/2023.07.26.23293038'> 10.1101/2023.07.26.23293038</a></p>
        <p class="abstract">Background and AimsNon-alcoholic fatty liver disease (NAFLD) is a progressive liver disease that ranges from simple steatosis to inflammation, fibrosis, and cirrhosis. To address the unmet need for new NAFLD biomarkers, we aimed to identify candidate biomarkers using publicly available RNA sequencing (RNA-seq) and proteomics data.

MethodsA novel approach involving unsupervised gene clustering was performed using homogeneously processed and integrated RNA-seq data of 625 liver specimens to screen for NAFLD biomarkers in combination with public proteomics data from healthy controls and NAFLD patients. Additionally, we validated the results in the NAFLD and healthy cohorts using enzyme-linked immunosorbent assay (ELISA) of plasma and immunohistochemical staining (IHC) of liver samples.

ResultsWe generated a database (https://dreamapp.biomed.au.dk/NAFLD/) for exploring gene expression changes along NAFLD progression to facilitate the identification of genes and pathways involved in the diseases progression. Through cross-analysis of the gene and protein clusters, we identified 38 genes as potential biomarkers for NAFLD severity. Up-regulation of Quiescin sulfhydryl oxidase 1 (QSOX1) and down-regulation of Interleukin-1 receptor accessory protein (IL1RAP) were associated with increasing NAFLD severity in RNA-seq and proteomics data. Remarkably, the QSOX1/IL1RAP ratio in plasma demonstrated effectiveness in diagnosing NAFLD, with an area under the receiver operating characteristic (AUROC) of up to 0.95 as quantified by proteomics profiling and an AUROC of 0.82 with ELISA.

ConclusionsWe discovered a significant association between the levels of QSOX1/IL1RAP and NAFLD severity. Furthermore, the QSOX1/IL1RAP ratio shows promise as a non-invasive biomarker for diagnosing NAFLD and assessing its severity. (ChiCTR2300073940).

Lay SummaryThis study aimed to find non-invasive biomarkers for non-alcoholic fatty liver disease (NAFLD). Researchers utilized a new gene clustering method to analyze RNA-seq data from 625 liver samples. The identified biomarkers were further validated using plasma proteomics profiling, enzyme-linked immunosorbent assay (ELISA), and liver immunohistochemical staining (IHC) in three separate groups of healthy controls and NAFLD patients. The study revealed that the levels of QSOX1 were elevated while IL1RAP levels were reduced with increasing severity of NAFLD. Notably, the ratio of QSOX1 to IL1RAP expression in plasma showed promise as a non-invasive diagnostic tool for assessing the severity of NAFLD, eliminating the reliance on liver biopsy.

HighlightsO_LIRNA-seq data from 625 liver specimens comprising healthy controls and NAFLD patients with increasing severity were utilized for screening NAFLD biomarkers.
C_LIO_LIA novel unsupervised method for clustering genes based on the similarity of gene expression across each sample enhanced the discovery of novel effective non-invasive NAFLD biomarkers.
C_LIO_LIQSOX1, IL1RAP, and especially the QSOX1/IL1RAP ratio, were found to be associated with NAFLD severity.
C_LIO_LIThe high sensitivity of the QSOX1/IL1RAP ratio in predicting NAFLD severity was validated with plasma proteomics quantification (AUROC = 0.95) and ELISA (AUROC = 0.82) in two independent patient cohorts.
C_LI

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=&#34;FIGDIR/small/23293038v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (49K):
org.highwire.dtl.DTLVardef@6d2fd6org.highwire.dtl.DTLVardef@5fa998org.highwire.dtl.DTLVardef@774cd0org.highwire.dtl.DTLVardef@53edfa_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.23292123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.23292123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.23292123">
        <p class="paperTitle">A SYSTEMATIC RE-ANALYSIS OF COPY NUMBER LOSSES OF UNCERTAIN CLINICAL SIGNIFICANCE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.23292123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.23292123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burghel, G. J.; Ellingford, J. M.; Wright, R.; Bradford, L.; Miller, J.; Watt, C.; Edgerley, J.; Naeem, F.; Banka, S.</p>
        <p class="info">Score: 14.6, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.23292123' target='https://doi.org/10.1101/2023.07.02.23292123'> 10.1101/2023.07.02.23292123</a></p>
        <p class="abstract">BackgroundRe-analysis of whole exome/genome data improves diagnostic yield. However, the value of re-analysis of clinical array comparative genomic hybridisation (aCGH) data has never been investigated. Case-by-case re-analysis is impractical in busy diagnostic laboratories.

Methods and ResultsWe harmonised historical post-natal clinical aCGH results from [~]16,000 patients tested via our diagnostic laboratory over [~]7 years with current clinical guidance. This led to identification of 33,857 benign, 2,173 class 3, and 979 pathogenic copy number losses (CNLs). We found benign CNLs to be significantly less likely to encompass haploinsufficient genes compared to the pathogenic or class 3 CNLs in our database. Using this observation, we developed a re-analysis pipeline (using up-to-date disease association data and haploinsufficiency scores) and shortlisted 207 class 3 CNLs encompassing at least one autosomal dominant disease-gene associated with haploinsufficiency or loss-of-function mechanism. Clinical scientist review led to reclassification of 7.2% shortlisted class 3 CNLs as pathogenic or likely pathogenic. This included first cases of CNV-mediated disease for some genes where all previously described cases involved only point variants. Interestingly, some CNLs could not be re-classified because the phenotypes of patients with CNLs seemed distinct from the known clinical features resulting from point variants, thus raising questions about accepted underlying disease mechanisms. Several potential novel disease-genes were identified that would need further validation.

ConclusionsRe-analysis of clinical aCGH data increases diagnostic yield and demonstrates their research value. In future, the aCGH reanalysis program should be expanded to include other copy number variant types.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
