<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Copy-number variants as modulators of common disease susceptibility
Authors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; Mägi, R.; Estonian Biobank Research Team, ; Reymond, A.; Kutalik, Z.
Score: 15.9, Published: 2023-08-05 DOI: 10.1101/2023.07.31.23293408
BackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described. MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Copy-number variants as modulators of common disease susceptibility
Authors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; Mägi, R.; Estonian Biobank Research Team, ; Reymond, A.; Kutalik, Z.
Score: 15.9, Published: 2023-08-05 DOI: 10.1101/2023.07.31.23293408
BackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described. MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Copy-number variants as modulators of common disease susceptibility
Authors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; Mägi, R.; Estonian Biobank Research Team, ; Reymond, A.; Kutalik, Z.
Score: 15.9, Published: 2023-08-05 DOI: 10.1101/2023.07.31.23293408
BackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described. MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Copy-number variants as modulators of common disease susceptibility\nAuthors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; Mägi, R.; Estonian Biobank Research Team, ; Reymond, A.; Kutalik, Z.\nScore: 15.9, Published: 2023-08-05 DOI: 10.1101/2023.07.31.23293408\nBackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described. MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank.",
  "keywords": [
    
  ],
  "articleBody": " Copy-number variants as modulators of common disease susceptibility\nAuthors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; Mägi, R.; Estonian Biobank Research Team, ; Reymond, A.; Kutalik, Z.\nScore: 15.9, Published: 2023-08-05 DOI: 10.1101/2023.07.31.23293408\nBackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described. MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank. ResultsWe identified 73 signals involving 40 diseases, all of which indicating that CNVs increased disease risk and caused earlier onset. Even after correcting for these signals, a higher CNV burden increased risk for 18 disorders, mainly through the number of deleted genes, suggesting a polygenic CNV architecture. Number and identity of genes disturbed by CNVs affected their pathogenicity, with many associations being supported by colocalization with both common and rare single nucleotide variant association signals. Dissection of association signals provided insights into the epidemiology of known gene-disease pairs (e.g., deletions in BRCA1 and LDLR increased risk for ovarian cancer and ischemic heart disease, respectively), clarified dosage mechanisms of action (e.g., both increased and decreased dosage of 17q12 impacts renal health), and identified putative causal genes (e.g., ABCC6 for kidney stones). Characterization of the pleiotropic pathological consequences of recurrent CNVs at 15q13, 16p13.11, 16p12.2, and 22q11.2 in adulthood indicated variable expressivity of these regions and the involvement of multiple genes. ConclusionsOur results shed light on the prominent role of CNVs in determining common disease susceptibility within the general population and provide actionable insights allowing to anticipate later-onset comorbidities in carriers of recurrent CNVs.\nA study of gene expression in the living human brain\nAuthors: Liharska, L. E.; Park, Y. J.; Ziafat, K.; Wilkins, L.; Silk, H.; Linares, L. M.; Vornholt, E.; Sullivan, B.; Cohen, V.; Kota, P.; Feng, C.; Cheng, E.; Moya, E.; Thompson, R. C.; Johnson, J. S.; Rieder, M.-K.; Huang, J.; Scarpa, J.; Hashemi, A.; Polanco, J.; Levin, M. A.; Nadkarni, G. N.; Sebra, R.; Crary, J.; Schadt, E. E.; Beckmann, N. D.; Kopell, B. H.; Charney, A. W.\nScore: 248.3, Published: 2023-08-01 DOI: 10.1101/2023.04.21.23288916\nA goal of medical research is to determine the molecular basis of human brain health and illness. One way to achieve this goal is through observational studies of gene expression in human brain tissue. Due to the unavailability of brain tissue from living people, most such studies are performed using tissue from postmortem brain donors. An assumption underlying this practice is that gene expression in the postmortem human brain is an accurate representation of gene expression in the living human brain. Here, this assumption - which, until now, had not been adequately tested - is tested by comparing human prefrontal cortex gene expression between 275 living samples and 243 postmortem samples. Expression levels differed significantly for nearly 80% of genes, and a systematic examination of alternative explanations for this observation determined that these differences are not a consequence of cell type composition, RNA quality, postmortem interval, age, medication, morbidity, symptom severity, tissue pathology, sample handling, batch effects, or computational methods utilized. Analyses integrating the data generated for this study with data from earlier landmark studies that used tissue from postmortem brain donors showed that postmortem brain gene expression signatures of neurological and mental illnesses, as well as of normal traits such as aging, may not be accurate representations of these gene expression signatures in the living brain. By using tissue from large cohorts living people, future observational studies of human brain biology have the potential to (1) determine the medical research questions that can be addressed using postmortem tissue as a proxy for living tissue and (2) expand the scope of medical research to include questions about the molecular basis of human brain health and illness that can only be addressed in living people (e.g., \"What happens at the molecular level in the brain as a person experiences an emotion?\").\nIntegration of polygenic and gut metagenomic risk prediction for common diseases\nAuthors: Liu, Y.; Ritchie, S. C.; Teo, S. M.; Ruuskanen, M. O.; Kambur, O.; Zhu, Q.; Sanders, J.; Vazquez-Baeza, Y.; Verspoor, K. M.; Jousilahti, P.; Lahti, L.; Niiranen, T.; Salomaa, V.; Havulinna, A.; Knight, R.; Meric, G.; Inouye, M.\nScore: 10.7, Published: 2023-08-05 DOI: 10.1101/2023.07.30.23293396\nMulti-omics has opened new avenues for non-invasive risk profiling and early detection of complex diseases. Both polygenic risk scores (PRSs) and the human microbiome have shown promise in improving risk assessment of various common diseases. Here, in a prospective population-based cohort (FINRISK 2002; n=5,676) with [~]18 years of e-health record follow-up, we assess the incremental and combined value of PRSs and gut metagenomic sequencing as compared to conventional risk factors for predicting incident coronary artery disease (CAD), type 2 diabetes (T2D), Alzheimers disease (AD) and prostate cancer. We found that PRSs improved predictive capacity over conventional risk factors for all diseases ({Delta}C-indices between 0.010 - 0.027). In sex-stratified analyses, gut metagenomics improved predictive capacity over baseline age for CAD, T2D and prostate cancer; however, improvement over all conventional risk factors was only observed for T2D ({Delta}C-index 0.004) and prostate cancer ({Delta}C-index 0.005). Integrated risk models of PRSs, gut metagenomic scores and conventional risk factors achieved the highest predictive performance for all diseases studied as compared to models based on conventional risk factors alone. We make our integrated risk models available for the wider research community. This study demonstrates that integrated PRS and gut metagenomic risk models improve the predictive value over conventional risk factors for common chronic diseases.\nCritical assessment of variant prioritization methods for rare disease diagnosis within the Rare Genomes Project\nAuthors: Stenton, S. L.; O'Leary, M.; Lemire, G.; VanNoy, G. E.; DiTroia, S.; Ganesh, V. S.; Groopman, E.; O'Heir, E.; Mangilog, B.; Osei-Owusu, I.; Pais, L. S.; Serrano, J.; Singer-Berk, M.; Weisburd, B.; Wilson, M.; Austin-Tse, C.; Abdelhakim, M.; Althagafi, A.; Babbi, G.; Bellazzi, R.; Bovo, S.; Carta, M. G.; Casadio, R.; Coenen, P.-J.; De Paoli, F.; Floris, M.; Gajapathy, M.; Hoehndorf, R.; Jacobsen, J. O. B.; Joseph, T.; Kamandula, A.; Katsonis, P.; Kint, C.; Lichtarge, O.; Limongelli, I.; Lu, Y.; Magni, P.; Mamidi, T. K. K.; Martelli, P. L.; Mulargia, M.; Nicora, G.; Nykamp, K.; Pejaver, V.; Peng\nScore: 7.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293212\nBackgroundA major obstacle faced by rare disease families is obtaining a genetic diagnosis. The average \"diagnostic odyssey\" lasts over five years, and causal variants are identified in under 50%. The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing (GS) for diagnosis and gene discovery. Families are consented for sharing of sequence and phenotype data with researchers, allowing development of a Critical Assessment of Genome Interpretation (CAGI) community challenge, placing variant prioritization models head-to-head in a real-life clinical diagnostic setting. MethodsPredictors were provided a dataset of phenotype terms and variant calls from GS of 175 RGP individuals (65 families), including 35 solved training set families, with causal variants specified, and 30 test set families (14 solved, 16 unsolved). The challenge tasked teams with identifying the causal variants in as many test set families as possible. Ranked variant predictions were submitted with estimated probability of causal relationship (EPCR) values. Model performance was determined by two metrics, a weighted score based on rank position of true positive causal variants and maximum F-measure, based on precision and recall of causal variants across EPCR thresholds. ResultsSixteen teams submitted predictions from 52 models, some with manual review incorporated. Top performing teams recalled the causal variants in up to 13 of 14 solved families by prioritizing high quality variant calls that were rare, predicted deleterious, segregating correctly, and consistent with reported phenotype. In unsolved families, newly discovered diagnostic variants were returned to two families following confirmatory RNA sequencing, and two prioritized novel disease gene candidates were entered into Matchmaker Exchange. In one example, RNA sequencing demonstrated aberrant splicing due to a deep intronic indel in ASNS, identified in trans with a frameshift variant, in an unsolved proband with phenotype overlap with asparagine synthetase deficiency. ConclusionsBy objective assessment of variant predictions, we provide insights into current state-of-the-art algorithms and platforms for genome sequencing analysis for rare disease diagnosis and explore areas for future optimization. Identification of diagnostic variants in unsolved families promotes synergy between researchers with clinical and computational expertise as a means of advancing the field of clinical genome interpretation.\nGenetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis\nAuthors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.\nScore: 51.5, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292611\nBackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.\nStrong protective effect of the APOL1 p.N264K variant against G2-associated FSGS and kidney disease\nAuthors: Gupta, Y.; Friedman, D. J.; McNulty, M.; Khan, A.; Lane, B.; Wang, C.; Ke, J.; Jin, G.; Wooden, B.; Knob, A. L.; Lim, T. Y.; Appel, G. B.; Huggins, K.; Liu, L.; Mitrotti, A.; Stangl, M. C.; Bomback, A.; Westland, R.; Bodria, M.; Marasa, M.; Shang, N.; Cohen, D. J.; Crew, R. J.; Morello, W.; Canetta, P.; Radhakrishnan, J.; Martino, J.; Liu, Q.; Chung, W. K.; Espinoza, A.; Luo, Y.; Wei, W.-Q.; Feng, Q.; Weng, C.; Fang, Y.; Kullo, I. J.; Naderian, M.; Limdi, N.; Irvin, M. R.; Tiwari, H.; Mohan, S.; Rao, M.; Dube, G.; Chaudhary, N. S.; Gutierrez, O. M.; Judd, S. E.; Cushman, M.; Lange, L. A.; Lang\nScore: 6.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293554\nBlack Americans have a significantly higher risk of developing chronic kidney disease (CKD), especially focal segmental glomerulosclerosis (FSGS), than European Americans. Two coding variants (G1 and G2) in the APOL1 gene play a major role in this disparity. While 13% of Black Americans carry the high-risk recessive genotypes, only a fraction of these individuals develops FSGS or kidney failure, indicating the involvement of additional disease modifiers. Here, we show that the presence of the APOL1 p.N264K missense variant, when co-inherited with the G2 APOL1 risk allele, substantially reduces the penetrance of the G1G2 and G2G2 high-risk genotypes by rendering these genotypes low-risk. These results align with prior functional evidence showing that the p.N264K variant reduces the toxicity of the APOL1 high-risk alleles. These findings have important implications for our understanding of the mechanisms of APOL1-associated nephropathy, as well as for the clinical management of individuals with high-risk genotypes that include the G2 allele.\nAssessing feasibility and risk to translate, de-identify and summarize medical letters using deep learning\nAuthors: Gauthier, L. W.; Willems, M.; Chatron, N.; Cenni, C.; Meyer, P.; Ruault, V.; Wells, C.; Sabbagh, Q.; Genevieve, D.; Yauy, K.\nScore: 2.6, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293234\nBackgroundPrecision medicine requires accurate phenotyping and data sharing, particularly for rare diseases. However, sharing medical letters across language barriers is challenging, as inconsistent and incomplete Human Phenotype Ontology (HPO) terms provided by physicians can lead to a loss of clinical information. MethodsTo assess the feasibility and the risk of using deep learning methods to translate, de-identify and summarize medical letters, we developed an open-source deep learning multi-language software in line with health data privacy. We conducted a non-inferiority clinical trial using deep learning methods versus a physician to de-identify protected health information (PHI) targeting a minimum sensitivity of 90% and specificity of 75%, and summarize non-English medical letters in HPO format, aiming a sensitivity of 75% and specificity of 90%. ResultsFrom March to April 2023, we evaluated 50 non-English medical letters from 8 physicians coming from 12 different indications of which neurodevelopmental disorders, congenital disorders, fetal pathology and oncology. Letter contains in median 15 PHI and 7 HPO terms. Deep learning method achieved a sensitivity of 99% and a specificity of 87% in de-identification, and a specificity of 92% in summarizing medical letters, reporting a median number of 6,6 HPO terms per letter, which is equivalent to the number of HPO terms provided usually by physicians in databases (6,8 in PhenoDB). ConclusionsDe-identification and summarization of non-English medical letters using deep learning methods, compared with the current manual physicians method reports non-inferior performance, providing insights on AI usage to facilitate precision medicine.\nExome sequencing in Asian populations identifies rare deficient SMPD1 alleles that increase risk of Parkinson's disease\nAuthors: Chew, E. G.; Liu, Z.; Li, Z.; Chung, S. J.; Lian, M. M.; Tandiono, M.; Ng, E. Y.; Tan, L. C.; Chng, W. L.; Tan, T. J.; Peh, E. K.; Ho, Y. S.; Chen, X. Y.; Lim, E. Y.; Chang, C. H.; Leong, J. J.; Heng, Y. J.; Peh, T. X.; Chan, L.-L.; Chao, Y.; Au, W.-L.; Prakash, K. M.; Lim, J. L.; Tay, Y. W.; Mok, V.; Chan, A. Y.; Lin, J.-J.; Jeon, B. S.; Song, K.; Tham, C. C.; Pang, C. P.; Ahn, J.; Park, K. H.; Wiggs, J. L.; Aung, T.; Tan, A.-H.; Ahmad Annuar, A.; Makarious, M. B.; Blauwendraat, C.; Nalls, M. A.; Robak, L. A.; Alcalay, R. N.; Gan-Or, Z.; Lim, S.-Y.; Khor, C.-C.; Tan, E.-K.; Wang, Z.; Foo, J. Score: 2.1, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293387\nParkinson's disease is an incurable and progressive disease that adversely affects balance, muscle control, and movement. We hypothesized that the landscape of rare, protein-altering genetic variants could provide further mechanistic insights into disease pathogenesis. We performed whole-exome sequencing on 4,298 persons with Parkinson's disease and 5,512 unaffected controls from Singapore, Malaysia, Hong Kong, South Korea, and Taiwan. We tested for association between gene-based burden of rare, predicted damaging variants and risk of Parkinson's disease. Genes surpassing exome-wide significance (P\u003c2.5x10-6) were tested for replication in sequencing data from a further 5,585 Parkinson's disease patients and 5,642 controls of Asian and European ancestry. We observed that carriage of rare, protein-altering variants that were predicted to impair protein function at SMPD1 (a gene encoding for acid sphingomyelinase) were significantly associated with increased risk of Parkinson's disease. Refinement of variant classification using functional acid sphingomyelinase assays suggest that individuals carrying SMPD1 variants with less than 44 percent of normal enzymatic activity show the strongest association with Parkinson's disease risk in both the discovery (odds ratio (OR) = 2.37, 95% CI = 1.68 - 3.35, P = 4.35 x 10-7) and replication collections (OR = 2.18, 95% CI = 1.69 - 2.81, P = 4.80 x 10-10), leading to a significant observation when all data were meta-analyzed (OR = 2.24, 95% CI = 1.83 - 2.76, P = 1.25 x 10-15). Our findings affirm the importance of sphingomyelin metabolism in the pathobiology of neurodegenerative diseases and highlights the utility of functional genomic assays in large-scale exome sequencing studies.\nGenome-wide analyses reveal widespread genetic overlap between neurological and psychiatric disorders and a convergence of biological associations related to the brain\nAuthors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC), ; The International Headache Genetics Consortium (IHGC), ; Steen, N. E.; van der Meer, D.; O'Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.\nScore: 30.7, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23292993\nNeurological and psychiatric disorders are considered to reflect distinct underlying pathogenic entities. However, the extent to which they share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate extensive genetic overlap across the disorders, with varying degrees of genetic correlations. In particular, migraine, essential tremor, stroke and multiple sclerosis were genetically correlated with several psychiatric disorders. Biological interrogation indicated heterogenous biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study demonstrates that neurological and psychiatric disorders are not genetically disparate, but share key etiological aspects, which have important implications for disease classification, clinical practice, and genomic precision medicine.\n",
  "wordCount" : "3194",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.23293408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.23293408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.23293408">
        <p class="paperTitle">Copy-number variants as modulators of common disease susceptibility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.23293408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.23293408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Auwerx, C.; Joeloo, M.; Sadler, M. C.; Tesio, N.; Ojavee, S. E.; Clark, C. J.; Mägi, R.; Estonian Biobank Research Team,  ; Reymond, A.; Kutalik, Z.</p>
        <p class="info">Score: 15.9, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.23293408' target='https://doi.org/10.1101/2023.07.31.23293408'> 10.1101/2023.07.31.23293408</a></p>
        <p class="abstract">BackgroundCopy-number variations (CNVs) have been associated with rare and debilitating genomic syndromes but their impact on health later in life in the general population remains poorly described.

MethodsAssessing four modes of CNV action, we performed genome-wide association scans (GWASs) between the copy-number of CNV-proxy probes and 60 curated ICD-10 based clinical diagnoses in 331,522 unrelated white UK Biobank participants with replication in the Estonian Biobank.

ResultsWe identified 73 signals involving 40 diseases, all of which indicating that CNVs increased disease risk and caused earlier onset. Even after correcting for these signals, a higher CNV burden increased risk for 18 disorders, mainly through the number of deleted genes, suggesting a polygenic CNV architecture. Number and identity of genes disturbed by CNVs affected their pathogenicity, with many associations being supported by colocalization with both common and rare single nucleotide variant association signals. Dissection of association signals provided insights into the epidemiology of known gene-disease pairs (e.g., deletions in BRCA1 and LDLR increased risk for ovarian cancer and ischemic heart disease, respectively), clarified dosage mechanisms of action (e.g., both increased and decreased dosage of 17q12 impacts renal health), and identified putative causal genes (e.g., ABCC6 for kidney stones). Characterization of the pleiotropic pathological consequences of recurrent CNVs at 15q13, 16p13.11, 16p12.2, and 22q11.2 in adulthood indicated variable expressivity of these regions and the involvement of multiple genes.

ConclusionsOur results shed light on the prominent role of CNVs in determining common disease susceptibility within the general population and provide actionable insights allowing to anticipate later-onset comorbidities in carriers of recurrent CNVs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.21.23288916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.21.23288916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.21.23288916">
        <p class="paperTitle">A study of gene expression in the living human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.21.23288916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.21.23288916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liharska, L. E.; Park, Y. J.; Ziafat, K.; Wilkins, L.; Silk, H.; Linares, L. M.; Vornholt, E.; Sullivan, B.; Cohen, V.; Kota, P.; Feng, C.; Cheng, E.; Moya, E.; Thompson, R. C.; Johnson, J. S.; Rieder, M.-K.; Huang, J.; Scarpa, J.; Hashemi, A.; Polanco, J.; Levin, M. A.; Nadkarni, G. N.; Sebra, R.; Crary, J.; Schadt, E. E.; Beckmann, N. D.; Kopell, B. H.; Charney, A. W.</p>
        <p class="info">Score: 248.3, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.21.23288916' target='https://doi.org/10.1101/2023.04.21.23288916'> 10.1101/2023.04.21.23288916</a></p>
        <p class="abstract">A goal of medical research is to determine the molecular basis of human brain health and illness. One way to achieve this goal is through observational studies of gene expression in human brain tissue. Due to the unavailability of brain tissue from living people, most such studies are performed using tissue from postmortem brain donors. An assumption underlying this practice is that gene expression in the postmortem human brain is an accurate representation of gene expression in the living human brain. Here, this assumption - which, until now, had not been adequately tested - is tested by comparing human prefrontal cortex gene expression between 275 living samples and 243 postmortem samples. Expression levels differed significantly for nearly 80% of genes, and a systematic examination of alternative explanations for this observation determined that these differences are not a consequence of cell type composition, RNA quality, postmortem interval, age, medication, morbidity, symptom severity, tissue pathology, sample handling, batch effects, or computational methods utilized. Analyses integrating the data generated for this study with data from earlier landmark studies that used tissue from postmortem brain donors showed that postmortem brain gene expression signatures of neurological and mental illnesses, as well as of normal traits such as aging, may not be accurate representations of these gene expression signatures in the living brain. By using tissue from large cohorts living people, future observational studies of human brain biology have the potential to (1) determine the medical research questions that can be addressed using postmortem tissue as a proxy for living tissue and (2) expand the scope of medical research to include questions about the molecular basis of human brain health and illness that can only be addressed in living people (e.g., &#34;What happens at the molecular level in the brain as a person experiences an emotion?&#34;).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.23293396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.23293396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.23293396">
        <p class="paperTitle">Integration of polygenic and gut metagenomic risk prediction for common diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.23293396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.23293396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Ritchie, S. C.; Teo, S. M.; Ruuskanen, M. O.; Kambur, O.; Zhu, Q.; Sanders, J.; Vazquez-Baeza, Y.; Verspoor, K. M.; Jousilahti, P.; Lahti, L.; Niiranen, T.; Salomaa, V.; Havulinna, A.; Knight, R.; Meric, G.; Inouye, M.</p>
        <p class="info">Score: 10.7, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.23293396' target='https://doi.org/10.1101/2023.07.30.23293396'> 10.1101/2023.07.30.23293396</a></p>
        <p class="abstract">Multi-omics has opened new avenues for non-invasive risk profiling and early detection of complex diseases. Both polygenic risk scores (PRSs) and the human microbiome have shown promise in improving risk assessment of various common diseases. Here, in a prospective population-based cohort (FINRISK 2002; n=5,676) with [~]18 years of e-health record follow-up, we assess the incremental and combined value of PRSs and gut metagenomic sequencing as compared to conventional risk factors for predicting incident coronary artery disease (CAD), type 2 diabetes (T2D), Alzheimers disease (AD) and prostate cancer. We found that PRSs improved predictive capacity over conventional risk factors for all diseases ({Delta}C-indices between 0.010 - 0.027). In sex-stratified analyses, gut metagenomics improved predictive capacity over baseline age for CAD, T2D and prostate cancer; however, improvement over all conventional risk factors was only observed for T2D ({Delta}C-index 0.004) and prostate cancer ({Delta}C-index 0.005). Integrated risk models of PRSs, gut metagenomic scores and conventional risk factors achieved the highest predictive performance for all diseases studied as compared to models based on conventional risk factors alone. We make our integrated risk models available for the wider research community. This study demonstrates that integrated PRS and gut metagenomic risk models improve the predictive value over conventional risk factors for common chronic diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293212">
        <p class="paperTitle">Critical assessment of variant prioritization methods for rare disease diagnosis within the Rare Genomes Project</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stenton, S. L.; O&#39;Leary, M.; Lemire, G.; VanNoy, G. E.; DiTroia, S.; Ganesh, V. S.; Groopman, E.; O&#39;Heir, E.; Mangilog, B.; Osei-Owusu, I.; Pais, L. S.; Serrano, J.; Singer-Berk, M.; Weisburd, B.; Wilson, M.; Austin-Tse, C.; Abdelhakim, M.; Althagafi, A.; Babbi, G.; Bellazzi, R.; Bovo, S.; Carta, M. G.; Casadio, R.; Coenen, P.-J.; De Paoli, F.; Floris, M.; Gajapathy, M.; Hoehndorf, R.; Jacobsen, J. O. B.; Joseph, T.; Kamandula, A.; Katsonis, P.; Kint, C.; Lichtarge, O.; Limongelli, I.; Lu, Y.; Magni, P.; Mamidi, T. K. K.; Martelli, P. L.; Mulargia, M.; Nicora, G.; Nykamp, K.; Pejaver, V.; Peng</p>
        <p class="info">Score: 7.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293212' target='https://doi.org/10.1101/2023.08.02.23293212'> 10.1101/2023.08.02.23293212</a></p>
        <p class="abstract">BackgroundA major obstacle faced by rare disease families is obtaining a genetic diagnosis. The average &#34;diagnostic odyssey&#34; lasts over five years, and causal variants are identified in under 50%. The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing (GS) for diagnosis and gene discovery. Families are consented for sharing of sequence and phenotype data with researchers, allowing development of a Critical Assessment of Genome Interpretation (CAGI) community challenge, placing variant prioritization models head-to-head in a real-life clinical diagnostic setting.

MethodsPredictors were provided a dataset of phenotype terms and variant calls from GS of 175 RGP individuals (65 families), including 35 solved training set families, with causal variants specified, and 30 test set families (14 solved, 16 unsolved). The challenge tasked teams with identifying the causal variants in as many test set families as possible. Ranked variant predictions were submitted with estimated probability of causal relationship (EPCR) values. Model performance was determined by two metrics, a weighted score based on rank position of true positive causal variants and maximum F-measure, based on precision and recall of causal variants across EPCR thresholds.

ResultsSixteen teams submitted predictions from 52 models, some with manual review incorporated. Top performing teams recalled the causal variants in up to 13 of 14 solved families by prioritizing high quality variant calls that were rare, predicted deleterious, segregating correctly, and consistent with reported phenotype. In unsolved families, newly discovered diagnostic variants were returned to two families following confirmatory RNA sequencing, and two prioritized novel disease gene candidates were entered into Matchmaker Exchange. In one example, RNA sequencing demonstrated aberrant splicing due to a deep intronic indel in ASNS, identified in trans with a frameshift variant, in an unsolved proband with phenotype overlap with asparagine synthetase deficiency.

ConclusionsBy objective assessment of variant predictions, we provide insights into current state-of-the-art algorithms and platforms for genome sequencing analysis for rare disease diagnosis and explore areas for future optimization. Identification of diagnostic variants in unsolved families promotes synergy between researchers with clinical and computational expertise as a means of advancing the field of clinical genome interpretation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292611">
        <p class="paperTitle">Genetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.</p>
        <p class="info">Score: 51.5, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292611' target='https://doi.org/10.1101/2023.07.13.23292611'> 10.1101/2023.07.13.23292611</a></p>
        <p class="abstract">BackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms.

MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms.

ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases.

Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3.

Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank.

Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS.

ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293554">
        <p class="paperTitle">Strong protective effect of the APOL1 p.N264K variant against G2-associated FSGS and kidney disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, Y.; Friedman, D. J.; McNulty, M.; Khan, A.; Lane, B.; Wang, C.; Ke, J.; Jin, G.; Wooden, B.; Knob, A. L.; Lim, T. Y.; Appel, G. B.; Huggins, K.; Liu, L.; Mitrotti, A.; Stangl, M. C.; Bomback, A.; Westland, R.; Bodria, M.; Marasa, M.; Shang, N.; Cohen, D. J.; Crew, R. J.; Morello, W.; Canetta, P.; Radhakrishnan, J.; Martino, J.; Liu, Q.; Chung, W. K.; Espinoza, A.; Luo, Y.; Wei, W.-Q.; Feng, Q.; Weng, C.; Fang, Y.; Kullo, I. J.; Naderian, M.; Limdi, N.; Irvin, M. R.; Tiwari, H.; Mohan, S.; Rao, M.; Dube, G.; Chaudhary, N. S.; Gutierrez, O. M.; Judd, S. E.; Cushman, M.; Lange, L. A.; Lang</p>
        <p class="info">Score: 6.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293554' target='https://doi.org/10.1101/2023.08.02.23293554'> 10.1101/2023.08.02.23293554</a></p>
        <p class="abstract">Black Americans have a significantly higher risk of developing chronic kidney disease (CKD), especially focal segmental glomerulosclerosis (FSGS), than European Americans. Two coding variants (G1 and G2) in the APOL1 gene play a major role in this disparity. While 13% of Black Americans carry the high-risk recessive genotypes, only a fraction of these individuals develops FSGS or kidney failure, indicating the involvement of additional disease modifiers.

Here, we show that the presence of the APOL1 p.N264K missense variant, when co-inherited with the G2 APOL1 risk allele, substantially reduces the penetrance of the G1G2 and G2G2 high-risk genotypes by rendering these genotypes low-risk. These results align with prior functional evidence showing that the p.N264K variant reduces the toxicity of the APOL1 high-risk alleles. These findings have important implications for our understanding of the mechanisms of APOL1-associated nephropathy, as well as for the clinical management of individuals with high-risk genotypes that include the G2 allele.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293234">
        <p class="paperTitle">Assessing feasibility and risk to translate, de-identify and summarize medical letters using deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gauthier, L. W.; Willems, M.; Chatron, N.; Cenni, C.; Meyer, P.; Ruault, V.; Wells, C.; Sabbagh, Q.; Genevieve, D.; Yauy, K.</p>
        <p class="info">Score: 2.6, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293234' target='https://doi.org/10.1101/2023.07.27.23293234'> 10.1101/2023.07.27.23293234</a></p>
        <p class="abstract">BackgroundPrecision medicine requires accurate phenotyping and data sharing, particularly for rare diseases. However, sharing medical letters across language barriers is challenging, as inconsistent and incomplete Human Phenotype Ontology (HPO) terms provided by physicians can lead to a loss of clinical information.

MethodsTo assess the feasibility and the risk of using deep learning methods to translate, de-identify and summarize medical letters, we developed an open-source deep learning multi-language software in line with health data privacy. We conducted a non-inferiority clinical trial using deep learning methods versus a physician to de-identify protected health information (PHI) targeting a minimum sensitivity of 90% and specificity of 75%, and summarize non-English medical letters in HPO format, aiming a sensitivity of 75% and specificity of 90%.

ResultsFrom March to April 2023, we evaluated 50 non-English medical letters from 8 physicians coming from 12 different indications of which neurodevelopmental disorders, congenital disorders, fetal pathology and oncology. Letter contains in median 15 PHI and 7 HPO terms. Deep learning method achieved a sensitivity of 99% and a specificity of 87% in de-identification, and a specificity of 92% in summarizing medical letters, reporting a median number of 6,6 HPO terms per letter, which is equivalent to the number of HPO terms provided usually by physicians in databases (6,8 in PhenoDB).

ConclusionsDe-identification and summarization of non-English medical letters using deep learning methods, compared with the current manual physicians method reports non-inferior performance, providing insights on AI usage to facilitate precision medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.23293387">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.23293387" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.23293387">
        <p class="paperTitle">Exome sequencing in Asian populations identifies rare deficient SMPD1 alleles that increase risk of Parkinson&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.23293387" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.23293387" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chew, E. G.; Liu, Z.; Li, Z.; Chung, S. J.; Lian, M. M.; Tandiono, M.; Ng, E. Y.; Tan, L. C.; Chng, W. L.; Tan, T. J.; Peh, E. K.; Ho, Y. S.; Chen, X. Y.; Lim, E. Y.; Chang, C. H.; Leong, J. J.; Heng, Y. J.; Peh, T. X.; Chan, L.-L.; Chao, Y.; Au, W.-L.; Prakash, K. M.; Lim, J. L.; Tay, Y. W.; Mok, V.; Chan, A. Y.; Lin, J.-J.; Jeon, B. S.; Song, K.; Tham, C. C.; Pang, C. P.; Ahn, J.; Park, K. H.; Wiggs, J. L.; Aung, T.; Tan, A.-H.; Ahmad Annuar, A.; Makarious, M. B.; Blauwendraat, C.; Nalls, M. A.; Robak, L. A.; Alcalay, R. N.; Gan-Or, Z.; Lim, S.-Y.; Khor, C.-C.; Tan, E.-K.; Wang, Z.; Foo, J. </p>
        <p class="info">Score: 2.1, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.23293387' target='https://doi.org/10.1101/2023.08.03.23293387'> 10.1101/2023.08.03.23293387</a></p>
        <p class="abstract">Parkinson&#39;s disease is an incurable and progressive disease that adversely affects balance, muscle control, and movement. We hypothesized that the landscape of rare, protein-altering genetic variants could provide further mechanistic insights into disease pathogenesis. We performed whole-exome sequencing on 4,298 persons with Parkinson&#39;s disease and 5,512 unaffected controls from Singapore, Malaysia, Hong Kong, South Korea, and Taiwan. We tested for association between gene-based burden of rare, predicted damaging variants and risk of Parkinson&#39;s disease. Genes surpassing exome-wide significance (P&lt;2.5x10-6) were tested for replication in sequencing data from a further 5,585 Parkinson&#39;s disease patients and 5,642 controls of Asian and European ancestry. We observed that carriage of rare, protein-altering variants that were predicted to impair protein function at SMPD1 (a gene encoding for acid sphingomyelinase) were significantly associated with increased risk of Parkinson&#39;s disease. Refinement of variant classification using functional acid sphingomyelinase assays suggest that individuals carrying SMPD1 variants with less than 44 percent of normal enzymatic activity show the strongest association with Parkinson&#39;s disease risk in both the discovery (odds ratio (OR) = 2.37, 95% CI = 1.68 - 3.35, P = 4.35 x 10-7) and replication collections (OR = 2.18, 95% CI = 1.69 - 2.81, P = 4.80 x 10-10), leading to a significant observation when all data were meta-analyzed (OR = 2.24, 95% CI = 1.83 - 2.76, P = 1.25 x 10-15). Our findings affirm the importance of sphingomyelin metabolism in the pathobiology of neurodegenerative diseases and highlights the utility of functional genomic assays in large-scale exome sequencing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292993">
        <p class="paperTitle">Genome-wide analyses reveal widespread genetic overlap between neurological and psychiatric disorders and a convergence of biological associations related to the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC),  ; The International Headache Genetics Consortium (IHGC),  ; Steen, N. E.; van der Meer, D.; O&#39;Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.</p>
        <p class="info">Score: 30.7, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292993' target='https://doi.org/10.1101/2023.07.21.23292993'> 10.1101/2023.07.21.23292993</a></p>
        <p class="abstract">Neurological and psychiatric disorders are considered to reflect distinct underlying pathogenic entities. However, the extent to which they share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate extensive genetic overlap across the disorders, with varying degrees of genetic correlations. In particular, migraine, essential tremor, stroke and multiple sclerosis were genetically correlated with several psychiatric disorders. Biological interrogation indicated heterogenous biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study demonstrates that neurological and psychiatric disorders are not genetically disparate, but share key etiological aspects, which have important implications for disease classification, clinical practice, and genomic precision medicine.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
