<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Distinct explanations underlie gene-environment interactions in the UK Biobank
Authors: Durvasula, A.; Price, A.
Score: 23.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969
The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Distinct explanations underlie gene-environment interactions in the UK Biobank
Authors: Durvasula, A.; Price, A.
Score: 23.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969
The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Distinct explanations underlie gene-environment interactions in the UK Biobank
Authors: Durvasula, A.; Price, A.
Score: 23.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969
The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Distinct explanations underlie gene-environment interactions in the UK Biobank\nAuthors: Durvasula, A.; Price, A.\nScore: 23.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969\nThe role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) \u0026lt; 1 across E bins.",
  "keywords": [
    
  ],
  "articleBody": " Distinct explanations underlie gene-environment interactions in the UK Biobank\nAuthors: Durvasula, A.; Price, A.\nScore: 23.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969\nThe role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) \u003c 1 across E bins. Second, we detect genome-wide effects of the E variable on genetic variance by leveraging polygenic risk scores (PRS) to test for significant PRSxE in a regression of phenotypes on PRS, E, and PRSxE, together with differences in SNP-heritability across E bins. Third, we detect genome-wide proportional amplification of genetic and environmental effects as a function of the E variable by testing for significant PRSxE with no differences in SNP-heritability across E bins. Simulations show that these approaches achieve high sensitivity and specificity in distinguishing these three GxE scenarios. We applied our framework to 33 UK Biobank diseases/traits (average N=325K) and 10 E variables spanning lifestyle, diet, and other environmental exposures. First, we identified 19 trait-E pairs with rg significantly \u003c 1 (FDR\u003c5%) (average rg=0.95); for example, white blood cell count had rg=0.95 (s.e. 0.01) between smokers and non-smokers. Second, we identified 28 trait-E pairs with significant PRSxE and significant SNP-heritability differences across E bins; for example, type 2 diabetes had a significant PRSxE for alcohol consumption (P=1e-13) with 4.2x larger SNP-heritability in the largest versus smallest quintiles of alcohol consumption (P\u003c1e-16). Third, we identified 15 trait-E pairs with significant PRSxE with no SNP-heritability differences across E bins; for example, triglyceride levels had a significant PRSxE effect for composite diet score (P=4e-5) with no SNP-heritability differences. Analyses using biological sex as the E variable produced additional significant findings in each of the three scenarios. Overall, we infer a substantial contribution of GxE and GxSex effects to disease and complex trait variance.\nJoint genotypic and phenotypic outcome modeling improves base editing variant effect quantification\nAuthors: Ryu, J.; Barkal, S.; Yu, T.; Jankowiak, M.; Zhou, Y.; Francoeur, M.; Phan, Q. V.; Li, Z.; Tognon, M.; Brown, L.; Love, M. I.; Lettre, G.; Ascher, D.; Cassa, C. A.; Sherwood, R. I.; Pinello, L.\nScore: 22.0, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295253\nCRISPR base editing screens are powerful tools for studying disease-associated variants at scale. However, the efficiency and precision of base editing perturbations vary, confounding the assessment of variant-induced phenotypic effects. Here, we provide an integrated pipeline that improves the estimation of variant impact in base editing screens. We perform high-throughput ABE8e-SpRY base editing screens with an integrated reporter construct to measure the editing efficiency and outcomes of each gRNA alongside their phenotypic consequences. We introduce BEAN, a Bayesian network that accounts for per-guide editing outcomes and target site chromatin accessibility to estimate variant impacts. We show this pipeline attains superior performance compared to existing tools in variant classification and effect size quantification. We use BEAN to pinpoint common variants that alter LDL uptake, implicating novel genes. Additionally, through saturation base editing of LDLR, we enable accurate quantitative prediction of the effects of missense variants on LDL-C levels, which aligns with measurements in UK Biobank individuals, and identify structural mechanisms underlying variant pathogenicity. This work provides a widely applicable approach to improve the power of base editor screens for disease-associated variant characterization.\nMulti-ancestry Polygenic Mechanisms of Type 2 Diabetes Elucidate Disease Processes and Clinical Heterogeneity\nAuthors: Smith, K.; Deutsch, A.; McGrail, C.; Kim, H.; Hsu, S.; Mandla, R.; Schroeder, P. H.; Westerman, K. E.; Szczerbinski, L.; Majarian, T. D.; Kaur, V.; Williamson, A.; Claussnitzer, M.; Florez, J. C.; Manning, A. K.; Mercader, J. M.; Gaulton, K. J.; Udler, M. S.\nScore: 14.3, Published: 2023-09-29 DOI: 10.1101/2023.09.28.23296294\nWe identified genetic subtypes of type 2 diabetes (T2D) by analyzing genetic data from diverse groups, including non-European populations. We implemented soft clustering with 650 T2D-associated genetic variants, capturing known and novel T2D subtypes with distinct cardiometabolic trait associations. The twelve genetic clusters were distinctively enriched for single-cell regulatory regions. Polygenic scores derived from the clusters differed in distribution between ancestry groups, including a significantly higher proportion of lipodystrophy-related polygenic risk in East Asian ancestry. T2D risk was equivalent at a BMI of 30 kg/m2 in the European subpopulation and 24.2 (22.9-25.5) kg/m2 in the East Asian subpopulation; after adjusting for cluster-specific genetic risk, the equivalent BMI threshold increased to 28.5 (27.1-30.0) kg/m2 in the East Asian group, explaining about 75% of the difference in BMI thresholds. Thus, these multi-ancestry T2D genetic subtypes encompass a broader range of biological mechanisms and help explain ancestry-associated differences in T2D risk profiles.\nGenome-wide association analysis reveals insights into the molecular aetiology underlying dilated cardiomyopathy\nAuthors: Zheng, S. L.; Henry, A.; Cannie, D.; Lee, M.; Miller, D.; McGurk, K. A.; Bond, I.; Xu, X.; Issa, H.; Francis, C.; De Marvao, A.; Theotokis, P. I.; Buchan, R. J.; Speed, D.; Abner, E.; Adams, L.; Aragam, K. G.; Arnlov, J.; Axelsson Raja, A.; Backman, J. D.; Baksi, J.; Barton, P. J.; Biddinger, K. J.; Boersma, E.; Brandimarto, J.; Brunak, S.; Bundgaard, H.; Carey, D. J.; Charron, P.; Cook, J. P.; Cook, S. A.; Denaxas, S.; Deleuze, J.-F.; Doney, A. S.; Elliot, P.; Erikstrup, C.; Esko, T.; Farber-Eger, E. H.; Finan, C.; Garnier, S.; Ghouse, J.; Giedraitis, V.; Gudbjartsson, D. F.; Haggerty, C. M.;\nScore: 11.7, Published: 2023-09-29 DOI: 10.1101/2023.09.28.23295408\nDilated cardiomyopathy (DCM) is a primary heart muscle disorder that has been associated with both monogenic and polygenic architectures, with the majority of cases having no molecular genetic diagnosis. To improve our understanding of the genetic basis of DCM, we perform a genome-wide association study (GWAS) meta-analysis comprising 14,255 DCM cases and 1,199,156 controls, and a multi-trait GWAS incorporating correlated cardiac magnetic resonance imaging traits of 36,203 participants. We identify 80 genetic susceptibility loci and prioritize 61 putative effector genes for DCM by synthesizing evidence from 8 gene prioritization strategies. Rare variant association testing identifies genes associated with DCM, including MAP3K7, NEDD4L, and SSPN. Through integration with single-nuclei transcriptomics from 52 end-stage DCM patients and 18 controls, we identify cellular states, biological pathways, and intercellular communications driving DCM pathogenesis. Finally, we demonstrate that a polygenic score predicts DCM in the general population and modulates the penetrance of rare pathogenic and likely pathogenic variants in DCM-causing genes.\nSystematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease\nAuthors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D'Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.\nScore: 40.2, Published: 2023-09-12 DOI: 10.1101/2023.09.12.23295416\nBackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown. MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls. ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations. ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.\nIncorporating Nanopore Sequencing into a Diverse Diagnostic Toolkit for Incontinentia Pigmenti\nAuthors: Ahting, S.; Popp, D.; Oppermann, H.; Strehlow, V.; Fasshauer, M.; Popp, B.; Karnstedt, M.; Schumann, I.\nScore: 7.0, Published: 2023-09-26 DOI: 10.1101/2023.09.26.23295778\nBackgroundIncontinentia pigmenti (IP) is a rare, hereditary multisystemic disorder affecting 1.2 in 100,000 live births, predominantly females. Conventional genetic analyses through short-read sequencing are complicated in case of IP due to the presence of a highly homologous pseudogene. Traditionally, long-range PCR is employed in order to overcome this challenge, however, detection of skewed X-Inactivation can also aid in correctly assign a variant to IKBKG. MethodsWe employed a comprehensive multi-method approach, incorporating whole-exome sequencing (WES), long-range PCR, RT-PCR, X-inactivation analysis, and nanopore sequencing of genomic DNA, to identify and accurately phase a small heterozygous deletion NM_001099857.5:c.363_367del, p.(Leu122Glyfs*14) to the IKBKG gene in a family affected with IP. ResultsThe deletion was initially detected through WES and skewed X-inactivation was observed in both the proband and her mother. Long-range PCR specific to IKBKG was utilized to verify that the variant is located in IKBKG and not in its highly homologous pseudogene. On RNA level, the variant was undetectable, suggesting nonsense-mediated decay (NMD) of the transcript containing the variant. We further utilized nanopore sequencing not only to pinpoint and accurately map the variant to the IKBKG gene but also to analyze methylation status of both alleles. This allowed us to confirm the skewed X-inactivation, with the variant-carrying allele found to be predominantly inactivated. ConclusionNanopore sequencing serves as a valuable tool in genetic diagnosis, enabling the precise localization of the variant in either the gene or the pseudogene. Furthermore, in females with skewed X-inactivation, this method facilitates the determination of whether the variant is predominantly located on the activated or inactivated X-chromosome.\nGenetics of NLRP3 suggests lack of involvement and inefficient druggability in Parkinson's disease\nAuthors: Senkevich, K.; Liu, L.; Alvarado, C. X.; Leonard, H. L.; Nalls, M. A.; Gan-Or, Z.\nScore: 6.4, Published: 2023-09-23 DOI: 10.1101/2023.09.20.23295790\nActivation of the NLRP3-inflammasome has been proposed to play a role in Parkinsons disease pathogenesis based on in vitro and in vivo studies. Currently, clinical trials targeting the NLRP3 pathway in Parkinsons disease are at early stages. However, the evidence supporting NLRP3s involvement in Parkinsons disease from human genetics data remains limited. In this study, we conducted comprehensive analyses of common and rare variants in genes related to the NLRP3-inflammasome in large Parkinsons disease cohorts. Furthermore, we performed pathway-specific analyses using polygenic risk scores and studied potential causal associations using Mendelian randomization with the NLRP3 components and the cytokines released by its activation, IL-1{beta} and IL-18. Our findings showed no associations of common or rare variants, nor of the pathway polygenic risk score for the NLRP3 inflammasome, with risk of Parkinsons disease. Mendelian randomization analyses suggest that altering the expression of the NLRP3 inflammasome, IL-1{beta} or IL-18, is not likely to affect Parkinsons disease risk, age-at-onset, or progression. Therefore, our results do not support an important role for the NLRP3 inflammasome in Parkinsons disease pathogenesis or as a strong target for drug development.\nThe shared genetic risk architecture of neurological and psychiatric disorders: a genome-wide analysis\nAuthors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC), ; The International Headache Genetics Consortium (IHGC), ; Steen, N. E.; van der Meer, D.; O'Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.\nScore: 34.2, Published: 2023-09-26 DOI: 10.1101/2023.07.21.23292993\nWhile neurological and psychiatric disorders have historically been considered to reflect distinct pathogenic entities, recent findings suggest shared pathobiological mechanisms. However, the extent to which these heritable disorders share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate widespread genetic overlap across the disorders, even in the absence of genetic correlations. This indicates that a large set of common variants impact risk of multiple neurological and psychiatric disorders, but with divergent effect sizes. Furthermore, biological interrogation revealed a range of biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study indicates that neurological and psychiatric disorders share key etiological aspects, which has important implications for disease classification, precision medicine, and clinical practice.\nHousehold income does not affect the pleiotropy of schizophrenia genetic liability with mental and physical health outcomes\nAuthors: Kouakou, M.; Cabrera Mendoza, B.; Pathak, G. A.; Cannon, T.; Polimanti, R.\nScore: 3.2, Published: 2023-09-26 DOI: 10.1101/2023.09.25.23296085\nBackground and HypothesisIndividuals with schizophrenia (SCZ) suffer from comorbidities that substantially reduce their life expectancy. Socioeconomic inequalities could contribute to many of the negative health outcomes associated with SCZ. Study DesignWe investigated genome-wide datasets related to SCZ (52,017 cases and 75,889 controls) from the Psychiatric Genomics Consortium, household income (HI; N=361,687) from UK Biobank, and 2,202 medical endpoints assessed in up to 342,499 FinnGen participants. A phenome-wide genetic correlation analysis of SCZ and HI was performed, also assessing whether SCZ genetic correlations were influenced by HI effect on SCZ. Additionally, SCZ and HI direct effects on medical endpoints were estimated using multivariable Mendelian randomization (MR). Study ResultsSCZ and HI showed overlapping genetic correlations with 70 traits (p\u003c2.89x10-5), including mental health, substance use, gastrointestinal illnesses, reproductive outcomes, liver diseases, respiratory problems, and musculoskeletal phenotypes. SCZ genetic correlations with these traits were not affected by HI effect on SCZ. Considering Bonferroni multiple testing correction (p\u003c7.14x10-4), MR analysis indicated that SCZ and HI may affect medical abortion (SCZ odds ratio, OR=1.07; HI OR=0.78), panic disorder (SCZ OR=1.20; HI OR=0.60), personality disorders (SCZ OR=1.31; HI OR=0.67), substance use (SCZ OR=1.2; HI OR=0.68), and adjustment disorders (SCZ OR=1.18; HI OR=0.78). Multivariable MR analysis confirmed that SCZ effects on these outcomes were independent of HI. ConclusionsThe effect of SCZ genetic liability on mental and physical health may not be strongly affected by socioeconomic differences. This suggests that SCZ-specific strategies are needed to reduce negative health outcomes affecting patients and high-risk individuals.\nLandscape of blood group antigens and alleles in the Indian population from whole genome sequences\nAuthors: Rophina, M.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.\nScore: 2.6, Published: 2023-09-26 DOI: 10.1101/2023.09.26.23296145\nBlood group antigens are genetically inherited macromolecular structures which form the underlying factor for inter individual variations in human blood. Currently there exists over 390 human blood group antigens corresponding to 44 blood group systems and 2 erythroid specific transcription factors. Distribution of these blood group antigens have been found to differ significantly among various ethnic populations. To date, there is a lack of comprehensive research that offers extensive blood group profiles for the Indian population. Whole genome sequence data (hg38) of 1029 self-declared healthy Indian individuals generated as a part of the pilot phase IndiGen programme were used for the analysis. Variants spanning the genes of 44 blood group systems and two transcription factors KLF1, GATA1 were fetched and annotated for their functional consequences. Our study reports a total of 40712 blood group related variants of which 695 were identified as non-synonymous variants in the coding region. Of the total non-synonymous variants, 105 were found to have a known blood phenotype. A total of 24 variants belonging to 12 blood groups were predicted to be deleterious by more than three computational tools. Our study was also able to identify a few rare blood phenotypes including Au(a-b+), Js(a+b+), Di(a+b-), In(a+b-) and KANNO-. This study is the first to use genomic data to understand the blood group antigen profiles of the Indian population, and it also systematically compares these profiles with those of other global populations. Key pointsO_LIAccurate characterization of the genomic landscape of known and rare blood group alleles and antigens in the Indian population using the whole genome sequencing data of 1029 self-declared healthy individuals C_LIO_LIUnderstanding the distinct similarities and differences in blood group genotypes and phenotypes across diverse global populations through systematic comparison of genomic datasets. C_LI Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=151 SRC=\"FIGDIR/small/23296145v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@1e76bbaorg.highwire.dtl.DTLVardef@51a537org.highwire.dtl.DTLVardef@35d395org.highwire.dtl.DTLVardef@1d6076a_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2727",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.23295969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.23295969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.23295969">
        <p class="paperTitle">Distinct explanations underlie gene-environment interactions in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.23295969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.23295969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durvasula, A.; Price, A.</p>
        <p class="info">Score: 23.2, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.23295969' target='https://doi.org/10.1101/2023.09.22.23295969'> 10.1101/2023.09.22.23295969</a></p>
        <p class="abstract">The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins. Second, we detect genome-wide effects of the E variable on genetic variance by leveraging polygenic risk scores (PRS) to test for significant PRSxE in a regression of phenotypes on PRS, E, and PRSxE, together with differences in SNP-heritability across E bins. Third, we detect genome-wide proportional amplification of genetic and environmental effects as a function of the E variable by testing for significant PRSxE with no differences in SNP-heritability across E bins. Simulations show that these approaches achieve high sensitivity and specificity in distinguishing these three GxE scenarios. We applied our framework to 33 UK Biobank diseases/traits (average N=325K) and 10 E variables spanning lifestyle, diet, and other environmental exposures. First, we identified 19 trait-E pairs with rg significantly &lt; 1 (FDR&lt;5%) (average rg=0.95); for example, white blood cell count had rg=0.95 (s.e. 0.01) between smokers and non-smokers. Second, we identified 28 trait-E pairs with significant PRSxE and significant SNP-heritability differences across E bins; for example, type 2 diabetes had a significant PRSxE for alcohol consumption (P=1e-13) with 4.2x larger SNP-heritability in the largest versus smallest quintiles of alcohol consumption (P&lt;1e-16). Third, we identified 15 trait-E pairs with significant PRSxE with no SNP-heritability differences across E bins; for example, triglyceride levels had a significant PRSxE effect for composite diet score (P=4e-5) with no SNP-heritability differences. Analyses using biological sex as the E variable produced additional significant findings in each of the three scenarios. Overall, we infer a substantial contribution of GxE and GxSex effects to disease and complex trait variance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295253">
        <p class="paperTitle">Joint genotypic and phenotypic outcome modeling improves base editing variant effect quantification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ryu, J.; Barkal, S.; Yu, T.; Jankowiak, M.; Zhou, Y.; Francoeur, M.; Phan, Q. V.; Li, Z.; Tognon, M.; Brown, L.; Love, M. I.; Lettre, G.; Ascher, D.; Cassa, C. A.; Sherwood, R. I.; Pinello, L.</p>
        <p class="info">Score: 22.0, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295253' target='https://doi.org/10.1101/2023.09.08.23295253'> 10.1101/2023.09.08.23295253</a></p>
        <p class="abstract">CRISPR base editing screens are powerful tools for studying disease-associated variants at scale. However, the efficiency and precision of base editing perturbations vary, confounding the assessment of variant-induced phenotypic effects. Here, we provide an integrated pipeline that improves the estimation of variant impact in base editing screens. We perform high-throughput ABE8e-SpRY base editing screens with an integrated reporter construct to measure the editing efficiency and outcomes of each gRNA alongside their phenotypic consequences. We introduce BEAN, a Bayesian network that accounts for per-guide editing outcomes and target site chromatin accessibility to estimate variant impacts. We show this pipeline attains superior performance compared to existing tools in variant classification and effect size quantification. We use BEAN to pinpoint common variants that alter LDL uptake, implicating novel genes. Additionally, through saturation base editing of LDLR, we enable accurate quantitative prediction of the effects of missense variants on LDL-C levels, which aligns with measurements in UK Biobank individuals, and identify structural mechanisms underlying variant pathogenicity. This work provides a widely applicable approach to improve the power of base editor screens for disease-associated variant characterization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.23296294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.23296294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.23296294">
        <p class="paperTitle">Multi-ancestry Polygenic Mechanisms of Type 2 Diabetes Elucidate Disease Processes and Clinical Heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.23296294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.23296294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, K.; Deutsch, A.; McGrail, C.; Kim, H.; Hsu, S.; Mandla, R.; Schroeder, P. H.; Westerman, K. E.; Szczerbinski, L.; Majarian, T. D.; Kaur, V.; Williamson, A.; Claussnitzer, M.; Florez, J. C.; Manning, A. K.; Mercader, J. M.; Gaulton, K. J.; Udler, M. S.</p>
        <p class="info">Score: 14.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.23296294' target='https://doi.org/10.1101/2023.09.28.23296294'> 10.1101/2023.09.28.23296294</a></p>
        <p class="abstract">We identified genetic subtypes of type 2 diabetes (T2D) by analyzing genetic data from diverse groups, including non-European populations. We implemented soft clustering with 650 T2D-associated genetic variants, capturing known and novel T2D subtypes with distinct cardiometabolic trait associations. The twelve genetic clusters were distinctively enriched for single-cell regulatory regions. Polygenic scores derived from the clusters differed in distribution between ancestry groups, including a significantly higher proportion of lipodystrophy-related polygenic risk in East Asian ancestry. T2D risk was equivalent at a BMI of 30 kg/m2 in the European subpopulation and 24.2 (22.9-25.5) kg/m2 in the East Asian subpopulation; after adjusting for cluster-specific genetic risk, the equivalent BMI threshold increased to 28.5 (27.1-30.0) kg/m2 in the East Asian group, explaining about 75% of the difference in BMI thresholds. Thus, these multi-ancestry T2D genetic subtypes encompass a broader range of biological mechanisms and help explain ancestry-associated differences in T2D risk profiles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.23295408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.23295408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.23295408">
        <p class="paperTitle">Genome-wide association analysis reveals insights into the molecular aetiology underlying dilated cardiomyopathy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.23295408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.23295408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, S. L.; Henry, A.; Cannie, D.; Lee, M.; Miller, D.; McGurk, K. A.; Bond, I.; Xu, X.; Issa, H.; Francis, C.; De Marvao, A.; Theotokis, P. I.; Buchan, R. J.; Speed, D.; Abner, E.; Adams, L.; Aragam, K. G.; Arnlov, J.; Axelsson Raja, A.; Backman, J. D.; Baksi, J.; Barton, P. J.; Biddinger, K. J.; Boersma, E.; Brandimarto, J.; Brunak, S.; Bundgaard, H.; Carey, D. J.; Charron, P.; Cook, J. P.; Cook, S. A.; Denaxas, S.; Deleuze, J.-F.; Doney, A. S.; Elliot, P.; Erikstrup, C.; Esko, T.; Farber-Eger, E. H.; Finan, C.; Garnier, S.; Ghouse, J.; Giedraitis, V.; Gudbjartsson, D. F.; Haggerty, C. M.;</p>
        <p class="info">Score: 11.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.23295408' target='https://doi.org/10.1101/2023.09.28.23295408'> 10.1101/2023.09.28.23295408</a></p>
        <p class="abstract">Dilated cardiomyopathy (DCM) is a primary heart muscle disorder that has been associated with both monogenic and polygenic architectures, with the majority of cases having no molecular genetic diagnosis. To improve our understanding of the genetic basis of DCM, we perform a genome-wide association study (GWAS) meta-analysis comprising 14,255 DCM cases and 1,199,156 controls, and a multi-trait GWAS incorporating correlated cardiac magnetic resonance imaging traits of 36,203 participants. We identify 80 genetic susceptibility loci and prioritize 61 putative effector genes for DCM by synthesizing evidence from 8 gene prioritization strategies. Rare variant association testing identifies genes associated with DCM, including MAP3K7, NEDD4L, and SSPN. Through integration with single-nuclei transcriptomics from 52 end-stage DCM patients and 18 controls, we identify cellular states, biological pathways, and intercellular communications driving DCM pathogenesis. Finally, we demonstrate that a polygenic score predicts DCM in the general population and modulates the penetrance of rare pathogenic and likely pathogenic variants in DCM-causing genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295416">
        <p class="paperTitle">Systematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D&#39;Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.</p>
        <p class="info">Score: 40.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295416' target='https://doi.org/10.1101/2023.09.12.23295416'> 10.1101/2023.09.12.23295416</a></p>
        <p class="abstract">BackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown.

MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls.

ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations.

ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23295778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23295778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23295778">
        <p class="paperTitle">Incorporating Nanopore Sequencing into a Diverse Diagnostic Toolkit for Incontinentia Pigmenti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23295778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23295778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahting, S.; Popp, D.; Oppermann, H.; Strehlow, V.; Fasshauer, M.; Popp, B.; Karnstedt, M.; Schumann, I.</p>
        <p class="info">Score: 7.0, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23295778' target='https://doi.org/10.1101/2023.09.26.23295778'> 10.1101/2023.09.26.23295778</a></p>
        <p class="abstract">BackgroundIncontinentia pigmenti (IP) is a rare, hereditary multisystemic disorder affecting 1.2 in 100,000 live births, predominantly females. Conventional genetic analyses through short-read sequencing are complicated in case of IP due to the presence of a highly homologous pseudogene. Traditionally, long-range PCR is employed in order to overcome this challenge, however, detection of skewed X-Inactivation can also aid in correctly assign a variant to IKBKG.

MethodsWe employed a comprehensive multi-method approach, incorporating whole-exome sequencing (WES), long-range PCR, RT-PCR, X-inactivation analysis, and nanopore sequencing of genomic DNA, to identify and accurately phase a small heterozygous deletion NM_001099857.5:c.363_367del, p.(Leu122Glyfs*14) to the IKBKG gene in a family affected with IP.

ResultsThe deletion was initially detected through WES and skewed X-inactivation was observed in both the proband and her mother. Long-range PCR specific to IKBKG was utilized to verify that the variant is located in IKBKG and not in its highly homologous pseudogene. On RNA level, the variant was undetectable, suggesting nonsense-mediated decay (NMD) of the transcript containing the variant. We further utilized nanopore sequencing not only to pinpoint and accurately map the variant to the IKBKG gene but also to analyze methylation status of both alleles. This allowed us to confirm the skewed X-inactivation, with the variant-carrying allele found to be predominantly inactivated.

ConclusionNanopore sequencing serves as a valuable tool in genetic diagnosis, enabling the precise localization of the variant in either the gene or the pseudogene. Furthermore, in females with skewed X-inactivation, this method facilitates the determination of whether the variant is predominantly located on the activated or inactivated X-chromosome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.23295790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.23295790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.23295790">
        <p class="paperTitle">Genetics of NLRP3 suggests lack of involvement and inefficient druggability in Parkinson&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.23295790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.23295790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Senkevich, K.; Liu, L.; Alvarado, C. X.; Leonard, H. L.; Nalls, M. A.; Gan-Or, Z.</p>
        <p class="info">Score: 6.4, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.23295790' target='https://doi.org/10.1101/2023.09.20.23295790'> 10.1101/2023.09.20.23295790</a></p>
        <p class="abstract">Activation of the NLRP3-inflammasome has been proposed to play a role in Parkinsons disease pathogenesis based on in vitro and in vivo studies. Currently, clinical trials targeting the NLRP3 pathway in Parkinsons disease are at early stages. However, the evidence supporting NLRP3s involvement in Parkinsons disease from human genetics data remains limited. In this study, we conducted comprehensive analyses of common and rare variants in genes related to the NLRP3-inflammasome in large Parkinsons disease cohorts. Furthermore, we performed pathway-specific analyses using polygenic risk scores and studied potential causal associations using Mendelian randomization with the NLRP3 components and the cytokines released by its activation, IL-1{beta} and IL-18. Our findings showed no associations of common or rare variants, nor of the pathway polygenic risk score for the NLRP3 inflammasome, with risk of Parkinsons disease. Mendelian randomization analyses suggest that altering the expression of the NLRP3 inflammasome, IL-1{beta} or IL-18, is not likely to affect Parkinsons disease risk, age-at-onset, or progression. Therefore, our results do not support an important role for the NLRP3 inflammasome in Parkinsons disease pathogenesis or as a strong target for drug development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292993">
        <p class="paperTitle">The shared genetic risk architecture of neurological and psychiatric disorders: a genome-wide analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC),  ; The International Headache Genetics Consortium (IHGC),  ; Steen, N. E.; van der Meer, D.; O&#39;Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.</p>
        <p class="info">Score: 34.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292993' target='https://doi.org/10.1101/2023.07.21.23292993'> 10.1101/2023.07.21.23292993</a></p>
        <p class="abstract">While neurological and psychiatric disorders have historically been considered to reflect distinct pathogenic entities, recent findings suggest shared pathobiological mechanisms. However, the extent to which these heritable disorders share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate widespread genetic overlap across the disorders, even in the absence of genetic correlations. This indicates that a large set of common variants impact risk of multiple neurological and psychiatric disorders, but with divergent effect sizes. Furthermore, biological interrogation revealed a range of biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study indicates that neurological and psychiatric disorders share key etiological aspects, which has important implications for disease classification, precision medicine, and clinical practice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.23296085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.23296085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.23296085">
        <p class="paperTitle">Household income does not affect the pleiotropy of schizophrenia genetic liability with mental and physical health outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.23296085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.23296085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kouakou, M.; Cabrera Mendoza, B.; Pathak, G. A.; Cannon, T.; Polimanti, R.</p>
        <p class="info">Score: 3.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.23296085' target='https://doi.org/10.1101/2023.09.25.23296085'> 10.1101/2023.09.25.23296085</a></p>
        <p class="abstract">Background and HypothesisIndividuals with schizophrenia (SCZ) suffer from comorbidities that substantially reduce their life expectancy. Socioeconomic inequalities could contribute to many of the negative health outcomes associated with SCZ.

Study DesignWe investigated genome-wide datasets related to SCZ (52,017 cases and 75,889 controls) from the Psychiatric Genomics Consortium, household income (HI; N=361,687) from UK Biobank, and 2,202 medical endpoints assessed in up to 342,499 FinnGen participants. A phenome-wide genetic correlation analysis of SCZ and HI was performed, also assessing whether SCZ genetic correlations were influenced by HI effect on SCZ. Additionally, SCZ and HI direct effects on medical endpoints were estimated using multivariable Mendelian randomization (MR).

Study ResultsSCZ and HI showed overlapping genetic correlations with 70 traits (p&lt;2.89x10-5), including mental health, substance use, gastrointestinal illnesses, reproductive outcomes, liver diseases, respiratory problems, and musculoskeletal phenotypes. SCZ genetic correlations with these traits were not affected by HI effect on SCZ. Considering Bonferroni multiple testing correction (p&lt;7.14x10-4), MR analysis indicated that SCZ and HI may affect medical abortion (SCZ odds ratio, OR=1.07; HI OR=0.78), panic disorder (SCZ OR=1.20; HI OR=0.60), personality disorders (SCZ OR=1.31; HI OR=0.67), substance use (SCZ OR=1.2; HI OR=0.68), and adjustment disorders (SCZ OR=1.18; HI OR=0.78). Multivariable MR analysis confirmed that SCZ effects on these outcomes were independent of HI.

ConclusionsThe effect of SCZ genetic liability on mental and physical health may not be strongly affected by socioeconomic differences. This suggests that SCZ-specific strategies are needed to reduce negative health outcomes affecting patients and high-risk individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23296145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23296145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23296145">
        <p class="paperTitle">Landscape of blood group antigens and alleles in the Indian population from whole genome sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23296145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23296145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rophina, M.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.</p>
        <p class="info">Score: 2.6, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23296145' target='https://doi.org/10.1101/2023.09.26.23296145'> 10.1101/2023.09.26.23296145</a></p>
        <p class="abstract">Blood group antigens are genetically inherited macromolecular structures which form the underlying factor for inter individual variations in human blood. Currently there exists over 390 human blood group antigens corresponding to 44 blood group systems and 2 erythroid specific transcription factors. Distribution of these blood group antigens have been found to differ significantly among various ethnic populations. To date, there is a lack of comprehensive research that offers extensive blood group profiles for the Indian population. Whole genome sequence data (hg38) of 1029 self-declared healthy Indian individuals generated as a part of the pilot phase IndiGen programme were used for the analysis. Variants spanning the genes of 44 blood group systems and two transcription factors KLF1, GATA1 were fetched and annotated for their functional consequences. Our study reports a total of 40712 blood group related variants of which 695 were identified as non-synonymous variants in the coding region. Of the total non-synonymous variants, 105 were found to have a known blood phenotype. A total of 24 variants belonging to 12 blood groups were predicted to be deleterious by more than three computational tools. Our study was also able to identify a few rare blood phenotypes including Au(a-b&#43;), Js(a&#43;b&#43;), Di(a&#43;b-), In(a&#43;b-) and KANNO-. This study is the first to use genomic data to understand the blood group antigen profiles of the Indian population, and it also systematically compares these profiles with those of other global populations.

Key pointsO_LIAccurate characterization of the genomic landscape of known and rare blood group alleles and antigens in the Indian population using the whole genome sequencing data of 1029 self-declared healthy individuals
C_LIO_LIUnderstanding the distinct similarities and differences in blood group genotypes and phenotypes across diverse global populations through systematic comparison of genomic datasets.
C_LI

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=151 SRC=&#34;FIGDIR/small/23296145v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@1e76bbaorg.highwire.dtl.DTLVardef@51a537org.highwire.dtl.DTLVardef@35d395org.highwire.dtl.DTLVardef@1d6076a_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
