<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA
Authors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O&#39;Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA
Authors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O&#39;Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA
Authors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O&#39;Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA\nAuthors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O\u0026#39;Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B.",
  "keywords": [
    
  ],
  "articleBody": " Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA\nAuthors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O'Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B.; Pengam, A.; Cogne, B.; MacArthur, D. G.; Ulitsky, I.; Carvill, G. L.; O'Donnell-Luria, A.\nScore: 95.1, Published: 2024-02-07 DOI: 10.1101/2024.01.31.24301497\nAbstract/SummaryGenes encoding long non-coding RNAs (lncRNAs) comprise a large fraction of the human genome, yet haploinsufficiency of a lncRNA has not been shown to cause a Mendelian disease. CHASERR is a highly conserved human lncRNA adjacent to CHD2-a coding gene in which de novo loss-of-function variants cause developmental and epileptic encephalopathy. Here we report three unrelated individuals each harboring an ultra-rare heterozygous de novo deletion in the CHASERR locus. We report similarities in severe developmental delay, facial dysmorphisms, and cerebral dysmyelination in these individuals, distinguishing them from the phenotypic spectrum of CHD2 haploinsufficiency. We demonstrate reduced CHASERR mRNA expression and corresponding increased CHD2 mRNA and protein in whole blood and patient-derived cell lines-specifically increased expression of the CHD2 allele in cis with the CHASERR deletion, as predicted from a prior mouse model of Chaserr haploinsufficiency. We show for the first time that de novo structural variants facilitated by Alu-mediated non-allelic homologous recombination led to deletion of a non-coding element (the lncRNA CHASERR) to cause a rare syndromic neurodevelopmental disorder. We also demonstrate that CHD2 has bidirectional dosage sensitivity in human disease. This work highlights the need to carefully evaluate other lncRNAs, particularly those upstream of genes associated with Mendelian disorders.\nWhole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy\nAuthors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Stamelou, M.; Arzberger, T.; Roeber, S.; Muller, U.; Hopfner, F.; Hopfner, F.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L\nScore: 13.2, Published: 2024-01-30 DOI: 10.1101/2023.12.28.23300612\nBackgroundProgressive supranuclear palsy (PSP) is a rare neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. Previous genome-wide association studies for PSP were based on genotype array, therefore, were inadequate for the analysis of rare variants as well as larger mutations, such as small insertions/deletions (indels) and structural variants (SVs). MethodIn this study, we performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), indels, and SVs, in a cohort of 1,718 cases and 2,944 controls of European ancestry. Of the 1,718 PSP individuals, 1,441 were autopsy-confirmed and 277 were clinically diagnosed. ResultsOur analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and further uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. Notably, in contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele in PSP. Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. Moreover, seven common SVs associated with PSP were observed in the H1/H2 haplotype region (17q21.31) and other loci, including IGH, PCMT1, CYP2A13, and SMCP. In the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP. ConclusionsThrough WGS, we significantly enhanced our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.\nGenome-wide association analysis of composite sleep health scores in 413,904 individuals\nAuthors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.; Wang, H.\nScore: 5.2, Published: 2024-02-03 DOI: 10.1101/2024.02.02.24302211\nRecent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, and together may provide a more complete picture of sleep health, while also illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p\u003c8.3e-9), along with 341 previously reported loci (p\u003c5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2=0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections to behavioral, psychological, and cardiometabolic traits.\nPrognostic Value of Multiplexed Assays of Variant Effect and Automated Patch-clamping for KCNH2-LQTS Risk Stratification\nAuthors: O'Neill, M. J.; Ng, C. A.; Aizawa, T.; Sala, L.; Bains, S.; Denjoy, I.; Winbo, A.; Ullah, R.; Shen, Q.; Tan, C.-Y.; Kozek, K.; Vanags, L.; Mitchell, D.; Shen, A.; Wada, Y.; Kashiwa, A.; Crotti, L.; Dagradi, F.; Musu, G.; Neves, R.; Bos, J. M.; Giudicessi, J.; Bledsoe, X.; Lancaster, M. C.; Glazer, A.; Roden, D. M.; Leenhardt, A.; Salem, J.-E.; Earle, N.; Stiles, R.; Agee, T.; Johnson, C.; Horie, M.; Skinner, J.; Extramiana, F.; Ackerman, M. J.; Schwartz, P.; Ohno, S.; Vandenberg, J.; Kroncke, B.\nScore: 5.1, Published: 2024-02-05 DOI: 10.1101/2024.02.01.24301443\nBackgroundLong QT syndrome (LQTS) is a lethal arrhythmia condition, frequently caused by rare loss-of-function variants in the cardiac potassium channel encoded by KCNH2. Variant-based risk stratification is complicated by heterogenous clinical data, incomplete penetrance, and low-throughput functional data. ObjectiveTo test the utility of variant-specific features, including high-throughput functional data, to predict cardiac events among KCNH2 variant heterozygotes. MethodsWe quantified cell-surface trafficking of 18,323 variants in KCNH2 and recorded potassium current densities for 506 KCNH2 variants. Next, we deeply phenotyped 1150 KCNH2 missense variant patients, including ECG features, cardiac event history (528 total cardiac events), and mortality. We then assessed variant functional, in silico, structural, and LQTS penetrance data to stratify event-free survival for cardiac events in the study cohort. ResultsVariant-specific current density (HR 0.28 [0.13-0.60]) and estimates of LQTS penetrance incorporating MAVE data (HR 3.16 [1.59-6.27]) were independently predictive of severe cardiac events when controlling for patient-specific features. Risk prediction models incorporating these data significantly improved prediction of 20 year cardiac events (AUC 0.79 [0.75-0.82]) over patient-only covariates (QTc and sex) (AUC 0.73 [0.70-0.77]). ConclusionWe show that high-throughput functional data, and other variant-specific features, meaningfully contribute to both diagnosis and prognosis of a clinically actionable monogenic disease. Graphical AbstractImplementation of KCNH2 variant functional studies, deep clinical phenotyping, and cardiac event risk stratification. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=127 SRC=\"FIGDIR/small/24301443v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (53K): org.highwire.dtl.DTLVardef@19961e3org.highwire.dtl.DTLVardef@e99eb3org.highwire.dtl.DTLVardef@1b232a8org.highwire.dtl.DTLVardef@1bca01_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPRESCOTT: a population aware, epistatic and structural model accurately predicts missense effect\nAuthors: Tekpinar, M.; David, L.; Henry, T.; Carbone, A.\nScore: 4.5, Published: 2024-02-06 DOI: 10.1101/2024.02.03.24302219\nPredicting the functional impact of point mutations is a complex yet vital task in genomics. PRESCOTT stands at the forefront of this challenge and reconstructs complete mutational landscapes of proteins, enables the identification of protein regions most vulnerable to mutations and assigns scores to individual mutations, assisting pathologists in evaluating the pathogenic potential of missense variants. PRESCOTT categorizes these variants into three distinct classes: Benign, Pathogenic, or Variants of Uncertain Significance (VUS). The model leverages protein sequences across millions of species, advanced protein structural models, and extensive genomic and exomic data from diverse human populations. By using only sequence and structural information, it significantly improves on current standards for predicting mutations in human proteins and matches AlphaMissense performance, which incorporates allele frequency data in its analysis. By including population-specific allele frequencies, PRESCOTT excels in genome-scale score separation of ClinVar benign and pathogenic variants and surpasses AlphaMissense in analyzing the ACMG reference human dataset and the over 1800 proteins from the Human Protein Dataset. Its efficacy is particularly notable in autoinflammatory diseases, accurately predicting pathogenic gain-of-function missense mutations, a task known for its difficulty. Efficiency and accessibility are key aspects of PRESCOTT. The user-friendly PRESCOTT webserver facilitates mutation effect calculations on any protein and protein variants. The server hosts a Comprehensive Human Protein Database for over 19,000 human proteins, based on sequences and structures, ready for a customized allele population analysis. Additionally, the tool provides open access to all intermediate scores, ensuring interpretability and transparency in variant analysis. PRESCOTT is a significant stride forward in the field of genomic medicine, offering unparalleled insights into protein mutational impacts.\nProteome-Wide Assessment of Clustering of Missense Variants in Neurodevelopmental Disorders Versus Cancer\nAuthors: Ng, J. K.; Chen, Y.; Akinwe, T. M.; Heins, H. B.; Mehinovic, E.; Chang, Y.; Payne, Z. L.; Manuel, J. G.; Karchin, R.; Turner, T. N.\nScore: 3.0, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302238\nMissense de novo variants (DNVs) and missense somatic variants contribute to neurodevelopmental disorders (NDDs) and cancer, respectively. Proteins with statistical enrichment based on analyses of these variants exhibit convergence in the differing NDD and cancer phenotypes. Herein, the question of why some of the same proteins are identified in both phenotypes is examined through investigation of clustering of missense variation at the protein level. Our hypothesis is that missense variation is present in different protein locations in the two phenotypes leading to the distinct phenotypic outcomes. We tested this hypothesis in 1D protein space using our software CLUMP. Furthermore, we newly developed 3D-CLUMP that uses 3D protein structures to spatially test clustering of missense variation for proteome-wide significance. We examined missense DNVs in 39,883 parent-child sequenced trios with NDDs and missense somatic variants from 10,543 sequenced tumors covering five TCGA cancer types and two COSMIC pan-cancer aggregates of tissue types. There were 57 proteins with proteome-wide significant missense variation clustering in NDDs when compared to cancers and 79 proteins with proteome-wide significant missense clustering in cancers compared to NDDs. While our main objective was to identify differences in patterns of missense variation, we also identified a novel NDD protein BLTP2. Overall, our study is innovative, provides new insights into differential missense variation in NDDs and cancer at the protein-level, and contributes necessary information toward building a framework for thinking about prognostic and therapeutic aspects of these proteins.\nA GWAS for grip strength in cohorts of children, advantages of analysing young participants for this trait.\nAuthors: Abbondanza, F.; Wang, C. A.; Schmitz, J.; Whitehouse, A. J. O.; Pennell, C. E.; Paracchini, S.\nScore: 2.2, Published: 2024-02-09 DOI: 10.1101/2024.02.09.24302539\nGrip strength (GS) is a proxy measure for muscular strength and a predictor for bone fracture risk among other diseases. Previous genomewide association studies (GWAS) have been conducted in large cohorts of adults focusing on scores collected for the dominant hand, therefore increasing the likelihood of confounding effects by environmental factors. Here, we perform the first GWAS meta-analyses on maximal GS with the dominant (GSD) and non-dominant (GSND) hand in two cohorts of children (ALSPAC, N = 5,450; age range = 10.65 / 13.61; Raine Study, N = 1,162, age range: 9.42 / 12.38 years). We identified a novel significant association for GSND (rs9546244, LINC02465, p = 3.43e-08) and replicated associations previously reported in adults including with a HOXB3 gene marker that shows an eQTL effect. Despite a much smaller sample (~3%) compared to the UK Biobank we replicated correlation and polygenic risk score (PRS) analyses previously reported in this much larger adult cohort. Specifically, we observed genetic correlations with coronary artery disease and a PRS association with the risk of overall fracture. Furthermore, we observed a higher SNP-heritability (24-41%) compared to previous studies (4-24%) in adults. Our results suggest that cohorts of children might be better suited for genetic studies of grip strength, possibly due to the shorter exposure to confounding environmental factors compared to adults.\nComparative analysis of 136,401 Admixed Americans and 419,228 Europeans reveals ancestry-specific genetic determinants of clonal haematopoiesis\nAuthors: Wen, S.; Kuri-Morales, P.; Hu, F.; Nag, A.; Tachmazidou, I.; Vishnu Vardhan Deevi, S.; Taiy, H.; Smith, K.; Loesch, D. P.; Burren, O. S.; Dhindsa, R. S.; Wasilewski, S.; Alegre-Diaz, J.; Berumen, J.; Emberson, J.; Torres, J. M.; Collins, R.; Carss, K.; Wang, Q.; Petrovski, S.; Tapia-Conyer, R.; Fabre, M. A.; Harper, A. R.; Vassiliou, G.; Mitchell, J.\nScore: 2.2, Published: 2024-02-09 DOI: 10.1101/2024.02.07.24302442\nThe development of clonal haematopoiesis (CH), the age-related expansion of mutated haematopoietic stem cell (HSC) clones, is influenced by genetic and non-genetic factors. To date, large-scale studies of CH have focused on individuals of European descent, such that the impact of genetic ancestry on CH development remains incompletely understood. Here, we investigate this by studying CH in 136,401 admixed participants from the Mexico City Prospective Study (MCPS) and 419,228 European participants from the UK Biobank (UKB). We observe that CH was significantly less common in MCPS compared to UKB (adjusted odds ratio (OR) = 0.56 [95% Cl = 0.55-0.59], P = 1.60 x 10-206), a difference that persisted when comparing MCPS participants whose genomes were \u003e50% ancestrally Indigenous American to those whose genomes were \u003e50% ancestrally European (adjusted OR = 0.76 [0.70-0.83], P = 1.78 x 10-10). Genome- and exome-wide association analyses in MCPS participants identified two novel loci associated with CH (CSGALNACT1 and DIAPH3), and ancestry-specific variants in the TCL1B locus with opposing effect on DNMT3A- versus non-DNMT3A-CH. Meta-analysis of the MCPS and UKB cohorts identified another five novel loci associated with overall or gene specific CH, including polymorphisms at PAPR11/CCND2, MEIS1 and UBE2G1/SPNS3. Our CH study, the largest in a non-European population to date, demonstrates the profound impact of ancestry on CH development and reveals the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis and advance health equity amongst different human populations.\nWhole genome sequencing of a family with autosomal dominant features within the oculoauriculovertebral spectrum\nAuthors: Petrin, A. L.; Machado-Paula, L. A.; Hinkle, A. B.; Hovey, L.; Awotoye, W.; Chimenti, M. S.; Darbro, B. W.; Ribeiro-Bicudo, L. A.; Dabdoub, S. M.; Peter, T. K.; Murray, J. C.; Van Otterloo, E.; Rengasamy Venugopalan, S.; Moreno-Uribe, L. M.\nScore: 2.2, Published: 2024-02-07 DOI: 10.1101/2024.02.07.24301824\nBackgroundOculoauriculovertebral Spectrum (OAVS) encompasses a wide variety of anomalies on derivatives from the first and second pharyngeal arches including macrostomia, hemifacial microsomia, micrognathia, preauricular tags, ocular and vertebral anomalies. We present the genetic findings of a large three-generation family with multiple members affected with macrostomia, preauricular tags and uni- or bilateral ptosis following an autosomal dominant segregation pattern. MethodsWe generated whole genome sequencing data for the proband, affected parent and unaffected paternal grandparent followed by Sanger sequencing on 23 family members for the top 10 candidate genes: KCND2, PDGFRA, CASP9, NCOA3, WNT10A, SIX1, MTF1, KDR/VEGFR2, LRRK1, and TRIM2. We performed parent and sibling-based transmission disequilibrium tests and burden analysis to explore segregation and burden of candidate gene mutations. Bioinformatic analyses investigated the biological connection between genes and the abnormal phenotypes. ResultsOverall, rare missense mutations in SIX1, KDR/VEGFR2, and PDGFRA showed the best evidence of segregation with the OAV phenotypes in this family. When considering affection with any of the 3 OAVS phenotypes as an outcome, parent-TDTs and sib-TDTs (unadjusted p-values) found that SIX1 (p=0.025, p=0.052), followed by PDGFRA (p=0.180, p=0.069) and KDR/VEGFR2 (p=0.180, p=0.069) have the strongest associations in this family. Burden analysis via a penalized linear mixed model identified SIX1 (RC=0.87) and PDGFRA (RC=0.98) as having the strongest association with OAVS severity. Using phenotype-specific ogfrautcomes, sib-TDTs identified associations between (1) SIX1 with uni- or bilateral ptosis (p=0.049) and ear tags (p=0.01), (2) PDGFRA and KDR/VEGFR2 with ear tags (both p\u003c0.01). ConclusionOur study reports the genomic findings of a large family with multiple individuals affected with OAVS phenotypes with autosomal dominant inheritance. Our findings narrow down to three potential candidate genes, SIX1, PDGFRA, and KDR/VEGFR2. Among these, SIX1 has been previously associated with OAVS ear malformations and it is co-expressed with EYA1 during ear development. Attempts to strengthen the genotype-phenotype co-relation underlying the OAVS of phenotypes are essential to discover the etiological factors leading to this complex and burdensome condition as well as for family counseling and prevention efforts.\nA reassessment of Hardy-Weinberg equilibrium filtering in large sample Genomic studies.\nAuthors: Greer, P. J.; Sedlakova, A.; Ellison, M.; Oranburg, T. D.; Maiers, M.; Whitcomb, D. C.; Busby, B.\nScore: 2.1, Published: 2024-02-08 DOI: 10.1101/2024.02.07.24301951\nHardy Weinberg Equilibrium (HWE) is a fundamental principle of population genetics. Adherence to HWE, using a p-value filter, is used as a quality control measure to remove potential genotyping errors prior to certain analyses. Larger sample sizes increase power to differentiate smaller effect sizes, but will also affect methods of quality control. Here, we test the effects of current methods of HWE QC filtering on varying sample sizes up to 486,178 subjects for imputed and Whole Exome Sequencing (WES) genotypes using data from the UK Biobank and propose potential alternative filtering methods. METHODS: Simulations were performed on imputed genotype data using chromosome 1. WES GWAS (Genome Wide Association Study) was performed using PLINK2. RESULTS: Our simulations on the imputed data from Chromosome 1 show a progressive increase in the number of SNPs eliminated from analysis as sample sizes increase. As the HWE p-value filter remains constant at p\u003c1e-15, the number of SNPs removed increases from 1.66% at n=10,000 to 18.86% at n=486,178 in a multi-ancestry cohort and from 0.002% at n=10,000 to 0.334% at n=300,000 in a European ancestry cohort. Greater reductions are shown in WES analysis with a 11.91% reduction in analyzed SNPs in a European ancestry cohort n=362,192, and a 32.70% reduction in SNPs in a multi- ancestry dataset n=463,605. Using a sample size specific HWE p-value cutoff removes ~2.25% of SNPs in the all ancestry cohort across all sample sizes, but does not currently scale beyond 300,000 samples. A hard cutoff of +/- 20% deviation from HWE produces the most consistent results and scales across all sample sizes but requires additional user steps. CONCLUSION: Testing for deviance from HWE may still be an important quality control step in GWAS studies, however we demonstrate here that using an HWE p-value threshold that is acceptable for smaller sample sizes will be inappropriate for large sample studies due to an unnecessarily high number of variants removed prior to analysis. Rather than exclude variants that fail HWE prior to analysis it may be better to include all variants in the analysis and examine their deviation from HWE afterward. We believe that adjusting the cutoffs will be even more important for large whole genome sequencing results and more diverse population studies.\n",
  "wordCount" : "3164",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.24301497">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.24301497" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.24301497">
        <p class="paperTitle">Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.24301497" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.24301497" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O&#39;Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B.; Pengam, A.; Cogne, B.; MacArthur, D. G.; Ulitsky, I.; Carvill, G. L.; O&#39;Donnell-Luria, A.</p>
        <p class="info">Score: 95.1, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.24301497' target='https://doi.org/10.1101/2024.01.31.24301497'> 10.1101/2024.01.31.24301497</a></p>
        <p class="abstract">Abstract/SummaryGenes encoding long non-coding RNAs (lncRNAs) comprise a large fraction of the human genome, yet haploinsufficiency of a lncRNA has not been shown to cause a Mendelian disease. CHASERR is a highly conserved human lncRNA adjacent to CHD2-a coding gene in which de novo loss-of-function variants cause developmental and epileptic encephalopathy. Here we report three unrelated individuals each harboring an ultra-rare heterozygous de novo deletion in the CHASERR locus. We report similarities in severe developmental delay, facial dysmorphisms, and cerebral dysmyelination in these individuals, distinguishing them from the phenotypic spectrum of CHD2 haploinsufficiency. We demonstrate reduced CHASERR mRNA expression and corresponding increased CHD2 mRNA and protein in whole blood and patient-derived cell lines-specifically increased expression of the CHD2 allele in cis with the CHASERR deletion, as predicted from a prior mouse model of Chaserr haploinsufficiency. We show for the first time that de novo structural variants facilitated by Alu-mediated non-allelic homologous recombination led to deletion of a non-coding element (the lncRNA CHASERR) to cause a rare syndromic neurodevelopmental disorder. We also demonstrate that CHD2 has bidirectional dosage sensitivity in human disease. This work highlights the need to carefully evaluate other lncRNAs, particularly those upstream of genes associated with Mendelian disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.23300612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.23300612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.23300612">
        <p class="paperTitle">Whole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.23300612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.23300612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Stamelou, M.; Arzberger, T.; Roeber, S.; Muller, U.; Hopfner, F.; Hopfner, F.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L</p>
        <p class="info">Score: 13.2, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.23300612' target='https://doi.org/10.1101/2023.12.28.23300612'> 10.1101/2023.12.28.23300612</a></p>
        <p class="abstract">BackgroundProgressive supranuclear palsy (PSP) is a rare neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. Previous genome-wide association studies for PSP were based on genotype array, therefore, were inadequate for the analysis of rare variants as well as larger mutations, such as small insertions/deletions (indels) and structural variants (SVs).

MethodIn this study, we performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), indels, and SVs, in a cohort of 1,718 cases and 2,944 controls of European ancestry. Of the 1,718 PSP individuals, 1,441 were autopsy-confirmed and 277 were clinically diagnosed.

ResultsOur analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and further uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. Notably, in contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele in PSP. Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. Moreover, seven common SVs associated with PSP were observed in the H1/H2 haplotype region (17q21.31) and other loci, including IGH, PCMT1, CYP2A13, and SMCP. In the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP.

ConclusionsThrough WGS, we significantly enhanced our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.24302211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.24302211" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.24302211">
        <p class="paperTitle">Genome-wide association analysis of composite sleep health scores in 413,904 individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.24302211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.24302211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.; Wang, H.</p>
        <p class="info">Score: 5.2, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.24302211' target='https://doi.org/10.1101/2024.02.02.24302211'> 10.1101/2024.02.02.24302211</a></p>
        <p class="abstract">Recent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, and together may provide a more complete picture of sleep health, while also illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p&lt;8.3e-9), along with 341 previously reported loci (p&lt;5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2=0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections to behavioral, psychological, and cardiometabolic traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.24301443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.24301443" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.24301443">
        <p class="paperTitle">Prognostic Value of Multiplexed Assays of Variant Effect and Automated Patch-clamping for KCNH2-LQTS Risk Stratification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.24301443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.24301443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Neill, M. J.; Ng, C. A.; Aizawa, T.; Sala, L.; Bains, S.; Denjoy, I.; Winbo, A.; Ullah, R.; Shen, Q.; Tan, C.-Y.; Kozek, K.; Vanags, L.; Mitchell, D.; Shen, A.; Wada, Y.; Kashiwa, A.; Crotti, L.; Dagradi, F.; Musu, G.; Neves, R.; Bos, J. M.; Giudicessi, J.; Bledsoe, X.; Lancaster, M. C.; Glazer, A.; Roden, D. M.; Leenhardt, A.; Salem, J.-E.; Earle, N.; Stiles, R.; Agee, T.; Johnson, C.; Horie, M.; Skinner, J.; Extramiana, F.; Ackerman, M. J.; Schwartz, P.; Ohno, S.; Vandenberg, J.; Kroncke, B.</p>
        <p class="info">Score: 5.1, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.24301443' target='https://doi.org/10.1101/2024.02.01.24301443'> 10.1101/2024.02.01.24301443</a></p>
        <p class="abstract">BackgroundLong QT syndrome (LQTS) is a lethal arrhythmia condition, frequently caused by rare loss-of-function variants in the cardiac potassium channel encoded by KCNH2. Variant-based risk stratification is complicated by heterogenous clinical data, incomplete penetrance, and low-throughput functional data.

ObjectiveTo test the utility of variant-specific features, including high-throughput functional data, to predict cardiac events among KCNH2 variant heterozygotes.

MethodsWe quantified cell-surface trafficking of 18,323 variants in KCNH2 and recorded potassium current densities for 506 KCNH2 variants. Next, we deeply phenotyped 1150 KCNH2 missense variant patients, including ECG features, cardiac event history (528 total cardiac events), and mortality. We then assessed variant functional, in silico, structural, and LQTS penetrance data to stratify event-free survival for cardiac events in the study cohort.

ResultsVariant-specific current density (HR 0.28 [0.13-0.60]) and estimates of LQTS penetrance incorporating MAVE data (HR 3.16 [1.59-6.27]) were independently predictive of severe cardiac events when controlling for patient-specific features. Risk prediction models incorporating these data significantly improved prediction of 20 year cardiac events (AUC 0.79 [0.75-0.82]) over patient-only covariates (QTc and sex) (AUC 0.73 [0.70-0.77]).

ConclusionWe show that high-throughput functional data, and other variant-specific features, meaningfully contribute to both diagnosis and prognosis of a clinically actionable monogenic disease.

Graphical AbstractImplementation of KCNH2 variant functional studies, deep clinical phenotyping, and cardiac event risk stratification.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=127 SRC=&#34;FIGDIR/small/24301443v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (53K):
org.highwire.dtl.DTLVardef@19961e3org.highwire.dtl.DTLVardef@e99eb3org.highwire.dtl.DTLVardef@1b232a8org.highwire.dtl.DTLVardef@1bca01_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.24302219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.24302219" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.24302219">
        <p class="paperTitle">PRESCOTT: a population aware, epistatic and structural model accurately predicts missense effect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.24302219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.24302219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tekpinar, M.; David, L.; Henry, T.; Carbone, A.</p>
        <p class="info">Score: 4.5, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.24302219' target='https://doi.org/10.1101/2024.02.03.24302219'> 10.1101/2024.02.03.24302219</a></p>
        <p class="abstract">Predicting the functional impact of point mutations is a complex yet vital task in genomics. PRESCOTT stands at the forefront of this challenge and reconstructs complete mutational landscapes of proteins, enables the identification of protein regions most vulnerable to mutations and assigns scores to individual mutations, assisting pathologists in evaluating the pathogenic potential of missense variants. PRESCOTT categorizes these variants into three distinct classes: Benign, Pathogenic, or Variants of Uncertain Significance (VUS). The model leverages protein sequences across millions of species, advanced protein structural models, and extensive genomic and exomic data from diverse human populations. By using only sequence and structural information, it significantly improves on current standards for predicting mutations in human proteins and matches AlphaMissense performance, which incorporates allele frequency data in its analysis. By including population-specific allele frequencies, PRESCOTT excels in genome-scale score separation of ClinVar benign and pathogenic variants and surpasses AlphaMissense in analyzing the ACMG reference human dataset and the over 1800 proteins from the Human Protein Dataset. Its efficacy is particularly notable in autoinflammatory diseases, accurately predicting pathogenic gain-of-function missense mutations, a task known for its difficulty. Efficiency and accessibility are key aspects of PRESCOTT. The user-friendly PRESCOTT webserver facilitates mutation effect calculations on any protein and protein variants. The server hosts a Comprehensive Human Protein Database for over 19,000 human proteins, based on sequences and structures, ready for a customized allele population analysis. Additionally, the tool provides open access to all intermediate scores, ensuring interpretability and transparency in variant analysis. PRESCOTT is a significant stride forward in the field of genomic medicine, offering unparalleled insights into protein mutational impacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.24302238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.24302238" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.24302238">
        <p class="paperTitle">Proteome-Wide Assessment of Clustering of Missense Variants in Neurodevelopmental Disorders Versus Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.24302238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.24302238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ng, J. K.; Chen, Y.; Akinwe, T. M.; Heins, H. B.; Mehinovic, E.; Chang, Y.; Payne, Z. L.; Manuel, J. G.; Karchin, R.; Turner, T. N.</p>
        <p class="info">Score: 3.0, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.24302238' target='https://doi.org/10.1101/2024.02.02.24302238'> 10.1101/2024.02.02.24302238</a></p>
        <p class="abstract">Missense de novo variants (DNVs) and missense somatic variants contribute to neurodevelopmental disorders (NDDs) and cancer, respectively. Proteins with statistical enrichment based on analyses of these variants exhibit convergence in the differing NDD and cancer phenotypes. Herein, the question of why some of the same proteins are identified in both phenotypes is examined through investigation of clustering of missense variation at the protein level. Our hypothesis is that missense variation is present in different protein locations in the two phenotypes leading to the distinct phenotypic outcomes. We tested this hypothesis in 1D protein space using our software CLUMP. Furthermore, we newly developed 3D-CLUMP that uses 3D protein structures to spatially test clustering of missense variation for proteome-wide significance. We examined missense DNVs in 39,883 parent-child sequenced trios with NDDs and missense somatic variants from 10,543 sequenced tumors covering five TCGA cancer types and two COSMIC pan-cancer aggregates of tissue types. There were 57 proteins with proteome-wide significant missense variation clustering in NDDs when compared to cancers and 79 proteins with proteome-wide significant missense clustering in cancers compared to NDDs. While our main objective was to identify differences in patterns of missense variation, we also identified a novel NDD protein BLTP2. Overall, our study is innovative, provides new insights into differential missense variation in NDDs and cancer at the protein-level, and contributes necessary information toward building a framework for thinking about prognostic and therapeutic aspects of these proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.24302539">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.24302539" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.24302539">
        <p class="paperTitle">A GWAS for grip strength in cohorts of children, advantages of analysing young participants for this trait.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.24302539" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.24302539" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abbondanza, F.; Wang, C. A.; Schmitz, J.; Whitehouse, A. J. O.; Pennell, C. E.; Paracchini, S.</p>
        <p class="info">Score: 2.2, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.24302539' target='https://doi.org/10.1101/2024.02.09.24302539'> 10.1101/2024.02.09.24302539</a></p>
        <p class="abstract">Grip strength (GS) is a proxy measure for muscular strength and a predictor for bone fracture risk among other diseases. Previous genomewide association studies (GWAS) have been conducted in large cohorts of adults focusing on scores collected for the dominant hand, therefore increasing the likelihood of confounding effects by environmental factors. Here, we perform the first GWAS meta-analyses on maximal GS with the dominant (GSD) and non-dominant (GSND) hand in two cohorts of children (ALSPAC, N = 5,450; age range = 10.65 / 13.61; Raine Study, N = 1,162, age range: 9.42 / 12.38 years). We identified a novel significant association for GSND (rs9546244, LINC02465, p = 3.43e-08) and replicated associations previously reported in adults including with a HOXB3 gene marker that shows an eQTL effect. Despite a much smaller sample (~3%) compared to the UK Biobank we replicated correlation and polygenic risk score (PRS) analyses previously reported in this much larger adult cohort. Specifically, we observed genetic correlations with coronary artery disease and a PRS association with the risk of overall fracture. Furthermore, we observed a higher SNP-heritability (24-41%) compared to previous studies (4-24%) in adults. Our results suggest that cohorts of children might be better suited for genetic studies of grip strength, possibly due to the shorter exposure to confounding environmental factors compared to adults.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.24302442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.24302442" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.24302442">
        <p class="paperTitle">Comparative analysis of 136,401 Admixed Americans and 419,228 Europeans reveals ancestry-specific genetic determinants of clonal haematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.24302442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.24302442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wen, S.; Kuri-Morales, P.; Hu, F.; Nag, A.; Tachmazidou, I.; Vishnu Vardhan Deevi, S.; Taiy, H.; Smith, K.; Loesch, D. P.; Burren, O. S.; Dhindsa, R. S.; Wasilewski, S.; Alegre-Diaz, J.; Berumen, J.; Emberson, J.; Torres, J. M.; Collins, R.; Carss, K.; Wang, Q.; Petrovski, S.; Tapia-Conyer, R.; Fabre, M. A.; Harper, A. R.; Vassiliou, G.; Mitchell, J.</p>
        <p class="info">Score: 2.2, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.24302442' target='https://doi.org/10.1101/2024.02.07.24302442'> 10.1101/2024.02.07.24302442</a></p>
        <p class="abstract">The development of clonal haematopoiesis (CH), the age-related expansion of mutated haematopoietic stem cell (HSC) clones, is influenced by genetic and non-genetic factors. To date, large-scale studies of CH have focused on individuals of European descent, such that the impact of genetic ancestry on CH development remains incompletely understood. Here, we investigate this by studying CH in 136,401 admixed participants from the Mexico City Prospective Study (MCPS) and 419,228 European participants from the UK Biobank (UKB). We observe that CH was significantly less common in MCPS compared to UKB (adjusted odds ratio (OR) = 0.56 [95% Cl = 0.55-0.59], P = 1.60 x 10-206), a difference that persisted when comparing MCPS participants whose genomes were &gt;50% ancestrally Indigenous American to those whose genomes were &gt;50% ancestrally European (adjusted OR = 0.76 [0.70-0.83], P = 1.78 x 10-10). Genome- and exome-wide association analyses in MCPS participants identified two novel loci associated with CH (CSGALNACT1 and DIAPH3), and ancestry-specific variants in the TCL1B locus with opposing effect on DNMT3A- versus non-DNMT3A-CH. Meta-analysis of the MCPS and UKB cohorts identified another five novel loci associated with overall or gene specific CH, including polymorphisms at PAPR11/CCND2, MEIS1 and UBE2G1/SPNS3. Our CH study, the largest in a non-European population to date, demonstrates the profound impact of ancestry on CH development and reveals the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis and advance health equity amongst different human populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.24301824">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.24301824" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.24301824">
        <p class="paperTitle">Whole genome sequencing of a family with autosomal dominant features within the oculoauriculovertebral spectrum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.24301824" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.24301824" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petrin, A. L.; Machado-Paula, L. A.; Hinkle, A. B.; Hovey, L.; Awotoye, W.; Chimenti, M. S.; Darbro, B. W.; Ribeiro-Bicudo, L. A.; Dabdoub, S. M.; Peter, T. K.; Murray, J. C.; Van Otterloo, E.; Rengasamy Venugopalan, S.; Moreno-Uribe, L. M.</p>
        <p class="info">Score: 2.2, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.24301824' target='https://doi.org/10.1101/2024.02.07.24301824'> 10.1101/2024.02.07.24301824</a></p>
        <p class="abstract">BackgroundOculoauriculovertebral Spectrum (OAVS) encompasses a wide variety of anomalies on derivatives from the first and second pharyngeal arches including macrostomia, hemifacial microsomia, micrognathia, preauricular tags, ocular and vertebral anomalies. We present the genetic findings of a large three-generation family with multiple members affected with macrostomia, preauricular tags and uni- or bilateral ptosis following an autosomal dominant segregation pattern.

MethodsWe generated whole genome sequencing data for the proband, affected parent and unaffected paternal grandparent followed by Sanger sequencing on 23 family members for the top 10 candidate genes: KCND2, PDGFRA, CASP9, NCOA3, WNT10A, SIX1, MTF1, KDR/VEGFR2, LRRK1, and TRIM2. We performed parent and sibling-based transmission disequilibrium tests and burden analysis to explore segregation and burden of candidate gene mutations. Bioinformatic analyses investigated the biological connection between genes and the abnormal phenotypes.

ResultsOverall, rare missense mutations in SIX1, KDR/VEGFR2, and PDGFRA showed the best evidence of segregation with the OAV phenotypes in this family. When considering affection with any of the 3 OAVS phenotypes as an outcome, parent-TDTs and sib-TDTs (unadjusted p-values) found that SIX1 (p=0.025, p=0.052), followed by PDGFRA (p=0.180, p=0.069) and KDR/VEGFR2 (p=0.180, p=0.069) have the strongest associations in this family. Burden analysis via a penalized linear mixed model identified SIX1 (RC=0.87) and PDGFRA (RC=0.98) as having the strongest association with OAVS severity. Using phenotype-specific ogfrautcomes, sib-TDTs identified associations between (1) SIX1 with uni- or bilateral ptosis (p=0.049) and ear tags (p=0.01), (2) PDGFRA and KDR/VEGFR2 with ear tags (both p&lt;0.01).

ConclusionOur study reports the genomic findings of a large family with multiple individuals affected with OAVS phenotypes with autosomal dominant inheritance. Our findings narrow down to three potential candidate genes, SIX1, PDGFRA, and KDR/VEGFR2. Among these, SIX1 has been previously associated with OAVS ear malformations and it is co-expressed with EYA1 during ear development. Attempts to strengthen the genotype-phenotype co-relation underlying the OAVS of phenotypes are essential to discover the etiological factors leading to this complex and burdensome condition as well as for family counseling and prevention efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.24301951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.24301951" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.24301951">
        <p class="paperTitle">A reassessment of Hardy-Weinberg equilibrium filtering in large sample Genomic studies.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.24301951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.24301951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greer, P. J.; Sedlakova, A.; Ellison, M.; Oranburg, T. D.; Maiers, M.; Whitcomb, D. C.; Busby, B.</p>
        <p class="info">Score: 2.1, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.24301951' target='https://doi.org/10.1101/2024.02.07.24301951'> 10.1101/2024.02.07.24301951</a></p>
        <p class="abstract">Hardy Weinberg Equilibrium (HWE) is a fundamental principle of population genetics. Adherence to HWE, using a p-value filter, is used as a quality control measure to remove potential genotyping errors prior to certain analyses. Larger sample sizes increase power to differentiate smaller effect sizes, but will also affect methods of quality control. Here, we test the effects of current methods of HWE QC filtering on varying sample sizes up to 486,178 subjects for imputed and Whole Exome Sequencing (WES) genotypes using data from the UK Biobank and propose potential alternative filtering methods. METHODS: Simulations were performed on imputed genotype data using chromosome 1. WES GWAS (Genome Wide Association Study) was performed using PLINK2. RESULTS: Our simulations on the imputed data from Chromosome 1 show a progressive increase in the number of SNPs eliminated from analysis as sample sizes increase. As the HWE p-value filter remains constant at p&lt;1e-15, the number of SNPs removed increases from 1.66% at n=10,000 to 18.86% at n=486,178 in a multi-ancestry cohort and from 0.002% at n=10,000 to 0.334% at n=300,000 in a European ancestry cohort. Greater reductions are shown in WES analysis with a 11.91% reduction in analyzed SNPs in a European ancestry cohort n=362,192, and a 32.70% reduction in SNPs in a multi- ancestry dataset n=463,605. Using a sample size specific HWE p-value cutoff removes ~2.25% of SNPs in the all ancestry cohort across all sample sizes, but does not currently scale beyond 300,000 samples. A hard cutoff of &#43;/- 20% deviation from HWE produces the most consistent results and scales across all sample sizes but requires additional user steps. CONCLUSION: Testing for deviance from HWE may still be an important quality control step in GWAS studies, however we demonstrate here that using an HWE p-value threshold that is acceptable for smaller sample sizes will be inappropriate for large sample studies due to an unnecessarily high number of variants removed prior to analysis. Rather than exclude variants that fail HWE prior to analysis it may be better to include all variants in the analysis and examine their deviation from HWE afterward. We believe that adjusting the cutoffs will be even more important for large whole genome sequencing results and more diverse population studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
