<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis
Authors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis
Authors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis
Authors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis\nAuthors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G.",
  "keywords": [
    
  ],
  "articleBody": " Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis\nAuthors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G.; Pierce, E. A.; Place, E. M.; Pajusalu, S.; Pais, L.; Ounap, K.; Osei-Owusu, I.; Okur, V.; Oja, K. T.; OLeary, M.; OHeir, E.; Morel, C.; Marchant, R. G.; Mangilog, B. E.; Madden, J. A.; MacArthur, D.; Lovgren, A.; Lerner-Ellis, J. P.; Lin, J.; Laing, N.; H\nScore: 21.9, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293829\nBackgroundCausal variants underlying rare disorders may remain elusive even after expansive gene panels or exome sequencing (ES). Clinicians and researchers may then turn to genome sequencing (GS), though the added value of this technique and its optimal use remain poorly defined. We therefore investigated the advantages of GS within a phenotypically diverse cohort. MethodsGS was performed for 744 individuals with rare disease who were genetically undiagnosed. Analysis included review of single nucleotide, indel, structural, and mitochondrial variants. ResultsWe successfully solved 218/744 (29.3%) cases using GS, with most solves involving established disease genes (157/218, 72.0%). Of all solved cases, 148 (67.9%) had previously had non-diagnostic ES. We systematically evaluated the 218 causal variants for features requiring GS to identify and 61/218 (28.0%) met these criteria, representing 8.2% of the entire cohort. These included small structural variants (13), copy neutral inversions and complex rearrangements (8), tandem repeat expansions (6), deep intronic variants (15), and coding variants that may be more easily found using GS related to uniformity of coverage (19). ConclusionWe describe the diagnostic yield of GS in a large and diverse cohort, illustrating several types of pathogenic variation eluding ES or other techniques. Our results reveal a higher diagnostic yield of GS, supporting the utility of a genome-first approach, with consideration of GS as a secondary or tertiary test when higher-resolution structural variant analysis is needed or there is a strong clinical suspicion for a condition and prior targeted genetic testing has been negative.\nGenetic influences on circulating retinol and its relationship to human health\nAuthors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.\nScore: 20.6, Published: 2023-08-10 DOI: 10.1101/2023.08.07.23293796\nRetinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan. Previous work has characterised genetic influences on circulating retinol; however, small sample sizes have limited our ability to fully appreciate the genetic architecture of this trait. In this study, we performed the largest genome-wide association study (GWAS) of retinol to date in up to 22,274 participants. We identified eight common variant loci associated with retinol, as well as a rare-variant signal. An integrative gene prioritisation pipeline supported novel retinol-associated genes outside of the main retinol transport complex (RBP4:TTR) related to lipid biology, energy homeostasis, and endocrine signalling. Genetic proxies of circulating retinol were then used to estimate causal relationships with almost 20,000 clinical phenotypes via a phenome-wide Mendelian randomisation study (MR-pheWAS). The MR-pheWAS suggested that retinol may exert causal effects on inflammation, adiposity, ocular measures, the microbiome, and MRI-derived brain phenotypes, amongst several others. Conversely, circulating retinol may be causally influenced by factors including lipids and renal function. Finally, we demonstrated how a retinol polygenic score could identify individuals who are more likely to fall outside of the normative range of circulating retinol for a given age. In summary, this study provides a comprehensive evaluation of the genetics of circulating retinol, as well as revealing traits which should be prioritised for further clinical investigation with respect to retinol related therapies or nutritional intervention.\nRare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank\nAuthors: Fenner, E.; Holmans, P.; O'Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.\nScore: 9.0, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294074\nImpairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.\nSevere obesity may be an oligogenic condition: evidence from 1,714 adults seeking treatment in the UK National Health Service\nAuthors: Almansoori, S.; Amin, H.; Alsters, S.; Handley, D. K.; Yiorkas, A. M.; Hashim, N. A. N.; Ramzi, N. H.; Bonaomi, G.; Ahmed, J.; Agrawal, S.; Small, P.; Purkayastha, S.; Haelst, M. V.; Walters, R. G.; Roux, C. W. l.; Chahal, H. S.; Drenos, F.; Blakemore, A. I.\nScore: 11.6, Published: 2023-08-08 DOI: 10.1101/2023.08.04.23293229\nSevere (class III) obesity is a chronic, relapsing condition, with a high burden of co-morbidity and mortality. Previous evidence has established genetics as an important contributing factor. We therefore designed a custom genotyping array to screen a cohort of UK patients seeking treatment for severe obesity. 1,714 participants were genotyped using a custom AXIOM array, focusing on rare (minor allele frequency \u003c0.01) variants, with CADD-PHRED [\u0026ge;]15 in 78 genes known/suspected to cause Mendelian forms of obesity. Concordance analyses of 22 duplicate samples and 66 samples with whole exome sequence data revealed good genotyping reliability. We identified the proportion of study participants who carried, or were homozygous for, rare, predicted-deleterious variants in genes with dominant and recessive modes of inheritance (MOI), respectively. 27% of patients carried relevant mutations consistent with the expected MOI, which was very similar to the rate observed in the UKB 50K whole exome sequencing dataset. However, the clinical obesity cohort were more likely to carry two or more such variants, in separate genes, than UK Biobank participants (p = 0.018). In conclusion, our results provide evidence: that (i) custom genotyping arrays, used with improved algorithms can allow reliable, cost-effective screening for rare genetic variants; (ii) rare mutations in \"obesity genes\" may be at high prevalence among bariatric patients, as well as in the general population; and (iii) that severe obesity may have an oligogenic pattern of inheritance in some cases.\nEstimating the direct effects of the genetic liabilities to bipolar disorder, schizophrenia, and behavioral traits on suicide attempt using a multivariable Mendelian randomization approach\nAuthors: Cabrera Mendoza, B.; Aydin, N.; Fries, G. R.; Docherty, A.; Walss-Bass, C.; Polimanti, R.\nScore: 6.8, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294083\nBipolar disorder (BD) and schizophrenia (SZ) are associated with higher odds of suicide attempt (SA). In this study, we aimed to explore the effect of BD and SZ genetic liabilities on SA, also considering the contribution of behavioral traits, socioeconomic factors, and substance use disorders. Leveraging large-scale genome-wide association data from the Psychiatric Genomics Consortium (PGC) and the UK Biobank (UKB), we conducted a two-sample Mendelian randomization (MR) analysis to evaluate the putative causal effect of BD (41,917 cases, 371,549 controls) and SZ (53,386 cases, 77,258 controls) on SA (26,590 cases, 492,022 controls). Then, we assessed the putative causal effect of BD and SZ on behavioral traits, socioeconomic factors, and substance use disorders. Considering the associations identified, we evaluated the direct causal effect of behavioral traits, socioeconomic factors, and substance use disorders on SA using a multivariable MR approach. The genetic liabilities to BD and SZ were associated with higher odds of SA (BD odds ratio (OR)=1.24, p=3.88x10-12; SZ OR=1.09, p=2.44x10-20). However, while the effect of mental distress (OR=1.17, p=1.02x10-4) and risk-taking (OR=1.52, p=0.028) on SA was independent of SZ genetic liability, the BD-SA relationship appeared to account for the effect of these risk factors. Similarly, the association with loneliness on SA was null after accounting for the effect of SZ genetic liability. These findings highlight the complex interplay between genetic risk of psychiatric disorders and behavioral traits in the context of SA, suggesting the need for a comprehensive mental health assessment for high-risk individuals.\nAn Evolutionary Perspective on the Genetics of Anorexia Nervosa\nAuthors: Breton, E.; Kaufmann, T.\nScore: 5.4, Published: 2023-08-15 DOI: 10.1101/2023.08.09.23293879\nAnorexia nervosa (AN) is a serious psychiatric disorder that typically emerges around adolescence and predominantly affects girls and women. Recent progress has been made in identifying biological correlates of this debilitating disorder, but more research is needed to pinpoint the specific mechanisms that lead to the development and maintenance of AN. There is a known phenotypic link between AN and growth as well as sexual maturation, yet the genetic overlap between these phenotypes has never been investigated in detail. One may hypothesize that shared factors between AN, energy metabolism and reproductive functions may have been under recent evolutionary selection. Here, we characterize the genetic overlap between AN, BMI and age at menarche, and took an evolutionary perspective, aiming to reveal potential evolutionary factors that may help explain the origin of AN in recent human evolution. We obtained publicly available GWAS summary statistics of AN, BMI and age at menarche and studied the polygenic overlap between them. We found 22 loci overlapping between AN and BMI, and 9 loci between AN and age at menarche, with twelve of these not previously associated with AN. Next, we used Neandertal Selective Sweep scores to explore potential positive evolutionary selection for loci associated with the three phenotypes, or overlapping between them. We found evidence that gene variants associated with AN may have been under particular evolutionary dynamic. The circadian cycle seemed to be highly relevant to the studied genetic overlaps and prone to recent evolutionary selection, offering a promising avenue for future research into chronobiology. Taken together, our findings contribute to the understanding of the genetic underpinning of AN. Ultimately, better knowledge of the biological origins of AN may help to target specific biological processes and facilitate early intervention in individuals who are most at risk.\nStrong protective effect of the APOL1 p.N264K variant against G2-associated FSGS and kidney disease\nAuthors: Gupta, Y.; Friedman, D. J.; McNulty, M.; Khan, A.; Lane, B.; Wang, C.; Ke, J.; Jin, G.; Wooden, B.; Knob, A. L.; Lim, T. Y.; Appel, G. B.; Huggins, K.; Liu, L.; Mitrotti, A.; Stangl, M. C.; Bomback, A.; Westland, R.; Bodria, M.; Marasa, M.; Shang, N.; Cohen, D. J.; Crew, R. J.; Morello, W.; Canetta, P.; Radhakrishnan, J.; Martino, J.; Liu, Q.; Chung, W. K.; Espinoza, A.; Luo, Y.; Wei, W.-Q.; Feng, Q.; Weng, C.; Fang, Y.; Kullo, I. J.; Naderian, M.; Limdi, N.; Irvin, M. R.; Tiwari, H.; Mohan, S.; Rao, M.; Dube, G.; Chaudhary, N. S.; Gutierrez, O. M.; Judd, S. E.; Cushman, M.; Lange, L. A.; Lang\nScore: 11.1, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293554\nBlack Americans have a significantly higher risk of developing chronic kidney disease (CKD), especially focal segmental glomerulosclerosis (FSGS), than European Americans. Two coding variants (G1 and G2) in the APOL1 gene play a major role in this disparity. While 13% of Black Americans carry the high-risk recessive genotypes, only a fraction of these individuals develops FSGS or kidney failure, indicating the involvement of additional disease modifiers. Here, we show that the presence of the APOL1 p.N264K missense variant, when co-inherited with the G2 APOL1 risk allele, substantially reduces the penetrance of the G1G2 and G2G2 high-risk genotypes by rendering these genotypes low-risk. These results align with prior functional evidence showing that the p.N264K variant reduces the toxicity of the APOL1 high-risk alleles. These findings have important implications for our understanding of the mechanisms of APOL1-associated nephropathy, as well as for the clinical management of individuals with high-risk genotypes that include the G2 allele.\nNovel risk loci for COVID-19 hospitalization among admixed American populations\nAuthors: Carracedo, A.; Spanish COalition to Unlock Research on host GEnetics on COVID-19 (SCOURGE), Score: 4.3, Published: 2023-08-15 DOI: 10.1101/2023.08.11.23293871\nThe genetic basis of severe COVID-19 has been thoroughly studied and many genetic risk factors shared between populations have been identified. However, reduced sample sizes from non-European groups have limited the discovery of population-specific common risk loci. In this second study nested in the SCOURGE consortium, we have conducted the largest GWAS meta-analysis for COVID-19 hospitalization in admixed Americans, comprising a total of 4,702 hospitalized cases recruited by SCOURGE and other seven participating studies in the COVID-19 Host Genetic Initiative. We identified four genome-wide significant associations, two of which constitute novel loci and first discovered in Latin-American populations (BAZ2B and DDIAS). A trans-ethnic meta-analysis revealed another novel cross-population risk locus in CREBBP. Finally, we assessed the performance of a cross-ancestry polygenic risk score in the SCOURGE admixed American cohort.\nHuman and computer attention in assessing genetic conditions\nAuthors: Duong, D.; Johny, A. R.; Ledgister Hanchard, S.; Fortney, C.; Hellmann, F.; Hu, P.; Moosa, S.; Patel, T.; Persky, S.; Sumer, O.; Tekendo-Ngongang, C.; Hsieh, T.-C.; Waikel, R. L.; Andre, E.; Krawitz, P.; Solomon, B. D.\nScore: 5.4, Published: 2023-07-28 DOI: 10.1101/2023.07.26.23293119\nDeep learning (DL) and other types of artificial intelligence (AI) are increasingly used in many biomedical areas, including genetics. One frequent use in medical genetics involves evaluating images of people with potential genetic conditions to help with diagnosis. A central question involves better understanding how AI classifiers assess images compared to humans. To explore this, we performed eye-tracking analyses of geneticist clinicians and non-clinicians. We compared results to DL-based saliency maps. We found that human visual attention when assessing images differs greatly from the parts of images weighted by the DL model. Further, individuals tend to have a specific pattern of image inspection, and clinicians demonstrate different visual attention patterns than non-clinicians.\nPregnancy-Associated Bleeding and Genetics: Five Sequence Variants in the Myometrium and Progesterone Signaling Pathway are associated with postpartum hemorrhage\nAuthors: Westergaard, D.; Steinthorsdottir, V.; Stefansdottir, L.; Rohde, P. D.; Wu, X.; Geller, F.; Tyrmi, J.; Havulinna, A. S.; Navais, P. S.; Flatley, C.; Ostrowski, S. R.; Pedersen, O. B.; Erikstrup, C.; Soerensen, E.; Mikkelsen, C.; Brun, M. T.; Jensen, B. A.; Brodersen, T.; Ullum, H.; FinnGen, ; Danish Blood Donor Study Genomic Consortium, ; Estonian Biobank Research Team, ; Nordic Collaboration for Womens and Reproductive Health, ; Magnus, P.; Andreassen, O. A.; Njolstad, P. R.; Kolte, A. M.; Krebs, L.; Nyegaard, M.; Hansen, T. F.; Fenstra, B.; Daly, M.; Lindgren, C. M.; Thorleifsson, G.; St\nScore: 2.5, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293932\nBleeding in early pregnancy and postpartum hemorrhage (PPH) bear substantial risks, with the former closely associated with pregnancy loss and the latter being the foremost cause of maternal death, underscoring the severity of these complications in maternal-fetal health. Here, we investigated the genetic variation underlying aspects of pregnancy-associated bleeding and identified five loci associated with PPH through a meta-analysis of 21,512 cases and 259,500 controls. Functional annotation analysis indicated candidate genes, HAND2, TBX3, and RAP2C/FRMD7, at three loci and showed that at each locus, associated variants were located within binding sites for progesterone receptors (PGR). Furthermore, there were strong genetic correlations with birth weight, gestational duration, and uterine fibroids. Early bleeding during pregnancy (28,898 cases and 302,894 controls) yielded no genome-wide association signals, but showed strong genetic correlation with a variety of human traits, indicative of polygenic and pleiotropic effects. Our results suggest that postpartum bleeding is related to myometrium dysregulation, whereas early bleeding is a complex trait related to underlying health and possibly socioeconomic status.\n",
  "wordCount" : "2660",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293829">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293829" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293829">
        <p class="paperTitle">Unique Capabilities of Genome Sequencing for Rare Disease Diagnosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293829" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293829" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wojcik, M. H.; Lemire, G.; Zaki, M. S.; Wissmann, M.; Win, W.; White, S.; Weisburd, B.; Waddell, L. B.; Verboon, J. M.; VanNoy, G. E.; Topf, A.; Tan, T. Y.; Straub, V.; Stenton, S. L.; Snow, H.; Singer-Berk, M.; Silver, J.; Shril, S.; Seaby, E. G.; Schneider, R.; Sankaran, V. G.; Sanchis-Juan, A.; Russell, K. A.; Reinson, K.; Ravenscroft, G.; Pierce, E. A.; Place, E. M.; Pajusalu, S.; Pais, L.; Ounap, K.; Osei-Owusu, I.; Okur, V.; Oja, K. T.; OLeary, M.; OHeir, E.; Morel, C.; Marchant, R. G.; Mangilog, B. E.; Madden, J. A.; MacArthur, D.; Lovgren, A.; Lerner-Ellis, J. P.; Lin, J.; Laing, N.; H</p>
        <p class="info">Score: 21.9, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293829' target='https://doi.org/10.1101/2023.08.08.23293829'> 10.1101/2023.08.08.23293829</a></p>
        <p class="abstract">BackgroundCausal variants underlying rare disorders may remain elusive even after expansive gene panels or exome sequencing (ES). Clinicians and researchers may then turn to genome sequencing (GS), though the added value of this technique and its optimal use remain poorly defined. We therefore investigated the advantages of GS within a phenotypically diverse cohort.

MethodsGS was performed for 744 individuals with rare disease who were genetically undiagnosed. Analysis included review of single nucleotide, indel, structural, and mitochondrial variants.

ResultsWe successfully solved 218/744 (29.3%) cases using GS, with most solves involving established disease genes (157/218, 72.0%). Of all solved cases, 148 (67.9%) had previously had non-diagnostic ES. We systematically evaluated the 218 causal variants for features requiring GS to identify and 61/218 (28.0%) met these criteria, representing 8.2% of the entire cohort. These included small structural variants (13), copy neutral inversions and complex rearrangements (8), tandem repeat expansions (6), deep intronic variants (15), and coding variants that may be more easily found using GS related to uniformity of coverage (19).

ConclusionWe describe the diagnostic yield of GS in a large and diverse cohort, illustrating several types of pathogenic variation eluding ES or other techniques. Our results reveal a higher diagnostic yield of GS, supporting the utility of a genome-first approach, with consideration of GS as a secondary or tertiary test when higher-resolution structural variant analysis is needed or there is a strong clinical suspicion for a condition and prior targeted genetic testing has been negative.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.23293796">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.23293796" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.23293796">
        <p class="paperTitle">Genetic influences on circulating retinol and its relationship to human health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.23293796" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.23293796" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reay, W. R.; Kiltschewskij, D. J.; Di Biase, M. A.; Gerring, Z. F.; Kundu, K.; Surendran, P.; Greco, L. A.; Clarke, E. D.; Collins, C. E.; Mondul, A. M.; Albanes, D.; Cairns, M. J.</p>
        <p class="info">Score: 20.6, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.23293796' target='https://doi.org/10.1101/2023.08.07.23293796'> 10.1101/2023.08.07.23293796</a></p>
        <p class="abstract">Retinol is a fat-soluble vitamin that plays an essential role in many biological processes throughout the human lifespan. Previous work has characterised genetic influences on circulating retinol; however, small sample sizes have limited our ability to fully appreciate the genetic architecture of this trait. In this study, we performed the largest genome-wide association study (GWAS) of retinol to date in up to 22,274 participants. We identified eight common variant loci associated with retinol, as well as a rare-variant signal. An integrative gene prioritisation pipeline supported novel retinol-associated genes outside of the main retinol transport complex (RBP4:TTR) related to lipid biology, energy homeostasis, and endocrine signalling. Genetic proxies of circulating retinol were then used to estimate causal relationships with almost 20,000 clinical phenotypes via a phenome-wide Mendelian randomisation study (MR-pheWAS). The MR-pheWAS suggested that retinol may exert causal effects on inflammation, adiposity, ocular measures, the microbiome, and MRI-derived brain phenotypes, amongst several others. Conversely, circulating retinol may be causally influenced by factors including lipids and renal function. Finally, we demonstrated how a retinol polygenic score could identify individuals who are more likely to fall outside of the normative range of circulating retinol for a given age. In summary, this study provides a comprehensive evaluation of the genetics of circulating retinol, as well as revealing traits which should be prioritised for further clinical investigation with respect to retinol related therapies or nutritional intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294074">
        <p class="paperTitle">Rare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fenner, E.; Holmans, P.; O&#39;Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.</p>
        <p class="info">Score: 9.0, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294074' target='https://doi.org/10.1101/2023.08.14.23294074'> 10.1101/2023.08.14.23294074</a></p>
        <p class="abstract">Impairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.23293229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.23293229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.23293229">
        <p class="paperTitle">Severe obesity may be an oligogenic condition: evidence from 1,714 adults seeking treatment in the UK National Health Service</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.23293229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.23293229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Almansoori, S.; Amin, H.; Alsters, S.; Handley, D. K.; Yiorkas, A. M.; Hashim, N. A. N.; Ramzi, N. H.; Bonaomi, G.; Ahmed, J.; Agrawal, S.; Small, P.; Purkayastha, S.; Haelst, M. V.; Walters, R. G.; Roux, C. W. l.; Chahal, H. S.; Drenos, F.; Blakemore, A. I.</p>
        <p class="info">Score: 11.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.23293229' target='https://doi.org/10.1101/2023.08.04.23293229'> 10.1101/2023.08.04.23293229</a></p>
        <p class="abstract">Severe (class III) obesity is a chronic, relapsing condition, with a high burden of co-morbidity and mortality. Previous evidence has established genetics as an important contributing factor. We therefore designed a custom genotyping array to screen a cohort of UK patients seeking treatment for severe obesity. 1,714 participants were genotyped using a custom AXIOM array, focusing on rare (minor allele frequency &lt;0.01) variants, with CADD-PHRED [&amp;ge;]15 in 78 genes known/suspected to cause Mendelian forms of obesity. Concordance analyses of 22 duplicate samples and 66 samples with whole exome sequence data revealed good genotyping reliability. We identified the proportion of study participants who carried, or were homozygous for, rare, predicted-deleterious variants in genes with dominant and recessive modes of inheritance (MOI), respectively. 27% of patients carried relevant mutations consistent with the expected MOI, which was very similar to the rate observed in the UKB 50K whole exome sequencing dataset. However, the clinical obesity cohort were more likely to carry two or more such variants, in separate genes, than UK Biobank participants (p = 0.018). In conclusion, our results provide evidence: that (i) custom genotyping arrays, used with improved algorithms can allow reliable, cost-effective screening for rare genetic variants; (ii) rare mutations in &#34;obesity genes&#34; may be at high prevalence among bariatric patients, as well as in the general population; and (iii) that severe obesity may have an oligogenic pattern of inheritance in some cases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294083">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294083" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294083">
        <p class="paperTitle">Estimating the direct effects of the genetic liabilities to bipolar disorder, schizophrenia, and behavioral traits on suicide attempt using a multivariable Mendelian randomization approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294083" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294083" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cabrera Mendoza, B.; Aydin, N.; Fries, G. R.; Docherty, A.; Walss-Bass, C.; Polimanti, R.</p>
        <p class="info">Score: 6.8, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294083' target='https://doi.org/10.1101/2023.08.14.23294083'> 10.1101/2023.08.14.23294083</a></p>
        <p class="abstract">Bipolar disorder (BD) and schizophrenia (SZ) are associated with higher odds of suicide attempt (SA). In this study, we aimed to explore the effect of BD and SZ genetic liabilities on SA, also considering the contribution of behavioral traits, socioeconomic factors, and substance use disorders. Leveraging large-scale genome-wide association data from the Psychiatric Genomics Consortium (PGC) and the UK Biobank (UKB), we conducted a two-sample Mendelian randomization (MR) analysis to evaluate the putative causal effect of BD (41,917 cases, 371,549 controls) and SZ (53,386 cases, 77,258 controls) on SA (26,590 cases, 492,022 controls). Then, we assessed the putative causal effect of BD and SZ on behavioral traits, socioeconomic factors, and substance use disorders. Considering the associations identified, we evaluated the direct causal effect of behavioral traits, socioeconomic factors, and substance use disorders on SA using a multivariable MR approach. The genetic liabilities to BD and SZ were associated with higher odds of SA (BD odds ratio (OR)=1.24, p=3.88x10-12; SZ OR=1.09, p=2.44x10-20). However, while the effect of mental distress (OR=1.17, p=1.02x10-4) and risk-taking (OR=1.52, p=0.028) on SA was independent of SZ genetic liability, the BD-SA relationship appeared to account for the effect of these risk factors. Similarly, the association with loneliness on SA was null after accounting for the effect of SZ genetic liability. These findings highlight the complex interplay between genetic risk of psychiatric disorders and behavioral traits in the context of SA, suggesting the need for a comprehensive mental health assessment for high-risk individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.23293879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.23293879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.23293879">
        <p class="paperTitle">An Evolutionary Perspective on the Genetics of Anorexia Nervosa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.23293879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.23293879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Breton, E.; Kaufmann, T.</p>
        <p class="info">Score: 5.4, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.23293879' target='https://doi.org/10.1101/2023.08.09.23293879'> 10.1101/2023.08.09.23293879</a></p>
        <p class="abstract">Anorexia nervosa (AN) is a serious psychiatric disorder that typically emerges around adolescence and predominantly affects girls and women. Recent progress has been made in identifying biological correlates of this debilitating disorder, but more research is needed to pinpoint the specific mechanisms that lead to the development and maintenance of AN. There is a known phenotypic link between AN and growth as well as sexual maturation, yet the genetic overlap between these phenotypes has never been investigated in detail. One may hypothesize that shared factors between AN, energy metabolism and reproductive functions may have been under recent evolutionary selection. Here, we characterize the genetic overlap between AN, BMI and age at menarche, and took an evolutionary perspective, aiming to reveal potential evolutionary factors that may help explain the origin of AN in recent human evolution. We obtained publicly available GWAS summary statistics of AN, BMI and age at menarche and studied the polygenic overlap between them. We found 22 loci overlapping between AN and BMI, and 9 loci between AN and age at menarche, with twelve of these not previously associated with AN. Next, we used Neandertal Selective Sweep scores to explore potential positive evolutionary selection for loci associated with the three phenotypes, or overlapping between them. We found evidence that gene variants associated with AN may have been under particular evolutionary dynamic. The circadian cycle seemed to be highly relevant to the studied genetic overlaps and prone to recent evolutionary selection, offering a promising avenue for future research into chronobiology. Taken together, our findings contribute to the understanding of the genetic underpinning of AN. Ultimately, better knowledge of the biological origins of AN may help to target specific biological processes and facilitate early intervention in individuals who are most at risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293554">
        <p class="paperTitle">Strong protective effect of the APOL1 p.N264K variant against G2-associated FSGS and kidney disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, Y.; Friedman, D. J.; McNulty, M.; Khan, A.; Lane, B.; Wang, C.; Ke, J.; Jin, G.; Wooden, B.; Knob, A. L.; Lim, T. Y.; Appel, G. B.; Huggins, K.; Liu, L.; Mitrotti, A.; Stangl, M. C.; Bomback, A.; Westland, R.; Bodria, M.; Marasa, M.; Shang, N.; Cohen, D. J.; Crew, R. J.; Morello, W.; Canetta, P.; Radhakrishnan, J.; Martino, J.; Liu, Q.; Chung, W. K.; Espinoza, A.; Luo, Y.; Wei, W.-Q.; Feng, Q.; Weng, C.; Fang, Y.; Kullo, I. J.; Naderian, M.; Limdi, N.; Irvin, M. R.; Tiwari, H.; Mohan, S.; Rao, M.; Dube, G.; Chaudhary, N. S.; Gutierrez, O. M.; Judd, S. E.; Cushman, M.; Lange, L. A.; Lang</p>
        <p class="info">Score: 11.1, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293554' target='https://doi.org/10.1101/2023.08.02.23293554'> 10.1101/2023.08.02.23293554</a></p>
        <p class="abstract">Black Americans have a significantly higher risk of developing chronic kidney disease (CKD), especially focal segmental glomerulosclerosis (FSGS), than European Americans. Two coding variants (G1 and G2) in the APOL1 gene play a major role in this disparity. While 13% of Black Americans carry the high-risk recessive genotypes, only a fraction of these individuals develops FSGS or kidney failure, indicating the involvement of additional disease modifiers.

Here, we show that the presence of the APOL1 p.N264K missense variant, when co-inherited with the G2 APOL1 risk allele, substantially reduces the penetrance of the G1G2 and G2G2 high-risk genotypes by rendering these genotypes low-risk. These results align with prior functional evidence showing that the p.N264K variant reduces the toxicity of the APOL1 high-risk alleles. These findings have important implications for our understanding of the mechanisms of APOL1-associated nephropathy, as well as for the clinical management of individuals with high-risk genotypes that include the G2 allele.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.23293871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.23293871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.23293871">
        <p class="paperTitle">Novel risk loci for COVID-19 hospitalization among admixed American populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.23293871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.23293871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carracedo, A.; Spanish COalition to Unlock Research on host GEnetics on COVID-19 (SCOURGE),  </p>
        <p class="info">Score: 4.3, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.23293871' target='https://doi.org/10.1101/2023.08.11.23293871'> 10.1101/2023.08.11.23293871</a></p>
        <p class="abstract">The genetic basis of severe COVID-19 has been thoroughly studied and many genetic risk factors shared between populations have been identified. However, reduced sample sizes from non-European groups have limited the discovery of population-specific common risk loci. In this second study nested in the SCOURGE consortium, we have conducted the largest GWAS meta-analysis for COVID-19 hospitalization in admixed Americans, comprising a total of 4,702 hospitalized cases recruited by SCOURGE and other seven participating studies in the COVID-19 Host Genetic Initiative. We identified four genome-wide significant associations, two of which constitute novel loci and first discovered in Latin-American populations (BAZ2B and DDIAS). A trans-ethnic meta-analysis revealed another novel cross-population risk locus in CREBBP. Finally, we assessed the performance of a cross-ancestry polygenic risk score in the SCOURGE admixed American cohort.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.23293119">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.23293119" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.23293119">
        <p class="paperTitle">Human and computer attention in assessing genetic conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.23293119" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.23293119" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duong, D.; Johny, A. R.; Ledgister Hanchard, S.; Fortney, C.; Hellmann, F.; Hu, P.; Moosa, S.; Patel, T.; Persky, S.; Sumer, O.; Tekendo-Ngongang, C.; Hsieh, T.-C.; Waikel, R. L.; Andre, E.; Krawitz, P.; Solomon, B. D.</p>
        <p class="info">Score: 5.4, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.23293119' target='https://doi.org/10.1101/2023.07.26.23293119'> 10.1101/2023.07.26.23293119</a></p>
        <p class="abstract">Deep learning (DL) and other types of artificial intelligence (AI) are increasingly used in many biomedical areas, including genetics. One frequent use in medical genetics involves evaluating images of people with potential genetic conditions to help with diagnosis. A central question involves better understanding how AI classifiers assess images compared to humans. To explore this, we performed eye-tracking analyses of geneticist clinicians and non-clinicians. We compared results to DL-based saliency maps. We found that human visual attention when assessing images differs greatly from the parts of images weighted by the DL model. Further, individuals tend to have a specific pattern of image inspection, and clinicians demonstrate different visual attention patterns than non-clinicians.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23293932">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23293932" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23293932">
        <p class="paperTitle">Pregnancy-Associated Bleeding and Genetics: Five Sequence Variants in the Myometrium and Progesterone Signaling Pathway are associated with postpartum hemorrhage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23293932" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23293932" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Westergaard, D.; Steinthorsdottir, V.; Stefansdottir, L.; Rohde, P. D.; Wu, X.; Geller, F.; Tyrmi, J.; Havulinna, A. S.; Navais, P. S.; Flatley, C.; Ostrowski, S. R.; Pedersen, O. B.; Erikstrup, C.; Soerensen, E.; Mikkelsen, C.; Brun, M. T.; Jensen, B. A.; Brodersen, T.; Ullum, H.; FinnGen,  ; Danish Blood Donor Study Genomic Consortium,  ; Estonian Biobank Research Team,  ; Nordic Collaboration for Womens and Reproductive Health,  ; Magnus, P.; Andreassen, O. A.; Njolstad, P. R.; Kolte, A. M.; Krebs, L.; Nyegaard, M.; Hansen, T. F.; Fenstra, B.; Daly, M.; Lindgren, C. M.; Thorleifsson, G.; St</p>
        <p class="info">Score: 2.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23293932' target='https://doi.org/10.1101/2023.08.10.23293932'> 10.1101/2023.08.10.23293932</a></p>
        <p class="abstract">Bleeding in early pregnancy and postpartum hemorrhage (PPH) bear substantial risks, with the former closely associated with pregnancy loss and the latter being the foremost cause of maternal death, underscoring the severity of these complications in maternal-fetal health. Here, we investigated the genetic variation underlying aspects of pregnancy-associated bleeding and identified five loci associated with PPH through a meta-analysis of 21,512 cases and 259,500 controls. Functional annotation analysis indicated candidate genes, HAND2, TBX3, and RAP2C/FRMD7, at three loci and showed that at each locus, associated variants were located within binding sites for progesterone receptors (PGR). Furthermore, there were strong genetic correlations with birth weight, gestational duration, and uterine fibroids. Early bleeding during pregnancy (28,898 cases and 302,894 controls) yielded no genome-wide association signals, but showed strong genetic correlation with a variety of human traits, indicative of polygenic and pleiotropic effects. Our results suggest that postpartum bleeding is related to myometrium dysregulation, whereas early bleeding is a complex trait related to underlying health and possibly socioeconomic status.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
