<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences
Authors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.
Score: 13.2, Published: 2023-09-19 DOI: 10.1101/2023.09.18.23295715
Telomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences
Authors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.
Score: 13.2, Published: 2023-09-19 DOI: 10.1101/2023.09.18.23295715
Telomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences
Authors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.
Score: 13.2, Published: 2023-09-19 DOI: 10.1101/2023.09.18.23295715
Telomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences\nAuthors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.\nScore: 13.2, Published: 2023-09-19 DOI: 10.1101/2023.09.18.23295715\nTelomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing.",
  "keywords": [
    
  ],
  "articleBody": " Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences\nAuthors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.\nScore: 13.2, Published: 2023-09-19 DOI: 10.1101/2023.09.18.23295715\nTelomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing. We developed a joint telomere length (TL) metric, combining both qPCR and whole genome sequencing (WGS) measurements across 462,675 UK Biobank participants that increased our ability to capture TL heritability by 36% (h2mean=0.058 to h2combined=0.079) and improved predictions of age. Exome-wide rare variant (minor allele frequency\u003c0.001) and gene-level collapsing association studies identified 53 variants and 22 genes significantly associated with TL that included allelic series in ACD and RTEL1. Five of the 31 rare-variant TL associated genes (16%) were also known drivers of clonal haematopoiesis (CH), prompting somatic variant analyses. Stratifying by CH clone size, we uncovered novel gene-specific associations with TL, including lengthened telomeres in individuals with large SRSF2-mutant clones, in contrast to the progressive telomere shortening observed with increasing clonal expansions driven by other CH genes. Our findings demonstrate the impact of rare variants on TL with larger effects in genes associated with CH, a precursor of myeloid cancers and several other non-malignant human diseases. Telomere biology is likely to be an important focus for the prevention and treatment of these conditions.\nClinical actionability of genetic findings in cerebral palsy\nAuthors: Lewis, S. A.; Chopra, M.; Cohen, J. S.; Bain, J.; Aravamuthan, B. R.; Carmel, J. B.; Fahey, M. C.; Segel, R.; Wintle, R. F.; Zech, M.; May, H.; Haque, N.; Fehlings, D.; Srivastava, S.; Kruer, M. C.\nScore: 10.9, Published: 2023-09-11 DOI: 10.1101/2023.09.08.23295195\nBackground and objectivesSingle gene mutations are increasingly recognized as causes of cerebral palsy (CP) phenotypes, yet there is currently no standardized framework for measuring their clinical impact. We evaluated Pathogenic/Likely Pathogenic (P/LP) variants identified in individuals with CP to determine how frequently genetic testing results would prompt changes in care. MethodsWe analyzed published P/LP variants in OMIM genes identified in clinical (n = 1,345 individuals) or research (n = 496) cohorts using exome sequencing of CP patients. We established a working group of clinical and research geneticists, developmental pediatricians, genetic counselors, and neurologists and performed a systematic review of existing literature for evidence of clinical management approaches linked to genetic disorders. Scoring rubrics were adapted, and a modified Delphi approach was used to build consensus and establish the anticipated impact on patient care. Overall clinical utility was calculated from metrics assessing outcome severity if left untreated, safety/practicality of the intervention, and anticipated intervention efficacy. ResultsWe found 140/1,841 (8%) of individuals in published CP cohorts had a genetic diagnosis classified as actionable, defined as prompting a change in clinical management based on knowledge related to the genetic etiology. 58/243 genes with P/LP variants were classified as actionable; 16 had treatment options targeting the primary disease mechanism, 16 had specific prevention strategies, and 26 had specific symptom management recommendations. The level of evidence was also graded according to ClinGen criteria; 44.6% of interventions had evidence class \"D\" or below. The potential interventions have clinical utility with 97% of outcomes being moderate-high severity if left untreated and 62% of interventions predicted to be of moderate-high efficacy. Most interventions (71%) were considered moderate-high safety/practicality. DiscussionOur findings indicate that actionable genetic findings occur in 8% of individuals referred for genetic testing with CP. Evaluation of potential efficacy, outcome severity, and intervention safety/practicality indicates moderate-high clinical utility of these genetic findings. Thus, genetic sequencing to identify these individuals for precision medicine interventions could improve outcomes and provide clinical benefit to individuals with CP. The relatively limited evidence base for most interventions underscores the need for additional research.\nA systematic review of genome-wide association studies on bladder cancer\nAuthors: Ahmad, A. A.; Muhammad, U.; Ibrahim, B.; Kwairanga, S. H.; Garkuwa, U. A.; Jibril, M. M.; Ahmad, U.\nScore: 3.3, Published: 2023-09-18 DOI: 10.1101/2023.09.17.23295670\nBackgroundBladder cancer (BC) is the most common cancer of the urinary tract worldwide with over 550,000 new cases each year, bladder cancer has drawn relatively limited research attention and healthcare interventions despite the escalating incidence and mortality rates, particularly in Africa. Historically, the clinical handling of bladder cancer remained largely unchanged for many years. However, novel research initiatives have heralded a fresh epoch in its diagnosis and treatment, fueled by detailed probing of molecular changes. AimThis study aimed to identify genetic susceptibility loci associated with bladder cancer by systematically reviewing previous Genome-Wide Association Studies (GWAS). MethodsIn line with this objective, comprehensive literature searches were conducted across PubMed, Google Scholar, and relevant genetic databases, focusing on bladder cancer GWAS studies from 2000 through to November 2022. This systematic review adhered to the robust PRISMA standards. To evaluate the credibility of the studies under scrutiny, the Newcastle-Ottawa Scale was employed, further assessing any potential bias risk. ResultsThe investigation identified chromosome 18q12.3 as the most vulnerable to bladder cancer, revealing four polymorphisms at this locus: rs7238033, rs10775480, rs11082469, and rs17674580. Furthermore, chromosome 5p15.3 emerged as the second most susceptible, with three noted polymorphisms: rs2736098 and two instances of rs401681. ConclusionDespite these findings, our understanding of genetic predisposition to bladder cancer remains rudimentary, with the majority of substantial data deriving from GWAS. No additional genetic association evidence emerged from this systematic review. Given the relatively minor influence of our current knowledge of genetic susceptibility to bladder cancer on public health, a call for larger cohort studies is necessary. These expanded studies can potentially unveil a broader range of significant polymorphisms across the genome, thereby enhancing our understanding and approach to bladder cancer.\nFrequentmers - a novel way to look at metagenomic Next Generation Sequencing data and an application in detecting liver cirrhosis\nAuthors: Mouratidis, I.; Chantzi, N.; Khan, U.; Konnaris, M. A.; Chan, C. S. Y.; Mareboina, M.; Georgakopoulos-Soares, I.\nScore: 3.2, Published: 2023-09-19 DOI: 10.1101/2023.09.19.23295771\nEarly detection of human disease is associated with improved clinical outcomes. However, many diseases are often detected at an advanced, symptomatic stage where patients are past efficacious treatment periods and can result in less favorable outcomes. Therefore, methods that can accurately detect human disease at a presymptomatic stage are urgently needed. Here, we introduce \"frequentmers\"; short sequences that are specific and recurrently observed in either patient or healthy control samples, but not in both. We showcase the utility of frequentmers for the detection of liver cirrhosis using metagenomic Next Generation Sequencing data from stool samples of patients and controls. We develop classification models for the detection of liver cirrhosis and achieve an AUC score of 0.91 using ten-fold cross-validation. A small subset of 200 frequentmers can achieve comparable results in detecting liver cirrhosis. Finally, we identify the microbial organisms in liver cirrhosis samples, which are associated with the most predictive frequentmer biomarkers.\nParSE-seq: A Calibrated Multiplexed Assay to Facilitate the Clinical Classification of Putative Splice-altering Variants\nAuthors: O'Neill, M. J.; Yang, T.; Laudeman, J.; Calandranis, M.; Solus, J.; Roden, D. M.; Glazer, A. M.\nScore: 3.5, Published: 2023-09-08 DOI: 10.1101/2023.09.04.23295019\nBackgroundInterpreting the clinical significance of putative splice-altering variants outside 2-base pair canonical splice sites remains difficult without functional studies. MethodsWe developed Parallel Splice Effect Sequencing (ParSE-seq), a multiplexed minigene-based assay, to test variant effects on RNA splicing quantified by high-throughput sequencing. We studied variants in SCN5A, an arrhythmia-associated gene which encodes the major cardiac voltage-gated sodium channel. We used the computational tool SpliceAI to prioritize exonic and intronic candidate splice variants, and ClinVar to select benign and pathogenic control variants. We generated a pool of 284 barcoded minigene plasmids, transfected them into Human Embryonic Kidney (HEK293) cells and induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs), sequenced the resulting pools of splicing products, and calibrated the assay to the American College of Medical Genetics and Genomics scheme. Variants were interpreted using the calibrated functional data, and experimental data were compared to SpliceAI predictions. We further studied some splice-altering missense variants by cDNA-based automated patch clamping (APC) in HEK cells and assessed splicing and sodium channel function in CRISPR-edited iPSC-CMs. ResultsParSE-seq revealed the splicing effect of 224 SCN5A variants in iPSC-CMs and 244 variants in HEK293 cells. The scores between the cell types were highly correlated (R2=0.84). In iPSCs, the assay had concordant scores for 21/22 benign/likely benign and 24/25 pathogenic/likely pathogenic control variants from ClinVar. 43/112 exonic variants and 35/70 intronic variants with determinate scores disrupted splicing. 11 of 42 variants of uncertain significance were reclassified, and 29 of 34 variants with conflicting interpretations were reclassified using the functional data. SpliceAI computational predictions correlated well with experimental data (AUC = 0.96). We identified 20 unique SCN5A missense variants that disrupted splicing, and 2 clinically observed splice-altering missense variants of uncertain significance had normal function when tested with the cDNA-based APC assay. A splice-altering intronic variant detected by ParSE-seq, c.1891-5C\u003eG, also disrupted splicing and sodium current when introduced into iPSC-CMs at the endogenous locus by CRISPR editing. ConclusionsParSE-seq is a calibrated, multiplexed, high-throughput assay to facilitate the classification of candidate splice-altering variants.\nSystematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease\nAuthors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D'Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.\nScore: 30.3, Published: 2023-09-12 DOI: 10.1101/2023.09.12.23295416\nBackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown. MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls. ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations. ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.\nThe genetic underpinnings of variable penetrance and expressivity of pathogenic mutations in cardiometabolic traits\nAuthors: Wei, A.; Border, R.; Fu, B.; Cullina, S.; Brandes, N.; Sankararaman, S.; Kenny, E.; Udler, M. S.; Ntranos, V.; Zaitlen, N.; Arboleda, V.\nScore: 1.2, Published: 2023-09-18 DOI: 10.1101/2023.09.14.23295564\nOver three percent of people carry a dominant pathogenic mutation, yet only a fraction of carriers develop disease (incomplete penetrance), and phenotypes from mutations in the same gene range from mild to severe (variable expressivity). Here, we investigate underlying mechanisms for this heterogeneity: variable variant effect sizes, carrier polygenic backgrounds, and modulation of carrier effect by genetic background (epistasis). We leveraged exomes and clinical phenotypes from the UK Biobank and the Mt. Sinai BioMe Biobank to identify carriers of pathogenic variants affecting cardiometabolic traits. We employed recently developed methods to study these cohorts, observing strong statistical support and clinical translational potential for all three mechanisms of variable penetrance and expressivity. For example, scores from our recent model of variant pathogenicity were tightly correlated with phenotype amongst clinical variant carriers, they predicted effects of variants of unknown significance, and they distinguished gain- from loss-of-function variants. We also found that polygenic scores predicted phenotypes amongst pathogenic carriers and that epistatic effects can exceed main carrier effects by an order of magnitude.\nGenome-wide association study and genomic risk prediction of age-related macular degeneration in Israel\nAuthors: Grunin, M.; Triffon, D.; Beykin, G.; Rahmani, E.; Schweiger, R.; Tiosano, L.; Khateb, S.; Hagbi-Levi, S.; Rinsky, B.; Munitz, R.; Winkler, T. W.; Heid, I. M.; Halperin, E.; Carmi, S.; Chowers, I.\nScore: 5.1, Published: 2023-09-06 DOI: 10.1101/2023.09.06.23295126\nPurposeThe risk of developing age-related macular degeneration(AMD) is influenced by genetic background. In 2016, International AMD Genomics Consortium(IAMDGC) identified 52 risk variants in 34 loci, and a polygenic risk score(PRS) based on these variants was associated with AMD. The Israeli population has a unique genetic composition: Ashkenazi Jewish(AJ), Jewish non-Ashkenazi, and Arab sub-populations. We aimed to perform a genome-wide association study(GWAS) for AMD in Israel, and to evaluate PRSs for AMD. MethodsFor our discovery set, we recruited 403 AMD patients and 256 controls at Hadassah Medical Center. We genotyped all individuals via custom exome chip. We imputed non-typed variants using cosmopolitan and AJ reference panels. We recruited additional 155 cases and 69 controls for validation. To evaluate predictive power of PRSs for AMD, we used IAMDGC summary statistics excluding our study and developed PRSs via either clumping/thresholding or LDpred2. ResultsIn our discovery set, 31/34 loci previously reported by the IAMDGC were AMD associated with P\u003c0.05. Of those, all effects were directionally consistent with the IAMDGC and 11 loci had a p-value under Bonferroni-corrected threshold(0.05/34=0.0015). At a threshold of 5x10-5, we discovered four suggestive associations in FAM189A1, IGDCC4, C7orf50, and CNTNAP4. However, only the FAM189A1 variant was AMD associated in the replication cohort after Bonferroni-correction. A prediction model including LDpred2-based PRS and other covariates had an AUC of 0.82(95%CI:0.79-0.85) and performed better than a covariates-only model(P=5.1x10-9). ConclusionsPreviously reported AMD-associated loci were nominally associated with AMD in Israel. A PRS developed based on a large international study is predictive in Israeli populations.\nRelationship between sex biases in gene expression and sex biases in autism and Alzheimer's disease\nAuthors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.\nScore: 15.2, Published: 2023-09-01 DOI: 10.1101/2023.08.29.23294773\nSex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimers disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=137 SRC=\"FIGDIR/small/23294773v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@1399eccorg.highwire.dtl.DTLVardef@1e8762borg.highwire.dtl.DTLVardef@a6375forg.highwire.dtl.DTLVardef@7d4caa_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2577",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.23295715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.23295715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.23295715">
        <p class="paperTitle">Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.23295715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.23295715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.</p>
        <p class="info">Score: 13.2, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.23295715' target='https://doi.org/10.1101/2023.09.18.23295715'> 10.1101/2023.09.18.23295715</a></p>
        <p class="abstract">Telomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing. We developed a joint telomere length (TL) metric, combining both qPCR and whole genome sequencing (WGS) measurements across 462,675 UK Biobank participants that increased our ability to capture TL heritability by 36% (h2mean=0.058 to h2combined=0.079) and improved predictions of age. Exome-wide rare variant (minor allele frequency&lt;0.001) and gene-level collapsing association studies identified 53 variants and 22 genes significantly associated with TL that included allelic series in ACD and RTEL1. Five of the 31 rare-variant TL associated genes (16%) were also known drivers of clonal haematopoiesis (CH), prompting somatic variant analyses. Stratifying by CH clone size, we uncovered novel gene-specific associations with TL, including lengthened telomeres in individuals with large SRSF2-mutant clones, in contrast to the progressive telomere shortening observed with increasing clonal expansions driven by other CH genes. Our findings demonstrate the impact of rare variants on TL with larger effects in genes associated with CH, a precursor of myeloid cancers and several other non-malignant human diseases. Telomere biology is likely to be an important focus for the prevention and treatment of these conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295195">
        <p class="paperTitle">Clinical actionability of genetic findings in cerebral palsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lewis, S. A.; Chopra, M.; Cohen, J. S.; Bain, J.; Aravamuthan, B. R.; Carmel, J. B.; Fahey, M. C.; Segel, R.; Wintle, R. F.; Zech, M.; May, H.; Haque, N.; Fehlings, D.; Srivastava, S.; Kruer, M. C.</p>
        <p class="info">Score: 10.9, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295195' target='https://doi.org/10.1101/2023.09.08.23295195'> 10.1101/2023.09.08.23295195</a></p>
        <p class="abstract">Background and objectivesSingle gene mutations are increasingly recognized as causes of cerebral palsy (CP) phenotypes, yet there is currently no standardized framework for measuring their clinical impact. We evaluated Pathogenic/Likely Pathogenic (P/LP) variants identified in individuals with CP to determine how frequently genetic testing results would prompt changes in care.

MethodsWe analyzed published P/LP variants in OMIM genes identified in clinical (n = 1,345 individuals) or research (n = 496) cohorts using exome sequencing of CP patients. We established a working group of clinical and research geneticists, developmental pediatricians, genetic counselors, and neurologists and performed a systematic review of existing literature for evidence of clinical management approaches linked to genetic disorders. Scoring rubrics were adapted, and a modified Delphi approach was used to build consensus and establish the anticipated impact on patient care. Overall clinical utility was calculated from metrics assessing outcome severity if left untreated, safety/practicality of the intervention, and anticipated intervention efficacy.

ResultsWe found 140/1,841 (8%) of individuals in published CP cohorts had a genetic diagnosis classified as actionable, defined as prompting a change in clinical management based on knowledge related to the genetic etiology. 58/243 genes with P/LP variants were classified as actionable; 16 had treatment options targeting the primary disease mechanism, 16 had specific prevention strategies, and 26 had specific symptom management recommendations. The level of evidence was also graded according to ClinGen criteria; 44.6% of interventions had evidence class &#34;D&#34; or below. The potential interventions have clinical utility with 97% of outcomes being moderate-high severity if left untreated and 62% of interventions predicted to be of moderate-high efficacy. Most interventions (71%) were considered moderate-high safety/practicality.

DiscussionOur findings indicate that actionable genetic findings occur in 8% of individuals referred for genetic testing with CP. Evaluation of potential efficacy, outcome severity, and intervention safety/practicality indicates moderate-high clinical utility of these genetic findings. Thus, genetic sequencing to identify these individuals for precision medicine interventions could improve outcomes and provide clinical benefit to individuals with CP. The relatively limited evidence base for most interventions underscores the need for additional research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.23295670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.23295670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.23295670">
        <p class="paperTitle">A systematic review of genome-wide association studies on bladder cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.23295670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.23295670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmad, A. A.; Muhammad, U.; Ibrahim, B.; Kwairanga, S. H.; Garkuwa, U. A.; Jibril, M. M.; Ahmad, U.</p>
        <p class="info">Score: 3.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.23295670' target='https://doi.org/10.1101/2023.09.17.23295670'> 10.1101/2023.09.17.23295670</a></p>
        <p class="abstract">BackgroundBladder cancer (BC) is the most common cancer of the urinary tract worldwide with over 550,000 new cases each year, bladder cancer has drawn relatively limited research attention and healthcare interventions despite the escalating incidence and mortality rates, particularly in Africa. Historically, the clinical handling of bladder cancer remained largely unchanged for many years. However, novel research initiatives have heralded a fresh epoch in its diagnosis and treatment, fueled by detailed probing of molecular changes.

AimThis study aimed to identify genetic susceptibility loci associated with bladder cancer by systematically reviewing previous Genome-Wide Association Studies (GWAS).

MethodsIn line with this objective, comprehensive literature searches were conducted across PubMed, Google Scholar, and relevant genetic databases, focusing on bladder cancer GWAS studies from 2000 through to November 2022. This systematic review adhered to the robust PRISMA standards. To evaluate the credibility of the studies under scrutiny, the Newcastle-Ottawa Scale was employed, further assessing any potential bias risk.

ResultsThe investigation identified chromosome 18q12.3 as the most vulnerable to bladder cancer, revealing four polymorphisms at this locus: rs7238033, rs10775480, rs11082469, and rs17674580. Furthermore, chromosome 5p15.3 emerged as the second most susceptible, with three noted polymorphisms: rs2736098 and two instances of rs401681.

ConclusionDespite these findings, our understanding of genetic predisposition to bladder cancer remains rudimentary, with the majority of substantial data deriving from GWAS. No additional genetic association evidence emerged from this systematic review. Given the relatively minor influence of our current knowledge of genetic susceptibility to bladder cancer on public health, a call for larger cohort studies is necessary. These expanded studies can potentially unveil a broader range of significant polymorphisms across the genome, thereby enhancing our understanding and approach to bladder cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.23295771">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.23295771" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.23295771">
        <p class="paperTitle">Frequentmers - a novel way to look at metagenomic Next Generation Sequencing data and an application in detecting liver cirrhosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.23295771" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.23295771" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mouratidis, I.; Chantzi, N.; Khan, U.; Konnaris, M. A.; Chan, C. S. Y.; Mareboina, M.; Georgakopoulos-Soares, I.</p>
        <p class="info">Score: 3.2, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.23295771' target='https://doi.org/10.1101/2023.09.19.23295771'> 10.1101/2023.09.19.23295771</a></p>
        <p class="abstract">Early detection of human disease is associated with improved clinical outcomes. However, many diseases are often detected at an advanced, symptomatic stage where patients are past efficacious treatment periods and can result in less favorable outcomes. Therefore, methods that can accurately detect human disease at a presymptomatic stage are urgently needed. Here, we introduce &#34;frequentmers&#34;; short sequences that are specific and recurrently observed in either patient or healthy control samples, but not in both. We showcase the utility of frequentmers for the detection of liver cirrhosis using metagenomic Next Generation Sequencing data from stool samples of patients and controls. We develop classification models for the detection of liver cirrhosis and achieve an AUC score of 0.91 using ten-fold cross-validation. A small subset of 200 frequentmers can achieve comparable results in detecting liver cirrhosis. Finally, we identify the microbial organisms in liver cirrhosis samples, which are associated with the most predictive frequentmer biomarkers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.23295019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.23295019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.23295019">
        <p class="paperTitle">ParSE-seq: A Calibrated Multiplexed Assay to Facilitate the Clinical Classification of Putative Splice-altering Variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.23295019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.23295019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Neill, M. J.; Yang, T.; Laudeman, J.; Calandranis, M.; Solus, J.; Roden, D. M.; Glazer, A. M.</p>
        <p class="info">Score: 3.5, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.23295019' target='https://doi.org/10.1101/2023.09.04.23295019'> 10.1101/2023.09.04.23295019</a></p>
        <p class="abstract">BackgroundInterpreting the clinical significance of putative splice-altering variants outside 2-base pair canonical splice sites remains difficult without functional studies.

MethodsWe developed Parallel Splice Effect Sequencing (ParSE-seq), a multiplexed minigene-based assay, to test variant effects on RNA splicing quantified by high-throughput sequencing. We studied variants in SCN5A, an arrhythmia-associated gene which encodes the major cardiac voltage-gated sodium channel. We used the computational tool SpliceAI to prioritize exonic and intronic candidate splice variants, and ClinVar to select benign and pathogenic control variants. We generated a pool of 284 barcoded minigene plasmids, transfected them into Human Embryonic Kidney (HEK293) cells and induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs), sequenced the resulting pools of splicing products, and calibrated the assay to the American College of Medical Genetics and Genomics scheme. Variants were interpreted using the calibrated functional data, and experimental data were compared to SpliceAI predictions. We further studied some splice-altering missense variants by cDNA-based automated patch clamping (APC) in HEK cells and assessed splicing and sodium channel function in CRISPR-edited iPSC-CMs.

ResultsParSE-seq revealed the splicing effect of 224 SCN5A variants in iPSC-CMs and 244 variants in HEK293 cells. The scores between the cell types were highly correlated (R2=0.84). In iPSCs, the assay had concordant scores for 21/22 benign/likely benign and 24/25 pathogenic/likely pathogenic control variants from ClinVar. 43/112 exonic variants and 35/70 intronic variants with determinate scores disrupted splicing. 11 of 42 variants of uncertain significance were reclassified, and 29 of 34 variants with conflicting interpretations were reclassified using the functional data. SpliceAI computational predictions correlated well with experimental data (AUC = 0.96). We identified 20 unique SCN5A missense variants that disrupted splicing, and 2 clinically observed splice-altering missense variants of uncertain significance had normal function when tested with the cDNA-based APC assay. A splice-altering intronic variant detected by ParSE-seq, c.1891-5C&gt;G, also disrupted splicing and sodium current when introduced into iPSC-CMs at the endogenous locus by CRISPR editing.

ConclusionsParSE-seq is a calibrated, multiplexed, high-throughput assay to facilitate the classification of candidate splice-altering variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295416">
        <p class="paperTitle">Systematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D&#39;Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.</p>
        <p class="info">Score: 30.3, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295416' target='https://doi.org/10.1101/2023.09.12.23295416'> 10.1101/2023.09.12.23295416</a></p>
        <p class="abstract">BackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown.

MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls.

ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations.

ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.23295564">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.23295564" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.23295564">
        <p class="paperTitle">The genetic underpinnings of variable penetrance and expressivity of pathogenic mutations in cardiometabolic traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.23295564" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.23295564" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, A.; Border, R.; Fu, B.; Cullina, S.; Brandes, N.; Sankararaman, S.; Kenny, E.; Udler, M. S.; Ntranos, V.; Zaitlen, N.; Arboleda, V.</p>
        <p class="info">Score: 1.2, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.23295564' target='https://doi.org/10.1101/2023.09.14.23295564'> 10.1101/2023.09.14.23295564</a></p>
        <p class="abstract">Over three percent of people carry a dominant pathogenic mutation, yet only a fraction of carriers develop disease (incomplete penetrance), and phenotypes from mutations in the same gene range from mild to severe (variable expressivity). Here, we investigate underlying mechanisms for this heterogeneity: variable variant effect sizes, carrier polygenic backgrounds, and modulation of carrier effect by genetic background (epistasis). We leveraged exomes and clinical phenotypes from the UK Biobank and the Mt. Sinai BioMe Biobank to identify carriers of pathogenic variants affecting cardiometabolic traits. We employed recently developed methods to study these cohorts, observing strong statistical support and clinical translational potential for all three mechanisms of variable penetrance and expressivity. For example, scores from our recent model of variant pathogenicity were tightly correlated with phenotype amongst clinical variant carriers, they predicted effects of variants of unknown significance, and they distinguished gain- from loss-of-function variants. We also found that polygenic scores predicted phenotypes amongst pathogenic carriers and that epistatic effects can exceed main carrier effects by an order of magnitude.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.23295126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.23295126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.23295126">
        <p class="paperTitle">Genome-wide association study and genomic risk prediction of age-related macular degeneration in Israel</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.23295126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.23295126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grunin, M.; Triffon, D.; Beykin, G.; Rahmani, E.; Schweiger, R.; Tiosano, L.; Khateb, S.; Hagbi-Levi, S.; Rinsky, B.; Munitz, R.; Winkler, T. W.; Heid, I. M.; Halperin, E.; Carmi, S.; Chowers, I.</p>
        <p class="info">Score: 5.1, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.23295126' target='https://doi.org/10.1101/2023.09.06.23295126'> 10.1101/2023.09.06.23295126</a></p>
        <p class="abstract">PurposeThe risk of developing age-related macular degeneration(AMD) is influenced by genetic background. In 2016, International AMD Genomics Consortium(IAMDGC) identified 52 risk variants in 34 loci, and a polygenic risk score(PRS) based on these variants was associated with AMD. The Israeli population has a unique genetic composition: Ashkenazi Jewish(AJ), Jewish non-Ashkenazi, and Arab sub-populations. We aimed to perform a genome-wide association study(GWAS) for AMD in Israel, and to evaluate PRSs for AMD.

MethodsFor our discovery set, we recruited 403 AMD patients and 256 controls at Hadassah Medical Center. We genotyped all individuals via custom exome chip. We imputed non-typed variants using cosmopolitan and AJ reference panels. We recruited additional 155 cases and 69 controls for validation. To evaluate predictive power of PRSs for AMD, we used IAMDGC summary statistics excluding our study and developed PRSs via either clumping/thresholding or LDpred2.

ResultsIn our discovery set, 31/34 loci previously reported by the IAMDGC were AMD associated with P&lt;0.05. Of those, all effects were directionally consistent with the IAMDGC and 11 loci had a p-value under Bonferroni-corrected threshold(0.05/34=0.0015). At a threshold of 5x10-5, we discovered four suggestive associations in FAM189A1, IGDCC4, C7orf50, and CNTNAP4. However, only the FAM189A1 variant was AMD associated in the replication cohort after Bonferroni-correction. A prediction model including LDpred2-based PRS and other covariates had an AUC of 0.82(95%CI:0.79-0.85) and performed better than a covariates-only model(P=5.1x10-9).

ConclusionsPreviously reported AMD-associated loci were nominally associated with AMD in Israel. A PRS developed based on a large international study is predictive in Israeli populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.23294773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.23294773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.23294773">
        <p class="paperTitle">Relationship between sex biases in gene expression and sex biases in autism and Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.23294773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.23294773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.</p>
        <p class="info">Score: 15.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.23294773' target='https://doi.org/10.1101/2023.08.29.23294773'> 10.1101/2023.08.29.23294773</a></p>
        <p class="abstract">Sex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimers disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=137 SRC=&#34;FIGDIR/small/23294773v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@1399eccorg.highwire.dtl.DTLVardef@1e8762borg.highwire.dtl.DTLVardef@a6375forg.highwire.dtl.DTLVardef@7d4caa_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
