<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 363.6, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nGenetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis\nAuthors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.\nScore: 34.6, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292611\nBackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.\nStrong Genetic Overlaps Between Dimensional and Categorical Models of Bipolar Disorders in a Family Sample\nAuthors: Arbona-Lampaya, A.; Sung, H.; D'Amico, A.; Knowles, E. E. M.; Besancon, E. K.; Freifeld, A.; Lacbawan, L.; Lopes, F.; Kassem, L.; Nardi, A. E.; McMahon, F. J.\nScore: 21.8, Published: 2023-07-10 DOI: 10.1101/2023.06.24.23291169\nObjectiveBipolar disorder (BD) often presents with a broad range of symptoms, but there is little agreement as to the heritability and genetic relationships between dimensional and categorical models of this often-disabling disorder. MethodsParticipants in the Amish-Mennonite Bipolar Genetics (AMBiGen) study, which enrolls families with BD and related disorders from Amish and Mennonite communities in North and South America, were assigned a categorical mood disorder diagnosis by structured psychiatric interview and asked to complete the Mood Disorder Questionnaire (MDQ), which assesses lifetime history of cardinal manic symptoms and associated impairment. Principal Component Analysis (PCA) was performed to analyze the dimensions of the MDQ in 726 participants, 212 of whom carried a categorical diagnosis of major mood disorder. SOLAR-ECLIPSE (v9.0.0) was used to estimate heritability and genetic overlaps between MDQ-derived measures and categorical diagnoses among 432 genotyped participants. ResultsAs expected, MDQ scores were significantly higher among individuals diagnosed with BD and related disorders. PCA suggested a three-component model for the MDQ, consistent with the literature. Heritability of the MDQ symptom score was estimated at 30% (p\u003c0.001), which was evenly distributed across its three principal components. Strong and significant genetic correlations were found between categorical diagnoses and most MDQ measures, especially impairment. ConclusionThe results support the MDQ as a dimensional measure of BD. Furthermore, significant heritability and high genetic correlations between MDQ scores and categorical diagnoses suggest a genetic continuity between dimensional and categorical measures of major mood disorders.\nPrevalence, Morbidity, and Mortality of 1,609 Men with Diagnosed and Undiagnosed Sex Chromosome Aneuploidy: Results from the Diverse Million Veteran Program (MVP) Cohort\nAuthors: Davis, S. M.; Teerlink, C.; Lynch, J. A.; Gorman, B. R.; Pagadala, M.; Liu, A.; Panizzon, M. S.; Merritt, V. C.; Genovese, G.; Pyarajan, S.; Ross, J. L.; Hauger, R. L.\nScore: 16.0, Published: 2023-07-16 DOI: 10.1101/2023.07.15.23292710\nImportance: The reported phenotypes of men with 47,XXY and 47,XYY syndromes include tall stature, multisystem comorbidities, and poor health-related quality of life (HRQoL). However, knowledge about these sex chromosome aneuploidy (SCA) conditions has been derived from studies in the \u003c15% of patients who are clinically diagnosed and also lack diversity in age and genetic ancestry. Objectives: Determine the prevalence of clinically diagnosed and undiagnosed X or Y chromosome aneuploidy among men enrolled in the Million Veteran Program (MVP); describe military service metrics of men with SCAs; compare morbidity and mortality outcomes between men with SCA with and without a clinical diagnosis to matched controls. Design: Cross-sectional, case-control Setting: United States Veterans Administration Healthcare System Participants: Biologic males enrolled in the MVP biobank with genomic identification of an additional X or Y chromosome (cases); controls matched 1:5 on sex, age, and genetic ancestry Main Outcome(s) and Measure(s): Prevalence of men with SCAs from genomic analysis; clinical SCA diagnosis; Charlson Comorbidity Index (CCI); rates of outpatient, inpatient, and emergency encounters per year; self-reported health outcomes; standardized mortality ratio (SMR) Results: An additional X or Y chromosome was present in 145 and 125 per 100,000 males in the MVP, respectively, with the highest prevalence among men with European and East Asian ancestry. At a mean age of 61 years, 74% of male veterans with 47,XXY and \u003e99% with 47,XYY remained undiagnosed. Individuals with 47,XXY (n=862) and 47,XYY (n=747) had similar military service history, all-cause SMR, and age of death compared to matched controls. CCI and healthcare utilization were higher among individuals with SCA, while several measures of HRQoL were lower. Men with a clinical diagnosis of 47,XXY had higher healthcare utilization but lower comorbidity score compared to those undiagnosed. Conclusion and Relevance: One in 370 males in the MVP cohort have SCA, a prevalence comparable to estimates in the general population. While these men have successfully served in the military, they have higher morbidity and report poorer HRQoL with aging. Longer longitudinal follow-up of this sample will be informative for clinical and patient-reported outcomes, the role of ancestry, and mortality statistics.\nFine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases\nAuthors: Yuan, K.; Longchamps, R. J.; Pardinas, A. F.; Yu, M.; Chen, T.-T.; Lin, S.-C.; Chen, Y.; Lam, M.; Liu, R.; Xia, Y.; Guo, Z.; Shi, W.; Shen, C.; The Schizophrenia Workgroup of Psychiatric Genomics Consortium, ; Daly, M. J.; Neale, B.; Feng, Y.-C. A.; Lin, Y.-F.; Chen, C.-Y.; O'Donovan, M.; Ge, T.; Huang, H.\nScore: 24.8, Published: 2023-07-09 DOI: 10.1101/2023.01.07.23284293\nGenome-wide association studies (GWAS) of human complex traits or diseases often implicate genetic loci that span hundreds or thousands of genetic variants, many of which have similar statistical significance. While statistical fine-mapping in individuals of European ancestries has made important discoveries, cross-population fine-mapping has the potential to improve power and resolution by capitalizing on the genomic diversity across ancestries. Here we present SuSiEx, an accurate and computationally efficient method for cross-population fine-mapping, which builds on the single-population fine-mapping framework, Sum of Single Effects (SuSiE). SuSiEx integrates data from an arbitrary number of ancestries, explicitly models population-specific allele frequencies and LD patterns, accounts for multiple causal variants in a genomic region, and can be applied to GWAS summary statistics. We comprehensively evaluated SuSiEx using simulations, a range of quantitative traits measured in both UK Biobank and Taiwan Biobank, and schizophrenia GWAS across East Asian and European ancestries. In all evaluations, SuSiEx fine-mapped more association signals, produced smaller credible sets and higher posterior inclusion probability (PIP) for putative causal variants, and captured population-specific causal variants.\nExome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank\nAuthors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.\nScore: 25.2, Published: 2023-07-03 DOI: 10.1101/2023.06.29.23291992\nExome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF \u003c 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( \u003c 1.68 x 10-7) and 17 nominally significant ( \u003c 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR \u003c 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.\nBayesian genome-wide TWAS with reference transcriptomic data of brain and blood tissues identified 93 risk genes for Alzheimer's disease dementia\nAuthors: Guo, S.; Yang, J.\nScore: 3.6, Published: 2023-07-12 DOI: 10.1101/2023.07.06.23292336\nBackgroundTranscriptome-wide association study (TWAS) is an influential tool for identifying novel genes associated with complex diseases, where their genetic effects may be mediated through transcriptome. TWAS utilizes reference genetic and transcriptomic data to estimate genetic effect sizes on expression quantitative traits of target genes (i.e., effect sizes of a broad sense of expression quantitative trait loci, eQTL). These estimated effect sizes are then employed as variant weights in burden gene-based association test statistics, facilitating the mapping of risk genes for complex diseases with genome-wide association study (GWAS) data. However, most existing TWAS of Alzheimers disease (AD) dementia have primarily focused on cis-eQTL, disregarding potential trans-eQTL. To overcome this limitation, we applied the Bayesian Genome-wide TWAS (BGW-TWAS) method which incorporated both cis- and trans-eQTL of brain and blood tissues to enhance mapping risk genes for AD dementia. MethodsWe first applied BGW-TWAS to the Genotype-Tissue Expression (GTEx) V8 dataset to estimate cis- and trans-eQTL effect sizes of the prefrontal cortex, cortex, and whole blood tissues. Subsequently, estimated eQTL effect sizes were integrated with the summary data of the most recent GWAS of AD dementia to obtain BGW-TWAS (i.e., gene-based association test) p-values of AD dementia per tissue type. Finally, we used the aggregated Cauchy association test to combine TWAS p-values across three tissues to obtain omnibus TWAS p-values per gene. ResultsWe identified 37 genes in prefrontal cortex, 55 in cortex, and 51 in whole blood that were significantly associated with AD dementia. By combining BGW-TWAS p-values across these three tissues, we obtained 93 significant risk genes including 29 genes primarily due to trans-eQTL and 50 novel genes. Utilizing protein-protein interaction network and phenotype enrichment analyses with these 93 significant risk genes, we detected 5 functional clusters comprised of both known and novel AD risk genes and 7 enriched phenotypes. ConclusionWe applied BGW-TWAS and aggregated Cauchy test methods to integrate both cis- and trans-eQTL data of brain and blood tissues with GWAS summary data to identify risk genes of AD dementia. The risk genes we identified provide novel insights into the underlying biological pathways implicated in AD dementia.\nPA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder\nAuthors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium, ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.\nScore: 7.5, Published: 2023-07-12 DOI: 10.1101/2023.06.23.23291611\nLarge biobank samples provide an opportunity to integrate broad phenotyping, familial records, and molecular genetics data to study complex traits and diseases. We introduce Pearson-Aitken Family Genetic Risk Scores (PA-FGRS), a new method for estimating disease liability from patterns of diagnoses in extended, age-censored genealogical records. We then apply the method to study a paradigmatic complex disorder, Major Depressive Disorder (MDD), using the iPSYCH2015 case-cohort study of 30,949 MDD cases, 39,655 random population controls, and more than 2 million relatives. We show that combining PA-FGRS liabilities estimated from family records with molecular genotypes of probands improves the three lines of inquiry. Incorporating PA-FGRS liabilities improves classification of MDD over and above polygenic scores, identifies robust genetic contributions to clinical heterogeneity in MDD associated with comorbidity, recurrence, and severity, and can improve the power of genome-wide association studies (GWAS). Our method is flexible and easy to use and our study approaches are generalizable to other data sets and other complex traits and diseases.\nThe NYCKidSeq randomized controlled trial: Impact of GUIA digitally enhanced genetic counseling in racially and ethnically diverse families\nAuthors: Suckiel, S. A.; Kelly, N. R.; Odgis, J. A.; Gallagher, K. M.; Sebastin, M.; Bonini, K. E.; Marathe, P. N.; Brown, K.; Di Biase, M.; Ramos, M.; Rodriguez, J. E.; Scarimbolo, L.; Insel, B. J.; Ferar, K. D. M.; Zinberg, R. E.; Diaz, G. A.; Greally, J. M.; Abul-Husn, N. S.; Bauman, L. J.; Gelb, B. D.; Horowitz, C. R.; Wasserstein, M. P.; Kenny, E. E.\nScore: 9.5, Published: 2023-07-07 DOI: 10.1101/2023.07.05.23292193\nBackgroundDigital solutions are needed to support rapid increases in the application of genetic and genomic tests (GT) in diverse clinical settings and patient populations. We developed GUIA, a bi-lingual web-based platform that facilitates disclosure of GT results. The NYCKidSeq randomized controlled trial evaluated GUIAs impact on understanding of GT results. MethodsNYCKidSeq enrolled diverse children with neurologic, cardiac, and immunologic conditions who underwent GT. Families were randomized to genetic counseling with GUIA (intervention) or standard of care (SOC) genetic counseling for results disclosure. Parents/legal guardians (participants) completed surveys at baseline, post-results disclosure, and 6-months later. Survey measures assessed the primary study outcomes of perceived understanding of and confidence in explaining their childs GT results and the secondary outcome of objective understanding. We used regression models to evaluate the association between the intervention and the study outcomes. ResultsThe analysis included 551 participants, 270 in the GUIA arm and 281 in SOC. Participants mean age was 41.1 years and 88.6% were mothers. Most participants were Hispanic/Latino(a) (46.3%), White/European American (24.5%), or Black/African American (15.8%). Participants in the GUIA arm had significantly higher perceived understanding post-results (OR=2.8, CI[1.004,7.617], P=0.049) and maintained higher objective understanding over time (OR=1.1, CI[1.004, 1.127], P=0.038) compared to those in the SOC arm. There was no impact on perceived confidence. Hispanic/Latino(a) individuals in the GUIA arm maintained higher perceived understanding (OR=3.9, CI[1.6, 9.3], P=0.003), confidence (OR=2.7, CI[1.021, 7.277], P=0.046), and objective understanding (OR=1.1, CI[1.009, 1.212], P=0.032) compared to SOC. ConclusionsThis trial demonstrates that GUIA positively impacts understanding of GT results in diverse parents of children with suspected genetic conditions. These findings build a case for utilizing GUIA to deliver complex and often ambiguous genetic results. Continued development and evaluation of digital applications in diverse populations are critical for equitably scaling GT offerings in specialty clinics.\nGenetic structure of major depression symptoms across clinical and community cohorts\nAuthors: Adams, M. J.; Thorp, J. G.; Jermy, B. S.; Kwong, A. S. F.; Koiv, K.; Grotzinger, A. D.; Nivard, M. G.; Marshall, S.; Milaneschi, Y.; Baune, B. T.; Mueller-Myhsok, B.; Penninx, B. W.; Boomsma, D. I.; Levinson, D. F.; Breen, G.; Pistis, G.; Grabe, H. J.; Tiemeier, H.; Berger, K.; Rietschel, M.; Magnusson, P. K.; Uher, R.; Hamilton, S. P.; Lucae, S.; Lehto, K.; Li, Q. S.; Byrne, E. M.; Hickie, I. B.; Martin, N. G.; Medland, S. E.; Wray, N. R.; Tucker-Drob, E. M.; Estonian Biobank Research Team, ; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, ; Lewis, C. M.; McI\nScore: 2.2, Published: 2023-07-07 DOI: 10.1101/2023.07.05.23292214\nDiagnostic criteria for major depressive disorder allow for heterogeneous symptom profiles but genetic analysis of major depressive symptoms has the potential to identify clinical and aetiological subtypes. There are several challenges to integrating symptom data from genetically-informative cohorts, such as sample size differences between clinical and community cohorts and various patterns of missing data. We conducted genome-wide association studies of major depressive symptoms in three clinical cohorts that were enriched for affected participants (Psychiatric Genomics Consortium, Australian Genetics of Depression Study, Generation Scotland) and three community cohorts (Avon Longitudinal Study of Parents and Children, Estonian Biobank, and UK Biobank). We fit a series of confirmatory factor models with factors that accounted for how symptom data was sampled and then compared alternative models with different symptom factors. The best fitting model had a distinct factor for Appetite/Weight symptoms and an additional measurement factor that accounted for missing data patterns in the community cohorts (use of Depression and Anhedonia as gating symptoms). The results show the importance of assessing the directionality of symptoms (such as hypersomnia versus insomnia) and of accounting for study and measurement design when meta-analysing genetic association data.\n",
  "wordCount" : "3435",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 363.6, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292611">
        <p class="paperTitle">Genetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.</p>
        <p class="info">Score: 34.6, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292611' target='https://doi.org/10.1101/2023.07.13.23292611'> 10.1101/2023.07.13.23292611</a></p>
        <p class="abstract">BackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms.

MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms.

ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases.

Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3.

Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank.

Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS.

ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.23291169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.23291169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.23291169">
        <p class="paperTitle">Strong Genetic Overlaps Between Dimensional and Categorical Models of Bipolar Disorders in a Family Sample</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.23291169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.23291169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arbona-Lampaya, A.; Sung, H.; D&#39;Amico, A.; Knowles, E. E. M.; Besancon, E. K.; Freifeld, A.; Lacbawan, L.; Lopes, F.; Kassem, L.; Nardi, A. E.; McMahon, F. J.</p>
        <p class="info">Score: 21.8, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.23291169' target='https://doi.org/10.1101/2023.06.24.23291169'> 10.1101/2023.06.24.23291169</a></p>
        <p class="abstract">ObjectiveBipolar disorder (BD) often presents with a broad range of symptoms, but there is little agreement as to the heritability and genetic relationships between dimensional and categorical models of this often-disabling disorder.

MethodsParticipants in the Amish-Mennonite Bipolar Genetics (AMBiGen) study, which enrolls families with BD and related disorders from Amish and Mennonite communities in North and South America, were assigned a categorical mood disorder diagnosis by structured psychiatric interview and asked to complete the Mood Disorder Questionnaire (MDQ), which assesses lifetime history of cardinal manic symptoms and associated impairment. Principal Component Analysis (PCA) was performed to analyze the dimensions of the MDQ in 726 participants, 212 of whom carried a categorical diagnosis of major mood disorder. SOLAR-ECLIPSE (v9.0.0) was used to estimate heritability and genetic overlaps between MDQ-derived measures and categorical diagnoses among 432 genotyped participants.

ResultsAs expected, MDQ scores were significantly higher among individuals diagnosed with BD and related disorders. PCA suggested a three-component model for the MDQ, consistent with the literature. Heritability of the MDQ symptom score was estimated at 30% (p&lt;0.001), which was evenly distributed across its three principal components. Strong and significant genetic correlations were found between categorical diagnoses and most MDQ measures, especially impairment.

ConclusionThe results support the MDQ as a dimensional measure of BD. Furthermore, significant heritability and high genetic correlations between MDQ scores and categorical diagnoses suggest a genetic continuity between dimensional and categorical measures of major mood disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.23292710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.23292710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.23292710">
        <p class="paperTitle">Prevalence, Morbidity, and Mortality of 1,609 Men with Diagnosed and Undiagnosed Sex Chromosome Aneuploidy: Results from the Diverse Million Veteran Program (MVP) Cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.23292710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.23292710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davis, S. M.; Teerlink, C.; Lynch, J. A.; Gorman, B. R.; Pagadala, M.; Liu, A.; Panizzon, M. S.; Merritt, V. C.; Genovese, G.; Pyarajan, S.; Ross, J. L.; Hauger, R. L.</p>
        <p class="info">Score: 16.0, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.23292710' target='https://doi.org/10.1101/2023.07.15.23292710'> 10.1101/2023.07.15.23292710</a></p>
        <p class="abstract">Importance: The reported phenotypes of men with 47,XXY and 47,XYY syndromes include tall stature, multisystem comorbidities, and poor health-related quality of life (HRQoL). However, knowledge about these sex chromosome aneuploidy (SCA) conditions has been derived from studies in the &lt;15% of patients who are clinically diagnosed and also lack diversity in age and genetic ancestry. Objectives: Determine the prevalence of clinically diagnosed and undiagnosed X or Y chromosome aneuploidy among men enrolled in the Million Veteran Program (MVP); describe military service metrics of men with SCAs; compare morbidity and mortality outcomes between men with SCA with and without a clinical diagnosis to matched controls. Design: Cross-sectional, case-control Setting: United States Veterans Administration Healthcare System Participants: Biologic males enrolled in the MVP biobank with genomic identification of an additional X or Y chromosome (cases); controls matched 1:5 on sex, age, and genetic ancestry Main Outcome(s) and Measure(s): Prevalence of men with SCAs from genomic analysis; clinical SCA diagnosis; Charlson Comorbidity Index (CCI); rates of outpatient, inpatient, and emergency encounters per year; self-reported health outcomes; standardized mortality ratio (SMR) Results: An additional X or Y chromosome was present in 145 and 125 per 100,000 males in the MVP, respectively, with the highest prevalence among men with European and East Asian ancestry. At a mean age of 61 years, 74% of male veterans with 47,XXY and &gt;99% with 47,XYY remained undiagnosed. Individuals with 47,XXY (n=862) and 47,XYY (n=747) had similar military service history, all-cause SMR, and age of death compared to matched controls. CCI and healthcare utilization were higher among individuals with SCA, while several measures of HRQoL were lower. Men with a clinical diagnosis of 47,XXY had higher healthcare utilization but lower comorbidity score compared to those undiagnosed. Conclusion and Relevance: One in 370 males in the MVP cohort have SCA, a prevalence comparable to estimates in the general population. While these men have successfully served in the military, they have higher morbidity and report poorer HRQoL with aging. Longer longitudinal follow-up of this sample will be informative for clinical and patient-reported outcomes, the role of ancestry, and mortality statistics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.07.23284293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.07.23284293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.07.23284293">
        <p class="paperTitle">Fine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.07.23284293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.07.23284293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, K.; Longchamps, R. J.; Pardinas, A. F.; Yu, M.; Chen, T.-T.; Lin, S.-C.; Chen, Y.; Lam, M.; Liu, R.; Xia, Y.; Guo, Z.; Shi, W.; Shen, C.; The Schizophrenia Workgroup of Psychiatric Genomics Consortium,  ; Daly, M. J.; Neale, B.; Feng, Y.-C. A.; Lin, Y.-F.; Chen, C.-Y.; O&#39;Donovan, M.; Ge, T.; Huang, H.</p>
        <p class="info">Score: 24.8, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.07.23284293' target='https://doi.org/10.1101/2023.01.07.23284293'> 10.1101/2023.01.07.23284293</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) of human complex traits or diseases often implicate genetic loci that span hundreds or thousands of genetic variants, many of which have similar statistical significance. While statistical fine-mapping in individuals of European ancestries has made important discoveries, cross-population fine-mapping has the potential to improve power and resolution by capitalizing on the genomic diversity across ancestries. Here we present SuSiEx, an accurate and computationally efficient method for cross-population fine-mapping, which builds on the single-population fine-mapping framework, Sum of Single Effects (SuSiE). SuSiEx integrates data from an arbitrary number of ancestries, explicitly models population-specific allele frequencies and LD patterns, accounts for multiple causal variants in a genomic region, and can be applied to GWAS summary statistics. We comprehensively evaluated SuSiEx using simulations, a range of quantitative traits measured in both UK Biobank and Taiwan Biobank, and schizophrenia GWAS across East Asian and European ancestries. In all evaluations, SuSiEx fine-mapped more association signals, produced smaller credible sets and higher posterior inclusion probability (PIP) for putative causal variants, and captured population-specific causal variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23291992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23291992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23291992">
        <p class="paperTitle">Exome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23291992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23291992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.</p>
        <p class="info">Score: 25.2, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23291992' target='https://doi.org/10.1101/2023.06.29.23291992'> 10.1101/2023.06.29.23291992</a></p>
        <p class="abstract">Exome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF &lt; 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( &lt; 1.68 x 10-7) and 17 nominally significant ( &lt; 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR &lt; 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.23292336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.23292336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.23292336">
        <p class="paperTitle">Bayesian genome-wide TWAS with reference transcriptomic data of brain and blood tissues identified 93 risk genes for Alzheimer&#39;s disease dementia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.23292336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.23292336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, S.; Yang, J.</p>
        <p class="info">Score: 3.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.23292336' target='https://doi.org/10.1101/2023.07.06.23292336'> 10.1101/2023.07.06.23292336</a></p>
        <p class="abstract">BackgroundTranscriptome-wide association study (TWAS) is an influential tool for identifying novel genes associated with complex diseases, where their genetic effects may be mediated through transcriptome. TWAS utilizes reference genetic and transcriptomic data to estimate genetic effect sizes on expression quantitative traits of target genes (i.e., effect sizes of a broad sense of expression quantitative trait loci, eQTL). These estimated effect sizes are then employed as variant weights in burden gene-based association test statistics, facilitating the mapping of risk genes for complex diseases with genome-wide association study (GWAS) data. However, most existing TWAS of Alzheimers disease (AD) dementia have primarily focused on cis-eQTL, disregarding potential trans-eQTL. To overcome this limitation, we applied the Bayesian Genome-wide TWAS (BGW-TWAS) method which incorporated both cis- and trans-eQTL of brain and blood tissues to enhance mapping risk genes for AD dementia.

MethodsWe first applied BGW-TWAS to the Genotype-Tissue Expression (GTEx) V8 dataset to estimate cis- and trans-eQTL effect sizes of the prefrontal cortex, cortex, and whole blood tissues. Subsequently, estimated eQTL effect sizes were integrated with the summary data of the most recent GWAS of AD dementia to obtain BGW-TWAS (i.e., gene-based association test) p-values of AD dementia per tissue type. Finally, we used the aggregated Cauchy association test to combine TWAS p-values across three tissues to obtain omnibus TWAS p-values per gene.

ResultsWe identified 37 genes in prefrontal cortex, 55 in cortex, and 51 in whole blood that were significantly associated with AD dementia. By combining BGW-TWAS p-values across these three tissues, we obtained 93 significant risk genes including 29 genes primarily due to trans-eQTL and 50 novel genes. Utilizing protein-protein interaction network and phenotype enrichment analyses with these 93 significant risk genes, we detected 5 functional clusters comprised of both known and novel AD risk genes and 7 enriched phenotypes.

ConclusionWe applied BGW-TWAS and aggregated Cauchy test methods to integrate both cis- and trans-eQTL data of brain and blood tissues with GWAS summary data to identify risk genes of AD dementia. The risk genes we identified provide novel insights into the underlying biological pathways implicated in AD dementia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291611">
        <p class="paperTitle">PA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium,  ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.</p>
        <p class="info">Score: 7.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291611' target='https://doi.org/10.1101/2023.06.23.23291611'> 10.1101/2023.06.23.23291611</a></p>
        <p class="abstract">Large biobank samples provide an opportunity to integrate broad phenotyping, familial records, and molecular genetics data to study complex traits and diseases. We introduce Pearson-Aitken Family Genetic Risk Scores (PA-FGRS), a new method for estimating disease liability from patterns of diagnoses in extended, age-censored genealogical records. We then apply the method to study a paradigmatic complex disorder, Major Depressive Disorder (MDD), using the iPSYCH2015 case-cohort study of 30,949 MDD cases, 39,655 random population controls, and more than 2 million relatives. We show that combining PA-FGRS liabilities estimated from family records with molecular genotypes of probands improves the three lines of inquiry. Incorporating PA-FGRS liabilities improves classification of MDD over and above polygenic scores, identifies robust genetic contributions to clinical heterogeneity in MDD associated with comorbidity, recurrence, and severity, and can improve the power of genome-wide association studies (GWAS). Our method is flexible and easy to use and our study approaches are generalizable to other data sets and other complex traits and diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292193">
        <p class="paperTitle">The NYCKidSeq randomized controlled trial: Impact of GUIA digitally enhanced genetic counseling in racially and ethnically diverse families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suckiel, S. A.; Kelly, N. R.; Odgis, J. A.; Gallagher, K. M.; Sebastin, M.; Bonini, K. E.; Marathe, P. N.; Brown, K.; Di Biase, M.; Ramos, M.; Rodriguez, J. E.; Scarimbolo, L.; Insel, B. J.; Ferar, K. D. M.; Zinberg, R. E.; Diaz, G. A.; Greally, J. M.; Abul-Husn, N. S.; Bauman, L. J.; Gelb, B. D.; Horowitz, C. R.; Wasserstein, M. P.; Kenny, E. E.</p>
        <p class="info">Score: 9.5, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292193' target='https://doi.org/10.1101/2023.07.05.23292193'> 10.1101/2023.07.05.23292193</a></p>
        <p class="abstract">BackgroundDigital solutions are needed to support rapid increases in the application of genetic and genomic tests (GT) in diverse clinical settings and patient populations. We developed GUIA, a bi-lingual web-based platform that facilitates disclosure of GT results. The NYCKidSeq randomized controlled trial evaluated GUIAs impact on understanding of GT results.

MethodsNYCKidSeq enrolled diverse children with neurologic, cardiac, and immunologic conditions who underwent GT. Families were randomized to genetic counseling with GUIA (intervention) or standard of care (SOC) genetic counseling for results disclosure. Parents/legal guardians (participants) completed surveys at baseline, post-results disclosure, and 6-months later. Survey measures assessed the primary study outcomes of perceived understanding of and confidence in explaining their childs GT results and the secondary outcome of objective understanding. We used regression models to evaluate the association between the intervention and the study outcomes.

ResultsThe analysis included 551 participants, 270 in the GUIA arm and 281 in SOC. Participants mean age was 41.1 years and 88.6% were mothers. Most participants were Hispanic/Latino(a) (46.3%), White/European American (24.5%), or Black/African American (15.8%). Participants in the GUIA arm had significantly higher perceived understanding post-results (OR=2.8, CI[1.004,7.617], P=0.049) and maintained higher objective understanding over time (OR=1.1, CI[1.004, 1.127], P=0.038) compared to those in the SOC arm. There was no impact on perceived confidence. Hispanic/Latino(a) individuals in the GUIA arm maintained higher perceived understanding (OR=3.9, CI[1.6, 9.3], P=0.003), confidence (OR=2.7, CI[1.021, 7.277], P=0.046), and objective understanding (OR=1.1, CI[1.009, 1.212], P=0.032) compared to SOC.

ConclusionsThis trial demonstrates that GUIA positively impacts understanding of GT results in diverse parents of children with suspected genetic conditions. These findings build a case for utilizing GUIA to deliver complex and often ambiguous genetic results. Continued development and evaluation of digital applications in diverse populations are critical for equitably scaling GT offerings in specialty clinics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292214">
        <p class="paperTitle">Genetic structure of major depression symptoms across clinical and community cohorts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adams, M. J.; Thorp, J. G.; Jermy, B. S.; Kwong, A. S. F.; Koiv, K.; Grotzinger, A. D.; Nivard, M. G.; Marshall, S.; Milaneschi, Y.; Baune, B. T.; Mueller-Myhsok, B.; Penninx, B. W.; Boomsma, D. I.; Levinson, D. F.; Breen, G.; Pistis, G.; Grabe, H. J.; Tiemeier, H.; Berger, K.; Rietschel, M.; Magnusson, P. K.; Uher, R.; Hamilton, S. P.; Lucae, S.; Lehto, K.; Li, Q. S.; Byrne, E. M.; Hickie, I. B.; Martin, N. G.; Medland, S. E.; Wray, N. R.; Tucker-Drob, E. M.; Estonian Biobank Research Team,  ; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium,  ; Lewis, C. M.; McI</p>
        <p class="info">Score: 2.2, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292214' target='https://doi.org/10.1101/2023.07.05.23292214'> 10.1101/2023.07.05.23292214</a></p>
        <p class="abstract">Diagnostic criteria for major depressive disorder allow for heterogeneous symptom profiles but genetic analysis of major depressive symptoms has the potential to identify clinical and aetiological subtypes. There are several challenges to integrating symptom data from genetically-informative cohorts, such as sample size differences between clinical and community cohorts and various patterns of missing data. We conducted genome-wide association studies of major depressive symptoms in three clinical cohorts that were enriched for affected participants (Psychiatric Genomics Consortium, Australian Genetics of Depression Study, Generation Scotland) and three community cohorts (Avon Longitudinal Study of Parents and Children, Estonian Biobank, and UK Biobank). We fit a series of confirmatory factor models with factors that accounted for how symptom data was sampled and then compared alternative models with different symptom factors. The best fitting model had a distinct factor for Appetite/Weight symptoms and an additional measurement factor that accounted for missing data patterns in the community cohorts (use of Depression and Anhedonia as gating symptoms). The results show the importance of assessing the directionality of symptoms (such as hypersomnia versus insomnia) and of accounting for study and measurement design when meta-analysing genetic association data.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
