<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Chromosome 20p11.2 deletions cause congenital hyperinsulinism via the likely disruption of FOXA2
Authors: Laver, T. W.; Wakeling, M. N.; Caswell, R. C.; Bunce, B.; Yau, D.; Houghton, J. A.; Hopkins, J. J.; Weedon, M. N.; Saraff, V.; Kershaw, M.; Honey, E.; Murphy, N.; Giri, D.; Nath, S.; Saredo, A. T.; Banerjee, I.; Hussain, K.; Owens, N. D.; Flanagan, S. E.
Score: 23.3, Published: 2023-08-21 DOI: 10.1101/2023.08.16.23294161
Persistent congenital hyperinsulinism (HI) is a rare genetically heterogeneous condition characterised by dysregulated insulin secretion leading to life-threatening hypoglycaemia.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Chromosome 20p11.2 deletions cause congenital hyperinsulinism via the likely disruption of FOXA2
Authors: Laver, T. W.; Wakeling, M. N.; Caswell, R. C.; Bunce, B.; Yau, D.; Houghton, J. A.; Hopkins, J. J.; Weedon, M. N.; Saraff, V.; Kershaw, M.; Honey, E.; Murphy, N.; Giri, D.; Nath, S.; Saredo, A. T.; Banerjee, I.; Hussain, K.; Owens, N. D.; Flanagan, S. E.
Score: 23.3, Published: 2023-08-21 DOI: 10.1101/2023.08.16.23294161
Persistent congenital hyperinsulinism (HI) is a rare genetically heterogeneous condition characterised by dysregulated insulin secretion leading to life-threatening hypoglycaemia." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Chromosome 20p11.2 deletions cause congenital hyperinsulinism via the likely disruption of FOXA2
Authors: Laver, T. W.; Wakeling, M. N.; Caswell, R. C.; Bunce, B.; Yau, D.; Houghton, J. A.; Hopkins, J. J.; Weedon, M. N.; Saraff, V.; Kershaw, M.; Honey, E.; Murphy, N.; Giri, D.; Nath, S.; Saredo, A. T.; Banerjee, I.; Hussain, K.; Owens, N. D.; Flanagan, S. E.
Score: 23.3, Published: 2023-08-21 DOI: 10.1101/2023.08.16.23294161
Persistent congenital hyperinsulinism (HI) is a rare genetically heterogeneous condition characterised by dysregulated insulin secretion leading to life-threatening hypoglycaemia."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Chromosome 20p11.2 deletions cause congenital hyperinsulinism via the likely disruption of FOXA2\nAuthors: Laver, T. W.; Wakeling, M. N.; Caswell, R. C.; Bunce, B.; Yau, D.; Houghton, J. A.; Hopkins, J. J.; Weedon, M. N.; Saraff, V.; Kershaw, M.; Honey, E.; Murphy, N.; Giri, D.; Nath, S.; Saredo, A. T.; Banerjee, I.; Hussain, K.; Owens, N. D.; Flanagan, S. E.\nScore: 23.3, Published: 2023-08-21 DOI: 10.1101/2023.08.16.23294161\nPersistent congenital hyperinsulinism (HI) is a rare genetically heterogeneous condition characterised by dysregulated insulin secretion leading to life-threatening hypoglycaemia.",
  "keywords": [
    
  ],
  "articleBody": " Chromosome 20p11.2 deletions cause congenital hyperinsulinism via the likely disruption of FOXA2\nAuthors: Laver, T. W.; Wakeling, M. N.; Caswell, R. C.; Bunce, B.; Yau, D.; Houghton, J. A.; Hopkins, J. J.; Weedon, M. N.; Saraff, V.; Kershaw, M.; Honey, E.; Murphy, N.; Giri, D.; Nath, S.; Saredo, A. T.; Banerjee, I.; Hussain, K.; Owens, N. D.; Flanagan, S. E.\nScore: 23.3, Published: 2023-08-21 DOI: 10.1101/2023.08.16.23294161\nPersistent congenital hyperinsulinism (HI) is a rare genetically heterogeneous condition characterised by dysregulated insulin secretion leading to life-threatening hypoglycaemia. For up to 50% of affected individuals screening of the known HI genes does not identify a disease-causing variant. Large deletions have previously been used to identify novel regulatory regions causing HI. Here, we used genome sequencing to search for novel large (\u003e1Mb) deletions in 180 probands with HI of unknown cause and replicated our findings in a large cohort of 883 genetically unsolved individuals with HI using off-target copy number variant calling from targeted gene panels. We identified overlapping heterozygous deletions in five individuals (range 3-8 Mb) spanning chromosome 20p11.2. The pancreatic beta-cell transcription factor gene, FOXA2, a known cause of HI was deleted in two of the five individuals. In the remaining three, we found a minimal deleted region of 2.4 Mb adjacent to FOXA2 that encompasses multiple non-coding regulatory elements that are in conformational contact with FOXA2. Our data suggests that the deletions in these three patients may cause disease through the dysregulation of FOXA2 expression. These findings provide new insights into the regulation of FOXA2 in the beta-cell and confirm an aetiological role for chromosome 20p deletions in syndromic HI.\nPathogenic paralogous variants can be used to apply the ACMG PS1 and PM5 variant interpretation criteria\nAuthors: Brünger, T.; Ivaniuk, A.; Perez-Palma, E.; Montanucci, L.; Cohen, S.; Smith, L.; Parthasarathy, S.; Helbig, I.; Nothnagel, M.; May, P.; Lal, D.\nScore: 7.3, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23294353\nPurpose The majority of missense variants in clinical genetic tests are classified as variants of uncertain significance. Broadening the evidence of the PS1 and PM5 criteria has the potential to increase conclusive variant interpretation. Methods We hypothesized that incorporation of pathogenic missense variants in conserved residues across paralogous genes can increase the number of variants where ACMG PS1/PM5 criteria can be applied. We mapped over 2.5 million pathogenic and general population variants from ClinVar, HGMD, and gnomAD databases onto 9,990 genes and aligned these by gene families. Subsequently, we developed a novel framework to extend PS1/PM5 by incorporating pathogenic paralogous variants annotations (para-PS1/PM5). Results We demonstrate that para-PS1/PM5 criteria increase the number of classifiable amino acids 3.6-fold compared to PS1 and PM5. Across all gene families with at least two disease-associated genes, the calculated likelihood ratios suggest moderate evidence for pathogenicity. Moreover, for 36 genes, the extended para-PS1/PM5 criteria reach strong evidence level. Conclusion We show that single pathogenic paralogous variants incorporation at paralogous protein positions increases the applicability of the PS1 and PM5 criteria, likely leading to a reduction of variants of uncertain significance across many monogenic disorders. Future iterations of the ACMG guidelines may consider para-PS1 and para-PM5.\nIdentification of a de novo mutation in TLK1 associated with a neurodevelopmental disorder and immunodeficiency\nAuthors: Villamor-Paya, M.; Sanchiz-Calvo, M.; Smak, J.; Pais, L.; Sud, M.; Shankavaram, U.; Lovgren, A. K.; Austin-Tse, C.; Ganesh, V. S.; Gay, M.; Vilaseca, M.; Arauz-Garofalo, G.; Palenzuela, L.; VanNoy, G.; O'Donnell-Luria, A.; Stracker, T. H.\nScore: 3.8, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23294267\nBackground The Tousled-like kinases 1 and 2 (TLK1/TLK2) regulate DNA replication, repair and chromatin maintenance. TLK2 variants are associated with Intellectual Disability, Autosomal Dominant 57 (MRD57), a neurodevelopmental disorder (NDD) characterized by intellectual disability (ID), autism spectrum disorder (ASD) and microcephaly. Several TLK1 variants have been reported in NDDs but their functional significance is unknown. Methods A male patient presenting with ID, seizures, global developmental delay, hypothyroidism, and primary immunodeficiency was determined to have a novel, heterozygous variant in TLK1 (c.1435C\u003eG, p.Q479E) by genome sequencing (GS). Single cell gel electrophoresis, western blot, flow cytometry and RNA-seq were performed in patient-derived lymphoblast cell lines. In silico, biochemical and proteomic analysis were used to determine the functional impact of the p.Q479E variant and previously reported NDD-associated TLK1 variant, p.M566T. Results Transcriptome sequencing in patient-derived cells confirmed expression of TLK1 transcripts carrying the p.Q479E variant and revealed alterations in genes involved in class switch recombination and cytokine signaling. Cells expressing the p.Q479E variant exhibited reduced cytokine responses and higher levels of spontaneous DNA damage but not increased sensitivity to radiation or DNA repair defects. The p.Q479E and p.M566T variants impaired kinase activity but did not strongly alter localization or proximal protein interactions. Conclusion Our study provides the first functional characterization of TLK1 variants associated with NDDs and suggests potential involvement in central nervous system and immune system development. Our results indicate that, like TLK2 variants, TLK1 variants may impact development in multiple tissues and should be considered in the diagnosis of rare NDDs.\nInvestigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores\nAuthors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes \u0026 Health Research Team, ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.\nScore: 2.1, Published: 2023-08-25 DOI: 10.1101/2023.08.23.23294497\nObjective: Correct classification of type 1 (T1D) and type 2 diabetes (T2D) is challenging due to overlapping clinical features and the increasingly early onset of T2D, particularly in South Asians. We used polygenic risk scores (PRSs) in a British Bangladeshi and Pakistani population with diabetes to estimate the proportion and misclassification rate of T1D in insulin-treated individuals with ambiguous features. Research Design and Methods: Using linked health record from the Genes \u0026 Health cohort (n=38,344) we defined four groups: 31 T1D cases, 1,842 T2D cases, and after excluding these, 839 insulin-treated individuals with ambiguous features and 5,174 controls. Combining these with 307 confirmed T1D cases and 307 controls from India, we calculated ancestry-corrected PRSs for T1D and T2D, with which we estimated the proportion of T1D cases within the ambiguous group and evaluated misclassification. Results: We estimated that the prevalence of T1D was ~6% within the ambiguous group, or ~4.5% within the subset who had T2D codes in their health records. We saw no significant association between the T1D or T2D PRS and BMI at diagnosis, time to insulin, or the presence of T1D or T2D diagnostic codes amongst the T2D or ambiguous cases. Conclusions: We estimate that 4.5% of British Pakistanis and Bangladeshis with diabetes who are treated with insulin and have ambiguous clinical features have been classified incorrectly in their health records, and in fact have T1D. These findings have significant implications for the delivery of optimal clinical care at population level.\nNeuroimaging-AI Endophenotypes of Brain Diseases in the General Population: Towards a Dimensional System of Vulnerability\nAuthors: WEN, J.; Skampardoni, I.; Tian, Y. E.; Yang, Z.; cui, y.; erus, g.; hwang, g.; Varol, E.; boquetipujadas, a.; Chand, G.; Nasrallah, I. M.; Satterthwaite, T.; Shou, H.; Shen, L.; Toga, A. W.; Zalesky, A.; Davatzikos, C.\nScore: 2.0, Published: 2023-08-24 DOI: 10.1101/2023.08.16.23294179\nDisease heterogeneity poses a significant challenge for precision diagnostics in both clinical and subclinical stages. Recent work leveraging artificial intelligence (AI) has offered promise to dissect this heterogeneity by identifying complex intermediate phenotypes, herein called dimensional neuroimaging endophenotypes (DNEs), which subtype various neurologic and neuropsychiatric diseases. We investigate the presence of nine such DNEs derived from independent yet harmonized studies on Alzheimer's disease (AD1-2), autism spectrum disorder (ASD1,2,3), late-life depression (LLD1,2), and schizophrenia (SCZ1,2), in the general population of 39,178 participants in the UK Biobank study. Phenome-wide associations revealed prominent associations between the nine DNEs and phenotypes related to the brain and other human organ systems. This phenotypic landscape aligns with the SNP-phenotype genome-wide associations, revealing 31 genomic loci associated with the nine DNEs (Bonferroni corrected P-value \u003c 5xE-8/9). The DNEs exhibited significant genetic correlations, colocalization, and causal relationships with multiple human organ systems and chronic diseases. A causal effect (odds ratio=1.25 [1.11, 1.40], P-value=8.72x10-4) was established from AD2, characterized by focal medial temporal lobe atrophy, to AD. The nine DNEs and their polygenic risk scores significantly improved the prediction accuracy for 14 systemic disease categories and mortality. These findings underscore the potential of the nine DNEs to identify individuals at a high risk of developing the four brain diseases during preclinical stages for precision diagnostics. All results are publicly available at: http://labs.loni.usc.edu/medicine/.\nRare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank\nAuthors: Fenner, E.; Holmans, P.; O'Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.\nScore: 9.8, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294074\nImpairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.\nEvaluation of optical genome mapping in clinical genetic testing of facioscapulohumeral muscular dystrophy\nAuthors: Kovanda, A.; Lovrecic, L.; Rudolf, G.; Babic Bozovic, I.; Jaklic, H.; Leonardis, L.; Peterlin, B.\nScore: 1.9, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23292816\nBackgroundFacioscapulohumeral muscular dystrophy (FSHD) is the third most common hereditary muscular dystrophy, caused by the contraction of the D4Z4 repeats on the permissive 4qA haplotype on chromosome 4, resulting in the faulty expression of the DUX4 gene. Traditional diagnostics is based on Southern blot, a time- and effort-intensive method that can be affected by single nucleotide (SNV), and copy number variants (CNV), as well as by the similarity of the D4Z4 repeats located on chromosome 10. We aimed to evaluate optical genome mapping (OGM) as an alternative molecular diagnostic method for the detection of FSHD. MethodsWe first performed optical genome mapping with EnFocus FSHD analysis using DLE-1 labeling and the Saphyr instrument, for our patients with inconclusive diagnostic Southern blot results, negative FSHD2 results, and clinically evident FSHD. Secondly, we performed OGM in parallel with the classical Southern blot analysis for our prospectively collected new FSHD cases. Finally, we used panel exome sequencing to confirm FSHD2. ResultsBy using OGM we could resolve two cases with diagnostically inconclusive Southern blot results as having shortened D4Z4 repeats on the permissive 4qA allele, consistent with their clinical presentation. Results of the prospectively collected patients tested in parallel using Southern blot and OGM showed full concordance, showing OGM to be a useful alternative to the classical Southern blot method for detecting FSHD1. For a patient showing clinical FSHD but no shortened D4Z4 repeat on the 4qA allele using OGM or Southern blot, a likely pathogenic variant in SMCHD1 was detected using exome sequencing, confirming FSHD2. OGM and panel exome sequencing can be used consecutively to detect FSHD2.\nCritical assessment of variant prioritization methods for rare disease diagnosis within the Rare Genomes Project\nAuthors: Stenton, S. L.; O'Leary, M.; Lemire, G.; VanNoy, G. E.; DiTroia, S.; Ganesh, V. S.; Groopman, E.; O'Heir, E.; Mangilog, B.; Osei-Owusu, I.; Pais, L. S.; Serrano, J.; Singer-Berk, M.; Weisburd, B.; Wilson, M.; Austin-Tse, C.; Abdelhakim, M.; Althagafi, A.; Babbi, G.; Bellazzi, R.; Bovo, S.; Carta, M. G.; Casadio, R.; Coenen, P.-J.; De Paoli, F.; Floris, M.; Gajapathy, M.; Hoehndorf, R.; Jacobsen, J. O. B.; Joseph, T.; Kamandula, A.; Katsonis, P.; Kint, C.; Lichtarge, O.; Limongelli, I.; Lu, Y.; Magni, P.; Mamidi, T. K. K.; Martelli, P. L.; Mulargia, M.; Nicora, G.; Nykamp, K.; Pejaver, V.; Peng\nScore: 9.8, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293212\nBackgroundA major obstacle faced by rare disease families is obtaining a genetic diagnosis. The average \"diagnostic odyssey\" lasts over five years, and causal variants are identified in under 50%. The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing (GS) for diagnosis and gene discovery. Families are consented for sharing of sequence and phenotype data with researchers, allowing development of a Critical Assessment of Genome Interpretation (CAGI) community challenge, placing variant prioritization models head-to-head in a real-life clinical diagnostic setting. MethodsPredictors were provided a dataset of phenotype terms and variant calls from GS of 175 RGP individuals (65 families), including 35 solved training set families, with causal variants specified, and 30 test set families (14 solved, 16 unsolved). The challenge tasked teams with identifying the causal variants in as many test set families as possible. Ranked variant predictions were submitted with estimated probability of causal relationship (EPCR) values. Model performance was determined by two metrics, a weighted score based on rank position of true positive causal variants and maximum F-measure, based on precision and recall of causal variants across EPCR thresholds. ResultsSixteen teams submitted predictions from 52 models, some with manual review incorporated. Top performing teams recalled the causal variants in up to 13 of 14 solved families by prioritizing high quality variant calls that were rare, predicted deleterious, segregating correctly, and consistent with reported phenotype. In unsolved families, newly discovered diagnostic variants were returned to two families following confirmatory RNA sequencing, and two prioritized novel disease gene candidates were entered into Matchmaker Exchange. In one example, RNA sequencing demonstrated aberrant splicing due to a deep intronic indel in ASNS, identified in trans with a frameshift variant, in an unsolved proband with phenotype overlap with asparagine synthetase deficiency. ConclusionsBy objective assessment of variant predictions, we provide insights into current state-of-the-art algorithms and platforms for genome sequencing analysis for rare disease diagnosis and explore areas for future optimization. Identification of diagnostic variants in unsolved families promotes synergy between researchers with clinical and computational expertise as a means of advancing the field of clinical genome interpretation.\nCirculating ADAM17 is associated with COVID-19 severity\nAuthors: Pan, M.; Goncalves, I.; Edsfeldt, A.; Sun, J.; Sward, P.\nScore: 1.0, Published: 2023-08-24 DOI: 10.1101/2023.08.23.23294465\nAbstract Background ADAM17 are emerging as an important role in the severe outcomes of COVID-19. This study aims to characterize causal relationship between ADAM17 and COVID-19. Methods Using mendelian randomization analyses, we examined the causal effects for circulating ADAM17 on COVID-19 outcomes using summary statistics from large genome wide association studies of ADAM17 (up to 35 559 individuals) from the Icelandic Cancer Project and deCODE genetics, critical COVID-19 (cases:13 769; controls:1 072 442), hospitalized COVID-19 (cases:32 519; controls: 2 062 805) and SARS-CoV-2 infection (cases:122 616; controls:2 475 240) from the COVID-19 Host Genetics Initiative. Results Mendelian randomization analyses demonstrated that 1 standard deviation increase of genetically determined circulating ADAM17 at extracellular domain were associated with increasing risk of developing critical COVID-19 (odds ratio [OR]=1.26, 95% CI 1.03-1.55). Multivariable MR analysis suggested a direct causal role of circulating ADAM17 at extracellular domain on the risk of critical COVID-19 (OR=1.09; 95% CI 1.01-1.17), accounting for body mass index. Casual effects for the cytoplasmic domain of ADAM17 on COVID-19 were not observed. Conclusion Our results suggest that the increased circulating ADAM17 at extracellular domain are associated with a high risk of critical COVID-19 strengthening that of ADAM17 may contribute to the risk stratification and a therapeutic option for severe COVID-19.\nStudy of the protective OAS1 rs10774671-G allele against severe COVID-19 in Moroccans suggests a North African origin for Neanderthals\nAuthors: El Youssfi, F. Z.; Haroun, A. E.; Nebhani, C.; Belayachi, J.; Askander, O.; El Fahime, E.; Fares, H.; Ennibi, K.; Abouqal, R.; Razine, R.; Bouhouche, A.\nScore: 1.0, Published: 2023-08-22 DOI: 10.1101/2023.08.19.23294314\nBackgroundThe clinical presentation of COVID-19 has shown high variability between individuals, which is partly due to genetic factors. The OAS1/2/3 cluster was found to be strongly associated with COVID-19 severity. We aimed to examine this locus for the occurrence of the critical variant, rs10774671, and its respective haplotype blocks within the Moroccan population. MethodsThe frequency of SNPs at the cluster of OAS immunity genes was assessed from an in-house database in 157 unrelated individuals of Moroccan origin. The OAS1 exon 6 was sequenced by Sangers method in 71 asymptomatic/mild and 74 moderate/severe individuals positive for SARS-CoV-2. Genotypic, allelic, and haplotype frequencies of three SNPs were compared between the two groups. Finally, males in our COVID-19 series were genotyped for the Berber-specific marker E-M81. ResultsThe prevalence of the OAS1 rs10774671-G allele in present-day Moroccans was 40.4%, close to that of Europeans. However, it was found equally on both the Neanderthal GGG haplotype and the African GAC haplotype with a frequency of 20% each. These two haplotypes, and hence the rs10774671-G allele, were significantly associated with the protection against severe COVID-19 (p = 0.034, p = 0.041, and p = 0.008 respectively). Surprisingly, among Berber men, the African haplotype was absent while the prevalence of the Neanderthal haplotype was close to that of Europeans. ConclusionThe protective rs10774671-G allele of OAS1 was found only in the Neanderthal haplotype in Berbers, the indigenous people of North Africa, suggesting that this region may have served as the stepping-stone for the passage of the hominids to the other continents.\n",
  "wordCount" : "2936",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.23294161">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.23294161" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.23294161">
        <p class="paperTitle">Chromosome 20p11.2 deletions cause congenital hyperinsulinism via the likely disruption of FOXA2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.23294161" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.23294161" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laver, T. W.; Wakeling, M. N.; Caswell, R. C.; Bunce, B.; Yau, D.; Houghton, J. A.; Hopkins, J. J.; Weedon, M. N.; Saraff, V.; Kershaw, M.; Honey, E.; Murphy, N.; Giri, D.; Nath, S.; Saredo, A. T.; Banerjee, I.; Hussain, K.; Owens, N. D.; Flanagan, S. E.</p>
        <p class="info">Score: 23.3, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.23294161' target='https://doi.org/10.1101/2023.08.16.23294161'> 10.1101/2023.08.16.23294161</a></p>
        <p class="abstract">Persistent congenital hyperinsulinism (HI) is a rare genetically heterogeneous condition characterised by dysregulated insulin secretion leading to life-threatening hypoglycaemia. For up to 50% of affected individuals screening of the known HI genes does not identify a disease-causing variant. Large deletions have previously been used to identify novel regulatory regions causing HI. Here, we used genome sequencing to search for novel large (&gt;1Mb) deletions in 180 probands with HI of unknown cause and replicated our findings in a large cohort of 883 genetically unsolved individuals with HI using off-target copy number variant calling from targeted gene panels. We identified overlapping heterozygous deletions in five individuals (range 3-8 Mb) spanning chromosome 20p11.2. The pancreatic beta-cell transcription factor gene, FOXA2, a known cause of HI was deleted in two of the five individuals. In the remaining three, we found a minimal deleted region of 2.4 Mb adjacent to FOXA2 that encompasses multiple non-coding regulatory elements that are in conformational contact with FOXA2. Our data suggests that the deletions in these three patients may cause disease through the dysregulation of FOXA2 expression. These findings provide new insights into the regulation of FOXA2 in the beta-cell and confirm an aetiological role for chromosome 20p deletions in syndromic HI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23294353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23294353" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23294353">
        <p class="paperTitle">Pathogenic paralogous variants can be used to apply the ACMG PS1 and PM5 variant interpretation criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23294353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23294353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brünger, T.; Ivaniuk, A.; Perez-Palma, E.; Montanucci, L.; Cohen, S.; Smith, L.; Parthasarathy, S.; Helbig, I.; Nothnagel, M.; May, P.; Lal, D.</p>
        <p class="info">Score: 7.3, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23294353' target='https://doi.org/10.1101/2023.08.22.23294353'> 10.1101/2023.08.22.23294353</a></p>
        <p class="abstract">Purpose The majority of missense variants in clinical genetic tests are classified as variants of uncertain significance. Broadening the evidence of the PS1 and PM5 criteria has the potential to increase conclusive variant interpretation. Methods We hypothesized that incorporation of pathogenic missense variants in conserved residues across paralogous genes can increase the number of variants where ACMG PS1/PM5 criteria can be applied. We mapped over 2.5 million pathogenic and general population variants from ClinVar, HGMD, and gnomAD databases onto 9,990 genes and aligned these by gene families. Subsequently, we developed a novel framework to extend PS1/PM5 by incorporating pathogenic paralogous variants annotations (para-PS1/PM5). Results We demonstrate that para-PS1/PM5 criteria increase the number of classifiable amino acids 3.6-fold compared to PS1 and PM5. Across all gene families with at least two disease-associated genes, the calculated likelihood ratios suggest moderate evidence for pathogenicity. Moreover, for 36 genes, the extended para-PS1/PM5 criteria reach strong evidence level. Conclusion We show that single pathogenic paralogous variants incorporation at paralogous protein positions increases the applicability of the PS1 and PM5 criteria, likely leading to a reduction of variants of uncertain significance across many monogenic disorders. Future iterations of the ACMG guidelines may consider para-PS1 and para-PM5.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23294267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23294267" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23294267">
        <p class="paperTitle">Identification of a de novo mutation in TLK1 associated with a neurodevelopmental disorder and immunodeficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23294267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23294267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Villamor-Paya, M.; Sanchiz-Calvo, M.; Smak, J.; Pais, L.; Sud, M.; Shankavaram, U.; Lovgren, A. K.; Austin-Tse, C.; Ganesh, V. S.; Gay, M.; Vilaseca, M.; Arauz-Garofalo, G.; Palenzuela, L.; VanNoy, G.; O&#39;Donnell-Luria, A.; Stracker, T. H.</p>
        <p class="info">Score: 3.8, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23294267' target='https://doi.org/10.1101/2023.08.22.23294267'> 10.1101/2023.08.22.23294267</a></p>
        <p class="abstract">Background The Tousled-like kinases 1 and 2 (TLK1/TLK2) regulate DNA replication, repair and chromatin maintenance. TLK2 variants are associated with Intellectual Disability, Autosomal Dominant 57 (MRD57), a neurodevelopmental disorder (NDD) characterized by intellectual disability (ID), autism spectrum disorder (ASD) and microcephaly. Several TLK1 variants have been reported in NDDs but their functional significance is unknown. Methods A male patient presenting with ID, seizures, global developmental delay, hypothyroidism, and primary immunodeficiency was determined to have a novel, heterozygous variant in TLK1 (c.1435C&gt;G, p.Q479E) by genome sequencing (GS). Single cell gel electrophoresis, western blot, flow cytometry and RNA-seq were performed in patient-derived lymphoblast cell lines. In silico, biochemical and proteomic analysis were used to determine the functional impact of the p.Q479E variant and previously reported NDD-associated TLK1 variant, p.M566T. Results Transcriptome sequencing in patient-derived cells confirmed expression of TLK1 transcripts carrying the p.Q479E variant and revealed alterations in genes involved in class switch recombination and cytokine signaling. Cells expressing the p.Q479E variant exhibited reduced cytokine responses and higher levels of spontaneous DNA damage but not increased sensitivity to radiation or DNA repair defects. The p.Q479E and p.M566T variants impaired kinase activity but did not strongly alter localization or proximal protein interactions. Conclusion Our study provides the first functional characterization of TLK1 variants associated with NDDs and suggests potential involvement in central nervous system and immune system development. Our results indicate that, like TLK2 variants, TLK1 variants may impact development in multiple tissues and should be considered in the diagnosis of rare NDDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.23294497">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.23294497" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.23294497">
        <p class="paperTitle">Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.23294497" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.23294497" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes &amp; Health Research Team,  ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.</p>
        <p class="info">Score: 2.1, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.23294497' target='https://doi.org/10.1101/2023.08.23.23294497'> 10.1101/2023.08.23.23294497</a></p>
        <p class="abstract">Objective: Correct classification of type 1 (T1D) and type 2 diabetes (T2D) is challenging due to overlapping clinical features and the increasingly early onset of T2D, particularly in South Asians. We used polygenic risk scores (PRSs) in a British Bangladeshi and Pakistani population with diabetes to estimate the proportion and misclassification rate of T1D in insulin-treated individuals with ambiguous features. Research Design and Methods: Using linked health record from the Genes &amp; Health cohort (n=38,344) we defined four groups: 31 T1D cases, 1,842 T2D cases, and after excluding these, 839 insulin-treated individuals with ambiguous features and 5,174 controls. Combining these with 307 confirmed T1D cases and 307 controls from India, we calculated ancestry-corrected PRSs for T1D and T2D, with which we estimated the proportion of T1D cases within the ambiguous group and evaluated misclassification. Results: We estimated that the prevalence of T1D was ~6% within the ambiguous group, or ~4.5% within the subset who had T2D codes in their health records. We saw no significant association between the T1D or T2D PRS and BMI at diagnosis, time to insulin, or the presence of T1D or T2D diagnostic codes amongst the T2D or ambiguous cases. Conclusions: We estimate that 4.5% of British Pakistanis and Bangladeshis with diabetes who are treated with insulin and have ambiguous clinical features have been classified incorrectly in their health records, and in fact have T1D. These findings have significant implications for the delivery of optimal clinical care at population level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.23294179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.23294179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.23294179">
        <p class="paperTitle">Neuroimaging-AI Endophenotypes of Brain Diseases in the General Population: Towards a Dimensional System of Vulnerability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.23294179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.23294179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: WEN, J.; Skampardoni, I.; Tian, Y. E.; Yang, Z.; cui, y.; erus, g.; hwang, g.; Varol, E.; boquetipujadas, a.; Chand, G.; Nasrallah, I. M.; Satterthwaite, T.; Shou, H.; Shen, L.; Toga, A. W.; Zalesky, A.; Davatzikos, C.</p>
        <p class="info">Score: 2.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.23294179' target='https://doi.org/10.1101/2023.08.16.23294179'> 10.1101/2023.08.16.23294179</a></p>
        <p class="abstract">Disease heterogeneity poses a significant challenge for precision diagnostics in both clinical and subclinical stages. Recent work leveraging artificial intelligence (AI) has offered promise to dissect this heterogeneity by identifying complex intermediate phenotypes, herein called dimensional neuroimaging endophenotypes (DNEs), which subtype various neurologic and neuropsychiatric diseases. We investigate the presence of nine such DNEs derived from independent yet harmonized studies on Alzheimer&#39;s disease (AD1-2), autism spectrum disorder (ASD1,2,3), late-life depression (LLD1,2), and schizophrenia (SCZ1,2), in the general population of 39,178 participants in the UK Biobank study. Phenome-wide associations revealed prominent associations between the nine DNEs and phenotypes related to the brain and other human organ systems. This phenotypic landscape aligns with the SNP-phenotype genome-wide associations, revealing 31 genomic loci associated with the nine DNEs (Bonferroni corrected P-value &lt; 5xE-8/9). The DNEs exhibited significant genetic correlations, colocalization, and causal relationships with multiple human organ systems and chronic diseases. A causal effect (odds ratio=1.25 [1.11, 1.40], P-value=8.72x10-4) was established from AD2, characterized by focal medial temporal lobe atrophy, to AD. The nine DNEs and their polygenic risk scores significantly improved the prediction accuracy for 14 systemic disease categories and mortality. These findings underscore the potential of the nine DNEs to identify individuals at a high risk of developing the four brain diseases during preclinical stages for precision diagnostics. All results are publicly available at: http://labs.loni.usc.edu/medicine/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294074">
        <p class="paperTitle">Rare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fenner, E.; Holmans, P.; O&#39;Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.</p>
        <p class="info">Score: 9.8, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294074' target='https://doi.org/10.1101/2023.08.14.23294074'> 10.1101/2023.08.14.23294074</a></p>
        <p class="abstract">Impairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23292816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23292816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23292816">
        <p class="paperTitle">Evaluation of optical genome mapping in clinical genetic testing of facioscapulohumeral muscular dystrophy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23292816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23292816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kovanda, A.; Lovrecic, L.; Rudolf, G.; Babic Bozovic, I.; Jaklic, H.; Leonardis, L.; Peterlin, B.</p>
        <p class="info">Score: 1.9, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23292816' target='https://doi.org/10.1101/2023.08.10.23292816'> 10.1101/2023.08.10.23292816</a></p>
        <p class="abstract">BackgroundFacioscapulohumeral muscular dystrophy (FSHD) is the third most common hereditary muscular dystrophy, caused by the contraction of the D4Z4 repeats on the permissive 4qA haplotype on chromosome 4, resulting in the faulty expression of the DUX4 gene. Traditional diagnostics is based on Southern blot, a time- and effort-intensive method that can be affected by single nucleotide (SNV), and copy number variants (CNV), as well as by the similarity of the D4Z4 repeats located on chromosome 10. We aimed to evaluate optical genome mapping (OGM) as an alternative molecular diagnostic method for the detection of FSHD.

MethodsWe first performed optical genome mapping with EnFocus FSHD analysis using DLE-1 labeling and the Saphyr instrument, for our patients with inconclusive diagnostic Southern blot results, negative FSHD2 results, and clinically evident FSHD. Secondly, we performed OGM in parallel with the classical Southern blot analysis for our prospectively collected new FSHD cases. Finally, we used panel exome sequencing to confirm FSHD2.

ResultsBy using OGM we could resolve two cases with diagnostically inconclusive Southern blot results as having shortened D4Z4 repeats on the permissive 4qA allele, consistent with their clinical presentation. Results of the prospectively collected patients tested in parallel using Southern blot and OGM showed full concordance, showing OGM to be a useful alternative to the classical Southern blot method for detecting FSHD1. For a patient showing clinical FSHD but no shortened D4Z4 repeat on the 4qA allele using OGM or Southern blot, a likely pathogenic variant in SMCHD1 was detected using exome sequencing, confirming FSHD2. OGM and panel exome sequencing can be used consecutively to detect FSHD2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293212">
        <p class="paperTitle">Critical assessment of variant prioritization methods for rare disease diagnosis within the Rare Genomes Project</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stenton, S. L.; O&#39;Leary, M.; Lemire, G.; VanNoy, G. E.; DiTroia, S.; Ganesh, V. S.; Groopman, E.; O&#39;Heir, E.; Mangilog, B.; Osei-Owusu, I.; Pais, L. S.; Serrano, J.; Singer-Berk, M.; Weisburd, B.; Wilson, M.; Austin-Tse, C.; Abdelhakim, M.; Althagafi, A.; Babbi, G.; Bellazzi, R.; Bovo, S.; Carta, M. G.; Casadio, R.; Coenen, P.-J.; De Paoli, F.; Floris, M.; Gajapathy, M.; Hoehndorf, R.; Jacobsen, J. O. B.; Joseph, T.; Kamandula, A.; Katsonis, P.; Kint, C.; Lichtarge, O.; Limongelli, I.; Lu, Y.; Magni, P.; Mamidi, T. K. K.; Martelli, P. L.; Mulargia, M.; Nicora, G.; Nykamp, K.; Pejaver, V.; Peng</p>
        <p class="info">Score: 9.8, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293212' target='https://doi.org/10.1101/2023.08.02.23293212'> 10.1101/2023.08.02.23293212</a></p>
        <p class="abstract">BackgroundA major obstacle faced by rare disease families is obtaining a genetic diagnosis. The average &#34;diagnostic odyssey&#34; lasts over five years, and causal variants are identified in under 50%. The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing (GS) for diagnosis and gene discovery. Families are consented for sharing of sequence and phenotype data with researchers, allowing development of a Critical Assessment of Genome Interpretation (CAGI) community challenge, placing variant prioritization models head-to-head in a real-life clinical diagnostic setting.

MethodsPredictors were provided a dataset of phenotype terms and variant calls from GS of 175 RGP individuals (65 families), including 35 solved training set families, with causal variants specified, and 30 test set families (14 solved, 16 unsolved). The challenge tasked teams with identifying the causal variants in as many test set families as possible. Ranked variant predictions were submitted with estimated probability of causal relationship (EPCR) values. Model performance was determined by two metrics, a weighted score based on rank position of true positive causal variants and maximum F-measure, based on precision and recall of causal variants across EPCR thresholds.

ResultsSixteen teams submitted predictions from 52 models, some with manual review incorporated. Top performing teams recalled the causal variants in up to 13 of 14 solved families by prioritizing high quality variant calls that were rare, predicted deleterious, segregating correctly, and consistent with reported phenotype. In unsolved families, newly discovered diagnostic variants were returned to two families following confirmatory RNA sequencing, and two prioritized novel disease gene candidates were entered into Matchmaker Exchange. In one example, RNA sequencing demonstrated aberrant splicing due to a deep intronic indel in ASNS, identified in trans with a frameshift variant, in an unsolved proband with phenotype overlap with asparagine synthetase deficiency.

ConclusionsBy objective assessment of variant predictions, we provide insights into current state-of-the-art algorithms and platforms for genome sequencing analysis for rare disease diagnosis and explore areas for future optimization. Identification of diagnostic variants in unsolved families promotes synergy between researchers with clinical and computational expertise as a means of advancing the field of clinical genome interpretation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.23294465">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.23294465" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.23294465">
        <p class="paperTitle">Circulating ADAM17 is associated with COVID-19 severity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.23294465" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.23294465" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pan, M.; Goncalves, I.; Edsfeldt, A.; Sun, J.; Sward, P.</p>
        <p class="info">Score: 1.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.23294465' target='https://doi.org/10.1101/2023.08.23.23294465'> 10.1101/2023.08.23.23294465</a></p>
        <p class="abstract">Abstract Background ADAM17 are emerging as an important role in the severe outcomes of COVID-19. This study aims to characterize causal relationship between ADAM17 and COVID-19. Methods Using mendelian randomization analyses, we examined the causal effects for circulating ADAM17 on COVID-19 outcomes using summary statistics from large genome wide association studies of ADAM17 (up to 35 559 individuals) from the Icelandic Cancer Project and deCODE genetics, critical COVID-19 (cases:13 769; controls:1 072 442), hospitalized COVID-19 (cases:32 519; controls: 2 062 805) and SARS-CoV-2 infection (cases:122 616; controls:2 475 240) from the COVID-19 Host Genetics Initiative. Results Mendelian randomization analyses demonstrated that 1 standard deviation increase of genetically determined circulating ADAM17 at extracellular domain were associated with increasing risk of developing critical COVID-19 (odds ratio [OR]=1.26, 95% CI 1.03-1.55). Multivariable MR analysis suggested a direct causal role of circulating ADAM17 at extracellular domain on the risk of critical COVID-19 (OR=1.09; 95% CI 1.01-1.17), accounting for body mass index. Casual effects for the cytoplasmic domain of ADAM17 on COVID-19 were not observed. Conclusion Our results suggest that the increased circulating ADAM17 at extracellular domain are associated with a high risk of critical COVID-19 strengthening that of ADAM17 may contribute to the risk stratification and a therapeutic option for severe COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.23294314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.23294314" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.23294314">
        <p class="paperTitle">Study of the protective OAS1 rs10774671-G allele against severe COVID-19 in Moroccans suggests a North African origin for Neanderthals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.23294314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.23294314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: El Youssfi, F. Z.; Haroun, A. E.; Nebhani, C.; Belayachi, J.; Askander, O.; El Fahime, E.; Fares, H.; Ennibi, K.; Abouqal, R.; Razine, R.; Bouhouche, A.</p>
        <p class="info">Score: 1.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.23294314' target='https://doi.org/10.1101/2023.08.19.23294314'> 10.1101/2023.08.19.23294314</a></p>
        <p class="abstract">BackgroundThe clinical presentation of COVID-19 has shown high variability between individuals, which is partly due to genetic factors. The OAS1/2/3 cluster was found to be strongly associated with COVID-19 severity. We aimed to examine this locus for the occurrence of the critical variant, rs10774671, and its respective haplotype blocks within the Moroccan population.

MethodsThe frequency of SNPs at the cluster of OAS immunity genes was assessed from an in-house database in 157 unrelated individuals of Moroccan origin. The OAS1 exon 6 was sequenced by Sangers method in 71 asymptomatic/mild and 74 moderate/severe individuals positive for SARS-CoV-2. Genotypic, allelic, and haplotype frequencies of three SNPs were compared between the two groups. Finally, males in our COVID-19 series were genotyped for the Berber-specific marker E-M81.

ResultsThe prevalence of the OAS1 rs10774671-G allele in present-day Moroccans was 40.4%, close to that of Europeans. However, it was found equally on both the Neanderthal GGG haplotype and the African GAC haplotype with a frequency of 20% each. These two haplotypes, and hence the rs10774671-G allele, were significantly associated with the protection against severe COVID-19 (p = 0.034, p = 0.041, and p = 0.008 respectively). Surprisingly, among Berber men, the African haplotype was absent while the prevalence of the Neanderthal haplotype was close to that of Europeans.

ConclusionThe protective rs10774671-G allele of OAS1 was found only in the Neanderthal haplotype in Berbers, the indigenous people of North Africa, suggesting that this region may have served as the stepping-stone for the passage of the hominids to the other continents.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
