<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 58.7, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 58.7, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.
Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.
Score: 58.7, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062
Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 58.7, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations.",
  "keywords": [
    
  ],
  "articleBody": " Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 58.7, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at pop.evemodel.org.\nGenetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes\nAuthors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.\nScore: 29.3, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23299014\nThe biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).\nLong-read RNA-seq atlas of novel microglia isoforms elucidates disease-associated genetic regulation of splicing\nAuthors: Humphrey, J.; Brophy, E.; Kosoy, R.; Zeng, B.; Coccia, E.; Mattei, D.; Ravi, A.; Efthymiou, A. G.; Navarro, E.; Muller, B. Z.; Snijders, G. J.; Allan, A.; Munch, A.; Birhanu Kitata, R.; Kleopoulos, S. P.; Argyriou, S.; Shao, Z.; Francoeur, N. J.; Tsai, C.-F.; Gritsenko, M. A.; Monroe, M. E.; Paurus, V. L.; Weitz, K. K.; Shi, T.; Sebra, R.; Liu, T.; Hoffman, G. E.; Goate, A. M.; de Witte, L. D.; Bennett, D.; Haroutunian, V.; Fullard, J. F.; Roussos, P.; Raj, T.\nScore: 8.2, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23299073\nMicroglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. We previously mapped the genetic regulation of gene expression and mRNA splicing in human microglia, identifying several loci where common genetic variants in microglia-specific regulatory elements explain disease risk loci identified by GWAS. However, identifying genetic effects on splicing has been challenging due to the use of short sequencing reads to identify causal isoforms. Here we present the isoform-centric microglia genomic atlas (isoMiGA) which leverages the power of long-read RNA-seq to identify 35,879 novel microglia isoforms. We show that the novel microglia isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ethnic meta-analysis of 555 human microglia short-read RNA-seq samples from 391 donors, the largest to date, and found associations with genetic risk loci in Alzheimers disease and Parkinsons disease. We nominate several loci that may act through complex changes in isoform and splice site usage.\nPervasive correlations between causal disease effects of proximal SNPs vary with functional annotations and implicate stabilizing selection\nAuthors: Zhang, M. J.; Durvasula, A.; Chiang, C.; Koch, E. M.; Strober, B. J.; Shi, H.; Barton, A. R.; Kim, S. S.; Weissbrod, O.; Loh, P.-R.; Gazal, S.; Sunyaev, S.; Price, A.\nScore: 6.4, Published: 2023-12-04 DOI: 10.1101/2023.12.04.23299391\nThe genetic architecture of human diseases and complex traits has been extensively studied, but little is known about the relationship of causal disease effect sizes between proximal SNPs, which have largely been assumed to be independent. We introduce a new method, LD SNP-pair effect correlation regression (LDSPEC), to estimate the correlation of causal disease effect sizes of derived alleles between proximal SNPs, depending on their allele frequencies, LD, and functional annotations; LDSPEC produced robust estimates in simulations across various genetic architectures. We applied LDSPEC to 70 diseases and complex traits from the UK Biobank (average N=306K), meta-analyzing results across diseases/traits. We detected significantly nonzero effect correlations for proximal SNP pairs (e.g., -0.37{+/-}0.09 for low-frequency positive-LD 0-100bp SNP pairs) that decayed with distance (e.g., -0.07{+/-}0.01 for low-frequency positive-LD 1-10kb), varied with allele frequency (e.g., -0.15{+/-}0.04 for common positive-LD 0-100bp), and varied with LD between SNPs (e.g., +0.12{+/-}0.05 for common negative-LD 0-100bp) (because we consider derived alleles, positive-LD and negative-LD SNP pairs may yield very different results). We further determined that SNP pairs with shared functions had stronger effect correlations that spanned longer genomic distances, e.g., -0.37{+/-}0.08 for low-frequency positive-LD same-gene promoter SNP pairs (average genomic distance of 47kb (due to alternative splicing)) and -0.32{+/-}0.04 for low-frequency positive-LD H3K27ac 0-1kb SNP pairs. Consequently, SNP-heritability estimates were substantially smaller than estimates of the sum of causal effect size variances across all SNPs (ratio of 0.87{+/-}0.02 across diseases/traits), particularly for certain functional annotations (e.g., 0.78{+/-}0.01 for common Super enhancer SNPs)--even though these quantities are widely assumed to be equal. We recapitulated our findings via forward simulations with an evolutionary model involving stabilizing selection, implicating the action of linkage masking, whereby haplotypes containing linked SNPs with opposite effects on disease have reduced effects on fitness and escape negative selection.\nPhenomAD-NDD: the Phenomics Aggregation Database of comorbidities in 51,227 pediatric individuals with NeuroDevelopmental Disorders\nAuthors: Dingemans, A. J. M.; Jansen, S.; van Reeuwijk, J.; de Leeuw, N.; Pfundt, R.; Schuurs-Hoeijmakers, J.; van Bon, B. W.; Marcelis, C.; Ockeloen, C. W.; Willemsen, M.; Van der Sluijs, P. J.; Santen, G. W. E.; Kooy, R. F.; Vult-van Silfhout, A. T.; Kleefstra, T.; Koolen, D. A.; Vissers, L. E. L. M.; de Vries, B. B. A.\nScore: 6.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.23299167\nThe prevalence of comorbidities in individuals with neurodevelopmental disorders (NDD) is not well understood, while these are important for accurate diagnosis and prognosis in routine care and for characterizing the clinical spectrum of NDD syndromes. Therefore, we developed PhenomAD-NDD: an aggregated database with comorbid phenotypic data of 51,227 individuals with NDD, all harmonized into Human Phenotype Ontology (HPO), with in total 3,054 unique HPO terms. We demonstrate that almost all congenital anomalies are more prevalent in the NDD population than in the general population and the NDD baseline prevalence allows for approximation of enrichment of symptoms. Such analyses for 33 genetic NDDs for instance shows that 32% of enriched phenotypes is currently not reported in the clinical synopsis in OMIM. PhenomAD-NDD is open to all via a visualization online tool and allows to determine enrichment of symptoms in NDD.\nGenetic association analysis of age of ischemic stroke onset: a caveat\nAuthors: von Berg, J.; McArdle, P.; Happola, P. J.; Haessler, J.; Kooperberg, C.; Lemmens, R.; Pezzini, A.; Thijs, V.; Pulit, S. L.; Kittner, S.; Mitchell, B. D.; de Ridder, J.; van der Laan, S. W.\nScore: 5.8, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23294385\nLarge genome-wide association studies (GWAS) employing case-control study designs have now identified tens of loci associated with ischemic stroke (IS). As a complement to these studies, we performed GWAS in a case-only design to identify loci influencing age at onset (AAO) of ischemic stroke. Analyses were conducted in a Discovery cohort of 10,857 ischemic stroke cases using a linear regression framework. We meta-analyzed all SNPs with p-value \u003c 1x10-5 in a sex-combined or sex-stratified analysis using summary data from two additional replication cohorts. In the women-only meta-analysis, we detected significant evidence for association of AAO with rs429358, an exonic variant in APOE that encodes for the APOE-[jukcy]4 allele. Each copy of the rs429358:T\u003eC allele was associated with a 1.29 years earlier stroke AOO (meta p-value = 2.48x10-11). This APOE variant has previously been associated with increased mortality and ischemic stroke AAO. We hypothesized that the association with AAO may reflect a survival bias attributable to an age-related decline in mortality among APOE-[jukcy]4 carriers and have no association to stroke AAO per se. Using a simulation study, we found that a variant associated with overall mortality might indeed be detected with an AAO analysis. A variant with a two-fold increase on mortality risk would lead to an observed effect of AAO that is comparable to what we found. In conclusion, we detected a robust association of the APOE locus with stroke AAO and provided simulations to suggest that this association may be unrelated to ischemic stroke per se but related to a general survival bias.\nGenomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in AR-altered lethal prostate cancer\nAuthors: Chauhan, P. S.; Alahi, I.; Sinha, S.; Shiang, A. L.; Mueller, R.; Webster, J.; Dang, H. X.; Saha, D.; Greiner, L.; Yang, B.; Ni, G.; Ledet, E. M.; Babbra, R. K.; Feng, W.; Harris, P. K.; Qaium, F.; Jaeger, E. B.; Miller, P. J.; Caputo, S. A.; Sartor, O.; Pachynski, R. K.; Maher, C. A.; Chaudhuri, A. A.\nScore: 5.5, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23299215\nMetastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor (AR)-targeted agents is often lethal. Unfortunately, biomarkers for this deadly disease remain under investigation, and underpinning mechanisms are ill-understood. Here, we applied deep sequencing to [~]100 mCRPC patients prior to the initiation of first-line AR-targeted therapy, which detected AR/enhancer alterations in over a third of patients, which correlated with lethality. To delve into the mechanism underlying why these patients with cell-free AR/enhancer alterations developed more lethal prostate cancer, we next performed genome-wide cell-free DNA epigenomics. Strikingly, we found that binding sites for transcription factors associated with developmental stemness were nucleosomally more accessible. These results were corroborated using cell-free DNA methylation data, as well as tumor RNA sequencing from a held-out cohort of mCRPC patients. Thus, we validated the importance of AR/enhancer alterations as a prognostic biomarker in lethal mCRPC, and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.\nGenetic analysis of 104 pregnancy phenotypes in 39,194 Chinese women\nAuthors: Xiao, H.; Li, L.; Yang, M.; Zeng, J.; Zhou, J.; Tao, Y.; Zhou, Y.; Cai, M.; Liu, J.; Huang, Y.; Zhong, Y.; Liu, P.; Cao, Z.; Mei, H.; Cai, X.; Hu, L.; Zhou, R.; Xu, X.; Yang, H.; Wang, J.; Zhu, H.; Zhou, A.; Jin, X.\nScore: 10.1, Published: 2023-11-24 DOI: 10.1101/2023.11.23.23298979\nMaternity is a special period in a womans life that involves substantial physiological, psychological, and hormonal changes. These changes may cause alterations in many clinical measurements during pregnancy, which can be used to monitor and diagnose maternal disorders and adverse postnatal outcomes. Exploring the genetic background of these phenotypes is key to elucidating the pathogenesis of pregnancy disorders. In this study, we conducted a large-scale molecular biology analysis of 104 pregnancy phenotypes based on genotype data from 39,194 Chinses women. Genome- wide association analysis identified a total of 407 trait-locus associations, of which 75.18% were previously reported. Among the 101 novel associations for 37 phenotypes, some were potentially pregnancy-specific and worth further experimental investigation. For example, ESR1 with fasting glucose, hemoglobin, hematocrit, and several leukocytoses; ZSCAN31 with blood urea nitrogen. We further performed pathway- based analysis and uncovered at least one significant pathway for 24 traits, in addition to previously known functional pathways, novel findings included birthweight with \"Reactome signaling by NODAL\", twin pregnancy with \"Reactome mitotic G1-G1/S phases\". The partitioning heritability analysis recapitulated known trait-relevant tissue/cell types, and also discovered interesting results including twin pregnancy with \"embryoid bodies\" cell-type enrichment, the delivery type cesarean section with \"fallopian tube\", and birth weight with \"ovary and embryonic stem cells\". In terms of both sample size and the variety of phenotypes, our work is one of the largest genetic studies of pregnancy phenotypes across all populations. We believe that this study will provide a valuable resource for exploring the genetic background of pregnancy phenotypes and also for further research on pregnancy-related diseases and adverse neonatal outcomes.\nCopy-number variants and polygenic risk for intelligence confer risk for autism spectrum disorder irrespective of their effects on cognitive ability\nAuthors: Schmilovich, Z.; Bourque, V.-R.; Douard, E.; Huguet, G.; Poulain, C.; Ross, J. P.; Alipour, P.; Castonguay, C.-E.; Younis, N.; Jean-Louis, M.; Saci, Z.; Pausova, Z.; Paus, T.; Schuman, G.; Porteous, D.; Davies, G.; Redmond, P.; Harris, S. E.; Deary, I. J.; Whalley, H.; Hayward, C.; Dion, P. A.; Jacquemont, S.; Rouleau, G. A.\nScore: 2.6, Published: 2023-11-29 DOI: 10.1101/2023.11.29.23299190\nRare copy number variants (CNVs) and polygenic risk for intelligence (PRS-IQ) both confer risk for autism spectrum disorder (ASD) but have opposing effects on cognitive ability. The field has struggled to disentangle the effects of these two classes of genomic variants on cognitive ability from their effects on ASD risk, in part because previous studies did not include controls with cognitive measures. We aim to investigate the impact of these genomic variants on ASD risk while adjusting for their known effects on cognitive ability. In a cohort of 8,426 subjects with ASD and 169,804 controls with cognitive assessments, we found that rare coding CNVs and PRS-IQ increased ASD risk, even after adjusting for their effects on cognitive ability. Bottom decile PRS-IQ and CNVs both decreased cognitive ability but had opposing effects on ASD risk. Models combining both classes of variants showed that the effects of rare CNVs and PRS-IQ on ASD risk and cognitive ability were largely additive, further suggesting that risk for ASD is conferred independently from its effects on cognitive ability. Despite imparting mostly additive effects on ASD risk, rare CNVs and PRS-IQ showed opposing effects on core and associated features and developmental history among subjects with ASD. Our findings suggest that cognitive ability itself may not be the factor driving the underlying risk for ASD conferred by these two classes of genomic variants. In other words, ASD risk and cognitive ability may be two distinct manifestations of CNVs and PRS-IQ. This study also highlights the challenge of understanding how genetic risk for ASD maps onto its dimensional traits.\nGenomic and immune determinants of resistance to anti-CD38 monoclonal antibody-based therapy in relapsed refractory multiple myeloma.\nAuthors: Ziccheddu, B.; Giannotta, C.; D'Agostino, M.; Bertuglia, G.; Saraci, E.; Oliva, S.; Genuardi, E.; Papadimitriou, M.; Diamond, B.; Corradini, P.; Coffey, D.; Landgren, O.; Bolli, N.; Bruno, B.; Boccadoro, M.; Massaia, M.; Maura, F.; Larocca, A.\nScore: 1.2, Published: 2023-12-04 DOI: 10.1101/2023.12.04.23299287\nAnti-CD38 antibody therapies have transformed multiple myeloma (MM) treatment. However, a large fraction of patients inevitably relapses. To understand this, we investigated 32 relapsed MM patients treated with daratumumab, lenalidomide, and dexamethasone (Dara-Rd; NCT03848676). Whole genome sequencing (WGS) before and after treatment pinpointed genomic drivers associated with early progression, including RPL5 loss and APOBEC mutagenesis. Flow cytometry on 202 blood samples, collected every three months until progression for 31 patients, revealed distinct immune changes significantly impacting clinical outcomes. Progressing patients exhibited significant depletion of CD38+ NK cells, persistence of T cell exhaustion, and reduced depletion of T-reg cells over time. These findings underscore the influence of immune composition and daratumumab-induced immune changes in promoting MM resistance. Integrating genomics and flow cytometry unveiled associations between adverse genomic features and immune patterns. Overall, this study sheds light on the intricate interplay between genomic complexity and the immune microenvironment driving resistance to Dara-Rd.\n",
  "wordCount" : "2772",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23299062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23299062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23299062">
        <p class="paperTitle">Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23299062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23299062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.</p>
        <p class="info">Score: 58.7, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23299062' target='https://doi.org/10.1101/2023.11.27.23299062'> 10.1101/2023.11.27.23299062</a></p>
        <p class="abstract">Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at pop.evemodel.org.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.23299014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.23299014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.23299014">
        <p class="paperTitle">Genetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.23299014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.23299014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.</p>
        <p class="info">Score: 29.3, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.23299014' target='https://doi.org/10.1101/2023.11.25.23299014'> 10.1101/2023.11.25.23299014</a></p>
        <p class="abstract">The biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23299073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23299073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23299073">
        <p class="paperTitle">Long-read RNA-seq atlas of novel microglia isoforms elucidates disease-associated genetic regulation of splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23299073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23299073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Humphrey, J.; Brophy, E.; Kosoy, R.; Zeng, B.; Coccia, E.; Mattei, D.; Ravi, A.; Efthymiou, A. G.; Navarro, E.; Muller, B. Z.; Snijders, G. J.; Allan, A.; Munch, A.; Birhanu Kitata, R.; Kleopoulos, S. P.; Argyriou, S.; Shao, Z.; Francoeur, N. J.; Tsai, C.-F.; Gritsenko, M. A.; Monroe, M. E.; Paurus, V. L.; Weitz, K. K.; Shi, T.; Sebra, R.; Liu, T.; Hoffman, G. E.; Goate, A. M.; de Witte, L. D.; Bennett, D.; Haroutunian, V.; Fullard, J. F.; Roussos, P.; Raj, T.</p>
        <p class="info">Score: 8.2, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23299073' target='https://doi.org/10.1101/2023.12.01.23299073'> 10.1101/2023.12.01.23299073</a></p>
        <p class="abstract">Microglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. We previously mapped the genetic regulation of gene expression and mRNA splicing in human microglia, identifying several loci where common genetic variants in microglia-specific regulatory elements explain disease risk loci identified by GWAS. However, identifying genetic effects on splicing has been challenging due to the use of short sequencing reads to identify causal isoforms. Here we present the isoform-centric microglia genomic atlas (isoMiGA) which leverages the power of long-read RNA-seq to identify 35,879 novel microglia isoforms. We show that the novel microglia isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ethnic meta-analysis of 555 human microglia short-read RNA-seq samples from 391 donors, the largest to date, and found associations with genetic risk loci in Alzheimers disease and Parkinsons disease. We nominate several loci that may act through complex changes in isoform and splice site usage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.23299391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.23299391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.23299391">
        <p class="paperTitle">Pervasive correlations between causal disease effects of proximal SNPs vary with functional annotations and implicate stabilizing selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.23299391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.23299391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, M. J.; Durvasula, A.; Chiang, C.; Koch, E. M.; Strober, B. J.; Shi, H.; Barton, A. R.; Kim, S. S.; Weissbrod, O.; Loh, P.-R.; Gazal, S.; Sunyaev, S.; Price, A.</p>
        <p class="info">Score: 6.4, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.23299391' target='https://doi.org/10.1101/2023.12.04.23299391'> 10.1101/2023.12.04.23299391</a></p>
        <p class="abstract">The genetic architecture of human diseases and complex traits has been extensively studied, but little is known about the relationship of causal disease effect sizes between proximal SNPs, which have largely been assumed to be independent. We introduce a new method, LD SNP-pair effect correlation regression (LDSPEC), to estimate the correlation of causal disease effect sizes of derived alleles between proximal SNPs, depending on their allele frequencies, LD, and functional annotations; LDSPEC produced robust estimates in simulations across various genetic architectures. We applied LDSPEC to 70 diseases and complex traits from the UK Biobank (average N=306K), meta-analyzing results across diseases/traits. We detected significantly nonzero effect correlations for proximal SNP pairs (e.g., -0.37{&#43;/-}0.09 for low-frequency positive-LD 0-100bp SNP pairs) that decayed with distance (e.g., -0.07{&#43;/-}0.01 for low-frequency positive-LD 1-10kb), varied with allele frequency (e.g., -0.15{&#43;/-}0.04 for common positive-LD 0-100bp), and varied with LD between SNPs (e.g., &#43;0.12{&#43;/-}0.05 for common negative-LD 0-100bp) (because we consider derived alleles, positive-LD and negative-LD SNP pairs may yield very different results). We further determined that SNP pairs with shared functions had stronger effect correlations that spanned longer genomic distances, e.g., -0.37{&#43;/-}0.08 for low-frequency positive-LD same-gene promoter SNP pairs (average genomic distance of 47kb (due to alternative splicing)) and -0.32{&#43;/-}0.04 for low-frequency positive-LD H3K27ac 0-1kb SNP pairs. Consequently, SNP-heritability estimates were substantially smaller than estimates of the sum of causal effect size variances across all SNPs (ratio of 0.87{&#43;/-}0.02 across diseases/traits), particularly for certain functional annotations (e.g., 0.78{&#43;/-}0.01 for common Super enhancer SNPs)--even though these quantities are widely assumed to be equal. We recapitulated our findings via forward simulations with an evolutionary model involving stabilizing selection, implicating the action of linkage masking, whereby haplotypes containing linked SNPs with opposite effects on disease have reduced effects on fitness and escape negative selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.23299167">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.23299167" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.23299167">
        <p class="paperTitle">PhenomAD-NDD: the Phenomics Aggregation Database of comorbidities in 51,227 pediatric individuals with NeuroDevelopmental Disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.23299167" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.23299167" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dingemans, A. J. M.; Jansen, S.; van Reeuwijk, J.; de Leeuw, N.; Pfundt, R.; Schuurs-Hoeijmakers, J.; van Bon, B. W.; Marcelis, C.; Ockeloen, C. W.; Willemsen, M.; Van der Sluijs, P. J.; Santen, G. W. E.; Kooy, R. F.; Vult-van Silfhout, A. T.; Kleefstra, T.; Koolen, D. A.; Vissers, L. E. L. M.; de Vries, B. B. A.</p>
        <p class="info">Score: 6.1, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.23299167' target='https://doi.org/10.1101/2023.11.29.23299167'> 10.1101/2023.11.29.23299167</a></p>
        <p class="abstract">The prevalence of comorbidities in individuals with neurodevelopmental disorders (NDD) is not well understood, while these are important for accurate diagnosis and prognosis in routine care and for characterizing the clinical spectrum of NDD syndromes. Therefore, we developed PhenomAD-NDD: an aggregated database with comorbid phenotypic data of 51,227 individuals with NDD, all harmonized into Human Phenotype Ontology (HPO), with in total 3,054 unique HPO terms. We demonstrate that almost all congenital anomalies are more prevalent in the NDD population than in the general population and the NDD baseline prevalence allows for approximation of enrichment of symptoms. Such analyses for 33 genetic NDDs for instance shows that 32% of enriched phenotypes is currently not reported in the clinical synopsis in OMIM. PhenomAD-NDD is open to all via a visualization online tool and allows to determine enrichment of symptoms in NDD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23294385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23294385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23294385">
        <p class="paperTitle">Genetic association analysis of age of ischemic stroke onset: a caveat</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23294385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23294385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: von Berg, J.; McArdle, P.; Happola, P. J.; Haessler, J.; Kooperberg, C.; Lemmens, R.; Pezzini, A.; Thijs, V.; Pulit, S. L.; Kittner, S.; Mitchell, B. D.; de Ridder, J.; van der Laan, S. W.</p>
        <p class="info">Score: 5.8, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23294385' target='https://doi.org/10.1101/2023.12.01.23294385'> 10.1101/2023.12.01.23294385</a></p>
        <p class="abstract">Large genome-wide association studies (GWAS) employing case-control study designs have now identified tens of loci associated with ischemic stroke (IS). As a complement to these studies, we performed GWAS in a case-only design to identify loci influencing age at onset (AAO) of ischemic stroke. Analyses were conducted in a Discovery cohort of 10,857 ischemic stroke cases using a linear regression framework. We meta-analyzed all SNPs with p-value &lt; 1x10-5 in a sex-combined or sex-stratified analysis using summary data from two additional replication cohorts. In the women-only meta-analysis, we detected significant evidence for association of AAO with rs429358, an exonic variant in APOE that encodes for the APOE-[jukcy]4 allele. Each copy of the rs429358:T&gt;C allele was associated with a 1.29 years earlier stroke AOO (meta p-value = 2.48x10-11).

This APOE variant has previously been associated with increased mortality and ischemic stroke AAO. We hypothesized that the association with AAO may reflect a survival bias attributable to an age-related decline in mortality among APOE-[jukcy]4 carriers and have no association to stroke AAO per se. Using a simulation study, we found that a variant associated with overall mortality might indeed be detected with an AAO analysis. A variant with a two-fold increase on mortality risk would lead to an observed effect of AAO that is comparable to what we found. In conclusion, we detected a robust association of the APOE locus with stroke AAO and provided simulations to suggest that this association may be unrelated to ischemic stroke per se but related to a general survival bias.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23299215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23299215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23299215">
        <p class="paperTitle">Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in AR-altered lethal prostate cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23299215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23299215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chauhan, P. S.; Alahi, I.; Sinha, S.; Shiang, A. L.; Mueller, R.; Webster, J.; Dang, H. X.; Saha, D.; Greiner, L.; Yang, B.; Ni, G.; Ledet, E. M.; Babbra, R. K.; Feng, W.; Harris, P. K.; Qaium, F.; Jaeger, E. B.; Miller, P. J.; Caputo, S. A.; Sartor, O.; Pachynski, R. K.; Maher, C. A.; Chaudhuri, A. A.</p>
        <p class="info">Score: 5.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23299215' target='https://doi.org/10.1101/2023.12.01.23299215'> 10.1101/2023.12.01.23299215</a></p>
        <p class="abstract">Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor (AR)-targeted agents is often lethal. Unfortunately, biomarkers for this deadly disease remain under investigation, and underpinning mechanisms are ill-understood. Here, we applied deep sequencing to [~]100 mCRPC patients prior to the initiation of first-line AR-targeted therapy, which detected AR/enhancer alterations in over a third of patients, which correlated with lethality. To delve into the mechanism underlying why these patients with cell-free AR/enhancer alterations developed more lethal prostate cancer, we next performed genome-wide cell-free DNA epigenomics. Strikingly, we found that binding sites for transcription factors associated with developmental stemness were nucleosomally more accessible. These results were corroborated using cell-free DNA methylation data, as well as tumor RNA sequencing from a held-out cohort of mCRPC patients. Thus, we validated the importance of AR/enhancer alterations as a prognostic biomarker in lethal mCRPC, and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.23298979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.23298979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.23298979">
        <p class="paperTitle">Genetic analysis of 104 pregnancy phenotypes in 39,194 Chinese women</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.23298979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.23298979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xiao, H.; Li, L.; Yang, M.; Zeng, J.; Zhou, J.; Tao, Y.; Zhou, Y.; Cai, M.; Liu, J.; Huang, Y.; Zhong, Y.; Liu, P.; Cao, Z.; Mei, H.; Cai, X.; Hu, L.; Zhou, R.; Xu, X.; Yang, H.; Wang, J.; Zhu, H.; Zhou, A.; Jin, X.</p>
        <p class="info">Score: 10.1, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.23298979' target='https://doi.org/10.1101/2023.11.23.23298979'> 10.1101/2023.11.23.23298979</a></p>
        <p class="abstract">Maternity is a special period in a womans life that involves substantial physiological, psychological, and hormonal changes. These changes may cause alterations in many clinical measurements during pregnancy, which can be used to monitor and diagnose maternal disorders and adverse postnatal outcomes. Exploring the genetic background of these phenotypes is key to elucidating the pathogenesis of pregnancy disorders. In this study, we conducted a large-scale molecular biology analysis of 104 pregnancy phenotypes based on genotype data from 39,194 Chinses women. Genome- wide association analysis identified a total of 407 trait-locus associations, of which 75.18% were previously reported. Among the 101 novel associations for 37 phenotypes, some were potentially pregnancy-specific and worth further experimental investigation. For example, ESR1 with fasting glucose, hemoglobin, hematocrit, and several leukocytoses; ZSCAN31 with blood urea nitrogen. We further performed pathway- based analysis and uncovered at least one significant pathway for 24 traits, in addition to previously known functional pathways, novel findings included birthweight with &#34;Reactome signaling by NODAL&#34;, twin pregnancy with &#34;Reactome mitotic G1-G1/S phases&#34;. The partitioning heritability analysis recapitulated known trait-relevant tissue/cell types, and also discovered interesting results including twin pregnancy with &#34;embryoid bodies&#34; cell-type enrichment, the delivery type cesarean section with &#34;fallopian tube&#34;, and birth weight with &#34;ovary and embryonic stem cells&#34;. In terms of both sample size and the variety of phenotypes, our work is one of the largest genetic studies of pregnancy phenotypes across all populations. We believe that this study will provide a valuable resource for exploring the genetic background of pregnancy phenotypes and also for further research on pregnancy-related diseases and adverse neonatal outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.23299190">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.23299190" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.23299190">
        <p class="paperTitle">Copy-number variants and polygenic risk for intelligence confer risk for autism spectrum disorder irrespective of their effects on cognitive ability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.23299190" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.23299190" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmilovich, Z.; Bourque, V.-R.; Douard, E.; Huguet, G.; Poulain, C.; Ross, J. P.; Alipour, P.; Castonguay, C.-E.; Younis, N.; Jean-Louis, M.; Saci, Z.; Pausova, Z.; Paus, T.; Schuman, G.; Porteous, D.; Davies, G.; Redmond, P.; Harris, S. E.; Deary, I. J.; Whalley, H.; Hayward, C.; Dion, P. A.; Jacquemont, S.; Rouleau, G. A.</p>
        <p class="info">Score: 2.6, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.23299190' target='https://doi.org/10.1101/2023.11.29.23299190'> 10.1101/2023.11.29.23299190</a></p>
        <p class="abstract">Rare copy number variants (CNVs) and polygenic risk for intelligence (PRS-IQ) both confer risk for autism spectrum disorder (ASD) but have opposing effects on cognitive ability. The field has struggled to disentangle the effects of these two classes of genomic variants on cognitive ability from their effects on ASD risk, in part because previous studies did not include controls with cognitive measures. We aim to investigate the impact of these genomic variants on ASD risk while adjusting for their known effects on cognitive ability. In a cohort of 8,426 subjects with ASD and 169,804 controls with cognitive assessments, we found that rare coding CNVs and PRS-IQ increased ASD risk, even after adjusting for their effects on cognitive ability. Bottom decile PRS-IQ and CNVs both decreased cognitive ability but had opposing effects on ASD risk. Models combining both classes of variants showed that the effects of rare CNVs and PRS-IQ on ASD risk and cognitive ability were largely additive, further suggesting that risk for ASD is conferred independently from its effects on cognitive ability. Despite imparting mostly additive effects on ASD risk, rare CNVs and PRS-IQ showed opposing effects on core and associated features and developmental history among subjects with ASD. Our findings suggest that cognitive ability itself may not be the factor driving the underlying risk for ASD conferred by these two classes of genomic variants. In other words, ASD risk and cognitive ability may be two distinct manifestations of CNVs and PRS-IQ. This study also highlights the challenge of understanding how genetic risk for ASD maps onto its dimensional traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.23299287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.23299287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.23299287">
        <p class="paperTitle">Genomic and immune determinants of resistance to anti-CD38 monoclonal antibody-based therapy in relapsed refractory multiple myeloma.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.23299287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.23299287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ziccheddu, B.; Giannotta, C.; D&#39;Agostino, M.; Bertuglia, G.; Saraci, E.; Oliva, S.; Genuardi, E.; Papadimitriou, M.; Diamond, B.; Corradini, P.; Coffey, D.; Landgren, O.; Bolli, N.; Bruno, B.; Boccadoro, M.; Massaia, M.; Maura, F.; Larocca, A.</p>
        <p class="info">Score: 1.2, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.23299287' target='https://doi.org/10.1101/2023.12.04.23299287'> 10.1101/2023.12.04.23299287</a></p>
        <p class="abstract">Anti-CD38 antibody therapies have transformed multiple myeloma (MM) treatment. However, a large fraction of patients inevitably relapses. To understand this, we investigated 32 relapsed MM patients treated with daratumumab, lenalidomide, and dexamethasone (Dara-Rd; NCT03848676). Whole genome sequencing (WGS) before and after treatment pinpointed genomic drivers associated with early progression, including RPL5 loss and APOBEC mutagenesis. Flow cytometry on 202 blood samples, collected every three months until progression for 31 patients, revealed distinct immune changes significantly impacting clinical outcomes. Progressing patients exhibited significant depletion of CD38&#43; NK cells, persistence of T cell exhaustion, and reduced depletion of T-reg cells over time. These findings underscore the influence of immune composition and daratumumab-induced immune changes in promoting MM resistance. Integrating genomics and flow cytometry unveiled associations between adverse genomic features and immune patterns. Overall, this study sheds light on the intricate interplay between genomic complexity and the immune microenvironment driving resistance to Dara-Rd.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
