<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Whole-genome sequencing of half-a-million UK Biobank participants
Authors: The UK Biobank Whole-Genome Sequencing Consortium, ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.
Score: 65.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299426
Summary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Whole-genome sequencing of half-a-million UK Biobank participants
Authors: The UK Biobank Whole-Genome Sequencing Consortium, ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.
Score: 65.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299426
Summary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Whole-genome sequencing of half-a-million UK Biobank participants
Authors: The UK Biobank Whole-Genome Sequencing Consortium, ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.
Score: 65.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299426
Summary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Whole-genome sequencing of half-a-million UK Biobank participants\nAuthors: The UK Biobank Whole-Genome Sequencing Consortium, ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.\nScore: 65.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299426\nSummary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3.",
  "keywords": [
    
  ],
  "articleBody": " Whole-genome sequencing of half-a-million UK Biobank participants\nAuthors: The UK Biobank Whole-Genome Sequencing Consortium, ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.\nScore: 65.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299426\nSummary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3. This advance deepens our understanding of how genetics influences disease biology and further enhances the value of this open resource for the study of human biology and health. Coupling this dataset with rich phenotypic data, we surveyed within- and cross-ancestry genomic associations with health-related phenotypes and identified novel genetic and clinical insights. While most genome-wide significant associations with disease traits were primarily observed in Europeans, we also identified strong or novel signals in individuals of African and Asian ancestries. Deeper capture of exonic variation in both coding and UTR sequences, strengthened and surfaced novel insights relative to WES analyses. This landmark dataset, representing the largest collection of WGS and available to the UK Biobank research community, will enable advances into our understanding of the human genome, and facilitate the discovery of new diagnostics, therapeutics with higher efficacy and improved safety profile, and enable precision medicine strategies with the potential to improve global health. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=114 SRC=\"FIGDIR/small/23299426v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (22K): org.highwire.dtl.DTLVardef@e03bc2org.highwire.dtl.DTLVardef@1736602org.highwire.dtl.DTLVardef@12e4050org.highwire.dtl.DTLVardef@1805b5c_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOGraphic summary.C_FLOATNO Framework of the WGS UKB study. This figure captures the flow of this manuscript. We start with the collection of patient samples by UK Biobank and followed by the strategy taken to perform WGS. We continue with quality control performed on GraphTyper and DRAGEN datasets, followed by variant calling of SNPs, in/dels, and structural variants (SV). Thereafter we defined the phenotypes (binary and quantitative) associated with SV, SNPs and at the gene level (rare variant analysis) and conclude with the definition of five ancestry groups and collective association effect as a cross-ancestry meta-analysis. C_FIG\nPervasive correlations between causal disease effects of proximal SNPs vary with functional annotations and implicate stabilizing selection\nAuthors: Zhang, M. J.; Durvasula, A.; Chiang, C.; Koch, E. M.; Strober, B. J.; Shi, H.; Barton, A. R.; Kim, S. S.; Weissbrod, O.; Loh, P.-R.; Gazal, S.; Sunyaev, S.; Price, A.\nScore: 13.2, Published: 2023-12-04 DOI: 10.1101/2023.12.04.23299391\nThe genetic architecture of human diseases and complex traits has been extensively studied, but little is known about the relationship of causal disease effect sizes between proximal SNPs, which have largely been assumed to be independent. We introduce a new method, LD SNP-pair effect correlation regression (LDSPEC), to estimate the correlation of causal disease effect sizes of derived alleles between proximal SNPs, depending on their allele frequencies, LD, and functional annotations; LDSPEC produced robust estimates in simulations across various genetic architectures. We applied LDSPEC to 70 diseases and complex traits from the UK Biobank (average N=306K), meta-analyzing results across diseases/traits. We detected significantly nonzero effect correlations for proximal SNP pairs (e.g., -0.37 {+/-}0.09 for low-frequency positive-LD 0-100bp SNP pairs) that decayed with distance (e.g., -0.07 {+/-}0.01 for low-frequency positive-LD 1-10kb), varied with allele frequency (e.g., -0.15 {+/-}0.04 for common positive-LD 0-100bp), and varied with LD between SNPs (e.g., +0.12 {+/-}0.05 for common negative-LD 0-100bp) (because we consider derived alleles, positive-LD and negative-LD SNP pairs may yield very different results). We further determined that SNP pairs with shared functions had stronger effect correlations that spanned longer genomic distances, e.g., -0.37 {+/-}0.08 for low-frequency positive-LD same-gene promoter SNP pairs (average genomic distance of 47kb (due to alternative splicing)) and -0.32 {+/-}0.04 for low-frequency positive-LD H3K27ac 0-1kb SNP pairs. Consequently, SNP-heritability estimates were substantially smaller than estimates of the sum of causal effect size variances across all SNPs (ratio of 0.87 {+/-}0.02 across diseases/traits), particularly for certain functional annotations (e.g., 0.78 {+/-}0.01 for common Super enhancer SNPs)--even though these quantities are widely assumed to be equal. We recapitulated our findings via forward simulations with an evolutionary model involving stabilizing selection, implicating the action of linkage masking, whereby haplotypes containing linked SNPs with opposite effects on disease have reduced effects on fitness and escape negative selection.\nPolygenic prediction of human longevity on the supposition of pervasive pleiotropy\nAuthors: Jabalameli, M. R.; Lin, J.-R.; Zhang, Q.; Wang, Z.; Mitra, J.; Nguyen, N.; Gao, T.; Khusidman, M.; Atzmon, G.; Milman, S.; Vijg, J.; Barzilai, N.; Zhang, Z.\nScore: 11.8, Published: 2023-12-11 DOI: 10.1101/2023.12.10.23299795\nThe highly polygenic nature of human longevity renders cross-trait pleiotropy an indispensable feature of its genetic architecture. Leveraging the genetic correlation between the aging-related traits (ARTs), we sought to model the additive variance in lifespan as a function of cumulative liability from pleiotropic segregating variants. We tracked allele frequency changes as a function of viability across different age bins and prioritized 34 variants with an immediate implication on lipid metabolism, body mass index (BMI), and cognitive performance, among other traits, revealed by PheWAS analysis in the UK Biobank. Given the highly complex and non-linear interactions between the genetic determinants of longevity, we reasoned that a composite polygenic score would approximate a substantial portion of the variance in lifespan and developed the integrated longevity genetic scores (iLGSs) for distinguishing exceptional survival. We showed that coefficients derived from our ensemble model could potentially reveal an interesting pattern of genomic pleiotropy specific to lifespan. We assessed the predictive performance of our model for distinguishing the enrichment of exceptional longevity among long-lived individuals in two replication cohorts and showed that the median lifespan in the highest decile of our composite prognostic index is up to 4.8 years longer. Finally, using the proteomic correlates of iLGS, we identified protein markers associated with exceptional longevity irrespective of chronological age and prioritized drugs with repurposing potentials for gerotherapeutics. Together, our approach demonstrates a promising framework for polygenic modeling of additive liability conferred by ARTs in defining exceptional longevity and assisting the identification of individuals at higher risk of mortality for targeted lifestyle modifications earlier in life. Furthermore, the proteomic signature associated with iLGS highlights the functional pathway upstream of the PI3K-Akt that can be effectively targeted to slow down aging and extend lifespan.\nLong-read RNA-seq atlas of novel microglia isoforms elucidates disease-associated genetic regulation of splicing\nAuthors: Humphrey, J.; Brophy, E.; Kosoy, R.; Zeng, B.; Coccia, E.; Mattei, D.; Ravi, A.; Efthymiou, A. G.; Navarro, E.; Muller, B. Z.; Snijders, G. J.; Allan, A.; Munch, A.; Birhanu Kitata, R.; Kleopoulos, S. P.; Argyriou, S.; Shao, Z.; Francoeur, N. J.; Tsai, C.-F.; Gritsenko, M. A.; Monroe, M. E.; Paurus, V. L.; Weitz, K. K.; Shi, T.; Sebra, R.; Liu, T.; Hoffman, G. E.; Goate, A. M.; de Witte, L. D.; Bennett, D.; Haroutunian, V.; Fullard, J. F.; Roussos, P.; Raj, T.\nScore: 14.5, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23299073\nMicroglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. We previously mapped the genetic regulation of gene expression and mRNA splicing in human microglia, identifying several loci where common genetic variants in microglia-specific regulatory elements explain disease risk loci identified by GWAS. However, identifying genetic effects on splicing has been challenging due to the use of short sequencing reads to identify causal isoforms. Here we present the isoform-centric microglia genomic atlas (isoMiGA) which leverages the power of long-read RNA-seq to identify 35,879 novel microglia isoforms. We show that the novel microglia isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ethnic meta-analysis of 555 human microglia short-read RNA-seq samples from 391 donors, the largest to date, and found associations with genetic risk loci in Alzheimers disease and Parkinsons disease. We nominate several loci that may act through complex changes in isoform and splice site usage.\nExome-wide tandem repeats confer large effects on subcortical volumes in UK Biobank participants\nAuthors: Panoyan, M. A.; Shi, Y.; Abbatangelo, C. L.; Adler, N.; Moo-Choy, A.; Parra, E.; Polimanti, R.; Hu, P.; Wendt, F.\nScore: 8.3, Published: 2023-12-11 DOI: 10.1101/2023.12.11.23299818\nThe human subcortex is involved in memory and cognition. Structural and functional changes in subcortical regions is implicated in psychiatric conditions. We performed an association study of subcortical volumes using 15,941 tandem repeats (TRs) derived from whole exome sequencing (WES) data in 16,527 unrelated European ancestry participants. We identified 17 loci, most of which were associated with accumbens volume, and nine of which had fine-mapping probability supporting their causal effect on subcortical volume independent of surrounding variation. The most significant association involved NTN1-[GCGG]N and increased accumbens volume ({beta}=5.93, P=8.16x10-9). Three exonic TRs had large effects on thalamus volume (LAT2-[CATC]N {beta}=-949, P=3.84x10-6 and SLC39A4-[CAG]N {beta}=-1599, P=2.42x10-8) and pallidum volume (MCM2-[AGG]N {beta}=-404.9, P=147x10-7). These genetic effects were consistent measurements of per-repeat expansion/contraction effects on organism fitness. With 3-dimensional modeling, we reinforced these effects to show that the expanded and contracted LAT2-[CATC]N repeat causes a frameshift mutation that prevents appropriate protein folding. These TRs also exhibited independent effects on several psychiatric symptoms, including LAT2-[CATC]N and the tiredness/low energy symptom of depression ({beta}=0.340, P=0.003). These findings link genetic variation to tractable biology in the brain and relevant psychiatric symptoms. We also chart one pathway for TR prioritization in future complex trait genetic studies.\nCausal Associations of Sleep Apnea with Alzheimer's Disease and Cardiovascular Disease: a Bidirectional Mendelian Randomization Analysis\nAuthors: Cavailles, C.; Andrews, S. J.; Leng, Y.; Chatterjee, A.; Daghlas, I.; Yaffe, K.\nScore: 9.2, Published: 2023-11-22 DOI: 10.1101/2023.11.20.23298793\nBackgroundSleep apnea (SA) has been linked to an increased risk of dementia in numerous observational studies; whether this is driven by neurodegenerative, vascular or other mechanisms is not clear. We sought to examine the bidirectional causal relationships between SA, Alzheimers disease (AD), coronary artery disease (CAD), and ischemic stroke using Mendelian randomization (MR). MethodsUsing summary statistics from four recent, large genome-wide association studies of SA (n=523,366), AD (n=64,437), CAD (n=1,165,690), and stroke (n=1,308,460), we conducted bidirectional two-sample MR analyses. Our primary analytic method was fixed-effects inverse variance weighted MR; diagnostics tests and sensitivity analyses were conducted to verify the robustness of the results. ResultsWe identified a significant causal effect of SA on the risk of CAD (odds ratio (ORIVW) =1.35 per log-odds increase in SA liability, 95% confidence interval (CI) =1.25-1.47) and stroke (ORIVW=1.13, 95% CI =1.01-1.25). These associations were somewhat attenuated after excluding single-nucleotide polymorphisms associated with body mass index (BMI) (ORIVW=1.26, 95% CI =1.15-1.39 for CAD risk; ORIVW=1.08, 95% CI =0.96-1.22 for stroke risk). SA was not causally associated with a higher risk of AD (ORIVW=1.14, 95% CI =0.91-1.43). We did not find causal effects of AD, CAD, or stroke on risk of SA. ConclusionsThese results suggest that SA increased the risk of CAD, and the identified causal association with stroke risk may be confounded by BMI. Moreover, no causal effect of SA on AD risk was found. Future studies are warranted to investigate cardiovascular pathways between sleep disorders, including SA, and dementia.\nClinical-grade whole genome sequencing-based haplarithmisis enables all forms of preimplantation genetic testing\nAuthors: Janssen, A. E. J.; Koeck, R. M.; Essers, R.; van Dijk, W.; Drusedau, M.; Meekels, J.; Yaldiz, B.; van de Vorst, M.; Cao, P.; de Koning, B.; Hellebrekers, D. M. E. I.; Stevens, S. J. C.; Sun, S. M.; Heijligers, M.; de Munnik, S. A.; van Uum, C. M. J.; Achten, J.; Hamers, L.; Naghdi, M.; Vissers, L. E. L. M.; van Golde, R. J. T.; de Wert, G.; Dreesen, J. C. F. M.; de Die-Smulders, C.; Coonen, E.; Brunner, H. G.; van den Wijngaard, A.; Paulussen, A. D. C.; Zamani Esteki, M.\nScore: 6.8, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299605\nHigh-throughput sequencing technologies have increasingly led to discovery of disease-causing genetic variants, primarily in postnatal multi-cell DNA samples. However, applying these technologies to preimplantation genetic testing (PGT) in nuclear or mitochondrial DNA from single or few-cells biopsied from in vitro fertilised (IVF) embryos is challenging. PGT aims to select IVF embryos without genetic abnormalities. Although genotyping-by-sequencing (GBS)-based haplotyping methods enabled PGT for monogenic disorders (PGT-M), structural rearrangements (PGT-SR), and aneuploidies (PGT-A), they are labour intensive, only partially cover the genome and are troublesome for difficult loci and consanguineous couples. Here, we devised a simple, scalable and universal whole genome sequencing haplarithmisis-based approach enabling all forms of PGT in a single assay. In a comparison to state-of-the-art GBS-based PGT for nuclear DNA (37 embryos, 18 families, 25 indications), shallow sequencing-based PGT (10 embryos, 3 families), and PCR-based PGT for mitochondrial DNA (10 embryos, 2 families), our approach alleviates technical limitations by decreasing whole genome amplification artifacts by 68.4%, increasing breadth of coverage by 4-fold, and reducing wet-lab turn-around-time by 2.5-fold. Importantly, this method enables trio-based PGT-A for aneuploidy origin, an approach we coin PGT-AO, detects translocation breakpoints, and nuclear and mitochondrial single nucleotide variants and indels in base-resolution.\nGenetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes\nAuthors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.\nScore: 31.5, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23299014\nThe biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).\nEndothelial sampling in situ enables genetic characterization of vein of Galen Malformation\nAuthors: Hale, A. T.; Liu, S.; Huang, F.; Song, Y.; Crowley, M. R.; Crossman, D. K.; Caudill, C.; Smith, A. A.; Chapman, L.; Feldman, M. J.; Saccomano, B. W.; Rocque, B. G.; Rozzelle, C. J.; Blount, J. P.; Johnston, J. M.; Chong, Z.; Jones, J. G.\nScore: 3.9, Published: 2023-12-05 DOI: 10.1101/2023.12.01.23299300\nBackground and ObjectivesVein of Galen malformation (VOGM) is the result of arteriovenous shunting between choroidal and/or subependymal arteries and the embryologic prosencephalic vein and is among the most severe cerebrovascular disorders of childhood. While endovascular treatments have improved outcomes, morbidity and mortality remain high. We hypothesized that in situ analysis of the VOGM lesion using endoluminal tissue sampling (ETS) is feasible and may identify somatic mutations and transcriptional aberrations. A mechanistic understanding of VOGM genetics, pathogenesis, and maintenance will guide future therapeutic efforts. MethodWe performed ETS under conditions where clinical factors alone determined vessel selection and access route. Coils were only placed in vessel locations desired for treatment. After a coil was deployed, we allowed 1 minute of endoluminal contact time before the coil was resheathed through the microcatheter, protecting it from contact with other vessel segments. Once the coil was removed, it was rinsed in PBS and dissociated with trypsin before flow cytometry was used to isolate experimental endothelial cells (EC), defined as CD31+ and CD45-. Single-cell RNA sequencing (scRNA-seq) of VOGM ECs was performed using autologous peripheral femoral access ECs as a control to demonstrate feasibility of downstream genomic analyses from ETS. ResultsOur cohort contains 6 patients who underwent 10 ETS procedures from arterial and/or venous access during routine VOGM treatment (range: 12 days to [~]6 years). No periprocedural complications attributable to ETS occurred. Six unique coil types were used. ETS captured 98 {+/-} 88 (mean {+/-} SD; range 17-256) experimental endothelial cells (CD31+ and CD45-). There was no discernable correlation between cell yield and coil type or route of access. Single cell RNA sequencing demonstrated hierarchical clustering and unique cell populations within the VOGM lesion compared to autologous femoral control ECs. ConclusionWe report the first successful utilization of ETS for VOGM. ETS appears safe and may supplement investigations aimed at development of a molecular-genetic taxonomic classification scheme. Moreover, results may eventually inform the selection of personalized pharmacologic or genetic therapies for VOGM and cerebrovascular disorders more broadly.\nIrritability in young people with copy number variants associated with neurodevelopmental disorders (ND-CNVs)\nAuthors: Hall, J. H.; Chawner, S.; IMAGINE-ID Consortium, ; Wolstencroft, J.; Skuse, D.; Holmans, P. H.; Owen, M.; van den Bree, M. B. M.\nScore: 2.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299440\nBackgroundA range of rare mutations involving micro-deletion or -duplication of genetic material (copy number variants (CNVs)) have been associated with high neurodevelopmental and psychiatric risk (ND-CNVs). Irritability is frequently observed in childhood neurodevelopmental conditions, yet its aetiology is largely unknown. Genetic variation may play a role, but there is a sparsity of studies investigating presentation of irritability in young people with ND-CNVs. AimsThis study aimed to investigate whether there is a difference in irritability in young people with rare ND-CNVs compared to those without ND-CNVs, and to what extent irritability is associated with psychiatric diagnoses and cognitive ability (IQ). MethodsIrritability and broader psychopathology was assessed in 485 young people with ND-CNVs and 164 sibling controls, using the child and adolescent psychiatric assessment (CAPA). Autism was assessed using the Social Communication Questionnaire (SCQ), and Intelligence Quotient (IQ) by the Wechsler Abbreviated Scale of Intelligence (WASI). Results54% of young people with ND-CNVs met the threshold for irritability; significantly more than controls (OR = 3.77, CI = 3.07-7.90, p= 5.31 x 10-11). When controlling for the presence of other psychiatric comorbidities, ND-CNV status was still associated with irritability. There was no evidence for a relationship between irritability and IQ. ConclusionsIrritability is an important aspect of the clinical picture in young people with ND-CNVs. This work shows that genetic variation is associated with irritability in young people with ND-CNVs, independent of psychiatric comorbidities or IQ impairment. Clinicians should be aware of this increased risk to inform management and interventions.\n",
  "wordCount" : "3082",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.23299426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.23299426" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.23299426">
        <p class="paperTitle">Whole-genome sequencing of half-a-million UK Biobank participants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.23299426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.23299426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: The UK Biobank Whole-Genome Sequencing Consortium,  ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.</p>
        <p class="info">Score: 65.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.23299426' target='https://doi.org/10.1101/2023.12.06.23299426'> 10.1101/2023.12.06.23299426</a></p>
        <p class="abstract">Summary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3. This advance deepens our understanding of how genetics influences disease biology and further enhances the value of this open resource for the study of human biology and health. Coupling this dataset with rich phenotypic data, we surveyed within- and cross-ancestry genomic associations with health-related phenotypes and identified novel genetic and clinical insights. While most genome-wide significant associations with disease traits were primarily observed in Europeans, we also identified strong or novel signals in individuals of African and Asian ancestries. Deeper capture of exonic variation in both coding and UTR sequences, strengthened and surfaced novel insights relative to WES analyses. This landmark dataset, representing the largest collection of WGS and available to the UK Biobank research community, will enable advances into our understanding of the human genome, and facilitate the discovery of new diagnostics, therapeutics with higher efficacy and improved safety profile, and enable precision medicine strategies with the potential to improve global health.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=114 SRC=&#34;FIGDIR/small/23299426v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (22K):
org.highwire.dtl.DTLVardef@e03bc2org.highwire.dtl.DTLVardef@1736602org.highwire.dtl.DTLVardef@12e4050org.highwire.dtl.DTLVardef@1805b5c_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOGraphic summary.C_FLOATNO Framework of the WGS UKB study. This figure captures the flow of this manuscript. We start with the collection of patient samples by UK Biobank and followed by the strategy taken to perform WGS. We continue with quality control performed on GraphTyper and DRAGEN datasets, followed by variant calling of SNPs, in/dels, and structural variants (SV). Thereafter we defined the phenotypes (binary and quantitative) associated with SV, SNPs and at the gene level (rare variant analysis) and conclude with the definition of five ancestry groups and collective association effect as a cross-ancestry meta-analysis.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.23299391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.23299391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.23299391">
        <p class="paperTitle">Pervasive correlations between causal disease effects of proximal SNPs vary with functional annotations and implicate stabilizing selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.23299391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.23299391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, M. J.; Durvasula, A.; Chiang, C.; Koch, E. M.; Strober, B. J.; Shi, H.; Barton, A. R.; Kim, S. S.; Weissbrod, O.; Loh, P.-R.; Gazal, S.; Sunyaev, S.; Price, A.</p>
        <p class="info">Score: 13.2, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.23299391' target='https://doi.org/10.1101/2023.12.04.23299391'> 10.1101/2023.12.04.23299391</a></p>
        <p class="abstract">The genetic architecture of human diseases and complex traits has been extensively studied, but little is known about the relationship of causal disease effect sizes between proximal SNPs, which have largely been assumed to be independent. We introduce a new method, LD SNP-pair effect correlation regression (LDSPEC), to estimate the correlation of causal disease effect sizes of derived alleles between proximal SNPs, depending on their allele frequencies, LD, and functional annotations; LDSPEC produced robust estimates in simulations across various genetic architectures. We applied LDSPEC to 70 diseases and complex traits from the UK Biobank (average N=306K), meta-analyzing results across diseases/traits. We detected significantly nonzero effect correlations for proximal SNP pairs (e.g., -0.37 {&#43;/-}0.09 for low-frequency positive-LD 0-100bp SNP pairs) that decayed with distance (e.g., -0.07 {&#43;/-}0.01 for low-frequency positive-LD 1-10kb), varied with allele frequency (e.g., -0.15 {&#43;/-}0.04 for common positive-LD 0-100bp), and varied with LD between SNPs (e.g., &#43;0.12 {&#43;/-}0.05 for common negative-LD 0-100bp) (because we consider derived alleles, positive-LD and negative-LD SNP pairs may yield very different results). We further determined that SNP pairs with shared functions had stronger effect correlations that spanned longer genomic distances, e.g., -0.37 {&#43;/-}0.08 for low-frequency positive-LD same-gene promoter SNP pairs (average genomic distance of 47kb (due to alternative splicing)) and -0.32 {&#43;/-}0.04 for low-frequency positive-LD H3K27ac 0-1kb SNP pairs. Consequently, SNP-heritability estimates were substantially smaller than estimates of the sum of causal effect size variances across all SNPs (ratio of 0.87 {&#43;/-}0.02 across diseases/traits), particularly for certain functional annotations (e.g., 0.78 {&#43;/-}0.01 for common Super enhancer SNPs)--even though these quantities are widely assumed to be equal. We recapitulated our findings via forward simulations with an evolutionary model involving stabilizing selection, implicating the action of linkage masking, whereby haplotypes containing linked SNPs with opposite effects on disease have reduced effects on fitness and escape negative selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.23299795">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.23299795" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.23299795">
        <p class="paperTitle">Polygenic prediction of human longevity on the supposition of pervasive pleiotropy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.23299795" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.23299795" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jabalameli, M. R.; Lin, J.-R.; Zhang, Q.; Wang, Z.; Mitra, J.; Nguyen, N.; Gao, T.; Khusidman, M.; Atzmon, G.; Milman, S.; Vijg, J.; Barzilai, N.; Zhang, Z.</p>
        <p class="info">Score: 11.8, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.23299795' target='https://doi.org/10.1101/2023.12.10.23299795'> 10.1101/2023.12.10.23299795</a></p>
        <p class="abstract">The highly polygenic nature of human longevity renders cross-trait pleiotropy an indispensable feature of its genetic architecture. Leveraging the genetic correlation between the aging-related traits (ARTs), we sought to model the additive variance in lifespan as a function of cumulative liability from pleiotropic segregating variants. We tracked allele frequency changes as a function of viability across different age bins and prioritized 34 variants with an immediate implication on lipid metabolism, body mass index (BMI), and cognitive performance, among other traits, revealed by PheWAS analysis in the UK Biobank. Given the highly complex and non-linear interactions between the genetic determinants of longevity, we reasoned that a composite polygenic score would approximate a substantial portion of the variance in lifespan and developed the integrated longevity genetic scores (iLGSs) for distinguishing exceptional survival. We showed that coefficients derived from our ensemble model could potentially reveal an interesting pattern of genomic pleiotropy specific to lifespan. We assessed the predictive performance of our model for distinguishing the enrichment of exceptional longevity among long-lived individuals in two replication cohorts and showed that the median lifespan in the highest decile of our composite prognostic index is up to 4.8 years longer. Finally, using the proteomic correlates of iLGS, we identified protein markers associated with exceptional longevity irrespective of chronological age and prioritized drugs with repurposing potentials for gerotherapeutics. Together, our approach demonstrates a promising framework for polygenic modeling of additive liability conferred by ARTs in defining exceptional longevity and assisting the identification of individuals at higher risk of mortality for targeted lifestyle modifications earlier in life. Furthermore, the proteomic signature associated with iLGS highlights the functional pathway upstream of the PI3K-Akt that can be effectively targeted to slow down aging and extend lifespan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23299073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23299073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23299073">
        <p class="paperTitle">Long-read RNA-seq atlas of novel microglia isoforms elucidates disease-associated genetic regulation of splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23299073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23299073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Humphrey, J.; Brophy, E.; Kosoy, R.; Zeng, B.; Coccia, E.; Mattei, D.; Ravi, A.; Efthymiou, A. G.; Navarro, E.; Muller, B. Z.; Snijders, G. J.; Allan, A.; Munch, A.; Birhanu Kitata, R.; Kleopoulos, S. P.; Argyriou, S.; Shao, Z.; Francoeur, N. J.; Tsai, C.-F.; Gritsenko, M. A.; Monroe, M. E.; Paurus, V. L.; Weitz, K. K.; Shi, T.; Sebra, R.; Liu, T.; Hoffman, G. E.; Goate, A. M.; de Witte, L. D.; Bennett, D.; Haroutunian, V.; Fullard, J. F.; Roussos, P.; Raj, T.</p>
        <p class="info">Score: 14.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23299073' target='https://doi.org/10.1101/2023.12.01.23299073'> 10.1101/2023.12.01.23299073</a></p>
        <p class="abstract">Microglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. We previously mapped the genetic regulation of gene expression and mRNA splicing in human microglia, identifying several loci where common genetic variants in microglia-specific regulatory elements explain disease risk loci identified by GWAS. However, identifying genetic effects on splicing has been challenging due to the use of short sequencing reads to identify causal isoforms. Here we present the isoform-centric microglia genomic atlas (isoMiGA) which leverages the power of long-read RNA-seq to identify 35,879 novel microglia isoforms. We show that the novel microglia isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ethnic meta-analysis of 555 human microglia short-read RNA-seq samples from 391 donors, the largest to date, and found associations with genetic risk loci in Alzheimers disease and Parkinsons disease. We nominate several loci that may act through complex changes in isoform and splice site usage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.23299818">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.23299818" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.23299818">
        <p class="paperTitle">Exome-wide tandem repeats confer large effects on subcortical volumes in UK Biobank participants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.23299818" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.23299818" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Panoyan, M. A.; Shi, Y.; Abbatangelo, C. L.; Adler, N.; Moo-Choy, A.; Parra, E.; Polimanti, R.; Hu, P.; Wendt, F.</p>
        <p class="info">Score: 8.3, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.23299818' target='https://doi.org/10.1101/2023.12.11.23299818'> 10.1101/2023.12.11.23299818</a></p>
        <p class="abstract">The human subcortex is involved in memory and cognition. Structural and functional changes in subcortical regions is implicated in psychiatric conditions. We performed an association study of subcortical volumes using 15,941 tandem repeats (TRs) derived from whole exome sequencing (WES) data in 16,527 unrelated European ancestry participants. We identified 17 loci, most of which were associated with accumbens volume, and nine of which had fine-mapping probability supporting their causal effect on subcortical volume independent of surrounding variation. The most significant association involved NTN1-[GCGG]N and increased accumbens volume ({beta}=5.93, P=8.16x10-9). Three exonic TRs had large effects on thalamus volume (LAT2-[CATC]N {beta}=-949, P=3.84x10-6 and SLC39A4-[CAG]N {beta}=-1599, P=2.42x10-8) and pallidum volume (MCM2-[AGG]N {beta}=-404.9, P=147x10-7). These genetic effects were consistent measurements of per-repeat expansion/contraction effects on organism fitness. With 3-dimensional modeling, we reinforced these effects to show that the expanded and contracted LAT2-[CATC]N repeat causes a frameshift mutation that prevents appropriate protein folding. These TRs also exhibited independent effects on several psychiatric symptoms, including LAT2-[CATC]N and the tiredness/low energy symptom of depression ({beta}=0.340, P=0.003). These findings link genetic variation to tractable biology in the brain and relevant psychiatric symptoms. We also chart one pathway for TR prioritization in future complex trait genetic studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.23298793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.23298793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.23298793">
        <p class="paperTitle">Causal Associations of Sleep Apnea with Alzheimer&#39;s Disease and Cardiovascular Disease: a Bidirectional Mendelian Randomization Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.23298793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.23298793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cavailles, C.; Andrews, S. J.; Leng, Y.; Chatterjee, A.; Daghlas, I.; Yaffe, K.</p>
        <p class="info">Score: 9.2, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.23298793' target='https://doi.org/10.1101/2023.11.20.23298793'> 10.1101/2023.11.20.23298793</a></p>
        <p class="abstract">BackgroundSleep apnea (SA) has been linked to an increased risk of dementia in numerous observational studies; whether this is driven by neurodegenerative, vascular or other mechanisms is not clear. We sought to examine the bidirectional causal relationships between SA, Alzheimers disease (AD), coronary artery disease (CAD), and ischemic stroke using Mendelian randomization (MR).

MethodsUsing summary statistics from four recent, large genome-wide association studies of SA (n=523,366), AD (n=64,437), CAD (n=1,165,690), and stroke (n=1,308,460), we conducted bidirectional two-sample MR analyses. Our primary analytic method was fixed-effects inverse variance weighted MR; diagnostics tests and sensitivity analyses were conducted to verify the robustness of the results.

ResultsWe identified a significant causal effect of SA on the risk of CAD (odds ratio (ORIVW) =1.35 per log-odds increase in SA liability, 95% confidence interval (CI) =1.25-1.47) and stroke (ORIVW=1.13, 95% CI =1.01-1.25). These associations were somewhat attenuated after excluding single-nucleotide polymorphisms associated with body mass index (BMI) (ORIVW=1.26, 95% CI =1.15-1.39 for CAD risk; ORIVW=1.08, 95% CI =0.96-1.22 for stroke risk). SA was not causally associated with a higher risk of AD (ORIVW=1.14, 95% CI =0.91-1.43). We did not find causal effects of AD, CAD, or stroke on risk of SA.

ConclusionsThese results suggest that SA increased the risk of CAD, and the identified causal association with stroke risk may be confounded by BMI. Moreover, no causal effect of SA on AD risk was found. Future studies are warranted to investigate cardiovascular pathways between sleep disorders, including SA, and dementia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.23299605">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.23299605" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.23299605">
        <p class="paperTitle">Clinical-grade whole genome sequencing-based haplarithmisis enables all forms of preimplantation genetic testing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.23299605" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.23299605" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janssen, A. E. J.; Koeck, R. M.; Essers, R.; van Dijk, W.; Drusedau, M.; Meekels, J.; Yaldiz, B.; van de Vorst, M.; Cao, P.; de Koning, B.; Hellebrekers, D. M. E. I.; Stevens, S. J. C.; Sun, S. M.; Heijligers, M.; de Munnik, S. A.; van Uum, C. M. J.; Achten, J.; Hamers, L.; Naghdi, M.; Vissers, L. E. L. M.; van Golde, R. J. T.; de Wert, G.; Dreesen, J. C. F. M.; de Die-Smulders, C.; Coonen, E.; Brunner, H. G.; van den Wijngaard, A.; Paulussen, A. D. C.; Zamani Esteki, M.</p>
        <p class="info">Score: 6.8, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.23299605' target='https://doi.org/10.1101/2023.12.06.23299605'> 10.1101/2023.12.06.23299605</a></p>
        <p class="abstract">High-throughput sequencing technologies have increasingly led to discovery of disease-causing genetic variants, primarily in postnatal multi-cell DNA samples. However, applying these technologies to preimplantation genetic testing (PGT) in nuclear or mitochondrial DNA from single or few-cells biopsied from in vitro fertilised (IVF) embryos is challenging. PGT aims to select IVF embryos without genetic abnormalities. Although genotyping-by-sequencing (GBS)-based haplotyping methods enabled PGT for monogenic disorders (PGT-M), structural rearrangements (PGT-SR), and aneuploidies (PGT-A), they are labour intensive, only partially cover the genome and are troublesome for difficult loci and consanguineous couples. Here, we devised a simple, scalable and universal whole genome sequencing haplarithmisis-based approach enabling all forms of PGT in a single assay. In a comparison to state-of-the-art GBS-based PGT for nuclear DNA (37 embryos, 18 families, 25 indications), shallow sequencing-based PGT (10 embryos, 3 families), and PCR-based PGT for mitochondrial DNA (10 embryos, 2 families), our approach alleviates technical limitations by decreasing whole genome amplification artifacts by 68.4%, increasing breadth of coverage by 4-fold, and reducing wet-lab turn-around-time by 2.5-fold. Importantly, this method enables trio-based PGT-A for aneuploidy origin, an approach we coin PGT-AO, detects translocation breakpoints, and nuclear and mitochondrial single nucleotide variants and indels in base-resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.23299014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.23299014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.23299014">
        <p class="paperTitle">Genetic determinants of blood gene expression and splicing and their contribution to molecular phenotypes and health outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.23299014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.23299014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tokolyi, A.; Persyn, E.; Nath, A. P.; Burnham, K. L.; Marten, J.; Vanderstichele, T.; Tardaguila, M.; Stacey, D.; Farr, B.; Iyer, V.; Jiang, X.; Lambert, S. A.; Noell, G.; Quail, M. A.; Rajan, D.; Ritchie, S. C.; Sun, B. B.; Thurston, S. A. J.; Xu, Y.; Whelan, C. D.; Runz, H.; Petrovski, S.; Gaffney, D. J.; Roberts, D. J.; Di Angelantonio, E.; Peters, J. E.; Soranzo, N.; Danesh, J.; Butterworth, A. S.; Inouye, M.; Davenport, E. E.; Paul, D. S.</p>
        <p class="info">Score: 31.5, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.23299014' target='https://doi.org/10.1101/2023.11.25.23299014'> 10.1101/2023.11.25.23299014</a></p>
        <p class="abstract">The biological mechanisms through which most non-protein-coding genetic variants affect disease risk are unknown. To investigate the gene-regulatory cascades that ensue from these variants, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA-sequencing in 4,732 participants, and integrated these data with protein, metabolite and lipid QTLs in the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Using colocalization analysis, we identified 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology through statistical mediation, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at GWAS disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access and interactive resource of the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans (https://IntervalRNA.org.uk).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23299300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23299300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23299300">
        <p class="paperTitle">Endothelial sampling in situ enables genetic characterization of vein of Galen Malformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23299300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23299300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hale, A. T.; Liu, S.; Huang, F.; Song, Y.; Crowley, M. R.; Crossman, D. K.; Caudill, C.; Smith, A. A.; Chapman, L.; Feldman, M. J.; Saccomano, B. W.; Rocque, B. G.; Rozzelle, C. J.; Blount, J. P.; Johnston, J. M.; Chong, Z.; Jones, J. G.</p>
        <p class="info">Score: 3.9, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23299300' target='https://doi.org/10.1101/2023.12.01.23299300'> 10.1101/2023.12.01.23299300</a></p>
        <p class="abstract">Background and ObjectivesVein of Galen malformation (VOGM) is the result of arteriovenous shunting between choroidal and/or subependymal arteries and the embryologic prosencephalic vein and is among the most severe cerebrovascular disorders of childhood. While endovascular treatments have improved outcomes, morbidity and mortality remain high. We hypothesized that in situ analysis of the VOGM lesion using endoluminal tissue sampling (ETS) is feasible and may identify somatic mutations and transcriptional aberrations. A mechanistic understanding of VOGM genetics, pathogenesis, and maintenance will guide future therapeutic efforts.

MethodWe performed ETS under conditions where clinical factors alone determined vessel selection and access route. Coils were only placed in vessel locations desired for treatment. After a coil was deployed, we allowed 1 minute of endoluminal contact time before the coil was resheathed through the microcatheter, protecting it from contact with other vessel segments. Once the coil was removed, it was rinsed in PBS and dissociated with trypsin before flow cytometry was used to isolate experimental  endothelial cells (EC), defined as CD31&#43; and CD45-. Single-cell RNA sequencing (scRNA-seq) of VOGM ECs was performed using autologous peripheral femoral access ECs as a control to demonstrate feasibility of downstream genomic analyses from ETS.

ResultsOur cohort contains 6 patients who underwent 10 ETS procedures from arterial and/or venous access during routine VOGM treatment (range: 12 days to [~]6 years). No periprocedural complications attributable to ETS occurred. Six unique coil types were used. ETS captured 98 {&#43;/-} 88 (mean {&#43;/-} SD; range 17-256) experimental  endothelial cells (CD31&#43; and CD45-). There was no discernable correlation between cell yield and coil type or route of access. Single cell RNA sequencing demonstrated hierarchical clustering and unique cell populations within the VOGM lesion compared to autologous femoral control ECs.

ConclusionWe report the first successful utilization of ETS for VOGM. ETS appears safe and may supplement investigations aimed at development of a molecular-genetic taxonomic classification scheme. Moreover, results may eventually inform the selection of personalized pharmacologic or genetic therapies for VOGM and cerebrovascular disorders more broadly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299440">
        <p class="paperTitle">Irritability in young people with copy number variants associated with neurodevelopmental disorders (ND-CNVs)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hall, J. H.; Chawner, S.; IMAGINE-ID Consortium,  ; Wolstencroft, J.; Skuse, D.; Holmans, P. H.; Owen, M.; van den Bree, M. B. M.</p>
        <p class="info">Score: 2.8, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299440' target='https://doi.org/10.1101/2023.12.05.23299440'> 10.1101/2023.12.05.23299440</a></p>
        <p class="abstract">BackgroundA range of rare mutations involving micro-deletion or -duplication of genetic material (copy number variants (CNVs)) have been associated with high neurodevelopmental and psychiatric risk (ND-CNVs). Irritability is frequently observed in childhood neurodevelopmental conditions, yet its aetiology is largely unknown. Genetic variation may play a role, but there is a sparsity of studies investigating presentation of irritability in young people with ND-CNVs.

AimsThis study aimed to investigate whether there is a difference in irritability in young people with rare ND-CNVs compared to those without ND-CNVs, and to what extent irritability is associated with psychiatric diagnoses and cognitive ability (IQ).

MethodsIrritability and broader psychopathology was assessed in 485 young people with ND-CNVs and 164 sibling controls, using the child and adolescent psychiatric assessment (CAPA). Autism was assessed using the Social Communication Questionnaire (SCQ), and Intelligence Quotient (IQ) by the Wechsler Abbreviated Scale of Intelligence (WASI).

Results54% of young people with ND-CNVs met the threshold for irritability; significantly more than controls (OR = 3.77, CI = 3.07-7.90, p= 5.31 x 10-11). When controlling for the presence of other psychiatric comorbidities, ND-CNV status was still associated with irritability. There was no evidence for a relationship between irritability and IQ.

ConclusionsIrritability is an important aspect of the clinical picture in young people with ND-CNVs. This work shows that genetic variation is associated with irritability in young people with ND-CNVs, independent of psychiatric comorbidities or IQ impairment. Clinicians should be aware of this increased risk to inform management and interventions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
