<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations
Authors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences, ; Wood, C.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations
Authors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences, ; Wood, C." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations
Authors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences, ; Wood, C."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations\nAuthors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences, ; Wood, C.",
  "keywords": [
    
  ],
  "articleBody": " Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations\nAuthors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences, ; Wood, C. R.; Kriegl, J. M.; Podduturi, N.; Jensen, J. N.; Stutzki, J.; Ding, Z.\nScore: 26.9, Published: 2023-12-22 DOI: 10.1101/2023.12.20.23300308\nAdvancements in long-read sequencing technology have accelerated the study of large structural variants (SVs). We created a curated, publicly available, multi-ancestry SV imputation panel by long-read sequencing 888 samples from the 1000 Genomes Project. This high-quality panel was used to impute SVs in approximately 500,000 UK Biobank participants. We demonstrated the feasibility of conducting genome-wide SV association studies at biobank scale using 32 disease-relevant phenotypes related to respiratory, cardiometabolic and liver diseases, in addition to 1,463 protein levels. This analysis identified thousands of genome-wide significant SV associations, including hundreds of conditionally independent signals, thereby enabling novel biological insights. Focusing on genetic association studies of lung function as an example, we demonstrate the added value of SVs for prioritising causal genes at gene-rich loci compared to traditional GWAS using only short variants. We envision that future post-GWAS gene-prioritisation workflows will incorporate SV analyses using this SV imputation panel and framework.\nA rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand\nAuthors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D.; Amnueypol, M.; Krittayaphong, R.; Prechawat, S.; Anannab, A.; Makarawate, P.; Ngarmukos, T.; Veerakul, G.; Kingsbury, Z.; Newington, T.; Maheswari, U.; Ross, M. T.; Grace, A.; Lambiase, P. D.; Behr, E. R.; Schott, J.-J.; Redon, R.; Barc, J.\nScore: 12.9, Published: 2023-12-20 DOI: 10.1101/2023.12.19.23299785\nBrugada syndrome (BrS) is a cardiac arrhythmia disorder that causes sudden death in young adults. Rare genetic variants in the SCN5A gene, encoding the Nav1.5 sodium channel, and common non-coding variants at this locus, are robustly associated with the condition. BrS is particularly prevalent in Southeast Asia but the underlying ancestry-specific factors remain largely unknown. Here, we performed genome sequencing of BrS probands from Thailand and population-matched controls and identified a rare non-coding variant in an SCN5A intronic enhancer that is highly enriched in BrS cases (3.9% in cases, odds ratio 20.2-45.2) and predicted to disrupt a Mef2 transcription factor binding site. Heterozygous introduction of the enhancer variant in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) caused significantly reduced SCN5A expression from the variant-containing allele and a 30% reduction in Nav1.5-mediated sodium-current density compared to isogenic controls. This is the first example of a validated rare non-coding variant at the SCN5A locus and partly explains the increased prevalence of BrS in this geographic region.\nEmpowering GWAS Discovery through Enhanced Genotype Imputation\nAuthors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.\nScore: 9.6, Published: 2023-12-21 DOI: 10.1101/2023.12.18.23300143\nGenotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.\nMulti-site validation of a functional assay to adjudicate SCN5A Brugada Syndrome-associated variants\nAuthors: Ma, J. G.; O'Neill, M. J.; Richardson, E.; Thomson, K.; Ingles, J.; Muhammad, A.; Solus, J. F.; Davogustto, G. E.; Anderson, K. C.; Shoemaker, M. B.; Stergachis, A. B.; Floyd, B. J.; Dunn, K.; Parikh, V. N.; Chubb, H.; Perrin, M.; Roden, D. M.; Vandenberg, J. I.; Ng, C. A.; Glazer, A. M.\nScore: 7.6, Published: 2023-12-20 DOI: 10.1101/2023.12.19.23299592\nBrugada Syndrome (BrS) is an inheritable arrhythmia condition that is associated with rare, loss-of-function variants in the cardiac sodium channel gene, SCN5A. Interpreting the pathogenicity of SCN5A missense variants is challenging and [~]79% of SCN5A missense variants in ClinVar are currently classified as Variants of Uncertain Significance (VUS). An in vitro SCN5A-BrS automated patch clamp assay was generated for high-throughput functional studies of NaV1.5. The assay was independently studied at two separate research sites - Vanderbilt University Medical Center and Victor Chang Cardiac Research Institute - revealing strong correlations, including peak INa density (R2=0.86). The assay was calibrated according to ClinGen Sequence Variant Interpretation recommendations using high-confidence variant controls (n=49). Normal and abnormal ranges of function were established based on the distribution of benign variant assay results. The assay accurately distinguished benign controls (24/25) from pathogenic controls (23/24). Odds of Pathogenicity values derived from the experimental results yielded 0.042 for normal function (BS3 criterion) and 24.0 for abnormal function (PS3 criterion), resulting in up to strong evidence for both ACMG criteria. The calibrated assay was then used to study SCN5A VUS observed in four families with BrS and other arrhythmia phenotypes associated with SCN5A loss-of-function. The assay revealed loss-of-function for three of four variants, enabling reclassification to likely pathogenic. This validated APC assay provides clinical-grade functional evidence for the reclassification of current VUS and will aid future SCN5A-BrS variant classification.\nIdentification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis\nAuthors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.\nScore: 25.4, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893\nClonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.\nA maternal germline mutator phenotype in a family affected by heritable colorectal cancer\nAuthors: Young, C. L.; Beichman, A. C.; Mas Ponte, D.; Hemker, S. L.; Zhu, L.; Kitzman, J.; Shirts, B. H.; Harris, K.\nScore: 11.3, Published: 2023-12-18 DOI: 10.1101/2023.12.08.23299304\nVariation in DNA repair genes can increase cancer risk by elevating the rate of oncogenic mutation. Defects in one such gene, MUTYH, are known to elevate the incidence of colorectal cancer in a recessive Mendelian manner, and some evidence has also linked MUTYH to elevated incidence of other cancers as well as elevated mutation rates in normal somatic and germline cells. Here, we use whole genome sequencing to measure germline de novo mutation rates in a large extended family affected by pathogenic MUTYH variation and a history of colorectal cancer. Although this familys genotype, p.Y179C/V234M (c.536A\u003eG/700G\u003eA on transcript NM_001128425), contains a variant with conflicting functional interpretations, we use an in vitro cell line assay to determine that it partially attenuates MUTYHs function. In the children of mothers affected by the Y179C/V234M genotype, we identify an elevation of the C\u003eA mutation rate that is weaker than mutator effects previously reported to be caused by other pathogenic MUTYH genotypes, suggesting that mutation rates in normal tissues may be useful for classifying cancer-associated variation along a continuum of severity. Surprisingly, we detect no significant elevation of the C\u003eA mutation rate in children born to a father with the same biallelic MUTYH genotype, despite calculating that we should have adequate power to detect such a mutator effect. This suggests that the oxidative stress repaired by MUTYH may contribute more to female reproductive aging than male reproductive aging in the general population.\nRare disease gene association discovery from burden analysis of the 100,000 Genomes Project data\nAuthors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O'Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra\nScore: 5.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300294\nTo discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.\nGenome-wide characterization of 54 urinary metabolites reveals molecular impact of kidney function\nAuthors: Valo, E.; Richmond, A.; Mutter, S.; Campbell, A.; Porteous, D.; Wilson, J. F.; FinnDiane Study Group, ; Groop, P.-H.; Hayward, C.; Sandholm, N.\nScore: 2.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300206\nDissecting the genetic mechanisms underlying urinary metabolite concentrations can provide molecular insights into kidney function and open possibilities for causal assessment of urinary metabolites with risk factors and disease outcomes. Proton nuclear magnetic resonance metabolomics provides a high-throughput means for urinary metabolite profiling, as widely applied for blood biomarker studies. Here we report a genome-wide association study meta-analysed for 3 European cohorts comprising 8,026 individuals, covering both people with type 1 diabetes and general population settings. We identified 52 associations (p\u003c9.3x10-10) for 19 of 54 studied metabolite concentrations. Out of these, 32 were not reported previously for relevant urinary or blood metabolite traits. Subsequent two-sample Mendelian randomization analysis suggests that estimated glomerular filtration rate (eGFR) causally affects 13 urinary metabolite concentrations whereas urinary ethanolamine, an initial precursor for phosphatidylcholine and phosphatidylethanolamine, was associated with higher eGFR lending support for a potential protective role.\nProgress Update for the Multisite Optical Genome Mapping Evaluation and Validation Study: Prenatal Applications\nAuthors: Levy, B.; Liu, J.; Iqbal, A.; Dupont, B.; Sahajpal, N.; Ho, M.; Yu, J.; Brody, S.; Mason-Suares, H.; Ganapathi, M.; Rajkovic, A.; Smolarek, T.; Toydemir, R.; Bui, P.; Kolhe, R.; Stevenson, R.\nScore: 1.9, Published: 2023-12-26 DOI: 10.1101/2023.12.22.23300469\nOptical genome mapping (OGM) is an emerging technology with great potential for prenatal diagnosis. OGM can identify and resolve all types of balanced and unbalanced cytogenomic abnormalities in a single test, which are typically assessed by multiple standard of care (SOC) methods including karyotyping, fluorescence in situ hybridization and chromosomal microarray. To assess OGM's viability as an alternative to conventional SOC testing, a comprehensive clinical research study was conducted across multiple sites, operators, and instruments to evaluate its accuracy and clinical utility. This report provides an update for the phase 2 results of the ongoing multisite evaluation and validation study evaluating OGM for prenatal applications. In phase 1, 123 prenatal cases were assessed by OGM, and in phase 2, 219 retrospective and prospective prenatal cases have been evaluated. For 71% of cases, at least two SOC tests were performed. The study found that OGM had an overall accuracy of 99.6% and positive predictive value of 100% when compared to all cytogenetic SOC results. With its standardized workflow, cost-effectiveness, and high-resolution cytogenomic analysis, OGM shows great promise as an alternative technology that uses a single assay to consolidate the multiple SOC tests usually used for prenatal cytogenetic diagnosis.\nMendelian randomization analysis identifies a causal effect of Streptococcus salivarius on DR mediating via the level of host fasting glucose\nAuthors: Li, J.; Zheng, G.; Jiang, D.; Deng, C.; Zhang, Y.; Ma, Y.; Su, J.\nScore: 1.8, Published: 2023-12-22 DOI: 10.1101/2023.12.19.23300249\nBackgroundDiabetic retinopathy (DR) is one of leading causes of vision loss in adults with increasing prevalence worldwide. Increasing evidence has emphasized the importance of gut microbiome in the etiology and development of DR. However, the causal relationship between gut microbes and DR remains largely unknown. MethodsTo investigate the causal associations of DR with gut microbes and DR risk factors, we employed two-sample Mendelian Randomization (MR) analyses to estimate the causal effects of 207 gut microbes on DR outcomes. Inputs for MR included Genome-wide Association Study (GWAS) summary statistics of 207 taxa of gut microbes (the Dutch Microbiome Project) and 21 risk factors for DR. The GWAS summary statistics data of DR was from the FinnGen Research Project. Data analysis was performed in May 2023. ResultsWe identified eight bacterial taxa that exhibited significant causal associations with DR (FDR \u003c 0.05). Among them, genus Collinsella and species Collinsella aerofaciens were associated with increased risk of DR, while the species Bacteroides faecis, Burkholderiales bacterium_1_1_47, Ruminococcus torques, Streptococcus salivarius, genus Burkholderiales_noname, and family Burkholderiales_noname showed protective effects against DR. Notably, we found that the causal effect of species Streptococcus salivarius on DR was mediated through the level of host fasting glucose, a well-established risk factor for DR. ConclusionsOur results reveal that specific gut microbes may be causally linked to DR via mediating host metabolic risk factors, highlighting potential novel therapeutic or preventive targets for DR.\n",
  "wordCount" : "2524",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300308">
        <p class="paperTitle">Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences,  ; Wood, C. R.; Kriegl, J. M.; Podduturi, N.; Jensen, J. N.; Stutzki, J.; Ding, Z.</p>
        <p class="info">Score: 26.9, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300308' target='https://doi.org/10.1101/2023.12.20.23300308'> 10.1101/2023.12.20.23300308</a></p>
        <p class="abstract">Advancements in long-read sequencing technology have accelerated the study of large structural variants (SVs). We created a curated, publicly available, multi-ancestry SV imputation panel by long-read sequencing 888 samples from the 1000 Genomes Project. This high-quality panel was used to impute SVs in approximately 500,000 UK Biobank participants. We demonstrated the feasibility of conducting genome-wide SV association studies at biobank scale using 32 disease-relevant phenotypes related to respiratory, cardiometabolic and liver diseases, in addition to 1,463 protein levels. This analysis identified thousands of genome-wide significant SV associations, including hundreds of conditionally independent signals, thereby enabling novel biological insights. Focusing on genetic association studies of lung function as an example, we demonstrate the added value of SVs for prioritising causal genes at gene-rich loci compared to traditional GWAS using only short variants. We envision that future post-GWAS gene-prioritisation workflows will incorporate SV analyses using this SV imputation panel and framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23299785">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23299785" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23299785">
        <p class="paperTitle">A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23299785" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23299785" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D.; Amnueypol, M.; Krittayaphong, R.; Prechawat, S.; Anannab, A.; Makarawate, P.; Ngarmukos, T.; Veerakul, G.; Kingsbury, Z.; Newington, T.; Maheswari, U.; Ross, M. T.; Grace, A.; Lambiase, P. D.; Behr, E. R.; Schott, J.-J.; Redon, R.; Barc, J.</p>
        <p class="info">Score: 12.9, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23299785' target='https://doi.org/10.1101/2023.12.19.23299785'> 10.1101/2023.12.19.23299785</a></p>
        <p class="abstract">Brugada syndrome (BrS) is a cardiac arrhythmia disorder that causes sudden death in young adults. Rare genetic variants in the SCN5A gene, encoding the Nav1.5 sodium channel, and common non-coding variants at this locus, are robustly associated with the condition. BrS is particularly prevalent in Southeast Asia but the underlying ancestry-specific factors remain largely unknown. Here, we performed genome sequencing of BrS probands from Thailand and population-matched controls and identified a rare non-coding variant in an SCN5A intronic enhancer that is highly enriched in BrS cases (3.9% in cases, odds ratio 20.2-45.2) and predicted to disrupt a Mef2 transcription factor binding site. Heterozygous introduction of the enhancer variant in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) caused significantly reduced SCN5A expression from the variant-containing allele and a 30% reduction in Nav1.5-mediated sodium-current density compared to isogenic controls. This is the first example of a validated rare non-coding variant at the SCN5A locus and partly explains the increased prevalence of BrS in this geographic region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.23300143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.23300143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.23300143">
        <p class="paperTitle">Empowering GWAS Discovery through Enhanced Genotype Imputation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.23300143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.23300143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.</p>
        <p class="info">Score: 9.6, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.23300143' target='https://doi.org/10.1101/2023.12.18.23300143'> 10.1101/2023.12.18.23300143</a></p>
        <p class="abstract">Genotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23299592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23299592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23299592">
        <p class="paperTitle">Multi-site validation of a functional assay to adjudicate SCN5A Brugada Syndrome-associated variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23299592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23299592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ma, J. G.; O&#39;Neill, M. J.; Richardson, E.; Thomson, K.; Ingles, J.; Muhammad, A.; Solus, J. F.; Davogustto, G. E.; Anderson, K. C.; Shoemaker, M. B.; Stergachis, A. B.; Floyd, B. J.; Dunn, K.; Parikh, V. N.; Chubb, H.; Perrin, M.; Roden, D. M.; Vandenberg, J. I.; Ng, C. A.; Glazer, A. M.</p>
        <p class="info">Score: 7.6, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23299592' target='https://doi.org/10.1101/2023.12.19.23299592'> 10.1101/2023.12.19.23299592</a></p>
        <p class="abstract">Brugada Syndrome (BrS) is an inheritable arrhythmia condition that is associated with rare, loss-of-function variants in the cardiac sodium channel gene, SCN5A. Interpreting the pathogenicity of SCN5A missense variants is challenging and [~]79% of SCN5A missense variants in ClinVar are currently classified as Variants of Uncertain Significance (VUS). An in vitro SCN5A-BrS automated patch clamp assay was generated for high-throughput functional studies of NaV1.5. The assay was independently studied at two separate research sites - Vanderbilt University Medical Center and Victor Chang Cardiac Research Institute - revealing strong correlations, including peak INa density (R2=0.86). The assay was calibrated according to ClinGen Sequence Variant Interpretation recommendations using high-confidence variant controls (n=49). Normal and abnormal ranges of function were established based on the distribution of benign variant assay results. The assay accurately distinguished benign controls (24/25) from pathogenic controls (23/24). Odds of Pathogenicity values derived from the experimental results yielded 0.042 for normal function (BS3 criterion) and 24.0 for abnormal function (PS3 criterion), resulting in up to strong evidence for both ACMG criteria. The calibrated assay was then used to study SCN5A VUS observed in four families with BrS and other arrhythmia phenotypes associated with SCN5A loss-of-function. The assay revealed loss-of-function for three of four variants, enabling reclassification to likely pathogenic. This validated APC assay provides clinical-grade functional evidence for the reclassification of current VUS and will aid future SCN5A-BrS variant classification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.23299893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.23299893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.23299893">
        <p class="paperTitle">Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.23299893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.23299893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.</p>
        <p class="info">Score: 25.4, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.23299893' target='https://doi.org/10.1101/2023.12.13.23299893'> 10.1101/2023.12.13.23299893</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals

SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.23299304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.23299304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.23299304">
        <p class="paperTitle">A maternal germline mutator phenotype in a family affected by heritable colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.23299304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.23299304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, C. L.; Beichman, A. C.; Mas Ponte, D.; Hemker, S. L.; Zhu, L.; Kitzman, J.; Shirts, B. H.; Harris, K.</p>
        <p class="info">Score: 11.3, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.23299304' target='https://doi.org/10.1101/2023.12.08.23299304'> 10.1101/2023.12.08.23299304</a></p>
        <p class="abstract">Variation in DNA repair genes can increase cancer risk by elevating the rate of oncogenic mutation. Defects in one such gene, MUTYH, are known to elevate the incidence of colorectal cancer in a recessive Mendelian manner, and some evidence has also linked MUTYH to elevated incidence of other cancers as well as elevated mutation rates in normal somatic and germline cells. Here, we use whole genome sequencing to measure germline de novo mutation rates in a large extended family affected by pathogenic MUTYH variation and a history of colorectal cancer. Although this familys genotype, p.Y179C/V234M (c.536A&gt;G/700G&gt;A on transcript NM_001128425), contains a variant with conflicting functional interpretations, we use an in vitro cell line assay to determine that it partially attenuates MUTYHs function. In the children of mothers affected by the Y179C/V234M genotype, we identify an elevation of the C&gt;A mutation rate that is weaker than mutator effects previously reported to be caused by other pathogenic MUTYH genotypes, suggesting that mutation rates in normal tissues may be useful for classifying cancer-associated variation along a continuum of severity. Surprisingly, we detect no significant elevation of the C&gt;A mutation rate in children born to a father with the same biallelic MUTYH genotype, despite calculating that we should have adequate power to detect such a mutator effect. This suggests that the oxidative stress repaired by MUTYH may contribute more to female reproductive aging than male reproductive aging in the general population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300294">
        <p class="paperTitle">Rare disease gene association discovery from burden analysis of the 100,000 Genomes Project data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O&#39;Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra</p>
        <p class="info">Score: 5.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300294' target='https://doi.org/10.1101/2023.12.20.23300294'> 10.1101/2023.12.20.23300294</a></p>
        <p class="abstract">To discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300206" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300206">
        <p class="paperTitle">Genome-wide characterization of 54 urinary metabolites reveals molecular impact of kidney function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Valo, E.; Richmond, A.; Mutter, S.; Campbell, A.; Porteous, D.; Wilson, J. F.; FinnDiane Study Group,  ; Groop, P.-H.; Hayward, C.; Sandholm, N.</p>
        <p class="info">Score: 2.0, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300206' target='https://doi.org/10.1101/2023.12.20.23300206'> 10.1101/2023.12.20.23300206</a></p>
        <p class="abstract">Dissecting the genetic mechanisms underlying urinary metabolite concentrations can provide molecular insights into kidney function and open possibilities for causal assessment of urinary metabolites with risk factors and disease outcomes. Proton nuclear magnetic resonance metabolomics provides a high-throughput means for urinary metabolite profiling, as widely applied for blood biomarker studies. Here we report a genome-wide association study meta-analysed for 3 European cohorts comprising 8,026 individuals, covering both people with type 1 diabetes and general population settings. We identified 52 associations (p&lt;9.3x10-10) for 19 of 54 studied metabolite concentrations. Out of these, 32 were not reported previously for relevant urinary or blood metabolite traits. Subsequent two-sample Mendelian randomization analysis suggests that estimated glomerular filtration rate (eGFR) causally affects 13 urinary metabolite concentrations whereas urinary ethanolamine, an initial precursor for phosphatidylcholine and phosphatidylethanolamine, was associated with higher eGFR lending support for a potential protective role.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.23300469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.23300469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.23300469">
        <p class="paperTitle">Progress Update for the Multisite Optical Genome Mapping Evaluation and Validation Study: Prenatal Applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.23300469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.23300469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levy, B.; Liu, J.; Iqbal, A.; Dupont, B.; Sahajpal, N.; Ho, M.; Yu, J.; Brody, S.; Mason-Suares, H.; Ganapathi, M.; Rajkovic, A.; Smolarek, T.; Toydemir, R.; Bui, P.; Kolhe, R.; Stevenson, R.</p>
        <p class="info">Score: 1.9, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.23300469' target='https://doi.org/10.1101/2023.12.22.23300469'> 10.1101/2023.12.22.23300469</a></p>
        <p class="abstract">Optical genome mapping (OGM) is an emerging technology with great potential for prenatal diagnosis. OGM can identify and resolve all types of balanced and unbalanced cytogenomic abnormalities in a single test, which are typically assessed by multiple standard of care (SOC) methods including karyotyping, fluorescence in situ hybridization and chromosomal microarray. To assess OGM&#39;s viability as an alternative to conventional SOC testing, a comprehensive clinical research study was conducted across multiple sites, operators, and instruments to evaluate its accuracy and clinical utility. This report provides an update for the phase 2 results of the ongoing multisite evaluation and validation study evaluating OGM for prenatal applications. In phase 1, 123 prenatal cases were assessed by OGM, and in phase 2, 219 retrospective and prospective prenatal cases have been evaluated. For 71% of cases, at least two SOC tests were performed. The study found that OGM had an overall accuracy of 99.6% and positive predictive value of 100% when compared to all cytogenetic SOC results. With its standardized workflow, cost-effectiveness, and high-resolution cytogenomic analysis, OGM shows great promise as an alternative technology that uses a single assay to consolidate the multiple SOC tests usually used for prenatal cytogenetic diagnosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23300249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23300249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23300249">
        <p class="paperTitle">Mendelian randomization analysis identifies a causal effect of Streptococcus salivarius on DR mediating via the level of host fasting glucose</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23300249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23300249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, J.; Zheng, G.; Jiang, D.; Deng, C.; Zhang, Y.; Ma, Y.; Su, J.</p>
        <p class="info">Score: 1.8, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23300249' target='https://doi.org/10.1101/2023.12.19.23300249'> 10.1101/2023.12.19.23300249</a></p>
        <p class="abstract">BackgroundDiabetic retinopathy (DR) is one of leading causes of vision loss in adults with increasing prevalence worldwide. Increasing evidence has emphasized the importance of gut microbiome in the etiology and development of DR. However, the causal relationship between gut microbes and DR remains largely unknown.

MethodsTo investigate the causal associations of DR with gut microbes and DR risk factors, we employed two-sample Mendelian Randomization (MR) analyses to estimate the causal effects of 207 gut microbes on DR outcomes. Inputs for MR included Genome-wide Association Study (GWAS) summary statistics of 207 taxa of gut microbes (the Dutch Microbiome Project) and 21 risk factors for DR. The GWAS summary statistics data of DR was from the FinnGen Research Project. Data analysis was performed in May 2023.

ResultsWe identified eight bacterial taxa that exhibited significant causal associations with DR (FDR &lt; 0.05). Among them, genus Collinsella and species Collinsella aerofaciens were associated with increased risk of DR, while the species Bacteroides faecis, Burkholderiales bacterium_1_1_47, Ruminococcus torques, Streptococcus salivarius, genus Burkholderiales_noname, and family Burkholderiales_noname showed protective effects against DR. Notably, we found that the causal effect of species Streptococcus salivarius on DR was mediated through the level of host fasting glucose, a well-established risk factor for DR.

ConclusionsOur results reveal that specific gut microbes may be causally linked to DR via mediating host metabolic risk factors, highlighting potential novel therapeutic or preventive targets for DR.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
