<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Valid inference for machine learning-assisted GWAS
Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.
Score: 20.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779
Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Valid inference for machine learning-assisted GWAS
Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.
Score: 20.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779
Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Valid inference for machine learning-assisted GWAS
Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.
Score: 20.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779
Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Valid inference for machine learning-assisted GWAS\nAuthors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.\nScore: 20.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779\nMachine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research.",
  "keywords": [
    
  ],
  "articleBody": " Valid inference for machine learning-assisted GWAS\nAuthors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.\nScore: 20.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779\nMachine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research. However, the validity of associations identified from ML-assisted GWAS has not been carefully evaluated. In this study, we report pervasive risks for false positive associations in ML-assisted GWAS, and introduce POP-GWAS, a novel statistical framework that reimagines GWAS on ML-imputed outcomes. POP-GWAS provides valid statistical inference irrespective of the quality of imputation or variables and algorithms used for imputation. It also only requires GWAS summary statistics as input. We employed POP-GWAS to perform the largest GWAS of bone mineral density (BMD) derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 novel loci reaching genome-wide significance and revealing skeletal site-specific genetic architecture of BMD. Our framework may fundamentally reshape the analytical strategies in future ML-assisted GWAS.\nA framework for accurate prediction of effector genes in genetic loci for complex traits\nAuthors: Schipper, M.; de Leeuw, C. A.; Maciel, B. A. P. C.; Wightman, D. P.; Hubers, N.; Boomsma, D. I.; O'Donovan, M. C.; Posthuma, D.\nScore: 12.8, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300360\nGenome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus associations remains challenging. Here we present the FLAMES framework, which predicts the most likely effector gene in a locus. FLAMES integrates machine learning with biological data linking single nucleotide polymorphisms (SNPs) to genes, and GWAS-wide convergence of gene interactions. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication, and domain knowledge of molecular traits. We demonstrate that combining SNP-based and convergence-based evidence outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find novel schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life.\nBenchmarking Mendelian Randomization methods for causal inference using genome-wide association study summary statistics\nAuthors: Hu, X.; Cai, M.; Xiao, J.; Wan, X.; Wang, Z.; Zhao, H.; Yang, C.\nScore: 10.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300765\nMendelian Randomization (MR), which utilizes genetic variants as instrumental variables (IVs), has gained popularity as a method for causal inference between phenotypes using genetic data. While efforts have been made to relax IV assumptions and develop new methods for causal inference in the presence of invalid IVs due to confounding, the reliability of MR methods in real-world applications remains uncertain. To bridge this gap, we conducted a benchmark study evaluating 15 MR methods using real-world genetic datasets. Our study focused on three crucial aspects: type I error control in the presence of various confounding scenarios (e.g., population stratification, pleiotropy, and assortative mating), the accuracy of causal effect estimates, replicability and power. By comprehensively evaluating the performance of compared methods over one thousand pairs of exposure-outcome traits, our study not only provides valuable insights into the performance and limitations of the compared methods but also offers practical guidance for researchers to choose appropriate MR methods for causal inference.\nThe Solute Carrier Family 26 Member 9 Is a Modifier of the Rapidly Progressing Cystic Fibrosis Associated with F508del CFTR Mutations\nAuthors: Luo, S.; Rollins, S.; Schmitz-Abe, K.; Tam, A.; Li, Q.; Shi, J.; Lin, J.; Wang, R.; Agrawal, P.\nScore: 6.8, Published: 2024-01-04 DOI: 10.1101/2024.01.04.23300546\nCystic fibrosis (CF) is an autosomal recessive disease caused by mutations to the CF transmembrane conductance regulator (CFTR). Symptoms and severity of the disease vary shown that modifier genes influence disease severity and clinical course. We previously reported epithelial sodium channel (ENaC) genes as modifiers of disease severity in long-term non-progressors sharing deltaF508 homozygous for CFTR genotype. Here we describe the opposite, modifier genes that may be associated with rapidly progressing CF (RPCF) in a cohort of patients with the shared deltaF508 homozygous genotype. We have identified three rare missense SLC26A9 variants in four individuals (out of six) deemed to have RPCF: c.229G\u003eA; p.G77S (present in two patients), c.1885C\u003eT; p.P629S and c.2546G\u003eA; p.R849Q. By analyzing publicly available single cell sequencing dataset from LungMAP, we revealed that both SLC26A9 and CFTR mRNA are highly enriched in Alveolar type 2 (AT2) cells, with a few (greater than 1%) in respiratory airway secretory (RAS) cells and ionocytes. Structural modeling suggests deleterious effects of these mutations as they are in critical protein domains which might affect the ion transportation capability of SLC26A9. The enrichment of rare and potentially deleterious SLC26A9 mutations in patients with RPCF suggests SLC26A9 is a modifier gene associated with RPCF.\nEffect of APOE e4 and its modification by sociodemographic characteristics on cognitive measures in South Asians from LASI-DAD\nAuthors: Wang, Y. Z.; Zhao, W.; Moorjani, P.; Gross, A. L.; Zhou, X.; Dey, A. B.; Lee, J.; Smith, J. A.; Kardia, S. L.\nScore: 5.1, Published: 2024-01-04 DOI: 10.1101/2024.01.04.24300840\nBACKGROUNDWe investigated effects of APOE {varepsilon}4 and its interactions with sociodemographic characteristics on cognitive measures in 2,563 South Asians from the Diagnostic Assessment of Dementia for the Longitudinal Aging Study of India (LASI-DAD). METHODSLinear regression models were used to assess the association between APOE {varepsilon}4 and global- and domain-specific cognitive function. Effect modification by age, sex, and education were explored using cross-product interaction terms and subgroup analyses. RESULTSAPOE {varepsilon}4 was inversely associated with most cognitive measures. This association was stronger in the older age group (age \u003e68) for general cognitive function, orientation, and language/fluency, as well as in females for memory and language/fluency. Interaction between APOE {varepsilon}4 and education was mostly nonsignificant. DISCUSSIONAPOE {varepsilon}4 is associated with lower cognitive function in South Asians from India, with a more pronounced impact observed in females and older individuals.\nGenomic analysis identifies risk factors in restless legs syndrome.\nAuthors: Akcimen, F.; Chia, R.; Saez-Atienzar, S.; Ruffo, P.; Rasheed, M.; Ross, J.; Liao, C.; Ray, A.; Dion, P. A.; Scholz, S. W.; Rouleau, G. A.; Traynor, B. J.\nScore: 5.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.23300211\nRestless legs syndrome (RLS) is a neurological condition that causes uncomfortable sensations in the legs and an irresistible urge to move them, typically during periods of rest. The genetic basis and pathophysiology of RLS are incompletely understood. Here, we present a whole-genome sequencing and genome-wide association meta-analysis of RLS cases (n = 9,851) and controls (n = 38,957) in three population-based biobanks (All of Us, Canadian Longitudinal Study on Aging, and CARTaGENE). Genome-wide association analysis identified nine independent risk loci, of which eight had been previously reported, and one was a novel risk locus (LMX1B, rs35196838, OR = 1.14, 95% CI = 1.09-1.19, p-value = 2.2 x 10-9). A genome-wide, gene-based common variant analysis identified GLO1 as an additional risk gene (p-value = 8.45 x 10-7). Furthermore, a transcriptome-wide association study also identified GLO1 and a previously unreported gene, ELFN1. A genetic correlation analysis revealed significant common variant overlaps between RLS and neuroticism (rg = 0.40, se = 0.08, p-value = 5.4 x 10-7), depression (rg = 0.35, se = 0.06, p-value = 2.17 x 10-8), and intelligence (rg = -0.20, se = 0.06, p-value = 4.0 x 10-4). Our study expands the understanding of the genetic architecture of RLS and highlights the contributions of common variants to this prevalent neurological disorder.\nGenome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health\nAuthors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.\nScore: 5.0, Published: 2023-12-29 DOI: 10.1101/2023.12.26.23300533\nNeuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field, brain age gap has emerged as promising MRI-based biomarker quantifying the deviation between an individuals biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimers disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.\nRare disease gene association discovery from burden analysis of the 100,000 Genomes Project data\nAuthors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O'Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra\nScore: 8.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300294\nTo discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.\nMulti-site validation of a functional assay to adjudicate SCN5A Brugada Syndrome-associated variants\nAuthors: Ma, J. G.; O'Neill, M. J.; Richardson, E.; Thomson, K.; Ingles, J.; Muhammad, A.; Solus, J. F.; Davogustto, G. E.; Anderson, K. C.; Shoemaker, M. B.; Stergachis, A. B.; Floyd, B. J.; Dunn, K.; Parikh, V. N.; Chubb, H.; Perrin, M.; Roden, D. M.; Vandenberg, J. I.; Ng, C. A.; Glazer, A. M.\nScore: 10.3, Published: 2023-12-20 DOI: 10.1101/2023.12.19.23299592\nBrugada Syndrome (BrS) is an inheritable arrhythmia condition that is associated with rare, loss-of-function variants in the cardiac sodium channel gene, SCN5A. Interpreting the pathogenicity of SCN5A missense variants is challenging and [~]79% of SCN5A missense variants in ClinVar are currently classified as Variants of Uncertain Significance (VUS). An in vitro SCN5A-BrS automated patch clamp assay was generated for high-throughput functional studies of NaV1.5. The assay was independently studied at two separate research sites - Vanderbilt University Medical Center and Victor Chang Cardiac Research Institute - revealing strong correlations, including peak INa density (R2=0.86). The assay was calibrated according to ClinGen Sequence Variant Interpretation recommendations using high-confidence variant controls (n=49). Normal and abnormal ranges of function were established based on the distribution of benign variant assay results. The assay accurately distinguished benign controls (24/25) from pathogenic controls (23/24). Odds of Pathogenicity values derived from the experimental results yielded 0.042 for normal function (BS3 criterion) and 24.0 for abnormal function (PS3 criterion), resulting in up to strong evidence for both ACMG criteria. The calibrated assay was then used to study SCN5A VUS observed in four families with BrS and other arrhythmia phenotypes associated with SCN5A loss-of-function. The assay revealed loss-of-function for three of four variants, enabling reclassification to likely pathogenic. This validated APC assay provides clinical-grade functional evidence for the reclassification of current VUS and will aid future SCN5A-BrS variant classification.\nEmpowering GWAS Discovery through Enhanced Genotype Imputation\nAuthors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.\nScore: 11.4, Published: 2023-12-21 DOI: 10.1101/2023.12.18.23300143\nGenotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.\n",
  "wordCount" : "2242",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300779">
        <p class="paperTitle">Valid inference for machine learning-assisted GWAS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.</p>
        <p class="info">Score: 20.8, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300779' target='https://doi.org/10.1101/2024.01.03.24300779'> 10.1101/2024.01.03.24300779</a></p>
        <p class="abstract">Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research. However, the validity of associations identified from ML-assisted GWAS has not been carefully evaluated. In this study, we report pervasive risks for false positive associations in ML-assisted GWAS, and introduce POP-GWAS, a novel statistical framework that reimagines GWAS on ML-imputed outcomes. POP-GWAS provides valid statistical inference irrespective of the quality of imputation or variables and algorithms used for imputation. It also only requires GWAS summary statistics as input. We employed POP-GWAS to perform the largest GWAS of bone mineral density (BMD) derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 novel loci reaching genome-wide significance and revealing skeletal site-specific genetic architecture of BMD. Our framework may fundamentally reshape the analytical strategies in future ML-assisted GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300360">
        <p class="paperTitle">A framework for accurate prediction of effector genes in genetic loci for complex traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schipper, M.; de Leeuw, C. A.; Maciel, B. A. P. C.; Wightman, D. P.; Hubers, N.; Boomsma, D. I.; O&#39;Donovan, M. C.; Posthuma, D.</p>
        <p class="info">Score: 12.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300360' target='https://doi.org/10.1101/2023.12.23.23300360'> 10.1101/2023.12.23.23300360</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus associations remains challenging. Here we present the FLAMES framework, which predicts the most likely effector gene in a locus. FLAMES integrates machine learning with biological data linking single nucleotide polymorphisms (SNPs) to genes, and GWAS-wide convergence of gene interactions. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication, and domain knowledge of molecular traits. We demonstrate that combining SNP-based and convergence-based evidence outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find novel schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300765" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300765">
        <p class="paperTitle">Benchmarking Mendelian Randomization methods for causal inference using genome-wide association study summary statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, X.; Cai, M.; Xiao, J.; Wan, X.; Wang, Z.; Zhao, H.; Yang, C.</p>
        <p class="info">Score: 10.8, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300765' target='https://doi.org/10.1101/2024.01.03.24300765'> 10.1101/2024.01.03.24300765</a></p>
        <p class="abstract">Mendelian Randomization (MR), which utilizes genetic variants as instrumental variables (IVs), has gained popularity as a method for causal inference between phenotypes using genetic data. While efforts have been made to relax IV assumptions and develop new methods for causal inference in the presence of invalid IVs due to confounding, the reliability of MR methods in real-world applications remains uncertain. To bridge this gap, we conducted a benchmark study evaluating 15 MR methods using real-world genetic datasets. Our study focused on three crucial aspects: type I error control in the presence of various confounding scenarios (e.g., population stratification, pleiotropy, and assortative mating), the accuracy of causal effect estimates, replicability and power. By comprehensively evaluating the performance of compared methods over one thousand pairs of exposure-outcome traits, our study not only provides valuable insights into the performance and limitations of the compared methods but also offers practical guidance for researchers to choose appropriate MR methods for causal inference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.23300546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.23300546" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.23300546">
        <p class="paperTitle">The Solute Carrier Family 26 Member 9 Is a Modifier of the Rapidly Progressing Cystic Fibrosis Associated with F508del CFTR Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.23300546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.23300546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, S.; Rollins, S.; Schmitz-Abe, K.; Tam, A.; Li, Q.; Shi, J.; Lin, J.; Wang, R.; Agrawal, P.</p>
        <p class="info">Score: 6.8, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.23300546' target='https://doi.org/10.1101/2024.01.04.23300546'> 10.1101/2024.01.04.23300546</a></p>
        <p class="abstract">Cystic fibrosis (CF) is an autosomal recessive disease caused by mutations to the CF transmembrane conductance regulator (CFTR). Symptoms and severity of the disease vary shown that modifier genes influence disease severity and clinical course. We previously reported epithelial sodium channel (ENaC) genes as modifiers of disease severity in long-term non-progressors sharing deltaF508 homozygous for CFTR genotype. Here we describe the opposite, modifier genes that may be associated with rapidly progressing CF (RPCF) in a cohort of patients with the shared deltaF508 homozygous genotype. We have identified three rare missense SLC26A9 variants in four individuals (out of six) deemed to have RPCF: c.229G&gt;A; p.G77S (present in two patients), c.1885C&gt;T; p.P629S and c.2546G&gt;A; p.R849Q. By analyzing publicly available single cell sequencing dataset from LungMAP, we revealed that both SLC26A9 and CFTR mRNA are highly enriched in Alveolar type 2 (AT2) cells, with a few (greater than 1%) in respiratory airway secretory (RAS) cells and ionocytes. Structural modeling suggests deleterious effects of these mutations as they are in critical protein domains which might affect the ion transportation capability of SLC26A9. The enrichment of rare and potentially deleterious SLC26A9 mutations in patients with RPCF suggests SLC26A9 is a modifier gene associated with RPCF.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.24300840">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.24300840" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.24300840">
        <p class="paperTitle">Effect of APOE e4 and its modification by sociodemographic characteristics on cognitive measures in South Asians from LASI-DAD</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.24300840" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.24300840" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y. Z.; Zhao, W.; Moorjani, P.; Gross, A. L.; Zhou, X.; Dey, A. B.; Lee, J.; Smith, J. A.; Kardia, S. L.</p>
        <p class="info">Score: 5.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.24300840' target='https://doi.org/10.1101/2024.01.04.24300840'> 10.1101/2024.01.04.24300840</a></p>
        <p class="abstract">BACKGROUNDWe investigated effects of APOE {varepsilon}4 and its interactions with sociodemographic characteristics on cognitive measures in 2,563 South Asians from the Diagnostic Assessment of Dementia for the Longitudinal Aging Study of India (LASI-DAD).

METHODSLinear regression models were used to assess the association between APOE {varepsilon}4 and global- and domain-specific cognitive function. Effect modification by age, sex, and education were explored using cross-product interaction terms and subgroup analyses.

RESULTSAPOE {varepsilon}4 was inversely associated with most cognitive measures. This association was stronger in the older age group (age &gt;68) for general cognitive function, orientation, and language/fluency, as well as in females for memory and language/fluency. Interaction between APOE {varepsilon}4 and education was mostly nonsignificant.

DISCUSSIONAPOE {varepsilon}4 is associated with lower cognitive function in South Asians from India, with a more pronounced impact observed in females and older individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23300211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23300211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23300211">
        <p class="paperTitle">Genomic analysis identifies risk factors in restless legs syndrome.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23300211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23300211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akcimen, F.; Chia, R.; Saez-Atienzar, S.; Ruffo, P.; Rasheed, M.; Ross, J.; Liao, C.; Ray, A.; Dion, P. A.; Scholz, S. W.; Rouleau, G. A.; Traynor, B. J.</p>
        <p class="info">Score: 5.0, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23300211' target='https://doi.org/10.1101/2023.12.19.23300211'> 10.1101/2023.12.19.23300211</a></p>
        <p class="abstract">Restless legs syndrome (RLS) is a neurological condition that causes uncomfortable sensations in the legs and an irresistible urge to move them, typically during periods of rest. The genetic basis and pathophysiology of RLS are incompletely understood. Here, we present a whole-genome sequencing and genome-wide association meta-analysis of RLS cases (n = 9,851) and controls (n = 38,957) in three population-based biobanks (All of Us, Canadian Longitudinal Study on Aging, and CARTaGENE). Genome-wide association analysis identified nine independent risk loci, of which eight had been previously reported, and one was a novel risk locus (LMX1B, rs35196838, OR = 1.14, 95% CI = 1.09-1.19, p-value = 2.2 x 10-9). A genome-wide, gene-based common variant analysis identified GLO1 as an additional risk gene (p-value = 8.45 x 10-7). Furthermore, a transcriptome-wide association study also identified GLO1 and a previously unreported gene, ELFN1. A genetic correlation analysis revealed significant common variant overlaps between RLS and neuroticism (rg = 0.40, se = 0.08, p-value = 5.4 x 10-7), depression (rg = 0.35, se = 0.06, p-value = 2.17 x 10-8), and intelligence (rg = -0.20, se = 0.06, p-value = 4.0 x 10-4). Our study expands the understanding of the genetic architecture of RLS and highlights the contributions of common variants to this prevalent neurological disorder.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.23300533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.23300533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.23300533">
        <p class="paperTitle">Genome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.23300533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.23300533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.</p>
        <p class="info">Score: 5.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.23300533' target='https://doi.org/10.1101/2023.12.26.23300533'> 10.1101/2023.12.26.23300533</a></p>
        <p class="abstract">Neuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field,  brain age gap has emerged as promising MRI-based biomarker quantifying the deviation between an individuals biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimers disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300294">
        <p class="paperTitle">Rare disease gene association discovery from burden analysis of the 100,000 Genomes Project data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O&#39;Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra</p>
        <p class="info">Score: 8.3, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300294' target='https://doi.org/10.1101/2023.12.20.23300294'> 10.1101/2023.12.20.23300294</a></p>
        <p class="abstract">To discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23299592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23299592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23299592">
        <p class="paperTitle">Multi-site validation of a functional assay to adjudicate SCN5A Brugada Syndrome-associated variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23299592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23299592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ma, J. G.; O&#39;Neill, M. J.; Richardson, E.; Thomson, K.; Ingles, J.; Muhammad, A.; Solus, J. F.; Davogustto, G. E.; Anderson, K. C.; Shoemaker, M. B.; Stergachis, A. B.; Floyd, B. J.; Dunn, K.; Parikh, V. N.; Chubb, H.; Perrin, M.; Roden, D. M.; Vandenberg, J. I.; Ng, C. A.; Glazer, A. M.</p>
        <p class="info">Score: 10.3, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23299592' target='https://doi.org/10.1101/2023.12.19.23299592'> 10.1101/2023.12.19.23299592</a></p>
        <p class="abstract">Brugada Syndrome (BrS) is an inheritable arrhythmia condition that is associated with rare, loss-of-function variants in the cardiac sodium channel gene, SCN5A. Interpreting the pathogenicity of SCN5A missense variants is challenging and [~]79% of SCN5A missense variants in ClinVar are currently classified as Variants of Uncertain Significance (VUS). An in vitro SCN5A-BrS automated patch clamp assay was generated for high-throughput functional studies of NaV1.5. The assay was independently studied at two separate research sites - Vanderbilt University Medical Center and Victor Chang Cardiac Research Institute - revealing strong correlations, including peak INa density (R2=0.86). The assay was calibrated according to ClinGen Sequence Variant Interpretation recommendations using high-confidence variant controls (n=49). Normal and abnormal ranges of function were established based on the distribution of benign variant assay results. The assay accurately distinguished benign controls (24/25) from pathogenic controls (23/24). Odds of Pathogenicity values derived from the experimental results yielded 0.042 for normal function (BS3 criterion) and 24.0 for abnormal function (PS3 criterion), resulting in up to strong evidence for both ACMG criteria. The calibrated assay was then used to study SCN5A VUS observed in four families with BrS and other arrhythmia phenotypes associated with SCN5A loss-of-function. The assay revealed loss-of-function for three of four variants, enabling reclassification to likely pathogenic. This validated APC assay provides clinical-grade functional evidence for the reclassification of current VUS and will aid future SCN5A-BrS variant classification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.23300143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.23300143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.23300143">
        <p class="paperTitle">Empowering GWAS Discovery through Enhanced Genotype Imputation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.23300143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.23300143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.</p>
        <p class="info">Score: 11.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.23300143' target='https://doi.org/10.1101/2023.12.18.23300143'> 10.1101/2023.12.18.23300143</a></p>
        <p class="abstract">Genotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
