<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders
Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders
Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-25T10:39:04+00:00" />
<meta property="article:modified_time" content="2023-10-25T10:39:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders
Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders\nAuthors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.",
  "keywords": [
    
  ],
  "articleBody": " Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders\nAuthors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.; Gardham, A.; Ghali, N.; Green, A.; Hanna, W.; Harrison, R.; Hegarty, M.; Higgs, J.; Holder, M.; Irving, R.; Jain, V.; Johnson, K.; Jolley, R.; Jones, W.; Jones, G.; Joss, S.; Kalinauskiene, R.; Kanini, F.; Kavanagh, K.; Khan, M.; Khan, N.; Kivuva, E.; Lahiri, N.; L\nScore: 21.0, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297202\nObjectiveWe sought to evaluate outcomes for clinical management following a genetic diagnosis from the Deciphering Developmental Disorders (DDD) Study. DesignIndividuals in the DDD study who had a pathogenic/likely pathogenic genotype in the DECIPHER database were selected for inclusion (n=5010). Clinical notes from regional clinical genetics services notes were reviewed to assess pre-defined clinical outcomes relating to interventions, prenatal choices, and information provision. ResultsOutcomes were recorded for 4237 diagnosed probands (85% of those eligible) from all 24 recruiting centres across the UK and Ireland. Additional diagnostic or screening tests were performed in 903 (21%) probands through referral to a range of different clinical specialties, and stopped or avoided in a further 26 (0.6%). Disease-specific treatment was started in 85 (2%) probands, including seizure-control medications and dietary supplements, and contra-indicated medications were stopped/avoided as no longer necessary in a further 20 (0.5%). The option of prenatal/preimplantation genetic testing was discussed with 1204 (28%) families, despite the relatively advanced age of the parents at the time of diagnosis. Importantly, condition-specific information or literature was given to 3214 (76%) families, and 880 (21%) were involved in family support groups. In the most common condition (KBG syndrome; 79 (2%) probands), clinical interventions only partially reflected the temporal development of phenotypes, highlighting the importance of consensus management guidelines and patient support groups. ConclusionsOur results underscore the importance of achieving a clinico-molecular diagnosis to ensure timely onward referral of patients, enabling appropriate care and anticipatory surveillance, and for accessing relevant patient support groups.\nDisentangling the link between maternal influences on birth weight and disease risk in 36,211 genotyped mother-child pairs\nAuthors: Leinonen, J. T.; FinnGen, ; Pirinen, M.; Tukiainen, T.\nScore: 12.6, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297154\nEpidemiological studies have robustly linked lower birth weight to later-life disease risks. These observations may reflect the adverse impact of intrauterine growth restriction on a childs health. However, causal evidence supporting such a mechanism in humans is largely lacking. Using Mendelian Randomization and 36,211 genotyped mother-child pairs from the FinnGen study, we assessed the relationship between intrauterine growth restriction and five common health outcomes (coronary heart disease (CHD), hypertension, statin use, type 2 diabetes and cancer). We proxied intrauterine growth restriction with polygenic scores for maternal effects on birth weight and took into account transmission of genetic variants between a mother and a child in the analyses. We find limited evidence for contribution of normal variation in maternally influenced intrauterine growth on later-life disease. Instead, we find support for genetic pleiotropy in the child genome linking birth weight to CHD and hypertension. Our study illustrates the opportunities that data from genotyped parent-child pairs from a population-based biobank provides for addressing causality of maternal influences.\nDeterminants of mosaic chromosomal alteration fitness\nAuthors: Pershad, Y.; Mack, T.; Poisner, H. M.; Jakubek, Y. A.; Stilp, A. M.; Mitchell, B. D.; Lewis, J. P.; Boerwinkle, E.; Loos, R. J.; Chami, N.; Wang, Z.; Barnes, K.; Pankratz, N.; Fornage, M.; Redline, S.; Psaty, B. M.; Bis, J. C.; Shojaie, A.; Silverman, E. K.; Cho, M. H.; Yun, J.; DeMeo, D.; Levy, D.; Johnson, A.; Mathias, R.; Taub, M.; Arnett, D.; North, K.; Raffield, L. M.; Carlson, A.; Doyle, M. F.; Rich, S. S.; Rotter, J. I.; Guo, X.; Cox, N.; Roden, D. M.; Franceschini, N.; Desai, P.; Reiner, A.; Auer, P. L.; Scheet, P.; Jaiswal, S.; Weinstock, J. S.; Bick, A. G.\nScore: 12.2, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297280\nClonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal Expansion Rate (PACER) method to estimate clonal expansion rate for 6,381 individuals in the NHLBI TOPMed cohort with gain, loss, and copy-neutral loss of heterozygosity mCAs. Our estimates of mCA fitness were correlated (R2 = 0.49) with an alternative approach that estimated fitness of mCAs in the UK Biobank using a theoretical probability distribution. Individuals with lymphoid-associated mCAs had a significantly higher white blood cell count and faster clonal expansion rate. In a cross-sectional analysis, genome-wide association study of estimates of mCA expansion rate identified TCL1A, NRIP1, and TERT locus variants as modulators of mCA clonal expansion rate.\nBidirectional relationship between olfaction and Parkinson's disease\nAuthors: Kim, J. J.; Bandres-Ciga, S.; Heilbron, K.; 23andMe Research Team, ; Blauwendraat, C.; Noyce, A. J.\nScore: 6.1, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297218\nBackgroundHyposmia (loss of smell) is a common early symptom of Parkinsons disease (PD). The shared genetic architecture between hyposmia and PD is unknown. MethodsWe leveraged genome-wide association study (GWAS) results for self-assessment of ability to smell and PD diagnosis. Linkage disequilibrium score regression (LDSC) and Local Analysis of [co]Variant Association (LAVA) were used to identify genome-wide and local genetic correlations. Mendelian randomization was used to identify potential causal relationships. ResultsLDSC found that sense of smell negatively correlated at a genome-wide level with PD. LAVA found negative correlations in four genetic loci near GBA1, ANAPC4, SNCA, and MAPT. Using Mendelian randomization we found evidence for strong causal relationship between PD and liability towards poorer sense of smell, but weaker evidence for the reverse direction. ConclusionsHyposmia and PD share genetic liability in only a subset of the major PD risk genes. While there was definitive evidence that PD can lower the sense of smell, there was only suggestive evidence for the reverse. This work highlights the heritability of olfactory function and its relationship with PD heritability and provides further insight into the association between PD and hyposmia.\niSCORED: nanopore-based random genomic sampling for intraoperative molecular diagnosis\nAuthors: Emiliani, F.; Ould Ismail, A. A.; Hughes, E.; Tsongalis, G.; Zanazzi, G.; Lin, C.-C.\nScore: 6.0, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297170\nCopy number variations (CNVs) are almost ubiquitous in cancer. In many cases, somatic CNV analysis has led to the identification of oncogenic pathways and suggested molecular-defined therapeutic targets. Here, we develop iSCORED, a one-step random genomic DNA reconstruction method that enables efficient, unbiased quantification of CNVs using a real-time Nanopore sequencer. By leveraging the long concatenated reads, we generate approximately 1-2 million genomic fragments within one hour of MinION sequencing, allowing for high-resolution genomic dosage comparisons. In our cohort of 26 malignant brain tumors, we demonstrated 100% concordance in CNV detections, including chromosomal alterations and oncogene amplifications when compared to clinically validated next generation sequencing and chromosomal microarray results. In addition, iSCORED allows concurrent brain tumor methylation classification without additional tissue preparation. The integrated methylation information revealed promoter hypomethylation in all detected amplified oncogenes. The entire workflow, including the automatic generation of CNV and methylation reports, can be accomplished within 120-140 minutes. Ultrafast molecular analysis can enhance clinical decision-making, optimize surgical planning and identify potential molecular therapies within surgical timeframes.\nUnderstanding and Predicting Polycystic Ovary Syndrome through Shared Genetics with Testosterone, SHBG, and Chronic Inflammation\nAuthors: Petersen, L. K.; Brixi, G.; Li, J.; Hu, J.; Wang, Z.; Han, X.; Meir, A. Y.; Tyrmi, J.; Mahalingaiah, S.; Piltonen, T.; Liang, L.\nScore: 5.0, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297115\nPolycystic ovary syndrome (PCOS) is a common hormonal disorder that affects one out of eight women and has high metabolic and psychological comorbidities. PCOS is thought to be associated with increased systemic low-grade inflammation, but the underlying mechanisms remain unclear. Here we study the genetic relationship between PCOS and obesity, testosterone, sex hormone binding globulin (SHBG), and chronic inflammation. First we create a large meta-analysis of PCOS (7,747 PCOS cases and 498,227 controls) and identify four novel genetic loci associated with PCOS. These novel loci have been previously associated with changes in gene expression in multiple tissues including the thyroid and ovary. We then analyze the PCOS meta analysis alongside GWASs for obesity (n=681,275), SHBG (n=190,366), testosterone (n=176,687), and meta-analyses of 130 inflammatory biomarkers (average n=30,000). We replicate potential causal relationships (via Mendelian randomization) from obesity and SHBG to PCOS and find extensive genetic correlations and causality between these traits and inflammatory biomarkers. We identify significant genetic correlations between PCOS and eight inflammatory biomarkers, including new relationships such as a strong correlation to death receptor 5 (LDSC rg = 0.54, FDR = 0.043). Although these results point to a shared genetic architecture between PCOS and inflammation, we did not find statistically significant causality between them; however, we did find 27 inflammatory biomarkers with significant causal effects on SHBG and four biomarkers with significant causal effects on testosterone. These results support the hypothesis that chronic inflammation can influence androgen and SHBG levels, though more research is needed to verify these results. Finally, we show that combining the polygenic risk scores of PCOS-related traits improves genetic prediction of PCOS cases in the UK Biobank and MGB Biobank compared to using only PCOS risk scores (0.72 AUC from 0.59 AUC and 0.61 AUC from 0.59 AUC in each biobank respectively).\nComprehensive reanalysis for CNVs in exome sequencing data from unsolved rare disease cases results in new diagnoses\nAuthors: Demidov, G.; Yaldiz, B.; Garcia-Pelaez, J.; de Boer, E.; Schuermans, N.; Van de Vondel, L.; Johansson, L. F.; Musacchia, F.; Paramonov, I.; Benetti, E.; Bullich, G.; Sablauskas, K.; Beltran, S.; Gilissen, C.; Hoischen, A.; Ossowski, S.; de Voer, R.; Lohmann, K.; Oliveira, C.; Topf, A.; Vissers, L. E. L. M.; Solve-RD Consortium, ; Laurie, S.\nScore: 4.6, Published: 2023-10-23 DOI: 10.1101/2023.10.22.23296993\nWe report the diagnostic results of a comprehensive copy number variant (CNV) reanalysis of 9,171 exome sequencing (ES) datasets from 5,757 families, including 6,143 individuals affected by a rare disease (RD). The data analysed was extremely heterogeneous, having been generated using 28 different exome enrichment kits, and sequenced on multiple short-read sequencing platforms, by 42 different research groups across Europe partnering in the Solve-RD project. Each of these research groups had previously undertaken their own analysis of the ES data but had failed to identify disease-causing variants. We applied three CNV calling algorithms to maximise sensitivity: ClinCNV, Conifer, and ExomeDepth. Rare CNVs overlapping genes of interest in custom lists provided by one of four partner European Reference Networks (ERN) were identified and taken forward for interpretation by clinical experts in RD. To facilitate interpretation, Integrative Genomics Viewer (IGV) screenshots incorporating a variety of custom-made tracks were generated for all prioritised CNVs. These analyses have resulted in a molecular diagnosis being provided for 51 families in this sample, with ClinCNV performing the best of the three algorithms in identifying disease-causing CNVs. We also identified pathogenic CNVs that are partially explanatory of the proband's phenotype in a further 34 individuals. This work illustrates the value of reanalysing ES cold cases for CNVs even where analyses had been undertaken previously. Crucially, identification of these previously undetected CNVs has resulted in the conclusion of the diagnostic odyssey for these RD families, some of which had endured decades.\nCould Metformin use reduce abdominal aortic aneurysm risk? A Mendelian randomisation study using known Metformin targets\nAuthors: Saxby, K. L.; Dudbridge, F.; Samani, N. J.; Bown, M. J.; Nelson, C. P.\nScore: 4.8, Published: 2023-10-16 DOI: 10.1101/2023.10.13.23293381\nIntroductionAn abdominal aortic aneurysm (AAA) is a swelling of the main artery in the body estimated to affect 0.92% of adults (aged 30-79) worldwide. Rupture is often fatal and surgical intervention may be offered if the risk of rupture is high. There is no treatment to prevent AAA or to slow aneurysm growth aside from dietary and lifestyle recommendations. Metformin, a drug prescribed to treat type 2 diabetes, has previously been associated with a potential reduction in AAA disease risk but no causal link has been shown. Here we investigate the causal link between Metformin and AAA risk through Mendelian randomisation (MR). MethodsWe conducted a two-sample MR analysis using genetic variants associated with gene expression of five Metformin drug targets that also show a genetic association with decreased glycated haemoglobin (HbA1c) levels. Effect sizes are obtained from within UK Biobank for HbA1c, and within AAAgen for AAA risk, a multi-ancestry meta-GWAS analysis of 39,221 cases and 1,086,107 controls. ResultsWe identified statistically significant evidence of a causal association between a genetic proxy for Metformin action and a decrease in AAA risk, OR=0.58 (95%CI: 0.37-0.90 p=0.015). We estimate that on average a one standard deviation decrease in HbA1c, measured via Metformin gene targets, reduces AAA risk by over 40%. ConclusionMetformin use in those at increased risk of AAA may reduce incidence of disease. Clinical trials are required to assess the efficacy of Metformin in reducing disease risk.\nPheNorm, a language model normalizer of physical examinations from genetics clinical notes\nAuthors: Weissenbacher, D.; Rawal, S.; Zhao, X.; Priestley, J. R. C.; Szigety, K. M.; Schmidt, S. F.; Higgins, M. J.; Magge, A.; O'Connor, K.; Gonzalez-Hernandez, G.; Campbell, I. M.\nScore: 2.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.23296894\nBackgroundPhenotypes identified during dysmorphology physical examinations are critical to genetic diagnosis and nearly universally documented as free-text in the electronic health record (EHR). Variation in how phenotypes are recorded in free-text makes large-scale computational analysis extremely challenging. Existing natural language processing (NLP) approaches to address phenotype extraction are trained largely on the biomedical literature or on case vignettes rather than actual EHR data. MethodsWe implemented a tailored system at the Childrens Hospital of Philadelpia that allows clinicians to document dysmorphology physical exam findings. From the underlying data, we manually annotated a corpus of 3136 organ system observations using the Human Phenotype Ontology (HPO). We provide this corpus publicly. We trained a transformer based NLP system to identify HPO terms from exam observations. The pipeline includes an extractor, which identifies tokens in the sentence expected to contain an HPO term, and a normalizer, which uses those tokens together with the original observation to determine the specific term mentioned. FindingsWe find that our labeler and normalizer NLP pipeline, which we call PheNorm, achieves state-of-the-art performance for the dysmorphology physical exam phenotype extraction task. PheNorms performance on the test set was 0.717, compared to the nearest baseline system (PhenoTagger) performance of 0.633. An analysis of our systems normalization errors shows possible imperfections in the HPO terminology itself but also reveals a lack of semantic understanding by our transformer models. InterpretationTransformers-based NLP models are a promising approach to genetic phenotype extraction and, with recent development of larger pre-trained causal language models, may improve semantic understanding in the future. We believe our results also have direct applicability to more general extraction of medical signs and symptoms. FundingUS National Institutes of Health\nSpinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs\nAuthors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.\nScore: 20.1, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296230\nBackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders. Spinocerebellar ataxia 4 (SCA4), one of the rarest SCAs, is characterized by adult-onset ataxia, polyneuropathy and linked to chromosome 16q22.1, the underlying mutation remains to be discovered. MethodsThree Swedish families affected by undiagnosed SCA went through detailed examinations, neurophysiological tests, neuroimaging studies and genetic testing. Imaging included MRI of the neuroaxis and brain PET. In 3 cases neuropathological assessments were performed. Genetic testing included SR WGS, STR analysis with Expansion Hunter de novo and long read (LR) WGS. ResultsNovel features such as dysautonomia, motor neuron affection, and abnormal eye movements, were found. Anticipation was documented as well. Atrophy in the cerebellum, brainstem and spinal cord was found in patients studied with MRI. [18F]FDG-PET demonstrated brain hypometabolism whereas [11C]Flumazenil-PET yielded reduced binding in several brain lobes, insula, thalamus, hypothalamus and cerebellum. Neuropathological assessment revealed moderate to severe loss of Purkinje cells in the cerebellum. In the spinal cord a marked loss of motor neurons in the anterior horns was seen as well as pronounced degeneration of posterior tracts. Intranuclear, mainly neuronal, inclusions positive for p62 and ubiquitin were sparse but widespread in the CNS. This finding prompted assessment for nucleotide expansions. A poly-glycine stretch encoding GGC expansions in the last exon of the zink finger homeobox 3 (ZFHX3) gene was identified segregating with disease, and not found in 1000 controls. ConclusionsSCA4 is a neurodegenerative disease, caused by a novel GGC expansion in ZFHX3, arguably the first polyglycine disorder in humans.\n",
  "wordCount" : "2838",
  "inLanguage": "en",
  "datePublished": "2023-10-25T10:39:04Z",
  "dateModified": "2023-10-25T10:39:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on October 25, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297202">
        <p class="paperTitle">Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.; Gardham, A.; Ghali, N.; Green, A.; Hanna, W.; Harrison, R.; Hegarty, M.; Higgs, J.; Holder, M.; Irving, R.; Jain, V.; Johnson, K.; Jolley, R.; Jones, W.; Jones, G.; Joss, S.; Kalinauskiene, R.; Kanini, F.; Kavanagh, K.; Khan, M.; Khan, N.; Kivuva, E.; Lahiri, N.; L</p>
        <p class="info">Score: 21.0, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297202' target='https://doi.org/10.1101/2023.10.18.23297202'> 10.1101/2023.10.18.23297202</a></p>
        <p class="abstract">ObjectiveWe sought to evaluate outcomes for clinical management following a genetic diagnosis from the Deciphering Developmental Disorders (DDD) Study.

DesignIndividuals in the DDD study who had a pathogenic/likely pathogenic genotype in the DECIPHER database were selected for inclusion (n=5010). Clinical notes from regional clinical genetics services notes were reviewed to assess pre-defined clinical outcomes relating to interventions, prenatal choices, and information provision.

ResultsOutcomes were recorded for 4237 diagnosed probands (85% of those eligible) from all 24 recruiting centres across the UK and Ireland. Additional diagnostic or screening tests were performed in 903 (21%) probands through referral to a range of different clinical specialties, and stopped or avoided in a further 26 (0.6%). Disease-specific treatment was started in 85 (2%) probands, including seizure-control medications and dietary supplements, and contra-indicated medications were stopped/avoided as no longer necessary in a further 20 (0.5%).

The option of prenatal/preimplantation genetic testing was discussed with 1204 (28%) families, despite the relatively advanced age of the parents at the time of diagnosis.

Importantly, condition-specific information or literature was given to 3214 (76%) families, and 880 (21%) were involved in family support groups. In the most common condition (KBG syndrome; 79 (2%) probands), clinical interventions only partially reflected the temporal development of phenotypes, highlighting the importance of consensus management guidelines and patient support groups.

ConclusionsOur results underscore the importance of achieving a clinico-molecular diagnosis to ensure timely onward referral of patients, enabling appropriate care and anticipatory surveillance, and for accessing relevant patient support groups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297154">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297154" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297154">
        <p class="paperTitle">Disentangling the link between maternal influences on birth weight and disease risk in 36,211 genotyped mother-child pairs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297154" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297154" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leinonen, J. T.; FinnGen,  ; Pirinen, M.; Tukiainen, T.</p>
        <p class="info">Score: 12.6, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297154' target='https://doi.org/10.1101/2023.10.17.23297154'> 10.1101/2023.10.17.23297154</a></p>
        <p class="abstract">Epidemiological studies have robustly linked lower birth weight to later-life disease risks. These observations may reflect the adverse impact of intrauterine growth restriction on a childs health. However, causal evidence supporting such a mechanism in humans is largely lacking.

Using Mendelian Randomization and 36,211 genotyped mother-child pairs from the FinnGen study, we assessed the relationship between intrauterine growth restriction and five common health outcomes (coronary heart disease (CHD), hypertension, statin use, type 2 diabetes and cancer). We proxied intrauterine growth restriction with polygenic scores for maternal effects on birth weight and took into account transmission of genetic variants between a mother and a child in the analyses.

We find limited evidence for contribution of normal variation in maternally influenced intrauterine growth on later-life disease. Instead, we find support for genetic pleiotropy in the child genome linking birth weight to CHD and hypertension. Our study illustrates the opportunities that data from genotyped parent-child pairs from a population-based biobank provides for addressing causality of maternal influences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297280">
        <p class="paperTitle">Determinants of mosaic chromosomal alteration fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pershad, Y.; Mack, T.; Poisner, H. M.; Jakubek, Y. A.; Stilp, A. M.; Mitchell, B. D.; Lewis, J. P.; Boerwinkle, E.; Loos, R. J.; Chami, N.; Wang, Z.; Barnes, K.; Pankratz, N.; Fornage, M.; Redline, S.; Psaty, B. M.; Bis, J. C.; Shojaie, A.; Silverman, E. K.; Cho, M. H.; Yun, J.; DeMeo, D.; Levy, D.; Johnson, A.; Mathias, R.; Taub, M.; Arnett, D.; North, K.; Raffield, L. M.; Carlson, A.; Doyle, M. F.; Rich, S. S.; Rotter, J. I.; Guo, X.; Cox, N.; Roden, D. M.; Franceschini, N.; Desai, P.; Reiner, A.; Auer, P. L.; Scheet, P.; Jaiswal, S.; Weinstock, J. S.; Bick, A. G.</p>
        <p class="info">Score: 12.2, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297280' target='https://doi.org/10.1101/2023.10.20.23297280'> 10.1101/2023.10.20.23297280</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal Expansion Rate (PACER) method to estimate clonal expansion rate for 6,381 individuals in the NHLBI TOPMed cohort with gain, loss, and copy-neutral loss of heterozygosity mCAs. Our estimates of mCA fitness were correlated (R2 = 0.49) with an alternative approach that estimated fitness of mCAs in the UK Biobank using a theoretical probability distribution. Individuals with lymphoid-associated mCAs had a significantly higher white blood cell count and faster clonal expansion rate. In a cross-sectional analysis, genome-wide association study of estimates of mCA expansion rate identified TCL1A, NRIP1, and TERT locus variants as modulators of mCA clonal expansion rate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297218">
        <p class="paperTitle">Bidirectional relationship between olfaction and Parkinson&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J. J.; Bandres-Ciga, S.; Heilbron, K.; 23andMe Research Team,  ; Blauwendraat, C.; Noyce, A. J.</p>
        <p class="info">Score: 6.1, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297218' target='https://doi.org/10.1101/2023.10.18.23297218'> 10.1101/2023.10.18.23297218</a></p>
        <p class="abstract">BackgroundHyposmia (loss of smell) is a common early symptom of Parkinsons disease (PD). The shared genetic architecture between hyposmia and PD is unknown.

MethodsWe leveraged genome-wide association study (GWAS) results for self-assessment of  ability to smell and PD diagnosis. Linkage disequilibrium score regression (LDSC) and Local Analysis of [co]Variant Association (LAVA) were used to identify genome-wide and local genetic correlations. Mendelian randomization was used to identify potential causal relationships.

ResultsLDSC found that sense of smell negatively correlated at a genome-wide level with PD. LAVA found negative correlations in four genetic loci near GBA1, ANAPC4, SNCA, and MAPT. Using Mendelian randomization we found evidence for strong causal relationship between PD and liability towards poorer sense of smell, but weaker evidence for the reverse direction.

ConclusionsHyposmia and PD share genetic liability in only a subset of the major PD risk genes. While there was definitive evidence that PD can lower the sense of smell, there was only suggestive evidence for the reverse. This work highlights the heritability of olfactory function and its relationship with PD heritability and provides further insight into the association between PD and hyposmia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297170">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297170" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297170">
        <p class="paperTitle">iSCORED: nanopore-based random genomic sampling for intraoperative molecular diagnosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297170" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297170" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emiliani, F.; Ould Ismail, A. A.; Hughes, E.; Tsongalis, G.; Zanazzi, G.; Lin, C.-C.</p>
        <p class="info">Score: 6.0, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297170' target='https://doi.org/10.1101/2023.10.17.23297170'> 10.1101/2023.10.17.23297170</a></p>
        <p class="abstract">Copy number variations (CNVs) are almost ubiquitous in cancer. In many cases, somatic CNV analysis has led to the identification of oncogenic pathways and suggested molecular-defined therapeutic targets. Here, we develop iSCORED, a one-step random genomic DNA reconstruction method that enables efficient, unbiased quantification of CNVs using a real-time Nanopore sequencer. By leveraging the long concatenated reads, we generate approximately 1-2 million genomic fragments within one hour of MinION sequencing, allowing for high-resolution genomic dosage comparisons. In our cohort of 26 malignant brain tumors, we demonstrated 100% concordance in CNV detections, including chromosomal alterations and oncogene amplifications when compared to clinically validated next generation sequencing and chromosomal microarray results. In addition, iSCORED allows concurrent brain tumor methylation classification without additional tissue preparation. The integrated methylation information revealed promoter hypomethylation in all detected amplified oncogenes. The entire workflow, including the automatic generation of CNV and methylation reports, can be accomplished within 120-140 minutes. Ultrafast molecular analysis can enhance clinical decision-making, optimize surgical planning and identify potential molecular therapies within surgical timeframes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297115" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297115">
        <p class="paperTitle">Understanding and Predicting Polycystic Ovary Syndrome through Shared Genetics with Testosterone, SHBG, and Chronic Inflammation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petersen, L. K.; Brixi, G.; Li, J.; Hu, J.; Wang, Z.; Han, X.; Meir, A. Y.; Tyrmi, J.; Mahalingaiah, S.; Piltonen, T.; Liang, L.</p>
        <p class="info">Score: 5.0, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297115' target='https://doi.org/10.1101/2023.10.17.23297115'> 10.1101/2023.10.17.23297115</a></p>
        <p class="abstract">Polycystic ovary syndrome (PCOS) is a common hormonal disorder that affects one out of eight women and has high metabolic and psychological comorbidities. PCOS is thought to be associated with increased systemic low-grade inflammation, but the underlying mechanisms remain unclear. Here we study the genetic relationship between PCOS and obesity, testosterone, sex hormone binding globulin (SHBG), and chronic inflammation. First we create a large meta-analysis of PCOS (7,747 PCOS cases and 498,227 controls) and identify four novel genetic loci associated with PCOS. These novel loci have been previously associated with changes in gene expression in multiple tissues including the thyroid and ovary. We then analyze the PCOS meta analysis alongside GWASs for obesity (n=681,275), SHBG (n=190,366), testosterone (n=176,687), and meta-analyses of 130 inflammatory biomarkers (average n=30,000). We replicate potential causal relationships (via Mendelian randomization) from obesity and SHBG to PCOS and find extensive genetic correlations and causality between these traits and inflammatory biomarkers. We identify significant genetic correlations between PCOS and eight inflammatory biomarkers, including new relationships such as a strong correlation to death receptor 5 (LDSC rg = 0.54, FDR = 0.043). Although these results point to a shared genetic architecture between PCOS and inflammation, we did not find statistically significant causality between them; however, we did find 27 inflammatory biomarkers with significant causal effects on SHBG and four biomarkers with significant causal effects on testosterone. These results support the hypothesis that chronic inflammation can influence androgen and SHBG levels, though more research is needed to verify these results. Finally, we show that combining the polygenic risk scores of PCOS-related traits improves genetic prediction of PCOS cases in the UK Biobank and MGB Biobank compared to using only PCOS risk scores (0.72 AUC from 0.59 AUC and 0.61 AUC from 0.59 AUC in each biobank respectively).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.23296993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.23296993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.23296993">
        <p class="paperTitle">Comprehensive reanalysis for CNVs in exome sequencing data from unsolved rare disease cases results in new diagnoses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.23296993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.23296993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demidov, G.; Yaldiz, B.; Garcia-Pelaez, J.; de Boer, E.; Schuermans, N.; Van de Vondel, L.; Johansson, L. F.; Musacchia, F.; Paramonov, I.; Benetti, E.; Bullich, G.; Sablauskas, K.; Beltran, S.; Gilissen, C.; Hoischen, A.; Ossowski, S.; de Voer, R.; Lohmann, K.; Oliveira, C.; Topf, A.; Vissers, L. E. L. M.; Solve-RD Consortium,  ; Laurie, S.</p>
        <p class="info">Score: 4.6, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.23296993' target='https://doi.org/10.1101/2023.10.22.23296993'> 10.1101/2023.10.22.23296993</a></p>
        <p class="abstract">We report the diagnostic results of a comprehensive copy number variant (CNV) reanalysis of 9,171 exome sequencing (ES) datasets from 5,757 families, including 6,143 individuals affected by a rare disease (RD). The data analysed was extremely heterogeneous, having been generated using 28 different exome enrichment kits, and sequenced on multiple short-read sequencing platforms, by 42 different research groups across Europe partnering in the Solve-RD project. Each of these research groups had previously undertaken their own analysis of the ES data but had failed to identify disease-causing variants. We applied three CNV calling algorithms to maximise sensitivity: ClinCNV, Conifer, and ExomeDepth. Rare CNVs overlapping genes of interest in custom lists provided by one of four partner European Reference Networks (ERN) were identified and taken forward for interpretation by clinical experts in RD. To facilitate interpretation, Integrative Genomics Viewer (IGV) screenshots incorporating a variety of custom-made tracks were generated for all prioritised CNVs. These analyses have resulted in a molecular diagnosis being provided for 51 families in this sample, with ClinCNV performing the best of the three algorithms in identifying disease-causing CNVs. We also identified pathogenic CNVs that are partially explanatory of the proband&#39;s phenotype in a further 34 individuals. This work illustrates the value of reanalysing ES cold cases for CNVs even where analyses had been undertaken previously. Crucially, identification of these previously undetected CNVs has resulted in the conclusion of the diagnostic odyssey for these RD families, some of which had endured decades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.23293381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.23293381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.23293381">
        <p class="paperTitle">Could Metformin use reduce abdominal aortic aneurysm risk? A Mendelian randomisation study using known Metformin targets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.23293381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.23293381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saxby, K. L.; Dudbridge, F.; Samani, N. J.; Bown, M. J.; Nelson, C. P.</p>
        <p class="info">Score: 4.8, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.23293381' target='https://doi.org/10.1101/2023.10.13.23293381'> 10.1101/2023.10.13.23293381</a></p>
        <p class="abstract">IntroductionAn abdominal aortic aneurysm (AAA) is a swelling of the main artery in the body estimated to affect 0.92% of adults (aged 30-79) worldwide. Rupture is often fatal and surgical intervention may be offered if the risk of rupture is high. There is no treatment to prevent AAA or to slow aneurysm growth aside from dietary and lifestyle recommendations. Metformin, a drug prescribed to treat type 2 diabetes, has previously been associated with a potential reduction in AAA disease risk but no causal link has been shown. Here we investigate the causal link between Metformin and AAA risk through Mendelian randomisation (MR).

MethodsWe conducted a two-sample MR analysis using genetic variants associated with gene expression of five Metformin drug targets that also show a genetic association with decreased glycated haemoglobin (HbA1c) levels. Effect sizes are obtained from within UK Biobank for HbA1c, and within AAAgen for AAA risk, a multi-ancestry meta-GWAS analysis of 39,221 cases and 1,086,107 controls.

ResultsWe identified statistically significant evidence of a causal association between a genetic proxy for Metformin action and a decrease in AAA risk, OR=0.58 (95%CI: 0.37-0.90 p=0.015). We estimate that on average a one standard deviation decrease in HbA1c, measured via Metformin gene targets, reduces AAA risk by over 40%.

ConclusionMetformin use in those at increased risk of AAA may reduce incidence of disease. Clinical trials are required to assess the efficacy of Metformin in reducing disease risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.23296894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.23296894" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.23296894">
        <p class="paperTitle">PheNorm, a language model normalizer of physical examinations from genetics clinical notes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.23296894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.23296894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weissenbacher, D.; Rawal, S.; Zhao, X.; Priestley, J. R. C.; Szigety, K. M.; Schmidt, S. F.; Higgins, M. J.; Magge, A.; O&#39;Connor, K.; Gonzalez-Hernandez, G.; Campbell, I. M.</p>
        <p class="info">Score: 2.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.23296894' target='https://doi.org/10.1101/2023.10.16.23296894'> 10.1101/2023.10.16.23296894</a></p>
        <p class="abstract">BackgroundPhenotypes identified during dysmorphology physical examinations are critical to genetic diagnosis and nearly universally documented as free-text in the electronic health record (EHR). Variation in how phenotypes are recorded in free-text makes large-scale computational analysis extremely challenging. Existing natural language processing (NLP) approaches to address phenotype extraction are trained largely on the biomedical literature or on case vignettes rather than actual EHR data.

MethodsWe implemented a tailored system at the Childrens Hospital of Philadelpia that allows clinicians to document dysmorphology physical exam findings. From the underlying data, we manually annotated a corpus of 3136 organ system observations using the Human Phenotype Ontology (HPO). We provide this corpus publicly. We trained a transformer based NLP system to identify HPO terms from exam observations. The pipeline includes an extractor, which identifies tokens in the sentence expected to contain an HPO term, and a normalizer, which uses those tokens together with the original observation to determine the specific term mentioned.

FindingsWe find that our labeler and normalizer NLP pipeline, which we call PheNorm, achieves state-of-the-art performance for the dysmorphology physical exam phenotype extraction task. PheNorms performance on the test set was 0.717, compared to the nearest baseline system (PhenoTagger) performance of 0.633. An analysis of our systems normalization errors shows possible imperfections in the HPO terminology itself but also reveals a lack of semantic understanding by our transformer models.

InterpretationTransformers-based NLP models are a promising approach to genetic phenotype extraction and, with recent development of larger pre-trained causal language models, may improve semantic understanding in the future. We believe our results also have direct applicability to more general extraction of medical signs and symptoms.

FundingUS National Institutes of Health</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.23296230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.23296230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.23296230">
        <p class="paperTitle">Spinocerebellar ataxia type 4 is caused by a GGC expansion in the ZFHX3 gene and is associated with prominent dysautonomia and motor neuron signs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.23296230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.23296230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paucar, M.; Nilsson, D.; Engvall, M.; Laffita-Mesa, J.; Soderhall, C.; Skorpil, M.; Halldin, C.; Fazio, P.; Lagerstedt-Robinson, K.; Solders, G.; Angeria, M.; Risling, M.; Jiao, H.; Nennesmo, I.; Wedell, A.; Svenningsson, P.</p>
        <p class="info">Score: 20.1, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.23296230' target='https://doi.org/10.1101/2023.10.03.23296230'> 10.1101/2023.10.03.23296230</a></p>
        <p class="abstract">BackgroundSpinocerebellar ataxias (SCAs) are a group of heterogeneous autosomal dominant disorders. Spinocerebellar ataxia 4 (SCA4), one of the rarest SCAs, is characterized by adult-onset ataxia, polyneuropathy and linked to chromosome 16q22.1, the underlying mutation remains to be discovered.

MethodsThree Swedish families affected by undiagnosed SCA went through detailed examinations, neurophysiological tests, neuroimaging studies and genetic testing. Imaging included MRI of the neuroaxis and brain PET. In 3 cases neuropathological assessments were performed. Genetic testing included SR WGS, STR analysis with Expansion Hunter de novo and long read (LR) WGS.

ResultsNovel features such as dysautonomia, motor neuron affection, and abnormal eye movements, were found. Anticipation was documented as well. Atrophy in the cerebellum, brainstem and spinal cord was found in patients studied with MRI. [18F]FDG-PET demonstrated brain hypometabolism whereas [11C]Flumazenil-PET yielded reduced binding in several brain lobes, insula, thalamus, hypothalamus and cerebellum. Neuropathological assessment revealed moderate to severe loss of Purkinje cells in the cerebellum. In the spinal cord a marked loss of motor neurons in the anterior horns was seen as well as pronounced degeneration of posterior tracts. Intranuclear, mainly neuronal, inclusions positive for p62 and ubiquitin were sparse but widespread in the CNS. This finding prompted assessment for nucleotide expansions. A poly-glycine stretch encoding GGC expansions in the last exon of the zink finger homeobox 3 (ZFHX3) gene was identified segregating with disease, and not found in 1000 controls.

ConclusionsSCA4 is a neurodegenerative disease, caused by a novel GGC expansion in ZFHX3, arguably the first polyglycine disorder in humans.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
