<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis
Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.
Score: 22.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893
Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis
Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.
Score: 22.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893
Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis
Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.
Score: 22.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893
Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis\nAuthors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.\nScore: 22.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893\nClonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions.",
  "keywords": [
    
  ],
  "articleBody": " Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis\nAuthors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.\nScore: 22.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893\nClonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.\nHuman genetic evidence enriched for side effects of approved drugs\nAuthors: Vallabh Minikel, E.; Nelson, M. R.\nScore: 17.4, Published: 2023-12-13 DOI: 10.1101/2023.12.12.23299869\nSafety failures are an important factor in low drug development success rates. Human genetic evidence can select drug targets causal in disease and enrich for successful programs. Here, we sought to determine whether human genetic evidence can also enrich for labeled side effects (SEs) of approved drugs. We combined the SIDER database of SEs with human genetic evidence from genome-wide association studies, Mendelian disease, and somatic mutations. SEs were 2.0 times more likely to occur for drugs whose target possessed human genetic evidence for a trait similar to the SE. Enrichment was highest when the trait and SE were most similar to each other, and was robust to removing drugs where the approved indication was also similar to the SE. The enrichment of genetic evidence was greatest for SEs that were more drug specific, affected more people, and were more severe. There was significant heterogeneity among disease areas the SEs mapped to, with the highest positive predictive value for cardiovascular SEs. This supports the integration of human genetic evidence early in the drug discovery process to identify potential SE risks to be monitored or mitigated in the course of drug development.\nA national pilot study: Clinical long-read sequencing of chromosomal rearrangements\nAuthors: Eisfeldt, J.; Ameur, A.; Lenner, F.; ten Berk de Boer, E.; Ek, M.; Wincent, J.; Vaz, R.; Ottosson, J.; Jonsson, T.; Ivarsson, S. E.; Thunstrom, S.; Topa, A.; Stenberg, S.; Rohlin, A.; Burstedt, M.; Nordin, F.; Sobol, M.; Baliakas, P.; Bondeson, M.-L.; Hojer, I.; Bilgrav Saether, K.; Lovmar, L.; Ehrencrona, H.; Melin, M.; Feuk, L.; Lindstrand, A.\nScore: 8.1, Published: 2023-12-17 DOI: 10.1101/2023.12.15.23299892\nBackground:Clinical genetic laboratories often require comprehensive analysis of chromosomal rearrangements/structural variants (SVs) which can range from gross chromosomal events, such as translocations and inversions, to supernumerary ring/marker chromosomes, and small deletions or duplications. To fully understand the complexity of a specific event and its associated clinical consequences, it is imperative to locate the breakpoint junctions (BPJs) and to resolve the derivative chromosome structure. This task, however, often surpasses the capabilities of conventional short-read sequencing technologies. In contrast, emerging long-read sequencing techniques present a compelling alternative for clinical diagnostics. Methods:Here, the Genomic Medicine Sweden Rare Diseases (GMS-RD) consortium validated HiFi Revio long-read whole genome sequencing (lrGS) for clinical digital karyotyping of SVs nationwide. The first 13 samples included in this study were collected from five of the seven Swedish regions with university hospitals. We established a national pipeline and a shared variant database for variant calling and filtering. The included validation samples cover a spectrum of simple and complex SVs including inversions, translocations and copy number variants. Results:The results from the lrGS analysis match the reported karyotype for 11/13 individuals and known SVs were mapped at nucleotide resolution. A complex rearrangement on chromosome 15 was identified only through read depth analysis and a mosaic ring chromosome 21 remained undetected. Depending on input DNA quality, the average read length ranged from 8.3-17.6 kb and the coverage was \u003e20x for all samples. De novo assembly resulted in a limited number of contigs per individual (N50 range 6-73 Mb) clearly separating the two alleles in most cases, enabling direct characterization of the chromosomal rearrangements. Conclusions:We have within a national pilot study successfully demonstrated the utility of HiFi Revio lrGS as a clinical analysis of chromosomal rearrangements.\nA maternal germline mutator phenotype in a family affected by heritable colorectal cancer\nAuthors: Young, C. L.; Beichman, A. C.; Mas Ponte, D.; Hemker, S. L.; Zhu, L.; Kitzman, J.; Shirts, B. H.; Harris, K.\nScore: 7.8, Published: 2023-12-18 DOI: 10.1101/2023.12.08.23299304\nVariation in DNA repair genes can increase cancer risk by elevating the rate of oncogenic mutation. Defects in one such gene, MUTYH, are known to elevate the incidence of colorectal cancer in a recessive Mendelian manner, and some evidence has also linked MUTYH to elevated incidence of other cancers as well as elevated mutation rates in normal somatic and germline cells. Here, we use whole genome sequencing to measure germline de novo mutation rates in a large extended family affected by pathogenic MUTYH variation and a history of colorectal cancer. Although this family's genotype, p. Y179C/V234M (transcript NM_001128425) contains a variant with conflicting functional interpretations, we use an in vitro cell line assay to determine that it partially attenuates MUTYH's function. In the children of mothers affected by the Y179C/V234M genotype, we identify an elevation of the C\u003eA mutation rate that is weaker than mutator effects previously reported to be caused by other pathogenic MUTYH genotypes, suggesting that mutation rates in normal tissues may be useful for classifying cancer-associated variation along a continuum of severity. Surprisingly, we detect no significant elevation of the C\u003eA mutation rate in children born to a father with the same biallelic MUTYH genotype, despite calculating that we should have adequate power to detect such a mutator effect. This suggests that the oxidative stress repaired by MUTYH may contribute more to female reproductive aging than male reproductive aging in the general population.\nEmpowering GWAS Discovery through Enhanced Genotype Imputation\nAuthors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.\nScore: 6.7, Published: 2023-12-19 DOI: 10.1101/2023.12.18.23300143\nGenotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.\nA probabilistic graphical model for estimating selection coefficient of nonsynonymous variants from human population sequence data\nAuthors: Zhao, Y.; Zhong, G.; Hagen, J.; Pan, H.; Chung, W. K.; Shen, Y.\nScore: 6.6, Published: 2023-12-13 DOI: 10.1101/2023.12.11.23299809\nAccurately predicting the effect of missense variants is important in discovering disease risk genes and clinical genetic diagnostics. Commonly used computational methods predict pathogenicity, which does not capture the quantitative impact on fitness in humans. We developed a method, MisFit, to estimate missense fitness effect using a graphical model. MisFit jointly models the effect at a molecular level (d) and a population level (selection coefficient, s), assuming that in the same gene, missense variants with similar d have similar s. We trained it by maximizing probability of observed allele counts in 236,017 European individuals. We show that s is informative in predicting allele frequency across ancestries and consistent with the fraction of de novo mutations in sites under strong selection. Further, s outperforms previous methods in prioritizing de novo missense variants in individuals with neurodevelopmental disorders. In conclusion, MisFit accurately predicts s and yields new insights from genomic data.\nThe counterpart congenital overgrowth syndromes Beckwith-Wiedemann Syndrome in human and large offspring syndrome in bovine involve alterations in DNA methylation, transcription, and chromatin configuration\nAuthors: Li, Y.; Boadu, F.; Xiao, P.; Goldkamp, A. K.; Nirgude, S.; Cheng, J.; Hagen, D. E.; Kalish, J.; Rivera, R. M.\nScore: 3.4, Published: 2023-12-16 DOI: 10.1101/2023.12.14.23299981\nBeckwith-Wiedemann Syndrome (BWS, OMIM #130650) is a congenital epigenetic disorder in humans which affects approximately 1 in 10,340 children. The incidence is likely an underestimation as the condition is usually recognized based on observable phenotypes at birth. BWS children have up to a 28% risk of developing tumors and currently, only 80% of patients can be corroborated molecularly (epimutations/variants). It is unknown how the subtypes of this condition are molecularly similar/dissimilar globally, therefore there is a need to deeply characterize the syndrome at the molecular level. Here we characterize the methylome, transcriptome and chromatin configuration of 18 BWS individuals together with the animal model of the condition, the bovine large offspring syndrome (LOS). Sex specific comparisons are performed for a subset of the BWS patients and LOS. Given that this epigenetic overgrowth syndrome has been characterized as a loss-of-imprinting condition, parental allele-specific comparisons were performed using the bovine animal model. In general, the differentially methylated regions (DMRs) detected in BWS and LOS showed significant enrichment for CTCF binding sites. Altered chromosome compartments in BWS and LOS were positively correlated with gene expression changes, and the promoters of differentially expressed genes showed significant enrichment for DMRs, differential topologically associating domains, and differential A/B compartments in some comparisons of BWS subtypes and LOS. We show shared regions of dysregulation between BWS and LOS, including several HOX gene clusters, and also demonstrate that altered DNA methylation differs between the clinically epigenetically identified BWS patients and those identified as having DNA variants (i.e. CDKN1C microdeletion). Lastly, we highlight additional genes and genomic regions that have the potential to serve as targets for biomarker development to improve current molecular methodologies. In summary, our results suggest that genome-wide alternation of chromosome architecture, which is partially caused by DNA methylation changes, also contribute to the development of BWS and LOS.\nDeep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.\nAuthors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.\nScore: 60.1, Published: 2023-11-28 DOI: 10.1101/2023.11.27.23299062\nIdentifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at pop.evemodel.org.\nEvaluating the cost-effectiveness of polygenic risk score-stratified screening for abdominal aortic aneurysm\nAuthors: Kelemen, M.; Roychowdhury, T.; Danesh, J.; Di Angelantonio, E.; Inouye, M.; O'Sullivan, J. W.; Pennells, L.; Wood, A.; Sweeting, M.; Harrison, S.; Kim, L.\nScore: 9.4, Published: 2023-12-07 DOI: 10.1101/2023.11.02.23297906\nAs the heritability of abdominal aortic aneurysm (AAA) is high and AAA partially shares genetic architecture with other cardiovascular diseases, there is interest in whether genetic information could inform AAA screening strategies. Exploiting pleiotropy and meta-analysing summary data from large AAA studies, we constructed a polygenic risk score (PRS) for AAA. Compared with the low PRS tertile, the intermediate and high PRS tertiles had hazard ratios for AAA of 2.13 (95%CI 1.61, 2.82) and 3.70 (95%CI 2.86, 4.80) respectively, after adjusting for known clinical risk factors. Using simulation modelling, we compared PRS- and smoking-stratified screening with inviting men at age 65 and not inviting women (current UK strategy). In a futuristic scenario where genomic information is available on the population, our modelling suggests inviting male current smokers with high PRS earlier than 65 and targeting screening in female smokers with high/intermediate PRS at 65 and 70 respectively, may improve cost-effectiveness.\nWhole-genome sequencing of half-a-million UK Biobank participants\nAuthors: The UK Biobank Whole-Genome Sequencing Consortium, ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.\nScore: 66.9, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299426\nSummary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3. This advance deepens our understanding of how genetics influences disease biology and further enhances the value of this open resource for the study of human biology and health. Coupling this dataset with rich phenotypic data, we surveyed within- and cross-ancestry genomic associations with health-related phenotypes and identified novel genetic and clinical insights. While most genome-wide significant associations with disease traits were primarily observed in Europeans, we also identified strong or novel signals in individuals of African and Asian ancestries. Deeper capture of exonic variation in both coding and UTR sequences, strengthened and surfaced novel insights relative to WES analyses. This landmark dataset, representing the largest collection of WGS and available to the UK Biobank research community, will enable advances into our understanding of the human genome, and facilitate the discovery of new diagnostics, therapeutics with higher efficacy and improved safety profile, and enable precision medicine strategies with the potential to improve global health. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=114 SRC=\"FIGDIR/small/23299426v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (22K): org.highwire.dtl.DTLVardef@14a2f80org.highwire.dtl.DTLVardef@1747f36org.highwire.dtl.DTLVardef@14bd7e6org.highwire.dtl.DTLVardef@4b7e54_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOGraphic summary.C_FLOATNO Framework of the WGS UKB study. This figure captures the flow of this manuscript. We start with the collection of patient samples by UK Biobank and followed by the strategy taken to perform WGS. We continue with quality control performed on GraphTyper and DRAGEN datasets, followed by variant calling of SNPs, in/dels, and structural variants (SV). Thereafter we defined the phenotypes (binary and quantitative) associated with SV, SNPs and at the gene level (rare variant analysis) and conclude with the definition of five ancestry groups and collective association effect as a cross-ancestry meta-analysis. C_FIG\n",
  "wordCount" : "2496",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.23299893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.23299893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.23299893">
        <p class="paperTitle">Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.23299893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.23299893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.</p>
        <p class="info">Score: 22.3, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.23299893' target='https://doi.org/10.1101/2023.12.13.23299893'> 10.1101/2023.12.13.23299893</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals

SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.23299869">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.23299869" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.23299869">
        <p class="paperTitle">Human genetic evidence enriched for side effects of approved drugs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.23299869" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.23299869" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vallabh Minikel, E.; Nelson, M. R.</p>
        <p class="info">Score: 17.4, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.23299869' target='https://doi.org/10.1101/2023.12.12.23299869'> 10.1101/2023.12.12.23299869</a></p>
        <p class="abstract">Safety failures are an important factor in low drug development success rates. Human genetic evidence can select drug targets causal in disease and enrich for successful programs. Here, we sought to determine whether human genetic evidence can also enrich for labeled side effects (SEs) of approved drugs. We combined the SIDER database of SEs with human genetic evidence from genome-wide association studies, Mendelian disease, and somatic mutations. SEs were 2.0 times more likely to occur for drugs whose target possessed human genetic evidence for a trait similar to the SE. Enrichment was highest when the trait and SE were most similar to each other, and was robust to removing drugs where the approved indication was also similar to the SE. The enrichment of genetic evidence was greatest for SEs that were more drug specific, affected more people, and were more severe. There was significant heterogeneity among disease areas the SEs mapped to, with the highest positive predictive value for cardiovascular SEs. This supports the integration of human genetic evidence early in the drug discovery process to identify potential SE risks to be monitored or mitigated in the course of drug development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.23299892">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.23299892" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.23299892">
        <p class="paperTitle">A national pilot study: Clinical long-read sequencing of chromosomal rearrangements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.23299892" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.23299892" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eisfeldt, J.; Ameur, A.; Lenner, F.; ten Berk de Boer, E.; Ek, M.; Wincent, J.; Vaz, R.; Ottosson, J.; Jonsson, T.; Ivarsson, S. E.; Thunstrom, S.; Topa, A.; Stenberg, S.; Rohlin, A.; Burstedt, M.; Nordin, F.; Sobol, M.; Baliakas, P.; Bondeson, M.-L.; Hojer, I.; Bilgrav Saether, K.; Lovmar, L.; Ehrencrona, H.; Melin, M.; Feuk, L.; Lindstrand, A.</p>
        <p class="info">Score: 8.1, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.23299892' target='https://doi.org/10.1101/2023.12.15.23299892'> 10.1101/2023.12.15.23299892</a></p>
        <p class="abstract">Background:Clinical genetic laboratories often require comprehensive analysis of chromosomal rearrangements/structural variants (SVs) which can range from gross chromosomal events, such as translocations and inversions, to supernumerary ring/marker chromosomes, and small deletions or duplications. To fully understand the complexity of a specific event and its associated clinical consequences, it is imperative to locate the breakpoint junctions (BPJs) and to resolve the derivative chromosome structure. This task, however, often surpasses the capabilities of conventional short-read sequencing technologies. In contrast, emerging long-read sequencing techniques present a compelling alternative for clinical diagnostics. Methods:Here, the Genomic Medicine Sweden Rare Diseases (GMS-RD) consortium validated HiFi Revio long-read whole genome sequencing (lrGS) for clinical digital karyotyping of SVs nationwide. The first 13 samples included in this study were collected from five of the seven Swedish regions with university hospitals. We established a national pipeline and a shared variant database for variant calling and filtering. The included validation samples cover a spectrum of simple and complex SVs including inversions, translocations and copy number variants. Results:The results from the lrGS analysis match the reported karyotype for 11/13 individuals and known SVs were mapped at nucleotide resolution. A complex rearrangement on chromosome 15 was identified only through read depth analysis and a mosaic ring chromosome 21 remained undetected. Depending on input DNA quality, the average read length ranged from 8.3-17.6 kb and the coverage was &gt;20x for all samples. De novo assembly resulted in a limited number of contigs per individual (N50 range 6-73 Mb) clearly separating the two alleles in most cases, enabling direct characterization of the chromosomal rearrangements. Conclusions:We have within a national pilot study successfully demonstrated the utility of HiFi Revio lrGS as a clinical analysis of chromosomal rearrangements.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.23299304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.23299304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.23299304">
        <p class="paperTitle">A maternal germline mutator phenotype in a family affected by heritable colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.23299304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.23299304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, C. L.; Beichman, A. C.; Mas Ponte, D.; Hemker, S. L.; Zhu, L.; Kitzman, J.; Shirts, B. H.; Harris, K.</p>
        <p class="info">Score: 7.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.23299304' target='https://doi.org/10.1101/2023.12.08.23299304'> 10.1101/2023.12.08.23299304</a></p>
        <p class="abstract">Variation in DNA repair genes can increase cancer risk by elevating the rate of oncogenic mutation. Defects in one such gene, MUTYH, are known to elevate the incidence of colorectal cancer in a recessive Mendelian manner, and some evidence has also linked MUTYH to elevated incidence of other cancers as well as elevated mutation rates in normal somatic and germline cells. Here, we use whole genome sequencing to measure germline de novo mutation rates in a large extended family affected by pathogenic MUTYH variation and a history of colorectal cancer. Although this family&#39;s genotype, p. Y179C/V234M (transcript NM_001128425) contains a variant with conflicting functional interpretations, we use an in vitro cell line assay to determine that it partially attenuates MUTYH&#39;s function. In the children of mothers affected by the Y179C/V234M genotype, we identify an elevation of the C&gt;A mutation rate that is weaker than mutator effects previously reported to be caused by other pathogenic MUTYH genotypes, suggesting that mutation rates in normal tissues may be useful for classifying cancer-associated variation along a continuum of severity. Surprisingly, we detect no significant elevation of the C&gt;A mutation rate in children born to a father with the same biallelic MUTYH genotype, despite calculating that we should have adequate power to detect such a mutator effect. This suggests that the oxidative stress repaired by MUTYH may contribute more to female reproductive aging than male reproductive aging in the general population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.23300143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.23300143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.23300143">
        <p class="paperTitle">Empowering GWAS Discovery through Enhanced Genotype Imputation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.23300143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.23300143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.</p>
        <p class="info">Score: 6.7, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.23300143' target='https://doi.org/10.1101/2023.12.18.23300143'> 10.1101/2023.12.18.23300143</a></p>
        <p class="abstract">Genotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.23299809">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.23299809" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.23299809">
        <p class="paperTitle">A probabilistic graphical model for estimating selection coefficient of nonsynonymous variants from human population sequence data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.23299809" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.23299809" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Y.; Zhong, G.; Hagen, J.; Pan, H.; Chung, W. K.; Shen, Y.</p>
        <p class="info">Score: 6.6, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.23299809' target='https://doi.org/10.1101/2023.12.11.23299809'> 10.1101/2023.12.11.23299809</a></p>
        <p class="abstract">Accurately predicting the effect of missense variants is important in discovering disease risk genes and clinical genetic diagnostics. Commonly used computational methods predict pathogenicity, which does not capture the quantitative impact on fitness in humans. We developed a method, MisFit, to estimate missense fitness effect using a graphical model. MisFit jointly models the effect at a molecular level (d) and a population level (selection coefficient, s), assuming that in the same gene, missense variants with similar d have similar s. We trained it by maximizing probability of observed allele counts in 236,017 European individuals. We show that s is informative in predicting allele frequency across ancestries and consistent with the fraction of de novo mutations in sites under strong selection. Further, s outperforms previous methods in prioritizing de novo missense variants in individuals with neurodevelopmental disorders. In conclusion, MisFit accurately predicts s and yields new insights from genomic data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.23299981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.23299981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.23299981">
        <p class="paperTitle">The counterpart congenital overgrowth syndromes Beckwith-Wiedemann Syndrome in human and large offspring syndrome in bovine involve alterations in DNA methylation, transcription, and chromatin configuration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.23299981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.23299981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Boadu, F.; Xiao, P.; Goldkamp, A. K.; Nirgude, S.; Cheng, J.; Hagen, D. E.; Kalish, J.; Rivera, R. M.</p>
        <p class="info">Score: 3.4, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.23299981' target='https://doi.org/10.1101/2023.12.14.23299981'> 10.1101/2023.12.14.23299981</a></p>
        <p class="abstract">Beckwith-Wiedemann Syndrome (BWS, OMIM #130650) is a congenital epigenetic disorder in humans which affects approximately 1 in 10,340 children. The incidence is likely an underestimation as the condition is usually recognized based on observable phenotypes at birth. BWS children have up to a 28% risk of developing tumors and currently, only 80% of patients can be corroborated molecularly (epimutations/variants). It is unknown how the subtypes of this condition are molecularly similar/dissimilar globally, therefore there is a need to deeply characterize the syndrome at the molecular level. Here we characterize the methylome, transcriptome and chromatin configuration of 18 BWS individuals together with the animal model of the condition, the bovine large offspring syndrome (LOS). Sex specific comparisons are performed for a subset of the BWS patients and LOS. Given that this epigenetic overgrowth syndrome has been characterized as a loss-of-imprinting condition, parental allele-specific comparisons were performed using the bovine animal model. In general, the differentially methylated regions (DMRs) detected in BWS and LOS showed significant enrichment for CTCF binding sites. Altered chromosome compartments in BWS and LOS were positively correlated with gene expression changes, and the promoters of differentially expressed genes showed significant enrichment for DMRs, differential topologically associating domains, and differential A/B compartments in some comparisons of BWS subtypes and LOS. We show shared regions of dysregulation between BWS and LOS, including several HOX gene clusters, and also demonstrate that altered DNA methylation differs between the clinically epigenetically identified BWS patients and those identified as having DNA variants (i.e. CDKN1C microdeletion). Lastly, we highlight additional genes and genomic regions that have the potential to serve as targets for biomarker development to improve current molecular methodologies. In summary, our results suggest that genome-wide alternation of chromosome architecture, which is partially caused by DNA methylation changes, also contribute to the development of BWS and LOS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23299062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23299062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23299062">
        <p class="paperTitle">Deep generative modeling of the human proteome reveals over a hundred novel genes involved in rare genetic disorders.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23299062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23299062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Orenbuch, R.; Kollasch, A. W.; Spinner, H. D.; Shearer, C. A.; Hopf, T. A.; Franceschi, D.; Dias, M.; Frazer, J.; Marks, D. S.</p>
        <p class="info">Score: 60.1, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23299062' target='https://doi.org/10.1101/2023.11.27.23299062'> 10.1101/2023.11.27.23299062</a></p>
        <p class="abstract">Identifying causal mutations accelerates genetic disease diagnosis, and therapeutic development. Missense variants present a bottleneck in genetic diagnoses as their effects are less straightforward than truncations or nonsense mutations. While computational prediction methods are increasingly successful at prediction for variants in known disease genes, they do not generalize well to other genes as the scores are not calibrated across the proteome. To address this, we developed a deep generative model, popEVE, that combines evolutionary information with population sequence data and achieves state-of-the-art performance at ranking variants by severity to distinguish patients with severe developmental disorders from potentially healthy individuals. popEVE identifies 442 genes in a cohort of developmental disorder cases, including evidence of 119 novel genetic disorders without the need for gene-level enrichment and without overestimating the prevalence of pathogenic variants in the population. By placing variants on a unified scale, our model offers a comprehensive perspective on the distribution of fitness effects across the entire proteome and the broader human population. popEVE provides compelling evidence for genetic diagnoses even in exceptionally rare single-patient disorders where conventional techniques relying on repeated observations may not be applicable. Interactive web viewer and downloads available at pop.evemodel.org.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297906">
        <p class="paperTitle">Evaluating the cost-effectiveness of polygenic risk score-stratified screening for abdominal aortic aneurysm</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelemen, M.; Roychowdhury, T.; Danesh, J.; Di Angelantonio, E.; Inouye, M.; O&#39;Sullivan, J. W.; Pennells, L.; Wood, A.; Sweeting, M.; Harrison, S.; Kim, L.</p>
        <p class="info">Score: 9.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297906' target='https://doi.org/10.1101/2023.11.02.23297906'> 10.1101/2023.11.02.23297906</a></p>
        <p class="abstract">As the heritability of abdominal aortic aneurysm (AAA) is high and AAA partially shares genetic architecture with other cardiovascular diseases, there is interest in whether genetic information could inform AAA screening strategies. Exploiting pleiotropy and meta-analysing summary data from large AAA studies, we constructed a polygenic risk score (PRS) for AAA. Compared with the low PRS tertile, the intermediate and high PRS tertiles had hazard ratios for AAA of 2.13 (95%CI 1.61, 2.82) and 3.70 (95%CI 2.86, 4.80) respectively, after adjusting for known clinical risk factors. Using simulation modelling, we compared PRS- and smoking-stratified screening with inviting men at age 65 and not inviting women (current UK strategy). In a futuristic scenario where genomic information is available on the population, our modelling suggests inviting male current smokers with high PRS earlier than 65 and targeting screening in female smokers with high/intermediate PRS at 65 and 70 respectively, may improve cost-effectiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.23299426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.23299426" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.23299426">
        <p class="paperTitle">Whole-genome sequencing of half-a-million UK Biobank participants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.23299426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.23299426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: The UK Biobank Whole-Genome Sequencing Consortium,  ; Li, S.; Carss, K. J.; Halldorsson, B. V.; Cortes, A.</p>
        <p class="info">Score: 66.9, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.23299426' target='https://doi.org/10.1101/2023.12.06.23299426'> 10.1101/2023.12.06.23299426</a></p>
        <p class="abstract">Summary ParagraphWhole-genome sequencing (WGS) provides a comprehensive view of the genome, enabling detection of coding and non-coding genetic variation, and surveying complex regions which are difficult to genotype. Here, we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping1 and whole-exome sequencing (WES) efforts2,3. This advance deepens our understanding of how genetics influences disease biology and further enhances the value of this open resource for the study of human biology and health. Coupling this dataset with rich phenotypic data, we surveyed within- and cross-ancestry genomic associations with health-related phenotypes and identified novel genetic and clinical insights. While most genome-wide significant associations with disease traits were primarily observed in Europeans, we also identified strong or novel signals in individuals of African and Asian ancestries. Deeper capture of exonic variation in both coding and UTR sequences, strengthened and surfaced novel insights relative to WES analyses. This landmark dataset, representing the largest collection of WGS and available to the UK Biobank research community, will enable advances into our understanding of the human genome, and facilitate the discovery of new diagnostics, therapeutics with higher efficacy and improved safety profile, and enable precision medicine strategies with the potential to improve global health.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=114 SRC=&#34;FIGDIR/small/23299426v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (22K):
org.highwire.dtl.DTLVardef@14a2f80org.highwire.dtl.DTLVardef@1747f36org.highwire.dtl.DTLVardef@14bd7e6org.highwire.dtl.DTLVardef@4b7e54_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOGraphic summary.C_FLOATNO Framework of the WGS UKB study. This figure captures the flow of this manuscript. We start with the collection of patient samples by UK Biobank and followed by the strategy taken to perform WGS. We continue with quality control performed on GraphTyper and DRAGEN datasets, followed by variant calling of SNPs, in/dels, and structural variants (SV). Thereafter we defined the phenotypes (binary and quantitative) associated with SV, SNPs and at the gene level (rare variant analysis) and conclude with the definition of five ancestry groups and collective association effect as a cross-ancestry meta-analysis.

C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
