<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics
Authors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics
Authors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics
Authors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics\nAuthors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A.",
  "keywords": [
    
  ],
  "articleBody": " Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics\nAuthors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A.; Chrisinger, J. S. A.; Dehner, C. A.; Widemann, B. C.; Hirbe, A. C.; Chaudhuri, A. A.; Shern, J. F.\nScore: 16.1, Published: 2024-01-20 DOI: 10.1101/2024.01.18.24301053\nEarly detection of neurofibromatosis type 1 (NF1) associated peripheral nerve sheath tumors (PNST) informs clinical decision-making, potentially averting deadly outcomes. Here, we describe a cell-free DNA (cfDNA) fragmentomic approach which distinguishes non-malignant, pre-malignant and malignant forms of NF1 PNST. Using plasma samples from a novel cohort of 101 NF1 patients and 21 healthy controls, we validated that our previous cfDNA copy number alteration (CNA)-based approach identifies malignant peripheral nerve sheath tumor (MPNST) but cannot distinguish among benign and premalignant states. We therefore investigated the ability of fragment-based cfDNA features to differentiate NF1-associated tumors including binned genome-wide fragment length ratios, end motif analysis, and non-negative matrix factorization deconvolution of fragment lengths. Fragmentomic methods were able to differentiate pre-malignant states including atypical neurofibromas (AN). Fragmentomics also adjudicated AN cases suspicious for MPNST, correctly diagnosing samples noninvasively, which could have informed clinical management. Overall, this study pioneers the early detection of malignant and premalignant peripheral nerve sheath tumors in NF1 patients using plasma cfDNA fragmentomics. In addition to screening applications, this novel approach distinguishes atypical neurofibromas from benign plexiform neurofibromas and malignant peripheral nerve sheath tumors, enabling more precise clinical diagnosis and management.\nPWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases\nAuthors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.\nScore: 11.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.23300645\nPWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score. Finally, standard statistical tests are activated in the case-control setting to yield statistically significant genes per phenotype. The PWAS Hub offers a user-friendly interface for an in-depth exploration of gene-disease associations from the UK Biobank (UKB). Results from PWAS cover 99 common diseases and conditions, each with over 10,000 diagnosed individuals per phenotype. Users can explore genes associated with these diseases, with separate analyses conducted for males and females. The PWAS Hub lists statistically significant genes associated with common diseases. It also indicates whether the analyzed damaged gene is associated with an increased or decreased risk. For each phenotype, the analyses account for sex-based genetic effects, inheritance modes (dominant and recessive), and the pleiotropic nature of associated genes. The PWAS Hub showcases its usefulness by navigating through such proteomic-genetic application for asthma. Graphical tools facilitate comparing genetic effects between the results of PWAS and coding GWAS, aiding in understanding the sex-specific genetic impact on common diseases. This adaptable platform is attractive for clinicians, researchers, and individuals interested in delving into gene-disease associations and sex-specific genetic effects. The PWAS Hub is accessible at http://pwas.huji.ac.il.\nAn Oxford Nanopore Technology-Based Hepatitis B Virus Sequencing Protocol Suitable For Genomic Surveillance Within Clinical Diagnostic Settings.\nAuthors: Tshiabuila, D.; Choga, W.; San, J. E.; Maponga, T.; Preiser, W.; van Zyl, G.; Moir, M.; van Wyk, S.; Giandhari, J.; Pillay, S.; Anyaneji, U. J.; Lessells, R.; Naidoo, Y.; Sanko, T. J.; Wilkinson, E.; Tegally, H.; Baxter, C.; Martin, D.; de Oliveira, T.\nScore: 10.9, Published: 2024-01-25 DOI: 10.1101/2024.01.19.24301519\nChronic hepatitis B virus (HBV) infection remains a significant public health concern, particularly in Africa, where there is a substantial. HBV is an enveloped virus, with isolates being classified into ten phylogenetically distinct genotypes (A - J) determined based on full-genome sequence data or reverse hybridization-based diagnostic tests. In practice, limitations are noted in that diagnostic sequencing, generally using Sanger sequencing, tends to focus only on the S-gene, yielding little or no information on intra-patient HBV genetic diversity with very low-frequency variants and reverse hybridization detects only known genotype-specific mutations. To resolve these limitations, we developed an Oxford Nanopore Technology (ONT)-based HBV genotyping protocol suitable for clinical virology, yielding complete HBV genome sequences and extensive data on intra-patient HBV diversity. Specifically, the protocol involves tiling-based PCR amplification of HBV sequences, library preparation using the ONT Rapid Barcoding Kit, ONT GridION sequencing, genotyping using Genome Detective software, recombination analysis using jpHMM and RDP5 software, and drug resistance profiling using Geno2pheno software. We prove the utility of our protocol by efficiently generating and characterizing high-quality near full-length HBV genomes from 148 left-over diagnostic Hepatitis B patient samples obtained in the Western Cape province of South Africa, providing valuable insights into the genetic diversity and epidemiology of HBV in this region of the world.\nExploration of the circulating human secretome through protein quantitative trait analysis identifies an association between circulating levels of apolipoprotein L1 and risk of giant cell arteritis\nAuthors: Chaddock, N.; UK GCA Consortium, ; Zulcinski, M.; Martin, J.; Mälarstig, A.; Peters, J. E.; Iles, M.; Morgan, A. W.\nScore: 5.3, Published: 2024-01-21 DOI: 10.1101/2024.01.19.24301534\nBackgroundGlucocorticoid monotherapy remains the principal treatment for giant cell arteritis (GCA), yet concurrent toxicity and adverse effects highlight the need for targeted therapies and improved risk stratification. Previous work suggests that evidence of genetic association can improve success rates in clinical trials and identify biomarkers for risk assessment, particularly when combined with other omics data, such as proteomics. However, relatively little is currently known about the genetic basis of GCA. MethodsPolygenic risk scores (PRS) were developed for 169 human plasma proteins and tested for association with GCA susceptibility (cases N=729, controls N=2,619). Associated PRS were replicated in an independent cohort (cases N=1,129, controls N=2,654) and their respective proteins were evaluated for causality using Mendelian randomization (MR). Finally, relationships between proteins with GCA-associated PRS were assessed using protein-protein interaction (PPI) network analysis ResultsThe Apolipoprotein L1 (APOL1) PRS had a statistically significant GCA association with a protective effect (P-value[P]=1 x 10-4), which replicated in an independent dataset (P=8.69 x 10- 4), and MR analysis supported a causal relationship (beta=-0.093; SE=0.02; P=4.42 x 10-9). PPI network analysis of proteins with GCA-associated PRS revealed enrichment for \"negative regulation of fibrinolysis\" and \"negative regulation of blood coagulation\" pathways. ConclusionsThis work emphasizes a potentially protective role of APOL1 and therefore reverse cholesterol transport in the pathogenesis of GCA. These findings also implicate fibrinolytic and coagulation cascades in GCA susceptibility, highlighting pathways that may be of interest for future pharmaceutical targeting. Non-standard Abbreviations and AcronymsGCA, giant cell arteritis; MHC, major histocompatibility complex; GWAS, genome-wide association study; SNPs, single nucleotide polymorphisms; PRS, polygenic risk score; pQTL, protein quantitative trait loci; MR, Mendelian randomization; QC, quality control; WTCCC, Wellcome Trust Case Control Consortium; PCA, principal component analysis; IV, instrumental variables; IVW, inverse-variance weighted; PheWAS, phenome-wide association study; GO, gene ontology; FUMA, Functional Mapping and Annotation of Genome-Wide Association Studies; MAGMA, Multi-marker Analysis of GenoMic Annotation; RCT, reverse cholesterol transport. Clinical PerspectiveO_ST_ABSWhat is new?C_ST_ABSO_LIAn apoliporotein-L1 polygenic risk score was associated with giant cell arteritis susceptibility, and replicated in an independent dataset. C_LIO_LIEvidence for causality of a protective effect of apolipoprotein-L1 in giant cell arteritis susceptibility was identified using Mendelian randomization. C_LIO_LIProteins with giant cell arteritis-associated polygenic risk scores were enriched in coagulation-related, fibrinolytic and immune response pathways. C_LI What are the clinical implications?O_LIFindings from this study indicate a protective role of apolipoprotein-L1 in giant cell arteritis susceptibility, highlighting a potential involvement of reverse cholesterol transport and lipid metabolism in disease pathogenesis. C_LIO_LIFibrinolytic and coagulation cascades were also implicated in the disease in addition to innate immune response pathways, redrawing attention to the role of thromboinflammation and the need to re-evaluate anti-platelet and anticoagulant therapies, particularly for those with impending visual loss and cranial ischaemic complications. C_LI\nVariant-specific pathophysiological mechanisms of AFF3 differently influence transcriptome profiles\nAuthors: Bassani, S.; Chrast, J.; Ambrosini, G.; Voisin, N.; Schütz, F.; Brusco, A.; Sirchia, F.; Turban, L.; Schubert, S.; Jamra, R. A.; Schlump, J.-U.; DeMille, D.; Bayrak-Toydemir, P.; Nelson, G. R.; Wong, K. N.; Duncan, L.; Mosera, M.; Gilissen, C.; Vissers, L. E. L. M.; Pfundt, R.; Kersseboom, R.; Yttervik, H.; Hansen, G. A. M.; Falkenberg Smeland, M.; Butler, K. M.; Lyons, M. J.; Carvalho, C. M. B.; Zhang, C.; Lupski, J. R.; Potocki, L.; Flores-Gallegos, L.; Morales-Toquero, R.; Petit, F.; Yalcin, B.; Tuttle, A.; Zghal Elloumi, H.; Mccormick, L.; Kukolich, M.; Klaas, O.; Horvath, J.; Scala, M.; Score: 6.0, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301100\nBackgroundWe previously described the KINSSHIP syndrome, an autosomal dominant disorder associated with intellectual disability (ID), mesomelic dysplasia and horseshoe kidney,caused by de novo variants in the degron of AFF3. Mouse knock-ins and overexpression in zebrafish provided evidence for a dominant-negative (DN) mode-of-action, wherein an increased level of AFF3 resulted in pathological effects. MethodsEvolutionary constraints suggest that other mode-of-inheritance could be at play. We challenged this hypothesis by screening ID cohorts for individuals with predicted-to-be deleterious variants in AFF3. We used both animal and cellular models to assess the deleteriousness of the identified variants. ResultsWe identified an individual with a KINSSHIP-like phenotype carrying a de novo partial duplication of AFF3 further strengthening the hypothesis that an increased level of AFF3 is pathological. We also detected seventeen individuals displaying a milder syndrome with either heterozygous LoF or biallelic missense variants in AFF3. Consistent with semi-dominance, we discovered three patients with homozygous LoF and one compound heterozygote for a LoF and a missense variant, who presented more severe phenotypes than their heterozygous parents. Matching zebrafish knockdowns exhibit neurological defects that could be rescued by expressing human AFF3 mRNA, confirming their association with the ablation of aff3. Conversely, some of the human AFF3 mRNAs carrying missense variants identified in affected individuals did not complement. Overexpression of mutated AFF3 mRNAs in zebrafish embryos produced a significant increase of abnormal larvae compared to wild-type overexpression further demonstrating deleteriousness. To further assess the effect of AFF3 variation, we profiled the transcriptome of fibroblasts from affected individuals and engineered isogenic cells harboring +/+, DN/DN, LoF/+, LoF/LoF or DN/LoF AFF3 genotypes. The expression of more than a third of the AFF3 bound loci is modified in either the DN/DN or the LoF/LoF lines. While the same pathways are affected, only about one-third of the differentially expressed genes are common to these homozygote datasets, indicating that AFF3 LoF and DN variants largely modulate transcriptomes differently, e.g. the DNA repair pathway displayed opposite modulation. ConclusionsOur results and the high pleiotropy shown by variation at this locus suggest that minute changes in AFF3 function are deleterious.\nEHMT2 LOSS-OF-FUNCTION ALTERATIONS CAUSE A KLEEFSTRA-LIKE SYNDROME\nAuthors: Martinez-Delgado, B.; Lopez-Martin, E.; Kerkhof, J.; Baladron, B.; Mielu, L. M.; Sanchez-Ponce, D.; Bada-Navarro, A.; Herrero-Matesanz, M.; Lopez-Jimenez, L.; Rzasa, J.; Rots, D.; Fernandez, M.; Hernandez-San Miguel, E.; Gomez-Mariano, G.; Marin-Reina, P.; Cazorla-Calleja, R.; Alonso, J.; Kleefstra, T.; Posada, M.; Bermejo-Sanchez, E.; Sadikovic, B.; Barrero, M. J.\nScore: 8.5, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24300997\nDysregulation of the epigenetic machinery is associated with neurodevelopmental defects in humans. Kleefstra syndrome (KS) is a neurodevelopmental syndrome caused by heterozygous alterations in the gene EHMT1 that cause loss-of-function. EHTM1 and EHMT2 are highly similar histone methyltransferases that play relevant roles in development. Despite their similarity, individuals with alterations in EHMT2 have never been described. Here, we describe a pediatric patient with a KS-overlapping phenotype and a single base de novo substitution in EHMT2 that causes the amino acid change p.Ala1077Ser in the catalytic SET domain. This change causes a reduction in the affinity of the catalytic domain for the H3 tail and in the activity of the enzyme by three- to five-fold. DNA methylation, histone methylation and gene expression profiles suggest a significant overlap between the EHMT2 p.Ala1077Ser variant and KS. Based on this evidence we suggest that EHMT2 haploinsufficiency causes a Kleefstra-like syndrome. Although we cannot rule out dominant negative effects caused by the EHMT2 p.Ala1077Ser variant, our data and previously published data suggest that loss of EHMT2 function is probably more detrimental to cells than loss of EHMT1, explaining why individuals with alterations in EHMT2 are very rare.\nCross-ancestry genetic investigation of schizophrenia, cannabis use disorder, and tobacco smoking\nAuthors: Johnson, E. C.; Austin-Zimmerman, I.; Thorpe, H. H.; Levey, D. F.; Baranger, D. A.; Colbert, S. M. C.; Demontis, D.; Khokhar, J. Y.; Davis, L. K.; Edenberg, H. J.; Di Forti, M.; Sanchez-Roige, S.; Gelernter, J.; Agrawal, A.\nScore: 3.4, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301430\nIndividuals with schizophrenia frequently experience co-occurring substance use, including tobacco smoking and heavy cannabis use, and substance use disorders. There is interest in understanding the extent to which these relationships are causal, and to what extent shared genetic factors play a role. We explored the relationships between schizophrenia (Scz), cannabis use disorder (CanUD), and ever-regular tobacco smoking (Smk) using the largest available genome-wide studies of these phenotypes in individuals of African and European ancestries. All three phenotypes were positively genetically correlated (rgs = 0.17 - 0.62).Causal inference analyses suggested the presence of horizontal pleiotropy, but evidence for bidirectional causal relationships was also found between all three phenotypes even after correcting for horizontal pleiotropy. We identified 439 pleiotropic loci in the European ancestry data, 150 of which were novel (i.e., not genome-wide significant in the original studies). Of these pleiotropic loci, 202 had lead variants which showed convergent effects (i.e., same direction of effect) on Scz, CanUD, and Smk. Genetic variants convergent across all three phenotypes showed strong genetic correlations with risk-taking, executive function, and several mental health conditions. Our results suggest that both horizontal pleiotropy and causal mechanisms may play a role in the relationship between CanUD, Smk, and Scz, but longitudinal, prospective studies are needed to confirm a causal relationship.\nA phenome-wide association study of tandem repeat variation in 168,554 individuals from the UK Biobank\nAuthors: Manigbas, C.; Jadhav, B.; Garg, P.; Shadrina, M.; Lee, W.; Martin-Trujillo, A.; Sharp, A.\nScore: 1.9, Published: 2024-01-23 DOI: 10.1101/2024.01.22.24301630\nMost genetic association studies focus on binary variants. To identify the effects of multi-allelic variation of tandem repeats (TRs) on human traits, we performed direct TR genotyping and phenome-wide association studies in 168,554 individuals from the UK Biobank, identifying 47 TRs showing causal associations with 73 traits. We replicated 23 of 31 (74%) of these causal associations in the All of Us cohort. While this set included several known repeat expansion disorders, novel associations we found were attributable to common polymorphic variation in TR length rather than rare expansions and include e.g. a coding polyhistidine motif in HRCT1 influencing risk of hypertension and a poly(CGC) in the 5UTR of GNB2 influencing heart rate. Causal TRs were strongly enriched for associations with local gene expression and DNA methylation. Our study highlights the contribution of multi-allelic TRs to the \"missing heritability\" of the human genome.\nIdentifying compounds to treat opiate use disorder by leveraging multi-omic data integration and multiple drug repurposing databases.\nAuthors: Stratford, J. K.; Carnes, M. U.; Willis, C.; Minto, M. S.; Elnimeiry, L.; Mathur, R.; Schu, M.; Quach, B. C.; Carter, J.; Nolen, T.; Vandergrift, N.; Kosten, T. R.; Johnson, E. O.; Webb, B. T.\nScore: 5.7, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301329\nGenes influencing opioid use disorder (OUD) biology have been identified via genome-wide association studies (GWAS), gene expression, and network analyses. These discoveries provide opportunities to identifying existing compounds targeting these genes for drug repurposing studies. However, systematically integrating discovery results and identifying relevant available pharmacotherapies for OUD repurposing studies is challenging. To address this, weve constructed a framework that leverages existing results and drug databases to identify candidate pharmacotherapies. For this study, two independent OUD related meta-analyses were used including a GWAS and a differential gene expression (DGE) study of post-mortem human brain. Protein-Protein Interaction (PPI) sub-networks enriched for GWAS risk loci were identified via network analyses. Drug databases Pharos, Open Targets, Therapeutic Target Database (TTD), and DrugBank were queried for clinical status and target selectivity. Cross-omic and drug query results were then integrated to identify candidate compounds. GWAS and DGE analyses revealed 3 and 335 target genes (FDR q\u003c0.05), respectively, while network analysis detected 70 genes in 22 enriched PPI networks. Four selection strategies were implemented, which yielded between 72 and 676 genes with statistically significant support and 110 to 683 drugs targeting these genes, respectively. After filtering out less specific compounds or those targeting well-established psychiatric-related receptors (OPRM1 and DRD2), between 2 and 329 approved drugs remained across the four strategies. By leveraging multiple lines of biological evidence and resources, we identified many FDA approved drugs that target genes associated with OUD. This approach a) allows high-throughput querying of OUD-related genes, b) detects OUD-related genes and compounds not identified using a single domain or resource, and c) produces a succinct summary of FDA approved compounds eligible for efficient expert review. Identifying larger pools of candidate pharmacotherapies and summarizing the supporting evidence bridges the gap between discovery and drug repurposing studies.\nBreaking down causes, consequences, and mediating effects of age-related telomere shortening on human health\nAuthors: Moix, S.; Sadler, M. C.; Kutalik, Z.; Auwerx, C.\nScore: 14.5, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301196\nTelomeres represent repeated DNA sequences at the ends of chromosomes, which shorten with each cell division. Factors modulating telomere attrition and the health consequences thereof are not fully understood. To address this, we leveraged data from 326,363 unrelated UK Biobank participants of European ancestry and used linear regression and bidirectional univariable and multivariable Mendelian randomization (MR and MVMR) to elucidate the relationships between leukocyte telomere length (LTL) and 141 complex traits, including diseases, biomarkers, and lifestyle factors. We confirm that telomeres shorten with age and show a stronger decline in males than in females, which cannot be explained by hormonal or lifestyle differences. MR revealed 23 traits modulating LTL; e.g., smoking cessation and high educational attainment associated with longer LTL, while weekly alcohol intake, body mass index, urate levels, and female reproductive events, such as childbirth, associated with shorter LTL. We also identified 26 traits affected by LTL, with risk for cardiovascular, pulmonary, renal, and some autoimmune diseases being increased by short LTL, while longer LTL increased risk for other autoimmune conditions and cancers. Through MVMR, we show that LTL partially mediates the impact of educational attainment, body mass index, and female age at childbirth on lifespan. These results provide new insights into the biology of telomere regulation by shedding light on the modulators, consequences, and the mediatory role of telomere shortening, portraying an intricate relationship between LTL, diseases, lifestyle, and socio-economic factors.\n",
  "wordCount" : "3096",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.24301053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.24301053" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.24301053">
        <p class="paperTitle">Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.24301053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.24301053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A.; Chrisinger, J. S. A.; Dehner, C. A.; Widemann, B. C.; Hirbe, A. C.; Chaudhuri, A. A.; Shern, J. F.</p>
        <p class="info">Score: 16.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.24301053' target='https://doi.org/10.1101/2024.01.18.24301053'> 10.1101/2024.01.18.24301053</a></p>
        <p class="abstract">Early detection of neurofibromatosis type 1 (NF1) associated peripheral nerve sheath tumors (PNST) informs clinical decision-making, potentially averting deadly outcomes. Here, we describe a cell-free DNA (cfDNA) fragmentomic approach which distinguishes non-malignant, pre-malignant and malignant forms of NF1 PNST. Using plasma samples from a novel cohort of 101 NF1 patients and 21 healthy controls, we validated that our previous cfDNA copy number alteration (CNA)-based approach identifies malignant peripheral nerve sheath tumor (MPNST) but cannot distinguish among benign and premalignant states. We therefore investigated the ability of fragment-based cfDNA features to differentiate NF1-associated tumors including binned genome-wide fragment length ratios, end motif analysis, and non-negative matrix factorization deconvolution of fragment lengths. Fragmentomic methods were able to differentiate pre-malignant states including atypical neurofibromas (AN). Fragmentomics also adjudicated AN cases suspicious for MPNST, correctly diagnosing samples noninvasively, which could have informed clinical management. Overall, this study pioneers the early detection of malignant and premalignant peripheral nerve sheath tumors in NF1 patients using plasma cfDNA fragmentomics. In addition to screening applications, this novel approach distinguishes atypical neurofibromas from benign plexiform neurofibromas and malignant peripheral nerve sheath tumors, enabling more precise clinical diagnosis and management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.23300645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.23300645" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.23300645">
        <p class="paperTitle">PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.23300645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.23300645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.</p>
        <p class="info">Score: 11.4, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.23300645' target='https://doi.org/10.1101/2024.01.20.23300645'> 10.1101/2024.01.20.23300645</a></p>
        <p class="abstract">PWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score. Finally, standard statistical tests are activated in the case-control setting to yield statistically significant genes per phenotype. The PWAS Hub offers a user-friendly interface for an in-depth exploration of gene-disease associations from the UK Biobank (UKB). Results from PWAS cover 99 common diseases and conditions, each with over 10,000 diagnosed individuals per phenotype. Users can explore genes associated with these diseases, with separate analyses conducted for males and females. The PWAS Hub lists statistically significant genes associated with common diseases. It also indicates whether the analyzed damaged gene is associated with an increased or decreased risk. For each phenotype, the analyses account for sex-based genetic effects, inheritance modes (dominant and recessive), and the pleiotropic nature of associated genes. The PWAS Hub showcases its usefulness by navigating through such proteomic-genetic application for asthma. Graphical tools facilitate comparing genetic effects between the results of PWAS and coding GWAS, aiding in understanding the sex-specific genetic impact on common diseases. This adaptable platform is attractive for clinicians, researchers, and individuals interested in delving into gene-disease associations and sex-specific genetic effects. The PWAS Hub is accessible at http://pwas.huji.ac.il.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.19.24301519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.19.24301519" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.19.24301519">
        <p class="paperTitle">An Oxford Nanopore Technology-Based Hepatitis B Virus Sequencing Protocol Suitable For Genomic Surveillance Within Clinical Diagnostic Settings.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.19.24301519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.19.24301519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tshiabuila, D.; Choga, W.; San, J. E.; Maponga, T.; Preiser, W.; van Zyl, G.; Moir, M.; van Wyk, S.; Giandhari, J.; Pillay, S.; Anyaneji, U. J.; Lessells, R.; Naidoo, Y.; Sanko, T. J.; Wilkinson, E.; Tegally, H.; Baxter, C.; Martin, D.; de Oliveira, T.</p>
        <p class="info">Score: 10.9, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.19.24301519' target='https://doi.org/10.1101/2024.01.19.24301519'> 10.1101/2024.01.19.24301519</a></p>
        <p class="abstract">Chronic hepatitis B virus (HBV) infection remains a significant public health concern, particularly in Africa, where there is a substantial. HBV is an enveloped virus, with isolates being classified into ten phylogenetically distinct genotypes (A - J) determined based on full-genome sequence data or reverse hybridization-based diagnostic tests. In practice, limitations are noted in that diagnostic sequencing, generally using Sanger sequencing, tends to focus only on the S-gene, yielding little or no information on intra-patient HBV genetic diversity with very low-frequency variants and reverse hybridization detects only known genotype-specific mutations. To resolve these limitations, we developed an Oxford Nanopore Technology (ONT)-based HBV genotyping protocol suitable for clinical virology, yielding complete HBV genome sequences and extensive data on intra-patient HBV diversity. Specifically, the protocol involves tiling-based PCR amplification of HBV sequences, library preparation using the ONT Rapid Barcoding Kit, ONT GridION sequencing, genotyping using Genome Detective software, recombination analysis using jpHMM and RDP5 software, and drug resistance profiling using Geno2pheno software. We prove the utility of our protocol by efficiently generating and characterizing high-quality near full-length HBV genomes from 148 left-over diagnostic Hepatitis B patient samples obtained in the Western Cape province of South Africa, providing valuable insights into the genetic diversity and epidemiology of HBV in this region of the world.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.19.24301534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.19.24301534" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.19.24301534">
        <p class="paperTitle">Exploration of the circulating human secretome through protein quantitative trait analysis identifies an association between circulating levels of apolipoprotein L1 and risk of giant cell arteritis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.19.24301534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.19.24301534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chaddock, N.; UK GCA Consortium,  ; Zulcinski, M.; Martin, J.; Mälarstig, A.; Peters, J. E.; Iles, M.; Morgan, A. W.</p>
        <p class="info">Score: 5.3, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.19.24301534' target='https://doi.org/10.1101/2024.01.19.24301534'> 10.1101/2024.01.19.24301534</a></p>
        <p class="abstract">BackgroundGlucocorticoid monotherapy remains the principal treatment for giant cell arteritis (GCA), yet concurrent toxicity and adverse effects highlight the need for targeted therapies and improved risk stratification. Previous work suggests that evidence of genetic association can improve success rates in clinical trials and identify biomarkers for risk assessment, particularly when combined with other  omics data, such as proteomics. However, relatively little is currently known about the genetic basis of GCA.

MethodsPolygenic risk scores (PRS) were developed for 169 human plasma proteins and tested for association with GCA susceptibility (cases N=729, controls N=2,619). Associated PRS were replicated in an independent cohort (cases N=1,129, controls N=2,654) and their respective proteins were evaluated for causality using Mendelian randomization (MR). Finally, relationships between proteins with GCA-associated PRS were assessed using protein-protein interaction (PPI) network analysis

ResultsThe Apolipoprotein L1 (APOL1) PRS had a statistically significant GCA association with a protective effect (P-value[P]=1 x 10-4), which replicated in an independent dataset (P=8.69 x 10- 4), and MR analysis supported a causal relationship (beta=-0.093; SE=0.02; P=4.42 x 10-9). PPI network analysis of proteins with GCA-associated PRS revealed enrichment for &#34;negative regulation of fibrinolysis&#34; and &#34;negative regulation of blood coagulation&#34; pathways.

ConclusionsThis work emphasizes a potentially protective role of APOL1 and therefore reverse cholesterol transport in the pathogenesis of GCA. These findings also implicate fibrinolytic and coagulation cascades in GCA susceptibility, highlighting pathways that may be of interest for future pharmaceutical targeting.

Non-standard Abbreviations and AcronymsGCA, giant cell arteritis; MHC, major histocompatibility complex; GWAS, genome-wide association study; SNPs, single nucleotide polymorphisms; PRS, polygenic risk score; pQTL, protein quantitative trait loci; MR, Mendelian randomization; QC, quality control; WTCCC, Wellcome Trust Case Control Consortium; PCA, principal component analysis; IV, instrumental variables; IVW, inverse-variance weighted; PheWAS, phenome-wide association study; GO, gene ontology; FUMA, Functional Mapping and Annotation of Genome-Wide Association Studies; MAGMA, Multi-marker Analysis of GenoMic Annotation; RCT, reverse cholesterol transport.

Clinical PerspectiveO_ST_ABSWhat is new?C_ST_ABSO_LIAn apoliporotein-L1 polygenic risk score was associated with giant cell arteritis susceptibility, and replicated in an independent dataset.
C_LIO_LIEvidence for causality of a protective effect of apolipoprotein-L1 in giant cell arteritis susceptibility was identified using Mendelian randomization.
C_LIO_LIProteins with giant cell arteritis-associated polygenic risk scores were enriched in coagulation-related, fibrinolytic and immune response pathways.
C_LI

What are the clinical implications?O_LIFindings from this study indicate a protective role of apolipoprotein-L1 in giant cell arteritis susceptibility, highlighting a potential involvement of reverse cholesterol transport and lipid metabolism in disease pathogenesis.
C_LIO_LIFibrinolytic and coagulation cascades were also implicated in the disease in addition to innate immune response pathways, redrawing attention to the role of thromboinflammation and the need to re-evaluate anti-platelet and anticoagulant therapies, particularly for those with impending visual loss and cranial ischaemic complications.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.24301100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.24301100" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.24301100">
        <p class="paperTitle">Variant-specific pathophysiological mechanisms of AFF3 differently influence transcriptome profiles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.24301100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.24301100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bassani, S.; Chrast, J.; Ambrosini, G.; Voisin, N.; Schütz, F.; Brusco, A.; Sirchia, F.; Turban, L.; Schubert, S.; Jamra, R. A.; Schlump, J.-U.; DeMille, D.; Bayrak-Toydemir, P.; Nelson, G. R.; Wong, K. N.; Duncan, L.; Mosera, M.; Gilissen, C.; Vissers, L. E. L. M.; Pfundt, R.; Kersseboom, R.; Yttervik, H.; Hansen, G. A. M.; Falkenberg Smeland, M.; Butler, K. M.; Lyons, M. J.; Carvalho, C. M. B.; Zhang, C.; Lupski, J. R.; Potocki, L.; Flores-Gallegos, L.; Morales-Toquero, R.; Petit, F.; Yalcin, B.; Tuttle, A.; Zghal Elloumi, H.; Mccormick, L.; Kukolich, M.; Klaas, O.; Horvath, J.; Scala, M.; </p>
        <p class="info">Score: 6.0, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.24301100' target='https://doi.org/10.1101/2024.01.14.24301100'> 10.1101/2024.01.14.24301100</a></p>
        <p class="abstract">BackgroundWe previously described the KINSSHIP syndrome, an autosomal dominant disorder associated with intellectual disability (ID), mesomelic dysplasia and horseshoe kidney,caused by de novo variants in the degron of AFF3. Mouse knock-ins and overexpression in zebrafish provided evidence for a dominant-negative (DN) mode-of-action, wherein an increased level of AFF3 resulted in pathological effects.

MethodsEvolutionary constraints suggest that other mode-of-inheritance could be at play. We challenged this hypothesis by screening ID cohorts for individuals with predicted-to-be deleterious variants in AFF3. We used both animal and cellular models to assess the deleteriousness of the identified variants.

ResultsWe identified an individual with a KINSSHIP-like phenotype carrying a de novo partial duplication of AFF3 further strengthening the hypothesis that an increased level of AFF3 is pathological. We also detected seventeen individuals displaying a milder syndrome with either heterozygous LoF or biallelic missense variants in AFF3. Consistent with semi-dominance, we discovered three patients with homozygous LoF and one compound heterozygote for a LoF and a missense variant, who presented more severe phenotypes than their heterozygous parents. Matching zebrafish knockdowns exhibit neurological defects that could be rescued by expressing human AFF3 mRNA, confirming their association with the ablation of aff3. Conversely, some of the human AFF3 mRNAs carrying missense variants identified in affected individuals did not complement. Overexpression of mutated AFF3 mRNAs in zebrafish embryos produced a significant increase of abnormal larvae compared to wild-type overexpression further demonstrating deleteriousness. To further assess the effect of AFF3 variation, we profiled the transcriptome of fibroblasts from affected individuals and engineered isogenic cells harboring &#43;/&#43;, DN/DN, LoF/&#43;, LoF/LoF or DN/LoF AFF3 genotypes. The expression of more than a third of the AFF3 bound loci is modified in either the DN/DN or the LoF/LoF lines. While the same pathways are affected, only about one-third of the differentially expressed genes are common to these homozygote datasets, indicating that AFF3 LoF and DN variants largely modulate transcriptomes differently, e.g. the DNA repair pathway displayed opposite modulation.

ConclusionsOur results and the high pleiotropy shown by variation at this locus suggest that minute changes in AFF3 function are deleterious.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24300997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24300997" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24300997">
        <p class="paperTitle">EHMT2 LOSS-OF-FUNCTION ALTERATIONS CAUSE A KLEEFSTRA-LIKE SYNDROME</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24300997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24300997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Delgado, B.; Lopez-Martin, E.; Kerkhof, J.; Baladron, B.; Mielu, L. M.; Sanchez-Ponce, D.; Bada-Navarro, A.; Herrero-Matesanz, M.; Lopez-Jimenez, L.; Rzasa, J.; Rots, D.; Fernandez, M.; Hernandez-San Miguel, E.; Gomez-Mariano, G.; Marin-Reina, P.; Cazorla-Calleja, R.; Alonso, J.; Kleefstra, T.; Posada, M.; Bermejo-Sanchez, E.; Sadikovic, B.; Barrero, M. J.</p>
        <p class="info">Score: 8.5, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24300997' target='https://doi.org/10.1101/2024.01.10.24300997'> 10.1101/2024.01.10.24300997</a></p>
        <p class="abstract">Dysregulation of the epigenetic machinery is associated with neurodevelopmental defects in humans. Kleefstra syndrome (KS) is a neurodevelopmental syndrome caused by heterozygous alterations in the gene EHMT1 that cause loss-of-function. EHTM1 and EHMT2 are highly similar histone methyltransferases that play relevant roles in development. Despite their similarity, individuals with alterations in EHMT2 have never been described. Here, we describe a pediatric patient with a KS-overlapping phenotype and a single base de novo substitution in EHMT2 that causes the amino acid change p.Ala1077Ser in the catalytic SET domain. This change causes a reduction in the affinity of the catalytic domain for the H3 tail and in the activity of the enzyme by three- to five-fold. DNA methylation, histone methylation and gene expression profiles suggest a significant overlap between the EHMT2 p.Ala1077Ser variant and KS. Based on this evidence we suggest that EHMT2 haploinsufficiency causes a Kleefstra-like syndrome. Although we cannot rule out dominant negative effects caused by the EHMT2 p.Ala1077Ser variant, our data and previously published data suggest that loss of EHMT2 function is probably more detrimental to cells than loss of EHMT1, explaining why individuals with alterations in EHMT2 are very rare.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301430" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301430">
        <p class="paperTitle">Cross-ancestry genetic investigation of schizophrenia, cannabis use disorder, and tobacco smoking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, E. C.; Austin-Zimmerman, I.; Thorpe, H. H.; Levey, D. F.; Baranger, D. A.; Colbert, S. M. C.; Demontis, D.; Khokhar, J. Y.; Davis, L. K.; Edenberg, H. J.; Di Forti, M.; Sanchez-Roige, S.; Gelernter, J.; Agrawal, A.</p>
        <p class="info">Score: 3.4, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301430' target='https://doi.org/10.1101/2024.01.17.24301430'> 10.1101/2024.01.17.24301430</a></p>
        <p class="abstract">Individuals with schizophrenia frequently experience co-occurring substance use, including tobacco smoking and heavy cannabis use, and substance use disorders. There is interest in understanding the extent to which these relationships are causal, and to what extent shared genetic factors play a role. We explored the relationships between schizophrenia (Scz), cannabis use disorder (CanUD), and ever-regular tobacco smoking (Smk) using the largest available genome-wide studies of these phenotypes in individuals of African and European ancestries. All three phenotypes were positively genetically correlated (rgs = 0.17 - 0.62).Causal inference analyses suggested the presence of horizontal pleiotropy, but evidence for bidirectional causal relationships was also found between all three phenotypes even after correcting for horizontal pleiotropy. We identified 439 pleiotropic loci in the European ancestry data, 150 of which were novel (i.e., not genome-wide significant in the original studies). Of these pleiotropic loci, 202 had lead variants which showed convergent effects (i.e., same direction of effect) on Scz, CanUD, and Smk. Genetic variants convergent across all three phenotypes showed strong genetic correlations with risk-taking, executive function, and several mental health conditions. Our results suggest that both horizontal pleiotropy and causal mechanisms may play a role in the relationship between CanUD, Smk, and Scz, but longitudinal, prospective studies are needed to confirm a causal relationship.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.24301630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.24301630" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.24301630">
        <p class="paperTitle">A phenome-wide association study of tandem repeat variation in 168,554 individuals from the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.24301630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.24301630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manigbas, C.; Jadhav, B.; Garg, P.; Shadrina, M.; Lee, W.; Martin-Trujillo, A.; Sharp, A.</p>
        <p class="info">Score: 1.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.24301630' target='https://doi.org/10.1101/2024.01.22.24301630'> 10.1101/2024.01.22.24301630</a></p>
        <p class="abstract">Most genetic association studies focus on binary variants. To identify the effects of multi-allelic variation of tandem repeats (TRs) on human traits, we performed direct TR genotyping and phenome-wide association studies in 168,554 individuals from the UK Biobank, identifying 47 TRs showing causal associations with 73 traits. We replicated 23 of 31 (74%) of these causal associations in the All of Us cohort. While this set included several known repeat expansion disorders, novel associations we found were attributable to common polymorphic variation in TR length rather than rare expansions and include e.g. a coding polyhistidine motif in HRCT1 influencing risk of hypertension and a poly(CGC) in the 5UTR of GNB2 influencing heart rate. Causal TRs were strongly enriched for associations with local gene expression and DNA methylation. Our study highlights the contribution of multi-allelic TRs to the &#34;missing heritability&#34; of the human genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301329">
        <p class="paperTitle">Identifying compounds to treat opiate use disorder by leveraging multi-omic data integration and multiple drug repurposing databases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stratford, J. K.; Carnes, M. U.; Willis, C.; Minto, M. S.; Elnimeiry, L.; Mathur, R.; Schu, M.; Quach, B. C.; Carter, J.; Nolen, T.; Vandergrift, N.; Kosten, T. R.; Johnson, E. O.; Webb, B. T.</p>
        <p class="info">Score: 5.7, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301329' target='https://doi.org/10.1101/2024.01.17.24301329'> 10.1101/2024.01.17.24301329</a></p>
        <p class="abstract">Genes influencing opioid use disorder (OUD) biology have been identified via genome-wide association studies (GWAS), gene expression, and network analyses. These discoveries provide opportunities to identifying existing compounds targeting these genes for drug repurposing studies. However, systematically integrating discovery results and identifying relevant available pharmacotherapies for OUD repurposing studies is challenging. To address this, weve constructed a framework that leverages existing results and drug databases to identify candidate pharmacotherapies.

For this study, two independent OUD related meta-analyses were used including a GWAS and a differential gene expression (DGE) study of post-mortem human brain. Protein-Protein Interaction (PPI) sub-networks enriched for GWAS risk loci were identified via network analyses. Drug databases Pharos, Open Targets, Therapeutic Target Database (TTD), and DrugBank were queried for clinical status and target selectivity. Cross-omic and drug query results were then integrated to identify candidate compounds.

GWAS and DGE analyses revealed 3 and 335 target genes (FDR q&lt;0.05), respectively, while network analysis detected 70 genes in 22 enriched PPI networks. Four selection strategies were implemented, which yielded between 72 and 676 genes with statistically significant support and 110 to 683 drugs targeting these genes, respectively. After filtering out less specific compounds or those targeting well-established psychiatric-related receptors (OPRM1 and DRD2), between 2 and 329 approved drugs remained across the four strategies.

By leveraging multiple lines of biological evidence and resources, we identified many FDA approved drugs that target genes associated with OUD. This approach a) allows high-throughput querying of OUD-related genes, b) detects OUD-related genes and compounds not identified using a single domain or resource, and c) produces a succinct summary of FDA approved compounds eligible for efficient expert review. Identifying larger pools of candidate pharmacotherapies and summarizing the supporting evidence bridges the gap between discovery and drug repurposing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301196" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301196">
        <p class="paperTitle">Breaking down causes, consequences, and mediating effects of age-related telomere shortening on human health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moix, S.; Sadler, M. C.; Kutalik, Z.; Auwerx, C.</p>
        <p class="info">Score: 14.5, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301196' target='https://doi.org/10.1101/2024.01.12.24301196'> 10.1101/2024.01.12.24301196</a></p>
        <p class="abstract">Telomeres represent repeated DNA sequences at the ends of chromosomes, which shorten with each cell division. Factors modulating telomere attrition and the health consequences thereof are not fully understood. To address this, we leveraged data from 326,363 unrelated UK Biobank participants of European ancestry and used linear regression and bidirectional univariable and multivariable Mendelian randomization (MR and MVMR) to elucidate the relationships between leukocyte telomere length (LTL) and 141 complex traits, including diseases, biomarkers, and lifestyle factors. We confirm that telomeres shorten with age and show a stronger decline in males than in females, which cannot be explained by hormonal or lifestyle differences. MR revealed 23 traits modulating LTL; e.g., smoking cessation and high educational attainment associated with longer LTL, while weekly alcohol intake, body mass index, urate levels, and female reproductive events, such as childbirth, associated with shorter LTL. We also identified 26 traits affected by LTL, with risk for cardiovascular, pulmonary, renal, and some autoimmune diseases being increased by short LTL, while longer LTL increased risk for other autoimmune conditions and cancers. Through MVMR, we show that LTL partially mediates the impact of educational attainment, body mass index, and female age at childbirth on lifespan. These results provide new insights into the biology of telomere regulation by shedding light on the modulators, consequences, and the mediatory role of telomere shortening, portraying an intricate relationship between LTL, diseases, lifestyle, and socio-economic factors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
