<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 328.3, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nRNA sequencing uplifts diagnostic rate in undiagnosed rare disease patients\nAuthors: Jaramillo Oquendo, C.; Wai, H. A.; Rich, W.; Bunyan, D. J.; Thomas, N. S.; Hunt, D.; Lord, J.; Douglas, A. G. L.; Baralle, D.\nScore: 21.2, Published: 2023-07-08 DOI: 10.1101/2023.07.05.23292254\nRNA-sequencing is increasingly being used as a complementary tool to DNA sequencing in diagnostics where DNA analysis has been uninformative as it enables the identification of alternative splicing (AS), aberrant gene expression and allele specific expression. Using RNA from patient blood, we have used both RNA-sequencing and RT-PCR to detect splicing and gene expression outliers in a heterogenous cohort of 87 patients with suspected Mendelian disorders, 38% of which did not have a candidate sequence variant. Expression outliers were detected using OUTRIDER, and we compared the performance of multiple open-source alternative splicing tools (MAJIQ, rMATS-turbo, and LeafCutterMD) in identifying alternative splicing events. As well as clarifying the impact of variants of uncertain significance (VUSs), we trialled two novel approaches to identify new potential diagnoses in patients with no candidate variants. We were able to assess 85% of VUSs and validate splicing abnormalities in 18/48 patients with a VUS. Furthermore, we identified four new diagnoses by detecting novel AS events in patients with no candidate sequence variants from prior genomic DNA testing (n=33) or those in which the candidate VUS did not affect splicing (n=23) and identified one additional diagnosis through detection of skewed X-inactivation. These results gave an overall uplift in diagnostic yield of 26%. In this work we demonstrate the utility of blood-based RNA analysis in improving diagnostic yields and highlight optimal approaches for such analysis.\nMulti-ancestry genome-wide meta-analysis of 56,241 individuals identifies LRRC4C, LHX5-AS1 and nominates ancestry-specific loci PTPRK, GRB14, and KIAA0825 as novel risk loci for Alzheimer disease: the Alzheimer Disease Genetics Consortium\nAuthors: Rajabli, F.; Benchek, P.; Tosto, G.; Kushch, N.; Sha, J.; Bazemore, K.; Zhu, C.; Lee, W.-P.; Haut, J.; Hamilton-Nelson, K. L.; Wheeler, N. R.; Zhao, Y.; Farrell, J. J.; Grunin, M. A.; Leung, Y. Y.; Kuksa, P. P.; Li, D.; Lucio da Fonseca, E.; Mez, J. B.; Palmer, E. L.; Pillai, J.; Sherva, R. M.; Song, Y. E.; Zhang, X.; Iqbal, T.; Pathak, O.; Valladares, O.; Kuzma, A. B.; Abner, E.; Adams, P. M.; Aguirre, A.; Albert, M. S.; Albin, R. L.; Allen, M.; Alvarez, L.; Apostolova, L. G.; Arnold, S. E.; Asthana, S.; Atwood, C. S.; Ayres, G.; Baldwin, C. T.; Barber, R. C.; Barnes, L. L.; Barral, S.; Beach\nScore: 20.1, Published: 2023-07-08 DOI: 10.1101/2023.07.06.23292311\nLimited ancestral diversity has impaired our ability to detect risk variants more prevalent in non-European ancestry groups in genome-wide association studies (GWAS). We constructed and analyzed a multi-ancestry GWAS dataset in the Alzheimers Disease (AD) Genetics Consortium (ADGC) to test for novel shared and ancestry-specific AD susceptibility loci and evaluate underlying genetic architecture in 37,382 non-Hispanic White (NHW), 6,728 African American, 8,899 Hispanic (HIS), and 3,232 East Asian individuals, performing within-ancestry fixed-effects meta-analysis followed by a cross-ancestry random-effects meta-analysis. We identified 13 loci with cross-ancestry associations including known loci at/near CR1, BIN1, TREM2, CD2AP, PTK2B, CLU, SHARPIN, MS4A6A, PICALM, ABCA7, APOE and two novel loci not previously reported at 11p12 (LRRC4C) and 12q24.13 (LHX5-AS1). Reflecting the power of diverse ancestry in GWAS, we observed the SHARPIN locus using 7.1% the sample size of the original discovering single-ancestry GWAS (n=788,989). We additionally identified three GWS ancestry-specific loci at/near (PTPRK (P=2.4x10-8) and GRB14 (P=1.7x10-8) in HIS), and KIAA0825 (P=2.9x10-8 in NHW). Pathway analysis implicated multiple amyloid regulation pathways (strongest with Padjusted=1.6x10-4) and the classical complement pathway (Padjusted=1.3x10-3). Genes at/near our novel loci have known roles in neuronal development (LRRC4C, LHX5-AS1, and PTPRK) and insulin receptor activity regulation (GRB14). These findings provide compelling support for using traditionally-underrepresented populations for gene discovery, even with smaller sample sizes.\nStrong Genetic Overlaps Between Dimensional and Categorical Models of Bipolar Disorders in a Family Sample\nAuthors: Arbona-Lampaya, A.; Sung, H.; D'Amico, A.; Knowles, E. E. M.; Besancon, E. K.; Freifeld, A.; Lacbawan, L.; Lopes, F.; Kassem, L.; Nardi, A. E.; McMahon, F. J.\nScore: 21.8, Published: 2023-07-10 DOI: 10.1101/2023.06.24.23291169\nObjectiveBipolar disorder (BD) often presents with a broad range of symptoms, but there is little agreement as to the heritability and genetic relationships between dimensional and categorical models of this often-disabling disorder. MethodsParticipants in the Amish-Mennonite Bipolar Genetics (AMBiGen) study, which enrolls families with BD and related disorders from Amish and Mennonite communities in North and South America, were assigned a categorical mood disorder diagnosis by structured psychiatric interview and asked to complete the Mood Disorder Questionnaire (MDQ), which assesses lifetime history of cardinal manic symptoms and associated impairment. Principal Component Analysis (PCA) was performed to analyze the dimensions of the MDQ in 726 participants, 212 of whom carried a categorical diagnosis of major mood disorder. SOLAR-ECLIPSE (v9.0.0) was used to estimate heritability and genetic overlaps between MDQ-derived measures and categorical diagnoses among 432 genotyped participants. ResultsAs expected, MDQ scores were significantly higher among individuals diagnosed with BD and related disorders. PCA suggested a three-component model for the MDQ, consistent with the literature. Heritability of the MDQ symptom score was estimated at 30% (p\u003c0.001), which was evenly distributed across its three principal components. Strong and significant genetic correlations were found between categorical diagnoses and most MDQ measures, especially impairment. ConclusionThe results support the MDQ as a dimensional measure of BD. Furthermore, significant heritability and high genetic correlations between MDQ scores and categorical diagnoses suggest a genetic continuity between dimensional and categorical measures of major mood disorders.\nNon-coding variants are a rare cause of recessive developmental disorders in trans with coding variants\nAuthors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium, ; Baralle, D.; Martin, H. C.; Whiffin, N.\nScore: 32.5, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291805\nPurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown. MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing. ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA). ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.\nThe NYCKidSeq randomized controlled trial: Impact of GUIA digitally enhanced genetic counseling in racially and ethnically diverse families\nAuthors: Suckiel, S. A.; Kelly, N. R.; Odgis, J. A.; Gallagher, K. M.; Sebastin, M.; Bonini, K. E.; Marathe, P. N.; Brown, K.; Di Biase, M.; Ramos, M.; Rodriguez, J. E.; Scarimbolo, L.; Insel, B. J.; Ferar, K. D. M.; Zinberg, R. E.; Diaz, G. A.; Greally, J. M.; Abul-Husn, N. S.; Bauman, L. J.; Gelb, B. D.; Horowitz, C. R.; Wasserstein, M. P.; Kenny, E. E.\nScore: 8.8, Published: 2023-07-07 DOI: 10.1101/2023.07.05.23292193\nBackgroundDigital solutions are needed to support rapid increases in the application of genetic and genomic tests (GT) in diverse clinical settings and patient populations. We developed GUIA, a bi-lingual web-based platform that facilitates disclosure of GT results. The NYCKidSeq randomized controlled trial evaluated GUIAs impact on understanding of GT results. MethodsNYCKidSeq enrolled diverse children with neurologic, cardiac, and immunologic conditions who underwent GT. Families were randomized to genetic counseling with GUIA (intervention) or standard of care (SOC) genetic counseling for results disclosure. Parents/legal guardians (participants) completed surveys at baseline, post-results disclosure, and 6-months later. Survey measures assessed the primary study outcomes of perceived understanding of and confidence in explaining their childs GT results and the secondary outcome of objective understanding. We used regression models to evaluate the association between the intervention and the study outcomes. ResultsThe analysis included 551 participants, 270 in the GUIA arm and 281 in SOC. Participants mean age was 41.1 years and 88.6% were mothers. Most participants were Hispanic/Latino(a) (46.3%), White/European American (24.5%), or Black/African American (15.8%). Participants in the GUIA arm had significantly higher perceived understanding post-results (OR=2.8, CI[1.004,7.617], P=0.049) and maintained higher objective understanding over time (OR=1.1, CI[1.004, 1.127], P=0.038) compared to those in the SOC arm. There was no impact on perceived confidence. Hispanic/Latino(a) individuals in the GUIA arm maintained higher perceived understanding (OR=3.9, CI[1.6, 9.3], P=0.003), confidence (OR=2.7, CI[1.021, 7.277], P=0.046), and objective understanding (OR=1.1, CI[1.009, 1.212], P=0.032) compared to SOC. ConclusionsThis trial demonstrates that GUIA positively impacts understanding of GT results in diverse parents of children with suspected genetic conditions. These findings build a case for utilizing GUIA to deliver complex and often ambiguous genetic results. Continued development and evaluation of digital applications in diverse populations are critical for equitably scaling GT offerings in specialty clinics.\nFine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases\nAuthors: Yuan, K.; Longchamps, R. J.; Pardinas, A. F.; Yu, M.; Chen, T.-T.; Lin, S.-C.; Chen, Y.; Lam, M.; Liu, R.; Xia, Y.; Guo, Z.; Shi, W.; Shen, C.; The Schizophrenia Workgroup of Psychiatric Genomics Consortium, ; Daly, M. J.; Neale, B.; Feng, Y.-C. A.; Lin, Y.-F.; Chen, C.-Y.; O'Donovan, M.; Ge, T.; Huang, H.\nScore: 24.8, Published: 2023-07-09 DOI: 10.1101/2023.01.07.23284293\nGenome-wide association studies (GWAS) of human complex traits or diseases often implicate genetic loci that span hundreds or thousands of genetic variants, many of which have similar statistical significance. While statistical fine-mapping in individuals of European ancestries has made important discoveries, cross-population fine-mapping has the potential to improve power and resolution by capitalizing on the genomic diversity across ancestries. Here we present SuSiEx, an accurate and computationally efficient method for cross-population fine-mapping, which builds on the single-population fine-mapping framework, Sum of Single Effects (SuSiE). SuSiEx integrates data from an arbitrary number of ancestries, explicitly models population-specific allele frequencies and LD patterns, accounts for multiple causal variants in a genomic region, and can be applied to GWAS summary statistics. We comprehensively evaluated SuSiEx using simulations, a range of quantitative traits measured in both UK Biobank and Taiwan Biobank, and schizophrenia GWAS across East Asian and European ancestries. In all evaluations, SuSiEx fine-mapped more association signals, produced smaller credible sets and higher posterior inclusion probability (PIP) for putative causal variants, and captured population-specific causal variants.\nExome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank\nAuthors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.\nScore: 23.5, Published: 2023-07-03 DOI: 10.1101/2023.06.29.23291992\nExome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF \u003c 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( \u003c 1.68 x 10-7) and 17 nominally significant ( \u003c 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR \u003c 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.\nBayesian genome-wide TWAS with reference transcriptomic data of brain and blood tissues identified 93 risk genes for Alzheimer's disease dementia\nAuthors: Guo, S.; Yang, J.\nScore: 3.6, Published: 2023-07-12 DOI: 10.1101/2023.07.06.23292336\nBackgroundTranscriptome-wide association study (TWAS) is an influential tool for identifying novel genes associated with complex diseases, where their genetic effects may be mediated through transcriptome. TWAS utilizes reference genetic and transcriptomic data to estimate genetic effect sizes on expression quantitative traits of target genes (i.e., effect sizes of a broad sense of expression quantitative trait loci, eQTL). These estimated effect sizes are then employed as variant weights in burden gene-based association test statistics, facilitating the mapping of risk genes for complex diseases with genome-wide association study (GWAS) data. However, most existing TWAS of Alzheimers disease (AD) dementia have primarily focused on cis-eQTL, disregarding potential trans-eQTL. To overcome this limitation, we applied the Bayesian Genome-wide TWAS (BGW-TWAS) method which incorporated both cis- and trans-eQTL of brain and blood tissues to enhance mapping risk genes for AD dementia. MethodsWe first applied BGW-TWAS to the Genotype-Tissue Expression (GTEx) V8 dataset to estimate cis- and trans-eQTL effect sizes of the prefrontal cortex, cortex, and whole blood tissues. Subsequently, estimated eQTL effect sizes were integrated with the summary data of the most recent GWAS of AD dementia to obtain BGW-TWAS (i.e., gene-based association test) p-values of AD dementia per tissue type. Finally, we used the aggregated Cauchy association test to combine TWAS p-values across three tissues to obtain omnibus TWAS p-values per gene. ResultsWe identified 37 genes in prefrontal cortex, 55 in cortex, and 51 in whole blood that were significantly associated with AD dementia. By combining BGW-TWAS p-values across these three tissues, we obtained 93 significant risk genes including 29 genes primarily due to trans-eQTL and 50 novel genes. Utilizing protein-protein interaction network and phenotype enrichment analyses with these 93 significant risk genes, we detected 5 functional clusters comprised of both known and novel AD risk genes and 7 enriched phenotypes. ConclusionWe applied BGW-TWAS and aggregated Cauchy test methods to integrate both cis- and trans-eQTL data of brain and blood tissues with GWAS summary data to identify risk genes of AD dementia. The risk genes we identified provide novel insights into the underlying biological pathways implicated in AD dementia.\nGenetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis\nAuthors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.\nScore: 1.9, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292611\nBackground Long COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. Methods We used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics' Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. Results Combinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. Conclusion This study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.\n",
  "wordCount" : "3356",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 328.3, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292254" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292254">
        <p class="paperTitle">RNA sequencing uplifts diagnostic rate in undiagnosed rare disease patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jaramillo Oquendo, C.; Wai, H. A.; Rich, W.; Bunyan, D. J.; Thomas, N. S.; Hunt, D.; Lord, J.; Douglas, A. G. L.; Baralle, D.</p>
        <p class="info">Score: 21.2, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292254' target='https://doi.org/10.1101/2023.07.05.23292254'> 10.1101/2023.07.05.23292254</a></p>
        <p class="abstract">RNA-sequencing is increasingly being used as a complementary tool to DNA sequencing in diagnostics where DNA analysis has been uninformative as it enables the identification of alternative splicing (AS), aberrant gene expression and allele specific expression. Using RNA from patient blood, we have used both RNA-sequencing and RT-PCR to detect splicing and gene expression outliers in a heterogenous cohort of 87 patients with suspected Mendelian disorders, 38% of which did not have a candidate sequence variant. Expression outliers were detected using OUTRIDER, and we compared the performance of multiple open-source alternative splicing tools (MAJIQ, rMATS-turbo, and LeafCutterMD) in identifying alternative splicing events. As well as clarifying the impact of variants of uncertain significance (VUSs), we trialled two novel approaches to identify new potential diagnoses in patients with no candidate variants. We were able to assess 85% of VUSs and validate splicing abnormalities in 18/48 patients with a VUS. Furthermore, we identified four new diagnoses by detecting novel AS events in patients with no candidate sequence variants from prior genomic DNA testing (n=33) or those in which the candidate VUS did not affect splicing (n=23) and identified one additional diagnosis through detection of skewed X-inactivation. These results gave an overall uplift in diagnostic yield of 26%. In this work we demonstrate the utility of blood-based RNA analysis in improving diagnostic yields and highlight optimal approaches for such analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.23292311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.23292311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.23292311">
        <p class="paperTitle">Multi-ancestry genome-wide meta-analysis of 56,241 individuals identifies LRRC4C, LHX5-AS1 and nominates ancestry-specific loci PTPRK, GRB14, and KIAA0825 as novel risk loci for Alzheimer disease: the Alzheimer Disease Genetics Consortium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.23292311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.23292311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rajabli, F.; Benchek, P.; Tosto, G.; Kushch, N.; Sha, J.; Bazemore, K.; Zhu, C.; Lee, W.-P.; Haut, J.; Hamilton-Nelson, K. L.; Wheeler, N. R.; Zhao, Y.; Farrell, J. J.; Grunin, M. A.; Leung, Y. Y.; Kuksa, P. P.; Li, D.; Lucio da Fonseca, E.; Mez, J. B.; Palmer, E. L.; Pillai, J.; Sherva, R. M.; Song, Y. E.; Zhang, X.; Iqbal, T.; Pathak, O.; Valladares, O.; Kuzma, A. B.; Abner, E.; Adams, P. M.; Aguirre, A.; Albert, M. S.; Albin, R. L.; Allen, M.; Alvarez, L.; Apostolova, L. G.; Arnold, S. E.; Asthana, S.; Atwood, C. S.; Ayres, G.; Baldwin, C. T.; Barber, R. C.; Barnes, L. L.; Barral, S.; Beach</p>
        <p class="info">Score: 20.1, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.23292311' target='https://doi.org/10.1101/2023.07.06.23292311'> 10.1101/2023.07.06.23292311</a></p>
        <p class="abstract">Limited ancestral diversity has impaired our ability to detect risk variants more prevalent in non-European ancestry groups in genome-wide association studies (GWAS). We constructed and analyzed a multi-ancestry GWAS dataset in the Alzheimers Disease (AD) Genetics Consortium (ADGC) to test for novel shared and ancestry-specific AD susceptibility loci and evaluate underlying genetic architecture in 37,382 non-Hispanic White (NHW), 6,728 African American, 8,899 Hispanic (HIS), and 3,232 East Asian individuals, performing within-ancestry fixed-effects meta-analysis followed by a cross-ancestry random-effects meta-analysis. We identified 13 loci with cross-ancestry associations including known loci at/near CR1, BIN1, TREM2, CD2AP, PTK2B, CLU, SHARPIN, MS4A6A, PICALM, ABCA7, APOE and two novel loci not previously reported at 11p12 (LRRC4C) and 12q24.13 (LHX5-AS1). Reflecting the power of diverse ancestry in GWAS, we observed the SHARPIN locus using 7.1% the sample size of the original discovering single-ancestry GWAS (n=788,989). We additionally identified three GWS ancestry-specific loci at/near (PTPRK (P=2.4x10-8) and GRB14 (P=1.7x10-8) in HIS), and KIAA0825 (P=2.9x10-8 in NHW). Pathway analysis implicated multiple amyloid regulation pathways (strongest with Padjusted=1.6x10-4) and the classical complement pathway (Padjusted=1.3x10-3). Genes at/near our novel loci have known roles in neuronal development (LRRC4C, LHX5-AS1, and PTPRK) and insulin receptor activity regulation (GRB14). These findings provide compelling support for using traditionally-underrepresented populations for gene discovery, even with smaller sample sizes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.23291169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.23291169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.23291169">
        <p class="paperTitle">Strong Genetic Overlaps Between Dimensional and Categorical Models of Bipolar Disorders in a Family Sample</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.23291169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.23291169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arbona-Lampaya, A.; Sung, H.; D&#39;Amico, A.; Knowles, E. E. M.; Besancon, E. K.; Freifeld, A.; Lacbawan, L.; Lopes, F.; Kassem, L.; Nardi, A. E.; McMahon, F. J.</p>
        <p class="info">Score: 21.8, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.23291169' target='https://doi.org/10.1101/2023.06.24.23291169'> 10.1101/2023.06.24.23291169</a></p>
        <p class="abstract">ObjectiveBipolar disorder (BD) often presents with a broad range of symptoms, but there is little agreement as to the heritability and genetic relationships between dimensional and categorical models of this often-disabling disorder.

MethodsParticipants in the Amish-Mennonite Bipolar Genetics (AMBiGen) study, which enrolls families with BD and related disorders from Amish and Mennonite communities in North and South America, were assigned a categorical mood disorder diagnosis by structured psychiatric interview and asked to complete the Mood Disorder Questionnaire (MDQ), which assesses lifetime history of cardinal manic symptoms and associated impairment. Principal Component Analysis (PCA) was performed to analyze the dimensions of the MDQ in 726 participants, 212 of whom carried a categorical diagnosis of major mood disorder. SOLAR-ECLIPSE (v9.0.0) was used to estimate heritability and genetic overlaps between MDQ-derived measures and categorical diagnoses among 432 genotyped participants.

ResultsAs expected, MDQ scores were significantly higher among individuals diagnosed with BD and related disorders. PCA suggested a three-component model for the MDQ, consistent with the literature. Heritability of the MDQ symptom score was estimated at 30% (p&lt;0.001), which was evenly distributed across its three principal components. Strong and significant genetic correlations were found between categorical diagnoses and most MDQ measures, especially impairment.

ConclusionThe results support the MDQ as a dimensional measure of BD. Furthermore, significant heritability and high genetic correlations between MDQ scores and categorical diagnoses suggest a genetic continuity between dimensional and categorical measures of major mood disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291805">
        <p class="paperTitle">Non-coding variants are a rare cause of recessive developmental disorders in trans with coding variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium,  ; Baralle, D.; Martin, H. C.; Whiffin, N.</p>
        <p class="info">Score: 32.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291805' target='https://doi.org/10.1101/2023.06.23.23291805'> 10.1101/2023.06.23.23291805</a></p>
        <p class="abstract">PurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown.

MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing.

ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA).

ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292193">
        <p class="paperTitle">The NYCKidSeq randomized controlled trial: Impact of GUIA digitally enhanced genetic counseling in racially and ethnically diverse families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suckiel, S. A.; Kelly, N. R.; Odgis, J. A.; Gallagher, K. M.; Sebastin, M.; Bonini, K. E.; Marathe, P. N.; Brown, K.; Di Biase, M.; Ramos, M.; Rodriguez, J. E.; Scarimbolo, L.; Insel, B. J.; Ferar, K. D. M.; Zinberg, R. E.; Diaz, G. A.; Greally, J. M.; Abul-Husn, N. S.; Bauman, L. J.; Gelb, B. D.; Horowitz, C. R.; Wasserstein, M. P.; Kenny, E. E.</p>
        <p class="info">Score: 8.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292193' target='https://doi.org/10.1101/2023.07.05.23292193'> 10.1101/2023.07.05.23292193</a></p>
        <p class="abstract">BackgroundDigital solutions are needed to support rapid increases in the application of genetic and genomic tests (GT) in diverse clinical settings and patient populations. We developed GUIA, a bi-lingual web-based platform that facilitates disclosure of GT results. The NYCKidSeq randomized controlled trial evaluated GUIAs impact on understanding of GT results.

MethodsNYCKidSeq enrolled diverse children with neurologic, cardiac, and immunologic conditions who underwent GT. Families were randomized to genetic counseling with GUIA (intervention) or standard of care (SOC) genetic counseling for results disclosure. Parents/legal guardians (participants) completed surveys at baseline, post-results disclosure, and 6-months later. Survey measures assessed the primary study outcomes of perceived understanding of and confidence in explaining their childs GT results and the secondary outcome of objective understanding. We used regression models to evaluate the association between the intervention and the study outcomes.

ResultsThe analysis included 551 participants, 270 in the GUIA arm and 281 in SOC. Participants mean age was 41.1 years and 88.6% were mothers. Most participants were Hispanic/Latino(a) (46.3%), White/European American (24.5%), or Black/African American (15.8%). Participants in the GUIA arm had significantly higher perceived understanding post-results (OR=2.8, CI[1.004,7.617], P=0.049) and maintained higher objective understanding over time (OR=1.1, CI[1.004, 1.127], P=0.038) compared to those in the SOC arm. There was no impact on perceived confidence. Hispanic/Latino(a) individuals in the GUIA arm maintained higher perceived understanding (OR=3.9, CI[1.6, 9.3], P=0.003), confidence (OR=2.7, CI[1.021, 7.277], P=0.046), and objective understanding (OR=1.1, CI[1.009, 1.212], P=0.032) compared to SOC.

ConclusionsThis trial demonstrates that GUIA positively impacts understanding of GT results in diverse parents of children with suspected genetic conditions. These findings build a case for utilizing GUIA to deliver complex and often ambiguous genetic results. Continued development and evaluation of digital applications in diverse populations are critical for equitably scaling GT offerings in specialty clinics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.07.23284293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.07.23284293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.07.23284293">
        <p class="paperTitle">Fine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.07.23284293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.07.23284293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, K.; Longchamps, R. J.; Pardinas, A. F.; Yu, M.; Chen, T.-T.; Lin, S.-C.; Chen, Y.; Lam, M.; Liu, R.; Xia, Y.; Guo, Z.; Shi, W.; Shen, C.; The Schizophrenia Workgroup of Psychiatric Genomics Consortium,  ; Daly, M. J.; Neale, B.; Feng, Y.-C. A.; Lin, Y.-F.; Chen, C.-Y.; O&#39;Donovan, M.; Ge, T.; Huang, H.</p>
        <p class="info">Score: 24.8, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.07.23284293' target='https://doi.org/10.1101/2023.01.07.23284293'> 10.1101/2023.01.07.23284293</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) of human complex traits or diseases often implicate genetic loci that span hundreds or thousands of genetic variants, many of which have similar statistical significance. While statistical fine-mapping in individuals of European ancestries has made important discoveries, cross-population fine-mapping has the potential to improve power and resolution by capitalizing on the genomic diversity across ancestries. Here we present SuSiEx, an accurate and computationally efficient method for cross-population fine-mapping, which builds on the single-population fine-mapping framework, Sum of Single Effects (SuSiE). SuSiEx integrates data from an arbitrary number of ancestries, explicitly models population-specific allele frequencies and LD patterns, accounts for multiple causal variants in a genomic region, and can be applied to GWAS summary statistics. We comprehensively evaluated SuSiEx using simulations, a range of quantitative traits measured in both UK Biobank and Taiwan Biobank, and schizophrenia GWAS across East Asian and European ancestries. In all evaluations, SuSiEx fine-mapped more association signals, produced smaller credible sets and higher posterior inclusion probability (PIP) for putative causal variants, and captured population-specific causal variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23291992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23291992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23291992">
        <p class="paperTitle">Exome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23291992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23291992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.</p>
        <p class="info">Score: 23.5, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23291992' target='https://doi.org/10.1101/2023.06.29.23291992'> 10.1101/2023.06.29.23291992</a></p>
        <p class="abstract">Exome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF &lt; 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( &lt; 1.68 x 10-7) and 17 nominally significant ( &lt; 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR &lt; 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.23292336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.23292336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.23292336">
        <p class="paperTitle">Bayesian genome-wide TWAS with reference transcriptomic data of brain and blood tissues identified 93 risk genes for Alzheimer&#39;s disease dementia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.23292336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.23292336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, S.; Yang, J.</p>
        <p class="info">Score: 3.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.23292336' target='https://doi.org/10.1101/2023.07.06.23292336'> 10.1101/2023.07.06.23292336</a></p>
        <p class="abstract">BackgroundTranscriptome-wide association study (TWAS) is an influential tool for identifying novel genes associated with complex diseases, where their genetic effects may be mediated through transcriptome. TWAS utilizes reference genetic and transcriptomic data to estimate genetic effect sizes on expression quantitative traits of target genes (i.e., effect sizes of a broad sense of expression quantitative trait loci, eQTL). These estimated effect sizes are then employed as variant weights in burden gene-based association test statistics, facilitating the mapping of risk genes for complex diseases with genome-wide association study (GWAS) data. However, most existing TWAS of Alzheimers disease (AD) dementia have primarily focused on cis-eQTL, disregarding potential trans-eQTL. To overcome this limitation, we applied the Bayesian Genome-wide TWAS (BGW-TWAS) method which incorporated both cis- and trans-eQTL of brain and blood tissues to enhance mapping risk genes for AD dementia.

MethodsWe first applied BGW-TWAS to the Genotype-Tissue Expression (GTEx) V8 dataset to estimate cis- and trans-eQTL effect sizes of the prefrontal cortex, cortex, and whole blood tissues. Subsequently, estimated eQTL effect sizes were integrated with the summary data of the most recent GWAS of AD dementia to obtain BGW-TWAS (i.e., gene-based association test) p-values of AD dementia per tissue type. Finally, we used the aggregated Cauchy association test to combine TWAS p-values across three tissues to obtain omnibus TWAS p-values per gene.

ResultsWe identified 37 genes in prefrontal cortex, 55 in cortex, and 51 in whole blood that were significantly associated with AD dementia. By combining BGW-TWAS p-values across these three tissues, we obtained 93 significant risk genes including 29 genes primarily due to trans-eQTL and 50 novel genes. Utilizing protein-protein interaction network and phenotype enrichment analyses with these 93 significant risk genes, we detected 5 functional clusters comprised of both known and novel AD risk genes and 7 enriched phenotypes.

ConclusionWe applied BGW-TWAS and aggregated Cauchy test methods to integrate both cis- and trans-eQTL data of brain and blood tissues with GWAS summary data to identify risk genes of AD dementia. The risk genes we identified provide novel insights into the underlying biological pathways implicated in AD dementia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292611">
        <p class="paperTitle">Genetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.</p>
        <p class="info">Score: 1.9, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292611' target='https://doi.org/10.1101/2023.07.13.23292611'> 10.1101/2023.07.13.23292611</a></p>
        <p class="abstract">Background Long COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. Methods We used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics&#39; Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. Results Combinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. Conclusion This study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
