<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 434.1, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nGenetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis\nAuthors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.\nScore: 46.2, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292611\nBackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.\nPrevalence, Morbidity, and Mortality of 1,609 Men with Diagnosed and Undiagnosed Sex Chromosome Aneuploidy: Results from the Diverse Million Veteran Program (MVP) Cohort\nAuthors: Davis, S. M.; Teerlink, C.; Lynch, J. A.; Gorman, B. R.; Pagadala, M.; Liu, A.; Panizzon, M. S.; Merritt, V. C.; Genovese, G.; Pyarajan, S.; Ross, J. L.; Hauger, R. L.\nScore: 29.5, Published: 2023-07-16 DOI: 10.1101/2023.07.15.23292710\nSTRUCTURED ABSTRACTO_ST_ABSImportanceC_ST_ABSThe reported phenotypes of men with 47,XXY and 47,XYY syndromes include tall stature, multisystem comorbidities, and poor health-related quality of life (HRQoL). However, knowledge about these sex chromosome aneuploidy (SCA) conditions has been derived from studies in the \u003c15% of patients who are clinically diagnosed and also lack diversity in age and genetic ancestry. ObjectivesDetermine the prevalence of clinically diagnosed and undiagnosed X or Y chromosome aneuploidy among men enrolled in the Million Veteran Program (MVP); describe military service metrics of men with SCAs; compare morbidity and mortality outcomes between men with SCA with and without a clinical diagnosis to matched controls. DesignCross-sectional, case-control SettingUnited States Veterans Administration Healthcare System ParticipantsBiologic males enrolled in the MVP biobank with genomic identification of an additional X or Y chromosome (cases); controls matched 1:5 on sex, age, and genetic ancestry Main Outcome(s) and Measure(s)Prevalence of men with SCAs from genomic analysis; clinical SCA diagnosis; Charlson Comorbidity Index (CCI); rates of outpatient, inpatient, and emergency encounters per year; self-reported health outcomes; standardized mortality ratio (SMR) ResultsAn additional X or Y chromosome was present in 145 and 125 per 100,000 males in the MVP, respectively, with the highest prevalence among men with European and East Asian ancestry. At a mean age of 61{+/-}12 years, 74% of male veterans with 47,XXY and \u003e99% with 47,XYY remained undiagnosed. Individuals with 47,XXY (n=862) and 47,XYY (n=747) had similar military service history, all-cause SMR, and age of death compared to matched controls. CCI and healthcare utilization were higher among individuals with SCA, while several measures of HRQoL were lower. Men with a clinical diagnosis of 47,XXY had higher healthcare utilization but lower comorbidity score compared to those undiagnosed. Conclusion and RelevanceOne in 370 males in the MVP cohort have SCA, a prevalence comparable to estimates in the general population. While these men have successfully served in the military, they have higher morbidity and report poorer HRQoL with aging. Longer longitudinal follow-up of this sample will be informative for clinical and patient-reported outcomes, the role of ancestry, and mortality statistics. KEY POINTSO_LIComparable to the general population, approximately 1 in 370 male veterans have a sex chromosome aneuploidy, but most are undiagnosed. C_LIO_LIMen with X or Y chromosome aneuploidy successfully complete US miliary duty with similar service history compared to their 46,XY peers. C_LIO_LIMedical comorbidities and healthcare utilization metrics are higher in male veterans with 47,XXY and 47,XYY during aging, however life expectancy is similar to matched controls. C_LI\nEffect of prioritized SORL1 missense variants supports clinical consideration for familial Alzheimer's Disease\nAuthors: Holstege, H.; de Waal, M. W. J.; Tesi, N.; van der Lee, S. J.; ADES consortium, ; ADSP consortium, ; StEP-AD consortium, ; Knight-ADRC, ; UCSF/NYGC/UAB, ; Vogel, M.; van Spaendonk, R.; Hulsman, M.; Andersen, O. M.\nScore: 3.5, Published: 2023-07-16 DOI: 10.1101/2023.07.13.23292622\nBackgroundRare variants in the SORL1 gene have been associated with increased risk of Alzheimers disease (AD). While protein-truncating variants (PTVs) are observed almost exclusively in AD patients, most variants are rare missense variants that can be benign, risk-increasing, and recent reports have indicated that some variants are causative for disease. However, since SORL1 is currently not considered an autosomal dominant Alzheimer Disease gene (ADAD), segregation analyses are not performed, which complicates the identification of additional clinically important missense variants. MethodsWe prioritized highly conserved and functionally relevant SORL1 missense variants by considering the functional effects of homologous variants on proteins that share domains with SORL1 (domain-mapping of disease mutations, DMDM) into. We used this variant prioritization approach to annotate SORL1 variants identified in a previously assembled exome sequencing dataset encompassing 18,959 AD cases and 21,893 non-demented controls, and we tested the effect of high, moderate, low and no priority missense variants and specific variant subtypes on disease risk and age at onset. ResultsHigh priority missense variants (HPV) associated with a 6.4-fold increased risk of AD (95%CI: 4.3 - 9.7, p=2.1x10-24), which concentrated on early onset AD (OREOAD 10.5, 95%CI: 6.8 - 16.3, p=3.0x10-29) vs. late onset AD (ORLOAD=4.5, 95%CI 2.85 - 6.94; p=4.9x10-11). The median age at onset of HPV carriers was \u003e8-years earlier than carriers of wild-type SORL1. Intriguingly, specific subtypes of HPVs, including those affecting residues in the YWTD-motif or the calcium cage, occurred only in AD cases and carriers of these variants had an earlier age at onset compared to carriers of PTVs, indicative of a dominant negative effect. Carriers of other HPVs had an age at onset that overlapped with carriers of PTVs, suggesting they lead to haploinsufficiency. Yet other variants had a slightly later age at onset than PTVs, suggesting that their effect on SORL1 function was milder than losing a copy. Variants annotated as moderate, low and no priority did not have an effect on AD. ConclusionsNext to carriers of SORL1 PTVs, carriers of selected missense variants should be considered for segregation analyses, which will likely provide evidence for autosomal dominant inheritance for additional SORL1 missense variants.\nMulti-ancestry genome-wide meta-analysis of 56,241 individuals identifies LRRC4C, LHX5-AS1 and nominates ancestry-specific loci PTPRK, GRB14, and KIAA0825 as novel risk loci for Alzheimer disease: the Alzheimer Disease Genetics Consortium\nAuthors: Rajabli, F.; Benchek, P.; Tosto, G.; Kushch, N.; Sha, J.; Bazemore, K.; Zhu, C.; Lee, W.-P.; Haut, J.; Hamilton-Nelson, K. L.; Wheeler, N. R.; Zhao, Y.; Farrell, J. J.; Grunin, M. A.; Leung, Y. Y.; Kuksa, P. P.; Li, D.; Lucio da Fonseca, E.; Mez, J. B.; Palmer, E. L.; Pillai, J.; Sherva, R. M.; Song, Y. E.; Zhang, X.; Iqbal, T.; Pathak, O.; Valladares, O.; Kuzma, A. B.; Abner, E.; Adams, P. M.; Aguirre, A.; Albert, M. S.; Albin, R. L.; Allen, M.; Alvarez, L.; Apostolova, L. G.; Arnold, S. E.; Asthana, S.; Atwood, C. S.; Ayres, G.; Baldwin, C. T.; Barber, R. C.; Barnes, L. L.; Barral, S.; Beach\nScore: 23.0, Published: 2023-07-08 DOI: 10.1101/2023.07.06.23292311\nLimited ancestral diversity has impaired our ability to detect risk variants more prevalent in non-European ancestry groups in genome-wide association studies (GWAS). We constructed and analyzed a multi-ancestry GWAS dataset in the Alzheimers Disease (AD) Genetics Consortium (ADGC) to test for novel shared and ancestry-specific AD susceptibility loci and evaluate underlying genetic architecture in 37,382 non-Hispanic White (NHW), 6,728 African American, 8,899 Hispanic (HIS), and 3,232 East Asian individuals, performing within-ancestry fixed-effects meta-analysis followed by a cross-ancestry random-effects meta-analysis. We identified 13 loci with cross-ancestry associations including known loci at/near CR1, BIN1, TREM2, CD2AP, PTK2B, CLU, SHARPIN, MS4A6A, PICALM, ABCA7, APOE and two novel loci not previously reported at 11p12 (LRRC4C) and 12q24.13 (LHX5-AS1). Reflecting the power of diverse ancestry in GWAS, we observed the SHARPIN locus using 7.1% the sample size of the original discovering single-ancestry GWAS (n=788,989). We additionally identified three GWS ancestry-specific loci at/near (PTPRK (P=2.4x10-8) and GRB14 (P=1.7x10-8) in HIS), and KIAA0825 (P=2.9x10-8 in NHW). Pathway analysis implicated multiple amyloid regulation pathways (strongest with Padjusted=1.6x10-4) and the classical complement pathway (Padjusted=1.3x10-3). Genes at/near our novel loci have known roles in neuronal development (LRRC4C, LHX5-AS1, and PTPRK) and insulin receptor activity regulation (GRB14). These findings provide compelling support for using traditionally-underrepresented populations for gene discovery, even with smaller sample sizes.\nPA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder\nAuthors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium, ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.\nScore: 7.5, Published: 2023-07-12 DOI: 10.1101/2023.06.23.23291611\nLarge biobank samples provide an opportunity to integrate broad phenotyping, familial records, and molecular genetics data to study complex traits and diseases. We introduce Pearson-Aitken Family Genetic Risk Scores (PA-FGRS), a new method for estimating disease liability from patterns of diagnoses in extended, age-censored genealogical records. We then apply the method to study a paradigmatic complex disorder, Major Depressive Disorder (MDD), using the iPSYCH2015 case-cohort study of 30,949 MDD cases, 39,655 random population controls, and more than 2 million relatives. We show that combining PA-FGRS liabilities estimated from family records with molecular genotypes of probands improves the three lines of inquiry. Incorporating PA-FGRS liabilities improves classification of MDD over and above polygenic scores, identifies robust genetic contributions to clinical heterogeneity in MDD associated with comorbidity, recurrence, and severity, and can improve the power of genome-wide association studies (GWAS). Our method is flexible and easy to use and our study approaches are generalizable to other data sets and other complex traits and diseases.\nGenetic structure of major depression symptoms across clinical and community cohorts\nAuthors: Adams, M. J.; Thorp, J. G.; Jermy, B. S.; Kwong, A. S. F.; Koiv, K.; Grotzinger, A. D.; Nivard, M. G.; Marshall, S.; Milaneschi, Y.; Baune, B. T.; Mueller-Myhsok, B.; Penninx, B. W.; Boomsma, D. I.; Levinson, D. F.; Breen, G.; Pistis, G.; Grabe, H. J.; Tiemeier, H.; Berger, K.; Rietschel, M.; Magnusson, P. K.; Uher, R.; Hamilton, S. P.; Lucae, S.; Lehto, K.; Li, Q. S.; Byrne, E. M.; Hickie, I. B.; Martin, N. G.; Medland, S. E.; Wray, N. R.; Tucker-Drob, E. M.; Estonian Biobank Research Team, ; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, ; Lewis, C. M.; McI\nScore: 2.2, Published: 2023-07-07 DOI: 10.1101/2023.07.05.23292214\nDiagnostic criteria for major depressive disorder allow for heterogeneous symptom profiles but genetic analysis of major depressive symptoms has the potential to identify clinical and aetiological subtypes. There are several challenges to integrating symptom data from genetically-informative cohorts, such as sample size differences between clinical and community cohorts and various patterns of missing data. We conducted genome-wide association studies of major depressive symptoms in three clinical cohorts that were enriched for affected participants (Psychiatric Genomics Consortium, Australian Genetics of Depression Study, Generation Scotland) and three community cohorts (Avon Longitudinal Study of Parents and Children, Estonian Biobank, and UK Biobank). We fit a series of confirmatory factor models with factors that accounted for how symptom data was sampled and then compared alternative models with different symptom factors. The best fitting model had a distinct factor for Appetite/Weight symptoms and an additional measurement factor that accounted for missing data patterns in the community cohorts (use of Depression and Anhedonia as gating symptoms). The results show the importance of assessing the directionality of symptoms (such as hypersomnia versus insomnia) and of accounting for study and measurement design when meta-analysing genetic association data.\nNovel genomic loci and pathways influence patterns of structural covariance in the human brain\nAuthors: WEN, J.; nasrallah, i.; Abdulkadir, A.; Satterthwaite, T. D.; yang, z.; erus, g.; Robert-Fitzgerald, T.; singh, a.; sotiras, a.; boquetipujadas, a.; mamourian, e.; doshi, j.; cui, y.; srinivasan, d.; Chen, J.; hwang, g.; bergman, m.; bao, j.; veturi, y.; zhou, z.; yang, s.; dazzan, p.; kahn, r.; Schnack, H.; Zanetti, M.; Meisenzahl, E.; Busatto, G.; Crespo-Facorro, B.; Pantelis, C.; Wood, S.; Zhuo, C.; Shinohara, R.; Gur, R.; gur, r.; koutsouleris, n.; Wolf, D.; Saykin, A. J.; Ritchie, M. D.; Shen, L.; Thompson, P.; colliot, o.; wittfeld, k.; Grabe, H.; Tosun, D.; Bilgel, M.; An, Y.; Marcus, D\nScore: 4.6, Published: 2023-07-17 DOI: 10.1101/2022.07.20.22277727\nNormal and pathologic neurobiological processes influence brain morphology in coordinated ways that give rise to patterns of structural covariance (PSC) across brain regions and individuals during brain aging and diseases. The genetic underpinnings of these patterns remain largely unknown. We apply a stochastic multivariate factorization method to a diverse population of 50,699 individuals (12 studies, 130 sites) and derive data-driven, multi-scale PSCs of regional brain size. PSCs were significantly correlated with 915 genomic loci in the discovery set, 617 of which are novel, and 72% were independently replicated. Key pathways influencing PSCs involve reelin signaling, apoptosis, neurogenesis, and appendage development, while pathways of breast cancer indicate potential interplays between brain metastasis and PSCs associated with neurodegeneration and dementia. Using support vector machines, multi-scale PSCs effectively derive imaging signatures of several brain diseases. Our results elucidate new genetic and biological underpinnings that influence structural covariance patterns in the human brain. Significance statementThe coordinated patterns of changes in the human brain throughout life, driven by brain development, aging, and diseases, remain largely unexplored regarding their underlying genetic determinants. This study delineates 2003 multi-scale patterns of structural covariance (PSCs) and identifies 617 novel genomic loci, with the mapped genes enriched in biological pathways implicated in reelin signaling, apoptosis, neurogenesis, and appendage development. Overall, the 2003 PSCs provide new genetic insights into understanding human brain morphological changes and demonstrate great potential in predicting various neurologic conditions.\nPOPULATION FREQUENCY OF REPEAT EXPANSIONS INDICATES INCREASED DISEASE PREVALENCE ESTIMATES ACROSS DIFFERENT POPULATIONS\nAuthors: Ibanez Garikano, K.; Jadhav, B.; Facchini, S.; Garg, P.; Zanovello, M.; Martin Trujillo, A.; Gies, S.; Galassi Deforie, V.; Gagliardi, D.; Hensman Moss, D.; Moutsianas, L.; EUROSCA, n.; Genomics England Research Consortium, ; Shoai, M.; Caulifield, M. J.; Cortese, A.; Escott Price, V.; Hardy, J.; Houlden, H.; Sharp, A. J.; Tucci, A.\nScore: 17.8, Published: 2023-07-06 DOI: 10.1101/2023.07.03.23292162\nRepeat expansion disorders (REDs) are a devastating group of predominantly neurological diseases. Together they are common, affecting 1 in 3,000 people worldwide with population-specific differences. However, prevalence estimates of REDs are hampered by heterogeneous clinical presentation, variable geographic distributions, and technological limitations leading to under-ascertainment. Here, leveraging whole genome sequencing data from 82,176 individuals from different populations we found an overall carrier frequency of REDs of 1 in 340 individuals. Modelling disease prevalence using genetic data, age at onset and survival, we show that REDs are up to 3-fold more prevalent than currently reported figures. While some REDs are population-specific, e.g. Huntingtons disease type 2, most REDs are represented in all broad genetic ancestries, including Africans and Asians, challenging the notion that some REDs are found only in European populations. These results have worldwide implications for local and global health communities in the diagnosis and management of REDs both at local and global levels.\nMulti-Omic Graph Diagnosis (MOGDx) : A data integration tool to perform classification tasks for heterogeneous diseases\nAuthors: Ryan, B.; Marioni, R. E.; Simpson, T. I.\nScore: 2.8, Published: 2023-07-09 DOI: 10.1101/2023.07.09.23292410\nHeterogeneity in human diseases presents challenges in diagnosis and treatments due to the broad range of manifestations and symptoms. With the rapid development of labelled multi-omic data, integrative machine learning methods have achieved breakthroughs in treatments by redefining these diseases at a more granular level. These approaches often have limitations in scalability, oversimplification, and handling of missing data. In this study, we introduce Multi-Omic Graph Diagnosis (MOGDx), a flexible command line tool for the integration of multi-omic data to perform classification tasks for heterogeneous diseases. MOGDx is a network integrative method that combines patient similarity networks with a reduced vector representation of genomic data. The reduced vector is derived from the latent embeddings of an auto-encoder and the combined network is fed into a graph convolutional network for classification. MOGDx was evaluated on three datasets from the cancer genome atlas for breast invasive carcinoma, kidney cancer, and low grade glioma. MOGDx demonstrated state-of-the-art performance and an ability to identify relevant multi-omic markers in each task. It did so while integrating more genomic measures with greater patient coverage compared to other network integrative methods. MOGDx is available to download from https://github.com/biomedicalinformaticsgroup/MOGDx. Overall, MOGDx is a promising tool for integrating multi-omic data, classifying heterogeneous diseases, and interpreting genomic markers.\n",
  "wordCount" : "3496",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 434.1, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292611">
        <p class="paperTitle">Genetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.</p>
        <p class="info">Score: 46.2, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292611' target='https://doi.org/10.1101/2023.07.13.23292611'> 10.1101/2023.07.13.23292611</a></p>
        <p class="abstract">BackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms.

MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms.

ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases.

Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3.

Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank.

Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS.

ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.23292710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.23292710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.23292710">
        <p class="paperTitle">Prevalence, Morbidity, and Mortality of 1,609 Men with Diagnosed and Undiagnosed Sex Chromosome Aneuploidy: Results from the Diverse Million Veteran Program (MVP) Cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.23292710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.23292710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davis, S. M.; Teerlink, C.; Lynch, J. A.; Gorman, B. R.; Pagadala, M.; Liu, A.; Panizzon, M. S.; Merritt, V. C.; Genovese, G.; Pyarajan, S.; Ross, J. L.; Hauger, R. L.</p>
        <p class="info">Score: 29.5, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.23292710' target='https://doi.org/10.1101/2023.07.15.23292710'> 10.1101/2023.07.15.23292710</a></p>
        <p class="abstract">STRUCTURED ABSTRACTO_ST_ABSImportanceC_ST_ABSThe reported phenotypes of men with 47,XXY and 47,XYY syndromes include tall stature, multisystem comorbidities, and poor health-related quality of life (HRQoL). However, knowledge about these sex chromosome aneuploidy (SCA) conditions has been derived from studies in the &lt;15% of patients who are clinically diagnosed and also lack diversity in age and genetic ancestry.

ObjectivesDetermine the prevalence of clinically diagnosed and undiagnosed X or Y chromosome aneuploidy among men enrolled in the Million Veteran Program (MVP); describe military service metrics of men with SCAs; compare morbidity and mortality outcomes between men with SCA with and without a clinical diagnosis to matched controls.

DesignCross-sectional, case-control

SettingUnited States Veterans Administration Healthcare System

ParticipantsBiologic males enrolled in the MVP biobank with genomic identification of an additional X or Y chromosome (cases); controls matched 1:5 on sex, age, and genetic ancestry

Main Outcome(s) and Measure(s)Prevalence of men with SCAs from genomic analysis; clinical SCA diagnosis; Charlson Comorbidity Index (CCI); rates of outpatient, inpatient, and emergency encounters per year; self-reported health outcomes; standardized mortality ratio (SMR)

ResultsAn additional X or Y chromosome was present in 145 and 125 per 100,000 males in the MVP, respectively, with the highest prevalence among men with European and East Asian ancestry. At a mean age of 61{&#43;/-}12 years, 74% of male veterans with 47,XXY and &gt;99% with 47,XYY remained undiagnosed. Individuals with 47,XXY (n=862) and 47,XYY (n=747) had similar military service history, all-cause SMR, and age of death compared to matched controls. CCI and healthcare utilization were higher among individuals with SCA, while several measures of HRQoL were lower. Men with a clinical diagnosis of 47,XXY had higher healthcare utilization but lower comorbidity score compared to those undiagnosed.

Conclusion and RelevanceOne in 370 males in the MVP cohort have SCA, a prevalence comparable to estimates in the general population. While these men have successfully served in the military, they have higher morbidity and report poorer HRQoL with aging. Longer longitudinal follow-up of this sample will be informative for clinical and patient-reported outcomes, the role of ancestry, and mortality statistics.

KEY POINTSO_LIComparable to the general population, approximately 1 in 370 male veterans have a sex chromosome aneuploidy, but most are undiagnosed.
C_LIO_LIMen with X or Y chromosome aneuploidy successfully complete US miliary duty with similar service history compared to their 46,XY peers.
C_LIO_LIMedical comorbidities and healthcare utilization metrics are higher in male veterans with 47,XXY and 47,XYY during aging, however life expectancy is similar to matched controls.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292622">
        <p class="paperTitle">Effect of prioritized SORL1 missense variants supports clinical consideration for familial Alzheimer&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holstege, H.; de Waal, M. W. J.; Tesi, N.; van der Lee, S. J.; ADES consortium,  ; ADSP consortium,  ; StEP-AD consortium,  ; Knight-ADRC,  ; UCSF/NYGC/UAB,  ; Vogel, M.; van Spaendonk, R.; Hulsman, M.; Andersen, O. M.</p>
        <p class="info">Score: 3.5, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292622' target='https://doi.org/10.1101/2023.07.13.23292622'> 10.1101/2023.07.13.23292622</a></p>
        <p class="abstract">BackgroundRare variants in the SORL1 gene have been associated with increased risk of Alzheimers disease (AD). While protein-truncating variants (PTVs) are observed almost exclusively in AD patients, most variants are rare missense variants that can be benign, risk-increasing, and recent reports have indicated that some variants are causative for disease. However, since SORL1 is currently not considered an autosomal dominant Alzheimer Disease gene (ADAD), segregation analyses are not performed, which complicates the identification of additional clinically important missense variants.

MethodsWe prioritized highly conserved and functionally relevant SORL1 missense variants by considering the functional effects of homologous variants on proteins that share domains with SORL1 (domain-mapping of disease mutations, DMDM) into. We used this variant prioritization approach to annotate SORL1 variants identified in a previously assembled exome sequencing dataset encompassing 18,959 AD cases and 21,893 non-demented controls, and we tested the effect of high, moderate, low and no priority missense variants and specific variant subtypes on disease risk and age at onset.

ResultsHigh priority missense variants (HPV) associated with a 6.4-fold increased risk of AD (95%CI: 4.3 - 9.7, p=2.1x10-24), which concentrated on early onset AD (OREOAD 10.5, 95%CI: 6.8 - 16.3, p=3.0x10-29) vs. late onset AD (ORLOAD=4.5, 95%CI 2.85 - 6.94; p=4.9x10-11). The median age at onset of HPV carriers was &gt;8-years earlier than carriers of wild-type SORL1. Intriguingly, specific subtypes of HPVs, including those affecting residues in the YWTD-motif or the calcium cage, occurred only in AD cases and carriers of these variants had an earlier age at onset compared to carriers of PTVs, indicative of a dominant negative effect. Carriers of other HPVs had an age at onset that overlapped with carriers of PTVs, suggesting they lead to haploinsufficiency. Yet other variants had a slightly later age at onset than PTVs, suggesting that their effect on SORL1 function was milder than losing a copy. Variants annotated as moderate, low and no priority did not have an effect on AD.

ConclusionsNext to carriers of SORL1 PTVs, carriers of selected missense variants should be considered for segregation analyses, which will likely provide evidence for autosomal dominant inheritance for additional SORL1 missense variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.23292311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.23292311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.23292311">
        <p class="paperTitle">Multi-ancestry genome-wide meta-analysis of 56,241 individuals identifies LRRC4C, LHX5-AS1 and nominates ancestry-specific loci PTPRK, GRB14, and KIAA0825 as novel risk loci for Alzheimer disease: the Alzheimer Disease Genetics Consortium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.23292311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.23292311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rajabli, F.; Benchek, P.; Tosto, G.; Kushch, N.; Sha, J.; Bazemore, K.; Zhu, C.; Lee, W.-P.; Haut, J.; Hamilton-Nelson, K. L.; Wheeler, N. R.; Zhao, Y.; Farrell, J. J.; Grunin, M. A.; Leung, Y. Y.; Kuksa, P. P.; Li, D.; Lucio da Fonseca, E.; Mez, J. B.; Palmer, E. L.; Pillai, J.; Sherva, R. M.; Song, Y. E.; Zhang, X.; Iqbal, T.; Pathak, O.; Valladares, O.; Kuzma, A. B.; Abner, E.; Adams, P. M.; Aguirre, A.; Albert, M. S.; Albin, R. L.; Allen, M.; Alvarez, L.; Apostolova, L. G.; Arnold, S. E.; Asthana, S.; Atwood, C. S.; Ayres, G.; Baldwin, C. T.; Barber, R. C.; Barnes, L. L.; Barral, S.; Beach</p>
        <p class="info">Score: 23.0, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.23292311' target='https://doi.org/10.1101/2023.07.06.23292311'> 10.1101/2023.07.06.23292311</a></p>
        <p class="abstract">Limited ancestral diversity has impaired our ability to detect risk variants more prevalent in non-European ancestry groups in genome-wide association studies (GWAS). We constructed and analyzed a multi-ancestry GWAS dataset in the Alzheimers Disease (AD) Genetics Consortium (ADGC) to test for novel shared and ancestry-specific AD susceptibility loci and evaluate underlying genetic architecture in 37,382 non-Hispanic White (NHW), 6,728 African American, 8,899 Hispanic (HIS), and 3,232 East Asian individuals, performing within-ancestry fixed-effects meta-analysis followed by a cross-ancestry random-effects meta-analysis. We identified 13 loci with cross-ancestry associations including known loci at/near CR1, BIN1, TREM2, CD2AP, PTK2B, CLU, SHARPIN, MS4A6A, PICALM, ABCA7, APOE and two novel loci not previously reported at 11p12 (LRRC4C) and 12q24.13 (LHX5-AS1). Reflecting the power of diverse ancestry in GWAS, we observed the SHARPIN locus using 7.1% the sample size of the original discovering single-ancestry GWAS (n=788,989). We additionally identified three GWS ancestry-specific loci at/near (PTPRK (P=2.4x10-8) and GRB14 (P=1.7x10-8) in HIS), and KIAA0825 (P=2.9x10-8 in NHW). Pathway analysis implicated multiple amyloid regulation pathways (strongest with Padjusted=1.6x10-4) and the classical complement pathway (Padjusted=1.3x10-3). Genes at/near our novel loci have known roles in neuronal development (LRRC4C, LHX5-AS1, and PTPRK) and insulin receptor activity regulation (GRB14). These findings provide compelling support for using traditionally-underrepresented populations for gene discovery, even with smaller sample sizes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291611">
        <p class="paperTitle">PA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium,  ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.</p>
        <p class="info">Score: 7.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291611' target='https://doi.org/10.1101/2023.06.23.23291611'> 10.1101/2023.06.23.23291611</a></p>
        <p class="abstract">Large biobank samples provide an opportunity to integrate broad phenotyping, familial records, and molecular genetics data to study complex traits and diseases. We introduce Pearson-Aitken Family Genetic Risk Scores (PA-FGRS), a new method for estimating disease liability from patterns of diagnoses in extended, age-censored genealogical records. We then apply the method to study a paradigmatic complex disorder, Major Depressive Disorder (MDD), using the iPSYCH2015 case-cohort study of 30,949 MDD cases, 39,655 random population controls, and more than 2 million relatives. We show that combining PA-FGRS liabilities estimated from family records with molecular genotypes of probands improves the three lines of inquiry. Incorporating PA-FGRS liabilities improves classification of MDD over and above polygenic scores, identifies robust genetic contributions to clinical heterogeneity in MDD associated with comorbidity, recurrence, and severity, and can improve the power of genome-wide association studies (GWAS). Our method is flexible and easy to use and our study approaches are generalizable to other data sets and other complex traits and diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292214">
        <p class="paperTitle">Genetic structure of major depression symptoms across clinical and community cohorts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adams, M. J.; Thorp, J. G.; Jermy, B. S.; Kwong, A. S. F.; Koiv, K.; Grotzinger, A. D.; Nivard, M. G.; Marshall, S.; Milaneschi, Y.; Baune, B. T.; Mueller-Myhsok, B.; Penninx, B. W.; Boomsma, D. I.; Levinson, D. F.; Breen, G.; Pistis, G.; Grabe, H. J.; Tiemeier, H.; Berger, K.; Rietschel, M.; Magnusson, P. K.; Uher, R.; Hamilton, S. P.; Lucae, S.; Lehto, K.; Li, Q. S.; Byrne, E. M.; Hickie, I. B.; Martin, N. G.; Medland, S. E.; Wray, N. R.; Tucker-Drob, E. M.; Estonian Biobank Research Team,  ; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium,  ; Lewis, C. M.; McI</p>
        <p class="info">Score: 2.2, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292214' target='https://doi.org/10.1101/2023.07.05.23292214'> 10.1101/2023.07.05.23292214</a></p>
        <p class="abstract">Diagnostic criteria for major depressive disorder allow for heterogeneous symptom profiles but genetic analysis of major depressive symptoms has the potential to identify clinical and aetiological subtypes. There are several challenges to integrating symptom data from genetically-informative cohorts, such as sample size differences between clinical and community cohorts and various patterns of missing data. We conducted genome-wide association studies of major depressive symptoms in three clinical cohorts that were enriched for affected participants (Psychiatric Genomics Consortium, Australian Genetics of Depression Study, Generation Scotland) and three community cohorts (Avon Longitudinal Study of Parents and Children, Estonian Biobank, and UK Biobank). We fit a series of confirmatory factor models with factors that accounted for how symptom data was sampled and then compared alternative models with different symptom factors. The best fitting model had a distinct factor for Appetite/Weight symptoms and an additional measurement factor that accounted for missing data patterns in the community cohorts (use of Depression and Anhedonia as gating symptoms). The results show the importance of assessing the directionality of symptoms (such as hypersomnia versus insomnia) and of accounting for study and measurement design when meta-analysing genetic association data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.20.22277727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.20.22277727" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.20.22277727">
        <p class="paperTitle">Novel genomic loci and pathways influence patterns of structural covariance in the human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.20.22277727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.20.22277727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: WEN, J.; nasrallah, i.; Abdulkadir, A.; Satterthwaite, T. D.; yang, z.; erus, g.; Robert-Fitzgerald, T.; singh, a.; sotiras, a.; boquetipujadas, a.; mamourian, e.; doshi, j.; cui, y.; srinivasan, d.; Chen, J.; hwang, g.; bergman, m.; bao, j.; veturi, y.; zhou, z.; yang, s.; dazzan, p.; kahn, r.; Schnack, H.; Zanetti, M.; Meisenzahl, E.; Busatto, G.; Crespo-Facorro, B.; Pantelis, C.; Wood, S.; Zhuo, C.; Shinohara, R.; Gur, R.; gur, r.; koutsouleris, n.; Wolf, D.; Saykin, A. J.; Ritchie, M. D.; Shen, L.; Thompson, P.; colliot, o.; wittfeld, k.; Grabe, H.; Tosun, D.; Bilgel, M.; An, Y.; Marcus, D</p>
        <p class="info">Score: 4.6, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.20.22277727' target='https://doi.org/10.1101/2022.07.20.22277727'> 10.1101/2022.07.20.22277727</a></p>
        <p class="abstract">Normal and pathologic neurobiological processes influence brain morphology in coordinated ways that give rise to patterns of structural covariance (PSC) across brain regions and individuals during brain aging and diseases. The genetic underpinnings of these patterns remain largely unknown. We apply a stochastic multivariate factorization method to a diverse population of 50,699 individuals (12 studies, 130 sites) and derive data-driven, multi-scale PSCs of regional brain size. PSCs were significantly correlated with 915 genomic loci in the discovery set, 617 of which are novel, and 72% were independently replicated. Key pathways influencing PSCs involve reelin signaling, apoptosis, neurogenesis, and appendage development, while pathways of breast cancer indicate potential interplays between brain metastasis and PSCs associated with neurodegeneration and dementia. Using support vector machines, multi-scale PSCs effectively derive imaging signatures of several brain diseases. Our results elucidate new genetic and biological underpinnings that influence structural covariance patterns in the human brain.

Significance statementThe coordinated patterns of changes in the human brain throughout life, driven by brain development, aging, and diseases, remain largely unexplored regarding their underlying genetic determinants. This study delineates 2003 multi-scale patterns of structural covariance (PSCs) and identifies 617 novel genomic loci, with the mapped genes enriched in biological pathways implicated in reelin signaling, apoptosis, neurogenesis, and appendage development. Overall, the 2003 PSCs provide new genetic insights into understanding human brain morphological changes and demonstrate great potential in predicting various neurologic conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.23292162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.23292162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.23292162">
        <p class="paperTitle">POPULATION FREQUENCY OF REPEAT EXPANSIONS INDICATES INCREASED DISEASE PREVALENCE ESTIMATES ACROSS DIFFERENT POPULATIONS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.23292162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.23292162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ibanez Garikano, K.; Jadhav, B.; Facchini, S.; Garg, P.; Zanovello, M.; Martin Trujillo, A.; Gies, S.; Galassi Deforie, V.; Gagliardi, D.; Hensman Moss, D.; Moutsianas, L.; EUROSCA, n.; Genomics England Research Consortium,  ; Shoai, M.; Caulifield, M. J.; Cortese, A.; Escott Price, V.; Hardy, J.; Houlden, H.; Sharp, A. J.; Tucci, A.</p>
        <p class="info">Score: 17.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.23292162' target='https://doi.org/10.1101/2023.07.03.23292162'> 10.1101/2023.07.03.23292162</a></p>
        <p class="abstract">Repeat expansion disorders (REDs) are a devastating group of predominantly neurological diseases. Together they are common, affecting 1 in 3,000 people worldwide with population-specific differences. However, prevalence estimates of REDs are hampered by heterogeneous clinical presentation, variable geographic distributions, and technological limitations leading to under-ascertainment. Here, leveraging whole genome sequencing data from 82,176 individuals from different populations we found an overall carrier frequency of REDs of 1 in 340 individuals. Modelling disease prevalence using genetic data, age at onset and survival, we show that REDs are up to 3-fold more prevalent than currently reported figures. While some REDs are population-specific, e.g. Huntingtons disease type 2, most REDs are represented in all broad genetic ancestries, including Africans and Asians, challenging the notion that some REDs are found only in European populations. These results have worldwide implications for local and global health communities in the diagnosis and management of REDs both at local and global levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.23292410">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.23292410" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.23292410">
        <p class="paperTitle">Multi-Omic Graph Diagnosis (MOGDx) : A data integration tool to perform classification tasks for heterogeneous diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.23292410" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.23292410" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ryan, B.; Marioni, R. E.; Simpson, T. I.</p>
        <p class="info">Score: 2.8, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.23292410' target='https://doi.org/10.1101/2023.07.09.23292410'> 10.1101/2023.07.09.23292410</a></p>
        <p class="abstract">Heterogeneity in human diseases presents challenges in diagnosis and treatments due to the broad range of manifestations and symptoms. With the rapid development of labelled multi-omic data, integrative machine learning methods have achieved breakthroughs in treatments by redefining these diseases at a more granular level. These approaches often have limitations in scalability, oversimplification, and handling of missing data. In this study, we introduce Multi-Omic Graph Diagnosis (MOGDx), a flexible command line tool for the integration of multi-omic data to perform classification tasks for heterogeneous diseases. MOGDx is a network integrative method that combines patient similarity networks with a reduced vector representation of genomic data. The reduced vector is derived from the latent embeddings of an auto-encoder and the combined network is fed into a graph convolutional network for classification. MOGDx was evaluated on three datasets from the cancer genome atlas for breast invasive carcinoma, kidney cancer, and low grade glioma. MOGDx demonstrated state-of-the-art performance and an ability to identify relevant multi-omic markers in each task. It did so while integrating more genomic measures with greater patient coverage compared to other network integrative methods. MOGDx is available to download from https://github.com/biomedicalinformaticsgroup/MOGDx. Overall, MOGDx is a promising tool for integrating multi-omic data, classifying heterogeneous diseases, and interpreting genomic markers.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
