<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 236.5, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nRNA sequencing uplifts diagnostic rate in undiagnosed rare disease patients\nAuthors: Jaramillo Oquendo, C.; Wai, H. A.; Rich, W.; Bunyan, D. J.; Thomas, N. S.; Hunt, D.; Lord, J.; Douglas, A. G. L.; Baralle, D.\nScore: 20.8, Published: 2023-07-08 DOI: 10.1101/2023.07.05.23292254\nRNA-sequencing is increasingly being used as a complementary tool to DNA sequencing in diagnostics where DNA analysis has been uninformative as it enables the identification of alternative splicing (AS), aberrant gene expression and allele specific expression. Using RNA from patient blood, we have used both RNA-sequencing and RT-PCR to detect splicing and gene expression outliers in a heterogenous cohort of 87 patients with suspected Mendelian disorders, 38% of which did not have a candidate sequence variant. Expression outliers were detected using OUTRIDER, and we compared the performance of multiple open-source alternative splicing tools (MAJIQ, rMATS-turbo, and LeafCutterMD) in identifying alternative splicing events. As well as clarifying the impact of variants of uncertain significance (VUSs), we trialled two novel approaches to identify new potential diagnoses in patients with no candidate variants. We were able to assess 85% of VUSs and validate splicing abnormalities in 18/48 patients with a VUS. Furthermore, we identified four new diagnoses by detecting novel AS events in patients with no candidate sequence variants from prior genomic DNA testing (n=33) or those in which the candidate VUS did not affect splicing (n=23) and identified one additional diagnosis through detection of skewed X-inactivation. These results gave an overall uplift in diagnostic yield of 26%. In this work we demonstrate the utility of blood-based RNA analysis in improving diagnostic yields and highlight optimal approaches for such analysis.\nMulti-ancestry genome-wide meta-analysis of 56,241 individuals identifies LRRC4C, LHX5-AS1 and nominates ancestry-specific loci PTPRK, GRB14, and KIAA0825 as novel risk loci for Alzheimer disease: the Alzheimer Disease Genetics Consortium\nAuthors: Rajabli, F.; Benchek, P.; Tosto, G.; Kushch, N.; Sha, J.; Bazemore, K.; Zhu, C.; Lee, W.-P.; Haut, J.; Hamilton-Nelson, K. L.; Wheeler, N. R.; Zhao, Y.; Farrell, J. J.; Grunin, M. A.; Leung, Y. Y.; Kuksa, P. P.; Li, D.; Lucio da Fonseca, E.; Mez, J. B.; Palmer, E. L.; Pillai, J.; Sherva, R. M.; Song, Y. E.; Zhang, X.; Iqbal, T.; Pathak, O.; Valladares, O.; Kuzma, A. B.; Abner, E.; Adams, P. M.; Aguirre, A.; Albert, M. S.; Albin, R. L.; Allen, M.; Alvarez, L.; Apostolova, L. G.; Arnold, S. E.; Asthana, S.; Atwood, C. S.; Ayres, G.; Baldwin, C. T.; Barber, R. C.; Barnes, L. L.; Barral, S.; Beach\nScore: 20.1, Published: 2023-07-08 DOI: 10.1101/2023.07.06.23292311\nLimited ancestral diversity has impaired our ability to detect risk variants more prevalent in non-European ancestry groups in genome-wide association studies (GWAS). We constructed and analyzed a multi-ancestry GWAS dataset in the Alzheimers Disease (AD) Genetics Consortium (ADGC) to test for novel shared and ancestry-specific AD susceptibility loci and evaluate underlying genetic architecture in 37,382 non-Hispanic White (NHW), 6,728 African American, 8,899 Hispanic (HIS), and 3,232 East Asian individuals, performing within-ancestry fixed-effects meta-analysis followed by a cross-ancestry random-effects meta-analysis. We identified 13 loci with cross-ancestry associations including known loci at/near CR1, BIN1, TREM2, CD2AP, PTK2B, CLU, SHARPIN, MS4A6A, PICALM, ABCA7, APOE and two novel loci not previously reported at 11p12 (LRRC4C) and 12q24.13 (LHX5-AS1). Reflecting the power of diverse ancestry in GWAS, we observed the SHARPIN locus using 7.1% the sample size of the original discovering single-ancestry GWAS (n=788,989). We additionally identified three GWS ancestry-specific loci at/near (PTPRK (P=2.4x10-8) and GRB14 (P=1.7x10-8) in HIS), and KIAA0825 (P=2.9x10-8 in NHW). Pathway analysis implicated multiple amyloid regulation pathways (strongest with Padjusted=1.6x10-4) and the classical complement pathway (Padjusted=1.3x10-3). Genes at/near our novel loci have known roles in neuronal development (LRRC4C, LHX5-AS1, and PTPRK) and insulin receptor activity regulation (GRB14). These findings provide compelling support for using traditionally-underrepresented populations for gene discovery, even with smaller sample sizes.\nPOPULATION FREQUENCY OF REPEAT EXPANSIONS INDICATES INCREASED DISEASE PREVALENCE ESTIMATES ACROSS DIFFERENT POPULATIONS\nAuthors: Ibanez Garikano, K.; Jadhav, B.; Facchini, S.; Garg, P.; Zanovello, M.; Martin Trujillo, A.; Gies, S.; Galassi Deforie, V.; Gagliardi, D.; Hensman Moss, D.; Moutsianas, L.; EUROSCA, n.; Genomics England Research Consortium, ; Shoai, M.; Caulifield, M. J.; Cortese, A.; Escott Price, V.; Hardy, J.; Houlden, H.; Sharp, A. J.; Tucci, A.\nScore: 16.6, Published: 2023-07-06 DOI: 10.1101/2023.07.03.23292162\nRepeat expansion disorders (REDs) are a devastating group of predominantly neurological diseases. Together they are common, affecting 1 in 3,000 people worldwide with population-specific differences. However, prevalence estimates of REDs are hampered by heterogeneous clinical presentation, variable geographic distributions, and technological limitations leading to under-ascertainment. Here, leveraging whole genome sequencing data from 82,176 individuals from different populations we found an overall carrier frequency of REDs of 1 in 340 individuals. Modelling disease prevalence using genetic data, age at onset and survival, we show that REDs are up to 3-fold more prevalent than currently reported figures. While some REDs are population-specific, e.g. Huntingtons disease type 2, most REDs are represented in all broad genetic ancestries, including Africans and Asians, challenging the notion that some REDs are found only in European populations. These results have worldwide implications for local and global health communities in the diagnosis and management of REDs both at local and global levels.\nExome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank\nAuthors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.\nScore: 21.5, Published: 2023-07-03 DOI: 10.1101/2023.06.29.23291992\nExome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF \u003c 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( \u003c 1.68 x 10-7) and 17 nominally significant ( \u003c 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR \u003c 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.\nNon-coding variants are a rare cause of recessive developmental disorders in trans with coding variants\nAuthors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium, ; Baralle, D.; Martin, H. C.; Whiffin, N.\nScore: 32.5, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291805\nPurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown. MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing. ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA). ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.\nThe NYCKidSeq randomized controlled trial: Impact of GUIA digitally enhanced genetic counseling in racially and ethnically diverse families\nAuthors: Suckiel, S. A.; Kelly, N. R.; Odgis, J. A.; Gallagher, K. M.; Sebastin, M.; Bonini, K. E.; Marathe, P. N.; Brown, K.; Di Biase, M.; Ramos, M.; Rodriguez, J. E.; Scarimbolo, L.; Insel, B. J.; Ferar, K. D. M.; Zinberg, R. E.; Diaz, G. A.; Greally, J. M.; Abul-Husn, N. S.; Bauman, L. J.; Gelb, B. D.; Horowitz, C. R.; Wasserstein, M. P.; Kenny, E. E.\nScore: 7.7, Published: 2023-07-07 DOI: 10.1101/2023.07.05.23292193\nBackgroundDigital solutions are needed to support rapid increases in the application of genetic and genomic tests (GT) in diverse clinical settings and patient populations. We developed GUIA, a bi-lingual web-based platform that facilitates disclosure of GT results. The NYCKidSeq randomized controlled trial evaluated GUIAs impact on understanding of GT results. MethodsNYCKidSeq enrolled diverse children with neurologic, cardiac, and immunologic conditions who underwent GT. Families were randomized to genetic counseling with GUIA (intervention) or standard of care (SOC) genetic counseling for results disclosure. Parents/legal guardians (participants) completed surveys at baseline, post-results disclosure, and 6-months later. Survey measures assessed the primary study outcomes of perceived understanding of and confidence in explaining their childs GT results and the secondary outcome of objective understanding. We used regression models to evaluate the association between the intervention and the study outcomes. ResultsThe analysis included 551 participants, 270 in the GUIA arm and 281 in SOC. Participants mean age was 41.1 years and 88.6% were mothers. Most participants were Hispanic/Latino(a) (46.3%), White/European American (24.5%), or Black/African American (15.8%). Participants in the GUIA arm had significantly higher perceived understanding post-results (OR=2.8, CI[1.004,7.617], P=0.049) and maintained higher objective understanding over time (OR=1.1, CI[1.004, 1.127], P=0.038) compared to those in the SOC arm. There was no impact on perceived confidence. Hispanic/Latino(a) individuals in the GUIA arm maintained higher perceived understanding (OR=3.9, CI[1.6, 9.3], P=0.003), confidence (OR=2.7, CI[1.021, 7.277], P=0.046), and objective understanding (OR=1.1, CI[1.009, 1.212], P=0.032) compared to SOC. ConclusionsThis trial demonstrates that GUIA positively impacts understanding of GT results in diverse parents of children with suspected genetic conditions. These findings build a case for utilizing GUIA to deliver complex and often ambiguous genetic results. Continued development and evaluation of digital applications in diverse populations are critical for equitably scaling GT offerings in specialty clinics.\nGenome-wide meta-analysis of CSF biomarkers in Alzheimer's disease and Parkinson's disease cohorts\nAuthors: Ta, M.; Blauwendraat, C.; Antar, T.; Leonard, H. L.; Singleton, A. B.; Nalls, M.; Iwaki, H.; Alzheimers Disease Neuroimaging Initiative (ADNI), ; the Fox Investigation for New Discovery of Biomarkers, Score: 9.2, Published: 2023-06-19 DOI: 10.1101/2023.06.13.23291354\nBackgroundAmyloid beta (A{beta}), phosphorylated tau (p-tau), and total tau (t-tau) in cerebrospinal fluid are established biomarkers for Alzheimers disease (AD). In other neurodegenerative diseases, such as Parkinsons disease (PD), these biomarkers have also been found to be altered, and the molecular mechanisms responsible for these alterations are still under investigation. Moreover, the interplay between these mechanisms and the diverse underlying disease states remains to be elucidated. ObjectivesTo investigate genetic contributions to the AD biomarkers and assess the commonality and heterogeneity of the associations per underlying disease status. MethodsWe conducted GWAS for the AD biomarkers on subjects from the Parkinsons Progression Markers Initiative (PPMI), the Fox Investigation for New Discovery of Biomarkers (BioFIND), and the Alzheimers Disease Neuroimaging Initiative (ADNI) and meta-analyzed with the largest AD GWAS.[7] We tested heterogeneity of associations of interest between different disease statuses (AD, PD, and control). ResultsWe observed three GWAS signals: the APOE locus for A{beta}, the 3q28 locus between GEMC1 and OSTN for p-tau and t-tau, and the 7p22 locus (top hit: rs60871478, an intronic variant for DNAAF5, also known as HEATR2) for p-tau. The 7p22 locus is novel and co-localized with the brain DNAAF5 expression. While no heterogeneity from underlying disease status was observed for the above GWAS signals, some disease risk loci suggested disease specific associations with these biomarkers. ConclusionsOur study identified a novel association at the intronic region of DNAAF5 associated with increased levels of p-tau across all diseases. We also observed some disease specific genetic associations with these biomarkers.\nA systematic assessment of the impact of rare canonical splice site variants on splicing using functional and in silico methods\nAuthors: Oh, R. Y.; AlMail, A.; Cheerie, D.; Guirguis, G.; Hou, H.; Yuki, K. E.; Haque, B.; Thiruvahindrapuram, B.; Marshall, C. R.; Mendoza-Londono, R.; Shlien, A.; Kyriakopoulou, L. G.; Walker, S.; Dowling, J. J.; Wilson, M. D.; Costain, G.\nScore: 5.8, Published: 2023-07-06 DOI: 10.1101/2023.06.29.23292012\nBackground/ObjectivesCanonical splice site variants (CSSVs) are often presumed to cause loss-of-function (LoF) and are assigned very strong evidence of pathogenicity (according to ACMG criterion PVS1). However, the exact nature and predictability of splicing effects of unselected rare CSSVs in blood-expressed genes is poorly understood. MethodsA total of 184 rare CSSVs in unselected blood-expressed genes were identified by genome sequencing in 121 individuals, and their impact on splicing was interrogated manually in RNA sequencing (RNA-seq) data. Blind to these RNA-seq data, we attempted to predict the precise impact of CSSVs by applying in silico tools and the ClinGen Sequence Variant Interpretation Working Group 2018 guidelines for applying PVS1 criterion. ResultsThere was no evidence of a frameshift nor of reduced expression consistent with nonsense-mediated decay (NMD) for 24% of CSSVs: 17% had wildtype splicing only and normal junction depths, 3.25% resulted in cryptic splice site usage and in-frame indels, 3.25% resulted in full exon skipping (in-frame), and 0.5% resulted in full intron inclusion (in-frame). Misclassification rates for splicing outcome (frameshift/NMD vs. no frameshift/no NMD) using (i) SpliceAI, (ii) MaxEntScan, and (iii) AutoPVS1 ranged from 30-41%, with none outperforming a simple \"zero rule\" classifier. ConclusionNearly 1 in 4 CSSVs may not cause LoF based on analysis of RNA-seq data. Predictions from in silico methods were often discordant with findings from RNA-seq. More caution may be warranted in applying PVS1-level evidence to CSSVs in the absence of functional data.\nMeta-analysis of CYP2C19 and CYP2D6 metabolic activity on antidepressant response from 13 clinical studies\nAuthors: Li, D.; Pain, O.; Fabbri, C.; Wong, W. L. E.; Lo, C. W. H.; Ripke, S.; Cattaneo, A.; Souery, D.; Dernovsek, M. Z.; Henigsberg, N.; Hauser, J.; Lewis, G.; Mors, O.; Perroud, N.; Rietschel, M.; Uher, R.; Maier, W.; Aitchison, K. J.; Baune, B. T.; Biernacka, J. M.; Bondolfi, G.; Domschke, K.; Kato, M.; Liu, Y.-L.; Serretti, A.; Tsai, S.-J.; Weinshilboum, R.; the GSRD Consortium, ; the Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, ; McIntosh, A. M.; Lewis, C. M.\nScore: 4.3, Published: 2023-06-28 DOI: 10.1101/2023.06.26.23291890\nCytochrome P450 enzymes including CYP2C19 and CYP2D6 are important for antidepressant metabolism and polymorphisms of these genes have been determined to predict metabolite levels. Nonetheless, more evidence is needed to understand the impact of genetic variations on antidepressant response. In this study, individual data from 13 clinical studies of European and East Asian ancestry populations were collected. The antidepressant response was clinically assessed as remission and percentage improvement. Imputed genotype was used to translate genetic polymorphisms to four metabolic phenotypes (poor, intermediate, normal, and ultrarapid) of CYP2C19 and CYP2D6. The association of CYP2C19 and CYP2D6 metabolic phenotypes with treatment response was examined using normal metabolizers as the reference. Among 5843 depression patients, a higher remission rate was found in CYP2C19 poor metabolizers compared to normal metabolizers at nominal significance (OR = 1.46, 95% CI [1.03, 2.06], p = 0.033) but did not survive after multiple testing correction. No metabolic phenotype was associated with percentage improvement from baseline. After stratifying by antidepressants primarily metabolized by CYP2C19 and CYP2D6, no association was found between metabolic phenotypes and antidepressant response. Metabolic phenotypes showed differences in frequency, but not effect, between European and East Asian studies. In conclusion, metabolic phenotypes imputed from genetic variants were not associated with antidepressant response. CYP2C19 poor metabolizers could potentially contribute to antidepressant efficacy with more evidence needed. Information including side effects, antidepressant dosage, as well as population from different ancestries could be involved to fully capture the influence of metabolic phenotypes and improve the power of effect assessment.\n",
  "wordCount" : "3036",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 236.5, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292254" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292254">
        <p class="paperTitle">RNA sequencing uplifts diagnostic rate in undiagnosed rare disease patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jaramillo Oquendo, C.; Wai, H. A.; Rich, W.; Bunyan, D. J.; Thomas, N. S.; Hunt, D.; Lord, J.; Douglas, A. G. L.; Baralle, D.</p>
        <p class="info">Score: 20.8, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292254' target='https://doi.org/10.1101/2023.07.05.23292254'> 10.1101/2023.07.05.23292254</a></p>
        <p class="abstract">RNA-sequencing is increasingly being used as a complementary tool to DNA sequencing in diagnostics where DNA analysis has been uninformative as it enables the identification of alternative splicing (AS), aberrant gene expression and allele specific expression. Using RNA from patient blood, we have used both RNA-sequencing and RT-PCR to detect splicing and gene expression outliers in a heterogenous cohort of 87 patients with suspected Mendelian disorders, 38% of which did not have a candidate sequence variant. Expression outliers were detected using OUTRIDER, and we compared the performance of multiple open-source alternative splicing tools (MAJIQ, rMATS-turbo, and LeafCutterMD) in identifying alternative splicing events. As well as clarifying the impact of variants of uncertain significance (VUSs), we trialled two novel approaches to identify new potential diagnoses in patients with no candidate variants. We were able to assess 85% of VUSs and validate splicing abnormalities in 18/48 patients with a VUS. Furthermore, we identified four new diagnoses by detecting novel AS events in patients with no candidate sequence variants from prior genomic DNA testing (n=33) or those in which the candidate VUS did not affect splicing (n=23) and identified one additional diagnosis through detection of skewed X-inactivation. These results gave an overall uplift in diagnostic yield of 26%. In this work we demonstrate the utility of blood-based RNA analysis in improving diagnostic yields and highlight optimal approaches for such analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.23292311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.23292311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.23292311">
        <p class="paperTitle">Multi-ancestry genome-wide meta-analysis of 56,241 individuals identifies LRRC4C, LHX5-AS1 and nominates ancestry-specific loci PTPRK, GRB14, and KIAA0825 as novel risk loci for Alzheimer disease: the Alzheimer Disease Genetics Consortium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.23292311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.23292311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rajabli, F.; Benchek, P.; Tosto, G.; Kushch, N.; Sha, J.; Bazemore, K.; Zhu, C.; Lee, W.-P.; Haut, J.; Hamilton-Nelson, K. L.; Wheeler, N. R.; Zhao, Y.; Farrell, J. J.; Grunin, M. A.; Leung, Y. Y.; Kuksa, P. P.; Li, D.; Lucio da Fonseca, E.; Mez, J. B.; Palmer, E. L.; Pillai, J.; Sherva, R. M.; Song, Y. E.; Zhang, X.; Iqbal, T.; Pathak, O.; Valladares, O.; Kuzma, A. B.; Abner, E.; Adams, P. M.; Aguirre, A.; Albert, M. S.; Albin, R. L.; Allen, M.; Alvarez, L.; Apostolova, L. G.; Arnold, S. E.; Asthana, S.; Atwood, C. S.; Ayres, G.; Baldwin, C. T.; Barber, R. C.; Barnes, L. L.; Barral, S.; Beach</p>
        <p class="info">Score: 20.1, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.23292311' target='https://doi.org/10.1101/2023.07.06.23292311'> 10.1101/2023.07.06.23292311</a></p>
        <p class="abstract">Limited ancestral diversity has impaired our ability to detect risk variants more prevalent in non-European ancestry groups in genome-wide association studies (GWAS). We constructed and analyzed a multi-ancestry GWAS dataset in the Alzheimers Disease (AD) Genetics Consortium (ADGC) to test for novel shared and ancestry-specific AD susceptibility loci and evaluate underlying genetic architecture in 37,382 non-Hispanic White (NHW), 6,728 African American, 8,899 Hispanic (HIS), and 3,232 East Asian individuals, performing within-ancestry fixed-effects meta-analysis followed by a cross-ancestry random-effects meta-analysis. We identified 13 loci with cross-ancestry associations including known loci at/near CR1, BIN1, TREM2, CD2AP, PTK2B, CLU, SHARPIN, MS4A6A, PICALM, ABCA7, APOE and two novel loci not previously reported at 11p12 (LRRC4C) and 12q24.13 (LHX5-AS1). Reflecting the power of diverse ancestry in GWAS, we observed the SHARPIN locus using 7.1% the sample size of the original discovering single-ancestry GWAS (n=788,989). We additionally identified three GWS ancestry-specific loci at/near (PTPRK (P=2.4x10-8) and GRB14 (P=1.7x10-8) in HIS), and KIAA0825 (P=2.9x10-8 in NHW). Pathway analysis implicated multiple amyloid regulation pathways (strongest with Padjusted=1.6x10-4) and the classical complement pathway (Padjusted=1.3x10-3). Genes at/near our novel loci have known roles in neuronal development (LRRC4C, LHX5-AS1, and PTPRK) and insulin receptor activity regulation (GRB14). These findings provide compelling support for using traditionally-underrepresented populations for gene discovery, even with smaller sample sizes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.23292162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.23292162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.23292162">
        <p class="paperTitle">POPULATION FREQUENCY OF REPEAT EXPANSIONS INDICATES INCREASED DISEASE PREVALENCE ESTIMATES ACROSS DIFFERENT POPULATIONS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.23292162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.23292162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ibanez Garikano, K.; Jadhav, B.; Facchini, S.; Garg, P.; Zanovello, M.; Martin Trujillo, A.; Gies, S.; Galassi Deforie, V.; Gagliardi, D.; Hensman Moss, D.; Moutsianas, L.; EUROSCA, n.; Genomics England Research Consortium,  ; Shoai, M.; Caulifield, M. J.; Cortese, A.; Escott Price, V.; Hardy, J.; Houlden, H.; Sharp, A. J.; Tucci, A.</p>
        <p class="info">Score: 16.6, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.23292162' target='https://doi.org/10.1101/2023.07.03.23292162'> 10.1101/2023.07.03.23292162</a></p>
        <p class="abstract">Repeat expansion disorders (REDs) are a devastating group of predominantly neurological diseases. Together they are common, affecting 1 in 3,000 people worldwide with population-specific differences. However, prevalence estimates of REDs are hampered by heterogeneous clinical presentation, variable geographic distributions, and technological limitations leading to under-ascertainment. Here, leveraging whole genome sequencing data from 82,176 individuals from different populations we found an overall carrier frequency of REDs of 1 in 340 individuals. Modelling disease prevalence using genetic data, age at onset and survival, we show that REDs are up to 3-fold more prevalent than currently reported figures. While some REDs are population-specific, e.g. Huntingtons disease type 2, most REDs are represented in all broad genetic ancestries, including Africans and Asians, challenging the notion that some REDs are found only in European populations. These results have worldwide implications for local and global health communities in the diagnosis and management of REDs both at local and global levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23291992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23291992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23291992">
        <p class="paperTitle">Exome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23291992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23291992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.</p>
        <p class="info">Score: 21.5, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23291992' target='https://doi.org/10.1101/2023.06.29.23291992'> 10.1101/2023.06.29.23291992</a></p>
        <p class="abstract">Exome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF [~] 0.001%) in exome sequencing data from 175,587 UK Biobank (UKBB) participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF &lt; 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( &lt; 1.68 x 10-7) and 17 nominally significant ( &lt; 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR &lt; 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291805">
        <p class="paperTitle">Non-coding variants are a rare cause of recessive developmental disorders in trans with coding variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium,  ; Baralle, D.; Martin, H. C.; Whiffin, N.</p>
        <p class="info">Score: 32.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291805' target='https://doi.org/10.1101/2023.06.23.23291805'> 10.1101/2023.06.23.23291805</a></p>
        <p class="abstract">PurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown.

MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing.

ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA).

ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.23292193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.23292193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.23292193">
        <p class="paperTitle">The NYCKidSeq randomized controlled trial: Impact of GUIA digitally enhanced genetic counseling in racially and ethnically diverse families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.23292193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.23292193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suckiel, S. A.; Kelly, N. R.; Odgis, J. A.; Gallagher, K. M.; Sebastin, M.; Bonini, K. E.; Marathe, P. N.; Brown, K.; Di Biase, M.; Ramos, M.; Rodriguez, J. E.; Scarimbolo, L.; Insel, B. J.; Ferar, K. D. M.; Zinberg, R. E.; Diaz, G. A.; Greally, J. M.; Abul-Husn, N. S.; Bauman, L. J.; Gelb, B. D.; Horowitz, C. R.; Wasserstein, M. P.; Kenny, E. E.</p>
        <p class="info">Score: 7.7, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.23292193' target='https://doi.org/10.1101/2023.07.05.23292193'> 10.1101/2023.07.05.23292193</a></p>
        <p class="abstract">BackgroundDigital solutions are needed to support rapid increases in the application of genetic and genomic tests (GT) in diverse clinical settings and patient populations. We developed GUIA, a bi-lingual web-based platform that facilitates disclosure of GT results. The NYCKidSeq randomized controlled trial evaluated GUIAs impact on understanding of GT results.

MethodsNYCKidSeq enrolled diverse children with neurologic, cardiac, and immunologic conditions who underwent GT. Families were randomized to genetic counseling with GUIA (intervention) or standard of care (SOC) genetic counseling for results disclosure. Parents/legal guardians (participants) completed surveys at baseline, post-results disclosure, and 6-months later. Survey measures assessed the primary study outcomes of perceived understanding of and confidence in explaining their childs GT results and the secondary outcome of objective understanding. We used regression models to evaluate the association between the intervention and the study outcomes.

ResultsThe analysis included 551 participants, 270 in the GUIA arm and 281 in SOC. Participants mean age was 41.1 years and 88.6% were mothers. Most participants were Hispanic/Latino(a) (46.3%), White/European American (24.5%), or Black/African American (15.8%). Participants in the GUIA arm had significantly higher perceived understanding post-results (OR=2.8, CI[1.004,7.617], P=0.049) and maintained higher objective understanding over time (OR=1.1, CI[1.004, 1.127], P=0.038) compared to those in the SOC arm. There was no impact on perceived confidence. Hispanic/Latino(a) individuals in the GUIA arm maintained higher perceived understanding (OR=3.9, CI[1.6, 9.3], P=0.003), confidence (OR=2.7, CI[1.021, 7.277], P=0.046), and objective understanding (OR=1.1, CI[1.009, 1.212], P=0.032) compared to SOC.

ConclusionsThis trial demonstrates that GUIA positively impacts understanding of GT results in diverse parents of children with suspected genetic conditions. These findings build a case for utilizing GUIA to deliver complex and often ambiguous genetic results. Continued development and evaluation of digital applications in diverse populations are critical for equitably scaling GT offerings in specialty clinics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.23291354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.23291354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.23291354">
        <p class="paperTitle">Genome-wide meta-analysis of CSF biomarkers in Alzheimer&#39;s disease and Parkinson&#39;s disease cohorts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.23291354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.23291354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ta, M.; Blauwendraat, C.; Antar, T.; Leonard, H. L.; Singleton, A. B.; Nalls, M.; Iwaki, H.; Alzheimers Disease Neuroimaging Initiative (ADNI),  ; the Fox Investigation for New Discovery of Biomarkers,  </p>
        <p class="info">Score: 9.2, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.23291354' target='https://doi.org/10.1101/2023.06.13.23291354'> 10.1101/2023.06.13.23291354</a></p>
        <p class="abstract">BackgroundAmyloid beta (A{beta}), phosphorylated tau (p-tau), and total tau (t-tau) in cerebrospinal fluid are established biomarkers for Alzheimers disease (AD). In other neurodegenerative diseases, such as Parkinsons disease (PD), these biomarkers have also been found to be altered, and the molecular mechanisms responsible for these alterations are still under investigation. Moreover, the interplay between these mechanisms and the diverse underlying disease states remains to be elucidated.

ObjectivesTo investigate genetic contributions to the AD biomarkers and assess the commonality and heterogeneity of the associations per underlying disease status.

MethodsWe conducted GWAS for the AD biomarkers on subjects from the Parkinsons Progression Markers Initiative (PPMI), the Fox Investigation for New Discovery of Biomarkers (BioFIND), and the Alzheimers Disease Neuroimaging Initiative (ADNI) and meta-analyzed with the largest AD GWAS.[7] We tested heterogeneity of associations of interest between different disease statuses (AD, PD, and control).

ResultsWe observed three GWAS signals: the APOE locus for A{beta}, the 3q28 locus between GEMC1 and OSTN for p-tau and t-tau, and the 7p22 locus (top hit: rs60871478, an intronic variant for DNAAF5, also known as HEATR2) for p-tau. The 7p22 locus is novel and co-localized with the brain DNAAF5 expression. While no heterogeneity from underlying disease status was observed for the above GWAS signals, some disease risk loci suggested disease specific associations with these biomarkers.

ConclusionsOur study identified a novel association at the intronic region of DNAAF5 associated with increased levels of p-tau across all diseases. We also observed some disease specific genetic associations with these biomarkers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292012" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292012">
        <p class="paperTitle">A systematic assessment of the impact of rare canonical splice site variants on splicing using functional and in silico methods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oh, R. Y.; AlMail, A.; Cheerie, D.; Guirguis, G.; Hou, H.; Yuki, K. E.; Haque, B.; Thiruvahindrapuram, B.; Marshall, C. R.; Mendoza-Londono, R.; Shlien, A.; Kyriakopoulou, L. G.; Walker, S.; Dowling, J. J.; Wilson, M. D.; Costain, G.</p>
        <p class="info">Score: 5.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292012' target='https://doi.org/10.1101/2023.06.29.23292012'> 10.1101/2023.06.29.23292012</a></p>
        <p class="abstract">Background/ObjectivesCanonical splice site variants (CSSVs) are often presumed to cause loss-of-function (LoF) and are assigned very strong evidence of pathogenicity (according to ACMG criterion PVS1). However, the exact nature and predictability of splicing effects of unselected rare CSSVs in blood-expressed genes is poorly understood.

MethodsA total of 184 rare CSSVs in unselected blood-expressed genes were identified by genome sequencing in 121 individuals, and their impact on splicing was interrogated manually in RNA sequencing (RNA-seq) data. Blind to these RNA-seq data, we attempted to predict the precise impact of CSSVs by applying in silico tools and the ClinGen Sequence Variant Interpretation Working Group 2018 guidelines for applying PVS1 criterion.

ResultsThere was no evidence of a frameshift nor of reduced expression consistent with nonsense-mediated decay (NMD) for 24% of CSSVs: 17% had wildtype splicing only and normal junction depths, 3.25% resulted in cryptic splice site usage and in-frame indels, 3.25% resulted in full exon skipping (in-frame), and 0.5% resulted in full intron inclusion (in-frame). Misclassification rates for splicing outcome (frameshift/NMD vs. no frameshift/no NMD) using (i) SpliceAI, (ii) MaxEntScan, and (iii) AutoPVS1 ranged from 30-41%, with none outperforming a simple &#34;zero rule&#34; classifier.

ConclusionNearly 1 in 4 CSSVs may not cause LoF based on analysis of RNA-seq data. Predictions from in silico methods were often discordant with findings from RNA-seq. More caution may be warranted in applying PVS1-level evidence to CSSVs in the absence of functional data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.23291890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.23291890" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.23291890">
        <p class="paperTitle">Meta-analysis of CYP2C19 and CYP2D6 metabolic activity on antidepressant response from 13 clinical studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.23291890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.23291890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, D.; Pain, O.; Fabbri, C.; Wong, W. L. E.; Lo, C. W. H.; Ripke, S.; Cattaneo, A.; Souery, D.; Dernovsek, M. Z.; Henigsberg, N.; Hauser, J.; Lewis, G.; Mors, O.; Perroud, N.; Rietschel, M.; Uher, R.; Maier, W.; Aitchison, K. J.; Baune, B. T.; Biernacka, J. M.; Bondolfi, G.; Domschke, K.; Kato, M.; Liu, Y.-L.; Serretti, A.; Tsai, S.-J.; Weinshilboum, R.; the GSRD Consortium,  ; the Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium,  ; McIntosh, A. M.; Lewis, C. M.</p>
        <p class="info">Score: 4.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.23291890' target='https://doi.org/10.1101/2023.06.26.23291890'> 10.1101/2023.06.26.23291890</a></p>
        <p class="abstract">Cytochrome P450 enzymes including CYP2C19 and CYP2D6 are important for antidepressant metabolism and polymorphisms of these genes have been determined to predict metabolite levels. Nonetheless, more evidence is needed to understand the impact of genetic variations on antidepressant response. In this study, individual data from 13 clinical studies of European and East Asian ancestry populations were collected. The antidepressant response was clinically assessed as remission and percentage improvement. Imputed genotype was used to translate genetic polymorphisms to four metabolic phenotypes (poor, intermediate, normal, and ultrarapid) of CYP2C19 and CYP2D6. The association of CYP2C19 and CYP2D6 metabolic phenotypes with treatment response was examined using normal metabolizers as the reference. Among 5843 depression patients, a higher remission rate was found in CYP2C19 poor metabolizers compared to normal metabolizers at nominal significance (OR = 1.46, 95% CI [1.03, 2.06], p = 0.033) but did not survive after multiple testing correction. No metabolic phenotype was associated with percentage improvement from baseline. After stratifying by antidepressants primarily metabolized by CYP2C19 and CYP2D6, no association was found between metabolic phenotypes and antidepressant response. Metabolic phenotypes showed differences in frequency, but not effect, between European and East Asian studies. In conclusion, metabolic phenotypes imputed from genetic variants were not associated with antidepressant response. CYP2C19 poor metabolizers could potentially contribute to antidepressant efficacy with more evidence needed. Information including side effects, antidepressant dosage, as well as population from different ancestries could be involved to fully capture the influence of metabolic phenotypes and improve the power of effect assessment.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
