<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program
Authors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program
Authors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program
Authors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program\nAuthors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.",
  "keywords": [
    
  ],
  "articleBody": " Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program\nAuthors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.; Shi, Y.; Nandi, T. N.; Assimes, T. L.; Brunette, C. A.; Carroll, R. J.; Clifford, R.; Duvall, S.; Gelernter, J.; Hung, A.; Iyengar, S. K.; Joseph, J.; Kember, R.; Kranzler, H.; Levey, D.; Luoh, S.-W.; Merritt, V. C.; Overstreet, C.; Deak, J. D.; Grant, S. F. A.; Poli\nScore: 26.0, Published: 2023-06-29 DOI: 10.1101/2023.06.28.23291975\nGenome-wide association studies (GWAS) have underrepresented individuals from non-European populations, impeding progress in characterizing the genetic architecture and consequences of health and disease traits. To address this, we present a population-stratified phenome-wide GWAS followed by a multi-population meta-analysis for 2,068 traits derived from electronic health records of 635,969 participants in the Million Veteran Program (MVP), a longitudinal cohort study of diverse U.S. Veterans genetically similar to the respective African (121,177), Admixed American (59,048), East Asian (6,702), and European (449,042) superpopulations defined by the 1000 Genomes Project. We identified 38,270 independent variants associating with one or more traits at experiment-wide (P \u003c 4.6x10-11) significance; fine-mapping 6,318 signals identified from 613 traits to single-variant resolution. Among these, a third (2,069) of the associations were found only among participants genetically similar to non-European reference populations, demonstrating the importance of expanding diversity in genetic studies. Our work provides a comprehensive atlas of phenome-wide genetic associations for future studies dissecting the architecture of complex traits in diverse populations.\nNon-coding variants are a rare cause of recessive developmental disorders in trans with coding variants\nAuthors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium, ; Baralle, D.; Martin, H. C.; Whiffin, N.\nScore: 20.7, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291805\nPurpose: Identifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding second hits in trans with these is unknown. Methods: In 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing. Results: We found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA). Conclusion: We developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.\nLoss-of-function variants in CUL3 cause a syndromic neurodevelopmental disorder\nAuthors: Blackburn, P. R.; Ebstein, F.; Hsieh, T.-C.; Motta, M.; Radio, F. C.; Herkert, J. C.; Rinne, T.; Thiffault, I.; Rapp, M.; Alders, M.; Maas, S.; Gerard, B.; Smol, T.; Vincent-Delorme, C.; Cogne, B.; Isidor, B.; Vincent, M.; Bachmann-Gagescu, R.; Rauch, A.; Joset, P.; Battista Ferrero, G.; Ciolfi, A.; Husson, T.; Guerrot, A.-M.; Bacino, C.; Macmurdo, C.; Thompson, S. S.; Rosenfeld, J. A.; Faivre, L.; Tran Mau-Them, F.; Deb, W.; Vignard, V.; Agrawal, P. B.; Madden, J. A.; Goldenberg, A.; Lecoquierre, F.; Zech, M.; Prokisch, H.; Necpal, J.; Jech, R.; Winkelmann, J.; Turcanova Koprusakova, M.; Kons\nScore: 23.1, Published: 2023-06-16 DOI: 10.1101/2023.06.13.23290941\nPurposeDe novo variants in CUL3 (Cullin-3 ubiquitin ligase) have been strongly associated with neurodevelopmental disorders (NDDs), but no large case series have been reported so far. Here we aimed to collect sporadic cases carrying rare variants in CUL3, describe the genotype-phenotype correlation, and investigate the underlying pathogenic mechanism. MethodsGenetic data and detailed clinical records were collected via multi-center collaboration. Dysmorphic facial features were analyzed using GestaltMatcher. Variant effects on CUL3 protein stability were assessed using patient-derived T-cells. ResultsWe assembled a cohort of 35 individuals with heterozygous CUL3 variants presenting a syndromic NDD characterized by intellectual disability with or without autistic features. Of these, 33 have loss-of-function (LoF) and two have missense variants. CUL3 LoF variants in patients may affect protein stability leading to perturbations in protein homeostasis, as evidenced by decreased ubiquitin-protein conjugates in vitro. Specifically, we show that cyclin E1 (CCNE1) and 4E-BP1 (EIF4EBP1), two prominent substrates of CUL3, fail to be targeted for proteasomal degradation in patient-derived cells. ConclusionOur study further refines the clinical and mutational spectrum of CUL3-associated NDDs, expands the spectrum of cullin RING E3 ligase-associated neuropsychiatric disorders, and suggests haploinsufficiency via LoF variants is the predominant pathogenic mechanism.\nCombining a prioritization strategy and functional studies nominates 5'UTR variants underlying inherited retinal disease\nAuthors: Duenas Rey, A.; del Pozo Valero, M.; Bouckaert, M.; Van Den Broeck, F.; Daich Varela, M.; Van Heetvelde, M.; De Bruyne, M.; Van de Sompele, S.; Bauwens, M.; Ellingford, J. M.; Lenaerts, H.; Mahieu, Q.; Josifova, D.; Genomics England Research Consortium, ; Rivolta, C.; Webster, A.; Arno, G.; Ayuso, C.; De Zaeytijd, J.; Leroy, B.; De Baere, E.; Coppieters, F.\nScore: 14.1, Published: 2023-06-27 DOI: 10.1101/2023.06.19.23291376\nBackground5 untranslated regions (5UTRs) are essential modulators of protein translation. Predicting the impact of 5UTR variants is challenging and typically not performed in routine diagnostics. Here, we present a combined approach of a comprehensive prioritization strategy and subsequent functional assays to evaluate 5UTR variation in two large cohorts of patients with inherited retinal diseases (IRDs). MethodsWe performed an isoform-level re-analysis of retinal RNA-seq data to identify the protein-coding transcripts of 378 IRD genes with highest expression in retina. We evaluated the coverage of these 5UTRs by different whole exome sequencing (WES) capture kits. The selected 5UTRs were analyzed in whole genome sequencing (WGS) and WES data from IRD sub-cohorts from the 100,000 Genomes Project (n = 2,417 WGS) and an in-house database (n = 1,682 WES), respectively. Identified variants were annotated for 5UTR-relevant features and classified into 7 distinct categories based on their predicted functional consequence. We developed a variant prioritization strategy by integrating population frequency, specific criteria for each category, and family and phenotypic data. A selection of candidate variants underwent functional validation using diverse experimental approaches. ResultsIsoform-level re-quantification of retinal gene expression revealed 76 IRD genes with a non-canonical retina-enriched isoform, of which 20 display a fully distinct 5UTR compared to that of their canonical isoform. Depending on the probe-design 3-20% of IRD genes have 5UTRs fully captured by WES. After analyzing these regions in both IRD cohorts we prioritized 11 (likely) pathogenic variants in 10 genes (ARL3, MERTK, NDP, NMNAT1, NPHP4, PAX6, PRPF31, PRPF4, RDH12, RD3), of which 8 were novel. Functional analyses further supported the pathogenicity of 2 variants. The MERTK:c.-125G\u003eA variant, overlapping a transcriptional start site, was shown to significantly reduce both luciferase mRNA levels and activity. The RDH12:c.-123C\u003eT variant was found in cis with the reported hypomorphic RDH12:c.701G\u003eA (p.Arg234His) variant in 11 patients. This 5UTR variant, predicted to introduce an upstream open reading frame, was shown to result in reduced RDH12 protein but unaltered mRNA levels. ConclusionsThis study demonstrates the importance of 5UTR variants implicated in IRDs and provides a systematic approach for 5UTR annotation and validation that is applicable to other inherited diseases.\nGenome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 9.9, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections(1,2). Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction(3-5). The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative(6,7) to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity(6), lung function(8), and cancers(9), suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nImproved Type 2 Diabetes Risk Stratification in the Qatar Biobank Cohort by Ensemble Learning Classifier Incorporating Multi-Trait, Population-Specific, Polygenic Risk Scores\nAuthors: Ahmed, I.; Ziab, M.; Mbarek, H.; Taheri, S.; Chagoury, O. L.; Hussain, S. A.; Lakshmi, J.; Bhat, A.; Fakhro, K. A.; Al-Shabeeb AKIL, A. S.\nScore: 9.4, Published: 2023-06-25 DOI: 10.1101/2023.06.23.23291830\nBackgroundType 2 Diabetes (T2D) is a pervasive chronic disease influenced by a complex interplay of environmental and genetic factors. To enhance T2D risk prediction, leveraging genetic information is essential, with polygenic risk scores (PRS) offering a promising tool for assessing individual genetic risk. Our study focuses on the comparison between multi-trait and single-trait PRS models and demonstrates how the incorporation of multi-trait PRS into risk prediction models can significantly augment T2D risk assessment accuracy and effectiveness. MethodsWe conducted genome-wide association studies (GWAS) on 12 distinct T2D-related traits within a cohort of 14,278 individuals, all sequenced under the Qatar Genome Programme (QGP). This in-depth genetic analysis yielded several novel genetic variants associated with T2D, which served as the foundation for constructing multiple weighted PRS models. To assess the cumulative risk from these predictors, we applied machine learning (ML) techniques, which allowed for a thorough risk assessment. ResultsOur research identified genetic variations tied to T2D risk and facilitated the construction of ML models integrating PRS predictors for an exhaustive risk evaluation. The top-performing ML model demonstrated a robust performance with an accuracy of 0.8549, AUC of 0.92, AUC-PR of 0.8522, and an F1 score of 0.757, reflecting its strong capacity to differentiate cases from controls. We are currently working on acquiring independent T2D cohorts to validate the efficacy of our final model. ConclusionOur research underscores the potential of PRS models in identifying individuals within the population who are at elevated risk of developing T2D and its associated complications. The use of multi-trait PRS and ML models for risk prediction could inform early interventions, potentially identifying T2D patients who stand to benefit most based on their individual genetic risk profile. This combined approach signifies a stride forward in the field of precision medicine, potentially enhancing T2D risk prediction, prevention, and management.\nBiallelic loss of function variants in WBP4, encoding a spliceosome protein, result in a variable neurodevelopmental delay syndrome\nAuthors: Engal, E.; Oja, K. T.; Maroofian, R.; Geminder, O.; Le, T.-L.; Mor, E.; Tzvi, N.; Elefant, N.; Zaki, M. S.; Gleeson, J. G.; Muru, K.; Pajusalu, S.; Wojcik, M. H.; Pachat, D.; Abd Elmaksoud, M.; Jeong, W. C.; Lee, H.; Bauer, P.; Zifarelli, G.; Houlden, H.; Elpeleg, O.; Gordon, C.; Harel, T.; Ounap, K.; Salton, M.; Mor-Shaked, H.\nScore: 9.1, Published: 2023-06-27 DOI: 10.1101/2023.06.19.23291425\nOver two dozen spliceosome proteins are involved in human diseases, also referred to as spliceosomopathies. WBP4 (WW Domain Binding Protein 4) is part of the early spliceosomal complex, and was not described before in the context of human pathologies. Ascertained through GeneMatcher we identified eleven patients from eight families, with a severe neurodevelopmental syndrome with variable manifestations. Clinical manifestations included hypotonia, global developmental delay, severe intellectual disability, brain abnormalities, musculoskeletal and gastrointestinal abnormalities. Genetic analysis revealed overall five different homozygous loss-of-function variants in WBP4. Immunoblotting on fibroblasts from two affected individuals with different genetic variants demonstrated complete loss of protein, and RNA sequencing analysis uncovered shared abnormal splicing patterns, including enrichment for abnormalities of the nervous system and musculoskeletal system genes, suggesting that the overlapping differentially spliced genes are related to the common phenotypes of the probands. We conclude that biallelic variants in WBP4 cause a spliceosomopathy. Further functional studies are called for better understanding of the mechanism of pathogenicity.\nPA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder\nAuthors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium, ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.\nScore: 6.2, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291611\nThe state-of-the-field in complex disorder genetics is marked by large-scale, biobank-ascertained data sets that integrate broad demographic, health, survey, and genetic data. Leveraging the richness of this data can advance our understanding of complex disorders by improving predictions, describing etiology, and augmenting gene-mapping. These themes are especially, but not exclusively, relevant to Major Depressive Disorder (MDD), a leading cause of disability for which genetic predictors underperform, clinical heterogeneity is etiologically enigmatic, and missing heritability persists. The iPSYCH 2015 case-cohort presents a unique opportunity to integrate genotypes, register-based clinical outcomes, and extended genealogies of 30,949 MDD cases and 39,655 random population controls. To make full use of our data, we introduce the Pearson-Aitken framework for Family Genetic Risk Scores (PA-FGRS) to estimate individuals' liabilities for major depressive disorder from extended genealogies of partially observed relatives. PA-FGRS extends previous methods by accounting for censoring, leveraging distant relatives, and utilizing a flexible, model-based approach amenable to analysis and extension, advantages we highlight in simulations. Combining PA-FGRS with genotype data improves classification, replicates and extends known genetic contributions to clinical heterogeneity, and increases power for genome-wide association studies. This study combines novel analyses of unique data and can serve as a model for studies of other outcomes.\nProtein-truncating variants in BSN are associated with severe adult-onset obesity, type 2 diabetes and fatty liver disease\nAuthors: Zhao, Y.; Chukanova, M.; Kentistou, K. A.; Fairhurst-Hunter, Z.; Siegert, A. M.; Jia, R.; Dowsett, G.; Gardner, E. J.; Day, F. R.; Kaisinger, L. R.; Tung, Y.-C. L.; Lam, B. Y. H.; Chen, H.-J. C.; Wang, Q.; Berumen-Campos, J.; Kuri-Morales, P.; Tapia-Conyer, R.; Alegre-Diaz, J.; Emberson, J.; Torres, J. M.; Collins, R.; Saleheen, D.; Smith, K. R.; Paul, D. R.; Merkle, F.; Wareham, N. J.; Petrovski, S.; O'Rahilly, S.; Ong, K. K.; Yeo, G. S. H.; Perry, J. R. B.\nScore: 9.1, Published: 2023-06-16 DOI: 10.1101/2023.06.14.23291368\nObesity is a major risk factor for many common diseases and has a significant heritable component. While clinical and large-scale population studies have identified several genes harbouring rare alleles with large effects on obesity risk, there are likely many unknown genes with highly penetrant effects remaining. To this end, we performed whole exome-sequence analyses for adult body mass index (BMI) in up to 587,027 individuals. We identified rare, loss of function variants in two genes - BSN and APBA1 - with effects on BMI substantially larger than well-established obesity genes such as MC4R. One in [~]6500 individuals carry a heterozygous protein truncating variant (PTV) in BSN, which confers a 6.6, 3.7 and 3-fold higher risk of severe obesity (BMI \u003e40kg/m2), non-alcoholic fatty liver disease and type 2 diabetes, respectively. In contrast to most other obesity-related genes, rare variants in BSN and APBA1 had no apparent effect on childhood adiposity. Furthermore, BSN PTVs magnified the influence of common genetic variants associated with BMI, with a common polygenic score exhibiting an effect on BMI twice as large in BSN PTV carriers than non-carriers. Finally, we explored the plasma proteomic signatures of BSN PTV carriers as well as the functional consequences of BSN deletion in human iPSC-derived hypothalamic neurons. These approaches highlighted a network of differentially expressed genes that were collectively enriched for genomic regions associated with BMI, and suggest a role for degenerative neuronal synaptic function and neurotransmitter release in the etiology of obesity.\nSR-TWAS: Leveraging Multiple Reference Panels to Improve TWAS Power by Ensemble Machine Learning\nAuthors: Parrish, R. L.; Buchman, A. S.; Tasaki, S.; Wang, Y.; Avey, D.; Xu, J.; De Jager, P. L.; Bennett, D. A.; Epstein, M. P.; Yang, J.\nScore: 3.5, Published: 2023-06-27 DOI: 10.1101/2023.06.20.23291605\nMultiple reference panels of a given tissue or multiple tissues often exist, and multiple regression methods could be used for training gene expression imputation models for TWAS. To leverage expression imputation models (i.e., base models) trained with multiple reference panels, regression methods, and tissues, we develop a Stacked Regression based TWAS (SR-TWAS) tool which can obtain optimal linear combinations of base models for a given validation transcriptomic dataset. Both simulation and real studies showed that SR-TWAS improved power due to increased effective training sample sizes and borrowed strength across multiple regression methods and tissues. Leveraging base models across multiple reference panels, tissues, and regression methods, our studies of Alzheimers disease (AD) dementia and Parkinsons disease (PD) identified respective 11 independent significant risk genes for AD (supplementary motor area tissue) and 12 independent significant risk genes for PD (substantia nigra tissue), including 6 novels for AD and 6 novels for PD.\n",
  "wordCount" : "3005",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.23291975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.23291975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.23291975">
        <p class="paperTitle">Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.23291975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.23291975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.; Shi, Y.; Nandi, T. N.; Assimes, T. L.; Brunette, C. A.; Carroll, R. J.; Clifford, R.; Duvall, S.; Gelernter, J.; Hung, A.; Iyengar, S. K.; Joseph, J.; Kember, R.; Kranzler, H.; Levey, D.; Luoh, S.-W.; Merritt, V. C.; Overstreet, C.; Deak, J. D.; Grant, S. F. A.; Poli</p>
        <p class="info">Score: 26.0, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.23291975' target='https://doi.org/10.1101/2023.06.28.23291975'> 10.1101/2023.06.28.23291975</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have underrepresented individuals from non-European populations, impeding progress in characterizing the genetic architecture and consequences of health and disease traits. To address this, we present a population-stratified phenome-wide GWAS followed by a multi-population meta-analysis for 2,068 traits derived from electronic health records of 635,969 participants in the Million Veteran Program (MVP), a longitudinal cohort study of diverse U.S. Veterans genetically similar to the respective African (121,177), Admixed American (59,048), East Asian (6,702), and European (449,042) superpopulations defined by the 1000 Genomes Project. We identified 38,270 independent variants associating with one or more traits at experiment-wide (P &lt; 4.6x10-11) significance; fine-mapping 6,318 signals identified from 613 traits to single-variant resolution. Among these, a third (2,069) of the associations were found only among participants genetically similar to non-European reference populations, demonstrating the importance of expanding diversity in genetic studies. Our work provides a comprehensive atlas of phenome-wide genetic associations for future studies dissecting the architecture of complex traits in diverse populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291805">
        <p class="paperTitle">Non-coding variants are a rare cause of recessive developmental disorders in trans with coding variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium,  ; Baralle, D.; Martin, H. C.; Whiffin, N.</p>
        <p class="info">Score: 20.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291805' target='https://doi.org/10.1101/2023.06.23.23291805'> 10.1101/2023.06.23.23291805</a></p>
        <p class="abstract">Purpose: Identifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding second hits in trans with these is unknown. Methods: In 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing. Results: We found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA). Conclusion: We developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.23290941">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.23290941" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.23290941">
        <p class="paperTitle">Loss-of-function variants in CUL3 cause a syndromic neurodevelopmental disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.23290941" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.23290941" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blackburn, P. R.; Ebstein, F.; Hsieh, T.-C.; Motta, M.; Radio, F. C.; Herkert, J. C.; Rinne, T.; Thiffault, I.; Rapp, M.; Alders, M.; Maas, S.; Gerard, B.; Smol, T.; Vincent-Delorme, C.; Cogne, B.; Isidor, B.; Vincent, M.; Bachmann-Gagescu, R.; Rauch, A.; Joset, P.; Battista Ferrero, G.; Ciolfi, A.; Husson, T.; Guerrot, A.-M.; Bacino, C.; Macmurdo, C.; Thompson, S. S.; Rosenfeld, J. A.; Faivre, L.; Tran Mau-Them, F.; Deb, W.; Vignard, V.; Agrawal, P. B.; Madden, J. A.; Goldenberg, A.; Lecoquierre, F.; Zech, M.; Prokisch, H.; Necpal, J.; Jech, R.; Winkelmann, J.; Turcanova Koprusakova, M.; Kons</p>
        <p class="info">Score: 23.1, Published: 2023-06-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.23290941' target='https://doi.org/10.1101/2023.06.13.23290941'> 10.1101/2023.06.13.23290941</a></p>
        <p class="abstract">PurposeDe novo variants in CUL3 (Cullin-3 ubiquitin ligase) have been strongly associated with neurodevelopmental disorders (NDDs), but no large case series have been reported so far. Here we aimed to collect sporadic cases carrying rare variants in CUL3, describe the genotype-phenotype correlation, and investigate the underlying pathogenic mechanism.

MethodsGenetic data and detailed clinical records were collected via multi-center collaboration. Dysmorphic facial features were analyzed using GestaltMatcher. Variant effects on CUL3 protein stability were assessed using patient-derived T-cells.

ResultsWe assembled a cohort of 35 individuals with heterozygous CUL3 variants presenting a syndromic NDD characterized by intellectual disability with or without autistic features. Of these, 33 have loss-of-function (LoF) and two have missense variants. CUL3 LoF variants in patients may affect protein stability leading to perturbations in protein homeostasis, as evidenced by decreased ubiquitin-protein conjugates in vitro. Specifically, we show that cyclin E1 (CCNE1) and 4E-BP1 (EIF4EBP1), two prominent substrates of CUL3, fail to be targeted for proteasomal degradation in patient-derived cells.

ConclusionOur study further refines the clinical and mutational spectrum of CUL3-associated NDDs, expands the spectrum of cullin RING E3 ligase-associated neuropsychiatric disorders, and suggests haploinsufficiency via LoF variants is the predominant pathogenic mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291376">
        <p class="paperTitle">Combining a prioritization strategy and functional studies nominates 5&#39;UTR variants underlying inherited retinal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duenas Rey, A.; del Pozo Valero, M.; Bouckaert, M.; Van Den Broeck, F.; Daich Varela, M.; Van Heetvelde, M.; De Bruyne, M.; Van de Sompele, S.; Bauwens, M.; Ellingford, J. M.; Lenaerts, H.; Mahieu, Q.; Josifova, D.; Genomics England Research Consortium,  ; Rivolta, C.; Webster, A.; Arno, G.; Ayuso, C.; De Zaeytijd, J.; Leroy, B.; De Baere, E.; Coppieters, F.</p>
        <p class="info">Score: 14.1, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291376' target='https://doi.org/10.1101/2023.06.19.23291376'> 10.1101/2023.06.19.23291376</a></p>
        <p class="abstract">Background5 untranslated regions (5UTRs) are essential modulators of protein translation. Predicting the impact of 5UTR variants is challenging and typically not performed in routine diagnostics. Here, we present a combined approach of a comprehensive prioritization strategy and subsequent functional assays to evaluate 5UTR variation in two large cohorts of patients with inherited retinal diseases (IRDs).

MethodsWe performed an isoform-level re-analysis of retinal RNA-seq data to identify the protein-coding transcripts of 378 IRD genes with highest expression in retina. We evaluated the coverage of these 5UTRs by different whole exome sequencing (WES) capture kits. The selected 5UTRs were analyzed in whole genome sequencing (WGS) and WES data from IRD sub-cohorts from the 100,000 Genomes Project (n = 2,417 WGS) and an in-house database (n = 1,682 WES), respectively. Identified variants were annotated for 5UTR-relevant features and classified into 7 distinct categories based on their predicted functional consequence. We developed a variant prioritization strategy by integrating population frequency, specific criteria for each category, and family and phenotypic data. A selection of candidate variants underwent functional validation using diverse experimental approaches.

ResultsIsoform-level re-quantification of retinal gene expression revealed 76 IRD genes with a non-canonical retina-enriched isoform, of which 20 display a fully distinct 5UTR compared to that of their canonical isoform. Depending on the probe-design 3-20% of IRD genes have 5UTRs fully captured by WES. After analyzing these regions in both IRD cohorts we prioritized 11 (likely) pathogenic variants in 10 genes (ARL3, MERTK, NDP, NMNAT1, NPHP4, PAX6, PRPF31, PRPF4, RDH12, RD3), of which 8 were novel. Functional analyses further supported the pathogenicity of 2 variants. The MERTK:c.-125G&gt;A variant, overlapping a transcriptional start site, was shown to significantly reduce both luciferase mRNA levels and activity. The RDH12:c.-123C&gt;T variant was found in cis with the reported hypomorphic RDH12:c.701G&gt;A (p.Arg234His) variant in 11 patients. This 5UTR variant, predicted to introduce an upstream open reading frame, was shown to result in reduced RDH12 protein but unaltered mRNA levels.

ConclusionsThis study demonstrates the importance of 5UTR variants implicated in IRDs and provides a systematic approach for 5UTR annotation and validation that is applicable to other inherited diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 9.9, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections(1,2). Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction(3-5). The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative(6,7) to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity(6), lung function(8), and cancers(9), suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291830">
        <p class="paperTitle">Improved Type 2 Diabetes Risk Stratification in the Qatar Biobank Cohort by Ensemble Learning Classifier Incorporating Multi-Trait, Population-Specific, Polygenic Risk Scores</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmed, I.; Ziab, M.; Mbarek, H.; Taheri, S.; Chagoury, O. L.; Hussain, S. A.; Lakshmi, J.; Bhat, A.; Fakhro, K. A.; Al-Shabeeb AKIL, A. S.</p>
        <p class="info">Score: 9.4, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291830' target='https://doi.org/10.1101/2023.06.23.23291830'> 10.1101/2023.06.23.23291830</a></p>
        <p class="abstract">BackgroundType 2 Diabetes (T2D) is a pervasive chronic disease influenced by a complex interplay of environmental and genetic factors. To enhance T2D risk prediction, leveraging genetic information is essential, with polygenic risk scores (PRS) offering a promising tool for assessing individual genetic risk. Our study focuses on the comparison between multi-trait and single-trait PRS models and demonstrates how the incorporation of multi-trait PRS into risk prediction models can significantly augment T2D risk assessment accuracy and effectiveness.

MethodsWe conducted genome-wide association studies (GWAS) on 12 distinct T2D-related traits within a cohort of 14,278 individuals, all sequenced under the Qatar Genome Programme (QGP). This in-depth genetic analysis yielded several novel genetic variants associated with T2D, which served as the foundation for constructing multiple weighted PRS models. To assess the cumulative risk from these predictors, we applied machine learning (ML) techniques, which allowed for a thorough risk assessment.

ResultsOur research identified genetic variations tied to T2D risk and facilitated the construction of ML models integrating PRS predictors for an exhaustive risk evaluation. The top-performing ML model demonstrated a robust performance with an accuracy of 0.8549, AUC of 0.92, AUC-PR of 0.8522, and an F1 score of 0.757, reflecting its strong capacity to differentiate cases from controls. We are currently working on acquiring independent T2D cohorts to validate the efficacy of our final model.

ConclusionOur research underscores the potential of PRS models in identifying individuals within the population who are at elevated risk of developing T2D and its associated complications. The use of multi-trait PRS and ML models for risk prediction could inform early interventions, potentially identifying T2D patients who stand to benefit most based on their individual genetic risk profile. This combined approach signifies a stride forward in the field of precision medicine, potentially enhancing T2D risk prediction, prevention, and management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291425">
        <p class="paperTitle">Biallelic loss of function variants in WBP4, encoding a spliceosome protein, result in a variable neurodevelopmental delay syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engal, E.; Oja, K. T.; Maroofian, R.; Geminder, O.; Le, T.-L.; Mor, E.; Tzvi, N.; Elefant, N.; Zaki, M. S.; Gleeson, J. G.; Muru, K.; Pajusalu, S.; Wojcik, M. H.; Pachat, D.; Abd Elmaksoud, M.; Jeong, W. C.; Lee, H.; Bauer, P.; Zifarelli, G.; Houlden, H.; Elpeleg, O.; Gordon, C.; Harel, T.; Ounap, K.; Salton, M.; Mor-Shaked, H.</p>
        <p class="info">Score: 9.1, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291425' target='https://doi.org/10.1101/2023.06.19.23291425'> 10.1101/2023.06.19.23291425</a></p>
        <p class="abstract">Over two dozen spliceosome proteins are involved in human diseases, also referred to as spliceosomopathies. WBP4 (WW Domain Binding Protein 4) is part of the early spliceosomal complex, and was not described before in the context of human pathologies. Ascertained through GeneMatcher we identified eleven patients from eight families, with a severe neurodevelopmental syndrome with variable manifestations. Clinical manifestations included hypotonia, global developmental delay, severe intellectual disability, brain abnormalities, musculoskeletal and gastrointestinal abnormalities. Genetic analysis revealed overall five different homozygous loss-of-function variants in WBP4. Immunoblotting on fibroblasts from two affected individuals with different genetic variants demonstrated complete loss of protein, and RNA sequencing analysis uncovered shared abnormal splicing patterns, including enrichment for abnormalities of the nervous system and musculoskeletal system genes, suggesting that the overlapping differentially spliced genes are related to the common phenotypes of the probands. We conclude that biallelic variants in WBP4 cause a spliceosomopathy. Further functional studies are called for better understanding of the mechanism of pathogenicity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291611">
        <p class="paperTitle">PA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium,  ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.</p>
        <p class="info">Score: 6.2, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291611' target='https://doi.org/10.1101/2023.06.23.23291611'> 10.1101/2023.06.23.23291611</a></p>
        <p class="abstract">The state-of-the-field in complex disorder genetics is marked by large-scale, biobank-ascertained data sets that integrate broad demographic, health, survey, and genetic data. Leveraging the richness of this data can advance our understanding of complex disorders by improving predictions, describing etiology, and augmenting gene-mapping. These themes are especially, but not exclusively, relevant to Major Depressive Disorder (MDD), a leading cause of disability for which genetic predictors underperform, clinical heterogeneity is etiologically enigmatic, and missing heritability persists. The iPSYCH 2015 case-cohort presents a unique opportunity to integrate genotypes, register-based clinical outcomes, and extended genealogies of 30,949 MDD cases and 39,655 random population controls. To make full use of our data, we introduce the Pearson-Aitken framework for Family Genetic Risk Scores (PA-FGRS) to estimate individuals&#39; liabilities for major depressive disorder from extended genealogies of partially observed relatives. PA-FGRS extends previous methods by accounting for censoring, leveraging distant relatives, and utilizing a flexible, model-based approach amenable to analysis and extension, advantages we highlight in simulations. Combining PA-FGRS with genotype data improves classification, replicates and extends known genetic contributions to clinical heterogeneity, and increases power for genome-wide association studies. This study combines novel analyses of unique data and can serve as a model for studies of other outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.23291368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.23291368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.23291368">
        <p class="paperTitle">Protein-truncating variants in BSN are associated with severe adult-onset obesity, type 2 diabetes and fatty liver disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.23291368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.23291368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Y.; Chukanova, M.; Kentistou, K. A.; Fairhurst-Hunter, Z.; Siegert, A. M.; Jia, R.; Dowsett, G.; Gardner, E. J.; Day, F. R.; Kaisinger, L. R.; Tung, Y.-C. L.; Lam, B. Y. H.; Chen, H.-J. C.; Wang, Q.; Berumen-Campos, J.; Kuri-Morales, P.; Tapia-Conyer, R.; Alegre-Diaz, J.; Emberson, J.; Torres, J. M.; Collins, R.; Saleheen, D.; Smith, K. R.; Paul, D. R.; Merkle, F.; Wareham, N. J.; Petrovski, S.; O&#39;Rahilly, S.; Ong, K. K.; Yeo, G. S. H.; Perry, J. R. B.</p>
        <p class="info">Score: 9.1, Published: 2023-06-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.23291368' target='https://doi.org/10.1101/2023.06.14.23291368'> 10.1101/2023.06.14.23291368</a></p>
        <p class="abstract">Obesity is a major risk factor for many common diseases and has a significant heritable component. While clinical and large-scale population studies have identified several genes harbouring rare alleles with large effects on obesity risk, there are likely many unknown genes with highly penetrant effects remaining. To this end, we performed whole exome-sequence analyses for adult body mass index (BMI) in up to 587,027 individuals. We identified rare, loss of function variants in two genes - BSN and APBA1 - with effects on BMI substantially larger than well-established obesity genes such as MC4R. One in [~]6500 individuals carry a heterozygous protein truncating variant (PTV) in BSN, which confers a 6.6, 3.7 and 3-fold higher risk of severe obesity (BMI &gt;40kg/m2), non-alcoholic fatty liver disease and type 2 diabetes, respectively. In contrast to most other obesity-related genes, rare variants in BSN and APBA1 had no apparent effect on childhood adiposity. Furthermore, BSN PTVs magnified the influence of common genetic variants associated with BMI, with a common polygenic score exhibiting an effect on BMI twice as large in BSN PTV carriers than non-carriers. Finally, we explored the plasma proteomic signatures of BSN PTV carriers as well as the functional consequences of BSN deletion in human iPSC-derived hypothalamic neurons. These approaches highlighted a network of differentially expressed genes that were collectively enriched for genomic regions associated with BMI, and suggest a role for degenerative neuronal synaptic function and neurotransmitter release in the etiology of obesity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.23291605">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.23291605" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.23291605">
        <p class="paperTitle">SR-TWAS: Leveraging Multiple Reference Panels to Improve TWAS Power by Ensemble Machine Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.23291605" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.23291605" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parrish, R. L.; Buchman, A. S.; Tasaki, S.; Wang, Y.; Avey, D.; Xu, J.; De Jager, P. L.; Bennett, D. A.; Epstein, M. P.; Yang, J.</p>
        <p class="info">Score: 3.5, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.23291605' target='https://doi.org/10.1101/2023.06.20.23291605'> 10.1101/2023.06.20.23291605</a></p>
        <p class="abstract">Multiple reference panels of a given tissue or multiple tissues often exist, and multiple regression methods could be used for training gene expression imputation models for TWAS. To leverage expression imputation models (i.e., base models) trained with multiple reference panels, regression methods, and tissues, we develop a Stacked Regression based TWAS (SR-TWAS) tool which can obtain optimal linear combinations of base models for a given validation transcriptomic dataset. Both simulation and real studies showed that SR-TWAS improved power due to increased effective training sample sizes and borrowed strength across multiple regression methods and tissues. Leveraging base models across multiple reference panels, tissues, and regression methods, our studies of Alzheimers disease (AD) dementia and Parkinsons disease (PD) identified respective 11 independent significant risk genes for AD (supplementary motor area tissue) and 12 independent significant risk genes for PD (substantia nigra tissue), including 6 novels for AD and 6 novels for PD.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
