<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Valid inference for machine learning-assisted GWAS
Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.
Score: 14.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779
Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Valid inference for machine learning-assisted GWAS
Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.
Score: 14.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779
Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Valid inference for machine learning-assisted GWAS
Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.
Score: 14.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779
Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Valid inference for machine learning-assisted GWAS\nAuthors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.\nScore: 14.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779\nMachine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research.",
  "keywords": [
    
  ],
  "articleBody": " Valid inference for machine learning-assisted GWAS\nAuthors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.\nScore: 14.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779\nMachine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research. However, the validity of associations identified from ML-assisted GWAS has not been carefully evaluated. In this study, we report pervasive risks for false positive associations in ML-assisted GWAS, and introduce POP-GWAS, a novel statistical framework that reimagines GWAS on ML-imputed outcomes. POP-GWAS provides valid statistical inference irrespective of the quality of imputation or variables and algorithms used for imputation. It also only requires GWAS summary statistics as input. We employed POP-GWAS to perform the largest GWAS of bone mineral density (BMD) derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 novel loci reaching genome-wide significance and revealing skeletal site-specific genetic architecture of BMD. Our framework may fundamentally reshape the analytical strategies in future ML-assisted GWAS.\nBenchmarking Mendelian Randomization methods for causal inference using genome-wide association study summary statistics\nAuthors: Hu, X.; Cai, M.; Xiao, J.; Wan, X.; Wang, Z.; Zhao, H.; Yang, C.\nScore: 10.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300765\nMendelian Randomization (MR), which utilizes genetic variants as instrumental variables (IVs), has gained popularity as a method for causal inference between phenotypes using genetic data. While efforts have been made to relax IV assumptions and develop new methods for causal inference in the presence of invalid IVs due to confounding, the reliability of MR methods in real-world applications remains uncertain. To bridge this gap, we conducted a benchmark study evaluating 15 MR methods using real-world genetic datasets. Our study focused on three crucial aspects: type I error control in the presence of various confounding scenarios (e.g., population stratification, pleiotropy, and assortative mating), the accuracy of causal effect estimates, replicability and power. By comprehensively evaluating the performance of compared methods over one thousand pairs of exposure-outcome traits, our study not only provides valuable insights into the performance and limitations of the compared methods but also offers practical guidance for researchers to choose appropriate MR methods for causal inference.\nWhole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy\nAuthors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Arzberger, T.; Roeber, S.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L.; Miller, B. L.; Seeley, W. W.; Van Deerlin, V. M.;\nScore: 6.1, Published: 2023-12-30 DOI: 10.1101/2023.12.28.23300612\nProgressive supranuclear palsy (PSP) is a neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. We performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), small insertions/deletions (indels), and structural variants (SVs) in a cohort of 1,718 individuals with PSP and 2,944 control subjects. Analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and also uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. In contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele and the {varepsilon}4 allele to be protective, a pattern similar to the association pattern observed in age-related macular degeneration (AMD) but the opposite observed for Alzheimers disease (AD). Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. We also observed seven common SVs associated with PSP on the H1/H2 haplotype region (17q21.31) and in a few other loci: IGH, PCMT1, CYP2A13, and SMCP. Particularly, in the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP. Through WGS, we significantly refine our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.\nGenomic analysis identifies risk factors in restless legs syndrome.\nAuthors: Akcimen, F.; Chia, R.; Saez-Atienzar, S.; Ruffo, P.; Rasheed, M.; Ross, J.; Liao, C.; Ray, A.; Dion, P. A.; Scholz, S. W.; Rouleau, G. A.; Traynor, B. J.\nScore: 5.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.23300211\nRestless legs syndrome (RLS) is a neurological condition that causes uncomfortable sensations in the legs and an irresistible urge to move them, typically during periods of rest. The genetic basis and pathophysiology of RLS are incompletely understood. Here, we present a whole-genome sequencing and genome-wide association meta-analysis of RLS cases (n = 9,851) and controls (n = 38,957) in three population-based biobanks (All of Us, Canadian Longitudinal Study on Aging, and CARTaGENE). Genome-wide association analysis identified nine independent risk loci, of which eight had been previously reported, and one was a novel risk locus (LMX1B, rs35196838, OR = 1.14, 95% CI = 1.09-1.19, p-value = 2.2 x 10-9). A genome-wide, gene-based common variant analysis identified GLO1 as an additional risk gene (p-value = 8.45 x 10-7). Furthermore, a transcriptome-wide association study also identified GLO1 and a previously unreported gene, ELFN1. A genetic correlation analysis revealed significant common variant overlaps between RLS and neuroticism (rg = 0.40, se = 0.08, p-value = 5.4 x 10-7), depression (rg = 0.35, se = 0.06, p-value = 2.17 x 10-8), and intelligence (rg = -0.20, se = 0.06, p-value = 4.0 x 10-4). Our study expands the understanding of the genetic architecture of RLS and highlights the contributions of common variants to this prevalent neurological disorder.\nGenome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health\nAuthors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.\nScore: 4.2, Published: 2023-12-29 DOI: 10.1101/2023.12.26.23300533\nNeuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field, brain age gap has emerged as promising MRI-based biomarker quantifying the deviation between an individuals biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimers disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.\nPolygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.\nAuthors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.\nScore: 8.1, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412\nIntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups. Very few studies have looked at the transferability of European-derived PRS for behavioural and mental health phenotypes to non-Europeans. We assessed the comparative accuracy of PRS for Major Depressive Disorder (MDD) trained on European and African Ancestries GWAS studies to predict MDD and related traits in African Ancestries participants from the UK Biobank. MethodsUK Biobank participants were selected based on Principal component analysis (PCA) clustering with an African genetic similarity reference population and MDD was assessed with the Composite International Diagnostic Interview (CIDI). Polygenic Risk Scores (PRS) were computed using PRSice2 using either European or African Ancestries GWAS summary statistics. ResultsPRS trained on European ancestry samples (246,363 cases) predicted case control status in Africans of the UK Biobank with similar accuracies (190 cases, R2=2%) to PRS trained on far much smaller samples of African Ancestries participants from 23andMe, Inc. (5045 cases, R2=1.8%). This suggests that prediction of MDD status from Africans to Africans had greater efficiency per unit increase in the discovery sample size than prediction of MDD from Europeans to Africans. Prediction of MDD status in African UK Biobank participants using GWAS findings of causal risk factors from European ancestries was non-significant. ConclusionGWAS studies of MDD in European ancestries are an inefficient means of improving polygenic prediction accuracy in African samples.\nRare disease gene association discovery from burden analysis of the 100,000 Genomes Project data\nAuthors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O'Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra\nScore: 8.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300294\nTo discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.\nEmpowering GWAS Discovery through Enhanced Genotype Imputation\nAuthors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.\nScore: 11.4, Published: 2023-12-21 DOI: 10.1101/2023.12.18.23300143\nGenotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.\nMulti-ancestry transcriptome-wide association studies of cognitive function, white matter hyperintensity, and Alzheimer's disease\nAuthors: Chaar, D. L.; Li, Z.; Shang, L.; Ratliff, S. M.; Mosley, T. H.; Kardia, S. L. R.; Zhao, W.; Zhou, X.; Smith, J. A.\nScore: 1.5, Published: 2024-01-05 DOI: 10.1101/2024.01.03.24300768\nGenetic variants increase the risk of neurocognitive disorders in later life including Vascular Dementia (VaD) and Alzheimers disease (AD), but the precise relationships between genetic risk factors and underlying disease etiology are not well understood. Transcriptome-wide association studies (TWAS) can be leveraged to better characterize the genes and biological pathways underlying genetic influences on disease. To date, almost all existing TWAS have been conducted using expression studies from individuals of a single genetic ancestry, primarily European. Using the joint likelihood-based inference framework in Multi-ancEstry TRanscriptOme-wide analysis (METRO), we leveraged gene expression data from European (EA) and African ancestries (AA) to identify genes associated with general cognitive function, white matter hyperintensity (WMH), and AD. Regions were fine-mapped using Fine-mapping Of CaUsal gene Sets (FOCUS). We identified 266, 23, 69, and 2 genes associated with general cognitive function, WMH, AD (using EA GWAS summary statistics), and AD (using AA GWAS), respectively (Bonferroni-corrected alpha=P\u003c2.9x10-6), some of which were previously identified. Enrichment analysis showed that many of the identified genes were in pathways related to innate immunity, vascular dysfunction, and neuroinflammation. Further, downregulation of ICA1L was associated with higher WMH and with AD, indicating its potential contribution to overlapping AD and VaD neuropathology. To our knowledge, our study is the first TWAS of cognitive function and neurocognitive disorders that used expression mapping studies in multiple ancestries. This work may expand the benefits of TWAS studies beyond a single ancestry group and help to identify gene targets for pharmaceutical or preventative treatment for dementia. Author SummaryTranscriptome-wide association studies (TWAS) can be used to understand the mechanisms of gene expression that underly disease etiology. However, to date, TWAS methods have mostly been used in a single ancestry group, especially European ancestry (EA), and few TWAS have focused on cognitive function or structural brain measures. We used a newly developed TWAS method called the Multi-ancEstry TRanscriptOme-wide analysis (METRO) to incorproate gene expression data from 801 EA and 1,032 African ancestry (AA) adults to identify genes associated with general cognitive function, structural brain changes called white matter hyperintensities (WMH) that predispose people to vascular dementia, and another form of dementia called Alzheimers disease (AD). We found that reduced gene expression of ICA1L was associated with more WMH and with AD, indicating its potential contribution to overlapping AD and vascular dementia neuropathologies. To our knowledge, our study is the first TWAS of cognitive function and neurocognitive disorders using multiple ancestries. This work may expand the benefits of TWAS studies beyond a single ancestry group and help to identify gene targets for pharmaceutical or preventative treatment for dementia.\nIdentification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis\nAuthors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.\nScore: 26.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893\nClonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.\n",
  "wordCount" : "2669",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300779">
        <p class="paperTitle">Valid inference for machine learning-assisted GWAS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.</p>
        <p class="info">Score: 14.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300779' target='https://doi.org/10.1101/2024.01.03.24300779'> 10.1101/2024.01.03.24300779</a></p>
        <p class="abstract">Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research. However, the validity of associations identified from ML-assisted GWAS has not been carefully evaluated. In this study, we report pervasive risks for false positive associations in ML-assisted GWAS, and introduce POP-GWAS, a novel statistical framework that reimagines GWAS on ML-imputed outcomes. POP-GWAS provides valid statistical inference irrespective of the quality of imputation or variables and algorithms used for imputation. It also only requires GWAS summary statistics as input. We employed POP-GWAS to perform the largest GWAS of bone mineral density (BMD) derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 novel loci reaching genome-wide significance and revealing skeletal site-specific genetic architecture of BMD. Our framework may fundamentally reshape the analytical strategies in future ML-assisted GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300765" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300765">
        <p class="paperTitle">Benchmarking Mendelian Randomization methods for causal inference using genome-wide association study summary statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, X.; Cai, M.; Xiao, J.; Wan, X.; Wang, Z.; Zhao, H.; Yang, C.</p>
        <p class="info">Score: 10.8, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300765' target='https://doi.org/10.1101/2024.01.03.24300765'> 10.1101/2024.01.03.24300765</a></p>
        <p class="abstract">Mendelian Randomization (MR), which utilizes genetic variants as instrumental variables (IVs), has gained popularity as a method for causal inference between phenotypes using genetic data. While efforts have been made to relax IV assumptions and develop new methods for causal inference in the presence of invalid IVs due to confounding, the reliability of MR methods in real-world applications remains uncertain. To bridge this gap, we conducted a benchmark study evaluating 15 MR methods using real-world genetic datasets. Our study focused on three crucial aspects: type I error control in the presence of various confounding scenarios (e.g., population stratification, pleiotropy, and assortative mating), the accuracy of causal effect estimates, replicability and power. By comprehensively evaluating the performance of compared methods over one thousand pairs of exposure-outcome traits, our study not only provides valuable insights into the performance and limitations of the compared methods but also offers practical guidance for researchers to choose appropriate MR methods for causal inference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.23300612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.23300612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.23300612">
        <p class="paperTitle">Whole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.23300612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.23300612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Arzberger, T.; Roeber, S.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L.; Miller, B. L.; Seeley, W. W.; Van Deerlin, V. M.;</p>
        <p class="info">Score: 6.1, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.23300612' target='https://doi.org/10.1101/2023.12.28.23300612'> 10.1101/2023.12.28.23300612</a></p>
        <p class="abstract">Progressive supranuclear palsy (PSP) is a neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. We performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), small insertions/deletions (indels), and structural variants (SVs) in a cohort of 1,718 individuals with PSP and 2,944 control subjects. Analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and also uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. In contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele and the {varepsilon}4 allele to be protective, a pattern similar to the association pattern observed in age-related macular degeneration (AMD) but the opposite observed for Alzheimers disease (AD). Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. We also observed seven common SVs associated with PSP on the H1/H2 haplotype region (17q21.31) and in a few other loci: IGH, PCMT1, CYP2A13, and SMCP. Particularly, in the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP. Through WGS, we significantly refine our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23300211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23300211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23300211">
        <p class="paperTitle">Genomic analysis identifies risk factors in restless legs syndrome.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23300211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23300211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akcimen, F.; Chia, R.; Saez-Atienzar, S.; Ruffo, P.; Rasheed, M.; Ross, J.; Liao, C.; Ray, A.; Dion, P. A.; Scholz, S. W.; Rouleau, G. A.; Traynor, B. J.</p>
        <p class="info">Score: 5.0, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23300211' target='https://doi.org/10.1101/2023.12.19.23300211'> 10.1101/2023.12.19.23300211</a></p>
        <p class="abstract">Restless legs syndrome (RLS) is a neurological condition that causes uncomfortable sensations in the legs and an irresistible urge to move them, typically during periods of rest. The genetic basis and pathophysiology of RLS are incompletely understood. Here, we present a whole-genome sequencing and genome-wide association meta-analysis of RLS cases (n = 9,851) and controls (n = 38,957) in three population-based biobanks (All of Us, Canadian Longitudinal Study on Aging, and CARTaGENE). Genome-wide association analysis identified nine independent risk loci, of which eight had been previously reported, and one was a novel risk locus (LMX1B, rs35196838, OR = 1.14, 95% CI = 1.09-1.19, p-value = 2.2 x 10-9). A genome-wide, gene-based common variant analysis identified GLO1 as an additional risk gene (p-value = 8.45 x 10-7). Furthermore, a transcriptome-wide association study also identified GLO1 and a previously unreported gene, ELFN1. A genetic correlation analysis revealed significant common variant overlaps between RLS and neuroticism (rg = 0.40, se = 0.08, p-value = 5.4 x 10-7), depression (rg = 0.35, se = 0.06, p-value = 2.17 x 10-8), and intelligence (rg = -0.20, se = 0.06, p-value = 4.0 x 10-4). Our study expands the understanding of the genetic architecture of RLS and highlights the contributions of common variants to this prevalent neurological disorder.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.23300533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.23300533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.23300533">
        <p class="paperTitle">Genome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.23300533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.23300533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.</p>
        <p class="info">Score: 4.2, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.23300533' target='https://doi.org/10.1101/2023.12.26.23300533'> 10.1101/2023.12.26.23300533</a></p>
        <p class="abstract">Neuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field,  brain age gap has emerged as promising MRI-based biomarker quantifying the deviation between an individuals biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimers disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.23300412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.23300412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.23300412">
        <p class="paperTitle">Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.23300412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.23300412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team,  ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.</p>
        <p class="info">Score: 8.1, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.23300412' target='https://doi.org/10.1101/2023.12.24.23300412'> 10.1101/2023.12.24.23300412</a></p>
        <p class="abstract">IntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups. Very few studies have looked at the transferability of European-derived PRS for behavioural and mental health phenotypes to non-Europeans. We assessed the comparative accuracy of PRS for Major Depressive Disorder (MDD) trained on European and African Ancestries GWAS studies to predict MDD and related traits in African Ancestries participants from the UK Biobank.

MethodsUK Biobank participants were selected based on Principal component analysis (PCA) clustering with an African genetic similarity reference population and MDD was assessed with the Composite International Diagnostic Interview (CIDI). Polygenic Risk Scores (PRS) were computed using PRSice2 using either European or African Ancestries GWAS summary statistics.

ResultsPRS trained on European ancestry samples (246,363 cases) predicted case control status in Africans of the UK Biobank with similar accuracies (190 cases, R2=2%) to PRS trained on far much smaller samples of African Ancestries participants from 23andMe, Inc. (5045 cases, R2=1.8%). This suggests that prediction of MDD status from Africans to Africans had greater efficiency per unit increase in the discovery sample size than prediction of MDD from Europeans to Africans. Prediction of MDD status in African UK Biobank participants using GWAS findings of causal risk factors from European ancestries was non-significant.

ConclusionGWAS studies of MDD in European ancestries are an inefficient means of improving polygenic prediction accuracy in African samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300294">
        <p class="paperTitle">Rare disease gene association discovery from burden analysis of the 100,000 Genomes Project data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O&#39;Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra</p>
        <p class="info">Score: 8.3, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300294' target='https://doi.org/10.1101/2023.12.20.23300294'> 10.1101/2023.12.20.23300294</a></p>
        <p class="abstract">To discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.23300143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.23300143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.23300143">
        <p class="paperTitle">Empowering GWAS Discovery through Enhanced Genotype Imputation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.23300143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.23300143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Marino, A.; Mahmoud, A. A.; Bohn, S.; Lerga-Jaso, J.; Novkovic, B.; Manson, C.; Loguercio, S.; Terpolovsky, A.; Matushyn, M.; Torkamani, A.; Yazdi, P. G.</p>
        <p class="info">Score: 11.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.23300143' target='https://doi.org/10.1101/2023.12.18.23300143'> 10.1101/2023.12.18.23300143</a></p>
        <p class="abstract">Genotype imputation, crucial in genomics research, often faces accuracy limitations, notably for rarer variants. Leveraging data from the 1000 Genomes Project, TOPMed and UK Biobank, we demonstrate that Selphi, our novel imputation method, significantly outperforms Beagle5.4, Minimac4 and IMPUTE5 across various metrics (12.5%-26.5% as measured by error count) and allele frequencies (13.0%-27.1% for low-frequency variants).This improvement in accuracy boosts variant discovery in GWAS and improves polygenic risk scores.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300768" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300768">
        <p class="paperTitle">Multi-ancestry transcriptome-wide association studies of cognitive function, white matter hyperintensity, and Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chaar, D. L.; Li, Z.; Shang, L.; Ratliff, S. M.; Mosley, T. H.; Kardia, S. L. R.; Zhao, W.; Zhou, X.; Smith, J. A.</p>
        <p class="info">Score: 1.5, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300768' target='https://doi.org/10.1101/2024.01.03.24300768'> 10.1101/2024.01.03.24300768</a></p>
        <p class="abstract">Genetic variants increase the risk of neurocognitive disorders in later life including Vascular Dementia (VaD) and Alzheimers disease (AD), but the precise relationships between genetic risk factors and underlying disease etiology are not well understood. Transcriptome-wide association studies (TWAS) can be leveraged to better characterize the genes and biological pathways underlying genetic influences on disease. To date, almost all existing TWAS have been conducted using expression studies from individuals of a single genetic ancestry, primarily European. Using the joint likelihood-based inference framework in Multi-ancEstry TRanscriptOme-wide analysis (METRO), we leveraged gene expression data from European (EA) and African ancestries (AA) to identify genes associated with general cognitive function, white matter hyperintensity (WMH), and AD. Regions were fine-mapped using Fine-mapping Of CaUsal gene Sets (FOCUS). We identified 266, 23, 69, and 2 genes associated with general cognitive function, WMH, AD (using EA GWAS summary statistics), and AD (using AA GWAS), respectively (Bonferroni-corrected alpha=P&lt;2.9x10-6), some of which were previously identified. Enrichment analysis showed that many of the identified genes were in pathways related to innate immunity, vascular dysfunction, and neuroinflammation. Further, downregulation of ICA1L was associated with higher WMH and with AD, indicating its potential contribution to overlapping AD and VaD neuropathology. To our knowledge, our study is the first TWAS of cognitive function and neurocognitive disorders that used expression mapping studies in multiple ancestries. This work may expand the benefits of TWAS studies beyond a single ancestry group and help to identify gene targets for pharmaceutical or preventative treatment for dementia.

Author SummaryTranscriptome-wide association studies (TWAS) can be used to understand the mechanisms of gene expression that underly disease etiology. However, to date, TWAS methods have mostly been used in a single ancestry group, especially European ancestry (EA), and few TWAS have focused on cognitive function or structural brain measures. We used a newly developed TWAS method called the Multi-ancEstry TRanscriptOme-wide analysis (METRO) to incorproate gene expression data from 801 EA and 1,032 African ancestry (AA) adults to identify genes associated with general cognitive function, structural brain changes called white matter hyperintensities (WMH) that predispose people to vascular dementia, and another form of dementia called Alzheimers disease (AD). We found that reduced gene expression of ICA1L was associated with more WMH and with AD, indicating its potential contribution to overlapping AD and vascular dementia neuropathologies. To our knowledge, our study is the first TWAS of cognitive function and neurocognitive disorders using multiple ancestries. This work may expand the benefits of TWAS studies beyond a single ancestry group and help to identify gene targets for pharmaceutical or preventative treatment for dementia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.23299893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.23299893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.23299893">
        <p class="paperTitle">Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.23299893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.23299893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.</p>
        <p class="info">Score: 26.3, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.23299893' target='https://doi.org/10.1101/2023.12.13.23299893'> 10.1101/2023.12.13.23299893</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals

SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
