<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores
Authors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes &amp; Health Research Team, ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.
Score: 23.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores
Authors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes &amp; Health Research Team, ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.
Score: 23." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores
Authors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes &amp; Health Research Team, ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.
Score: 23."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores\nAuthors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes \u0026amp; Health Research Team, ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.\nScore: 23.",
  "keywords": [
    
  ],
  "articleBody": " Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores\nAuthors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes \u0026 Health Research Team, ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.\nScore: 23.6, Published: 2023-08-25 DOI: 10.1101/2023.08.23.23294497\nObjectiveCorrect classification of type 1 (T1D) and type 2 diabetes (T2D) is challenging due to overlapping clinical features and the increasingly early onset of T2D, particularly in South Asians. We used polygenic risk scores (PRSs) in a British Bangladeshi and Pakistani population with diabetes to estimate the proportion and misclassification rate of T1D in insulin-treated individuals with ambiguous features. Research Design and MethodsUsing linked health record from the Genes \u0026 Health cohort (n=38,344) we defined four groups: 31 T1D cases, 1,842 T2D cases, and after excluding these, 839 insulin-treated individuals with ambiguous features and 5,174 controls. Combining these with 307 confirmed T1D cases and 307 controls from India, we calculated ancestry-corrected PRSs for T1D and T2D, with which we estimated the proportion of T1D cases within the ambiguous group and evaluated misclassification. ResultsWe estimated that the prevalence of T1D was [~]6% within the ambiguous group, or [~]4.5% within the subset who had T2D codes in their health records. We saw no significant association between the T1D or T2D PRS and BMI at diagnosis, time to insulin, or the presence of T1D or T2D diagnostic codes amongst the T2D or ambiguous cases. ConclusionsWe estimate that 4.5% of British Pakistanis and Bangladeshis with diabetes who are treated with insulin and have ambiguous clinical features have been classified incorrectly in their health records, and in fact have T1D. These findings have significant implications for the delivery of optimal clinical care at population level.\nThe pivotal role of the X-chromosome in the genetic architecture of the human brain\nAuthors: Jiang, Z.; Sullivan, P. F.; Li, T.; Zhao, B.; Wang, X.; Luo, T.; Huang, S.; Guan, P. Y.; Chen, J.; Yang, Y.; Stein, J. L.; Li, Y.; Liu, D.; Sun, L.; Zhu, H.\nScore: 5.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.23294848\nGenes on the X-chromosome are extensively expressed in the human brain, resulting in substantial influences on brain development, intellectual disability, and other brain-related disorders. To comprehensively investigate the X-chromosome's impact on the cerebral cortex, white matter tract microstructures, and intrinsic and extrinsic brain functions, we examined 2,822 complex brain imaging traits obtained from n = 34,000 subjects in the UK Biobank. We unveiled potential autosome-X-chromosome interaction, while proposing an atlas of dosage compensation (DC) for each set of traits. We observed a pronounced X-chromosome impact on the corticospinal tract and the functional amplitude and connectivity of visual networks. In association studies, we identified 50 genome-wide significant trait-locus pairs enriched in Xq28, 22 of which replicated in independent datasets (n = 4,900). Notably, 13 newly identified pairs were in the X-chromosome's non-pseudo-autosomal regions (NPR). The volume of the right ventral diencephalon shared genetic architecture with schizophrenia and educational attainment in a locus indexed by rs2361468 (located ~3kb upstream of PJA1, a conserved and ubiquitously expressed gene implicated in multiple psychiatric disorders). No significant associations were identified in the pseudo-autosomal regions (PAR) or the Y-chromosome. Finally, we explored sex-specific associations on the X-chromosome and compared differing genetic effects between sexes. We found much more associations can be identified in males (33 versus 9) given a similar sample size. In conclusion, our research provides invaluable insights into the X-chromosome's role in the human brain, contributing to the observed sex differences in brain structure and function.\nRelationship between sex biases in gene expression and sex biases in autism and Alzheimer's disease\nAuthors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.\nScore: 5.2, Published: 2023-09-01 DOI: 10.1101/2023.08.29.23294773\nSex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimer's disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences.\nThe predictive capacity of polygenic risk scores for disease risk is only moderately influenced by imputation panels tailored to the target population\nAuthors: Levi, H.; Elkon, R.; Shamir, R.\nScore: 4.6, Published: 2023-09-01 DOI: 10.1101/2023.08.29.23294769\nPolygenic risk scores (PRS) predict individuals genetic risk of developing complex diseases. They summarize the effect of many genetic variants discovered in genome-wide association studies (GWASs). However, to date, large GWASs exist primarily for the European population and the quality of PRS prediction declines when applied to target sets of other ethnicities. A key step in using a PRS is imputation, which is the inference of un-typed SNPs using a set of fully-sequenced individuals, called the imputation panel. The SNP genotypes called by the imputation process depend on the ethnic composition of the imputation panel. Several studies have shown that imputing genotypes using a panel that contains individuals of the same ethnicity as the genotyped individuals improves imputation accuracy. However, until now, there has been no systematic investigation into the influence of the ethnic composition of imputation panels on the accuracy of PRS predictions when applied to ethnic groups that differ from the population used in the GWAS. In this study we estimated the effect of imputation of the target set on prediction accuracy of PRS when the discovery (GWAS) and the target sets come from different ethnic groups. We analyzed twelve binary phenotypes and three populations from the UK Biobank (Europeans, South-Asians, and Africans). We generated imputation panels from several ethnic groups, imputed the target set using each panel, and generated PRS to compute individuals risk scores. Then, we assessed the prediction accuracy obtained from each imputation panel. Our analysis indicates that using an imputation panel matched to the ethnic group of the target population yields only a marginal improvement and only under specific conditions. Hence, while a target-matched imputation panel can potentially improve prediction accuracy of European PRSs in non-EUR populations, the improvement is limited.\nThe Iberian Roma genetic variant server; population structure, susceptibility to disease and adaptive traits.\nAuthors: Mavillard, F.; Perez-Florido, J.; Ortuno, F.; Valladares, A.; Alvarez-Villegas, M. L.; Roldan, G.; Carmona, R.; Soriano, M.; Susarte, S.; Fuentes, P.; Lopez-Lopez, D.; Nunez-Negrillo, A. M.; Carvajal, A.; Morgado, Y.; Arteaga, D.; Ufano, R.; Mir, P.; Gamella, J.; Dopazo, J.; Paradas, c.; Cabrera Serrano, M.\nScore: 13.9, Published: 2023-08-25 DOI: 10.1101/2023.08.25.23294490\nThe Roma are the most numerous ethnic minority in Europe. The Iberian Roma arrived in the Iberian Peninsula five centuries ago and still today, they keep a strong group identity. Demographic and cultural reasons lie behind a high rate of Mendelian disease often related to founder variants. We have analysed exome data from 119 Iberian Roma individuals collected from 2018 to 2020. A database of variant frequency has been implemented (IRPVS) and made available online. We have analysed the carrier rate of founder private alleles as well as pathogenic variants present in the general population. Significant enrichment in structural variants involving gene clusters related to keratinization and epidermal growth suggest that evolutive mechanisms have developed towards climate and environmental adaptation. IRPVS can be accessed at http://irpvs.clinbioinfosspa.es/ AUTHOR SUMMARYReference data is necessary for the correct interpretation of genetic studies. Although most genetic variants are present in all populations, ancestry has an important impact in the genetic background. For that reason databases of genetic variant in populations are developed specifically for different ethnicities, being an important tool for genetic diagnosis. The Roma are the most numerous ethnic minority in Europe. In this study we have collected samples from healthy Roma individuals from Iberian descent and implemented a database of genetic variant to facilitate genetic diagnosis in this population. Analysis of structural variants that are specific to the Iberian Roma not found in other healthy population for which genetic data are available suggest evolution towards environmental adaptation.\nIdentifying X-Chromosome Variants Associated with Age-Related Macular Degeneration\nAuthors: Grunin, M.; Igo, R. P.; Song, Y. E.; Blanton, S. H.; Pericak-Vance, M. A.; Haines, J. L.; International Age-related Macular Degeneration Genomics Consortium, Score: 3.9, Published: 2023-08-31 DOI: 10.1101/2023.08.28.23294688\nPurposeIn genome-wide association studies (GWAS), X chromosome (ChrX) variants are often not investigated. Sex-specific effects and ChrX-specific quality control (QC) are needed to examine these effects. Previous analyses identified 52 autosomal variants associated with age-related macular degeneration (AMD) via the International AMD Genomics Consortium (IAMDGC), but did not analyze ChrX. Therefore, our goal was to investigate ChrX variants for association with AMD. MethodsWe genotyped 29,629 non-Hispanic White (NHW) individuals (M/F:10,404/18,865; AMD12,087/14723) via a custom chip and imputed after ChrX-specific QC (XWAS 3.0) using the Michigan Imputation Server. Imputation generated 1,221,623 variants on ChrX. Age, informative PCs, and subphenotyeps were covariates for logistic association analyses with Fishers correction. Gene/pathway analyses were performed with VEGAS, GSEASNP, ICSNPathway, DAVID, and mirPath. ResultsLogistic association on NHW individuals with sex correction, identified variants in/near the genes SLITRK4, ARHGAP6, FGF13 and DMD associated with AMD (P\u003c1x10-6,Fishers combined-corrected). Via association testing of subphenotypes of choroidal neovascularization and geographic atrophy (GA), variants in DMD associated with GA (P\u003c1x10-6, Fishers combined-corrected). Via gene-based analysis with VEGAS, several genes were associated with AMD (P\u003c0.05, both truncated tail strength/truncated product P) including SLITRK4 and BHLHB9. Pathway analysis using GSEASNP and DAVID showed genes associated with nervous system development (FDR: P:0.02), and blood coagulation (FDR: P:0.03). Variants in the region of a microRNA (miR) were associated with AMD (P\u003c0.05, truncated tail strength/truncated product P). Via DIANA mirPath analysis, downstream targets of miRs show association with brain disorders and fatty acid elongation (P\u003c0.05). A long-non coding RNA on ChrX near the DMD locus was also associated with AMD (P=4x10-7). Epistatic analysis (t-statistic) for a quantitative trait of AMD vs control including covariates found a suggestive association in the XG gene (P=2x10^-5). ConclusionsAnalysis of ChrX variants demonstrates association with AMD and these variants may be linked to novel pathways. Further analysis is needed to confirm results and to understand their biological significance and relationship with AMD development in worldwide populations.\nRare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank\nAuthors: Fenner, E.; Holmans, P.; O'Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.\nScore: 12.8, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294074\nImpairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.\nSmoking during pregnancy and its effect on placental weight: A Mendelian randomization study\nAuthors: Jaitner, A.; Vaudel, M.; Tsaneva-Atanasova, K.; Njolstad, P. R.; Jacobsson, B.; Bowden, J.; Johansson, S.; Freathy, R.\nScore: 7.7, Published: 2023-08-25 DOI: 10.1101/2023.08.24.23294537\nBackgroundThe causal relationship between maternal smoking in pregnancy and reduced offspring birth weight is well established and is likely due to impaired placental function. However, observational studies have given conflicting results on the association between smoking and placental weight. We aimed to estimate the causal effect of newly pregnant mothers quitting smoking on their placental weight at the time of delivery. MethodsWe used one-sample Mendelian randomization, drawing data from the Avon Longitudinal Study of Parents and Children (ALSPAC) (up to N = 805) and the Norwegian Mother, Father and Child Cohort Study (MoBa) (up to N = 4475). The analysis was performed in pre-pregnancy smokers only, due to the specific role of the genetic instrument SNP rs1051730 (CHRNA5 - CHRNA3 - CHRNB4) in affecting smoking cessation but not initiation. ResultsFixed effect meta-analysis showed a 175 g [95%CI: 16, 334] higher placental weight for pre-pregnancy smoking mothers who continued smoking at the beginning of pregnancy, compared with those who stopped smoking. Using the number of cigarettes smoked per day in the first trimester as the exposure, the causal estimate was a 12 g [95%CI: 2,22] higher placental weight per cigarette per day. Results were similar when the smoking exposures were measured at the end of pregnancy. Using the residuals of birth weight regressed on placental weight as the outcome, we showed weak evidence of lower offspring birth weight relative to the placental weight for continuing smoking. ConclusionOur results suggest that continued smoking during pregnancy causes higher placental weights. Key MessagesO_LIIt is well known that maternal smoking in pregnancy causes a lower birth weight on average, but the relationship between maternal smoking and placental weight is less clear, with observational studies showing conflicting results. C_LIO_LIOur Mendelian randomization study suggests that for pre-pregnancy smokers, continuing smoking during pregnancy causes higher placental weight at term than quitting smoking. C_LIO_LIOur study also suggests that a greater number of cigarettes smoked per day during pregnancy causes a larger placental weight at term. C_LIO_LIA possible explanation for our findings is that the placenta grows larger in mothers who smoke during pregnancy to compensate for the lower oxygen availability, but further work is needed to confirm and further investigate this hypothesis. C_LI\nAnalysis of rare variants in 470,000 exome-sequenced UK Biobank participants implicates novel genes affecting risk of hypertension\nAuthors: Curtis, D.\nScore: 1.2, Published: 2023-09-04 DOI: 10.1101/2023.09.03.23294987\nBackground A previous study of 200,000 exome-sequenced UK Biobank participants to test for association of rare coding variants with hypertension implicated two genes at exome-wide significance, DNMT3A and FES. A total of 42 genes had an uncorrected p value \u003c 0.001. These results were followed up in a larger sample of 470,000 exome-sequenced participants. Methods and Results Weighted burden analysis of rare coding variants in a new sample of 97,050 cases and 172,263 controls was carried out for these 42 genes. Those showing evidence for association were then analysed in the combined sample of 167,127 cases and 302,691 controls. The association of DNMT3A and FES with hypertension was replicated in the new sample and they and the previously implicated gene NPR1 were all exome-wide significant in the combined sample. Also exome-wide significant as risk genes for hypertension were GUCY1A1, ASXL1 and SMAD6, while GUCY1B1 had a nominal p value of \u003c 0.0001. For two genes, INPPL1 and DBH, rare coding variants predicted to impair gene function were protective against hypertension, again with exome-wide significance. Conclusions The findings offer new insights into biological risk factors for hypertension which could be the subject of further investigation. In particular, genetic variants predicted to impair the function of either membrane-bound guanylate cyclase, activated by natriuretic peptides, or soluble guanylate cyclase, activated by nitric oxide, increase risk of hypertension. Conversely, variants impairing the function of dopamine beta hydroxylase, responsible for the synthesis of norepinephrine, reduce hypertension risk. This research has been conducted using the UK Biobank Resource.\nPrioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach\nAuthors: Pena-Martinez, E. G.; Pomales-Matos, D. A.; Rivera-Madera, A.; Messon-Bird, J. L.; Medina-Feliciano, J. G.; Sanabria-Alberto, L.; Barreiro-Rosario, A. C.; Rodriguez-Rios, J. M.; Rodriguez-Martinez, J. A.\nScore: 1.2, Published: 2023-09-02 DOI: 10.1101/2023.09.01.23294951\nCardiovascular diseases (CVDs) are the leading cause of death worldwide and are heavily influenced by genetic factors. Genome-wide association studies (GWAS) have mapped \u003e 90% of CVD-associated variants within the non-coding genome, which can alter the function of regulatory proteins, like transcription factors (TFs). However, due to the overwhelming number of GWAS single nucleotide polymorphisms (SNPs) (\u003e500,000), prioritizing variants for in vitro analysis remains challenging. In this work, we implemented a computational approach that considers support vector machine (SVM)-based TF binding site classification and cardiac expression quantitative trait loci (eQTL) analysis to identify and prioritize potential CVD-causing SNPs. We identified 1,535 CVD-associated SNPs that occur within human heart footprints/enhancers and 9,309 variants in linkage disequilibrium (LD) with differential gene expression profiles in cardiac tissue. Using hiPSC-CM ChIP-seq data from NKX2-5 and TBX5, two cardiac TFs essential for proper heart development, we trained a large-scale gapped k-mer SVM (LS-GKM-SVM) predictive model that can identify binding sites altered by CVD-associated SNPs. The computational predictive model was tested by scoring human heart footprints and enhancers in vitro through electrophoretic mobility shift assay (EMSA). Three variants (rs59310144, rs6715570, and rs61872084) were prioritized for in vitro validation based on their eQTL in cardiac tissue and LS-GKM-SVM prediction to alter NKX2-5 DNA binding. All three variants altered NKX2-5 DNA binding. In summary, we present a bioinformatic approach that considers tissue-specific eQTL analysis and SVM-based TF binding site classification to prioritize CVD-associated variants for in experimental analysis.\n",
  "wordCount" : "2986",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.23294497">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.23294497" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.23294497">
        <p class="paperTitle">Investigating misclassification of type 1 diabetes in a population-based cohort of British Pakistanis and Bangladeshis using polygenic risk scores</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.23294497" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.23294497" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, T.; Sankareswaran, A.; Paterson, G.; Genes &amp; Health Research Team,  ; Fraser, D. P.; Hodgson, S.; Huang, Q. Q.; Heng, T. H.; Ladwa, M.; Thomas, N.; van Heel, D. A.; Weedon, M. N.; Yajnik, C. S.; Oram, R. A.; Chandak, G. R.; Martin, H. C.; Finer, S.</p>
        <p class="info">Score: 23.6, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.23294497' target='https://doi.org/10.1101/2023.08.23.23294497'> 10.1101/2023.08.23.23294497</a></p>
        <p class="abstract">ObjectiveCorrect classification of type 1 (T1D) and type 2 diabetes (T2D) is challenging due to overlapping clinical features and the increasingly early onset of T2D, particularly in South Asians. We used polygenic risk scores (PRSs) in a British Bangladeshi and Pakistani population with diabetes to estimate the proportion and misclassification rate of T1D in insulin-treated individuals with ambiguous features.

Research Design and MethodsUsing linked health record from the Genes &amp; Health cohort (n=38,344) we defined four groups: 31 T1D cases, 1,842 T2D cases, and after excluding these, 839 insulin-treated individuals with ambiguous features and 5,174 controls. Combining these with 307 confirmed T1D cases and 307 controls from India, we calculated ancestry-corrected PRSs for T1D and T2D, with which we estimated the proportion of T1D cases within the ambiguous group and evaluated misclassification.

ResultsWe estimated that the prevalence of T1D was [~]6% within the ambiguous group, or [~]4.5% within the subset who had T2D codes in their health records. We saw no significant association between the T1D or T2D PRS and BMI at diagnosis, time to insulin, or the presence of T1D or T2D diagnostic codes amongst the T2D or ambiguous cases.

ConclusionsWe estimate that 4.5% of British Pakistanis and Bangladeshis with diabetes who are treated with insulin and have ambiguous clinical features have been classified incorrectly in their health records, and in fact have T1D. These findings have significant implications for the delivery of optimal clinical care at population level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.23294848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.23294848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.23294848">
        <p class="paperTitle">The pivotal role of the X-chromosome in the genetic architecture of the human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.23294848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.23294848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Z.; Sullivan, P. F.; Li, T.; Zhao, B.; Wang, X.; Luo, T.; Huang, S.; Guan, P. Y.; Chen, J.; Yang, Y.; Stein, J. L.; Li, Y.; Liu, D.; Sun, L.; Zhu, H.</p>
        <p class="info">Score: 5.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.23294848' target='https://doi.org/10.1101/2023.08.30.23294848'> 10.1101/2023.08.30.23294848</a></p>
        <p class="abstract">Genes on the X-chromosome are extensively expressed in the human brain, resulting in substantial influences on brain development, intellectual disability, and other brain-related disorders. To comprehensively investigate the X-chromosome&#39;s impact on the cerebral cortex, white matter tract microstructures, and intrinsic and extrinsic brain functions, we examined 2,822 complex brain imaging traits obtained from n = 34,000 subjects in the UK Biobank. We unveiled potential autosome-X-chromosome interaction, while proposing an atlas of dosage compensation (DC) for each set of traits. We observed a pronounced X-chromosome impact on the corticospinal tract and the functional amplitude and connectivity of visual networks. In association studies, we identified 50 genome-wide significant trait-locus pairs enriched in Xq28, 22 of which replicated in independent datasets (n = 4,900). Notably, 13 newly identified pairs were in the X-chromosome&#39;s non-pseudo-autosomal regions (NPR). The volume of the right ventral diencephalon shared genetic architecture with schizophrenia and educational attainment in a locus indexed by rs2361468 (located ~3kb upstream of PJA1, a conserved and ubiquitously expressed gene implicated in multiple psychiatric disorders). No significant associations were identified in the pseudo-autosomal regions (PAR) or the Y-chromosome. Finally, we explored sex-specific associations on the X-chromosome and compared differing genetic effects between sexes. We found much more associations can be identified in males (33 versus 9) given a similar sample size. In conclusion, our research provides invaluable insights into the X-chromosome&#39;s role in the human brain, contributing to the observed sex differences in brain structure and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.23294773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.23294773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.23294773">
        <p class="paperTitle">Relationship between sex biases in gene expression and sex biases in autism and Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.23294773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.23294773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.</p>
        <p class="info">Score: 5.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.23294773' target='https://doi.org/10.1101/2023.08.29.23294773'> 10.1101/2023.08.29.23294773</a></p>
        <p class="abstract">Sex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimer&#39;s disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.23294769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.23294769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.23294769">
        <p class="paperTitle">The predictive capacity of polygenic risk scores for disease risk is only moderately influenced by imputation panels tailored to the target population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.23294769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.23294769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levi, H.; Elkon, R.; Shamir, R.</p>
        <p class="info">Score: 4.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.23294769' target='https://doi.org/10.1101/2023.08.29.23294769'> 10.1101/2023.08.29.23294769</a></p>
        <p class="abstract">Polygenic risk scores (PRS) predict individuals genetic risk of developing complex diseases. They summarize the effect of many genetic variants discovered in genome-wide association studies (GWASs). However, to date, large GWASs exist primarily for the European population and the quality of PRS prediction declines when applied to target sets of other ethnicities. A key step in using a PRS is imputation, which is the inference of un-typed SNPs using a set of fully-sequenced individuals, called the imputation panel. The SNP genotypes called by the imputation process depend on the ethnic composition of the imputation panel. Several studies have shown that imputing genotypes using a panel that contains individuals of the same ethnicity as the genotyped individuals improves imputation accuracy. However, until now, there has been no systematic investigation into the influence of the ethnic composition of imputation panels on the accuracy of PRS predictions when applied to ethnic groups that differ from the population used in the GWAS.

In this study we estimated the effect of imputation of the target set on prediction accuracy of PRS when the discovery (GWAS) and the target sets come from different ethnic groups. We analyzed twelve binary phenotypes and three populations from the UK Biobank (Europeans, South-Asians, and Africans). We generated imputation panels from several ethnic groups, imputed the target set using each panel, and generated PRS to compute individuals risk scores. Then, we assessed the prediction accuracy obtained from each imputation panel. Our analysis indicates that using an imputation panel matched to the ethnic group of the target population yields only a marginal improvement and only under specific conditions. Hence, while a target-matched imputation panel can potentially improve prediction accuracy of European PRSs in non-EUR populations, the improvement is limited.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.23294490">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.23294490" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.23294490">
        <p class="paperTitle">The Iberian Roma genetic variant server; population structure, susceptibility to disease and adaptive traits.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.23294490" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.23294490" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mavillard, F.; Perez-Florido, J.; Ortuno, F.; Valladares, A.; Alvarez-Villegas, M. L.; Roldan, G.; Carmona, R.; Soriano, M.; Susarte, S.; Fuentes, P.; Lopez-Lopez, D.; Nunez-Negrillo, A. M.; Carvajal, A.; Morgado, Y.; Arteaga, D.; Ufano, R.; Mir, P.; Gamella, J.; Dopazo, J.; Paradas, c.; Cabrera Serrano, M.</p>
        <p class="info">Score: 13.9, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.23294490' target='https://doi.org/10.1101/2023.08.25.23294490'> 10.1101/2023.08.25.23294490</a></p>
        <p class="abstract">The Roma are the most numerous ethnic minority in Europe. The Iberian Roma arrived in the Iberian Peninsula five centuries ago and still today, they keep a strong group identity. Demographic and cultural reasons lie behind a high rate of Mendelian disease often related to founder variants. We have analysed exome data from 119 Iberian Roma individuals collected from 2018 to 2020. A database of variant frequency has been implemented (IRPVS) and made available online. We have analysed the carrier rate of founder private alleles as well as pathogenic variants present in the general population. Significant enrichment in structural variants involving gene clusters related to keratinization and epidermal growth suggest that evolutive mechanisms have developed towards climate and environmental adaptation. IRPVS can be accessed at http://irpvs.clinbioinfosspa.es/

AUTHOR SUMMARYReference data is necessary for the correct interpretation of genetic studies. Although most genetic variants are present in all populations, ancestry has an important impact in the genetic background. For that reason databases of genetic variant in populations are developed specifically for different ethnicities, being an important tool for genetic diagnosis. The Roma are the most numerous ethnic minority in Europe. In this study we have collected samples from healthy Roma individuals from Iberian descent and implemented a database of genetic variant to facilitate genetic diagnosis in this population. Analysis of structural variants that are specific to the Iberian Roma not found in other healthy population for which genetic data are available suggest evolution towards environmental adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.23294688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.23294688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.23294688">
        <p class="paperTitle">Identifying X-Chromosome Variants Associated with Age-Related Macular Degeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.23294688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.23294688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grunin, M.; Igo, R. P.; Song, Y. E.; Blanton, S. H.; Pericak-Vance, M. A.; Haines, J. L.; International Age-related Macular Degeneration Genomics Consortium,  </p>
        <p class="info">Score: 3.9, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.23294688' target='https://doi.org/10.1101/2023.08.28.23294688'> 10.1101/2023.08.28.23294688</a></p>
        <p class="abstract">PurposeIn genome-wide association studies (GWAS), X chromosome (ChrX) variants are often not investigated. Sex-specific effects and ChrX-specific quality control (QC) are needed to examine these effects. Previous analyses identified 52 autosomal variants associated with age-related macular degeneration (AMD) via the International AMD Genomics Consortium (IAMDGC), but did not analyze ChrX. Therefore, our goal was to investigate ChrX variants for association with AMD.

MethodsWe genotyped 29,629 non-Hispanic White (NHW) individuals (M/F:10,404/18,865; AMD12,087/14723) via a custom chip and imputed after ChrX-specific QC (XWAS 3.0) using the Michigan Imputation Server. Imputation generated 1,221,623 variants on ChrX. Age, informative PCs, and subphenotyeps were covariates for logistic association analyses with Fishers correction. Gene/pathway analyses were performed with VEGAS, GSEASNP, ICSNPathway, DAVID, and mirPath.

ResultsLogistic association on NHW individuals with sex correction, identified variants in/near the genes SLITRK4, ARHGAP6, FGF13 and DMD associated with AMD (P&lt;1x10-6,Fishers combined-corrected). Via association testing of subphenotypes of choroidal neovascularization and geographic atrophy (GA), variants in DMD associated with GA (P&lt;1x10-6, Fishers combined-corrected). Via gene-based analysis with VEGAS, several genes were associated with AMD (P&lt;0.05, both truncated tail strength/truncated product P) including SLITRK4 and BHLHB9. Pathway analysis using GSEASNP and DAVID showed genes associated with nervous system development (FDR: P:0.02), and blood coagulation (FDR: P:0.03). Variants in the region of a microRNA (miR) were associated with AMD (P&lt;0.05, truncated tail strength/truncated product P). Via DIANA mirPath analysis, downstream targets of miRs show association with brain disorders and fatty acid elongation (P&lt;0.05). A long-non coding RNA on ChrX near the DMD locus was also associated with AMD (P=4x10-7). Epistatic analysis (t-statistic) for a quantitative trait of AMD vs control including covariates found a suggestive association in the XG gene (P=2x10^-5).

ConclusionsAnalysis of ChrX variants demonstrates association with AMD and these variants may be linked to novel pathways. Further analysis is needed to confirm results and to understand their biological significance and relationship with AMD development in worldwide populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294074">
        <p class="paperTitle">Rare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fenner, E.; Holmans, P.; O&#39;Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.</p>
        <p class="info">Score: 12.8, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294074' target='https://doi.org/10.1101/2023.08.14.23294074'> 10.1101/2023.08.14.23294074</a></p>
        <p class="abstract">Impairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.23294537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.23294537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.23294537">
        <p class="paperTitle">Smoking during pregnancy and its effect on placental weight: A Mendelian randomization study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.23294537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.23294537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jaitner, A.; Vaudel, M.; Tsaneva-Atanasova, K.; Njolstad, P. R.; Jacobsson, B.; Bowden, J.; Johansson, S.; Freathy, R.</p>
        <p class="info">Score: 7.7, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.23294537' target='https://doi.org/10.1101/2023.08.24.23294537'> 10.1101/2023.08.24.23294537</a></p>
        <p class="abstract">BackgroundThe causal relationship between maternal smoking in pregnancy and reduced offspring birth weight is well established and is likely due to impaired placental function. However, observational studies have given conflicting results on the association between smoking and placental weight. We aimed to estimate the causal effect of newly pregnant mothers quitting smoking on their placental weight at the time of delivery.

MethodsWe used one-sample Mendelian randomization, drawing data from the Avon Longitudinal Study of Parents and Children (ALSPAC) (up to N = 805) and the Norwegian Mother, Father and Child Cohort Study (MoBa) (up to N = 4475). The analysis was performed in pre-pregnancy smokers only, due to the specific role of the genetic instrument SNP rs1051730 (CHRNA5 - CHRNA3 - CHRNB4) in affecting smoking cessation but not initiation.

ResultsFixed effect meta-analysis showed a 175 g [95%CI: 16, 334] higher placental weight for pre-pregnancy smoking mothers who continued smoking at the beginning of pregnancy, compared with those who stopped smoking. Using the number of cigarettes smoked per day in the first trimester as the exposure, the causal estimate was a 12 g [95%CI: 2,22] higher placental weight per cigarette per day. Results were similar when the smoking exposures were measured at the end of pregnancy. Using the residuals of birth weight regressed on placental weight as the outcome, we showed weak evidence of lower offspring birth weight relative to the placental weight for continuing smoking.

ConclusionOur results suggest that continued smoking during pregnancy causes higher placental weights.

Key MessagesO_LIIt is well known that maternal smoking in pregnancy causes a lower birth weight on average, but the relationship between maternal smoking and placental weight is less clear, with observational studies showing conflicting results.
C_LIO_LIOur Mendelian randomization study suggests that for pre-pregnancy smokers, continuing smoking during pregnancy causes higher placental weight at term than quitting smoking.
C_LIO_LIOur study also suggests that a greater number of cigarettes smoked per day during pregnancy causes a larger placental weight at term.
C_LIO_LIA possible explanation for our findings is that the placenta grows larger in mothers who smoke during pregnancy to compensate for the lower oxygen availability, but further work is needed to confirm and further investigate this hypothesis.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.23294987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.23294987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.23294987">
        <p class="paperTitle">Analysis of rare variants in 470,000 exome-sequenced UK Biobank participants implicates novel genes affecting risk of hypertension</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.23294987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.23294987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Curtis, D.</p>
        <p class="info">Score: 1.2, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.23294987' target='https://doi.org/10.1101/2023.09.03.23294987'> 10.1101/2023.09.03.23294987</a></p>
        <p class="abstract">Background A previous study of 200,000 exome-sequenced UK Biobank participants to test for association of rare coding variants with hypertension implicated two genes at exome-wide significance, DNMT3A and FES. A total of 42 genes had an uncorrected p value &lt; 0.001. These results were followed up in a larger sample of 470,000 exome-sequenced participants. Methods and Results Weighted burden analysis of rare coding variants in a new sample of 97,050 cases and 172,263 controls was carried out for these 42 genes. Those showing evidence for association were then analysed in the combined sample of 167,127 cases and 302,691 controls. The association of DNMT3A and FES with hypertension was replicated in the new sample and they and the previously implicated gene NPR1 were all exome-wide significant in the combined sample. Also exome-wide significant as risk genes for hypertension were GUCY1A1, ASXL1 and SMAD6, while GUCY1B1 had a nominal p value of &lt; 0.0001. For two genes, INPPL1 and DBH, rare coding variants predicted to impair gene function were protective against hypertension, again with exome-wide significance. Conclusions The findings offer new insights into biological risk factors for hypertension which could be the subject of further investigation. In particular, genetic variants predicted to impair the function of either membrane-bound guanylate cyclase, activated by natriuretic peptides, or soluble guanylate cyclase, activated by nitric oxide, increase risk of hypertension. Conversely, variants impairing the function of dopamine beta hydroxylase, responsible for the synthesis of norepinephrine, reduce hypertension risk. This research has been conducted using the UK Biobank Resource.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.23294951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.23294951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.23294951">
        <p class="paperTitle">Prioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.23294951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.23294951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pena-Martinez, E. G.; Pomales-Matos, D. A.; Rivera-Madera, A.; Messon-Bird, J. L.; Medina-Feliciano, J. G.; Sanabria-Alberto, L.; Barreiro-Rosario, A. C.; Rodriguez-Rios, J. M.; Rodriguez-Martinez, J. A.</p>
        <p class="info">Score: 1.2, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.23294951' target='https://doi.org/10.1101/2023.09.01.23294951'> 10.1101/2023.09.01.23294951</a></p>
        <p class="abstract">Cardiovascular diseases (CVDs) are the leading cause of death worldwide and are heavily influenced by genetic factors. Genome-wide association studies (GWAS) have mapped &gt; 90% of CVD-associated variants within the non-coding genome, which can alter the function of regulatory proteins, like transcription factors (TFs). However, due to the overwhelming number of GWAS single nucleotide polymorphisms (SNPs) (&gt;500,000), prioritizing variants for in vitro analysis remains challenging. In this work, we implemented a computational approach that considers support vector machine (SVM)-based TF binding site classification and cardiac expression quantitative trait loci (eQTL) analysis to identify and prioritize potential CVD-causing SNPs. We identified 1,535 CVD-associated SNPs that occur within human heart footprints/enhancers and 9,309 variants in linkage disequilibrium (LD) with differential gene expression profiles in cardiac tissue. Using hiPSC-CM ChIP-seq data from NKX2-5 and TBX5, two cardiac TFs essential for proper heart development, we trained a large-scale gapped k-mer SVM (LS-GKM-SVM) predictive model that can identify binding sites altered by CVD-associated SNPs. The computational predictive model was tested by scoring human heart footprints and enhancers in vitro through electrophoretic mobility shift assay (EMSA). Three variants (rs59310144, rs6715570, and rs61872084) were prioritized for in vitro validation based on their eQTL in cardiac tissue and LS-GKM-SVM prediction to alter NKX2-5 DNA binding. All three variants altered NKX2-5 DNA binding. In summary, we present a bioinformatic approach that considers tissue-specific eQTL analysis and SVM-based TF binding site classification to prioritize CVD-associated variants for in experimental analysis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
