<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand
Authors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand
Authors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand
Authors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand\nAuthors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D.",
  "keywords": [
    
  ],
  "articleBody": " A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand\nAuthors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D.; Amnueypol, M.; Krittayaphong, R.; Prechawat, S.; Anannab, A.; Makarawate, P.; Ngarmukos, T.; Veerakul, G.; Kingsbury, Z.; Newington, T.; Maheswari, U.; Ross, M. T.; Grace, A.; Lambiase, P. D.; Behr, E. R.; Schott, J.-J.; Redon, R.; Barc, J.\nScore: 12.9, Published: 2023-12-20 DOI: 10.1101/2023.12.19.23299785\nBrugada syndrome (BrS) is a cardiac arrhythmia disorder that causes sudden death in young adults. Rare genetic variants in the SCN5A gene, encoding the Nav1.5 sodium channel, and common non-coding variants at this locus, are robustly associated with the condition. BrS is particularly prevalent in Southeast Asia but the underlying ancestry-specific factors remain largely unknown. Here, we performed genome sequencing of BrS probands from Thailand and population-matched controls and identified a rare non-coding variant in an SCN5A intronic enhancer that is highly enriched in BrS cases (3.9% in cases, odds ratio 20.2-45.2) and predicted to disrupt a Mef2 transcription factor binding site. Heterozygous introduction of the enhancer variant in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) caused significantly reduced SCN5A expression from the variant-containing allele and a 30% reduction in Nav1.5-mediated sodium-current density compared to isogenic controls. This is the first example of a validated rare non-coding variant at the SCN5A locus and partly explains the increased prevalence of BrS in this geographic region.\nPolygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.\nAuthors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.\nScore: 5.1, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412\nIntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups. Very few studies have looked at the transferability of European-derived PRS for behavioural and mental health phenotypes to non-Europeans. We assessed the comparative accuracy of PRS for Major Depressive Disorder (MDD) trained on European and African Ancestries GWAS studies to predict MDD and related traits in African Ancestries participants from the UK Biobank. MethodsUK Biobank participants were selected based on Principal component analysis (PCA) clustering with an African genetic similarity reference population and MDD was assessed with the Composite International Diagnostic Interview (CIDI). Polygenic Risk Scores (PRS) were computed using PRSice2 using either European or African Ancestries GWAS summary statistics. ResultsPRS trained on European ancestry samples (246,363 cases) predicted case control status in Africans of the UK Biobank with similar accuracies (190 cases, R2=2%) to PRS trained on far much smaller samples of African Ancestries participants from 23andMe, Inc. (5045 cases, R2=1.8%). This suggests that prediction of MDD status from Africans to Africans had greater efficiency per unit increase in the discovery sample size than prediction of MDD from Europeans to Africans. Prediction of MDD status in African UK Biobank participants using GWAS findings of causal risk factors from European ancestries was non-significant. ConclusionGWAS studies of MDD in European ancestries are an inefficient means of improving polygenic prediction accuracy in African samples.\nRare disease gene association discovery from burden analysis of the 100,000 Genomes Project data\nAuthors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O'Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra\nScore: 5.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300294\nTo discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.\nImputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations\nAuthors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences, ; Wood, C. R.; Kriegl, J. M.; Podduturi, N.; Jensen, J. N.; Stutzki, J.; Ding, Z.\nScore: 27.3, Published: 2023-12-22 DOI: 10.1101/2023.12.20.23300308\nAdvancements in long-read sequencing technology have accelerated the study of large structural variants (SVs). We created a curated, publicly available, multi-ancestry SV imputation panel by long-read sequencing 888 samples from the 1000 Genomes Project. This high-quality panel was used to impute SVs in approximately 500,000 UK Biobank participants. We demonstrated the feasibility of conducting genome-wide SV association studies at biobank scale using 32 disease-relevant phenotypes related to respiratory, cardiometabolic and liver diseases, in addition to 1,463 protein levels. This analysis identified thousands of genome-wide significant SV associations, including hundreds of conditionally independent signals, thereby enabling novel biological insights. Focusing on genetic association studies of lung function as an example, we demonstrate the added value of SVs for prioritising causal genes at gene-rich loci compared to traditional GWAS using only short variants. We envision that future post-GWAS gene-prioritisation workflows will incorporate SV analyses using this SV imputation panel and framework.\nIdentification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis\nAuthors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.\nScore: 25.4, Published: 2023-12-14 DOI: 10.1101/2023.12.13.23299893\nClonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.\nProgress Update for the Multisite Optical Genome Mapping Evaluation and Validation Study: Prenatal Applications\nAuthors: Levy, B.; Liu, J.; Iqbal, A.; Dupont, B.; Sahajpal, N.; Ho, M.; Yu, J.; Brody, S.; Mason-Suares, H.; Ganapathi, M.; Rajkovic, A.; Smolarek, T.; Toydemir, R.; Bui, P.; Kolhe, R.; Stevenson, R.\nScore: 1.9, Published: 2023-12-26 DOI: 10.1101/2023.12.22.23300469\nOptical genome mapping (OGM) is an emerging technology with great potential for prenatal diagnosis. OGM can identify and resolve all types of balanced and unbalanced cytogenomic abnormalities in a single test, which are typically assessed by multiple standard of care (SOC) methods including karyotyping, fluorescence in situ hybridization and chromosomal microarray. To assess OGMs viability as an alternative to conventional SOC testing, a comprehensive clinical research study was conducted across multiple sites, operators, and instruments to evaluate its accuracy and clinical utility. This report provides an update for the phase 2 results of the ongoing multisite evaluation and validation study evaluating OGM for prenatal applications. In phase 1, 123 prenatal cases were assessed by OGM, and in phase 2, 219 retrospective and prospective prenatal cases have been evaluated. For 71% of cases, at least two SOC tests were performed. The study found that OGM had an overall accuracy of 99.6% and positive predictive value of 100% when compared to all cytogenetic SOC results. With its standardized workflow, cost-effectiveness, and high-resolution cytogenomic analysis, OGM shows great promise as an alternative technology that uses a single assay to consolidate the multiple SOC tests usually used for prenatal cytogenetic diagnosis.\nGenome-wide characterization of 54 urinary metabolites reveals molecular impact of kidney function\nAuthors: Valo, E.; Richmond, A.; Mutter, S.; Campbell, A.; Porteous, D.; Wilson, J. F.; FinnDiane Study Group, ; Groop, P.-H.; Hayward, C.; Sandholm, N.\nScore: 2.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300206\nDissecting the genetic mechanisms underlying urinary metabolite concentrations can provide molecular insights into kidney function and open possibilities for causal assessment of urinary metabolites with risk factors and disease outcomes. Proton nuclear magnetic resonance metabolomics provides a high-throughput means for urinary metabolite profiling, as widely applied for blood biomarker studies. Here we report a genome-wide association study meta-analysed for 3 European cohorts comprising 8,026 individuals, covering both people with type 1 diabetes and general population settings. We identified 52 associations (p\u003c9.3x10-10) for 19 of 54 studied metabolite concentrations. Out of these, 32 were not reported previously for relevant urinary or blood metabolite traits. Subsequent two-sample Mendelian randomization analysis suggests that estimated glomerular filtration rate (eGFR) causally affects 13 urinary metabolite concentrations whereas urinary ethanolamine, an initial precursor for phosphatidylcholine and phosphatidylethanolamine, was associated with higher eGFR lending support for a potential protective role.\nLoss of coactosin-like F-actin binding protein 1 (Cotl1) decreases platelet-mediated osteoclastogenesis and causes osteopetrosis phenotypes in mouse\nAuthors: Park, E.; Yun, S.-H.; Jin, H.-S.; Lee, C.-G.; Lee, S.-H.; Choi, S.-Y.; Woo, H. G.; Lim, J. E.; Oh, B.; Jeong, S.-Y.\nScore: 1.5, Published: 2023-12-24 DOI: 10.1101/2023.12.20.23300337\nBACKGROUNDSOsteopetrosis, a rare skeletal disease, is characterized by an increased bone mass resulting from impaired bone remodeling process. Platelet is the major bone-healing blood component involved in the regulation of bone resorption, particularly in the removal of compromised bones. Several actin-associated proteins contribute to the orchestration of actin ring formation in osteoclasts closely related to bone resorption. However, the role of coactosin-like F-actin binding protein 1 (Cotl1) in actin ring formation and platelet-mediated bone resorption mechanisms remains unclear. METHODSWhole-mount in situ RNA hybridization was performed to detect cotl1 expression pattern in zebrafish. cotl1 gene knockdown zebrafish using morpholino oligonucleotides and platelet marker-expressing transgenic zebrafish were investigated for finding the phenotypic clues. Cotl1 knockout (Cotl1-/-) mice were generated using Cre/loxP recombination systems. In silico network analysis of the differentially expressed genes between bone marrow samples of wild type and Cotl1-/- mice was conducted. Primary-cultured monocytes from Cotl1-/- mice were examined for osteoclast differentiation and mRNA and protein expression patterns. Cotl1-/- mice underwent hematological examination and bone phenotype assessments including micro-CT, bone density, histology, immunohistochemistry, electron microscopy, and mechanical testing. Genetic association of SNPs in human COTL1 gene with estimated bone mineral density was analyzed. RESULTSZebrafish cotl1 mRNA was highly expressed in the caudal hematopoietic tissue region. Knockdown of cotl1 in zebrafish embryos decreased the expression of c-myb, a marker of hematopoietic stem cells (HSCs). Notably, the platelet receptor CD41 was reduced in the HSCs of cotl1-depleted zebrafish and Cotl1-/- mice showed reduced platelet production with platelet surface markers of CD41 and CD61. Significantly reduced osteoclast differentiation and bone resorption pit, and impaired actin ring formation were observed in the primary myocytes from Cotl1-/- mice. Structural and histological analyses of the femur revealed sclerotic bone phenotypes in Cotl1-/- mice. Mechanical assessment of Cotl1-/- mouse femoral bones revealed osteopetrotic phenotypes. Association analysis of genetic variants in COTL1 gene in subjects from the UK Biobank suggested that COTL1 is susceptible to bone density in humans. CONCLUSIONSOur results provide insights into the role of Cotl1 in platelet-mediated osteoclastogenesis and the novel finding that the loss of Cotl1-/- mice causes osteopetrosis phenotypes. Clinical PerspectiveWhat Is New? O_LIDeficiency of Cotl1 decreased platelet production in zebrafish and mice. C_LIO_LIAbsence of Cotl1 disrupted the actin ring formation which is crucial for osteoclast differentiation in bone remodeling process. C_LIO_LICotl1 knockout mice displayed sclerotic bone phenotypes and increased bone density that are representative characteristics of osteopetrosis. C_LIO_LIGenetic variants in COTL1 gene in subjects from the UK Biobank are significantly associated with bone density. C_LI What Are the Clinical Implications? O_LIThe current findings suggest that Cotl1 plays a fundamental role in platelet production-mediated osteoclastogenesis during bone remodeling, providing valuable insights into novel strategies for bone health maintenance. C_LIO_LICotl1 may be a promising target for novel therapeutic strategies for the treatment and/or prevention of impaired osteoclastogenesis-mediated bone diseases such as osteopetrosis and osteoporosis. C_LI\nGenome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health\nAuthors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.\nScore: 1.0, Published: 2023-12-29 DOI: 10.1101/2023.12.26.23300533\nNeuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field, 'brain age gap' has emerged as promising MRI-based biomarker quantifying the deviation between an individual's biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimer's disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.\nSEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population\nAuthors: Yang, M.-Y.; Zhong, J.-D.; Li, X.; Bai, W.-Y.; Yuan, C.-D.; Qiu, M.-C.; Liu, K.-Q.; Yu, C.-F.; Li, N.; Yang, J.-J.; Liu, Y.-H.; Yu, S.-H.; Zhao, W.-W.; Liu, J.-Q.; Sun, Y.; Cong, P.-K.; Khederzadeh, S.; Zhao, P.-P.; Qian, Y.; Guan, P.-L.; Gu, J.-X.; Gai, S.-R.; Yi, X.-J.; Tao, J.-G.; Chen, X.; Miao, M.-M.; Lei, L.-X.; Xu, L.; Xie, S.-Y.; Tian, G.; Li, J.-C.; Guo, J.-F.; Karasik, D.; Yang, L.; Tang, B.-S.; Huang, F.; Zheng, H.-F.\nScore: 1.0, Published: 2023-12-27 DOI: 10.1101/2023.12.23.23300480\nHere, we present the South and East Asian Reference Database (SEAD) reference panel (https://imputationserver.westlake.edu.cn/), which comprises whole genome sequencing data from 11,067 individuals across 17 countries in Asia. The SEAD panel, which excludes singleton variants, consists of 22,134 haplotypes and 80,367,720 variants. Firstly, we assessed the concordance rate in global populations using HGDP datasets, notably, the SEAD panel showed advantage in East Asia, Central and South Asia, and Oceania populations. When imputing the disease-associated variants of Asian population, the SEAD panel displayed a distinct preponderance in imputing low-frequency and rare variants. In imputation of Chinese population, the SEAD panel imputed a larger number of well-imputed sites across all minor allele frequency (MAF) bins. Additionally, the SEAD panel exhibited higher imputation accuracy for shared sites in all MAF bins. Finally, we applied the augmented SEAD panel to conduct a discovery and replication genome-wide association study (GWAS) for hip and femoral neck (FN) bone mineral density (BMD) traits within the 5,369 Westlake BioBank for Chinese (WBBC) samples. The single-variant test suggests that rare variants near SNTG1 gene are associated with hip BMD (rs60103302, MAF=0.0091, P=4.79x10-8). The spatial clustering analysis also suggests the association of this gene (Pslide_window=1.08x10-8, Pgene_centric=4.72x10-8). The gene and variants achieved a suggestive level for FN BMD. This gene was not reported previously, and the preliminary experiment demonstrated that the identified rare variant can upregulate the SNTG1 expression, which in turn inhibits the proliferation and differentiation of preosteoblast.\n",
  "wordCount" : "2904",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23299785">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23299785" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23299785">
        <p class="paperTitle">A rare non-coding enhancer variant in SCN5A contributes to the high prevalence of Brugada syndrome in Thailand</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23299785" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23299785" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walsh, R.; Mauleekoonphairoj, J.; Mengarelli, I.; Verkerk, A. O.; Bosada, F. M.; van Duijvenboden, K.; Poovorawan, Y.; Wongcharoen, W.; Sutjaporn, B.; Wandee, P.; Chimparlee, N.; Chokesuwattanaskul, R.; Vongpaisarnsin, K.; Dangkao, P.; Wu, C.-I.; Tadros, R.; Amin, A. S.; Lieve, K. V. V.; Postema, P. G.; Kooyman, M.; Beekman, L.; Phusanti, K.; Sahasatas, D.; Amnueypol, M.; Krittayaphong, R.; Prechawat, S.; Anannab, A.; Makarawate, P.; Ngarmukos, T.; Veerakul, G.; Kingsbury, Z.; Newington, T.; Maheswari, U.; Ross, M. T.; Grace, A.; Lambiase, P. D.; Behr, E. R.; Schott, J.-J.; Redon, R.; Barc, J.</p>
        <p class="info">Score: 12.9, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23299785' target='https://doi.org/10.1101/2023.12.19.23299785'> 10.1101/2023.12.19.23299785</a></p>
        <p class="abstract">Brugada syndrome (BrS) is a cardiac arrhythmia disorder that causes sudden death in young adults. Rare genetic variants in the SCN5A gene, encoding the Nav1.5 sodium channel, and common non-coding variants at this locus, are robustly associated with the condition. BrS is particularly prevalent in Southeast Asia but the underlying ancestry-specific factors remain largely unknown. Here, we performed genome sequencing of BrS probands from Thailand and population-matched controls and identified a rare non-coding variant in an SCN5A intronic enhancer that is highly enriched in BrS cases (3.9% in cases, odds ratio 20.2-45.2) and predicted to disrupt a Mef2 transcription factor binding site. Heterozygous introduction of the enhancer variant in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) caused significantly reduced SCN5A expression from the variant-containing allele and a 30% reduction in Nav1.5-mediated sodium-current density compared to isogenic controls. This is the first example of a validated rare non-coding variant at the SCN5A locus and partly explains the increased prevalence of BrS in this geographic region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.23300412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.23300412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.23300412">
        <p class="paperTitle">Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.23300412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.23300412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team,  ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.</p>
        <p class="info">Score: 5.1, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.23300412' target='https://doi.org/10.1101/2023.12.24.23300412'> 10.1101/2023.12.24.23300412</a></p>
        <p class="abstract">IntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups. Very few studies have looked at the transferability of European-derived PRS for behavioural and mental health phenotypes to non-Europeans. We assessed the comparative accuracy of PRS for Major Depressive Disorder (MDD) trained on European and African Ancestries GWAS studies to predict MDD and related traits in African Ancestries participants from the UK Biobank.

MethodsUK Biobank participants were selected based on Principal component analysis (PCA) clustering with an African genetic similarity reference population and MDD was assessed with the Composite International Diagnostic Interview (CIDI). Polygenic Risk Scores (PRS) were computed using PRSice2 using either European or African Ancestries GWAS summary statistics.

ResultsPRS trained on European ancestry samples (246,363 cases) predicted case control status in Africans of the UK Biobank with similar accuracies (190 cases, R2=2%) to PRS trained on far much smaller samples of African Ancestries participants from 23andMe, Inc. (5045 cases, R2=1.8%). This suggests that prediction of MDD status from Africans to Africans had greater efficiency per unit increase in the discovery sample size than prediction of MDD from Europeans to Africans. Prediction of MDD status in African UK Biobank participants using GWAS findings of causal risk factors from European ancestries was non-significant.

ConclusionGWAS studies of MDD in European ancestries are an inefficient means of improving polygenic prediction accuracy in African samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300294">
        <p class="paperTitle">Rare disease gene association discovery from burden analysis of the 100,000 Genomes Project data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O&#39;Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra</p>
        <p class="info">Score: 5.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300294' target='https://doi.org/10.1101/2023.12.20.23300294'> 10.1101/2023.12.20.23300294</a></p>
        <p class="abstract">To discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300308">
        <p class="paperTitle">Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noyvert, B.; Erzurumluoglu, A. M.; Drichel, D.; Omland, S.; Andlauer, T. F. M.; Mueller, S.; Sennels, L.; Becker, C.; Kantorovich, A.; Bartholdy, B. A.; Braenne, I.; Bolivar-Lopez, J. C.; Mistrellides, C.; Belbin, G. M.; Li, J. H.; Pickrell, J. K.; de Jong, J.; Arora, J.; Hu, Y.; Boehringer Ingelheim - Global Computational Biology and Digital Sciences,  ; Wood, C. R.; Kriegl, J. M.; Podduturi, N.; Jensen, J. N.; Stutzki, J.; Ding, Z.</p>
        <p class="info">Score: 27.3, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300308' target='https://doi.org/10.1101/2023.12.20.23300308'> 10.1101/2023.12.20.23300308</a></p>
        <p class="abstract">Advancements in long-read sequencing technology have accelerated the study of large structural variants (SVs). We created a curated, publicly available, multi-ancestry SV imputation panel by long-read sequencing 888 samples from the 1000 Genomes Project. This high-quality panel was used to impute SVs in approximately 500,000 UK Biobank participants. We demonstrated the feasibility of conducting genome-wide SV association studies at biobank scale using 32 disease-relevant phenotypes related to respiratory, cardiometabolic and liver diseases, in addition to 1,463 protein levels. This analysis identified thousands of genome-wide significant SV associations, including hundreds of conditionally independent signals, thereby enabling novel biological insights. Focusing on genetic association studies of lung function as an example, we demonstrate the added value of SVs for prioritising causal genes at gene-rich loci compared to traditional GWAS using only short variants. We envision that future post-GWAS gene-prioritisation workflows will incorporate SV analyses using this SV imputation panel and framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.23299893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.23299893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.23299893">
        <p class="paperTitle">Identification of Clonal Hematopoiesis Driver Mutations through In Silico Saturation Mutagenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.23299893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.23299893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Demajo, S.; Ramis-Zaldivar, J. E.; Muinos, F.; Grau, M. L.; Andrianova, M.; Lopez-Bigas, N.; Gonzalez-Perez, A.</p>
        <p class="info">Score: 25.4, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.23299893' target='https://doi.org/10.1101/2023.12.13.23299893'> 10.1101/2023.12.13.23299893</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is a phenomenon of clonal expansion of hematopoietic stem cells driven by somatic mutations affecting certain genes. Recently, CH has been linked to the development of a number of hematologic malignancies, cardiovascular diseases and other conditions. Although the most frequently mutated CH driver genes have been identified, a systematic landscape of the mutations capable of initiating this phenomenon is still lacking. Here, we train high-quality machine-learning models for 12 of the most recurrent CH driver genes to identify their driver mutations. These models outperform an experimental base-editing approach and expert-curated rules based on prior knowledge of the function of these genes. Moreover, their application to identify CH driver mutations across almost half a million donors of the UK Biobank reproduces known associations between CH driver mutations and age, and the prevalence of several diseases and conditions. We thus propose that these models support the accurate identification of CH across healthy individuals

SignificanceWe developed and validated 12 gene-specific machine learning models to identify CH driver mutations, showing their advantage with respect to expert-curated rules. These models can support the identification and clinical interpretation of CH mutations in newly sequenced individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.23300469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.23300469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.23300469">
        <p class="paperTitle">Progress Update for the Multisite Optical Genome Mapping Evaluation and Validation Study: Prenatal Applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.23300469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.23300469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levy, B.; Liu, J.; Iqbal, A.; Dupont, B.; Sahajpal, N.; Ho, M.; Yu, J.; Brody, S.; Mason-Suares, H.; Ganapathi, M.; Rajkovic, A.; Smolarek, T.; Toydemir, R.; Bui, P.; Kolhe, R.; Stevenson, R.</p>
        <p class="info">Score: 1.9, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.23300469' target='https://doi.org/10.1101/2023.12.22.23300469'> 10.1101/2023.12.22.23300469</a></p>
        <p class="abstract">Optical genome mapping (OGM) is an emerging technology with great potential for prenatal diagnosis. OGM can identify and resolve all types of balanced and unbalanced cytogenomic abnormalities in a single test, which are typically assessed by multiple standard of care (SOC) methods including karyotyping, fluorescence in situ hybridization and chromosomal microarray.

To assess OGMs viability as an alternative to conventional SOC testing, a comprehensive clinical research study was conducted across multiple sites, operators, and instruments to evaluate its accuracy and clinical utility. This report provides an update for the phase 2 results of the ongoing multisite evaluation and validation study evaluating OGM for prenatal applications. In phase 1, 123 prenatal cases were assessed by OGM, and in phase 2, 219 retrospective and prospective prenatal cases have been evaluated. For 71% of cases, at least two SOC tests were performed. The study found that OGM had an overall accuracy of 99.6% and positive predictive value of 100% when compared to all cytogenetic SOC results.

With its standardized workflow, cost-effectiveness, and high-resolution cytogenomic analysis, OGM shows great promise as an alternative technology that uses a single assay to consolidate the multiple SOC tests usually used for prenatal cytogenetic diagnosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300206" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300206">
        <p class="paperTitle">Genome-wide characterization of 54 urinary metabolites reveals molecular impact of kidney function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Valo, E.; Richmond, A.; Mutter, S.; Campbell, A.; Porteous, D.; Wilson, J. F.; FinnDiane Study Group,  ; Groop, P.-H.; Hayward, C.; Sandholm, N.</p>
        <p class="info">Score: 2.0, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300206' target='https://doi.org/10.1101/2023.12.20.23300206'> 10.1101/2023.12.20.23300206</a></p>
        <p class="abstract">Dissecting the genetic mechanisms underlying urinary metabolite concentrations can provide molecular insights into kidney function and open possibilities for causal assessment of urinary metabolites with risk factors and disease outcomes. Proton nuclear magnetic resonance metabolomics provides a high-throughput means for urinary metabolite profiling, as widely applied for blood biomarker studies. Here we report a genome-wide association study meta-analysed for 3 European cohorts comprising 8,026 individuals, covering both people with type 1 diabetes and general population settings. We identified 52 associations (p&lt;9.3x10-10) for 19 of 54 studied metabolite concentrations. Out of these, 32 were not reported previously for relevant urinary or blood metabolite traits. Subsequent two-sample Mendelian randomization analysis suggests that estimated glomerular filtration rate (eGFR) causally affects 13 urinary metabolite concentrations whereas urinary ethanolamine, an initial precursor for phosphatidylcholine and phosphatidylethanolamine, was associated with higher eGFR lending support for a potential protective role.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300337">
        <p class="paperTitle">Loss of coactosin-like F-actin binding protein 1 (Cotl1) decreases platelet-mediated osteoclastogenesis and causes osteopetrosis phenotypes in mouse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, E.; Yun, S.-H.; Jin, H.-S.; Lee, C.-G.; Lee, S.-H.; Choi, S.-Y.; Woo, H. G.; Lim, J. E.; Oh, B.; Jeong, S.-Y.</p>
        <p class="info">Score: 1.5, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300337' target='https://doi.org/10.1101/2023.12.20.23300337'> 10.1101/2023.12.20.23300337</a></p>
        <p class="abstract">BACKGROUNDSOsteopetrosis, a rare skeletal disease, is characterized by an increased bone mass resulting from impaired bone remodeling process. Platelet is the major bone-healing blood component involved in the regulation of bone resorption, particularly in the removal of compromised bones. Several actin-associated proteins contribute to the orchestration of actin ring formation in osteoclasts closely related to bone resorption. However, the role of coactosin-like F-actin binding protein 1 (Cotl1) in actin ring formation and platelet-mediated bone resorption mechanisms remains unclear.

METHODSWhole-mount in situ RNA hybridization was performed to detect cotl1 expression pattern in zebrafish. cotl1 gene knockdown zebrafish using morpholino oligonucleotides and platelet marker-expressing transgenic zebrafish were investigated for finding the phenotypic clues. Cotl1 knockout (Cotl1-/-) mice were generated using Cre/loxP recombination systems. In silico network analysis of the differentially expressed genes between bone marrow samples of wild type and Cotl1-/- mice was conducted. Primary-cultured monocytes from Cotl1-/- mice were examined for osteoclast differentiation and mRNA and protein expression patterns. Cotl1-/- mice underwent hematological examination and bone phenotype assessments including micro-CT, bone density, histology, immunohistochemistry, electron microscopy, and mechanical testing. Genetic association of SNPs in human COTL1 gene with estimated bone mineral density was analyzed.

RESULTSZebrafish cotl1 mRNA was highly expressed in the caudal hematopoietic tissue region. Knockdown of cotl1 in zebrafish embryos decreased the expression of c-myb, a marker of hematopoietic stem cells (HSCs). Notably, the platelet receptor CD41 was reduced in the HSCs of cotl1-depleted zebrafish and Cotl1-/- mice showed reduced platelet production with platelet surface markers of CD41 and CD61. Significantly reduced osteoclast differentiation and bone resorption pit, and impaired actin ring formation were observed in the primary myocytes from Cotl1-/- mice. Structural and histological analyses of the femur revealed sclerotic bone phenotypes in Cotl1-/- mice. Mechanical assessment of Cotl1-/- mouse femoral bones revealed osteopetrotic phenotypes. Association analysis of genetic variants in COTL1 gene in subjects from the UK Biobank suggested that COTL1 is susceptible to bone density in humans.

CONCLUSIONSOur results provide insights into the role of Cotl1 in platelet-mediated osteoclastogenesis and the novel finding that the loss of Cotl1-/- mice causes osteopetrosis phenotypes.

Clinical PerspectiveWhat Is New?

O_LIDeficiency of Cotl1 decreased platelet production in zebrafish and mice.
C_LIO_LIAbsence of Cotl1 disrupted the actin ring formation which is crucial for osteoclast differentiation in bone remodeling process.
C_LIO_LICotl1 knockout mice displayed sclerotic bone phenotypes and increased bone density that are representative characteristics of osteopetrosis.
C_LIO_LIGenetic variants in COTL1 gene in subjects from the UK Biobank are significantly associated with bone density.
C_LI

What Are the Clinical Implications?

O_LIThe current findings suggest that Cotl1 plays a fundamental role in platelet production-mediated osteoclastogenesis during bone remodeling, providing valuable insights into novel strategies for bone health maintenance.
C_LIO_LICotl1 may be a promising target for novel therapeutic strategies for the treatment and/or prevention of impaired osteoclastogenesis-mediated bone diseases such as osteopetrosis and osteoporosis.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.23300533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.23300533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.23300533">
        <p class="paperTitle">Genome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.23300533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.23300533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.</p>
        <p class="info">Score: 1.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.23300533' target='https://doi.org/10.1101/2023.12.26.23300533'> 10.1101/2023.12.26.23300533</a></p>
        <p class="abstract">Neuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field, &#39;brain age gap&#39; has emerged as promising MRI-based biomarker quantifying the deviation between an individual&#39;s biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimer&#39;s disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300480" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300480">
        <p class="paperTitle">SEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, M.-Y.; Zhong, J.-D.; Li, X.; Bai, W.-Y.; Yuan, C.-D.; Qiu, M.-C.; Liu, K.-Q.; Yu, C.-F.; Li, N.; Yang, J.-J.; Liu, Y.-H.; Yu, S.-H.; Zhao, W.-W.; Liu, J.-Q.; Sun, Y.; Cong, P.-K.; Khederzadeh, S.; Zhao, P.-P.; Qian, Y.; Guan, P.-L.; Gu, J.-X.; Gai, S.-R.; Yi, X.-J.; Tao, J.-G.; Chen, X.; Miao, M.-M.; Lei, L.-X.; Xu, L.; Xie, S.-Y.; Tian, G.; Li, J.-C.; Guo, J.-F.; Karasik, D.; Yang, L.; Tang, B.-S.; Huang, F.; Zheng, H.-F.</p>
        <p class="info">Score: 1.0, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300480' target='https://doi.org/10.1101/2023.12.23.23300480'> 10.1101/2023.12.23.23300480</a></p>
        <p class="abstract">Here, we present the South and East Asian Reference Database (SEAD) reference panel (https://imputationserver.westlake.edu.cn/), which comprises whole genome sequencing data from 11,067 individuals across 17 countries in Asia. The SEAD panel, which excludes singleton variants, consists of 22,134 haplotypes and 80,367,720 variants. Firstly, we assessed the concordance rate in global populations using HGDP datasets, notably, the SEAD panel showed advantage in East Asia, Central and South Asia, and Oceania populations. When imputing the disease-associated variants of Asian population, the SEAD panel displayed a distinct preponderance in imputing low-frequency and rare variants. In imputation of Chinese population, the SEAD panel imputed a larger number of well-imputed sites across all minor allele frequency (MAF) bins. Additionally, the SEAD panel exhibited higher imputation accuracy for shared sites in all MAF bins. Finally, we applied the augmented SEAD panel to conduct a discovery and replication genome-wide association study (GWAS) for hip and femoral neck (FN) bone mineral density (BMD) traits within the 5,369 Westlake BioBank for Chinese (WBBC) samples. The single-variant test suggests that rare variants near SNTG1 gene are associated with hip BMD (rs60103302, MAF=0.0091, P=4.79x10-8). The spatial clustering analysis also suggests the association of this gene (Pslide_window=1.08x10-8, Pgene_centric=4.72x10-8). The gene and variants achieved a suggestive level for FN BMD. This gene was not reported previously, and the preliminary experiment demonstrated that the identified rare variant can upregulate the SNTG1 expression, which in turn inhibits the proliferation and differentiation of preosteoblast.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
