<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.
Authors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium, ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.
Score: 24.3, Published: 2023-06-23 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.
Authors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium, ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.
Score: 24.3, Published: 2023-06-23 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.
Authors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium, ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.
Score: 24.3, Published: 2023-06-23 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.\nAuthors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium, ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.\nScore: 24.3, Published: 2023-06-23 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.\nAuthors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium, ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.\nScore: 24.3, Published: 2023-06-23 DOI: 10.1101/2023.06.19.23291606\nGenetics as a science has roots in studying phenotypes of relatives, but molecular approaches facilitate direct measurements of genomic variation within individuals. Agricultural and human biomedical research are both emphasizing genotype-based instruments, like polygenic scores, as the future of breeding programs or precision medicine and genetic epidemiology. However, unlike in agriculture, there is an emerging consensus that family variables act near independent of genotypes in models of human disease. To advance our understanding of this phenomenon, we use 2,066,057 family records of 99,645 genotyped probands from the iPSYCH2015 case-cohort study to show that state-of-the-field genotype- and phenotype-based genetic instruments explain essentially independent components of liability to psychiatric disorders. We support these empirical results with novel theoretical analysis and simulations to describe, in a human biomedical context, parameters affecting current and future performance of the two approaches, their expected interrelationships, relative sample size efficiencies, and the striking consistency of observed results with expectations under simple additive, polygenic liability models of disease. We conclude, at least for psychiatric disorders, that phenotype- and genotype-based genetic instruments are likely to be very noisy measures of the same underlying additive genetic liability, should be seen, in the near future, as complementary, and integrated to a greater extent going forward.\nLoss-of-function variants in CUL3 cause a syndromic neurodevelopmental disorder\nAuthors: Blackburn, P. R.; Ebstein, F.; Hsieh, T.-C.; Motta, M.; Radio, F. C.; Herkert, J. C.; Rinne, T.; Thiffault, I.; Rapp, M.; Alders, M.; Maas, S.; Gerard, B.; Smol, T.; Vincent-Delorme, C.; Cogne, B.; Isidor, B.; Vincent, M.; Bachmann-Gagescu, R.; Rauch, A.; Joset, P.; Battista Ferrero, G.; Ciolfi, A.; Husson, T.; Guerrot, A.-M.; Bacino, C.; Macmurdo, C.; Thompson, S. S.; Rosenfeld, J. A.; Faivre, L.; Tran Mau-Them, F.; Deb, W.; Vignard, V.; Agrawal, P. B.; Madden, J. A.; Goldenberg, A.; Lecoquierre, F.; Zech, M.; Prokisch, H.; Necpal, J.; Jech, R.; Winkelmann, J.; Turcanova Koprusakova, M.; Kons\nScore: 23.1, Published: 2023-06-16 DOI: 10.1101/2023.06.13.23290941\nPurposeDe novo variants in CUL3 (Cullin-3 ubiquitin ligase) have been strongly associated with neurodevelopmental disorders (NDDs), but no large case series have been reported so far. Here we aimed to collect sporadic cases carrying rare variants in CUL3, describe the genotype-phenotype correlation, and investigate the underlying pathogenic mechanism. MethodsGenetic data and detailed clinical records were collected via multi-center collaboration. Dysmorphic facial features were analyzed using GestaltMatcher. Variant effects on CUL3 protein stability were assessed using patient-derived T-cells. ResultsWe assembled a cohort of 35 individuals with heterozygous CUL3 variants presenting a syndromic NDD characterized by intellectual disability with or without autistic features. Of these, 33 have loss-of-function (LoF) and two have missense variants. CUL3 LoF variants in patients may affect protein stability leading to perturbations in protein homeostasis, as evidenced by decreased ubiquitin-protein conjugates in vitro. Specifically, we show that cyclin E1 (CCNE1) and 4E-BP1 (EIF4EBP1), two prominent substrates of CUL3, fail to be targeted for proteasomal degradation in patient-derived cells. ConclusionOur study further refines the clinical and mutational spectrum of CUL3-associated NDDs, expands the spectrum of cullin RING E3 ligase-associated neuropsychiatric disorders, and suggests haploinsufficiency via LoF variants is the predominant pathogenic mechanism.\nCombining a prioritization strategy and functional studies nominates 5'UTR variants underlying inherited retinal disease\nAuthors: Duenas Rey, A.; del Pozo Valero, M.; Bouckaert, M.; Van Den Broeck, F.; Daich Varela, M.; Van Heetvelde, M.; De Bruyne, M.; Van de Sompele, S.; Bauwens, M.; Ellingford, J. M.; Lenaerts, H.; Mahieu, Q.; Josifova, D.; Genomics England Research Consortium, ; Rivolta, C.; Webster, A.; Arno, G.; Ayuso, C.; De Zaeytijd, J.; Leroy, B.; De Baere, E.; Coppieters, F.\nScore: 11.4, Published: 2023-06-27 DOI: 10.1101/2023.06.19.23291376\nBackground| 5' untranslated regions (5'UTRs) are essential modulators of protein translation. Predicting the impact of 5'UTR variants is challenging and typically not performed in routine diagnostics. Here, we present a combined approach of a comprehensive prioritization strategy and subsequent functional assays to evaluate 5'UTR variation in two large cohorts of patients with inherited retinal diseases (IRDs). Methods| We performed an isoform-level re-analysis of retinal RNA-seq data to identify the protein-coding transcripts of 378 IRD genes with highest expression in retina. We evaluated the coverage of these 5'UTRs by different whole exome sequencing (WES) capture kits. The selected 5'UTRs were analyzed in whole genome sequencing (WGS) and WES data from IRD sub-cohorts from the 100,000 Genomes Project (n = 2,417 WGS) and an in-house database (n = 1,682 WES), respectively. Identified variants were annotated for 5'UTR-relevant features and classified into 7 distinct categories based on their predicted functional consequence. We developed a variant prioritization strategy by integrating population frequency, specific criteria for each category, and family and phenotypic data. A selection of candidate variants underwent functional validation using diverse experimental approaches. Results| Isoform-level re-quantification of retinal gene expression revealed 76 IRD genes with a non-canonical retina-enriched isoform, of which 20 display a fully distinct 5'UTR compared to that of their canonical isoform. Depending on the probe-design 3-20% of IRD genes have 5'UTRs fully captured by WES. After analyzing these regions in both IRD cohorts we prioritized 11 (likely) pathogenic variants in 10 genes (ARL3, MERTK, NDP, NMNAT1, NPHP4, PAX6, PRPF31, PRPF4, RDH12, RD3), of which 8 were novel. Functional analyses further supported the pathogenicity of 2 variants. The MERTK:c.-125G\u003eA variant, overlapping a transcriptional start site, was shown to significantly reduce both luciferase mRNA levels and activity. The RDH12:c.-123C\u003eT variant was found in cis with the reported hypomorphic RDH12:c.701G\u003eA (p.Arg234His) variant in 11 patients. This 5'UTR variant, predicted to introduce an upstream open reading frame, was shown to result in reduced RDH12 protein but unaltered mRNA levels. Conclusions| This study demonstrates the importance of 5'UTR variants implicated in IRDs and provides a systematic approach for 5'UTR annotation and validation that is applicable to other inherited diseases.\nImproved Type 2 Diabetes Risk Stratification in the Qatar Biobank Cohort by Ensemble Learning Classifier Incorporating Multi-Trait, Population-Specific, Polygenic Risk Scores\nAuthors: Ahmed, I.; Ziab, M.; Mbarek, H.; Taheri, S.; Chagoury, O. L.; Hussain, S. A.; Lakshmi, J.; Bhat, A.; Fakhro, K. A.; Al-Shabeeb AKIL, A. S.\nScore: 8.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.23291830\nBackground: Type 2 Diabetes (T2D) is a pervasive chronic disease influenced by a complex interplay of environmental and genetic factors. To enhance T2D risk prediction, leveraging genetic information is essential, with polygenic risk scores (PRS) offering a promising tool for assessing individual genetic risk. Our study focuses on the comparison between multi-trait and single-trait PRS models and demonstrates how the incorporation of multi-trait PRS into risk prediction models can significantly augment T2D risk assessment accuracy and effectiveness. Methods: We conducted genome-wide association studies (GWAS) on 12 distinct T2D-related traits within a cohort of 14,278 individuals, all sequenced under the Qatar Genome Programme (QGP). This in-depth genetic analysis yielded several novel genetic variants associated with T2D, which served as the foundation for constructing multiple weighted PRS models. To assess the cumulative risk from these predictors, we applied machine learning (ML) techniques, which allowed for a thorough risk assessment. Results: Our research identified genetic variations tied to T2D risk and facilitated the construction of ML models integrating PRS predictors for an exhaustive risk evaluation. The top-performing ML model demonstrated a robust performance with an accuracy of 0.8549, AUC of 0.92, AUC-PR of 0.8522, and an F1 score of 0.757, reflecting its strong capacity to differentiate cases from controls. We are currently working on acquiring independent T2D cohorts to validate the efficacy of our final model. Conclusion: Our research underscores the potential of PRS models in identifying individuals within the population who are at elevated risk of developing T2D and its associated complications. The use of multi-trait PRS and ML models for risk prediction could inform early interventions, potentially identifying T2D patients who stand to benefit most based on their individual genetic risk profile. This combined approach signifies a stride forward in the field of precision medicine, potentially enhancing T2D risk prediction, prevention, and management.\nMetabolomic and genomic prediction of common diseases in 477,706 participants in three national biobanks\nAuthors: Nightingale Health Biobank Collaborative Group, ; Barrett, J. C.; Esko, T.; Fischer, K.; Jostins-Dean, L.; Jousilahti, P.; Julkunen, H.; Jaaskelainen, T.; Kerimov, N.; Kerminen, S.; Kolde, A.; Koskela, H.; Kronberg, J.; Lundgren, S. N.; Lundqvist, A.; Makela, V.; Nybo, K.; Perola, M.; Salomaa, V.; Schut, K.; Soikkeli, M.; Soininen, P.; Tiainen, M.; Tillmann`, T.; Wurtz, P.; Estonian Biobank Research Team, Score: 32.8, Published: 2023-06-12 DOI: 10.1101/2023.06.09.23291213\nIdentifying individuals at high risk of chronic diseases via easily measured biomarkers could improve public health efforts to prevent avoidable illness and death. Here we present nuclear magnetic resonance blood metabolomics from half a million samples from three national biobanks. We built metabolomic risk scores that identify a high-risk group for each of 12 diseases that cause the most morbidity in high-income countries and show consistent cross-biobank replication of the relative risk of disease for these groups. We show that these metabolomic risk scores are more strongly associated with future disease onset than polygenic scores for most of these diseases. In a subset of 18,000 individuals with metabolomic biomarkers measured at two time points we show that people whose scores change have dramatically different future risk of disease, suggesting that repeat measurements capture the benefits of lifestyle change. We show cross-biobank calibration of our scores. Since metabolomics can be measured from a standard blood sample, we propose such tests can be feasibly implemented today in preventative health programs. One-Sentence SummaryBiomarkers from half a million blood samples identifies people at increased risk of chronic diseases and can be used for early detection today.\nAutoencoder-based phenotyping of ophthalmic images highlights genetic loci influencing retinal morphology and provides epidemiologically informative biomarkers.\nAuthors: Sergouniotis, P. I.; Diakite, A.; Gaurav, K.; Birney, E.; Fitzgerald, T.\nScore: 5.8, Published: 2023-06-20 DOI: 10.1101/2023.06.15.23291410\nGenome-wide association studies (GWAS) have been remarkably successful in identifying associations between genetic variation and imaging-derived phenotypes. To date, the main focus of these analyses has been established, clinically-used imaging features. Here, we sought to investigate if deep learning approaches can help detect more nuanced patterns of image variability. To this end, we used an autoencoder to represent retinal optical coherence tomography (OCT) images from 31,135 UK Biobank participants. For each study subject, we obtained a 64-dimensional vector representing features of retinal structure. GWAS of these autoencoder-derived imaging parameters identified 113 genome-wide-significant loci. These encompassed variants previously linked with retinal thickness measurements, ophthalmic disorders and/or neurodegenerative conditions (including dementia). Notably, the generated retinal phenotypes were found to contribute to predictive models for glaucoma and cardiovascular disorders. Overall, we demonstrate that self-supervised phenotyping of OCT images enhances the discoverability of genetic factors influencing retinal morphology and provides epidemiologically informative biomarkers.\nTracing the evolutionary path of the CCR5delta32 deletion via ancient and modern genomes\nAuthors: Ravn, K.; Cobuccio, L.; Muktupavela, R. A.; Meisner, J. M.; Benros, M. E.; Korneliussen, T. S.; Sikora, M.; Willerslev, E.; Allentoft, M. E.; Irving-Pease, E. K.; Racimo, F.; Rasmussen, S.\nScore: 5.0, Published: 2023-06-20 DOI: 10.1101/2023.06.15.23290026\nThe chemokine receptor variant CCR5delta32 is linked to HIV-1 infection resistance and other pathological conditions. In European populations, the allele frequency ranges from 10-16%, and its evolution has been extensively debated throughout the years. We provide a detailed perspective of the evolutionary history of the deletion through time and space. We discovered that the CCR5delta32 allele arose on a pre-existing haplotype consisting of 84 variants. Using this information, we developed a haplotype-aware probabilistic model to screen for this deletion across 860 low-coverage ancient genomes and we found evidence that CCR5delta32 arose at least 7,000 years BP, with a likely origin somewhere in the Western Eurasian Steppe region. We further show evidence that the CCR5delta32 haplotype underwent positive selection between 7,000-2,000 BP in Western Eurasia and that the presence of the haplotype in Latin America can be explained by post-Columbian genetic exchanges. Finally, we point to new complex CCR5delta32 genotype-haplotype-phenotype relationships, which demand consideration when targeting the CCR5 receptor for therapeutic strategies.\nProtein-truncating variants in BSN are associated with severe adult-onset obesity, type 2 diabetes and fatty liver disease\nAuthors: Zhao, Y.; Chukanova, M.; Kentistou, K. A.; Fairhurst-Hunter, Z.; Siegert, A. M.; Jia, R.; Dowsett, G.; Gardner, E. J.; Day, F. R.; Kaisinger, L. R.; Tung, Y.-C. L.; Lam, B. Y. H.; Chen, H.-J. C.; Wang, Q.; Berumen-Campos, J.; Kuri-Morales, P.; Tapia-Conyer, R.; Alegre-Diaz, J.; Emberson, J.; Torres, J. M.; Collins, R.; Saleheen, D.; Smith, K. R.; Paul, D. R.; Merkle, F.; Wareham, N. J.; Petrovski, S.; O'Rahilly, S.; Ong, K. K.; Yeo, G. S. H.; Perry, J. R. B.\nScore: 8.9, Published: 2023-06-16 DOI: 10.1101/2023.06.14.23291368\nObesity is a major risk factor for many common diseases and has a significant heritable component. While clinical and large-scale population studies have identified several genes harbouring rare alleles with large effects on obesity risk, there are likely many unknown genes with highly penetrant effects remaining. To this end, we performed whole exome-sequence analyses for adult body mass index (BMI) in up to 587,027 individuals. We identified rare, loss of function variants in two genes - BSN and APBA1 - with effects on BMI substantially larger than well-established obesity genes such as MC4R. One in [~]6500 individuals carry a heterozygous protein truncating variant (PTV) in BSN, which confers a 6.6, 3.7 and 3-fold higher risk of severe obesity (BMI \u003e40kg/m2), non-alcoholic fatty liver disease and type 2 diabetes, respectively. In contrast to most other obesity-related genes, rare variants in BSN and APBA1 had no apparent effect on childhood adiposity. Furthermore, BSN PTVs magnified the influence of common genetic variants associated with BMI, with a common polygenic score exhibiting an effect on BMI twice as large in BSN PTV carriers than non-carriers. Finally, we explored the plasma proteomic signatures of BSN PTV carriers as well as the functional consequences of BSN deletion in human iPSC-derived hypothalamic neurons. These approaches highlighted a network of differentially expressed genes that were collectively enriched for genomic regions associated with BMI, and suggest a role for degenerative neuronal synaptic function and neurotransmitter release in the etiology of obesity.\nUnderstanding the genetic complexity of puberty timing across the allele frequency spectrum\nAuthors: Kentistou, K. A.; Kaisinger, L. R.; Stankovic, S.; Vaudel, M.; de Oliveira, E. M.; Messina, A.; Walters, R. G.; Liu, X.; Busch, A. S.; Helgason, H.; Thompson, D. J.; Santon, F.; Petricek, K. M.; Zouaghi, Y.; Huang-Doran, I.; Gudbjartsson, D. F.; Bratland, E.; Lin, K.; Gardner, E. J.; Zhao, Y.; Jia, R.; Terao, C.; Riggan, M.; Bolla, M. K.; Yazdanpanah, M.; Yazdanpanah, N.; Bradfield, J. P.; Broer, L.; Campbell, A.; Chasman, D. I.; Cousminer, D. L.; Franceschini, N.; Franke, L. H.; Girotto, G.; He, C.; Jarvelin, M.-R.; Joshi, P. K.; Kamatani, Y.; Karlsson, R.; Luan, J.; Lunetta, K. L.; Magi, R.;\nScore: 4.0, Published: 2023-06-20 DOI: 10.1101/2023.06.14.23291322\nPubertal timing varies considerably and has been associated with a range of health outcomes in later life. To elucidate the underlying biological mechanisms, we performed multi-ancestry genetic analyses in [~]800,000 women, identifying 1,080 independent signals associated with age at menarche. Collectively these loci explained 11% of the trait variance in an independent sample, with women at the top and bottom 1% of polygenic risk exhibiting a [~]11 and [~]14-fold higher risk of delayed and precocious pubertal development, respectively. These common variant analyses were supported by exome sequence analysis of [~]220,000 women, identifying several genes, including rare loss of function variants in ZNF483 which abolished the impact of polygenic risk. Next, we implicated 660 genes in pubertal development using a combination of in silico variant-to-gene mapping approaches and integration with dynamic gene expression data from mouse embryonic GnRH neurons. This included an uncharacterized G-protein coupled receptor GPR83, which we demonstrate amplifies signaling of MC3R, a key sensor of nutritional status. Finally, we identified several genes, including ovary-expressed genes involved in DNA damage response that co-localize with signals associated with menopause timing, leading us to hypothesize that the ovarian reserve might signal centrally to trigger puberty. Collectively these findings extend our understanding of the biological complexity of puberty timing and highlight body size dependent and independent mechanisms that potentially link reproductive timing to later life disease.\nInvestigating the genetic relationship between depression symptoms and Alzheimer's disease in clinically diagnosed and proxy cases\nAuthors: Gilchrist, L.; Spargo, T. P.; Green, R. E.; Coleman, J. R. I.; Howard, D. M.; Thorp, J. G.; Adey, B.; Lord, J.; Davies, H. L.; Mundy, J.; Davies, M. R.; HuÌˆbel, C.; Bristow, S.; Lee, S. H.; Rogers, H.; Curtis, C.; Kalsi, G.; Arathimos, R.; Corbett, A.; Ballard, C.; Brooker, H.; Creese, B.; Aarsland, D.; Hampshire, A.; Velayudhan, L.; Eley, T. C.; Breen, G.; Iacoangeli, A.; Koks, S.; Lewis, C. M.; Proitsi, P.; Alzheimer's Disease Neuroimaging Initiative, ; AddNeuroMed, ; GERAD1 Consortium, Score: 16.5, Published: 2023-06-06 DOI: 10.1101/2023.06.05.23290588\nDepression is a risk factor for the later development of Alzheimers disease (AD), but evidence for their genetic relationship is mixed. Assessing depression symptom specific genetic associations may better clarify this relationship. Using data from the UK Biobank, the GLAD Study and PROTECT, we performed the largest genome-wide meta-analyses (GWAS) of the nine depression symptom items, plus their sum score, on the Patient Health Questionnaire (PHQ-9) (GWAS equivalent N: 224,535-308,421). We then assessed global and local genetic correlations and statistical colocalisation between depression/depression symptoms and AD across six AD GWAS with varying proportions of clinical and proxy (family history) case ascertainment. We assessed bi-directional causal associations using Mendelian randomisation (MR) and tested the predictive utility of depression/depression symptom polygenic risk scores (PRS) for AD case/control status in three clinical AD cohorts. Our GWAS meta-analyses identified 37 genomic risk loci across the ten depression symptom phenotypes. Of the 72 global genetic correlation tests conducted between depression/depression symptoms and AD, 20 were significant at pFDR [\u0026le;] 0.05. Only one of these was identified with AD GWAS containing clinical-only cases. Local genetic correlations were detected at 14 loci. Statistical colocalisation was not identified at these loci but was identified in the region of transmembrane protein 106B (TMEM106B) between multiple depression phenotypes and both clinical-only and clinical+proxy AD. MR and PRS analyses did not yield significant results after multiple testing correction. Our findings do not demonstrate a causal role of depression/depression symptoms on AD and suggest that previous evidence of genetic overlap between depression and AD may be driven by the inclusion of family-history-based proxy cases/controls. However, the identification of colocalisation at TMEM106B warrants further investigation.\n",
  "wordCount" : "3103",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291606">
        <p class="paperTitle">The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium,  ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.</p>
        <p class="info">Score: 24.3, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291606' target='https://doi.org/10.1101/2023.06.19.23291606'> 10.1101/2023.06.19.23291606</a></p>
        <p class="abstract">Genetics as a science has roots in studying phenotypes of relatives, but molecular approaches facilitate direct measurements of genomic variation within individuals. Agricultural and human biomedical research are both emphasizing genotype-based instruments, like polygenic scores, as the future of breeding programs or precision medicine and genetic epidemiology. However, unlike in agriculture, there is an emerging consensus that family variables act near independent of genotypes in models of human disease. To advance our understanding of this phenomenon, we use 2,066,057 family records of 99,645 genotyped probands from the iPSYCH2015 case-cohort study to show that state-of-the-field genotype- and phenotype-based genetic instruments explain essentially independent components of liability to psychiatric disorders. We support these empirical results with novel theoretical analysis and simulations to describe, in a human biomedical context, parameters affecting current and future performance of the two approaches, their expected interrelationships, relative sample size efficiencies, and the striking consistency of observed results with expectations under simple additive, polygenic liability models of disease. We conclude, at least for psychiatric disorders, that phenotype- and genotype-based genetic instruments are likely to be very noisy measures of the same underlying additive genetic liability, should be seen, in the near future, as complementary, and integrated to a greater extent going forward.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.23290941">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.23290941" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.23290941">
        <p class="paperTitle">Loss-of-function variants in CUL3 cause a syndromic neurodevelopmental disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.23290941" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.23290941" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blackburn, P. R.; Ebstein, F.; Hsieh, T.-C.; Motta, M.; Radio, F. C.; Herkert, J. C.; Rinne, T.; Thiffault, I.; Rapp, M.; Alders, M.; Maas, S.; Gerard, B.; Smol, T.; Vincent-Delorme, C.; Cogne, B.; Isidor, B.; Vincent, M.; Bachmann-Gagescu, R.; Rauch, A.; Joset, P.; Battista Ferrero, G.; Ciolfi, A.; Husson, T.; Guerrot, A.-M.; Bacino, C.; Macmurdo, C.; Thompson, S. S.; Rosenfeld, J. A.; Faivre, L.; Tran Mau-Them, F.; Deb, W.; Vignard, V.; Agrawal, P. B.; Madden, J. A.; Goldenberg, A.; Lecoquierre, F.; Zech, M.; Prokisch, H.; Necpal, J.; Jech, R.; Winkelmann, J.; Turcanova Koprusakova, M.; Kons</p>
        <p class="info">Score: 23.1, Published: 2023-06-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.23290941' target='https://doi.org/10.1101/2023.06.13.23290941'> 10.1101/2023.06.13.23290941</a></p>
        <p class="abstract">PurposeDe novo variants in CUL3 (Cullin-3 ubiquitin ligase) have been strongly associated with neurodevelopmental disorders (NDDs), but no large case series have been reported so far. Here we aimed to collect sporadic cases carrying rare variants in CUL3, describe the genotype-phenotype correlation, and investigate the underlying pathogenic mechanism.

MethodsGenetic data and detailed clinical records were collected via multi-center collaboration. Dysmorphic facial features were analyzed using GestaltMatcher. Variant effects on CUL3 protein stability were assessed using patient-derived T-cells.

ResultsWe assembled a cohort of 35 individuals with heterozygous CUL3 variants presenting a syndromic NDD characterized by intellectual disability with or without autistic features. Of these, 33 have loss-of-function (LoF) and two have missense variants. CUL3 LoF variants in patients may affect protein stability leading to perturbations in protein homeostasis, as evidenced by decreased ubiquitin-protein conjugates in vitro. Specifically, we show that cyclin E1 (CCNE1) and 4E-BP1 (EIF4EBP1), two prominent substrates of CUL3, fail to be targeted for proteasomal degradation in patient-derived cells.

ConclusionOur study further refines the clinical and mutational spectrum of CUL3-associated NDDs, expands the spectrum of cullin RING E3 ligase-associated neuropsychiatric disorders, and suggests haploinsufficiency via LoF variants is the predominant pathogenic mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291376">
        <p class="paperTitle">Combining a prioritization strategy and functional studies nominates 5&#39;UTR variants underlying inherited retinal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duenas Rey, A.; del Pozo Valero, M.; Bouckaert, M.; Van Den Broeck, F.; Daich Varela, M.; Van Heetvelde, M.; De Bruyne, M.; Van de Sompele, S.; Bauwens, M.; Ellingford, J. M.; Lenaerts, H.; Mahieu, Q.; Josifova, D.; Genomics England Research Consortium,  ; Rivolta, C.; Webster, A.; Arno, G.; Ayuso, C.; De Zaeytijd, J.; Leroy, B.; De Baere, E.; Coppieters, F.</p>
        <p class="info">Score: 11.4, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291376' target='https://doi.org/10.1101/2023.06.19.23291376'> 10.1101/2023.06.19.23291376</a></p>
        <p class="abstract">Background| 5&#39; untranslated regions (5&#39;UTRs) are essential modulators of protein translation. Predicting the impact of 5&#39;UTR variants is challenging and typically not performed in routine diagnostics. Here, we present a combined approach of a comprehensive prioritization strategy and subsequent functional assays to evaluate 5&#39;UTR variation in two large cohorts of patients with inherited retinal diseases (IRDs). Methods| We performed an isoform-level re-analysis of retinal RNA-seq data to identify the protein-coding transcripts of 378 IRD genes with highest expression in retina. We evaluated the coverage of these 5&#39;UTRs by different whole exome sequencing (WES) capture kits. The selected 5&#39;UTRs were analyzed in whole genome sequencing (WGS) and WES data from IRD sub-cohorts from the 100,000 Genomes Project (n = 2,417 WGS) and an in-house database (n = 1,682 WES), respectively. Identified variants were annotated for 5&#39;UTR-relevant features and classified into 7 distinct categories based on their predicted functional consequence. We developed a variant prioritization strategy by integrating population frequency, specific criteria for each category, and family and phenotypic data. A selection of candidate variants underwent functional validation using diverse experimental approaches. Results| Isoform-level re-quantification of retinal gene expression revealed 76 IRD genes with a non-canonical retina-enriched isoform, of which 20 display a fully distinct 5&#39;UTR compared to that of their canonical isoform. Depending on the probe-design 3-20% of IRD genes have 5&#39;UTRs fully captured by WES. After analyzing these regions in both IRD cohorts we prioritized 11 (likely) pathogenic variants in 10 genes (ARL3, MERTK, NDP, NMNAT1, NPHP4, PAX6, PRPF31, PRPF4, RDH12, RD3), of which 8 were novel. Functional analyses further supported the pathogenicity of 2 variants. The MERTK:c.-125G&gt;A variant, overlapping a transcriptional start site, was shown to significantly reduce both luciferase mRNA levels and activity. The RDH12:c.-123C&gt;T variant was found in cis with the reported hypomorphic RDH12:c.701G&gt;A (p.Arg234His) variant in 11 patients. This 5&#39;UTR variant, predicted to introduce an upstream open reading frame, was shown to result in reduced RDH12 protein but unaltered mRNA levels. Conclusions| This study demonstrates the importance of 5&#39;UTR variants implicated in IRDs and provides a systematic approach for 5&#39;UTR annotation and validation that is applicable to other inherited diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291830">
        <p class="paperTitle">Improved Type 2 Diabetes Risk Stratification in the Qatar Biobank Cohort by Ensemble Learning Classifier Incorporating Multi-Trait, Population-Specific, Polygenic Risk Scores</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmed, I.; Ziab, M.; Mbarek, H.; Taheri, S.; Chagoury, O. L.; Hussain, S. A.; Lakshmi, J.; Bhat, A.; Fakhro, K. A.; Al-Shabeeb AKIL, A. S.</p>
        <p class="info">Score: 8.9, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291830' target='https://doi.org/10.1101/2023.06.23.23291830'> 10.1101/2023.06.23.23291830</a></p>
        <p class="abstract">Background: Type 2 Diabetes (T2D) is a pervasive chronic disease influenced by a complex interplay of environmental and genetic factors. To enhance T2D risk prediction, leveraging genetic information is essential, with polygenic risk scores (PRS) offering a promising tool for assessing individual genetic risk. Our study focuses on the comparison between multi-trait and single-trait PRS models and demonstrates how the incorporation of multi-trait PRS into risk prediction models can significantly augment T2D risk assessment accuracy and effectiveness. Methods: We conducted genome-wide association studies (GWAS) on 12 distinct T2D-related traits within a cohort of 14,278 individuals, all sequenced under the Qatar Genome Programme (QGP). This in-depth genetic analysis yielded several novel genetic variants associated with T2D, which served as the foundation for constructing multiple weighted PRS models. To assess the cumulative risk from these predictors, we applied machine learning (ML) techniques, which allowed for a thorough risk assessment. Results: Our research identified genetic variations tied to T2D risk and facilitated the construction of ML models integrating PRS predictors for an exhaustive risk evaluation. The top-performing ML model demonstrated a robust performance with an accuracy of 0.8549, AUC of 0.92, AUC-PR of 0.8522, and an F1 score of 0.757, reflecting its strong capacity to differentiate cases from controls. We are currently working on acquiring independent T2D cohorts to validate the efficacy of our final model. Conclusion: Our research underscores the potential of PRS models in identifying individuals within the population who are at elevated risk of developing T2D and its associated complications. The use of multi-trait PRS and ML models for risk prediction could inform early interventions, potentially identifying T2D patients who stand to benefit most based on their individual genetic risk profile. This combined approach signifies a stride forward in the field of precision medicine, potentially enhancing T2D risk prediction, prevention, and management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.23291213">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.23291213" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.23291213">
        <p class="paperTitle">Metabolomic and genomic prediction of common diseases in 477,706 participants in three national biobanks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.23291213" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.23291213" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nightingale Health Biobank Collaborative Group,  ; Barrett, J. C.; Esko, T.; Fischer, K.; Jostins-Dean, L.; Jousilahti, P.; Julkunen, H.; Jaaskelainen, T.; Kerimov, N.; Kerminen, S.; Kolde, A.; Koskela, H.; Kronberg, J.; Lundgren, S. N.; Lundqvist, A.; Makela, V.; Nybo, K.; Perola, M.; Salomaa, V.; Schut, K.; Soikkeli, M.; Soininen, P.; Tiainen, M.; Tillmann`, T.; Wurtz, P.; Estonian Biobank Research Team,  </p>
        <p class="info">Score: 32.8, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.23291213' target='https://doi.org/10.1101/2023.06.09.23291213'> 10.1101/2023.06.09.23291213</a></p>
        <p class="abstract">Identifying individuals at high risk of chronic diseases via easily measured biomarkers could improve public health efforts to prevent avoidable illness and death. Here we present nuclear magnetic resonance blood metabolomics from half a million samples from three national biobanks. We built metabolomic risk scores that identify a high-risk group for each of 12 diseases that cause the most morbidity in high-income countries and show consistent cross-biobank replication of the relative risk of disease for these groups. We show that these metabolomic risk scores are more strongly associated with future disease onset than polygenic scores for most of these diseases. In a subset of 18,000 individuals with metabolomic biomarkers measured at two time points we show that people whose scores change have dramatically different future risk of disease, suggesting that repeat measurements capture the benefits of lifestyle change. We show cross-biobank calibration of our scores. Since metabolomics can be measured from a standard blood sample, we propose such tests can be feasibly implemented today in preventative health programs.

One-Sentence SummaryBiomarkers from half a million blood samples identifies people at increased risk of chronic diseases and can be used for early detection today.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.23291410">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.23291410" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.23291410">
        <p class="paperTitle">Autoencoder-based phenotyping of ophthalmic images highlights genetic loci influencing retinal morphology and provides epidemiologically informative biomarkers.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.23291410" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.23291410" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sergouniotis, P. I.; Diakite, A.; Gaurav, K.; Birney, E.; Fitzgerald, T.</p>
        <p class="info">Score: 5.8, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.23291410' target='https://doi.org/10.1101/2023.06.15.23291410'> 10.1101/2023.06.15.23291410</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have been remarkably successful in identifying associations between genetic variation and imaging-derived phenotypes. To date, the main focus of these analyses has been established, clinically-used imaging features. Here, we sought to investigate if deep learning approaches can help detect more nuanced patterns of image variability. To this end, we used an autoencoder to represent retinal optical coherence tomography (OCT) images from 31,135 UK Biobank participants. For each study subject, we obtained a 64-dimensional vector representing features of retinal structure. GWAS of these autoencoder-derived imaging parameters identified 113 genome-wide-significant loci. These encompassed variants previously linked with retinal thickness measurements, ophthalmic disorders and/or neurodegenerative conditions (including dementia). Notably, the generated retinal phenotypes were found to contribute to predictive models for glaucoma and cardiovascular disorders. Overall, we demonstrate that self-supervised phenotyping of OCT images enhances the discoverability of genetic factors influencing retinal morphology and provides epidemiologically informative biomarkers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.23290026">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.23290026" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.23290026">
        <p class="paperTitle">Tracing the evolutionary path of the CCR5delta32 deletion via ancient and modern genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.23290026" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.23290026" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ravn, K.; Cobuccio, L.; Muktupavela, R. A.; Meisner, J. M.; Benros, M. E.; Korneliussen, T. S.; Sikora, M.; Willerslev, E.; Allentoft, M. E.; Irving-Pease, E. K.; Racimo, F.; Rasmussen, S.</p>
        <p class="info">Score: 5.0, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.23290026' target='https://doi.org/10.1101/2023.06.15.23290026'> 10.1101/2023.06.15.23290026</a></p>
        <p class="abstract">The chemokine receptor variant CCR5delta32 is linked to HIV-1 infection resistance and other pathological conditions. In European populations, the allele frequency ranges from 10-16%, and its evolution has been extensively debated throughout the years. We provide a detailed perspective of the evolutionary history of the deletion through time and space. We discovered that the CCR5delta32 allele arose on a pre-existing haplotype consisting of 84 variants. Using this information, we developed a haplotype-aware probabilistic model to screen for this deletion across 860 low-coverage ancient genomes and we found evidence that CCR5delta32 arose at least 7,000 years BP, with a likely origin somewhere in the Western Eurasian Steppe region. We further show evidence that the CCR5delta32 haplotype underwent positive selection between 7,000-2,000 BP in Western Eurasia and that the presence of the haplotype in Latin America can be explained by post-Columbian genetic exchanges. Finally, we point to new complex CCR5delta32 genotype-haplotype-phenotype relationships, which demand consideration when targeting the CCR5 receptor for therapeutic strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.23291368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.23291368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.23291368">
        <p class="paperTitle">Protein-truncating variants in BSN are associated with severe adult-onset obesity, type 2 diabetes and fatty liver disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.23291368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.23291368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Y.; Chukanova, M.; Kentistou, K. A.; Fairhurst-Hunter, Z.; Siegert, A. M.; Jia, R.; Dowsett, G.; Gardner, E. J.; Day, F. R.; Kaisinger, L. R.; Tung, Y.-C. L.; Lam, B. Y. H.; Chen, H.-J. C.; Wang, Q.; Berumen-Campos, J.; Kuri-Morales, P.; Tapia-Conyer, R.; Alegre-Diaz, J.; Emberson, J.; Torres, J. M.; Collins, R.; Saleheen, D.; Smith, K. R.; Paul, D. R.; Merkle, F.; Wareham, N. J.; Petrovski, S.; O&#39;Rahilly, S.; Ong, K. K.; Yeo, G. S. H.; Perry, J. R. B.</p>
        <p class="info">Score: 8.9, Published: 2023-06-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.23291368' target='https://doi.org/10.1101/2023.06.14.23291368'> 10.1101/2023.06.14.23291368</a></p>
        <p class="abstract">Obesity is a major risk factor for many common diseases and has a significant heritable component. While clinical and large-scale population studies have identified several genes harbouring rare alleles with large effects on obesity risk, there are likely many unknown genes with highly penetrant effects remaining. To this end, we performed whole exome-sequence analyses for adult body mass index (BMI) in up to 587,027 individuals. We identified rare, loss of function variants in two genes - BSN and APBA1 - with effects on BMI substantially larger than well-established obesity genes such as MC4R. One in [~]6500 individuals carry a heterozygous protein truncating variant (PTV) in BSN, which confers a 6.6, 3.7 and 3-fold higher risk of severe obesity (BMI &gt;40kg/m2), non-alcoholic fatty liver disease and type 2 diabetes, respectively. In contrast to most other obesity-related genes, rare variants in BSN and APBA1 had no apparent effect on childhood adiposity. Furthermore, BSN PTVs magnified the influence of common genetic variants associated with BMI, with a common polygenic score exhibiting an effect on BMI twice as large in BSN PTV carriers than non-carriers. Finally, we explored the plasma proteomic signatures of BSN PTV carriers as well as the functional consequences of BSN deletion in human iPSC-derived hypothalamic neurons. These approaches highlighted a network of differentially expressed genes that were collectively enriched for genomic regions associated with BMI, and suggest a role for degenerative neuronal synaptic function and neurotransmitter release in the etiology of obesity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.23291322">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.23291322" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.23291322">
        <p class="paperTitle">Understanding the genetic complexity of puberty timing across the allele frequency spectrum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.23291322" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.23291322" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kentistou, K. A.; Kaisinger, L. R.; Stankovic, S.; Vaudel, M.; de Oliveira, E. M.; Messina, A.; Walters, R. G.; Liu, X.; Busch, A. S.; Helgason, H.; Thompson, D. J.; Santon, F.; Petricek, K. M.; Zouaghi, Y.; Huang-Doran, I.; Gudbjartsson, D. F.; Bratland, E.; Lin, K.; Gardner, E. J.; Zhao, Y.; Jia, R.; Terao, C.; Riggan, M.; Bolla, M. K.; Yazdanpanah, M.; Yazdanpanah, N.; Bradfield, J. P.; Broer, L.; Campbell, A.; Chasman, D. I.; Cousminer, D. L.; Franceschini, N.; Franke, L. H.; Girotto, G.; He, C.; Jarvelin, M.-R.; Joshi, P. K.; Kamatani, Y.; Karlsson, R.; Luan, J.; Lunetta, K. L.; Magi, R.;</p>
        <p class="info">Score: 4.0, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.23291322' target='https://doi.org/10.1101/2023.06.14.23291322'> 10.1101/2023.06.14.23291322</a></p>
        <p class="abstract">Pubertal timing varies considerably and has been associated with a range of health outcomes in later life. To elucidate the underlying biological mechanisms, we performed multi-ancestry genetic analyses in [~]800,000 women, identifying 1,080 independent signals associated with age at menarche. Collectively these loci explained 11% of the trait variance in an independent sample, with women at the top and bottom 1% of polygenic risk exhibiting a [~]11 and [~]14-fold higher risk of delayed and precocious pubertal development, respectively. These common variant analyses were supported by exome sequence analysis of [~]220,000 women, identifying several genes, including rare loss of function variants in ZNF483 which abolished the impact of polygenic risk. Next, we implicated 660 genes in pubertal development using a combination of in silico variant-to-gene mapping approaches and integration with dynamic gene expression data from mouse embryonic GnRH neurons. This included an uncharacterized G-protein coupled receptor GPR83, which we demonstrate amplifies signaling of MC3R, a key sensor of nutritional status. Finally, we identified several genes, including ovary-expressed genes involved in DNA damage response that co-localize with signals associated with menopause timing, leading us to hypothesize that the ovarian reserve might signal centrally to trigger puberty. Collectively these findings extend our understanding of the biological complexity of puberty timing and highlight body size dependent and independent mechanisms that potentially link reproductive timing to later life disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.23290588">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.23290588" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.23290588">
        <p class="paperTitle">Investigating the genetic relationship between depression symptoms and Alzheimer&#39;s disease in clinically diagnosed and proxy cases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.23290588" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.23290588" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilchrist, L.; Spargo, T. P.; Green, R. E.; Coleman, J. R. I.; Howard, D. M.; Thorp, J. G.; Adey, B.; Lord, J.; Davies, H. L.; Mundy, J.; Davies, M. R.; HuÌˆbel, C.; Bristow, S.; Lee, S. H.; Rogers, H.; Curtis, C.; Kalsi, G.; Arathimos, R.; Corbett, A.; Ballard, C.; Brooker, H.; Creese, B.; Aarsland, D.; Hampshire, A.; Velayudhan, L.; Eley, T. C.; Breen, G.; Iacoangeli, A.; Koks, S.; Lewis, C. M.; Proitsi, P.; Alzheimer&#39;s Disease Neuroimaging Initiative,  ; AddNeuroMed,  ; GERAD1 Consortium,  </p>
        <p class="info">Score: 16.5, Published: 2023-06-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.23290588' target='https://doi.org/10.1101/2023.06.05.23290588'> 10.1101/2023.06.05.23290588</a></p>
        <p class="abstract">Depression is a risk factor for the later development of Alzheimers disease (AD), but evidence for their genetic relationship is mixed. Assessing depression symptom specific genetic associations may better clarify this relationship.

Using data from the UK Biobank, the GLAD Study and PROTECT, we performed the largest genome-wide meta-analyses (GWAS) of the nine depression symptom items, plus their sum score, on the Patient Health Questionnaire (PHQ-9) (GWAS equivalent N: 224,535-308,421). We then assessed global and local genetic correlations and statistical colocalisation between depression/depression symptoms and AD across six AD GWAS with varying proportions of clinical and proxy (family history) case ascertainment. We assessed bi-directional causal associations using Mendelian randomisation (MR) and tested the predictive utility of depression/depression symptom polygenic risk scores (PRS) for AD case/control status in three clinical AD cohorts.

Our GWAS meta-analyses identified 37 genomic risk loci across the ten depression symptom phenotypes. Of the 72 global genetic correlation tests conducted between depression/depression symptoms and AD, 20 were significant at pFDR [&amp;le;] 0.05. Only one of these was identified with AD GWAS containing clinical-only cases. Local genetic correlations were detected at 14 loci. Statistical colocalisation was not identified at these loci but was identified in the region of transmembrane protein 106B (TMEM106B) between multiple depression phenotypes and both clinical-only and clinical&#43;proxy AD. MR and PRS analyses did not yield significant results after multiple testing correction.

Our findings do not demonstrate a causal role of depression/depression symptoms on AD and suggest that previous evidence of genetic overlap between depression and AD may be driven by the inclusion of family-history-based proxy cases/controls. However, the identification of colocalisation at TMEM106B warrants further investigation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
