<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Clonal hematopoiesis associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution
Authors: Vlasschaert, C.; Buttigieg, M.; Pershad, Y.; Lanktree, M. B.; Aldrich, M. C.; Rauh, M. J.; Bick, A. G.
Score: 26.2, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301439
Small particulate matter air pollution (PM2.5) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM2.5 recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Clonal hematopoiesis associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution
Authors: Vlasschaert, C.; Buttigieg, M.; Pershad, Y.; Lanktree, M. B.; Aldrich, M. C.; Rauh, M. J.; Bick, A. G.
Score: 26.2, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301439
Small particulate matter air pollution (PM2.5) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM2.5 recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Clonal hematopoiesis associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution
Authors: Vlasschaert, C.; Buttigieg, M.; Pershad, Y.; Lanktree, M. B.; Aldrich, M. C.; Rauh, M. J.; Bick, A. G.
Score: 26.2, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301439
Small particulate matter air pollution (PM2.5) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM2.5 recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Clonal hematopoiesis associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution\nAuthors: Vlasschaert, C.; Buttigieg, M.; Pershad, Y.; Lanktree, M. B.; Aldrich, M. C.; Rauh, M. J.; Bick, A. G.\nScore: 26.2, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301439\nSmall particulate matter air pollution (PM2.5) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM2.5 recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer.",
  "keywords": [
    
  ],
  "articleBody": " Clonal hematopoiesis associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution\nAuthors: Vlasschaert, C.; Buttigieg, M.; Pershad, Y.; Lanktree, M. B.; Aldrich, M. C.; Rauh, M. J.; Bick, A. G.\nScore: 26.2, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301439\nSmall particulate matter air pollution (PM2.5) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM2.5 recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM2.5-associated NSCLC in non-smokers using genetic, environmental, and phenotypic data from 413,901 individuals in the UK Biobank. Among non-smokers, PM2.5 is not associated with NSCLC and not associated with prevalence of CHIP, but CHIP is associated with a doubling of NSCLC risk (hazard ratio (HR) 2.01, 95% confidence interval (CI): 1.34-3.00). Moreover, CHIP-associated NSCLC risk is exacerbated in the setting of above-median PM2.5 levels (HR 2.70, 95% CI: 1.60-4.55). PM2.5 x CHIP is also associated with significantly greater markers of systemic inflammation (CRP, IL-6, and IL-1B) than expected. Altogether, these results suggest CHIP and PM2.5 form a novel gene x environment interaction promoting NSCLC tumorigenesis in non-smokers.\nConcordance of whole-genome amplified embryonic DNA with the subsequently born child\nAuthors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.\nScore: 21.7, Published: 2024-01-15 DOI: 10.1101/2024.01.12.24301086\nBefore implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGSs ability to identify de novo variants we compared the childs genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimers Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGSs capability to identify genetic variants not explicitly targeted for monogenic screening.\nEstimating Disorder Probability Based on Polygenic Prediction Using the BPC Approach\nAuthors: Uffelmann, E.; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, ; Schizophrenia Working Group of the Psychiatric Genomics Consortium, ; Price, A. L.; Posthuma, D.; Peyrot, W. J.\nScore: 21.6, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301157\nPolygenic Scores (PGSs) summarize an individuals genetic propensity for a given trait in a single value, based on SNP effect sizes derived from Genome-Wide Association Study (GWAS) results. Methods have been developed that apply Bayesian approaches to improve the prediction accuracy of PGSs through optimization of estimated effect sizes. While these methods are generally well-calibrated for continuous traits (implying the predicted values are on average equal to the true trait values), they are not well-calibrated for binary disorder traits in ascertained samples. This is a problem because well-calibrated PGSs are needed to reliably compute the absolute disorder probability for an individual to facilitate future clinical implementation. Here we introduce the Bayesian polygenic score Probability Conversion (BPC) approach, which computes an individuals predicted disorder probability using GWAS summary statistics, an existing Bayesian PGS method (e.g. PRScs, SBayesR), the individuals genotype data, and a prior disorder probability. The BPC approach transforms the PGS to its underlying liability scale, computes the variances of the PGS in cases and controls, and applies Bayes Theorem to compute the absolute disorder probability; it is practical in its application as it does not require a tuning dataset with both genotype and phenotype data. We applied the BPC approach to extensive simulated data and empirical data of nine disorders. The BPC approach yielded well-calibrated results that were consistently better than the results of another recently published approach.\nImpact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 35.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.\nBreaking down causes, consequences, and mediating effects of age-related telomere shortening on human health\nAuthors: Moix, S.; Sadler, M. C.; Kutalik, Z.; Auwerx, C.\nScore: 13.3, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301196\nTelomeres represent repeated DNA sequences at the ends of chromosomes, which shorten with each cell division. Factors modulating telomere attrition and the health consequences thereof are not fully understood. To address this, we leveraged data from 326,363 unrelated UK Biobank participants of European ancestry and used linear regression and bidirectional univariable and multivariable Mendelian randomization (MR and MVMR) to elucidate the relationships between leukocyte telomere length (LTL) and 141 complex traits, including diseases, biomarkers, and lifestyle factors. We confirm that telomeres shorten with age and show a stronger decline in males than in females, which cannot be explained by hormonal or lifestyle differences. MR revealed 23 traits modulating LTL; e.g., smoking cessation and high educational attainment associated with longer LTL, while weekly alcohol intake, body mass index, urate levels, and female reproductive events, such as childbirth, associated with shorter LTL. We also identified 26 traits affected by LTL, with risk for cardiovascular, pulmonary, renal, and some autoimmune diseases being increased by short LTL, while longer LTL increased risk for other autoimmune conditions and cancers. Through MVMR, we show that LTL partially mediates the impact of educational attainment, body mass index, and female age at childbirth on lifespan. These results provide new insights into the biology of telomere regulation by shedding light on the modulators, consequences, and the mediatory role of telomere shortening, portraying an intricate relationship between LTL, diseases, lifestyle, and socio-economic factors.\nLethal phenotypes in Mendelian disorders\nAuthors: Cacheiro, P.; Lawson, S.; Van den Veyver, I. B.; Marengo, G.; Zocche, D.; Murray, S. A.; Duyzend, M.; Robinson, P. N.; Smedley, D.\nScore: 5.8, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301168\nEssential genes are those whose function is required for cell proliferation and/or organism survival. A genes intolerance to loss-of-function can be allocated within a spectrum, as opposed to being considered a binary feature, since this function might be essential at different stages of development, genetic backgrounds or other contexts. Existing resources that collect and characterise the essentiality status of genes are based on either proliferation assessment in human cell lines, embryonic and postnatal viability evaluation in different model organisms, and gene metrics such as intolerance to variation scores derived from human population sequencing studies. There are also several repositories available that document phenotypic annotations for rare disorders in humans such as the Online Mendelian Inheritance in Man (OMIM) and the Human Phenotype Ontology (HPO) knowledgebases. This raises the prospect of being able to use clinical data, including lethality as the most severe phenotypic manifestation, to further our characterisation of gene essentiality. Here we queried OMIM for terms related to lethality and classified all Mendelian genes into categories, according to the earliest age of death recorded for the associated disorders, from prenatal death to no reports of premature death. To showcase this curated catalogue of human essential genes, we developed the Lethal Phenotypes Portal (https://lethalphenotypes.research.its.qmul.ac.uk), where we also explore the relationships between these lethality categories, constraint metrics and viability in cell lines and mouse. Further analysis of the genes in these categories reveals differences in the mode of inheritance of the associated disorders, physiological systems affected and disease class. We highlight how the phenotypic similarity between genes in the same lethality category combined with gene family/group information can be used for novel disease gene discovery. Finally, we explore the overlaps and discrepancies between the lethal phenotypes observed in mouse and human and discuss potential explanations that include differences in transcriptional regulation, functional compensation and molecular disease mechanisms. We anticipate that this resource will aid clinicians in the diagnosis of early lethal conditions and assist researchers in investigating the properties that make these genes essential for human development.\nBenchmarking Mendelian Randomization methods for causal inference using genome-wide association study summary statistics\nAuthors: Hu, X.; Cai, M.; Xiao, J.; Wan, X.; Wang, Z.; Zhao, H.; Yang, C.\nScore: 16.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300765\nMendelian Randomization (MR), which utilizes genetic variants as instrumental variables (IVs), has gained popularity as a method for causal inference between phenotypes using genetic data. While efforts have been made to relax IV assumptions and develop new methods for causal inference in the presence of invalid IVs due to confounding, the reliability of MR methods in real-world applications remains uncertain. To bridge this gap, we conducted a benchmark study evaluating 15 MR methods using real-world genetic datasets. Our study focused on three crucial aspects: type I error control in the presence of various confounding scenarios (e.g., population stratification, pleiotropy, and assortative mating), the accuracy of causal effect estimates, replicability and power. By comprehensively evaluating the performance of compared methods over one thousand pairs of exposure-outcome traits, our study not only provides valuable insights into the performance and limitations of the compared methods but also offers practical guidance for researchers to choose appropriate MR methods for causal inference.\nIdentifying compounds to treat opiate use disorder by leveraging multi-omic data integration and multiple drug repurposing databases.\nAuthors: Stratford, J. K.; Carnes, M. U.; Willis, C.; Minto, M. S.; Elnimeiry, L.; Mathur, R.; Schu, M.; Quach, B. C.; Carter, J.; Nolen, T.; Vandergrift, N.; Kosten, T. R.; Johnson, E. O.; Webb, B. T.\nScore: 4.9, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301329\nGenes influencing opioid use disorder (OUD) biology have been identified via genome-wide association studies (GWAS), gene expression, and network analyses. These discoveries provide opportunities to identifying existing compounds targeting these genes for drug repurposing studies. However, systematically integrating discovery results and identifying relevant available pharmacotherapies for OUD repurposing studies is challenging. To address this, we've constructed a framework that leverages existing results and drug databases to identify candidate pharmacotherapies. For this study, two independent OUD related meta-analyses were used including a GWAS and a differential gene expression (DGE) study of post-mortem human brain. Protein-Protein Interaction (PPI) sub-networks enriched for GWAS risk loci were identified via network analyses. Drug databases Pharos, Open Targets, Therapeutic Target Database (TTD), and DrugBank were queried for clinical status and target selectivity. Cross-omic and drug query results were then integrated to identify candidate compounds. GWAS and DGE analyses revealed 3 and 335 target genes (FDR q\u003c0.05), respectively while network analysis detected 70 genes in 22 enriched PPI networks. Four selection strategies with different statistical thresholds were implemented, which yielded between 72 and 676 genes with statistically significant support and 110 to 683 drugs targeting these genes, respectively. After filtering out less specific compounds or those targeting well-established psychiatric-related receptors (OPRM1 and DRD2), between 2 and 329 approved drugs remained across the four strategies. By leveraging multiple lines of biological evidence and resources, we identified many FDA approved drugs that target genes associated with OUD. This approach a) allows high-throughput querying of OUD-related genes, b) detects OUD-related genes and compounds not identified using a single domain or resource, and c) produces a succinct summary of FDA approved compounds eligible for efficient expert review. Identifying larger pools of candidate pharmacotherapies and summarizing the supporting evidence bridges the gap between discovery and drug repurposing studies.\nPrimateAI-3D outperforms AlphaMissense in real-world cohorts\nAuthors: Parry, D. A.; Bosc, T.; Hamp, T.; Fiziev, P.; Sharma, A.; Kassam, I.; McRae, J.; Farh, K.\nScore: 5.8, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301193\nAccurately predicting the impact of genetic variants is essential for interpreting genomic data, yet no consensus exists on how to measure classifier performance. We prepared the most comprehensive set of benchmarks to date and applied them to the recently published models PrimateAI-3D and AlphaMissense. PrimateAI-3D outperforms AlphaMissense on rare-disease cohort and biobank benchmarks, indicating that performance on clinical databases or in vitro assays does not reliably generalize to real-world cohorts.\nCross-ancestry genetic investigation of schizophrenia, cannabis use disorder, and tobacco smoking\nAuthors: Johnson, E. C.; Austin-Zimmerman, I.; Thorpe, H. H.; Levey, D. F.; Baranger, D. A.; Colbert, S. M. C.; Demontis, D.; Khokhar, J. Y.; Davis, L. K.; Edenberg, H. J.; Di Forti, M.; Sanchez-Roige, S.; Gelernter, J.; Agrawal, A.\nScore: 2.9, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301430\nIndividuals with schizophrenia frequently experience co-occurring substance use, including tobacco smoking and heavy cannabis use, and substance use disorders. There is interest in understanding the extent to which these relationships are causal, and to what extent shared genetic factors play a role. We explored the relationships between schizophrenia (Scz), cannabis use disorder (CanUD), and ever-regular tobacco smoking (Smk) using the largest available genome-wide studies of these phenotypes in individuals of African and European ancestries. All three phenotypes were positively genetically correlated (rgs = 0.17 - 0.62). Causal inference analyses suggested the presence of horizontal pleiotropy, but evidence for bidirectional causal relationships was also found between all three phenotypes even after correcting for horizontal pleiotropy. We identified 439 pleiotropic loci in the European ancestry data, 150 of which were novel (i.e., not genome-wide significant in the original studies). Of these pleiotropic loci, 202 had lead variants which showed convergent effects (i.e., same direction of effect) on Scz, CanUD, and Smk. Genetic variants convergent across all three phenotypes showed strong genetic correlations with risk-taking, executive function, and several mental health conditions. Our results suggest that both horizontal pleiotropy and causal mechanisms may play a role in the relationship between CanUD, Smk, and Scz, but longitudinal, prospective studies are needed to confirm a causal relationship.\n",
  "wordCount" : "2524",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301439" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301439">
        <p class="paperTitle">Clonal hematopoiesis associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vlasschaert, C.; Buttigieg, M.; Pershad, Y.; Lanktree, M. B.; Aldrich, M. C.; Rauh, M. J.; Bick, A. G.</p>
        <p class="info">Score: 26.2, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301439' target='https://doi.org/10.1101/2024.01.17.24301439'> 10.1101/2024.01.17.24301439</a></p>
        <p class="abstract">Small particulate matter air pollution (PM2.5) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM2.5 recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM2.5-associated NSCLC in non-smokers using genetic, environmental, and phenotypic data from 413,901 individuals in the UK Biobank. Among non-smokers, PM2.5 is not associated with NSCLC and not associated with prevalence of CHIP, but CHIP is associated with a doubling of NSCLC risk (hazard ratio (HR) 2.01, 95% confidence interval (CI): 1.34-3.00). Moreover, CHIP-associated NSCLC risk is exacerbated in the setting of above-median PM2.5 levels (HR 2.70, 95% CI: 1.60-4.55). PM2.5 x CHIP is also associated with significantly greater markers of systemic inflammation (CRP, IL-6, and IL-1B) than expected. Altogether, these results suggest CHIP and PM2.5 form a novel gene x environment interaction promoting NSCLC tumorigenesis in non-smokers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301086" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301086">
        <p class="paperTitle">Concordance of whole-genome amplified embryonic DNA with the subsequently born child</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.</p>
        <p class="info">Score: 21.7, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301086' target='https://doi.org/10.1101/2024.01.12.24301086'> 10.1101/2024.01.12.24301086</a></p>
        <p class="abstract">Before implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGSs ability to identify de novo variants we compared the childs genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimers Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGSs capability to identify genetic variants not explicitly targeted for monogenic screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301157" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301157">
        <p class="paperTitle">Estimating Disorder Probability Based on Polygenic Prediction Using the BPC Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uffelmann, E.; Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium,  ; Schizophrenia Working Group of the Psychiatric Genomics Consortium,  ; Price, A. L.; Posthuma, D.; Peyrot, W. J.</p>
        <p class="info">Score: 21.6, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301157' target='https://doi.org/10.1101/2024.01.12.24301157'> 10.1101/2024.01.12.24301157</a></p>
        <p class="abstract">Polygenic Scores (PGSs) summarize an individuals genetic propensity for a given trait in a single value, based on SNP effect sizes derived from Genome-Wide Association Study (GWAS) results. Methods have been developed that apply Bayesian approaches to improve the prediction accuracy of PGSs through optimization of estimated effect sizes. While these methods are generally well-calibrated for continuous traits (implying the predicted values are on average equal to the true trait values), they are not well-calibrated for binary disorder traits in ascertained samples. This is a problem because well-calibrated PGSs are needed to reliably compute the absolute disorder probability for an individual to facilitate future clinical implementation. Here we introduce the Bayesian polygenic score Probability Conversion (BPC) approach, which computes an individuals predicted disorder probability using GWAS summary statistics, an existing Bayesian PGS method (e.g. PRScs, SBayesR), the individuals genotype data, and a prior disorder probability. The BPC approach transforms the PGS to its underlying liability scale, computes the variances of the PGS in cases and controls, and applies Bayes Theorem to compute the absolute disorder probability; it is practical in its application as it does not require a tuning dataset with both genotype and phenotype data. We applied the BPC approach to extensive simulated data and empirical data of nine disorders. The BPC approach yielded well-calibrated results that were consistently better than the results of another recently published approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301165">
        <p class="paperTitle">Impact of genome build on RNA-seq interpretation and diagnostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network,  ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.</p>
        <p class="info">Score: 35.0, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301165' target='https://doi.org/10.1101/2024.01.11.24301165'> 10.1101/2024.01.11.24301165</a></p>
        <p class="abstract">Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301196" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301196">
        <p class="paperTitle">Breaking down causes, consequences, and mediating effects of age-related telomere shortening on human health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moix, S.; Sadler, M. C.; Kutalik, Z.; Auwerx, C.</p>
        <p class="info">Score: 13.3, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301196' target='https://doi.org/10.1101/2024.01.12.24301196'> 10.1101/2024.01.12.24301196</a></p>
        <p class="abstract">Telomeres represent repeated DNA sequences at the ends of chromosomes, which shorten with each cell division. Factors modulating telomere attrition and the health consequences thereof are not fully understood. To address this, we leveraged data from 326,363 unrelated UK Biobank participants of European ancestry and used linear regression and bidirectional univariable and multivariable Mendelian randomization (MR and MVMR) to elucidate the relationships between leukocyte telomere length (LTL) and 141 complex traits, including diseases, biomarkers, and lifestyle factors. We confirm that telomeres shorten with age and show a stronger decline in males than in females, which cannot be explained by hormonal or lifestyle differences. MR revealed 23 traits modulating LTL; e.g., smoking cessation and high educational attainment associated with longer LTL, while weekly alcohol intake, body mass index, urate levels, and female reproductive events, such as childbirth, associated with shorter LTL. We also identified 26 traits affected by LTL, with risk for cardiovascular, pulmonary, renal, and some autoimmune diseases being increased by short LTL, while longer LTL increased risk for other autoimmune conditions and cancers. Through MVMR, we show that LTL partially mediates the impact of educational attainment, body mass index, and female age at childbirth on lifespan. These results provide new insights into the biology of telomere regulation by shedding light on the modulators, consequences, and the mediatory role of telomere shortening, portraying an intricate relationship between LTL, diseases, lifestyle, and socio-economic factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301168" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301168">
        <p class="paperTitle">Lethal phenotypes in Mendelian disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cacheiro, P.; Lawson, S.; Van den Veyver, I. B.; Marengo, G.; Zocche, D.; Murray, S. A.; Duyzend, M.; Robinson, P. N.; Smedley, D.</p>
        <p class="info">Score: 5.8, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301168' target='https://doi.org/10.1101/2024.01.12.24301168'> 10.1101/2024.01.12.24301168</a></p>
        <p class="abstract">Essential genes are those whose function is required for cell proliferation and/or organism survival. A genes intolerance to loss-of-function can be allocated within a spectrum, as opposed to being considered a binary feature, since this function might be essential at different stages of development, genetic backgrounds or other contexts. Existing resources that collect and characterise the essentiality status of genes are based on either proliferation assessment in human cell lines, embryonic and postnatal viability evaluation in different model organisms, and gene metrics such as intolerance to variation scores derived from human population sequencing studies. There are also several repositories available that document phenotypic annotations for rare disorders in humans such as the Online Mendelian Inheritance in Man (OMIM) and the Human Phenotype Ontology (HPO) knowledgebases. This raises the prospect of being able to use clinical data, including lethality as the most severe phenotypic manifestation, to further our characterisation of gene essentiality. Here we queried OMIM for terms related to lethality and classified all Mendelian genes into categories, according to the earliest age of death recorded for the associated disorders, from prenatal death to no reports of premature death. To showcase this curated catalogue of human essential genes, we developed the Lethal Phenotypes Portal (https://lethalphenotypes.research.its.qmul.ac.uk), where we also explore the relationships between these lethality categories, constraint metrics and viability in cell lines and mouse. Further analysis of the genes in these categories reveals differences in the mode of inheritance of the associated disorders, physiological systems affected and disease class. We highlight how the phenotypic similarity between genes in the same lethality category combined with gene family/group information can be used for novel disease gene discovery. Finally, we explore the overlaps and discrepancies between the lethal phenotypes observed in mouse and human and discuss potential explanations that include differences in transcriptional regulation, functional compensation and molecular disease mechanisms. We anticipate that this resource will aid clinicians in the diagnosis of early lethal conditions and assist researchers in investigating the properties that make these genes essential for human development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300765" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300765">
        <p class="paperTitle">Benchmarking Mendelian Randomization methods for causal inference using genome-wide association study summary statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, X.; Cai, M.; Xiao, J.; Wan, X.; Wang, Z.; Zhao, H.; Yang, C.</p>
        <p class="info">Score: 16.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300765' target='https://doi.org/10.1101/2024.01.03.24300765'> 10.1101/2024.01.03.24300765</a></p>
        <p class="abstract">Mendelian Randomization (MR), which utilizes genetic variants as instrumental variables (IVs), has gained popularity as a method for causal inference between phenotypes using genetic data. While efforts have been made to relax IV assumptions and develop new methods for causal inference in the presence of invalid IVs due to confounding, the reliability of MR methods in real-world applications remains uncertain. To bridge this gap, we conducted a benchmark study evaluating 15 MR methods using real-world genetic datasets. Our study focused on three crucial aspects: type I error control in the presence of various confounding scenarios (e.g., population stratification, pleiotropy, and assortative mating), the accuracy of causal effect estimates, replicability and power. By comprehensively evaluating the performance of compared methods over one thousand pairs of exposure-outcome traits, our study not only provides valuable insights into the performance and limitations of the compared methods but also offers practical guidance for researchers to choose appropriate MR methods for causal inference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301329">
        <p class="paperTitle">Identifying compounds to treat opiate use disorder by leveraging multi-omic data integration and multiple drug repurposing databases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stratford, J. K.; Carnes, M. U.; Willis, C.; Minto, M. S.; Elnimeiry, L.; Mathur, R.; Schu, M.; Quach, B. C.; Carter, J.; Nolen, T.; Vandergrift, N.; Kosten, T. R.; Johnson, E. O.; Webb, B. T.</p>
        <p class="info">Score: 4.9, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301329' target='https://doi.org/10.1101/2024.01.17.24301329'> 10.1101/2024.01.17.24301329</a></p>
        <p class="abstract">Genes influencing opioid use disorder (OUD) biology have been identified via genome-wide association studies (GWAS), gene expression, and network analyses. These discoveries provide opportunities to identifying existing compounds targeting these genes for drug repurposing studies. However, systematically integrating discovery results and identifying relevant available pharmacotherapies for OUD repurposing studies is challenging. To address this, we&#39;ve constructed a framework that leverages existing results and drug databases to identify candidate pharmacotherapies. For this study, two independent OUD related meta-analyses were used including a GWAS and a differential gene expression (DGE) study of post-mortem human brain. Protein-Protein Interaction (PPI) sub-networks enriched for GWAS risk loci were identified via network analyses. Drug databases Pharos, Open Targets, Therapeutic Target Database (TTD), and DrugBank were queried for clinical status and target selectivity. Cross-omic and drug query results were then integrated to identify candidate compounds. GWAS and DGE analyses revealed 3 and 335 target genes (FDR q&lt;0.05), respectively while network analysis detected 70 genes in 22 enriched PPI networks. Four selection strategies with different statistical thresholds were implemented, which yielded between 72 and 676 genes with statistically significant support and 110 to 683 drugs targeting these genes, respectively. After filtering out less specific compounds or those targeting well-established psychiatric-related receptors (OPRM1 and DRD2), between 2 and 329 approved drugs remained across the four strategies. By leveraging multiple lines of biological evidence and resources, we identified many FDA approved drugs that target genes associated with OUD. This approach a) allows high-throughput querying of OUD-related genes, b) detects OUD-related genes and compounds not identified using a single domain or resource, and c) produces a succinct summary of FDA approved compounds eligible for efficient expert review. Identifying larger pools of candidate pharmacotherapies and summarizing the supporting evidence bridges the gap between discovery and drug repurposing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301193" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301193">
        <p class="paperTitle">PrimateAI-3D outperforms AlphaMissense in real-world cohorts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parry, D. A.; Bosc, T.; Hamp, T.; Fiziev, P.; Sharma, A.; Kassam, I.; McRae, J.; Farh, K.</p>
        <p class="info">Score: 5.8, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301193' target='https://doi.org/10.1101/2024.01.12.24301193'> 10.1101/2024.01.12.24301193</a></p>
        <p class="abstract">Accurately predicting the impact of genetic variants is essential for interpreting genomic data, yet no consensus exists on how to measure classifier performance. We prepared the most comprehensive set of benchmarks to date and applied them to the recently published models PrimateAI-3D and AlphaMissense. PrimateAI-3D outperforms AlphaMissense on rare-disease cohort and biobank benchmarks, indicating that performance on clinical databases or in vitro assays does not reliably generalize to real-world cohorts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301430" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301430">
        <p class="paperTitle">Cross-ancestry genetic investigation of schizophrenia, cannabis use disorder, and tobacco smoking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, E. C.; Austin-Zimmerman, I.; Thorpe, H. H.; Levey, D. F.; Baranger, D. A.; Colbert, S. M. C.; Demontis, D.; Khokhar, J. Y.; Davis, L. K.; Edenberg, H. J.; Di Forti, M.; Sanchez-Roige, S.; Gelernter, J.; Agrawal, A.</p>
        <p class="info">Score: 2.9, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301430' target='https://doi.org/10.1101/2024.01.17.24301430'> 10.1101/2024.01.17.24301430</a></p>
        <p class="abstract">Individuals with schizophrenia frequently experience co-occurring substance use, including tobacco smoking and heavy cannabis use, and substance use disorders. There is interest in understanding the extent to which these relationships are causal, and to what extent shared genetic factors play a role. We explored the relationships between schizophrenia (Scz), cannabis use disorder (CanUD), and ever-regular tobacco smoking (Smk) using the largest available genome-wide studies of these phenotypes in individuals of African and European ancestries. All three phenotypes were positively genetically correlated (rgs = 0.17 - 0.62). Causal inference analyses suggested the presence of horizontal pleiotropy, but evidence for bidirectional causal relationships was also found between all three phenotypes even after correcting for horizontal pleiotropy. We identified 439 pleiotropic loci in the European ancestry data, 150 of which were novel (i.e., not genome-wide significant in the original studies). Of these pleiotropic loci, 202 had lead variants which showed convergent effects (i.e., same direction of effect) on Scz, CanUD, and Smk. Genetic variants convergent across all three phenotypes showed strong genetic correlations with risk-taking, executive function, and several mental health conditions. Our results suggest that both horizontal pleiotropy and causal mechanisms may play a role in the relationship between CanUD, Smk, and Scz, but longitudinal, prospective studies are needed to confirm a causal relationship.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
