<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Impact of genome build on RNA-seq interpretation and diagnostics
Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.
Score: 15.9, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165
Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Impact of genome build on RNA-seq interpretation and diagnostics
Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.
Score: 15.9, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165
Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Impact of genome build on RNA-seq interpretation and diagnostics
Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.
Score: 15.9, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165
Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Impact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 15.9, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses.",
  "keywords": [
    
  ],
  "articleBody": " Impact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 15.9, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.\nA framework for accurate prediction of effector genes in genetic loci for complex traits\nAuthors: Schipper, M.; de Leeuw, C. A.; Maciel, B. A. P. C.; Wightman, D. P.; Hubers, N.; Boomsma, D. I.; O'Donovan, M. C.; Posthuma, D.\nScore: 14.5, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300360\nGenome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus associations remains challenging. Here we present the FLAMES framework, which predicts the most likely effector gene in a locus. FLAMES integrates machine learning with biological data linking single nucleotide polymorphisms (SNPs) to genes, and GWAS-wide convergence of gene interactions. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication, and domain knowledge of molecular traits. We demonstrate that combining SNP-based and convergence-based evidence outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find novel schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life.\nThe Metabolic Role of MAP3K15: Genetic and Phenotypic Insights from the 23andMe Research Database and Genetics-Driven Recruitment\nAuthors: Brady, J. J.; Kalkus, K.; Nguyen, D. T.; Wen, J.; Fuller, Z. L.; Qi, Y.; Deering, S. S.; Selcer, L.; Shringarpure, S. S.; The 23andMe Research Team, ; Holmes, M. V.; Wang, X.\nScore: 13.0, Published: 2024-01-10 DOI: 10.1101/2024.01.09.24301012\nMAP3K15 has been previously associated with protection from type 2 diabetes (T2D), prompting interest in the development of MAP3K15 inhibitors as a potential therapeutic option for diabetes. The trans-ancestry genome-wide association study (GWAS) meta-analysis and loss-of-function (LoF) burden testing methods that implicate association with T2D greatly benefit from large sample size. The direct-to-consumer genetic testing company, 23andMe, Inc., is the worlds largest research consented genetic database. We leveraged the 23andMe database to further inform the metabolic role of MAP3K15, using a variety of genetic analysis methods. We find that MAP3K15 LoF carriers show a significant delay of 4.5 years in the median age of T2D diagnosis among individuals at high polygenic risk and uncover a novel burden association of MAP3K15 LoF with protection against high cholesterol. We expanded these findings by establishing a capability to recruit consented participants on the basis of genetics unknown to them (specifically, a single LoF variant in MAP3K15, rs148312150), and obtained clinical laboratory evidence of a modest reduction in median cholesterol and LDL/HDL ratio in MAP3K15 LoF carriers. Our findings demonstrate the discovery power of the 23andMe database, including the feasibility of consented participant recruitment to inform therapeutic discovery and development.\nValid inference for machine learning-assisted GWAS\nAuthors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.\nScore: 20.8, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300779\nMachine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research. However, the validity of associations identified from ML-assisted GWAS has not been carefully evaluated. In this study, we report pervasive risks for false positive associations in ML-assisted GWAS, and introduce POP-GWAS, a novel statistical framework that reimagines GWAS on ML-imputed outcomes. POP-GWAS provides valid statistical inference irrespective of the quality of imputation or variables and algorithms used for imputation. It also only requires GWAS summary statistics as input. We employed POP-GWAS to perform the largest GWAS of bone mineral density (BMD) derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 novel loci reaching genome-wide significance and revealing skeletal site-specific genetic architecture of BMD. Our framework may fundamentally reshape the analytical strategies in future ML-assisted GWAS.\nThe Solute Carrier Family 26 Member 9 Is a Modifier of the Rapidly Progressing Cystic Fibrosis Associated with F508del CFTR Mutations\nAuthors: Luo, S.; Rollins, S.; Schmitz-Abe, K.; Tam, A.; Li, Q.; Shi, J.; Lin, J.; Wang, R.; Agrawal, P.\nScore: 7.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.23300546\nCystic fibrosis (CF) is an autosomal recessive disease caused by mutations to the CF transmembrane conductance regulator (CFTR). Symptoms and severity of the disease vary shown that modifier genes influence disease severity and clinical course. We previously reported epithelial sodium channel (ENaC) genes as modifiers of disease severity in long-term non-progressors sharing deltaF508 homozygous for CFTR genotype. Here we describe the opposite, modifier genes that may be associated with rapidly progressing CF (RPCF) in a cohort of patients with the shared deltaF508 homozygous genotype. We have identified three rare missense SLC26A9 variants in four individuals (out of six) deemed to have RPCF: c.229G\u003eA; p.G77S (present in two patients), c.1885C\u003eT; p.P629S and c.2546G\u003eA; p.R849Q. By analyzing publicly available single cell sequencing dataset from LungMAP, we revealed that both SLC26A9 and CFTR mRNA are highly enriched in Alveolar type 2 (AT2) cells, with a few (greater than 1%) in respiratory airway secretory (RAS) cells and ionocytes. Structural modeling suggests deleterious effects of these mutations as they are in critical protein domains which might affect the ion transportation capability of SLC26A9. The enrichment of rare and potentially deleterious SLC26A9 mutations in patients with RPCF suggests SLC26A9 is a modifier gene associated with RPCF.\nRNA methyltransferase SPOUT1/CENP-32 links mitotic spindle organization with the neurodevelopmental disorder SpADMiSS\nAuthors: Dharmadhikari, A. V.; Abad, M. A.; Khan, S.; Maroofian, R.; Sands, T. T.; Ullah, F.; Samejima, I.; Wear, M. A.; Moore, K. E.; Kondakova, E.; Mitina, N.; Schaub, T.; Lee, G. K.; Umandap, C. H.; Berger, S. M.; Iglesias, A. D.; Popp, B.; Jamra, R. A.; Gabriel, H.; Rentas, S.; Rippert, A. L.; Izumi, K.; Conlin, L. K.; Koboldt, D. C.; Mihalic Mosher, T.; Hickey, S. E.; Albert, D. V.; Norwood, H.; Lewanda, A. F.; Dai, H.; Liu, P.; Mitani, T.; Marafi, D.; Pehlivan, D.; Posey, J. E.; Lippa, N.; Vena, N.; Heinzen, E. L.; Goldstein, D. B.; Mignot, C.; Agathe, J.-M. d. S.; Al-Sannaa, N. A.; Zamani, M.; S\nScore: 6.6, Published: 2024-01-09 DOI: 10.1101/2024.01.09.23300329\nSPOUT1/CENP-32 encodes a putative SPOUT RNA methyltransferase previously identified as a mitotic chromosome associated protein. SPOUT1/CENP-32 depletion leads to centrosome detachment from the spindle poles and chromosome misalignment. Aided by gene matching platforms, we identified 24 individuals with neurodevelopmental delays from 18 families with bi-allelic variants in SPOUT1/CENP-32 detected by exome/genome sequencing. Zebrafish spout1/cenp-32 mutants showed reduction in larval head size with concomitant apoptosis likely associated with altered cell cycle progression. In vivo complementation assays in zebrafish indicated that SPOUT1/CENP-32 missense variants identified in humans are pathogenic. Crystal structure analysis of SPOUT1/CENP-32 revealed that most disease-associated missense variants mapped to the catalytic domain. Additionally, SPOUT1/CENP-32 recurrent missense variants had reduced methyltransferase activity in vitro and compromised centrosome tethering to the spindle poles in human cells. Thus, SPOUT1/CENP-32 pathogenic variants cause an autosomal recessive neurodevelopmental disorder: SpADMiSS (SPOUT1 Associated Development delay Microcephaly Seizures Short stature) underpinned by mitotic spindle organization defects and consequent chromosome segregation errors.\nEHMT2 LOSS-OF-FUNCTION ALTERATIONS CAUSE A KLEEFSTRA-LIKE SYNDROME\nAuthors: Martinez-Delgado, B.; Lopez-Martin, E.; Kerkhof, J.; Baladron, B.; Mielu, L. M.; Sanchez-Ponce, D.; Bada-Navarro, A.; Herrero-Matesanz, M.; Lopez-Jimenez, L.; Rzasa, J.; Rots, D.; Fernandez, M.; Hernandez-San Miguel, E.; Gomez-Mariano, G.; Marin-Reina, P.; Cazorla-Calleja, R.; Alonso, J.; Kleefstra, T.; Posada, M.; Bermejo-Sanchez, E.; Sadikovic, B.; Barrero, M. J.\nScore: 5.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24300997\nDysregulation of the epigenetic machinery is associated with neurodevelopmental defects in humans. Kleefstra syndrome (KS) is a neurodevelopmental syndrome caused by heterozygous alterations in the gene EHMT1 that cause loss-of-function. EHTM1 and EHMT2 are highly similar histone methyltransferases that play relevant roles in development. Despite their similarity, individuals with alterations in EHMT2 have never been described. Here, we describe a pediatric patient with a KS-overlapping phenotype and a single base de novo substitution in EHMT2 that causes the amino acid change p.Ala1077Ser in the catalytic SET domain. This change causes a reduction in the affinity of the catalytic domain for the H3 tail and in the activity of the enzyme by three- to five-fold. DNA methylation, histone methylation and gene expression profiles suggest a significant overlap between the EHMT2 p.Ala1077Ser variant and KS. Based on this evidence we suggest that EHMT2 haploinsufficiency causes a Kleefstra-like syndrome. Although we cannot rule out dominant negative effects caused by the EHMT2 p.Ala1077Ser variant, our data and previously published data suggest that loss of EHMT2 function is probably more detrimental to cells than loss of EHMT1, explaining why individuals with alterations in EHMT2 are very rare.\nComplex trait associations in rare diseases and impacts on Mendelian variant interpretation\nAuthors: Smail, C.; Ge, B.; Keever-Keigher, M. R.; Schwendinger-Schreck, C.; Cheung, W. A.; Johnston, J. J.; Barrett, C.; Genomic Answers for Kids Consortium, ; Feldman, K.; Cohen, A. S.; Farrow, E. G.; Thiffault, I.; Grundberg, E.; Pastinen, T.\nScore: 5.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24301111\nEmerging evidence implicates common genetic variation - aggregated into polygenic scores (PGS) - impacting the onset and phenotypic presentation of rare diseases. In this study, we quantified individual polygenic liability for 1,151 previously published PGS in a cohort of 2,374 probands enrolled in the Genomic Answers for Kids (GA4K) rare disease study, revealing widespread associations between rare disease phenotypes and PGSs for common complex diseases and traits, blood protein levels, and brain and other organ morphological measurements. We observed increased polygenic burden in probands with variants of unknown significance (VUS) compared to unaffected carrier parents. We further observed an enrichment in overlap between diagnostic and candidate rare disease genes and large-effect PGS genes. Overall, our study supports and expands on previous findings of complex trait associations in rare disease phenotypes and provides a framework for identifying novel candidate rare disease genes and in understanding variable penetrance of candidate Mendelian disease variants.\nAssessing the lack of diversity in genetics research across neurodegenerative diseases: a systematic review of the GWAS Catalog and literature\nAuthors: Leonard, H.; Jonson, C.; Levine, K.; Lake, J.; Hertslet, L.; Jones, L.; Patel, D.; Kim, J.; Bandres-Ciga, S.; Terry, N.; Mata, I. F.; Blauwendraat, C.; Singleton, A. B.; Yokoyama, J. S.\nScore: 4.9, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301007\nImportanceThe under-representation of participants with non-European ancestry in genome-wide association studies (GWAS) is a critical issue that has significant implications, including hindering the progress of precision medicine initiatives. This issue is particularly significant in the context of neurodegenerative diseases (NDDs), where current therapeutic approaches have shown limited success. Addressing this under-representation is crucial to harnessing the full potential of genomic medicine in underserved communities and improving outcomes for NDD patients. ObjectiveOur primary objective was to assess the representation of non-European ancestry participants in genetic discovery efforts related to NDDs. We aimed to quantify the extent of inclusion of diverse ancestry groups in NDD studies and determine the number of associated loci identified in more inclusive studies. Specifically, we sought to highlight the disparities in research efforts and outcomes between studies predominantly involving European ancestry participants and those deliberately targeting non-European or multi-ancestry populations across NDDs. Evidence ReviewWe conducted a systematic review utilizing existing GWAS results and publications to assess the inclusion of diverse ancestry groups in neurodegeneration and neurogenetics studies. Our search encompassed studies published up to the end of 2022, with a focus on identifying research that deliberately included non-European or multi-ancestry cohorts. We employed rigorous methods for the inclusion of identified articles and quality assessment. FindingsOur review identified a total of 123 NDD GWAS. Strikingly, 82% of these studies predominantly featured participants of European ancestry. Endeavors specifically targeting non-European or multi-ancestry populations across NDDs identified only 52 risk loci. This contrasts with predominantly European studies, which reported over 90 risk loci for a single disease. Encouragingly, over 65% of these discoveries occurred in 2020 or later, indicating a recent increase in studies deliberately including non-European cohorts. Conclusions and relevanceOur findings underscore the pressing need for increased diversity in neurodegenerative research. The significant under-representation of non-European ancestry participants in NDD GWAS limits our understanding of the genetic underpinnings of these diseases. To advance the field of neurodegenerative research and develop more effective therapies, it is imperative that future investigations prioritize and harness the genomic diversity present within and across global populations. Abstract and highlightsO_ST_ABSKey PointsC_ST_ABSQuestionWhat is the state of ancestral inclusivity in genetic studies of neurodegenerative diseases? FindingsA systematic review of 123 publications on neurodegenerative diseases shows a focus on European populations, with only 18% of studies including any non-European ancestry data. Among 52 novel loci identified in non-European studies, 28 were from multi-ancestry studies (which included Europeans), 21 from East Asian studies, and 3 from other populations. MeaningThis significant disparity underscores the need for more inclusive research approaches in neurodegenerative diseases, emphasizing multi-ancestry and non-European populations to advance precision medicine and develop treatments effective for diverse populations.\nmixWAS: An efficient distributed algorithm for mixed-outcomes genome-wide association studies\nAuthors: Li, R.; Benz, L.; Duan, R.; Denny, J.; Hakonarson, H.; Mosley, J.; Smoller, J. W.; Wei, W.-Q.; Ritchie, M. D.; Moore, J. H.; Chen, Y.\nScore: 4.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.24301073\nGenome-wide association studies (GWAS) have been instrumental in identifying genetic associations for various diseases and traits. However, uncovering genetic underpinnings among traits beyond univariate phenotype associations remains a challenge. Multi-phenotype associations (MPA), or genetic pleiotropy, offer important insights into shared genes and pathways among traits, enhancing our understanding of genetic architectures of complex diseases. GWAS of biobank-linked electronic health record (EHR) data are increasingly being utilized to identify MPA among various traits and diseases. However, methodologies that can efficiently take advantage of distributed EHR to detect MPA are still lacking. Here, we introduce mixWAS, a novel algorithm that efficiently and losslessly integrates multiple EHRs via summary statistics, allowing the detection of MPA among mixed phenotypes while accounting for heterogeneities across EHRs. Simulations demonstrate that mixWAS outperforms the widely used MPA detection method, Phenome-wide association study (PheWAS), across diverse scenarios. Applying mixWAS to data from seven EHRs in the US, we identified 4,534 MPA among blood lipids, BMI, and circulatory diseases. Validation in an independent EHR data from UK confirmed 97.7% of the associations. mixWAS fundamentally improves the detection of MPA and is available as a free, open-source software.\n",
  "wordCount" : "2563",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301165">
        <p class="paperTitle">Impact of genome build on RNA-seq interpretation and diagnostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network,  ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.</p>
        <p class="info">Score: 15.9, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301165' target='https://doi.org/10.1101/2024.01.11.24301165'> 10.1101/2024.01.11.24301165</a></p>
        <p class="abstract">Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300360">
        <p class="paperTitle">A framework for accurate prediction of effector genes in genetic loci for complex traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schipper, M.; de Leeuw, C. A.; Maciel, B. A. P. C.; Wightman, D. P.; Hubers, N.; Boomsma, D. I.; O&#39;Donovan, M. C.; Posthuma, D.</p>
        <p class="info">Score: 14.5, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300360' target='https://doi.org/10.1101/2023.12.23.23300360'> 10.1101/2023.12.23.23300360</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus associations remains challenging. Here we present the FLAMES framework, which predicts the most likely effector gene in a locus. FLAMES integrates machine learning with biological data linking single nucleotide polymorphisms (SNPs) to genes, and GWAS-wide convergence of gene interactions. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication, and domain knowledge of molecular traits. We demonstrate that combining SNP-based and convergence-based evidence outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find novel schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.24301012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.24301012" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.24301012">
        <p class="paperTitle">The Metabolic Role of MAP3K15: Genetic and Phenotypic Insights from the 23andMe Research Database and Genetics-Driven Recruitment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.24301012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.24301012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brady, J. J.; Kalkus, K.; Nguyen, D. T.; Wen, J.; Fuller, Z. L.; Qi, Y.; Deering, S. S.; Selcer, L.; Shringarpure, S. S.; The 23andMe Research Team,  ; Holmes, M. V.; Wang, X.</p>
        <p class="info">Score: 13.0, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.24301012' target='https://doi.org/10.1101/2024.01.09.24301012'> 10.1101/2024.01.09.24301012</a></p>
        <p class="abstract">MAP3K15 has been previously associated with protection from type 2 diabetes (T2D), prompting interest in the development of MAP3K15 inhibitors as a potential therapeutic option for diabetes. The trans-ancestry genome-wide association study (GWAS) meta-analysis and loss-of-function (LoF) burden testing methods that implicate association with T2D greatly benefit from large sample size. The direct-to-consumer genetic testing company, 23andMe, Inc., is the worlds largest research consented genetic database. We leveraged the 23andMe database to further inform the metabolic role of MAP3K15, using a variety of genetic analysis methods. We find that MAP3K15 LoF carriers show a significant delay of 4.5 years in the median age of T2D diagnosis among individuals at high polygenic risk and uncover a novel burden association of MAP3K15 LoF with protection against high cholesterol. We expanded these findings by establishing a capability to recruit consented participants on the basis of genetics unknown to them (specifically, a single LoF variant in MAP3K15, rs148312150), and obtained clinical laboratory evidence of a modest reduction in median cholesterol and LDL/HDL ratio in MAP3K15 LoF carriers. Our findings demonstrate the discovery power of the 23andMe database, including the feasibility of consented participant recruitment to inform therapeutic discovery and development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300779" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300779">
        <p class="paperTitle">Valid inference for machine learning-assisted GWAS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miao, J.; Wu, Y.; Sun, Z.; Miao, X.; Lu, T.; Zhao, J.; Lu, Q.</p>
        <p class="info">Score: 20.8, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300779' target='https://doi.org/10.1101/2024.01.03.24300779'> 10.1101/2024.01.03.24300779</a></p>
        <p class="abstract">Machine learning (ML) has revolutionized analytical strategies in almost all scientific disciplines including human genetics and genomics. Due to challenges in sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS) which uses sophisticated ML to impute phenotypes and then performs GWAS on imputed outcomes has quickly gained popularity in complex trait genetics research. However, the validity of associations identified from ML-assisted GWAS has not been carefully evaluated. In this study, we report pervasive risks for false positive associations in ML-assisted GWAS, and introduce POP-GWAS, a novel statistical framework that reimagines GWAS on ML-imputed outcomes. POP-GWAS provides valid statistical inference irrespective of the quality of imputation or variables and algorithms used for imputation. It also only requires GWAS summary statistics as input. We employed POP-GWAS to perform the largest GWAS of bone mineral density (BMD) derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 novel loci reaching genome-wide significance and revealing skeletal site-specific genetic architecture of BMD. Our framework may fundamentally reshape the analytical strategies in future ML-assisted GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.23300546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.23300546" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.23300546">
        <p class="paperTitle">The Solute Carrier Family 26 Member 9 Is a Modifier of the Rapidly Progressing Cystic Fibrosis Associated with F508del CFTR Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.23300546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.23300546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, S.; Rollins, S.; Schmitz-Abe, K.; Tam, A.; Li, Q.; Shi, J.; Lin, J.; Wang, R.; Agrawal, P.</p>
        <p class="info">Score: 7.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.23300546' target='https://doi.org/10.1101/2024.01.04.23300546'> 10.1101/2024.01.04.23300546</a></p>
        <p class="abstract">Cystic fibrosis (CF) is an autosomal recessive disease caused by mutations to the CF transmembrane conductance regulator (CFTR). Symptoms and severity of the disease vary shown that modifier genes influence disease severity and clinical course. We previously reported epithelial sodium channel (ENaC) genes as modifiers of disease severity in long-term non-progressors sharing deltaF508 homozygous for CFTR genotype. Here we describe the opposite, modifier genes that may be associated with rapidly progressing CF (RPCF) in a cohort of patients with the shared deltaF508 homozygous genotype. We have identified three rare missense SLC26A9 variants in four individuals (out of six) deemed to have RPCF: c.229G&gt;A; p.G77S (present in two patients), c.1885C&gt;T; p.P629S and c.2546G&gt;A; p.R849Q. By analyzing publicly available single cell sequencing dataset from LungMAP, we revealed that both SLC26A9 and CFTR mRNA are highly enriched in Alveolar type 2 (AT2) cells, with a few (greater than 1%) in respiratory airway secretory (RAS) cells and ionocytes. Structural modeling suggests deleterious effects of these mutations as they are in critical protein domains which might affect the ion transportation capability of SLC26A9. The enrichment of rare and potentially deleterious SLC26A9 mutations in patients with RPCF suggests SLC26A9 is a modifier gene associated with RPCF.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.23300329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.23300329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.23300329">
        <p class="paperTitle">RNA methyltransferase SPOUT1/CENP-32 links mitotic spindle organization with the neurodevelopmental disorder SpADMiSS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.23300329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.23300329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dharmadhikari, A. V.; Abad, M. A.; Khan, S.; Maroofian, R.; Sands, T. T.; Ullah, F.; Samejima, I.; Wear, M. A.; Moore, K. E.; Kondakova, E.; Mitina, N.; Schaub, T.; Lee, G. K.; Umandap, C. H.; Berger, S. M.; Iglesias, A. D.; Popp, B.; Jamra, R. A.; Gabriel, H.; Rentas, S.; Rippert, A. L.; Izumi, K.; Conlin, L. K.; Koboldt, D. C.; Mihalic Mosher, T.; Hickey, S. E.; Albert, D. V.; Norwood, H.; Lewanda, A. F.; Dai, H.; Liu, P.; Mitani, T.; Marafi, D.; Pehlivan, D.; Posey, J. E.; Lippa, N.; Vena, N.; Heinzen, E. L.; Goldstein, D. B.; Mignot, C.; Agathe, J.-M. d. S.; Al-Sannaa, N. A.; Zamani, M.; S</p>
        <p class="info">Score: 6.6, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.23300329' target='https://doi.org/10.1101/2024.01.09.23300329'> 10.1101/2024.01.09.23300329</a></p>
        <p class="abstract">SPOUT1/CENP-32 encodes a putative SPOUT RNA methyltransferase previously identified as a mitotic chromosome associated protein. SPOUT1/CENP-32 depletion leads to centrosome detachment from the spindle poles and chromosome misalignment. Aided by gene matching platforms, we identified 24 individuals with neurodevelopmental delays from 18 families with bi-allelic variants in SPOUT1/CENP-32 detected by exome/genome sequencing. Zebrafish spout1/cenp-32 mutants showed reduction in larval head size with concomitant apoptosis likely associated with altered cell cycle progression. In vivo complementation assays in zebrafish indicated that SPOUT1/CENP-32 missense variants identified in humans are pathogenic. Crystal structure analysis of SPOUT1/CENP-32 revealed that most disease-associated missense variants mapped to the catalytic domain. Additionally, SPOUT1/CENP-32 recurrent missense variants had reduced methyltransferase activity in vitro and compromised centrosome tethering to the spindle poles in human cells. Thus, SPOUT1/CENP-32 pathogenic variants cause an autosomal recessive neurodevelopmental disorder: SpADMiSS (SPOUT1 Associated Development delay Microcephaly Seizures Short stature) underpinned by mitotic spindle organization defects and consequent chromosome segregation errors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24300997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24300997" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24300997">
        <p class="paperTitle">EHMT2 LOSS-OF-FUNCTION ALTERATIONS CAUSE A KLEEFSTRA-LIKE SYNDROME</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24300997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24300997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Delgado, B.; Lopez-Martin, E.; Kerkhof, J.; Baladron, B.; Mielu, L. M.; Sanchez-Ponce, D.; Bada-Navarro, A.; Herrero-Matesanz, M.; Lopez-Jimenez, L.; Rzasa, J.; Rots, D.; Fernandez, M.; Hernandez-San Miguel, E.; Gomez-Mariano, G.; Marin-Reina, P.; Cazorla-Calleja, R.; Alonso, J.; Kleefstra, T.; Posada, M.; Bermejo-Sanchez, E.; Sadikovic, B.; Barrero, M. J.</p>
        <p class="info">Score: 5.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24300997' target='https://doi.org/10.1101/2024.01.10.24300997'> 10.1101/2024.01.10.24300997</a></p>
        <p class="abstract">Dysregulation of the epigenetic machinery is associated with neurodevelopmental defects in humans. Kleefstra syndrome (KS) is a neurodevelopmental syndrome caused by heterozygous alterations in the gene EHMT1 that cause loss-of-function. EHTM1 and EHMT2 are highly similar histone methyltransferases that play relevant roles in development. Despite their similarity, individuals with alterations in EHMT2 have never been described. Here, we describe a pediatric patient with a KS-overlapping phenotype and a single base de novo substitution in EHMT2 that causes the amino acid change p.Ala1077Ser in the catalytic SET domain. This change causes a reduction in the affinity of the catalytic domain for the H3 tail and in the activity of the enzyme by three- to five-fold. DNA methylation, histone methylation and gene expression profiles suggest a significant overlap between the EHMT2 p.Ala1077Ser variant and KS. Based on this evidence we suggest that EHMT2 haploinsufficiency causes a Kleefstra-like syndrome. Although we cannot rule out dominant negative effects caused by the EHMT2 p.Ala1077Ser variant, our data and previously published data suggest that loss of EHMT2 function is probably more detrimental to cells than loss of EHMT1, explaining why individuals with alterations in EHMT2 are very rare.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24301111">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24301111" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24301111">
        <p class="paperTitle">Complex trait associations in rare diseases and impacts on Mendelian variant interpretation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24301111" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24301111" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smail, C.; Ge, B.; Keever-Keigher, M. R.; Schwendinger-Schreck, C.; Cheung, W. A.; Johnston, J. J.; Barrett, C.; Genomic Answers for Kids Consortium,  ; Feldman, K.; Cohen, A. S.; Farrow, E. G.; Thiffault, I.; Grundberg, E.; Pastinen, T.</p>
        <p class="info">Score: 5.0, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24301111' target='https://doi.org/10.1101/2024.01.10.24301111'> 10.1101/2024.01.10.24301111</a></p>
        <p class="abstract">Emerging evidence implicates common genetic variation - aggregated into polygenic scores (PGS) - impacting the onset and phenotypic presentation of rare diseases. In this study, we quantified individual polygenic liability for 1,151 previously published PGS in a cohort of 2,374 probands enrolled in the Genomic Answers for Kids (GA4K) rare disease study, revealing widespread associations between rare disease phenotypes and PGSs for common complex diseases and traits, blood protein levels, and brain and other organ morphological measurements. We observed increased polygenic burden in probands with variants of unknown significance (VUS) compared to unaffected carrier parents. We further observed an enrichment in overlap between diagnostic and candidate rare disease genes and large-effect PGS genes. Overall, our study supports and expands on previous findings of complex trait associations in rare disease phenotypes and provides a framework for identifying novel candidate rare disease genes and in understanding variable penetrance of candidate Mendelian disease variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24301007">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24301007" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24301007">
        <p class="paperTitle">Assessing the lack of diversity in genetics research across neurodegenerative diseases: a systematic review of the GWAS Catalog and literature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24301007" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24301007" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leonard, H.; Jonson, C.; Levine, K.; Lake, J.; Hertslet, L.; Jones, L.; Patel, D.; Kim, J.; Bandres-Ciga, S.; Terry, N.; Mata, I. F.; Blauwendraat, C.; Singleton, A. B.; Yokoyama, J. S.</p>
        <p class="info">Score: 4.9, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24301007' target='https://doi.org/10.1101/2024.01.08.24301007'> 10.1101/2024.01.08.24301007</a></p>
        <p class="abstract">ImportanceThe under-representation of participants with non-European ancestry in genome-wide association studies (GWAS) is a critical issue that has significant implications, including hindering the progress of precision medicine initiatives. This issue is particularly significant in the context of neurodegenerative diseases (NDDs), where current therapeutic approaches have shown limited success. Addressing this under-representation is crucial to harnessing the full potential of genomic medicine in underserved communities and improving outcomes for NDD patients.

ObjectiveOur primary objective was to assess the representation of non-European ancestry participants in genetic discovery efforts related to NDDs. We aimed to quantify the extent of inclusion of diverse ancestry groups in NDD studies and determine the number of associated loci identified in more inclusive studies. Specifically, we sought to highlight the disparities in research efforts and outcomes between studies predominantly involving European ancestry participants and those deliberately targeting non-European or multi-ancestry populations across NDDs.

Evidence ReviewWe conducted a systematic review utilizing existing GWAS results and publications to assess the inclusion of diverse ancestry groups in neurodegeneration and neurogenetics studies. Our search encompassed studies published up to the end of 2022, with a focus on identifying research that deliberately included non-European or multi-ancestry cohorts. We employed rigorous methods for the inclusion of identified articles and quality assessment.

FindingsOur review identified a total of 123 NDD GWAS. Strikingly, 82% of these studies predominantly featured participants of European ancestry. Endeavors specifically targeting non-European or multi-ancestry populations across NDDs identified only 52 risk loci. This contrasts with predominantly European studies, which reported over 90 risk loci for a single disease.

Encouragingly, over 65% of these discoveries occurred in 2020 or later, indicating a recent increase in studies deliberately including non-European cohorts.

Conclusions and relevanceOur findings underscore the pressing need for increased diversity in neurodegenerative research. The significant under-representation of non-European ancestry participants in NDD GWAS limits our understanding of the genetic underpinnings of these diseases. To advance the field of neurodegenerative research and develop more effective therapies, it is imperative that future investigations prioritize and harness the genomic diversity present within and across global populations.

Abstract and highlightsO_ST_ABSKey PointsC_ST_ABSQuestionWhat is the state of ancestral inclusivity in genetic studies of neurodegenerative diseases?

FindingsA systematic review of 123 publications on neurodegenerative diseases shows a focus on European populations, with only 18% of studies including any non-European ancestry data. Among 52 novel loci identified in non-European studies, 28 were from multi-ancestry studies (which included Europeans), 21 from East Asian studies, and 3 from other populations.

MeaningThis significant disparity underscores the need for more inclusive research approaches in neurodegenerative diseases, emphasizing multi-ancestry and non-European populations to advance precision medicine and develop treatments effective for diverse populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.24301073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.24301073" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.24301073">
        <p class="paperTitle">mixWAS: An efficient distributed algorithm for mixed-outcomes genome-wide association studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.24301073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.24301073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, R.; Benz, L.; Duan, R.; Denny, J.; Hakonarson, H.; Mosley, J.; Smoller, J. W.; Wei, W.-Q.; Ritchie, M. D.; Moore, J. H.; Chen, Y.</p>
        <p class="info">Score: 4.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.24301073' target='https://doi.org/10.1101/2024.01.09.24301073'> 10.1101/2024.01.09.24301073</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have been instrumental in identifying genetic associations for various diseases and traits. However, uncovering genetic underpinnings among traits beyond univariate phenotype associations remains a challenge. Multi-phenotype associations (MPA), or genetic pleiotropy, offer important insights into shared genes and pathways among traits, enhancing our understanding of genetic architectures of complex diseases. GWAS of biobank-linked electronic health record (EHR) data are increasingly being utilized to identify MPA among various traits and diseases. However, methodologies that can efficiently take advantage of distributed EHR to detect MPA are still lacking. Here, we introduce mixWAS, a novel algorithm that efficiently and losslessly integrates multiple EHRs via summary statistics, allowing the detection of MPA among mixed phenotypes while accounting for heterogeneities across EHRs. Simulations demonstrate that mixWAS outperforms the widely used MPA detection method, Phenome-wide association study (PheWAS), across diverse scenarios. Applying mixWAS to data from seven EHRs in the US, we identified 4,534 MPA among blood lipids, BMI, and circulatory diseases. Validation in an independent EHR data from UK confirmed 97.7% of the associations. mixWAS fundamentally improves the detection of MPA and is available as a free, open-source software.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
