<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 438.8, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nPrevalence, Morbidity, and Mortality of 1,609 Men with Diagnosed and Undiagnosed Sex Chromosome Aneuploidy: Results from the Diverse Million Veteran Program (MVP) Cohort\nAuthors: Davis, S. M.; Teerlink, C.; Lynch, J. A.; Gorman, B. R.; Pagadala, M.; Liu, A.; Panizzon, M. S.; Merritt, V. C.; Genovese, G.; Pyarajan, S.; Ross, J. L.; Hauger, R. L.\nScore: 29.5, Published: 2023-07-16 DOI: 10.1101/2023.07.15.23292710\nSTRUCTURED ABSTRACTO_ST_ABSImportanceC_ST_ABSThe reported phenotypes of men with 47,XXY and 47,XYY syndromes include tall stature, multisystem comorbidities, and poor health-related quality of life (HRQoL). However, knowledge about these sex chromosome aneuploidy (SCA) conditions has been derived from studies in the \u003c15% of patients who are clinically diagnosed and also lack diversity in age and genetic ancestry. ObjectivesDetermine the prevalence of clinically diagnosed and undiagnosed X or Y chromosome aneuploidy among men enrolled in the Million Veteran Program (MVP); describe military service metrics of men with SCAs; compare morbidity and mortality outcomes between men with SCA with and without a clinical diagnosis to matched controls. DesignCross-sectional, case-control SettingUnited States Veterans Administration Healthcare System ParticipantsBiologic males enrolled in the MVP biobank with genomic identification of an additional X or Y chromosome (cases); controls matched 1:5 on sex, age, and genetic ancestry Main Outcome(s) and Measure(s)Prevalence of men with SCAs from genomic analysis; clinical SCA diagnosis; Charlson Comorbidity Index (CCI); rates of outpatient, inpatient, and emergency encounters per year; self-reported health outcomes; standardized mortality ratio (SMR) ResultsAn additional X or Y chromosome was present in 145 and 125 per 100,000 males in the MVP, respectively, with the highest prevalence among men with European and East Asian ancestry. At a mean age of 61{+/-}12 years, 74% of male veterans with 47,XXY and \u003e99% with 47,XYY remained undiagnosed. Individuals with 47,XXY (n=862) and 47,XYY (n=747) had similar military service history, all-cause SMR, and age of death compared to matched controls. CCI and healthcare utilization were higher among individuals with SCA, while several measures of HRQoL were lower. Men with a clinical diagnosis of 47,XXY had higher healthcare utilization but lower comorbidity score compared to those undiagnosed. Conclusion and RelevanceOne in 370 males in the MVP cohort have SCA, a prevalence comparable to estimates in the general population. While these men have successfully served in the military, they have higher morbidity and report poorer HRQoL with aging. Longer longitudinal follow-up of this sample will be informative for clinical and patient-reported outcomes, the role of ancestry, and mortality statistics. KEY POINTSO_LIComparable to the general population, approximately 1 in 370 male veterans have a sex chromosome aneuploidy, but most are undiagnosed. C_LIO_LIMen with X or Y chromosome aneuploidy successfully complete US miliary duty with similar service history compared to their 46,XY peers. C_LIO_LIMedical comorbidities and healthcare utilization metrics are higher in male veterans with 47,XXY and 47,XYY during aging, however life expectancy is similar to matched controls. C_LI\nFederated analysis of the contribution of recessive coding variants to 29,745 developmental disorder patients from diverse populations\nAuthors: Chundru, V. K.; Zhang, Z.; Walter, K.; Lindsay, S.; Danecek, P.; Eberhardt, R. Y.; Gardner, E. J.; Malawsky, D. S.; Wigdor, E. M.; Torene, R.; Retterer, K.; Wright, C. F.; McWalter, K.; Sheridan, E.; Firth, H. V.; Hurles, M. E.; Samocha, K. E.; Ustach, V. D.; Martin, H. C.\nScore: 25.9, Published: 2023-07-24 DOI: 10.1101/2023.07.24.23293070\nAutosomal recessive (AR) coding variants are a well-known cause of rare disorders. We quantified the contribution of these variants to developmental disorders (DDs) in the largest and most ancestrally diverse sample to date, comprising 29,745 trios from the Deciphering Developmental Disorders (DDD) study and the genetic diagnostics company GeneDx, of whom 20.4% have genetically-inferred non-European ancestries. The estimated fraction of patients attributable to exome-wide AR coding variants ranged from ~2% to ~18% across genetically-inferred ancestry groups, and was significantly correlated with the average autozygosity (r=0.99, p=5x10-6). Established AR DD-associated (ARDD) genes explained 90% of the total AR coding burden, and this was not significantly different between probands with genetically-inferred European versus non-European ancestries. Approximately half the burden in these established genes was explained by variants not already reported as pathogenic in ClinVar. We estimated that ~1% of undiagnosed patients in both cohorts were attributable to damaging biallelic genotypes involving missense variants in established ARDD genes, highlighting the challenge in interpreting these. By testing for gene-specific enrichment of damaging biallelic genotypes, we identified two novel ARDD genes passing Bonferroni correction, KBTBD2 (p=1x10-7) and CRELD1 (p=9x10-8). Several other novel or recently-reported candidate genes were identified at a more lenient 5% false-discovery rate, including ZDHHC16 and HECTD4. This study expands our understanding of the genetic architecture of DDs across diverse genetically-inferred ancestry groups and suggests that improving strategies for interpreting missense variants in known ARDD genes may allow us to diagnose more patients than discovering the remaining genes.\nGenetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis\nAuthors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.\nScore: 46.7, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292611\nBackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms. MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms. ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases. Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3. Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank. Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS. ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.\nGenome-wide analyses reveal widespread genetic overlap between neurological and psychiatric disorders and a convergence of biological associations related to the brain\nAuthors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC), ; The International Headache Genetics Consortium (IHGC), ; Steen, N. E.; van der Meer, D.; O'Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.\nScore: 13.9, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23292993\nNeurological and psychiatric disorders are considered to reflect distinct underlying pathogenic entities. However, the extent to which they share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate extensive genetic overlap across the disorders, with varying degrees of genetic correlations. In particular, migraine, essential tremor, stroke and multiple sclerosis were genetically correlated with several psychiatric disorders. Biological interrogation indicated heterogenous biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study demonstrates that neurological and psychiatric disorders are not genetically disparate, but share key etiological aspects, which have important implications for disease classification, clinical practice, and genomic precision medicine.\nAssociation study of human leukocyte antigen (HLA) variants and idiopathic pulmonary fibrosis\nAuthors: Guillen-Guio, B.; Paynton, M. L.; Allen, R. J.; Chin, D. P. W.; Donoghue, L. J.; Stockwell, A.; Leavy, O. C.; Hernandez-Beeftink, T.; Reynolds, C.; Cullinan, P.; Martinez, F.; CleanUP-IPF Investigators of the Pulmonary Trials Cooperative, ; Booth, H. L.; Fahy, W. A.; Hall, I. P.; Hart, S. P.; Hill, M. R.; Hirani, N.; Hubbard, R. B.; McAnulty, R. J.; Millar, A. B.; Navaratnam, V.; Oballa, E.; Parfrey, H.; Saini, G.; Sayers, I.; Tobin, M. D.; Whyte, M. K. B.; Adegunsoye, A.; Kaminski, N.; Ma, S.-F.; Strek, M. E.; Zhang, Y.; Fingerlin, T. E.; Molina-Molina, M.; Neighbors, M.; Sheng, X. R.; Oldha\nScore: 4.8, Published: 2023-07-24 DOI: 10.1101/2023.07.20.23292940\nIntroductionIdiopathic pulmonary fibrosis (IPF) is a chronic interstitial pneumonia marked by progressive lung fibrosis and a poor prognosis. Recent studies have highlighted the potential role of infection in the pathogenesis of IPF and a prior association of the HLA-DQB1 gene with idiopathic fibrotic interstitial pneumonia (including IPF) has been reported. Due to the important role that the Human Leukocyte Antigen (HLA) region plays in the immune response, here we evaluated if HLA genetic variation was associated specifically with IPF risk. MethodsWe performed a meta-analysis of associations of the HLA region with IPF risk in individuals of European ancestry from seven independent case-control studies of IPF (comprising a total of 5,159 cases and 27,459 controls, including the prior study of fibrotic interstitial pneumonia). Single nucleotide polymorphisms, classical HLA alleles and amino acids were analysed and signals meeting a region-wide association threshold p\u003c4.5x10-4 and a posterior probability of replication \u003e90% were considered significant. We sought to replicate the previously reported HLA-DQB1 association in the subset of studies independent of the original report. ResultsThe meta-analysis of all seven studies identified four significant independent single nucleotide polymorphisms associated with IPF risk. However, none met the posterior probability for replication criterion. The HLA-DQB1 association was not replicated in the independent IPF studies. ConclusionVariation in the HLA region was not consistently associated with risk in studies of IPF. However, this does not preclude the possibility that other genomic regions linked to the immune response may be involved in the aetiology of IPF.\nUnraveling COVID-19 Relationship with Anxiety Disorders and Symptoms\nAuthors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.\nScore: 3.8, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23293001\nBackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes. MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms. ResultsWe observed a strong genetic correlation of anxiety disorder with COVID-19 positive status (rg=0.35, p=2 x 10-4) and COVID-19 hospitalization (rg=0.31, p=7.2 x 10-4). Among anxiety symptoms, \"Tense, sore, or aching muscles during worst period of anxiety\" was genetically correlated with COVID-19 positive status (rg=0.33, p=0.001), while \"Frequent trouble falling or staying asleep during worst period of anxiety\" was genetically correlated with COVID-19 hospitalization (rg=0.24, p=0.004). Through a latent causal variable analysis, we observed that COVID-19 outcomes have statistically significant genetic causality proportion (gcp) on anxiety symptoms (e.g., COVID-19 positive status[-\u0026gt;]\"Recent easy annoyance or irritability\" [boxv]gcp[boxv]=0.18, p=6.72 x 10-17). Conversely, anxiety disorders appear to have a possible causal effect on COVID-19 ([boxv]gcp[boxv]=0.38, p=3.17 x 10-9). Additionally, we also identified multiple loci with evidence of local genetic correlation between anxiety and COVID-19. These appear to be related to genetic effects shared with lung function, brain morphology, alcohol and tobacco use, and hematologic parameters. ConclusionsThis study provided important insights into the relationship between COVID-19 and mental health, differentiating the dynamics linking anxiety disorders to COVID-19 from the effect of COVID-19 on anxiety symptoms.\nEffect of prioritized SORL1 missense variants supports clinical consideration for familial Alzheimer's Disease\nAuthors: Holstege, H.; de Waal, M. W. J.; Tesi, N.; van der Lee, S. J.; ADES consortium, ; ADSP consortium, ; StEP-AD consortium, ; Knight-ADRC, ; UCSF/NYGC/UAB, ; Vogel, M.; van Spaendonk, R.; Hulsman, M.; Andersen, O. M.\nScore: 3.5, Published: 2023-07-16 DOI: 10.1101/2023.07.13.23292622\nBackgroundRare variants in the SORL1 gene have been associated with increased risk of Alzheimers disease (AD). While protein-truncating variants (PTVs) are observed almost exclusively in AD patients, most variants are rare missense variants that can be benign, risk-increasing, and recent reports have indicated that some variants are causative for disease. However, since SORL1 is currently not considered an autosomal dominant Alzheimer Disease gene (ADAD), segregation analyses are not performed, which complicates the identification of additional clinically important missense variants. MethodsWe prioritized highly conserved and functionally relevant SORL1 missense variants by considering the functional effects of homologous variants on proteins that share domains with SORL1 (domain-mapping of disease mutations, DMDM) into. We used this variant prioritization approach to annotate SORL1 variants identified in a previously assembled exome sequencing dataset encompassing 18,959 AD cases and 21,893 non-demented controls, and we tested the effect of high, moderate, low and no priority missense variants and specific variant subtypes on disease risk and age at onset. ResultsHigh priority missense variants (HPV) associated with a 6.4-fold increased risk of AD (95%CI: 4.3 - 9.7, p=2.1x10-24), which concentrated on early onset AD (OREOAD 10.5, 95%CI: 6.8 - 16.3, p=3.0x10-29) vs. late onset AD (ORLOAD=4.5, 95%CI 2.85 - 6.94; p=4.9x10-11). The median age at onset of HPV carriers was \u003e8-years earlier than carriers of wild-type SORL1. Intriguingly, specific subtypes of HPVs, including those affecting residues in the YWTD-motif or the calcium cage, occurred only in AD cases and carriers of these variants had an earlier age at onset compared to carriers of PTVs, indicative of a dominant negative effect. Carriers of other HPVs had an age at onset that overlapped with carriers of PTVs, suggesting they lead to haploinsufficiency. Yet other variants had a slightly later age at onset than PTVs, suggesting that their effect on SORL1 function was milder than losing a copy. Variants annotated as moderate, low and no priority did not have an effect on AD. ConclusionsNext to carriers of SORL1 PTVs, carriers of selected missense variants should be considered for segregation analyses, which will likely provide evidence for autosomal dominant inheritance for additional SORL1 missense variants.\nSubset scanning for multi-trait analysis using GWAS summary statistics\nAuthors: Cao, R.; Olawsky, E.; McFowland, E.; Marcotte, E.; Spector, L.; Yang, T.\nScore: 2.4, Published: 2023-07-23 DOI: 10.1101/2023.07.19.23292708\nMulti-trait analysis has been shown to have greater statistical power than single-trait analysis. Most of the existing multi-trait analysis methods only work with a limited number of traits and usually prioritize high statistical power over identifying relevant traits, which heavily rely on domain knowledge. To handle diseases and traits with obscure etiology, we developed TraitScan, a powerful and fast algorithm that agnostically searches and tests a subset of traits from a moderate or large number of traits (e.g., dozens to thousands) based on either individual-level or summary-level genetic data. We evaluated TraitScan using extensive simulations and found that it outperformed existing methods in terms of both testing power and trait selection when sparsity was low or modest. We then applied it to search for traits associated with Ewing Sarcoma, a rare bone tumor with peak onset in adolescence, among 706 traits in UK Biobank. Our analysis revealed a few promising traits worthy of further investigation, highlighting the use of TraitScan for more effective multi-trait analysis as biobanks emerge. Our algorithm is implemented in an R package TraitScan available at https://github.com/RuiCao34/TraitScan.\nAllelic strengths of encephalopathy-associated UBA5 variants correlate between in vivo and in vitro assays.\nAuthors: Pan, X.; Alvarez, A. N.; Ma, M.; Lu, S.; Crawford, M. W.; Briere, L. C.; Kanca, O.; Yamamoto, S.; Sweetser, D. A.; Wilson, J. L.; Napier, R. J.; Pruneda, J. N.; Bellen, H. J.\nScore: 1.9, Published: 2023-07-23 DOI: 10.1101/2023.07.17.23292782\nProtein UFMylation downstream of the E1 enzyme UBA5 plays essential roles in development and ER stress. Variants in the UBA5 gene are associated with developmental and epileptic encephalopathy 44 (DEE44), an autosomal recessive disorder characterized by early-onset encephalopathy, movement abnormalities, global developmental delay, intellectual disability, and seizures. DEE44 is caused by at least twelve different missense variants described as loss of function (LoF), but the relationships between genotypes and molecular or clinical phenotypes remains to be established. We developed a humanized UBA5 fly model and biochemical activity assays in order to describe in vivo and in vitro genotype-phenotype relationships across the UBA5 allelic series. In vivo, we observed a broad spectrum of phenotypes in viability, developmental timing, lifespan, locomotor activity, and bang sensitivity. A range of functional effects was also observed in vitro across comprehensive biochemical assays for protein stability, ATP binding, UFM1 activation, and UFM1 transthiolation. Importantly, there is a strong correlation between in vivo and in vitro phenotypes, establishing a classification of LoF variants into mild, intermediate, and severe allelic strengths. By systemically evaluating UBA5 variants across in vivo and in vitro platforms, this study provides a foundation for more basic and translational UBA5 research, as well as a basis for evaluating current and future individuals afflicted with this rare disease.\n",
  "wordCount" : "3431",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 438.8, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections1, 2. Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction3-5. The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative6, 7 to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity6, lung function8, and cancers9, suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.23292710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.23292710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.23292710">
        <p class="paperTitle">Prevalence, Morbidity, and Mortality of 1,609 Men with Diagnosed and Undiagnosed Sex Chromosome Aneuploidy: Results from the Diverse Million Veteran Program (MVP) Cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.23292710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.23292710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davis, S. M.; Teerlink, C.; Lynch, J. A.; Gorman, B. R.; Pagadala, M.; Liu, A.; Panizzon, M. S.; Merritt, V. C.; Genovese, G.; Pyarajan, S.; Ross, J. L.; Hauger, R. L.</p>
        <p class="info">Score: 29.5, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.23292710' target='https://doi.org/10.1101/2023.07.15.23292710'> 10.1101/2023.07.15.23292710</a></p>
        <p class="abstract">STRUCTURED ABSTRACTO_ST_ABSImportanceC_ST_ABSThe reported phenotypes of men with 47,XXY and 47,XYY syndromes include tall stature, multisystem comorbidities, and poor health-related quality of life (HRQoL). However, knowledge about these sex chromosome aneuploidy (SCA) conditions has been derived from studies in the &lt;15% of patients who are clinically diagnosed and also lack diversity in age and genetic ancestry.

ObjectivesDetermine the prevalence of clinically diagnosed and undiagnosed X or Y chromosome aneuploidy among men enrolled in the Million Veteran Program (MVP); describe military service metrics of men with SCAs; compare morbidity and mortality outcomes between men with SCA with and without a clinical diagnosis to matched controls.

DesignCross-sectional, case-control

SettingUnited States Veterans Administration Healthcare System

ParticipantsBiologic males enrolled in the MVP biobank with genomic identification of an additional X or Y chromosome (cases); controls matched 1:5 on sex, age, and genetic ancestry

Main Outcome(s) and Measure(s)Prevalence of men with SCAs from genomic analysis; clinical SCA diagnosis; Charlson Comorbidity Index (CCI); rates of outpatient, inpatient, and emergency encounters per year; self-reported health outcomes; standardized mortality ratio (SMR)

ResultsAn additional X or Y chromosome was present in 145 and 125 per 100,000 males in the MVP, respectively, with the highest prevalence among men with European and East Asian ancestry. At a mean age of 61{&#43;/-}12 years, 74% of male veterans with 47,XXY and &gt;99% with 47,XYY remained undiagnosed. Individuals with 47,XXY (n=862) and 47,XYY (n=747) had similar military service history, all-cause SMR, and age of death compared to matched controls. CCI and healthcare utilization were higher among individuals with SCA, while several measures of HRQoL were lower. Men with a clinical diagnosis of 47,XXY had higher healthcare utilization but lower comorbidity score compared to those undiagnosed.

Conclusion and RelevanceOne in 370 males in the MVP cohort have SCA, a prevalence comparable to estimates in the general population. While these men have successfully served in the military, they have higher morbidity and report poorer HRQoL with aging. Longer longitudinal follow-up of this sample will be informative for clinical and patient-reported outcomes, the role of ancestry, and mortality statistics.

KEY POINTSO_LIComparable to the general population, approximately 1 in 370 male veterans have a sex chromosome aneuploidy, but most are undiagnosed.
C_LIO_LIMen with X or Y chromosome aneuploidy successfully complete US miliary duty with similar service history compared to their 46,XY peers.
C_LIO_LIMedical comorbidities and healthcare utilization metrics are higher in male veterans with 47,XXY and 47,XYY during aging, however life expectancy is similar to matched controls.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.23293070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.23293070" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.23293070">
        <p class="paperTitle">Federated analysis of the contribution of recessive coding variants to 29,745 developmental disorder patients from diverse populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.23293070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.23293070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chundru, V. K.; Zhang, Z.; Walter, K.; Lindsay, S.; Danecek, P.; Eberhardt, R. Y.; Gardner, E. J.; Malawsky, D. S.; Wigdor, E. M.; Torene, R.; Retterer, K.; Wright, C. F.; McWalter, K.; Sheridan, E.; Firth, H. V.; Hurles, M. E.; Samocha, K. E.; Ustach, V. D.; Martin, H. C.</p>
        <p class="info">Score: 25.9, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.23293070' target='https://doi.org/10.1101/2023.07.24.23293070'> 10.1101/2023.07.24.23293070</a></p>
        <p class="abstract">Autosomal recessive (AR) coding variants are a well-known cause of rare disorders. We quantified the contribution of these variants to developmental disorders (DDs) in the largest and most ancestrally diverse sample to date, comprising 29,745 trios from the Deciphering Developmental Disorders (DDD) study and the genetic diagnostics company GeneDx, of whom 20.4% have genetically-inferred non-European ancestries. The estimated fraction of patients attributable to exome-wide AR coding variants ranged from ~2% to ~18% across genetically-inferred ancestry groups, and was significantly correlated with the average autozygosity (r=0.99, p=5x10-6). Established AR DD-associated (ARDD) genes explained 90% of the total AR coding burden, and this was not significantly different between probands with genetically-inferred European versus non-European ancestries. Approximately half the burden in these established genes was explained by variants not already reported as pathogenic in ClinVar. We estimated that ~1% of undiagnosed patients in both cohorts were attributable to damaging biallelic genotypes involving missense variants in established ARDD genes, highlighting the challenge in interpreting these. By testing for gene-specific enrichment of damaging biallelic genotypes, we identified two novel ARDD genes passing Bonferroni correction, KBTBD2 (p=1x10-7) and CRELD1 (p=9x10-8). Several other novel or recently-reported candidate genes were identified at a more lenient 5% false-discovery rate, including ZDHHC16 and HECTD4. This study expands our understanding of the genetic architecture of DDs across diverse genetically-inferred ancestry groups and suggests that improving strategies for interpreting missense variants in known ARDD genes may allow us to diagnose more patients than discovering the remaining genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292611">
        <p class="paperTitle">Genetic Risk Factors for Severe and Fatigue Dominant Long COVID and Commonalities with ME/CFS Identified by Combinatorial Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Pearson, M.; Das, S.; Sardell, J.; Chocian, K.; Gardner, S.</p>
        <p class="info">Score: 46.7, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292611' target='https://doi.org/10.1101/2023.07.13.23292611'> 10.1101/2023.07.13.23292611</a></p>
        <p class="abstract">BackgroundLong COVID is a debilitating chronic condition that has affected over 100 million people globally. It is characterized by a diverse array of symptoms, including fatigue, cognitive dysfunction and respiratory problems. Studies have so far largely failed to identify genetic associations, the mechanisms behind the disease, or any common pathophysiology with other conditions such as ME/CFS that present with similar symptoms.

MethodsWe used a combinatorial analysis approach to identify combinations of genetic variants significantly associated with the development of long COVID and to examine the biological mechanisms underpinning its various symptoms. We compared two subpopulations of long COVID patients from Sano Genetics Long COVID GOLD study cohort, focusing on patients with severe or fatigue dominant phenotypes. We evaluated the genetic signatures previously identified in an ME/CFS population against this long COVID population to understand similarities with other fatigue disorders that may be triggered by a prior viral infection. Finally, we also compared the output of this long COVID analysis against known genetic associations in other chronic diseases, including a range of metabolic and neurological disorders, to understand the overlap of pathophysiological mechanisms.

ResultsCombinatorial analysis identified 73 genes that were highly associated with at least one of the long COVID populations included in this analysis. Of these, 9 genes have prior associations with acute COVID-19, and 14 were differentially expressed in a transcriptomic analysis of long COVID patients. A pathway enrichment analysis revealed that the biological pathways most significantly associated with the 73 long COVID genes were mainly aligned with neurological and cardiometabolic diseases.

Expanded genotype analysis suggests that specific SNX9 genotypes are a significant contributor to the risk of or protection against severe long COVID infection, but that the gene-disease relationship is context dependent and mediated by interactions with KLF15 and RYR3.

Comparison of the genes uniquely associated with the Severe and Fatigue Dominant long COVID patients revealed significant differences between the pathways enriched in each subgroup. The genes unique to Severe long COVID patients were associated with immune pathways such as myeloid differentiation and macrophage foam cells. Genes unique to the Fatigue Dominant subgroup were enriched in metabolic pathways such as MAPK/JNK signaling. We also identified overlap in the genes associated with Fatigue Dominant long COVID and ME/CFS, including several involved in circadian rhythm regulation and insulin regulation. Overall, 39 SNPs associated in this study with long COVID can be linked to 9 genes identified in a recent combinatorial analysis of ME/CFS patient from UK Biobank.

Among the 73 genes associated with long COVID, 42 are potentially tractable for novel drug discovery approaches, with 13 of these already targeted by drugs in clinical development pipelines. From this analysis for example, we identified TLR4 antagonists as repurposing candidates with potential to protect against long term cognitive impairment pathology caused by SARS-CoV-2. We are currently evaluating the repurposing potential of these drug targets for use in treating long COVID and/or ME/CFS.

ConclusionThis study demonstrates the power of combinatorial analytics for stratifying heterogeneous populations in complex diseases that do not have simple monogenic etiologies. These results build upon the genetic findings from combinatorial analyses of severe acute COVID-19 patients and an ME/CFS population and we expect that access to additional independent, larger patient datasets will further improve the disease insights and validate potential treatment options in long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292993">
        <p class="paperTitle">Genome-wide analyses reveal widespread genetic overlap between neurological and psychiatric disorders and a convergence of biological associations related to the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC),  ; The International Headache Genetics Consortium (IHGC),  ; Steen, N. E.; van der Meer, D.; O&#39;Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.</p>
        <p class="info">Score: 13.9, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292993' target='https://doi.org/10.1101/2023.07.21.23292993'> 10.1101/2023.07.21.23292993</a></p>
        <p class="abstract">Neurological and psychiatric disorders are considered to reflect distinct underlying pathogenic entities. However, the extent to which they share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate extensive genetic overlap across the disorders, with varying degrees of genetic correlations. In particular, migraine, essential tremor, stroke and multiple sclerosis were genetically correlated with several psychiatric disorders. Biological interrogation indicated heterogenous biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study demonstrates that neurological and psychiatric disorders are not genetically disparate, but share key etiological aspects, which have important implications for disease classification, clinical practice, and genomic precision medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.23292940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.23292940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.23292940">
        <p class="paperTitle">Association study of human leukocyte antigen (HLA) variants and idiopathic pulmonary fibrosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.23292940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.23292940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guillen-Guio, B.; Paynton, M. L.; Allen, R. J.; Chin, D. P. W.; Donoghue, L. J.; Stockwell, A.; Leavy, O. C.; Hernandez-Beeftink, T.; Reynolds, C.; Cullinan, P.; Martinez, F.; CleanUP-IPF Investigators of the Pulmonary Trials Cooperative,  ; Booth, H. L.; Fahy, W. A.; Hall, I. P.; Hart, S. P.; Hill, M. R.; Hirani, N.; Hubbard, R. B.; McAnulty, R. J.; Millar, A. B.; Navaratnam, V.; Oballa, E.; Parfrey, H.; Saini, G.; Sayers, I.; Tobin, M. D.; Whyte, M. K. B.; Adegunsoye, A.; Kaminski, N.; Ma, S.-F.; Strek, M. E.; Zhang, Y.; Fingerlin, T. E.; Molina-Molina, M.; Neighbors, M.; Sheng, X. R.; Oldha</p>
        <p class="info">Score: 4.8, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.23292940' target='https://doi.org/10.1101/2023.07.20.23292940'> 10.1101/2023.07.20.23292940</a></p>
        <p class="abstract">IntroductionIdiopathic pulmonary fibrosis (IPF) is a chronic interstitial pneumonia marked by progressive lung fibrosis and a poor prognosis. Recent studies have highlighted the potential role of infection in the pathogenesis of IPF and a prior association of the HLA-DQB1 gene with idiopathic fibrotic interstitial pneumonia (including IPF) has been reported. Due to the important role that the Human Leukocyte Antigen (HLA) region plays in the immune response, here we evaluated if HLA genetic variation was associated specifically with IPF risk.

MethodsWe performed a meta-analysis of associations of the HLA region with IPF risk in individuals of European ancestry from seven independent case-control studies of IPF (comprising a total of 5,159 cases and 27,459 controls, including the prior study of fibrotic interstitial pneumonia). Single nucleotide polymorphisms, classical HLA alleles and amino acids were analysed and signals meeting a region-wide association threshold p&lt;4.5x10-4 and a posterior probability of replication &gt;90% were considered significant. We sought to replicate the previously reported HLA-DQB1 association in the subset of studies independent of the original report.

ResultsThe meta-analysis of all seven studies identified four significant independent single nucleotide polymorphisms associated with IPF risk. However, none met the posterior probability for replication criterion. The HLA-DQB1 association was not replicated in the independent IPF studies.

ConclusionVariation in the HLA region was not consistently associated with risk in studies of IPF. However, this does not preclude the possibility that other genomic regions linked to the immune response may be involved in the aetiology of IPF.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23293001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23293001" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23293001">
        <p class="paperTitle">Unraveling COVID-19 Relationship with Anxiety Disorders and Symptoms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23293001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23293001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Asgel, Z.; Kouakou, M.; Koller, D.; Pathak, G. A.; Cabrera-Mendoza, B.; Polimanti, R.</p>
        <p class="info">Score: 3.8, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23293001' target='https://doi.org/10.1101/2023.07.21.23293001'> 10.1101/2023.07.21.23293001</a></p>
        <p class="abstract">BackgroundWhile COVID-19 outcomes are associated with increased anxiety, individuals affected by anxiety disorders are more likely to develop severe COVID-19 outcomes.

MethodsWe used genome-wide data from UK Biobank (up to 420,531 participants), FinnGen Project (up to 329,077 participants), Million Veteran Program (175,163 participants), and COVID-19 Host Genetics Initiative (up to 122,616 cases and 2,475,240 controls) to investigate possible causal effects and shared genetic mechanisms linking COVID-19 outcomes to anxiety disorders and symptoms.

ResultsWe observed a strong genetic correlation of anxiety disorder with COVID-19 positive status (rg=0.35, p=2 x 10-4) and COVID-19 hospitalization (rg=0.31, p=7.2 x 10-4). Among anxiety symptoms, &#34;Tense, sore, or aching muscles during worst period of anxiety&#34; was genetically correlated with COVID-19 positive status (rg=0.33, p=0.001), while &#34;Frequent trouble falling or staying asleep during worst period of anxiety&#34; was genetically correlated with COVID-19 hospitalization (rg=0.24, p=0.004). Through a latent causal variable analysis, we observed that COVID-19 outcomes have statistically significant genetic causality proportion (gcp) on anxiety symptoms (e.g., COVID-19 positive status[-&amp;gt;]&#34;Recent easy annoyance or irritability&#34; [boxv]gcp[boxv]=0.18, p=6.72 x 10-17). Conversely, anxiety disorders appear to have a possible causal effect on COVID-19 ([boxv]gcp[boxv]=0.38, p=3.17 x 10-9). Additionally, we also identified multiple loci with evidence of local genetic correlation between anxiety and COVID-19. These appear to be related to genetic effects shared with lung function, brain morphology, alcohol and tobacco use, and hematologic parameters.

ConclusionsThis study provided important insights into the relationship between COVID-19 and mental health, differentiating the dynamics linking anxiety disorders to COVID-19 from the effect of COVID-19 on anxiety symptoms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292622">
        <p class="paperTitle">Effect of prioritized SORL1 missense variants supports clinical consideration for familial Alzheimer&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holstege, H.; de Waal, M. W. J.; Tesi, N.; van der Lee, S. J.; ADES consortium,  ; ADSP consortium,  ; StEP-AD consortium,  ; Knight-ADRC,  ; UCSF/NYGC/UAB,  ; Vogel, M.; van Spaendonk, R.; Hulsman, M.; Andersen, O. M.</p>
        <p class="info">Score: 3.5, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292622' target='https://doi.org/10.1101/2023.07.13.23292622'> 10.1101/2023.07.13.23292622</a></p>
        <p class="abstract">BackgroundRare variants in the SORL1 gene have been associated with increased risk of Alzheimers disease (AD). While protein-truncating variants (PTVs) are observed almost exclusively in AD patients, most variants are rare missense variants that can be benign, risk-increasing, and recent reports have indicated that some variants are causative for disease. However, since SORL1 is currently not considered an autosomal dominant Alzheimer Disease gene (ADAD), segregation analyses are not performed, which complicates the identification of additional clinically important missense variants.

MethodsWe prioritized highly conserved and functionally relevant SORL1 missense variants by considering the functional effects of homologous variants on proteins that share domains with SORL1 (domain-mapping of disease mutations, DMDM) into. We used this variant prioritization approach to annotate SORL1 variants identified in a previously assembled exome sequencing dataset encompassing 18,959 AD cases and 21,893 non-demented controls, and we tested the effect of high, moderate, low and no priority missense variants and specific variant subtypes on disease risk and age at onset.

ResultsHigh priority missense variants (HPV) associated with a 6.4-fold increased risk of AD (95%CI: 4.3 - 9.7, p=2.1x10-24), which concentrated on early onset AD (OREOAD 10.5, 95%CI: 6.8 - 16.3, p=3.0x10-29) vs. late onset AD (ORLOAD=4.5, 95%CI 2.85 - 6.94; p=4.9x10-11). The median age at onset of HPV carriers was &gt;8-years earlier than carriers of wild-type SORL1. Intriguingly, specific subtypes of HPVs, including those affecting residues in the YWTD-motif or the calcium cage, occurred only in AD cases and carriers of these variants had an earlier age at onset compared to carriers of PTVs, indicative of a dominant negative effect. Carriers of other HPVs had an age at onset that overlapped with carriers of PTVs, suggesting they lead to haploinsufficiency. Yet other variants had a slightly later age at onset than PTVs, suggesting that their effect on SORL1 function was milder than losing a copy. Variants annotated as moderate, low and no priority did not have an effect on AD.

ConclusionsNext to carriers of SORL1 PTVs, carriers of selected missense variants should be considered for segregation analyses, which will likely provide evidence for autosomal dominant inheritance for additional SORL1 missense variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.23292708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.23292708" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.23292708">
        <p class="paperTitle">Subset scanning for multi-trait analysis using GWAS summary statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.23292708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.23292708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, R.; Olawsky, E.; McFowland, E.; Marcotte, E.; Spector, L.; Yang, T.</p>
        <p class="info">Score: 2.4, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.23292708' target='https://doi.org/10.1101/2023.07.19.23292708'> 10.1101/2023.07.19.23292708</a></p>
        <p class="abstract">Multi-trait analysis has been shown to have greater statistical power than single-trait analysis. Most of the existing multi-trait analysis methods only work with a limited number of traits and usually prioritize high statistical power over identifying relevant traits, which heavily rely on domain knowledge. To handle diseases and traits with obscure etiology, we developed TraitScan, a powerful and fast algorithm that agnostically searches and tests a subset of traits from a moderate or large number of traits (e.g., dozens to thousands) based on either individual-level or summary-level genetic data. We evaluated TraitScan using extensive simulations and found that it outperformed existing methods in terms of both testing power and trait selection when sparsity was low or modest. We then applied it to search for traits associated with Ewing Sarcoma, a rare bone tumor with peak onset in adolescence, among 706 traits in UK Biobank. Our analysis revealed a few promising traits worthy of further investigation, highlighting the use of TraitScan for more effective multi-trait analysis as biobanks emerge. Our algorithm is implemented in an R package  TraitScan available at https://github.com/RuiCao34/TraitScan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.23292782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.23292782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.23292782">
        <p class="paperTitle">Allelic strengths of encephalopathy-associated UBA5 variants correlate between in vivo and in vitro assays.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.23292782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.23292782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pan, X.; Alvarez, A. N.; Ma, M.; Lu, S.; Crawford, M. W.; Briere, L. C.; Kanca, O.; Yamamoto, S.; Sweetser, D. A.; Wilson, J. L.; Napier, R. J.; Pruneda, J. N.; Bellen, H. J.</p>
        <p class="info">Score: 1.9, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.23292782' target='https://doi.org/10.1101/2023.07.17.23292782'> 10.1101/2023.07.17.23292782</a></p>
        <p class="abstract">Protein UFMylation downstream of the E1 enzyme UBA5 plays essential roles in development and ER stress. Variants in the UBA5 gene are associated with developmental and epileptic encephalopathy 44 (DEE44), an autosomal recessive disorder characterized by early-onset encephalopathy, movement abnormalities, global developmental delay, intellectual disability, and seizures. DEE44 is caused by at least twelve different missense variants described as loss of function (LoF), but the relationships between genotypes and molecular or clinical phenotypes remains to be established. We developed a humanized UBA5 fly model and biochemical activity assays in order to describe in vivo and in vitro genotype-phenotype relationships across the UBA5 allelic series. In vivo, we observed a broad spectrum of phenotypes in viability, developmental timing, lifespan, locomotor activity, and bang sensitivity. A range of functional effects was also observed in vitro across comprehensive biochemical assays for protein stability, ATP binding, UFM1 activation, and UFM1 transthiolation. Importantly, there is a strong correlation between in vivo and in vitro phenotypes, establishing a classification of LoF variants into mild, intermediate, and severe allelic strengths. By systemically evaluating UBA5 variants across in vivo and in vitro platforms, this study provides a foundation for more basic and translational UBA5 research, as well as a basis for evaluating current and future individuals afflicted with this rare disease.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
