<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis
Authors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.
Score: 8.4, Published: 2023-09-07 DOI: 10.1101/2023.09.05.23295093
Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis
Authors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.
Score: 8.4, Published: 2023-09-07 DOI: 10.1101/2023.09.05.23295093
Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis
Authors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.
Score: 8.4, Published: 2023-09-07 DOI: 10.1101/2023.09.05.23295093
Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis\nAuthors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.\nScore: 8.4, Published: 2023-09-07 DOI: 10.1101/2023.09.05.23295093\nClonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD).",
  "keywords": [
    
  ],
  "articleBody": " Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis\nAuthors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.\nScore: 8.4, Published: 2023-09-07 DOI: 10.1101/2023.09.05.23295093\nClonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in Communities Study (ARIC) participants over a median follow-up of 21 years. During this period, 735 participants developed incident CH. We found that age at baseline, sex, and dyslipidemia significantly influence the incidence of CH, while ASCVD and other traditional risk factors for ASCVD did not exhibit such associations. Our study also revealed associations between germline genetic variants and incident CH, prioritizing genes in CH development. Our comprehensive longitudinal assessment yields novel insights into the factors contributing to incident CH in older adults.\nThe predictive capacity of polygenic risk scores for disease risk is only moderately influenced by imputation panels tailored to the target population\nAuthors: Levi, H.; Elkon, R.; Shamir, R.\nScore: 4.6, Published: 2023-09-01 DOI: 10.1101/2023.08.29.23294769\nPolygenic risk scores (PRS) predict individuals genetic risk of developing complex diseases. They summarize the effect of many genetic variants discovered in genome-wide association studies (GWASs). However, to date, large GWASs exist primarily for the European population and the quality of PRS prediction declines when applied to target sets of other ethnicities. A key step in using a PRS is imputation, which is the inference of un-typed SNPs using a set of fully-sequenced individuals, called the imputation panel. The SNP genotypes called by the imputation process depend on the ethnic composition of the imputation panel. Several studies have shown that imputing genotypes using a panel that contains individuals of the same ethnicity as the genotyped individuals improves imputation accuracy. However, until now, there has been no systematic investigation into the influence of the ethnic composition of imputation panels on the accuracy of PRS predictions when applied to ethnic groups that differ from the population used in the GWAS. In this study we estimated the effect of imputation of the target set on prediction accuracy of PRS when the discovery (GWAS) and the target sets come from different ethnic groups. We analyzed twelve binary phenotypes and three populations from the UK Biobank (Europeans, South-Asians, and Africans). We generated imputation panels from several ethnic groups, imputed the target set using each panel, and generated PRS to compute individuals risk scores. Then, we assessed the prediction accuracy obtained from each imputation panel. Our analysis indicates that using an imputation panel matched to the ethnic group of the target population yields only a marginal improvement and only under specific conditions. Hence, while a target-matched imputation panel can potentially improve prediction accuracy of European PRSs in non-EUR populations, the improvement is limited.\nRare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank\nAuthors: Fenner, E.; Holmans, P.; O'Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.\nScore: 13.7, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294074\nImpairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.\nRelationship between sex biases in gene expression and sex biases in autism and Alzheimer's disease\nAuthors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.\nScore: 5.2, Published: 2023-09-01 DOI: 10.1101/2023.08.29.23294773\nSex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimers disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=137 SRC=\"FIGDIR/small/23294773v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@9b00a1org.highwire.dtl.DTLVardef@467465org.highwire.dtl.DTLVardef@183b2d4org.highwire.dtl.DTLVardef@1517e19_HPS_FORMAT_FIGEXP M_FIG C_FIG\nGenome-wide association study and genomic risk prediction of age-related macular degeneration in Israel\nAuthors: Grunin, M.; Triffon, D.; Beykin, G.; Rahmani, E.; Schweiger, R.; Tiosano, L.; Khateb, S.; Hagbi-Levi, S.; Rinsky, B.; Munitz, R.; Winkler, T. W.; Heid, I. M.; Halperin, E.; Carmi, S.; Chowers, I.\nScore: 3.6, Published: 2023-09-06 DOI: 10.1101/2023.09.06.23295126\nPurposeThe risk of developing age-related macular degeneration(AMD) is influenced by genetic background. In 2016, International AMD Genomics Consortium(IAMDGC) identified 52 risk variants in 34 loci, and a polygenic risk score(PRS) based on these variants was associated with AMD. The Israeli population has a unique genetic composition: Ashkenazi Jewish(AJ), Jewish non-Ashkenazi, and Arab sub-populations. We aimed to perform a genome-wide association study(GWAS) for AMD in Israel, and to evaluate PRSs for AMD. MethodsFor our discovery set, we recruited 403 AMD patients and 256 controls at Hadassah Medical Center. We genotyped all individuals via custom exome chip. We imputed non-typed variants using cosmopolitan and AJ reference panels. We recruited additional 155 cases and 69 controls for validation. To evaluate predictive power of PRSs for AMD, we used IAMDGC summary statistics excluding our study and developed PRSs via either clumping/thresholding or LDpred2. ResultsIn our discovery set, 31/34 loci previously reported by the IAMDGC were AMD associated with P\u003c0.05. Of those, all effects were directionally consistent with the IAMDGC and 11 loci had a p-value under Bonferroni-corrected threshold(0.05/34=0.0015). At a threshold of 5x10-5, we discovered four suggestive associations in FAM189A1, IGDCC4, C7orf50, and CNTNAP4. However, only the FAM189A1 variant was AMD associated in the replication cohort after Bonferroni-correction. A prediction model including LDpred2-based PRS and other covariates had an AUC of 0.82(95%CI:0.79-0.85) and performed better than a covariates-only model(P=5.1x10-9). ConclusionsPreviously reported AMD-associated loci were nominally associated with AMD in Israel. A PRS developed based on a large international study is predictive in Israeli populations.\nIdentifying X-Chromosome Variants Associated with Age-Related Macular Degeneration\nAuthors: Grunin, M.; Igo, R. P.; Song, Y. E.; Blanton, S. H.; Pericak-Vance, M. A.; Haines, J. L.; International Age-related Macular Degeneration Genomics Consortium, Score: 6.2, Published: 2023-08-31 DOI: 10.1101/2023.08.28.23294688\nPurposeIn genome-wide association studies (GWAS), X chromosome (ChrX) variants are often not investigated. Sex-specific effects and ChrX-specific quality control (QC) are needed to examine these effects. Previous analyses identified 52 autosomal variants associated with age-related macular degeneration (AMD) via the International AMD Genomics Consortium (IAMDGC), but did not analyze ChrX. Therefore, our goal was to investigate ChrX variants for association with AMD. MethodsWe genotyped 29,629 non-Hispanic White (NHW) individuals (M/F:10,404/18,865; AMD12,087/14723) via a custom chip and imputed after ChrX-specific QC (XWAS 3.0) using the Michigan Imputation Server. Imputation generated 1,221,623 variants on ChrX. Age, informative PCs, and subphenotyeps were covariates for logistic association analyses with Fishers correction. Gene/pathway analyses were performed with VEGAS, GSEASNP, ICSNPathway, DAVID, and mirPath. ResultsLogistic association on NHW individuals with sex correction, identified variants in/near the genes SLITRK4, ARHGAP6, FGF13 and DMD associated with AMD (P\u003c1x10-6,Fishers combined-corrected). Via association testing of subphenotypes of choroidal neovascularization and geographic atrophy (GA), variants in DMD associated with GA (P\u003c1x10-6, Fishers combined-corrected). Via gene-based analysis with VEGAS, several genes were associated with AMD (P\u003c0.05, both truncated tail strength/truncated product P) including SLITRK4 and BHLHB9. Pathway analysis using GSEASNP and DAVID showed genes associated with nervous system development (FDR: P:0.02), and blood coagulation (FDR: P:0.03). Variants in the region of a microRNA (miR) were associated with AMD (P\u003c0.05, truncated tail strength/truncated product P). Via DIANA mirPath analysis, downstream targets of miRs show association with brain disorders and fatty acid elongation (P\u003c0.05). A long-non coding RNA on ChrX near the DMD locus was also associated with AMD (P=4x10-7). Epistatic analysis (t-statistic) for a quantitative trait of AMD vs control including covariates found a suggestive association in the XG gene (P=2x10^-5). ConclusionsAnalysis of ChrX variants demonstrates association with AMD and these variants may be linked to novel pathways. Further analysis is needed to confirm results and to understand their biological significance and relationship with AMD development in worldwide populations.\nThe pivotal role of the X-chromosome in the genetic architecture of the human brain\nAuthors: Jiang, Z.; Sullivan, P. F.; Li, T.; Zhao, B.; Wang, X.; Luo, T.; Huang, S.; Guan, P. Y.; Chen, J.; Yang, Y.; Stein, J. L.; Li, Y.; Liu, D.; Sun, L.; Zhu, H.\nScore: 6.4, Published: 2023-09-01 DOI: 10.1101/2023.08.30.23294848\nGenes on the X-chromosome are extensively expressed in the human brain, resulting in substantial influences on brain development, intellectual disability, and other brain-related disorders. To comprehensively investigate the X-chromosomes impact on the cerebral cortex, white matter tract microstructures, and intrinsic and extrinsic brain functions, we examined 2,822 complex brain imaging traits obtained from n = 34,000 subjects in the UK Biobank. We unveiled potential autosome-X-chromosome interaction, while proposing an atlas of dosage compensation (DC) for each set of traits. We observed a pronounced X-chromosome impact on the corticospinal tract and the functional amplitude and connectivity of visual networks. In association studies, we identified 50 genome-wide significant trait-locus pairs enriched in Xq28, 22 of which replicated in independent datasets (n = 4,900). Notably, 13 newly identified pairs were in the X-chromosomes non-pseudo-autosomal regions (NPR). The volume of the right ventral diencephalon shared genetic architecture with schizophrenia and educational attainment in a locus indexed by rs2361468 (located [~]3kb upstream of PJA1, a conserved and ubiquitously expressed gene implicated in multiple psychiatric disorders). No significant associations were identified in the pseudo-autosomal regions (PAR) or the Y-chromosome. Finally, we explored sex-specific associations on the X-chromosome and compared differing genetic effects between sexes. We found much more associations can be identified in males (33 versus 9) given a similar sample size. In conclusion, our research provides invaluable insights into the X-chromosomes role in the human brain, contributing to the observed sex differences in brain structure and function.\nSpatial Distribution of Missense Variants within Complement Proteins Associates with Age Related Macular Degeneration\nAuthors: Grunin, M.; de Jong, S.; Palmer, E. L.; Jin, B.; Rinker, D.; Moth, C.; Capra, J. A.; Haines, J. L.; Bush, W.; den Hollander, A.; International Age-related Macular Degeneration Genomics Consortium, Score: 2.1, Published: 2023-08-31 DOI: 10.1101/2023.08.28.23294686\nPurposeGenetic variants in complement genes are associated with age-related macular degeneration (AMD). However, many rare variants have been identified in these genes, but have an unknown significance, and their impact on protein function and structure is still unknown. We set out to address this issue by evaluating the spatial placement and impact on protein structureof these variants by developing an analytical pipeline and applying it to the International AMD Genomics Consortium (IAMDGC) dataset (16,144 AMD cases, 17,832 controls). MethodsThe IAMDGC dataset was imputed using the Haplotype Reference Consortium (HRC), leading to an improvement of over 30% more imputed variants, over the original 1000 Genomes imputation. Variants were extracted for the CFH, CFI, CFB, C9, and C3 genes, and filtered for missense variants in solved protein structures. We evaluated these variants as to their placement in the three-dimensional structure of the protein (i.e. spatial proximity in the protein), as well as AMD association. We applied several pipelines to a) calculate spatial proximity to known AMD variants versus gnomAD variants, b) assess a variants likelihood of causing protein destabilization via calculation of predicted free energy change (ddG) using Rosetta, and c) whole gene-based testing to test for statistical associations. Gene-based testing using seqMeta was performed using a) all variants b) variants near known AMD variants or c) with a ddG \u003e|2|. Further, we applied a structural kernel adaptation of SKAT testing (POKEMON) to confirm the association of spatial distributions of missense variants to AMD. Finally, we used logistic regression on known AMD variants in CFI to identify variants leading to \u003e50% reduction in protein expression from known AMD patient carriers of CFI variants compared to wild type (as determined by in vitro experiments) to determine the pipelines robustness in identifying AMD-relevant variants. These results were compared to functional impact scores, ie CADD values \u003e 10, which indicate if a variant may have a large functional impact genomewide, to determine if our metrics have better discriminative power than existing variant assessment methods. Once our pipeline had been validated, we then performed a priori selection of variants using this pipeline methodology, and tested AMD patient cell lines that carried those selected variants from the EUGENDA cohort (n=34). We investigated complement pathway protein expression in vitro, looking at multiple components of the complement factor pathway in patient carriers of bioinformatically identified variants. ResultsMultiple variants were found with a ddG\u003e|2| in each complement gene investigated. Gene-based tests using known and novel missense variants identified significant associations of the C3, C9, CFB, and CFH genes with AMD risk after controlling for age and sex (P=3.22x10-5;7.58x10-6;2.1x10-3;1.2x10-31). ddG filtering and SKAT-O tests indicate that missense variants that are predicted to destabilize the protein, in both CFI and CFH, are associated with AMD (P=CFH:0.05, CFI:0.01, threshold of 0.05 significance). Our structural kernel approach identified spatial associations for AMD risk within the protein structures for C3, C9, CFB, CFH, and CFI at a nominal p-value of 0.05. Both ddG and CADD scores were predictive of reduced CFI protein expression, with ROC curve analyses indicating ddG is a better predictor (AUCs of 0.76 and 0.69, respectively). A priori in vitro analysis of variants in all complement factor genes indicated that several variants identified via bioinformatics programs PathProx/POKEMON in our pipeline via in vitro experiments caused significant change in complement protein expression (P=0.04) in actual patient carriers of those variants, via ELISA testing of proteins in the complement factor pathway, and were previously unknown to contribute to AMD pathogenesis. ConclusionWe demonstrate for the first time that missense variants in complement genes cluster together spatially and are associated with AMD case/control status. Using this method, we can identify CFI and CFH variants of previously unknown significance that are predicted to destabilize the proteins. These variants, both in and outside spatial clusters, can predict in-vitro tested CFI protein expression changes, and we hypothesize the same is true for CFH. A priori identification of variants that impact gene expression allow for classification for previously classified as VUS. Further investigation is needed to validate the models for additional variants and to be applied to all AMD-associated genes.\nCirculating anti-Müllerian hormone levels in pre-menopausal women: novel genetic insights from a GWAS meta-analysis\nAuthors: Pujol Gualdo, N.; Karjalainen, M. K.; Vosa, U.; Arffman, R. K.; Mägi, R.; Ronkainen, J.; Laisk, T.; Piltonen, T. T.\nScore: 1.8, Published: 2023-09-08 DOI: 10.1101/2023.09.07.23295182\nStudy question: Can a genome-wide association study (GWAS) meta-analysis, including a large sample of young premenopausal women from a founder population from Northern Finland, identify novel genetic variants for circulating anti-Mullerian hormone (AMH) levels and provide insights into biological pathways and tissues involved in AMH regulation? Summary answer: We identified six loci associated with AMH levels at P \u003c 5 x 10-8, including the previously reported MCM8, AMH and TEX41 loci, and three novel signals in or near CHEK2, BMP4 and EIF4EBP1. Gene set enrichment analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranks the pituitary gland as a top associated tissue. What is known already: AMH is expressed by preantral and small antral stage ovarian follicles in women, and variation in age-specific circulating AMH levels has been associated with several health conditions. However, the biological mechanisms underlying the association between health conditions and AMH levels are not yet fully understood. Previous GWAS have identified loci associated with AMH levels in pre-menopausal women, but they were limited by small sample sizes or focused mostly on older pre-menopausal women. Study design, size, duration: We performed a GWAS meta-analysis for AMH level measurements in 9,668 pre-menopausal women. Participants/materials, setting, methods: We performed a GWAS meta-analysis in which we combined 2,619 AMH measurements (at age 31 years old) from a prospective founder population cohort (Northern Finland Birth Cohort 1966, NFBC1966) with a previous GWAS meta-analysis that included 7,049 pre-menopausal women (spanning age range 15-48). NFBC1966 AMH measurements were quantified using an automated assay (Elecsys AMH Plus (Roche)). We annotated the genetic variants, combined different data layers to prioritise potential candidate genes, described significant pathways and tissues enriched by the GWAS signals, identified plausible regulatory roles using colocalization analysis and leveraged publicly available summary statistics to assess genetic and phenotypic correlations with multiple traits. Main results and the role of chance: Three novel genome-wide significant loci were identified. One of these is in complete linkage disequilibrium with c.1100delC in CHEK2, which is found to be 4-fold enriched in the Finnish population compared to other European populations. We propose a plausible regulatory effect of some of the GWAS variants linked to AMH, as they colocalise with GWAS signals associated with gene expression levels of BMP4, TEX41 and EIFBP41. Gene set analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranked the pituitary gland as the top association. Large scale data: The GWAS meta-analysis summary statistics will be available for download from the GWAS Catalog. Accession numbers will be provided upon publication. Limitations, reasons for caution: This study only included women of European ancestry and the unavailability of sufficiently sized relevant tissue data in gene expression datasets hinders the assessment of potential regulatory effects in reproductive tissues. Wider implications of the findings: Our results highlight the increased power of founder populations and larger sample sizes to boost the discovery of novel trait-associated variants underlying variation in AMH levels, which aided to characterise novel biological pathways and plausible genetic regulatory effects linked with AMH levels variation for the first time.\nPrioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach\nAuthors: Pena-Martinez, E. G.; Pomales-Matos, D. A.; Rivera-Madera, A.; Messon-Bird, J. L.; Medina-Feliciano, J. G.; Sanabria-Alberto, L.; Barreiro-Rosario, A. C.; Rodriguez-Rios, J. M.; Rodriguez-Martinez, J. A.\nScore: 1.8, Published: 2023-09-02 DOI: 10.1101/2023.09.01.23294951\nCardiovascular diseases (CVDs) are the leading cause of death worldwide and are heavily influenced by genetic factors. Genome-wide association studies (GWAS) have mapped \u003e 90% of CVD-associated variants within the non-coding genome, which can alter the function of regulatory proteins, like transcription factors (TFs). However, due to the overwhelming number of GWAS single nucleotide polymorphisms (SNPs) (\u003e500,000), prioritizing variants for in vitro analysis remains challenging. In this work, we implemented a computational approach that considers support vector machine (SVM)-based TF binding site classification and cardiac expression quantitative trait loci (eQTL) analysis to identify and prioritize potential CVD-causing SNPs. We identified 1,535 CVD-associated SNPs that occur within human heart footprints/enhancers and 9,309 variants in linkage disequilibrium (LD) with differential gene expression profiles in cardiac tissue. Using hiPSC-CM ChIP-seq data from NKX2-5 and TBX5, two cardiac TFs essential for proper heart development, we trained a large-scale gapped k-mer SVM (LS-GKM-SVM) predictive model that can identify binding sites altered by CVD-associated SNPs. The computational predictive model was tested by scoring human heart footprints and enhancers in vitro through electrophoretic mobility shift assay (EMSA). Three variants (rs59310144, rs6715570, and rs61872084) were prioritized for in vitro validation based on their eQTL in cardiac tissue and LS-GKM-SVM prediction to alter NKX2-5 DNA binding. All three variants altered NKX2-5 DNA binding. In summary, we present a bioinformatic approach that considers tissue-specific eQTL analysis and SVM-based TF binding site classification to prioritize CVD-associated variants for in vitro experimental analysis. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=55 SRC=\"FIGDIR/small/23294951v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (18K): org.highwire.dtl.DTLVardef@ad40c2org.highwire.dtl.DTLVardef@768501org.highwire.dtl.DTLVardef@1161861org.highwire.dtl.DTLVardef@19aa06d_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "3504",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.23295093">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.23295093" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.23295093">
        <p class="paperTitle">Long-term longitudinal analysis of 4,187 participants reveals new insights into determinants of incident clonal hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.23295093" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.23295093" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uddin, M. M.; Saadatagah, S.; Niroula, A.; Yu, B.; Hornsby, W.; Ganesh, S.; Lannery, K.; Shuermans, A.; Honigberg, M. C.; Bick, A. G.; Libby, P.; Ebert, B. L.; Ballantyne, C. M.; Natarajan, P.</p>
        <p class="info">Score: 8.4, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.23295093' target='https://doi.org/10.1101/2023.09.05.23295093'> 10.1101/2023.09.05.23295093</a></p>
        <p class="abstract">Clonal hematopoiesis (CH), characterized by blood cells predominantly originating from a single mutated hematopoietic stem cell, is linked to diverse aging-related diseases, including hematologic malignancy and atherosclerotic cardiovascular disease (ASCVD). While CH is common among older adults, the underlying factors driving its development are largely unknown. To address this, we performed whole-exome sequencing on 8,374 blood DNA samples collected from 4,187 Atherosclerosis Risk in Communities Study (ARIC) participants over a median follow-up of 21 years. During this period, 735 participants developed incident CH. We found that age at baseline, sex, and dyslipidemia significantly influence the incidence of CH, while ASCVD and other traditional risk factors for ASCVD did not exhibit such associations. Our study also revealed associations between germline genetic variants and incident CH, prioritizing genes in CH development. Our comprehensive longitudinal assessment yields novel insights into the factors contributing to incident CH in older adults.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.23294769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.23294769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.23294769">
        <p class="paperTitle">The predictive capacity of polygenic risk scores for disease risk is only moderately influenced by imputation panels tailored to the target population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.23294769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.23294769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levi, H.; Elkon, R.; Shamir, R.</p>
        <p class="info">Score: 4.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.23294769' target='https://doi.org/10.1101/2023.08.29.23294769'> 10.1101/2023.08.29.23294769</a></p>
        <p class="abstract">Polygenic risk scores (PRS) predict individuals genetic risk of developing complex diseases. They summarize the effect of many genetic variants discovered in genome-wide association studies (GWASs). However, to date, large GWASs exist primarily for the European population and the quality of PRS prediction declines when applied to target sets of other ethnicities. A key step in using a PRS is imputation, which is the inference of un-typed SNPs using a set of fully-sequenced individuals, called the imputation panel. The SNP genotypes called by the imputation process depend on the ethnic composition of the imputation panel. Several studies have shown that imputing genotypes using a panel that contains individuals of the same ethnicity as the genotyped individuals improves imputation accuracy. However, until now, there has been no systematic investigation into the influence of the ethnic composition of imputation panels on the accuracy of PRS predictions when applied to ethnic groups that differ from the population used in the GWAS.

In this study we estimated the effect of imputation of the target set on prediction accuracy of PRS when the discovery (GWAS) and the target sets come from different ethnic groups. We analyzed twelve binary phenotypes and three populations from the UK Biobank (Europeans, South-Asians, and Africans). We generated imputation panels from several ethnic groups, imputed the target set using each panel, and generated PRS to compute individuals risk scores. Then, we assessed the prediction accuracy obtained from each imputation panel. Our analysis indicates that using an imputation panel matched to the ethnic group of the target population yields only a marginal improvement and only under specific conditions. Hence, while a target-matched imputation panel can potentially improve prediction accuracy of European PRSs in non-EUR populations, the improvement is limited.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294074">
        <p class="paperTitle">Rare coding variants in schizophrenia-associated genes affect generalised cognition in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fenner, E.; Holmans, P.; O&#39;Donovan, M. C.; Owen, M. J.; Walters, J. T.; Rees, E.</p>
        <p class="info">Score: 13.7, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294074' target='https://doi.org/10.1101/2023.08.14.23294074'> 10.1101/2023.08.14.23294074</a></p>
        <p class="abstract">Impairments in cognitive function are a feature of schizophrenia that strongly predict functional outcome and are generally not improved by current medications. However, the nature of the relationship between cognitive impairment and schizophrenia risk, and particularly the extent to which this reflects shared underlying biology, remains uncertain. We analysed exome-sequencing data from the UK Biobank to test for association between generalised cognition and damaging rare coding variation in genes and loci associated with schizophrenia in 30,487 people without the disorder. Rare protein-truncating variants (PTVs) and damaging missense variants in loss-of-function intolerant (LoFi) genes were associated with lower generalised cognition. Moreover, we found significantly stronger effects for damaging missense variants in credible causal genes at schizophrenia GWAS loci and for rare PTVs affecting LoFi genes in regions defined by schizophrenia-enriched CNVs. This suggests shared underlying biology between schizophrenia risk and general cognitive function in the population, and that exploiting large population sequencing datasets to identify genes with shared effects on cognition and schizophrenia can provide a route towards determining biological processes underlying cognitive impairment in schizophrenia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.23294773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.23294773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.23294773">
        <p class="paperTitle">Relationship between sex biases in gene expression and sex biases in autism and Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.23294773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.23294773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fass, S. B.; Mulvey, B.; Yang, W.; Selmanovic, D.; Chaturvedi, S.; Tycksen, E.; Weiss, L.; Dougherty, J.</p>
        <p class="info">Score: 5.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.23294773' target='https://doi.org/10.1101/2023.08.29.23294773'> 10.1101/2023.08.29.23294773</a></p>
        <p class="abstract">Sex differences in the brain may play an important role in sex-differential prevalence of neuropsychiatric conditions. In order to understand the transcriptional basis of sex differences, we analyzed multiple, large-scale, human postmortem brain RNA-seq datasets using both within-region and pan-regional frameworks. We find evidence of sex-biased transcription in many autosomal genes, some of which provide evidence for pathways and cell population differences between chromosomally male and female individuals. These analyses also highlight regional differences in the extent of sex-differential gene expression. We observe an increase in specific neuronal transcripts in male brains and an increase in immune and glial function-related transcripts in female brains. Integration with single-cell data suggests this corresponds to sex differences in cellular states rather than cell abundance. Integration with case-control gene expression studies suggests a female molecular predisposition towards Alzheimers disease, a female-biased disease. Autism, a male-biased diagnosis, does not exhibit a male predisposition pattern in our analysis. Finally, we provide region specific analyses of sex differences in brain gene expression to enable additional studies at the interface of gene expression and diagnostic differences.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=137 SRC=&#34;FIGDIR/small/23294773v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@9b00a1org.highwire.dtl.DTLVardef@467465org.highwire.dtl.DTLVardef@183b2d4org.highwire.dtl.DTLVardef@1517e19_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.23295126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.23295126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.23295126">
        <p class="paperTitle">Genome-wide association study and genomic risk prediction of age-related macular degeneration in Israel</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.23295126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.23295126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grunin, M.; Triffon, D.; Beykin, G.; Rahmani, E.; Schweiger, R.; Tiosano, L.; Khateb, S.; Hagbi-Levi, S.; Rinsky, B.; Munitz, R.; Winkler, T. W.; Heid, I. M.; Halperin, E.; Carmi, S.; Chowers, I.</p>
        <p class="info">Score: 3.6, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.23295126' target='https://doi.org/10.1101/2023.09.06.23295126'> 10.1101/2023.09.06.23295126</a></p>
        <p class="abstract">PurposeThe risk of developing age-related macular degeneration(AMD) is influenced by genetic background. In 2016, International AMD Genomics Consortium(IAMDGC) identified 52 risk variants in 34 loci, and a polygenic risk score(PRS) based on these variants was associated with AMD. The Israeli population has a unique genetic composition: Ashkenazi Jewish(AJ), Jewish non-Ashkenazi, and Arab sub-populations. We aimed to perform a genome-wide association study(GWAS) for AMD in Israel, and to evaluate PRSs for AMD.

MethodsFor our discovery set, we recruited 403 AMD patients and 256 controls at Hadassah Medical Center. We genotyped all individuals via custom exome chip. We imputed non-typed variants using cosmopolitan and AJ reference panels. We recruited additional 155 cases and 69 controls for validation. To evaluate predictive power of PRSs for AMD, we used IAMDGC summary statistics excluding our study and developed PRSs via either clumping/thresholding or LDpred2.

ResultsIn our discovery set, 31/34 loci previously reported by the IAMDGC were AMD associated with P&lt;0.05. Of those, all effects were directionally consistent with the IAMDGC and 11 loci had a p-value under Bonferroni-corrected threshold(0.05/34=0.0015). At a threshold of 5x10-5, we discovered four suggestive associations in FAM189A1, IGDCC4, C7orf50, and CNTNAP4. However, only the FAM189A1 variant was AMD associated in the replication cohort after Bonferroni-correction. A prediction model including LDpred2-based PRS and other covariates had an AUC of 0.82(95%CI:0.79-0.85) and performed better than a covariates-only model(P=5.1x10-9).

ConclusionsPreviously reported AMD-associated loci were nominally associated with AMD in Israel. A PRS developed based on a large international study is predictive in Israeli populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.23294688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.23294688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.23294688">
        <p class="paperTitle">Identifying X-Chromosome Variants Associated with Age-Related Macular Degeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.23294688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.23294688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grunin, M.; Igo, R. P.; Song, Y. E.; Blanton, S. H.; Pericak-Vance, M. A.; Haines, J. L.; International Age-related Macular Degeneration Genomics Consortium,  </p>
        <p class="info">Score: 6.2, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.23294688' target='https://doi.org/10.1101/2023.08.28.23294688'> 10.1101/2023.08.28.23294688</a></p>
        <p class="abstract">PurposeIn genome-wide association studies (GWAS), X chromosome (ChrX) variants are often not investigated. Sex-specific effects and ChrX-specific quality control (QC) are needed to examine these effects. Previous analyses identified 52 autosomal variants associated with age-related macular degeneration (AMD) via the International AMD Genomics Consortium (IAMDGC), but did not analyze ChrX. Therefore, our goal was to investigate ChrX variants for association with AMD.

MethodsWe genotyped 29,629 non-Hispanic White (NHW) individuals (M/F:10,404/18,865; AMD12,087/14723) via a custom chip and imputed after ChrX-specific QC (XWAS 3.0) using the Michigan Imputation Server. Imputation generated 1,221,623 variants on ChrX. Age, informative PCs, and subphenotyeps were covariates for logistic association analyses with Fishers correction. Gene/pathway analyses were performed with VEGAS, GSEASNP, ICSNPathway, DAVID, and mirPath.

ResultsLogistic association on NHW individuals with sex correction, identified variants in/near the genes SLITRK4, ARHGAP6, FGF13 and DMD associated with AMD (P&lt;1x10-6,Fishers combined-corrected). Via association testing of subphenotypes of choroidal neovascularization and geographic atrophy (GA), variants in DMD associated with GA (P&lt;1x10-6, Fishers combined-corrected). Via gene-based analysis with VEGAS, several genes were associated with AMD (P&lt;0.05, both truncated tail strength/truncated product P) including SLITRK4 and BHLHB9. Pathway analysis using GSEASNP and DAVID showed genes associated with nervous system development (FDR: P:0.02), and blood coagulation (FDR: P:0.03). Variants in the region of a microRNA (miR) were associated with AMD (P&lt;0.05, truncated tail strength/truncated product P). Via DIANA mirPath analysis, downstream targets of miRs show association with brain disorders and fatty acid elongation (P&lt;0.05). A long-non coding RNA on ChrX near the DMD locus was also associated with AMD (P=4x10-7). Epistatic analysis (t-statistic) for a quantitative trait of AMD vs control including covariates found a suggestive association in the XG gene (P=2x10^-5).

ConclusionsAnalysis of ChrX variants demonstrates association with AMD and these variants may be linked to novel pathways. Further analysis is needed to confirm results and to understand their biological significance and relationship with AMD development in worldwide populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.23294848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.23294848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.23294848">
        <p class="paperTitle">The pivotal role of the X-chromosome in the genetic architecture of the human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.23294848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.23294848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Z.; Sullivan, P. F.; Li, T.; Zhao, B.; Wang, X.; Luo, T.; Huang, S.; Guan, P. Y.; Chen, J.; Yang, Y.; Stein, J. L.; Li, Y.; Liu, D.; Sun, L.; Zhu, H.</p>
        <p class="info">Score: 6.4, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.23294848' target='https://doi.org/10.1101/2023.08.30.23294848'> 10.1101/2023.08.30.23294848</a></p>
        <p class="abstract">Genes on the X-chromosome are extensively expressed in the human brain, resulting in substantial influences on brain development, intellectual disability, and other brain-related disorders. To comprehensively investigate the X-chromosomes impact on the cerebral cortex, white matter tract microstructures, and intrinsic and extrinsic brain functions, we examined 2,822 complex brain imaging traits obtained from n = 34,000 subjects in the UK Biobank. We unveiled potential autosome-X-chromosome interaction, while proposing an atlas of dosage compensation (DC) for each set of traits. We observed a pronounced X-chromosome impact on the corticospinal tract and the functional amplitude and connectivity of visual networks. In association studies, we identified 50 genome-wide significant trait-locus pairs enriched in Xq28, 22 of which replicated in independent datasets (n = 4,900). Notably, 13 newly identified pairs were in the X-chromosomes non-pseudo-autosomal regions (NPR). The volume of the right ventral diencephalon shared genetic architecture with schizophrenia and educational attainment in a locus indexed by rs2361468 (located [~]3kb upstream of PJA1, a conserved and ubiquitously expressed gene implicated in multiple psychiatric disorders). No significant associations were identified in the pseudo-autosomal regions (PAR) or the Y-chromosome. Finally, we explored sex-specific associations on the X-chromosome and compared differing genetic effects between sexes. We found much more associations can be identified in males (33 versus 9) given a similar sample size. In conclusion, our research provides invaluable insights into the X-chromosomes role in the human brain, contributing to the observed sex differences in brain structure and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.23294686">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.23294686" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.23294686">
        <p class="paperTitle">Spatial Distribution of Missense Variants within Complement Proteins Associates with Age Related Macular Degeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.23294686" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.23294686" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grunin, M.; de Jong, S.; Palmer, E. L.; Jin, B.; Rinker, D.; Moth, C.; Capra, J. A.; Haines, J. L.; Bush, W.; den Hollander, A.; International Age-related Macular Degeneration Genomics Consortium,  </p>
        <p class="info">Score: 2.1, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.23294686' target='https://doi.org/10.1101/2023.08.28.23294686'> 10.1101/2023.08.28.23294686</a></p>
        <p class="abstract">PurposeGenetic variants in complement genes are associated with age-related macular degeneration (AMD). However, many rare variants have been identified in these genes, but have an unknown significance, and their impact on protein function and structure is still unknown. We set out to address this issue by evaluating the spatial placement and impact on protein structureof these variants by developing an analytical pipeline and applying it to the International AMD Genomics Consortium (IAMDGC) dataset (16,144 AMD cases, 17,832 controls).

MethodsThe IAMDGC dataset was imputed using the Haplotype Reference Consortium (HRC), leading to an improvement of over 30% more imputed variants, over the original 1000 Genomes imputation. Variants were extracted for the CFH, CFI, CFB, C9, and C3 genes, and filtered for missense variants in solved protein structures. We evaluated these variants as to their placement in the three-dimensional structure of the protein (i.e. spatial proximity in the protein), as well as AMD association. We applied several pipelines to a) calculate spatial proximity to known AMD variants versus gnomAD variants, b) assess a variants likelihood of causing protein destabilization via calculation of predicted free energy change (ddG) using Rosetta, and c) whole gene-based testing to test for statistical associations. Gene-based testing using seqMeta was performed using a) all variants b) variants near known AMD variants or c) with a ddG &gt;|2|. Further, we applied a structural kernel adaptation of SKAT testing (POKEMON) to confirm the association of spatial distributions of missense variants to AMD. Finally, we used logistic regression on known AMD variants in CFI to identify variants leading to &gt;50% reduction in protein expression from known AMD patient carriers of CFI variants compared to wild type (as determined by in vitro experiments) to determine the pipelines robustness in identifying AMD-relevant variants. These results were compared to functional impact scores, ie CADD values &gt; 10, which indicate if a variant may have a large functional impact genomewide, to determine if our metrics have better discriminative power than existing variant assessment methods. Once our pipeline had been validated, we then performed a priori selection of variants using this pipeline methodology, and tested AMD patient cell lines that carried those selected variants from the EUGENDA cohort (n=34). We investigated complement pathway protein expression in vitro, looking at multiple components of the complement factor pathway in patient carriers of bioinformatically identified variants.

ResultsMultiple variants were found with a ddG&gt;|2| in each complement gene investigated. Gene-based tests using known and novel missense variants identified significant associations of the C3, C9, CFB, and CFH genes with AMD risk after controlling for age and sex (P=3.22x10-5;7.58x10-6;2.1x10-3;1.2x10-31). ddG filtering and SKAT-O tests indicate that missense variants that are predicted to destabilize the protein, in both CFI and CFH, are associated with AMD (P=CFH:0.05, CFI:0.01, threshold of 0.05 significance). Our structural kernel approach identified spatial associations for AMD risk within the protein structures for C3, C9, CFB, CFH, and CFI at a nominal p-value of 0.05. Both ddG and CADD scores were predictive of reduced CFI protein expression, with ROC curve analyses indicating ddG is a better predictor (AUCs of 0.76 and 0.69, respectively). A priori in vitro analysis of variants in all complement factor genes indicated that several variants identified via bioinformatics programs PathProx/POKEMON in our pipeline via in vitro experiments caused significant change in complement protein expression (P=0.04) in actual patient carriers of those variants, via ELISA testing of proteins in the complement factor pathway, and were previously unknown to contribute to AMD pathogenesis.

ConclusionWe demonstrate for the first time that missense variants in complement genes cluster together spatially and are associated with AMD case/control status. Using this method, we can identify CFI and CFH variants of previously unknown significance that are predicted to destabilize the proteins. These variants, both in and outside spatial clusters, can predict in-vitro tested CFI protein expression changes, and we hypothesize the same is true for CFH. A priori identification of variants that impact gene expression allow for classification for previously classified as VUS. Further investigation is needed to validate the models for additional variants and to be applied to all AMD-associated genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.23295182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.23295182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.23295182">
        <p class="paperTitle">Circulating anti-Müllerian hormone levels in pre-menopausal women: novel genetic insights from a GWAS meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.23295182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.23295182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pujol Gualdo, N.; Karjalainen, M. K.; Vosa, U.; Arffman, R. K.; Mägi, R.; Ronkainen, J.; Laisk, T.; Piltonen, T. T.</p>
        <p class="info">Score: 1.8, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.23295182' target='https://doi.org/10.1101/2023.09.07.23295182'> 10.1101/2023.09.07.23295182</a></p>
        <p class="abstract">Study question: Can a genome-wide association study (GWAS) meta-analysis, including a large sample of young premenopausal women from a founder population from Northern Finland, identify novel genetic variants for circulating anti-Mullerian hormone (AMH) levels and provide insights into biological pathways and tissues involved in AMH regulation? Summary answer: We identified six loci associated with AMH levels at P &lt; 5 x 10-8, including the previously reported MCM8, AMH and TEX41 loci, and three novel signals in or near CHEK2, BMP4 and EIF4EBP1. Gene set enrichment analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranks the pituitary gland as a top associated tissue. What is known already: AMH is expressed by preantral and small antral stage ovarian follicles in women, and variation in age-specific circulating AMH levels has been associated with several health conditions. However, the biological mechanisms underlying the association between health conditions and AMH levels are not yet fully understood. Previous GWAS have identified loci associated with AMH levels in pre-menopausal women, but they were limited by small sample sizes or focused mostly on older pre-menopausal women. Study design, size, duration: We performed a GWAS meta-analysis for AMH level measurements in 9,668 pre-menopausal women. Participants/materials, setting, methods: We performed a GWAS meta-analysis in which we combined 2,619 AMH measurements (at age 31 years old) from a prospective founder population cohort (Northern Finland Birth Cohort 1966, NFBC1966) with a previous GWAS meta-analysis that included 7,049 pre-menopausal women (spanning age range 15-48). NFBC1966 AMH measurements were quantified using an automated assay (Elecsys AMH Plus (Roche)). We annotated the genetic variants, combined different data layers to prioritise potential candidate genes, described significant pathways and tissues enriched by the GWAS signals, identified plausible regulatory roles using colocalization analysis and leveraged publicly available summary statistics to assess genetic and phenotypic correlations with multiple traits. Main results and the role of chance: Three novel genome-wide significant loci were identified. One of these is in complete linkage disequilibrium with c.1100delC in CHEK2, which is found to be 4-fold enriched in the Finnish population compared to other European populations. We propose a plausible regulatory effect of some of the GWAS variants linked to AMH, as they colocalise with GWAS signals associated with gene expression levels of BMP4, TEX41 and EIFBP41. Gene set analysis highlighted significant enrichment in renal system vasculature morphogenesis and tissue enrichment analysis ranked the pituitary gland as the top association. Large scale data: The GWAS meta-analysis summary statistics will be available for download from the GWAS Catalog. Accession numbers will be provided upon publication. Limitations, reasons for caution: This study only included women of European ancestry and the unavailability of sufficiently sized relevant tissue data in gene expression datasets hinders the assessment of potential regulatory effects in reproductive tissues. Wider implications of the findings: Our results highlight the increased power of founder populations and larger sample sizes to boost the discovery of novel trait-associated variants underlying variation in AMH levels, which aided to characterise novel biological pathways and plausible genetic regulatory effects linked with AMH levels variation for the first time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.23294951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.23294951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.23294951">
        <p class="paperTitle">Prioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.23294951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.23294951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pena-Martinez, E. G.; Pomales-Matos, D. A.; Rivera-Madera, A.; Messon-Bird, J. L.; Medina-Feliciano, J. G.; Sanabria-Alberto, L.; Barreiro-Rosario, A. C.; Rodriguez-Rios, J. M.; Rodriguez-Martinez, J. A.</p>
        <p class="info">Score: 1.8, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.23294951' target='https://doi.org/10.1101/2023.09.01.23294951'> 10.1101/2023.09.01.23294951</a></p>
        <p class="abstract">Cardiovascular diseases (CVDs) are the leading cause of death worldwide and are heavily influenced by genetic factors. Genome-wide association studies (GWAS) have mapped &gt; 90% of CVD-associated variants within the non-coding genome, which can alter the function of regulatory proteins, like transcription factors (TFs). However, due to the overwhelming number of GWAS single nucleotide polymorphisms (SNPs) (&gt;500,000), prioritizing variants for in vitro analysis remains challenging. In this work, we implemented a computational approach that considers support vector machine (SVM)-based TF binding site classification and cardiac expression quantitative trait loci (eQTL) analysis to identify and prioritize potential CVD-causing SNPs. We identified 1,535 CVD-associated SNPs that occur within human heart footprints/enhancers and 9,309 variants in linkage disequilibrium (LD) with differential gene expression profiles in cardiac tissue. Using hiPSC-CM ChIP-seq data from NKX2-5 and TBX5, two cardiac TFs essential for proper heart development, we trained a large-scale gapped k-mer SVM

(LS-GKM-SVM) predictive model that can identify binding sites altered by CVD-associated SNPs. The computational predictive model was tested by scoring human heart footprints and enhancers in vitro through electrophoretic mobility shift assay (EMSA). Three variants (rs59310144, rs6715570, and rs61872084) were prioritized for in vitro validation based on their eQTL in cardiac tissue and LS-GKM-SVM prediction to alter NKX2-5 DNA binding. All three variants altered NKX2-5 DNA binding. In summary, we present a bioinformatic approach that considers tissue-specific eQTL analysis and SVM-based TF binding site classification to prioritize CVD-associated variants for in vitro experimental analysis.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=55 SRC=&#34;FIGDIR/small/23294951v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (18K):
org.highwire.dtl.DTLVardef@ad40c2org.highwire.dtl.DTLVardef@768501org.highwire.dtl.DTLVardef@1161861org.highwire.dtl.DTLVardef@19aa06d_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
