<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors
Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.
Score: 35.3, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414
BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors
Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.
Score: 35.3, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414
BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors
Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.
Score: 35.3, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414
BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors\nAuthors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.\nScore: 35.3, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414\nBackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1.",
  "keywords": [
    
  ],
  "articleBody": " Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors\nAuthors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.\nScore: 35.3, Published: 2023-10-24 DOI: 10.1101/2023.10.23.23297414\nBackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS). MethodWe used a dataset from 23andMe (6.77M European, 1.30M Latine, and 0.45M African American individuals). Using cross-sectional data for PRS construction and a prospective cohort for evaluation, we estimated PRS-associated cumulative incidences after one year of follow-up for 12 clinical endpoints. ResultsThe cumulative incidence of disease at one year was consistently higher among individuals in the top 10% of each PRS. Relative risks (RRs) comparing top to bottom 10% of PRS differed across diseases (e.g. European RR 2.12 for eczema vs 12.53 for T2D). Estimates were similar between Europeans and Latines however were more modest for African Americans (e.g. T2D RR 10.92 for Latines vs. 4.00 for African Americans). Clinical manifestation occurred earlier among those in top vs bottom 10% of polygenic risk: 16yrs for hypertension, and 9.5yrs for T2D. Among participants at elevated conventional risk of CHD or T2D, those in the top 10% PRS had a 10-20 fold higher RR of disease incidence vs those not at conventional risk. Among individuals at high polygenic risk of CHD or T2D, favorable lifestyle characteristics associated with 64-73% lower RR of developing disease over 1-year, with cumulative incidence equivalent to the population average. ConclusionIn an ancestrally-diverse cohort, individuals in the top 10% PRS had higher 1-year disease incidence and earlier age of clinical manifestation. PRS provided risk stratification beyond conventional risk factors. Lifestyle characteristics markedly lowered disease incidence among those at elevated polygenic risk.\nDecoding Genetics, Ancestry, and Geospatial Context for Precision Health\nAuthors: Koyama, S.; Wang, Y.; Paruchuri, K.; Uddin, M. M.; Cho, S. M. J.; Urbut, S. M.; Haidermota, S.; Hornsby, W. E.; Green, R. C.; Daly, M. J.; Neale, B. M.; Ellinor, P. T.; Smoller, J. W.; Lebo, M. S.; Karlson, E. W.; Martin, A. R.; Natarajan, P.\nScore: 32.2, Published: 2023-10-25 DOI: 10.1101/2023.10.24.23297096\nMass General Brigham, an integrated healthcare system based in the Greater Boston area of Massachusetts, annually serves 1.5 million patients. We established the Mass General Brigham Biobank (MGBB), encompassing 142,238 participants, to unravel the intricate relationships among genomic profiles, environmental context, and disease manifestations within clinical practice. In this study, we highlight the impact of ancestral diversity in the MGBB by employing population genetics, geospatial assessment, and association analyses of rare and common genetic variants. The population structures captured by the genetics mirror the sequential immigration to the Greater Boston area throughout American history, highlighting communities tied to shared genetic and environmental factors. Our investigation underscores the potency of unbiased, large-scale analyses in a healthcare-affiliated biobank, elucidating the dynamic interplay across genetics, immigration, structural geospatial factors, and health outcomes in one of the earliest American sites of European colonization.\nPlasticity of Human Microglia and Brain Perivascular Macrophages in Aging and Alzheimer's Disease\nAuthors: Lee, D.; Porras, C.; Spencer, C.; Pjanic, M.; Weiler, P.; Kosoy, R.; Bendl, J.; N.M., P.; Wang, X.; Zheng, S.; Therrien, K.; Mathur, D.; Kleopoulos, S. P.; Shao, Z.; Argyriou, S.; Alvia, M.; Casey, C.; Hong, A.; Beaumont, K. G.; Sebra, R.; Kellner, C. P.; Bennett, D. A.; Yuan, G.-C.; Voloudakis, G.; Theis, F. J.; Haroutunian, V.; Hoffman, G. E.; Fullard, J. F.; Roussos, P.\nScore: 12.5, Published: 2023-10-26 DOI: 10.1101/2023.10.25.23297558\nMicroglia and perivascular macrophages, myeloid-origin resident immune cells in the human brain, play crucial roles in Alzheimers disease (AD)1-4. However, the field lacks a unified taxonomy describing their heterogeneity and plasticity5. To address this, we applied single-cell profiling to two independent, demographically diverse cohorts. The first comprises 543,012 viable myeloid cells from 137 unique postmortem brain specimens, while the second consists of 289,493 myeloid nuclei from 1,470 donors. Collectively, they cover the human lifespan and varying degrees of AD neuropathology. We identify 13 transcriptionally distinct myeloid subtypes, including the \"GPNMB\" subtype that proliferates with AD. We distinguish two contrasting homeostatic microglial states in AD and with aging: the first (\"FRMD4A\") wanes over time, while the second (\"PICALM\") becomes more prevalent. By prioritizing AD-risk genes, including PTPRG, DPYD, and IL15, and placing them into a regulatory hierarchy, we identify common upstream transcriptional regulators, namely MITF and KLF12, that regulate the expression of AD-risk genes in the opposite directions. Through the construction of cell-to-cell interaction networks, we identify candidate ligand-receptor pairs, including APOE:SORL1 and APOE:TREM2, associated with AD progression. We show polygenic risk for AD predisposes and prioritize the GPNMB subtype as a therapeutic target of early intervention. Our findings delineate the relationship between distinct functional states of myeloid cells and their pathophysiological response to aging and AD, providing a significant step toward the mechanistic understanding of the roles of microglia in AD and the identification of novel therapeutics.\nCXCL12 regulates coronary artery dominance in diverse populations and links development to disease\nAuthors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.\nScore: 10.8, Published: 2023-10-27 DOI: 10.1101/2023.10.27.23297507\nMammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.\nDNAm scores for serum GDF15 and NT-proBNP levels associate with a range of traits affecting the body and brain\nAuthors: Gadd, D. A.; Smith, H.; Mullin, D.; Chybowska, A.; Hillary, R. F.; Kimenai, D. M.; Bernabeu, E.; Cheng, Y.; Fawns-Ritchie, C.; Campbell, A.; Page, D.; Taylor, A.; Corley, J.; Valdes Hernandez, M. D. C.; Munoz Maniega, S.; Bastin, M. E.; Wardlaw, J. M.; Walker, R. M.; Evans, K. L.; McIntosh, A. M.; Hayward, C.; Russ, T. C.; Harris, S. E.; Welsh, P.; Sattar, N.; Cox, S. R.; McCartney, D. L.; Marioni, R. E.\nScore: 9.8, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297200\nBackgroundPlasma growth differentiation factor 15 (GDF15) and N-terminal pro-B-type natriuretic peptide (NT-proBNP) are cardiovascular biomarkers that associate with a range of diseases. Epigenetic scores (EpiScores) for GDF15 and NT-proBNP may provide new routes for risk stratification. ResultsIn the Generation Scotland cohort (N [\u0026ge;] 16,963), GDF15 levels were associated with incident dementia, ischaemic stroke and type 2 diabetes, whereas NT-proBNP levels were associated with incident ischaemic heart disease, ischaemic stroke and type 2 diabetes (all PFDR \u003c 0.05). Bayesian Epigenome-wide association studies (EWAS) identified 12 and 4 DNA methylation (DNAm) CpG sites associated (Posterior Inclusion Probability [PIP] \u003e 95%) with levels of GDF15 and NT-proBNP, respectively. EpiScores for GDF15 and NT-proBNP that were trained in a subset of the population. The GDF15 EpiScore replicated protein associations with incident dementia, type 2 diabetes and ischaemic stroke in the Generation Scotland test set (Hazard Ratios (HR) range 1.36 - 1.41, PFDR \u003c0.03). The EpiScore for NT-proBNP replicated the protein association with type 2 diabetes, but failed to replicate an association with ischaemic stroke. EpiScores explained comparable variance in protein levels across both the Generation Scotland test set and the external LBC1936 test cohort (R2 range of 5.7-12.2%). In LBC1936, both EpiScores were associated with indicators of poorer brain health. Neither EpiScore was associated with incident dementia in the LBC1936 population. ConclusionsEpiScores for serum levels of GDF15 and Nt-proBNP associate with body and brain health traits. These EpiScores are provided as potential tools for disease risk stratification.\nMendelian randomization study of birth weight and risk of psychiatric disorders later in life\nAuthors: Wu, X.; Geller, F.; Helenius, D.; Grove, J.; Albinana, C.; Petersen, L. V.; Bulik, C.; Borglum, A.; Werge, T.; Vilhjalmsson, B. J.; Feenstra, B.\nScore: 5.5, Published: 2023-10-26 DOI: 10.1101/2023.10.26.23297618\nLow birth weight has been associated with a higher risk of psychiatric disorders later in life. The underlying causal mechanisms of this relationship are however not clear. In this study, we investigate whether variation in fetal growth has a direct causal effect on mental health. Using birth weight as a proxy measure for fetal growth, we first assessed associations between observed birth weight and later diagnosis of psychiatric disorders in the Danish iPSYCH and ANGI-DK cohorts. Next, we constructed a polygenic score for birth weight based on \u003e1 million variants and tested for association with psychiatric disorders. Finally, using 86 single-nucleotide polymorphisms with robust fetal-only genetic associations with birth weight, we assessed the causal relationship of genetically mediated fetal growth and psychiatric disorders using Mendelian randomization analyses. We found that higher observed birth weight was associated with lower risk of several psychiatric disorders. Polygenic score analyses supported this pattern for attention deficit/hyperactivity disorder, where an increase of one standard deviation in the score for birth weight corresponded to an odds ratio of 0.85 (95% confidence interval 0.79-0.92, P=6.27x10-5). However, one- and two-sample Mendelian randomization analyses did not indicate a direct causal relationship between the birth weight of children and their risk of psychiatric disorders. In conclusion, our study does not support a direct causal effect of fetal growth (as proxied by birth weight) on the risk of psychiatric disorders later in life, suggesting that the observed association between birth weight and mental health is likely to be caused by other factors.\nBiallelic variants in HMGCS1 are a novel cause of rare rigid spine syndrome\nAuthors: Dofash, L. N. H.; Miles, L. B.; Saito, Y.; Rivas, E.; Calcinotto, V.; Oveissi, S.; Serrano, R. J.; Templin, R.; Ramm, G.; Rodger, A.; Haywood, J.; Ingley, E.; Clayton, J. S.; Taylor, R. L.; Folland, C. L.; Groth, D.; Hock, D. H.; Stroud, D. A.; Gorokhova, S.; Donkervoort, S.; Bönnemann, C. G.; Sud, M.; VanNoy, G.; Mangilog, B. E.; Pais, L.; Madruga-Garrido, M.; Scala, M.; Fiorillo, C.; Baratto, S.; Traverso, M.; Bruno, C.; Zara, F.; Paradas, C.; Ogata, K.; Nishino, I.; Laing, N. G.; Bryson-Richardson, R. J.; Cabrera-Serrano, M.; Ravenscroft, G.\nScore: 4.6, Published: 2023-10-28 DOI: 10.1101/2023.10.25.23297129\nRigid spine syndrome is a rare childhood-onset myopathy characterised by slowly progressive or non-progressive scoliosis, neck and spine contractures, hypotonia, and respiratory insufficiency. Biallelic variants in SELENON account for most cases of rigid spine syndrome, however, the underlying genetic cause in some patients remains unexplained. In this study, we used exome and genome sequencing to investigate the genetic basis of rigid spine syndrome in patients without a genetic diagnosis. In five patients from four unrelated families, we identified biallelic variants in HMGCS1 (3-hydroxy-3-methylglutaryl-coenzyme A synthase). These included six missense variants and one frameshift variant distributed throughout HMGCS1. All patients presented with spinal rigidity primarily affecting the cervical and dorsolumbar regions, scoliosis, and respiratory insufficiency. Creatine kinase levels were variably elevated. The clinical course worsened with intercurrent disease or certain drugs in some patients; one patient died from respiratory failure following infection. Muscle biopsies revealed irregularities in oxidative enzyme staining with occasional internal nuclei and rimmed vacuoles. HMGCS1 encodes a key enzyme of the mevalonate pathway, disturbance of which is also associated with HMGCR-limb girdle muscular dystrophy and GGPS1-muscular dystrophy. Quantitative PCR and western blotting confirmed HMGCS1 abundance in skeletal muscle and myogenic precursors. HMGCS1 levels in skeletal muscle were comparable between healthy controls and the index case with a homozygous p{middle dot}(S447P) substitution. Muscle RNA-seq for a patient with a frameshift variant (c{middle dot}344_345del:p{middle dot}S115Wfs*12) and an in trans substitution (p{middle dot}(Q29L)) showed HMGCS1 transcript levels reduced to 53% compared to controls. The substitution appeared homozygous on RNA-seq, suggesting the allele harbouring the frameshift variant undergoes nonsense mediated decay. hmgcs1 null zebrafish displayed severe early defects, including immobility at 2 days and death by days 3 post-fertilisation. We anticipate that the variants observed in this cohort have subtle effects on HMGCS1 function given most patients survived to adulthood. In support of the variants being hypomorphic, analyses of recombinant human HMGCS1 protein and four mutants (p{middle dot}S447P, p{middle dot}Q29L, p{middle dot}M70T, p{middle dot}C268S) showed all mutants maintained their secondary structure and dimerized state and had enzymatic activity comparable to the wildtype. Thermal stability of the mutants was similar or slightly reduced compared to the wildtype. Altogether, our analyses suggest that the identified missense variants in HMGCS1 act through a hypomorphic mechanism yet to be elucidated. Here, we report an additional component of the mevalonate pathway associated with myopathy and suggest biallelic variants in HMGCS1 should be considered in patients presenting with an unresolved rigid spine phenotype.\nGenome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves cross-ancestry prediction\nAuthors: Hoffmann, T. J.; Graff, R. E.; Madduri, R. K.; Rodriguez, A. A.; Cario, C. L.; Feng, K.; Jiang, Y.; Wang, A.; Klein, R. J.; Pierce, B. L.; Eggener, S.; Tong, L.; Blot, W.; Long, J.; Rebbeck, T.; Lachance, J.; Andrews, C.; Adebiyi, A. O.; Adusei, B.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Jalloh, M.; Janivara, R.; Chen, W. C.; Mensah, J. E.; Agalliu, I.; Berndt, S. I.; Shelley, J. P.; Schaffer, K.; Machiela, M. J.; Freedman, N. D.; Huang, W.-Y.; Li, S. A.; Goodman, P. J.; Till, C.; Thompson, I.; Lilja, H.; Van Den Eeden, S. K.; Chanock, S. J.; Mosley, J. D.; Conti, D. V.; Haiman, C. A.;\nScore: 4.5, Published: 2023-10-30 DOI: 10.1101/2023.10.27.23297676\nWe conducted a multi-ancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry; 58,236 African ancestry; 23,546 Hispanic/Latino; 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (p[\u0026le;]5e-8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n=95,768). Meta-analyzing discovery and replication (n=392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our new polygenic risk score reached 16.9% (95% CI=16.1%-17.8%) in European ancestry, 9.5% (95% CI=7.0%-12.2%) in African ancestry, 18.6% (95% CI=15.8%-21.4%) in Hispanic/Latino, and 15.3% (95% CI=12.7%-18.1%) in Asian ancestry, and lower for higher age. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, with potential to personalize prostate cancer screening.\nDeterminants of mosaic chromosomal alteration fitness\nAuthors: Pershad, Y.; Mack, T.; Poisner, H. M.; Jakubek, Y. A.; Stilp, A. M.; Mitchell, B. D.; Lewis, J. P.; Boerwinkle, E.; Loos, R. J.; Chami, N.; Wang, Z.; Barnes, K.; Pankratz, N.; Fornage, M.; Redline, S.; Psaty, B. M.; Bis, J. C.; Shojaie, A.; Silverman, E. K.; Cho, M. H.; Yun, J.; DeMeo, D.; Levy, D.; Johnson, A.; Mathias, R.; Taub, M.; Arnett, D.; North, K.; Raffield, L. M.; Carlson, A.; Doyle, M. F.; Rich, S. S.; Rotter, J. I.; Guo, X.; Cox, N.; Roden, D. M.; Franceschini, N.; Desai, P.; Reiner, A.; Auer, P. L.; Scheet, P.; Jaiswal, S.; Weinstock, J. S.; Bick, A. G.\nScore: 14.8, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297280\nClonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal Expansion Rate (PACER) method to estimate clonal expansion rate for 6,381 individuals in the NHLBI TOPMed cohort with gain, loss, and copy-neutral loss of heterozygosity mCAs. Our estimates of mCA fitness were correlated (R2 = 0.49) with an alternative approach that estimated fitness of mCAs in the UK Biobank using a theoretical probability distribution. Individuals with lymphoid-associated mCAs had a significantly higher white blood cell count and faster clonal expansion rate. In a cross-sectional analysis, genome-wide association study of estimates of mCA expansion rate identified TCL1A, NRIP1, and TERT locus variants as modulators of mCA clonal expansion rate.\nGenome-wide DNA methylation, imprinting, and gene expression in human placentas derived from Assisted Reproductive Technology\nAuthors: Auvinen, P.; Vehvilainen, J.; Ramo, K.; Laukkanen, I.; Marjonen-Lindblad, H.; Wallen, E.; Soderstrom-Anttila, V.; Kahila, H.; Hyden-Granskog, C.; Tuuri, T.; Tiitinen, A.; Kaminen-Ahola, N.\nScore: 2.5, Published: 2023-10-27 DOI: 10.1101/2023.10.25.23297514\nAssisted reproductive technology (ART) has been associated with increased risk for growth disturbance and imprinting disorders, but the molecular mechanisms and whether they are a result of the ART procedures or the underlying subfertility are unknown. Here we performed genome-wide DNA methylation analysis by EPIC Illumina microarrays and gene expression analysis by mRNA sequencing for a total of 80 ART and 77 control placentas, including separate procedure- and sex-specific analyses. ART-associated changes enriched in the pathways of hormonal regulation, insulin resistance, neuronal development, and vascularization. Observed changes in the number of stromal cells as well as TRIM28 and NOTCH3 expressions in ART placentas indicated impaired angiogenesis and growth. The enrichment of DNA methylation changes in the imprinted regions and alterations in TRIM28, ZFP57, and NLRP5 suggested defective stabilization of the imprinting. Furthermore, downregulated expression of imprinted endocrine signaling molecule DLK1, associated with both ART and subfertility, provides a potential mechanism for the metabolic and phenotypic features associated with ART.\n",
  "wordCount" : "3012",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.23297414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.23297414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.23297414">
        <p class="paperTitle">Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.23297414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.23297414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, W.; Eriksson, N.; McIntyre, M.; Bagur Quetglas, R.; Koelsch, B. L.; Hinds, D. A.; Aslibekyan, S.; Auton, A.; Holmes, M. V.; Shringarpure, S. S.</p>
        <p class="info">Score: 35.3, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.23297414' target='https://doi.org/10.1101/2023.10.23.23297414'> 10.1101/2023.10.23.23297414</a></p>
        <p class="abstract">BackgroundHuman genetics provides opportunities for enhancing disease prediction through polygenic risk scores (PRS).

MethodWe used a dataset from 23andMe (6.77M European, 1.30M Latine, and 0.45M African American individuals). Using cross-sectional data for PRS construction and a prospective cohort for evaluation, we estimated PRS-associated cumulative incidences after one year of follow-up for 12 clinical endpoints.

ResultsThe cumulative incidence of disease at one year was consistently higher among individuals in the top 10% of each PRS. Relative risks (RRs) comparing top to bottom 10% of PRS differed across diseases (e.g. European RR 2.12 for eczema vs 12.53 for T2D). Estimates were similar between Europeans and Latines however were more modest for African Americans (e.g. T2D RR 10.92 for Latines vs. 4.00 for African Americans). Clinical manifestation occurred earlier among those in top vs bottom 10% of polygenic risk: 16yrs for hypertension, and 9.5yrs for T2D. Among participants at elevated conventional risk of CHD or T2D, those in the top 10% PRS had a 10-20 fold higher RR of disease incidence vs those not at conventional risk. Among individuals at high polygenic risk of CHD or T2D, favorable lifestyle characteristics associated with 64-73% lower RR of developing disease over 1-year, with cumulative incidence equivalent to the population average.

ConclusionIn an ancestrally-diverse cohort, individuals in the top 10% PRS had higher 1-year disease incidence and earlier age of clinical manifestation. PRS provided risk stratification beyond conventional risk factors. Lifestyle characteristics markedly lowered disease incidence among those at elevated polygenic risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.23297096">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.23297096" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.23297096">
        <p class="paperTitle">Decoding Genetics, Ancestry, and Geospatial Context for Precision Health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.23297096" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.23297096" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koyama, S.; Wang, Y.; Paruchuri, K.; Uddin, M. M.; Cho, S. M. J.; Urbut, S. M.; Haidermota, S.; Hornsby, W. E.; Green, R. C.; Daly, M. J.; Neale, B. M.; Ellinor, P. T.; Smoller, J. W.; Lebo, M. S.; Karlson, E. W.; Martin, A. R.; Natarajan, P.</p>
        <p class="info">Score: 32.2, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.23297096' target='https://doi.org/10.1101/2023.10.24.23297096'> 10.1101/2023.10.24.23297096</a></p>
        <p class="abstract">Mass General Brigham, an integrated healthcare system based in the Greater Boston area of Massachusetts, annually serves 1.5 million patients. We established the Mass General Brigham Biobank (MGBB), encompassing 142,238 participants, to unravel the intricate relationships among genomic profiles, environmental context, and disease manifestations within clinical practice. In this study, we highlight the impact of ancestral diversity in the MGBB by employing population genetics, geospatial assessment, and association analyses of rare and common genetic variants. The population structures captured by the genetics mirror the sequential immigration to the Greater Boston area throughout American history, highlighting communities tied to shared genetic and environmental factors. Our investigation underscores the potency of unbiased, large-scale analyses in a healthcare-affiliated biobank, elucidating the dynamic interplay across genetics, immigration, structural geospatial factors, and health outcomes in one of the earliest American sites of European colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.23297558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.23297558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.23297558">
        <p class="paperTitle">Plasticity of Human Microglia and Brain Perivascular Macrophages in Aging and Alzheimer&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.23297558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.23297558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, D.; Porras, C.; Spencer, C.; Pjanic, M.; Weiler, P.; Kosoy, R.; Bendl, J.; N.M., P.; Wang, X.; Zheng, S.; Therrien, K.; Mathur, D.; Kleopoulos, S. P.; Shao, Z.; Argyriou, S.; Alvia, M.; Casey, C.; Hong, A.; Beaumont, K. G.; Sebra, R.; Kellner, C. P.; Bennett, D. A.; Yuan, G.-C.; Voloudakis, G.; Theis, F. J.; Haroutunian, V.; Hoffman, G. E.; Fullard, J. F.; Roussos, P.</p>
        <p class="info">Score: 12.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.23297558' target='https://doi.org/10.1101/2023.10.25.23297558'> 10.1101/2023.10.25.23297558</a></p>
        <p class="abstract">Microglia and perivascular macrophages, myeloid-origin resident immune cells in the human brain, play crucial roles in Alzheimers disease (AD)1-4. However, the field lacks a unified taxonomy describing their heterogeneity and plasticity5. To address this, we applied single-cell profiling to two independent, demographically diverse cohorts. The first comprises 543,012 viable myeloid cells from 137 unique postmortem brain specimens, while the second consists of 289,493 myeloid nuclei from 1,470 donors. Collectively, they cover the human lifespan and varying degrees of AD neuropathology. We identify 13 transcriptionally distinct myeloid subtypes, including the &#34;GPNMB&#34; subtype that proliferates with AD. We distinguish two contrasting homeostatic microglial states in AD and with aging: the first (&#34;FRMD4A&#34;) wanes over time, while the second (&#34;PICALM&#34;) becomes more prevalent. By prioritizing AD-risk genes, including PTPRG, DPYD, and IL15, and placing them into a regulatory hierarchy, we identify common upstream transcriptional regulators, namely MITF and KLF12, that regulate the expression of AD-risk genes in the opposite directions. Through the construction of cell-to-cell interaction networks, we identify candidate ligand-receptor pairs, including APOE:SORL1 and APOE:TREM2, associated with AD progression. We show polygenic risk for AD predisposes and prioritize the GPNMB subtype as a therapeutic target of early intervention. Our findings delineate the relationship between distinct functional states of myeloid cells and their pathophysiological response to aging and AD, providing a significant step toward the mechanistic understanding of the roles of microglia in AD and the identification of novel therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297507">
        <p class="paperTitle">CXCL12 regulates coronary artery dominance in diverse populations and links development to disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.</p>
        <p class="info">Score: 10.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297507' target='https://doi.org/10.1101/2023.10.27.23297507'> 10.1101/2023.10.27.23297507</a></p>
        <p class="abstract">Mammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297200">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297200" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297200">
        <p class="paperTitle">DNAm scores for serum GDF15 and NT-proBNP levels associate with a range of traits affecting the body and brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297200" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297200" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gadd, D. A.; Smith, H.; Mullin, D.; Chybowska, A.; Hillary, R. F.; Kimenai, D. M.; Bernabeu, E.; Cheng, Y.; Fawns-Ritchie, C.; Campbell, A.; Page, D.; Taylor, A.; Corley, J.; Valdes Hernandez, M. D. C.; Munoz Maniega, S.; Bastin, M. E.; Wardlaw, J. M.; Walker, R. M.; Evans, K. L.; McIntosh, A. M.; Hayward, C.; Russ, T. C.; Harris, S. E.; Welsh, P.; Sattar, N.; Cox, S. R.; McCartney, D. L.; Marioni, R. E.</p>
        <p class="info">Score: 9.8, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297200' target='https://doi.org/10.1101/2023.10.18.23297200'> 10.1101/2023.10.18.23297200</a></p>
        <p class="abstract">BackgroundPlasma growth differentiation factor 15 (GDF15) and N-terminal pro-B-type natriuretic peptide (NT-proBNP) are cardiovascular biomarkers that associate with a range of diseases. Epigenetic scores (EpiScores) for GDF15 and NT-proBNP may provide new routes for risk stratification.

ResultsIn the Generation Scotland cohort (N [&amp;ge;] 16,963), GDF15 levels were associated with incident dementia, ischaemic stroke and type 2 diabetes, whereas NT-proBNP levels were associated with incident ischaemic heart disease, ischaemic stroke and type 2 diabetes (all PFDR &lt; 0.05). Bayesian Epigenome-wide association studies (EWAS) identified 12 and 4 DNA methylation (DNAm) CpG sites associated (Posterior Inclusion Probability [PIP] &gt; 95%) with levels of GDF15 and NT-proBNP, respectively. EpiScores for GDF15 and NT-proBNP that were trained in a subset of the population. The GDF15 EpiScore replicated protein associations with incident dementia, type 2 diabetes and ischaemic stroke in the Generation Scotland test set (Hazard Ratios (HR) range 1.36 - 1.41, PFDR &lt;0.03). The EpiScore for NT-proBNP replicated the protein association with type 2 diabetes, but failed to replicate an association with ischaemic stroke. EpiScores explained comparable variance in protein levels across both the Generation Scotland test set and the external LBC1936 test cohort (R2 range of 5.7-12.2%). In LBC1936, both EpiScores were associated with indicators of poorer brain health. Neither EpiScore was associated with incident dementia in the LBC1936 population.

ConclusionsEpiScores for serum levels of GDF15 and Nt-proBNP associate with body and brain health traits. These EpiScores are provided as potential tools for disease risk stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.23297618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.23297618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.23297618">
        <p class="paperTitle">Mendelian randomization study of birth weight and risk of psychiatric disorders later in life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.23297618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.23297618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, X.; Geller, F.; Helenius, D.; Grove, J.; Albinana, C.; Petersen, L. V.; Bulik, C.; Borglum, A.; Werge, T.; Vilhjalmsson, B. J.; Feenstra, B.</p>
        <p class="info">Score: 5.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.23297618' target='https://doi.org/10.1101/2023.10.26.23297618'> 10.1101/2023.10.26.23297618</a></p>
        <p class="abstract">Low birth weight has been associated with a higher risk of psychiatric disorders later in life. The underlying causal mechanisms of this relationship are however not clear. In this study, we investigate whether variation in fetal growth has a direct causal effect on mental health. Using birth weight as a proxy measure for fetal growth, we first assessed associations between observed birth weight and later diagnosis of psychiatric disorders in the Danish iPSYCH and ANGI-DK cohorts. Next, we constructed a polygenic score for birth weight based on &gt;1 million variants and tested for association with psychiatric disorders. Finally, using 86 single-nucleotide polymorphisms with robust fetal-only genetic associations with birth weight, we assessed the causal relationship of genetically mediated fetal growth and psychiatric disorders using Mendelian randomization analyses. We found that higher observed birth weight was associated with lower risk of several psychiatric disorders. Polygenic score analyses supported this pattern for attention deficit/hyperactivity disorder, where an increase of one standard deviation in the score for birth weight corresponded to an odds ratio of 0.85 (95% confidence interval 0.79-0.92, P=6.27x10-5). However, one- and two-sample Mendelian randomization analyses did not indicate a direct causal relationship between the birth weight of children and their risk of psychiatric disorders. In conclusion, our study does not support a direct causal effect of fetal growth (as proxied by birth weight) on the risk of psychiatric disorders later in life, suggesting that the observed association between birth weight and mental health is likely to be caused by other factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.23297129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.23297129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.23297129">
        <p class="paperTitle">Biallelic variants in HMGCS1 are a novel cause of rare rigid spine syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.23297129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.23297129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dofash, L. N. H.; Miles, L. B.; Saito, Y.; Rivas, E.; Calcinotto, V.; Oveissi, S.; Serrano, R. J.; Templin, R.; Ramm, G.; Rodger, A.; Haywood, J.; Ingley, E.; Clayton, J. S.; Taylor, R. L.; Folland, C. L.; Groth, D.; Hock, D. H.; Stroud, D. A.; Gorokhova, S.; Donkervoort, S.; Bönnemann, C. G.; Sud, M.; VanNoy, G.; Mangilog, B. E.; Pais, L.; Madruga-Garrido, M.; Scala, M.; Fiorillo, C.; Baratto, S.; Traverso, M.; Bruno, C.; Zara, F.; Paradas, C.; Ogata, K.; Nishino, I.; Laing, N. G.; Bryson-Richardson, R. J.; Cabrera-Serrano, M.; Ravenscroft, G.</p>
        <p class="info">Score: 4.6, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.23297129' target='https://doi.org/10.1101/2023.10.25.23297129'> 10.1101/2023.10.25.23297129</a></p>
        <p class="abstract">Rigid spine syndrome is a rare childhood-onset myopathy characterised by slowly progressive or non-progressive scoliosis, neck and spine contractures, hypotonia, and respiratory insufficiency. Biallelic variants in SELENON account for most cases of rigid spine syndrome, however, the underlying genetic cause in some patients remains unexplained. In this study, we used exome and genome sequencing to investigate the genetic basis of rigid spine syndrome in patients without a genetic diagnosis. In five patients from four unrelated families, we identified biallelic variants in HMGCS1 (3-hydroxy-3-methylglutaryl-coenzyme A synthase). These included six missense variants and one frameshift variant distributed throughout HMGCS1. All patients presented with spinal rigidity primarily affecting the cervical and dorsolumbar regions, scoliosis, and respiratory insufficiency. Creatine kinase levels were variably elevated. The clinical course worsened with intercurrent disease or certain drugs in some patients; one patient died from respiratory failure following infection. Muscle biopsies revealed irregularities in oxidative enzyme staining with occasional internal nuclei and rimmed vacuoles. HMGCS1 encodes a key enzyme of the mevalonate pathway, disturbance of which is also associated with HMGCR-limb girdle muscular dystrophy and GGPS1-muscular dystrophy. Quantitative PCR and western blotting confirmed HMGCS1 abundance in skeletal muscle and myogenic precursors. HMGCS1 levels in skeletal muscle were comparable between healthy controls and the index case with a homozygous p{middle dot}(S447P) substitution. Muscle RNA-seq for a patient with a frameshift variant (c{middle dot}344_345del:p{middle dot}S115Wfs*12) and an in trans substitution (p{middle dot}(Q29L)) showed HMGCS1 transcript levels reduced to 53% compared to controls. The substitution appeared homozygous on RNA-seq, suggesting the allele harbouring the frameshift variant undergoes nonsense mediated decay. hmgcs1 null zebrafish displayed severe early defects, including immobility at 2 days and death by days 3 post-fertilisation. We anticipate that the variants observed in this cohort have subtle effects on HMGCS1 function given most patients survived to adulthood. In support of the variants being hypomorphic, analyses of recombinant human HMGCS1 protein and four mutants (p{middle dot}S447P, p{middle dot}Q29L, p{middle dot}M70T, p{middle dot}C268S) showed all mutants maintained their secondary structure and dimerized state and had enzymatic activity comparable to the wildtype. Thermal stability of the mutants was similar or slightly reduced compared to the wildtype. Altogether, our analyses suggest that the identified missense variants in HMGCS1 act through a hypomorphic mechanism yet to be elucidated. Here, we report an additional component of the mevalonate pathway associated with myopathy and suggest biallelic variants in HMGCS1 should be considered in patients presenting with an unresolved rigid spine phenotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297676">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297676" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297676">
        <p class="paperTitle">Genome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves cross-ancestry prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297676" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297676" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, T. J.; Graff, R. E.; Madduri, R. K.; Rodriguez, A. A.; Cario, C. L.; Feng, K.; Jiang, Y.; Wang, A.; Klein, R. J.; Pierce, B. L.; Eggener, S.; Tong, L.; Blot, W.; Long, J.; Rebbeck, T.; Lachance, J.; Andrews, C.; Adebiyi, A. O.; Adusei, B.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Jalloh, M.; Janivara, R.; Chen, W. C.; Mensah, J. E.; Agalliu, I.; Berndt, S. I.; Shelley, J. P.; Schaffer, K.; Machiela, M. J.; Freedman, N. D.; Huang, W.-Y.; Li, S. A.; Goodman, P. J.; Till, C.; Thompson, I.; Lilja, H.; Van Den Eeden, S. K.; Chanock, S. J.; Mosley, J. D.; Conti, D. V.; Haiman, C. A.;</p>
        <p class="info">Score: 4.5, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297676' target='https://doi.org/10.1101/2023.10.27.23297676'> 10.1101/2023.10.27.23297676</a></p>
        <p class="abstract">We conducted a multi-ancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry; 58,236 African ancestry; 23,546 Hispanic/Latino; 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (p[&amp;le;]5e-8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n=95,768). Meta-analyzing discovery and replication (n=392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our new polygenic risk score reached 16.9% (95% CI=16.1%-17.8%) in European ancestry, 9.5% (95% CI=7.0%-12.2%) in African ancestry, 18.6% (95% CI=15.8%-21.4%) in Hispanic/Latino, and 15.3% (95% CI=12.7%-18.1%) in Asian ancestry, and lower for higher age. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, with potential to personalize prostate cancer screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297280">
        <p class="paperTitle">Determinants of mosaic chromosomal alteration fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pershad, Y.; Mack, T.; Poisner, H. M.; Jakubek, Y. A.; Stilp, A. M.; Mitchell, B. D.; Lewis, J. P.; Boerwinkle, E.; Loos, R. J.; Chami, N.; Wang, Z.; Barnes, K.; Pankratz, N.; Fornage, M.; Redline, S.; Psaty, B. M.; Bis, J. C.; Shojaie, A.; Silverman, E. K.; Cho, M. H.; Yun, J.; DeMeo, D.; Levy, D.; Johnson, A.; Mathias, R.; Taub, M.; Arnett, D.; North, K.; Raffield, L. M.; Carlson, A.; Doyle, M. F.; Rich, S. S.; Rotter, J. I.; Guo, X.; Cox, N.; Roden, D. M.; Franceschini, N.; Desai, P.; Reiner, A.; Auer, P. L.; Scheet, P.; Jaiswal, S.; Weinstock, J. S.; Bick, A. G.</p>
        <p class="info">Score: 14.8, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297280' target='https://doi.org/10.1101/2023.10.20.23297280'> 10.1101/2023.10.20.23297280</a></p>
        <p class="abstract">Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal Expansion Rate (PACER) method to estimate clonal expansion rate for 6,381 individuals in the NHLBI TOPMed cohort with gain, loss, and copy-neutral loss of heterozygosity mCAs. Our estimates of mCA fitness were correlated (R2 = 0.49) with an alternative approach that estimated fitness of mCAs in the UK Biobank using a theoretical probability distribution. Individuals with lymphoid-associated mCAs had a significantly higher white blood cell count and faster clonal expansion rate. In a cross-sectional analysis, genome-wide association study of estimates of mCA expansion rate identified TCL1A, NRIP1, and TERT locus variants as modulators of mCA clonal expansion rate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.23297514">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.23297514" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.23297514">
        <p class="paperTitle">Genome-wide DNA methylation, imprinting, and gene expression in human placentas derived from Assisted Reproductive Technology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.23297514" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.23297514" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Auvinen, P.; Vehvilainen, J.; Ramo, K.; Laukkanen, I.; Marjonen-Lindblad, H.; Wallen, E.; Soderstrom-Anttila, V.; Kahila, H.; Hyden-Granskog, C.; Tuuri, T.; Tiitinen, A.; Kaminen-Ahola, N.</p>
        <p class="info">Score: 2.5, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.23297514' target='https://doi.org/10.1101/2023.10.25.23297514'> 10.1101/2023.10.25.23297514</a></p>
        <p class="abstract">Assisted reproductive technology (ART) has been associated with increased risk for growth disturbance and imprinting disorders, but the molecular mechanisms and whether they are a result of the ART procedures or the underlying subfertility are unknown. Here we performed genome-wide DNA methylation analysis by EPIC Illumina microarrays and gene expression analysis by mRNA sequencing for a total of 80 ART and 77 control placentas, including separate procedure- and sex-specific analyses. ART-associated changes enriched in the pathways of hormonal regulation, insulin resistance, neuronal development, and vascularization. Observed changes in the number of stromal cells as well as TRIM28 and NOTCH3 expressions in ART placentas indicated impaired angiogenesis and growth. The enrichment of DNA methylation changes in the imprinted regions and alterations in TRIM28, ZFP57, and NLRP5 suggested defective stabilization of the imprinting. Furthermore, downregulated expression of imprinted endocrine signaling molecule DLK1, associated with both ART and subfertility, provides a potential mechanism for the metabolic and phenotypic features associated with ART.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
