<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders
Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders
Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders
Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders\nAuthors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.",
  "keywords": [
    
  ],
  "articleBody": " Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders\nAuthors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.; Gardham, A.; Ghali, N.; Green, A.; Hanna, W.; Harrison, R.; Hegarty, M.; Higgs, J.; Holder, M.; Irving, R.; Jain, V.; Johnson, K.; Jolley, R.; Jones, W.; Jones, G.; Joss, S.; Kalinauskiene, R.; Kanini, F.; Kavanagh, K.; Khan, M.; Khan, N.; Kivuva, E.; Lahiri, N.; L\nScore: 17.8, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297202\nObjective: We sought to evaluate outcomes for clinical management following a genetic diagnosis from the Deciphering Developmental Disorders (DDD) Study. Design: Individuals in the DDD study who had a pathogenic/likely pathogenic genotype in the DECIPHER database were selected for inclusion (n=5010). Clinical notes from regional clinical genetics services notes were reviewed to assess pre-defined clinical outcomes relating to interventions, prenatal choices, and information provision. Results: Outcomes were recorded for 4237 diagnosed probands (85% of those eligible) from all 24 recruiting centres across the UK and Ireland. Additional diagnostic or screening tests were performed in 903 (21%) probands through referral to a range of different clinical specialties, and stopped or avoided in a further 26 (0.6%). Disease-specific treatment was started in 85 (2%) probands, including seizure-control medications and dietary supplements, and contra-indicated medications were stopped/avoided as no longer necessary in a further 20 (0.5%). The option of prenatal/preimplantation genetic testing was discussed with 1204 (28%) families, despite the relatively advanced age of the parents at the time of diagnosis. Importantly, condition-specific information or literature was given to 3214 (76%) families, and 880 (21%) were involved in family support groups. In the most common condition (KBG syndrome; 79 (2%) probands), clinical interventions only partially reflected the temporal development of phenotypes, highlighting the importance of consensus management guidelines and patient support groups. Conclusions: Our results underscore the importance of achieving a clinico-molecular diagnosis to ensure timely onward referral of patients, enabling appropriate care and anticipatory surveillance, and for accessing relevant patient support groups.\nPolygenic Embryo Screening: High Approval Despite Substantial Concerns from the U.S. Public\nAuthors: Furrer, R. A.; Barlevy, D.; Pereira, S.; Carmi, S.; Lencz, T.; Lazaro-Munoz, G.\nScore: 12.0, Published: 2023-10-14 DOI: 10.1101/2023.10.13.23297022\nPolygenic embryo screening (PES) is a novel - yet commercially available - technology that can compute genetic likelihood estimates of polygenic conditions (e.g., diabetes, depression) and traits (e.g., height, cognitive ability) in embryos. Patients undergoing in vitro fertilization (IVF) can use these polygenic scores to select which embryos to transfer for implantation with the goal of having a child. We conducted a survey of the U.S. public to examine attitudes toward PES encompassing acceptance, interest, potential uses, and concerns (n=1435). Our results indicate 72% public approval for PES, with 82% expressing some interest in using PES if already undergoing IVF. Approval for using PES for embryo selection is notably high for physical health (77%) and psychiatric conditions (72%). In contrast, there is minority approval for embryo selection based on PES for behavioral traits (36%) and physical traits (30%). Nevertheless, concerns about PES leading to false expectations, eugenic practices, and stigma are pronounced (54-55% find them \"very\" to \"extremely\" concerning). In a second sample of participants (n=192), presenting concerns at survey onset (vs. end) reduced approval (-28%) by mostly increasing ambivalence (+24%), and only slightly increasing disapproval (+4%). Given its commercial availability, practical limitations, and ethical concerns among physicians, patients, geneticists, bioethicists, and legal scholars, it is notable that there is such high public approval and interest in using PES. Understanding these attitudes is essential for informing policymakers, healthcare professionals, and researchers about the publics perspectives on this novel biotechnology and debate about the role of medicine in regulating the use of PES.\nDisentangling the link between maternal influences on birth weight and disease risk in 36,211 genotyped mother-child pairs\nAuthors: Leinonen, J. T.; FinnGen, ; Pirinen, M.; Tukiainen, T.\nScore: 9.0, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297154\nEpidemiological studies have robustly linked lower birth weight to later-life disease risks. These observations may reflect the adverse impact of intrauterine growth restriction on a childs health. However, causal evidence supporting such a mechanism in humans is largely lacking. Using Mendelian Randomization and 36,211 genotyped mother-child pairs from the FinnGen study, we assessed the relationship between intrauterine growth restriction and five common health outcomes (coronary heart disease (CHD), hypertension, statin use, type 2 diabetes and cancer). We proxied intrauterine growth restriction with polygenic scores for maternal effects on birth weight and took into account transmission of genetic variants between a mother and a child in the analyses. We find limited evidence for contribution of normal variation in maternally influenced intrauterine growth on later-life disease. Instead, we find support for genetic pleiotropy in the child genome linking birth weight to CHD and hypertension. Our study illustrates the opportunities that data from genotyped parent-child pairs from a population-based biobank provides for addressing causality of maternal influences.\nBidirectional relationship between olfaction and Parkinson's disease\nAuthors: Kim, J. J.; Bandres-Ciga, S.; Heilbron, K.; 23andMe Research Team, ; Blauwendraat, C.; Noyce, A. J.\nScore: 6.1, Published: 2023-10-19 DOI: 10.1101/2023.10.18.23297218\nIntroduction Hyposmia (loss of smell) is a common early symptom of Parkinson's disease (PD). The shared genetic architecture between hyposmia and PD is unknown. Objectives To assess the shared genetic architecture and causal relationship between hyposmia and PD. Methods We leveraged genome-wide association study (GWAS) results for self-assessment of 'ability to smell' and PD diagnosis. Linkage disequilibrium score regression (LDSC) and Local Analysis of [co]Variant Association (LAVA) were used to identify genome-wide and local genetic correlations. Mendelian randomization was used to identify potential causal relationships. Results LDSC found that sense of smell negatively correlated at a genome-wide level with PD. LAVA found negative correlations in four genetic loci near GBA1, ANAPC4, SNCA, and MAPT. Using Mendelian randomization we found evidence for strong causal relationship between PD and liability towards poorer sense of smell, but weaker evidence for the reverse direction. Conclusions Hyposmia and PD share genetic liability in only a subset of the major PD risk genes. While there was definitive evidence that PD can lower the sense of smell, there was only suggestive evidence for the reverse. This work highlights the heritability of olfactory function and its relationship with PD heritability and provides further insight into the association between PD and hyposmia.\nCould Metformin use reduce abdominal aortic aneurysm risk? A Mendelian randomisation study using known Metformin targets\nAuthors: Saxby, K. L.; Dudbridge, F.; Samani, N. J.; Bown, M. J.; Nelson, C. P.\nScore: 4.8, Published: 2023-10-16 DOI: 10.1101/2023.10.13.23293381\nIntroductionAn abdominal aortic aneurysm (AAA) is a swelling of the main artery in the body estimated to affect 0.92% of adults (aged 30-79) worldwide. Rupture is often fatal and surgical intervention may be offered if the risk of rupture is high. There is no treatment to prevent AAA or to slow aneurysm growth aside from dietary and lifestyle recommendations. Metformin, a drug prescribed to treat type 2 diabetes, has previously been associated with a potential reduction in AAA disease risk but no causal link has been shown. Here we investigate the causal link between Metformin and AAA risk through Mendelian randomisation (MR). MethodsWe conducted a two-sample MR analysis using genetic variants associated with gene expression of five Metformin drug targets that also show a genetic association with decreased glycated haemoglobin (HbA1c) levels. Effect sizes are obtained from within UK Biobank for HbA1c, and within AAAgen for AAA risk, a multi-ancestry meta-GWAS analysis of 39,221 cases and 1,086,107 controls. ResultsWe identified statistically significant evidence of a causal association between a genetic proxy for Metformin action and a decrease in AAA risk, OR=0.58 (95%CI: 0.37-0.90 p=0.015). We estimate that on average a one standard deviation decrease in HbA1c, measured via Metformin gene targets, reduces AAA risk by over 40%. ConclusionMetformin use in those at increased risk of AAA may reduce incidence of disease. Clinical trials are required to assess the efficacy of Metformin in reducing disease risk.\niSCORED: nanopore-based random genomic sampling for intraoperative molecular diagnosis\nAuthors: Emiliani, F.; Ould Ismail, A. A.; Hughes, E.; Tsongalis, G.; Zanazzi, G.; Lin, C.-C.\nScore: 4.5, Published: 2023-10-18 DOI: 10.1101/2023.10.17.23297170\nCopy number variations (CNVs) are almost ubiquitous in cancer. In many cases, somatic CNV analysis has led to the identification of oncogenic pathways and suggested molecular-defined therapeutic targets. Here, we develop iSCORED, a one-step random genomic DNA reconstruction method that enables efficient, unbiased quantification of CNVs using a real-time Nanopore sequencer. By leveraging the long concatenated reads, we generate approximately 1-2 million genomic fragments within one hour of MinION sequencing, allowing for high-resolution genomic dosage comparisons. In our cohort of 26 malignant brain tumors, we demonstrated 100% concordance in CNV detections, including chromosomal alterations and oncogene amplifications when compared to clinically validated next generation sequencing and chromosomal microarray results. In addition, iSCORED allows concurrent brain tumor methylation classification without additional tissue preparation. The integrated methylation information revealed promoter hypomethylation in all detected amplified oncogenes. The entire workflow, including the automatic generation of CNV and methylation reports, can be accomplished within 120-140 minutes. Ultrafast molecular analysis can enhance clinical decision-making, optimize surgical planning and identify potential molecular therapies within surgical timeframes.\nNanoRanger enables rapid single base-pair resolution of genomic disorders\nAuthors: Zhang, Y.; Bi, C.; Nadeef, S. S.; Maddirevula, S.; Alqahtani, M.; Alkuraya, F.; Li, M.\nScore: 2.7, Published: 2023-10-15 DOI: 10.1101/2023.10.14.23296971\nRare diseases affect around 350 million individuals globally, yet at least half of those with suspected Mendelian disorders remain without a precise molecular diagnosis despite advanced genetic testing using short read sequencing (SRS). Long-read sequencing (LRS) holds a promise in addressing this diagnostic gap although its clinical application is hampered by its complicated workflow, demanding sample requirements, and exorbitant cost. Genomic disorders represent an opportunity to demonstrate the unique capability of LRS. This study addresses the challenges of identifying missing disease-causing breakpoints in complex genomic disorders by employing multiple LRS strategies, including a novel strategy named nanopore-based rapid acquisition of neighboring genomic regions (NanoRanger). NanoRanger requires neither detailed prior genetic mapping nor large amounts of ultrahigh-molecular-weight DNA, and it stands out for its ease of use and ultra-rapid acquisition of large genomic regions of interest with deep coverage. We describe a cohort of 13 families, each harboring a different homozygous disease-causing genomic rearrangement that defied breakpoint determination by SRS and Optical Genome Mapping (OGM). In each case, NanoRanger identified the breakpoints with a single base-pair resolution. This has enabled the accurate determination of the carrier status of unaffected family members as well as the founder nature of these genomic lesions and their frequency in the local population. It has also enabled an unprecedented analysis of the DNA motifs to discern the mechanism that predisposed to these recessive rearrangements. Our data suggest that NanoRanger can greatly accelerate the clinical adoption of LRS and expand its access for the benefit of patients with rare diseases.\nPKHD1L1, A Gene Involved in the Stereociliary Coat, Causes Autosomal Recessive Nonsyndromic Hearing Loss\nAuthors: Redfield, S. E.; De-la-Torre, P.; Zamani, M.; Khan, H.; Morris, T.; Shariati, G.; Karimi, M.; Kenna, M. A.; Seo, G. H.; Naz, S.; Galehdari, H.; Indzhykulian, A. A.; Shearer, A. E.; Vona, B.\nScore: 9.3, Published: 2023-10-09 DOI: 10.1101/2023.10.08.23296081\nIdentification of genes associated with nonsyndromic hearing loss is a crucial endeavor, given the substantial number of individuals who remain without a diagnosis after even the most advanced genetic testing. PKHD1L1 was established as necessary for the formation of the cochlear hair-cell stereociliary coat and causes hearing loss in mice and zebrafish when mutated. We sought to determine if biallelic variants in PKHD1L1 also cause hearing loss in humans. Exome sequencing was performed on DNA of three families segregating autosomal recessive moderate to severe nonsyndromic sensorineural hearing loss. Compound heterozygous missense p.[(Gly129Ser)];p.[(Gly1314Val)], homozygous missense p.(His2479Gln) and nonsense p.(Arg3381Ter) variants were identified in PKHD1L1 that were predicted to be damaging using in silico pathogenicity prediction methods. In vitro functional analysis of two missense variants was performed using purified recombinant PKHD1L1 protein fragments. We then evaluated protein thermodynamic stability with and without the missense variants found in one of the families. In vitro functional assessment indicated that both engineered PKHD1L1 mutant p.(Gly129Ser) and p.(Gly1314Val) constructs significantly reduced the folding and structural stabilities of the expressed protein fragments, providing further evidence to support pathogenicity of these variants. In silico molecular modelling using AlphaFold2 and protein sequence alignment analysis were carried out to further explore potential variant effects on protein folding and stability and exposed key structural features that might suggest PKHD1L1 protein destabilization. Multiple lines of evidence collectively associate PKHD1L1 with nonsyndromic mild-moderate to severe sensorineural hearing loss. PKHD1L1 testing in individuals with mild-moderate hearing loss may identify further affected families.\nPheNorm, a language model normalizer of physical examinations from genetics clinical notes\nAuthors: Weissenbacher, D.; Rawal, S.; Zhao, X.; Priestley, J. R. C.; Szigety, K. M.; Schmidt, S. F.; Higgins, M. J.; Magge, A.; O'Connor, K.; Gonzalez-Hernandez, G.; Campbell, I. M.\nScore: 2.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.23296894\nBackgroundPhenotypes identified during dysmorphology physical examinations are critical to genetic diagnosis and nearly universally documented as free-text in the electronic health record (EHR). Variation in how phenotypes are recorded in free-text makes large-scale computational analysis extremely challenging. Existing natural language processing (NLP) approaches to address phenotype extraction are trained largely on the biomedical literature or on case vignettes rather than actual EHR data. MethodsWe implemented a tailored system at the Childrens Hospital of Philadelpia that allows clinicians to document dysmorphology physical exam findings. From the underlying data, we manually annotated a corpus of 3136 organ system observations using the Human Phenotype Ontology (HPO). We provide this corpus publicly. We trained a transformer based NLP system to identify HPO terms from exam observations. The pipeline includes an extractor, which identifies tokens in the sentence expected to contain an HPO term, and a normalizer, which uses those tokens together with the original observation to determine the specific term mentioned. FindingsWe find that our labeler and normalizer NLP pipeline, which we call PheNorm, achieves state-of-the-art performance for the dysmorphology physical exam phenotype extraction task. PheNorms performance on the test set was 0.717, compared to the nearest baseline system (PhenoTagger) performance of 0.633. An analysis of our systems normalization errors shows possible imperfections in the HPO terminology itself but also reveals a lack of semantic understanding by our transformer models. InterpretationTransformers-based NLP models are a promising approach to genetic phenotype extraction and, with recent development of larger pre-trained causal language models, may improve semantic understanding in the future. We believe our results also have direct applicability to more general extraction of medical signs and symptoms. FundingUS National Institutes of Health\nGenetic predisposition to altered blood cell homeostasis is associated with glioma risk and survival\nAuthors: Kachuri, L.; Guerra, G. A.; Wendt, G. A.; Hansen, H. M.; Molinaro, A.; Bracci, P.; McCoy, L.; Rice, T.; Wiencke, J. K.; Eckel- Passow, J. E.; Jenkins, R. B.; Wrensch, M.; Francis, S. S.\nScore: 2.1, Published: 2023-10-16 DOI: 10.1101/2023.10.15.23296448\nGlioma is a highly fatal brain tumor comprised of molecular subtypes with distinct clinical trajectories. Observational studies have suggested that variability in immune response may play a role in glioma etiology. However, their findings have been inconsistent and susceptible to reverse causation due to treatment effects and the immunosuppressive nature of glioma. We applied genetic variants associated (p\u003c5x10-8) with blood cell traits to a meta-analysis of 3418 glioma cases and 8156 controls. Genetically predicted increase in the platelet to lymphocyte ratio (PLR) was associated with an increased risk of glioma (odds ratio (OR)=1.25, p=0.005), especially in IDH-mutant (IDHmut OR=1.38, p=0.007) and IDHmut 1p/19q non-codeleted (IDHmut-noncodel OR=1.53, p=0.004) tumors. However, reduced glioma risk was observed for higher counts of lymphocytes (IDHmut-noncodel OR=0.70, p=0.004) and neutrophils (IDHmut OR=0.69, p=0.019; IDHmut-noncodel OR=0.60, p=0.009), which may reflect genetic predisposition to enhanced immune-surveillance. In contrast to susceptibility, there was no association with survival in IDHmut-noncodel; however, in IDHmut 1p/19q co-deleted tumors, we observed higher mortality with increasing genetically predicted counts of lymphocytes (hazard ratio (HR)=1.65, 95% CI: 1.24-2.20), neutrophils (HR=1.49, 1.13-1.97), and eosinophils (HR=1.59, 1.18-2.14). Polygenic scores for blood cell traits were also associated with tumor immune microenvironment features, with heterogeneity by IDH status observed for 17 signatures related to interferon signaling, PD-1 expression, and T-cell/Cytotoxic responses. In summary, we identified novel, immune-mediated susceptibility mechanisms for glioma with potential disease management implications. SIGNIFICANCEThis study suggests that genetic determinants of peripheral blood cell counts influence subtype-specific glioma susceptibility and mortality, with potential effects on the tumor immune microenvironment. These findings may have disease management implications.\n",
  "wordCount" : "2780",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297202">
        <p class="paperTitle">Large-scale evaluation of outcomes following a genetic diagnosis in children with severe developmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Copeland, H.; Low, K. J.; Wynn, S.; Ahmed, A.; Arthur, V.; Balasubramanian, M.; Bennett, K.; Berg, J.; Bertoli, M.; Bryson, L.; Bucknall, C.; Campbell, J.; Chandler, K.; Chauhan, J.; Clarkson, A.; Coles, R.; Conti, H.; Costello, P.; Coupar, T.; Craig, A.; Dean, J.; Dillon, A.; Dixit, A.; Drew, K.; Eason, J.; Forzano, F.; Foulds, N.; Gardham, A.; Ghali, N.; Green, A.; Hanna, W.; Harrison, R.; Hegarty, M.; Higgs, J.; Holder, M.; Irving, R.; Jain, V.; Johnson, K.; Jolley, R.; Jones, W.; Jones, G.; Joss, S.; Kalinauskiene, R.; Kanini, F.; Kavanagh, K.; Khan, M.; Khan, N.; Kivuva, E.; Lahiri, N.; L</p>
        <p class="info">Score: 17.8, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297202' target='https://doi.org/10.1101/2023.10.18.23297202'> 10.1101/2023.10.18.23297202</a></p>
        <p class="abstract">Objective: We sought to evaluate outcomes for clinical management following a genetic diagnosis from the Deciphering Developmental Disorders (DDD) Study. Design: Individuals in the DDD study who had a pathogenic/likely pathogenic genotype in the DECIPHER database were selected for inclusion (n=5010). Clinical notes from regional clinical genetics services notes were reviewed to assess pre-defined clinical outcomes relating to interventions, prenatal choices, and information provision. Results: Outcomes were recorded for 4237 diagnosed probands (85% of those eligible) from all 24 recruiting centres across the UK and Ireland. Additional diagnostic or screening tests were performed in 903 (21%) probands through referral to a range of different clinical specialties, and stopped or avoided in a further 26 (0.6%). Disease-specific treatment was started in 85 (2%) probands, including seizure-control medications and dietary supplements, and contra-indicated medications were stopped/avoided as no longer necessary in a further 20 (0.5%). The option of prenatal/preimplantation genetic testing was discussed with 1204 (28%) families, despite the relatively advanced age of the parents at the time of diagnosis. Importantly, condition-specific information or literature was given to 3214 (76%) families, and 880 (21%) were involved in family support groups. In the most common condition (KBG syndrome; 79 (2%) probands), clinical interventions only partially reflected the temporal development of phenotypes, highlighting the importance of consensus management guidelines and patient support groups. Conclusions: Our results underscore the importance of achieving a clinico-molecular diagnosis to ensure timely onward referral of patients, enabling appropriate care and anticipatory surveillance, and for accessing relevant patient support groups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.23297022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.23297022" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.23297022">
        <p class="paperTitle">Polygenic Embryo Screening: High Approval Despite Substantial Concerns from the U.S. Public</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.23297022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.23297022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Furrer, R. A.; Barlevy, D.; Pereira, S.; Carmi, S.; Lencz, T.; Lazaro-Munoz, G.</p>
        <p class="info">Score: 12.0, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.23297022' target='https://doi.org/10.1101/2023.10.13.23297022'> 10.1101/2023.10.13.23297022</a></p>
        <p class="abstract">Polygenic embryo screening (PES) is a novel - yet commercially available - technology that can compute genetic likelihood estimates of polygenic conditions (e.g., diabetes, depression) and traits (e.g., height, cognitive ability) in embryos. Patients undergoing in vitro fertilization (IVF) can use these polygenic scores to select which embryos to transfer for implantation with the goal of having a child. We conducted a survey of the U.S. public to examine attitudes toward PES encompassing acceptance, interest, potential uses, and concerns (n=1435). Our results indicate 72% public approval for PES, with 82% expressing some interest in using PES if already undergoing IVF. Approval for using PES for embryo selection is notably high for physical health (77%) and psychiatric conditions (72%). In contrast, there is minority approval for embryo selection based on PES for behavioral traits (36%) and physical traits (30%). Nevertheless, concerns about PES leading to false expectations, eugenic practices, and stigma are pronounced (54-55% find them &#34;very&#34; to &#34;extremely&#34; concerning). In a second sample of participants (n=192), presenting concerns at survey onset (vs. end) reduced approval (-28%) by mostly increasing ambivalence (&#43;24%), and only slightly increasing disapproval (&#43;4%). Given its commercial availability, practical limitations, and ethical concerns among physicians, patients, geneticists, bioethicists, and legal scholars, it is notable that there is such high public approval and interest in using PES. Understanding these attitudes is essential for informing policymakers, healthcare professionals, and researchers about the publics perspectives on this novel biotechnology and debate about the role of medicine in regulating the use of PES.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297154">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297154" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297154">
        <p class="paperTitle">Disentangling the link between maternal influences on birth weight and disease risk in 36,211 genotyped mother-child pairs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297154" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297154" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leinonen, J. T.; FinnGen,  ; Pirinen, M.; Tukiainen, T.</p>
        <p class="info">Score: 9.0, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297154' target='https://doi.org/10.1101/2023.10.17.23297154'> 10.1101/2023.10.17.23297154</a></p>
        <p class="abstract">Epidemiological studies have robustly linked lower birth weight to later-life disease risks. These observations may reflect the adverse impact of intrauterine growth restriction on a childs health. However, causal evidence supporting such a mechanism in humans is largely lacking.

Using Mendelian Randomization and 36,211 genotyped mother-child pairs from the FinnGen study, we assessed the relationship between intrauterine growth restriction and five common health outcomes (coronary heart disease (CHD), hypertension, statin use, type 2 diabetes and cancer). We proxied intrauterine growth restriction with polygenic scores for maternal effects on birth weight and took into account transmission of genetic variants between a mother and a child in the analyses.

We find limited evidence for contribution of normal variation in maternally influenced intrauterine growth on later-life disease. Instead, we find support for genetic pleiotropy in the child genome linking birth weight to CHD and hypertension. Our study illustrates the opportunities that data from genotyped parent-child pairs from a population-based biobank provides for addressing causality of maternal influences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.23297218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.23297218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.23297218">
        <p class="paperTitle">Bidirectional relationship between olfaction and Parkinson&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.23297218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.23297218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J. J.; Bandres-Ciga, S.; Heilbron, K.; 23andMe Research Team,  ; Blauwendraat, C.; Noyce, A. J.</p>
        <p class="info">Score: 6.1, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.23297218' target='https://doi.org/10.1101/2023.10.18.23297218'> 10.1101/2023.10.18.23297218</a></p>
        <p class="abstract">Introduction Hyposmia (loss of smell) is a common early symptom of Parkinson&#39;s disease (PD). The shared genetic architecture between hyposmia and PD is unknown. Objectives To assess the shared genetic architecture and causal relationship between hyposmia and PD. Methods We leveraged genome-wide association study (GWAS) results for self-assessment of &#39;ability to smell&#39; and PD diagnosis. Linkage disequilibrium score regression (LDSC) and Local Analysis of [co]Variant Association (LAVA) were used to identify genome-wide and local genetic correlations. Mendelian randomization was used to identify potential causal relationships. Results LDSC found that sense of smell negatively correlated at a genome-wide level with PD. LAVA found negative correlations in four genetic loci near GBA1, ANAPC4, SNCA, and MAPT. Using Mendelian randomization we found evidence for strong causal relationship between PD and liability towards poorer sense of smell, but weaker evidence for the reverse direction. Conclusions Hyposmia and PD share genetic liability in only a subset of the major PD risk genes. While there was definitive evidence that PD can lower the sense of smell, there was only suggestive evidence for the reverse. This work highlights the heritability of olfactory function and its relationship with PD heritability and provides further insight into the association between PD and hyposmia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.23293381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.23293381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.23293381">
        <p class="paperTitle">Could Metformin use reduce abdominal aortic aneurysm risk? A Mendelian randomisation study using known Metformin targets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.23293381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.23293381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saxby, K. L.; Dudbridge, F.; Samani, N. J.; Bown, M. J.; Nelson, C. P.</p>
        <p class="info">Score: 4.8, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.23293381' target='https://doi.org/10.1101/2023.10.13.23293381'> 10.1101/2023.10.13.23293381</a></p>
        <p class="abstract">IntroductionAn abdominal aortic aneurysm (AAA) is a swelling of the main artery in the body estimated to affect 0.92% of adults (aged 30-79) worldwide. Rupture is often fatal and surgical intervention may be offered if the risk of rupture is high. There is no treatment to prevent AAA or to slow aneurysm growth aside from dietary and lifestyle recommendations. Metformin, a drug prescribed to treat type 2 diabetes, has previously been associated with a potential reduction in AAA disease risk but no causal link has been shown. Here we investigate the causal link between Metformin and AAA risk through Mendelian randomisation (MR).

MethodsWe conducted a two-sample MR analysis using genetic variants associated with gene expression of five Metformin drug targets that also show a genetic association with decreased glycated haemoglobin (HbA1c) levels. Effect sizes are obtained from within UK Biobank for HbA1c, and within AAAgen for AAA risk, a multi-ancestry meta-GWAS analysis of 39,221 cases and 1,086,107 controls.

ResultsWe identified statistically significant evidence of a causal association between a genetic proxy for Metformin action and a decrease in AAA risk, OR=0.58 (95%CI: 0.37-0.90 p=0.015). We estimate that on average a one standard deviation decrease in HbA1c, measured via Metformin gene targets, reduces AAA risk by over 40%.

ConclusionMetformin use in those at increased risk of AAA may reduce incidence of disease. Clinical trials are required to assess the efficacy of Metformin in reducing disease risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.23297170">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.23297170" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.23297170">
        <p class="paperTitle">iSCORED: nanopore-based random genomic sampling for intraoperative molecular diagnosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.23297170" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.23297170" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emiliani, F.; Ould Ismail, A. A.; Hughes, E.; Tsongalis, G.; Zanazzi, G.; Lin, C.-C.</p>
        <p class="info">Score: 4.5, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.23297170' target='https://doi.org/10.1101/2023.10.17.23297170'> 10.1101/2023.10.17.23297170</a></p>
        <p class="abstract">Copy number variations (CNVs) are almost ubiquitous in cancer. In many cases, somatic CNV analysis has led to the identification of oncogenic pathways and suggested molecular-defined therapeutic targets. Here, we develop iSCORED, a one-step random genomic DNA reconstruction method that enables efficient, unbiased quantification of CNVs using a real-time Nanopore sequencer. By leveraging the long concatenated reads, we generate approximately 1-2 million genomic fragments within one hour of MinION sequencing, allowing for high-resolution genomic dosage comparisons. In our cohort of 26 malignant brain tumors, we demonstrated 100% concordance in CNV detections, including chromosomal alterations and oncogene amplifications when compared to clinically validated next generation sequencing and chromosomal microarray results. In addition, iSCORED allows concurrent brain tumor methylation classification without additional tissue preparation. The integrated methylation information revealed promoter hypomethylation in all detected amplified oncogenes. The entire workflow, including the automatic generation of CNV and methylation reports, can be accomplished within 120-140 minutes. Ultrafast molecular analysis can enhance clinical decision-making, optimize surgical planning and identify potential molecular therapies within surgical timeframes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.23296971">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.23296971" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.23296971">
        <p class="paperTitle">NanoRanger enables rapid single base-pair resolution of genomic disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.23296971" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.23296971" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Bi, C.; Nadeef, S. S.; Maddirevula, S.; Alqahtani, M.; Alkuraya, F.; Li, M.</p>
        <p class="info">Score: 2.7, Published: 2023-10-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.23296971' target='https://doi.org/10.1101/2023.10.14.23296971'> 10.1101/2023.10.14.23296971</a></p>
        <p class="abstract">Rare diseases affect around 350 million individuals globally, yet at least half of those with suspected Mendelian disorders remain without a precise molecular diagnosis despite advanced genetic testing using short read sequencing (SRS). Long-read sequencing (LRS) holds a promise in addressing this diagnostic gap although its clinical application is hampered by its complicated workflow, demanding sample requirements, and exorbitant cost. Genomic disorders represent an opportunity to demonstrate the unique capability of LRS. This study addresses the challenges of identifying missing disease-causing breakpoints in complex genomic disorders by employing multiple LRS strategies, including a novel strategy named nanopore-based rapid acquisition of neighboring genomic regions (NanoRanger). NanoRanger requires neither detailed prior genetic mapping nor large amounts of ultrahigh-molecular-weight DNA, and it stands out for its ease of use and ultra-rapid acquisition of large genomic regions of interest with deep coverage. We describe a cohort of 13 families, each harboring a different homozygous disease-causing genomic rearrangement that defied breakpoint determination by SRS and Optical Genome Mapping (OGM). In each case, NanoRanger identified the breakpoints with a single base-pair resolution. This has enabled the accurate determination of the carrier status of unaffected family members as well as the founder nature of these genomic lesions and their frequency in the local population. It has also enabled an unprecedented analysis of the DNA motifs to discern the mechanism that predisposed to these recessive rearrangements. Our data suggest that NanoRanger can greatly accelerate the clinical adoption of LRS and expand its access for the benefit of patients with rare diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.23296081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.23296081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.23296081">
        <p class="paperTitle">PKHD1L1, A Gene Involved in the Stereociliary Coat, Causes Autosomal Recessive Nonsyndromic Hearing Loss</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.23296081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.23296081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redfield, S. E.; De-la-Torre, P.; Zamani, M.; Khan, H.; Morris, T.; Shariati, G.; Karimi, M.; Kenna, M. A.; Seo, G. H.; Naz, S.; Galehdari, H.; Indzhykulian, A. A.; Shearer, A. E.; Vona, B.</p>
        <p class="info">Score: 9.3, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.23296081' target='https://doi.org/10.1101/2023.10.08.23296081'> 10.1101/2023.10.08.23296081</a></p>
        <p class="abstract">Identification of genes associated with nonsyndromic hearing loss is a crucial endeavor, given the substantial number of individuals who remain without a diagnosis after even the most advanced genetic testing. PKHD1L1 was established as necessary for the formation of the cochlear hair-cell stereociliary coat and causes hearing loss in mice and zebrafish when mutated. We sought to determine if biallelic variants in PKHD1L1 also cause hearing loss in humans.

Exome sequencing was performed on DNA of three families segregating autosomal recessive moderate to severe nonsyndromic sensorineural hearing loss. Compound heterozygous missense p.[(Gly129Ser)];p.[(Gly1314Val)], homozygous missense p.(His2479Gln) and nonsense p.(Arg3381Ter) variants were identified in PKHD1L1 that were predicted to be damaging using in silico pathogenicity prediction methods. In vitro functional analysis of two missense variants was performed using purified recombinant PKHD1L1 protein fragments. We then evaluated protein thermodynamic stability with and without the missense variants found in one of the families. In vitro functional assessment indicated that both engineered PKHD1L1 mutant p.(Gly129Ser) and p.(Gly1314Val) constructs significantly reduced the folding and structural stabilities of the expressed protein fragments, providing further evidence to support pathogenicity of these variants. In silico molecular modelling using AlphaFold2 and protein sequence alignment analysis were carried out to further explore potential variant effects on protein folding and stability and exposed key structural features that might suggest PKHD1L1 protein destabilization.

Multiple lines of evidence collectively associate PKHD1L1 with nonsyndromic mild-moderate to severe sensorineural hearing loss. PKHD1L1 testing in individuals with mild-moderate hearing loss may identify further affected families.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.23296894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.23296894" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.23296894">
        <p class="paperTitle">PheNorm, a language model normalizer of physical examinations from genetics clinical notes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.23296894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.23296894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weissenbacher, D.; Rawal, S.; Zhao, X.; Priestley, J. R. C.; Szigety, K. M.; Schmidt, S. F.; Higgins, M. J.; Magge, A.; O&#39;Connor, K.; Gonzalez-Hernandez, G.; Campbell, I. M.</p>
        <p class="info">Score: 2.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.23296894' target='https://doi.org/10.1101/2023.10.16.23296894'> 10.1101/2023.10.16.23296894</a></p>
        <p class="abstract">BackgroundPhenotypes identified during dysmorphology physical examinations are critical to genetic diagnosis and nearly universally documented as free-text in the electronic health record (EHR). Variation in how phenotypes are recorded in free-text makes large-scale computational analysis extremely challenging. Existing natural language processing (NLP) approaches to address phenotype extraction are trained largely on the biomedical literature or on case vignettes rather than actual EHR data.

MethodsWe implemented a tailored system at the Childrens Hospital of Philadelpia that allows clinicians to document dysmorphology physical exam findings. From the underlying data, we manually annotated a corpus of 3136 organ system observations using the Human Phenotype Ontology (HPO). We provide this corpus publicly. We trained a transformer based NLP system to identify HPO terms from exam observations. The pipeline includes an extractor, which identifies tokens in the sentence expected to contain an HPO term, and a normalizer, which uses those tokens together with the original observation to determine the specific term mentioned.

FindingsWe find that our labeler and normalizer NLP pipeline, which we call PheNorm, achieves state-of-the-art performance for the dysmorphology physical exam phenotype extraction task. PheNorms performance on the test set was 0.717, compared to the nearest baseline system (PhenoTagger) performance of 0.633. An analysis of our systems normalization errors shows possible imperfections in the HPO terminology itself but also reveals a lack of semantic understanding by our transformer models.

InterpretationTransformers-based NLP models are a promising approach to genetic phenotype extraction and, with recent development of larger pre-trained causal language models, may improve semantic understanding in the future. We believe our results also have direct applicability to more general extraction of medical signs and symptoms.

FundingUS National Institutes of Health</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.23296448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.23296448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.23296448">
        <p class="paperTitle">Genetic predisposition to altered blood cell homeostasis is associated with glioma risk and survival</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.23296448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.23296448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kachuri, L.; Guerra, G. A.; Wendt, G. A.; Hansen, H. M.; Molinaro, A.; Bracci, P.; McCoy, L.; Rice, T.; Wiencke, J. K.; Eckel- Passow, J. E.; Jenkins, R. B.; Wrensch, M.; Francis, S. S.</p>
        <p class="info">Score: 2.1, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.23296448' target='https://doi.org/10.1101/2023.10.15.23296448'> 10.1101/2023.10.15.23296448</a></p>
        <p class="abstract">Glioma is a highly fatal brain tumor comprised of molecular subtypes with distinct clinical trajectories. Observational studies have suggested that variability in immune response may play a role in glioma etiology. However, their findings have been inconsistent and susceptible to reverse causation due to treatment effects and the immunosuppressive nature of glioma. We applied genetic variants associated (p&lt;5x10-8) with blood cell traits to a meta-analysis of 3418 glioma cases and 8156 controls. Genetically predicted increase in the platelet to lymphocyte ratio (PLR) was associated with an increased risk of glioma (odds ratio (OR)=1.25, p=0.005), especially in IDH-mutant (IDHmut OR=1.38, p=0.007) and IDHmut 1p/19q non-codeleted (IDHmut-noncodel OR=1.53, p=0.004) tumors. However, reduced glioma risk was observed for higher counts of lymphocytes (IDHmut-noncodel OR=0.70, p=0.004) and neutrophils (IDHmut OR=0.69, p=0.019; IDHmut-noncodel OR=0.60, p=0.009), which may reflect genetic predisposition to enhanced immune-surveillance. In contrast to susceptibility, there was no association with survival in IDHmut-noncodel; however, in IDHmut 1p/19q co-deleted tumors, we observed higher mortality with increasing genetically predicted counts of lymphocytes (hazard ratio (HR)=1.65, 95% CI: 1.24-2.20), neutrophils (HR=1.49, 1.13-1.97), and eosinophils (HR=1.59, 1.18-2.14). Polygenic scores for blood cell traits were also associated with tumor immune microenvironment features, with heterogeneity by IDH status observed for 17 signatures related to interferon signaling, PD-1 expression, and T-cell/Cytotoxic responses. In summary, we identified novel, immune-mediated susceptibility mechanisms for glioma with potential disease management implications.

SIGNIFICANCEThis study suggests that genetic determinants of peripheral blood cell counts influence subtype-specific glioma susceptibility and mortality, with potential effects on the tumor immune microenvironment. These findings may have disease management implications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
