<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel
Authors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel
Authors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel
Authors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel\nAuthors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A.",
  "keywords": [
    
  ],
  "articleBody": " Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel\nAuthors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A. R.; Monteiro, A. N.; Nathan, V.; O'Connor, R.; Pesaran, T.; Radice, P.; Schmidt, G.; Sokilde-Pedersen, I.; Southey, M.; Tavtigian, S. V.; Thompson, B. A.; Toland, A. E.; Turnbull, C.; Vogel, M. J.; Weyandt, J.; Wiggins, G. A. R.; Zec, L.; Couch, F. J.; Walker, L.; Vreeswijk, M. P. G.; Gold\nScore: 5.7, Published: 2024-01-23 DOI: 10.1101/2024.01.22.24301588\nThe ENIGMA research consortium (https://enigmaconsortium.org/) develops and applies methods to determine clinical significance of variants in Hereditary Breast and Ovarian Cancer genes. An ENIGMA BRCA1/2 classification sub-group, originally formed in 2016 as a ClinGen external expert panel, evolved into a ClinGen internal Variant Curation Expert Panel (VCEP) to align with Federal Drug Administration recognized processes for ClinVar contributions. The VCEP reviewed American College of Medical Genetics/Association of Molecular Pathology (ACMG/AMP) classification criteria for relevance to interpreting BRCA1 and BRCA2 variants. Statistical methods were used to calibrate evidence strength for different data types. Pilot specifications were tested on 40 variants, and documentation revised for clarity and ease-of-use. The original criterion descriptions for 13 evidence codes were considered non-applicable or overlapping with other criteria. Scenario of use was extended or re-purposed for eight codes. Extensive analysis and/or data review informed specification descriptions and weights for all codes. Specifications were applied to pilot variants with pre-existing ClinVar classification as follows: 13 Uncertain Significance or Conflicting, 14 Pathogenic and/or Likely Pathogenic, and 13 Benign and/or Likely Benign. Review resolved classification for 11/13 Uncertain Significance or Conflicting variants, and retained or improved confidence in classification for the remaining variants. Alignment of pre-existing ENIGMA research classification processes with ACMG/AMP classification guidelines highlighted several gaps in both the research processes and the baseline ACMG/AMP criteria. Calibration of evidence types was key to justify utility and strength of different evidence types for gene-specific application. The gene-specific criteria demonstrated value for improving ACMG/AMP-aligned classification of BRCA1 and BRCA2 variants.\nA genome-wide investigation into the underlying genetic architecture of personality traits and overlap with psychopathology.\nAuthors: Gupta, P.; Galimberti, M.; Liu, Y.; Beck, S.; Wingo, A.; Wingo, T.; Adhikari, K.; VA Million Veteran Program, ; Stein, M.; Gelernter, J.; Levey, D. F.\nScore: 5.6, Published: 2024-01-17 DOI: 10.1101/2024.01.17.24301428\nPersonality is influenced by both genetic and environmental factors and is associated with other psychiatric traits such as anxiety and depression. The \"Big Five\" personality traits, which include neuroticism, extraversion, agreeableness, conscientiousness, and openness, are a widely accepted and influential framework for understanding and describing human personality. Of the big five personality traits, neuroticism has most often been the focus of genetic studies and is linked to various mental illnesses including depression, anxiety, and schizophrenia. Our knowledge of the genetic architecture of the other four personality traits is more limited. Utilizing the Million Veteran Program (MVP) cohort we conducted a genome-wide association study (GWAS) in individuals of European and African ancestry. Adding other published data, we performed GWAS meta-analysis for each of the five personality traits with sample sizes ranging from 237,390 to 682,688. We identified 158, 14, 3, 2, and 7 independent genome-wide significant (GWS) loci associated with neuroticism, extraversion, agreeableness, conscientiousness, and openness, respectively. These findings represent 55 novel loci for neuroticism, as well as the first GWS loci discovered for extraversion and agreeableness. Gene-based association testing revealed 254 genes showing significant association with at least one of the five personality traits. Transcriptome-wide and proteome-wide analysis identified altered expression of genes and proteins such as CRHR1, SLC12A5, MAPT, and STX4. Pathway enrichment and drug perturbation analyses identified complex biology underlying human personality traits. We also studied the inter-relationship of personality traits with 1,437 other traits in a phenome-wide genetic correlation analysis, identifying new associations. Mendelian randomization showed positive bidirectional effects between neuroticism and depression and anxiety while a negative bidirectional effect was observed for agreeableness and these psychiatric traits. This study improves our comprehensive understanding of the genetic architecture underlying personality traits and their relationship to other complex human traits.\nAn Oxford Nanopore Technology-Based Hepatitis B Virus Sequencing Protocol Suitable For Genomic Surveillance Within Clinical Diagnostic Settings.\nAuthors: Tshiabuila, D.; Choga, W.; San, J. E.; Maponga, T.; Preiser, W.; van Zyl, G.; Moir, M.; van Wyk, S.; Giandhari, J.; Pillay, S.; Anyaneji, U. J.; Lessells, R.; Naidoo, Y.; Sanko, T. J.; Wilkinson, E.; Tegally, H.; Baxter, C.; Martin, D.; de Oliveira, T.\nScore: 13.9, Published: 2024-01-25 DOI: 10.1101/2024.01.19.24301519\nChronic hepatitis B virus (HBV) infection remains a significant public health concern, particularly in Africa, where there is a substantial. HBV is an enveloped virus, with isolates being classified into ten phylogenetically distinct genotypes (A - J) determined based on full-genome sequence data or reverse hybridization-based diagnostic tests. In practice, limitations are noted in that diagnostic sequencing, generally using Sanger sequencing, tends to focus only on the S-gene, yielding little or no information on intra-patient HBV genetic diversity with very low-frequency variants and reverse hybridization detects only known genotype-specific mutations. To resolve these limitations, we developed an Oxford Nanopore Technology (ONT)-based HBV genotyping protocol suitable for clinical virology, yielding complete HBV genome sequences and extensive data on intra-patient HBV diversity. Specifically, the protocol involves tiling-based PCR amplification of HBV sequences, library preparation using the ONT Rapid Barcoding Kit, ONT GridION sequencing, genotyping using Genome Detective software, recombination analysis using jpHMM and RDP5 software, and drug resistance profiling using Geno2pheno software. We prove the utility of our protocol by efficiently generating and characterizing high-quality near full-length HBV genomes from 148 left-over diagnostic Hepatitis B patient samples obtained in the Western Cape province of South Africa, providing valuable insights into the genetic diversity and epidemiology of HBV in this region of the world.\nAutScore- An integrative scoring approach for prioritization of ultra-rare autism spectrum disorder candidate variants from whole exome sequencing data\nAuthors: Shil, A.; Arava, N.; Levi, N.; Levine, L.; Golan, H.; Meiri, G.; Michaelovski, A.; Tsadaka, Y.; Aran, A.; Menashe, I.\nScore: 1.2, Published: 2024-02-01 DOI: 10.1101/2024.01.24.24301544\nBackgroundDiscerning clinically relevant ASD candidate variants from whole-exome sequencing (WES) data is complex, time-consuming, and labor-intensive. To this end, we developed AutScore, an integrative prioritization algorithm of ASD candidate variants from WES data, and assessed its performance to detect clinically relevant variants. MethodsWe studied WES data from 581 ASD probands, and their parents registered in the Azrieli National Center database for Autism and Neurodevelopment Research. We focused on rare allele frequency \u003c1%), high-quality proband-specific variants affecting genes associated with ASD or other neurodevelopmental disorders (NDDs). We assigned a score (i.e., AutScore) to each such variant based on their pathogenicity, clinical relevance, gene-disease association, and inheritance patterns. Finally, we compared the AutScore performance with the rating of clinical experts and the NDD variants prioritization algorithm, AutoCasC. ResultsOverall, 1161 ultra-rare variants distributed in 687 genes in 441 ASD probands were evaluated by AutScore with scores ranging from -4 to 25, with a mean {+/-} SD of 5.89 {+/-} 4.18. AutScore cut-off of [\u0026ge;] 12 outperforms AutoCasC in detecting clinically relevant ASD variants, with a detection accuracy rate of 72.3% and an overall diagnostic yield of 11.9%. Sixteen variants with AutScore of [\u0026ge;] 12 were distributed in fifteen novel ASD genes. ConclusionAutScore is an effective automated ranking system for ASD candidate variants that could be implemented in ASD clinical genetics pipelines.\nConcordance of whole-genome amplified embryonic DNA with the subsequently born child\nAuthors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.\nScore: 22.5, Published: 2024-01-15 DOI: 10.1101/2024.01.12.24301086\nBefore implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGSs ability to identify de novo variants we compared the childs genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimers Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGSs capability to identify genetic variants not explicitly targeted for monogenic screening.\nIdentifying compounds to treat opiate use disorder by leveraging multi-omic data integration and multiple drug repurposing databases.\nAuthors: Stratford, J. K.; Carnes, M. U.; Willis, C.; Minto, M. S.; Elnimeiry, L.; Mathur, R.; Schu, M.; Quach, B. C.; Carter, J.; Nolen, T.; Vandergrift, N.; Kosten, T. R.; Johnson, E. O.; Webb, B. T.\nScore: 6.6, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301329\nGenes influencing opioid use disorder (OUD) biology have been identified via genome-wide association studies (GWAS), gene expression, and network analyses. These discoveries provide opportunities to identifying existing compounds targeting these genes for drug repurposing studies. However, systematically integrating discovery results and identifying relevant available pharmacotherapies for OUD repurposing studies is challenging. To address this, weve constructed a framework that leverages existing results and drug databases to identify candidate pharmacotherapies. For this study, two independent OUD related meta-analyses were used including a GWAS and a differential gene expression (DGE) study of post-mortem human brain. Protein-Protein Interaction (PPI) sub-networks enriched for GWAS risk loci were identified via network analyses. Drug databases Pharos, Open Targets, Therapeutic Target Database (TTD), and DrugBank were queried for clinical status and target selectivity. Cross-omic and drug query results were then integrated to identify candidate compounds. GWAS and DGE analyses revealed 3 and 335 target genes (FDR q\u003c0.05), respectively, while network analysis detected 70 genes in 22 enriched PPI networks. Four selection strategies were implemented, which yielded between 72 and 676 genes with statistically significant support and 110 to 683 drugs targeting these genes, respectively. After filtering out less specific compounds or those targeting well-established psychiatric-related receptors (OPRM1 and DRD2), between 2 and 329 approved drugs remained across the four strategies. By leveraging multiple lines of biological evidence and resources, we identified many FDA approved drugs that target genes associated with OUD. This approach a) allows high-throughput querying of OUD-related genes, b) detects OUD-related genes and compounds not identified using a single domain or resource, and c) produces a succinct summary of FDA approved compounds eligible for efficient expert review. Identifying larger pools of candidate pharmacotherapies and summarizing the supporting evidence bridges the gap between discovery and drug repurposing studies.\nImpact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 34.9, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.\nGenome-wide association analysis of composite sleep health scores in 413,904 individuals\nAuthors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.; Wang, H.\nScore: 0.8, Published: 2024-02-03 DOI: 10.1101/2024.02.02.24302211\nRecent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, and together may provide a more complete picture of sleep health, while also illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p\u003c8.3e-9), along with 341 previously reported loci (p\u003c5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2=0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections to behavioral, psychological, and cardiometabolic traits.\n",
  "wordCount" : "2327",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.24301588">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.24301588" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.24301588">
        <p class="paperTitle">Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.24301588" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.24301588" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A. R.; Monteiro, A. N.; Nathan, V.; O&#39;Connor, R.; Pesaran, T.; Radice, P.; Schmidt, G.; Sokilde-Pedersen, I.; Southey, M.; Tavtigian, S. V.; Thompson, B. A.; Toland, A. E.; Turnbull, C.; Vogel, M. J.; Weyandt, J.; Wiggins, G. A. R.; Zec, L.; Couch, F. J.; Walker, L.; Vreeswijk, M. P. G.; Gold</p>
        <p class="info">Score: 5.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.24301588' target='https://doi.org/10.1101/2024.01.22.24301588'> 10.1101/2024.01.22.24301588</a></p>
        <p class="abstract">The ENIGMA research consortium (https://enigmaconsortium.org/) develops and applies methods to determine clinical significance of variants in Hereditary Breast and Ovarian Cancer genes. An ENIGMA BRCA1/2 classification sub-group, originally formed in 2016 as a ClinGen external expert panel, evolved into a ClinGen internal Variant Curation Expert Panel (VCEP) to align with Federal Drug Administration recognized processes for ClinVar contributions.

The VCEP reviewed American College of Medical Genetics/Association of Molecular Pathology (ACMG/AMP) classification criteria for relevance to interpreting BRCA1 and BRCA2 variants. Statistical methods were used to calibrate evidence strength for different data types. Pilot specifications were tested on 40 variants, and documentation revised for clarity and ease-of-use.

The original criterion descriptions for 13 evidence codes were considered non-applicable or overlapping with other criteria. Scenario of use was extended or re-purposed for eight codes. Extensive analysis and/or data review informed specification descriptions and weights for all codes. Specifications were applied to pilot variants with pre-existing ClinVar classification as follows: 13 Uncertain Significance or Conflicting, 14 Pathogenic and/or Likely Pathogenic, and 13 Benign and/or Likely Benign. Review resolved classification for 11/13 Uncertain Significance or Conflicting variants, and retained or improved confidence in classification for the remaining variants.

Alignment of pre-existing ENIGMA research classification processes with ACMG/AMP classification guidelines highlighted several gaps in both the research processes and the baseline ACMG/AMP criteria. Calibration of evidence types was key to justify utility and strength of different evidence types for gene-specific application. The gene-specific criteria demonstrated value for improving ACMG/AMP-aligned classification of BRCA1 and BRCA2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301428" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301428">
        <p class="paperTitle">A genome-wide investigation into the underlying genetic architecture of personality traits and overlap with psychopathology.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, P.; Galimberti, M.; Liu, Y.; Beck, S.; Wingo, A.; Wingo, T.; Adhikari, K.; VA Million Veteran Program,  ; Stein, M.; Gelernter, J.; Levey, D. F.</p>
        <p class="info">Score: 5.6, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301428' target='https://doi.org/10.1101/2024.01.17.24301428'> 10.1101/2024.01.17.24301428</a></p>
        <p class="abstract">Personality is influenced by both genetic and environmental factors and is associated with other psychiatric traits such as anxiety and depression. The &#34;Big Five&#34; personality traits, which include neuroticism, extraversion, agreeableness, conscientiousness, and openness, are a widely accepted and influential framework for understanding and describing human personality. Of the big five personality traits, neuroticism has most often been the focus of genetic studies and is linked to various mental illnesses including depression, anxiety, and schizophrenia. Our knowledge of the genetic architecture of the other four personality traits is more limited. Utilizing the Million Veteran Program (MVP) cohort we conducted a genome-wide association study (GWAS) in individuals of European and African ancestry. Adding other published data, we performed GWAS meta-analysis for each of the five personality traits with sample sizes ranging from 237,390 to 682,688. We identified 158, 14, 3, 2, and 7 independent genome-wide significant (GWS) loci associated with neuroticism, extraversion, agreeableness, conscientiousness, and openness, respectively. These findings represent 55 novel loci for neuroticism, as well as the first GWS loci discovered for extraversion and agreeableness. Gene-based association testing revealed 254 genes showing significant association with at least one of the five personality traits. Transcriptome-wide and proteome-wide analysis identified altered expression of genes and proteins such as CRHR1, SLC12A5, MAPT, and STX4. Pathway enrichment and drug perturbation analyses identified complex biology underlying human personality traits. We also studied the inter-relationship of personality traits with 1,437 other traits in a phenome-wide genetic correlation analysis, identifying new associations. Mendelian randomization showed positive bidirectional effects between neuroticism and depression and anxiety while a negative bidirectional effect was observed for agreeableness and these psychiatric traits. This study improves our comprehensive understanding of the genetic architecture underlying personality traits and their relationship to other complex human traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.19.24301519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.19.24301519" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.19.24301519">
        <p class="paperTitle">An Oxford Nanopore Technology-Based Hepatitis B Virus Sequencing Protocol Suitable For Genomic Surveillance Within Clinical Diagnostic Settings.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.19.24301519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.19.24301519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tshiabuila, D.; Choga, W.; San, J. E.; Maponga, T.; Preiser, W.; van Zyl, G.; Moir, M.; van Wyk, S.; Giandhari, J.; Pillay, S.; Anyaneji, U. J.; Lessells, R.; Naidoo, Y.; Sanko, T. J.; Wilkinson, E.; Tegally, H.; Baxter, C.; Martin, D.; de Oliveira, T.</p>
        <p class="info">Score: 13.9, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.19.24301519' target='https://doi.org/10.1101/2024.01.19.24301519'> 10.1101/2024.01.19.24301519</a></p>
        <p class="abstract">Chronic hepatitis B virus (HBV) infection remains a significant public health concern, particularly in Africa, where there is a substantial. HBV is an enveloped virus, with isolates being classified into ten phylogenetically distinct genotypes (A - J) determined based on full-genome sequence data or reverse hybridization-based diagnostic tests. In practice, limitations are noted in that diagnostic sequencing, generally using Sanger sequencing, tends to focus only on the S-gene, yielding little or no information on intra-patient HBV genetic diversity with very low-frequency variants and reverse hybridization detects only known genotype-specific mutations. To resolve these limitations, we developed an Oxford Nanopore Technology (ONT)-based HBV genotyping protocol suitable for clinical virology, yielding complete HBV genome sequences and extensive data on intra-patient HBV diversity. Specifically, the protocol involves tiling-based PCR amplification of HBV sequences, library preparation using the ONT Rapid Barcoding Kit, ONT GridION sequencing, genotyping using Genome Detective software, recombination analysis using jpHMM and RDP5 software, and drug resistance profiling using Geno2pheno software. We prove the utility of our protocol by efficiently generating and characterizing high-quality near full-length HBV genomes from 148 left-over diagnostic Hepatitis B patient samples obtained in the Western Cape province of South Africa, providing valuable insights into the genetic diversity and epidemiology of HBV in this region of the world.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.24301544">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.24301544" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.24301544">
        <p class="paperTitle">AutScore- An integrative scoring approach for prioritization of ultra-rare autism spectrum disorder candidate variants from whole exome sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.24301544" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.24301544" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shil, A.; Arava, N.; Levi, N.; Levine, L.; Golan, H.; Meiri, G.; Michaelovski, A.; Tsadaka, Y.; Aran, A.; Menashe, I.</p>
        <p class="info">Score: 1.2, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.24301544' target='https://doi.org/10.1101/2024.01.24.24301544'> 10.1101/2024.01.24.24301544</a></p>
        <p class="abstract">BackgroundDiscerning clinically relevant ASD candidate variants from whole-exome sequencing (WES) data is complex, time-consuming, and labor-intensive. To this end, we developed AutScore, an integrative prioritization algorithm of ASD candidate variants from WES data, and assessed its performance to detect clinically relevant variants.

MethodsWe studied WES data from 581 ASD probands, and their parents registered in the Azrieli National Center database for Autism and Neurodevelopment Research. We focused on rare allele frequency &lt;1%), high-quality proband-specific variants affecting genes associated with ASD or other neurodevelopmental disorders (NDDs). We assigned a score (i.e., AutScore) to each such variant based on their pathogenicity, clinical relevance, gene-disease association, and inheritance patterns. Finally, we compared the AutScore performance with the rating of clinical experts and the NDD variants prioritization algorithm, AutoCasC.

ResultsOverall, 1161 ultra-rare variants distributed in 687 genes in 441 ASD probands were evaluated by AutScore with scores ranging from -4 to 25, with a mean {&#43;/-} SD of 5.89 {&#43;/-} 4.18. AutScore cut-off of [&amp;ge;] 12 outperforms AutoCasC in detecting clinically relevant ASD variants, with a detection accuracy rate of 72.3% and an overall diagnostic yield of 11.9%. Sixteen variants with AutScore of [&amp;ge;] 12 were distributed in fifteen novel ASD genes.

ConclusionAutScore is an effective automated ranking system for ASD candidate variants that could be implemented in ASD clinical genetics pipelines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301086" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301086">
        <p class="paperTitle">Concordance of whole-genome amplified embryonic DNA with the subsequently born child</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Giardina, T.; Katz, M.; Chandramohan, D.; Slotnick, N.; Behr, B.; Siddiqui, N.; Xia, Y.; Podgursky, B.</p>
        <p class="info">Score: 22.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301086' target='https://doi.org/10.1101/2024.01.12.24301086'> 10.1101/2024.01.12.24301086</a></p>
        <p class="abstract">Before implantation subsequent to in vitro fertilization (IVF), the current options for Preimplantation Genetic Testing (PGT) are PGT for Aneuploidy (PGT-A) and, if clinically indicated, PGT for monogenic conditions (PGT-M). A more comprehensive approach involves PGT whole genome sequencing (PGT-WGS). PGT-WGS incorporates PGT-A, screens for hundreds of monogenic conditions, and can evaluate polygenic risk. Here we compare PGT-WGS results against the genome of the subsequently born child. We demonstrated high levels of concordance (both in sensitivity and precision) in exome variant calls between amplified embryonic DNA and sequenced fetal cord blood. This concordance was higher when filtering against 1300 targeted monogenic conditions implicated in birth defects, neurodevelopmental disorders, and hereditary cancer. To evaluate PGT-WGSs ability to identify de novo variants we compared the childs genome to parental genomes and demonstrated that PGT-WGS successfully identified 5/5 confirmed de-novo variants. We further demonstrated concordance in polygenic risk scores calculated for both the embryo and the subsequently born child. This agreement extended to both traditional polygenic scores and oligogenic scores (Type 1 diabetes, Celiac disease, and Alzheimers Disease), which heavily rely on accurate genotyping of HLA and APOE sites. To our knowledge, this is the first direct concordance study between a whole-genome sequencing of a trophectoderm biopsy and the DNA of the subsequently born child. By demonstrating a high degree of whole-exome concordance and adept detection of de novo variants, this approach showcases PGT-WGSs capability to identify genetic variants not explicitly targeted for monogenic screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301329">
        <p class="paperTitle">Identifying compounds to treat opiate use disorder by leveraging multi-omic data integration and multiple drug repurposing databases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stratford, J. K.; Carnes, M. U.; Willis, C.; Minto, M. S.; Elnimeiry, L.; Mathur, R.; Schu, M.; Quach, B. C.; Carter, J.; Nolen, T.; Vandergrift, N.; Kosten, T. R.; Johnson, E. O.; Webb, B. T.</p>
        <p class="info">Score: 6.6, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301329' target='https://doi.org/10.1101/2024.01.17.24301329'> 10.1101/2024.01.17.24301329</a></p>
        <p class="abstract">Genes influencing opioid use disorder (OUD) biology have been identified via genome-wide association studies (GWAS), gene expression, and network analyses. These discoveries provide opportunities to identifying existing compounds targeting these genes for drug repurposing studies. However, systematically integrating discovery results and identifying relevant available pharmacotherapies for OUD repurposing studies is challenging. To address this, weve constructed a framework that leverages existing results and drug databases to identify candidate pharmacotherapies.

For this study, two independent OUD related meta-analyses were used including a GWAS and a differential gene expression (DGE) study of post-mortem human brain. Protein-Protein Interaction (PPI) sub-networks enriched for GWAS risk loci were identified via network analyses. Drug databases Pharos, Open Targets, Therapeutic Target Database (TTD), and DrugBank were queried for clinical status and target selectivity. Cross-omic and drug query results were then integrated to identify candidate compounds.

GWAS and DGE analyses revealed 3 and 335 target genes (FDR q&lt;0.05), respectively, while network analysis detected 70 genes in 22 enriched PPI networks. Four selection strategies were implemented, which yielded between 72 and 676 genes with statistically significant support and 110 to 683 drugs targeting these genes, respectively. After filtering out less specific compounds or those targeting well-established psychiatric-related receptors (OPRM1 and DRD2), between 2 and 329 approved drugs remained across the four strategies.

By leveraging multiple lines of biological evidence and resources, we identified many FDA approved drugs that target genes associated with OUD. This approach a) allows high-throughput querying of OUD-related genes, b) detects OUD-related genes and compounds not identified using a single domain or resource, and c) produces a succinct summary of FDA approved compounds eligible for efficient expert review. Identifying larger pools of candidate pharmacotherapies and summarizing the supporting evidence bridges the gap between discovery and drug repurposing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301165">
        <p class="paperTitle">Impact of genome build on RNA-seq interpretation and diagnostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network,  ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.</p>
        <p class="info">Score: 34.9, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301165' target='https://doi.org/10.1101/2024.01.11.24301165'> 10.1101/2024.01.11.24301165</a></p>
        <p class="abstract">Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.24302211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.24302211" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.24302211">
        <p class="paperTitle">Genome-wide association analysis of composite sleep health scores in 413,904 individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.24302211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.24302211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goodman, M.; Faquih, T.; Paz, V.; Nagarajan, P.; Lane, J.; Spitzer, B.; Maher, M.; Chung, J.; Cade, B. E.; Purcell, S. M.; Zhu, X.; Noordam, R.; Phillips, A. J. K.; Kyle, S.; Spiegelhalder, K.; Weedon, M. N.; Lawlor, D. A.; Rotter, J. I.; Taylor, K.; Isasi, C.; Sofer, T.; Dashti, H. S.; Rutter, M.; Redline, S.; Saxena, R.; Wang, H.</p>
        <p class="info">Score: 0.8, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.24302211' target='https://doi.org/10.1101/2024.02.02.24302211'> 10.1101/2024.02.02.24302211</a></p>
        <p class="abstract">Recent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, and together may provide a more complete picture of sleep health, while also illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p&lt;8.3e-9), along with 341 previously reported loci (p&lt;5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2=0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections to behavioral, psychological, and cardiometabolic traits.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
