<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Genome-wide Association Study of Long COVID
Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O\u0026#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Association Study of Long COVID\nAuthors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O'Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative, ; FinnGen, ; DBDS Ge\nScore: 155.1, Published: 2023-07-01 DOI: 10.1101/2023.06.29.23292056\nInfections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections(1,2). Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction(3-5). The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative(6,7) to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity(6), lung function(8), and cancers(9), suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.\nDiversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program\nAuthors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.; Shi, Y.; Nandi, T. N.; Assimes, T. L.; Brunette, C. A.; Carroll, R. J.; Clifford, R.; Duvall, S.; Gelernter, J.; Hung, A.; Iyengar, S. K.; Joseph, J.; Kember, R.; Kranzler, H.; Levey, D.; Luoh, S.-W.; Merritt, V. C.; Overstreet, C.; Deak, J. D.; Grant, S. F. A.; Poli\nScore: 27.8, Published: 2023-06-29 DOI: 10.1101/2023.06.28.23291975\nGenome-wide association studies (GWAS) have underrepresented individuals from non-European populations, impeding progress in characterizing the genetic architecture and consequences of health and disease traits. To address this, we present a population-stratified phenome-wide GWAS followed by a multi-population meta-analysis for 2,068 traits derived from electronic health records of 635,969 participants in the Million Veteran Program (MVP), a longitudinal cohort study of diverse U.S. Veterans genetically similar to the respective African (121,177), Admixed American (59,048), East Asian (6,702), and European (449,042) superpopulations defined by the 1000 Genomes Project. We identified 38,270 independent variants associating with one or more traits at experiment-wide (P \u003c 4.6x10-11) significance; fine-mapping 6,318 signals identified from 613 traits to single-variant resolution. Among these, a third (2,069) of the associations were found only among participants genetically similar to non-European reference populations, demonstrating the importance of expanding diversity in genetic studies. Our work provides a comprehensive atlas of phenome-wide genetic associations for future studies dissecting the architecture of complex traits in diverse populations. One Sentence SummaryTo address the underrepresentation of non-European individuals in genome-wide association studies (GWAS), we conducted a population-stratified phenome-wide GWAS across 2,068 traits in 635,969 participants from the diverse U.S. Department of Veterans Affairs Million Veteran Program, with results expanding our knowledge of variant-trait associations and highlighting the importance of genetic diversity in understanding the architecture of complex health and disease traits.\nNon-coding variants are a rare cause of recessive developmental disorders in trans with coding variants\nAuthors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium, ; Baralle, D.; Martin, H. C.; Whiffin, N.\nScore: 25.7, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291805\nPurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown. MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing. ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA). ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.\nExome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank\nAuthors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.\nScore: 19.1, Published: 2023-07-03 DOI: 10.1101/2023.06.29.23291992\nExome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF ~ 0.001%) in exome sequencing data from 175,587 UK Biobank participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF \u003c 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( \u003c 1.68 x 10-7) and 17 nominally significant ( \u003c 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR \u003c 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.\nCombining a prioritization strategy and functional studies nominates 5'UTR variants underlying inherited retinal disease\nAuthors: Duenas Rey, A.; del Pozo Valero, M.; Bouckaert, M.; Van Den Broeck, F.; Daich Varela, M.; Van Heetvelde, M.; De Bruyne, M.; Van de Sompele, S.; Bauwens, M.; Ellingford, J. M.; Lenaerts, H.; Mahieu, Q.; Josifova, D.; Genomics England Research Consortium, ; Rivolta, C.; Webster, A.; Arno, G.; Ayuso, C.; De Zaeytijd, J.; Leroy, B.; De Baere, E.; Coppieters, F.\nScore: 14.4, Published: 2023-06-27 DOI: 10.1101/2023.06.19.23291376\nBackground5 untranslated regions (5UTRs) are essential modulators of protein translation. Predicting the impact of 5UTR variants is challenging and typically not performed in routine diagnostics. Here, we present a combined approach of a comprehensive prioritization strategy and subsequent functional assays to evaluate 5UTR variation in two large cohorts of patients with inherited retinal diseases (IRDs). MethodsWe performed an isoform-level re-analysis of retinal RNA-seq data to identify the protein-coding transcripts of 378 IRD genes with highest expression in retina. We evaluated the coverage of these 5UTRs by different whole exome sequencing (WES) capture kits. The selected 5UTRs were analyzed in whole genome sequencing (WGS) and WES data from IRD sub-cohorts from the 100,000 Genomes Project (n = 2,417 WGS) and an in-house database (n = 1,682 WES), respectively. Identified variants were annotated for 5UTR-relevant features and classified into 7 distinct categories based on their predicted functional consequence. We developed a variant prioritization strategy by integrating population frequency, specific criteria for each category, and family and phenotypic data. A selection of candidate variants underwent functional validation using diverse experimental approaches. ResultsIsoform-level re-quantification of retinal gene expression revealed 76 IRD genes with a non-canonical retina-enriched isoform, of which 20 display a fully distinct 5UTR compared to that of their canonical isoform. Depending on the probe-design 3-20% of IRD genes have 5UTRs fully captured by WES. After analyzing these regions in both IRD cohorts we prioritized 11 (likely) pathogenic variants in 10 genes (ARL3, MERTK, NDP, NMNAT1, NPHP4, PAX6, PRPF31, PRPF4, RDH12, RD3), of which 8 were novel. Functional analyses further supported the pathogenicity of 2 variants. The MERTK:c.-125G\u003eA variant, overlapping a transcriptional start site, was shown to significantly reduce both luciferase mRNA levels and activity. The RDH12:c.-123C\u003eT variant was found in cis with the reported hypomorphic RDH12:c.701G\u003eA (p.Arg234His) variant in 11 patients. This 5UTR variant, predicted to introduce an upstream open reading frame, was shown to result in reduced RDH12 protein but unaltered mRNA levels. ConclusionsThis study demonstrates the importance of 5UTR variants implicated in IRDs and provides a systematic approach for 5UTR annotation and validation that is applicable to other inherited diseases.\nBiallelic loss of function variants in WBP4, encoding a spliceosome protein, result in a variable neurodevelopmental delay syndrome\nAuthors: Engal, E.; Oja, K. T.; Maroofian, R.; Geminder, O.; Le, T.-L.; Mor, E.; Tzvi, N.; Elefant, N.; Zaki, M. S.; Gleeson, J. G.; Muru, K.; Pajusalu, S.; Wojcik, M. H.; Pachat, D.; Abd Elmaksoud, M.; Jeong, W. C.; Lee, H.; Bauer, P.; Zifarelli, G.; Houlden, H.; Elpeleg, O.; Gordon, C.; Harel, T.; Ounap, K.; Salton, M.; Mor-Shaked, H.\nScore: 10.1, Published: 2023-06-27 DOI: 10.1101/2023.06.19.23291425\nOver two dozen spliceosome proteins are involved in human diseases, also referred to as spliceosomopathies. WBP4 (WW Domain Binding Protein 4) is part of the early spliceosomal complex, and was not described before in the context of human pathologies. Ascertained through GeneMatcher we identified eleven patients from eight families, with a severe neurodevelopmental syndrome with variable manifestations. Clinical manifestations included hypotonia, global developmental delay, severe intellectual disability, brain abnormalities, musculoskeletal and gastrointestinal abnormalities. Genetic analysis revealed overall five different homozygous loss-of-function variants in WBP4. Immunoblotting on fibroblasts from two affected individuals with different genetic variants demonstrated complete loss of protein, and RNA sequencing analysis uncovered shared abnormal splicing patterns, including enrichment for abnormalities of the nervous system and musculoskeletal system genes, suggesting that the overlapping differentially spliced genes are related to the common phenotypes of the probands. We conclude that biallelic variants in WBP4 cause a spliceosomopathy. Further functional studies are called for better understanding of the mechanism of pathogenicity.\nPA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder\nAuthors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium, ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.\nScore: 6.2, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291611\nThe state-of-the-field in complex disorder genetics is marked by large-scale, biobank-ascertained data sets that integrate broad demographic, health, survey, and genetic data. Leveraging the richness of this data can advance our understanding of complex disorders by improving predictions, describing etiology, and augmenting gene-mapping. These themes are especially, but not exclusively, relevant to Major Depressive Disorder (MDD), a leading cause of disability for which genetic predictors underperform, clinical heterogeneity is etiologically enigmatic, and missing heritability persists. The iPSYCH 2015 case-cohort presents a unique opportunity to integrate genotypes, register-based clinical outcomes, and extended genealogies of 30,949 MDD cases and 39,655 random population controls. To make full use of our data, we introduce the Pearson-Aitken framework for Family Genetic Risk Scores (PA-FGRS) to estimate individuals liabilities for major depressive disorder from extended genealogies of partially observed relatives. PA-FGRS extends previous methods by accounting for censoring, leveraging distant relatives, and utilizing a flexible, model-based approach amenable to analysis and extension - advantages we highlight in simulations. Combining PA-FGRS with genotype data improves classification, replicates and extends known genetic contributions to clinical heterogeneity, and increases power for genome-wide association studies. This study combines novel analyses of unique data and can serve as a model for studies of other outcomes.\nLongitudinal multi-omics study reveals common etiology underlying association between plasma proteome and BMI trajectories in adolescent and young adult twins\nAuthors: Drouard, G.; Hagenbeek, F. A.; Whipp, A.; Pool, R.; Hottenga, J.-J.; Jansen, R.; Hubers, N.; Afonin, A.; BIOS Consortium, ; BBMRI-NL Metabolomics Consortium, ; Willemsen, G.; de Geus, E. J. C.; Ripatti, S.; Pirinen, M.; Kanninen, K. M.; Boomsma, D. I.; van Dongen, J.; Kaprio, J.\nScore: 4.1, Published: 2023-07-01 DOI: 10.1101/2023.06.28.23291995\nBackground: The influence of genetics and environment on the association of the plasma proteome with body mass index (BMI) and changes in BMI remain underexplored, and the links to other omics in these associations remain to be investigated. We characterized protein-BMI trajectory associations in adolescents and adults and how these connect to other omics layers. Methods: Our study included two cohorts of longitudinally followed twins: FinnTwin12 (N=651) and the Netherlands Twin Register (NTR) (N=665). Follow-up comprised four BMI measurements over approximately 6 (NTR: 23-27 years old) to 10 years (FinnTwin12: 12-22 years old), with omics data collected at the last BMI measurement. BMI changes were calculated using latent growth curve models. Mixed-effects models were used to quantify the associations between the abundance of 439 plasma proteins with BMI at blood sampling and changes in BMI. The sources of genetic and environmental variation underlying the protein abundances were quantified using twin models, as were the associations of proteins with BMI and BMI changes. In NTR, we investigated the association of gene expression of genes encoding proteins identified in FinnTwin12 with BMI and changes in BMI. We linked identified proteins and their coding genes to plasma metabolites and polygenic risk scores (PRS) using mixed-effect models and correlation networks. Results: We identified 66 and 14 proteins associated with BMI at blood sampling and changes in BMI, respectively. The average heritability of these proteins was 35%. Of the 66 BMI-protein associations, 43 and 12 showed genetic and environmental correlations, respectively, including 8 proteins showing both. Similarly, we observed 6 and 4 genetic and environmental correlations between changes in BMI and protein abundance, respectively. S100A8 gene expression was associated with BMI at blood sampling, and the PRG4 and CFI genes were associated with BMI changes. Proteins showed strong connections with many metabolites and PRSs, but we observed no multi-omics connections among gene expression and other omics layers. Conclusions: Associations between the proteome and BMI trajectories are characterized by shared genetic, environmental, and metabolic etiologies. We observed few gene-protein pairs associated with BMI or changes in BMI at the proteome and transcriptome levels.\nRare variants in long non-coding RNAs are associated with blood lipid levels in the TOPMed Whole Genome Sequencing Study\nAuthors: Wang, Y.; Selvaraj, M. S.; Li, X.; Li, Z.; Holdcraft, J. A.; Arnett, D. K.; Bis, J. C.; Blangero, J.; Boerwinkle, E.; Bowden, D. W.; Cade, B. E.; Carlson, J. C.; Carson, A. P.; Chen, Y.-D. I.; Curran, J. E.; de Vries, P. S.; Dutcher, S. K.; Ellinor, P. T.; Floyd, J. S.; Fornage, M.; Freedman, B. I.; Gabriel, S.; Germer, S.; Gibbs, R. A.; Guo, X.; He, J.; Heard-Costa, N.; Hildalgo, B.; Hou, L.; Irvin, M. R.; Joehanes, R.; Kaplan, R. C.; Kardia, S. L.; Kelly, T. N.; Kim, R.; Kooperberg, C.; Kral, B. G.; Levy, D.; Li, C.; Liu, C.; Lloyd-Jone, D.; Loos, R. J.; Mahaney, M. C.; Martin, L. W.; Mathia\nScore: 3.8, Published: 2023-06-29 DOI: 10.1101/2023.06.28.23291966\nLong non-coding RNAs (lncRNAs) are known to perform important regulatory functions. Large-scale whole genome sequencing (WGS) studies and new statistical methods for variant set tests now provide an opportunity to assess the associations between rare variants in lncRNA genes and complex traits across the genome. In this study, we used high-coverage WGS from 66,329 participants of diverse ancestries with blood lipid levels (LDL-C, HDL-C, TC, and TG) in the National Heart, Lung, and Blood Institute (NHLBI) Trans-Omics for Precision Medicine (TOPMed) program to investigate the role of lncRNAs in lipid variability. We aggregated rare variants for 165,375 lncRNA genes based on their genomic locations and conducted rare variant aggregate association tests using the STAAR (variant-Set Test for Association using Annotation infoRmation) framework. We performed STAAR conditional analysis adjusting for common variants in known lipid GWAS loci and rare coding variants in nearby protein coding genes. Our analyses revealed 83 rare lncRNA variant sets significantly associated with blood lipid levels, all of which were located in known lipid GWAS loci (in a {+/-}500 kb window of a Global Lipids Genetics Consortium index variant). Notably, 61 out of 83 signals (73%) were conditionally independent of common regulatory variations and rare protein coding variations at the same loci. We replicated 34 out of 61 (56%) conditionally independent associations using the independent UK Biobank WGS data. Our results expand the genetic architecture of blood lipids to rare variants in lncRNA, implicating new therapeutic opportunities.\nThe relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.\nAuthors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium, ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.\nScore: 26.2, Published: 2023-06-23 DOI: 10.1101/2023.06.19.23291606\nGenetics as a science has roots in studying phenotypes of relatives, but molecular approaches facilitate direct measurements of genomic variation within individuals. Agricultural and human biomedical research are both emphasizing genotype-based instruments, like polygenic scores, as the future of breeding programs or precision medicine and genetic epidemiology. However, unlike in agriculture, there is an emerging consensus that family variables act near independent of genotypes in models of human disease. To advance our understanding of this phenomenon, we use 2,066,057 family records of 99,645 genotyped probands from the iPSYCH2015 case-cohort study to show that state-of-the-field genotype- and phenotype-based genetic instruments explain essentially independent components of liability to psychiatric disorders. We support these empirical results with novel theoretical analysis and simulations to describe, in a human biomedical context, parameters affecting current and future performance of the two approaches, their expected interrelationships, relative sample size efficiencies, and the striking consistency of observed results with expectations under simple additive, polygenic liability models of disease. We conclude, at least for psychiatric disorders, that phenotype- and genotype-based genetic instruments are likely to be very noisy measures of the same underlying additive genetic liability, should be seen, in the near future, as complementary, and integrated to a greater extent going forward.\n",
  "wordCount" : "3226",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292056">
        <p class="paperTitle">Genome-wide Association Study of Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammi, V.; Nakanishi, T.; Jones, S. E.; Andrews, S. J.; Karjalainen, J.; Cortes, B.; O&#39;Brien, H. E.; Fulton-Howard, B. E.; Haapaniemi, H. H.; Schmidt, A.; Mitchell, R. E.; Mousas, A.; Mangino, M.; Huerta-Chagoya, A.; Sinnott-Armstrong, N.; Cirulli, E. T.; Vaudel, M.; Kwong, A. S.; Maiti, A. K.; Marttila, M. M.; Batini, C.; Minnai, F.; Dearman, A. R.; Warmerdam, C. R.; Sequeros, C. B.; Winkler, T. W.; Jordan, D. M.; Guare, L.; Vergasova, E.; Marouli, E.; Striano, P.; Zainulabid, U. A.; Kumar, A.; Ahmad, H. F.; Edahiro, R.; Azekawa, S.; Long COVID Host Genetics Initiative,  ; FinnGen,  ; DBDS Ge</p>
        <p class="info">Score: 155.1, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292056' target='https://doi.org/10.1101/2023.06.29.23292056'> 10.1101/2023.06.29.23292056</a></p>
        <p class="abstract">Infections can lead to persistent or long-term symptoms and diseases such as shingles after varicella zoster, cancers after human papillomavirus, or rheumatic fever after streptococcal infections(1,2). Similarly, infection by SARS-CoV-2 can result in Long COVID, a condition characterized by symptoms of fatigue and pulmonary and cognitive dysfunction(3-5). The biological mechanisms that contribute to the development of Long COVID remain to be clarified. We leveraged the COVID-19 Host Genetics Initiative(6,7) to perform a genome-wide association study for Long COVID including up to 6,450 Long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We identified the first genome-wide significant association for Long COVID at the FOXP4 locus. FOXP4 has been previously associated with COVID-19 severity(6), lung function(8), and cancers(9), suggesting a broader role for lung function in the pathophysiology of Long COVID. While we identify COVID-19 severity as a causal risk factor for Long COVID, the impact of the genetic risk factor located in the FOXP4 locus could not be solely explained by its association to severe COVID-19. Our findings further support the role of pulmonary dysfunction and COVID-19 severity in the development of Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.23291975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.23291975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.23291975">
        <p class="paperTitle">Diversity and Scale: Genetic Architecture of 2,068 Traits in the VA Million Veteran Program</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.23291975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.23291975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, A.; Huffman, J. E.; Rodriguez, A.; Conery, M.; Liu, M.; Ho, Y.-L.; Kim, Y.; Heise, D. A.; Guare, L.; Ayakulangara Panickan, V.; Garcon, H.; Linares, F.; Costa, L.; Goethert, I.; Tipton, R.; Honerlaw, J.; Davies, L.; Whitbourne, S.; Cohen, J.; Posner, D. C.; Sangar, R.; Murray, M.; Wang, X.; Dochtermann, D. R.; Devineni, P.; Shi, Y.; Nandi, T. N.; Assimes, T. L.; Brunette, C. A.; Carroll, R. J.; Clifford, R.; Duvall, S.; Gelernter, J.; Hung, A.; Iyengar, S. K.; Joseph, J.; Kember, R.; Kranzler, H.; Levey, D.; Luoh, S.-W.; Merritt, V. C.; Overstreet, C.; Deak, J. D.; Grant, S. F. A.; Poli</p>
        <p class="info">Score: 27.8, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.23291975' target='https://doi.org/10.1101/2023.06.28.23291975'> 10.1101/2023.06.28.23291975</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have underrepresented individuals from non-European populations, impeding progress in characterizing the genetic architecture and consequences of health and disease traits. To address this, we present a population-stratified phenome-wide GWAS followed by a multi-population meta-analysis for 2,068 traits derived from electronic health records of 635,969 participants in the Million Veteran Program (MVP), a longitudinal cohort study of diverse U.S. Veterans genetically similar to the respective African (121,177), Admixed American (59,048), East Asian (6,702), and European (449,042) superpopulations defined by the 1000 Genomes Project. We identified 38,270 independent variants associating with one or more traits at experiment-wide (P &lt; 4.6x10-11) significance; fine-mapping 6,318 signals identified from 613 traits to single-variant resolution. Among these, a third (2,069) of the associations were found only among participants genetically similar to non-European reference populations, demonstrating the importance of expanding diversity in genetic studies. Our work provides a comprehensive atlas of phenome-wide genetic associations for future studies dissecting the architecture of complex traits in diverse populations.

One Sentence SummaryTo address the underrepresentation of non-European individuals in genome-wide association studies (GWAS), we conducted a population-stratified phenome-wide GWAS across 2,068 traits in 635,969 participants from the diverse U.S. Department of Veterans Affairs Million Veteran Program, with results expanding our knowledge of variant-trait associations and highlighting the importance of genetic diversity in understanding the architecture of complex health and disease traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291805">
        <p class="paperTitle">Non-coding variants are a rare cause of recessive developmental disorders in trans with coding variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lord, J.; Jaramillo Oquendo, C.; Martin-Geary, A. C.; Blakes, A. J.; Arciero, E.; Domcke, S.; Childs, A.-M.; Low, K.; Rankin, J.; Genomics England Research Consortium,  ; Baralle, D.; Martin, H. C.; Whiffin, N.</p>
        <p class="info">Score: 25.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291805' target='https://doi.org/10.1101/2023.06.23.23291805'> 10.1101/2023.06.23.23291805</a></p>
        <p class="abstract">PurposeIdentifying pathogenic non-coding variants in individuals with developmental disorders (DD) is challenging due to the large search space. It is common to find a single protein-altering variant in a recessive gene in DD patients, but the prevalence of pathogenic non-coding ;second hits; in trans with these is unknown.

MethodsIn 4,073 genetically undiagnosed rare disease trio probands from the 100,000 Genomes project, we identified rare heterozygous loss-of-function (LoF) or ClinVar pathogenic variants in recessive DD-associated genes. Using stringent region-specific filtering, we identified rare non-coding variants on the other haplotype. Identified genes were clinically evaluated for phenotypic fit, and where possible, we performed functional testing using RNA-sequencing.

ResultsWe found 2,430 probands with one or more rare heterozygous pLoF or ClinVar pathogenic variants in recessive DD-associated genes, for a total of 3,761 proband-variant pairs. For 1,366 (36.3%) of these pairs, we identified at least one rare non-coding variant in trans. After stringent bioinformatic filtering and clinical review, five were determined to be a good clinical fit (in ALMS1, NPHP3, LAMA2, IGHMBP2 and GAA).

ConclusionWe developed a pipeline to systematically identify and annotate compound heterozygous coding/non-coding genotypes. Using this approach we uncovered new diagnoses and conclude that this mechanism is a rare cause of DDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23291992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23291992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23291992">
        <p class="paperTitle">Exome-wide evidence of compound heterozygous effects across common phenotypes in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23291992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23291992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lassen, F. H.; Venkatesh, S. S.; Baya, N.; Zhou, W.; Bloemendal, A.; Neale, B. M.; Kessler, B. M.; Whiffin, N.; Lindgren, C. M.; Palmer, D. S.</p>
        <p class="info">Score: 19.1, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23291992' target='https://doi.org/10.1101/2023.06.29.23291992'> 10.1101/2023.06.29.23291992</a></p>
        <p class="abstract">Exome-sequencing association studies have successfully linked rare protein-coding variation to risk of thousands of diseases. However, the relationship between rare deleterious compound heterozygous (CH) variation and their phenotypic impact has not been fully investigated. Here, we leverage advances in statistical phasing to accurately phase rare variants (MAF ~ 0.001%) in exome sequencing data from 175,587 UK Biobank participants, which we then systematically annotate to identify putatively deleterious CH coding variation. We show that 6.5% of individuals carry such damaging variants in the CH state, with 90% of variants occurring at MAF &lt; 0.34%. Using a logistic mixed model framework, systematically accounting for relatedness, polygenic risk, nearby common variants, and rare variant burden, we investigate recessive effects in common complex diseases. We find six exome-wide significant ( &lt; 1.68 x 10-7) and 17 nominally significant ( &lt; 5.25 x 10-5) gene-trait associations. Among these, only four would have been identified without accounting for CH variation in the gene. We further incorporate age-at-diagnosis information from primary care electronic health records, to show that genetic phase influences lifetime risk of disease across 20 gene-trait combinations (FDR &lt; 5%). Using a permutation approach, we find evidence for genetic phase contributing to disease susceptibility for a collection of gene-trait pairs, including FLG-asthma ( = 0.00205) and USH2A-visual impairment ( = 0.0084). Taken together, we demonstrate the utility of phasing large-scale genetic sequencing cohorts for robust identification of the phenome-wide consequences of compound heterozygosity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291376">
        <p class="paperTitle">Combining a prioritization strategy and functional studies nominates 5&#39;UTR variants underlying inherited retinal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duenas Rey, A.; del Pozo Valero, M.; Bouckaert, M.; Van Den Broeck, F.; Daich Varela, M.; Van Heetvelde, M.; De Bruyne, M.; Van de Sompele, S.; Bauwens, M.; Ellingford, J. M.; Lenaerts, H.; Mahieu, Q.; Josifova, D.; Genomics England Research Consortium,  ; Rivolta, C.; Webster, A.; Arno, G.; Ayuso, C.; De Zaeytijd, J.; Leroy, B.; De Baere, E.; Coppieters, F.</p>
        <p class="info">Score: 14.4, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291376' target='https://doi.org/10.1101/2023.06.19.23291376'> 10.1101/2023.06.19.23291376</a></p>
        <p class="abstract">Background5 untranslated regions (5UTRs) are essential modulators of protein translation. Predicting the impact of 5UTR variants is challenging and typically not performed in routine diagnostics. Here, we present a combined approach of a comprehensive prioritization strategy and subsequent functional assays to evaluate 5UTR variation in two large cohorts of patients with inherited retinal diseases (IRDs).

MethodsWe performed an isoform-level re-analysis of retinal RNA-seq data to identify the protein-coding transcripts of 378 IRD genes with highest expression in retina. We evaluated the coverage of these 5UTRs by different whole exome sequencing (WES) capture kits. The selected 5UTRs were analyzed in whole genome sequencing (WGS) and WES data from IRD sub-cohorts from the 100,000 Genomes Project (n = 2,417 WGS) and an in-house database (n = 1,682 WES), respectively. Identified variants were annotated for 5UTR-relevant features and classified into 7 distinct categories based on their predicted functional consequence. We developed a variant prioritization strategy by integrating population frequency, specific criteria for each category, and family and phenotypic data. A selection of candidate variants underwent functional validation using diverse experimental approaches.

ResultsIsoform-level re-quantification of retinal gene expression revealed 76 IRD genes with a non-canonical retina-enriched isoform, of which 20 display a fully distinct 5UTR compared to that of their canonical isoform. Depending on the probe-design 3-20% of IRD genes have 5UTRs fully captured by WES. After analyzing these regions in both IRD cohorts we prioritized 11 (likely) pathogenic variants in 10 genes (ARL3, MERTK, NDP, NMNAT1, NPHP4, PAX6, PRPF31, PRPF4, RDH12, RD3), of which 8 were novel. Functional analyses further supported the pathogenicity of 2 variants. The MERTK:c.-125G&gt;A variant, overlapping a transcriptional start site, was shown to significantly reduce both luciferase mRNA levels and activity. The RDH12:c.-123C&gt;T variant was found in cis with the reported hypomorphic RDH12:c.701G&gt;A (p.Arg234His) variant in 11 patients. This 5UTR variant, predicted to introduce an upstream open reading frame, was shown to result in reduced RDH12 protein but unaltered mRNA levels.

ConclusionsThis study demonstrates the importance of 5UTR variants implicated in IRDs and provides a systematic approach for 5UTR annotation and validation that is applicable to other inherited diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291425">
        <p class="paperTitle">Biallelic loss of function variants in WBP4, encoding a spliceosome protein, result in a variable neurodevelopmental delay syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engal, E.; Oja, K. T.; Maroofian, R.; Geminder, O.; Le, T.-L.; Mor, E.; Tzvi, N.; Elefant, N.; Zaki, M. S.; Gleeson, J. G.; Muru, K.; Pajusalu, S.; Wojcik, M. H.; Pachat, D.; Abd Elmaksoud, M.; Jeong, W. C.; Lee, H.; Bauer, P.; Zifarelli, G.; Houlden, H.; Elpeleg, O.; Gordon, C.; Harel, T.; Ounap, K.; Salton, M.; Mor-Shaked, H.</p>
        <p class="info">Score: 10.1, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291425' target='https://doi.org/10.1101/2023.06.19.23291425'> 10.1101/2023.06.19.23291425</a></p>
        <p class="abstract">Over two dozen spliceosome proteins are involved in human diseases, also referred to as spliceosomopathies. WBP4 (WW Domain Binding Protein 4) is part of the early spliceosomal complex, and was not described before in the context of human pathologies. Ascertained through GeneMatcher we identified eleven patients from eight families, with a severe neurodevelopmental syndrome with variable manifestations. Clinical manifestations included hypotonia, global developmental delay, severe intellectual disability, brain abnormalities, musculoskeletal and gastrointestinal abnormalities. Genetic analysis revealed overall five different homozygous loss-of-function variants in WBP4. Immunoblotting on fibroblasts from two affected individuals with different genetic variants demonstrated complete loss of protein, and RNA sequencing analysis uncovered shared abnormal splicing patterns, including enrichment for abnormalities of the nervous system and musculoskeletal system genes, suggesting that the overlapping differentially spliced genes are related to the common phenotypes of the probands. We conclude that biallelic variants in WBP4 cause a spliceosomopathy. Further functional studies are called for better understanding of the mechanism of pathogenicity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291611">
        <p class="paperTitle">PA-FGRS is a novel estimator of pedigree-based genetic liability that complements genotype-based inferences into the genetic architecture of major depressive disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dybdahl Krebs, M.; Georgii Hellberg, K.-L.; Lundberg, M.; Appadurai, V.; Ohlsson, H.; Pedersen, E. M.; Steinbach, J.; Matthews, J.; LaBianca, S.; Calle Sanchez, X.; Meijsen, J.; iPSYCH Study Consortium,  ; Ingasson, A.; Buil Demur, A.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Cai, N.; Dahl, A. W.; Zaitlen, N.; Werge, T.; Kendler, K. S.; Schork, A.</p>
        <p class="info">Score: 6.2, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291611' target='https://doi.org/10.1101/2023.06.23.23291611'> 10.1101/2023.06.23.23291611</a></p>
        <p class="abstract">The state-of-the-field in complex disorder genetics is marked by large-scale, biobank-ascertained data sets that integrate broad demographic, health, survey, and genetic data. Leveraging the richness of this data can advance our understanding of complex disorders by improving predictions, describing etiology, and augmenting gene-mapping. These themes are especially, but not exclusively, relevant to Major Depressive Disorder (MDD), a leading cause of disability for which genetic predictors underperform, clinical heterogeneity is etiologically enigmatic, and missing heritability persists. The iPSYCH 2015 case-cohort presents a unique opportunity to integrate genotypes, register-based clinical outcomes, and extended genealogies of 30,949 MDD cases and 39,655 random population controls. To make full use of our data, we introduce the Pearson-Aitken framework for Family Genetic Risk Scores (PA-FGRS) to estimate individuals liabilities for major depressive disorder from extended genealogies of partially observed relatives. PA-FGRS extends previous methods by accounting for censoring, leveraging distant relatives, and utilizing a flexible, model-based approach amenable to analysis and extension - advantages we highlight in simulations. Combining PA-FGRS with genotype data improves classification, replicates and extends known genetic contributions to clinical heterogeneity, and increases power for genome-wide association studies. This study combines novel analyses of unique data and can serve as a model for studies of other outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.23291995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.23291995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.23291995">
        <p class="paperTitle">Longitudinal multi-omics study reveals common etiology underlying association between plasma proteome and BMI trajectories in adolescent and young adult twins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.23291995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.23291995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drouard, G.; Hagenbeek, F. A.; Whipp, A.; Pool, R.; Hottenga, J.-J.; Jansen, R.; Hubers, N.; Afonin, A.; BIOS Consortium,  ; BBMRI-NL Metabolomics Consortium,  ; Willemsen, G.; de Geus, E. J. C.; Ripatti, S.; Pirinen, M.; Kanninen, K. M.; Boomsma, D. I.; van Dongen, J.; Kaprio, J.</p>
        <p class="info">Score: 4.1, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.23291995' target='https://doi.org/10.1101/2023.06.28.23291995'> 10.1101/2023.06.28.23291995</a></p>
        <p class="abstract">Background: The influence of genetics and environment on the association of the plasma proteome with body mass index (BMI) and changes in BMI remain underexplored, and the links to other omics in these associations remain to be investigated. We characterized protein-BMI trajectory associations in adolescents and adults and how these connect to other omics layers. Methods: Our study included two cohorts of longitudinally followed twins: FinnTwin12 (N=651) and the Netherlands Twin Register (NTR) (N=665). Follow-up comprised four BMI measurements over approximately 6 (NTR: 23-27 years old) to 10 years (FinnTwin12: 12-22 years old), with omics data collected at the last BMI measurement. BMI changes were calculated using latent growth curve models. Mixed-effects models were used to quantify the associations between the abundance of 439 plasma proteins with BMI at blood sampling and changes in BMI. The sources of genetic and environmental variation underlying the protein abundances were quantified using twin models, as were the associations of proteins with BMI and BMI changes. In NTR, we investigated the association of gene expression of genes encoding proteins identified in FinnTwin12 with BMI and changes in BMI. We linked identified proteins and their coding genes to plasma metabolites and polygenic risk scores (PRS) using mixed-effect models and correlation networks. Results: We identified 66 and 14 proteins associated with BMI at blood sampling and changes in BMI, respectively. The average heritability of these proteins was 35%. Of the 66 BMI-protein associations, 43 and 12 showed genetic and environmental correlations, respectively, including 8 proteins showing both. Similarly, we observed 6 and 4 genetic and environmental correlations between changes in BMI and protein abundance, respectively. S100A8 gene expression was associated with BMI at blood sampling, and the PRG4 and CFI genes were associated with BMI changes. Proteins showed strong connections with many metabolites and PRSs, but we observed no multi-omics connections among gene expression and other omics layers. Conclusions: Associations between the proteome and BMI trajectories are characterized by shared genetic, environmental, and metabolic etiologies. We observed few gene-protein pairs associated with BMI or changes in BMI at the proteome and transcriptome levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.23291966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.23291966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.23291966">
        <p class="paperTitle">Rare variants in long non-coding RNAs are associated with blood lipid levels in the TOPMed Whole Genome Sequencing Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.23291966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.23291966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Selvaraj, M. S.; Li, X.; Li, Z.; Holdcraft, J. A.; Arnett, D. K.; Bis, J. C.; Blangero, J.; Boerwinkle, E.; Bowden, D. W.; Cade, B. E.; Carlson, J. C.; Carson, A. P.; Chen, Y.-D. I.; Curran, J. E.; de Vries, P. S.; Dutcher, S. K.; Ellinor, P. T.; Floyd, J. S.; Fornage, M.; Freedman, B. I.; Gabriel, S.; Germer, S.; Gibbs, R. A.; Guo, X.; He, J.; Heard-Costa, N.; Hildalgo, B.; Hou, L.; Irvin, M. R.; Joehanes, R.; Kaplan, R. C.; Kardia, S. L.; Kelly, T. N.; Kim, R.; Kooperberg, C.; Kral, B. G.; Levy, D.; Li, C.; Liu, C.; Lloyd-Jone, D.; Loos, R. J.; Mahaney, M. C.; Martin, L. W.; Mathia</p>
        <p class="info">Score: 3.8, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.23291966' target='https://doi.org/10.1101/2023.06.28.23291966'> 10.1101/2023.06.28.23291966</a></p>
        <p class="abstract">Long non-coding RNAs (lncRNAs) are known to perform important regulatory functions. Large-scale whole genome sequencing (WGS) studies and new statistical methods for variant set tests now provide an opportunity to assess the associations between rare variants in lncRNA genes and complex traits across the genome. In this study, we used high-coverage WGS from 66,329 participants of diverse ancestries with blood lipid levels (LDL-C, HDL-C, TC, and TG) in the National Heart, Lung, and Blood Institute (NHLBI) Trans-Omics for Precision Medicine (TOPMed) program to investigate the role of lncRNAs in lipid variability. We aggregated rare variants for 165,375 lncRNA genes based on their genomic locations and conducted rare variant aggregate association tests using the STAAR (variant-Set Test for Association using Annotation infoRmation) framework. We performed STAAR conditional analysis adjusting for common variants in known lipid GWAS loci and rare coding variants in nearby protein coding genes. Our analyses revealed 83 rare lncRNA variant sets significantly associated with blood lipid levels, all of which were located in known lipid GWAS loci (in a {&#43;/-}500 kb window of a Global Lipids Genetics Consortium index variant). Notably, 61 out of 83 signals (73%) were conditionally independent of common regulatory variations and rare protein coding variations at the same loci. We replicated 34 out of 61 (56%) conditionally independent associations using the independent UK Biobank WGS data. Our results expand the genetic architecture of blood lipids to rare variants in lncRNA, implicating new therapeutic opportunities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291606">
        <p class="paperTitle">The relationship between genotype- and phenotype-based estimates of genetic liability to human psychiatric disorders, in practice and in theory.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dybdahl Krebs, M.; Appadurai, V.; Georgii Hellberg, K.-L.; Ohlsson, H.; Steinbach, J.; Pedersen, E. M.; iPSYCH Study Consortium,  ; Werge, T.; Sundquist, J.; Sundquist, K.; Cai, N.; Zaitlen, N.; Dahl, A. W.; Vilhjalmsson, B. J.; Flint, J.; Bacanu, S.-A.; Schork, A. J.; Kendler, K. S.</p>
        <p class="info">Score: 26.2, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291606' target='https://doi.org/10.1101/2023.06.19.23291606'> 10.1101/2023.06.19.23291606</a></p>
        <p class="abstract">Genetics as a science has roots in studying phenotypes of relatives, but molecular approaches facilitate direct measurements of genomic variation within individuals. Agricultural and human biomedical research are both emphasizing genotype-based instruments, like polygenic scores, as the future of breeding programs or precision medicine and genetic epidemiology. However, unlike in agriculture, there is an emerging consensus that family variables act near independent of genotypes in models of human disease. To advance our understanding of this phenomenon, we use 2,066,057 family records of 99,645 genotyped probands from the iPSYCH2015 case-cohort study to show that state-of-the-field genotype- and phenotype-based genetic instruments explain essentially independent components of liability to psychiatric disorders. We support these empirical results with novel theoretical analysis and simulations to describe, in a human biomedical context, parameters affecting current and future performance of the two approaches, their expected interrelationships, relative sample size efficiencies, and the striking consistency of observed results with expectations under simple additive, polygenic liability models of disease. We conclude, at least for psychiatric disorders, that phenotype- and genotype-based genetic instruments are likely to be very noisy measures of the same underlying additive genetic liability, should be seen, in the near future, as complementary, and integrated to a greater extent going forward.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
