<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.
Authors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.
Score: 7.3, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412
IntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.
Authors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.
Score: 7.3, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412
IntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.
Authors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.
Score: 7.3, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412
IntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.\nAuthors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.\nScore: 7.3, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412\nIntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups.",
  "keywords": [
    
  ],
  "articleBody": " Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.\nAuthors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team, ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.\nScore: 7.3, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300412\nIntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups. Very few studies have looked at the transferability of European-derived PRS for behavioural and mental health phenotypes to non-Europeans. We assessed the comparative accuracy of PRS for Major Depressive Disorder (MDD) trained on European and African Ancestries GWAS studies to predict MDD and related traits in African Ancestries participants from the UK Biobank. MethodsUK Biobank participants were selected based on Principal component analysis (PCA) clustering with an African genetic similarity reference population and MDD was assessed with the Composite International Diagnostic Interview (CIDI). Polygenic Risk Scores (PRS) were computed using PRSice2 using either European or African Ancestries GWAS summary statistics. ResultsPRS trained on European ancestry samples (246,363 cases) predicted case control status in Africans of the UK Biobank with similar accuracies (190 cases, R2=2%) to PRS trained on far much smaller samples of African Ancestries participants from 23andMe, Inc. (5045 cases, R2=1.8%). This suggests that prediction of MDD status from Africans to Africans had greater efficiency per unit increase in the discovery sample size than prediction of MDD from Europeans to Africans. Prediction of MDD status in African UK Biobank participants using GWAS findings of causal risk factors from European ancestries was non-significant. ConclusionGWAS studies of MDD in European ancestries are an inefficient means of improving polygenic prediction accuracy in African samples.\nWhole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy\nAuthors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Arzberger, T.; Roeber, S.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L.; Miller, B. L.; Seeley, W. W.; Van Deerlin, V. M.;\nScore: 5.6, Published: 2023-12-30 DOI: 10.1101/2023.12.28.23300612\nProgressive supranuclear palsy (PSP) is a neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. We performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), small insertions/deletions (indels), and structural variants (SVs) in a cohort of 1,718 individuals with PSP and 2,944 control subjects. Analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and also uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. In contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele and the {varepsilon}4 allele to be protective, a pattern similar to the association pattern observed in age-related macular degeneration (AMD) but the opposite observed for Alzheimers disease (AD). Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. We also observed seven common SVs associated with PSP on the H1/H2 haplotype region (17q21.31) and in a few other loci: IGH, PCMT1, CYP2A13, and SMCP. Particularly, in the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP. Through WGS, we significantly refine our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.\nRare disease gene association discovery from burden analysis of the 100,000 Genomes Project data\nAuthors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O'Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra\nScore: 7.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.23300294\nTo discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.\nGenome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health\nAuthors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.\nScore: 1.0, Published: 2023-12-29 DOI: 10.1101/2023.12.26.23300533\nNeuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field, brain age gap has emerged as promising MRI-based biomarker quantifying the deviation between an individuals biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimers disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.\nSEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population\nAuthors: Yang, M.-Y.; Zhong, J.-D.; Li, X.; Bai, W.-Y.; Yuan, C.-D.; Qiu, M.-C.; Liu, K.-Q.; Yu, C.-F.; Li, N.; Yang, J.-J.; Liu, Y.-H.; Yu, S.-H.; Zhao, W.-W.; Liu, J.-Q.; Sun, Y.; Cong, P.-K.; Khederzadeh, S.; Zhao, P.-P.; Qian, Y.; Guan, P.-L.; Gu, J.-X.; Gai, S.-R.; Yi, X.-J.; Tao, J.-G.; Chen, X.; Miao, M.-M.; Lei, L.-X.; Xu, L.; Xie, S.-Y.; Tian, G.; Li, J.-C.; Guo, J.-F.; Karasik, D.; Yang, L.; Tang, B.-S.; Huang, F.; Zheng, H.-F.\nScore: 1.0, Published: 2023-12-27 DOI: 10.1101/2023.12.23.23300480\nHere, we present the South and East Asian Reference Database (SEAD) reference panel (https://imputationserver.westlake.edu.cn/), which comprises whole genome sequencing data from 11,067 individuals across 17 countries in Asia. The SEAD panel, which excludes singleton variants, consists of 22,134 haplotypes and 80,367,720 variants. Firstly, we assessed the concordance rate in global populations using HGDP datasets, notably, the SEAD panel showed advantage in East Asia, Central and South Asia, and Oceania populations. When imputing the disease-associated variants of Asian population, the SEAD panel displayed a distinct preponderance in imputing low-frequency and rare variants. In imputation of Chinese population, the SEAD panel imputed a larger number of well-imputed sites across all minor allele frequency (MAF) bins. Additionally, the SEAD panel exhibited higher imputation accuracy for shared sites in all MAF bins. Finally, we applied the augmented SEAD panel to conduct a discovery and replication genome-wide association study (GWAS) for hip and femoral neck (FN) bone mineral density (BMD) traits within the 5,369 Westlake BioBank for Chinese (WBBC) samples. The single-variant test suggests that rare variants near SNTG1 gene are associated with hip BMD (rs60103302, MAF=0.0091, P=4.79x10-8). The spatial clustering analysis also suggests the association of this gene (Pslide_window=1.08x10-8, Pgene_centric=4.72x10-8). The gene and variants achieved a suggestive level for FN BMD. This gene was not reported previously, and the preliminary experiment demonstrated that the identified rare variant can upregulate the SNTG1 expression, which in turn inhibits the proliferation and differentiation of preosteoblast.\nEthnic and region-specific genetic risk variants of stroke and its comorbid conditions may better define the variations in the burden of stroke and its phenotypic traits\nAuthors: Sukumaran, R.; Nair, A. S.; Banerjee, M.\nScore: 2.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299453\nBurden of stroke differs by region, which could be attributed to differences in comorbid conditions and ethnicity. Genomewide variation acts as a proxy marker for ethnicity, and comorbid conditions. We present an integrated approach to understand this variation by considering prevalence and mortality rates of stroke and its comorbid risk for 204 countries from 2009 to 2019, and GWAS risk variant for all these conditions. Global and regional trend analysis of rates using linear regression, correlation and proportion analysis, signify ethnogeographic differences. Interestingly, the comorbid conditions that act as risk drivers for stroke differed by regions, with more of metabolic risk in America and Europe, in contrast to high SBP in Asian and African regions. GWAS risk loci of stroke and its comorbid conditions indicate distinct population stratification for each of these conditions, signifying for population specific risk. Unique and shared genetic risk variants for stroke, and its comorbid and followed up with ethnic specific variation can help in determining regional risk drivers for stroke. Unique ethnic specific risk variants and their distinct patterns of Linkage Disequilibrium further uncover the drivers for phenotypic variation. Therefore, identifying population and comorbidity specific risk variants might help in defining the threshold for risk, and aid in developing population specific prevention strategies for stroke. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=59 SRC=\"FIGDIR/small/23299453v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (19K): org.highwire.dtl.DTLVardef@a1d264org.highwire.dtl.DTLVardef@1fbe222org.highwire.dtl.DTLVardef@1bb3174org.highwire.dtl.DTLVardef@abdd83_HPS_FORMAT_FIGEXP M_FIG C_FIG\nProgress Update for the Multisite Optical Genome Mapping Evaluation and Validation Study: Prenatal Applications\nAuthors: Levy, B.; Liu, J.; Iqbal, A.; Dupont, B.; Sahajpal, N.; Ho, M.; Yu, J.; Brody, S.; Mason-Suares, H.; Ganapathi, M.; Rajkovic, A.; Smolarek, T.; Toydemir, R.; Bui, P.; Kolhe, R.; Stevenson, R.\nScore: 1.9, Published: 2023-12-26 DOI: 10.1101/2023.12.22.23300469\nOptical genome mapping (OGM) is an emerging technology with great potential for prenatal diagnosis. OGM can identify and resolve all types of balanced and unbalanced cytogenomic abnormalities in a single test, which are typically assessed by multiple standard of care (SOC) methods including karyotyping, fluorescence in situ hybridization and chromosomal microarray. To assess OGMs viability as an alternative to conventional SOC testing, a comprehensive clinical research study was conducted across multiple sites, operators, and instruments to evaluate its accuracy and clinical utility. This report provides an update for the phase 2 results of the ongoing multisite evaluation and validation study evaluating OGM for prenatal applications. In phase 1, 123 prenatal cases were assessed by OGM, and in phase 2, 219 retrospective and prospective prenatal cases have been evaluated. For 71% of cases, at least two SOC tests were performed. The study found that OGM had an overall accuracy of 99.6% and positive predictive value of 100% when compared to all cytogenetic SOC results. With its standardized workflow, cost-effectiveness, and high-resolution cytogenomic analysis, OGM shows great promise as an alternative technology that uses a single assay to consolidate the multiple SOC tests usually used for prenatal cytogenetic diagnosis.\nDiscovery of genomic and transcriptomic pleiotropy between kidney function and soluble receptor for advanced glycation end-products using correlated meta-analyses: The Long Life Family Study (LLFS)\nAuthors: Feitosa, M. F.; Lin, S. J.; Acharya, S.; Thyagarajan, B.; Wojczynski, M. K.; Kuipers, A. L.; Kulminski, A.; Christensen, K.; Zmuda, J. M.; Brent, M. R.; Province, M. A.\nScore: 0.8, Published: 2023-12-29 DOI: 10.1101/2023.12.27.23300583\nPatients with chronic kidney disease (CKD) have increased oxidative stress and chronic inflammation, which may escalate the production of advanced glycation end-products (AGE). High soluble receptor for AGE (sRAGE) and low estimated glomerular filtration rate (eGFR) levels are associated with CKD and aging. We evaluated whether eGFR calculated from creatinine and cystatin C share pleiotropic genetic factors with sRAGE. We employed whole-genome sequencing and correlated meta-analyses on combined genomewide association study (GWAS) p-values in 4,182 individuals (age range: 24-110) from the Long Life Family Study (LLFS). We also conducted transcriptome-wide association studies (TWAS) on whole blood in a subset of 1,209 individuals. We identified 59 pleiotropic GWAS loci (p\u003c5x10-8) and 17 TWAS genes (Bonferroni-p\u003c2.73x10-6) for eGFR traits and sRAGE. TWAS genes, LSP1 and MIR23AHG, were associated with eGFR and sRAGE located within GWAS loci, lncRNA-KCNQ1OT1 and CACNA1A/CCDC130, respectively. GWAS variants were eQTLs in the kidney glomeruli and tubules, and GWAS genes predicted kidney carcinoma. TWAS genes harbored eQTLs in the kidney, predicted kidney carcinoma, and connected enhancer-promoter variants with kidney function-related phenotypes at p\u003c5x10-8. Additionally, higher allele frequencies of protective variants for eGFR traits were detected in LLFS than in ALFA-Europeans and TOPMed, suggesting better kidney function in healthy-aging LLFS than in general populations. Integrating genomic annotation and transcriptional gene activity revealed the enrichment of genetic elements in kidney function and kidney diseases. The identified pleiotropic loci and gene expressions for eGFR and sRAGE suggest their underlying shared genetic effects and highlight their roles in kidney- and aging-related signaling pathways.\nMetDecode: methylation-based deconvolution of cell-free DNA for non-invasive multi-cancer typing\nAuthors: Sudhakaran, D.; Tuveri, S.; Passemiers, A.; Jatsenko, T.; Laga, T.; Punie, K.; Tejpar, S.; Coosemans, A.; Van Nieuwenhuysen, E.; Timmermans, D.; Floris, G.; Van Rompuy, A.-S.; Sagaert, X.; Testa, A. C.; Fischerova, D.; Raimondi, D.; Amant, F.; Lenaerts, L.; Moreau, Y.; Vermeesch, J.\nScore: 0.5, Published: 2023-12-29 DOI: 10.1101/2023.12.29.23300371\nCell-free DNA (cfDNA) mediated early cancer detection is based on detecting alterations in the cfDNA components. However, the underlying pathology can usually not be readily identified. We built a reference atlas based on the methylome of multiple cancer and blood-cell types and developed MetDecode, an epigenetic signature-based deconvolution algorithm. MetDecode accurately estimates the tumour proportion in in-silico mixtures and identifies the tissue of origin in 81.25% cfDNA samples from cancer patients. This method will complement cancer screening programs and guide clinical follow-up.\nA framework for accurate prediction of effector genes in genetic loci for complex traits\nAuthors: Schipper, M.; de Leeuw, C. A.; Maciel, B. A. P. C.; Wightman, D. P.; Hubers, N.; Boomsma, D. I.; O'Donovan, M. C.; Posthuma, D.\nScore: 0.5, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300360\nGenome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus associations remains challenging. Here we present the FLAMES framework, which predicts the most likely effector gene in a locus. FLAMES integrates machine learning with biological data linking single nucleotide polymorphisms (SNPs) to genes, and GWAS-wide convergence of gene interactions. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication, and domain knowledge of molecular traits. We demonstrate that combining SNP-based and convergence-based evidence outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find novel schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life.\n",
  "wordCount" : "2706",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.23300412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.23300412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.23300412">
        <p class="paperTitle">Polygenic prediction of major depressive disorder and related traits in African ancestries UK Biobank participants.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.23300412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.23300412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kanjira, S. C.; Adams, M. C.; Jiang, Y.; Tian, C.; 23andMe Research Team,  ; Lewis, C.; Kuchenbaecker, K.; McIntosh, A. M.</p>
        <p class="info">Score: 7.3, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.23300412' target='https://doi.org/10.1101/2023.12.24.23300412'> 10.1101/2023.12.24.23300412</a></p>
        <p class="abstract">IntroductionGenome-Wide Association Studies (GWAS) over-represent European ancestries compared to the global population, neglecting all other ancestry groups and low-income nations. Consequently, polygenic risk scores (PRS) more accurately predict complex traits in Europeans than African Ancestries groups. Very few studies have looked at the transferability of European-derived PRS for behavioural and mental health phenotypes to non-Europeans. We assessed the comparative accuracy of PRS for Major Depressive Disorder (MDD) trained on European and African Ancestries GWAS studies to predict MDD and related traits in African Ancestries participants from the UK Biobank.

MethodsUK Biobank participants were selected based on Principal component analysis (PCA) clustering with an African genetic similarity reference population and MDD was assessed with the Composite International Diagnostic Interview (CIDI). Polygenic Risk Scores (PRS) were computed using PRSice2 using either European or African Ancestries GWAS summary statistics.

ResultsPRS trained on European ancestry samples (246,363 cases) predicted case control status in Africans of the UK Biobank with similar accuracies (190 cases, R2=2%) to PRS trained on far much smaller samples of African Ancestries participants from 23andMe, Inc. (5045 cases, R2=1.8%). This suggests that prediction of MDD status from Africans to Africans had greater efficiency per unit increase in the discovery sample size than prediction of MDD from Europeans to Africans. Prediction of MDD status in African UK Biobank participants using GWAS findings of causal risk factors from European ancestries was non-significant.

ConclusionGWAS studies of MDD in European ancestries are an inefficient means of improving polygenic prediction accuracy in African samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.23300612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.23300612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.23300612">
        <p class="paperTitle">Whole-Genome Sequencing Analysis Reveals New Susceptibility Loci and Structural Variants Associated with Progressive Supranuclear Palsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.23300612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.23300612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Chang, T. S.; Dombroski, B. A.; Cheng, P.-L.; Patil, V.; Valiente-Banuet, L.; Farrell, K.; McLean, C.; Molina-Porcel, L.; Rajput, A.; Paul De Deyn, P.; Le Bastard, N.; Gearing, M.; Donker Kaat, L.; Van Swieten, J. C.; Dopper, E.; Ghetti, B. F.; Newell, K. L.; Troakes, C.; G de Yebenes, J.; Rabano-Gutierrez, A.; Meller, T.; Oertel, W. H.; Respondek, G.; Arzberger, T.; Roeber, S.; Pastor, P.; Brice, A.; Durr, A.; Le Ber, I.; Beach, T. G.; Serrano, G. E.; Hazrati, L.-N.; Litvan, I.; Rademakers, R.; Ross, O. A.; Galasko, D.; Boxer, A. L.; Miller, B. L.; Seeley, W. W.; Van Deerlin, V. M.;</p>
        <p class="info">Score: 5.6, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.23300612' target='https://doi.org/10.1101/2023.12.28.23300612'> 10.1101/2023.12.28.23300612</a></p>
        <p class="abstract">Progressive supranuclear palsy (PSP) is a neurodegenerative disease characterized by the accumulation of aggregated tau proteins in astrocytes, neurons, and oligodendrocytes. We performed whole genome sequencing (WGS) and conducted association analysis for single nucleotide variants (SNVs), small insertions/deletions (indels), and structural variants (SVs) in a cohort of 1,718 individuals with PSP and 2,944 control subjects. Analysis of common SNVs and indels confirmed known genetic loci at MAPT, MOBP, STX6, SLCO1A2, DUSP10, and SP1, and also uncovered novel signals in APOE, FCHO1/MAP1S, KIF13A, TRIM24, TNXB, and ELOVL1. In contrast to Alzheimers disease (AD), we observed the APOE {varepsilon}2 allele to be the risk allele and the {varepsilon}4 allele to be protective, a pattern similar to the association pattern observed in age-related macular degeneration (AMD) but the opposite observed for Alzheimers disease (AD). Analysis of rare SNVs and indels identified significant association in ZNF592 and further gene network analysis identified a module of neuronal genes dysregulated in PSP. We also observed seven common SVs associated with PSP on the H1/H2 haplotype region (17q21.31) and in a few other loci: IGH, PCMT1, CYP2A13, and SMCP. Particularly, in the H1/H2 haplotype region, there is a burden of rare deletions and duplications (P = 6.73x10-3) in PSP. Through WGS, we significantly refine our understanding of the genetic basis of PSP, providing new targets for exploring disease mechanisms and therapeutic interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300294">
        <p class="paperTitle">Rare disease gene association discovery from burden analysis of the 100,000 Genomes Project data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cipriani, V.; Vestito, L.; Magavern, E. F.; Jacobsen, J. O.; Arno, G.; Behr, E. R.; Benson, K. A.; Bertoli, M.; Bockenhauer, D.; Bowl, M. R.; Burley, K.; Chan, L. F.; Chinnery, P.; Conlon, P.; Costa, M.; Davidson, A. E.; Dawson, S. J.; Elhassan, E.; Flanagan, S. E.; Futema, M.; Gale, D. P.; Garcia-Ruiz, S.; Gonzalez Corcia, C.; Griffin, H. R.; Hambleton, S.; Hicks, A. R.; Houlden, H.; Houlston, R. S.; Howles, S. A.; Kleta, R.; Lekkerkerker, I.; Lin, S.; Liskova, P.; Mitchison, H.; Morsy, H.; Mumford, A. D.; Newman, W. G.; Neatu, R.; O&#39;Toole, E. A.; Ong, A. C.; Pagnamenta, A. T.; Rahman, S.; Ra</p>
        <p class="info">Score: 7.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300294' target='https://doi.org/10.1101/2023.12.20.23300294'> 10.1101/2023.12.20.23300294</a></p>
        <p class="abstract">To discover rare disease-gene associations, we developed a gene burden analytical framework and applied it to rare, protein-coding variants from whole genome sequencing of 35,008 cases with rare diseases and their family members recruited to the 100,000 Genomes Project (100KGP). Following in silico triaging of the results, 88 novel associations were identified including 38 with existing experimental evidence. We have published the confirmation of one of these associations, hereditary ataxia with UCHL1, and independent confirmatory evidence has recently been published for four more. We highlight a further seven compelling associations: hypertrophic cardiomyopathy with DYSF and SLC4A3 where both genes show high/specific heart expression and existing associations to skeletal dystrophies or short QT syndrome respectively; monogenic diabetes with UNC13A with a known role in the regulation of {beta} cells and a mouse model with impaired glucose tolerance; epilepsy with KCNQ1 where a mouse model shows seizures and the existing long QT syndrome association may be linked; early onset Parkinsons disease with RYR1 with existing links to tremor pathophysiology and a mouse model with neurological phenotypes; anterior segment ocular abnormalities associated with POMK showing expression in corneal cells and with a zebrafish model with developmental ocular abnormalities; and cystic kidney disease with COL4A3 showing high renal expression and prior evidence for a digenic or modifying role in renal disease. Confirmation of all 88 associations would lead to potential diagnoses in 456 molecularly undiagnosed cases within the 100KGP, as well as other rare disease patients worldwide, highlighting the clinical impact of a large-scale statistical approach to rare disease gene discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.23300533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.23300533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.23300533">
        <p class="paperTitle">Genome-wide analysis of brain age identifies 25 associated loci and unveils relationships with mental and physical health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.23300533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.23300533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jawinski, P.; Forstbach, H.; Kirsten, H.; Beyer, F.; Villringer, A.; Witte, V.; Scholz, M.; Ripke, S.; Markett, S.</p>
        <p class="info">Score: 1.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.23300533' target='https://doi.org/10.1101/2023.12.26.23300533'> 10.1101/2023.12.26.23300533</a></p>
        <p class="abstract">Neuroimaging and machine learning are opening up new opportunities in studying biological aging mechanisms. In this field,  brain age gap has emerged as promising MRI-based biomarker quantifying the deviation between an individuals biological and chronological age of the brain - an indicator of accelerated/decelerated aging. Here, we investigated the genetic architecture of brain age gap and its relationships with over 1,000 health traits. Genome-wide analyses in 32,634 UK Biobank individuals unveiled a 30% SNP-based heritability and highlighted 25 associated loci. Of these, 23 showed sign-consistency and 16 replicated in another 7,259 individuals. The leading locus encompasses MAPT, encoding the tau protein central to Alzheimers disease. Genetic correlations revealed relationships with various mental health (depression), physical health (diabetes), and socioeconomic variables (education). Mendelian Randomization indicated a causal role of enhanced blood pressure on accelerated brain aging. This work refines our understanding of genetically modulated brain aging and its implications for human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300480" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300480">
        <p class="paperTitle">SEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, M.-Y.; Zhong, J.-D.; Li, X.; Bai, W.-Y.; Yuan, C.-D.; Qiu, M.-C.; Liu, K.-Q.; Yu, C.-F.; Li, N.; Yang, J.-J.; Liu, Y.-H.; Yu, S.-H.; Zhao, W.-W.; Liu, J.-Q.; Sun, Y.; Cong, P.-K.; Khederzadeh, S.; Zhao, P.-P.; Qian, Y.; Guan, P.-L.; Gu, J.-X.; Gai, S.-R.; Yi, X.-J.; Tao, J.-G.; Chen, X.; Miao, M.-M.; Lei, L.-X.; Xu, L.; Xie, S.-Y.; Tian, G.; Li, J.-C.; Guo, J.-F.; Karasik, D.; Yang, L.; Tang, B.-S.; Huang, F.; Zheng, H.-F.</p>
        <p class="info">Score: 1.0, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300480' target='https://doi.org/10.1101/2023.12.23.23300480'> 10.1101/2023.12.23.23300480</a></p>
        <p class="abstract">Here, we present the South and East Asian Reference Database (SEAD) reference panel (https://imputationserver.westlake.edu.cn/), which comprises whole genome sequencing data from 11,067 individuals across 17 countries in Asia. The SEAD panel, which excludes singleton variants, consists of 22,134 haplotypes and 80,367,720 variants. Firstly, we assessed the concordance rate in global populations using HGDP datasets, notably, the SEAD panel showed advantage in East Asia, Central and South Asia, and Oceania populations. When imputing the disease-associated variants of Asian population, the SEAD panel displayed a distinct preponderance in imputing low-frequency and rare variants. In imputation of Chinese population, the SEAD panel imputed a larger number of well-imputed sites across all minor allele frequency (MAF) bins. Additionally, the SEAD panel exhibited higher imputation accuracy for shared sites in all MAF bins. Finally, we applied the augmented SEAD panel to conduct a discovery and replication genome-wide association study (GWAS) for hip and femoral neck (FN) bone mineral density (BMD) traits within the 5,369 Westlake BioBank for Chinese (WBBC) samples. The single-variant test suggests that rare variants near SNTG1 gene are associated with hip BMD (rs60103302, MAF=0.0091, P=4.79x10-8). The spatial clustering analysis also suggests the association of this gene (Pslide_window=1.08x10-8, Pgene_centric=4.72x10-8). The gene and variants achieved a suggestive level for FN BMD. This gene was not reported previously, and the preliminary experiment demonstrated that the identified rare variant can upregulate the SNTG1 expression, which in turn inhibits the proliferation and differentiation of preosteoblast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299453">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299453" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299453">
        <p class="paperTitle">Ethnic and region-specific genetic risk variants of stroke and its comorbid conditions may better define the variations in the burden of stroke and its phenotypic traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299453" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299453" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sukumaran, R.; Nair, A. S.; Banerjee, M.</p>
        <p class="info">Score: 2.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299453' target='https://doi.org/10.1101/2023.12.05.23299453'> 10.1101/2023.12.05.23299453</a></p>
        <p class="abstract">Burden of stroke differs by region, which could be attributed to differences in comorbid conditions and ethnicity. Genomewide variation acts as a proxy marker for ethnicity, and comorbid conditions. We present an integrated approach to understand this variation by considering prevalence and mortality rates of stroke and its comorbid risk for 204 countries from 2009 to 2019, and GWAS risk variant for all these conditions. Global and regional trend analysis of rates using linear regression, correlation and proportion analysis, signify ethnogeographic differences. Interestingly, the comorbid conditions that act as risk drivers for stroke differed by regions, with more of metabolic risk in America and Europe, in contrast to high SBP in Asian and African regions. GWAS risk loci of stroke and its comorbid conditions indicate distinct population stratification for each of these conditions, signifying for population specific risk. Unique and shared genetic risk variants for stroke, and its comorbid and followed up with ethnic specific variation can help in determining regional risk drivers for stroke. Unique ethnic specific risk variants and their distinct patterns of Linkage Disequilibrium further uncover the drivers for phenotypic variation. Therefore, identifying population and comorbidity specific risk variants might help in defining the threshold for risk, and aid in developing population specific prevention strategies for stroke.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=59 SRC=&#34;FIGDIR/small/23299453v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (19K):
org.highwire.dtl.DTLVardef@a1d264org.highwire.dtl.DTLVardef@1fbe222org.highwire.dtl.DTLVardef@1bb3174org.highwire.dtl.DTLVardef@abdd83_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.23300469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.23300469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.23300469">
        <p class="paperTitle">Progress Update for the Multisite Optical Genome Mapping Evaluation and Validation Study: Prenatal Applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.23300469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.23300469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levy, B.; Liu, J.; Iqbal, A.; Dupont, B.; Sahajpal, N.; Ho, M.; Yu, J.; Brody, S.; Mason-Suares, H.; Ganapathi, M.; Rajkovic, A.; Smolarek, T.; Toydemir, R.; Bui, P.; Kolhe, R.; Stevenson, R.</p>
        <p class="info">Score: 1.9, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.23300469' target='https://doi.org/10.1101/2023.12.22.23300469'> 10.1101/2023.12.22.23300469</a></p>
        <p class="abstract">Optical genome mapping (OGM) is an emerging technology with great potential for prenatal diagnosis. OGM can identify and resolve all types of balanced and unbalanced cytogenomic abnormalities in a single test, which are typically assessed by multiple standard of care (SOC) methods including karyotyping, fluorescence in situ hybridization and chromosomal microarray.

To assess OGMs viability as an alternative to conventional SOC testing, a comprehensive clinical research study was conducted across multiple sites, operators, and instruments to evaluate its accuracy and clinical utility. This report provides an update for the phase 2 results of the ongoing multisite evaluation and validation study evaluating OGM for prenatal applications. In phase 1, 123 prenatal cases were assessed by OGM, and in phase 2, 219 retrospective and prospective prenatal cases have been evaluated. For 71% of cases, at least two SOC tests were performed. The study found that OGM had an overall accuracy of 99.6% and positive predictive value of 100% when compared to all cytogenetic SOC results.

With its standardized workflow, cost-effectiveness, and high-resolution cytogenomic analysis, OGM shows great promise as an alternative technology that uses a single assay to consolidate the multiple SOC tests usually used for prenatal cytogenetic diagnosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.23300583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.23300583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.23300583">
        <p class="paperTitle">Discovery of genomic and transcriptomic pleiotropy between kidney function and soluble receptor for advanced glycation end-products using correlated meta-analyses: The Long Life Family Study (LLFS)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.23300583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.23300583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feitosa, M. F.; Lin, S. J.; Acharya, S.; Thyagarajan, B.; Wojczynski, M. K.; Kuipers, A. L.; Kulminski, A.; Christensen, K.; Zmuda, J. M.; Brent, M. R.; Province, M. A.</p>
        <p class="info">Score: 0.8, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.23300583' target='https://doi.org/10.1101/2023.12.27.23300583'> 10.1101/2023.12.27.23300583</a></p>
        <p class="abstract">Patients with chronic kidney disease (CKD) have increased oxidative stress and chronic inflammation, which may escalate the production of advanced glycation end-products (AGE). High soluble receptor for AGE (sRAGE) and low estimated glomerular filtration rate (eGFR) levels are associated with CKD and aging. We evaluated whether eGFR calculated from creatinine and cystatin C share pleiotropic genetic factors with sRAGE. We employed whole-genome sequencing and correlated meta-analyses on combined genomewide association study (GWAS) p-values in 4,182 individuals (age range: 24-110) from the Long Life Family Study (LLFS). We also conducted transcriptome-wide association studies (TWAS) on whole blood in a subset of 1,209 individuals. We identified 59 pleiotropic GWAS loci (p&lt;5x10-8) and 17 TWAS genes (Bonferroni-p&lt;2.73x10-6) for eGFR traits and sRAGE. TWAS genes, LSP1 and MIR23AHG, were associated with eGFR and sRAGE located within GWAS loci, lncRNA-KCNQ1OT1 and CACNA1A/CCDC130, respectively. GWAS variants were eQTLs in the kidney glomeruli and tubules, and GWAS genes predicted kidney carcinoma. TWAS genes harbored eQTLs in the kidney, predicted kidney carcinoma, and connected enhancer-promoter variants with kidney function-related phenotypes at p&lt;5x10-8. Additionally, higher allele frequencies of protective variants for eGFR traits were detected in LLFS than in ALFA-Europeans and TOPMed, suggesting better kidney function in healthy-aging LLFS than in general populations. Integrating genomic annotation and transcriptional gene activity revealed the enrichment of genetic elements in kidney function and kidney diseases. The identified pleiotropic loci and gene expressions for eGFR and sRAGE suggest their underlying shared genetic effects and highlight their roles in kidney- and aging-related signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.23300371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.23300371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.23300371">
        <p class="paperTitle">MetDecode: methylation-based deconvolution of cell-free DNA for non-invasive multi-cancer typing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.23300371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.23300371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sudhakaran, D.; Tuveri, S.; Passemiers, A.; Jatsenko, T.; Laga, T.; Punie, K.; Tejpar, S.; Coosemans, A.; Van Nieuwenhuysen, E.; Timmermans, D.; Floris, G.; Van Rompuy, A.-S.; Sagaert, X.; Testa, A. C.; Fischerova, D.; Raimondi, D.; Amant, F.; Lenaerts, L.; Moreau, Y.; Vermeesch, J.</p>
        <p class="info">Score: 0.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.23300371' target='https://doi.org/10.1101/2023.12.29.23300371'> 10.1101/2023.12.29.23300371</a></p>
        <p class="abstract">Cell-free DNA (cfDNA) mediated early cancer detection is based on detecting alterations in the cfDNA components. However, the underlying pathology can usually not be readily identified. We built a reference atlas based on the methylome of multiple cancer and blood-cell types and developed MetDecode, an epigenetic signature-based deconvolution algorithm. MetDecode accurately estimates the tumour proportion in in-silico mixtures and identifies the tissue of origin in 81.25% cfDNA samples from cancer patients. This method will complement cancer screening programs and guide clinical follow-up.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300360">
        <p class="paperTitle">A framework for accurate prediction of effector genes in genetic loci for complex traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schipper, M.; de Leeuw, C. A.; Maciel, B. A. P. C.; Wightman, D. P.; Hubers, N.; Boomsma, D. I.; O&#39;Donovan, M. C.; Posthuma, D.</p>
        <p class="info">Score: 0.5, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300360' target='https://doi.org/10.1101/2023.12.23.23300360'> 10.1101/2023.12.23.23300360</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus associations remains challenging. Here we present the FLAMES framework, which predicts the most likely effector gene in a locus. FLAMES integrates machine learning with biological data linking single nucleotide polymorphisms (SNPs) to genes, and GWAS-wide convergence of gene interactions. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication, and domain knowledge of molecular traits. We demonstrate that combining SNP-based and convergence-based evidence outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find novel schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
