<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics
Authors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.
Score: 11.5, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298270
Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics
Authors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.
Score: 11.5, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298270
Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics
Authors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.
Score: 11.5, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298270
Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics\nAuthors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.\nScore: 11.5, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298270\nClonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally.",
  "keywords": [
    
  ],
  "articleBody": " Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics\nAuthors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.\nScore: 11.5, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298270\nClonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable (~$8/sample), accurate and highly scalable. To demonstrate the utility of this assay, we detected CHIP in a cohort of 456 individuals with DNA collected at multiple timepoints in the Vanderbilt BioVU biobank and quantified clonal expansion rates over time. A total of 101 individuals with CHIP were identified, and individual-level clonal expansion rate was calculated using the variant allele fraction (VAF) at both timepoints. Differences in clonal expansion rate by driver gene were observed, but there was also significant individual-level heterogeneity, emphasizing the multifactorial nature of clonal expansion. We further describe the mutation co-occurrence and clonal competition between multiple driver mutations.\nSingle locus theory of admixture is insufficient for the study of complex traits in admixed populations\nAuthors: Lee, H.; Lee, M.-H.; Hou, K.; Pasaniuc, B.; Han, B.\nScore: 10.3, Published: 2023-11-08 DOI: 10.1101/2023.11.07.23298201\nAdmixed populations offer valuable insights into the genetic architecture of complex traits. Investigation of such a topic demands a delicate theory that can handle the complex demographic history of admixed populations. With additional assumptions, we show that Pritchard-Stephens-Donnelly (PSD) can predict empirical GWAS findings. We name this the extended PSD (ePSD) model and evaluate the predictions using real data. When applied to real data of admixed genomes, the prediction of the ePSD model is very successful in explaining single-locus behaviors while falling short in predicting two-locus phenomena involving linkage disequilibrium. Our results show that a mosaic of independent single-continental segments is an insufficient approximation of contemporary admixed populations. A more advanced theory that better models linkage disequilibrium of admixed populations will be crucial to better understanding the genetic architecture of complex traits.\nA Massively Parallel Screen of 5'UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes\nAuthors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.\nScore: 16.7, Published: 2023-11-03 DOI: 10.1101/2023.11.02.23297961\nDe novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5'UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5'UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5'UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.\nEvaluating the cost-effectiveness of polygenic risk score-stratified screening for abdominal aortic aneurysm\nAuthors: Kelemen, M.; Danesh, J.; Di Angelantonio, E.; Inouye, M.; O'Sullivan, J. W.; Pennells, L.; Wood, A.; Sweeting, M.; Harrison, S.; Kim, L.\nScore: 8.2, Published: 2023-11-06 DOI: 10.1101/2023.11.02.23297906\nAs the heritability of abdominal aortic aneurysm (AAA) is high and AAA partially shares genetic architecture with other cardiovascular diseases, there is interest in whether genetic information could inform AAA screening strategies. Exploiting pleiotropy and meta-analysing summary data from large AAA studies, we constructed a polygenic risk score (PRS) for AAA. Compared with the low PRS tertile, the intermediate and high PRS tertiles had hazard ratios for AAA of 2.13 (95%CI 1.61, 2.82) and 3.70 (95%CI 2.86, 4.80) respectively, after adjusting for known clinical risk factors. Using simulation modelling, we compared PRS- and smoking-stratified screening with inviting men at age 65 and not inviting women (current UK strategy). In a futuristic scenario where genomic information is available on the population, our modelling suggests inviting male current smokers with high PRS earlier than 65 and targeting screening in female smokers with high/intermediate PRS at 65 and 70 respectively, may improve cost-effectiveness.\nNetwork medicine-based epistasis detection in complex diseases: ready for quantum computing\nAuthors: Hoffmann, M.; Incudini, M.; Poschenrieder, J. M.; Baier, S.; Fitz, A.; Maier, A.; Hartung, M.; Hoffmann, C. M.; Trummer, N.; Adamowicz, K.; Picciani, M.; Scheibling, E.; Harl, M.; Lesch, I.; Frey, H.; Kayser, S.; Wissenberg, P.; Schwartz, L.; Hafner, L.; Acharya, A.; Hackl, L.; Grabert, G.; Lee, S.-G.; Cho, G.; Cloward, M. E.; Jankowski, J.; Lee, H. K. D.; Tsoy, O.; Wenke, N.; Pedersen, A. G.; Bonnelykke, K.; Mandarino, A.; Melograna, F.; Schulz, L.; Climente-Gonzalez, H.; Wilhelm, M.; Iapichino, L.; Wienbrandt, L.; Ellinghaus, D.; Van Steen, K.; Grossi, M.; Furth, P. A.; Hennighausen, L.; Di Score: 5.8, Published: 2023-11-09 DOI: 10.1101/2023.11.07.23298205\nMost heritable diseases are polygenic. To comprehend the underlying genetic architecture, it is crucial to discover the clinically relevant epistatic interactions (EIs) between genomic single nucleotide polymorphisms (SNPs). Existing statistical computational methods for EI detection are mostly limited to pairs of SNPs due to the combinatorial explosion of higher-order EIs. With NeEDL network-based epistasis detection via local search), we leverage network medicine to inform the selection of EIs that are an order of magnitude more statistically significant compared to existing tools and consist, on average, of five SNPs. We further show that this computationally demanding task can be substantially accelerated once quantum computing hardware becomes available. We apply NeEDL to eight different diseases and discover genes (affected by EIs of SNPs) that are partly known to affect the disease, additionally, these results are reproducible across independent cohorts. EIs for these eight diseases can be interactively explored in the Epistasis Disease Atlas: https://epistasis-disease-atlas.com. In summary, NeEDL is the first application that demonstrates the potential of seamlessly integrated quantum computing techniques to accelerate biomedical research. Our network medicine approach detects higher-order EIs with unprecedented statistical and biological evidence, yielding unique insights into polygenic diseases and providing a basis for the development of improved risk scores and combination therapies.\nComparison of gene expression in living and postmortem human brain\nAuthors: Collado-Torres, L.; Klei, L.; Liu, C.; Kleinman, J. E.; Hyde, T. M.; Geschwind, D. H.; Gandal, M. J.; Devlin, B.; Weinberger, D. R.\nScore: 1.8, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298172\nMolecular mechanisms of neuropsychiatric disorders are challenging to study in human brain. For decades, the preferred model has been to study postmortem human brain samples despite the limitations they entail. A recent study generated RNA sequencing data from biopsies of prefrontal cortex from living patients with Parkinson's Disease and compared gene expression to postmortem tissue samples, from which they found vast differences between the two. This led the authors to question the utility of postmortem human brain studies. Through re-analysis of the same data, we unexpectedly found that the living brain tissue samples were of much lower quality than the postmortem samples across multiple standard metrics. We also performed simulations that illustrate the effects of ignoring RNA degradation in differential gene expression analyses, showing the effects can be substantial and of similar magnitude to what the authors find. For these reasons, we believe the authors' conclusions are unjustified. To the contrary, while opportunities to study gene expression in the living brain are welcome, evidence that this eclipses the value of postmortem analyses is not apparent.\nA Genomics England haplotype reference panel and the imputation of the UK Biobank\nAuthors: Shi, S.; Rubinacci, S.; Hu, S.; Moutsianas, L.; Stuckey, A.; Need, A. C.; The Genomics England Resarch Consortium, ; Caulfield, M.; Marchini, J.; Myers, S.\nScore: 1.8, Published: 2023-11-08 DOI: 10.1101/2023.11.06.23298035\nThe choice of reference panels significantly impacts phasing, imputation and GWAS results. In this study, we built a haplotype reference panel using the Genomics England (GEL) high-coverage sequencing dataset, one of the largest genetic variation resources ever collected in the UK. The resulting reference panel consists of 156,390 haplotypes and 342 million autosomal variants. The GEL reference panel demonstrates reliable imputation of variants as rare as 1 in 10,000 within the White British population, with an imputation r2 value of 0.75. The resulting imputed UKB data (GEL-UKB) contains three times more variants, predominantly rare variants, compared to the UKB data previously imputed using the HRC and UK10K reference panel. The GEL-UKB presents a unique opportunity for the reliable discovery of rare associations across the whole genome, especially within the regions not covered by the exome sequencing data. Rare variant signals with high confidence are predominantly from rare coding variants, implying firstly, a probable tendency for existing rare non-coding mutations to not reach a disruptive level comparable to that of coding variants. Secondly, it raises the possibility that the current sample size of UK Biobank may be insufficient for detecting rare variants with a moderate effect size, even with the whole genome sequencing. The resulting GEL phased haplotype reference panel has been made available on the GEL platform and widely used by GEL users. Our GEL imputed UKB data has been adopted as one of the UKB official imputed data resources (Data Field 21008).\nCXCL12 regulates coronary artery dominance in diverse populations and links development to disease\nAuthors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.\nScore: 28.4, Published: 2023-10-27 DOI: 10.1101/2023.10.27.23297507\nMammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.\nDNA methylation effects on Van der Woude Syndrome phenotypic variability\nAuthors: Seaberg, A.; Awotoye, W.; Qian, F.; Dunlay, L.; Butali, A.; Murray, J. C.; Moreno-Uribe, L. M.; Petrin, A.\nScore: 1.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.23298094\nOBJECTIVEVan der Woude Syndrome (VWS) classically presents with combinations of lip pits (LP) and orofacial clefts, with marked phenotypic discordance even amongst individuals carrying the same mutation. Such discordance suggests a possible role for epigenetic factors as phenotypic modifiers. Both IRF6, causal for 70% of VWS cases, and TP63 interact in a regulatory loop to coordinate epithelial proliferation and differentiation for palatogenesis. We hypothesize that differential DNA methylation (DNAm) in CpG sites within regulatory regions of IRF6 and TP63 are associated with VWS phenotypic discordance. METHODSWe measured DNAm levels of CpG sites located in the promoter regions of IRF6 and TP63 and in an IRF6 enhancer element (MCS9.7) in 83 individuals with VWS grouped within 5 phenotypes for primary analysis: 1=CL+/-P+LP, 2=CL+/-P, 3=CP+LP, 4=CP, 5=LP and 2 phenotypes for secondary analysis: 1=any cleft and LP, 2= any cleft without LP. DNA samples were bisulfite converted and pyrosequenced with target-specific primers. Methylation levels were compared amongst phenotypes. RESULTSCpG sites in the IRF6 promoter showed statistically significant differences in methylation among phenotypic groups in both analyses (P\u003c0.05). Individuals with any form of cleft (Groups 1-4) had significantly higher methylation levels than individuals with lip pits only (Group 5). In the secondary analysis, individuals in Group 1 (cleft+LP) had significantly higher methylation than Group 2 (cleft only). CONCLUSIONResults indicated that hypermethylation of the IRF6 promoter is associated with more severe phenotypes (any cleft +/-lip pits); thus, possibly impacting an already genetically weakened IRF6 protein and leading to a more severe phenotype.\nThe qMini assay identifies an overlooked class of splice variants\nAuthors: Guan, B.; Bender, C.; Pantrangi, M.; Moore, N.; Reeves, M.; Naik, A.; Li, H.; Goetz, K.; Blain, D.; Agather, A.; Cukras, C.; Zein, W. M.; Huryn, L. A.; Brooks, B. P.; Hufnagel, R. B.\nScore: 3.1, Published: 2023-11-03 DOI: 10.1101/2023.11.02.23297963\nSplice variants are known to cause diseases by utilizing alternative splice sites, potentially resulting in protein truncation or mRNA degradation by nonsense-mediated decay. Splice variants are verified when altered mature mRNA sequences are identified in RNA analyses or minigene assays. Using a quantitative minigene assay, qMini, we uncovered a previously overlooked class of disease-associated splice variants that did not alter mRNA sequence but decreased mature mRNA level, suggesting a potentially new pathogenic mechanism.\n",
  "wordCount" : "2387",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.23298270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.23298270" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.23298270">
        <p class="paperTitle">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.23298270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.23298270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.</p>
        <p class="info">Score: 11.5, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.23298270' target='https://doi.org/10.1101/2023.11.08.23298270'> 10.1101/2023.11.08.23298270</a></p>
        <p class="abstract">Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable (~$8/sample), accurate and highly scalable. To demonstrate the utility of this assay, we detected CHIP in a cohort of 456 individuals with DNA collected at multiple timepoints in the Vanderbilt BioVU biobank and quantified clonal expansion rates over time. A total of 101 individuals with CHIP were identified, and individual-level clonal expansion rate was calculated using the variant allele fraction (VAF) at both timepoints. Differences in clonal expansion rate by driver gene were observed, but there was also significant individual-level heterogeneity, emphasizing the multifactorial nature of clonal expansion. We further describe the mutation co-occurrence and clonal competition between multiple driver mutations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.23298201">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.23298201" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.23298201">
        <p class="paperTitle">Single locus theory of admixture is insufficient for the study of complex traits in admixed populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.23298201" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.23298201" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, H.; Lee, M.-H.; Hou, K.; Pasaniuc, B.; Han, B.</p>
        <p class="info">Score: 10.3, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.23298201' target='https://doi.org/10.1101/2023.11.07.23298201'> 10.1101/2023.11.07.23298201</a></p>
        <p class="abstract">Admixed populations offer valuable insights into the genetic architecture of complex traits. Investigation of such a topic demands a delicate theory that can handle the complex demographic history of admixed populations. With additional assumptions, we show that Pritchard-Stephens-Donnelly (PSD) can predict empirical GWAS findings. We name this the extended PSD (ePSD) model and evaluate the predictions using real data. When applied to real data of admixed genomes, the prediction of the ePSD model is very successful in explaining single-locus behaviors while falling short in predicting two-locus phenomena involving linkage disequilibrium. Our results show that a mosaic of independent single-continental segments is an insufficient approximation of contemporary admixed populations. A more advanced theory that better models linkage disequilibrium of admixed populations will be crucial to better understanding the genetic architecture of complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297961">
        <p class="paperTitle">A Massively Parallel Screen of 5&#39;UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.</p>
        <p class="info">Score: 16.7, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297961' target='https://doi.org/10.1101/2023.11.02.23297961'> 10.1101/2023.11.02.23297961</a></p>
        <p class="abstract">De novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5&#39;UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5&#39;UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5&#39;UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297906">
        <p class="paperTitle">Evaluating the cost-effectiveness of polygenic risk score-stratified screening for abdominal aortic aneurysm</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelemen, M.; Danesh, J.; Di Angelantonio, E.; Inouye, M.; O&#39;Sullivan, J. W.; Pennells, L.; Wood, A.; Sweeting, M.; Harrison, S.; Kim, L.</p>
        <p class="info">Score: 8.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297906' target='https://doi.org/10.1101/2023.11.02.23297906'> 10.1101/2023.11.02.23297906</a></p>
        <p class="abstract">As the heritability of abdominal aortic aneurysm (AAA) is high and AAA partially shares genetic architecture with other cardiovascular diseases, there is interest in whether genetic information could inform AAA screening strategies. Exploiting pleiotropy and meta-analysing summary data from large AAA studies, we constructed a polygenic risk score (PRS) for AAA. Compared with the low PRS tertile, the intermediate and high PRS tertiles had hazard ratios for AAA of 2.13 (95%CI 1.61, 2.82) and 3.70 (95%CI 2.86, 4.80) respectively, after adjusting for known clinical risk factors. Using simulation modelling, we compared PRS- and smoking-stratified screening with inviting men at age 65 and not inviting women (current UK strategy). In a futuristic scenario where genomic information is available on the population, our modelling suggests inviting male current smokers with high PRS earlier than 65 and targeting screening in female smokers with high/intermediate PRS at 65 and 70 respectively, may improve cost-effectiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.23298205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.23298205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.23298205">
        <p class="paperTitle">Network medicine-based epistasis detection in complex diseases: ready for quantum computing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.23298205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.23298205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, M.; Incudini, M.; Poschenrieder, J. M.; Baier, S.; Fitz, A.; Maier, A.; Hartung, M.; Hoffmann, C. M.; Trummer, N.; Adamowicz, K.; Picciani, M.; Scheibling, E.; Harl, M.; Lesch, I.; Frey, H.; Kayser, S.; Wissenberg, P.; Schwartz, L.; Hafner, L.; Acharya, A.; Hackl, L.; Grabert, G.; Lee, S.-G.; Cho, G.; Cloward, M. E.; Jankowski, J.; Lee, H. K. D.; Tsoy, O.; Wenke, N.; Pedersen, A. G.; Bonnelykke, K.; Mandarino, A.; Melograna, F.; Schulz, L.; Climente-Gonzalez, H.; Wilhelm, M.; Iapichino, L.; Wienbrandt, L.; Ellinghaus, D.; Van Steen, K.; Grossi, M.; Furth, P. A.; Hennighausen, L.; Di </p>
        <p class="info">Score: 5.8, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.23298205' target='https://doi.org/10.1101/2023.11.07.23298205'> 10.1101/2023.11.07.23298205</a></p>
        <p class="abstract">Most heritable diseases are polygenic. To comprehend the underlying genetic architecture, it is crucial to discover the clinically relevant epistatic interactions (EIs) between genomic single nucleotide polymorphisms (SNPs). Existing statistical computational methods for EI detection are mostly limited to pairs of SNPs due to the combinatorial explosion of higher-order EIs. With NeEDL network-based epistasis detection via local search), we leverage network medicine to inform the selection of EIs that are an order of magnitude more statistically significant compared to existing tools and consist, on average, of five SNPs. We further show that this computationally demanding task can be substantially accelerated once quantum computing hardware becomes available. We apply NeEDL to eight different diseases and discover genes (affected by EIs of SNPs) that are partly known to affect the disease, additionally, these results are reproducible across independent cohorts. EIs for these eight diseases can be interactively explored in the Epistasis Disease Atlas: https://epistasis-disease-atlas.com. In summary, NeEDL is the first application that demonstrates the potential of seamlessly integrated quantum computing techniques to accelerate biomedical research. Our network medicine approach detects higher-order EIs with unprecedented statistical and biological evidence, yielding unique insights into polygenic diseases and providing a basis for the development of improved risk scores and combination therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.23298172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.23298172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.23298172">
        <p class="paperTitle">Comparison of gene expression in living and postmortem human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.23298172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.23298172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collado-Torres, L.; Klei, L.; Liu, C.; Kleinman, J. E.; Hyde, T. M.; Geschwind, D. H.; Gandal, M. J.; Devlin, B.; Weinberger, D. R.</p>
        <p class="info">Score: 1.8, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.23298172' target='https://doi.org/10.1101/2023.11.08.23298172'> 10.1101/2023.11.08.23298172</a></p>
        <p class="abstract">Molecular mechanisms of neuropsychiatric disorders are challenging to study in human brain. For decades, the preferred model has been to study postmortem human brain samples despite the limitations they entail. A recent study generated RNA sequencing data from biopsies of prefrontal cortex from living patients with Parkinson&#39;s Disease and compared gene expression to postmortem tissue samples, from which they found vast differences between the two. This led the authors to question the utility of postmortem human brain studies. Through re-analysis of the same data, we unexpectedly found that the living brain tissue samples were of much lower quality than the postmortem samples across multiple standard metrics. We also performed simulations that illustrate the effects of ignoring RNA degradation in differential gene expression analyses, showing the effects can be substantial and of similar magnitude to what the authors find. For these reasons, we believe the authors&#39; conclusions are unjustified. To the contrary, while opportunities to study gene expression in the living brain are welcome, evidence that this eclipses the value of postmortem analyses is not apparent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.23298035">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.23298035" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.23298035">
        <p class="paperTitle">A Genomics England haplotype reference panel and the imputation of the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.23298035" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.23298035" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, S.; Rubinacci, S.; Hu, S.; Moutsianas, L.; Stuckey, A.; Need, A. C.; The Genomics England Resarch Consortium,  ; Caulfield, M.; Marchini, J.; Myers, S.</p>
        <p class="info">Score: 1.8, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.23298035' target='https://doi.org/10.1101/2023.11.06.23298035'> 10.1101/2023.11.06.23298035</a></p>
        <p class="abstract">The choice of reference panels significantly impacts phasing, imputation and GWAS results. In this study, we built a haplotype reference panel using the Genomics England (GEL) high-coverage sequencing dataset, one of the largest genetic variation resources ever collected in the UK. The resulting reference panel consists of 156,390 haplotypes and 342 million autosomal variants. The GEL reference panel demonstrates reliable imputation of variants as rare as 1 in 10,000 within the White British population, with an imputation r2 value of 0.75. The resulting imputed UKB data (GEL-UKB) contains three times more variants, predominantly rare variants, compared to the UKB data previously imputed using the HRC and UK10K reference panel. The GEL-UKB presents a unique opportunity for the reliable discovery of rare associations across the whole genome, especially within the regions not covered by the exome sequencing data. Rare variant signals with high confidence are predominantly from rare coding variants, implying firstly, a probable tendency for existing rare non-coding mutations to not reach a disruptive level comparable to that of coding variants. Secondly, it raises the possibility that the current sample size of UK Biobank may be insufficient for detecting rare variants with a moderate effect size, even with the whole genome sequencing. The resulting GEL phased haplotype reference panel has been made available on the GEL platform and widely used by GEL users. Our GEL imputed UKB data has been adopted as one of the UKB official imputed data resources (Data Field 21008).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297507">
        <p class="paperTitle">CXCL12 regulates coronary artery dominance in diverse populations and links development to disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.</p>
        <p class="info">Score: 28.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297507' target='https://doi.org/10.1101/2023.10.27.23297507'> 10.1101/2023.10.27.23297507</a></p>
        <p class="abstract">Mammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.23298094">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.23298094" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.23298094">
        <p class="paperTitle">DNA methylation effects on Van der Woude Syndrome phenotypic variability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.23298094" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.23298094" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seaberg, A.; Awotoye, W.; Qian, F.; Dunlay, L.; Butali, A.; Murray, J. C.; Moreno-Uribe, L. M.; Petrin, A.</p>
        <p class="info">Score: 1.1, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.23298094' target='https://doi.org/10.1101/2023.11.04.23298094'> 10.1101/2023.11.04.23298094</a></p>
        <p class="abstract">OBJECTIVEVan der Woude Syndrome (VWS) classically presents with combinations of lip pits (LP) and orofacial clefts, with marked phenotypic discordance even amongst individuals carrying the same mutation. Such discordance suggests a possible role for epigenetic factors as phenotypic modifiers. Both IRF6, causal for 70% of VWS cases, and TP63 interact in a regulatory loop to coordinate epithelial proliferation and differentiation for palatogenesis. We hypothesize that differential DNA methylation (DNAm) in CpG sites within regulatory regions of IRF6 and TP63 are associated with VWS phenotypic discordance.

METHODSWe measured DNAm levels of CpG sites located in the promoter regions of IRF6 and TP63 and in an IRF6 enhancer element (MCS9.7) in 83 individuals with VWS grouped within 5 phenotypes for primary analysis: 1=CL&#43;/-P&#43;LP, 2=CL&#43;/-P, 3=CP&#43;LP, 4=CP, 5=LP and 2 phenotypes for secondary analysis: 1=any cleft and LP, 2= any cleft without LP. DNA samples were bisulfite converted and pyrosequenced with target-specific primers. Methylation levels were compared amongst phenotypes.

RESULTSCpG sites in the IRF6 promoter showed statistically significant differences in methylation among phenotypic groups in both analyses (P&lt;0.05). Individuals with any form of cleft (Groups 1-4) had significantly higher methylation levels than individuals with lip pits only (Group 5). In the secondary analysis, individuals in Group 1 (cleft&#43;LP) had significantly higher methylation than Group 2 (cleft only).

CONCLUSIONResults indicated that hypermethylation of the IRF6 promoter is associated with more severe phenotypes (any cleft &#43;/-lip pits); thus, possibly impacting an already genetically weakened IRF6 protein and leading to a more severe phenotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297963">
        <p class="paperTitle">The qMini assay identifies an overlooked class of splice variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guan, B.; Bender, C.; Pantrangi, M.; Moore, N.; Reeves, M.; Naik, A.; Li, H.; Goetz, K.; Blain, D.; Agather, A.; Cukras, C.; Zein, W. M.; Huryn, L. A.; Brooks, B. P.; Hufnagel, R. B.</p>
        <p class="info">Score: 3.1, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297963' target='https://doi.org/10.1101/2023.11.02.23297963'> 10.1101/2023.11.02.23297963</a></p>
        <p class="abstract">Splice variants are known to cause diseases by utilizing alternative splice sites, potentially resulting in protein truncation or mRNA degradation by nonsense-mediated decay. Splice variants are verified when altered mature mRNA sequences are identified in RNA analyses or minigene assays. Using a quantitative minigene assay, qMini, we uncovered a previously overlooked class of disease-associated splice variants that did not alter mRNA sequence but decreased mature mRNA level, suggesting a potentially new pathogenic mechanism.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
