<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A rare splice-site variant in cardiac troponin-T (TNNT2): The need for ancestral diversity in genomic reference datasets
Authors: Butters, A.; Thomson, K.; Harrington, F.; Henden, N.; McGuire, K.; Byrne, A.; Bryen, S.; Leask, M.; Ackerman, M. J.; Atherton, J. J.; Bos, J. M.; Caleshu, C.; Parikh, V. N.; Day, S.; Tardiff, J. C.; Dunn, K.; Hayes, I.; Juang, J.; McGaughran, J.; Nowak, N.; Ronan, A.; Semsarian, C.; Tiemensma, M.; Merriman, T.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="A rare splice-site variant in cardiac troponin-T (TNNT2): The need for ancestral diversity in genomic reference datasets
Authors: Butters, A.; Thomson, K.; Harrington, F.; Henden, N.; McGuire, K.; Byrne, A.; Bryen, S.; Leask, M.; Ackerman, M. J.; Atherton, J. J.; Bos, J. M.; Caleshu, C.; Parikh, V. N.; Day, S.; Tardiff, J. C.; Dunn, K.; Hayes, I.; Juang, J.; McGaughran, J.; Nowak, N.; Ronan, A.; Semsarian, C.; Tiemensma, M.; Merriman, T." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="A rare splice-site variant in cardiac troponin-T (TNNT2): The need for ancestral diversity in genomic reference datasets
Authors: Butters, A.; Thomson, K.; Harrington, F.; Henden, N.; McGuire, K.; Byrne, A.; Bryen, S.; Leask, M.; Ackerman, M. J.; Atherton, J. J.; Bos, J. M.; Caleshu, C.; Parikh, V. N.; Day, S.; Tardiff, J. C.; Dunn, K.; Hayes, I.; Juang, J.; McGaughran, J.; Nowak, N.; Ronan, A.; Semsarian, C.; Tiemensma, M.; Merriman, T."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "A rare splice-site variant in cardiac troponin-T (TNNT2): The need for ancestral diversity in genomic reference datasets\nAuthors: Butters, A.; Thomson, K.; Harrington, F.; Henden, N.; McGuire, K.; Byrne, A.; Bryen, S.; Leask, M.; Ackerman, M. J.; Atherton, J. J.; Bos, J. M.; Caleshu, C.; Parikh, V. N.; Day, S.; Tardiff, J. C.; Dunn, K.; Hayes, I.; Juang, J.; McGaughran, J.; Nowak, N.; Ronan, A.; Semsarian, C.; Tiemensma, M.; Merriman, T.",
  "keywords": [
    
  ],
  "articleBody": " A rare splice-site variant in cardiac troponin-T (TNNT2): The need for ancestral diversity in genomic reference datasets\nAuthors: Butters, A.; Thomson, K.; Harrington, F.; Henden, N.; McGuire, K.; Byrne, A.; Bryen, S.; Leask, M.; Ackerman, M. J.; Atherton, J. J.; Bos, J. M.; Caleshu, C.; Parikh, V. N.; Day, S.; Tardiff, J. C.; Dunn, K.; Hayes, I.; Juang, J.; McGaughran, J.; Nowak, N.; Ronan, A.; Semsarian, C.; Tiemensma, M.; Merriman, T.; Skinner, J. R.; MacArthur, D. G.; Siggs, O. M.; Bagnall, R. D.; Ingles, J.\nScore: 28.4, Published: 2024-02-11 DOI: 10.1101/2024.02.08.24302375\nThe underrepresentation of different ancestry groups in large genomic datasets creates difficulties in interpreting the pathogenicity of monogenic variants. Genetic testing for individuals with non-European ancestry results in higher rates of uncertain variants and a greater risk of misclassification. We report a rare variant in the cardiac troponin T gene, TNNT2; NM_001001430.3: c.571-1G\u003eA (rs483352835) identified via research-based whole exome sequencing in two unrelated probands of Oceanian ancestry with cardiac phenotypes. The variant disrupts the canonical splice acceptor site, activating a cryptic acceptor and resulting in an in-frame deletion (p.Gln191del). The variant is rare in gnomAD v4.0.0 (13/780,762; 0.002%), with the highest frequency in South Asians (5/74,486; 0.007%) and has 16 ClinVar assertions (13 diagnostic clinical laboratories classify as variant of uncertain significance). There are at least 28 reported cases, many with Oceanian ancestry and diverse cardiac phenotypes. Indeed, among Oceanian-ancestry-matched datasets, the allele frequency ranges from 2.9-8.8%, and is present in 2/4 (50%) Indigenous Australian alleles in Genome Asia 100K, with one participant being homozygous. With Oceanians deriving greater than 3% of their DNA from archaic genomes, we found c.571-1G\u003eA in Vindija and Altai Neanderthal, but not the Altai Denisovan, suggesting an origin post Neanderthal divergence from modern humans 130-145 thousand years ago. Based on these data, we classify this variant as benign, and conclude it is not a monogenic cause of disease. Even with ongoing efforts to increase representation in genomics, we highlight the need for caution in assuming rarity of genetic variants in largely European datasets. Efforts to enhance diversity in genomic databases remain crucial.\nNovel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA\nAuthors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O'Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B.; Pengam, A.; Cogne, B.; MacArthur, D. G.; Ulitsky, I.; Carvill, G. L.; O'Donnell-Luria, A.\nScore: 112.7, Published: 2024-02-07 DOI: 10.1101/2024.01.31.24301497\nAbstract/SummaryGenes encoding long non-coding RNAs (lncRNAs) comprise a large fraction of the human genome, yet haploinsufficiency of a lncRNA has not been shown to cause a Mendelian disease. CHASERR is a highly conserved human lncRNA adjacent to CHD2-a coding gene in which de novo loss-of-function variants cause developmental and epileptic encephalopathy. Here we report three unrelated individuals each harboring an ultra-rare heterozygous de novo deletion in the CHASERR locus. We report similarities in severe developmental delay, facial dysmorphisms, and cerebral dysmyelination in these individuals, distinguishing them from the phenotypic spectrum of CHD2 haploinsufficiency. We demonstrate reduced CHASERR mRNA expression and corresponding increased CHD2 mRNA and protein in whole blood and patient-derived cell lines-specifically increased expression of the CHD2 allele in cis with the CHASERR deletion, as predicted from a prior mouse model of Chaserr haploinsufficiency. We show for the first time that de novo structural variants facilitated by Alu-mediated non-allelic homologous recombination led to deletion of a non-coding element (the lncRNA CHASERR) to cause a rare syndromic neurodevelopmental disorder. We also demonstrate that CHD2 has bidirectional dosage sensitivity in human disease. This work highlights the need to carefully evaluate other lncRNAs, particularly those upstream of genes associated with Mendelian disorders.\nGene Discovery and Biological Insights into Anxiety Disorders from a Multi-Ancestry Genome-wide Association Study of \u003e1.2 Million Participants\nAuthors: Friligkou, E.; Lokhammer, S.; Cabrera Mendoza, B.; Shen, J.; He, J.; Deiana, G.; Zanoaga, M. D.; Asgel, Z.; Pilcher, A.; Di Lascio, L.; Makharashvili, A.; Koller, D.; Tylee, D.; Pathak, G. A.; Polimanti, R.\nScore: 10.5, Published: 2024-02-15 DOI: 10.1101/2024.02.14.24302836\nWe leveraged information from more than 1.2 million participants to investigate the genetics of anxiety disorders across five continental ancestral groups. Ancestry-specific and cross-ancestry genome-wide association studies identified 51 anxiety-associated loci, 39 of which are novel. Additionally, polygenic risk scores derived from individuals of European descent were associated with anxiety in African, Admixed-American, and East Asian groups. The heritability of anxiety was enriched for genes expressed in the limbic system, the cerebral cortex, the cerebellum, the metencephalon, the entorhinal cortex, and the brain stem. Transcriptome- and proteome-wide analyses highlighted 115 genes associated with anxiety through brain-specific and cross-tissue regulation. We also observed global and local genetic correlations with depression, schizophrenia, and bipolar disorder and putative causal relationships with several physical health conditions. Overall, this study expands the knowledge regarding the genetic risk and pathogenesis of anxiety disorders, highlighting the importance of investigating diverse populations and integrating multi-omics information.\nDifferences in polygenic score distributions in European ancestry populations: implications for breast cancer risk prediction\nAuthors: Yiangou, K.; Mavaddat, N.; Dennis, J.; Zanti, M.; Wang, Q.; Bolla, M. K.; Abubakar, M.; Ahearn, T. U.; Andrulis, I. L.; Anton-Culver, H.; Antonenkova, N. N.; Arndt, V.; Aronson, K. J.; Augustinsson, A.; Baten, A.; Behrens, S.; Bermisheva, M.; Berrington de Gonzalez, A.; Bialkowska, K.; Boddicker, N.; Bodelon, C.; Bogdanova, N. V.; Bojesen, S. E.; Brantley, K. D.; Brauch, H.; Brenner, H.; Camp, N. J.; Canzian, F.; Castelao, J. E.; Cessna, M. H.; Chang-Claude, J.; Chenevix-Trench, G.; Chung, W. K.; Collaborators, N.; Colonna, S. V.; Couch, F. J.; Cox, A.; Cross, S. S.; Czene, K.; Daly, M. B.; De\nScore: 10.5, Published: 2024-02-13 DOI: 10.1101/2024.02.12.24302043\nThe 313-variant polygenic risk score (PRS313) provides a promising tool for breast cancer risk prediction. However, evaluation of the PRS313 across different European populations which could influence risk estimation has not been performed. Here, we explored the distribution of PRS313 across European populations using genotype data from 94,072 females without breast cancer, of European-ancestry from 21 countries participating in the Breast Cancer Association Consortium (BCAC) and 225,105 female participants from the UK Biobank. The mean PRS313 differed markedly across European countries, being highest in south-eastern Europe and lowest in north-western Europe. Using the overall European PRS313 distribution to categorise individuals leads to overestimation and underestimation of risk in some individuals from south-eastern and north-western countries, respectively. Adjustment for principal components explained most of the observed heterogeneity in mean PRS. Country-specific PRS distributions may be used to calibrate risk categories in individuals from different countries.\nComparative analysis of 136,401 Admixed Americans and 419,228 Europeans reveals ancestry-specific genetic determinants of clonal haematopoiesis\nAuthors: Wen, S.; Kuri-Morales, P.; Hu, F.; Nag, A.; Tachmazidou, I.; Vishnu Vardhan Deevi, S.; Taiy, H.; Smith, K.; Loesch, D. P.; Burren, O. S.; Dhindsa, R. S.; Wasilewski, S.; Alegre-Diaz, J.; Berumen, J.; Emberson, J.; Torres, J. M.; Collins, R.; Carss, K.; Wang, Q.; Petrovski, S.; Tapia-Conyer, R.; Fabre, M. A.; Harper, A. R.; Vassiliou, G.; Mitchell, J.\nScore: 8.7, Published: 2024-02-09 DOI: 10.1101/2024.02.07.24302442\nThe development of clonal haematopoiesis (CH), the age-related expansion of mutated haematopoietic stem cell (HSC) clones, is influenced by genetic and non-genetic factors. To date, large-scale studies of CH have focused on individuals of European descent, such that the impact of genetic ancestry on CH development remains incompletely understood. Here, we investigate this by studying CH in 136,401 admixed participants from the Mexico City Prospective Study (MCPS) and 419,228 European participants from the UK Biobank (UKB). We observe that CH was significantly less common in MCPS compared to UKB (adjusted odds ratio (OR) = 0.56 [95% Cl = 0.55-0.59], P = 1.60 x 10-206), a difference that persisted when comparing MCPS participants whose genomes were \u003e50% ancestrally Indigenous American to those whose genomes were \u003e50% ancestrally European (adjusted OR = 0.76 [0.70-0.83], P = 1.78 x 10-10). Genome- and exome-wide association analyses in MCPS participants identified two novel loci associated with CH (CSGALNACT1 and DIAPH3), and ancestry-specific variants in the TCL1B locus with opposing effect on DNMT3A-versus non-DNMT3A-CH. Meta-analysis of the MCPS and UKB cohorts identified another five novel loci associated with overall or gene specific CH, including polymorphisms at PAPR11/CCND2, MEIS1 and UBE2G1/SPNS3. Our CH study, the largest in a non-European population to date, demonstrates the profound impact of ancestry on CH development and reveals the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis and advance health equity amongst different human populations.\nConnecting genomic results for psychiatric disorders to human brain cell types and regions reveals convergence with functional connectivity\nAuthors: Yao, S.; Harder, A.; Darki, F.; Chang, Y.-W.; Li, A.; Nikouei, K.; Volpe, G.; Lundstrom, J. N.; Zeng, J.; Wray, N. R.; Lu, Y.; Sullivan, P. F.; Hjerling-Leffler, J.\nScore: 8.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.24301478\nUnderstanding the temporal and spatial brain locations etiological for psychiatric disorders is essential for targeted neurobiological research. Integration of genomic insights from genome-wide association studies with single-cell transcriptomics is a powerful approach although past efforts have necessarily relied on mouse atlases. Leveraging a comprehensive atlas of the adult human brain, we prioritized cell types via the enrichment of SNP-heritabilities for brain diseases, disorders, and traits, progressing from individual cell types to brain regions. Our findings highlight specific neuronal clusters significantly enriched for the SNP-heritabilities for schizophrenia, bipolar disorder, and major depressive disorder along with intelligence, education, and neuroticism. Extrapolation of cell-type results to brain regions reveals important patterns for schizophrenia with distinct subregions in the hippocampus and amygdala exhibiting the highest significance. Cerebral cortical regions display similar enrichments despite the known prefrontal dysfunction in those with schizophrenia highlighting the importance of subcortical connectivity. Using functional MRI connectivity from cases with schizophrenia and neurotypical controls, we identified brain networks that distinguished cases from controls that also confirmed involvement of the central and lateral amygdala, hippocampal body, and prefrontal cortex. Our findings underscore the value of single-cell transcriptomics in decoding the polygenicity of psychiatric disorders and offer a promising convergence of genomic, transcriptomic, and brain imaging modalities toward common biological targets.\nDistinct impact modes of polygenic disposition to dyslexia in the adult brain\nAuthors: Soheili-Nezhad, S.; Schijven, D.; Mars, R. B.; Fisher, S. E.; Francks, C.\nScore: 7.6, Published: 2024-02-15 DOI: 10.1101/2024.02.14.24302815\nDyslexia is a common condition that impacts reading ability. Identifying affected brain networks has been hampered by limited sample sizes of imaging case-control studies. We focused instead on brain structural correlates of genetic disposition to dyslexia in large-scale population data. In over 30,000 adults (UK Biobank), higher polygenic disposition to dyslexia was associated with lower head and brain size, and especially reduced volume and/or altered fiber density in networks involved in motor control, language and vision. However, individual genetic variants disposing to dyslexia often had quite distinct patterns of association with brain structural features. Independent component analysis applied to brain-wide association maps for thousands of dyslexia-disposing genetic variants revealed multiple impact modes on the brain, that corresponded to anatomically distinct areas with their own genomic profiles of association. Polygenic scores for dyslexia-related cognitive and educational measures, as well as attention-deficit/hyperactivity disorder, showed similarities to dyslexia polygenic disposition in terms of brain-wide associations, with microstructure of the internal capsule consistently implicated. In contrast, lower volume of the primary motor cortex was only associated with higher dyslexia polygenic disposition among all traits. These findings robustly reveal heterogeneous neurobiological aspects of dyslexia genetic disposition, and whether they are shared or unique with respect to other genetically correlated traits.\nCharacterization of the common genetic variation in the Spanish population of Navarre\nAuthors: Maillo, A.; Huergo, E.; Apellaniz-Ruiz, M.; Urrutia, E.; Miranda, M.; Salgado, J.; Pasalodos-Sanchez, S.; Delgado-Mora, L.; Teijido, O.; Carmona, R.; Perez-Florido, J.; Aquino, V.; Lopez-Lopez, D.; Pena-Chilet, M.; Beltran, S.; Dopazo, J.; Lasa, I.; Beloqui, J. J.; Alonso, A.; Gomez-Cabrero, D.\nScore: 6.0, Published: 2024-02-13 DOI: 10.1101/2024.02.11.24302643\nLarge-scale genomic studies have significantly increased our knowledge of genetic variability across populations. Regional genetic profiling is essential for distinguishing common benign variants from those associated with disease. To this end, we conducted a comprehensive characterization of variants in the population of Navarre (Spain), utilizing whole genome sequencing data from 358 unrelated individuals of Spanish origin. Our analysis revealed 61,410 biallelic single nucleotide variants (SNV) within the Navarrese cohort, with 35% classified as common using a minor allele frequency (MAF) \u003e 1%. By comparing allele frequency data from 1000 Genome Project Phase 3 (excluding the Iberian cohort of Spain, IBS), Genome Aggregation Database, and a Spanish cohort including IBS individuals as well as data from Medical Genome Project, we identified 1,069 SNVs common in Navarre but rare (MAF [\u0026le;] 1%) in all other populations. This observation was further corroborated by a second regional cohort of 239 unrelated exomes, which confirmed 676 of the 1,069 SNVs as common in Navarre. In conclusion, this study highlights the importance of population-specific characterization of genetic variation to improve allele frequency filtering in sequencing data analysis to identify disease-causing variants.\nFeature pre-selection for the development of epigenetic biomarkers\nAuthors: Cheng, Y.; Gieger, C.; Campbell, A.; McIntosh, A. M.; Waldenberger, M.; McCartney, D. L.; Marioni, R. E.; Vallejos, C. A.\nScore: 4.7, Published: 2024-02-15 DOI: 10.1101/2024.02.14.24302694\nOver the last decade, a plethora of blood-based DNA methylation biomarkers have been developed to track differences in ageing, lifestyle, health, and biological outcomes. Typically, penalised regression models are used to generate these predictors, with hundreds or thousands of CpGs included as potential features. However, in such ultra high-dimensional settings, the effectiveness of these methods may be reduced. Here, we introduce Related Trait-based Feature Screening (RTFS), a method for performing CpG pre-selection for incident disease prediction models by utilising associations between CpGs and health-related continuous traits. In a comparison with commonly used CpG pre-selection methods, we evaluate resulting downstream Cox proportional-hazards prediction models for 10-year type 2 diabetes (T2D) onset risk in Generation Scotland (n=18,414). The top performing models utilised incident T2D EWAS (AUC=0.881, PRAUC=0.279) and RTFS (AUC=0.877, PRAUC=0.277). The resulting models also improve prediction over a model using standard risk factors only (AUC=0.841, PRAUC=0.194) and replication was observed in the German-based KORA study (n=4,261) RTFS is a flexible and generalisable framework that can help to refine biomarker development for incident disease outcomes.\nDetecting missed diagnoses of spinal muscular atrophy in genome, exome, and panel sequencing datasets\nAuthors: Weisburd, B.; Sharma, R.; Pata, V.; Reimand, T.; Ganesh, V. S.; Austin-Tse, C.; Osei-Owusu, I.; O'Heir, E.; O'Leary, M.; Pais, L.; Stafki, S. A.; Daugherty, A. L.; Bonnemann, C. G.; Donkervoort, S.; Haliloglu, G.; Kang, P. B.; Ravenscroft, G.; Laing, N.; Scott, H. S.; Topf, A.; Straub, V.; Pajusalu, S.; Ounap, K.; Tiao, G.; Rehm, H. L.; O'Donnell-Luria, A.\nScore: 1.5, Published: 2024-02-13 DOI: 10.1101/2024.02.11.24302646\nSpinal muscular atrophy (SMA) is a genetic disorder that causes progressive degeneration of lower motor neurons and the subsequent loss of muscle function throughout the body. It is the second most common recessive disorder in individuals of European descent and is present in all populations. Accurate tools exist for diagnosing SMA from short read and long read genome sequencing data. However, there are no publicly available tools for GRCh38-aligned data from panel or exome sequencing assays which continue to be used as first line tests for neuromuscular disorders. We therefore developed and extensively validated a new tool - SMA Finder - that can diagnose SMA not only in genome, but also exome and targeted sequencing samples aligned to GRCh37, GRCh38, or T2T-CHM13. It works by evaluating aligned reads that overlap the c.840 position of SMN1 and SMN2 in order to detect the most common molecular causes of SMA. We applied SMA Finder to 16,626 exomes and 3,911 genomes from heterogeneous rare disease cohorts sequenced at the Broad Institute Center for Mendelian Genomics as well as 1,157 exomes and 8,766 targeted sequencing samples from Tartu University Hospital. SMA Finder correctly identified all 16 known SMA cases and reported nine novel diagnoses which have since been confirmed by clinical testing, with another four novel diagnoses undergoing validation. Notably, out of the 29 total SMA positive cases, 21 had an initial clinical diagnosis of muscular dystrophy, congenital myasthenic syndrome, or congenital myopathy. This underscored the frequency with which SMA can be misdiagnosed as other neuromuscular disorders and confirmed the utility of using SMA Finder to reanalyze phenotypically diverse neuromuscular disease cohorts. Finally, we evaluated SMA Finder on 198,868 individuals that had both exome and genome sequencing data within the UK Biobank (UKBB) and found that SMA Finders overall false positive rate was less than 1 / 200,000 exome samples, and its positive predictive value (PPV) was 96%. We also observed 100% concordance between UKBB exome and genome calls. This analysis showed that, even though it is located within a segmental duplication, the most common causal variant for SMA can be detected with comparable accuracy to monogenic disease variants in non-repetitive regions. Additionally, the high PPV demonstrated by SMA Finder, the existence of treatment options for SMA in which early diagnosis is imperative for therapeutic benefit, as well as widespread availability of clinical confirmatory testing for SMA, may warrant the addition of SMN1 to the ACMG list of genes with reportable secondary findings after genome and exome sequencing.\n",
  "wordCount" : "2874",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.24302375">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.24302375" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.24302375">
        <p class="paperTitle">A rare splice-site variant in cardiac troponin-T (TNNT2): The need for ancestral diversity in genomic reference datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.24302375" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.24302375" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Butters, A.; Thomson, K.; Harrington, F.; Henden, N.; McGuire, K.; Byrne, A.; Bryen, S.; Leask, M.; Ackerman, M. J.; Atherton, J. J.; Bos, J. M.; Caleshu, C.; Parikh, V. N.; Day, S.; Tardiff, J. C.; Dunn, K.; Hayes, I.; Juang, J.; McGaughran, J.; Nowak, N.; Ronan, A.; Semsarian, C.; Tiemensma, M.; Merriman, T.; Skinner, J. R.; MacArthur, D. G.; Siggs, O. M.; Bagnall, R. D.; Ingles, J.</p>
        <p class="info">Score: 28.4, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.24302375' target='https://doi.org/10.1101/2024.02.08.24302375'> 10.1101/2024.02.08.24302375</a></p>
        <p class="abstract">The underrepresentation of different ancestry groups in large genomic datasets creates difficulties in interpreting the pathogenicity of monogenic variants. Genetic testing for individuals with non-European ancestry results in higher rates of uncertain variants and a greater risk of misclassification. We report a rare variant in the cardiac troponin T gene, TNNT2; NM_001001430.3: c.571-1G&gt;A (rs483352835) identified via research-based whole exome sequencing in two unrelated probands of Oceanian ancestry with cardiac phenotypes. The variant disrupts the canonical splice acceptor site, activating a cryptic acceptor and resulting in an in-frame deletion (p.Gln191del). The variant is rare in gnomAD v4.0.0 (13/780,762; 0.002%), with the highest frequency in South Asians (5/74,486; 0.007%) and has 16 ClinVar assertions (13 diagnostic clinical laboratories classify as variant of uncertain significance). There are at least 28 reported cases, many with Oceanian ancestry and diverse cardiac phenotypes. Indeed, among Oceanian-ancestry-matched datasets, the allele frequency ranges from 2.9-8.8%, and is present in 2/4 (50%) Indigenous Australian alleles in Genome Asia 100K, with one participant being homozygous. With Oceanians deriving greater than 3% of their DNA from archaic genomes, we found c.571-1G&gt;A in Vindija and Altai Neanderthal, but not the Altai Denisovan, suggesting an origin post Neanderthal divergence from modern humans 130-145 thousand years ago. Based on these data, we classify this variant as benign, and conclude it is not a monogenic cause of disease. Even with ongoing efforts to increase representation in genomics, we highlight the need for caution in assuming rarity of genetic variants in largely European datasets. Efforts to enhance diversity in genomic databases remain crucial.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.24301497">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.24301497" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.24301497">
        <p class="paperTitle">Novel syndromic neurodevelopmental disorder caused by de novo deletion of CHASERR, a long noncoding RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.24301497" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.24301497" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ganesh, V. S.; Riquin, K.; Chatron, N.; Lamar, K.-M.; Aziz, M.; Monin, P.; O&#39;Leary, M.; Goodrich, J. K.; Garimella, K. V.; England, E.; Yoon, E.; Weisburd, B.; Aguet, F.; Bacino, C. A.; Murdock, D. R.; Dai, H.; Rosenfeld, J. A.; Emrick, L. T.; Ketkar, S.; Sarusi, Y.; Sanlaville, D.; Kayani, S.; Broadbent, B.; Isidor, B.; Pengam, A.; Cogne, B.; MacArthur, D. G.; Ulitsky, I.; Carvill, G. L.; O&#39;Donnell-Luria, A.</p>
        <p class="info">Score: 112.7, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.24301497' target='https://doi.org/10.1101/2024.01.31.24301497'> 10.1101/2024.01.31.24301497</a></p>
        <p class="abstract">Abstract/SummaryGenes encoding long non-coding RNAs (lncRNAs) comprise a large fraction of the human genome, yet haploinsufficiency of a lncRNA has not been shown to cause a Mendelian disease. CHASERR is a highly conserved human lncRNA adjacent to CHD2-a coding gene in which de novo loss-of-function variants cause developmental and epileptic encephalopathy. Here we report three unrelated individuals each harboring an ultra-rare heterozygous de novo deletion in the CHASERR locus. We report similarities in severe developmental delay, facial dysmorphisms, and cerebral dysmyelination in these individuals, distinguishing them from the phenotypic spectrum of CHD2 haploinsufficiency. We demonstrate reduced CHASERR mRNA expression and corresponding increased CHD2 mRNA and protein in whole blood and patient-derived cell lines-specifically increased expression of the CHD2 allele in cis with the CHASERR deletion, as predicted from a prior mouse model of Chaserr haploinsufficiency. We show for the first time that de novo structural variants facilitated by Alu-mediated non-allelic homologous recombination led to deletion of a non-coding element (the lncRNA CHASERR) to cause a rare syndromic neurodevelopmental disorder. We also demonstrate that CHD2 has bidirectional dosage sensitivity in human disease. This work highlights the need to carefully evaluate other lncRNAs, particularly those upstream of genes associated with Mendelian disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.24302836">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.24302836" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.24302836">
        <p class="paperTitle">Gene Discovery and Biological Insights into Anxiety Disorders from a Multi-Ancestry Genome-wide Association Study of &gt;1.2 Million Participants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.24302836" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.24302836" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Friligkou, E.; Lokhammer, S.; Cabrera Mendoza, B.; Shen, J.; He, J.; Deiana, G.; Zanoaga, M. D.; Asgel, Z.; Pilcher, A.; Di Lascio, L.; Makharashvili, A.; Koller, D.; Tylee, D.; Pathak, G. A.; Polimanti, R.</p>
        <p class="info">Score: 10.5, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.24302836' target='https://doi.org/10.1101/2024.02.14.24302836'> 10.1101/2024.02.14.24302836</a></p>
        <p class="abstract">We leveraged information from more than 1.2 million participants to investigate the genetics of anxiety disorders across five continental ancestral groups. Ancestry-specific and cross-ancestry genome-wide association studies identified 51 anxiety-associated loci, 39 of which are novel. Additionally, polygenic risk scores derived from individuals of European descent were associated with anxiety in African, Admixed-American, and East Asian groups. The heritability of anxiety was enriched for genes expressed in the limbic system, the cerebral cortex, the cerebellum, the metencephalon, the entorhinal cortex, and the brain stem. Transcriptome- and proteome-wide analyses highlighted 115 genes associated with anxiety through brain-specific and cross-tissue regulation. We also observed global and local genetic correlations with depression, schizophrenia, and bipolar disorder and putative causal relationships with several physical health conditions. Overall, this study expands the knowledge regarding the genetic risk and pathogenesis of anxiety disorders, highlighting the importance of investigating diverse populations and integrating multi-omics information.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.24302043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.24302043" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.24302043">
        <p class="paperTitle">Differences in polygenic score distributions in European ancestry populations: implications for breast cancer risk prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.24302043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.24302043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yiangou, K.; Mavaddat, N.; Dennis, J.; Zanti, M.; Wang, Q.; Bolla, M. K.; Abubakar, M.; Ahearn, T. U.; Andrulis, I. L.; Anton-Culver, H.; Antonenkova, N. N.; Arndt, V.; Aronson, K. J.; Augustinsson, A.; Baten, A.; Behrens, S.; Bermisheva, M.; Berrington de Gonzalez, A.; Bialkowska, K.; Boddicker, N.; Bodelon, C.; Bogdanova, N. V.; Bojesen, S. E.; Brantley, K. D.; Brauch, H.; Brenner, H.; Camp, N. J.; Canzian, F.; Castelao, J. E.; Cessna, M. H.; Chang-Claude, J.; Chenevix-Trench, G.; Chung, W. K.; Collaborators, N.; Colonna, S. V.; Couch, F. J.; Cox, A.; Cross, S. S.; Czene, K.; Daly, M. B.; De</p>
        <p class="info">Score: 10.5, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.24302043' target='https://doi.org/10.1101/2024.02.12.24302043'> 10.1101/2024.02.12.24302043</a></p>
        <p class="abstract">The 313-variant polygenic risk score (PRS313) provides a promising tool for breast cancer risk prediction. However, evaluation of the PRS313 across different European populations which could influence risk estimation has not been performed. Here, we explored the distribution of PRS313 across European populations using genotype data from 94,072 females without breast cancer, of European-ancestry from 21 countries participating in the Breast Cancer Association Consortium (BCAC) and 225,105 female participants from the UK Biobank. The mean PRS313 differed markedly across European countries, being highest in south-eastern Europe and lowest in north-western Europe. Using the overall European PRS313 distribution to categorise individuals leads to overestimation and underestimation of risk in some individuals from south-eastern and north-western countries, respectively. Adjustment for principal components explained most of the observed heterogeneity in mean PRS. Country-specific PRS distributions may be used to calibrate risk categories in individuals from different countries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.24302442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.24302442" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.24302442">
        <p class="paperTitle">Comparative analysis of 136,401 Admixed Americans and 419,228 Europeans reveals ancestry-specific genetic determinants of clonal haematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.24302442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.24302442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wen, S.; Kuri-Morales, P.; Hu, F.; Nag, A.; Tachmazidou, I.; Vishnu Vardhan Deevi, S.; Taiy, H.; Smith, K.; Loesch, D. P.; Burren, O. S.; Dhindsa, R. S.; Wasilewski, S.; Alegre-Diaz, J.; Berumen, J.; Emberson, J.; Torres, J. M.; Collins, R.; Carss, K.; Wang, Q.; Petrovski, S.; Tapia-Conyer, R.; Fabre, M. A.; Harper, A. R.; Vassiliou, G.; Mitchell, J.</p>
        <p class="info">Score: 8.7, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.24302442' target='https://doi.org/10.1101/2024.02.07.24302442'> 10.1101/2024.02.07.24302442</a></p>
        <p class="abstract">The development of clonal haematopoiesis (CH), the age-related expansion of mutated haematopoietic stem cell (HSC) clones, is influenced by genetic and non-genetic factors. To date, large-scale studies of CH have focused on individuals of European descent, such that the impact of genetic ancestry on CH development remains incompletely understood. Here, we investigate this by studying CH in 136,401 admixed participants from the Mexico City Prospective Study (MCPS) and 419,228 European participants from the UK Biobank (UKB). We observe that CH was significantly less common in MCPS compared to UKB (adjusted odds ratio (OR) = 0.56 [95% Cl = 0.55-0.59], P = 1.60 x 10-206), a difference that persisted when comparing MCPS participants whose genomes were &gt;50% ancestrally Indigenous American to those whose genomes were &gt;50% ancestrally European (adjusted OR = 0.76 [0.70-0.83], P = 1.78 x 10-10). Genome- and exome-wide association analyses in MCPS participants identified two novel loci associated with CH (CSGALNACT1 and DIAPH3), and ancestry-specific variants in the TCL1B locus with opposing effect on DNMT3A-versus non-DNMT3A-CH. Meta-analysis of the MCPS and UKB cohorts identified another five novel loci associated with overall or gene specific CH, including polymorphisms at PAPR11/CCND2, MEIS1 and UBE2G1/SPNS3. Our CH study, the largest in a non-European population to date, demonstrates the profound impact of ancestry on CH development and reveals the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis and advance health equity amongst different human populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.24301478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.24301478" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.24301478">
        <p class="paperTitle">Connecting genomic results for psychiatric disorders to human brain cell types and regions reveals convergence with functional connectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.24301478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.24301478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yao, S.; Harder, A.; Darki, F.; Chang, Y.-W.; Li, A.; Nikouei, K.; Volpe, G.; Lundstrom, J. N.; Zeng, J.; Wray, N. R.; Lu, Y.; Sullivan, P. F.; Hjerling-Leffler, J.</p>
        <p class="info">Score: 8.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.24301478' target='https://doi.org/10.1101/2024.01.18.24301478'> 10.1101/2024.01.18.24301478</a></p>
        <p class="abstract">Understanding the temporal and spatial brain locations etiological for psychiatric disorders is essential for targeted neurobiological research. Integration of genomic insights from genome-wide association studies with single-cell transcriptomics is a powerful approach although past efforts have necessarily relied on mouse atlases. Leveraging a comprehensive atlas of the adult human brain, we prioritized cell types via the enrichment of SNP-heritabilities for brain diseases, disorders, and traits, progressing from individual cell types to brain regions. Our findings highlight specific neuronal clusters significantly enriched for the SNP-heritabilities for schizophrenia, bipolar disorder, and major depressive disorder along with intelligence, education, and neuroticism. Extrapolation of cell-type results to brain regions reveals important patterns for schizophrenia with distinct subregions in the hippocampus and amygdala exhibiting the highest significance. Cerebral cortical regions display similar enrichments despite the known prefrontal dysfunction in those with schizophrenia highlighting the importance of subcortical connectivity. Using functional MRI connectivity from cases with schizophrenia and neurotypical controls, we identified brain networks that distinguished cases from controls that also confirmed involvement of the central and lateral amygdala, hippocampal body, and prefrontal cortex. Our findings underscore the value of single-cell transcriptomics in decoding the polygenicity of psychiatric disorders and offer a promising convergence of genomic, transcriptomic, and brain imaging modalities toward common biological targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.24302815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.24302815" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.24302815">
        <p class="paperTitle">Distinct impact modes of polygenic disposition to dyslexia in the adult brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.24302815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.24302815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soheili-Nezhad, S.; Schijven, D.; Mars, R. B.; Fisher, S. E.; Francks, C.</p>
        <p class="info">Score: 7.6, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.24302815' target='https://doi.org/10.1101/2024.02.14.24302815'> 10.1101/2024.02.14.24302815</a></p>
        <p class="abstract">Dyslexia is a common condition that impacts reading ability. Identifying affected brain networks has been hampered by limited sample sizes of imaging case-control studies. We focused instead on brain structural correlates of genetic disposition to dyslexia in large-scale population data. In over 30,000 adults (UK Biobank), higher polygenic disposition to dyslexia was associated with lower head and brain size, and especially reduced volume and/or altered fiber density in networks involved in motor control, language and vision. However, individual genetic variants disposing to dyslexia often had quite distinct patterns of association with brain structural features. Independent component analysis applied to brain-wide association maps for thousands of dyslexia-disposing genetic variants revealed multiple impact modes on the brain, that corresponded to anatomically distinct areas with their own genomic profiles of association. Polygenic scores for dyslexia-related cognitive and educational measures, as well as attention-deficit/hyperactivity disorder, showed similarities to dyslexia polygenic disposition in terms of brain-wide associations, with microstructure of the internal capsule consistently implicated. In contrast, lower volume of the primary motor cortex was only associated with higher dyslexia polygenic disposition among all traits. These findings robustly reveal heterogeneous neurobiological aspects of dyslexia genetic disposition, and whether they are shared or unique with respect to other genetically correlated traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.24302643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.24302643" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.24302643">
        <p class="paperTitle">Characterization of the common genetic variation in the Spanish population of Navarre</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.24302643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.24302643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maillo, A.; Huergo, E.; Apellaniz-Ruiz, M.; Urrutia, E.; Miranda, M.; Salgado, J.; Pasalodos-Sanchez, S.; Delgado-Mora, L.; Teijido, O.; Carmona, R.; Perez-Florido, J.; Aquino, V.; Lopez-Lopez, D.; Pena-Chilet, M.; Beltran, S.; Dopazo, J.; Lasa, I.; Beloqui, J. J.; Alonso, A.; Gomez-Cabrero, D.</p>
        <p class="info">Score: 6.0, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.24302643' target='https://doi.org/10.1101/2024.02.11.24302643'> 10.1101/2024.02.11.24302643</a></p>
        <p class="abstract">Large-scale genomic studies have significantly increased our knowledge of genetic variability across populations. Regional genetic profiling is essential for distinguishing common benign variants from those associated with disease. To this end, we conducted a comprehensive characterization of variants in the population of Navarre (Spain), utilizing whole genome sequencing data from 358 unrelated individuals of Spanish origin. Our analysis revealed 61,410 biallelic single nucleotide variants (SNV) within the Navarrese cohort, with 35% classified as common using a minor allele frequency (MAF) &gt; 1%. By comparing allele frequency data from 1000 Genome Project Phase 3 (excluding the Iberian cohort of Spain, IBS), Genome Aggregation Database, and a Spanish cohort including IBS individuals as well as data from Medical Genome Project, we identified 1,069 SNVs common in Navarre but rare (MAF [&amp;le;] 1%) in all other populations. This observation was further corroborated by a second regional cohort of 239 unrelated exomes, which confirmed 676 of the 1,069 SNVs as common in Navarre. In conclusion, this study highlights the importance of population-specific characterization of genetic variation to improve allele frequency filtering in sequencing data analysis to identify disease-causing variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.24302694">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.24302694" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.24302694">
        <p class="paperTitle">Feature pre-selection for the development of epigenetic biomarkers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.24302694" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.24302694" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cheng, Y.; Gieger, C.; Campbell, A.; McIntosh, A. M.; Waldenberger, M.; McCartney, D. L.; Marioni, R. E.; Vallejos, C. A.</p>
        <p class="info">Score: 4.7, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.24302694' target='https://doi.org/10.1101/2024.02.14.24302694'> 10.1101/2024.02.14.24302694</a></p>
        <p class="abstract">Over the last decade, a plethora of blood-based DNA methylation biomarkers have been developed to track differences in ageing, lifestyle, health, and biological outcomes. Typically, penalised regression models are used to generate these predictors, with hundreds or thousands of CpGs included as potential features. However, in such ultra high-dimensional settings, the effectiveness of these methods may be reduced.

Here, we introduce Related Trait-based Feature Screening (RTFS), a method for performing CpG pre-selection for incident disease prediction models by utilising associations between CpGs and health-related continuous traits. In a comparison with commonly used CpG pre-selection methods, we evaluate resulting downstream Cox proportional-hazards prediction models for 10-year type 2 diabetes (T2D) onset risk in Generation Scotland (n=18,414). The top performing models utilised incident T2D EWAS (AUC=0.881, PRAUC=0.279) and RTFS (AUC=0.877, PRAUC=0.277). The resulting models also improve prediction over a model using standard risk factors only (AUC=0.841, PRAUC=0.194) and replication was observed in the German-based KORA study (n=4,261)

RTFS is a flexible and generalisable framework that can help to refine biomarker development for incident disease outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.24302646">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.24302646" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.24302646">
        <p class="paperTitle">Detecting missed diagnoses of spinal muscular atrophy in genome, exome, and panel sequencing datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.24302646" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.24302646" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weisburd, B.; Sharma, R.; Pata, V.; Reimand, T.; Ganesh, V. S.; Austin-Tse, C.; Osei-Owusu, I.; O&#39;Heir, E.; O&#39;Leary, M.; Pais, L.; Stafki, S. A.; Daugherty, A. L.; Bonnemann, C. G.; Donkervoort, S.; Haliloglu, G.; Kang, P. B.; Ravenscroft, G.; Laing, N.; Scott, H. S.; Topf, A.; Straub, V.; Pajusalu, S.; Ounap, K.; Tiao, G.; Rehm, H. L.; O&#39;Donnell-Luria, A.</p>
        <p class="info">Score: 1.5, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.24302646' target='https://doi.org/10.1101/2024.02.11.24302646'> 10.1101/2024.02.11.24302646</a></p>
        <p class="abstract">Spinal muscular atrophy (SMA) is a genetic disorder that causes progressive degeneration of lower motor neurons and the subsequent loss of muscle function throughout the body. It is the second most common recessive disorder in individuals of European descent and is present in all populations. Accurate tools exist for diagnosing SMA from short read and long read genome sequencing data. However, there are no publicly available tools for GRCh38-aligned data from panel or exome sequencing assays which continue to be used as first line tests for neuromuscular disorders. We therefore developed and extensively validated a new tool - SMA Finder - that can diagnose SMA not only in genome, but also exome and targeted sequencing samples aligned to GRCh37, GRCh38, or T2T-CHM13. It works by evaluating aligned reads that overlap the c.840 position of SMN1 and SMN2 in order to detect the most common molecular causes of SMA. We applied SMA Finder to 16,626 exomes and 3,911 genomes from heterogeneous rare disease cohorts sequenced at the Broad Institute Center for Mendelian Genomics as well as 1,157 exomes and 8,766 targeted sequencing samples from Tartu University Hospital. SMA Finder correctly identified all 16 known SMA cases and reported nine novel diagnoses which have since been confirmed by clinical testing, with another four novel diagnoses undergoing validation. Notably, out of the 29 total SMA positive cases, 21 had an initial clinical diagnosis of muscular dystrophy, congenital myasthenic syndrome, or congenital myopathy. This underscored the frequency with which SMA can be misdiagnosed as other neuromuscular disorders and confirmed the utility of using SMA Finder to reanalyze phenotypically diverse neuromuscular disease cohorts. Finally, we evaluated SMA Finder on 198,868 individuals that had both exome and genome sequencing data within the UK Biobank (UKBB) and found that SMA Finders overall false positive rate was less than 1 / 200,000 exome samples, and its positive predictive value (PPV) was 96%. We also observed 100% concordance between UKBB exome and genome calls. This analysis showed that, even though it is located within a segmental duplication, the most common causal variant for SMA can be detected with comparable accuracy to monogenic disease variants in non-repetitive regions. Additionally, the high PPV demonstrated by SMA Finder, the existence of treatment options for SMA in which early diagnosis is imperative for therapeutic benefit, as well as widespread availability of clinical confirmatory testing for SMA, may warrant the addition of SMN1 to the ACMG list of genes with reportable secondary findings after genome and exome sequencing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
