<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Distinct explanations underlie gene-environment interactions in the UK Biobank
Authors: Durvasula, A.; Price, A.
Score: 22.7, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969
The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Distinct explanations underlie gene-environment interactions in the UK Biobank
Authors: Durvasula, A.; Price, A.
Score: 22.7, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969
The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Distinct explanations underlie gene-environment interactions in the UK Biobank
Authors: Durvasula, A.; Price, A.
Score: 22.7, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969
The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Distinct explanations underlie gene-environment interactions in the UK Biobank\nAuthors: Durvasula, A.; Price, A.\nScore: 22.7, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969\nThe role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) \u0026lt; 1 across E bins.",
  "keywords": [
    
  ],
  "articleBody": " Distinct explanations underlie gene-environment interactions in the UK Biobank\nAuthors: Durvasula, A.; Price, A.\nScore: 22.7, Published: 2023-09-23 DOI: 10.1101/2023.09.22.23295969\nThe role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) \u003c 1 across E bins. Second, we detect genome-wide effects of the E variable on genetic variance by leveraging polygenic risk scores (PRS) to test for significant PRSxE in a regression of phenotypes on PRS, E, and PRSxE, together with differences in SNP-heritability across E bins. Third, we detect genome-wide proportional amplification of genetic and environmental effects as a function of the E variable by testing for significant PRSxE with no differences in SNP-heritability across E bins. Simulations show that these approaches achieve high sensitivity and specificity in distinguishing these three GxE scenarios. We applied our framework to 33 UK Biobank diseases/traits (average N=325K) and 10 E variables spanning lifestyle, diet, and other environmental exposures. First, we identified 19 trait-E pairs with rg significantly \u003c 1 (FDR\u003c5%) (average rg=0.95); for example, white blood cell count had rg=0.95 (s.e. 0.01) between smokers and non-smokers. Second, we identified 28 trait-E pairs with significant PRSxE and significant SNP-heritability differences across E bins; for example, type 2 diabetes had a significant PRSxE for alcohol consumption (P=1e-13) with 4.2x larger SNP-heritability in the largest versus smallest quintiles of alcohol consumption (P\u003c1e-16). Third, we identified 15 trait-E pairs with significant PRSxE with no SNP-heritability differences across E bins; for example, triglyceride levels had a significant PRSxE effect for composite diet score (P=4e-5) with no SNP-heritability differences. Analyses using biological sex as the E variable produced additional significant findings in each of the three scenarios. Overall, we infer a substantial contribution of GxE and GxSex effects to disease and complex trait variance.\nJoint genotypic and phenotypic outcome modeling improves base editing variant effect quantification\nAuthors: Ryu, J.; Barkal, S.; Yu, T.; Jankowiak, M.; Zhou, Y.; Francoeur, M.; Phan, Q. V.; Li, Z.; Tognon, M.; Brown, L.; Love, M. I.; Lettre, G.; Ascher, D.; Cassa, C. A.; Sherwood, R. I.; Pinello, L.\nScore: 16.0, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295253\nCRISPR base editing screens are powerful tools for studying disease-associated variants at scale. However, the efficiency and precision of base editing perturbations vary, confounding the assessment of variant-induced phenotypic effects. Here, we provide an integrated pipeline that improves the estimation of variant impact in base editing screens. We perform high-throughput ABE8e-SpRY base editing screens with an integrated reporter construct to measure the editing efficiency and outcomes of each gRNA alongside their phenotypic consequences. We introduce BEAN, a Bayesian network that accounts for per-guide editing outcomes and target site chromatin accessibility to estimate variant impacts. We show this pipeline attains superior performance compared to existing tools in variant classification and effect size quantification. We use BEAN to pinpoint common variants that alter LDL uptake, implicating novel genes. Additionally, through saturation base editing of LDLR, we enable accurate quantitative prediction of the effects of missense variants on LDL-C levels, which aligns with measurements in UK Biobank individuals, and identify structural mechanisms underlying variant pathogenicity. This work provides a widely applicable approach to improve the power of base editor screens for disease-associated variant characterization.\nGenetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences\nAuthors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.\nScore: 13.7, Published: 2023-09-19 DOI: 10.1101/2023.09.18.23295715\nTelomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing. We developed a joint telomere length (TL) metric, combining both qPCR and whole genome sequencing (WGS) measurements across 462,675 UK Biobank participants that increased our ability to capture TL heritability by 36% (h2mean=0.058 to h2combined=0.079) and improved predictions of age. Exome-wide rare variant (minor allele frequency\u003c0.001) and gene-level collapsing association studies identified 53 variants and 22 genes significantly associated with TL that included allelic series in ACD and RTEL1. Five of the 31 rare-variant TL associated genes (16%) were also known drivers of clonal haematopoiesis (CH), prompting somatic variant analyses. Stratifying by CH clone size, we uncovered novel gene-specific associations with TL, including lengthened telomeres in individuals with large SRSF2-mutant clones, in contrast to the progressive telomere shortening observed with increasing clonal expansions driven by other CH genes. Our findings demonstrate the impact of rare variants on TL with larger effects in genes associated with CH, a precursor of myeloid cancers and several other non-malignant human diseases. Telomere biology is likely to be an important focus for the prevention and treatment of these conditions.\nSystematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease\nAuthors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D'Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.\nScore: 40.2, Published: 2023-09-12 DOI: 10.1101/2023.09.12.23295416\nBackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown. MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls. ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations. ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.\nGenetics of NLRP3 suggests lack of involvement and inefficient druggability in Parkinson's disease\nAuthors: Senkevich, K.; Liu, L.; Alvarado, C. X.; Leonard, H. L.; Nalls, M. A.; Gan-Or, Z.\nScore: 6.1, Published: 2023-09-23 DOI: 10.1101/2023.09.20.23295790\nActivation of the NLRP3-inflammasome has been proposed to play a role in Parkinsons disease pathogenesis based on in vitro and in vivo studies. Currently, clinical trials targeting the NLRP3 pathway in Parkinsons disease are at early stages. However, the evidence supporting NLRP3s involvement in Parkinsons disease from human genetics data remains limited. In this study, we conducted comprehensive analyses of common and rare variants in genes related to the NLRP3-inflammasome in large Parkinsons disease cohorts. Furthermore, we performed pathway-specific analyses using polygenic risk scores and studied potential causal associations using Mendelian randomization with the NLRP3 components and the cytokines released by its activation, IL-1{beta} and IL-18. Our findings showed no associations of common or rare variants, nor of the pathway polygenic risk score for the NLRP3 inflammasome, with risk of Parkinsons disease. Mendelian randomization analyses suggest that altering the expression of the NLRP3 inflammasome, IL-1{beta} or IL-18, is not likely to affect Parkinsons disease risk, age-at-onset, or progression. Therefore, our results do not support an important role for the NLRP3 inflammasome in Parkinsons disease pathogenesis or as a strong target for drug development.\nParSE-seq: A Calibrated Multiplexed Assay to Facilitate the Clinical Classification of Putative Splice-altering Variants\nAuthors: O'Neill, M. J.; Yang, T.; Laudeman, J.; Calandranis, M.; Solus, J.; Roden, D. M.; Glazer, A. M.\nScore: 5.0, Published: 2023-09-08 DOI: 10.1101/2023.09.04.23295019\nBackgroundInterpreting the clinical significance of putative splice-altering variants outside 2-base pair canonical splice sites remains difficult without functional studies. MethodsWe developed Parallel Splice Effect Sequencing (ParSE-seq), a multiplexed minigene-based assay, to test variant effects on RNA splicing quantified by high-throughput sequencing. We studied variants in SCN5A, an arrhythmia-associated gene which encodes the major cardiac voltage-gated sodium channel. We used the computational tool SpliceAI to prioritize exonic and intronic candidate splice variants, and ClinVar to select benign and pathogenic control variants. We generated a pool of 284 barcoded minigene plasmids, transfected them into Human Embryonic Kidney (HEK293) cells and induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs), sequenced the resulting pools of splicing products, and calibrated the assay to the American College of Medical Genetics and Genomics scheme. Variants were interpreted using the calibrated functional data, and experimental data were compared to SpliceAI predictions. We further studied some splice-altering missense variants by cDNA-based automated patch clamping (APC) in HEK cells and assessed splicing and sodium channel function in CRISPR-edited iPSC-CMs. ResultsParSE-seq revealed the splicing effect of 224 SCN5A variants in iPSC-CMs and 244 variants in HEK293 cells. The scores between the cell types were highly correlated (R2=0.84). In iPSCs, the assay had concordant scores for 21/22 benign/likely benign and 24/25 pathogenic/likely pathogenic control variants from ClinVar. 43/112 exonic variants and 35/70 intronic variants with determinate scores disrupted splicing. 11 of 42 variants of uncertain significance were reclassified, and 29 of 34 variants with conflicting interpretations were reclassified using the functional data. SpliceAI computational predictions correlated well with experimental data (AUC = 0.96). We identified 20 unique SCN5A missense variants that disrupted splicing, and 2 clinically observed splice-altering missense variants of uncertain significance had normal function when tested with the cDNA-based APC assay. A splice-altering intronic variant detected by ParSE-seq, c.1891-5C\u003eG, also disrupted splicing and sodium current when introduced into iPSC-CMs at the endogenous locus by CRISPR editing. ConclusionsParSE-seq is a calibrated, multiplexed, high-throughput assay to facilitate the classification of candidate splice-altering variants.\nClinical actionability of genetic findings in cerebral palsy\nAuthors: Lewis, S. A.; Chopra, M.; Cohen, J. S.; Bain, J.; Aravamuthan, B. R.; Carmel, J. B.; Fahey, M. C.; Segel, R.; Wintle, R. F.; Zech, M.; May, H.; Haque, N.; Fehlings, D.; Srivastava, S.; Kruer, M. C.\nScore: 10.9, Published: 2023-09-11 DOI: 10.1101/2023.09.08.23295195\nBackground and objectivesSingle gene mutations are increasingly recognized as causes of cerebral palsy (CP) phenotypes, yet there is currently no standardized framework for measuring their clinical impact. We evaluated Pathogenic/Likely Pathogenic (P/LP) variants identified in individuals with CP to determine how frequently genetic testing results would prompt changes in care. MethodsWe analyzed published P/LP variants in OMIM genes identified in clinical (n = 1,345 individuals) or research (n = 496) cohorts using exome sequencing of CP patients. We established a working group of clinical and research geneticists, developmental pediatricians, genetic counselors, and neurologists and performed a systematic review of existing literature for evidence of clinical management approaches linked to genetic disorders. Scoring rubrics were adapted, and a modified Delphi approach was used to build consensus and establish the anticipated impact on patient care. Overall clinical utility was calculated from metrics assessing outcome severity if left untreated, safety/practicality of the intervention, and anticipated intervention efficacy. ResultsWe found 140/1,841 (8%) of individuals in published CP cohorts had a genetic diagnosis classified as actionable, defined as prompting a change in clinical management based on knowledge related to the genetic etiology. 58/243 genes with P/LP variants were classified as actionable; 16 had treatment options targeting the primary disease mechanism, 16 had specific prevention strategies, and 26 had specific symptom management recommendations. The level of evidence was also graded according to ClinGen criteria; 44.6% of interventions had evidence class \"D\" or below. The potential interventions have clinical utility with 97% of outcomes being moderate-high severity if left untreated and 62% of interventions predicted to be of moderate-high efficacy. Most interventions (71%) were considered moderate-high safety/practicality. DiscussionOur findings indicate that actionable genetic findings occur in 8% of individuals referred for genetic testing with CP. Evaluation of potential efficacy, outcome severity, and intervention safety/practicality indicates moderate-high clinical utility of these genetic findings. Thus, genetic sequencing to identify these individuals for precision medicine interventions could improve outcomes and provide clinical benefit to individuals with CP. The relatively limited evidence base for most interventions underscores the need for additional research.\nThe shared genetic risk architecture of neurological and psychiatric disorders: a genome-wide analysis\nAuthors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC), ; The International Headache Genetics Consortium (IHGC), ; Steen, N. E.; van der Meer, D.; O'Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.\nScore: 33.5, Published: 2023-09-26 DOI: 10.1101/2023.07.21.23292993\nWhile neurological and psychiatric disorders have historically been considered to reflect distinct pathogenic entities, recent findings suggest shared pathobiological mechanisms. However, the extent to which these heritable disorders share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate widespread genetic overlap across the disorders, even in the absence of genetic correlations. This indicates that a large set of common variants impact risk of multiple neurological and psychiatric disorders, but with divergent effect sizes. Furthermore, biological interrogation revealed a range of biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study indicates that neurological and psychiatric disorders share key etiological aspects, which has important implications for disease classification, precision medicine, and clinical practice.\nFrequentmers - a novel way to look at metagenomic Next Generation Sequencing data and an application in detecting liver cirrhosis\nAuthors: Mouratidis, I.; Chantzi, N.; Khan, U.; Konnaris, M. A.; Chan, C. S. Y.; Mareboina, M.; Georgakopoulos-Soares, I.\nScore: 2.8, Published: 2023-09-19 DOI: 10.1101/2023.09.19.23295771\nEarly detection of human disease is associated with improved clinical outcomes. However, many diseases are often detected at an advanced, symptomatic stage where patients are past efficacious treatment periods and can result in less favorable outcomes. Therefore, methods that can accurately detect human disease at a presymptomatic stage are urgently needed. Here, we introduce \"frequentmers\"; short sequences that are specific and recurrently observed in either patient or healthy control samples, but not in both. We showcase the utility of frequentmers for the detection of liver cirrhosis using metagenomic Next Generation Sequencing data from stool samples of patients and controls. We develop classification models for the detection of liver cirrhosis and achieve an AUC score of 0.91 using ten-fold cross-validation. A small subset of 200 frequentmers can achieve comparable results in detecting liver cirrhosis. Finally, we identify the microbial organisms in liver cirrhosis samples, which are associated with the most predictive frequentmer biomarkers.\nLandscape of blood group antigens and alleles in the Indian population from whole genome sequences\nAuthors: Rophina, M.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.\nScore: 1.2, Published: 2023-09-26 DOI: 10.1101/2023.09.26.23296145\nBlood group antigens are genetically inherited macromolecular structures which form the underlying factor for inter individual variations in human blood. Currently there exists over 390 human blood group antigens corresponding to 44 blood group systems and 2 erythroid specific transcription factors. Distribution of these blood group antigens have been found to differ significantly among various ethnic populations. To date, there is a lack of comprehensive research that offers extensive blood group profiles for the Indian population. Whole genome sequence data (hg38) of 1029 self-declared healthy Indian individuals generated as a part of the pilot phase IndiGen programme were used for the analysis. Variants spanning the genes of 44 blood group systems and two transcription factors KLF1, GATA1 were fetched and annotated for their functional consequences. Our study reports a total of 40712 blood group related variants of which 695 were identified as non-synonymous variants in the coding region. Of the total non-synonymous variants, 105 were found to have a known blood phenotype. A total of 24 variants belonging to 12 blood groups were predicted to be deleterious by more than three computational tools. Our study was also able to identify a few rare blood phenotypes including Au(a-b+), Js(a+b+), Di(a+b-), In(a+b-) and KANNO-. This study is the first to use genomic data to understand the blood group antigen profiles of the Indian population, and it also systematically compares these profiles with those of other global populations.\n",
  "wordCount" : "2807",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.23295969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.23295969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.23295969">
        <p class="paperTitle">Distinct explanations underlie gene-environment interactions in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.23295969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.23295969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durvasula, A.; Price, A.</p>
        <p class="info">Score: 22.7, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.23295969' target='https://doi.org/10.1101/2023.09.22.23295969'> 10.1101/2023.09.22.23295969</a></p>
        <p class="abstract">The role of gene-environment (GxE) interaction in disease and complex trait architectures is widely hypothesized, but currently unknown. Here, we apply three statistical approaches to quantify and distinguish three different types of GxE interaction for a given disease/trait and E variable. First, we detect locus-specific GxE interaction by testing for genetic correlation (rg) &lt; 1 across E bins. Second, we detect genome-wide effects of the E variable on genetic variance by leveraging polygenic risk scores (PRS) to test for significant PRSxE in a regression of phenotypes on PRS, E, and PRSxE, together with differences in SNP-heritability across E bins. Third, we detect genome-wide proportional amplification of genetic and environmental effects as a function of the E variable by testing for significant PRSxE with no differences in SNP-heritability across E bins. Simulations show that these approaches achieve high sensitivity and specificity in distinguishing these three GxE scenarios. We applied our framework to 33 UK Biobank diseases/traits (average N=325K) and 10 E variables spanning lifestyle, diet, and other environmental exposures. First, we identified 19 trait-E pairs with rg significantly &lt; 1 (FDR&lt;5%) (average rg=0.95); for example, white blood cell count had rg=0.95 (s.e. 0.01) between smokers and non-smokers. Second, we identified 28 trait-E pairs with significant PRSxE and significant SNP-heritability differences across E bins; for example, type 2 diabetes had a significant PRSxE for alcohol consumption (P=1e-13) with 4.2x larger SNP-heritability in the largest versus smallest quintiles of alcohol consumption (P&lt;1e-16). Third, we identified 15 trait-E pairs with significant PRSxE with no SNP-heritability differences across E bins; for example, triglyceride levels had a significant PRSxE effect for composite diet score (P=4e-5) with no SNP-heritability differences. Analyses using biological sex as the E variable produced additional significant findings in each of the three scenarios. Overall, we infer a substantial contribution of GxE and GxSex effects to disease and complex trait variance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295253">
        <p class="paperTitle">Joint genotypic and phenotypic outcome modeling improves base editing variant effect quantification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ryu, J.; Barkal, S.; Yu, T.; Jankowiak, M.; Zhou, Y.; Francoeur, M.; Phan, Q. V.; Li, Z.; Tognon, M.; Brown, L.; Love, M. I.; Lettre, G.; Ascher, D.; Cassa, C. A.; Sherwood, R. I.; Pinello, L.</p>
        <p class="info">Score: 16.0, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295253' target='https://doi.org/10.1101/2023.09.08.23295253'> 10.1101/2023.09.08.23295253</a></p>
        <p class="abstract">CRISPR base editing screens are powerful tools for studying disease-associated variants at scale. However, the efficiency and precision of base editing perturbations vary, confounding the assessment of variant-induced phenotypic effects. Here, we provide an integrated pipeline that improves the estimation of variant impact in base editing screens. We perform high-throughput ABE8e-SpRY base editing screens with an integrated reporter construct to measure the editing efficiency and outcomes of each gRNA alongside their phenotypic consequences. We introduce BEAN, a Bayesian network that accounts for per-guide editing outcomes and target site chromatin accessibility to estimate variant impacts. We show this pipeline attains superior performance compared to existing tools in variant classification and effect size quantification. We use BEAN to pinpoint common variants that alter LDL uptake, implicating novel genes. Additionally, through saturation base editing of LDLR, we enable accurate quantitative prediction of the effects of missense variants on LDL-C levels, which aligns with measurements in UK Biobank individuals, and identify structural mechanisms underlying variant pathogenicity. This work provides a widely applicable approach to improve the power of base editor screens for disease-associated variant characterization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.23295715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.23295715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.23295715">
        <p class="paperTitle">Genetic architecture of telomere length in 462,675 UK Biobank whole-genome sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.23295715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.23295715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burren, O. S.; Dhindsa, R.; Deevi, S. V. V.; Wen, S.; Nag, A.; Mitchell, J.; Hu, F.; Smith, K. R.; Razdan, N.; Olsson, H.; Platt, A.; Codd, V.; Nelson, C.; Samani, N.; March, R.; Wasilewski, S.; Carss, K.; Fabre, M.; Wang, Q.; Pangalos, M.; Petrovski, S.</p>
        <p class="info">Score: 13.7, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.23295715' target='https://doi.org/10.1101/2023.09.18.23295715'> 10.1101/2023.09.18.23295715</a></p>
        <p class="abstract">Telomeres protect the ends of chromosomes from damage, and genetic regulation of their length is associated with human disease and ageing. We developed a joint telomere length (TL) metric, combining both qPCR and whole genome sequencing (WGS) measurements across 462,675 UK Biobank participants that increased our ability to capture TL heritability by 36% (h2mean=0.058 to h2combined=0.079) and improved predictions of age. Exome-wide rare variant (minor allele frequency&lt;0.001) and gene-level collapsing association studies identified 53 variants and 22 genes significantly associated with TL that included allelic series in ACD and RTEL1. Five of the 31 rare-variant TL associated genes (16%) were also known drivers of clonal haematopoiesis (CH), prompting somatic variant analyses. Stratifying by CH clone size, we uncovered novel gene-specific associations with TL, including lengthened telomeres in individuals with large SRSF2-mutant clones, in contrast to the progressive telomere shortening observed with increasing clonal expansions driven by other CH genes. Our findings demonstrate the impact of rare variants on TL with larger effects in genes associated with CH, a precursor of myeloid cancers and several other non-malignant human diseases. Telomere biology is likely to be an important focus for the prevention and treatment of these conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295416">
        <p class="paperTitle">Systematic identification of disease-causing promoter and untranslated region variants in 8,040 undiagnosed individuals with rare disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin Geary, A. C.; Blakes, A. J.; Dawes, R.; Findlay, S. D.; Lord, J. C.; Walker, S.; Talbot-Martin, J.; Wieder, N.; D&#39;Souza, E. N.; Fernandes, M.; Hilton, S.; Lahiri, N.; Campbell, C.; Jenkinson, S.; De Goede, C. G.; Anderson, E. R.; Burge, C. B.; Sanders, S. J.; Ellingford, J.; Baralle, D.; Banka, S.; Whiffin, N.</p>
        <p class="info">Score: 40.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295416' target='https://doi.org/10.1101/2023.09.12.23295416'> 10.1101/2023.09.12.23295416</a></p>
        <p class="abstract">BackgroundBoth promoters and untranslated regions (UTRs) have critical regulatory roles, yet variants in these regions are largely excluded from clinical genetic testing due to difficulty in interpreting pathogenicity. The extent to which these regions may harbour diagnoses for individuals with rare disease is currently unknown.

MethodsWe present a framework for the identification and annotation of potentially deleterious proximal promoter and UTR variants in known dominant disease genes. We use this framework to annotate de novo variants (DNVs) in 8,040 undiagnosed individuals in the Genomics England 100,000 genomes project, which were subject to strict region-based filtering, clinical review, and validation studies where possible. In addition, we performed region and variant annotation-based burden testing in 7,862 unrelated probands against matched unaffected controls.

ResultsWe prioritised eleven DNVs and identified an additional variant overlapping one of the eleven. Ten of these twelve variants (82%) are in genes that are a strong match to the individuals phenotype and six had not previously been identified. Through burden testing, we did not observe a significant enrichment of potentially deleterious promoter and/or UTR variants in individuals with rare disease collectively across any of our region or variant annotations.

ConclusionsOverall, we demonstrate the value of screening promoters and UTRs to uncover additional diagnoses for previously undiagnosed individuals with rare disease and provide a framework for doing so without dramatically increasing interpretation burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.23295790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.23295790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.23295790">
        <p class="paperTitle">Genetics of NLRP3 suggests lack of involvement and inefficient druggability in Parkinson&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.23295790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.23295790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Senkevich, K.; Liu, L.; Alvarado, C. X.; Leonard, H. L.; Nalls, M. A.; Gan-Or, Z.</p>
        <p class="info">Score: 6.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.23295790' target='https://doi.org/10.1101/2023.09.20.23295790'> 10.1101/2023.09.20.23295790</a></p>
        <p class="abstract">Activation of the NLRP3-inflammasome has been proposed to play a role in Parkinsons disease pathogenesis based on in vitro and in vivo studies. Currently, clinical trials targeting the NLRP3 pathway in Parkinsons disease are at early stages. However, the evidence supporting NLRP3s involvement in Parkinsons disease from human genetics data remains limited. In this study, we conducted comprehensive analyses of common and rare variants in genes related to the NLRP3-inflammasome in large Parkinsons disease cohorts. Furthermore, we performed pathway-specific analyses using polygenic risk scores and studied potential causal associations using Mendelian randomization with the NLRP3 components and the cytokines released by its activation, IL-1{beta} and IL-18. Our findings showed no associations of common or rare variants, nor of the pathway polygenic risk score for the NLRP3 inflammasome, with risk of Parkinsons disease. Mendelian randomization analyses suggest that altering the expression of the NLRP3 inflammasome, IL-1{beta} or IL-18, is not likely to affect Parkinsons disease risk, age-at-onset, or progression. Therefore, our results do not support an important role for the NLRP3 inflammasome in Parkinsons disease pathogenesis or as a strong target for drug development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.23295019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.23295019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.23295019">
        <p class="paperTitle">ParSE-seq: A Calibrated Multiplexed Assay to Facilitate the Clinical Classification of Putative Splice-altering Variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.23295019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.23295019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Neill, M. J.; Yang, T.; Laudeman, J.; Calandranis, M.; Solus, J.; Roden, D. M.; Glazer, A. M.</p>
        <p class="info">Score: 5.0, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.23295019' target='https://doi.org/10.1101/2023.09.04.23295019'> 10.1101/2023.09.04.23295019</a></p>
        <p class="abstract">BackgroundInterpreting the clinical significance of putative splice-altering variants outside 2-base pair canonical splice sites remains difficult without functional studies.

MethodsWe developed Parallel Splice Effect Sequencing (ParSE-seq), a multiplexed minigene-based assay, to test variant effects on RNA splicing quantified by high-throughput sequencing. We studied variants in SCN5A, an arrhythmia-associated gene which encodes the major cardiac voltage-gated sodium channel. We used the computational tool SpliceAI to prioritize exonic and intronic candidate splice variants, and ClinVar to select benign and pathogenic control variants. We generated a pool of 284 barcoded minigene plasmids, transfected them into Human Embryonic Kidney (HEK293) cells and induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs), sequenced the resulting pools of splicing products, and calibrated the assay to the American College of Medical Genetics and Genomics scheme. Variants were interpreted using the calibrated functional data, and experimental data were compared to SpliceAI predictions. We further studied some splice-altering missense variants by cDNA-based automated patch clamping (APC) in HEK cells and assessed splicing and sodium channel function in CRISPR-edited iPSC-CMs.

ResultsParSE-seq revealed the splicing effect of 224 SCN5A variants in iPSC-CMs and 244 variants in HEK293 cells. The scores between the cell types were highly correlated (R2=0.84). In iPSCs, the assay had concordant scores for 21/22 benign/likely benign and 24/25 pathogenic/likely pathogenic control variants from ClinVar. 43/112 exonic variants and 35/70 intronic variants with determinate scores disrupted splicing. 11 of 42 variants of uncertain significance were reclassified, and 29 of 34 variants with conflicting interpretations were reclassified using the functional data. SpliceAI computational predictions correlated well with experimental data (AUC = 0.96). We identified 20 unique SCN5A missense variants that disrupted splicing, and 2 clinically observed splice-altering missense variants of uncertain significance had normal function when tested with the cDNA-based APC assay. A splice-altering intronic variant detected by ParSE-seq, c.1891-5C&gt;G, also disrupted splicing and sodium current when introduced into iPSC-CMs at the endogenous locus by CRISPR editing.

ConclusionsParSE-seq is a calibrated, multiplexed, high-throughput assay to facilitate the classification of candidate splice-altering variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295195">
        <p class="paperTitle">Clinical actionability of genetic findings in cerebral palsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lewis, S. A.; Chopra, M.; Cohen, J. S.; Bain, J.; Aravamuthan, B. R.; Carmel, J. B.; Fahey, M. C.; Segel, R.; Wintle, R. F.; Zech, M.; May, H.; Haque, N.; Fehlings, D.; Srivastava, S.; Kruer, M. C.</p>
        <p class="info">Score: 10.9, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295195' target='https://doi.org/10.1101/2023.09.08.23295195'> 10.1101/2023.09.08.23295195</a></p>
        <p class="abstract">Background and objectivesSingle gene mutations are increasingly recognized as causes of cerebral palsy (CP) phenotypes, yet there is currently no standardized framework for measuring their clinical impact. We evaluated Pathogenic/Likely Pathogenic (P/LP) variants identified in individuals with CP to determine how frequently genetic testing results would prompt changes in care.

MethodsWe analyzed published P/LP variants in OMIM genes identified in clinical (n = 1,345 individuals) or research (n = 496) cohorts using exome sequencing of CP patients. We established a working group of clinical and research geneticists, developmental pediatricians, genetic counselors, and neurologists and performed a systematic review of existing literature for evidence of clinical management approaches linked to genetic disorders. Scoring rubrics were adapted, and a modified Delphi approach was used to build consensus and establish the anticipated impact on patient care. Overall clinical utility was calculated from metrics assessing outcome severity if left untreated, safety/practicality of the intervention, and anticipated intervention efficacy.

ResultsWe found 140/1,841 (8%) of individuals in published CP cohorts had a genetic diagnosis classified as actionable, defined as prompting a change in clinical management based on knowledge related to the genetic etiology. 58/243 genes with P/LP variants were classified as actionable; 16 had treatment options targeting the primary disease mechanism, 16 had specific prevention strategies, and 26 had specific symptom management recommendations. The level of evidence was also graded according to ClinGen criteria; 44.6% of interventions had evidence class &#34;D&#34; or below. The potential interventions have clinical utility with 97% of outcomes being moderate-high severity if left untreated and 62% of interventions predicted to be of moderate-high efficacy. Most interventions (71%) were considered moderate-high safety/practicality.

DiscussionOur findings indicate that actionable genetic findings occur in 8% of individuals referred for genetic testing with CP. Evaluation of potential efficacy, outcome severity, and intervention safety/practicality indicates moderate-high clinical utility of these genetic findings. Thus, genetic sequencing to identify these individuals for precision medicine interventions could improve outcomes and provide clinical benefit to individuals with CP. The relatively limited evidence base for most interventions underscores the need for additional research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292993">
        <p class="paperTitle">The shared genetic risk architecture of neurological and psychiatric disorders: a genome-wide analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smeland, O. B.; Kutrolli, G.; Bahrami, S.; Fominykh, V.; Parker, N.; Hindley, G.; Rodevand, L.; Jaholkowski, P.; Tesfaye, M.; Parekh, P.; Elvsaashagen, T.; Grotzinger, A. D.; The International Multiple Sclerosis Genetics Consortium (IMSGC),  ; The International Headache Genetics Consortium (IHGC),  ; Steen, N. E.; van der Meer, D.; O&#39;Connell, K. S.; Djurovic, S. D.; Dale, A.; Shadrin, A.; Frei, O.; Andreassen, O.</p>
        <p class="info">Score: 33.5, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292993' target='https://doi.org/10.1101/2023.07.21.23292993'> 10.1101/2023.07.21.23292993</a></p>
        <p class="abstract">While neurological and psychiatric disorders have historically been considered to reflect distinct pathogenic entities, recent findings suggest shared pathobiological mechanisms. However, the extent to which these heritable disorders share genetic influences remains unclear. Here, we performed a comprehensive analysis of GWAS data, involving nearly 1 million cases across ten neurological diseases and ten psychiatric disorders, to compare their common genetic risk and biological underpinnings. Using complementary statistical tools, we demonstrate widespread genetic overlap across the disorders, even in the absence of genetic correlations. This indicates that a large set of common variants impact risk of multiple neurological and psychiatric disorders, but with divergent effect sizes. Furthermore, biological interrogation revealed a range of biological processes associated with neurological diseases, while psychiatric disorders consistently implicated neuronal biology. Altogether, the study indicates that neurological and psychiatric disorders share key etiological aspects, which has important implications for disease classification, precision medicine, and clinical practice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.23295771">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.23295771" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.23295771">
        <p class="paperTitle">Frequentmers - a novel way to look at metagenomic Next Generation Sequencing data and an application in detecting liver cirrhosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.23295771" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.23295771" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mouratidis, I.; Chantzi, N.; Khan, U.; Konnaris, M. A.; Chan, C. S. Y.; Mareboina, M.; Georgakopoulos-Soares, I.</p>
        <p class="info">Score: 2.8, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.23295771' target='https://doi.org/10.1101/2023.09.19.23295771'> 10.1101/2023.09.19.23295771</a></p>
        <p class="abstract">Early detection of human disease is associated with improved clinical outcomes. However, many diseases are often detected at an advanced, symptomatic stage where patients are past efficacious treatment periods and can result in less favorable outcomes. Therefore, methods that can accurately detect human disease at a presymptomatic stage are urgently needed. Here, we introduce &#34;frequentmers&#34;; short sequences that are specific and recurrently observed in either patient or healthy control samples, but not in both. We showcase the utility of frequentmers for the detection of liver cirrhosis using metagenomic Next Generation Sequencing data from stool samples of patients and controls. We develop classification models for the detection of liver cirrhosis and achieve an AUC score of 0.91 using ten-fold cross-validation. A small subset of 200 frequentmers can achieve comparable results in detecting liver cirrhosis. Finally, we identify the microbial organisms in liver cirrhosis samples, which are associated with the most predictive frequentmer biomarkers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23296145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23296145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23296145">
        <p class="paperTitle">Landscape of blood group antigens and alleles in the Indian population from whole genome sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23296145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23296145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rophina, M.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.</p>
        <p class="info">Score: 1.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23296145' target='https://doi.org/10.1101/2023.09.26.23296145'> 10.1101/2023.09.26.23296145</a></p>
        <p class="abstract">Blood group antigens are genetically inherited macromolecular structures which form the underlying factor for inter individual variations in human blood. Currently there exists over 390 human blood group antigens corresponding to 44 blood group systems and 2 erythroid specific transcription factors. Distribution of these blood group antigens have been found to differ significantly among various ethnic populations. To date, there is a lack of comprehensive research that offers extensive blood group profiles for the Indian population. Whole genome sequence data (hg38) of 1029 self-declared healthy Indian individuals generated as a part of the pilot phase IndiGen programme were used for the analysis. Variants spanning the genes of 44 blood group systems and two transcription factors KLF1, GATA1 were fetched and annotated for their functional consequences. Our study reports a total of 40712 blood group related variants of which 695 were identified as non-synonymous variants in the coding region. Of the total non-synonymous variants, 105 were found to have a known blood phenotype. A total of 24 variants belonging to 12 blood groups were predicted to be deleterious by more than three computational tools. Our study was also able to identify a few rare blood phenotypes including Au(a-b&#43;), Js(a&#43;b&#43;), Di(a&#43;b-), In(a&#43;b-) and KANNO-. This study is the first to use genomic data to understand the blood group antigen profiles of the Indian population, and it also systematically compares these profiles with those of other global populations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
