<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease
Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.
Score: 18.3, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623
Studies of the genetics of Alzheimer&#39;s disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease
Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.
Score: 18.3, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623
Studies of the genetics of Alzheimer&#39;s disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease
Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.
Score: 18.3, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623
Studies of the genetics of Alzheimer&#39;s disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "Polygenic burden of short tandem repeat expansions promote risk for Alzheimer\u0026#39;s disease\nAuthors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.\nScore: 18.3, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623\nStudies of the genetics of Alzheimer\u0026#39;s disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation.",
  "keywords": [
    
  ],
  "articleBody": " Polygenic burden of short tandem repeat expansions promote risk for Alzheimer's disease\nAuthors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.\nScore: 18.3, Published: 2023-11-16 DOI: 10.1101/2023.11.16.23298623\nStudies of the genetics of Alzheimer's disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation. There are over one million short tandem repeats (STRs) in the genome, and their link to AD risk has not been assessed. As pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk. Here, we genotyped 321,742 polymorphic STR tracts genome-wide using PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals). We implemented an approach to identify STR expansions as STRs with tract lengths that are outliers from the population. We then tested for differences in aggregate burden of expansions in case versus control individuals. AD patients had a 1.19-fold increase of STR expansions compared to healthy elderly controls (p=8.27x10-3, two-sided Mann Whitney test). Individuals carrying \u003e 30 STR expansions had 3.62-fold higher odds of having AD and had more severe AD neuropathology. AD STR expansions were highly enriched within active promoters in post-mortem hippocampal brain tissues and particularly within SINE-VNTR-Alu (SVA) retrotransposons. Together, these results demonstrate that expanded STRs within active promoter regions of the genome promote risk of AD.\nEstimating cancer risk in carriers of Lynch syndrome variants in UK Biobank\nAuthors: Fummey, E.; Navarro, P.; Plazzer, J.-P.; Frayling, I. M.; Knott, S.; Tenesa, A.\nScore: 2.5, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298308\nBackgroundLynch syndrome (LS) is an inherited cancer predisposition syndrome caused by genetic variants affecting DNA mismatch repair (MMR) genes MLH1, MSH2, MSH6, and PMS2. Cancer risk in LS is estimated from cohorts of individuals ascertained by family history of cancer, which is known to upwardly bias estimates. MethodsThe InSiGHT Database classifies MMR gene variants by pathogenicity through expert panel review of published evidence. 830 carriers of pathogenic or likely pathogenic (path_MMR) MMR gene variants from InSiGHT were identified in 454,756 UK Biobank participants using whole exome sequence. Nelson-Aalen survival analysis was used to estimate cumulative incidence of colorectal, endometrial, and breast cancer. ResultsCumulative incidence of colorectal and endometrial cancer by age 70 was elevated in path_MMR carriers compared to non-carriers (colorectal: 11.8% (95% CI: 9.5 - 14.6) vs. 1.7% (1.6 - 1.7), endometrial: 13.4% (10.2 - 17.6) vs. 1.0% (0.9 - 1.0)), but the magnitude of this increase differed between genes. Cumulative breast cancer incidence by age 70 was not elevated in path_MMR carriers compared to non-carriers (8.9% (6.3 - 12.4) vs. 7.5% (7.4 - 7.6)). Cumulative cancer incidence estimates in UK Biobank were similar to estimates from the Prospective Lynch Syndrome Database for all genes and cancers, except there was no evidence for elevated endometrial cancer risk in carriers of pathogenic PMS2 variants in UK Biobank. ConclusionThese results can be used to inform the management of incidentally identified cases of LS. For example, they support the application of existing colorectal cancer surveillance strategies for LS in incidentally identified cases.\nCXCL12 regulates coronary artery dominance in diverse populations and links development to disease\nAuthors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.\nScore: 29.4, Published: 2023-10-27 DOI: 10.1101/2023.10.27.23297507\nMammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.\nMaSk-LMM: A Matrix Sketching Framework for Linear Mixed Models in Association Studies\nAuthors: Burch, M. C.; Bose, A.; Dexter, G.; PARIDA, L.; Drineas, P.\nScore: 1.2, Published: 2023-11-13 DOI: 10.1101/2023.11.13.23298469\nLinear mixed models (LMMs) have been widely used in genome-wide association studies (GWAS) to control for population stratification and cryptic relatedness. Unfortunately, estimating LMM parameters is computationally expensive, necessitating large-scale matrix operations to build the genetic relatedness matrix (GRM). Over the past 25 years, Randomized Linear Algebra has provided alternative approaches to such matrix operations by leveraging matrix sketching, which often results in provably accurate fast and efficient approximations. We leverage matrix sketching to develop a fast and efficient LMM method called Matrix-Sketching LMM (MaSk-LMM) by sketching the genotype matrix to reduce its dimensions and speed up computations. Our framework comes with both theoretical guarantees and a strong empirical performance compared to current state-of-the-art.\nA Massively Parallel Screen of 5'UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes\nAuthors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.\nScore: 17.9, Published: 2023-11-03 DOI: 10.1101/2023.11.02.23297961\nDe novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5'UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5'UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5'UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.\nCost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics\nAuthors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.\nScore: 12.5, Published: 2023-11-09 DOI: 10.1101/2023.11.08.23298270\nClonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable ([~]$8/sample), accurate and highly scalable. To demonstrate the utility of this assay, we detected CHIP in a cohort of 456 individuals with DNA collected at multiple timepoints in the Vanderbilt BioVU biobank and quantified clonal expansion rates over time. A total of 101 individuals with CHIP were identified, and individual-level clonal expansion rate was calculated using the variant allele fraction (VAF) at both timepoints. Differences in clonal expansion rate by driver gene were observed, but there was also significant individual-level heterogeneity, emphasizing the multifactorial nature of clonal expansion. We further describe the mutation co-occurrence and clonal competition between multiple driver mutations.\nGenome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves cross-ancestry prediction\nAuthors: Hoffmann, T. J.; Graff, R. E.; Madduri, R. K.; Rodriguez, A. A.; Cario, C. L.; Feng, K.; Jiang, Y.; Wang, A.; Klein, R. J.; Pierce, B. L.; Eggener, S.; Tong, L.; Blot, W.; Long, J.; Rebbeck, T.; Lachance, J.; Andrews, C.; Adebiyi, A. O.; Adusei, B.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Jalloh, M.; Janivara, R.; Chen, W. C.; Mensah, J. E.; Agalliu, I.; Berndt, S. I.; Shelley, J. P.; Schaffer, K.; Machiela, M. J.; Freedman, N. D.; Huang, W.-Y.; Li, S. A.; Goodman, P. J.; Till, C.; Thompson, I.; Lilja, H.; Van Den Eeden, S. K.; Chanock, S. J.; Mosley, J. D.; Conti, D. V.; Haiman, C. A.;\nScore: 8.0, Published: 2023-10-30 DOI: 10.1101/2023.10.27.23297676\nWe conducted a multi-ancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry; 58,236 African ancestry; 23,546 Hispanic/Latino; 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (p[\u0026le;]5e-8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n=95,768). Meta-analyzing discovery and replication (n=392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our new polygenic risk score reached 16.9% (95% CI=16.1%-17.8%) in European ancestry, 9.5% (95% CI=7.0%-12.2%) in African ancestry, 18.6% (95% CI=15.8%-21.4%) in Hispanic/Latino, and 15.3% (95% CI=12.7%-18.1%) in Asian ancestry, and lower for higher age. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, with potential to personalize prostate cancer screening.\nA Biallelic Variant of the RNA Exosome Gene EXOSC4 Causes Translational Defects Associated with a Neurodevelopmental Disorder\nAuthors: Fasken, M. B.; Leung, S. W.; Cureton, L. A.; Al-Awadi, M.; Al-Kindy, A.; Khoshnevis, S.; Ghalei, H.; Al-Maawali, A.; Corbett, A. H.\nScore: 1.2, Published: 2023-10-28 DOI: 10.1101/2023.10.24.23297197\nThe RNA exosome is an evolutionarily conserved complex required for both precise RNA processing and decay. Mutations in EXOSC genes encoding structural subunits of the complex are linked to several autosomal recessive disorders. Here, we describe a missense allele of the EXOSC4 gene, which causes a collection of clinical features in two affected siblings. This missense mutation (NM_019037.3: exon3:c.560T\u003eC), changes a leucine residue within a highly conserved region of EXOSC4 to proline (p.Leu187Pro). The two affected individuals presented with prenatal growth restriction, failure to thrive, global developmental delay, intracerebral and basal ganglia calcifications, and kidney failure. Homozygosity for the damaging variant was identified through exome sequencing and Sanger sequencing confirmed segregation. To explore the functional consequences of this amino acid change, we modeled EXOSC4-L187P in the corresponding budding yeast protein, Rrp41 (Rrp41-L187P). Cells that express Rrp41-L187P as the sole copy of the essential Rrp41 protein show significant growth defects. The steady-state level of both the Rrp41-L187P and the EXOSC4-L187P proteins is significantly decreased compared to control Rrp41/EXOSC4. Consistent with this observation, targets of the RNA exosome accumulate in rrp41-L187P cells, including the 7S precursor of 5.8S rRNA. Polysome profiles show a significant decrease in translation in rrp41-L187P cells as compared to control cells with apparent incorporation of 7S pre-rRNA into polysomes. Taken together, this work adds the EXOSC4 subunit of the RNA exosome to the structural subunits of this complex that have been linked to human disease and defines foundational molecular defects that could contribute to the adverse growth phenotypes caused by this novel EXOSC4 pathogenic variant.\n",
  "wordCount" : "2058",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.23298623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.23298623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.23298623">
        <p class="paperTitle">Polygenic burden of short tandem repeat expansions promote risk for Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.23298623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.23298623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, M. H.; Lee, W.-P.; Vardarajan, B. N.; Schellenberg, G. D.; Phillips-Cremins, J. E.</p>
        <p class="info">Score: 18.3, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.23298623' target='https://doi.org/10.1101/2023.11.16.23298623'> 10.1101/2023.11.16.23298623</a></p>
        <p class="abstract">Studies of the genetics of Alzheimer&#39;s disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, most of the disease heritability has yet to be uncovered, suggesting that there is substantial genetic risk conferred by other forms of genetic variation. There are over one million short tandem repeats (STRs) in the genome, and their link to AD risk has not been assessed. As pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk. Here, we genotyped 321,742 polymorphic STR tracts genome-wide using PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals). We implemented an approach to identify STR expansions as STRs with tract lengths that are outliers from the population. We then tested for differences in aggregate burden of expansions in case versus control individuals. AD patients had a 1.19-fold increase of STR expansions compared to healthy elderly controls (p=8.27x10-3, two-sided Mann Whitney test). Individuals carrying &gt; 30 STR expansions had 3.62-fold higher odds of having AD and had more severe AD neuropathology. AD STR expansions were highly enriched within active promoters in post-mortem hippocampal brain tissues and particularly within SINE-VNTR-Alu (SVA) retrotransposons. Together, these results demonstrate that expanded STRs within active promoter regions of the genome promote risk of AD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298308">
        <p class="paperTitle">Estimating cancer risk in carriers of Lynch syndrome variants in UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fummey, E.; Navarro, P.; Plazzer, J.-P.; Frayling, I. M.; Knott, S.; Tenesa, A.</p>
        <p class="info">Score: 2.5, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298308' target='https://doi.org/10.1101/2023.11.10.23298308'> 10.1101/2023.11.10.23298308</a></p>
        <p class="abstract">BackgroundLynch syndrome (LS) is an inherited cancer predisposition syndrome caused by genetic variants affecting DNA mismatch repair (MMR) genes MLH1, MSH2, MSH6, and PMS2. Cancer risk in LS is estimated from cohorts of individuals ascertained by family history of cancer, which is known to upwardly bias estimates.

MethodsThe InSiGHT Database classifies MMR gene variants by pathogenicity through expert panel review of published evidence. 830 carriers of pathogenic or likely pathogenic (path_MMR) MMR gene variants from InSiGHT were identified in 454,756 UK Biobank participants using whole exome sequence. Nelson-Aalen survival analysis was used to estimate cumulative incidence of colorectal, endometrial, and breast cancer.

ResultsCumulative incidence of colorectal and endometrial cancer by age 70 was elevated in path_MMR carriers compared to non-carriers (colorectal: 11.8% (95% CI: 9.5 - 14.6) vs. 1.7% (1.6 - 1.7), endometrial: 13.4% (10.2 - 17.6) vs. 1.0% (0.9 - 1.0)), but the magnitude of this increase differed between genes. Cumulative breast cancer incidence by age 70 was not elevated in path_MMR carriers compared to non-carriers (8.9% (6.3 - 12.4) vs. 7.5% (7.4 - 7.6)). Cumulative cancer incidence estimates in UK Biobank were similar to estimates from the Prospective Lynch Syndrome Database for all genes and cancers, except there was no evidence for elevated endometrial cancer risk in carriers of pathogenic PMS2 variants in UK Biobank.

ConclusionThese results can be used to inform the management of incidentally identified cases of LS. For example, they support the application of existing colorectal cancer surveillance strategies for LS in incidentally identified cases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297507">
        <p class="paperTitle">CXCL12 regulates coronary artery dominance in diverse populations and links development to disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rios Coronado, P. E.; Zanetti, D.; Zhou, J.; Naftaly, J. A.; Prabala, P.; Kho, P. F.; Martinez Jaimes, A. M.; Hilliard, A. T.; Pyarajan, S.; Dochtermann, D.; Chang, K.-M.; Winn, V. D.; Pasca, A. M.; Plomondon, M. E.; Waldo, S. W.; Tsao, P. S.; Clarke, S. L.; Red-Horse, K.; Assimes, T. L.</p>
        <p class="info">Score: 29.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297507' target='https://doi.org/10.1101/2023.10.27.23297507'> 10.1101/2023.10.27.23297507</a></p>
        <p class="abstract">Mammalian cardiac muscle is supplied with blood by right and left coronary arteries that form branches covering both ventricles of the heart. Whether branches of the right or left coronary arteries wrap around to the inferior side of the left ventricle is variable in humans and termed right or left dominance. Coronary dominance is likely a heritable trait, but its genetic architecture has never been explored. Here, we present the first large-scale multi-ancestry genome-wide association study of dominance in 61,043 participants of the VA Million Veteran Program, including over 10,300 Africans and 4,400 Admixed Americans. Dominance was moderately heritable with ten loci reaching genome wide significance. The most significant mapped to the chemokine CXCL12 in both Europeans and Africans. Whole-organ imaging of human fetal hearts revealed that dominance is established during development in locations where CXCL12 is expressed. In mice, dominance involved the septal coronary artery, and its patterning was altered with Cxcl12 deficiency. Finally, we linked human dominance patterns with coronary artery disease through colocalization, genome-wide genetic correlation and Mendelian Randomization analyses. Together, our data supports CXCL12 as a primary determinant of coronary artery dominance in humans of diverse backgrounds and suggests that developmental patterning of arteries may influence ones susceptibility to ischemic heart disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.23298469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.23298469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.23298469">
        <p class="paperTitle">MaSk-LMM: A Matrix Sketching Framework for Linear Mixed Models in Association Studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.23298469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.23298469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burch, M. C.; Bose, A.; Dexter, G.; PARIDA, L.; Drineas, P.</p>
        <p class="info">Score: 1.2, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.23298469' target='https://doi.org/10.1101/2023.11.13.23298469'> 10.1101/2023.11.13.23298469</a></p>
        <p class="abstract">Linear mixed models (LMMs) have been widely used in genome-wide association studies (GWAS) to control for population stratification and cryptic relatedness. Unfortunately, estimating LMM parameters is computationally expensive, necessitating large-scale matrix operations to build the genetic relatedness matrix (GRM). Over the past 25 years, Randomized Linear Algebra has provided alternative approaches to such matrix operations by leveraging matrix sketching, which often results in provably accurate fast and efficient approximations. We leverage matrix sketching to develop a fast and efficient LMM method called Matrix-Sketching LMM (MaSk-LMM) by sketching the genotype matrix to reduce its dimensions and speed up computations. Our framework comes with both theoretical guarantees and a strong empirical performance compared to current state-of-the-art.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.23297961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.23297961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.23297961">
        <p class="paperTitle">A Massively Parallel Screen of 5&#39;UTR Mutations Identifies Variants Impacting Translation and Protein Production in Neurodevelopmental Disorder Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.23297961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.23297961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plassmeyer, S. P.; Florian, C. P.; Kasper, M. J.; Chase, R.; Mueller, S.; Liu, Y.; McFarland White, K.; Jungers, C. F.; Pavlovic Djuranovic, S.; Djuranovic, S.; Dougherty, J. D.</p>
        <p class="info">Score: 17.9, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.23297961' target='https://doi.org/10.1101/2023.11.02.23297961'> 10.1101/2023.11.02.23297961</a></p>
        <p class="abstract">De novo mutations cause a variety of neurodevelopmental disorders including autism. Recent whole genome sequencing from individuals with autism has shown that many de novo mutations also occur in untranslated regions (UTRs) of genes, but it is difficult to predict from sequence alone which mutations are functional, let alone causal. Therefore, we developed a high throughput assay to screen the transcriptional and translational effects of 997 variants from 5&#39;UTR patient mutations. This assay successfully enriched for elements that alter reporter translation, identifying over 100 potentially functional mutations from probands. Studies in patient-derived cell lines further confirmed that these mutations can alter protein production in individuals with autism, and some variants fall in genes known to cause syndromic forms of autism, suggesting a diagnosis for these individual patients. Since UTR function varies by cell type, we further optimized this high throughput assay to enable assessment of mutations in neurons in vivo. First, comparing in cellulo to in vivo results, we demonstrate neurons have different principles of regulation by 5&#39;UTRs, consistent with a more robust mechanism for reducing the impact of RNA secondary structure. Finally, we discovered patient mutations specifically altering the translational activity of additional known syndromic genes LRRC4 and ZNF644 in neurons of the brain. Overall our results highlight a new approach for assessing the impact of 5&#39;UTR mutations across cell types and suggest that some cases of neurodevelopmental disorder may be caused by such variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.23298270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.23298270" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.23298270">
        <p class="paperTitle">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.23298270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.23298270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mack, T.; Vlasschaert, C.; von Beck, K.; Silver, A. J.; Heimlich, J. B.; Poisner, H.; Condon, H. R.; Ulloa, J.; Sochacki, A. L.; Spaulding, T. P.; Kishtagari, A.; Bejan, C. A.; Xu, Y.; Savona, M. R.; Jones, A.; Bick, A. G.</p>
        <p class="info">Score: 12.5, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.23298270' target='https://doi.org/10.1101/2023.11.08.23298270'> 10.1101/2023.11.08.23298270</a></p>
        <p class="abstract">Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable ([~]$8/sample), accurate and highly scalable. To demonstrate the utility of this assay, we detected CHIP in a cohort of 456 individuals with DNA collected at multiple timepoints in the Vanderbilt BioVU biobank and quantified clonal expansion rates over time. A total of 101 individuals with CHIP were identified, and individual-level clonal expansion rate was calculated using the variant allele fraction (VAF) at both timepoints. Differences in clonal expansion rate by driver gene were observed, but there was also significant individual-level heterogeneity, emphasizing the multifactorial nature of clonal expansion. We further describe the mutation co-occurrence and clonal competition between multiple driver mutations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297676">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297676" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297676">
        <p class="paperTitle">Genome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves cross-ancestry prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297676" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297676" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, T. J.; Graff, R. E.; Madduri, R. K.; Rodriguez, A. A.; Cario, C. L.; Feng, K.; Jiang, Y.; Wang, A.; Klein, R. J.; Pierce, B. L.; Eggener, S.; Tong, L.; Blot, W.; Long, J.; Rebbeck, T.; Lachance, J.; Andrews, C.; Adebiyi, A. O.; Adusei, B.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Jalloh, M.; Janivara, R.; Chen, W. C.; Mensah, J. E.; Agalliu, I.; Berndt, S. I.; Shelley, J. P.; Schaffer, K.; Machiela, M. J.; Freedman, N. D.; Huang, W.-Y.; Li, S. A.; Goodman, P. J.; Till, C.; Thompson, I.; Lilja, H.; Van Den Eeden, S. K.; Chanock, S. J.; Mosley, J. D.; Conti, D. V.; Haiman, C. A.;</p>
        <p class="info">Score: 8.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297676' target='https://doi.org/10.1101/2023.10.27.23297676'> 10.1101/2023.10.27.23297676</a></p>
        <p class="abstract">We conducted a multi-ancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry; 58,236 African ancestry; 23,546 Hispanic/Latino; 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (p[&amp;le;]5e-8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n=95,768). Meta-analyzing discovery and replication (n=392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our new polygenic risk score reached 16.9% (95% CI=16.1%-17.8%) in European ancestry, 9.5% (95% CI=7.0%-12.2%) in African ancestry, 18.6% (95% CI=15.8%-21.4%) in Hispanic/Latino, and 15.3% (95% CI=12.7%-18.1%) in Asian ancestry, and lower for higher age. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, with potential to personalize prostate cancer screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.23297197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.23297197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.23297197">
        <p class="paperTitle">A Biallelic Variant of the RNA Exosome Gene EXOSC4 Causes Translational Defects Associated with a Neurodevelopmental Disorder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.23297197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.23297197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fasken, M. B.; Leung, S. W.; Cureton, L. A.; Al-Awadi, M.; Al-Kindy, A.; Khoshnevis, S.; Ghalei, H.; Al-Maawali, A.; Corbett, A. H.</p>
        <p class="info">Score: 1.2, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.23297197' target='https://doi.org/10.1101/2023.10.24.23297197'> 10.1101/2023.10.24.23297197</a></p>
        <p class="abstract">The RNA exosome is an evolutionarily conserved complex required for both precise RNA processing and decay. Mutations in EXOSC genes encoding structural subunits of the complex are linked to several autosomal recessive disorders. Here, we describe a missense allele of the EXOSC4 gene, which causes a collection of clinical features in two affected siblings. This missense mutation (NM_019037.3: exon3:c.560T&gt;C), changes a leucine residue within a highly conserved region of EXOSC4 to proline (p.Leu187Pro). The two affected individuals presented with prenatal growth restriction, failure to thrive, global developmental delay, intracerebral and basal ganglia calcifications, and kidney failure. Homozygosity for the damaging variant was identified through exome sequencing and Sanger sequencing confirmed segregation. To explore the functional consequences of this amino acid change, we modeled EXOSC4-L187P in the corresponding budding yeast protein, Rrp41 (Rrp41-L187P). Cells that express Rrp41-L187P as the sole copy of the essential Rrp41 protein show significant growth defects. The steady-state level of both the Rrp41-L187P and the EXOSC4-L187P proteins is significantly decreased compared to control Rrp41/EXOSC4. Consistent with this observation, targets of the RNA exosome accumulate in rrp41-L187P cells, including the 7S precursor of 5.8S rRNA. Polysome profiles show a significant decrease in translation in rrp41-L187P cells as compared to control cells with apparent incorporation of 7S pre-rRNA into polysomes. Taken together, this work adds the EXOSC4 subunit of the RNA exosome to the structural subunits of this complex that have been linked to human disease and defines foundational molecular defects that could contribute to the adverse growth phenotypes caused by this novel EXOSC4 pathogenic variant.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
