<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetic and genomic medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetic and genomic medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases
Authors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.
Score: 11.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.23300645
PWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetic and genomic medicine" />
<meta property="og:description" content="PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases
Authors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.
Score: 11.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.23300645
PWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetic and genomic medicine"/>
<meta name="twitter:description" content="PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases
Authors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.
Score: 11.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.23300645
PWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetic and genomic medicine",
      "item": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetic and genomic medicine",
  "name": "genetic and genomic medicine",
  "description": "PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases\nAuthors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.\nScore: 11.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.23300645\nPWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score.",
  "keywords": [
    
  ],
  "articleBody": " PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases\nAuthors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.\nScore: 11.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.23300645\nPWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score. Finally, standard statistical tests are activated in the case-control setting to yield statistically significant genes per phenotype. The PWAS Hub offers a user-friendly interface for an in-depth exploration of gene-disease associations from the UK Biobank (UKB). Results from PWAS cover 99 common diseases and conditions, each with over 10,000 diagnosed individuals per phenotype. Users can explore genes associated with these diseases, with separate analyses conducted for males and females. The PWAS Hub lists statistically significant genes associated with common diseases. It also indicates whether the analyzed damaged gene is associated with an increased or decreased risk. For each phenotype, the analyses account for sex-based genetic effects, inheritance modes (dominant and recessive), and the pleiotropic nature of associated genes. The PWAS Hub showcases its usefulness by navigating through such proteomic-genetic application for asthma. Graphical tools facilitate comparing genetic effects between the results of PWAS and coding GWAS, aiding in understanding the sex-specific genetic impact on common diseases. This adaptable platform is attractive for clinicians, researchers, and individuals interested in delving into gene-disease associations and sex-specific genetic effects. The PWAS Hub is accessible at http://pwas.huji.ac.il.\nEvidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel\nAuthors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A. R.; Monteiro, A. N.; Nathan, V.; O'Connor, R.; Pesaran, T.; Radice, P.; Schmidt, G.; Sokilde-Pedersen, I.; Southey, M.; Tavtigian, S. V.; Thompson, B. A.; Toland, A. E.; Turnbull, C.; Vogel, M. J.; Weyandt, J.; Wiggins, G. A. R.; Zec, L.; Couch, F. J.; Walker, L.; Vreeswijk, M. P. G.; Gold\nScore: 5.7, Published: 2024-01-23 DOI: 10.1101/2024.01.22.24301588\nThe ENIGMA research consortium (https://enigmaconsortium.org/) develops and applies methods to determine clinical significance of variants in Hereditary Breast and Ovarian Cancer genes. An ENIGMA BRCA1/2 classification sub-group, originally formed in 2016 as a ClinGen external expert panel, evolved into a ClinGen internal Variant Curation Expert Panel (VCEP) to align with Federal Drug Administration recognized processes for ClinVar contributions. The VCEP reviewed American College of Medical Genetics/Association of Molecular Pathology (ACMG/AMP) classification criteria for relevance to interpreting BRCA1 and BRCA2 variants. Statistical methods were used to calibrate evidence strength for different data types. Pilot specifications were tested on 40 variants, and documentation revised for clarity and ease-of-use. The original criterion descriptions for 13 evidence codes were considered non-applicable or overlapping with other criteria. Scenario of use was extended or re-purposed for eight codes. Extensive analysis and/or data review informed specification descriptions and weights for all codes. Specifications were applied to pilot variants with pre-existing ClinVar classification as follows: 13 Uncertain Significance or Conflicting, 14 Pathogenic and/or Likely Pathogenic, and 13 Benign and/or Likely Benign. Review resolved classification for 11/13 Uncertain Significance or Conflicting variants, and retained or improved confidence in classification for the remaining variants. Alignment of pre-existing ENIGMA research classification processes with ACMG/AMP classification guidelines highlighted several gaps in both the research processes and the baseline ACMG/AMP criteria. Calibration of evidence types was key to justify utility and strength of different evidence types for gene-specific application. The gene-specific criteria demonstrated value for improving ACMG/AMP-aligned classification of BRCA1 and BRCA2 variants.\nExploration of the circulating human secretome through protein quantitative trait analysis identifies an association between circulating levels of apolipoprotein L1 and risk of giant cell arteritis\nAuthors: Chaddock, N.; UK GCA Consortium, ; Zulcinski, M.; Martin, J.; Mälarstig, A.; Peters, J. E.; Iles, M.; Morgan, A. W.\nScore: 5.1, Published: 2024-01-21 DOI: 10.1101/2024.01.19.24301534\nBackgroundGlucocorticoid monotherapy remains the principal treatment for giant cell arteritis (GCA), yet concurrent toxicity and adverse effects highlight the need for targeted therapies and improved risk stratification. Previous work suggests that evidence of genetic association can improve success rates in clinical trials and identify biomarkers for risk assessment, particularly when combined with other omics data, such as proteomics. However, relatively little is currently known about the genetic basis of GCA. MethodsPolygenic risk scores (PRS) were developed for 169 human plasma proteins and tested for association with GCA susceptibility (cases N=729, controls N=2,619). Associated PRS were replicated in an independent cohort (cases N=1,129, controls N=2,654) and their respective proteins were evaluated for causality using Mendelian randomization (MR). Finally, relationships between proteins with GCA-associated PRS were assessed using protein-protein interaction (PPI) network analysis ResultsThe Apolipoprotein L1 (APOL1) PRS had a statistically significant GCA association with a protective effect (P-value[P]=1 x 10-4), which replicated in an independent dataset (P=8.69 x 10- 4), and MR analysis supported a causal relationship (beta=-0.093; SE=0.02; P=4.42 x 10-9). PPI network analysis of proteins with GCA-associated PRS revealed enrichment for \"negative regulation of fibrinolysis\" and \"negative regulation of blood coagulation\" pathways. ConclusionsThis work emphasizes a potentially protective role of APOL1 and therefore reverse cholesterol transport in the pathogenesis of GCA. These findings also implicate fibrinolytic and coagulation cascades in GCA susceptibility, highlighting pathways that may be of interest for future pharmaceutical targeting. Non-standard Abbreviations and AcronymsGCA, giant cell arteritis; MHC, major histocompatibility complex; GWAS, genome-wide association study; SNPs, single nucleotide polymorphisms; PRS, polygenic risk score; pQTL, protein quantitative trait loci; MR, Mendelian randomization; QC, quality control; WTCCC, Wellcome Trust Case Control Consortium; PCA, principal component analysis; IV, instrumental variables; IVW, inverse-variance weighted; PheWAS, phenome-wide association study; GO, gene ontology; FUMA, Functional Mapping and Annotation of Genome-Wide Association Studies; MAGMA, Multi-marker Analysis of GenoMic Annotation; RCT, reverse cholesterol transport. Clinical PerspectiveO_ST_ABSWhat is new?C_ST_ABSO_LIAn apoliporotein-L1 polygenic risk score was associated with giant cell arteritis susceptibility, and replicated in an independent dataset. C_LIO_LIEvidence for causality of a protective effect of apolipoprotein-L1 in giant cell arteritis susceptibility was identified using Mendelian randomization. C_LIO_LIProteins with giant cell arteritis-associated polygenic risk scores were enriched in coagulation-related, fibrinolytic and immune response pathways. C_LI What are the clinical implications?O_LIFindings from this study indicate a protective role of apolipoprotein-L1 in giant cell arteritis susceptibility, highlighting a potential involvement of reverse cholesterol transport and lipid metabolism in disease pathogenesis. C_LIO_LIFibrinolytic and coagulation cascades were also implicated in the disease in addition to innate immune response pathways, redrawing attention to the role of thromboinflammation and the need to re-evaluate anti-platelet and anticoagulant therapies, particularly for those with impending visual loss and cranial ischaemic complications. C_LI\nEarly detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics\nAuthors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A.; Chrisinger, J. S. A.; Dehner, C. A.; Widemann, B. C.; Hirbe, A. C.; Chaudhuri, A. A.; Shern, J. F.\nScore: 16.1, Published: 2024-01-20 DOI: 10.1101/2024.01.18.24301053\nEarly detection of neurofibromatosis type 1 (NF1) associated peripheral nerve sheath tumors (PNST) informs clinical decision-making, potentially averting deadly outcomes. Here, we describe a cell-free DNA (cfDNA) fragmentomic approach which distinguishes non-malignant, pre-malignant and malignant forms of NF1 PNST. Using plasma samples from a novel cohort of 101 NF1 patients and 21 healthy controls, we validated that our previous cfDNA copy number alteration (CNA)-based approach identifies malignant peripheral nerve sheath tumor (MPNST) but cannot distinguish among benign and premalignant states. We therefore investigated the ability of fragment-based cfDNA features to differentiate NF1-associated tumors including binned genome-wide fragment length ratios, end motif analysis, and non-negative matrix factorization deconvolution of fragment lengths. Fragmentomic methods were able to differentiate pre-malignant states including atypical neurofibromas (AN). Fragmentomics also adjudicated AN cases suspicious for MPNST, correctly diagnosing samples noninvasively, which could have informed clinical management. Overall, this study pioneers the early detection of malignant and premalignant peripheral nerve sheath tumors in NF1 patients using plasma cfDNA fragmentomics. In addition to screening applications, this novel approach distinguishes atypical neurofibromas from benign plexiform neurofibromas and malignant peripheral nerve sheath tumors, enabling more precise clinical diagnosis and management.\nA phenome-wide association study of tandem repeat variation in 168,554 individuals from the UK Biobank\nAuthors: Manigbas, C.; Jadhav, B.; Garg, P.; Shadrina, M.; Lee, W.; Martin-Trujillo, A.; Sharp, A.\nScore: 1.9, Published: 2024-01-23 DOI: 10.1101/2024.01.22.24301630\nMost genetic association studies focus on binary variants. To identify the effects of multi-allelic variation of tandem repeats (TRs) on human traits, we performed direct TR genotyping and phenome-wide association studies in 168,554 individuals from the UK Biobank, identifying 47 TRs showing causal associations with 73 traits. We replicated 23 of 31 (74%) of these causal associations in the All of Us cohort. While this set included several known repeat expansion disorders, novel associations we found were attributable to common polymorphic variation in TR length rather than rare expansions and include e.g. a coding polyhistidine motif in HRCT1 influencing risk of hypertension and a poly(CGC) in the 5UTR of GNB2 influencing heart rate. Causal TRs were strongly enriched for associations with local gene expression and DNA methylation. Our study highlights the contribution of multi-allelic TRs to the \"missing heritability\" of the human genome.\nImpact of genome build on RNA-seq interpretation and diagnostics\nAuthors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network, ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.\nScore: 35.4, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24301165\nTranscriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.\nAutScore- An integrative scoring approach for prioritization of ultra-rare autism spectrum disorder candidate variants from whole exome sequencing data\nAuthors: Shil, A.; Arava, N.; Levi, N.; Levine, L.; Golan, H.; Meiri, G.; Michaelovski, A.; Tsadaka, Y.; Aran, A.; Menashe, I.\nScore: 1.2, Published: 2024-01-25 DOI: 10.1101/2024.01.24.24301544\nBackgroundDiscerning clinically relevant ASD candidate variants from whole-exome sequencing (WES) data is complex, time-consuming, and labor-intensive. To this end, we developed AutScore, an integrative prioritization algorithm of ASD candidate variants from WES data, and assessed its performance to detect clinically relevant variants. MethodsWe studied WES data from 581 ASD probands, and their parents registered in the Azrieli National Center database for Autism and Neurodevelopment Research. We focused on rare allele frequency \u003c1%), high-quality proband-specific variants affecting genes associated with ASD or other neurodevelopmental disorders (NDDs). We assigned a score (i.e., AutScore) to each such variant based on their pathogenicity, clinical relevance, gene-disease association, and inheritance patterns. Finally, we compared the AutScore performance with the rating of clinical experts and the NDD variants prioritization algorithm, AutoCasC. ResultsOverall, 1161 ultra-rare variants distributed in 687 genes in 441 ASD probands were evaluated by AutScore with scores ranging from -4 to 25, with a mean {+/-} SD of 5.89 {+/-} 4.18. AutScore cut-off of [\u0026ge;] 12 outperforms AutoCasC in detecting clinically relevant ASD variants, with a detection accuracy rate of 72.3% and an overall diagnostic yield of 11.9%. Sixteen variants with AutScore of [\u0026ge;] 12 were distributed in fifteen novel ASD genes. ConclusionAutScore is an effective automated ranking system for ASD candidate variants that could be implemented in ASD clinical genetics pipelines.\n",
  "wordCount" : "2079",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetic-and-genomic-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetic and genomic medicine
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.23300645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.23300645" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.23300645">
        <p class="paperTitle">PWAS Hub: Exploring Gene-Based Associations of Common Complex Diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.23300645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.23300645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelman, G.; Zucker, R.; Brandes, N.; Linial, M.</p>
        <p class="info">Score: 11.4, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.23300645' target='https://doi.org/10.1101/2024.01.20.23300645'> 10.1101/2024.01.20.23300645</a></p>
        <p class="abstract">PWAS (Proteome-Wide Association Study) is an innovative genetic association approach that complements widely-used methods like GWAS (Genome-Wide Association Study). The PWAS platform involves consecutive phases. Initially, machine learning modeling and probabilistic considerations quantified the impact of genetic variants on protein-coding genes biochemical functions. Secondly, aggregating the variants per gene for each individual determines a gene-damaging score. Finally, standard statistical tests are activated in the case-control setting to yield statistically significant genes per phenotype. The PWAS Hub offers a user-friendly interface for an in-depth exploration of gene-disease associations from the UK Biobank (UKB). Results from PWAS cover 99 common diseases and conditions, each with over 10,000 diagnosed individuals per phenotype. Users can explore genes associated with these diseases, with separate analyses conducted for males and females. The PWAS Hub lists statistically significant genes associated with common diseases. It also indicates whether the analyzed damaged gene is associated with an increased or decreased risk. For each phenotype, the analyses account for sex-based genetic effects, inheritance modes (dominant and recessive), and the pleiotropic nature of associated genes. The PWAS Hub showcases its usefulness by navigating through such proteomic-genetic application for asthma. Graphical tools facilitate comparing genetic effects between the results of PWAS and coding GWAS, aiding in understanding the sex-specific genetic impact on common diseases. This adaptable platform is attractive for clinicians, researchers, and individuals interested in delving into gene-disease associations and sex-specific genetic effects. The PWAS Hub is accessible at http://pwas.huji.ac.il.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.24301588">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.24301588" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.24301588">
        <p class="paperTitle">Evidence-based recommendations for gene-specific ACMG/AMP variant classification from the ClinGen ENIGMA BRCA1 and BRCA2 Variant Curation Expert Panel</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.24301588" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.24301588" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parsons, M. T.; de la Hoya, M.; Richardson, M. E.; Tudini, E.; Anderson, M.; Berkofsky-Fessler, W.; Caputo, S.; Chan, R. C.; Cline, M. C.; Feng, B.-J.; Fortuno, C.; Gomez-Garcia, E.; Hadler, J.; Hiraki, S.; Holdren, M.; Houdayer, C.; Hruska, K.; James, P.; Karam, R.; Leong, H. S.; Martins, A.; Mensenkamp, A. R.; Monteiro, A. N.; Nathan, V.; O&#39;Connor, R.; Pesaran, T.; Radice, P.; Schmidt, G.; Sokilde-Pedersen, I.; Southey, M.; Tavtigian, S. V.; Thompson, B. A.; Toland, A. E.; Turnbull, C.; Vogel, M. J.; Weyandt, J.; Wiggins, G. A. R.; Zec, L.; Couch, F. J.; Walker, L.; Vreeswijk, M. P. G.; Gold</p>
        <p class="info">Score: 5.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.24301588' target='https://doi.org/10.1101/2024.01.22.24301588'> 10.1101/2024.01.22.24301588</a></p>
        <p class="abstract">The ENIGMA research consortium (https://enigmaconsortium.org/) develops and applies methods to determine clinical significance of variants in Hereditary Breast and Ovarian Cancer genes. An ENIGMA BRCA1/2 classification sub-group, originally formed in 2016 as a ClinGen external expert panel, evolved into a ClinGen internal Variant Curation Expert Panel (VCEP) to align with Federal Drug Administration recognized processes for ClinVar contributions.

The VCEP reviewed American College of Medical Genetics/Association of Molecular Pathology (ACMG/AMP) classification criteria for relevance to interpreting BRCA1 and BRCA2 variants. Statistical methods were used to calibrate evidence strength for different data types. Pilot specifications were tested on 40 variants, and documentation revised for clarity and ease-of-use.

The original criterion descriptions for 13 evidence codes were considered non-applicable or overlapping with other criteria. Scenario of use was extended or re-purposed for eight codes. Extensive analysis and/or data review informed specification descriptions and weights for all codes. Specifications were applied to pilot variants with pre-existing ClinVar classification as follows: 13 Uncertain Significance or Conflicting, 14 Pathogenic and/or Likely Pathogenic, and 13 Benign and/or Likely Benign. Review resolved classification for 11/13 Uncertain Significance or Conflicting variants, and retained or improved confidence in classification for the remaining variants.

Alignment of pre-existing ENIGMA research classification processes with ACMG/AMP classification guidelines highlighted several gaps in both the research processes and the baseline ACMG/AMP criteria. Calibration of evidence types was key to justify utility and strength of different evidence types for gene-specific application. The gene-specific criteria demonstrated value for improving ACMG/AMP-aligned classification of BRCA1 and BRCA2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.19.24301534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.19.24301534" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.19.24301534">
        <p class="paperTitle">Exploration of the circulating human secretome through protein quantitative trait analysis identifies an association between circulating levels of apolipoprotein L1 and risk of giant cell arteritis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.19.24301534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.19.24301534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chaddock, N.; UK GCA Consortium,  ; Zulcinski, M.; Martin, J.; Mälarstig, A.; Peters, J. E.; Iles, M.; Morgan, A. W.</p>
        <p class="info">Score: 5.1, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.19.24301534' target='https://doi.org/10.1101/2024.01.19.24301534'> 10.1101/2024.01.19.24301534</a></p>
        <p class="abstract">BackgroundGlucocorticoid monotherapy remains the principal treatment for giant cell arteritis (GCA), yet concurrent toxicity and adverse effects highlight the need for targeted therapies and improved risk stratification. Previous work suggests that evidence of genetic association can improve success rates in clinical trials and identify biomarkers for risk assessment, particularly when combined with other  omics data, such as proteomics. However, relatively little is currently known about the genetic basis of GCA.

MethodsPolygenic risk scores (PRS) were developed for 169 human plasma proteins and tested for association with GCA susceptibility (cases N=729, controls N=2,619). Associated PRS were replicated in an independent cohort (cases N=1,129, controls N=2,654) and their respective proteins were evaluated for causality using Mendelian randomization (MR). Finally, relationships between proteins with GCA-associated PRS were assessed using protein-protein interaction (PPI) network analysis

ResultsThe Apolipoprotein L1 (APOL1) PRS had a statistically significant GCA association with a protective effect (P-value[P]=1 x 10-4), which replicated in an independent dataset (P=8.69 x 10- 4), and MR analysis supported a causal relationship (beta=-0.093; SE=0.02; P=4.42 x 10-9). PPI network analysis of proteins with GCA-associated PRS revealed enrichment for &#34;negative regulation of fibrinolysis&#34; and &#34;negative regulation of blood coagulation&#34; pathways.

ConclusionsThis work emphasizes a potentially protective role of APOL1 and therefore reverse cholesterol transport in the pathogenesis of GCA. These findings also implicate fibrinolytic and coagulation cascades in GCA susceptibility, highlighting pathways that may be of interest for future pharmaceutical targeting.

Non-standard Abbreviations and AcronymsGCA, giant cell arteritis; MHC, major histocompatibility complex; GWAS, genome-wide association study; SNPs, single nucleotide polymorphisms; PRS, polygenic risk score; pQTL, protein quantitative trait loci; MR, Mendelian randomization; QC, quality control; WTCCC, Wellcome Trust Case Control Consortium; PCA, principal component analysis; IV, instrumental variables; IVW, inverse-variance weighted; PheWAS, phenome-wide association study; GO, gene ontology; FUMA, Functional Mapping and Annotation of Genome-Wide Association Studies; MAGMA, Multi-marker Analysis of GenoMic Annotation; RCT, reverse cholesterol transport.

Clinical PerspectiveO_ST_ABSWhat is new?C_ST_ABSO_LIAn apoliporotein-L1 polygenic risk score was associated with giant cell arteritis susceptibility, and replicated in an independent dataset.
C_LIO_LIEvidence for causality of a protective effect of apolipoprotein-L1 in giant cell arteritis susceptibility was identified using Mendelian randomization.
C_LIO_LIProteins with giant cell arteritis-associated polygenic risk scores were enriched in coagulation-related, fibrinolytic and immune response pathways.
C_LI

What are the clinical implications?O_LIFindings from this study indicate a protective role of apolipoprotein-L1 in giant cell arteritis susceptibility, highlighting a potential involvement of reverse cholesterol transport and lipid metabolism in disease pathogenesis.
C_LIO_LIFibrinolytic and coagulation cascades were also implicated in the disease in addition to innate immune response pathways, redrawing attention to the role of thromboinflammation and the need to re-evaluate anti-platelet and anticoagulant therapies, particularly for those with impending visual loss and cranial ischaemic complications.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.24301053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.24301053" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.24301053">
        <p class="paperTitle">Early detection of malignant and pre-malignant peripheral nerve tumors using cell-free DNA fragmentomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.24301053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.24301053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sundby, R. T.; Szymanski, J. J.; Pan, A.; Jones, P. A.; Mahmood, S. Z.; Reid, O. H.; Srihari, D.; Armstrong, A. E.; Chamberlain, S.; Burgic, S.; Weekley, K.; Murray, B.; Patel, S.; Lucas, A. N.; Fagan, M.; Dufek, A.; Meyer, C. F.; Collins, N. B.; Van Tine, B. A.; Dombi, E.; Gross, A. M.; Kim, A.; Chrisinger, J. S. A.; Dehner, C. A.; Widemann, B. C.; Hirbe, A. C.; Chaudhuri, A. A.; Shern, J. F.</p>
        <p class="info">Score: 16.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.24301053' target='https://doi.org/10.1101/2024.01.18.24301053'> 10.1101/2024.01.18.24301053</a></p>
        <p class="abstract">Early detection of neurofibromatosis type 1 (NF1) associated peripheral nerve sheath tumors (PNST) informs clinical decision-making, potentially averting deadly outcomes. Here, we describe a cell-free DNA (cfDNA) fragmentomic approach which distinguishes non-malignant, pre-malignant and malignant forms of NF1 PNST. Using plasma samples from a novel cohort of 101 NF1 patients and 21 healthy controls, we validated that our previous cfDNA copy number alteration (CNA)-based approach identifies malignant peripheral nerve sheath tumor (MPNST) but cannot distinguish among benign and premalignant states. We therefore investigated the ability of fragment-based cfDNA features to differentiate NF1-associated tumors including binned genome-wide fragment length ratios, end motif analysis, and non-negative matrix factorization deconvolution of fragment lengths. Fragmentomic methods were able to differentiate pre-malignant states including atypical neurofibromas (AN). Fragmentomics also adjudicated AN cases suspicious for MPNST, correctly diagnosing samples noninvasively, which could have informed clinical management. Overall, this study pioneers the early detection of malignant and premalignant peripheral nerve sheath tumors in NF1 patients using plasma cfDNA fragmentomics. In addition to screening applications, this novel approach distinguishes atypical neurofibromas from benign plexiform neurofibromas and malignant peripheral nerve sheath tumors, enabling more precise clinical diagnosis and management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.24301630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.24301630" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.24301630">
        <p class="paperTitle">A phenome-wide association study of tandem repeat variation in 168,554 individuals from the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.24301630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.24301630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manigbas, C.; Jadhav, B.; Garg, P.; Shadrina, M.; Lee, W.; Martin-Trujillo, A.; Sharp, A.</p>
        <p class="info">Score: 1.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.24301630' target='https://doi.org/10.1101/2024.01.22.24301630'> 10.1101/2024.01.22.24301630</a></p>
        <p class="abstract">Most genetic association studies focus on binary variants. To identify the effects of multi-allelic variation of tandem repeats (TRs) on human traits, we performed direct TR genotyping and phenome-wide association studies in 168,554 individuals from the UK Biobank, identifying 47 TRs showing causal associations with 73 traits. We replicated 23 of 31 (74%) of these causal associations in the All of Us cohort. While this set included several known repeat expansion disorders, novel associations we found were attributable to common polymorphic variation in TR length rather than rare expansions and include e.g. a coding polyhistidine motif in HRCT1 influencing risk of hypertension and a poly(CGC) in the 5UTR of GNB2 influencing heart rate. Causal TRs were strongly enriched for associations with local gene expression and DNA methylation. Our study highlights the contribution of multi-allelic TRs to the &#34;missing heritability&#34; of the human genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301165">
        <p class="paperTitle">Impact of genome build on RNA-seq interpretation and diagnostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ungar, R. A.; Goddard, P. C.; Jensen, T. D.; Degalez, F.; Smith, K. S.; Jin, C. A.; Undiagnosed Diseases Network,  ; Bonner, D. E.; Bernstein, J. A.; Wheeler, M. T.; Montgomery, S. B.</p>
        <p class="info">Score: 35.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301165' target='https://doi.org/10.1101/2024.01.11.24301165'> 10.1101/2024.01.11.24301165</a></p>
        <p class="abstract">Transcriptomics is a powerful tool for unraveling the molecular effects of genetic variants and disease diagnosis. Prior studies have demonstrated that choice of genome build impacts variant interpretation and diagnostic yield for genomic analyses. To identify the extent genome build also impacts transcriptomics analyses, we studied the effect of the hg19, hg38, and CHM13 genome builds on expression quantification and outlier detection in 386 rare disease and familial control samples from both the Undiagnosed Diseases Network (UDN) and Genomics Research to Elucidate the Genetics of Rare Disease (GREGoR) Consortium. We identified 2,800 genes with build-dependent quantification across six routinely-collected biospecimens, including 1,391 protein-coding genes and 341 known rare disease genes. We further observed multiple genes that only have detectable expression in a subset of genome builds. Finally, we characterized how genome build impacts the detection of outlier transcriptomic events. Combined, we provide a database of genes impacted by build choice, and recommend that transcriptomics-guided analyses and diagnoses are cross-referenced with these data for robustness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.24301544">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.24301544" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.24301544">
        <p class="paperTitle">AutScore- An integrative scoring approach for prioritization of ultra-rare autism spectrum disorder candidate variants from whole exome sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.24301544" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.24301544" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shil, A.; Arava, N.; Levi, N.; Levine, L.; Golan, H.; Meiri, G.; Michaelovski, A.; Tsadaka, Y.; Aran, A.; Menashe, I.</p>
        <p class="info">Score: 1.2, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.24301544' target='https://doi.org/10.1101/2024.01.24.24301544'> 10.1101/2024.01.24.24301544</a></p>
        <p class="abstract">BackgroundDiscerning clinically relevant ASD candidate variants from whole-exome sequencing (WES) data is complex, time-consuming, and labor-intensive. To this end, we developed AutScore, an integrative prioritization algorithm of ASD candidate variants from WES data, and assessed its performance to detect clinically relevant variants.

MethodsWe studied WES data from 581 ASD probands, and their parents registered in the Azrieli National Center database for Autism and Neurodevelopment Research. We focused on rare allele frequency &lt;1%), high-quality proband-specific variants affecting genes associated with ASD or other neurodevelopmental disorders (NDDs). We assigned a score (i.e., AutScore) to each such variant based on their pathogenicity, clinical relevance, gene-disease association, and inheritance patterns. Finally, we compared the AutScore performance with the rating of clinical experts and the NDD variants prioritization algorithm, AutoCasC.

ResultsOverall, 1161 ultra-rare variants distributed in 687 genes in 441 ASD probands were evaluated by AutScore with scores ranging from -4 to 25, with a mean {&#43;/-} SD of 5.89 {&#43;/-} 4.18. AutScore cut-off of [&amp;ge;] 12 outperforms AutoCasC in detecting clinically relevant ASD variants, with a detection accuracy rate of 72.3% and an overall diagnostic yield of 11.9%. Sixteen variants with AutScore of [&amp;ge;] 12 were distributed in fifteen novel ASD genes.

ConclusionAutScore is an effective automated ranking system for ASD candidate variants that could be implemented in ASD clinical genetics pipelines.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
