<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes
Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.
Score: 284.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799
The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes
Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.
Score: 284.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799
The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes
Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.
Score: 284.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799
The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes\nAuthors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.\nScore: 284.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799\nThe potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis.",
  "keywords": [
    
  ],
  "articleBody": " Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes\nAuthors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.\nScore: 284.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799\nThe potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis. An alternative promising approach involves expanding the capabilities of existing natural enzymes to enable functionality across new substrates and operational parameters. To this end we introduce CoSaNN (Conformation Sampling using Neural Network), a novel strategy for enzyme design that utilizes advances in deep learning for structure prediction and sequence optimization. By controlling enzyme conformations, we can expand the chemical space beyond the reach of simple mutagenesis. CoSaNN uses a context-dependent approach that accurately generates novel enzyme designs by considering non-linear relationships in both sequence and structure space. Additionally, we have further developed SolvIT, a graph neural network trained to predict protein solubility in E.Coli, as an additional optimization layer for producing highly expressed enzymes. Through this approach, we have engineered novel enzymes exhibiting superior expression levels, with 54% of our designs expressed in E.Coli, and increased thermal stability with more than 30% of our designs having a higher Tm than the template enzyme. Furthermore, our research underscores the transformative potential of AI in protein design, adeptly capturing high order interactions and preserving allosteric mechanisms in extensively modified enzymes. These advancements pave the way for the creation of diverse, functional, and robust enzymes, thereby opening new avenues for targeted biotechnological applications.\nVoyager: exploratory single-cell genomics data analysis with geospatial statistics\nAuthors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.\nScore: 84.4, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945\nExploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis. We introduce the Voyager platform, which systematically brings the geospatial ESDA tradition to (spatial) -omics, with local, bivariate, and multivariate spatial methods not yet commonly applied to spatial -omics, united by a uniform user interface. Using Voyager, we showcase biological insights that can be derived with its methods, such as biologically relevant negative spatial autocorrelation. Underlying Voyager is the SpatialFeatureExperiment data structure, which combines Simple Feature with SingleCellExperiment and AnnData to represent and operate on geometries bundled with gene expression data. Voyager has comprehensive tutorials demonstrating ESDA built on GitHub Actions to ensure reproducibility and scalability, using data from popular commercial technologies. Voyager is implemented in both R/Bioconductor and Python/PyPI, and features compatibility tests to ensure that both implementations return consistent results.\nNanoPyx: super-fast bioimage analysis powered by adaptive machine learning\nAuthors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.\nScore: 60.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080\nTo overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks. Unlike traditional single-algorithm methods, the Liquid Engine generates multiple CPU and GPU code variations using a meta-programming system, creating a competitive environment where different algorithms are benchmarked against each other to achieve optimal performance under the user\"s computational environment. In initial experiments focusing on super-resolution analysis methods, the Liquid Engine demonstrated an over 10-fold computational speed improvement by accurately predicting the ideal scenarios to switch between algorithmic implementations. NanoPyx is accessible to users through a Python library, code-free Jupyter notebooks, and a napari plugin, making it suitable for individuals regardless of their coding proficiency. Furthermore, the optimisation principles embodied by the Liquid Engine have broader implications, extending their applicability to various high-performance computing fields.\nDesign of Cyclic Peptides Targeting Protein-Protein Interactions using AlphaFold\nAuthors: Kosugi, T.; Ohue, M.\nScore: 32.9, Published: 2023-08-21 DOI: 10.1101/2023.08.20.554056\nMore than 930,000 protein-protein interactions (PPIs) have been identified in recent years, but their physicochemical properties differ from conventional drug targets, complicating the use of conventional small molecules as modalities. Cyclic peptides are a promising modality for targeting protein-protein interactions (PPIs), but it is difficult to predict the structure of a target protein-cyclic peptide complex or to design a cyclic peptide sequence that binds to the target protein using computational methods. Recently, AlphaFold with a cyclic offset has enabled predicting the structure of cyclic peptides, thereby enabling de novo cyclic peptide designs. We developed a cyclic peptide complex offset to enable the structural prediction of target proteins and cyclic peptide complexes and found AlphaFold2 with a cyclic peptide complex offset can predict structures with high accuracy. We also applied the cyclic peptide complex offset to the binder hallucination protocol of AfDesign, a de novo protein design method using AlphaFold, and we could design a high predicted local-distance difference test and lower separated binding energy per unit interface area than the native MDM2/p53 structure. Furthermore, the method was applied to 12 other protein-peptide complexes and one protein-protein complex. Our approach shows that it is possible to design putative cyclic peptide sequences targeting PPI.\nDNA language models are powerful predictors of genome-wide variant effects\nAuthors: Benegas, G.; Batra, S. S.; Song, Y. S.\nScore: 171.3, Published: 2023-08-04 DOI: 10.1101/2022.08.22.504706\nThe expanding catalog of genome-wide association studies (GWAS) provides biological insights across a variety of species, but identifying the causal variants behind these associations remains a significant challenge. Experimental validation is both labor-intensive and costly, highlighting the need for accurate, scalable computational methods to predict the effects of genetic variants across the entire genome. Inspired by recent progress in natural language processing, unsupervised pre-training on large protein sequence databases has proven successful in extracting complex information related to proteins. These models showcase their ability to learn variant effects in coding regions using an unsupervised approach. Expanding on this idea, we here introduce the Genomic Pre-trained Network (GPN), a model designed to learn genome-wide variant effects through unsupervised pre-training on genomic DNA sequences. Our model also successfully learns gene structure and DNA motifs without any supervision. To demonstrate its utility, we train GPN on unaligned reference genomes of Arabidopsis thaliana and seven related species within the Brassicales order, and evaluate its ability to predict the functional impact of genetic variants in Arabidopsis thaliana by utilizing allele frequencies from the 1001 Genomes Project and a comprehensive database of GWAS. Notably, GPN outperforms predictors based on popular conservation scores such as phyloP and phastCons. Our predictions for Arabidopsis thaliana can be visualized as sequence logos in the UCSC Genome Browser (https://genome.ucsc.edu/s/gbenegas/gpn-arabidopsis). We provide code (https://github.com/songlab-cal/gpn) to train GPN for any given species using its DNA sequence alone, enabling unsupervised prediction of variant effects across the entire genome.\nProtein Structure Refinement via DeepTracer and AlphaFold2\nAuthors: Chen, J.; Zia, A.; Hou, J.; Wang, F.; Cao, R.; Si, D.\nScore: 25.8, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553616\nUnderstanding the structures of proteins has numerous applications, such as vaccine development. It is a slow and labor-intensive task to manually build protein structures from experimental electron density maps, therefore, machine learning approaches have been proposed to automate this process. However, most of the experimental maps are not atomic resolution, so they are insufficient for computer vision-based machine learning methods to precisely determine the protein structure. On the other hand, methods that utilize evolutionary information from protein sequences to predict structures, like AlphaFold2, have recently achieved groundbreaking accuracy but often require manual effort to refine the results. We propose DeepTracer-Refine, an automated method to refine AlphaFold structures by aligning them to DeepTracers predicted structure. We tested our method on 39 multi-domain proteins and we improved the average residue coverage from 78.2% to 90.0% and average lDDT score from 0.67 to 0.71. We also compared DeepTracer-Refine against another method, Phenixs AlphaFold refinement, to demonstrate that our method not only performs better when the initial AlphaFold model is less precise but also exceeds Phenix in run-time performance.\nSearch and Match across Spatial Omics Samples at Single-cell Resolution\nAuthors: Tang, Z.; Luo, S.; Zeng, H.; Huang, J.; Wu, M.; Wang, X.\nScore: 23.4, Published: 2023-08-15 DOI: 10.1101/2023.08.13.552987\nSpatial omics technologies characterize tissue molecular properties with spatial information, but integrating and comparing spatial data across different technologies and modalities is challenging. A comparative analysis tool that can search, match, and visualize both similarities and differences of molecular features in space across multiple samples is lacking. To address this, we introduce CAST (Cross-sample Alignment of SpaTial omics), a deep graph neural network (GNN)-based method enabling spatial-to-spatial searching and matching at the single-cell level. CAST aligns tissues based on intrinsic similarities of spatial molecular features and reconstructs spatially resolved single-cell multi-omic profiles. CAST enables spatially resolved differential analysis ({Delta}Analysis) to pinpoint and visualize disease-associated molecular pathways and cell-cell interactions, and single-cell relative translational efficiency (scRTE) profiling to reveal variations in translational control across cell types and regions. CAST serves as an integrative framework for seamless single-cell spatial data searching and matching across technologies, modalities, and disease conditions, analogous to BLAST in sequence alignment.\nDeGenPrime provides robust primer design and optimization1 unlocking the biosphere\nAuthors: Fulghum, B. W.; Tanker, S.; White, R. A.\nScore: 18.7, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553048\nPolymerase chain reaction (PCR) is the worlds most important molecular diagnostic with applications ranging from medicine to ecology. PCR can fail because of poor primer design. The nearest-neighbor thermodynamic properties, picking conserved regions, and filtration via penalty of oligonucleotides form the basis for good primer design. DeGenPrime is a console-based high quality PCR primer design tool that can utilize MSA formats and degenerate bases expanding target range for a single primer set. Our software utilizes thermodynamic properties, filtration metrics, penalty scoring, and conserved region finding of any proposed primer. It has degeneracy, repeated k-mers, relative GC content, and temperature range filters. Minimal penalty scoring is included according to secondary structure self-dimerization metrics, GC clamping, tri- and tetra-loop hairpins and internal repetition. We compared PrimerDesign-M, DegePrime, ConsensusPrimer, and DeGenPrime on acceptable primer yield. PrimerDesign-M, DegePrime, and ConsensusPrimer provided 0%, 11%, and 17% yield respectively for alternative iron nitrogenase (anfD) gene target. DeGenPrime successfully identified quality primers within the conserved regions of the T4-like phage major capsid protein (g23), conserved regions of molybdenum-based nitrogenase (nif), and its alternatives vanadium (vnf) and iron (anf) nitrogenase. DeGenPrime provides a universal and scalable primer design tool for the entire tree of life.\nClairS: a deep-learning method for long-read somatic small variant calling\nAuthors: Zheng, Z.; Su, J.; Chen, L.; Lee, Y.-L.; Lam, T.-W.; Luo, R.\nScore: 15.0, Published: 2023-08-21 DOI: 10.1101/2023.08.17.553778\nIdentifying somatic variants in tumor samples is a crucial task, which is often performed using statistical methods and heuristic filters applied to short-read data. However, with the increasing demand for long-read somatic variant calling, existing methods have fallen short. To address this gap, we present ClairS, the first deep-learning-based, long-read somatic small variant caller. ClairS was trained on massive synthetic somatic variants with diverse coverages and variant allele frequencies (VAF), enabling it to accurately detect a wide range of somatic variants from paired tumor and normal samples. We evaluated ClairS using the latest Nanopore Q20+ HCC1395-HCC1395BL dataset. With 50-fold/25-fold tumor/normal, ClairS achieved a 93.01%/86.86% precision/recall rate for Single Nucleotide Variation (SNVs), and 66.54%/66.89% for somatic insertions and deletions (Indels). Applying ClairS to short-read datasets from multiple sources showed comparable or better performance than Strelka2 and Mutect2. Our findings suggest that improved read phasing enabled by long-read sequencing is key to accurate long-read SNV calling, especially for variants with low VAF. Through experiments across various coverage, purity, and contamination settings, we demonstrated that ClairS is a reliable somatic variant caller. ClairS is open-source at https://github.com/HKU-BAL/ClairS.\nAlignment of spatial transcriptomics data using diffeomorphic metric mapping\nAuthors: Clifton, K.; Anant, M.; Aihara, G.; Atta, L.; Aimiuwu, O. K.; Kebschull, J. M.; Miller, M. I.; Tward, D.; Fan, J.\nScore: 43.4, Published: 2023-08-19 DOI: 10.1101/2023.04.11.534630\nSpatial transcriptomics (ST) technologies enable high throughput gene expression characterization within thin tissue sections. However, comparing spatial observations across sections, samples, and technologies remains challenging. To address this challenge, we developed STalign to align ST datasets in a manner that accounts for partially matched tissue sections and other local non-linear distortions using diffeomorphic metric mapping. We apply STalign to align ST datasets within and across technologies as well as to align ST datasets to a 3D common coordinate framework. We show that STalign achieves high gene expression and cell-type correspondence across matched spatial locations that is significantly improved over landmark-based affine alignments. Applying STalign to align ST datasets of the mouse brain to the 3D common coordinate framework from the Allen Brain Atlas, we highlight how STalign can be used to lift over brain region annotations and enable the interrogation of compositional heterogeneity across anatomical structures. STalign is available as an open-source Python toolkit at https://github.com/JEFworks-Lab/STalign and as supplementary software with additional documentation and tutorials available at https://jef.works/STalign.\n",
  "wordCount" : "2200",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550799">
        <p class="paperTitle">Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.</p>
        <p class="info">Score: 284.3, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550799' target='https://doi.org/10.1101/2023.07.27.550799'> 10.1101/2023.07.27.550799</a></p>
        <p class="abstract">The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis. An alternative promising approach involves expanding the capabilities of existing natural enzymes to enable functionality across new substrates and operational parameters. To this end we introduce CoSaNN (Conformation Sampling using Neural Network), a novel strategy for enzyme design that utilizes advances in deep learning for structure prediction and sequence optimization. By controlling enzyme conformations, we can expand the chemical space beyond the reach of simple mutagenesis. CoSaNN uses a context-dependent approach that accurately generates novel enzyme designs by considering non-linear relationships in both sequence and structure space. Additionally, we have further developed SolvIT, a graph neural network trained to predict protein solubility in E.Coli, as an additional optimization layer for producing highly expressed enzymes. Through this approach, we have engineered novel enzymes exhibiting superior expression levels, with 54% of our designs expressed in E.Coli, and increased thermal stability with more than 30% of our designs having a higher Tm than the template enzyme. Furthermore, our research underscores the transformative potential of AI in protein design, adeptly capturing high order interactions and preserving allosteric mechanisms in extensively modified enzymes. These advancements pave the way for the creation of diverse, functional, and robust enzymes, thereby opening new avenues for targeted biotechnological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549945">
        <p class="paperTitle">Voyager: exploratory single-cell genomics data analysis with geospatial statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.</p>
        <p class="info">Score: 84.4, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549945' target='https://doi.org/10.1101/2023.07.20.549945'> 10.1101/2023.07.20.549945</a></p>
        <p class="abstract">Exploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis. We introduce the Voyager platform, which systematically brings the geospatial ESDA tradition to (spatial) -omics, with local, bivariate, and multivariate spatial methods not yet commonly applied to spatial -omics, united by a uniform user interface. Using Voyager, we showcase biological insights that can be derived with its methods, such as biologically relevant negative spatial autocorrelation. Underlying Voyager is the SpatialFeatureExperiment data structure, which combines Simple Feature with SingleCellExperiment and AnnData to represent and operate on geometries bundled with gene expression data. Voyager has comprehensive tutorials demonstrating ESDA built on GitHub Actions to ensure reproducibility and scalability, using data from popular commercial technologies. Voyager is implemented in both R/Bioconductor and Python/PyPI, and features compatibility tests to ensure that both implementations return consistent results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553080">
        <p class="paperTitle">NanoPyx: super-fast bioimage analysis powered by adaptive machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.</p>
        <p class="info">Score: 60.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553080' target='https://doi.org/10.1101/2023.08.13.553080'> 10.1101/2023.08.13.553080</a></p>
        <p class="abstract">To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks. Unlike traditional single-algorithm methods, the Liquid Engine generates multiple CPU and GPU code variations using a meta-programming system, creating a competitive environment where different algorithms are benchmarked against each other to achieve optimal performance under the user&#34;s computational environment. In initial experiments focusing on super-resolution analysis methods, the Liquid Engine demonstrated an over 10-fold computational speed improvement by accurately predicting the ideal scenarios to switch between algorithmic implementations. NanoPyx is accessible to users through a Python library, code-free Jupyter notebooks, and a napari plugin, making it suitable for individuals regardless of their coding proficiency. Furthermore, the optimisation principles embodied by the Liquid Engine have broader implications, extending their applicability to various high-performance computing fields.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554056">
        <p class="paperTitle">Design of Cyclic Peptides Targeting Protein-Protein Interactions using AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kosugi, T.; Ohue, M.</p>
        <p class="info">Score: 32.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554056' target='https://doi.org/10.1101/2023.08.20.554056'> 10.1101/2023.08.20.554056</a></p>
        <p class="abstract">More than 930,000 protein-protein interactions (PPIs) have been identified in recent years, but their physicochemical properties differ from conventional drug targets, complicating the use of conventional small molecules as modalities. Cyclic peptides are a promising modality for targeting protein-protein interactions (PPIs), but it is difficult to predict the structure of a target protein-cyclic peptide complex or to design a cyclic peptide sequence that binds to the target protein using computational methods. Recently, AlphaFold with a cyclic offset has enabled predicting the structure of cyclic peptides, thereby enabling de novo cyclic peptide designs. We developed a cyclic peptide complex offset to enable the structural prediction of target proteins and cyclic peptide complexes and found AlphaFold2 with a cyclic peptide complex offset can predict structures with high accuracy. We also applied the cyclic peptide complex offset to the binder hallucination protocol of AfDesign, a de novo protein design method using AlphaFold, and we could design a high predicted local-distance difference test and lower separated binding energy per unit interface area than the native MDM2/p53 structure. Furthermore, the method was applied to 12 other protein-peptide complexes and one protein-protein complex. Our approach shows that it is possible to design putative cyclic peptide sequences targeting PPI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.22.504706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.22.504706" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.22.504706">
        <p class="paperTitle">DNA language models are powerful predictors of genome-wide variant effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.22.504706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.22.504706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benegas, G.; Batra, S. S.; Song, Y. S.</p>
        <p class="info">Score: 171.3, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.22.504706' target='https://doi.org/10.1101/2022.08.22.504706'> 10.1101/2022.08.22.504706</a></p>
        <p class="abstract">The expanding catalog of genome-wide association studies (GWAS) provides biological insights across a variety of species, but identifying the causal variants behind these associations remains a significant challenge. Experimental validation is both labor-intensive and costly, highlighting the need for accurate, scalable computational methods to predict the effects of genetic variants across the entire genome. Inspired by recent progress in natural language processing, unsupervised pre-training on large protein sequence databases has proven successful in extracting complex information related to proteins. These models showcase their ability to learn variant effects in coding regions using an unsupervised approach. Expanding on this idea, we here introduce the Genomic Pre-trained Network (GPN), a model designed to learn genome-wide variant effects through unsupervised pre-training on genomic DNA sequences. Our model also successfully learns gene structure and DNA motifs without any supervision. To demonstrate its utility, we train GPN on unaligned reference genomes of Arabidopsis thaliana and seven related species within the Brassicales order, and evaluate its ability to predict the functional impact of genetic variants in Arabidopsis thaliana by utilizing allele frequencies from the 1001 Genomes Project and a comprehensive database of GWAS. Notably, GPN outperforms predictors based on popular conservation scores such as phyloP and phastCons. Our predictions for Arabidopsis thaliana can be visualized as sequence logos in the UCSC Genome Browser (https://genome.ucsc.edu/s/gbenegas/gpn-arabidopsis). We provide code (https://github.com/songlab-cal/gpn) to train GPN for any given species using its DNA sequence alone, enabling unsupervised prediction of variant effects across the entire genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553616">
        <p class="paperTitle">Protein Structure Refinement via DeepTracer and AlphaFold2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Zia, A.; Hou, J.; Wang, F.; Cao, R.; Si, D.</p>
        <p class="info">Score: 25.8, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553616' target='https://doi.org/10.1101/2023.08.16.553616'> 10.1101/2023.08.16.553616</a></p>
        <p class="abstract">Understanding the structures of proteins has numerous applications, such as vaccine development. It is a slow and labor-intensive task to manually build protein structures from experimental electron density maps, therefore, machine learning approaches have been proposed to automate this process. However, most of the experimental maps are not atomic resolution, so they are insufficient for computer vision-based machine learning methods to precisely determine the protein structure. On the other hand, methods that utilize evolutionary information from protein sequences to predict structures, like AlphaFold2, have recently achieved groundbreaking accuracy but often require manual effort to refine the results. We propose DeepTracer-Refine, an automated method to refine AlphaFold structures by aligning them to DeepTracers predicted structure. We tested our method on 39 multi-domain proteins and we improved the average residue coverage from 78.2% to 90.0% and average lDDT score from 0.67 to 0.71. We also compared DeepTracer-Refine against another method, Phenixs AlphaFold refinement, to demonstrate that our method not only performs better when the initial AlphaFold model is less precise but also exceeds Phenix in run-time performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.552987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.552987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.552987">
        <p class="paperTitle">Search and Match across Spatial Omics Samples at Single-cell Resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.552987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.552987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tang, Z.; Luo, S.; Zeng, H.; Huang, J.; Wu, M.; Wang, X.</p>
        <p class="info">Score: 23.4, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.552987' target='https://doi.org/10.1101/2023.08.13.552987'> 10.1101/2023.08.13.552987</a></p>
        <p class="abstract">Spatial omics technologies characterize tissue molecular properties with spatial information, but integrating and comparing spatial data across different technologies and modalities is challenging. A comparative analysis tool that can search, match, and visualize both similarities and differences of molecular features in space across multiple samples is lacking. To address this, we introduce CAST (Cross-sample Alignment of SpaTial omics), a deep graph neural network (GNN)-based method enabling spatial-to-spatial searching and matching at the single-cell level. CAST aligns tissues based on intrinsic similarities of spatial molecular features and reconstructs spatially resolved single-cell multi-omic profiles. CAST enables spatially resolved differential analysis ({Delta}Analysis) to pinpoint and visualize disease-associated molecular pathways and cell-cell interactions, and single-cell relative translational efficiency (scRTE) profiling to reveal variations in translational control across cell types and regions. CAST serves as an integrative framework for seamless single-cell spatial data searching and matching across technologies, modalities, and disease conditions, analogous to BLAST in sequence alignment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553048" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553048">
        <p class="paperTitle">DeGenPrime provides robust primer design and optimization1 unlocking the biosphere</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fulghum, B. W.; Tanker, S.; White, R. A.</p>
        <p class="info">Score: 18.7, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553048' target='https://doi.org/10.1101/2023.08.11.553048'> 10.1101/2023.08.11.553048</a></p>
        <p class="abstract">Polymerase chain reaction (PCR) is the worlds most important molecular diagnostic with applications ranging from medicine to ecology. PCR can fail because of poor primer design. The nearest-neighbor thermodynamic properties, picking conserved regions, and filtration via penalty of oligonucleotides form the basis for good primer design.

DeGenPrime is a console-based high quality PCR primer design tool that can utilize MSA formats and degenerate bases expanding target range for a single primer set. Our software utilizes thermodynamic properties, filtration metrics, penalty scoring, and conserved region finding of any proposed primer. It has degeneracy, repeated k-mers, relative GC content, and temperature range filters. Minimal penalty scoring is included according to secondary structure self-dimerization metrics, GC clamping, tri- and tetra-loop hairpins and internal repetition.

We compared PrimerDesign-M, DegePrime, ConsensusPrimer, and DeGenPrime on acceptable primer yield. PrimerDesign-M, DegePrime, and ConsensusPrimer provided 0%, 11%, and 17% yield respectively for alternative iron nitrogenase (anfD) gene target. DeGenPrime successfully identified quality primers within the conserved regions of the T4-like phage major capsid protein (g23), conserved regions of molybdenum-based nitrogenase (nif), and its alternatives vanadium (vnf) and iron (anf) nitrogenase. DeGenPrime provides a universal and scalable primer design tool for the entire tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553778">
        <p class="paperTitle">ClairS: a deep-learning method for long-read somatic small variant calling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, Z.; Su, J.; Chen, L.; Lee, Y.-L.; Lam, T.-W.; Luo, R.</p>
        <p class="info">Score: 15.0, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553778' target='https://doi.org/10.1101/2023.08.17.553778'> 10.1101/2023.08.17.553778</a></p>
        <p class="abstract">Identifying somatic variants in tumor samples is a crucial task, which is often performed using statistical methods and heuristic filters applied to short-read data. However, with the increasing demand for long-read somatic variant calling, existing methods have fallen short. To address this gap, we present ClairS, the first deep-learning-based, long-read somatic small variant caller. ClairS was trained on massive synthetic somatic variants with diverse coverages and variant allele frequencies (VAF), enabling it to accurately detect a wide range of somatic variants from paired tumor and normal samples. We evaluated ClairS using the latest Nanopore Q20&#43; HCC1395-HCC1395BL dataset. With 50-fold/25-fold tumor/normal, ClairS achieved a 93.01%/86.86% precision/recall rate for Single Nucleotide Variation (SNVs), and 66.54%/66.89% for somatic insertions and deletions (Indels). Applying ClairS to short-read datasets from multiple sources showed comparable or better performance than Strelka2 and Mutect2. Our findings suggest that improved read phasing enabled by long-read sequencing is key to accurate long-read SNV calling, especially for variants with low VAF. Through experiments across various coverage, purity, and contamination settings, we demonstrated that ClairS is a reliable somatic variant caller. ClairS is open-source at https://github.com/HKU-BAL/ClairS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.11.534630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.11.534630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.11.534630">
        <p class="paperTitle">Alignment of spatial transcriptomics data using diffeomorphic metric mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.11.534630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.11.534630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clifton, K.; Anant, M.; Aihara, G.; Atta, L.; Aimiuwu, O. K.; Kebschull, J. M.; Miller, M. I.; Tward, D.; Fan, J.</p>
        <p class="info">Score: 43.4, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.11.534630' target='https://doi.org/10.1101/2023.04.11.534630'> 10.1101/2023.04.11.534630</a></p>
        <p class="abstract">Spatial transcriptomics (ST) technologies enable high throughput gene expression characterization within thin tissue sections. However, comparing spatial observations across sections, samples, and technologies remains challenging. To address this challenge, we developed STalign to align ST datasets in a manner that accounts for partially matched tissue sections and other local non-linear distortions using diffeomorphic metric mapping. We apply STalign to align ST datasets within and across technologies as well as to align ST datasets to a 3D common coordinate framework. We show that STalign achieves high gene expression and cell-type correspondence across matched spatial locations that is significantly improved over landmark-based affine alignments. Applying STalign to align ST datasets of the mouse brain to the 3D common coordinate framework from the Allen Brain Atlas, we highlight how STalign can be used to lift over brain region annotations and enable the interrogation of compositional heterogeneity across anatomical structures. STalign is available as an open-source Python toolkit at https://github.com/JEFworks-Lab/STalign and as supplementary software with additional documentation and tutorials available at https://jef.works/STalign.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
