<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Petascale Homology Search for Structure Prediction
Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.
Score: 72.9, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308
The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Petascale Homology Search for Structure Prediction
Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.
Score: 72.9, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308
The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Petascale Homology Search for Structure Prediction
Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.
Score: 72.9, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308
The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Petascale Homology Search for Structure Prediction\nAuthors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.\nScore: 72.9, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308\nThe recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets.",
  "keywords": [
    
  ],
  "articleBody": " Petascale Homology Search for Structure Prediction\nAuthors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.\nScore: 72.9, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308\nThe recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets. These were merged with default MSAs produced by ColabFold-search and provided to ColabFold-predict. By using SRA data, we achieved highly accurate predictions (GDT_TS \u003e 70) for 66% of the non-easy targets, whereas using ColabFold-search default MSAs scored highly in only 52%. Next, we tested the effect of deep homology search and ColabFolds advanced features, such as more recycles, on prediction accuracy. While SRA homologs were most significant for improving ColabFolds CASP15 ranking from 11th to 3rd place, other strategies contributed too. We analyze these in the context of existing strategies to improve prediction.\nSimulations of sequence evolution: how (un)realistic they really are and why\nAuthors: Trost, J.; Haag, J.; Hoehler, D.; Nesterenko, L.; Jacob, L.; Stamatakis, A.; Boussau, B.\nScore: 30.3, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548509\nMotivationSimulating sequence evolution plays an important role in the development and evaluation of phylogenetic inference tools. Naturally, the simulated data needs to be as realistic as possible to be indicative of the performance of the developed tools on empirical data. Over the years, numerous phylogenetic sequence simulators, employing various models of evolution, have been published with the goal to simulate such empirical-like data. In this study, we simulated DNA and protein Multiple Sequence Alignments (MSAs) under increasingly complex models of evolution with and without insertion/deletion (indel) events using a state-of-the-art sequence simulator. We assessed their realism by quantifying how well supervised learning methods are able to predict whether a given MSA is simulated or empirical. ResultsOur results show that we can distinguish between empirical and simulated MSAs with high accuracy using two distinct and independently developed classification approaches across all tested models of sequence evolution. Our findings suggest that the current state-of-the-art models fail to accurately replicate the process of evolution. Data and Code AvailabilityAll simulated and empirical MSAs, as well as all analysis results, are available at https://cme.h-its.org/exelixis/material/simulation_study.tar.gz. All scripts required to reproduce our results are available at https://github.com/tschuelia/SimulationStudy and https://github.com/JohannaTrost/seqsharp. Contactjulia.haag@h-its.org\nEfficient High-Quality Metagenome Assembly from Long Accurate Reads using Minimizer-space de Bruijn Graphs\nAuthors: Benoit, G.; Raguideau, S.; James, R.; Phillippy, A. M.; Chikhi, R.; Quince, C.\nScore: 58.7, Published: 2023-07-08 DOI: 10.1101/2023.07.07.548136\nWe introduce a novel metagenomics assembler for high-accuracy long reads. Our approach, implemented as metaMDBG, combines highly efficient de Bruijn graph assembly in minimizer space, with both a multi-k' approach for dealing with variations in genome coverage depth and an abundance-based filtering strategy for simplifying strain complexity. The resulting algorithm is more efficient than the state-of-the-art but with better assembly results. metaMDBG was 1.5 to 12 times faster than competing assemblers and requires between one-tenth and one-thirtieth of the memory across a range of data sets. We obtained up to twice as many high-quality circularised prokaryotic metagenome assembled genomes (MAGs) on the most complex communities, and a better recovery of viruses and plasmids. metaMDBG performs particularly well for abundant organisms whilst being robust to the presence of strain diversity. The result is that for the first time it is possible to efficiently reconstruct the majority of complex communities by abundance as nearcomplete MAGs.\nIdentifying Protein Conformational States in the PDB and Comparison to AlphaFold2 Predictions\nAuthors: Ellaway, J. I. J.; Anyango, S.; Nair, S.; Zaki, H. A.; Nadzirin, N.; Powell, H. R.; Gutmanas, A.; Varadi, M.; Velankar, S.\nScore: 28.6, Published: 2023-07-14 DOI: 10.1101/2023.07.13.545008\nProteins, as molecular machines, are necessarily dynamic macromolecules that carry out essential cellular functions. Recognising their stable conformations is important for understanding the molecular mechanisms of disease. While AI-based computational methods have enabled protein structure prediction, the prediction of protein dynamics remains a challenge. Here, we present a deterministic pipeline that clusters experimentally determined protein structures to comprehensively recognise conformational states across the Protein Data Bank. Our approach clusters protein chains based on a GLObal CONformation (GLOCON) difference score, which is computed from pairwise C-alpha distances. By superposing the clustered structures, differences and similarities in conformational states can be observed. Additionally, we offer users the ability to superpose predicted models from the AlphaFold Database to the clusters of PDB structures. This clustering pipeline significantly advances researchers ability to explore the conformational landscape within the PDB. All clustered and superposed models can be viewed in Mol* on the PDBe Knowledge Base website, or accessed in as raw annotations via our GraphAPI and FTP server. The clustering package is made available as an open-source Python3 package under the Apache-2.0 license.\nFLAG: Find, Label Annotate Genomes, a fully automated tool for genome gene structural and functional annotation of highly fragmented non-model species\nAuthors: Troy, W.; Damas, J.; Titus, A. J.; Cantarel, B. L.\nScore: 27.1, Published: 2023-07-17 DOI: 10.1101/2023.07.14.548907\nRecent advances in long-read sequencing technologies and the efforts of projects aimed at increasing the universe sequenced reference genomes, such as the Earth Biogenome project have led to a growth in the number of whole genome sequencing projects for non-model organisms. Still, the vast majority (6,880) of the 28,727 unique publicly available eukaryotic genomes lack gene structure annotations. Annotating genes in a genome of interest is a multistep process, and while there are many tools available for each step of the annotation, there are few end-to-end workflows that (i) run on multiple computing environments, (ii) automatically run without initial data training and (iii) perform accurately with fragmented genomes. Here we present \"Find, Label, Annotate Genomes\" (FLAG), a fully automated genome annotation workflow that does not require species-specific extrinsic evidence to create highly complete and accurate gene annotations of even highly fragmented genomes. Using FLAG, various eukaryotic genome assemblies, including 1 plant, 5 living animals, and 1 extinct animal were annotated. In all living species, FLAG annotations provided an average of 18% increase in complete BUSCO scores when compared to BRAKER2. Furthermore, FLAG has more accurately predicted the number of protein-coding genes, with an average error rate 15x lower than BRAKER2.\nARCTIC-3D: Automatic Retrieval and ClusTering of Interfaces in Complexes from 3D structural information\nAuthors: Giulini, M.; Honorato, R. V.; Rivera, J. L.; Bonvin, A. M. J. J.\nScore: 26.7, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548477\nThe formation of a stable complex between proteins lies at the core of a wide variety of biological processes and has been the focus of countless experiments. The huge amount of information contained in the protein structural interactome in the Protein Data Bank can now be used to characterise and classify the existing biological interfaces. We here introduce ARCTIC-3D, a fast and user-friendly data mining and clustering software to retrieve data and rationalise the interface information associated with the protein input data. We demonstrate its use by various examples ranging from showing the increased interaction complexity of eukaryotic proteins, 20% of which on average have more than 3 different interfaces compared to only 10% for prokaryotes, to associating different functions to different interfaces. In the context of modelling biomolecular assemblies, we introduce the concept of \"recognition entropy\", related to the number of possible interfaces of the components of a protein-protein complex, which we demonstrate to correlate with the modelling difficulty. The identified interface clusters can also be used to generate various combinations of interface-specific restraints for integrative modelling. The ARCTIC-3D software is freely available at https://github.com/haddocking/arctic3d and can be accessed as a web-service at https://wenmr.science.uu.nl/arctic-3d\nTutorial: guidelines for manual cell type annotation of single-cell multi-omics datasets using interactive software\nAuthors: Kim, Y. J.; Tarashansky, A.; Liang, K.; Urisko, M.; Dorman, L.; Borja, M.; Neff, N.; Pisco, A. O.; Granados, A.\nScore: 23.2, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548639\nAssigning cell identity to clusters of single cells is an essential step towards extracting biological insights from many genomics datasets. Although annotation workflows for datasets built with a single modality are well established, limitations exist in annotating cell types in datasets with multiple modalities due to the need for a framework to exploit them jointly. While, in principle, different modalities could convey complementary information about cell identity, it is unclear to what extent they can be combined to improve the accuracy and resolution of cell type annotations. Here, we present a conceptual framework to examine and jointly interrogate distinct modalities to identify cell types. We integrated our framework into a series of vignettes, using immune cells as a well-studied example, and demonstrate cell type annotation workflows ranging from using single-cell RNA-seq datasets alone, to using multiple modalities such as single-cell Multiome (RNA and chromatin accessibility), CITE-seq (RNA and surface proteins). In some cases, one or other single modality is superior to the other for identification of specific cell types, in others combining the two modalities improves resolution and the ability to identify finer subpopulations. Finally, we use interactive software from CZ CELLxGENE community tools to visualize and integrate histological and spatial transcriptomic data.\nThe Patent and Literature Antibody Database (PLAbDab): an evolving reference set of functionally diverse, literature-annotated antibody sequences and structures\nAuthors: Abanades, B.; Olsen, T. H.; Raybould, M. I. J.; Aguilar-Sanjuan, B.; Wong, W. K.; Georges, G.; Bujotzek, A.; Deane, C. M.\nScore: 21.8, Published: 2023-07-16 DOI: 10.1101/2023.07.15.549143\nAntibodies are key proteins of the adaptive immune system, and there exists a large body of academic literature and patents dedicated to their study and concomitant conversion into therapeutics, diagnostics, or reagents. These documents often contain extensive functional characterisations of the sets of antibodies the describe. However, leveraging these heterogeneous reports, for example to offer insights into the properties of query antibodies of interest, is currently challenging as there is no central repository through which this wide corpus can be mined by sequence or structure. Here, we present PLAbDab (the Patent and Literature Antibody Database), a self-updating repository containing over 150,000 paired antibody sequences and 3D structural models, of which over 65,000 are unique. Each entry in the database also contains the title and authors of its literature source. Here we describe the methods used to extract, filter, pair, and model the antibodies in PLAbDab, and showcase how PLAbDab can be searched by sequence, structure, or keyword. PLAbDab uses include annotating query antibodies with potential antigen information from similar entries, analysing structural models of existing antibodies to identify modifications that could improve their properties, and compiling bespoke datasets of antibody sequences/structures known to bind to a specific antigen. PLAbDab is freely available via Github (https://github.com/oxpig/PLAbDab) and as a searchable webserver (https://opig.stats.ox.ac.uk/webapps/plabdab).\nAI2BMD: efficient characterization of protein dynamics with ab initio accuracy\nAuthors: Wang, T.; He, X.; Li, M.; Wang, Y.; Wang, Z.; Li, S.; Shao, B.; Liu, T.-Y.\nScore: 20.6, Published: 2023-07-13 DOI: 10.1101/2023.07.12.548519\nBiomolecular dynamics simulation is a fundamental technology for life sciences research, and its usefulness depends on its accuracy and efficiency. Classical molecular dynamics simulation is fast but lacks chemical accuracy. Quantum chemistry methods like density functional theory (DFT) can reach chemical accuracy but cannot scale to support large biomolecules. We introduce an AI-based ab initio biomolecular dynamics system (AI2BMD) that can efficiently simulate large biomolecules with ab initio accuracy. AI2BMD uses a protein fragmentation scheme and machine learning force field to achieve generalizable ab initio accuracy for energy and force calculations for various proteins comprising over 10,000 atoms. Compared to DFT, it reduces computational time by several orders of magnitude. With several hundred nanoseconds of dynamics simulations, AI2BMD demonstrated its capability of efficiently exploring the conformational space of peptides and proteins, deriving accurate 3J-couplings that match NMR experiments, and showing protein folding and unfolding tendencies. Furthermore, AI2BMD enables precise free energy calculations for protein folding, and the estimated melting temperatures are well aligned with experiments. AI2BMD could potentially complement wet-lab experiments, detect the dynamic processes of bioactivities, and enable biomedical research that is currently impossible to conduct.\nUNI-RNA: UNIVERSAL PRE-TRAINED MODELS REVOLUTIONIZE RNA RESEARCH\nAuthors: Wang, X.; Gu, R.; Chen, Z.; Li, Y.; Ji, X.; Ke, G.; Wen, H.\nScore: 18.1, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548588\nAO_SCPLOWBSTRACTC_SCPLOWRNA molecules play a crucial role as intermediaries in diverse biological processes. Attaining a profound understanding of their function can substantially enhance our comprehension of lifes activities and facilitate drug development for numerous diseases. The advent of high-throughput sequencing technologies makes vast amounts of RNA sequence data accessible, which contains invaluable information and knowledge. However, deriving insights for further application from such an immense volume of data poses a significant challenge. Fortunately, recent advancements in pre-trained models have surfaced as a revolutionary solution for addressing such challenges owing to their exceptional ability to automatically mine and extract hidden knowledge from massive datasets. Inspired by the past successes, we developed a novel context-aware deep learning model named Uni-RNA that performs pre-training on the largest dataset of RNA sequences at the unprecedented scale to date. During this process, our model autonomously unraveled the obscured evolutionary and structural information embedded within the RNA sequences. As a result, through fine-tuning, our model achieved the state-of-the-art (SOTA) performances in a spectrum of downstream tasks, including both structural and functional predictions. Overall, Uni-RNA established a new research paradigm empowered by the large pre-trained model in the field of RNA, enabling the community to unlock the power of AI at a whole new level to significantly expedite the pace of research and foster groundbreaking discoveries.\n",
  "wordCount" : "2244",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548308">
        <p class="paperTitle">Petascale Homology Search for Structure Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.</p>
        <p class="info">Score: 72.9, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548308' target='https://doi.org/10.1101/2023.07.10.548308'> 10.1101/2023.07.10.548308</a></p>
        <p class="abstract">The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets. These were merged with default MSAs produced by ColabFold-search and provided to ColabFold-predict. By using SRA data, we achieved highly accurate predictions (GDT_TS &gt; 70) for 66% of the non-easy targets, whereas using ColabFold-search default MSAs scored highly in only 52%. Next, we tested the effect of deep homology search and ColabFolds advanced features, such as more recycles, on prediction accuracy. While SRA homologs were most significant for improving ColabFolds CASP15 ranking from 11th to 3rd place, other strategies contributed too. We analyze these in the context of existing strategies to improve prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548509">
        <p class="paperTitle">Simulations of sequence evolution: how (un)realistic they really are and why</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trost, J.; Haag, J.; Hoehler, D.; Nesterenko, L.; Jacob, L.; Stamatakis, A.; Boussau, B.</p>
        <p class="info">Score: 30.3, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548509' target='https://doi.org/10.1101/2023.07.11.548509'> 10.1101/2023.07.11.548509</a></p>
        <p class="abstract">MotivationSimulating sequence evolution plays an important role in the development and evaluation of phylogenetic inference tools. Naturally, the simulated data needs to be as realistic as possible to be indicative of the performance of the developed tools on empirical data. Over the years, numerous phylogenetic sequence simulators, employing various models of evolution, have been published with the goal to simulate such empirical-like data. In this study, we simulated DNA and protein Multiple Sequence Alignments (MSAs) under increasingly complex models of evolution with and without insertion/deletion (indel) events using a state-of-the-art sequence simulator. We assessed their realism by quantifying how well supervised learning methods are able to predict whether a given MSA is simulated or empirical.

ResultsOur results show that we can distinguish between empirical and simulated MSAs with high accuracy using two distinct and independently developed classification approaches across all tested models of sequence evolution. Our findings suggest that the current state-of-the-art models fail to accurately replicate the process of evolution.

Data and Code AvailabilityAll simulated and empirical MSAs, as well as all analysis results, are available at https://cme.h-its.org/exelixis/material/simulation_study.tar.gz. All scripts required to reproduce our results are available at https://github.com/tschuelia/SimulationStudy and https://github.com/JohannaTrost/seqsharp.

Contactjulia.haag@h-its.org</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548136" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548136">
        <p class="paperTitle">Efficient High-Quality Metagenome Assembly from Long Accurate Reads using Minimizer-space de Bruijn Graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benoit, G.; Raguideau, S.; James, R.; Phillippy, A. M.; Chikhi, R.; Quince, C.</p>
        <p class="info">Score: 58.7, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548136' target='https://doi.org/10.1101/2023.07.07.548136'> 10.1101/2023.07.07.548136</a></p>
        <p class="abstract">We introduce a novel metagenomics assembler for high-accuracy long reads. Our approach, implemented as metaMDBG, combines highly efficient de Bruijn graph assembly in minimizer space, with both a multi-k&#39; approach for dealing with variations in genome coverage depth and an abundance-based filtering strategy for simplifying strain complexity. The resulting algorithm is more efficient than the state-of-the-art but with better assembly results. metaMDBG was 1.5 to 12 times faster than competing assemblers and requires between one-tenth and one-thirtieth of the memory across a range of data sets. We obtained up to twice as many high-quality circularised prokaryotic metagenome assembled genomes (MAGs) on the most complex communities, and a better recovery of viruses and plasmids. metaMDBG performs particularly well for abundant organisms whilst being robust to the presence of strain diversity. The result is that for the first time it is possible to efficiently reconstruct the majority of complex communities by abundance as nearcomplete MAGs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.545008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.545008" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.545008">
        <p class="paperTitle">Identifying Protein Conformational States in the PDB and Comparison to AlphaFold2 Predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.545008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.545008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ellaway, J. I. J.; Anyango, S.; Nair, S.; Zaki, H. A.; Nadzirin, N.; Powell, H. R.; Gutmanas, A.; Varadi, M.; Velankar, S.</p>
        <p class="info">Score: 28.6, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.545008' target='https://doi.org/10.1101/2023.07.13.545008'> 10.1101/2023.07.13.545008</a></p>
        <p class="abstract">Proteins, as molecular machines, are necessarily dynamic macromolecules that carry out essential cellular functions. Recognising their stable conformations is important for understanding the molecular mechanisms of disease. While AI-based computational methods have enabled protein structure prediction, the prediction of protein dynamics remains a challenge. Here, we present a deterministic pipeline that clusters experimentally determined protein structures to comprehensively recognise conformational states across the Protein Data Bank. Our approach clusters protein chains based on a GLObal CONformation (GLOCON) difference score, which is computed from pairwise C-alpha distances. By superposing the clustered structures, differences and similarities in conformational states can be observed. Additionally, we offer users the ability to superpose predicted models from the AlphaFold Database to the clusters of PDB structures. This clustering pipeline significantly advances researchers ability to explore the conformational landscape within the PDB. All clustered and superposed models can be viewed in Mol* on the PDBe Knowledge Base website, or accessed in as raw annotations via our GraphAPI and FTP server. The clustering package is made available as an open-source Python3 package under the Apache-2.0 license.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548907">
        <p class="paperTitle">FLAG: Find, Label Annotate Genomes, a fully automated tool for genome gene structural and functional annotation of highly fragmented non-model species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Troy, W.; Damas, J.; Titus, A. J.; Cantarel, B. L.</p>
        <p class="info">Score: 27.1, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548907' target='https://doi.org/10.1101/2023.07.14.548907'> 10.1101/2023.07.14.548907</a></p>
        <p class="abstract">Recent advances in long-read sequencing technologies and the efforts of projects aimed at increasing the universe sequenced reference genomes, such as the Earth Biogenome project have led to a growth in the number of whole genome sequencing projects for non-model organisms. Still, the vast majority (6,880) of the 28,727 unique publicly available eukaryotic genomes lack gene structure annotations. Annotating genes in a genome of interest is a multistep process, and while there are many tools available for each step of the annotation, there are few end-to-end workflows that (i) run on multiple computing environments, (ii) automatically run without initial data training and (iii) perform accurately with fragmented genomes. Here we present &#34;Find, Label, Annotate Genomes&#34; (FLAG), a fully automated genome annotation workflow that does not require species-specific extrinsic evidence to create highly complete and accurate gene annotations of even highly fragmented genomes. Using FLAG, various eukaryotic genome assemblies, including 1 plant, 5 living animals, and 1 extinct animal were annotated. In all living species, FLAG annotations provided an average of 18% increase in complete BUSCO scores when compared to BRAKER2. Furthermore, FLAG has more accurately predicted the number of protein-coding genes, with an average error rate 15x lower than BRAKER2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548477" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548477">
        <p class="paperTitle">ARCTIC-3D: Automatic Retrieval and ClusTering of Interfaces in Complexes from 3D structural information</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giulini, M.; Honorato, R. V.; Rivera, J. L.; Bonvin, A. M. J. J.</p>
        <p class="info">Score: 26.7, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548477' target='https://doi.org/10.1101/2023.07.10.548477'> 10.1101/2023.07.10.548477</a></p>
        <p class="abstract">The formation of a stable complex between proteins lies at the core of a wide variety of biological processes and has been the focus of countless experiments. The huge amount of information contained in the protein structural interactome in the Protein Data Bank can now be used to characterise and classify the existing biological interfaces. We here introduce ARCTIC-3D, a fast and user-friendly data mining and clustering software to retrieve data and rationalise the interface information associated with the protein input data. We demonstrate its use by various examples ranging from showing the increased interaction complexity of eukaryotic proteins, 20% of which on average have more than 3 different interfaces compared to only 10% for prokaryotes, to associating different functions to different interfaces. In the context of modelling biomolecular assemblies, we introduce the concept of &#34;recognition entropy&#34;, related to the number of possible interfaces of the components of a protein-protein complex, which we demonstrate to correlate with the modelling difficulty. The identified interface clusters can also be used to generate various combinations of interface-specific restraints for integrative modelling. The ARCTIC-3D software is freely available at https://github.com/haddocking/arctic3d and can be accessed as a web-service at https://wenmr.science.uu.nl/arctic-3d</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548639">
        <p class="paperTitle">Tutorial: guidelines for manual cell type annotation of single-cell multi-omics datasets using interactive software</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, Y. J.; Tarashansky, A.; Liang, K.; Urisko, M.; Dorman, L.; Borja, M.; Neff, N.; Pisco, A. O.; Granados, A.</p>
        <p class="info">Score: 23.2, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548639' target='https://doi.org/10.1101/2023.07.11.548639'> 10.1101/2023.07.11.548639</a></p>
        <p class="abstract">Assigning cell identity to clusters of single cells is an essential step towards extracting biological insights from many genomics datasets. Although annotation workflows for datasets built with a single modality are well established, limitations exist in annotating cell types in datasets with multiple modalities due to the need for a framework to exploit them jointly. While, in principle, different modalities could convey complementary information about cell identity, it is unclear to what extent they can be combined to improve the accuracy and resolution of cell type annotations.

Here, we present a conceptual framework to examine and jointly interrogate distinct modalities to identify cell types. We integrated our framework into a series of vignettes, using immune cells as a well-studied example, and demonstrate cell type annotation workflows ranging from using single-cell RNA-seq datasets alone, to using multiple modalities such as single-cell Multiome (RNA and chromatin accessibility), CITE-seq (RNA and surface proteins). In some cases, one or other single modality is superior to the other for identification of specific cell types, in others combining the two modalities improves resolution and the ability to identify finer subpopulations. Finally, we use interactive software from CZ CELLxGENE community tools to visualize and integrate histological and spatial transcriptomic data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.549143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.549143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.549143">
        <p class="paperTitle">The Patent and Literature Antibody Database (PLAbDab): an evolving reference set of functionally diverse, literature-annotated antibody sequences and structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.549143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.549143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abanades, B.; Olsen, T. H.; Raybould, M. I. J.; Aguilar-Sanjuan, B.; Wong, W. K.; Georges, G.; Bujotzek, A.; Deane, C. M.</p>
        <p class="info">Score: 21.8, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.549143' target='https://doi.org/10.1101/2023.07.15.549143'> 10.1101/2023.07.15.549143</a></p>
        <p class="abstract">Antibodies are key proteins of the adaptive immune system, and there exists a large body of academic literature and patents dedicated to their study and concomitant conversion into therapeutics, diagnostics, or reagents. These documents often contain extensive functional characterisations of the sets of antibodies the describe. However, leveraging these heterogeneous reports, for example to offer insights into the properties of query antibodies of interest, is currently challenging as there is no central repository through which this wide corpus can be mined by sequence or structure. Here, we present PLAbDab (the Patent and Literature Antibody Database), a self-updating repository containing over 150,000 paired antibody sequences and 3D structural models, of which over 65,000 are unique. Each entry in the database also contains the title and authors of its literature source. Here we describe the methods used to extract, filter, pair, and model the antibodies in PLAbDab, and showcase how PLAbDab can be searched by sequence, structure, or keyword. PLAbDab uses include annotating query antibodies with potential antigen information from similar entries, analysing structural models of existing antibodies to identify modifications that could improve their properties, and compiling bespoke datasets of antibody sequences/structures known to bind to a specific antigen. PLAbDab is freely available via Github (https://github.com/oxpig/PLAbDab) and as a searchable webserver (https://opig.stats.ox.ac.uk/webapps/plabdab).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548519">
        <p class="paperTitle">AI2BMD: efficient characterization of protein dynamics with ab initio accuracy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, T.; He, X.; Li, M.; Wang, Y.; Wang, Z.; Li, S.; Shao, B.; Liu, T.-Y.</p>
        <p class="info">Score: 20.6, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548519' target='https://doi.org/10.1101/2023.07.12.548519'> 10.1101/2023.07.12.548519</a></p>
        <p class="abstract">Biomolecular dynamics simulation is a fundamental technology for life sciences research, and its usefulness depends on its accuracy and efficiency. Classical molecular dynamics simulation is fast but lacks chemical accuracy. Quantum chemistry methods like density functional theory (DFT) can reach chemical accuracy but cannot scale to support large biomolecules. We introduce an AI-based ab initio biomolecular dynamics system (AI2BMD) that can efficiently simulate large biomolecules with ab initio accuracy. AI2BMD uses a protein fragmentation scheme and machine learning force field to achieve generalizable ab initio accuracy for energy and force calculations for various proteins comprising over 10,000 atoms. Compared to DFT, it reduces computational time by several orders of magnitude. With several hundred nanoseconds of dynamics simulations, AI2BMD demonstrated its capability of efficiently exploring the conformational space of peptides and proteins, deriving accurate 3J-couplings that match NMR experiments, and showing protein folding and unfolding tendencies. Furthermore, AI2BMD enables precise free energy calculations for protein folding, and the estimated melting temperatures are well aligned with experiments. AI2BMD could potentially complement wet-lab experiments, detect the dynamic processes of bioactivities, and enable biomedical research that is currently impossible to conduct.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548588">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548588" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548588">
        <p class="paperTitle">UNI-RNA: UNIVERSAL PRE-TRAINED MODELS REVOLUTIONIZE RNA RESEARCH</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548588" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548588" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, X.; Gu, R.; Chen, Z.; Li, Y.; Ji, X.; Ke, G.; Wen, H.</p>
        <p class="info">Score: 18.1, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548588' target='https://doi.org/10.1101/2023.07.11.548588'> 10.1101/2023.07.11.548588</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWRNA molecules play a crucial role as intermediaries in diverse biological processes. Attaining a profound understanding of their function can substantially enhance our comprehension of lifes activities and facilitate drug development for numerous diseases. The advent of high-throughput sequencing technologies makes vast amounts of RNA sequence data accessible, which contains invaluable information and knowledge. However, deriving insights for further application from such an immense volume of data poses a significant challenge. Fortunately, recent advancements in pre-trained models have surfaced as a revolutionary solution for addressing such challenges owing to their exceptional ability to automatically mine and extract hidden knowledge from massive datasets. Inspired by the past successes, we developed a novel context-aware deep learning model named Uni-RNA that performs pre-training on the largest dataset of RNA sequences at the unprecedented scale to date. During this process, our model autonomously unraveled the obscured evolutionary and structural information embedded within the RNA sequences. As a result, through fine-tuning, our model achieved the state-of-the-art (SOTA) performances in a spectrum of downstream tasks, including both structural and functional predictions. Overall, Uni-RNA established a new research paradigm empowered by the large pre-trained model in the field of RNA, enabling the community to unlock the power of AI at a whole new level to significantly expedite the pace of research and foster groundbreaking discoveries.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
