<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics
Authors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics
Authors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics
Authors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics\nAuthors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.",
  "keywords": [
    
  ],
  "articleBody": " An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics\nAuthors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.; Vaz, B.; Bhargava, S.; Yusuf, F.; Samuel, S.; Alibai, S.; Baghestani, Z.; He, X.; Krastel, K.; Oladapo, O.; Mohan, A.; Shanavas, A.; Bugno, M.; Bogojeski, J.; Schmitges, F.; Kim, C.; Grant, S.; Jayaraman, R.; Masud, T.; Deshwar, A.; Gandhi, S.; Frey, B. J.\nScore: 74.7, Published: 2023-09-26 DOI: 10.1101/2023.09.20.558508\nAccurately modeling and predicting RNA biology has been a long-standing challenge, bearing significant clinical ramifications for variant interpretation and the formulation of tailored therapeutics. We describe a foundation model for RNA biology, \"BigRNA\", which was trained on thousands of genome-matched datasets to predict tissue-specific RNA expression, splicing, microRNA sites, and RNA binding protein specificity from DNA sequence. Unlike approaches that are restricted to missense variants, BigRNA can identify pathogenic non-coding variant effects across diverse mechanisms, including polyadenylation, exon skipping and intron retention. BigRNA accurately predicted the effects of steric blocking oligonucleotides (SBOs) on increasing the expression of 4 out of 4 genes, and on splicing for 18 out of 18 exons across 14 genes, including those involved in Wilson disease and spinal muscular atrophy. We anticipate that BigRNA and foundation models like it will have widespread applications in the field of personalized RNA therapeutics.\nStructural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses\nAuthors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.\nScore: 212.0, Published: 2023-10-02 DOI: 10.1101/2023.09.19.558401\nRecent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences. This feature of structures could in principle be used to reconstruct phylogenetic trees over longer evolutionary timescales than sequence-based approaches, but until now a reliable structure-based tree building method has been elusive. Here, we demonstrate that the use of structure-based phylogenies can outperform sequence-based ones not only for distantly related proteins but also, remarkably, for more closely related ones. This is achieved by inferring trees from protein structures using a local structural alphabet, an approach robust to conformational changes that confound traditional structural distance measures. As an illustration, we used structures to decipher the evolutionary diversification of a particularly challenging family: the fast-evolving RRNPPA quorum sensing receptors enabling gram-positive bacteria, plasmids and bacteriophages to communicate and coordinate key behaviors such as sporulation, virulence, antibiotic resistance, conjugation or phage lysis/lysogeny decision. The advent of high-accuracy structural phylogenetics enables myriad of applications across biology, such as uncovering deeper evolutionary relationships, elucidating unknown protein functions, or refining the design of bioengineered molecules.\nSaProt: Protein Language Modeling with Structure-aware Vocabulary\nAuthors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.\nScore: 36.0, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349\nLarge-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a 'structure-aware vocabulary\" that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures. Through extensive evaluation, our SaProt model surpasses well-established and renowned baselines across 10 significant downstream tasks, demonstrating its exceptional capacity and broad applicability. We have made the code, pre-trained model, and all relevant materials available at https://github.com/westlake-repl/SaProt.\nThe Topological Properties of the Protein Universe\nAuthors: Madsen, C. D.; Barbensi, A.; Zhang, S. Y.; Ham, L.; David, A.; Pires, D. E. V.; Stumpf, M. P. H.\nScore: 33.9, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559443\nDeep learning methods have revolutionized our ability to predict protein structures, allowing us a glimpse into the entire protein universe. As a result, our understanding of how protein structure drives function is now lagging behind our ability to determine and predict protein structure. Here, we describe how topology, the branch of mathematics concerned with qualitative properties of spatial structures, provides a lens through which we can identify fundamental organizing features across the known protein universe. We identify topological determinants that capture global features of the protein universe, such as domain architecture and binding sites. Additionally, our analysis also identified highly specific properties, so-called topological generators, that can be used to provide deeper insights into protein structure-function and evolutionary relationships. We used our approach to determine structural, functional and disease consequences of mutations, explain differences in properties of proteins in mesophiles and thermophiles, and the likely structural and functional consequences of polymorphisms in a protein. Overall, we present a practical methodology for mapping the topology of the known protein universe at scale.\nSeqVerify: A quality-assurance pipeline for whole-genome sequencing data\nAuthors: Pierson Smela, M. D.; Pepe, V.; Church, G. M.\nScore: 26.0, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559766\n1Over the last decade, advances in genome editing and pluripotent stem cell (PSC) culture have let researchers generate edited PSC lines to study a wide variety of biological questions. However, abnormalities in cell lines can arise during PSC culture or due to undesired editing outcomes. These can include aneuploidy, on-target and off-target editing errors, and microbial contamination. Any of these abnormalities can invalidate experiments, so detecting them is crucial. The ongoing decline of next-generation sequencing prices has made whole genome sequencing (WGS) an effective quality control option, since WGS can detect any abnormality involving changes to DNA sequences or presence of unwanted sequences. However, until now, this approach has suffered from a lack of easily usable data analysis software. Here, we present SeqVerify, a computational pipeline designed to take raw WGS data and a list of intended edits, and verify that the edits are present and that there are no abnormalities. We anticipate that SeqVerify will be a useful tool for researchers generating edited PSCs, and more broadly, for cell line quality control in general.\nGenerative Diffusion Models for Antibody Design, Docking, and Optimization\nAuthors: Peng, Z.; Han, C.; Wang, X.; Li, D.; Yuan, F.\nScore: 21.3, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559190\nIn recent years, optimizing antibody binding affinity for biomedical applications has become increasingly important. However, traditional wet-experiment-based approaches are time-consuming and inefficient. To address this issue, we propose a diffusion model-based antibody optimization pipeline to improve binding affinity. Our approach involves two key models: AbDesign for designing antibody sequences and structures, and AbDock, a paratope-epitope docking model, used for screening designed CDRs. On an independent test set, our AbDesign demonstrates the exceptional performance of an RMSD of 2.56[A] in structure design and an amino acid recovery of 36.47% in sequence design. In a paratope-epitope docking test set, our AbDock achieves a state-of-the-art performance of DockQ 0.44, irms 2.71[A], fnat 0.40, and Lrms 6.29[A]. The effectiveness of the optimization pipeline is further experimentally validated by optimizing a flaviviruse antibody 1G5.3, resulting in a broad-spectrum antibody that demonstrates improved binding to 6 out of the nine tested flaviviruses. This research offers a general-purpose methodology to enhance antibody functionality without training on data from specific antigens.\nProteinFlow: a Python Library to Pre-Process Protein Structure Data for Deep Learning Applications\nAuthors: Kozlova, E.; Valentin, A.; Khadhraoui, A.; Nakhaee-Zadeh Gutierrez, D.\nScore: 23.2, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559346\nOver the past few years, deep learning tools for protein design have made significant advances in the field of bioengineering, opening up new opportunities for drug discovery, disease prevention or industrial biotechnology. However, despite the growing interest and excitement surrounding these tools, progress in the field is hindered by a lack of standardized datasets for benchmarking. Most models are trained on data from the Protein Data Bank (PDB), the largest repository of experimentally determined biological macromolecular structures. But filtering and processing this data involves many hyperparameter choices that are often not harmonized across the research community. Moreover, the task of splitting protein data into training and validation subsets with minimal data leakage is not trivial and often overlooked. Here we present ProteinFlow, a computational pipeline to pre-process protein sequence and structural data for deep learning applications. The pipeline is fully configurable and allows the extraction of all levels of protein organization (primary to quaternary), allowing end-users to cater the dataset for a multitude of downstream tasks, such as protein sequence design, protein folding modeling or protein-protein interaction prediction. In addition, we curate a feature-rich benchmarking dataset based on the latest annual release of the PDB and a selection of preprocessing parameters that are widely used across the research community. We showcase its utility by benchmarking a state-of-the-art (SOTA) deep learning model for protein sequence design. The open source code is packaged as a python library and can be accessed on https://github.com/adaptyvbio/ProteinFlow.\nGeneCompass: Deciphering Universal Gene Regulatory Mechanisms with Knowledge-Informed Cross-Species Foundation Model\nAuthors: Yang, X.; Liu, G.; Feng, G.; Bu, D.; Wang, P.; Jiang, J.; Chen, S.; Yang, Q.; Zhang, Y.; Man, Z.; Liang, Z.; Wang, Z.; Li, Y.; Li, Z.; Liu, Y.; Tian, Y.; Li, A.; Dong, J.; Hu, Z.; Fang, C.; Miao, H.; Cui, L.; Deng, Z.; Jiang, H.; Cui, W.; Zhang, J.; Yang, Z.; Li, H.; He, X.; Zhong, L.; Zhou, J.; Wang, Z.; Long, Q.; Xu, P.; Wang, H.; Meng, Z.; Wang, X.; Wang, Y.; Wang, Y.; Zhang, S.; Guo, J.; Zhao, Y.; Zhou, Y.; Li, F.; Liu, J.; Chen, Y.; Yang, G.; Li, X.\nScore: 18.1, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559542\nDeciphering the universal gene regulatory mechanisms in diverse organisms holds great potential to advance our knowledge of fundamental life process and facilitate research on clinical applications. However, the traditional research paradigm primarily focuses on individual model organisms, resulting in limited collection and integration of complex features on various cell types across species. Recent breakthroughs in single-cell sequencing and advancements in deep learning techniques present an unprecedented opportunity to tackle this challenge. In this study, we developed GeneCompass, the first knowledge-informed, cross-species foundation model pre-trained on an extensive dataset of over 120 million single-cell transcriptomes from human and mouse. During pre-training, GeneCompass effectively integrates four types of biological prior knowledge to enhance the understanding of gene regulatory mechanisms in a self-supervised manner. Fine-tuning towards multiple downstream tasks, GeneCompass outperforms competing state-of-the-art models in multiple tasks on single species and unlocks new realms of cross-species biological investigation. Overall, GeneCompass marks a milestone in advancing knowledge of universal gene regulatory mechanisms and accelerating the discovery of key cell fate regulators and candidate targets for drug development.\nCellPolaris: Decoding Cell Fate through Generalization Transfer Learning of Gene Regulatory Networks\nAuthors: Feng, G.; Qin, X.; Zhang, J.; Huang, W.; Zhang, Y.; Cui, W.; Li, S.; Chen, Y.; Liu, W.; Tian, Y.; Liu, Y.; Dong, J.; Xu, P.; Man, Z.; Liu, G.; Liang, Z.; Jiang, X.; Yang, X.; Wang, P.; Yang, G.; Wang, H.; Wang, X.; Tong, M.-H.; Zhou, Y.; Zhang, S.; Chen, Y.; Wang, Y.; Li, X.\nScore: 13.4, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559244\nCell fate changes are determined by gene regulatory network (GRN), a sophisticated system regulating gene expression in precise spatial and temporal patterns. However, existing methods for reconstructing GRNs suffer from inherent limitations, leading to compromised accuracy and application generalizability. In this study, we introduce CellPolaris, a computational system that leverages transfer learning algorithms to generate high-quality, cell-type-specific GRNs. Diverging from conventional GRN inference models, which heavily rely on integrating epigenomic data with transcriptomic information or adopt causal strategies through gene co-expression networks, CellPolaris employs high-confidence GRN sources for model training, relying exclusively on transcriptomic data to generate previously unknown cell-type-specific GRNs. Applications of CellPolaris demonstrate remarkable efficacy in predicting master regulatory factors and simulating in-silico perturbations of transcription factors during cell fate transition, attaining state-of-the-art performance in accurately predicting candidate key factors and outcomes in cell reprogramming and spermatogenesis with validated datasets. It is worth noting that, with a transfer learning framework, CellPolaris can perform GRN based predictions in all cell types even across species. Together, CellPolaris represents a significant advancement in deciphering the mechanisms of cell fate regulation, thereby enhancing the precision and efficiency of cell fate manipulation at high resolution.\nGut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles\nAuthors: Chang, D.; Gupta, V. K.; Hur, B.; Cobo-Lopez, S.; Cunningham, K. Y.; Han, N.; Lee, I.; Kronzer, V. L.; Teigen, L.; Karnatovskaia, L.; Longbrake, E. E.; Davis, J. M.; Nelson, H.; Sung, J.\nScore: 9.7, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560294\nRecent advancements in human gut microbiome research have revealed its crucial role in shaping innovative predictive healthcare applications. We introduce Gut Microbiome Wellness Index 2 (GMWI2), an advanced iteration of our original GMWI prototype, designed as a robust, disease-agnostic health status indicator based on gut microbiome taxonomic profiles. Our analysis involved pooling existing 8069 stool shotgun metagenome data across a global demographic landscape to effectively capture biological signals linking gut taxonomies to health. GMWI2 achieves a cross-validation balanced accuracy of 80% in distinguishing healthy (no disease) from non-healthy (diseased) individuals and surpasses 90% accuracy for samples with higher confidence (i.e., outside the reject option). The enhanced classification accuracy of GMWI2 outperforms both the original GMWI model and traditional species-level -diversity indices, suggesting a more reliable tool for differentiating between healthy and non-healthy phenotypes using gut microbiome data. Furthermore, by reevaluating and reinterpreting previously published data, GMWI2 provides fresh insights into the established understanding of how diet, antibiotic exposure, and fecal microbiota transplantation influence gut health. Looking ahead, GMWI2 represents a timely pivotal tool for evaluating health based on an individuals unique gut microbial composition, paving the way for the early screening of adverse gut health shifts. GMWI2 is offered as an open-source command-line tool, ensuring it is both accessible to and adaptable for researchers interested in the translational applications of human gut microbiome science.\n",
  "wordCount" : "2369",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558508">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558508" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558508">
        <p class="paperTitle">An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558508" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558508" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.; Vaz, B.; Bhargava, S.; Yusuf, F.; Samuel, S.; Alibai, S.; Baghestani, Z.; He, X.; Krastel, K.; Oladapo, O.; Mohan, A.; Shanavas, A.; Bugno, M.; Bogojeski, J.; Schmitges, F.; Kim, C.; Grant, S.; Jayaraman, R.; Masud, T.; Deshwar, A.; Gandhi, S.; Frey, B. J.</p>
        <p class="info">Score: 74.7, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558508' target='https://doi.org/10.1101/2023.09.20.558508'> 10.1101/2023.09.20.558508</a></p>
        <p class="abstract">Accurately modeling and predicting RNA biology has been a long-standing challenge, bearing significant clinical ramifications for variant interpretation and the formulation of tailored therapeutics. We describe a foundation model for RNA biology, &#34;BigRNA&#34;, which was trained on thousands of genome-matched datasets to predict tissue-specific RNA expression, splicing, microRNA sites, and RNA binding protein specificity from DNA sequence. Unlike approaches that are restricted to missense variants, BigRNA can identify pathogenic non-coding variant effects across diverse mechanisms, including polyadenylation, exon skipping and intron retention. BigRNA accurately predicted the effects of steric blocking oligonucleotides (SBOs) on increasing the expression of 4 out of 4 genes, and on splicing for 18 out of 18 exons across 14 genes, including those involved in Wilson disease and spinal muscular atrophy. We anticipate that BigRNA and foundation models like it will have widespread applications in the field of personalized RNA therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558401">
        <p class="paperTitle">Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.</p>
        <p class="info">Score: 212.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558401' target='https://doi.org/10.1101/2023.09.19.558401'> 10.1101/2023.09.19.558401</a></p>
        <p class="abstract">Recent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences. This feature of structures could in principle be used to reconstruct phylogenetic trees over longer evolutionary timescales than sequence-based approaches, but until now a reliable structure-based tree building method has been elusive. Here, we demonstrate that the use of structure-based phylogenies can outperform sequence-based ones not only for distantly related proteins but also, remarkably, for more closely related ones. This is achieved by inferring trees from protein structures using a local structural alphabet, an approach robust to conformational changes that confound traditional structural distance measures. As an illustration, we used structures to decipher the evolutionary diversification of a particularly challenging family: the fast-evolving RRNPPA quorum sensing receptors enabling gram-positive bacteria, plasmids and bacteriophages to communicate and coordinate key behaviors such as sporulation, virulence, antibiotic resistance, conjugation or phage lysis/lysogeny decision. The advent of high-accuracy structural phylogenetics enables myriad of applications across biology, such as uncovering deeper evolutionary relationships, elucidating unknown protein functions, or refining the design of bioengineered molecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560349">
        <p class="paperTitle">SaProt: Protein Language Modeling with Structure-aware Vocabulary</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.</p>
        <p class="info">Score: 36.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560349' target='https://doi.org/10.1101/2023.10.01.560349'> 10.1101/2023.10.01.560349</a></p>
        <p class="abstract">Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a &#39;structure-aware vocabulary&#34; that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures. Through extensive evaluation, our SaProt model surpasses well-established and renowned baselines across 10 significant downstream tasks, demonstrating its exceptional capacity and broad applicability. We have made the code, pre-trained model, and all relevant materials available at https://github.com/westlake-repl/SaProt.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559443">
        <p class="paperTitle">The Topological Properties of the Protein Universe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madsen, C. D.; Barbensi, A.; Zhang, S. Y.; Ham, L.; David, A.; Pires, D. E. V.; Stumpf, M. P. H.</p>
        <p class="info">Score: 33.9, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559443' target='https://doi.org/10.1101/2023.09.25.559443'> 10.1101/2023.09.25.559443</a></p>
        <p class="abstract">Deep learning methods have revolutionized our ability to predict protein structures, allowing us a glimpse into the entire protein universe. As a result, our understanding of how protein structure drives function is now lagging behind our ability to determine and predict protein structure. Here, we describe how topology, the branch of mathematics concerned with qualitative properties of spatial structures, provides a lens through which we can identify fundamental organizing features across the known protein universe. We identify topological determinants that capture global features of the protein universe, such as domain architecture and binding sites. Additionally, our analysis also identified highly specific properties, so-called topological generators, that can be used to provide deeper insights into protein structure-function and evolutionary relationships. We used our approach to determine structural, functional and disease consequences of mutations, explain differences in properties of proteins in mesophiles and thermophiles, and the likely structural and functional consequences of polymorphisms in a protein. Overall, we present a practical methodology for mapping the topology of the known protein universe at scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559766">
        <p class="paperTitle">SeqVerify: A quality-assurance pipeline for whole-genome sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pierson Smela, M. D.; Pepe, V.; Church, G. M.</p>
        <p class="info">Score: 26.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559766' target='https://doi.org/10.1101/2023.09.27.559766'> 10.1101/2023.09.27.559766</a></p>
        <p class="abstract">1Over the last decade, advances in genome editing and pluripotent stem cell (PSC) culture have let researchers generate edited PSC lines to study a wide variety of biological questions. However, abnormalities in cell lines can arise during PSC culture or due to undesired editing outcomes. These can include aneuploidy, on-target and off-target editing errors, and microbial contamination. Any of these abnormalities can invalidate experiments, so detecting them is crucial. The ongoing decline of next-generation sequencing prices has made whole genome sequencing (WGS) an effective quality control option, since WGS can detect any abnormality involving changes to DNA sequences or presence of unwanted sequences. However, until now, this approach has suffered from a lack of easily usable data analysis software. Here, we present SeqVerify, a computational pipeline designed to take raw WGS data and a list of intended edits, and verify that the edits are present and that there are no abnormalities. We anticipate that SeqVerify will be a useful tool for researchers generating edited PSCs, and more broadly, for cell line quality control in general.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559190">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559190" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559190">
        <p class="paperTitle">Generative Diffusion Models for Antibody Design, Docking, and Optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559190" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559190" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, Z.; Han, C.; Wang, X.; Li, D.; Yuan, F.</p>
        <p class="info">Score: 21.3, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559190' target='https://doi.org/10.1101/2023.09.25.559190'> 10.1101/2023.09.25.559190</a></p>
        <p class="abstract">In recent years, optimizing antibody binding affinity for biomedical applications has become increasingly important. However, traditional wet-experiment-based approaches are time-consuming and inefficient. To address this issue, we propose a diffusion model-based antibody optimization pipeline to improve binding affinity. Our approach involves two key models: AbDesign for designing antibody sequences and structures, and AbDock, a paratope-epitope docking model, used for screening designed CDRs. On an independent test set, our AbDesign demonstrates the exceptional performance of an RMSD of 2.56[A] in structure design and an amino acid recovery of 36.47% in sequence design. In a paratope-epitope docking test set, our AbDock achieves a state-of-the-art performance of DockQ 0.44, irms 2.71[A], fnat 0.40, and Lrms 6.29[A]. The effectiveness of the optimization pipeline is further experimentally validated by optimizing a flaviviruse antibody 1G5.3, resulting in a broad-spectrum antibody that demonstrates improved binding to 6 out of the nine tested flaviviruses. This research offers a general-purpose methodology to enhance antibody functionality without training on data from specific antigens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559346">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559346" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559346">
        <p class="paperTitle">ProteinFlow: a Python Library to Pre-Process Protein Structure Data for Deep Learning Applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559346" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559346" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozlova, E.; Valentin, A.; Khadhraoui, A.; Nakhaee-Zadeh Gutierrez, D.</p>
        <p class="info">Score: 23.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559346' target='https://doi.org/10.1101/2023.09.25.559346'> 10.1101/2023.09.25.559346</a></p>
        <p class="abstract">Over the past few years, deep learning tools for protein design have made significant advances in the field of bioengineering, opening up new opportunities for drug discovery, disease prevention or industrial biotechnology. However, despite the growing interest and excitement surrounding these tools, progress in the field is hindered by a lack of standardized datasets for benchmarking. Most models are trained on data from the Protein Data Bank (PDB), the largest repository of experimentally determined biological macromolecular structures. But filtering and processing this data involves many hyperparameter choices that are often not harmonized across the research community. Moreover, the task of splitting protein data into training and validation subsets with minimal data leakage is not trivial and often overlooked. Here we present ProteinFlow, a computational pipeline to pre-process protein sequence and structural data for deep learning applications. The pipeline is fully configurable and allows the extraction of all levels of protein organization (primary to quaternary), allowing end-users to cater the dataset for a multitude of downstream tasks, such as protein sequence design, protein folding modeling or protein-protein interaction prediction. In addition, we curate a feature-rich benchmarking dataset based on the latest annual release of the PDB and a selection of preprocessing parameters that are widely used across the research community. We showcase its utility by benchmarking a state-of-the-art (SOTA) deep learning model for protein sequence design. The open source code is packaged as a python library and can be accessed on https://github.com/adaptyvbio/ProteinFlow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559542">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559542" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559542">
        <p class="paperTitle">GeneCompass: Deciphering Universal Gene Regulatory Mechanisms with Knowledge-Informed Cross-Species Foundation Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559542" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559542" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Liu, G.; Feng, G.; Bu, D.; Wang, P.; Jiang, J.; Chen, S.; Yang, Q.; Zhang, Y.; Man, Z.; Liang, Z.; Wang, Z.; Li, Y.; Li, Z.; Liu, Y.; Tian, Y.; Li, A.; Dong, J.; Hu, Z.; Fang, C.; Miao, H.; Cui, L.; Deng, Z.; Jiang, H.; Cui, W.; Zhang, J.; Yang, Z.; Li, H.; He, X.; Zhong, L.; Zhou, J.; Wang, Z.; Long, Q.; Xu, P.; Wang, H.; Meng, Z.; Wang, X.; Wang, Y.; Wang, Y.; Zhang, S.; Guo, J.; Zhao, Y.; Zhou, Y.; Li, F.; Liu, J.; Chen, Y.; Yang, G.; Li, X.</p>
        <p class="info">Score: 18.1, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559542' target='https://doi.org/10.1101/2023.09.26.559542'> 10.1101/2023.09.26.559542</a></p>
        <p class="abstract">Deciphering the universal gene regulatory mechanisms in diverse organisms holds great potential to advance our knowledge of fundamental life process and facilitate research on clinical applications. However, the traditional research paradigm primarily focuses on individual model organisms, resulting in limited collection and integration of complex features on various cell types across species. Recent breakthroughs in single-cell sequencing and advancements in deep learning techniques present an unprecedented opportunity to tackle this challenge. In this study, we developed GeneCompass, the first knowledge-informed, cross-species foundation model pre-trained on an extensive dataset of over 120 million single-cell transcriptomes from human and mouse. During pre-training, GeneCompass effectively integrates four types of biological prior knowledge to enhance the understanding of gene regulatory mechanisms in a self-supervised manner. Fine-tuning towards multiple downstream tasks, GeneCompass outperforms competing state-of-the-art models in multiple tasks on single species and unlocks new realms of cross-species biological investigation. Overall, GeneCompass marks a milestone in advancing knowledge of universal gene regulatory mechanisms and accelerating the discovery of key cell fate regulators and candidate targets for drug development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559244">
        <p class="paperTitle">CellPolaris: Decoding Cell Fate through Generalization Transfer Learning of Gene Regulatory Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, G.; Qin, X.; Zhang, J.; Huang, W.; Zhang, Y.; Cui, W.; Li, S.; Chen, Y.; Liu, W.; Tian, Y.; Liu, Y.; Dong, J.; Xu, P.; Man, Z.; Liu, G.; Liang, Z.; Jiang, X.; Yang, X.; Wang, P.; Yang, G.; Wang, H.; Wang, X.; Tong, M.-H.; Zhou, Y.; Zhang, S.; Chen, Y.; Wang, Y.; Li, X.</p>
        <p class="info">Score: 13.4, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559244' target='https://doi.org/10.1101/2023.09.25.559244'> 10.1101/2023.09.25.559244</a></p>
        <p class="abstract">Cell fate changes are determined by gene regulatory network (GRN), a sophisticated system regulating gene expression in precise spatial and temporal patterns. However, existing methods for reconstructing GRNs suffer from inherent limitations, leading to compromised accuracy and application generalizability. In this study, we introduce CellPolaris, a computational system that leverages transfer learning algorithms to generate high-quality, cell-type-specific GRNs. Diverging from conventional GRN inference models, which heavily rely on integrating epigenomic data with transcriptomic information or adopt causal strategies through gene co-expression networks, CellPolaris employs high-confidence GRN sources for model training, relying exclusively on transcriptomic data to generate previously unknown cell-type-specific GRNs. Applications of CellPolaris demonstrate remarkable efficacy in predicting master regulatory factors and simulating in-silico perturbations of transcription factors during cell fate transition, attaining state-of-the-art performance in accurately predicting candidate key factors and outcomes in cell reprogramming and spermatogenesis with validated datasets. It is worth noting that, with a transfer learning framework, CellPolaris can perform GRN based predictions in all cell types even across species. Together, CellPolaris represents a significant advancement in deciphering the mechanisms of cell fate regulation, thereby enhancing the precision and efficiency of cell fate manipulation at high resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560294">
        <p class="paperTitle">Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, D.; Gupta, V. K.; Hur, B.; Cobo-Lopez, S.; Cunningham, K. Y.; Han, N.; Lee, I.; Kronzer, V. L.; Teigen, L.; Karnatovskaia, L.; Longbrake, E. E.; Davis, J. M.; Nelson, H.; Sung, J.</p>
        <p class="info">Score: 9.7, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560294' target='https://doi.org/10.1101/2023.09.30.560294'> 10.1101/2023.09.30.560294</a></p>
        <p class="abstract">Recent advancements in human gut microbiome research have revealed its crucial role in shaping innovative predictive healthcare applications. We introduce Gut Microbiome Wellness Index 2 (GMWI2), an advanced iteration of our original GMWI prototype, designed as a robust, disease-agnostic health status indicator based on gut microbiome taxonomic profiles. Our analysis involved pooling existing 8069 stool shotgun metagenome data across a global demographic landscape to effectively capture biological signals linking gut taxonomies to health. GMWI2 achieves a cross-validation balanced accuracy of 80% in distinguishing healthy (no disease) from non-healthy (diseased) individuals and surpasses 90% accuracy for samples with higher confidence (i.e., outside the reject option). The enhanced classification accuracy of GMWI2 outperforms both the original GMWI model and traditional species-level -diversity indices, suggesting a more reliable tool for differentiating between healthy and non-healthy phenotypes using gut microbiome data. Furthermore, by reevaluating and reinterpreting previously published data, GMWI2 provides fresh insights into the established understanding of how diet, antibiotic exposure, and fecal microbiota transplantation influence gut health. Looking ahead, GMWI2 represents a timely pivotal tool for evaluating health based on an individuals unique gut microbial composition, paving the way for the early screening of adverse gut health shifts. GMWI2 is offered as an open-source command-line tool, ensuring it is both accessible to and adaptable for researchers interested in the translational applications of human gut microbiome science.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
