<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sequence clustering confounds AlphaFold2
Authors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.
Score: 23.3, Published: 2024-01-09 DOI: 10.1101/2024.01.05.574434
Though typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Sequence clustering confounds AlphaFold2
Authors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.
Score: 23.3, Published: 2024-01-09 DOI: 10.1101/2024.01.05.574434
Though typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Sequence clustering confounds AlphaFold2
Authors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.
Score: 23.3, Published: 2024-01-09 DOI: 10.1101/2024.01.05.574434
Though typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Sequence clustering confounds AlphaFold2\nAuthors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.\nScore: 23.3, Published: 2024-01-09 DOI: 10.1101/2024.01.05.574434\nThough typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al.",
  "keywords": [
    
  ],
  "articleBody": " Sequence clustering confounds AlphaFold2\nAuthors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.\nScore: 23.3, Published: 2024-01-09 DOI: 10.1101/2024.01.05.574434\nThough typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al. steered AF2 to predict alternative structures of a few metamorphic proteins using a method they call AF-cluster5. However, their Paper lacks some essential controls needed to assess AF-clusters reliability. We find that these controls show AF-cluster to be a poor predictor of metamorphic proteins. First, closer examination of the Papers results reveals that random sequence sampling outperforms sequence clustering, challenging the claim that AF-cluster works by \"deconvolving conflicting sets of couplings.\" Further, we observe that AF-cluster mistakes some single-folding KaiB homologs for fold switchers, a critical flaw bound to mislead users. Finally, proper error analysis reveals that AF-cluster predicts many correct structures with low confidence and some experimentally unobserved conformations with confidences similar to experimentally observed ones. For these reasons, we suggest using ColabFold6-based random sequence sampling7-augmented by other predictive approaches-as a more accurate and less computationally intense alternative to AF-cluster.\nThe Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes\nAuthors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.\nScore: 17.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574205\nThe Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze. The issue stems from VCFs requirement to densely represent both reference-genotypes and allele-indexed arrays. These requirements lead to unnecessary data duplication and, ultimately, very large files. To address these challenges, we introduce the Scalable Variant Call Representation (SVCR). This representation reduces file sizes by ensuring they scale linearly with samples. SVCR achieves this by adopting reference blocks from the Genomic Variant Call Format (GVCF) and employing local allele indices. SVCR is also lossless and mergeable, allowing for N+1 and N+K incremental joint-calling. We present two implementations of SVCR: SVCR-VCF, which encodes SVCR in VCF format, and VDS, which uses Hails native format. Our experiments confirm the linear scalability of SVCR-VCF and VDS, in contrast to the super-linear growth seen with standard VCF files. We also discuss the VDS Combiner, a scalable, open-source tool for producing a VDS from GVCFs and unique features of VDS which enable rapid data analysis. SVCR, and VDS in particular, ensure the scientific community can generate, analyze, and disseminate genetics datasets with millions of samples.\nPCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation\nAuthors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.\nScore: 50.9, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573793\nBackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples. We initially used conventional PCA as a tool for critical quality control of batch and trend effects in multi-omic profiling data produced by The Cancer Genome Atlas (TCGA) project of the NCI. We found, however, that conventional PCA visualizations were often hard to interpret when inter-batch differences were moderate in comparison with intra-batch differences; it was also difficult to quantify batch effects objectively. We, therefore, sought enhancements to make the method more informative in those and analogous settings. ResultsWe have developed algorithms and a toolbox of enhancements to conventional PCA that improve the detection, diagnosis, and quantitation of differences between or among groups, e.g., groups of molecularly profiled biological samples. The enhancements include (i) computed group centroids; (ii) sample-dispersion rays; (iii) differential coloring of centroids, rays, and sample data points; (iii) trend trajectories; and (iv) a novel separation index (DSC) for quantitation of differences among groups. ConclusionsPCA-Plus has been our most useful single tool for analyzing, visualizing, and quantitating batch effects, trend effects, and class differences in molecular profiling data of many types: mRNA expression, microRNA expression, DNA methylation, and DNA copy number. An early version of PCA-Plus has been used as the central graphical visualization in our MBatch package for near-real-time surveillance of data for analysis working groups in more than 70 TCGA, PanCancer Atlas, PanCancer Analysis of Whole Genomes, and Genome Data Analysis Network projects of the NCI. The algorithms and software are generic, hence applicable more generally to other types of multivariate data as well. PCA-Plus is freely available in a down-loadable R package at our MBatch website.\nBeyond benchmarking: towards predictive models of dataset-specific single-cell RNA-seq pipeline performance\nAuthors: Fang, C.; Selega, A.; Campbell, K. R.\nScore: 25.4, Published: 2024-01-02 DOI: 10.1101/2024.01.02.572650\nThe advent of single-cell RNA-sequencing (scRNA-seq) has driven significant computational methods development for all steps in the scRNA-seq data analysis pipeline, including filtering, normalization, and clustering. The large number of methods and their resulting parameter combinations has created a combinatorial set of possible pipelines to analyze scRNA-seq data, which leads to the obvious question: which is best? Several benchmarking studies have sought to compare methods to answer this, but frequently find variable performance depending on dataset and pipeline characteristics. Alternatively, the large number of publicly available scRNA-seq datasets along with advances in supervised machine learning raise a tantalizing possibility: could the optimal pipeline be predicted for a given dataset? Here we begin to answer this question by applying 288 scRNA-seq analysis pipelines to 86 datasets and quantifying pipeline success via a range of measures evaluating cluster purity and biological plausibility. We build supervised machine learning models to predict pipeline success given a range of dataset and pipeline characteristics. We find both that prediction performance is significantly better than random and that in many cases pipelines predicted to perform well provide clustering outputs similar to expert-annotated cell type labels. Finally, we identify characteristics of scRNA-seq datasets that correlate with strong prediction performance that could guide when such prediction models may be useful.\nscDirect: key transcription factor identification for directing cell state transitions based on single-cell multi-omics data\nAuthors: Li, C.; Chen, S.; Chen, Y.; Bian, H.; Hao, M.; Wei, L.; Zhang, X.\nScore: 11.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574757\nCell state transitions are complicated processes that occur in various life activities. Understanding and artificially manipulating them have been longstanding challenges. Substantial experiments reveal that the transitions could be directed by several key transcription factors (TFs), and some computational methods have been developed to alleviate the burden of biological experiments on identifying key TFs. However, most existing methods employ data with resolution on the cell population, instead of individual cells, which will influence the identification quality due to cell heterogeneity. Besides, they require collecting abundant samples for candidate cell states and are not available for unknown cell states. As for the alternative single-cell analysis methods, they generally concentrate on differentially expressed genes between cell states but can hardly identify key TFs responsible for directing cells state transition. Here we present scDirect, a computational framework to identify key TFs based on single-cell multi-omics data. scDirect models the TF identification task as a linear inverse problem, and solve it with gene regulatory networks enhanced by a graph attention network. Through a benchmarking on a single-cell human embryonic stem cell atlas, we systematically demonstrate the robustness and superiority of scDirect against other alternative single-cell analysis methods on TF identification. With application on various single-cell datasets, scDirect exhibits high capability in identifying key TFs in cell differentiation, somatic cell conversion, and cell reprogramming. Furthermore, scDirect can quantitatively identify experimentally validated reprogramming TF combinations, which indicates the potential of scDirect to guide and assist the experimental design in cellular engineering. We envision that scDirect could utilize rapidly increasing single-cell datasets to identify key TFs for directing cell state transitions, and become an effective tool to facilitate regenerative medicine and therapeutic discovery.\nGenopyc: a python library for investigating the genomic basis of complex diseases\nAuthors: Gualdi, F.; Oliva, B.; Pinero, J.\nScore: 9.3, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575316\nMotivation: Understanding the genetic basis of complex diseases is a paramount challenge in modern genomics. However, current tools often lack the versatility to efficiently analyze the intri-cate relationships between genetic variations and disease outcomes. To address this, we intro-duce Genopyc, a novel Python library designed for comprehensive investigation of the genetics underlying complex diseases. Genopyc offers an extensive suite of functions for heterogeneous data mining and visualization, enabling researchers to delve into and integrate biological infor-mation from large-scale genomic datasets with ease. Results: In this study, we present the Genopyc library through application to real-world genome wide association studies variants. Using Genopyc to investigate variants associated to interverte-bral disc degeneration (IDD) enabled a deeper understanding of the potential dysregulated path-ways involved in the disease, which can be explored and visualized by exploiting the functionali-ties featured in the package. Genopyc emerges as a powerful asset for researchers, fostering ad-vancements in the understanding of complex diseases and thus paving the way for more targeted therapeutic interventions. Availability: Genopyc is available at pip (https://pypi.org/project/genopyc/) and the source code of Genopyc is available at https://github.com/freh-g/genopyc\nADMET-AI: A machine learning ADMET platform for evaluation of large-scale chemical libraries\nAuthors: Swanson, K.; Walther, P.; Leitz, J.; Mukherjee, S.; Wu, J. C.; Shivnaraine, R. V.; Zou, J.\nScore: 20.7, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573531\nSummaryThe emergence of large chemical repositories and combinatorial chemical spaces, coupled with high-throughput docking and generative AI, have greatly expanded the chemical diversity of small molecules for drug discovery. Selecting compounds for experimental validation requires filtering these molecules based on favourable druglike properties, such as Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET). We developed ADMET-AI, a machine learning platform that provides fast and accurate ADMET predictions both as a website and as a Python package. ADMET-AI has the highest average rank on the TDC ADMET Benchmark Group leaderboard, and it is currently the fastest web-based ADMET predictor, with a 45% reduction in time compared to the next fastest ADMET web server. ADMET-AI can also be run locally with predictions for one million molecules taking just 3.1 hours. Availability and ImplementationThe ADMET-AI platform is freely available both as a web server at admet.ai.greenstonebio.com and as an open-source Python package for local batch prediction at github.com/swansonk14/admet_ai (also archived on Zenodo at doi.org/10.5281/zenodo.10372930). All data and models are archived on Zenodo at doi.org/10.5281/zenodo.10372418.\nA catalogue of small proteins from the global microbiome\nAuthors: Duan, Y.; Santos Junior, C. D.; Schmidt, T. S. B.; Fullam, A.; de Almeida, B. L. S.; Zhu, C.; Michael, K.; Zhao, X.-M.; Bork, P.; Coelho, L. P.\nScore: 25.2, Published: 2023-12-28 DOI: 10.1101/2023.12.27.573469\nSmall open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we constructed a global microbial smORFs catalogue (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We found that archaea harbor more small proteins proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.\nInferring strain-level mutational drivers of phage-bacteria interaction phenotypes\nAuthors: Lucia-Sanz, A.; Peng, S.; Leung, J.; Gupta, A.; Meyer, J. R.; Weitz, J. S.\nScore: 7.3, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574707\nThe enormous diversity of bacteriophages and their bacterial hosts presents a significant challenge to predict which phages infect a focal set of bacteria. Infection is largely determined by complementary -and largely uncharacterized-genetics of adsorption, injection, and cell take-over. Here we present a machine learning (ML) approach to predict phage-bacteria interactions trained on genome sequences of and phenotypic interactions amongst 51 Escherichia coli strains and 45 phage {lambda} strains that coevolved in laboratory conditions for 37 days. Leveraging multiple inference strategies and without a priori knowledge of driver mutations, this framework predicts both who infects whom and the quantitative levels of infections across a suite of 2,295 potential interactions. The most effective ML approach inferred interaction phenotypes from independent contributions from phage and bacteria mutations, predicting phage host range with 86% mean classification accuracy while reducing the relative error in the estimated strength of the infection phenotype by 40%. Further, transparent feature selection in the predictive model revealed 18 of 176 phage {lambda} and 6 of 18 E. coli mutations that have a significant influence on the outcome of phage-bacteria interactions, corroborating sites previously known to affect phage {lambda} infections, as well as identifying mutations in genes of unknown function not previously shown to influence bacterial resistance. While the genetic variation studied was limited to a focal, coevolved phage-bacteria system, the methods success at recapitulating strain-level infection outcomes provides a path forward towards developing strategies for inferring interactions in non-model systems, including those of therapeutic significance.\nGenerative artificial intelligence performs rudimentary structural biology modelling\nAuthors: Ille, A. M.; Markosian, C.; Burley, S. K.; Mathews, M. B.; Pasqualini, R.; Arap, W.\nScore: 7.2, Published: 2024-01-13 DOI: 10.1101/2024.01.10.575113\nGenerative artificial intelligence (AI) is being increasingly used by researchers in numerous fields of study, including the biological sciences. To date, the most commonly used tool grounded in this technology has been Chat Generative Pre-trained Transformer (ChatGPT). While ChatGPT is typically applied for natural language text generation, other application modes include coding and mathematical problem-solving. We have recently reported the ability of ChatGPT to interpret the central dogma of molecular biology and the genetic code. Here we explored how ChatGPT performs rudimentary structural biology modelling in order to gain evaluative insight. We prompted ChatGPT to model 3D structures for the 20 standard amino acids as well as an -helical polypeptide chain, with the latter involving incorporation of the Wolfram plugin for advanced mathematical computation. For amino acid modelling, distances and angles between atoms of the generated structures in most cases approximated to around experimentally-determined values. For -helix modelling, the generated structures were comparable to that of an experimentally-determined -helical structure. However, both amino acid and -helix modelling were sporadically error-prone and increased molecular complexity was not well tolerated. Despite current limitations, our findings show the capability of generative AI to perform basic structural biology modelling with atomic-scale accuracy. These results provide a precedent for the potential use of generative AI in structural biology as this technology continues to advance.\n",
  "wordCount" : "2435",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574434" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574434">
        <p class="paperTitle">Sequence clustering confounds AlphaFold2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.</p>
        <p class="info">Score: 23.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574434' target='https://doi.org/10.1101/2024.01.05.574434'> 10.1101/2024.01.05.574434</a></p>
        <p class="abstract">Though typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al. steered AF2 to predict alternative structures of a few metamorphic proteins using a method they call AF-cluster5. However, their Paper lacks some essential controls needed to assess AF-clusters reliability. We find that these controls show AF-cluster to be a poor predictor of metamorphic proteins. First, closer examination of the Papers results reveals that random sequence sampling outperforms sequence clustering, challenging the claim that AF-cluster works by &#34;deconvolving conflicting sets of couplings.&#34; Further, we observe that AF-cluster mistakes some single-folding KaiB homologs for fold switchers, a critical flaw bound to mislead users. Finally, proper error analysis reveals that AF-cluster predicts many correct structures with low confidence and some experimentally unobserved conformations with confidences similar to experimentally observed ones. For these reasons, we suggest using ColabFold6-based random sequence sampling7-augmented by other predictive approaches-as a more accurate and less computationally intense alternative to AF-cluster.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574205" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574205">
        <p class="paperTitle">The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.</p>
        <p class="info">Score: 17.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574205' target='https://doi.org/10.1101/2024.01.09.574205'> 10.1101/2024.01.09.574205</a></p>
        <p class="abstract">The Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze. The issue stems from VCFs requirement to densely represent both reference-genotypes and allele-indexed arrays. These requirements lead to unnecessary data duplication and, ultimately, very large files.

To address these challenges, we introduce the Scalable Variant Call Representation (SVCR). This representation reduces file sizes by ensuring they scale linearly with samples. SVCR achieves this by adopting reference blocks from the Genomic Variant Call Format (GVCF) and employing local allele indices. SVCR is also lossless and mergeable, allowing for N&#43;1 and N&#43;K incremental joint-calling.

We present two implementations of SVCR: SVCR-VCF, which encodes SVCR in VCF format, and VDS, which uses Hails native format. Our experiments confirm the linear scalability of SVCR-VCF and VDS, in contrast to the super-linear growth seen with standard VCF files. We also discuss the VDS Combiner, a scalable, open-source tool for producing a VDS from GVCFs and unique features of VDS which enable rapid data analysis. SVCR, and VDS in particular, ensure the scientific community can generate, analyze, and disseminate genetics datasets with millions of samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573793" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573793">
        <p class="paperTitle">PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.</p>
        <p class="info">Score: 50.9, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573793' target='https://doi.org/10.1101/2024.01.02.573793'> 10.1101/2024.01.02.573793</a></p>
        <p class="abstract">BackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples. We initially used conventional PCA as a tool for critical quality control of batch and trend effects in multi-omic profiling data produced by The Cancer Genome Atlas (TCGA) project of the NCI. We found, however, that conventional PCA visualizations were often hard to interpret when inter-batch differences were moderate in comparison with intra-batch differences; it was also difficult to quantify batch effects objectively. We, therefore, sought enhancements to make the method more informative in those and analogous settings.

ResultsWe have developed algorithms and a toolbox of enhancements to conventional PCA that improve the detection, diagnosis, and quantitation of differences between or among groups, e.g., groups of molecularly profiled biological samples. The enhancements include (i) computed group centroids; (ii) sample-dispersion rays; (iii) differential coloring of centroids, rays, and sample data points; (iii) trend trajectories; and (iv) a novel separation index (DSC) for quantitation of differences among groups.

ConclusionsPCA-Plus has been our most useful single tool for analyzing, visualizing, and quantitating batch effects, trend effects, and class differences in molecular profiling data of many types: mRNA expression, microRNA expression, DNA methylation, and DNA copy number. An early version of PCA-Plus has been used as the central graphical visualization in our MBatch package for near-real-time surveillance of data for analysis working groups in more than 70 TCGA, PanCancer Atlas, PanCancer Analysis of Whole Genomes, and Genome Data Analysis Network projects of the NCI. The algorithms and software are generic, hence applicable more generally to other types of multivariate data as well. PCA-Plus is freely available in a down-loadable R package at our MBatch website.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.572650">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.572650" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.572650">
        <p class="paperTitle">Beyond benchmarking: towards predictive models of dataset-specific single-cell RNA-seq pipeline performance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.572650" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.572650" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fang, C.; Selega, A.; Campbell, K. R.</p>
        <p class="info">Score: 25.4, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.572650' target='https://doi.org/10.1101/2024.01.02.572650'> 10.1101/2024.01.02.572650</a></p>
        <p class="abstract">The advent of single-cell RNA-sequencing (scRNA-seq) has driven significant computational methods development for all steps in the scRNA-seq data analysis pipeline, including filtering, normalization, and clustering. The large number of methods and their resulting parameter combinations has created a combinatorial set of possible pipelines to analyze scRNA-seq data, which leads to the obvious question: which is best? Several benchmarking studies have sought to compare methods to answer this, but frequently find variable performance depending on dataset and pipeline characteristics. Alternatively, the large number of publicly available scRNA-seq datasets along with advances in supervised machine learning raise a tantalizing possibility: could the optimal pipeline be predicted for a given dataset? Here we begin to answer this question by applying 288 scRNA-seq analysis pipelines to 86 datasets and quantifying pipeline success via a range of measures evaluating cluster purity and biological plausibility. We build supervised machine learning models to predict pipeline success given a range of dataset and pipeline characteristics. We find both that prediction performance is significantly better than random and that in many cases pipelines predicted to perform well provide clustering outputs similar to expert-annotated cell type labels. Finally, we identify characteristics of scRNA-seq datasets that correlate with strong prediction performance that could guide when such prediction models may be useful.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574757" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574757">
        <p class="paperTitle">scDirect: key transcription factor identification for directing cell state transitions based on single-cell multi-omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, C.; Chen, S.; Chen, Y.; Bian, H.; Hao, M.; Wei, L.; Zhang, X.</p>
        <p class="info">Score: 11.2, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574757' target='https://doi.org/10.1101/2024.01.08.574757'> 10.1101/2024.01.08.574757</a></p>
        <p class="abstract">Cell state transitions are complicated processes that occur in various life activities. Understanding and artificially manipulating them have been longstanding challenges. Substantial experiments reveal that the transitions could be directed by several key transcription factors (TFs), and some computational methods have been developed to alleviate the burden of biological experiments on identifying key TFs. However, most existing methods employ data with resolution on the cell population, instead of individual cells, which will influence the identification quality due to cell heterogeneity. Besides, they require collecting abundant samples for candidate cell states and are not available for unknown cell states. As for the alternative single-cell analysis methods, they generally concentrate on differentially expressed genes between cell states but can hardly identify key TFs responsible for directing cells state transition. Here we present scDirect, a computational framework to identify key TFs based on single-cell multi-omics data. scDirect models the TF identification task as a linear inverse problem, and solve it with gene regulatory networks enhanced by a graph attention network. Through a benchmarking on a single-cell human embryonic stem cell atlas, we systematically demonstrate the robustness and superiority of scDirect against other alternative single-cell analysis methods on TF identification. With application on various single-cell datasets, scDirect exhibits high capability in identifying key TFs in cell differentiation, somatic cell conversion, and cell reprogramming. Furthermore, scDirect can quantitatively identify experimentally validated reprogramming TF combinations, which indicates the potential of scDirect to guide and assist the experimental design in cellular engineering. We envision that scDirect could utilize rapidly increasing single-cell datasets to identify key TFs for directing cell state transitions, and become an effective tool to facilitate regenerative medicine and therapeutic discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575316" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575316">
        <p class="paperTitle">Genopyc: a python library for investigating the genomic basis of complex diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gualdi, F.; Oliva, B.; Pinero, J.</p>
        <p class="info">Score: 9.3, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575316' target='https://doi.org/10.1101/2024.01.11.575316'> 10.1101/2024.01.11.575316</a></p>
        <p class="abstract">Motivation: Understanding the genetic basis of complex diseases is a paramount challenge in modern genomics. However, current tools often lack the versatility to efficiently analyze the intri-cate relationships between genetic variations and disease outcomes. To address this, we intro-duce Genopyc, a novel Python library designed for comprehensive investigation of the genetics underlying complex diseases. Genopyc offers an extensive suite of functions for heterogeneous data mining and visualization, enabling researchers to delve into and integrate biological infor-mation from large-scale genomic datasets with ease. Results: In this study, we present the Genopyc library through application to real-world genome wide association studies variants. Using Genopyc to investigate variants associated to interverte-bral disc degeneration (IDD) enabled a deeper understanding of the potential dysregulated path-ways involved in the disease, which can be explored and visualized by exploiting the functionali-ties featured in the package. Genopyc emerges as a powerful asset for researchers, fostering ad-vancements in the understanding of complex diseases and thus paving the way for more targeted therapeutic interventions. Availability: Genopyc is available at pip (https://pypi.org/project/genopyc/) and the source code of Genopyc is available at https://github.com/freh-g/genopyc</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573531">
        <p class="paperTitle">ADMET-AI: A machine learning ADMET platform for evaluation of large-scale chemical libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swanson, K.; Walther, P.; Leitz, J.; Mukherjee, S.; Wu, J. C.; Shivnaraine, R. V.; Zou, J.</p>
        <p class="info">Score: 20.7, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573531' target='https://doi.org/10.1101/2023.12.28.573531'> 10.1101/2023.12.28.573531</a></p>
        <p class="abstract">SummaryThe emergence of large chemical repositories and combinatorial chemical spaces, coupled with high-throughput docking and generative AI, have greatly expanded the chemical diversity of small molecules for drug discovery. Selecting compounds for experimental validation requires filtering these molecules based on favourable druglike properties, such as Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET). We developed ADMET-AI, a machine learning platform that provides fast and accurate ADMET predictions both as a website and as a Python package. ADMET-AI has the highest average rank on the TDC ADMET Benchmark Group leaderboard, and it is currently the fastest web-based ADMET predictor, with a 45% reduction in time compared to the next fastest ADMET web server. ADMET-AI can also be run locally with predictions for one million molecules taking just 3.1 hours.

Availability and ImplementationThe ADMET-AI platform is freely available both as a web server at admet.ai.greenstonebio.com and as an open-source Python package for local batch prediction at github.com/swansonk14/admet_ai (also archived on Zenodo at doi.org/10.5281/zenodo.10372930). All data and models are archived on Zenodo at doi.org/10.5281/zenodo.10372418.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573469">
        <p class="paperTitle">A catalogue of small proteins from the global microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duan, Y.; Santos Junior, C. D.; Schmidt, T. S. B.; Fullam, A.; de Almeida, B. L. S.; Zhu, C.; Michael, K.; Zhao, X.-M.; Bork, P.; Coelho, L. P.</p>
        <p class="info">Score: 25.2, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573469' target='https://doi.org/10.1101/2023.12.27.573469'> 10.1101/2023.12.27.573469</a></p>
        <p class="abstract">Small open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we constructed a global microbial smORFs catalogue (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We found that archaea harbor more small proteins proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574707" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574707">
        <p class="paperTitle">Inferring strain-level mutational drivers of phage-bacteria interaction phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lucia-Sanz, A.; Peng, S.; Leung, J.; Gupta, A.; Meyer, J. R.; Weitz, J. S.</p>
        <p class="info">Score: 7.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574707' target='https://doi.org/10.1101/2024.01.08.574707'> 10.1101/2024.01.08.574707</a></p>
        <p class="abstract">The enormous diversity of bacteriophages and their bacterial hosts presents a significant challenge to predict which phages infect a focal set of bacteria. Infection is largely determined by complementary -and largely uncharacterized-genetics of adsorption, injection, and cell take-over. Here we present a machine learning (ML) approach to predict phage-bacteria interactions trained on genome sequences of and phenotypic interactions amongst 51 Escherichia coli strains and 45 phage {lambda} strains that coevolved in laboratory conditions for 37 days. Leveraging multiple inference strategies and without a priori knowledge of driver mutations, this framework predicts both who infects whom and the quantitative levels of infections across a suite of 2,295 potential interactions. The most effective ML approach inferred interaction phenotypes from independent contributions from phage and bacteria mutations, predicting phage host range with 86% mean classification accuracy while reducing the relative error in the estimated strength of the infection phenotype by 40%. Further, transparent feature selection in the predictive model revealed 18 of 176 phage {lambda} and 6 of 18 E. coli mutations that have a significant influence on the outcome of phage-bacteria interactions, corroborating sites previously known to affect phage {lambda} infections, as well as identifying mutations in genes of unknown function not previously shown to influence bacterial resistance. While the genetic variation studied was limited to a focal, coevolved phage-bacteria system, the methods success at recapitulating strain-level infection outcomes provides a path forward towards developing strategies for inferring interactions in non-model systems, including those of therapeutic significance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575113" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575113">
        <p class="paperTitle">Generative artificial intelligence performs rudimentary structural biology modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ille, A. M.; Markosian, C.; Burley, S. K.; Mathews, M. B.; Pasqualini, R.; Arap, W.</p>
        <p class="info">Score: 7.2, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575113' target='https://doi.org/10.1101/2024.01.10.575113'> 10.1101/2024.01.10.575113</a></p>
        <p class="abstract">Generative artificial intelligence (AI) is being increasingly used by researchers in numerous fields of study, including the biological sciences. To date, the most commonly used tool grounded in this technology has been Chat Generative Pre-trained Transformer (ChatGPT). While ChatGPT is typically applied for natural language text generation, other application modes include coding and mathematical problem-solving. We have recently reported the ability of ChatGPT to interpret the central dogma of molecular biology and the genetic code. Here we explored how ChatGPT performs rudimentary structural biology modelling in order to gain evaluative insight. We prompted ChatGPT to model 3D structures for the 20 standard amino acids as well as an -helical polypeptide chain, with the latter involving incorporation of the Wolfram plugin for advanced mathematical computation. For amino acid modelling, distances and angles between atoms of the generated structures in most cases approximated to around experimentally-determined values. For -helix modelling, the generated structures were comparable to that of an experimentally-determined -helical structure. However, both amino acid and -helix modelling were sporadically error-prone and increased molecular complexity was not well tolerated. Despite current limitations, our findings show the capability of generative AI to perform basic structural biology modelling with atomic-scale accuracy. These results provide a precedent for the potential use of generative AI in structural biology as this technology continues to advance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
