<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages
Authors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Bravo, H. C.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.
Score: 150.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549537
Single-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages
Authors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Bravo, H. C.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.
Score: 150.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549537
Single-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages
Authors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Bravo, H. C.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.
Score: 150.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549537
Single-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages\nAuthors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Bravo, H. C.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.\nScore: 150.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549537\nSingle-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date.",
  "keywords": [
    
  ],
  "articleBody": " Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages\nAuthors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Bravo, H. C.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.\nScore: 150.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549537\nSingle-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date. A singular view of this vast and growing expression landscape could help reveal novel associations between cell states and diseases, discover cell states in unexpected tissue contexts, and relate in vivo cells to in vitro models. However, these require a common, scalable representation of cell profiles from across the body, a general measure of their similarity, and an efficient way to query these data. Here, we present SCimilarity, a metric learning framework to learn and search a unified and interpretable representation that annotates cell types and instantaneously queries for a cell state across tens of millions of profiles. We demonstrate SCimilarity on a 22.7 million cell corpus assembled across 399 published scRNA-seq studies, showing accurate integration, annotation and querying. We experimentally validated SCimilarity by querying across tissues for a macrophage subset originally identified in interstitial lung disease, and showing that cells with similar profiles are found in other fibrotic diseases, tissues, and a 3D hydrogel system, which we then repurposed to yield this cell state in vitro. SCimilarity serves as a foundational model for single cell gene expression data and enables researchers to query for similar cellular states across the entire human body, providing a powerful tool for generating novel biological insights from the growing Human Cell Atlas.\nUnified fate mapping in multiview single-cell data\nAuthors: Weiler, P.; Lange, M.; Klein, M.; Pe'er, D.; Theis, F.\nScore: 94.8, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549685\nSingle-cell RNA sequencing allows us to model cellular state dynamics and fate decisions using expression similarity or RNA velocity to reconstruct state-change trajectories. However, trajectory inference does not incorporate valuable time point information or utilize additional modalities, while methods that address these different data views cannot be combined and do not scale. Here, we present CellRank 2, a versatile and scalable framework to study cellular fate using multiview single-cell data of up to millions of cells in a unified fashion. CellRank 2 consistently recovers terminal states and fate probabilities across data modalities in human hematopoiesis and mouse endodermal development. Our framework also allows combining transitions within and across experimental time points, a feature we use to recover genes promoting medullary thymic epithelial cell formation during pharyngeal endoderm development. Moreover, we enable estimating cell-specific transcription and degradation rates from metabolic labeling data, which we apply to an intestinal organoid system to delineate differentiation trajectories and pinpoint regulatory strategies.\nkmindex and ORA: indexing and real-time user-friendly queries in terabytes-sized complex genomic datasets\nAuthors: Lemane, T.; Lezzoche, N.; Lecubin, J.; Pelletier, E.; Lescot, M.; Chikhi, R.; Peterlongo, P.\nScore: 54.0, Published: 2023-07-17 DOI: 10.1101/2023.05.31.543043\nPublic sequencing databases contain vast amounts of biological information, yet they are largely underutilized as one cannot efficiently search them for any sequence of interest. We present kmindex, an innovative approach that can index thousands of highly complex metagenomes and perform sequence searches in a fraction of a second. The construction of the index is an order of magnitude faster than previous methods, while search times are two orders of magnitude faster. With negligible false positive rates below 0.01%, kmindex outperforms the precision of existing approaches by four orders of magnitude. We demonstrate the scalability of kmindex by successfully indexing 1,393 complex marine seawater metagenome samples from the Tara Oceans project. Additionally, we introduce the publicly accessible web server \"Ocean Read Atlas\" (ORA) at https://ocean-read-atlas.mio.osupytheas.fr/, which enables real-time queries on the entire Tara Oceans dataset. The open-source kmindex software is available at https://github.com/tlemane/kmindex.\nScalable and efficient DNA sequencing analysis on different compute infrastructures aiding variant discovery\nAuthors: Hanssen, F.; Garcia, M. U.; Folkersen, L.; Pedersen, A. S.; Lescai, F.; Jodoin, S.; Miller, E.; Wacker, O.; Smith, N.; nf-core community, ; Gabernet, G.; Nahnsen, S.\nScore: 25.6, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549462\nDNA variation analysis has become indispensable in many aspects of modern biomedicine, most prominently in the comparison of normal and tumor samples. Thousands of samples are collected in local sequencing efforts and public databases requiring highly scalable, portable, and automated workflows for streamlined processing. Here, we present nf-core/sarek 3.x, a well-established, comprehensive variant calling and annotation pipeline for germline and somatic samples. It is suitable for any genome with a known reference. We present a full rewrite of the original pipeline showing a significant reduction of storage requirements by using the CRAM format and runtime by increasing intra-sample parallelization. Both are leading to a 70% cost reduction in commercial clouds enabling users to do large-scale and cross-platform data analysis while keeping costs and CO2 emissions low. The code is available at https://nf-co.re/sarek.\nPredicting cell morphological responses to perturbations using generative modeling\nAuthors: Palma, A.; Theis, F. J.; Lotfollahi, M.\nScore: 22.2, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549216\nAdvancements in high-throughput screening have enabled the exploration of rich phenotypic readouts like high-content microscopy, expediting drug target identification and mode of action studies. However, scaling these experiments to the vast space of drug or genetic manipulations poses challenges, as only a small subset of compounds show activity in screenings. Despite being widely used in various applications, machine learning methods have not shown a reliable ability to extrapolate predictions to scenarios involving unseen phenomena, specifically transforming an unseen control cell image into a desired perturbation. We present a generative model, the IMage Perturbation Autoencoder (IMPA), which predicts cellular morphological effects of chemical and genetic perturbations using untreated cells as input. IMPA learns perturbation-specific styles from generalized embeddings and generates counterfactual treatment response predictions in control cells. We demonstrate IMPA can predict morphological changes caused by small molecule perturbations on breast cancer cells. Additionally, we test IMPA on the unseen drug effect prediction task, showing improved performance over state-of-the-art generative models when compounds are structurally related to the training set. Finally, generalizability and capability to predict more subtle effects are showcased through its application to large microscopy datasets with hundreds of genetic perturbations on U2OS cells. We envision IMPA to become a valuable tool in computational microscopy for aiding phenotypic drug discovery, facilitating navigation of the perturbation space, and rational experimental design.\nThe Patent and Literature Antibody Database (PLAbDab): an evolving reference set of functionally diverse, literature-annotated antibody sequences and structures\nAuthors: Abanades, B.; Olsen, T. H.; Raybould, M. I. J.; Aguilar-Sanjuan, B.; Wong, W. K.; Georges, G.; Bujotzek, A.; Deane, C. M.\nScore: 21.8, Published: 2023-07-16 DOI: 10.1101/2023.07.15.549143\nAntibodies are key proteins of the adaptive immune system, and there exists a large body of academic literature and patents dedicated to their study and concomitant conversion into therapeutics, diagnostics, or reagents. These documents often contain extensive functional characterisations of the sets of antibodies the describe. However, leveraging these heterogeneous reports, for example to offer insights into the properties of query antibodies of interest, is currently challenging as there is no central repository through which this wide corpus can be mined by sequence or structure. Here, we present PLAbDab (the Patent and Literature Antibody Database), a self-updating repository containing over 150,000 paired antibody sequences and 3D structural models, of which over 65,000 are unique. Each entry in the database also contains the title and authors of its literature source. Here we describe the methods used to extract, filter, pair, and model the antibodies in PLAbDab, and showcase how PLAbDab can be searched by sequence, structure, or keyword. PLAbDab uses include annotating query antibodies with potential antigen information from similar entries, analysing structural models of existing antibodies to identify modifications that could improve their properties, and compiling bespoke datasets of antibody sequences/structures known to bind to a specific antigen. PLAbDab is freely available via Github (https://github.com/oxpig/PLAbDab) and as a searchable webserver (https://opig.stats.ox.ac.uk/webapps/plabdab/).\nTopOMetry systematically learns and evaluates the latent dimensions of single-cell atlases\nAuthors: Sidarta-Oliveira, D.; Velloso, L. A.\nScore: 37.0, Published: 2023-07-11 DOI: 10.1101/2022.03.14.484134\nA core task in single-cell data analysis is recovering the latent dimensions encoding the genetic and epigenetic landscapes inhabited by cell types and lineages. However, consensus is lacking for optimal modeling and visualization approaches. Here, we propose these landscapes are ideally modeled as Riemannian manifolds and present TopOMetry, a computational toolkit based on Laplacian-type operators to learn these manifolds. TopOMetry systematically learns and evaluates dozens of possible representations, eliminating the need to choose a single dimensional reduction method a priori. The learned visualizations preserve more original information than current PCA-based standards across single-cell and non-biological datasets. TopOMetry allows users to estimate intrinsic dimensionalities and visualize distortions with the Riemannian metric, among other challenging tasks. Illustrating its hypothesis generation power, TopOMetry suggests the existence of dozens of novel T cell subpopulations consistently found across public datasets that correspond to specific clonotypes. TopOMetry is available at https://github.com/davisidarta/topometry.\nFLAG: Find, Label Annotate Genomes, a fully automated tool for genome gene structural and functional annotation of highly fragmented non-model species\nAuthors: Troy, W.; Damas, J.; Titus, A. J.; Cantarel, B. L.\nScore: 29.5, Published: 2023-07-17 DOI: 10.1101/2023.07.14.548907\nRecent advances in long-read sequencing technologies and the efforts of projects aimed at increasing the universe sequenced reference genomes, such as the Earth Biogenome project have led to a growth in the number of whole genome sequencing projects for non-model organisms. Still, the vast majority (6,880) of the 28,727 unique publicly available eukaryotic genomes lack gene structure annotations. Annotating genes in a genome of interest is a multistep process, and while there are many tools available for each step of the annotation, there are few end-to-end workflows that (i) run on multiple computing environments, (ii) automatically run without initial data training and (iii) perform accurately with fragmented genomes. Here we present \"Find, Label, Annotate Genomes\" (FLAG), a fully automated genome annotation workflow that does not require species-specific extrinsic evidence to create highly complete and accurate gene annotations of even highly fragmented genomes. Using FLAG, various eukaryotic genome assemblies, including 1 plant, 5 living animals, and 1 extinct animal were annotated. In all living species, FLAG annotations provided an average of 18% increase in complete BUSCO scores when compared to BRAKER2. Furthermore, FLAG has more accurately predicted the number of protein-coding genes, with an average error rate 15x lower than BRAKER2.\nBenchmarking software tools for trimming adapters and merging next-generation sequencing data for ancient DNA\nAuthors: Lien, A.; Pestana Legori, L.; Kraft, L.; Wad Sackett, P.; Renaud, G.\nScore: 20.6, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549303\nAncient DNA is highly degraded, resulting in very short sequences. Reads generated with modern high-throughput sequencing machines are generally longer than ancient DNA molecules, therefore the reads often contain some portion of the sequencing adaptors. It is crucial to remove those adaptors, as they can interfere with downstream analysis. Furthermore, overlapping portions when DNA has been read forward and backward (paired-end) can be merged to correct sequencing errors and improve read quality. Several tools have been developed for adapter trimming and read merging, however, no one has attempted to evaluate their accuracy and evaluate their potential impact on downstream analyses. Through the simulation of sequencing data, seven commonly used tools were analyzed in their ability to reconstruct ancient DNA sequences through read merging. The analyzed tools exhibit notable differences in their abilities to correct sequence errors and identify the correct read overlap, but the most substantial difference is observed in their ability to calculate quality scores for merged bases. Selecting the most appropriate tool for a given project depends on several factors, although some tools such as fastp have some shortcomings, whereas others like leeHom outperform the other tools in most aspects. While the choice of tool did not result in a measurable difference when analyzing population genetics using principal component analysis, it is important to note that downstream analyses that rely on quality scores can be significantly impacted by the choice of tool.\nRandom, de novo and conserved proteins: How structure and disorder predictors perform differently\nAuthors: Middendorf, L.; Eicholt, L. A.\nScore: 14.7, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549582\nUnderstanding the emergence and structural characteristics of de novo and random proteins is crucial for unraveling protein evolution and designing novel enzymes. However, experimental determination of their structures remains challenging. Recent advancements in protein structure prediction, particularly with AlphaFold2 (AF2), have expanded our knowledge of protein structures, but their applicability to de novo and random proteins is unclear. In this study, we investigate the structural predictions and confidence scores of AF2 and protein language model (pLM)-based predictor ESMFold for de novo, random, and conserved proteins. We find that the structural predictions for de novo and random proteins differ significantly from conserved proteins. Interestingly, a positive correlation between disorder and confidence scores (pLDDT) is observed for de novo and random proteins, in contrast to the negative correlation observed for conserved proteins. Furthermore, the performance of structure predictors for de novo and random proteins is hampered by the lack of sequence identity. We also observe varying predicted disorder among different sequence length quartiles for random proteins, suggesting an influence of sequence length on disorder predictions. In conclusion, while structure predictors provide initial insights into the structural composition of de novo and random proteins, their accuracy and applicability to such proteins remain limited. Experimental determination of their structures is necessary for a comprehensive understanding. The positive correlation between disorder and pLDDT could imply a potential for conditional folding and transient binding interactions of de novo and random proteins.\n",
  "wordCount" : "2253",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549537">
        <p class="paperTitle">Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Bravo, H. C.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.</p>
        <p class="info">Score: 150.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549537' target='https://doi.org/10.1101/2023.07.18.549537'> 10.1101/2023.07.18.549537</a></p>
        <p class="abstract">Single-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date. A singular view of this vast and growing expression landscape could help reveal novel associations between cell states and diseases, discover cell states in unexpected tissue contexts, and relate in vivo cells to in vitro models. However, these require a common, scalable representation of cell profiles from across the body, a general measure of their similarity, and an efficient way to query these data. Here, we present SCimilarity, a metric learning framework to learn and search a unified and interpretable representation that annotates cell types and instantaneously queries for a cell state across tens of millions of profiles. We demonstrate SCimilarity on a 22.7 million cell corpus assembled across 399 published scRNA-seq studies, showing accurate integration, annotation and querying. We experimentally validated SCimilarity by querying across tissues for a macrophage subset originally identified in interstitial lung disease, and showing that cells with similar profiles are found in other fibrotic diseases, tissues, and a 3D hydrogel system, which we then repurposed to yield this cell state in vitro. SCimilarity serves as a foundational model for single cell gene expression data and enables researchers to query for similar cellular states across the entire human body, providing a powerful tool for generating novel biological insights from the growing Human Cell Atlas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549685">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549685" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549685">
        <p class="paperTitle">Unified fate mapping in multiview single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549685" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549685" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weiler, P.; Lange, M.; Klein, M.; Pe&#39;er, D.; Theis, F.</p>
        <p class="info">Score: 94.8, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549685' target='https://doi.org/10.1101/2023.07.19.549685'> 10.1101/2023.07.19.549685</a></p>
        <p class="abstract">Single-cell RNA sequencing allows us to model cellular state dynamics and fate decisions using expression similarity or RNA velocity to reconstruct state-change trajectories. However, trajectory inference does not incorporate valuable time point information or utilize additional modalities, while methods that address these different data views cannot be combined and do not scale. Here, we present CellRank 2, a versatile and scalable framework to study cellular fate using multiview single-cell data of up to millions of cells in a unified fashion. CellRank 2 consistently recovers terminal states and fate probabilities across data modalities in human hematopoiesis and mouse endodermal development. Our framework also allows combining transitions within and across experimental time points, a feature we use to recover genes promoting medullary thymic epithelial cell formation during pharyngeal endoderm development. Moreover, we enable estimating cell-specific transcription and degradation rates from metabolic labeling data, which we apply to an intestinal organoid system to delineate differentiation trajectories and pinpoint regulatory strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.31.543043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.31.543043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.31.543043">
        <p class="paperTitle">kmindex and ORA: indexing and real-time user-friendly queries in terabytes-sized complex genomic datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.31.543043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.31.543043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lemane, T.; Lezzoche, N.; Lecubin, J.; Pelletier, E.; Lescot, M.; Chikhi, R.; Peterlongo, P.</p>
        <p class="info">Score: 54.0, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.31.543043' target='https://doi.org/10.1101/2023.05.31.543043'> 10.1101/2023.05.31.543043</a></p>
        <p class="abstract">Public sequencing databases contain vast amounts of biological information, yet they are largely underutilized as one cannot efficiently search them for any sequence of interest. We present kmindex, an innovative approach that can index thousands of highly complex metagenomes and perform sequence searches in a fraction of a second. The construction of the index is an order of magnitude faster than previous methods, while search times are two orders of magnitude faster. With negligible false positive rates below 0.01%, kmindex outperforms the precision of existing approaches by four orders of magnitude. We demonstrate the scalability of kmindex by successfully indexing 1,393 complex marine seawater metagenome samples from the Tara Oceans project. Additionally, we introduce the publicly accessible web server &#34;Ocean Read Atlas&#34; (ORA) at https://ocean-read-atlas.mio.osupytheas.fr/, which enables real-time queries on the entire Tara Oceans dataset. The open-source kmindex software is available at https://github.com/tlemane/kmindex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549462">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549462" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549462">
        <p class="paperTitle">Scalable and efficient DNA sequencing analysis on different compute infrastructures aiding variant discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549462" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549462" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hanssen, F.; Garcia, M. U.; Folkersen, L.; Pedersen, A. S.; Lescai, F.; Jodoin, S.; Miller, E.; Wacker, O.; Smith, N.; nf-core community,  ; Gabernet, G.; Nahnsen, S.</p>
        <p class="info">Score: 25.6, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549462' target='https://doi.org/10.1101/2023.07.19.549462'> 10.1101/2023.07.19.549462</a></p>
        <p class="abstract">DNA variation analysis has become indispensable in many aspects of modern biomedicine, most prominently in the comparison of normal and tumor samples. Thousands of samples are collected in local sequencing efforts and public databases requiring highly scalable, portable, and automated workflows for streamlined processing. Here, we present nf-core/sarek 3.x, a well-established, comprehensive variant calling and annotation pipeline for germline and somatic samples. It is suitable for any genome with a known reference. We present a full rewrite of the original pipeline showing a significant reduction of storage requirements by using the CRAM format and runtime by increasing intra-sample parallelization. Both are leading to a 70% cost reduction in commercial clouds enabling users to do large-scale and cross-platform data analysis while keeping costs and CO2 emissions low. The code is available at https://nf-co.re/sarek.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549216" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549216">
        <p class="paperTitle">Predicting cell morphological responses to perturbations using generative modeling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Palma, A.; Theis, F. J.; Lotfollahi, M.</p>
        <p class="info">Score: 22.2, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549216' target='https://doi.org/10.1101/2023.07.17.549216'> 10.1101/2023.07.17.549216</a></p>
        <p class="abstract">Advancements in high-throughput screening have enabled the exploration of rich phenotypic readouts like high-content microscopy, expediting drug target identification and mode of action studies. However, scaling these experiments to the vast space of drug or genetic manipulations poses challenges, as only a small subset of compounds show activity in screenings. Despite being widely used in various applications, machine learning methods have not shown a reliable ability to extrapolate predictions to scenarios involving unseen phenomena, specifically transforming an unseen control cell image into a desired perturbation. We present a generative model, the IMage Perturbation Autoencoder (IMPA), which predicts cellular morphological effects of chemical and genetic perturbations using untreated cells as input. IMPA learns perturbation-specific styles from generalized embeddings and generates counterfactual treatment response predictions in control cells. We demonstrate IMPA can predict morphological changes caused by small molecule perturbations on breast cancer cells. Additionally, we test IMPA on the unseen drug effect prediction task, showing improved performance over state-of-the-art generative models when compounds are structurally related to the training set. Finally, generalizability and capability to predict more subtle effects are showcased through its application to large microscopy datasets with hundreds of genetic perturbations on U2OS cells. We envision IMPA to become a valuable tool in computational microscopy for aiding phenotypic drug discovery, facilitating navigation of the perturbation space, and rational experimental design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.549143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.549143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.549143">
        <p class="paperTitle">The Patent and Literature Antibody Database (PLAbDab): an evolving reference set of functionally diverse, literature-annotated antibody sequences and structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.549143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.549143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abanades, B.; Olsen, T. H.; Raybould, M. I. J.; Aguilar-Sanjuan, B.; Wong, W. K.; Georges, G.; Bujotzek, A.; Deane, C. M.</p>
        <p class="info">Score: 21.8, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.549143' target='https://doi.org/10.1101/2023.07.15.549143'> 10.1101/2023.07.15.549143</a></p>
        <p class="abstract">Antibodies are key proteins of the adaptive immune system, and there exists a large body of academic literature and patents dedicated to their study and concomitant conversion into therapeutics, diagnostics, or reagents. These documents often contain extensive functional characterisations of the sets of antibodies the describe. However, leveraging these heterogeneous reports, for example to offer insights into the properties of query antibodies of interest, is currently challenging as there is no central repository through which this wide corpus can be mined by sequence or structure.

Here, we present PLAbDab (the Patent and Literature Antibody Database), a self-updating repository containing over 150,000 paired antibody sequences and 3D structural models, of which over 65,000 are unique. Each entry in the database also contains the title and authors of its literature source. Here we describe the methods used to extract, filter, pair, and model the antibodies in PLAbDab, and showcase how PLAbDab can be searched by sequence, structure, or keyword.

PLAbDab uses include annotating query antibodies with potential antigen information from similar entries, analysing structural models of existing antibodies to identify modifications that could improve their properties, and compiling bespoke datasets of antibody sequences/structures known to bind to a specific antigen. PLAbDab is freely available via Github (https://github.com/oxpig/PLAbDab) and as a searchable webserver (https://opig.stats.ox.ac.uk/webapps/plabdab/).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.03.14.484134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.03.14.484134" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.03.14.484134">
        <p class="paperTitle">TopOMetry systematically learns and evaluates the latent dimensions of single-cell atlases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.03.14.484134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.03.14.484134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sidarta-Oliveira, D.; Velloso, L. A.</p>
        <p class="info">Score: 37.0, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.03.14.484134' target='https://doi.org/10.1101/2022.03.14.484134'> 10.1101/2022.03.14.484134</a></p>
        <p class="abstract">A core task in single-cell data analysis is recovering the latent dimensions encoding the genetic and epigenetic landscapes inhabited by cell types and lineages. However, consensus is lacking for optimal modeling and visualization approaches. Here, we propose these landscapes are ideally modeled as Riemannian manifolds and present TopOMetry, a computational toolkit based on Laplacian-type operators to learn these manifolds. TopOMetry systematically learns and evaluates dozens of possible representations, eliminating the need to choose a single dimensional reduction method a priori. The learned visualizations preserve more original information than current PCA-based standards across single-cell and non-biological datasets. TopOMetry allows users to estimate intrinsic dimensionalities and visualize distortions with the Riemannian metric, among other challenging tasks. Illustrating its hypothesis generation power, TopOMetry suggests the existence of dozens of novel T cell subpopulations consistently found across public datasets that correspond to specific clonotypes. TopOMetry is available at https://github.com/davisidarta/topometry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548907">
        <p class="paperTitle">FLAG: Find, Label Annotate Genomes, a fully automated tool for genome gene structural and functional annotation of highly fragmented non-model species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Troy, W.; Damas, J.; Titus, A. J.; Cantarel, B. L.</p>
        <p class="info">Score: 29.5, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548907' target='https://doi.org/10.1101/2023.07.14.548907'> 10.1101/2023.07.14.548907</a></p>
        <p class="abstract">Recent advances in long-read sequencing technologies and the efforts of projects aimed at increasing the universe sequenced reference genomes, such as the Earth Biogenome project have led to a growth in the number of whole genome sequencing projects for non-model organisms. Still, the vast majority (6,880) of the 28,727 unique publicly available eukaryotic genomes lack gene structure annotations. Annotating genes in a genome of interest is a multistep process, and while there are many tools available for each step of the annotation, there are few end-to-end workflows that (i) run on multiple computing environments, (ii) automatically run without initial data training and (iii) perform accurately with fragmented genomes. Here we present &#34;Find, Label, Annotate Genomes&#34; (FLAG), a fully automated genome annotation workflow that does not require species-specific extrinsic evidence to create highly complete and accurate gene annotations of even highly fragmented genomes. Using FLAG, various eukaryotic genome assemblies, including 1 plant, 5 living animals, and 1 extinct animal were annotated. In all living species, FLAG annotations provided an average of 18% increase in complete BUSCO scores when compared to BRAKER2. Furthermore, FLAG has more accurately predicted the number of protein-coding genes, with an average error rate 15x lower than BRAKER2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549303">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549303" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549303">
        <p class="paperTitle">Benchmarking software tools for trimming adapters and merging next-generation sequencing data for ancient DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549303" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549303" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lien, A.; Pestana Legori, L.; Kraft, L.; Wad Sackett, P.; Renaud, G.</p>
        <p class="info">Score: 20.6, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549303' target='https://doi.org/10.1101/2023.07.17.549303'> 10.1101/2023.07.17.549303</a></p>
        <p class="abstract">Ancient DNA is highly degraded, resulting in very short sequences. Reads generated with modern high-throughput sequencing machines are generally longer than ancient DNA molecules, therefore the reads often contain some portion of the sequencing adaptors. It is crucial to remove those adaptors, as they can interfere with downstream analysis. Furthermore, overlapping portions when DNA has been read forward and backward (paired-end) can be merged to correct sequencing errors and improve read quality. Several tools have been developed for adapter trimming and read merging, however, no one has attempted to evaluate their accuracy and evaluate their potential impact on downstream analyses. Through the simulation of sequencing data, seven commonly used tools were analyzed in their ability to reconstruct ancient DNA sequences through read merging. The analyzed tools exhibit notable differences in their abilities to correct sequence errors and identify the correct read overlap, but the most substantial difference is observed in their ability to calculate quality scores for merged bases. Selecting the most appropriate tool for a given project depends on several factors, although some tools such as fastp have some shortcomings, whereas others like leeHom outperform the other tools in most aspects. While the choice of tool did not result in a measurable difference when analyzing population genetics using principal component analysis, it is important to note that downstream analyses that rely on quality scores can be significantly impacted by the choice of tool.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549582">
        <p class="paperTitle">Random, de novo and conserved proteins: How structure and disorder predictors perform differently</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Middendorf, L.; Eicholt, L. A.</p>
        <p class="info">Score: 14.7, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549582' target='https://doi.org/10.1101/2023.07.18.549582'> 10.1101/2023.07.18.549582</a></p>
        <p class="abstract">Understanding the emergence and structural characteristics of de novo and random proteins is crucial for unraveling protein evolution and designing novel enzymes. However, experimental determination of their structures remains challenging. Recent advancements in protein structure prediction, particularly with AlphaFold2 (AF2), have expanded our knowledge of protein structures, but their applicability to de novo and random proteins is unclear. In this study, we investigate the structural predictions and confidence scores of AF2 and protein language model (pLM)-based predictor ESMFold for de novo, random, and conserved proteins. We find that the structural predictions for de novo and random proteins differ significantly from conserved proteins. Interestingly, a positive correlation between disorder and confidence scores (pLDDT) is observed for de novo and random proteins, in contrast to the negative correlation observed for conserved proteins. Furthermore, the performance of structure predictors for de novo and random proteins is hampered by the lack of sequence identity. We also observe varying predicted disorder among different sequence length quartiles for random proteins, suggesting an influence of sequence length on disorder predictions. In conclusion, while structure predictors provide initial insights into the structural composition of de novo and random proteins, their accuracy and applicability to such proteins remain limited. Experimental determination of their structures is necessary for a comprehensive understanding. The positive correlation between disorder and pLDDT could imply a potential for conditional folding and transient binding interactions of de novo and random proteins.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
