<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction
Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.
Score: 52.4, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776
Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction
Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.
Score: 52.4, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776
Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction
Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.
Score: 52.4, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776
Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction\nAuthors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.\nScore: 52.4, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776\nWhereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train.",
  "keywords": [
    
  ],
  "articleBody": " GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction\nAuthors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.\nScore: 52.4, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776\nWhereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train. Across several benchmarks on clinical databases (ClinVar, COSMIC, and OMIM) and population genomic data (gnomAD), our model for the human genome achieves outstanding performance on deleteriousness prediction for both coding and non-coding variants.\nAleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.\nAuthors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.\nScore: 35.1, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561091\nMotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics. Ideally, phylogenetic methods should not only model substitutions along gene trees, which explain differences between homologous gene sequences, but also the processes that generate the gene trees themselves along a shared species tree. To conduct accurate inferences, one needs to account for uncertainty at both levels, that is, in gene trees estimated from inherently short sequences and in their diverse evolutionary histories along a shared species tree. ResultsWe present AleRax, a software that can infer reconciled gene trees together with a shared species tree using a simple, yet powerful, probabilistic model of gene duplication, transfer, and loss. A key feature of AleRax is its ability to account for uncertainty in the gene tree and its reconciliation by using an efficient approximation to calculate the joint phylogenetic-reconciliation likelihood and sample reconciled gene trees accordingly. Simulations and analyses of empirical data show that AleRax is one order of magnitude faster than competing gene tree inference tools while attaining the same accuracy. It is consistently more robust than species tree inference methods such as SpeciesRax and ASTRAL-Pro 2 under gene tree uncertainty. Finally, AleRax can process multiple gene families in parallel thereby allowing users to compare competing phylogenetic hypotheses and estimate model parameters, such as DTL probabilities for genome-scale datasets with hundreds of taxa Availability and ImplementationGNU GPL at https://github.com/BenoitMorel/AleRax and data are made available at https://cme.h-its.org/exelixis/material/alerax_data.tar.gz. ContactBenoit.Morel@h-its.org Supplementary informationSupplementary material is available.\nOn the relationship between protist metabarcoding and protist metagenome-assembled genomes\nAuthors: Zavadska, D.; Henry, N.; Auladell, A.; Berney, C.; Richter, D. J.\nScore: 29.6, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561583\nAO_SCPLOWBSTRACTC_SCPLOWThe two most commonly used approaches to study the composition of environmental protist communities are marker gene metabarcoding and whole genome analysis through metagenomics. Raw metabarcoding data are usually processed into Operational Taxonomic Units (OTUs) or amplicon sequence variants (ASVs) through clustering or denoising approaches, respectively. Analogous approaches have been developed to assemble metagenomic sequence reads into metagenome-assembled genomes (MAGs). Understanding the correspondence between the data produced by these two approaches can help to integrate information between the datasets and to explain how metabarcoding OTUs and MAGs are related with the underlying biological entities they are hypothesised to represent. Due to the nature of their construction, MAGs do not contain the most commonly used barcoding loci, meaning that sequence homology approaches cannot be used to match OTUs and MAGs. We made an attempt to match V9 metabarcoding OTUs from the 18S rRNA gene (V9 OTUs) and MAGs from the Tara Oceans expedition (2009-2013) based on the correspondence of their relative abundances across the same set of samples. We evaluated the performance of several methods for detecting correspondence between features in these two compositional datasets and developed a series of controls to filter artefacts of data structure and processing. After selecting the best-performing correspondence metrics, ranking the V9 OTU/MAG matches by their proportionality/correlation coefficients and applying a set of selection criteria, we identified candidate matches between V9 OTUs and MAGs. In a subset of cases, V9 OTUs and MAGs could be successfully matched with one another with a one-to-one correspondence, implying that they likely represent the same underlying biological entity. More generally, matches we observed could be classified into 4 scenarios: Scenario I - one V9 OTU matches more than one MAG; Scenario II - more than one V9 OTU matches more than one MAG; Scenario III - more than one V9 OTU matches one MAG; Scenario IV - one V9 OTU matches one MAG. These diverse scenarios for V9 OTU-MAG matches illustrate the complex nature of the OTU/MAG relationship. Notably, we found some instances in which different OTU-MAG matches from the same taxonomic group were not classified in the same scenario, with all four scenarios possible even within the same taxonomic group, illustrating that factors beyond taxonomic lineage influence the relationship between OTUs and MAGs. Overall, each scenario produces a different interpretation of V9 OTUs, MAGs and how they compare in terms of the genomic and ecological diversity that they represent.\nCharacterizing cell-type spatial relationships across length scales in spatially resolved omics data\nAuthors: dos Santos Peixoto, R.; Miller, B. F.; Brusko, M. A.; Atta, L.; Anant, M.; Atkinson, M. A.; Brusko, T. M.; Wasserfall, C. H.; Fan, J.\nScore: 29.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560733\nSpatially resolved omics technologies provide molecular profiling of cells while preserving their organization within tissues, allowing for the evaluation of cell-type spatial relationships. We developed CRAWDAD to quantify cell-type spatial relationships across length scales. We highlight the utility of such multi-scale characterization on simulated data, recapitulate expected cell-type spatial relationships in tissues such as the mouse brain and embryo, and delineate functionally relevant spatial-defined cell-type subsets in the human spleen.\nUncover spatially informed shared variations for single-cell spatial transcriptomics with STew\nAuthors: Guo, N.; Vargas, J.; Fritz, D.; Krishna, R.; Zhang, F.\nScore: 20.8, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561789\nMotivation: The recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge. Results: We introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods. STew is also robust to generate consistent results insensitive to model parameters, including sparsity constraints. We next applied STew to various ST data acquired from 10x Visium and Slide-seqV2, encompassing samples from both mouse and human brain, which revealed spatially informed cell type clusters. We further identified a pro-inflammatory fibroblast spatial niche using ST data from psoriatic skins. Hence, STew is a generalized method to identify both spatially informed clusters and disease-relevant niches in complex tissues. Availability: Source code and the R software tool STew are available from github.com/fanzhanglab/STew.\nScaling cross-tissue single-cell annotation models\nAuthors: Fischer, F.; Fischer, D. S.; Biederstedt, E.; Villani, A.-C.; Theis, F. J.\nScore: 19.3, Published: 2023-10-10 DOI: 10.1101/2023.10.07.561331\nIdentifying cellular identities (both novel and well-studied) is one of the key use cases in single-cell transcriptomics. While supervised machine learning has been leveraged to automate cell annotation predictions for some time, there has been relatively little progress both in scaling neural networks to large data sets and in constructing models that generalize well across diverse tissues and biological contexts up to whole organisms. Here, we propose scTab, an automated, feature-attention-based cell type prediction model specific to tabular data, and train it using a novel data augmentation scheme across a large corpus of single-cell RNA-seq observations (22.2 million human cells in total). In addition, scTab leverages deep ensembles for uncertainty quantification. Moreover, we account for ontological relationships between labels in the model evaluation to accommodate for differences in annotation granularity across datasets. On this large-scale corpus, we show that cross-tissue annotation requires nonlinear models and that the performance of scTab scales in terms of training dataset size as well as model size - demonstrating the advantage of scTab over current state-of-the-art linear models in this context. Additionally, we show that the proposed data augmentation schema improves model generalization. In summary, we introduce a de novo cell type prediction model for single-cell RNA-seq data that can be trained across a large-scale collection of curated datasets from a diverse selection of human tissues and demonstrate the benefits of using deep learning methods in this paradigm. Our codebase, training data, and model checkpoints are publicly available at https://github.com/theislab/scTab to further enable rigorous benchmarks of foundation models for single-cell RNA-seq data.\nComputational design of soluble analogues of integral membrane protein structures\nAuthors: Goverde, C. A.; Pacesa, M.; Dornfeld, L. J.; Goldbach, N.; Georgeon, S.; Rosset, S.; Dauparas, J.; Shellhaas, C.; Kozlov, S.; Baker, D.; Ovchinnikov, S.; Correia, B.\nScore: 66.3, Published: 2023-10-12 DOI: 10.1101/2023.05.09.540044\nDe novo design of complex protein folds using solely computational means remains a significant challenge. Here, we use a robust deep learning pipeline to design complex folds and soluble analogues of integral membrane proteins. Unique membrane topologies, such as the GPCRs, are not found in the soluble proteome and we demonstrate that their structural features can be recapitulated in solution. Biophysical analyses reveal high thermal stability of the designs and experimental structures show remarkable design accuracy. Transplantation of native structural motifs demonstrates their potential for further functionalization as novel small molecule binders and new approaches to drug discovery. In summary, our workflow enables the design of complex protein topologies with high experimental success rates and low sequence similarity to natural proteins, leading to a de facto expansion of the soluble fold space.\nCUDASW++4.0: Ultra-fast GPU-based Smith-Waterman Protein Sequence Database Search\nAuthors: Schmidt, B.; Kallenborn, F.; Chacon, A.; Hundt, C.\nScore: 14.9, Published: 2023-10-12 DOI: 10.1101/2023.10.09.561526\nBackground: The maximal sensitivity for local pairwise alignment makes the Smith-Waterman algorithm a popular choice forprotein sequence database search. However, its quadratic time complexity makes it compute-intensive. Unfortunately, current state-of-the-art software tools are not able to leverage the massively parallel processing capabilities of modern GPUs with close-to-peak performance. This motivates the need for more efficient implementations. Results: CUDASW++4.0 is a fast software tool for scanning protein sequence databases with the Smith-Waterman algorithm on CUDA-enabled GPUs. Our approach achieves high efficiency for dynamic programming-based alignment computation by minimizing memory accesses and instructions. We provide both efficient matrixtiling and sequence database partitioning schemes, and exploit next generation floating point arithmetic and novel DPX instructions. This leads to close-to-peak performance on modern GPU generations (Ampere, Ada, Hopper) with throughput rates of up to 1.94 TCUPS, 5.01 TCUPS, 5.71 TCUPS on an A100, L40S, and H100, respectively. Evaluation on the Swiss-Prot, UniRef50, and TrEMBL databases shows that CUDASW++4.0 gains over an order-of-magnitude performance improvements over previous GPU-based approaches (CUDASW++3.0, ADEPT). In addition, our algorithm demonstrates significant speedups over top-performing CPU-based tools (BLASTP, SWIPE, SWIMM2.0), can exploit multi-GPU nodes with linear scaling, and features an impressive energy efficiency of up to 15.7 GCUPS/Watt. Conclusion: CUDASW++4.0 changes the standing of GPUs in protein sequence database search with Smith-Waterman alignment by providing close-to-peak performance on modern GPUs. It is freely available at https://github.com/asbschmidt/CUDASW4.\nBuDDI: Bulk Deconvolution with Domain Invariance to predict cell-type-specific perturbations from bulk\nAuthors: Davidson, N. R.; Greene, C. S.\nScore: 16.5, Published: 2023-10-05 DOI: 10.1101/2023.07.20.549951\nWhile single-cell experiments provide deep cellular resolution within a single sample, some single-cell experiments are inherently more challenging than bulk experiments due to dissociation difficulties, cost, or limited tissue availability. This creates a situation where we have deep cellular profiles of one sample or condition, and bulk profiles across multiple samples and conditions. To bridge this gap, we propose BuDDI (BUlk Deconvolution with Domain Invariance). BuDDI utilizes domain adaptation techniques to effectively integrate available corpora of case-control bulk and reference scRNA-seq observations to infer cell-type-specific perturbation effects. BuDDI achieves this by learning independent latent spaces within a single variational autoencoder (VAE) encompassing at least four sources of variability: 1) cell-type proportion, 2) perturbation effect, 3) structured experimental variability, and 4) remaining variability. Since each latent space is encouraged to be independent, we simulate perturbation responses by independently composing each latent space to simulate cell-type-specific perturbation responses. We evaluated BuDDIs performance on simulated and real data with experimental designs of increasing complexity. We first validated that BuDDI could learn domain invariant latent spaces on data with matched samples across each source of variability. Then we validated that BuDDI could accurately predict cell-type-specific perturbation response when no single-cell perturbed profiles were used during training; instead, only bulk samples had both perturbed and non-perturbed observations. Finally, we validated BuDDI on predicting sex-specific differences, an experimental design where it is not possible to have matched samples. In each experiment, BuDDI outperformed all other comparative methods and baselines. As more reference atlases are completed, BuDDI provides a path to combine these resources with bulk-profiled treatment or disease signatures to study perturbations, sex differences, or other factors at single-cell resolution.\nDeciphering Cancer Genomes with GenomeSpy: A Grammar-Based Visualization Toolkit\nAuthors: Lavikka, K.; Oikkonen, J.; Li, Y.; Muranen, T.; Micoli, G.; Marchi, G.; Lahtinen, A.; Huhtinen, K.; Lehtonen, R.; Hietanen, S.; Hynninen, J.; Virtanen, A.; Hautaniemi, S.\nScore: 8.6, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561159\nBackgroundVisualization is an indispensable facet of genomic data analysis. Despite the abundance of specialized visualization tools, there remains a distinct need for tailored solutions. However, their implementation typically requires extensive programming expertise from bioinformaticians and software developers, especially when building interactive applications. Toolkits based on visualization grammars offer a more accessible, declarative way to author new visualizations. Nevertheless, current grammar-based solutions fall short in adequately supporting the interactive analysis of large data sets with extensive sample collections, a pivotal task often encountered in cancer research. ResultsWe present GenomeSpy, a grammar-based toolkit for authoring tailored, interactive visualizations for genomic data analysis. Users can implement new visualization designs with little effort by using combinatorial building blocks that are put together with a declarative language. These fully customizable visualizations can be embedded in web pages or end-user-oriented applications. The toolkit also includes a fully customizable but user-friendly application for analyzing sample collections, which may comprise genomic and clinical data. Findings can be bookmarked and shared as links that incorporate provenance information. A distinctive element of GenomeSpys architecture is its effective use of the graphics processing unit (GPU) in all rendering. GPU usage enables a high frame rate and smoothly animated interactions, such as navigation within a genome. We demonstrate the utility of GenomeSpy by characterizing the genomic landscape of 753 ovarian cancer samples from patients in the DECIDER clinical trial. Our results expand the understanding of the genomic architecture in ovarian cancer, particularly the diversity of chromosomal instability. We also show how GenomeSpy enabled the discovery of clinically actionable genomic aberrations. ConclusionsGenomeSpy is a visualization toolkit applicable to a wide range of tasks pertinent to genome analysis. It offers high flexibility and exceptional performance in interactive analysis. The toolkit is open source with an MIT license, implemented in JavaScript, and available at https://genomespy.app/.\n",
  "wordCount" : "2554",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561776">
        <p class="paperTitle">GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.</p>
        <p class="info">Score: 52.4, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561776' target='https://doi.org/10.1101/2023.10.10.561776'> 10.1101/2023.10.10.561776</a></p>
        <p class="abstract">Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train. Across several benchmarks on clinical databases (ClinVar, COSMIC, and OMIM) and population genomic data (gnomAD), our model for the human genome achieves outstanding performance on deleteriousness prediction for both coding and non-coding variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561091" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561091">
        <p class="paperTitle">AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.</p>
        <p class="info">Score: 35.1, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561091' target='https://doi.org/10.1101/2023.10.06.561091'> 10.1101/2023.10.06.561091</a></p>
        <p class="abstract">MotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics. Ideally, phylogenetic methods should not only model substitutions along gene trees, which explain differences between homologous gene sequences, but also the processes that generate the gene trees themselves along a shared species tree. To conduct accurate inferences, one needs to account for uncertainty at both levels, that is, in gene trees estimated from inherently short sequences and in their diverse evolutionary histories along a shared species tree.

ResultsWe present AleRax, a software that can infer reconciled gene trees together with a shared species tree using a simple, yet powerful, probabilistic model of gene duplication, transfer, and loss. A key feature of AleRax is its ability to account for uncertainty in the gene tree and its reconciliation by using an efficient approximation to calculate the joint phylogenetic-reconciliation likelihood and sample reconciled gene trees accordingly. Simulations and analyses of empirical data show that AleRax is one order of magnitude faster than competing gene tree inference tools while attaining the same accuracy. It is consistently more robust than species tree inference methods such as SpeciesRax and ASTRAL-Pro 2 under gene tree uncertainty. Finally, AleRax can process multiple gene families in parallel thereby allowing users to compare competing phylogenetic hypotheses and estimate model parameters, such as DTL probabilities for genome-scale datasets with hundreds of taxa

Availability and ImplementationGNU GPL at https://github.com/BenoitMorel/AleRax and data are made available at https://cme.h-its.org/exelixis/material/alerax_data.tar.gz.

ContactBenoit.Morel@h-its.org

Supplementary informationSupplementary material is available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561583">
        <p class="paperTitle">On the relationship between protist metabarcoding and protist metagenome-assembled genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zavadska, D.; Henry, N.; Auladell, A.; Berney, C.; Richter, D. J.</p>
        <p class="info">Score: 29.6, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561583' target='https://doi.org/10.1101/2023.10.09.561583'> 10.1101/2023.10.09.561583</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe two most commonly used approaches to study the composition of environmental protist communities are marker gene metabarcoding and whole genome analysis through metagenomics. Raw metabarcoding data are usually processed into Operational Taxonomic Units (OTUs) or amplicon sequence variants (ASVs) through clustering or denoising approaches, respectively. Analogous approaches have been developed to assemble metagenomic sequence reads into metagenome-assembled genomes (MAGs). Understanding the correspondence between the data produced by these two approaches can help to integrate information between the datasets and to explain how metabarcoding OTUs and MAGs are related with the underlying biological entities they are hypothesised to represent. Due to the nature of their construction, MAGs do not contain the most commonly used barcoding loci, meaning that sequence homology approaches cannot be used to match OTUs and MAGs.

We made an attempt to match V9 metabarcoding OTUs from the 18S rRNA gene (V9 OTUs) and MAGs from the Tara Oceans expedition (2009-2013) based on the correspondence of their relative abundances across the same set of samples. We evaluated the performance of several methods for detecting correspondence between features in these two compositional datasets and developed a series of controls to filter artefacts of data structure and processing. After selecting the best-performing correspondence metrics, ranking the V9 OTU/MAG matches by their proportionality/correlation coefficients and applying a set of selection criteria, we identified candidate matches between V9 OTUs and MAGs.

In a subset of cases, V9 OTUs and MAGs could be successfully matched with one another with a one-to-one correspondence, implying that they likely represent the same underlying biological entity. More generally, matches we observed could be classified into 4 scenarios: Scenario I - one V9 OTU matches more than one MAG; Scenario II - more than one V9 OTU matches more than one MAG; Scenario III - more than one V9 OTU matches one MAG; Scenario IV - one V9 OTU matches one MAG.

These diverse scenarios for V9 OTU-MAG matches illustrate the complex nature of the OTU/MAG relationship. Notably, we found some instances in which different OTU-MAG matches from the same taxonomic group were not classified in the same scenario, with all four scenarios possible even within the same taxonomic group, illustrating that factors beyond taxonomic lineage influence the relationship between OTUs and MAGs. Overall, each scenario produces a different interpretation of V9 OTUs, MAGs and how they compare in terms of the genomic and ecological diversity that they represent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560733">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560733" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560733">
        <p class="paperTitle">Characterizing cell-type spatial relationships across length scales in spatially resolved omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560733" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560733" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: dos Santos Peixoto, R.; Miller, B. F.; Brusko, M. A.; Atta, L.; Anant, M.; Atkinson, M. A.; Brusko, T. M.; Wasserfall, C. H.; Fan, J.</p>
        <p class="info">Score: 29.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560733' target='https://doi.org/10.1101/2023.10.05.560733'> 10.1101/2023.10.05.560733</a></p>
        <p class="abstract">Spatially resolved omics technologies provide molecular profiling of cells while preserving their organization within tissues, allowing for the evaluation of cell-type spatial relationships. We developed CRAWDAD to quantify cell-type spatial relationships across length scales. We highlight the utility of such multi-scale characterization on simulated data, recapitulate expected cell-type spatial relationships in tissues such as the mouse brain and embryo, and delineate functionally relevant spatial-defined cell-type subsets in the human spleen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561789" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561789">
        <p class="paperTitle">Uncover spatially informed shared variations for single-cell spatial transcriptomics with STew</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, N.; Vargas, J.; Fritz, D.; Krishna, R.; Zhang, F.</p>
        <p class="info">Score: 20.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561789' target='https://doi.org/10.1101/2023.10.10.561789'> 10.1101/2023.10.10.561789</a></p>
        <p class="abstract">Motivation: The recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge. Results: We introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods. STew is also robust to generate consistent results insensitive to model parameters, including sparsity constraints. We next applied STew to various ST data acquired from 10x Visium and Slide-seqV2, encompassing samples from both mouse and human brain, which revealed spatially informed cell type clusters. We further identified a pro-inflammatory fibroblast spatial niche using ST data from psoriatic skins. Hence, STew is a generalized method to identify both spatially informed clusters and disease-relevant niches in complex tissues. Availability: Source code and the R software tool STew are available from github.com/fanzhanglab/STew.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561331">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561331" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561331">
        <p class="paperTitle">Scaling cross-tissue single-cell annotation models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561331" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561331" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, F.; Fischer, D. S.; Biederstedt, E.; Villani, A.-C.; Theis, F. J.</p>
        <p class="info">Score: 19.3, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561331' target='https://doi.org/10.1101/2023.10.07.561331'> 10.1101/2023.10.07.561331</a></p>
        <p class="abstract">Identifying cellular identities (both novel and well-studied) is one of the key use cases in single-cell transcriptomics. While supervised machine learning has been leveraged to automate cell annotation predictions for some time, there has been relatively little progress both in scaling neural networks to large data sets and in constructing models that generalize well across diverse tissues and biological contexts up to whole organisms. Here, we propose scTab, an automated, feature-attention-based cell type prediction model specific to tabular data, and train it using a novel data augmentation scheme across a large corpus of single-cell RNA-seq observations (22.2 million human cells in total). In addition, scTab leverages deep ensembles for uncertainty quantification. Moreover, we account for ontological relationships between labels in the model evaluation to accommodate for differences in annotation granularity across datasets. On this large-scale corpus, we show that cross-tissue annotation requires nonlinear models and that the performance of scTab scales in terms of training dataset size as well as model size - demonstrating the advantage of scTab over current state-of-the-art linear models in this context. Additionally, we show that the proposed data augmentation schema improves model generalization. In summary, we introduce a de novo cell type prediction model for single-cell RNA-seq data that can be trained across a large-scale collection of curated datasets from a diverse selection of human tissues and demonstrate the benefits of using deep learning methods in this paradigm. Our codebase, training data, and model checkpoints are publicly available at https://github.com/theislab/scTab to further enable rigorous benchmarks of foundation models for single-cell RNA-seq data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.09.540044">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.09.540044" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.09.540044">
        <p class="paperTitle">Computational design of soluble analogues of integral membrane protein structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.09.540044" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.09.540044" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goverde, C. A.; Pacesa, M.; Dornfeld, L. J.; Goldbach, N.; Georgeon, S.; Rosset, S.; Dauparas, J.; Shellhaas, C.; Kozlov, S.; Baker, D.; Ovchinnikov, S.; Correia, B.</p>
        <p class="info">Score: 66.3, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.09.540044' target='https://doi.org/10.1101/2023.05.09.540044'> 10.1101/2023.05.09.540044</a></p>
        <p class="abstract">De novo design of complex protein folds using solely computational means remains a significant challenge. Here, we use a robust deep learning pipeline to design complex folds and soluble analogues of integral membrane proteins. Unique membrane topologies, such as the GPCRs, are not found in the soluble proteome and we demonstrate that their structural features can be recapitulated in solution. Biophysical analyses reveal high thermal stability of the designs and experimental structures show remarkable design accuracy. Transplantation of native structural motifs demonstrates their potential for further functionalization as novel small molecule binders and new approaches to drug discovery. In summary, our workflow enables the design of complex protein topologies with high experimental success rates and low sequence similarity to natural proteins, leading to a de facto expansion of the soluble fold space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561526">
        <p class="paperTitle">CUDASW&#43;&#43;4.0: Ultra-fast GPU-based Smith-Waterman Protein Sequence Database Search</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmidt, B.; Kallenborn, F.; Chacon, A.; Hundt, C.</p>
        <p class="info">Score: 14.9, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561526' target='https://doi.org/10.1101/2023.10.09.561526'> 10.1101/2023.10.09.561526</a></p>
        <p class="abstract">Background: The maximal sensitivity for local pairwise alignment makes the Smith-Waterman algorithm a popular choice forprotein sequence database search. However, its quadratic time complexity makes it compute-intensive. Unfortunately, current state-of-the-art software tools are not able to leverage the massively parallel processing capabilities of modern GPUs with close-to-peak performance. This motivates the need for more efficient implementations. Results: CUDASW&#43;&#43;4.0 is a fast software tool for scanning protein sequence databases with the Smith-Waterman algorithm on CUDA-enabled GPUs. Our approach achieves high efficiency for dynamic programming-based alignment computation by minimizing memory accesses and instructions. We provide both efficient matrixtiling and sequence database partitioning schemes, and exploit next generation floating point arithmetic and novel DPX instructions. This leads to close-to-peak performance on modern GPU generations (Ampere, Ada, Hopper) with throughput rates of up to 1.94 TCUPS, 5.01 TCUPS, 5.71 TCUPS on an A100, L40S, and H100, respectively. Evaluation on the Swiss-Prot, UniRef50, and TrEMBL databases shows that CUDASW&#43;&#43;4.0 gains over an order-of-magnitude performance improvements over previous GPU-based approaches (CUDASW&#43;&#43;3.0, ADEPT). In addition, our algorithm demonstrates significant speedups over top-performing CPU-based tools (BLASTP, SWIPE, SWIMM2.0), can exploit multi-GPU nodes with linear scaling, and features an impressive energy efficiency of up to 15.7 GCUPS/Watt. Conclusion: CUDASW&#43;&#43;4.0 changes the standing of GPUs in protein sequence database search with Smith-Waterman alignment by providing close-to-peak performance on modern GPUs. It is freely available at https://github.com/asbschmidt/CUDASW4.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549951">
        <p class="paperTitle">BuDDI: Bulk Deconvolution with Domain Invariance to predict cell-type-specific perturbations from bulk</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, N. R.; Greene, C. S.</p>
        <p class="info">Score: 16.5, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549951' target='https://doi.org/10.1101/2023.07.20.549951'> 10.1101/2023.07.20.549951</a></p>
        <p class="abstract">While single-cell experiments provide deep cellular resolution within a single sample, some single-cell experiments are inherently more challenging than bulk experiments due to dissociation difficulties, cost, or limited tissue availability. This creates a situation where we have deep cellular profiles of one sample or condition, and bulk profiles across multiple samples and conditions. To bridge this gap, we propose BuDDI (BUlk Deconvolution with Domain Invariance). BuDDI utilizes domain adaptation techniques to effectively integrate available corpora of case-control bulk and reference scRNA-seq observations to infer cell-type-specific perturbation effects. BuDDI achieves this by learning independent latent spaces within a single variational autoencoder (VAE) encompassing at least four sources of variability: 1) cell-type proportion, 2) perturbation effect, 3) structured experimental variability, and 4) remaining variability. Since each latent space is encouraged to be independent, we simulate perturbation responses by independently composing each latent space to simulate cell-type-specific perturbation responses.

We evaluated BuDDIs performance on simulated and real data with experimental designs of increasing complexity. We first validated that BuDDI could learn domain invariant latent spaces on data with matched samples across each source of variability. Then we validated that BuDDI could accurately predict cell-type-specific perturbation response when no single-cell perturbed profiles were used during training; instead, only bulk samples had both perturbed and non-perturbed observations. Finally, we validated BuDDI on predicting sex-specific differences, an experimental design where it is not possible to have matched samples. In each experiment, BuDDI outperformed all other comparative methods and baselines. As more reference atlases are completed, BuDDI provides a path to combine these resources with bulk-profiled treatment or disease signatures to study perturbations, sex differences, or other factors at single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561159" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561159">
        <p class="paperTitle">Deciphering Cancer Genomes with GenomeSpy: A Grammar-Based Visualization Toolkit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lavikka, K.; Oikkonen, J.; Li, Y.; Muranen, T.; Micoli, G.; Marchi, G.; Lahtinen, A.; Huhtinen, K.; Lehtonen, R.; Hietanen, S.; Hynninen, J.; Virtanen, A.; Hautaniemi, S.</p>
        <p class="info">Score: 8.6, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561159' target='https://doi.org/10.1101/2023.10.06.561159'> 10.1101/2023.10.06.561159</a></p>
        <p class="abstract">BackgroundVisualization is an indispensable facet of genomic data analysis. Despite the abundance of specialized visualization tools, there remains a distinct need for tailored solutions. However, their implementation typically requires extensive programming expertise from bioinformaticians and software developers, especially when building interactive applications. Toolkits based on visualization grammars offer a more accessible, declarative way to author new visualizations. Nevertheless, current grammar-based solutions fall short in adequately supporting the interactive analysis of large data sets with extensive sample collections, a pivotal task often encountered in cancer research.

ResultsWe present GenomeSpy, a grammar-based toolkit for authoring tailored, interactive visualizations for genomic data analysis. Users can implement new visualization designs with little effort by using combinatorial building blocks that are put together with a declarative language. These fully customizable visualizations can be embedded in web pages or end-user-oriented applications. The toolkit also includes a fully customizable but user-friendly application for analyzing sample collections, which may comprise genomic and clinical data. Findings can be bookmarked and shared as links that incorporate provenance information. A distinctive element of GenomeSpys architecture is its effective use of the graphics processing unit (GPU) in all rendering. GPU usage enables a high frame rate and smoothly animated interactions, such as navigation within a genome. We demonstrate the utility of GenomeSpy by characterizing the genomic landscape of 753 ovarian cancer samples from patients in the DECIDER clinical trial. Our results expand the understanding of the genomic architecture in ovarian cancer, particularly the diversity of chromosomal instability. We also show how GenomeSpy enabled the discovery of clinically actionable genomic aberrations.

ConclusionsGenomeSpy is a visualization toolkit applicable to a wide range of tasks pertinent to genome analysis. It offers high flexibility and exceptional performance in interactive analysis. The toolkit is open source with an MIT license, implemented in JavaScript, and available at https://genomespy.app/.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
