<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Voyager: exploratory single-cell genomics data analysis with geospatial statistics
Authors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.
Score: 130.7, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945
Exploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Voyager: exploratory single-cell genomics data analysis with geospatial statistics
Authors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.
Score: 130.7, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945
Exploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Voyager: exploratory single-cell genomics data analysis with geospatial statistics
Authors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.
Score: 130.7, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945
Exploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Voyager: exploratory single-cell genomics data analysis with geospatial statistics\nAuthors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.\nScore: 130.7, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945\nExploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis.",
  "keywords": [
    
  ],
  "articleBody": " Voyager: exploratory single-cell genomics data analysis with geospatial statistics\nAuthors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.\nScore: 130.7, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945\nExploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis. We introduce the Voyager platform, which systematically brings the geospatial ESDA tradition to (spatial) -omics, with local, bivariate, and multivariate spatial methods not yet commonly applied to spatial -omics, united by a uniform user interface. Using Voyager, we showcase biological insights that can be derived with its methods, such as biologically relevant negative spatial autocorrelation. Underlying Voyager is the SpatialFeatureExperiment data structure, which combines Simple Feature with SingleCellExperiment and AnnData to represent and operate on geometries bundled with gene expression data. Voyager has comprehensive tutorials demonstrating ESDA built on GitHub Actions to ensure reproducibility and scalability, using data from popular commercial technologies. Voyager is implemented in both R/Bioconductor and Python/PyPI, and features compatibility tests to ensure that both implementations return consistent results.\nSSDraw: software for generating comparative protein secondary structure diagrams\nAuthors: Chen, E. A.; Porter, L. L.\nScore: 50.8, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554905\nThe program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment. Linear visualization allows the user to stack registered diagrams, facilitating comparison of secondary structure and other properties among homologous proteins. SSDraw can be used to compare secondary structures of homologous proteins with both conserved and divergent folds. It can also generate one secondary structure diagram from an input protein structure of interest. The source code can be downloaded and run locally for rapid structure generation, while a Google Colab notebook allows for easy use.\nSegment Anything for Microscopy\nAuthors: Archit, A.; Nair, S.; Khalid, N.; Hilt, P.; Rajashekar, V.; Freitag, M.; Gupta, S.; Dengel, A.; Ahmed, S.; Pape, C.\nScore: 40.8, Published: 2023-08-26 DOI: 10.1101/2023.08.21.554208\nWe present Segment Anything for Microscopy, a tool for interactive and automatic segmentation and tracking of objects in multi-dimensional microscopy data. Our method is based on Segment Anything, a vision foundation model for image segmentation. We extend it by training specialized models for microscopy data that significantly improve segmentation quality for a wide range of imaging conditions. We also implement annotation tools for interactive (volumetric) segmentation and tracking, that speed up data annotation significantly compared to established tools. Our work constitutes the first application of vision foundation models to microscopy, laying the groundwork for solving image analysis problems in these domains with a small set of powerful deep learning architectures.\nA multimodal Transformer Network for protein-small molecule interactions enhances drug-target affinity and enzyme-substrate predictions\nAuthors: Kroll, A.; Ranjan, S.; Lercher, M. J.\nScore: 29.7, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554147\nThe activities of most enzymes and drugs rely on interactions between proteins and small molecules. Accurate predictions of these interactions could accelerate pharmaceutical and biotechnological research massively. Machine learning models designed for this task are currently limited by the lack of information exchange between the protein and the small molecule during the creation of the required numerical representations. Here, we introduce ProSmith, a machine learning framework that employs a multimodal Transformer Network to simultaneously process protein amino acid sequences and small molecule strings in the same input. This approach facilitates the exchange of all relevant information between the two molecule types during the calculation of their numerical representations, allowing the model to account for their structural and functional interactions. Our final model combines gradient boosting predictions based on the resulting multimodal Transformer Network with independent predictions based on separate deep learning representations of the proteins and small molecules. The corresponding predictions outperform all previous models for predicting protein-small molecule interactions across three diverse tasks: predicting Michaelis constants KM; inferring potential substrates for enzymes; and predicting protein-drug affinities. The Python code provided can be used to easily implement and improve machine learning predictions involving arbitrary protein-small molecule interactions.\nA fast comparative genome browser for diverse bacteria and archaea\nAuthors: Price, M. N.; Arkin, A. P.\nScore: 28.1, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554478\nGenome sequencing has revealed an incredible diversity of bacteria and archaea, but there are no fast and convenient tools for browsing across these genomes. It is cumbersome to view the prevalence of homologs for a protein of interest, or the gene neighborhoods of those homologs, across the diversity of the prokaryotes. We developed a web-based tool, fast.genomics, that uses two strategies to support fast browsing across the diversity of prokaryotes. First, the database of genomes is split up. The main database contains one representative from each of the 6,377 genera that have a high-quality genome, and additional databases for each taxonomic order contain up to 10 representatives of each species. Second, homologs of proteins of interest are identified quickly by using accelerated searches, usually in a few seconds. Once homologs are identified, fast.genomics can quickly show their prevalence across taxa, view their neighboring genes, or compare the prevalence of two different proteins. Fast.genomics is available at https://fast.genomics.lbl.gov.\npyScoMotif: Discovery of similar 3D structural motifs across proteins\nAuthors: Cia, G.; Kwasigroch, J. M.; Stamatopoulos, B.; Rooman, M.; Pucci, F.\nScore: 26.7, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554982\nThe fast and accurate detection of similar geometrical arrangements of protein residues, known as 3-dimensional (3D) structural motifs, is highly relevant for many applications such as binding region and catalytic site detection, drug discovery and structure conservation analyses. With the recent publication of new protein structure prediction methods, the number of available protein structures is exploding, which makes efficient and easy-to-use tools for identifying 3D structural motifs essential. We present an open-source Python package that enables the search for both exact and mutated motifs with position-specific residue substitutions. The tool is fast, accurate and suitable to run both on computer clusters and personal laptops. Successful application of pyScoMotif to catalytic site identification is showcased. The pyScoMotif package can be installed from the PyPI repository and is also available at https://github.com/3BioCompBio/pyScoMotif. It is free to use for non-commercial purposes; for commercial purposes, please contact us.\nHighfold: accurately predicting cyclic peptide monomers and complexes with AlphaFold\nAuthors: Zhang, C.; Zhang, C.; Shang, T.; Wu, X.; Duan, H.\nScore: 23.3, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554979\nIn recent years, cyclic peptides have gained growing traction as a therapeutic modality owing to their diverse biological activities. Understanding the structures of these cyclic peptides and their complexes can provide valuable insights. However, experimental observation needs much time and money, and there still are many limitations to CADD methods. As for DL-based models, the scarcity of training data poses a formidable challenge in predicting cyclic peptides and their complexes. In this work, we present \"Highfold,\" an AlphaFold-based algorithm that addresses this issue. By incorporating pertinent information about head-to-tailed circular and disulfide bridge structures, Highfold reaches the best performance in comparison to other various approaches. This model enables accurate prediction of cyclic peptides and their complexes, making a step towards resolving its structure-activity research.\nAnalyzing microbial evolution through gene and genome phylogenies\nAuthors: Teichman, S.; Lee, M. D.; Willis, A. D.\nScore: 33.5, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553440\nAO_SCPLOWBSTRACTC_SCPLOWMicrobiome scientists critically need modern tools to explore and analyze microbial evolution. Often this involves studying the evolution of microbial genomes as a whole. However, different genes in a single genome can be subject to different evolutionary pressures, which can result in distinct gene-level evolutionary histories. To address this challenge, we propose to treat estimated gene-level phylogenies as data objects, and present an interactive method for the analysis of a collection of gene phylogenies. We use a local linear approximation of phylogenetic tree space to visualize estimated gene trees as points in low-dimensional Euclidean space, and address important practical limitations of existing related approaches, allowing an intuitive visualization of complex data objects. We demonstrate the utility of our proposed approach through microbial data analyses, including by identifying outlying gene histories in strains of Prevotella, and by contrasting Streptococcus phylogenies estimated using different gene sets. Our method is available as an open-source R package, and assists with estimating, visualizing and interacting with a collection of bacterial gene phylogenies. dimension reduction, microbiome, non-Euclidean, statistical genetics, visualization\nScalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages\nAuthors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Corrada Bravo, H.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.\nScore: 159.0, Published: 2023-08-26 DOI: 10.1101/2023.07.18.549537\nSingle-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date. A singular view of this vast and growing expression landscape could help reveal novel associations between cell states and diseases, discover cell states in unexpected tissue contexts, and relate in vivo cells to in vitro models. However, these require a common, scalable representation of cell profiles from across the body, a general measure of their similarity, and an efficient way to query these data. Here, we present SCimilarity, a metric learning framework to learn and search a unified and interpretable representation that annotates cell types and instantaneously queries for a cell state across tens of millions of profiles. We demonstrate SCimilarity on a 22.7 million cell corpus assembled across 399 published scRNA-seq studies, showing accurate integration, annotation and querying. We experimentally validated SCimilarity by querying across tissues for a macrophage subset originally identified in interstitial lung disease, and showing that cells with similar profiles are found in other fibrotic diseases, tissues, and a 3D hydrogel system, which we then repurposed to yield this cell state in vitro. SCimilarity serves as a foundational model for single cell gene expression data and enables researchers to query for similar cellular states across the entire human body, providing a powerful tool for generating novel biological insights from the growing Human Cell Atlas.\nSelf-supervised learning for characterising histomorphological diversity and spatial RNA expression prediction across 23 human tissue types\nAuthors: Cisternino, F.; Ometto, S.; Chatterjee, S.; Giacopuzzi, E.; Levine, A. P.; Glastonbury, C. A.\nScore: 21.8, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554251\nAs vast histological archives are digitised, there is a pressing need to be able to associate specific tissue substructures and incident pathology to disease outcomes without arduous annotation. Such automation provides an opportunity to learn fundamental biology about how tissue structure and function varies in a population. Recently, self-supervised learning has proven competitive to supervised machine learning approaches in classification, segmentation and representation learning. Here, we leverage self-supervised learning to generate histology feature representations using 1.7M images across 23 healthy tissues in 838 donors from GTEx. Using these representations, we demonstrate we can automatically segment tissues into their constituent tissue substructures and pathology proportions, and surpass the performance of conventionally used pre-trained models. We observe striking population variability in canonical tissue substructures, highlight examples of missing pathological diagnoses, incorrect assignment of target tissue and cross-tissue contamination. We demonstrate that this variability in tissue composition leads to a likely overestimation of eQTL tissue sharing and drives dramatic differential gene expression changes. We use derived tissue substructures to detect 284 tissue substructures and pathology specific eQTLs. As our derived histology representations are rich morphological descriptors of the underlying tissue, we introduce a multiple instance learning model that can predict and spatially localise individual RNA expression levels directly from histology to specific substructures and pathological features. We validate our RNA spatial predictions with matched ground truth immunohistochemistry (IHC) for several well characterised marker genes, recapitulating their known spatial specificity. Finally, we derive a gene expression spatial enrichment metric, allowing us to detect genes specifically expressed within sites of pathology (e.g. arterial calcification). Together, these results demonstrate the power of self-supervised machine learning when applied to vast histological datasets to allow researchers to pose and answer questions about tissue pathology, its spatial organisation and the interplay between morphological tissue variability and gene expression.\n",
  "wordCount" : "2036",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549945">
        <p class="paperTitle">Voyager: exploratory single-cell genomics data analysis with geospatial statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.</p>
        <p class="info">Score: 130.7, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549945' target='https://doi.org/10.1101/2023.07.20.549945'> 10.1101/2023.07.20.549945</a></p>
        <p class="abstract">Exploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis. We introduce the Voyager platform, which systematically brings the geospatial ESDA tradition to (spatial) -omics, with local, bivariate, and multivariate spatial methods not yet commonly applied to spatial -omics, united by a uniform user interface. Using Voyager, we showcase biological insights that can be derived with its methods, such as biologically relevant negative spatial autocorrelation. Underlying Voyager is the SpatialFeatureExperiment data structure, which combines Simple Feature with SingleCellExperiment and AnnData to represent and operate on geometries bundled with gene expression data. Voyager has comprehensive tutorials demonstrating ESDA built on GitHub Actions to ensure reproducibility and scalability, using data from popular commercial technologies. Voyager is implemented in both R/Bioconductor and Python/PyPI, and features compatibility tests to ensure that both implementations return consistent results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554905">
        <p class="paperTitle">SSDraw: software for generating comparative protein secondary structure diagrams</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, E. A.; Porter, L. L.</p>
        <p class="info">Score: 50.8, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554905' target='https://doi.org/10.1101/2023.08.25.554905'> 10.1101/2023.08.25.554905</a></p>
        <p class="abstract">The program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment. Linear visualization allows the user to stack registered diagrams, facilitating comparison of secondary structure and other properties among homologous proteins. SSDraw can be used to compare secondary structures of homologous proteins with both conserved and divergent folds. It can also generate one secondary structure diagram from an input protein structure of interest. The source code can be downloaded and run locally for rapid structure generation, while a Google Colab notebook allows for easy use.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554208">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554208" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554208">
        <p class="paperTitle">Segment Anything for Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554208" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554208" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Archit, A.; Nair, S.; Khalid, N.; Hilt, P.; Rajashekar, V.; Freitag, M.; Gupta, S.; Dengel, A.; Ahmed, S.; Pape, C.</p>
        <p class="info">Score: 40.8, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554208' target='https://doi.org/10.1101/2023.08.21.554208'> 10.1101/2023.08.21.554208</a></p>
        <p class="abstract">We present Segment Anything for Microscopy, a tool for interactive and automatic segmentation and tracking of objects in multi-dimensional microscopy data. Our method is based on Segment Anything, a vision foundation model for image segmentation. We extend it by training specialized models for microscopy data that significantly improve segmentation quality for a wide range of imaging conditions. We also implement annotation tools for interactive (volumetric) segmentation and tracking, that speed up data annotation significantly compared to established tools. Our work constitutes the first application of vision foundation models to microscopy, laying the groundwork for solving image analysis problems in these domains with a small set of powerful deep learning architectures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554147">
        <p class="paperTitle">A multimodal Transformer Network for protein-small molecule interactions enhances drug-target affinity and enzyme-substrate predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kroll, A.; Ranjan, S.; Lercher, M. J.</p>
        <p class="info">Score: 29.7, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554147' target='https://doi.org/10.1101/2023.08.21.554147'> 10.1101/2023.08.21.554147</a></p>
        <p class="abstract">The activities of most enzymes and drugs rely on interactions between proteins and small molecules. Accurate predictions of these interactions could accelerate pharmaceutical and biotechnological research massively. Machine learning models designed for this task are currently limited by the lack of information exchange between the protein and the small molecule during the creation of the required numerical representations. Here, we introduce ProSmith, a machine learning framework that employs a multimodal Transformer Network to simultaneously process protein amino acid sequences and small molecule strings in the same input. This approach facilitates the exchange of all relevant information between the two molecule types during the calculation of their numerical representations, allowing the model to account for their structural and functional interactions. Our final model combines gradient boosting predictions based on the resulting multimodal Transformer Network with independent predictions based on separate deep learning representations of the proteins and small molecules. The corresponding predictions outperform all previous models for predicting protein-small molecule interactions across three diverse tasks: predicting Michaelis constants KM; inferring potential substrates for enzymes; and predicting protein-drug affinities. The Python code provided can be used to easily implement and improve machine learning predictions involving arbitrary protein-small molecule interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554478">
        <p class="paperTitle">A fast comparative genome browser for diverse bacteria and archaea</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Price, M. N.; Arkin, A. P.</p>
        <p class="info">Score: 28.1, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554478' target='https://doi.org/10.1101/2023.08.23.554478'> 10.1101/2023.08.23.554478</a></p>
        <p class="abstract">Genome sequencing has revealed an incredible diversity of bacteria and archaea, but there are no fast and convenient tools for browsing across these genomes. It is cumbersome to view the prevalence of homologs for a protein of interest, or the gene neighborhoods of those homologs, across the diversity of the prokaryotes. We developed a web-based tool, fast.genomics, that uses two strategies to support fast browsing across the diversity of prokaryotes. First, the database of genomes is split up. The main database contains one representative from each of the 6,377 genera that have a high-quality genome, and additional databases for each taxonomic order contain up to 10 representatives of each species. Second, homologs of proteins of interest are identified quickly by using accelerated searches, usually in a few seconds. Once homologs are identified, fast.genomics can quickly show their prevalence across taxa, view their neighboring genes, or compare the prevalence of two different proteins. Fast.genomics is available at https://fast.genomics.lbl.gov.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554982" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554982">
        <p class="paperTitle">pyScoMotif: Discovery of similar 3D structural motifs across proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cia, G.; Kwasigroch, J. M.; Stamatopoulos, B.; Rooman, M.; Pucci, F.</p>
        <p class="info">Score: 26.7, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554982' target='https://doi.org/10.1101/2023.08.27.554982'> 10.1101/2023.08.27.554982</a></p>
        <p class="abstract">The fast and accurate detection of similar geometrical arrangements of protein residues, known as 3-dimensional (3D) structural motifs, is highly relevant for many applications such as binding region and catalytic site detection, drug discovery and structure conservation analyses. With the recent publication of new protein structure prediction methods, the number of available protein structures is exploding, which makes efficient and easy-to-use tools for identifying 3D structural motifs essential. We present an open-source Python package that enables the search for both exact and mutated motifs with position-specific residue substitutions. The tool is fast, accurate and suitable to run both on computer clusters and personal laptops. Successful application of pyScoMotif to catalytic site identification is showcased. The pyScoMotif package can be installed from the PyPI repository and is also available at https://github.com/3BioCompBio/pyScoMotif. It is free to use for non-commercial purposes; for commercial purposes, please contact us.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554979">
        <p class="paperTitle">Highfold: accurately predicting cyclic peptide monomers and complexes with AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, C.; Zhang, C.; Shang, T.; Wu, X.; Duan, H.</p>
        <p class="info">Score: 23.3, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554979' target='https://doi.org/10.1101/2023.08.27.554979'> 10.1101/2023.08.27.554979</a></p>
        <p class="abstract">In recent years, cyclic peptides have gained growing traction as a therapeutic modality owing to their diverse biological activities. Understanding the structures of these cyclic peptides and their complexes can provide valuable insights. However, experimental observation needs much time and money, and there still are many limitations to CADD methods. As for DL-based models, the scarcity of training data poses a formidable challenge in predicting cyclic peptides and their complexes. In this work, we present &#34;Highfold,&#34; an AlphaFold-based algorithm that addresses this issue. By incorporating pertinent information about head-to-tailed circular and disulfide bridge structures, Highfold reaches the best performance in comparison to other various approaches. This model enables accurate prediction of cyclic peptides and their complexes, making a step towards resolving its structure-activity research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553440">
        <p class="paperTitle">Analyzing microbial evolution through gene and genome phylogenies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Teichman, S.; Lee, M. D.; Willis, A. D.</p>
        <p class="info">Score: 33.5, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553440' target='https://doi.org/10.1101/2023.08.15.553440'> 10.1101/2023.08.15.553440</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWMicrobiome scientists critically need modern tools to explore and analyze microbial evolution. Often this involves studying the evolution of microbial genomes as a whole. However, different genes in a single genome can be subject to different evolutionary pressures, which can result in distinct gene-level evolutionary histories. To address this challenge, we propose to treat estimated gene-level phylogenies as data objects, and present an interactive method for the analysis of a collection of gene phylogenies. We use a local linear approximation of phylogenetic tree space to visualize estimated gene trees as points in low-dimensional Euclidean space, and address important practical limitations of existing related approaches, allowing an intuitive visualization of complex data objects. We demonstrate the utility of our proposed approach through microbial data analyses, including by identifying outlying gene histories in strains of Prevotella, and by contrasting Streptococcus phylogenies estimated using different gene sets. Our method is available as an open-source R package, and assists with estimating, visualizing and interacting with a collection of bacterial gene phylogenies. dimension reduction, microbiome, non-Euclidean, statistical genetics, visualization</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549537">
        <p class="paperTitle">Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Corrada Bravo, H.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.</p>
        <p class="info">Score: 159.0, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549537' target='https://doi.org/10.1101/2023.07.18.549537'> 10.1101/2023.07.18.549537</a></p>
        <p class="abstract">Single-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date. A singular view of this vast and growing expression landscape could help reveal novel associations between cell states and diseases, discover cell states in unexpected tissue contexts, and relate in vivo cells to in vitro models. However, these require a common, scalable representation of cell profiles from across the body, a general measure of their similarity, and an efficient way to query these data. Here, we present SCimilarity, a metric learning framework to learn and search a unified and interpretable representation that annotates cell types and instantaneously queries for a cell state across tens of millions of profiles. We demonstrate SCimilarity on a 22.7 million cell corpus assembled across 399 published scRNA-seq studies, showing accurate integration, annotation and querying. We experimentally validated SCimilarity by querying across tissues for a macrophage subset originally identified in interstitial lung disease, and showing that cells with similar profiles are found in other fibrotic diseases, tissues, and a 3D hydrogel system, which we then repurposed to yield this cell state in vitro. SCimilarity serves as a foundational model for single cell gene expression data and enables researchers to query for similar cellular states across the entire human body, providing a powerful tool for generating novel biological insights from the growing Human Cell Atlas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554251">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554251" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554251">
        <p class="paperTitle">Self-supervised learning for characterising histomorphological diversity and spatial RNA expression prediction across 23 human tissue types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554251" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554251" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cisternino, F.; Ometto, S.; Chatterjee, S.; Giacopuzzi, E.; Levine, A. P.; Glastonbury, C. A.</p>
        <p class="info">Score: 21.8, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554251' target='https://doi.org/10.1101/2023.08.22.554251'> 10.1101/2023.08.22.554251</a></p>
        <p class="abstract">As vast histological archives are digitised, there is a pressing need to be able to associate specific tissue substructures and incident pathology to disease outcomes without arduous annotation. Such automation provides an opportunity to learn fundamental biology about how tissue structure and function varies in a population. Recently, self-supervised learning has proven competitive to supervised machine learning approaches in classification, segmentation and representation learning. Here, we leverage self-supervised learning to generate histology feature representations using 1.7M images across 23 healthy tissues in 838 donors from GTEx. Using these representations, we demonstrate we can automatically segment tissues into their constituent tissue substructures and pathology proportions, and surpass the performance of conventionally used pre-trained models. We observe striking population variability in canonical tissue substructures, highlight examples of missing pathological diagnoses, incorrect assignment of target tissue and cross-tissue contamination. We demonstrate that this variability in tissue composition leads to a likely overestimation of eQTL tissue sharing and drives dramatic differential gene expression changes. We use derived tissue substructures to detect 284 tissue substructures and pathology specific eQTLs. As our derived histology representations are rich morphological descriptors of the underlying tissue, we introduce a multiple instance learning model that can predict and spatially localise individual RNA expression levels directly from histology to specific substructures and pathological features. We validate our RNA spatial predictions with matched ground truth immunohistochemistry (IHC) for several well characterised marker genes, recapitulating their known spatial specificity. Finally, we derive a gene expression spatial enrichment metric, allowing us to detect genes specifically expressed within sites of pathology (e.g. arterial calcification). Together, these results demonstrate the power of self-supervised machine learning when applied to vast histological datasets to allow researchers to pose and answer questions about tissue pathology, its spatial organisation and the interplay between morphological tissue variability and gene expression.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
