<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records
Authors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records
Authors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records
Authors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records\nAuthors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J.",
  "keywords": [
    
  ],
  "articleBody": " OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records\nAuthors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J. A.\nScore: 579.9, Published: 2023-10-24 DOI: 10.1101/2023.10.16.562114\nBiological aging is a multifactorial process involving complex interactions of cellular and biochemical processes that is reflected in omic profiles. Using common clinical laboratory measures in ~30,000 individuals from the MGB-Biobank, we developed a robust, predictive biological aging phenotype, EMRAge, that balances clinical biomarkers with overall mortality risk and can be broadly recapitulated across EMRs. We then applied elastic-net regression to model EMRAge with DNA-methylation (DNAm) and multiple omics, generating DNAmEMRAge and OMICmAge, respectively. Both biomarkers demonstrated strong associations with chronic diseases and mortality that outperform current biomarkers across our discovery (MGB-ABC, n=3,451) and validation (TruDiagnostic, n=12,666) cohorts. Through the use of epigenetic biomarker proxies, OMICmAge has the unique advantage of expanding the predictive search space to include epigenomic, proteomic, metabolomic, and clinical data while distilling this in a measure with DNAm alone, providing opportunities to identify clinically-relevant interconnections central to the aging process.\nCellTracksColab, A platform for compiling, analyzing, and exploring tracking data\nAuthors: Jacquemet, G.\nScore: 40.2, Published: 2023-10-26 DOI: 10.1101/2023.10.20.563252\nIn life sciences, tracking objects from movies enables researchers to quantify the behavior of single particles, organelles, bacteria, cells, and even whole animals. While numerous tools now allow automated tracking from video, a significant challenge persists in compiling, analyzing, and exploring the large datasets generated by these approaches. Here, we introduce CellTracksColab, an online platform tailored to simplify the exploration and analysis of tracking data by leveraging the free, cloud-based computational resources of Google Colab. Cell-TracksColab facilitates the amalgamation and analysis of tracking results across multiple fields of view, conditions, and repeats, ensuring a holistic dataset overview. CellTracksColab also harnesses the power of high-dimensional data reduction and clustering, using Uniform Manifold Approximation and Projection and Hierarchical Density-Based Spatial Clustering of Applications with Noise, enabling researchers to identify distinct behavioral patterns and trends without bias. CellTracksColab is available for the broader scientific community here.\nSimulation of adaptive immune receptors and repertoires with complex immune information to guide the development and benchmarking of AIRR machine learning\nAuthors: Chernigovskaya, M.; Pavlovic, M.; Kanduri, C.; Gielis, S.; Robert, P. A.; Scheffer, L.; Slabodkin, A.; Haff, I. H.; Meysman, P.; Yaari, G.; Sandve, G. K.; Greiff, V.\nScore: 28.1, Published: 2023-10-23 DOI: 10.1101/2023.10.20.562936\nMachine-learning methods (ML) have shown great potential in the adaptive immune receptor repertoire (AIRR) field. However, there is a lack of large-scale ground-truth experimental AIRR data suitable for AIRR-ML-based disease diagnostics and therapeutics discovery. Simulated ground-truth AIRR data are required to complement the development and benchmarking of robust and interpretable AIRR-ML approaches where experimental data is inaccessible or insufficient as of yet. The challenge for simulated data to be useful is the ability to incorporate key features observed in experimental repertoires. These features, such as complex antigen or disease-associated immune information, cause AIRR-ML problems to be challenging. Here, we introduce LIgO, a modular software suite, which simulates AIRR data for the development and benchmarking of AIRR-based machine learning. LIgO incorporates different types of immune information both on the receptor and the repertoire level and preserves native-like generation probability distribution. Additionally, LIgO assists users in determining the computational feasibility of their simulations. We show two examples where LIgO simulation supports the development and validation of AIRR-ML methods: (1) how individuals carrying out-of-distribution immune information impacts receptor-level prediction performance and (2) how immune information co-occurring in the same AIRs have an impact on the performance of conventional receptor-level encoding and repertoire-level classification approaches. The LIgO software guides the advancement and assessment of interpretable AIRR-ML methods.\nProtein Language Models Uncover Carbohydrate-Active Enzyme Function in Metagenomic\nAuthors: Thurimella, K.; Mohamed, A. M. T.; Graham, D. B.; Owens, R. M.; La Rosa, S. L.; Plichta, D. R.; Bacallado, S.; Xavier, R. J.\nScore: 21.8, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563620\nIn metagenomics, the pool of uncharacterized microbial enzymes presents a challenge for functional annotation. Among these, carbohydrate-active enzymes (CAZymes) stand out due to their pivotal roles in various biological processes related to host health and nutrition. Here, we present CAZyLingua, the first tool that harnesses protein language model embeddings to build a deep learning framework that facilitates the annotation of CAZymes in metagenomic datasets. Our benchmarking results showed on average a higher F1 score (reflecting an average of precision and recall) on the annotated genomes of Bacteroides thetaiotaomicron, Eggerthella lenta and Ruminococcus gnavus compared to the traditional sequence homology-based method in dbCAN2. We applied our tool to a paired mother/infant longitudinal dataset and revealed unannotated CAZymes linked to microbial development during infancy. When applied to metagenomic datasets derived from patients affected by fibrosis-prone diseases such as Crohns disease and IgG4-related disease, CAZyLingua uncovered CAZymes associated with disease and healthy states. In each of these metagenomic catalogs, CAZyLingua discovered new annotations that were previously overlooked by traditional sequence homology tools. Overall, the deep learning model CAZyLingua can be applied in combination with existing tools to unravel intricate CAZyme evolutionary profiles and patterns, contributing to a more comprehensive understanding of microbial metabolic dynamics.\nVIPERA: Viral Intra-Patient Evolution Reporting and Analysis\nAuthors: Alvarez-Herrera, M.; Sevilla, J.; Ruiz-Rodriguez, P.; Vergara, A.; Vila, J.; Cano-Jimenez, P.; Gonzalez-Candelas, F.; Comas, I.; Coscolla, M.\nScore: 19.4, Published: 2023-10-25 DOI: 10.1101/2023.10.24.561010\nViral mutations within patients nurture the adaptive potential of SARS-CoV-2 during chronic infections, which are a potential source of variants of concern. However, there is no integrated framework for the evolutionary analysis of intra-patient SARS-CoV-2 serial samples. Herein we describe VIPERA (Viral Intra-Patient Evolution Reporting and Analysis), a new software that integrates the evaluation of the intra-patient ancestry of SARS-CoV-2 sequences with the analysis of evolutionary trajectories of serial sequences from the same viral infection. It has been validated using positive and negative control datasets and successfully applied to a novel case, contributing to easy and automatic analysis of intra-patient SARS-CoV-2 sequences.\nAssessing the limits of zero-shot foundation models in single-cell biology\nAuthors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.\nScore: 80.0, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085\nThe advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2\nVRP Assembler: haplotype-resolved de novo assembly of diploid and polyploid genomes using quantum computing\nAuthors: Chen, Y.; Huang, J.-H.; Sun, Y.; Zhang, Y.; Li, Y.; Xu, X.\nScore: 17.6, Published: 2023-10-21 DOI: 10.1101/2023.10.19.563028\nPrecision medicines emphasis on individual genetic variants highlights the importance of haplotype-resolved assembly, a computational challenge in bioinformatics given its combinatorial nature. While classical algorithms have made strides in addressing this issue, the potential of quantum computing remains largely untapped. Here, we present the VRP assembler: a novel approach that transforms this task into a vehicle routing problem, a combinatorial optimization problem solvable on a quantum computer. We demonstrate its potential and feasibility through a proof of concept on short synthetic diploid and triploid genomes using a D-Wave quantum annealer. To tackle larger-scale assembly problems, we integrate the VRP assembler with Googles OR-Tools, achieving a haplotype-resolved assembly across the human major histocompatibility complex (MHC) region. Our results show encouraging performance compared to hifiasm with the phasing accuracy first approaching the theoretical limit, underscoring the promising future of quantum computing in bioinformatics.\nA deep learning model embedded framework to distinguish DNA and RNA mutations directly from RNA-seq\nAuthors: Fu, Z.-C.; Gao, B.-Q.; Nan, F.; Ma, X.-K.; Yang, L.\nScore: 28.2, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562625\nWe develop a stepwise computational framework, called DEMINING, to directly detect expressed DNA and RNA mutations in RNA deep sequencing data. DEMINING incorporates a deep learning model named DeepDDR, which facilitates the separation of expressed DNA mutations from RNA mutations after RNA-seq read mapping and pileup. When applied in RNA-seq of acute myeloid leukemia patients, DEMINING uncovered previously-underappreciated DNA and RNA mutations, some associated with the upregulated expression of host genes or the production of neoantigens. Finally, we demonstrate that DEMINING could precisely classify DNA and RNA mutations in RNA-seq data from non-primate species through the utilization of transfer learning.\nPyTME (Python Template Matching Engine): A fast, flexible, and multi-purpose template matching library for cryogenic electron microscopy data\nAuthors: Maurer, V. J.; Siggel, M.; Kosinski, J.\nScore: 15.5, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563472\nCryogenic electron microscopy (cryo-EM) is a key method in structural and cell biology. Analysis of cryo-EM images requires interpretation of noisy, low-resolution densities which relies on identifying the most probable orientation of macromolecules in a target using template matching. Many method-specific template matching software exist for single-particle cryo-EM, cryo-electron tomography (cryo-ET), or fitting atomic structures into averaged 3D maps of macromolecules. Here, we report the Python Template Matching Engine (pyTME), a software engine that consolidates method-specific template matching problems. The underlying library provides highly efficient template-matching implementation and abstract data structures for storing and manipulating input and output data. It scales favorable to large datasets, both with multiple CPUs and GPUs, compared to existing software enabling template matching of even unbinned cryo-ET data in hours, which was previously nearly impossible due to technical restraints. Any hardware-specific optimization needed for dealing with large data is automatically performed to increase ease of use and minimize user intervention. The efficiency and simplicity of pyTME will enable high throughput mining of a variety of cryo-EM and ET datasets in the future.\nAutomated Bioinformatics Analysis via AutoBA\nAuthors: Zhou, J.; Zhang, B.; Chen, X.; Li, H.; Xu, X.; Chen, S.; Gao, X.\nScore: 26.0, Published: 2023-10-22 DOI: 10.1101/2023.09.08.556814\nWith the fast-growing and evolving omics data, the demand for streamlined and adaptable tools to handle the bioinformatics analysis continues to grow. In response to this need, we introduce Automated Bioinformatics Analysis (AutoBA), an autonomous AI agent designed explicitly for regular omics data analysis based on large language models. AutoBA simplifies the analytical process by requiring minimal user input while delivering detailed step-by-step plans for various bioinformatics tasks. Through rigorous validation by expert bioinformaticians, AutoBAs robustness and adaptability are affirmed across a diverse range of omics analysis cases, including whole genome/exome sequencing (WGS/WES), chromatin immunoprecipitation assays with sequencing (ChIP-seq), RNA sequencing (RNA-seq), single-cell RNA-seq, spatial transcriptomics and so on. AutoBAs unique capacity to self-design analysis processes based on input data variations further underscores its versatility. Compared with online bioinformatic services, AutoBA offers five LLM backends, with options for both online and local usage, prioritizing data security and user privacy. Moreover, different from the predefined pipeline, AutoBA has adaptability in sync with emerging bioinformatics tools. Overall, AutoBA represents an advanced and convenient tool, offering robustness and adaptability for conventional bioinformatics analysis.\n",
  "wordCount" : "1949",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562114">
        <p class="paperTitle">OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J. A.</p>
        <p class="info">Score: 579.9, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562114' target='https://doi.org/10.1101/2023.10.16.562114'> 10.1101/2023.10.16.562114</a></p>
        <p class="abstract">Biological aging is a multifactorial process involving complex interactions of cellular and biochemical processes that is reflected in omic profiles. Using common clinical laboratory measures in ~30,000 individuals from the MGB-Biobank, we developed a robust, predictive biological aging phenotype, EMRAge, that balances clinical biomarkers with overall mortality risk and can be broadly recapitulated across EMRs. We then applied elastic-net regression to model EMRAge with DNA-methylation (DNAm) and multiple omics, generating DNAmEMRAge and OMICmAge, respectively. Both biomarkers demonstrated strong associations with chronic diseases and mortality that outperform current biomarkers across our discovery (MGB-ABC, n=3,451) and validation (TruDiagnostic, n=12,666) cohorts. Through the use of epigenetic biomarker proxies, OMICmAge has the unique advantage of expanding the predictive search space to include epigenomic, proteomic, metabolomic, and clinical data while distilling this in a measure with DNAm alone, providing opportunities to identify clinically-relevant interconnections central to the aging process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563252">
        <p class="paperTitle">CellTracksColab, A platform for compiling, analyzing, and exploring tracking data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacquemet, G.</p>
        <p class="info">Score: 40.2, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563252' target='https://doi.org/10.1101/2023.10.20.563252'> 10.1101/2023.10.20.563252</a></p>
        <p class="abstract">In life sciences, tracking objects from movies enables researchers to quantify the behavior of single particles, organelles, bacteria, cells, and even whole animals. While numerous tools now allow automated tracking from video, a significant challenge persists in compiling, analyzing, and exploring the large datasets generated by these approaches. Here, we introduce CellTracksColab, an online platform tailored to simplify the exploration and analysis of tracking data by leveraging the free, cloud-based computational resources of Google Colab. Cell-TracksColab facilitates the amalgamation and analysis of tracking results across multiple fields of view, conditions, and repeats, ensuring a holistic dataset overview. CellTracksColab also harnesses the power of high-dimensional data reduction and clustering, using Uniform Manifold Approximation and Projection and Hierarchical Density-Based Spatial Clustering of Applications with Noise, enabling researchers to identify distinct behavioral patterns and trends without bias. CellTracksColab is available for the broader scientific community here.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.562936">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.562936" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.562936">
        <p class="paperTitle">Simulation of adaptive immune receptors and repertoires with complex immune information to guide the development and benchmarking of AIRR machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.562936" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.562936" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chernigovskaya, M.; Pavlovic, M.; Kanduri, C.; Gielis, S.; Robert, P. A.; Scheffer, L.; Slabodkin, A.; Haff, I. H.; Meysman, P.; Yaari, G.; Sandve, G. K.; Greiff, V.</p>
        <p class="info">Score: 28.1, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.562936' target='https://doi.org/10.1101/2023.10.20.562936'> 10.1101/2023.10.20.562936</a></p>
        <p class="abstract">Machine-learning methods (ML) have shown great potential in the adaptive immune receptor repertoire (AIRR) field. However, there is a lack of large-scale ground-truth experimental AIRR data suitable for AIRR-ML-based disease diagnostics and therapeutics discovery. Simulated ground-truth AIRR data are required to complement the development and benchmarking of robust and interpretable AIRR-ML approaches where experimental data is inaccessible or insufficient as of yet. The challenge for simulated data to be useful is the ability to incorporate key features observed in experimental repertoires. These features, such as complex antigen or disease-associated immune information, cause AIRR-ML problems to be challenging. Here, we introduce LIgO, a modular software suite, which simulates AIRR data for the development and benchmarking of AIRR-based machine learning. LIgO incorporates different types of immune information both on the receptor and the repertoire level and preserves native-like generation probability distribution. Additionally, LIgO assists users in determining the computational feasibility of their simulations. We show two examples where LIgO simulation supports the development and validation of AIRR-ML methods: (1) how individuals carrying out-of-distribution immune information impacts receptor-level prediction performance and (2) how immune information co-occurring in the same AIRs have an impact on the performance of conventional receptor-level encoding and repertoire-level classification approaches. The LIgO software guides the advancement and assessment of interpretable AIRR-ML methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563620">
        <p class="paperTitle">Protein Language Models Uncover Carbohydrate-Active Enzyme Function in Metagenomic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thurimella, K.; Mohamed, A. M. T.; Graham, D. B.; Owens, R. M.; La Rosa, S. L.; Plichta, D. R.; Bacallado, S.; Xavier, R. J.</p>
        <p class="info">Score: 21.8, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563620' target='https://doi.org/10.1101/2023.10.23.563620'> 10.1101/2023.10.23.563620</a></p>
        <p class="abstract">In metagenomics, the pool of uncharacterized microbial enzymes presents a challenge for functional annotation. Among these, carbohydrate-active enzymes (CAZymes) stand out due to their pivotal roles in various biological processes related to host health and nutrition. Here, we present CAZyLingua, the first tool that harnesses protein language model embeddings to build a deep learning framework that facilitates the annotation of CAZymes in metagenomic datasets. Our benchmarking results showed on average a higher F1 score (reflecting an average of precision and recall) on the annotated genomes of Bacteroides thetaiotaomicron, Eggerthella lenta and Ruminococcus gnavus compared to the traditional sequence homology-based method in dbCAN2. We applied our tool to a paired mother/infant longitudinal dataset and revealed unannotated CAZymes linked to microbial development during infancy. When applied to metagenomic datasets derived from patients affected by fibrosis-prone diseases such as Crohns disease and IgG4-related disease, CAZyLingua uncovered CAZymes associated with disease and healthy states. In each of these metagenomic catalogs, CAZyLingua discovered new annotations that were previously overlooked by traditional sequence homology tools. Overall, the deep learning model CAZyLingua can be applied in combination with existing tools to unravel intricate CAZyme evolutionary profiles and patterns, contributing to a more comprehensive understanding of microbial metabolic dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.561010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.561010" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.561010">
        <p class="paperTitle">VIPERA: Viral Intra-Patient Evolution Reporting and Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.561010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.561010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alvarez-Herrera, M.; Sevilla, J.; Ruiz-Rodriguez, P.; Vergara, A.; Vila, J.; Cano-Jimenez, P.; Gonzalez-Candelas, F.; Comas, I.; Coscolla, M.</p>
        <p class="info">Score: 19.4, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.561010' target='https://doi.org/10.1101/2023.10.24.561010'> 10.1101/2023.10.24.561010</a></p>
        <p class="abstract">Viral mutations within patients nurture the adaptive potential of SARS-CoV-2 during chronic infections, which are a potential source of variants of concern. However, there is no integrated framework for the evolutionary analysis of intra-patient SARS-CoV-2 serial samples. Herein we describe VIPERA (Viral Intra-Patient Evolution Reporting and Analysis), a new software that integrates the evaluation of the intra-patient ancestry of SARS-CoV-2 sequences with the analysis of evolutionary trajectories of serial sequences from the same viral infection. It has been validated using positive and negative control datasets and successfully applied to a novel case, contributing to easy and automatic analysis of intra-patient SARS-CoV-2 sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.561085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.561085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.561085">
        <p class="paperTitle">Assessing the limits of zero-shot foundation models in single-cell biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.561085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.561085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.</p>
        <p class="info">Score: 80.0, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.561085' target='https://doi.org/10.1101/2023.10.16.561085'> 10.1101/2023.10.16.561085</a></p>
        <p class="abstract">The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563028">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563028" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563028">
        <p class="paperTitle">VRP Assembler: haplotype-resolved de novo assembly of diploid and polyploid genomes using quantum computing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563028" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563028" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Huang, J.-H.; Sun, Y.; Zhang, Y.; Li, Y.; Xu, X.</p>
        <p class="info">Score: 17.6, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563028' target='https://doi.org/10.1101/2023.10.19.563028'> 10.1101/2023.10.19.563028</a></p>
        <p class="abstract">Precision medicines emphasis on individual genetic variants highlights the importance of haplotype-resolved assembly, a computational challenge in bioinformatics given its combinatorial nature. While classical algorithms have made strides in addressing this issue, the potential of quantum computing remains largely untapped. Here, we present the VRP assembler: a novel approach that transforms this task into a vehicle routing problem, a combinatorial optimization problem solvable on a quantum computer. We demonstrate its potential and feasibility through a proof of concept on short synthetic diploid and triploid genomes using a D-Wave quantum annealer. To tackle larger-scale assembly problems, we integrate the VRP assembler with Googles OR-Tools, achieving a haplotype-resolved assembly across the human major histocompatibility complex (MHC) region. Our results show encouraging performance compared to hifiasm with the phasing accuracy first approaching the theoretical limit, underscoring the promising future of quantum computing in bioinformatics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562625">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562625" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562625">
        <p class="paperTitle">A deep learning model embedded framework to distinguish DNA and RNA mutations directly from RNA-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562625" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562625" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Z.-C.; Gao, B.-Q.; Nan, F.; Ma, X.-K.; Yang, L.</p>
        <p class="info">Score: 28.2, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562625' target='https://doi.org/10.1101/2023.10.17.562625'> 10.1101/2023.10.17.562625</a></p>
        <p class="abstract">We develop a stepwise computational framework, called DEMINING, to directly detect expressed DNA and RNA mutations in RNA deep sequencing data. DEMINING incorporates a deep learning model named DeepDDR, which facilitates the separation of expressed DNA mutations from RNA mutations after RNA-seq read mapping and pileup. When applied in RNA-seq of acute myeloid leukemia patients, DEMINING uncovered previously-underappreciated DNA and RNA mutations, some associated with the upregulated expression of host genes or the production of neoantigens. Finally, we demonstrate that DEMINING could precisely classify DNA and RNA mutations in RNA-seq data from non-primate species through the utilization of transfer learning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563472">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563472" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563472">
        <p class="paperTitle">PyTME (Python Template Matching Engine): A fast, flexible, and multi-purpose template matching library for cryogenic electron microscopy data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563472" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563472" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maurer, V. J.; Siggel, M.; Kosinski, J.</p>
        <p class="info">Score: 15.5, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563472' target='https://doi.org/10.1101/2023.10.23.563472'> 10.1101/2023.10.23.563472</a></p>
        <p class="abstract">Cryogenic electron microscopy (cryo-EM) is a key method in structural and cell biology. Analysis of cryo-EM images requires interpretation of noisy, low-resolution densities which relies on identifying the most probable orientation of macromolecules in a target using template matching. Many method-specific template matching software exist for single-particle cryo-EM, cryo-electron tomography (cryo-ET), or fitting atomic structures into averaged 3D maps of macromolecules. Here, we report the Python Template Matching Engine (pyTME), a software engine that consolidates method-specific template matching problems. The underlying library provides highly efficient template-matching implementation and abstract data structures for storing and manipulating input and output data. It scales favorable to large datasets, both with multiple CPUs and GPUs, compared to existing software enabling template matching of even unbinned cryo-ET data in hours, which was previously nearly impossible due to technical restraints. Any hardware-specific optimization needed for dealing with large data is automatically performed to increase ease of use and minimize user intervention. The efficiency and simplicity of pyTME will enable high throughput mining of a variety of cryo-EM and ET datasets in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556814">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556814" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556814">
        <p class="paperTitle">Automated Bioinformatics Analysis via AutoBA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556814" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556814" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, J.; Zhang, B.; Chen, X.; Li, H.; Xu, X.; Chen, S.; Gao, X.</p>
        <p class="info">Score: 26.0, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556814' target='https://doi.org/10.1101/2023.09.08.556814'> 10.1101/2023.09.08.556814</a></p>
        <p class="abstract">With the fast-growing and evolving omics data, the demand for streamlined and adaptable tools to handle the bioinformatics analysis continues to grow. In response to this need, we introduce Automated Bioinformatics Analysis (AutoBA), an autonomous AI agent designed explicitly for regular omics data analysis based on large language models. AutoBA simplifies the analytical process by requiring minimal user input while delivering detailed step-by-step plans for various bioinformatics tasks. Through rigorous validation by expert bioinformaticians, AutoBAs robustness and adaptability are affirmed across a diverse range of omics analysis cases, including whole genome/exome sequencing (WGS/WES), chromatin immunoprecipitation assays with sequencing (ChIP-seq), RNA sequencing (RNA-seq), single-cell RNA-seq, spatial transcriptomics and so on. AutoBAs unique capacity to self-design analysis processes based on input data variations further underscores its versatility. Compared with online bioinformatic services, AutoBA offers five LLM backends, with options for both online and local usage, prioritizing data security and user privacy. Moreover, different from the predefined pipeline, AutoBA has adaptability in sync with emerging bioinformatics tools. Overall, AutoBA represents an advanced and convenient tool, offering robustness and adaptability for conventional bioinformatics analysis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
