<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics
Authors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.
Score: 40.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572483
Large Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics
Authors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.
Score: 40.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572483
Large Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics
Authors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.
Score: 40.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572483
Large Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics\nAuthors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.\nScore: 40.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572483\nLarge Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks.",
  "keywords": [
    
  ],
  "articleBody": " BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics\nAuthors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.\nScore: 40.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572483\nLarge Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks. Through BioLLMBench, we conducted a thorough evaluation of 2,160 experimental runs of the three most widely used models, GPT-4, Bard and LLaMA, focusing on 36 distinct tasks within the field of bioinformatics. The tasks come from six key areas of emphasis within bioinformatics that directly relate to the daily challenges and tasks faced by individuals within the field. These areas are domain expertise, mathematical problem-solving, coding proficiency, data visualization, summarizing research papers, and developing machine learning models. The tasks also span across varying levels of complexity, ranging from fundamental concepts to expert-level challenges. Each key area was evaluated using seven specifically designed task metrics, which were then used to conduct an overall evaluation of the LLMs response. To enhance our understanding of model responses under varying conditions, we implemented a Contextual Response Variability Analysis. Our results reveal a diverse spectrum of model performance, with GPT-4 leading in all tasks except mathematical problem solving. GPT4 was able to achieve an overall proficiency score of 91.3% in domain knowledge tasks, while Bard excelled in mathematical problem-solving with a 97.5% success rate. While GPT-4 outperformed in machine learning model development tasks with an average accuracy of 65.32%, both Bard and LLaMA were unable to generate executable end-to-end code. All models faced considerable challenges in research paper summarization, with none of them exceeding a 40% score in our evaluation using the Recall-Oriented Understudy for Gisting Evaluation (ROUGE) score, highlighting a significant area for future improvement. We observed an increase in model performance variance when using a new chatting window compared to using the same chat, although the average scores between the two contextual environments remained similar. Lastly, we discuss various limitations of these models and acknowledge the risks associated with their potential misuse.\nPredicting the Structural Impact of Human Alternative Splicing\nAuthors: Song, Y.; ZHANG, C.; Omenn, G. S.; O'Meara, M. J.; Welch, J. D.\nScore: 30.3, Published: 2023-12-24 DOI: 10.1101/2023.12.21.572928\nProtein structure prediction with neural networks is a powerful new method for linking protein sequence, structure, and function, but structures have generally been predicted for only a single isoform of each gene, neglecting splice variants. To investigate the structural implications of alternative splicing, we used AlphaFold2 to predict the structures of more than 11,000 human isoforms. We employed multiple metrics to identify splicing-induced structural alterations, including template matching score, secondary structure composition, surface charge distribution, radius of gyration, accessibility of post-translational modification sites, and structure-based function prediction. We identified examples of how alternative splicing induced clear changes in each of these properties. Structural similarity between isoforms largely correlated with degree of sequence identity, but we identified a subset of isoforms with low structural similarity despite high sequence similarity. Exon skipping and alternative last exons tended to increase the surface charge and radius of gyration. Splicing also buried or exposed numerous post-translational modification sites, most notably among the isoforms of BAX. Functional prediction nominated numerous functional differences among isoforms of the same gene, with loss of function compared to the reference predominating. Finally, we used single-cell RNA-seq data from the Tabula Sapiens to determine the cell types in which each structure is expressed. Our work represents an important resource for studying the structure and function of splice isoforms across the cell types of the human body.\nA long-context language model for the generation of bacteriophage genomes\nAuthors: Shao, B.\nScore: 32.6, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572218\nGenerative pre-trained transformers (GPTs) have revolutionized the field of natural language processing. Inspired by this success, we develop a long-context generative model for genomes. Our multiscale transformer model was pre-trained on unannotated bacteriophage genomes with byte-level tokenization. It generates de novo sequences up to 96K with functional genomic structure, including regulatory elements and novel proteins with phage-related functions. Our work paves the way for the de novo design of the whole genome.\nFURNA: a database for function annotations of RNA structures\nAuthors: Zhang, C.; Freddolino, P. L.\nScore: 30.1, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572314\nDespite the increasing number of 3D RNA structures in the Protein Data Bank, the majority of experimental RNA structures lack thorough functional annotations. As the significance of the functional roles played by non-coding RNAs becomes increasingly apparent, comprehensive annotation of RNA function is becoming a pressing concern. In response to this need, we have developed FURNA (Functions of RNAs), the first database for experimental RNA structures that aims to provide a comprehensive repository of high-quality functional annotations. These include Gene Ontology terms, Enzyme Commission numbers, ligand binding sites, RNA families, protein binding motifs, and cross-references to related databases. FURNA is available at https://seq2fun.dcmb.med.umich.edu/furna/ to enable quick discovery of RNA functions from their structures and sequences.\nInvestigating the impact of whole genome duplication on transposable element evolution in ray-finned fishes\nAuthors: Mallik, R.; Wcisel, D. J.; Yoder, J. A.; Near, T. J.; Dornburg, A.\nScore: 27.6, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572151\nTransposable elements (TEs) can make up more than 50% of any given vertebrate's genome, with substantial variability in TE composition among lineages. TE variation is often linked to changes in gene regulation, genome size, and speciation. However, the role that genome duplication events have played in generating abrupt shifts in the composition of the mobilome over macroevolutionary timescales remains unclear. We investigated the degree to which the teleost genome duplication (TGD) shaped the diversification trajectory of the ray-finned fish mobilome. We integrate a new high coverage genome of Polypterus bichir with data from over 100 publicly available actinopterygian genomes to assess the macroevolutionary implications of genome duplication events on TE evolution. Our results provide no evidence for a substantial shift in mobilome composition following the TGD event. Instead, the diversity of the actinopterygian mobilome appears to have been shaped by a history of lineage specific shifts in composition that are not correlated with commonly evoked drivers of diversification such as body size, water column usage, or latitude. Collectively, these results provide a new perspective on the early diversification of the actinopterygian mobilome and suggest that historic ploidy events may not necessarily catalyze bursts of TE diversification and innovation.\nSOAPy: a Python package to dissect spatial architecture, dynamics and communication\nAuthors: Wang, H.; Li, J.; Jing, S.; Lin, P.; Li, Y.; Zhang, H.; Chen, Y.; Wang, Z.; Li, H.\nScore: 26.3, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572725\nAdvances in spatial omics technologies have brought opportunities to dissect tissue microenvironment, while also posing more requirements and challenges for computational methods. Here we developed a package SOAPy to systematically dissect spatial architecture, dynamics and communication from spatial omics data. Specifically, it provides analysis methods for multiple spatial-related tasks, including spatial domain, spatial expression tendency, spatiotemporal expression pattern, cellular co-localization, multi-cellular niches, and ligand-receptor-mediated and spatial-constrained cell communication. Applying SOAPy on different spatial omics technologies and diverse biological fields has demonstrated its power on elucidation of biological questions about tumors, embryonic development, and normal physiological structures. Overall, SOAPy is a universal tool for spatial omics analysis, providing a foundation for continued investigation of the microenvironment.\nPredicting stop codon reassignment improves functional annotation of bacteriophages\nAuthors: Cook, R.; Telatin, A.; Bouras, G.; Camargo, A. P.; Larralde, M.; Edwards, R. A.; Adriaenssens, E. M.\nScore: 28.0, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572299\nThe majority of bacteriophage diversity remains uncharacterised, and new intriguing mechanisms of their biology are being continually described. Members of some phage lineages, such as the Crassvirales, repurpose stop codons to encode an amino acid by using alternate genetic codes. Here, we investigated the prevalence of stop codon reassignment in phage genomes and subsequent impacts on functional annotation. We predicted 76 genomes within INPHARED and 712 vOTUs from the Unified Human Gut Virome catalogue (UHGV) that repurpose a stop codon to encode an amino acid. We re-annotated these sequences with modified versions of Pharokka and Prokka, called Pharokka-gv and Prokkagv, to automatically predict stop codon reassignment prior to annotation. Both tools significantly improved the quality of annotations, with Pharokka-gv performing best. For sequences predicted to repurpose TAG to glutamine (translation table 15), Pharokka-gv increased the median gene length (median of per genome medians) from 287 to 481 bp for UHGV sequences (67.8% increase) and from 318 to 550 bp for INPHARED sequences (72.9% increase). The re-annotation increased mean coding density from 66.8% to 90.0%, and from 69.0% to 89.8% for UHGV and INPHARED sequences. Furthermore, the proportion of genes that could be assigned functional annotation increased, including an increase in the number of major capsid proteins that could be identified. We propose that automatic prediction of stop codon reassignment before annotation is beneficial to downstream viral genomic and metagenomic analyses.\nDifferential detection workflows for multi-sample single-cell RNA-seq data\nAuthors: Gilis, J.; Perin, L.; Malfait, M.; Van den Berge, K.; Assefa Takele, A.; Verbist, B.; Risso, D.; Clement, L.\nScore: 19.0, Published: 2023-12-19 DOI: 10.1101/2023.12.17.572043\nIn single-cell transcriptomics, differential gene expression (DE) analyses typically focus on testing differences in the average expression of genes between cell types or conditions of interest. Single-cell transcriptomics, however, also has the promise to prioritise genes for which the expression differ in other aspects of the distribution. Here we develop a workflow for assessing differential detection (DD), which tests for differences in the average fraction of samples or cells in which a gene is detected. After benchmarking eight different DD data analysis strategies, we provide a unified workflow for jointly assessing DE and DD. Using simulations and two case studies, we show that DE and DD analysis provide complementary information, both in terms of the individual genes they report and in the functional interpretation of those genes.\nSingle-cell and spatial multiomic inference of gene regulatory networks using SCRIPro\nAuthors: Chang, Z.; Xu, Y.; Dong, X.; Gao, Y.; Wang, C.\nScore: 16.0, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572934\nThe accurate reconstruction of gene regulation networks (GRNs) from sparse and noisy single-cell or spatial multi-omics data remains a challenge. Here, we present SCRIPro, a comprehensive computational framework that robustly infers GRNs for both single-cell and spatial multi-omics data. SCRIPro first addresses sample sparseness by a density clustering approach. SCRIPro assesses transcriptional regulator (TR) importance through chromatin reconstruction and in silico deletion, referencing 1,292 human and 994 mouse TRs. It combines TR-target importance scores with expression levels for precise GRN reconstruction. Finally, we benchmarked SCRIPro on diverse datasets, it outperforms existing motif-based methods and accurately reconstructs cell type-specific, stage-specific, and region-specific GRNs.\nFlowAtlas.jl: an interactive tool bridging FlowJo with computational tools in Julia\nAuthors: Coppard, V.; Szep, G.; Georgieva, Z.; Howlett, S. K.; Jarvis, L. B.; Rainbow, D. B.; Suchanek, O.; Needham, E. J.; Mousa, H. S.; Menon, D. K.; Feyertag, F.; Mahbubani, K. T.; Saeb-Parsy, K.; Jones, J. L.\nScore: 13.9, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572741\nAs the dimensionality, throughput, and complexity of cytometry data increases, so does the demand for user-friendly, interactive analysis tools that leverage high-performance machine learning frameworks. Here we introduce FlowAtlas.jl: an interactive web application that bridges the user-friendly environment of FlowJo and computational tools in Julia developed by the scientific machine learning community. We demonstrate the capabilities of FlowAtlas using a novel human multi-tissue, multi-donor immune cell dataset, highlighting key immunological findings.\n",
  "wordCount" : "1915",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572483">
        <p class="paperTitle">BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.</p>
        <p class="info">Score: 40.0, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572483' target='https://doi.org/10.1101/2023.12.19.572483'> 10.1101/2023.12.19.572483</a></p>
        <p class="abstract">Large Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks. Through BioLLMBench, we conducted a thorough evaluation of 2,160 experimental runs of the three most widely used models, GPT-4, Bard and LLaMA, focusing on 36 distinct tasks within the field of bioinformatics. The tasks come from six key areas of emphasis within bioinformatics that directly relate to the daily challenges and tasks faced by individuals within the field. These areas are domain expertise, mathematical problem-solving, coding proficiency, data visualization, summarizing research papers, and developing machine learning models. The tasks also span across varying levels of complexity, ranging from fundamental concepts to expert-level challenges. Each key area was evaluated using seven specifically designed task metrics, which were then used to conduct an overall evaluation of the LLMs response. To enhance our understanding of model responses under varying conditions, we implemented a Contextual Response Variability Analysis. Our results reveal a diverse spectrum of model performance, with GPT-4 leading in all tasks except mathematical problem solving. GPT4 was able to achieve an overall proficiency score of 91.3% in domain knowledge tasks, while Bard excelled in mathematical problem-solving with a 97.5% success rate. While GPT-4 outperformed in machine learning model development tasks with an average accuracy of 65.32%, both Bard and LLaMA were unable to generate executable end-to-end code. All models faced considerable challenges in research paper summarization, with none of them exceeding a 40% score in our evaluation using the Recall-Oriented Understudy for Gisting Evaluation (ROUGE) score, highlighting a significant area for future improvement. We observed an increase in model performance variance when using a new chatting window compared to using the same chat, although the average scores between the two contextual environments remained similar. Lastly, we discuss various limitations of these models and acknowledge the risks associated with their potential misuse.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572928">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572928" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572928">
        <p class="paperTitle">Predicting the Structural Impact of Human Alternative Splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572928" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572928" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Song, Y.; ZHANG, C.; Omenn, G. S.; O&#39;Meara, M. J.; Welch, J. D.</p>
        <p class="info">Score: 30.3, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572928' target='https://doi.org/10.1101/2023.12.21.572928'> 10.1101/2023.12.21.572928</a></p>
        <p class="abstract">Protein structure prediction with neural networks is a powerful new method for linking protein sequence, structure, and function, but structures have generally been predicted for only a single isoform of each gene, neglecting splice variants. To investigate the structural implications of alternative splicing, we used AlphaFold2 to predict the structures of more than 11,000 human isoforms. We employed multiple metrics to identify splicing-induced structural alterations, including template matching score, secondary structure composition, surface charge distribution, radius of gyration, accessibility of post-translational modification sites, and structure-based function prediction. We identified examples of how alternative splicing induced clear changes in each of these properties. Structural similarity between isoforms largely correlated with degree of sequence identity, but we identified a subset of isoforms with low structural similarity despite high sequence similarity. Exon skipping and alternative last exons tended to increase the surface charge and radius of gyration. Splicing also buried or exposed numerous post-translational modification sites, most notably among the isoforms of BAX. Functional prediction nominated numerous functional differences among isoforms of the same gene, with loss of function compared to the reference predominating. Finally, we used single-cell RNA-seq data from the Tabula Sapiens to determine the cell types in which each structure is expressed. Our work represents an important resource for studying the structure and function of splice isoforms across the cell types of the human body.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572218">
        <p class="paperTitle">A long-context language model for the generation of bacteriophage genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shao, B.</p>
        <p class="info">Score: 32.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572218' target='https://doi.org/10.1101/2023.12.18.572218'> 10.1101/2023.12.18.572218</a></p>
        <p class="abstract">Generative pre-trained transformers (GPTs) have revolutionized the field of natural language processing. Inspired by this success, we develop a long-context generative model for genomes. Our multiscale transformer model was pre-trained on unannotated bacteriophage genomes with byte-level tokenization. It generates de novo sequences up to 96K with functional genomic structure, including regulatory elements and novel proteins with phage-related functions. Our work paves the way for the de novo design of the whole genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572314" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572314">
        <p class="paperTitle">FURNA: a database for function annotations of RNA structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, C.; Freddolino, P. L.</p>
        <p class="info">Score: 30.1, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572314' target='https://doi.org/10.1101/2023.12.19.572314'> 10.1101/2023.12.19.572314</a></p>
        <p class="abstract">Despite the increasing number of 3D RNA structures in the Protein Data Bank, the majority of experimental RNA structures lack thorough functional annotations. As the significance of the functional roles played by non-coding RNAs becomes increasingly apparent, comprehensive annotation of RNA function is becoming a pressing concern. In response to this need, we have developed FURNA (Functions of RNAs), the first database for experimental RNA structures that aims to provide a comprehensive repository of high-quality functional annotations. These include Gene Ontology terms, Enzyme Commission numbers, ligand binding sites, RNA families, protein binding motifs, and cross-references to related databases. FURNA is available at https://seq2fun.dcmb.med.umich.edu/furna/ to enable quick discovery of RNA functions from their structures and sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572151">
        <p class="paperTitle">Investigating the impact of whole genome duplication on transposable element evolution in ray-finned fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mallik, R.; Wcisel, D. J.; Yoder, J. A.; Near, T. J.; Dornburg, A.</p>
        <p class="info">Score: 27.6, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572151' target='https://doi.org/10.1101/2023.12.22.572151'> 10.1101/2023.12.22.572151</a></p>
        <p class="abstract">Transposable elements (TEs) can make up more than 50% of any given vertebrate&#39;s genome, with substantial variability in TE composition among lineages. TE variation is often linked to changes in gene regulation, genome size, and speciation. However, the role that genome duplication events have played in generating abrupt shifts in the composition of the mobilome over macroevolutionary timescales remains unclear. We investigated the degree to which the teleost genome duplication (TGD) shaped the diversification trajectory of the ray-finned fish mobilome. We integrate a new high coverage genome of Polypterus bichir with data from over 100 publicly available actinopterygian genomes to assess the macroevolutionary implications of genome duplication events on TE evolution. Our results provide no evidence for a substantial shift in mobilome composition following the TGD event. Instead, the diversity of the actinopterygian mobilome appears to have been shaped by a history of lineage specific shifts in composition that are not correlated with commonly evoked drivers of diversification such as body size, water column usage, or latitude. Collectively, these results provide a new perspective on the early diversification of the actinopterygian mobilome and suggest that historic ploidy events may not necessarily catalyze bursts of TE diversification and innovation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572725">
        <p class="paperTitle">SOAPy: a Python package to dissect spatial architecture, dynamics and communication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Li, J.; Jing, S.; Lin, P.; Li, Y.; Zhang, H.; Chen, Y.; Wang, Z.; Li, H.</p>
        <p class="info">Score: 26.3, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572725' target='https://doi.org/10.1101/2023.12.21.572725'> 10.1101/2023.12.21.572725</a></p>
        <p class="abstract">Advances in spatial omics technologies have brought opportunities to dissect tissue microenvironment, while also posing more requirements and challenges for computational methods. Here we developed a package SOAPy to systematically dissect spatial architecture, dynamics and communication from spatial omics data. Specifically, it provides analysis methods for multiple spatial-related tasks, including spatial domain, spatial expression tendency, spatiotemporal expression pattern, cellular co-localization, multi-cellular niches, and ligand-receptor-mediated and spatial-constrained cell communication. Applying SOAPy on different spatial omics technologies and diverse biological fields has demonstrated its power on elucidation of biological questions about tumors, embryonic development, and normal physiological structures. Overall, SOAPy is a universal tool for spatial omics analysis, providing a foundation for continued investigation of the microenvironment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572299" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572299">
        <p class="paperTitle">Predicting stop codon reassignment improves functional annotation of bacteriophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, R.; Telatin, A.; Bouras, G.; Camargo, A. P.; Larralde, M.; Edwards, R. A.; Adriaenssens, E. M.</p>
        <p class="info">Score: 28.0, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572299' target='https://doi.org/10.1101/2023.12.19.572299'> 10.1101/2023.12.19.572299</a></p>
        <p class="abstract">The majority of bacteriophage diversity remains uncharacterised, and new intriguing mechanisms of their biology are being continually described. Members of some phage lineages, such as the Crassvirales, repurpose stop codons to encode an amino acid by using alternate genetic codes. Here, we investigated the prevalence of stop codon reassignment in phage genomes and subsequent impacts on functional annotation. We predicted 76 genomes within INPHARED and 712 vOTUs from the Unified Human Gut Virome catalogue (UHGV) that repurpose a stop codon to encode an amino acid. We re-annotated these sequences with modified versions of Pharokka and Prokka, called Pharokka-gv and Prokkagv, to automatically predict stop codon reassignment prior to annotation. Both tools significantly improved the quality of annotations, with Pharokka-gv performing best. For sequences predicted to repurpose TAG to glutamine (translation table 15), Pharokka-gv increased the median gene length (median of per genome medians) from 287 to 481 bp for UHGV sequences (67.8% increase) and from 318 to 550 bp for INPHARED sequences (72.9% increase). The re-annotation increased mean coding density from 66.8% to 90.0%, and from 69.0% to 89.8% for UHGV and INPHARED sequences. Furthermore, the proportion of genes that could be assigned functional annotation increased, including an increase in the number of major capsid proteins that could be identified. We propose that automatic prediction of stop codon reassignment before annotation is beneficial to downstream viral genomic and metagenomic analyses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.17.572043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.17.572043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.17.572043">
        <p class="paperTitle">Differential detection workflows for multi-sample single-cell RNA-seq data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.17.572043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.17.572043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilis, J.; Perin, L.; Malfait, M.; Van den Berge, K.; Assefa Takele, A.; Verbist, B.; Risso, D.; Clement, L.</p>
        <p class="info">Score: 19.0, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.17.572043' target='https://doi.org/10.1101/2023.12.17.572043'> 10.1101/2023.12.17.572043</a></p>
        <p class="abstract">In single-cell transcriptomics, differential gene expression (DE) analyses typically focus on testing differences in the average expression of genes between cell types or conditions of interest. Single-cell transcriptomics, however, also has the promise to prioritise genes for which the expression differ in other aspects of the distribution. Here we develop a workflow for assessing differential detection (DD), which tests for differences in the average fraction of samples or cells in which a gene is detected. After benchmarking eight different DD data analysis strategies, we provide a unified workflow for jointly assessing DE and DD. Using simulations and two case studies, we show that DE and DD analysis provide complementary information, both in terms of the individual genes they report and in the functional interpretation of those genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572934">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572934" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572934">
        <p class="paperTitle">Single-cell and spatial multiomic inference of gene regulatory networks using SCRIPro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572934" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572934" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, Z.; Xu, Y.; Dong, X.; Gao, Y.; Wang, C.</p>
        <p class="info">Score: 16.0, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572934' target='https://doi.org/10.1101/2023.12.21.572934'> 10.1101/2023.12.21.572934</a></p>
        <p class="abstract">The accurate reconstruction of gene regulation networks (GRNs) from sparse and noisy single-cell or spatial multi-omics data remains a challenge. Here, we present SCRIPro, a comprehensive computational framework that robustly infers GRNs for both single-cell and spatial multi-omics data. SCRIPro first addresses sample sparseness by a density clustering approach. SCRIPro assesses transcriptional regulator (TR) importance through chromatin reconstruction and in silico deletion, referencing 1,292 human and 994 mouse TRs. It combines TR-target importance scores with expression levels for precise GRN reconstruction. Finally, we benchmarked SCRIPro on diverse datasets, it outperforms existing motif-based methods and accurately reconstructs cell type-specific, stage-specific, and region-specific GRNs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572741" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572741">
        <p class="paperTitle">FlowAtlas.jl: an interactive tool bridging FlowJo with computational tools in Julia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coppard, V.; Szep, G.; Georgieva, Z.; Howlett, S. K.; Jarvis, L. B.; Rainbow, D. B.; Suchanek, O.; Needham, E. J.; Mousa, H. S.; Menon, D. K.; Feyertag, F.; Mahbubani, K. T.; Saeb-Parsy, K.; Jones, J. L.</p>
        <p class="info">Score: 13.9, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572741' target='https://doi.org/10.1101/2023.12.21.572741'> 10.1101/2023.12.21.572741</a></p>
        <p class="abstract">As the dimensionality, throughput, and complexity of cytometry data increases, so does the demand for user-friendly, interactive analysis tools that leverage high-performance machine learning frameworks. Here we introduce FlowAtlas.jl: an interactive web application that bridges the user-friendly environment of FlowJo and computational tools in Julia developed by the scientific machine learning community. We demonstrate the capabilities of FlowAtlas using a novel human multi-tissue, multi-donor immune cell dataset, highlighting key immunological findings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
