<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation
Authors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.
Score: 46.6, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573793
BackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation
Authors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.
Score: 46.6, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573793
BackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation
Authors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.
Score: 46.6, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573793
BackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation\nAuthors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.\nScore: 46.6, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573793\nBackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples.",
  "keywords": [
    
  ],
  "articleBody": " PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation\nAuthors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.\nScore: 46.6, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573793\nBackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples. We initially used conventional PCA as a tool for critical quality control of batch and trend effects in multi-omic profiling data produced by The Cancer Genome Atlas (TCGA) project of the NCI. We found, however, that conventional PCA visualizations were often hard to interpret when inter-batch differences were moderate in comparison with intra-batch differences; it was also difficult to quantify batch effects objectively. We, therefore, sought enhancements to make the method more informative in those and analogous settings. ResultsWe have developed algorithms and a toolbox of enhancements to conventional PCA that improve the detection, diagnosis, and quantitation of differences between or among groups, e.g., groups of molecularly profiled biological samples. The enhancements include (i) computed group centroids; (ii) sample-dispersion rays; (iii) differential coloring of centroids, rays, and sample data points; (iii) trend trajectories; and (iv) a novel separation index (DSC) for quantitation of differences among groups. ConclusionsPCA-Plus has been our most useful single tool for analyzing, visualizing, and quantitating batch effects, trend effects, and class differences in molecular profiling data of many types: mRNA expression, microRNA expression, DNA methylation, and DNA copy number. An early version of PCA-Plus has been used as the central graphical visualization in our MBatch package for near-real-time surveillance of data for analysis working groups in more than 70 TCGA, PanCancer Atlas, PanCancer Analysis of Whole Genomes, and Genome Data Analysis Network projects of the NCI. The algorithms and software are generic, hence applicable more generally to other types of multivariate data as well. PCA-Plus is freely available in a down-loadable R package at our MBatch website.\nSequence clustering confounds AlphaFold2\nAuthors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.\nScore: 22.8, Published: 2024-01-09 DOI: 10.1101/2024.01.05.574434\nThough typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al. steered AF2 to predict alternative structures of a few metamorphic proteins using a method they call AF-cluster5. However, their Paper lacks some essential controls needed to assess AF-clusters reliability. We find that these controls show AF-cluster to be a poor predictor of metamorphic proteins. First, closer examination of the Papers results reveals that random sequence sampling outperforms sequence clustering, challenging the claim that AF-cluster works by \"deconvolving conflicting sets of couplings.\" Further, we observe that AF-cluster mistakes some single-folding KaiB homologs for fold switchers, a critical flaw bound to mislead users. Finally, proper error analysis reveals that AF-cluster predicts many correct structures with low confidence and some experimentally unobserved conformations with confidences similar to experimentally observed ones. For these reasons, we suggest using ColabFold6-based random sequence sampling7-augmented by other predictive approaches-as a more accurate and less computationally intense alternative to AF-cluster.\nBeyond benchmarking: towards predictive models of dataset-specific single-cell RNA-seq pipeline performance\nAuthors: Fang, C.; Selega, A.; Campbell, K. R.\nScore: 15.8, Published: 2024-01-02 DOI: 10.1101/2024.01.02.572650\nThe advent of single-cell RNA-sequencing (scRNA-seq) has driven significant computational methods development for all steps in the scRNA-seq data analysis pipeline, including filtering, normalization, and clustering. The large number of methods and their resulting parameter combinations has created a combinatorial set of possible pipelines to analyze scRNA-seq data, which leads to the obvious question: which is best? Several benchmarking studies have sought to compare methods to answer this, but frequently find variable performance depending on dataset and pipeline characteristics. Alternatively, the large number of publicly available scRNA-seq datasets along with advances in supervised machine learning raise a tantalizing possibility: could the optimal pipeline be predicted for a given dataset? Here we begin to answer this question by applying 288 scRNA-seq analysis pipelines to 86 datasets and quantifying pipeline success via a range of measures evaluating cluster purity and biological plausibility. We build supervised machine learning models to predict pipeline success given a range of dataset and pipeline characteristics. We find both that prediction performance is significantly better than random and that in many cases pipelines predicted to perform well provide clustering outputs similar to expert-annotated cell type labels. Finally, we identify characteristics of scRNA-seq datasets that correlate with strong prediction performance that could guide when such prediction models may be useful.\nADMET-AI: A machine learning ADMET platform for evaluation of large-scale chemical libraries\nAuthors: Swanson, K.; Walther, P.; Leitz, J.; Mukherjee, S.; Wu, J. C.; Shivnaraine, R. V.; Zou, J.\nScore: 20.3, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573531\nSummaryThe emergence of large chemical repositories and combinatorial chemical spaces, coupled with high-throughput docking and generative AI, have greatly expanded the chemical diversity of small molecules for drug discovery. Selecting compounds for experimental validation requires filtering these molecules based on favourable druglike properties, such as Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET). We developed ADMET-AI, a machine learning platform that provides fast and accurate ADMET predictions both as a website and as a Python package. ADMET-AI has the highest average rank on the TDC ADMET Benchmark Group leaderboard, and it is currently the fastest web-based ADMET predictor, with a 45% reduction in time compared to the next fastest ADMET web server. ADMET-AI can also be run locally with predictions for one million molecules taking just 3.1 hours. Availability and ImplementationThe ADMET-AI platform is freely available both as a web server at admet.ai.greenstonebio.com and as an open-source Python package for local batch prediction at github.com/swansonk14/admet_ai (also archived on Zenodo at doi.org/10.5281/zenodo.10372930). All data and models are archived on Zenodo at doi.org/10.5281/zenodo.10372418.\nNovel metagenomics analysis suggests a Vibrio species is associated with stony coral tissue loss disease\nAuthors: Heinz, J. M.; Sommer, M.; Rosales, S. M.; Lu, J.; Huebner, L. K.; Salzberg, S. L.\nScore: 9.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573916\nStony coral tissue loss disease (SCTLD) has devastated coral reefs off the coast of Florida and continues to spread throughout the Caribbean. Although a number of bacterial taxa have consistently been associated with SCTLD, no pathogen has been definitively implicated in the etiology of SCTLD. Previous studies have predominantly focused on the prokaryotic community through 16S rRNA sequencing of healthy and infected tissues. Here, we provide a different analytical approach by applying a bioinformatics pipeline to publicly available whole genome sequencing samples of SCTLD lesions and healthy tissues from four stony coral species. To compensate for the lack of coral reference genomes, we used data from apparently healthy coral samples to approximate a host genome and healthy microbiome reference. The healthy reference reads were then used to filter the reads from diseased lesion tissue samples, and the remaining data were taxonomically classified at the DNA and protein levels. For DNA classifications, we used a pathogen identification protocol originally designed to identify pathogens in human tissue samples, and for protein classifications, we used a fast protein sequence aligner. Although these data were previously analyzed, our approach revealed unique patterns that were not identified in the previous work. We found a relatively high abundance of the Vibrio genus across diseased samples as well as a number of enriched Vibrio phages that further support the presence of this genus in diseased samples, suggesting that a member of the Vibrio genus may be involved in the visual lesion formation stage of SCTLD. Article SummaryStudies of stony coral tissue loss disease (SCTLD), a devastating coral disease, have primary used 16S rRNA sequencing approaches to identify putative pathogens. This study applied human tissue pathogen identification protocols to SCTLD whole genome DNA samples. A k-mer based filtering method for diseased samples was utilized to compensate for the lack of host reference genomes. DNA and protein level classifications revealed a relatively abundant member of the Vibrio genus associated with SCTLD across four stony coral species.\nProtein superfolds are characterised as frustration-free topologies: A case study of pure parallel beta-sheet topologies\nAuthors: Murata, H.; Toko, K.; Chikenji, G.\nScore: 8.5, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574326\nA protein superfold is a type of protein fold that is observed in at least three distinct, non-homologous protein families. Structural classification studies have revealed a limited number of prevalent superfolds alongside several infrequent occurring folds, and in /{beta} type superfolds, the C-terminal {beta}-strand tends to favor the edge of the {beta}-sheet, while the N-terminal {beta}-strand is often found in the middle. The reasons behind these observations, whether they are due to evolutionary sampling bias or physical interactions, remain unclear. This article offers a physics-based explanation for these observations, specifically for pure parallel {beta}-sheet topologies. Our investigation is grounded in three established structural rules that are based on physical interactions. We have identified \"frustration-free topologies\" which are topologies that can satisfy all three rules simultaneously. In contrast, topologies that cannot are termed \"frustrated topologies.\" Our findings reveal that frustration-free topologies represent only a fraction of all theoretically possible patterns, these topologies strongly favor positioning the C-terminal {beta}-strand at the edge of the {beta}-sheet and the N-terminal {beta}-strand in the middle, and there is significant overlap between frustration-free topologies and superfolds. We also used a lattice protein model to thoroughly investigate sequence-structure relationships. Our results show that frustration-free structures are highly designable, while frustrated structures are poorly designable. These findings suggest that superfolds are highly designable due to their lack of frustration, and the preference for positioning C-terminal {beta}-strands at the edge of the {beta}-sheet is a direct result of frustration-free topologies. These insights not only enhance our understanding of sequence-structure relationships but also have significant implications for de novo protein design. Author summaryA protein superfold is a protein fold that appears in at least three different non-homologous protein families. Superfolds are unique in their ability to accommodate multiple functions within a single fold, a feature not typically seen in other folds. Studies in structural classification have led to two notable observations: the existence of a limited number of common superfolds contrasted with a larger variety of less frequent folds, and a recurring pattern in /{beta} type superfolds where the C-terminal {beta}-strand often occupies the edge of the {beta}-sheet, while the N-terminal {beta}-strand is usually found in the middle. The origins of these patterns, whether they stem from evolutionary sampling bias or physical interaction mechanisms, remain unclear. This article provides a physics-oriented explanation for these observations, specifically concentrating on pure parallel {beta}-sheet topologies. The insights gained from this research are crucial in enhancing our understanding of the relationship between protein sequences and structures, and are expected to contribute significantly to the de novo design of new proteins.\nGenomics 2 Proteins portal: A resource and discovery tool for linking genetic screening outputs to protein sequences and structures\nAuthors: Kwon, S.; Safer, J.; Nguyen, D. T.; Hoksza, D.; May, P.; Arbesfeld, J.; Rubin, A. F.; Campbell, A. J.; Burgin, A.; Iqbal, S.\nScore: 8.3, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573913\nRecent advances in AI-based methods have revolutionized the field of structural biology. Concomitantly, high-throughput sequencing and functional genomics technologies have enabled the detection and generation of variants at an unprecedented scale. However, efficient tools and resources are needed to link these two disparate data types - to \"map\" variants onto protein structures, to better understand how the variation causes disease and thereby design therapeutics. Here we present the Genomics 2 Proteins Portal (G2P; g2p.broadinstitute.org/): a human proteome-wide resource that maps 19,996,443 genetic variants onto 42,413 protein sequences and 77,923 structures, with a comprehensive set of structural and functional features. Additionally, the G2P portal generalizes the capability of linking genomics to proteins beyond databases by allowing users to interactively upload protein residue-wise annotations (variants, scores, etc.) as well as the protein structure to establish the connection. The portal serves as an easy-to-use discovery tool for researchers and scientists to hypothesize the structure-function relationship between natural or synthetic variations and their molecular phenotype.\nCombinatorial prediction of therapeutic perturbations using causally-inspired neural networks\nAuthors: Gonzalez, G.; Herath, I.; Veselkov, K.; Bronstein, M.; Zitnik, M.\nScore: 7.9, Published: 2024-01-08 DOI: 10.1101/2024.01.03.573985\nAs an alternative to target-driven drug discovery, phenotype-driven approaches identify compounds that counteract the overall disease effects by analyzing phenotypic signatures. Our study introduces a novel approach to this field, aiming to expand the search space for new therapeutic agents. We introduce PDGrapher, a causally-inspired graph neural network model designed to predict arbitrary perturbagens - sets of therapeutic targets - capable of reversing disease effects. Unlike existing methods that learn responses to perturbations, PDGrapher solves the inverse problem, which is to infer the perturbagens necessary to achieve a specific response - i.e., directly predicting perturbagens by learning which perturbations elicit a desired response. Experiments across eight datasets of genetic and chemical perturbations show that PDGrapher successfully predicted effective perturbagens in up to 9% additional test samples and ranked therapeutic targets up to 35% higher than competing methods. A key innovation of PDGrapher is its direct prediction capability, which contrasts with the indirect, computationally intensive models traditionally used in phenotype-driven drug discovery that only predict changes in phenotypes due to perturbations. The direct approach enables PDGrapher to train up to 30 times faster, representing a significant leap in efficiency. Our results suggest that PDGrapher can advance phenotype-driven drug discovery, offering a fast and comprehensive approach to identifying therapeutically useful perturbations.\nFinaleMe: Predicting DNA methylation by the fragmentation patterns of plasma cell-free DNA\nAuthors: Liu, Y.; Reed, S. C.; Lo, C.; Choudhury, A. D.; Parsons, H. A.; Stover, D. G.; Ha, G.; Gydush, G.; Rhoades, J.; Rotem, D.; Freeman, S.; Katz, D.; Bandaru, R.; Zheng, H.; Fu, H.; Adalsteinsson, V. A.; Kellis, M.\nScore: 7.8, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573710\nAnalysis of DNA methylation in cell-free DNA (cfDNA) reveals clinically relevant biomarkers but requires specialized protocols and sufficient input material that limits its applicability. Millions of cfDNA samples have been profiled by genomic sequencing. To maximize the gene regulation information from the existing dataset, we developed FinaleMe, a non-homogeneous Hidden Markov Model (HMM), to predict DNA methylation of cfDNA and, therefore, tissues-of-origin directly from plasma whole-genome sequencing (WGS). We validated the performance with 80 pairs of deep and shallow-coverage WGS and whole-genome bisulfite sequencing (WGBS) data.\nTEMINET: A Co-Informative and Trustworthy Multi-Omics Integration Network for Diagnostic Prediction\nAuthors: Luo, H.; Liang, H.; Liu, H.; Fan, Z.; Wei, Y.; Yao, X.; Cong, S.\nScore: 7.3, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574118\nAdvancing the domain of biomedical investigation, integrated multi-omics data have shown exceptional performance in elucidating complex human diseases. However, as the variety of omics information expands, precisely perceiving the informativeness of intra- and inter-omics becomes challenging due to the intricate interrelations, thus posing significant obstacles in multi-omics data integration. To address this, we introduce a novel multi-omics integration approach, referred to as TEMINET. This approach enhances diagnostic prediction by leveraging an intra-omics co-informative representation method and a trustworthy learning strategy used to address inter-omics fusion. Considering the multifactorial nature of complex diseases, TEMINET utilizes intra-omics features to construct disease-specific networks, then applies graph attention networks and a multi-level framework to capture more collective informativeness than pairwise relations. To perceive the contribution of co-informative representations within intra-omics, we design a trustworthy learning strategy to identify the reliability of each omics in integration. To integrate inter-omics information, a combined beliefs fusion approach is deployed to harmonize the trustworthy representations of different omics types effectively. Our experiments across four different diseases using mRNA, methylation, and miRNA data demonstrate that TEMINET achieves advanced performance and robustness in classification tasks.\n",
  "wordCount" : "2611",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573793" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573793">
        <p class="paperTitle">PCA-Plus: Enhanced principal component analysis with illustrative applications to batch effects and their quantitation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, N.; Casasent, T. D.; Casasent, A. K.; Kumar, S. V.; Wakefield, C.; Broom, B. M.; Weinstein, J. N.; Akbani, R.</p>
        <p class="info">Score: 46.6, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573793' target='https://doi.org/10.1101/2024.01.02.573793'> 10.1101/2024.01.02.573793</a></p>
        <p class="abstract">BackgroundPrincipal component analysis (PCA), a standard approach to analysis and visualization of large datasets, is commonly used in biomedical research for detecting similarities and differences among groups of samples. We initially used conventional PCA as a tool for critical quality control of batch and trend effects in multi-omic profiling data produced by The Cancer Genome Atlas (TCGA) project of the NCI. We found, however, that conventional PCA visualizations were often hard to interpret when inter-batch differences were moderate in comparison with intra-batch differences; it was also difficult to quantify batch effects objectively. We, therefore, sought enhancements to make the method more informative in those and analogous settings.

ResultsWe have developed algorithms and a toolbox of enhancements to conventional PCA that improve the detection, diagnosis, and quantitation of differences between or among groups, e.g., groups of molecularly profiled biological samples. The enhancements include (i) computed group centroids; (ii) sample-dispersion rays; (iii) differential coloring of centroids, rays, and sample data points; (iii) trend trajectories; and (iv) a novel separation index (DSC) for quantitation of differences among groups.

ConclusionsPCA-Plus has been our most useful single tool for analyzing, visualizing, and quantitating batch effects, trend effects, and class differences in molecular profiling data of many types: mRNA expression, microRNA expression, DNA methylation, and DNA copy number. An early version of PCA-Plus has been used as the central graphical visualization in our MBatch package for near-real-time surveillance of data for analysis working groups in more than 70 TCGA, PanCancer Atlas, PanCancer Analysis of Whole Genomes, and Genome Data Analysis Network projects of the NCI. The algorithms and software are generic, hence applicable more generally to other types of multivariate data as well. PCA-Plus is freely available in a down-loadable R package at our MBatch website.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574434" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574434">
        <p class="paperTitle">Sequence clustering confounds AlphaFold2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schafer, J. W.; Chakravarty, D.; Chen, E. A.; Porter, L. L.</p>
        <p class="info">Score: 22.8, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574434' target='https://doi.org/10.1101/2024.01.05.574434'> 10.1101/2024.01.05.574434</a></p>
        <p class="abstract">Though typically associated with a single folded state, some globular proteins remodel their secondary and/or tertiary structures in response to cellular stimuli. AlphaFold21 (AF2) readily generates one dominant protein structure for these fold-switching (a.k.a. metamorphic) proteins2, but it often fails to predict their alternative experimentally observed structures3,4. Wayment-Steele, et al. steered AF2 to predict alternative structures of a few metamorphic proteins using a method they call AF-cluster5. However, their Paper lacks some essential controls needed to assess AF-clusters reliability. We find that these controls show AF-cluster to be a poor predictor of metamorphic proteins. First, closer examination of the Papers results reveals that random sequence sampling outperforms sequence clustering, challenging the claim that AF-cluster works by &#34;deconvolving conflicting sets of couplings.&#34; Further, we observe that AF-cluster mistakes some single-folding KaiB homologs for fold switchers, a critical flaw bound to mislead users. Finally, proper error analysis reveals that AF-cluster predicts many correct structures with low confidence and some experimentally unobserved conformations with confidences similar to experimentally observed ones. For these reasons, we suggest using ColabFold6-based random sequence sampling7-augmented by other predictive approaches-as a more accurate and less computationally intense alternative to AF-cluster.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.572650">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.572650" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.572650">
        <p class="paperTitle">Beyond benchmarking: towards predictive models of dataset-specific single-cell RNA-seq pipeline performance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.572650" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.572650" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fang, C.; Selega, A.; Campbell, K. R.</p>
        <p class="info">Score: 15.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.572650' target='https://doi.org/10.1101/2024.01.02.572650'> 10.1101/2024.01.02.572650</a></p>
        <p class="abstract">The advent of single-cell RNA-sequencing (scRNA-seq) has driven significant computational methods development for all steps in the scRNA-seq data analysis pipeline, including filtering, normalization, and clustering. The large number of methods and their resulting parameter combinations has created a combinatorial set of possible pipelines to analyze scRNA-seq data, which leads to the obvious question: which is best? Several benchmarking studies have sought to compare methods to answer this, but frequently find variable performance depending on dataset and pipeline characteristics. Alternatively, the large number of publicly available scRNA-seq datasets along with advances in supervised machine learning raise a tantalizing possibility: could the optimal pipeline be predicted for a given dataset? Here we begin to answer this question by applying 288 scRNA-seq analysis pipelines to 86 datasets and quantifying pipeline success via a range of measures evaluating cluster purity and biological plausibility. We build supervised machine learning models to predict pipeline success given a range of dataset and pipeline characteristics. We find both that prediction performance is significantly better than random and that in many cases pipelines predicted to perform well provide clustering outputs similar to expert-annotated cell type labels. Finally, we identify characteristics of scRNA-seq datasets that correlate with strong prediction performance that could guide when such prediction models may be useful.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573531">
        <p class="paperTitle">ADMET-AI: A machine learning ADMET platform for evaluation of large-scale chemical libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swanson, K.; Walther, P.; Leitz, J.; Mukherjee, S.; Wu, J. C.; Shivnaraine, R. V.; Zou, J.</p>
        <p class="info">Score: 20.3, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573531' target='https://doi.org/10.1101/2023.12.28.573531'> 10.1101/2023.12.28.573531</a></p>
        <p class="abstract">SummaryThe emergence of large chemical repositories and combinatorial chemical spaces, coupled with high-throughput docking and generative AI, have greatly expanded the chemical diversity of small molecules for drug discovery. Selecting compounds for experimental validation requires filtering these molecules based on favourable druglike properties, such as Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET). We developed ADMET-AI, a machine learning platform that provides fast and accurate ADMET predictions both as a website and as a Python package. ADMET-AI has the highest average rank on the TDC ADMET Benchmark Group leaderboard, and it is currently the fastest web-based ADMET predictor, with a 45% reduction in time compared to the next fastest ADMET web server. ADMET-AI can also be run locally with predictions for one million molecules taking just 3.1 hours.

Availability and ImplementationThe ADMET-AI platform is freely available both as a web server at admet.ai.greenstonebio.com and as an open-source Python package for local batch prediction at github.com/swansonk14/admet_ai (also archived on Zenodo at doi.org/10.5281/zenodo.10372930). All data and models are archived on Zenodo at doi.org/10.5281/zenodo.10372418.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573916" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573916">
        <p class="paperTitle">Novel metagenomics analysis suggests a Vibrio species is associated with stony coral tissue loss disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heinz, J. M.; Sommer, M.; Rosales, S. M.; Lu, J.; Huebner, L. K.; Salzberg, S. L.</p>
        <p class="info">Score: 9.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573916' target='https://doi.org/10.1101/2024.01.02.573916'> 10.1101/2024.01.02.573916</a></p>
        <p class="abstract">Stony coral tissue loss disease (SCTLD) has devastated coral reefs off the coast of Florida and continues to spread throughout the Caribbean. Although a number of bacterial taxa have consistently been associated with SCTLD, no pathogen has been definitively implicated in the etiology of SCTLD. Previous studies have predominantly focused on the prokaryotic community through 16S rRNA sequencing of healthy and infected tissues. Here, we provide a different analytical approach by applying a bioinformatics pipeline to publicly available whole genome sequencing samples of SCTLD lesions and healthy tissues from four stony coral species. To compensate for the lack of coral reference genomes, we used data from apparently healthy coral samples to approximate a host genome and healthy microbiome reference. The healthy reference reads were then used to filter the reads from diseased lesion tissue samples, and the remaining data were taxonomically classified at the DNA and protein levels. For DNA classifications, we used a pathogen identification protocol originally designed to identify pathogens in human tissue samples, and for protein classifications, we used a fast protein sequence aligner. Although these data were previously analyzed, our approach revealed unique patterns that were not identified in the previous work. We found a relatively high abundance of the Vibrio genus across diseased samples as well as a number of enriched Vibrio phages that further support the presence of this genus in diseased samples, suggesting that a member of the Vibrio genus may be involved in the visual lesion formation stage of SCTLD.

Article SummaryStudies of stony coral tissue loss disease (SCTLD), a devastating coral disease, have primary used 16S rRNA sequencing approaches to identify putative pathogens. This study applied human tissue pathogen identification protocols to SCTLD whole genome DNA samples. A k-mer based filtering method for diseased samples was utilized to compensate for the lack of host reference genomes. DNA and protein level classifications revealed a relatively abundant member of the Vibrio genus associated with SCTLD across four stony coral species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574326" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574326">
        <p class="paperTitle">Protein superfolds are characterised as frustration-free topologies: A case study of pure parallel beta-sheet topologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murata, H.; Toko, K.; Chikenji, G.</p>
        <p class="info">Score: 8.5, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574326' target='https://doi.org/10.1101/2024.01.05.574326'> 10.1101/2024.01.05.574326</a></p>
        <p class="abstract">A protein superfold is a type of protein fold that is observed in at least three distinct, non-homologous protein families. Structural classification studies have revealed a limited number of prevalent superfolds alongside several infrequent occurring folds, and in /{beta} type superfolds, the C-terminal {beta}-strand tends to favor the edge of the {beta}-sheet, while the N-terminal {beta}-strand is often found in the middle. The reasons behind these observations, whether they are due to evolutionary sampling bias or physical interactions, remain unclear. This article offers a physics-based explanation for these observations, specifically for pure parallel {beta}-sheet topologies. Our investigation is grounded in three established structural rules that are based on physical interactions. We have identified &#34;frustration-free topologies&#34; which are topologies that can satisfy all three rules simultaneously. In contrast, topologies that cannot are termed &#34;frustrated topologies.&#34; Our findings reveal that frustration-free topologies represent only a fraction of all theoretically possible patterns, these topologies strongly favor positioning the C-terminal {beta}-strand at the edge of the {beta}-sheet and the N-terminal {beta}-strand in the middle, and there is significant overlap between frustration-free topologies and superfolds. We also used a lattice protein model to thoroughly investigate sequence-structure relationships. Our results show that frustration-free structures are highly designable, while frustrated structures are poorly designable. These findings suggest that superfolds are highly designable due to their lack of frustration, and the preference for positioning C-terminal {beta}-strands at the edge of the {beta}-sheet is a direct result of frustration-free topologies. These insights not only enhance our understanding of sequence-structure relationships but also have significant implications for de novo protein design.

Author summaryA protein superfold is a protein fold that appears in at least three different non-homologous protein families. Superfolds are unique in their ability to accommodate multiple functions within a single fold, a feature not typically seen in other folds. Studies in structural classification have led to two notable observations: the existence of a limited number of common superfolds contrasted with a larger variety of less frequent folds, and a recurring pattern in /{beta} type superfolds where the C-terminal {beta}-strand often occupies the edge of the {beta}-sheet, while the N-terminal {beta}-strand is usually found in the middle. The origins of these patterns, whether they stem from evolutionary sampling bias or physical interaction mechanisms, remain unclear. This article provides a physics-oriented explanation for these observations, specifically concentrating on pure parallel {beta}-sheet topologies. The insights gained from this research are crucial in enhancing our understanding of the relationship between protein sequences and structures, and are expected to contribute significantly to the de novo design of new proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573913" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573913">
        <p class="paperTitle">Genomics 2 Proteins portal: A resource and discovery tool for linking genetic screening outputs to protein sequences and structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kwon, S.; Safer, J.; Nguyen, D. T.; Hoksza, D.; May, P.; Arbesfeld, J.; Rubin, A. F.; Campbell, A. J.; Burgin, A.; Iqbal, S.</p>
        <p class="info">Score: 8.3, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573913' target='https://doi.org/10.1101/2024.01.02.573913'> 10.1101/2024.01.02.573913</a></p>
        <p class="abstract">Recent advances in AI-based methods have revolutionized the field of structural biology. Concomitantly, high-throughput sequencing and functional genomics technologies have enabled the detection and generation of variants at an unprecedented scale. However, efficient tools and resources are needed to link these two disparate data types - to &#34;map&#34; variants onto protein structures, to better understand how the variation causes disease and thereby design therapeutics. Here we present the Genomics 2 Proteins Portal (G2P; g2p.broadinstitute.org/): a human proteome-wide resource that maps 19,996,443 genetic variants onto 42,413 protein sequences and 77,923 structures, with a comprehensive set of structural and functional features. Additionally, the G2P portal generalizes the capability of linking genomics to proteins beyond databases by allowing users to interactively upload protein residue-wise annotations (variants, scores, etc.) as well as the protein structure to establish the connection. The portal serves as an easy-to-use discovery tool for researchers and scientists to hypothesize the structure-function relationship between natural or synthetic variations and their molecular phenotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573985" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573985">
        <p class="paperTitle">Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez, G.; Herath, I.; Veselkov, K.; Bronstein, M.; Zitnik, M.</p>
        <p class="info">Score: 7.9, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573985' target='https://doi.org/10.1101/2024.01.03.573985'> 10.1101/2024.01.03.573985</a></p>
        <p class="abstract">As an alternative to target-driven drug discovery, phenotype-driven approaches identify compounds that counteract the overall disease effects by analyzing phenotypic signatures. Our study introduces a novel approach to this field, aiming to expand the search space for new therapeutic agents. We introduce PDGrapher, a causally-inspired graph neural network model designed to predict arbitrary perturbagens - sets of therapeutic targets - capable of reversing disease effects. Unlike existing methods that learn responses to perturbations, PDGrapher solves the inverse problem, which is to infer the perturbagens necessary to achieve a specific response - i.e., directly predicting perturbagens by learning which perturbations elicit a desired response. Experiments across eight datasets of genetic and chemical perturbations show that PDGrapher successfully predicted effective perturbagens in up to 9% additional test samples and ranked therapeutic targets up to 35% higher than competing methods. A key innovation of PDGrapher is its direct prediction capability, which contrasts with the indirect, computationally intensive models traditionally used in phenotype-driven drug discovery that only predict changes in phenotypes due to perturbations. The direct approach enables PDGrapher to train up to 30 times faster, representing a significant leap in efficiency. Our results suggest that PDGrapher can advance phenotype-driven drug discovery, offering a fast and comprehensive approach to identifying therapeutically useful perturbations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573710" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573710">
        <p class="paperTitle">FinaleMe: Predicting DNA methylation by the fragmentation patterns of plasma cell-free DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Reed, S. C.; Lo, C.; Choudhury, A. D.; Parsons, H. A.; Stover, D. G.; Ha, G.; Gydush, G.; Rhoades, J.; Rotem, D.; Freeman, S.; Katz, D.; Bandaru, R.; Zheng, H.; Fu, H.; Adalsteinsson, V. A.; Kellis, M.</p>
        <p class="info">Score: 7.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573710' target='https://doi.org/10.1101/2024.01.02.573710'> 10.1101/2024.01.02.573710</a></p>
        <p class="abstract">Analysis of DNA methylation in cell-free DNA (cfDNA) reveals clinically relevant biomarkers but requires specialized protocols and sufficient input material that limits its applicability. Millions of cfDNA samples have been profiled by genomic sequencing. To maximize the gene regulation information from the existing dataset, we developed FinaleMe, a non-homogeneous Hidden Markov Model (HMM), to predict DNA methylation of cfDNA and, therefore, tissues-of-origin directly from plasma whole-genome sequencing (WGS). We validated the performance with 80 pairs of deep and shallow-coverage WGS and whole-genome bisulfite sequencing (WGBS) data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574118" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574118">
        <p class="paperTitle">TEMINET: A Co-Informative and Trustworthy Multi-Omics Integration Network for Diagnostic Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, H.; Liang, H.; Liu, H.; Fan, Z.; Wei, Y.; Yao, X.; Cong, S.</p>
        <p class="info">Score: 7.3, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574118' target='https://doi.org/10.1101/2024.01.03.574118'> 10.1101/2024.01.03.574118</a></p>
        <p class="abstract">Advancing the domain of biomedical investigation, integrated multi-omics data have shown exceptional performance in elucidating complex human diseases. However, as the variety of omics information expands, precisely perceiving the informativeness of intra- and inter-omics becomes challenging due to the intricate interrelations, thus posing significant obstacles in multi-omics data integration. To address this, we introduce a novel multi-omics integration approach, referred to as TEMINET. This approach enhances diagnostic prediction by leveraging an intra-omics co-informative representation method and a trustworthy learning strategy used to address inter-omics fusion. Considering the multifactorial nature of complex diseases, TEMINET utilizes intra-omics features to construct disease-specific networks, then applies graph attention networks and a multi-level framework to capture more collective informativeness than pairwise relations. To perceive the contribution of co-informative representations within intra-omics, we design a trustworthy learning strategy to identify the reliability of each omics in integration. To integrate inter-omics information, a combined beliefs fusion approach is deployed to harmonize the trustworthy representations of different omics types effectively. Our experiments across four different diseases using mRNA, methylation, and miRNA data demonstrate that TEMINET achieves advanced performance and robustness in classification tasks.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
