<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.
Authors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.
Score: 34.4, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561091
MotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.
Authors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.
Score: 34.4, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561091
MotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.
Authors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.
Score: 34.4, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561091
MotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.\nAuthors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.\nScore: 34.4, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561091\nMotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics.",
  "keywords": [
    
  ],
  "articleBody": " AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.\nAuthors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.\nScore: 34.4, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561091\nMotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics. Ideally, phylogenetic methods should not only model substitutions along gene trees, which explain differences between homologous gene sequences, but also the processes that generate the gene trees themselves along a shared species tree. To conduct accurate inferences, one needs to account for uncertainty at both levels, that is, in gene trees estimated from inherently short sequences and in their diverse evolutionary histories along a shared species tree. ResultsWe present AleRax, a software that can infer reconciled gene trees together with a shared species tree using a simple, yet powerful, probabilistic model of gene duplication, transfer, and loss. A key feature of AleRax is its ability to account for uncertainty in the gene tree and its reconciliation by using an efficient approximation to calculate the joint phylogenetic -reconciliation likelihood and sample reconciled gene trees accordingly. Simulations and analyses of empirical data show that AleRax is one order of magnitude faster than competing gene tree inference tools while attaining the same accuracy. It is consistently more robust than species tree inference methods such as SpeciesRax and ASTRAL-Pro 2 under gene tree uncertainty. Finally, AleRax can process multiple gene families in parallel thereby allowing users to compare competing phylogenetic hypotheses and estimate model parameters, such as DTL probabilities for genome-scale datasets with hundreds of taxa Availability and ImplementationGNU GPL at https://github.com/BenoitMorel/AleRax and data are made available at https://cme.h-its.org/exelixis/material/alerax_data.tar.gz. ContactBenoit.Morel@h-its.org Supplementary informationSupplementary material is available.\nPROTOSPACEJAM: AN OPEN-SOURCE, CUSTOMIZABLE AND WEB-ACCESSIBLE DESIGN PLATFORM FOR CRISPR/CAS9 INSERTIONAL KNOCK-IN\nAuthors: Peng, D.; Vangipuram, M.; Wong, J.; Leonetti, M.\nScore: 34.3, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560793\nCRISPR/Cas9-mediated knock-in of DNA sequences enables precise genome engineering for research and therapeutic applications. However, designing effective guide RNAs (gRNAs) and homology-directed repair (HDR) donors remains a bottleneck. Here, we present protoSpaceJAM, an open-source algorithm to automate and optimize gRNA and HDR donor design for CRISPR/Cas9 insertional knock-in experiments. protoSpaceJAM utilizes biological rules to rank gRNAs based on specificity, distance to insertion site, and position relative to regulatory regions. protoSpaceJAM can introduce recoding mutations (silent mutations and mutations in non-coding sequences) in HDR donors to prevent re-cutting and increase knock-in efficiency. Users can customize parameters and design double-stranded or single-stranded donors. We validated protoSpaceJAMs design rules by demonstrating increased knock-in efficiency with recoding mutations and optimal strand selection for single-stranded donors. An additional module enables the design of genotyping primers for next-generation sequencing of edited alleles. Overall, protoSpaceJAM streamlines and optimizes CRISPR knock-in experimental design in a flexible and modular manner to benefit diverse research and therapeutic applications. protoSpaceJAM is available open-source as an interactive web tool at protospacejam.czbiohub.org or as a standalone Python package at github.com/czbiohub-sf/protoSpaceJAM.\nCharacterizing cell-type spatial relationships across length scales in spatially resolved omics data\nAuthors: dos Santos Peixoto, R.; Miller, B. F.; Brusko, M. A.; Atta, L.; Anant, M.; Atkinson, M. A.; Brusko, T. M.; Wasserfall, C. H.; Fan, J.\nScore: 26.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560733\nSpatially resolved omics technologies provide molecular profiling of cells while preserving their organization within tissues, allowing for the evaluation of cell-type spatial relationships. We developed CRAWDAD to quantify cell-type spatial relationships across length scales. We highlight the utility of such multi-scale characterization on simulated data, recapitulate expected cell-type spatial relationships in tissues such as the mouse brain and embryo, and delineate functionally relevant spatial-defined cell-type subsets in the human spleen.\nThe structural landscape of the immunoglobulin fold by large-scale de novo design\nAuthors: Roel-Touris, J.; Carcelen, L.; Marcos, E.\nScore: 24.4, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560637\nDe novo designing immunoglobulin-like frameworks that allow for functional loop diversification shows great potential for crafting antibody-like scaffolds with fully customizable structures and functions. In this work, we combined de novo parametric design with deep-learning methods for protein structure prediction and design to explore the structural landscape of 7-stranded immunoglobulin domains. After screening folding of nearly 4 million designs, we have assembled a structurally diverse library of [~]50,000 immunoglobulin domains with high-confidence AlphaFold2 predictions and structures diverging from naturally occurring ones. The designed dataset enabled us to identify structural requirements for the correct folding of immunoglobulin domains, shed light on {beta}-sheet-{beta}-sheet rotational preferences and how these are linked to functional properties. Our approach eliminates the need for preset loop conformations and opens the route to large-scale de novo design of immunoglobulin-like frameworks.\nSimulation-based Benchmarking of Ancient Haplotype Inference for Detecting Population Structure\nAuthors: Medina-Tretmanis, J.; Jay, F.; Avila-Arcos, M. C.; Huerta-Sanchez, E.\nScore: 20.7, Published: 2023-10-03 DOI: 10.1101/2023.09.28.560049\nPaleogenomic data has informed us about the movements, growth, and relationships of ancient populations. It has also given us context for medically relevant adaptations that appear in present-day humans due to introgression from other hominids, and it continues to help us characterize the evolutionary history of humans. However, ancient DNA (aDNA) presents several practical challenges as various factors such as deamination, high fragmentation, environmental contamination of aDNA, and low amounts of recoverable endogenous DNA, make aDNA recovery and analysis more difficult than modern DNA. Most studies with aDNA leverage only SNP data, and only a few studies have made inferences on human demographic history based on haplotype data, possibly because haplotype estimation (or phasing) has not yet been systematically evaluated in the context of aDNA. Here, we evaluate how the unique challenges of aDNA can impact phasing quality. We also develop a software tool that simulates aDNA taking into account the features of aDNA as well as the evolutionary history of the population. We measured phasing error as a function of aDNA quality and demographic history, and found that low phasing error is achievable even for very ancient individuals ([~] 400 generations in the past) as long as contamination and read depth are adequate. Our results show that population splits or bottleneck events occurring between the reference and phased populations affect phasing quality, with bottlenecks resulting in the highest average error rates. Finally, we found that using estimated haplotypes, even if not completely accurate, is superior to using the simulated genotype data when reconstructing changes in population structure after population splits between present-day and ancient populations. AvailabilityAll software used for simulation and analysis is available at github.com/Jazpy/Paleogenomic-Datasim\nBuDDI: Bulk Deconvolution with Domain Invariance to predict cell-type-specific perturbations from bulk\nAuthors: Davidson, N. R.; Greene, C. S.\nScore: 15.7, Published: 2023-10-05 DOI: 10.1101/2023.07.20.549951\nWhile single-cell experiments provide deep cellular resolution within a single sample, some single-cell experiments are inherently more challenging than bulk experiments due to dissociation difficulties, cost, or limited tissue availability. This creates a situation where we have deep cellular profiles of one sample or condition, and bulk profiles across multiple samples and conditions. To bridge this gap, we propose BuDDI (BUlk Deconvolution with Domain Invariance). BuDDI utilizes domain adaptation techniques to effectively integrate available corpora of case-control bulk and reference scRNA-seq observations to infer cell-type-specific perturbation effects. BuDDI achieves this by learning independent latent spaces within a single variational autoencoder (VAE) encompassing at least four sources of variability: 1) cell-type proportion, 2) perturbation effect, 3) structured experimental variability, and 4) remaining variability. Since each latent space is encouraged to be independent, we simulate perturbation responses by independently composing each latent space to simulate cell-type-specific perturbation responses. We evaluated BuDDI's performance on simulated and real data with experimental designs of increasing complexity. We first validated that BuDDI could learn domain invariant latent spaces on data with matched samples across each source of variability. Then we validated that BuDDI could accurately predict cell-type-specific perturbation response when no single-cell perturbed profiles were used during training; instead, only bulk samples had both perturbed and non-perturbed observations. Finally, we validated BuDDI on predicting sex-specific differences, an experimental design where it is not possible to have matched samples. In each experiment, BuDDI outperformed all other comparative methods and baselines. As more reference atlases are completed, BuDDI provides a path to combine these resources with bulk-profiled treatment or disease signatures to study perturbations, sex differences, or other factors at single-cell resolution.\nSegzoo: a turnkey system that summarizes genome annotations\nAuthors: Mendez, M.; Liu, Y.; Asenjo Ponce de Leon, M.; Hoffman, M. M.\nScore: 10.9, Published: 2023-10-05 DOI: 10.1101/2023.10.03.559369\nSegmentation and automated genome annotation (SAGA) techniques, such as Segway and ChromHMM, assign labels to every part of the genome, identifying similar patterns across multiple genomic input signals. Inferring biological meaning in these patterns remains challenging. Doing so requires a time-consuming process of manually downloading reference data, running multiple analysis methods, and interpreting many individual results. To simplify these tasks, we developed the turnkey system Segzoo. As input, Segzoo only requires a genome annotation file in browser extensible data (BED) format. It automatically downloads the rest of the data required for comparisons. Segzoo performs analyses using these data and summarizes results in a single visualization. Availability and ImplementationSource code for Python [\u0026ge;]3.7 on Linux freely available for download at https://github.com/hoffmangroup/segzoo under the GNU General Public License (GPL) version 2. Segzoo is also available in the Bioconda package segzoo: https://anaconda.org/bioconda/segzoo.\nCellPLM: Pre-training of Cell Language Model Beyond Single Cells\nAuthors: Wen, H.; Tang, W.; Dai, X.; Ding, J.; Jin, W.; Xie, Y.; Tang, J.\nScore: 10.9, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560734\nThe current state-of-the-art single-cell pre-trained models are greatly inspired by the success of large language models. They trained transformers by treating genes as tokens and cells as sentences. However, three fundamental differences between single-cell data and natural language data are overlooked: (1) scRNA-seq data are presented as bag-of-genes instead of sequences of RNAs; (2) Cell-cell relations are more intricate and important than inter-sentence relations; and (3) The quantity of single-cell data is considerably inferior to text data, and they are very noisy. In light of these characteristics, we propose a new pre-trained model CellPLM, which takes cells as tokens and tissues as sentences. In addition, we leverage spatially-resolved transcriptomic data in pre-training to facilitate learning cell-cell relationships and introduce a Gaussian mixture prior distribution as an additional inductive bias to overcome data limitation. CellPLM is the first single-cell pre-trained transformer that encodes cell-cell relations and it consistently outperforms existing pre-trained and non-pre-trained models in diverse downstream tasks, with 100x times higher inference speed compared to existing pre-trained models.\nSaProt: Protein Language Modeling with Structure-aware Vocabulary\nAuthors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.\nScore: 43.5, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349\nLarge-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a \"structure-aware vocabulary\" that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures. Through extensive evaluation, our SaProt model surpasses well-established and renowned baselines across 10 significant downstream tasks, demonstrating its exceptional capacity and broad applicability. We have made the code1, pretrained model, and all relevant materials available at https://github.com/westlake-repl/SaProt.\nRummagene: Mining Gene Sets from Supporting Materials of PMC Publications\nAuthors: Clarke, D. J. B.; Marino, G. B.; Deng, E. Z.; Xie, Z.; Evangelista, J. E.; Ma'ayan, A.\nScore: 9.9, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560783\nEvery week thousands of biomedical research papers are published with a portion of them containing supporting tables with data about genes, transcripts, variants, and proteins. For example, supporting tables may contain differentially expressed genes and proteins from transcriptomics and proteomics assays, targets of transcription factors from ChIP-seq experiments, hits from genome-wide CRISPR screens, or genes identified to harbor mutations from GWAS studies. Because these gene sets are commonly buried in the supplemental tables of research publications, they are not widely available for search and reuse. Rummagene, available from https://rummagene.com, is a web server application that provides access to hundreds of thousands human and mouse gene sets extracted from supporting materials of publications listed on PubMed Central (PMC). To create Rummagene, we first developed a softbot that extracts human and mouse gene sets from supporting tables of PMC publications. So far, the softbot has scanned 5,448,589 PMC articles to find 121,237 articles that contain 642,389 gene sets. These gene sets are served for enrichment analysis, free text, and table title search. Users of Rummagene can submit their own gene sets to find matching gene sets ranked by their overlap with the input gene set. In addition to providing the extracted gene sets for search, we investigated the massive corpus of these gene sets for statistical patterns. We show that the number of gene sets reported in publications is rapidly increasing, containing both short sets that are highly enriched in highly studied genes, and long sets from omics profiling. We also demonstrate that the gene sets in Rummagene can be used for transcription factor and kinase enrichment analyses, and for gene function predictions. By combining gene set similarity with abstract similarity, Rummagene can be used to find surprising relationships between unexpected biological processes, concepts, and named entities. Finally, by overlaying the Rummagene gene set space with the Enrichr gene set space we can discover areas of biological and biomedical knowledge unique to each resource.\n",
  "wordCount" : "2279",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561091" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561091">
        <p class="paperTitle">AleRax: A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer and loss.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morel, B.; Williams, T. A.; Stamatakis, A.; Szollosi, G.</p>
        <p class="info">Score: 34.4, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561091' target='https://doi.org/10.1101/2023.10.06.561091'> 10.1101/2023.10.06.561091</a></p>
        <p class="abstract">MotivationGenomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics. Ideally, phylogenetic methods should not only model substitutions along gene trees, which explain differences between homologous gene sequences, but also the processes that generate the gene trees themselves along a shared species tree. To conduct accurate inferences, one needs to account for uncertainty at both levels, that is, in gene trees estimated from inherently short sequences and in their diverse evolutionary histories along a shared species tree.

ResultsWe present AleRax, a software that can infer reconciled gene trees together with a shared species tree using a simple, yet powerful, probabilistic model of gene duplication, transfer, and loss. A key feature of AleRax is its ability to account for uncertainty in the gene tree and its reconciliation by using an efficient approximation to calculate the joint phylogenetic -reconciliation likelihood and sample reconciled gene trees accordingly. Simulations and analyses of empirical data show that AleRax is one order of magnitude faster than competing gene tree inference tools while attaining the same accuracy. It is consistently more robust than species tree inference methods such as SpeciesRax and ASTRAL-Pro 2 under gene tree uncertainty. Finally, AleRax can process multiple gene families in parallel thereby allowing users to compare competing phylogenetic hypotheses and estimate model parameters, such as DTL probabilities for genome-scale datasets with hundreds of taxa

Availability and ImplementationGNU GPL at https://github.com/BenoitMorel/AleRax and data are made available at https://cme.h-its.org/exelixis/material/alerax_data.tar.gz.

ContactBenoit.Morel@h-its.org

Supplementary informationSupplementary material is available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560793">
        <p class="paperTitle">PROTOSPACEJAM: AN OPEN-SOURCE, CUSTOMIZABLE AND WEB-ACCESSIBLE DESIGN PLATFORM FOR CRISPR/CAS9 INSERTIONAL KNOCK-IN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, D.; Vangipuram, M.; Wong, J.; Leonetti, M.</p>
        <p class="info">Score: 34.3, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560793' target='https://doi.org/10.1101/2023.10.04.560793'> 10.1101/2023.10.04.560793</a></p>
        <p class="abstract">CRISPR/Cas9-mediated knock-in of DNA sequences enables precise genome engineering for research and therapeutic applications. However, designing effective guide RNAs (gRNAs) and homology-directed repair (HDR) donors remains a bottleneck. Here, we present protoSpaceJAM, an open-source algorithm to automate and optimize gRNA and HDR donor design for CRISPR/Cas9 insertional knock-in experiments. protoSpaceJAM utilizes biological rules to rank gRNAs based on specificity, distance to insertion site, and position relative to regulatory regions. protoSpaceJAM can introduce recoding mutations (silent mutations and mutations in non-coding sequences) in HDR donors to prevent re-cutting and increase knock-in efficiency. Users can customize parameters and design double-stranded or single-stranded donors. We validated protoSpaceJAMs design rules by demonstrating increased knock-in efficiency with recoding mutations and optimal strand selection for single-stranded donors. An additional module enables the design of genotyping primers for next-generation sequencing of edited alleles. Overall, protoSpaceJAM streamlines and optimizes CRISPR knock-in experimental design in a flexible and modular manner to benefit diverse research and therapeutic applications. protoSpaceJAM is available open-source as an interactive web tool at protospacejam.czbiohub.org or as a standalone Python package at github.com/czbiohub-sf/protoSpaceJAM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560733">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560733" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560733">
        <p class="paperTitle">Characterizing cell-type spatial relationships across length scales in spatially resolved omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560733" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560733" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: dos Santos Peixoto, R.; Miller, B. F.; Brusko, M. A.; Atta, L.; Anant, M.; Atkinson, M. A.; Brusko, T. M.; Wasserfall, C. H.; Fan, J.</p>
        <p class="info">Score: 26.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560733' target='https://doi.org/10.1101/2023.10.05.560733'> 10.1101/2023.10.05.560733</a></p>
        <p class="abstract">Spatially resolved omics technologies provide molecular profiling of cells while preserving their organization within tissues, allowing for the evaluation of cell-type spatial relationships. We developed CRAWDAD to quantify cell-type spatial relationships across length scales. We highlight the utility of such multi-scale characterization on simulated data, recapitulate expected cell-type spatial relationships in tissues such as the mouse brain and embryo, and delineate functionally relevant spatial-defined cell-type subsets in the human spleen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560637">
        <p class="paperTitle">The structural landscape of the immunoglobulin fold by large-scale de novo design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roel-Touris, J.; Carcelen, L.; Marcos, E.</p>
        <p class="info">Score: 24.4, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560637' target='https://doi.org/10.1101/2023.10.03.560637'> 10.1101/2023.10.03.560637</a></p>
        <p class="abstract">De novo designing immunoglobulin-like frameworks that allow for functional loop diversification shows great potential for crafting antibody-like scaffolds with fully customizable structures and functions. In this work, we combined de novo parametric design with deep-learning methods for protein structure prediction and design to explore the structural landscape of 7-stranded immunoglobulin domains. After screening folding of nearly 4 million designs, we have assembled a structurally diverse library of [~]50,000 immunoglobulin domains with high-confidence AlphaFold2 predictions and structures diverging from naturally occurring ones. The designed dataset enabled us to identify structural requirements for the correct folding of immunoglobulin domains, shed light on {beta}-sheet-{beta}-sheet rotational preferences and how these are linked to functional properties. Our approach eliminates the need for preset loop conformations and opens the route to large-scale de novo design of immunoglobulin-like frameworks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.560049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.560049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.560049">
        <p class="paperTitle">Simulation-based Benchmarking of Ancient Haplotype Inference for Detecting Population Structure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.560049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.560049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Medina-Tretmanis, J.; Jay, F.; Avila-Arcos, M. C.; Huerta-Sanchez, E.</p>
        <p class="info">Score: 20.7, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.560049' target='https://doi.org/10.1101/2023.09.28.560049'> 10.1101/2023.09.28.560049</a></p>
        <p class="abstract">Paleogenomic data has informed us about the movements, growth, and relationships of ancient populations. It has also given us context for medically relevant adaptations that appear in present-day humans due to introgression from other hominids, and it continues to help us characterize the evolutionary history of humans. However, ancient DNA (aDNA) presents several practical challenges as various factors such as deamination, high fragmentation, environmental contamination of aDNA, and low amounts of recoverable endogenous DNA, make aDNA recovery and analysis more difficult than modern DNA. Most studies with aDNA leverage only SNP data, and only a few studies have made inferences on human demographic history based on haplotype data, possibly because haplotype estimation (or phasing) has not yet been systematically evaluated in the context of aDNA. Here, we evaluate how the unique challenges of aDNA can impact phasing quality. We also develop a software tool that simulates aDNA taking into account the features of aDNA as well as the evolutionary history of the population. We measured phasing error as a function of aDNA quality and demographic history, and found that low phasing error is achievable even for very ancient individuals ([~] 400 generations in the past) as long as contamination and read depth are adequate. Our results show that population splits or bottleneck events occurring between the reference and phased populations affect phasing quality, with bottlenecks resulting in the highest average error rates. Finally, we found that using estimated haplotypes, even if not completely accurate, is superior to using the simulated genotype data when reconstructing changes in population structure after population splits between present-day and ancient populations.

AvailabilityAll software used for simulation and analysis is available at github.com/Jazpy/Paleogenomic-Datasim</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549951">
        <p class="paperTitle">BuDDI: Bulk Deconvolution with Domain Invariance to predict cell-type-specific perturbations from bulk</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, N. R.; Greene, C. S.</p>
        <p class="info">Score: 15.7, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549951' target='https://doi.org/10.1101/2023.07.20.549951'> 10.1101/2023.07.20.549951</a></p>
        <p class="abstract">While single-cell experiments provide deep cellular resolution within a single sample, some single-cell experiments are inherently more challenging than bulk experiments due to dissociation difficulties, cost, or limited tissue availability. This creates a situation where we have deep cellular profiles of one sample or condition, and bulk profiles across multiple samples and conditions. To bridge this gap, we propose BuDDI (BUlk Deconvolution with Domain Invariance). BuDDI utilizes domain adaptation techniques to effectively integrate available corpora of case-control bulk and reference scRNA-seq observations to infer cell-type-specific perturbation effects. BuDDI achieves this by learning independent latent spaces within a single variational autoencoder (VAE) encompassing at least four sources of variability: 1) cell-type proportion, 2) perturbation effect, 3) structured experimental variability, and 4) remaining variability. Since each latent space is encouraged to be independent, we simulate perturbation responses by independently composing each latent space to simulate cell-type-specific perturbation responses. We evaluated BuDDI&#39;s performance on simulated and real data with experimental designs of increasing complexity. We first validated that BuDDI could learn domain invariant latent spaces on data with matched samples across each source of variability. Then we validated that BuDDI could accurately predict cell-type-specific perturbation response when no single-cell perturbed profiles were used during training; instead, only bulk samples had both perturbed and non-perturbed observations. Finally, we validated BuDDI on predicting sex-specific differences, an experimental design where it is not possible to have matched samples. In each experiment, BuDDI outperformed all other comparative methods and baselines. As more reference atlases are completed, BuDDI provides a path to combine these resources with bulk-profiled treatment or disease signatures to study perturbations, sex differences, or other factors at single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.559369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.559369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.559369">
        <p class="paperTitle">Segzoo: a turnkey system that summarizes genome annotations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.559369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.559369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mendez, M.; Liu, Y.; Asenjo Ponce de Leon, M.; Hoffman, M. M.</p>
        <p class="info">Score: 10.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.559369' target='https://doi.org/10.1101/2023.10.03.559369'> 10.1101/2023.10.03.559369</a></p>
        <p class="abstract">Segmentation and automated genome annotation (SAGA) techniques, such as Segway and ChromHMM, assign labels to every part of the genome, identifying similar patterns across multiple genomic input signals. Inferring biological meaning in these patterns remains challenging. Doing so requires a time-consuming process of manually downloading reference data, running multiple analysis methods, and interpreting many individual results.

To simplify these tasks, we developed the turnkey system Segzoo. As input, Segzoo only requires a genome annotation file in browser extensible data (BED) format. It automatically downloads the rest of the data required for comparisons. Segzoo performs analyses using these data and summarizes results in a single visualization.

Availability and ImplementationSource code for Python [&amp;ge;]3.7 on Linux freely available for download at https://github.com/hoffmangroup/segzoo under the GNU General Public License (GPL) version 2. Segzoo is also available in the Bioconda package segzoo: https://anaconda.org/bioconda/segzoo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560734">
        <p class="paperTitle">CellPLM: Pre-training of Cell Language Model Beyond Single Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wen, H.; Tang, W.; Dai, X.; Ding, J.; Jin, W.; Xie, Y.; Tang, J.</p>
        <p class="info">Score: 10.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560734' target='https://doi.org/10.1101/2023.10.03.560734'> 10.1101/2023.10.03.560734</a></p>
        <p class="abstract">The current state-of-the-art single-cell pre-trained models are greatly inspired by the success of large language models. They trained transformers by treating genes as tokens and cells as sentences. However, three fundamental differences between single-cell data and natural language data are overlooked: (1) scRNA-seq data are presented as bag-of-genes instead of sequences of RNAs; (2) Cell-cell relations are more intricate and important than inter-sentence relations; and (3) The quantity of single-cell data is considerably inferior to text data, and they are very noisy. In light of these characteristics, we propose a new pre-trained model CellPLM, which takes cells as tokens and tissues as sentences. In addition, we leverage spatially-resolved transcriptomic data in pre-training to facilitate learning cell-cell relationships and introduce a Gaussian mixture prior distribution as an additional inductive bias to overcome data limitation. CellPLM is the first single-cell pre-trained transformer that encodes cell-cell relations and it consistently outperforms existing pre-trained and non-pre-trained models in diverse downstream tasks, with 100x times higher inference speed compared to existing pre-trained models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560349">
        <p class="paperTitle">SaProt: Protein Language Modeling with Structure-aware Vocabulary</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.</p>
        <p class="info">Score: 43.5, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560349' target='https://doi.org/10.1101/2023.10.01.560349'> 10.1101/2023.10.01.560349</a></p>
        <p class="abstract">Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a &#34;structure-aware vocabulary&#34; that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures. Through extensive evaluation, our SaProt model surpasses well-established and renowned baselines across 10 significant downstream tasks, demonstrating its exceptional capacity and broad applicability. We have made the code1, pretrained model, and all relevant materials available at https://github.com/westlake-repl/SaProt.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560783">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560783" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560783">
        <p class="paperTitle">Rummagene: Mining Gene Sets from Supporting Materials of PMC Publications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560783" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560783" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clarke, D. J. B.; Marino, G. B.; Deng, E. Z.; Xie, Z.; Evangelista, J. E.; Ma&#39;ayan, A.</p>
        <p class="info">Score: 9.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560783' target='https://doi.org/10.1101/2023.10.03.560783'> 10.1101/2023.10.03.560783</a></p>
        <p class="abstract">Every week thousands of biomedical research papers are published with a portion of them containing supporting tables with data about genes, transcripts, variants, and proteins. For example, supporting tables may contain differentially expressed genes and proteins from transcriptomics and proteomics assays, targets of transcription factors from ChIP-seq experiments, hits from genome-wide CRISPR screens, or genes identified to harbor mutations from GWAS studies. Because these gene sets are commonly buried in the supplemental tables of research publications, they are not widely available for search and reuse. Rummagene, available from https://rummagene.com, is a web server application that provides access to hundreds of thousands human and mouse gene sets extracted from supporting materials of publications listed on PubMed Central (PMC). To create Rummagene, we first developed a softbot that extracts human and mouse gene sets from supporting tables of PMC publications. So far, the softbot has scanned 5,448,589 PMC articles to find 121,237 articles that contain 642,389 gene sets. These gene sets are served for enrichment analysis, free text, and table title search. Users of Rummagene can submit their own gene sets to find matching gene sets ranked by their overlap with the input gene set. In addition to providing the extracted gene sets for search, we investigated the massive corpus of these gene sets for statistical patterns. We show that the number of gene sets reported in publications is rapidly increasing, containing both short sets that are highly enriched in highly studied genes, and long sets from omics profiling. We also demonstrate that the gene sets in Rummagene can be used for transcription factor and kinase enrichment analyses, and for gene function predictions. By combining gene set similarity with abstract similarity, Rummagene can be used to find surprising relationships between unexpected biological processes, concepts, and named entities. Finally, by overlaying the Rummagene gene set space with the Enrichr gene set space we can discover areas of biological and biomedical knowledge unique to each resource.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
