<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure prediction of protein-ligand complexes from sequence information with Umol
Authors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.
Score: 89.3, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565471
Protein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Structure prediction of protein-ligand complexes from sequence information with Umol
Authors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.
Score: 89.3, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565471
Protein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Structure prediction of protein-ligand complexes from sequence information with Umol
Authors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.
Score: 89.3, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565471
Protein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Structure prediction of protein-ligand complexes from sequence information with Umol\nAuthors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.\nScore: 89.3, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565471\nProtein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand.",
  "keywords": [
    
  ],
  "articleBody": " Structure prediction of protein-ligand complexes from sequence information with Umol\nAuthors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.\nScore: 89.3, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565471\nProtein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand. At a high accuracy threshold, unseen protein-ligand complexes can be predicted more accurately than for RoseTTAFold-AA, and at medium accuracy even classical docking methods that use known protein structures as input are surpassed. The high accuracy presented here suggests that the goal of AI-based drug discovery is one step closer, but there is still a way to go to fully grasp the complexity of protein-ligand interactions. Umol is available at: https://github.com/patrickbryant1/Umol .\nCellChat for systematic analysis of cell-cell communication from single-cell and spatially resolved transcriptomics\nAuthors: Jin, S.; Plikus, M. V.; Nie, Q.\nScore: 48.5, Published: 2023-11-05 DOI: 10.1101/2023.11.05.565674\nRecent advances in single-cell sequencing technologies offer an opportunity to explore cell-cell communication in tissues systematically and with reduced bias. A key challenge is the integration between known molecular interactions and measurements into a framework to identify and analyze complex cell-cell communication networks. Previously, we developed a computational tool, named CellChat that infers and analyzes cell-cell communication networks from single-cell RNA-sequencing (scRNA-seq) data within an easily interpretable framework. CellChat quantifies the signaling communication probability between two cell groups using a simplified mass action-based model, which incorporates the core interaction between ligands and receptors with multi-subunit structure along with modulation by cofactors. CellChat v2 is an updated version that includes direct incorporation of spatial locations of cells, if available, to infer spatially proximal cell-cell communication, additional comparison functionalities, expanded database of ligand-receptor pairs along with rich annotations, and an Interactive CellChat Explorer. Here we provide a step-by-step protocol for using CellChat v2 that can be used for both scRNA-seq and spatially resolved transcriptomic data, including inference and analysis of cell-cell communication from one dataset and identification of altered signaling across different datasets. The key steps of applying CellChat v2 to spatially resolved transcriptomics are described in detail. The R implementation of CellChat v2 toolkit and tutorials with the graphic outputs are available at https://github.com/jinworks/CellChat. This protocol typically takes around 20 minutes, and no specialized prior bioinformatics training is required to complete the task.\nIdentifying cell states in single-cell RNA-seq data at statistically maximal resolution\nAuthors: Grobecker, P.; van Nimwegen, E.\nScore: 45.5, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564980\nSingle-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties. Although a vast number of different methods is applied in the literature for clustering cells into subsets with similar expression profiles, these methods generally lack rigorously specified objectives, involve multiple complex layers of normalization, filtering, feature selection, dimensionalityreduction, employ ad hoc measures of distance or similarity between cells, often ignore the known measurement noise properties of scRNA-seq measurements, and include a large number of tunable parameters. Consequently, it is virtually impossible to assign concrete biophysical meaning to the clusterings that result from these methods. Here we address the following problem: Given raw unique molecule identifier (UMI) counts of an scRNA-seq dataset, partition the cells into subsets such that the gene expression states of the cells in each subset are statistically indistinguishable, and each subset corresponds to a distinct gene expression state. That is, we aim to partition cells so as to maximally reduce the complexity of the dataset without removing any of its meaningful structure. We show that, given the known measurement noise structure of scRNA-seq data, this problem is mathematically well-defined and derive its unique solution from first principles. We have implemented this solution in a tool called CO_SCPLOWELLSTATESC_SCPLOW which operates directly on the raw data and automatically determines the optimal partition and cluster number, with zero tunable parameters. We show that, on synthetic datasets, CO_SCPLOWELLSTATESC_SCPLOW almost perfectly recovers optimal partitions. On real data, CO_SCPLOWELLSTATESC_SCPLOW robustly identifies subtle substructure within groups of cells that are traditionally annotated as a common cell type. Moreover, we show that the diversity of gene expression states that CO_SCPLOWELLSTATESC_SCPLOW identifies systematically depends on the tissue of origin and not on technical features of the experiments such as the total number of cells and total UMI count per cell. In addition to the CO_SCPLOWELLSTATESC_SCPLOW tool we also provide a small toolbox of software to place the identified cellstates into a hierarchical tree of higher-order clusters, to identify the most important marker genes at each branch of this hierarchy, and to visualize these results.\nPredmoter - Cross-species prediction of plant promoter and enhancer regions\nAuthors: Kindel, F.; Triesch, S.; Schlueter, U.; Randarevitch, L. A.; Reichel-Deland, V.; Weber, A.; Denton, A. K.\nScore: 37.1, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565452\nMotivation: The identification of cis-regulatory elements (CREs) is crucial for the analysis of gene regulatory networks in plants. Several next generation sequencing (NGS)-based methods were developed to identify CREs. However, these methods can be time-consuming and costly. They also involve creating sequencing libraries for the entire genome. Since many research efforts only focus on specific genomic loci, this presents a considerable expenditure. Computational prediction of the outputs of specialized NGS methods to analyze CREs, like Assay for Transposase Accessible Chromatin using sequencing (ATAC-seq), would significantly cut costs and time investment. Yet, no such method is available to date. Results: We present Predmoter, a deep neural network able to predict base-wise ATAC-seq and histone Chromatin immunoprecipitation DNA-sequencing (ChIP-seq) read coverage for plant genomes. Predmoter uses only the DNA sequence as input. We evaluated our model on two plant genomes, the genome of the dicot Arabidopsis thaliana and of the monocot Oryza sativa. We trained our models on 10 species with publicly available ATAC-seq data and 15 species with ChIP-seq data. Our best models showed accurate predictions in peak positions and the overall pattern of peaks for ATAC- and Histone H3 trimethylated at lysine 4 (H3K4me3) ChIP-seq. Annotating putatively accessible chromatin regions provides valuable input for the identification of CREs. In conjunction with other in silico data, such as predicted binding affinities for transcription factors (TFs), this can significantly narrow down the search space to a manageable number of experimentally verifiable DNA-protein interaction pairs. Availability and Implementation: The source code for Predmoter is available at: https://github.com/weberlab-hhu/Predmoter along with documentation for installation and usage. Predmoter uses a single-command inference, Predmoter.py, for both training and prediction. Predmoter takes a fasta file as input and outputs an h5 file and optionally bigWig and bedGraph files.\nIntegrating single-cell RNA-seq datasets with substantial batch effects\nAuthors: Hrovatin, K.; Moinfar, A. A.; Tejada Lapuerta, A.; Zappia, L.; Lengerich, B.; Kellis, M.; Theis, F. J.\nScore: 31.0, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565463\nComputational methods for integrating scRNA-seq datasets often struggle to harmonize datasets with substantial differences driven by technical or biological variation, such as between different species, organoids and primary tissue, or different scRNA-seq protocols, including single-cell and single-nuclei. Given that many widely adopted and scalable methods are based on conditional variational autoencoders (cVAE), we hypothesize that machine learning interventions to standard cVAEs can help improve batch effect removal while potentially preserving biological variation more effectively. To address this, we assess four strategies applied to commonly used cVAE models: the previously proposed Kullback-Leibler divergence (KL) regularization tuning and adversarial learning, as well as cycle-consistency loss (previously applied to multi-omic integration) and the multimodal variational mixture of posteriors prior (VampPrior) that has not yet been applied to integration. We evaluated performance in three data settings, namely cross-species, organoid-tissue, and cell-nuclei integration. Cycle-consistency and VampPrior improved batch correction while retaining high biological preservation, with their combination further increasing performance. While adversarial learning led to the strongest batch correction, its preservation of within-cell type variation did not match that of VampPrior or cycle-consistency models, and it was also prone to mixing unrelated cell types with different proportions across batches. KL regularization strength tuning had the least favorable performance, as it jointly removed biological and batch variation by reducing the number of effectively used embedding dimensions. Based on our findings, we recommend the adoption of the VampPrior in combination with the cycle-consistency loss for integrating datasets with substantial batch effects.\ncisDynet: an integrated platform for modeling gene-regulatory dynamics and networks\nAuthors: Zhu, T.; Zhou, X.; You, Y.; Wang, L.; He, Z.; Chen, D.\nScore: 30.0, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564662\nChromatin accessibility sequencing has been widely used for uncovering genetic regulatory mechanisms and inferring gene regulatory networks. However, effectively integrating large-scale chromatin accessibility datasets has posed a significant challenge. This is due to the lack of a comprehensive end-to-end solution, as many existing tools primarily emphasize data pre-processing and overlook downstream analyses. To bridge this gap, we have introduced cisDynet, a holistic solution that combines streamlined data pre-processing using Snakemake and R functions with advanced downstream analysis capabilities. cisDynet excels in conventional data analyses, encompassing peak statistics, peak annotation, differential analysis, motif enrichment analysis and more. Additionally, it allows to perform sophisticated data exploration such as tissue-specific peak identification, time-course data fitting, integration of RNA-seq data to establish peak-to-gene associations, constructing regulatory networks, and conducting enrichment analysis of GWAS variants. As a proof of concept, we applied cisDynet to re-analyze the comprehensive ATAC-seq datasets across various tissues from the ENCODE project. The analysis successfully delineated tissue-specific open chromatin regions (OCRs), established connections between OCRs and target genes, and effectively linked these discoveries with 1,861 GWAS variants. Furthermore, cisDynet was instrumental in dissecting the time-course open chromatin data of mouse embryonic development, revealing the dynamic behavior of OCRs over time and identifying key transcription factors governing differentiation trajectories. In summary, cisDynet offers researchers a user-friendly solution that minimizes the need for extensive coding, ensures the reproducibility of results, and greatly simplifies the exploration of epigenomic data. Graphical AbstractcisDynet enables the exploration of the cis-regulatory chromatin dynamics and networks. It offers a streamlined preprocessing pipeline built on Snakemake, along with an R package for advanced downstream data analysis and visualization. It is free to access on GitHub (https://github.com/tzhu-bio/cisDynet). HighlightsO_LIThe cisDynet enables comprehensive and efficient processing of chromatin accessibility data, including pre-processing, advanced downstream data analysis and visualization. C_LIO_LIcisDynet provides a range of analytical features such as processing of time course data, co-accessibility analysis, linking OCRs to genes, building regulatory networks, and GWAS variant enrichment analysis. C_LIO_LIcisDynet simplifies the identification of tissue/cell type-specific OCRs or dynamic OCR changes over time and facilitates the integration of RNA-seq data to depict temporal trajectories. C_LI\nCoenzyme-Protein Interactions since Early Life\nAuthors: Sanchez Rocha, A. C.; Makarov, M.; Pravda, L.; Novotny, M.; Hlouchova, K.\nScore: 25.5, Published: 2023-11-01 DOI: 10.1101/2023.10.28.563965\nRecent findings in protein evolution and peptide prebiotic plausibility have been setting the stage for reconsidering the role of peptides in the early stages of lifes origin. Ancient protein families have been found to share common themes and proteins reduced in composition to prebiotically plausible amino acids have been reported capable of structure formation and key functions, such as binding to RNA. While this may suggest peptide relevance in early life, their functional repertoire when composed of a limited number of early residues (missing some of the most sophisticated functional groups of todays alphabet) has been debated. Cofactors enrich the functional scope of about half of extant enzymes but whether they could also bind to peptides lacking the evolutionary late amino acids remains speculative. The aim of this study was to resolve the early peptide propensity to bind organic cofactors by analysis of protein-coenzyme interactions across the Protein Data Bank (PDB). We find that the prebiotically plausible amino acids are more abundant in the binding sites of the most ancient coenzymes and that such interactions rely more frequently on the involvement of the protein backbone atoms and metal ion cofactors. Moreover, we have identified a few select examples in todays enzymes where coenzyme binding is supported solely by prebiotically available amino acids. These results imply the plausibility of a coenzyme-peptide functional collaboration preceding the establishment of the Central Dogma and full protein alphabet evolution.\nBenchmarking of small and large variants across tandem repeats\nAuthors: English, A.; Dolzhenko, E.; Ziaei-Jam, H.; Olson, N. D.; Mckenzie, S.; De Coster, W.; Park, J.; Gu, B.; Wagner, J.; Eberle, M.; Gymrek, M.; Chaisson, M.; Zook, J. M.; Sedlazeck, F. J.\nScore: 25.2, Published: 2023-11-01 DOI: 10.1101/2023.10.29.564632\nTandem repeats (TRs) are highly polymorphic in the human genome, have thousands of associated molecular traits, and are linked to over 60 disease phenotypes. However, their complexity often excludes them from at-scale studies due to challenges with variant calling, representation, and lack of a genome-wide standard. To promote TR methods development, we create a comprehensive catalog of TR regions and explore its properties across 86 samples. We then curate variants from the GIAB HG002 individual to create a tandem repeat benchmark. We also present a variant comparison method that handles small and large alleles and varying allelic representation. The 8.1% of the genome covered by the TR catalog holds [~]24.9% of variants per individual, including 124,728 small and 17,988 large variants for the GIAB HG002 TR benchmark. We work with the GIAB community to demonstrate the utility of this benchmark across short and long read technologies.\nTRAFICA: Improving Transcription Factor Binding Affinity Prediction using Large Language Model on ATAC-seq Data\nAuthors: Xu, Y.; Wang, C.; Xu, K.; Ding, Y.; Lyu, A.; Zhang, L.\nScore: 24.6, Published: 2023-11-05 DOI: 10.1101/2023.11.02.565416\nIn silico prediction of transcription factor (TF) binding affinity plays a vital role in determining TF binding preferences and understanding gene regulation. Many existing tools utilize known TF-DNA binding data to train conventional machine learning and deep learning models to predict TF-DNA binding affinities. The majority of these tools do not consider the natural preferences of DNA sequences binding to TFs, which could be influenced by chromatin accessibility. In this study, we developed TRAFICA, a deep language model to predict TF-DNA binding affinities by integrating chromatin accessibility from ATAC-seq and known TF-DNA binding data. We pre-trained TRAFICA on an integrated ATAC-seq dataset with over 13 million nucleotide sequences in a self-supervised manner to learn potential TF-DNA binding preferences and contextual relationships within DNA sequences. TRAFICA was fine-tuned using PBM and HT-SELEX datasets to predict in vitro TF-DNA binding affinities. We also implemented AdapterFusion to enhance its ability to predict in vivo TF-DNA binding affinities. We observed TRAFICA significantly outperformed the six existing tools in predicting in vitro TF-DNA binding affinities and was comparable with the best tool on in vivo prediction.\ndms-viz: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets\nAuthors: Hannon, W. W.; Bloom, J. D.\nScore: 22.6, Published: 2023-11-01 DOI: 10.1101/2023.10.29.564578\nSummary and PurposeUnderstanding how mutations impact a proteins functions is valuable for many types of biological questions. High-throughput techniques such as deep-mutational scanning (DMS) have greatly expanded the number of mutation-function datasets. For instance, DMS has been used to determine how mutations to viral proteins affect antibody escape (Dadonaite et al. 2023), receptor affinity (Starr et al. 2020), and essential functions such as viral genome transcription and replication (Li et al. 2023). With the growth of sequence databases, in some cases the effects of mutations can also be inferred from phylogenies of natural sequences (Bloom and Neher 2023) (Figure 1). O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=\"FIGDIR/small/564578v1_fig1.gif\" ALT=\"Figure 1\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@6c623forg.highwire.dtl.DTLVardef@1a62199org.highwire.dtl.DTLVardef@1e7d71eorg.highwire.dtl.DTLVardef@1dc1252_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure 1.C_FLOATNO Large mutation-associated datasets are used in a variety of experimental contexts. They can be used to map antibody footprints on viral glycoproteins, assess the impact of mutations on protein function in a laboratory setting, and identify patterns of selection from natural mutation frequencies. C_FIG The mutation-based data generated by these approaches is often best understood in the context of a proteins 3D structure; for instance, to assess questions like how mutations that affect antibody escape relate to the physical antibody binding epitope on the protein. However, current approaches for visualizing mutation data in the context of a proteins structure are often cumbersome and require multiple steps and softwares. To streamline the visualization of mutation-associated data in the context of a protein structure, we developed a web-based tool, dms-viz. With dms-viz, users can straightforwardly visualize mutation-based data such as those from DMS experiments in the context of a 3D protein model in an interactive format. See https://dms-viz.github.io/ to use dms-viz.\n",
  "wordCount" : "2778",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565471">
        <p class="paperTitle">Structure prediction of protein-ligand complexes from sequence information with Umol</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.</p>
        <p class="info">Score: 89.3, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565471' target='https://doi.org/10.1101/2023.11.03.565471'> 10.1101/2023.11.03.565471</a></p>
        <p class="abstract">Protein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand. At a high accuracy threshold, unseen protein-ligand complexes can be predicted more accurately than for RoseTTAFold-AA, and at medium accuracy even classical docking methods that use known protein structures as input are surpassed. The high accuracy presented here suggests that the goal of AI-based drug discovery is one step closer, but there is still a way to go to fully grasp the complexity of protein-ligand interactions. Umol is available at: https://github.com/patrickbryant1/Umol .</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.05.565674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.05.565674" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.05.565674">
        <p class="paperTitle">CellChat for systematic analysis of cell-cell communication from single-cell and spatially resolved transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.05.565674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.05.565674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, S.; Plikus, M. V.; Nie, Q.</p>
        <p class="info">Score: 48.5, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.05.565674' target='https://doi.org/10.1101/2023.11.05.565674'> 10.1101/2023.11.05.565674</a></p>
        <p class="abstract">Recent advances in single-cell sequencing technologies offer an opportunity to explore cell-cell communication in tissues systematically and with reduced bias. A key challenge is the integration between known molecular interactions and measurements into a framework to identify and analyze complex cell-cell communication networks. Previously, we developed a computational tool, named CellChat that infers and analyzes cell-cell communication networks from single-cell RNA-sequencing (scRNA-seq) data within an easily interpretable framework. CellChat quantifies the signaling communication probability between two cell groups using a simplified mass action-based model, which incorporates the core interaction between ligands and receptors with multi-subunit structure along with modulation by cofactors. CellChat v2 is an updated version that includes direct incorporation of spatial locations of cells, if available, to infer spatially proximal cell-cell communication, additional comparison functionalities, expanded database of ligand-receptor pairs along with rich annotations, and an Interactive CellChat Explorer. Here we provide a step-by-step protocol for using CellChat v2 that can be used for both scRNA-seq and spatially resolved transcriptomic data, including inference and analysis of cell-cell communication from one dataset and identification of altered signaling across different datasets. The key steps of applying CellChat v2 to spatially resolved transcriptomics are described in detail. The R implementation of CellChat v2 toolkit and tutorials with the graphic outputs are available at https://github.com/jinworks/CellChat. This protocol typically takes around 20 minutes, and no specialized prior bioinformatics training is required to complete the task.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564980">
        <p class="paperTitle">Identifying cell states in single-cell RNA-seq data at statistically maximal resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grobecker, P.; van Nimwegen, E.</p>
        <p class="info">Score: 45.5, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564980' target='https://doi.org/10.1101/2023.10.31.564980'> 10.1101/2023.10.31.564980</a></p>
        <p class="abstract">Single-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties. Although a vast number of different methods is applied in the literature for clustering cells into subsets with  similar expression profiles, these methods generally lack rigorously specified objectives, involve multiple complex layers of normalization, filtering, feature selection, dimensionalityreduction, employ ad hoc measures of distance or similarity between cells, often ignore the known measurement noise properties of scRNA-seq measurements, and include a large number of tunable parameters. Consequently, it is virtually impossible to assign concrete biophysical meaning to the clusterings that result from these methods.

Here we address the following problem: Given raw unique molecule identifier (UMI) counts of an scRNA-seq dataset, partition the cells into subsets such that the gene expression states of the cells in each subset are statistically indistinguishable, and each subset corresponds to a distinct gene expression state. That is, we aim to partition cells so as to maximally reduce the complexity of the dataset without removing any of its meaningful structure. We show that, given the known measurement noise structure of scRNA-seq data, this problem is mathematically well-defined and derive its unique solution from first principles. We have implemented this solution in a tool called CO_SCPLOWELLSTATESC_SCPLOW which operates directly on the raw data and automatically determines the optimal partition and cluster number, with zero tunable parameters.

We show that, on synthetic datasets, CO_SCPLOWELLSTATESC_SCPLOW almost perfectly recovers optimal partitions. On real data, CO_SCPLOWELLSTATESC_SCPLOW robustly identifies subtle substructure within groups of cells that are traditionally annotated as a common cell type. Moreover, we show that the diversity of gene expression states that CO_SCPLOWELLSTATESC_SCPLOW identifies systematically depends on the tissue of origin and not on technical features of the experiments such as the total number of cells and total UMI count per cell. In addition to the CO_SCPLOWELLSTATESC_SCPLOW tool we also provide a small toolbox of software to place the identified cellstates into a hierarchical tree of higher-order clusters, to identify the most important marker genes at each branch of this hierarchy, and to visualize these results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565452" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565452">
        <p class="paperTitle">Predmoter - Cross-species prediction of plant promoter and enhancer regions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kindel, F.; Triesch, S.; Schlueter, U.; Randarevitch, L. A.; Reichel-Deland, V.; Weber, A.; Denton, A. K.</p>
        <p class="info">Score: 37.1, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565452' target='https://doi.org/10.1101/2023.11.03.565452'> 10.1101/2023.11.03.565452</a></p>
        <p class="abstract">Motivation: The identification of cis-regulatory elements (CREs) is crucial for the analysis of gene regulatory networks in plants. Several next generation sequencing (NGS)-based methods were developed to identify CREs. However, these methods can be time-consuming and costly. They also involve creating sequencing libraries for the entire genome. Since many research efforts only focus on specific genomic loci, this presents a considerable expenditure. Computational prediction of the outputs of specialized NGS methods to analyze CREs, like Assay for Transposase Accessible Chromatin using sequencing (ATAC-seq), would significantly cut costs and time investment. Yet, no such method is available to date. Results: We present Predmoter, a deep neural network able to predict base-wise ATAC-seq and histone Chromatin immunoprecipitation DNA-sequencing (ChIP-seq) read coverage for plant genomes. Predmoter uses only the DNA sequence as input. We evaluated our model on two plant genomes, the genome of the dicot Arabidopsis thaliana and of the monocot Oryza sativa. We trained our models on 10 species with publicly available ATAC-seq data and 15 species with ChIP-seq data. Our best models showed accurate predictions in peak positions and the overall pattern of peaks for ATAC- and Histone H3 trimethylated at lysine 4 (H3K4me3) ChIP-seq. Annotating putatively accessible chromatin regions provides valuable input for the identification of CREs. In conjunction with other in silico data, such as predicted binding affinities for transcription factors (TFs), this can significantly narrow down the search space to a manageable number of experimentally verifiable DNA-protein interaction pairs. Availability and Implementation: The source code for Predmoter is available at: https://github.com/weberlab-hhu/Predmoter along with documentation for installation and usage. Predmoter uses a single-command inference, Predmoter.py, for both training and prediction. Predmoter takes a fasta file as input and outputs an h5 file and optionally bigWig and bedGraph files.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565463">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565463" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565463">
        <p class="paperTitle">Integrating single-cell RNA-seq datasets with substantial batch effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565463" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565463" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hrovatin, K.; Moinfar, A. A.; Tejada Lapuerta, A.; Zappia, L.; Lengerich, B.; Kellis, M.; Theis, F. J.</p>
        <p class="info">Score: 31.0, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565463' target='https://doi.org/10.1101/2023.11.03.565463'> 10.1101/2023.11.03.565463</a></p>
        <p class="abstract">Computational methods for integrating scRNA-seq datasets often struggle to harmonize datasets with substantial differences driven by technical or biological variation, such as between different species, organoids and primary tissue, or different scRNA-seq protocols, including single-cell and single-nuclei. Given that many widely adopted and scalable methods are based on conditional variational autoencoders (cVAE), we hypothesize that machine learning interventions to standard cVAEs can help improve batch effect removal while potentially preserving biological variation more effectively. To address this, we assess four strategies applied to commonly used cVAE models: the previously proposed Kullback-Leibler divergence (KL) regularization tuning and adversarial learning, as well as cycle-consistency loss (previously applied to multi-omic integration) and the multimodal variational mixture of posteriors prior (VampPrior) that has not yet been applied to integration. We evaluated performance in three data settings, namely cross-species, organoid-tissue, and cell-nuclei integration. Cycle-consistency and VampPrior improved batch correction while retaining high biological preservation, with their combination further increasing performance. While adversarial learning led to the strongest batch correction, its preservation of within-cell type variation did not match that of VampPrior or cycle-consistency models, and it was also prone to mixing unrelated cell types with different proportions across batches. KL regularization strength tuning had the least favorable performance, as it jointly removed biological and batch variation by reducing the number of effectively used embedding dimensions. Based on our findings, we recommend the adoption of the VampPrior in combination with the cycle-consistency loss for integrating datasets with substantial batch effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564662">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564662" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564662">
        <p class="paperTitle">cisDynet: an integrated platform for modeling gene-regulatory dynamics and networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564662" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564662" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, T.; Zhou, X.; You, Y.; Wang, L.; He, Z.; Chen, D.</p>
        <p class="info">Score: 30.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564662' target='https://doi.org/10.1101/2023.10.30.564662'> 10.1101/2023.10.30.564662</a></p>
        <p class="abstract">Chromatin accessibility sequencing has been widely used for uncovering genetic regulatory mechanisms and inferring gene regulatory networks. However, effectively integrating large-scale chromatin accessibility datasets has posed a significant challenge. This is due to the lack of a comprehensive end-to-end solution, as many existing tools primarily emphasize data pre-processing and overlook downstream analyses. To bridge this gap, we have introduced cisDynet, a holistic solution that combines streamlined data pre-processing using Snakemake and R functions with advanced downstream analysis capabilities. cisDynet excels in conventional data analyses, encompassing peak statistics, peak annotation, differential analysis, motif enrichment analysis and more. Additionally, it allows to perform sophisticated data exploration such as tissue-specific peak identification, time-course data fitting, integration of RNA-seq data to establish peak-to-gene associations, constructing regulatory networks, and conducting enrichment analysis of GWAS variants. As a proof of concept, we applied cisDynet to re-analyze the comprehensive ATAC-seq datasets across various tissues from the ENCODE project. The analysis successfully delineated tissue-specific open chromatin regions (OCRs), established connections between OCRs and target genes, and effectively linked these discoveries with 1,861 GWAS variants. Furthermore, cisDynet was instrumental in dissecting the time-course open chromatin data of mouse embryonic development, revealing the dynamic behavior of OCRs over time and identifying key transcription factors governing differentiation trajectories. In summary, cisDynet offers researchers a user-friendly solution that minimizes the need for extensive coding, ensures the reproducibility of results, and greatly simplifies the exploration of epigenomic data.

Graphical AbstractcisDynet enables the exploration of the cis-regulatory chromatin dynamics and networks. It offers a streamlined preprocessing pipeline built on Snakemake, along with an R package for advanced downstream data analysis and visualization. It is free to access on GitHub (https://github.com/tzhu-bio/cisDynet).

HighlightsO_LIThe cisDynet enables comprehensive and efficient processing of chromatin accessibility data, including pre-processing, advanced downstream data analysis and visualization.
C_LIO_LIcisDynet provides a range of analytical features such as processing of time course data, co-accessibility analysis, linking OCRs to genes, building regulatory networks, and GWAS variant enrichment analysis.
C_LIO_LIcisDynet simplifies the identification of tissue/cell type-specific OCRs or dynamic OCR changes over time and facilitates the integration of RNA-seq data to depict temporal trajectories.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.563965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.563965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.563965">
        <p class="paperTitle">Coenzyme-Protein Interactions since Early Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.563965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.563965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez Rocha, A. C.; Makarov, M.; Pravda, L.; Novotny, M.; Hlouchova, K.</p>
        <p class="info">Score: 25.5, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.563965' target='https://doi.org/10.1101/2023.10.28.563965'> 10.1101/2023.10.28.563965</a></p>
        <p class="abstract">Recent findings in protein evolution and peptide prebiotic plausibility have been setting the stage for reconsidering the role of peptides in the early stages of lifes origin. Ancient protein families have been found to share common themes and proteins reduced in composition to prebiotically plausible amino acids have been reported capable of structure formation and key functions, such as binding to RNA. While this may suggest peptide relevance in early life, their functional repertoire when composed of a limited number of early residues (missing some of the most sophisticated functional groups of todays alphabet) has been debated. Cofactors enrich the functional scope of about half of extant enzymes but whether they could also bind to peptides lacking the evolutionary late amino acids remains speculative. The aim of this study was to resolve the early peptide propensity to bind organic cofactors by analysis of protein-coenzyme interactions across the Protein Data Bank (PDB). We find that the prebiotically plausible amino acids are more abundant in the binding sites of the most ancient coenzymes and that such interactions rely more frequently on the involvement of the protein backbone atoms and metal ion cofactors. Moreover, we have identified a few select examples in todays enzymes where coenzyme binding is supported solely by prebiotically available amino acids. These results imply the plausibility of a coenzyme-peptide functional collaboration preceding the establishment of the Central Dogma and full protein alphabet evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564632">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564632" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564632">
        <p class="paperTitle">Benchmarking of small and large variants across tandem repeats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564632" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564632" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: English, A.; Dolzhenko, E.; Ziaei-Jam, H.; Olson, N. D.; Mckenzie, S.; De Coster, W.; Park, J.; Gu, B.; Wagner, J.; Eberle, M.; Gymrek, M.; Chaisson, M.; Zook, J. M.; Sedlazeck, F. J.</p>
        <p class="info">Score: 25.2, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564632' target='https://doi.org/10.1101/2023.10.29.564632'> 10.1101/2023.10.29.564632</a></p>
        <p class="abstract">Tandem repeats (TRs) are highly polymorphic in the human genome, have thousands of associated molecular traits, and are linked to over 60 disease phenotypes. However, their complexity often excludes them from at-scale studies due to challenges with variant calling, representation, and lack of a genome-wide standard. To promote TR methods development, we create a comprehensive catalog of TR regions and explore its properties across 86 samples. We then curate variants from the GIAB HG002 individual to create a tandem repeat benchmark. We also present a variant comparison method that handles small and large alleles and varying allelic representation. The 8.1% of the genome covered by the TR catalog holds [~]24.9% of variants per individual, including 124,728 small and 17,988 large variants for the GIAB HG002 TR benchmark. We work with the GIAB community to demonstrate the utility of this benchmark across short and long read technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565416">
        <p class="paperTitle">TRAFICA: Improving Transcription Factor Binding Affinity Prediction using Large Language Model on ATAC-seq Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Wang, C.; Xu, K.; Ding, Y.; Lyu, A.; Zhang, L.</p>
        <p class="info">Score: 24.6, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565416' target='https://doi.org/10.1101/2023.11.02.565416'> 10.1101/2023.11.02.565416</a></p>
        <p class="abstract">In silico prediction of transcription factor (TF) binding affinity plays a vital role in determining TF binding preferences and understanding gene regulation. Many existing tools utilize known TF-DNA binding data to train conventional machine learning and deep learning models to predict TF-DNA binding affinities. The majority of these tools do not consider the natural preferences of DNA sequences binding to TFs, which could be influenced by chromatin accessibility. In this study, we developed TRAFICA, a deep language model to predict TF-DNA binding affinities by integrating chromatin accessibility from ATAC-seq and known TF-DNA binding data. We pre-trained TRAFICA on an integrated ATAC-seq dataset with over 13 million nucleotide sequences in a self-supervised manner to learn potential TF-DNA binding preferences and contextual relationships within DNA sequences. TRAFICA was fine-tuned using PBM and HT-SELEX datasets to predict in vitro TF-DNA binding affinities. We also implemented AdapterFusion to enhance its ability to predict in vivo TF-DNA binding affinities. We observed TRAFICA significantly outperformed the six existing tools in predicting in vitro TF-DNA binding affinities and was comparable with the best tool on in vivo prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564578">
        <p class="paperTitle">dms-viz: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hannon, W. W.; Bloom, J. D.</p>
        <p class="info">Score: 22.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564578' target='https://doi.org/10.1101/2023.10.29.564578'> 10.1101/2023.10.29.564578</a></p>
        <p class="abstract">Summary and PurposeUnderstanding how mutations impact a proteins functions is valuable for many types of biological questions. High-throughput techniques such as deep-mutational scanning (DMS) have greatly expanded the number of mutation-function datasets. For instance, DMS has been used to determine how mutations to viral proteins affect antibody escape (Dadonaite et al. 2023), receptor affinity (Starr et al. 2020), and essential functions such as viral genome transcription and replication (Li et al. 2023). With the growth of sequence databases, in some cases the effects of mutations can also be inferred from phylogenies of natural sequences (Bloom and Neher 2023) (Figure 1).

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=&#34;FIGDIR/small/564578v1_fig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@6c623forg.highwire.dtl.DTLVardef@1a62199org.highwire.dtl.DTLVardef@1e7d71eorg.highwire.dtl.DTLVardef@1dc1252_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure 1.C_FLOATNO Large mutation-associated datasets are used in a variety of experimental contexts. They can be used to map antibody footprints on viral glycoproteins, assess the impact of mutations on protein function in a laboratory setting, and identify patterns of selection from natural mutation frequencies.

C_FIG The mutation-based data generated by these approaches is often best understood in the context of a proteins 3D structure; for instance, to assess questions like how mutations that affect antibody escape relate to the physical antibody binding epitope on the protein. However, current approaches for visualizing mutation data in the context of a proteins structure are often cumbersome and require multiple steps and softwares. To streamline the visualization of mutation-associated data in the context of a protein structure, we developed a web-based tool, dms-viz. With dms-viz, users can straightforwardly visualize mutation-based data such as those from DMS experiments in the context of a 3D protein model in an interactive format. See https://dms-viz.github.io/ to use dms-viz.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
