<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SSDraw: software for generating comparative protein secondary structure diagrams
Authors: Chen, E. A.; Porter, L. L.
Score: 53.5, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554905
The program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="SSDraw: software for generating comparative protein secondary structure diagrams
Authors: Chen, E. A.; Porter, L. L.
Score: 53.5, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554905
The program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="SSDraw: software for generating comparative protein secondary structure diagrams
Authors: Chen, E. A.; Porter, L. L.
Score: 53.5, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554905
The program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "SSDraw: software for generating comparative protein secondary structure diagrams\nAuthors: Chen, E. A.; Porter, L. L.\nScore: 53.5, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554905\nThe program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment.",
  "keywords": [
    
  ],
  "articleBody": " SSDraw: software for generating comparative protein secondary structure diagrams\nAuthors: Chen, E. A.; Porter, L. L.\nScore: 53.5, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554905\nThe program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment. Linear visualization allows the user to stack registered diagrams, facilitating comparison of secondary structure and other properties among homologous proteins. SSDraw can be used to compare secondary structures of homologous proteins with both conserved and divergent folds. It can also generate one secondary structure diagram from an input protein structure of interest. The source code can be downloaded and run locally for rapid structure generation, while a Google Colab notebook allows for easy use.\nExploring \"dark matter\" protein folds using deep learning\nAuthors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.\nScore: 38.0, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555621\nDe novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures. Here, we present an approach to learn patterns of protein structure based on a convolutional variational autoencoder, dubbed Genesis. We coupled Genesis with trRosetta to design sequences for a set of protein folds and found that Genesis is capable of reconstructing native-like distance- and angle distributions for five native folds and three novel, so-called dark-matter folds as a demonstration of generalizability. We used a high-throughput assay to characterize protease resistance of the designs, obtaining encouraging success rates for folded proteins and further biochemically characterized folded designs. The Genesis framework enables the exploration of the protein sequence and fold space within minutes and is not bound to specific protein topologies. Our approach addresses the backbone designability problem, showing that structural patterns in proteins can be efficiently learned by small neural networks and could ultimately contribute to the de novo design of proteins with new functions.\nHighfold: accurately predicting cyclic peptide monomers and complexes with AlphaFold\nAuthors: Zhang, C.; Zhang, C.; Shang, T.; Wu, X.; Duan, H.\nScore: 29.7, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554979\nIn recent years, cyclic peptides have gained growing traction as a therapeutic modality owing to their diverse biological activities. Understanding the structures of these cyclic peptides and their complexes can provide valuable insights. However, experimental observation needs much time and money, and there still are many limitations to CADD methods. As for DL-based models, the scarcity of training data poses a formidable challenge in predicting cyclic peptides and their complexes. In this work, we present \"High-fold,\" an AlphaFold-based algorithm that addresses this issue. By incorporating pertinent information about head-to-tailed circular and disulfide bridge structures, Highfold reaches the best performance in comparison to other various approaches. This model enables accurate prediction of cyclic peptides and their complexes, making a step to-wards resolving its structure-activity research.\npyScoMotif: Discovery of similar 3D structural motifs across proteins\nAuthors: Cia, G.; Kwasigroch, J. M.; Stamatopoulos, B.; Rooman, M.; Pucci, F.\nScore: 29.4, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554982\nMotivationThe fast and accurate detection of similar geometrical arrangements of protein residues, known as 3-dimensional (3D) structural motifs, is highly relevant for many applications such as binding region and catalytic site detection, drug discovery and structure conservation analyses. With the recent publication of new protein structure prediction methods, the number of available protein structures is exploding, which makes efficient and easy-to-use tools for identifying 3D structural motifs essential. ResultsWe present an open-source Python package that enables the search for both exact and mutated motifs with position-specific residue substitutions. The tool is fast, accurate and suitable to run both on computer clusters and personal laptops. Successful application of pyScoMotif to catalytic site identification is showcased. AvailabilityThe pyScoMotif package can be installed from the PyPI repository and is also available at https://github.com/3BioCompBio/pyScoMotif. It is free to use for non-commercial purposes; for commercial purposes, please contact us.\nFlow: a web platform and open database to analyse, store, curate and share bioinformatics data at scale\nAuthors: Capitanchik, C.; Ireland, S.; Harston, A.; Cheshire, C.; Jones, D. M.; Lee, F. C. Y.; Ruiz de los Mozos, I.; Iosub, I. A.; Kuret, K.; Faraway, R.; Wilkins, O. G.; Arora, R.; Hallegger, M.; Modic, M.; Chakrabarti, A. M.; Luscombe, N. M.; Ule, J.\nScore: 23.6, Published: 2023-08-29 DOI: 10.1101/2023.08.22.544179\nEver-increasing volumes of sequencing data offer potential for large-scale meta-analyses to address significant biological questions. However, challenges such as insufficient data processing information, data quality concerns, and issues related to accessibility and curation often present obstacles. Additionally, most experimental biologists lack the time and expertise needed to independently analyse, manage and share their own data. To overcome these hurdles, we present Flow, a web-based platform that links bioinformatic analysis and database solutions with a user-friendly interface and web API. Flow currently accommodates a range of genomics methods and further DSL2-compliant Nextflow pipelines can be added via a simple JSON schema file. Deployable on local systems or cloud services, an instance is freely accessible to academic researchers at https://flow.bio.\nSandy: A user-friendly and versatile NGS simulator to facilitate sequencing assay design and optimization\nAuthors: Miller, T. L. A.; Conceicao, H. B.; Mercuri, R. L.; Santos, F. R. C.; Barreiro, R.; Buzzo, J. L.; Rego, F. O.; Guardia, G. D. A.; Galante, P. A. F.\nScore: 22.1, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554791\nNext-generation sequencing (NGS) is currently the gold standard technique for large-scale genome and transcriptome studies. However, the downstream processing of NGS data is a critical bottleneck that requires difficult decisions regarding data analysis methods and parameters. Simulated or synthetic NGS datasets are practical and cost-effective alternatives for overcoming these difficulties. Simulated NGS datasets have known true values and provide a standardized scenario for driving the development of data analysis methodologies and tuning cut-off values. Although tools for simulating NGS data are available, they have limitations in terms of their overall usability and documentation. Here, we present Sandy, an open-source simulator that generates synthetic reads that mimic DNA or RNA next-generation sequencing on the Illumina, Oxford Nanopore, and Pacific Bioscience platforms. Sandy is designed to be user-friendly, computationally efficient, and capable of simulating data resembling a wide range of features of real NGS assays, including sequencing quality, genomic variations, and gene expression profiles per tissue. To demonstrate Sandys versatility, we used it to address two critical questions in designing an NGS assay: (i) How many reads should be sequenced to ensure unbiased analysis of gene expression in an RNA sequencing run? (ii) What is the lowest genome coverage required to identify most (90%) of the single nucleotide variants and structural variations in whole-genome sequencing? In summary, Sandy is an ideal tool for assessing and validating pipelines for processing, optimizing results, and defining the costs of NGS assays. Sandy runs on Linux, MacOS, and Microsoft Windows and can provide feasible results, even on personal computers. Availability: Sandy is freely available at https://galantelab.github.io/sandy.\nA multimodal Transformer Network for protein-small molecule interactions enhances drug-target affinity and enzyme-substrate predictions\nAuthors: Kroll, A.; Ranjan, S.; Lercher, M. J.\nScore: 33.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554147\nThe activities of most enzymes and drugs rely on interactions between proteins and small molecules. Accurate predictions of these interactions could accelerate pharmaceutical and biotechnological research massively. Machine learning models designed for this task are currently limited by the lack of information exchange between the protein and the small molecule during the creation of the required numerical representations. Here, we introduce ProSmith, a machine learning framework that employs a multimodal Transformer Network to simultaneously process protein amino acid sequences and small molecule strings in the same input. This approach facilitates the exchange of all relevant information between the two molecule types during the calculation of their numerical representations, allowing the model to account for their structural and functional interactions. Our final model combines gradient boosting predictions based on the resulting multimodal Transformer Network with independent predictions based on separate deep learning representations of the proteins and small molecules. The corresponding predictions outperform all previous models for predicting protein-small molecule interactions across three diverse tasks: predicting Michaelis constants KM; inferring potential substrates for enzymes; and predicting protein-drug affinities. The Python code provided can be used to easily implement and improve machine learning predictions involving arbitrary protein-small molecule interactions.\nHigh-accuracy mapping of human and viral direct physical protein-protein interactions using the novel computational system AlphaFold-pairs\nAuthors: Poitras, C.; Lamontagne, F.; Grandvaux, N.; Song, H.; Pinard, M.; Coulombe, B.\nScore: 15.9, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555151\nProtein-protein interactions are central, highly flexible components of regulatory mechanisms in all living cells. Over the years, diverse methods have been developed to map protein-protein interactions. These methods have revealed the organization of protein complexes and networks in numerous cells and conditions. However, these methods are also time consuming, costly and sensitive to various experimental artifacts. To avoid these caveats, we have taken advantage of the AlphaFold-Multimer software, which succeeded in predicting the structure of many protein complexes. We designed a relatively simple algorithm based on assessing the physical proximity of a test protein with other AlphaFold structures. Using this method, named AlphaFold-pairs, we have successfully defined the probability of a protein-protein interaction forming. AlphaFold-pairs was validated using well-defined protein-protein interactions found in the literature and specialized databases. All pairwise interactions forming within the 12-subunit transcription machinery RNA Polymerase II, according to available structures, have been identified. Out of 66 possible interactions (excluding homodimers), 19 specific interactions have been found, and an additional previously unknown interaction has been unveiled. The SARS-CoV-2 surface glycoprotein Spike (or S) was confirmed to interact with high preference with the human ACE2 receptor when compared to other human receptors. Notably, two additional receptors, INSR and FLT4, were found to interact with S. For the first time, we have successfully identified protein-protein interactions that are likely to form within the reassortant Eurasian avian-like (EA) H1N1 swine G4 genotype Influenza A virus, which poses a potential zoonotic threat. Testing G4 proteins against human transcription factors and molecular chaperones (a total of 100 proteins) revealed strong specific interactions between the G4 HA and HSP90B1, the G4 NS and the PAQosome subunit RPAP3, as well as the G4 PA and the POLR2A subunit. We predict that AlphaFold-pairs will revolutionize the study of protein-protein interactions in a large number of healthy and diseased systems in the years to come.\nScalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages\nAuthors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Corrada Bravo, H.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.\nScore: 160.6, Published: 2023-08-26 DOI: 10.1101/2023.07.18.549537\nSingle-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date. A singular view of this vast and growing expression landscape could help reveal novel associations between cell states and diseases, discover cell states in unexpected tissue contexts, and relate in vivo cells to in vitro models. However, these require a common, scalable representation of cell profiles from across the body, a general measure of their similarity, and an efficient way to query these data. Here, we present SCimilarity, a metric learning framework to learn and search a unified and interpretable representation that annotates cell types and instantaneously queries for a cell state across tens of millions of profiles. We demonstrate SCimilarity on a 22.7 million cell corpus assembled across 399 published scRNA-seq studies, showing accurate integration, annotation and querying. We experimentally validated SCimilarity by querying across tissues for a macrophage subset originally identified in interstitial lung disease, and showing that cells with similar profiles are found in other fibrotic diseases, tissues, and a 3D hydrogel system, which we then repurposed to yield this cell state in vitro. SCimilarity serves as a foundational model for single cell gene expression data and enables researchers to query for similar cellular states across the entire human body, providing a powerful tool for generating novel biological insights from the growing Human Cell Atlas.\nTCCIA: A Comprehensive Resource for Exploring CircRNA in Cancer Immunotherapy\nAuthors: Wang, S.; Xiong, Y.; Zhang, Y.; Wang, H.; Chen, M.; Li, J.; Luo, P.; Luo, Y.-H.; Hecht, M.; Frey, B.; Gaipl, U. S.; Li, X.; Zhao, Q.; Ma, H.; Zhou, J.-G.\nScore: 16.6, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554049\nBackgroundImmunotherapies targeting immune checkpoints have gained increasing attention in cancer treatment, emphasizing the need for predictive biomarkers. Circular RNAs (circRNAs) have emerged as critical regulators of tumor immunity, particularly in the PD- 1/PD-L1 pathway, and have shown potential in predicting the efficacy of immunotherapies. However, the precise roles of circRNAs in cancer immunotherapy remain incompletely understood. While existing databases focus on either circRNA profiles or immunotherapy cohorts, there is currently no platform that enables the exploration of the intricate interplay between circRNAs and anti-tumor immunotherapy. Therefore, the development of a comprehensive resource that integrates circRNA profiles, immunotherapy response data, and clinical benefits is crucial for advancing our understanding of circRNA-mediated tumor-immune interactions and developing effective immunotherapy biomarkers. MethodsTo address these gaps, we constructed the Cancer CircRNA Immunome Atlas (TCCIA), the first database that combines circRNA profiles, immunotherapy response data, and clinical outcomes across multi-cancer types. The construction of TCCIA involved applying standardized preprocessing to the raw sequencing FASTQ files, characterizing circRNA profiles using CIRCexplorer2, analyzing tumor immunophenotypes through IOBR, and compiling immunotherapy response data from diverse cohorts treated with immune-checkpoint blockades (ICBs). ResultsTCCIA encompasses over 3,700 clinical samples obtained from 18 cohorts treated with ICBs, including PD-1/PD-L1 and CTLA-4 inhibitors, along with other treatment modalities. The database provides researchers and clinicians with a cloud-based platform that enables interactive exploration of circRNA data in the context of ICB. The platform offers a range of analytical tools, including visualization of circRNA abundance and correlation, association analysis between circRNAs and clinical variables, assessment of the tumor immune microenvironment, exploration of tumor molecular signatures, evaluation of treatment response or prognosis, and identification of altered circRNAs in immunotherapy-sensitive and resistant tumors. To illustrate the utility of TCCIA, we performed a re-analysis on a melanoma cohort with TCCIA, and found that an isoform of circTMTC3, TMTC3:+:chr12:88148287:88176319, played a significant role in predicting unfavorable survival outcomes and treatment nonresponse. ConclusionsTCCIA represents a significant advancement over existing resources, providing a comprehensive platform to investigate the role of circRNAs in immune oncology. What is already known on this topicPrior knowledge indicated that circRNAs are involved in tumor immunity and have potential as predictive biomarkers for immunotherapy efficacy. However, there lacked a comprehensive database that integrated circRNA profiles and immunotherapy response data, necessitating this study. What this study addsThis study introduces TCCIA, a database that combines circRNA profiles, immunotherapy response data, and clinical outcomes. It provides a diverse collection of clinical samples and an interactive platform, enabling in-depth exploration of circRNAs in the context of checkpoint-blockade immunotherapy. How this study might affect research, practice or policyThe findings of this study offer valuable insights into the roles of circRNAs in tumor-immune interactions and provide a resource for researchers and clinicians in the field of immune-oncology. TCCIA has the potential to guide personalized immunotherapeutic strategies and contribute to future research, clinical practice, and policy decisions in checkpoint-blockade immunotherapy and biomarker development.\n",
  "wordCount" : "2571",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554905">
        <p class="paperTitle">SSDraw: software for generating comparative protein secondary structure diagrams</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, E. A.; Porter, L. L.</p>
        <p class="info">Score: 53.5, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554905' target='https://doi.org/10.1101/2023.08.25.554905'> 10.1101/2023.08.25.554905</a></p>
        <p class="abstract">The program SSDraw generates publication-quality protein secondary structure diagrams from three-dimensional protein structures. To depict relationships between secondary structure and other protein features, diagrams can be colored by conservation score, B-factor, or custom scoring. Diagrams of homologous proteins can be registered according to an input multiple sequence alignment. Linear visualization allows the user to stack registered diagrams, facilitating comparison of secondary structure and other properties among homologous proteins. SSDraw can be used to compare secondary structures of homologous proteins with both conserved and divergent folds. It can also generate one secondary structure diagram from an input protein structure of interest. The source code can be downloaded and run locally for rapid structure generation, while a Google Colab notebook allows for easy use.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555621">
        <p class="paperTitle">Exploring &#34;dark matter&#34; protein folds using deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.</p>
        <p class="info">Score: 38.0, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555621' target='https://doi.org/10.1101/2023.08.30.555621'> 10.1101/2023.08.30.555621</a></p>
        <p class="abstract">De novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures. Here, we present an approach to learn patterns of protein structure based on a convolutional variational autoencoder, dubbed Genesis. We coupled Genesis with trRosetta to design sequences for a set of protein folds and found that Genesis is capable of reconstructing native-like distance- and angle distributions for five native folds and three novel, so-called dark-matter folds as a demonstration of generalizability. We used a high-throughput assay to characterize protease resistance of the designs, obtaining encouraging success rates for folded proteins and further biochemically characterized folded designs. The Genesis framework enables the exploration of the protein sequence and fold space within minutes and is not bound to specific protein topologies. Our approach addresses the backbone designability problem, showing that structural patterns in proteins can be efficiently learned by small neural networks and could ultimately contribute to the de novo design of proteins with new functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554979">
        <p class="paperTitle">Highfold: accurately predicting cyclic peptide monomers and complexes with AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, C.; Zhang, C.; Shang, T.; Wu, X.; Duan, H.</p>
        <p class="info">Score: 29.7, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554979' target='https://doi.org/10.1101/2023.08.27.554979'> 10.1101/2023.08.27.554979</a></p>
        <p class="abstract">In recent years, cyclic peptides have gained growing traction as a therapeutic modality owing to their diverse biological activities. Understanding the structures of these cyclic peptides and their complexes can provide valuable insights. However, experimental observation needs much time and money, and there still are many limitations to CADD methods. As for DL-based models, the scarcity of training data poses a formidable challenge in predicting cyclic peptides and their complexes. In this work, we present &#34;High-fold,&#34; an AlphaFold-based algorithm that addresses this issue. By incorporating pertinent information about head-to-tailed circular and disulfide bridge structures, Highfold reaches the best performance in comparison to other various approaches. This model enables accurate prediction of cyclic peptides and their complexes, making a step to-wards resolving its structure-activity research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554982" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554982">
        <p class="paperTitle">pyScoMotif: Discovery of similar 3D structural motifs across proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cia, G.; Kwasigroch, J. M.; Stamatopoulos, B.; Rooman, M.; Pucci, F.</p>
        <p class="info">Score: 29.4, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554982' target='https://doi.org/10.1101/2023.08.27.554982'> 10.1101/2023.08.27.554982</a></p>
        <p class="abstract">MotivationThe fast and accurate detection of similar geometrical arrangements of protein residues, known as 3-dimensional (3D) structural motifs, is highly relevant for many applications such as binding region and catalytic site detection, drug discovery and structure conservation analyses. With the recent publication of new protein structure prediction methods, the number of available protein structures is exploding, which makes efficient and easy-to-use tools for identifying 3D structural motifs essential.

ResultsWe present an open-source Python package that enables the search for both exact and mutated motifs with position-specific residue substitutions. The tool is fast, accurate and suitable to run both on computer clusters and personal laptops. Successful application of pyScoMotif to catalytic site identification is showcased.

AvailabilityThe pyScoMotif package can be installed from the PyPI repository and is also available at https://github.com/3BioCompBio/pyScoMotif. It is free to use for non-commercial purposes; for commercial purposes, please contact us.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.544179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.544179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.544179">
        <p class="paperTitle">Flow: a web platform and open database to analyse, store, curate and share bioinformatics data at scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.544179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.544179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Capitanchik, C.; Ireland, S.; Harston, A.; Cheshire, C.; Jones, D. M.; Lee, F. C. Y.; Ruiz de los Mozos, I.; Iosub, I. A.; Kuret, K.; Faraway, R.; Wilkins, O. G.; Arora, R.; Hallegger, M.; Modic, M.; Chakrabarti, A. M.; Luscombe, N. M.; Ule, J.</p>
        <p class="info">Score: 23.6, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.544179' target='https://doi.org/10.1101/2023.08.22.544179'> 10.1101/2023.08.22.544179</a></p>
        <p class="abstract">Ever-increasing volumes of sequencing data offer potential for large-scale meta-analyses to address significant biological questions. However, challenges such as insufficient data processing information, data quality concerns, and issues related to accessibility and curation often present obstacles. Additionally, most experimental biologists lack the time and expertise needed to independently analyse, manage and share their own data. To overcome these hurdles, we present Flow, a web-based platform that links bioinformatic analysis and database solutions with a user-friendly interface and web API. Flow currently accommodates a range of genomics methods and further DSL2-compliant Nextflow pipelines can be added via a simple JSON schema file. Deployable on local systems or cloud services, an instance is freely accessible to academic researchers at https://flow.bio.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554791">
        <p class="paperTitle">Sandy: A user-friendly and versatile NGS simulator to facilitate sequencing assay design and optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, T. L. A.; Conceicao, H. B.; Mercuri, R. L.; Santos, F. R. C.; Barreiro, R.; Buzzo, J. L.; Rego, F. O.; Guardia, G. D. A.; Galante, P. A. F.</p>
        <p class="info">Score: 22.1, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554791' target='https://doi.org/10.1101/2023.08.25.554791'> 10.1101/2023.08.25.554791</a></p>
        <p class="abstract">Next-generation sequencing (NGS) is currently the gold standard technique for large-scale genome and transcriptome studies. However, the downstream processing of NGS data is a critical bottleneck that requires difficult decisions regarding data analysis methods and parameters. Simulated or synthetic NGS datasets are practical and cost-effective alternatives for overcoming these difficulties. Simulated NGS datasets have known true values and provide a standardized scenario for driving the development of data analysis methodologies and tuning cut-off values. Although tools for simulating NGS data are available, they have limitations in terms of their overall usability and documentation. Here, we present Sandy, an open-source simulator that generates synthetic reads that mimic DNA or RNA next-generation sequencing on the Illumina, Oxford Nanopore, and Pacific Bioscience platforms. Sandy is designed to be user-friendly, computationally efficient, and capable of simulating data resembling a wide range of features of real NGS assays, including sequencing quality, genomic variations, and gene expression profiles per tissue. To demonstrate Sandys versatility, we used it to address two critical questions in designing an NGS assay: (i) How many reads should be sequenced to ensure unbiased analysis of gene expression in an RNA sequencing run? (ii) What is the lowest genome coverage required to identify most (90%) of the single nucleotide variants and structural variations in whole-genome sequencing? In summary, Sandy is an ideal tool for assessing and validating pipelines for processing, optimizing results, and defining the costs of NGS assays. Sandy runs on Linux, MacOS, and Microsoft Windows and can provide feasible results, even on personal computers. Availability: Sandy is freely available at https://galantelab.github.io/sandy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554147">
        <p class="paperTitle">A multimodal Transformer Network for protein-small molecule interactions enhances drug-target affinity and enzyme-substrate predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kroll, A.; Ranjan, S.; Lercher, M. J.</p>
        <p class="info">Score: 33.6, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554147' target='https://doi.org/10.1101/2023.08.21.554147'> 10.1101/2023.08.21.554147</a></p>
        <p class="abstract">The activities of most enzymes and drugs rely on interactions between proteins and small molecules. Accurate predictions of these interactions could accelerate pharmaceutical and biotechnological research massively. Machine learning models designed for this task are currently limited by the lack of information exchange between the protein and the small molecule during the creation of the required numerical representations. Here, we introduce ProSmith, a machine learning framework that employs a multimodal Transformer Network to simultaneously process protein amino acid sequences and small molecule strings in the same input. This approach facilitates the exchange of all relevant information between the two molecule types during the calculation of their numerical representations, allowing the model to account for their structural and functional interactions. Our final model combines gradient boosting predictions based on the resulting multimodal Transformer Network with independent predictions based on separate deep learning representations of the proteins and small molecules. The corresponding predictions outperform all previous models for predicting protein-small molecule interactions across three diverse tasks: predicting Michaelis constants KM; inferring potential substrates for enzymes; and predicting protein-drug affinities. The Python code provided can be used to easily implement and improve machine learning predictions involving arbitrary protein-small molecule interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555151">
        <p class="paperTitle">High-accuracy mapping of human and viral direct physical protein-protein interactions using the novel computational system AlphaFold-pairs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poitras, C.; Lamontagne, F.; Grandvaux, N.; Song, H.; Pinard, M.; Coulombe, B.</p>
        <p class="info">Score: 15.9, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555151' target='https://doi.org/10.1101/2023.08.29.555151'> 10.1101/2023.08.29.555151</a></p>
        <p class="abstract">Protein-protein interactions are central, highly flexible components of regulatory mechanisms in all living cells. Over the years, diverse methods have been developed to map protein-protein interactions. These methods have revealed the organization of protein complexes and networks in numerous cells and conditions. However, these methods are also time consuming, costly and sensitive to various experimental artifacts. To avoid these caveats, we have taken advantage of the AlphaFold-Multimer software, which succeeded in predicting the structure of many protein complexes. We designed a relatively simple algorithm based on assessing the physical proximity of a test protein with other AlphaFold structures. Using this method, named AlphaFold-pairs, we have successfully defined the probability of a protein-protein interaction forming. AlphaFold-pairs was validated using well-defined protein-protein interactions found in the literature and specialized databases. All pairwise interactions forming within the 12-subunit transcription machinery RNA Polymerase II, according to available structures, have been identified. Out of 66 possible interactions (excluding homodimers), 19 specific interactions have been found, and an additional previously unknown interaction has been unveiled. The SARS-CoV-2 surface glycoprotein Spike (or S) was confirmed to interact with high preference with the human ACE2 receptor when compared to other human receptors. Notably, two additional receptors, INSR and FLT4, were found to interact with S. For the first time, we have successfully identified protein-protein interactions that are likely to form within the reassortant Eurasian avian-like (EA) H1N1 swine G4 genotype Influenza A virus, which poses a potential zoonotic threat. Testing G4 proteins against human transcription factors and molecular chaperones (a total of 100 proteins) revealed strong specific interactions between the G4 HA and HSP90B1, the G4 NS and the PAQosome subunit RPAP3, as well as the G4 PA and the POLR2A subunit. We predict that AlphaFold-pairs will revolutionize the study of protein-protein interactions in a large number of healthy and diseased systems in the years to come.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549537">
        <p class="paperTitle">Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heimberg, G.; Kuo, T. C.; DePianto, D.; Heigl, T.; Diamant, N.; Salem, O.; Scalia, G.; Biancalani, T.; Rock, J.; Turley, S.; Corrada Bravo, H.; Kaminker, J.; Vander Heiden, J. A.; Regev, A.</p>
        <p class="info">Score: 160.6, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549537' target='https://doi.org/10.1101/2023.07.18.549537'> 10.1101/2023.07.18.549537</a></p>
        <p class="abstract">Single-cell RNA-seq (scRNA-seq) studies have profiled over 100 million human cells across diseases, developmental stages, and perturbations to date. A singular view of this vast and growing expression landscape could help reveal novel associations between cell states and diseases, discover cell states in unexpected tissue contexts, and relate in vivo cells to in vitro models. However, these require a common, scalable representation of cell profiles from across the body, a general measure of their similarity, and an efficient way to query these data. Here, we present SCimilarity, a metric learning framework to learn and search a unified and interpretable representation that annotates cell types and instantaneously queries for a cell state across tens of millions of profiles. We demonstrate SCimilarity on a 22.7 million cell corpus assembled across 399 published scRNA-seq studies, showing accurate integration, annotation and querying. We experimentally validated SCimilarity by querying across tissues for a macrophage subset originally identified in interstitial lung disease, and showing that cells with similar profiles are found in other fibrotic diseases, tissues, and a 3D hydrogel system, which we then repurposed to yield this cell state in vitro. SCimilarity serves as a foundational model for single cell gene expression data and enables researchers to query for similar cellular states across the entire human body, providing a powerful tool for generating novel biological insights from the growing Human Cell Atlas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554049">
        <p class="paperTitle">TCCIA: A Comprehensive Resource for Exploring CircRNA in Cancer Immunotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, S.; Xiong, Y.; Zhang, Y.; Wang, H.; Chen, M.; Li, J.; Luo, P.; Luo, Y.-H.; Hecht, M.; Frey, B.; Gaipl, U. S.; Li, X.; Zhao, Q.; Ma, H.; Zhou, J.-G.</p>
        <p class="info">Score: 16.6, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554049' target='https://doi.org/10.1101/2023.08.24.554049'> 10.1101/2023.08.24.554049</a></p>
        <p class="abstract">BackgroundImmunotherapies targeting immune checkpoints have gained increasing attention in cancer treatment, emphasizing the need for predictive biomarkers. Circular RNAs (circRNAs) have emerged as critical regulators of tumor immunity, particularly in the PD- 1/PD-L1 pathway, and have shown potential in predicting the efficacy of immunotherapies. However, the precise roles of circRNAs in cancer immunotherapy remain incompletely understood. While existing databases focus on either circRNA profiles or immunotherapy cohorts, there is currently no platform that enables the exploration of the intricate interplay between circRNAs and anti-tumor immunotherapy. Therefore, the development of a comprehensive resource that integrates circRNA profiles, immunotherapy response data, and clinical benefits is crucial for advancing our understanding of circRNA-mediated tumor-immune interactions and developing effective immunotherapy biomarkers.

MethodsTo address these gaps, we constructed the Cancer CircRNA Immunome Atlas (TCCIA), the first database that combines circRNA profiles, immunotherapy response data, and clinical outcomes across multi-cancer types. The construction of TCCIA involved applying standardized preprocessing to the raw sequencing FASTQ files, characterizing circRNA profiles using CIRCexplorer2, analyzing tumor immunophenotypes through IOBR, and compiling immunotherapy response data from diverse cohorts treated with immune-checkpoint blockades (ICBs).

ResultsTCCIA encompasses over 3,700 clinical samples obtained from 18 cohorts treated with ICBs, including PD-1/PD-L1 and CTLA-4 inhibitors, along with other treatment modalities. The database provides researchers and clinicians with a cloud-based platform that enables interactive exploration of circRNA data in the context of ICB. The platform offers a range of analytical tools, including visualization of circRNA abundance and correlation, association analysis between circRNAs and clinical variables, assessment of the tumor immune microenvironment, exploration of tumor molecular signatures, evaluation of treatment response or prognosis, and identification of altered circRNAs in immunotherapy-sensitive and resistant tumors. To illustrate the utility of TCCIA, we performed a re-analysis on a melanoma cohort with TCCIA, and found that an isoform of circTMTC3, TMTC3:&#43;:chr12:88148287:88176319, played a significant role in predicting unfavorable survival outcomes and treatment nonresponse.

ConclusionsTCCIA represents a significant advancement over existing resources, providing a comprehensive platform to investigate the role of circRNAs in immune oncology.

What is already known on this topicPrior knowledge indicated that circRNAs are involved in tumor immunity and have potential as predictive biomarkers for immunotherapy efficacy. However, there lacked a comprehensive database that integrated circRNA profiles and immunotherapy response data, necessitating this study.

What this study addsThis study introduces TCCIA, a database that combines circRNA profiles, immunotherapy response data, and clinical outcomes. It provides a diverse collection of clinical samples and an interactive platform, enabling in-depth exploration of circRNAs in the context of checkpoint-blockade immunotherapy.

How this study might affect research, practice or policyThe findings of this study offer valuable insights into the roles of circRNAs in tumor-immune interactions and provide a resource for researchers and clinicians in the field of immune-oncology. TCCIA has the potential to guide personalized immunotherapeutic strategies and contribute to future research, clinical practice, and policy decisions in checkpoint-blockade immunotherapy and biomarker development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
