<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Assessing the limits of zero-shot foundation models in single-cell biology
Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.
Score: 79.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085
The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Assessing the limits of zero-shot foundation models in single-cell biology
Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.
Score: 79.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085
The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-25T10:39:04+00:00" />
<meta property="article:modified_time" content="2023-10-25T10:39:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Assessing the limits of zero-shot foundation models in single-cell biology
Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.
Score: 79.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085
The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Assessing the limits of zero-shot foundation models in single-cell biology\nAuthors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.\nScore: 79.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085\nThe advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.",
  "keywords": [
    
  ],
  "articleBody": " Assessing the limits of zero-shot foundation models in single-cell biology\nAuthors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.\nScore: 79.6, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085\nThe advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2\nAF2BIND: Predicting ligand-binding sites using the pair representation of AlphaFold2\nAuthors: Gazizov, A.; Lian, A.; Goverde, C. A.; Ovchinnikov, S.; Polizzi, N. F.\nScore: 46.1, Published: 2023-10-18 DOI: 10.1101/2023.10.15.562410\nPredicting ligand-binding sites, particularly in the absence of previously resolved homologous structures, presents a significant challenge in structural biology. Here, we leverage the internal pairwise representation of AlphaFold2 (AF2) to train a model, AF2BIND, to accurately predict small-molecule-binding residues given only a target protein. AF2BIND uses 20 \"bait\" amino acids to optimally extract the binding signal in the absence of a small-molecule ligand. We find that the AF2 pair representation outperforms other neural-network representations for binding-site prediction. Moreover, unique combinations of the 20 bait amino acids are correlated with chemical properties of the ligand.\nCellTracksColab, A platform for compiling, analyzing, and exploring tracking data\nAuthors: Jacquemet, G.\nScore: 36.0, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563252\nIn life sciences, tracking objects from movies enables researchers to quantify the behavior of single particles, organelles, bacteria, cells, and even whole animals. While numerous tools now allow automated tracking from video, a significant challenge persists in compiling, analyzing, and exploring the large datasets generated by these approaches. Here, we introduce CellTracksColab, an online platform tailored to simplify the exploration and analysis of tracking data by leveraging the free, cloud-based computational resources of Google Colab. CellTracksColab facilitates the amalgamation and analysis of tracking results across multiple fields of view, conditions, and repeats, ensuring a holistic dataset overview. CellTracksColab also harnesses the power of high-dimensional data reduction and clustering, using Uniform Manifold Approximation and Projection and Hierarchical Density-Based Spatial Clustering of Applications with Noise, enabling researchers to identify distinct behavioral patterns and trends without bias. CellTracksColab is available for the broader scientific community at https://github.com/guijacquemet/CellTracksColab.\nGENEPT: A SIMPLE BUT HARD-TO-BEAT FOUNDATION MODEL FOR GENES AND CELLS BUILT FROM CHATGPT\nAuthors: Chen, Y. T.; Zou, J.\nScore: 29.4, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562533\nAO_SCPLOWBSTRACTC_SCPLOWThere has been significant recent progress in leveraging large-scale gene expression data to develop foundation models for single-cell transcriptomes such as Geneformer [1], scGPT [2], and scBERT [3]. These models infer gene functions and interrelations from the gene expression profiles of millions of cells, which requires extensive data curation and resource-intensive training. Here, we explore a much simpler alternative by leveraging ChatGPT embeddings of genes based on literature. Our proposal, GenePT, uses NCBI text descriptions of individual genes with GPT-3.5 to generate gene embeddings. From there, GenePT generates single-cell embeddings in two ways: (i) by averaging the gene embeddings, weighted by each genes expression level; or (ii) by creating a sentence embedding for each cell, using gene names ordered by the expression level. Without the need for dataset curation and additional pretraining, GenePT is efficient and easy to use. On many downstream tasks used to evaluate recent single-cell foundation models -- e.g., classifying gene properties and cell types -- GenePT achieves comparable, and often better, performance than Geneformer and other methods. GenePT demonstrates that large language model embedding of literature is a simple and effective path for biological foundation models.\nA deep learning model embedded framework to distinguish DNA and RNA mutations directly from RNA-seq\nAuthors: Fu, Z.-C.; Gao, B.-Q.; Nan, F.; Ma, X.-K.; Yang, L.\nScore: 28.6, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562625\nWe develop a stepwise computational framework, called DEMINING, to directly detect expressed DNA and RNA mutations in RNA deep sequencing data. DEMINING incorporates a deep learning model named DeepDDR, which facilitates the separation of expressed DNA mutations from RNA mutations after RNA-seq read mapping and pileup. When applied in RNA-seq of acute myeloid leukemia patients, DEMINING uncovered previously-underappreciated DNA and RNA mutations, some associated with the upregulated expression of host genes or the production of neoantigens. Finally, we demonstrate that DEMINING could precisely classify DNA and RNA mutations in RNA-seq data from non-primate species through the utilization of transfer learning.\nOMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records\nAuthors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J. A.\nScore: 27.4, Published: 2023-10-24 DOI: 10.1101/2023.10.16.562114\nBiological aging is a multifactorial process involving complex interactions of cellular and biochemical processes that is reflected in omic profiles. Using common clinical laboratory measures in [~]30,000 individuals from the MGB-Biobank, we developed a robust, predictive biological aging phenotype, EMRAge, that balances clinical biomarkers with overall mortality risk and can be broadly recapitulated across EMRs. We then applied elastic-net regression to model EMRAge with DNA-methylation (DNAm) and multiple omics, generating DNAmEMRAge and OMICmAge, respectively. Both biomarkers demonstrated strong associations with chronic diseases and mortality that outperform current biomarkers across our discovery (MGB-ABC, n=3,451) and validation (TruDiagnostic, n=12,666) cohorts. Through the use of epigenetic biomarker proxies, OMICmAge has the unique advantage of expanding the predictive search space to include epigenomic, proteomic, metabolomic, and clinical data while distilling this in a measure with DNAm alone, providing opportunities to identify clinically-relevant interconnections central to the aging process.\nAccurate model and ensemble refinement using cryo-electron microscopy maps and Bayesian inference\nAuthors: Hoff, S. E.; Thomasen, F. E.; Lindorff-Larsen, K.; Bonomi, M.\nScore: 24.5, Published: 2023-10-19 DOI: 10.1101/2023.10.18.562710\nConverting cryo-electron microscopy (cryo-EM) data into high-quality structural models is a challenging problem of outstanding importance. Current refinement methods often generate unbalanced models in which physico-chemical quality is sacrificed for excellent fit to the data. Furthermore, these techniques struggle to represent the conformational heterogeneity averaged out in low-resolution regions of density maps. Here we introduce EMMIVox, a Bayesian inference approach to determine single-structure models as well as structural ensembles from cryo-EM maps. EMMIVox automatically balances experimental information with accurate physico-chemical models of the system and the surrounding environment, including waters, lipids, and ions. Explicit treatment of data correlation and noise as well as inference of accurate B-factors enable determination of structural models and ensembles with both excellent fit to the data and high stereochemical quality, thus outperforming state-of-the-art refinement techniques. EMMIVox represents a flexible approach to determine high-quality structural models that will contribute to advancing our understanding of the molecular mechanisms underlying biological functions.\nBTR: A Bioinformatics Tool Recommendation System\nAuthors: Green, R.; Qu, X.; Liu, J.; Yu, T.\nScore: 23.8, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562252\nMotivationThe rapid expansion of Bioinformatics research has resulted in a vast array of computational tools utilized in the development of scientific analysis pipelines. However, constructing these pipelines is a laborious and intricate task, one which demands extensive domain knowledge and careful consideration at all stages. As the Bioinformatics landscape continues to evolve, researchers, both novice and expert, may find themselves overwhelmed when working in unfamiliar fields. Consequently, this may result in the selection of unsuitable or suboptimal tools during workflow development. ResultsIn this paper, we propose the Bioinformatics Tool Recommendation system (BTR), an innovative deep learning model designed to recommend the most suitable tools for a given workflow-in-progress. BTR utilizes recent advances in graph neural network technology and introduces a novel approach, representing the entire workflow as a graph to capture essential context and structural information. Additionally, natural language processing techniques are integrated to enhance the quality of tool recommendations by analyzing associated tool descriptions. Experiments demonstrate that BTR outperforms the existing Galaxy tool recommendation system, highlighting its potential to greatly facilitate scientific workflow construction. Availability and implementationThe Python source code is available at https://github.com/ryangreenj/bioinformatics_tool_recommendation\nSimulation of adaptive immune receptors and repertoires with complex immune information to guide the development and benchmarking of AIRR machine learning\nAuthors: Chernigovskaya, M.; Pavlovic, M.; Kanduri, C.; Gielis, S.; Robert, P. A.; Scheffer, L.; Slabodkin, A.; Haff, I. H.; Meysman, P.; Yaari, G.; Sandve, G. K.; Greiff, V.\nScore: 22.5, Published: 2023-10-23 DOI: 10.1101/2023.10.20.562936\nMachine-learning methods (ML) have shown great potential in the adaptive immune receptor repertoire (AIRR) field. However, there is a lack of large-scale ground-truth experimental AIRR data suitable for AIRR-ML-based disease diagnostics and therapeutics discovery. Simulated ground-truth AIRR data are required to complement the development and benchmarking of robust and interpretable AIRR-ML approaches where experimental data is inaccessible or insufficient as of yet. The challenge for simulated data to be useful is the ability to incorporate key features observed in experimental repertoires. These features, such as complex antigen or disease-associated immune information, cause AIRR-ML problems to be challenging. Here, we introduce LIgO, a modular software suite, which simulates AIRR data for the development and benchmarking of AIRR-based machine learning. LIgO incorporates different types of immune information both on the receptor and the repertoire level and preserves native-like generation probability distribution. Additionally, LIgO assists users in determining the computational feasibility of their simulations. We show two examples where LIgO simulation supports the development and validation of AIRR-ML methods: (1) how individuals carrying out-of-distribution immune information impacts receptor-level prediction performance and (2) how immune information co-occurring in the same AIRs have an impact on the performance of conventional receptor-level encoding and repertoire-level classification approaches. The LIgO software guides the advancement and assessment of interpretable AIRR-ML methods.\nVRP Assembler: haplotype-resolved de novo assembly of diploid and polyploid genomes using quantum computing\nAuthors: Chen, Y.; Huang, J.-H.; Sun, Y.; Zhang, Y.; Li, Y.; Xu, X.\nScore: 17.6, Published: 2023-10-21 DOI: 10.1101/2023.10.19.563028\nPrecision medicine's emphasis on individual genetic variants highlights the importance of haplotype-resolved assembly, a computational challenge in bioinformatics given its combinatorial nature. While classical algorithms have made strides in addressing this issue, the potential of quantum computing remains largely untapped. Here, we present the VRP assembler: a novel approach that transforms this task into a vehicle routing problem, a combinatorial optimization problem solvable on a quantum computer. We demonstrate its potential and feasibility through a proof of concept on short synthetic diploid and triploid genomes using a D-Wave quantum annealer. To tackle larger-scale assembly problems, we integrate the VRP assembler with Google's OR-Tools, achieving a haplotype-resolved assembly across the human major histocompatibility complex (MHC) region. Our results show encouraging performance compared to hifiasm with the phasing accuracy first approaching the theoretical limit, underscoring the promising future of quantum computing in bioinformatics.\n",
  "wordCount" : "1888",
  "inLanguage": "en",
  "datePublished": "2023-10-25T10:39:04Z",
  "dateModified": "2023-10-25T10:39:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 25, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.561085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.561085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.561085">
        <p class="paperTitle">Assessing the limits of zero-shot foundation models in single-cell biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.561085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.561085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.</p>
        <p class="info">Score: 79.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.561085' target='https://doi.org/10.1101/2023.10.16.561085'> 10.1101/2023.10.16.561085</a></p>
        <p class="abstract">The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562410">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562410" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562410">
        <p class="paperTitle">AF2BIND: Predicting ligand-binding sites using the pair representation of AlphaFold2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562410" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562410" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gazizov, A.; Lian, A.; Goverde, C. A.; Ovchinnikov, S.; Polizzi, N. F.</p>
        <p class="info">Score: 46.1, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562410' target='https://doi.org/10.1101/2023.10.15.562410'> 10.1101/2023.10.15.562410</a></p>
        <p class="abstract">Predicting ligand-binding sites, particularly in the absence of previously resolved homologous structures, presents a significant challenge in structural biology. Here, we leverage the internal pairwise representation of AlphaFold2 (AF2) to train a model, AF2BIND, to accurately predict small-molecule-binding residues given only a target protein. AF2BIND uses 20 &#34;bait&#34; amino acids to optimally extract the binding signal in the absence of a small-molecule ligand. We find that the AF2 pair representation outperforms other neural-network representations for binding-site prediction. Moreover, unique combinations of the 20 bait amino acids are correlated with chemical properties of the ligand.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563252">
        <p class="paperTitle">CellTracksColab, A platform for compiling, analyzing, and exploring tracking data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacquemet, G.</p>
        <p class="info">Score: 36.0, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563252' target='https://doi.org/10.1101/2023.10.20.563252'> 10.1101/2023.10.20.563252</a></p>
        <p class="abstract">In life sciences, tracking objects from movies enables researchers to quantify the behavior of single particles, organelles, bacteria, cells, and even whole animals. While numerous tools now allow automated tracking from video, a significant challenge persists in compiling, analyzing, and exploring the large datasets generated by these approaches. Here, we introduce CellTracksColab, an online platform tailored to simplify the exploration and analysis of tracking data by leveraging the free, cloud-based computational resources of Google Colab. CellTracksColab facilitates the amalgamation and analysis of tracking results across multiple fields of view, conditions, and repeats, ensuring a holistic dataset overview. CellTracksColab also harnesses the power of high-dimensional data reduction and clustering, using Uniform Manifold Approximation and Projection and Hierarchical Density-Based Spatial Clustering of Applications with Noise, enabling researchers to identify distinct behavioral patterns and trends without bias. CellTracksColab is available for the broader scientific community at https://github.com/guijacquemet/CellTracksColab.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562533">
        <p class="paperTitle">GENEPT: A SIMPLE BUT HARD-TO-BEAT FOUNDATION MODEL FOR GENES AND CELLS BUILT FROM CHATGPT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y. T.; Zou, J.</p>
        <p class="info">Score: 29.4, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562533' target='https://doi.org/10.1101/2023.10.16.562533'> 10.1101/2023.10.16.562533</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThere has been significant recent progress in leveraging large-scale gene expression data to develop foundation models for single-cell transcriptomes such as Geneformer [1], scGPT [2], and scBERT [3]. These models infer gene functions and interrelations from the gene expression profiles of millions of cells, which requires extensive data curation and resource-intensive training. Here, we explore a much simpler alternative by leveraging ChatGPT embeddings of genes based on literature. Our proposal, GenePT, uses NCBI text descriptions of individual genes with GPT-3.5 to generate gene embeddings. From there, GenePT generates single-cell embeddings in two ways: (i) by averaging the gene embeddings, weighted by each genes expression level; or (ii) by creating a sentence embedding for each cell, using gene names ordered by the expression level. Without the need for dataset curation and additional pretraining, GenePT is efficient and easy to use. On many downstream tasks used to evaluate recent single-cell foundation models -- e.g., classifying gene properties and cell types -- GenePT achieves comparable, and often better, performance than Geneformer and other methods. GenePT demonstrates that large language model embedding of literature is a simple and effective path for biological foundation models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562625">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562625" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562625">
        <p class="paperTitle">A deep learning model embedded framework to distinguish DNA and RNA mutations directly from RNA-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562625" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562625" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Z.-C.; Gao, B.-Q.; Nan, F.; Ma, X.-K.; Yang, L.</p>
        <p class="info">Score: 28.6, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562625' target='https://doi.org/10.1101/2023.10.17.562625'> 10.1101/2023.10.17.562625</a></p>
        <p class="abstract">We develop a stepwise computational framework, called DEMINING, to directly detect expressed DNA and RNA mutations in RNA deep sequencing data. DEMINING incorporates a deep learning model named DeepDDR, which facilitates the separation of expressed DNA mutations from RNA mutations after RNA-seq read mapping and pileup. When applied in RNA-seq of acute myeloid leukemia patients, DEMINING uncovered previously-underappreciated DNA and RNA mutations, some associated with the upregulated expression of host genes or the production of neoantigens. Finally, we demonstrate that DEMINING could precisely classify DNA and RNA mutations in RNA-seq data from non-primate species through the utilization of transfer learning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562114">
        <p class="paperTitle">OMICmAge: An integrative multi-omics approach to quantify biological age with electronic medical records</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Q.; Dwaraka, V. B.; Carreras-Gallo, N.; Mendez, K.; Chen, Y.; Kachroo, P.; Prince, N.; Went, H.; Mendez, T.; Lin, A.; Turner, L.; Moqri, M.; Chu, S. H.; Kelly, R. S.; Weiss, S. T.; Rattray, N. J.; Gladyshev, V. N.; Karlson, E.; Wheelock, C.; Mathe, E. A.; Dahlin, A.; McGeachie, M. J.; Smith, R.; Lasky-Su, J. A.</p>
        <p class="info">Score: 27.4, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562114' target='https://doi.org/10.1101/2023.10.16.562114'> 10.1101/2023.10.16.562114</a></p>
        <p class="abstract">Biological aging is a multifactorial process involving complex interactions of cellular and biochemical processes that is reflected in omic profiles. Using common clinical laboratory measures in [~]30,000 individuals from the MGB-Biobank, we developed a robust, predictive biological aging phenotype, EMRAge, that balances clinical biomarkers with overall mortality risk and can be broadly recapitulated across EMRs. We then applied elastic-net regression to model EMRAge with DNA-methylation (DNAm) and multiple omics, generating DNAmEMRAge and OMICmAge, respectively. Both biomarkers demonstrated strong associations with chronic diseases and mortality that outperform current biomarkers across our discovery (MGB-ABC, n=3,451) and validation (TruDiagnostic, n=12,666) cohorts. Through the use of epigenetic biomarker proxies, OMICmAge has the unique advantage of expanding the predictive search space to include epigenomic, proteomic, metabolomic, and clinical data while distilling this in a measure with DNAm alone, providing opportunities to identify clinically-relevant interconnections central to the aging process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562710">
        <p class="paperTitle">Accurate model and ensemble refinement using cryo-electron microscopy maps and Bayesian inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoff, S. E.; Thomasen, F. E.; Lindorff-Larsen, K.; Bonomi, M.</p>
        <p class="info">Score: 24.5, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562710' target='https://doi.org/10.1101/2023.10.18.562710'> 10.1101/2023.10.18.562710</a></p>
        <p class="abstract">Converting cryo-electron microscopy (cryo-EM) data into high-quality structural models is a challenging problem of outstanding importance. Current refinement methods often generate unbalanced models in which physico-chemical quality is sacrificed for excellent fit to the data. Furthermore, these techniques struggle to represent the conformational heterogeneity averaged out in low-resolution regions of density maps. Here we introduce EMMIVox, a Bayesian inference approach to determine single-structure models as well as structural ensembles from cryo-EM maps. EMMIVox automatically balances experimental information with accurate physico-chemical models of the system and the surrounding environment, including waters, lipids, and ions. Explicit treatment of data correlation and noise as well as inference of accurate B-factors enable determination of structural models and ensembles with both excellent fit to the data and high stereochemical quality, thus outperforming state-of-the-art refinement techniques. EMMIVox represents a flexible approach to determine high-quality structural models that will contribute to advancing our understanding of the molecular mechanisms underlying biological functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562252">
        <p class="paperTitle">BTR: A Bioinformatics Tool Recommendation System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Green, R.; Qu, X.; Liu, J.; Yu, T.</p>
        <p class="info">Score: 23.8, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562252' target='https://doi.org/10.1101/2023.10.13.562252'> 10.1101/2023.10.13.562252</a></p>
        <p class="abstract">MotivationThe rapid expansion of Bioinformatics research has resulted in a vast array of computational tools utilized in the development of scientific analysis pipelines. However, constructing these pipelines is a laborious and intricate task, one which demands extensive domain knowledge and careful consideration at all stages. As the Bioinformatics landscape continues to evolve, researchers, both novice and expert, may find themselves overwhelmed when working in unfamiliar fields. Consequently, this may result in the selection of unsuitable or suboptimal tools during workflow development.

ResultsIn this paper, we propose the Bioinformatics Tool Recommendation system (BTR), an innovative deep learning model designed to recommend the most suitable tools for a given workflow-in-progress. BTR utilizes recent advances in graph neural network technology and introduces a novel approach, representing the entire workflow as a graph to capture essential context and structural information. Additionally, natural language processing techniques are integrated to enhance the quality of tool recommendations by analyzing associated tool descriptions. Experiments demonstrate that BTR outperforms the existing Galaxy tool recommendation system, highlighting its potential to greatly facilitate scientific workflow construction.

Availability and implementationThe Python source code is available at https://github.com/ryangreenj/bioinformatics_tool_recommendation</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.562936">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.562936" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.562936">
        <p class="paperTitle">Simulation of adaptive immune receptors and repertoires with complex immune information to guide the development and benchmarking of AIRR machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.562936" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.562936" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chernigovskaya, M.; Pavlovic, M.; Kanduri, C.; Gielis, S.; Robert, P. A.; Scheffer, L.; Slabodkin, A.; Haff, I. H.; Meysman, P.; Yaari, G.; Sandve, G. K.; Greiff, V.</p>
        <p class="info">Score: 22.5, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.562936' target='https://doi.org/10.1101/2023.10.20.562936'> 10.1101/2023.10.20.562936</a></p>
        <p class="abstract">Machine-learning methods (ML) have shown great potential in the adaptive immune receptor repertoire (AIRR) field. However, there is a lack of large-scale ground-truth experimental AIRR data suitable for AIRR-ML-based disease diagnostics and therapeutics discovery. Simulated ground-truth AIRR data are required to complement the development and benchmarking of robust and interpretable AIRR-ML approaches where experimental data is inaccessible or insufficient as of yet. The challenge for simulated data to be useful is the ability to incorporate key features observed in experimental repertoires. These features, such as complex antigen or disease-associated immune information, cause AIRR-ML problems to be challenging. Here, we introduce LIgO, a modular software suite, which simulates AIRR data for the development and benchmarking of AIRR-based machine learning. LIgO incorporates different types of immune information both on the receptor and the repertoire level and preserves native-like generation probability distribution. Additionally, LIgO assists users in determining the computational feasibility of their simulations. We show two examples where LIgO simulation supports the development and validation of AIRR-ML methods: (1) how individuals carrying out-of-distribution immune information impacts receptor-level prediction performance and (2) how immune information co-occurring in the same AIRs have an impact on the performance of conventional receptor-level encoding and repertoire-level classification approaches. The LIgO software guides the advancement and assessment of interpretable AIRR-ML methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563028">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563028" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563028">
        <p class="paperTitle">VRP Assembler: haplotype-resolved de novo assembly of diploid and polyploid genomes using quantum computing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563028" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563028" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Huang, J.-H.; Sun, Y.; Zhang, Y.; Li, Y.; Xu, X.</p>
        <p class="info">Score: 17.6, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563028' target='https://doi.org/10.1101/2023.10.19.563028'> 10.1101/2023.10.19.563028</a></p>
        <p class="abstract">Precision medicine&#39;s emphasis on individual genetic variants highlights the importance of haplotype-resolved assembly, a computational challenge in bioinformatics given its combinatorial nature. While classical algorithms have made strides in addressing this issue, the potential of quantum computing remains largely untapped. Here, we present the VRP assembler: a novel approach that transforms this task into a vehicle routing problem, a combinatorial optimization problem solvable on a quantum computer. We demonstrate its potential and feasibility through a proof of concept on short synthetic diploid and triploid genomes using a D-Wave quantum annealer. To tackle larger-scale assembly problems, we integrate the VRP assembler with Google&#39;s OR-Tools, achieving a haplotype-resolved assembly across the human major histocompatibility complex (MHC) region. Our results show encouraging performance compared to hifiasm with the phasing accuracy first approaching the theoretical limit, underscoring the promising future of quantum computing in bioinformatics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
