<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="QuantUMS: uncertainty minimisation enables confident quantification in proteomics
Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.
Score: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604
Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/bioinformatics/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="QuantUMS: uncertainty minimisation enables confident quantification in proteomics
Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.
Score: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604
Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T14:29:33+08:00" />
<meta property="article:modified_time" content="2023-06-28T14:29:33+08:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="QuantUMS: uncertainty minimisation enables confident quantification in proteomics
Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.
Score: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604
Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://yorks0n.github.io/TRxiv2/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "QuantUMS: uncertainty minimisation enables confident quantification in proteomics\nAuthors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.\nScore: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604\nMass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information.",
  "keywords": [
    
  ],
  "articleBody": " QuantUMS: uncertainty minimisation enables confident quantification in proteomics\nAuthors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.\nScore: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604\nMass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information. In combination with data-independent acquisition proteomics, QuantUMS boosts accuracy and precision of quantities, as well as reports an uncertainty metric, enabling effective filtering of data for downstream analysis. The algorithm has linear complexity with respect to the number of mass spectrometry acquisitions in the experiment and is thus scalable to infinitely large proteomic experiments. For an easy implementation in a proteomics laboratory, we integrate QuantUMS in our automated DIA-NN software suite.\nTooManyCellsInteractive: a visualization tool for dynamic exploration of single-cell data\nAuthors: Klamann, C.; Lau, C.; Schwartz, G. W.\nScore: 34.5, Published: 2023-06-18 DOI: 10.1101/2023.06.16.544954\nAs single-cell sequencing data sets grow in size, visualizations of large cellular populations become difficult to parse and require extensive processing to identify subpopulations of cells. Managing many of these charts is laborious for technical users and unintuitive for non-technical users. To address this issue, we developed TooManyCellsInteractive (TMCI), a browser-based JavaScript application for visualizing hierarchical cellular populations as an interactive radial tree. TMCI allows users to explore, filter, and manipulate hierarchical data structures through an intuitive interface while also enabling batch export of high-quality custom graphics. Here we describe the software architecture and illustrate how TMCI has identified unique survival pathways among drug-tolerant persister cells in a pan-cancer analysis. TMCI will help guide increasingly large data visualizations and facilitate multi-resolution data exploration in a user-friendly way.\nComprehensive annotation of 3'UTRs from primary cells and their quantification from scRNA-seq data\nAuthors: Fansler, M. M.; Mitschka, S.; Mayr, C.\nScore: 33.1, Published: 2023-06-22 DOI: 10.1101/2021.11.22.469635\nApproximately half of human genes generate mRNA isoforms that differ in their 3'UTRs while encoding the same protein. 3'UTR and mRNA length is determined by 3' end cleavage sites (CS). Here, we mapped and categorized mRNA 3' end CS in more than 200 primary human and mouse cell types, resulting in a 40% increase of CS annotations relative to the GENCODE database. We incorporated these annotations into a novel computational pipeline, called scUTRquant, for rapid, precise, and accurate quantification of gene and 3'UTR isoform expression from single-cell RNA sequencing (scRNA-seq) data. When applying scUTRquant to data from 474 cell types and 2,134 perturbations, we discovered extensive 3'UTR length changes across cell types that are as widespread and dynamically regulated as gene expression changes. Our data indicate that mRNA abundance and mRNA length are two independent axes of gene regulation that together determine the amount and spatial organization of protein synthesis.\nBrainRBPedia: a resource for RNA-binding proteins relevant to neurodevelopmental disorders\nAuthors: Han, K.; Wainberg, M.; Calarco, J. A.; Smibert, C. A.; Lipshitz, H. D.; Lee, H. O.; Tripathy, S. J.\nScore: 23.8, Published: 2023-06-09 DOI: 10.1101/2023.06.07.542483\nRNA-binding proteins (RBPs) are crucial players in the post-transcriptional regulation of mRNA and play major roles in ensuring proper neuronal development and function. Deficits in RBP function have been implicated in a number of neurodevelopmental disorders including autism spectrum disorder (ASD) and intellectual disability (ID), yet we lack resources that integrate current knowledge of RBP function, tissue expression, and disease association in one place to aid in their experimental characterization. Here we introduce BrainRBPedia - a database of 1072 RBPs with both disease annotations for neurodevelopmental disorders and functional annotations relevant to these disorders, including loss-of-function intolerance and expression specificity to the brain, neurons, and neuronal development. Using these functional annotations, we develop a machine learning model to prioritize RBPs likely to be involved in ASD and ID. Our model indicates that RBPs with high loss-of-function intolerance and those upregulated during neuronal differentiation are disproportionately likely to contribute to ASD and ID etiology. In summary, BrainRBPedia comprises a unique resource for researchers interested in the experimental characterization of RBPs in relation to neurodevelopmental disorders and suggests functional signatures of RBPs likely to play a role in neurodevelopment.\nDeepRank-GNN-esm: A Graph Neural Network for Scoring Protein-Protein Models using Protein Language Model\nAuthors: Xu, X.; Bonvin, A. M.\nScore: 18.3, Published: 2023-06-24 DOI: 10.1101/2023.06.22.546080\nProtein-Protein interactions (PPIs) play critical roles in numerous cellular processes. By modelling the three-dimensional structures of the correspond protein complexes valuable insights can be obtained, providing, for example, starting points for drug and protein design. One challenge in the modelling process is however the identification of near-native models from the large pool of generated models. To this end we previously developed DeepRank-GNN, a graph neural network that integrates structural and sequence information to enable effective pattern learning at PPI interfaces. Its main features are related to the Position Specific Scoring Matrices (PSSM), which are computationally expensive to generate and significantly limit the algorithm's usability. We introduce here DeepRank-GNN-esm that includes as additional features protein language model embeddings from the EMS-2 model. We show that the ESM-2 embeddings can actually replace the PSSM features at no cost in-, or even better performance on two PPI-related tasks: scoring docking poses and detecting crystal artifacts. This new DeepRank version bypasses thus the need of generating PSSM, greatly improving the usability of the software and opening new application opportunities for systems for which PSSM profiles cannot be obtained or are irrelevant (e.g. antibody-antigen complexes). DeepRank-GNN-esm is freely available from https://github.com/DeepRank/DeepRank-GNN-esm .\n",
  "wordCount" : "945",
  "inLanguage": "en",
  "datePublished": "2023-06-28T14:29:33+08:00",
  "dateModified": "2023-06-28T14:29:33+08:00",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545604">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545604" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545604">
        <p class="paperTitle">QuantUMS: uncertainty minimisation enables confident quantification in proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545604" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545604" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.</p>
        <p class="info">Score: 49.3, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545604' target='https://doi.org/10.1101/2023.06.20.545604'> 10.1101/2023.06.20.545604</a></p>
        <p class="abstract">Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information. In combination with data-independent acquisition proteomics, QuantUMS boosts accuracy and precision of quantities, as well as reports an uncertainty metric, enabling effective filtering of data for downstream analysis. The algorithm has linear complexity with respect to the number of mass spectrometry acquisitions in the experiment and is thus scalable to infinitely large proteomic experiments. For an easy implementation in a proteomics laboratory, we integrate QuantUMS in our automated DIA-NN software suite.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.544954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.544954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.544954">
        <p class="paperTitle">TooManyCellsInteractive: a visualization tool for dynamic exploration of single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.544954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.544954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klamann, C.; Lau, C.; Schwartz, G. W.</p>
        <p class="info">Score: 34.5, Published: 2023-06-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.544954' target='https://doi.org/10.1101/2023.06.16.544954'> 10.1101/2023.06.16.544954</a></p>
        <p class="abstract">As single-cell sequencing data sets grow in size, visualizations of large cellular populations become difficult to parse and require extensive processing to identify subpopulations of cells. Managing many of these charts is laborious for technical users and unintuitive for non-technical users. To address this issue, we developed TooManyCellsInteractive (TMCI), a browser-based JavaScript application for visualizing hierarchical cellular populations as an interactive radial tree. TMCI allows users to explore, filter, and manipulate hierarchical data structures through an intuitive interface while also enabling batch export of high-quality custom graphics. Here we describe the software architecture and illustrate how TMCI has identified unique survival pathways among drug-tolerant persister cells in a pan-cancer analysis. TMCI will help guide increasingly large data visualizations and facilitate multi-resolution data exploration in a user-friendly way.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.11.22.469635">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.11.22.469635" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.11.22.469635">
        <p class="paperTitle">Comprehensive annotation of 3&#39;UTRs from primary cells and their quantification from scRNA-seq data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.11.22.469635" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.11.22.469635" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fansler, M. M.; Mitschka, S.; Mayr, C.</p>
        <p class="info">Score: 33.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.11.22.469635' target='https://doi.org/10.1101/2021.11.22.469635'> 10.1101/2021.11.22.469635</a></p>
        <p class="abstract">Approximately half of human genes generate mRNA isoforms that differ in their 3&#39;UTRs while encoding the same protein. 3&#39;UTR and mRNA length is determined by 3&#39; end cleavage sites (CS). Here, we mapped and categorized mRNA 3&#39; end CS in more than 200 primary human and mouse cell types, resulting in a 40% increase of CS annotations relative to the GENCODE database. We incorporated these annotations into a novel computational pipeline, called scUTRquant, for rapid, precise, and accurate quantification of gene and 3&#39;UTR isoform expression from single-cell RNA sequencing (scRNA-seq) data. When applying scUTRquant to data from 474 cell types and 2,134 perturbations, we discovered extensive 3&#39;UTR length changes across cell types that are as widespread and dynamically regulated as gene expression changes. Our data indicate that mRNA abundance and mRNA length are two independent axes of gene regulation that together determine the amount and spatial organization of protein synthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.542483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.542483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.542483">
        <p class="paperTitle">BrainRBPedia: a resource for RNA-binding proteins relevant to neurodevelopmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.542483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.542483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, K.; Wainberg, M.; Calarco, J. A.; Smibert, C. A.; Lipshitz, H. D.; Lee, H. O.; Tripathy, S. J.</p>
        <p class="info">Score: 23.8, Published: 2023-06-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.542483' target='https://doi.org/10.1101/2023.06.07.542483'> 10.1101/2023.06.07.542483</a></p>
        <p class="abstract">RNA-binding proteins (RBPs) are crucial players in the post-transcriptional regulation of mRNA and play major roles in ensuring proper neuronal development and function. Deficits in RBP function have been implicated in a number of neurodevelopmental disorders including autism spectrum disorder (ASD) and intellectual disability (ID), yet we lack resources that integrate current knowledge of RBP function, tissue expression, and disease association in one place to aid in their experimental characterization. Here we introduce BrainRBPedia - a database of 1072 RBPs with both disease annotations for neurodevelopmental disorders and functional annotations relevant to these disorders, including loss-of-function intolerance and expression specificity to the brain, neurons, and neuronal development. Using these functional annotations, we develop a machine learning model to prioritize RBPs likely to be involved in ASD and ID. Our model indicates that RBPs with high loss-of-function intolerance and those upregulated during neuronal differentiation are disproportionately likely to contribute to ASD and ID etiology. In summary, BrainRBPedia comprises a unique resource for researchers interested in the experimental characterization of RBPs in relation to neurodevelopmental disorders and suggests functional signatures of RBPs likely to play a role in neurodevelopment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546080">
        <p class="paperTitle">DeepRank-GNN-esm: A Graph Neural Network for Scoring Protein-Protein Models using Protein Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, X.; Bonvin, A. M.</p>
        <p class="info">Score: 18.3, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546080' target='https://doi.org/10.1101/2023.06.22.546080'> 10.1101/2023.06.22.546080</a></p>
        <p class="abstract">Protein-Protein interactions (PPIs) play critical roles in numerous cellular processes. By modelling the three-dimensional structures of the correspond protein complexes valuable insights can be obtained, providing, for example, starting points for drug and protein design. One challenge in the modelling process is however the identification of near-native models from the large pool of generated models. To this end we previously developed DeepRank-GNN, a graph neural network that integrates structural and sequence information to enable effective pattern learning at PPI interfaces. Its main features are related to the Position Specific Scoring Matrices (PSSM), which are computationally expensive to generate and significantly limit the algorithm&#39;s usability. We introduce here DeepRank-GNN-esm that includes as additional features protein language model embeddings from the EMS-2 model. We show that the ESM-2 embeddings can actually replace the PSSM features at no cost in-, or even better performance on two PPI-related tasks: scoring docking poses and detecting crystal artifacts. This new DeepRank version bypasses thus the need of generating PSSM, greatly improving the usability of the software and opening new application opportunities for systems for which PSSM profiles cannot be obtained or are irrelevant (e.g. antibody-antigen complexes). DeepRank-GNN-esm is freely available from https://github.com/DeepRank/DeepRank-GNN-esm .</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
