<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)
Authors: Revell, L. J.
Score: 198.2, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791
Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)
Authors: Revell, L. J.
Score: 198.2, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791
Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)
Authors: Revell, L. J.
Score: 198.2, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791
Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)\nAuthors: Revell, L. J.\nScore: 198.2, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791\nPhylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis.",
  "keywords": [
    
  ],
  "articleBody": " phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)\nAuthors: Revell, L. J.\nScore: 198.2, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791\nPhylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis. phytools is a diverse contributed R library now consisting of hundreds of different functions covering a variety of methods and purposes in phylogenetic biology. As of the time of writing, phytools included functionality for fitting models of trait evolution, for reconstructing ancestral states, for studying diversification on trees, and for visualizing phylogenies, comparative data, and fitted models, as well numerous other tasks related to phylogenetic biology. Here, I describe some significant features of and recent updates to phytools, while also illustrating several popular workflows of the phytools computational software.\nFrom Proteins to Ligands: Decoding Deep Learning Methods for Binding Affinity Prediction\nAuthors: Gorantla, R.; Kubincova, A.; Weisse, A. Y.; Mey, A. S. J. S.\nScore: 36.1, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551483\nAccurate in silico prediction of protein-ligand binding affinity is important in the early stages of drug discovery. Deep learning-based methods exist but have yet to overtake more conventional methods such as giga-docking largely due to their lack of generalisability. To improve generalizability we need to understand what these models learn from input protein and ligand data. We systematically investigated a sequence-based deep learning framework to assess the impact of protein and ligand encodings on predicting binding affinities for commonly used kinase data sets. The role of proteins is studied using convolutional neural network-based encodings obtained from sequences and graph neural network-based encodings enriched with structural information from contact maps. Ligand-based encodings are generated from graph-neural networks. We test different ligand perturbations by randomizing node and edge properties. For proteins we make use of 3 different protein contact generation methods (AlphaFold2, Pconsc4, and ESM-1b) and compare these with a random control. Our investigation shows that protein encodings do not substantially impact the binding predictions, with no statistically significant difference in binding affinity for KIBA in the investigated metrics (concordance index, Pearsons R Spearmans Rank, and RMSE). Significant differences are seen for ligand encodings with random ligands and random ligand node properties, suggesting a much bigger reliance on ligand data for the learning tasks. Using different ways to combine protein and ligand encodings, did not show a significant change in performance. TOC Graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=\"FIGDIR/small/551483v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@138b03corg.highwire.dtl.DTLVardef@ba5fe0org.highwire.dtl.DTLVardef@1ec9174org.highwire.dtl.DTLVardef@415fdd_HPS_FORMAT_FIGEXP M_FIG C_FIG\ngetphylo: rapid and automatic generation of multi-locus phylogenetic trees\nAuthors: Booth, T. J.; Shaw, S.; Weber, T.\nScore: 37.4, Published: 2023-07-31 DOI: 10.1101/2023.07.26.550493\nMotivationPhylogenetic trees are the primary tool for visualising evolutionary relationships. Traditionally, phylogenies are inferred from manually curated sets of marker genes. As available genomic data increases, there is increasing demand for tools to automatically build phylogenies from assembled genomes. Existing tools rely on reference databases of preselected marker genes, limiting their taxonomic scope. We sought to develop a tool that could quickly build phylogeny from input genomes alone. ResultsWe developed getphylo, a tool to automatically generate multi-locus phylogenetic trees from GenBank files. It has a low barrier to entry with minimal dependencies. getphylo uses a parallelised, heuristic workflow to keep runtime and system requirements as low as possible. getphylo consistently produces trees with topologies comparable to other tools in less time. Furthermore, as getphylo does not rely on reference databases, it has a virtually unlimited scope in terms of taxonomy (e.g., not limited to bacteria) and genetic scale (e.g., can analyse plasmids, prophage, and gene clusters). This combination of speed and flexibility makes getphylo a valuable addition to the phylogenetics toolkit. Availabilitygetphylo is freely available and is downloadable through the Python Package Index (pip install getphylo; https://pypi.org/project/getphylo/) and GitHub (https://github.com/drboothtj/getphylo).\nChallenges in identifying mRNA transcript starts and ends from long-read sequencing data\nAuthors: Calvo-Roitberg, E.; Daniels, R. F.; Pai, A. A.\nScore: 32.5, Published: 2023-07-27 DOI: 10.1101/2023.07.26.550536\nLong-read sequencing (LRS) technologies have the potential to revolutionize scientific discoveries in RNA biology, especially by enabling the comprehensive identification and quantification of full length mRNA isoforms. However, inherently high error rates make the analysis of long-read sequencing data challenging. While these error rates have been characterized for sequence and splice site identification, it is still unclear how accurately LRS reads represent transcript start and end sites. Here, we systematically assess the variability and accuracy of mRNA terminal ends identified by LRS reads across multiple sequencing platforms. We find substantial inconsistencies in both the start and end coordinates of LRS reads spanning a gene, such that LRS reads often fail to accurately recapitulate annotated or empirically derived terminal ends of mRNA molecules. To address this challenge, we introduce an approach to condition reads based on empirically derived terminal ends and identified a subset of reads that are more likely to represent full-length transcripts. Our approach can improve transcriptome analyses by enhancing the fidelity of transcript terminal end identification, but may result in lower power to quantify genes or discover novel isoforms. Thus, it is necessary to be cautious when selecting sequencing approaches and/or interpreting data from long-read RNA sequencing.\nContext-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes\nAuthors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.\nScore: 28.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799\nThe potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis. An alternative promising approach involves expanding the capabilities of existing natural enzymes to enable functionality across new substrates and operational parameters. To this end we introduce CoSaNN (Conformation Sampling using Neural Network), a novel strategy for enzyme design that utilizes advances in deep learning for structure prediction and sequence optimization. By controlling enzyme conformations, we can expand the chemical space beyond the reach of simple mutagenesis. CoSaNN uses a context-dependent approach that accurately generates novel enzyme designs by considering non-linear relationships in both sequence and structure space. Additionally, we have further developed SolvIT, a graph neural network trained to predict protein solubility in E.Coli, as an additional optimization layer for producing highly expressed enzymes. Through this approach, we have engineered novel enzymes exhibiting superior expression levels, with 54% of our designs expressed in E.Coli, and increased thermal stability with more than 30% of our designs having a higher Tm than the template enzyme. Furthermore, our research underscores the transformative potential of AI in protein design, adeptly capturing high order interactions and preserving allosteric mechanisms in extensively modified enzymes. These advancements pave the way for the creation of diverse, functional, and robust enzymes, thereby opening new avenues for targeted biotechnological applications.\nQuantifying Cell-State Densities in Single-Cell Phenotypic Landscapes using Mellon\nAuthors: Otto, D. J.; Jordan, C.; Dury, B.; Dien, C.; Setty, M.\nScore: 47.9, Published: 2023-07-11 DOI: 10.1101/2023.07.09.548272\nCell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive cellular differentiation, regeneration, and disease. Here, we present Mellon, a novel computational algorithm for high-resolution estimation of cell-state densities from single-cell data. We demonstrate Mellons efficacy by dissecting the density landscape of various differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. Utilizing hematopoietic stem cell fate specification to B-cells as a case study, we present evidence implicating enhancer priming and the activation of master regulators in the emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during the inherently continuous developmental processes. Scalable and adaptable, Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide new insights into the regulatory mechanisms guiding cellular fate decisions.\nFragment ion intensity prediction improves the identification rate of non-tryptic peptides in TimsTOF\nAuthors: Adams, C.; Gabriel, W.; Laukens, K.; Wilhelm, M.; Bittremieux, W.; Boonen, K.\nScore: 17.1, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549401\nImmunopeptidomics plays a crucial role in identifying targets for immunotherapy and vaccine development. Because the generation of immunopeptides from their parent proteins does not adhere to clear-cut rules, rather than being able to use known digestion patterns, every possible protein subsequence within HLA class-specific length restrictions needs to be considered. This leads to an inflation of the search space and results in lower spectrum annotation rates. Rescoring is a powerful enhancement of standard sequence database searching that boosts the spectrum annotation performance. In the field of immunopeptidomics low abundant peptides often occur, which is why the highly sensitive timsTOF instruments are increasingly gaining popularity. To improve rescoring for immunopeptides measured using timsTOF instruments, we trained a deep learning-based fragment ion intensity prediction model. Over 300,000 synthesized non-tryptic peptides from the ProteomeTools project were analyzed on a timsTOF-Pro to generate a dataset that was used to fine-tune an existing Prosit model. By applying our fragment ion intensity prediction model, we demonstrate up to 3-fold improvement in the identification of immunopeptides. Furthermore, our approach increased detection of immunopeptides even from low input samples.\neinprot: flexible, easy-to-use, reproducible workflows for statistical analysis of quantitative proteomics data\nAuthors: Soneson, C.; Iesmantavicius, V.; Hess, D.; Stadler, M. B.; Seebacher, J.\nScore: 12.2, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550821\nWe describe einprot, an R package providing easy-to-use reproducible workflows for quality control, statistical analysis and visualization of quantitative proteomics data. einprot is applicable to tabular output from MaxQuant, Proteome Discoverer and FragPipe, and a single function call generates an html report that describes the full analysis pipeline applied to the data and contains static and interactive figures and tables for further exploration. This has the potential to facilitate routine analyses as well as to provide a standardized, yet comprehensive way to communicate results to collaborators and the broader community. The source file underlying the report is also returned, giving the user full flexibility to further modify the workflow according to their needs.\ndisperseNN2: a neural network for estimating dispersal distance from georeferenced polymorphism data\nAuthors: Smith, C. C.; Kern, A. D.\nScore: 11.9, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551115\nWe present a deep learning tool, disperseNN2, for estimating the mean per-generation dispersal distance from georeferenced polymorphism data. Our neural network performs feature extraction on pairs of genotypes, and uses the geographic information that comes with each sample. These attributes led disperseNN2 to outperform a state-of-the-art deep learning method that does not use explicit spatial information: error was reduced by 33% and 42% using sample sizes of 10 and 100 individuals, respectively. disperseNN2 is particularly useful for non-model organisms or systems with sparse genomic resources, as it uses unphased, single nucleotide polymorphisms as its input. Availability and Implementationhttps://github.com/kr-colab/disperseNN2 Contactchriscs@uoregon.edu\nSensitive remote homology search by local alignment of small positional embeddings from protein language models\nAuthors: Johnson, S. R.; Peshwa, M.; Sun, Z.\nScore: 16.4, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550718\nAccurately detecting distant evolutionary relationships between proteins remains an ongoing challenge in bioinformatics. Search methods based on primary sequence struggle to accurately detect homology between sequences with less than 20% amino acid identity. Profile- and structure-based strategies extend sensitive search capabilities into this twilight zone of sequence similarity but require slow pre-processing steps. Recently, whole-protein and positional embeddings from deep neural networks have shown promise for providing sensitive sequence comparison and annotation at long evolutionary distances. Embeddings are generally faster to compute than profiles and predicted structures but still suffer several drawbacks related to the ability of whole-protein embeddings to discriminate domain-level homology, and the database size and search speed of methods using positional embeddings. In this work, we show that low-dimensionality positional embeddings can be used directly in speed-optimized local search algorithms. As a proof of concept, we use the ESM2 3B model to convert primary sequences directly into the 3Di alphabet or amino acid profiles and use these embeddings as input to the highly optimized Foldseek, HMMER3, and HH-suite search algorithms. Our results suggest that positional embeddings as small as a single byte can provide sufficient information for dramatically improved sensitivity over amino acid sequence searches without sacrificing search speed.\n",
  "wordCount" : "2083",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.08.531791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.08.531791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.08.531791">
        <p class="paperTitle">phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.08.531791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.08.531791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Revell, L. J.</p>
        <p class="info">Score: 198.2, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.08.531791' target='https://doi.org/10.1101/2023.03.08.531791'> 10.1101/2023.03.08.531791</a></p>
        <p class="abstract">Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis. phytools is a diverse contributed R library now consisting of hundreds of different functions covering a variety of methods and purposes in phylogenetic biology. As of the time of writing, phytools included functionality for fitting models of trait evolution, for reconstructing ancestral states, for studying diversification on trees, and for visualizing phylogenies, comparative data, and fitted models, as well numerous other tasks related to phylogenetic biology. Here, I describe some significant features of and recent updates to phytools, while also illustrating several popular workflows of the phytools computational software.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551483">
        <p class="paperTitle">From Proteins to Ligands: Decoding Deep Learning Methods for Binding Affinity Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gorantla, R.; Kubincova, A.; Weisse, A. Y.; Mey, A. S. J. S.</p>
        <p class="info">Score: 36.1, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551483' target='https://doi.org/10.1101/2023.08.01.551483'> 10.1101/2023.08.01.551483</a></p>
        <p class="abstract">Accurate in silico prediction of protein-ligand binding affinity is important in the early stages of drug discovery. Deep learning-based methods exist but have yet to overtake more conventional methods such as giga-docking largely due to their lack of generalisability. To improve generalizability we need to understand what these models learn from input protein and ligand data. We systematically investigated a sequence-based deep learning framework to assess the impact of protein and ligand encodings on predicting binding affinities for commonly used kinase data sets. The role of proteins is studied using convolutional neural network-based encodings obtained from sequences and graph neural network-based encodings enriched with structural information from contact maps. Ligand-based encodings are generated from graph-neural networks. We test different ligand perturbations by randomizing node and edge properties. For proteins we make use of 3 different protein contact generation methods (AlphaFold2, Pconsc4, and ESM-1b) and compare these with a random control. Our investigation shows that protein encodings do not substantially impact the binding predictions, with no statistically significant difference in binding affinity for KIBA in the investigated metrics (concordance index, Pearsons R Spearmans Rank, and RMSE). Significant differences are seen for ligand encodings with random ligands and random ligand node properties, suggesting a much bigger reliance on ligand data for the learning tasks. Using different ways to combine protein and ligand encodings, did not show a significant change in performance.

TOC Graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=&#34;FIGDIR/small/551483v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@138b03corg.highwire.dtl.DTLVardef@ba5fe0org.highwire.dtl.DTLVardef@1ec9174org.highwire.dtl.DTLVardef@415fdd_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550493">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550493" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550493">
        <p class="paperTitle">getphylo: rapid and automatic generation of multi-locus phylogenetic trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550493" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550493" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Booth, T. J.; Shaw, S.; Weber, T.</p>
        <p class="info">Score: 37.4, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550493' target='https://doi.org/10.1101/2023.07.26.550493'> 10.1101/2023.07.26.550493</a></p>
        <p class="abstract">MotivationPhylogenetic trees are the primary tool for visualising evolutionary relationships. Traditionally, phylogenies are inferred from manually curated sets of marker genes. As available genomic data increases, there is increasing demand for tools to automatically build phylogenies from assembled genomes. Existing tools rely on reference databases of preselected marker genes, limiting their taxonomic scope. We sought to develop a tool that could quickly build phylogeny from input genomes alone.

ResultsWe developed getphylo, a tool to automatically generate multi-locus phylogenetic trees from GenBank files. It has a low barrier to entry with minimal dependencies. getphylo uses a parallelised, heuristic workflow to keep runtime and system requirements as low as possible. getphylo consistently produces trees with topologies comparable to other tools in less time. Furthermore, as getphylo does not rely on reference databases, it has a virtually unlimited scope in terms of taxonomy (e.g., not limited to bacteria) and genetic scale (e.g., can analyse plasmids, prophage, and gene clusters). This combination of speed and flexibility makes getphylo a valuable addition to the phylogenetics toolkit.

Availabilitygetphylo is freely available and is downloadable through the Python Package Index (pip install getphylo; https://pypi.org/project/getphylo/) and GitHub (https://github.com/drboothtj/getphylo).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550536" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550536">
        <p class="paperTitle">Challenges in identifying mRNA transcript starts and ends from long-read sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calvo-Roitberg, E.; Daniels, R. F.; Pai, A. A.</p>
        <p class="info">Score: 32.5, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550536' target='https://doi.org/10.1101/2023.07.26.550536'> 10.1101/2023.07.26.550536</a></p>
        <p class="abstract">Long-read sequencing (LRS) technologies have the potential to revolutionize scientific discoveries in RNA biology, especially by enabling the comprehensive identification and quantification of full length mRNA isoforms. However, inherently high error rates make the analysis of long-read sequencing data challenging. While these error rates have been characterized for sequence and splice site identification, it is still unclear how accurately LRS reads represent transcript start and end sites. Here, we systematically assess the variability and accuracy of mRNA terminal ends identified by LRS reads across multiple sequencing platforms. We find substantial inconsistencies in both the start and end coordinates of LRS reads spanning a gene, such that LRS reads often fail to accurately recapitulate annotated or empirically derived terminal ends of mRNA molecules. To address this challenge, we introduce an approach to condition reads based on empirically derived terminal ends and identified a subset of reads that are more likely to represent full-length transcripts. Our approach can improve transcriptome analyses by enhancing the fidelity of transcript terminal end identification, but may result in lower power to quantify genes or discover novel isoforms. Thus, it is necessary to be cautious when selecting sequencing approaches and/or interpreting data from long-read RNA sequencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550799">
        <p class="paperTitle">Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.</p>
        <p class="info">Score: 28.3, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550799' target='https://doi.org/10.1101/2023.07.27.550799'> 10.1101/2023.07.27.550799</a></p>
        <p class="abstract">The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis. An alternative promising approach involves expanding the capabilities of existing natural enzymes to enable functionality across new substrates and operational parameters. To this end we introduce CoSaNN (Conformation Sampling using Neural Network), a novel strategy for enzyme design that utilizes advances in deep learning for structure prediction and sequence optimization. By controlling enzyme conformations, we can expand the chemical space beyond the reach of simple mutagenesis. CoSaNN uses a context-dependent approach that accurately generates novel enzyme designs by considering non-linear relationships in both sequence and structure space. Additionally, we have further developed SolvIT, a graph neural network trained to predict protein solubility in E.Coli, as an additional optimization layer for producing highly expressed enzymes. Through this approach, we have engineered novel enzymes exhibiting superior expression levels, with 54% of our designs expressed in E.Coli, and increased thermal stability with more than 30% of our designs having a higher Tm than the template enzyme. Furthermore, our research underscores the transformative potential of AI in protein design, adeptly capturing high order interactions and preserving allosteric mechanisms in extensively modified enzymes. These advancements pave the way for the creation of diverse, functional, and robust enzymes, thereby opening new avenues for targeted biotechnological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548272">
        <p class="paperTitle">Quantifying Cell-State Densities in Single-Cell Phenotypic Landscapes using Mellon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otto, D. J.; Jordan, C.; Dury, B.; Dien, C.; Setty, M.</p>
        <p class="info">Score: 47.9, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548272' target='https://doi.org/10.1101/2023.07.09.548272'> 10.1101/2023.07.09.548272</a></p>
        <p class="abstract">Cell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive cellular differentiation, regeneration, and disease. Here, we present Mellon, a novel computational algorithm for high-resolution estimation of cell-state densities from single-cell data. We demonstrate Mellons efficacy by dissecting the density landscape of various differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. Utilizing hematopoietic stem cell fate specification to B-cells as a case study, we present evidence implicating enhancer priming and the activation of master regulators in the emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during the inherently continuous developmental processes. Scalable and adaptable, Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide new insights into the regulatory mechanisms guiding cellular fate decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549401">
        <p class="paperTitle">Fragment ion intensity prediction improves the identification rate of non-tryptic peptides in TimsTOF</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adams, C.; Gabriel, W.; Laukens, K.; Wilhelm, M.; Bittremieux, W.; Boonen, K.</p>
        <p class="info">Score: 17.1, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549401' target='https://doi.org/10.1101/2023.07.17.549401'> 10.1101/2023.07.17.549401</a></p>
        <p class="abstract">Immunopeptidomics plays a crucial role in identifying targets for immunotherapy and vaccine development. Because the generation of immunopeptides from their parent proteins does not adhere to clear-cut rules, rather than being able to use known digestion patterns, every possible protein subsequence within HLA class-specific length restrictions needs to be considered. This leads to an inflation of the search space and results in lower spectrum annotation rates. Rescoring is a powerful enhancement of standard sequence database searching that boosts the spectrum annotation performance. In the field of immunopeptidomics low abundant peptides often occur, which is why the highly sensitive timsTOF instruments are increasingly gaining popularity. To improve rescoring for immunopeptides measured using timsTOF instruments, we trained a deep learning-based fragment ion intensity prediction model. Over 300,000 synthesized non-tryptic peptides from the ProteomeTools project were analyzed on a timsTOF-Pro to generate a dataset that was used to fine-tune an existing Prosit model. By applying our fragment ion intensity prediction model, we demonstrate up to 3-fold improvement in the identification of immunopeptides. Furthermore, our approach increased detection of immunopeptides even from low input samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550821" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550821">
        <p class="paperTitle">einprot: flexible, easy-to-use, reproducible workflows for statistical analysis of quantitative proteomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soneson, C.; Iesmantavicius, V.; Hess, D.; Stadler, M. B.; Seebacher, J.</p>
        <p class="info">Score: 12.2, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550821' target='https://doi.org/10.1101/2023.07.27.550821'> 10.1101/2023.07.27.550821</a></p>
        <p class="abstract">We describe einprot, an R package providing easy-to-use reproducible workflows for quality control, statistical analysis and visualization of quantitative proteomics data. einprot is applicable to tabular output from MaxQuant, Proteome Discoverer and FragPipe, and a single function call generates an html report that describes the full analysis pipeline applied to the data and contains static and interactive figures and tables for further exploration. This has the potential to facilitate routine analyses as well as to provide a standardized, yet comprehensive way to communicate results to collaborators and the broader community. The source file underlying the report is also returned, giving the user full flexibility to further modify the workflow according to their needs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551115" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551115">
        <p class="paperTitle">disperseNN2: a neural network for estimating dispersal distance from georeferenced polymorphism data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, C. C.; Kern, A. D.</p>
        <p class="info">Score: 11.9, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551115' target='https://doi.org/10.1101/2023.07.30.551115'> 10.1101/2023.07.30.551115</a></p>
        <p class="abstract">We present a deep learning tool, disperseNN2, for estimating the mean per-generation dispersal distance from georeferenced polymorphism data. Our neural network performs feature extraction on pairs of genotypes, and uses the geographic information that comes with each sample. These attributes led disperseNN2 to outperform a state-of-the-art deep learning method that does not use explicit spatial information: error was reduced by 33% and 42% using sample sizes of 10 and 100 individuals, respectively. disperseNN2 is particularly useful for non-model organisms or systems with sparse genomic resources, as it uses unphased, single nucleotide polymorphisms as its input.

Availability and Implementationhttps://github.com/kr-colab/disperseNN2

Contactchriscs@uoregon.edu</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550718" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550718">
        <p class="paperTitle">Sensitive remote homology search by local alignment of small positional embeddings from protein language models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, S. R.; Peshwa, M.; Sun, Z.</p>
        <p class="info">Score: 16.4, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550718' target='https://doi.org/10.1101/2023.07.26.550718'> 10.1101/2023.07.26.550718</a></p>
        <p class="abstract">Accurately detecting distant evolutionary relationships between proteins remains an ongoing challenge in bioinformatics. Search methods based on primary sequence struggle to accurately detect homology between sequences with less than 20% amino acid identity. Profile- and structure-based strategies extend sensitive search capabilities into this twilight zone of sequence similarity but require slow pre-processing steps. Recently, whole-protein and positional embeddings from deep neural networks have shown promise for providing sensitive sequence comparison and annotation at long evolutionary distances. Embeddings are generally faster to compute than profiles and predicted structures but still suffer several drawbacks related to the ability of whole-protein embeddings to discriminate domain-level homology, and the database size and search speed of methods using positional embeddings. In this work, we show that low-dimensionality positional embeddings can be used directly in speed-optimized local search algorithms. As a proof of concept, we use the ESM2 3B model to convert primary sequences directly into the 3Di alphabet or amino acid profiles and use these embeddings as input to the highly optimized Foldseek, HMMER3, and HH-suite search algorithms. Our results suggest that positional embeddings as small as a single byte can provide sufficient information for dramatically improved sensitivity over amino acid sequence searches without sacrificing search speed.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
