<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Identifying cell states in single-cell RNA-seq data at statistically maximal resolution
Authors: Grobecker, P.; van Nimwegen, E.
Score: 32.0, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564980
Single-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Identifying cell states in single-cell RNA-seq data at statistically maximal resolution
Authors: Grobecker, P.; van Nimwegen, E.
Score: 32.0, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564980
Single-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Identifying cell states in single-cell RNA-seq data at statistically maximal resolution
Authors: Grobecker, P.; van Nimwegen, E.
Score: 32.0, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564980
Single-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Identifying cell states in single-cell RNA-seq data at statistically maximal resolution\nAuthors: Grobecker, P.; van Nimwegen, E.\nScore: 32.0, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564980\nSingle-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties.",
  "keywords": [
    
  ],
  "articleBody": " Identifying cell states in single-cell RNA-seq data at statistically maximal resolution\nAuthors: Grobecker, P.; van Nimwegen, E.\nScore: 32.0, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564980\nSingle-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties. Although a vast number of different methods is applied in the literature for clustering cells into subsets with `similar' expression profiles, these methods generally lack rigorously specified objectives, involve multiple complex layers of normalization, filtering, feature selection, dimensionality-reduction, employ ad hoc measures of distance or similarity between cells, often ignore the known measurement noise properties of scRNA-seq measurements, and include a large number of tunable parameters. Consequently, it is virtually impossible to assign concrete biophysical meaning to the clusterings that result from these methods. Here we address the following problem: Given raw unique molecule identifier (UMI) counts of an scRNA-seq dataset, partition the cells into subsets such that the gene expression states of the cells in each subset are statistically indistinguishable, and each subset corresponds to a distinct gene expression state. That is, we aim to partition cells so as to maximally reduce the complexity of the dataset without removing any of its meaningful structure. We show that, given the known measurement noise structure of scRNA-seq data, this problem is mathematically well-defined and derive its unique solution from first principles. We have implemented this solution in a tool called Cellstates which operates directly on the raw data and automatically determines the optimal partition and cluster number, with zero tunable parameters. We show that, on synthetic datasets, Cellstates almost perfectly recovers optimal partitions. On real data, Cellstates robustly identifies subtle substructure within groups of cells that are traditionally annotated as a common cell type. Moreover, we show that the diversity of gene expression states that Cellstates identifies systematically depends on the tissue of origin and not on technical features of the experiments such as the total number of cells and total UMI count per cell. In addition to the Cellstates tool we also provide a small toolbox of software to place the identified cellstates into a hierarchical tree of higher-order clusters, to identify the most important marker genes at each branch of this hierarchy, and to visualize these results.\nRosace: a robust deep mutational scanning analysis framework employing position and mean-variance shrinkage\nAuthors: Rao, J.; Xin, R.; Macdonald, C.; Howard, M.; Estevam, G. O.; Yee, S. W.; Wang, M.; Fraser, J. S.; Coyote-Maestas, W.; Pimentel, H.\nScore: 25.7, Published: 2023-10-29 DOI: 10.1101/2023.10.24.562292\nDeep mutational scanning (DMS) enables functional insight into protein mutations with multiplexed measurements of thousands of genetic variants in a protein simultaneously. The small sample size of DMS renders classical statistical methods ineffective, for example, p-values cannot be correctly calibrated when treating variants independently. We propose Rosace, a Bayesian framework for analyzing growth-based deep mutational scanning data. Rosace leverages amino acid position information to increase power and control the false discovery rate by sharing information across parameters via shrinkage. To benchmark Rosace against existing methods, we developed Rosette, a simulation framework that simulates the distributional properties of DMS. Further, we show that Rosace is robust to the violation of model assumptions and is more powerful than existing tools under Rosette simulation and real data.\ncisDynet: an integrated platform for modeling gene-regulatory dynamics and networks\nAuthors: Zhu, T.; Zhou, X.; You, Y.; Wang, L.; He, Z.; Chen, D.\nScore: 24.0, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564662\nChromatin accessibility sequencing has been widely used for uncovering genetic regulatory mechanisms and inferring gene regulatory networks. However, effectively integrating large-scale chromatin accessibility datasets has posed a significant challenge. This is due to the lack of a comprehensive end-to-end solution, as many existing tools primarily emphasize data pre-processing and overlook downstream analyses. To bridge this gap, we have introduced cisDynet, a holistic solution that combines streamlined data pre-processing using Snakemake and R functions with advanced downstream analysis capabilities. cisDynet excels in conventional data analyses, encompassing peak statistics, peak annotation, differential analysis, motif enrichment analysis and more. Additionally, it allows to perform sophisticated data exploration such as tissue-specific peak identification, time-course data fitting, integration of RNA-seq data to establish peak-to-gene associations, constructing regulatory networks, and conducting enrichment analysis of GWAS variants. As a proof of concept, we applied cisDynet to re-analyze the comprehensive ATAC-seq datasets across various tissues from the ENCODE project. The analysis successfully delineated tissue-specific open chromatin regions (OCRs), established connections between OCRs and target genes, and effectively linked these discoveries with 1,861 GWAS variants. Furthermore, cisDynet was instrumental in dissecting the time-course open chromatin data of mouse embryonic development, revealing the dynamic behavior of OCRs over time and identifying key transcription factors governing differentiation trajectories. In summary, cisDynet offers researchers a user-friendly solution that minimizes the need for extensive coding, ensures the reproducibility of results, and greatly simplifies the exploration of epigenomic data.\ndms-viz: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets\nAuthors: Hannon, W. W.; Bloom, J. D.\nScore: 21.5, Published: 2023-11-01 DOI: 10.1101/2023.10.29.564578\nUnderstanding how mutations impact a protein's functions is valuable for many types of biological questions. High-throughput techniques such as deep-mutational scanning (DMS) have greatly expanded the number of mutation-function datasets. For instance, DMS has been used to determine how mutations to viral proteins affect antibody escape, receptor affinity, and essential functions such as viral genome transcription and replication. With the growth of sequence databases, in some cases the effects of mutations can also be inferred from phylogenies of natural sequences. The mutation-based data generated by these approaches is often best understood in the context of a protein's 3D structure; for instance, to assess questions like how mutations that affect antibody escape relate to the physical antibody binding epitope on the protein. However, current approaches for visualizing mutation data in the context of a protein's structure are often cumbersome and require multiple steps and softwares. To streamline the visualization of mutation-associated data in the context of a protein structure, we developed a web-based tool, dms-viz. With dms-viz, users can straightforwardly visualize mutation-based data such as those from DMS experiments in the context of a 3D protein model in an interactive format. See https://dms-viz.github.io/ to use dms-viz.\nDeLoop: a deep learning model for chromatin loop prediction from sparse ATAC-seq data\nAuthors: Luo, Y.; Zhang, Z.\nScore: 17.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564594\nDeciphering gene regulation and understanding the functional implications of disease-associated non-coding variants require the identification of cell-type-specific 3D chromatin interactions. Current chromosome conformation capture technologies fall short in resolution when handling limited cell inputs. To address this limitation, we introduce DeLoop, a deep learning model designed to predict CTCF-mediated chromatin loops from sparse ATAC-seq data by leveraging multitask learning techniques and attention mechanisms. Our model utilizes ATAC-seq data and DNA sequence features, showcasing superior performance compared to existing state-of-the-art models, particularly under low read depth conditions, enabling accurate chromatin loop inference when sufficient cells are infeasible. In addition, generalizing across cell types, DeLoop proves effective in de novo prediction tasks and its potential for predicting functional interactions.\nNetREm: Network Regression Embeddings reveal cell-type transcription factor coordination for gene regulation\nAuthors: Khullar, S.; Huang, X.; Ramesh, R.; Svaren, J.; Wang, D.\nScore: 17.0, Published: 2023-10-30 DOI: 10.1101/2023.10.25.563769\nTranscription factor (TF) coordination plays a key role in gene regulation such as protein-protein interactions (PPIs) and DNA co-bindings. Single-cell technologies facilitate gene expression measurement for individual cells and cell-type identification, yet the connection between TF coordination and gene regulation of various cell types remains unclear. To address this, we have developed a novel computational approach, Network Regression Embedding (NetREm), to reveal cell-type TF-TF coordination activities for gene regulation. NetREm leverages network-constrained regularization using prior interaction knowledge (e.g., protein, chromatin, TF binding) to analyze single-cell gene expression data. We test NetREm by simulation data and apply it to analyze various cell types in both central and peripheral nerve systems (PNS) such as neuronal, glial and Schwann cells as well as in Alzheimers disease (AD). Our findings uncover cell-type coordinating TFs and identify new TF-target gene candidate links. We also validate our top predictions using Cut\u0026Run and knockout loss-of-function expression data in rat and mouse models and compare our results with additional functional genomic data including expression quantitative trait loci (eQTL) and Genome-Wide Association Studies (GWAS) to link genetic variants to TF coordination. NetREm is open-source available at https://github.com/SaniyaKhullar/NetREm.\nCartography of the developability landscapes of native and human-engineered antibodies\nAuthors: Bashour, H.; Smorodina, E.; Pariset, M.; Zhong, J.; Akbar, R.; Chernigovskaya, M.; Le Quy, K.; Snapkov, I.; Rawat, P.; Krawczyk, K.; Sandve, G. K.; Gutierrez-Marcos, J.; Nakhaee-Zadeh Gutierrez, D.; Andersen, J. T.; Greiff, V.\nScore: 16.9, Published: 2023-10-30 DOI: 10.1101/2023.10.26.563958\nDesigning effective monoclonal antibody (mAb) therapeutics face a significant challenge known as \"developability\", which reflects an antibodys ability to progress through development stages based on its physicochemical properties. While natural antibody repertoires may provide valuable guidance for antibody selection, we lack a comprehensive understanding of natural developability parameter (DP) plasticity (redundancy, predictability, sensitivity) and how the DP landscapes of human-engineered and natural antibodies relate to one another. These knowledge gaps hinder fundamental developability profile cartography. To chart natural and engineered DP landscapes, we computed 40 sequence- and 46 structure-based DPs of over two million native and human-engineered single-chain antibody sequences. We found lower redundancy among structure-based compared to sequence-based DPs. Sequence DP sensitivity to single amino acid substitutions varied by antibody region and DPs, and structure DP values varied across the conformational ensemble of antibody structures. Sequence DPs were more predictable than structure-based ones across different machine learning (ML) tasks and embeddings, indicating a constrained sequence-based design space. Human-engineered antibodies were localized within the developability and sequence landscapes of natural antibodies, suggesting that human-engineered antibodies explore mere subspaces of the natural one. Our work charts a roadmap to evaluate the plasticity of antibody developability, providing a more rational perspective for therapeutic mAb design.\nCoenzyme-Protein Interactions since Early Life\nAuthors: Sanchez Rocha, A. C.; Makarov, M.; Pravda, L.; Novotny, M.; Hlouchova, K.\nScore: 16.5, Published: 2023-11-01 DOI: 10.1101/2023.10.28.563965\nRecent findings in protein evolution and peptide prebiotic plausibility have been setting the stage for reconsidering the role of peptides in the early stages of life's origin. Ancient protein families have been found to share common themes and proteins reduced in composition to prebiotically plausible amino acids have been reported capable of structure formation and key functions, such as binding to RNA. While this may suggest peptide relevance in early life, their functional repertoire when composed of a limited number of early residues (missing some of the most sophisticated functional groups of today's alphabet) has been debated. Cofactors enrich the functional scope of about half of extant enzymes but whether they could also bind to peptides lacking the evolutionary late amino acids remains speculative. The aim of this study was to resolve the early peptide propensity to bind organic cofactors by analysis of protein-coenzyme interactions across the Protein Data Bank (PDB). We find that the prebiotically plausible amino acids are more abundant in the binding sites of the most ancient coenzymes and that such interactions rely more frequently on the involvement of the protein backbone atoms and metal ion cofactors. Moreover, we have identified a few select examples in today's enzymes where coenzyme binding is supported solely by prebiotically available amino acids. These results imply the plausibility of a coenzyme-peptide functional collaboration preceding the establishment of the Central Dogma and full protein alphabet evolution.\nBack to sequences: find the origin of kmers\nAuthors: Baire, A.; Peterlongo, P.\nScore: 15.9, Published: 2023-10-29 DOI: 10.1101/2023.10.26.564040\nA vast majority of bioinformatics tools dedicated to the treatment of raw sequencing data heavily use the concept of k-mers. This enables us to reduce the data redundancy (and thus the memory pressure), to discard sequencing errors, and to dispose of objects of fixed size that can be manipulated and easily compared to each others. A drawback is that the link between each k-mer and the original set of sequences it belongs to is generally lost. Given the volume of data considered in this context, finding back this association is costly. In this work, we present \"back_to_sequences\", a simple tool designed to index a set of k-mers of interests, and to stream a set of sequences, extracting those containing at least one of the indexed k-mer. In addition, the number of occurrences of k-mers in the sequences is provided. Our results show that back_to_sequences streams{approx} 200 short read per millisecond, enabling to search k-mers in hundreds of millions of reads in a matter of a few minutes. Availabilitygithub.com/pierrepeterlongo/back_to_sequences\nBioMANIA: Simplifying bioinformatics data analysis through conversation\nAuthors: Dong, Z.; Zhong, V.; Lu, Y.\nScore: 13.2, Published: 2023-11-01 DOI: 10.1101/2023.10.29.564479\nThe burgeoning field of bioinformatics has been revolutionized by the rapid growth of omics data, providing insights into various biological processes. However, the complexity of bioinformatics tools and the rapidly evolving nature of data analysis pipelines present significant challenges for researchers, especially those lacking extensive programming expertise. To address these challenges, we introduce BioMANIA, an artificial intelligence-driven, natural language-oriented bioinformatics data analysis pipeline. BioMANIA comprises two key components: a ChatBot generation pipeline and a user-friendly ChatBot back-end service. The generation pipeline takes as input an open-source (e.g., hosted in GitHub) and well-documented (e.g., hosted in ReadTheDocs) Python tool, extracting API attributes and generating synthetic instructions that train a ChatBot to understand and perform specific data analysis tasks. We identified 11 common issues to provide a practical guideline for designing more ChatBot-compatible tools, which we categorized into five groups, while analyzing 12 well-documented open-source Python tools across various bioinformatics settings. The ChatBot service then assists users in selecting the appropriate analysis API and parameters, significantly reducing the programming barrier. We applied BioMANIA to analyze single-cell gene expression data, demonstrating its effectiveness in simplifying complex omics data analysis. BioMANIA has the potential to transform the landscape of bioinformatics research, making data analysis more accessible and accelerating discoveries in the field.\n",
  "wordCount" : "2343",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564980">
        <p class="paperTitle">Identifying cell states in single-cell RNA-seq data at statistically maximal resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grobecker, P.; van Nimwegen, E.</p>
        <p class="info">Score: 32.0, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564980' target='https://doi.org/10.1101/2023.10.31.564980'> 10.1101/2023.10.31.564980</a></p>
        <p class="abstract">Single-cell RNA sequencing (scRNA-seq) has become a popular experimental method to study variation of gene expression within a population of cells. However, obtaining an accurate picture of the diversity of distinct gene expression states that are present in a given dataset is highly challenging because the sparsity of the scRNA-seq data and its inhomogeneous measurement noise properties. Although a vast number of different methods is applied in the literature for clustering cells into subsets with `similar&#39; expression profiles, these methods generally lack rigorously specified objectives, involve multiple complex layers of normalization, filtering, feature selection, dimensionality-reduction, employ ad hoc measures of distance or similarity between cells, often ignore the known measurement noise properties of scRNA-seq measurements, and include a large number of tunable parameters. Consequently, it is virtually impossible to assign concrete biophysical meaning to the clusterings that result from these methods. Here we address the following problem: Given raw unique molecule identifier (UMI) counts of an scRNA-seq dataset, partition the cells into subsets such that the gene expression states of the cells in each subset are statistically indistinguishable, and each subset corresponds to a distinct gene expression state. That is, we aim to partition cells so as to maximally reduce the complexity of the dataset without removing any of its meaningful structure. We show that, given the known measurement noise structure of scRNA-seq data, this problem is mathematically well-defined and derive its unique solution from first principles. We have implemented this solution in a tool called Cellstates which operates directly on the raw data and automatically determines the optimal partition and cluster number, with zero tunable parameters. We show that, on synthetic datasets, Cellstates almost perfectly recovers optimal partitions. On real data, Cellstates robustly identifies subtle substructure within groups of cells that are traditionally annotated as a common cell type. Moreover, we show that the diversity of gene expression states that Cellstates identifies systematically depends on the tissue of origin and not on technical features of the experiments such as the total number of cells and total UMI count per cell. In addition to the Cellstates tool we also provide a small toolbox of software to place the identified cellstates into a hierarchical tree of higher-order clusters, to identify the most important marker genes at each branch of this hierarchy, and to visualize these results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.562292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.562292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.562292">
        <p class="paperTitle">Rosace: a robust deep mutational scanning analysis framework employing position and mean-variance shrinkage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.562292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.562292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rao, J.; Xin, R.; Macdonald, C.; Howard, M.; Estevam, G. O.; Yee, S. W.; Wang, M.; Fraser, J. S.; Coyote-Maestas, W.; Pimentel, H.</p>
        <p class="info">Score: 25.7, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.562292' target='https://doi.org/10.1101/2023.10.24.562292'> 10.1101/2023.10.24.562292</a></p>
        <p class="abstract">Deep mutational scanning (DMS) enables functional insight into protein mutations with multiplexed measurements of thousands of genetic variants in a protein simultaneously. The small sample size of DMS renders classical statistical methods ineffective, for example, p-values cannot be correctly calibrated when treating variants independently. We propose Rosace, a Bayesian framework for analyzing growth-based deep mutational scanning data. Rosace leverages amino acid position information to increase power and control the false discovery rate by sharing information across parameters via shrinkage. To benchmark Rosace against existing methods, we developed Rosette, a simulation framework that simulates the distributional properties of DMS. Further, we show that Rosace is robust to the violation of model assumptions and is more powerful than existing tools under Rosette simulation and real data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564662">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564662" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564662">
        <p class="paperTitle">cisDynet: an integrated platform for modeling gene-regulatory dynamics and networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564662" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564662" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, T.; Zhou, X.; You, Y.; Wang, L.; He, Z.; Chen, D.</p>
        <p class="info">Score: 24.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564662' target='https://doi.org/10.1101/2023.10.30.564662'> 10.1101/2023.10.30.564662</a></p>
        <p class="abstract">Chromatin accessibility sequencing has been widely used for uncovering genetic regulatory mechanisms and inferring gene regulatory networks. However, effectively integrating large-scale chromatin accessibility datasets has posed a significant challenge. This is due to the lack of a comprehensive end-to-end solution, as many existing tools primarily emphasize data pre-processing and overlook downstream analyses. To bridge this gap, we have introduced cisDynet, a holistic solution that combines streamlined data pre-processing using Snakemake and R functions with advanced downstream analysis capabilities. cisDynet excels in conventional data analyses, encompassing peak statistics, peak annotation, differential analysis, motif enrichment analysis and more. Additionally, it allows to perform sophisticated data exploration such as tissue-specific peak identification, time-course data fitting, integration of RNA-seq data to establish peak-to-gene associations, constructing regulatory networks, and conducting enrichment analysis of GWAS variants. As a proof of concept, we applied cisDynet to re-analyze the comprehensive ATAC-seq datasets across various tissues from the ENCODE project. The analysis successfully delineated tissue-specific open chromatin regions (OCRs), established connections between OCRs and target genes, and effectively linked these discoveries with 1,861 GWAS variants. Furthermore, cisDynet was instrumental in dissecting the time-course open chromatin data of mouse embryonic development, revealing the dynamic behavior of OCRs over time and identifying key transcription factors governing differentiation trajectories. In summary, cisDynet offers researchers a user-friendly solution that minimizes the need for extensive coding, ensures the reproducibility of results, and greatly simplifies the exploration of epigenomic data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564578">
        <p class="paperTitle">dms-viz: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hannon, W. W.; Bloom, J. D.</p>
        <p class="info">Score: 21.5, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564578' target='https://doi.org/10.1101/2023.10.29.564578'> 10.1101/2023.10.29.564578</a></p>
        <p class="abstract">Understanding how mutations impact a protein&#39;s functions is valuable for many types of biological questions. High-throughput techniques such as deep-mutational scanning (DMS) have greatly expanded the number of mutation-function datasets. For instance, DMS has been used to determine how mutations to viral proteins affect antibody escape, receptor affinity, and essential functions such as viral genome transcription and replication. With the growth of sequence databases, in some cases the effects of mutations can also be inferred from phylogenies of natural sequences. The mutation-based data generated by these approaches is often best understood in the context of a protein&#39;s 3D structure; for instance, to assess questions like how mutations that affect antibody escape relate to the physical antibody binding epitope on the protein. However, current approaches for visualizing mutation data in the context of a protein&#39;s structure are often cumbersome and require multiple steps and softwares. To streamline the visualization of mutation-associated data in the context of a protein structure, we developed a web-based tool, dms-viz. With dms-viz, users can straightforwardly visualize mutation-based data such as those from DMS experiments in the context of a 3D protein model in an interactive format. See https://dms-viz.github.io/ to use dms-viz.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564594" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564594">
        <p class="paperTitle">DeLoop: a deep learning model for chromatin loop prediction from sparse ATAC-seq data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, Y.; Zhang, Z.</p>
        <p class="info">Score: 17.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564594' target='https://doi.org/10.1101/2023.11.01.564594'> 10.1101/2023.11.01.564594</a></p>
        <p class="abstract">Deciphering gene regulation and understanding the functional implications of disease-associated non-coding variants require the identification of cell-type-specific 3D chromatin interactions. Current chromosome conformation capture technologies fall short in resolution when handling limited cell inputs. To address this limitation, we introduce DeLoop, a deep learning model designed to predict CTCF-mediated chromatin loops from sparse ATAC-seq data by leveraging multitask learning techniques and attention mechanisms. Our model utilizes ATAC-seq data and DNA sequence features, showcasing superior performance compared to existing state-of-the-art models, particularly under low read depth conditions, enabling accurate chromatin loop inference when sufficient cells are infeasible. In addition, generalizing across cell types, DeLoop proves effective in de novo prediction tasks and its potential for predicting functional interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563769">
        <p class="paperTitle">NetREm: Network Regression Embeddings reveal cell-type transcription factor coordination for gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khullar, S.; Huang, X.; Ramesh, R.; Svaren, J.; Wang, D.</p>
        <p class="info">Score: 17.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563769' target='https://doi.org/10.1101/2023.10.25.563769'> 10.1101/2023.10.25.563769</a></p>
        <p class="abstract">Transcription factor (TF) coordination plays a key role in gene regulation such as protein-protein interactions (PPIs) and DNA co-bindings. Single-cell technologies facilitate gene expression measurement for individual cells and cell-type identification, yet the connection between TF coordination and gene regulation of various cell types remains unclear. To address this, we have developed a novel computational approach, Network Regression Embedding (NetREm), to reveal cell-type TF-TF coordination activities for gene regulation. NetREm leverages network-constrained regularization using prior interaction knowledge (e.g., protein, chromatin, TF binding) to analyze single-cell gene expression data. We test NetREm by simulation data and apply it to analyze various cell types in both central and peripheral nerve systems (PNS) such as neuronal, glial and Schwann cells as well as in Alzheimers disease (AD). Our findings uncover cell-type coordinating TFs and identify new TF-target gene candidate links. We also validate our top predictions using Cut&amp;Run and knockout loss-of-function expression data in rat and mouse models and compare our results with additional functional genomic data including expression quantitative trait loci (eQTL) and Genome-Wide Association Studies (GWAS) to link genetic variants to TF coordination. NetREm is open-source available at https://github.com/SaniyaKhullar/NetREm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563958">
        <p class="paperTitle">Cartography of the developability landscapes of native and human-engineered antibodies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bashour, H.; Smorodina, E.; Pariset, M.; Zhong, J.; Akbar, R.; Chernigovskaya, M.; Le Quy, K.; Snapkov, I.; Rawat, P.; Krawczyk, K.; Sandve, G. K.; Gutierrez-Marcos, J.; Nakhaee-Zadeh Gutierrez, D.; Andersen, J. T.; Greiff, V.</p>
        <p class="info">Score: 16.9, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563958' target='https://doi.org/10.1101/2023.10.26.563958'> 10.1101/2023.10.26.563958</a></p>
        <p class="abstract">Designing effective monoclonal antibody (mAb) therapeutics face a significant challenge known as &#34;developability&#34;, which reflects an antibodys ability to progress through development stages based on its physicochemical properties. While natural antibody repertoires may provide valuable guidance for antibody selection, we lack a comprehensive understanding of natural developability parameter (DP) plasticity (redundancy, predictability, sensitivity) and how the DP landscapes of human-engineered and natural antibodies relate to one another. These knowledge gaps hinder fundamental developability profile cartography. To chart natural and engineered DP landscapes, we computed 40 sequence- and 46 structure-based DPs of over two million native and human-engineered single-chain antibody sequences. We found lower redundancy among structure-based compared to sequence-based DPs. Sequence DP sensitivity to single amino acid substitutions varied by antibody region and DPs, and structure DP values varied across the conformational ensemble of antibody structures. Sequence DPs were more predictable than structure-based ones across different machine learning (ML) tasks and embeddings, indicating a constrained sequence-based design space. Human-engineered antibodies were localized within the developability and sequence landscapes of natural antibodies, suggesting that human-engineered antibodies explore mere subspaces of the natural one. Our work charts a roadmap to evaluate the plasticity of antibody developability, providing a more rational perspective for therapeutic mAb design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.563965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.563965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.563965">
        <p class="paperTitle">Coenzyme-Protein Interactions since Early Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.563965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.563965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez Rocha, A. C.; Makarov, M.; Pravda, L.; Novotny, M.; Hlouchova, K.</p>
        <p class="info">Score: 16.5, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.563965' target='https://doi.org/10.1101/2023.10.28.563965'> 10.1101/2023.10.28.563965</a></p>
        <p class="abstract">Recent findings in protein evolution and peptide prebiotic plausibility have been setting the stage for reconsidering the role of peptides in the early stages of life&#39;s origin. Ancient protein families have been found to share common themes and proteins reduced in composition to prebiotically plausible amino acids have been reported capable of structure formation and key functions, such as binding to RNA. While this may suggest peptide relevance in early life, their functional repertoire when composed of a limited number of early residues (missing some of the most sophisticated functional groups of today&#39;s alphabet) has been debated. Cofactors enrich the functional scope of about half of extant enzymes but whether they could also bind to peptides lacking the evolutionary late amino acids remains speculative. The aim of this study was to resolve the early peptide propensity to bind organic cofactors by analysis of protein-coenzyme interactions across the Protein Data Bank (PDB). We find that the prebiotically plausible amino acids are more abundant in the binding sites of the most ancient coenzymes and that such interactions rely more frequently on the involvement of the protein backbone atoms and metal ion cofactors. Moreover, we have identified a few select examples in today&#39;s enzymes where coenzyme binding is supported solely by prebiotically available amino acids. These results imply the plausibility of a coenzyme-peptide functional collaboration preceding the establishment of the Central Dogma and full protein alphabet evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564040">
        <p class="paperTitle">Back to sequences: find the origin of kmers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baire, A.; Peterlongo, P.</p>
        <p class="info">Score: 15.9, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564040' target='https://doi.org/10.1101/2023.10.26.564040'> 10.1101/2023.10.26.564040</a></p>
        <p class="abstract">A vast majority of bioinformatics tools dedicated to the treatment of raw sequencing data heavily use the concept of k-mers. This enables us to reduce the data redundancy (and thus the memory pressure), to discard sequencing errors, and to dispose of objects of fixed size that can be manipulated and easily compared to each others. A drawback is that the link between each k-mer and the original set of sequences it belongs to is generally lost. Given the volume of data considered in this context, finding back this association is costly. In this work, we present &#34;back_to_sequences&#34;, a simple tool designed to index a set of k-mers of interests, and to stream a set of sequences, extracting those containing at least one of the indexed k-mer. In addition, the number of occurrences of k-mers in the sequences is provided. Our results show that back_to_sequences streams{approx} 200 short read per millisecond, enabling to search k-mers in hundreds of millions of reads in a matter of a few minutes.

Availabilitygithub.com/pierrepeterlongo/back_to_sequences</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564479">
        <p class="paperTitle">BioMANIA: Simplifying bioinformatics data analysis through conversation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dong, Z.; Zhong, V.; Lu, Y.</p>
        <p class="info">Score: 13.2, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564479' target='https://doi.org/10.1101/2023.10.29.564479'> 10.1101/2023.10.29.564479</a></p>
        <p class="abstract">The burgeoning field of bioinformatics has been revolutionized by the rapid growth of omics data, providing insights into various biological processes. However, the complexity of bioinformatics tools and the rapidly evolving nature of data analysis pipelines present significant challenges for researchers, especially those lacking extensive programming expertise. To address these challenges, we introduce BioMANIA, an artificial intelligence-driven, natural language-oriented bioinformatics data analysis pipeline. BioMANIA comprises two key components: a ChatBot generation pipeline and a user-friendly ChatBot back-end service. The generation pipeline takes as input an open-source (e.g., hosted in GitHub) and well-documented (e.g., hosted in ReadTheDocs) Python tool, extracting API attributes and generating synthetic instructions that train a ChatBot to understand and perform specific data analysis tasks. We identified 11 common issues to provide a practical guideline for designing more ChatBot-compatible tools, which we categorized into five groups, while analyzing 12 well-documented open-source Python tools across various bioinformatics settings. The ChatBot service then assists users in selecting the appropriate analysis API and parameters, significantly reducing the programming barrier. We applied BioMANIA to analyze single-cell gene expression data, demonstrating its effectiveness in simplifying complex omics data analysis. BioMANIA has the potential to transform the landscape of bioinformatics research, making data analysis more accessible and accelerating discoveries in the field.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
