<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Efficient and accurate detection of viral sequences at single-cell resolution reveals novel viruses perturbing host gene expression
Authors: Luebbert, L.; Sullivan, D. K.; Carilli, M.; Eldjarn Hjorleifsson, K.; Winnett, A. V.; Chari, T.; Pachter, L.
Score: 44.3, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571168
More than 300,000 mammalian virus species are estimated to cause disease in humans. They inhabit human tissues such as the lungs, blood, and brain and often remain undetected. Efficient and accurate detection of viral infection is vital to understanding its impact on human health and to make accurate predictions to limit adverse effects, such as future epidemics.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Efficient and accurate detection of viral sequences at single-cell resolution reveals novel viruses perturbing host gene expression
Authors: Luebbert, L.; Sullivan, D. K.; Carilli, M.; Eldjarn Hjorleifsson, K.; Winnett, A. V.; Chari, T.; Pachter, L.
Score: 44.3, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571168
More than 300,000 mammalian virus species are estimated to cause disease in humans. They inhabit human tissues such as the lungs, blood, and brain and often remain undetected. Efficient and accurate detection of viral infection is vital to understanding its impact on human health and to make accurate predictions to limit adverse effects, such as future epidemics." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Efficient and accurate detection of viral sequences at single-cell resolution reveals novel viruses perturbing host gene expression
Authors: Luebbert, L.; Sullivan, D. K.; Carilli, M.; Eldjarn Hjorleifsson, K.; Winnett, A. V.; Chari, T.; Pachter, L.
Score: 44.3, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571168
More than 300,000 mammalian virus species are estimated to cause disease in humans. They inhabit human tissues such as the lungs, blood, and brain and often remain undetected. Efficient and accurate detection of viral infection is vital to understanding its impact on human health and to make accurate predictions to limit adverse effects, such as future epidemics."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Efficient and accurate detection of viral sequences at single-cell resolution reveals novel viruses perturbing host gene expression\nAuthors: Luebbert, L.; Sullivan, D. K.; Carilli, M.; Eldjarn Hjorleifsson, K.; Winnett, A. V.; Chari, T.; Pachter, L.\nScore: 44.3, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571168\nMore than 300,000 mammalian virus species are estimated to cause disease in humans. They inhabit human tissues such as the lungs, blood, and brain and often remain undetected. Efficient and accurate detection of viral infection is vital to understanding its impact on human health and to make accurate predictions to limit adverse effects, such as future epidemics.",
  "keywords": [
    
  ],
  "articleBody": " Efficient and accurate detection of viral sequences at single-cell resolution reveals novel viruses perturbing host gene expression\nAuthors: Luebbert, L.; Sullivan, D. K.; Carilli, M.; Eldjarn Hjorleifsson, K.; Winnett, A. V.; Chari, T.; Pachter, L.\nScore: 44.3, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571168\nMore than 300,000 mammalian virus species are estimated to cause disease in humans. They inhabit human tissues such as the lungs, blood, and brain and often remain undetected. Efficient and accurate detection of viral infection is vital to understanding its impact on human health and to make accurate predictions to limit adverse effects, such as future epidemics. The increasing use of high-throughput sequencing methods in research, agriculture, and healthcare provides an opportunity for the cost-effective surveillance of viral diversity and investigation of virus-disease correlation. However, existing methods for identifying viruses in sequencing data rely on and are limited to reference genomes or cannot retain single-cell resolution through cell barcode tracking. We introduce a method that accurately and rapidly detects viral sequences in bulk and single-cell transcriptomics data based on highly conserved amino acid domains, which enables the detection of RNA viruses covering up to 1012 virus species. The analysis of viral presence and host gene expression in parallel at single-cell resolution allows for the characterization of host viromes and the identification of viral tropism and host responses. We applied our method to identify novel viruses in rhesus macaque PBMC data that display cell type specificity and whose presence correlates with altered host gene expression.\nAlphaFold2 has more to learn about protein energy landscapes\nAuthors: Chakravarty, D.; Schafer, J. W.; Chen, E. A.; Thole, J.; Porter, L.\nScore: 36.7, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571380\nRecent work suggests that AlphaFold2 (AF2)-a deep learning-based model that can accurately infer protein structure from sequence-may discern important features of folded protein energy landscapes, defined by the diversity and frequency of different conformations in the folded state. Here, we test the limits of its predictive power on fold-switching proteins, which assume two structures with regions of distinct secondary and/or tertiary structure. Using several implementations of AF2, including two published enhanced sampling approaches, we generated \u003e280,000 models of 93 fold-switching proteins whose experimentally determined conformations were likely in AF2s training set. Combining all models, AF2 predicted fold switching with a modest success rate of [~]25%, indicating that it does not readily sample both experimentally characterized conformations of most fold switchers. Further, AF2s confidence metrics selected against models consistent with experimentally determined fold-switching conformations in favor of inconsistent models. Accordingly, these confidence metrics-though suggested to evaluate protein energetics reliably-did not discriminate between low and high energy states of fold-switching proteins. We then evaluated AF2s performance on seven fold-switching proteins outside of its training set, generating \u003e159,000 models in total. Fold switching was accurately predicted in one of seven targets with moderate confidence. Further, AF2 demonstrated no ability to predict alternative conformations of two newly discovered targets without homologs in the set of 93 fold switchers. These results indicate that AF2 has more to learn about the underlying energetics of protein ensembles and highlight the need for further developments of methods that readily predict multiple protein conformations.\nDeepPBS: Geometric deep learning for interpretable prediction of protein-DNA binding specificity\nAuthors: Mitra, R.; Li, J.; Sagendorf, J. M.; Jiang, Y.; Chiu, T.-P.; Rohs, R.\nScore: 35.2, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571942\nPredicting specificity in protein-DNA interactions is a challenging yet essential task for understanding gene regulation. Here, we present Deep Predictor of Binding Specificity (DeepPBS), a geometric deep-learning model designed to predict binding specificity across protein families based on protein-DNA structures. The DeepPBS architecture allows investigation of different family-specific recognition patterns. DeepPBS can be applied to predicted structures, and can aid in the modeling of protein-DNA complexes. DeepPBS is interpretable and can be used to calculate protein heavy atom-level importance scores, demonstrated as a case-study on p53-DNA interface. When aggregated at the protein residue level, these scores conform well with alanine scanning mutagenesis experimental data. The inference time for DeepPBS is sufficiently fast for analyzing simulation trajectories, as demonstrated on a molecular-dynamics simulation of a Drosophila Hox-DNA tertiary complex with its cofactor. DeepPBS and its corresponding data resources offer a foundation for machine-aided protein-DNA interaction studies, guiding experimental choices and complex design, as well as advancing our understanding of molecular interactions.\nA joint embedding of protein sequence and structure enables robust variant effect predictions\nAuthors: Blaabjerg, L. M.; Jonsson, N.; Boomsma, W.; Stein, A.; Lindorff-Larsen, K.\nScore: 26.4, Published: 2023-12-16 DOI: 10.1101/2023.12.14.571755\nThe ability to predict how amino acid changes may affect protein function has a wide range of applications including in disease variant classification and protein engineering. Many existing methods focus on learning from patterns found in either protein sequences or protein structures. Here, we present a method for integrating information from protein sequences and structures in a single model that we term SSEmb (Sequence Structure Embedding). SSEmb combines a graph representation for the protein structure with a transformer model for processing multiple sequence alignments, and we show that by integrating both types of information we obtain a variant effect prediction model that is more robust to cases where sequence information is scarce. Furthermore, we find that SSEmb learns embeddings of the sequence and structural properties that are useful for other downstream tasks. We exemplify this by training a downstream model to predict protein-protein binding sites at high accuracy using only the SSEmb embeddings as input. We envisage that SSEmb may be useful both for zero-shot predictions of variant effects and as a representation for predicting protein properties that depend on protein sequence and structure.\nHybracter: Enabling Scalable, Automated, Complete and Accurate Bacterial Genome Assemblies\nAuthors: Bouras, G.; Houtak, G.; Wick, R. R.; Mallawaarachchi, V.; Roach, M. J.; Papudeshi, B.; Judd, L. M.; Sheppard, A. E.; Edwards, R. A.; Vreugde, S.\nScore: 24.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571215\nImprovements in the accuracy and availability of long-read sequencing mean that complete bacterial genomes are now routinely reconstructed using hybrid (i.e. short- and long-reads) assembly approaches. Complete genomes allow a deeper understanding of bacterial evolution and genomic variation beyond small nucleotide variants (SNVs). They are also crucial for identifying plasmids, which often carry medically significant antimicrobial resistance (AMR) genes. However, small plasmids are often missed or misassembled by long-read assembly algorithms. Here, we present Hybracter, method for fast, automatic and scalable recovery of near-perfect complete bacterial genomes using a long-read first assembly approach. We compared Hybracter to existing automated hybrid assembly tools using a diverse panel of samples with manually curated ground truth reference genomes. We demonstrate that Hybracter is more accurate and faster than the existing gold standard automated hybrid assembler Unicycler. We also show that Hybracter with long-reads only is comparable to hybrid methods in recovering small plasmids.\nMAGqual: A standalone pipeline to assess the quality of metagenome-assembled genomes\nAuthors: Cansdale, A.; Chong, J. P. J.\nScore: 19.4, Published: 2023-12-13 DOI: 10.1101/2023.12.13.571510\nMetagenomics, the whole genome sequencing of microbial communities, has provided insight into complex ecosystems. It has facilitated the discovery of novel microorganisms, explained community interactions, and found applications in various fields. Advances in high-throughput and third-generation sequencing technologies have further fuelled its popularity. Nevertheless, managing the vast data produced and addressing variable dataset quality remain ongoing challenges. Another challenge arises from the number of assembly and binning strategies used across studies. Comparing datasets and analysis tools is complex as it requires a measure of metagenome quality. The inherent limitations of metagenomic sequencing, which often involves sequencing complex communities means community members are challenging to interrogate with traditional culturing methods leading to many lacking reference sequences. The MIMAG standards (Bowers et al., 2017) aim to provide a method to assess metagenome quality for comparison but have not been widely adopted. To bridge this gap, the MAGqual pipeline outlined here offers an accessible way to evaluate metagenome quality and generate metadata on a large scale. MAGqual is built in Snakemake to ensure readability and scalability and its open-source nature promotes accessibility, community development, and ease of updates. Here, we introduce the pipeline MAGqual (metagenome-assembled genome qualifier) and demonstrate its effectiveness at determining metagenomic dataset quality when compared to the MIMAG standards. MAGqual is built in Snakemake, R, and Python and is available under the MIT License on GitHub at https://github.com/ac1513/MAGqual.\nATOM-1: A Foundation Model for RNA Structure and Function Built on Chemical Mapping Data\nAuthors: Boyd, N.; Anderson, B. M.; Townshend, B.; Chow, R.; Stephens, C. J.; Rangan, R.; Kaplan, M.; Corley, M.; Tambe, A.; Ido, Y.; Yukich, J.; Tcheau, T.; Abdeldayem, A.; Ferns, G.; Patel, H.; Barman, S.; Schleck, A.; Sanborn, A. L.; Eismann, S.; Townshend, R. J. L.\nScore: 18.2, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571579\nRNA-based medicines and RNA-targeting drugs are emerging as promising new approaches for treating disease. Optimizing these therapeutics by naive experimental screening is a time-consuming and expensive process, while rational design requires an accurate understanding of the structure and function of RNA. To address this design challenge, we present ATOM-1, the first RNA foundation model trained on chemical mapping data, enabled by data collection strategies purposely developed for machine learning training. Using small probe neural networks on top of ATOM-1 embeddings, we demonstrate that this model has developed rich internal representations of RNA. Trained on limited amounts of additional data, these small networks achieve state-of-the-art accuracy on key RNA prediction tasks, suggesting that this approach can enable the design of therapies across the RNA landscape.\nPersonalized Pangenome References\nAuthors: Siren, J.; Eskandar, P.; Ungaro, M. T.; Hickey, G.; Eizenga, J. M.; Novak, A. M.; Chang, X.; Chang, P.-C.; Kolmogorov, M.; Carroll, A.; Monlong, J.; Paten, B.\nScore: 18.1, Published: 2023-12-15 DOI: 10.1101/2023.12.13.571553\nPangenomes, by including genetic diversity, should reduce reference bias by better representing new samples compared to them. Yet when comparing a new sample to a pangenome, variants in the pangenome that are not part of the sample can be misleading, for example, causing false read mappings. These irrelevant variants are generally rarer in terms of allele frequency, and have previously been dealt with using allele frequency filters. However, this is a blunt heuristic that both fails to remove some irrelevant variants and removes many relevant variants. We propose a new approach, inspired by local ancestry inference methods, that imputes a personalized pangenome subgraph based on sampling local haplotypes according to k-mer counts in the reads. Our approach is tailored for the Giraffe short read aligner, as the indexes it needs for read mapping can be built quickly. We compare the accuracy of our approach to state-of-the-art methods using graphs from the Human Pangenome Reference Consortium. The resulting personalized pangenome pipelines provide faster pangenome read mapping than comparable pipelines that use a linear reference, reduce small variant genotyping errors by 4x relative to the Genome Analysis Toolkit (GATK) best-practice pipeline, and for the first time make short-read structural variant genotyping competitive with long-read discovery methods.\nEncompassing view of spatial and single-cell RNA-seq renews the role of the microvasculature in human atherosclerosis\nAuthors: Bleckwehl, T.; Maryam, S.; Babler, A.; Bosteen, M.; Nyberg, M.; Halder, M.; Hvid, H.; Pyke, C.; Voetmann, L. M.; Das, V.; Baumgart, S.; Kramann, R.; Hayat, S.\nScore: 16.2, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571796\nAtherosclerosis is a pervasive contributor to cardiovascular diseases including ischemic heart disease and stroke. Despite the advance and success of effective lipid lowering- therapies and hypertensive agents, the residual risk of an atherosclerotic event remains high and improving disease understanding and development of novel therapeutic strategies has proven to be challenging. This is largely due to the complexity of atherosclerosis with a spatial interplay of multiple cell types within the vascular wall. Here, we generated an integrative high-resolution map of human atherosclerotic plaques by combining single-cell RNA-seq from multiple studies and novel spatial transcriptomics data from 12 human specimens to gain insights into disease mechanisms. Comparative analyses revealed cell-type and atherosclerosis-specific expression changes and associated alterations in cell-cell communication. We highlight the possible recruitment of lymphocytes via different endothelial cells of the vasa vasorum, the migration of vascular smooth muscle cells towards the lumen to become fibromyocytes, and cell-cell communication in the plaque, indicating an intricate cellular interplay within the adventitia and the subendothelial space in human atherosclerosis.\nMelon: metagenomic long-read-based taxonomic identification and quantification using marker genes\nAuthors: Chen, X.; Yin, X.; Shi, X.; Yan, W.; Yang, Y.; Liu, L.; Zhang, T.\nScore: 16.1, Published: 2023-12-18 DOI: 10.1101/2023.12.17.572079\nLong-read sequencing holds great potential for characterizing complex microbial communities, yet taxonomic profiling tools designed specifically for long reads remain lacking. Here we introduce Melon, a novel marker-based taxonomic profiler that capitalizes on the unique attributes of long reads. Melon employs a two-stage classification scheme to reduce computational time and is equipped with an expectation-maximization-based post-correction module to handle ambiguous reads. Melon achieves superior performance compared to other existing tools in both mock and simulated samples. Using wastewater metagenomic samples, we demonstrate the applicability of Melon by showing it is capable in providing reliable estimates of overall genome copies, and species-level taxonomic profiles.\n",
  "wordCount" : "2151",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571168" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571168">
        <p class="paperTitle">Efficient and accurate detection of viral sequences at single-cell resolution reveals novel viruses perturbing host gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luebbert, L.; Sullivan, D. K.; Carilli, M.; Eldjarn Hjorleifsson, K.; Winnett, A. V.; Chari, T.; Pachter, L.</p>
        <p class="info">Score: 44.3, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571168' target='https://doi.org/10.1101/2023.12.11.571168'> 10.1101/2023.12.11.571168</a></p>
        <p class="abstract">More than 300,000 mammalian virus species are estimated to cause disease in humans. They inhabit human tissues such as the lungs, blood, and brain and often remain undetected. Efficient and accurate detection of viral infection is vital to understanding its impact on human health and to make accurate predictions to limit adverse effects, such as future epidemics. The increasing use of high-throughput sequencing methods in research, agriculture, and healthcare provides an opportunity for the cost-effective surveillance of viral diversity and investigation of virus-disease correlation. However, existing methods for identifying viruses in sequencing data rely on and are limited to reference genomes or cannot retain single-cell resolution through cell barcode tracking. We introduce a method that accurately and rapidly detects viral sequences in bulk and single-cell transcriptomics data based on highly conserved amino acid domains, which enables the detection of RNA viruses covering up to 1012 virus species. The analysis of viral presence and host gene expression in parallel at single-cell resolution allows for the characterization of host viromes and the identification of viral tropism and host responses. We applied our method to identify novel viruses in rhesus macaque PBMC data that display cell type specificity and whose presence correlates with altered host gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571380" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571380">
        <p class="paperTitle">AlphaFold2 has more to learn about protein energy landscapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakravarty, D.; Schafer, J. W.; Chen, E. A.; Thole, J.; Porter, L.</p>
        <p class="info">Score: 36.7, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571380' target='https://doi.org/10.1101/2023.12.12.571380'> 10.1101/2023.12.12.571380</a></p>
        <p class="abstract">Recent work suggests that AlphaFold2 (AF2)-a deep learning-based model that can accurately infer protein structure from sequence-may discern important features of folded protein energy landscapes, defined by the diversity and frequency of different conformations in the folded state. Here, we test the limits of its predictive power on fold-switching proteins, which assume two structures with regions of distinct secondary and/or tertiary structure. Using several implementations of AF2, including two published enhanced sampling approaches, we generated &gt;280,000 models of 93 fold-switching proteins whose experimentally determined conformations were likely in AF2s training set. Combining all models, AF2 predicted fold switching with a modest success rate of [~]25%, indicating that it does not readily sample both experimentally characterized conformations of most fold switchers. Further, AF2s confidence metrics selected against models consistent with experimentally determined fold-switching conformations in favor of inconsistent models. Accordingly, these confidence metrics-though suggested to evaluate protein energetics reliably-did not discriminate between low and high energy states of fold-switching proteins. We then evaluated AF2s performance on seven fold-switching proteins outside of its training set, generating &gt;159,000 models in total. Fold switching was accurately predicted in one of seven targets with moderate confidence. Further, AF2 demonstrated no ability to predict alternative conformations of two newly discovered targets without homologs in the set of 93 fold switchers. These results indicate that AF2 has more to learn about the underlying energetics of protein ensembles and highlight the need for further developments of methods that readily predict multiple protein conformations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571942">
        <p class="paperTitle">DeepPBS: Geometric deep learning for interpretable prediction of protein-DNA binding specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitra, R.; Li, J.; Sagendorf, J. M.; Jiang, Y.; Chiu, T.-P.; Rohs, R.</p>
        <p class="info">Score: 35.2, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571942' target='https://doi.org/10.1101/2023.12.15.571942'> 10.1101/2023.12.15.571942</a></p>
        <p class="abstract">Predicting specificity in protein-DNA interactions is a challenging yet essential task for understanding gene regulation. Here, we present Deep Predictor of Binding Specificity (DeepPBS), a geometric deep-learning model designed to predict binding specificity across protein families based on protein-DNA structures. The DeepPBS architecture allows investigation of different family-specific recognition patterns. DeepPBS can be applied to predicted structures, and can aid in the modeling of protein-DNA complexes. DeepPBS is interpretable and can be used to calculate protein heavy atom-level importance scores, demonstrated as a case-study on p53-DNA interface. When aggregated at the protein residue level, these scores conform well with alanine scanning mutagenesis experimental data. The inference time for DeepPBS is sufficiently fast for analyzing simulation trajectories, as demonstrated on a molecular-dynamics simulation of a Drosophila Hox-DNA tertiary complex with its cofactor. DeepPBS and its corresponding data resources offer a foundation for machine-aided protein-DNA interaction studies, guiding experimental choices and complex design, as well as advancing our understanding of molecular interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571755">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571755" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571755">
        <p class="paperTitle">A joint embedding of protein sequence and structure enables robust variant effect predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571755" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571755" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blaabjerg, L. M.; Jonsson, N.; Boomsma, W.; Stein, A.; Lindorff-Larsen, K.</p>
        <p class="info">Score: 26.4, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571755' target='https://doi.org/10.1101/2023.12.14.571755'> 10.1101/2023.12.14.571755</a></p>
        <p class="abstract">The ability to predict how amino acid changes may affect protein function has a wide range of applications including in disease variant classification and protein engineering. Many existing methods focus on learning from patterns found in either protein sequences or protein structures. Here, we present a method for integrating information from protein sequences and structures in a single model that we term SSEmb (Sequence Structure Embedding). SSEmb combines a graph representation for the protein structure with a transformer model for processing multiple sequence alignments, and we show that by integrating both types of information we obtain a variant effect prediction model that is more robust to cases where sequence information is scarce. Furthermore, we find that SSEmb learns embeddings of the sequence and structural properties that are useful for other downstream tasks. We exemplify this by training a downstream model to predict protein-protein binding sites at high accuracy using only the SSEmb embeddings as input. We envisage that SSEmb may be useful both for zero-shot predictions of variant effects and as a representation for predicting protein properties that depend on protein sequence and structure.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571215">
        <p class="paperTitle">Hybracter: Enabling Scalable, Automated, Complete and Accurate Bacterial Genome Assemblies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bouras, G.; Houtak, G.; Wick, R. R.; Mallawaarachchi, V.; Roach, M. J.; Papudeshi, B.; Judd, L. M.; Sheppard, A. E.; Edwards, R. A.; Vreugde, S.</p>
        <p class="info">Score: 24.5, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571215' target='https://doi.org/10.1101/2023.12.12.571215'> 10.1101/2023.12.12.571215</a></p>
        <p class="abstract">Improvements in the accuracy and availability of long-read sequencing mean that complete bacterial genomes are now routinely reconstructed using hybrid (i.e. short- and long-reads) assembly approaches. Complete genomes allow a deeper understanding of bacterial evolution and genomic variation beyond small nucleotide variants (SNVs). They are also crucial for identifying plasmids, which often carry medically significant antimicrobial resistance (AMR) genes. However, small plasmids are often missed or misassembled by long-read assembly algorithms. Here, we present Hybracter, method for fast, automatic and scalable recovery of near-perfect complete bacterial genomes using a long-read first assembly approach. We compared Hybracter to existing automated hybrid assembly tools using a diverse panel of samples with manually curated ground truth reference genomes. We demonstrate that Hybracter is more accurate and faster than the existing gold standard automated hybrid assembler Unicycler. We also show that Hybracter with long-reads only is comparable to hybrid methods in recovering small plasmids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571510">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571510" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571510">
        <p class="paperTitle">MAGqual: A standalone pipeline to assess the quality of metagenome-assembled genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571510" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571510" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cansdale, A.; Chong, J. P. J.</p>
        <p class="info">Score: 19.4, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571510' target='https://doi.org/10.1101/2023.12.13.571510'> 10.1101/2023.12.13.571510</a></p>
        <p class="abstract">Metagenomics, the whole genome sequencing of microbial communities, has provided insight into complex ecosystems. It has facilitated the discovery of novel microorganisms, explained community interactions, and found applications in various fields. Advances in high-throughput and third-generation sequencing technologies have further fuelled its popularity. Nevertheless, managing the vast data produced and addressing variable dataset quality remain ongoing challenges. Another challenge arises from the number of assembly and binning strategies used across studies. Comparing datasets and analysis tools is complex as it requires a measure of metagenome quality. The inherent limitations of metagenomic sequencing, which often involves sequencing complex communities means community members are challenging to interrogate with traditional culturing methods leading to many lacking reference sequences.

The MIMAG standards (Bowers et al., 2017) aim to provide a method to assess metagenome quality for comparison but have not been widely adopted. To bridge this gap, the MAGqual pipeline outlined here offers an accessible way to evaluate metagenome quality and generate metadata on a large scale. MAGqual is built in Snakemake to ensure readability and scalability and its open-source nature promotes accessibility, community development, and ease of updates. Here, we introduce the pipeline MAGqual (metagenome-assembled genome qualifier) and demonstrate its effectiveness at determining metagenomic dataset quality when compared to the MIMAG standards. MAGqual is built in Snakemake, R, and Python and is available under the MIT License on GitHub at https://github.com/ac1513/MAGqual.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571579">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571579" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571579">
        <p class="paperTitle">ATOM-1: A Foundation Model for RNA Structure and Function Built on Chemical Mapping Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571579" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571579" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boyd, N.; Anderson, B. M.; Townshend, B.; Chow, R.; Stephens, C. J.; Rangan, R.; Kaplan, M.; Corley, M.; Tambe, A.; Ido, Y.; Yukich, J.; Tcheau, T.; Abdeldayem, A.; Ferns, G.; Patel, H.; Barman, S.; Schleck, A.; Sanborn, A. L.; Eismann, S.; Townshend, R. J. L.</p>
        <p class="info">Score: 18.2, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571579' target='https://doi.org/10.1101/2023.12.13.571579'> 10.1101/2023.12.13.571579</a></p>
        <p class="abstract">RNA-based medicines and RNA-targeting drugs are emerging as promising new approaches for treating disease. Optimizing these therapeutics by naive experimental screening is a time-consuming and expensive process, while rational design requires an accurate understanding of the structure and function of RNA. To address this design challenge, we present ATOM-1, the first RNA foundation model trained on chemical mapping data, enabled by data collection strategies purposely developed for machine learning training. Using small probe neural networks on top of ATOM-1 embeddings, we demonstrate that this model has developed rich internal representations of RNA. Trained on limited amounts of additional data, these small networks achieve state-of-the-art accuracy on key RNA prediction tasks, suggesting that this approach can enable the design of therapies across the RNA landscape.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571553" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571553">
        <p class="paperTitle">Personalized Pangenome References</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Siren, J.; Eskandar, P.; Ungaro, M. T.; Hickey, G.; Eizenga, J. M.; Novak, A. M.; Chang, X.; Chang, P.-C.; Kolmogorov, M.; Carroll, A.; Monlong, J.; Paten, B.</p>
        <p class="info">Score: 18.1, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571553' target='https://doi.org/10.1101/2023.12.13.571553'> 10.1101/2023.12.13.571553</a></p>
        <p class="abstract">Pangenomes, by including genetic diversity, should reduce reference bias by better representing new samples compared to them. Yet when comparing a new sample to a pangenome, variants in the pangenome that are not part of the sample can be misleading, for example, causing false read mappings. These irrelevant variants are generally rarer in terms of allele frequency, and have previously been dealt with using allele frequency filters. However, this is a blunt heuristic that both fails to remove some irrelevant variants and removes many relevant variants. We propose a new approach, inspired by local ancestry inference methods, that imputes a personalized pangenome subgraph based on sampling local haplotypes according to k-mer counts in the reads. Our approach is tailored for the Giraffe short read aligner, as the indexes it needs for read mapping can be built quickly. We compare the accuracy of our approach to state-of-the-art methods using graphs from the Human Pangenome Reference Consortium. The resulting personalized pangenome pipelines provide faster pangenome read mapping than comparable pipelines that use a linear reference, reduce small variant genotyping errors by 4x relative to the Genome Analysis Toolkit (GATK) best-practice pipeline, and for the first time make short-read structural variant genotyping competitive with long-read discovery methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571796">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571796" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571796">
        <p class="paperTitle">Encompassing view of spatial and single-cell RNA-seq renews the role of the microvasculature in human atherosclerosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571796" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571796" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bleckwehl, T.; Maryam, S.; Babler, A.; Bosteen, M.; Nyberg, M.; Halder, M.; Hvid, H.; Pyke, C.; Voetmann, L. M.; Das, V.; Baumgart, S.; Kramann, R.; Hayat, S.</p>
        <p class="info">Score: 16.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571796' target='https://doi.org/10.1101/2023.12.15.571796'> 10.1101/2023.12.15.571796</a></p>
        <p class="abstract">Atherosclerosis is a pervasive contributor to cardiovascular diseases including ischemic heart disease and stroke. Despite the advance and success of effective lipid lowering- therapies and hypertensive agents, the residual risk of an atherosclerotic event remains high and improving disease understanding and development of novel therapeutic strategies has proven to be challenging. This is largely due to the complexity of atherosclerosis with a spatial interplay of multiple cell types within the vascular wall. Here, we generated an integrative high-resolution map of human atherosclerotic plaques by combining single-cell RNA-seq from multiple studies and novel spatial transcriptomics data from 12 human specimens to gain insights into disease mechanisms. Comparative analyses revealed cell-type and atherosclerosis-specific expression changes and associated alterations in cell-cell communication. We highlight the possible recruitment of lymphocytes via different endothelial cells of the vasa vasorum, the migration of vascular smooth muscle cells towards the lumen to become fibromyocytes, and cell-cell communication in the plaque, indicating an intricate cellular interplay within the adventitia and the subendothelial space in human atherosclerosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.17.572079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.17.572079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.17.572079">
        <p class="paperTitle">Melon: metagenomic long-read-based taxonomic identification and quantification using marker genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.17.572079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.17.572079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, X.; Yin, X.; Shi, X.; Yan, W.; Yang, Y.; Liu, L.; Zhang, T.</p>
        <p class="info">Score: 16.1, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.17.572079' target='https://doi.org/10.1101/2023.12.17.572079'> 10.1101/2023.12.17.572079</a></p>
        <p class="abstract">Long-read sequencing holds great potential for characterizing complex microbial communities, yet taxonomic profiling tools designed specifically for long reads remain lacking. Here we introduce Melon, a novel marker-based taxonomic profiler that capitalizes on the unique attributes of long reads. Melon employs a two-stage classification scheme to reduce computational time and is equipped with an expectation-maximization-based post-correction module to handle ambiguous reads. Melon achieves superior performance compared to other existing tools in both mock and simulated samples. Using wastewater metagenomic samples, we demonstrate the applicability of Melon by showing it is capable in providing reliable estimates of overall genome copies, and species-level taxonomic profiles.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
