<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 142.2, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 142.2, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 142.2, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets\nAuthors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.\nScore: 142.2, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131\nedgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq.",
  "keywords": [
    
  ],
  "articleBody": " edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets\nAuthors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.\nScore: 142.2, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131\nedgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. edgeR pioneered the use of the negative binomial distribution to model read count data with replicates and the use of generalized linear models to analyse complex experimental designs. edgeR implements empirical Bayes moderation methods to allow reliable inference when the number of replicates is small. This article announces edgeR version 4, which includes new developments across a range of application areas. Infrastructure improvements include support for fractional counts, implementation of model fitting in C++, and a new statistical treatment of the quasi-likelihood pipeline that improves accuracy for small counts. The revised package has new functionality for differential methylation analysis, differential transcript expression, differential transcript and exon usage, testing relative to a fold-change threshold and pathway analysis. This article reviews the statistical framework and computational implementation of edgeR, briefly summarizing all the existing features and functionalities but with special attention to new features and those that have not been described previously.\nMetagenomic estimation of dietary intake from human stool\nAuthors: Diener, C.; Gibbons, S. M.\nScore: 53.4, Published: 2024-02-06 DOI: 10.1101/2024.02.02.578701\nDietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (\u003e10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.\nSpotiflow: accurate and efficient spot detection for imaging-based spatial transcriptomics with stereographic flow regression\nAuthors: Dominguez Mantes, A.; Herrera, A.; Khven, I.; Schlaeppi, A.; Kyriacou, E.; Tsissios, G.; Aztekin, C.; Ligner, J.; La Manno, G.; Weigert, M.\nScore: 47.1, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578426\nThe identification of spot-like structures in large and noisy microscopy images is an important task in many life science techniques, and it is essential to their quantitative performance. For example, imaging-based spatial transcriptomics (iST) methods rely critically on the accurate detection of millions of transcripts in low signal-to-noise ratio (SNR) images. While recent developments in computer vision have revolutionized many bioimage tasks, currently adopted spot detection approaches for iST still rely on classical signal processing methods that are fragile and require manually tuning. In this work we introduce Spotiflow, a deep-learning method that casts the spot-detection problem as a multiscale stereographic flow regression problem that yields subpixel-accurate localizations. Spotiflow is robust to different noise conditions and generalizes across different chemistries while being up to an order-of-magnitude more time and memory efficient than commonly used methods. We show the efficacy of Spotiflow by comprehensive quantitative comparisons against other methods on a variety of datasets and demonstrate the impact of its increased accuracy on the biological conclusions drawn from iST and live imaging experiments. Spotiflow is available as an easy-to-use Python library as well as a napari plugin at http://www.github.com/weigertlab/spotiflow.\nMulti-genome synteny detection using minimizer graph mappings\nAuthors: Coombe, L.; Kazemi, P.; Wong, J.; Birol, I.; Warren, R. L.\nScore: 24.3, Published: 2024-02-13 DOI: 10.1101/2024.02.07.579356\nIn recent years, the landscape of reference-grade genome assemblies has seen substantial diversification. With such rich data, there is pressing demand for robust tools for scalable, multi-species comparative genomics analyses, including detecting genome synteny, which informs on the sequence conservation between genomes and contributes crucial insights into species evolution. Here, we introduce ntSynt, a scalable utility for computing large-scale multi-genome synteny blocks using a minimizer graph-based approach. Through extensive testing utilizing multiple ~3 Gbp genomes, we demonstrate how ntSynt produces synteny blocks with coverages between 79-100% in at most 2h using 34 GB of memory, even for genomes with appreciable (\u003e15%) sequence divergence. Compared to existing state-of-the-art methodologies, ntSynt offers enhanced flexibility to diverse input genome sequences and synteny block granularity. We expect the macrosyntenic genome analyses facilitated by ntSynt will have broad utility in generating critical evolutionary insights within and between species across the tree of life.\nGenerRNA: A generative pre-trained language model for de novo RNA design\nAuthors: Zhao, Y.; Oono, K.; Takizawa, H.; Kotera, M.\nScore: 22.4, Published: 2024-02-08 DOI: 10.1101/2024.02.01.578496\nAO_SCPLOWBSTRACTC_SCPLOWThe design of RNA plays a crucial role in developing RNA vaccines, nucleic acid therapeutics, and innovative biotechnological tools. Nevertheless, existing techniques lack versatility across various tasks and frequently suffer from a deficiency of automated generation. Inspired by the remarkable success of Large Language Models (LLMs) in the realm of protein and molecule design, we present GenerRNA, the first large-scale pre-trained model for RNA generation, aiming to further automate RNA design. Our approach eliminates the need for secondary structure or other prior knowledge and is capable of de novo generation of RNA with stable secondary structures while ensuring its distinctiveness from existing sequences. This widens our exploration of RNA space, thereby enriching our understanding of RNA structures and functions. Moreover, GenerRNA is fine-tunable on smaller, more specialized datasets for particular subtasks. This flexibility and versatility enables the generation of RNAs with desired specific functionalities or properties. Upon fine-tuning GenerRNA, we successfully generated novel RNA sequences exhibiting high affinity for target proteins. GenerRNA is freely available at the following repository: https://github.com/pfnet-research/GenerRNA\nProteus: pioneering protein structure generation for enhanced designability and efficiency\nAuthors: Wang, C.; Qu, Y.; Peng, Z.; Wang, Y.; Zhu, H.; Chen, D.; Cao, L.\nScore: 16.5, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579791\nThe development of de novo protein design method is crucial for widespread applications in biology and chemistry. Protein backbone diffusion aims to generate designable protein structures with high efficiency. Although there have been great strides in protein structure prediction, applying these methods to protein diffusion has been challenging and inefficient. We introduce Proteus, an innovative approach that uses graph-based triangle methods and a multi-track interaction network. Proteus demonstrated cutting-edge designability and efficiency in computational evaluations. We tested the reliability of the model by experimental characterization. Our analysis indicates that from both computational and experimental perspectives, it is able to generate proteins with a remarkably high success rate. We believe Proteus's capacity to rapidly create highly designable protein backbone without the need for pre-training techniques will greatly enhance our understanding of protein structure diffusion and contribute to advances in protein design.\nCellpose3: one-click image restoration for improved cellular segmentation\nAuthors: Stringer, C.; Pachitariu, M.\nScore: 16.2, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579780\nGeneralist methods for cellular segmentation have good out-of-the-box performance on a variety of image types. However, existing methods struggle for images that are degraded by noise, blurred or undersampled, all of which are common in microscopy. We focused the development of Cellpose3 on addressing these cases, and here we demonstrate substantial out-of-the-box gains in segmentation and image quality for noisy, blurry or undersampled images. Unlike previous approaches, which train models to restore pixel values, we trained Cellpose3 to output images that are well-segmented by a generalist segmentation model, while maintaining perceptual similarity to the target images. Furthermore, we trained the restoration models on a large, varied collection of datasets, thus ensuring good generalization to user images. We provide these tools as \"one-click\" buttons inside the graphical interface of Cellpose as well as in the Cellpose API.\nAnalysis of multi-condition single-cell data with latent embedding multivariate regression\nAuthors: Ahlmann-Eltze, C.; Huber, W.\nScore: 138.4, Published: 2024-02-07 DOI: 10.1101/2023.03.06.531268\nIdentifying gene expression differences in heterogeneous samples across multiple experimental or observational conditions is a fundamental biological task, enabled by single-cell resolution assays such as multi-condition sc-RNA-seq. Current approaches divide the constituent cells into discrete clusters meant to represent cell types, and identify differentially expressed genes for each cluster. However, such discrete categorization is often unsatisfactory: if too coarse, more subtle but important differences may be overlooked, if too fine-grained, statistical power is lost and results become unwieldy. Here, we introduce Latent Embedding Multivariate Regression (LEMUR), which enables differential expression analysis using a continuous lowdimensional latent space representation of cell type and state diversity, and thus operates without (or before) commitment to discrete categorization. LEMUR (1) aligns the data from the different conditions, (2) predicts how a cells gene expression changes as a function of the conditions and its position in latent space, and (3) identifies compact neighborhoods of cells with consistent differential expression for each gene. Unlike static clusters, these neighborhoods adapt to the underlying expression changes per gene. We validate LEMUR on a large compendium of single-cell datasets and show applications to the identification of subpopulations of tumor cells with distinct drug responses, to the characterization of the interplay between cell state and developmental time in Zebrafish embryos, and to the discovery of cell state-specific gene regulation in a spatial single-cell dataset of Alzheimers plaques. Software availabilityhttps://bioconductor.org/packages/lemur Contactconstantin.ahlmann@embl.de\nBiaPy: A unified framework for versatile bioimage analysis with deep learning\nAuthors: Franco-Barranco, D.; Andres-San Roman, J. A.; Hidalgo-Cenalmor, I.; Backova, L.; Gonzalez-Marfil, A.; Caporal, C.; Chessel, A.; Gomez-Galvez, P.; Escudero, L. M.; Wei, D.; Munoz-Barrutia, A.; Arganda-Carreras, I.\nScore: 25.9, Published: 2024-02-05 DOI: 10.1101/2024.02.03.576026\nBiaPy, a unified open-source bioimage analysis library, offers a comprehensive suite of deep learning-powered workflows. Tailored for users of all levels, BiaPy features an intuitive interface, zero-code notebooks, and Docker integration. With support for 2D and 3D image data, it addresses existing gaps by providing multi-GPU capabilities, memory optimization, and compatibility with large datasets. As a collaborative and accessible solution, BiaPy aims to empower researchers by democratizing the use of sophisticated and efficient bioimage analysis workflows.\nInvestigating the performance of foundation models on human 3'UTR sequences\nAuthors: Vilov, S.; Heinig, M.\nScore: 13.2, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579631\nFoundation models, such as DNABERT and Nucleotide Transformer have recently shaped a new direction in DNA research. Trained in an unsupervised manner on a vast quantity of genomic data, they can be used for a variety of downstream tasks, such as promoter prediction, DNA methylation prediction, gene network prediction or functional variant prioritization. However, these models are often trained and evaluated on entire genomes, neglecting genome partitioning into different functional regions. In our study, we investigate the efficacy of various unsupervised approaches, including genome-wide and 3'UTR-specific foundation models on human 3'UTR regions. Our evaluation includes downstream tasks specific for RNA biology, such as recognition of binding motifs of RNA binding proteins, detection of functional genetic variants, prediction of expression levels in massively parallel reporter assays, and estimation of mRNA half-life. Remarkably, models specifically trained on 3'UTR sequences demonstrate superior performance when compared to the established genome-wide foundation models in three out of four downstream tasks. Our results underscore the importance of considering genome partitioning into functional regions when training and evaluating foundation models.\n",
  "wordCount" : "1923",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576131" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576131">
        <p class="paperTitle">edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.</p>
        <p class="info">Score: 142.2, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576131' target='https://doi.org/10.1101/2024.01.21.576131'> 10.1101/2024.01.21.576131</a></p>
        <p class="abstract">edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. edgeR pioneered the use of the negative binomial distribution to model read count data with replicates and the use of generalized linear models to analyse complex experimental designs. edgeR implements empirical Bayes moderation methods to allow reliable inference when the number of replicates is small. This article announces edgeR version 4, which includes new developments across a range of application areas. Infrastructure improvements include support for fractional counts, implementation of model fitting in C&#43;&#43;, and a new statistical treatment of the quasi-likelihood pipeline that improves accuracy for small counts. The revised package has new functionality for differential methylation analysis, differential transcript expression, differential transcript and exon usage, testing relative to a fold-change threshold and pathway analysis. This article reviews the statistical framework and computational implementation of edgeR, briefly summarizing all the existing features and functionalities but with special attention to new features and those that have not been described previously.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578701">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578701" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578701">
        <p class="paperTitle">Metagenomic estimation of dietary intake from human stool</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578701" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578701" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diener, C.; Gibbons, S. M.</p>
        <p class="info">Score: 53.4, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578701' target='https://doi.org/10.1101/2024.02.02.578701'> 10.1101/2024.02.02.578701</a></p>
        <p class="abstract">Dietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (&gt;10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578426" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578426">
        <p class="paperTitle">Spotiflow: accurate and efficient spot detection for imaging-based spatial transcriptomics with stereographic flow regression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dominguez Mantes, A.; Herrera, A.; Khven, I.; Schlaeppi, A.; Kyriacou, E.; Tsissios, G.; Aztekin, C.; Ligner, J.; La Manno, G.; Weigert, M.</p>
        <p class="info">Score: 47.1, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578426' target='https://doi.org/10.1101/2024.02.01.578426'> 10.1101/2024.02.01.578426</a></p>
        <p class="abstract">The identification of spot-like structures in large and noisy microscopy images is an important task in many life science techniques, and it is essential to their quantitative performance. For example, imaging-based spatial transcriptomics (iST) methods rely critically on the accurate detection of millions of transcripts in low signal-to-noise ratio (SNR) images. While recent developments in computer vision have revolutionized many bioimage tasks, currently adopted spot detection approaches for iST still rely on classical signal processing methods that are fragile and require manually tuning. In this work we introduce Spotiflow, a deep-learning method that casts the spot-detection problem as a multiscale stereographic flow regression problem that yields subpixel-accurate localizations. Spotiflow is robust to different noise conditions and generalizes across different chemistries while being up to an order-of-magnitude more time and memory efficient than commonly used methods. We show the efficacy of Spotiflow by comprehensive quantitative comparisons against other methods on a variety of datasets and demonstrate the impact of its increased accuracy on the biological conclusions drawn from iST and live imaging experiments. Spotiflow is available as an easy-to-use Python library as well as a napari plugin at http://www.github.com/weigertlab/spotiflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579356" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579356">
        <p class="paperTitle">Multi-genome synteny detection using minimizer graph mappings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coombe, L.; Kazemi, P.; Wong, J.; Birol, I.; Warren, R. L.</p>
        <p class="info">Score: 24.3, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579356' target='https://doi.org/10.1101/2024.02.07.579356'> 10.1101/2024.02.07.579356</a></p>
        <p class="abstract">In recent years, the landscape of reference-grade genome assemblies has seen substantial diversification. With such rich data, there is pressing demand for robust tools for scalable, multi-species comparative genomics analyses, including detecting genome synteny, which informs on the sequence conservation between genomes and contributes crucial insights into species evolution. Here, we introduce ntSynt, a scalable utility for computing large-scale multi-genome synteny blocks using a minimizer graph-based approach. Through extensive testing utilizing multiple ~3 Gbp genomes, we demonstrate how ntSynt produces synteny blocks with coverages between 79-100% in at most 2h using 34 GB of memory, even for genomes with appreciable (&gt;15%) sequence divergence. Compared to existing state-of-the-art methodologies, ntSynt offers enhanced flexibility to diverse input genome sequences and synteny block granularity. We expect the macrosyntenic genome analyses facilitated by ntSynt will have broad utility in generating critical evolutionary insights within and between species across the tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578496">
        <p class="paperTitle">GenerRNA: A generative pre-trained language model for de novo RNA design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Y.; Oono, K.; Takizawa, H.; Kotera, M.</p>
        <p class="info">Score: 22.4, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578496' target='https://doi.org/10.1101/2024.02.01.578496'> 10.1101/2024.02.01.578496</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe design of RNA plays a crucial role in developing RNA vaccines, nucleic acid therapeutics, and innovative biotechnological tools. Nevertheless, existing techniques lack versatility across various tasks and frequently suffer from a deficiency of automated generation. Inspired by the remarkable success of Large Language Models (LLMs) in the realm of protein and molecule design, we present GenerRNA, the first large-scale pre-trained model for RNA generation, aiming to further automate RNA design. Our approach eliminates the need for secondary structure or other prior knowledge and is capable of de novo generation of RNA with stable secondary structures while ensuring its distinctiveness from existing sequences. This widens our exploration of RNA space, thereby enriching our understanding of RNA structures and functions. Moreover, GenerRNA is fine-tunable on smaller, more specialized datasets for particular subtasks. This flexibility and versatility enables the generation of RNAs with desired specific functionalities or properties. Upon fine-tuning GenerRNA, we successfully generated novel RNA sequences exhibiting high affinity for target proteins. GenerRNA is freely available at the following repository: https://github.com/pfnet-research/GenerRNA</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579791" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579791">
        <p class="paperTitle">Proteus: pioneering protein structure generation for enhanced designability and efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, C.; Qu, Y.; Peng, Z.; Wang, Y.; Zhu, H.; Chen, D.; Cao, L.</p>
        <p class="info">Score: 16.5, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579791' target='https://doi.org/10.1101/2024.02.10.579791'> 10.1101/2024.02.10.579791</a></p>
        <p class="abstract">The development of de novo protein design method is crucial for widespread applications in biology and chemistry. Protein backbone diffusion aims to generate designable protein structures with high efficiency. Although there have been great strides in protein structure prediction, applying these methods to protein diffusion has been challenging and inefficient. We introduce Proteus, an innovative approach that uses graph-based triangle methods and a multi-track interaction network. Proteus demonstrated cutting-edge designability and efficiency in computational evaluations. We tested the reliability of the model by experimental characterization. Our analysis indicates that from both computational and experimental perspectives, it is able to generate proteins with a remarkably high success rate. We believe Proteus&#39;s capacity to rapidly create highly designable protein backbone without the need for pre-training techniques will greatly enhance our understanding of protein structure diffusion and contribute to advances in protein design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579780">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579780" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579780">
        <p class="paperTitle">Cellpose3: one-click image restoration for improved cellular segmentation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579780" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579780" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stringer, C.; Pachitariu, M.</p>
        <p class="info">Score: 16.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579780' target='https://doi.org/10.1101/2024.02.10.579780'> 10.1101/2024.02.10.579780</a></p>
        <p class="abstract">Generalist methods for cellular segmentation have good out-of-the-box performance on a variety of image types. However, existing methods struggle for images that are degraded by noise, blurred or undersampled, all of which are common in microscopy. We focused the development of Cellpose3 on addressing these cases, and here we demonstrate substantial out-of-the-box gains in segmentation and image quality for noisy, blurry or undersampled images. Unlike previous approaches, which train models to restore pixel values, we trained Cellpose3 to output images that are well-segmented by a generalist segmentation model, while maintaining perceptual similarity to the target images. Furthermore, we trained the restoration models on a large, varied collection of datasets, thus ensuring good generalization to user images. We provide these tools as &#34;one-click&#34; buttons inside the graphical interface of Cellpose as well as in the Cellpose API.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.06.531268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.06.531268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.06.531268">
        <p class="paperTitle">Analysis of multi-condition single-cell data with latent embedding multivariate regression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.06.531268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.06.531268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahlmann-Eltze, C.; Huber, W.</p>
        <p class="info">Score: 138.4, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.06.531268' target='https://doi.org/10.1101/2023.03.06.531268'> 10.1101/2023.03.06.531268</a></p>
        <p class="abstract">Identifying gene expression differences in heterogeneous samples across multiple experimental or observational conditions is a fundamental biological task, enabled by single-cell resolution assays such as multi-condition sc-RNA-seq. Current approaches divide the constituent cells into discrete clusters meant to represent cell types, and identify differentially expressed genes for each cluster. However, such discrete categorization is often unsatisfactory: if too coarse, more subtle but important differences may be overlooked, if too fine-grained, statistical power is lost and results become unwieldy. Here, we introduce Latent Embedding Multivariate Regression (LEMUR), which enables differential expression analysis using a continuous lowdimensional latent space representation of cell type and state diversity, and thus operates without (or before) commitment to discrete categorization. LEMUR (1) aligns the data from the different conditions, (2) predicts how a cells gene expression changes as a function of the conditions and its position in latent space, and (3) identifies compact neighborhoods of cells with consistent differential expression for each gene. Unlike static clusters, these neighborhoods adapt to the underlying expression changes per gene. We validate LEMUR on a large compendium of single-cell datasets and show applications to the identification of subpopulations of tumor cells with distinct drug responses, to the characterization of the interplay between cell state and developmental time in Zebrafish embryos, and to the discovery of cell state-specific gene regulation in a spatial single-cell dataset of Alzheimers plaques.

Software availabilityhttps://bioconductor.org/packages/lemur

Contactconstantin.ahlmann@embl.de</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.576026">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.576026" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.576026">
        <p class="paperTitle">BiaPy: A unified framework for versatile bioimage analysis with deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.576026" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.576026" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Franco-Barranco, D.; Andres-San Roman, J. A.; Hidalgo-Cenalmor, I.; Backova, L.; Gonzalez-Marfil, A.; Caporal, C.; Chessel, A.; Gomez-Galvez, P.; Escudero, L. M.; Wei, D.; Munoz-Barrutia, A.; Arganda-Carreras, I.</p>
        <p class="info">Score: 25.9, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.576026' target='https://doi.org/10.1101/2024.02.03.576026'> 10.1101/2024.02.03.576026</a></p>
        <p class="abstract">BiaPy, a unified open-source bioimage analysis library, offers a comprehensive suite of deep learning-powered workflows. Tailored for users of all levels, BiaPy features an intuitive interface, zero-code notebooks, and Docker integration. With support for 2D and 3D image data, it addresses existing gaps by providing multi-GPU capabilities, memory optimization, and compatibility with large datasets. As a collaborative and accessible solution, BiaPy aims to empower researchers by democratizing the use of sophisticated and efficient bioimage analysis workflows.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579631">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579631" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579631">
        <p class="paperTitle">Investigating the performance of foundation models on human 3&#39;UTR sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579631" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579631" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vilov, S.; Heinig, M.</p>
        <p class="info">Score: 13.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579631' target='https://doi.org/10.1101/2024.02.09.579631'> 10.1101/2024.02.09.579631</a></p>
        <p class="abstract">Foundation models, such as DNABERT and Nucleotide Transformer have recently shaped a new direction in DNA research. Trained in an unsupervised manner on a vast quantity of genomic data, they can be used for a variety of downstream tasks, such as promoter prediction, DNA methylation prediction, gene network prediction or functional variant prioritization. However, these models are often trained and evaluated on entire genomes, neglecting genome partitioning into different functional regions. In our study, we investigate the efficacy of various unsupervised approaches, including genome-wide and 3&#39;UTR-specific foundation models on human 3&#39;UTR regions. Our evaluation includes downstream tasks specific for RNA biology, such as recognition of binding motifs of RNA binding proteins, detection of functional genetic variants, prediction of expression levels in massively parallel reporter assays, and estimation of mRNA half-life. Remarkably, models specifically trained on 3&#39;UTR sequences demonstrate superior performance when compared to the established genome-wide foundation models in three out of four downstream tasks. Our results underscore the importance of considering genome partitioning into functional regions when training and evaluating foundation models.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
